
<html lang="en"     class="pb-page"  data-request-id="4495bbca-f800-4521-a579-88d86e74f8e0"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";issue:issue:10.1021/jmcmar.2017.60.issue-22;requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.7b01320;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Discovery, Development, and Cellular Delivery of Potent and Selective Bicyclic Peptide Inhibitors of Grb7 Cancer Target" /></meta><meta name="dc.Creator" content="Gabrielle M.  Watson" /></meta><meta name="dc.Creator" content="Ketav  Kulkarni" /></meta><meta name="dc.Creator" content="Jianrong  Sang" /></meta><meta name="dc.Creator" content="Xiuquan  Ma" /></meta><meta name="dc.Creator" content="Menachem J.  Gunzburg" /></meta><meta name="dc.Creator" content="Patrick  Perlmutter" /></meta><meta name="dc.Creator" content="Matthew C.J.  Wilce" /></meta><meta name="dc.Creator" content="Jacqueline A.  Wilce" /></meta><meta name="dc.Description" content="Grb7 is a signaling protein with critical roles in tumor cell proliferation and migration and an established cancer therapeutic target. Here we explore chemical space to develop a new bicyclic pept..." /></meta><meta name="Description" content="Grb7 is a signaling protein with critical roles in tumor cell proliferation and migration and an established cancer therapeutic target. Here we explore chemical space to develop a new bicyclic pept..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="November 10, 2017" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.7b01320" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2017 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.7b01320" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.7b01320" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.7b01320" /></link>
        
    
    

<title>Discovery, Development, and Cellular Delivery of Potent and Selective Bicyclic Peptide Inhibitors of Grb7 Cancer Target | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.7b01320" /></meta><meta property="og:title" content="Discovery, Development, and Cellular Delivery of Potent and Selective Bicyclic Peptide Inhibitors of Grb7 Cancer Target" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01320/20171117/images/large/jm-2017-01320h_0006.jpeg" /></meta><meta property="og:description" content="Grb7 is a signaling protein with critical roles in tumor cell proliferation and migration and an established cancer therapeutic target. Here we explore chemical space to develop a new bicyclic peptide inhibitor, incorporating thioether and lactam linkers that binds with affinity (KD = 1.1 μM) and specificity to the Grb7–SH2 domain. Structural analysis of the Grb7–SH2/peptide complex revealed an unexpected binding orientation underlying the binding selectivity by this new scaffold. We further incorporated carboxymethylphenylalanine and carboxyphenylalanine phosphotyrosine mimetics and arrived at an optimized inhibitor that potently binds Grb7–SH2 (KD = 0.13 μM) under physiological conditions. X-ray crystal structures of these Grb7–SH2/peptide complexes reveal the structural basis for the most potent and specific inhibitors of Grb7 developed to date. Finally, we demonstrate that cell permeable versions of these peptides successfully block Grb7 mediated interactions in a breast cancer cell line, establishing the potential of these peptides in the development of novel therapeutics targeted to Grb7." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.7b01320"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.7b01320">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.7b01320&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.7b01320&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.7b01320&amp;href=/doi/10.1021/acs.jmedchem.7b01320" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2017</span><span class="cit-fg-volume">, 60</span><span class="cit-fg-issue">, 22</span><span class="cit-fg-pageRange">, 9349-9359</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/60/22" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.7b01313" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.7b01356" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Discovery, Development, and Cellular Delivery of Potent and Selective Bicyclic Peptide Inhibitors of Grb7 Cancer Target</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Gabrielle+M.++Watson">Gabrielle M. Watson</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Ketav++Kulkarni">Ketav Kulkarni</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jianrong++Sang">Jianrong Sang</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Xiuquan++Ma">Xiuquan Ma</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Menachem+J.++Gunzburg">Menachem J. Gunzburg</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Patrick++Perlmutter">Patrick Perlmutter</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Matthew+C.J.++Wilce">Matthew C.J. Wilce</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jacqueline+A.++Wilce">Jacqueline A. Wilce</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="loa-info-orcid"><a href="http://orcid.org/0000-0002-8344-2626" title="Orcid link"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img></a></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-symbol">†</span> <span class="aff-text">Biomedicine Discovery Institute, Department of Biochemistry and Molecular Biology, Monash University, Wellington Road, Clayton, VIC 3800, Australia</span></div><div class="aff-info" id="aff2"><span class="aff-symbol">‡</span> <span class="aff-text">Department of Physiology, School of Medicine, Jiangsu University, Zhenjiang, Jiangsu 212013, P.R. China</span></div><div class="aff-info" id="aff3"><span class="aff-symbol">§</span> <span class="aff-text">School of Chemistry, Monash University, Wellington Road, Clayton, VIC 3800, Australia</span></div><div class="corresp-info"><strong>*</strong>Jacqueline Wilce. E-mail: <a href="/cdn-cgi/l/email-protection#d6bcb7b5bdbfb3f8a1bfbab5b396bbb9b8b7a5bef8b3b2a3"><span class="__cf_email__" data-cfemail="1e747f7d75777b306977727d7b5e7371707f6d76307b7a6b">[email protected]</span></a>. Phone: + 613 9902 9226. Fax: + 613 9902 9500.</div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.7b01320&amp;href=/doi/10.1021%2Facs.jmedchem.7b01320" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2017</span></span><span class="cit-volume">, 60</span><span class="cit-issue">, 22</span><span class="cit-pageRange">, 9349–9359</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">October 30, 2017</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>7 September 2017</li><li><span class="item_label"><b>Published</b> online</span>10 November 2017</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 22 November 2017</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.7b01320" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b01320</a></div><div class="article_header-article-copyright"><strong>Copyright © 2017 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D9349%26pageCount%3D11%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DGabrielle%2BM.%2BWatson%252C%2BKetav%2BKulkarni%252C%2BJianrong%2BSang%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D60%26issueNum%3D22%26contentID%3Dacs.jmedchem.7b01320%26title%3DDiscovery%252C%2BDevelopment%252C%2Band%2BCellular%2BDelivery%2Bof%2BPotent%2Band%2BSelective%2BBicyclic%2BPeptide%2BInhibitors%2Bof%2BGrb7%2BCancer%2BTarget%26numPages%3D11%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D9359%26publicationDate%3DNovember%2B2017">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.7b01320"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1481</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">13</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.7b01320" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Discovery, Development, and Cellular Delivery of Potent and Selective Bicyclic Peptide Inhibitors of Grb7 Cancer Target&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Gabrielle&quot;,&quot;last_name&quot;:&quot;M. Watson&quot;},{&quot;first_name&quot;:&quot;Ketav&quot;,&quot;last_name&quot;:&quot;Kulkarni&quot;},{&quot;first_name&quot;:&quot;Jianrong&quot;,&quot;last_name&quot;:&quot;Sang&quot;},{&quot;first_name&quot;:&quot;Xiuquan&quot;,&quot;last_name&quot;:&quot;Ma&quot;},{&quot;first_name&quot;:&quot;Menachem&quot;,&quot;last_name&quot;:&quot;J. Gunzburg&quot;},{&quot;first_name&quot;:&quot;Patrick&quot;,&quot;last_name&quot;:&quot;Perlmutter&quot;},{&quot;first_name&quot;:&quot;Matthew&quot;,&quot;last_name&quot;:&quot;C.J. Wilce&quot;},{&quot;first_name&quot;:&quot;Jacqueline&quot;,&quot;last_name&quot;:&quot;A. Wilce&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2017&quot;,&quot;month&quot;:&quot;11&quot;,&quot;day&quot;:&quot;10&quot;,&quot;issue&quot;:&quot;22&quot;,&quot;volume&quot;:&quot;60&quot;,&quot;pages&quot;:&quot;9349-9359&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.7b01320&quot;},&quot;abstract&quot;:&quot;Grb7 is a signaling protein with critical roles in tumor cell proliferation and migration and an established cancer therapeutic target. Here we explore chemical space to develop a new bicyclic peptide inhibitor, incorporating thioether and lactam linkers that binds with affinity (KD = 1.1 μM) and specificity to the Grb7–SH2 domain. Structural analysis of the Grb7–SH2/peptide complex revealed an unexpected binding orientation underlying the binding selectivity by this new scaffold. We further incorporated carboxymethylphenylalanine and carboxyphenylalanine phosphotyrosine mimetics and arrived at an optimized inhibitor that potently binds Grb7–SH2 (KD = 0.13 μM) under physiological conditions. X-ray crystal structures of these Grb7–SH2/peptide complexes reveal the structural basis for the most potent and specific inhibitors of Grb7 developed to date. Finally, we demonstrate that cell permeable versions of these peptides successfully block Grb7 mediated interactions in a breast cancer cell line, establishing the&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b01320&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b01320" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b01320&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b01320" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b01320&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b01320" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.7b01320&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b01320&amp;href=/doi/10.1021/acs.jmedchem.7b01320" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.7b01320" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.7b01320" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (5 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b01320&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.7b01320%26sid%3Dliteratum%253Aachs%26pmid%3D29083893%26genre%3Darticle%26aulast%3DWatson%26date%3D2017%26atitle%3DDiscovery%252C%2BDevelopment%252C%2Band%2BCellular%2BDelivery%2Bof%2BPotent%2Band%2BSelective%2BBicyclic%2BPeptide%2BInhibitors%2Bof%2BGrb7%2BCancer%2BTarget%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D60%26issue%3D22%26spage%3D9349%26epage%3D9359%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (3)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292258" title="Cancer">Cancer</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=290831" title="Monomers">Monomers</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=290672" title="Phosphates">Phosphates</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/60/22" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/jmcmar.2017.60.issue-22/20171122/jmcmar.2017.60.issue-22.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01320/20171117/images/medium/jm-2017-01320h_0006.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01320/20171117/images/large/jm-2017-01320h_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01320&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Grb7 is a signaling protein with critical roles in tumor cell proliferation and migration and an established cancer therapeutic target. Here we explore chemical space to develop a new bicyclic peptide inhibitor, incorporating thioether and lactam linkers that binds with affinity (<i>K</i><sub>D</sub> = 1.1 μM) and specificity to the Grb7–SH2 domain. Structural analysis of the Grb7–SH2/peptide complex revealed an unexpected binding orientation underlying the binding selectivity by this new scaffold. We further incorporated carboxymethylphenylalanine and carboxyphenylalanine phosphotyrosine mimetics and arrived at an optimized inhibitor that potently binds Grb7–SH2 (<i>K</i><sub>D</sub> = 0.13 μM) under physiological conditions. X-ray crystal structures of these Grb7–SH2/peptide complexes reveal the structural basis for the most potent and specific inhibitors of Grb7 developed to date. Finally, we demonstrate that cell permeable versions of these peptides successfully block Grb7 mediated interactions in a breast cancer cell line, establishing the potential of these peptides in the development of novel therapeutics targeted to Grb7.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_24322" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_24322" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Targeting protein–protein interactions (PPIs) in signaling pathways is an attractive strategy in the development of anticancer therapeutics.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> Signaling pathways regulating fundamental cellular processes (such as proliferation, migration, and survival) utilize PPIs to rapidly and specifically transmit external signals to the cell, and it is often the deregulation of these pathways that leads to the development of tumors and their advancement into a metastatic state.<a onclick="showRef(event, 'ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref2 ref3">(2, 3)</a> However, targeting PPIs requires the development of inhibitors able to form interactions with large and relatively featureless protein surfaces. Here, the use of peptides can overcome this challenge by spatially and functionally mimicking the natural binding partner, leading to potent and selective inhibitors.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a></div><div class="NLM_p">Growth factor receptor bound protein 7 (Grb7) is an intracellular signaling protein with critical roles in cancer cell proliferation and migration that has been found to be overexpressed in cancers of the breast, pancreas, ovaries, and esophagus.<a onclick="showRef(event, 'ref5 ref6 ref7 ref8'); return false;" href="javascript:void(0);" class="ref ref5 ref6 ref7 ref8">(5-8)</a> Grb7 propagates proliferation and survival signals from phosphorylated tyrosine kinases including HER2/3, the well-known predictor of poor prognosis in breast cancer patients.<a onclick="showRef(event, 'ref9 ref10'); return false;" href="javascript:void(0);" class="ref ref9 ref10">(9, 10)</a> Grb7 also interacts with focal adhesion kinase (FAK) promoting cell migration and thus, when overexpressed, contributes to the metastatic potential of the cell.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> Overexpression of Grb7 has now been identified as a more significant predictor than HER2 overexpression for reduced cancer-free periods and a worse prognosis for breast cancer patients.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> Consistent with its effects when overexpressed, Grb7 knockdown results in inhibited proliferation and growth in HER2+ breast cancer cell lines,<a onclick="showRef(event, 'ref10 ref13'); return false;" href="javascript:void(0);" class="ref ref10 ref13">(10, 13)</a> as well as inhibited migration, invasion, and colony growth in triple negative breast cancer cell lines.<a onclick="showRef(event, 'ref14 ref15'); return false;" href="javascript:void(0);" class="ref ref14 ref15">(14, 15)</a> Grb7 is thus identified as a therapeutic target for the treatment of a number of cancers, including HER2+ breast cancer and triple negative breast cancer.</div><div class="NLM_p">Grb7 is a 532 amino acid modular protein consisting of an N-terminal proline-rich region, a central GM (for <u class="uu">G</u>rb and <u class="uu">M</u>ig) domain, and a C-terminal SH2 domain. The central GM region encompasses the Ras-associating (RA) domain, pleckstrin homology (PH) domain, and the <u class="uu">b</u>etween <u class="uu">P</u>H and <u class="uu">S</u>H2 (BPS) domain. It is the C-terminal SH2 domain that interacts with the HER2/3 receptors and the focal adhesion kinase (FAK) through binding phosphorylated tyrosines (pY), and it is therefore this domain that is currently being targeted for therapeutic development.<a onclick="showRef(event, 'ref16 ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref16 ref17 ref18">(16-18)</a></div><div class="NLM_p">A nonphosphorylated thioether-linked cyclic peptide <b>1</b>, named G7-18NATE (cyclo-(CH<sub>2</sub>CO-WFEGYDNTFPC)-amide), was identified via phage display to inhibit interactions between Grb7 and HER3 in breast cancer cell extracts.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> When peptide <b>1</b> was attached to the penetratin cell permeability sequence (<b>1P</b>), it was shown to inhibit migration and proliferation in breast and pancreatic cancer cell lines, as well as reduce tumor metastasis in a pancreatic mouse model derived from Grb7 overexpressing cells.<a onclick="showRef(event, 'ref7 ref10 ref14'); return false;" href="javascript:void(0);" class="ref ref7 ref10 ref14">(7, 10, 14)</a> Furthermore, peptide <b>1P</b> (as well as other cell permeable forms of peptide <b>1</b>) was identified to have synergistic effects with the currently available therapeutics doxorubicin and trastuzumab in inhibiting proliferation and migration in breast cancer cell lines.<a onclick="showRef(event, 'ref10 ref19'); return false;" href="javascript:void(0);" class="ref ref10 ref19">(10, 19)</a> Subsequent biophysical analyses of the Grb7–SH2/peptide <b>1</b> interaction identified, however, that the interaction was only of modest affinity with a <i>K</i><sub>D</sub> of 4.1 μM in a buffer of high phosphate concentration (50 mM) and 18.1 μM in buffer containing a physiological concentration of phosphate (1 mM).<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> To improve on this, the peptide backbone was further constrained through the introduction of a second linker (via incorporation of an <i>O</i>-allylserine-based olefin linkage) and removal of two amino acids residues to arrive at peptide <b>2</b> (cyclo-(CH<sub>2</sub>CO-XFEGYDNXC)-amide where X= <i>O</i>-allylserine) with a <i>K</i><sub>D</sub> of 0.83 μM in high phosphate concentration buffer.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> Interestingly, the X-ray crystal structure of the Grb7–SH2/peptide <b>2</b> complex revealed that the cyclic inhibitor bound with a rearrangement of the peptide interaction surface.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> This unexpected orientation exemplifies one of the challenges of rational inhibitor design.</div><div class="NLM_p last">Building on the success of this bicyclic peptide strategy, we then embarked on further exploration of chemical space to discover even more potent Grb7–SH2 domain bicyclic inhibitors. In this report, we describe the synthesis and analysis of triazole- (peptide <b>3</b>) and lactam-linked (peptides <b>4</b> and <b>5</b>) bicyclic Grb7–SH2 inhibitors that are specific for Grb7 and achieve binding affinities up to <i>K</i><sub>D</sub> = 270 nM. The X-ray crystal structure of the highest affinity peptide <b>5</b>/Grb7–SH2 complex revealed, yet again, a rearranged mode of peptide binding and identified the importance of engaging the pY binding pocket of the Grb7–SH2 domain. This led to the further modification of the bicyclic inhibitors to incorporate carboxymethylphenylalanine (cmF) or carboxyphenylalanine (cF) pY mimetics (peptides <b>6</b> and <b>7</b>). These inhibitors bound Grb7–SH2 with affinities up to <i>K</i><sub>D</sub> = 130 nM, and the X-ray crystal structures provided insight into the molecular basis for the enhanced affinity of the slightly shorter cF pY mimetic. Finally, cell-permeable versions of the optimized peptide inhibitors were shown to block Grb7 interactions with HER2, SHC, and FAK upstream binding partners in a HER2+ breast cancer cell line. Together this work demonstrates the rational structure-based progression toward the most potent Grb7–SH2 domain peptides developed to date. These peptides represent invaluable tools for understanding Grb7 function and are a significant advancement in the development of Grb7-targeted anticancer agents.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_41167" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_41167" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i3" class="anchor-spacer"></div><h3 class="article-section__title" id="_i3"> Redesign of Bicyclic Grb7 Targeting Inhibitors</h3><div class="NLM_p">The X-ray crystal structure of the Grb7–SH2/peptide <b>2</b> complex was recently solved to 2.47 Å resolution and revealed the structural basis for the <i>K</i><sub>D</sub> = 0.83 μM binding affinity observed between the bicyclic peptide and the Grb7–SH2 domain (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EEL">5EEL</a>).<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> Note that the nomenclature used to describe SH2 domain secondary structure is as derived by Eck et al. (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>A).<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> For clarity SH2 domain amino acids will be denoted by the three letter code, and peptide amino acids by the single letter code. Close inspection of this X-ray structure revealed that Asp497 in the Grb7–SH2 EF loop is surface exposed and <4 Å in distance from the bicyclic hydrocarbon staple (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>A).</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01320/20171117/images/medium/jm-2017-01320h_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01320/20171117/images/large/jm-2017-01320h_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Redesigning Grb7 targeted bicyclics. (A) X-ray crystal structure of peptide <b>2</b> (blue sticks) bound to the Grb7–SH2 (gray cartoon and surface representation) identifying a surface exposed aspartic acid (orange sticks) in the Grb7–SH2 EF loop (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EEL">5EEL</a>), <b>(</b>B<b>)</b> Schematic and chemical structure of peptides <b>1</b>–<b>5</b>. For the schematics, the amino acid code is displayed in color and the remainder of the schematic conveys the chemical structure including the amidated C-terminus.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01320/20171117/images/large/jm-2017-01320h_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01320&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">In light of the structural information, three new bicyclic peptides were synthesized that incorporated electron accepting functionalities that were predicted to interact with the Asp497 (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>B). These peptides all retained the thioether linker and core interaction amino acids (FEGYDN) present in peptide <b>2</b> but differed in their linkage chemistry between residues 1 and 8. Peptide <b>3</b> incorporated propargyl and azido functionalities at positions 1 and 8, respectively, and was cyclized via click chemistry to produce a triazole linkage. Peptides <b>4</b> and <b>5</b> incorporated glutamic acid and lysine amino acids and were cyclized through lactamization of the E and K side chains. All three peptides were synthesized via Fmoc-solid phase peptide synthesis on Rink amide resin, and the thioether linkage was introduced postcleavage between the N-terminal chloroacetyl group and the C-terminal side chain thiol.</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Improved Potency by Incorporating a Lactam Macrocycle</h3><div class="NLM_p">To test the binding affinity of the newly synthesized peptides for Grb7–SH2, we utilized surface plasmon resonance (SPR) under the same experimental conditions as previously reported.<a onclick="showRef(event, 'ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref17 ref18">(17, 18)</a> The triazole containing peptide <b>3</b> and lactam containing peptide <b>4</b> bound to Grb7–SH2 with affinities of <i>K</i><sub>D</sub> = 3.03 and 2.56 μM, a 3.7-fold and 3.1-fold loss of binding affinity, respectively, compared with peptide <b>2</b> (<i>K</i><sub>D</sub> = 0.83 μM; <a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>A; <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>).<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> While peptide <b>4</b> displayed a classical rapid association/dissociation binding profile, similar to peptide <b>1</b>,<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> peptide <b>3</b> displayed a two-step association and two-step dissociation binding profile that suggested nonspecific and irreversible binding (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>B,C).</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01320/20171117/images/medium/jm-2017-01320h_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01320/20171117/images/large/jm-2017-01320h_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Binding studies of bicyclic inhibitors for Grb7–SH2. (A) Equilibrium binding curves for peptides <b>3</b>–<b>5</b> binding to Grb7–SH2 in high phosphate buffer (50 mM, +PO<sub>4</sub><sup>3–</sup>), (B–D) corresponding SPR sensorgrams, (E) equilibrium binding curves for peptides <b>3</b>–<b>5</b> binding to the Grb7–SH2 in low phosphate buffer (1 mM; −PO<sub>4</sub><sup>3–</sup>), (F–H) Representative SPR sensorgrams. The peptide concentrations utilized are listed on the left of the sensorgrams, with lower concentrations omitted for clarity.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01320/20171117/images/large/jm-2017-01320h_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01320&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Binding Parameters of Peptides <b>3–5</b> for the Grb7–SH2 Domain Measured in a High Phosphate (50 mM) Buffer</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="−" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">peptide</th><th class="colsep0 rowsep0" align="center" char="."><i>K</i><sub>D</sub><a class="ref internalNav" href="#t1fn1" aria-label="a">a</a> (μM)</th><th class="colsep0 rowsep0" align="center" char="−"><i>B</i><sub>max</sub>/theoretical <i>B</i><sub>max</sub><a class="ref internalNav" href="#t1fn2" aria-label="b">b</a> (%)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3</b></td><td class="colsep0 rowsep0" align="char" char=".">3.03 ± 0.05</td><td class="colsep0 rowsep0" align="char" char="−">67–78</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4</b></td><td class="colsep0 rowsep0" align="char" char=".">2.56 ± 0.05</td><td class="colsep0 rowsep0" align="char" char="−">46–64</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5</b></td><td class="colsep0 rowsep0" align="char" char=".">0.27 ± 0.001</td><td class="colsep0 rowsep0" align="char" char="−">54–64</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup>a</sup><p class="last">Equilibrium dissociation constants calculated from fits by a single-site binding model. The errors shown are the standard error arising from the fits.</p></div><div class="footnote" id="t1fn2"><sup>b</sup><p class="last">The range of <i>B</i><sub>max</sub> values that were derived from fits by a single-site binding model are displayed as a % of the calculated theoretical <i>B</i><sub>max</sub>.</p></div></div></div><div class="NLM_p">The last peptide in the series, <b>5</b>, also utilized lactam chemistry, but with the reverse orientation compared to peptide <b>4</b>. Surprisingly it was found to bind to the Grb7–SH2 with the highest affinity that has been described to date, with a <i>K</i><sub>D</sub> of 0.27 μM (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>A). The Grb7–SH2/peptide <b>5</b> interaction displayed a slowed dissociation rate, indicating a more stable complex, consistent with the measured increased binding affinity (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>D).</div><div class="NLM_p">The aforementioned experiments were all conducted in a buffer containing 50 mM phosphate, a higher concentration than would be present in a physiological setting.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> A previous study identified the effect of phosphate on the binding affinity of peptide <b>1</b>, with the <i>K</i><sub>D</sub> of the interaction decreasing >4-fold when the phosphate concentration was decreased to 1 mM (a concentration representative of physiological levels).<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> As peptides <b>3</b>–<b>5</b> were not designed to engage the pY binding pocket, this reliance on high concentrations of phosphate was also anticipated for these peptides. To determine this, the SPR experiments were repeated in a buffer containing 1 mM phosphate. As expected, binding of all three inhibitors to the Grb7–SH2 decreased >2.5-fold as evidenced by a shift to the right in the binding curve (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>E; <a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>). While the sensorgrams did not markedly differ for the peptide <b>3</b> and <b>4</b> interactions with the Grb7–SH2, the Grb7–SH2/peptide <b>5</b> interaction no longer displayed the slower off-rate (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>F–H). This suggested that the engagement of the pY binding pocket via a phosphate was required for formation of the more stable complex.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Binding Parameters of Peptides <b>3–5</b> for the Grb7–SH2 Domain in a Buffer Reflecting Physiological Phosphate Conditions (1 mM)</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="−" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">peptide</th><th class="colsep0 rowsep0" align="center" char="."><i>K</i><sub>D</sub><a class="ref internalNav" href="#t2fn1" aria-label="a">a</a> (μM)</th><th class="colsep0 rowsep0" align="center" char="−"><i>B</i><sub>max</sub>/theoretical <i>B</i><sub>max</sub><a class="ref internalNav" href="#t2fn2" aria-label="b">b</a> (%)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3</b></td><td class="colsep0 rowsep0" align="char" char=".">7.81 ± 0.07</td><td class="colsep0 rowsep0" align="char" char="−">55</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4</b></td><td class="colsep0 rowsep0" align="char" char=".">15.7 ± 0.06</td><td class="colsep0 rowsep0" align="char" char="−">37</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5</b></td><td class="colsep0 rowsep0" align="char" char=".">1.10 ± 0.01</td><td class="colsep0 rowsep0" align="char" char="−">29–39</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup>a</sup><p class="last">Equilibrium dissociation constants calculated from fits by a single-site binding model. The errors shown are the standard error arising from the fits.</p></div><div class="footnote" id="t2fn2"><sup>b</sup><p class="last">The range of <i>B</i><sub>max</sub> values that were derived from fits by a single-site binding model are displayed as a % of the calculated theoretical <i>B</i><sub>max.</sub></p></div></div></div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> The Crystal Structure of the Grb7–SH2/Peptide <b>5</b> Complex Revealed an Unexpected Binding Orientation of the Lactam</h3><div class="NLM_p">To understand the structural basis for the improved affinity of the Grb7–SH2/peptide <b>5</b> interaction, we solved the X-ray crystal structure of the complex to 2.1 Å (<a class="ref internalNav" href="#tbl3" aria-label="Table 3">Table 3</a>). Grb7–SH2 adopted the canonical SH2 domain arrangement with peptide <b>5</b> positioned unambiguously across the βD strand (<a href="/doi/suppl/10.1021/acs.jmedchem.7b01320/suppl_file/jm7b01320_si_001.pdf" class="ext-link">Figure S1</a>). The core Grb7–SH2 binding motif (YXN) of peptide <b>5</b> maintained similar hydrogen bonding interactions as have been observed previously for Grb7–SH2/peptide interactions.<a onclick="showRef(event, 'ref17 ref18 ref23'); return false;" href="javascript:void(0);" class="ref ref17 ref18 ref23">(17, 18, 23)</a> However, the structure revealed an unexpected positioning of the thioether linkage to the surface of the Grb7–SH2 domain and concomitant positioning of the lactam linker away from the surface (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>A,B). In this position, the C-terminal amide is able to interact with Gln499 in the EF loop (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>B). This binding mode is reminiscent of peptide <b>1</b> binding to the Grb7–SH2; however, the removal of F9 and P10 from peptide <b>1</b> enables <b>5</b> to have improved shape complementarity along the groove formed by the EF loop (<a href="/doi/suppl/10.1021/acs.jmedchem.7b01320/suppl_file/jm7b01320_si_001.pdf" class="ext-link">Figure S1</a>).</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01320/20171117/images/medium/jm-2017-01320h_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01320/20171117/images/large/jm-2017-01320h_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. X-ray crystal structure of the Grb7–SH2/peptide <b>5</b> complex and specificity analysis. (A, B) Surface representations of the Grb7–SH2 (gray) with either bound peptide <b>2</b> (blue sticks) or peptide <b>5</b> (green sticks), (C, D) Close-up of the Grb7–SH2 pY binding pocket with key amino acids shown in stick format and hydrogen bonds displayed as dashed lines (PDB IDs <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EEL">5EEL</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5U1Q">5U1Q</a>). (E, F) SPR equilibrium binding curves for peptide <b>5</b> binding to the Grb7–SH2 (gray), Grb2–SH2 (wheat), and Grb10–SH2 (purple) in either high phosphate buffer (50 mM; +PO<sub>4</sub><sup>3–</sup>) or low phosphate buffer (1 mM; −PO<sub>4</sub><sup>3–</sup>). Representative sensorgrams are provided as <a href="/doi/suppl/10.1021/acs.jmedchem.7b01320/suppl_file/jm7b01320_si_001.pdf" class="ext-link">Supporting Information</a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01320/20171117/images/large/jm-2017-01320h_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01320&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. X-ray Data Statistics</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center">Grb7–SH2/peptide <b>5</b></th><th class="colsep0 rowsep0" align="center">Grb7–SH2/peptide <b>6</b></th><th class="colsep0 rowsep0" align="center">Grb7–SH2/peptide <b>7</b></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="4" align="left">Data Collection</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">wavelength (Å)</td><td class="colsep0 rowsep0" align="left">0.954</td><td class="colsep0 rowsep0" align="left">0.954</td><td class="colsep0 rowsep0" align="left">0.947</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">space group</td><td class="colsep0 rowsep0" align="left"><i>C</i>2</td><td class="colsep0 rowsep0" align="left"><i>P</i>2<sub>1</sub></td><td class="colsep0 rowsep0" align="left"><i>P</i>2<sub>1</sub></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">unit cell dimensions</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>a</i>, <i>b</i>, <i>c</i> (Å)</td><td class="colsep0 rowsep0" align="left">122.18, 47.09, 94.61</td><td class="colsep0 rowsep0" align="left">47.98, 109.74, 48.33</td><td class="colsep0 rowsep0" align="left">45.07, 107.61, 48.01</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">α, β, γ (deg)</td><td class="colsep0 rowsep0" align="left">90, 97.49, 90</td><td class="colsep0 rowsep0" align="left">90, 103.44, 90</td><td class="colsep0 rowsep0" align="left">90, 101.38, 90</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">resolution (Å)</td><td class="colsep0 rowsep0" align="left">40.49–2.10 (2.16–2.10)</td><td class="colsep0 rowsep0" align="left">35.74–2.10 (2.16–2.10)</td><td class="colsep0 rowsep0" align="left">47.07–2.15 (2.22–2.15)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>R</i><sub>merge</sub><a class="ref internalNav" href="#t3fn1" aria-label="a">a</a> (%)</td><td class="colsep0 rowsep0" align="left">5.8 (40.6)</td><td class="colsep0 rowsep0" align="left">13.8 (75.5)</td><td class="colsep0 rowsep0" align="left">12.0 (48.4)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Wilson B factor</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">22.96</td><td class="colsep0 rowsep0" align="left">24.21</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CC (1/2) (%)</td><td class="colsep0 rowsep0" align="left">99.7 (78.8)</td><td class="colsep0 rowsep0" align="left">99.6 (58.8)</td><td class="colsep0 rowsep0" align="left">85.2 (63.8)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>I</i>/σ<i>I</i></td><td class="colsep0 rowsep0" align="left">9.7 (2.0)</td><td class="colsep0 rowsep0" align="left">5.1 (1.7)</td><td class="colsep0 rowsep0" align="left">4.9 (1.8)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">unique reflns measd</td><td class="colsep0 rowsep0" align="left">30704 (2519)</td><td class="colsep0 rowsep0" align="left">28374 (2300)</td><td class="colsep0 rowsep0" align="left">24095 (2044)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">completeness (%)</td><td class="colsep0 rowsep0" align="left">97.7 (98.1)</td><td class="colsep0 rowsep0" align="left">99.9 (99.8)</td><td class="colsep0 rowsep0" align="left">98.7 (98.0)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">multiplicity</td><td class="colsep0 rowsep0" align="left">2.5 (2.6)</td><td class="colsep0 rowsep0" align="left">3.6 (3.6)</td><td class="colsep0 rowsep0" align="left">2.8 (2.9)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Refinement</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>R</i><sub>work</sub> (%)</td><td class="colsep0 rowsep0" align="left">19.66 (28.62)</td><td class="colsep0 rowsep0" align="left">21.79 (29.55)</td><td class="colsep0 rowsep0" align="left">20.97 (25.35)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>R</i><sub>free</sub> (%)</td><td class="colsep0 rowsep0" align="left">24.95 (35.98)</td><td class="colsep0 rowsep0" align="left">26.09 (34.48)</td><td class="colsep0 rowsep0" align="left">22.05 (28.26)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">no. of atoms</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">macromolecules</td><td class="colsep0 rowsep0" align="left">3511</td><td class="colsep0 rowsep0" align="left">3595</td><td class="colsep0 rowsep0" align="left">3433</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ligands</td><td class="colsep0 rowsep0" align="left">44</td><td class="colsep0 rowsep0" align="left">101</td><td class="colsep0 rowsep0" align="left">97</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">solvent</td><td class="colsep0 rowsep0" align="left">135</td><td class="colsep0 rowsep0" align="left">191</td><td class="colsep0 rowsep0" align="left">122</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">mean B-factors (Å<sup>2</sup>)</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">macromolecules</td><td class="colsep0 rowsep0" align="left">41.25</td><td class="colsep0 rowsep0" align="left">27.02</td><td class="colsep0 rowsep0" align="left">27.27</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ligands</td><td class="colsep0 rowsep0" align="left">45.46</td><td class="colsep0 rowsep0" align="left">23.02</td><td class="colsep0 rowsep0" align="left">23.73</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">solvent</td><td class="colsep0 rowsep0" align="left">41.77</td><td class="colsep0 rowsep0" align="left">27.93</td><td class="colsep0 rowsep0" align="left">26.46</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">RMSDs</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">bond lengths (Å)</td><td class="colsep0 rowsep0" align="left">0.005</td><td class="colsep0 rowsep0" align="left">0.005</td><td class="colsep0 rowsep0" align="left">0.009</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">bond angles (deg)</td><td class="colsep0 rowsep0" align="left">0.80</td><td class="colsep0 rowsep0" align="left">0.69</td><td class="colsep0 rowsep0" align="left">1.12</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Ramachandran plot (%)</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">favored regions</td><td class="colsep0 rowsep0" align="left">98.4</td><td class="colsep0 rowsep0" align="left">98.1</td><td class="colsep0 rowsep0" align="left">97.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">allowed regions</td><td class="colsep0 rowsep0" align="left">1.6</td><td class="colsep0 rowsep0" align="left">1.9</td><td class="colsep0 rowsep0" align="left">2.7</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup>a</sup><p class="last"><i>R</i><sub>merge</sub> = ∑<sub><i>hkl</i></sub>∑<sub><i>i</i></sub>|<i>I</i><sub><i>i</i></sub>(<i>hkl</i>) – ⟨<i>I</i>(<i>hkl</i>)⟩|/∑<sub><i>hkl</i></sub>∑<sub><i>i</i></sub><i>I</i><sub><i>i</i></sub>(<i>hkl</i>), where <i>I</i><sub><i>i</i></sub>(<i>hkl</i>) is the <i>i</i>th intensity measurement of reflection <i>hkl</i> and ⟨<i>I</i>(<i>hkl</i>)⟩ is its average. Values given in parentheses are for the high resolution shell.</p></div></div></div><div class="NLM_p last">It was also noted that the BC loop of the Grb7–SH2 adopted two conformations in the asymmetric unit of the peptide <b>5</b> bound crystal structure. In two of the four chains, a chloride and water molecule were positioned in the pY-binding pocket, whereas the pY-binding pocket of the other two chains were “empty”. The chloride forms ionic bonds with the guanidino side chains of two essential SH2 domain pY-binding pocket residues, Arg438 and Arg458 (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>C). The adjacent water molecule in the binding pocket also coordinates the chloride, with the water forming hydrogen bonds with the Ser460 side chain hydroxyl and Gln461 backbone amine (from the Grb7–SH2) as well as the Y5 hydroxyl from the peptide <b>5</b>. In the remaining two chains of the Grb7–SH2 pY-binding pocket the coordinating chloride is not present and the BC loop exists in a more “open state” and is less engaged with the peptide (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>D). Thus, the Grb7–SH2/peptide <b>5</b> crystal structure provides insight into two possible conformational states of the Grb7–SH2 domain pY binding pocket, one in which electronegative ions act to coordinate additional interactions between the protein and ligand forming a “closed” conformation and one in which there is no electronegative ion and the BC loop exists in a more “open” conformation. This likely reflects the molecular basis for the enhanced peptide binding observed in the presence of higher phosphate concentration–with the highly electronegative phosphate ion effecting coordination between peptide and protein and promoting the “closed” conformation of the BC loop.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> The Lactam Peptide <b>5</b> Selectively Binds Grb7–SH2</h3><div class="NLM_p">As well as developing high affinity binding peptides, it was also important to assess whether the designed molecules maintained specificity for the Grb7–SH2 target. Establishing inhibitor specificity against the Grb7–SH2 is critical, with over 120 human proteins bearing SH2 domains. We chose to test binding against two closely related SH2 domains, Grb2 and Grb10, with Grb2 sharing the same preferential pYXN binding motif as Grb7, and Grb10 representing a member of the Grb7 family.<a onclick="showRef(event, 'ref24 ref25'); return false;" href="javascript:void(0);" class="ref ref24 ref25">(24, 25)</a> Using SPR, we determined that peptide <b>5</b> showed negligible binding to both Grb2–SH2 and Grb10–SH2 with binding responses that were too weak for equilibrium dissociation constants to be determined (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>E,F; <a href="/doi/suppl/10.1021/acs.jmedchem.7b01320/suppl_file/jm7b01320_si_001.pdf" class="ext-link">Figure S2</a>). The preferential binding to Grb7–SH2 was shown to occur regardless of the phosphate concentration, reinforcing that bicyclic peptide <b>5</b> is robustly preferential for the Grb7–SH2 domain. Peptides <b>3</b> and <b>4</b> also showed negligible binding to the Grb10–SH2 and Grb2–SH2 showing that the linkage chemistry used between residues 1 and 8 does not impact on peptide binding specificity (<a href="/doi/suppl/10.1021/acs.jmedchem.7b01320/suppl_file/jm7b01320_si_001.pdf" class="ext-link">Figure S2</a>).</div><div class="NLM_p last">A structural overlay of peptide <b>5</b> onto the Grb10–SH2 and Grb2–SH2 structures (solved in the absence of peptide ligand) provides possible insight into the specificity defining features of Grb7. It has previously been determined that the βD6 residue, Leu481, can dictate the ligand binding preference of the Grb7–SH2 domain.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> Structural analysis of the Grb7–SH2/peptide <b>5</b> complex clearly identifies the way in which Leu481 influences binding, with F2 and Y5 from peptide <b>5</b> straddling this hydrophobic residue (<a href="/doi/suppl/10.1021/acs.jmedchem.7b01320/suppl_file/jm7b01320_si_001.pdf" class="ext-link">Figure S3</a>). In contrast, the glutamine at the equivalent position in Grb10–SH2 would likely prevent binding due to steric clashes (<a href="/doi/suppl/10.1021/acs.jmedchem.7b01320/suppl_file/jm7b01320_si_001.pdf" class="ext-link">Figure S3</a>). This is also likely to occur for the Grb2–SH2 domain with a bulky lysine side chain potentially clashing with F2 positioning, as well as the Grb2–SH2 having an extended BG loop that could interfere with bicyclic peptide binding (<a href="/doi/suppl/10.1021/acs.jmedchem.7b01320/suppl_file/jm7b01320_si_001.pdf" class="ext-link">Figure S3</a>).</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Incorporating pY Mimetics Leads to Engagement of the Grb7–SH2 BC Loop and Removes the Requirement for Phosphate</h3><div class="NLM_p">Based on the structural insights from the Grb7–SH2/peptide <b>5</b> X-ray crystal structure, we hypothesized that replacing Y5 with a pY mimetic would enhance binding affinity and remove the requirement for high concentrations of phosphate. We have previously described the use of carboxymethylphenylalanine (cmF) or carboxyphenylalanine (cF) as pY mimetics;<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> therefore, we incorporated these same non-natural amino acids into the peptide <b>5</b> bicyclic backbone (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>A). The new pY containing bicyclic peptides, <b>6</b> (containing cmF) and <b>7</b> (containing cF), were tested for binding to the Grb7–SH2 in the presence of a physiological concentration of phosphate. Both peptides displayed enhanced affinity for the Grb7–SH2 with equilibrium dissociation constants of <i>K</i><sub>D</sub> = 220 nM and 130 nM, respectively, representing an approximately 80-fold and 140-fold improvement over with the lead inhibitor peptide <b>1</b> (<i>K</i><sub>D</sub> = 18.1 μM under the same conditions) (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>B; <a href="/doi/suppl/10.1021/acs.jmedchem.7b01320/suppl_file/jm7b01320_si_001.pdf" class="ext-link">Figure S4</a>; <a class="ref internalNav" href="#tbl4" aria-label="Table 4">Table 4</a>). Both peptides displayed the slowed dissociation rate that was described for peptide <b>5</b> binding to the Grb7–SH2 in a high phosphate buffer.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01320/20171117/images/medium/jm-2017-01320h_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01320/20171117/images/large/jm-2017-01320h_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Redesigning pY mimetic containing bicyclic peptides. (A) Chemical structure of peptides <b>6</b> and <b>7</b> with the pY mimetics (cmF and cF) highlighted by the pink and purple boxes, respectively. (B) SPR equilibrium binding curves of peptide <b>6</b> (top) and peptide <b>7</b> (bottom) binding to the Grb7–SH2 in both high (50 mM; +PO<sub>4</sub><sup>3–</sup>) and low (1 mM; −PO<sub>4</sub><sup>3–</sup>) phosphate containing buffer conditions. Representative sensorgrams are provided as <a href="/doi/suppl/10.1021/acs.jmedchem.7b01320/suppl_file/jm7b01320_si_001.pdf" class="ext-link">Supporting Information</a>. (C) Close-up of the Grb7–SH2 pY binding pocket (gray) highlighting peptide <b>6</b> (top) and peptide <b>7</b> (bottom) interacting residues (stick format). Waters are displayed as red spheres and hydrogen bonds as dashed lines. (D) Surface representation of the Grb7–SH2 (colored according to atom type: N = blue, O = red, S = yellow, C = gray), with either peptide <b>6</b> (top) or peptide <b>7</b> (bottom). Grb7–SH2/peptide <b>6</b> PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5U06">5U06</a>, Grb7–SH2/peptide <b>7</b> PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5TYI">5TYI</a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01320/20171117/images/large/jm-2017-01320h_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01320&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Binding Parameters of Peptides <b>6</b> and <b>7</b> for the Grb7–SH2 Measured in a High (50 mM; +PO<sub>4</sub><sup>3–</sup>) or Low (1 mM; −PO<sub>4</sub><sup>3–</sup>) Phosphate Concentration</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center" char=".">–PO<sub>4</sub><sup>3–</sup></th><th class="rowsep1 colsep0" colspan="2" align="center" char=".">+PO<sub>4</sub><sup>3–</sup></th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">peptide</th><th class="colsep0 rowsep0" align="center" char="."><i>K</i><sub>D</sub><a class="ref internalNav" href="#t4fn1" aria-label="a">a</a> (μM)</th><th class="colsep0 rowsep0" align="center" char="."><i>B</i><sub>max</sub>/theoretical <i>B</i><sub>max</sub><a class="ref internalNav" href="#t4fn2" aria-label="b">b</a> (%)</th><th class="colsep0 rowsep0" align="center" valign="top" char="."><i>K</i><sub>D</sub><a class="ref internalNav" href="#t4fn1" aria-label="a">a</a> (μM)</th><th class="colsep0 rowsep0" align="center" valign="top" char="."><i>B</i><sub>max</sub>/theoretical <i>B</i><sub>max</sub><a class="ref internalNav" href="#t4fn2" aria-label="b">b</a> (%)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6</b></td><td class="colsep0 rowsep0" align="char" char=".">0.22 ± 0.0004</td><td class="colsep0 rowsep0" align="char" char=".">30</td><td class="colsep0 rowsep0" align="char" char=".">2.94 ± 0.008</td><td class="colsep0 rowsep0" align="char" char=".">52</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7</b></td><td class="colsep0 rowsep0" align="char" char=".">0.13 ± 0.0004</td><td class="colsep0 rowsep0" align="char" char=".">32</td><td class="colsep0 rowsep0" align="char" char=".">0.83 ± 0.001</td><td class="colsep0 rowsep0" align="char" char=".">53</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup>a</sup><p class="last">Equilibrium dissociation constants calculated from fits by a single-site binding model. The errors shown are the standard error arising from the fits.</p></div><div class="footnote" id="t4fn2"><sup>b</sup><p class="last">The range of <i>B</i><sub>max</sub> values that were derived from fits by a single-site binding model are displayed as a % of the calculated theoretical <i>B</i><sub>max</sub>.</p></div></div></div><div class="NLM_p last">To deduce the effectiveness of the cmF and cF as pY mimetics, we also measured the binding affinity of the two peptides for the Grb7–SH2 in the high phosphate buffer (50 mM). As expected, the affinity decreased 14-fold and 6-fold, respectively, consistent with the phosphate outcompeting the peptides for Grb7–SH2 binding (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>B; <a href="/doi/suppl/10.1021/acs.jmedchem.7b01320/suppl_file/jm7b01320_si_001.pdf" class="ext-link">Figure S4</a>; <a class="ref internalNav" href="#tbl4" aria-label="Table 4">Table 4</a>). Thus, by mimicking the positioning of phosphate, the incorporation of the cmF and cF into the bicyclic backbone has resulted in potent Grb7–SH2 binding peptides under physiologically relevant buffer conditions.</div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> The cF Phosphotyrosine Mimetic Effectively Closes off the pY Binding Pocket</h3><div class="NLM_p">When the two pY mimetics were designed, it was anticipated that the cmF containing peptide (<b>6</b>) would have enhanced affinity for the Grb7–SH2 compared with the cF bearing peptide (<b>7</b>). The cF group was predicted to have a less ideal geometry than the cmF, with the absence of the methylene group that would spatially mimic the oxygen in a phosphotyrosine. However, our binding experiments indicated that peptide <b>7</b> was the higher affinity ligand. To understand the molecular basis for this unexpected finding, we determined the X-ray crystal structures of the Grb7–SH2/peptide <b>7</b> and Grb7–SH2/peptide <b>6</b> complexes (<a class="ref internalNav" href="#tbl3" aria-label="Table 3">Table 3</a>).</div><div class="NLM_p">The Grb7–SH2 domain in both structures adopted the classical fold, with peptides <b>6</b> and <b>7</b> unambiguously positioned across the βD strand as described for the Grb7–SH2/peptide <b>5</b> complex structure (<a href="/doi/suppl/10.1021/acs.jmedchem.7b01320/suppl_file/jm7b01320_si_001.pdf" class="ext-link">Figure S5</a>). The peptide <b>6</b> and peptide <b>7</b> backbones maintained similar positioning to peptide <b>5</b> with an α-carbon RMSD of 0.2 and 0.3 Å respectively, and formed the same interactions with Grb7–SH2 amino acids (hydrogen bonding with Arg438, His479, Leu481, and Met495) (<a href="/doi/suppl/10.1021/acs.jmedchem.7b01320/suppl_file/jm7b01320_si_001.pdf" class="ext-link">Figure S5</a>).</div><div class="NLM_p">As expected, the only difference between the three complex structures was in the engagement of the BC loop and interactions with the pY binding pocket. In the case of peptide <b>6</b>, the cmF group extends into the back of the pY binding pocket with the carboxylate forming bidentate salt bridges with Arg458 at the base of the pY binding pocket, as well as with the Arg438 guanidino group and hydrogen bonds with the Ser460 side-chain oxygen (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>C). Through these interactions, the BC loop is held in the optimal closed conformation that was described for the Grb7–SH2/peptide <b>5</b> structure via chloride and water ion interactions.</div><div class="NLM_p last">In contrast, the cF moiety of peptide <b>7</b> does not extend deeply into the pY pocket and instead forms hydrogen bonds with the Asn463 and Ser460 side chain oxygens that are closer to the entry of the binding pocket (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>C). A coordinating water hydrogen bonds with the cF carboxyl leading to engagement of Gln461 and Arg458. This positioning places the aromatic ring of cF in a more planar arrangement against the Grb7–SH2 binding surface and allows Arg462 to extend over the pY pocket via interactions with the G4 carbonyl (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>D). Thus, although both pY mimetics engage the BC loop and hold it in the optimal closed conformation, the shortened pY mimetic appears to facilitate the most stabilized complex with the Grb7–SH2 domain through additional favorable interactions.</div></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> The Optimized Inhibitors Successfully Block Grb7 Mediated Interactions in Breast Cancer Cells</h3><div class="NLM_p">We next sought to determine whether the optimized inhibitors were effective at disrupting Grb7 interactions with HER2, SHC, and FAK in a HER2+ breast cancer cell line.<a onclick="showRef(event, 'ref5 ref11'); return false;" href="javascript:void(0);" class="ref ref5 ref11">(5, 11)</a> To ensure cell entry, the peptides were conjugated to the cell penetrating peptide (CPP) penetratin (denoted by <b>P</b>). We also tested first generation monocyclic peptide <b>1</b> and its cF containing equivalent (peptide <b>8</b>) as a comparison.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> Whole cells were treated with peptide (to test their internalization) and washed, and then Grb7 immunoprecipitated from the cell lysate and associated partners were identified by Western blot (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>A). All of the penetratin conjugated peptides (<b>1P</b>, <b>5P</b>, <b>7P</b>, and <b>8P</b>) disrupted Grb7 interactions with SHC, HER2, and FAK to a similar extent (The full blots are provided in <a href="/doi/suppl/10.1021/acs.jmedchem.7b01320/suppl_file/jm7b01320_si_001.pdf" class="ext-link">Figure S6</a>).</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01320/20171117/images/medium/jm-2017-01320h_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01320/20171117/images/large/jm-2017-01320h_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Optimized inhibitors block Grb7 interactions with cellular partners. (A) SKBR3 cells were serum starved overnight and treated for 1 h with Grb7 targeting peptides (20 μM) prior to stimulation for 10 min with fibronectin (10 μg·mL<sup>–1</sup>). Washed cells were lysed and immunoprecipitated using Grb7 antibodies before being probed with various antibodies for the detection of Grb7 binding partners: SHC, top panel; HER2, second panel; FAK, third panel; Grb7 on the bottom panel served as a loading control. FN = fibronectin, P = penetratin control peptide. (B) As per panel A, with the exception that the Grb7 targeting peptides were added following cell lysis. Schematics of the penetratin coupled peptides tested are shown on the right of the image with the peptide sequence conveyed as single letter code and code displayed in color and the remainder of the schematic conveying the peptide chemical structure.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01320/20171117/images/large/jm-2017-01320h_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01320&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">To test whether there was any discrimination between the ability of the peptides to disrupt Grb7 interactions with binding partners more directly we also tested their ability to block Grb7 mediated interactions when added directly to cell lysate (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>B). The optimized inhibitors (<b>5P</b>, <b>7P</b> and <b>8P</b>) all blocked Grb7 mediated interactions to a greater extent than the original lead <b>1P</b> inhibitor. Thus, the incorporation of the lactam linker and/or phosphotyrosine mimetic has effectively translated into improved inhibitors of Grb7 interactions.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i14">Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_55422" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_55422" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The development of biological agents that target disease related PPIs is a burgeoning area in the field of therapeutic design.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> SH2 domains are an obvious target for the development of therapeutics with the negatively charged pY binding pocket, a distinctive feature for small molecules to bind. However, ensuring selectivity over other SH2 domains has led to larger ligands, such as peptides, being utilized in SH2 inhibitor design, with surfaces adjacent to the pY binding pocket helping to confer substrate specificity.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> Here, we have described the development of peptides that potently and selectively target the SH2 domain of the breast cancer target Grb7.</div><div class="NLM_p">Based on a combination of structural and binding information, we developed bicyclic peptide inhibitors that bind to the Grb7–SH2 with nanomolar affinity, representing an overall 140-fold improvement on the first-generation monocyclic inhibitor peptide <b>1</b>.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> We previously discovered a bicyclic peptide tethered by an olefin linkage that showed enhanced affinity over peptide <b>1</b>, likely due to a reduced entropic loss effect.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a></div><div class="NLM_p">Here we tested the effects of alternative linkage chemistries on the Grb7–SH2/peptide binding affinity using SPR. These were designed to maintain peptide rigidity but also include potential electron accepting functionalities that could form additional contacts with the Grb7–SH2 domain. This strategy identified that the lactam linker in peptide <b>5</b> facilitates an 18-fold increase to <i>K</i><sub>D</sub> = 1 μM (in physiologically relevant phosphate concentrations). Fascinatingly, this improvement was not seen for peptide <b>4</b> that only differed in the orientation of the lysine and glutamic acid lactam precursors within the amino acid sequence. To understand the structural basis for the improved interaction, we solved the X-ray crystal structure of the Grb7–SH2/peptide <b>5</b> complex. This structure revealed a novel binding mode adopted by the bicyclic peptide, with the thioether linkage forming part of the binding surface and the lactam linker not engaging the Grb7–SH2 as predicted. Thus, the basis for the peptide binding was revealed, but not the reason for the favored lactam orientation.</div><div class="NLM_p">Peptide stabilization through lactam linkers has been used previously in peptide design, mainly to constrain peptides in an α-helical conformation.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> Examples of the use of lactam constraints in other structural motifs are rarer but include successful application in the development of SH2 domain inhibitors. Inhibitors of Src–SH2 cyclized through a lactam linkage displayed >60-fold improved affinity compared with the linear peptides,<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> and Grb2–SH2 targeting lactam-linked bicyclic inhibitors improved the IC<sub>50</sub> 17-fold compared to the monocyclic peptide.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> Interestingly, in the case of these Grb2–SH2 targeted inhibitors, selectivity depending upon the lactam amino acid orientation was also described suggesting an important structural difference that can occur when the lactam orientation is reversed.</div><div class="NLM_p">Our binding experiments revealed that a high concentration of phosphate in the buffer was required for high affinity binding between Grb7–SH2 and the tyrosine containing bicyclic peptides, consistent with our previous studies of Grb7 targeting peptides.<a onclick="showRef(event, 'ref18 ref20'); return false;" href="javascript:void(0);" class="ref ref18 ref20">(18, 20)</a> The phosphate is likely to play a role in the pY binding pocket analogous to that played by the pY phosphate of a cellular ligand, contributing greatly to the overall affinity of binding through interactions with the surrounding residues including the BC loop. The Grb7–SH2/peptide <b>5</b> complex structure revealed the presence of a chloride ion that could effect the equivalent interactions alongside peptide <b>5</b>. In light of this structural information, we incorporated either a carboxymethylphenylalanine (cmF) or carboxyphenylalanine (cF) into the peptide <b>5</b> bicyclic backbone resulting in the affinity improving a further 8-fold to <i>K</i><sub>D</sub> = 130 nM (in a physiological concentration of phosphate). Both non-natural amino acids successfully acted as phosphotyrosine mimetics with the SPR analysis revealing that phosphate competes for Grb7–SH2 binding, and the X-ray crystal structures of Grb7–SH2 in complex with peptides <b>6</b> or <b>7</b> revealing engagement of the BC loop and pY binding pocket residues.</div><div class="NLM_p">Phosphotyrosine mimetics are typically designed to spatially mimic a phosphotyrosine. In this study, we discovered that the shorter cF pY mimetic (lacking the oxygen mimicking methylene of cmF) increased the binding affinity to a greater extent than the more traditional cmF pY mimetic and was less affected by the presence of high concentrations of phosphate (6-fold lower affinity compared with 14-fold difference). This is consistent with our studies of monocyclic peptides;<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> however, we have now revealed the structural basis for the improvement in the interaction with crystal structures of Grb7–SH2 domain in complex with peptides <b>6</b> and <b>7</b>. While the cmF group extends to the base of the pY binding pocket, the shorter cF group binds BC loop residues at the entry to the pocket. Through these interactions, the binding pocket is effectively closed off, forming a tighter interaction surface. This finding suggests that shortened pY mimetics such as carboxyphenylalanine could also be applicable in the design of peptides targeted to other SH2 domains to produce high affinity interactions.</div><div class="NLM_p">We have also determined that the peptide <b>5</b> scaffold is specific for Grb7–SH2 over the closely related SH2 domains of Grb2 and Grb10. Our structural investigations show the way in which this bicyclic scaffold complements the surface of Grb7–SH2, particularly across the βD6 amino acid Leu481 over which F2 and Y5 are straddled. Despite the very close homology and similar ligand binding preference of Grb10–SH2 and Grb2–SH2 respectively, the equivalent interaction is predicted to be impeded for these SH2 domains. Thus, the optimized peptides not only possess higher affinity but also maintain specificity for Grb7.</div><div class="NLM_p last">Lastly, we explored whether the optimized inhibitors could disrupt Grb7 mediated interactions when attached to the cell penetrating peptide, penetratin. This confirmed that the peptide inhibitors could successfully enter the cell and block Grb7 interactions with HER2, SHC, and FAK. Improved inhibition of Grb7 interactions by the bicyclic peptides over the first-generation monocyclic peptides was apparent, but only when the peptides were delivered directly to the cell lysate. Thus, we postulate that there could be differences in the effective delivery of the bicyclic peptides across the cell membrane. It has been demonstrated that the incorporation of different cargoes can alter the cellular uptake efficiencies by CPPs.<a onclick="showRef(event, 'ref33 ref34 ref35'); return false;" href="javascript:void(0);" class="ref ref33 ref34 ref35">(33-35)</a> It is possible that incorporating the lactam-linker has altered the ability of penetratin to deliver the peptides to the cytosol, and a different CPP is required for optimal uptake.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i15">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_02224" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_02224" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">This study thus describes a major advance in the development of potent and specific Grb7 targeting peptides. We propose that the bicyclic scaffold described herein not only confers rigidity to the peptide helping to enhance binding affinity but would also be anticipated to protect the peptide against proteases <i>in vivo</i>. We also show that these peptides are deliverable to cells. They therefore not only represent tools to elucidate the specific function of Grb7 in cells but also represent a significant step toward the development of novel Grb7-targeted therapeutics for the treatment of breast and other cancers.</div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i16">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_74634" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_74634" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec5_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Statement of Purity</h3><div class="NLM_p last">All proteins were purified ≥95% purity using size-exclusion chromatography as assessed using SDS-PAGE. All peptides were purified to ≥95% purity using reversed-phase HPLC and mass spectrometry for verification (as shown in <a href="/doi/suppl/10.1021/acs.jmedchem.7b01320/suppl_file/jm7b01320_si_001.pdf" class="ext-link">Supporting Information</a>).</div></div><div id="sec5_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Peptide Synthesis</h3><div id="sec5_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i19" class="anchor-spacer"></div><h4 class="article-section__title" id="_i19"> Peptide Synthesis Materials</h4><div class="NLM_p last">The allyloxycarbamate protected <i>N</i>-Fmoc lysine and the N-Fmoc-<span class="smallcaps smallerCapital">l</span>-glutamic acid α-allyl ester, to synthesize the lactam peptides (<b>4</b>–<b>7</b>), were purchased from Purar chemicals (Melbourne, Victoria, Australia). The triazole linker containing peptide (<b>3</b>) was synthesized from N-Fmoc-5-azidopentanoic acid, purchased from AAPPTec LLC (Louisville, Kentucky, USA) and <i>N</i>-Fmoc-<span class="smallcaps smallerCapital">l</span>-propargyl serine, prepared from <i>N</i>-Fmoc-<span class="smallcaps smallerCapital">l</span>-serine. To further incorporate a phosphotyrosine mimetic within the peptides required the substitution of the tyrosine residue within peptides (<b>6</b> and <b>7</b>) with either <i>N</i>-Fmoc-(4-<i>tert</i>-butyloxycarbonyl)-<span class="smallcaps smallerCapital">l</span>-phenylalanine, purchased from Santa Cruz Biotechnology Inc. (Dallas, Texas, USA) or <i>N</i>-Fmoc-(4-methoxycarbonylmethyl)-<span class="smallcaps smallerCapital">l</span>-phenylalanine, purchased from (Epichem Pty Ltd., Bentley, Western Australia, Australia). All other <i>N</i>-Fmoc protected α amino acids incorporated within the peptide sequences were purchased from GL Biochem Shanghai Ltd. (Shanghai, China). At the end of the peptide elongation cycle, the resin was thoroughly washed with CH<sub>2</sub>Cl<sub>2</sub> and Et<sub>2</sub>O and air-dried before further manipulation (see below).</div></div><div id="sec5_2_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20"> Peptide Synthesis (Peptides <b>3</b>–<b>7</b>)</h4><div class="NLM_p last">Peptides <b>3</b>–<b>7</b> were synthesized via solid phase peptide synthesis on a 0.25 mmol scale using standard Fmoc-chemistry on Rink amide resin (0.7 mmol/g), similarly to previously reported syntheses.<a onclick="showRef(event, 'ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref17 ref18">(17, 18)</a> Details of the procedure for click chemistry, lactamization, thioether formation, and peptide verification are provided below.</div></div><div id="sec5_2_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> Click Chemistry (Peptide <b>3</b>)</h4><div class="NLM_p last">Click chemistry was per that performed, on resin, on a 0.25 mmol scale. 2,6-lutidine (70 μL, 0.6 mmol), <span class="smallcaps smallerCapital">l</span>-ascorbic acid (70 mg, 0.4 mmol), and NaHCO<sub>3</sub> (33 mg, 0.4 mmol) in H<sub>2</sub>O (1.5 mL) were added to a solution of CuI (29 mg, 0.15 mmol) in CH<sub>3</sub>CN (12 mL) and DMSO (3 mL). The resulting solution was degassed by bubbling a stream of argon for 10 min and then added to the peptide loaded resin followed by gentle agitation for 16 h at room temperature. The resin was filtered and washed sequentially with CH<sub>3</sub>CN, H<sub>2</sub>O, DMF, and CH<sub>2</sub>Cl<sub>2</sub>, and the resin was soaked in DMF before continuing to the N-terminus deprotection step.</div></div><div id="sec5_2_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> Lactamization (Peptides <b>4</b>–<b>7</b>)</h4><div class="NLM_p">The glutamic acid and lysine residue protecting groups (allyl ester and allyloxycarbonyl, respectively) were removed simultaneously, on resin, on a 0.25 mmol scale. CHCl<sub>3</sub> (15 mL) was degassed by bubbling a stream of argon for 10 min. A portion of the CHCl<sub>3</sub> (2 mL) was used to preswell the resin, while PhSiH<sub>3</sub> (740 μL, 6 mmol) and Pd(PPh<sub>3</sub>)<sub>4</sub> (580 mg, 0.5 mmol) were added to the remainder of the sparged CHCl<sub>3</sub>. The mixture was vigorously shaken until homogeneity was achieved. The resin was subsequently soaked in the Pd(PPh<sub>3</sub>)<sub>4</sub> solution for 2 h with gentle agitation. The resin was filtered and washed with CH<sub>2</sub>Cl<sub>2</sub> and DMF to remove the catalyst.</div><div class="NLM_p last">Lactamization of the lysine and glutamic acid residues was achieved on a 0.25 mmol scale. The resin was washed with DMF, and then a solution of HBTU (3 equiv to resin loading), HOBt (3 equiv to resin loading), and DIPEA (4.5 equiv to resin loading), dissolved in DMF (4 mL), was added to the resin and gently agitated for 45 min. The resin was subsequently washed with DMF before continuing to the N-terminus deprotection step.</div></div><div id="sec5_2_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> N-Terminus Deprotection, Thioether Formation, and Peptide Analysis (Peptides <b>3</b>–<b>7</b>)</h4><div class="NLM_p last">The terminal Fmoc protecting group was removed with 2× wash (5 mL each) of 20% (v/v) piperidine in DMF (with 0.1 M HOBt) for 15 min each, and the resin was treated with chloroacetic anhydride (171 mg, 1 mmol) and DIPEA (100 μL) in DMF (3 mL) for 45 min to afford a choloroacetyl-capped N-terminus. The resin was subsequently washed with DMF, CH<sub>2</sub>Cl<sub>2</sub>, and Et<sub>2</sub>O and air-dried. Cleavage was performed on 0.25 mmol of the resin, by treating the resin with a cleavage solution (15 mL) comprising 2.5% (v/v) distilled water (375 μL), 2.5% (v/v) triisopropylsilane (375 μL), and 0.5% (v/v) ethanedithiol (75 μL) in TFA for 3 h. The TFA was evaporated under a stream of N<sub>2</sub>, the peptide was precipitated by addition of Et<sub>2</sub>O (40 mL), and the precipitate was filtered and redissolved in 50% aqueous CH<sub>3</sub>CN for lyophilization. Thioether formation occurred by dissolving the crude peptides in 100 mM aqueous NH<sub>4</sub>HCO<sub>3</sub> solution made up in 50% aqueous CH<sub>3</sub>CN for 2 h at room temp. The cyclized peptides were purified using reversed-phase HPLC.</div></div><div id="sec5_2_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> Hydrolysis of cmF Methyl Ester (Peptide <b>6</b>)</h4><div class="NLM_p">The <i>para</i>-carboxymethyl phenylalanine methyl ester was hydrolyzed by dissolving the peptide (∼5 mg·mL<sup>–1</sup>) in aqueous LiOH solution for 2 h at room temp. The success of hydrolysis was confirmed using mass spectrometry, and all peptides were purified to ≥95% purity using reversed-phase HPLC (as shown in <a href="/doi/suppl/10.1021/acs.jmedchem.7b01320/suppl_file/jm7b01320_si_001.pdf" class="ext-link">Supporting Information</a>).</div><div class="NLM_p last">All peptide identities were verified by mass spectrometry. Peptide <b>3</b>: <i>M</i><sub>expected</sub> (C<sub>49</sub>H<sub>61</sub>N<sub>14</sub>O<sub>17</sub>S)<sup>−</sup> = 1149.4; <i>M</i><sub>measured</sub> (C<sub>49</sub>H<sub>61</sub>N<sub>14</sub>O<sub>17</sub>S)<sup>−</sup> = 1149.3; <i>M</i><sub>expected</sub> (C<sub>49</sub>H<sub>60</sub>N<sub>14</sub>O<sub>17</sub>S)<sup>2–</sup> = 574.2; <i>M</i><sub>measured</sub> (C<sub>49</sub>H<sub>60</sub>N<sub>14</sub>O<sub>17</sub>S)<sup>2–</sup> = 574.2. Peptide <b>4</b>: <i>M</i><sub>expected</sub> (C<sub>49</sub>H<sub>63</sub>N<sub>12</sub>O<sub>17</sub>S)<sup>−</sup> = 1123.4; <i>M</i><sub>measured</sub> (C<sub>49</sub>H<sub>63</sub>N<sub>12</sub>O<sub>17</sub>S)<sup>−</sup> = 1123.1; <i>M</i><sub>expected</sub> (C<sub>49</sub>H<sub>62</sub>N<sub>12</sub>O<sub>17</sub>S)<sup>2–</sup> = 561.2; <i>M</i><sub>measured</sub> (C<sub>49</sub>H<sub>62</sub>N<sub>12</sub>O<sub>17</sub>S)<sup>2–</sup> = 561.1. Peptide <b>5</b>: <i>M</i><sub>expected</sub> (C<sub>49</sub>H<sub>63</sub>N<sub>12</sub>O<sub>17</sub>S)<sup>−</sup> = 1123.4; <i>M</i><sub>measured</sub> (C<sub>49</sub>H<sub>63</sub>N<sub>12</sub>O<sub>17</sub>S)<sup>−</sup> = 1123.4; <i>M</i><sub>expected</sub> (C<sub>49</sub>H<sub>62</sub>N<sub>12</sub>O<sub>17</sub>S)<sup>2–</sup> = 561.2; <i>M</i><sub>measured</sub> (C<sub>49</sub>H<sub>62</sub>N<sub>12</sub>O<sub>17</sub>S)<sup>2–</sup> = 561.1. Peptide <b>6</b>: <i>M</i><sub>expected</sub> (C<sub>51</sub>H<sub>65</sub>N<sub>12</sub>O<sub>18</sub>S)<sup>−</sup> = 1165.4; <i>M</i><sub>measured</sub> (C<sub>51</sub>H<sub>65</sub>N<sub>12</sub>O<sub>18</sub>S)<sup>−</sup> = 1165.1; <i>M</i><sub>expected</sub> (C<sub>51</sub>H<sub>64</sub>N<sub>12</sub>O<sub>18</sub>S)<sup>2–</sup> = 582.2; <i>M</i><sub>measured</sub> (C<sub>51</sub>H<sub>64</sub>N<sub>12</sub>O<sub>18</sub>S)<sup>2–</sup> = 582.0. Peptide <b>7</b>: <i>M</i><sub>expected</sub> (C<sub>50</sub>H<sub>63</sub>N<sub>12</sub>O<sub>18</sub>S)<sup>−</sup> = 1151.4; <i>M</i><sub>measured</sub> (C<sub>50</sub>H<sub>63</sub>N<sub>12</sub>O<sub>18</sub>S)<sup>−</sup> = 1151.0; <i>M</i><sub>expected</sub> (C<sub>50</sub>H<sub>62</sub>N<sub>12</sub>O<sub>18</sub>S)<sup>2–</sup> = 575.2; <i>M</i><sub>measured</sub> (C<sub>50</sub>H<sub>62</sub>N<sub>12</sub>O<sub>18</sub>S)<sup>2–</sup> = 575.1.</div></div><div id="sec5_2_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> Penetratin Coupled Peptides</h4><div class="NLM_p last">Penetratin coupled peptide 1 (<b>1P</b>) was prepared as previously described.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a><i>M</i><sub>expected</sub> (C<sub>171</sub>H<sub>246</sub>N<sub>48</sub>O<sub>38</sub>S<sub>2</sub>)<sup>6+</sup> = 608.3; <i>M</i><sub>measured</sub> (C<sub>171</sub>H<sub>246</sub>N<sub>48</sub>O<sub>38</sub>S<sub>2</sub>)<sup>6+</sup> = 608.6. Penetratin coupled peptides <b>5</b>, <b>7</b>, and <b>8</b> (<b>5P</b>, <b>7P</b>, and <b>8P</b>), and a propargyl penetratin control peptide were synthesized by Purar Chemicals (Australia) to >95% purity based on LC-MS (as shown in <a href="/doi/suppl/10.1021/acs.jmedchem.7b01320/suppl_file/jm7b01320_si_001.pdf" class="ext-link">Supporting Information</a>). The synthesis of peptide <b>8</b> (without penetratin) has been previously described.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> Peptide <b>5P</b>: <i>M</i><sub>expected</sub> (C<sub>153</sub>H<sub>230</sub>N<sub>46</sub>O<sub>36</sub>S<sub>2</sub>)<sup>4+</sup> = 839.5; <i>M</i><sub>measured</sub>(C<sub>153</sub>H<sub>230</sub>N<sub>46</sub>O<sub>36</sub>S<sub>2</sub>)<sup>4+</sup> = 839.8; <i>M</i><sub>expected</sub> (C<sub>153</sub>H<sub>230</sub>N<sub>46</sub>O<sub>36</sub>S<sub>2</sub>)<sup>5+</sup> = 671.8; <i>M</i><sub>measured</sub>(C<sub>153</sub>H<sub>230</sub>N<sub>46</sub>O<sub>36</sub>S<sub>2</sub>)<sup>5+</sup> = 672.0. Peptide <b>7P</b>: <i>M</i><sub>expected</sub> (C<sub>155</sub>H<sub>231</sub>N<sub>45</sub>O<sub>37</sub>S<sub>2</sub>)<sup>4+</sup> = 846.5; <i>M</i><sub>measured</sub>(C<sub>155</sub>H<sub>231</sub>N<sub>45</sub>O<sub>37</sub>S<sub>2</sub>)<sup>4+</sup> = 847.0; <i>M</i><sub>expected</sub> (C<sub>155</sub>H<sub>231</sub>N<sub>45</sub>O<sub>37</sub>S<sub>2</sub>)<sup>5+</sup> = 677.4; <i>M</i><sub>measured</sub>(C<sub>155</sub>H<sub>231</sub>N<sub>45</sub>O<sub>37</sub>S<sub>2</sub>)<sup>5+</sup> = 677.8. Peptide <b>8P</b>: <i>M</i><sub>expected</sub> (C<sub>172</sub>H<sub>246</sub>N<sub>48</sub>O<sub>39</sub>S<sub>2</sub>)<sup>4+</sup> = 919.6; <i>M</i><sub>measured</sub>(C<sub>172</sub>H<sub>246</sub>N<sub>48</sub>O<sub>39</sub>S<sub>2</sub>)<sup>4+</sup> = 919.6; <i>M</i><sub>expected</sub> (C<sub>172</sub>H<sub>246</sub>N<sub>48</sub>O<sub>39</sub>S<sub>2</sub>)<sup>5+</sup> = 735.9; <i>M</i><sub>measured</sub>(C<sub>172</sub>H<sub>246</sub>N<sub>48</sub>O<sub>39</sub>S<sub>2</sub>)<sup>5+</sup> = 735.9. Penetratin control peptide: <i>M</i><sub>expected</sub> (C<sub>111</sub>H<sub>180</sub>N<sub>36</sub>O<sub>21</sub>)<sup>4+</sup> = 596.6; <i>M</i><sub>measured</sub> (C<sub>111</sub>H<sub>180</sub>N<sub>36</sub>O<sub>21</sub>)<sup>4+</sup> = 596.5; <i>M</i><sub>expected</sub> (C<sub>111</sub>H<sub>181</sub>N<sub>36</sub>O<sub>21</sub>)<sup>5+</sup> = 477.5; <i>M</i><sub>measured</sub> (C<sub>111</sub>H<sub>181</sub>N<sub>36</sub>O<sub>21</sub>)<sup>5+</sup> = 477.4; <i>M</i><sub>expected</sub> (C<sub>111</sub>H<sub>182</sub>N<sub>36</sub>O<sub>21</sub>)<sup>6+</sup> = 398.1; <i>M</i><sub>measured</sub> (C<sub>111</sub>H<sub>182</sub>N<sub>36</sub>O<sub>21</sub>)<sup>6+</sup> = 398.0.</div></div><div id="sec5_2_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> Peptide Concentration Determination</h4><div class="NLM_p last">Solution peptide concentrations were determined spectrophotometrically based on absorbance at 280 nm using extinction coefficients derived from amino acid content.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> If the absorbance could not be measured at 280 nm, the DirectDetect (Millipore) system was used.</div></div></div><div id="sec5_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> Protein Expression and Purification</h3><div class="NLM_p last">GST tagged Grb7–SH2, Grb2–SH2, and Grb10–SH2, GST alone, and untagged Grb7–SH2 were expressed and purified as previously described.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a></div></div><div id="sec5_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> Surface Plasmon Resonance</h3><div class="NLM_p last">SPR experiments were conducted at 25 °C as previously described on a BIAcore T100 using BIAcore CM5 series S sensor chips (GE Life Sciences).<a onclick="showRef(event, 'ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref17 ref18">(17, 18)</a> Briefly, polyclonal anti-GST antibody (Abcam, Cambridge) was immobilized on all flow cells via amine coupling with capture levels between 2280 RU and 6700 RU (GE Life Sciences GST capture kit). As a control, GST alone was captured on the reference flow cell (between 520 RU and 1280 RU) and the GST tagged proteins were immobilized on the active flow cells (between 690 RU and 1990 RU). Lyophilized peptides (in triplicate) were resuspended in analysis buffer and injected over all four flow cells at 30 μL·min<sup>–1</sup> for 60–80 s. The +PO<sub>4</sub><sup>3–</sup> buffer consisted of 50 mM Na<sub>3</sub>PO<sub>4</sub>, 150–300 mM NaCl, and 1 mM DTT (pH 7.4), whereas the −PO<sub>4</sub><sup>3–</sup> buffer consisted of 1 mM Na<sub>3</sub>PO<sub>4</sub> (reflecting a physiological concentration), 150 mM NaCl, 20 mM Tris, 1 mM DTT (pH 7.4). Peptide <b>3</b> required regeneration with 2 M NaCl before injection of the next peptide cycle. Data were analyzed using Scrubber2.0 (BioLogic Software, Campbell, ACT, Australia) and Prism 6.0 (GraphPad Software, California, USA).</div></div><div id="sec5_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> Structure Determination of Grb7–SH2/Peptide Complexes</h3><div class="NLM_p">The Grb7–SH2/peptide <b>5</b> complex formed diffracting crystals using 7.5 mg·mL<sup>–1</sup> protein in a 1:1.5 M ratio of protein/peptide in a condition containing 18% (w/v) PEG 6000, 0.1 M MES (pH 5.5), and 0.2 M CaCl<sub>2</sub>. The crystals were cryoprotected in mother liquor supplemented with 15% (v/v) glycerol and flash-cooled in liquid nitrogen. Diffraction images were collected at the Australian Synchrotron MX2 high-throughput protein crystallography beamline with an ADSC Quantum 315r CCD detector using the BLU-ICE acquisition software.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> Diffraction images were indexed using IMOSFLM<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> and scaled using AIMLESS<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> from the CCP4 suite.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> Phases were generated via molecular replacement (using PHASER) with one chain of apo Grb7–SH2 as the search model (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2QMS">2QMS</a>).<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> Subsequent model refinement was conducted using PHENIX<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> and model building was performed with COOT.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a></div><div class="NLM_p">The Grb7–SH2/peptide <b>6</b> complex crystals formed using 10 mg·mL<sup>–1</sup> protein in a 1:1.5 M ratio of protein/peptide in a condition containing 25% (w/v) PEG3350, 0.2 M KSCN, and 0.1 M BTP (pH 8.5). The crystals were cryoprotected in mother liquor supplemented with 10% (v/v) glycerol and flash-cooled in liquid nitrogen. Diffraction data was collected at the Australian Synchrotron as per that of the Grb7–SH2/peptide <b>5</b> complex and subsequently indexed and using IMOSFLM<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> and scaled using AIMLESS<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> from the CCP4 suite.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> Molecular replacement, model building, and refinement occurred as per that of the Grb7–SH2/peptide <b>5</b> complex.</div><div class="NLM_p">The Grb7–SH2/peptide <b>7</b> complex formed diffracting crystals using 10 mg·mL<sup>–1</sup> protein in a 1:1.5 M ratio of protein/peptide in a condition containing 20% (w/v) PEG3350 and 0.2 M NaSCN. The crystals were cryoprotected in mother liquor saturated with sucrose and flash-cooled in liquid nitrogen. Diffraction images were collected at the Australian Synchrotron as described above. The images were indexed using XDS<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> and scaled using AIMLESS.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> Molecular replacement, model building, and refinement occurred as per that of the Grb7–SH2/peptide <b>5</b> complex.</div><div class="NLM_p last">The restraint files for all peptides were generated using eLBOW from SMILES strings.<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> MOLPROBITY was used to assess the quality of the final models.<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> The data collection and processing statistics are displayed in <a class="ref internalNav" href="#tbl3" aria-label="Table 3">Table 3</a>.</div></div><div id="sec5_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> Cell Culture and Coimmunoprecipitation Experiments</h3><div class="NLM_p">The SKBR3 HER2+ breast cancer cell line was obtained from the American Type Culture Collection (ATCC, USA) and maintained according to ATCC guidelines. Overnight serum starved cells in a 10 cm dish were stimulated for 10 min with fibronectin (10 μg.mL<sup>–1</sup>, 37 °C), washed 2 times with ice cold PBS and lysed with 1 mL of RIPA buffer (50 mM Tris-HCl, pH 7.4, 150 mM NaCl, 10% (v/v) glycerol, 1% (v/v) Triton X-100, 10 mM sodium pyrophosphate, 20 mM NaF, 1 mM phenylmethylsulfonyl fluoride, 1 mM sodium orthovanadate, 10 μg·mL<sup>–1</sup> aprotinin, and 10 μg·mL<sup>–1</sup> leupeptin). Supernatant protein concentration was determined using the BradfordUltra reagent from Expedeon (catalog no. BFU1L).<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> Rabbit Grb7 (N-20) antibodies (Santa Cruz Biotechnology) were used to immunoprecipiate Grb7 and associating proteins, and this was collected with protein A sepharose beads (Zymed Laboratories). The Grb7 targeting peptides (20 μM) were either added to 1 mg of cell lysate or, prior to fibronectin stimulation, live cells were treated with peptide for 1 h.</div><div class="NLM_p last">Immunoprecipitate samples were resolved by SDS-PAGE (8–12%) and subject to Western blotting using the appropriate antibodies. Grb7 and its bound proteins were visualized by chemiluminescence from the corresponding secondary antibodies. Mouse FAK (catalog no. F15020) and mouse SHC (catalog no. 610879) were obtained from BD Transduction Laboratories. Rabbit HER2 (catalog no. 06-562) was obtained from Merck Millipore. Donkey anti-mouse HRP (catalog no. 1706515) and goat anti-rabbit HRP antibodies were obtained from Bio-RadAbcam.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i31"><a href="/doi/suppl/10.1021/acs.jmedchem.7b01320">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_54167" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_54167" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.7b01320" class="ext-link">10.1021/acs.jmedchem.7b01320</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">X-ray crystal structure of the Grb7–SH2/peptide <b>5</b> complex, SPR specificity experiments for peptides <b>3</b>–<b>5</b> binding to Grb7–SH2 domain, surface representation of peptide <b>5</b> bound to SH2 domains, SPR binding experiments for peptides <b>6</b> and <b>7</b> binding to Grb7–SH2, X-ray crystal structures of the Grb7–SH2/peptide <b>6</b> complex and the Grb7–SH2/peptide <b>7</b> complex, full Western blots and LC chromatograms and MS for peptides (<a href="/doi/suppl/10.1021/acs.jmedchem.7b01320/suppl_file/jm7b01320_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.7b01320/suppl_file/jm7b01320_si_002.csv" class="ext-link">CSV</a>)</p></li><li><p class="inline">Chemical structure data (<a href="/doi/suppl/10.1021/acs.jmedchem.7b01320/suppl_file/jm7b01320_si_003.cdx" class="ext-link">CDX</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b01320/suppl_file/jm7b01320_si_001.pdf">jm7b01320_si_001.pdf (12.79 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b01320/suppl_file/jm7b01320_si_002.csv">jm7b01320_si_002.csv (4.42 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b01320/suppl_file/jm7b01320_si_003.cdx">jm7b01320_si_003.cdx (167.78 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Atomic coordinates and structure factors have been deposited in the RCSB protein data bank under the accession numbers <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5U1Q">5U1Q</a> (Grb7–SH2/peptide <b>5</b> complex), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5U06">5U06</a> (Grb7–SH2/peptide <b>6</b> complex), and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5TYI">5TYI</a> (Grb7–SH2/peptide <b>7</b> complex).</p><div class="testing" data-doi="10.1021/acs.jmedchem.7b01320" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_22646" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_22646" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jacqueline A. Wilce</span> - <span class="hlFld-Affiliation affiliation">Biomedicine Discovery
Institute, Department of Biochemistry and Molecular Biology, Monash University, Wellington Road, Clayton, VIC 3800, Australia</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-8344-2626" title="Orcid link">http://orcid.org/0000-0002-8344-2626</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#600a01030b09054e17090c0305200d0f0e0113084e050415"><span class="__cf_email__" data-cfemail="e08a81838b8985ce97898c8385a08d8f8e819388ce858495">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Gabrielle M. Watson</span> - <span class="hlFld-Affiliation affiliation">Biomedicine Discovery
Institute, Department of Biochemistry and Molecular Biology, Monash University, Wellington Road, Clayton, VIC 3800, Australia</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ketav Kulkarni</span> - <span class="hlFld-Affiliation affiliation">Biomedicine Discovery
Institute, Department of Biochemistry and Molecular Biology, Monash University, Wellington Road, Clayton, VIC 3800, Australia</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jianrong Sang</span> - <span class="hlFld-Affiliation affiliation">Biomedicine Discovery
Institute, Department of Biochemistry and Molecular Biology, Monash University, Wellington Road, Clayton, VIC 3800, Australia</span>; 
    <span class="hlFld-Affiliation affiliation">Department of Physiology, School of Medicine, Jiangsu University, Zhenjiang, Jiangsu 212013, P.R. China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xiuquan Ma</span> - <span class="hlFld-Affiliation affiliation">Biomedicine Discovery
Institute, Department of Biochemistry and Molecular Biology, Monash University, Wellington Road, Clayton, VIC 3800, Australia</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Menachem J. Gunzburg</span> - <span class="hlFld-Affiliation affiliation">Biomedicine Discovery
Institute, Department of Biochemistry and Molecular Biology, Monash University, Wellington Road, Clayton, VIC 3800, Australia</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Patrick Perlmutter</span> - <span class="hlFld-Affiliation affiliation">School of Chemistry, Monash University, Wellington
Road, Clayton, VIC 3800, Australia</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Matthew C.J. Wilce</span> - <span class="hlFld-Affiliation affiliation">Biomedicine Discovery
Institute, Department of Biochemistry and Molecular Biology, Monash University, Wellington Road, Clayton, VIC 3800, Australia</span></div></li></ul></li><li></li><li><span class="author-information-subsection-header">Funding</span><p>This work was supported by a grant from the National Health and Medical Research Council awarded to J.A.W. (APP1045309) and a National Health and Medical Research Senior Research Fellowship awarded to M.C.J.W. (FP1079611), as well as the Victorian Government’s Operational Infrastructure Support Program.</p></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d7e3355-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i34">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_54165" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_54165" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We thank Danielle Cini for assistance with setting up crystallization experiments, Roger Daly for input towards the cellular experiments, and all the staff at the MX2 beamline at the Australian Synchrotron, Victoria, Australia.</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i35" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i35"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i36" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i36"> Abbreviations</h2><tr><td class="NLM_term">Grb7</td><td class="NLM_def"><p class="first last">growth receptor bound protein</p></td></tr><tr><td class="NLM_term">SH2 domain</td><td class="NLM_def"><p class="first last">Src homology domain</p></td></tr><tr><td class="NLM_term">PPI</td><td class="NLM_def"><p class="first last">protein–protein interaction</p></td></tr><tr><td class="NLM_term">HER2</td><td class="NLM_def"><p class="first last">human epidermal growth factor receptor 2</p></td></tr><tr><td class="NLM_term">FAK</td><td class="NLM_def"><p class="first last">focal adhesion kinase</p></td></tr><tr><td class="NLM_term">SHC</td><td class="NLM_def"><p class="first last">SH2-containing protein</p></td></tr><tr><td class="NLM_term">cmF</td><td class="NLM_def"><p class="first last">carboxymethylphenylalanine</p></td></tr><tr><td class="NLM_term">cF</td><td class="NLM_def"><p class="first last">carboxyphenylalanine</p></td></tr><tr><td class="NLM_term">SPR</td><td class="NLM_def"><p class="first last">surface plasmon resonance</p></td></tr><tr><td class="NLM_term">pY</td><td class="NLM_def"><p class="first last">phosphotyrosine</p></td></tr><tr><td class="NLM_term">CPP</td><td class="NLM_def"><p class="first last">cell penetrating peptide</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i37">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_73063" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_73063" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 48 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Ivanov, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Khuri, F. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fu, H.</span><span> </span><span class="NLM_article-title">Targeting protein-protein interactions as an anticancer strategy</span> <span class="citation_source-journal">Trends Pharmacol. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">34</span><span class="NLM_x">, </span> <span class="NLM_fpage">393</span><span class="NLM_x">–</span> <span class="NLM_lpage">400</span><span class="refDoi"> DOI: 10.1016/j.tips.2013.04.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=10.1016%2Fj.tips.2013.04.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=23725674" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=1%3ACAS%3A528%3ADC%252BC3sXosVGjt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2013&pages=393-400&author=A.+A.+Ivanovauthor=F.+R.+Khuriauthor=H.+Fu&title=Targeting+protein-protein+interactions+as+an+anticancer+strategy&doi=10.1016%2Fj.tips.2013.04.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting protein-protein interactions as an anticancer strategy</span></div><div class="casAuthors">Ivanov, Andrei A.; Khuri, Fadlo R.; Fu, Haian</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Pharmacological Sciences</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">393-400</span>CODEN:
                <span class="NLM_cas:coden">TPHSDY</span>;
        ISSN:<span class="NLM_cas:issn">0165-6147</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  The emergence and convergence of cancer genomics, targeted therapies, and network oncol. have significantly expanded the landscape of protein-protein interaction (PPI) networks in cancer for therapeutic discovery.  Extensive biol. and clin. investigations have led to the identification of protein interaction hubs and nodes that are crit. for the acquisition and maintenance of characteristics of cancer essential for cell transformation.  Such cancer-enabling PPIs have become promising therapeutic targets.  With technol. advances in PPI modulator discovery and validation of PPI-targeting agents in clin. settings, targeting of PPI interfaces as an anticancer strategy has become a reality.  Future research directed at genomics-based PPI target discovery, PPI interface characterization, PPI-focused chem. library design, and patient-genomic subpopulation-driven clin. studies is expected to accelerate the development of the next generation of PPI-based anticancer agents for personalized precision medicine.  Here we briefly review prominent PPIs that mediate cancer-acquired properties, highlight recognized challenges and promising clin. results in targeting PPIs, and outline emerging opportunities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqKeFT8Id1vhbVg90H21EOLACvtfcHk0ljFibPoGdeD2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXosVGjt7g%253D&md5=2f4772a4abfa6640b6af922671834fa3</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.tips.2013.04.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tips.2013.04.007%26sid%3Dliteratum%253Aachs%26aulast%3DIvanov%26aufirst%3DA.%2BA.%26aulast%3DKhuri%26aufirst%3DF.%2BR.%26aulast%3DFu%26aufirst%3DH.%26atitle%3DTargeting%2520protein-protein%2520interactions%2520as%2520an%2520anticancer%2520strategy%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2013%26volume%3D34%26spage%3D393%26epage%3D400%26doi%3D10.1016%2Fj.tips.2013.04.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Pawson, T.</span><span> </span><span class="NLM_article-title">Specificity in signal transduction: from phosphotyrosine-SH2 domain interactions to complex cellular systems</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">116</span><span class="NLM_x">, </span> <span class="NLM_fpage">191</span><span class="NLM_x">–</span> <span class="NLM_lpage">203</span><span class="refDoi"> DOI: 10.1016/S0092-8674(03)01077-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=10.1016%2FS0092-8674%2803%2901077-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=14744431" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtValsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=2004&pages=191-203&author=T.+Pawson&title=Specificity+in+signal+transduction%3A+from+phosphotyrosine-SH2+domain+interactions+to+complex+cellular+systems&doi=10.1016%2FS0092-8674%2803%2901077-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Specificity in signal transduction: From phosphotyrosine-SH2 domain interactions to complex cellular systems</span></div><div class="casAuthors">Pawson, Tony</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">191-203</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review and discussion.  Over the last 2 decades, a new and unifying concept of cellular organization has emerged in which modular protein-protein interactions provide an underlying framework through which signaling pathways are assembled and controlled.  In this scheme, post-translational modifications such as phosphorylation commonly exert their biol. effects by regulating mol. interactions, exemplified by the ability of phosphotyrosine sites to bind selectively to SH2 domains.  Although these interactions are rather simple in isolation, they can nonetheless be exploited to generate complex cellular systems.  Here, the author discusses expts. that have led to this view of dynamic cellular behavior and identifies some current and future areas of interest in cell signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrolI53c79a6rVg90H21EOLACvtfcHk0ljFibPoGdeD2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtValsrk%253D&md5=8521703a8fe32fda8ddb93adb8055af3</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2FS0092-8674%2803%2901077-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0092-8674%252803%252901077-8%26sid%3Dliteratum%253Aachs%26aulast%3DPawson%26aufirst%3DT.%26atitle%3DSpecificity%2520in%2520signal%2520transduction%253A%2520from%2520phosphotyrosine-SH2%2520domain%2520interactions%2520to%2520complex%2520cellular%2520systems%26jtitle%3DCell%26date%3D2004%26volume%3D116%26spage%3D191%26epage%3D203%26doi%3D10.1016%2FS0092-8674%2803%2901077-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Hanahan, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weinberg, R. A.</span><span> </span><span class="NLM_article-title">Hallmarks of cancer: the next generation</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">144</span><span class="NLM_x">, </span> <span class="NLM_fpage">646</span><span class="NLM_x">–</span> <span class="NLM_lpage">674</span><span class="refDoi"> DOI: 10.1016/j.cell.2011.02.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=10.1016%2Fj.cell.2011.02.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=21376230" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=1%3ACAS%3A528%3ADC%252BC3MXjsFeqtrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=144&publication_year=2011&pages=646-674&author=D.+Hanahanauthor=R.+A.+Weinberg&title=Hallmarks+of+cancer%3A+the+next+generation&doi=10.1016%2Fj.cell.2011.02.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Hallmarks of cancer: the next generation</span></div><div class="casAuthors">Hanahan, Douglas; Weinberg, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">144</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">646-674</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  The hallmarks of cancer comprise six biol. capabilities acquired during the multistep development of human tumors.  The hallmarks constitute an organizing principle for rationalizing the complexities of neoplastic disease.  They include sustaining proliferative signaling, evading growth suppressors, resisting cell death, enabling replicative immortality, inducing angiogenesis, and activating invasion and metastasis.  Underlying these hallmarks are genome instability, which generates the genetic diversity that expedites their acquisition, and inflammation, which fosters multiple hallmark functions.  Conceptual progress in the last decade has added two emerging hallmarks of potential generality to this list-reprogramming of energy metab. and evading immune destruction.  In addn. to cancer cells, tumors exhibit another dimension of complexity: they contain a repertoire of recruited, ostensibly normal cells that contribute to the acquisition of hallmark traits by creating the "tumor microenvironment.".  Recognition of the widespread applicability of these concepts will increasingly affect the development of new means to treat human cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTcpqXALcb6LVg90H21EOLACvtfcHk0ljFibPoGdeD2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXjsFeqtrk%253D&md5=b36f160c417a712e83cb83c577f0018e</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2011.02.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2011.02.013%26sid%3Dliteratum%253Aachs%26aulast%3DHanahan%26aufirst%3DD.%26aulast%3DWeinberg%26aufirst%3DR.%2BA.%26atitle%3DHallmarks%2520of%2520cancer%253A%2520the%2520next%2520generation%26jtitle%3DCell%26date%3D2011%26volume%3D144%26spage%3D646%26epage%3D674%26doi%3D10.1016%2Fj.cell.2011.02.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Gao, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yin, H.</span><span> </span><span class="NLM_article-title">Targeting protein-protein interfaces using macrocyclic peptides</span> <span class="citation_source-journal">Biopolymers</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">104</span><span class="NLM_x">, </span> <span class="NLM_fpage">310</span><span class="NLM_x">–</span> <span class="NLM_lpage">316</span><span class="refDoi"> DOI: 10.1002/bip.22625</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=10.1002%2Fbip.22625" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=25664609" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1egu7jP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=2015&pages=310-316&author=M.+Gaoauthor=K.+Chengauthor=H.+Yin&title=Targeting+protein-protein+interfaces+using+macrocyclic+peptides&doi=10.1002%2Fbip.22625"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting protein-protein interfaces using macrocyclic peptides</span></div><div class="casAuthors">Gao, Meng; Cheng, Kui; Yin, Hang</div><div class="citationInfo"><span class="NLM_cas:title">Biopolymers</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">104</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">310-316</span>CODEN:
                <span class="NLM_cas:coden">BIPMAA</span>;
        ISSN:<span class="NLM_cas:issn">0006-3525</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A review.  Protein-protein interactions (PPIs) are crit. in numerous biol. processes including signaling transduction, function regulations, and disease development.  To regulate PPIs has been thought to be challenging due to their highly dynamic and expansive interfacial areas.  Nonetheless, successful examples have been reported of targeting PPIs using small mols., peptides, and proteins.  Peptides, esp. macrocyclic peptides have proven to be a particularly useful tool to inhibit PPIs for their exquisite potency, stability and selectivity.  Herein we review the recent developments of this area of research, focusing on the macrocyclic peptides isolated from natural products, identified from library screening, and rationally designed based on structures, as PPI regulators. © 2015 Wiley Periodicals, Inc. Biopolymers (Pept Sci) 104: 310-316, 2015.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfC3LNeS7WxrVg90H21EOLACvtfcHk0lgH8KLwm1zjCA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1egu7jP&md5=340696f3a54e9cb6f9b48fed6c7a5e45</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1002%2Fbip.22625&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fbip.22625%26sid%3Dliteratum%253Aachs%26aulast%3DGao%26aufirst%3DM.%26aulast%3DCheng%26aufirst%3DK.%26aulast%3DYin%26aufirst%3DH.%26atitle%3DTargeting%2520protein-protein%2520interfaces%2520using%2520macrocyclic%2520peptides%26jtitle%3DBiopolymers%26date%3D2015%26volume%3D104%26spage%3D310%26epage%3D316%26doi%3D10.1002%2Fbip.22625" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Stein, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fuqua, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roonprapunt, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yajnik, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">D’Eustachio, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moskow, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buchberg, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Osborne, C. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Margolis, B.</span><span> </span><span class="NLM_article-title">The SH2 domain protein GRB-7 is co-amplified, overexpressed and in a tight complex with HER2 in breast cancer</span> <span class="citation_source-journal">EMBO J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">1331</span><span class="NLM_x">–</span> <span class="NLM_lpage">1340</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=1994&pages=1331-1340&author=D.+Steinauthor=J.+Wuauthor=S.+A.+Fuquaauthor=C.+Roonprapuntauthor=V.+Yajnikauthor=P.+D%E2%80%99Eustachioauthor=J.+J.+Moskowauthor=A.+M.+Buchbergauthor=C.+K.+Osborneauthor=B.+Margolis&title=The+SH2+domain+protein+GRB-7+is+co-amplified%2C+overexpressed+and+in+a+tight+complex+with+HER2+in+breast+cancer"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DStein%26aufirst%3DD.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DFuqua%26aufirst%3DS.%2BA.%26aulast%3DRoonprapunt%26aufirst%3DC.%26aulast%3DYajnik%26aufirst%3DV.%26aulast%3DD%25E2%2580%2599Eustachio%26aufirst%3DP.%26aulast%3DMoskow%26aufirst%3DJ.%2BJ.%26aulast%3DBuchberg%26aufirst%3DA.%2BM.%26aulast%3DOsborne%26aufirst%3DC.%2BK.%26aulast%3DMargolis%26aufirst%3DB.%26atitle%3DThe%2520SH2%2520domain%2520protein%2520GRB-7%2520is%2520co-amplified%252C%2520overexpressed%2520and%2520in%2520a%2520tight%2520complex%2520with%2520HER2%2520in%2520breast%2520cancer%26jtitle%3DEMBO%2520J.%26date%3D1994%26volume%3D13%26spage%3D1331%26epage%3D1340" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Tanaka, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mori, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Akiyoshi, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tanaka, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mafune, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wands, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sugimachi, K.</span><span> </span><span class="NLM_article-title">Coexpression of Grb7 with epidermal growth factor receptor or Her2/erbB2 in human advanced esophageal carcinoma</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">28</span><span class="NLM_x">–</span> <span class="NLM_lpage">31</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=1997&pages=28-31&author=S.+Tanakaauthor=M.+Moriauthor=T.+Akiyoshiauthor=Y.+Tanakaauthor=K.+Mafuneauthor=J.+R.+Wandsauthor=K.+Sugimachi&title=Coexpression+of+Grb7+with+epidermal+growth+factor+receptor+or+Her2%2FerbB2+in+human+advanced+esophageal+carcinoma"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DTanaka%26aufirst%3DS.%26aulast%3DMori%26aufirst%3DM.%26aulast%3DAkiyoshi%26aufirst%3DT.%26aulast%3DTanaka%26aufirst%3DY.%26aulast%3DMafune%26aufirst%3DK.%26aulast%3DWands%26aufirst%3DJ.%2BR.%26aulast%3DSugimachi%26aufirst%3DK.%26atitle%3DCoexpression%2520of%2520Grb7%2520with%2520epidermal%2520growth%2520factor%2520receptor%2520or%2520Her2%252FerbB2%2520in%2520human%2520advanced%2520esophageal%2520carcinoma%26jtitle%3DCancer%2520Res.%26date%3D1997%26volume%3D57%26spage%3D28%26epage%3D31" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Tanaka, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pero, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taguchi, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shimada, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mori, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krag, D. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arii, S.</span><span> </span><span class="NLM_article-title">Specific peptide ligand for Grb7 signal transduction protein and pancreatic cancer metastasis</span> <span class="citation_source-journal">J. Natl. Cancer Inst.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">98</span><span class="NLM_x">, </span> <span class="NLM_fpage">491</span><span class="NLM_x">–</span> <span class="NLM_lpage">498</span><span class="refDoi"> DOI: 10.1093/jnci/djj105</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=10.1093%2Fjnci%2Fdjj105" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=16595785" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=1%3ACAS%3A528%3ADC%252BD28XjtlamtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=2006&pages=491-498&author=S.+Tanakaauthor=S.+C.+Peroauthor=K.+Taguchiauthor=M.+Shimadaauthor=M.+Moriauthor=D.+N.+Kragauthor=S.+Arii&title=Specific+peptide+ligand+for+Grb7+signal+transduction+protein+and+pancreatic+cancer+metastasis&doi=10.1093%2Fjnci%2Fdjj105"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Specific Peptide Ligand for Grb7 Signal Transduction Protein and Pancreatic Cancer Metastasis</span></div><div class="casAuthors">Tanaka, Shinji; Pero, Stephanie C.; Taguchi, Kenichi; Shimada, Mitsuo; Mori, Masaki; Krag, David N.; Arii, Shigeki</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the National Cancer Institute</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">491-498</span>CODEN:
                <span class="NLM_cas:coden">JNCIEQ</span>;
        ISSN:<span class="NLM_cas:issn">0027-8874</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Background: Pancreatic cancer is one of the most aggressive malignancies, with high rates of invasion and metastasis and with generally poor prognosis.  We previously found that metastasis was strongly assocd. with the expression of growth factor receptor-bound protein 7 (Grb7), which contains a Src homol. 2 (SH2) domain.  In this study, we evaluated Grb7 protein as a mol. target of therapy for metastatic pancreatic cancer.  Methods: Grb7 protein expression was measured by immunohistochem. in 36 human pancreatic cancer specimens and adjacent normal pancreatic tissue.  We synthesized a nonphosphorylated peptide inhibitor that binds specifically to the SH2 domain of Grb7.  Intracellular signaling was assessed by immunopptn. and immunoblot assays in cultured human pancreatic cancer cells.  Cell migration was measured with a modified Boyden chamber method.  Peritoneal metastasis of the pancreatic cancer cells was measured with a mouse model.  All statistical tests were two-sided.  Results: We found that 22 (61%) of 36 pancreatic cancer specimens had higher levels of Grb7 protein than their corresponding normal pancreatic tissue specimens.  Grb7 expression was statistically significantly different between specimens from patients without lymph node metastasis (stage N0; two of the 10 patients) and patients with lymph node metastasis (stages N1 + N2; 20 of the 26 patients) (P = .006).  The Grb7 peptide inhibitor selectively blocked the interaction between Grb7 and focal adhesion kinase and blocked the phosphorylation of Grb7 protein.  In vivo Grb7 peptide inhibitor statistically significantly attenuated cell migration (for control peptide, 87.5 cells migrated, 95% confidence interval [CI] = 82.6 to 92.4 cells; for Grb7 peptide, 5.7 cells migrated, 95% CI = 2.3 to 9.0 cells; P<.001) and peritoneal metastasis of the pancreatic cancer cells in a mouse model, as assessed by the no. of nodules (control = 72.6 nodules, 95% CI = 55.8 to 89.4 nodules; and for Grb7 peptide = 3.2 nodules, 95% CI = 1.6 to 4.8 nodules; P<.001, t test) and their wt. (control = 4.13 g, 95% CI = 3.40 to 4.86 g; Grb7 peptide = 0.19 g, 95% CI = 0.06 to 0.32 g; P<.001, t test).  Conclusions: The Grb7 peptide inhibitor appears to be a promising molecularly targeted therapeutic agent against metastatic pancreatic cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpletsYvkiEs7Vg90H21EOLACvtfcHk0lgH8KLwm1zjCA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XjtlamtL8%253D&md5=8bf8b5624a76b39cde03cd269bb16f27</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1093%2Fjnci%2Fdjj105&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjnci%252Fdjj105%26sid%3Dliteratum%253Aachs%26aulast%3DTanaka%26aufirst%3DS.%26aulast%3DPero%26aufirst%3DS.%2BC.%26aulast%3DTaguchi%26aufirst%3DK.%26aulast%3DShimada%26aufirst%3DM.%26aulast%3DMori%26aufirst%3DM.%26aulast%3DKrag%26aufirst%3DD.%2BN.%26aulast%3DArii%26aufirst%3DS.%26atitle%3DSpecific%2520peptide%2520ligand%2520for%2520Grb7%2520signal%2520transduction%2520protein%2520and%2520pancreatic%2520cancer%2520metastasis%26jtitle%3DJ.%2520Natl.%2520Cancer%2520Inst.%26date%3D2006%26volume%3D98%26spage%3D491%26epage%3D498%26doi%3D10.1093%2Fjnci%2Fdjj105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chan, D. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, V. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chiu, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ngan, H. Y.</span><span> </span><span class="NLM_article-title">Differential functions of growth factor receptor-bound protein 7 (GRB7) and its variant GRB7v in ovarian carcinogenesis</span> <span class="citation_source-journal">Clin. Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">2529</span><span class="NLM_x">–</span> <span class="NLM_lpage">2539</span><span class="refDoi"> DOI: 10.1158/1078-0432.CCR-10-0018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=10.1158%2F1078-0432.CCR-10-0018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=20388850" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=1%3ACAS%3A528%3ADC%252BC3cXltlegtrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2010&pages=2529-2539&author=Y.+Wangauthor=D.+W.+Chanauthor=V.+W.+Liuauthor=P.+Chiuauthor=H.+Y.+Ngan&title=Differential+functions+of+growth+factor+receptor-bound+protein+7+%28GRB7%29+and+its+variant+GRB7v+in+ovarian+carcinogenesis&doi=10.1158%2F1078-0432.CCR-10-0018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Differential Functions of Growth Factor Receptor-Bound Protein 7 (GRB7) and Its Variant GRB7v in Ovarian Carcinogenesis</span></div><div class="casAuthors">Wang, Yajun; Chan, David W.; Liu, Vincent W. S.; Chiu, P. M.; Ngan, Hextan Y. S.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2529-2539</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">PURPOSE: Aberrant overexpression of growth factor receptor-bound protein 7 (GRB7) and its variant GRB7v has been found in numerous human cancers.  The goal of this study was to characterize the functions of GRB7 and GRB7v in the ovarian carcinogenesis and to investigate the differential roles of GRB7 and GRB7v in the modulation of signaling pathways.  Exptl. Design: Quant. reverse transcription-PCR, Western blot, and immunohistochem. analyses were used to evaluate the levels of GRB7 and GRB7v.  The cellular localization, functions, and signaling pathways regulated by GRB7 and GRB7v were investigated by enforced expression of GRB7 and GRB7v.  RESULTS: Quant. reverse transcription-PCR and Western blot analyses showed that GRB7 and GRB7v were frequently upregulated in ovarian cancer samples.  The overexpressed GRB7 (P = 0.009) and GRB7v (P = 0.017) were significantly correlated with high-grade ovarian cancer.  Immunohistochem. anal. on ovarian cancer tissue array confirmed that the upregulated GRB7 was significantly correlated with high-grade ovarian cancer (P = 0.001).  Confocal microscopy anal. showed that GRB7 and GRB7v predominately localized in cytoplasm of ovarian cancer cells, consistent with their roles as signaling adaptors.  Enforced expression of GRB7 promoted cell proliferation, migration, and invasion, whereas GRB7v only increased cell proliferation and anchorage-independent growth ability.  With the treatment of specific kinase inhibitors, we showed that both GRB7 and GRB7v promoted cell proliferation through activating extracellular signal-regulated kinase signaling, whereas GRB7 enhanced cell migration/invasion by activating c-Jun NH2 terminal kinase signaling.  CONCLUSIONS: Our studies implicate that the overexpressed GRB7 and GRB7v are assocd. with high-grade tumors and exert distinct tumorigenic functions through regulating different signaling pathways in ovarian cancer cells.  Clin Cancer Res; 16(9); 2529-39.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdUrMOZ0FjhbVg90H21EOLACvtfcHk0lgsSyAQHRzz-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXltlegtrs%253D&md5=6d74a2b3a3ce1fafe4ff2c27ed6ba323</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-10-0018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-10-0018%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DY.%26aulast%3DChan%26aufirst%3DD.%2BW.%26aulast%3DLiu%26aufirst%3DV.%2BW.%26aulast%3DChiu%26aufirst%3DP.%26aulast%3DNgan%26aufirst%3DH.%2BY.%26atitle%3DDifferential%2520functions%2520of%2520growth%2520factor%2520receptor-bound%2520protein%25207%2520%2528GRB7%2529%2520and%2520its%2520variant%2520GRB7v%2520in%2520ovarian%2520carcinogenesis%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2010%26volume%3D16%26spage%3D2529%26epage%3D2539%26doi%3D10.1158%2F1078-0432.CCR-10-0018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Slamon, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clark, G. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong, S. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Levin, W. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ullrich, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McGuire, W. L.</span><span> </span><span class="NLM_article-title">Human breast cancer: correlation of relapse and survival with amplification of the HER-2/neu oncogene</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1987</span><span class="NLM_x">, </span> <span class="NLM_volume">235</span><span class="NLM_x">, </span> <span class="NLM_fpage">177</span><span class="NLM_x">–</span> <span class="NLM_lpage">182</span><span class="refDoi"> DOI: 10.1126/science.3798106</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=10.1126%2Fscience.3798106" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=3798106" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=1%3ACAS%3A528%3ADyaL2sXhtVSht7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=235&publication_year=1987&pages=177-182&author=D.+J.+Slamonauthor=G.+M.+Clarkauthor=S.+G.+Wongauthor=W.+J.+Levinauthor=A.+Ullrichauthor=W.+L.+McGuire&title=Human+breast+cancer%3A+correlation+of+relapse+and+survival+with+amplification+of+the+HER-2%2Fneu+oncogene&doi=10.1126%2Fscience.3798106"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Human breast cancer:  correlation of relapse and survival with amplification of the HER-2/neu oncogene</span></div><div class="casAuthors">Slamon, Dennis J.; Clark, Gary M.; Wong, Steven G.; Levin, Wendy J.; Ullrich, Axel; McGuire, William L.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">1987</span>),
    <span class="NLM_cas:volume">235</span>
        (<span class="NLM_cas:issue">4785</span>),
    <span class="NLM_cas:pages">177-82</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    </div><div class="casAbstract">The HER-2/neu oncogene is a member of the erbB-like oncogene family, and is related to, but distinct from, the epidermal growth factor receptor.  This gene has been shown to be amplified in human breast cancer cell lines.  In the current study, alterations of the gene in 189 primary human breast cancers were investigated.  HER-2/neu was found to be amplified from 2- to >20-fold in 30% of the tumors.  Correlation of gene amplification with several disease parameters was evaluated.  Amplification of the HER-2neu gene was a significant predictor of both overall survival and time to relapse in patients with breast cancer.  It retained its significance even when adjustments were made for other known prognostic factors.  Moreover, HER-2/neu amplification had greater prognostic value than most currently used prognostic factors, including hormonal-receptor status, in lymph node-pos. disease.  These data indicate that this gene may play a role in the biol. behavior and(or) pathogenesis of human breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWPpNbN8S_QrVg90H21EOLACvtfcHk0lgsSyAQHRzz-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL2sXhtVSht7s%253D&md5=0889410d6e47651de4d8085e9bbd70d9</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1126%2Fscience.3798106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.3798106%26sid%3Dliteratum%253Aachs%26aulast%3DSlamon%26aufirst%3DD.%2BJ.%26aulast%3DClark%26aufirst%3DG.%2BM.%26aulast%3DWong%26aufirst%3DS.%2BG.%26aulast%3DLevin%26aufirst%3DW.%2BJ.%26aulast%3DUllrich%26aufirst%3DA.%26aulast%3DMcGuire%26aufirst%3DW.%2BL.%26atitle%3DHuman%2520breast%2520cancer%253A%2520correlation%2520of%2520relapse%2520and%2520survival%2520with%2520amplification%2520of%2520the%2520HER-2%252Fneu%2520oncogene%26jtitle%3DScience%26date%3D1987%26volume%3D235%26spage%3D177%26epage%3D182%26doi%3D10.1126%2Fscience.3798106" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Pradip, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bouzyk, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dey, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leyland-Jones, B.</span><span> </span><span class="NLM_article-title">Dissecting GRB7-mediated signals for proliferation and migration in HER2 overexpressing breast tumor cells: GTP-ase rules</span> <span class="citation_source-journal">Am. J. Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">173</span><span class="NLM_x">–</span> <span class="NLM_lpage">195</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=23593540" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=1%3ACAS%3A280%3ADC%252BC3srlvFOqtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2013&pages=173-195&author=D.+Pradipauthor=M.+Bouzykauthor=N.+Deyauthor=B.+Leyland-Jones&title=Dissecting+GRB7-mediated+signals+for+proliferation+and+migration+in+HER2+overexpressing+breast+tumor+cells%3A+GTP-ase+rules"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Dissecting GRB7-mediated signals for proliferation and migration in HER2 overexpressing breast tumor cells: GTP-ase rules</span></div><div class="casAuthors">Pradip De; Bouzyk Mark; Dey Nandini; Leyland-Jones Brian</div><div class="citationInfo"><span class="NLM_cas:title">American journal of cancer research</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">173-95</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Amplification of human Her2 and its aberrant signaling in 20-30% of early breast cancer patients is responsible for highly aggressive tumors with poor outcome.  Grb7 is reported to be co-amplified with Her2.  We report a concurrent high expression of mRNA (from FFPE tumor samples; mRNA correlation, Pearson r(2)= 0.806), and high levels of GRB7 protein (immunoblot) in HER2+ breast cancer cell lines.  We demonstrated the signaling mechanism of HER2 and downstream effectors that contributes to proliferation and migration.  Using HER2+ and trastuzumab-resistant breast cancer cell lines, we identified the interaction between GRB7 and HER2 in the control of HER2+ cell proliferation.  Our co-IP data show that GRB7 recruits SHC into the HER2-GRB7 signaling complex.  This complex formation leads to activation of RAS-GTP.  We also observed that following integrin engagement, GRB7 is phosphorylated at tyrosine in a p-FAK (Y397) dependent manner.  This FAK-GRB7 complex leads to downstream activation of RAC1-GTP (responsible for migration) probably through the recruitment of VAV2.  Our CO-IP data demonstrate that GRB7 directly binds with VAV2 following fibronectin engagement in HER2+ cells.  To address whether GRB7 could serve as a pathway specific therapeutic target, we used siRNA to suppress GRB7 expression.  Knockdown of GRB7 expression in the HER2+ breast cancer cell lines decreases RAS activation, cell proliferation, 2D and 3D colony formation and also blocked integrin-mediated RAC1 activation along with integrin-directed cell migration.  These findings dissected the HER2-mediated signaling cascade into (1) HER2+ cell proliferation (HER2-GRB7-SHC-RAS) and (2) HER2+ cell migration (alpha5 beta1/alpha4 beta1-FAK-GRB7-VAV2-RAC1).  Our data clearly demonstrate that a coupling of GRB7 with HER2 is required for the proliferative and migratory signals in HER2+ breast tumor cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQo49zaaj9yMMyail4HZd-ifW6udTcc2ebdFDLtN53Pmbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3srlvFOqtw%253D%253D&md5=a09635d02e188705bcb24085c4b94c4b</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DPradip%26aufirst%3DD.%26aulast%3DBouzyk%26aufirst%3DM.%26aulast%3DDey%26aufirst%3DN.%26aulast%3DLeyland-Jones%26aufirst%3DB.%26atitle%3DDissecting%2520GRB7-mediated%2520signals%2520for%2520proliferation%2520and%2520migration%2520in%2520HER2%2520overexpressing%2520breast%2520tumor%2520cells%253A%2520GTP-ase%2520rules%26jtitle%3DAm.%2520J.%2520Cancer%2520Res.%26date%3D2013%26volume%3D3%26spage%3D173%26epage%3D195" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Han, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guan, J. L.</span><span> </span><span class="NLM_article-title">Association of focal adhesion kinase with Grb7 and its role in cell migration</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">274</span><span class="NLM_x">, </span> <span class="NLM_fpage">24425</span><span class="NLM_x">–</span> <span class="NLM_lpage">24430</span><span class="refDoi"> DOI: 10.1074/jbc.274.34.24425</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=10.1074%2Fjbc.274.34.24425" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=274&publication_year=1999&pages=24425-24430&author=D.+C.+Hanauthor=J.+L.+Guan&title=Association+of+focal+adhesion+kinase+with+Grb7+and+its+role+in+cell+migration&doi=10.1074%2Fjbc.274.34.24425"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1074%2Fjbc.274.34.24425&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.274.34.24425%26sid%3Dliteratum%253Aachs%26aulast%3DHan%26aufirst%3DD.%2BC.%26aulast%3DGuan%26aufirst%3DJ.%2BL.%26atitle%3DAssociation%2520of%2520focal%2520adhesion%2520kinase%2520with%2520Grb7%2520and%2520its%2520role%2520in%2520cell%2520migration%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1999%26volume%3D274%26spage%3D24425%26epage%3D24430%26doi%3D10.1074%2Fjbc.274.34.24425" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Ramsey, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bai, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hanlon Newell, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Troxell, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Park, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olson, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keenan, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luoh, S. W.</span><span> </span><span class="NLM_article-title">GRB7 protein over-expression and clinical outcome in breast cancer</span> <span class="citation_source-journal">Breast Cancer Res. Treat.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">127</span><span class="NLM_x">, </span> <span class="NLM_fpage">659</span><span class="NLM_x">–</span> <span class="NLM_lpage">669</span><span class="refDoi"> DOI: 10.1007/s10549-010-1010-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=10.1007%2Fs10549-010-1010-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=20635137" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmtlyku7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=127&publication_year=2011&pages=659-669&author=B.+Ramseyauthor=T.+Baiauthor=A.+Hanlon+Newellauthor=M.+Troxellauthor=B.+Parkauthor=S.+Olsonauthor=E.+Keenanauthor=S.+W.+Luoh&title=GRB7+protein+over-expression+and+clinical+outcome+in+breast+cancer&doi=10.1007%2Fs10549-010-1010-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">GRB7 protein over-expression and clinical outcome in breast cancer</span></div><div class="casAuthors">Ramsey, Betsy; Bai, Tao; Newell, Amy Hanlon; Troxell, Megan; Park, Byung; Olson, Susan; Keenan, Edward; Luoh, Shiuh-Wen</div><div class="citationInfo"><span class="NLM_cas:title">Breast Cancer Research and Treatment</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">127</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">659-669</span>CODEN:
                <span class="NLM_cas:coden">BCTRD6</span>;
        ISSN:<span class="NLM_cas:issn">0167-6806</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">The growth factor receptor-bound protein-7 gene (GRB7) encodes a multi-domain signal transduction mol.  The purpose of this study was to examine the clin. significance of GRB7 protein expression in human breast cancer.  Western blotting anal. of protein exts. from 563 annotated frozen breast tumors was performed.  Expression status of GRB7 and HER-2 was correlated with clin. covariates and outcomes.  Cox proportional hazards were used to identify factors assocd. with breast cancer-free interval.  The median follow-up was 71 mo.  P values <0.05 were considered statistically significant (two-sided).  A discrepancy between HER-2 and GRB7 protein over-expression was obsd.  GRB7 protein over-expression was assocd. with neg. estrogen and progesterone receptor status, higher tumor grade, larger primary tumor size, (more) axillary lymph node involvement, higher clin. stage, and shortened breast cancer-free interval.  HER-2 protein over-expression was assocd. only with higher tumor grade.  Multi-variate anal. revealed that GRB7 protein over-expression was an independent adverse prognostic factor for breast cancer-free interval (hazard ratio 1.69, 95% confidence interval 1.07-2.67; P = 0.024).  The same was true of the subset of patients who did not receive any adjuvant systemic therapy (hazard ratio 1.68, 95% confidence interval 1.16-2.31; P = 0.0055).  Using FISH anal., 32/32 (100%; 95% CI 89-100%) tumors which over-expressed both HER-2 and GRB7 proteins and 1/35 (3%; 95% CI 0-15%) tumors with HER-2 but no GRB7 protein over-expression with Western blotting anal. demonstrated HER-2 gene amplification.  GRB7 protein over-expression is an independent adverse prognostic factor in human breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrChZB-XTObp7Vg90H21EOLACvtfcHk0lg4FF7XYTXp2A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmtlyku7Y%253D&md5=600c4cb4b85ba9646ee9a9ee2e21864c</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1007%2Fs10549-010-1010-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10549-010-1010-0%26sid%3Dliteratum%253Aachs%26aulast%3DRamsey%26aufirst%3DB.%26aulast%3DBai%26aufirst%3DT.%26aulast%3DHanlon%2BNewell%26aufirst%3DA.%26aulast%3DTroxell%26aufirst%3DM.%26aulast%3DPark%26aufirst%3DB.%26aulast%3DOlson%26aufirst%3DS.%26aulast%3DKeenan%26aufirst%3DE.%26aulast%3DLuoh%26aufirst%3DS.%2BW.%26atitle%3DGRB7%2520protein%2520over-expression%2520and%2520clinical%2520outcome%2520in%2520breast%2520cancer%26jtitle%3DBreast%2520Cancer%2520Res.%2520Treat.%26date%3D2011%26volume%3D127%26spage%3D659%26epage%3D669%26doi%3D10.1007%2Fs10549-010-1010-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Chu, P. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, T. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ding, S. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lai, I. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, T. L.</span><span> </span><span class="NLM_article-title">EGF-induced Grb7 recruits and promotes Ras activity essential for the tumorigenicity of Sk-Br3 breast cancer cells</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">285</span><span class="NLM_x">, </span> <span class="NLM_fpage">29279</span><span class="NLM_x">–</span> <span class="NLM_lpage">29285</span><span class="refDoi"> DOI: 10.1074/jbc.C110.114124</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=10.1074%2Fjbc.C110.114124" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=285&publication_year=2010&pages=29279-29285&author=P.+Y.+Chuauthor=T.+K.+Liauthor=S.+T.+Dingauthor=I.+R.+Laiauthor=T.+L.+Shen&title=EGF-induced+Grb7+recruits+and+promotes+Ras+activity+essential+for+the+tumorigenicity+of+Sk-Br3+breast+cancer+cells&doi=10.1074%2Fjbc.C110.114124"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1074%2Fjbc.C110.114124&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.C110.114124%26sid%3Dliteratum%253Aachs%26aulast%3DChu%26aufirst%3DP.%2BY.%26aulast%3DLi%26aufirst%3DT.%2BK.%26aulast%3DDing%26aufirst%3DS.%2BT.%26aulast%3DLai%26aufirst%3DI.%2BR.%26aulast%3DShen%26aufirst%3DT.%2BL.%26atitle%3DEGF-induced%2520Grb7%2520recruits%2520and%2520promotes%2520Ras%2520activity%2520essential%2520for%2520the%2520tumorigenicity%2520of%2520Sk-Br3%2520breast%2520cancer%2520cells%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2010%26volume%3D285%26spage%3D29279%26epage%3D29285%26doi%3D10.1074%2Fjbc.C110.114124" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Giricz, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Calvo, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pero, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krag, D. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sparano, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kenny, P. A.</span><span> </span><span class="NLM_article-title">GRB7 is required for triple-negative breast cancer cell invasion and survival</span> <span class="citation_source-journal">Breast Cancer Res. Treat.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">133</span><span class="NLM_x">, </span> <span class="NLM_fpage">607</span><span class="NLM_x">–</span> <span class="NLM_lpage">615</span><span class="refDoi"> DOI: 10.1007/s10549-011-1822-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=10.1007%2Fs10549-011-1822-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=22005836" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=1%3ACAS%3A528%3ADC%252BC38XnvFOksbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=133&publication_year=2012&pages=607-615&author=O.+Giriczauthor=V.+Calvoauthor=S.+C.+Peroauthor=D.+N.+Kragauthor=J.+A.+Sparanoauthor=P.+A.+Kenny&title=GRB7+is+required+for+triple-negative+breast+cancer+cell+invasion+and+survival&doi=10.1007%2Fs10549-011-1822-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">GRB7 is required for triple-negative breast cancer cell invasion and survival</span></div><div class="casAuthors">Giricz, Orsi; Calvo, Veronica; Pero, Stephanie C.; Krag, David N.; Sparano, Joseph A.; Kenny, Paraic A.</div><div class="citationInfo"><span class="NLM_cas:title">Breast Cancer Research and Treatment</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">133</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">607-615</span>CODEN:
                <span class="NLM_cas:coden">BCTRD6</span>;
        ISSN:<span class="NLM_cas:issn">0167-6806</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Triple-neg. breast cancer (TNBC) is a heterogeneous disease that is usually assocd. with poor prognosis, and frequently assocd. with the basal-like breast cancer gene expression profile.  There are no targeted therapeutic modalities for this disease, and no useful biomarkers.  High GRB7 RNA expression levels are assocd. with an elevated risk of recurrence in patients with operable TNBC treated with std. adjuvant anthracycline and taxane therapy.  To det. whether GRB7 is involved in the pathobiol. of TNBC, we evaluated the biol. effects of GRB7 inhibition in a panel of triple-neg. cell lines-MDA-MB-468, MDA-MB-231, HCC70, and T4-2.  We found GRB7 inhibition reduced cell motility and invasion of these cell lines and promoted cell death by apoptosis in 3D culture.  These data suggest that GRB7 itself, or GRB7-dependent pathways, may prove to be important therapeutic targets in this disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreB0Epb_bbMLVg90H21EOLACvtfcHk0lg4FF7XYTXp2A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XnvFOksbc%253D&md5=8ad42003c4e5536023f81377e6f1bbc1</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1007%2Fs10549-011-1822-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10549-011-1822-6%26sid%3Dliteratum%253Aachs%26aulast%3DGiricz%26aufirst%3DO.%26aulast%3DCalvo%26aufirst%3DV.%26aulast%3DPero%26aufirst%3DS.%2BC.%26aulast%3DKrag%26aufirst%3DD.%2BN.%26aulast%3DSparano%26aufirst%3DJ.%2BA.%26aulast%3DKenny%26aufirst%3DP.%2BA.%26atitle%3DGRB7%2520is%2520required%2520for%2520triple-negative%2520breast%2520cancer%2520cell%2520invasion%2520and%2520survival%26jtitle%3DBreast%2520Cancer%2520Res.%2520Treat.%26date%3D2012%26volume%3D133%26spage%3D607%26epage%3D615%26doi%3D10.1007%2Fs10549-011-1822-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Lim, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Price, J. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilce, J. A.</span><span> </span><span class="NLM_article-title">Context-dependent role of Grb7 in HER2+ve and triple-negative breast cancer cell lines</span> <span class="citation_source-journal">Breast Cancer Res. Treat.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">143</span><span class="NLM_x">, </span> <span class="NLM_fpage">593</span><span class="NLM_x">–</span> <span class="NLM_lpage">603</span><span class="refDoi"> DOI: 10.1007/s10549-014-2838-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=10.1007%2Fs10549-014-2838-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=24464577" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtlSqsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=143&publication_year=2014&pages=593-603&author=R.+C.+Limauthor=J.+T.+Priceauthor=J.+A.+Wilce&title=Context-dependent+role+of+Grb7+in+HER2%2Bve+and+triple-negative+breast+cancer+cell+lines&doi=10.1007%2Fs10549-014-2838-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Context-dependent role of Grb7 in HER2+ve and triple-negative breast cancer cell lines</span></div><div class="casAuthors">Lim, Reece C. C.; Price, John T.; Wilce, Jacqueline A.</div><div class="citationInfo"><span class="NLM_cas:title">Breast Cancer Research and Treatment</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">143</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">593-603</span>CODEN:
                <span class="NLM_cas:coden">BCTRD6</span>;
        ISSN:<span class="NLM_cas:issn">0167-6806</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Grb7 is an adapter protein, aberrantly co-overexpressed with HER2 and identified as an independent prognostic marker in breast cancer.  It has been established that Grb7 exacerbates the cellular growth and migratory behavior of HER2+ve breast cancer cells.  Less is known about Grb7's role in the context of HER2-ve cells.  Here we directly compare the effect of stable Grb7 knockdown in estrogen sensitive (T47D), HER2+ve (SKBR3) and triple-neg. (MDA-MB-468 and MDA-MB-231) breast cancer cell lines on anchorage dependent and independent cell growth, wound healing and chemotaxis.  All cell lines showed reduced ability to migrate upon Grb7 knockdown, despite their greatly varied endogenous levels of Grb7.  Decreased cell proliferation was not obsd. in any of the cell lines upon Grb7 knockdown; however, decreased ability to form colonies was obsd. for all but the estrogen sensitive cell line, depending upon the stringency of the growth conditions.  The data reveal that Grb7 plays an important role in breast cancer progression, beyond the context of HER2+ve cell types.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4vHleefRXL7Vg90H21EOLACvtfcHk0lhja__zEn_G1Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtlSqsr8%253D&md5=121ce4f7e3fe1fa16c790a63bdf6dce4</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1007%2Fs10549-014-2838-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10549-014-2838-5%26sid%3Dliteratum%253Aachs%26aulast%3DLim%26aufirst%3DR.%2BC.%26aulast%3DPrice%26aufirst%3DJ.%2BT.%26aulast%3DWilce%26aufirst%3DJ.%2BA.%26atitle%3DContext-dependent%2520role%2520of%2520Grb7%2520in%2520HER2%252Bve%2520and%2520triple-negative%2520breast%2520cancer%2520cell%2520lines%26jtitle%3DBreast%2520Cancer%2520Res.%2520Treat.%26date%3D2014%26volume%3D143%26spage%3D593%26epage%3D603%26doi%3D10.1007%2Fs10549-014-2838-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Pero, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oligino, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Daly, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soden, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roller, P. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krag, D. N.</span><span> </span><span class="NLM_article-title">Identification of novel non-phosphorylated ligands, which bind selectively to the SH2 domain of Grb7</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">277</span><span class="NLM_x">, </span> <span class="NLM_fpage">11918</span><span class="NLM_x">–</span> <span class="NLM_lpage">11926</span><span class="refDoi"> DOI: 10.1074/jbc.M111816200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=10.1074%2Fjbc.M111816200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=277&publication_year=2002&pages=11918-11926&author=S.+C.+Peroauthor=L.+Oliginoauthor=R.+J.+Dalyauthor=A.+L.+Sodenauthor=C.+Liuauthor=P.+P.+Rollerauthor=P.+Liauthor=D.+N.+Krag&title=Identification+of+novel+non-phosphorylated+ligands%2C+which+bind+selectively+to+the+SH2+domain+of+Grb7&doi=10.1074%2Fjbc.M111816200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M111816200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M111816200%26sid%3Dliteratum%253Aachs%26aulast%3DPero%26aufirst%3DS.%2BC.%26aulast%3DOligino%26aufirst%3DL.%26aulast%3DDaly%26aufirst%3DR.%2BJ.%26aulast%3DSoden%26aufirst%3DA.%2BL.%26aulast%3DLiu%26aufirst%3DC.%26aulast%3DRoller%26aufirst%3DP.%2BP.%26aulast%3DLi%26aufirst%3DP.%26aulast%3DKrag%26aufirst%3DD.%2BN.%26atitle%3DIdentification%2520of%2520novel%2520non-phosphorylated%2520ligands%252C%2520which%2520bind%2520selectively%2520to%2520the%2520SH2%2520domain%2520of%2520Grb7%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2002%26volume%3D277%26spage%3D11918%26epage%3D11926%26doi%3D10.1074%2Fjbc.M111816200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Watson, G. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gunzburg, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ambaye, N. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lucas, W. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Traore, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kulkarni, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cergol, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Payne, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Panjikar, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pero, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perlmutter, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilce, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilce, J. A.</span><span> </span><span class="NLM_article-title">Cyclic peptides incorporating phosphotyrosine mimetics as potent and specific inhibitors of the Grb7 breast cancer target</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">7707</span><span class="NLM_x">–</span> <span class="NLM_lpage">7718</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00609</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00609" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVyktrrF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=7707-7718&author=G.+M.+Watsonauthor=M.+J.+Gunzburgauthor=N.+D.+Ambayeauthor=W.+A.+Lucasauthor=D.+A.+Traoreauthor=K.+Kulkarniauthor=K.+M.+Cergolauthor=R.+J.+Payneauthor=S.+Panjikarauthor=S.+C.+Peroauthor=P.+Perlmutterauthor=M.+C.+Wilceauthor=J.+A.+Wilce&title=Cyclic+peptides+incorporating+phosphotyrosine+mimetics+as+potent+and+specific+inhibitors+of+the+Grb7+breast+cancer+target&doi=10.1021%2Facs.jmedchem.5b00609"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Cyclic Peptides Incorporating Phosphotyrosine Mimetics as Potent and Specific Inhibitors of the Grb7 Breast Cancer Target</span></div><div class="casAuthors">Watson, Gabrielle M.; Gunzburg, Menachem J.; Ambaye, Nigus D.; Lucas, William A. H.; Traore, Daouda A.; Kulkarni, Ketav; Cergol, Katie M.; Payne, Richard J.; Panjikar, Santosh; Pero, Stephanie C.; Perlmutter, Patrick; Wilce, Matthew C. J.; Wilce, Jacqueline A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">7707-7718</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The Grb7 adaptor protein is a therapeutic target for both TNBC and HER2+ breast cancer.  A nonphosphorylated cyclic peptide 1 (known as G7-18NATE) inhibits Grb7 via targeting the Grb7-SH2 domain, but requires the presence of phosphate ions for both affinity and specificity.  Here, the authors report the discovery of malonate bound in the phosphotyrosine binding pocket of the apo-Grb7-SH2 structure.  Based on this, the authors carried out the rational design and synthesis of two analogs of peptide 1 that incorporate carboxymethylphenylalanine (cmF) and carboxyphenylalanine (cF) as mimics of phosphotyrosine (pY).  Binding studies using SPR confirmed that affinity for Grb7-SH2 domain is improved up to 9-fold over peptide 1 under physiol. phosphate conditions (KD = 2.1-5.7 μM) and that binding is specific for Grb7-SH2 over closely related domains (low or no detectable binding to Grb2-SH2 and Grb10-SH2).  X-ray crystallog. structural anal. of the analog bearing a cmF moiety in complex with Grb7-SH2 has identified the precise contacts conferred by the pY mimic that underpin this improved mol. interaction.  This study identifies and characterizes the tightest specific inhibitor of Grb7 to date, representing a significant development toward a new Grb7-targeted therapeutic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBV3XGn3QVZLVg90H21EOLACvtfcHk0lhja__zEn_G1Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVyktrrF&md5=c28d4bbd140c6eced3e6256964461612</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00609&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00609%26sid%3Dliteratum%253Aachs%26aulast%3DWatson%26aufirst%3DG.%2BM.%26aulast%3DGunzburg%26aufirst%3DM.%2BJ.%26aulast%3DAmbaye%26aufirst%3DN.%2BD.%26aulast%3DLucas%26aufirst%3DW.%2BA.%26aulast%3DTraore%26aufirst%3DD.%2BA.%26aulast%3DKulkarni%26aufirst%3DK.%26aulast%3DCergol%26aufirst%3DK.%2BM.%26aulast%3DPayne%26aufirst%3DR.%2BJ.%26aulast%3DPanjikar%26aufirst%3DS.%26aulast%3DPero%26aufirst%3DS.%2BC.%26aulast%3DPerlmutter%26aufirst%3DP.%26aulast%3DWilce%26aufirst%3DM.%2BC.%26aulast%3DWilce%26aufirst%3DJ.%2BA.%26atitle%3DCyclic%2520peptides%2520incorporating%2520phosphotyrosine%2520mimetics%2520as%2520potent%2520and%2520specific%2520inhibitors%2520of%2520the%2520Grb7%2520breast%2520cancer%2520target%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D7707%26epage%3D7718%26doi%3D10.1021%2Facs.jmedchem.5b00609" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Gunzburg, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kulkarni, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Watson, G. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ambaye, N. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Del Borgo, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brandt, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pero, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perlmutter, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilce, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilce, J. A.</span><span> </span><span class="NLM_article-title">Unexpected involvement of staple leads to redesign of selective bicyclic peptide inhibitor of Grb7</span> <span class="citation_source-journal">Sci. Rep.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">27060</span><span class="refDoi"> DOI: 10.1038/srep27060</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=10.1038%2Fsrep27060" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=27257138" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=1%3ACAS%3A528%3ADC%252BC28XpsFCgsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2016&pages=27060&author=M.+J.+Gunzburgauthor=K.+Kulkarniauthor=G.+M.+Watsonauthor=N.+D.+Ambayeauthor=M.+P.+Del+Borgoauthor=R.+Brandtauthor=S.+C.+Peroauthor=P.+Perlmutterauthor=M.+C.+Wilceauthor=J.+A.+Wilce&title=Unexpected+involvement+of+staple+leads+to+redesign+of+selective+bicyclic+peptide+inhibitor+of+Grb7&doi=10.1038%2Fsrep27060"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Unexpected involvement of staple leads to redesign of selective bicyclic peptide inhibitor of Grb7</span></div><div class="casAuthors">Gunzburg, Menachem J.; Kulkarni, Ketav; Watson, Gabrielle M.; Ambaye, Nigus D.; Del Borgo, Mark P.; Brandt, Rebecca; Pero, Stephanie C.; Perlmutter, Patrick; Wilce, Matthew C. J.; Wilce, Jacqueline A.</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">27060</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The design of potent and specific peptide inhibitors to therapeutic targets is of enormous utility for both proof-of-concept studies and for the development of potential new therapeutics.  Grb7 is a key signaling mol. in the progression of HER2 pos. and triple neg. breast cancers.  Here we report the crystal structure of a stapled bicyclic peptide inhibitor G7-B1 in complex with the Grb7-SH2 domain.  This revealed an unexpected binding mode of the peptide, in which the staple forms an alternative contact with the surface of the target protein.  Based on this structural information, we designed a new series of bicyclic G7 peptides that progressively constrain the starting peptide, to arrive at the G7-B4 peptide that binds with an approx. 2-fold enhanced affinity to the Grb7-SH2 domain (KD = 0.83 μM) compared to G7-B1 and shows low affinity binding to Grb2-, Grb10- and Grb14-SH2 domains (KD > 100 μM).  Furthermore, we detd. the structure of the G7-B4 bicyclic peptide in complex with the Grb7-SH2 domain, both before and after ring closing metathesis to show that the closed staple is essential to the target interaction.  The G7-B4 peptide represents an advance in the development of Grb7 inhibitors and is a classical example of structure aided inhibitor development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGLvbEuvyHLrVg90H21EOLACvtfcHk0lh_IWn0-H_WTg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XpsFCgsLo%253D&md5=141b7b4db6aa6178e9449e628947d706</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1038%2Fsrep27060&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep27060%26sid%3Dliteratum%253Aachs%26aulast%3DGunzburg%26aufirst%3DM.%2BJ.%26aulast%3DKulkarni%26aufirst%3DK.%26aulast%3DWatson%26aufirst%3DG.%2BM.%26aulast%3DAmbaye%26aufirst%3DN.%2BD.%26aulast%3DDel%2BBorgo%26aufirst%3DM.%2BP.%26aulast%3DBrandt%26aufirst%3DR.%26aulast%3DPero%26aufirst%3DS.%2BC.%26aulast%3DPerlmutter%26aufirst%3DP.%26aulast%3DWilce%26aufirst%3DM.%2BC.%26aulast%3DWilce%26aufirst%3DJ.%2BA.%26atitle%3DUnexpected%2520involvement%2520of%2520staple%2520leads%2520to%2520redesign%2520of%2520selective%2520bicyclic%2520peptide%2520inhibitor%2520of%2520Grb7%26jtitle%3DSci.%2520Rep.%26date%3D2016%26volume%3D6%26spage%3D27060%26doi%3D10.1038%2Fsrep27060" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Pero, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shukla, G. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cookson, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flemer, S.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krag, D. N.</span><span> </span><span class="NLM_article-title">Combination treatment with Grb7 peptide and Doxorubicin or Trastuzumab (Herceptin) results in cooperative cell growth inhibition in breast cancer cells</span> <span class="citation_source-journal">Br. J. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">96</span><span class="NLM_x">, </span> <span class="NLM_fpage">1520</span><span class="NLM_x">–</span> <span class="NLM_lpage">1525</span><span class="refDoi"> DOI: 10.1038/sj.bjc.6603732</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=10.1038%2Fsj.bjc.6603732" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=17426702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=1%3ACAS%3A528%3ADC%252BD2sXlt1KmtLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=96&publication_year=2007&pages=1520-1525&author=S.+C.+Peroauthor=G.+S.+Shuklaauthor=M.+M.+Cooksonauthor=S.+Flemerauthor=D.+N.+Krag&title=Combination+treatment+with+Grb7+peptide+and+Doxorubicin+or+Trastuzumab+%28Herceptin%29+results+in+cooperative+cell+growth+inhibition+in+breast+cancer+cells&doi=10.1038%2Fsj.bjc.6603732"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Combination treatment with Grb7 peptide and Doxorubicin or Trastuzumab (Herceptin) results in cooperative cell growth inhibition in breast cancer cells</span></div><div class="casAuthors">Pero, S. C.; Shukla, G. S.; Cookson, M. M.; Flemer, S., Jr.; Krag, D. N.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Cancer</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">96</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1520-1525</span>CODEN:
                <span class="NLM_cas:coden">BJCAAI</span>;
        ISSN:<span class="NLM_cas:issn">0007-0920</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Grb7 has potential importance in the progression of cancer.  We have previously identified a novel peptide that binds to the SH2 domain of Grb7 and inhibits its assocn. with several different receptor tyrosine kinases.  We have synthesized the Grb7 peptide, G7-18NATE, with two different cell penetrating peptides, Penetratin and Tat.  In this study, we have shown that both Penetratin- and Tat-conjugated G7-18NATE peptides are able to inhibit the proliferation of SK-BR-3, ZR-75-30, MDA-MB-361 and MDA-MB-231 breast cancer cells.  There was no significant effects on breast cancer MCF-7cells, non-malignant MCF 10A or 3T3 cells.  In addn., there was no significant inhibition of proliferation by Penetratin or Tat alone or by their conjugates with arbitrary peptide sequence in any of the cell lines tested.  We detd. the EC50 of G7-18NATE-P peptide for SK-BR-3 cell proliferation to be 7.663×10-6 M.  Co-treatment of G7-18NATE-P peptide plus Doxorubicin in SK-BR-3 breast cancer cells resulted in an addnl. inhibition of proliferation, resulting in 56 and 84% decreases in the Doxorubicin EC50 value in the presence of 5×10-6 and 1.0×10-5 M G7-18NATE-P peptide, resp.  Importantly, the co-treatment with Doxorubicin and the delivery peptide did not change the Doxorubicin EC50.  Since Grb7 assocs. with ErbB2, we assessed whether the peptide inhibitor would have a combined effect with a mol. that targets ErbB2, Herceptin.  Co-treatment of Herceptin plus 1.0×10-5 M G7-18NATE-P peptide in SK-BR-3 cells resulted in a 46% decrease in the Herceptin EC50 value and no decrease following the co-treatment with Herceptin and penetratin alone.  This Grb7 peptide has potential to be developed as a therapeutic agent alone, in combination with traditional chemotherapy, or in combination with other targeting mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrtYqBV3GdCebVg90H21EOLACvtfcHk0lh_IWn0-H_WTg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXlt1KmtLc%253D&md5=ef6c823762d834e077e70657ea28162d</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1038%2Fsj.bjc.6603732&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.bjc.6603732%26sid%3Dliteratum%253Aachs%26aulast%3DPero%26aufirst%3DS.%2BC.%26aulast%3DShukla%26aufirst%3DG.%2BS.%26aulast%3DCookson%26aufirst%3DM.%2BM.%26aulast%3DFlemer%26aufirst%3DS.%26aulast%3DKrag%26aufirst%3DD.%2BN.%26atitle%3DCombination%2520treatment%2520with%2520Grb7%2520peptide%2520and%2520Doxorubicin%2520or%2520Trastuzumab%2520%2528Herceptin%2529%2520results%2520in%2520cooperative%2520cell%2520growth%2520inhibition%2520in%2520breast%2520cancer%2520cells%26jtitle%3DBr.%2520J.%2520Cancer%26date%3D2007%26volume%3D96%26spage%3D1520%26epage%3D1525%26doi%3D10.1038%2Fsj.bjc.6603732" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Gunzburg, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ambaye, N. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Del Borgo, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pero, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krag, D. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilce, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilce, J. A.</span><span> </span><span class="NLM_article-title">Interaction of the non-phosphorylated peptide G7–18NATE with Grb7-SH2 domain requires phosphate for enhanced affinity and specificity</span> <span class="citation_source-journal">J. Mol. Recognit.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">57</span><span class="NLM_x">–</span> <span class="NLM_lpage">67</span><span class="refDoi"> DOI: 10.1002/jmr.2148</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=10.1002%2Fjmr.2148" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=22213451" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs1yntrzK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2012&pages=57-67&author=M.+J.+Gunzburgauthor=N.+D.+Ambayeauthor=M.+P.+Del+Borgoauthor=S.+C.+Peroauthor=D.+N.+Kragauthor=M.+C.+Wilceauthor=J.+A.+Wilce&title=Interaction+of+the+non-phosphorylated+peptide+G7%E2%80%9318NATE+with+Grb7-SH2+domain+requires+phosphate+for+enhanced+affinity+and+specificity&doi=10.1002%2Fjmr.2148"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Interaction of the non-phosphorylated peptide G7-18NATE with Grb7-SH2 domain requires phosphate for enhanced affinity and specificity</span></div><div class="casAuthors">Gunzburg, Menachem J.; Ambaye, Nigus D.; Del Borgo, Mark P.; Pero, Stephanie C.; Krag, David N.; Wilce, Matthew C. J.; Wilce, Jacqueline A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Recognition</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">57-67</span>CODEN:
                <span class="NLM_cas:coden">JMORE4</span>;
        ISSN:<span class="NLM_cas:issn">0952-3499</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Src-homol. (SH2) domains are an attractive target for the inhibition of specific signaling pathways but pose the challenge of developing a truly specific inhibitor.  The G7-18NATE cyclic peptide is reported to specifically inhibit the growth factor receptor bound protein 7 (Grb7) adapter protein, implicated in the progression of several cancer types, via interactions with its SH2 domain.  G7-18NATE effectively inhibits the interaction of Grb7 with ErbB3 and focal adhesion kinase in cell lysates and, with the addn. of a cell permeability sequence, inhibits the growth and migration of a no. of breast cancer cell lines.  It is thus a promising lead in the development of therapeutics targeted to Grb7.  Here the authors investigate the degree to which G7-18NATE is specific for the Grb7-SH2 domain compared with closely related SH2 domains including those of Grb10, Grb14, and Grb2 using surface plasmon resonance.  The authors demonstrate that G7-18NATE binds with micromolar binding affinity to Grb7-SH2 domain (KD = 4-6 μm) compared with 50-200 times lower affinity for Grb10, Grb14, and Grb2 but that this specificity depends critically on the presence of phosphate in millimolar concns.  Other differences in buffer compn., including use of Tris or 2-(N-Morpholino)ethanesulfonic acid or varying the pH, do not impact on the interaction.  This suggests that under cellular conditions, G7-18NATE binds with highest affinity to Grb7.  In addn., the authors' findings demonstrate that the basis of specificity of G7-18NATE binding to the Grb7-SH2 domain is via other than intrinsic structural features of the protein, representing an unexpected mode of mol. recognition.  Copyright © 2011 John Wiley & Sons, Ltd.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVOz6dLoaKNLVg90H21EOLACvtfcHk0lh_IWn0-H_WTg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs1yntrzK&md5=77073aec5afc99cfbf1f3b7d62039b8f</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1002%2Fjmr.2148&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjmr.2148%26sid%3Dliteratum%253Aachs%26aulast%3DGunzburg%26aufirst%3DM.%2BJ.%26aulast%3DAmbaye%26aufirst%3DN.%2BD.%26aulast%3DDel%2BBorgo%26aufirst%3DM.%2BP.%26aulast%3DPero%26aufirst%3DS.%2BC.%26aulast%3DKrag%26aufirst%3DD.%2BN.%26aulast%3DWilce%26aufirst%3DM.%2BC.%26aulast%3DWilce%26aufirst%3DJ.%2BA.%26atitle%3DInteraction%2520of%2520the%2520non-phosphorylated%2520peptide%2520G7%25E2%2580%259318NATE%2520with%2520Grb7-SH2%2520domain%2520requires%2520phosphate%2520for%2520enhanced%2520affinity%2520and%2520specificity%26jtitle%3DJ.%2520Mol.%2520Recognit.%26date%3D2012%26volume%3D25%26spage%3D57%26epage%3D67%26doi%3D10.1002%2Fjmr.2148" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Eck, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoelson, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harrison, S. C.</span><span> </span><span class="NLM_article-title">Recognition of a high-affinity phosphotyrosyl peptide by the Src homology-2 domain of p56lck</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1993</span><span class="NLM_x">, </span> <span class="NLM_volume">362</span><span class="NLM_x">, </span> <span class="NLM_fpage">87</span><span class="NLM_x">–</span> <span class="NLM_lpage">91</span><span class="refDoi"> DOI: 10.1038/362087a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=10.1038%2F362087a0" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=362&publication_year=1993&pages=87-91&author=M.+J.+Eckauthor=S.+E.+Shoelsonauthor=S.+C.+Harrison&title=Recognition+of+a+high-affinity+phosphotyrosyl+peptide+by+the+Src+homology-2+domain+of+p56lck&doi=10.1038%2F362087a0"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2F362087a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F362087a0%26sid%3Dliteratum%253Aachs%26aulast%3DEck%26aufirst%3DM.%2BJ.%26aulast%3DShoelson%26aufirst%3DS.%2BE.%26aulast%3DHarrison%26aufirst%3DS.%2BC.%26atitle%3DRecognition%2520of%2520a%2520high-affinity%2520phosphotyrosyl%2520peptide%2520by%2520the%2520Src%2520homology-2%2520domain%2520of%2520p56lck%26jtitle%3DNature%26date%3D1993%26volume%3D362%26spage%3D87%26epage%3D91%26doi%3D10.1038%2F362087a0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Bevington, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mundy, K. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yates, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kanis, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Russell, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rajagopalan, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Radda, G. K.</span><span> </span><span class="NLM_article-title">A study of intracellular orthophosphate concentration in human muscle and erythrocytes by 31P nuclear magnetic resonance spectroscopy and selective chemical assay</span> <span class="citation_source-journal">Clin. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1986</span><span class="NLM_x">, </span> <span class="NLM_volume">71</span><span class="NLM_x">, </span> <span class="NLM_fpage">729</span><span class="NLM_x">–</span> <span class="NLM_lpage">735</span><span class="refDoi"> DOI: 10.1042/cs0710729</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=10.1042%2Fcs0710729" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=3024899" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=1%3ACAS%3A528%3ADyaL2sXkvVWnsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=1986&pages=729-735&author=A.+Bevingtonauthor=K.+I.+Mundyauthor=A.+J.+Yatesauthor=J.+A.+Kanisauthor=R.+G.+Russellauthor=D.+J.+Taylorauthor=B.+Rajagopalanauthor=G.+K.+Radda&title=A+study+of+intracellular+orthophosphate+concentration+in+human+muscle+and+erythrocytes+by+31P+nuclear+magnetic+resonance+spectroscopy+and+selective+chemical+assay&doi=10.1042%2Fcs0710729"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">A study of intracellular orthophosphate concentration in human muscle and erythrocytes by phosphorus-31 nuclear magnetic resonance spectroscopy and selective chemical assay</span></div><div class="casAuthors">Bevington, A.; Mundy, K. I.; Yates, A. J. P.; Kanis, J. A.; Russell, R. G. G.; Taylor, D. J.; Rajagopalan, B.; Radda, G. K.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Science</span>
        (<span class="NLM_cas:date">1986</span>),
    <span class="NLM_cas:volume">71</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">729-35</span>CODEN:
                <span class="NLM_cas:coden">CSCIAE</span>;
        ISSN:<span class="NLM_cas:issn">0143-5221</span>.
    </div><div class="casAbstract">To study the relation between extracellular and intracellular concns. of orthophosphate (Pi), phosphorus NMR spectra were recorded, at rest, from the flexor digitorum superficialis muscle of hypophosphatemic patients with vitamin D-resistant rickets, and patients with Paget's disease of bone before and after they had been made hyperphosphatemic by treatment with ethylidene-1-hydroxy-1,1-bisphosphonate.  Changes in i.m. Pi were estd. from the ratio of the areas of the Pi to ATP peaks.  Even though the plasma Pi concn. in these patients spanned a 4-fold range (0.5-2.0 mmol/L) the corresponding i.m. Pi concn. increased by only 70%.  A similar effect was obsd. in erythrocytes from patients with these disorders which were incubated in autologous plasma at 37°, under an atm. of O2 + CO2.  However, there was no general effect on passive anion distribution.  When erythrocytes from normal subjects were incubated in autologous plasma and in plasma supplemented with Pi the Pi concn. in the cells, at steady state, increased only from 0.57 to 0.78 mmol/L cells, suggesting that effect was not an artifact of disease or drug therapy.  Thus, in human skeletal myocytes and erythrocytes, the percentage change in the concn. of cytoplasmic Pi is lower than that in plasma.  This implies that these cells can buffer or regulate cytoplasmic Pi when the extracellular concn. is disturbed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVQuVo95gnZbVg90H21EOLACvtfcHk0ljSc5CdL8yGaQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL2sXkvVWnsA%253D%253D&md5=55a9afb0860b8079327f9bc6814ab888</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1042%2Fcs0710729&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252Fcs0710729%26sid%3Dliteratum%253Aachs%26aulast%3DBevington%26aufirst%3DA.%26aulast%3DMundy%26aufirst%3DK.%2BI.%26aulast%3DYates%26aufirst%3DA.%2BJ.%26aulast%3DKanis%26aufirst%3DJ.%2BA.%26aulast%3DRussell%26aufirst%3DR.%2BG.%26aulast%3DTaylor%26aufirst%3DD.%2BJ.%26aulast%3DRajagopalan%26aufirst%3DB.%26aulast%3DRadda%26aufirst%3DG.%2BK.%26atitle%3DA%2520study%2520of%2520intracellular%2520orthophosphate%2520concentration%2520in%2520human%2520muscle%2520and%2520erythrocytes%2520by%252031P%2520nuclear%2520magnetic%2520resonance%2520spectroscopy%2520and%2520selective%2520chemical%2520assay%26jtitle%3DClin.%2520Sci.%26date%3D1986%26volume%3D71%26spage%3D729%26epage%3D735%26doi%3D10.1042%2Fcs0710729" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Ambaye, N. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pero, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gunzburg, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yap, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clayton, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Del Borgo, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perlmutter, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aguilar, M. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shukla, G. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peletskaya, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cookson, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krag, D. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilce, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilce, J. A.</span><span> </span><span class="NLM_article-title">Structural basis of binding by cyclic nonphosphorylated peptide antagonists of Grb7 implicated in breast cancer progression</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">412</span><span class="NLM_x">, </span> <span class="NLM_fpage">397</span><span class="NLM_x">–</span> <span class="NLM_lpage">411</span><span class="refDoi"> DOI: 10.1016/j.jmb.2011.07.030</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=10.1016%2Fj.jmb.2011.07.030" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=21802427" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFarurbF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=412&publication_year=2011&pages=397-411&author=N.+D.+Ambayeauthor=S.+C.+Peroauthor=M.+J.+Gunzburgauthor=M.+Yapauthor=D.+J.+Claytonauthor=M.+P.+Del+Borgoauthor=P.+Perlmutterauthor=M.+I.+Aguilarauthor=G.+S.+Shuklaauthor=E.+Peletskayaauthor=M.+M.+Cooksonauthor=D.+N.+Kragauthor=M.+C.+Wilceauthor=J.+A.+Wilce&title=Structural+basis+of+binding+by+cyclic+nonphosphorylated+peptide+antagonists+of+Grb7+implicated+in+breast+cancer+progression&doi=10.1016%2Fj.jmb.2011.07.030"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Basis of Binding by Cyclic Nonphosphorylated Peptide Antagonists of Grb7 Implicated in Breast Cancer Progression</span></div><div class="casAuthors">Ambaye, Nigus D.; Pero, Stephanie C.; Gunzburg, Menachem J.; Yap, Min Yin; Clayton, Daniel J.; Del Borgo, Mark P.; Perlmutter, Patrick; Aguilar, Marie-Isabel; Shukla, Girja S.; Peletskaya, Elena; Cookson, Michelle M.; Krag, David N.; Wilce, Matthew C. J.; Wilce, Jacqueline A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">412</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">397-411</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Growth-receptor-bound protein (Grb)7 is an adapter protein aberrantly overexpressed, along with the erbB-2 receptor in breast cancer and in other cancers.  Normally recruited to focal adhesions with a role in cell migration, it is assocd. with erbB-2 in cancer cells and is found to exacerbate cancer progression via stimulation of cell migration and proliferation.  The G7-18NATE peptide (sequence: WFEGYDNTFPC cyclized via a thioether bond) is a nonphosphorylated peptide that was developed for the specific inhibition of Grb7 by blocking its SH2 domain.  Cell-permeable versions of G7-18NATE are effective in the redn. of migration and proliferation in Grb7-overexpressing cells.  It thus represents a promising starting point for the development of a therapeutic against Grb7.  Here, the authors report the crystal structure of the G7-18NATE peptide in complex with the Grb7-SH2 domain, revealing the structural basis for its interaction.  The authors also report further rounds of phage display that have identified G7-18NATE analogs with micromolar affinity for Grb7-SH2.  These peptides retained amino acids F2, G4, and F9, as well as the YDN motif that the structural biol. study showed to be the main residues in contact with the Grb7-SH2 domain.  Isothermal titrn. calorimetry measurements reveal similar and better binding affinity of these peptides compared with G7-18NATE.  Together, this study facilitates the optimization of second-generation inhibitors of Grb7.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEQ_mNcka417Vg90H21EOLACvtfcHk0ljSc5CdL8yGaQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFarurbF&md5=d00128a2cfbc35681abb5fd4f569840f</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2011.07.030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2011.07.030%26sid%3Dliteratum%253Aachs%26aulast%3DAmbaye%26aufirst%3DN.%2BD.%26aulast%3DPero%26aufirst%3DS.%2BC.%26aulast%3DGunzburg%26aufirst%3DM.%2BJ.%26aulast%3DYap%26aufirst%3DM.%26aulast%3DClayton%26aufirst%3DD.%2BJ.%26aulast%3DDel%2BBorgo%26aufirst%3DM.%2BP.%26aulast%3DPerlmutter%26aufirst%3DP.%26aulast%3DAguilar%26aufirst%3DM.%2BI.%26aulast%3DShukla%26aufirst%3DG.%2BS.%26aulast%3DPeletskaya%26aufirst%3DE.%26aulast%3DCookson%26aufirst%3DM.%2BM.%26aulast%3DKrag%26aufirst%3DD.%2BN.%26aulast%3DWilce%26aufirst%3DM.%2BC.%26aulast%3DWilce%26aufirst%3DJ.%2BA.%26atitle%3DStructural%2520basis%2520of%2520binding%2520by%2520cyclic%2520nonphosphorylated%2520peptide%2520antagonists%2520of%2520Grb7%2520implicated%2520in%2520breast%2520cancer%2520progression%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2011%26volume%3D412%26spage%3D397%26epage%3D411%26doi%3D10.1016%2Fj.jmb.2011.07.030" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Pero, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Daly, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krag, D. N.</span><span> </span><span class="NLM_article-title">Grb7-based molecular therapeutics in cancer</span> <span class="citation_source-journal">Expert Rev. Mol. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">11</span><span class="refDoi"> DOI: 10.1017/S1462399403006227</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=10.1017%2FS1462399403006227" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=14585167" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=1%3ACAS%3A280%3ADC%252BD3srgvFaltw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2003&pages=1-11&author=S.+C.+Peroauthor=R.+J.+Dalyauthor=D.+N.+Krag&title=Grb7-based+molecular+therapeutics+in+cancer&doi=10.1017%2FS1462399403006227"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Grb7-based molecular therapeutics in cancer</span></div><div class="casAuthors">Pero Stephanie C; Daly Roger J; Krag David N</div><div class="citationInfo"><span class="NLM_cas:title">Expert reviews in molecular medicine</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">1-11</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Traditional anti-cancer drugs preferentially kill rapidly growing tumour cells rather than normal cells.  However, most of these drugs have no preferential selection towards cancer cells and are taken up by the whole body, resulting in significant adverse side effects.  Therapeutic molecules that could specifically inhibit undesirable phenotypes are an attractive way of eliminating cancer cells.  There is a widespread effort to develop inhibitors against signal transduction molecules that play a key role in the proliferative, migratory and invasive properties of a cancer cell.  Grb7 is an adaptor-type signalling protein that is recruited via its Src-homology 2 (SH2) domain to a variety of tyrosine kinases.  Grb7 is overexpressed in breast, oesophageal and gastric cancers, and may contribute to the invasive potential of cancer cells.  Molecular interactions involving Grb7 therefore provide attractive targets for therapeutic intervention.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT7Lgiv6JFRiJ6KlWarZTAzfW6udTcc2earFPFUQ39AHLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD3srgvFaltw%253D%253D&md5=584ec9b9538c8b523524d2ad26c52fc8</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1017%2FS1462399403006227&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1017%252FS1462399403006227%26sid%3Dliteratum%253Aachs%26aulast%3DPero%26aufirst%3DS.%2BC.%26aulast%3DDaly%26aufirst%3DR.%2BJ.%26aulast%3DKrag%26aufirst%3DD.%2BN.%26atitle%3DGrb7-based%2520molecular%2520therapeutics%2520in%2520cancer%26jtitle%3DExpert%2520Rev.%2520Mol.%2520Med.%26date%3D2003%26volume%3D5%26spage%3D1%26epage%3D11%26doi%3D10.1017%2FS1462399403006227" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Songyang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoelson, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McGlade, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olivier, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pawson, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bustelo, X. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barbacid, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sabe, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hanafusa, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yi, T.</span><span> </span><span class="NLM_article-title">Specific motifs recognized by the SH2 domains of Csk, 3BP2, fps/fes, GRB-2, HCP, SHC, Syk, and Vav</span> <span class="citation_source-journal">Mol. Cell. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">2777</span><span class="NLM_x">–</span> <span class="NLM_lpage">2785</span><span class="refDoi"> DOI: 10.1128/MCB.14.4.2777</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=10.1128%2FMCB.14.4.2777" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=7511210" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=1%3ACAS%3A528%3ADyaK2cXktVenur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=1994&pages=2777-2785&author=Z.+Songyangauthor=S.+E.+Shoelsonauthor=J.+McGladeauthor=P.+Olivierauthor=T.+Pawsonauthor=X.+R.+Busteloauthor=M.+Barbacidauthor=H.+Sabeauthor=H.+Hanafusaauthor=T.+Yi&title=Specific+motifs+recognized+by+the+SH2+domains+of+Csk%2C+3BP2%2C+fps%2Ffes%2C+GRB-2%2C+HCP%2C+SHC%2C+Syk%2C+and+Vav&doi=10.1128%2FMCB.14.4.2777"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Specific motifs recognized by the SH2 domains of Csk, 3BP2, fps/fes, GRB-2, HCP, SHC, Syk, and Vav</span></div><div class="casAuthors">Songyang, Z.; Shoelson, S. E.; McGlade, J.; Olivier, P.; Pawson, T.; Bustelo, X. R.; Barbacid, M.; Sabe, H.; Hanafusa, H.; et al.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">2777-85</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    </div><div class="casAbstract">Src homol. 2 (SH2) domains provide specificity to intracellular signaling by binding to specific phosphotyrosine (phospho-Tyr)-contg. sequences.  The authors recently developed a technique using a degenerate phosphopeptide library to predict the specificity of individual SH2 domains src family members, Abl, Nck, Sem5, phospholipase C-γ, p85 subunit of phosphatidylinositol-3-kinase, and SHPTP2 (Z. Songyang, S. E. Shoelson, M. Chaudhuri, G. Gish, T. Pawson, W. G. Haser, F. King, T. Roberts, S. Ratnofsky, R. J. Lechleider, B. G. Neel, R. B. Birge, J. E. Fajardo, M. M. Chou, H. Hanafusa, B. Schaffhausen, and L. C. Cantley, Cell, 72:767-778, 1993).  The authors report here the optimal recognition motifs for SH2 domains from GRB-2, Drk, Csk, Vav, fps/fes, SHC, Syk (carboxy-terminal SH2), 3BP2, and HCP (amino-terminal SH2 domain, also called PTP1C and SHPTP1).  As predicted, SH2 domains from proteins that fall into group I on the basis of a Phe or Tyr at the βD5 position (GRB-2, 3BP2, Csk, fps/fes, Syk C-terminal SH2) select phosphopeptides with the general motif phospho-Tyr-hydrophilic (residue)-hydrophilic (residue)-hydrophobic (residue).  The SH2 domains of SHC and HCP (group III proteins with Ile, Leu, or Cys at the βD5 position) selected the general motif phospho-Tyr-hydrophobic-Xxx-hydrophobic, also as predicted.  Vav, which has a Thr at the βD5 position, selected phospho-Tyr-Met-Glu-Pro as the optimal motif.  Each SH2 domain selected a unique optimal motif distinct from motifs previously detd. for other SH2 domains.  These motifs are used to predict potential sites in signaling proteins for interaction with specific SH2 domain-contg. proteins.  The Syk SH2 domain is predicted to bind to Tyr-hydrophilic-hydrophilic-Leu/Ile motifs like those repeated at 10-residue intervals in T- and B-cell receptor-assocd. proteins.  SHC is predicted to bind to a subgroup of these same motifs.  A structural basis for the assocn. of Csk with Src family members is also suggested from these studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrDwa25r4DEDbVg90H21EOLACvtfcHk0lj6bwYENz8cSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXktVenur8%253D&md5=51bfdaf8c43a4b01c39011f79930e85c</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1128%2FMCB.14.4.2777&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.14.4.2777%26sid%3Dliteratum%253Aachs%26aulast%3DSongyang%26aufirst%3DZ.%26aulast%3DShoelson%26aufirst%3DS.%2BE.%26aulast%3DMcGlade%26aufirst%3DJ.%26aulast%3DOlivier%26aufirst%3DP.%26aulast%3DPawson%26aufirst%3DT.%26aulast%3DBustelo%26aufirst%3DX.%2BR.%26aulast%3DBarbacid%26aufirst%3DM.%26aulast%3DSabe%26aufirst%3DH.%26aulast%3DHanafusa%26aufirst%3DH.%26aulast%3DYi%26aufirst%3DT.%26atitle%3DSpecific%2520motifs%2520recognized%2520by%2520the%2520SH2%2520domains%2520of%2520Csk%252C%25203BP2%252C%2520fps%252Ffes%252C%2520GRB-2%252C%2520HCP%252C%2520SHC%252C%2520Syk%252C%2520and%2520Vav%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D1994%26volume%3D14%26spage%3D2777%26epage%3D2785%26doi%3D10.1128%2FMCB.14.4.2777" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Janes, P. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lackmann, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Church, W. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sanderson, G. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sutherland, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Daly, R. J.</span><span> </span><span class="NLM_article-title">Structural determinants of the interaction between the erbB2 receptor and the Src homology 2 domain of Grb7</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">272</span><span class="NLM_x">, </span> <span class="NLM_fpage">8490</span><span class="NLM_x">–</span> <span class="NLM_lpage">8497</span><span class="refDoi"> DOI: 10.1074/jbc.272.13.8490</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=10.1074%2Fjbc.272.13.8490" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=272&publication_year=1997&pages=8490-8497&author=P.+W.+Janesauthor=M.+Lackmannauthor=W.+B.+Churchauthor=G.+M.+Sandersonauthor=R.+L.+Sutherlandauthor=R.+J.+Daly&title=Structural+determinants+of+the+interaction+between+the+erbB2+receptor+and+the+Src+homology+2+domain+of+Grb7&doi=10.1074%2Fjbc.272.13.8490"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1074%2Fjbc.272.13.8490&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.272.13.8490%26sid%3Dliteratum%253Aachs%26aulast%3DJanes%26aufirst%3DP.%2BW.%26aulast%3DLackmann%26aufirst%3DM.%26aulast%3DChurch%26aufirst%3DW.%2BB.%26aulast%3DSanderson%26aufirst%3DG.%2BM.%26aulast%3DSutherland%26aufirst%3DR.%2BL.%26aulast%3DDaly%26aufirst%3DR.%2BJ.%26atitle%3DStructural%2520determinants%2520of%2520the%2520interaction%2520between%2520the%2520erbB2%2520receptor%2520and%2520the%2520Src%2520homology%25202%2520domain%2520of%2520Grb7%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1997%26volume%3D272%26spage%3D8490%26epage%3D8497%26doi%3D10.1074%2Fjbc.272.13.8490" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Scott, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bayly, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abell, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Skidmore, J.</span><span> </span><span class="NLM_article-title">Small molecules, big targets: drug discovery faces the protein-protein interaction challenge</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">533</span><span class="NLM_x">–</span> <span class="NLM_lpage">550</span><span class="refDoi"> DOI: 10.1038/nrd.2016.29</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=10.1038%2Fnrd.2016.29" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=27050677" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=1%3ACAS%3A528%3ADC%252BC28XlsFOgur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=533-550&author=D.+E.+Scottauthor=A.+R.+Baylyauthor=C.+Abellauthor=J.+Skidmore&title=Small+molecules%2C+big+targets%3A+drug+discovery+faces+the+protein-protein+interaction+challenge&doi=10.1038%2Fnrd.2016.29"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Small molecules, big targets: drug discovery faces the protein-protein interaction challenge</span></div><div class="casAuthors">Scott, Duncan E.; Bayly, Andrew R.; Abell, Chris; Skidmore, John</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">533-550</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Protein-protein interactions (PPIs) are of pivotal importance in the regulation of biol. systems and are consequently implicated in the development of disease states.  Recent work has begun to show that, with the right tools, certain classes of PPI can yield to the efforts of medicinal chemists to develop inhibitors, and the first PPI inhibitors have reached clin. development.  In this Review, we describe the research leading to these breakthroughs and highlight the existence of groups of structurally related PPIs within the PPI target class.  For each of these groups, we use examples of successful discovery efforts to illustrate the research strategies that have proved most useful.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoK-aGzHa9mP7Vg90H21EOLACvtfcHk0lj6bwYENz8cSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XlsFOgur4%253D&md5=9f0953567ae9338b8b36aa78b8160655</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.29&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.29%26sid%3Dliteratum%253Aachs%26aulast%3DScott%26aufirst%3DD.%2BE.%26aulast%3DBayly%26aufirst%3DA.%2BR.%26aulast%3DAbell%26aufirst%3DC.%26aulast%3DSkidmore%26aufirst%3DJ.%26atitle%3DSmall%2520molecules%252C%2520big%2520targets%253A%2520drug%2520discovery%2520faces%2520the%2520protein-protein%2520interaction%2520challenge%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2016%26volume%3D15%26spage%3D533%26epage%3D550%26doi%3D10.1038%2Fnrd.2016.29" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Kraskouskaya, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duodu, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arpin, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gunning, P. T.</span><span> </span><span class="NLM_article-title">Progress towards the development of SH2 domain inhibitors</span> <span class="citation_source-journal">Chem. Soc. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">42</span><span class="NLM_x">, </span> <span class="NLM_fpage">3337</span><span class="NLM_x">–</span> <span class="NLM_lpage">3370</span><span class="refDoi"> DOI: 10.1039/c3cs35449k</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=10.1039%2Fc3cs35449k" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2013&pages=3337-3370&author=D.+Kraskouskayaauthor=E.+Duoduauthor=C.+C.+Arpinauthor=P.+T.+Gunning&title=Progress+towards+the+development+of+SH2+domain+inhibitors&doi=10.1039%2Fc3cs35449k"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1039%2Fc3cs35449k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc3cs35449k%26sid%3Dliteratum%253Aachs%26aulast%3DKraskouskaya%26aufirst%3DD.%26aulast%3DDuodu%26aufirst%3DE.%26aulast%3DArpin%26aufirst%3DC.%2BC.%26aulast%3DGunning%26aufirst%3DP.%2BT.%26atitle%3DProgress%2520towards%2520the%2520development%2520of%2520SH2%2520domain%2520inhibitors%26jtitle%3DChem.%2520Soc.%2520Rev.%26date%3D2013%26volume%3D42%26spage%3D3337%26epage%3D3370%26doi%3D10.1039%2Fc3cs35449k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">Gunzburg, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ambaye, N. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Del Borgo, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perlmutter, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilce, J. A.</span><span> </span><span class="NLM_article-title">Design and testing of bicyclic inhibitors of Grb7—are two cycles better than one?</span> <span class="citation_source-journal">Biopolymers</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">100</span><span class="NLM_x">, </span> <span class="NLM_fpage">543</span><span class="NLM_x">–</span> <span class="NLM_lpage">549</span><span class="refDoi"> DOI: 10.1002/bip.22237</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=10.1002%2Fbip.22237" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=100&publication_year=2013&pages=543-549&author=M.+J.+Gunzburgauthor=N.+D.+Ambayeauthor=M.+P.+Del+Borgoauthor=P.+Perlmutterauthor=J.+A.+Wilce&title=Design+and+testing+of+bicyclic+inhibitors+of+Grb7%E2%80%94are+two+cycles+better+than+one%3F&doi=10.1002%2Fbip.22237"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1002%2Fbip.22237&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fbip.22237%26sid%3Dliteratum%253Aachs%26aulast%3DGunzburg%26aufirst%3DM.%2BJ.%26aulast%3DAmbaye%26aufirst%3DN.%2BD.%26aulast%3DDel%2BBorgo%26aufirst%3DM.%2BP.%26aulast%3DPerlmutter%26aufirst%3DP.%26aulast%3DWilce%26aufirst%3DJ.%2BA.%26atitle%3DDesign%2520and%2520testing%2520of%2520bicyclic%2520inhibitors%2520of%2520Grb7%25E2%2580%2594are%2520two%2520cycles%2520better%2520than%2520one%253F%26jtitle%3DBiopolymers%26date%3D2013%26volume%3D100%26spage%3D543%26epage%3D549%26doi%3D10.1002%2Fbip.22237" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Lau, Y. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Andrade, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spring, D. R.</span><span> </span><span class="NLM_article-title">Peptide stapling techniques based on different macrocyclisation chemistries</span> <span class="citation_source-journal">Chem. Soc. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">44</span><span class="NLM_x">, </span> <span class="NLM_fpage">91</span><span class="NLM_x">–</span> <span class="NLM_lpage">102</span><span class="refDoi"> DOI: 10.1039/C4CS00246F</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=10.1039%2FC4CS00246F" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=25199043" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsV2mtb%252FL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2015&pages=91-102&author=Y.+H.+Lauauthor=P.+de+Andradeauthor=Y.+Wuauthor=D.+R.+Spring&title=Peptide+stapling+techniques+based+on+different+macrocyclisation+chemistries&doi=10.1039%2FC4CS00246F"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Peptide stapling techniques based on different macrocyclization chemistries</span></div><div class="casAuthors">Lau, Yu Heng; de Andrade, Peterson; Wu, Yuteng; Spring, David R.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Society Reviews</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">91-102</span>CODEN:
                <span class="NLM_cas:coden">CSRVBR</span>;
        ISSN:<span class="NLM_cas:issn">0306-0012</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  Peptide stapling is a strategy for constraining short peptides typically in an alpha-helical conformation.  Stapling is carried out by covalently linking the side-chains of two amino acids, thereby forming a peptide macrocycle.  There is an expanding repertoire of stapling techniques based on different macrocyclization chemistries.  In this tutorial review, the authors categorize and analyze key examples of peptide stapling in terms of their synthesis and applicability to biol. systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkqQ6yGsikIbVg90H21EOLACvtfcHk0lh24udDoQ0Ilg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsV2mtb%252FL&md5=0efb31241f9809b61a81474131a54de3</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1039%2FC4CS00246F&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC4CS00246F%26sid%3Dliteratum%253Aachs%26aulast%3DLau%26aufirst%3DY.%2BH.%26aulast%3Dde%2BAndrade%26aufirst%3DP.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DSpring%26aufirst%3DD.%2BR.%26atitle%3DPeptide%2520stapling%2520techniques%2520based%2520on%2520different%2520macrocyclisation%2520chemistries%26jtitle%3DChem.%2520Soc.%2520Rev.%26date%3D2015%26volume%3D44%26spage%3D91%26epage%3D102%26doi%3D10.1039%2FC4CS00246F" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group">Nam, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ye, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parang, K.</span><span> </span><span class="NLM_article-title">Conformationally constrained peptide analogues of pTyr-Glu-Glu-Ile as inhibitors of the Src SH2 domain binding</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">3131</span><span class="NLM_x">–</span> <span class="NLM_lpage">3141</span><span class="refDoi"> DOI: 10.1021/jm040008+</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm040008%2B" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=1%3ACAS%3A528%3ADC%252BD2cXjslOhs7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=3131-3141&author=N.+Namauthor=G.+Yeauthor=G.+Sunauthor=K.+Parang&title=Conformationally+constrained+peptide+analogues+of+pTyr-Glu-Glu-Ile+as+inhibitors+of+the+Src+SH2+domain+binding&doi=10.1021%2Fjm040008%2B"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Conformationally Constrained Peptide Analogues of pTyr-Glu-Glu-Ile as Inhibitors of the Src SH2 Domain Binding</span></div><div class="casAuthors">Nam, Nguyen-Hai; Ye, Guofeng; Sun, Gongqin; Parang, Keykavous</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3131-3141</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A series of conformationally-constrained peptides were designed and synthesized as the Src SH2 domain ligands based on a tetrapeptide sequence pTyr-Glu-Glu-Ile (pYEEI).  In general, the constrained peptides showed higher binding affinities to the Src SH2 domain relative to the corresponding linear peptides as evaluated by a fluorescence polarization assay.  Mol. modeling studies revealed that in constrained peptides, the isoleucine side chain penetrates very deeply into the hydrophobic binding pocket (P + 3 site) of the Src SH2 domain.  These constrained peptides can serve as novel templates for the design of small and nonpeptidic inhibitors of the Src SH2 domain.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpsUsOw9C8CmLVg90H21EOLACvtfcHk0lh24udDoQ0Ilg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXjslOhs7s%253D&md5=af4660ffefc20ac0ef23804370b2bccc</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fjm040008%2B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm040008%252B%26sid%3Dliteratum%253Aachs%26aulast%3DNam%26aufirst%3DN.%26aulast%3DYe%26aufirst%3DG.%26aulast%3DSun%26aufirst%3DG.%26aulast%3DParang%26aufirst%3DK.%26atitle%3DConformationally%2520constrained%2520peptide%2520analogues%2520of%2520pTyr-Glu-Glu-Ile%2520as%2520inhibitors%2520of%2520the%2520Src%2520SH2%2520domain%2520binding%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D3131%26epage%3D3141%26doi%3D10.1021%2Fjm040008%2B" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group">Quartararo, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kritzer, J. A.</span><span> </span><span class="NLM_article-title">Peptide bicycles that inhibit the Grb2 SH2 domain</span> <span class="citation_source-journal">ChemBioChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">1490</span><span class="NLM_x">–</span> <span class="NLM_lpage">1496</span><span class="refDoi"> DOI: 10.1002/cbic.201200175</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=10.1002%2Fcbic.201200175" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=22689355" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=1%3ACAS%3A528%3ADC%252BC38Xot1KgsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2012&pages=1490-1496&author=J.+S.+Quartararoauthor=P.+Wuauthor=J.+A.+Kritzer&title=Peptide+bicycles+that+inhibit+the+Grb2+SH2+domain&doi=10.1002%2Fcbic.201200175"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Peptide Bicycles that Inhibit the Grb2 SH2 Domain</span></div><div class="casAuthors">Quartararo, Justin S.; Wu, Pianpian; Kritzer, Joshua A.</div><div class="citationInfo"><span class="NLM_cas:title">ChemBioChem</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1490-1496</span>CODEN:
                <span class="NLM_cas:coden">CBCHFX</span>;
        ISSN:<span class="NLM_cas:issn">1439-4227</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Developing short peptides into useful probes and therapeutic leads remains a difficult challenge.  Structural rigidification is a proven method for improving the properties of short peptides.  In this work, we report a strategy for stabilizing peptide macrocycles by introducing side-chain-to-side-chain staples to produce peptide bicycles with higher affinity, selectivity, and resistance to degrdn.  We have applied this strategy to G1, an 11-residue peptide macrocycle that binds the Src homol. 2 (SH2) domain of growth-factor-bound protein 2 (Grb2).  Several homodetic peptide bicycles were synthesized entirely on-resin with high yields.  Two rounds of iterative design produced peptide bicycle BC1, which is 60 times more potent than G1 and 200 times more selective.  Moreover, BC1 is completely intact after 24 h in buffered human serum, conditions under which G1 is completely degraded.  Our peptide-bicycle approach holds promise for the development of selective inhibitors of SH2 domains and other phosphotyrosine (pTyr)-binding proteins, as well as inhibitors of many other protein-protein interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGogV33oAc3rpbVg90H21EOLACvtfcHk0lhtpykm66wEjw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xot1KgsLY%253D&md5=0a23f863636c340cd4dcff0c58321661</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1002%2Fcbic.201200175&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbic.201200175%26sid%3Dliteratum%253Aachs%26aulast%3DQuartararo%26aufirst%3DJ.%2BS.%26aulast%3DWu%26aufirst%3DP.%26aulast%3DKritzer%26aufirst%3DJ.%2BA.%26atitle%3DPeptide%2520bicycles%2520that%2520inhibit%2520the%2520Grb2%2520SH2%2520domain%26jtitle%3DChemBioChem%26date%3D2012%26volume%3D13%26spage%3D1490%26epage%3D1496%26doi%3D10.1002%2Fcbic.201200175" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group">Fischer, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waizenegger, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kohler, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brock, R.</span><span> </span><span class="NLM_article-title">A quantitative validation of fluorophore-labelled cell-permeable peptide conjugates: fluorophore and cargo dependence of import</span> <span class="citation_source-journal">Biochim. Biophys. Acta, Biomembr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">1564</span><span class="NLM_x">, </span> <span class="NLM_fpage">365</span><span class="NLM_x">–</span> <span class="NLM_lpage">374</span><span class="refDoi"> DOI: 10.1016/S0005-2736(02)00471-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=10.1016%2FS0005-2736%2802%2900471-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=12175919" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=1%3ACAS%3A528%3ADC%252BD38XmtlCku7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1564&publication_year=2002&pages=365-374&author=R.+Fischerauthor=T.+Waizeneggerauthor=K.+Kohlerauthor=R.+Brock&title=A+quantitative+validation+of+fluorophore-labelled+cell-permeable+peptide+conjugates%3A+fluorophore+and+cargo+dependence+of+import&doi=10.1016%2FS0005-2736%2802%2900471-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">A quantitative validation of fluorophore-labelled cell-permeable peptide conjugates: fluorophore and cargo dependence of import</span></div><div class="casAuthors">Fischer, Rainer; Waizenegger, Thomas; Kohler, Karsten; Brock, Roland</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Biomembranes</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">1564</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">365-374</span>CODEN:
                <span class="NLM_cas:coden">BBBMBS</span>;
        ISSN:<span class="NLM_cas:issn">0005-2736</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Cell-permeable peptides were evaluated for a quant. controlled import of small mols.  The dependence of the import efficiency on the fluorophore, on the position of the fluorophore as well as on the nature of the cargo were addressed.  Cellular uptake was quantitated by flow cytometry and fluorescence correlation microscopy (FCM).  Fluorophores with different spectral characteristics, covering the whole visible spectral range, were selected in order to enable the simultaneous detection of several cell-permeable peptide constructs.  The transcytosis sequences were based either on the sequence of the Antennapedia homeodomain protein (AntpHD)-derived penetratin peptide or the Kaposi fibroblast growth factor (FGF)-derived membrane translocating sequence (MTS)-peptide.  In general, the AntpHD-derived peptides had a three- to fourfold higher import efficiency than the MTS-derived peptides.  In spite of the very different physicochem. characteristics of the fluorophores, the import efficiencies for analogs labeled at different positions within the sequence of the import peptides showed a strong pos. correlation.  However, even for peptide cargos of very similar size, pronounced differences in import efficiency were obsd.  The use of cell-permeable peptide/cargo constructs for intracellular analyses of structure-function relationships therefore requires the detn. of the intracellular concns. for each construct individually.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpS1odJZP8c_7Vg90H21EOLACvtfcHk0lhtpykm66wEjw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XmtlCku7c%253D&md5=e20688aef28a746afc276cd97700a245</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2FS0005-2736%2802%2900471-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0005-2736%252802%252900471-6%26sid%3Dliteratum%253Aachs%26aulast%3DFischer%26aufirst%3DR.%26aulast%3DWaizenegger%26aufirst%3DT.%26aulast%3DKohler%26aufirst%3DK.%26aulast%3DBrock%26aufirst%3DR.%26atitle%3DA%2520quantitative%2520validation%2520of%2520fluorophore-labelled%2520cell-permeable%2520peptide%2520conjugates%253A%2520fluorophore%2520and%2520cargo%2520dependence%2520of%2520import%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Biomembr.%26date%3D2002%26volume%3D1564%26spage%3D365%26epage%3D374%26doi%3D10.1016%2FS0005-2736%2802%2900471-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group">Maiolo, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferrer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ottinger, E. A.</span><span> </span><span class="NLM_article-title">Effects of cargo molecules on the cellular uptake of arginine-rich cell-penetrating peptides</span> <span class="citation_source-journal">Biochim. Biophys. Acta, Biomembr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">1712</span><span class="NLM_x">, </span> <span class="NLM_fpage">161</span><span class="NLM_x">–</span> <span class="NLM_lpage">172</span><span class="refDoi"> DOI: 10.1016/j.bbamem.2005.04.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=10.1016%2Fj.bbamem.2005.04.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=15935328" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=1%3ACAS%3A528%3ADC%252BD2MXltFSku7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1712&publication_year=2005&pages=161-172&author=J.+R.+Maioloauthor=M.+Ferrerauthor=E.+A.+Ottinger&title=Effects+of+cargo+molecules+on+the+cellular+uptake+of+arginine-rich+cell-penetrating+peptides&doi=10.1016%2Fj.bbamem.2005.04.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Effects of cargo molecules on the cellular uptake of arginine-rich cell-penetrating peptides</span></div><div class="casAuthors">Maiolo, James R.; Ferrer, Marc; Ottinger, Elizabeth A.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Biomembranes</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">1712</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">161-172</span>CODEN:
                <span class="NLM_cas:coden">BBBMBS</span>;
        ISSN:<span class="NLM_cas:issn">0005-2736</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The identification of cell-penetrating peptides (CPPs) as vectors for the intracellular delivery of conjugated mols. such as peptides, proteins, and oligonucleotides has emerged as a significant tool to modulate biol. activities inside cells.  The mechanism of CPP uptake by the cells is still unclear, and appears to be both endocytotic and non-endocytotic, depending on the CPP and cell type.  Moreover, it is also unknown whether cargo sequences have an effect on the uptake and cellular distribution properties of CPP sequences.  Here, we combine results from quant. fluorescence microscopy and binding to lipid membrane models to det. the effect of cargo peptide mols. on the cellular uptake and distribution of the arginine-rich CPPs, R7, and R7W, in live cells.  Image anal. algorithms that quantify fluorescence were used to measure the relative amt. of peptide taken up by the cell, as well as the extent to which the uptake was endocytotic in nature.  The results presented here indicate that fusion of arginine-rich CPPs to peptide sequences reduces the efficiency of uptake, and dramatically changes the cellular distribution of the CPP from a diffuse pattern to one in which the peptides are mostly retained in endosomal compartments.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQi9cX9dekbbVg90H21EOLACvtfcHk0lhtpykm66wEjw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXltFSku7k%253D&md5=e2e2cb45c5100f509b8d450991d32147</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.bbamem.2005.04.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbamem.2005.04.010%26sid%3Dliteratum%253Aachs%26aulast%3DMaiolo%26aufirst%3DJ.%2BR.%26aulast%3DFerrer%26aufirst%3DM.%26aulast%3DOttinger%26aufirst%3DE.%2BA.%26atitle%3DEffects%2520of%2520cargo%2520molecules%2520on%2520the%2520cellular%2520uptake%2520of%2520arginine-rich%2520cell-penetrating%2520peptides%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Biomembr.%26date%3D2005%26volume%3D1712%26spage%3D161%26epage%3D172%26doi%3D10.1016%2Fj.bbamem.2005.04.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group">El-Andaloussi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jarver, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johansson, H. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Langel, U.</span><span> </span><span class="NLM_article-title">Cargo-dependent cytotoxicity and delivery efficacy of cell-penetrating peptides: a comparative study</span> <span class="citation_source-journal">Biochem. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">407</span><span class="NLM_x">, </span> <span class="NLM_fpage">285</span><span class="NLM_x">–</span> <span class="NLM_lpage">292</span><span class="refDoi"> DOI: 10.1042/BJ20070507</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=10.1042%2FBJ20070507" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=17627607" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtVyrtr3P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=407&publication_year=2007&pages=285-292&author=S.+El-Andaloussiauthor=P.+Jarverauthor=H.+J.+Johanssonauthor=U.+Langel&title=Cargo-dependent+cytotoxicity+and+delivery+efficacy+of+cell-penetrating+peptides%3A+a+comparative+study&doi=10.1042%2FBJ20070507"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Cargo-dependent cytotoxicity and delivery efficacy of cell-penetrating peptides: A comparative study</span></div><div class="casAuthors">El-Andaloussi, Samir; Jaerver, Peter; Johansson, Henrik J.; Langel, Uelo</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Journal</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">407</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">285-292</span>CODEN:
                <span class="NLM_cas:coden">BIJOAK</span>;
        ISSN:<span class="NLM_cas:issn">0264-6021</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">The use of CPPs (cell-penetrating peptides) as delivery vectors for bioactive mols. has been an emerging field since 1994 when the first CPP, penetratin, was discovered.  Since then, several CPPs, including the widely used Tat (transactivator of transcription) peptide, have been developed and utilized to translocate a wide range of compds. across the plasma membrane of cells both in vivo and in vitro.  Although the field has emerged as a possible future candidate for drug delivery, little attention has been given to the potential toxic side effects that these peptides might exhibit in cargo delivery.  Also, no comprehensive study has been performed to evaluate the relative efficacy of single CPPs to convey different cargos.  Therefore the authors selected three of the major CPPs, penetratin, Tat and transportan 10, and evaluated their ability to deliver commonly used cargos, including fluoresceinyl moiety, double-stranded DNA and proteins (i.e. avidin and streptavidin), and studied their effect on membrane integrity and cell viability.  The results demonstrate the unfeasibility to use the translocation efficacy of fluorescein moiety as a gauge for CPP efficiency, since the delivery properties are dependent on the cargo used.  Furthermore, and no less importantly, the toxicity of CPPs depends heavily on peptide concn., cargo mol. and coupling strategy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqMuztXTLEVq7Vg90H21EOLACvtfcHk0lgK_n2JGOuYCA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtVyrtr3P&md5=41e2f8e70acb7e80314938a7dbc56d37</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1042%2FBJ20070507&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBJ20070507%26sid%3Dliteratum%253Aachs%26aulast%3DEl-Andaloussi%26aufirst%3DS.%26aulast%3DJarver%26aufirst%3DP.%26aulast%3DJohansson%26aufirst%3DH.%2BJ.%26aulast%3DLangel%26aufirst%3DU.%26atitle%3DCargo-dependent%2520cytotoxicity%2520and%2520delivery%2520efficacy%2520of%2520cell-penetrating%2520peptides%253A%2520a%2520comparative%2520study%26jtitle%3DBiochem.%2520J.%26date%3D2007%26volume%3D407%26spage%3D285%26epage%3D292%26doi%3D10.1042%2FBJ20070507" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group">Ambaye, N. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lim, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clayton, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gunzburg, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Price, J. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pero, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krag, D. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilce, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aguilar, M. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perlmutter, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilce, J. A.</span><span> </span><span class="NLM_article-title">Uptake of a cell permeable G7–18NATE contruct into cells and binding with the Grb-7-SH2 domain</span> <span class="citation_source-journal">Biopolymers</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">96</span><span class="NLM_x">, </span> <span class="NLM_fpage">181</span><span class="NLM_x">–</span> <span class="NLM_lpage">188</span><span class="refDoi"> DOI: 10.1002/bip.21403</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=10.1002%2Fbip.21403" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=96&publication_year=2011&pages=181-188&author=N.+D.+Ambayeauthor=R.+C.+Limauthor=D.+J.+Claytonauthor=M.+J.+Gunzburgauthor=J.+T.+Priceauthor=S.+C.+Peroauthor=D.+N.+Kragauthor=M.+C.+Wilceauthor=M.+I.+Aguilarauthor=P.+Perlmutterauthor=J.+A.+Wilce&title=Uptake+of+a+cell+permeable+G7%E2%80%9318NATE+contruct+into+cells+and+binding+with+the+Grb-7-SH2+domain&doi=10.1002%2Fbip.21403"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1002%2Fbip.21403&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fbip.21403%26sid%3Dliteratum%253Aachs%26aulast%3DAmbaye%26aufirst%3DN.%2BD.%26aulast%3DLim%26aufirst%3DR.%2BC.%26aulast%3DClayton%26aufirst%3DD.%2BJ.%26aulast%3DGunzburg%26aufirst%3DM.%2BJ.%26aulast%3DPrice%26aufirst%3DJ.%2BT.%26aulast%3DPero%26aufirst%3DS.%2BC.%26aulast%3DKrag%26aufirst%3DD.%2BN.%26aulast%3DWilce%26aufirst%3DM.%2BC.%26aulast%3DAguilar%26aufirst%3DM.%2BI.%26aulast%3DPerlmutter%26aufirst%3DP.%26aulast%3DWilce%26aufirst%3DJ.%2BA.%26atitle%3DUptake%2520of%2520a%2520cell%2520permeable%2520G7%25E2%2580%259318NATE%2520contruct%2520into%2520cells%2520and%2520binding%2520with%2520the%2520Grb-7-SH2%2520domain%26jtitle%3DBiopolymers%26date%3D2011%26volume%3D96%26spage%3D181%26epage%3D188%26doi%3D10.1002%2Fbip.21403" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group">Pace, C. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vajdos, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fee, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grimsley, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, T.</span><span> </span><span class="NLM_article-title">How to measure and predict the molar absorption coefficient of a protein</span> <span class="citation_source-journal">Protein Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">2411</span><span class="NLM_x">–</span> <span class="NLM_lpage">2423</span><span class="refDoi"> DOI: 10.1002/pro.5560041120</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=10.1002%2Fpro.5560041120" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=8563639" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=1%3ACAS%3A528%3ADyaK2MXpsVGqt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=1995&pages=2411-2423&author=C.+N.+Paceauthor=F.+Vajdosauthor=L.+Feeauthor=G.+Grimsleyauthor=T.+Gray&title=How+to+measure+and+predict+the+molar+absorption+coefficient+of+a+protein&doi=10.1002%2Fpro.5560041120"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">How to measure and predict the molar absorption coefficient of a protein</span></div><div class="casAuthors">Pace, C. Nick; Vajdos, Felix; Fee, Lanette; Grimsley, Gerald; Gray, Theronica</div><div class="citationInfo"><span class="NLM_cas:title">Protein Science</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2411-23</span>CODEN:
                <span class="NLM_cas:coden">PRCIEI</span>;
        ISSN:<span class="NLM_cas:issn">0961-8368</span>.
    
            (<span class="NLM_cas:orgname">Cambridge University Press</span>)
        </div><div class="casAbstract">The molar absorption coeff., ε, of a protein is usually based on concns. measured by dry wt., nitrogen, or amino acid anal.  The studies reported here suggest that the Edelhoch method is the best method for measuring ε for a protein.  (This method is described by Gill and von Hippel [1989, Anal Biochem 182:319-326] and is based on data from Edelhoch [1967, Biochem. 6:1948-1954].).  The av. ε values for these chromophores in a sample of 18 well-characterized proteins have been estd., and the ε values in water, propanol, 6 M guanidine hydrochloride (GdnHCl), and 8 M urea have been measured.  For Trp, the av. ε values for the proteins are intermediate between those measured in 6 M GdnHCl and those measured in propanol.  Based on a sample of 116 measured ε values for 80 proteins, the ε at 280 nm of a folded protein in water, ε(280), can best be predicted with this equation L ε(280) (M-1 cm-1) = (#Trp)(5,500) + (#Tyr)(1,490) + (#cystine)(125).  These ε(280) values are quite reliable for proteins contg. Trp residues, and less reliable for proteins that do not.  However, the Edelhoch method is convenient and accurate, and the best approach is to measure rather than predict ε.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDyrAj6uk11bVg90H21EOLACvtfcHk0lgK_n2JGOuYCA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXpsVGqt7k%253D&md5=033237a125c7486331bf5180047c3cd4</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1002%2Fpro.5560041120&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpro.5560041120%26sid%3Dliteratum%253Aachs%26aulast%3DPace%26aufirst%3DC.%2BN.%26aulast%3DVajdos%26aufirst%3DF.%26aulast%3DFee%26aufirst%3DL.%26aulast%3DGrimsley%26aufirst%3DG.%26aulast%3DGray%26aufirst%3DT.%26atitle%3DHow%2520to%2520measure%2520and%2520predict%2520the%2520molar%2520absorption%2520coefficient%2520of%2520a%2520protein%26jtitle%3DProtein%2520Sci.%26date%3D1995%26volume%3D4%26spage%3D2411%26epage%3D2423%26doi%3D10.1002%2Fpro.5560041120" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group">McPhillips, T. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McPhillips, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chiu, H. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cohen, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deacon, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ellis, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garman, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gonzalez, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sauter, N. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Phizackerley, R. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soltis, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuhn, P.</span><span> </span><span class="NLM_article-title">Blu-Ice and the Distributed Control System: software for data acquisition and instrument control at macromolecular crystallography beamlines</span> <span class="citation_source-journal">J. Synchrotron Radiat.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">401</span><span class="NLM_x">–</span> <span class="NLM_lpage">406</span><span class="refDoi"> DOI: 10.1107/S0909049502015170</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=10.1107%2FS0909049502015170" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=12409628" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=1%3ACAS%3A528%3ADC%252BD38Xotleluro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2002&pages=401-406&author=T.+M.+McPhillipsauthor=S.+E.+McPhillipsauthor=H.+J.+Chiuauthor=A.+E.+Cohenauthor=A.+M.+Deaconauthor=P.+J.+Ellisauthor=E.+Garmanauthor=A.+Gonzalezauthor=N.+K.+Sauterauthor=R.+P.+Phizackerleyauthor=S.+M.+Soltisauthor=P.+Kuhn&title=Blu-Ice+and+the+Distributed+Control+System%3A+software+for+data+acquisition+and+instrument+control+at+macromolecular+crystallography+beamlines&doi=10.1107%2FS0909049502015170"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Blu-Ice and the Distributed Control System: software for data acquisition and instrument control at macromolecular crystallography beamlines</span></div><div class="casAuthors">McPhillips, Timothy M.; McPhillips, Scott E.; Chiu, Hsiu Ju; Cohen, Aina E.; Deacon, Ashley M.; Ellis, Paul J.; Garman, Elspeth; Gonzalez, Ana; Sauter, Nicholas K.; Phizackerley, R. Paul; Soltis, S. Michael; Kuhn, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Synchrotron Radiation</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">401-406</span>CODEN:
                <span class="NLM_cas:coden">JSYRES</span>;
        ISSN:<span class="NLM_cas:issn">0909-0495</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Munksgaard</span>)
        </div><div class="casAbstract">The Blu-Ice and Distributed Control System (DCS) software packages were developed to provide unified control over the disparate hardware resources available at a macromol. crystallog. beamline.  Blu-Ice is a user interface that provides scientific experimenters and beamline support staff with intuitive graphical tools for collecting diffraction data and configuring beamlines for expts.  Blu-Ice communicates with the hardware at a beamline via DCS, an instrument-control and data-acquisition package designed to integrate hardware resources in a highly heterogeneous networked computing environment.  Together, Blu-Ice and DCS provide a flexible platform for increasing the ease of use, the level of automation and the remote accessibility of beamlines.  Blu-Ice and DCS are currently installed on four Stanford Synchrotron Radiation Lab. crystallog. beamlines and are being implemented at sister light sources.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHVsQUIFJPBLVg90H21EOLACvtfcHk0lgK_n2JGOuYCA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xotleluro%253D&md5=c77228bd50890463f3e43e981a2e238a</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1107%2FS0909049502015170&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0909049502015170%26sid%3Dliteratum%253Aachs%26aulast%3DMcPhillips%26aufirst%3DT.%2BM.%26aulast%3DMcPhillips%26aufirst%3DS.%2BE.%26aulast%3DChiu%26aufirst%3DH.%2BJ.%26aulast%3DCohen%26aufirst%3DA.%2BE.%26aulast%3DDeacon%26aufirst%3DA.%2BM.%26aulast%3DEllis%26aufirst%3DP.%2BJ.%26aulast%3DGarman%26aufirst%3DE.%26aulast%3DGonzalez%26aufirst%3DA.%26aulast%3DSauter%26aufirst%3DN.%2BK.%26aulast%3DPhizackerley%26aufirst%3DR.%2BP.%26aulast%3DSoltis%26aufirst%3DS.%2BM.%26aulast%3DKuhn%26aufirst%3DP.%26atitle%3DBlu-Ice%2520and%2520the%2520Distributed%2520Control%2520System%253A%2520software%2520for%2520data%2520acquisition%2520and%2520instrument%2520control%2520at%2520macromolecular%2520crystallography%2520beamlines%26jtitle%3DJ.%2520Synchrotron%2520Radiat.%26date%3D2002%26volume%3D9%26spage%3D401%26epage%3D406%26doi%3D10.1107%2FS0909049502015170" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group">Battye, T. G. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kontogiannis, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Powell, H. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leslie, A. G. W.</span><span> </span><span class="NLM_article-title">IMosflm: a new graphical interface for diffraction-image processing with MOSFLM</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">67</span><span class="NLM_x">, </span> <span class="NLM_fpage">271</span><span class="NLM_x">–</span> <span class="NLM_lpage">281</span><span class="refDoi"> DOI: 10.1107/S0907444910048675</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=10.1107%2FS0907444910048675" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=21460445" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqtLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=271-281&author=T.+G.+G.+Battyeauthor=L.+Kontogiannisauthor=O.+Johnsonauthor=H.+R.+Powellauthor=A.+G.+W.+Leslie&title=IMosflm%3A+a+new+graphical+interface+for+diffraction-image+processing+with+MOSFLM&doi=10.1107%2FS0907444910048675"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">iMOSFLM: A new graphical interface for diffraction-image processing with MOSFLM</span></div><div class="casAuthors">Battye, T. Geoff G.; Kontogiannis, Luke; Johnson, Owen; Powell, Harold R.; Leslie, Andrew G. W.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">271-281</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">IMOSFLM is a graphical user interface to the diffraction data-integration program MOSFLM.  It is designed to simplify data processing by dividing the process into a series of steps, which are normally carried out sequentially.  Each step has its own display pane, allowing control over parameters that influence that step and providing graphical feedback to the user.  Suitable values for integration parameters are set automatically, but addnl. menus provide a detailed level of control for experienced users.  The image display and the interfaces to the different tasks (indexing, strategy calcn., cell refinement, integration and history) are described.  The most important parameters for each step and the best way of assessing success or failure are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqeZl2UOMg3_bVg90H21EOLACvtfcHk0ljx0pT3MokjyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqtLw%253D&md5=852651253f8d9c9a81e0513b6633c705</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1107%2FS0907444910048675&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910048675%26sid%3Dliteratum%253Aachs%26aulast%3DBattye%26aufirst%3DT.%2BG.%2BG.%26aulast%3DKontogiannis%26aufirst%3DL.%26aulast%3DJohnson%26aufirst%3DO.%26aulast%3DPowell%26aufirst%3DH.%2BR.%26aulast%3DLeslie%26aufirst%3DA.%2BG.%2BW.%26atitle%3DIMosflm%253A%2520a%2520new%2520graphical%2520interface%2520for%2520diffraction-image%2520processing%2520with%2520MOSFLM%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D271%26epage%3D281%26doi%3D10.1107%2FS0907444910048675" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group">Evans, P. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murshudov, G. N.</span><span> </span><span class="NLM_article-title">How good are my data and what is the resolution?</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">69</span><span class="NLM_x">, </span> <span class="NLM_fpage">1204</span><span class="NLM_x">–</span> <span class="NLM_lpage">1214</span><span class="refDoi"> DOI: 10.1107/S0907444913000061</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=10.1107%2FS0907444913000061" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=23793146" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=1%3ACAS%3A528%3ADC%252BC3sXpvVejsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2013&pages=1204-1214&author=P.+R.+Evansauthor=G.+N.+Murshudov&title=How+good+are+my+data+and+what+is+the+resolution%3F&doi=10.1107%2FS0907444913000061"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">How good are my data and what is the resolution?</span></div><div class="casAuthors">Evans, Philip R.; Murshudov, Garib N.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1204-1214</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Following integration of the obsd. diffraction spots, the process of  data redn.' initially aims to det. the point-group symmetry of the data and the likely space group.  This can be performed with the program POINTLESS.  The scaling program then puts all the measurements on a common scale, avs. measurements of symmetry-related reflections (using the symmetry detd. previously) and produces many statistics that provide the first important measures of data quality.  A new scaling program, AIMLESS, implements scaling models similar to those in SCALA but adds some addnl. analyses.  From the analyses, a no. of decisions can be made about the quality of the data and whether some measurements should be discarded.  The effective  resoln.' of a data set is a difficult and possibly contentious question (particularly with referees of papers) and this is discussed in the light of tests comparing the data-processing statistics with trials of refinement against obsd. and simulated data, and automated model-building and comparison of maps calcd. with different resoln. limits.  These trials show that adding weak high-resoln. data beyond the commonly used limits may make some improvement and does no harm.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreOyPAR8GLBrVg90H21EOLACvtfcHk0ljx0pT3MokjyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXpvVejsL8%253D&md5=3f649426c21e9d423f94ef8d7c568233</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1107%2FS0907444913000061&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444913000061%26sid%3Dliteratum%253Aachs%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26atitle%3DHow%2520good%2520are%2520my%2520data%2520and%2520what%2520is%2520the%2520resolution%253F%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2013%26volume%3D69%26spage%3D1204%26epage%3D1214%26doi%3D10.1107%2FS0907444913000061" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group">Winn, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ballard, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowtan, K. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dodson, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Emsley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Evans, P. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keegan, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krissinel, E. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leslie, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCoy, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McNicholas, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murshudov, G. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pannu, N. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Potterton, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Powell, H. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Read, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vagin, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilson, K. S.</span><span> </span><span class="NLM_article-title">Overview of the CCP4 suite and current developments</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">67</span><span class="NLM_x">, </span> <span class="NLM_fpage">235</span><span class="NLM_x">–</span> <span class="NLM_lpage">242</span><span class="refDoi"> DOI: 10.1107/S0907444910045749</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=10.1107%2FS0907444910045749" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=21460441" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=235-242&author=M.+D.+Winnauthor=C.+C.+Ballardauthor=K.+D.+Cowtanauthor=E.+J.+Dodsonauthor=P.+Emsleyauthor=P.+R.+Evansauthor=R.+M.+Keeganauthor=E.+B.+Krissinelauthor=A.+G.+Leslieauthor=A.+McCoyauthor=S.+J.+McNicholasauthor=G.+N.+Murshudovauthor=N.+S.+Pannuauthor=E.+A.+Pottertonauthor=H.+R.+Powellauthor=R.+J.+Readauthor=A.+Vaginauthor=K.+S.+Wilson&title=Overview+of+the+CCP4+suite+and+current+developments&doi=10.1107%2FS0907444910045749"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Overview of the CCP4 suite and current developments</span></div><div class="casAuthors">Winn, Martyn D.; Ballard, Charles C.; Cowtan, Kevin D.; Dodson, Eleanor J.; Emsley, Paul; Evans, Phil R.; Keegan, Ronan M.; Krissinel, Eugene B.; Leslie, Andrew G. W.; McCoy, Airlie; McNicholas, Stuart J.; Murshudov, Garib N.; Pannu, Navraj S.; Potterton, Elizabeth A.; Powell, Harold R.; Read, Randy J.; Vagin, Alexei; Wilson, Keith S.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  The CCP4 (Collaborative Computational Project, No. 4) software suite is a collection of programs and assocd. data and software libraries which can be used for macromol. structure detn. by X-ray crystallog.  The suite is designed to be flexible, allowing users a no. of methods of achieving their aims.  The programs are from a wide variety of sources but are connected by a common infrastructure provided by std. file formats, data objects and graphical interfaces.  Structure soln. by macromol. crystallog. is becoming increasingly automated and the CCP4 suite includes several automation pipelines.  After giving a brief description of the evolution of CCP4 over the last 30 years, an overview of the current suite is given.  While detailed descriptions are given in the accompanying articles, here it is shown how the individual programs contribute to a complete software package.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_wWMMIol6UrVg90H21EOLACvtfcHk0ljx0pT3MokjyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D&md5=c407e4d47bef46864be336d60147c17d</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1107%2FS0907444910045749&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910045749%26sid%3Dliteratum%253Aachs%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DBallard%26aufirst%3DC.%2BC.%26aulast%3DCowtan%26aufirst%3DK.%2BD.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DKeegan%26aufirst%3DR.%2BM.%26aulast%3DKrissinel%26aufirst%3DE.%2BB.%26aulast%3DLeslie%26aufirst%3DA.%2BG.%26aulast%3DMcCoy%26aufirst%3DA.%26aulast%3DMcNicholas%26aufirst%3DS.%2BJ.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DPannu%26aufirst%3DN.%2BS.%26aulast%3DPotterton%26aufirst%3DE.%2BA.%26aulast%3DPowell%26aufirst%3DH.%2BR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DWilson%26aufirst%3DK.%2BS.%26atitle%3DOverview%2520of%2520the%2520CCP4%2520suite%2520and%2520current%2520developments%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D235%26epage%3D242%26doi%3D10.1107%2FS0907444910045749" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group">McCoy, A. J.</span><span> </span><span class="NLM_article-title">Solving structures of protein complexes by molecular replacement with Phaser</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">63</span><span class="NLM_x">, </span> <span class="NLM_fpage">32</span><span class="NLM_x">–</span> <span class="NLM_lpage">41</span><span class="refDoi"> DOI: 10.1107/S0907444906045975</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=10.1107%2FS0907444906045975" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=17164524" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtlSqu77F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2007&pages=32-41&author=A.+J.+McCoy&title=Solving+structures+of+protein+complexes+by+molecular+replacement+with+Phaser&doi=10.1107%2FS0907444906045975"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Solving structures of protein complexes by molecular replacement with Phaser</span></div><div class="casAuthors">McCoy, Airlie J.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">32-41</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Mol. replacement (MR) generally becomes more difficult as the no. of components in the asym. unit requiring sep. MR models (i.e. the dimensionality of the search) increases.  When the proportion of the total scattering contributed by each search component is small, the signal in the search for each component in isolation is weak or non-existent.  Maximum-likelihood MR functions enable complex asym. units to be built up from individual components with a 'tree search with pruning' approach.  This method, as implemented in the automated search procedure of the program Phaser, has been very successful in solving many previously intractable MR problems.  However, there are a no. of cases in which the automated search procedure of Phaser is suboptimal or encounters difficulties.  These include cases where there are a large no. of copies of the same component in the asym. unit or where the components of the asym. unit have greatly varying B factors.  Two case studies are presented to illustrate how Phaser can be used to best advantage in the std. 'automated MR' mode and two case studies are used to show how to modify the automated search strategy for problematic cases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpAvs9W8hQ3l7Vg90H21EOLACvtfcHk0liJ9sqaMeucig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtlSqu77F&md5=323a5297bdf622ebaf398a87c39dc794</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1107%2FS0907444906045975&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444906045975%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26atitle%3DSolving%2520structures%2520of%2520protein%2520complexes%2520by%2520molecular%2520replacement%2520with%2520Phaser%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2007%26volume%3D63%26spage%3D32%26epage%3D41%26doi%3D10.1107%2FS0907444906045975" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group">Afonine, P. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grosse-Kunstleve, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Echols, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Headd, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moriarty, N. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mustyakimov, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Terwilliger, T. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Urzhumtsev, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zwart, P. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adams, P. D.</span><span> </span><span class="NLM_article-title">Towards automated crystallographic structure refinement with phenix.refine</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">68</span><span class="NLM_x">, </span> <span class="NLM_fpage">352</span><span class="NLM_x">–</span> <span class="NLM_lpage">367</span><span class="refDoi"> DOI: 10.1107/S0907444912001308</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=10.1107%2FS0907444912001308" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=22505256" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=1%3ACAS%3A528%3ADC%252BC38Xlt1Ggu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2012&pages=352-367&author=P.+V.+Afonineauthor=R.+W.+Grosse-Kunstleveauthor=N.+Echolsauthor=J.+J.+Headdauthor=N.+W.+Moriartyauthor=M.+Mustyakimovauthor=T.+C.+Terwilligerauthor=A.+Urzhumtsevauthor=P.+H.+Zwartauthor=P.+D.+Adams&title=Towards+automated+crystallographic+structure+refinement+with+phenix.refine&doi=10.1107%2FS0907444912001308"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Towards automated crystallographic structure refinement with phenix.refine</span></div><div class="casAuthors">Afonine, Pavel V.; Grosse-Kunstleve, Ralf W.; Echols, Nathaniel; Headd, Jeffrey J.; Moriarty, Nigel W.; Mustyakimov, Marat; Terwilliger, Thomas C.; Urzhumtsev, Alexandre; Zwart, Peter H.; Adams, Paul D.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">352-367</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phenix.refine is a program within the PHENIX package that supports crystallog. structure refinement against exptl. data with a wide range of upper resoln. limits using a large repertoire of model parameterizations.  It has several automation features and is also highly flexible.  Several hundred parameters enable extensive customizations for complex use cases.  Multiple user-defined refinement strategies can be applied to specific parts of the model in a single refinement run.  An intuitive graphical user interface is available to guide novice users and to assist advanced users in managing refinement projects.  X-ray or neutron diffraction data can be used sep. or jointly in refinement. phenix.refine is tightly integrated into the PHENIX suite, where it serves as a crit. component in automated model building, final structure refinement, structure validation and deposition to the wwPDB.  This paper presents an overview of the major phenix.refine features, with extensive literature refs. for readers interested in more detailed discussions of the methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnhn9QwIEkTbVg90H21EOLACvtfcHk0liJ9sqaMeucig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xlt1Ggu7c%253D&md5=1cc7da2cea396f1a75646ea709140cf1</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1107%2FS0907444912001308&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444912001308%26sid%3Dliteratum%253Aachs%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DMustyakimov%26aufirst%3DM.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DUrzhumtsev%26aufirst%3DA.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26atitle%3DTowards%2520automated%2520crystallographic%2520structure%2520refinement%2520with%2520phenix.refine%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2012%26volume%3D68%26spage%3D352%26epage%3D367%26doi%3D10.1107%2FS0907444912001308" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group">Emsley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowtan, K.</span><span> </span><span class="NLM_article-title">Coot: model-building tools for molecular graphics</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">60</span><span class="NLM_x">, </span> <span class="NLM_fpage">2126</span><span class="NLM_x">–</span> <span class="NLM_lpage">2132</span><span class="refDoi"> DOI: 10.1107/S0907444904019158</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=10.1107%2FS0907444904019158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=15572765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2126-2132&author=P.+Emsleyauthor=K.+Cowtan&title=Coot%3A+model-building+tools+for+molecular+graphics&doi=10.1107%2FS0907444904019158"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Coot: model-building tools for molecular graphics</span></div><div class="casAuthors">Emsley, Paul; Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2126-2132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4mg is a project that aims to provide a general-purpose tool for structural biologists, providing tools for x-ray structure soln., structure comparison and anal., and publication-quality graphics.  The map-fitting tools are available as a stand-alone package, distributed as 'Coot'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMisM42468k7Vg90H21EOLACvtfcHk0liJ9sqaMeucig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P&md5=1be390f3bb6fd584468499ad0921161e</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019158%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DCoot%253A%2520model-building%2520tools%2520for%2520molecular%2520graphics%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D2126%26epage%3D2132%26doi%3D10.1107%2FS0907444904019158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group">Kabsch, W.</span><span> </span><span class="NLM_article-title">Integration, scaling, space-group assignment and post-refinement</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">133</span><span class="NLM_x">–</span> <span class="NLM_lpage">144</span><span class="refDoi"> DOI: 10.1107/S0907444909047374</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=10.1107%2FS0907444909047374" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=20124693" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=133-144&author=W.+Kabsch&title=Integration%2C+scaling%2C+space-group+assignment+and+post-refinement&doi=10.1107%2FS0907444909047374"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Integration, scaling, space-group assignment and post-refinement</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">133-144</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Important steps in the processing of rotation data are described that are common to most software packages.  These programs differ in the details and in the methods implemented to carry out the tasks.  Here, the working principles underlying the data-redn. package XDS are explained, including the new features of automatic detn. of spot size and reflecting range, recognition and assignment of crystal symmetry and a highly efficient algorithm for the detn. of correction/scaling factors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGW7t3Bb1P_rVg90H21EOLACvtfcHk0lgftWPgXh14vQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisb4%253D&md5=eb1db34f3c959b4921771eb9f57b2159</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047374&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047374%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DIntegration%252C%2520scaling%252C%2520space-group%2520assignment%2520and%2520post-refinement%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D133%26epage%3D144%26doi%3D10.1107%2FS0907444909047374" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group">Moriarty, N. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grosse-Kunstleve, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adams, P. D.</span><span> </span><span class="NLM_article-title">electronic Ligand Builder and Optimization Workbench (eLBOW): a tool for ligand coordinate and restraint generation</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">65</span><span class="NLM_x">, </span> <span class="NLM_fpage">1074</span><span class="NLM_x">–</span> <span class="NLM_lpage">1080</span><span class="refDoi"> DOI: 10.1107/S0907444909029436</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=10.1107%2FS0907444909029436" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=19770504" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtFKnsLnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2009&pages=1074-1080&author=N.+W.+Moriartyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adams&title=electronic+Ligand+Builder+and+Optimization+Workbench+%28eLBOW%29%3A+a+tool+for+ligand+coordinate+and+restraint+generation&doi=10.1107%2FS0907444909029436"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">electronic Ligand Builder and Optimization Workbench (eLBOW): a tool for ligand coordinate and restraint generation</span></div><div class="casAuthors">Moriarty, Nigel W.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">65</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1074-1080</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The electronic Ligand Builder and Optimization Workbench (eLBOW) is a program module of the PHENIX suite of computational crystallog. software.  It is designed to be a flexible procedure that uses simple and fast quantum-chem. techniques to provide chem. accurate information for novel and known ligands alike.  A variety of input formats and options allow the attainment of a no. of diverse goals including geometry optimization and generation of restraints.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojnN4AVY_DwrVg90H21EOLACvtfcHk0lgftWPgXh14vQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtFKnsLnN&md5=3f9a9c6c31c9a4b014a9f151fb6aa8d9</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1107%2FS0907444909029436&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909029436%26sid%3Dliteratum%253Aachs%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26atitle%3Delectronic%2520Ligand%2520Builder%2520and%2520Optimization%2520Workbench%2520%2528eLBOW%2529%253A%2520a%2520tool%2520for%2520ligand%2520coordinate%2520and%2520restraint%2520generation%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2009%26volume%3D65%26spage%3D1074%26epage%3D1080%26doi%3D10.1107%2FS0907444909029436" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group">Chen, V. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arendall, W. B.,  3rd</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Headd, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keedy, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Immormino, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kapral, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, L. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, D. C.</span><span> </span><span class="NLM_article-title">MolProbity: all-atom structure validation for macromolecular crystallography</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">12</span><span class="NLM_x">–</span> <span class="NLM_lpage">21</span><span class="refDoi"> DOI: 10.1107/S0907444909042073</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=10.1107%2FS0907444909042073" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=20057044" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=1%3ACAS%3A528%3ADC%252BC3cXit1Kktg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=12-21&author=V.+B.+Chenauthor=W.+B.+Arendallauthor=J.+J.+Headdauthor=D.+A.+Keedyauthor=R.+M.+Immorminoauthor=G.+J.+Kapralauthor=L.+W.+Murrayauthor=J.+S.+Richardsonauthor=D.+C.+Richardson&title=MolProbity%3A+all-atom+structure+validation+for+macromolecular+crystallography&doi=10.1107%2FS0907444909042073"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">MolProbity: all-atom structure validation for macromolecular crystallography</span></div><div class="casAuthors">Chen, Vincent B.; Arendall, W. Bryan, III; Headd, Jeffrey J.; Keedy, Daniel A.; Immormino, Robert M.; Kapral, Gary J.; Murray, Laura W.; Richardson, Jane S.; Richardson, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">12-21</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">MolProbity is a structure-validation web service that provides broad-spectrum solidly based evaluation of model quality at both the global and local levels for both proteins and nucleic acids.  It relies heavily on the power and sensitivity provided by optimized hydrogen placement and all-atom contact anal., complemented by updated versions of covalent-geometry and torsion-angle criteria.  Some of the local corrections can be performed automatically in MolProbity and all of the diagnostics are presented in chart and graphical forms that help guide manual rebuilding.  X-ray crystallog. provides a wealth of biol. important mol. data in the form of at. three-dimensional structures of proteins, nucleic acids and increasingly large complexes in multiple forms and states.  Advances in automation, in everything from crystn. to data collection to phasing to model building to refinement, have made solving a structure using crystallog. easier than ever.  However, despite these improvements, local errors that can affect biol. interpretation are widespread at low resoln. and even high-resoln. structures nearly all contain at least a few local errors such as Ramachandran outliers, flipped branched protein side chains and incorrect sugar puckers.  It is crit. both for the crystallographer and for the end user that there are easy and reliable methods to diagnose and correct these sorts of errors in structures.  MolProbity is the authors' contribution to helping solve this problem and this article reviews its general capabilities, reports on recent enhancements and usage, and presents evidence that the resulting improvements are now beneficially affecting the global database.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIjgsh59dO-7Vg90H21EOLACvtfcHk0lgftWPgXh14vQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXit1Kktg%253D%253D&md5=b5fc7574f43f01dd6e43c3663ca4f779</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1107%2FS0907444909042073&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909042073%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DArendall%26aufirst%3DW.%2BB.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DKeedy%26aufirst%3DD.%2BA.%26aulast%3DImmormino%26aufirst%3DR.%2BM.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DMurray%26aufirst%3DL.%2BW.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26atitle%3DMolProbity%253A%2520all-atom%2520structure%2520validation%2520for%2520macromolecular%2520crystallography%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D12%26epage%3D21%26doi%3D10.1107%2FS0907444909042073" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group">Bradford, M. M.</span><span> </span><span class="NLM_article-title">A rapid and sensitive method for the quantitation of microgram quantities of protein utilizing the principle of protein-dye binding</span> <span class="citation_source-journal">Anal. Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1976</span><span class="NLM_x">, </span> <span class="NLM_volume">72</span><span class="NLM_x">, </span> <span class="NLM_fpage">248</span><span class="NLM_x">–</span> <span class="NLM_lpage">254</span><span class="refDoi"> DOI: 10.1016/0003-2697(76)90527-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=10.1016%2F0003-2697%2876%2990527-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=942051" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=1%3ACAS%3A528%3ADyaE28XksVehtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=1976&pages=248-254&author=M.+M.+Bradford&title=A+rapid+and+sensitive+method+for+the+quantitation+of+microgram+quantities+of+protein+utilizing+the+principle+of+protein-dye+binding&doi=10.1016%2F0003-2697%2876%2990527-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">A rapid and sensitive method for the quantitation of microgram quantities of protein utilizing the principle of protein-dye binding</span></div><div class="casAuthors">Bradford, Marion M.</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Biochemistry</span>
        (<span class="NLM_cas:date">1976</span>),
    <span class="NLM_cas:volume">72</span>
        (<span class="NLM_cas:issue">1-2</span>),
    <span class="NLM_cas:pages">248-54</span>CODEN:
                <span class="NLM_cas:coden">ANBCA2</span>;
        ISSN:<span class="NLM_cas:issn">0003-2697</span>.
    </div><div class="casAbstract">A protein detn. method that involves the binding of coomassie Brilliant Blue G 250 to protein is described.  The binding of the dye to protein causes a shift in the absorption max. of the dye from 465 to 595 nm, and it is the increase in absorption at 595 nm that is monitored.  This assay is very reproducible and rapid with the dye binding process virtually complete in ∼ 2 min with good color stability for 1 hr.  There is little or no interference from cations such as Na+ or K+ nor from carbohydrates such as sucrose.  A small amt. of color is developed in the presence of strongly alk. buffering agents, but the assay may be run accurately by the use of proper buffer controls.  The only components found to give excessive interfering color in the assay are relatively large amts. of detergents such as Na dodecyl sulfate, Triton X 100, and commercial glassware detergents.  Interference by small amts. of detergent may be eliminated by the use of proper control.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrN_T_d0trcNrVg90H21EOLACvtfcHk0lgnjQ65epl1wg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE28XksVehtrY%253D&md5=43f388a493becc4193c7a49d9de0e1db</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1016%2F0003-2697%2876%2990527-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0003-2697%252876%252990527-3%26sid%3Dliteratum%253Aachs%26aulast%3DBradford%26aufirst%3DM.%2BM.%26atitle%3DA%2520rapid%2520and%2520sensitive%2520method%2520for%2520the%2520quantitation%2520of%2520microgram%2520quantities%2520of%2520protein%2520utilizing%2520the%2520principle%2520of%2520protein-dye%2520binding%26jtitle%3DAnal.%2520Biochem.%26date%3D1976%26volume%3D72%26spage%3D248%26epage%3D254%26doi%3D10.1016%2F0003-2697%2876%2990527-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 13 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jovana R  Gotovac</span>, <span class="hlFld-ContribAuthor ">David SH  Liu</span>, <span class="hlFld-ContribAuthor ">Michael J  Yates</span>, <span class="hlFld-ContribAuthor ">Julia V  Milne</span>, <span class="hlFld-ContribAuthor ">Arthi A  Macpherson</span>, <span class="hlFld-ContribAuthor ">Kaylene J  Simpson</span>, <span class="hlFld-ContribAuthor ">Guy D  Eslick</span>, <span class="hlFld-ContribAuthor ">Catherine  Mitchell</span>, <span class="hlFld-ContribAuthor ">Cuong P  Duong</span>, <span class="hlFld-ContribAuthor ">Wayne A  Phillips</span>, <span class="hlFld-ContribAuthor ">Nicholas J  Clemons</span>. </span><span class="cited-content_cbyCitation_article-title">GRB7
              is an oncogenic driver and potential therapeutic target in oesophageal adenocarcinoma. </span><span class="cited-content_cbyCitation_journal-name">The Journal of Pathology</span><span> <strong>2020,</strong> <em>252 </em>
                                    (3)
                                     , 317-329. <a href="https://doi.org/10.1002/path.5528" title="DOI URL">https://doi.org/10.1002/path.5528</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/path.5528&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fpath.5528%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Journal%2520of%2520Pathology%26atitle%3DGRB7%252Bis%252Ban%252Boncogenic%252Bdriver%252Band%252Bpotential%252Btherapeutic%252Btarget%252Bin%252Boesophageal%252Badenocarcinoma%26aulast%3DGotovac%26aufirst%3DJovana%2BR%26date%3D2020%26date%3D2020%26volume%3D252%26issue%3D3%26spage%3D317%26epage%3D329" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ketav  Kulkarni</span>, <span class="hlFld-ContribAuthor ">Jianrong  Sang</span>, <span class="hlFld-ContribAuthor ">Xiuquan  Ma</span>, <span class="hlFld-ContribAuthor ">Jacqueline A.  Wilce</span>. </span><span class="cited-content_cbyCitation_article-title">Comparison between clickable cyclic TAT and penetratin for delivery of cyclic and bicyclic‐peptide cargos. </span><span class="cited-content_cbyCitation_journal-name">Peptide Science</span><span> <strong>2020,</strong> <em>112 </em>
                                    (4)
                                     <a href="https://doi.org/10.1002/pep2.24163" title="DOI URL">https://doi.org/10.1002/pep2.24163</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/pep2.24163&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fpep2.24163%26sid%3Dliteratum%253Aachs%26jtitle%3DPeptide%2520Science%26atitle%3DComparison%252Bbetween%252Bclickable%252Bcyclic%252BTAT%252Band%252Bpenetratin%252Bfor%252Bdelivery%252Bof%252Bcyclic%252Band%252Bbicyclic%2525E2%252580%252590peptide%252Bcargos%26aulast%3DKulkarni%26aufirst%3DKetav%26date%3D2020%26date%3D2020%26volume%3D112%26issue%3D4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Robert A.  Cerulli</span>, <span class="hlFld-ContribAuthor ">Livia  Shehaj</span>, <span class="hlFld-ContribAuthor ">Isidora  Tosic</span>, <span class="hlFld-ContribAuthor ">Kevin  Jiang</span>, <span class="hlFld-ContribAuthor ">Jing  Wang</span>, <span class="hlFld-ContribAuthor ">David A.  Frank</span>, <span class="hlFld-ContribAuthor ">Joshua A.  Kritzer</span>. </span><span class="cited-content_cbyCitation_article-title">Cytosolic delivery of peptidic STAT3 SH2 domain inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2020,</strong> <em>28 </em>
                                    (12)
                                     , 115542. <a href="https://doi.org/10.1016/j.bmc.2020.115542" title="DOI URL">https://doi.org/10.1016/j.bmc.2020.115542</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2020.115542&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2020.115542%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DCytosolic%252Bdelivery%252Bof%252Bpeptidic%252BSTAT3%252BSH2%252Bdomain%252Binhibitors%26aulast%3DCerulli%26aufirst%3DRobert%2BA.%26date%3D2020%26volume%3D28%26issue%3D12%26spage%3D115542" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Juan  Alcalde</span>, <span class="hlFld-ContribAuthor ">María  González-Muñoz</span>, <span class="hlFld-ContribAuthor ">Antonio  Villalobo</span>. </span><span class="cited-content_cbyCitation_article-title">Grb7-derived calmodulin-binding peptides inhibit proliferation, migration and invasiveness of tumor cells while they enhance attachment to the substrate. </span><span class="cited-content_cbyCitation_journal-name">Heliyon</span><span> <strong>2020,</strong> <em>6 </em>
                                    (5)
                                     , e03922. <a href="https://doi.org/10.1016/j.heliyon.2020.e03922" title="DOI URL">https://doi.org/10.1016/j.heliyon.2020.e03922</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.heliyon.2020.e03922&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.heliyon.2020.e03922%26sid%3Dliteratum%253Aachs%26jtitle%3DHeliyon%26atitle%3DGrb7-derived%252Bcalmodulin-binding%252Bpeptides%252Binhibit%252Bproliferation%25252C%252Bmigration%252Band%252Binvasiveness%252Bof%252Btumor%252Bcells%252Bwhile%252Bthey%252Benhance%252Battachment%252Bto%252Bthe%252Bsubstrate%26aulast%3DAlcalde%26aufirst%3DJuan%26date%3D2020%26volume%3D6%26issue%3D5%26spage%3De03922" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zachary R.  Crook</span>, <span class="hlFld-ContribAuthor ">Natalie W.  Nairn</span>, <span class="hlFld-ContribAuthor ">James M.  Olson</span>. </span><span class="cited-content_cbyCitation_article-title">Miniproteins as a Powerful Modality in Drug Development. </span><span class="cited-content_cbyCitation_journal-name">Trends in Biochemical Sciences</span><span> <strong>2020,</strong> <em>45 </em>
                                    (4)
                                     , 332-346. <a href="https://doi.org/10.1016/j.tibs.2019.12.008" title="DOI URL">https://doi.org/10.1016/j.tibs.2019.12.008</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.tibs.2019.12.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.tibs.2019.12.008%26sid%3Dliteratum%253Aachs%26jtitle%3DTrends%2520in%2520Biochemical%2520Sciences%26atitle%3DMiniproteins%252Bas%252Ba%252BPowerful%252BModality%252Bin%252BDrug%252BDevelopment%26aulast%3DCrook%26aufirst%3DZachary%2BR.%26date%3D2020%26volume%3D45%26issue%3D4%26spage%3D332%26epage%3D346" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Gabrielle M.  Watson</span>, <span class="hlFld-ContribAuthor ">Jacqueline A.  Wilce</span>. </span><span class="cited-content_cbyCitation_article-title">Direct Interaction between Calmodulin and the Grb7 RA-PH Domain. </span><span class="cited-content_cbyCitation_journal-name">International Journal of Molecular Sciences</span><span> <strong>2020,</strong> <em>21 </em>
                                    (4)
                                     , 1336. <a href="https://doi.org/10.3390/ijms21041336" title="DOI URL">https://doi.org/10.3390/ijms21041336</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/ijms21041336&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fijms21041336%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Molecular%2520Sciences%26atitle%3DDirect%252BInteraction%252Bbetween%252BCalmodulin%252Band%252Bthe%252BGrb7%252BRA-PH%252BDomain%26aulast%3DWatson%26aufirst%3DGabrielle%2BM.%26date%3D2020%26date%3D2020%26volume%3D21%26issue%3D4%26spage%3D1336" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Robert A.  Cerulli</span>, <span class="hlFld-ContribAuthor ">Joshua A.  Kritzer</span>. </span><span class="cited-content_cbyCitation_article-title">Phosphotyrosine isosteres: past, present and future. </span><span class="cited-content_cbyCitation_journal-name">Organic & Biomolecular Chemistry</span><span> <strong>2020,</strong> <em>18 </em>
                                    (4)
                                     , 583-605. <a href="https://doi.org/10.1039/C9OB01998G" title="DOI URL">https://doi.org/10.1039/C9OB01998G</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C9OB01998G&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC9OB01998G%26sid%3Dliteratum%253Aachs%26jtitle%3DOrganic%2520%2526%2520Biomolecular%2520Chemistry%26atitle%3DPhosphotyrosine%252Bisosteres%25253A%252Bpast%25252C%252Bpresent%252Band%252Bfuture%26aulast%3DCerulli%26aufirst%3DRobert%2BA.%26date%3D2020%26date%3D2020%26volume%3D18%26issue%3D4%26spage%3D583%26epage%3D605" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yexian  Zhang</span>, <span class="hlFld-ContribAuthor ">Ruoyao  Shi</span>, <span class="hlFld-ContribAuthor ">Chaorong  Chen</span>, <span class="hlFld-ContribAuthor ">Meiyu  Duan</span>, <span class="hlFld-ContribAuthor ">Shuai  Liu</span>, <span class="hlFld-ContribAuthor ">Yanjiao  Ren</span>, <span class="hlFld-ContribAuthor ">Lan  Huang</span>, <span class="hlFld-ContribAuthor ">Xiaofeng  Dai</span>, <span class="hlFld-ContribAuthor ">Fengfeng  Zhou</span>. </span><span class="cited-content_cbyCitation_article-title">ELMO: An Efficient Logistic Regression-Based Multi-Omic Integrated Analysis Method for Breast Cancer Intrinsic Subtypes. </span><span class="cited-content_cbyCitation_journal-name">IEEE Access</span><span> <strong>2020,</strong> <em>8 </em>, 5121-5130. <a href="https://doi.org/10.1109/ACCESS.2019.2960373" title="DOI URL">https://doi.org/10.1109/ACCESS.2019.2960373</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1109/ACCESS.2019.2960373&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1109%2FACCESS.2019.2960373%26sid%3Dliteratum%253Aachs%26jtitle%3DIEEE%2520Access%26atitle%3DELMO%25253A%252BAn%252BEfficient%252BLogistic%252BRegression-Based%252BMulti-Omic%252BIntegrated%252BAnalysis%252BMethod%252Bfor%252BBreast%252BCancer%252BIntrinsic%252BSubtypes%26aulast%3DZhang%26aufirst%3DYexian%26date%3D2020%26volume%3D8%26spage%3D5121%26epage%3D5130" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiaqing  Yu</span>, <span class="hlFld-ContribAuthor ">Peng  Zhong</span>, <span class="hlFld-ContribAuthor ">Yali  Han</span>, <span class="hlFld-ContribAuthor ">Qingqing  Huang</span>, <span class="hlFld-ContribAuthor ">Jian  Wang</span>, <span class="hlFld-ContribAuthor ">Chengyou  Jia</span>, <span class="hlFld-ContribAuthor ">Zhongwei  Lv</span>. </span><span class="cited-content_cbyCitation_article-title">Key candidate genes associated with BRAF
              V600E
              in papillary thyroid carcinoma on microarray analysis. </span><span class="cited-content_cbyCitation_journal-name">Journal of Cellular Physiology</span><span> <strong>2019,</strong> <em>234 </em>
                                    (12)
                                     , 23369-23378. <a href="https://doi.org/10.1002/jcp.28906" title="DOI URL">https://doi.org/10.1002/jcp.28906</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/jcp.28906&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fjcp.28906%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Cellular%2520Physiology%26atitle%3DKey%252Bcandidate%252Bgenes%252Bassociated%252Bwith%252BBRAF%252BV600E%252Bin%252Bpapillary%252Bthyroid%252Bcarcinoma%252Bon%252Bmicroarray%252Banalysis%26aulast%3DYu%26aufirst%3DXiaqing%26date%3D2019%26date%3D2019%26volume%3D234%26issue%3D12%26spage%3D23369%26epage%3D23378" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jianrong  Sang</span>, <span class="hlFld-ContribAuthor ">Ketav  Kulkarni</span>, <span class="hlFld-ContribAuthor ">Gabrielle M.  Watson</span>, <span class="hlFld-ContribAuthor ">Xiuquan  Ma</span>, <span class="hlFld-ContribAuthor ">David J.  Craik</span>, <span class="hlFld-ContribAuthor ">Sónia T.  Henriques</span>, <span class="hlFld-ContribAuthor ">Aaron G.  Poth</span>, <span class="hlFld-ContribAuthor ">Aurélie H.  Benfield</span>, <span class="hlFld-ContribAuthor ">Jacqueline A.  Wilce</span>. </span><span class="cited-content_cbyCitation_article-title">Evaluation of Cyclic Peptide Inhibitors of the Grb7 Breast Cancer Target: Small Change in Cargo Results in Large Change in Cellular Activity. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2019,</strong> <em>24 </em>
                                    (20)
                                     , 3739. <a href="https://doi.org/10.3390/molecules24203739" title="DOI URL">https://doi.org/10.3390/molecules24203739</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules24203739&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules24203739%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DEvaluation%252Bof%252BCyclic%252BPeptide%252BInhibitors%252Bof%252Bthe%252BGrb7%252BBreast%252BCancer%252BTarget%25253A%252BSmall%252BChange%252Bin%252BCargo%252BResults%252Bin%252BLarge%252BChange%252Bin%252BCellular%252BActivity%26aulast%3DSang%26aufirst%3DJianrong%26date%3D2019%26date%3D2019%26volume%3D24%26issue%3D20%26spage%3D3739" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Pei-Yu  Chu</span>, <span class="hlFld-ContribAuthor ">Yu-Ling  Tai</span>, <span class="hlFld-ContribAuthor ">Tang-Long  Shen</span>. </span><span class="cited-content_cbyCitation_article-title">Grb7, a Critical Mediator of EGFR/ErbB Signaling, in Cancer Development and as a Potential Therapeutic Target. </span><span class="cited-content_cbyCitation_journal-name">Cells</span><span> <strong>2019,</strong> <em>8 </em>
                                    (5)
                                     , 435. <a href="https://doi.org/10.3390/cells8050435" title="DOI URL">https://doi.org/10.3390/cells8050435</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/cells8050435&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fcells8050435%26sid%3Dliteratum%253Aachs%26jtitle%3DCells%26atitle%3DGrb7%25252C%252Ba%252BCritical%252BMediator%252Bof%252BEGFR%25252FErbB%252BSignaling%25252C%252Bin%252BCancer%252BDevelopment%252Band%252Bas%252Ba%252BPotential%252BTherapeutic%252BTarget%26aulast%3DChu%26aufirst%3DPei-Yu%26date%3D2019%26date%3D2019%26volume%3D8%26issue%3D5%26spage%3D435" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Iteng  Ng-Choi</span>, <span class="hlFld-ContribAuthor ">Àngel  Oliveras</span>, <span class="hlFld-ContribAuthor ">Lidia  Feliu</span>, <span class="hlFld-ContribAuthor ">Marta  Planas</span>. </span><span class="cited-content_cbyCitation_article-title">Solid-phase synthesis of biaryl bicyclic peptides containing a 3-aryltyrosine or a 4-arylphenylalanine moiety. </span><span class="cited-content_cbyCitation_journal-name">Beilstein Journal of Organic Chemistry</span><span> <strong>2019,</strong> <em>15 </em>, 761-768. <a href="https://doi.org/10.3762/bjoc.15.72" title="DOI URL">https://doi.org/10.3762/bjoc.15.72</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3762/bjoc.15.72&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3762%2Fbjoc.15.72%26sid%3Dliteratum%253Aachs%26jtitle%3DBeilstein%2520Journal%2520of%2520Organic%2520Chemistry%26atitle%3DSolid-phase%252Bsynthesis%252Bof%252Bbiaryl%252Bbicyclic%252Bpeptides%252Bcontaining%252Ba%252B3-aryltyrosine%252Bor%252Ba%252B4-arylphenylalanine%252Bmoiety%26aulast%3DNg-Choi%26aufirst%3DIteng%26date%3D2019%26date%3D2019%26volume%3D15%26spage%3D761%26epage%3D768" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ketav  Kulkarni</span>, <span class="hlFld-ContribAuthor ">Gabrielle M.  Watson</span>, <span class="hlFld-ContribAuthor ">Jianrong  Sang</span>, <span class="hlFld-ContribAuthor ">Jacqueline A.  Wilce</span>. </span><span class="cited-content_cbyCitation_article-title">Preparation and cellular uptake of bicyclic-peptide cargo clicked to cell penetrating peptides. </span><span class="cited-content_cbyCitation_journal-name">Peptide Science</span><span> <strong>2018,</strong> <em>110 </em>
                                    (3)
                                     , e24037. <a href="https://doi.org/10.1002/pep2.24037" title="DOI URL">https://doi.org/10.1002/pep2.24037</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/pep2.24037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fpep2.24037%26sid%3Dliteratum%253Aachs%26jtitle%3DPeptide%2520Science%26atitle%3DPreparation%252Band%252Bcellular%252Buptake%252Bof%252Bbicyclic-peptide%252Bcargo%252Bclicked%252Bto%252Bcell%252Bpenetrating%252Bpeptides%26aulast%3DKulkarni%26aufirst%3DKetav%26date%3D2018%26date%3D2018%26volume%3D110%26issue%3D3%26spage%3De24037" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01320/20171117/images/medium/jm-2017-01320h_0006.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01320/20171117/images/large/jm-2017-01320h_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01320&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01320/20171117/images/medium/jm-2017-01320h_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01320/20171117/images/large/jm-2017-01320h_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Redesigning Grb7 targeted bicyclics. (A) X-ray crystal structure of peptide <b>2</b> (blue sticks) bound to the Grb7–SH2 (gray cartoon and surface representation) identifying a surface exposed aspartic acid (orange sticks) in the Grb7–SH2 EF loop (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EEL">5EEL</a>), <b>(</b>B<b>)</b> Schematic and chemical structure of peptides <b>1</b>–<b>5</b>. For the schematics, the amino acid code is displayed in color and the remainder of the schematic conveys the chemical structure including the amidated C-terminus.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01320/20171117/images/large/jm-2017-01320h_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01320&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01320/20171117/images/medium/jm-2017-01320h_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01320/20171117/images/large/jm-2017-01320h_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Binding studies of bicyclic inhibitors for Grb7–SH2. (A) Equilibrium binding curves for peptides <b>3</b>–<b>5</b> binding to Grb7–SH2 in high phosphate buffer (50 mM, +PO<sub>4</sub><sup>3–</sup>), (B–D) corresponding SPR sensorgrams, (E) equilibrium binding curves for peptides <b>3</b>–<b>5</b> binding to the Grb7–SH2 in low phosphate buffer (1 mM; −PO<sub>4</sub><sup>3–</sup>), (F–H) Representative SPR sensorgrams. The peptide concentrations utilized are listed on the left of the sensorgrams, with lower concentrations omitted for clarity.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01320/20171117/images/large/jm-2017-01320h_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01320&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01320/20171117/images/medium/jm-2017-01320h_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01320/20171117/images/large/jm-2017-01320h_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. X-ray crystal structure of the Grb7–SH2/peptide <b>5</b> complex and specificity analysis. (A, B) Surface representations of the Grb7–SH2 (gray) with either bound peptide <b>2</b> (blue sticks) or peptide <b>5</b> (green sticks), (C, D) Close-up of the Grb7–SH2 pY binding pocket with key amino acids shown in stick format and hydrogen bonds displayed as dashed lines (PDB IDs <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EEL">5EEL</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5U1Q">5U1Q</a>). (E, F) SPR equilibrium binding curves for peptide <b>5</b> binding to the Grb7–SH2 (gray), Grb2–SH2 (wheat), and Grb10–SH2 (purple) in either high phosphate buffer (50 mM; +PO<sub>4</sub><sup>3–</sup>) or low phosphate buffer (1 mM; −PO<sub>4</sub><sup>3–</sup>). Representative sensorgrams are provided as <a href="/doi/suppl/10.1021/acs.jmedchem.7b01320/suppl_file/jm7b01320_si_001.pdf" class="ext-link">Supporting Information</a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01320/20171117/images/large/jm-2017-01320h_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01320&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01320/20171117/images/medium/jm-2017-01320h_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01320/20171117/images/large/jm-2017-01320h_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Redesigning pY mimetic containing bicyclic peptides. (A) Chemical structure of peptides <b>6</b> and <b>7</b> with the pY mimetics (cmF and cF) highlighted by the pink and purple boxes, respectively. (B) SPR equilibrium binding curves of peptide <b>6</b> (top) and peptide <b>7</b> (bottom) binding to the Grb7–SH2 in both high (50 mM; +PO<sub>4</sub><sup>3–</sup>) and low (1 mM; −PO<sub>4</sub><sup>3–</sup>) phosphate containing buffer conditions. Representative sensorgrams are provided as <a href="/doi/suppl/10.1021/acs.jmedchem.7b01320/suppl_file/jm7b01320_si_001.pdf" class="ext-link">Supporting Information</a>. (C) Close-up of the Grb7–SH2 pY binding pocket (gray) highlighting peptide <b>6</b> (top) and peptide <b>7</b> (bottom) interacting residues (stick format). Waters are displayed as red spheres and hydrogen bonds as dashed lines. (D) Surface representation of the Grb7–SH2 (colored according to atom type: N = blue, O = red, S = yellow, C = gray), with either peptide <b>6</b> (top) or peptide <b>7</b> (bottom). Grb7–SH2/peptide <b>6</b> PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5U06">5U06</a>, Grb7–SH2/peptide <b>7</b> PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5TYI">5TYI</a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01320/20171117/images/large/jm-2017-01320h_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01320&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01320/20171117/images/medium/jm-2017-01320h_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01320/20171117/images/large/jm-2017-01320h_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Optimized inhibitors block Grb7 interactions with cellular partners. (A) SKBR3 cells were serum starved overnight and treated for 1 h with Grb7 targeting peptides (20 μM) prior to stimulation for 10 min with fibronectin (10 μg·mL<sup>–1</sup>). Washed cells were lysed and immunoprecipitated using Grb7 antibodies before being probed with various antibodies for the detection of Grb7 binding partners: SHC, top panel; HER2, second panel; FAK, third panel; Grb7 on the bottom panel served as a loading control. FN = fibronectin, P = penetratin control peptide. (B) As per panel A, with the exception that the Grb7 targeting peptides were added following cell lysis. Schematics of the penetratin coupled peptides tested are shown on the right of the image with the peptide sequence conveyed as single letter code and code displayed in color and the remainder of the schematic conveying the peptide chemical structure.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-22/acs.jmedchem.7b01320/20171117/images/large/jm-2017-01320h_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01320&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i37">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_76446" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_76446" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 48 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Ivanov, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Khuri, F. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fu, H.</span><span> </span><span class="NLM_article-title">Targeting protein-protein interactions as an anticancer strategy</span> <span class="citation_source-journal">Trends Pharmacol. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">34</span><span class="NLM_x">, </span> <span class="NLM_fpage">393</span><span class="NLM_x">–</span> <span class="NLM_lpage">400</span><span class="refDoi"> DOI: 10.1016/j.tips.2013.04.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=10.1016%2Fj.tips.2013.04.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=23725674" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=1%3ACAS%3A528%3ADC%252BC3sXosVGjt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2013&pages=393-400&author=A.+A.+Ivanovauthor=F.+R.+Khuriauthor=H.+Fu&title=Targeting+protein-protein+interactions+as+an+anticancer+strategy&doi=10.1016%2Fj.tips.2013.04.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting protein-protein interactions as an anticancer strategy</span></div><div class="casAuthors">Ivanov, Andrei A.; Khuri, Fadlo R.; Fu, Haian</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Pharmacological Sciences</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">393-400</span>CODEN:
                <span class="NLM_cas:coden">TPHSDY</span>;
        ISSN:<span class="NLM_cas:issn">0165-6147</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  The emergence and convergence of cancer genomics, targeted therapies, and network oncol. have significantly expanded the landscape of protein-protein interaction (PPI) networks in cancer for therapeutic discovery.  Extensive biol. and clin. investigations have led to the identification of protein interaction hubs and nodes that are crit. for the acquisition and maintenance of characteristics of cancer essential for cell transformation.  Such cancer-enabling PPIs have become promising therapeutic targets.  With technol. advances in PPI modulator discovery and validation of PPI-targeting agents in clin. settings, targeting of PPI interfaces as an anticancer strategy has become a reality.  Future research directed at genomics-based PPI target discovery, PPI interface characterization, PPI-focused chem. library design, and patient-genomic subpopulation-driven clin. studies is expected to accelerate the development of the next generation of PPI-based anticancer agents for personalized precision medicine.  Here we briefly review prominent PPIs that mediate cancer-acquired properties, highlight recognized challenges and promising clin. results in targeting PPIs, and outline emerging opportunities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqKeFT8Id1vhbVg90H21EOLACvtfcHk0lgB9bn2G-_fcA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXosVGjt7g%253D&md5=2f4772a4abfa6640b6af922671834fa3</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.tips.2013.04.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tips.2013.04.007%26sid%3Dliteratum%253Aachs%26aulast%3DIvanov%26aufirst%3DA.%2BA.%26aulast%3DKhuri%26aufirst%3DF.%2BR.%26aulast%3DFu%26aufirst%3DH.%26atitle%3DTargeting%2520protein-protein%2520interactions%2520as%2520an%2520anticancer%2520strategy%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2013%26volume%3D34%26spage%3D393%26epage%3D400%26doi%3D10.1016%2Fj.tips.2013.04.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Pawson, T.</span><span> </span><span class="NLM_article-title">Specificity in signal transduction: from phosphotyrosine-SH2 domain interactions to complex cellular systems</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">116</span><span class="NLM_x">, </span> <span class="NLM_fpage">191</span><span class="NLM_x">–</span> <span class="NLM_lpage">203</span><span class="refDoi"> DOI: 10.1016/S0092-8674(03)01077-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=10.1016%2FS0092-8674%2803%2901077-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=14744431" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtValsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=2004&pages=191-203&author=T.+Pawson&title=Specificity+in+signal+transduction%3A+from+phosphotyrosine-SH2+domain+interactions+to+complex+cellular+systems&doi=10.1016%2FS0092-8674%2803%2901077-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Specificity in signal transduction: From phosphotyrosine-SH2 domain interactions to complex cellular systems</span></div><div class="casAuthors">Pawson, Tony</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">191-203</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review and discussion.  Over the last 2 decades, a new and unifying concept of cellular organization has emerged in which modular protein-protein interactions provide an underlying framework through which signaling pathways are assembled and controlled.  In this scheme, post-translational modifications such as phosphorylation commonly exert their biol. effects by regulating mol. interactions, exemplified by the ability of phosphotyrosine sites to bind selectively to SH2 domains.  Although these interactions are rather simple in isolation, they can nonetheless be exploited to generate complex cellular systems.  Here, the author discusses expts. that have led to this view of dynamic cellular behavior and identifies some current and future areas of interest in cell signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrolI53c79a6rVg90H21EOLACvtfcHk0liw3hcKVoSupg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtValsrk%253D&md5=8521703a8fe32fda8ddb93adb8055af3</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2FS0092-8674%2803%2901077-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0092-8674%252803%252901077-8%26sid%3Dliteratum%253Aachs%26aulast%3DPawson%26aufirst%3DT.%26atitle%3DSpecificity%2520in%2520signal%2520transduction%253A%2520from%2520phosphotyrosine-SH2%2520domain%2520interactions%2520to%2520complex%2520cellular%2520systems%26jtitle%3DCell%26date%3D2004%26volume%3D116%26spage%3D191%26epage%3D203%26doi%3D10.1016%2FS0092-8674%2803%2901077-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Hanahan, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weinberg, R. A.</span><span> </span><span class="NLM_article-title">Hallmarks of cancer: the next generation</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">144</span><span class="NLM_x">, </span> <span class="NLM_fpage">646</span><span class="NLM_x">–</span> <span class="NLM_lpage">674</span><span class="refDoi"> DOI: 10.1016/j.cell.2011.02.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=10.1016%2Fj.cell.2011.02.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=21376230" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=1%3ACAS%3A528%3ADC%252BC3MXjsFeqtrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=144&publication_year=2011&pages=646-674&author=D.+Hanahanauthor=R.+A.+Weinberg&title=Hallmarks+of+cancer%3A+the+next+generation&doi=10.1016%2Fj.cell.2011.02.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Hallmarks of cancer: the next generation</span></div><div class="casAuthors">Hanahan, Douglas; Weinberg, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">144</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">646-674</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  The hallmarks of cancer comprise six biol. capabilities acquired during the multistep development of human tumors.  The hallmarks constitute an organizing principle for rationalizing the complexities of neoplastic disease.  They include sustaining proliferative signaling, evading growth suppressors, resisting cell death, enabling replicative immortality, inducing angiogenesis, and activating invasion and metastasis.  Underlying these hallmarks are genome instability, which generates the genetic diversity that expedites their acquisition, and inflammation, which fosters multiple hallmark functions.  Conceptual progress in the last decade has added two emerging hallmarks of potential generality to this list-reprogramming of energy metab. and evading immune destruction.  In addn. to cancer cells, tumors exhibit another dimension of complexity: they contain a repertoire of recruited, ostensibly normal cells that contribute to the acquisition of hallmark traits by creating the "tumor microenvironment.".  Recognition of the widespread applicability of these concepts will increasingly affect the development of new means to treat human cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTcpqXALcb6LVg90H21EOLACvtfcHk0liw3hcKVoSupg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXjsFeqtrk%253D&md5=b36f160c417a712e83cb83c577f0018e</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2011.02.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2011.02.013%26sid%3Dliteratum%253Aachs%26aulast%3DHanahan%26aufirst%3DD.%26aulast%3DWeinberg%26aufirst%3DR.%2BA.%26atitle%3DHallmarks%2520of%2520cancer%253A%2520the%2520next%2520generation%26jtitle%3DCell%26date%3D2011%26volume%3D144%26spage%3D646%26epage%3D674%26doi%3D10.1016%2Fj.cell.2011.02.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Gao, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yin, H.</span><span> </span><span class="NLM_article-title">Targeting protein-protein interfaces using macrocyclic peptides</span> <span class="citation_source-journal">Biopolymers</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">104</span><span class="NLM_x">, </span> <span class="NLM_fpage">310</span><span class="NLM_x">–</span> <span class="NLM_lpage">316</span><span class="refDoi"> DOI: 10.1002/bip.22625</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=10.1002%2Fbip.22625" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=25664609" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1egu7jP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=2015&pages=310-316&author=M.+Gaoauthor=K.+Chengauthor=H.+Yin&title=Targeting+protein-protein+interfaces+using+macrocyclic+peptides&doi=10.1002%2Fbip.22625"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting protein-protein interfaces using macrocyclic peptides</span></div><div class="casAuthors">Gao, Meng; Cheng, Kui; Yin, Hang</div><div class="citationInfo"><span class="NLM_cas:title">Biopolymers</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">104</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">310-316</span>CODEN:
                <span class="NLM_cas:coden">BIPMAA</span>;
        ISSN:<span class="NLM_cas:issn">0006-3525</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A review.  Protein-protein interactions (PPIs) are crit. in numerous biol. processes including signaling transduction, function regulations, and disease development.  To regulate PPIs has been thought to be challenging due to their highly dynamic and expansive interfacial areas.  Nonetheless, successful examples have been reported of targeting PPIs using small mols., peptides, and proteins.  Peptides, esp. macrocyclic peptides have proven to be a particularly useful tool to inhibit PPIs for their exquisite potency, stability and selectivity.  Herein we review the recent developments of this area of research, focusing on the macrocyclic peptides isolated from natural products, identified from library screening, and rationally designed based on structures, as PPI regulators. © 2015 Wiley Periodicals, Inc. Biopolymers (Pept Sci) 104: 310-316, 2015.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfC3LNeS7WxrVg90H21EOLACvtfcHk0liw3hcKVoSupg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1egu7jP&md5=340696f3a54e9cb6f9b48fed6c7a5e45</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1002%2Fbip.22625&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fbip.22625%26sid%3Dliteratum%253Aachs%26aulast%3DGao%26aufirst%3DM.%26aulast%3DCheng%26aufirst%3DK.%26aulast%3DYin%26aufirst%3DH.%26atitle%3DTargeting%2520protein-protein%2520interfaces%2520using%2520macrocyclic%2520peptides%26jtitle%3DBiopolymers%26date%3D2015%26volume%3D104%26spage%3D310%26epage%3D316%26doi%3D10.1002%2Fbip.22625" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Stein, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fuqua, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roonprapunt, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yajnik, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">D’Eustachio, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moskow, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buchberg, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Osborne, C. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Margolis, B.</span><span> </span><span class="NLM_article-title">The SH2 domain protein GRB-7 is co-amplified, overexpressed and in a tight complex with HER2 in breast cancer</span> <span class="citation_source-journal">EMBO J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">1331</span><span class="NLM_x">–</span> <span class="NLM_lpage">1340</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=1994&pages=1331-1340&author=D.+Steinauthor=J.+Wuauthor=S.+A.+Fuquaauthor=C.+Roonprapuntauthor=V.+Yajnikauthor=P.+D%E2%80%99Eustachioauthor=J.+J.+Moskowauthor=A.+M.+Buchbergauthor=C.+K.+Osborneauthor=B.+Margolis&title=The+SH2+domain+protein+GRB-7+is+co-amplified%2C+overexpressed+and+in+a+tight+complex+with+HER2+in+breast+cancer"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DStein%26aufirst%3DD.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DFuqua%26aufirst%3DS.%2BA.%26aulast%3DRoonprapunt%26aufirst%3DC.%26aulast%3DYajnik%26aufirst%3DV.%26aulast%3DD%25E2%2580%2599Eustachio%26aufirst%3DP.%26aulast%3DMoskow%26aufirst%3DJ.%2BJ.%26aulast%3DBuchberg%26aufirst%3DA.%2BM.%26aulast%3DOsborne%26aufirst%3DC.%2BK.%26aulast%3DMargolis%26aufirst%3DB.%26atitle%3DThe%2520SH2%2520domain%2520protein%2520GRB-7%2520is%2520co-amplified%252C%2520overexpressed%2520and%2520in%2520a%2520tight%2520complex%2520with%2520HER2%2520in%2520breast%2520cancer%26jtitle%3DEMBO%2520J.%26date%3D1994%26volume%3D13%26spage%3D1331%26epage%3D1340" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Tanaka, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mori, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Akiyoshi, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tanaka, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mafune, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wands, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sugimachi, K.</span><span> </span><span class="NLM_article-title">Coexpression of Grb7 with epidermal growth factor receptor or Her2/erbB2 in human advanced esophageal carcinoma</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">28</span><span class="NLM_x">–</span> <span class="NLM_lpage">31</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=1997&pages=28-31&author=S.+Tanakaauthor=M.+Moriauthor=T.+Akiyoshiauthor=Y.+Tanakaauthor=K.+Mafuneauthor=J.+R.+Wandsauthor=K.+Sugimachi&title=Coexpression+of+Grb7+with+epidermal+growth+factor+receptor+or+Her2%2FerbB2+in+human+advanced+esophageal+carcinoma"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DTanaka%26aufirst%3DS.%26aulast%3DMori%26aufirst%3DM.%26aulast%3DAkiyoshi%26aufirst%3DT.%26aulast%3DTanaka%26aufirst%3DY.%26aulast%3DMafune%26aufirst%3DK.%26aulast%3DWands%26aufirst%3DJ.%2BR.%26aulast%3DSugimachi%26aufirst%3DK.%26atitle%3DCoexpression%2520of%2520Grb7%2520with%2520epidermal%2520growth%2520factor%2520receptor%2520or%2520Her2%252FerbB2%2520in%2520human%2520advanced%2520esophageal%2520carcinoma%26jtitle%3DCancer%2520Res.%26date%3D1997%26volume%3D57%26spage%3D28%26epage%3D31" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Tanaka, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pero, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taguchi, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shimada, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mori, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krag, D. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arii, S.</span><span> </span><span class="NLM_article-title">Specific peptide ligand for Grb7 signal transduction protein and pancreatic cancer metastasis</span> <span class="citation_source-journal">J. Natl. Cancer Inst.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">98</span><span class="NLM_x">, </span> <span class="NLM_fpage">491</span><span class="NLM_x">–</span> <span class="NLM_lpage">498</span><span class="refDoi"> DOI: 10.1093/jnci/djj105</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=10.1093%2Fjnci%2Fdjj105" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=16595785" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=1%3ACAS%3A528%3ADC%252BD28XjtlamtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=2006&pages=491-498&author=S.+Tanakaauthor=S.+C.+Peroauthor=K.+Taguchiauthor=M.+Shimadaauthor=M.+Moriauthor=D.+N.+Kragauthor=S.+Arii&title=Specific+peptide+ligand+for+Grb7+signal+transduction+protein+and+pancreatic+cancer+metastasis&doi=10.1093%2Fjnci%2Fdjj105"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Specific Peptide Ligand for Grb7 Signal Transduction Protein and Pancreatic Cancer Metastasis</span></div><div class="casAuthors">Tanaka, Shinji; Pero, Stephanie C.; Taguchi, Kenichi; Shimada, Mitsuo; Mori, Masaki; Krag, David N.; Arii, Shigeki</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the National Cancer Institute</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">491-498</span>CODEN:
                <span class="NLM_cas:coden">JNCIEQ</span>;
        ISSN:<span class="NLM_cas:issn">0027-8874</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Background: Pancreatic cancer is one of the most aggressive malignancies, with high rates of invasion and metastasis and with generally poor prognosis.  We previously found that metastasis was strongly assocd. with the expression of growth factor receptor-bound protein 7 (Grb7), which contains a Src homol. 2 (SH2) domain.  In this study, we evaluated Grb7 protein as a mol. target of therapy for metastatic pancreatic cancer.  Methods: Grb7 protein expression was measured by immunohistochem. in 36 human pancreatic cancer specimens and adjacent normal pancreatic tissue.  We synthesized a nonphosphorylated peptide inhibitor that binds specifically to the SH2 domain of Grb7.  Intracellular signaling was assessed by immunopptn. and immunoblot assays in cultured human pancreatic cancer cells.  Cell migration was measured with a modified Boyden chamber method.  Peritoneal metastasis of the pancreatic cancer cells was measured with a mouse model.  All statistical tests were two-sided.  Results: We found that 22 (61%) of 36 pancreatic cancer specimens had higher levels of Grb7 protein than their corresponding normal pancreatic tissue specimens.  Grb7 expression was statistically significantly different between specimens from patients without lymph node metastasis (stage N0; two of the 10 patients) and patients with lymph node metastasis (stages N1 + N2; 20 of the 26 patients) (P = .006).  The Grb7 peptide inhibitor selectively blocked the interaction between Grb7 and focal adhesion kinase and blocked the phosphorylation of Grb7 protein.  In vivo Grb7 peptide inhibitor statistically significantly attenuated cell migration (for control peptide, 87.5 cells migrated, 95% confidence interval [CI] = 82.6 to 92.4 cells; for Grb7 peptide, 5.7 cells migrated, 95% CI = 2.3 to 9.0 cells; P<.001) and peritoneal metastasis of the pancreatic cancer cells in a mouse model, as assessed by the no. of nodules (control = 72.6 nodules, 95% CI = 55.8 to 89.4 nodules; and for Grb7 peptide = 3.2 nodules, 95% CI = 1.6 to 4.8 nodules; P<.001, t test) and their wt. (control = 4.13 g, 95% CI = 3.40 to 4.86 g; Grb7 peptide = 0.19 g, 95% CI = 0.06 to 0.32 g; P<.001, t test).  Conclusions: The Grb7 peptide inhibitor appears to be a promising molecularly targeted therapeutic agent against metastatic pancreatic cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpletsYvkiEs7Vg90H21EOLACvtfcHk0ljaB0OU9t6xEg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XjtlamtL8%253D&md5=8bf8b5624a76b39cde03cd269bb16f27</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1093%2Fjnci%2Fdjj105&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjnci%252Fdjj105%26sid%3Dliteratum%253Aachs%26aulast%3DTanaka%26aufirst%3DS.%26aulast%3DPero%26aufirst%3DS.%2BC.%26aulast%3DTaguchi%26aufirst%3DK.%26aulast%3DShimada%26aufirst%3DM.%26aulast%3DMori%26aufirst%3DM.%26aulast%3DKrag%26aufirst%3DD.%2BN.%26aulast%3DArii%26aufirst%3DS.%26atitle%3DSpecific%2520peptide%2520ligand%2520for%2520Grb7%2520signal%2520transduction%2520protein%2520and%2520pancreatic%2520cancer%2520metastasis%26jtitle%3DJ.%2520Natl.%2520Cancer%2520Inst.%26date%3D2006%26volume%3D98%26spage%3D491%26epage%3D498%26doi%3D10.1093%2Fjnci%2Fdjj105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chan, D. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, V. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chiu, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ngan, H. Y.</span><span> </span><span class="NLM_article-title">Differential functions of growth factor receptor-bound protein 7 (GRB7) and its variant GRB7v in ovarian carcinogenesis</span> <span class="citation_source-journal">Clin. Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">2529</span><span class="NLM_x">–</span> <span class="NLM_lpage">2539</span><span class="refDoi"> DOI: 10.1158/1078-0432.CCR-10-0018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=10.1158%2F1078-0432.CCR-10-0018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=20388850" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=1%3ACAS%3A528%3ADC%252BC3cXltlegtrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2010&pages=2529-2539&author=Y.+Wangauthor=D.+W.+Chanauthor=V.+W.+Liuauthor=P.+Chiuauthor=H.+Y.+Ngan&title=Differential+functions+of+growth+factor+receptor-bound+protein+7+%28GRB7%29+and+its+variant+GRB7v+in+ovarian+carcinogenesis&doi=10.1158%2F1078-0432.CCR-10-0018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Differential Functions of Growth Factor Receptor-Bound Protein 7 (GRB7) and Its Variant GRB7v in Ovarian Carcinogenesis</span></div><div class="casAuthors">Wang, Yajun; Chan, David W.; Liu, Vincent W. S.; Chiu, P. M.; Ngan, Hextan Y. S.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2529-2539</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">PURPOSE: Aberrant overexpression of growth factor receptor-bound protein 7 (GRB7) and its variant GRB7v has been found in numerous human cancers.  The goal of this study was to characterize the functions of GRB7 and GRB7v in the ovarian carcinogenesis and to investigate the differential roles of GRB7 and GRB7v in the modulation of signaling pathways.  Exptl. Design: Quant. reverse transcription-PCR, Western blot, and immunohistochem. analyses were used to evaluate the levels of GRB7 and GRB7v.  The cellular localization, functions, and signaling pathways regulated by GRB7 and GRB7v were investigated by enforced expression of GRB7 and GRB7v.  RESULTS: Quant. reverse transcription-PCR and Western blot analyses showed that GRB7 and GRB7v were frequently upregulated in ovarian cancer samples.  The overexpressed GRB7 (P = 0.009) and GRB7v (P = 0.017) were significantly correlated with high-grade ovarian cancer.  Immunohistochem. anal. on ovarian cancer tissue array confirmed that the upregulated GRB7 was significantly correlated with high-grade ovarian cancer (P = 0.001).  Confocal microscopy anal. showed that GRB7 and GRB7v predominately localized in cytoplasm of ovarian cancer cells, consistent with their roles as signaling adaptors.  Enforced expression of GRB7 promoted cell proliferation, migration, and invasion, whereas GRB7v only increased cell proliferation and anchorage-independent growth ability.  With the treatment of specific kinase inhibitors, we showed that both GRB7 and GRB7v promoted cell proliferation through activating extracellular signal-regulated kinase signaling, whereas GRB7 enhanced cell migration/invasion by activating c-Jun NH2 terminal kinase signaling.  CONCLUSIONS: Our studies implicate that the overexpressed GRB7 and GRB7v are assocd. with high-grade tumors and exert distinct tumorigenic functions through regulating different signaling pathways in ovarian cancer cells.  Clin Cancer Res; 16(9); 2529-39.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdUrMOZ0FjhbVg90H21EOLACvtfcHk0ljaB0OU9t6xEg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXltlegtrs%253D&md5=6d74a2b3a3ce1fafe4ff2c27ed6ba323</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-10-0018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-10-0018%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DY.%26aulast%3DChan%26aufirst%3DD.%2BW.%26aulast%3DLiu%26aufirst%3DV.%2BW.%26aulast%3DChiu%26aufirst%3DP.%26aulast%3DNgan%26aufirst%3DH.%2BY.%26atitle%3DDifferential%2520functions%2520of%2520growth%2520factor%2520receptor-bound%2520protein%25207%2520%2528GRB7%2529%2520and%2520its%2520variant%2520GRB7v%2520in%2520ovarian%2520carcinogenesis%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2010%26volume%3D16%26spage%3D2529%26epage%3D2539%26doi%3D10.1158%2F1078-0432.CCR-10-0018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Slamon, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clark, G. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong, S. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Levin, W. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ullrich, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McGuire, W. L.</span><span> </span><span class="NLM_article-title">Human breast cancer: correlation of relapse and survival with amplification of the HER-2/neu oncogene</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1987</span><span class="NLM_x">, </span> <span class="NLM_volume">235</span><span class="NLM_x">, </span> <span class="NLM_fpage">177</span><span class="NLM_x">–</span> <span class="NLM_lpage">182</span><span class="refDoi"> DOI: 10.1126/science.3798106</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=10.1126%2Fscience.3798106" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=3798106" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=1%3ACAS%3A528%3ADyaL2sXhtVSht7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=235&publication_year=1987&pages=177-182&author=D.+J.+Slamonauthor=G.+M.+Clarkauthor=S.+G.+Wongauthor=W.+J.+Levinauthor=A.+Ullrichauthor=W.+L.+McGuire&title=Human+breast+cancer%3A+correlation+of+relapse+and+survival+with+amplification+of+the+HER-2%2Fneu+oncogene&doi=10.1126%2Fscience.3798106"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Human breast cancer:  correlation of relapse and survival with amplification of the HER-2/neu oncogene</span></div><div class="casAuthors">Slamon, Dennis J.; Clark, Gary M.; Wong, Steven G.; Levin, Wendy J.; Ullrich, Axel; McGuire, William L.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">1987</span>),
    <span class="NLM_cas:volume">235</span>
        (<span class="NLM_cas:issue">4785</span>),
    <span class="NLM_cas:pages">177-82</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    </div><div class="casAbstract">The HER-2/neu oncogene is a member of the erbB-like oncogene family, and is related to, but distinct from, the epidermal growth factor receptor.  This gene has been shown to be amplified in human breast cancer cell lines.  In the current study, alterations of the gene in 189 primary human breast cancers were investigated.  HER-2/neu was found to be amplified from 2- to >20-fold in 30% of the tumors.  Correlation of gene amplification with several disease parameters was evaluated.  Amplification of the HER-2neu gene was a significant predictor of both overall survival and time to relapse in patients with breast cancer.  It retained its significance even when adjustments were made for other known prognostic factors.  Moreover, HER-2/neu amplification had greater prognostic value than most currently used prognostic factors, including hormonal-receptor status, in lymph node-pos. disease.  These data indicate that this gene may play a role in the biol. behavior and(or) pathogenesis of human breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWPpNbN8S_QrVg90H21EOLACvtfcHk0ljKVmiYQ_zpIw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL2sXhtVSht7s%253D&md5=0889410d6e47651de4d8085e9bbd70d9</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1126%2Fscience.3798106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.3798106%26sid%3Dliteratum%253Aachs%26aulast%3DSlamon%26aufirst%3DD.%2BJ.%26aulast%3DClark%26aufirst%3DG.%2BM.%26aulast%3DWong%26aufirst%3DS.%2BG.%26aulast%3DLevin%26aufirst%3DW.%2BJ.%26aulast%3DUllrich%26aufirst%3DA.%26aulast%3DMcGuire%26aufirst%3DW.%2BL.%26atitle%3DHuman%2520breast%2520cancer%253A%2520correlation%2520of%2520relapse%2520and%2520survival%2520with%2520amplification%2520of%2520the%2520HER-2%252Fneu%2520oncogene%26jtitle%3DScience%26date%3D1987%26volume%3D235%26spage%3D177%26epage%3D182%26doi%3D10.1126%2Fscience.3798106" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Pradip, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bouzyk, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dey, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leyland-Jones, B.</span><span> </span><span class="NLM_article-title">Dissecting GRB7-mediated signals for proliferation and migration in HER2 overexpressing breast tumor cells: GTP-ase rules</span> <span class="citation_source-journal">Am. J. Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">173</span><span class="NLM_x">–</span> <span class="NLM_lpage">195</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=23593540" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=1%3ACAS%3A280%3ADC%252BC3srlvFOqtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2013&pages=173-195&author=D.+Pradipauthor=M.+Bouzykauthor=N.+Deyauthor=B.+Leyland-Jones&title=Dissecting+GRB7-mediated+signals+for+proliferation+and+migration+in+HER2+overexpressing+breast+tumor+cells%3A+GTP-ase+rules"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Dissecting GRB7-mediated signals for proliferation and migration in HER2 overexpressing breast tumor cells: GTP-ase rules</span></div><div class="casAuthors">Pradip De; Bouzyk Mark; Dey Nandini; Leyland-Jones Brian</div><div class="citationInfo"><span class="NLM_cas:title">American journal of cancer research</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">173-95</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Amplification of human Her2 and its aberrant signaling in 20-30% of early breast cancer patients is responsible for highly aggressive tumors with poor outcome.  Grb7 is reported to be co-amplified with Her2.  We report a concurrent high expression of mRNA (from FFPE tumor samples; mRNA correlation, Pearson r(2)= 0.806), and high levels of GRB7 protein (immunoblot) in HER2+ breast cancer cell lines.  We demonstrated the signaling mechanism of HER2 and downstream effectors that contributes to proliferation and migration.  Using HER2+ and trastuzumab-resistant breast cancer cell lines, we identified the interaction between GRB7 and HER2 in the control of HER2+ cell proliferation.  Our co-IP data show that GRB7 recruits SHC into the HER2-GRB7 signaling complex.  This complex formation leads to activation of RAS-GTP.  We also observed that following integrin engagement, GRB7 is phosphorylated at tyrosine in a p-FAK (Y397) dependent manner.  This FAK-GRB7 complex leads to downstream activation of RAC1-GTP (responsible for migration) probably through the recruitment of VAV2.  Our CO-IP data demonstrate that GRB7 directly binds with VAV2 following fibronectin engagement in HER2+ cells.  To address whether GRB7 could serve as a pathway specific therapeutic target, we used siRNA to suppress GRB7 expression.  Knockdown of GRB7 expression in the HER2+ breast cancer cell lines decreases RAS activation, cell proliferation, 2D and 3D colony formation and also blocked integrin-mediated RAC1 activation along with integrin-directed cell migration.  These findings dissected the HER2-mediated signaling cascade into (1) HER2+ cell proliferation (HER2-GRB7-SHC-RAS) and (2) HER2+ cell migration (alpha5 beta1/alpha4 beta1-FAK-GRB7-VAV2-RAC1).  Our data clearly demonstrate that a coupling of GRB7 with HER2 is required for the proliferative and migratory signals in HER2+ breast tumor cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQo49zaaj9yMMyail4HZd-ifW6udTcc2ebMdQ2djB-hJ7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3srlvFOqtw%253D%253D&md5=a09635d02e188705bcb24085c4b94c4b</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DPradip%26aufirst%3DD.%26aulast%3DBouzyk%26aufirst%3DM.%26aulast%3DDey%26aufirst%3DN.%26aulast%3DLeyland-Jones%26aufirst%3DB.%26atitle%3DDissecting%2520GRB7-mediated%2520signals%2520for%2520proliferation%2520and%2520migration%2520in%2520HER2%2520overexpressing%2520breast%2520tumor%2520cells%253A%2520GTP-ase%2520rules%26jtitle%3DAm.%2520J.%2520Cancer%2520Res.%26date%3D2013%26volume%3D3%26spage%3D173%26epage%3D195" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Han, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guan, J. L.</span><span> </span><span class="NLM_article-title">Association of focal adhesion kinase with Grb7 and its role in cell migration</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">274</span><span class="NLM_x">, </span> <span class="NLM_fpage">24425</span><span class="NLM_x">–</span> <span class="NLM_lpage">24430</span><span class="refDoi"> DOI: 10.1074/jbc.274.34.24425</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=10.1074%2Fjbc.274.34.24425" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=274&publication_year=1999&pages=24425-24430&author=D.+C.+Hanauthor=J.+L.+Guan&title=Association+of+focal+adhesion+kinase+with+Grb7+and+its+role+in+cell+migration&doi=10.1074%2Fjbc.274.34.24425"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1074%2Fjbc.274.34.24425&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.274.34.24425%26sid%3Dliteratum%253Aachs%26aulast%3DHan%26aufirst%3DD.%2BC.%26aulast%3DGuan%26aufirst%3DJ.%2BL.%26atitle%3DAssociation%2520of%2520focal%2520adhesion%2520kinase%2520with%2520Grb7%2520and%2520its%2520role%2520in%2520cell%2520migration%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1999%26volume%3D274%26spage%3D24425%26epage%3D24430%26doi%3D10.1074%2Fjbc.274.34.24425" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Ramsey, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bai, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hanlon Newell, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Troxell, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Park, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olson, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keenan, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luoh, S. W.</span><span> </span><span class="NLM_article-title">GRB7 protein over-expression and clinical outcome in breast cancer</span> <span class="citation_source-journal">Breast Cancer Res. Treat.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">127</span><span class="NLM_x">, </span> <span class="NLM_fpage">659</span><span class="NLM_x">–</span> <span class="NLM_lpage">669</span><span class="refDoi"> DOI: 10.1007/s10549-010-1010-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=10.1007%2Fs10549-010-1010-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=20635137" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmtlyku7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=127&publication_year=2011&pages=659-669&author=B.+Ramseyauthor=T.+Baiauthor=A.+Hanlon+Newellauthor=M.+Troxellauthor=B.+Parkauthor=S.+Olsonauthor=E.+Keenanauthor=S.+W.+Luoh&title=GRB7+protein+over-expression+and+clinical+outcome+in+breast+cancer&doi=10.1007%2Fs10549-010-1010-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">GRB7 protein over-expression and clinical outcome in breast cancer</span></div><div class="casAuthors">Ramsey, Betsy; Bai, Tao; Newell, Amy Hanlon; Troxell, Megan; Park, Byung; Olson, Susan; Keenan, Edward; Luoh, Shiuh-Wen</div><div class="citationInfo"><span class="NLM_cas:title">Breast Cancer Research and Treatment</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">127</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">659-669</span>CODEN:
                <span class="NLM_cas:coden">BCTRD6</span>;
        ISSN:<span class="NLM_cas:issn">0167-6806</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">The growth factor receptor-bound protein-7 gene (GRB7) encodes a multi-domain signal transduction mol.  The purpose of this study was to examine the clin. significance of GRB7 protein expression in human breast cancer.  Western blotting anal. of protein exts. from 563 annotated frozen breast tumors was performed.  Expression status of GRB7 and HER-2 was correlated with clin. covariates and outcomes.  Cox proportional hazards were used to identify factors assocd. with breast cancer-free interval.  The median follow-up was 71 mo.  P values <0.05 were considered statistically significant (two-sided).  A discrepancy between HER-2 and GRB7 protein over-expression was obsd.  GRB7 protein over-expression was assocd. with neg. estrogen and progesterone receptor status, higher tumor grade, larger primary tumor size, (more) axillary lymph node involvement, higher clin. stage, and shortened breast cancer-free interval.  HER-2 protein over-expression was assocd. only with higher tumor grade.  Multi-variate anal. revealed that GRB7 protein over-expression was an independent adverse prognostic factor for breast cancer-free interval (hazard ratio 1.69, 95% confidence interval 1.07-2.67; P = 0.024).  The same was true of the subset of patients who did not receive any adjuvant systemic therapy (hazard ratio 1.68, 95% confidence interval 1.16-2.31; P = 0.0055).  Using FISH anal., 32/32 (100%; 95% CI 89-100%) tumors which over-expressed both HER-2 and GRB7 proteins and 1/35 (3%; 95% CI 0-15%) tumors with HER-2 but no GRB7 protein over-expression with Western blotting anal. demonstrated HER-2 gene amplification.  GRB7 protein over-expression is an independent adverse prognostic factor in human breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrChZB-XTObp7Vg90H21EOLACvtfcHk0ljKVmiYQ_zpIw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmtlyku7Y%253D&md5=600c4cb4b85ba9646ee9a9ee2e21864c</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1007%2Fs10549-010-1010-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10549-010-1010-0%26sid%3Dliteratum%253Aachs%26aulast%3DRamsey%26aufirst%3DB.%26aulast%3DBai%26aufirst%3DT.%26aulast%3DHanlon%2BNewell%26aufirst%3DA.%26aulast%3DTroxell%26aufirst%3DM.%26aulast%3DPark%26aufirst%3DB.%26aulast%3DOlson%26aufirst%3DS.%26aulast%3DKeenan%26aufirst%3DE.%26aulast%3DLuoh%26aufirst%3DS.%2BW.%26atitle%3DGRB7%2520protein%2520over-expression%2520and%2520clinical%2520outcome%2520in%2520breast%2520cancer%26jtitle%3DBreast%2520Cancer%2520Res.%2520Treat.%26date%3D2011%26volume%3D127%26spage%3D659%26epage%3D669%26doi%3D10.1007%2Fs10549-010-1010-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Chu, P. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, T. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ding, S. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lai, I. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, T. L.</span><span> </span><span class="NLM_article-title">EGF-induced Grb7 recruits and promotes Ras activity essential for the tumorigenicity of Sk-Br3 breast cancer cells</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">285</span><span class="NLM_x">, </span> <span class="NLM_fpage">29279</span><span class="NLM_x">–</span> <span class="NLM_lpage">29285</span><span class="refDoi"> DOI: 10.1074/jbc.C110.114124</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=10.1074%2Fjbc.C110.114124" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=285&publication_year=2010&pages=29279-29285&author=P.+Y.+Chuauthor=T.+K.+Liauthor=S.+T.+Dingauthor=I.+R.+Laiauthor=T.+L.+Shen&title=EGF-induced+Grb7+recruits+and+promotes+Ras+activity+essential+for+the+tumorigenicity+of+Sk-Br3+breast+cancer+cells&doi=10.1074%2Fjbc.C110.114124"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1074%2Fjbc.C110.114124&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.C110.114124%26sid%3Dliteratum%253Aachs%26aulast%3DChu%26aufirst%3DP.%2BY.%26aulast%3DLi%26aufirst%3DT.%2BK.%26aulast%3DDing%26aufirst%3DS.%2BT.%26aulast%3DLai%26aufirst%3DI.%2BR.%26aulast%3DShen%26aufirst%3DT.%2BL.%26atitle%3DEGF-induced%2520Grb7%2520recruits%2520and%2520promotes%2520Ras%2520activity%2520essential%2520for%2520the%2520tumorigenicity%2520of%2520Sk-Br3%2520breast%2520cancer%2520cells%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2010%26volume%3D285%26spage%3D29279%26epage%3D29285%26doi%3D10.1074%2Fjbc.C110.114124" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Giricz, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Calvo, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pero, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krag, D. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sparano, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kenny, P. A.</span><span> </span><span class="NLM_article-title">GRB7 is required for triple-negative breast cancer cell invasion and survival</span> <span class="citation_source-journal">Breast Cancer Res. Treat.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">133</span><span class="NLM_x">, </span> <span class="NLM_fpage">607</span><span class="NLM_x">–</span> <span class="NLM_lpage">615</span><span class="refDoi"> DOI: 10.1007/s10549-011-1822-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=10.1007%2Fs10549-011-1822-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=22005836" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=1%3ACAS%3A528%3ADC%252BC38XnvFOksbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=133&publication_year=2012&pages=607-615&author=O.+Giriczauthor=V.+Calvoauthor=S.+C.+Peroauthor=D.+N.+Kragauthor=J.+A.+Sparanoauthor=P.+A.+Kenny&title=GRB7+is+required+for+triple-negative+breast+cancer+cell+invasion+and+survival&doi=10.1007%2Fs10549-011-1822-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">GRB7 is required for triple-negative breast cancer cell invasion and survival</span></div><div class="casAuthors">Giricz, Orsi; Calvo, Veronica; Pero, Stephanie C.; Krag, David N.; Sparano, Joseph A.; Kenny, Paraic A.</div><div class="citationInfo"><span class="NLM_cas:title">Breast Cancer Research and Treatment</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">133</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">607-615</span>CODEN:
                <span class="NLM_cas:coden">BCTRD6</span>;
        ISSN:<span class="NLM_cas:issn">0167-6806</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Triple-neg. breast cancer (TNBC) is a heterogeneous disease that is usually assocd. with poor prognosis, and frequently assocd. with the basal-like breast cancer gene expression profile.  There are no targeted therapeutic modalities for this disease, and no useful biomarkers.  High GRB7 RNA expression levels are assocd. with an elevated risk of recurrence in patients with operable TNBC treated with std. adjuvant anthracycline and taxane therapy.  To det. whether GRB7 is involved in the pathobiol. of TNBC, we evaluated the biol. effects of GRB7 inhibition in a panel of triple-neg. cell lines-MDA-MB-468, MDA-MB-231, HCC70, and T4-2.  We found GRB7 inhibition reduced cell motility and invasion of these cell lines and promoted cell death by apoptosis in 3D culture.  These data suggest that GRB7 itself, or GRB7-dependent pathways, may prove to be important therapeutic targets in this disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreB0Epb_bbMLVg90H21EOLACvtfcHk0ljoxLp32MvWWg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XnvFOksbc%253D&md5=8ad42003c4e5536023f81377e6f1bbc1</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1007%2Fs10549-011-1822-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10549-011-1822-6%26sid%3Dliteratum%253Aachs%26aulast%3DGiricz%26aufirst%3DO.%26aulast%3DCalvo%26aufirst%3DV.%26aulast%3DPero%26aufirst%3DS.%2BC.%26aulast%3DKrag%26aufirst%3DD.%2BN.%26aulast%3DSparano%26aufirst%3DJ.%2BA.%26aulast%3DKenny%26aufirst%3DP.%2BA.%26atitle%3DGRB7%2520is%2520required%2520for%2520triple-negative%2520breast%2520cancer%2520cell%2520invasion%2520and%2520survival%26jtitle%3DBreast%2520Cancer%2520Res.%2520Treat.%26date%3D2012%26volume%3D133%26spage%3D607%26epage%3D615%26doi%3D10.1007%2Fs10549-011-1822-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Lim, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Price, J. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilce, J. A.</span><span> </span><span class="NLM_article-title">Context-dependent role of Grb7 in HER2+ve and triple-negative breast cancer cell lines</span> <span class="citation_source-journal">Breast Cancer Res. Treat.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">143</span><span class="NLM_x">, </span> <span class="NLM_fpage">593</span><span class="NLM_x">–</span> <span class="NLM_lpage">603</span><span class="refDoi"> DOI: 10.1007/s10549-014-2838-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=10.1007%2Fs10549-014-2838-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=24464577" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtlSqsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=143&publication_year=2014&pages=593-603&author=R.+C.+Limauthor=J.+T.+Priceauthor=J.+A.+Wilce&title=Context-dependent+role+of+Grb7+in+HER2%2Bve+and+triple-negative+breast+cancer+cell+lines&doi=10.1007%2Fs10549-014-2838-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Context-dependent role of Grb7 in HER2+ve and triple-negative breast cancer cell lines</span></div><div class="casAuthors">Lim, Reece C. C.; Price, John T.; Wilce, Jacqueline A.</div><div class="citationInfo"><span class="NLM_cas:title">Breast Cancer Research and Treatment</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">143</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">593-603</span>CODEN:
                <span class="NLM_cas:coden">BCTRD6</span>;
        ISSN:<span class="NLM_cas:issn">0167-6806</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Grb7 is an adapter protein, aberrantly co-overexpressed with HER2 and identified as an independent prognostic marker in breast cancer.  It has been established that Grb7 exacerbates the cellular growth and migratory behavior of HER2+ve breast cancer cells.  Less is known about Grb7's role in the context of HER2-ve cells.  Here we directly compare the effect of stable Grb7 knockdown in estrogen sensitive (T47D), HER2+ve (SKBR3) and triple-neg. (MDA-MB-468 and MDA-MB-231) breast cancer cell lines on anchorage dependent and independent cell growth, wound healing and chemotaxis.  All cell lines showed reduced ability to migrate upon Grb7 knockdown, despite their greatly varied endogenous levels of Grb7.  Decreased cell proliferation was not obsd. in any of the cell lines upon Grb7 knockdown; however, decreased ability to form colonies was obsd. for all but the estrogen sensitive cell line, depending upon the stringency of the growth conditions.  The data reveal that Grb7 plays an important role in breast cancer progression, beyond the context of HER2+ve cell types.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4vHleefRXL7Vg90H21EOLACvtfcHk0ljoxLp32MvWWg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtlSqsr8%253D&md5=121ce4f7e3fe1fa16c790a63bdf6dce4</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1007%2Fs10549-014-2838-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10549-014-2838-5%26sid%3Dliteratum%253Aachs%26aulast%3DLim%26aufirst%3DR.%2BC.%26aulast%3DPrice%26aufirst%3DJ.%2BT.%26aulast%3DWilce%26aufirst%3DJ.%2BA.%26atitle%3DContext-dependent%2520role%2520of%2520Grb7%2520in%2520HER2%252Bve%2520and%2520triple-negative%2520breast%2520cancer%2520cell%2520lines%26jtitle%3DBreast%2520Cancer%2520Res.%2520Treat.%26date%3D2014%26volume%3D143%26spage%3D593%26epage%3D603%26doi%3D10.1007%2Fs10549-014-2838-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Pero, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oligino, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Daly, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soden, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roller, P. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krag, D. N.</span><span> </span><span class="NLM_article-title">Identification of novel non-phosphorylated ligands, which bind selectively to the SH2 domain of Grb7</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">277</span><span class="NLM_x">, </span> <span class="NLM_fpage">11918</span><span class="NLM_x">–</span> <span class="NLM_lpage">11926</span><span class="refDoi"> DOI: 10.1074/jbc.M111816200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=10.1074%2Fjbc.M111816200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=277&publication_year=2002&pages=11918-11926&author=S.+C.+Peroauthor=L.+Oliginoauthor=R.+J.+Dalyauthor=A.+L.+Sodenauthor=C.+Liuauthor=P.+P.+Rollerauthor=P.+Liauthor=D.+N.+Krag&title=Identification+of+novel+non-phosphorylated+ligands%2C+which+bind+selectively+to+the+SH2+domain+of+Grb7&doi=10.1074%2Fjbc.M111816200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M111816200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M111816200%26sid%3Dliteratum%253Aachs%26aulast%3DPero%26aufirst%3DS.%2BC.%26aulast%3DOligino%26aufirst%3DL.%26aulast%3DDaly%26aufirst%3DR.%2BJ.%26aulast%3DSoden%26aufirst%3DA.%2BL.%26aulast%3DLiu%26aufirst%3DC.%26aulast%3DRoller%26aufirst%3DP.%2BP.%26aulast%3DLi%26aufirst%3DP.%26aulast%3DKrag%26aufirst%3DD.%2BN.%26atitle%3DIdentification%2520of%2520novel%2520non-phosphorylated%2520ligands%252C%2520which%2520bind%2520selectively%2520to%2520the%2520SH2%2520domain%2520of%2520Grb7%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2002%26volume%3D277%26spage%3D11918%26epage%3D11926%26doi%3D10.1074%2Fjbc.M111816200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Watson, G. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gunzburg, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ambaye, N. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lucas, W. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Traore, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kulkarni, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cergol, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Payne, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Panjikar, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pero, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perlmutter, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilce, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilce, J. A.</span><span> </span><span class="NLM_article-title">Cyclic peptides incorporating phosphotyrosine mimetics as potent and specific inhibitors of the Grb7 breast cancer target</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">7707</span><span class="NLM_x">–</span> <span class="NLM_lpage">7718</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00609</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00609" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVyktrrF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=7707-7718&author=G.+M.+Watsonauthor=M.+J.+Gunzburgauthor=N.+D.+Ambayeauthor=W.+A.+Lucasauthor=D.+A.+Traoreauthor=K.+Kulkarniauthor=K.+M.+Cergolauthor=R.+J.+Payneauthor=S.+Panjikarauthor=S.+C.+Peroauthor=P.+Perlmutterauthor=M.+C.+Wilceauthor=J.+A.+Wilce&title=Cyclic+peptides+incorporating+phosphotyrosine+mimetics+as+potent+and+specific+inhibitors+of+the+Grb7+breast+cancer+target&doi=10.1021%2Facs.jmedchem.5b00609"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Cyclic Peptides Incorporating Phosphotyrosine Mimetics as Potent and Specific Inhibitors of the Grb7 Breast Cancer Target</span></div><div class="casAuthors">Watson, Gabrielle M.; Gunzburg, Menachem J.; Ambaye, Nigus D.; Lucas, William A. H.; Traore, Daouda A.; Kulkarni, Ketav; Cergol, Katie M.; Payne, Richard J.; Panjikar, Santosh; Pero, Stephanie C.; Perlmutter, Patrick; Wilce, Matthew C. J.; Wilce, Jacqueline A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">7707-7718</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The Grb7 adaptor protein is a therapeutic target for both TNBC and HER2+ breast cancer.  A nonphosphorylated cyclic peptide 1 (known as G7-18NATE) inhibits Grb7 via targeting the Grb7-SH2 domain, but requires the presence of phosphate ions for both affinity and specificity.  Here, the authors report the discovery of malonate bound in the phosphotyrosine binding pocket of the apo-Grb7-SH2 structure.  Based on this, the authors carried out the rational design and synthesis of two analogs of peptide 1 that incorporate carboxymethylphenylalanine (cmF) and carboxyphenylalanine (cF) as mimics of phosphotyrosine (pY).  Binding studies using SPR confirmed that affinity for Grb7-SH2 domain is improved up to 9-fold over peptide 1 under physiol. phosphate conditions (KD = 2.1-5.7 μM) and that binding is specific for Grb7-SH2 over closely related domains (low or no detectable binding to Grb2-SH2 and Grb10-SH2).  X-ray crystallog. structural anal. of the analog bearing a cmF moiety in complex with Grb7-SH2 has identified the precise contacts conferred by the pY mimic that underpin this improved mol. interaction.  This study identifies and characterizes the tightest specific inhibitor of Grb7 to date, representing a significant development toward a new Grb7-targeted therapeutic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBV3XGn3QVZLVg90H21EOLACvtfcHk0lii2HSFwldtJg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVyktrrF&md5=c28d4bbd140c6eced3e6256964461612</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00609&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00609%26sid%3Dliteratum%253Aachs%26aulast%3DWatson%26aufirst%3DG.%2BM.%26aulast%3DGunzburg%26aufirst%3DM.%2BJ.%26aulast%3DAmbaye%26aufirst%3DN.%2BD.%26aulast%3DLucas%26aufirst%3DW.%2BA.%26aulast%3DTraore%26aufirst%3DD.%2BA.%26aulast%3DKulkarni%26aufirst%3DK.%26aulast%3DCergol%26aufirst%3DK.%2BM.%26aulast%3DPayne%26aufirst%3DR.%2BJ.%26aulast%3DPanjikar%26aufirst%3DS.%26aulast%3DPero%26aufirst%3DS.%2BC.%26aulast%3DPerlmutter%26aufirst%3DP.%26aulast%3DWilce%26aufirst%3DM.%2BC.%26aulast%3DWilce%26aufirst%3DJ.%2BA.%26atitle%3DCyclic%2520peptides%2520incorporating%2520phosphotyrosine%2520mimetics%2520as%2520potent%2520and%2520specific%2520inhibitors%2520of%2520the%2520Grb7%2520breast%2520cancer%2520target%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D7707%26epage%3D7718%26doi%3D10.1021%2Facs.jmedchem.5b00609" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Gunzburg, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kulkarni, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Watson, G. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ambaye, N. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Del Borgo, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brandt, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pero, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perlmutter, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilce, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilce, J. A.</span><span> </span><span class="NLM_article-title">Unexpected involvement of staple leads to redesign of selective bicyclic peptide inhibitor of Grb7</span> <span class="citation_source-journal">Sci. Rep.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">27060</span><span class="refDoi"> DOI: 10.1038/srep27060</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=10.1038%2Fsrep27060" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=27257138" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=1%3ACAS%3A528%3ADC%252BC28XpsFCgsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2016&pages=27060&author=M.+J.+Gunzburgauthor=K.+Kulkarniauthor=G.+M.+Watsonauthor=N.+D.+Ambayeauthor=M.+P.+Del+Borgoauthor=R.+Brandtauthor=S.+C.+Peroauthor=P.+Perlmutterauthor=M.+C.+Wilceauthor=J.+A.+Wilce&title=Unexpected+involvement+of+staple+leads+to+redesign+of+selective+bicyclic+peptide+inhibitor+of+Grb7&doi=10.1038%2Fsrep27060"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Unexpected involvement of staple leads to redesign of selective bicyclic peptide inhibitor of Grb7</span></div><div class="casAuthors">Gunzburg, Menachem J.; Kulkarni, Ketav; Watson, Gabrielle M.; Ambaye, Nigus D.; Del Borgo, Mark P.; Brandt, Rebecca; Pero, Stephanie C.; Perlmutter, Patrick; Wilce, Matthew C. J.; Wilce, Jacqueline A.</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">27060</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The design of potent and specific peptide inhibitors to therapeutic targets is of enormous utility for both proof-of-concept studies and for the development of potential new therapeutics.  Grb7 is a key signaling mol. in the progression of HER2 pos. and triple neg. breast cancers.  Here we report the crystal structure of a stapled bicyclic peptide inhibitor G7-B1 in complex with the Grb7-SH2 domain.  This revealed an unexpected binding mode of the peptide, in which the staple forms an alternative contact with the surface of the target protein.  Based on this structural information, we designed a new series of bicyclic G7 peptides that progressively constrain the starting peptide, to arrive at the G7-B4 peptide that binds with an approx. 2-fold enhanced affinity to the Grb7-SH2 domain (KD = 0.83 μM) compared to G7-B1 and shows low affinity binding to Grb2-, Grb10- and Grb14-SH2 domains (KD > 100 μM).  Furthermore, we detd. the structure of the G7-B4 bicyclic peptide in complex with the Grb7-SH2 domain, both before and after ring closing metathesis to show that the closed staple is essential to the target interaction.  The G7-B4 peptide represents an advance in the development of Grb7 inhibitors and is a classical example of structure aided inhibitor development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGLvbEuvyHLrVg90H21EOLACvtfcHk0lii2HSFwldtJg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XpsFCgsLo%253D&md5=141b7b4db6aa6178e9449e628947d706</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1038%2Fsrep27060&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep27060%26sid%3Dliteratum%253Aachs%26aulast%3DGunzburg%26aufirst%3DM.%2BJ.%26aulast%3DKulkarni%26aufirst%3DK.%26aulast%3DWatson%26aufirst%3DG.%2BM.%26aulast%3DAmbaye%26aufirst%3DN.%2BD.%26aulast%3DDel%2BBorgo%26aufirst%3DM.%2BP.%26aulast%3DBrandt%26aufirst%3DR.%26aulast%3DPero%26aufirst%3DS.%2BC.%26aulast%3DPerlmutter%26aufirst%3DP.%26aulast%3DWilce%26aufirst%3DM.%2BC.%26aulast%3DWilce%26aufirst%3DJ.%2BA.%26atitle%3DUnexpected%2520involvement%2520of%2520staple%2520leads%2520to%2520redesign%2520of%2520selective%2520bicyclic%2520peptide%2520inhibitor%2520of%2520Grb7%26jtitle%3DSci.%2520Rep.%26date%3D2016%26volume%3D6%26spage%3D27060%26doi%3D10.1038%2Fsrep27060" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Pero, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shukla, G. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cookson, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flemer, S.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krag, D. N.</span><span> </span><span class="NLM_article-title">Combination treatment with Grb7 peptide and Doxorubicin or Trastuzumab (Herceptin) results in cooperative cell growth inhibition in breast cancer cells</span> <span class="citation_source-journal">Br. J. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">96</span><span class="NLM_x">, </span> <span class="NLM_fpage">1520</span><span class="NLM_x">–</span> <span class="NLM_lpage">1525</span><span class="refDoi"> DOI: 10.1038/sj.bjc.6603732</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=10.1038%2Fsj.bjc.6603732" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=17426702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=1%3ACAS%3A528%3ADC%252BD2sXlt1KmtLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=96&publication_year=2007&pages=1520-1525&author=S.+C.+Peroauthor=G.+S.+Shuklaauthor=M.+M.+Cooksonauthor=S.+Flemerauthor=D.+N.+Krag&title=Combination+treatment+with+Grb7+peptide+and+Doxorubicin+or+Trastuzumab+%28Herceptin%29+results+in+cooperative+cell+growth+inhibition+in+breast+cancer+cells&doi=10.1038%2Fsj.bjc.6603732"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Combination treatment with Grb7 peptide and Doxorubicin or Trastuzumab (Herceptin) results in cooperative cell growth inhibition in breast cancer cells</span></div><div class="casAuthors">Pero, S. C.; Shukla, G. S.; Cookson, M. M.; Flemer, S., Jr.; Krag, D. N.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Cancer</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">96</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1520-1525</span>CODEN:
                <span class="NLM_cas:coden">BJCAAI</span>;
        ISSN:<span class="NLM_cas:issn">0007-0920</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Grb7 has potential importance in the progression of cancer.  We have previously identified a novel peptide that binds to the SH2 domain of Grb7 and inhibits its assocn. with several different receptor tyrosine kinases.  We have synthesized the Grb7 peptide, G7-18NATE, with two different cell penetrating peptides, Penetratin and Tat.  In this study, we have shown that both Penetratin- and Tat-conjugated G7-18NATE peptides are able to inhibit the proliferation of SK-BR-3, ZR-75-30, MDA-MB-361 and MDA-MB-231 breast cancer cells.  There was no significant effects on breast cancer MCF-7cells, non-malignant MCF 10A or 3T3 cells.  In addn., there was no significant inhibition of proliferation by Penetratin or Tat alone or by their conjugates with arbitrary peptide sequence in any of the cell lines tested.  We detd. the EC50 of G7-18NATE-P peptide for SK-BR-3 cell proliferation to be 7.663×10-6 M.  Co-treatment of G7-18NATE-P peptide plus Doxorubicin in SK-BR-3 breast cancer cells resulted in an addnl. inhibition of proliferation, resulting in 56 and 84% decreases in the Doxorubicin EC50 value in the presence of 5×10-6 and 1.0×10-5 M G7-18NATE-P peptide, resp.  Importantly, the co-treatment with Doxorubicin and the delivery peptide did not change the Doxorubicin EC50.  Since Grb7 assocs. with ErbB2, we assessed whether the peptide inhibitor would have a combined effect with a mol. that targets ErbB2, Herceptin.  Co-treatment of Herceptin plus 1.0×10-5 M G7-18NATE-P peptide in SK-BR-3 cells resulted in a 46% decrease in the Herceptin EC50 value and no decrease following the co-treatment with Herceptin and penetratin alone.  This Grb7 peptide has potential to be developed as a therapeutic agent alone, in combination with traditional chemotherapy, or in combination with other targeting mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrtYqBV3GdCebVg90H21EOLACvtfcHk0lii2HSFwldtJg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXlt1KmtLc%253D&md5=ef6c823762d834e077e70657ea28162d</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1038%2Fsj.bjc.6603732&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.bjc.6603732%26sid%3Dliteratum%253Aachs%26aulast%3DPero%26aufirst%3DS.%2BC.%26aulast%3DShukla%26aufirst%3DG.%2BS.%26aulast%3DCookson%26aufirst%3DM.%2BM.%26aulast%3DFlemer%26aufirst%3DS.%26aulast%3DKrag%26aufirst%3DD.%2BN.%26atitle%3DCombination%2520treatment%2520with%2520Grb7%2520peptide%2520and%2520Doxorubicin%2520or%2520Trastuzumab%2520%2528Herceptin%2529%2520results%2520in%2520cooperative%2520cell%2520growth%2520inhibition%2520in%2520breast%2520cancer%2520cells%26jtitle%3DBr.%2520J.%2520Cancer%26date%3D2007%26volume%3D96%26spage%3D1520%26epage%3D1525%26doi%3D10.1038%2Fsj.bjc.6603732" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Gunzburg, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ambaye, N. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Del Borgo, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pero, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krag, D. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilce, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilce, J. A.</span><span> </span><span class="NLM_article-title">Interaction of the non-phosphorylated peptide G7–18NATE with Grb7-SH2 domain requires phosphate for enhanced affinity and specificity</span> <span class="citation_source-journal">J. Mol. Recognit.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">57</span><span class="NLM_x">–</span> <span class="NLM_lpage">67</span><span class="refDoi"> DOI: 10.1002/jmr.2148</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=10.1002%2Fjmr.2148" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=22213451" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs1yntrzK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2012&pages=57-67&author=M.+J.+Gunzburgauthor=N.+D.+Ambayeauthor=M.+P.+Del+Borgoauthor=S.+C.+Peroauthor=D.+N.+Kragauthor=M.+C.+Wilceauthor=J.+A.+Wilce&title=Interaction+of+the+non-phosphorylated+peptide+G7%E2%80%9318NATE+with+Grb7-SH2+domain+requires+phosphate+for+enhanced+affinity+and+specificity&doi=10.1002%2Fjmr.2148"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Interaction of the non-phosphorylated peptide G7-18NATE with Grb7-SH2 domain requires phosphate for enhanced affinity and specificity</span></div><div class="casAuthors">Gunzburg, Menachem J.; Ambaye, Nigus D.; Del Borgo, Mark P.; Pero, Stephanie C.; Krag, David N.; Wilce, Matthew C. J.; Wilce, Jacqueline A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Recognition</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">57-67</span>CODEN:
                <span class="NLM_cas:coden">JMORE4</span>;
        ISSN:<span class="NLM_cas:issn">0952-3499</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Src-homol. (SH2) domains are an attractive target for the inhibition of specific signaling pathways but pose the challenge of developing a truly specific inhibitor.  The G7-18NATE cyclic peptide is reported to specifically inhibit the growth factor receptor bound protein 7 (Grb7) adapter protein, implicated in the progression of several cancer types, via interactions with its SH2 domain.  G7-18NATE effectively inhibits the interaction of Grb7 with ErbB3 and focal adhesion kinase in cell lysates and, with the addn. of a cell permeability sequence, inhibits the growth and migration of a no. of breast cancer cell lines.  It is thus a promising lead in the development of therapeutics targeted to Grb7.  Here the authors investigate the degree to which G7-18NATE is specific for the Grb7-SH2 domain compared with closely related SH2 domains including those of Grb10, Grb14, and Grb2 using surface plasmon resonance.  The authors demonstrate that G7-18NATE binds with micromolar binding affinity to Grb7-SH2 domain (KD = 4-6 μm) compared with 50-200 times lower affinity for Grb10, Grb14, and Grb2 but that this specificity depends critically on the presence of phosphate in millimolar concns.  Other differences in buffer compn., including use of Tris or 2-(N-Morpholino)ethanesulfonic acid or varying the pH, do not impact on the interaction.  This suggests that under cellular conditions, G7-18NATE binds with highest affinity to Grb7.  In addn., the authors' findings demonstrate that the basis of specificity of G7-18NATE binding to the Grb7-SH2 domain is via other than intrinsic structural features of the protein, representing an unexpected mode of mol. recognition.  Copyright © 2011 John Wiley & Sons, Ltd.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVOz6dLoaKNLVg90H21EOLACvtfcHk0ljKeIt2eLATrw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs1yntrzK&md5=77073aec5afc99cfbf1f3b7d62039b8f</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1002%2Fjmr.2148&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjmr.2148%26sid%3Dliteratum%253Aachs%26aulast%3DGunzburg%26aufirst%3DM.%2BJ.%26aulast%3DAmbaye%26aufirst%3DN.%2BD.%26aulast%3DDel%2BBorgo%26aufirst%3DM.%2BP.%26aulast%3DPero%26aufirst%3DS.%2BC.%26aulast%3DKrag%26aufirst%3DD.%2BN.%26aulast%3DWilce%26aufirst%3DM.%2BC.%26aulast%3DWilce%26aufirst%3DJ.%2BA.%26atitle%3DInteraction%2520of%2520the%2520non-phosphorylated%2520peptide%2520G7%25E2%2580%259318NATE%2520with%2520Grb7-SH2%2520domain%2520requires%2520phosphate%2520for%2520enhanced%2520affinity%2520and%2520specificity%26jtitle%3DJ.%2520Mol.%2520Recognit.%26date%3D2012%26volume%3D25%26spage%3D57%26epage%3D67%26doi%3D10.1002%2Fjmr.2148" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Eck, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoelson, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harrison, S. C.</span><span> </span><span class="NLM_article-title">Recognition of a high-affinity phosphotyrosyl peptide by the Src homology-2 domain of p56lck</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1993</span><span class="NLM_x">, </span> <span class="NLM_volume">362</span><span class="NLM_x">, </span> <span class="NLM_fpage">87</span><span class="NLM_x">–</span> <span class="NLM_lpage">91</span><span class="refDoi"> DOI: 10.1038/362087a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=10.1038%2F362087a0" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=362&publication_year=1993&pages=87-91&author=M.+J.+Eckauthor=S.+E.+Shoelsonauthor=S.+C.+Harrison&title=Recognition+of+a+high-affinity+phosphotyrosyl+peptide+by+the+Src+homology-2+domain+of+p56lck&doi=10.1038%2F362087a0"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2F362087a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F362087a0%26sid%3Dliteratum%253Aachs%26aulast%3DEck%26aufirst%3DM.%2BJ.%26aulast%3DShoelson%26aufirst%3DS.%2BE.%26aulast%3DHarrison%26aufirst%3DS.%2BC.%26atitle%3DRecognition%2520of%2520a%2520high-affinity%2520phosphotyrosyl%2520peptide%2520by%2520the%2520Src%2520homology-2%2520domain%2520of%2520p56lck%26jtitle%3DNature%26date%3D1993%26volume%3D362%26spage%3D87%26epage%3D91%26doi%3D10.1038%2F362087a0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Bevington, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mundy, K. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yates, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kanis, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Russell, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rajagopalan, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Radda, G. K.</span><span> </span><span class="NLM_article-title">A study of intracellular orthophosphate concentration in human muscle and erythrocytes by 31P nuclear magnetic resonance spectroscopy and selective chemical assay</span> <span class="citation_source-journal">Clin. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1986</span><span class="NLM_x">, </span> <span class="NLM_volume">71</span><span class="NLM_x">, </span> <span class="NLM_fpage">729</span><span class="NLM_x">–</span> <span class="NLM_lpage">735</span><span class="refDoi"> DOI: 10.1042/cs0710729</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=10.1042%2Fcs0710729" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=3024899" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=1%3ACAS%3A528%3ADyaL2sXkvVWnsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=1986&pages=729-735&author=A.+Bevingtonauthor=K.+I.+Mundyauthor=A.+J.+Yatesauthor=J.+A.+Kanisauthor=R.+G.+Russellauthor=D.+J.+Taylorauthor=B.+Rajagopalanauthor=G.+K.+Radda&title=A+study+of+intracellular+orthophosphate+concentration+in+human+muscle+and+erythrocytes+by+31P+nuclear+magnetic+resonance+spectroscopy+and+selective+chemical+assay&doi=10.1042%2Fcs0710729"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">A study of intracellular orthophosphate concentration in human muscle and erythrocytes by phosphorus-31 nuclear magnetic resonance spectroscopy and selective chemical assay</span></div><div class="casAuthors">Bevington, A.; Mundy, K. I.; Yates, A. J. P.; Kanis, J. A.; Russell, R. G. G.; Taylor, D. J.; Rajagopalan, B.; Radda, G. K.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Science</span>
        (<span class="NLM_cas:date">1986</span>),
    <span class="NLM_cas:volume">71</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">729-35</span>CODEN:
                <span class="NLM_cas:coden">CSCIAE</span>;
        ISSN:<span class="NLM_cas:issn">0143-5221</span>.
    </div><div class="casAbstract">To study the relation between extracellular and intracellular concns. of orthophosphate (Pi), phosphorus NMR spectra were recorded, at rest, from the flexor digitorum superficialis muscle of hypophosphatemic patients with vitamin D-resistant rickets, and patients with Paget's disease of bone before and after they had been made hyperphosphatemic by treatment with ethylidene-1-hydroxy-1,1-bisphosphonate.  Changes in i.m. Pi were estd. from the ratio of the areas of the Pi to ATP peaks.  Even though the plasma Pi concn. in these patients spanned a 4-fold range (0.5-2.0 mmol/L) the corresponding i.m. Pi concn. increased by only 70%.  A similar effect was obsd. in erythrocytes from patients with these disorders which were incubated in autologous plasma at 37°, under an atm. of O2 + CO2.  However, there was no general effect on passive anion distribution.  When erythrocytes from normal subjects were incubated in autologous plasma and in plasma supplemented with Pi the Pi concn. in the cells, at steady state, increased only from 0.57 to 0.78 mmol/L cells, suggesting that effect was not an artifact of disease or drug therapy.  Thus, in human skeletal myocytes and erythrocytes, the percentage change in the concn. of cytoplasmic Pi is lower than that in plasma.  This implies that these cells can buffer or regulate cytoplasmic Pi when the extracellular concn. is disturbed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVQuVo95gnZbVg90H21EOLACvtfcHk0ljKeIt2eLATrw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL2sXkvVWnsA%253D%253D&md5=55a9afb0860b8079327f9bc6814ab888</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1042%2Fcs0710729&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252Fcs0710729%26sid%3Dliteratum%253Aachs%26aulast%3DBevington%26aufirst%3DA.%26aulast%3DMundy%26aufirst%3DK.%2BI.%26aulast%3DYates%26aufirst%3DA.%2BJ.%26aulast%3DKanis%26aufirst%3DJ.%2BA.%26aulast%3DRussell%26aufirst%3DR.%2BG.%26aulast%3DTaylor%26aufirst%3DD.%2BJ.%26aulast%3DRajagopalan%26aufirst%3DB.%26aulast%3DRadda%26aufirst%3DG.%2BK.%26atitle%3DA%2520study%2520of%2520intracellular%2520orthophosphate%2520concentration%2520in%2520human%2520muscle%2520and%2520erythrocytes%2520by%252031P%2520nuclear%2520magnetic%2520resonance%2520spectroscopy%2520and%2520selective%2520chemical%2520assay%26jtitle%3DClin.%2520Sci.%26date%3D1986%26volume%3D71%26spage%3D729%26epage%3D735%26doi%3D10.1042%2Fcs0710729" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Ambaye, N. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pero, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gunzburg, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yap, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clayton, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Del Borgo, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perlmutter, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aguilar, M. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shukla, G. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peletskaya, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cookson, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krag, D. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilce, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilce, J. A.</span><span> </span><span class="NLM_article-title">Structural basis of binding by cyclic nonphosphorylated peptide antagonists of Grb7 implicated in breast cancer progression</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">412</span><span class="NLM_x">, </span> <span class="NLM_fpage">397</span><span class="NLM_x">–</span> <span class="NLM_lpage">411</span><span class="refDoi"> DOI: 10.1016/j.jmb.2011.07.030</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=10.1016%2Fj.jmb.2011.07.030" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=21802427" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFarurbF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=412&publication_year=2011&pages=397-411&author=N.+D.+Ambayeauthor=S.+C.+Peroauthor=M.+J.+Gunzburgauthor=M.+Yapauthor=D.+J.+Claytonauthor=M.+P.+Del+Borgoauthor=P.+Perlmutterauthor=M.+I.+Aguilarauthor=G.+S.+Shuklaauthor=E.+Peletskayaauthor=M.+M.+Cooksonauthor=D.+N.+Kragauthor=M.+C.+Wilceauthor=J.+A.+Wilce&title=Structural+basis+of+binding+by+cyclic+nonphosphorylated+peptide+antagonists+of+Grb7+implicated+in+breast+cancer+progression&doi=10.1016%2Fj.jmb.2011.07.030"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Basis of Binding by Cyclic Nonphosphorylated Peptide Antagonists of Grb7 Implicated in Breast Cancer Progression</span></div><div class="casAuthors">Ambaye, Nigus D.; Pero, Stephanie C.; Gunzburg, Menachem J.; Yap, Min Yin; Clayton, Daniel J.; Del Borgo, Mark P.; Perlmutter, Patrick; Aguilar, Marie-Isabel; Shukla, Girja S.; Peletskaya, Elena; Cookson, Michelle M.; Krag, David N.; Wilce, Matthew C. J.; Wilce, Jacqueline A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">412</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">397-411</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Growth-receptor-bound protein (Grb)7 is an adapter protein aberrantly overexpressed, along with the erbB-2 receptor in breast cancer and in other cancers.  Normally recruited to focal adhesions with a role in cell migration, it is assocd. with erbB-2 in cancer cells and is found to exacerbate cancer progression via stimulation of cell migration and proliferation.  The G7-18NATE peptide (sequence: WFEGYDNTFPC cyclized via a thioether bond) is a nonphosphorylated peptide that was developed for the specific inhibition of Grb7 by blocking its SH2 domain.  Cell-permeable versions of G7-18NATE are effective in the redn. of migration and proliferation in Grb7-overexpressing cells.  It thus represents a promising starting point for the development of a therapeutic against Grb7.  Here, the authors report the crystal structure of the G7-18NATE peptide in complex with the Grb7-SH2 domain, revealing the structural basis for its interaction.  The authors also report further rounds of phage display that have identified G7-18NATE analogs with micromolar affinity for Grb7-SH2.  These peptides retained amino acids F2, G4, and F9, as well as the YDN motif that the structural biol. study showed to be the main residues in contact with the Grb7-SH2 domain.  Isothermal titrn. calorimetry measurements reveal similar and better binding affinity of these peptides compared with G7-18NATE.  Together, this study facilitates the optimization of second-generation inhibitors of Grb7.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEQ_mNcka417Vg90H21EOLACvtfcHk0lg433ovDEfxVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFarurbF&md5=d00128a2cfbc35681abb5fd4f569840f</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2011.07.030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2011.07.030%26sid%3Dliteratum%253Aachs%26aulast%3DAmbaye%26aufirst%3DN.%2BD.%26aulast%3DPero%26aufirst%3DS.%2BC.%26aulast%3DGunzburg%26aufirst%3DM.%2BJ.%26aulast%3DYap%26aufirst%3DM.%26aulast%3DClayton%26aufirst%3DD.%2BJ.%26aulast%3DDel%2BBorgo%26aufirst%3DM.%2BP.%26aulast%3DPerlmutter%26aufirst%3DP.%26aulast%3DAguilar%26aufirst%3DM.%2BI.%26aulast%3DShukla%26aufirst%3DG.%2BS.%26aulast%3DPeletskaya%26aufirst%3DE.%26aulast%3DCookson%26aufirst%3DM.%2BM.%26aulast%3DKrag%26aufirst%3DD.%2BN.%26aulast%3DWilce%26aufirst%3DM.%2BC.%26aulast%3DWilce%26aufirst%3DJ.%2BA.%26atitle%3DStructural%2520basis%2520of%2520binding%2520by%2520cyclic%2520nonphosphorylated%2520peptide%2520antagonists%2520of%2520Grb7%2520implicated%2520in%2520breast%2520cancer%2520progression%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2011%26volume%3D412%26spage%3D397%26epage%3D411%26doi%3D10.1016%2Fj.jmb.2011.07.030" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Pero, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Daly, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krag, D. N.</span><span> </span><span class="NLM_article-title">Grb7-based molecular therapeutics in cancer</span> <span class="citation_source-journal">Expert Rev. Mol. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">11</span><span class="refDoi"> DOI: 10.1017/S1462399403006227</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=10.1017%2FS1462399403006227" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=14585167" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=1%3ACAS%3A280%3ADC%252BD3srgvFaltw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2003&pages=1-11&author=S.+C.+Peroauthor=R.+J.+Dalyauthor=D.+N.+Krag&title=Grb7-based+molecular+therapeutics+in+cancer&doi=10.1017%2FS1462399403006227"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Grb7-based molecular therapeutics in cancer</span></div><div class="casAuthors">Pero Stephanie C; Daly Roger J; Krag David N</div><div class="citationInfo"><span class="NLM_cas:title">Expert reviews in molecular medicine</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">1-11</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Traditional anti-cancer drugs preferentially kill rapidly growing tumour cells rather than normal cells.  However, most of these drugs have no preferential selection towards cancer cells and are taken up by the whole body, resulting in significant adverse side effects.  Therapeutic molecules that could specifically inhibit undesirable phenotypes are an attractive way of eliminating cancer cells.  There is a widespread effort to develop inhibitors against signal transduction molecules that play a key role in the proliferative, migratory and invasive properties of a cancer cell.  Grb7 is an adaptor-type signalling protein that is recruited via its Src-homology 2 (SH2) domain to a variety of tyrosine kinases.  Grb7 is overexpressed in breast, oesophageal and gastric cancers, and may contribute to the invasive potential of cancer cells.  Molecular interactions involving Grb7 therefore provide attractive targets for therapeutic intervention.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT7Lgiv6JFRiJ6KlWarZTAzfW6udTcc2eZ2dgGo5QsBgrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD3srgvFaltw%253D%253D&md5=584ec9b9538c8b523524d2ad26c52fc8</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1017%2FS1462399403006227&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1017%252FS1462399403006227%26sid%3Dliteratum%253Aachs%26aulast%3DPero%26aufirst%3DS.%2BC.%26aulast%3DDaly%26aufirst%3DR.%2BJ.%26aulast%3DKrag%26aufirst%3DD.%2BN.%26atitle%3DGrb7-based%2520molecular%2520therapeutics%2520in%2520cancer%26jtitle%3DExpert%2520Rev.%2520Mol.%2520Med.%26date%3D2003%26volume%3D5%26spage%3D1%26epage%3D11%26doi%3D10.1017%2FS1462399403006227" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Songyang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoelson, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McGlade, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olivier, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pawson, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bustelo, X. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barbacid, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sabe, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hanafusa, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yi, T.</span><span> </span><span class="NLM_article-title">Specific motifs recognized by the SH2 domains of Csk, 3BP2, fps/fes, GRB-2, HCP, SHC, Syk, and Vav</span> <span class="citation_source-journal">Mol. Cell. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">2777</span><span class="NLM_x">–</span> <span class="NLM_lpage">2785</span><span class="refDoi"> DOI: 10.1128/MCB.14.4.2777</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=10.1128%2FMCB.14.4.2777" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=7511210" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=1%3ACAS%3A528%3ADyaK2cXktVenur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=1994&pages=2777-2785&author=Z.+Songyangauthor=S.+E.+Shoelsonauthor=J.+McGladeauthor=P.+Olivierauthor=T.+Pawsonauthor=X.+R.+Busteloauthor=M.+Barbacidauthor=H.+Sabeauthor=H.+Hanafusaauthor=T.+Yi&title=Specific+motifs+recognized+by+the+SH2+domains+of+Csk%2C+3BP2%2C+fps%2Ffes%2C+GRB-2%2C+HCP%2C+SHC%2C+Syk%2C+and+Vav&doi=10.1128%2FMCB.14.4.2777"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Specific motifs recognized by the SH2 domains of Csk, 3BP2, fps/fes, GRB-2, HCP, SHC, Syk, and Vav</span></div><div class="casAuthors">Songyang, Z.; Shoelson, S. E.; McGlade, J.; Olivier, P.; Pawson, T.; Bustelo, X. R.; Barbacid, M.; Sabe, H.; Hanafusa, H.; et al.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">2777-85</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    </div><div class="casAbstract">Src homol. 2 (SH2) domains provide specificity to intracellular signaling by binding to specific phosphotyrosine (phospho-Tyr)-contg. sequences.  The authors recently developed a technique using a degenerate phosphopeptide library to predict the specificity of individual SH2 domains src family members, Abl, Nck, Sem5, phospholipase C-γ, p85 subunit of phosphatidylinositol-3-kinase, and SHPTP2 (Z. Songyang, S. E. Shoelson, M. Chaudhuri, G. Gish, T. Pawson, W. G. Haser, F. King, T. Roberts, S. Ratnofsky, R. J. Lechleider, B. G. Neel, R. B. Birge, J. E. Fajardo, M. M. Chou, H. Hanafusa, B. Schaffhausen, and L. C. Cantley, Cell, 72:767-778, 1993).  The authors report here the optimal recognition motifs for SH2 domains from GRB-2, Drk, Csk, Vav, fps/fes, SHC, Syk (carboxy-terminal SH2), 3BP2, and HCP (amino-terminal SH2 domain, also called PTP1C and SHPTP1).  As predicted, SH2 domains from proteins that fall into group I on the basis of a Phe or Tyr at the βD5 position (GRB-2, 3BP2, Csk, fps/fes, Syk C-terminal SH2) select phosphopeptides with the general motif phospho-Tyr-hydrophilic (residue)-hydrophilic (residue)-hydrophobic (residue).  The SH2 domains of SHC and HCP (group III proteins with Ile, Leu, or Cys at the βD5 position) selected the general motif phospho-Tyr-hydrophobic-Xxx-hydrophobic, also as predicted.  Vav, which has a Thr at the βD5 position, selected phospho-Tyr-Met-Glu-Pro as the optimal motif.  Each SH2 domain selected a unique optimal motif distinct from motifs previously detd. for other SH2 domains.  These motifs are used to predict potential sites in signaling proteins for interaction with specific SH2 domain-contg. proteins.  The Syk SH2 domain is predicted to bind to Tyr-hydrophilic-hydrophilic-Leu/Ile motifs like those repeated at 10-residue intervals in T- and B-cell receptor-assocd. proteins.  SHC is predicted to bind to a subgroup of these same motifs.  A structural basis for the assocn. of Csk with Src family members is also suggested from these studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrDwa25r4DEDbVg90H21EOLACvtfcHk0lg433ovDEfxVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXktVenur8%253D&md5=51bfdaf8c43a4b01c39011f79930e85c</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1128%2FMCB.14.4.2777&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.14.4.2777%26sid%3Dliteratum%253Aachs%26aulast%3DSongyang%26aufirst%3DZ.%26aulast%3DShoelson%26aufirst%3DS.%2BE.%26aulast%3DMcGlade%26aufirst%3DJ.%26aulast%3DOlivier%26aufirst%3DP.%26aulast%3DPawson%26aufirst%3DT.%26aulast%3DBustelo%26aufirst%3DX.%2BR.%26aulast%3DBarbacid%26aufirst%3DM.%26aulast%3DSabe%26aufirst%3DH.%26aulast%3DHanafusa%26aufirst%3DH.%26aulast%3DYi%26aufirst%3DT.%26atitle%3DSpecific%2520motifs%2520recognized%2520by%2520the%2520SH2%2520domains%2520of%2520Csk%252C%25203BP2%252C%2520fps%252Ffes%252C%2520GRB-2%252C%2520HCP%252C%2520SHC%252C%2520Syk%252C%2520and%2520Vav%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D1994%26volume%3D14%26spage%3D2777%26epage%3D2785%26doi%3D10.1128%2FMCB.14.4.2777" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Janes, P. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lackmann, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Church, W. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sanderson, G. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sutherland, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Daly, R. J.</span><span> </span><span class="NLM_article-title">Structural determinants of the interaction between the erbB2 receptor and the Src homology 2 domain of Grb7</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">272</span><span class="NLM_x">, </span> <span class="NLM_fpage">8490</span><span class="NLM_x">–</span> <span class="NLM_lpage">8497</span><span class="refDoi"> DOI: 10.1074/jbc.272.13.8490</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=10.1074%2Fjbc.272.13.8490" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=272&publication_year=1997&pages=8490-8497&author=P.+W.+Janesauthor=M.+Lackmannauthor=W.+B.+Churchauthor=G.+M.+Sandersonauthor=R.+L.+Sutherlandauthor=R.+J.+Daly&title=Structural+determinants+of+the+interaction+between+the+erbB2+receptor+and+the+Src+homology+2+domain+of+Grb7&doi=10.1074%2Fjbc.272.13.8490"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1074%2Fjbc.272.13.8490&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.272.13.8490%26sid%3Dliteratum%253Aachs%26aulast%3DJanes%26aufirst%3DP.%2BW.%26aulast%3DLackmann%26aufirst%3DM.%26aulast%3DChurch%26aufirst%3DW.%2BB.%26aulast%3DSanderson%26aufirst%3DG.%2BM.%26aulast%3DSutherland%26aufirst%3DR.%2BL.%26aulast%3DDaly%26aufirst%3DR.%2BJ.%26atitle%3DStructural%2520determinants%2520of%2520the%2520interaction%2520between%2520the%2520erbB2%2520receptor%2520and%2520the%2520Src%2520homology%25202%2520domain%2520of%2520Grb7%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1997%26volume%3D272%26spage%3D8490%26epage%3D8497%26doi%3D10.1074%2Fjbc.272.13.8490" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Scott, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bayly, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abell, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Skidmore, J.</span><span> </span><span class="NLM_article-title">Small molecules, big targets: drug discovery faces the protein-protein interaction challenge</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">533</span><span class="NLM_x">–</span> <span class="NLM_lpage">550</span><span class="refDoi"> DOI: 10.1038/nrd.2016.29</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=10.1038%2Fnrd.2016.29" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=27050677" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=1%3ACAS%3A528%3ADC%252BC28XlsFOgur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=533-550&author=D.+E.+Scottauthor=A.+R.+Baylyauthor=C.+Abellauthor=J.+Skidmore&title=Small+molecules%2C+big+targets%3A+drug+discovery+faces+the+protein-protein+interaction+challenge&doi=10.1038%2Fnrd.2016.29"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Small molecules, big targets: drug discovery faces the protein-protein interaction challenge</span></div><div class="casAuthors">Scott, Duncan E.; Bayly, Andrew R.; Abell, Chris; Skidmore, John</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">533-550</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Protein-protein interactions (PPIs) are of pivotal importance in the regulation of biol. systems and are consequently implicated in the development of disease states.  Recent work has begun to show that, with the right tools, certain classes of PPI can yield to the efforts of medicinal chemists to develop inhibitors, and the first PPI inhibitors have reached clin. development.  In this Review, we describe the research leading to these breakthroughs and highlight the existence of groups of structurally related PPIs within the PPI target class.  For each of these groups, we use examples of successful discovery efforts to illustrate the research strategies that have proved most useful.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoK-aGzHa9mP7Vg90H21EOLACvtfcHk0lifbJShHETklw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XlsFOgur4%253D&md5=9f0953567ae9338b8b36aa78b8160655</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.29&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.29%26sid%3Dliteratum%253Aachs%26aulast%3DScott%26aufirst%3DD.%2BE.%26aulast%3DBayly%26aufirst%3DA.%2BR.%26aulast%3DAbell%26aufirst%3DC.%26aulast%3DSkidmore%26aufirst%3DJ.%26atitle%3DSmall%2520molecules%252C%2520big%2520targets%253A%2520drug%2520discovery%2520faces%2520the%2520protein-protein%2520interaction%2520challenge%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2016%26volume%3D15%26spage%3D533%26epage%3D550%26doi%3D10.1038%2Fnrd.2016.29" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Kraskouskaya, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duodu, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arpin, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gunning, P. T.</span><span> </span><span class="NLM_article-title">Progress towards the development of SH2 domain inhibitors</span> <span class="citation_source-journal">Chem. Soc. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">42</span><span class="NLM_x">, </span> <span class="NLM_fpage">3337</span><span class="NLM_x">–</span> <span class="NLM_lpage">3370</span><span class="refDoi"> DOI: 10.1039/c3cs35449k</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=10.1039%2Fc3cs35449k" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2013&pages=3337-3370&author=D.+Kraskouskayaauthor=E.+Duoduauthor=C.+C.+Arpinauthor=P.+T.+Gunning&title=Progress+towards+the+development+of+SH2+domain+inhibitors&doi=10.1039%2Fc3cs35449k"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1039%2Fc3cs35449k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc3cs35449k%26sid%3Dliteratum%253Aachs%26aulast%3DKraskouskaya%26aufirst%3DD.%26aulast%3DDuodu%26aufirst%3DE.%26aulast%3DArpin%26aufirst%3DC.%2BC.%26aulast%3DGunning%26aufirst%3DP.%2BT.%26atitle%3DProgress%2520towards%2520the%2520development%2520of%2520SH2%2520domain%2520inhibitors%26jtitle%3DChem.%2520Soc.%2520Rev.%26date%3D2013%26volume%3D42%26spage%3D3337%26epage%3D3370%26doi%3D10.1039%2Fc3cs35449k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">Gunzburg, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ambaye, N. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Del Borgo, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perlmutter, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilce, J. A.</span><span> </span><span class="NLM_article-title">Design and testing of bicyclic inhibitors of Grb7—are two cycles better than one?</span> <span class="citation_source-journal">Biopolymers</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">100</span><span class="NLM_x">, </span> <span class="NLM_fpage">543</span><span class="NLM_x">–</span> <span class="NLM_lpage">549</span><span class="refDoi"> DOI: 10.1002/bip.22237</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=10.1002%2Fbip.22237" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=100&publication_year=2013&pages=543-549&author=M.+J.+Gunzburgauthor=N.+D.+Ambayeauthor=M.+P.+Del+Borgoauthor=P.+Perlmutterauthor=J.+A.+Wilce&title=Design+and+testing+of+bicyclic+inhibitors+of+Grb7%E2%80%94are+two+cycles+better+than+one%3F&doi=10.1002%2Fbip.22237"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1002%2Fbip.22237&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fbip.22237%26sid%3Dliteratum%253Aachs%26aulast%3DGunzburg%26aufirst%3DM.%2BJ.%26aulast%3DAmbaye%26aufirst%3DN.%2BD.%26aulast%3DDel%2BBorgo%26aufirst%3DM.%2BP.%26aulast%3DPerlmutter%26aufirst%3DP.%26aulast%3DWilce%26aufirst%3DJ.%2BA.%26atitle%3DDesign%2520and%2520testing%2520of%2520bicyclic%2520inhibitors%2520of%2520Grb7%25E2%2580%2594are%2520two%2520cycles%2520better%2520than%2520one%253F%26jtitle%3DBiopolymers%26date%3D2013%26volume%3D100%26spage%3D543%26epage%3D549%26doi%3D10.1002%2Fbip.22237" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Lau, Y. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Andrade, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spring, D. R.</span><span> </span><span class="NLM_article-title">Peptide stapling techniques based on different macrocyclisation chemistries</span> <span class="citation_source-journal">Chem. Soc. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">44</span><span class="NLM_x">, </span> <span class="NLM_fpage">91</span><span class="NLM_x">–</span> <span class="NLM_lpage">102</span><span class="refDoi"> DOI: 10.1039/C4CS00246F</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=10.1039%2FC4CS00246F" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=25199043" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsV2mtb%252FL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2015&pages=91-102&author=Y.+H.+Lauauthor=P.+de+Andradeauthor=Y.+Wuauthor=D.+R.+Spring&title=Peptide+stapling+techniques+based+on+different+macrocyclisation+chemistries&doi=10.1039%2FC4CS00246F"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Peptide stapling techniques based on different macrocyclization chemistries</span></div><div class="casAuthors">Lau, Yu Heng; de Andrade, Peterson; Wu, Yuteng; Spring, David R.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Society Reviews</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">91-102</span>CODEN:
                <span class="NLM_cas:coden">CSRVBR</span>;
        ISSN:<span class="NLM_cas:issn">0306-0012</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  Peptide stapling is a strategy for constraining short peptides typically in an alpha-helical conformation.  Stapling is carried out by covalently linking the side-chains of two amino acids, thereby forming a peptide macrocycle.  There is an expanding repertoire of stapling techniques based on different macrocyclization chemistries.  In this tutorial review, the authors categorize and analyze key examples of peptide stapling in terms of their synthesis and applicability to biol. systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkqQ6yGsikIbVg90H21EOLACvtfcHk0liMAMmDsJdoCA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsV2mtb%252FL&md5=0efb31241f9809b61a81474131a54de3</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1039%2FC4CS00246F&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC4CS00246F%26sid%3Dliteratum%253Aachs%26aulast%3DLau%26aufirst%3DY.%2BH.%26aulast%3Dde%2BAndrade%26aufirst%3DP.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DSpring%26aufirst%3DD.%2BR.%26atitle%3DPeptide%2520stapling%2520techniques%2520based%2520on%2520different%2520macrocyclisation%2520chemistries%26jtitle%3DChem.%2520Soc.%2520Rev.%26date%3D2015%26volume%3D44%26spage%3D91%26epage%3D102%26doi%3D10.1039%2FC4CS00246F" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group">Nam, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ye, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parang, K.</span><span> </span><span class="NLM_article-title">Conformationally constrained peptide analogues of pTyr-Glu-Glu-Ile as inhibitors of the Src SH2 domain binding</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">3131</span><span class="NLM_x">–</span> <span class="NLM_lpage">3141</span><span class="refDoi"> DOI: 10.1021/jm040008+</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm040008%2B" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=1%3ACAS%3A528%3ADC%252BD2cXjslOhs7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=3131-3141&author=N.+Namauthor=G.+Yeauthor=G.+Sunauthor=K.+Parang&title=Conformationally+constrained+peptide+analogues+of+pTyr-Glu-Glu-Ile+as+inhibitors+of+the+Src+SH2+domain+binding&doi=10.1021%2Fjm040008%2B"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Conformationally Constrained Peptide Analogues of pTyr-Glu-Glu-Ile as Inhibitors of the Src SH2 Domain Binding</span></div><div class="casAuthors">Nam, Nguyen-Hai; Ye, Guofeng; Sun, Gongqin; Parang, Keykavous</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3131-3141</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A series of conformationally-constrained peptides were designed and synthesized as the Src SH2 domain ligands based on a tetrapeptide sequence pTyr-Glu-Glu-Ile (pYEEI).  In general, the constrained peptides showed higher binding affinities to the Src SH2 domain relative to the corresponding linear peptides as evaluated by a fluorescence polarization assay.  Mol. modeling studies revealed that in constrained peptides, the isoleucine side chain penetrates very deeply into the hydrophobic binding pocket (P + 3 site) of the Src SH2 domain.  These constrained peptides can serve as novel templates for the design of small and nonpeptidic inhibitors of the Src SH2 domain.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpsUsOw9C8CmLVg90H21EOLACvtfcHk0liMAMmDsJdoCA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXjslOhs7s%253D&md5=af4660ffefc20ac0ef23804370b2bccc</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fjm040008%2B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm040008%252B%26sid%3Dliteratum%253Aachs%26aulast%3DNam%26aufirst%3DN.%26aulast%3DYe%26aufirst%3DG.%26aulast%3DSun%26aufirst%3DG.%26aulast%3DParang%26aufirst%3DK.%26atitle%3DConformationally%2520constrained%2520peptide%2520analogues%2520of%2520pTyr-Glu-Glu-Ile%2520as%2520inhibitors%2520of%2520the%2520Src%2520SH2%2520domain%2520binding%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D3131%26epage%3D3141%26doi%3D10.1021%2Fjm040008%2B" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group">Quartararo, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kritzer, J. A.</span><span> </span><span class="NLM_article-title">Peptide bicycles that inhibit the Grb2 SH2 domain</span> <span class="citation_source-journal">ChemBioChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">1490</span><span class="NLM_x">–</span> <span class="NLM_lpage">1496</span><span class="refDoi"> DOI: 10.1002/cbic.201200175</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=10.1002%2Fcbic.201200175" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=22689355" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=1%3ACAS%3A528%3ADC%252BC38Xot1KgsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2012&pages=1490-1496&author=J.+S.+Quartararoauthor=P.+Wuauthor=J.+A.+Kritzer&title=Peptide+bicycles+that+inhibit+the+Grb2+SH2+domain&doi=10.1002%2Fcbic.201200175"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Peptide Bicycles that Inhibit the Grb2 SH2 Domain</span></div><div class="casAuthors">Quartararo, Justin S.; Wu, Pianpian; Kritzer, Joshua A.</div><div class="citationInfo"><span class="NLM_cas:title">ChemBioChem</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1490-1496</span>CODEN:
                <span class="NLM_cas:coden">CBCHFX</span>;
        ISSN:<span class="NLM_cas:issn">1439-4227</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Developing short peptides into useful probes and therapeutic leads remains a difficult challenge.  Structural rigidification is a proven method for improving the properties of short peptides.  In this work, we report a strategy for stabilizing peptide macrocycles by introducing side-chain-to-side-chain staples to produce peptide bicycles with higher affinity, selectivity, and resistance to degrdn.  We have applied this strategy to G1, an 11-residue peptide macrocycle that binds the Src homol. 2 (SH2) domain of growth-factor-bound protein 2 (Grb2).  Several homodetic peptide bicycles were synthesized entirely on-resin with high yields.  Two rounds of iterative design produced peptide bicycle BC1, which is 60 times more potent than G1 and 200 times more selective.  Moreover, BC1 is completely intact after 24 h in buffered human serum, conditions under which G1 is completely degraded.  Our peptide-bicycle approach holds promise for the development of selective inhibitors of SH2 domains and other phosphotyrosine (pTyr)-binding proteins, as well as inhibitors of many other protein-protein interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGogV33oAc3rpbVg90H21EOLACvtfcHk0lhevFVM3xn4tQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xot1KgsLY%253D&md5=0a23f863636c340cd4dcff0c58321661</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1002%2Fcbic.201200175&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbic.201200175%26sid%3Dliteratum%253Aachs%26aulast%3DQuartararo%26aufirst%3DJ.%2BS.%26aulast%3DWu%26aufirst%3DP.%26aulast%3DKritzer%26aufirst%3DJ.%2BA.%26atitle%3DPeptide%2520bicycles%2520that%2520inhibit%2520the%2520Grb2%2520SH2%2520domain%26jtitle%3DChemBioChem%26date%3D2012%26volume%3D13%26spage%3D1490%26epage%3D1496%26doi%3D10.1002%2Fcbic.201200175" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group">Fischer, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waizenegger, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kohler, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brock, R.</span><span> </span><span class="NLM_article-title">A quantitative validation of fluorophore-labelled cell-permeable peptide conjugates: fluorophore and cargo dependence of import</span> <span class="citation_source-journal">Biochim. Biophys. Acta, Biomembr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">1564</span><span class="NLM_x">, </span> <span class="NLM_fpage">365</span><span class="NLM_x">–</span> <span class="NLM_lpage">374</span><span class="refDoi"> DOI: 10.1016/S0005-2736(02)00471-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=10.1016%2FS0005-2736%2802%2900471-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=12175919" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=1%3ACAS%3A528%3ADC%252BD38XmtlCku7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1564&publication_year=2002&pages=365-374&author=R.+Fischerauthor=T.+Waizeneggerauthor=K.+Kohlerauthor=R.+Brock&title=A+quantitative+validation+of+fluorophore-labelled+cell-permeable+peptide+conjugates%3A+fluorophore+and+cargo+dependence+of+import&doi=10.1016%2FS0005-2736%2802%2900471-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">A quantitative validation of fluorophore-labelled cell-permeable peptide conjugates: fluorophore and cargo dependence of import</span></div><div class="casAuthors">Fischer, Rainer; Waizenegger, Thomas; Kohler, Karsten; Brock, Roland</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Biomembranes</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">1564</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">365-374</span>CODEN:
                <span class="NLM_cas:coden">BBBMBS</span>;
        ISSN:<span class="NLM_cas:issn">0005-2736</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Cell-permeable peptides were evaluated for a quant. controlled import of small mols.  The dependence of the import efficiency on the fluorophore, on the position of the fluorophore as well as on the nature of the cargo were addressed.  Cellular uptake was quantitated by flow cytometry and fluorescence correlation microscopy (FCM).  Fluorophores with different spectral characteristics, covering the whole visible spectral range, were selected in order to enable the simultaneous detection of several cell-permeable peptide constructs.  The transcytosis sequences were based either on the sequence of the Antennapedia homeodomain protein (AntpHD)-derived penetratin peptide or the Kaposi fibroblast growth factor (FGF)-derived membrane translocating sequence (MTS)-peptide.  In general, the AntpHD-derived peptides had a three- to fourfold higher import efficiency than the MTS-derived peptides.  In spite of the very different physicochem. characteristics of the fluorophores, the import efficiencies for analogs labeled at different positions within the sequence of the import peptides showed a strong pos. correlation.  However, even for peptide cargos of very similar size, pronounced differences in import efficiency were obsd.  The use of cell-permeable peptide/cargo constructs for intracellular analyses of structure-function relationships therefore requires the detn. of the intracellular concns. for each construct individually.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpS1odJZP8c_7Vg90H21EOLACvtfcHk0lhevFVM3xn4tQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XmtlCku7c%253D&md5=e20688aef28a746afc276cd97700a245</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2FS0005-2736%2802%2900471-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0005-2736%252802%252900471-6%26sid%3Dliteratum%253Aachs%26aulast%3DFischer%26aufirst%3DR.%26aulast%3DWaizenegger%26aufirst%3DT.%26aulast%3DKohler%26aufirst%3DK.%26aulast%3DBrock%26aufirst%3DR.%26atitle%3DA%2520quantitative%2520validation%2520of%2520fluorophore-labelled%2520cell-permeable%2520peptide%2520conjugates%253A%2520fluorophore%2520and%2520cargo%2520dependence%2520of%2520import%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Biomembr.%26date%3D2002%26volume%3D1564%26spage%3D365%26epage%3D374%26doi%3D10.1016%2FS0005-2736%2802%2900471-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group">Maiolo, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferrer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ottinger, E. A.</span><span> </span><span class="NLM_article-title">Effects of cargo molecules on the cellular uptake of arginine-rich cell-penetrating peptides</span> <span class="citation_source-journal">Biochim. Biophys. Acta, Biomembr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">1712</span><span class="NLM_x">, </span> <span class="NLM_fpage">161</span><span class="NLM_x">–</span> <span class="NLM_lpage">172</span><span class="refDoi"> DOI: 10.1016/j.bbamem.2005.04.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=10.1016%2Fj.bbamem.2005.04.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=15935328" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=1%3ACAS%3A528%3ADC%252BD2MXltFSku7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1712&publication_year=2005&pages=161-172&author=J.+R.+Maioloauthor=M.+Ferrerauthor=E.+A.+Ottinger&title=Effects+of+cargo+molecules+on+the+cellular+uptake+of+arginine-rich+cell-penetrating+peptides&doi=10.1016%2Fj.bbamem.2005.04.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Effects of cargo molecules on the cellular uptake of arginine-rich cell-penetrating peptides</span></div><div class="casAuthors">Maiolo, James R.; Ferrer, Marc; Ottinger, Elizabeth A.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Biomembranes</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">1712</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">161-172</span>CODEN:
                <span class="NLM_cas:coden">BBBMBS</span>;
        ISSN:<span class="NLM_cas:issn">0005-2736</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The identification of cell-penetrating peptides (CPPs) as vectors for the intracellular delivery of conjugated mols. such as peptides, proteins, and oligonucleotides has emerged as a significant tool to modulate biol. activities inside cells.  The mechanism of CPP uptake by the cells is still unclear, and appears to be both endocytotic and non-endocytotic, depending on the CPP and cell type.  Moreover, it is also unknown whether cargo sequences have an effect on the uptake and cellular distribution properties of CPP sequences.  Here, we combine results from quant. fluorescence microscopy and binding to lipid membrane models to det. the effect of cargo peptide mols. on the cellular uptake and distribution of the arginine-rich CPPs, R7, and R7W, in live cells.  Image anal. algorithms that quantify fluorescence were used to measure the relative amt. of peptide taken up by the cell, as well as the extent to which the uptake was endocytotic in nature.  The results presented here indicate that fusion of arginine-rich CPPs to peptide sequences reduces the efficiency of uptake, and dramatically changes the cellular distribution of the CPP from a diffuse pattern to one in which the peptides are mostly retained in endosomal compartments.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQi9cX9dekbbVg90H21EOLACvtfcHk0lizU_kl01dCGw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXltFSku7k%253D&md5=e2e2cb45c5100f509b8d450991d32147</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.bbamem.2005.04.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbamem.2005.04.010%26sid%3Dliteratum%253Aachs%26aulast%3DMaiolo%26aufirst%3DJ.%2BR.%26aulast%3DFerrer%26aufirst%3DM.%26aulast%3DOttinger%26aufirst%3DE.%2BA.%26atitle%3DEffects%2520of%2520cargo%2520molecules%2520on%2520the%2520cellular%2520uptake%2520of%2520arginine-rich%2520cell-penetrating%2520peptides%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Biomembr.%26date%3D2005%26volume%3D1712%26spage%3D161%26epage%3D172%26doi%3D10.1016%2Fj.bbamem.2005.04.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group">El-Andaloussi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jarver, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johansson, H. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Langel, U.</span><span> </span><span class="NLM_article-title">Cargo-dependent cytotoxicity and delivery efficacy of cell-penetrating peptides: a comparative study</span> <span class="citation_source-journal">Biochem. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">407</span><span class="NLM_x">, </span> <span class="NLM_fpage">285</span><span class="NLM_x">–</span> <span class="NLM_lpage">292</span><span class="refDoi"> DOI: 10.1042/BJ20070507</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=10.1042%2FBJ20070507" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=17627607" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtVyrtr3P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=407&publication_year=2007&pages=285-292&author=S.+El-Andaloussiauthor=P.+Jarverauthor=H.+J.+Johanssonauthor=U.+Langel&title=Cargo-dependent+cytotoxicity+and+delivery+efficacy+of+cell-penetrating+peptides%3A+a+comparative+study&doi=10.1042%2FBJ20070507"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Cargo-dependent cytotoxicity and delivery efficacy of cell-penetrating peptides: A comparative study</span></div><div class="casAuthors">El-Andaloussi, Samir; Jaerver, Peter; Johansson, Henrik J.; Langel, Uelo</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Journal</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">407</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">285-292</span>CODEN:
                <span class="NLM_cas:coden">BIJOAK</span>;
        ISSN:<span class="NLM_cas:issn">0264-6021</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">The use of CPPs (cell-penetrating peptides) as delivery vectors for bioactive mols. has been an emerging field since 1994 when the first CPP, penetratin, was discovered.  Since then, several CPPs, including the widely used Tat (transactivator of transcription) peptide, have been developed and utilized to translocate a wide range of compds. across the plasma membrane of cells both in vivo and in vitro.  Although the field has emerged as a possible future candidate for drug delivery, little attention has been given to the potential toxic side effects that these peptides might exhibit in cargo delivery.  Also, no comprehensive study has been performed to evaluate the relative efficacy of single CPPs to convey different cargos.  Therefore the authors selected three of the major CPPs, penetratin, Tat and transportan 10, and evaluated their ability to deliver commonly used cargos, including fluoresceinyl moiety, double-stranded DNA and proteins (i.e. avidin and streptavidin), and studied their effect on membrane integrity and cell viability.  The results demonstrate the unfeasibility to use the translocation efficacy of fluorescein moiety as a gauge for CPP efficiency, since the delivery properties are dependent on the cargo used.  Furthermore, and no less importantly, the toxicity of CPPs depends heavily on peptide concn., cargo mol. and coupling strategy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqMuztXTLEVq7Vg90H21EOLACvtfcHk0lizU_kl01dCGw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtVyrtr3P&md5=41e2f8e70acb7e80314938a7dbc56d37</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1042%2FBJ20070507&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBJ20070507%26sid%3Dliteratum%253Aachs%26aulast%3DEl-Andaloussi%26aufirst%3DS.%26aulast%3DJarver%26aufirst%3DP.%26aulast%3DJohansson%26aufirst%3DH.%2BJ.%26aulast%3DLangel%26aufirst%3DU.%26atitle%3DCargo-dependent%2520cytotoxicity%2520and%2520delivery%2520efficacy%2520of%2520cell-penetrating%2520peptides%253A%2520a%2520comparative%2520study%26jtitle%3DBiochem.%2520J.%26date%3D2007%26volume%3D407%26spage%3D285%26epage%3D292%26doi%3D10.1042%2FBJ20070507" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group">Ambaye, N. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lim, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clayton, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gunzburg, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Price, J. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pero, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krag, D. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilce, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aguilar, M. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perlmutter, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilce, J. A.</span><span> </span><span class="NLM_article-title">Uptake of a cell permeable G7–18NATE contruct into cells and binding with the Grb-7-SH2 domain</span> <span class="citation_source-journal">Biopolymers</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">96</span><span class="NLM_x">, </span> <span class="NLM_fpage">181</span><span class="NLM_x">–</span> <span class="NLM_lpage">188</span><span class="refDoi"> DOI: 10.1002/bip.21403</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=10.1002%2Fbip.21403" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=96&publication_year=2011&pages=181-188&author=N.+D.+Ambayeauthor=R.+C.+Limauthor=D.+J.+Claytonauthor=M.+J.+Gunzburgauthor=J.+T.+Priceauthor=S.+C.+Peroauthor=D.+N.+Kragauthor=M.+C.+Wilceauthor=M.+I.+Aguilarauthor=P.+Perlmutterauthor=J.+A.+Wilce&title=Uptake+of+a+cell+permeable+G7%E2%80%9318NATE+contruct+into+cells+and+binding+with+the+Grb-7-SH2+domain&doi=10.1002%2Fbip.21403"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1002%2Fbip.21403&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fbip.21403%26sid%3Dliteratum%253Aachs%26aulast%3DAmbaye%26aufirst%3DN.%2BD.%26aulast%3DLim%26aufirst%3DR.%2BC.%26aulast%3DClayton%26aufirst%3DD.%2BJ.%26aulast%3DGunzburg%26aufirst%3DM.%2BJ.%26aulast%3DPrice%26aufirst%3DJ.%2BT.%26aulast%3DPero%26aufirst%3DS.%2BC.%26aulast%3DKrag%26aufirst%3DD.%2BN.%26aulast%3DWilce%26aufirst%3DM.%2BC.%26aulast%3DAguilar%26aufirst%3DM.%2BI.%26aulast%3DPerlmutter%26aufirst%3DP.%26aulast%3DWilce%26aufirst%3DJ.%2BA.%26atitle%3DUptake%2520of%2520a%2520cell%2520permeable%2520G7%25E2%2580%259318NATE%2520contruct%2520into%2520cells%2520and%2520binding%2520with%2520the%2520Grb-7-SH2%2520domain%26jtitle%3DBiopolymers%26date%3D2011%26volume%3D96%26spage%3D181%26epage%3D188%26doi%3D10.1002%2Fbip.21403" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group">Pace, C. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vajdos, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fee, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grimsley, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, T.</span><span> </span><span class="NLM_article-title">How to measure and predict the molar absorption coefficient of a protein</span> <span class="citation_source-journal">Protein Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">2411</span><span class="NLM_x">–</span> <span class="NLM_lpage">2423</span><span class="refDoi"> DOI: 10.1002/pro.5560041120</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=10.1002%2Fpro.5560041120" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=8563639" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=1%3ACAS%3A528%3ADyaK2MXpsVGqt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=1995&pages=2411-2423&author=C.+N.+Paceauthor=F.+Vajdosauthor=L.+Feeauthor=G.+Grimsleyauthor=T.+Gray&title=How+to+measure+and+predict+the+molar+absorption+coefficient+of+a+protein&doi=10.1002%2Fpro.5560041120"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">How to measure and predict the molar absorption coefficient of a protein</span></div><div class="casAuthors">Pace, C. Nick; Vajdos, Felix; Fee, Lanette; Grimsley, Gerald; Gray, Theronica</div><div class="citationInfo"><span class="NLM_cas:title">Protein Science</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2411-23</span>CODEN:
                <span class="NLM_cas:coden">PRCIEI</span>;
        ISSN:<span class="NLM_cas:issn">0961-8368</span>.
    
            (<span class="NLM_cas:orgname">Cambridge University Press</span>)
        </div><div class="casAbstract">The molar absorption coeff., ε, of a protein is usually based on concns. measured by dry wt., nitrogen, or amino acid anal.  The studies reported here suggest that the Edelhoch method is the best method for measuring ε for a protein.  (This method is described by Gill and von Hippel [1989, Anal Biochem 182:319-326] and is based on data from Edelhoch [1967, Biochem. 6:1948-1954].).  The av. ε values for these chromophores in a sample of 18 well-characterized proteins have been estd., and the ε values in water, propanol, 6 M guanidine hydrochloride (GdnHCl), and 8 M urea have been measured.  For Trp, the av. ε values for the proteins are intermediate between those measured in 6 M GdnHCl and those measured in propanol.  Based on a sample of 116 measured ε values for 80 proteins, the ε at 280 nm of a folded protein in water, ε(280), can best be predicted with this equation L ε(280) (M-1 cm-1) = (#Trp)(5,500) + (#Tyr)(1,490) + (#cystine)(125).  These ε(280) values are quite reliable for proteins contg. Trp residues, and less reliable for proteins that do not.  However, the Edelhoch method is convenient and accurate, and the best approach is to measure rather than predict ε.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDyrAj6uk11bVg90H21EOLACvtfcHk0lizU_kl01dCGw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXpsVGqt7k%253D&md5=033237a125c7486331bf5180047c3cd4</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1002%2Fpro.5560041120&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpro.5560041120%26sid%3Dliteratum%253Aachs%26aulast%3DPace%26aufirst%3DC.%2BN.%26aulast%3DVajdos%26aufirst%3DF.%26aulast%3DFee%26aufirst%3DL.%26aulast%3DGrimsley%26aufirst%3DG.%26aulast%3DGray%26aufirst%3DT.%26atitle%3DHow%2520to%2520measure%2520and%2520predict%2520the%2520molar%2520absorption%2520coefficient%2520of%2520a%2520protein%26jtitle%3DProtein%2520Sci.%26date%3D1995%26volume%3D4%26spage%3D2411%26epage%3D2423%26doi%3D10.1002%2Fpro.5560041120" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group">McPhillips, T. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McPhillips, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chiu, H. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cohen, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deacon, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ellis, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garman, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gonzalez, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sauter, N. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Phizackerley, R. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soltis, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuhn, P.</span><span> </span><span class="NLM_article-title">Blu-Ice and the Distributed Control System: software for data acquisition and instrument control at macromolecular crystallography beamlines</span> <span class="citation_source-journal">J. Synchrotron Radiat.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">401</span><span class="NLM_x">–</span> <span class="NLM_lpage">406</span><span class="refDoi"> DOI: 10.1107/S0909049502015170</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=10.1107%2FS0909049502015170" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=12409628" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=1%3ACAS%3A528%3ADC%252BD38Xotleluro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2002&pages=401-406&author=T.+M.+McPhillipsauthor=S.+E.+McPhillipsauthor=H.+J.+Chiuauthor=A.+E.+Cohenauthor=A.+M.+Deaconauthor=P.+J.+Ellisauthor=E.+Garmanauthor=A.+Gonzalezauthor=N.+K.+Sauterauthor=R.+P.+Phizackerleyauthor=S.+M.+Soltisauthor=P.+Kuhn&title=Blu-Ice+and+the+Distributed+Control+System%3A+software+for+data+acquisition+and+instrument+control+at+macromolecular+crystallography+beamlines&doi=10.1107%2FS0909049502015170"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Blu-Ice and the Distributed Control System: software for data acquisition and instrument control at macromolecular crystallography beamlines</span></div><div class="casAuthors">McPhillips, Timothy M.; McPhillips, Scott E.; Chiu, Hsiu Ju; Cohen, Aina E.; Deacon, Ashley M.; Ellis, Paul J.; Garman, Elspeth; Gonzalez, Ana; Sauter, Nicholas K.; Phizackerley, R. Paul; Soltis, S. Michael; Kuhn, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Synchrotron Radiation</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">401-406</span>CODEN:
                <span class="NLM_cas:coden">JSYRES</span>;
        ISSN:<span class="NLM_cas:issn">0909-0495</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Munksgaard</span>)
        </div><div class="casAbstract">The Blu-Ice and Distributed Control System (DCS) software packages were developed to provide unified control over the disparate hardware resources available at a macromol. crystallog. beamline.  Blu-Ice is a user interface that provides scientific experimenters and beamline support staff with intuitive graphical tools for collecting diffraction data and configuring beamlines for expts.  Blu-Ice communicates with the hardware at a beamline via DCS, an instrument-control and data-acquisition package designed to integrate hardware resources in a highly heterogeneous networked computing environment.  Together, Blu-Ice and DCS provide a flexible platform for increasing the ease of use, the level of automation and the remote accessibility of beamlines.  Blu-Ice and DCS are currently installed on four Stanford Synchrotron Radiation Lab. crystallog. beamlines and are being implemented at sister light sources.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHVsQUIFJPBLVg90H21EOLACvtfcHk0ljmJOAVM1qrRA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xotleluro%253D&md5=c77228bd50890463f3e43e981a2e238a</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1107%2FS0909049502015170&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0909049502015170%26sid%3Dliteratum%253Aachs%26aulast%3DMcPhillips%26aufirst%3DT.%2BM.%26aulast%3DMcPhillips%26aufirst%3DS.%2BE.%26aulast%3DChiu%26aufirst%3DH.%2BJ.%26aulast%3DCohen%26aufirst%3DA.%2BE.%26aulast%3DDeacon%26aufirst%3DA.%2BM.%26aulast%3DEllis%26aufirst%3DP.%2BJ.%26aulast%3DGarman%26aufirst%3DE.%26aulast%3DGonzalez%26aufirst%3DA.%26aulast%3DSauter%26aufirst%3DN.%2BK.%26aulast%3DPhizackerley%26aufirst%3DR.%2BP.%26aulast%3DSoltis%26aufirst%3DS.%2BM.%26aulast%3DKuhn%26aufirst%3DP.%26atitle%3DBlu-Ice%2520and%2520the%2520Distributed%2520Control%2520System%253A%2520software%2520for%2520data%2520acquisition%2520and%2520instrument%2520control%2520at%2520macromolecular%2520crystallography%2520beamlines%26jtitle%3DJ.%2520Synchrotron%2520Radiat.%26date%3D2002%26volume%3D9%26spage%3D401%26epage%3D406%26doi%3D10.1107%2FS0909049502015170" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group">Battye, T. G. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kontogiannis, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Powell, H. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leslie, A. G. W.</span><span> </span><span class="NLM_article-title">IMosflm: a new graphical interface for diffraction-image processing with MOSFLM</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">67</span><span class="NLM_x">, </span> <span class="NLM_fpage">271</span><span class="NLM_x">–</span> <span class="NLM_lpage">281</span><span class="refDoi"> DOI: 10.1107/S0907444910048675</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=10.1107%2FS0907444910048675" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=21460445" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqtLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=271-281&author=T.+G.+G.+Battyeauthor=L.+Kontogiannisauthor=O.+Johnsonauthor=H.+R.+Powellauthor=A.+G.+W.+Leslie&title=IMosflm%3A+a+new+graphical+interface+for+diffraction-image+processing+with+MOSFLM&doi=10.1107%2FS0907444910048675"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">iMOSFLM: A new graphical interface for diffraction-image processing with MOSFLM</span></div><div class="casAuthors">Battye, T. Geoff G.; Kontogiannis, Luke; Johnson, Owen; Powell, Harold R.; Leslie, Andrew G. W.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">271-281</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">IMOSFLM is a graphical user interface to the diffraction data-integration program MOSFLM.  It is designed to simplify data processing by dividing the process into a series of steps, which are normally carried out sequentially.  Each step has its own display pane, allowing control over parameters that influence that step and providing graphical feedback to the user.  Suitable values for integration parameters are set automatically, but addnl. menus provide a detailed level of control for experienced users.  The image display and the interfaces to the different tasks (indexing, strategy calcn., cell refinement, integration and history) are described.  The most important parameters for each step and the best way of assessing success or failure are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqeZl2UOMg3_bVg90H21EOLACvtfcHk0ljmJOAVM1qrRA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqtLw%253D&md5=852651253f8d9c9a81e0513b6633c705</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1107%2FS0907444910048675&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910048675%26sid%3Dliteratum%253Aachs%26aulast%3DBattye%26aufirst%3DT.%2BG.%2BG.%26aulast%3DKontogiannis%26aufirst%3DL.%26aulast%3DJohnson%26aufirst%3DO.%26aulast%3DPowell%26aufirst%3DH.%2BR.%26aulast%3DLeslie%26aufirst%3DA.%2BG.%2BW.%26atitle%3DIMosflm%253A%2520a%2520new%2520graphical%2520interface%2520for%2520diffraction-image%2520processing%2520with%2520MOSFLM%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D271%26epage%3D281%26doi%3D10.1107%2FS0907444910048675" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group">Evans, P. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murshudov, G. N.</span><span> </span><span class="NLM_article-title">How good are my data and what is the resolution?</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">69</span><span class="NLM_x">, </span> <span class="NLM_fpage">1204</span><span class="NLM_x">–</span> <span class="NLM_lpage">1214</span><span class="refDoi"> DOI: 10.1107/S0907444913000061</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=10.1107%2FS0907444913000061" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=23793146" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=1%3ACAS%3A528%3ADC%252BC3sXpvVejsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2013&pages=1204-1214&author=P.+R.+Evansauthor=G.+N.+Murshudov&title=How+good+are+my+data+and+what+is+the+resolution%3F&doi=10.1107%2FS0907444913000061"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">How good are my data and what is the resolution?</span></div><div class="casAuthors">Evans, Philip R.; Murshudov, Garib N.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1204-1214</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Following integration of the obsd. diffraction spots, the process of  data redn.' initially aims to det. the point-group symmetry of the data and the likely space group.  This can be performed with the program POINTLESS.  The scaling program then puts all the measurements on a common scale, avs. measurements of symmetry-related reflections (using the symmetry detd. previously) and produces many statistics that provide the first important measures of data quality.  A new scaling program, AIMLESS, implements scaling models similar to those in SCALA but adds some addnl. analyses.  From the analyses, a no. of decisions can be made about the quality of the data and whether some measurements should be discarded.  The effective  resoln.' of a data set is a difficult and possibly contentious question (particularly with referees of papers) and this is discussed in the light of tests comparing the data-processing statistics with trials of refinement against obsd. and simulated data, and automated model-building and comparison of maps calcd. with different resoln. limits.  These trials show that adding weak high-resoln. data beyond the commonly used limits may make some improvement and does no harm.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreOyPAR8GLBrVg90H21EOLACvtfcHk0ljmJOAVM1qrRA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXpvVejsL8%253D&md5=3f649426c21e9d423f94ef8d7c568233</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1107%2FS0907444913000061&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444913000061%26sid%3Dliteratum%253Aachs%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26atitle%3DHow%2520good%2520are%2520my%2520data%2520and%2520what%2520is%2520the%2520resolution%253F%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2013%26volume%3D69%26spage%3D1204%26epage%3D1214%26doi%3D10.1107%2FS0907444913000061" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group">Winn, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ballard, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowtan, K. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dodson, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Emsley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Evans, P. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keegan, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krissinel, E. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leslie, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCoy, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McNicholas, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murshudov, G. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pannu, N. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Potterton, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Powell, H. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Read, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vagin, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilson, K. S.</span><span> </span><span class="NLM_article-title">Overview of the CCP4 suite and current developments</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">67</span><span class="NLM_x">, </span> <span class="NLM_fpage">235</span><span class="NLM_x">–</span> <span class="NLM_lpage">242</span><span class="refDoi"> DOI: 10.1107/S0907444910045749</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=10.1107%2FS0907444910045749" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=21460441" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=235-242&author=M.+D.+Winnauthor=C.+C.+Ballardauthor=K.+D.+Cowtanauthor=E.+J.+Dodsonauthor=P.+Emsleyauthor=P.+R.+Evansauthor=R.+M.+Keeganauthor=E.+B.+Krissinelauthor=A.+G.+Leslieauthor=A.+McCoyauthor=S.+J.+McNicholasauthor=G.+N.+Murshudovauthor=N.+S.+Pannuauthor=E.+A.+Pottertonauthor=H.+R.+Powellauthor=R.+J.+Readauthor=A.+Vaginauthor=K.+S.+Wilson&title=Overview+of+the+CCP4+suite+and+current+developments&doi=10.1107%2FS0907444910045749"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Overview of the CCP4 suite and current developments</span></div><div class="casAuthors">Winn, Martyn D.; Ballard, Charles C.; Cowtan, Kevin D.; Dodson, Eleanor J.; Emsley, Paul; Evans, Phil R.; Keegan, Ronan M.; Krissinel, Eugene B.; Leslie, Andrew G. W.; McCoy, Airlie; McNicholas, Stuart J.; Murshudov, Garib N.; Pannu, Navraj S.; Potterton, Elizabeth A.; Powell, Harold R.; Read, Randy J.; Vagin, Alexei; Wilson, Keith S.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  The CCP4 (Collaborative Computational Project, No. 4) software suite is a collection of programs and assocd. data and software libraries which can be used for macromol. structure detn. by X-ray crystallog.  The suite is designed to be flexible, allowing users a no. of methods of achieving their aims.  The programs are from a wide variety of sources but are connected by a common infrastructure provided by std. file formats, data objects and graphical interfaces.  Structure soln. by macromol. crystallog. is becoming increasingly automated and the CCP4 suite includes several automation pipelines.  After giving a brief description of the evolution of CCP4 over the last 30 years, an overview of the current suite is given.  While detailed descriptions are given in the accompanying articles, here it is shown how the individual programs contribute to a complete software package.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_wWMMIol6UrVg90H21EOLACvtfcHk0lifgeDjQvvrLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D&md5=c407e4d47bef46864be336d60147c17d</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1107%2FS0907444910045749&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910045749%26sid%3Dliteratum%253Aachs%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DBallard%26aufirst%3DC.%2BC.%26aulast%3DCowtan%26aufirst%3DK.%2BD.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DKeegan%26aufirst%3DR.%2BM.%26aulast%3DKrissinel%26aufirst%3DE.%2BB.%26aulast%3DLeslie%26aufirst%3DA.%2BG.%26aulast%3DMcCoy%26aufirst%3DA.%26aulast%3DMcNicholas%26aufirst%3DS.%2BJ.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DPannu%26aufirst%3DN.%2BS.%26aulast%3DPotterton%26aufirst%3DE.%2BA.%26aulast%3DPowell%26aufirst%3DH.%2BR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DWilson%26aufirst%3DK.%2BS.%26atitle%3DOverview%2520of%2520the%2520CCP4%2520suite%2520and%2520current%2520developments%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D235%26epage%3D242%26doi%3D10.1107%2FS0907444910045749" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group">McCoy, A. J.</span><span> </span><span class="NLM_article-title">Solving structures of protein complexes by molecular replacement with Phaser</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">63</span><span class="NLM_x">, </span> <span class="NLM_fpage">32</span><span class="NLM_x">–</span> <span class="NLM_lpage">41</span><span class="refDoi"> DOI: 10.1107/S0907444906045975</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=10.1107%2FS0907444906045975" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=17164524" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtlSqu77F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2007&pages=32-41&author=A.+J.+McCoy&title=Solving+structures+of+protein+complexes+by+molecular+replacement+with+Phaser&doi=10.1107%2FS0907444906045975"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Solving structures of protein complexes by molecular replacement with Phaser</span></div><div class="casAuthors">McCoy, Airlie J.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">32-41</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Mol. replacement (MR) generally becomes more difficult as the no. of components in the asym. unit requiring sep. MR models (i.e. the dimensionality of the search) increases.  When the proportion of the total scattering contributed by each search component is small, the signal in the search for each component in isolation is weak or non-existent.  Maximum-likelihood MR functions enable complex asym. units to be built up from individual components with a 'tree search with pruning' approach.  This method, as implemented in the automated search procedure of the program Phaser, has been very successful in solving many previously intractable MR problems.  However, there are a no. of cases in which the automated search procedure of Phaser is suboptimal or encounters difficulties.  These include cases where there are a large no. of copies of the same component in the asym. unit or where the components of the asym. unit have greatly varying B factors.  Two case studies are presented to illustrate how Phaser can be used to best advantage in the std. 'automated MR' mode and two case studies are used to show how to modify the automated search strategy for problematic cases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpAvs9W8hQ3l7Vg90H21EOLACvtfcHk0lifgeDjQvvrLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtlSqu77F&md5=323a5297bdf622ebaf398a87c39dc794</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1107%2FS0907444906045975&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444906045975%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26atitle%3DSolving%2520structures%2520of%2520protein%2520complexes%2520by%2520molecular%2520replacement%2520with%2520Phaser%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2007%26volume%3D63%26spage%3D32%26epage%3D41%26doi%3D10.1107%2FS0907444906045975" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group">Afonine, P. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grosse-Kunstleve, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Echols, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Headd, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moriarty, N. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mustyakimov, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Terwilliger, T. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Urzhumtsev, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zwart, P. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adams, P. D.</span><span> </span><span class="NLM_article-title">Towards automated crystallographic structure refinement with phenix.refine</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">68</span><span class="NLM_x">, </span> <span class="NLM_fpage">352</span><span class="NLM_x">–</span> <span class="NLM_lpage">367</span><span class="refDoi"> DOI: 10.1107/S0907444912001308</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=10.1107%2FS0907444912001308" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=22505256" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=1%3ACAS%3A528%3ADC%252BC38Xlt1Ggu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2012&pages=352-367&author=P.+V.+Afonineauthor=R.+W.+Grosse-Kunstleveauthor=N.+Echolsauthor=J.+J.+Headdauthor=N.+W.+Moriartyauthor=M.+Mustyakimovauthor=T.+C.+Terwilligerauthor=A.+Urzhumtsevauthor=P.+H.+Zwartauthor=P.+D.+Adams&title=Towards+automated+crystallographic+structure+refinement+with+phenix.refine&doi=10.1107%2FS0907444912001308"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Towards automated crystallographic structure refinement with phenix.refine</span></div><div class="casAuthors">Afonine, Pavel V.; Grosse-Kunstleve, Ralf W.; Echols, Nathaniel; Headd, Jeffrey J.; Moriarty, Nigel W.; Mustyakimov, Marat; Terwilliger, Thomas C.; Urzhumtsev, Alexandre; Zwart, Peter H.; Adams, Paul D.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">352-367</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phenix.refine is a program within the PHENIX package that supports crystallog. structure refinement against exptl. data with a wide range of upper resoln. limits using a large repertoire of model parameterizations.  It has several automation features and is also highly flexible.  Several hundred parameters enable extensive customizations for complex use cases.  Multiple user-defined refinement strategies can be applied to specific parts of the model in a single refinement run.  An intuitive graphical user interface is available to guide novice users and to assist advanced users in managing refinement projects.  X-ray or neutron diffraction data can be used sep. or jointly in refinement. phenix.refine is tightly integrated into the PHENIX suite, where it serves as a crit. component in automated model building, final structure refinement, structure validation and deposition to the wwPDB.  This paper presents an overview of the major phenix.refine features, with extensive literature refs. for readers interested in more detailed discussions of the methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnhn9QwIEkTbVg90H21EOLACvtfcHk0lifgeDjQvvrLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xlt1Ggu7c%253D&md5=1cc7da2cea396f1a75646ea709140cf1</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1107%2FS0907444912001308&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444912001308%26sid%3Dliteratum%253Aachs%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DMustyakimov%26aufirst%3DM.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DUrzhumtsev%26aufirst%3DA.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26atitle%3DTowards%2520automated%2520crystallographic%2520structure%2520refinement%2520with%2520phenix.refine%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2012%26volume%3D68%26spage%3D352%26epage%3D367%26doi%3D10.1107%2FS0907444912001308" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group">Emsley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowtan, K.</span><span> </span><span class="NLM_article-title">Coot: model-building tools for molecular graphics</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">60</span><span class="NLM_x">, </span> <span class="NLM_fpage">2126</span><span class="NLM_x">–</span> <span class="NLM_lpage">2132</span><span class="refDoi"> DOI: 10.1107/S0907444904019158</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=10.1107%2FS0907444904019158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=15572765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2126-2132&author=P.+Emsleyauthor=K.+Cowtan&title=Coot%3A+model-building+tools+for+molecular+graphics&doi=10.1107%2FS0907444904019158"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Coot: model-building tools for molecular graphics</span></div><div class="casAuthors">Emsley, Paul; Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2126-2132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4mg is a project that aims to provide a general-purpose tool for structural biologists, providing tools for x-ray structure soln., structure comparison and anal., and publication-quality graphics.  The map-fitting tools are available as a stand-alone package, distributed as 'Coot'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMisM42468k7Vg90H21EOLACvtfcHk0lhmgsZXQk5xsw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P&md5=1be390f3bb6fd584468499ad0921161e</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019158%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DCoot%253A%2520model-building%2520tools%2520for%2520molecular%2520graphics%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D2126%26epage%3D2132%26doi%3D10.1107%2FS0907444904019158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group">Kabsch, W.</span><span> </span><span class="NLM_article-title">Integration, scaling, space-group assignment and post-refinement</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">133</span><span class="NLM_x">–</span> <span class="NLM_lpage">144</span><span class="refDoi"> DOI: 10.1107/S0907444909047374</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=10.1107%2FS0907444909047374" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=20124693" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=133-144&author=W.+Kabsch&title=Integration%2C+scaling%2C+space-group+assignment+and+post-refinement&doi=10.1107%2FS0907444909047374"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Integration, scaling, space-group assignment and post-refinement</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">133-144</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Important steps in the processing of rotation data are described that are common to most software packages.  These programs differ in the details and in the methods implemented to carry out the tasks.  Here, the working principles underlying the data-redn. package XDS are explained, including the new features of automatic detn. of spot size and reflecting range, recognition and assignment of crystal symmetry and a highly efficient algorithm for the detn. of correction/scaling factors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGW7t3Bb1P_rVg90H21EOLACvtfcHk0lhmgsZXQk5xsw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisb4%253D&md5=eb1db34f3c959b4921771eb9f57b2159</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047374&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047374%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DIntegration%252C%2520scaling%252C%2520space-group%2520assignment%2520and%2520post-refinement%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D133%26epage%3D144%26doi%3D10.1107%2FS0907444909047374" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group">Moriarty, N. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grosse-Kunstleve, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adams, P. D.</span><span> </span><span class="NLM_article-title">electronic Ligand Builder and Optimization Workbench (eLBOW): a tool for ligand coordinate and restraint generation</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">65</span><span class="NLM_x">, </span> <span class="NLM_fpage">1074</span><span class="NLM_x">–</span> <span class="NLM_lpage">1080</span><span class="refDoi"> DOI: 10.1107/S0907444909029436</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=10.1107%2FS0907444909029436" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=19770504" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtFKnsLnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2009&pages=1074-1080&author=N.+W.+Moriartyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adams&title=electronic+Ligand+Builder+and+Optimization+Workbench+%28eLBOW%29%3A+a+tool+for+ligand+coordinate+and+restraint+generation&doi=10.1107%2FS0907444909029436"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">electronic Ligand Builder and Optimization Workbench (eLBOW): a tool for ligand coordinate and restraint generation</span></div><div class="casAuthors">Moriarty, Nigel W.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">65</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1074-1080</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The electronic Ligand Builder and Optimization Workbench (eLBOW) is a program module of the PHENIX suite of computational crystallog. software.  It is designed to be a flexible procedure that uses simple and fast quantum-chem. techniques to provide chem. accurate information for novel and known ligands alike.  A variety of input formats and options allow the attainment of a no. of diverse goals including geometry optimization and generation of restraints.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojnN4AVY_DwrVg90H21EOLACvtfcHk0lhmgsZXQk5xsw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtFKnsLnN&md5=3f9a9c6c31c9a4b014a9f151fb6aa8d9</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1107%2FS0907444909029436&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909029436%26sid%3Dliteratum%253Aachs%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26atitle%3Delectronic%2520Ligand%2520Builder%2520and%2520Optimization%2520Workbench%2520%2528eLBOW%2529%253A%2520a%2520tool%2520for%2520ligand%2520coordinate%2520and%2520restraint%2520generation%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2009%26volume%3D65%26spage%3D1074%26epage%3D1080%26doi%3D10.1107%2FS0907444909029436" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group">Chen, V. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arendall, W. B.,  3rd</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Headd, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keedy, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Immormino, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kapral, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, L. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, D. C.</span><span> </span><span class="NLM_article-title">MolProbity: all-atom structure validation for macromolecular crystallography</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">12</span><span class="NLM_x">–</span> <span class="NLM_lpage">21</span><span class="refDoi"> DOI: 10.1107/S0907444909042073</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=10.1107%2FS0907444909042073" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=20057044" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=1%3ACAS%3A528%3ADC%252BC3cXit1Kktg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=12-21&author=V.+B.+Chenauthor=W.+B.+Arendallauthor=J.+J.+Headdauthor=D.+A.+Keedyauthor=R.+M.+Immorminoauthor=G.+J.+Kapralauthor=L.+W.+Murrayauthor=J.+S.+Richardsonauthor=D.+C.+Richardson&title=MolProbity%3A+all-atom+structure+validation+for+macromolecular+crystallography&doi=10.1107%2FS0907444909042073"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">MolProbity: all-atom structure validation for macromolecular crystallography</span></div><div class="casAuthors">Chen, Vincent B.; Arendall, W. Bryan, III; Headd, Jeffrey J.; Keedy, Daniel A.; Immormino, Robert M.; Kapral, Gary J.; Murray, Laura W.; Richardson, Jane S.; Richardson, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">12-21</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">MolProbity is a structure-validation web service that provides broad-spectrum solidly based evaluation of model quality at both the global and local levels for both proteins and nucleic acids.  It relies heavily on the power and sensitivity provided by optimized hydrogen placement and all-atom contact anal., complemented by updated versions of covalent-geometry and torsion-angle criteria.  Some of the local corrections can be performed automatically in MolProbity and all of the diagnostics are presented in chart and graphical forms that help guide manual rebuilding.  X-ray crystallog. provides a wealth of biol. important mol. data in the form of at. three-dimensional structures of proteins, nucleic acids and increasingly large complexes in multiple forms and states.  Advances in automation, in everything from crystn. to data collection to phasing to model building to refinement, have made solving a structure using crystallog. easier than ever.  However, despite these improvements, local errors that can affect biol. interpretation are widespread at low resoln. and even high-resoln. structures nearly all contain at least a few local errors such as Ramachandran outliers, flipped branched protein side chains and incorrect sugar puckers.  It is crit. both for the crystallographer and for the end user that there are easy and reliable methods to diagnose and correct these sorts of errors in structures.  MolProbity is the authors' contribution to helping solve this problem and this article reviews its general capabilities, reports on recent enhancements and usage, and presents evidence that the resulting improvements are now beneficially affecting the global database.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIjgsh59dO-7Vg90H21EOLACvtfcHk0lhTzzZthRH5Og"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXit1Kktg%253D%253D&md5=b5fc7574f43f01dd6e43c3663ca4f779</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1107%2FS0907444909042073&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909042073%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DArendall%26aufirst%3DW.%2BB.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DKeedy%26aufirst%3DD.%2BA.%26aulast%3DImmormino%26aufirst%3DR.%2BM.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DMurray%26aufirst%3DL.%2BW.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26atitle%3DMolProbity%253A%2520all-atom%2520structure%2520validation%2520for%2520macromolecular%2520crystallography%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D12%26epage%3D21%26doi%3D10.1107%2FS0907444909042073" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group">Bradford, M. M.</span><span> </span><span class="NLM_article-title">A rapid and sensitive method for the quantitation of microgram quantities of protein utilizing the principle of protein-dye binding</span> <span class="citation_source-journal">Anal. Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1976</span><span class="NLM_x">, </span> <span class="NLM_volume">72</span><span class="NLM_x">, </span> <span class="NLM_fpage">248</span><span class="NLM_x">–</span> <span class="NLM_lpage">254</span><span class="refDoi"> DOI: 10.1016/0003-2697(76)90527-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=10.1016%2F0003-2697%2876%2990527-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=942051" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;key=1%3ACAS%3A528%3ADyaE28XksVehtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=1976&pages=248-254&author=M.+M.+Bradford&title=A+rapid+and+sensitive+method+for+the+quantitation+of+microgram+quantities+of+protein+utilizing+the+principle+of+protein-dye+binding&doi=10.1016%2F0003-2697%2876%2990527-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">A rapid and sensitive method for the quantitation of microgram quantities of protein utilizing the principle of protein-dye binding</span></div><div class="casAuthors">Bradford, Marion M.</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Biochemistry</span>
        (<span class="NLM_cas:date">1976</span>),
    <span class="NLM_cas:volume">72</span>
        (<span class="NLM_cas:issue">1-2</span>),
    <span class="NLM_cas:pages">248-54</span>CODEN:
                <span class="NLM_cas:coden">ANBCA2</span>;
        ISSN:<span class="NLM_cas:issn">0003-2697</span>.
    </div><div class="casAbstract">A protein detn. method that involves the binding of coomassie Brilliant Blue G 250 to protein is described.  The binding of the dye to protein causes a shift in the absorption max. of the dye from 465 to 595 nm, and it is the increase in absorption at 595 nm that is monitored.  This assay is very reproducible and rapid with the dye binding process virtually complete in ∼ 2 min with good color stability for 1 hr.  There is little or no interference from cations such as Na+ or K+ nor from carbohydrates such as sucrose.  A small amt. of color is developed in the presence of strongly alk. buffering agents, but the assay may be run accurately by the use of proper buffer controls.  The only components found to give excessive interfering color in the assay are relatively large amts. of detergents such as Na dodecyl sulfate, Triton X 100, and commercial glassware detergents.  Interference by small amts. of detergent may be eliminated by the use of proper control.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrN_T_d0trcNrVg90H21EOLACvtfcHk0lhTzzZthRH5Og"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE28XksVehtrY%253D&md5=43f388a493becc4193c7a49d9de0e1db</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1016%2F0003-2697%2876%2990527-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0003-2697%252876%252990527-3%26sid%3Dliteratum%253Aachs%26aulast%3DBradford%26aufirst%3DM.%2BM.%26atitle%3DA%2520rapid%2520and%2520sensitive%2520method%2520for%2520the%2520quantitation%2520of%2520microgram%2520quantities%2520of%2520protein%2520utilizing%2520the%2520principle%2520of%2520protein-dye%2520binding%26jtitle%3DAnal.%2520Biochem.%26date%3D1976%26volume%3D72%26spage%3D248%26epage%3D254%26doi%3D10.1016%2F0003-2697%2876%2990527-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EEL" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EEL','PDB','5EEL'); return false;">PDB: 5EEL</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5U1Q" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5U1Q','PDB','5U1Q'); return false;">PDB: 5U1Q</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5U06" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5U06','PDB','5U06'); return false;">PDB: 5U06</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5TYI" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5TYI','PDB','5TYI'); return false;">PDB: 5TYI</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2QMS" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2QMS','PDB','2QMS'); return false;">PDB: 2QMS</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i31"><a href="/doi/suppl/10.1021/acs.jmedchem.7b01320">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_37887"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.7b01320">10.1021/acs.jmedchem.7b01320</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">X-ray crystal structure of the Grb7–SH2/peptide <b>5</b> complex, SPR specificity experiments for peptides <b>3</b>–<b>5</b> binding to Grb7–SH2 domain, surface representation of peptide <b>5</b> bound to SH2 domains, SPR binding experiments for peptides <b>6</b> and <b>7</b> binding to Grb7–SH2, X-ray crystal structures of the Grb7–SH2/peptide <b>6</b> complex and the Grb7–SH2/peptide <b>7</b> complex, full Western blots and LC chromatograms and MS for peptides (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.7b01320/suppl_file/jm7b01320_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.7b01320/suppl_file/jm7b01320_si_002.csv">CSV</a>)</p></li><li><p class="inline">Chemical structure data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.7b01320/suppl_file/jm7b01320_si_003.cdx">CDX</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b01320/suppl_file/jm7b01320_si_001.pdf">jm7b01320_si_001.pdf (12.79 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b01320/suppl_file/jm7b01320_si_002.csv">jm7b01320_si_002.csv (4.42 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b01320/suppl_file/jm7b01320_si_003.cdx">jm7b01320_si_003.cdx (167.78 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Atomic coordinates and structure factors have been deposited in the RCSB protein data bank under the accession numbers <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5U1Q">5U1Q</a> (Grb7–SH2/peptide <b>5</b> complex), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5U06">5U06</a> (Grb7–SH2/peptide <b>6</b> complex), and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5TYI">5TYI</a> (Grb7–SH2/peptide <b>7</b> complex).</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.7b01320&amp;pbContext=%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2017.60.issue-22%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.7b01320%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.7b01320" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                5MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679903ca08333dac","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
