
<html lang="en"     class="pb-page"  data-request-id="59bb861e-85d7-4f56-8af4-fcd60c416eb1"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;issue:issue:10.1021/jmcmar.2019.62.issue-3;website:website:acspubs;article:article:10.1021/acs.jmedchem.8b01572;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Discovery of ERD-308 as a Highly Potent Proteolysis Targeting Chimera (PROTAC) Degrader of Estrogen Receptor (ER)" /></meta><meta name="dc.Creator" content="Jiantao  Hu" /></meta><meta name="dc.Creator" content="Biao  Hu" /></meta><meta name="dc.Creator" content="Mingliang  Wang" /></meta><meta name="dc.Creator" content="Fuming  Xu" /></meta><meta name="dc.Creator" content="Bukeyan  Miao" /></meta><meta name="dc.Creator" content="Chao-Yie  Yang" /></meta><meta name="dc.Creator" content="Mi  Wang" /></meta><meta name="dc.Creator" content="Zhaomin  Liu" /></meta><meta name="dc.Creator" content="Daniel F.  Hayes" /></meta><meta name="dc.Creator" content="Krishnapriya  Chinnaswamy" /></meta><meta name="dc.Creator" content="James  Delproposto" /></meta><meta name="dc.Creator" content="Jeanne  Stuckey" /></meta><meta name="dc.Creator" content="Shaomeng  Wang" /></meta><meta name="dc.Description" content="The estrogen receptor (ER) is a validated target for the treatment of estrogen receptor-positive (ER+) breast cancer. Here, we describe the design, synthesis, and extensive structure–activity relat..." /></meta><meta name="Description" content="The estrogen receptor (ER) is a validated target for the treatment of estrogen receptor-positive (ER+) breast cancer. Here, we describe the design, synthesis, and extensive structure–activity relat..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="January 18, 2019" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.8b01572" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2019 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.8b01572" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.8b01572" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.8b01572" /></link>
        
    
    

<title>Discovery of ERD-308 as a Highly Potent Proteolysis Targeting Chimera (PROTAC) Degrader of Estrogen Receptor (ER) | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.8b01572" /></meta><meta property="og:title" content="Discovery of ERD-308 as a Highly Potent Proteolysis Targeting Chimera (PROTAC) Degrader of Estrogen Receptor (ER)" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/large/jm-2018-015727_0017.jpeg" /></meta><meta property="og:description" content="The estrogen receptor (ER) is a validated target for the treatment of estrogen receptor-positive (ER+) breast cancer. Here, we describe the design, synthesis, and extensive structure–activity relationship (SAR) studies of small-molecule ERα degraders based on the proteolysis targeting chimeras (PROTAC) concept. Our efforts have resulted in the discovery of highly potent and effective PROTAC ER degraders, as exemplified by ERD-308 (32). ERD-308 achieves DC50 (concentration causing 50% of protein degradation) values of 0.17 and 0.43 nM in MCF-7 and T47D ER+ breast cancer cell lines, respectively, and induces &gt;95% of ER degradation at concentrations as low as 5 nM in both cell lines. Significantly, ERD-308 induces more complete ER degradation than fulvestrant, the only approved selective ER degrader (SERD), and is more effective in inhibition of cell proliferation than fulvestrant in MCF-7 cells. Further optimization of ERD-308 may lead to a new therapy for advanced ER+ breast cancer." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.8b01572"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.8b01572">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.8b01572&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.8b01572&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.8b01572&amp;href=/doi/10.1021/acs.jmedchem.8b01572" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2019</span><span class="cit-fg-volume">, 62</span><span class="cit-fg-issue">, 3</span><span class="cit-fg-pageRange">, 1420-1442</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/62/3" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.8b01558" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.8b01593" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Discovery of ERD-308 as a Highly Potent Proteolysis Targeting Chimera (PROTAC) Degrader of Estrogen Receptor (ER)</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Jiantao Hu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jiantao Hu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">The Rogel Cancer Center,  Departments of Internal Medicine, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jiantao++Hu">Jiantao Hu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Biao Hu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Biao Hu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">The Rogel Cancer Center,  Departments of Internal Medicine, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Biao++Hu">Biao Hu</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-4691-6490" title="Orcid link">http://orcid.org/0000-0002-4691-6490</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Mingliang Wang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Mingliang Wang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">The Rogel Cancer Center,  Departments of Internal Medicine, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Mingliang++Wang">Mingliang Wang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Fuming Xu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Fuming Xu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">The Rogel Cancer Center,  Departments of Internal Medicine, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Fuming++Xu">Fuming Xu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Bukeyan Miao</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Bukeyan Miao</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">The Rogel Cancer Center,  Departments of Internal Medicine, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Bukeyan++Miao">Bukeyan Miao</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Chao-Yie Yang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Chao-Yie Yang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">The Rogel Cancer Center,  Departments of Internal Medicine, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Chao-Yie++Yang">Chao-Yie Yang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Mi Wang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Mi Wang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">The Rogel Cancer Center,  Departments of Internal Medicine, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Mi++Wang">Mi Wang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Zhaomin Liu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Zhaomin Liu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">The Rogel Cancer Center,  Departments of Internal Medicine, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Zhaomin++Liu">Zhaomin Liu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Daniel F. Hayes</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Daniel F. Hayes</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">The Rogel Cancer Center,  Departments of Internal Medicine, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Daniel+F.++Hayes">Daniel F. Hayes</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Krishnapriya Chinnaswamy</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Krishnapriya Chinnaswamy</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Life Sciences Institute, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Krishnapriya++Chinnaswamy">Krishnapriya Chinnaswamy</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">James Delproposto</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">James Delproposto</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Life Sciences Institute, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=James++Delproposto">James Delproposto</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jeanne Stuckey</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jeanne Stuckey</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Life Sciences Institute, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jeanne++Stuckey">Jeanne Stuckey</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Shaomeng Wang</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Shaomeng Wang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">The Rogel Cancer Center,  Departments of Internal Medicine,  Pharmacology, University of Michigan Medical School,  Department of Medicinal Chemistry, College of Pharmacy, University of Michigan, Ann Arbor, Michigan 48109, United States</div></div><span class="conrtib-corresp"><strong>*</strong>Phone: (734)615-0362. Fax: (734)647-9647. E-mail: <a href="/cdn-cgi/l/email-protection#cdbea5aca2a0a8a3aa8db8a0a4aea5e3a8a9b8"><span class="__cf_email__" data-cfemail="f6859e97999b939891b6839b9f959ed8939283">[email protected]</span></a></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Shaomeng++Wang">Shaomeng Wang</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-8782-6950" title="Orcid link">http://orcid.org/0000-0002-8782-6950</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.8b01572&amp;href=/doi/10.1021%2Facs.jmedchem.8b01572" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2019</span></span><span class="cit-volume">, 62</span><span class="cit-issue">, 3</span><span class="cit-pageRange">, 1420–1442</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">January 18, 2019</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>9 October 2018</li><li><span class="item_label"><b>Published</b> online</span>18 January 2019</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 14 February 2019</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.8b01572" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b01572</a></div><div class="article_header-article-copyright"><strong>Copyright © 2019 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D1420%26pageCount%3D23%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DJiantao%2BHu%252C%2BBiao%2BHu%252C%2BMingliang%2BWang%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D62%26issueNum%3D3%26contentID%3Dacs.jmedchem.8b01572%26title%3DDiscovery%2Bof%2BERD-308%2Bas%2Ba%2BHighly%2BPotent%2BProteolysis%2BTargeting%2BChimera%2B%2528PROTAC%2529%2BDegrader%2Bof%2BEstrogen%2BReceptor%2B%2528ER%2529%26numPages%3D23%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D1442%26publicationDate%3DFebruary%2B2019">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.8b01572"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">7893</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">66</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.8b01572" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Discovery of ERD-308 as a Highly Potent Proteolysis Targeting Chimera (PROTAC) Degrader of Estrogen Receptor (ER)&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Jiantao&quot;,&quot;last_name&quot;:&quot;Hu&quot;},{&quot;first_name&quot;:&quot;Biao&quot;,&quot;last_name&quot;:&quot;Hu&quot;},{&quot;first_name&quot;:&quot;Mingliang&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;Fuming&quot;,&quot;last_name&quot;:&quot;Xu&quot;},{&quot;first_name&quot;:&quot;Bukeyan&quot;,&quot;last_name&quot;:&quot;Miao&quot;},{&quot;first_name&quot;:&quot;Chao-Yie&quot;,&quot;last_name&quot;:&quot;Yang&quot;},{&quot;first_name&quot;:&quot;Mi&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;Zhaomin&quot;,&quot;last_name&quot;:&quot;Liu&quot;},{&quot;first_name&quot;:&quot;Daniel&quot;,&quot;last_name&quot;:&quot;F. Hayes&quot;},{&quot;first_name&quot;:&quot;Krishnapriya&quot;,&quot;last_name&quot;:&quot;Chinnaswamy&quot;},{&quot;first_name&quot;:&quot;James&quot;,&quot;last_name&quot;:&quot;Delproposto&quot;},{&quot;first_name&quot;:&quot;Jeanne&quot;,&quot;last_name&quot;:&quot;Stuckey&quot;},{&quot;first_name&quot;:&quot;Shaomeng&quot;,&quot;last_name&quot;:&quot;Wang&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2019&quot;,&quot;month&quot;:&quot;01&quot;,&quot;day&quot;:&quot;18&quot;,&quot;issue&quot;:&quot;3&quot;,&quot;volume&quot;:&quot;62&quot;,&quot;pages&quot;:&quot;1420-1442&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.8b01572&quot;},&quot;abstract&quot;:&quot;The estrogen receptor (ER) is a validated target for the treatment of estrogen receptor-positive (ER+) breast cancer. Here, we describe the design, synthesis, and extensive structure–activity relationship (SAR) studies of small-molecule ERα degraders based on the proteolysis targeting chimeras (PROTAC) concept. Our efforts have resulted in the discovery of highly potent and effective PROTAC ER degraders, as exemplified by ERD-308 (32). ERD-308 achieves DC50 (concentration causing 50% of protein degradation) values of 0.17 and 0.43 nM in MCF-7 and T47D ER+ breast cancer cell lines, respectively, and induces &gt;95% of ER degradation at concentrations as low as 5 nM in both cell lines. Significantly, ERD-308 induces more complete ER degradation than fulvestrant, the only approved selective ER degrader (SERD), and is more effective in inhibition of cell proliferation than fulvestrant in MCF-7 cells. Further optimization of ERD-308 may lead to a new therapy for advanced ER+ breast cancer.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b01572&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b01572" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b01572&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b01572" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b01572&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b01572" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.8b01572&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b01572&amp;href=/doi/10.1021/acs.jmedchem.8b01572" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.8b01572" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.8b01572" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (7 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b01572&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.8b01572%26sid%3Dliteratum%253Aachs%26genre%3Darticle%26aulast%3DHu%26date%3D2019%26atitle%3DDiscovery%2Bof%2BERD-308%2Bas%2Ba%2BHighly%2BPotent%2BProteolysis%2BTargeting%2BChimera%2B%2528PROTAC%2529%2BDegrader%2Bof%2BEstrogen%2BReceptor%2B%2528ER%2529%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D62%26issue%3D3%26spage%3D1420%26epage%3D1442%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291781" title="Degradation">Degradation</a>,</li><li><a href="/action/doSearch?ConceptID=292258" title="Cancer">Cancer</a>,</li><li><a href="/action/doSearch?ConceptID=291011" title="High-performance liquid chromatography">High-performance liquid chromatography</a>,</li><li><a href="/action/doSearch?ConceptID=292366" title="Ligands">Ligands</a>,</li><li><a href="/action/doSearch?ConceptID=291316" title="Mixtures">Mixtures</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/62/3" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/jmcmar.2019.62.issue-3/20190214/jmcmar.2019.62.issue-3.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/medium/jm-2018-015727_0017.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/large/jm-2018-015727_0017.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01572&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">The estrogen receptor (ER) is a validated target for the treatment of estrogen receptor-positive (ER+) breast cancer. Here, we describe the design, synthesis, and extensive structure–activity relationship (SAR) studies of small-molecule ERα degraders based on the proteolysis targeting chimeras (PROTAC) concept. Our efforts have resulted in the discovery of highly potent and effective PROTAC ER degraders, as exemplified by ERD-308 (<b>32</b>). ERD-308 achieves DC<sub>50</sub> (concentration causing 50% of protein degradation) values of 0.17 and 0.43 nM in MCF-7 and T47D ER+ breast cancer cell lines, respectively, and induces >95% of ER degradation at concentrations as low as 5 nM in both cell lines. Significantly, ERD-308 induces more complete ER degradation than fulvestrant, the only approved selective ER degrader (SERD), and is more effective in inhibition of cell proliferation than fulvestrant in MCF-7 cells. Further optimization of ERD-308 may lead to a new therapy for advanced ER+ breast cancer.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_53234" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_53234" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Breast cancer (BC) is one of the most common malignancies in women worldwide. On the basis of the status of the tumor receptor, breast cancer can be further subdivided into estrogen receptor-positive (ER+), human epidermal growth factor receptor 2 (HER2)-positive (HER2+), and triple-negative subtypes.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> ER+ breast cancer occurs in approximately 80% of newly diagnosed breast cancer cases.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> As members of the nuclear receptor family, estrogen receptors ERα and ERβ are transcription factors regulating gene expression and mediating the biological effects of the estrogen. Both ERα and ERβ are widely expressed in different tissues, and ERα is considered to be the major mediator which transduces the estrogen signaling in the female reproductive tract and mammary glands.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> ERα has therefore been pursued as a promising therapeutic target in multiple pathological settings, particularly in cancer and osteoporosis, and this is highlighted by the clinical success of tamoxifen for the treatment of ER+ BC and raloxifene for the prevention and treatment of osteoporosis in postmenopausal women.<a onclick="showRef(event, 'ref4 ref5'); return false;" href="javascript:void(0);" class="ref ref4 ref5">(4,5)</a></div><div class="NLM_p">Although inhibition of estrogen synthesis by aromatase inhibitors and inhibition of ER pathway signaling by selective estrogen receptor modulators (SERM) (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>) have demonstrated considerable clinical benefits in the treatment of ER+ BC, the development of intrinsic and acquired resistance to those drug classes presents an impediment for patients with advanced and metastatic breast cancer.<a onclick="showRef(event, 'ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref6 ref7">(6,7)</a> While there are clearly multiple resistance mechanisms to aromatase inhibitors and SERMs, recent studies have demonstrated that in most of the cases of drug-resistance, continued dependence on ERα signaling for tumor growth and disease progression is retained and the ER protein remains a principal driver in ER+ metastatic breast cancer.<a onclick="showRef(event, 'ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref8 ref9">(8,9)</a></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/medium/jm-2018-015727_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/large/jm-2018-015727_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Representative SERMs and SERDs.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/large/jm-2018-015727_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01572&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Selective estrogen receptor degraders (SERD) are small molecules that target ERα for proteasome-dependent degradation. Currently, fulvestrant (<b>5</b>, <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>) is the only SERD that has been approved for the treatment of postmenopausal women with advanced ER+ breast cancer with standard endocrine therapies.<a onclick="showRef(event, 'ref10 ref11'); return false;" href="javascript:void(0);" class="ref ref10 ref11">(10,11)</a> The clinical success enjoyed by fulvestrant suggests that degradation of the ER protein is beneficial to patients with ER+ breast cancer, particularly those whose disease continues to progress after standard endocrine therapies. Because fulvestrant has poor solubility and is not orally bioavailable, it is administered clinically by a monthly intramuscular injection.<a onclick="showRef(event, 'ref12 ref13'); return false;" href="javascript:void(0);" class="ref ref12 ref13">(12,13)</a> To address the shortcomings of fulvestrant, orally bioavailable SERD molecules have been developed and a number of them, shown in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>, are currently being evaluated in clinical trials as new therapies for the treatment of ER+, metastatic BC.<a onclick="showRef(event, 'ref14 ref15 ref16 ref17 ref18 ref19'); return false;" href="javascript:void(0);" class="ref ref14 ref15 ref16 ref17 ref18 ref19">(14−19)</a></div><div class="NLM_p">The proposed mechanism of action for traditional SERDs such as fulvestrant and those shown in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a> is induction of misfolding of the ER protein, which ultimately leads to proteasome-dependent ERα protein degradation.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> The SERD molecules are typically potent and effective in inducing degradation of ER protein in ER+ breast cancer cells, but they are only able to achieve partial degradation of the ER protein.<a onclick="showRef(event, 'ref21 ref22'); return false;" href="javascript:void(0);" class="ref ref21 ref22">(21,22)</a> Consequently, we propose that novel therapeutic agents, which can achieve more complete degradation of ER, could be more efficacious than the traditional SERD molecules for the treatment of ER+ metastatic breast cancer.</div><div class="NLM_p">The proteolysis targeting chimera (PROTAC) concept was first introduced in 2001,<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> with the objective of induction of selective target protein degradation by hijacking the cellular E3 ubiquitination ligase systems.<a onclick="showRef(event, 'ref24 ref25 ref26 ref27 ref28'); return false;" href="javascript:void(0);" class="ref ref24 ref25 ref26 ref27 ref28">(24−28)</a> PROTACs are heterobifunctional small molecules containing a ligand, which binds to the target protein of interest, and another ligand for an E3 ligase system. These two ligands are tethered together by a chemical linker (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A). The PROTAC strategy has recently gained momentum due in part to the availability of potent and druglike small-molecule ligands for a number of E3 ligase systems, and it has been employed for the design of small-molecule degraders for a number of proteins.<a onclick="showRef(event, 'ref29 ref30 ref31 ref32 ref33 ref34 ref35 ref36 ref37 ref38 ref39 ref40 ref41 ref42 ref43'); return false;" href="javascript:void(0);" class="ref ref29 ref30 ref31 ref32 ref33 ref34 ref35 ref36 ref37 ref38 ref39 ref40 ref41 ref42 ref43">(29−43)</a> Recently, Naito et al. reported several PROTAC-like ERα degraders, which were named specific and nongenetic IAP-dependent protein erasers (SNIPERs).<a onclick="showRef(event, 'ref44 ref45'); return false;" href="javascript:void(0);" class="ref ref44 ref45">(44,45)</a> They designed ERα SNIPER molecules using an ERα antagonist and a ligand for inhibitors of apoptosis protein (IAPs), which are E3 ligases. However, while SNIPER ER degraders effectively induce partial degradation of the ER protein, they also induce autoubiquitylation and proteasomal degradation of the E3 ligase, the cIAP1 protein, thus limiting their degradation and therapeutic efficacy.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/medium/jm-2018-015727_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/large/jm-2018-015727_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (A) A general scheme for the design of ER degraders based upon the PROTAC concept. (B) Structures of CRBN ligand and VHL ligand. (C) Crystal structure of ERα ligand-binding domain in a complex with raloxifene (<b>1</b>) (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1ERR">1ERR</a>). (D) Predicted binding models of <i>N</i>,<i>N</i>-diethylamino analogue of raloxifene in complex with ERα LBD. Hydrogen bonds are depicted by yellow dashed lines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/large/jm-2018-015727_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01572&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">In the present study, we report our design, synthesis, and evaluation of PROTAC degraders of ERα protein by employing the cereblon/cullin 4A and VHL/cullin 2 neddylation degradation systems and several different classes of selective ER modulators/antagonists.<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> Our study has resulted in the discovery of highly potent PROTAC ERα degraders (hereafter called ER degraders) with DC<sub>50</sub> < 1 nM and <i>D</i><sub>max</sub> > 95% in MCF-7 and T47D ER+ breast cancer cells. This study lays a foundation for the development of a completely new class of therapeutics for the treatment of ER+ metastatic breast cancer.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_35363" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_35363" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Raloxifene (<b>1</b>, <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>), a highly potent SERM, has been approved for the treatment of osteoporosis and breast cancer in women and was employed in our design of PROTAC ER degraders. Both cereblon/cullin 4A and VHL/cullin 2 have been successfully used for the design of PROTAC degraders.<a onclick="showRef(event, 'ref47 ref48'); return false;" href="javascript:void(0);" class="ref ref47 ref48">(47,48)</a> Therefore, we employed phthalimide analogues to recruit cereblon and compound <b>11</b> to recruit VHL for the design of putative PROTAC ERα degraders (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B).</div><div class="NLM_p">The cocrystal structure of human ER ligand-binding domain in a complex with raloxifene shows that the piperidine ring in raloxifene is buried in the hydrophobic pocket of ERα (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>C).<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> For the synthesis of PROTAC ERα degraders, we replaced the piperidine group with an <i>N</i>,<i>N</i>-diethylamino group, similar to the <i>N</i>,<i>N</i>-dimethylamino group in tamoxifen (<b>2a</b>, <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>). Our modeling shows that one of the ethyl groups is exposed to solvent and is therefore suitable as the linking site in the design of PROTAC ER degraders (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>D).</div><div class="NLM_p">We first synthesized compounds <b>12</b> and <b>13</b> using a cereblon ligand and two different linkers and also prepared compounds <b>14</b> and <b>15</b> using the VHL ligand (<b>11</b>) and two different linkers as putative PROTAC ER degraders (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A). These four compounds (<b>12</b>–<b>15</b>) were evaluated for their ability to induce ER degradation in the MCF-7 ER+ breast cancer cell line, with fulvestrant used as the control.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/medium/jm-2018-015727_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/large/jm-2018-015727_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Degradation of ER protein by PROTACs. (A) Structures of synthetic putative ER degraders. (B) Western blotting analysis of ER protein in MCF-7 cells treated with putative PROTAC ER degraders and control compounds. Cells were treated with different compounds for 4 h, and whole cell lysates were then analyzed by Western blotting to examine the level of ER protein. The membranes were stripped and reblotted for GAPDH as the loading control. The numbers below the panel represent the ERα/GADPH ratio normalized with the DMSO control at 100.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/large/jm-2018-015727_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01572&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Western blotting data (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B) showed that, consistent with its mode of action as a potent SERD molecule, fulvestrant very effectively induces ER protein degradation in MCF-7 cells. Interestingly, while compounds <b>14</b> and <b>15</b>, designed using a VHL ligand, are very effective in inducing ER degradation with 4 h treatment at concentrations as low as 1–10 nM in MCF-7 cells, compounds <b>12</b> and <b>13</b>, designed using a cereblon ligand, are completely ineffective in inducing ER degradation at concentrations of 1–1000 nM. Although further investigation is clearly needed before it can be concluded that the cereblon/cullin 4A system is less effective or ineffective for the design of ER degraders, our data for compounds <b>14</b> and <b>15</b> clearly demonstrate that the VHL/cullin 2 system can be successfully employed for the design of potent and highly effective PROTAC ER degraders. We have therefore focused on our further design and optimization of PROTAC ER degraders using VHL ligands.</div><div class="NLM_p">Our previous studies on bromodomain and extra-terminal (BET) degraders demonstrated that the length and chemical composition of the linker in a PROTAC degrader molecule were critical in achieving effective protein degradation.<a onclick="showRef(event, 'ref48 ref50 ref51'); return false;" href="javascript:void(0);" class="ref ref48 ref50 ref51">(48,50,51)</a> We therefore synthesized a series of analogues of compound <b>15</b> with the linker length varying from 3 to 9 atoms (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A). These compounds were evaluated for their ability to induce ER degradation in MCF-7 cells at concentrations of 1, 10, and 100 nM, with compound <b>15</b>, fulvestrant (<b>5</b>), another SERD molecule RAD1901 (<b>9</b>), and raloxifene (<b>1</b>) included as controls. Our Western blotting data (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>B) showed that compounds <b>16</b> and <b>17</b> with a (CH<sub>2</sub>)<sub>3</sub> or (CH<sub>2</sub>)<sub>4</sub> linker are ineffective in inducing ER degradation at 1–100 nM concentrations. However, compounds <b>15</b>, <b>18</b>, <b>19</b>, <b>20</b>, and <b>21</b>, with linkers containing 6–9 carbon atoms, are all highly effective in inducing ER degradation at concentrations as low as 1 nM. These compounds display similar potencies, with <b>15</b> being slightly more potent than the other compounds. Fulvestrant effectively induces ER degradation at concentrations of 1, 10, and 100 nM, but RAD1901 fails to induce ER degradation at 1–100 nM. Consistent with its mode of action as a SERM, raloxifene fails to induce any ER degradation.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/medium/jm-2018-015727_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/large/jm-2018-015727_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Optimization of linker length with linear aliphatic linkers. (A) Structures of ER degraders. (B) Western blotting analysis of ER protein in the MCF-7 cells treated with indicated compounds at 1, 10, and 100 nM. MCF-7 cells were treated with different compounds for 4 h and whole cell lysates were analyzed by Western blotting to examine the level of ER protein. The membranes were stripped and reblotted for GAPDH as the loading control. The numbers below the panel represent the ERα/GADPH ratio normalized with the DMSO control at 100.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/large/jm-2018-015727_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01572&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Upon the basis of our modeling (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>D), an <i>N</i>-ethyl group in the ER antagonist portion of compound <b>15</b> binds to a hydrophobic pocket in the ER protein and presumably contributes to its binding to ER. We therefore synthesized and tested a series of analogues of <b>15</b> with different <i>N</i>-substituent alkyl groups (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A). As shown in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B, compound <b>22</b> with a <i>N</i>-methyl group is ineffective in inducing ER degradation at 1 and 10 nM and induces only moderate ER degradation at 100 nM. In contrast, compounds <b>23</b>–<b>29</b>, with an isopropyl, <i>tert</i>-butyl, cyclopropyl, cyclobutyl, cyclopentyl, cyclohexyl, or cyclobutylmethyl group, respectively, all induce essentially complete ER degradation in MCF-7 cells at concentrations of 10 and 100 nM. Upon the basis of the Western blotting data from these compounds at a 10 nM concentration, compound <b>26</b> with a cyclobutyl group is the most potent ER degrader among this series of compounds, followed by compound <b>25</b> with a cyclopropyl group, and compounds <b>15</b> and <b>24</b> with an ethyl or a <i>tert</i>-butyl group. However, in view of the fact that compound <b>15</b> possesses the lowest molecular weight (MW: 1002.3) and achieves an excellent degradation potency, we employed it as the template molecule for further optimization.</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/medium/jm-2018-015727_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/large/jm-2018-015727_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Optimization of N-substituent groups (A) Structures of ER degraders. (B) Western blotting analysis of ER protein in the MCF-7 cells treated with indicated compounds at 1, 10, and 100 nM. MCF-7 cells were treated with different compounds for 4 h, and whole cell lysates were analyzed by Western blotting to examine the level of ER protein. The membranes were stripped and reblotted for GAPDH as the loading control. The numbers below the panel represent the ERα/GADPH ratio normalized with the DMSO control at 100.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/large/jm-2018-015727_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01572&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Our previous studies showed that in addition to the linker length, the linker composition plays a key role for protein degradation potencies in our BET PROTAC molecules.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> In view of the fact that the linker in compound <b>15</b> is quite hydrophobic and that this compound displays poor aqueous solubility, we synthesized a series of new analogues with linkers to improve solubility of the resulting compounds (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A). These new analogues were evaluated together with <b>14</b> and <b>15</b> for their ability to induce ER degradation in MCF-7 cells (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>B).</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/medium/jm-2018-015727_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/large/jm-2018-015727_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Optimization of the linker composition. (A) Structures of the ER degraders. (B) Western blotting analysis of ER protein in the MCF-7 cells treated with indicated compounds at 1, 10, and 100 nM. MCF-7 cells were treated with different compounds for 4 h, and whole cell lysates were analyzed by Western blotting to examine the level of ER protein.The membranes were stripped and reblotted for GAPDH as the loading control. The numbers below the panel represent the ERα/GADPH ratio normalized with the DMSO control at 100.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/large/jm-2018-015727_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01572&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Compounds <b>14</b> and <b>30</b>–<b>32</b> with a polyethylene glycol unit (PEG) embedded in the linker are all effective in inducing ER degradation at 10 and 100 nM in MCF-7 cells. Upon the basis of the Western blotting data from these compounds tested at 1 nM, compound <b>32</b> is the most potent compound, followed by compound <b>31</b>.</div><div class="NLM_p">Compounds <b>33</b> and <b>34</b> containing a positively charged piperidinyl or piperazinyl group are much less potent than compound <b>15</b> and fail in fact to induce any ER degradation at 1 and 10 nM in MCF-7 cells. While <b>33</b> is still ineffective in inducing ER degradation at 100 nM, compound <b>34</b> at 100 nM clearly induces ER degradation.</div><div class="NLM_p">Compound <b>15</b> has a very flexible linker. We sought to determine if introduction of a conformationally rigid linker using a charge-neutral, aromatic ring can result in potent PROTAC ER degraders. This led to the design and synthesis of compounds <b>35</b>–<b>37</b>. Western blotting analysis showed that while compounds <b>35</b>–<b>37</b> effectively induce ER degradation, these three compounds are much less potent than compounds <b>15</b> and <b>32</b>.</div><div class="NLM_p">Compound <b>15</b> contains the core structure of the ER antagonist raloxifene (<b>1</b>), and we next designed and synthesized a series of ER degraders using the core structures of several different classes of selective ER modulators, including tamoxifen (<b>2a</b>), 4-hydroxytamoxifen (<b>2b</b>), bazedoxifene (<b>3</b>), and lasofoxifene (<b>4</b>). These efforts resulted in compounds <b>38</b>–<b>41</b> (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>A), which are all very effective in inducing ER degradation with compound <b>39</b>, with the tamoxifen core structure being the least potent. Our data showed that compound <b>15</b> is still more potent than compounds <b>38</b>–<b>41</b> in induction of ER degradation (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>B).</div><figure id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/medium/jm-2018-015727_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/large/jm-2018-015727_0007.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. SAR study for ER ligands. (A) Structures of ER degraders. (B) Western blotting analysis of ER protein in the MCF-7 cells treated with indicated compounds at 1, 10, and 100 nM. MCF-7 cells were treated with different compounds for 4 h, and whole cell lysates were analyzed by Western blotting to examine the level of ER protein. The membranes were stripped and reblotted for GAPDH as the loading control. The numbers below the panel represent the ERα/GADPH ratio normalized with the DMSO control at 100.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/large/jm-2018-015727_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01572&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">We next investigated the effects of the VHL ligand portion in our PROTAC ER degraders on ER degradation, focusing particularly on the benzylamine moiety because this portion is known to be critical in achieving high binding affinities to VHL (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>A).<a onclick="showRef(event, 'ref52 ref53 ref54 ref55'); return false;" href="javascript:void(0);" class="ref ref52 ref53 ref54 ref55">(52−55)</a> First, we synthesized a series of VHL ligands. We established a fluorescence polarization (FP) assay for VHL (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01572/suppl_file/jm8b01572_si_001.pdf" class="ext-link">Figure S1</a> in Supporting Information) and tested the binding affinities of VHL ligands <b>11</b> and <b>43a</b>–<b>48a</b>, with a previously reported VHL ligand (VH032)<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> included as a control (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>A, <a href="/doi/suppl/10.1021/acs.jmedchem.8b01572/suppl_file/jm8b01572_si_001.pdf" class="ext-link">Figure S2</a> in Supporting Information). Among these VHL ligands, compound <b>11</b> is twice as potent as VH032 with an IC<sub>50</sub> value of 196 nM in our FP assay. However, substitution of the 4-methylthiazole group with other smaller aliphatic groups decreases the binding affinity significantly. Among them, an ethynyl group substituent displayed acceptable binding affinity with an IC<sub>50</sub> value of 879 nM.</div><figure id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/medium/jm-2018-015727_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/large/jm-2018-015727_0008.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. SAR study for VHL ligands. (A) Structures of ER degraders and binding affinities of relative VHL ligands against VHL protein by fluorescence polarization (FP) assay. (B) Western blotting analysis of ER protein in the MCF-7 cells treated with indicated compounds at 1, 10, and 100 nM. MCF-7 cells were treated with different compounds for 4 h, and whole cell lysates were analyzed by Western blotting to examine the level of ER protein. The membranes were stripped and reblotted for GAPDH as the loading control. The numbers below the panel represent the ERα/GADPH ratio normalized with the DMSO control at 100.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/large/jm-2018-015727_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01572&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Next, we employed those VHL ligands in the design of ER degraders to investigate the influence of VHL binding affinity on ER degradation. As shown in <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>B, compound <b>42</b>, which lacks the chiral methyl group in <b>15</b>, is much less potent than <b>15</b> in inducing ER degradation. Replacement of the methylthiazole group in <b>15</b> with other hydrophobic groups, such as chloro (<b>43</b>), cyano (<b>44</b>), cyclopropyl (<b>46</b>), isopropyl (<b>47</b>), and <i>tert</i>-butyl (<b>48</b>), led to significantly reduced ER degradation. An exception is compound <b>45</b> with an ethynyl group, which displays good or moderate degradation activity at 10 and 100 nM. Collectively, our data show that, consistent with the mechanism of action of PROTAC molecules, the binding affinity of degraders to VHL protein plays an important role in induction of ER degradation.</div><div class="NLM_p">Through extensive optimization of the three structural moieties in our PROTAC ER degraders, we have obtained a number of highly potent ER PROTAC molecules in MCF-7 cells, as exemplified by <b>32</b> (ERD-308). We next performed further investigations to characterize the action of compound <b>32</b>.</div><div class="NLM_p">We first examined the ER degradation by compound <b>32</b> in a wide range of concentrations to determine its DC<sub>50</sub> (concentration to achieve 50% of protein degradation) in MCF-7 cells (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>A). Quantification of our Western blotting data showed that compound <b>32</b> achieves a DC<sub>50</sub> value of 0.17 nM in the MCF-7 cells with a 4 h treatment. Compound <b>32</b> achieves a maximum ER degradation of >95% based upon our quantification at concentrations as low as 5 nM. It is noteworthy that compound <b>32</b> induces less ER degradation at 1 μM than at lower concentrations, a known “hook” effect that has been observed previously in PROTAC degraders.<a onclick="showRef(event, 'ref56 ref57'); return false;" href="javascript:void(0);" class="ref ref56 ref57">(56,57)</a></div><figure id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/medium/jm-2018-015727_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/large/jm-2018-015727_0009.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Further characterization of compound <b>32</b> (ERD-308). (A) ERα degradation dose-response for compound <b>32</b> with a treatment time of 4 h in MCF-7 cells. ER protein was examined by Western blotting and ER protein level was quantified by densitometry and normalized to the corresponding density of GAPDH protein. (B) ERα degradation dose-response for compound <b>32</b> with a treatment time of 4 h in T47D cells. ER protein was examined by Western blotting and ER protein level was quantified by densitometry and normalized to the corresponding density of GAPDH protein. (C) Time course of ERα degradation by compound <b>32</b> (30 nM) and fulvestrant (30 nM) in the MCF-7 cells. (D) Time course of ERα degradation by compound <b>32</b> (30 nM) and fulvestrant (30 nM) in the T47D cells.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/large/jm-2018-015727_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01572&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">We also evaluated compound <b>32</b> for its ability to induce ER degradation in the T47D ER+ breast cancer cell line. As shown in <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>B, compound <b>32</b> achieves a DC<sub>50</sub> value of 0.43 nM and a maximum degradation of >95% at 5 nM. Compound <b>32</b> at 1 μM also demonstrates a hook effect in the T47D cells.</div><div class="NLM_p">Next, we examined the kinetics of ER degradation induced by compound <b>32</b> in MCF-7 cells. As shown in <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>C, at a concentration of 30 nM, compound <b>32</b> reduces >80% of the ER protein level with a 1 h treatment and achieves essentially complete ER degradation at the 3 h time-point, indicating fast kinetics. In comparison, fulvestrant, a conventional SERD, has only a modest effect on reduction of the level of ER at 1 h and achieves a maximum of approximately 90% of ER degradation after a 24 h treatment. The kinetic data obtained for <b>32</b> and fulvestrant in the T47D cells were similar to those observed in MCF-7 cells (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>D). Taken together, our data demonstrate that compound <b>32</b> displays faster degradation kinetics and more complete degradation than fulvestrant.</div><div class="NLM_p">Upon the basis of the PROTAC design, degradation of ER protein induced by the PROTAC degrader <b>32</b> should depend upon its binding to both the ER protein and the VHL protein. Furthermore, the induced ER degradation by a PROTAC ER degrader such as <b>32</b> should be proteasome-dependent. We therefore investigated the mechanism of action of ER degradation induced by <b>32</b> (<a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>). Consistent with the PROTAC design, ER degradation induced by compound <b>32</b> at a 30 nM concentration is significantly reduced by addition of 1 μM of raloxifene or 1 μM of the proteasome inhibitor carfilzomib, but raloxifene or carfilzomib alone have no effect on the ER protein levels. Interestingly, 1 μM of the VHL ligand (<b>11</b>) blocks the degradation by 30 nM of compound <b>32</b> only slightly (<a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>A). To further confirm that the degradation is VHL-dependent, we performed a dose–response experiment with VHL ligand <b>11</b>. As shown in <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>B, the degradation by compound <b>32</b> was completely blocked with 5 or 10 μM of <b>11</b>. Taken together, these mechanistic data provide clear evidence that compound <b>32</b> acts as a bona fide PROTAC ER degrader.</div><figure id="fig10" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/medium/jm-2018-015727_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/large/jm-2018-015727_0010.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. Degradation is dependent on VHL, ER, and proteasome. (A) MCF-7 cells were pretreated with VHL ligand <b>11</b> (1 μM), or ER ligand raloxifene (<b>1</b>) (1 μm), or the proteasome inhibitor carfilzomib (1 μM) for 2 h, followed by treatment with DMSO or compound <b>32</b> (30 nM) for 4 h. Then whole-cell lysates were analyzed by Western blotting. (B) MCF-7 cells were pretreated with VHL ligand <b>11</b> (+, 0.5 μM; ++, 1 μM; +++, 5 μM; ++++, 10 μM) for 2 h, followed by treatment with DMSO or compound <b>32</b> (30 nM) for 4 h. Then whole-cell lysates were analyzed by Western blotting.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/large/jm-2018-015727_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01572&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Because compound <b>32</b> is highly potent and effective in induction of ER degradation, we used the WST-8 cell proliferation assay to evaluate its ability to inhibit cell proliferation in MCF-7 cells, with raloxifene and fulvestrant included as controls (<a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">11</a></a>A). Our data showed that compound <b>32</b> is highly potent and effective in inhibition of cell proliferation and achieves an IC<sub>50</sub> value of 0.77 nM and a maximum inhibition (<i>I</i><sub>max</sub>) of 57.5% in MCF-7 cells. Fulvestrant is also very effective and potent in inhibition of cell proliferation, but it achieves an <i>I</i><sub>max</sub> value of 43.8%. Raloxifene, on the other hand, achieves an <i>I</i><sub>max</sub> value of only 34.0%. Interestingly, RAD1901, a previously reported SERD molecule,<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> achieves an <i>I</i><sub>max</sub> value of only 25.7%. Hence the more complete ER degradation achieved by compound <b>32</b> translates into a greater maximum cell growth inhibition than that of fulvestrant or RAD1901, two conventional SERD molecules. Furthermore, compound <b>32</b> achieves a much greater maximum cell growth inhibition than raloxifene, a SERM molecule. As expected, compound <b>32</b> does not exhibit the cell proliferation inhibition effects in triple-negative breast cancer cell MDA-MB-231 (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01572/suppl_file/jm8b01572_si_001.pdf" class="ext-link">Figure S3</a> in Supporting Information) and primary human mammary epithelial cells (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01572/suppl_file/jm8b01572_si_001.pdf" class="ext-link">Figure S4</a> in Supporting Information).</div><figure id="fig11" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 11</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/medium/jm-2018-015727_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/large/jm-2018-015727_0011.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 11. Cellular effects of compound <b>32</b> (ERD-308) in MCF-7 cells. (A) Cell viability assay. Cells were treated with indicated doses for 4 days. Cell viability was determined by colorimetric WST-8 assay. GI<sub>50</sub> values were calculated using the GraphPad Prism 7 software. (B) Crystal violet staining. Cells were treated with indicated doses for 5 days, and then fixed and stained with crystal violet. (C,D) qRT-PCR analysis for pGR gene and GREB1 mRNA expression in MCF-7 cells. Cells were treated with DMSO, raloxifene (<b>1</b>), fulvestrant (<b>5</b>), or compound <b>32</b> at indicated doses for 8 h. Multiple student’s <i>t</i> tests were conducted using the Holm–Sidak method. * Significant difference between the compound treated group and the vehicle treated control group (DMSO). <sup>&</sup> Significant difference between the compound treated group and the ER inhibitor (raloxifene) treated group. <sup>#</sup> Significant difference between fulvestrant and PROTAC degrader treated groups.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/large/jm-2018-015727_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01572&amp;id=fig11"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To visually evaluate the cellular effect, we conducted a crystal violet staining experiment for compound <b>32</b> at 10, 100, and 300 nM with raloxifene and fulvestrant as controls (<a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">11</a></a>B). Consistent with the WST-8 cell proliferation assay, treatment of MCF-7 cells with compound <b>32</b> reduced cell proliferation more significantly than raloxifene or fulvestrant at all three of the concentrations tested (<a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">11</a></a>B).</div><div class="NLM_p last">We next performed quantitative reverse transcription-polymerase chain reaction (qRT-PCR) analysis to evaluate the ability of compound <b>32</b> to suppress the mRNA levels of pGR and GREB1, two ER-regulated genes in MCF-7 cells. As shown in <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">11</a></a>C,D, the expression of both genes is strongly suppressed by compound <b>32</b>. Compound <b>32</b> is only slightly more effective than fulvestrant in suppressing the expression of pGR and GREB1 at both 10 and 100 nM, but it is significantly more effective than raloxifene.</div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i4">Chemistry</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_48867" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_48867" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The syntheses of the presented final compounds were outlined in <a class="ref internalNav" href="#sch3" aria-label="Schemes 3">Schemes 3</a>–<a class="ref internalNav" href="#sch5" aria-label="5">5</a>. First, key common intermediates <b>53</b> and <b>58</b> were synthesized as shown in <a class="ref internalNav" href="#sch1" aria-label="Schemes 1">Schemes 1</a> and <a class="ref internalNav" href="#sch2" aria-label="2">2</a>, respectively. The commercial 4-acetoxybenzoic acid (<b>49</b>) was converted to the acyl chloride, which, after Friedel–Crafts acylation of commercial 6-methoxy-2-(4-methoxyphenyl)benzo-[<i>b</i>]thiophene, furnished compound <b>50</b>. After deacetylation of compound <b>50</b> under basic and aqueous conditions, compound <b>51</b> was converted to the alkyl bromide, which was substituted with excess ethylamine to afford the secondary amine (<b>52</b>). Cleavage of both aryl methoxy ethers in <b>52</b> with boron tribromide furnished the dihydroxy intermediate <b>53</b>. Following the literature reported procedure,<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> the synthesis of compound <b>58</b> commenced with the <i>tert</i>-butyloxycarbonyl protection of commercial (<i>S</i>)-1-(4-bromophenyl)ethan-1-amine (<b>54</b>). Subsequent Suzuki coupling with 4-methylthiazole afforded compound <b>55</b>, and this was followed by deprotection under acidic conditions and amide coupling with commercial (2<i>S</i>,4<i>R</i>)-1-(<i>tert</i>-butoxycarbonyl)-4-hydroxypyrrolidine-2-carboxylic acid. Under the same conditions, compound <b>56</b> was deprotected, followed by amide coupling with commercial (<i>S</i>)-1-(<i>tert</i>-butoxycarbonyl)piperidine-2-carboxylic acid to afford compound <b>57</b>, which after acidic deprotection afforded compound <b>58</b>.</div><figure id="sch1" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/medium/jm-2018-015727_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/large/jm-2018-015727_0012.jpeg" id="gr12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of the Key Intermediate <b>53</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/large/jm-2018-015727_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01572&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) oxalyl chloride, DMF, DCM, 0 °C to RT, 1 h; (b) 6-methoxy-2-(4-methoxyphenyl)benzo[<i>b</i>]thiophene, AlCl<sub>3</sub>, DCM, 0 °C to RT, 1 h; (c) NaOAc, EtOH/H<sub>2</sub>O, 80 °C, 12 h; (d) 1,2-dibromoethane, Cs<sub>2</sub>CO<sub>3</sub>, MeCN, reflux, 12 h; (e) EtNH<sub>2</sub>, DIPEA, DMF, 80 °C, 12 h; (f) BBr<sub>3</sub>, DCM, 0 °C to RT, 1 h.</p></p></figure><figure id="sch2" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/medium/jm-2018-015727_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/large/jm-2018-015727_0013.jpeg" id="gr13" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of the Key Intermediate <b>58</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/large/jm-2018-015727_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01572&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) Boc<sub>2</sub>O, NaHCO<sub>3</sub>, EtOAc/H<sub>2</sub>O, 2 h; (b) 4-methylthiazole, Pd(OAc)<sub>2</sub>, KOAc, DMA, 90 °C, 12 h; (c) 4 N HCl in dioxane/MeOH, RT, 12 h; (d) (2<i>S</i>,4<i>R</i>)-1-(<i>tert</i>-butoxycarbonyl)-4-hydroxypyrrolidine-2-carboxylic acid, HATU, DIPEA, DMF, 0 °C to RT, 12 h; (e) 4 N HCl in dioxane/MeOH, RT, 12 h; (f) (<i>S</i>)-1-(<i>tert</i>-butoxycarbonyl)piperidine-2-carboxylic acid, HATU, DIPEA, DMF, 0 °C to RT, 12 h; (g) 4 N HCl in dioxane/MeOH, RT, 12 h.</p></p></figure><div class="NLM_p">As shown in <a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>, synthesis of compound <b>12</b> commenced with the conversion of commercial 2-(2-(2-(prop-2-yn-1-yloxy)ethoxy)ethoxy)ethan-1-ol (<b>59a</b>) to compound <b>60a</b> using methanesulfonyl chloride with trimethylamine as base. Nucleophilic substitution of <b>60a</b> with compound <b>53</b> under mild basic conditions afforded the <i>N</i>-substituted compound (<b>61a</b>). Sonogashira coupling of compound <b>61a</b> with the previously published compound 3-(4-iodo-1-oxoisoindolin-2-yl)piperidine-2,6-dione<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> afforded the compound <b>12</b> in high yield. Compound <b>13</b> was synthesized using the procedure described for the synthesis of compound <b>12</b> with oct-7-yn-1-ol <b>59b</b> as the starting material.</div><figure id="sch3" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/medium/jm-2018-015727_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/large/jm-2018-015727_0014.jpeg" id="gr14" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Compounds <b>12</b> and <b>13</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/large/jm-2018-015727_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01572&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) MsCl, TEA, DCM, 0 °C to RT, 1 h; (b) <b>53</b>, DIPEA, DMF, 80 °C, 12 h; (c) 3-(4-iodo-1-oxoisoindolin-2-yl)piperidine-2,6-dione, Pd(PPh<sub>3</sub>)Cl<sub>2</sub>, CuI, DMF/TEA, 80 °C, 1 h.</p></p></figure><div class="NLM_p">As shown in <a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>, diversity of the linkers commenced with the preparation of compound <b>63</b> or <b>65</b>, which are commercially available or can be prepared from <b>62</b> or <b>64</b>, respectively. The substitution reaction of compound <b>63</b> or <b>65</b> with compound <b>53</b> furnished compound <b>66</b>, which underwent acidic deprotection to afford the acid (<b>67</b>). Amide coupling between compound <b>67</b> and <b>58</b> afforded the final compounds <b>14</b>–<b>21</b> and <b>30</b>–<b>37</b> in high yields.</div><figure id="sch4" data-index="15" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/medium/jm-2018-015727_0015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/large/jm-2018-015727_0015.jpeg" id="gr15" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of Compounds <b>14</b>–<b>21</b>, <b>30</b>–<b>37</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/large/jm-2018-015727_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01572&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) TsCl or MsCl, TEA, DCM, 0 °C to RT, 1 h; (b) TFAA, <i>tert</i>-BuOH, DCM, 0 °C to RT, 12 h; (c) <b>53</b>, DIPEA, DMF, 80 °C, 12 h; (d) TFA/DCM, 0 °C to RT, 6 h; (e) <b>58</b>, HATU, DIPEA, DMF, RT, 12 h.</p></p></figure><div class="NLM_p">As shown in <a class="ref internalNav" href="#sch5" aria-label="Scheme 5">Scheme 5</a>, the common intermediate <b>51</b> was used for the SAR studies of the <i>N</i>-substituent groups. Compound <b>51</b> was first converted to the alkyl bromide, which underwent nucleophilic attack with excess primary amines furnishing compound <b>68</b>. Substitution reaction of compound <b>68</b> with <i>tert</i>-butyl 8-bromooctanoate (<b>65</b>) furnished the linker attached intermediate, which underwent boron tribromide-mediated demethylation and deprotection to afford the acid (<b>69</b>). Amide coupling between compounds <b>69</b> and <b>58</b> afforded the final compounds <b>22</b>–<b>29</b> in high yields. Compounds <b>38</b>-<b>48</b> were synthesized using the general procedure used to prepare compound <b>15</b>.</div><figure id="sch5" data-index="16" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/medium/jm-2018-015727_0016.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/large/jm-2018-015727_0016.jpeg" id="gr16" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Synthesis of Compounds <b>22</b>–<b>29</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/large/jm-2018-015727_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01572&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) 1,2-dibromoethane, Cs<sub>2</sub>CO<sub>3</sub>, MeCN, reflux, 12 h; (b) RNH<sub>2</sub>, K<sub>2</sub>CO<sub>3</sub>, MeCN, 80 °C; (c) <b>65</b>, K<sub>2</sub>CO<sub>3</sub>, DMF, 80 °C, 12 h; (d) BBr<sub>3</sub>, DCM, 0 °C to RT, 1 h; (e) <b>58</b>, HATU, DIPEA, DMF, RT, 12 h.</p></p></figure></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i10">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_40310" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_40310" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">We have designed, synthesized, and evaluated a series of PROTAC ER degraders. Through extensive optimization of three structural components in our designed ER degraders, we have successfully obtained a number of highly potent ER degraders, exemplified by <b>32</b> (ERD-308). With 4 h treatment, compound <b>32</b> achieves DC<sub>50</sub> values of 0.17 and 0.43 nM in MCF-7 and T47D cells, respectively, and induces over 95% of ER degradation at concentrations as low as 5 nM. In direct comparison, compound <b>32</b> can achieve more complete degradation than the only approved SERD molecule fulvestrant. Consistently, compound <b>32</b> achieves more complete cell growth inhibition than fulvestrant in MCF-7 cells. Our data suggest that further optimization of PROTAC ER degraders may lead to a class of completely new and effective therapeutic agents for the treatment of advanced and metastatic ER+ breast cancer.</div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i11">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_16366" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_16366" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec5_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> General Chemical Methods</h3><div class="NLM_p">Unless otherwise noted, all purchased reagents were used as received without further purification. <sup>1</sup>H NMR and <sup>13</sup>C NMR spectra were recorded on a Bruker Advance 400 MHz spectrometer. <sup>1</sup>H NMR spectra are reported in parts per million (ppm) downfield from tetramethylsilane (TMS). All <sup>13</sup>C NMR spectra are reported in ppm and obtained with <sup>1</sup>H decoupling. In the spectral data reported, the format (δ) chemical shift (multiplicity, <i>J</i> values in Hz, integration) was used with the following abbreviations: s = singlet, d = doublet, t = triplet, q = quartet, m = multiplet. MS analyses were carried out with a H2Os UPLC–mass spectrometer. The final compounds were all purified by C18 reverse phase preparative HPLC column with solvent A (0.1% TFA in H<sub>2</sub>O) and solvent B (0.1% TFA in MeCN) as eluents. The purities of all the final compounds were determined to be over 95% by UPLC-MS.</div><div id="sec5_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i13" class="anchor-spacer"></div><h4 class="article-section__title" id="_i13"> (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-2-Acetamido-3,3-dimethylbutanoyl)-4-hydroxy-<i>N</i>-((<i>S</i>)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)pyrrolidine-2-carboxamide (<b>11</b>)</h4><div class="NLM_p last">HATU (21 mg, 0.055 mmol, 1.1 equiv) was added to a mixture of compound <b>58</b> (23 mg, 0.05 mmol, 1.0 equiv), acetic acid (4 μL, 0.06 mmol, 1.2 equiv), and DIPEA (26 μL, 0.15 mmol, 3.0 equiv) in DMF (2 mL) at 0 °C under N<sub>2</sub>. The mixture was stirred at ambient temperature for 1 h, then the crude mixture was purified by reversed-phase preparative HPLC to afford the title compound as a white solid (19 mg, 80% yield). <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz) δ (ppm) 9.02 (s, 1H), 7.47–7.42 (m, 4H), 5.04–4.98 (m, 1H), 4.62–4.55 (m, 2H), 4.43–4.41 (m, 1H), 3.88 (d, <i>J</i> = 1.08 Hz, 1H), 3.74 (dd, <i>J</i> = 10.8 Hz, <i>J</i> = 4.0 Hz, 1H), 2.50 (s, 3H), 2.22–2.16 (m, 1H), 2.00 (s, 3H), 1.98–1.91 (m, 1H), 1.51 (d, <i>J</i> = 6.8 Hz, 3H), 1.05 (s, 9H). <sup>13</sup>C NMR (CD<sub>3</sub>OD, 100 MHz) δ (ppm) 173.26, 173.11, 172.28, 153.34, 148.20, 146.01, 133.91, 131.04, 130.51, 127.69, 127.52, 70.97, 60.55, 59.22, 57.97, 50.14, 38.77, 36.41, 26.99, 22.38, 22.29, 15.41. UPLC-MS (ESI<sup>+</sup>) calcd for C<sub>25</sub>H<sub>35</sub>N<sub>4</sub>O<sub>4</sub>S [M + 1]<sup>+</sup>: 487.24, found 487.43. Purity 98.5% (HPLC).</div></div><div id="sec5_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i14" class="anchor-spacer"></div><h4 class="article-section__title" id="_i14"> 3-(4-(3-Ethyl-1-(4-(6-hydroxy-2-(4-hydroxyphenyl)benzo[<i>b</i>]thiophene-3-carbonyl)phenoxy)-6,9,12-trioxa-3-azapentadec-14-yn-15-yl)-1-oxoisoindolin-2-yl)piperidine-2,6-dione (<b>12</b>)</h4><div class="NLM_p">Methanesulfonyl chloride (0.35 mL, 4.5 mmol, 1.5 equiv) and triethylamine (0.84 mL, 6.0 mmol, 2.0 equiv) were added sequentially to a solution of commercial compound 2-(2-(2-(prop-2-yn-1-yloxy)ethoxy)ethoxy)ethan-1-ol (<b>59a</b>) (565 mg, 3.0 mmol, 1.0 equiv) in DCM (10 mL) at 0 °C. The mixture was warmed to rt and stirred for 1 h. After concentration, the residue was purified by silica gel flash column chromatography with hexane:EtOAc (2:1–1:2) to afford the title compound (<b>60a</b>) as a colorless oil (710 mg, 89% yield). <sup>1</sup>H NMR (CDCl<sub>3</sub>, 400 MHz) δ (ppm) 4.21–4.18 (m, 2H), 4.01 (d, <i>J</i> = 2.4 Hz, 2H), 3.61–3.58 (m, 2H), 3.51–3.46 (m, 8H), 2.92 (s, 3H), 2.41 (t, <i>J</i> = 2.4 Hz, 1H). <sup>13</sup>C NMR (CDCl<sub>3</sub>, 100 MHz) δ (ppm) 79.71, 74.91, 70.33, 70.25, 70.09, 69.51, 68.90, 68.77, 58.09, 37.43.</div><div class="NLM_p">DIPEA (0.09 mL, 0.5 mmol, 5.0 equiv) was added to a solution of compound <b>53</b> (44 mg, 0.1 mmol, 1.0 equiv) and <b>60a</b> (40 mg, 0.15 mmol, 1.5 equiv) in DMF (3.0 mL). The solution was stirred at 100 °C for 12 h. After cooling to rt, the residue was purified by reversed-phase preparative HPLC to afford the title compound (<b>61a</b>) as a white solid (30 mg, 50% yield). UPLC-MS (ESI<sup>+</sup>) calcd for C<sub>34</sub>H<sub>38</sub>NO<sub>7</sub>S [M + 1]<sup>+</sup>: 604.24, found 604.30.</div><div class="NLM_p last">3-(4-Iodo-1-oxoisoindolin-2-yl)piperidine-2,6-dione (19 mg, 0.05 mmol, 1.0 equiv) was added to a solution of compound <b>61a</b> (30 mg, 0.05 mmol, 1.0 equiv) in DMF (2.0 mL). The solution was purged and refilled with nitrogen three times with sonication then Pd(PPh<sub>3</sub>)<sub>2</sub>Cl<sub>2</sub> (3.5 mg, 0.005 mmol, 0.1 equiv), CuI (2.0 mg, 0.01 mmol, 0.2 equiv), and trimethylamine (2.0 mL) were added sequentially. The solution was purged and refilled with N<sub>2</sub> again. The solution was stirred at 80 °C for 1 h and was then cooled to rt. EtOAc and H<sub>2</sub>O were added, and the aqueous layer was extracted with EtOAc twice. The combined organic layer was dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>. After filtration and concentration, the residue was purified by reversed-phase preparative HPLC to afford the title compound (<b>12</b>) as a yellow solid (18 mg, 43% yield). <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz) δ (ppm) 7.76–7.71 (m, 3H), 7.58 (d, <i>J</i> = 7.6 Hz, 1H), 7.46–7.40 (m, 2H), 7.25 (d, <i>J</i> = 2.0 Hz, 1H), 7.16 (d, <i>J</i> = 8.8 Hz, 2H), 6.89 (d, <i>J</i> = 8.8 Hz, 2H), 6.85 (dd, <i>J</i> = 8.8 Hz, <i>J</i> = 2.0 Hz, 1H), 6.60 (d, <i>J</i> = 8.8 Hz, 2H), 5.12 (dd, <i>J</i> = 13.2 Hz, <i>J</i> = 5.2 Hz, 1H), 4.42–4.34 (m, 6H), 3.81–3.78 (m, 2H), 3.70–3.59 (m, 10H), 3.43 (t, <i>J</i> = 4.8 Hz, 2H), 3.35 (q, <i>J</i> = 7.6 Hz, 2H), 2.90–2.81 (m, 1H), 2.74–2.68 (m, 1H), 2.37–2.30 (m, 1H), 2.11–2.05 (m, 1H), 1.32 (t, <i>J</i> = 7.6 Hz, 3H). <sup>13</sup>C NMR (CD<sub>3</sub>OD, 100 MHz) δ (ppm) 195.30, 174.53, 172.14, 170.68, 163.12, 159.25, 156.85, 145.48, 144.12, 141.47, 135.99, 134.20, 133.51, 133.10, 132.62, 131.44, 130.98, 129.77, 126.01, 124.75, 124.71, 119.34, 116.50, 116.12, 115.49, 107.94, 91.94, 82.61, 71.34, 71.24, 70.43, 65.49, 63.61, 59.69, 54.01, 53.58, 53.13, 51.37, 32.30, 24.02, 9.07. UPLC-MS (ESI<sup>+</sup>) calcd for C<sub>47</sub>H<sub>48</sub>N<sub>3</sub>O<sub>10</sub>S [M + 1]<sup>+</sup>: 846.31, found 846.52. Purity 99.1% (HPLC).</div></div><div id="sec5_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i15" class="anchor-spacer"></div><h4 class="article-section__title" id="_i15"> 3-(4-(8-(Ethyl(2-(4-(6-hydroxy-2-(4-hydroxyphenyl)benzo[<i>b</i>]thiophene-3-carbonyl)-phenoxy)ethyl)amino)oct-1-yn-1-yl)-1-oxoisoindolin-2-yl)piperidine-2,6-dione (<b>13</b>)</h4><div class="NLM_p last">This compound was prepared using a procedure similar to that used for compound <b>12</b>. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz) δ (ppm) 7.73–7.71 (m, 3H), 7.56 (d, <i>J</i> = 7.2 Hz, 1H), 7.47–7.42 (m, 2H), 7.26 (d, <i>J</i> = 2.0 Hz, 1H), 7.16 (d, <i>J</i> = 8.4 Hz, 2H), 6.91–6.86 (m, 3H), 6.60 (d, <i>J</i> = 8.8 Hz, 2H), 5.16 (dd, <i>J</i> = 13.6 Hz, <i>J</i> = 5.2 Hz, 1H), 4.50 (d, <i>J</i> = 17.2 Hz, 1H), 4.44 (d, <i>J</i> = 17.2 Hz, 1H), 4.35 (t, <i>J</i> = 4.4 Hz, 2H), 3.58 (t, <i>J</i> = 4.4 Hz, 2H), 3.32–3.30 (m, 2H), 3.21 (q, <i>J</i> = 7.6 Hz, 2H), 2.93–2.84 (m, 1H), 2.77–2.73 (m, 1H), 2.50–2.44 (m, 3H), 2.17–2.12 (m, 1H), 1.79–1.72 (m, 2H), 1.68–1.61 (m, 2H), 1.58–1.51 (m, 2H), 1.47–1.42 (m, 2H), 1.33 (t, <i>J</i> = 7.2 Hz, 3H). <sup>13</sup>C NMR (CD<sub>3</sub>OD, 100 MHz) δ (ppm) 195.28, 174.58, 172.22, 170.98, 163.03, 159.27, 156.87, 145.24, 144.38, 141.48, 135.77, 134.20, 133.51, 132.92, 132.75, 131.47, 130.99, 129.62, 126.02, 124.72, 123.74, 120.90, 116.46, 116.12, 115.38, 107.92, 97.08, 77.41, 63.59, 54.42, 53.66, 52.53, 50.31, 32.33, 29.39, 27.05, 24.68, 24.08, 19.94, 9.10. UPLC-MS (ESI<sup>+</sup>) calcd for C<sub>46</sub>H<sub>46</sub>N<sub>3</sub>O<sub>7</sub>S [M + 1]<sup>+</sup>: 784.31, found 784.27. Purity 98.9% (HPLC).</div></div><div id="sec5_1_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i16" class="anchor-spacer"></div><h4 class="article-section__title" id="_i16"> (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-17-(<i>tert</i>-Butyl)-3-ethyl-1-(4-(6-hydroxy-2-(4-hydroxyphenyl)benzo[<i>b</i>]thiophene-3-carbonyl)phenoxy)-15-oxo-6,9,12-trioxa-3,16-diazaoctadecan-18-oyl)-4-hydroxy-<i>N</i>-((<i>S</i>)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)pyrrolidine-2-carboxamide (<b>14</b>)</h4><div class="NLM_p last">This compound was prepared using a procedure similar to that used for compound <b>32</b>. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz) δ (ppm) 8.87 (s, 1H), 7.72 (d, <i>J</i> = 8.8 Hz, 2H), 7.44–7.39 (m, 5H), 7.26 (d, <i>J</i> = 2.0 Hz, 1H), 7.19 (d, <i>J</i> = 8.4 Hz, 2H), 6.89–6.85 (m, 3H), 6.63 (d, <i>J</i> = 8.4 Hz, 2H), 5.02–4.96 (m, 1H), 4.64–4.54 (m, 2H), 4.43–4.41 (m, 1H), 4.20 (t, <i>J</i> = 5.2 Hz, 2H), 3.85 (d, <i>J</i> = 11.2 Hz, 1H), 3.75–3.64 (m, 5H), 3.60–3.55 (m, 8H), 3.20–2.93 (m, 6H), 2.54–2.46 (m, 5H), 2.20–2.17 (m, 1H), 1.98–1.92 (m, 1H), 1.49 (d, <i>J</i> = 7.2 Hz, 3H), 1.67 (t, <i>J</i> = 7.2 Hz, 3H), 1.02 (s, 9H). UPLC-MS (ESI<sup>+</sup>) calcd for C<sub>57</sub>H<sub>70</sub>N<sub>5</sub>O<sub>11</sub>S<sub>2</sub> [M + 1]<sup>+</sup>: 1064.45, found 1064.74. Purity 96.4% (HPLC).</div></div><div id="sec5_1_99" class="NLM_sec NLM_sec_level_3"><div id="ac_i17" class="anchor-spacer"></div><h4 class="article-section__title" id="_i17"> (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-2-(8-(Ethyl(2-(4-(6-hydroxy-2-(4-hydroxyphenyl)benzo[<i>b</i>]thiophene-3-carbonyl)phenoxy)ethyl)amino)octanamido)-3,3-dimethylbutanoyl)-4-hydroxy-<i>N</i>-((<i>S</i>)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)pyrrolidine-2-carboxamide (<b>15</b>, ERD-148)</h4><div class="NLM_p">Trifluoroacetic anhydride (3.80 mL, 27.34 mmol, 2.0 equiv) was added at 0 °C to a solution of commercial 8-bromooctanoic acid <b>64</b> (3.05 g, 13.67 mmol, 1.0 equiv) in 50 mL of DCM. The solution was stirred at rt for 2 h. Then <i>tert</i>-butanol (3.92 mL, 41.01 mmol, 3.0 equiv) was added. The solution was stirred at rt for 12 h. Saturated aqueous NaHCO<sub>3</sub> was added. The organic layer was separated and dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>. After filtration and concentration, the residue was purified by silica gel flash column chromatography with hexane:EtOAc (20:1–5:1) to afford <i>tert</i>-butyl 8-bromooctanoate (<b>65</b>) as a colorless oil (2.48 g, 65% yield). <sup>1</sup>H NMR (CDCl<sub>3</sub>, 400 MHz) δ (ppm) 3.34 (t, <i>J</i> = 6.8 Hz, 2H), 2.14 (t, <i>J</i> = 7.6 Hz, 2H), 1.83–1.75 (m, 2H), 1.54–1.49 (m, 2H), 1.40–1.36 (m, 11H), 1.29–1.25 (m, 4H). <sup>13</sup>C NMR (CDCl<sub>3</sub>, 100 MHz) δ (ppm) 173.08, 79.88, 35.47, 33.81, 32.73, 28.86, 28.44, 28.12, 27.99, 24.95;</div><div class="NLM_p last">Compound <b>15</b> (ERD-148) was prepared using a procedure similar to that used for compound <b>32</b> with intermediate <b>65</b> instead of compound <b>63</b> as the starting material. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz) δ (ppm) 9.09 (s, 1H), 7.75 (d, <i>J</i> = 8.8 Hz, 2H), 7.47–7.41 (m, 5H), 7.27 (d, <i>J</i> = 2.4 Hz, 1H), 7.19–7.15 (m, 2H), 6.92 (d, <i>J</i> = 8.8 Hz, 2H), 6.87 (dd, <i>J</i> = 8.8 Hz, <i>J</i> = 2.4 Hz, 1H), 6.63–6.59 (m, 2H), 5.02–4.90 (m, 1H), 4.64–4.54 (m, 2H), 4.43–4.41 (m, 1H), 4.35 (t, <i>J</i> = 4.4 Hz, 2H), 3.88 (d, <i>J</i> = 11.2 Hz, 1H), 3.74 (dd, <i>J</i> = 11.2 Hz, <i>J</i> = 4.0 Hz, 1H), 3.60 (t, <i>J</i> = 4.8 Hz, 2H), 3.31–3.17 (m, 4H), 2.50 (s, 3H), 2.32–2.17 (m, 3H), 1.98–1.91 (m, 1H), 1.75–1.65 (m, 2H), 1.65–1.55 (m, 2H), 1.50 (d, <i>J</i> = 6.8 Hz, 3H), 1.43–1.29 (m, 9H), 1.03 (s, 9H). <sup>13</sup>C NMR (CD<sub>3</sub>OD, 100 MHz) δ (ppm) 193.96, 174.53, 171.84, 170.93, 161.69, 157.88, 155.47, 152.13, 144.63, 142.99, 140.09, 132.82, 132.14, 131.39, 130.07, 129.62, 129.12, 126.33, 124.64, 123.31, 115.08, 114.74, 114.01, 106.51, 69.57, 62.17, 59.21, 57.61, 56.61, 53.15, 51.19, 48.85, 48.76, 37.41, 35.10, 28.54, 28.41, 25.93, 25.65, 25.33, 23.31, 20.96, 13.89, 7.67. UPLC-MS (ESI<sup>+</sup>) calcd for C<sub>56</sub>H<sub>68</sub>N<sub>5</sub>O<sub>8</sub>S<sub>2</sub> [M + 1]<sup>+</sup>: 1002.45, found 1002.51. Purity 97.5% (HPLC).</div></div><div id="sec5_1_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i18" class="anchor-spacer"></div><h4 class="article-section__title" id="_i18"> (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-2-(4-(Ethyl(2-(4-(6-hydroxy-2-(4-hydroxyphenyl)benzo[b]thiophene-3-carbonyl)phenoxy)ethyl)amino)butanamido)-3,3-dimethylbutanoyl)-4-hydroxy-<i>N</i>-((<i>S</i>)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)pyrrolidine-2-carboxamide (<b>16</b>)</h4><div class="NLM_p last">This compound was prepared using a procedure similar to that used for compound <b>15</b>. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz) δ (ppm) 8.87 (s, 1H), 7.73 (d, <i>J</i> = 8.8 Hz, 2H), 7.45–7.39 (m, 5H), 7.26 (d, <i>J</i> = 2.0 Hz, 1H), 7.18 (d, <i>J</i> = 8.8 Hz, 1H), 6.90 (d, <i>J</i> = 8.8 Hz, 2H), 6.86 (dd, <i>J</i> = 8.8 Hz, <i>J</i> = 2.4 Hz, 1H), 6.62 (d, <i>J</i> = 8.8 Hz, 2H), 5.00–4.95 (m, 1H), 4.56–4.50 (m, 2H), 4.38–4.36 (m, 1H), 4.26 (t, <i>J</i> = 4.8 Hz, 2H), 3.83 (d, <i>J</i> = 11.2 Hz, 1H), 3.66 (dd, <i>J</i> = 10.8 Hz, <i>J</i> = 4.0 Hz, 1H), 3.06–2.99 (m, 4H), 2.47 (s, 3H), 2.43 (t, <i>J</i> = 6.4 Hz, 2H), 2.20–2.15 (m, 1H), 1.97–1.89 (m, 3H), 1.49 (d, <i>J</i> = 6.8 Hz, 3H), 1.23 (t, <i>J</i> = 7.2 Hz, 3H), 1.00 (s, 9H). UPLC-MS (ESI<sup>+</sup>) calcd for C<sub>52</sub>H<sub>60</sub>N<sub>5</sub>O<sub>8</sub>S<sub>2</sub> [M + 1]<sup>+</sup>: 946.39, found 946.41. Purity 97.4% (HPLC);</div></div><div id="sec5_1_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i19" class="anchor-spacer"></div><h4 class="article-section__title" id="_i19"> (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-2-(5-(Ethyl(2-(4-(6-hydroxy-2-(4-hydroxyphenyl)benzo[b]thiophene-3 carbonyl)phenoxy)ethyl)amino)pentanamido)-3,3-dimethylbutanoyl)-4-hydroxy-<i>N</i>-((<i>S</i>)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)pyrrolidine-2-carboxamide (<b>17</b>)</h4><div class="NLM_p last">This compound was prepared using a procedure similar to that used for compound <b>15</b>. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz) δ (ppm) 8.87 (s, 1H), 7.76–7.73 (m, 1H), 7.45–7.38 (m, 5H), 7.27 (d, <i>J</i> = 2.0 Hz, 1H), 7.20–7.14 (m, 2H), 6.94–6.92 (m, 2H), 6.89–6.85 (m, 1H), 6.62–6.59 (m, 2H), 5.01–4.97 (m, 1H), 4.59–4.52 (m, 2H), 4.43–4.41 (m, 1H), 4.36 (t, <i>J</i> = 4.8 Hz, 2H), 3.86 (d, <i>J</i> = 11.2 Hz, 1H), 3.72 (dd, <i>J</i> = 10.8 Hz, <i>J</i> = 4.0 Hz, 1H), 3.59 (t, <i>J</i> = 4.8 Hz, 2H), 3.31–3.21 (m, 4H), 2.47 (s, 3H), 2.37 (t, <i>J</i> = 6.8 Hz, 2H), 2.21–2.16 (m, 1H), 1.98–1.91 (m, 1H), 1.79–1.68 (m, 4H), 1.48 (d, <i>J</i> = 7.2 Hz, 3H), 1.34 (t, <i>J</i> = 7.2 Hz, 3H), 1.04 (s, 9H). UPLC-MS (ESI<sup>+</sup>) calcd for C<sub>53</sub>H<sub>62</sub>N<sub>5</sub>O<sub>8</sub>S<sub>2</sub> [M + 1]<sup>+</sup>: 960.40, found 960.84. Purity 96.9% (HPLC).</div></div><div id="sec5_1_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20"> (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-2-(6-(Ethyl(2-(4-(6-hydroxy-2-(4-hydroxyphenyl)benzo[<i>b</i>]thiophene-3-carbonyl)phenoxy)ethyl)amino)hexanamido)-3,3-dimethylbutanoyl)-4-hydroxy-<i>N</i>-((<i>S</i>)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)pyrrolidine-2-carboxamide (<b>18</b>)</h4><div class="NLM_p last">This compound was prepared using a procedure similar to that used for compound <b>15</b>. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz) δ (ppm) 8.87 (s, 1H), 7.73 (d, <i>J</i> = 9.2 Hz, 2H), 7.44–7.38 (m, 5H), 7.26 (d, <i>J</i> = 2.4 Hz, 1H), 7.16 (d, <i>J</i> = 8.8 Hz, 2H), 6.90 (d, <i>J</i> = 9.2 Hz, 2H), 6.87 (dd, <i>J</i> = 8.8 Hz, <i>J</i> = 2.0 Hz, 1H), 6.61 (d, <i>J</i> = 8.8 Hz, 2H), 5.03–4.96 (m, 1H), 4.61–4.54 (m, 2H), 4.41–4.39 (m, 1H), 4.35 (t, <i>J</i> = 4.8 Hz, 2H), 3.87–3.84 (m, 1H), 3.73–3.71 (m, 1H), 3.58 (t, <i>J</i> = 4.4 Hz, 2H), 3.31–3.17 (m, 4H),2.47 (s, 3H), 2.37–2.26 (m, 2H), 2.21–2.16 (m, 1H), 1.97–1.91 (m, 1H), 1.77–1.62 (m, 4H), 1.49 (d, <i>J</i> = 7.2 Hz, 3H), 1.45–1.30 (m, 5H), 1.02 (s, 9H). UPLC-MS (ESI<sup>+</sup>) calcd for C<sub>54</sub>H<sub>64</sub>N<sub>5</sub>O<sub>8</sub>S<sub>2</sub> [M + 1]<sup>+</sup>: 974.42, found 974.63. Purity 99.6% (HPLC).</div></div><div id="sec5_1_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-2-(7-(Ethyl(2-(4-(6-hydroxy-2-(4-hydroxyphenyl)benzo[b]thiophene-3-carbonyl)phenoxy)ethyl)amino)heptanamido)-3,3-dimethylbutanoyl)-4-hydroxy-<i>N</i>-((<i>S</i>)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)pyrrolidine-2-carboxamide (<b>19</b>)</h4><div class="NLM_p last">This compound was prepared using a procedure similar to that used for compound <b>15</b>. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz) δ (ppm) 8.86 (s, 1H), 7.71 (d, <i>J</i> = 8.8 Hz, 2H), 7.44–7.38 (m, 5H), 7.26 (d, <i>J</i> = 2.0 Hz, 1H), 7.17 (d, <i>J</i> = 8.8 Hz, 2H), 6.89–6.80 (m, 3H), 6.62 (d, <i>J</i> = 8.8 Hz, 2H), 5.02–4.97 (m, 1H), 4.63–4.55 (m, 2H), 4.43–4.41 (m, 1H), 4.24 (t, <i>J</i> = 4.8 Hz, 2H), 3.87 (d, <i>J</i> = 10.8 Hz, 1H), 3.74 (dd, <i>J</i> = 10.8 Hz, <i>J</i> = 4.0 Hz, 1H), 3.32–3.30 (m, 2H), 3.04–2.90 (m, 4H), 2.47 (s, 3H), 2.34–2.16 (m, 3H), 1.98–1.91 (m, 1H), 1.63–1.60 (m, 4H), 1.49 (d, <i>J</i> = 7.2 Hz, 3H), 1.37–1.35 (m, 4H), 1.25–1.18 (m, 3H), 1.01 (s, 9H). UPLC-MS (ESI<sup>+</sup>) calcd for C<sub>55</sub>H<sub>66</sub>N<sub>5</sub>O<sub>8</sub>S<sub>2</sub> [M + 1]<sup>+</sup>: 988.44, found 988.60. Purity 96.2% (HPLC).</div></div><div id="sec5_1_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-2-(9-(Ethyl(2-(4-(6-hydroxy-2-(4-hydroxyphenyl)benzo[b]thiophene-3-carbonyl)phenoxy)ethyl)amino)nonanamido)-3,3-dimethylbutanoyl)-4-hydroxy-<i>N</i>-((<i>S</i>)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)pyrrolidine-2-carboxamide (<b>20</b>)</h4><div class="NLM_p last">This compound was prepared using a procedure similar to that for compound <b>15</b>. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz) δ (ppm) 8.87 (s, 1H), 7.75–7.70 (m, 2H), 7.44–7.39 (m, 5H), 7.26 (d, <i>J</i> = 2.4 Hz, 1H), 7.18–7.15 (m, 2H), 6.90 (d, <i>J</i> = 8.8 Hz, 2H), 6.87 (dd, <i>J</i> = 8.8 Hz, <i>J</i> = 2.4 Hz, 1H), 6.61 (d, <i>J</i> = 8.8 Hz, 2H), 5.04–4.88 (m, 1H), 4.64–4.55 (m, 2H), 4.45–4.40 (m, 1H), 4.35 (t, <i>J</i> = 4.8 Hz, 2H), 3.89–3.86 (m, 1H), 3.74 (dd, <i>J</i> = 10.8 Hz, <i>J</i> = 4.0 Hz, 1H), 3.58 (t, <i>J</i> = 4.8 Hz, 2H), 3.31–3.28 (m, 2H), 3.21–3.15 (m, 2H), 2.47 (s, 3H), 2.31–2.17 (m, 3H), 1.98–1.92 (m, 1H), 1.75–1.55 (m, 4H), 1.50 (d, <i>J</i> = 7.2 Hz, 3H), 1.36–1.31 (m, 11H), 1.03 (s, 9H). UPLC-MS (ESI<sup>+</sup>) calcd for C<sub>57</sub>H<sub>70</sub>N<sub>5</sub>O<sub>8</sub>S<sub>2</sub> [M + 1]<sup>+</sup>: 1016.47, found 1016.53. Purity 95.7% (HPLC).</div></div><div id="sec5_1_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-2-(10-(Ethyl(2-(4-(6-hydroxy-2-(4-hydroxyphenyl)benzo[<i>b</i>]thiophene-3-carbonyl)phenoxy)ethyl)amino)decanamido)-3,3-dimethylbutanoyl)-4-hydroxy-<i>N</i>-((<i>S</i>)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)pyrrolidine-2-carboxamide (<b>21</b>)</h4><div class="NLM_p last">This compound was prepared using a procedure similar to that used for compound <b>15</b>. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz) δ (ppm) 8.86 (s, 1H), 7.72 (d, <i>J</i> = 8.8 Hz, 2H), 7.43–7.36 (m, 5H), 7.26 (d, <i>J</i> = 2.0 Hz, 1H), 7.17 (d, <i>J</i> = 8.8 Hz, 2H), 6.89–6.84 (m, 3H), 6.61 (d, <i>J</i> = 8.8 Hz, 2H), 5.02–4.97 (m, 1H), 4.64–4.55 (m, 2H), 4.43–4.41 (m, 1H), 4.27 (t, <i>J</i> = 4.8 Hz, 2H), 3.88 (d, <i>J</i> = 11.2 Hz, 1H), 3.74 (dd, <i>J</i> = 11.2 Hz, <i>J</i> = 4.0 Hz, 1H), 3.39 (t, <i>J</i> = 4.4 Hz, 2H), 3.15–3.09 (m, 2H), 3.03–2.98 (m, 2H), 2.46 (s, 3H), 2.33–2.17 (m, 3H), 1.98–1.92 (m, 1H), 1.69–1.50 (m, 4H), 1.49 (d, <i>J</i> = 7.2 Hz, 3H), 1.33–1.23 (m, 13H), 1.03 (s, 9H). UPLC-MS (ESI<sup>+</sup>) calcd for C<sub>58</sub>H<sub>72</sub>N<sub>5</sub>O<sub>8</sub>S<sub>2</sub> [M + 1]<sup>+</sup>: 1030.48, found 1030.46. Purity 96.4% (HPLC).</div></div><div id="sec5_1_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> (2<i>S</i>,4<i>R</i>)-4-Hydroxy-1-((<i>S</i>)-2-(8-((2-(4-(6-hydroxy-2-(4-hydroxyphenyl)benzo[<i>b</i>]thiophene-3-carbonyl)phenoxy)ethyl)(methyl)amino)octanamido)-3,3-dimethylbutanoyl)-<i>N</i>-((<i>S</i>)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)pyrrolidine-2-carboxamide (<b>22</b>)</h4><div class="NLM_p last">This compound was prepared using a procedure similar to that used for compound <b>15</b>. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz) δ (ppm) 8.95 (s, 1H), 7.74 (d, <i>J</i> = 9.2 Hz, 2H), 7.45–7.42 (m, 5H), 7.27 (d, <i>J</i> = 2.0 Hz, 1H), 7.17 (d, <i>J</i> = 8.4 Hz, 2H), 6.92 (d, <i>J</i> = 8.8 Hz, 2H), 6.87 (dd, <i>J</i> = 8.8 Hz, <i>J</i> = 2.4 Hz, 1H), 6.61 (d, <i>J</i> = 8.8 Hz, 2H), 5.02–4.97 (m, 1H), 4.64–4.54 (m, 2H), 4.42–4.37 (m, 3H), 3.87 (d, <i>J</i> = 11.2 Hz, 1H), 3.74 (dd, <i>J</i> = 11.2 Hz, <i>J</i> = 4.4 Hz, 1H), 3.66–3.48 (m, 2H), 3.23–3.13 (m, 2H), 2.93 (s, 3H), 2.48 (s, 3H), 2.34–2.17 (m, 3H), 1.98–1.91 (m, 1H), 1.80–1.70 (m, 2H), 1.63–1.55 (m, 2H), 1.50 (d, <i>J</i> = 7.2 Hz, 3H), 1.45–1.35 (m, 6H), 1.03 (s, 9H). UPLC-MS (ESI<sup>+</sup>) calcd for C<sub>55</sub>H<sub>66</sub>N<sub>5</sub>O<sub>8</sub>S<sub>2</sub> [M+1]<sup>+</sup>: 988.44, found 988.54. Purity 95.0% (HPLC).</div></div><div id="sec5_1_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> (2<i>S</i>,4<i>R</i>)-4-Hydroxy-1-((<i>S</i>)-2-(8-((2-(4-(6-hydroxy-2-(4-hydroxyphenyl)benzo[<i>b</i>]thiophene-3-carbonyl)phenoxy)ethyl)(isopropyl)amino)octanamido)-3,3-dimethylbutanoyl)-<i>N</i>-((<i>S</i>)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)pyrrolidine-2-carboxamide (<b>23</b>)</h4><div class="NLM_p last">This compound was prepared using a procedure similar to that used for compound <b>15</b>. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz) δ (ppm) 8.86 (s, 1H), 7.72 (d, <i>J</i> = 8.8 Hz, 2H), 7.43–7.38 (m, 5H), 7.26 (d, <i>J</i> = 2.0 Hz, 1H), 7.17 (d, <i>J</i> = 8.8 Hz, 2H), 6.88–6.84 (m, 3H), 6.62 (d, <i>J</i> = 8.8 Hz, 2H), 5.02–4.97 (m, 1H), 4.63–4.55 (m, 2H), 4.43–4.41 (m, 1H), 4.16 (t, <i>J</i> = 5.2 Hz, 2H), 3.87 (d, <i>J</i> = 11.2 Hz, 1H), 3.74 (dd, <i>J</i> = 10.8 Hz, <i>J</i> = 4.0 Hz, 1H), 3.37–3.35 (m, 1H), 2.86–2.82 (m, 2H), 2.46 (s, 3H), 2.31–2.16 (m, 3H), 1.98–1.92 (m, 1H), 1.59–1.56 (m, 4H), 1.49 (d, <i>J</i> = 6.8 Hz, 3H), 1.33–1.27 (m, 10H), 1.18 (d, <i>J</i> = 6.8 Hz, 6H), 1.03 (s, 9H). UPLC-MS (ESI<sup>+</sup>) calcd for C<sub>57</sub>H<sub>70</sub>N<sub>5</sub>O<sub>8</sub>S<sub>2</sub> [M + 1]<sup>+</sup>: 1016.47, found 1016.55. Purity 96.0% (HPLC).</div></div><div id="sec5_1_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-2-(8-(<i>tert</i>-Butyl(2-(4-(6-hydroxy-2-(4-hydroxyphenyl)benzo[<i>b</i>]thiophene-3-carbonyl)phenoxy)ethyl)amino)octanamido)-3,3-dimethylbutanoyl)-4-hydroxy-<i>N</i>-((<i>S</i>)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)pyrrolidine-2-carboxamide (<b>24</b>)</h4><div class="NLM_p last">This compound was prepared using a procedure similar to that used for compound <b>15</b>. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz) δ (ppm) 8.97 (s, 1H), 7.76 (d, <i>J</i> = 8.8 Hz, 2H), 7.47–7.40 (m, 5H), 7.26 (d, <i>J</i> = 2.4 Hz, 1H), 7.18 (d, <i>J</i> = 8.8 Hz, 2H), 6.92–6.86 (m, 3H), 6.62–6.60 (m, 2H), 5.03–4.97 (m, 1H), 4.62 (s, 1H), 4.59–4.55 (m, 1H), 4.45–4.41 (m, 1H), 4.32 (t, <i>J</i> = 4.4 Hz, 2H), 3.91–3.83 (m, 2H), 3.74 (dd, <i>J</i> = 11.2 Hz, <i>J</i> = 4.0 Hz, 1H), 3.49–3.38 (m, 2H), 3.14–3.10 (m, 1H), 2.48 (s, 3H), 2.33–2.17 (m, 3H), 1.99–1.92 (m, 1H), 1.83–1.73 (m, 1H), 1.70–1.55 (m, 4H), 1.51–1.47 (m, 3H), 1.47–1.45 (m, 9H), 1.39–1.29 (m, 6H), 1.03 (s, 9H). <sup>13</sup>C NMR (CD<sub>3</sub>OD, 100 MHz) δ (ppm) 195.34, 175.92, 173.23, 172.33, 163.09, 159.30, 156.89, 153.17, 148.54, 145.80, 144.45, 141.51, 134.22, 133.59, 132.77, 131.50, 131.25, 130.98, 130.51, 127.67, 127.45, 126.04, 124.72, 116.47, 116.15, 115.30, 107.91, 70.97, 67.43, 65.32, 60.63, 59.02, 58.00, 53.38, 51.70, 50.16, 49.71, 38.82, 36.50, 29.99, 29.86, 27.63, 27.53, 27.06, 26.76, 24.94, 22.37, 15.57. UPLC-MS (ESI<sup>+</sup>) calcd for C<sub>58</sub>H<sub>72</sub>N<sub>5</sub>O<sub>8</sub>S<sub>2</sub> [M + 1]<sup>+</sup>: 1030.48, found 1030.52.</div></div><div id="sec5_1_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-2-(8-(Cyclopropyl(2-(4-(6-hydroxy-2-(4-hydroxyphenyl)benzo[<i>b</i>]thiophene-3-carbonyl)phenoxy)ethyl)amino)octanamido)-3,3-dimethylbutanoyl)-4-hydroxy-<i>N</i>-((<i>S</i>)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)pyrrolidine-2-carboxamide (<b>25</b>)</h4><div class="NLM_p last">This compound was prepared using a procedure similar to that used for compound <b>15</b>. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz) δ (ppm) 9.04 (s, 1H), 7.73 (d, <i>J</i> = 9.2 Hz, 2H), 7.46–7.41 (m, 5H), 7.26 (d, <i>J</i> = 2.0 Hz, 1H), 7.16 (d, <i>J</i> = 8.8 Hz, 2H), 6.91–6.86 (m, 3H), 6.61 (d, <i>J</i> = 8.8 Hz, 2H), 5.03–4.97 (m, 1H), 4.64–4.61 (m, 1H), 4.57 (t, <i>J</i> = 8.4 Hz, 1H), 4.42–4.29 (m, 3H), 3.87 (d, <i>J</i> = 11.2 Hz, 1H), 3.76–3.65 (m, 3H), 3.32–3.30 (m, 1H), 2.89–2.84 (m, 1H), 2.49 (s, 3H), 2.33–2.17 (m, 3H), 1.99–1.92 (m, 1H), 1.85–1.75 (m, 2H), 1.63–1.55 (m, 2H), 1.50 (d, <i>J</i> = 7.2 Hz, 3H), 1.45–1.35 (m, 6H), 1.03 (s, 9H), 1.00–0.90 (m, 4H). <sup>13</sup>C NMR (CD<sub>3</sub>OD, 100 MHz) δ (ppm) 195.30, 175.92, 173.23, 172.33, 162.98, 159.26, 156.84, 153.43, 145.96, 144.57, 141.46, 134.21, 133.53, 132.77, 131.50, 131.02, 130.49, 127.70, 126.01, 124.75, 116.45, 116.14, 115.38, 107.92, 70.95, 63.22, 60.61, 59.01, 57.98, 57.85, 55.33, 50.14, 38.78, 38.54, 36.47, 29.94, 29.81, 27.42, 27.04, 26.72, 24.72, 22.34, 15.31. UPLC-MS (ESI<sup>+</sup>) calcd for C<sub>57</sub>H<sub>68</sub>N<sub>5</sub>O<sub>8</sub>S<sub>2</sub> [M + 1]<sup>+</sup>: 1014.45, found 1014.61. Purity 96.1% (HPLC).</div></div><div id="sec5_1_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-2-(8-(Cyclobutyl(2-(4-(6-hydroxy-2-(4-hydroxyphenyl)benzo[<i>b</i>]thiophene-3-carbonyl)phenoxy)ethyl)amino)octanamido)-3,3-dimethylbutanoyl)-4-hydroxy-<i>N</i>-((<i>S</i>)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)pyrrolidine-2-carboxamide (<b>26</b>)</h4><div class="NLM_p last">This compound was prepared using a procedure similar to that used for compound <b>15</b>. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz) δ (ppm) 9.04 (s, 1H), 7.74 (d, <i>J</i> = 8.8 Hz, 2H), 7.44–7.42 (m, 5H), 7.26 (d, <i>J</i> = 2.4 Hz, 1H), 7.17 (d, <i>J</i> = 8.8 Hz, 2H), 6.91–6.85 (m, 3H), 6.62 (d, <i>J</i> = 8.8 Hz, 2H), 5.03–4.97 (m, 1H), 4.64–4.55 (m, 2H), 4.45–4.41 (m, 1H), 4.32–4.29 (m, 2H), 3.91–3.86 (m, 2H), 3.74 (dd, <i>J</i> = 11.2 Hz, <i>J</i> = 4.0 Hz, 1H), 3.49 (t, <i>J</i> = 4.8 Hz, 2H), 3.09 (t, <i>J</i> = 8.8 Hz, 2H), 2.48 (s, 3H), 2.35–2.15 (m, 7H), 1.99–1.92 (m, 1H), 1.87–1.55 (m, 6H), 1.50 (d, <i>J</i> = 7.2 Hz, 3H), 1.40–1.30 (m, 6H), 1.03 (s, 9H). <sup>13</sup>C NMR (CD<sub>3</sub>OD, 100 MHz) δ (ppm) 195.32, 175.91, 173.22, 172.33, 163.01, 159.27, 156.85, 153.42, 145.95, 144.39, 141.46, 134.22, 133.54, 132.73, 131.45, 131.01, 130.49, 127.70, 125.99, 124.73, 116.49, 116.15, 115.37, 107.94, 70.95, 63.46, 60.60, 59.61, 59.01, 57.98, 52.11, 50.49, 50.14, 38.78, 36.49, 36.47, 29.94, 29.79, 27.40, 27.05, 26.71, 24.30, 22.35, 15.33, 14.18. UPLC-MS (ESI<sup>+</sup>) calcd for C<sub>58</sub>H<sub>70</sub>N<sub>5</sub>O<sub>8</sub>S<sub>2</sub> [M + 1]<sup>+</sup>: 1028.47, found 1029.18. Purity 97.7% (HPLC).</div></div><div id="sec5_1_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-2-(8-(Cyclopentyl(2-(4-(6-hydroxy-2-(4-hydroxyphenyl)benzo[<i>b</i>]thiophene-3-carbonyl)phenoxy)ethyl)amino)octanamido)-3,3-dimethylbutanoyl)-4-hydroxy-<i>N</i>-((<i>S</i>)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)pyrrolidine-2-carboxamide (<b>27</b>)</h4><div class="NLM_p last">This compound was prepared using a procedure similar to that used for compound <b>15</b>. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz) δ (ppm) 8.93 (s, 1H), 7.75 (d, <i>J</i> = 8.8 Hz, 2H), 7.45–7.41 (m, 5H), 7.26 (d, <i>J</i> = 2.4 Hz, 1H), 7.18 (d, <i>J</i> = 8.8 Hz, 2H), 6.92–6.85 (m, 3H), 6.61 (d, <i>J</i> = 8.8 Hz, 2H), 5.02–4.98 (m, 1H), 4.64–4.55 (m, 2H), 4.43–4.41 (m, 1H), 4.37–4.33 (m, 2H), 3.89–3.73 (m, 3H), 3.65–3.55 (m, 2H), 3.22 (t, <i>J</i> = 8.4 Hz, 2H), 2.48 (s, 3H), 2.30–2.15 (m, 5H), 2.03–1.94 (m, 1H), 1.84–1.57 (m, 10H), 1.50 (d, <i>J</i> = 7.2 Hz, 3H), 1.45–1.30 (m, 6H), 1.03 (s, 9H). <sup>13</sup>C NMR (CD<sub>3</sub>OD, 100 MHz) δ (ppm) 195.34, 175.89, 173.21, 172.34, 163.06, 159.29, 156.87, 153.06, 148.67, 145.73, 144.37, 141.49, 134.22, 133.55, 132.78, 131.46, 131.32, 131.01, 130.49, 127.65, 127.42, 126.03, 124.71, 116.48, 116.14, 115.36, 107.92, 70.96, 67.31, 63.94, 60.62, 59.00, 57.99, 54.01, 52.09, 38.80, 36.50, 29.97, 29.84, 29.20, 29.14, 27.39, 27.05, 26.72, 24.85, 24.81, 24.66, 22.34, 15.61. UPLC-MS (ESI<sup>+</sup>) calcd for C<sub>59</sub>H<sub>72</sub>N<sub>5</sub>O<sub>8</sub>S<sub>2</sub> [M + 1]<sup>+</sup>: 1042.48, found 1042.39. Purity >99.5% (HPLC).</div></div><div id="sec5_1_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-2-(8-(Cyclohexyl(2-(4-(6-hydroxy-2-(4-hydroxyphenyl)benzo[<i>b</i>]thiophene-3-carbonyl)phenoxy)ethyl)amino)octanamido)-3,3-dimethylbutanoyl)-4-hydroxy-<i>N</i>-((<i>S</i>)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)pyrrolidine-2-carboxamide (<b>28</b>)</h4><div class="NLM_p last">This compound was prepared using a procedure similar to that used for compound <b>15</b>. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz) δ (ppm) 8.87 (s, 1H), 7.75 (d, <i>J</i> = 8.8 Hz, 2H), 7.45–7.39 (m, 5H), 7.26 (d, <i>J</i> = 2.0 Hz, 1H), 7.18 (d, <i>J</i> = 8.4 Hz, 2H), 6.91–6.84 (m, 3H), 6.62–6.60 (m, 2H), 5.02–4.97 (m, 1H), 4.64–4.54 (m, 2H), 4.45–4.41 (m, 1H), 4.32 (t, <i>J</i> = 4.4 Hz, 2H), 3.89–3.85 (m, 1H), 3.74 (dd, <i>J</i> = 11.2 Hz, <i>J</i> = 4.0 Hz, 1H), 3.59–3.45 (m, 2H), 3.18–3.08 (m, 2H), 2.47 (s, 3H), 2.34–2.16 (m, 3H), 2.03–1.90 (m, 5H), 1.74–1.31 (m, 20H), 1.03 (s, 9H). <sup>13</sup>C NMR (CD<sub>3</sub>OD, 100 MHz) δ (ppm) 195.37, 175.94, 173.22, 172.33, 163.29, 159.29, 156.87, 152.87, 149.07, 145.61, 144.35, 141.49, 134.23, 133.56, 133.34, 131.53, 131.47, 131.03, 130.50, 127.62, 127.41, 126.04, 116.47, 116.13, 115.36, 107.91, 70.96, 60.62, 59.01, 58.00, 53.41, 51.06, 50.15, 49.28, 38.81, 37.63, 36.50, 30.00, 29.89, 27.58, 27.05, 26.20, 26.19, 26.18, 26.17, 26.15, 26.14, 22.36, 15.79. UPLC-MS (ESI<sup>+</sup>) calcd for C<sub>60</sub>H<sub>74</sub>N<sub>5</sub>O<sub>8</sub>S<sub>2</sub> [M + 1]<sup>+</sup>: 1056.50, found 1056.54.</div></div><div id="sec5_1_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-2-(8-((Cyclobutylmethyl)(2-(4-(6-hydroxy-2-(4-hydroxyphenyl)benzo-[<i>b</i>]thiophene-3-carbonyl)phenoxy)ethyl)amino)octanamido)-3,3-dimethylbutanoyl)-4-hydroxy-<i>N</i>-((<i>S</i>)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)pyrrolidine-2-carboxamide (<b>29</b>)</h4><div class="NLM_p last">This compound was prepared using a procedure similar to that used for compound <b>15</b>. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz) δ (ppm) 8.98 (s, 1H), 7.73 (d, <i>J</i> = 8.8 Hz, 2H), 7.44–7.36 (m, 5H), 7.26 (d, <i>J</i> = 2.0 Hz, 1H), 7.17 (d, <i>J</i> = 8.4 Hz, 2H), 6.90–6.85 (m, 3H), 6.61(d, <i>J</i> = 8.4 Hz, 2H), 5.02–4.97 (m, 1H), 4.64–4.55 (m, 2H), 4.45–4.41 (m, 1H), 4.33 (t, <i>J</i> = 4.4 Hz, 2H), 3.87 (d, <i>J</i> = 11.2 Hz, 1H), 3.74 (dd, <i>J</i> = 11.2 Hz, <i>J</i> = 4.0 Hz, 1H), 3.54–3.52 (m, 2H), 3.28–3.25 (m, 2H), 3.16–3.11 (m, 2H), 2.80–2.73 (m, 1H), 2.48 (s, 3H), 2.32–2.14 (m, 5H), 2.04–1.84 (m, 5H), 1.75–1.65 (m, 2H), 1.65–1.55 (m, 2H), 1.49 (d, <i>J</i> = 7.2 Hz, 3H), 1.42–1.30 (m, 6H), 1.03 (s, 9H). <sup>13</sup>C NMR (CD<sub>3</sub>OD, 100 MHz) δ (ppm) 195.33, 175.89, 173.20, 172.32, 163.05, 159.27, 156.84, 153.25, 148.24, 145.84, 144.30, 141.45, 134.22, 133.52, 132.72, 131.43, 131.07, 131.01, 130.48, 127.67, 125.97, 124.72, 116.50, 116.14, 115.39, 107.95, 70.94, 63.50, 60.60, 60.11, 59.00, 57.98, 55.21, 51.06, 50.13, 38.78, 36.48, 31.78, 29.94, 29.81, 28.20, 28.09, 27.35, 27.05, 26.71, 24.52, 22.34, 19.42, 15.46. UPLC-MS (ESI<sup>+</sup>) calcd for C<sub>59</sub>H<sub>72</sub>N<sub>5</sub>O<sub>8</sub>S<sub>2</sub> [M + 1]<sup>+</sup>: 1042.48, found 1042.82. Purity >99.5% (HPLC).</div></div><div id="sec5_1_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-14-(<i>tert</i>-Butyl)-3-ethyl-1-(4-(6-hydroxy-2-(4-hydroxyphenyl)benzo[<i>b</i>]-thiophene-3-carbonyl)phenoxy)-12-oxo-6,9-dioxa-3,13-diazapentadecan-15-oyl)-4-hydroxy-<i>N</i>-((<i>S</i>)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)pyrrolidine-2-carboxamide (<b>30</b>)</h4><div class="NLM_p last">This compound was prepared using a procedure similar to that used for compound <b>32</b>. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz) δ (ppm) 8.86 (s, 1H), 7.74 (d, <i>J</i> = 8.8 Hz, 2H), 7.44–7.37 (m, 5H), 7.27 (d, <i>J</i> = 2.4 Hz, 1H), 7.18 (d, <i>J</i> = 8.8 Hz, 2H), 6.93 (d, <i>J</i> = 8.8 Hz, 2H), 6.87 (dd, <i>J</i> = 8.8 Hz, <i>J</i> = 2.4 Hz, 1H), 6.62 (d, <i>J</i> = 8.4 Hz, 2H), 5.02–4.95 (m, 1H), 4.57–4.53 (m, 2H), 4.40–4.37 (m, 3H), 3.85–3.80 (m, 3H), 3.73–3.58 (m, 9H), 3.45 (t, <i>J</i> = 8.8 Hz, 2H), 3.37 (q, <i>J</i> = 7.2 Hz, 2H), 2.55–2.44 (m, 5H), 2.22–2.17 (m, 1H), 1.97–1.91 (m, 1H), 1.48 (d, <i>J</i> = 7.2 Hz, 3H), 1.35 (t, <i>J</i> = 7.2 Hz, 3H), 1.01 (s, 9H). UPLC-MS (ESI<sup>+</sup>) calcd for C<sub>55</sub>H<sub>66</sub>N<sub>5</sub>O<sub>10</sub>S<sub>2</sub> [M + 1]<sup>+</sup>: 1020.43, found 1020.77. Purity 97.2% (HPLC).</div></div><div id="sec5_1_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-2-(<i>tert</i>-Butyl)-12-ethyl-14-(4-(6-hydroxy-2-(4-hydroxyphenyl)benzo[<i>b</i>]-thiophene-3-carbonyl)phenoxy)-4-oxo-6,9-dioxa-3,12-diazatetradecanoyl)-4-hydroxy-<i>N</i>-((<i>S</i>)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)pyrrolidine-2-carboxamide (<b>31</b>)</h4><div class="NLM_p last">This compound was prepared using a procedure similar to that used for compound <b>32</b>. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz) δ (ppm) 9.11 (s, 1H), 7.73 (d, <i>J</i> = 8.4 Hz, 2H), 7.44–7.37 (m, 5H), 7.26 (d, <i>J</i> = 2.0 Hz, 1H), 7.17 (d, <i>J</i> = 8.8 Hz, 2H), 6.92–6.84 (m, 3H), 6.61 (d, <i>J</i> = 8.4 Hz, 2H), 4.97–4.91 (m, 1H), 4.71–4.68 (m, 1H), 4.57–4.54 (m, 1H), 4.41–4.38 (m, 3H), 4.02–3.40 (m, 16H), 2.48 (s, 3H), 2.36–2.20 (m, 1H), 1.95–1.89 (m, 1H), 1.45 (d, <i>J</i> = 7.2 Hz, 2H), 1.37 (t, <i>J</i> = 7.6 Hz, 3H), 1.01 (s, 9H). UPLC-MS (ESI<sup>+</sup>) calcd for C<sub>54</sub>H<sub>64</sub>N<sub>5</sub>O<sub>10</sub>S<sub>2</sub> [M + 1]<sup>+</sup>: 1006.41, found 1006.66. Purity 95.1% (HPLC).</div></div><div id="sec5_1_98" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-2-(2-((5-(Ethyl(2-(4-(6-hydroxy-2-(4-hydroxyphenyl)benzo[<i>b</i>]thiophene-3-carbonyl)phenoxy)ethyl)amino)pentyl)oxy)acetamido)-3,3-dimethylbutanoyl)-4-hydroxy-<i>N</i>-((<i>S</i>)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)pyrrolidine-2-carboxamide (<b>32</b>, ERD-308)</h4><div class="NLM_p">Sodium hydroxide (4.0 g, 100.0 mmol, 10.0 equiv) and tetrabutyl ammonium chloride (2.78 g, 10.0 mmol, 1.0 equiv) were added sequentially to a solution of 5-(benzyloxy)pentan-1-ol (1.94 g, 10.0 mmol, 1.0 equiv) and <i>tert</i>-butyl 2-bromoacetate (3.90 g, 20.0 mmol, 2.0 equiv) in H<sub>2</sub>O (20 mL) and DCM (20 mL). The solution was stirred vigorously at rt overnight when TLC showed that the reaction was complete. The mixture was partitioned between DCM (100 mL) and H<sub>2</sub>O (100 mL), and the organic layer was collected, washed with brine (100 mL), dried over anhydrous sodium sulfate, and concentrated under reduced pressure to give a residue that was purified by silica gel flash column chromatography with hexane:EtOAc (10:1–5:1) to afford <i>tert</i>-butyl 2-((5-(benzyloxy)pentyl)oxy)acetate as a colorless oil (987 mg, 32% yield).</div><div class="NLM_p">A mixture of <i>tert</i>-butyl 2-((5-(benzyloxy)pentyl)oxy)acetate (770 mg, 2.5 mmol, 1.0 equiv) and 10 wt % palladium on carbon (100 mg) in MeOH (20 mL) was stirred at rt overnight under a hydrogen atmosphere. TLC showed that the reaction was complete and the solution was filtered through Celite and washed with MeOH. The combined filtrate was concentrated under reduced pressure. The residue was purified by silica gel flash column chromatography with hexane:EtOAc (2:1–1:1) to afford <i>tert</i>-butyl 2-((5-hydroxypentyl)oxy)acetate (<b>62</b>) as a colorless oil (671 mg, 95% yield). <sup>1</sup>H NMR (CDCl<sub>3</sub>, 400 MHz) δ (ppm) 3.58 (s, 2H), 3.21 (t, <i>J</i> = 6.8 Hz, 2H), 3.15 (t, <i>J</i> = 6.8 Hz, 2H), 1.31–1.14 (m, 4H), 1.12–1.05 (m, 12H). <sup>13</sup>C NMR (CDCl<sub>3</sub>, 100 MHz) δ (ppm) 169.51, 80.94, 71.18, 68.20, 61.54, 31.96, 28.97, 27.67, 21.95.</div><div class="NLM_p">4-Toluenesulfonyl chloride (879 mg, 4.6 mmol, 1.5 equiv) and Et<sub>3</sub>N (0.86 mL, 6.14 mmol, 2.0 equiv) were added sequentially to a solution of <i>tert</i>-butyl 2-((5-hydroxypentyl)oxy)acetate (<b>62</b>) (671 mg, 3.07 mmol, 1.0 equiv) in DCM (10 mL) at 0 °C. The mixture was warmed to rt and stirred for 1 h. After concentration, the residue was purified by silica gel flash column chromatography with hexane:EtOAc (5:1–2:1) to afford the intermediate <i>tert</i>-butyl 2-((5-(tosyloxy)pentyl)oxy)acetate (<b>63</b>) as a colorless oil (1.02 g, 89% yield). <sup>1</sup>H NMR (CDCl<sub>3</sub>, 400 MHz) δ (ppm) 7.75 (d, <i>J</i> = 8.0 Hz, 2H), 7.32 (d, <i>J</i> = 8.0 Hz, 2H), 3.99 (t, <i>J</i> = 6.4 Hz, 2H), 3.88 (s, 2H), 3.43 (t, <i>J</i> = 6.4 Hz, 2H), 2.42 (s, 3H), 1.68–1.61 (m, 2H), 1.57–1.50 (m, 2H), 1.44 (s, 9H), 1.42–1.36 (m, 2H). <sup>13</sup>C NMR (CDCl<sub>3</sub>, 100 MHz) δ (ppm) 169.76, 144.75, 133.16, 129.89, 127.92, 81.54, 71.19, 70.55, 68.76, 28.99, 28.65, 28.15, 22.07, 21.67. UPLC-MS (ESI<sup>+</sup>) calcd for C<sub>18</sub>H<sub>28</sub>NaO<sub>6</sub>S [M + 23]<sup>+</sup>: 395.15, found 395.36.</div><div class="NLM_p">DIPEA (0.18 mL, 1.0 mmol, 5.0 equiv) was added to a solution of intermediate <b>53</b> (87 mg, 0.2 mmol, 1.0 equiv) and <i>tert</i>-butyl 2-((5-(tosyloxy)pentyl)oxy)acetate <b>63</b> (223 mg, 0.6 mmol, 3.0 equiv) in DMF (3.0 mL). The solution was stirred at 80 °C for 12 h. After cooling to temperature, the solution was diluted with EtOAc and H<sub>2</sub>O. The organic layer was separated and dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>. After filtration and concentration, the residue was purified by silica gel flash column chromatography with DCM:MeOH (10:1) to afford the intermediate (<b>66</b>) as colorless oil (114 mg, 90% yield). <sup>1</sup>H NMR (CDCl<sub>3</sub>, 400 MHz) δ (ppm) 7.65 (d, <i>J</i> = 8.8 Hz, 2H), 7.42 (d, <i>J</i> = 7.2 Hz, 1H), 7.19 (s, 1H), 7.08 (d, <i>J</i> = 8.0 Hz, 2H), 6.81 (d, <i>J</i> = 9.2 Hz, 1H), 6.59–6.54 (m, 4H), 3.99–3.95 (m, 2H), 3.92 (s, 2H), 3.48–3.40 (m, 4H), 2.86–2.82 (m, 2H), 2.64 (q, <i>J</i> = 6.8 Hz, 2H), 2.55–2.51 (m, 2H), 1.59–1.52 (m, 2H), 1.44 (s, 9H), 1.31–1.25 (m, 2H), 1.01 (t, <i>J</i> = 6.8 Hz, 3H). <sup>13</sup>C NMR (CDCl<sub>3</sub>, 100 MHz) δ (ppm) 194.16, 170.28, 162.73, 157.39, 154.83, 143.60, 140.09, 133.37, 132.59, 130.47, 129.90, 125.01, 124.06, 116.02, 115.57, 114.20, 107.67, 82.05, 71.67, 68.75, 53.54, 53.32, 51.72, 47.86, 29.78, 29.38, 28.18, 24.01, 10.64. UPLC-MS (ESI<sup>+</sup>) calcd for C<sub>36</sub>H<sub>44</sub>NO<sub>7</sub>S [M + 23]<sup>+</sup>: 634.28, found 634.18.</div><div class="NLM_p">Trifluoroacetic acid (5.0 mL) was added to a solution of intermediate <b>66</b> (114 mg, 0.18 mmol) in DCM (10 mL) at 0 °C. The solution was stirred at rt for 6 h. After concentration, the residue was purified by reversed-phase preparative HPLC to afford the title compound (<b>67</b>) as a slightly yellow solid (81 mg, 78% yield). UPLC-MS (ESI<sup>+</sup>) calcd for C<sub>32</sub>H<sub>36</sub>NO<sub>7</sub>S [M + 23]<sup>+</sup>: 578.22, found 578.06.</div><div class="NLM_p last">HATU (53 mg, 0.14 mmol, 1.0 equiv) was added to a solution of intermediate <b>67</b> (81 mg, 0.14 mmol, 1.0 equiv), intermediate <b>58</b> (67 mg, 0.15 mmol, 1.1 equiv), and DIPEA (0.12 mL, 0.70 mmol, 5.0 equiv) in DMF (2 mL). The mixture was stirred at ambient temperature for 1 h then was purified by reversed-phase preparative HPLC to afford the title compound <b>32</b> (ERD-308) as a yellow solid (56 mg, 40% yield). <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz) δ (ppm) 8.86 (s, 1H), 7.74 (d, <i>J</i> = 9.2 Hz, 2H), 7.43–7.35 (m, 5H), 7.26 (d, <i>J</i> = 2.0 Hz, 1H), 7.18 (d, <i>J</i> = 8.4 Hz, 2H), 6.91–6.85 (m, 3H), 6.62 (d, <i>J</i> = 8.4 Hz, 2H), 4.98–4.95 (m, 1H), 4.89 (s, 2H), 4.69–4.64 (m, 1H), 4.59–4.53 (m, 1H), 4.45–4.41 (m, 1H), 4.31 (t, <i>J</i> = 4.8 Hz, 2H), 4.02–3.92 (m, 2H), 3.84 (d, <i>J</i> = 11.2 Hz, 1H), 3.74 (dd, <i>J</i> = 10.8 Hz, <i>J</i> = 3.6 Hz, 1H), 3.56 (t, <i>J</i> = 6.8 Hz, 2H), 3.45 (t, <i>J</i> = 4.8 Hz, 2H), 3.11–3.07 (m, 2H), 2.47 (s, 3H), 2.22–2.19 (m, 1H), 1.98–1.92 (m, 1H), 1.76–1.66 (m, 4H), 1.57–1.46 (m, 5H), 1.02 (s, 9H). <sup>13</sup>C NMR (CD<sub>3</sub>OD, 100 MHz) δ (ppm) 195.36, 173.13, 173.05, 171.91, 171.80, 163.11, 159.29, 156.87, 152.90, 149.01, 145.60, 144.31, 141.47, 134.22, 133.55, 132.72, 131.45, 131.02, 130.49, 127.61, 127.38, 126.00, 124.72, 116.50, 116.13, 115.45, 107.93, 72.40, 70.90, 70.73, 69.08, 63.61, 60.68, 58.14, 54.33, 52.59, 50.26, 38.90, 37.80, 37.13, 29.90, 26.93, 24.56, 24.40, 22.43, 15.79, 9.18. UPLC-MS (ESI<sup>+</sup>) calcd for C<sub>55</sub>H<sub>66</sub>N<sub>5</sub>O<sub>9</sub>S<sub>2</sub> [M + 1]<sup>+</sup>: 1004.43, found 1004.11. Purity 97.4% (HPLC).</div></div><div id="sec5_1_21" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-2-(4-(4-(2-(Ethyl(2-(4-(6-hydroxy-2-(4-hydroxyphenyl)benzo[<i>b</i>]thiophene-3-carbonyl)phenoxy)ethyl)amino)ethyl)piperidin-1-yl)butanamido)-3,3-dimethylbutanoyl)-4-hydroxy-<i>N</i>-((<i>S</i>)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)pyrrolidine-2-carboxamide (<b>33</b>)</h4><div class="NLM_p last">This compound was prepared using a procedure similar to that used for compound <b>32</b>. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz) δ (ppm) 8.95 (s, 1H), 7.74 (d, <i>J</i> = 8.8 Hz, 2H), 7.45–7.40 (m, 5H), 7.27 (d, <i>J</i> = 2.4 Hz, 1H), 7.17 (d, <i>J</i> = 8.4 Hz, 2H), 6.93 (d, <i>J</i> = 9.2 Hz, 2H), 6.87 (dd, <i>J</i> = 8.8 Hz, <i>J</i> = 2.4 Hz, 1H), 6.60 (d, <i>J</i> = 8.4 Hz, 2H), 5.03–4.90 (m, 1H), 4.59–4.53 (m, 2H), 4.43–4.41 (m, 1H), 4.37 (t, <i>J</i> = 4.8 Hz, 2H), 3.90 (d, <i>J</i> = 10.8 Hz, 1H), 3.74 (dd, <i>J</i> = 10.8 Hz, <i>J</i> = 4.0 Hz, 1H), 3.61–3.56 (m, 4H), 3.35–3.27 (m, 4H), 3.10 (t, <i>J</i> = 6.8 Hz, 2H), 2.94–2.86 (m, 2H), 2.51 (t, <i>J</i> = 6.4 Hz, 2H), 2.48 (s, 3H), 2.22–2.17 (m, 1H), 2.01–1.95 (m, 5H), 1.78–1.68 (m, 3H), 1.59–1.57 (m, 2H), 1.50 (d, <i>J</i> = 7.2 Hz, 3H), 1.34 (t, <i>J</i> = 7.6 Hz, 3H), 1.06 (s, 9H). UPLC-MS (ESI<sup>+</sup>) calcd for C<sub>59</sub>H<sub>73</sub>N<sub>6</sub>O<sub>8</sub>S<sub>2</sub> [M + 1]<sup>+</sup>: 1057.49, found 1057.90. Purity 99.1% (HPLC).</div></div><div id="sec5_1_22" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-2-(4-(4-(2-(Ethyl(2-(4-(6-hydroxy-2-(4-hydroxyphenyl)benzo[<i>b</i>]thiophene-3-carbonyl)phenoxy)ethyl)amino)ethyl)piperazin-1-yl)butanamido)-3,3-dimethylbutanoyl)-4-hydroxy-<i>N</i>-((<i>S</i>)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)pyrrolidine-2-carboxamide (<b>34</b>)</h4><div class="NLM_p last">This compound was prepared using a procedure similar to that used for compound <b>32</b>. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz) δ (ppm) 8.94 (s, 1H), 7.75 (d, <i>J</i> = 8.8 Hz, 2H), 7.45–7.40 (m, 5H), 7.27 (d, <i>J</i> = 2.4 Hz, 1H), 7.18 (d, <i>J</i> = 8.4 Hz, 2H), 6.93 (d, <i>J</i> = 9.2 Hz, 2H), 6.87 (dd, <i>J</i> = 8.8 Hz, <i>J</i> = 2.4 Hz, 1H), 6.61 (d, <i>J</i> = 8.4 Hz, 2H), 5.03–4.99 (m, 1H), 4.58–4.54 (m, 2H), 4.44–4.39 (m, 3H), 3.90 (d, <i>J</i> = 10.8 Hz, 1H), 3.76–3.70 (m, 3H), 3.51–3.48 (m, 2H), 3.42–3.37 (m, 2H), 3.14–3.12 (m, 2H), 2.85 (t, <i>J</i> = 6.4 Hz, 2H), 2.53 (t, <i>J</i> = 6.4 Hz, 2H), 2.48 (s, 3H), 2.23–2.18 (m, 1H), 2.01–1.92 (m, 3H), 1.50 (d, <i>J</i> = 6.8 Hz, 3H), 1.36 (t, <i>J</i> = 7.2 Hz, 3H), 1.06 (s, 9H). UPLC-MS (ESI<sup>+</sup>) calcd for C<sub>58</sub>H<sub>72</sub>N<sub>7</sub>O<sub>8</sub>S<sub>2</sub> [M + 1]<sup>+</sup>: 1058.49, found 1058.72. Purity 99.3% (HPLC).</div></div><div id="sec5_1_23" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-2-(3-(4-(5-(Ethyl(2-(4-(6-hydroxy-2-(4-hydroxyphenyl)benzo[<i>b</i>]thiophene-3-carbonyl)phenoxy)ethyl)amino)pent-1-yn-1-yl)-1<i>H</i>-pyrazol-1-yl)propanamido)-3,3-dimethylbutanoyl)-4-hydroxy-<i>N</i>-((<i>S</i>)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)pyrrolidine-2-carboxamide (<b>35</b>)</h4><div class="NLM_p last">This compound was prepared using a procedure similar to that used for compound <b>32</b>. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz) δ (ppm) 9.13 (s, 1H), 7.69 (d, <i>J</i> = 8.8 Hz, 2H), 7.62 (s, 1H), 7.46–7.41 (m, 6H), 7.27 (d, <i>J</i> = 2.0 Hz, 1H), 7.17 (d, <i>J</i> = 8.4 Hz, 2H), 6.90–6.86 (m, 3H), 6.61 (d, <i>J</i> = 8.8 Hz, 2H), 5.01–4.96 (m, 1H), 4.59–4.55 (m, 2H), 4.42–4.26 (m, 5H), 3.86 (d, <i>J</i> = 10.8 Hz, 1H), 3.72 (dd, <i>J</i> = 10.8 Hz, <i>J</i> = 4.0 Hz, 1H), 3.62–3.57 (m, 2H), 3.41–3.30 (m, 4H), 2.92–2.85 (m, 1H), 2.75–2.70 (m, 1H), 2.54 (t, <i>J</i> = 6.8 Hz, 2H), 2.50 (s, 3H), 2.22–2.17 (m, 1H), 2.01–1.91 (m, 3H), 1.50 (d, <i>J</i> = 6.8 Hz, 3H), 1.36 (t, <i>J</i> = 7.2 Hz, 3H), 0.95 (s, 9H). <sup>13</sup>C NMR (CD<sub>3</sub>OD, 100 MHz) δ (ppm) 195.31, 173.25, 172.24, 172.10, 163.01, 159.26, 156.86, 153.68, 146.13, 144.43, 142.84, 141.47, 134.21, 133.48, 132.76, 131.49, 131.00, 130.51, 127.74, 126.02, 124.77, 116.48, 116.15, 115.42, 107.92, 104.32, 88.60, 74.07, 70.96, 63.58, 60.60, 59.18, 57.88, 53.41, 52.83, 50.53, 50.45, 38.77, 36.74, 36.42, 26.98, 23.92, 22.36, 17.27, 15.15, 9.16. UPLC-MS (ESI<sup>+</sup>) calcd for C<sub>59</sub>H<sub>66</sub>N<sub>7</sub>O<sub>8</sub>S<sub>2</sub> [M + 1]<sup>+</sup>: 1064.44, found 1064.89. Purity 95.1% (HPLC).</div></div><div id="sec5_1_24" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-2-(2-(4-(4-(Ethyl(2-(4-(6-hydroxy-2-(4-hydroxyphenyl)benzo[<i>b</i>]thiophene-3-carbonyl)phenoxy)ethyl)amino)but-1-yn-1-yl)-1<i>H</i>-pyrazol-1-yl)acetamido)-3,3-dimethyl-butanoyl)-4-hydroxy-<i>N</i>-((<i>S</i>)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)pyrrolidine-2-carboxamide (<b>36</b>)</h4><div class="NLM_p last">This compound was prepared using a procedure similar to that used for compound <b>32</b>. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz) δ (ppm) 8.87 (s, 1H), 7.74–7.70 (m, 3H), 7.52 (s, 1H), 7.44–7.41 (m, 5H), 7.26 (d, <i>J</i> = 2.0 Hz, 1H), 7.18 (d, <i>J</i> = 8.4 Hz, 2H), 6.88–6.85 (m, 3H), 6.63 (d, <i>J</i> = 8.8 Hz, 2H), 5.01–4.98 (m, 1H), 4.62–4.52 (m, 2H), 4.42–4.39 (m, 3H), 3.82 (d, <i>J</i> = 11.2 Hz, 1H), 3.76–3.64 (m, 3H), 3.50–3.39 (m, 6H), 2.96 (t, <i>J</i> = 7.2 Hz, 2H), 2.47 (s, 3H), 2.26–2.15 (m, 1H), 1.96–1.90 (m, 1H), 1.49 (d, <i>J</i> = 7.2 Hz, 3H), 1.38 (t, <i>J</i> = 7.2 Hz, 3H), 1.02 (s, 9H). UPLC-MS (ESI<sup>+</sup>) calcd for C<sub>57</sub>H<sub>62</sub>N<sub>7</sub>O<sub>8</sub>S<sub>2</sub> [M + 1]<sup>+</sup>: 1036.41, found 1035.92. Purity 98.8% (HPLC).</div></div><div id="sec5_1_25" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-2-(2-(4-(4-(Ethyl(2-(4-(6-hydroxy-2-(4-hydroxyphenyl)benzo[<i>b</i>]thiophene-3-carbonyl)phenoxy)ethyl)amino)butyl)-1<i>H</i>-pyrazol-1-yl)acetamido)-3,3-dimethylbutanoyl)-4-hydroxy-<i>N</i>-((<i>S</i>)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)pyrrolidine-2-carboxamide (<b>37</b>)</h4><div class="NLM_p last">This compound was prepared using a procedure similar to that used for compound <b>32</b>. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz) δ (ppm) 9.23 (s, 1H), 7.70 (d, <i>J</i> = 8.8 Hz, 2H), 7.49 (s, 1H), 7.44–7.37 (m, 6H), 7.25 (d, <i>J</i> = 2.4 Hz, 1H), 7.14 (d, <i>J</i> = 8.8 Hz, 2H), 6.87–6.83 (m, 3H), 6.59 (d, <i>J</i> = 8.4 Hz, 2H), 4.99–4.83 (m, 3H), 4.61–4.52 (m, 2H), 4.40–4.37 (m, 1H), 4.30 (t, <i>J</i> = 4.4 Hz, 2H), 3.81 (d, <i>J</i> = 11.2 Hz, 1H), 3.69 (dd, <i>J</i> = 11.2 Hz, <i>J</i> = 4.0 Hz, 1H), 3.54 (d, <i>J</i> = 4.0 Hz, 2H), 3.28–3.16 (m, 4H), 2.54–2.46 (m, 5H), 2.22–2.17 (m, 1H), 1.96–1.89 (m, 1H), 1.73–1.60 (m, 4H), 1.47 (d, <i>J</i> = 7.2 Hz, 3H), 1.30 (t, <i>J</i> = 7.2 Hz, 3H), 1.00 (s, 9H). <sup>13</sup>C NMR (CD<sub>3</sub>OD, 100 MHz) δ (ppm) 195.28, 173.08, 171.67, 169.31, 163.05, 159.21, 156.78, 153.34, 147.93, 145.88, 144.26, 141.40, 140.79, 134.19, 133.92, 133.49, 132.59, 131.61, 131.41, 130.96, 130.89, 130.45, 127.65, 125.90, 124.74, 122.64, 116.52, 116.16, 115.42, 108.00, 70.90, 63.51, 60.56, 59.13, 57.96, 54.70, 54.16, 52.44, 50.14, 38.80, 36.70, 28.55, 26.93, 26.82, 24.24, 24.05, 22.38, 15.39, 9.07. UPLC-MS (ESI<sup>+</sup>) calcd for C<sub>57</sub>H<sub>66</sub>N<sub>7</sub>O<sub>8</sub>S<sub>2</sub> [M + 1]<sup>+</sup>: 1040.44, found 1040.17. Purity 98.7% (HPLC).</div></div><div id="sec5_1_26" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-2-(8-(Ethyl(2-(4-((<i>Z</i>)-1-(4-hydroxyphenyl)-2-phenylbut-1-en-1-yl)-phenoxy)ethyl)amino)octanamido)-3,3-dimethylbutanoyl)-4-hydroxy-<i>N</i>-((<i>S</i>)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)pyrrolidine-2-carboxamide (<b>38</b>)</h4><div class="NLM_p last">This compound was prepared using a procedure similar to that used for compound <b>15</b>. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz) δ (ppm) 8.98 (s, 1H), 7.47–7.36 (m, 4H), 7.13–7.01 (m, 7H), 6.84–6.75 (m, 3H), 6.66–6.60 (m, 2H), 5.03–4.98 (m, 1H), 4.64–4.62 (m, 1H), 4.56 (t, <i>J</i> = 8.4 Hz, 1H), 4.45–4.41 (m, 1H), 4.22–4.18 (m, 2H), 3.89–3.86 (m, 1H), 3.74 (dd, <i>J</i> = 11.2 Hz, <i>J</i> = 4.0 Hz, 1H), 3.55–3.52 (m, 2H), 3.21–3.15 (m, 2H), 2.52–2.45 (m, 5H), 2.32–2.17 (m, 3H), 1.99–1.92 (m, 1H), 1.78–1.56 (m, 5H), 1.52–1.49 (m, 3H), 1.45–1.30 (m, 9H), 1.04 (s, 9H), 0.91 (t, <i>J</i> = 7.2 Hz, 3H). <sup>13</sup>C NMR (CD<sub>3</sub>OD, 100 MHz) δ (ppm) 175.90, 173.22, 172.31, 157.41, 156.91, 153.20, 145.85, 144.03, 142.58, 139.49, 138.76, 136.07, 133.23, 131.55, 130.91, 130.51, 128.88, 127.67, 127.04, 115.92, 114.46, 70.96, 63.11, 60.60, 58.99, 58.00, 54.48, 52.76, 50.15, 38.82, 37.64, 36.51, 33.74, 29.95, 29.93, 29.86, 29.81, 27.36, 27.05, 26.73, 24.71, 22.37, 15.54, 13.86, 9.04. UPLC-MS (ESI<sup>+</sup>) calcd for C<sub>57</sub>H<sub>74</sub>N<sub>5</sub>O<sub>6</sub>S [M + 1]<sup>+</sup>: 956.54, found 956.51.</div></div><div id="sec5_1_27" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-2-(8-((2-(4-(1,2-Diphenylbut-1-en-1-yl)phenoxy)ethyl)(ethyl)amino)-octanamido)-3,3-dimethylbutanoyl)-4-hydroxy-<i>N</i>-((<i>S</i>)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)pyrrolidine-2-carboxamide (<b>39</b>)</h4><div class="NLM_p last">This compound was prepared using a procedure similar to that used for compound <b>15</b>. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz) δ (ppm) 9.08 (s, 1H), 7.47–7.42 (m, 4H), 7.36–7.32 (m, 1H), 7.28–7.22 (m, 0.5H), 7.21–7.07 (m, 7H), 7.03–6.95 (m, 2.5H), 6.86–6.82 (m, 2H), 6.66–6.64 (m, 1H), 5.03–4.98 (m, 1H), 4.64–4.62 (m, 1H), 4.57 (t, <i>J</i> = 8.4 Hz, 1H), 4.43–4.41 (m, 1H), 4.38 (t, <i>J</i> = 4.8 Hz, 1H), 4.21 (t, <i>J</i> = 4.8 Hz, 1H), 3.88 (d, <i>J</i> = 10.8 Hz, 1H), 3.74 (dd, <i>J</i> = 10.8 Hz, <i>J</i> = 4.0 Hz, 1H), 3.64 (t, <i>J</i> = 4.8 Hz, 1H), 3.53 (t, <i>J</i> = 4.8 Hz, 1H), 3.38–3.34 (m, 1H), 3.31–3.17 (m, 3H), 2.49–2.42 (m, 5H), 2.30–2.19 (m, 3H), 1.99–1.95 (m, 1H), 1.70–1.56 (m, 4H), 1.50 (d, <i>J</i> = 7.2 Hz, 3H), 1.40–1.30 (m, 9H), 1.04 (s, 9H), 0.94–0.88 (m, 3H). UPLC-MS (ESI<sup>+</sup>) calcd for C<sub>57</sub>H<sub>74</sub>N<sub>5</sub>O<sub>5</sub>S [M + 1]<sup>+</sup>: 940.54, found 940.82. Purity 97.0% (HPLC).</div></div><div id="sec5_1_28" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-2-(8-(Ethyl(2-(4-((5-hydroxy-2-(4-hydroxyphenyl)-3-methyl-1<i>H</i>-indol-1-yl)methyl)phenoxy)ethyl)amino)octanamido)-3,3-dimethylbutanoyl)-4-hydroxy-<i>N</i>-((<i>S</i>)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)pyrrolidine-2-carboxamide (<b>40</b>)</h4><div class="NLM_p last">This compound was prepared using a procedure similar to that used for compound <b>15</b>. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz) δ (ppm) 9.08 (s, 1H), 7.45–7.40 (m, 4H), 7.12 (d, <i>J</i> = 8.4 Hz, 2H), 6.99 (d, <i>J</i> = 8.8 Hz, 1H), 6.91 (d, <i>J</i> = 2.4 Hz, 1H), 6.84–6.75 (m, 6H), 6.63 (dd, <i>J</i> = 8.8 Hz, <i>J</i> = 2.4 Hz, 1H), 5.11 (s, 2H), 5.02–4.97 (m, 1H), 4.64–4.62 (m, 1H), 4.56 (t, <i>J</i> = 8.4 Hz, 1H), 4.43–4.41 (m, 1H), 4.24 (t, <i>J</i> = 4.8 Hz, 1H), 3.87 (d, <i>J</i> = 11.2 Hz, 1H), 3.73 (dd, <i>J</i> = 11.2 Hz, <i>J</i> = 4.0 Hz, 1H), 3.54 (t, <i>J</i> = 4.8 Hz, 1H), 2.49 (s, 3H), 2.31–2.15 (m, 6H), 1.98–1.92 (m, 1H), 1.73–1.65 (m, 2H), 1.59–1.55 (m, 2H), 1.50 (d, <i>J</i> = 7.2 Hz, 3H), 1.40–1.29 (m, 9H), 1.03 (s, 9H). <sup>13</sup>C NMR (CD<sub>3</sub>OD, 100 MHz) δ (ppm) 175.91, 173.21, 172.33, 158.52, 157.92, 153.66, 151.65, 146.06, 139.86, 133.87, 133.08, 132.79, 131.04, 130.50, 128.68, 127.72, 124.47, 116.23, 115.54, 112.17, 111.59, 108.66, 103.92, 70.94, 63.27, 60.60, 59.00, 57.98, 54.43, 52.72, 50.15, 47.58, 38.78, 36.47, 29.90, 29.80, 29.76, 27.04, 24.69, 22.35, 15.21, 9.62, 9.05. UPLC-MS (ESI<sup>+</sup>) calcd for C<sub>57</sub>H<sub>73</sub>N<sub>6</sub>O<sub>7</sub>S [M + 1]<sup>+</sup>: 985.53, found 985.82. Purity >99.5% (HPLC).</div></div><div id="sec5_1_29" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43"> (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-2-(8-(Ethyl(2-(4-((1<i>R</i>,2<i>S</i>)-6-hydroxy-2-phenyl-1,2,3,4-tetrahydronaphthalen-1-yl)phenoxy)ethyl)amino)octanamido)-3,3-dimethylbutanoyl)-4-hydroxy-<i>N</i>-((<i>S</i>)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)pyrrolidine-2-carboxamide (<b>41</b>)</h4><div class="NLM_p last">This compound was prepared using a procedure similar to that used for compound <b>15</b>. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz) δ (ppm) 8.94 (s, 1H), 7.46–7.35 (m, 4H), 7.13–7.09 (m, 3H), 6.83–6.80 (m, 2H), 6.68–6.61 (m, 4H), 6.52 (dd, <i>J</i> = 8.4 Hz, <i>J</i> = 4.0 Hz 1H), 6.38 (d, <i>J</i> = 8.4 Hz, 2H), 5.03–4.98 (m, 1H), 4.64–4.54 (m, 2H), 4.43–4.41 (m, 1H), 4.25–4.20 (m, 3H), 3.89–3.86 (m, 1H), 3.75 (dd, <i>J</i> = 11.2 Hz, <i>J</i> = 4.0 Hz, 1H), 3.56–3.53 (m, 2H), 3.37–3.35 (m, 1H), 3.23–3.16 (m, 2H), 3.06–2.99 (m, 2H), 2.49 (s, 3H), 2.34–2.14 (m, 4H), 1.99–1.92 (m, 1H), 1.79–1.50 (m, 8H), 1.38–1.29 (m, 9H), 1.03 (s, 9H). <sup>13</sup>C NMR (CD<sub>3</sub>OD, 100 MHz) δ (ppm) 175.89, 173.23, 172.31, 157.05, 156.64, 153.10, 148.68, 145.79, 138.82, 137.76, 132.43, 132.06, 131.33, 130.51, 129.21, 128.72, 127.66, 127.66, 127.44, 126.97, 126.63, 115.50, 114.68, 113.98, 70.96, 63.16, 61.04, 60.61, 58.98, 58.01, 54.48, 52.81, 52.24, 51.57, 50.15, 46.73, 38.82, 36.52, 36.49, 30.97, 29.97, 28.83, 27.38, 27.05, 26.74, 25.59, 24.73, 24.44, 23.24, 22.37, 15.62, 9.06. UPLC-MS (ESI<sup>+</sup>) calcd for C<sub>57</sub>H<sub>74</sub>N<sub>5</sub>O<sub>6</sub>S [M + 1]<sup>+</sup>: 956.54, found 956.48.</div></div><div id="sec5_1_30" class="NLM_sec NLM_sec_level_3"><div id="ac_i44" class="anchor-spacer"></div><h4 class="article-section__title" id="_i44"> (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-2-(8-(Ethyl(2-(4-(6-hydroxy-2-(4-hydroxyphenyl)benzo[<i>b</i>]thiophene-3-carbonyl)phenoxy)ethyl)amino)octanamido)-3,3-dimethylbutanoyl)-4-hydroxy-<i>N</i>-(4-(4-methylthiazol-5-yl)benzyl)pyrrolidine-2-carboxamide (<b>42</b>)</h4><div class="NLM_p last">This compound was prepared using a procedure similar to that used for compound <b>15</b>. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz) δ (ppm) 8.86 (s, 1H), 7.74 (d, <i>J</i> = 9.2 Hz, 2H), 7.46–7.39 (m, 5H), 7.27 (d, <i>J</i> = 2.4 Hz, 1H), 7.18 (d, <i>J</i> = 8.8 Hz, 2H), 6.91 (d, <i>J</i> = 8.8 Hz, 2H), 6.87 (dd, <i>J</i> = 8.8 Hz, <i>J</i> = 2.0 Hz, 1H), 6.61 (d, <i>J</i> = 8.8 Hz, 2H), 4.64 (d, <i>J</i> = 8.8 Hz, 1H), 4.58–4.49 (m, 3H), 4.38–4.33 (m, 3H), 3.90 (d, <i>J</i> = 11.2 Hz, 1H), 3.80 (dd, <i>J</i> = 10.8 Hz, <i>J</i> = 4.0 Hz, 1H), 3.59 (t, <i>J</i> = 4.8 Hz, 2H), 3.21–3.17 (m, 2H), 2.46 (s, 3H), 2.32–2.19 (m, 3H), 2.11–2.03 (m, 1H), 1.73–1.71 (m, 2H), 1.62–1.59 (m, 2H), 1.38–1.29 (m, 9H), 1.02 (s, 9H). UPLC-MS (ESI<sup>+</sup>) calcd for C<sub>55</sub>H<sub>66</sub>N<sub>5</sub>O<sub>8</sub>S<sub>2</sub> [M + 1]<sup>+</sup>: 988.44, found 988.98. Purity 97.8% (HPLC).</div></div><div id="sec5_1_31" class="NLM_sec NLM_sec_level_3"><div id="ac_i45" class="anchor-spacer"></div><h4 class="article-section__title" id="_i45"> (2<i>S</i>,4<i>R</i>)-<i>N</i>-((<i>S</i>)-1-(4-Chlorophenyl)ethyl)-1-((<i>S</i>)-2-(8-(ethyl(2-(4-(6-hydroxy-2-(4-hydroxyphenyl)benzo[<i>b</i>]thiophene-3-carbonyl)phenoxy)ethyl)amino)octanamido)-3,3-dimethylbutanoyl)-4-hydroxypyrrolidine-2-carboxamide (<b>43</b>)</h4><div class="NLM_p last">This compound was prepared using a procedure similar to that used for compound <b>15</b>. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz) δ (ppm) 7.75 (d, <i>J</i> = 8.8 Hz, 2H), 7.44 (d, <i>J</i> = 8.8 Hz, 1H), 7.32–7.27 (m, 5H), 7.21–7.15 (m, 2H), 6.95–6.86 (m, 3H), 6.65–6.60 (m, 2H), 4.95–4.87 (m, 1H), 4.63–4.61 (m, 1H), 4.53 (t, <i>J</i> = 8.4 Hz, 2H), 3.87–3.84 (m, 1H), 3.73 (dd, <i>J</i> = 11.2 Hz, <i>J</i> = 4.0 Hz, 1H), 3.64–3.54 (m, 3H), 3.26–3.16 (m, 3H), 3.07–2.96 (m, 1H), 2.33–2.13 (m, 3H), 1.95–1.88 (m, 1H), 1.78–1.58 (m, 5H), 1.51 (d, <i>J</i> = 7.2 Hz, 2H), 1.41–1.29 (m, 11H), 1.02 (s, 9H). <sup>13</sup>C NMR (CD<sub>3</sub>OD, 100 MHz) δ (ppm) 195.37, 175.91, 173.18, 172.29, 163.09, 159.28, 156.88, 144.38, 144.13, 141.49, 134.22, 133.71, 133.53, 132.77, 131.65, 131.47, 131.02, 129.56, 128.70, 128.70, 128.47, 126.04, 124.71, 116.59, 124.71, 116.59, 116.47, 116.13, 115.41, 107.91, 70.94, 63.57, 60.57, 58.99, 57.98, 54.55, 52.59, 50.24, 49.82, 38.78, 36.49, 34.62, 29.95, 29.81, 27.33, 27.03, 26.73, 25.75, 24.71, 22.28, 9.06. UPLC-MS (ESI<sup>+</sup>) calcd for C<sub>52</sub>H<sub>64</sub>ClN<sub>4</sub>O<sub>8</sub>S [M + 1]<sup>+</sup>: 939.41, found 939.45.</div></div><div id="sec5_1_32" class="NLM_sec NLM_sec_level_3"><div id="ac_i46" class="anchor-spacer"></div><h4 class="article-section__title" id="_i46"> (2<i>S</i>,4<i>R</i>)-<i>N</i>-((<i>S</i>)-1-(4-Chlorophenyl)ethyl)-1-((<i>S</i>)-2-(8-(ethyl(2-(4-(6-hydroxy-2-(4-hydroxyphenyl)benzo[<i>b</i>]thiophene-3-carbonyl)phenoxy)ethyl)amino)octanamido)-3,3-dimethylbutanoyl)-4-hydroxypyrrolidine-2-carboxamide (<b>44</b>)</h4><div class="NLM_p last">This compound was prepared using a procedure similar to that used for compound <b>15</b>. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz) δ (ppm) 7.75 (d, <i>J</i> = 8.8 Hz, 2H), 7.68 (d, <i>J</i> = 8.8 Hz, 2H), 7.49–7.40 (m, 3H), 7.27 (d, <i>J</i> = 2.4 Hz, 1H), 7.18 (d, <i>J</i> = 8.8 Hz, 2H), 6.94–6.86 (m, 3H), 6.64–6.60 (m, 2H), 5.01–4.96 (m, 1H), 4.63–4.61 (m, 1H), 4.47–4.38 (m, 1H), 4.35 (t, <i>J</i> = 4.8 Hz, 2H), 3.88–3.85 (m, 1H), 3.72 (dd, <i>J</i> = 11.2 Hz, <i>J</i> = 4.0 Hz, 1H), 3.60–3.58 (m, 2H), 3.23–3.17 (m, 2H), 2.32–2.16 (m, 3H), 1.94–1.87 (m, 1H), 1.73–1.60 (m, 5H), 1.54–1.46 (m, 3H), 1.38–1.29 (m, 10H), 1.02 (s, 9H). <sup>13</sup>C NMR (CD<sub>3</sub>OD, 100 MHz) δ (ppm) 195.36, 175.91, 173.38, 172.30, 163.09, 159.28, 156.88, 151.22, 144.39, 141.49, 134.22, 131.47, 131.02, 128.06, 127.85, 126.04, 124.71, 119.67, 116.48, 116.13, 115.40, 111.78, 107.91, 70.95, 63.56, 60.51, 58.99, 57.99, 54.54, 52.58, 50.33, 50.22, 38.81, 36.49, 29.96, 29.82, 27.34, 27.02, 26.73, 24.72, 22.09, 9.06. UPLC-MS (ESI<sup>+</sup>) calcd for C<sub>53</sub>H<sub>64</sub>N<sub>5</sub>O<sub>8</sub>S [M + 1]<sup>+</sup>: 930.45, found 930.48.</div></div><div id="sec5_1_33" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-2-(8-(Ethyl(2-(4-(6-hydroxy-2-(4-hydroxyphenyl)benzo[<i>b</i>]thiophene-3-carbonyl)phenoxy)ethyl)amino)octanamido)-3,3-dimethylbutanoyl)-<i>N</i>-((<i>S</i>)-1-(4-ethynylphenyl)ethyl)-4-hydroxypyrrolidine-2-carboxamide (<b>45</b>)</h4><div class="NLM_p last">This compound was prepared using a procedure similar to that used for compound <b>15</b>. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz) δ (ppm) 7.75 (d, <i>J</i> = 8.8 Hz, 2H), 7.46–7.38 (m, 3H), 7.29–7.22 (m, 3H), 7.19–7.15 (m, 2H), 6.93–6.86 (m, 3H), 6.61(d, <i>J</i> = 8.8 Hz, 2H), 4.97–4.92 (m, 1H), 4.63–4.61 (m, 1H), 4.54 (t, <i>J</i> = 8.4 Hz, 2H), 4.42–4.39 (m, 1H), 4.36 (t, <i>J</i> = 4.8 Hz, 2H), 3.88–3.85 (m, 1H), 3.75–3.71 (m, 2H), 3.63–3.58 (m, 3H), 3.44 (s, 1H), 3.26–3.17 (m, 3H), 2.33–2.14 (m, 4H), 1.95–1.89 (m, 1H), 1.80–1.65 (m, 3H), 1.67–1.54 (m, 3H), 1.46–1.30 (m, 10H), 1.02 (s, 9H). <sup>13</sup>C NMR (CD<sub>3</sub>OD, 100 MHz) δ (ppm) 195.38, 175.91, 173.20, 172.29, 163.10, 161.00, 159.28, 156.88, 155.70, 146.12, 144.39, 141.49, 134.22, 133.53, 133.18, 132.77, 131.47, 131.02, 130.05, 129.93, 127.14, 126.04, 124.71, 122.43, 116.47, 116.12, 115.41, 107.90, 84.19, 78.50, 70.95, 64.27, 60.58, 59.62, 58.99, 57.98, 55.93, 55.83, 55.65, 54.56, 53.65, 52.59, 50.17, 36.49, 29.94, 29.81, 27.03, 26.73, 24.71, 22.23, 18.70, 17.26, 13.17, 9.07. UPLC-MS (ESI<sup>+</sup>) calcd for C<sub>53</sub>H<sub>64</sub>N<sub>5</sub>O<sub>8</sub>S [M + 1]<sup>+</sup>: 929.45, found 929.49.</div></div><div id="sec5_1_34" class="NLM_sec NLM_sec_level_3"><div id="ac_i48" class="anchor-spacer"></div><h4 class="article-section__title" id="_i48"> (2<i>S</i>,4<i>R</i>)-<i>N</i>-((<i>S</i>)-1-(4-Cyclopropylphenyl)ethyl)-1-((<i>S</i>)-2-(8-(ethyl(2-(4-(6-hydroxy-2-(4-hydroxyphenyl)benzo[<i>b</i>]thiophene-3-carbonyl)phenoxy)ethyl)amino)octanamido)-3,3-dimethylbutanoyl)-4-hydroxypyrrolidine-2-carboxamide (<b>46</b>)</h4><div class="NLM_p last">This compound was prepared using a procedure similar to that used for compound <b>15</b>. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz) δ (ppm) 7.76 (d, <i>J</i> = 8.8 Hz, 2H), 7.45 (d, <i>J</i> = 8.8 Hz, 1H), 7.27 (d, <i>J</i> = 2.0 Hz, 2H), 7.20–7.10 (m, 4H), 7.02–6.98 (m, 2H), 6.94–6.86 (m, 3H), 6.63–6.59 (m, 2H), 4.95–4.91 (m, 1H), 4.63–4.61 (m, 1H), 4.54 (t, <i>J</i> = 8.4 Hz, 1H), 4.43–4.39 (m, 1H), 4.36–4.34 (m, 2H), 3.87–3.85 (m, 1H), 3.75–3.71 (m, 1H), 3.63–3.58 (m, 2H), 3.23–3.17 (m, 2H), 2.31–2.23 (m, 2H), 2.18–2.11 (m, 1H), 1.96–1.83 (m, 2H), 1.72–1.59 (m, 4H), 1.51–1.28 (m, 12H), 1.02 (s, 9H), 1.00–0.90 (m, 2H), 0.64–0.60 (m, 2H). <sup>13</sup>C NMR (CD<sub>3</sub>OD, 100 MHz) δ (ppm) 195.36, 175.88, 172.98, 172.30, 163.07, 159.29, 156.90, 144.39, 144.17, 142.11, 141.50, 134.22, 133.54, 131.48, 131.02, 126.95, 126.72, 126.06, 124.70, 116.47, 116.13, 115.40, 107.90, 70.94, 63.55, 60.64, 58.98, 57.98, 55.90, 54.57, 53.58, 52.62, 51.85, 51.29, 38.76, 36.51, 29.93, 29.83, 27.36, 27.04, 26.72, 24.72, 15.77, 9.51, 9.06. UPLC-MS (ESI<sup>+</sup>) calcd for C<sub>55</sub>H<sub>69</sub>N<sub>4</sub>O<sub>8</sub>S [M + 1]<sup>+</sup>: 945.48, found 945.51.</div></div><div id="sec5_1_35" class="NLM_sec NLM_sec_level_3"><div id="ac_i49" class="anchor-spacer"></div><h4 class="article-section__title" id="_i49"> (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-2-(8-(Ethyl(2-(4-(6-hydroxy-2-(4-hydroxyphenyl)benzo[<i>b</i>]thiophene-3-carbonyl)phenoxy)ethyl)amino)octanamido)-3,3-dimethylbutanoyl)-4-hydroxy-<i>N</i>-((<i>S</i>)-1-(4-isopropylphenyl)ethyl)pyrrolidine-2-carboxamide (<b>47</b>)</h4><div class="NLM_p last">This compound was prepared using a procedure similar to that used for compound <b>15</b>. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz) δ (ppm) 7.75 (d, <i>J</i> = 8.8 Hz, 2H), 7.44 (d, <i>J</i> = 8.8 Hz, 2H), 7.27 (d, <i>J</i> = 2.0 Hz, 1H), 7.22–7.15 (m, 6H), 6.93–6.86 (m, 3H), 6.63–6.59 (m, 2H), 4.96–4.91 (m, 1H), 4.62 (d, <i>J</i> = 8.8 Hz, 1H), 4.53 (d, <i>J</i> = 8.4 Hz, 1H), 4.43–4.36 (m, 1H), 4.35–4.29 (m, 2H), 3.87–3.84 (m, 1H), 3.75–3.71 (m, 1H), 3.60–3.53 (m, 2H), 3.24–3.15 (m, 2H), 2.89–2.81 (m, 1H), 2.34–2.21 (m, 2H), 2.18–2.12 (m, 1H), 1.98–1.91 (m, 1H), 1.73–1.70 (m, 2H), 1.63–1.58 (m, 2H), 1.52–1.29 (m, 12H), 1.22 (d, <i>J</i> = 7.2 Hz, 6H), 1.02 (s, 9H). <sup>13</sup>C NMR (CD<sub>3</sub>OD, 100 MHz) δ (ppm) 195.35, 175.89, 173.00, 172.30, 163.08, 159.29, 156.88, 148.92, 144.37, 142.57, 141.49, 134.22, 133.53, 132.79, 131.47, 131.01, 127.50, 126.76, 126.03, 124.71, 116.13, 115.40, 107.91, 70.94, 63.55, 60.63, 58.98, 57.98, 54.53, 52.57, 50.20, 50.10, 38.76, 36.51, 35.06, 29.94, 29.81, 27.34, 27.04, 26.72, 24.70, 24.45, 22.49, 9.06. UPLC-MS (ESI<sup>+</sup>) calcd for C<sub>55</sub>H<sub>71</sub>N<sub>4</sub>O<sub>8</sub>S [M + 1]<sup>+</sup>: 947.50, found 947.53.</div></div><div id="sec5_1_36" class="NLM_sec NLM_sec_level_3"><div id="ac_i50" class="anchor-spacer"></div><h4 class="article-section__title" id="_i50"> (2<i>S</i>,4<i>R</i>)-<i>N</i>-((<i>S</i>)-1-(4-(<i>tert</i>-Butyl)phenyl)ethyl)-1-((<i>S</i>)-2-(8-(ethyl(2-(4-(6-hydroxy-2-(4-hydroxyphenyl)benzo[<i>b</i>]thiophene-3-carbonyl)phenoxy)ethyl)amino)octanamido)-3,3-dimethylbutanoyl)-4-hydroxypyrrolidine-2-carboxamide (<b>48</b>)</h4><div class="NLM_p last">This compound was prepared using a procedure similar to that used for compound <b>15</b>. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz) δ (ppm) 7.75 (d, <i>J</i> = 8.8 Hz, 2H), 7.45 (d, <i>J</i> = 8.8 Hz, 1H), 7.34 (d, <i>J</i> = 8.4 Hz, 2H), 7.27 (d, <i>J</i> = 2.0 Hz, 1H), 7.22–7.16 (m, 4H), 6.93–6.86 (m, 3H), 6.63–6.59 (m, 2H), 4.95–4.91 (m, 1H), 4.65–4.60 (m, 1H), 4.53 (t, <i>J</i> = 8.4 Hz, 1H), 4.42–4.40 (m, 1H), 4.35 (t, <i>J</i> = 4.4 Hz, 2H), 3.87–3.84 (m, 1H), 3.75–3.71 (m, 1H), 3.60–3.56 (m, 2H), 3.25–3.16 (m, 2H), 2.34–2.14 (m, 3H), 1.98–1.91 (m, 1H), 1.74–1.71 (m, 2H), 1.64–1.55 (m, 2H), 1.45–1.32 (m, 1H), 1.29 (s, 9H), 1.02 (s, 9H). <sup>13</sup>C NMR (CD<sub>3</sub>OD, 100 MHz) δ (ppm) 193.36, 173.88, 171.01, 170.31, 161.08, 157.29, 154.88, 149.06, 142.37, 140.13, 139.49, 132.22, 131.53, 130.80, 129.47, 129.02, 124.71, 124.48, 124.39, 124.04, 122.70, 114.47, 114.13, 113.40, 105.91, 68.95, 61.56, 58.64, 56.98, 55.98, 52.54, 50.58, 48.21, 48.02, 36.77, 34.51, 33.26, 29.78, 27.94, 27.81, 25.34, 25.04, 24.72, 22.70, 20.45, 7.06. UPLC-MS (ESI<sup>+</sup>) calcd for C<sub>56</sub>H<sub>73</sub>N<sub>4</sub>O<sub>8</sub>S [M + 1]<sup>+</sup>: 961.51, found 961.55.</div></div><div id="sec5_1_37" class="NLM_sec NLM_sec_level_3"><div id="ac_i51" class="anchor-spacer"></div><h4 class="article-section__title" id="_i51"> 4-(6-Methoxy-2-(4-methoxyphenyl)benzo[<i>b</i>]thiophene-3-carbonyl)phenyl acetate (<b>50</b>)</h4><div class="NLM_p last">Oxalyl chloride (9.70 mL, 120 mmol, 3.0 equiv) was added dropwise under N<sub>2</sub> to a solution of 4-acetoxybenzoic acid (<b>49</b>) (7.206 g, 40 mmol, 1.0 equiv) in anhydrous DCM (80 mL) at 0 °C. Then several drops of DMF were added. The solution was warmed to rt and stirred for 1 h. The solution was concentrated and dried to obtain the acyl chloride as a white solid. The intermediate was dissolved in anhydrous DCM (150 mL), then 6-methoxy-2-(4-methoxyphenyl)-benzo[<i>b</i>]thiophene (8.65 g, 32 mmol, 0.8 equiv) was added followed by addition of AlCl<sub>3</sub> (8.00 g, 60 mmol, 1.5 equiv) in three portions over a period of 5 min with vigorous stirring at 0 °C under N<sub>2</sub>. The mixture was warmed to rt and stirred for 1 h. The reaction was quenched by slow addition of ice–H<sub>2</sub>O followed by 1 N HCl (aq). The layers were separated, and the aqueous layer was extracted twice with DCM. The combined organic layer was dried by anhydrous Na<sub>2</sub>SO<sub>4</sub>. After filtration and concentration, the residue was purified on a silica gel flash column with hexane:DCM (100:1–1:100) to afford the intermediate (<b>50</b>) as a yellow solid (5.517 g, 40% yield). <sup>1</sup>H NMR (CDCl<sub>3</sub>, 400 MHz) δ (ppm) 7.81 (d, <i>J</i> = 8.8 Hz, 2H), 7.61 (d, <i>J</i> = 8.8 Hz, 1H), 7.32–7.29 (m, 3H), 7.02–6.99 (m, 3H), 6.74 (d, <i>J</i> = 8.8 Hz, 2H), 3.86 (s, 3H), 3.73 (s, 3H), 2.25 (s, 3H). <sup>13</sup>C NMR (CDCl<sub>3</sub>, 100 MHz) δ (ppm) 193.15, 168.63, 159.99, 157.78, 154.38, 144.16, 140.10, 135.03, 133.76, 131.52, 130.48, 130.02, 125.76, 124.16, 121.54, 114.99, 114.13, 104.54, 55.65, 55.28, 21.16. UPLC-MS (ESI<sup>+</sup>) calcd for C<sub>25</sub>H<sub>21</sub>O<sub>5</sub>S [M + 1]<sup>+</sup>: 433.11, found 433.37.</div></div><div id="sec5_1_38" class="NLM_sec NLM_sec_level_3"><div id="ac_i52" class="anchor-spacer"></div><h4 class="article-section__title" id="_i52"> (4-Hydroxyphenyl)(6-methoxy-2-(4-methoxyphenyl)benzo[<i>b</i>]thiophen-3-yl)methanone (<b>51</b>)</h4><div class="NLM_p last">Compound <b>50</b> (5.517 g, 12.76 mmol, 1.0 equiv) was dissolved in EtOH (70 mL) and H<sub>2</sub>O (30 mL). Then sodium acetate (5.23 g, 63.8 mmol, 5.0 equiv) was added. The solution was heated to 90–100 °C and stirred for 12 h. The solution was cooled to rt and concentrated. The residue was diluted in EtOAc and H<sub>2</sub>O. The organic layer was separated, and the aqueous layer was extracted twice with EtOAc. The combined organic layer was dried by anhydrous Na<sub>2</sub>SO<sub>4</sub>. After filtration and concentration, the residue was purified by silica gel flash column chromatography with hexane:EtOAc (5:1–2:1) to afford intermediate <b>51</b> as yellow oil (4.7 g, 95% yield). <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz) δ (ppm) 7.64 (d, <i>J</i> = 9.2 Hz, 2H), 7.43 (d, <i>J</i> = 8.8 Hz, 1H), 7.30 (d, <i>J</i> = 2.4 Hz, 1H), 7.24 (d, <i>J</i> = 8.8 Hz, 2H), 6.89 (dd, <i>J</i> = 8.8 Hz, <i>J</i> = 2.4 Hz, 1H), 6.69–6.64 (m, 4H), 3.73 (s, 3H), 3.59 (s, 3H). <sup>13</sup>C NMR (CD<sub>3</sub>OD, 100 MHz) δ (ppm) 193.95, 162.85, 159.92, 157.81, 142.33, 140.08, 133.78, 132.55, 130.34, 129.91, 129.06, 125.78, 123.43, 115.04, 114.63, 113.79, 104.34, 54.78, 54.39. UPLC-MS (ESI<sup>+</sup>) calcd for C<sub>23</sub>H<sub>19</sub>O<sub>4</sub>S [M + 1]<sup>+</sup>: 391.10, found 391.42.</div></div><div id="sec5_1_39" class="NLM_sec NLM_sec_level_3"><div id="ac_i53" class="anchor-spacer"></div><h4 class="article-section__title" id="_i53"> (4-(2-(Ethylamino)ethoxy)phenyl)(6-methoxy-2-(4-methoxyphenyl)benzo[<i>b</i>]thiophen-3-yl)methanone (<b>52</b>)</h4><div class="NLM_p last">1,2-Dibromoethane (2.0 mL, 24.0 mmol, 2.0 equiv) and Cs<sub>2</sub>CO<sub>3</sub> (5.86 g, 18.0 mmol, 1.5 equiv) were added sequentially to a solution of compound <b>51</b> (4.7 g, 12.0 mmol, 1.0 equiv) in 200 mL of MeCN. The solution was heated to reflux for 12 h. The solution was filtered and washed with MeCN. The concentrated residue was used in next step without further column purification. Ethylamine (2.0 M in THF) (60 mL, 120 mmol, 10.0 equiv) was added to a solution of the residue in DMF. The solution was heated to 80 °C and stirred for 12 h. After cooling to rt, the reaction mixture was diluted in EtOAc and saturated brine. The aqueous layer was extracted with EtOAc twice. The combined organic layer was dried and concentrated. The residue was purified by silica gel flash column chromatography with DCM:MeOH (10:1) to afford the compound <b>52</b> as a yellow solid (4.43 g, 80% yield). <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz) δ (ppm) 7.63 (d, <i>J</i> = 8.8 Hz, 2H), 7.42 (d, <i>J</i> = 8.8 Hz, 1H), 7.32 (d, <i>J</i> = 2.4 Hz, 1H), 7.20 (d, <i>J</i> = 8.8 Hz, 2H), 6.88 (dd, <i>J</i> = 8.8 Hz, <i>J</i> = 2.4 Hz, 1H), 6.71 (d, <i>J</i> = 8.8 Hz, 2H), 6.64 (d, <i>J</i> = 8.8 Hz, 2H), 3.93 (t, <i>J</i> = 4.2 Hz, 2H), 3.75 (s, 3H), 3.59 (s, 3H), 2.83 (t, <i>J</i> = 5.2 Hz, 2H), 2.59 (q, <i>J</i> = 7.2 Hz, 2H), 1.06 (t, <i>J</i> = 7.2 Hz, 3H). <sup>13</sup>C NMR (CD<sub>3</sub>OD, 100 MHz) δ (ppm) 194.88, 164.48, 161.27, 159.20, 143.95, 141.41, 135.02, 133.39, 131.61, 131.48, 131.26, 127.01, 124.76, 115.97, 115.30, 115.10, 105.67, 68.13, 56.10, 55.70, 49.65, 44.52, 14.68. UPLC-MS (ESI<sup>+</sup>) calcd for C<sub>27</sub>H<sub>28</sub>NO<sub>4</sub>S [M + 1]<sup>+</sup>: 462.17, found 462.27.</div></div><div id="sec5_1_40" class="NLM_sec NLM_sec_level_3"><div id="ac_i54" class="anchor-spacer"></div><h4 class="article-section__title" id="_i54"> (4-(2-(Ethylamino)ethoxy)phenyl)(6-hydroxy-2-(4-hydroxyphenyl)benzo[<i>b</i>]thiophen-3-yl)methanone (<b>53</b>)</h4><div class="NLM_p last">BBr<sub>3</sub> solution (8.0 mL, 1.0 M in DCM) (8.0 mmol, 4.0 equiv) was slowly added under N<sub>2</sub> to a solution of <b>52</b> (923 mg, 2.0 mmol, 1.0 equiv) in 30 mL of anhydrous DCM at 0 °C. The dark-red solution was stirred at rt for 2 h, then MeOH (1.0 mL) was added dropwise to quench the reaction. The solution was concentrated, and the residue was dissolved in EtOAc (50 mL) then aqueous saturated NaHCO<sub>3</sub> (50 mL) and EtOH (5 mL) were added. The organic layer was separated and dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>. After filtration, the solution was concentrated and the residue was purified by silica gel flash column chromatography with DCM:MeOH (10:1–5:1) to afford the key intermediate (<b>53</b>) as a yellow solid (520 mg, 60% yield). <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz) δ (ppm) 7.72 (d, <i>J</i> = 8.8 Hz, 2H), 7.43 (d, <i>J</i> = 8.8 Hz, 1H), 7.27 (d, <i>J</i> = 2.0 Hz, 1H), 7.17 (d, <i>J</i> = 8.8 Hz, 2H), 6.91 (d, <i>J</i> = 8.8 Hz, 2H), 6.87 (dd, <i>J</i> = 8.8 Hz, <i>J</i> = 2.4 Hz, 1H), 6.61 (d, <i>J</i> = 8.8 Hz, 2H), 4.27 (t, <i>J</i> = 4.8 Hz, 2H), 3.42 (t, <i>J</i> = 4.8 Hz, 2H), 3.14 (q, <i>J</i> = 7.2 Hz, 2H), 1.33 (t, <i>J</i> = 7.2 Hz, 3H). <sup>13</sup>C NMR (CD<sub>3</sub>OD, 100 MHz) δ (ppm) 195.41, 163.41, 159.22, 156.80, 144.30, 141.45, 134.21, 133.45, 132.55, 131.42, 131.00, 125.99, 124.68, 116.43, 116.08, 115.39, 107.90, 64.68, 47.41, 44.36, 11.40. UPLC-MS (ESI<sup>+</sup>) calcd for C<sub>25</sub>H<sub>24</sub>NO<sub>4</sub>S [M + 1]<sup>+</sup>: 434.14, found 434.11.</div></div><div id="sec5_1_41" class="NLM_sec NLM_sec_level_3"><div id="ac_i55" class="anchor-spacer"></div><h4 class="article-section__title" id="_i55"> <i>tert</i>-Butyl (2<i>S</i>,4<i>R</i>)-4-Hydroxy-2-(((<i>S</i>)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)carbamoyl)-pyrrolidine-1-carboxylate (<b>56</b>)</h4><div class="NLM_p">Compound <b>55</b>, synthesized using the reported procedure,<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> was dissolved in 4 N HCl in dioxane (25 mL, 100 mmol) and MeOH (25 mL), and the mixture was stirred at ambient temperature for 12 h. The mixture was concentrated, and the residue was dried under vacuum to afford the intermediate, which was used in next step without further purification.</div><div class="NLM_p last">HATU (14.51 g, 38.2 mmol, 1.2 equiv) was added to a solution of the intermediate (<b>55</b>) obtained above (6.95 g, 31.8 mmol, 1.0 equiv), (2<i>S</i>,4<i>R</i>)-1-(<i>tert</i>-Butoxycarbonyl)-4-hydroxypyrrolidine-2-carboxylic acid (7.36 g, 31.8 mmol, 1.0 equiv), and DIPEA (11.08 mL, 63.6 mmol, 2.0 equiv) in DMF (36 mL) at 0 °C under nitrogen. The mixture was stirred at ambient temperature for 12 h. TLC showed that the reaction was complete. The reaction mixture was quenched with H<sub>2</sub>O (200 mL) and extracted with EtOAc (150 mL × 2). The combined organic layer was washed with brine (200 mL) and dried over Na<sub>2</sub>SO<sub>4</sub>. The organic solution was filtered and concentrated. The residue was purified by silica gel flash column chromatography with hexane:EtOAc (100:1–1:100), then DCM:MeOH (10:1) to afford the intermediate (<b>56</b>) as white solid (10.98 g, 80% yield). <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz) δ (ppm) 8.84 (s, 1H), 7.43–7.37 (m, 4H), 5.11–5.07 (m, 1H), 4.44–4.37 (m, 2H), 3.60–3.46 (m, 2H), 2.44 (s, 3H), 2.27–2.22 (m, 1H), 1.98–1.91 (m, 1H), 1.50 (d, <i>J</i> = 7.2 Hz, 3H), 1.46 (s, 9H). UPLC-MS (ESI<sup>+</sup>) calcd for C<sub>22</sub>H<sub>30</sub>N<sub>3</sub>O<sub>4</sub>S [M + 1]<sup>+</sup>: 432.20, found 432.20.</div></div><div id="sec5_1_42" class="NLM_sec NLM_sec_level_3"><div id="ac_i56" class="anchor-spacer"></div><h4 class="article-section__title" id="_i56"> <i>tert</i>-Butyl ((<i>S</i>)-1-((2<i>S</i>,4<i>R</i>)-4-Hydroxy-2-(((<i>S</i>)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)carbamoyl)pyrrolidin-1-yl)-3,3-dimethyl-1-oxobutan-2-yl)carbamate (<b>57</b>)</h4><div class="NLM_p last">This solid material (<b>56</b>), obtained as described above was dissolved in 4 N HCl in dioxane (25 mL, 100 mmol) and 25 mL of MeOH, and the mixture was stirred at ambient temperature for 12 h. The mixture was then concentrated, and the residue was dried under vacuum to afford intermediate, which was used in next step without further purification. UPLC-MS (ESI<sup>+</sup>) calcd for C<sub>17</sub>H<sub>22</sub>N<sub>3</sub>O<sub>2</sub>S [M + 1]<sup>+</sup>: 332.14, found 332.11. HATU (1.37 g, 3.6 mmol, 1.2 equiv) was added to a solution of this intermediate (994 mg, 3.0 mmol, 1.0 equiv), (<i>S</i>)-2-((<i>tert</i>-butoxycarbonyl)amino)-3,3-dimethylbutanoic acid (694 mg, 3.0 mmol, 1.0 equiv), and DIPEA (1.57 mL, 9.0 mmol, 3.0 equiv) in DMF (10 mL) at 0 °C under N<sub>2</sub>. The mixture was stirred at ambient temperature for 12 h when TLC showed that the reaction was complete. The reaction mixture was quenched with H<sub>2</sub>O (100 mL) and extracted with EtOAc (75 mL × 2). The combined organic layer was washed with brine (100 mL) and dried over Na<sub>2</sub>SO<sub>4</sub>. The organic solution was filtered and concentrated. The residue was purified by silica gel flash column chromatography with hexane:EtOAc then DCM:MeOH to afford the desired compound (<b>57</b>) as white solid (1.31 g, 80% yield). <sup>1</sup>H NMR (CDCl<sub>3</sub>, 400 MHz) δ (ppm) 8.65 (s, 1H), 7.70 (d, <i>J</i> = 8.0 Hz, 1H), 7.35–7.31 (m, 4H), 5.29 (d, <i>J</i> = 9.2 Hz, 1H), 5.06–5.01 (m, 1H), 4.67 (t, <i>J</i> = 8.0 Hz, 1H), 4.46–4.44 (m, 1H), 4.22–4.19 (m, 1H), 3.91 (d, <i>J</i> = 17.2 Hz, 1H), 3.61–3.58 (m, 1H), 2.46 (s, 3H), 2.37–2.30 (m, 1H), 2.04–1.99 (m, 1H), 1.44 (d, <i>J</i> = 7.2 Hz, 3H), 1.35 (s, 9H), 0.96 (s, 9H). <sup>13</sup>C NMR (CDCl<sub>3</sub>, 100 MHz) δ (ppm) 172.22, 170.13, 156.15, 150.56, 148.21, 143.43, 131.74, 130.59, 129.49, 126.46, 80.18, 69.91, 58.86, 56.58, 48.74, 38.60, 36.02, 35.48, 28.34, 26.39, 22.17, 15.95. UPLC-MS (ESI<sup>+</sup>) calcd for C<sub>28</sub>H<sub>41</sub>N<sub>4</sub>O<sub>5</sub>S [M + 1]<sup>+</sup>: 545.28, found 545.35.</div></div><div id="sec5_1_43" class="NLM_sec NLM_sec_level_3"><div id="ac_i57" class="anchor-spacer"></div><h4 class="article-section__title" id="_i57"> (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-2-Amino-3,3-dimethylbutanoyl)-4-hydroxy-<i>N</i>-((<i>S</i>)-1-(4-(4-methylthiazol-5-yl)phenyl)ethyl)pyrrolidine-2-carboxamide (<b>58</b>)</h4><div class="NLM_p last">The solid material (<b>57</b>) obtained above was dissolved in 4 N HCl in dioxane (4 mL, 16 mmol) and MeOH (4.0 mL), and the mixture was stirred at ambient temperature for 12 h. The mixture was then concentrated and the residue was dried under vacuum to afford crude compound, which was purified by reversed-phase preparative HPLC to afford the pure final compound (<b>58</b>) as an off-white solid. UPLC-MS (ESI<sup>+</sup>) calcd for C<sub>23</sub>H<sub>33</sub>N<sub>4</sub>O<sub>3</sub>S [M + 1]<sup>+</sup>: 445.23, found 445.44.</div></div></div><div id="sec5_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i58" class="anchor-spacer"></div><h3 class="article-section__title" id="_i58"> Cell Culture</h3><div class="NLM_p last">Human breast cancer cell lines MCF-7 (ATCC HTB-22) and T47D (ATCC HTB-133) were purchased from the American Type Culture Collection (ATCC), Manassas, VA. They were maintained and cultured in Dulbecco’s Modified Eagle’s medium (DMEM) containing 10% fetal bovine serum, 1 unit/mL of penicillin, and 1 μg/mL of streptomycin. Cells with 3–8 passages after purchasing were used in experiments as indicated.</div></div><div id="sec5_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i59" class="anchor-spacer"></div><h3 class="article-section__title" id="_i59"> Western Blot Analysis</h3><div class="NLM_p last">Western blot analysis was performed essentially as described previously (Hu et al, 2015; PMID 26358219). Cells treated with indicated compounds were lysed in Radioimmunoprecipitation Assay Protein Lysis and Extraction Buffer (25 mmol/L Tris·HCl, pH 7.6, 150 mmol/L NaCl, 1% Nonidet P-40, 1% sodium deoxycholate, and 0.1% sodium dodecyl sulfate) containing proteinase inhibitor cocktail (Roche Diagnostics, Mannheim, Germany). After determination of protein concentration by BCA assay (Fisher Scientific, Pittsburgh, PA), equal amounts of total protein were electrophoresed through 10% SDS-polyacrylamide gels. The separated protein bands were transferred onto PVDF membranes (GE Healthcare Life Sciences, Marlborough, MA) and blotted against different antibodies as indicated. The human estrogen receptor α antibody (AB16460) were purchased from Abcam, Inc., Cambridge, MA. The membranes were reblotted with horseradish peroxidase-conjugated anti-glyceraldehyde-3-phosphate dehydrogenase antibody (G9295) from Sigma-Aldrich Corporation, St. Louis, MO. The blots were scanned, and the band intensities were quantified using <a href="http://GelQuant.NET" class="ext-link">GelQuant.NET</a> software provided by <a href="http://biochemlabsolutions.com" class="ext-link">biochemlabsolutions.com</a>. The relative mean intensity of target proteins was expressed after normalization to the intensity of glyceraldehyde-3-phosphate dehydrogenase bands from individual repeats.</div></div><div id="sec5_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i60" class="anchor-spacer"></div><h3 class="article-section__title" id="_i60"> Cell Growth Assay</h3><div class="NLM_p last">MCF-7 cancer cells or human mammary epithelial cells (HMEC) were seeded at 1500/well in 96-well plates overnight. One day after the seeding, they were treated with indicated doses of compounds, respectively. The growth of the cells was evaluated by colorimetric wst-8 assay 4 days after the compound treatment by following the instructions of the manufacturer, Cayman Chemical, Ann Arbor, MI.</div></div><div id="sec5_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i61" class="anchor-spacer"></div><h3 class="article-section__title" id="_i61"> Molecular Modeling</h3><div class="NLM_p last">The binding pose of the <i>N</i>,<i>N</i>-diethylamino analogue of raloxifene with ER was modeled with the structure (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1ERR">1ERR</a>)<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> cocrystallized with raloxifene using MOE program. Residues were rebuilt if atoms were missing based on the amber10 library in MOE, and protons were added using the “protonate 3D” module considered by setting pH at 7, temperature at 300 K, and salt concentration at 0.1 mol/L. Docking simulations were then performed using raloxifene to define the binding site with crystallized H<sub>2</sub>O molecules preserved. The ligand was placed by “Triangle matcher” and evaluated by London dG scoring. DGVI/WSA dG scoring was then applied to rank the poses, and the top ranked pose was selected. Figures appeared in this paper were prepared using the PyMOL program (<a href="http://www.pymol.org" class="extLink">www.pymol.org</a>).</div></div><div id="sec5_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i62" class="anchor-spacer"></div><h3 class="article-section__title" id="_i62"> Cloning and Purification of VHL-ElonginB/C Complex</h3><div class="NLM_p last">The DNA sequence of VHL (coding for residues 54–213) was constructed by PCR and inserted into a His-TEV expression vector<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(58)</a> using ligation-independent cloning. The DNA sequences of Elongin B (encoding residues 1–118) and Elongin C (encoding residues 1–96) were constructed by PCR and inserted into pCDFDuet 1 using Gibson assembly.<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(59)</a> BL21(DE3) cells were transformed simultaneously with both plasmids and grown in Terrific broth at 37 °C until an OD600 of 1.2. The cells were induced overnight with 0.4 mM IPTG at 24 °C. Pelleted cells were freeze–thawed then resuspended in 20 mM Tris HCl pH 7.0, 200 mM NaCl, and 0.1% β-mercaptoethanol (bME) containing protease inhibitors. The cell suspension was lysed by sonication and debris removed via centrifugation. The supernatant was incubated at 4 °C for 1 h with Ni-NTA (Qiagen) prewashed in 20 mM Tris-HCl pH 7.0, 200 mM NaCl, and 10 mM imidazole. The protein complex was eluted in 20 mM Tris-HCl pH 7.0, 200 mM NaCl, and 300 mM imidazole, dialyzed into 20 mM Tris-HCl pH 7.0, 150 mM NaCl, and 0.01% bME, and incubated with TEV protease overnight at 4 °C. The protein sample was reapplied to the Ni-NTA column to remove the His-tag. The flow through containing the VHL complex was diluted to 75 mM NaCl and applied to a HiTrap Q column (GE Healthcare). The sample was eluted with a salt gradient (0.075–1 M NaCl), concentrated and further purified on a Superdex S75 column (GE Healthcare) pre-equilibrated with 20 mM Bis-Tris 7.0, 150 mM NaCl, and 1 mM DTT. Samples were aliquoted and stored at −80 °C.</div></div><div id="sec5_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i63" class="anchor-spacer"></div><h3 class="article-section__title" id="_i63"> Determination of Binding Affinities of Synthesized VHL Ligands to VHL-ElonginB/C Complex (VHL)</h3><div class="NLM_p">We have developed a competitive, fluorescence polarization (FP) assay to determine the binding affinities of our synthesized VHL ligands to VHL-ElonginB/C complex (VHL) protein. For this purpose, we synthesized a fluorencently tagged VHL ligand (VHL-F1) (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01572/suppl_file/jm8b01572_si_001.pdf" class="ext-link">Figure S1(A)</a> in Supporting Information) and determined its <i>K</i><sub>d</sub> value to be 2.3 ± 0.3 nM (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01572/suppl_file/jm8b01572_si_001.pdf" class="ext-link">Figure S1(B)</a> in Supporting Information).</div><div class="NLM_p last">IC<sub>50</sub> and <i>K</i><sub>i</sub> values of VHL ligands to VHL protein were then determined in competitive binding experiments. Mixtures of 5 μL of each compound in DMSO and 95 μL of preincubated VHL protein/VHL-F1 tracer complex solution were added into assay plates which were incubated at room temperature for 60 min with gentle shaking. Final concentrations of VHL protein and VHL-F1 fluorescent probe were both 5 nM. Negative controls containing protein/probe complex only (equivalent to 0% inhibition) and positive controls containing only the probe (equivalent to 100% inhibition) were included in each assay plate. FP values in millipolarization units (mP) were measured using the PHERAstar FSX HTS plate reader (BMG Labtech, 13000 Weston Parkway, Suite 109, Cary, NC 27513, USA) in 96-well, black, flat bottom microplates (Greiner Bio-One North America, Inc. 4238 Capital Drive Monroe, NC, 28110, USA) at an excitation wavelength of 485 nm and an emission wavelength of 530 nm. IC<sub>50</sub> values were determined by nonlinear regression fitting of the competition curves. <i>K</i><sub>i</sub> values of competitive inhibitors were obtained directly by nonlinear regression fitting, based upon the <i>K</i><sub>D</sub> values of the probe and concentrations of the protein and probe in the competitive assays. All the FP competitive experiments were performed in duplicate in three independent experiments.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i64"><a href="/doi/suppl/10.1021/acs.jmedchem.8b01572" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_06326" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_06326" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.8b01572" class="ext-link">10.1021/acs.jmedchem.8b01572</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Structure characterization for compound <b>11</b>, VH032, <b>43a</b>–<b>48a</b>; <sup>1</sup>H NMR, <sup>13</sup>C NMR, and UPLC-MS analysis of compounds <b>15</b> (ERD-148) and <b>32</b> (ERD-308 (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01572/suppl_file/jm8b01572_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular string files for all the final target compounds (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01572/suppl_file/jm8b01572_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b01572/suppl_file/jm8b01572_si_001.pdf">jm8b01572_si_001.pdf (1.57 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b01572/suppl_file/jm8b01572_si_002.csv">jm8b01572_si_002.csv (7.47 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.8b01572" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_46454" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_46454" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Shaomeng Wang</span> - <span class="hlFld-Affiliation affiliation">The
Rogel Cancer Center, 
    Departments of Internal Medicine, 
    Pharmacology, University of Michigan
Medical School, 
    Department of Medicinal Chemistry, College of Pharmacy, 
    ,  and , , , University of Michigan, Ann Arbor, Michigan 48109, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-8782-6950" title="Orcid link">http://orcid.org/0000-0002-8782-6950</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#22514a434d4f474c4562574f4b414a0c474657"><span class="__cf_email__" data-cfemail="72011a131d1f171c1532071f1b111a5c171607">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jiantao Hu</span> - <span class="hlFld-Affiliation affiliation">The
Rogel Cancer Center, 
    Departments of Internal Medicine, 
    ,  and , , , University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Biao Hu</span> - <span class="hlFld-Affiliation affiliation">The
Rogel Cancer Center, 
    Departments of Internal Medicine, 
    ,  and , , , University of Michigan, Ann Arbor, Michigan 48109, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-4691-6490" title="Orcid link">http://orcid.org/0000-0002-4691-6490</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mingliang Wang</span> - <span class="hlFld-Affiliation affiliation">The
Rogel Cancer Center, 
    Departments of Internal Medicine, 
    ,  and , , , University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Fuming Xu</span> - <span class="hlFld-Affiliation affiliation">The
Rogel Cancer Center, 
    Departments of Internal Medicine, 
    ,  and , , , University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Bukeyan Miao</span> - <span class="hlFld-Affiliation affiliation">The
Rogel Cancer Center, 
    Departments of Internal Medicine, 
    ,  and , , , University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Chao-Yie Yang</span> - <span class="hlFld-Affiliation affiliation">The
Rogel Cancer Center, 
    Departments of Internal Medicine, 
    ,  and , , , University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mi Wang</span> - <span class="hlFld-Affiliation affiliation">The
Rogel Cancer Center, 
    Departments of Internal Medicine, 
    ,  and , , , University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Zhaomin Liu</span> - <span class="hlFld-Affiliation affiliation">The
Rogel Cancer Center, 
    Departments of Internal Medicine, 
    ,  and , , , University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Daniel F. Hayes</span> - <span class="hlFld-Affiliation affiliation">The
Rogel Cancer Center, 
    Departments of Internal Medicine, 
    ,  and , , , University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Krishnapriya Chinnaswamy</span> - <span class="hlFld-Affiliation affiliation">Life Sciences Institute,  and , , , University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">James Delproposto</span> - <span class="hlFld-Affiliation affiliation">Life Sciences Institute,  and , , , University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jeanne Stuckey</span> - <span class="hlFld-Affiliation affiliation">Life Sciences Institute,  and , , , University of Michigan, Ann Arbor, Michigan 48109, United States</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>Jiantao Hu, Biao Hu, Mingliang Wang, Fuming Xu, and Bukeyan Miao contributed equally.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare the following competing financial interest(s): The University of Michigan has filed a patent application on these ER degraders, which has been licensed by Oncopia Therapeutics LLC. S. Wang, J. Hu, B. Hu, M. Wang, F. Xu, and B. Miao are co-inventors on this patent application. The University of Michigan has received a research grant from Oncopia and owns equity in Oncipia. S. Wang is a co-founder of Oncopia, owns equity in Oncopia and is a paid consultant for Oncopia.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i66">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_02686" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_02686" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This study is supported in part by funding from Oncopia Therapeutics, LLC, the University of Michigan Comprehensive Cancer Center Strategic Fund for Breast Cancer (to S.W.), the University of Michigan Comprehensive Cancer Center Core Grant from the National Cancer Institute, NIH (grant P30CA046592).</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">BC</td><td class="NLM_def"><p class="first last">breast cancer</p></td></tr><tr><td class="NLM_term">ER</td><td class="NLM_def"><p class="first last">estrogen receptor</p></td></tr><tr><td class="NLM_term">HER2</td><td class="NLM_def"><p class="first last">human epidermal growth factor 2</p></td></tr><tr><td class="NLM_term">SERM</td><td class="NLM_def"><p class="first last">selective estrogen receptor modulator</p></td></tr><tr><td class="NLM_term">SERD</td><td class="NLM_def"><p class="first last">selective estrogen receptor degrader</p></td></tr><tr><td class="NLM_term">VHL</td><td class="NLM_def"><p class="first last">von Hippel-Lindau</p></td></tr><tr><td class="NLM_term">CRBN</td><td class="NLM_def"><p class="first last">cereblon</p></td></tr><tr><td class="NLM_term">FP</td><td class="NLM_def"><p class="first last">fluorescence polarization</p></td></tr><tr><td class="NLM_term">GAPDH</td><td class="NLM_def"><p class="first last">glyceraldehyde 3-phosphate dehydrogenase</p></td></tr><tr><td class="NLM_term">PBS</td><td class="NLM_def"><p class="first last">phosphate buffered saline</p></td></tr><tr><td class="NLM_term">SNIPER</td><td class="NLM_def"><p class="first last">specific and nongenetic IAP-dependent protein erasers</p></td></tr><tr><td class="NLM_term">PROTAC</td><td class="NLM_def"><p class="first last">proteolysis targeting chimera</p></td></tr><tr><td class="NLM_term">DMSO</td><td class="NLM_def"><p class="first last">dimethyl sulfoxide</p></td></tr><tr><td class="NLM_term">HATU</td><td class="NLM_def"><p class="first last">hexafluorophosphate azabenzotriazole tetramethyl uronium</p></td></tr><tr><td class="NLM_term">DIPEA</td><td class="NLM_def"><p class="first last"><i>N</i>,<i>N</i>-diisopropylethylamine</p></td></tr><tr><td class="NLM_term">DMF</td><td class="NLM_def"><p class="first last"><i>N</i>,<i>N</i>-dimethylformamide</p></td></tr><tr><td class="NLM_term">DMA</td><td class="NLM_def"><p class="first last"><i>N</i>,<i>N</i>-dimethylacetamide</p></td></tr><tr><td class="NLM_term">qPCR</td><td class="NLM_def"><p class="first last">quantitative polymerase chain reaction</p></td></tr><tr><td class="NLM_term">pGR</td><td class="NLM_def"><p class="first last">progesterone receptor</p></td></tr><tr><td class="NLM_term">GREB1</td><td class="NLM_def"><p class="first last">growth regulation by estrogen in breast cancer 1</p></td></tr><tr><td class="NLM_term">HMEC</td><td class="NLM_def"><p class="first last">human mammary epithelial cells.</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i68">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_66000" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_66000" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 59 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tong, C. W. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, W. C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">To, K. K. W.</span></span> <span> </span><span class="NLM_article-title">Recent advances in the treatment of breast cancer</span>. <i>Front. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">227</span>, <span class="refDoi"> DOI: 10.3389/fonc.2018.00227</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=10.3389%2Ffonc.2018.00227" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=29963498" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A280%3ADC%252BB3c%252FjvVylsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2018&pages=227&author=C.+W.+S.+Tongauthor=M.+Wuauthor=W.+C.+S.+Choauthor=K.+K.+W.+To&title=Recent+advances+in+the+treatment+of+breast+cancer&doi=10.3389%2Ffonc.2018.00227"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Recent Advances in the Treatment of Breast Cancer</span></div><div class="casAuthors">Tong Christy W S; Wu Mingxia; To Kenneth K W; Cho William C S</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in oncology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">227</span>
        ISSN:<span class="NLM_cas:issn">2234-943X</span>.
    </div><div class="casAbstract">Breast cancer (BC) is the most common malignancy in women.  It is classified into a few major molecular subtypes according to hormone and growth factor receptor expression.  Over the past few years, substantial advances have been made in the discovery of new drugs for treating BC.  Improved understanding of the biologic heterogeneity of BC has allowed the development of more effective and individualized approach to treatment.  In this review, we provide an update about the current treatment strategy and discuss the various emerging novel therapies for the major molecular subtypes of BC.  A brief account of the clinical development of inhibitors of poly(ADP-ribose) polymerase, cyclin-dependent kinases 4 and 6, phosphatidylinositol 3-kinase/protein kinase B/mammalian target of rapamycin pathway, histone deacetylation, multi-targeting tyrosine kinases, and immune checkpoints for personalized treatment of BC is included.  However, no targeted drug has been approved for the most aggressive subtype-triple negative breast cancer (TNBC).  Thus, we discuss the heterogeneity of TNBC and how molecular subtyping of TNBC may help drug discovery for this deadly disease.  The emergence of drug resistance also poses threat to the successful development of targeted therapy in various molecular subtypes of BC.  New clinical trials should incorporate advanced methods to identify changes induced by drug treatment, which may be associated with the upregulation of compensatory signaling pathways in drug resistant cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS9aYrYWlne0BJ5E29YNGvQfW6udTcc2eZvkxoa8lX_urntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3c%252FjvVylsw%253D%253D&md5=a116dcf63d5640023aac066b920bdc5f</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.3389%2Ffonc.2018.00227&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffonc.2018.00227%26sid%3Dliteratum%253Aachs%26aulast%3DTong%26aufirst%3DC.%2BW.%2BS.%26aulast%3DWu%26aufirst%3DM.%26aulast%3DCho%26aufirst%3DW.%2BC.%2BS.%26aulast%3DTo%26aufirst%3DK.%2BK.%2BW.%26atitle%3DRecent%2520advances%2520in%2520the%2520treatment%2520of%2520breast%2520cancer%26jtitle%3DFront.%2520Oncol.%26date%3D2018%26volume%3D8%26spage%3D227%26doi%3D10.3389%2Ffonc.2018.00227" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, W. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katki, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenberg, P. S.</span></span> <span> </span><span class="NLM_article-title">Incidence of breast cancer in the United States: current and future trends</span>. <i>J. Natl. Cancer Inst.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>103</i></span>,  <span class="NLM_fpage">1397</span>– <span class="NLM_lpage">1402</span>, <span class="refDoi"> DOI: 10.1093/jnci/djr257</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=10.1093%2Fjnci%2Fdjr257" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=21753181" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A280%3ADC%252BC3MfltVSgtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=2011&pages=1397-1402&author=W.+F.+Andersonauthor=H.+A.+Katkiauthor=P.+S.+Rosenberg&title=Incidence+of+breast+cancer+in+the+United+States%3A+current+and+future+trends&doi=10.1093%2Fjnci%2Fdjr257"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Incidence of breast cancer in the United States: current and future trends</span></div><div class="casAuthors">Anderson William F; Katki Hormuzd A; Rosenberg Philip S</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the National Cancer Institute</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">103</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">1397-402</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  The incidence of breast cancer increased in the United States until circa 2000 then decreased, mostly among women with estrogen receptor (ER)-positive cancers.  Time trends provide important clues for cancer etiology and prevention; however, the observed trends of ER-positive and ER-negative breast cancers can be biased by missing ER data.  METHODS:  We developed a simple imputation method to correct invasive female breast cancer incidence for missing or unknown ER expression, using nationally representative data from the National Cancer Institute's Surveillance, Epidemiology, and End Results Program during 1980-2008, including 588,720 invasive female breast cancer patients with 471,336,233 woman-years of follow-up.  Corrected rates of ER-positive and ER-negative breast cancers were used to calculate age-standardized incidence rates, estimated annual percentage changes, and projections derived from age-period-cohort models.  RESULTS:  The recent decrease in the incidence of breast cancer overall stabilized near 200 per 100,000 woman-years by 2007-2008, reflecting a transient decrease in ER-positive cancers and a steady decrease in ER-negative cancers.  The projected incidence rate for breast cancer overall through the year 2016 was similar to the incidence rate during 2007-2008.  In contrast, rates of ER-positive breast cancers were projected to increase 5.3% (95% confidence interval = 5.2% to 5.4%), whereas rates of ER-negative breast cancers were projected to decrease 11.4% (95% confidence interval = 11.3% to 11.6%) during 2009-2016.  CONCLUSION:  Recent changes in breast cancer incidence overall reflect the superimposition of divergent trends in ER-positive and ER-negative cancers.  If current trends continue, the incidence of ER-positive breast cancers will increase, the incidence of ER-negative breast cancers will continue to decrease, and the incidence of breast cancer overall will remain similar to its current level.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ2E9JRP4-GvK__Jf7-eMvJfW6udTcc2eZvkxoa8lX_urntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3MfltVSgtg%253D%253D&md5=c4468ec6b42936d658f0274561120a6c</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1093%2Fjnci%2Fdjr257&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjnci%252Fdjr257%26sid%3Dliteratum%253Aachs%26aulast%3DAnderson%26aufirst%3DW.%2BF.%26aulast%3DKatki%26aufirst%3DH.%2BA.%26aulast%3DRosenberg%26aufirst%3DP.%2BS.%26atitle%3DIncidence%2520of%2520breast%2520cancer%2520in%2520the%2520United%2520States%253A%2520current%2520and%2520future%2520trends%26jtitle%3DJ.%2520Natl.%2520Cancer%2520Inst.%26date%3D2011%26volume%3D103%26spage%3D1397%26epage%3D1402%26doi%3D10.1093%2Fjnci%2Fdjr257" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nilsson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koehler, K. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gustafsson, J. A.</span></span> <span> </span><span class="NLM_article-title">Development of subtype-selective oestrogen receptor-based therapeutics</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">778</span>– <span class="NLM_lpage">792</span>, <span class="refDoi"> DOI: 10.1038/nrd3551</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=10.1038%2Fnrd3551" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2011&pages=778-792&author=S.+Nilssonauthor=K.+F.+Koehlerauthor=J.+A.+Gustafsson&title=Development+of+subtype-selective+oestrogen+receptor-based+therapeutics&doi=10.1038%2Fnrd3551"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnrd3551&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3551%26sid%3Dliteratum%253Aachs%26aulast%3DNilsson%26aufirst%3DS.%26aulast%3DKoehler%26aufirst%3DK.%2BF.%26aulast%3DGustafsson%26aufirst%3DJ.%2BA.%26atitle%3DDevelopment%2520of%2520subtype-selective%2520oestrogen%2520receptor-based%2520therapeutics%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2011%26volume%3D10%26spage%3D778%26epage%3D792%26doi%3D10.1038%2Fnrd3551" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jordan, V. C.</span></span> <span> </span><span class="NLM_article-title">Tamoxifen: a most unlikely pioneering medicine</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">205</span>– <span class="NLM_lpage">213</span>, <span class="refDoi"> DOI: 10.1038/nrd1031</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=10.1038%2Fnrd1031" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=12612646" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A528%3ADC%252BD3sXhsFKrtL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2003&pages=205-213&author=V.+C.+Jordan&title=Tamoxifen%3A+a+most+unlikely+pioneering+medicine&doi=10.1038%2Fnrd1031"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Tamoxifen: A most unlikely pioneering medicine</span></div><div class="casAuthors">Jordan, V. Craig</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">205-213</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  For >25 yr, tamoxifen has been the gold std. for the endocrine treatment of all stages of estrogen receptor-pos. breast cancer, and the World Health Organization lists tamoxifen as an essential drug for the treatment of breast cancer.  It is estd. that >400,000 women are alive today as a result of tamoxifen therapy, and millions more have benefited from palliation and extended disease-free survival.  Interestingly, tamoxifen also became the first cancer chemopreventive approved by the Food and Drug Administration (FDA) for the redn. of breast-cancer incidence in both pre- and postmenopausal women at high risk.  However, 40 yr ago, it was hard to imagine that a nontoxic targeted treatment for breast cancer could be developed at all.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9q3QlfhnxLrVg90H21EOLACvtfcHk0liTlVh33B_01Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXhsFKrtL0%253D&md5=f5574615d30cb7dab87cf9d812104a83</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fnrd1031&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd1031%26sid%3Dliteratum%253Aachs%26aulast%3DJordan%26aufirst%3DV.%2BC.%26atitle%3DTamoxifen%253A%2520a%2520most%2520unlikely%2520pioneering%2520medicine%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2003%26volume%3D2%26spage%3D205%26epage%3D213%26doi%3D10.1038%2Fnrd1031" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Das, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crockett, J. C.</span></span> <span> </span><span class="NLM_article-title">Osteoporosis - a current view of pharmacological prevention and treatment</span>. <i>Drug Des., Dev. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">435</span>– <span class="NLM_lpage">448</span>, <span class="refDoi"> DOI: 10.2147/DDDT.S31504</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=10.2147%2FDDDT.S31504" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=23807838" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A528%3ADC%252BC3sXpslyis7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2013&pages=435-448&author=S.+Dasauthor=J.+C.+Crockett&title=Osteoporosis+-+a+current+view+of+pharmacological+prevention+and+treatment&doi=10.2147%2FDDDT.S31504"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Osteoporosis - a current view of pharmacological prevention and treatment</span></div><div class="casAuthors">Das, Subhajit; Crockett, Julie C.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Design, Development and Therapy</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">435-448</span>CODEN:
                <span class="NLM_cas:coden">DDDTAQ</span>;
        ISSN:<span class="NLM_cas:issn">1177-8881</span>.
    
            (<span class="NLM_cas:orgname">Dove Medical Press Ltd.</span>)
        </div><div class="casAbstract">A review.  Postmenopausal osteoporosis is the most common bone disease, assocd. with low bone mineral d. (BMD) and pathol. fractures which lead to significant morbidity.  It is defined clin. by a BMD of 2.5 std. deviations or more below the young female adult mean (T-score = -2.5).  Osteoporosis was a huge global problem both socially and economically - in the UK alone, in 2011 £6 million per day was spent on treatment and social care of the 230,000 osteoporotic fracture patients - and therefore viable preventative and therapeutic approaches are key to managing this problem within the aging population of today.  One of the main issues surrounding the potential of osteoporosis management is diagnosing patients at risk before they develop a fracture.  We discuss the current and future possibilities for identifying susceptible patients, from fracture risk assessment to shape modeling and in relation to the high heritability of osteoporosis now that a plethora of genes have been assocd. with low BMD and osteoporotic fracture.  This review highlights the current therapeutics in clin. use (including bisphosphonates, anti-RANKL [receptor activator of NF-κB ligand], intermittent low dose parathyroid hormone, and strontium ranelate) and some of those in development (anti-sclerostin antibodies and cathepsin K inhibitors).  By highlighting the intimate relationship between the activities of bone forming (osteoblasts) and bone-resorbing (osteoclasts) cells, we include an overview and comparison of the mol. mechanisms exploited in each therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbQ-qJlOT4Z7Vg90H21EOLACvtfcHk0liTlVh33B_01Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXpslyis7k%253D&md5=dd22c21c6d0f305321ebdc1f32051000</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.2147%2FDDDT.S31504&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2147%252FDDDT.S31504%26sid%3Dliteratum%253Aachs%26aulast%3DDas%26aufirst%3DS.%26aulast%3DCrockett%26aufirst%3DJ.%2BC.%26atitle%3DOsteoporosis%2520-%2520a%2520current%2520view%2520of%2520pharmacological%2520prevention%2520and%2520treatment%26jtitle%3DDrug%2520Des.%252C%2520Dev.%2520Ther.%26date%3D2013%26volume%3D7%26spage%3D435%26epage%3D448%26doi%3D10.2147%2FDDDT.S31504" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">De
Marchi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foekens, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Umar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martens, J. W.</span></span> <span> </span><span class="NLM_article-title">Endocrine therapy resistance in estrogen receptor (ER)-positive breast cancer</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">1181</span>– <span class="NLM_lpage">1188</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2016.05.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=10.1016%2Fj.drudis.2016.05.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=27233379" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A528%3ADC%252BC28Xoslagsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2016&pages=1181-1188&author=T.+De%0AMarchiauthor=J.+A.+Foekensauthor=A.+Umarauthor=J.+W.+Martens&title=Endocrine+therapy+resistance+in+estrogen+receptor+%28ER%29-positive+breast+cancer&doi=10.1016%2Fj.drudis.2016.05.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Endocrine therapy resistance in estrogen receptor (ER)-positive breast cancer</span></div><div class="casAuthors">De Marchi, Tommaso; Foekens, John A.; Umar, Arzu; Martens, John W. M.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1181-1188</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Estrogen receptor (ER)-pos. breast cancer represents the majority (∼70%) of all breast malignancies.  In this subgroup of breast cancers, endocrine therapies are effective both in the adjuvant and recurrent settings, although resistance remains a major issue.  Several high-throughput approaches have been used to elucidate mechanisms of resistance and to derive potential predictive markers or alternative therapies.  In this review, we cover the state-of-the-art of endocrine-resistance biomarker discovery with regard to the latest technol. developments, and discuss current opportunities and restrictions for their implementation into a clin. setting.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovT4-cqEBi7bVg90H21EOLACvtfcHk0liTlVh33B_01Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xoslagsb8%253D&md5=133921c6a09077c5c2f1bcab1c565c1a</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2016.05.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2016.05.012%26sid%3Dliteratum%253Aachs%26aulast%3DDe%2BMarchi%26aufirst%3DT.%26aulast%3DFoekens%26aufirst%3DJ.%2BA.%26aulast%3DUmar%26aufirst%3DA.%26aulast%3DMartens%26aufirst%3DJ.%2BW.%26atitle%3DEndocrine%2520therapy%2520resistance%2520in%2520estrogen%2520receptor%2520%2528ER%2529-positive%2520breast%2520cancer%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2016%26volume%3D21%26spage%3D1181%26epage%3D1188%26doi%3D10.1016%2Fj.drudis.2016.05.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">AlFakeeh, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brezden-Masley, C.</span></span> <span> </span><span class="NLM_article-title">Overcoming endocrine resistance in hormone receptor-positive breast cancer</span>. <i>Curr. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">18</span>– <span class="NLM_lpage">27</span>, <span class="refDoi"> DOI: 10.3747/co.25.3752</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=10.3747%2Fco.25.3752" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=18-27&author=A.+AlFakeehauthor=C.+Brezden-Masley&title=Overcoming+endocrine+resistance+in+hormone+receptor-positive+breast+cancer&doi=10.3747%2Fco.25.3752"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.3747%2Fco.25.3752&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3747%252Fco.25.3752%26sid%3Dliteratum%253Aachs%26aulast%3DAlFakeeh%26aufirst%3DA.%26aulast%3DBrezden-Masley%26aufirst%3DC.%26atitle%3DOvercoming%2520endocrine%2520resistance%2520in%2520hormone%2520receptor-positive%2520breast%2520cancer%26jtitle%3DCurr.%2520Oncol.%26date%3D2018%26volume%3D25%26spage%3D18%26epage%3D27%26doi%3D10.3747%2Fco.25.3752" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Martin, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ribas, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simigdala, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuster, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pancholi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tenev, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gellert, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buluwela, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harrod, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thornhill, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nikitorowicz-Buniak, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhamra, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turgeon, M. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poulogiannis, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martins, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hills, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia-Murillas, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fribbens, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patani, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sikora, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turner, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwart, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oesterreich, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carroll, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ali, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dowsett, M.</span></span> <span> </span><span class="NLM_article-title">Discovery of naturally occurring ESR1 mutations in breast cancer cell lines modelling endocrine resistance</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1865</span>, <span class="refDoi"> DOI: 10.1038/s41467-017-01864-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=10.1038%2Fs41467-017-01864-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=29192207" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A280%3ADC%252BC1M3ms1CqtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=1865&author=L.+A.+Martinauthor=R.+Ribasauthor=N.+Simigdalaauthor=E.+Schusterauthor=S.+Pancholiauthor=T.+Tenevauthor=P.+Gellertauthor=L.+Buluwelaauthor=A.+Harrodauthor=A.+Thornhillauthor=J.+Nikitorowicz-Buniakauthor=A.+Bhamraauthor=M.+O.+Turgeonauthor=G.+Poulogiannisauthor=Q.+Gaoauthor=V.+Martinsauthor=M.+Hillsauthor=I.+Garcia-Murillasauthor=C.+Fribbensauthor=N.+Pataniauthor=Z.+Liauthor=M.+J.+Sikoraauthor=N.+Turnerauthor=W.+Zwartauthor=S.+Oesterreichauthor=J.+Carrollauthor=S.+Aliauthor=M.+Dowsett&title=Discovery+of+naturally+occurring+ESR1+mutations+in+breast+cancer+cell+lines+modelling+endocrine+resistance&doi=10.1038%2Fs41467-017-01864-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of naturally occurring ESR1 mutations in breast cancer cell lines modelling endocrine resistance</span></div><div class="casAuthors">Martin Lesley-Ann; Ribas Ricardo; Simigdala Nikiana; Schuster Eugene; Pancholi Sunil; Tenev Tencho; Gellert Pascal; Nikitorowicz-Buniak Joanna; Gao Qiong; Garcia-Murillas Isaac; Fribbens Charlotte; Patani Neill; Turner Nicholas; Dowsett Mitch; Buluwela Laki; Harrod Alison; Ali Simak; Thornhill Allan; Bhamra Amandeep; Turgeon Marc-Olivier; Poulogiannis George; Poulogiannis George; Martins Vera; Hills Margaret; Dowsett Mitch; Li Zheqi; Sikora Matthew J; Oesterreich Steffi; Zwart Wilbert; Carroll Jason</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1865</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Resistance to endocrine therapy remains a major clinical problem in breast cancer.  Genetic studies highlight the potential role of estrogen receptor-α (ESR1) mutations, which show increased prevalence in the metastatic, endocrine-resistant setting.  No naturally occurring ESR1 mutations have been reported in in vitro models of BC either before or after the acquisition of endocrine resistance making functional consequences difficult to study.  We report the first discovery of naturally occurring ESR1 (Y537C) and ESR1 (Y537S) mutations in MCF7 and SUM44 ESR1-positive cell lines after acquisition of resistance to long-term-estrogen-deprivation (LTED) and subsequent resistance to fulvestrant (ICIR).  Mutations were enriched with time, impacted on ESR1 binding to the genome and altered the ESR1 interactome.  The results highlight the importance and functional consequence of these mutations and provide an important resource for studying endocrine resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTMmRurTV1xdhBrI9DTGjC0fW6udTcc2eZTLAe1K-4kw7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M3ms1CqtA%253D%253D&md5=0ff681e49e4de73dd6822b02fe85e096</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fs41467-017-01864-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-017-01864-y%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DL.%2BA.%26aulast%3DRibas%26aufirst%3DR.%26aulast%3DSimigdala%26aufirst%3DN.%26aulast%3DSchuster%26aufirst%3DE.%26aulast%3DPancholi%26aufirst%3DS.%26aulast%3DTenev%26aufirst%3DT.%26aulast%3DGellert%26aufirst%3DP.%26aulast%3DBuluwela%26aufirst%3DL.%26aulast%3DHarrod%26aufirst%3DA.%26aulast%3DThornhill%26aufirst%3DA.%26aulast%3DNikitorowicz-Buniak%26aufirst%3DJ.%26aulast%3DBhamra%26aufirst%3DA.%26aulast%3DTurgeon%26aufirst%3DM.%2BO.%26aulast%3DPoulogiannis%26aufirst%3DG.%26aulast%3DGao%26aufirst%3DQ.%26aulast%3DMartins%26aufirst%3DV.%26aulast%3DHills%26aufirst%3DM.%26aulast%3DGarcia-Murillas%26aufirst%3DI.%26aulast%3DFribbens%26aufirst%3DC.%26aulast%3DPatani%26aufirst%3DN.%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DSikora%26aufirst%3DM.%2BJ.%26aulast%3DTurner%26aufirst%3DN.%26aulast%3DZwart%26aufirst%3DW.%26aulast%3DOesterreich%26aufirst%3DS.%26aulast%3DCarroll%26aufirst%3DJ.%26aulast%3DAli%26aufirst%3DS.%26aulast%3DDowsett%26aufirst%3DM.%26atitle%3DDiscovery%2520of%2520naturally%2520occurring%2520ESR1%2520mutations%2520in%2520breast%2520cancer%2520cell%2520lines%2520modelling%2520endocrine%2520resistance%26jtitle%3DNat.%2520Commun.%26date%3D2017%26volume%3D8%26spage%3D1865%26doi%3D10.1038%2Fs41467-017-01864-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nardone, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Angelis, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trivedi, M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osborne, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiff, R.</span></span> <span> </span><span class="NLM_article-title">The changing role of ER in endocrine resistance</span>. <i>Breast</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">S60</span>– <span class="NLM_lpage">S66</span>, <span class="refDoi"> DOI: 10.1016/j.breast.2015.07.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=10.1016%2Fj.breast.2015.07.015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=26271713" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2015&pages=S60-S66&author=A.+Nardoneauthor=C.+De+Angelisauthor=M.+V.+Trivediauthor=C.+K.+Osborneauthor=R.+Schiff&title=The+changing+role+of+ER+in+endocrine+resistance&doi=10.1016%2Fj.breast.2015.07.015"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.breast.2015.07.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.breast.2015.07.015%26sid%3Dliteratum%253Aachs%26aulast%3DNardone%26aufirst%3DA.%26aulast%3DDe%2BAngelis%26aufirst%3DC.%26aulast%3DTrivedi%26aufirst%3DM.%2BV.%26aulast%3DOsborne%26aufirst%3DC.%2BK.%26aulast%3DSchiff%26aufirst%3DR.%26atitle%3DThe%2520changing%2520role%2520of%2520ER%2520in%2520endocrine%2520resistance%26jtitle%3DBreast%26date%3D2015%26volume%3D24%26spage%3DS60%26epage%3DS66%26doi%3D10.1016%2Fj.breast.2015.07.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Robertson, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harrison, M.</span></span> <span> </span><span class="NLM_article-title">Fulvestrant: pharmacokinetics and pharmacology</span>. <i>Br. J. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>90</i></span>,  <span class="NLM_fpage">S7</span>– <span class="NLM_lpage">S10</span>, <span class="refDoi"> DOI: 10.1038/sj.bjc.6601630</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=10.1038%2Fsj.bjc.6601630" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=15094758" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A528%3ADC%252BD2cXjtFaktbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=90&publication_year=2004&pages=S7-S10&author=J.+F.+Robertsonauthor=M.+Harrison&title=Fulvestrant%3A+pharmacokinetics+and+pharmacology&doi=10.1038%2Fsj.bjc.6601630"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Fulvestrant: pharmacokinetics and pharmacology</span></div><div class="casAuthors">Robertson, J. F. R.; Harrison, M.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Cancer</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">90</span>
        (<span class="NLM_cas:issue">Suppl. 1</span>),
    <span class="NLM_cas:pages">S7-S10</span>CODEN:
                <span class="NLM_cas:coden">BJCAAI</span>;
        ISSN:<span class="NLM_cas:issn">0007-0920</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Fulvestrant is a new type of estrogen receptor (ER) antagonist with no agonist activity and a novel pharmacol. profile.  Fulvestrant has been shown to significantly reduce cellular levels of the ER and progesterone receptor in both preclin. studies and in clin. trials of postmenopausal women with primary breast cancer.  This paper reviews the pharmacokinetics and metab. of fulvestrant, which support the rationale for drug delivery as a single, once-monthly i.m. injection, and show that this agent has minimal potential to be the subject, or cause, of significant cytochrome P 450-mediated drug interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqThGy3GHRDRrVg90H21EOLACvtfcHk0lglCJL1kf4p_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXjtFaktbc%253D&md5=d4a14a84081b408e9c60a3e89968a70a</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fsj.bjc.6601630&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.bjc.6601630%26sid%3Dliteratum%253Aachs%26aulast%3DRobertson%26aufirst%3DJ.%2BF.%26aulast%3DHarrison%26aufirst%3DM.%26atitle%3DFulvestrant%253A%2520pharmacokinetics%2520and%2520pharmacology%26jtitle%3DBr.%2520J.%2520Cancer%26date%3D2004%26volume%3D90%26spage%3DS7%26epage%3DS10%26doi%3D10.1038%2Fsj.bjc.6601630" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Osborne, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wakeling, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicholson, R. I.</span></span> <span> </span><span class="NLM_article-title">Fulvestrant: an oestrogen receptor antagonist with a novel mechanism of action</span>. <i>Br. J. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>90</i></span>,  <span class="NLM_fpage">S2</span>– <span class="NLM_lpage">S6</span>, <span class="refDoi"> DOI: 10.1038/sj.bjc.6601629</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=10.1038%2Fsj.bjc.6601629" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=15094757" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A528%3ADC%252BD2cXjtFaktbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=90&publication_year=2004&pages=S2-S6&author=C.+K.+Osborneauthor=A.+Wakelingauthor=R.+I.+Nicholson&title=Fulvestrant%3A+an+oestrogen+receptor+antagonist+with+a+novel+mechanism+of+action&doi=10.1038%2Fsj.bjc.6601629"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Fulvestrant: an estrogen receptor antagonist with a novel mechanism of action</span></div><div class="casAuthors">Osborne, C. K.; Wakeling, A.; Nicholson, R. I.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Cancer</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">90</span>
        (<span class="NLM_cas:issue">Suppl. 1</span>),
    <span class="NLM_cas:pages">S2-S6</span>CODEN:
                <span class="NLM_cas:coden">BJCAAI</span>;
        ISSN:<span class="NLM_cas:issn">0007-0920</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Due to their favorable tolerability profiles, endocrine therapies have long been considered the treatment of choice for hormone-sensitive metastatic breast cancer.  However, the estrogen agonist effects of the available selective estrogen receptor modulators, such as tamoxifen, and the development of cross-resistance between endocrine therapies with similar modes of action have led to the need for new treatments that act through different mechanisms.  Fulvestrant (Faslodex') is the first of a new type of endocrine treatment-an estrogen receptor (ER) antagonist that downregulates the ER and has no agonist effects.  This article provides an overview of the current understanding of ER signaling and illustrates the unique mode of action of fulvestrant.  Preclin. and clin. study data are presented in support of the novel mechanism of action of this new type of ER antagonist.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8myn0Hcy0CrVg90H21EOLACvtfcHk0lglCJL1kf4p_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXjtFaktbY%253D&md5=291a738f1d67db894e83d88ddfa000e4</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fsj.bjc.6601629&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.bjc.6601629%26sid%3Dliteratum%253Aachs%26aulast%3DOsborne%26aufirst%3DC.%2BK.%26aulast%3DWakeling%26aufirst%3DA.%26aulast%3DNicholson%26aufirst%3DR.%2BI.%26atitle%3DFulvestrant%253A%2520an%2520oestrogen%2520receptor%2520antagonist%2520with%2520a%2520novel%2520mechanism%2520of%2520action%26jtitle%3DBr.%2520J.%2520Cancer%26date%3D2004%26volume%3D90%26spage%3DS2%26epage%3DS6%26doi%3D10.1038%2Fsj.bjc.6601629" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Howell, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sapunar, F.</span></span> <span> </span><span class="NLM_article-title">Fulvestrant revisited: efficacy and safety of the 500-mg dose</span>. <i>Clin. Breast Cancer</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">204</span>– <span class="NLM_lpage">210</span>, <span class="refDoi"> DOI: 10.1016/j.clbc.2011.02.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=10.1016%2Fj.clbc.2011.02.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=21729658" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlKisr7E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2011&pages=204-210&author=A.+Howellauthor=F.+Sapunar&title=Fulvestrant+revisited%3A+efficacy+and+safety+of+the+500-mg+dose&doi=10.1016%2Fj.clbc.2011.02.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Fulvestrant revisited: efficacy and safety of the 500-mg dose</span></div><div class="casAuthors">Howell, Anthony; Sapunar, Francisco</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Breast Cancer</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">204-210</span>CODEN:
                <span class="NLM_cas:coden">CBCLB7</span>;
        ISSN:<span class="NLM_cas:issn">1526-8209</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review.  Postmenopausal women with hormone receptor-pos. advanced breast cancer are candidates for endocrine therapy.  As the disease will eventually progress in most patients, it is important to investigate agents with novel modes of action to reduce the likelihood of treatment cross-resistance.  Fulvestrant is an estrogen receptor antagonist with no known agonist effects that has been shown to be as effective as anastrozole following failure on tamoxifen, at the approved dose of 250 mg/mo.  However, pharmacokinetic modeling and evidence of clin. efficacy in early trials, together with the favorable tolerability profile of fulvestrant 250 mg, led to suggestions that increasing the fulvestrant dose would lead to an improved benefit-risk profile.  This review describes the rationale behind the development of a 500 mg/mo higher dose of fulvestrant and details relevant clin. trials, including the pivotal phase III COmparisoN of Faslodex In Recurrent or Metastatic breast cancer (CONFIRM) study.  CONFIRM demonstrated a significant improvement in progression-free survival for fulvestrant 500 mg vs. 250 mg in postmenopausal patients who had progressed on previous endocrine therapy.  Here, we present and discuss a pooled safety anal. of CONFIRM and three further clin. studies demonstrating fulvestrant 500 mg to be well-tolerated with no evidence of dose-related adverse events.  Overall, these data indicate an improved benefit-risk profile for fulvestrant 500 mg vs. 250 mg following failure on prior endocrine therapy, and suggest that fulvestrant 500 mg may be considered in future as initial endocrine treatment for advanced breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4U1d7Q6edlbVg90H21EOLACvtfcHk0lgRArth21UzCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlKisr7E&md5=0d1063c9546320294193e92d3f2097dd</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.clbc.2011.02.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.clbc.2011.02.002%26sid%3Dliteratum%253Aachs%26aulast%3DHowell%26aufirst%3DA.%26aulast%3DSapunar%26aufirst%3DF.%26atitle%3DFulvestrant%2520revisited%253A%2520efficacy%2520and%2520safety%2520of%2520the%2520500-mg%2520dose%26jtitle%3DClin.%2520Breast%2520Cancer%26date%3D2011%26volume%3D11%26spage%3D204%26epage%3D210%26doi%3D10.1016%2Fj.clbc.2011.02.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Robertson, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindemann, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garnett, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicholson, R. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuter, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gee, J. M.</span></span> <span> </span><span class="NLM_article-title">A good drug made better: the fulvestrant dose-response story</span>. <i>Clin. Breast Cancer</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">381</span>– <span class="NLM_lpage">389</span>, <span class="refDoi"> DOI: 10.1016/j.clbc.2014.06.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=10.1016%2Fj.clbc.2014.06.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=25457991" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFKmtrvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2014&pages=381-389&author=J.+F.+Robertsonauthor=J.+Lindemannauthor=S.+Garnettauthor=E.+Andersonauthor=R.+I.+Nicholsonauthor=I.+Kuterauthor=J.+M.+Gee&title=A+good+drug+made+better%3A+the+fulvestrant+dose-response+story&doi=10.1016%2Fj.clbc.2014.06.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">A Good Drug Made Better: The Fulvestrant Dose-Response Story</span></div><div class="casAuthors">Robertson, John F. R.; Lindemann, Justin; Garnett, Sally; Anderson, Elizabeth; Nicholson, Robert I.; Kuter, Irene; Gee, Julia M. W.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Breast Cancer</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">381-389</span>CODEN:
                <span class="NLM_cas:coden">CBCLB7</span>;
        ISSN:<span class="NLM_cas:issn">1526-8209</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review.  Sequential use of endocrine therapies remains the cornerstone of treatment for hormone receptor-pos. advanced breast cancer, before the use of cytotoxic chemotherapy for unresponsive disease.  Fulvestrant is an estrogen receptor (ER) antagonist approved for the treatment of postmenopausal women with ER+ advanced breast cancer after failure of prior antiestrogen therapy.  Initially approved at a monthly dose of 250 mg, the recommended fulvestrant dose was revised to 500 mg (500 mg/mo plus 500 mg on day 14 of month 1) after demonstration of improved progression-free survival vs. fulvestrant 250 mg.  We have reviewed the dose-dependent effects of fulvestrant, both from a retrospective combined anal. of dose-dependent redn. of tumor biomarkers in the presurgical setting (3 previously reported studies: Study 18, Neoadjuvant Endocrine Therapy for Women with Estrogen-Sensitive Tumors, and Trial 57) and from a review of clin. studies for advanced breast cancer in postmenopausal women.  Anal. of presurgical data revealed a consistent dose-dependent effect for fulvestrant on tumor biomarkers, with increasing fulvestrant dose resulting in greater redns. in ER, progesterone receptor, and Ki67 labeling index.  The dose-dependent biol. effect corresponds with the dose-dependent clin. efficacy obsd. in the treatment of advanced breast cancer after failure of prior antiestrogen therapy.  Although it remains to be detd. in a phase III trial, cross-trial comparisons suggest a dose-dependent relationship for fulvestrant as first-line treatment for advanced breast cancer.  Overall, biol. and clin. data demonstrate a strong dose-dependent relationship for fulvestrant, supporting the efficacy benefit seen with fulvestrant 500 mg over the 250 mg dose.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbnkP0JX-1rbVg90H21EOLACvtfcHk0lgRArth21UzCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFKmtrvI&md5=88d83ed816d271f73ad84521982b8411</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.clbc.2014.06.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.clbc.2014.06.005%26sid%3Dliteratum%253Aachs%26aulast%3DRobertson%26aufirst%3DJ.%2BF.%26aulast%3DLindemann%26aufirst%3DJ.%26aulast%3DGarnett%26aufirst%3DS.%26aulast%3DAnderson%26aufirst%3DE.%26aulast%3DNicholson%26aufirst%3DR.%2BI.%26aulast%3DKuter%26aufirst%3DI.%26aulast%3DGee%26aufirst%3DJ.%2BM.%26atitle%3DA%2520good%2520drug%2520made%2520better%253A%2520the%2520fulvestrant%2520dose-response%2520story%26jtitle%3DClin.%2520Breast%2520Cancer%26date%3D2014%26volume%3D14%26spage%3D381%26epage%3D389%26doi%3D10.1016%2Fj.clbc.2014.06.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McDonnell, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wardell, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norris, J. D.</span></span> <span> </span><span class="NLM_article-title">Oral selective estrogen receptor downregulators (SERDs), a breakthrough endocrine therapy for breast cancer</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">4883</span>– <span class="NLM_lpage">4887</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00760</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00760" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A528%3ADC%252BC2MXpsFaktL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=4883-4887&author=D.+P.+McDonnellauthor=S.+E.+Wardellauthor=J.+D.+Norris&title=Oral+selective+estrogen+receptor+downregulators+%28SERDs%29%2C+a+breakthrough+endocrine+therapy+for+breast+cancer&doi=10.1021%2Facs.jmedchem.5b00760"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Oral Selective Estrogen Receptor Downregulators (SERDs), a Breakthrough Endocrine Therapy for Breast Cancer</span></div><div class="casAuthors">McDonnell, Donald P.; Wardell, Suzanne E.; Norris, John D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">4883-4887</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Drugs that inhibit estrogen receptor alpha (ERα) or that block the prodn. of estrogens remain frontline interventions in the treatment and management of breast cancer at all stages.  However, resistance to endocrine therapies, esp. in the setting of advanced disease, remains an impediment to durable clin. responses.  Although the mechanisms underlying resistance to existing agents are complex, preclin. studies suggest that selective estrogen receptor downregulators (SERDs), mols. which eliminate ERα expression, may have particular utility in the treatment of breast cancers that have progressed on tamoxifen and/or aromatase inhibitors.  The discovery and development of orally bioavailable SERDs provide the opportunity to evaluate the utility of eliminating ERα expression in advanced metastatic breast cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpQkRKbmjeJdLVg90H21EOLACvtfcHk0lgRArth21UzCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXpsFaktL8%253D&md5=127869f40034451832a82fb2d7816528</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00760&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00760%26sid%3Dliteratum%253Aachs%26aulast%3DMcDonnell%26aufirst%3DD.%2BP.%26aulast%3DWardell%26aufirst%3DS.%2BE.%26aulast%3DNorris%26aufirst%3DJ.%2BD.%26atitle%3DOral%2520selective%2520estrogen%2520receptor%2520downregulators%2520%2528SERDs%2529%252C%2520a%2520breakthrough%2520endocrine%2520therapy%2520for%2520breast%2520cancer%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D4883%26epage%3D4887%26doi%3D10.1021%2Facs.jmedchem.5b00760" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abdel-Magid, A. F.</span></span> <span> </span><span class="NLM_article-title">Selective estrogen receptor degraders (SERDs): a promising treatment to overcome resistance to endocrine therapy in ERα-positive breast cancer</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1129</span>– <span class="NLM_lpage">1131</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.7b00424</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.7b00424" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=1129-1131&author=A.+F.+Abdel-Magid&title=Selective+estrogen+receptor+degraders+%28SERDs%29%3A+a+promising+treatment+to+overcome+resistance+to+endocrine+therapy+in+ER%CE%B1-positive+breast+cancer&doi=10.1021%2Facsmedchemlett.7b00424"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.7b00424&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.7b00424%26sid%3Dliteratum%253Aachs%26aulast%3DAbdel-Magid%26aufirst%3DA.%2BF.%26atitle%3DSelective%2520estrogen%2520receptor%2520degraders%2520%2528SERDs%2529%253A%2520a%2520promising%2520treatment%2520to%2520overcome%2520resistance%2520to%2520endocrine%2520therapy%2520in%2520ER%25CE%25B1-positive%2520breast%2520cancer%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2017%26volume%3D8%26spage%3D1129%26epage%3D1131%26doi%3D10.1021%2Facsmedchemlett.7b00424" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Weir, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradbury, R. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rabow, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buttar, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Callis, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curwen, J. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Almeida, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ballard, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hulse, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donald, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feron, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karoutchi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacFaul, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moss, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norman, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pearson, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tonge, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rowlinson, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sadler, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richmond, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ladd, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pazolli, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mazzola, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">D’Cruz, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Savi, C.</span></span> <span> </span><span class="NLM_article-title">AZD9496: an oral estrogen receptor inhibitor that blocks the growth of ER-positive and ESR1-mutant breast tumors in preclinical models</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>76</i></span>,  <span class="NLM_fpage">3307</span>– <span class="NLM_lpage">3318</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-15-2357</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=10.1158%2F0008-5472.CAN-15-2357" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=27020862" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A528%3ADC%252BC28XovVyktLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=76&publication_year=2016&pages=3307-3318&author=H.+M.+Weirauthor=R.+H.+Bradburyauthor=M.+Lawsonauthor=A.+A.+Rabowauthor=D.+Buttarauthor=R.+J.+Callisauthor=J.+O.+Curwenauthor=C.+de+Almeidaauthor=P.+Ballardauthor=M.+Hulseauthor=C.+S.+Donaldauthor=L.+J.+Feronauthor=G.+Karoutchiauthor=P.+MacFaulauthor=T.+Mossauthor=R.+A.+Normanauthor=S.+E.+Pearsonauthor=M.+Tongeauthor=G.+Daviesauthor=G.+E.+Walkerauthor=Z.+Wilsonauthor=R.+Rowlinsonauthor=S.+Powellauthor=C.+Sadlerauthor=G.+Richmondauthor=B.+Laddauthor=E.+Pazolliauthor=A.+M.+Mazzolaauthor=C.+D%E2%80%99Cruzauthor=C.+De+Savi&title=AZD9496%3A+an+oral+estrogen+receptor+inhibitor+that+blocks+the+growth+of+ER-positive+and+ESR1-mutant+breast+tumors+in+preclinical+models&doi=10.1158%2F0008-5472.CAN-15-2357"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">AZD9496: An Oral Estrogen Receptor Inhibitor That Blocks the Growth of ER-Positive and ESR1-Mutant Breast Tumors in Preclinical Models</span></div><div class="casAuthors">Weir, Hazel M.; Bradbury, Robert H.; Lawson, Mandy; Rabow, Alfred A.; Buttar, David; Callis, Rowena J.; Curwen, Jon O.; de Almeida, Camila; Ballard, Peter; Hulse, Michael; Donald, Craig S.; Feron, Lyman J. L.; Karoutchi, Galith; MacFaul, Philip; Moss, Thomas; Norman, Richard A.; Pearson, Stuart E.; Tonge, Michael; Davies, Gareth; Walker, Graeme E.; Wilson, Zena; Rowlinson, Rachel; Powell, Steve; Sadler, Claire; Richmond, Graham; Ladd, Brendon; Pazolli, Ermira; Mazzola, Anne Marie; D'Cruz, Celina; De Savi, Chris</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">76</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">3307-3318</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Fulvestrant is an estrogen receptor (ER) antagonist administered to breast cancer patients by monthly i.m. injection.  Given its present limitations of dosing and route of administration, a more flexible orally available compd. has been sought to pursue the potential benefits of this drug in patients with advanced metastatic disease.  Here we report the identification and characterization of AZD9496, a nonsteroidal small-mol. inhibitor of ERα, which is a potent and selective antagonist and downregulator of ERα in vitro and in vivo in ER-pos. models of breast cancer.  Significant tumor growth inhibition was obsd. as low as 0.5 mg/kg dose in the estrogen-dependent MCF-7 xenograft model, where this effect was accompanied by a dose-dependent decrease in PR protein levels, demonstrating potent antagonist activity.  Combining AZD9496 with PI3K pathway and CDK4/6 inhibitors led to further growth-inhibitory effects compared with monotherapy alone.  Tumor regressions were also seen in a long-term estrogen-deprived breast model, where significant downregulation of ERα protein was obsd.  AZD9496 bound and downregulated clin. relevant ESR1 mutants in vitro and inhibited tumor growth in an ESR1-mutant patient-derived xenograft model that included a D538G mutation.  Collectively, the pharmacol. evidence showed that AZD9496 is an oral, nonsteroidal, selective estrogen receptor antagonist and downregulator in ER+ breast cells that could provide meaningful benefit to ER+ breast cancer patients.  AZD9496 is currently being evaluated in a phase I clin. trial.  Cancer Res; 76(11); 3307-18. ©2016 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoP4X3e2iYdcLVg90H21EOLACvtfcHk0lgHFR2eCU2H5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XovVyktLo%253D&md5=4ff8ab9a9cd379acc031b7c6fd27134c</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-15-2357&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-15-2357%26sid%3Dliteratum%253Aachs%26aulast%3DWeir%26aufirst%3DH.%2BM.%26aulast%3DBradbury%26aufirst%3DR.%2BH.%26aulast%3DLawson%26aufirst%3DM.%26aulast%3DRabow%26aufirst%3DA.%2BA.%26aulast%3DButtar%26aufirst%3DD.%26aulast%3DCallis%26aufirst%3DR.%2BJ.%26aulast%3DCurwen%26aufirst%3DJ.%2BO.%26aulast%3Dde%2BAlmeida%26aufirst%3DC.%26aulast%3DBallard%26aufirst%3DP.%26aulast%3DHulse%26aufirst%3DM.%26aulast%3DDonald%26aufirst%3DC.%2BS.%26aulast%3DFeron%26aufirst%3DL.%2BJ.%26aulast%3DKaroutchi%26aufirst%3DG.%26aulast%3DMacFaul%26aufirst%3DP.%26aulast%3DMoss%26aufirst%3DT.%26aulast%3DNorman%26aufirst%3DR.%2BA.%26aulast%3DPearson%26aufirst%3DS.%2BE.%26aulast%3DTonge%26aufirst%3DM.%26aulast%3DDavies%26aufirst%3DG.%26aulast%3DWalker%26aufirst%3DG.%2BE.%26aulast%3DWilson%26aufirst%3DZ.%26aulast%3DRowlinson%26aufirst%3DR.%26aulast%3DPowell%26aufirst%3DS.%26aulast%3DSadler%26aufirst%3DC.%26aulast%3DRichmond%26aufirst%3DG.%26aulast%3DLadd%26aufirst%3DB.%26aulast%3DPazolli%26aufirst%3DE.%26aulast%3DMazzola%26aufirst%3DA.%2BM.%26aulast%3DD%25E2%2580%2599Cruz%26aufirst%3DC.%26aulast%3DDe%2BSavi%26aufirst%3DC.%26atitle%3DAZD9496%253A%2520an%2520oral%2520estrogen%2520receptor%2520inhibitor%2520that%2520blocks%2520the%2520growth%2520of%2520ER-positive%2520and%2520ESR1-mutant%2520breast%2520tumors%2520in%2520preclinical%2520models%26jtitle%3DCancer%2520Res.%26date%3D2016%26volume%3D76%26spage%3D3307%26epage%3D3318%26doi%3D10.1158%2F0008-5472.CAN-15-2357" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Joseph, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darimont, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrazate, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ingalla, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walter, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blake, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nonomiya, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guan, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kategaya, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Govek, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kahraman, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brigham, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sensintaffar, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grillot, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prudente, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bischoff, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonnefous, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Douglas, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Julien, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagasawa, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aparicio, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaufman, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haley, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giltnane, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wertz, I. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lackner, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nannini, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sampath, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwarz, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manning, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tantawy, M. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arteaga, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heyman, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rix, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friedman, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, N. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Metcalfe, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hager, J. H.</span></span> <span> </span><span class="NLM_article-title">The selective estrogen receptor downregulator GDC-0810 is efficacious in diverse models of ER+ breast cancer</span>. <i>eLife</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>5</i></span>, <span class="NLM_elocation-id">e15828</span> <span class="refDoi"> DOI: 10.7554/eLife.15828</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=10.7554%2FeLife.15828" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=27410477" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A528%3ADC%252BC1cXmslahtb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2016&author=J.+D.+Josephauthor=B.+Darimontauthor=W.+Zhouauthor=A.+Arrazateauthor=A.+Youngauthor=E.+Ingallaauthor=K.+Walterauthor=R.+A.+Blakeauthor=J.+Nonomiyaauthor=Z.+Guanauthor=L.+Kategayaauthor=S.+P.+Govekauthor=A.+G.+Laiauthor=M.+Kahramanauthor=D.+Brighamauthor=J.+Sensintaffarauthor=N.+Luauthor=G.+Shaoauthor=J.+Qianauthor=K.+Grillotauthor=M.+Moonauthor=R.+Prudenteauthor=E.+Bischoffauthor=K.+J.+Leeauthor=C.+Bonnefousauthor=K.+L.+Douglasauthor=J.+D.+Julienauthor=J.+Y.+Nagasawaauthor=A.+Aparicioauthor=J.+Kaufmanauthor=B.+Haleyauthor=J.+M.+Giltnaneauthor=I.+E.+Wertzauthor=M.+R.+Lacknerauthor=M.+A.+Nanniniauthor=D.+Sampathauthor=L.+Schwarzauthor=H.+C.+Manningauthor=M.+N.+Tantawyauthor=C.+L.+Arteagaauthor=R.+A.+Heymanauthor=P.+J.+Rixauthor=L.+Friedmanauthor=N.+D.+Smithauthor=C.+Metcalfeauthor=J.+H.+Hager&title=The+selective+estrogen+receptor+downregulator+GDC-0810+is+efficacious+in+diverse+models+of+ER%2B+breast+cancer&doi=10.7554%2FeLife.15828"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">The selective estrogen receptor downregulator GDC-0810 is efficacious in diverse models of ER+ breast cancer</span></div><div class="casAuthors">Joseph, James D.; Darimont, Beatrice; Zhou, Wei; Arrazate, Alfonso; Young, Amy; Ingalla, Ellen; Walter, Kimberly; Blake, Robert A.; Nonomiya, Jim; Guan, Zhengyu; Kategaya, Lorna; Govek, Steven P.; Lai, Andiliy G.; Kahraman, Mehmet; Brigham, Dan; Sensintaffar, John; Lu, Nhin; Shao, Gang; Qian, Jing; Grillot, Kate; Moon, Michael; Prudente, Rene; Bischoff, Eric; Lee, Kyoung-Jin; Bonnefous, Celine; Douglas, Karensa L.; Julien, Jackaline D.; Nagasawa, Johnny Y.; Aparicio, Anna; Kaufman, Josh; Haley, Benjamin; Giltnane, Jennifer M.; Wertz, Ingrid E.; Lackner, Mark R.; Nannini, Michelle A.; Sampath, Deepak; Schwarz, Luis; Manning, Henry Charles; Tantawy, Mohammed Noor; Arteaga, Carlos L.; Heyman, Richard A.; Rix, Peter J.; Friedman, Lori; Smith, Nicholas D.; Metcalfe, Ciara; Hager, Jeffrey H.</div><div class="citationInfo"><span class="NLM_cas:title">eLife</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">e15828/1-e15828/34</span>CODEN:
                <span class="NLM_cas:coden">ELIFA8</span>;
        ISSN:<span class="NLM_cas:issn">2050-084X</span>.
    
            (<span class="NLM_cas:orgname">eLife Sciences Publications Ltd.</span>)
        </div><div class="casAbstract">ER-targeted therapeutics provide valuable treatment options for patients with ER+ breast cancer, however, current relapse and mortality rates emphasize the need for improved therapeutic strategies.  The recent discovery of prevalent ESR1 mutations in relapsed tumors underscores a sustained reliance of advanced tumors on ERa signaling, and provides a strong rationale for continued targeting of ERa.  Here we describe GDC-0810, a novel, non-steroidal, orally bioavailable selective ER downregulator (SERD), which was identified by prospectively optimizing ERa degrdn., antagonism and pharmacokinetic properties.  GDC-0810 induces a distinct ERa conformation, relative to that induced by currently approved therapeutics, suggesting a unique mechanism of action.  GDC-0810 has robust in vitro and in vivo activity against a variety of human breast cancer cell lines and patient derived xenografts, including a tamoxifen-resistant model and those that harbor ERa mutations.  GDC-0810 is currently being evaluated in Phase II clin. studies in women with ER+ breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoML4fZixufB7Vg90H21EOLACvtfcHk0lj7kEEsT-Hauw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXmslahtb0%253D&md5=cfdf9624f8b260a732b92caab0bb2d58</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.7554%2FeLife.15828&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7554%252FeLife.15828%26sid%3Dliteratum%253Aachs%26aulast%3DJoseph%26aufirst%3DJ.%2BD.%26aulast%3DDarimont%26aufirst%3DB.%26aulast%3DZhou%26aufirst%3DW.%26aulast%3DArrazate%26aufirst%3DA.%26aulast%3DYoung%26aufirst%3DA.%26aulast%3DIngalla%26aufirst%3DE.%26aulast%3DWalter%26aufirst%3DK.%26aulast%3DBlake%26aufirst%3DR.%2BA.%26aulast%3DNonomiya%26aufirst%3DJ.%26aulast%3DGuan%26aufirst%3DZ.%26aulast%3DKategaya%26aufirst%3DL.%26aulast%3DGovek%26aufirst%3DS.%2BP.%26aulast%3DLai%26aufirst%3DA.%2BG.%26aulast%3DKahraman%26aufirst%3DM.%26aulast%3DBrigham%26aufirst%3DD.%26aulast%3DSensintaffar%26aufirst%3DJ.%26aulast%3DLu%26aufirst%3DN.%26aulast%3DShao%26aufirst%3DG.%26aulast%3DQian%26aufirst%3DJ.%26aulast%3DGrillot%26aufirst%3DK.%26aulast%3DMoon%26aufirst%3DM.%26aulast%3DPrudente%26aufirst%3DR.%26aulast%3DBischoff%26aufirst%3DE.%26aulast%3DLee%26aufirst%3DK.%2BJ.%26aulast%3DBonnefous%26aufirst%3DC.%26aulast%3DDouglas%26aufirst%3DK.%2BL.%26aulast%3DJulien%26aufirst%3DJ.%2BD.%26aulast%3DNagasawa%26aufirst%3DJ.%2BY.%26aulast%3DAparicio%26aufirst%3DA.%26aulast%3DKaufman%26aufirst%3DJ.%26aulast%3DHaley%26aufirst%3DB.%26aulast%3DGiltnane%26aufirst%3DJ.%2BM.%26aulast%3DWertz%26aufirst%3DI.%2BE.%26aulast%3DLackner%26aufirst%3DM.%2BR.%26aulast%3DNannini%26aufirst%3DM.%2BA.%26aulast%3DSampath%26aufirst%3DD.%26aulast%3DSchwarz%26aufirst%3DL.%26aulast%3DManning%26aufirst%3DH.%2BC.%26aulast%3DTantawy%26aufirst%3DM.%2BN.%26aulast%3DArteaga%26aufirst%3DC.%2BL.%26aulast%3DHeyman%26aufirst%3DR.%2BA.%26aulast%3DRix%26aufirst%3DP.%2BJ.%26aulast%3DFriedman%26aufirst%3DL.%26aulast%3DSmith%26aufirst%3DN.%2BD.%26aulast%3DMetcalfe%26aufirst%3DC.%26aulast%3DHager%26aufirst%3DJ.%2BH.%26atitle%3DThe%2520selective%2520estrogen%2520receptor%2520downregulator%2520GDC-0810%2520is%2520efficacious%2520in%2520diverse%2520models%2520of%2520ER%252B%2520breast%2520cancer%26jtitle%3DeLife%26date%3D2016%26volume%3D5%26doi%3D10.7554%2FeLife.15828" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bihani, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, H. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arlt, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tao, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Purandare, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hattersley, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garner, F.</span></span> <span> </span><span class="NLM_article-title">Elacestrant (RAD1901), a selective estrogen receptor degrader (SERD), has antitumor activity in multiple ER+ breast cancer patient-derived xenograft models</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">4793</span>– <span class="NLM_lpage">4804</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.CCR-16-2561</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=10.1158%2F1078-0432.CCR-16-2561" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=28473534" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtlCmu73O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2017&pages=4793-4804&author=T.+Bihaniauthor=H.+K.+Patelauthor=H.+Arltauthor=N.+Taoauthor=H.+Jiangauthor=J.+L.+Brownauthor=D.+M.+Purandareauthor=G.+Hattersleyauthor=F.+Garner&title=Elacestrant+%28RAD1901%29%2C+a+selective+estrogen+receptor+degrader+%28SERD%29%2C+has+antitumor+activity+in+multiple+ER%2B+breast+cancer+patient-derived+xenograft+models&doi=10.1158%2F1078-0432.CCR-16-2561"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Elacestrant (RAD1901), a Selective Estrogen Receptor Degrader (SERD), Has Antitumor Activity in Multiple ER+ Breast Cancer Patient-derived Xenograft Models</span></div><div class="casAuthors">Bihani, Teeru; Patel, Hitisha K.; Arlt, Heike; Tao, Nianjun; Jiang, Hai; Brown, Jeffrey L.; Purandare, Dinesh M.; Hattersley, Gary; Garner, Fiona</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">4793-4804</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Purpose: Estrogen receptor-pos. (ER+) breast cancers are typically treated with endocrine agents, and dependence on the ER pathway is often retained even after multiple rounds of antiestrogen therapy.  Selective estrogen receptor degraders (SERD) are being developed as a strategy to more effectively target ER and exploit ER dependence in these cancers, which includes inhibiting both wild-type and mutant forms of ER.  The purpose of this study was to evaluate the efficacy of a novel orally bioavailable SERD, elacestrant (RAD1901), in preclin. models of ER+ breast cancer.  Exptl. Design: Elacestrant was evaluated as a single agent and in combination with palbociclib or everolimus in multiple ER+ breast cancer models, including several patient-derived xenograft models.  Results: Elacestrant induces the degrdn. of ER, inhibits ER-mediated signaling and growth of ER+ breast cancer cell lines in vitro and in vivo, and significantly inhibits tumor growth of multiple PDX models.  Furthermore, we demonstrate that elacestrant in combination with palbociclib or everolimus can lead to greater efficacy in certain contexts.  Finally, elacestrant exhibits significant antitumor activity both as a single agent and in combination with palbociclib in two patient-derived breast cancer xenograft models harboring ESR1 mutations.  Conclusions: These data underscore the potential clin. utility of elacestrant as a single agent and as a combination therapy, for both early- and late-stage ER+ disease.  Clin Cancer Res; 23(16); 4793-804. ©2017 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpASoatzBVHxbVg90H21EOLACvtfcHk0lj7kEEsT-Hauw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtlCmu73O&md5=4a6811274e95e6401bbfe0e56ccb1afc</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-16-2561&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-16-2561%26sid%3Dliteratum%253Aachs%26aulast%3DBihani%26aufirst%3DT.%26aulast%3DPatel%26aufirst%3DH.%2BK.%26aulast%3DArlt%26aufirst%3DH.%26aulast%3DTao%26aufirst%3DN.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DBrown%26aufirst%3DJ.%2BL.%26aulast%3DPurandare%26aufirst%3DD.%2BM.%26aulast%3DHattersley%26aufirst%3DG.%26aulast%3DGarner%26aufirst%3DF.%26atitle%3DElacestrant%2520%2528RAD1901%2529%252C%2520a%2520selective%2520estrogen%2520receptor%2520degrader%2520%2528SERD%2529%252C%2520has%2520antitumor%2520activity%2520in%2520multiple%2520ER%252B%2520breast%2520cancer%2520patient-derived%2520xenograft%2520models%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2017%26volume%3D23%26spage%3D4793%26epage%3D4804%26doi%3D10.1158%2F1078-0432.CCR-16-2561" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tria, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abrams, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baird, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burks, H. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Firestone, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaither, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamann, L. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirby, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lombardo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macchi, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDonnell, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mishina, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norris, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nunez, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Springer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomsen, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tiong-Yip, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peukert, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of LSZ102, a potent, orally bioavailable selective estrogen receptor degrader (SERD) for the treatment of estrogen receptor positive breast cancer</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">2837</span>– <span class="NLM_lpage">2864</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01682</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01682" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A528%3ADC%252BC1cXltFynurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=2837-2864&author=G.+S.+Triaauthor=T.+Abramsauthor=J.+Bairdauthor=H.+E.+Burksauthor=B.+Firestoneauthor=L.+A.+Gaitherauthor=L.+G.+Hamannauthor=G.+Heauthor=C.+A.+Kirbyauthor=S.+Kimauthor=F.+Lombardoauthor=K.+J.+Macchiauthor=D.+P.+McDonnellauthor=Y.+Mishinaauthor=J.+D.+Norrisauthor=J.+Nunezauthor=C.+Springerauthor=Y.+Sunauthor=N.+M.+Thomsenauthor=C.+Wangauthor=J.+Wangauthor=B.+Yuauthor=C.+L.+Tiong-Yipauthor=S.+Peukert&title=Discovery+of+LSZ102%2C+a+potent%2C+orally+bioavailable+selective+estrogen+receptor+degrader+%28SERD%29+for+the+treatment+of+estrogen+receptor+positive+breast+cancer&doi=10.1021%2Facs.jmedchem.7b01682"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of LSZ102, a Potent, Orally Bioavailable Selective Estrogen Receptor Degrader (SERD) for the Treatment of Estrogen Receptor Positive Breast Cancer</span></div><div class="casAuthors">Tria, George S.; Abrams, Tinya; Baird, Jason; Burks, Heather E.; Firestone, Brant; Gaither, L. Alex; Hamann, Lawrence G.; He, Guo; Kirby, Christina A.; Kim, Sunkyu; Lombardo, Franco; Macchi, Kaitlin J.; McDonnell, Donald P.; Mishina, Yuji; Norris, John D.; Nunez, Jill; Springer, Clayton; Sun, Yingchuan; Thomsen, Noel M.; Wang, Chunrong; Wang, Jianling; Yu, Bing; Tiong-Yip, Choi-Lai; Peukert, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2837-2864</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In breast cancer, estrogen receptor alpha (ERα) pos. cancer accounts for approx. 74% of all diagnoses, and in these settings, it is a primary driver of cell proliferation.  Treatment of ERα pos. breast cancer has long relied on endocrine therapies such as selective estrogen receptor modulators, aromatase inhibitors, and selective estrogen receptor degraders (SERDs).  The steroid-based anti-estrogen fulvestrant, the only approved SERD, is effective in patients who have not previously been treated with endocrine therapy as well as in patients who have progressed after receiving other endocrine therapies.  Its efficacy, however, may be limited due to its poor physicochem. properties.  THe authors describe the design and synthesis of a series of potent benzothiophene-contg. compds. that exhibit oral bioavailability and preclin. activity as SERDs.  This article culminates in the identification of LSZ102 (I), a compd. in clin. development for the treatment of ERα pos. breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcHgOrWwlRvrVg90H21EOLACvtfcHk0lj7kEEsT-Hauw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXltFynurc%253D&md5=dba954f9f11121fae1bb50979428a83b</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01682&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01682%26sid%3Dliteratum%253Aachs%26aulast%3DTria%26aufirst%3DG.%2BS.%26aulast%3DAbrams%26aufirst%3DT.%26aulast%3DBaird%26aufirst%3DJ.%26aulast%3DBurks%26aufirst%3DH.%2BE.%26aulast%3DFirestone%26aufirst%3DB.%26aulast%3DGaither%26aufirst%3DL.%2BA.%26aulast%3DHamann%26aufirst%3DL.%2BG.%26aulast%3DHe%26aufirst%3DG.%26aulast%3DKirby%26aufirst%3DC.%2BA.%26aulast%3DKim%26aufirst%3DS.%26aulast%3DLombardo%26aufirst%3DF.%26aulast%3DMacchi%26aufirst%3DK.%2BJ.%26aulast%3DMcDonnell%26aufirst%3DD.%2BP.%26aulast%3DMishina%26aufirst%3DY.%26aulast%3DNorris%26aufirst%3DJ.%2BD.%26aulast%3DNunez%26aufirst%3DJ.%26aulast%3DSpringer%26aufirst%3DC.%26aulast%3DSun%26aufirst%3DY.%26aulast%3DThomsen%26aufirst%3DN.%2BM.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DYu%26aufirst%3DB.%26aulast%3DTiong-Yip%26aufirst%3DC.%2BL.%26aulast%3DPeukert%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520LSZ102%252C%2520a%2520potent%252C%2520orally%2520bioavailable%2520selective%2520estrogen%2520receptor%2520degrader%2520%2528SERD%2529%2520for%2520the%2520treatment%2520of%2520estrogen%2520receptor%2520positive%2520breast%2520cancer%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D2837%26epage%3D2864%26doi%3D10.1021%2Facs.jmedchem.7b01682" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Carlson, R. W.</span></span> <span> </span><span class="NLM_article-title">The history and mechanism of action of fulvestrant</span>. <i>Clin. Breast Cancer</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">S5</span>– <span class="NLM_lpage">S8</span>, <span class="refDoi"> DOI: 10.3816/CBC.2005.s.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=10.3816%2FCBC.2005.s.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=15865849" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A528%3ADC%252BD2MXltV2ksbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2005&pages=S5-S8&author=R.+W.+Carlson&title=The+history+and+mechanism+of+action+of+fulvestrant&doi=10.3816%2FCBC.2005.s.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">The history and mechanism of action of fulvestrant</span></div><div class="casAuthors">Carlson, Robert W.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Breast Cancer</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">Suppl. 1</span>),
    <span class="NLM_cas:pages">S5-S8</span>CODEN:
                <span class="NLM_cas:coden">CBCLB7</span>;
        ISSN:<span class="NLM_cas:issn">1526-8209</span>.
    
            (<span class="NLM_cas:orgname">Cancer Information Group, LP</span>)
        </div><div class="casAbstract">A review.  Fulvestrant is a pure antiestrogen that emerged from a systematic medicinal chem. strategy of modification of long-chain alkyl substitutes in the 7α-position of estradiol.  Fulvestrant has no uterotrophic effects on the immature or ovariectomized rat and blocks the agonistic effects of estradiol and tamoxifen in a dose-dependent manner.  In in vivo and in vitro breast cancer models, fulvestrant has anticancer activity at least as good as tamoxifen and is superior to tamoxifen in some models.  Fulvestrant requires i.m. administration in a proprietary formulation of castor oil and alcs.  When fulvestrant binds to estrogen receptor monomers it inhibits receptor dimerization, activating function 1 (AF1) and AF2 are rendered inactive, translocation of receptor to the nucleus is reduced, and degrdn. of the estrogen receptor is accelerated.  This results in pure antiestrogenic effects.  There is substantial preclin. evidence that the nonsteroidal hormone-dependent mechanisms of estrogen receptor activation and regulation via growth factor receptors and their signal transduction pathways are important in the development of breast cancer hormonal resistance.  Methods of exploiting the interactions between these nonsteroidal hormone-dependent mechanisms of resistance and hormonal agents such as fulvestrant are an active area for drug development and clin. investigation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr6cUux0p2XbbVg90H21EOLACvtfcHk0liuXSNdDkdNhQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXltV2ksbs%253D&md5=76569c36605521476f8458d614670b04</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.3816%2FCBC.2005.s.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3816%252FCBC.2005.s.008%26sid%3Dliteratum%253Aachs%26aulast%3DCarlson%26aufirst%3DR.%2BW.%26atitle%3DThe%2520history%2520and%2520mechanism%2520of%2520action%2520of%2520fulvestrant%26jtitle%3DClin.%2520Breast%2520Cancer%26date%3D2005%26volume%3D6%26spage%3DS5%26epage%3DS8%26doi%3D10.3816%2FCBC.2005.s.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Marsaud, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gougelet, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maillard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renoir, J. M.</span></span> <span> </span><span class="NLM_article-title">Various phosphorylation pathways, depending on agonist and antagonist binding to endogenous estrogen receptor α (ERα), differentially affect ERα extractability, proteasome-mediated stability, and transcriptional activity in human breast cancer cells</span>. <i>Mol. Endocrinol.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">2013</span>– <span class="NLM_lpage">2027</span>, <span class="refDoi"> DOI: 10.1210/me.2002-0269</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=10.1210%2Fme.2002-0269" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=12855746" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A528%3ADC%252BD3sXotFSrsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2003&pages=2013-2027&author=V.+Marsaudauthor=A.+Gougeletauthor=S.+Maillardauthor=J.+M.+Renoir&title=Various+phosphorylation+pathways%2C+depending+on+agonist+and+antagonist+binding+to+endogenous+estrogen+receptor+%CE%B1+%28ER%CE%B1%29%2C+differentially+affect+ER%CE%B1+extractability%2C+proteasome-mediated+stability%2C+and+transcriptional+activity+in+human+breast+cancer+cells&doi=10.1210%2Fme.2002-0269"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Various phosphorylation pathways, depending on agonist and antagonist binding to endogenous estrogen receptor α (ERα), differentially affect ERα extractability, proteasome-mediated stability, and transcriptional activity in human breast cancer cells</span></div><div class="casAuthors">Marsaud, Veronique; Gougelet, Angelique; Maillard, Sebastien; Renoir, Jack-Michel</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Endocrinology</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2013-2027</span>CODEN:
                <span class="NLM_cas:coden">MOENEN</span>;
        ISSN:<span class="NLM_cas:issn">0888-8809</span>.
    
            (<span class="NLM_cas:orgname">Endocrine Society</span>)
        </div><div class="casAbstract">Estrogen receptor-α (ER) is down-regulated in the presence of its cognate ligand, estradiol (E2), as well as in the presence of antiestrogens, through the ubiquitin proteasome pathway.  Here, we show that, at pharmacol. concns., the degrdn. rate of pure antagonist/endogenous ER complexes from human breast cancer MCF-7 cells is 10 times faster than that of ER-E2 complexes, while 4-hydroxy-tamoxifen (4-OH-T)-ER complexes are stable.  Whereas pure antagonist-ER complexes are firmly bound to a nuclear compartment from which they are not extractable, the 4-OH-T-ER accumulates in a sol. cell compartment.  No difference was obsd. in the fate of ER whether bound to pure antiestrogens ICI 182,780 or RU 58668.  Cycloheximide expts. showed that, while the proteasome-mediated destruction of E2-ER (unlike that of RU 58668- and ICI 182,780-ER) complexes could implicate (or not) a protein synthesis-dependent process, both MAPKs (p38 and ERKs p44 and p42) are activated.  By using a panel of kinase inhibitors/activators to study the impact of phosphorylation pathways on ER degrdn., we found that protein kinase C is an enhancer of proteasome-mediated degrdn. of both ligand-free and ER bound to either E2, 4-OH-T, and pure antagonists.  On the contrary, protein kinase A, MAPKs, and phosphatidyl-inositol-3 kinase all impede proteasome-mediated destruction of ligand free and E2-bound ER while only MAPKs inhibit the degrdn. of pure antiestrogens/ER species.  In addn., no correlation was found between the capacity of kinase inhibitors to affect ER stability and the basal or E2-induced transcription.  These results suggest that, in MCF-7 breast cancer cells, ER turnover, localization, and activity are maintained by an equil. between various phosphorylation pathways, which are differently modulated by ER ligands and protein kinases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGofjCUE8Z2B7bVg90H21EOLACvtfcHk0liuXSNdDkdNhQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXotFSrsLc%253D&md5=d7f2244b364abdc94a0730f3140f29a8</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1210%2Fme.2002-0269&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1210%252Fme.2002-0269%26sid%3Dliteratum%253Aachs%26aulast%3DMarsaud%26aufirst%3DV.%26aulast%3DGougelet%26aufirst%3DA.%26aulast%3DMaillard%26aufirst%3DS.%26aulast%3DRenoir%26aufirst%3DJ.%2BM.%26atitle%3DVarious%2520phosphorylation%2520pathways%252C%2520depending%2520on%2520agonist%2520and%2520antagonist%2520binding%2520to%2520endogenous%2520estrogen%2520receptor%2520%25CE%25B1%2520%2528ER%25CE%25B1%2529%252C%2520differentially%2520affect%2520ER%25CE%25B1%2520extractability%252C%2520proteasome-mediated%2520stability%252C%2520and%2520transcriptional%2520activity%2520in%2520human%2520breast%2520cancer%2520cells%26jtitle%3DMol.%2520Endocrinol.%26date%3D2003%26volume%3D17%26spage%3D2013%26epage%3D2027%26doi%3D10.1210%2Fme.2002-0269" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wittmann, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sherk, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDonnell, D. P.</span></span> <span> </span><span class="NLM_article-title">Definition of functionally important mechanistic differences among selective estrogen receptor down-regulators</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">9549</span>– <span class="NLM_lpage">9560</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-07-1590</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=10.1158%2F0008-5472.CAN-07-1590" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2007&pages=9549-9560&author=B.+M.+Wittmannauthor=A.+Sherkauthor=D.+P.+McDonnell&title=Definition+of+functionally+important+mechanistic+differences+among+selective+estrogen+receptor+down-regulators&doi=10.1158%2F0008-5472.CAN-07-1590"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-07-1590&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-07-1590%26sid%3Dliteratum%253Aachs%26aulast%3DWittmann%26aufirst%3DB.%2BM.%26aulast%3DSherk%26aufirst%3DA.%26aulast%3DMcDonnell%26aufirst%3DD.%2BP.%26atitle%3DDefinition%2520of%2520functionally%2520important%2520mechanistic%2520differences%2520among%2520selective%2520estrogen%2520receptor%2520down-regulators%26jtitle%3DCancer%2520Res.%26date%3D2007%26volume%3D67%26spage%3D9549%26epage%3D9560%26doi%3D10.1158%2F0008-5472.CAN-07-1590" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sakamoto, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, K. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumagai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mercurio, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deshaies, R. J.</span></span> <span> </span><span class="NLM_article-title">Protacs: chimeric molecules that target proteins to the Skp1-Cullin-F box complex for ubiquitination and degradation</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>98</i></span>,  <span class="NLM_fpage">8554</span>– <span class="NLM_lpage">8559</span>, <span class="refDoi"> DOI: 10.1073/pnas.141230798</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=10.1073%2Fpnas.141230798" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=11438690" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A528%3ADC%252BD3MXls1Wisbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=2001&pages=8554-8559&author=K.+M.+Sakamotoauthor=K.+B.+Kimauthor=A.+Kumagaiauthor=F.+Mercurioauthor=C.+M.+Crewsauthor=R.+J.+Deshaies&title=Protacs%3A+chimeric+molecules+that+target+proteins+to+the+Skp1-Cullin-F+box+complex+for+ubiquitination+and+degradation&doi=10.1073%2Fpnas.141230798"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Protacs: chimeric molecules that target proteins to the Skp1-Cullin-F box complex for ubiquitination and degradation</span></div><div class="casAuthors">Sakamoto, Kathleen M.; Kim, Kyung B.; Kumagai, Akiko; Mercurio, Frank; Crews, Craig M.; Deshaies, Raymond J.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">8554-8559</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The intracellular levels of many proteins are regulated by ubiquitin-dependent proteolysis.  One of the best-characterized enzymes that catalyzes the attachment of ubiquitin to proteins is a ubiquitin ligase complex, Skp1-Cullin-F box complex contg. Hrt1 (SCF).  We sought to artificially target a protein to the SCF complex for ubiquitination and degrdn.  To this end, we tested methionine aminopeptidase-2 (MetAP-2), which covalently binds the angiogenesis inhibitor ovalicin.  A chimeric compd., protein-targeting chimeric mol. 1 (Protac-1), was synthesized to recruit MetAP-2 to SCF.  One domain of Protac-1 contains the IκBα phosphopeptide that is recognized by the F-box protein β-TRCP, whereas the other domain is composed of ovalicin.  We show that MetAP-2 can be tethered to SCFβ-TRCP, ubiquitinated, and degraded in a Protac-1-dependent manner.  In the future, this approach may be useful for conditional inactivation of proteins, and for targeting disease-causing proteins for destruction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp49fYtuEVlJrVg90H21EOLACvtfcHk0ljPL7-OC3aFzQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXls1Wisbk%253D&md5=540358035222f745f7b6367b38781a21</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1073%2Fpnas.141230798&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.141230798%26sid%3Dliteratum%253Aachs%26aulast%3DSakamoto%26aufirst%3DK.%2BM.%26aulast%3DKim%26aufirst%3DK.%2BB.%26aulast%3DKumagai%26aufirst%3DA.%26aulast%3DMercurio%26aufirst%3DF.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26aulast%3DDeshaies%26aufirst%3DR.%2BJ.%26atitle%3DProtacs%253A%2520chimeric%2520molecules%2520that%2520target%2520proteins%2520to%2520the%2520Skp1-Cullin-F%2520box%2520complex%2520for%2520ubiquitination%2520and%2520degradation%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2001%26volume%3D98%26spage%3D8554%26epage%3D8559%26doi%3D10.1073%2Fpnas.141230798" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Toure, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Small-molecule PROTACS: new approaches to protein degradation</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">1966</span>– <span class="NLM_lpage">1973</span>, <span class="refDoi"> DOI: 10.1002/anie.201507978</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=10.1002%2Fanie.201507978" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A528%3ADC%252BC28XnsF2ltA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2016&pages=1966-1973&author=M.+Toureauthor=C.+M.+Crews&title=Small-molecule+PROTACS%3A+new+approaches+to+protein+degradation&doi=10.1002%2Fanie.201507978"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Small-Molecule PROTACS: New Approaches to Protein Degradation</span></div><div class="casAuthors">Toure, Momar; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1966-1973</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  The current inhibitor-based approach to therapeutics has inherent limitations owing to its occupancy-based model: there is a need to maintain high systemic exposure to ensure sufficient in vivo inhibition, high in vivo concns. bring potential for off-target side effects, and there is a need to bind to an active site, thus limiting the drug target space.  As an alternative, induced protein degrdn. lacks these limitations.  Based on an event-driven model, this approach offers a novel catalytic mechanism to irreversibly inhibit protein function by targeting protein destruction through recruitment to the cellular quality control machinery.  Prior protein degrading strategies have lacked therapeutic potential.  However, recent reports of small-mol.-based proteolysis-targeting chimeras (PROTACs) have demonstrated that this technol. can effectively decrease the cellular levels of several protein classes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdaqlIjCSsgrVg90H21EOLACvtfcHk0ljPL7-OC3aFzQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XnsF2ltA%253D%253D&md5=47838cd15719757317cb8282c26cc59e</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1002%2Fanie.201507978&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201507978%26sid%3Dliteratum%253Aachs%26aulast%3DToure%26aufirst%3DM.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DSmall-molecule%2520PROTACS%253A%2520new%2520approaches%2520to%2520protein%2520degradation%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2016%26volume%3D55%26spage%3D1966%26epage%3D1973%26doi%3D10.1002%2Fanie.201507978" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lai, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Induced protein degradation: an emerging drug discovery paradigm</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">101</span>– <span class="NLM_lpage">114</span>, <span class="refDoi"> DOI: 10.1038/nrd.2016.211</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=10.1038%2Fnrd.2016.211" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=27885283" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFCit7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=101-114&author=A.+C.+Laiauthor=C.+M.+Crews&title=Induced+protein+degradation%3A+an+emerging+drug+discovery+paradigm&doi=10.1038%2Fnrd.2016.211"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Induced protein degradation: an emerging drug discovery paradigm</span></div><div class="casAuthors">Lai, Ashton C.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">101-114</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Small-mol. drug discovery has traditionally focused on occupancy of a binding site that directly affects protein function, and this approach typically precludes targeting proteins that lack such amenable sites.  Furthermore, high systemic drug exposures may be needed to maintain sufficient target inhibition in vivo, increasing the risk of undesirable off-target effects.  Induced protein degrdn. is an alternative approach that is event-driven: upon drug binding, the target protein is tagged for elimination.  Emerging technologies based on proteolysis-targeting chimaeras (PROTACs) that exploit cellular quality control machinery to selectively degrade target proteins are attracting considerable attention in the pharmaceutical industry owing to the advantages they could offer over traditional small-mol. strategies.  These advantages include the potential to reduce systemic drug exposure, the ability to counteract increased target protein expression that often accompanies inhibition of protein function and the potential ability to target proteins that are not currently therapeutically tractable, such as transcription factors, scaffolding and regulatory proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoj-O7zXGOpXLVg90H21EOLACvtfcHk0ljPL7-OC3aFzQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFCit7rP&md5=a01ff43e9b696fe4dc03f0815158a98f</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.211&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.211%26sid%3Dliteratum%253Aachs%26aulast%3DLai%26aufirst%3DA.%2BC.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DInduced%2520protein%2520degradation%253A%2520an%2520emerging%2520drug%2520discovery%2520paradigm%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2017%26volume%3D16%26spage%3D101%26epage%3D114%26doi%3D10.1038%2Fnrd.2016.211" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Burslem, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Small-molecule modulation of protein homeostasis</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>117</i></span>,  <span class="NLM_fpage">11269</span>– <span class="NLM_lpage">11301</span>, <span class="refDoi"> DOI: 10.1021/acs.chemrev.7b00077</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.7b00077" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1Ohur3K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=117&publication_year=2017&pages=11269-11301&author=G.+M.+Burslemauthor=C.+M.+Crews&title=Small-molecule+modulation+of+protein+homeostasis&doi=10.1021%2Facs.chemrev.7b00077"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Small-Molecule Modulation of Protein Homeostasis</span></div><div class="casAuthors">Burslem, George M.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">117</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">11269-11301</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Control of protein levels by nucleic-acid-based technologies has proven to be a useful research tool but lacks the advantages of small mols. with respect to cell permeability, temporal control, and the potential generation of therapeutics.  In this Review, we discuss the technologies available for the control of intracellular protein levels with small mols. and compare the various systems available.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpcARydbgPK9bVg90H21EOLACvtfcHk0ljPL7-OC3aFzQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1Ohur3K&md5=f9f8dd98e2c71c7166d2d500b514d578</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.7b00077&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.7b00077%26sid%3Dliteratum%253Aachs%26aulast%3DBurslem%26aufirst%3DG.%2BM.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DSmall-molecule%2520modulation%2520of%2520protein%2520homeostasis%26jtitle%3DChem.%2520Rev.%26date%3D2017%26volume%3D117%26spage%3D11269%26epage%3D11301%26doi%3D10.1021%2Facs.chemrev.7b00077" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cromm, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Targeted protein degradation: from chemical biology to drug discovery</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">1181</span>– <span class="NLM_lpage">1190</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2017.05.024</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=10.1016%2Fj.chembiol.2017.05.024" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=28648379" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVOjtrrO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2017&pages=1181-1190&author=P.+M.+Crommauthor=C.+M.+Crews&title=Targeted+protein+degradation%3A+from+chemical+biology+to+drug+discovery&doi=10.1016%2Fj.chembiol.2017.05.024"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted Protein Degradation: from Chemical Biology to Drug Discovery</span></div><div class="casAuthors">Cromm, Philipp M.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1181-1190</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Traditional pharmaceutical drug discovery is almost exclusively focused on directly controlling protein activity to cure diseases.  Modulators of protein activity, esp. inhibitors, are developed and applied at high concn. to achieve maximal effects.  Thereby, reduced bioavailability and off-target effects can hamper compd. efficacy.  Nucleic acid-based strategies that control protein function by affecting expression have emerged as an alternative.  However, metabolic stability and broad bioavailability represent development hurdles that remain to be overcome for these approaches.  More recently, utilizing the cell's own protein destruction machinery for selective degrdn. of essential drivers of human disorders has opened up a new and exciting area of drug discovery.  Small-mol.-induced proteolysis of selected substrates offers the potential of reaching beyond the limitations of the current pharmaceutical paradigm to expand the druggable target space.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrosuhmnPE0RrVg90H21EOLACvtfcHk0lhYR4fmTuKoLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVOjtrrO&md5=3c8c6d9488396d5eb08ca51cee20bbf4</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2017.05.024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2017.05.024%26sid%3Dliteratum%253Aachs%26aulast%3DCromm%26aufirst%3DP.%2BM.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DTargeted%2520protein%2520degradation%253A%2520from%2520chemical%2520biology%2520to%2520drug%2520discovery%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2017%26volume%3D24%26spage%3D1181%26epage%3D1190%26doi%3D10.1016%2Fj.chembiol.2017.05.024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ottis, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Proteolysis-targeting chimeras: induced protein degradation as a therapeutic strategy</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">892</span>– <span class="NLM_lpage">898</span>, <span class="refDoi"> DOI: 10.1021/acschembio.6b01068</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.6b01068" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjs12ns70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=892-898&author=P.+Ottisauthor=C.+M.+Crews&title=Proteolysis-targeting+chimeras%3A+induced+protein+degradation+as+a+therapeutic+strategy&doi=10.1021%2Facschembio.6b01068"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Proteolysis-Targeting Chimeras: Induced Protein Degradation as a Therapeutic Strategy</span></div><div class="casAuthors">Ottis, Philipp; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">892-898</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Until recently, the only ways to reduce specific protein signaling were to either knock down the target by RNAi or to interfere with the signaling by inhibiting an enzyme or receptor within the signal transduction cascade.  Herein, we review an emerging class of small mol. pharmacol. agents, called PROTACs, that present a novel approach to specifically target proteins and their resp. signaling pathways.  These heterobifunctional mols. utilize endogenous cellular quality control machinery by recruiting it to target proteins in order to induce their degrdn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohB6xwkQMTprVg90H21EOLACvtfcHk0lhYR4fmTuKoLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjs12ns70%253D&md5=eada521c10faa1e456e0885fd390c554</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Facschembio.6b01068&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.6b01068%26sid%3Dliteratum%253Aachs%26aulast%3DOttis%26aufirst%3DP.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DProteolysis-targeting%2520chimeras%253A%2520induced%2520protein%2520degradation%2520as%2520a%2520therapeutic%2520strategy%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2017%26volume%3D12%26spage%3D892%26epage%3D898%26doi%3D10.1021%2Facschembio.6b01068" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguez-Gonzalez, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cyrus, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salcius, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deshaies, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakamoto, K. M.</span></span> <span> </span><span class="NLM_article-title">Targeting steroid hormone receptors for ubiquitination and degradation in breast and prostate cancer</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">7201</span>– <span class="NLM_lpage">7211</span>, <span class="refDoi"> DOI: 10.1038/onc.2008.320</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=10.1038%2Fonc.2008.320" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=18794799" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsVGlu7%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2008&pages=7201-7211&author=A.+Rodriguez-Gonzalezauthor=K.+Cyrusauthor=M.+Salciusauthor=K.+Kimauthor=C.+M.+Crewsauthor=R.+J.+Deshaiesauthor=K.+M.+Sakamoto&title=Targeting+steroid+hormone+receptors+for+ubiquitination+and+degradation+in+breast+and+prostate+cancer&doi=10.1038%2Fonc.2008.320"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting steroid hormone receptors for ubiquitination and degradation in breast and prostate cancer</span></div><div class="casAuthors">Rodriguez-Gonzalez, A.; Cyrus, K.; Salcius, M.; Kim, K.; Crews, C. M.; Deshaies, R. J.; Sakamoto, K. M.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">57</span>),
    <span class="NLM_cas:pages">7201-7211</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Proteolysis targeting chimeric mols. (Protacs) target proteins for destruction by exploiting the ubiquitin-dependent proteolytic system of eukaryotic cells.  We designed two Protacs that contain the peptide degron' from hypoxia-inducible factor-1α, which binds to the Von -Hippel-Lindau (VHL) E3 ubiquitin ligase complex, linked to either dihydroxytestosterone that targets the androgen receptor (AR; Protac-A), or linked to estradiol (E2) that targets the estrogen receptor-α (ERα; Protac-B).  We hypothesized that these Protacs would recruit hormone receptors to the VHL E3 ligase complex, resulting in the degrdn. of receptors, and decreased proliferation of hormone-dependent cell lines.  Treatment of estrogen-dependent breast cancer cells with Protac-B induced the degrdn. of ERα in a proteasome-dependent manner.  Protac-B inhibited the proliferation of ERα-dependent breast cancer cells by inducing G1 arrest, inhibition of retinoblastoma phosphorylation and decreasing expression of cyclin D1, progesterone receptors A and B.  Protac-B treatment did not affect the proliferation of estrogen-independent breast cancer cells that lacked ERα expression.  Similarly, Protac-A treatment of androgen-dependent prostate cancer cells induced G1 arrest but did not affect cells that do not express AR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_DgqkmmrRTbVg90H21EOLACvtfcHk0lhYR4fmTuKoLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsVGlu7%252FM&md5=acd4620a0c095a25bcf733fc2d74e84a</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1038%2Fonc.2008.320&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fonc.2008.320%26sid%3Dliteratum%253Aachs%26aulast%3DRodriguez-Gonzalez%26aufirst%3DA.%26aulast%3DCyrus%26aufirst%3DK.%26aulast%3DSalcius%26aufirst%3DM.%26aulast%3DKim%26aufirst%3DK.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26aulast%3DDeshaies%26aufirst%3DR.%2BJ.%26aulast%3DSakamoto%26aufirst%3DK.%2BM.%26atitle%3DTargeting%2520steroid%2520hormone%2520receptors%2520for%2520ubiquitination%2520and%2520degradation%2520in%2520breast%2520and%2520prostate%2520cancer%26jtitle%3DOncogene%26date%3D2008%26volume%3D27%26spage%3D7201%26epage%3D7211%26doi%3D10.1038%2Fonc.2008.320" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span> <span> </span><span class="NLM_article-title">Development of stabilized peptide-based PROTACs against estrogen receptor α</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">628</span>– <span class="NLM_lpage">635</span>, <span class="refDoi"> DOI: 10.1021/acschembio.7b00985</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.7b00985" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A528%3ADC%252BC2sXitVaksr3M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=628-635&author=Y.+Jiangauthor=Q.+Dengauthor=H.+Zhaoauthor=M.+Xieauthor=L.+Chenauthor=F.+Yinauthor=X.+Qinauthor=W.+Zhengauthor=Y.+Zhaoauthor=Z.+Li&title=Development+of+stabilized+peptide-based+PROTACs+against+estrogen+receptor+%CE%B1&doi=10.1021%2Facschembio.7b00985"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Development of Stabilized Peptide-Based PROTACs against Estrogen Receptor α</span></div><div class="casAuthors">Jiang, Yanhong; Deng, Qiwen; Zhao, Hui; Xie, Mingsheng; Chen, Longjian; Yin, Feng; Qin, Xuan; Zheng, Weihao; Zhao, Yongjuan; Li, Zigang</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">628-635</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Peptide modulators targeting protein-protein interactions (PPIs) exhibit greater potential than small-mol. drugs in several important aspects including facile modification and relative large contact surface area.  Stabilized peptides constructed by variable chem. methods exhibit improved peptide stability and cell permeability compared to that of the linears.  Herein, we designed a stabilized peptide-based proteolysis-targeting chimera (PROTAC) targeting estrogen receptor α (ERα) by tethering an N-terminal aspartic acid cross-linked stabilized peptide ERα modulator (TD-PERM) with a pentapeptide that binds the Von Hippel-Lindau (VHL) E3 ubiquitin ligase complex.  The resulting heterobifunctional peptide (TD-PROTAC) selectively recruits ERα to the VHL E3 ligase complex, leading to the degrdn. of ERα in a proteasome-dependent manner.  Compared with the control peptides, TD-PROTAC shows significantly enhanced activities in reducing the transcription of the ERα-downstream genes and inhibiting the proliferation of ERα-pos. breast cancer cells.  In addn., in vivo expts. indicate that TD-PROTAC leads to tumor regression in the MCF-7 mouse xenograft model.  This work is a successful attempt to construct PROTACs based on cell-permeable stabilized peptides, which significantly broadens the chem. space of PROTACs and stabilized peptides.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovSDWxTR3LbLVg90H21EOLACvtfcHk0lhYR4fmTuKoLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXitVaksr3M&md5=abcd6f72223751db6d96f8027cbb8c23</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Facschembio.7b00985&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.7b00985%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DDeng%26aufirst%3DQ.%26aulast%3DZhao%26aufirst%3DH.%26aulast%3DXie%26aufirst%3DM.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DYin%26aufirst%3DF.%26aulast%3DQin%26aufirst%3DX.%26aulast%3DZheng%26aufirst%3DW.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DZ.%26atitle%3DDevelopment%2520of%2520stabilized%2520peptide-based%2520PROTACs%2520against%2520estrogen%2520receptor%2520%25CE%25B1%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2018%26volume%3D13%26spage%3D628%26epage%3D635%26doi%3D10.1021%2Facschembio.7b00985" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G. E. B. D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulk, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Souza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhe-Paganon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">Phthalimide conjugation as a strategy for in vivo target protein degradation</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>348</i></span>,  <span class="NLM_fpage">1376</span>– <span class="NLM_lpage">1381</span>, <span class="refDoi"> DOI: 10.1126/science.aab1433</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=10.1126%2Fscience.aab1433" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=25999370" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A528%3ADC%252BC2MXpvFansLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=348&publication_year=2015&pages=1376-1381&author=G.+E.+B.+D.+L.+Winterauthor=D.+L.+Buckleyauthor=J.+Paulkauthor=J.+M.+Robertsauthor=A.+Souzaauthor=S.+Dhe-Paganonauthor=J.+E.+Bradner&title=Phthalimide+conjugation+as+a+strategy+for+in+vivo+target+protein+degradation&doi=10.1126%2Fscience.aab1433"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Phthalimide conjugation as a strategy for in vivo target protein degradation</span></div><div class="casAuthors">Winter, Georg E.; Buckley, Dennis L.; Paulk, Joshiawa; Roberts, Justin M.; Souza, Amanda; Dhe-Paganon, Sirano; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">348</span>
        (<span class="NLM_cas:issue">6241</span>),
    <span class="NLM_cas:pages">1376-1381</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The development of effective pharmacol. inhibitors of multidomain scaffold proteins, notably transcription factors, is a particularly challenging problem.  In part, this is because many small-mol. antagonists disrupt the activity of only one domain in the target protein.  The authors devised a chem. strategy that promotes ligand-dependent target protein degrdn. using as an example the transcriptional coactivator BRD4, a protein crit. for cancer cell growth and survival.  The authors appended a competitive antagonist of BET bromodomains to a phthalimide moiety to hijack the cereblon E3 ubiquitin ligase complex.  The resultant compd., dBET1, induced highly selective cereblon-dependent BET protein degrdn. in vitro and in vivo and delayed leukemia progression in mice.  A second series of probes resulted in selective degrdn. of the cytosolic protein FKBP12.  This chem. strategy for controlling target protein stability may have implications for therapeutically targeting previously intractable proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1ZW4zjRjXi7Vg90H21EOLACvtfcHk0lg13wyGlLOz4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXpvFansLk%253D&md5=b0277cc95d318d926b96e7a119066e3b</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1126%2Fscience.aab1433&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aab1433%26sid%3Dliteratum%253Aachs%26aulast%3DWinter%26aufirst%3DG.%2BE.%2BB.%2BD.%2BL.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DPaulk%26aufirst%3DJ.%26aulast%3DRoberts%26aufirst%3DJ.%2BM.%26aulast%3DSouza%26aufirst%3DA.%26aulast%3DDhe-Paganon%26aufirst%3DS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DPhthalimide%2520conjugation%2520as%2520a%2520strategy%2520for%2520in%2520vivo%2520target%2520protein%2520degradation%26jtitle%3DScience%26date%3D2015%26volume%3D348%26spage%3D1376%26epage%3D1381%26doi%3D10.1126%2Fscience.aab1433" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altieri, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raina, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hines, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crew, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Hijacking the E3 ubiquitin ligase cereblon to efficiently target BRD4</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">755</span>– <span class="NLM_lpage">763</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2015.05.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=10.1016%2Fj.chembiol.2015.05.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=26051217" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVWrtbrM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2015&pages=755-763&author=J.+Luauthor=Y.+Qianauthor=M.+Altieriauthor=H.+Dongauthor=J.+Wangauthor=K.+Rainaauthor=J.+Hinesauthor=J.+D.+Winklerauthor=A.+P.+Crewauthor=K.+Colemanauthor=C.+M.+Crews&title=Hijacking+the+E3+ubiquitin+ligase+cereblon+to+efficiently+target+BRD4&doi=10.1016%2Fj.chembiol.2015.05.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Hijacking the E3 Ubiquitin Ligase Cereblon to Efficiently Target BRD4</span></div><div class="casAuthors">Lu, Jing; Qian, Yimin; Altieri, Martha; Dong, Hanqing; Wang, Jing; Raina, Kanak; Hines, John; Winkler, James D.; Crew, Andrew P.; Coleman, Kevin; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">755-763</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">BRD4, a bromodomain and extraterminal domain (BET) family member, is an attractive target in multiple pathol. settings, particularly cancer.  While BRD4 inhibitors have shown some promise in MYC-driven malignancies such as Burkitt's lymphoma (BL), we show that BRD4 inhibitors lead to robust BRD4 protein accumulation, which may account for their limited suppression of MYC expression, modest antiproliferative activity, and lack of apoptotic induction.  To address these limitations we designed ARV-825, a hetero-bifunctional PROTAC (Proteolysis Targeting Chimera) that recruits BRD4 to the E3 ubiquitin ligase cereblon, leading to fast, efficient, and prolonged degrdn. of BRD4 in all BL cell lines tested.  Consequently, ARV-825 more effectively suppresses c-MYC levels and downstream signaling than small-mol. BRD4 inhibitors, resulting in more effective cell proliferation inhibition and apoptosis induction in BL.  Our findings provide strong evidence that cereblon-based PROTACs provide a better and more efficient strategy in targeting BRD4 than traditional small-mol. inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6Ol4grCG2RbVg90H21EOLACvtfcHk0lg13wyGlLOz4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVWrtbrM&md5=a64675e7a0cd006a8737be9730f447e3</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2015.05.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2015.05.009%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DQian%26aufirst%3DY.%26aulast%3DAltieri%26aufirst%3DM.%26aulast%3DDong%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DRaina%26aufirst%3DK.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DWinkler%26aufirst%3DJ.%2BD.%26aulast%3DCrew%26aufirst%3DA.%2BP.%26aulast%3DColeman%26aufirst%3DK.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DHijacking%2520the%2520E3%2520ubiquitin%2520ligase%2520cereblon%2520to%2520efficiently%2520target%2520BRD4%26jtitle%3DChem.%2520Biol.%26date%3D2015%26volume%3D22%26spage%3D755%26epage%3D763%26doi%3D10.1016%2Fj.chembiol.2015.05.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bondeson, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mares, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, I. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ko, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campos, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miah, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulholland, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Routly, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gustafson, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zinn, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shimamura, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergamini, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faelth-Savitski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bantscheff, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willard, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flanagan, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casillas, L. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Votta, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">den Besten, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Famm, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kruidenier, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harling, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Churcher, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Catalytic in vivo protein knockdown by small-molecule PROTACs</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">611</span>– <span class="NLM_lpage">617</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1858</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=10.1038%2Fnchembio.1858" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=26075522" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFeju7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=611-617&author=D.+P.+Bondesonauthor=A.+Maresauthor=I.+E.+Smithauthor=E.+Koauthor=S.+Camposauthor=A.+H.+Miahauthor=K.+E.+Mulhollandauthor=N.+Routlyauthor=D.+L.+Buckleyauthor=J.+L.+Gustafsonauthor=N.+Zinnauthor=P.+Grandiauthor=S.+Shimamuraauthor=G.+Bergaminiauthor=M.+Faelth-Savitskiauthor=M.+Bantscheffauthor=C.+Coxauthor=D.+A.+Gordonauthor=R.+R.+Willardauthor=J.+J.+Flanaganauthor=L.+N.+Casillasauthor=B.+J.+Vottaauthor=W.+den+Bestenauthor=K.+Fammauthor=L.+Kruidenierauthor=P.+S.+Carterauthor=J.+D.+Harlingauthor=I.+Churcherauthor=C.+M.+Crews&title=Catalytic+in+vivo+protein+knockdown+by+small-molecule+PROTACs&doi=10.1038%2Fnchembio.1858"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Catalytic in vivo protein knockdown by small-molecule PROTACs</span></div><div class="casAuthors">Bondeson, Daniel P.; Mares, Alina; Smith, Ian E. D.; Ko, Eunhwa; Campos, Sebastien; Miah, Afjal H.; Mulholland, Katie E.; Routly, Natasha; Buckley, Dennis L.; Gustafson, Jeffrey L.; Zinn, Nico; Grandi, Paola; Shimamura, Satoko; Bergamini, Giovanna; Faelth-Savitski, Maria; Bantscheff, Marcus; Cox, Carly; Gordon, Deborah A.; Willard, Ryan R.; Flanagan, John J.; Casillas, Linda N.; Votta, Bartholomew J.; den Besten, Willem; Famm, Kristoffer; Kruidenier, Laurens; Carter, Paul S.; Harling, John D.; Churcher, Ian; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">611-617</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The current predominant therapeutic paradigm is based on maximizing drug-receptor occupancy to achieve clin. benefit.  This strategy, however, generally requires excessive drug concns. to ensure sufficient occupancy, often leading to adverse side effects.  Here, we describe major improvements to the proteolysis targeting chimeras (PROTACs) method, a chem. knockdown strategy in which a heterobifunctional mol. recruits a specific protein target to an E3 ubiquitin ligase, resulting in the target's ubiquitination and degrdn.  These compds. behave catalytically in their ability to induce the ubiquitination of super-stoichiometric quantities of proteins, providing efficacy that is not limited by equil. occupancy.  We present two PROTACs that are capable of specifically reducing protein levels by >90% at nanomolar concns.  In addn., mouse studies indicate that they provide broad tissue distribution and knockdown of the targeted protein in tumor xenografts.  Together, these data demonstrate a protein knockdown system combining many of the favorable properties of small-mol. agents with the potent protein knockdown of RNAi and CRISPR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonhnajD3CxS7Vg90H21EOLACvtfcHk0lg13wyGlLOz4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFeju7rP&md5=31eb8756c5661f78ab4527a50dc1e251</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1858&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1858%26sid%3Dliteratum%253Aachs%26aulast%3DBondeson%26aufirst%3DD.%2BP.%26aulast%3DMares%26aufirst%3DA.%26aulast%3DSmith%26aufirst%3DI.%2BE.%26aulast%3DKo%26aufirst%3DE.%26aulast%3DCampos%26aufirst%3DS.%26aulast%3DMiah%26aufirst%3DA.%2BH.%26aulast%3DMulholland%26aufirst%3DK.%2BE.%26aulast%3DRoutly%26aufirst%3DN.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DGustafson%26aufirst%3DJ.%2BL.%26aulast%3DZinn%26aufirst%3DN.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DShimamura%26aufirst%3DS.%26aulast%3DBergamini%26aufirst%3DG.%26aulast%3DFaelth-Savitski%26aufirst%3DM.%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DCox%26aufirst%3DC.%26aulast%3DGordon%26aufirst%3DD.%2BA.%26aulast%3DWillard%26aufirst%3DR.%2BR.%26aulast%3DFlanagan%26aufirst%3DJ.%2BJ.%26aulast%3DCasillas%26aufirst%3DL.%2BN.%26aulast%3DVotta%26aufirst%3DB.%2BJ.%26aulast%3Dden%2BBesten%26aufirst%3DW.%26aulast%3DFamm%26aufirst%3DK.%26aulast%3DKruidenier%26aufirst%3DL.%26aulast%3DCarter%26aufirst%3DP.%2BS.%26aulast%3DHarling%26aufirst%3DJ.%2BD.%26aulast%3DChurcher%26aufirst%3DI.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DCatalytic%2520in%2520vivo%2520protein%2520knockdown%2520by%2520small-molecule%2520PROTACs%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26spage%3D611%26epage%3D617%26doi%3D10.1038%2Fnchembio.1858" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lai, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toure, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hellerschmied, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salami, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaime-Figueroa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ko, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hines, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Modular PROTAC design for the degradation of oncogenic BCR-ABL</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">807</span>– <span class="NLM_lpage">810</span>, <span class="refDoi"> DOI: 10.1002/anie.201507634</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=10.1002%2Fanie.201507634" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVOlt7rN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2016&pages=807-810&author=A.+C.+Laiauthor=M.+Toureauthor=D.+Hellerschmiedauthor=J.+Salamiauthor=S.+Jaime-Figueroaauthor=E.+Koauthor=J.+Hinesauthor=C.+M.+Crews&title=Modular+PROTAC+design+for+the+degradation+of+oncogenic+BCR-ABL&doi=10.1002%2Fanie.201507634"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Modular PROTAC Design for the Degradation of Oncogenic BCR-ABL</span></div><div class="casAuthors">Lai, Ashton C.; Toure, Momar; Hellerschmied, Doris; Salami, Jemilat; Jaime-Figueroa, Saul; Ko, Eunhwa; Hines, John; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">807-810</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Proteolysis Targeting Chimera (PROTAC) technol. is a rapidly emerging alternative therapeutic strategy with the potential to address many of the challenges currently faced in modern drug development programs.  PROTAC technol. employs small mols. that recruit target proteins for ubiquitination and removal by the proteasome.  The synthesis of PROTAC compds. that mediate the degrdn. of c-ABL and BCR-ABL by recruiting either Cereblon or Von Hippel Lindau E3 ligases is reported.  During the course of their development, the authors discovered that the capacity of a PROTAC to induce degrdn. involves more than just target binding: the identity of the inhibitor warhead and the recruited E3 ligase largely det. the degrdn. profiles of the compds.; thus, as a starting point for PROTAC development, both the target ligand and the recruited E3 ligase should be varied to rapidly generate a PROTAC with the desired degrdn. profile.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdcSTHt5WfILVg90H21EOLACvtfcHk0ljfESQfajNg5g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVOlt7rN&md5=58a5d78f32bc8a21fd1a6e0c38b8979c</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1002%2Fanie.201507634&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201507634%26sid%3Dliteratum%253Aachs%26aulast%3DLai%26aufirst%3DA.%2BC.%26aulast%3DToure%26aufirst%3DM.%26aulast%3DHellerschmied%26aufirst%3DD.%26aulast%3DSalami%26aufirst%3DJ.%26aulast%3DJaime-Figueroa%26aufirst%3DS.%26aulast%3DKo%26aufirst%3DE.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DModular%2520PROTAC%2520design%2520for%2520the%2520degradation%2520of%2520oncogenic%2520BCR-ABL%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2016%26volume%3D55%26spage%3D807%26epage%3D810%26doi%3D10.1002%2Fanie.201507634" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Robb, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Contreras, J. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kour, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abid, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sonawane, Y. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zahid, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murry, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Natarajan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rana, S.</span></span> <span> </span><span class="NLM_article-title">Chemically induced degradation of CDK9 by a proteolysis targeting chimera (PROTAC)</span>. <i>Chem. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">7577</span>– <span class="NLM_lpage">7580</span>, <span class="refDoi"> DOI: 10.1039/C7CC03879H</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=10.1039%2FC7CC03879H" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=28636052" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A528%3ADC%252BC2sXpvFyjsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2017&pages=7577-7580&author=C.+M.+Robbauthor=J.+I.+Contrerasauthor=S.+Kourauthor=M.+A.+Taylorauthor=M.+Abidauthor=Y.+A.+Sonawaneauthor=M.+Zahidauthor=D.+J.+Murryauthor=A.+Natarajanauthor=S.+Rana&title=Chemically+induced+degradation+of+CDK9+by+a+proteolysis+targeting+chimera+%28PROTAC%29&doi=10.1039%2FC7CC03879H"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Chemically induced degradation of CDK9 by a proteolysis targeting chimera (PROTAC)</span></div><div class="casAuthors">Robb, Caroline M.; Contreras, Jacob I.; Kour, Smit; Taylor, Margaret A.; Abid, Mohammad; Sonawane, Yogesh A.; Zahid, Muhammad; Murry, Daryl J.; Natarajan, Amarnath; Rana, Sandeep</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Communications (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">54</span>),
    <span class="NLM_cas:pages">7577-7580</span>CODEN:
                <span class="NLM_cas:coden">CHCOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-7345</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Cyclin-dependent kinase 9 (CDK9), a member of the cyclin-dependent protein kinase (CDK) family, is involved in transcriptional elongation of several target genes.  CDK9 is ubiquitously expressed and has been shown to contribute to a variety of malignancies such as pancreatic, prostate and breast cancers.  Here we report the development of a heterobifunctional small mol. proteolysis targeting chimera (PROTAC) capable of cereblon (CRBN)-mediated proteasomal degrdn. of CDK9.  In HCT116 cells, it selectively degrades CDK9 while sparing other CDK family members.  This is the first example of a PROTAC that selectively degrades CDK9.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXgLtOTwvlnbVg90H21EOLACvtfcHk0ljfESQfajNg5g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXpvFyjsLw%253D&md5=a9aa8ebf8109f11206740efd1b5e09a2</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1039%2FC7CC03879H&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC7CC03879H%26sid%3Dliteratum%253Aachs%26aulast%3DRobb%26aufirst%3DC.%2BM.%26aulast%3DContreras%26aufirst%3DJ.%2BI.%26aulast%3DKour%26aufirst%3DS.%26aulast%3DTaylor%26aufirst%3DM.%2BA.%26aulast%3DAbid%26aufirst%3DM.%26aulast%3DSonawane%26aufirst%3DY.%2BA.%26aulast%3DZahid%26aufirst%3DM.%26aulast%3DMurry%26aufirst%3DD.%2BJ.%26aulast%3DNatarajan%26aufirst%3DA.%26aulast%3DRana%26aufirst%3DS.%26atitle%3DChemically%2520induced%2520degradation%2520of%2520CDK9%2520by%2520a%2520proteolysis%2520targeting%2520chimera%2520%2528PROTAC%2529%26jtitle%3DChem.%2520Commun.%26date%3D2017%26volume%3D53%26spage%3D7577%26epage%3D7580%26doi%3D10.1039%2FC7CC03879H" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, X. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Proteolysis targeting chimeras (PROTACs) of anaplastic lymphoma kinase (ALK)</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>151</i></span>,  <span class="NLM_fpage">304</span>– <span class="NLM_lpage">314</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2018.03.071</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=10.1016%2Fj.ejmech.2018.03.071" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=29627725" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A528%3ADC%252BC1cXnt1ejs7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=151&publication_year=2018&pages=304-314&author=C.+Zhangauthor=X.+R.+Hanauthor=X.+Yangauthor=B.+Jiangauthor=J.+Liuauthor=Y.+Xiongauthor=J.+Jin&title=Proteolysis+targeting+chimeras+%28PROTACs%29+of+anaplastic+lymphoma+kinase+%28ALK%29&doi=10.1016%2Fj.ejmech.2018.03.071"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Proteolysis Targeting Chimeras (PROTACs) of Anaplastic Lymphoma Kinase (ALK)</span></div><div class="casAuthors">Zhang, Chengwei; Han, Xiao-Ran; Yang, Xiaobao; Jiang, Biao; Liu, Jing; Xiong, Yue; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">151</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">304-314</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Anaplastic lymphoma kinase (ALK) activation has been assocd. with many types of human cancer.  Significant efforts have been devoted to the development of ALK inhibitors to antagonize the kinase activity of ALK.  Four ALK inhibitors have been approved by the FDA to date for treating patients with ALK-pos. nonsmall cell lung cancers (NSCLC).  However, drug resistance has been obsd. in the majority of patients treated with these inhibitors.  New therapeutic strategies (e.g., compds. with novel mechanisms of action) are needed to overcome the drug resistance issue.  The emerging PROTAC (Proteolysis Targeting Chimera) technol. has been successfully applied to selective degrdn. of multiple protein targets, but not ALK.  Since ALK protein levels are not important for viability in mammals, ALK PROTACs could lead to novel therapeutics with minimal toxicity.  Here the authors report the design, synthesis and biol. evaluation of novel PROTACs (degraders) of ALK.  MS4077 (I) and MS4078 (II) potently decreased cellular levels of oncogenic active ALK fusion proteins in a concn.- and time-dependent manner in SU-DHL-1 lymphoma and NCI-H2228 lung cancer cells.  The ALK protein degrdn. induced by compds. I and II was cereblon and proteasome dependent.  In addn., compds. I and II potently inhibited proliferation of SU-DHL-1 cells.  Furthermore, compd. II displayed good plasma exposure in a mouse pharmacokinetic study, thus is suitable for in vivo efficacy studies.  The authors also developed MS4748 and MS4740 , very close analogs of I and II resp., which are incapable to degrade the ALK fusion proteins, as neg. controls.  All 4 compds. are valuable chem. tools for investigating effects of ALK pharmacol. degrdn.  The study paved the way for developing the next generation of ALK PROTACs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPGM3aAZZ6h7Vg90H21EOLACvtfcHk0ljfESQfajNg5g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXnt1ejs7Y%253D&md5=804324b95034f38b46e45f89e18e55be</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2018.03.071&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2018.03.071%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DHan%26aufirst%3DX.%2BR.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DJiang%26aufirst%3DB.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DXiong%26aufirst%3DY.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DProteolysis%2520targeting%2520chimeras%2520%2528PROTACs%2529%2520of%2520anaplastic%2520lymphoma%2520kinase%2520%2528ALK%2529%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2018%26volume%3D151%26spage%3D304%26epage%3D314%26doi%3D10.1016%2Fj.ejmech.2018.03.071" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Z.</span></span> <span> </span><span class="NLM_article-title">Discovery of a Keap1-dependent peptide PROTAC to knockdown Tau by ubiquitination-proteasome degradation pathway</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>146</i></span>,  <span class="NLM_fpage">251</span>– <span class="NLM_lpage">259</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2018.01.063</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=10.1016%2Fj.ejmech.2018.01.063" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=29407955" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisFehsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=146&publication_year=2018&pages=251-259&author=M.+Luauthor=T.+Liuauthor=Q.+Jiaoauthor=J.+Jiauthor=M.+Taoauthor=Y.+Liuauthor=Q.+Youauthor=Z.+Jiang&title=Discovery+of+a+Keap1-dependent+peptide+PROTAC+to+knockdown+Tau+by+ubiquitination-proteasome+degradation+pathway&doi=10.1016%2Fj.ejmech.2018.01.063"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Keap1-dependent peptide PROTAC to knockdown Tau by ubiquitination-proteasome degradation pathway</span></div><div class="casAuthors">Lu, Mengchen; Liu, Tian; Jiao, Qiong; Ji, Jianai; Tao, Mengmin; Liu, Yijun; You, Qidong; Jiang, Zhengyu</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">146</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">251-259</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Induced protein degrdn. by PROTACs has emerged as a promising strategy to target nonenzymic proteins inside the cell.  The aim of this study was to identify Keap1, a substrate adaptor protein for ubiquitin E3 ligase involved in oxidative stress regulation, as a novel candidate for PROTACs that can be applied in the degrdn. of the nonenzymic protein Tau.  A peptide PROTAC by recruiting Keap1-Cul3 ubiquitin E3 ligase was developed and applied in the degrdn. of intracellular Tau.  Peptide 1 showed strong in vitro binding with Keap1 and Tau.  With proper cell permeability, peptide 1 was found to colocalize with cellular Keap1 and resulted in the coimmunopptn. of Tau and Keap1.  The results of flow cytometry and western blotting assays showed that peptide 1 can downregulate the intracellular Tau level in both time- and concn.-dependent manner.  The application of Keap1 siRNA silencing and the proteasome inhibitor MG132 confirmed that peptide 1 could promote the Keap1-dependent poly-ubiquitination and proteasome-dependent degrdn. of Tau.  The results suggested that using PROTACs to recruit Keap1 to induce the degrdn. of Tau may show promising character in the treatment of neurodegenerative disease.  In addn., our research demonstrated that Keap1 should be a promising E3 ligase adaptor to be used in the design of novel PROTACs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9KC7pnPqlL7Vg90H21EOLACvtfcHk0ljH1On7RoVyCw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisFehsbY%253D&md5=165e6f510e4e2024aca6c4135d2840b2</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2018.01.063&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2018.01.063%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DT.%26aulast%3DJiao%26aufirst%3DQ.%26aulast%3DJi%26aufirst%3DJ.%26aulast%3DTao%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DYou%26aufirst%3DQ.%26aulast%3DJiang%26aufirst%3DZ.%26atitle%3DDiscovery%2520of%2520a%2520Keap1-dependent%2520peptide%2520PROTAC%2520to%2520knockdown%2520Tau%2520by%2520ubiquitination-proteasome%2520degradation%2520pathway%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2018%26volume%3D146%26spage%3D251%26epage%3D259%26doi%3D10.1016%2Fj.ejmech.2018.01.063" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Burslem, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaime-Figueroa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McQuaid, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bondeson, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toure, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crew, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hines, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">The advantages of targeted protein degradation over inhibition: an RTK case study</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">67</span>– <span class="NLM_lpage">77</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2017.09.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=10.1016%2Fj.chembiol.2017.09.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=29129716" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrjM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=67-77&author=G.+M.+Burslemauthor=B.+E.+Smithauthor=A.+C.+Laiauthor=S.+Jaime-Figueroaauthor=D.+C.+McQuaidauthor=D.+P.+Bondesonauthor=M.+Toureauthor=H.+Dongauthor=Y.+Qianauthor=J.+Wangauthor=A.+P.+Crewauthor=J.+Hinesauthor=C.+M.+Crews&title=The+advantages+of+targeted+protein+degradation+over+inhibition%3A+an+RTK+case+study&doi=10.1016%2Fj.chembiol.2017.09.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">The Advantages of Targeted Protein Degradation Over Inhibition: An RTK Case Study</span></div><div class="casAuthors">Burslem, George M.; Smith, Blake E.; Lai, Ashton C.; Jaime-Figueroa, Saul; McQuaid, Daniel C.; Bondeson, Daniel P.; Toure, Momar; Dong, Hanqing; Qian, Yimin; Wang, Jing; Crew, Andrew P.; Hines, John; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">67-77.e3</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Proteolysis targeting chimera (PROTAC) technol. has emerged over the last two decades as a powerful tool for targeted degrdn. of endogenous proteins.  Herein we describe the development of PROTACs for receptor tyrosine kinases, a protein family yet to be targeted for induced protein degrdn.  The use of VHL-recruiting PROTACs against this protein family reveals several advantages of degrdn. over inhibition alone: direct comparisons of fully functional, target-degrading PROTACs with target-inhibiting variants that contain an inactivated E3 ligase-recruiting ligand show that degrdn. leads to more potent inhibition of cell proliferation and a more durable and sustained downstream signaling response, and thus addresses the kinome rewiring challenge seen with many receptor tyrosine kinase inhibitors.  Combined, these findings demonstrate the ability to target receptor tyrosine kinases for degrdn. using the PROTAC technol. and outline the advantages of this degrdn.-based approach.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4yDhRDWqNv7Vg90H21EOLACvtfcHk0ljH1On7RoVyCw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrjM&md5=0ac39a6f5836ea3772a0396356efb630</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2017.09.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2017.09.009%26sid%3Dliteratum%253Aachs%26aulast%3DBurslem%26aufirst%3DG.%2BM.%26aulast%3DSmith%26aufirst%3DB.%2BE.%26aulast%3DLai%26aufirst%3DA.%2BC.%26aulast%3DJaime-Figueroa%26aufirst%3DS.%26aulast%3DMcQuaid%26aufirst%3DD.%2BC.%26aulast%3DBondeson%26aufirst%3DD.%2BP.%26aulast%3DToure%26aufirst%3DM.%26aulast%3DDong%26aufirst%3DH.%26aulast%3DQian%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DCrew%26aufirst%3DA.%2BP.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DThe%2520advantages%2520of%2520targeted%2520protein%2520degradation%2520over%2520inhibition%253A%2520an%2520RTK%2520case%2520study%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2018%26volume%3D25%26spage%3D67%26epage%3D77%26doi%3D10.1016%2Fj.chembiol.2017.09.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schiedel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herp, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hammelmann, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swyter, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lehotzky, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robaa, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olah, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ovadi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sippl, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jung, M.</span></span> <span> </span><span class="NLM_article-title">Chemically induced degradation of sirtuin 2 (sirt2) by a proteolysis targeting chimera (PROTAC) based on sirtuin rearranging ligands (SirReals)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">482</span>– <span class="NLM_lpage">491</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01872</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01872" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A528%3ADC%252BC2sXlslSltrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=482-491&author=M.+Schiedelauthor=D.+Herpauthor=S.+Hammelmannauthor=S.+Swyterauthor=A.+Lehotzkyauthor=D.+Robaaauthor=J.+Olahauthor=J.+Ovadiauthor=W.+Sipplauthor=M.+Jung&title=Chemically+induced+degradation+of+sirtuin+2+%28sirt2%29+by+a+proteolysis+targeting+chimera+%28PROTAC%29+based+on+sirtuin+rearranging+ligands+%28SirReals%29&doi=10.1021%2Facs.jmedchem.6b01872"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Chemically Induced Degradation of Sirtuin 2 (Sirt2) by a Proteolysis Targeting Chimera (PROTAC) Based on Sirtuin Rearranging Ligands (SirReals)</span></div><div class="casAuthors">Schiedel, Matthias; Herp, Daniel; Hammelmann, Soeren; Swyter, Soeren; Lehotzky, Attila; Robaa, Dina; Olah, Judit; Ovadi, Judit; Sippl, Wolfgang; Jung, Manfred</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">482-491</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Here the authors report the development of a proteolysis targeting chimera (PROTAC) I based on the combination of the unique features of the sirtuin rearranging ligands (SirReals) as highly potent and isotype-selective Sirt2 inhibitors with thalidomide, a bona fide cereblon ligand.  For the first time, the authors report the formation of a PROTAC by Cu(I)-catalyzed cycloaddn. of a thalidomide-derived azide to an alkynylated inhibitor.  This thalidomide-derived azide as well as the highly versatile linking strategy can be readily adapted to alkynylated ligands of other targets.  In HeLa cells, the SirReal-based PROTAC induced isotype-selective Sirt2 degrdn. that results in the hyperacetylation of the microtubule network coupled with enhanced process elongation.  Thus, the SirReal-based PROTAC is the first example of a probe that is able to chem. induce the degrdn. of an epigenetic eraser protein.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrLUKzL4R5C0bVg90H21EOLACvtfcHk0ljH1On7RoVyCw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXlslSltrw%253D&md5=3be73e320c6d40bb87a18cad75c16bf4</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01872&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01872%26sid%3Dliteratum%253Aachs%26aulast%3DSchiedel%26aufirst%3DM.%26aulast%3DHerp%26aufirst%3DD.%26aulast%3DHammelmann%26aufirst%3DS.%26aulast%3DSwyter%26aufirst%3DS.%26aulast%3DLehotzky%26aufirst%3DA.%26aulast%3DRobaa%26aufirst%3DD.%26aulast%3DOlah%26aufirst%3DJ.%26aulast%3DOvadi%26aufirst%3DJ.%26aulast%3DSippl%26aufirst%3DW.%26aulast%3DJung%26aufirst%3DM.%26atitle%3DChemically%2520induced%2520degradation%2520of%2520sirtuin%25202%2520%2528sirt2%2529%2520by%2520a%2520proteolysis%2520targeting%2520chimera%2520%2528PROTAC%2529%2520based%2520on%2520sirtuin%2520rearranging%2520ligands%2520%2528SirReals%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D482%26epage%3D491%26doi%3D10.1021%2Facs.jmedchem.6b01872" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shibata, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagai, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morita, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ujikawa, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohoka, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hattori, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koyama, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sano, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imaeda, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nara, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naito, M.</span></span> <span> </span><span class="NLM_article-title">Development of protein degradation inducers of androgen receptor by conjugation of androgen receptor ligands and inhibitor of apoptosis protein ligands</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">543</span>– <span class="NLM_lpage">575</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00168</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00168" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A528%3ADC%252BC2sXpslSlu7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=543-575&author=N.+Shibataauthor=K.+Nagaiauthor=Y.+Moritaauthor=O.+Ujikawaauthor=N.+Ohokaauthor=T.+Hattoriauthor=R.+Koyamaauthor=O.+Sanoauthor=Y.+Imaedaauthor=H.+Naraauthor=N.+Choauthor=M.+Naito&title=Development+of+protein+degradation+inducers+of+androgen+receptor+by+conjugation+of+androgen+receptor+ligands+and+inhibitor+of+apoptosis+protein+ligands&doi=10.1021%2Facs.jmedchem.7b00168"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Development of Protein Degradation Inducers of Androgen Receptor by Conjugation of Androgen Receptor Ligands and Inhibitor of Apoptosis Protein Ligands</span></div><div class="casAuthors">Shibata, Norihito; Nagai, Katsunori; Morita, Yoko; Ujikawa, Osamu; Ohoka, Nobumichi; Hattori, Takayuki; Koyama, Ryokichi; Sano, Osamu; Imaeda, Yasuhiro; Nara, Hiroshi; Cho, Nobuo; Naito, Mikihiko</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">543-575</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Targeted protein degrdn. using small mols. is a novel strategy for drug development.  The authors have developed hybrid mols. named specific and nongenetic inhibitor of apoptosis protein [IAP]-dependent protein erasers (SNIPERs) that recruit IAP ubiquitin ligases to degrade target proteins.  Here, the authors show novel SNIPERs capable of inducing proteasomal degrdn. of the androgen receptor (AR).  Through derivatization of the SNIPER(AR) mol. at the AR ligand and IAP ligand and linker, the authors developed SNIPER(AR)-51 (I), which shows effective protein knockdown activity against AR.  Consistent with the degrdn. of the AR protein, I inhibits AR-mediated gene expression and proliferation of androgen-dependent prostate cancer cells.  In addn., I efficiently induces caspase activation and apoptosis in prostate cancer cells, which was not obsd. in the cells treated with AR antagonists.  These results suggest that SNIPER(AR)s could be leads for an anticancer drug against prostate cancers that exhibit AR-dependent proliferation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqW6OG4BZYg1rVg90H21EOLACvtfcHk0liuXY6EvmPLTA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXpslSlu7k%253D&md5=eeefbe187f2f93ca32518f6a3258b903</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00168&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00168%26sid%3Dliteratum%253Aachs%26aulast%3DShibata%26aufirst%3DN.%26aulast%3DNagai%26aufirst%3DK.%26aulast%3DMorita%26aufirst%3DY.%26aulast%3DUjikawa%26aufirst%3DO.%26aulast%3DOhoka%26aufirst%3DN.%26aulast%3DHattori%26aufirst%3DT.%26aulast%3DKoyama%26aufirst%3DR.%26aulast%3DSano%26aufirst%3DO.%26aulast%3DImaeda%26aufirst%3DY.%26aulast%3DNara%26aufirst%3DH.%26aulast%3DCho%26aufirst%3DN.%26aulast%3DNaito%26aufirst%3DM.%26atitle%3DDevelopment%2520of%2520protein%2520degradation%2520inducers%2520of%2520androgen%2520receptor%2520by%2520conjugation%2520of%2520androgen%2520receptor%2520ligands%2520and%2520inhibitor%2520of%2520apoptosis%2520protein%2520ligands%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D543%26epage%3D575%26doi%3D10.1021%2Facs.jmedchem.7b00168" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Crew, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raina, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vigil, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Serebrenik, Y. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamman, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morgan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferraro, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neklesa, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Identification and characterization of von Hippel-Lindau-recruiting proteolysis targeting chimeras (PROTACs) of TANK-binding kinase 1</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">583</span>– <span class="NLM_lpage">598</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00635</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00635" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtFelu7rK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=583-598&author=A.+P.+Crewauthor=K.+Rainaauthor=H.+Dongauthor=Y.+Qianauthor=J.+Wangauthor=D.+Vigilauthor=Y.+V.+Serebrenikauthor=B.+D.+Hammanauthor=A.+Morganauthor=C.+Ferraroauthor=K.+Siuauthor=T.+K.+Neklesaauthor=J.+D.+Winklerauthor=K.+G.+Colemanauthor=C.+M.+Crews&title=Identification+and+characterization+of+von+Hippel-Lindau-recruiting+proteolysis+targeting+chimeras+%28PROTACs%29+of+TANK-binding+kinase+1&doi=10.1021%2Facs.jmedchem.7b00635"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and Characterization of Von Hippel-Lindau-Recruiting Proteolysis Targeting Chimeras (PROTACs) of TANK-Binding Kinase 1</span></div><div class="casAuthors">Crew, Andrew P.; Raina, Kanak; Dong, Hanqing; Qian, Yimin; Wang, Jing; Vigil, Dominico; Serebrenik, Yevgeniy V.; Hamman, Brian D.; Morgan, Alicia; Ferraro, Caterina; Siu, Kam; Neklesa, Taavi K.; Winkler, James D.; Coleman, Kevin G.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">583-598</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Proteolysis Targeting Chimeras (PROTACs) are bifunctional mols. that recruit an E3 ligase to a target protein to facilitate ubiquitination and subsequent degrdn. of that protein.  While the field of targeted degraders is still relatively young, the potential for this modality to become a differentiated and therapeutic reality is strong, such that both academic and pharmaceutical institutions are now entering this interesting area of research.  In this paper, the authors describe a broadly applicable process for identifying degrader hits based around the serine/threonine kinase TANK-binding kinase 1 (TBK1), and have generalized the key structural elements assocd. with degrdn. activities.  Compd. I is a potent hit (TBK1 DC50 = 12 nM, Dmax = 96%) with excellent selectivity against a related kinase IKKε, which was further used as a chem. tool to assess TBK1 as a target in mutant K-Ras cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoIJWCnFncQRLVg90H21EOLACvtfcHk0liuXY6EvmPLTA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtFelu7rK&md5=41c3627e05ca31aee3ba85815685fcb2</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00635&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00635%26sid%3Dliteratum%253Aachs%26aulast%3DCrew%26aufirst%3DA.%2BP.%26aulast%3DRaina%26aufirst%3DK.%26aulast%3DDong%26aufirst%3DH.%26aulast%3DQian%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DVigil%26aufirst%3DD.%26aulast%3DSerebrenik%26aufirst%3DY.%2BV.%26aulast%3DHamman%26aufirst%3DB.%2BD.%26aulast%3DMorgan%26aufirst%3DA.%26aulast%3DFerraro%26aufirst%3DC.%26aulast%3DSiu%26aufirst%3DK.%26aulast%3DNeklesa%26aufirst%3DT.%2BK.%26aulast%3DWinkler%26aufirst%3DJ.%2BD.%26aulast%3DColeman%26aufirst%3DK.%2BG.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DIdentification%2520and%2520characterization%2520of%2520von%2520Hippel-Lindau-recruiting%2520proteolysis%2520targeting%2520chimeras%2520%2528PROTACs%2529%2520of%2520TANK-binding%2520kinase%25201%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D583%26epage%3D598%26doi%3D10.1021%2Facs.jmedchem.7b00635" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hwang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, Y.</span></span> <span> </span><span class="NLM_article-title">PROTAC-induced BTK degradation as a novel therapy for mutated BTK C481S induced ibrutinib-resistant B-cell malignancies</span>. <i>Cell Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">779</span>– <span class="NLM_lpage">781</span>, <span class="refDoi"> DOI: 10.1038/s41422-018-0055-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=10.1038%2Fs41422-018-0055-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=29875397" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1ektbzO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2018&pages=779-781&author=Y.+Sunauthor=X.+Zhaoauthor=N.+Dingauthor=H.+Gaoauthor=Y.+Wuauthor=Y.+Yangauthor=M.+Zhaoauthor=J.+Hwangauthor=Y.+Songauthor=W.+Liuauthor=Y.+Rao&title=PROTAC-induced+BTK+degradation+as+a+novel+therapy+for+mutated+BTK+C481S+induced+ibrutinib-resistant+B-cell+malignancies&doi=10.1038%2Fs41422-018-0055-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">PROTAC-induced BTK degradation as a novel therapy for mutated BTK C481S induced ibrutinib-resistant B-cell malignancies</span></div><div class="casAuthors">Sun, Yonghui; Zhao, Xingwang; Ding, Ning; Gao, Hongying; Wu, Yue; Yang, Yiqing; Zhao, Meng; Hwang, Jinseok; Song, Yuqin; Liu, Wanli; Rao, Yu</div><div class="citationInfo"><span class="NLM_cas:title">Cell Research</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">779-781</span>CODEN:
                <span class="NLM_cas:coden">CREEB6</span>;
        ISSN:<span class="NLM_cas:issn">1001-0602</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">In this study, for the first time, we report the development of BTK-targeting degraders using the PROTAC strategy.  These PROTACs could ef fi ciently degrade ibrutinib-sensitive BTK-WT (wild type).  More importantly, our newly designed PROTACs also significantly induced the degrdn. of ibrutinib resistant BTK-C481S (50% degrdn. efficiency at 30 nM).  Furthermore, our PROTAC mols. ef fi ciently inhibited cell proliferation and colony formation, while exhibited no obvious inhibition (>1000 nM) of ITK, EGFR, and TEC, which are major off-targets of ibrutinib.  These data demonstrate the strong potential for developing PROTAC-based therapeutic mols.  The results indicate that the PROTAC strategy (protein degrdn. instead of inhibition) could be adopted as a general and powerful therapeutic treatment for drug-resistant cancers in the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqg_FLdaUP_HLVg90H21EOLACvtfcHk0liuXY6EvmPLTA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1ektbzO&md5=1cca51ac76ecb3b491da0117b21271fb</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1038%2Fs41422-018-0055-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41422-018-0055-1%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DY.%26aulast%3DZhao%26aufirst%3DX.%26aulast%3DDing%26aufirst%3DN.%26aulast%3DGao%26aufirst%3DH.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DZhao%26aufirst%3DM.%26aulast%3DHwang%26aufirst%3DJ.%26aulast%3DSong%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DW.%26aulast%3DRao%26aufirst%3DY.%26atitle%3DPROTAC-induced%2520BTK%2520degradation%2520as%2520a%2520novel%2520therapy%2520for%2520mutated%2520BTK%2520C481S%2520induced%2520ibrutinib-resistant%2520B-cell%2520malignancies%26jtitle%3DCell%2520Res.%26date%3D2018%26volume%3D28%26spage%3D779%26epage%3D781%26doi%3D10.1038%2Fs41422-018-0055-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leisten, E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, W.</span></span> <span> </span><span class="NLM_article-title">Development of the first small molecule histone deacetylase 6 (HDAC6) degraders</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">2493</span>– <span class="NLM_lpage">2497</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2018.05.057</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=10.1016%2Fj.bmcl.2018.05.057" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=29871848" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtVOlsrfF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2018&pages=2493-2497&author=K.+Yangauthor=Y.+Songauthor=H.+Xieauthor=H.+Wuauthor=Y.+T.+Wuauthor=E.+D.+Leistenauthor=W.+Tang&title=Development+of+the+first+small+molecule+histone+deacetylase+6+%28HDAC6%29+degraders&doi=10.1016%2Fj.bmcl.2018.05.057"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Development of the first small molecule histone deacetylase 6 (HDAC6) degraders</span></div><div class="casAuthors">Yang, Ka; Song, Yanling; Xie, Haibo; Wu, Hao; Wu, Yi-Ting; Leisten, Eric D.; Tang, Weiping</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">2493-2497</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Histone deacetylases (HDACs) decrease the acetylation level of histones and other non-histone proteins.  Over expression of HDACs have been obsd. in cancers and other diseases.  Targeted protein degrdn. by "hijacking" the natural ubiquitin-proteasome-system (UPS) recently emerged as a novel technol. to "knock-out" endogenous disease-causing proteins.  The authors applied this strategy to the development of the first small mol. degraders for zinc-dependent HDACs by conjugating non-selective HDAC inhibitors with E3 ubiquitin ligase ligands.  Through cell-based assays, the authors discovered novel bifunctional mols. (dHDAC6) that could selectively degrade HDAC6.  Further mechanistic studies indicated that HDAC6 was selectively removed by the UPS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9t1stJ7Qkf7Vg90H21EOLACvtfcHk0ljSgLhoAsyM0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtVOlsrfF&md5=20be52b05a327184f824ccf9cf88333f</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2018.05.057&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2018.05.057%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DK.%26aulast%3DSong%26aufirst%3DY.%26aulast%3DXie%26aufirst%3DH.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DWu%26aufirst%3DY.%2BT.%26aulast%3DLeisten%26aufirst%3DE.%2BD.%26aulast%3DTang%26aufirst%3DW.%26atitle%3DDevelopment%2520of%2520the%2520first%2520small%2520molecule%2520histone%2520deacetylase%25206%2520%2528HDAC6%2529%2520degraders%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2018%26volume%3D28%26spage%3D2493%26epage%3D2497%26doi%3D10.1016%2Fj.bmcl.2018.05.057" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ohoka, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okuhira, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ito, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagai, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shibata, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hattori, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ujikawa, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shimokawa, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sano, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koyama, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujita, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teratani, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsumoto, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imaeda, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nara, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naito, M.</span></span> <span> </span><span class="NLM_article-title">In vivo knockdown of pathogenic proteins via specific and nongenetic inhibitor of apoptosis protein (IAP)-dependent protein erasers (SNIPERs)</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>292</i></span>,  <span class="NLM_fpage">4556</span>– <span class="NLM_lpage">4570</span>, <span class="refDoi"> DOI: 10.1074/jbc.M116.768853</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=10.1074%2Fjbc.M116.768853" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=28154167" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A528%3ADC%252BC2sXksVClur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=292&publication_year=2017&pages=4556-4570&author=N.+Ohokaauthor=K.+Okuhiraauthor=M.+Itoauthor=K.+Nagaiauthor=N.+Shibataauthor=T.+Hattoriauthor=O.+Ujikawaauthor=K.+Shimokawaauthor=O.+Sanoauthor=R.+Koyamaauthor=H.+Fujitaauthor=M.+Terataniauthor=H.+Matsumotoauthor=Y.+Imaedaauthor=H.+Naraauthor=N.+Choauthor=M.+Naito&title=In+vivo+knockdown+of+pathogenic+proteins+via+specific+and+nongenetic+inhibitor+of+apoptosis+protein+%28IAP%29-dependent+protein+erasers+%28SNIPERs%29&doi=10.1074%2Fjbc.M116.768853"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">In Vivo Knockdown of Pathogenic Proteins via Specific and Nongenetic Inhibitor of Apoptosis Protein (IAP)-dependent Protein Erasers (SNIPERs)</span></div><div class="casAuthors">Ohoka, Nobumichi; Okuhira, Keiichiro; Ito, Masahiro; Nagai, Katsunori; Shibata, Norihito; Hattori, Takayuki; Ujikawa, Osamu; Shimokawa, Kenichiro; Sano, Osamu; Koyama, Ryokichi; Fujita, Hisashi; Teratani, Mika; Matsumoto, Hirokazu; Imaeda, Yasuhiro; Nara, Hiroshi; Cho, Nobuo; Naito, Mikihiko</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">292</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4556-4570</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Many diseases, esp. cancers, result from aberrant or overexpression of pathogenic proteins.  Specific inhibitors against these proteins have shown remarkable therapeutic effects, but these are limited mainly to enzymes.  An alternative approach that may have utility in drug development relies on selective degrdn. of pathogenic proteins via small chimeric mols. linking an E3 ubiquitin ligase to the targeted protein for proteasomal degrdn.  To this end, the authors recently developed a protein knockdown system based on hybrid small mol. SNIPERs (Specific and Nongenetic IAP-dependent Protein Erasers) that recruit inhibitor of the apoptosis protein (IAP) ubiquitin ligases to specifically degrade targeted proteins.  Here, the authors extend the authors' previous study to show a proof of concept of the SNIPER technol. in vivo.  By incorporating a high affinity IAP ligand, the authors developed a novel SNIPER against estrogen receptor α (ERα), SNIPER(ER)-87, that has a potent protein knockdown activity.  The SNIPER(ER) reduced ERα levels in tumor xenografts and suppressed the growth of ERα-pos. breast tumors in mice.  Mechanistically, it preferentially recruits X-linked IAP (XIAP) rather than cellular IAP1, to degrade ERα via the ubiquitin-proteasome pathway.  With this IAP ligand, potent SNIPERs against other pathogenic proteins, BCR-ABL, bromodomain-contg. protein 4 (BRD4), and phosphodiesterase-4 (PDE4) could also be developed.  These results indicate that forced ubiquitylation by SNIPERs is a useful method to achieve efficient protein knockdown with potential therapeutic activities and could also be applied to study the role of ubiquitylation in many cellular processes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqEzH5uNeAFabVg90H21EOLACvtfcHk0ljSgLhoAsyM0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXksVClur8%253D&md5=b09264d53c47c54343ed92d5a6d0eb79</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M116.768853&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M116.768853%26sid%3Dliteratum%253Aachs%26aulast%3DOhoka%26aufirst%3DN.%26aulast%3DOkuhira%26aufirst%3DK.%26aulast%3DIto%26aufirst%3DM.%26aulast%3DNagai%26aufirst%3DK.%26aulast%3DShibata%26aufirst%3DN.%26aulast%3DHattori%26aufirst%3DT.%26aulast%3DUjikawa%26aufirst%3DO.%26aulast%3DShimokawa%26aufirst%3DK.%26aulast%3DSano%26aufirst%3DO.%26aulast%3DKoyama%26aufirst%3DR.%26aulast%3DFujita%26aufirst%3DH.%26aulast%3DTeratani%26aufirst%3DM.%26aulast%3DMatsumoto%26aufirst%3DH.%26aulast%3DImaeda%26aufirst%3DY.%26aulast%3DNara%26aufirst%3DH.%26aulast%3DCho%26aufirst%3DN.%26aulast%3DNaito%26aufirst%3DM.%26atitle%3DIn%2520vivo%2520knockdown%2520of%2520pathogenic%2520proteins%2520via%2520specific%2520and%2520nongenetic%2520inhibitor%2520of%2520apoptosis%2520protein%2520%2528IAP%2529-dependent%2520protein%2520erasers%2520%2528SNIPERs%2529%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2017%26volume%3D292%26spage%3D4556%26epage%3D4570%26doi%3D10.1074%2Fjbc.M116.768853" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ohoka, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morita, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagai, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shimokawa, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ujikawa, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujimori, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ito, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayase, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okuhira, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shibata, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hattori, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sameshima, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sano, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koyama, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imaeda, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nara, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naito, M.</span></span> <span> </span><span class="NLM_article-title">Derivatization of inhibitor of apoptosis protein (IAP) ligands yields improved inducers of estrogen receptor α degradation</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>293</i></span>,  <span class="NLM_fpage">6776</span>– <span class="NLM_lpage">6790</span>, <span class="refDoi"> DOI: 10.1074/jbc.RA117.001091</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=10.1074%2Fjbc.RA117.001091" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=293&publication_year=2018&pages=6776-6790&author=N.+Ohokaauthor=Y.+Moritaauthor=K.+Nagaiauthor=K.+Shimokawaauthor=O.+Ujikawaauthor=I.+Fujimoriauthor=M.+Itoauthor=Y.+Hayaseauthor=K.+Okuhiraauthor=N.+Shibataauthor=T.+Hattoriauthor=T.+Sameshimaauthor=O.+Sanoauthor=R.+Koyamaauthor=Y.+Imaedaauthor=H.+Naraauthor=N.+Choauthor=M.+Naito&title=Derivatization+of+inhibitor+of+apoptosis+protein+%28IAP%29+ligands+yields+improved+inducers+of+estrogen+receptor+%CE%B1+degradation&doi=10.1074%2Fjbc.RA117.001091"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1074%2Fjbc.RA117.001091&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.RA117.001091%26sid%3Dliteratum%253Aachs%26aulast%3DOhoka%26aufirst%3DN.%26aulast%3DMorita%26aufirst%3DY.%26aulast%3DNagai%26aufirst%3DK.%26aulast%3DShimokawa%26aufirst%3DK.%26aulast%3DUjikawa%26aufirst%3DO.%26aulast%3DFujimori%26aufirst%3DI.%26aulast%3DIto%26aufirst%3DM.%26aulast%3DHayase%26aufirst%3DY.%26aulast%3DOkuhira%26aufirst%3DK.%26aulast%3DShibata%26aufirst%3DN.%26aulast%3DHattori%26aufirst%3DT.%26aulast%3DSameshima%26aufirst%3DT.%26aulast%3DSano%26aufirst%3DO.%26aulast%3DKoyama%26aufirst%3DR.%26aulast%3DImaeda%26aufirst%3DY.%26aulast%3DNara%26aufirst%3DH.%26aulast%3DCho%26aufirst%3DN.%26aulast%3DNaito%26aufirst%3DM.%26atitle%3DDerivatization%2520of%2520inhibitor%2520of%2520apoptosis%2520protein%2520%2528IAP%2529%2520ligands%2520yields%2520improved%2520inducers%2520of%2520estrogen%2520receptor%2520%25CE%25B1%2520degradation%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2018%26volume%3D293%26spage%3D6776%26epage%3D6790%26doi%3D10.1074%2Fjbc.RA117.001091" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ottis, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toure, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cromm, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ko, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gustafson, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Assessing different E3 ligases for small molecule induced protein ubiquitination and degradation</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">2570</span>– <span class="NLM_lpage">2578</span>, <span class="refDoi"> DOI: 10.1021/acschembio.7b00485</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.7b00485" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1Kisr3O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=2570-2578&author=P.+Ottisauthor=M.+Toureauthor=P.+M.+Crommauthor=E.+Koauthor=J.+L.+Gustafsonauthor=C.+M.+Crews&title=Assessing+different+E3+ligases+for+small+molecule+induced+protein+ubiquitination+and+degradation&doi=10.1021%2Facschembio.7b00485"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Assessing Different E3 Ligases for Small Molecule Induced Protein Ubiquitination and Degradation</span></div><div class="casAuthors">Ottis, Philipp; Toure, Momar; Cromm, Philipp M.; Ko, Eunhwa; Gustafson, Jeffrey L.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2570-2578</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Proteolysis Targeting Chimera (PROTAC) technol., the recruitment of E3 ubiquitin ligases to induce the degrdn. of a protein target, is rapidly impacting chem. biol., as well as modern drug development.  Here, we explore the breadth of this approach by evaluating different E3 ubiquitin ligases engineered in their substrate binding domains to accept a recruiting ligand.  Five out of six E3 ligases were found to be amenable to recruitment for target degrdn.  Taking advantage of the tight spatio-temporal control of inducing ubiquitination on a pre-selected target in living cells, we focused on two of the engineered E3 ligases, βTRCP and parkin, to characterize their ability to induce ubiquitination in comparison with the PROTAC-recruited endogenous E3 ligases VHL and cereblon.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZhBU1NFnjNbVg90H21EOLACvtfcHk0lghlcHAPsgocg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1Kisr3O&md5=b643d34cb33b330a7658b8b7feb2c6cc</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Facschembio.7b00485&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.7b00485%26sid%3Dliteratum%253Aachs%26aulast%3DOttis%26aufirst%3DP.%26aulast%3DToure%26aufirst%3DM.%26aulast%3DCromm%26aufirst%3DP.%2BM.%26aulast%3DKo%26aufirst%3DE.%26aulast%3DGustafson%26aufirst%3DJ.%2BL.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DAssessing%2520different%2520E3%2520ligases%2520for%2520small%2520molecule%2520induced%2520protein%2520ubiquitination%2520and%2520degradation%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2017%26volume%3D12%26spage%3D2570%26epage%3D2578%26doi%3D10.1021%2Facschembio.7b00485" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Raina, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altieri, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossi, A. M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crew, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, K. G.</span></span> <span> </span><span class="NLM_article-title">PROTAC-induced BET protein degradation as a therapy for castration-resistant prostate cancer</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>113</i></span>,  <span class="NLM_fpage">7124</span>– <span class="NLM_lpage">7129</span>, <span class="refDoi"> DOI: 10.1073/pnas.1521738113</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=10.1073%2Fpnas.1521738113" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=27274052" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A528%3ADC%252BC28Xptlyqs7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2016&pages=7124-7129&author=K.+Rainaauthor=J.+Luauthor=Y.+Qianauthor=M.+Altieriauthor=D.+Gordonauthor=A.+M.+K.+Rossiauthor=J.+Wangauthor=X.+Chenauthor=H.+Dongauthor=K.+Siuauthor=J.+D.+Winklerauthor=A.+P.+Crewauthor=C.+M.+Crewsauthor=K.+G.+Coleman&title=PROTAC-induced+BET+protein+degradation+as+a+therapy+for+castration-resistant+prostate+cancer&doi=10.1073%2Fpnas.1521738113"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">PROTAC-induced BET protein degradation as a therapy for castration-resistant prostate cancer</span></div><div class="casAuthors">Raina, Kanak; Lu, Jing; Qian, Yimin; Altieri, Martha; Gordon, Deborah; Rossi, Ann Marie K.; Wang, Jing; Chen, Xin; Dong, Hanqing; Siu, Kam; Winkler, James D.; Crew, Andrew P.; Crews, Craig M.; Coleman, Kevin G.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">7124-7129</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Prostate cancer has the second highest incidence among cancers in men worldwide and is the second leading cause of cancer deaths of men in the United States.  Although androgen deprivation can initially lead to remission, the disease often progresses to castration-resistant prostate cancer (CRPC), which is still reliant on androgen receptor (AR) signaling and is assocd. with a poor prognosis.  Some success against CRPC has been achieved by drugs that target AR signaling, but secondary resistance invariably emerges, and new therapies are urgently needed.  Recently, inhibitors of bromodomain and extra-terminal (BET) family proteins have shown growth-inhibitory activity in preclin. models of CRPC.  Here, we demonstrate that ARV-771, a small-mol. pan-BET degrader based on proteolysis-targeting chimera (PROTAC) technol., demonstrates dramatically improved efficacy in cellular models of CRPC as compared with BET inhibition.  Unlike BET inhibitors, ARV-771 results in suppression of both AR signaling and AR levels and leads to tumor regression in a CRPC mouse xenograft model.  This study is, to our knowledge, the first to demonstrate efficacy with a small-mol. BET degrader in a solid-tumor malignancy and potentially represents an important therapeutic advance in the treatment of CRPC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrD7QtYgZVig7Vg90H21EOLACvtfcHk0lghlcHAPsgocg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xptlyqs7s%253D&md5=34218e685ad72950f3eb85586110c43d</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1521738113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1521738113%26sid%3Dliteratum%253Aachs%26aulast%3DRaina%26aufirst%3DK.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DQian%26aufirst%3DY.%26aulast%3DAltieri%26aufirst%3DM.%26aulast%3DGordon%26aufirst%3DD.%26aulast%3DRossi%26aufirst%3DA.%2BM.%2BK.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DDong%26aufirst%3DH.%26aulast%3DSiu%26aufirst%3DK.%26aulast%3DWinkler%26aufirst%3DJ.%2BD.%26aulast%3DCrew%26aufirst%3DA.%2BP.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26aulast%3DColeman%26aufirst%3DK.%2BG.%26atitle%3DPROTAC-induced%2520BET%2520protein%2520degradation%2520as%2520a%2520therapy%2520for%2520castration-resistant%2520prostate%2520cancer%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2016%26volume%3D113%26spage%3D7124%26epage%3D7129%26doi%3D10.1073%2Fpnas.1521738113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandez-Salas, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McEachern, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Przybranowski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of a small-molecule degrader of bromodomain and extra-terminal (BET) proteins with picomolar cellular potencies and capable of achieving tumor regression</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">462</span>– <span class="NLM_lpage">481</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01816</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01816" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkvVajsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=462-481&author=B.+Zhouauthor=J.+Huauthor=F.+Xuauthor=Z.+Chenauthor=L.+Baiauthor=E.+Fernandez-Salasauthor=M.+Linauthor=L.+Liuauthor=C.+Y.+Yangauthor=Y.+Zhaoauthor=D.+McEachernauthor=S.+Przybranowskiauthor=B.+Wenauthor=D.+Sunauthor=S.+Wang&title=Discovery+of+a+small-molecule+degrader+of+bromodomain+and+extra-terminal+%28BET%29+proteins+with+picomolar+cellular+potencies+and+capable+of+achieving+tumor+regression&doi=10.1021%2Facs.jmedchem.6b01816"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Small-Molecule Degrader of Bromodomain and Extra-Terminal (BET) Proteins with Picomolar Cellular Potencies and Capable of Achieving Tumor Regression</span></div><div class="casAuthors">Zhou, Bing; Hu, Jiantao; Xu, Fuming; Chen, Zhuo; Bai, Longchuan; Fernandez-Salas, Ester; Lin, Mei; Liu, Liu; Yang, Chao-Yie; Zhao, Yujun; McEachern, Donna; Przybranowski, Sally; Wen, Bo; Sun, Duxin; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">462-481</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The bromodomain and extra-terminal (BET) family proteins, consisting of BRD2, BRD3, BRD4, and testis-specific BRDT members, are epigenetic "readers" and play a key role in the regulation of gene transcription.  BET proteins are considered to be attractive therapeutic targets for cancer and other human diseases.  Recently, heterobifunctional small-mol. BET degraders have been designed based upon the proteolysis targeting chimera (PROTAC) concept to induce BET protein degrdn.  Herein, we present our design, synthesis, and evaluation of a new class of PROTAC BET degraders.  One of the most promising compds., 23, effectively degrades BRD4 protein at concns. as low as 30 pM in the RS4;11 leukemia cell line, achieves an IC50 value of 51 pM in inhibition of RS4;11 cell growth and induces rapid tumor regression in vivo against RS4;11 xenograft tumors.  These data establish that compd. 23 (BETd-260/ZBC260) is a highly potent and efficacious BET degrader.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqz2Lwp52x1rVg90H21EOLACvtfcHk0lghlcHAPsgocg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkvVajsb8%253D&md5=6b3a10ecd192a5812ef581dca1fdc82a</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01816&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01816%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DB.%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DF.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DBai%26aufirst%3DL.%26aulast%3DFernandez-Salas%26aufirst%3DE.%26aulast%3DLin%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DYang%26aufirst%3DC.%2BY.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DMcEachern%26aufirst%3DD.%26aulast%3DPrzybranowski%26aufirst%3DS.%26aulast%3DWen%26aufirst%3DB.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520a%2520small-molecule%2520degrader%2520of%2520bromodomain%2520and%2520extra-terminal%2520%2528BET%2529%2520proteins%2520with%2520picomolar%2520cellular%2520potencies%2520and%2520capable%2520of%2520achieving%2520tumor%2520regression%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D462%26epage%3D481%26doi%3D10.1021%2Facs.jmedchem.6b01816" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brzozowski, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pike, A. C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dauter, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubbard, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonn, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engstrom, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohman, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greene, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gustafsson, J.-Å.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlquist, M.</span></span> <span> </span><span class="NLM_article-title">Molecular basis of agonism and antagonism in the oestrogen receptor</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>389</i></span>,  <span class="NLM_fpage">753</span>– <span class="NLM_lpage">758</span>, <span class="refDoi"> DOI: 10.1038/39645</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=10.1038%2F39645" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=9338790" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A528%3ADyaK2sXmvVSnt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=389&publication_year=1997&pages=753-758&author=A.+M.+Brzozowskiauthor=A.+C.+W.+Pikeauthor=Z.+Dauterauthor=R.+E.+Hubbardauthor=T.+Bonnauthor=O.+Engstromauthor=L.+Ohmanauthor=G.+L.+Greeneauthor=J.-%C3%85.+Gustafssonauthor=M.+Carlquist&title=Molecular+basis+of+agonism+and+antagonism+in+the+oestrogen+receptor&doi=10.1038%2F39645"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular basis of agonism and antagonism in the estrogen receptor</span></div><div class="casAuthors">Brzozowski, Andrzej M.; Pike, Ashley C. W.; Dauter, Zbigniew; Hubbard, Roderick E.; Bonn, Tomas; Engstrom, Owe; Ohman, Lars; Greene, Geoffrey L.; Gustafsson, Jan-Ake; Carlquist, Mats</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London)</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">389</span>
        (<span class="NLM_cas:issue">6652</span>),
    <span class="NLM_cas:pages">753-758</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Macmillan Magazines</span>)
        </div><div class="casAbstract">Estrogens are involved in the growth, development and homeostasis of a no. of tissues.  The physiol. effects of these steroids are mediated by a ligand-inducible nuclear transcription factor, the estrogen receptor (ER).  Hormone binding to the ligand-binding domain (LBD) of the ER initiates a series of mol. events culminating in the activation or repression of target genes.  Transcriptional regulation arises from the direct interaction of the ER with components of the cellular transcription machinery.  Here we report the crystal structures of the LBD of ER in complex with the endogenous estrogen, 17β-estradiol, and the selective antagonist raloxifene, at resolns. of 3.1 and 2.6Å, resp.  The structures provide a mol. basis for the distinctive pharmacophore of the ER and its catholic binding properties.  Agonist and antagonist bind at the same site within the core of the LBD but demonstrate different binding modes.  In addn., each class of ligand induces a distinct conformation in the transactivation domain of the LBD, providing structural evidence of the mechanism of antagonism.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAXbSo66V1LrVg90H21EOLACvtfcHk0lguOS15tFMHpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXmvVSnt74%253D&md5=2a63951b881b329ff142c9fbc82a13e3</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1038%2F39645&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F39645%26sid%3Dliteratum%253Aachs%26aulast%3DBrzozowski%26aufirst%3DA.%2BM.%26aulast%3DPike%26aufirst%3DA.%2BC.%2BW.%26aulast%3DDauter%26aufirst%3DZ.%26aulast%3DHubbard%26aufirst%3DR.%2BE.%26aulast%3DBonn%26aufirst%3DT.%26aulast%3DEngstrom%26aufirst%3DO.%26aulast%3DOhman%26aufirst%3DL.%26aulast%3DGreene%26aufirst%3DG.%2BL.%26aulast%3DGustafsson%26aufirst%3DJ.-%25C3%2585.%26aulast%3DCarlquist%26aufirst%3DM.%26atitle%3DMolecular%2520basis%2520of%2520agonism%2520and%2520antagonism%2520in%2520the%2520oestrogen%2520receptor%26jtitle%3DNature%26date%3D1997%26volume%3D389%26spage%3D753%26epage%3D758%26doi%3D10.1038%2F39645" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Qin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandez-Salas, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McEachern, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Przybranowski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meagher, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ziazadeh, D. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of QCA570 as an exceptionally potent and efficacious proteolysis targeting chimera (PROTAC) degrader of the bromodomain and extra-terminal (BET) proteins capable of inducing complete and durable tumor regression</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">6685</span>– <span class="NLM_lpage">6704</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00506</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00506" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtlaqsLrE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=6685-6704&author=C.+Qinauthor=Y.+Huauthor=B.+Zhouauthor=E.+Fernandez-Salasauthor=C.+Y.+Yangauthor=L.+Liuauthor=D.+McEachernauthor=S.+Przybranowskiauthor=M.+Wangauthor=J.+Stuckeyauthor=J.+Meagherauthor=L.+Baiauthor=Z.+Chenauthor=M.+Linauthor=J.+Yangauthor=D.+N.+Ziazadehauthor=F.+Xuauthor=J.+Huauthor=W.+Xiangauthor=L.+Huangauthor=S.+Liauthor=B.+Wenauthor=D.+Sunauthor=S.+Wang&title=Discovery+of+QCA570+as+an+exceptionally+potent+and+efficacious+proteolysis+targeting+chimera+%28PROTAC%29+degrader+of+the+bromodomain+and+extra-terminal+%28BET%29+proteins+capable+of+inducing+complete+and+durable+tumor+regression&doi=10.1021%2Facs.jmedchem.8b00506"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of QCA570 as an Exceptionally Potent and Efficacious Proteolysis Targeting Chimera (PROTAC) Degrader of the Bromodomain and Extra-Terminal (BET) Proteins Capable of Inducing Complete and Durable Tumor Regression</span></div><div class="casAuthors">Qin, Chong; Hu, Yang; Zhou, Bing; Fernandez-Salas, Ester; Yang, Chao-Yie; Liu, Liu; McEachern, Donna; Przybranowski, Sally; Wang, Mi; Stuckey, Jeanne; Meagher, Jennifer; Bai, Longchuan; Chen, Zhuo; Lin, Mei; Yang, Jiuling; Ziazadeh, Danya N.; Xu, Fuming; Hu, Jiantao; Xing, Weiguo; Huang, Liyue; Li, Siwei; Wen, Bo; Sun, Duxin; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">6685-6704</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Proteins of the bromodomain and extra-terminal (BET) family are epigenetics "readers" and promising therapeutic targets for cancer and other human diseases.  We describe herein a structure-guided design of [1,4]oxazepines as a new class of BET inhibitors and our subsequent design, synthesis, and evaluation of proteolysis-targeting chimeric (PROTAC) small-mol. BET degraders.  Our efforts have led to the discovery of extremely potent BET degraders, exemplified by QCA570, which effectively induces degrdn. of BET proteins and inhibits cell growth in human acute leukemia cell lines even at low picomolar concns.  QCA570 achieves complete and durable tumor regression in leukemia xenograft models in mice at well-tolerated dose-schedules.  QCA570 is the most potent and efficacious BET degrader reported to date.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6ZUIsfX3p6LVg90H21EOLACvtfcHk0lguOS15tFMHpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtlaqsLrE&md5=2a731730641372f4526543e03d9ac76f</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00506&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00506%26sid%3Dliteratum%253Aachs%26aulast%3DQin%26aufirst%3DC.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DB.%26aulast%3DFernandez-Salas%26aufirst%3DE.%26aulast%3DYang%26aufirst%3DC.%2BY.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DMcEachern%26aufirst%3DD.%26aulast%3DPrzybranowski%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DStuckey%26aufirst%3DJ.%26aulast%3DMeagher%26aufirst%3DJ.%26aulast%3DBai%26aufirst%3DL.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DLin%26aufirst%3DM.%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DZiazadeh%26aufirst%3DD.%2BN.%26aulast%3DXu%26aufirst%3DF.%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DXiang%26aufirst%3DW.%26aulast%3DHuang%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DWen%26aufirst%3DB.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520QCA570%2520as%2520an%2520exceptionally%2520potent%2520and%2520efficacious%2520proteolysis%2520targeting%2520chimera%2520%2528PROTAC%2529%2520degrader%2520of%2520the%2520bromodomain%2520and%2520extra-terminal%2520%2528BET%2529%2520proteins%2520capable%2520of%2520inducing%2520complete%2520and%2520durable%2520tumor%2520regression%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D6685%26epage%3D6704%26doi%3D10.1021%2Facs.jmedchem.8b00506" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McEachern, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Przybranowski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandez-Salas, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meagher, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayes, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellis, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Targeted degradation of BET proteins in triple-negative breast cancer</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>77</i></span>,  <span class="NLM_fpage">2476</span>– <span class="NLM_lpage">2487</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-16-2622</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=10.1158%2F0008-5472.CAN-16-2622" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=28209615" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmvFakurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=77&publication_year=2017&pages=2476-2487&author=L.+Baiauthor=B.+Zhouauthor=C.+Y.+Yangauthor=J.+Jiauthor=D.+McEachernauthor=S.+Przybranowskiauthor=H.+Jiangauthor=J.+Huauthor=F.+Xuauthor=Y.+Zhaoauthor=L.+Liuauthor=E.+Fernandez-Salasauthor=J.+Xuauthor=Y.+Douauthor=B.+Wenauthor=D.+Sunauthor=J.+Meagherauthor=J.+Stuckeyauthor=D.+F.+Hayesauthor=S.+Liauthor=M.+J.+Ellisauthor=S.+Wang&title=Targeted+degradation+of+BET+proteins+in+triple-negative+breast+cancer&doi=10.1158%2F0008-5472.CAN-16-2622"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted Degradation of BET Proteins in Triple-Negative Breast Cancer</span></div><div class="casAuthors">Bai, Longchuan; Zhou, Bing; Yang, Chao-Yie; Ji, Jiao; McEachern, Donna; Przybranowski, Sally; Jiang, Hui; Hu, Jiantao; Xu, Fuming; Zhao, Yujun; Liu, Liu; Fernandez-Salas, Ester; Xu, Jing; Dou, Yali; Wen, Bo; Sun, Duxin; Meagher, Jennifer; Stuckey, Jeanne; Hayes, Daniel F.; Li, Shunqiang; Ellis, Matthew J.; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">77</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2476-2487</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Triple-neg. breast cancers (TNBC) remain clin. challenging with a lack of options for targeted therapy.  In this study, we report the development of a second-generation BET protein degrader, BETd-246, which exhibits superior selectivity, potency, and antitumor activity.  In human TNBC cells, BETd-246 induced degrdn. of BET proteins at low nanomolar concns. within 1 h of exposure, resulting in robust growth inhibition and apoptosis.  BETd-246 was more potent and effective in TNBC cells than its parental BET inhibitor compd. BETi-211.  RNA-seq anal. revealed predominant downregulation of a large no. of genes involved in proliferation and apoptosis in cells treated with BETd-246, as compared with BETi-211 treatment that upregulated and downregulated a similar no. of genes.  Functional investigations identified the MCL1 gene as a crit. downstream effector for BET degraders, which synergized with small-mol. inhibitors of BCL-xL in triggering apoptosis.  In multiple murine xenograft models of human breast cancer, BETd-246 and a further optimized analog BETd-260 effectively depleted BET proteins in tumors and exhibited strong antitumor activities at well-tolerated dosing schedules.  Overall, our findings show that targeting BET proteins for degrdn. represents an effective therapeutic strategy for TNBC treatment.  Cancer Res; 77(9); 2476-87. ©2017 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8M1VaSNIBCLVg90H21EOLACvtfcHk0lguOS15tFMHpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmvFakurg%253D&md5=84c0e0b23e61a5503f8d88385cc88ae5</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-16-2622&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-16-2622%26sid%3Dliteratum%253Aachs%26aulast%3DBai%26aufirst%3DL.%26aulast%3DZhou%26aufirst%3DB.%26aulast%3DYang%26aufirst%3DC.%2BY.%26aulast%3DJi%26aufirst%3DJ.%26aulast%3DMcEachern%26aufirst%3DD.%26aulast%3DPrzybranowski%26aufirst%3DS.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DF.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DFernandez-Salas%26aufirst%3DE.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DDou%26aufirst%3DY.%26aulast%3DWen%26aufirst%3DB.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DMeagher%26aufirst%3DJ.%26aulast%3DStuckey%26aufirst%3DJ.%26aulast%3DHayes%26aufirst%3DD.%2BF.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DEllis%26aufirst%3DM.%2BJ.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DTargeted%2520degradation%2520of%2520BET%2520proteins%2520in%2520triple-negative%2520breast%2520cancer%26jtitle%3DCancer%2520Res.%26date%3D2017%26volume%3D77%26spage%3D2476%26epage%3D2487%26doi%3D10.1158%2F0008-5472.CAN-16-2622" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Molle, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gareiss, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tae, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noblin, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jorgensen, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Targeting the von Hippel-Lindau E3 ubiquitin ligase using small molecules to disrupt the VHL/HIF-1α interaction</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>134</i></span>,  <span class="NLM_fpage">4465</span>– <span class="NLM_lpage">4468</span>, <span class="refDoi"> DOI: 10.1021/ja209924v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja209924v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A528%3ADC%252BC38XivVOgurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=134&publication_year=2012&pages=4465-4468&author=D.+L.+Buckleyauthor=I.+Van+Molleauthor=P.+C.+Gareissauthor=H.+S.+Taeauthor=J.+Michelauthor=D.+J.+Noblinauthor=W.+L.+Jorgensenauthor=A.+Ciulliauthor=C.+M.+Crews&title=Targeting+the+von+Hippel-Lindau+E3+ubiquitin+ligase+using+small+molecules+to+disrupt+the+VHL%2FHIF-1%CE%B1+interaction&doi=10.1021%2Fja209924v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting the von Hippel-Lindau E3 Ubiquitin Ligase Using Small Molecules To Disrupt the VHL/HIF-1α Interaction</span></div><div class="casAuthors">Buckley, Dennis L.; Van Molle, Inge; Gareiss, Peter C.; Tae, Hyun Seop; Michel, Julien; Noblin, Devin J.; Jorgensen, William L.; Ciulli, Alessio; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">134</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4465-4468</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">E3 ubiquitin ligases, which bind protein targets, leading to their ubiquitination and subsequent degrdn., are attractive drug targets due to their exquisite substrate specificity.  However, the development of small-mol. inhibitors has proven extraordinarily challenging as modulation of E3 ligase activities requires the targeting of protein-protein interactions.  Using rational design, we have generated the first small mol. targeting the von Hippel-Lindau protein (VHL), the substrate recognition subunit of an E3 ligase, and an important target in cancer, chronic anemia, and ischemia.  We have also obtained the crystal structure of VHL bound to our most potent inhibitor, confirming that the compd. mimics the binding mode of the transcription factor HIF-1α, a substrate of VHL.  These results have the potential to guide future development of improved lead compds. as therapeutics for the treatment of chronic anemia and ischemia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGolmBVM_mfWzrVg90H21EOLACvtfcHk0lgDF2L8fjoibQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XivVOgurw%253D&md5=60f7384b6bc932c9828dc12999283a1e</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1021%2Fja209924v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja209924v%26sid%3Dliteratum%253Aachs%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DVan%2BMolle%26aufirst%3DI.%26aulast%3DGareiss%26aufirst%3DP.%2BC.%26aulast%3DTae%26aufirst%3DH.%2BS.%26aulast%3DMichel%26aufirst%3DJ.%26aulast%3DNoblin%26aufirst%3DD.%2BJ.%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26aulast%3DCiulli%26aufirst%3DA.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DTargeting%2520the%2520von%2520Hippel-Lindau%2520E3%2520ubiquitin%2520ligase%2520using%2520small%2520molecules%2520to%2520disrupt%2520the%2520VHL%252FHIF-1%25CE%25B1%2520interaction%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2012%26volume%3D134%26spage%3D4465%26epage%3D4468%26doi%3D10.1021%2Fja209924v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gustafson, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Molle, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tae, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gareiss, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jorgensen, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Small-molecule inhibitors of the interaction between the E3 ligase VHL and HIF-1α</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">11463</span>– <span class="NLM_lpage">11467</span>, <span class="refDoi"> DOI: 10.1002/anie.201206231</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=10.1002%2Fanie.201206231" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2012&pages=11463-11467&author=D.+L.+Buckleyauthor=J.+L.+Gustafsonauthor=I.+Van+Molleauthor=A.+G.+Rothauthor=H.+S.+Taeauthor=P.+C.+Gareissauthor=W.+L.+Jorgensenauthor=A.+Ciulliauthor=C.+M.+Crews&title=Small-molecule+inhibitors+of+the+interaction+between+the+E3+ligase+VHL+and+HIF-1%CE%B1&doi=10.1002%2Fanie.201206231"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1002%2Fanie.201206231&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201206231%26sid%3Dliteratum%253Aachs%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DGustafson%26aufirst%3DJ.%2BL.%26aulast%3DVan%2BMolle%26aufirst%3DI.%26aulast%3DRoth%26aufirst%3DA.%2BG.%26aulast%3DTae%26aufirst%3DH.%2BS.%26aulast%3DGareiss%26aufirst%3DP.%2BC.%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26aulast%3DCiulli%26aufirst%3DA.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DSmall-molecule%2520inhibitors%2520of%2520the%2520interaction%2520between%2520the%2520E3%2520ligase%2520VHL%2520and%2520HIF-1%25CE%25B1%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2012%26volume%3D51%26spage%3D11463%26epage%3D11467%26doi%3D10.1002%2Fanie.201206231" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Galdeano, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gadd, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soares, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scaffidi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Molle, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Birced, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hewitt, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dias, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Structure-guided design and optimization of small molecules targeting the protein-protein interaction between the von Hippel-Lindau (VHL) E3 ubiquitin ligase and the hypoxia inducible factor (HIF) α subunit with in vitro nanomolar affinities</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">8657</span>– <span class="NLM_lpage">8663</span>, <span class="refDoi"> DOI: 10.1021/jm5011258</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm5011258" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=8657-8663&author=C.+Galdeanoauthor=M.+S.+Gaddauthor=P.+Soaresauthor=S.+Scaffidiauthor=I.+Van+Molleauthor=I.+Bircedauthor=S.+Hewittauthor=D.+M.+Diasauthor=A.+Ciulli&title=Structure-guided+design+and+optimization+of+small+molecules+targeting+the+protein-protein+interaction+between+the+von+Hippel-Lindau+%28VHL%29+E3+ubiquitin+ligase+and+the+hypoxia+inducible+factor+%28HIF%29+%CE%B1+subunit+with+in+vitro+nanomolar+affinities&doi=10.1021%2Fjm5011258"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1021%2Fjm5011258&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm5011258%26sid%3Dliteratum%253Aachs%26aulast%3DGaldeano%26aufirst%3DC.%26aulast%3DGadd%26aufirst%3DM.%2BS.%26aulast%3DSoares%26aufirst%3DP.%26aulast%3DScaffidi%26aufirst%3DS.%26aulast%3DVan%2BMolle%26aufirst%3DI.%26aulast%3DBirced%26aufirst%3DI.%26aulast%3DHewitt%26aufirst%3DS.%26aulast%3DDias%26aufirst%3DD.%2BM.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DStructure-guided%2520design%2520and%2520optimization%2520of%2520small%2520molecules%2520targeting%2520the%2520protein-protein%2520interaction%2520between%2520the%2520von%2520Hippel-Lindau%2520%2528VHL%2529%2520E3%2520ubiquitin%2520ligase%2520and%2520the%2520hypoxia%2520inducible%2520factor%2520%2528HIF%2529%2520%25CE%25B1%2520subunit%2520with%2520in%2520vitro%2520nanomolar%2520affinities%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D8657%26epage%3D8663%26doi%3D10.1021%2Fjm5011258" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Soares, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gadd, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frost, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galdeano, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellis, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Epemolu, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rocha, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Group-based optimization of potent and cell-active inhibitors of the von Hippel-Lindau (VHL) E3 ubiquitin ligase: structure-activity relationships leading to the chemical probe (2S,4R)-1-((S)-2-(1-cyanocyclopropanecarboxamido)-3,3-dimethylbutanoyl)-4-hydroxy -N-(4-(4-methylthiazol-5-yl)benzyl)pyrrolidine-2-carboxamide (VH298)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">599</span>– <span class="NLM_lpage">618</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00675</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00675" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=599-618&author=P.+Soaresauthor=M.+S.+Gaddauthor=J.+Frostauthor=C.+Galdeanoauthor=L.+Ellisauthor=O.+Epemoluauthor=S.+Rochaauthor=K.+D.+Readauthor=A.+Ciulli&title=Group-based+optimization+of+potent+and+cell-active+inhibitors+of+the+von+Hippel-Lindau+%28VHL%29+E3+ubiquitin+ligase%3A+structure-activity+relationships+leading+to+the+chemical+probe+%282S%2C4R%29-1-%28%28S%29-2-%281-cyanocyclopropanecarboxamido%29-3%2C3-dimethylbutanoyl%29-4-hydroxy+-N-%284-%284-methylthiazol-5-yl%29benzyl%29pyrrolidine-2-carboxamide+%28VH298%29&doi=10.1021%2Facs.jmedchem.7b00675"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00675&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00675%26sid%3Dliteratum%253Aachs%26aulast%3DSoares%26aufirst%3DP.%26aulast%3DGadd%26aufirst%3DM.%2BS.%26aulast%3DFrost%26aufirst%3DJ.%26aulast%3DGaldeano%26aufirst%3DC.%26aulast%3DEllis%26aufirst%3DL.%26aulast%3DEpemolu%26aufirst%3DO.%26aulast%3DRocha%26aufirst%3DS.%26aulast%3DRead%26aufirst%3DK.%2BD.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DGroup-based%2520optimization%2520of%2520potent%2520and%2520cell-active%2520inhibitors%2520of%2520the%2520von%2520Hippel-Lindau%2520%2528VHL%2529%2520E3%2520ubiquitin%2520ligase%253A%2520structure-activity%2520relationships%2520leading%2520to%2520the%2520chemical%2520probe%2520%25282S%252C4R%2529-1-%2528%2528S%2529-2-%25281-cyanocyclopropanecarboxamido%2529-3%252C3-dimethylbutanoyl%2529-4-hydroxy%2520-N-%25284-%25284-methylthiazol-5-yl%2529benzyl%2529pyrrolidine-2-carboxamide%2520%2528VH298%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D599%26epage%3D618%26doi%3D10.1021%2Facs.jmedchem.7b00675" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gadd, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Testa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucas, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamont, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zengerle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Structural basis of PROTAC cooperative recognition for selective protein degradation</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">514</span>– <span class="NLM_lpage">521</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2329</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=10.1038%2Fnchembio.2329" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=28288108" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkvFahtbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&pages=514-521&author=M.+S.+Gaddauthor=A.+Testaauthor=X.+Lucasauthor=K.+H.+Chanauthor=W.+Chenauthor=D.+J.+Lamontauthor=M.+Zengerleauthor=A.+Ciulli&title=Structural+basis+of+PROTAC+cooperative+recognition+for+selective+protein+degradation&doi=10.1038%2Fnchembio.2329"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis of PROTAC cooperative recognition for selective protein degradation</span></div><div class="casAuthors">Gadd, Morgan S.; Testa, Andrea; Lucas, Xavier; Chan, Kwok-Ho; Chen, Wenzhang; Lamont, Douglas J.; Zengerle, Michael; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">514-521</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Inducing macromol. interactions with small mols. to activate cellular signaling is a challenging goal.  PROTACs (proteolysis-targeting chimeras) are bifunctional mols. that recruit a target protein in proximity to an E3 ubiquitin ligase to trigger protein degrdn.  Structural elucidation of the key ternary ligase-PROTAC-target species and its impact on target degrdn. selectivity remain elusive.  Here, we solved the crystal structure of Brd4 degrader MZ1 in complex with human VHL and the Brd4 bromodomain (Brd4BD2).  The ligand folds into itself to allow formation of specific intermol. interactions in the ternary complex.  Isothermal titrn. calorimetry studies, supported by surface mutagenesis and proximity assays, were consistent with pronounced cooperative formation of ternary complexes with Brd4BD2.  Structure-based-designed compd. AT1 exhibited highly selective depletion of Brd4 in cells.  These results elucidate how PROTAC-induced de novo contacts dictate preferential recruitment of a target protein into a stable and cooperative complex with an E3 ligase for selective degrdn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJri00jB5YJLVg90H21EOLACvtfcHk0lhNLFqY2G64BA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkvFahtbg%253D&md5=dfee13913106d362d0972d7b1a62d700</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2329&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2329%26sid%3Dliteratum%253Aachs%26aulast%3DGadd%26aufirst%3DM.%2BS.%26aulast%3DTesta%26aufirst%3DA.%26aulast%3DLucas%26aufirst%3DX.%26aulast%3DChan%26aufirst%3DK.%2BH.%26aulast%3DChen%26aufirst%3DW.%26aulast%3DLamont%26aufirst%3DD.%2BJ.%26aulast%3DZengerle%26aufirst%3DM.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DStructural%2520basis%2520of%2520PROTAC%2520cooperative%2520recognition%2520for%2520selective%2520protein%2520degradation%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2017%26volume%3D13%26spage%3D514%26epage%3D521%26doi%3D10.1038%2Fnchembio.2329" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Long, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poganik, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aye, Y.</span></span> <span> </span><span class="NLM_article-title">On-demand targeting: investigating biology with proximity-directed chemistry</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>138</i></span>,  <span class="NLM_fpage">3610</span>– <span class="NLM_lpage">3622</span>, <span class="refDoi"> DOI: 10.1021/jacs.5b12608</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.5b12608" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A528%3ADC%252BC28XjtVCis7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=138&publication_year=2016&pages=3610-3622&author=M.+J.+Longauthor=J.+R.+Poganikauthor=Y.+Aye&title=On-demand+targeting%3A+investigating+biology+with+proximity-directed+chemistry&doi=10.1021%2Fjacs.5b12608"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">On-Demand Targeting: Investigating Biology with Proximity-Directed Chemistry</span></div><div class="casAuthors">Long, Marcus J. C.; Poganik, Jesse R.; Aye, Yimon</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">138</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">3610-3622</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Proximity enhancement is a central chem. tenet underpinning an exciting suite of small-mol. toolsets that have allowed us to unravel many biol. complexities.  The leitmotif of this opus is "tethering"-a strategy in which a multifunctional small mol. serves as a template to bring proteins/biomols. together.  Scaffolding approaches have been powerfully applied to control diverse biol. outcomes such as protein-protein assocn., protein stability, activity, and improve imaging capabilities.  A new twist on this strategy has recently appeared, in which the small-mol. probe is engineered to unleash controlled amts. of reactive chem. signals within the microenvironment of a target protein.  Modification of a specific target elicits a precisely timed and spatially controlled gain-of-function (or dominant loss-of-function) signaling response.  Presented herein is a unique personal outlook conceptualizing the powerful proximity-enhanced chem. biol. toolsets into two paradigms: "multifunctional scaffolding" vs. "on-demand targeting".  By addressing the latest advances and challenges in the established yet constantly evolving multifunctional scaffolding strategies as well as in the emerging on-demand precision targeting (and related) systems, this Perspective is aimed at choosing when it is best to employ each of the two strategies, with an emphasis toward further promoting novel applications and discoveries stemming from these innovative chem. biol. platforms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGro6dKG5laUM7Vg90H21EOLACvtfcHk0lhNLFqY2G64BA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XjtVCis7g%253D&md5=40fa3d02e8de215dd7c596aa7a32af78</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1021%2Fjacs.5b12608&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.5b12608%26sid%3Dliteratum%253Aachs%26aulast%3DLong%26aufirst%3DM.%2BJ.%26aulast%3DPoganik%26aufirst%3DJ.%2BR.%26aulast%3DAye%26aufirst%3DY.%26atitle%3DOn-demand%2520targeting%253A%2520investigating%2520biology%2520with%2520proximity-directed%2520chemistry%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2016%26volume%3D138%26spage%3D3610%26epage%3D3622%26doi%3D10.1021%2Fjacs.5b12608" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stols, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dieckman, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raffen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collart, F. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donnelly, M. I.</span></span> <span> </span><span class="NLM_article-title">A new vector for high-throughput, ligation-independent cloning encoding a tobacco etch virus protease cleavage site</span>. <i>Protein Expression Purif.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">8</span>– <span class="NLM_lpage">15</span>, <span class="refDoi"> DOI: 10.1006/prep.2001.1603</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=10.1006%2Fprep.2001.1603" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=12071693" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A528%3ADC%252BD38XksF2gsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2002&pages=8-15&author=L.+Stolsauthor=M.+Guauthor=L.+Dieckmanauthor=R.+Raffenauthor=F.+R.+Collartauthor=M.+I.+Donnelly&title=A+new+vector+for+high-throughput%2C+ligation-independent+cloning+encoding+a+tobacco+etch+virus+protease+cleavage+site&doi=10.1006%2Fprep.2001.1603"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">A New Vector for High-Throughput, Ligation-Independent Cloning Encoding a Tobacco Etch Virus Protease Cleavage Site</span></div><div class="casAuthors">Stols, Lucy; Gu, Minyi; Dieckman, Lynda; Raffen, Rosemarie; Collart, Frank R.; Donnelly, Mark I.</div><div class="citationInfo"><span class="NLM_cas:title">Protein Expression and Purification</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">8-15</span>CODEN:
                <span class="NLM_cas:coden">PEXPEJ</span>;
        ISSN:<span class="NLM_cas:issn">1046-5928</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science</span>)
        </div><div class="casAbstract">To establish high-throughput methods for protein crystallog., all aspects of the prodn. and anal. of protein crystals must be accelerated.  Automated, plate-based methods for cloning, expression, and evaluation of target proteins will help researchers investigate the vast nos. of proteins now available from sequenced genomes.  Ligation-independent cloning (LIC) is well suited to robotic cloning and expression, but few LIC vectors are available com.  The authors have developed a new LIC vector, pMCSG7, that incorporates the tobacco etch virus (TEV) protease cleavage site into the leader sequence.  This protease is highly specific and functions under a wide range of conditions.  The new vector incorporates an N-terminal his-tag followed by the TEV protease recognition site and a SspI restriction site used for LIC.  The vector functioned as expected, giving high cloning efficiencies and strong expression of proteins.  Purifn. and cleavage of a target protein showed that the his-tag and the TEV cleavage site function properly.  The protein was purified and cleaved under different conditions to simulate both plate-based screening methods and large-scale purifications for crystal prodn.  The vector also includes a pair of adjacent, unique restriction sites that will allow insertion of addnl. modules between the his-tag and the cleavage site of the leader sequence to generate a family of vectors suitable for high-throughput prodn. of proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqcWlZD9MM0mbVg90H21EOLACvtfcHk0lhNLFqY2G64BA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XksF2gsb4%253D&md5=a2baff9a27cd5caf32358891635de4f5</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1006%2Fprep.2001.1603&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fprep.2001.1603%26sid%3Dliteratum%253Aachs%26aulast%3DStols%26aufirst%3DL.%26aulast%3DGu%26aufirst%3DM.%26aulast%3DDieckman%26aufirst%3DL.%26aulast%3DRaffen%26aufirst%3DR.%26aulast%3DCollart%26aufirst%3DF.%2BR.%26aulast%3DDonnelly%26aufirst%3DM.%2BI.%26atitle%3DA%2520new%2520vector%2520for%2520high-throughput%252C%2520ligation-independent%2520cloning%2520encoding%2520a%2520tobacco%2520etch%2520virus%2520protease%2520cleavage%2520site%26jtitle%3DProtein%2520Expression%2520Purif.%26date%3D2002%26volume%3D25%26spage%3D8%26epage%3D15%26doi%3D10.1006%2Fprep.2001.1603" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Benoit, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ostermeier, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geiser, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Widmer, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Auer, M.</span></span> <span> </span><span class="NLM_article-title">Seamless insert-plasmid assembly at high efficiency and low cost</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>, <span class="NLM_elocation-id">e0153158</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0153158</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=10.1371%2Fjournal.pone.0153158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=27073895" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFyjs7%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&author=R.+M.+Benoitauthor=C.+Ostermeierauthor=M.+Geiserauthor=J.+S.+Liauthor=H.+Widmerauthor=M.+Auer&title=Seamless+insert-plasmid+assembly+at+high+efficiency+and+low+cost&doi=10.1371%2Fjournal.pone.0153158"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Seamless insert-plasmid assembly at high efficiency and low cost</span></div><div class="casAuthors">Benoit, Roger M.; Ostermeier, Christian; Geiser, Martin; Li, Julia Su Zhou; Widmer, Hans; Auer, Manfred</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">e0153158/1-e0153158/13</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Seamless cloning methods, such as co-transformation cloning, sequence- and ligation-independent cloning (SLIC) or the Gibson assembly, are essential tools for the precise construction of plasmids.  The efficiency of co-transformation cloning is however low and the Gibson assembly reagents are expensive.  With the aim to improve the robustness of seamless cloning expts. while keeping costs low, we examd. the importance of complementary single-stranded DNA ends for co-transformation cloning and the influence of single-stranded gaps in circular plasmids on SLIC cloning efficiency.  Most importantly, our data show that single-stranded gaps in double-stranded plasmids, which occur in typical SLIC protocols, can drastically decrease the efficiency at which the DNA transforms competent E. coli bacteria.  Accordingly, filling-in of single-stranded gaps using DNA polymerase resulted in increased transformation efficiency.  Ligation of the remaining nicks did not lead to a further increase in transformation efficiency.  These findings demonstrate that highly efficient insert-plasmid assembly can be achieved by using only T5 exonuclease and Phusion DNA polymerase, without Taq DNA ligase from the original Gibson protocol, which significantly reduces the cost of the reactions.  We successfully used this modified Gibson assembly protocol with two short insert-plasmid overlap regions, each counting only 15 nucleotides.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWj6JIvwfOybVg90H21EOLACvtfcHk0lhNLFqY2G64BA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFyjs7%252FP&md5=f5978304dfcf58f93be15833fc30a5d6</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0153158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0153158%26sid%3Dliteratum%253Aachs%26aulast%3DBenoit%26aufirst%3DR.%2BM.%26aulast%3DOstermeier%26aufirst%3DC.%26aulast%3DGeiser%26aufirst%3DM.%26aulast%3DLi%26aufirst%3DJ.%2BS.%26aulast%3DWidmer%26aufirst%3DH.%26aulast%3DAuer%26aufirst%3DM.%26atitle%3DSeamless%2520insert-plasmid%2520assembly%2520at%2520high%2520efficiency%2520and%2520low%2520cost%26jtitle%3DPLoS%2520One%26date%3D2016%26volume%3D11%26doi%3D10.1371%2Fjournal.pone.0153158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 66 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Liqiang Fu, Jing Zhang, Bin Shen, Linglong Kong, Yingtao Liu, Wangyang Tu, Wenqian Wang, Xin Cai, Xiaotao Wang, Na Cheng, Mingxuan Xia, Tianyuan Zhou, Qian Liu, Yanping Xu, Jennifer Yang, Paul Gavine, Ulrike Philippar, Ricardo Attar, James P. Edwards, Jennifer D. Venable, <span class="NLM_string-name hlFld-ContribAuthor">Xuedong Dai</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of Highly Potent and Selective IRAK1 Degraders to Probe Scaffolding Functions of IRAK1 in ABC DLBCL. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021</strong>, Article ASAP.</span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00103&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.1c00103%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252BHighly%252BPotent%252Band%252BSelective%252BIRAK1%252BDegraders%252Bto%252BProbe%252BScaffolding%252BFunctions%252Bof%252BIRAK1%252Bin%252BABC%252BDLBCL%26aulast%3DFu%26aufirst%3DLiqiang%26date%3D2021%26date%3D2021%26date%3D20012021%26volume%3D0%26issue%3D0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Jeremy S. Disch, Jennifer M. Duffy, Esther C. Y. Lee, Diana Gikunju, Betty Chan, Benjamin Levin, Michael I. Monteiro, Sarah A. Talcott, Anthony C. Lau, Fei Zhou, Anton Kozhushnyan, Neil E. Westlund, Patrick B. Mullins, Yan Yu, Moritz von Rechenberg, Junyi Zhang, Yelena A. Arnautova, Yanbin Liu, Ying Zhang, Andrew J. McRiner, Anthony D. Keefe, Anna Kohlmann, Matthew A. Clark, John W. Cuozzo, Christelle Huguet, <span class="NLM_string-name hlFld-ContribAuthor">Shilpi Arora</span>. </span><span class="cited-content_cbyCitation_article-title">Bispecific Estrogen Receptor α Degraders Incorporating Novel Binders Identified Using DNA-Encoded Chemical Library Screening. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (8)
                                     , 5049-5066. <a href="https://doi.org/10.1021/acs.jmedchem.1c00127" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.1c00127</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00127&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.1c00127%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DBispecific%252BEstrogen%252BReceptor%252B%2525CE%2525B1%252BDegraders%252BIncorporating%252BNovel%252BBinders%252BIdentified%252BUsing%252BDNA-Encoded%252BChemical%252BLibrary%252BScreening%26aulast%3DDisch%26aufirst%3DJeremy%2BS.%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D22012021%26date%3D12042021%26volume%3D64%26issue%3D8%26spage%3D5049%26epage%3D5066" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Madhusoodanan Mottamal, Borui Kang, Xianyou Peng, <span class="NLM_string-name hlFld-ContribAuthor">Guangdi Wang</span>. </span><span class="cited-content_cbyCitation_article-title">From Pure Antagonists to Pure Degraders of the Estrogen Receptor: Evolving Strategies for the Same Target. </span><span class="cited-content_cbyCitation_journal-name">ACS Omega</span><span> <strong>2021,</strong> <em>6 </em>
                                    (14)
                                     , 9334-9343. <a href="https://doi.org/10.1021/acsomega.0c06362" title="DOI URL">https://doi.org/10.1021/acsomega.0c06362</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsomega.0c06362&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsomega.0c06362%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Omega%26atitle%3DFrom%252BPure%252BAntagonists%252Bto%252BPure%252BDegraders%252Bof%252Bthe%252BEstrogen%252BReceptor%25253A%252BEvolving%252BStrategies%252Bfor%252Bthe%252BSame%252BTarget%26aulast%3DMottamal%26aufirst%3DMadhusoodanan%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D31122020%26date%3D19032021%26date%3D30032021%26volume%3D6%26issue%3D14%26spage%3D9334%26epage%3D9343" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Anh S. Lu, Milad Rouhimoghadam, Christopher K. Arnatt, Edward J. Filardo, <span class="NLM_string-name hlFld-ContribAuthor">Aliasger K. Salem</span>. </span><span class="cited-content_cbyCitation_article-title">Proteolytic Targeting Chimeras with Specificity for Plasma Membrane and Intracellular Estrogen Receptors. </span><span class="cited-content_cbyCitation_journal-name">Molecular Pharmaceutics</span><span> <strong>2021,</strong> <em>18 </em>
                                    (3)
                                     , 1455-1469. <a href="https://doi.org/10.1021/acs.molpharmaceut.1c00018" title="DOI URL">https://doi.org/10.1021/acs.molpharmaceut.1c00018</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.molpharmaceut.1c00018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.molpharmaceut.1c00018%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Pharmaceutics%26atitle%3DProteolytic%252BTargeting%252BChimeras%252Bwith%252BSpecificity%252Bfor%252BPlasma%252BMembrane%252Band%252BIntracellular%252BEstrogen%252BReceptors%26aulast%3DLu%26aufirst%3DAnh%2BS.%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D09012021%26date%3D04022021%26date%3D03022021%26date%3D18022021%26volume%3D18%26issue%3D3%26spage%3D1455%26epage%3D1469" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Yunlong Lu, <span class="NLM_string-name hlFld-ContribAuthor">Wukun Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Selective Estrogen Receptor Degraders (SERDs): A Promising Strategy for Estrogen Receptor Positive Endocrine-Resistant Breast Cancer. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (24)
                                     , 15094-15114. <a href="https://doi.org/10.1021/acs.jmedchem.0c00913" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00913</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00913&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00913%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DSelective%252BEstrogen%252BReceptor%252BDegraders%252B%252528SERDs%252529%25253A%252BA%252BPromising%252BStrategy%252Bfor%252BEstrogen%252BReceptor%252BPositive%252BEndocrine-Resistant%252BBreast%252BCancer%26aulast%3DLu%26aufirst%3DYunlong%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D29052020%26date%3D02112020%26volume%3D63%26issue%3D24%26spage%3D15094%26epage%3D15114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Chaoguo Cao, Jie Yang, Yong Chen, Peiting Zhou, Yingwei Wang, Wu Du, Lifeng Zhao, <span class="NLM_string-name hlFld-ContribAuthor">Yuanwei Chen</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of SK-575 as a Highly Potent and Efficacious Proteolysis-Targeting Chimera Degrader of PARP1 for Treating Cancers. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (19)
                                     , 11012-11033. <a href="https://doi.org/10.1021/acs.jmedchem.0c00821" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00821</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00821&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00821%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252BSK-575%252Bas%252Ba%252BHighly%252BPotent%252Band%252BEfficacious%252BProteolysis-Targeting%252BChimera%252BDegrader%252Bof%252BPARP1%252Bfor%252BTreating%252BCancers%26aulast%3DCao%26aufirst%3DChaoguo%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D01062020%26date%3D24092020%26date%3D14092020%26volume%3D63%26issue%3D19%26spage%3D11012%26epage%3D11033" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Yudao Shen, Guozhen Gao, Xufen Yu, Huensuk Kim, Li Wang, Ling Xie, Megan Schwarz, Xian Chen, Ernesto Guccione, Jing Liu, Mark T. Bedford, <span class="NLM_string-name hlFld-ContribAuthor">Jian Jin</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of First-in-Class Protein Arginine Methyltransferase 5 (PRMT5) Degraders. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (17)
                                     , 9977-9989. <a href="https://doi.org/10.1021/acs.jmedchem.0c01111" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01111</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01111%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252BFirst-in-Class%252BProtein%252BArginine%252BMethyltransferase%252B5%252B%252528PRMT5%252529%252BDegraders%26aulast%3DShen%26aufirst%3DYudao%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D29062020%26date%3D21082020%26date%3D11082020%26volume%3D63%26issue%3D17%26spage%3D9977%26epage%3D9989" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Mingliang Wang, Jianfeng Lu, Mi Wang, Chao-Yie Yang, <span class="NLM_string-name hlFld-ContribAuthor">Shaomeng Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of SHP2-D26 as a First, Potent, and Effective PROTAC Degrader of SHP2 Protein. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (14)
                                     , 7510-7528. <a href="https://doi.org/10.1021/acs.jmedchem.0c00471" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00471</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00471&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00471%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252BSHP2-D26%252Bas%252Ba%252BFirst%25252C%252BPotent%25252C%252Band%252BEffective%252BPROTAC%252BDegrader%252Bof%252BSHP2%252BProtein%26aulast%3DWang%26aufirst%3DMingliang%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D20032020%26date%3D21052020%26volume%3D63%26issue%3D14%26spage%3D7510%26epage%3D7528" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Junfei Cheng, Yu Li, Xu Wang, Guoqiang Dong, <span class="NLM_string-name hlFld-ContribAuthor">Chunquan Sheng</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of Novel PDEδ Degraders for the Treatment of KRAS Mutant Colorectal Cancer. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (14)
                                     , 7892-7905. <a href="https://doi.org/10.1021/acs.jmedchem.0c00929" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00929</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00929&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00929%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252BNovel%252BPDE%2525CE%2525B4%252BDegraders%252Bfor%252Bthe%252BTreatment%252Bof%252BKRAS%252BMutant%252BColorectal%252BCancer%26aulast%3DCheng%26aufirst%3DJunfei%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D01062020%26date%3D14072020%26date%3D30062020%26volume%3D63%26issue%3D14%26spage%3D7892%26epage%3D7905" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Brett L. Roberts, Zhi-Xiong Ma, Ang Gao, Eric D. Leisten, Dan Yin, Wei Xu, <span class="NLM_string-name hlFld-ContribAuthor">Weiping Tang</span>. </span><span class="cited-content_cbyCitation_article-title">Two-Stage Strategy for Development of Proteolysis Targeting Chimeras and its Application for Estrogen Receptor Degraders. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Biology</span><span> <strong>2020,</strong> <em>15 </em>
                                    (6)
                                     , 1487-1496. <a href="https://doi.org/10.1021/acschembio.0c00140" title="DOI URL">https://doi.org/10.1021/acschembio.0c00140</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acschembio.0c00140&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facschembio.0c00140%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Biology%26atitle%3DTwo-Stage%252BStrategy%252Bfor%252BDevelopment%252Bof%252BProteolysis%252BTargeting%252BChimeras%252Band%252Bits%252BApplication%252Bfor%252BEstrogen%252BReceptor%252BDegraders%26aulast%3DRoberts%26aufirst%3DBrett%2BL.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D26022020%26date%3D07042020%26date%3D16042020%26date%3D07042020%26volume%3D15%26issue%3D6%26spage%3D1487%26epage%3D1496" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Marı́a Maneiro, Nafsika Forte, Maria M. Shchepinova, Cyrille S. Kounde, Vijay Chudasama, James Richard Baker, <span class="NLM_string-name hlFld-ContribAuthor">Edward W. Tate</span>. </span><span class="cited-content_cbyCitation_article-title">Antibody–PROTAC Conjugates Enable HER2-Dependent Targeted Protein Degradation of BRD4. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Biology</span><span> <strong>2020,</strong> <em>15 </em>
                                    (6)
                                     , 1306-1312. <a href="https://doi.org/10.1021/acschembio.0c00285" title="DOI URL">https://doi.org/10.1021/acschembio.0c00285</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acschembio.0c00285&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facschembio.0c00285%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Biology%26atitle%3DAntibody%2525E2%252580%252593PROTAC%252BConjugates%252BEnable%252BHER2-Dependent%252BTargeted%252BProtein%252BDegradation%252Bof%252BBRD4%26aulast%3DManeiro%26aufirst%3DMar%25C4%25B1%25CC%2581a%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D15042020%26date%3D27042020%26date%3D30042020%26date%3D27042020%26volume%3D15%26issue%3D6%26spage%3D1306%26epage%3D1312" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Jun Liang, Robert Blake, Jae Chang, Lori S. Friedman, Simon Goodacre, Steven Hartman, Ellen Rei Ingalla, James R. Kiefer, Tracy Kleinheinz, Sharada Labadie, Jun Li, Kwong Wah Lai, Jiangpeng Liao, Vidhi Mody, Neville McLean, Ciara Metcalfe, Michelle Nannini, Daniel Otwine, Yingqing Ran, Nick Ray, Fabien Roussel, Amy Sambrone, Deepak Sampath, Maia Vinogradova, John Wai, Tao Wang, Kuen Yeap, Amy Young, Jason Zbieg, Birong Zhang, Xiaoping Zheng, Yu Zhong, <span class="NLM_string-name hlFld-ContribAuthor">Xiaojing Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of GNE-149 as a Full Antagonist and Efficient Degrader of Estrogen Receptor alpha for ER+ Breast Cancer. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2020,</strong> <em>11 </em>
                                    (6)
                                     , 1342-1347. <a href="https://doi.org/10.1021/acsmedchemlett.0c00224" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.0c00224</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.0c00224&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.0c00224%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DDiscovery%252Bof%252BGNE-149%252Bas%252Ba%252BFull%252BAntagonist%252Band%252BEfficient%252BDegrader%252Bof%252BEstrogen%252BReceptor%252Balpha%252Bfor%252BER%25252B%252BBreast%252BCancer%26aulast%3DLiang%26aufirst%3DJun%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D28042020%26date%3D26052020%26date%3D29052020%26date%3D26052020%26volume%3D11%26issue%3D6%26spage%3D1342%26epage%3D1347" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Mei-Xin Li, Yiqing Yang, Qiuye Zhao, Yue Wu, Lei Song, Haiyan Yang, Ming He, Hongying Gao, Bao-Liang Song, Jie Luo, <span class="NLM_string-name hlFld-ContribAuthor">Yu Rao</span>. </span><span class="cited-content_cbyCitation_article-title">Degradation versus Inhibition: Development of Proteolysis-Targeting Chimeras for Overcoming Statin-Induced Compensatory Upregulation of 3-Hydroxy-3-methylglutaryl Coenzyme A Reductase. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (9)
                                     , 4908-4928. <a href="https://doi.org/10.1021/acs.jmedchem.0c00339" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00339</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00339&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00339%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDegradation%252Bversus%252BInhibition%25253A%252BDevelopment%252Bof%252BProteolysis-Targeting%252BChimeras%252Bfor%252BOvercoming%252BStatin-Induced%252BCompensatory%252BUpregulation%252Bof%252B3-Hydroxy-3-methylglutaryl%252BCoenzyme%252BA%252BReductase%26aulast%3DLi%26aufirst%3DMei-Xin%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D26022020%26date%3D04052020%26date%3D22042020%26volume%3D63%26issue%3D9%26spage%3D4908%26epage%3D4928" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Ka Yang, Hao Wu, Zhongrui Zhang, Eric D. Leisten, Xueqing Nie, Binkai Liu, Zhi Wen, Jing Zhang, Michael D. Cunningham, <span class="NLM_string-name hlFld-ContribAuthor">Weiping Tang</span>. </span><span class="cited-content_cbyCitation_article-title">Development of Selective Histone Deacetylase 6 (HDAC6) Degraders Recruiting Von Hippel–Lindau (VHL) E3 Ubiquitin Ligase. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2020,</strong> <em>11 </em>
                                    (4)
                                     , 575-581. <a href="https://doi.org/10.1021/acsmedchemlett.0c00046" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.0c00046</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.0c00046&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.0c00046%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DDevelopment%252Bof%252BSelective%252BHistone%252BDeacetylase%252B6%252B%252528HDAC6%252529%252BDegraders%252BRecruiting%252BVon%252BHippel%2525E2%252580%252593Lindau%252B%252528VHL%252529%252BE3%252BUbiquitin%252BLigase%26aulast%3DYang%26aufirst%3DKa%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D28012020%26date%3D13032020%26date%3D18032020%26volume%3D11%26issue%3D4%26spage%3D575%26epage%3D581" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Xin Han, Lijie Zhao, Weiguo Xiang, Chong Qin, Bukeyan Miao, Tianfeng Xu, Mi Wang, Chao-Yie Yang, Krishnapriya Chinnaswamy, Jeanne Stuckey, <span class="NLM_string-name hlFld-ContribAuthor">Shaomeng Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of Highly Potent and Efficient PROTAC Degraders of Androgen Receptor (AR) by Employing Weak Binding Affinity VHL E3 Ligase Ligands. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (24)
                                     , 11218-11231. <a href="https://doi.org/10.1021/acs.jmedchem.9b01393" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01393</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01393&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b01393%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252BHighly%252BPotent%252Band%252BEfficient%252BPROTAC%252BDegraders%252Bof%252BAndrogen%252BReceptor%252B%252528AR%252529%252Bby%252BEmploying%252BWeak%252BBinding%252BAffinity%252BVHL%252BE3%252BLigase%252BLigands%26aulast%3DHan%26aufirst%3DXin%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D22082019%26date%3D05122019%26volume%3D62%26issue%3D24%26spage%3D11218%26epage%3D11231" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Jieli Wei, Jianping Hu, Li Wang, Ling Xie, Margaret S. Jin, Xian Chen, Jing Liu, <span class="NLM_string-name hlFld-ContribAuthor">Jian Jin</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of a First-in-Class Mitogen-Activated Protein Kinase Kinase 1/2 Degrader. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (23)
                                     , 10897-10911. <a href="https://doi.org/10.1021/acs.jmedchem.9b01528" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01528</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01528&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b01528%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252Ba%252BFirst-in-Class%252BMitogen-Activated%252BProtein%252BKinase%252BKinase%252B1%25252F2%252BDegrader%26aulast%3DWei%26aufirst%3DJieli%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D13092019%26date%3D27112019%26date%3D15112019%26volume%3D62%26issue%3D23%26spage%3D10897%26epage%3D10911" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Christopher Heim, Dimanthi Pliatsika, Farnoush Mousavizadeh, Kerstin Bär, Birte Hernandez Alvarez, Athanassios Giannis, <span class="NLM_string-name hlFld-ContribAuthor">Marcus D. Hartmann</span>. </span><span class="cited-content_cbyCitation_article-title">De-Novo Design of Cereblon (CRBN) Effectors Guided by Natural Hydrolysis Products of Thalidomide Derivatives. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (14)
                                     , 6615-6629. <a href="https://doi.org/10.1021/acs.jmedchem.9b00454" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b00454</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b00454&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b00454%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDe-Novo%252BDesign%252Bof%252BCereblon%252B%252528CRBN%252529%252BEffectors%252BGuided%252Bby%252BNatural%252BHydrolysis%252BProducts%252Bof%252BThalidomide%252BDerivatives%26aulast%3DHeim%26aufirst%3DChristopher%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D15032019%26date%3D12072019%26date%3D28062019%26volume%3D62%26issue%3D14%26spage%3D6615%26epage%3D6629" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Cristina  Hernando</span>, <span class="hlFld-ContribAuthor ">Belén  Ortega-Morillo</span>, <span class="hlFld-ContribAuthor ">Marta  Tapia</span>, <span class="hlFld-ContribAuthor ">Santiago  Moragón</span>, <span class="hlFld-ContribAuthor ">María Teresa  Martínez</span>, <span class="hlFld-ContribAuthor ">Pilar  Eroles</span>, <span class="hlFld-ContribAuthor ">Iris  Garrido-Cano</span>, <span class="hlFld-ContribAuthor ">Anna  Adam-Artigues</span>, <span class="hlFld-ContribAuthor ">Ana  Lluch</span>, <span class="hlFld-ContribAuthor ">Begoña  Bermejo</span>, <span class="hlFld-ContribAuthor ">Juan Miguel  Cejalvo</span>. </span><span class="cited-content_cbyCitation_article-title">Oral Selective Estrogen Receptor Degraders (SERDs) as a Novel Breast Cancer Therapy: Present and Future from a Clinical Perspective. </span><span class="cited-content_cbyCitation_journal-name">International Journal of Molecular Sciences</span><span> <strong>2021,</strong> <em>22 </em>
                                    (15)
                                     , 7812. <a href="https://doi.org/10.3390/ijms22157812" title="DOI URL">https://doi.org/10.3390/ijms22157812</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/ijms22157812&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fijms22157812%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Molecular%2520Sciences%26atitle%3DOral%252BSelective%252BEstrogen%252BReceptor%252BDegraders%252B%252528SERDs%252529%252Bas%252Ba%252BNovel%252BBreast%252BCancer%252BTherapy%25253A%252BPresent%252Band%252BFuture%252Bfrom%252Ba%252BClinical%252BPerspective%26aulast%3DHernando%26aufirst%3DCristina%26date%3D2021%26date%3D2021%26volume%3D22%26issue%3D15%26spage%3D7812" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jing  Liu</span>, <span class="hlFld-ContribAuthor ">Yunhua  Peng</span>, <span class="hlFld-ContribAuthor ">Wenyi  Wei</span>. </span><span class="cited-content_cbyCitation_article-title">Light-Controllable PROTACs for Temporospatial Control of Protein Degradation. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Cell and Developmental Biology</span><span> <strong>2021,</strong> <em>9 </em><a href="https://doi.org/10.3389/fcell.2021.678077" title="DOI URL">https://doi.org/10.3389/fcell.2021.678077</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fcell.2021.678077&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffcell.2021.678077%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Cell%2520and%2520Developmental%2520Biology%26atitle%3DLight-Controllable%252BPROTACs%252Bfor%252BTemporospatial%252BControl%252Bof%252BProtein%252BDegradation%26aulast%3DLiu%26aufirst%3DJing%26date%3D2021%26date%3D2021%26volume%3D9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Aleša  Bricelj</span>, <span class="hlFld-ContribAuthor ">Christian  Steinebach</span>, <span class="hlFld-ContribAuthor ">Robert  Kuchta</span>, <span class="hlFld-ContribAuthor ">Michael  Gütschow</span>, <span class="hlFld-ContribAuthor ">Izidor  Sosič</span>. </span><span class="cited-content_cbyCitation_article-title">E3 Ligase Ligands in Successful PROTACs: An Overview of Syntheses and Linker Attachment Points. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Chemistry</span><span> <strong>2021,</strong> <em>9 </em><a href="https://doi.org/10.3389/fchem.2021.707317" title="DOI URL">https://doi.org/10.3389/fchem.2021.707317</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fchem.2021.707317&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffchem.2021.707317%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Chemistry%26atitle%3DE3%252BLigase%252BLigands%252Bin%252BSuccessful%252BPROTACs%25253A%252BAn%252BOverview%252Bof%252BSyntheses%252Band%252BLinker%252BAttachment%252BPoints%26aulast%3DBricelj%26aufirst%3DAle%25C5%25A1a%26date%3D2021%26date%3D2021%26volume%3D9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Brandon  Dale</span>, <span class="hlFld-ContribAuthor ">Meng  Cheng</span>, <span class="hlFld-ContribAuthor ">Kwang-Su  Park</span>, <span class="hlFld-ContribAuthor ">H. Ümit  Kaniskan</span>, <span class="hlFld-ContribAuthor ">Yue  Xiong</span>, <span class="hlFld-ContribAuthor ">Jian  Jin</span>. </span><span class="cited-content_cbyCitation_article-title">Advancing targeted protein degradation for cancer therapy. </span><span class="cited-content_cbyCitation_journal-name">Nature Reviews Cancer</span><span> <strong>2021,</strong> <em>170 </em><a href="https://doi.org/10.1038/s41568-021-00365-x" title="DOI URL">https://doi.org/10.1038/s41568-021-00365-x</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41568-021-00365-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41568-021-00365-x%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Reviews%2520Cancer%26atitle%3DAdvancing%252Btargeted%252Bprotein%252Bdegradation%252Bfor%252Bcancer%252Btherapy%26aulast%3DDale%26aufirst%3DBrandon%26date%3D2021%26date%3D2021%26volume%3D170" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiangbo  Yang</span>, <span class="hlFld-ContribAuthor ">Zhijia  Wang</span>, <span class="hlFld-ContribAuthor ">Yuan  Pei</span>, <span class="hlFld-ContribAuthor ">Ning  Song</span>, <span class="hlFld-ContribAuthor ">Lei  Xu</span>, <span class="hlFld-ContribAuthor ">Bo  Feng</span>, <span class="hlFld-ContribAuthor ">Hanlin  Wang</span>, <span class="hlFld-ContribAuthor ">Xiaomin  Luo</span>, <span class="hlFld-ContribAuthor ">Xiaobei  Hu</span>, <span class="hlFld-ContribAuthor ">Xiaohui  Qiu</span>, <span class="hlFld-ContribAuthor ">Huijin  Feng</span>, <span class="hlFld-ContribAuthor ">Yaxi  Yang</span>, <span class="hlFld-ContribAuthor ">Yubo  Zhou</span>, <span class="hlFld-ContribAuthor ">Jia  Li</span>, <span class="hlFld-ContribAuthor ">Bing  Zhou</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of thalidomide-based PROTAC small molecules as the highly efficient SHP2 degraders. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>218 </em>, 113341. <a href="https://doi.org/10.1016/j.ejmech.2021.113341" title="DOI URL">https://doi.org/10.1016/j.ejmech.2021.113341</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2021.113341&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2021.113341%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252Bthalidomide-based%252BPROTAC%252Bsmall%252Bmolecules%252Bas%252Bthe%252Bhighly%252Befficient%252BSHP2%252Bdegraders%26aulast%3DYang%26aufirst%3DXiangbo%26date%3D2021%26volume%3D218%26spage%3D113341" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Matthew J.  Henley</span>, <span class="hlFld-ContribAuthor ">Angela N.  Koehler</span>. </span><span class="cited-content_cbyCitation_article-title">Advances in targeting ‘undruggable’ transcription factors with small molecules. </span><span class="cited-content_cbyCitation_journal-name">Nature Reviews Drug Discovery</span><span> <strong>2021,</strong> <em>172 </em><a href="https://doi.org/10.1038/s41573-021-00199-0" title="DOI URL">https://doi.org/10.1038/s41573-021-00199-0</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41573-021-00199-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41573-021-00199-0%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Reviews%2520Drug%2520Discovery%26atitle%3DAdvances%252Bin%252Btargeting%252B%2525E2%252580%252598undruggable%2525E2%252580%252599%252Btranscription%252Bfactors%252Bwith%252Bsmall%252Bmolecules%26aulast%3DHenley%26aufirst%3DMatthew%2BJ.%26date%3D2021%26date%3D2021%26volume%3D172" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Guoshun  Luo</span>, <span class="hlFld-ContribAuthor ">Zhenbang  Li</span>, <span class="hlFld-ContribAuthor ">Xin  Lin</span>, <span class="hlFld-ContribAuthor ">Xinyu  Li</span>, <span class="hlFld-ContribAuthor ">Yu  Chen</span>, <span class="hlFld-ContribAuthor ">Kun  Xi</span>, <span class="hlFld-ContribAuthor ">Maoxu  Xiao</span>, <span class="hlFld-ContribAuthor ">Hanlin  Wei</span>, <span class="hlFld-ContribAuthor ">Lizhe  Zhu</span>, <span class="hlFld-ContribAuthor ">Hua  Xiang</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of an orally active VHL-recruiting PROTAC that achieves robust HMGCR degradation and potent hypolipidemic activity in vivo. </span><span class="cited-content_cbyCitation_journal-name">Acta Pharmaceutica Sinica B</span><span> <strong>2021,</strong> <em>11 </em>
                                    (5)
                                     , 1300-1314. <a href="https://doi.org/10.1016/j.apsb.2020.11.001" title="DOI URL">https://doi.org/10.1016/j.apsb.2020.11.001</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.apsb.2020.11.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.apsb.2020.11.001%26sid%3Dliteratum%253Aachs%26jtitle%3DActa%2520Pharmaceutica%2520Sinica%2520B%26atitle%3DDiscovery%252Bof%252Ban%252Borally%252Bactive%252BVHL-recruiting%252BPROTAC%252Bthat%252Bachieves%252Brobust%252BHMGCR%252Bdegradation%252Band%252Bpotent%252Bhypolipidemic%252Bactivity%252Bin%2525C2%2525A0vivo%26aulast%3DLuo%26aufirst%3DGuoshun%26date%3D2021%26volume%3D11%26issue%3D5%26spage%3D1300%26epage%3D1314" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kusal T.G.  Samarasinghe</span>, <span class="hlFld-ContribAuthor ">Saul  Jaime-Figueroa</span>, <span class="hlFld-ContribAuthor ">Michael  Burgess</span>, <span class="hlFld-ContribAuthor ">Dhanusha A.  Nalawansha</span>, <span class="hlFld-ContribAuthor ">Katherine  Dai</span>, <span class="hlFld-ContribAuthor ">Zhenyi  Hu</span>, <span class="hlFld-ContribAuthor ">Adrian  Bebenek</span>, <span class="hlFld-ContribAuthor ">Scott A.  Holley</span>, <span class="hlFld-ContribAuthor ">Craig M.  Crews</span>. </span><span class="cited-content_cbyCitation_article-title">Targeted degradation of transcription factors by TRAFTACs: TRAnscription Factor TArgeting Chimeras. </span><span class="cited-content_cbyCitation_journal-name">Cell Chemical Biology</span><span> <strong>2021,</strong> <em>28 </em>
                                    (5)
                                     , 648-661.e5. <a href="https://doi.org/10.1016/j.chembiol.2021.03.011" title="DOI URL">https://doi.org/10.1016/j.chembiol.2021.03.011</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.chembiol.2021.03.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.chembiol.2021.03.011%26sid%3Dliteratum%253Aachs%26jtitle%3DCell%2520Chemical%2520Biology%26atitle%3DTargeted%252Bdegradation%252Bof%252Btranscription%252Bfactors%252Bby%252BTRAFTACs%25253A%252BTRAnscription%252BFactor%252BTArgeting%252BChimeras%26aulast%3DSamarasinghe%26aufirst%3DKusal%2BT.G.%26date%3D2021%26volume%3D28%26issue%3D5%26spage%3D648%26epage%3D661.e5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zunyuan  Wang</span>, <span class="hlFld-ContribAuthor ">Zhen  Ma</span>, <span class="hlFld-ContribAuthor ">Zhengrong  Shen</span>. </span><span class="cited-content_cbyCitation_article-title">Selective degradation of the estrogen receptor in the treatment of cancers. </span><span class="cited-content_cbyCitation_journal-name">The Journal of Steroid Biochemistry and Molecular Biology</span><span> <strong>2021,</strong> <em>209 </em>, 105848. <a href="https://doi.org/10.1016/j.jsbmb.2021.105848" title="DOI URL">https://doi.org/10.1016/j.jsbmb.2021.105848</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.jsbmb.2021.105848&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.jsbmb.2021.105848%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Journal%2520of%2520Steroid%2520Biochemistry%2520and%2520Molecular%2520Biology%26atitle%3DSelective%252Bdegradation%252Bof%252Bthe%252Bestrogen%252Breceptor%252Bin%252Bthe%252Btreatment%252Bof%252Bcancers%26aulast%3DWang%26aufirst%3DZunyuan%26date%3D2021%26volume%3D209%26spage%3D105848" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Carlo I.  Rosales</span>, <span class="hlFld-ContribAuthor ">Lauren M.  Gutgesell</span>, <span class="hlFld-ContribAuthor ">Kiira M.  Ratia</span>, <span class="hlFld-ContribAuthor ">Gregory R.J.  Thatcher</span>. </span><span class="cited-content_cbyCitation_article-title">Estrogen Receptor Modulators. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,, 1-74. <a href="https://doi.org/10.1002/0471266949.bmc054.pub2" title="DOI URL">https://doi.org/10.1002/0471266949.bmc054.pub2</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/0471266949.bmc054.pub2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F0471266949.bmc054.pub2%26sid%3Dliteratum%253Aachs%26atitle%3DEstrogen%252BReceptor%252BModulators%26aulast%3DRosales%26aufirst%3DCarlo%2BI.%26date%3D2021%26date%3D2021%26spage%3D1%26epage%3D74%26pub%3DWiley%26atitle%3DBurger%252527s%252BMedicinal%252BChemistry%252Band%252BDrug%252BDiscovery%26aulast%3DAbraham%26aufirst%3DDonald%2BJ.%26date%3D2003%26date%3D2003%26volume%3D11" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiang  Lu</span>, <span class="hlFld-ContribAuthor ">Yu  Teng</span>, <span class="hlFld-ContribAuthor ">Xin  Lin</span>, <span class="hlFld-ContribAuthor ">Maoxu  Xiao</span>, <span class="hlFld-ContribAuthor ">Chenna  Liu</span>, <span class="hlFld-ContribAuthor ">Xinglong  Chi</span>, <span class="hlFld-ContribAuthor ">Yulan  Zhang</span>, <span class="hlFld-ContribAuthor ">Guoshun  Luo</span>, <span class="hlFld-ContribAuthor ">Hua  Xiang</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of novel 2H-chromene-3-carbonyl derivatives as selective estrogen receptor degraders (SERDs): Design, synthesis and biological evaluation. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic Chemistry</span><span> <strong>2021,</strong> <em>109 </em>, 104714. <a href="https://doi.org/10.1016/j.bioorg.2021.104714" title="DOI URL">https://doi.org/10.1016/j.bioorg.2021.104714</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bioorg.2021.104714&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bioorg.2021.104714%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520Chemistry%26atitle%3DDiscovery%252Bof%252Bnovel%252B2H-chromene-3-carbonyl%252Bderivatives%252Bas%252Bselective%252Bestrogen%252Breceptor%252Bdegraders%252B%252528SERDs%252529%25253A%252BDesign%25252C%252Bsynthesis%252Band%252Bbiological%252Bevaluation%26aulast%3DLu%26aufirst%3DXiang%26date%3D2021%26volume%3D109%26spage%3D104714" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yvonne A.  Nagel</span>, <span class="hlFld-ContribAuthor ">Adrian  Britschgi</span>, <span class="hlFld-ContribAuthor ">Antonio  Ricci</span>. </span><span class="cited-content_cbyCitation_article-title">From Degraders to Molecular Glues: New Ways of Breaking Down Disease‐Associated Proteins. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,, 47-85. <a href="https://doi.org/10.1002/9783527826872.ch2" title="DOI URL">https://doi.org/10.1002/9783527826872.ch2</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/9783527826872.ch2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F9783527826872.ch2%26sid%3Dliteratum%253Aachs%26atitle%3DFrom%252BDegraders%252Bto%252BMolecular%252BGlues%25253A%252BNew%252BWays%252Bof%252BBreaking%252BDown%252BDisease%2525E2%252580%252590Associated%252BProteins%26aulast%3DNagel%26aufirst%3DYvonne%2BA.%26date%3D2021%26date%3D2021%26spage%3D47%26epage%3D85%26pub%3DWiley%26atitle%3DSuccessful%252BDrug%252BDiscovery%26aulast%3DFischer%26aufirst%3DJ%25C3%25A1nos%26date%3D2021%26date%3D2021%26volume%3D115" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yasser  Tabana</span>, <span class="hlFld-ContribAuthor ">Isobel S.  Okoye</span>, <span class="hlFld-ContribAuthor ">Arno  Siraki</span>, <span class="hlFld-ContribAuthor ">Shokrollah  Elahi</span>, <span class="hlFld-ContribAuthor ">Khaled H.  Barakat</span>. </span><span class="cited-content_cbyCitation_article-title">Tackling Immune Targets for Breast Cancer: Beyond PD-1/PD-L1 Axis. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Oncology</span><span> <strong>2021,</strong> <em>11 </em><a href="https://doi.org/10.3389/fonc.2021.628138" title="DOI URL">https://doi.org/10.3389/fonc.2021.628138</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fonc.2021.628138&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffonc.2021.628138%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Oncology%26atitle%3DTackling%252BImmune%252BTargets%252Bfor%252BBreast%252BCancer%25253A%252BBeyond%252BPD-1%25252FPD-L1%252BAxis%26aulast%3DTabana%26aufirst%3DYasser%26date%3D2021%26date%3D2021%26volume%3D11" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Andrew B.  Benowitz</span>, <span class="hlFld-ContribAuthor ">Katherine L.  Jones</span>, <span class="hlFld-ContribAuthor ">John D.  Harling</span>. </span><span class="cited-content_cbyCitation_article-title">The therapeutic potential of PROTACs. </span><span class="cited-content_cbyCitation_journal-name">Expert Opinion on Therapeutic Patents</span><span> <strong>2021,</strong> <em>31 </em>
                                    (1)
                                     , 1-24. <a href="https://doi.org/10.1080/13543776.2021.1840553" title="DOI URL">https://doi.org/10.1080/13543776.2021.1840553</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/13543776.2021.1840553&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F13543776.2021.1840553%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Opinion%2520on%2520Therapeutic%2520Patents%26atitle%3DThe%252Btherapeutic%252Bpotential%252Bof%252BPROTACs%26aulast%3DBenowitz%26aufirst%3DAndrew%2BB.%26date%3D2021%26date%3D2021%26volume%3D31%26issue%3D1%26spage%3D1%26epage%3D24" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xinrui  Yang</span>, <span class="hlFld-ContribAuthor ">He  Yin</span>, <span class="hlFld-ContribAuthor ">Richard D.  Kim</span>, <span class="hlFld-ContribAuthor ">Jason B.  Fleming</span>, <span class="hlFld-ContribAuthor ">Hao  Xie</span>. </span><span class="cited-content_cbyCitation_article-title">Preclinical and Clinical Advances of Targeted Protein Degradation as a Novel Cancer Therapeutic Strategy: An Oncologist Perspective. </span><span class="cited-content_cbyCitation_journal-name">Targeted Oncology</span><span> <strong>2021,</strong> <em>16 </em>
                                    (1)
                                     , 1-12. <a href="https://doi.org/10.1007/s11523-020-00782-2" title="DOI URL">https://doi.org/10.1007/s11523-020-00782-2</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s11523-020-00782-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs11523-020-00782-2%26sid%3Dliteratum%253Aachs%26jtitle%3DTargeted%2520Oncology%26atitle%3DPreclinical%252Band%252BClinical%252BAdvances%252Bof%252BTargeted%252BProtein%252BDegradation%252Bas%252Ba%252BNovel%252BCancer%252BTherapeutic%252BStrategy%25253A%252BAn%252BOncologist%252BPerspective%26aulast%3DYang%26aufirst%3DXinrui%26date%3D2021%26date%3D2020%26volume%3D16%26issue%3D1%26spage%3D1%26epage%3D12" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shenxin  Zeng</span>, <span class="hlFld-ContribAuthor ">Wenhai  Huang</span>, <span class="hlFld-ContribAuthor ">Xiaoliang  Zheng</span>, <span class="hlFld-ContribAuthor ">Liyan cheng</span>, <span class="hlFld-ContribAuthor ">Zhimin  Zhang</span>, <span class="hlFld-ContribAuthor ">Jian  Wang</span>, <span class="hlFld-ContribAuthor ">Zhengrong  Shen</span>. </span><span class="cited-content_cbyCitation_article-title">Proteolysis targeting chimera (PROTAC) in drug discovery paradigm: Recent progress and future challenges. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>210 </em>, 112981. <a href="https://doi.org/10.1016/j.ejmech.2020.112981" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112981</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112981&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112981%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DProteolysis%252Btargeting%252Bchimera%252B%252528PROTAC%252529%252Bin%252Bdrug%252Bdiscovery%252Bparadigm%25253A%252BRecent%252Bprogress%252Band%252Bfuture%252Bchallenges%26aulast%3DZeng%26aufirst%3DShenxin%26date%3D2021%26volume%3D210%26spage%3D112981" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">M.  Maneiro</span>, <span class="hlFld-ContribAuthor ">E.  De Vita</span>, <span class="hlFld-ContribAuthor ">D.  Conole</span>, <span class="hlFld-ContribAuthor ">C.S.  Kounde</span>, <span class="hlFld-ContribAuthor ">Q.  Zhang</span>, <span class="hlFld-ContribAuthor ">E.W.  Tate</span>. </span><span class="cited-content_cbyCitation_article-title">PROTACs, molecular glues and bifunctionals from bench to bedside: Unlocking the clinical potential of catalytic drugs. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,<a href="https://doi.org/10.1016/bs.pmch.2021.01.002" title="DOI URL">https://doi.org/10.1016/bs.pmch.2021.01.002</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/bs.pmch.2021.01.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fbs.pmch.2021.01.002%26sid%3Dliteratum%253Aachs%26atitle%3DPROTACs%25252C%252Bmolecular%252Bglues%252Band%252Bbifunctionals%252Bfrom%252Bbench%252Bto%252Bbedside%25253A%252BUnlocking%252Bthe%252Bclinical%252Bpotential%252Bof%252Bcatalytic%252Bdrugs%26aulast%3DManeiro%26aufirst%3DM.%26date%3D2021%26pub%3DElsevier%26date%3D2021%26volume%3D61" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Nuno A.  Padrão</span>, <span class="hlFld-ContribAuthor ">Isabel  Mayayo-Peralta</span>, <span class="hlFld-ContribAuthor ">Wilbert  Zwart</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting mutated estrogen receptor alpha: Rediscovering old and identifying new therapeutic strategies in metastatic breast cancer treatment. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Endocrine and Metabolic Research</span><span> <strong>2020,</strong> <em>15 </em>, 43-48. <a href="https://doi.org/10.1016/j.coemr.2020.10.008" title="DOI URL">https://doi.org/10.1016/j.coemr.2020.10.008</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.coemr.2020.10.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.coemr.2020.10.008%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Endocrine%2520and%2520Metabolic%2520Research%26atitle%3DTargeting%252Bmutated%252Bestrogen%252Breceptor%252Balpha%25253A%252BRediscovering%252Bold%252Band%252Bidentifying%252Bnew%252Btherapeutic%252Bstrategies%252Bin%252Bmetastatic%252Bbreast%252Bcancer%252Btreatment%26aulast%3DPadr%25C3%25A3o%26aufirst%3DNuno%2BA.%26date%3D2020%26volume%3D15%26spage%3D43%26epage%3D48" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Akshay D.  Takwale</span>, <span class="hlFld-ContribAuthor ">Seung-Hyun  Jo</span>, <span class="hlFld-ContribAuthor ">Yeong Uk  Jeon</span>, <span class="hlFld-ContribAuthor ">Hyung Soo  Kim</span>, <span class="hlFld-ContribAuthor ">Choong Hoon  Shin</span>, <span class="hlFld-ContribAuthor ">Heung Kyoung  Lee</span>, <span class="hlFld-ContribAuthor ">Sunjoo  Ahn</span>, <span class="hlFld-ContribAuthor ">Chong Ock  Lee</span>, <span class="hlFld-ContribAuthor ">Jae  Du Ha</span>, <span class="hlFld-ContribAuthor ">Jeong-Hoon  Kim</span>, <span class="hlFld-ContribAuthor ">Jong Yeon  Hwang</span>. </span><span class="cited-content_cbyCitation_article-title">Design and characterization of cereblon-mediated androgen receptor proteolysis-targeting chimeras. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>208 </em>, 112769. <a href="https://doi.org/10.1016/j.ejmech.2020.112769" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112769</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112769&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112769%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDesign%252Band%252Bcharacterization%252Bof%252Bcereblon-mediated%252Bandrogen%252Breceptor%252Bproteolysis-targeting%252Bchimeras%26aulast%3DTakwale%26aufirst%3DAkshay%2BD.%26date%3D2020%26volume%3D208%26spage%3D112769" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jing  Liu</span>, <span class="hlFld-ContribAuthor ">Jia  Ma</span>, <span class="hlFld-ContribAuthor ">Yi  Liu</span>, <span class="hlFld-ContribAuthor ">Jun  Xia</span>, <span class="hlFld-ContribAuthor ">Yuyun  Li</span>, <span class="hlFld-ContribAuthor ">Z. Peter  Wang</span>, <span class="hlFld-ContribAuthor ">Wenyi  Wei</span>. </span><span class="cited-content_cbyCitation_article-title">PROTACs: A novel strategy for cancer therapy. </span><span class="cited-content_cbyCitation_journal-name">Seminars in Cancer Biology</span><span> <strong>2020,</strong> <em>67 </em>, 171-179. <a href="https://doi.org/10.1016/j.semcancer.2020.02.006" title="DOI URL">https://doi.org/10.1016/j.semcancer.2020.02.006</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.semcancer.2020.02.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.semcancer.2020.02.006%26sid%3Dliteratum%253Aachs%26jtitle%3DSeminars%2520in%2520Cancer%2520Biology%26atitle%3DPROTACs%25253A%252BA%252Bnovel%252Bstrategy%252Bfor%252Bcancer%252Btherapy%26aulast%3DLiu%26aufirst%3DJing%26date%3D2020%26volume%3D67%26spage%3D171%26epage%3D179" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jifa  Zhang</span>, <span class="hlFld-ContribAuthor ">Liang  Ouyang</span>. </span><span class="cited-content_cbyCitation_article-title">Proteolysis-targeting chimeras in breast cancer therapy. </span><span class="cited-content_cbyCitation_journal-name">Future Medicinal Chemistry</span><span> <strong>2020,</strong> <em>12 </em>
                                    (24)
                                     , 2175-2177. <a href="https://doi.org/10.4155/fmc-2020-0279" title="DOI URL">https://doi.org/10.4155/fmc-2020-0279</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fmc-2020-0279&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffmc-2020-0279%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Medicinal%2520Chemistry%26atitle%3DProteolysis-targeting%252Bchimeras%252Bin%252Bbreast%252Bcancer%252Btherapy%26aulast%3DZhang%26aufirst%3DJifa%26date%3D2020%26volume%3D12%26issue%3D24%26spage%3D2175%26epage%3D2177" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lucia  Wang</span>, <span class="hlFld-ContribAuthor ">Abhishek  Sharma</span>. </span><span class="cited-content_cbyCitation_article-title">The Quest for Orally Available Selective Estrogen Receptor Degraders (SERDs). </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2020,</strong> <em>15 </em>
                                    (22)
                                     , 2072-2097. <a href="https://doi.org/10.1002/cmdc.202000473" title="DOI URL">https://doi.org/10.1002/cmdc.202000473</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.202000473&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.202000473%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DThe%252BQuest%252Bfor%252BOrally%252BAvailable%252BSelective%252BEstrogen%252BReceptor%252BDegraders%252B%252528SERDs%252529%26aulast%3DWang%26aufirst%3DLucia%26date%3D2020%26date%3D2020%26volume%3D15%26issue%3D22%26spage%3D2072%26epage%3D2097" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jie  Li</span>, <span class="hlFld-ContribAuthor ">Jieqing  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">PROTAC: A Novel Technology for Drug Development**. </span><span class="cited-content_cbyCitation_journal-name">ChemistrySelect</span><span> <strong>2020,</strong> <em>5 </em>
                                    (42)
                                     , 13232-13247. <a href="https://doi.org/10.1002/slct.202003162" title="DOI URL">https://doi.org/10.1002/slct.202003162</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/slct.202003162&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fslct.202003162%26sid%3Dliteratum%253Aachs%26jtitle%3DChemistrySelect%26atitle%3DPROTAC%25253A%252BA%252BNovel%252BTechnology%252Bfor%252BDrug%252BDevelopment%252A%252A%26aulast%3DLi%26aufirst%3DJie%26date%3D2020%26date%3D2020%26volume%3D5%26issue%3D42%26spage%3D13232%26epage%3D13247" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lina  Yin</span>, <span class="hlFld-ContribAuthor ">Qingzhong  Hu</span>. </span><span class="cited-content_cbyCitation_article-title">Chimera induced protein degradation: PROTACs and beyond. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>206 </em>, 112494. <a href="https://doi.org/10.1016/j.ejmech.2020.112494" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112494</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112494&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112494%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DChimera%252Binduced%252Bprotein%252Bdegradation%25253A%252BPROTACs%252Band%252Bbeyond%26aulast%3DYin%26aufirst%3DLina%26date%3D2020%26volume%3D206%26spage%3D112494" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xin  Lin</span>, <span class="hlFld-ContribAuthor ">Hua  Xiang</span>, <span class="hlFld-ContribAuthor ">Guoshun  Luo</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting estrogen receptor α for degradation with PROTACs: A promising approach to overcome endocrine resistance. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>206 </em>, 112689. <a href="https://doi.org/10.1016/j.ejmech.2020.112689" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112689</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112689&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112689%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DTargeting%252Bestrogen%252Breceptor%252B%2525CE%2525B1%252Bfor%252Bdegradation%252Bwith%252BPROTACs%25253A%252BA%252Bpromising%252Bapproach%252Bto%252Bovercome%252Bendocrine%252Bresistance%26aulast%3DLin%26aufirst%3DXin%26date%3D2020%26volume%3D206%26spage%3D112689" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xin  Zhou</span>, <span class="hlFld-ContribAuthor ">Ru  Dong</span>, <span class="hlFld-ContribAuthor ">Jin-Yang  Zhang</span>, <span class="hlFld-ContribAuthor ">Xin  Zheng</span>, <span class="hlFld-ContribAuthor ">Li-Ping  Sun</span>. </span><span class="cited-content_cbyCitation_article-title">PROTAC: A promising technology for cancer treatment. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>203 </em>, 112539. <a href="https://doi.org/10.1016/j.ejmech.2020.112539" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112539</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112539&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112539%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DPROTAC%25253A%252BA%252Bpromising%252Btechnology%252Bfor%252Bcancer%252Btreatment%26aulast%3DZhou%26aufirst%3DXin%26date%3D2020%26volume%3D203%26spage%3D112539" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Francisco J.  Cimas</span>, <span class="hlFld-ContribAuthor ">Enrique  Niza</span>, <span class="hlFld-ContribAuthor ">Alberto  Juan</span>, <span class="hlFld-ContribAuthor ">María del Mar  Noblejas-López</span>, <span class="hlFld-ContribAuthor ">Iván  Bravo</span>, <span class="hlFld-ContribAuthor ">Agustín  Lara-Sanchez</span>, <span class="hlFld-ContribAuthor ">Carlos  Alonso-Moreno</span>, <span class="hlFld-ContribAuthor ">Alberto  Ocaña</span>. </span><span class="cited-content_cbyCitation_article-title">Controlled Delivery of BET-PROTACs: In Vitro Evaluation of MZ1-Loaded Polymeric Antibody Conjugated Nanoparticles in Breast Cancer. </span><span class="cited-content_cbyCitation_journal-name">Pharmaceutics</span><span> <strong>2020,</strong> <em>12 </em>
                                    (10)
                                     , 986. <a href="https://doi.org/10.3390/pharmaceutics12100986" title="DOI URL">https://doi.org/10.3390/pharmaceutics12100986</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/pharmaceutics12100986&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fpharmaceutics12100986%26sid%3Dliteratum%253Aachs%26jtitle%3DPharmaceutics%26atitle%3DControlled%252BDelivery%252Bof%252BBET-PROTACs%25253A%252BIn%252BVitro%252BEvaluation%252Bof%252BMZ1-Loaded%252BPolymeric%252BAntibody%252BConjugated%252BNanoparticles%252Bin%252BBreast%252BCancer%26aulast%3DCimas%26aufirst%3DFrancisco%2BJ.%26date%3D2020%26date%3D2020%26volume%3D12%26issue%3D10%26spage%3D986" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shuhua  Zheng</span>, <span class="hlFld-ContribAuthor ">Wensi  Tao</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting Cullin-RING E3 Ligases for Radiosensitization: From NEDDylation Inhibition to PROTACs. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Oncology</span><span> <strong>2020,</strong> <em>10 </em><a href="https://doi.org/10.3389/fonc.2020.01517" title="DOI URL">https://doi.org/10.3389/fonc.2020.01517</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fonc.2020.01517&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffonc.2020.01517%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Oncology%26atitle%3DTargeting%252BCullin-RING%252BE3%252BLigases%252Bfor%252BRadiosensitization%25253A%252BFrom%252BNEDDylation%252BInhibition%252Bto%252BPROTACs%26aulast%3DZheng%26aufirst%3DShuhua%26date%3D2020%26date%3D2020%26volume%3D10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dhanusha A.  Nalawansha</span>, <span class="hlFld-ContribAuthor ">Craig M.  Crews</span>. </span><span class="cited-content_cbyCitation_article-title">PROTACs: An Emerging Therapeutic Modality in Precision Medicine. </span><span class="cited-content_cbyCitation_journal-name">Cell Chemical Biology</span><span> <strong>2020,</strong> <em>27 </em>
                                    (8)
                                     , 998-1014. <a href="https://doi.org/10.1016/j.chembiol.2020.07.020" title="DOI URL">https://doi.org/10.1016/j.chembiol.2020.07.020</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.chembiol.2020.07.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.chembiol.2020.07.020%26sid%3Dliteratum%253Aachs%26jtitle%3DCell%2520Chemical%2520Biology%26atitle%3DPROTACs%25253A%252BAn%252BEmerging%252BTherapeutic%252BModality%252Bin%252BPrecision%252BMedicine%26aulast%3DNalawansha%26aufirst%3DDhanusha%2BA.%26date%3D2020%26volume%3D27%26issue%3D8%26spage%3D998%26epage%3D1014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kailun  He</span>, <span class="hlFld-ContribAuthor ">Zhuo  Zhang</span>, <span class="hlFld-ContribAuthor ">Wenbing  Wang</span>, <span class="hlFld-ContribAuthor ">Xiaoliang  Zheng</span>, <span class="hlFld-ContribAuthor ">Xiaoju  Wang</span>, <span class="hlFld-ContribAuthor ">Xingxian  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery and biological evaluation of proteolysis targeting chimeras (PROTACs) as an EGFR degraders based on osimertinib and lenalidomide. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2020,</strong> <em>30 </em>
                                    (12)
                                     , 127167. <a href="https://doi.org/10.1016/j.bmcl.2020.127167" title="DOI URL">https://doi.org/10.1016/j.bmcl.2020.127167</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2020.127167&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2020.127167%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DDiscovery%252Band%252Bbiological%252Bevaluation%252Bof%252Bproteolysis%252Btargeting%252Bchimeras%252B%252528PROTACs%252529%252Bas%252Ban%252BEGFR%252Bdegraders%252Bbased%252Bon%252Bosimertinib%252Band%252Blenalidomide%26aulast%3DHe%26aufirst%3DKailun%26date%3D2020%26volume%3D30%26issue%3D12%26spage%3D127167" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Divya  Bafna</span>, <span class="hlFld-ContribAuthor ">Fuqiang  Ban</span>, <span class="hlFld-ContribAuthor ">Paul S.  Rennie</span>, <span class="hlFld-ContribAuthor ">Kriti  Singh</span>, <span class="hlFld-ContribAuthor ">Artem  Cherkasov</span>. </span><span class="cited-content_cbyCitation_article-title">Computer-Aided Ligand Discovery for Estrogen Receptor Alpha. </span><span class="cited-content_cbyCitation_journal-name">International Journal of Molecular Sciences</span><span> <strong>2020,</strong> <em>21 </em>
                                    (12)
                                     , 4193. <a href="https://doi.org/10.3390/ijms21124193" title="DOI URL">https://doi.org/10.3390/ijms21124193</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/ijms21124193&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fijms21124193%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Molecular%2520Sciences%26atitle%3DComputer-Aided%252BLigand%252BDiscovery%252Bfor%252BEstrogen%252BReceptor%252BAlpha%26aulast%3DBafna%26aufirst%3DDivya%26date%3D2020%26date%3D2020%26volume%3D21%26issue%3D12%26spage%3D4193" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sara S  Bashraheel</span>, <span class="hlFld-ContribAuthor ">Alexander  Domling</span>, <span class="hlFld-ContribAuthor ">Sayed K  Goda</span>. </span><span class="cited-content_cbyCitation_article-title">Update on targeted cancer therapies, single or in combination, and their fine tuning for precision medicine. </span><span class="cited-content_cbyCitation_journal-name">Biomedicine & Pharmacotherapy</span><span> <strong>2020,</strong> <em>125 </em>, 110009. <a href="https://doi.org/10.1016/j.biopha.2020.110009" title="DOI URL">https://doi.org/10.1016/j.biopha.2020.110009</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.biopha.2020.110009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.biopha.2020.110009%26sid%3Dliteratum%253Aachs%26jtitle%3DBiomedicine%2520%2526%2520Pharmacotherapy%26atitle%3DUpdate%252Bon%252Btargeted%252Bcancer%252Btherapies%25252C%252Bsingle%252Bor%252Bin%252Bcombination%25252C%252Band%252Btheir%252Bfine%252Btuning%252Bfor%252Bprecision%252Bmedicine%26aulast%3DBashraheel%26aufirst%3DSara%2BS%26date%3D2020%26volume%3D125%26spage%3D110009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ning  Sun</span>, <span class="hlFld-ContribAuthor ">Chaowei  Ren</span>, <span class="hlFld-ContribAuthor ">Ying  Kong</span>, <span class="hlFld-ContribAuthor ">Hui  Zhong</span>, <span class="hlFld-ContribAuthor ">Jinju  Chen</span>, <span class="hlFld-ContribAuthor ">Yan  Li</span>, <span class="hlFld-ContribAuthor ">Jianshui  Zhang</span>, <span class="hlFld-ContribAuthor ">Yuedong  Zhou</span>, <span class="hlFld-ContribAuthor ">Xing  Qiu</span>, <span class="hlFld-ContribAuthor ">Haifan  Lin</span>, <span class="hlFld-ContribAuthor ">Xiaoling  Song</span>, <span class="hlFld-ContribAuthor ">Xiaobao  Yang</span>, <span class="hlFld-ContribAuthor ">Biao  Jiang</span>. </span><span class="cited-content_cbyCitation_article-title">Development of a Brigatinib degrader (SIAIS117) as a potential treatment for ALK positive cancer resistance. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>193 </em>, 112190. <a href="https://doi.org/10.1016/j.ejmech.2020.112190" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112190</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112190&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112190%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDevelopment%252Bof%252Ba%252BBrigatinib%252Bdegrader%252B%252528SIAIS117%252529%252Bas%252Ba%252Bpotential%252Btreatment%252Bfor%252BALK%252Bpositive%252Bcancer%252Bresistance%26aulast%3DSun%26aufirst%3DNing%26date%3D2020%26volume%3D193%26spage%3D112190" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shagufta</span>, <span class="hlFld-ContribAuthor ">Irshad  Ahmad</span>, <span class="hlFld-ContribAuthor ">Shimy  Mathew</span>, <span class="hlFld-ContribAuthor ">Sofia  Rahman</span>. </span><span class="cited-content_cbyCitation_article-title">Recent progress in selective estrogen receptor downregulators (SERDs) for the treatment of breast cancer. </span><span class="cited-content_cbyCitation_journal-name">RSC Medicinal Chemistry</span><span> <strong>2020,</strong> <em>11 </em>
                                    (4)
                                     , 438-454. <a href="https://doi.org/10.1039/C9MD00570F" title="DOI URL">https://doi.org/10.1039/C9MD00570F</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C9MD00570F&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC9MD00570F%26sid%3Dliteratum%253Aachs%26jtitle%3DRSC%2520Medicinal%2520Chemistry%26atitle%3DRecent%252Bprogress%252Bin%252Bselective%252Bestrogen%252Breceptor%252Bdownregulators%252B%252528SERDs%252529%252Bfor%252Bthe%252Btreatment%252Bof%252Bbreast%252Bcancer%26aulast%3DShagufta%26date%3D2020%26date%3D2020%26volume%3D11%26issue%3D4%26spage%3D438%26epage%3D454" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Thomas L.  Gonzalez</span>, <span class="hlFld-ContribAuthor ">Molly  Hancock</span>, <span class="hlFld-ContribAuthor ">Siqi  Sun</span>, <span class="hlFld-ContribAuthor ">Christina L.  Gersch</span>, <span class="hlFld-ContribAuthor ">Jose M.  Larios</span>, <span class="hlFld-ContribAuthor ">Wadie  David</span>, <span class="hlFld-ContribAuthor ">Jiantao  Hu</span>, <span class="hlFld-ContribAuthor ">Daniel F.  Hayes</span>, <span class="hlFld-ContribAuthor ">Shaomeng  Wang</span>, <span class="hlFld-ContribAuthor ">James M.  Rae</span>. </span><span class="cited-content_cbyCitation_article-title">Targeted degradation of activating estrogen receptor α ligand-binding domain mutations in human breast cancer. </span><span class="cited-content_cbyCitation_journal-name">Breast Cancer Research and Treatment</span><span> <strong>2020,</strong> <em>180 </em>
                                    (3)
                                     , 611-622. <a href="https://doi.org/10.1007/s10549-020-05564-y" title="DOI URL">https://doi.org/10.1007/s10549-020-05564-y</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s10549-020-05564-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs10549-020-05564-y%26sid%3Dliteratum%253Aachs%26jtitle%3DBreast%2520Cancer%2520Research%2520and%2520Treatment%26atitle%3DTargeted%252Bdegradation%252Bof%252Bactivating%252Bestrogen%252Breceptor%252B%2525CE%2525B1%252Bligand-binding%252Bdomain%252Bmutations%252Bin%252Bhuman%252Bbreast%252Bcancer%26aulast%3DGonzalez%26aufirst%3DThomas%2BL.%26date%3D2020%26date%3D2020%26volume%3D180%26issue%3D3%26spage%3D611%26epage%3D622" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xuan  Zhang</span>, <span class="hlFld-ContribAuthor ">Dinesh  Thummuri</span>, <span class="hlFld-ContribAuthor ">Xingui  Liu</span>, <span class="hlFld-ContribAuthor ">Wanyi  Hu</span>, <span class="hlFld-ContribAuthor ">Peiyi  Zhang</span>, <span class="hlFld-ContribAuthor ">Sajid  Khan</span>, <span class="hlFld-ContribAuthor ">Yaxia  Yuan</span>, <span class="hlFld-ContribAuthor ">Daohong  Zhou</span>, <span class="hlFld-ContribAuthor ">Guangrong  Zheng</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of PROTAC BCL-XL degraders as potent anticancer agents with low on-target platelet toxicity. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>192 </em>, 112186. <a href="https://doi.org/10.1016/j.ejmech.2020.112186" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112186</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112186&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112186%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252BPROTAC%252BBCL-XL%252Bdegraders%252Bas%252Bpotent%252Banticancer%252Bagents%252Bwith%252Blow%252Bon-target%252Bplatelet%252Btoxicity%26aulast%3DZhang%26aufirst%3DXuan%26date%3D2020%26volume%3D192%26spage%3D112186" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">George M.  Burslem</span>, <span class="hlFld-ContribAuthor ">Craig M.  Crews</span>. </span><span class="cited-content_cbyCitation_article-title">Proteolysis-Targeting Chimeras as Therapeutics and Tools for Biological Discovery. </span><span class="cited-content_cbyCitation_journal-name">Cell</span><span> <strong>2020,</strong> <em>181 </em>
                                    (1)
                                     , 102-114. <a href="https://doi.org/10.1016/j.cell.2019.11.031" title="DOI URL">https://doi.org/10.1016/j.cell.2019.11.031</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.cell.2019.11.031&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.cell.2019.11.031%26sid%3Dliteratum%253Aachs%26jtitle%3DCell%26atitle%3DProteolysis-Targeting%252BChimeras%252Bas%252BTherapeutics%252Band%252BTools%252Bfor%252BBiological%252BDiscovery%26aulast%3DBurslem%26aufirst%3DGeorge%2BM.%26date%3D2020%26volume%3D181%26issue%3D1%26spage%3D102%26epage%3D114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Qiuyao  Huang</span>, <span class="hlFld-ContribAuthor ">Yan  Zhong</span>, <span class="hlFld-ContribAuthor ">Hui  Dong</span>, <span class="hlFld-ContribAuthor ">Qiyao  Zheng</span>, <span class="hlFld-ContribAuthor ">Shuo  Shi</span>, <span class="hlFld-ContribAuthor ">Kai  Zhu</span>, <span class="hlFld-ContribAuthor ">Xinming  Qu</span>, <span class="hlFld-ContribAuthor ">Wenhao  Hu</span>, <span class="hlFld-ContribAuthor ">Xiaolei  Zhang</span>, <span class="hlFld-ContribAuthor ">Yuanxiang  Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Revisiting signal transducer and activator of transcription 3 (STAT3) as an anticancer target and its inhibitor discovery: Where are we and where should we go?. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>187 </em>, 111922. <a href="https://doi.org/10.1016/j.ejmech.2019.111922" title="DOI URL">https://doi.org/10.1016/j.ejmech.2019.111922</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2019.111922&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2019.111922%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DRevisiting%252Bsignal%252Btransducer%252Band%252Bactivator%252Bof%252Btranscription%252B3%252B%252528STAT3%252529%252Bas%252Ban%252Banticancer%252Btarget%252Band%252Bits%252Binhibitor%252Bdiscovery%25253A%252BWhere%252Bare%252Bwe%252Band%252Bwhere%252Bshould%252Bwe%252Bgo%25253F%26aulast%3DHuang%26aufirst%3DQiuyao%26date%3D2020%26volume%3D187%26spage%3D111922" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Martin  Reynders</span>, <span class="hlFld-ContribAuthor ">Bryan S.  Matsuura</span>, <span class="hlFld-ContribAuthor ">Marleen  Bérouti</span>, <span class="hlFld-ContribAuthor ">Daniele  Simoneschi</span>, <span class="hlFld-ContribAuthor ">Antonio  Marzio</span>, <span class="hlFld-ContribAuthor ">Michele  Pagano</span>, <span class="hlFld-ContribAuthor ">Dirk  Trauner</span>. </span><span class="cited-content_cbyCitation_article-title">PHOTACs enable optical control of protein degradation. </span><span class="cited-content_cbyCitation_journal-name">Science Advances</span><span> <strong>2020,</strong> <em>6 </em>
                                    (8)
                                     , eaay5064. <a href="https://doi.org/10.1126/sciadv.aay5064" title="DOI URL">https://doi.org/10.1126/sciadv.aay5064</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1126/sciadv.aay5064&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1126%2Fsciadv.aay5064%26sid%3Dliteratum%253Aachs%26jtitle%3DScience%2520Advances%26atitle%3DPHOTACs%252Benable%252Boptical%252Bcontrol%252Bof%252Bprotein%252Bdegradation%26aulast%3DReynders%26aufirst%3DMartin%26date%3D2020%26date%3D2020%26volume%3D6%26issue%3D8%26spage%3Deaay5064" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">V. Prakash  Reddy</span>. </span><span class="cited-content_cbyCitation_article-title">Pharmaceutical applications of organofluorine compounds. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2020,</strong>,, 133-214. <a href="https://doi.org/10.1016/B978-0-12-813286-9.00005-5" title="DOI URL">https://doi.org/10.1016/B978-0-12-813286-9.00005-5</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-813286-9.00005-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-813286-9.00005-5%26sid%3Dliteratum%253Aachs%26atitle%3DPharmaceutical%252Bapplications%252Bof%252Borganofluorine%252Bcompounds%26aulast%3DReddy%26aufirst%3DV.%2BPrakash%26date%3D2020%26spage%3D133%26epage%3D214%26pub%3DElsevier%26atitle%3DOrganofluorine%252BChemistry%26date%3D2020%26volume%3D58" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Samantha J.  Allen</span>, <span class="hlFld-ContribAuthor ">Kevin J.  Lumb</span>. </span><span class="cited-content_cbyCitation_article-title">Protein-protein interactions: a structural view of inhibition strategies and the IL-23/IL-17 axis. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2020,</strong>,, 253-303. <a href="https://doi.org/10.1016/bs.apcsb.2019.12.006" title="DOI URL">https://doi.org/10.1016/bs.apcsb.2019.12.006</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/bs.apcsb.2019.12.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fbs.apcsb.2019.12.006%26sid%3Dliteratum%253Aachs%26atitle%3DProtein-protein%252Binteractions%25253A%252Ba%252Bstructural%252Bview%252Bof%252Binhibition%252Bstrategies%252Band%252Bthe%252BIL-23%25252FIL-17%252Baxis%26aulast%3DAllen%26aufirst%3DSamantha%2BJ.%26date%3D2020%26spage%3D253%26epage%3D303%26pub%3DElsevier%26date%3D2020%26volume%3D121" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zhimin  Zhang</span>, <span class="hlFld-ContribAuthor ">Xinyue  Chang</span>, <span class="hlFld-ContribAuthor ">Chixiao  Zhang</span>, <span class="hlFld-ContribAuthor ">Shenxin  Zeng</span>, <span class="hlFld-ContribAuthor ">Meihao  Liang</span>, <span class="hlFld-ContribAuthor ">Zhen  Ma</span>, <span class="hlFld-ContribAuthor ">Zunyuan  Wang</span>, <span class="hlFld-ContribAuthor ">Wenhai  Huang</span>, <span class="hlFld-ContribAuthor ">Zhengrong  Shen</span>. </span><span class="cited-content_cbyCitation_article-title">Identification of probe-quality degraders for Poly(ADP-ribose) polymerase-1 (PARP-1). </span><span class="cited-content_cbyCitation_journal-name">Journal of Enzyme Inhibition and Medicinal Chemistry</span><span> <strong>2020,</strong> <em>35 </em>
                                    (1)
                                     , 1606-1615. <a href="https://doi.org/10.1080/14756366.2020.1804382" title="DOI URL">https://doi.org/10.1080/14756366.2020.1804382</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/14756366.2020.1804382&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F14756366.2020.1804382%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Enzyme%2520Inhibition%2520and%2520Medicinal%2520Chemistry%26atitle%3DIdentification%252Bof%252Bprobe-quality%252Bdegraders%252Bfor%252BPoly%252528ADP-ribose%252529%252Bpolymerase-1%252B%252528PARP-1%252529%26aulast%3DZhang%26aufirst%3DZhimin%26date%3D2020%26date%3D2020%26volume%3D35%26issue%3D1%26spage%3D1606%26epage%3D1615" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xuan  Zhang</span>, <span class="hlFld-ContribAuthor ">Dinesh  Thummuri</span>, <span class="hlFld-ContribAuthor ">Yonghan  He</span>, <span class="hlFld-ContribAuthor ">Xingui  Liu</span>, <span class="hlFld-ContribAuthor ">Peiyi  Zhang</span>, <span class="hlFld-ContribAuthor ">Daohong  Zhou</span>, <span class="hlFld-ContribAuthor ">Guangrong  Zheng</span>. </span><span class="cited-content_cbyCitation_article-title">Utilizing PROTAC technology to address the on-target platelet toxicity associated with inhibition of BCL-X
              L. </span><span class="cited-content_cbyCitation_journal-name">Chemical Communications</span><span> <strong>2019,</strong> <em>55 </em>
                                    (98)
                                     , 14765-14768. <a href="https://doi.org/10.1039/C9CC07217A" title="DOI URL">https://doi.org/10.1039/C9CC07217A</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C9CC07217A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC9CC07217A%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Communications%26atitle%3DUtilizing%252BPROTAC%252Btechnology%252Bto%252Baddress%252Bthe%252Bon-target%252Bplatelet%252Btoxicity%252Bassociated%252Bwith%252Binhibition%252Bof%252BBCL-X%252BL%26aulast%3DZhang%26aufirst%3DXuan%26date%3D2019%26date%3D2019%26volume%3D55%26issue%3D98%26spage%3D14765%26epage%3D14768" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiuyun  Sun</span>, <span class="hlFld-ContribAuthor ">Hongying  Gao</span>, <span class="hlFld-ContribAuthor ">Yiqing  Yang</span>, <span class="hlFld-ContribAuthor ">Ming  He</span>, <span class="hlFld-ContribAuthor ">Yue  Wu</span>, <span class="hlFld-ContribAuthor ">Yugang  Song</span>, <span class="hlFld-ContribAuthor ">Yan  Tong</span>, <span class="hlFld-ContribAuthor ">Yu  Rao</span>. </span><span class="cited-content_cbyCitation_article-title">PROTACs: great opportunities for academia and industry. </span><span class="cited-content_cbyCitation_journal-name">Signal Transduction and Targeted Therapy</span><span> <strong>2019,</strong> <em>4 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s41392-019-0101-6" title="DOI URL">https://doi.org/10.1038/s41392-019-0101-6</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41392-019-0101-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41392-019-0101-6%26sid%3Dliteratum%253Aachs%26jtitle%3DSignal%2520Transduction%2520and%2520Targeted%2520Therapy%26atitle%3DPROTACs%25253A%252Bgreat%252Bopportunities%252Bfor%252Bacademia%252Band%252Bindustry%26aulast%3DSun%26aufirst%3DXiuyun%26date%3D2019%26date%3D2019%26volume%3D4%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Eric  Valeur</span>, <span class="hlFld-ContribAuthor ">Frank  Narjes</span>, <span class="hlFld-ContribAuthor ">Christian  Ottmann</span>, <span class="hlFld-ContribAuthor ">Alleyn T.  Plowright</span>. </span><span class="cited-content_cbyCitation_article-title">Emerging modes-of-action in drug discovery. </span><span class="cited-content_cbyCitation_journal-name">MedChemComm</span><span> <strong>2019,</strong> <em>10 </em>
                                    (9)
                                     , 1550-1568. <a href="https://doi.org/10.1039/C9MD00263D" title="DOI URL">https://doi.org/10.1039/C9MD00263D</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C9MD00263D&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC9MD00263D%26sid%3Dliteratum%253Aachs%26jtitle%3DMedChemComm%26atitle%3DEmerging%252Bmodes-of-action%252Bin%252Bdrug%252Bdiscovery%26aulast%3DValeur%26aufirst%3DEric%26date%3D2019%26date%3D2019%26volume%3D10%26issue%3D9%26spage%3D1550%26epage%3D1568" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">John J.  Flanagan</span>, <span class="hlFld-ContribAuthor ">Taavi K.  Neklesa</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting Nuclear Receptors with PROTAC degraders. </span><span class="cited-content_cbyCitation_journal-name">Molecular and Cellular Endocrinology</span><span> <strong>2019,</strong> <em>493 </em>, 110452. <a href="https://doi.org/10.1016/j.mce.2019.110452" title="DOI URL">https://doi.org/10.1016/j.mce.2019.110452</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.mce.2019.110452&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.mce.2019.110452%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520and%2520Cellular%2520Endocrinology%26atitle%3DTargeting%252BNuclear%252BReceptors%252Bwith%252BPROTAC%252Bdegraders%26aulast%3DFlanagan%26aufirst%3DJohn%2BJ.%26date%3D2019%26volume%3D493%26spage%3D110452" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Meiyang  Xi</span>, <span class="hlFld-ContribAuthor ">Yi  Chen</span>, <span class="hlFld-ContribAuthor ">Hongyu  Yang</span>, <span class="hlFld-ContribAuthor ">Huiting  Xu</span>, <span class="hlFld-ContribAuthor ">Kui  Du</span>, <span class="hlFld-ContribAuthor ">Chunlei  Wu</span>, <span class="hlFld-ContribAuthor ">Yanfei  Xu</span>, <span class="hlFld-ContribAuthor ">Liping  Deng</span>, <span class="hlFld-ContribAuthor ">Xiang  Luo</span>, <span class="hlFld-ContribAuthor ">Lemao  Yu</span>, <span class="hlFld-ContribAuthor ">Yonghua  Wu</span>, <span class="hlFld-ContribAuthor ">Xiaozhong  Gao</span>, <span class="hlFld-ContribAuthor ">Tao  Cai</span>, <span class="hlFld-ContribAuthor ">Bin  Chen</span>, <span class="hlFld-ContribAuthor ">Runpu  Shen</span>, <span class="hlFld-ContribAuthor ">Haopeng  Sun</span>. </span><span class="cited-content_cbyCitation_article-title">Small molecule PROTACs in targeted therapy: An emerging strategy to induce protein degradation. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>174 </em>, 159-180. <a href="https://doi.org/10.1016/j.ejmech.2019.04.036" title="DOI URL">https://doi.org/10.1016/j.ejmech.2019.04.036</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2019.04.036&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2019.04.036%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DSmall%252Bmolecule%252BPROTACs%252Bin%252Btargeted%252Btherapy%25253A%252BAn%252Bemerging%252Bstrategy%252Bto%252Binduce%252Bprotein%252Bdegradation%26aulast%3DXi%26aufirst%3DMeiyang%26date%3D2019%26volume%3D174%26spage%3D159%26epage%3D180" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Miriam  Girardini</span>, <span class="hlFld-ContribAuthor ">Chiara  Maniaci</span>, <span class="hlFld-ContribAuthor ">Scott J.  Hughes</span>, <span class="hlFld-ContribAuthor ">Andrea  Testa</span>, <span class="hlFld-ContribAuthor ">Alessio  Ciulli</span>. </span><span class="cited-content_cbyCitation_article-title">Cereblon versus VHL: Hijacking E3 ligases against each other using PROTACs. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2019,</strong> <em>27 </em>
                                    (12)
                                     , 2466-2479. <a href="https://doi.org/10.1016/j.bmc.2019.02.048" title="DOI URL">https://doi.org/10.1016/j.bmc.2019.02.048</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2019.02.048&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2019.02.048%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DCereblon%252Bversus%252BVHL%25253A%252BHijacking%252BE3%252Bligases%252Bagainst%252Beach%252Bother%252Busing%252BPROTACs%26aulast%3DGirardini%26aufirst%3DMiriam%26date%3D2019%26volume%3D27%26issue%3D12%26spage%3D2466%26epage%3D2479" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Haixiang  Pei</span>, <span class="hlFld-ContribAuthor ">Yangrui  Peng</span>, <span class="hlFld-ContribAuthor ">Qiuhua  Zhao</span>, <span class="hlFld-ContribAuthor ">Yihua  Chen</span>. </span><span class="cited-content_cbyCitation_article-title">Small molecule PROTACs: an emerging technology for targeted therapy in drug discovery. </span><span class="cited-content_cbyCitation_journal-name">RSC Advances</span><span> <strong>2019,</strong> <em>9 </em>
                                    (30)
                                     , 16967-16976. <a href="https://doi.org/10.1039/C9RA03423D" title="DOI URL">https://doi.org/10.1039/C9RA03423D</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C9RA03423D&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC9RA03423D%26sid%3Dliteratum%253Aachs%26jtitle%3DRSC%2520Advances%26atitle%3DSmall%252Bmolecule%252BPROTACs%25253A%252Ban%252Bemerging%252Btechnology%252Bfor%252Btargeted%252Btherapy%252Bin%252Bdrug%252Bdiscovery%26aulast%3DPei%26aufirst%3DHaixiang%26date%3D2019%26date%3D2019%26volume%3D9%26issue%3D30%26spage%3D16967%26epage%3D16976" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/medium/jm-2018-015727_0017.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/large/jm-2018-015727_0017.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01572&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/medium/jm-2018-015727_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/large/jm-2018-015727_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Representative SERMs and SERDs.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/large/jm-2018-015727_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01572&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/medium/jm-2018-015727_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/large/jm-2018-015727_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (A) A general scheme for the design of ER degraders based upon the PROTAC concept. (B) Structures of CRBN ligand and VHL ligand. (C) Crystal structure of ERα ligand-binding domain in a complex with raloxifene (<b>1</b>) (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1ERR">1ERR</a>). (D) Predicted binding models of <i>N</i>,<i>N</i>-diethylamino analogue of raloxifene in complex with ERα LBD. Hydrogen bonds are depicted by yellow dashed lines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/large/jm-2018-015727_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01572&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/medium/jm-2018-015727_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/large/jm-2018-015727_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Degradation of ER protein by PROTACs. (A) Structures of synthetic putative ER degraders. (B) Western blotting analysis of ER protein in MCF-7 cells treated with putative PROTAC ER degraders and control compounds. Cells were treated with different compounds for 4 h, and whole cell lysates were then analyzed by Western blotting to examine the level of ER protein. The membranes were stripped and reblotted for GAPDH as the loading control. The numbers below the panel represent the ERα/GADPH ratio normalized with the DMSO control at 100.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/large/jm-2018-015727_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01572&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/medium/jm-2018-015727_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/large/jm-2018-015727_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Optimization of linker length with linear aliphatic linkers. (A) Structures of ER degraders. (B) Western blotting analysis of ER protein in the MCF-7 cells treated with indicated compounds at 1, 10, and 100 nM. MCF-7 cells were treated with different compounds for 4 h and whole cell lysates were analyzed by Western blotting to examine the level of ER protein. The membranes were stripped and reblotted for GAPDH as the loading control. The numbers below the panel represent the ERα/GADPH ratio normalized with the DMSO control at 100.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/large/jm-2018-015727_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01572&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/medium/jm-2018-015727_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/large/jm-2018-015727_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Optimization of N-substituent groups (A) Structures of ER degraders. (B) Western blotting analysis of ER protein in the MCF-7 cells treated with indicated compounds at 1, 10, and 100 nM. MCF-7 cells were treated with different compounds for 4 h, and whole cell lysates were analyzed by Western blotting to examine the level of ER protein. The membranes were stripped and reblotted for GAPDH as the loading control. The numbers below the panel represent the ERα/GADPH ratio normalized with the DMSO control at 100.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/large/jm-2018-015727_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01572&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/medium/jm-2018-015727_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/large/jm-2018-015727_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Optimization of the linker composition. (A) Structures of the ER degraders. (B) Western blotting analysis of ER protein in the MCF-7 cells treated with indicated compounds at 1, 10, and 100 nM. MCF-7 cells were treated with different compounds for 4 h, and whole cell lysates were analyzed by Western blotting to examine the level of ER protein.The membranes were stripped and reblotted for GAPDH as the loading control. The numbers below the panel represent the ERα/GADPH ratio normalized with the DMSO control at 100.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/large/jm-2018-015727_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01572&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/medium/jm-2018-015727_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/large/jm-2018-015727_0007.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. SAR study for ER ligands. (A) Structures of ER degraders. (B) Western blotting analysis of ER protein in the MCF-7 cells treated with indicated compounds at 1, 10, and 100 nM. MCF-7 cells were treated with different compounds for 4 h, and whole cell lysates were analyzed by Western blotting to examine the level of ER protein. The membranes were stripped and reblotted for GAPDH as the loading control. The numbers below the panel represent the ERα/GADPH ratio normalized with the DMSO control at 100.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/large/jm-2018-015727_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01572&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/medium/jm-2018-015727_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/large/jm-2018-015727_0008.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. SAR study for VHL ligands. (A) Structures of ER degraders and binding affinities of relative VHL ligands against VHL protein by fluorescence polarization (FP) assay. (B) Western blotting analysis of ER protein in the MCF-7 cells treated with indicated compounds at 1, 10, and 100 nM. MCF-7 cells were treated with different compounds for 4 h, and whole cell lysates were analyzed by Western blotting to examine the level of ER protein. The membranes were stripped and reblotted for GAPDH as the loading control. The numbers below the panel represent the ERα/GADPH ratio normalized with the DMSO control at 100.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/large/jm-2018-015727_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01572&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/medium/jm-2018-015727_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/large/jm-2018-015727_0009.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Further characterization of compound <b>32</b> (ERD-308). (A) ERα degradation dose-response for compound <b>32</b> with a treatment time of 4 h in MCF-7 cells. ER protein was examined by Western blotting and ER protein level was quantified by densitometry and normalized to the corresponding density of GAPDH protein. (B) ERα degradation dose-response for compound <b>32</b> with a treatment time of 4 h in T47D cells. ER protein was examined by Western blotting and ER protein level was quantified by densitometry and normalized to the corresponding density of GAPDH protein. (C) Time course of ERα degradation by compound <b>32</b> (30 nM) and fulvestrant (30 nM) in the MCF-7 cells. (D) Time course of ERα degradation by compound <b>32</b> (30 nM) and fulvestrant (30 nM) in the T47D cells.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/large/jm-2018-015727_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01572&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig10" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/medium/jm-2018-015727_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/large/jm-2018-015727_0010.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. Degradation is dependent on VHL, ER, and proteasome. (A) MCF-7 cells were pretreated with VHL ligand <b>11</b> (1 μM), or ER ligand raloxifene (<b>1</b>) (1 μm), or the proteasome inhibitor carfilzomib (1 μM) for 2 h, followed by treatment with DMSO or compound <b>32</b> (30 nM) for 4 h. Then whole-cell lysates were analyzed by Western blotting. (B) MCF-7 cells were pretreated with VHL ligand <b>11</b> (+, 0.5 μM; ++, 1 μM; +++, 5 μM; ++++, 10 μM) for 2 h, followed by treatment with DMSO or compound <b>32</b> (30 nM) for 4 h. Then whole-cell lysates were analyzed by Western blotting.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/large/jm-2018-015727_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01572&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig11" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 11</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/medium/jm-2018-015727_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/large/jm-2018-015727_0011.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 11. Cellular effects of compound <b>32</b> (ERD-308) in MCF-7 cells. (A) Cell viability assay. Cells were treated with indicated doses for 4 days. Cell viability was determined by colorimetric WST-8 assay. GI<sub>50</sub> values were calculated using the GraphPad Prism 7 software. (B) Crystal violet staining. Cells were treated with indicated doses for 5 days, and then fixed and stained with crystal violet. (C,D) qRT-PCR analysis for pGR gene and GREB1 mRNA expression in MCF-7 cells. Cells were treated with DMSO, raloxifene (<b>1</b>), fulvestrant (<b>5</b>), or compound <b>32</b> at indicated doses for 8 h. Multiple student’s <i>t</i> tests were conducted using the Holm–Sidak method. * Significant difference between the compound treated group and the vehicle treated control group (DMSO). <sup>&</sup> Significant difference between the compound treated group and the ER inhibitor (raloxifene) treated group. <sup>#</sup> Significant difference between fulvestrant and PROTAC degrader treated groups.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/large/jm-2018-015727_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01572&amp;id=fig11"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/medium/jm-2018-015727_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/large/jm-2018-015727_0012.jpeg" id="rightTab-gr12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of the Key Intermediate <b>53</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/large/jm-2018-015727_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01572&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) oxalyl chloride, DMF, DCM, 0 °C to RT, 1 h; (b) 6-methoxy-2-(4-methoxyphenyl)benzo[<i>b</i>]thiophene, AlCl<sub>3</sub>, DCM, 0 °C to RT, 1 h; (c) NaOAc, EtOH/H<sub>2</sub>O, 80 °C, 12 h; (d) 1,2-dibromoethane, Cs<sub>2</sub>CO<sub>3</sub>, MeCN, reflux, 12 h; (e) EtNH<sub>2</sub>, DIPEA, DMF, 80 °C, 12 h; (f) BBr<sub>3</sub>, DCM, 0 °C to RT, 1 h.</p></p></figure><figure data-id="sch2" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/medium/jm-2018-015727_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/large/jm-2018-015727_0013.jpeg" id="rightTab-gr13" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of the Key Intermediate <b>58</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/large/jm-2018-015727_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01572&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) Boc<sub>2</sub>O, NaHCO<sub>3</sub>, EtOAc/H<sub>2</sub>O, 2 h; (b) 4-methylthiazole, Pd(OAc)<sub>2</sub>, KOAc, DMA, 90 °C, 12 h; (c) 4 N HCl in dioxane/MeOH, RT, 12 h; (d) (2<i>S</i>,4<i>R</i>)-1-(<i>tert</i>-butoxycarbonyl)-4-hydroxypyrrolidine-2-carboxylic acid, HATU, DIPEA, DMF, 0 °C to RT, 12 h; (e) 4 N HCl in dioxane/MeOH, RT, 12 h; (f) (<i>S</i>)-1-(<i>tert</i>-butoxycarbonyl)piperidine-2-carboxylic acid, HATU, DIPEA, DMF, 0 °C to RT, 12 h; (g) 4 N HCl in dioxane/MeOH, RT, 12 h.</p></p></figure><figure data-id="sch3" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/medium/jm-2018-015727_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/large/jm-2018-015727_0014.jpeg" id="rightTab-gr14" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Compounds <b>12</b> and <b>13</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/large/jm-2018-015727_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01572&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) MsCl, TEA, DCM, 0 °C to RT, 1 h; (b) <b>53</b>, DIPEA, DMF, 80 °C, 12 h; (c) 3-(4-iodo-1-oxoisoindolin-2-yl)piperidine-2,6-dione, Pd(PPh<sub>3</sub>)Cl<sub>2</sub>, CuI, DMF/TEA, 80 °C, 1 h.</p></p></figure><figure data-id="sch4" data-index="15" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/medium/jm-2018-015727_0015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/large/jm-2018-015727_0015.jpeg" id="rightTab-gr15" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of Compounds <b>14</b>–<b>21</b>, <b>30</b>–<b>37</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/large/jm-2018-015727_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01572&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) TsCl or MsCl, TEA, DCM, 0 °C to RT, 1 h; (b) TFAA, <i>tert</i>-BuOH, DCM, 0 °C to RT, 12 h; (c) <b>53</b>, DIPEA, DMF, 80 °C, 12 h; (d) TFA/DCM, 0 °C to RT, 6 h; (e) <b>58</b>, HATU, DIPEA, DMF, RT, 12 h.</p></p></figure><figure data-id="sch5" data-index="16" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/medium/jm-2018-015727_0016.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/large/jm-2018-015727_0016.jpeg" id="rightTab-gr16" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Synthesis of Compounds <b>22</b>–<b>29</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01572/20190211/images/large/jm-2018-015727_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01572&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) 1,2-dibromoethane, Cs<sub>2</sub>CO<sub>3</sub>, MeCN, reflux, 12 h; (b) RNH<sub>2</sub>, K<sub>2</sub>CO<sub>3</sub>, MeCN, 80 °C; (c) <b>65</b>, K<sub>2</sub>CO<sub>3</sub>, DMF, 80 °C, 12 h; (d) BBr<sub>3</sub>, DCM, 0 °C to RT, 1 h; (e) <b>58</b>, HATU, DIPEA, DMF, RT, 12 h.</p></p></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i68">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_71411" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_71411" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 59 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tong, C. W. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, W. C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">To, K. K. W.</span></span> <span> </span><span class="NLM_article-title">Recent advances in the treatment of breast cancer</span>. <i>Front. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">227</span>, <span class="refDoi"> DOI: 10.3389/fonc.2018.00227</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=10.3389%2Ffonc.2018.00227" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=29963498" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A280%3ADC%252BB3c%252FjvVylsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2018&pages=227&author=C.+W.+S.+Tongauthor=M.+Wuauthor=W.+C.+S.+Choauthor=K.+K.+W.+To&title=Recent+advances+in+the+treatment+of+breast+cancer&doi=10.3389%2Ffonc.2018.00227"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Recent Advances in the Treatment of Breast Cancer</span></div><div class="casAuthors">Tong Christy W S; Wu Mingxia; To Kenneth K W; Cho William C S</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in oncology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">227</span>
        ISSN:<span class="NLM_cas:issn">2234-943X</span>.
    </div><div class="casAbstract">Breast cancer (BC) is the most common malignancy in women.  It is classified into a few major molecular subtypes according to hormone and growth factor receptor expression.  Over the past few years, substantial advances have been made in the discovery of new drugs for treating BC.  Improved understanding of the biologic heterogeneity of BC has allowed the development of more effective and individualized approach to treatment.  In this review, we provide an update about the current treatment strategy and discuss the various emerging novel therapies for the major molecular subtypes of BC.  A brief account of the clinical development of inhibitors of poly(ADP-ribose) polymerase, cyclin-dependent kinases 4 and 6, phosphatidylinositol 3-kinase/protein kinase B/mammalian target of rapamycin pathway, histone deacetylation, multi-targeting tyrosine kinases, and immune checkpoints for personalized treatment of BC is included.  However, no targeted drug has been approved for the most aggressive subtype-triple negative breast cancer (TNBC).  Thus, we discuss the heterogeneity of TNBC and how molecular subtyping of TNBC may help drug discovery for this deadly disease.  The emergence of drug resistance also poses threat to the successful development of targeted therapy in various molecular subtypes of BC.  New clinical trials should incorporate advanced methods to identify changes induced by drug treatment, which may be associated with the upregulation of compensatory signaling pathways in drug resistant cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS9aYrYWlne0BJ5E29YNGvQfW6udTcc2eau6S9FToVgFrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3c%252FjvVylsw%253D%253D&md5=a116dcf63d5640023aac066b920bdc5f</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.3389%2Ffonc.2018.00227&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffonc.2018.00227%26sid%3Dliteratum%253Aachs%26aulast%3DTong%26aufirst%3DC.%2BW.%2BS.%26aulast%3DWu%26aufirst%3DM.%26aulast%3DCho%26aufirst%3DW.%2BC.%2BS.%26aulast%3DTo%26aufirst%3DK.%2BK.%2BW.%26atitle%3DRecent%2520advances%2520in%2520the%2520treatment%2520of%2520breast%2520cancer%26jtitle%3DFront.%2520Oncol.%26date%3D2018%26volume%3D8%26spage%3D227%26doi%3D10.3389%2Ffonc.2018.00227" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, W. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katki, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenberg, P. S.</span></span> <span> </span><span class="NLM_article-title">Incidence of breast cancer in the United States: current and future trends</span>. <i>J. Natl. Cancer Inst.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>103</i></span>,  <span class="NLM_fpage">1397</span>– <span class="NLM_lpage">1402</span>, <span class="refDoi"> DOI: 10.1093/jnci/djr257</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=10.1093%2Fjnci%2Fdjr257" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=21753181" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A280%3ADC%252BC3MfltVSgtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=2011&pages=1397-1402&author=W.+F.+Andersonauthor=H.+A.+Katkiauthor=P.+S.+Rosenberg&title=Incidence+of+breast+cancer+in+the+United+States%3A+current+and+future+trends&doi=10.1093%2Fjnci%2Fdjr257"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Incidence of breast cancer in the United States: current and future trends</span></div><div class="casAuthors">Anderson William F; Katki Hormuzd A; Rosenberg Philip S</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the National Cancer Institute</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">103</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">1397-402</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  The incidence of breast cancer increased in the United States until circa 2000 then decreased, mostly among women with estrogen receptor (ER)-positive cancers.  Time trends provide important clues for cancer etiology and prevention; however, the observed trends of ER-positive and ER-negative breast cancers can be biased by missing ER data.  METHODS:  We developed a simple imputation method to correct invasive female breast cancer incidence for missing or unknown ER expression, using nationally representative data from the National Cancer Institute's Surveillance, Epidemiology, and End Results Program during 1980-2008, including 588,720 invasive female breast cancer patients with 471,336,233 woman-years of follow-up.  Corrected rates of ER-positive and ER-negative breast cancers were used to calculate age-standardized incidence rates, estimated annual percentage changes, and projections derived from age-period-cohort models.  RESULTS:  The recent decrease in the incidence of breast cancer overall stabilized near 200 per 100,000 woman-years by 2007-2008, reflecting a transient decrease in ER-positive cancers and a steady decrease in ER-negative cancers.  The projected incidence rate for breast cancer overall through the year 2016 was similar to the incidence rate during 2007-2008.  In contrast, rates of ER-positive breast cancers were projected to increase 5.3% (95% confidence interval = 5.2% to 5.4%), whereas rates of ER-negative breast cancers were projected to decrease 11.4% (95% confidence interval = 11.3% to 11.6%) during 2009-2016.  CONCLUSION:  Recent changes in breast cancer incidence overall reflect the superimposition of divergent trends in ER-positive and ER-negative cancers.  If current trends continue, the incidence of ER-positive breast cancers will increase, the incidence of ER-negative breast cancers will continue to decrease, and the incidence of breast cancer overall will remain similar to its current level.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ2E9JRP4-GvK__Jf7-eMvJfW6udTcc2eau6S9FToVgFrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3MfltVSgtg%253D%253D&md5=c4468ec6b42936d658f0274561120a6c</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1093%2Fjnci%2Fdjr257&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjnci%252Fdjr257%26sid%3Dliteratum%253Aachs%26aulast%3DAnderson%26aufirst%3DW.%2BF.%26aulast%3DKatki%26aufirst%3DH.%2BA.%26aulast%3DRosenberg%26aufirst%3DP.%2BS.%26atitle%3DIncidence%2520of%2520breast%2520cancer%2520in%2520the%2520United%2520States%253A%2520current%2520and%2520future%2520trends%26jtitle%3DJ.%2520Natl.%2520Cancer%2520Inst.%26date%3D2011%26volume%3D103%26spage%3D1397%26epage%3D1402%26doi%3D10.1093%2Fjnci%2Fdjr257" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nilsson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koehler, K. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gustafsson, J. A.</span></span> <span> </span><span class="NLM_article-title">Development of subtype-selective oestrogen receptor-based therapeutics</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">778</span>– <span class="NLM_lpage">792</span>, <span class="refDoi"> DOI: 10.1038/nrd3551</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=10.1038%2Fnrd3551" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2011&pages=778-792&author=S.+Nilssonauthor=K.+F.+Koehlerauthor=J.+A.+Gustafsson&title=Development+of+subtype-selective+oestrogen+receptor-based+therapeutics&doi=10.1038%2Fnrd3551"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnrd3551&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3551%26sid%3Dliteratum%253Aachs%26aulast%3DNilsson%26aufirst%3DS.%26aulast%3DKoehler%26aufirst%3DK.%2BF.%26aulast%3DGustafsson%26aufirst%3DJ.%2BA.%26atitle%3DDevelopment%2520of%2520subtype-selective%2520oestrogen%2520receptor-based%2520therapeutics%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2011%26volume%3D10%26spage%3D778%26epage%3D792%26doi%3D10.1038%2Fnrd3551" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jordan, V. C.</span></span> <span> </span><span class="NLM_article-title">Tamoxifen: a most unlikely pioneering medicine</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">205</span>– <span class="NLM_lpage">213</span>, <span class="refDoi"> DOI: 10.1038/nrd1031</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=10.1038%2Fnrd1031" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=12612646" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A528%3ADC%252BD3sXhsFKrtL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2003&pages=205-213&author=V.+C.+Jordan&title=Tamoxifen%3A+a+most+unlikely+pioneering+medicine&doi=10.1038%2Fnrd1031"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Tamoxifen: A most unlikely pioneering medicine</span></div><div class="casAuthors">Jordan, V. Craig</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">205-213</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  For >25 yr, tamoxifen has been the gold std. for the endocrine treatment of all stages of estrogen receptor-pos. breast cancer, and the World Health Organization lists tamoxifen as an essential drug for the treatment of breast cancer.  It is estd. that >400,000 women are alive today as a result of tamoxifen therapy, and millions more have benefited from palliation and extended disease-free survival.  Interestingly, tamoxifen also became the first cancer chemopreventive approved by the Food and Drug Administration (FDA) for the redn. of breast-cancer incidence in both pre- and postmenopausal women at high risk.  However, 40 yr ago, it was hard to imagine that a nontoxic targeted treatment for breast cancer could be developed at all.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9q3QlfhnxLrVg90H21EOLACvtfcHk0lgqq6bPu_whOQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXhsFKrtL0%253D&md5=f5574615d30cb7dab87cf9d812104a83</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fnrd1031&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd1031%26sid%3Dliteratum%253Aachs%26aulast%3DJordan%26aufirst%3DV.%2BC.%26atitle%3DTamoxifen%253A%2520a%2520most%2520unlikely%2520pioneering%2520medicine%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2003%26volume%3D2%26spage%3D205%26epage%3D213%26doi%3D10.1038%2Fnrd1031" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Das, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crockett, J. C.</span></span> <span> </span><span class="NLM_article-title">Osteoporosis - a current view of pharmacological prevention and treatment</span>. <i>Drug Des., Dev. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">435</span>– <span class="NLM_lpage">448</span>, <span class="refDoi"> DOI: 10.2147/DDDT.S31504</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=10.2147%2FDDDT.S31504" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=23807838" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A528%3ADC%252BC3sXpslyis7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2013&pages=435-448&author=S.+Dasauthor=J.+C.+Crockett&title=Osteoporosis+-+a+current+view+of+pharmacological+prevention+and+treatment&doi=10.2147%2FDDDT.S31504"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Osteoporosis - a current view of pharmacological prevention and treatment</span></div><div class="casAuthors">Das, Subhajit; Crockett, Julie C.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Design, Development and Therapy</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">435-448</span>CODEN:
                <span class="NLM_cas:coden">DDDTAQ</span>;
        ISSN:<span class="NLM_cas:issn">1177-8881</span>.
    
            (<span class="NLM_cas:orgname">Dove Medical Press Ltd.</span>)
        </div><div class="casAbstract">A review.  Postmenopausal osteoporosis is the most common bone disease, assocd. with low bone mineral d. (BMD) and pathol. fractures which lead to significant morbidity.  It is defined clin. by a BMD of 2.5 std. deviations or more below the young female adult mean (T-score = -2.5).  Osteoporosis was a huge global problem both socially and economically - in the UK alone, in 2011 £6 million per day was spent on treatment and social care of the 230,000 osteoporotic fracture patients - and therefore viable preventative and therapeutic approaches are key to managing this problem within the aging population of today.  One of the main issues surrounding the potential of osteoporosis management is diagnosing patients at risk before they develop a fracture.  We discuss the current and future possibilities for identifying susceptible patients, from fracture risk assessment to shape modeling and in relation to the high heritability of osteoporosis now that a plethora of genes have been assocd. with low BMD and osteoporotic fracture.  This review highlights the current therapeutics in clin. use (including bisphosphonates, anti-RANKL [receptor activator of NF-κB ligand], intermittent low dose parathyroid hormone, and strontium ranelate) and some of those in development (anti-sclerostin antibodies and cathepsin K inhibitors).  By highlighting the intimate relationship between the activities of bone forming (osteoblasts) and bone-resorbing (osteoclasts) cells, we include an overview and comparison of the mol. mechanisms exploited in each therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbQ-qJlOT4Z7Vg90H21EOLACvtfcHk0lixCPjSlK36vg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXpslyis7k%253D&md5=dd22c21c6d0f305321ebdc1f32051000</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.2147%2FDDDT.S31504&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2147%252FDDDT.S31504%26sid%3Dliteratum%253Aachs%26aulast%3DDas%26aufirst%3DS.%26aulast%3DCrockett%26aufirst%3DJ.%2BC.%26atitle%3DOsteoporosis%2520-%2520a%2520current%2520view%2520of%2520pharmacological%2520prevention%2520and%2520treatment%26jtitle%3DDrug%2520Des.%252C%2520Dev.%2520Ther.%26date%3D2013%26volume%3D7%26spage%3D435%26epage%3D448%26doi%3D10.2147%2FDDDT.S31504" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">De
Marchi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foekens, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Umar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martens, J. W.</span></span> <span> </span><span class="NLM_article-title">Endocrine therapy resistance in estrogen receptor (ER)-positive breast cancer</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">1181</span>– <span class="NLM_lpage">1188</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2016.05.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=10.1016%2Fj.drudis.2016.05.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=27233379" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A528%3ADC%252BC28Xoslagsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2016&pages=1181-1188&author=T.+De%0AMarchiauthor=J.+A.+Foekensauthor=A.+Umarauthor=J.+W.+Martens&title=Endocrine+therapy+resistance+in+estrogen+receptor+%28ER%29-positive+breast+cancer&doi=10.1016%2Fj.drudis.2016.05.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Endocrine therapy resistance in estrogen receptor (ER)-positive breast cancer</span></div><div class="casAuthors">De Marchi, Tommaso; Foekens, John A.; Umar, Arzu; Martens, John W. M.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1181-1188</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Estrogen receptor (ER)-pos. breast cancer represents the majority (∼70%) of all breast malignancies.  In this subgroup of breast cancers, endocrine therapies are effective both in the adjuvant and recurrent settings, although resistance remains a major issue.  Several high-throughput approaches have been used to elucidate mechanisms of resistance and to derive potential predictive markers or alternative therapies.  In this review, we cover the state-of-the-art of endocrine-resistance biomarker discovery with regard to the latest technol. developments, and discuss current opportunities and restrictions for their implementation into a clin. setting.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovT4-cqEBi7bVg90H21EOLACvtfcHk0lixCPjSlK36vg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xoslagsb8%253D&md5=133921c6a09077c5c2f1bcab1c565c1a</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2016.05.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2016.05.012%26sid%3Dliteratum%253Aachs%26aulast%3DDe%2BMarchi%26aufirst%3DT.%26aulast%3DFoekens%26aufirst%3DJ.%2BA.%26aulast%3DUmar%26aufirst%3DA.%26aulast%3DMartens%26aufirst%3DJ.%2BW.%26atitle%3DEndocrine%2520therapy%2520resistance%2520in%2520estrogen%2520receptor%2520%2528ER%2529-positive%2520breast%2520cancer%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2016%26volume%3D21%26spage%3D1181%26epage%3D1188%26doi%3D10.1016%2Fj.drudis.2016.05.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">AlFakeeh, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brezden-Masley, C.</span></span> <span> </span><span class="NLM_article-title">Overcoming endocrine resistance in hormone receptor-positive breast cancer</span>. <i>Curr. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">18</span>– <span class="NLM_lpage">27</span>, <span class="refDoi"> DOI: 10.3747/co.25.3752</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=10.3747%2Fco.25.3752" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=18-27&author=A.+AlFakeehauthor=C.+Brezden-Masley&title=Overcoming+endocrine+resistance+in+hormone+receptor-positive+breast+cancer&doi=10.3747%2Fco.25.3752"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.3747%2Fco.25.3752&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3747%252Fco.25.3752%26sid%3Dliteratum%253Aachs%26aulast%3DAlFakeeh%26aufirst%3DA.%26aulast%3DBrezden-Masley%26aufirst%3DC.%26atitle%3DOvercoming%2520endocrine%2520resistance%2520in%2520hormone%2520receptor-positive%2520breast%2520cancer%26jtitle%3DCurr.%2520Oncol.%26date%3D2018%26volume%3D25%26spage%3D18%26epage%3D27%26doi%3D10.3747%2Fco.25.3752" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Martin, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ribas, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simigdala, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuster, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pancholi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tenev, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gellert, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buluwela, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harrod, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thornhill, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nikitorowicz-Buniak, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhamra, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turgeon, M. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poulogiannis, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martins, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hills, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia-Murillas, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fribbens, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patani, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sikora, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turner, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwart, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oesterreich, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carroll, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ali, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dowsett, M.</span></span> <span> </span><span class="NLM_article-title">Discovery of naturally occurring ESR1 mutations in breast cancer cell lines modelling endocrine resistance</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1865</span>, <span class="refDoi"> DOI: 10.1038/s41467-017-01864-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=10.1038%2Fs41467-017-01864-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=29192207" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A280%3ADC%252BC1M3ms1CqtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=1865&author=L.+A.+Martinauthor=R.+Ribasauthor=N.+Simigdalaauthor=E.+Schusterauthor=S.+Pancholiauthor=T.+Tenevauthor=P.+Gellertauthor=L.+Buluwelaauthor=A.+Harrodauthor=A.+Thornhillauthor=J.+Nikitorowicz-Buniakauthor=A.+Bhamraauthor=M.+O.+Turgeonauthor=G.+Poulogiannisauthor=Q.+Gaoauthor=V.+Martinsauthor=M.+Hillsauthor=I.+Garcia-Murillasauthor=C.+Fribbensauthor=N.+Pataniauthor=Z.+Liauthor=M.+J.+Sikoraauthor=N.+Turnerauthor=W.+Zwartauthor=S.+Oesterreichauthor=J.+Carrollauthor=S.+Aliauthor=M.+Dowsett&title=Discovery+of+naturally+occurring+ESR1+mutations+in+breast+cancer+cell+lines+modelling+endocrine+resistance&doi=10.1038%2Fs41467-017-01864-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of naturally occurring ESR1 mutations in breast cancer cell lines modelling endocrine resistance</span></div><div class="casAuthors">Martin Lesley-Ann; Ribas Ricardo; Simigdala Nikiana; Schuster Eugene; Pancholi Sunil; Tenev Tencho; Gellert Pascal; Nikitorowicz-Buniak Joanna; Gao Qiong; Garcia-Murillas Isaac; Fribbens Charlotte; Patani Neill; Turner Nicholas; Dowsett Mitch; Buluwela Laki; Harrod Alison; Ali Simak; Thornhill Allan; Bhamra Amandeep; Turgeon Marc-Olivier; Poulogiannis George; Poulogiannis George; Martins Vera; Hills Margaret; Dowsett Mitch; Li Zheqi; Sikora Matthew J; Oesterreich Steffi; Zwart Wilbert; Carroll Jason</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1865</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Resistance to endocrine therapy remains a major clinical problem in breast cancer.  Genetic studies highlight the potential role of estrogen receptor-α (ESR1) mutations, which show increased prevalence in the metastatic, endocrine-resistant setting.  No naturally occurring ESR1 mutations have been reported in in vitro models of BC either before or after the acquisition of endocrine resistance making functional consequences difficult to study.  We report the first discovery of naturally occurring ESR1 (Y537C) and ESR1 (Y537S) mutations in MCF7 and SUM44 ESR1-positive cell lines after acquisition of resistance to long-term-estrogen-deprivation (LTED) and subsequent resistance to fulvestrant (ICIR).  Mutations were enriched with time, impacted on ESR1 binding to the genome and altered the ESR1 interactome.  The results highlight the importance and functional consequence of these mutations and provide an important resource for studying endocrine resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTMmRurTV1xdhBrI9DTGjC0fW6udTcc2ea-7E9xMEhctrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M3ms1CqtA%253D%253D&md5=0ff681e49e4de73dd6822b02fe85e096</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fs41467-017-01864-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-017-01864-y%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DL.%2BA.%26aulast%3DRibas%26aufirst%3DR.%26aulast%3DSimigdala%26aufirst%3DN.%26aulast%3DSchuster%26aufirst%3DE.%26aulast%3DPancholi%26aufirst%3DS.%26aulast%3DTenev%26aufirst%3DT.%26aulast%3DGellert%26aufirst%3DP.%26aulast%3DBuluwela%26aufirst%3DL.%26aulast%3DHarrod%26aufirst%3DA.%26aulast%3DThornhill%26aufirst%3DA.%26aulast%3DNikitorowicz-Buniak%26aufirst%3DJ.%26aulast%3DBhamra%26aufirst%3DA.%26aulast%3DTurgeon%26aufirst%3DM.%2BO.%26aulast%3DPoulogiannis%26aufirst%3DG.%26aulast%3DGao%26aufirst%3DQ.%26aulast%3DMartins%26aufirst%3DV.%26aulast%3DHills%26aufirst%3DM.%26aulast%3DGarcia-Murillas%26aufirst%3DI.%26aulast%3DFribbens%26aufirst%3DC.%26aulast%3DPatani%26aufirst%3DN.%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DSikora%26aufirst%3DM.%2BJ.%26aulast%3DTurner%26aufirst%3DN.%26aulast%3DZwart%26aufirst%3DW.%26aulast%3DOesterreich%26aufirst%3DS.%26aulast%3DCarroll%26aufirst%3DJ.%26aulast%3DAli%26aufirst%3DS.%26aulast%3DDowsett%26aufirst%3DM.%26atitle%3DDiscovery%2520of%2520naturally%2520occurring%2520ESR1%2520mutations%2520in%2520breast%2520cancer%2520cell%2520lines%2520modelling%2520endocrine%2520resistance%26jtitle%3DNat.%2520Commun.%26date%3D2017%26volume%3D8%26spage%3D1865%26doi%3D10.1038%2Fs41467-017-01864-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nardone, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Angelis, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trivedi, M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osborne, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiff, R.</span></span> <span> </span><span class="NLM_article-title">The changing role of ER in endocrine resistance</span>. <i>Breast</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">S60</span>– <span class="NLM_lpage">S66</span>, <span class="refDoi"> DOI: 10.1016/j.breast.2015.07.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=10.1016%2Fj.breast.2015.07.015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=26271713" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2015&pages=S60-S66&author=A.+Nardoneauthor=C.+De+Angelisauthor=M.+V.+Trivediauthor=C.+K.+Osborneauthor=R.+Schiff&title=The+changing+role+of+ER+in+endocrine+resistance&doi=10.1016%2Fj.breast.2015.07.015"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.breast.2015.07.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.breast.2015.07.015%26sid%3Dliteratum%253Aachs%26aulast%3DNardone%26aufirst%3DA.%26aulast%3DDe%2BAngelis%26aufirst%3DC.%26aulast%3DTrivedi%26aufirst%3DM.%2BV.%26aulast%3DOsborne%26aufirst%3DC.%2BK.%26aulast%3DSchiff%26aufirst%3DR.%26atitle%3DThe%2520changing%2520role%2520of%2520ER%2520in%2520endocrine%2520resistance%26jtitle%3DBreast%26date%3D2015%26volume%3D24%26spage%3DS60%26epage%3DS66%26doi%3D10.1016%2Fj.breast.2015.07.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Robertson, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harrison, M.</span></span> <span> </span><span class="NLM_article-title">Fulvestrant: pharmacokinetics and pharmacology</span>. <i>Br. J. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>90</i></span>,  <span class="NLM_fpage">S7</span>– <span class="NLM_lpage">S10</span>, <span class="refDoi"> DOI: 10.1038/sj.bjc.6601630</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=10.1038%2Fsj.bjc.6601630" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=15094758" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A528%3ADC%252BD2cXjtFaktbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=90&publication_year=2004&pages=S7-S10&author=J.+F.+Robertsonauthor=M.+Harrison&title=Fulvestrant%3A+pharmacokinetics+and+pharmacology&doi=10.1038%2Fsj.bjc.6601630"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Fulvestrant: pharmacokinetics and pharmacology</span></div><div class="casAuthors">Robertson, J. F. R.; Harrison, M.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Cancer</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">90</span>
        (<span class="NLM_cas:issue">Suppl. 1</span>),
    <span class="NLM_cas:pages">S7-S10</span>CODEN:
                <span class="NLM_cas:coden">BJCAAI</span>;
        ISSN:<span class="NLM_cas:issn">0007-0920</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Fulvestrant is a new type of estrogen receptor (ER) antagonist with no agonist activity and a novel pharmacol. profile.  Fulvestrant has been shown to significantly reduce cellular levels of the ER and progesterone receptor in both preclin. studies and in clin. trials of postmenopausal women with primary breast cancer.  This paper reviews the pharmacokinetics and metab. of fulvestrant, which support the rationale for drug delivery as a single, once-monthly i.m. injection, and show that this agent has minimal potential to be the subject, or cause, of significant cytochrome P 450-mediated drug interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqThGy3GHRDRrVg90H21EOLACvtfcHk0lhoIxs5xSLpbg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXjtFaktbc%253D&md5=d4a14a84081b408e9c60a3e89968a70a</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fsj.bjc.6601630&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.bjc.6601630%26sid%3Dliteratum%253Aachs%26aulast%3DRobertson%26aufirst%3DJ.%2BF.%26aulast%3DHarrison%26aufirst%3DM.%26atitle%3DFulvestrant%253A%2520pharmacokinetics%2520and%2520pharmacology%26jtitle%3DBr.%2520J.%2520Cancer%26date%3D2004%26volume%3D90%26spage%3DS7%26epage%3DS10%26doi%3D10.1038%2Fsj.bjc.6601630" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Osborne, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wakeling, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicholson, R. I.</span></span> <span> </span><span class="NLM_article-title">Fulvestrant: an oestrogen receptor antagonist with a novel mechanism of action</span>. <i>Br. J. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>90</i></span>,  <span class="NLM_fpage">S2</span>– <span class="NLM_lpage">S6</span>, <span class="refDoi"> DOI: 10.1038/sj.bjc.6601629</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=10.1038%2Fsj.bjc.6601629" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=15094757" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A528%3ADC%252BD2cXjtFaktbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=90&publication_year=2004&pages=S2-S6&author=C.+K.+Osborneauthor=A.+Wakelingauthor=R.+I.+Nicholson&title=Fulvestrant%3A+an+oestrogen+receptor+antagonist+with+a+novel+mechanism+of+action&doi=10.1038%2Fsj.bjc.6601629"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Fulvestrant: an estrogen receptor antagonist with a novel mechanism of action</span></div><div class="casAuthors">Osborne, C. K.; Wakeling, A.; Nicholson, R. I.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Cancer</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">90</span>
        (<span class="NLM_cas:issue">Suppl. 1</span>),
    <span class="NLM_cas:pages">S2-S6</span>CODEN:
                <span class="NLM_cas:coden">BJCAAI</span>;
        ISSN:<span class="NLM_cas:issn">0007-0920</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Due to their favorable tolerability profiles, endocrine therapies have long been considered the treatment of choice for hormone-sensitive metastatic breast cancer.  However, the estrogen agonist effects of the available selective estrogen receptor modulators, such as tamoxifen, and the development of cross-resistance between endocrine therapies with similar modes of action have led to the need for new treatments that act through different mechanisms.  Fulvestrant (Faslodex') is the first of a new type of endocrine treatment-an estrogen receptor (ER) antagonist that downregulates the ER and has no agonist effects.  This article provides an overview of the current understanding of ER signaling and illustrates the unique mode of action of fulvestrant.  Preclin. and clin. study data are presented in support of the novel mechanism of action of this new type of ER antagonist.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8myn0Hcy0CrVg90H21EOLACvtfcHk0lhoIxs5xSLpbg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXjtFaktbY%253D&md5=291a738f1d67db894e83d88ddfa000e4</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fsj.bjc.6601629&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.bjc.6601629%26sid%3Dliteratum%253Aachs%26aulast%3DOsborne%26aufirst%3DC.%2BK.%26aulast%3DWakeling%26aufirst%3DA.%26aulast%3DNicholson%26aufirst%3DR.%2BI.%26atitle%3DFulvestrant%253A%2520an%2520oestrogen%2520receptor%2520antagonist%2520with%2520a%2520novel%2520mechanism%2520of%2520action%26jtitle%3DBr.%2520J.%2520Cancer%26date%3D2004%26volume%3D90%26spage%3DS2%26epage%3DS6%26doi%3D10.1038%2Fsj.bjc.6601629" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Howell, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sapunar, F.</span></span> <span> </span><span class="NLM_article-title">Fulvestrant revisited: efficacy and safety of the 500-mg dose</span>. <i>Clin. Breast Cancer</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">204</span>– <span class="NLM_lpage">210</span>, <span class="refDoi"> DOI: 10.1016/j.clbc.2011.02.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=10.1016%2Fj.clbc.2011.02.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=21729658" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlKisr7E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2011&pages=204-210&author=A.+Howellauthor=F.+Sapunar&title=Fulvestrant+revisited%3A+efficacy+and+safety+of+the+500-mg+dose&doi=10.1016%2Fj.clbc.2011.02.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Fulvestrant revisited: efficacy and safety of the 500-mg dose</span></div><div class="casAuthors">Howell, Anthony; Sapunar, Francisco</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Breast Cancer</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">204-210</span>CODEN:
                <span class="NLM_cas:coden">CBCLB7</span>;
        ISSN:<span class="NLM_cas:issn">1526-8209</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review.  Postmenopausal women with hormone receptor-pos. advanced breast cancer are candidates for endocrine therapy.  As the disease will eventually progress in most patients, it is important to investigate agents with novel modes of action to reduce the likelihood of treatment cross-resistance.  Fulvestrant is an estrogen receptor antagonist with no known agonist effects that has been shown to be as effective as anastrozole following failure on tamoxifen, at the approved dose of 250 mg/mo.  However, pharmacokinetic modeling and evidence of clin. efficacy in early trials, together with the favorable tolerability profile of fulvestrant 250 mg, led to suggestions that increasing the fulvestrant dose would lead to an improved benefit-risk profile.  This review describes the rationale behind the development of a 500 mg/mo higher dose of fulvestrant and details relevant clin. trials, including the pivotal phase III COmparisoN of Faslodex In Recurrent or Metastatic breast cancer (CONFIRM) study.  CONFIRM demonstrated a significant improvement in progression-free survival for fulvestrant 500 mg vs. 250 mg in postmenopausal patients who had progressed on previous endocrine therapy.  Here, we present and discuss a pooled safety anal. of CONFIRM and three further clin. studies demonstrating fulvestrant 500 mg to be well-tolerated with no evidence of dose-related adverse events.  Overall, these data indicate an improved benefit-risk profile for fulvestrant 500 mg vs. 250 mg following failure on prior endocrine therapy, and suggest that fulvestrant 500 mg may be considered in future as initial endocrine treatment for advanced breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4U1d7Q6edlbVg90H21EOLACvtfcHk0lhoIxs5xSLpbg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlKisr7E&md5=0d1063c9546320294193e92d3f2097dd</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.clbc.2011.02.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.clbc.2011.02.002%26sid%3Dliteratum%253Aachs%26aulast%3DHowell%26aufirst%3DA.%26aulast%3DSapunar%26aufirst%3DF.%26atitle%3DFulvestrant%2520revisited%253A%2520efficacy%2520and%2520safety%2520of%2520the%2520500-mg%2520dose%26jtitle%3DClin.%2520Breast%2520Cancer%26date%3D2011%26volume%3D11%26spage%3D204%26epage%3D210%26doi%3D10.1016%2Fj.clbc.2011.02.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Robertson, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindemann, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garnett, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicholson, R. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuter, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gee, J. M.</span></span> <span> </span><span class="NLM_article-title">A good drug made better: the fulvestrant dose-response story</span>. <i>Clin. Breast Cancer</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">381</span>– <span class="NLM_lpage">389</span>, <span class="refDoi"> DOI: 10.1016/j.clbc.2014.06.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=10.1016%2Fj.clbc.2014.06.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=25457991" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFKmtrvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2014&pages=381-389&author=J.+F.+Robertsonauthor=J.+Lindemannauthor=S.+Garnettauthor=E.+Andersonauthor=R.+I.+Nicholsonauthor=I.+Kuterauthor=J.+M.+Gee&title=A+good+drug+made+better%3A+the+fulvestrant+dose-response+story&doi=10.1016%2Fj.clbc.2014.06.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">A Good Drug Made Better: The Fulvestrant Dose-Response Story</span></div><div class="casAuthors">Robertson, John F. R.; Lindemann, Justin; Garnett, Sally; Anderson, Elizabeth; Nicholson, Robert I.; Kuter, Irene; Gee, Julia M. W.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Breast Cancer</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">381-389</span>CODEN:
                <span class="NLM_cas:coden">CBCLB7</span>;
        ISSN:<span class="NLM_cas:issn">1526-8209</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review.  Sequential use of endocrine therapies remains the cornerstone of treatment for hormone receptor-pos. advanced breast cancer, before the use of cytotoxic chemotherapy for unresponsive disease.  Fulvestrant is an estrogen receptor (ER) antagonist approved for the treatment of postmenopausal women with ER+ advanced breast cancer after failure of prior antiestrogen therapy.  Initially approved at a monthly dose of 250 mg, the recommended fulvestrant dose was revised to 500 mg (500 mg/mo plus 500 mg on day 14 of month 1) after demonstration of improved progression-free survival vs. fulvestrant 250 mg.  We have reviewed the dose-dependent effects of fulvestrant, both from a retrospective combined anal. of dose-dependent redn. of tumor biomarkers in the presurgical setting (3 previously reported studies: Study 18, Neoadjuvant Endocrine Therapy for Women with Estrogen-Sensitive Tumors, and Trial 57) and from a review of clin. studies for advanced breast cancer in postmenopausal women.  Anal. of presurgical data revealed a consistent dose-dependent effect for fulvestrant on tumor biomarkers, with increasing fulvestrant dose resulting in greater redns. in ER, progesterone receptor, and Ki67 labeling index.  The dose-dependent biol. effect corresponds with the dose-dependent clin. efficacy obsd. in the treatment of advanced breast cancer after failure of prior antiestrogen therapy.  Although it remains to be detd. in a phase III trial, cross-trial comparisons suggest a dose-dependent relationship for fulvestrant as first-line treatment for advanced breast cancer.  Overall, biol. and clin. data demonstrate a strong dose-dependent relationship for fulvestrant, supporting the efficacy benefit seen with fulvestrant 500 mg over the 250 mg dose.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbnkP0JX-1rbVg90H21EOLACvtfcHk0lgtxwJTOwaD_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFKmtrvI&md5=88d83ed816d271f73ad84521982b8411</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.clbc.2014.06.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.clbc.2014.06.005%26sid%3Dliteratum%253Aachs%26aulast%3DRobertson%26aufirst%3DJ.%2BF.%26aulast%3DLindemann%26aufirst%3DJ.%26aulast%3DGarnett%26aufirst%3DS.%26aulast%3DAnderson%26aufirst%3DE.%26aulast%3DNicholson%26aufirst%3DR.%2BI.%26aulast%3DKuter%26aufirst%3DI.%26aulast%3DGee%26aufirst%3DJ.%2BM.%26atitle%3DA%2520good%2520drug%2520made%2520better%253A%2520the%2520fulvestrant%2520dose-response%2520story%26jtitle%3DClin.%2520Breast%2520Cancer%26date%3D2014%26volume%3D14%26spage%3D381%26epage%3D389%26doi%3D10.1016%2Fj.clbc.2014.06.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McDonnell, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wardell, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norris, J. D.</span></span> <span> </span><span class="NLM_article-title">Oral selective estrogen receptor downregulators (SERDs), a breakthrough endocrine therapy for breast cancer</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">4883</span>– <span class="NLM_lpage">4887</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00760</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00760" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A528%3ADC%252BC2MXpsFaktL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=4883-4887&author=D.+P.+McDonnellauthor=S.+E.+Wardellauthor=J.+D.+Norris&title=Oral+selective+estrogen+receptor+downregulators+%28SERDs%29%2C+a+breakthrough+endocrine+therapy+for+breast+cancer&doi=10.1021%2Facs.jmedchem.5b00760"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Oral Selective Estrogen Receptor Downregulators (SERDs), a Breakthrough Endocrine Therapy for Breast Cancer</span></div><div class="casAuthors">McDonnell, Donald P.; Wardell, Suzanne E.; Norris, John D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">4883-4887</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Drugs that inhibit estrogen receptor alpha (ERα) or that block the prodn. of estrogens remain frontline interventions in the treatment and management of breast cancer at all stages.  However, resistance to endocrine therapies, esp. in the setting of advanced disease, remains an impediment to durable clin. responses.  Although the mechanisms underlying resistance to existing agents are complex, preclin. studies suggest that selective estrogen receptor downregulators (SERDs), mols. which eliminate ERα expression, may have particular utility in the treatment of breast cancers that have progressed on tamoxifen and/or aromatase inhibitors.  The discovery and development of orally bioavailable SERDs provide the opportunity to evaluate the utility of eliminating ERα expression in advanced metastatic breast cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpQkRKbmjeJdLVg90H21EOLACvtfcHk0lgtxwJTOwaD_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXpsFaktL8%253D&md5=127869f40034451832a82fb2d7816528</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00760&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00760%26sid%3Dliteratum%253Aachs%26aulast%3DMcDonnell%26aufirst%3DD.%2BP.%26aulast%3DWardell%26aufirst%3DS.%2BE.%26aulast%3DNorris%26aufirst%3DJ.%2BD.%26atitle%3DOral%2520selective%2520estrogen%2520receptor%2520downregulators%2520%2528SERDs%2529%252C%2520a%2520breakthrough%2520endocrine%2520therapy%2520for%2520breast%2520cancer%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D4883%26epage%3D4887%26doi%3D10.1021%2Facs.jmedchem.5b00760" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abdel-Magid, A. F.</span></span> <span> </span><span class="NLM_article-title">Selective estrogen receptor degraders (SERDs): a promising treatment to overcome resistance to endocrine therapy in ERα-positive breast cancer</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1129</span>– <span class="NLM_lpage">1131</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.7b00424</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.7b00424" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=1129-1131&author=A.+F.+Abdel-Magid&title=Selective+estrogen+receptor+degraders+%28SERDs%29%3A+a+promising+treatment+to+overcome+resistance+to+endocrine+therapy+in+ER%CE%B1-positive+breast+cancer&doi=10.1021%2Facsmedchemlett.7b00424"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.7b00424&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.7b00424%26sid%3Dliteratum%253Aachs%26aulast%3DAbdel-Magid%26aufirst%3DA.%2BF.%26atitle%3DSelective%2520estrogen%2520receptor%2520degraders%2520%2528SERDs%2529%253A%2520a%2520promising%2520treatment%2520to%2520overcome%2520resistance%2520to%2520endocrine%2520therapy%2520in%2520ER%25CE%25B1-positive%2520breast%2520cancer%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2017%26volume%3D8%26spage%3D1129%26epage%3D1131%26doi%3D10.1021%2Facsmedchemlett.7b00424" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Weir, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradbury, R. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rabow, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buttar, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Callis, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curwen, J. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Almeida, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ballard, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hulse, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donald, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feron, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karoutchi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacFaul, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moss, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norman, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pearson, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tonge, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rowlinson, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sadler, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richmond, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ladd, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pazolli, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mazzola, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">D’Cruz, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Savi, C.</span></span> <span> </span><span class="NLM_article-title">AZD9496: an oral estrogen receptor inhibitor that blocks the growth of ER-positive and ESR1-mutant breast tumors in preclinical models</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>76</i></span>,  <span class="NLM_fpage">3307</span>– <span class="NLM_lpage">3318</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-15-2357</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=10.1158%2F0008-5472.CAN-15-2357" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=27020862" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A528%3ADC%252BC28XovVyktLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=76&publication_year=2016&pages=3307-3318&author=H.+M.+Weirauthor=R.+H.+Bradburyauthor=M.+Lawsonauthor=A.+A.+Rabowauthor=D.+Buttarauthor=R.+J.+Callisauthor=J.+O.+Curwenauthor=C.+de+Almeidaauthor=P.+Ballardauthor=M.+Hulseauthor=C.+S.+Donaldauthor=L.+J.+Feronauthor=G.+Karoutchiauthor=P.+MacFaulauthor=T.+Mossauthor=R.+A.+Normanauthor=S.+E.+Pearsonauthor=M.+Tongeauthor=G.+Daviesauthor=G.+E.+Walkerauthor=Z.+Wilsonauthor=R.+Rowlinsonauthor=S.+Powellauthor=C.+Sadlerauthor=G.+Richmondauthor=B.+Laddauthor=E.+Pazolliauthor=A.+M.+Mazzolaauthor=C.+D%E2%80%99Cruzauthor=C.+De+Savi&title=AZD9496%3A+an+oral+estrogen+receptor+inhibitor+that+blocks+the+growth+of+ER-positive+and+ESR1-mutant+breast+tumors+in+preclinical+models&doi=10.1158%2F0008-5472.CAN-15-2357"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">AZD9496: An Oral Estrogen Receptor Inhibitor That Blocks the Growth of ER-Positive and ESR1-Mutant Breast Tumors in Preclinical Models</span></div><div class="casAuthors">Weir, Hazel M.; Bradbury, Robert H.; Lawson, Mandy; Rabow, Alfred A.; Buttar, David; Callis, Rowena J.; Curwen, Jon O.; de Almeida, Camila; Ballard, Peter; Hulse, Michael; Donald, Craig S.; Feron, Lyman J. L.; Karoutchi, Galith; MacFaul, Philip; Moss, Thomas; Norman, Richard A.; Pearson, Stuart E.; Tonge, Michael; Davies, Gareth; Walker, Graeme E.; Wilson, Zena; Rowlinson, Rachel; Powell, Steve; Sadler, Claire; Richmond, Graham; Ladd, Brendon; Pazolli, Ermira; Mazzola, Anne Marie; D'Cruz, Celina; De Savi, Chris</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">76</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">3307-3318</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Fulvestrant is an estrogen receptor (ER) antagonist administered to breast cancer patients by monthly i.m. injection.  Given its present limitations of dosing and route of administration, a more flexible orally available compd. has been sought to pursue the potential benefits of this drug in patients with advanced metastatic disease.  Here we report the identification and characterization of AZD9496, a nonsteroidal small-mol. inhibitor of ERα, which is a potent and selective antagonist and downregulator of ERα in vitro and in vivo in ER-pos. models of breast cancer.  Significant tumor growth inhibition was obsd. as low as 0.5 mg/kg dose in the estrogen-dependent MCF-7 xenograft model, where this effect was accompanied by a dose-dependent decrease in PR protein levels, demonstrating potent antagonist activity.  Combining AZD9496 with PI3K pathway and CDK4/6 inhibitors led to further growth-inhibitory effects compared with monotherapy alone.  Tumor regressions were also seen in a long-term estrogen-deprived breast model, where significant downregulation of ERα protein was obsd.  AZD9496 bound and downregulated clin. relevant ESR1 mutants in vitro and inhibited tumor growth in an ESR1-mutant patient-derived xenograft model that included a D538G mutation.  Collectively, the pharmacol. evidence showed that AZD9496 is an oral, nonsteroidal, selective estrogen receptor antagonist and downregulator in ER+ breast cells that could provide meaningful benefit to ER+ breast cancer patients.  AZD9496 is currently being evaluated in a phase I clin. trial.  Cancer Res; 76(11); 3307-18. ©2016 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoP4X3e2iYdcLVg90H21EOLACvtfcHk0lh7Xc9YJ1gHMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XovVyktLo%253D&md5=4ff8ab9a9cd379acc031b7c6fd27134c</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-15-2357&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-15-2357%26sid%3Dliteratum%253Aachs%26aulast%3DWeir%26aufirst%3DH.%2BM.%26aulast%3DBradbury%26aufirst%3DR.%2BH.%26aulast%3DLawson%26aufirst%3DM.%26aulast%3DRabow%26aufirst%3DA.%2BA.%26aulast%3DButtar%26aufirst%3DD.%26aulast%3DCallis%26aufirst%3DR.%2BJ.%26aulast%3DCurwen%26aufirst%3DJ.%2BO.%26aulast%3Dde%2BAlmeida%26aufirst%3DC.%26aulast%3DBallard%26aufirst%3DP.%26aulast%3DHulse%26aufirst%3DM.%26aulast%3DDonald%26aufirst%3DC.%2BS.%26aulast%3DFeron%26aufirst%3DL.%2BJ.%26aulast%3DKaroutchi%26aufirst%3DG.%26aulast%3DMacFaul%26aufirst%3DP.%26aulast%3DMoss%26aufirst%3DT.%26aulast%3DNorman%26aufirst%3DR.%2BA.%26aulast%3DPearson%26aufirst%3DS.%2BE.%26aulast%3DTonge%26aufirst%3DM.%26aulast%3DDavies%26aufirst%3DG.%26aulast%3DWalker%26aufirst%3DG.%2BE.%26aulast%3DWilson%26aufirst%3DZ.%26aulast%3DRowlinson%26aufirst%3DR.%26aulast%3DPowell%26aufirst%3DS.%26aulast%3DSadler%26aufirst%3DC.%26aulast%3DRichmond%26aufirst%3DG.%26aulast%3DLadd%26aufirst%3DB.%26aulast%3DPazolli%26aufirst%3DE.%26aulast%3DMazzola%26aufirst%3DA.%2BM.%26aulast%3DD%25E2%2580%2599Cruz%26aufirst%3DC.%26aulast%3DDe%2BSavi%26aufirst%3DC.%26atitle%3DAZD9496%253A%2520an%2520oral%2520estrogen%2520receptor%2520inhibitor%2520that%2520blocks%2520the%2520growth%2520of%2520ER-positive%2520and%2520ESR1-mutant%2520breast%2520tumors%2520in%2520preclinical%2520models%26jtitle%3DCancer%2520Res.%26date%3D2016%26volume%3D76%26spage%3D3307%26epage%3D3318%26doi%3D10.1158%2F0008-5472.CAN-15-2357" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Joseph, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darimont, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrazate, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ingalla, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walter, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blake, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nonomiya, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guan, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kategaya, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Govek, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kahraman, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brigham, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sensintaffar, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grillot, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prudente, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bischoff, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonnefous, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Douglas, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Julien, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagasawa, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aparicio, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaufman, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haley, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giltnane, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wertz, I. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lackner, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nannini, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sampath, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwarz, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manning, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tantawy, M. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arteaga, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heyman, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rix, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friedman, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, N. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Metcalfe, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hager, J. H.</span></span> <span> </span><span class="NLM_article-title">The selective estrogen receptor downregulator GDC-0810 is efficacious in diverse models of ER+ breast cancer</span>. <i>eLife</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>5</i></span>, <span class="NLM_elocation-id">e15828</span> <span class="refDoi"> DOI: 10.7554/eLife.15828</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=10.7554%2FeLife.15828" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=27410477" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A528%3ADC%252BC1cXmslahtb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2016&author=J.+D.+Josephauthor=B.+Darimontauthor=W.+Zhouauthor=A.+Arrazateauthor=A.+Youngauthor=E.+Ingallaauthor=K.+Walterauthor=R.+A.+Blakeauthor=J.+Nonomiyaauthor=Z.+Guanauthor=L.+Kategayaauthor=S.+P.+Govekauthor=A.+G.+Laiauthor=M.+Kahramanauthor=D.+Brighamauthor=J.+Sensintaffarauthor=N.+Luauthor=G.+Shaoauthor=J.+Qianauthor=K.+Grillotauthor=M.+Moonauthor=R.+Prudenteauthor=E.+Bischoffauthor=K.+J.+Leeauthor=C.+Bonnefousauthor=K.+L.+Douglasauthor=J.+D.+Julienauthor=J.+Y.+Nagasawaauthor=A.+Aparicioauthor=J.+Kaufmanauthor=B.+Haleyauthor=J.+M.+Giltnaneauthor=I.+E.+Wertzauthor=M.+R.+Lacknerauthor=M.+A.+Nanniniauthor=D.+Sampathauthor=L.+Schwarzauthor=H.+C.+Manningauthor=M.+N.+Tantawyauthor=C.+L.+Arteagaauthor=R.+A.+Heymanauthor=P.+J.+Rixauthor=L.+Friedmanauthor=N.+D.+Smithauthor=C.+Metcalfeauthor=J.+H.+Hager&title=The+selective+estrogen+receptor+downregulator+GDC-0810+is+efficacious+in+diverse+models+of+ER%2B+breast+cancer&doi=10.7554%2FeLife.15828"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">The selective estrogen receptor downregulator GDC-0810 is efficacious in diverse models of ER+ breast cancer</span></div><div class="casAuthors">Joseph, James D.; Darimont, Beatrice; Zhou, Wei; Arrazate, Alfonso; Young, Amy; Ingalla, Ellen; Walter, Kimberly; Blake, Robert A.; Nonomiya, Jim; Guan, Zhengyu; Kategaya, Lorna; Govek, Steven P.; Lai, Andiliy G.; Kahraman, Mehmet; Brigham, Dan; Sensintaffar, John; Lu, Nhin; Shao, Gang; Qian, Jing; Grillot, Kate; Moon, Michael; Prudente, Rene; Bischoff, Eric; Lee, Kyoung-Jin; Bonnefous, Celine; Douglas, Karensa L.; Julien, Jackaline D.; Nagasawa, Johnny Y.; Aparicio, Anna; Kaufman, Josh; Haley, Benjamin; Giltnane, Jennifer M.; Wertz, Ingrid E.; Lackner, Mark R.; Nannini, Michelle A.; Sampath, Deepak; Schwarz, Luis; Manning, Henry Charles; Tantawy, Mohammed Noor; Arteaga, Carlos L.; Heyman, Richard A.; Rix, Peter J.; Friedman, Lori; Smith, Nicholas D.; Metcalfe, Ciara; Hager, Jeffrey H.</div><div class="citationInfo"><span class="NLM_cas:title">eLife</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">e15828/1-e15828/34</span>CODEN:
                <span class="NLM_cas:coden">ELIFA8</span>;
        ISSN:<span class="NLM_cas:issn">2050-084X</span>.
    
            (<span class="NLM_cas:orgname">eLife Sciences Publications Ltd.</span>)
        </div><div class="casAbstract">ER-targeted therapeutics provide valuable treatment options for patients with ER+ breast cancer, however, current relapse and mortality rates emphasize the need for improved therapeutic strategies.  The recent discovery of prevalent ESR1 mutations in relapsed tumors underscores a sustained reliance of advanced tumors on ERa signaling, and provides a strong rationale for continued targeting of ERa.  Here we describe GDC-0810, a novel, non-steroidal, orally bioavailable selective ER downregulator (SERD), which was identified by prospectively optimizing ERa degrdn., antagonism and pharmacokinetic properties.  GDC-0810 induces a distinct ERa conformation, relative to that induced by currently approved therapeutics, suggesting a unique mechanism of action.  GDC-0810 has robust in vitro and in vivo activity against a variety of human breast cancer cell lines and patient derived xenografts, including a tamoxifen-resistant model and those that harbor ERa mutations.  GDC-0810 is currently being evaluated in Phase II clin. studies in women with ER+ breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoML4fZixufB7Vg90H21EOLACvtfcHk0lh7Xc9YJ1gHMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXmslahtb0%253D&md5=cfdf9624f8b260a732b92caab0bb2d58</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.7554%2FeLife.15828&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7554%252FeLife.15828%26sid%3Dliteratum%253Aachs%26aulast%3DJoseph%26aufirst%3DJ.%2BD.%26aulast%3DDarimont%26aufirst%3DB.%26aulast%3DZhou%26aufirst%3DW.%26aulast%3DArrazate%26aufirst%3DA.%26aulast%3DYoung%26aufirst%3DA.%26aulast%3DIngalla%26aufirst%3DE.%26aulast%3DWalter%26aufirst%3DK.%26aulast%3DBlake%26aufirst%3DR.%2BA.%26aulast%3DNonomiya%26aufirst%3DJ.%26aulast%3DGuan%26aufirst%3DZ.%26aulast%3DKategaya%26aufirst%3DL.%26aulast%3DGovek%26aufirst%3DS.%2BP.%26aulast%3DLai%26aufirst%3DA.%2BG.%26aulast%3DKahraman%26aufirst%3DM.%26aulast%3DBrigham%26aufirst%3DD.%26aulast%3DSensintaffar%26aufirst%3DJ.%26aulast%3DLu%26aufirst%3DN.%26aulast%3DShao%26aufirst%3DG.%26aulast%3DQian%26aufirst%3DJ.%26aulast%3DGrillot%26aufirst%3DK.%26aulast%3DMoon%26aufirst%3DM.%26aulast%3DPrudente%26aufirst%3DR.%26aulast%3DBischoff%26aufirst%3DE.%26aulast%3DLee%26aufirst%3DK.%2BJ.%26aulast%3DBonnefous%26aufirst%3DC.%26aulast%3DDouglas%26aufirst%3DK.%2BL.%26aulast%3DJulien%26aufirst%3DJ.%2BD.%26aulast%3DNagasawa%26aufirst%3DJ.%2BY.%26aulast%3DAparicio%26aufirst%3DA.%26aulast%3DKaufman%26aufirst%3DJ.%26aulast%3DHaley%26aufirst%3DB.%26aulast%3DGiltnane%26aufirst%3DJ.%2BM.%26aulast%3DWertz%26aufirst%3DI.%2BE.%26aulast%3DLackner%26aufirst%3DM.%2BR.%26aulast%3DNannini%26aufirst%3DM.%2BA.%26aulast%3DSampath%26aufirst%3DD.%26aulast%3DSchwarz%26aufirst%3DL.%26aulast%3DManning%26aufirst%3DH.%2BC.%26aulast%3DTantawy%26aufirst%3DM.%2BN.%26aulast%3DArteaga%26aufirst%3DC.%2BL.%26aulast%3DHeyman%26aufirst%3DR.%2BA.%26aulast%3DRix%26aufirst%3DP.%2BJ.%26aulast%3DFriedman%26aufirst%3DL.%26aulast%3DSmith%26aufirst%3DN.%2BD.%26aulast%3DMetcalfe%26aufirst%3DC.%26aulast%3DHager%26aufirst%3DJ.%2BH.%26atitle%3DThe%2520selective%2520estrogen%2520receptor%2520downregulator%2520GDC-0810%2520is%2520efficacious%2520in%2520diverse%2520models%2520of%2520ER%252B%2520breast%2520cancer%26jtitle%3DeLife%26date%3D2016%26volume%3D5%26doi%3D10.7554%2FeLife.15828" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bihani, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, H. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arlt, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tao, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Purandare, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hattersley, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garner, F.</span></span> <span> </span><span class="NLM_article-title">Elacestrant (RAD1901), a selective estrogen receptor degrader (SERD), has antitumor activity in multiple ER+ breast cancer patient-derived xenograft models</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">4793</span>– <span class="NLM_lpage">4804</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.CCR-16-2561</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=10.1158%2F1078-0432.CCR-16-2561" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=28473534" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtlCmu73O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2017&pages=4793-4804&author=T.+Bihaniauthor=H.+K.+Patelauthor=H.+Arltauthor=N.+Taoauthor=H.+Jiangauthor=J.+L.+Brownauthor=D.+M.+Purandareauthor=G.+Hattersleyauthor=F.+Garner&title=Elacestrant+%28RAD1901%29%2C+a+selective+estrogen+receptor+degrader+%28SERD%29%2C+has+antitumor+activity+in+multiple+ER%2B+breast+cancer+patient-derived+xenograft+models&doi=10.1158%2F1078-0432.CCR-16-2561"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Elacestrant (RAD1901), a Selective Estrogen Receptor Degrader (SERD), Has Antitumor Activity in Multiple ER+ Breast Cancer Patient-derived Xenograft Models</span></div><div class="casAuthors">Bihani, Teeru; Patel, Hitisha K.; Arlt, Heike; Tao, Nianjun; Jiang, Hai; Brown, Jeffrey L.; Purandare, Dinesh M.; Hattersley, Gary; Garner, Fiona</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">4793-4804</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Purpose: Estrogen receptor-pos. (ER+) breast cancers are typically treated with endocrine agents, and dependence on the ER pathway is often retained even after multiple rounds of antiestrogen therapy.  Selective estrogen receptor degraders (SERD) are being developed as a strategy to more effectively target ER and exploit ER dependence in these cancers, which includes inhibiting both wild-type and mutant forms of ER.  The purpose of this study was to evaluate the efficacy of a novel orally bioavailable SERD, elacestrant (RAD1901), in preclin. models of ER+ breast cancer.  Exptl. Design: Elacestrant was evaluated as a single agent and in combination with palbociclib or everolimus in multiple ER+ breast cancer models, including several patient-derived xenograft models.  Results: Elacestrant induces the degrdn. of ER, inhibits ER-mediated signaling and growth of ER+ breast cancer cell lines in vitro and in vivo, and significantly inhibits tumor growth of multiple PDX models.  Furthermore, we demonstrate that elacestrant in combination with palbociclib or everolimus can lead to greater efficacy in certain contexts.  Finally, elacestrant exhibits significant antitumor activity both as a single agent and in combination with palbociclib in two patient-derived breast cancer xenograft models harboring ESR1 mutations.  Conclusions: These data underscore the potential clin. utility of elacestrant as a single agent and as a combination therapy, for both early- and late-stage ER+ disease.  Clin Cancer Res; 23(16); 4793-804. ©2017 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpASoatzBVHxbVg90H21EOLACvtfcHk0lh7Xc9YJ1gHMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtlCmu73O&md5=4a6811274e95e6401bbfe0e56ccb1afc</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-16-2561&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-16-2561%26sid%3Dliteratum%253Aachs%26aulast%3DBihani%26aufirst%3DT.%26aulast%3DPatel%26aufirst%3DH.%2BK.%26aulast%3DArlt%26aufirst%3DH.%26aulast%3DTao%26aufirst%3DN.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DBrown%26aufirst%3DJ.%2BL.%26aulast%3DPurandare%26aufirst%3DD.%2BM.%26aulast%3DHattersley%26aufirst%3DG.%26aulast%3DGarner%26aufirst%3DF.%26atitle%3DElacestrant%2520%2528RAD1901%2529%252C%2520a%2520selective%2520estrogen%2520receptor%2520degrader%2520%2528SERD%2529%252C%2520has%2520antitumor%2520activity%2520in%2520multiple%2520ER%252B%2520breast%2520cancer%2520patient-derived%2520xenograft%2520models%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2017%26volume%3D23%26spage%3D4793%26epage%3D4804%26doi%3D10.1158%2F1078-0432.CCR-16-2561" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tria, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abrams, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baird, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burks, H. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Firestone, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaither, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamann, L. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirby, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lombardo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macchi, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDonnell, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mishina, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norris, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nunez, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Springer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomsen, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tiong-Yip, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peukert, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of LSZ102, a potent, orally bioavailable selective estrogen receptor degrader (SERD) for the treatment of estrogen receptor positive breast cancer</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">2837</span>– <span class="NLM_lpage">2864</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01682</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01682" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A528%3ADC%252BC1cXltFynurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=2837-2864&author=G.+S.+Triaauthor=T.+Abramsauthor=J.+Bairdauthor=H.+E.+Burksauthor=B.+Firestoneauthor=L.+A.+Gaitherauthor=L.+G.+Hamannauthor=G.+Heauthor=C.+A.+Kirbyauthor=S.+Kimauthor=F.+Lombardoauthor=K.+J.+Macchiauthor=D.+P.+McDonnellauthor=Y.+Mishinaauthor=J.+D.+Norrisauthor=J.+Nunezauthor=C.+Springerauthor=Y.+Sunauthor=N.+M.+Thomsenauthor=C.+Wangauthor=J.+Wangauthor=B.+Yuauthor=C.+L.+Tiong-Yipauthor=S.+Peukert&title=Discovery+of+LSZ102%2C+a+potent%2C+orally+bioavailable+selective+estrogen+receptor+degrader+%28SERD%29+for+the+treatment+of+estrogen+receptor+positive+breast+cancer&doi=10.1021%2Facs.jmedchem.7b01682"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of LSZ102, a Potent, Orally Bioavailable Selective Estrogen Receptor Degrader (SERD) for the Treatment of Estrogen Receptor Positive Breast Cancer</span></div><div class="casAuthors">Tria, George S.; Abrams, Tinya; Baird, Jason; Burks, Heather E.; Firestone, Brant; Gaither, L. Alex; Hamann, Lawrence G.; He, Guo; Kirby, Christina A.; Kim, Sunkyu; Lombardo, Franco; Macchi, Kaitlin J.; McDonnell, Donald P.; Mishina, Yuji; Norris, John D.; Nunez, Jill; Springer, Clayton; Sun, Yingchuan; Thomsen, Noel M.; Wang, Chunrong; Wang, Jianling; Yu, Bing; Tiong-Yip, Choi-Lai; Peukert, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2837-2864</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In breast cancer, estrogen receptor alpha (ERα) pos. cancer accounts for approx. 74% of all diagnoses, and in these settings, it is a primary driver of cell proliferation.  Treatment of ERα pos. breast cancer has long relied on endocrine therapies such as selective estrogen receptor modulators, aromatase inhibitors, and selective estrogen receptor degraders (SERDs).  The steroid-based anti-estrogen fulvestrant, the only approved SERD, is effective in patients who have not previously been treated with endocrine therapy as well as in patients who have progressed after receiving other endocrine therapies.  Its efficacy, however, may be limited due to its poor physicochem. properties.  THe authors describe the design and synthesis of a series of potent benzothiophene-contg. compds. that exhibit oral bioavailability and preclin. activity as SERDs.  This article culminates in the identification of LSZ102 (I), a compd. in clin. development for the treatment of ERα pos. breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcHgOrWwlRvrVg90H21EOLACvtfcHk0lg0BIOD0rMeQg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXltFynurc%253D&md5=dba954f9f11121fae1bb50979428a83b</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01682&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01682%26sid%3Dliteratum%253Aachs%26aulast%3DTria%26aufirst%3DG.%2BS.%26aulast%3DAbrams%26aufirst%3DT.%26aulast%3DBaird%26aufirst%3DJ.%26aulast%3DBurks%26aufirst%3DH.%2BE.%26aulast%3DFirestone%26aufirst%3DB.%26aulast%3DGaither%26aufirst%3DL.%2BA.%26aulast%3DHamann%26aufirst%3DL.%2BG.%26aulast%3DHe%26aufirst%3DG.%26aulast%3DKirby%26aufirst%3DC.%2BA.%26aulast%3DKim%26aufirst%3DS.%26aulast%3DLombardo%26aufirst%3DF.%26aulast%3DMacchi%26aufirst%3DK.%2BJ.%26aulast%3DMcDonnell%26aufirst%3DD.%2BP.%26aulast%3DMishina%26aufirst%3DY.%26aulast%3DNorris%26aufirst%3DJ.%2BD.%26aulast%3DNunez%26aufirst%3DJ.%26aulast%3DSpringer%26aufirst%3DC.%26aulast%3DSun%26aufirst%3DY.%26aulast%3DThomsen%26aufirst%3DN.%2BM.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DYu%26aufirst%3DB.%26aulast%3DTiong-Yip%26aufirst%3DC.%2BL.%26aulast%3DPeukert%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520LSZ102%252C%2520a%2520potent%252C%2520orally%2520bioavailable%2520selective%2520estrogen%2520receptor%2520degrader%2520%2528SERD%2529%2520for%2520the%2520treatment%2520of%2520estrogen%2520receptor%2520positive%2520breast%2520cancer%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D2837%26epage%3D2864%26doi%3D10.1021%2Facs.jmedchem.7b01682" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Carlson, R. W.</span></span> <span> </span><span class="NLM_article-title">The history and mechanism of action of fulvestrant</span>. <i>Clin. Breast Cancer</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">S5</span>– <span class="NLM_lpage">S8</span>, <span class="refDoi"> DOI: 10.3816/CBC.2005.s.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=10.3816%2FCBC.2005.s.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=15865849" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A528%3ADC%252BD2MXltV2ksbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2005&pages=S5-S8&author=R.+W.+Carlson&title=The+history+and+mechanism+of+action+of+fulvestrant&doi=10.3816%2FCBC.2005.s.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">The history and mechanism of action of fulvestrant</span></div><div class="casAuthors">Carlson, Robert W.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Breast Cancer</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">Suppl. 1</span>),
    <span class="NLM_cas:pages">S5-S8</span>CODEN:
                <span class="NLM_cas:coden">CBCLB7</span>;
        ISSN:<span class="NLM_cas:issn">1526-8209</span>.
    
            (<span class="NLM_cas:orgname">Cancer Information Group, LP</span>)
        </div><div class="casAbstract">A review.  Fulvestrant is a pure antiestrogen that emerged from a systematic medicinal chem. strategy of modification of long-chain alkyl substitutes in the 7α-position of estradiol.  Fulvestrant has no uterotrophic effects on the immature or ovariectomized rat and blocks the agonistic effects of estradiol and tamoxifen in a dose-dependent manner.  In in vivo and in vitro breast cancer models, fulvestrant has anticancer activity at least as good as tamoxifen and is superior to tamoxifen in some models.  Fulvestrant requires i.m. administration in a proprietary formulation of castor oil and alcs.  When fulvestrant binds to estrogen receptor monomers it inhibits receptor dimerization, activating function 1 (AF1) and AF2 are rendered inactive, translocation of receptor to the nucleus is reduced, and degrdn. of the estrogen receptor is accelerated.  This results in pure antiestrogenic effects.  There is substantial preclin. evidence that the nonsteroidal hormone-dependent mechanisms of estrogen receptor activation and regulation via growth factor receptors and their signal transduction pathways are important in the development of breast cancer hormonal resistance.  Methods of exploiting the interactions between these nonsteroidal hormone-dependent mechanisms of resistance and hormonal agents such as fulvestrant are an active area for drug development and clin. investigation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr6cUux0p2XbbVg90H21EOLACvtfcHk0lg0BIOD0rMeQg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXltV2ksbs%253D&md5=76569c36605521476f8458d614670b04</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.3816%2FCBC.2005.s.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3816%252FCBC.2005.s.008%26sid%3Dliteratum%253Aachs%26aulast%3DCarlson%26aufirst%3DR.%2BW.%26atitle%3DThe%2520history%2520and%2520mechanism%2520of%2520action%2520of%2520fulvestrant%26jtitle%3DClin.%2520Breast%2520Cancer%26date%3D2005%26volume%3D6%26spage%3DS5%26epage%3DS8%26doi%3D10.3816%2FCBC.2005.s.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Marsaud, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gougelet, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maillard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renoir, J. M.</span></span> <span> </span><span class="NLM_article-title">Various phosphorylation pathways, depending on agonist and antagonist binding to endogenous estrogen receptor α (ERα), differentially affect ERα extractability, proteasome-mediated stability, and transcriptional activity in human breast cancer cells</span>. <i>Mol. Endocrinol.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">2013</span>– <span class="NLM_lpage">2027</span>, <span class="refDoi"> DOI: 10.1210/me.2002-0269</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=10.1210%2Fme.2002-0269" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=12855746" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A528%3ADC%252BD3sXotFSrsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2003&pages=2013-2027&author=V.+Marsaudauthor=A.+Gougeletauthor=S.+Maillardauthor=J.+M.+Renoir&title=Various+phosphorylation+pathways%2C+depending+on+agonist+and+antagonist+binding+to+endogenous+estrogen+receptor+%CE%B1+%28ER%CE%B1%29%2C+differentially+affect+ER%CE%B1+extractability%2C+proteasome-mediated+stability%2C+and+transcriptional+activity+in+human+breast+cancer+cells&doi=10.1210%2Fme.2002-0269"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Various phosphorylation pathways, depending on agonist and antagonist binding to endogenous estrogen receptor α (ERα), differentially affect ERα extractability, proteasome-mediated stability, and transcriptional activity in human breast cancer cells</span></div><div class="casAuthors">Marsaud, Veronique; Gougelet, Angelique; Maillard, Sebastien; Renoir, Jack-Michel</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Endocrinology</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2013-2027</span>CODEN:
                <span class="NLM_cas:coden">MOENEN</span>;
        ISSN:<span class="NLM_cas:issn">0888-8809</span>.
    
            (<span class="NLM_cas:orgname">Endocrine Society</span>)
        </div><div class="casAbstract">Estrogen receptor-α (ER) is down-regulated in the presence of its cognate ligand, estradiol (E2), as well as in the presence of antiestrogens, through the ubiquitin proteasome pathway.  Here, we show that, at pharmacol. concns., the degrdn. rate of pure antagonist/endogenous ER complexes from human breast cancer MCF-7 cells is 10 times faster than that of ER-E2 complexes, while 4-hydroxy-tamoxifen (4-OH-T)-ER complexes are stable.  Whereas pure antagonist-ER complexes are firmly bound to a nuclear compartment from which they are not extractable, the 4-OH-T-ER accumulates in a sol. cell compartment.  No difference was obsd. in the fate of ER whether bound to pure antiestrogens ICI 182,780 or RU 58668.  Cycloheximide expts. showed that, while the proteasome-mediated destruction of E2-ER (unlike that of RU 58668- and ICI 182,780-ER) complexes could implicate (or not) a protein synthesis-dependent process, both MAPKs (p38 and ERKs p44 and p42) are activated.  By using a panel of kinase inhibitors/activators to study the impact of phosphorylation pathways on ER degrdn., we found that protein kinase C is an enhancer of proteasome-mediated degrdn. of both ligand-free and ER bound to either E2, 4-OH-T, and pure antagonists.  On the contrary, protein kinase A, MAPKs, and phosphatidyl-inositol-3 kinase all impede proteasome-mediated destruction of ligand free and E2-bound ER while only MAPKs inhibit the degrdn. of pure antiestrogens/ER species.  In addn., no correlation was found between the capacity of kinase inhibitors to affect ER stability and the basal or E2-induced transcription.  These results suggest that, in MCF-7 breast cancer cells, ER turnover, localization, and activity are maintained by an equil. between various phosphorylation pathways, which are differently modulated by ER ligands and protein kinases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGofjCUE8Z2B7bVg90H21EOLACvtfcHk0lj6vj-909OymQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXotFSrsLc%253D&md5=d7f2244b364abdc94a0730f3140f29a8</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1210%2Fme.2002-0269&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1210%252Fme.2002-0269%26sid%3Dliteratum%253Aachs%26aulast%3DMarsaud%26aufirst%3DV.%26aulast%3DGougelet%26aufirst%3DA.%26aulast%3DMaillard%26aufirst%3DS.%26aulast%3DRenoir%26aufirst%3DJ.%2BM.%26atitle%3DVarious%2520phosphorylation%2520pathways%252C%2520depending%2520on%2520agonist%2520and%2520antagonist%2520binding%2520to%2520endogenous%2520estrogen%2520receptor%2520%25CE%25B1%2520%2528ER%25CE%25B1%2529%252C%2520differentially%2520affect%2520ER%25CE%25B1%2520extractability%252C%2520proteasome-mediated%2520stability%252C%2520and%2520transcriptional%2520activity%2520in%2520human%2520breast%2520cancer%2520cells%26jtitle%3DMol.%2520Endocrinol.%26date%3D2003%26volume%3D17%26spage%3D2013%26epage%3D2027%26doi%3D10.1210%2Fme.2002-0269" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wittmann, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sherk, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDonnell, D. P.</span></span> <span> </span><span class="NLM_article-title">Definition of functionally important mechanistic differences among selective estrogen receptor down-regulators</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">9549</span>– <span class="NLM_lpage">9560</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-07-1590</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=10.1158%2F0008-5472.CAN-07-1590" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2007&pages=9549-9560&author=B.+M.+Wittmannauthor=A.+Sherkauthor=D.+P.+McDonnell&title=Definition+of+functionally+important+mechanistic+differences+among+selective+estrogen+receptor+down-regulators&doi=10.1158%2F0008-5472.CAN-07-1590"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-07-1590&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-07-1590%26sid%3Dliteratum%253Aachs%26aulast%3DWittmann%26aufirst%3DB.%2BM.%26aulast%3DSherk%26aufirst%3DA.%26aulast%3DMcDonnell%26aufirst%3DD.%2BP.%26atitle%3DDefinition%2520of%2520functionally%2520important%2520mechanistic%2520differences%2520among%2520selective%2520estrogen%2520receptor%2520down-regulators%26jtitle%3DCancer%2520Res.%26date%3D2007%26volume%3D67%26spage%3D9549%26epage%3D9560%26doi%3D10.1158%2F0008-5472.CAN-07-1590" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sakamoto, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, K. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumagai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mercurio, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deshaies, R. J.</span></span> <span> </span><span class="NLM_article-title">Protacs: chimeric molecules that target proteins to the Skp1-Cullin-F box complex for ubiquitination and degradation</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>98</i></span>,  <span class="NLM_fpage">8554</span>– <span class="NLM_lpage">8559</span>, <span class="refDoi"> DOI: 10.1073/pnas.141230798</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=10.1073%2Fpnas.141230798" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=11438690" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A528%3ADC%252BD3MXls1Wisbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=2001&pages=8554-8559&author=K.+M.+Sakamotoauthor=K.+B.+Kimauthor=A.+Kumagaiauthor=F.+Mercurioauthor=C.+M.+Crewsauthor=R.+J.+Deshaies&title=Protacs%3A+chimeric+molecules+that+target+proteins+to+the+Skp1-Cullin-F+box+complex+for+ubiquitination+and+degradation&doi=10.1073%2Fpnas.141230798"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Protacs: chimeric molecules that target proteins to the Skp1-Cullin-F box complex for ubiquitination and degradation</span></div><div class="casAuthors">Sakamoto, Kathleen M.; Kim, Kyung B.; Kumagai, Akiko; Mercurio, Frank; Crews, Craig M.; Deshaies, Raymond J.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">8554-8559</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The intracellular levels of many proteins are regulated by ubiquitin-dependent proteolysis.  One of the best-characterized enzymes that catalyzes the attachment of ubiquitin to proteins is a ubiquitin ligase complex, Skp1-Cullin-F box complex contg. Hrt1 (SCF).  We sought to artificially target a protein to the SCF complex for ubiquitination and degrdn.  To this end, we tested methionine aminopeptidase-2 (MetAP-2), which covalently binds the angiogenesis inhibitor ovalicin.  A chimeric compd., protein-targeting chimeric mol. 1 (Protac-1), was synthesized to recruit MetAP-2 to SCF.  One domain of Protac-1 contains the IκBα phosphopeptide that is recognized by the F-box protein β-TRCP, whereas the other domain is composed of ovalicin.  We show that MetAP-2 can be tethered to SCFβ-TRCP, ubiquitinated, and degraded in a Protac-1-dependent manner.  In the future, this approach may be useful for conditional inactivation of proteins, and for targeting disease-causing proteins for destruction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp49fYtuEVlJrVg90H21EOLACvtfcHk0lj6vj-909OymQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXls1Wisbk%253D&md5=540358035222f745f7b6367b38781a21</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1073%2Fpnas.141230798&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.141230798%26sid%3Dliteratum%253Aachs%26aulast%3DSakamoto%26aufirst%3DK.%2BM.%26aulast%3DKim%26aufirst%3DK.%2BB.%26aulast%3DKumagai%26aufirst%3DA.%26aulast%3DMercurio%26aufirst%3DF.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26aulast%3DDeshaies%26aufirst%3DR.%2BJ.%26atitle%3DProtacs%253A%2520chimeric%2520molecules%2520that%2520target%2520proteins%2520to%2520the%2520Skp1-Cullin-F%2520box%2520complex%2520for%2520ubiquitination%2520and%2520degradation%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2001%26volume%3D98%26spage%3D8554%26epage%3D8559%26doi%3D10.1073%2Fpnas.141230798" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Toure, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Small-molecule PROTACS: new approaches to protein degradation</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">1966</span>– <span class="NLM_lpage">1973</span>, <span class="refDoi"> DOI: 10.1002/anie.201507978</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=10.1002%2Fanie.201507978" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A528%3ADC%252BC28XnsF2ltA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2016&pages=1966-1973&author=M.+Toureauthor=C.+M.+Crews&title=Small-molecule+PROTACS%3A+new+approaches+to+protein+degradation&doi=10.1002%2Fanie.201507978"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Small-Molecule PROTACS: New Approaches to Protein Degradation</span></div><div class="casAuthors">Toure, Momar; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1966-1973</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  The current inhibitor-based approach to therapeutics has inherent limitations owing to its occupancy-based model: there is a need to maintain high systemic exposure to ensure sufficient in vivo inhibition, high in vivo concns. bring potential for off-target side effects, and there is a need to bind to an active site, thus limiting the drug target space.  As an alternative, induced protein degrdn. lacks these limitations.  Based on an event-driven model, this approach offers a novel catalytic mechanism to irreversibly inhibit protein function by targeting protein destruction through recruitment to the cellular quality control machinery.  Prior protein degrading strategies have lacked therapeutic potential.  However, recent reports of small-mol.-based proteolysis-targeting chimeras (PROTACs) have demonstrated that this technol. can effectively decrease the cellular levels of several protein classes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdaqlIjCSsgrVg90H21EOLACvtfcHk0lj6vj-909OymQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XnsF2ltA%253D%253D&md5=47838cd15719757317cb8282c26cc59e</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1002%2Fanie.201507978&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201507978%26sid%3Dliteratum%253Aachs%26aulast%3DToure%26aufirst%3DM.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DSmall-molecule%2520PROTACS%253A%2520new%2520approaches%2520to%2520protein%2520degradation%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2016%26volume%3D55%26spage%3D1966%26epage%3D1973%26doi%3D10.1002%2Fanie.201507978" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lai, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Induced protein degradation: an emerging drug discovery paradigm</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">101</span>– <span class="NLM_lpage">114</span>, <span class="refDoi"> DOI: 10.1038/nrd.2016.211</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=10.1038%2Fnrd.2016.211" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=27885283" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFCit7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=101-114&author=A.+C.+Laiauthor=C.+M.+Crews&title=Induced+protein+degradation%3A+an+emerging+drug+discovery+paradigm&doi=10.1038%2Fnrd.2016.211"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Induced protein degradation: an emerging drug discovery paradigm</span></div><div class="casAuthors">Lai, Ashton C.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">101-114</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Small-mol. drug discovery has traditionally focused on occupancy of a binding site that directly affects protein function, and this approach typically precludes targeting proteins that lack such amenable sites.  Furthermore, high systemic drug exposures may be needed to maintain sufficient target inhibition in vivo, increasing the risk of undesirable off-target effects.  Induced protein degrdn. is an alternative approach that is event-driven: upon drug binding, the target protein is tagged for elimination.  Emerging technologies based on proteolysis-targeting chimaeras (PROTACs) that exploit cellular quality control machinery to selectively degrade target proteins are attracting considerable attention in the pharmaceutical industry owing to the advantages they could offer over traditional small-mol. strategies.  These advantages include the potential to reduce systemic drug exposure, the ability to counteract increased target protein expression that often accompanies inhibition of protein function and the potential ability to target proteins that are not currently therapeutically tractable, such as transcription factors, scaffolding and regulatory proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoj-O7zXGOpXLVg90H21EOLACvtfcHk0lj6vj-909OymQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFCit7rP&md5=a01ff43e9b696fe4dc03f0815158a98f</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.211&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.211%26sid%3Dliteratum%253Aachs%26aulast%3DLai%26aufirst%3DA.%2BC.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DInduced%2520protein%2520degradation%253A%2520an%2520emerging%2520drug%2520discovery%2520paradigm%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2017%26volume%3D16%26spage%3D101%26epage%3D114%26doi%3D10.1038%2Fnrd.2016.211" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Burslem, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Small-molecule modulation of protein homeostasis</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>117</i></span>,  <span class="NLM_fpage">11269</span>– <span class="NLM_lpage">11301</span>, <span class="refDoi"> DOI: 10.1021/acs.chemrev.7b00077</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.7b00077" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1Ohur3K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=117&publication_year=2017&pages=11269-11301&author=G.+M.+Burslemauthor=C.+M.+Crews&title=Small-molecule+modulation+of+protein+homeostasis&doi=10.1021%2Facs.chemrev.7b00077"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Small-Molecule Modulation of Protein Homeostasis</span></div><div class="casAuthors">Burslem, George M.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">117</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">11269-11301</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Control of protein levels by nucleic-acid-based technologies has proven to be a useful research tool but lacks the advantages of small mols. with respect to cell permeability, temporal control, and the potential generation of therapeutics.  In this Review, we discuss the technologies available for the control of intracellular protein levels with small mols. and compare the various systems available.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpcARydbgPK9bVg90H21EOLACvtfcHk0lg2CQ4C6CBrgA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1Ohur3K&md5=f9f8dd98e2c71c7166d2d500b514d578</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.7b00077&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.7b00077%26sid%3Dliteratum%253Aachs%26aulast%3DBurslem%26aufirst%3DG.%2BM.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DSmall-molecule%2520modulation%2520of%2520protein%2520homeostasis%26jtitle%3DChem.%2520Rev.%26date%3D2017%26volume%3D117%26spage%3D11269%26epage%3D11301%26doi%3D10.1021%2Facs.chemrev.7b00077" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cromm, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Targeted protein degradation: from chemical biology to drug discovery</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">1181</span>– <span class="NLM_lpage">1190</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2017.05.024</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=10.1016%2Fj.chembiol.2017.05.024" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=28648379" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVOjtrrO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2017&pages=1181-1190&author=P.+M.+Crommauthor=C.+M.+Crews&title=Targeted+protein+degradation%3A+from+chemical+biology+to+drug+discovery&doi=10.1016%2Fj.chembiol.2017.05.024"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted Protein Degradation: from Chemical Biology to Drug Discovery</span></div><div class="casAuthors">Cromm, Philipp M.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1181-1190</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Traditional pharmaceutical drug discovery is almost exclusively focused on directly controlling protein activity to cure diseases.  Modulators of protein activity, esp. inhibitors, are developed and applied at high concn. to achieve maximal effects.  Thereby, reduced bioavailability and off-target effects can hamper compd. efficacy.  Nucleic acid-based strategies that control protein function by affecting expression have emerged as an alternative.  However, metabolic stability and broad bioavailability represent development hurdles that remain to be overcome for these approaches.  More recently, utilizing the cell's own protein destruction machinery for selective degrdn. of essential drivers of human disorders has opened up a new and exciting area of drug discovery.  Small-mol.-induced proteolysis of selected substrates offers the potential of reaching beyond the limitations of the current pharmaceutical paradigm to expand the druggable target space.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrosuhmnPE0RrVg90H21EOLACvtfcHk0lg2CQ4C6CBrgA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVOjtrrO&md5=3c8c6d9488396d5eb08ca51cee20bbf4</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2017.05.024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2017.05.024%26sid%3Dliteratum%253Aachs%26aulast%3DCromm%26aufirst%3DP.%2BM.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DTargeted%2520protein%2520degradation%253A%2520from%2520chemical%2520biology%2520to%2520drug%2520discovery%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2017%26volume%3D24%26spage%3D1181%26epage%3D1190%26doi%3D10.1016%2Fj.chembiol.2017.05.024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ottis, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Proteolysis-targeting chimeras: induced protein degradation as a therapeutic strategy</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">892</span>– <span class="NLM_lpage">898</span>, <span class="refDoi"> DOI: 10.1021/acschembio.6b01068</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.6b01068" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjs12ns70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=892-898&author=P.+Ottisauthor=C.+M.+Crews&title=Proteolysis-targeting+chimeras%3A+induced+protein+degradation+as+a+therapeutic+strategy&doi=10.1021%2Facschembio.6b01068"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Proteolysis-Targeting Chimeras: Induced Protein Degradation as a Therapeutic Strategy</span></div><div class="casAuthors">Ottis, Philipp; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">892-898</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Until recently, the only ways to reduce specific protein signaling were to either knock down the target by RNAi or to interfere with the signaling by inhibiting an enzyme or receptor within the signal transduction cascade.  Herein, we review an emerging class of small mol. pharmacol. agents, called PROTACs, that present a novel approach to specifically target proteins and their resp. signaling pathways.  These heterobifunctional mols. utilize endogenous cellular quality control machinery by recruiting it to target proteins in order to induce their degrdn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohB6xwkQMTprVg90H21EOLACvtfcHk0lg2CQ4C6CBrgA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjs12ns70%253D&md5=eada521c10faa1e456e0885fd390c554</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Facschembio.6b01068&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.6b01068%26sid%3Dliteratum%253Aachs%26aulast%3DOttis%26aufirst%3DP.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DProteolysis-targeting%2520chimeras%253A%2520induced%2520protein%2520degradation%2520as%2520a%2520therapeutic%2520strategy%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2017%26volume%3D12%26spage%3D892%26epage%3D898%26doi%3D10.1021%2Facschembio.6b01068" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguez-Gonzalez, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cyrus, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salcius, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deshaies, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakamoto, K. M.</span></span> <span> </span><span class="NLM_article-title">Targeting steroid hormone receptors for ubiquitination and degradation in breast and prostate cancer</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">7201</span>– <span class="NLM_lpage">7211</span>, <span class="refDoi"> DOI: 10.1038/onc.2008.320</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=10.1038%2Fonc.2008.320" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=18794799" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsVGlu7%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2008&pages=7201-7211&author=A.+Rodriguez-Gonzalezauthor=K.+Cyrusauthor=M.+Salciusauthor=K.+Kimauthor=C.+M.+Crewsauthor=R.+J.+Deshaiesauthor=K.+M.+Sakamoto&title=Targeting+steroid+hormone+receptors+for+ubiquitination+and+degradation+in+breast+and+prostate+cancer&doi=10.1038%2Fonc.2008.320"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting steroid hormone receptors for ubiquitination and degradation in breast and prostate cancer</span></div><div class="casAuthors">Rodriguez-Gonzalez, A.; Cyrus, K.; Salcius, M.; Kim, K.; Crews, C. M.; Deshaies, R. J.; Sakamoto, K. M.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">57</span>),
    <span class="NLM_cas:pages">7201-7211</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Proteolysis targeting chimeric mols. (Protacs) target proteins for destruction by exploiting the ubiquitin-dependent proteolytic system of eukaryotic cells.  We designed two Protacs that contain the peptide degron' from hypoxia-inducible factor-1α, which binds to the Von -Hippel-Lindau (VHL) E3 ubiquitin ligase complex, linked to either dihydroxytestosterone that targets the androgen receptor (AR; Protac-A), or linked to estradiol (E2) that targets the estrogen receptor-α (ERα; Protac-B).  We hypothesized that these Protacs would recruit hormone receptors to the VHL E3 ligase complex, resulting in the degrdn. of receptors, and decreased proliferation of hormone-dependent cell lines.  Treatment of estrogen-dependent breast cancer cells with Protac-B induced the degrdn. of ERα in a proteasome-dependent manner.  Protac-B inhibited the proliferation of ERα-dependent breast cancer cells by inducing G1 arrest, inhibition of retinoblastoma phosphorylation and decreasing expression of cyclin D1, progesterone receptors A and B.  Protac-B treatment did not affect the proliferation of estrogen-independent breast cancer cells that lacked ERα expression.  Similarly, Protac-A treatment of androgen-dependent prostate cancer cells induced G1 arrest but did not affect cells that do not express AR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_DgqkmmrRTbVg90H21EOLACvtfcHk0lgYxBoXe6nfhQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsVGlu7%252FM&md5=acd4620a0c095a25bcf733fc2d74e84a</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1038%2Fonc.2008.320&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fonc.2008.320%26sid%3Dliteratum%253Aachs%26aulast%3DRodriguez-Gonzalez%26aufirst%3DA.%26aulast%3DCyrus%26aufirst%3DK.%26aulast%3DSalcius%26aufirst%3DM.%26aulast%3DKim%26aufirst%3DK.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26aulast%3DDeshaies%26aufirst%3DR.%2BJ.%26aulast%3DSakamoto%26aufirst%3DK.%2BM.%26atitle%3DTargeting%2520steroid%2520hormone%2520receptors%2520for%2520ubiquitination%2520and%2520degradation%2520in%2520breast%2520and%2520prostate%2520cancer%26jtitle%3DOncogene%26date%3D2008%26volume%3D27%26spage%3D7201%26epage%3D7211%26doi%3D10.1038%2Fonc.2008.320" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span> <span> </span><span class="NLM_article-title">Development of stabilized peptide-based PROTACs against estrogen receptor α</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">628</span>– <span class="NLM_lpage">635</span>, <span class="refDoi"> DOI: 10.1021/acschembio.7b00985</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.7b00985" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A528%3ADC%252BC2sXitVaksr3M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=628-635&author=Y.+Jiangauthor=Q.+Dengauthor=H.+Zhaoauthor=M.+Xieauthor=L.+Chenauthor=F.+Yinauthor=X.+Qinauthor=W.+Zhengauthor=Y.+Zhaoauthor=Z.+Li&title=Development+of+stabilized+peptide-based+PROTACs+against+estrogen+receptor+%CE%B1&doi=10.1021%2Facschembio.7b00985"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Development of Stabilized Peptide-Based PROTACs against Estrogen Receptor α</span></div><div class="casAuthors">Jiang, Yanhong; Deng, Qiwen; Zhao, Hui; Xie, Mingsheng; Chen, Longjian; Yin, Feng; Qin, Xuan; Zheng, Weihao; Zhao, Yongjuan; Li, Zigang</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">628-635</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Peptide modulators targeting protein-protein interactions (PPIs) exhibit greater potential than small-mol. drugs in several important aspects including facile modification and relative large contact surface area.  Stabilized peptides constructed by variable chem. methods exhibit improved peptide stability and cell permeability compared to that of the linears.  Herein, we designed a stabilized peptide-based proteolysis-targeting chimera (PROTAC) targeting estrogen receptor α (ERα) by tethering an N-terminal aspartic acid cross-linked stabilized peptide ERα modulator (TD-PERM) with a pentapeptide that binds the Von Hippel-Lindau (VHL) E3 ubiquitin ligase complex.  The resulting heterobifunctional peptide (TD-PROTAC) selectively recruits ERα to the VHL E3 ligase complex, leading to the degrdn. of ERα in a proteasome-dependent manner.  Compared with the control peptides, TD-PROTAC shows significantly enhanced activities in reducing the transcription of the ERα-downstream genes and inhibiting the proliferation of ERα-pos. breast cancer cells.  In addn., in vivo expts. indicate that TD-PROTAC leads to tumor regression in the MCF-7 mouse xenograft model.  This work is a successful attempt to construct PROTACs based on cell-permeable stabilized peptides, which significantly broadens the chem. space of PROTACs and stabilized peptides.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovSDWxTR3LbLVg90H21EOLACvtfcHk0lgYxBoXe6nfhQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXitVaksr3M&md5=abcd6f72223751db6d96f8027cbb8c23</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Facschembio.7b00985&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.7b00985%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DDeng%26aufirst%3DQ.%26aulast%3DZhao%26aufirst%3DH.%26aulast%3DXie%26aufirst%3DM.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DYin%26aufirst%3DF.%26aulast%3DQin%26aufirst%3DX.%26aulast%3DZheng%26aufirst%3DW.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DZ.%26atitle%3DDevelopment%2520of%2520stabilized%2520peptide-based%2520PROTACs%2520against%2520estrogen%2520receptor%2520%25CE%25B1%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2018%26volume%3D13%26spage%3D628%26epage%3D635%26doi%3D10.1021%2Facschembio.7b00985" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G. E. B. D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulk, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Souza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhe-Paganon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">Phthalimide conjugation as a strategy for in vivo target protein degradation</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>348</i></span>,  <span class="NLM_fpage">1376</span>– <span class="NLM_lpage">1381</span>, <span class="refDoi"> DOI: 10.1126/science.aab1433</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=10.1126%2Fscience.aab1433" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=25999370" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A528%3ADC%252BC2MXpvFansLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=348&publication_year=2015&pages=1376-1381&author=G.+E.+B.+D.+L.+Winterauthor=D.+L.+Buckleyauthor=J.+Paulkauthor=J.+M.+Robertsauthor=A.+Souzaauthor=S.+Dhe-Paganonauthor=J.+E.+Bradner&title=Phthalimide+conjugation+as+a+strategy+for+in+vivo+target+protein+degradation&doi=10.1126%2Fscience.aab1433"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Phthalimide conjugation as a strategy for in vivo target protein degradation</span></div><div class="casAuthors">Winter, Georg E.; Buckley, Dennis L.; Paulk, Joshiawa; Roberts, Justin M.; Souza, Amanda; Dhe-Paganon, Sirano; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">348</span>
        (<span class="NLM_cas:issue">6241</span>),
    <span class="NLM_cas:pages">1376-1381</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The development of effective pharmacol. inhibitors of multidomain scaffold proteins, notably transcription factors, is a particularly challenging problem.  In part, this is because many small-mol. antagonists disrupt the activity of only one domain in the target protein.  The authors devised a chem. strategy that promotes ligand-dependent target protein degrdn. using as an example the transcriptional coactivator BRD4, a protein crit. for cancer cell growth and survival.  The authors appended a competitive antagonist of BET bromodomains to a phthalimide moiety to hijack the cereblon E3 ubiquitin ligase complex.  The resultant compd., dBET1, induced highly selective cereblon-dependent BET protein degrdn. in vitro and in vivo and delayed leukemia progression in mice.  A second series of probes resulted in selective degrdn. of the cytosolic protein FKBP12.  This chem. strategy for controlling target protein stability may have implications for therapeutically targeting previously intractable proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1ZW4zjRjXi7Vg90H21EOLACvtfcHk0lgYxBoXe6nfhQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXpvFansLk%253D&md5=b0277cc95d318d926b96e7a119066e3b</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1126%2Fscience.aab1433&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aab1433%26sid%3Dliteratum%253Aachs%26aulast%3DWinter%26aufirst%3DG.%2BE.%2BB.%2BD.%2BL.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DPaulk%26aufirst%3DJ.%26aulast%3DRoberts%26aufirst%3DJ.%2BM.%26aulast%3DSouza%26aufirst%3DA.%26aulast%3DDhe-Paganon%26aufirst%3DS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DPhthalimide%2520conjugation%2520as%2520a%2520strategy%2520for%2520in%2520vivo%2520target%2520protein%2520degradation%26jtitle%3DScience%26date%3D2015%26volume%3D348%26spage%3D1376%26epage%3D1381%26doi%3D10.1126%2Fscience.aab1433" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altieri, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raina, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hines, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crew, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Hijacking the E3 ubiquitin ligase cereblon to efficiently target BRD4</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">755</span>– <span class="NLM_lpage">763</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2015.05.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=10.1016%2Fj.chembiol.2015.05.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=26051217" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVWrtbrM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2015&pages=755-763&author=J.+Luauthor=Y.+Qianauthor=M.+Altieriauthor=H.+Dongauthor=J.+Wangauthor=K.+Rainaauthor=J.+Hinesauthor=J.+D.+Winklerauthor=A.+P.+Crewauthor=K.+Colemanauthor=C.+M.+Crews&title=Hijacking+the+E3+ubiquitin+ligase+cereblon+to+efficiently+target+BRD4&doi=10.1016%2Fj.chembiol.2015.05.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Hijacking the E3 Ubiquitin Ligase Cereblon to Efficiently Target BRD4</span></div><div class="casAuthors">Lu, Jing; Qian, Yimin; Altieri, Martha; Dong, Hanqing; Wang, Jing; Raina, Kanak; Hines, John; Winkler, James D.; Crew, Andrew P.; Coleman, Kevin; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">755-763</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">BRD4, a bromodomain and extraterminal domain (BET) family member, is an attractive target in multiple pathol. settings, particularly cancer.  While BRD4 inhibitors have shown some promise in MYC-driven malignancies such as Burkitt's lymphoma (BL), we show that BRD4 inhibitors lead to robust BRD4 protein accumulation, which may account for their limited suppression of MYC expression, modest antiproliferative activity, and lack of apoptotic induction.  To address these limitations we designed ARV-825, a hetero-bifunctional PROTAC (Proteolysis Targeting Chimera) that recruits BRD4 to the E3 ubiquitin ligase cereblon, leading to fast, efficient, and prolonged degrdn. of BRD4 in all BL cell lines tested.  Consequently, ARV-825 more effectively suppresses c-MYC levels and downstream signaling than small-mol. BRD4 inhibitors, resulting in more effective cell proliferation inhibition and apoptosis induction in BL.  Our findings provide strong evidence that cereblon-based PROTACs provide a better and more efficient strategy in targeting BRD4 than traditional small-mol. inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6Ol4grCG2RbVg90H21EOLACvtfcHk0lidTGKFnigoPg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVWrtbrM&md5=a64675e7a0cd006a8737be9730f447e3</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2015.05.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2015.05.009%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DQian%26aufirst%3DY.%26aulast%3DAltieri%26aufirst%3DM.%26aulast%3DDong%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DRaina%26aufirst%3DK.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DWinkler%26aufirst%3DJ.%2BD.%26aulast%3DCrew%26aufirst%3DA.%2BP.%26aulast%3DColeman%26aufirst%3DK.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DHijacking%2520the%2520E3%2520ubiquitin%2520ligase%2520cereblon%2520to%2520efficiently%2520target%2520BRD4%26jtitle%3DChem.%2520Biol.%26date%3D2015%26volume%3D22%26spage%3D755%26epage%3D763%26doi%3D10.1016%2Fj.chembiol.2015.05.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bondeson, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mares, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, I. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ko, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campos, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miah, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulholland, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Routly, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gustafson, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zinn, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shimamura, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergamini, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faelth-Savitski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bantscheff, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willard, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flanagan, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casillas, L. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Votta, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">den Besten, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Famm, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kruidenier, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harling, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Churcher, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Catalytic in vivo protein knockdown by small-molecule PROTACs</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">611</span>– <span class="NLM_lpage">617</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1858</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=10.1038%2Fnchembio.1858" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=26075522" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFeju7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=611-617&author=D.+P.+Bondesonauthor=A.+Maresauthor=I.+E.+Smithauthor=E.+Koauthor=S.+Camposauthor=A.+H.+Miahauthor=K.+E.+Mulhollandauthor=N.+Routlyauthor=D.+L.+Buckleyauthor=J.+L.+Gustafsonauthor=N.+Zinnauthor=P.+Grandiauthor=S.+Shimamuraauthor=G.+Bergaminiauthor=M.+Faelth-Savitskiauthor=M.+Bantscheffauthor=C.+Coxauthor=D.+A.+Gordonauthor=R.+R.+Willardauthor=J.+J.+Flanaganauthor=L.+N.+Casillasauthor=B.+J.+Vottaauthor=W.+den+Bestenauthor=K.+Fammauthor=L.+Kruidenierauthor=P.+S.+Carterauthor=J.+D.+Harlingauthor=I.+Churcherauthor=C.+M.+Crews&title=Catalytic+in+vivo+protein+knockdown+by+small-molecule+PROTACs&doi=10.1038%2Fnchembio.1858"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Catalytic in vivo protein knockdown by small-molecule PROTACs</span></div><div class="casAuthors">Bondeson, Daniel P.; Mares, Alina; Smith, Ian E. D.; Ko, Eunhwa; Campos, Sebastien; Miah, Afjal H.; Mulholland, Katie E.; Routly, Natasha; Buckley, Dennis L.; Gustafson, Jeffrey L.; Zinn, Nico; Grandi, Paola; Shimamura, Satoko; Bergamini, Giovanna; Faelth-Savitski, Maria; Bantscheff, Marcus; Cox, Carly; Gordon, Deborah A.; Willard, Ryan R.; Flanagan, John J.; Casillas, Linda N.; Votta, Bartholomew J.; den Besten, Willem; Famm, Kristoffer; Kruidenier, Laurens; Carter, Paul S.; Harling, John D.; Churcher, Ian; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">611-617</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The current predominant therapeutic paradigm is based on maximizing drug-receptor occupancy to achieve clin. benefit.  This strategy, however, generally requires excessive drug concns. to ensure sufficient occupancy, often leading to adverse side effects.  Here, we describe major improvements to the proteolysis targeting chimeras (PROTACs) method, a chem. knockdown strategy in which a heterobifunctional mol. recruits a specific protein target to an E3 ubiquitin ligase, resulting in the target's ubiquitination and degrdn.  These compds. behave catalytically in their ability to induce the ubiquitination of super-stoichiometric quantities of proteins, providing efficacy that is not limited by equil. occupancy.  We present two PROTACs that are capable of specifically reducing protein levels by >90% at nanomolar concns.  In addn., mouse studies indicate that they provide broad tissue distribution and knockdown of the targeted protein in tumor xenografts.  Together, these data demonstrate a protein knockdown system combining many of the favorable properties of small-mol. agents with the potent protein knockdown of RNAi and CRISPR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonhnajD3CxS7Vg90H21EOLACvtfcHk0lidTGKFnigoPg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFeju7rP&md5=31eb8756c5661f78ab4527a50dc1e251</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1858&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1858%26sid%3Dliteratum%253Aachs%26aulast%3DBondeson%26aufirst%3DD.%2BP.%26aulast%3DMares%26aufirst%3DA.%26aulast%3DSmith%26aufirst%3DI.%2BE.%26aulast%3DKo%26aufirst%3DE.%26aulast%3DCampos%26aufirst%3DS.%26aulast%3DMiah%26aufirst%3DA.%2BH.%26aulast%3DMulholland%26aufirst%3DK.%2BE.%26aulast%3DRoutly%26aufirst%3DN.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DGustafson%26aufirst%3DJ.%2BL.%26aulast%3DZinn%26aufirst%3DN.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DShimamura%26aufirst%3DS.%26aulast%3DBergamini%26aufirst%3DG.%26aulast%3DFaelth-Savitski%26aufirst%3DM.%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DCox%26aufirst%3DC.%26aulast%3DGordon%26aufirst%3DD.%2BA.%26aulast%3DWillard%26aufirst%3DR.%2BR.%26aulast%3DFlanagan%26aufirst%3DJ.%2BJ.%26aulast%3DCasillas%26aufirst%3DL.%2BN.%26aulast%3DVotta%26aufirst%3DB.%2BJ.%26aulast%3Dden%2BBesten%26aufirst%3DW.%26aulast%3DFamm%26aufirst%3DK.%26aulast%3DKruidenier%26aufirst%3DL.%26aulast%3DCarter%26aufirst%3DP.%2BS.%26aulast%3DHarling%26aufirst%3DJ.%2BD.%26aulast%3DChurcher%26aufirst%3DI.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DCatalytic%2520in%2520vivo%2520protein%2520knockdown%2520by%2520small-molecule%2520PROTACs%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26spage%3D611%26epage%3D617%26doi%3D10.1038%2Fnchembio.1858" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lai, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toure, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hellerschmied, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salami, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaime-Figueroa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ko, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hines, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Modular PROTAC design for the degradation of oncogenic BCR-ABL</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">807</span>– <span class="NLM_lpage">810</span>, <span class="refDoi"> DOI: 10.1002/anie.201507634</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=10.1002%2Fanie.201507634" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVOlt7rN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2016&pages=807-810&author=A.+C.+Laiauthor=M.+Toureauthor=D.+Hellerschmiedauthor=J.+Salamiauthor=S.+Jaime-Figueroaauthor=E.+Koauthor=J.+Hinesauthor=C.+M.+Crews&title=Modular+PROTAC+design+for+the+degradation+of+oncogenic+BCR-ABL&doi=10.1002%2Fanie.201507634"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Modular PROTAC Design for the Degradation of Oncogenic BCR-ABL</span></div><div class="casAuthors">Lai, Ashton C.; Toure, Momar; Hellerschmied, Doris; Salami, Jemilat; Jaime-Figueroa, Saul; Ko, Eunhwa; Hines, John; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">807-810</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Proteolysis Targeting Chimera (PROTAC) technol. is a rapidly emerging alternative therapeutic strategy with the potential to address many of the challenges currently faced in modern drug development programs.  PROTAC technol. employs small mols. that recruit target proteins for ubiquitination and removal by the proteasome.  The synthesis of PROTAC compds. that mediate the degrdn. of c-ABL and BCR-ABL by recruiting either Cereblon or Von Hippel Lindau E3 ligases is reported.  During the course of their development, the authors discovered that the capacity of a PROTAC to induce degrdn. involves more than just target binding: the identity of the inhibitor warhead and the recruited E3 ligase largely det. the degrdn. profiles of the compds.; thus, as a starting point for PROTAC development, both the target ligand and the recruited E3 ligase should be varied to rapidly generate a PROTAC with the desired degrdn. profile.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdcSTHt5WfILVg90H21EOLACvtfcHk0lidTGKFnigoPg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVOlt7rN&md5=58a5d78f32bc8a21fd1a6e0c38b8979c</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1002%2Fanie.201507634&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201507634%26sid%3Dliteratum%253Aachs%26aulast%3DLai%26aufirst%3DA.%2BC.%26aulast%3DToure%26aufirst%3DM.%26aulast%3DHellerschmied%26aufirst%3DD.%26aulast%3DSalami%26aufirst%3DJ.%26aulast%3DJaime-Figueroa%26aufirst%3DS.%26aulast%3DKo%26aufirst%3DE.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DModular%2520PROTAC%2520design%2520for%2520the%2520degradation%2520of%2520oncogenic%2520BCR-ABL%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2016%26volume%3D55%26spage%3D807%26epage%3D810%26doi%3D10.1002%2Fanie.201507634" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Robb, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Contreras, J. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kour, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abid, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sonawane, Y. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zahid, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murry, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Natarajan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rana, S.</span></span> <span> </span><span class="NLM_article-title">Chemically induced degradation of CDK9 by a proteolysis targeting chimera (PROTAC)</span>. <i>Chem. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">7577</span>– <span class="NLM_lpage">7580</span>, <span class="refDoi"> DOI: 10.1039/C7CC03879H</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=10.1039%2FC7CC03879H" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=28636052" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A528%3ADC%252BC2sXpvFyjsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2017&pages=7577-7580&author=C.+M.+Robbauthor=J.+I.+Contrerasauthor=S.+Kourauthor=M.+A.+Taylorauthor=M.+Abidauthor=Y.+A.+Sonawaneauthor=M.+Zahidauthor=D.+J.+Murryauthor=A.+Natarajanauthor=S.+Rana&title=Chemically+induced+degradation+of+CDK9+by+a+proteolysis+targeting+chimera+%28PROTAC%29&doi=10.1039%2FC7CC03879H"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Chemically induced degradation of CDK9 by a proteolysis targeting chimera (PROTAC)</span></div><div class="casAuthors">Robb, Caroline M.; Contreras, Jacob I.; Kour, Smit; Taylor, Margaret A.; Abid, Mohammad; Sonawane, Yogesh A.; Zahid, Muhammad; Murry, Daryl J.; Natarajan, Amarnath; Rana, Sandeep</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Communications (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">54</span>),
    <span class="NLM_cas:pages">7577-7580</span>CODEN:
                <span class="NLM_cas:coden">CHCOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-7345</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Cyclin-dependent kinase 9 (CDK9), a member of the cyclin-dependent protein kinase (CDK) family, is involved in transcriptional elongation of several target genes.  CDK9 is ubiquitously expressed and has been shown to contribute to a variety of malignancies such as pancreatic, prostate and breast cancers.  Here we report the development of a heterobifunctional small mol. proteolysis targeting chimera (PROTAC) capable of cereblon (CRBN)-mediated proteasomal degrdn. of CDK9.  In HCT116 cells, it selectively degrades CDK9 while sparing other CDK family members.  This is the first example of a PROTAC that selectively degrades CDK9.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXgLtOTwvlnbVg90H21EOLACvtfcHk0lgDEOdw0AETlA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXpvFyjsLw%253D&md5=a9aa8ebf8109f11206740efd1b5e09a2</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1039%2FC7CC03879H&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC7CC03879H%26sid%3Dliteratum%253Aachs%26aulast%3DRobb%26aufirst%3DC.%2BM.%26aulast%3DContreras%26aufirst%3DJ.%2BI.%26aulast%3DKour%26aufirst%3DS.%26aulast%3DTaylor%26aufirst%3DM.%2BA.%26aulast%3DAbid%26aufirst%3DM.%26aulast%3DSonawane%26aufirst%3DY.%2BA.%26aulast%3DZahid%26aufirst%3DM.%26aulast%3DMurry%26aufirst%3DD.%2BJ.%26aulast%3DNatarajan%26aufirst%3DA.%26aulast%3DRana%26aufirst%3DS.%26atitle%3DChemically%2520induced%2520degradation%2520of%2520CDK9%2520by%2520a%2520proteolysis%2520targeting%2520chimera%2520%2528PROTAC%2529%26jtitle%3DChem.%2520Commun.%26date%3D2017%26volume%3D53%26spage%3D7577%26epage%3D7580%26doi%3D10.1039%2FC7CC03879H" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, X. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Proteolysis targeting chimeras (PROTACs) of anaplastic lymphoma kinase (ALK)</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>151</i></span>,  <span class="NLM_fpage">304</span>– <span class="NLM_lpage">314</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2018.03.071</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=10.1016%2Fj.ejmech.2018.03.071" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=29627725" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A528%3ADC%252BC1cXnt1ejs7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=151&publication_year=2018&pages=304-314&author=C.+Zhangauthor=X.+R.+Hanauthor=X.+Yangauthor=B.+Jiangauthor=J.+Liuauthor=Y.+Xiongauthor=J.+Jin&title=Proteolysis+targeting+chimeras+%28PROTACs%29+of+anaplastic+lymphoma+kinase+%28ALK%29&doi=10.1016%2Fj.ejmech.2018.03.071"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Proteolysis Targeting Chimeras (PROTACs) of Anaplastic Lymphoma Kinase (ALK)</span></div><div class="casAuthors">Zhang, Chengwei; Han, Xiao-Ran; Yang, Xiaobao; Jiang, Biao; Liu, Jing; Xiong, Yue; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">151</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">304-314</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Anaplastic lymphoma kinase (ALK) activation has been assocd. with many types of human cancer.  Significant efforts have been devoted to the development of ALK inhibitors to antagonize the kinase activity of ALK.  Four ALK inhibitors have been approved by the FDA to date for treating patients with ALK-pos. nonsmall cell lung cancers (NSCLC).  However, drug resistance has been obsd. in the majority of patients treated with these inhibitors.  New therapeutic strategies (e.g., compds. with novel mechanisms of action) are needed to overcome the drug resistance issue.  The emerging PROTAC (Proteolysis Targeting Chimera) technol. has been successfully applied to selective degrdn. of multiple protein targets, but not ALK.  Since ALK protein levels are not important for viability in mammals, ALK PROTACs could lead to novel therapeutics with minimal toxicity.  Here the authors report the design, synthesis and biol. evaluation of novel PROTACs (degraders) of ALK.  MS4077 (I) and MS4078 (II) potently decreased cellular levels of oncogenic active ALK fusion proteins in a concn.- and time-dependent manner in SU-DHL-1 lymphoma and NCI-H2228 lung cancer cells.  The ALK protein degrdn. induced by compds. I and II was cereblon and proteasome dependent.  In addn., compds. I and II potently inhibited proliferation of SU-DHL-1 cells.  Furthermore, compd. II displayed good plasma exposure in a mouse pharmacokinetic study, thus is suitable for in vivo efficacy studies.  The authors also developed MS4748 and MS4740 , very close analogs of I and II resp., which are incapable to degrade the ALK fusion proteins, as neg. controls.  All 4 compds. are valuable chem. tools for investigating effects of ALK pharmacol. degrdn.  The study paved the way for developing the next generation of ALK PROTACs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPGM3aAZZ6h7Vg90H21EOLACvtfcHk0lgDEOdw0AETlA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXnt1ejs7Y%253D&md5=804324b95034f38b46e45f89e18e55be</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2018.03.071&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2018.03.071%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DHan%26aufirst%3DX.%2BR.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DJiang%26aufirst%3DB.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DXiong%26aufirst%3DY.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DProteolysis%2520targeting%2520chimeras%2520%2528PROTACs%2529%2520of%2520anaplastic%2520lymphoma%2520kinase%2520%2528ALK%2529%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2018%26volume%3D151%26spage%3D304%26epage%3D314%26doi%3D10.1016%2Fj.ejmech.2018.03.071" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Z.</span></span> <span> </span><span class="NLM_article-title">Discovery of a Keap1-dependent peptide PROTAC to knockdown Tau by ubiquitination-proteasome degradation pathway</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>146</i></span>,  <span class="NLM_fpage">251</span>– <span class="NLM_lpage">259</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2018.01.063</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=10.1016%2Fj.ejmech.2018.01.063" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=29407955" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisFehsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=146&publication_year=2018&pages=251-259&author=M.+Luauthor=T.+Liuauthor=Q.+Jiaoauthor=J.+Jiauthor=M.+Taoauthor=Y.+Liuauthor=Q.+Youauthor=Z.+Jiang&title=Discovery+of+a+Keap1-dependent+peptide+PROTAC+to+knockdown+Tau+by+ubiquitination-proteasome+degradation+pathway&doi=10.1016%2Fj.ejmech.2018.01.063"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Keap1-dependent peptide PROTAC to knockdown Tau by ubiquitination-proteasome degradation pathway</span></div><div class="casAuthors">Lu, Mengchen; Liu, Tian; Jiao, Qiong; Ji, Jianai; Tao, Mengmin; Liu, Yijun; You, Qidong; Jiang, Zhengyu</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">146</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">251-259</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Induced protein degrdn. by PROTACs has emerged as a promising strategy to target nonenzymic proteins inside the cell.  The aim of this study was to identify Keap1, a substrate adaptor protein for ubiquitin E3 ligase involved in oxidative stress regulation, as a novel candidate for PROTACs that can be applied in the degrdn. of the nonenzymic protein Tau.  A peptide PROTAC by recruiting Keap1-Cul3 ubiquitin E3 ligase was developed and applied in the degrdn. of intracellular Tau.  Peptide 1 showed strong in vitro binding with Keap1 and Tau.  With proper cell permeability, peptide 1 was found to colocalize with cellular Keap1 and resulted in the coimmunopptn. of Tau and Keap1.  The results of flow cytometry and western blotting assays showed that peptide 1 can downregulate the intracellular Tau level in both time- and concn.-dependent manner.  The application of Keap1 siRNA silencing and the proteasome inhibitor MG132 confirmed that peptide 1 could promote the Keap1-dependent poly-ubiquitination and proteasome-dependent degrdn. of Tau.  The results suggested that using PROTACs to recruit Keap1 to induce the degrdn. of Tau may show promising character in the treatment of neurodegenerative disease.  In addn., our research demonstrated that Keap1 should be a promising E3 ligase adaptor to be used in the design of novel PROTACs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9KC7pnPqlL7Vg90H21EOLACvtfcHk0lgDEOdw0AETlA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisFehsbY%253D&md5=165e6f510e4e2024aca6c4135d2840b2</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2018.01.063&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2018.01.063%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DT.%26aulast%3DJiao%26aufirst%3DQ.%26aulast%3DJi%26aufirst%3DJ.%26aulast%3DTao%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DYou%26aufirst%3DQ.%26aulast%3DJiang%26aufirst%3DZ.%26atitle%3DDiscovery%2520of%2520a%2520Keap1-dependent%2520peptide%2520PROTAC%2520to%2520knockdown%2520Tau%2520by%2520ubiquitination-proteasome%2520degradation%2520pathway%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2018%26volume%3D146%26spage%3D251%26epage%3D259%26doi%3D10.1016%2Fj.ejmech.2018.01.063" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Burslem, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaime-Figueroa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McQuaid, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bondeson, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toure, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crew, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hines, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">The advantages of targeted protein degradation over inhibition: an RTK case study</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">67</span>– <span class="NLM_lpage">77</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2017.09.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=10.1016%2Fj.chembiol.2017.09.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=29129716" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrjM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=67-77&author=G.+M.+Burslemauthor=B.+E.+Smithauthor=A.+C.+Laiauthor=S.+Jaime-Figueroaauthor=D.+C.+McQuaidauthor=D.+P.+Bondesonauthor=M.+Toureauthor=H.+Dongauthor=Y.+Qianauthor=J.+Wangauthor=A.+P.+Crewauthor=J.+Hinesauthor=C.+M.+Crews&title=The+advantages+of+targeted+protein+degradation+over+inhibition%3A+an+RTK+case+study&doi=10.1016%2Fj.chembiol.2017.09.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">The Advantages of Targeted Protein Degradation Over Inhibition: An RTK Case Study</span></div><div class="casAuthors">Burslem, George M.; Smith, Blake E.; Lai, Ashton C.; Jaime-Figueroa, Saul; McQuaid, Daniel C.; Bondeson, Daniel P.; Toure, Momar; Dong, Hanqing; Qian, Yimin; Wang, Jing; Crew, Andrew P.; Hines, John; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">67-77.e3</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Proteolysis targeting chimera (PROTAC) technol. has emerged over the last two decades as a powerful tool for targeted degrdn. of endogenous proteins.  Herein we describe the development of PROTACs for receptor tyrosine kinases, a protein family yet to be targeted for induced protein degrdn.  The use of VHL-recruiting PROTACs against this protein family reveals several advantages of degrdn. over inhibition alone: direct comparisons of fully functional, target-degrading PROTACs with target-inhibiting variants that contain an inactivated E3 ligase-recruiting ligand show that degrdn. leads to more potent inhibition of cell proliferation and a more durable and sustained downstream signaling response, and thus addresses the kinome rewiring challenge seen with many receptor tyrosine kinase inhibitors.  Combined, these findings demonstrate the ability to target receptor tyrosine kinases for degrdn. using the PROTAC technol. and outline the advantages of this degrdn.-based approach.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4yDhRDWqNv7Vg90H21EOLACvtfcHk0ljbACTz6As5iQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrjM&md5=0ac39a6f5836ea3772a0396356efb630</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2017.09.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2017.09.009%26sid%3Dliteratum%253Aachs%26aulast%3DBurslem%26aufirst%3DG.%2BM.%26aulast%3DSmith%26aufirst%3DB.%2BE.%26aulast%3DLai%26aufirst%3DA.%2BC.%26aulast%3DJaime-Figueroa%26aufirst%3DS.%26aulast%3DMcQuaid%26aufirst%3DD.%2BC.%26aulast%3DBondeson%26aufirst%3DD.%2BP.%26aulast%3DToure%26aufirst%3DM.%26aulast%3DDong%26aufirst%3DH.%26aulast%3DQian%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DCrew%26aufirst%3DA.%2BP.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DThe%2520advantages%2520of%2520targeted%2520protein%2520degradation%2520over%2520inhibition%253A%2520an%2520RTK%2520case%2520study%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2018%26volume%3D25%26spage%3D67%26epage%3D77%26doi%3D10.1016%2Fj.chembiol.2017.09.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schiedel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herp, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hammelmann, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swyter, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lehotzky, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robaa, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olah, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ovadi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sippl, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jung, M.</span></span> <span> </span><span class="NLM_article-title">Chemically induced degradation of sirtuin 2 (sirt2) by a proteolysis targeting chimera (PROTAC) based on sirtuin rearranging ligands (SirReals)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">482</span>– <span class="NLM_lpage">491</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01872</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01872" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A528%3ADC%252BC2sXlslSltrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=482-491&author=M.+Schiedelauthor=D.+Herpauthor=S.+Hammelmannauthor=S.+Swyterauthor=A.+Lehotzkyauthor=D.+Robaaauthor=J.+Olahauthor=J.+Ovadiauthor=W.+Sipplauthor=M.+Jung&title=Chemically+induced+degradation+of+sirtuin+2+%28sirt2%29+by+a+proteolysis+targeting+chimera+%28PROTAC%29+based+on+sirtuin+rearranging+ligands+%28SirReals%29&doi=10.1021%2Facs.jmedchem.6b01872"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Chemically Induced Degradation of Sirtuin 2 (Sirt2) by a Proteolysis Targeting Chimera (PROTAC) Based on Sirtuin Rearranging Ligands (SirReals)</span></div><div class="casAuthors">Schiedel, Matthias; Herp, Daniel; Hammelmann, Soeren; Swyter, Soeren; Lehotzky, Attila; Robaa, Dina; Olah, Judit; Ovadi, Judit; Sippl, Wolfgang; Jung, Manfred</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">482-491</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Here the authors report the development of a proteolysis targeting chimera (PROTAC) I based on the combination of the unique features of the sirtuin rearranging ligands (SirReals) as highly potent and isotype-selective Sirt2 inhibitors with thalidomide, a bona fide cereblon ligand.  For the first time, the authors report the formation of a PROTAC by Cu(I)-catalyzed cycloaddn. of a thalidomide-derived azide to an alkynylated inhibitor.  This thalidomide-derived azide as well as the highly versatile linking strategy can be readily adapted to alkynylated ligands of other targets.  In HeLa cells, the SirReal-based PROTAC induced isotype-selective Sirt2 degrdn. that results in the hyperacetylation of the microtubule network coupled with enhanced process elongation.  Thus, the SirReal-based PROTAC is the first example of a probe that is able to chem. induce the degrdn. of an epigenetic eraser protein.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrLUKzL4R5C0bVg90H21EOLACvtfcHk0ljbACTz6As5iQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXlslSltrw%253D&md5=3be73e320c6d40bb87a18cad75c16bf4</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01872&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01872%26sid%3Dliteratum%253Aachs%26aulast%3DSchiedel%26aufirst%3DM.%26aulast%3DHerp%26aufirst%3DD.%26aulast%3DHammelmann%26aufirst%3DS.%26aulast%3DSwyter%26aufirst%3DS.%26aulast%3DLehotzky%26aufirst%3DA.%26aulast%3DRobaa%26aufirst%3DD.%26aulast%3DOlah%26aufirst%3DJ.%26aulast%3DOvadi%26aufirst%3DJ.%26aulast%3DSippl%26aufirst%3DW.%26aulast%3DJung%26aufirst%3DM.%26atitle%3DChemically%2520induced%2520degradation%2520of%2520sirtuin%25202%2520%2528sirt2%2529%2520by%2520a%2520proteolysis%2520targeting%2520chimera%2520%2528PROTAC%2529%2520based%2520on%2520sirtuin%2520rearranging%2520ligands%2520%2528SirReals%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D482%26epage%3D491%26doi%3D10.1021%2Facs.jmedchem.6b01872" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shibata, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagai, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morita, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ujikawa, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohoka, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hattori, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koyama, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sano, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imaeda, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nara, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naito, M.</span></span> <span> </span><span class="NLM_article-title">Development of protein degradation inducers of androgen receptor by conjugation of androgen receptor ligands and inhibitor of apoptosis protein ligands</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">543</span>– <span class="NLM_lpage">575</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00168</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00168" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A528%3ADC%252BC2sXpslSlu7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=543-575&author=N.+Shibataauthor=K.+Nagaiauthor=Y.+Moritaauthor=O.+Ujikawaauthor=N.+Ohokaauthor=T.+Hattoriauthor=R.+Koyamaauthor=O.+Sanoauthor=Y.+Imaedaauthor=H.+Naraauthor=N.+Choauthor=M.+Naito&title=Development+of+protein+degradation+inducers+of+androgen+receptor+by+conjugation+of+androgen+receptor+ligands+and+inhibitor+of+apoptosis+protein+ligands&doi=10.1021%2Facs.jmedchem.7b00168"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Development of Protein Degradation Inducers of Androgen Receptor by Conjugation of Androgen Receptor Ligands and Inhibitor of Apoptosis Protein Ligands</span></div><div class="casAuthors">Shibata, Norihito; Nagai, Katsunori; Morita, Yoko; Ujikawa, Osamu; Ohoka, Nobumichi; Hattori, Takayuki; Koyama, Ryokichi; Sano, Osamu; Imaeda, Yasuhiro; Nara, Hiroshi; Cho, Nobuo; Naito, Mikihiko</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">543-575</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Targeted protein degrdn. using small mols. is a novel strategy for drug development.  The authors have developed hybrid mols. named specific and nongenetic inhibitor of apoptosis protein [IAP]-dependent protein erasers (SNIPERs) that recruit IAP ubiquitin ligases to degrade target proteins.  Here, the authors show novel SNIPERs capable of inducing proteasomal degrdn. of the androgen receptor (AR).  Through derivatization of the SNIPER(AR) mol. at the AR ligand and IAP ligand and linker, the authors developed SNIPER(AR)-51 (I), which shows effective protein knockdown activity against AR.  Consistent with the degrdn. of the AR protein, I inhibits AR-mediated gene expression and proliferation of androgen-dependent prostate cancer cells.  In addn., I efficiently induces caspase activation and apoptosis in prostate cancer cells, which was not obsd. in the cells treated with AR antagonists.  These results suggest that SNIPER(AR)s could be leads for an anticancer drug against prostate cancers that exhibit AR-dependent proliferation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqW6OG4BZYg1rVg90H21EOLACvtfcHk0ljulLFZVTa2UQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXpslSlu7k%253D&md5=eeefbe187f2f93ca32518f6a3258b903</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00168&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00168%26sid%3Dliteratum%253Aachs%26aulast%3DShibata%26aufirst%3DN.%26aulast%3DNagai%26aufirst%3DK.%26aulast%3DMorita%26aufirst%3DY.%26aulast%3DUjikawa%26aufirst%3DO.%26aulast%3DOhoka%26aufirst%3DN.%26aulast%3DHattori%26aufirst%3DT.%26aulast%3DKoyama%26aufirst%3DR.%26aulast%3DSano%26aufirst%3DO.%26aulast%3DImaeda%26aufirst%3DY.%26aulast%3DNara%26aufirst%3DH.%26aulast%3DCho%26aufirst%3DN.%26aulast%3DNaito%26aufirst%3DM.%26atitle%3DDevelopment%2520of%2520protein%2520degradation%2520inducers%2520of%2520androgen%2520receptor%2520by%2520conjugation%2520of%2520androgen%2520receptor%2520ligands%2520and%2520inhibitor%2520of%2520apoptosis%2520protein%2520ligands%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D543%26epage%3D575%26doi%3D10.1021%2Facs.jmedchem.7b00168" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Crew, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raina, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vigil, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Serebrenik, Y. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamman, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morgan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferraro, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neklesa, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Identification and characterization of von Hippel-Lindau-recruiting proteolysis targeting chimeras (PROTACs) of TANK-binding kinase 1</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">583</span>– <span class="NLM_lpage">598</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00635</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00635" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtFelu7rK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=583-598&author=A.+P.+Crewauthor=K.+Rainaauthor=H.+Dongauthor=Y.+Qianauthor=J.+Wangauthor=D.+Vigilauthor=Y.+V.+Serebrenikauthor=B.+D.+Hammanauthor=A.+Morganauthor=C.+Ferraroauthor=K.+Siuauthor=T.+K.+Neklesaauthor=J.+D.+Winklerauthor=K.+G.+Colemanauthor=C.+M.+Crews&title=Identification+and+characterization+of+von+Hippel-Lindau-recruiting+proteolysis+targeting+chimeras+%28PROTACs%29+of+TANK-binding+kinase+1&doi=10.1021%2Facs.jmedchem.7b00635"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and Characterization of Von Hippel-Lindau-Recruiting Proteolysis Targeting Chimeras (PROTACs) of TANK-Binding Kinase 1</span></div><div class="casAuthors">Crew, Andrew P.; Raina, Kanak; Dong, Hanqing; Qian, Yimin; Wang, Jing; Vigil, Dominico; Serebrenik, Yevgeniy V.; Hamman, Brian D.; Morgan, Alicia; Ferraro, Caterina; Siu, Kam; Neklesa, Taavi K.; Winkler, James D.; Coleman, Kevin G.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">583-598</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Proteolysis Targeting Chimeras (PROTACs) are bifunctional mols. that recruit an E3 ligase to a target protein to facilitate ubiquitination and subsequent degrdn. of that protein.  While the field of targeted degraders is still relatively young, the potential for this modality to become a differentiated and therapeutic reality is strong, such that both academic and pharmaceutical institutions are now entering this interesting area of research.  In this paper, the authors describe a broadly applicable process for identifying degrader hits based around the serine/threonine kinase TANK-binding kinase 1 (TBK1), and have generalized the key structural elements assocd. with degrdn. activities.  Compd. I is a potent hit (TBK1 DC50 = 12 nM, Dmax = 96%) with excellent selectivity against a related kinase IKKε, which was further used as a chem. tool to assess TBK1 as a target in mutant K-Ras cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoIJWCnFncQRLVg90H21EOLACvtfcHk0ljulLFZVTa2UQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtFelu7rK&md5=41c3627e05ca31aee3ba85815685fcb2</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00635&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00635%26sid%3Dliteratum%253Aachs%26aulast%3DCrew%26aufirst%3DA.%2BP.%26aulast%3DRaina%26aufirst%3DK.%26aulast%3DDong%26aufirst%3DH.%26aulast%3DQian%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DVigil%26aufirst%3DD.%26aulast%3DSerebrenik%26aufirst%3DY.%2BV.%26aulast%3DHamman%26aufirst%3DB.%2BD.%26aulast%3DMorgan%26aufirst%3DA.%26aulast%3DFerraro%26aufirst%3DC.%26aulast%3DSiu%26aufirst%3DK.%26aulast%3DNeklesa%26aufirst%3DT.%2BK.%26aulast%3DWinkler%26aufirst%3DJ.%2BD.%26aulast%3DColeman%26aufirst%3DK.%2BG.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DIdentification%2520and%2520characterization%2520of%2520von%2520Hippel-Lindau-recruiting%2520proteolysis%2520targeting%2520chimeras%2520%2528PROTACs%2529%2520of%2520TANK-binding%2520kinase%25201%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D583%26epage%3D598%26doi%3D10.1021%2Facs.jmedchem.7b00635" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hwang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, Y.</span></span> <span> </span><span class="NLM_article-title">PROTAC-induced BTK degradation as a novel therapy for mutated BTK C481S induced ibrutinib-resistant B-cell malignancies</span>. <i>Cell Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">779</span>– <span class="NLM_lpage">781</span>, <span class="refDoi"> DOI: 10.1038/s41422-018-0055-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=10.1038%2Fs41422-018-0055-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=29875397" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1ektbzO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2018&pages=779-781&author=Y.+Sunauthor=X.+Zhaoauthor=N.+Dingauthor=H.+Gaoauthor=Y.+Wuauthor=Y.+Yangauthor=M.+Zhaoauthor=J.+Hwangauthor=Y.+Songauthor=W.+Liuauthor=Y.+Rao&title=PROTAC-induced+BTK+degradation+as+a+novel+therapy+for+mutated+BTK+C481S+induced+ibrutinib-resistant+B-cell+malignancies&doi=10.1038%2Fs41422-018-0055-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">PROTAC-induced BTK degradation as a novel therapy for mutated BTK C481S induced ibrutinib-resistant B-cell malignancies</span></div><div class="casAuthors">Sun, Yonghui; Zhao, Xingwang; Ding, Ning; Gao, Hongying; Wu, Yue; Yang, Yiqing; Zhao, Meng; Hwang, Jinseok; Song, Yuqin; Liu, Wanli; Rao, Yu</div><div class="citationInfo"><span class="NLM_cas:title">Cell Research</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">779-781</span>CODEN:
                <span class="NLM_cas:coden">CREEB6</span>;
        ISSN:<span class="NLM_cas:issn">1001-0602</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">In this study, for the first time, we report the development of BTK-targeting degraders using the PROTAC strategy.  These PROTACs could ef fi ciently degrade ibrutinib-sensitive BTK-WT (wild type).  More importantly, our newly designed PROTACs also significantly induced the degrdn. of ibrutinib resistant BTK-C481S (50% degrdn. efficiency at 30 nM).  Furthermore, our PROTAC mols. ef fi ciently inhibited cell proliferation and colony formation, while exhibited no obvious inhibition (>1000 nM) of ITK, EGFR, and TEC, which are major off-targets of ibrutinib.  These data demonstrate the strong potential for developing PROTAC-based therapeutic mols.  The results indicate that the PROTAC strategy (protein degrdn. instead of inhibition) could be adopted as a general and powerful therapeutic treatment for drug-resistant cancers in the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqg_FLdaUP_HLVg90H21EOLACvtfcHk0ljulLFZVTa2UQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1ektbzO&md5=1cca51ac76ecb3b491da0117b21271fb</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1038%2Fs41422-018-0055-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41422-018-0055-1%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DY.%26aulast%3DZhao%26aufirst%3DX.%26aulast%3DDing%26aufirst%3DN.%26aulast%3DGao%26aufirst%3DH.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DZhao%26aufirst%3DM.%26aulast%3DHwang%26aufirst%3DJ.%26aulast%3DSong%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DW.%26aulast%3DRao%26aufirst%3DY.%26atitle%3DPROTAC-induced%2520BTK%2520degradation%2520as%2520a%2520novel%2520therapy%2520for%2520mutated%2520BTK%2520C481S%2520induced%2520ibrutinib-resistant%2520B-cell%2520malignancies%26jtitle%3DCell%2520Res.%26date%3D2018%26volume%3D28%26spage%3D779%26epage%3D781%26doi%3D10.1038%2Fs41422-018-0055-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leisten, E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, W.</span></span> <span> </span><span class="NLM_article-title">Development of the first small molecule histone deacetylase 6 (HDAC6) degraders</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">2493</span>– <span class="NLM_lpage">2497</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2018.05.057</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=10.1016%2Fj.bmcl.2018.05.057" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=29871848" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtVOlsrfF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2018&pages=2493-2497&author=K.+Yangauthor=Y.+Songauthor=H.+Xieauthor=H.+Wuauthor=Y.+T.+Wuauthor=E.+D.+Leistenauthor=W.+Tang&title=Development+of+the+first+small+molecule+histone+deacetylase+6+%28HDAC6%29+degraders&doi=10.1016%2Fj.bmcl.2018.05.057"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Development of the first small molecule histone deacetylase 6 (HDAC6) degraders</span></div><div class="casAuthors">Yang, Ka; Song, Yanling; Xie, Haibo; Wu, Hao; Wu, Yi-Ting; Leisten, Eric D.; Tang, Weiping</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">2493-2497</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Histone deacetylases (HDACs) decrease the acetylation level of histones and other non-histone proteins.  Over expression of HDACs have been obsd. in cancers and other diseases.  Targeted protein degrdn. by "hijacking" the natural ubiquitin-proteasome-system (UPS) recently emerged as a novel technol. to "knock-out" endogenous disease-causing proteins.  The authors applied this strategy to the development of the first small mol. degraders for zinc-dependent HDACs by conjugating non-selective HDAC inhibitors with E3 ubiquitin ligase ligands.  Through cell-based assays, the authors discovered novel bifunctional mols. (dHDAC6) that could selectively degrade HDAC6.  Further mechanistic studies indicated that HDAC6 was selectively removed by the UPS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9t1stJ7Qkf7Vg90H21EOLACvtfcHk0lgHwXAImbLUFg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtVOlsrfF&md5=20be52b05a327184f824ccf9cf88333f</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2018.05.057&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2018.05.057%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DK.%26aulast%3DSong%26aufirst%3DY.%26aulast%3DXie%26aufirst%3DH.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DWu%26aufirst%3DY.%2BT.%26aulast%3DLeisten%26aufirst%3DE.%2BD.%26aulast%3DTang%26aufirst%3DW.%26atitle%3DDevelopment%2520of%2520the%2520first%2520small%2520molecule%2520histone%2520deacetylase%25206%2520%2528HDAC6%2529%2520degraders%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2018%26volume%3D28%26spage%3D2493%26epage%3D2497%26doi%3D10.1016%2Fj.bmcl.2018.05.057" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ohoka, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okuhira, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ito, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagai, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shibata, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hattori, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ujikawa, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shimokawa, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sano, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koyama, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujita, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teratani, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsumoto, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imaeda, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nara, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naito, M.</span></span> <span> </span><span class="NLM_article-title">In vivo knockdown of pathogenic proteins via specific and nongenetic inhibitor of apoptosis protein (IAP)-dependent protein erasers (SNIPERs)</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>292</i></span>,  <span class="NLM_fpage">4556</span>– <span class="NLM_lpage">4570</span>, <span class="refDoi"> DOI: 10.1074/jbc.M116.768853</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=10.1074%2Fjbc.M116.768853" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=28154167" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A528%3ADC%252BC2sXksVClur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=292&publication_year=2017&pages=4556-4570&author=N.+Ohokaauthor=K.+Okuhiraauthor=M.+Itoauthor=K.+Nagaiauthor=N.+Shibataauthor=T.+Hattoriauthor=O.+Ujikawaauthor=K.+Shimokawaauthor=O.+Sanoauthor=R.+Koyamaauthor=H.+Fujitaauthor=M.+Terataniauthor=H.+Matsumotoauthor=Y.+Imaedaauthor=H.+Naraauthor=N.+Choauthor=M.+Naito&title=In+vivo+knockdown+of+pathogenic+proteins+via+specific+and+nongenetic+inhibitor+of+apoptosis+protein+%28IAP%29-dependent+protein+erasers+%28SNIPERs%29&doi=10.1074%2Fjbc.M116.768853"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">In Vivo Knockdown of Pathogenic Proteins via Specific and Nongenetic Inhibitor of Apoptosis Protein (IAP)-dependent Protein Erasers (SNIPERs)</span></div><div class="casAuthors">Ohoka, Nobumichi; Okuhira, Keiichiro; Ito, Masahiro; Nagai, Katsunori; Shibata, Norihito; Hattori, Takayuki; Ujikawa, Osamu; Shimokawa, Kenichiro; Sano, Osamu; Koyama, Ryokichi; Fujita, Hisashi; Teratani, Mika; Matsumoto, Hirokazu; Imaeda, Yasuhiro; Nara, Hiroshi; Cho, Nobuo; Naito, Mikihiko</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">292</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4556-4570</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Many diseases, esp. cancers, result from aberrant or overexpression of pathogenic proteins.  Specific inhibitors against these proteins have shown remarkable therapeutic effects, but these are limited mainly to enzymes.  An alternative approach that may have utility in drug development relies on selective degrdn. of pathogenic proteins via small chimeric mols. linking an E3 ubiquitin ligase to the targeted protein for proteasomal degrdn.  To this end, the authors recently developed a protein knockdown system based on hybrid small mol. SNIPERs (Specific and Nongenetic IAP-dependent Protein Erasers) that recruit inhibitor of the apoptosis protein (IAP) ubiquitin ligases to specifically degrade targeted proteins.  Here, the authors extend the authors' previous study to show a proof of concept of the SNIPER technol. in vivo.  By incorporating a high affinity IAP ligand, the authors developed a novel SNIPER against estrogen receptor α (ERα), SNIPER(ER)-87, that has a potent protein knockdown activity.  The SNIPER(ER) reduced ERα levels in tumor xenografts and suppressed the growth of ERα-pos. breast tumors in mice.  Mechanistically, it preferentially recruits X-linked IAP (XIAP) rather than cellular IAP1, to degrade ERα via the ubiquitin-proteasome pathway.  With this IAP ligand, potent SNIPERs against other pathogenic proteins, BCR-ABL, bromodomain-contg. protein 4 (BRD4), and phosphodiesterase-4 (PDE4) could also be developed.  These results indicate that forced ubiquitylation by SNIPERs is a useful method to achieve efficient protein knockdown with potential therapeutic activities and could also be applied to study the role of ubiquitylation in many cellular processes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqEzH5uNeAFabVg90H21EOLACvtfcHk0lgHwXAImbLUFg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXksVClur8%253D&md5=b09264d53c47c54343ed92d5a6d0eb79</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M116.768853&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M116.768853%26sid%3Dliteratum%253Aachs%26aulast%3DOhoka%26aufirst%3DN.%26aulast%3DOkuhira%26aufirst%3DK.%26aulast%3DIto%26aufirst%3DM.%26aulast%3DNagai%26aufirst%3DK.%26aulast%3DShibata%26aufirst%3DN.%26aulast%3DHattori%26aufirst%3DT.%26aulast%3DUjikawa%26aufirst%3DO.%26aulast%3DShimokawa%26aufirst%3DK.%26aulast%3DSano%26aufirst%3DO.%26aulast%3DKoyama%26aufirst%3DR.%26aulast%3DFujita%26aufirst%3DH.%26aulast%3DTeratani%26aufirst%3DM.%26aulast%3DMatsumoto%26aufirst%3DH.%26aulast%3DImaeda%26aufirst%3DY.%26aulast%3DNara%26aufirst%3DH.%26aulast%3DCho%26aufirst%3DN.%26aulast%3DNaito%26aufirst%3DM.%26atitle%3DIn%2520vivo%2520knockdown%2520of%2520pathogenic%2520proteins%2520via%2520specific%2520and%2520nongenetic%2520inhibitor%2520of%2520apoptosis%2520protein%2520%2528IAP%2529-dependent%2520protein%2520erasers%2520%2528SNIPERs%2529%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2017%26volume%3D292%26spage%3D4556%26epage%3D4570%26doi%3D10.1074%2Fjbc.M116.768853" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ohoka, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morita, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagai, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shimokawa, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ujikawa, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujimori, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ito, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayase, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okuhira, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shibata, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hattori, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sameshima, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sano, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koyama, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imaeda, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nara, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naito, M.</span></span> <span> </span><span class="NLM_article-title">Derivatization of inhibitor of apoptosis protein (IAP) ligands yields improved inducers of estrogen receptor α degradation</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>293</i></span>,  <span class="NLM_fpage">6776</span>– <span class="NLM_lpage">6790</span>, <span class="refDoi"> DOI: 10.1074/jbc.RA117.001091</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=10.1074%2Fjbc.RA117.001091" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=293&publication_year=2018&pages=6776-6790&author=N.+Ohokaauthor=Y.+Moritaauthor=K.+Nagaiauthor=K.+Shimokawaauthor=O.+Ujikawaauthor=I.+Fujimoriauthor=M.+Itoauthor=Y.+Hayaseauthor=K.+Okuhiraauthor=N.+Shibataauthor=T.+Hattoriauthor=T.+Sameshimaauthor=O.+Sanoauthor=R.+Koyamaauthor=Y.+Imaedaauthor=H.+Naraauthor=N.+Choauthor=M.+Naito&title=Derivatization+of+inhibitor+of+apoptosis+protein+%28IAP%29+ligands+yields+improved+inducers+of+estrogen+receptor+%CE%B1+degradation&doi=10.1074%2Fjbc.RA117.001091"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1074%2Fjbc.RA117.001091&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.RA117.001091%26sid%3Dliteratum%253Aachs%26aulast%3DOhoka%26aufirst%3DN.%26aulast%3DMorita%26aufirst%3DY.%26aulast%3DNagai%26aufirst%3DK.%26aulast%3DShimokawa%26aufirst%3DK.%26aulast%3DUjikawa%26aufirst%3DO.%26aulast%3DFujimori%26aufirst%3DI.%26aulast%3DIto%26aufirst%3DM.%26aulast%3DHayase%26aufirst%3DY.%26aulast%3DOkuhira%26aufirst%3DK.%26aulast%3DShibata%26aufirst%3DN.%26aulast%3DHattori%26aufirst%3DT.%26aulast%3DSameshima%26aufirst%3DT.%26aulast%3DSano%26aufirst%3DO.%26aulast%3DKoyama%26aufirst%3DR.%26aulast%3DImaeda%26aufirst%3DY.%26aulast%3DNara%26aufirst%3DH.%26aulast%3DCho%26aufirst%3DN.%26aulast%3DNaito%26aufirst%3DM.%26atitle%3DDerivatization%2520of%2520inhibitor%2520of%2520apoptosis%2520protein%2520%2528IAP%2529%2520ligands%2520yields%2520improved%2520inducers%2520of%2520estrogen%2520receptor%2520%25CE%25B1%2520degradation%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2018%26volume%3D293%26spage%3D6776%26epage%3D6790%26doi%3D10.1074%2Fjbc.RA117.001091" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ottis, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toure, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cromm, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ko, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gustafson, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Assessing different E3 ligases for small molecule induced protein ubiquitination and degradation</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">2570</span>– <span class="NLM_lpage">2578</span>, <span class="refDoi"> DOI: 10.1021/acschembio.7b00485</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.7b00485" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1Kisr3O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=2570-2578&author=P.+Ottisauthor=M.+Toureauthor=P.+M.+Crommauthor=E.+Koauthor=J.+L.+Gustafsonauthor=C.+M.+Crews&title=Assessing+different+E3+ligases+for+small+molecule+induced+protein+ubiquitination+and+degradation&doi=10.1021%2Facschembio.7b00485"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Assessing Different E3 Ligases for Small Molecule Induced Protein Ubiquitination and Degradation</span></div><div class="casAuthors">Ottis, Philipp; Toure, Momar; Cromm, Philipp M.; Ko, Eunhwa; Gustafson, Jeffrey L.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2570-2578</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Proteolysis Targeting Chimera (PROTAC) technol., the recruitment of E3 ubiquitin ligases to induce the degrdn. of a protein target, is rapidly impacting chem. biol., as well as modern drug development.  Here, we explore the breadth of this approach by evaluating different E3 ubiquitin ligases engineered in their substrate binding domains to accept a recruiting ligand.  Five out of six E3 ligases were found to be amenable to recruitment for target degrdn.  Taking advantage of the tight spatio-temporal control of inducing ubiquitination on a pre-selected target in living cells, we focused on two of the engineered E3 ligases, βTRCP and parkin, to characterize their ability to induce ubiquitination in comparison with the PROTAC-recruited endogenous E3 ligases VHL and cereblon.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZhBU1NFnjNbVg90H21EOLACvtfcHk0lgDmt1YLd2x4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1Kisr3O&md5=b643d34cb33b330a7658b8b7feb2c6cc</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Facschembio.7b00485&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.7b00485%26sid%3Dliteratum%253Aachs%26aulast%3DOttis%26aufirst%3DP.%26aulast%3DToure%26aufirst%3DM.%26aulast%3DCromm%26aufirst%3DP.%2BM.%26aulast%3DKo%26aufirst%3DE.%26aulast%3DGustafson%26aufirst%3DJ.%2BL.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DAssessing%2520different%2520E3%2520ligases%2520for%2520small%2520molecule%2520induced%2520protein%2520ubiquitination%2520and%2520degradation%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2017%26volume%3D12%26spage%3D2570%26epage%3D2578%26doi%3D10.1021%2Facschembio.7b00485" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Raina, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altieri, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossi, A. M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crew, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, K. G.</span></span> <span> </span><span class="NLM_article-title">PROTAC-induced BET protein degradation as a therapy for castration-resistant prostate cancer</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>113</i></span>,  <span class="NLM_fpage">7124</span>– <span class="NLM_lpage">7129</span>, <span class="refDoi"> DOI: 10.1073/pnas.1521738113</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=10.1073%2Fpnas.1521738113" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=27274052" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A528%3ADC%252BC28Xptlyqs7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2016&pages=7124-7129&author=K.+Rainaauthor=J.+Luauthor=Y.+Qianauthor=M.+Altieriauthor=D.+Gordonauthor=A.+M.+K.+Rossiauthor=J.+Wangauthor=X.+Chenauthor=H.+Dongauthor=K.+Siuauthor=J.+D.+Winklerauthor=A.+P.+Crewauthor=C.+M.+Crewsauthor=K.+G.+Coleman&title=PROTAC-induced+BET+protein+degradation+as+a+therapy+for+castration-resistant+prostate+cancer&doi=10.1073%2Fpnas.1521738113"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">PROTAC-induced BET protein degradation as a therapy for castration-resistant prostate cancer</span></div><div class="casAuthors">Raina, Kanak; Lu, Jing; Qian, Yimin; Altieri, Martha; Gordon, Deborah; Rossi, Ann Marie K.; Wang, Jing; Chen, Xin; Dong, Hanqing; Siu, Kam; Winkler, James D.; Crew, Andrew P.; Crews, Craig M.; Coleman, Kevin G.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">7124-7129</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Prostate cancer has the second highest incidence among cancers in men worldwide and is the second leading cause of cancer deaths of men in the United States.  Although androgen deprivation can initially lead to remission, the disease often progresses to castration-resistant prostate cancer (CRPC), which is still reliant on androgen receptor (AR) signaling and is assocd. with a poor prognosis.  Some success against CRPC has been achieved by drugs that target AR signaling, but secondary resistance invariably emerges, and new therapies are urgently needed.  Recently, inhibitors of bromodomain and extra-terminal (BET) family proteins have shown growth-inhibitory activity in preclin. models of CRPC.  Here, we demonstrate that ARV-771, a small-mol. pan-BET degrader based on proteolysis-targeting chimera (PROTAC) technol., demonstrates dramatically improved efficacy in cellular models of CRPC as compared with BET inhibition.  Unlike BET inhibitors, ARV-771 results in suppression of both AR signaling and AR levels and leads to tumor regression in a CRPC mouse xenograft model.  This study is, to our knowledge, the first to demonstrate efficacy with a small-mol. BET degrader in a solid-tumor malignancy and potentially represents an important therapeutic advance in the treatment of CRPC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrD7QtYgZVig7Vg90H21EOLACvtfcHk0lgDmt1YLd2x4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xptlyqs7s%253D&md5=34218e685ad72950f3eb85586110c43d</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1521738113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1521738113%26sid%3Dliteratum%253Aachs%26aulast%3DRaina%26aufirst%3DK.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DQian%26aufirst%3DY.%26aulast%3DAltieri%26aufirst%3DM.%26aulast%3DGordon%26aufirst%3DD.%26aulast%3DRossi%26aufirst%3DA.%2BM.%2BK.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DDong%26aufirst%3DH.%26aulast%3DSiu%26aufirst%3DK.%26aulast%3DWinkler%26aufirst%3DJ.%2BD.%26aulast%3DCrew%26aufirst%3DA.%2BP.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26aulast%3DColeman%26aufirst%3DK.%2BG.%26atitle%3DPROTAC-induced%2520BET%2520protein%2520degradation%2520as%2520a%2520therapy%2520for%2520castration-resistant%2520prostate%2520cancer%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2016%26volume%3D113%26spage%3D7124%26epage%3D7129%26doi%3D10.1073%2Fpnas.1521738113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandez-Salas, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McEachern, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Przybranowski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of a small-molecule degrader of bromodomain and extra-terminal (BET) proteins with picomolar cellular potencies and capable of achieving tumor regression</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">462</span>– <span class="NLM_lpage">481</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01816</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01816" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkvVajsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=462-481&author=B.+Zhouauthor=J.+Huauthor=F.+Xuauthor=Z.+Chenauthor=L.+Baiauthor=E.+Fernandez-Salasauthor=M.+Linauthor=L.+Liuauthor=C.+Y.+Yangauthor=Y.+Zhaoauthor=D.+McEachernauthor=S.+Przybranowskiauthor=B.+Wenauthor=D.+Sunauthor=S.+Wang&title=Discovery+of+a+small-molecule+degrader+of+bromodomain+and+extra-terminal+%28BET%29+proteins+with+picomolar+cellular+potencies+and+capable+of+achieving+tumor+regression&doi=10.1021%2Facs.jmedchem.6b01816"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Small-Molecule Degrader of Bromodomain and Extra-Terminal (BET) Proteins with Picomolar Cellular Potencies and Capable of Achieving Tumor Regression</span></div><div class="casAuthors">Zhou, Bing; Hu, Jiantao; Xu, Fuming; Chen, Zhuo; Bai, Longchuan; Fernandez-Salas, Ester; Lin, Mei; Liu, Liu; Yang, Chao-Yie; Zhao, Yujun; McEachern, Donna; Przybranowski, Sally; Wen, Bo; Sun, Duxin; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">462-481</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The bromodomain and extra-terminal (BET) family proteins, consisting of BRD2, BRD3, BRD4, and testis-specific BRDT members, are epigenetic "readers" and play a key role in the regulation of gene transcription.  BET proteins are considered to be attractive therapeutic targets for cancer and other human diseases.  Recently, heterobifunctional small-mol. BET degraders have been designed based upon the proteolysis targeting chimera (PROTAC) concept to induce BET protein degrdn.  Herein, we present our design, synthesis, and evaluation of a new class of PROTAC BET degraders.  One of the most promising compds., 23, effectively degrades BRD4 protein at concns. as low as 30 pM in the RS4;11 leukemia cell line, achieves an IC50 value of 51 pM in inhibition of RS4;11 cell growth and induces rapid tumor regression in vivo against RS4;11 xenograft tumors.  These data establish that compd. 23 (BETd-260/ZBC260) is a highly potent and efficacious BET degrader.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqz2Lwp52x1rVg90H21EOLACvtfcHk0lgDmt1YLd2x4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkvVajsb8%253D&md5=6b3a10ecd192a5812ef581dca1fdc82a</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01816&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01816%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DB.%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DF.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DBai%26aufirst%3DL.%26aulast%3DFernandez-Salas%26aufirst%3DE.%26aulast%3DLin%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DYang%26aufirst%3DC.%2BY.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DMcEachern%26aufirst%3DD.%26aulast%3DPrzybranowski%26aufirst%3DS.%26aulast%3DWen%26aufirst%3DB.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520a%2520small-molecule%2520degrader%2520of%2520bromodomain%2520and%2520extra-terminal%2520%2528BET%2529%2520proteins%2520with%2520picomolar%2520cellular%2520potencies%2520and%2520capable%2520of%2520achieving%2520tumor%2520regression%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D462%26epage%3D481%26doi%3D10.1021%2Facs.jmedchem.6b01816" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brzozowski, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pike, A. C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dauter, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubbard, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonn, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engstrom, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohman, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greene, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gustafsson, J.-Å.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlquist, M.</span></span> <span> </span><span class="NLM_article-title">Molecular basis of agonism and antagonism in the oestrogen receptor</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>389</i></span>,  <span class="NLM_fpage">753</span>– <span class="NLM_lpage">758</span>, <span class="refDoi"> DOI: 10.1038/39645</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=10.1038%2F39645" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=9338790" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A528%3ADyaK2sXmvVSnt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=389&publication_year=1997&pages=753-758&author=A.+M.+Brzozowskiauthor=A.+C.+W.+Pikeauthor=Z.+Dauterauthor=R.+E.+Hubbardauthor=T.+Bonnauthor=O.+Engstromauthor=L.+Ohmanauthor=G.+L.+Greeneauthor=J.-%C3%85.+Gustafssonauthor=M.+Carlquist&title=Molecular+basis+of+agonism+and+antagonism+in+the+oestrogen+receptor&doi=10.1038%2F39645"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular basis of agonism and antagonism in the estrogen receptor</span></div><div class="casAuthors">Brzozowski, Andrzej M.; Pike, Ashley C. W.; Dauter, Zbigniew; Hubbard, Roderick E.; Bonn, Tomas; Engstrom, Owe; Ohman, Lars; Greene, Geoffrey L.; Gustafsson, Jan-Ake; Carlquist, Mats</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London)</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">389</span>
        (<span class="NLM_cas:issue">6652</span>),
    <span class="NLM_cas:pages">753-758</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Macmillan Magazines</span>)
        </div><div class="casAbstract">Estrogens are involved in the growth, development and homeostasis of a no. of tissues.  The physiol. effects of these steroids are mediated by a ligand-inducible nuclear transcription factor, the estrogen receptor (ER).  Hormone binding to the ligand-binding domain (LBD) of the ER initiates a series of mol. events culminating in the activation or repression of target genes.  Transcriptional regulation arises from the direct interaction of the ER with components of the cellular transcription machinery.  Here we report the crystal structures of the LBD of ER in complex with the endogenous estrogen, 17β-estradiol, and the selective antagonist raloxifene, at resolns. of 3.1 and 2.6Å, resp.  The structures provide a mol. basis for the distinctive pharmacophore of the ER and its catholic binding properties.  Agonist and antagonist bind at the same site within the core of the LBD but demonstrate different binding modes.  In addn., each class of ligand induces a distinct conformation in the transactivation domain of the LBD, providing structural evidence of the mechanism of antagonism.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAXbSo66V1LrVg90H21EOLACvtfcHk0ljjxQD2k9Jfbw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXmvVSnt74%253D&md5=2a63951b881b329ff142c9fbc82a13e3</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1038%2F39645&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F39645%26sid%3Dliteratum%253Aachs%26aulast%3DBrzozowski%26aufirst%3DA.%2BM.%26aulast%3DPike%26aufirst%3DA.%2BC.%2BW.%26aulast%3DDauter%26aufirst%3DZ.%26aulast%3DHubbard%26aufirst%3DR.%2BE.%26aulast%3DBonn%26aufirst%3DT.%26aulast%3DEngstrom%26aufirst%3DO.%26aulast%3DOhman%26aufirst%3DL.%26aulast%3DGreene%26aufirst%3DG.%2BL.%26aulast%3DGustafsson%26aufirst%3DJ.-%25C3%2585.%26aulast%3DCarlquist%26aufirst%3DM.%26atitle%3DMolecular%2520basis%2520of%2520agonism%2520and%2520antagonism%2520in%2520the%2520oestrogen%2520receptor%26jtitle%3DNature%26date%3D1997%26volume%3D389%26spage%3D753%26epage%3D758%26doi%3D10.1038%2F39645" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Qin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandez-Salas, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McEachern, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Przybranowski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meagher, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ziazadeh, D. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of QCA570 as an exceptionally potent and efficacious proteolysis targeting chimera (PROTAC) degrader of the bromodomain and extra-terminal (BET) proteins capable of inducing complete and durable tumor regression</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">6685</span>– <span class="NLM_lpage">6704</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00506</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00506" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtlaqsLrE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=6685-6704&author=C.+Qinauthor=Y.+Huauthor=B.+Zhouauthor=E.+Fernandez-Salasauthor=C.+Y.+Yangauthor=L.+Liuauthor=D.+McEachernauthor=S.+Przybranowskiauthor=M.+Wangauthor=J.+Stuckeyauthor=J.+Meagherauthor=L.+Baiauthor=Z.+Chenauthor=M.+Linauthor=J.+Yangauthor=D.+N.+Ziazadehauthor=F.+Xuauthor=J.+Huauthor=W.+Xiangauthor=L.+Huangauthor=S.+Liauthor=B.+Wenauthor=D.+Sunauthor=S.+Wang&title=Discovery+of+QCA570+as+an+exceptionally+potent+and+efficacious+proteolysis+targeting+chimera+%28PROTAC%29+degrader+of+the+bromodomain+and+extra-terminal+%28BET%29+proteins+capable+of+inducing+complete+and+durable+tumor+regression&doi=10.1021%2Facs.jmedchem.8b00506"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of QCA570 as an Exceptionally Potent and Efficacious Proteolysis Targeting Chimera (PROTAC) Degrader of the Bromodomain and Extra-Terminal (BET) Proteins Capable of Inducing Complete and Durable Tumor Regression</span></div><div class="casAuthors">Qin, Chong; Hu, Yang; Zhou, Bing; Fernandez-Salas, Ester; Yang, Chao-Yie; Liu, Liu; McEachern, Donna; Przybranowski, Sally; Wang, Mi; Stuckey, Jeanne; Meagher, Jennifer; Bai, Longchuan; Chen, Zhuo; Lin, Mei; Yang, Jiuling; Ziazadeh, Danya N.; Xu, Fuming; Hu, Jiantao; Xing, Weiguo; Huang, Liyue; Li, Siwei; Wen, Bo; Sun, Duxin; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">6685-6704</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Proteins of the bromodomain and extra-terminal (BET) family are epigenetics "readers" and promising therapeutic targets for cancer and other human diseases.  We describe herein a structure-guided design of [1,4]oxazepines as a new class of BET inhibitors and our subsequent design, synthesis, and evaluation of proteolysis-targeting chimeric (PROTAC) small-mol. BET degraders.  Our efforts have led to the discovery of extremely potent BET degraders, exemplified by QCA570, which effectively induces degrdn. of BET proteins and inhibits cell growth in human acute leukemia cell lines even at low picomolar concns.  QCA570 achieves complete and durable tumor regression in leukemia xenograft models in mice at well-tolerated dose-schedules.  QCA570 is the most potent and efficacious BET degrader reported to date.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6ZUIsfX3p6LVg90H21EOLACvtfcHk0ljjxQD2k9Jfbw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtlaqsLrE&md5=2a731730641372f4526543e03d9ac76f</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00506&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00506%26sid%3Dliteratum%253Aachs%26aulast%3DQin%26aufirst%3DC.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DB.%26aulast%3DFernandez-Salas%26aufirst%3DE.%26aulast%3DYang%26aufirst%3DC.%2BY.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DMcEachern%26aufirst%3DD.%26aulast%3DPrzybranowski%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DStuckey%26aufirst%3DJ.%26aulast%3DMeagher%26aufirst%3DJ.%26aulast%3DBai%26aufirst%3DL.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DLin%26aufirst%3DM.%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DZiazadeh%26aufirst%3DD.%2BN.%26aulast%3DXu%26aufirst%3DF.%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DXiang%26aufirst%3DW.%26aulast%3DHuang%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DWen%26aufirst%3DB.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520QCA570%2520as%2520an%2520exceptionally%2520potent%2520and%2520efficacious%2520proteolysis%2520targeting%2520chimera%2520%2528PROTAC%2529%2520degrader%2520of%2520the%2520bromodomain%2520and%2520extra-terminal%2520%2528BET%2529%2520proteins%2520capable%2520of%2520inducing%2520complete%2520and%2520durable%2520tumor%2520regression%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D6685%26epage%3D6704%26doi%3D10.1021%2Facs.jmedchem.8b00506" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McEachern, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Przybranowski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandez-Salas, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meagher, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayes, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellis, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Targeted degradation of BET proteins in triple-negative breast cancer</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>77</i></span>,  <span class="NLM_fpage">2476</span>– <span class="NLM_lpage">2487</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-16-2622</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=10.1158%2F0008-5472.CAN-16-2622" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=28209615" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmvFakurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=77&publication_year=2017&pages=2476-2487&author=L.+Baiauthor=B.+Zhouauthor=C.+Y.+Yangauthor=J.+Jiauthor=D.+McEachernauthor=S.+Przybranowskiauthor=H.+Jiangauthor=J.+Huauthor=F.+Xuauthor=Y.+Zhaoauthor=L.+Liuauthor=E.+Fernandez-Salasauthor=J.+Xuauthor=Y.+Douauthor=B.+Wenauthor=D.+Sunauthor=J.+Meagherauthor=J.+Stuckeyauthor=D.+F.+Hayesauthor=S.+Liauthor=M.+J.+Ellisauthor=S.+Wang&title=Targeted+degradation+of+BET+proteins+in+triple-negative+breast+cancer&doi=10.1158%2F0008-5472.CAN-16-2622"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted Degradation of BET Proteins in Triple-Negative Breast Cancer</span></div><div class="casAuthors">Bai, Longchuan; Zhou, Bing; Yang, Chao-Yie; Ji, Jiao; McEachern, Donna; Przybranowski, Sally; Jiang, Hui; Hu, Jiantao; Xu, Fuming; Zhao, Yujun; Liu, Liu; Fernandez-Salas, Ester; Xu, Jing; Dou, Yali; Wen, Bo; Sun, Duxin; Meagher, Jennifer; Stuckey, Jeanne; Hayes, Daniel F.; Li, Shunqiang; Ellis, Matthew J.; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">77</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2476-2487</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Triple-neg. breast cancers (TNBC) remain clin. challenging with a lack of options for targeted therapy.  In this study, we report the development of a second-generation BET protein degrader, BETd-246, which exhibits superior selectivity, potency, and antitumor activity.  In human TNBC cells, BETd-246 induced degrdn. of BET proteins at low nanomolar concns. within 1 h of exposure, resulting in robust growth inhibition and apoptosis.  BETd-246 was more potent and effective in TNBC cells than its parental BET inhibitor compd. BETi-211.  RNA-seq anal. revealed predominant downregulation of a large no. of genes involved in proliferation and apoptosis in cells treated with BETd-246, as compared with BETi-211 treatment that upregulated and downregulated a similar no. of genes.  Functional investigations identified the MCL1 gene as a crit. downstream effector for BET degraders, which synergized with small-mol. inhibitors of BCL-xL in triggering apoptosis.  In multiple murine xenograft models of human breast cancer, BETd-246 and a further optimized analog BETd-260 effectively depleted BET proteins in tumors and exhibited strong antitumor activities at well-tolerated dosing schedules.  Overall, our findings show that targeting BET proteins for degrdn. represents an effective therapeutic strategy for TNBC treatment.  Cancer Res; 77(9); 2476-87. ©2017 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8M1VaSNIBCLVg90H21EOLACvtfcHk0ljjxQD2k9Jfbw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmvFakurg%253D&md5=84c0e0b23e61a5503f8d88385cc88ae5</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-16-2622&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-16-2622%26sid%3Dliteratum%253Aachs%26aulast%3DBai%26aufirst%3DL.%26aulast%3DZhou%26aufirst%3DB.%26aulast%3DYang%26aufirst%3DC.%2BY.%26aulast%3DJi%26aufirst%3DJ.%26aulast%3DMcEachern%26aufirst%3DD.%26aulast%3DPrzybranowski%26aufirst%3DS.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DF.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DFernandez-Salas%26aufirst%3DE.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DDou%26aufirst%3DY.%26aulast%3DWen%26aufirst%3DB.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DMeagher%26aufirst%3DJ.%26aulast%3DStuckey%26aufirst%3DJ.%26aulast%3DHayes%26aufirst%3DD.%2BF.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DEllis%26aufirst%3DM.%2BJ.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DTargeted%2520degradation%2520of%2520BET%2520proteins%2520in%2520triple-negative%2520breast%2520cancer%26jtitle%3DCancer%2520Res.%26date%3D2017%26volume%3D77%26spage%3D2476%26epage%3D2487%26doi%3D10.1158%2F0008-5472.CAN-16-2622" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Molle, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gareiss, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tae, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noblin, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jorgensen, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Targeting the von Hippel-Lindau E3 ubiquitin ligase using small molecules to disrupt the VHL/HIF-1α interaction</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>134</i></span>,  <span class="NLM_fpage">4465</span>– <span class="NLM_lpage">4468</span>, <span class="refDoi"> DOI: 10.1021/ja209924v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja209924v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A528%3ADC%252BC38XivVOgurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=134&publication_year=2012&pages=4465-4468&author=D.+L.+Buckleyauthor=I.+Van+Molleauthor=P.+C.+Gareissauthor=H.+S.+Taeauthor=J.+Michelauthor=D.+J.+Noblinauthor=W.+L.+Jorgensenauthor=A.+Ciulliauthor=C.+M.+Crews&title=Targeting+the+von+Hippel-Lindau+E3+ubiquitin+ligase+using+small+molecules+to+disrupt+the+VHL%2FHIF-1%CE%B1+interaction&doi=10.1021%2Fja209924v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting the von Hippel-Lindau E3 Ubiquitin Ligase Using Small Molecules To Disrupt the VHL/HIF-1α Interaction</span></div><div class="casAuthors">Buckley, Dennis L.; Van Molle, Inge; Gareiss, Peter C.; Tae, Hyun Seop; Michel, Julien; Noblin, Devin J.; Jorgensen, William L.; Ciulli, Alessio; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">134</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4465-4468</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">E3 ubiquitin ligases, which bind protein targets, leading to their ubiquitination and subsequent degrdn., are attractive drug targets due to their exquisite substrate specificity.  However, the development of small-mol. inhibitors has proven extraordinarily challenging as modulation of E3 ligase activities requires the targeting of protein-protein interactions.  Using rational design, we have generated the first small mol. targeting the von Hippel-Lindau protein (VHL), the substrate recognition subunit of an E3 ligase, and an important target in cancer, chronic anemia, and ischemia.  We have also obtained the crystal structure of VHL bound to our most potent inhibitor, confirming that the compd. mimics the binding mode of the transcription factor HIF-1α, a substrate of VHL.  These results have the potential to guide future development of improved lead compds. as therapeutics for the treatment of chronic anemia and ischemia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGolmBVM_mfWzrVg90H21EOLACvtfcHk0lhFuDn9qxAtAw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XivVOgurw%253D&md5=60f7384b6bc932c9828dc12999283a1e</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1021%2Fja209924v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja209924v%26sid%3Dliteratum%253Aachs%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DVan%2BMolle%26aufirst%3DI.%26aulast%3DGareiss%26aufirst%3DP.%2BC.%26aulast%3DTae%26aufirst%3DH.%2BS.%26aulast%3DMichel%26aufirst%3DJ.%26aulast%3DNoblin%26aufirst%3DD.%2BJ.%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26aulast%3DCiulli%26aufirst%3DA.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DTargeting%2520the%2520von%2520Hippel-Lindau%2520E3%2520ubiquitin%2520ligase%2520using%2520small%2520molecules%2520to%2520disrupt%2520the%2520VHL%252FHIF-1%25CE%25B1%2520interaction%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2012%26volume%3D134%26spage%3D4465%26epage%3D4468%26doi%3D10.1021%2Fja209924v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gustafson, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Molle, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tae, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gareiss, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jorgensen, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Small-molecule inhibitors of the interaction between the E3 ligase VHL and HIF-1α</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">11463</span>– <span class="NLM_lpage">11467</span>, <span class="refDoi"> DOI: 10.1002/anie.201206231</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=10.1002%2Fanie.201206231" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2012&pages=11463-11467&author=D.+L.+Buckleyauthor=J.+L.+Gustafsonauthor=I.+Van+Molleauthor=A.+G.+Rothauthor=H.+S.+Taeauthor=P.+C.+Gareissauthor=W.+L.+Jorgensenauthor=A.+Ciulliauthor=C.+M.+Crews&title=Small-molecule+inhibitors+of+the+interaction+between+the+E3+ligase+VHL+and+HIF-1%CE%B1&doi=10.1002%2Fanie.201206231"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1002%2Fanie.201206231&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201206231%26sid%3Dliteratum%253Aachs%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DGustafson%26aufirst%3DJ.%2BL.%26aulast%3DVan%2BMolle%26aufirst%3DI.%26aulast%3DRoth%26aufirst%3DA.%2BG.%26aulast%3DTae%26aufirst%3DH.%2BS.%26aulast%3DGareiss%26aufirst%3DP.%2BC.%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26aulast%3DCiulli%26aufirst%3DA.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DSmall-molecule%2520inhibitors%2520of%2520the%2520interaction%2520between%2520the%2520E3%2520ligase%2520VHL%2520and%2520HIF-1%25CE%25B1%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2012%26volume%3D51%26spage%3D11463%26epage%3D11467%26doi%3D10.1002%2Fanie.201206231" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Galdeano, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gadd, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soares, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scaffidi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Molle, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Birced, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hewitt, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dias, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Structure-guided design and optimization of small molecules targeting the protein-protein interaction between the von Hippel-Lindau (VHL) E3 ubiquitin ligase and the hypoxia inducible factor (HIF) α subunit with in vitro nanomolar affinities</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">8657</span>– <span class="NLM_lpage">8663</span>, <span class="refDoi"> DOI: 10.1021/jm5011258</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm5011258" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=8657-8663&author=C.+Galdeanoauthor=M.+S.+Gaddauthor=P.+Soaresauthor=S.+Scaffidiauthor=I.+Van+Molleauthor=I.+Bircedauthor=S.+Hewittauthor=D.+M.+Diasauthor=A.+Ciulli&title=Structure-guided+design+and+optimization+of+small+molecules+targeting+the+protein-protein+interaction+between+the+von+Hippel-Lindau+%28VHL%29+E3+ubiquitin+ligase+and+the+hypoxia+inducible+factor+%28HIF%29+%CE%B1+subunit+with+in+vitro+nanomolar+affinities&doi=10.1021%2Fjm5011258"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1021%2Fjm5011258&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm5011258%26sid%3Dliteratum%253Aachs%26aulast%3DGaldeano%26aufirst%3DC.%26aulast%3DGadd%26aufirst%3DM.%2BS.%26aulast%3DSoares%26aufirst%3DP.%26aulast%3DScaffidi%26aufirst%3DS.%26aulast%3DVan%2BMolle%26aufirst%3DI.%26aulast%3DBirced%26aufirst%3DI.%26aulast%3DHewitt%26aufirst%3DS.%26aulast%3DDias%26aufirst%3DD.%2BM.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DStructure-guided%2520design%2520and%2520optimization%2520of%2520small%2520molecules%2520targeting%2520the%2520protein-protein%2520interaction%2520between%2520the%2520von%2520Hippel-Lindau%2520%2528VHL%2529%2520E3%2520ubiquitin%2520ligase%2520and%2520the%2520hypoxia%2520inducible%2520factor%2520%2528HIF%2529%2520%25CE%25B1%2520subunit%2520with%2520in%2520vitro%2520nanomolar%2520affinities%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D8657%26epage%3D8663%26doi%3D10.1021%2Fjm5011258" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Soares, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gadd, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frost, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galdeano, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellis, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Epemolu, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rocha, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Group-based optimization of potent and cell-active inhibitors of the von Hippel-Lindau (VHL) E3 ubiquitin ligase: structure-activity relationships leading to the chemical probe (2S,4R)-1-((S)-2-(1-cyanocyclopropanecarboxamido)-3,3-dimethylbutanoyl)-4-hydroxy -N-(4-(4-methylthiazol-5-yl)benzyl)pyrrolidine-2-carboxamide (VH298)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">599</span>– <span class="NLM_lpage">618</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00675</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00675" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=599-618&author=P.+Soaresauthor=M.+S.+Gaddauthor=J.+Frostauthor=C.+Galdeanoauthor=L.+Ellisauthor=O.+Epemoluauthor=S.+Rochaauthor=K.+D.+Readauthor=A.+Ciulli&title=Group-based+optimization+of+potent+and+cell-active+inhibitors+of+the+von+Hippel-Lindau+%28VHL%29+E3+ubiquitin+ligase%3A+structure-activity+relationships+leading+to+the+chemical+probe+%282S%2C4R%29-1-%28%28S%29-2-%281-cyanocyclopropanecarboxamido%29-3%2C3-dimethylbutanoyl%29-4-hydroxy+-N-%284-%284-methylthiazol-5-yl%29benzyl%29pyrrolidine-2-carboxamide+%28VH298%29&doi=10.1021%2Facs.jmedchem.7b00675"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00675&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00675%26sid%3Dliteratum%253Aachs%26aulast%3DSoares%26aufirst%3DP.%26aulast%3DGadd%26aufirst%3DM.%2BS.%26aulast%3DFrost%26aufirst%3DJ.%26aulast%3DGaldeano%26aufirst%3DC.%26aulast%3DEllis%26aufirst%3DL.%26aulast%3DEpemolu%26aufirst%3DO.%26aulast%3DRocha%26aufirst%3DS.%26aulast%3DRead%26aufirst%3DK.%2BD.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DGroup-based%2520optimization%2520of%2520potent%2520and%2520cell-active%2520inhibitors%2520of%2520the%2520von%2520Hippel-Lindau%2520%2528VHL%2529%2520E3%2520ubiquitin%2520ligase%253A%2520structure-activity%2520relationships%2520leading%2520to%2520the%2520chemical%2520probe%2520%25282S%252C4R%2529-1-%2528%2528S%2529-2-%25281-cyanocyclopropanecarboxamido%2529-3%252C3-dimethylbutanoyl%2529-4-hydroxy%2520-N-%25284-%25284-methylthiazol-5-yl%2529benzyl%2529pyrrolidine-2-carboxamide%2520%2528VH298%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D599%26epage%3D618%26doi%3D10.1021%2Facs.jmedchem.7b00675" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gadd, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Testa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucas, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamont, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zengerle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Structural basis of PROTAC cooperative recognition for selective protein degradation</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">514</span>– <span class="NLM_lpage">521</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2329</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=10.1038%2Fnchembio.2329" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=28288108" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkvFahtbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&pages=514-521&author=M.+S.+Gaddauthor=A.+Testaauthor=X.+Lucasauthor=K.+H.+Chanauthor=W.+Chenauthor=D.+J.+Lamontauthor=M.+Zengerleauthor=A.+Ciulli&title=Structural+basis+of+PROTAC+cooperative+recognition+for+selective+protein+degradation&doi=10.1038%2Fnchembio.2329"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis of PROTAC cooperative recognition for selective protein degradation</span></div><div class="casAuthors">Gadd, Morgan S.; Testa, Andrea; Lucas, Xavier; Chan, Kwok-Ho; Chen, Wenzhang; Lamont, Douglas J.; Zengerle, Michael; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">514-521</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Inducing macromol. interactions with small mols. to activate cellular signaling is a challenging goal.  PROTACs (proteolysis-targeting chimeras) are bifunctional mols. that recruit a target protein in proximity to an E3 ubiquitin ligase to trigger protein degrdn.  Structural elucidation of the key ternary ligase-PROTAC-target species and its impact on target degrdn. selectivity remain elusive.  Here, we solved the crystal structure of Brd4 degrader MZ1 in complex with human VHL and the Brd4 bromodomain (Brd4BD2).  The ligand folds into itself to allow formation of specific intermol. interactions in the ternary complex.  Isothermal titrn. calorimetry studies, supported by surface mutagenesis and proximity assays, were consistent with pronounced cooperative formation of ternary complexes with Brd4BD2.  Structure-based-designed compd. AT1 exhibited highly selective depletion of Brd4 in cells.  These results elucidate how PROTAC-induced de novo contacts dictate preferential recruitment of a target protein into a stable and cooperative complex with an E3 ligase for selective degrdn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJri00jB5YJLVg90H21EOLACvtfcHk0lhVX-7cpnrOFg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkvFahtbg%253D&md5=dfee13913106d362d0972d7b1a62d700</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2329&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2329%26sid%3Dliteratum%253Aachs%26aulast%3DGadd%26aufirst%3DM.%2BS.%26aulast%3DTesta%26aufirst%3DA.%26aulast%3DLucas%26aufirst%3DX.%26aulast%3DChan%26aufirst%3DK.%2BH.%26aulast%3DChen%26aufirst%3DW.%26aulast%3DLamont%26aufirst%3DD.%2BJ.%26aulast%3DZengerle%26aufirst%3DM.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DStructural%2520basis%2520of%2520PROTAC%2520cooperative%2520recognition%2520for%2520selective%2520protein%2520degradation%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2017%26volume%3D13%26spage%3D514%26epage%3D521%26doi%3D10.1038%2Fnchembio.2329" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Long, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poganik, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aye, Y.</span></span> <span> </span><span class="NLM_article-title">On-demand targeting: investigating biology with proximity-directed chemistry</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>138</i></span>,  <span class="NLM_fpage">3610</span>– <span class="NLM_lpage">3622</span>, <span class="refDoi"> DOI: 10.1021/jacs.5b12608</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.5b12608" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A528%3ADC%252BC28XjtVCis7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=138&publication_year=2016&pages=3610-3622&author=M.+J.+Longauthor=J.+R.+Poganikauthor=Y.+Aye&title=On-demand+targeting%3A+investigating+biology+with+proximity-directed+chemistry&doi=10.1021%2Fjacs.5b12608"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">On-Demand Targeting: Investigating Biology with Proximity-Directed Chemistry</span></div><div class="casAuthors">Long, Marcus J. C.; Poganik, Jesse R.; Aye, Yimon</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">138</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">3610-3622</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Proximity enhancement is a central chem. tenet underpinning an exciting suite of small-mol. toolsets that have allowed us to unravel many biol. complexities.  The leitmotif of this opus is "tethering"-a strategy in which a multifunctional small mol. serves as a template to bring proteins/biomols. together.  Scaffolding approaches have been powerfully applied to control diverse biol. outcomes such as protein-protein assocn., protein stability, activity, and improve imaging capabilities.  A new twist on this strategy has recently appeared, in which the small-mol. probe is engineered to unleash controlled amts. of reactive chem. signals within the microenvironment of a target protein.  Modification of a specific target elicits a precisely timed and spatially controlled gain-of-function (or dominant loss-of-function) signaling response.  Presented herein is a unique personal outlook conceptualizing the powerful proximity-enhanced chem. biol. toolsets into two paradigms: "multifunctional scaffolding" vs. "on-demand targeting".  By addressing the latest advances and challenges in the established yet constantly evolving multifunctional scaffolding strategies as well as in the emerging on-demand precision targeting (and related) systems, this Perspective is aimed at choosing when it is best to employ each of the two strategies, with an emphasis toward further promoting novel applications and discoveries stemming from these innovative chem. biol. platforms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGro6dKG5laUM7Vg90H21EOLACvtfcHk0lhVX-7cpnrOFg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XjtVCis7g%253D&md5=40fa3d02e8de215dd7c596aa7a32af78</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1021%2Fjacs.5b12608&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.5b12608%26sid%3Dliteratum%253Aachs%26aulast%3DLong%26aufirst%3DM.%2BJ.%26aulast%3DPoganik%26aufirst%3DJ.%2BR.%26aulast%3DAye%26aufirst%3DY.%26atitle%3DOn-demand%2520targeting%253A%2520investigating%2520biology%2520with%2520proximity-directed%2520chemistry%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2016%26volume%3D138%26spage%3D3610%26epage%3D3622%26doi%3D10.1021%2Fjacs.5b12608" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stols, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dieckman, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raffen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collart, F. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donnelly, M. I.</span></span> <span> </span><span class="NLM_article-title">A new vector for high-throughput, ligation-independent cloning encoding a tobacco etch virus protease cleavage site</span>. <i>Protein Expression Purif.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">8</span>– <span class="NLM_lpage">15</span>, <span class="refDoi"> DOI: 10.1006/prep.2001.1603</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=10.1006%2Fprep.2001.1603" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=12071693" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A528%3ADC%252BD38XksF2gsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2002&pages=8-15&author=L.+Stolsauthor=M.+Guauthor=L.+Dieckmanauthor=R.+Raffenauthor=F.+R.+Collartauthor=M.+I.+Donnelly&title=A+new+vector+for+high-throughput%2C+ligation-independent+cloning+encoding+a+tobacco+etch+virus+protease+cleavage+site&doi=10.1006%2Fprep.2001.1603"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">A New Vector for High-Throughput, Ligation-Independent Cloning Encoding a Tobacco Etch Virus Protease Cleavage Site</span></div><div class="casAuthors">Stols, Lucy; Gu, Minyi; Dieckman, Lynda; Raffen, Rosemarie; Collart, Frank R.; Donnelly, Mark I.</div><div class="citationInfo"><span class="NLM_cas:title">Protein Expression and Purification</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">8-15</span>CODEN:
                <span class="NLM_cas:coden">PEXPEJ</span>;
        ISSN:<span class="NLM_cas:issn">1046-5928</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science</span>)
        </div><div class="casAbstract">To establish high-throughput methods for protein crystallog., all aspects of the prodn. and anal. of protein crystals must be accelerated.  Automated, plate-based methods for cloning, expression, and evaluation of target proteins will help researchers investigate the vast nos. of proteins now available from sequenced genomes.  Ligation-independent cloning (LIC) is well suited to robotic cloning and expression, but few LIC vectors are available com.  The authors have developed a new LIC vector, pMCSG7, that incorporates the tobacco etch virus (TEV) protease cleavage site into the leader sequence.  This protease is highly specific and functions under a wide range of conditions.  The new vector incorporates an N-terminal his-tag followed by the TEV protease recognition site and a SspI restriction site used for LIC.  The vector functioned as expected, giving high cloning efficiencies and strong expression of proteins.  Purifn. and cleavage of a target protein showed that the his-tag and the TEV cleavage site function properly.  The protein was purified and cleaved under different conditions to simulate both plate-based screening methods and large-scale purifications for crystal prodn.  The vector also includes a pair of adjacent, unique restriction sites that will allow insertion of addnl. modules between the his-tag and the cleavage site of the leader sequence to generate a family of vectors suitable for high-throughput prodn. of proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqcWlZD9MM0mbVg90H21EOLACvtfcHk0lhVX-7cpnrOFg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XksF2gsb4%253D&md5=a2baff9a27cd5caf32358891635de4f5</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1006%2Fprep.2001.1603&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fprep.2001.1603%26sid%3Dliteratum%253Aachs%26aulast%3DStols%26aufirst%3DL.%26aulast%3DGu%26aufirst%3DM.%26aulast%3DDieckman%26aufirst%3DL.%26aulast%3DRaffen%26aufirst%3DR.%26aulast%3DCollart%26aufirst%3DF.%2BR.%26aulast%3DDonnelly%26aufirst%3DM.%2BI.%26atitle%3DA%2520new%2520vector%2520for%2520high-throughput%252C%2520ligation-independent%2520cloning%2520encoding%2520a%2520tobacco%2520etch%2520virus%2520protease%2520cleavage%2520site%26jtitle%3DProtein%2520Expression%2520Purif.%26date%3D2002%26volume%3D25%26spage%3D8%26epage%3D15%26doi%3D10.1006%2Fprep.2001.1603" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Benoit, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ostermeier, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geiser, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Widmer, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Auer, M.</span></span> <span> </span><span class="NLM_article-title">Seamless insert-plasmid assembly at high efficiency and low cost</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>, <span class="NLM_elocation-id">e0153158</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0153158</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=10.1371%2Fjournal.pone.0153158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=27073895" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFyjs7%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&author=R.+M.+Benoitauthor=C.+Ostermeierauthor=M.+Geiserauthor=J.+S.+Liauthor=H.+Widmerauthor=M.+Auer&title=Seamless+insert-plasmid+assembly+at+high+efficiency+and+low+cost&doi=10.1371%2Fjournal.pone.0153158"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Seamless insert-plasmid assembly at high efficiency and low cost</span></div><div class="casAuthors">Benoit, Roger M.; Ostermeier, Christian; Geiser, Martin; Li, Julia Su Zhou; Widmer, Hans; Auer, Manfred</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">e0153158/1-e0153158/13</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Seamless cloning methods, such as co-transformation cloning, sequence- and ligation-independent cloning (SLIC) or the Gibson assembly, are essential tools for the precise construction of plasmids.  The efficiency of co-transformation cloning is however low and the Gibson assembly reagents are expensive.  With the aim to improve the robustness of seamless cloning expts. while keeping costs low, we examd. the importance of complementary single-stranded DNA ends for co-transformation cloning and the influence of single-stranded gaps in circular plasmids on SLIC cloning efficiency.  Most importantly, our data show that single-stranded gaps in double-stranded plasmids, which occur in typical SLIC protocols, can drastically decrease the efficiency at which the DNA transforms competent E. coli bacteria.  Accordingly, filling-in of single-stranded gaps using DNA polymerase resulted in increased transformation efficiency.  Ligation of the remaining nicks did not lead to a further increase in transformation efficiency.  These findings demonstrate that highly efficient insert-plasmid assembly can be achieved by using only T5 exonuclease and Phusion DNA polymerase, without Taq DNA ligase from the original Gibson protocol, which significantly reduces the cost of the reactions.  We successfully used this modified Gibson assembly protocol with two short insert-plasmid overlap regions, each counting only 15 nucleotides.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWj6JIvwfOybVg90H21EOLACvtfcHk0lgazX0xFozWdw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFyjs7%252FP&md5=f5978304dfcf58f93be15833fc30a5d6</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0153158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0153158%26sid%3Dliteratum%253Aachs%26aulast%3DBenoit%26aufirst%3DR.%2BM.%26aulast%3DOstermeier%26aufirst%3DC.%26aulast%3DGeiser%26aufirst%3DM.%26aulast%3DLi%26aufirst%3DJ.%2BS.%26aulast%3DWidmer%26aufirst%3DH.%26aulast%3DAuer%26aufirst%3DM.%26atitle%3DSeamless%2520insert-plasmid%2520assembly%2520at%2520high%2520efficiency%2520and%2520low%2520cost%26jtitle%3DPLoS%2520One%26date%3D2016%26volume%3D11%26doi%3D10.1371%2Fjournal.pone.0153158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1ERR" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1ERR','PDB','1ERR'); return false;">PDB: 1ERR</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i64"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b01572">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_25341"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.8b01572">10.1021/acs.jmedchem.8b01572</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Structure characterization for compound <b>11</b>, VH032, <b>43a</b>–<b>48a</b>; <sup>1</sup>H NMR, <sup>13</sup>C NMR, and UPLC-MS analysis of compounds <b>15</b> (ERD-148) and <b>32</b> (ERD-308 (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b01572/suppl_file/jm8b01572_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular string files for all the final target compounds (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b01572/suppl_file/jm8b01572_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b01572/suppl_file/jm8b01572_si_001.pdf">jm8b01572_si_001.pdf (1.57 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b01572/suppl_file/jm8b01572_si_002.csv">jm8b01572_si_002.csv (7.47 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.8b01572&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2019.62.issue-3%3Bwebsite%3Awebsite%3Aacspubs%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.8b01572%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.8b01572" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                7MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679950a4f9a424ad","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
