
<html lang="en"     class="pb-page"  data-request-id="89673890-8963-4424-87ce-a07a9f1bc014"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;issue:issue:10.1021/jmcmar.2018.61.issue-23;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.8b01219;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Structure-Based Engineering of Irreversible Inhibitors against Histone Lysine Demethylase KDM5A" /></meta><meta name="dc.Creator" content="John R.  Horton" /></meta><meta name="dc.Creator" content="Clayton B.  Woodcock" /></meta><meta name="dc.Creator" content="Qin  Chen" /></meta><meta name="dc.Creator" content="Xu  Liu" /></meta><meta name="dc.Creator" content="Xing  Zhang" /></meta><meta name="dc.Creator" content="John  Shanks" /></meta><meta name="dc.Creator" content="Ganesha  Rai" /></meta><meta name="dc.Creator" content="Bryan T.  Mott" /></meta><meta name="dc.Creator" content="Daniel J.  Jansen" /></meta><meta name="dc.Creator" content="Stephen C.  Kales" /></meta><meta name="dc.Creator" content="Mark J.  Henderson" /></meta><meta name="dc.Creator" content="Matthew  Cyr" /></meta><meta name="dc.Creator" content="Katherine  Pohida" /></meta><meta name="dc.Creator" content="Xin  Hu" /></meta><meta name="dc.Creator" content="Pranav  Shah" /></meta><meta name="dc.Creator" content="Xin  Xu" /></meta><meta name="dc.Creator" content="Ajit  Jadhav" /></meta><meta name="dc.Creator" content="David J.  Maloney" /></meta><meta name="dc.Creator" content="Matthew D.  Hall" /></meta><meta name="dc.Creator" content="Anton  Simeonov" /></meta><meta name="dc.Creator" content="Haian  Fu" /></meta><meta name="dc.Creator" content="Paula M.  Vertino" /></meta><meta name="dc.Creator" content="Xiaodong  Cheng" /></meta><meta name="dc.Description" content="The active sites of hundreds of human α-ketoglutarate (αKG) and Fe(II)-dependent dioxygenases are exceedingly well preserved, which challenges the design of selective inhibitors. We identified a no..." /></meta><meta name="Description" content="The active sites of hundreds of human α-ketoglutarate (αKG) and Fe(II)-dependent dioxygenases are exceedingly well preserved, which challenges the design of selective inhibitors. We identified a no..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="November 3, 2018" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.8b01219" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2018 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.8b01219" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.8b01219" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.8b01219" /></link>
        
    
    

<title>Structure-Based Engineering of Irreversible Inhibitors against Histone Lysine Demethylase KDM5A | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.8b01219" /></meta><meta property="og:title" content="Structure-Based Engineering of Irreversible Inhibitors against Histone Lysine Demethylase KDM5A" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01219/20181207/images/large/jm-2018-01219j_0009.jpeg" /></meta><meta property="og:description" content="The active sites of hundreds of human α-ketoglutarate (αKG) and Fe(II)-dependent dioxygenases are exceedingly well preserved, which challenges the design of selective inhibitors. We identified a noncatalytic cysteine (Cys481 in KDM5A) near the active sites of KDM5 histone H3 lysine 4 demethylases, which is absent in other histone demethylase families, that could be explored for interaction with the cysteine-reactive electrophile acrylamide. We synthesized analogs of a thienopyridine-based inhibitor chemotype, namely, 2-((3-aminophenyl)(2-(piperidin-1-yl)ethoxy)methyl)thieno[3,2-b]pyridine-7-carboxylic acid (N70) and a derivative containing a (dimethylamino)but-2-enamido)phenyl moiety (N71) designed to form a covalent interaction with Cys481. We characterized the inhibitory and binding activities against KDM5A and determined the cocrystal structures of the catalytic domain of KDM5A in complex with N70 and N71. Whereas the noncovalent inhibitor N70 displayed αKG-competitive inhibition that could be reversed after dialysis, inhibition by N71 was dependent on enzyme concentration and persisted even after dialysis, consistent with covalent modification." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.8b01219"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.8b01219">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.8b01219&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.8b01219&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.8b01219&amp;href=/doi/10.1021/acs.jmedchem.8b01219" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2018</span><span class="cit-fg-volume">, 61</span><span class="cit-fg-issue">, 23</span><span class="cit-fg-pageRange">, 10588-10601</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/61/23" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.8b01165" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.8b01222" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Structure-Based Engineering of Irreversible Inhibitors against Histone Lysine Demethylase KDM5A</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">John R. Horton</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">John R. Horton</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Molecular and Cellular Oncology, The University of Texas MD Anderson Cancer Center, Houston, Texas 77030, United States</div><div class="loa-info-affiliations-info">Department of Biochemistry, Emory University School of Medicine, Atlanta, Georgia 30322, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=John+R.++Horton">John R. Horton</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Clayton B. Woodcock</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Clayton B. Woodcock</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Molecular and Cellular Oncology, The University of Texas MD Anderson Cancer Center, Houston, Texas 77030, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Clayton+B.++Woodcock">Clayton B. Woodcock</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Qin Chen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Qin Chen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Molecular and Cellular Oncology, The University of Texas MD Anderson Cancer Center, Houston, Texas 77030, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Qin++Chen">Qin Chen</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Xu Liu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xu Liu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biochemistry, Emory University School of Medicine, Atlanta, Georgia 30322, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xu++Liu">Xu Liu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Xing Zhang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xing Zhang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Molecular and Cellular Oncology, The University of Texas MD Anderson Cancer Center, Houston, Texas 77030, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xing++Zhang">Xing Zhang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">John Shanks</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">John Shanks</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biochemistry, Emory University School of Medicine, Atlanta, Georgia 30322, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=John++Shanks">John Shanks</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ganesha Rai</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ganesha Rai</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">National Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ganesha++Rai">Ganesha Rai</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-9763-9641" title="Orcid link">http://orcid.org/0000-0001-9763-9641</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Bryan T. Mott</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Bryan T. Mott</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">National Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Bryan+T.++Mott">Bryan T. Mott</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Daniel J. Jansen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Daniel J. Jansen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">National Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Daniel+J.++Jansen">Daniel J. Jansen</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Stephen C. Kales</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Stephen C. Kales</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">National Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Stephen+C.++Kales">Stephen C. Kales</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Mark J. Henderson</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Mark J. Henderson</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">National Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Mark+J.++Henderson">Mark J. Henderson</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Matthew Cyr</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Matthew Cyr</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">National Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Matthew++Cyr">Matthew Cyr</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Katherine Pohida</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Katherine Pohida</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">National Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Katherine++Pohida">Katherine Pohida</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Xin Hu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xin Hu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">National Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xin++Hu">Xin Hu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Pranav Shah</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Pranav Shah</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">National Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Pranav++Shah">Pranav Shah</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Xin Xu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xin Xu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">National Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xin++Xu">Xin Xu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ajit Jadhav</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ajit Jadhav</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">National Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ajit++Jadhav">Ajit Jadhav</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">David J. Maloney</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">David J. Maloney</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">National Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=David+J.++Maloney">David J. Maloney</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Matthew D. Hall</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Matthew D. Hall</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">National Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Matthew+D.++Hall">Matthew D. Hall</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-5073-442X" title="Orcid link">http://orcid.org/0000-0002-5073-442X</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Anton Simeonov</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Anton Simeonov</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">National Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Anton++Simeonov">Anton Simeonov</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Haian Fu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Haian Fu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Departments of Pharmacology,  Hematology and Medical Oncology,  Emory Chemical Biology Discovery Center,  The Winship Cancer Institute, Emory University, Atlanta, Georgia 30322, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Haian++Fu">Haian Fu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Paula M. Vertino</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Paula M. Vertino</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">The Winship Cancer Institute,  Department of Radiation Oncology, Emory University, Atlanta, Georgia 30322, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Paula+M.++Vertino">Paula M. Vertino</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Xiaodong Cheng</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xiaodong Cheng</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Molecular and Cellular Oncology, The University of Texas MD Anderson Cancer Center, Houston, Texas 77030, United States</div><div class="loa-info-affiliations-info">Department of Biochemistry, Emory University School of Medicine, Atlanta, Georgia 30322, United States</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#19617a717c777e2c59747d78777d7c6b6a767737766b7e"><span class="__cf_email__" data-cfemail="5e263d363b30396b1e333a3f303a3b2c2d313070312c39">[email protected]</span></a></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xiaodong++Cheng">Xiaodong Cheng</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-6967-6362" title="Orcid link">http://orcid.org/0000-0002-6967-6362</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.8b01219&amp;href=/doi/10.1021%2Facs.jmedchem.8b01219" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2018</span></span><span class="cit-volume">, 61</span><span class="cit-issue">, 23</span><span class="cit-pageRange">, 10588–10601</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">November 3, 2018</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>2 August 2018</li><li><span class="item_label"><b>Published</b> online</span>3 November 2018</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 13 December 2018</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.8b01219" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b01219</a></div><div class="article_header-article-copyright"><strong>Copyright © 2018 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D10588%26pageCount%3D14%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DJohn%2BR.%2BHorton%252C%2BClayton%2BB.%2BWoodcock%252C%2BQin%2BChen%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D61%26issueNum%3D23%26contentID%3Dacs.jmedchem.8b01219%26title%3DStructure-Based%2BEngineering%2Bof%2BIrreversible%2BInhibitors%2Bagainst%2BHistone%2BLysine%2BDemethylase%2BKDM5A%26numPages%3D14%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D10601%26publicationDate%3DDecember%2B2018">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.8b01219"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">2546</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">13</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.8b01219" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Structure-Based Engineering of Irreversible Inhibitors against Histone Lysine Demethylase KDM5A&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;John&quot;,&quot;last_name&quot;:&quot;R. Horton&quot;},{&quot;first_name&quot;:&quot;Clayton&quot;,&quot;last_name&quot;:&quot;B. Woodcock&quot;},{&quot;first_name&quot;:&quot;Qin&quot;,&quot;last_name&quot;:&quot;Chen&quot;},{&quot;first_name&quot;:&quot;Xu&quot;,&quot;last_name&quot;:&quot;Liu&quot;},{&quot;first_name&quot;:&quot;Xing&quot;,&quot;last_name&quot;:&quot;Zhang&quot;},{&quot;first_name&quot;:&quot;John&quot;,&quot;last_name&quot;:&quot;Shanks&quot;},{&quot;first_name&quot;:&quot;Ganesha&quot;,&quot;last_name&quot;:&quot;Rai&quot;},{&quot;first_name&quot;:&quot;Bryan&quot;,&quot;last_name&quot;:&quot;T. Mott&quot;},{&quot;first_name&quot;:&quot;Daniel&quot;,&quot;last_name&quot;:&quot;J. Jansen&quot;},{&quot;first_name&quot;:&quot;Stephen&quot;,&quot;last_name&quot;:&quot;C. Kales&quot;},{&quot;first_name&quot;:&quot;Mark&quot;,&quot;last_name&quot;:&quot;J. Henderson&quot;},{&quot;first_name&quot;:&quot;Matthew&quot;,&quot;last_name&quot;:&quot;Cyr&quot;},{&quot;first_name&quot;:&quot;Katherine&quot;,&quot;last_name&quot;:&quot;Pohida&quot;},{&quot;first_name&quot;:&quot;Xin&quot;,&quot;last_name&quot;:&quot;Hu&quot;},{&quot;first_name&quot;:&quot;Pranav&quot;,&quot;last_name&quot;:&quot;Shah&quot;},{&quot;first_name&quot;:&quot;Xin&quot;,&quot;last_name&quot;:&quot;Xu&quot;},{&quot;first_name&quot;:&quot;Ajit&quot;,&quot;last_name&quot;:&quot;Jadhav&quot;},{&quot;first_name&quot;:&quot;David&quot;,&quot;last_name&quot;:&quot;J. Maloney&quot;},{&quot;first_name&quot;:&quot;Matthew&quot;,&quot;last_name&quot;:&quot;D. Hall&quot;},{&quot;first_name&quot;:&quot;Anton&quot;,&quot;last_name&quot;:&quot;Simeonov&quot;},{&quot;first_name&quot;:&quot;Haian&quot;,&quot;last_name&quot;:&quot;Fu&quot;},{&quot;first_name&quot;:&quot;Paula&quot;,&quot;last_name&quot;:&quot;M. Vertino&quot;},{&quot;first_name&quot;:&quot;Xiaodong&quot;,&quot;last_name&quot;:&quot;Cheng&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2018&quot;,&quot;month&quot;:&quot;11&quot;,&quot;day&quot;:&quot;03&quot;,&quot;issue&quot;:&quot;23&quot;,&quot;volume&quot;:&quot;61&quot;,&quot;pages&quot;:&quot;10588-10601&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.8b01219&quot;},&quot;abstract&quot;:&quot;The active sites of hundreds of human α-ketoglutarate (αKG) and Fe(II)-dependent dioxygenases are exceedingly well preserved, which challenges the design of selective inhibitors. We identified a noncatalytic cysteine (Cys481 in KDM5A) near the active sites of KDM5 histone H3 lysine 4 demethylases, which is absent in other histone demethylase families, that could be explored for interaction with the cysteine-reactive electrophile acrylamide. We synthesized analogs of a thienopyridine-based inhibitor chemotype, namely, 2-((3-aminophenyl)(2-(piperidin-1-yl)ethoxy)methyl)thieno[3,2-b]pyridine-7-carboxylic acid (N70) and a derivative containing a (dimethylamino)but-2-enamido)phenyl moiety (N71) designed to form a covalent interaction with Cys481. We characterized the inhibitory and binding activities against KDM5A and determined the cocrystal structures of the catalytic domain of KDM5A in complex with N70 and N71. Whereas the noncovalent inhibitor N70 displayed αKG-competitive inhibition that could be reversed aft&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b01219&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b01219" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b01219&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b01219" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b01219&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b01219" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.8b01219&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b01219&amp;href=/doi/10.1021/acs.jmedchem.8b01219" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.8b01219" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.8b01219" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (7 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b01219&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.8b01219%26sid%3Dliteratum%253Aachs%26pmid%3D30392349%26genre%3Darticle%26aulast%3DHorton%26date%3D2018%26atitle%3DStructure-Based%2BEngineering%2Bof%2BIrreversible%2BInhibitors%2Bagainst%2BHistone%2BLysine%2BDemethylase%2BKDM5A%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D61%26issue%3D23%26spage%3D10588%26epage%3D10601%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=290831" title="Monomers">Monomers</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a>,</li><li><a href="/action/doSearch?ConceptID=292178" title="Organic compounds">Organic compounds</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/61/23" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/jmcmar.2018.61.issue-23/20181213/jmcmar.2018.61.issue-23.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01219/20181207/images/medium/jm-2018-01219j_0009.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01219/20181207/images/large/jm-2018-01219j_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01219&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">The active sites of hundreds of human α-ketoglutarate (αKG) and Fe(II)-dependent dioxygenases are exceedingly well preserved, which challenges the design of selective inhibitors. We identified a noncatalytic cysteine (Cys481 in KDM5A) near the active sites of KDM5 histone H3 lysine 4 demethylases, which is absent in other histone demethylase families, that could be explored for interaction with the cysteine-reactive electrophile acrylamide. We synthesized analogs of a thienopyridine-based inhibitor chemotype, namely, 2-((3-aminophenyl)(2-(piperidin-1-yl)ethoxy)methyl)thieno[3,2-<i>b</i>]pyridine-7-carboxylic acid (<b>N70</b>) and a derivative containing a (dimethylamino)but-2-enamido)phenyl moiety (<b>N71</b>) designed to form a covalent interaction with Cys481. We characterized the inhibitory and binding activities against KDM5A and determined the cocrystal structures of the catalytic domain of KDM5A in complex with <b>N70</b> and <b>N71</b>. Whereas the noncovalent inhibitor <b>N70</b> displayed αKG-competitive inhibition that could be reversed after dialysis, inhibition by <b>N71</b> was dependent on enzyme concentration and persisted even after dialysis, consistent with covalent modification.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_57113" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_57113" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Protein (histone) lysine methylation has been implicated in regulating a broad array of biological processes including epigenetic memory, genome stability, cell cycle, gene expression and nuclear architecture, and is often compromised in diseases.<a onclick="showRef(event, 'ref1 ref2'); return false;" href="javascript:void(0);" class="ref ref1 ref2">(1,2)</a> Recent meta-analysis indicates a strong correlation between the degree of methylation at lysine 4 of histone H3 (H3K4) and the prognosis of patients with malignant tumors.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> To enable epigenetic reprogramming of gene expression and chromatin structure, histone modifications are reversible. Since the discovery of LSD1 (also known as KDM1A), the first protein lysine-specific demethylase,<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> protein (histone) lysine demethylases have emerged as essential medicinal targets in human diseases such as cancers where these enzymes are often misregulated and/or mutated.<a onclick="showRef(event, 'ref5 ref6'); return false;" href="javascript:void(0);" class="ref ref5 ref6">(5,6)</a></div><div class="NLM_p">Two dissimilar demethylase families act on methylated H3K4: KDM1/LSD1 removes methyl groups from mono- or dimethylated H3K4 using a flavin-dependent reaction,<a onclick="showRef(event, 'ref4 ref7'); return false;" href="javascript:void(0);" class="ref ref4 ref7">(4,7)</a> whereas KDM5/JARID1 eliminates methyl groups from tri- or dimethylated H3K4 via an α-ketoglutarate (αKG) and Fe(II)-dependent reaction.<a onclick="showRef(event, 'ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref8 ref9">(8,9)</a> All KDM5 family members (A-to-D) demethylate the same histone methyl mark but are present in different protein complexes and have tissue-specific expression profiles, implying distinct cellular roles.<a onclick="showRef(event, 'ref9 ref10'); return false;" href="javascript:void(0);" class="ref ref9 ref10">(9,10)</a></div><div class="NLM_p">Increasing evidence from animal models and human tumors support a role for the KDM5A and KDM5B as oncogenic drivers.<a onclick="showRef(event, 'ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref11 ref12">(11,12)</a> Significant efforts are being devoted to develop KDM5 family specific inhibitors, which include cell-permeable small molecule compounds <b>N54</b> (Patent WO2016057924),<a onclick="showRef(event, 'ref13 ref14'); return false;" href="javascript:void(0);" class="ref ref13 ref14">(13,14)</a><b>CPI-455</b>,<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a><b>KDOAM-25</b>,<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> and <b>KDM5-C70</b> (Patent WO2014053491).<a onclick="showRef(event, 'ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref17 ref18">(17,18)</a> We observed that these compounds inhibit KDM5 activity by competing with the cofactor αKG and providing one-or-two metal–ligand bonds that coordinate Fe(II) in the highly conserved active site constructed by a 2-His-1-carboxylate motif (HxE/D···H). KDM5 belongs to a large family of αKG and Fe(II)-dependent dioxygenases with importance in a broad range of critical cellular activities and includes enzymes that act on DNA (e.g., AlkB DNA repair enzymes and Tet 5-methylcytosine dioxygenases), RNA (e.g., mRNA 6-methyladenine demethylases), and proteins (e.g., hypoxia-inducible factor HIF1 proline hydroxylase and protein lysine demethylases KDM2–8), as well as enzymes that act on lipids, cellular metabolites, and signaling molecules.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> Due to the highly conserved nature of active sites in this mega family, ongoing efforts for the development of therapeutics to find selective small molecule inhibitors targeting specific family members have been challenging.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a></div><div class="NLM_p">In recent years, new approaches toward the development of selective covalent-modifier inhibitors have emerged wherein targeted reactivity with the thiol group of noncatalytic cysteine residues is employed.<a onclick="showRef(event, 'ref21 ref22'); return false;" href="javascript:void(0);" class="ref ref21 ref22">(21,22)</a> These nonenzymatic cysteines are less likely to be conserved across a large protein family, which gives an opportunity to identify a particular enzyme target and minimizes the off-target reactions. The approach of using a reactive warhead-like acrylamide to target a noncatalytic cysteine to accomplish tunable and prolonged residence time was validated for several protein kinases<a onclick="showRef(event, 'ref23 ref24 ref25'); return false;" href="javascript:void(0);" class="ref ref23 ref24 ref25">(23−25)</a> and a protein lysine methyltransferase.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> The reaction may require the deprotonation of the thiol (SH) to form a thiolate (S:) which functions as a nucleophile, resulting in an enolate intermediate, followed by protonation of the enolate to generate a thio-ether product.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> Previously, we identified a cysteine located near the active site in a stretch of loop containing two metal coordinating residues (C<sub>481</sub>WH<sub>483</sub>IE<sub>485</sub>), which is invariant among the four members of the KDM5 subfamily (e.g., Cys481 of KDM5A) but absent in other dioxygenases including other histone demethylases such as the KDM4 and KDM6 families (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A).<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> Here, we explore the interaction between this noncatalytic Cys481 of KDM5A and inhibitors with the cysteine-reactive electrophile acrylamide.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01219/20181207/images/medium/jm-2018-01219j_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01219/20181207/images/large/jm-2018-01219j_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Binding of KDM5A by <b>GSK-J1</b> and its derivatives. (A) Sequence alignment of region of KDM5A containing Cys481 (in magenta) and corresponding regions in KDM4A (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6CG1">6CG1</a>), KDM6A (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3AVS">3AVS</a>), KDM2A (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4QXB">4QXB</a>), PHF8 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3KV4">3KV4</a>), and KDM3B (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4C8D">4C8D</a>). KDM5A and KDM4A share the highest sequence conservation (white letters against black background), followed by KDM6A. (B) Structure of ternary complex of GSK-J1-KDM5A-Mn(II) at 1.39 Å resolution (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6DQ4">6DQ4</a>). The omit electron density, contoured at 10σ above the mean, is shown for GSK-J1 (gray mesh). (C) Two alternative conformations are observed for Tyr409. The 2Fo–Fc electron density, contoured at 1.5σ above the mean, is shown for KDM5 protein residues (light gray mesh). (D) Cys481 is 4.2 Å away from the tip of GSK-J1. (E) Two orthogonal orientations of superimposed <b>GSK-J1</b> bound in KDM5A (yellow) and KDM6B (magenta). Cys481 of KDM5A replaces Pro-1388 of KDM6B. (F) Summary of the IC<sub>50</sub> values of inhibition of demethylation for catalytic domains of KDM4A, KDM5A, and KDM6A by FDH-coupled assay and isothermal titration calorimetry measurement of dissociation constants (<i>K</i><sub>D</sub>) of compounds to KDM5A catalytic domain (see <a class="ref internalNav" href="#notes1" aria-label="Figure S1">Figure S1</a> for original experimental data). (G) Chemical structures of <b>GSK-J1</b> and two related derivatives. Included are summary of inhibitor-bound X-ray structures of KDM5A (see <a class="ref internalNav" href="#notes1" aria-label="Table S1">Table S1</a> for detailed statistics of X-ray diffraction and refinement). (H) Structural details showing interactions between KDM5A and <b>N43</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6DQ5">6DQ5</a>). (I) Interactions displayed between KDM5A and <b>N44</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6DQ6">6DQ6</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01219/20181207/images/large/jm-2018-01219j_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01219&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_04777" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_04777" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">In this study, we synthesized and characterized structurally and biochemically modified variants of a known KDM inhibitor, GSK-J1,<a onclick="showRef(event, 'ref28 ref29'); return false;" href="javascript:void(0);" class="ref ref28 ref29">(28,29)</a> as well as analogs of a new, more potent, thienopyridine-based chemotype with the goal of engaging Cys481. In total, we determined six complex structures of KDM5A bound with inhibitors to a resolution of 1.39–1.89 Å (<a class="ref internalNav" href="#notes1" aria-label="Table S1">Table S1</a>). We define a derivative of 2-((3-aminophenyl)(2-(piperidin-1-yl)ethoxy)methyl)thieno[3,2-<i>b</i>]pyridine-7-carboxylic acid containing a ((dimethylamino)but-2-enamido)phenyl moiety (<b>N71</b>) as a KDM5A inhibitor that covalently interacts with Cys481.</div><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> GSK-J1 Occupies the Space in Vicinity of Cys481 of KDM5A</h3><div class="NLM_p">Previously, we modeled GSK-J1, a small molecule inhibitor initially developed for the KDM6 subfamily (which removes methyl groups from H3K27me3)<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> that also cross-inhibits the KDM5 subfamily,<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> in the active site of KDM5A.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> Using the purified minimal catalytic domain of KDM5A,<a onclick="showRef(event, 'ref14 ref17 ref27'); return false;" href="javascript:void(0);" class="ref ref14 ref17 ref27">(14,17,27)</a> we began by cocrystallizing GSK-J1 in complex with KDM5A to a resolution of 1.39 Å (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B). The near atomic resolution structure allowed us to position accurately every atom of GSK-J1 and many alternative conformations of protein side chains, particularly those of residues involved in binding of the inhibitor in the active site (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>C). Most importantly, the tip of the tetrahydrobenzazepine moiety of GSK-J1 is only 4.2 Å away from the sulfur atom of Cys481 (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>D).</div><div class="NLM_p">The GSK-J1-bound KDM5A catalytic domain is similar in overall structure to the GSK-J1-bound KDM6B (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4ASK">4ASK</a>) [or KDM6A (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ZPO">3ZPO</a>)] with a root-mean-squared deviation of ∼2 Å across more than 200 pairs of aligned Cα atoms. As with KDM6, the pyridyl-pyrimidine biaryl of GSK-J1 makes a bidentate contact with the metal ion and the carboxylate group of propanoic acid moiety of GSK-J1 bridging between Lys501 and Tyr409 (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B). One notable difference is that the seven-membered azepine ring is in a twist boat conformation bound in KDM5A, whereas it was modeled as a chair conformation bound in KDM6 (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>E). Furthermore, the residues surrounding GSK-J1 are highly conserved between KDM5 and KDM6, supporting the cross-inhibition of GSK-J1 against the members of KDM5 family<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>F), except that Cys481 of KDM5A replaces Pro1388 of KDM6B, which packs against the aromatic face of tetrahydrobenzazepine (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>E).</div><div class="NLM_p">Next, we modified GSK-J1 by substituting the tetrahydrobenzazepine moiety with acryloyl-1,4-diazepine (<b>N43</b>) or vinylsulfonyl-1,4-diazepine (<b>N44</b>) (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>G). These substitutions replaced the rigid benzene ring in GSK-J1 with a more flexible moiety containing the reactive electrophile. The common constituents of the three compounds displayed nearly identical contacts with the metal ion and the protein residues in the active site of KDM5A (compare <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>D and <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>H,I). However, neither of these two new compounds appeared to form a covalent bond with Cys481 in our crystal structures. In <b>N43</b>, the carbonyl oxygen of acrylamide is 3.4 Å away from the sulfur atom of Cys481, forming a hydrogen-bond-like interaction (C═O···H–S), while the terminal CH<sub>2</sub> group gains an additional van der Waals contact with the imidazole ring of His483 (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>H). This new interaction might be responsible for the moderate improvement from the original GSK-J1 in binding affinity (2× reduced dissociation constant <i>K</i><sub>D</sub>) and enzyme inhibitory activity (1.5× reduced IC<sub>50</sub> value for KDM5A catalytic domain and 4.5× reduced for near full-length KDM5A) (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>F and <a href="/doi/suppl/10.1021/acs.jmedchem.8b01219/suppl_file/jm8b01219_si_002.pdf" class="ext-link">S1–S2</a>).</div><div class="NLM_p last">Similarly, in <b>N44</b>, the side chain of Cys481 adopts two alternative conformations in interaction with one of the sulfonyl oxygen atoms (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>I). These observations suggested that rotating the torsional angle along the C<sub>3</sub>–N<sub>4</sub> bond in <b>N43</b>, or S<sub>3</sub>–N<sub>4</sub> bond in <b>N44</b>, might place the terminal CH<sub>2</sub> group of the double bond close to the sulfur atom of Cys481, within covalent bonding distance, but the geometry of bond angle along the three atoms S···C<sub>1</sub>═C<sub>2</sub> is ∼160°, which is not optimal for a covalent bond formation, which would require the sulfur nucleophile to approach the double bond in an angle close to 116°.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a></div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Synthesis of Analogs of GSK-J1</h3><div class="NLM_p">Analogs of GSK-J1 (compounds <b>N43</b> and <b>N44</b>) were synthesized by the S<sub>N</sub>Ar reaction of monoprotected homopiperazine with the corresponding chloropyrimidine (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>). The warheads were attached via acid chloride or HATU-mediated coupling. Hydrolysis of the ethyl ester was performed with trimethyltinhydroxide to afford the final compounds.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a></div><figure id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01219/20181207/images/medium/jm-2018-01219j_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01219/20181207/images/large/jm-2018-01219j_0007.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Compounds <b>N43</b> and <b>N44</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01219/20181207/images/large/jm-2018-01219j_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01219&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Thienopyridine–Carboxylate Scaffold</h3><div class="NLM_p">Recently, we characterized a collection of compounds containing a 1<i>H</i>-pyrrolo[3,2-<i>b</i>]pyridine-7-carboxylate (pyridine-7-carboxylate) moiety and 2-chlorophenyl moiety connected to a variable length of alkoxyether bound to KDM5A (e.g., compound <b>N46</b>; <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A).<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> These compounds are racemic mixtures containing a chiral secondary carbon atom (highlighted by a red star in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A) connected by the three different substituents and a hydrogen atom. We replaced the pyrrolo ring NH group with a sulfur atom and generated compound <b>N49</b> [2-((2-chlorophenyl)(2-(1-methylpyrrolidin-2-yl)ethoxy)methyl)thieno[3,2-<i>b</i>]pyridine-7-carboxylic acid] (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B). The dissociation constants (<i>K</i><sub>D</sub>), measured by isothermal titration calorimetry, indicated that the difference in binding affinity to KDM5A between <b>N46</b> and <b>N49</b> is approximately 2-fold (<a class="ref internalNav" href="#notes1" aria-label="Figure S1B">Figure S1B</a>). In comparison, two pure enantiomers of a related compound (e.g., <b>N40</b>) exhibited approximately a 4-fold difference in binding affinity;<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> we thus concluded that the thienopyridine binds in a similar manner to pyrrolopyridine carboxylate scaffold.</div><figure id="fig2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01219/20181207/images/medium/jm-2018-01219j_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01219/20181207/images/large/jm-2018-01219j_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Comparison of <b>N46</b> and <b>N49</b>. (A,B) Chemical structures of <b>N46</b> and <b>N49</b> are identical except containing a 1<i>H</i>-pyrrolo[3,2-<i>b</i>]pyridine-7-carboxylate moiety (panel A) or thienopyridine-7-carboxylate (panel B). Included are a summary of ITC measurements of dissociation constants (<i>K</i><sub>D</sub>) (see <a class="ref internalNav" href="#notes1" aria-label="Figure S1B">Figure S1B</a> for original binding data) and a summary of inhibitor-bound X-ray structures of KDM5A (<a class="ref internalNav" href="#notes1" aria-label="Table S1">Table S1</a>). (C) In the racemic mixture of compound <b>N49</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6DQ8">6DQ8</a>), two possible conformations of 2-chlorophenyl moiety can be modeled into the electron density. (D) View rotated ∼180° from panel C.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01219/20181207/images/large/jm-2018-01219j_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01219&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">As expected from the racemic mixture of compound <b>N49</b>, we observed two conformations (<i>S</i> and <i>R</i>) in the electron density, each with approximately 50% occupancy (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>C). In the <i>S</i>-enantiomer (cyan in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>C,D), the chlorine atom of 2-chlorophenyl makes van der Waals interactions with Ala411 and Tyr409, whereas in the <i>R</i>-enantiomer (yellow in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>C,D), the corresponding chlorine atom forms a van der Waals contact with His483, a mimic to what was observed between KDM5A and the terminal CH<sub>2</sub> group of acrylamide moiety in <b>N43</b> (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>H). The distance between Cys481 and the phenyl ring carbon-3 position of (<i>R</i>)-<b>N49</b> is 4.5 Å.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Engaging Cys481</h3><div class="NLM_p">We next replaced the 2-chlorophenyl moiety in <b>N49</b> with a 3-aminophenyl group, as well as a substitution of piperidine for pyrrolidine to generate <b>N70</b> (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A). The latter replacement did not affect binding affinity to KDM5A.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> This resulted in an amino group in close contact (3.2 Å) with Cys481, forming a hydrogen-bond-like N–H···S interaction (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B). It appears that the sulfur atom of Cys481 is deprotonated and functioning as a proton acceptor. Although the racemic mixture of <b>N70</b> was used for crystallization, the electron density mapped well with the <i>R</i>-enantiomer (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B), signifying the preferential binding of this enantiomer to the protein.</div><figure id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01219/20181207/images/medium/jm-2018-01219j_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01219/20181207/images/large/jm-2018-01219j_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Structural snapshots of covalent modifiers bound into KDM5A active site. (A) Chemical structure of <b>N70</b>. (B) The amino group of 3-aminophenyl moiety of <b>N70</b> involves in a H-bond-like interaction with Cys481 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6DQA">6DQA</a>). The omit electron density, contoured at 5σ above the mean, is shown for the inhibitor (gray mesh). (C) Chemical structure of <b>N71</b>. (D) A covalent bond is formed between Cys481 and the C<sub>1</sub> carbon of <b>N71</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6DQB">6DQB</a>). (E) Two methyl groups of the capping dimethylamino-2-butenamide moiety formed weak H-bond contacts with main chain carbonyl oxygen atoms of Ser479 and Arg76, whereas the nitrogen was solvent exposed to water molecules (away from the viewer). (F) Superimposition of <b>N70</b> and <b>N71</b> bound with KDM5A. Note the rotation along the chiral carbon and pyrrolopyridine ring resulting in movement of 3-aminophenyl ring and alkoxyether piperidine moiety. (G) Chemical structure of Afatinib contains the same moiety of dimethylamino-2-butenamide. (H) Afatinib covalently inhibits EGFR kinase (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4G5J">4G5J</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01219/20181207/images/large/jm-2018-01219j_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01219&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">The first step of cysteine covalent modification involves deprotonation of the thiol (SH) to form a thiolate (S:), which functions as a nucleophile.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> Encouraged by the potentially reactive Cys481 thiolate noted in the <b>N70</b> structure above, we replaced the amino group at the phenyl ring carbon-3 position in <b>N70</b> with a dimethylamino-2-butenamide moiety, harboring a reactive acrylamide group to yield <b>N71</b> (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>C). A covalent bond is clearly formed between Cys481 and the Michael acceptor group of <b>N71</b> (S–C distance = 1.8 Å and Cβ–S–C<sub>1</sub> angle = 110°; <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>D). The dimethylamino group H-bonds with a water molecule and interacts with the main chain carbonyl oxygen atoms of Arg76 and Ser479, respectively, via two C–H···O═C type of hydrogen bonds<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>E). In addition, the carbonyl oxygen of the acrylamide group of <b>N71</b> makes a hydrogen bond with the main-chain amide nitrogen of Cys481 (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>D), a conformation that is rotated from that of <b>N70</b> (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>F). The same moiety, dimethylamino-2-butenamide, has been used in Afatinib (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>G), a protein kinase inhibitor that irreversibly inhibits human epidermal growth factor receptor (EGFR) family tyrosine kinases<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> (S–C distance = 1.8 Å and Cβ–S–C<sub>1</sub> angle = 111°; <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>H). The conformation of the covalent bond formation is highly similar between KDM5A-<b>N71</b> and EGFR kinase bound to Afatinib<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> (compare <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>D and H).</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Covalent Bond Formation in Solution</h3><div class="NLM_p">We explored several approaches to evaluate the compound behavior in solution. Because the pyrrolopyridine-7-carboxylate-based compounds (<b>N49</b>, <b>N70</b>, and <b>N71</b>) are intrinsically fluorescent and interfere with the fluorescence signal of the formaldehyde dehydrogenase (FDH)-coupled demethylase assay,<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> we first utilized an AlphaLISA-based assay to measure potency and selectivity of these compounds against the activities of KDM5A, KDM5B, KDM4A, and KDM6A. This assay employs an antibody against histone H3 methylated at K4 (for KDM5), K9 (for KDM4A), or K27 (for KDM6A) to measure the demethylation of histone peptide substrates specific for each enzyme (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A and <a href="/doi/suppl/10.1021/acs.jmedchem.8b01219/suppl_file/jm8b01219_si_002.pdf" class="ext-link">S2</a>). Interestingly, the selectivity for KDM5B over KDM4A was 3 times greater (24-fold) for the covalent inhibitor <b>N71</b> compared with two noncovalent inhibitors (<b>N49</b> and <b>N70</b>), which is approximately 7–8-fold (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A; both enzymes are recombinant catalytic domains purified from <i>Escherichia coli</i> and tested under the same enzyme concentration). All three inhibitors have the same high selectivity against KDM6A (<a class="ref internalNav" href="#notes1" aria-label="Figure S2">Figure S2</a>). In addition to the enhanced selectivity, the strength of the covalent bond also conferred a 2-fold increase in enzyme inhibitory activity (i.e., IC<sub>50</sub> value decreased ∼2× for <b>N71</b> as compared to the noncovalent <b>N70</b>). As expected, the inhibition constants of <b>N70</b> and <b>N71</b> are about the same against KDM4A, which shares the highest sequence conservation to KDM5A but lacks the cysteine at the corresponding position (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A).</div><figure id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01219/20181207/images/medium/jm-2018-01219j_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01219/20181207/images/large/jm-2018-01219j_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Irreversible inhibition of KDM5A in solution. (A) Summary of AlphaLISA derived IC<sub>50</sub> values from inhibition of demethylation by KDM5B and KDM4A (see <a class="ref internalNav" href="#notes1" aria-label="Figure S2">Figure S2</a> for original inhibition data). (B–G) IC<sub>50</sub> values of compounds <b>N70</b> and <b>N71</b> against KDM5A(1–739)ΔAP using the Promega Succinate-Glo assay under three αKG concentrations. The order involves preincubation (20 min) with the inhibitors before the addition of αKG to initiate the reaction. (B) Inhibition of KDM5A by <b>N70</b> is αKG dependent. (C) <b>N71</b> has no significant change to IC<sub>50</sub> values with variation in αKG concentration. (D) Inhibition of KDM5A by <b>N71</b> is enzyme concentration dependent. (E,F) Inhibition of KDM5A with <b>N71</b> (E) or <b>N70</b> (F) with or without incubation for 20 min. (G) Preformation of enzyme–inhibitor complex ([E] = 5 μM and initial [I] = 100 μM) followed by four times of buffer exchange removes the noncovalent modifying <b>N70</b> from solution but does not perturb <b>N71</b> complex with KDM5A. (H) KDM5A ([E] = 0.8 μM) was preincubated with five concentrations of <b>N71</b> ([I] = 0.01–5 μM). At each time point, the enzymatic reaction started by adding 5 μL of preincubated E+I complex to a solution containing H3 peptide ([S] = 60 μM) and αKG (100 μM), and the reaction was stopped after 7 min. (I) Inactivation plot, logarithm of the % remaining activity versus preincubation time. (J) Nonlinear regression analysis of the negative slopes against inhibitor concentration gives rise to <i>k</i><sub>inact</sub> and <i>K</i><sub>I</sub> values as indicated. (K) Summary of electrospray ionization mass spectrometry (ESI-tandem-MS) of KDM5A in the absence and presence of inhibitors (see <a class="ref internalNav" href="#notes1" aria-label="Figure S3">Figure S3</a> for raw spectrum). KDM5A (50 μM) was incubated with excess inhibitor (0.5 mM) for 30 min at ambient temperature.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01219/20181207/images/large/jm-2018-01219j_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01219&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Next, we used an alternative approach to assess enzyme activity utilizing the Promega Succinate-Glo demethylase assay, a bioluminescence-based assay for detecting the activity of demethylases that use αKG as substrate and release succinate as a product.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> As expected, the inhibition of KDM5A by the noncovalent inhibitor <b>N70</b> was dependent on αKG concentration (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>B), whereas that by the covalent inhibitor <b>N71</b> did not appear to be sensitive to the αKG concentration (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>C). Instead, inhibition by <b>N71</b> was dependent on enzyme concentration (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>D), a feature expected for a covalent modifier exhibiting tight binding. As a result, we observed enhanced inhibition for the covalent modifier, i.e., the IC<sub>50</sub> value for <b>N71</b> was now ∼50-fold lower than that of <b>N70</b> at 100 μM αKG concentration. In addition, preincubation with KDM5A increased inhibition potency of <b>N71</b>, whereas <b>N70</b> was insensitive (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>E,F), suggesting that <b>N71</b> undergoes a time-dependent inhibition. Furthermore, whereas enzyme activity could be restored after buffer exchange (or dilution) for the <b>N70</b>-bound KDM5A, it had no effect on the inhibition of <b>N71</b>-bound KDM5A (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>G) consistent with very tight or irreversible binding between the compound and KDM5A. We also characterized parameters of <i>k</i><sub>inact</sub> (maximal rate of inactivation) and <i>K</i><sub>I</sub> (concentration at 50% <i>k</i><sub>inact</sub>) of <b>N71</b> by varying five different inhibitor concentrations (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>H–J).<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> The value of <i>k</i><sub>inact</sub>/<i>K</i><sub>I</sub> of <b>N71</b> against KDM5A is approximately 8.0 × 10<sup>3</sup> M<sup>–1</sup> s<sup>–1</sup>. Finally, we confirmed by mass spectrometry that <b>N71</b>, but not <b>N70</b>, covalently modified KDM5A enzyme completely (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>K and <a href="/doi/suppl/10.1021/acs.jmedchem.8b01219/suppl_file/jm8b01219_si_002.pdf" class="ext-link">S3</a>). The addition of mass equals to one <b>N71</b> molecule in the presence excess of <b>N71</b>, despite the enzyme containing multiple cysteine residues on its surface. Together with structural observation, these data support <b>N71</b> as a KDM5A inhibitor that covalently interacts with Cys481.</div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Isopropyl Ester Derivative of Thienopyridine–Carboxylic Acid Is Cytotoxic</h3><div class="NLM_p">As with the pyrrolopyridine-7-carboxylate-based compounds described here and in our earlier work,<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> some known KDM inhibitors, such as <b>KDM5-C49</b> (Patent WO2014053491) and <b>GSK-J1</b>,<a onclick="showRef(event, 'ref28 ref29'); return false;" href="javascript:void(0);" class="ref ref28 ref29">(28,29)</a> contain a carboxylate moiety that engages hydrogen bonding and ionic interactions with the active-site residues Tyr409 and Lys501 (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B,C), but precludes cell uptake. For this reason, the respective ethyl ester derivatives, <b>KDM5-C70</b> and <b>GSK-J4</b>, were synthesized as cell-permeable prodrugs that are hydrolyzed by cellular esterase(s) to generate active <b>KDM5-C49</b> and <b>GSK-J1</b>, respectively. We therefore synthesized <b>SI-5</b> and <b>N73</b>, the isopropyl ester derivatives of <b>N70</b> and <b>N71</b>, respectively, and tested their cellular activities (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A). We measured the effective permeability using the parallel artificial membrane permeability assay (PAMPA).<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> In each case, the ester derivative is much more permeable than the corresponding free acid (<b>N73</b> > <b>N71</b> and <b>SI-5</b> > <b>N70</b>) (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B). The permeability of compounds is in the order <b>SI-5</b> (high) > <b>N73</b> ≈ <b>N70</b> (moderate) > <b>N71</b> (low).</div><figure id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01219/20181207/images/medium/jm-2018-01219j_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01219/20181207/images/large/jm-2018-01219j_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Concentration dependence of cytotoxicity of <b>N73</b>. (A) Chemical structures of ethyl (<b>GSK-J4</b> and <b>KDM5-C70</b>) or isopropyl ester (<b>N73</b> and <b>SI-5</b>) derivatives of indicated compounds. (B) PAMPA permeability. (C,D) Western blot analyses of MCF7 cells treated with the indicated concentration of compounds or DMSO (0.1%) control for 30 or 48 h. Total cells lysates were used for assays using indicated antibodies. (E) <b>N73</b> (isopropyl ester derivative of <b>N71</b>) does not increase H3K4me3 level in BT474 cells at 48 h as measured by immunocytochemistry. (F) Rapid cytotoxicity in BT474 cells treated with <b>N73</b> at concentrations >5 μM. Cell viability was measured using a CellTiter-Glo assay 24 h after treatment. (G,H) Proliferation of BT474 cells treated with <b>N73 or SI-5</b>. Percent confluence was tracked over 300 h by imaging the cells every 4 h using an IncuCyte live cell imaging system. (I) Real-time RT-PCR analysis of the indicated mRNAs in MCF7 cells treated with the indicated concentration of compounds or DMSO for 48 h. Data from biological triplicate experiments are shown.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01219/20181207/images/large/jm-2018-01219j_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01219&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Exposure of MCF7 (ER+) or BT474 (HER2+) breast cancer cells to 1 μM <b>KDM5-C70</b> resulted in an increase in cellular H3K4me3 levels, as previously described<a onclick="showRef(event, 'ref14 ref17'); return false;" href="javascript:void(0);" class="ref ref14 ref17">(14,17)</a> (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>C,D), the isopropyl ester derivative <b>N73</b> had no impact on cellular H3K4me3 methylation levels at doses up to 5 μM (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>E). We note that GSK-J4 influences little of the overall levels of H3K4me3 or H3K27me3 in MCF7 under the same conditions tested (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>C). Nevertheless, <b>N73</b> did affect cellular proliferation and viability of BT474 cells rapidly within 24 h (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>F). Interestingly, the steep loss of proliferation and viability only occurs at doses of <b>N73</b> above 5 μM (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>F), whereas the low doses do not affect cell growth over the period of >12 days (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>G). In contrast, <b>SI-5</b>, which has greater permeability but lacks the reactive acrylamide group, only showed some growth inhibition at the highest concentration tested (28 μM) (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>H).</div><div class="NLM_p">Similarly, we observed a noteworthy difference in cell killing within 24 h between 5 and 10 μM of <b>N73</b> concentrations in cells of MCF7, MDA-MB-231 (triple negative for HER2, PR, and ER), and HEK293T (human embryonic kidney cells) (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A–C). These observations suggested that <b>N73</b> at concentrations above 5 μM has strong and rapid growth inhibitory/cell killing activity among the cell types tested, although whether this is tied to the inhibition of endogenous KDM5 enzymes will require further studies. Several target genes of KDM5-mediated repression have been identified, and we focused here on three genes whose expression was up-regulated upon <b>KDM5-C70</b> treatment in MCF7 cells: <i>STING</i><a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> and <i>MT1F</i> and <i>MT1H</i><a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>D,I). Treatment of MCF7 with <b>N73</b> at concentrations of 1 or 5 μM for 2 days led to small changes (<2×) in the expression of <i>STING</i>, <i>MT1F</i>, and <i>MT1H</i> (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>D,I). In contrast, admission with <b>KDM5-C70</b> under the same condition (1 μM for 2 days) increased the expression of the three targets up to 6-fold.</div><figure id="fig6" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01219/20181207/images/medium/jm-2018-01219j_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01219/20181207/images/large/jm-2018-01219j_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Images of bright-field microscopy of live cells after 24 h post-treatment with indicated concentrations of <b>KDM5-C70</b> (top), <b>N71</b> (middle), and <b>N73</b> (bottom) or 0.1% DMSO (upper left corner) in MCF7 (A), MDA-MB-231 (B), and HEK293T (C). Scale bar is 50 μM.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01219/20181207/images/large/jm-2018-01219j_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01219&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Synthesis of Covalent KDM5 Inhibitors N71 (Acid) and N73 (Isopropyl Ester)</h3><div class="NLM_p">Synthesis of the covalent modifiers started with esterification of the commercially available 2-bromothieno[3,2-<i>b</i>]pyridine-7-carboxylic acid with Boc-anhydride.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> The corresponding Grignard reagent was then prepared and reacted with di<i>tert</i>-butyl (3-formylphenyl) imidodicarbonate to provide the benzyl alcohol as a mixture of mono- and bis-<i>tert</i>-butyl carbamates.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> Solvolysis of the benzyl alcohol with excess 1-(2-hydroxyethyl)piperidine in methanesulfonic acid followed by alkaline hydrolysis and reverse phase purification (0.1% TFA modifier) afforded <b>N70</b> as the TFA salt. The isopropyl ester <b>SI-5</b> could be synthesized by treatment with TMS-Cl in anhydrous isopropanol. Formation of the final covalent modifiers <b>N71</b> and <b>N73</b> was accomplished by treatment with (<i>E</i>)-4-(dimethylamino)but-2-enoic acid in the presence of HATU (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>).</div><figure id="sch2" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01219/20181207/images/medium/jm-2018-01219j_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01219/20181207/images/large/jm-2018-01219j_0008.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Compounds <b>N70</b>, <b>N71</b>, <b>N73</b>, and <b>SI-5</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01219/20181207/images/large/jm-2018-01219j_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01219&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The stability of acid/ester pairs <b>N71</b>/<b>N73</b> and <b>N70</b>/<b>SI-5</b> in culture media was assessed after 24 h. Approximately 80% of <b>N73</b> remained unchanged (no <b>N71</b> was detected), while only 60% of <b>N71</b> remained unchanged (<a class="ref internalNav" href="#notes1" aria-label="Figure S4A">Figure S4A</a>). This suggests that the covalent group is reasonably stable over 24 h. Correspondingly, 75% of <b>SI-5</b> and 93% of <b>N70</b> remained unchanged after 24 h. Approximately 1% of <b>SI-5</b> is converted into <b>N70</b> under the same conditions and suggests that the isopropyl ester imparts good stability to these series of compounds in culture media.</div><div class="NLM_p last">To understand how much of the acid/ester pair <b>N71</b>/<b>N73</b> was present in cells in our growth inhibition assays, we performed a cellular extract experiment. These compounds were individually incubated with BT474 cells for 24 h and lysed to determine cell extract concentrations. The ester <b>N73</b> appeared to be highly permeable and was detected at levels 100-fold greater than the acid <b>N71</b> (<a class="ref internalNav" href="#notes1" aria-label="Figure S4B">Figure S4B</a>), in agreement with the PAMPA measurement (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B). Subsequent intracellular esterase-mediated conversion of the ester <b>N73</b> to acid <b>N71</b> appeared to be inefficient under the conditions tested.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i13">Discussion and Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_12670" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_12670" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Studies in cancer-related processes (of metastasis, drug resistance, and cell proliferation) and in genetically engineered mouse tumor models<a onclick="showRef(event, 'ref9 ref10 ref11 ref41 ref42 ref43'); return false;" href="javascript:void(0);" class="ref ref9 ref10 ref11 ref41 ref42 ref43">(9−11,41−43)</a> suggested that KDM5 inhibitors can be used for treatment of cancers, particularly in multidrug-resistant cells where expression of many histone demethylases are increased and KDM5 depletion reversed the resistant phenotype.<a onclick="showRef(event, 'ref44 ref45 ref46'); return false;" href="javascript:void(0);" class="ref ref44 ref45 ref46">(44−46)</a> Our efforts to depict the interactions of KDM5A with diverse chemotypes of inhibitors provide fundamental knowledge into structure–activity relationships of inhibitors against KDM5.<a onclick="showRef(event, 'ref14 ref17 ref27'); return false;" href="javascript:void(0);" class="ref ref14 ref17 ref27">(14,17,27)</a></div><div class="NLM_p">Here, we present <b>N71</b> [(<i>E</i>)-2-((3-(4-(dimethylamino)but-2-enamido)phenyl)(2-(piperidin-1-yl)ethoxy)methyl)thieno[3,2-<i>b</i>]pyridine-7-carboxylic acid] as the first covalent modifier of KDM5 family enzymes. <i>In vitro</i>, the reactive ((dimethylamino)but-2-enamido)phenyl moiety of <b>N71</b> forms a covalent interaction with Cys481 of KDM5A. The covalently modified KDM5A exhibited an inhibition mode dependent on enzyme concentration, unlike most of the other KDM5 inhibitors to date, which act as αKG-competitive inhibitors, yielding an increased selectivity for the KDM5 family over KDM4A (24×) relative to other KDM5 inhibitors, including <b>KDM5-C70</b> (10×),<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> which has shown the most promise to date in cell-based studies.</div><div class="NLM_p">Our study aids in the design and synthesis of potent and selective inhibitors of KDM5 demethylases. A lasting challenge is to make these compounds cell permissible. Low permeability of the potent KDM5A inhibitors is probably owing to the overall compound polarity and/or acidic nature of the carboxylate moiety. Chemical modifications are required to increase cell permeability of these compounds. Several cell-permeable compounds are available that lack a carboxylate moiety, including (<i>R</i>)-<b>N54</b>,<a onclick="showRef(event, 'ref13 ref14'); return false;" href="javascript:void(0);" class="ref ref13 ref14">(13,14)</a><b>CPI-455</b>,<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> and <b>KDOAM-25</b><a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> (which replaces the carboxylate of <b>KDM5-C49</b> with a carboxyamide), as well as a range of noncarboxylate (and nonselective) inhibitors for the families of KDM5 and KDM4.<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> In addition to the thienopyridine-carboxylate (involved in interactions with metal ion and Lys501/Tyr409 cluster) and the Cys481-reactive acrylamide group of 3-aminophenyl moiety, the third component of <b>N71</b> (alkoxyether piperidine moiety) is relatively flexible (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>D). It may be beneficial to substitute/add chemical moieties<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> along this third branch to make more cell penetrant compounds.</div><div class="NLM_p last">While the isopropyl ester derivative <b>N73</b> showed some growth inhibitory and cytotoxic activity against various cell lines of breast cancer, the molecular basis of this activity is currently unknown, and whether it derives from engagement of its intended cellular target (KDM5) remains to be determined. Under similar conditions, treatment with <b>N71</b> had no effect on cell growth (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A–C). Nevertheless, whether there are compensatory mechanisms at play to balance H3K4me levels (e.g., increased activity of cognate histone lysine methyltransferase(s)) remains to be determined. Our previous work has revealed that the impact on total H3K4me3 levels is not always a reliable predictor of growth inhibitory activity across different cell lines suggesting that other factors beyond enzyme inhibition contribute to the cytotoxic effects of KDM5 inhibitors.<a onclick="showRef(event, 'ref14 ref17'); return false;" href="javascript:void(0);" class="ref ref14 ref17">(14,17)</a> This result could stem from a combination of incomplete hydrolysis of the isopropyl ester derivative in the cell and/or potentially interaction with off-target proteins. Covalent inhibitors of other epigenetic modifiers have been successfully translated to clinical practice despite the high cytotoxicity such as the aforementioned Afatinib, which demonstrated complete cytotoxicity for the EGFR expressing K562 cells at a very low dose (10 nM),<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> and nucleoside analogs.<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> DNA methyltransferase (DNMT) inhibitors, including 5-azacytidine and decitabine, become incorporated into replicating DNA as an altered cytosine base and create a DNA–DNMT covalent complex that ultimately drives the destruction of DNMT1 and hypomethylation of DNA.<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> Similarly, the primary anticancer effect of poly(ADP-ribose) polymerase (PARP) inhibitors correlates with trapping of DNA–PARP complexes at sites of DNA damage.<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> Combining DNMT and PARP inhibitors enhanced cell death and increased antitumor effects <i>in vivo</i>.<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> At present, we do not know how and where, at a molecular level, the isopropyl ester derivative of <b>N71</b> acts to inhibit growth and viability, but we note that the closely related methyl ester derivatives of pyrrolopyridine-7-carboxylate-based compounds showed no growth inhibitory activity when similarly tested<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> (and the methyl ester is not appreciably hydrolyzed in cells; <a href="/doi/suppl/10.1021/acs.jmedchem.8b01219/suppl_file/jm8b01219_si_002.pdf" class="ext-link">Figure S4C</a>). Further study will not only be invaluable in deciphering the pathway(s) affected by these molecules but also may serve as a starting point for a new class of potent inhibitors of histone lysine demethylase as anticancer agents.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i14">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_62706" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_62706" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The <a href="/doi/suppl/10.1021/acs.jmedchem.8b01219/suppl_file/jm8b01219_si_002.pdf" class="ext-link">Supporting Information</a> contains previously described experimental methods including protein crystallography, isothermal titration calorimetry, formaldehyde dehydrogenase (FDH)-coupled demethylase assay, AlphaLISA assay, Western blot analysis, H3K4me3 immunocytochemistry, and IncuCyte live cell analysis. Compound synthesis and assays new to this study are described here.</div><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Compound Synthesis</h3><div class="NLM_p">Compounds were synthesized at the National Center for Advancing Translational Sciences. Purity determination was performed using an Agilent Diode Array Detector for both Method 1 and Method 2 (below). Mass determination was performed using an Agilent 6130 mass spectrometer with electrospray ionization in the positive mode. <sup>1</sup>H NMR spectra were recorded on Varian 400 MHz spectrometers. Chemical shifts for final compounds are reported in ppm with undeuterated solvent (DMSO-<i>d</i><sub>6</sub> at 2.50 ppm) as internal standard for DMSO-<i>d</i><sub>6</sub> solutions. Chemical shifts for intermediate compounds are reported in ppm with undeuterated solvent (CDCl<sub>3</sub> at 7.26 ppm) as internal standard for CDCl<sub>3</sub> solutions. All the analogs tested in the biological assays have purity greater than 95%, based on both analytical methods. High resolution mass spectrometry was recorded on Agilent 6210 Time-of-Flight LC/MS system. Confirmation of molecular formula was accomplished using electrospray ionization in the positive mode with the Agilent Masshunter software (version B.02).</div><div id="sec4_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i16" class="anchor-spacer"></div><h4 class="article-section__title" id="_i16"> Method 1</h4><div class="NLM_p last">A 7 min gradient of 4% to 100% acetonitrile (containing 0.025% trifluoroacetic acid) in water (containing 0.05% trifluoroacetic acid) was used with an 8 min run time at a flow rate of 1 mL/min. A Phenomenex Luna C18 column (3 μm, 3 × 75 mm) was used at a temperature of 50 °C.</div></div><div id="sec4_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i17" class="anchor-spacer"></div><h4 class="article-section__title" id="_i17"> Method 2</h4><div class="NLM_p">A 3 min gradient of 4% to 100% acetonitrile (containing 0.025% trifluoroacetic acid) in water (containing 0.05% trifluoroacetic acid) was used with a 4.5 min run time at a flow rate of 1 mL/min. A Phenomenex Gemini Phenyl column (3 μm, 3 × 100 mm) was used at a temperature of 50 °C.</div><div class="NLM_p last">All air or moisture sensitive reactions were performed under positive pressure of nitrogen or argon with oven-dried glassware. Anhydrous solvents and bases such as dichloromethane, <i>N</i>,<i>N</i>-dimethylforamide (DMF), acetonitrile, ethanol, DMSO, dioxane, DIPEA (diisopropylethylamine), and triethylamine were purchased from Sigma-Aldrich. Palladium catalysts were purchased from Strem Chemicals and used as such. Preparative purification was performed on a Waters semipreparative HPLC system using a Phenomenex Luna C18 column (5 μm, 30 × 75 mm) at a flow rate of 45 mL/min (For all compounds tested, the HPLC traces were provided in <a href="/doi/suppl/10.1021/acs.jmedchem.8b01219/suppl_file/jm8b01219_si_002.pdf" class="ext-link">Supplementary Figure S5</a>). The mobile phase consisted of acetonitrile and water (each containing 0.1% trifluoroacetic acid). A gradient of 10% to 50% acetonitrile over 8 min was used during the purification. Fraction collection was triggered by UV detection (220 nm). Analytical analysis was performed on an Agilent LC/MS (Agilent Technologies, Santa Clara, CA).<img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01219/20181207/images/medium/jm-2018-01219j_0010.gif" alt="" id="dgr1" /></img></div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i18" class="anchor-spacer"></div><h4 class="article-section__title" id="_i18"> Synthesis of Ethyl 3-((6-(1,4-Diazepan-1-yl)-2-(pyridin-2-yl)pyrimidin-4-yl)amino)propanoate (<b>SI-0</b>)</h4><div class="NLM_p last">(Patent WO2012/52390A1; page 55.) To a solution of ethyl 3-((6-chloro-2-(pyridin-2-yl)pyrimidin-4-yl)amino)propanoate (1.2 g, 3.91 mmol) and 1-Boc-homopiperazine (0.838 mL, 4.30 mmol) in dioxane was added DIPEA (2.050 mL, 11.74 mmol), then heated in microwave for 1.5 h at 150 °C. The mixture was the concentrated <i>in vacuo</i>, and the residue was extracted with ethyl acetate and washed with ammonium chloride and brine. The organic layer was dried with sodium sulfate, filtered, and concentrated <i>in vacuo</i>. The residue was then purified on icso-normal phase eluting with 40–100% EA (contains 1% TEA) in hexanes using an 80 g column. The last fraction was pooled and dried to get a white solid, which was then treated with a 1:1 mixture of DCM/TFA to provide <b>SI-0</b> as the TFA salt. LC–MS retention time: <i>t</i><sub>1</sub> = 2.75 min; LC–MS M + H = 373.<img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01219/20181207/images/medium/jm-2018-01219j_0011.gif" alt="" id="dgr2" /></img></div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i19" class="anchor-spacer"></div><h4 class="article-section__title" id="_i19"> Synthesis of 3-((6-(4-Acryloyl-1,4-diazepan-1-yl)-2-(pyridin-2-yl)pyrimidin-4-yl)amino)propanoic Acid (<b>N43</b>)<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a></h4><div class="NLM_p last">To a solution of ethyl 3-((6-(1,4-diazepan-1-yl)-2-(pyridin-2-yl)pyrimidin-4-yl)amino)propanoate (0.1 g, 0.255 mmol, 1 equiv) <b>SI-0</b> in DCM (2 mL) at 0 °C, DIPEA (0.13 mL, 0.76 mmol, 3 equiv) and acryloyl chloride (0.021 mL, 0.255 mmol, 1 equiv) were added. The reaction was allowed to stir at room temperature for 1 h and was then concentrated <i>in vacuo</i>. The crude ethyl ester residue was taken up in DCE (2 mL), and hydroxytrimethylstannane (0.194 g, 1.072 mmol, 1 equiv) was added. The reaction was then capped and placed in a microwave and heated at 120 °C for 1 h. The reaction was then passed through a syringe filter and purified on HPLC to provide <b>N43</b> as the TFA salt. LC–MS retention time: <i>t</i><sub>1</sub> = 2.43 min; <i>t</i><sub>2</sub> = 3.46 min LC–MS M + H = 397. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.67–8.65 (m, 1H), 8.23–8.20 (m, 1H), 7.89–7.85 (m, 1H), 7.43–7.40 (m, 1H), 6.79–6.70 (m, 2H), 6.16–5.96 (ddd, <i>J</i> = 60.1, 16.6, 2.4 Hz, 1H), 5.68–5.51 (m, 2H), 3.82–3.59 (m, 6H), 3.53–3.43 (m, 4H), 2.55–2.51 (m, 2H), 1.87–1.80 (m, 2H).<img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01219/20181207/images/medium/jm-2018-01219j_0012.gif" alt="" id="dgr3" /></img></div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20"> Synthesis of 3-((2-(Pyridin-2-yl)-6-(4-(vinylsulfonyl)-1,4-diazepan-1-yl)pyrimidin-4-yl)amino)propanoic Acid (<b>N44</b>)</h4><div class="NLM_p last">The compound was synthesized as above except using ethenesulfonyl chloride in place of acryloyl chloride. The mixture was purified on HPLC to provide <b>N44</b> as the TFA salt. LC–MS retention time: <i>t</i><sub>1</sub> = 2.60 min; <i>t</i><sub>2</sub> = 3.73 min; LC–MS M + H = 433. <sup>1</sup>H NMR (400 MHz, DMSO-d6) δ 12.13 (br s, 1H), 8.65–8.59 (m, 1H), 8.18 (d, <i>J</i> = 7.9 Hz, 1H), 7.83 (td, <i>J</i> = 7.7, 1.8 Hz, 1H), 7.40–7.37 (m, 1H), 6.78–6.63 (m, 2H), 5.99 (d, <i>J</i> = 16.5 Hz, 1H), 5.89 (d, <i>J</i> = 10.0 Hz, 1H), 5.57 (s, 1H), 3.82–3.77 (m, 2H), 3.71–3.65 (m, 2H), 3.50–3.44 (m, 2H), 3.36 (t, <i>J</i> = 5.4 Hz, 2H), 3.22–3.11 (m, 2H), 1.88–1.81 (m, 2H).<img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01219/20181207/images/medium/jm-2018-01219j_0013.gif" alt="" id="dgr4" /></img></div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> Synthesis of <i>tert</i>-Butyl 2-bromothieno[3,2-<i>b</i>]pyridine-7-carboxylate (<b>SI-1</b>)</h4><div class="NLM_p last">To a solution of 2-bromothieno[3,2-<i>b</i>]pyridine-7-carboxylic acid (1 g, 3.87 mmol, 1 equiv) and DMAP (0.237 g, 1.937 mmol, 0.5 equiv) in 5 mL of THF was added Boc-anhydride (2.249 mL, 9.69 mmol, 2.5 equiv) in 5 mL. The reaction was heated to 65 °C and stirred for 4 h at the same temperature. The reaction was monitored by LC–MS for disappearance of starting material. The reaction was concentrated, and the residue was extracted with ethyl acetate (50 mL) and washed with aqueous sodium bicarbonate (1 × 10 mL). The organic layer was dried with sodium sulfate, filtered, and concentrated <i>in vacuo</i>. The residue was purified on isco-normal phase using an 80 g silica column eluting with 0–40% ethyl acetate in hexanes over 18 CV (elutes around 20–25% ethyl acetate). The fractions were collected and concentrated <i>in vacuo</i> to provide 1.1 g of <i>tert</i>-butyl 2-bromothieno[3,2-<i>b</i>]pyridine-7-carboxylate <b>SI-1</b> (90% yield). LC–MS retention time: <i>t</i><sub>1</sub> = 3.87 min; LC–MS M + H = 314/316. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.75 (d, <i>J</i> = 4.9 Hz, 1H), 7.84–7.74 (m, 1H), 7.67 (s, 1H), 1.68 (s, 9H).<img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01219/20181207/images/medium/jm-2018-01219j_0014.gif" alt="" id="dgr5" /></img></div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> Synthesis of <i>tert</i>-Butyl 2-((2-Chlorophenyl)(hydroxy)methyl)thieno[3,2-<i>b</i>]pyridine-7-carboxylate (<b>SI-2</b>)<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a></h4><div class="NLM_p last">A small vial capped with a rubber septa and nitrogen line was cooled with an external ice water bath. A solution of isopropylmagnesium chloride (2.0 M in THF, 0.38 mL, 0.76 mmol, 1.2 equiv) was added followed by an additional 2 mL of THF. Then, 2,2′-oxybis(<i>N</i>,<i>N</i>-dimethylethanamine) (0.15 mL, 0.76 mmol, 1.2 equiv) was added neat dropwise over 20 s. A white precipitate forms, and the reaction was warmed to room temperature by removing the cooling bath. (Note: if reaction completely solidifies, additional THF can be added.) After 5 min, the vial was again placed in the ice water cooling bath, and <i>tert</i>-butyl 2-bromothieno[3,2-<i>b</i>]pyridine-7-carboxylate (200 mg, 0.637 mmol) was added as a solution in 1 mL of THF. The reaction turned orange and was kept at 0 °C for 5–10 min. Then, 2-chlorobenzaldehyde (0.09 mL, 0.82 mmol) was added as a solution in 1 mL of THF. The reaction was slowly warmed to room temperature and stirred for at least 1 h (as monitored by LC–MS). Depending on the amount of 2-chlorobenzaldehyde added, the ketone could form in addition to the alcohol (possibly via a magnesium-mediated Oppenauer-type oxidation). If this occurs, the reaction can be diluted with methanol and a small amount of sodium borohydride can be added (1 equiv). The reaction was then concentrated <i>in vacuo</i>, and the residue was taken up in ethyl acetate (20 mL). The organic layer was washed with water (1 × 10 mL) and brine (1 × 10 mL). The organic layer was separated, dried with sodium sulfate, filtered, and concentrated <i>in vacuo</i>. The residue was taken up in a small amount of DCM and purified on normal phase isco using a 24 g gold column eluting with 0 → 50% ethyl acetate in hexanes over 12 min (product elutes at around 50% ethyl acetate). The pure fractions were collected and concentrated <i>in vacuo</i> to provide 154 mg of <i>tert</i>-butyl 2-((2-chlorophenyl)(hydroxy)methyl)thieno[3,2-<i>b</i>]pyridine-7-carboxylate <b>SI-2</b> (64% yield). LC–MS retention time: <i>t</i><sub>1</sub> = 3.59 min; LC–MS M + H = 376. <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 8.68 (d, <i>J</i> = 4.9 Hz, 1H), 7.76–7.71 (m, 2H), 7.43 (d, <i>J</i> = 1.1 Hz, 1H), 7.38–7.29 (m, 2H), 7.27–7.23 (m, 1H), 6.56 (s, 1H), 3.81 (br s, 1H), 1.65 (s, 9H).<img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01219/20181207/images/medium/jm-2018-01219j_0015.gif" alt="" id="dgr6" /></img></div></div><div id="sec4_6_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> Synthesis of 2-((2-Chlorophenyl)(2-(1-methylpyrrolidin-2-yl)ethoxy)methyl)thieno[3,2-<i>b</i>]pyridine-7-carboxylic Acid (<b>N49</b>)</h4><div class="NLM_p last"><i>tert</i>-Butyl 2-((2-chlorophenyl)(2-(1-methylpyrrolidin-2-yl)ethoxy)methyl)thieno[3,2-<i>b</i>]pyridine-7-carboxylate (110 mg, 0.29 mmol, 1 equiv) was put into a 2–5 mL microwave vessel along with a small stir bar. 2-(1-Methylpyrrolidin-2-yl)ethanol (119 μL, 0.88 mmol) was added to the bottom of the microwave vessel, and then methanesulfonic acid (330 μL, 5.12 mmol) was added. The vessel was then purged with a nitrogen stream, and the vessel was sealed with a microwave cap. The reaction was then placed in a preheated reaction block set to 150 °C, and the reaction was stirred for 10–15 min. The reaction turns slightly orange over 10–15 min (if it turns brown or black, the product will have likely decomposed). It is important not to heat this reaction at 150 °C for too long because it will decompose. The reaction was monitored by LC–MS and usually contained a transesterification product in addition to the free acid product. After 10–15 min, the reaction was removed from the heating block and allowed to cool to room temperature. A small amount of water (∼1 mL) was added slowly (heat produced) followed by lithium hydroxide (140 mg, 5.85 mmol, 20 equiv). A small amount of methanol was added (<1 mL) to decrease viscosity to allow for stirring and to solubilize organics. The mixture was then loaded directly onto a 100 g gold reverse phase isco column eluting with 10% acetonitrile (0.1% TFA modifier) for 3 min, then 10 → 100% acetonitrile (0.1% TFA modifier) in water (0.1% TFA modifier) from minutes 3–17. The product elutes around 40–50% acetonitrile. The pure fractions were collected and concentrated <i>in vacuo</i> to provide 2-((2-chlorophenyl)(2-(1-methylpyrrolidin-2-yl)ethoxy)methyl)thieno[3,2-<i>b</i>]pyridine-7-carboxylic acid as a TFA salt. LC–MS retention time: <i>t</i><sub>1</sub> = 2.30 min; <i>t</i><sub>2</sub> = 2.73 min. LC–MS M + H = 431/433. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12–11.5 (br s, 1H), 8.61 (dd, <i>J</i> = 4.7, 0.5 Hz, 1H), 7.72–7.68 (m, 1H), 7.62 (d, <i>J</i> = 4.7 Hz, 1H), 7.49–7.46 (m, 1H), 7.44–7.34 (m, 3H), 6.14–6.12 (d, <i>J</i> = 2.5 Hz, 1H), 3.73–3.54 (m, 2H), 3.44 (br s, 1H), 3.17 (br s, 1H), 2.87 (br s, 1H), 2.71–2.70 (m, 3H), 2.27–2.21 (m, 2H), 1.91–1.79 (m, 3H), 1.74–1.64 (m, 1H).<img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01219/20181207/images/medium/jm-2018-01219j_0016.gif" alt="" id="dgr7" /></img></div></div><div id="sec4_6_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> Synthesis of Di<i>tert</i>-butyl (3-formylphenyl) Imidodicarbonate (<b>SI-3</b>)</h4><div class="NLM_p last">To a solution of <i>tert</i>-butyl (3-formylphenyl)carbamate (3.19 g, 14.42 mmol, 1 equiv) in 25 mL of <i>t</i>-BuOH, DMAP (0.176 g, 1.442 mmol, 0.1 equiv) and Boc-anhydride (3.78 g, 17.3 mmol, 1.2 equiv) were added. The reaction was then heated to 60 °C for 18 h and then an additional 0.5 equiv of Boc-anhydride was added. The reaction was heated at 60 °C for an additional 4 h. As monitored by LC–MS, the starting material was consumed. The reaction was then concentrated <i>in vacuo</i>, and the residue was diluted in a small amount of DCM and then purified on normal phase isco with a 120 g column eluting with 0 → 20% ethyl acetate in hexanes. LC–MS retention time: <i>t</i><sub>1</sub> = 3.57 min; LC–MS 2M + Na = 665. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 10.02 (s, 1H), 7.84–7.81 (m, 1H), 7.68–7.67 (m, 1H), 7.54 (t, <i>J</i> = 7.7 Hz, 1H), 7.44–7.41 (m, 1H), 1.43 (s, 18H).<img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01219/20181207/images/medium/jm-2018-01219j_0017.gif" alt="" id="dgr8" /></img></div></div><div id="sec4_6_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> Synthesis of <i>tert</i>-Butyl 2-((3-((di-<i>tert</i>-butoxycarbonyl)imido)phenyl)(hydroxy)methyl)thieno[3,2-<i>b</i>]pyridine-7-carboxylate (<b>SI-4</b>)</h4><div class="NLM_p last">The Grignard reagent was prepared as before and was reacted with di<i>tert</i>-butyl (3-formylphenyl) imidodicarbonate. The reaction was slowly warmed to room temperature and stirred for at least 1 h (as monitored by LC–MS). The reaction was then concentrated <i>in vacuo</i>, and the residue was taken up in ethyl acetate (20 mL). The organic layer was washed with water (1 × 10 mL) and brine (1 × 10 mL). The organic layer was separated, dried with sodium sulfate, filtered, and concentrated <i>in vacuo</i>. The residue was taken up in a small amount of DCM and purified on normal phase isco using a 40 g gold column eluting with 0 → 50% ethyl acetate in hexanes over 12 min (product elutes at around 50% ethyl acetate). The product elutes as a mixture of monoprotected and bis-protected aniline. The mixture of products was taken forward to the next reaction without further purification. LC–MS retention time: <i>t</i><sub>1</sub> = 3.98 min; LC–MS M + H = 557. <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 8.75 (d, <i>J</i> = 4.8 Hz, 1H), 7.75 (d, <i>J</i> = 4.8 Hz, 1H), 7.49–7.40 (m, 3H), 7.31 (t, <i>J</i> = 7.9 Hz, 1H), 7.17 (d, <i>J</i> = 7.6 Hz, 1H), 6.89 (s, 1H), 6.56 (s, 1H), 1.65 (s, 9H), 1.49 (d, <i>J</i> = 3.5 Hz, 18H).<img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01219/20181207/images/medium/jm-2018-01219j_0018.gif" alt="" id="dgr9" /></img></div></div><div id="sec4_6_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> Synthesis of 2-((3-aminophenyl)(2-(piperidin-1-yl)ethoxy)methyl)thieno[3,2-<i>b</i>]pyridine-7-carboxylic acid (<b>N70</b>)</h4><div class="NLM_p last"><i>tert</i>-Butyl 2-((3-((di-<i>tert</i>-butoxycarbonyl)imido)phenyl)(hydroxy)methyl)thieno[3,2-<i>b</i>]pyridine-7-carboxylate (0.9 g, 1.617 mmol, 1 equiv) was put into a 20 mL microwave vessel along with a small stir bar. 2-(Piperidin-1-yl)ethan-1-ol (0.785 mL, 5.91 mmol, 3 equiv) was added to the bottom of the microwave vessel, and then methanesulfonic acid (2.24 mL, 34.5 mmol, 17.5 equiv) was added. The vessel was then purged with a nitrogen stream, and the vessel was sealed with a microwave cap. The reaction was then placed in a preheated reaction block set to 120 °C, and the reaction was stirred for 4 h. The reaction was monitored by LC–MS and usually contained a transesterification product in addition to the free acid product. After 4 h, the reaction iswas removed from the heating block and allowed to cool to room temperature. A small amount of water (∼5 mL) was added slowly (heat produced) followed by lithium hydroxide (944 mg, 39.4 mmol, 20 equiv). A small amount of methanol was added (<3 mL) to decrease viscosity to allow for stirring and to solubilize organics. The mixture was then loaded directly onto a 100 g gold reverse phase isco column eluting with 10% acetonitrile (0.1% TFA modifier) for 3 min, then 10 → 100% acetonitrile (0.1% TFA modifier) in water (0.1% TFA modifier) to obtain the TFA salt. LC–MS retention time: <i>t</i><sub>1</sub> = 2.14 min; <i>t</i><sub>2</sub> = 2.81 min; LC–MS M + H = 412. <sup>1</sup>H NMR (400 MHz, DMSO-d6) δ 9.18 (br s, 1H), 8.82 (d, <i>J</i> = 4.8 Hz, 1H), 7.80 (d, <i>J</i> = 4.8 Hz, 1H), 7.57 (s, 1H), 7.10 (t, <i>J</i> = 7.8 Hz, 1H), 6.80–6.55 (m, 3H), 5.86 (s, 1H), 3.84–3.79 (m, 2H), 3.49–3.46 (m, 2H), 3.38–3.37 (m, 2H), 3.00–2.97 (m, 2H), 1.84–1.80 (m, 2H), 1.68–1.62 (m, 3H), 1.44–1.37 (m, 1H).<img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01219/20181207/images/medium/jm-2018-01219j_0019.gif" alt="" id="dgr10" /></img></div></div><div id="sec4_6_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> Synthesis of (<i>E</i>)-2-((3-(4-(Dimethylamino)but-2-enamido)phenyl)(2-(piperidin-1-yl)ethoxy)methyl)thieno[3,2-<i>b</i>]pyridine-7-carboxylic Acid (<b>N71</b>)</h4><div class="NLM_p last">To a solution of (<i>E</i>)-4-(dimethylamino)but-2-enoic acid hydrochloride (28.4 mg, 0.171 mmol, 1 equiv) in DMF (2 mL), DIPEA (0.090 mL, 0.514 mmol, 3 equiv) was added, followed by HATU (65.1 mg, 0.171 mmol, 1 equiv), and the mixture was stirred at room temperature for 10 min, then 2-((3-aminophenyl)(2-(piperidin-1-yl)ethoxy)methyl)thieno[3,2-<i>b</i>]pyridine-7-carboxylic acid·TFA (90 mg, 0.171 mmol, 1 equiv) was added and the reaction was stirred for 1 h at room temperature. The mixture was concentrated, dissolved in a small amount of DMSO, and purified on HPLC to obtain <b>N71</b> as the TFA salt. LC–MS retention time: <i>t</i><sub>1</sub> = 2.48 min; <i>t</i><sub>2</sub> = 2.77 min; LC–MS M + H = 523. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.40 (s, 1H), 9.82 (br s, 1H), 9.24 (br s, 1H), 8.83 (d, <i>J</i> = 4.8 Hz, 1H), 7.85 (s, 1H), 7.83–7.79 (d, <i>J</i> = 4.8 Hz 1H), 7.63 (s, 1H), 7.61–7.59 (m, 1H), 7.43–7.39 (t, <i>J</i> = 7.9 Hz, 1H), 7.31–7.29 (m, 1H), 6.75–6.68 (m, 1H), 6.44–6.41 (m, 1H), 6.05 (s, 1H), 3.92–3.79 (m, 5H), 3.52–3.46 (m, 2H), 3.05–2.95 (m, 3H), 2.78 (s, 6H), 1.84–1.81 (m, 2H), 1.72–1.62 (m, 2H), 1.44–1.36 (m, 2H).<img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01219/20181207/images/medium/jm-2018-01219j_0020.gif" alt="" id="dgr11" /></img></div></div><div id="sec4_6_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> Synthesis of Isopropyl 2-((3-Aminophenyl)(2-(piperidin-1-yl)ethoxy)methyl)thieno[3,2-<i>b</i>]pyridine-7-carboxylate (<b>SI-5</b>)</h4><div class="NLM_p last">To a solution of 2-((3-aminophenyl)(2-(piperidin-1-yl)ethoxy)methyl)thieno[3,2-<i>b</i>]pyridine-7-carboxylic acid (0.06g, 0.146 mmol, 1 equiv) in isopropanol (1 mL), TMS-Cl (0.186 mL, 1.458 mmol, 10 equiv) was added, and the reaction was heated to 100 °C in a microwave vial for 30 min. The reaction was then concentrated <i>in vacuo</i>, and the residue was purified on HPLC to provide the isopropyl 2-((3-aminophenyl)(2-(piperidin-1-yl)ethoxy)methyl)thieno[3,2-<i>b</i>]pyridine-7-carboxylate <b>SI-5</b> as the TFA salt. LC–MS retention time: <i>t</i><sub>1</sub> = 2.94 min; <i>t</i><sub>2</sub> = 3.65 min; LC–MS M + H = 454. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.18 (s, 1H), 8.83 (d, <i>J</i> = 4.8 Hz, 1H), 7.81 (d, <i>J</i> = 4.8 Hz, 1H), 7.57 (s, 1H), 7.12 (t, <i>J</i> = 7.7 Hz, 1H), 6.84–6.61 (m, 3H), 5.88 (s, 1H), 5.27–5.21 (m, 1H), 4.4–3.9 (br s, 2H), 3.87–3.76 (qt, <i>J</i> = 11.2, 4.9 Hz, 2H), 3.50–3.47 (m, 2H), 3.40–3.37 (m, 2H), 3.03–2.95 (m, 2H), 1.85–1.78 (m, 2H), 1.68 (m, 3H), 1.71–1.65 (m, 1H), 1.38 (dd, <i>J</i> = 6.3, 1.3 Hz, 6H).<img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01219/20181207/images/medium/jm-2018-01219j_0021.gif" alt="" id="dgr12" /></img></div></div><div id="sec4_6_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> Synthesis of Isopropyl (<i>E</i>)-2-((3-(4-(Dimethylamino)but-2-enamido)phenyl)(2-(piperidin-1-yl)ethoxy)methyl)thieno[3,2-<i>b</i>]pyridine-7-carboxylate (<b>N73</b>)</h4><div class="NLM_p last">To a solution of (<i>E</i>)-4-(dimethylamino)but-2-enoic acid hydrochloride (26.3 mg, 0.159 mmol, 1 equiv) in DMF (2 mL), DIPEA (0.055 mL, 0.317 mmol, 2 equiv) was added, followed by HATU (60.3 mg, 0.159 mmol, 1 equiv), and the mixture was stirred at room temperature for 10 min. Then isopropyl 2-((3-aminophenyl)(2-(piperidin-1-yl)ethoxy)methyl)thieno[3,2-<i>b</i>]pyridine-7-carboxylate (90 mg, 0.159 mmol, 1 equiv) was added, and the reaction was stirred for 1 h at room temperature. The mixture was concentrated, dissolved in a small amount of DMSO, and purified on HPLC to obtain <b>N73</b> as the TFA salt. LC–MS retention time: <i>t</i><sub>1</sub> = 2.91 min; <i>t</i><sub>2</sub> = 4.17 min; LC–MS M + H = 565; (M + 2H)/2 = 283. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.42 (s, 1H), 9.96 (br s, 1H), 9.35 (br s, 1H), 8.85 (d, <i>J</i> = 4.8 Hz, 1H), 7.88 (t, <i>J</i> = 1.9 Hz, 1H), 7.83 (d, <i>J</i> = 4.8 Hz, 1H), 7.60 (m, 2H), 7.41 (t, <i>J</i> = 7.9 Hz, 1H), 7.29 (d, <i>J</i> = 7.7 Hz, 1H), 6.73 (dq, <i>J</i> = 14.8, 7.7, 7.2 Hz, 1H), 6.43 (d, <i>J</i> = 15.3 Hz, 1H), 6.06 (s, 1H), 5.27–5.20 (m, 1H), 3.96–3.91 (m, 2H), 3.90–3.80 (m, 2H), 3.54–3.47 (m, 2H), 3.42–3.37 (m, 2H), 3.08–2.95 (m, 2H), 2.79 (s, 6H), 1.87–1.78 (m, 2H), 1.74–1.62 (m, 3H), 1.46–1.31 (m, 7H).</div></div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> Constructs</h3><div class="NLM_p last">The KDM5 family is unique among histone demethylases in that each member contains an atypical split catalytic domain with insertion of a DNA-binding ARID and histone-interacting PHD1 domain separating it into two segments. Deletion of internal ARID and PHD1 domains (ΔAP) generated KDM5A(1–739)ΔAP and KDM5B(1–755)ΔAP, used for enzyme activity studies, and KDM5A(1–588)ΔAP, used for compound binding assays and structural studies.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a></div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> Promega Succinate-Glo Demethylase Assay</h3><div class="NLM_p last">The inhibition of KDM5A(1–739)ΔAP by compounds <b>N49</b>, <b>N70</b>, and <b>N71</b> was obtained using the Promega Succinate-Glo kit. The demethylase reaction was conducted for 10 min with 0.8 μM KDM5A preincubated for 20 min with a 2=fold serial dilution of compound. The reaction was initiated by the addition of αKG (10, 100, or 1000 μM) and substrate peptide H3(1–24)K4me3 (60 μM) in the presence of 10 μM (NH<sub>4</sub>)<sub>2</sub>Fe(SO<sub>4</sub>)<sub>2</sub>, 100 μM ascorbic acid, 50 mM HEPES pH 8.0, and 5% DMSO at ambient temperature (<i>n</i> = 3). The reaction was terminated by the addition of equal volume of Succinate Detection Reagent I (SDI), supplied by the manufacture, followed by 1 h incubation. After the incubation period with SDI the second reagent SDII was added to a final ratio of 1:2 (SDI/SDII). A 15 μL aliquot of the final solution was loaded into each well of a 384 low volume white plate. Data was collected using a BioTek Synergy 4, and IC<sub>50</sub> values were fitted using a four-parameter model in GraphPad Prism 7.</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> PAMPA Permeability Assay</h3><div class="NLM_p last">The stirring double-sink PAMPA method patented by pION Inc. (Billerica, MA) was employed to determine the permeability of compounds via PAMPA passive diffusion. The PAMPA lipid membrane consisted of an artificial membrane of a proprietary lipid mixture and dodecane (Pion Inc.), optimized to predict gastrointestinal tract (GIT) passive diffusion permeability, and was immobilized on a plastic matrix of a 96-well “donor” filter plate placed above a 96-well “acceptor” plate. A pH 7.4 solution was used in both donor and acceptor wells. The test articles, stocked in 10 mM DMSO solutions, were diluted to 0.05 mM in aqueous buffer (pH 7.4), and the concentration of DMSO was 0.5% in the final solution. During the 30 min permeation period at room temperature, the test samples in the donor compartment were stirred using the Gutbox technology (Pion Inc.) to reduce the unstirred water layer. The test article concentrations in the donor and acceptor compartments were measured using an UV plate reader (Nano Quant, Infinite 200 PRO, Tecan Inc., Mannedorf, Switzerland). Permeability calculations were performed using Pion Inc. software and were expressed in units of 10<sup>–6</sup> cm s<sup>–1</sup>. Compounds with values >100 are considered to have high permeability, compounds with values <100 but >10 are considered moderate permeability, and compounds with values <10 are considered to have low permeability.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i33"><a href="/doi/suppl/10.1021/acs.jmedchem.8b01219" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_12850" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_12850" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.8b01219" class="ext-link">10.1021/acs.jmedchem.8b01219</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01219/suppl_file/jm8b01219_si_001.csv" class="ext-link">CSV</a>)</p></li><li><p class="inline">Figures S1–S5 and Table S1 (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01219/suppl_file/jm8b01219_si_002.pdf" class="ext-link">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b01219/suppl_file/jm8b01219_si_001.csv">jm8b01219_si_001.csv (0.34 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b01219/suppl_file/jm8b01219_si_002.pdf">jm8b01219_si_002.pdf (4.18 MB)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The atomic coordinates and structure factors of KDM5A in complex with <b>GSK-J1</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6DQ4">6DQ4</a>), <b>N43</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6DQ5">6DQ5</a>), <b>N44</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6DQ6">6DQ6</a>), <b>N49</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6DQ8">6DQ8</a>), <b>N70</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6DQA">6DQA</a>), and <b>N71</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6DQB">6DQB</a>) have been deposited in the Protein Data Bank.</p><div class="testing" data-doi="10.1021/acs.jmedchem.8b01219" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_32267" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_32267" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xiaodong Cheng</span> - <span class="hlFld-Affiliation affiliation">Department
of Molecular and Cellular Oncology, The
University of Texas MD Anderson Cancer Center, Houston, Texas 77030, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Biochemistry, Emory University School
of Medicine, Atlanta, Georgia 30322, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-6967-6362" title="Orcid link">http://orcid.org/0000-0002-6967-6362</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#f8809b909d969fcdb8959c99969c9d8a8b9796d6978a9f"><span class="__cf_email__" data-cfemail="79011a111c171e4c39141d18171d1c0b0a161757160b1e">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">John R. Horton</span> - <span class="hlFld-Affiliation affiliation">Department
of Molecular and Cellular Oncology, The
University of Texas MD Anderson Cancer Center, Houston, Texas 77030, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Biochemistry, Emory University School
of Medicine, Atlanta, Georgia 30322, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Clayton B. Woodcock</span> - <span class="hlFld-Affiliation affiliation">Department
of Molecular and Cellular Oncology, The
University of Texas MD Anderson Cancer Center, Houston, Texas 77030, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Qin Chen</span> - <span class="hlFld-Affiliation affiliation">Department
of Molecular and Cellular Oncology, The
University of Texas MD Anderson Cancer Center, Houston, Texas 77030, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xu Liu</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry, Emory University School
of Medicine, Atlanta, Georgia 30322, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xing Zhang</span> - <span class="hlFld-Affiliation affiliation">Department
of Molecular and Cellular Oncology, The
University of Texas MD Anderson Cancer Center, Houston, Texas 77030, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">John Shanks</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry, Emory University School
of Medicine, Atlanta, Georgia 30322, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ganesha Rai</span> - <span class="hlFld-Affiliation affiliation">National
Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-9763-9641" title="Orcid link">http://orcid.org/0000-0001-9763-9641</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Bryan T. Mott</span> - <span class="hlFld-Affiliation affiliation">National
Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Daniel J. Jansen</span> - <span class="hlFld-Affiliation affiliation">National
Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Stephen C. Kales</span> - <span class="hlFld-Affiliation affiliation">National
Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mark J. Henderson</span> - <span class="hlFld-Affiliation affiliation">National
Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Matthew Cyr</span> - <span class="hlFld-Affiliation affiliation">National
Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Katherine Pohida</span> - <span class="hlFld-Affiliation affiliation">National
Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xin Hu</span> - <span class="hlFld-Affiliation affiliation">National
Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Pranav Shah</span> - <span class="hlFld-Affiliation affiliation">National
Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xin Xu</span> - <span class="hlFld-Affiliation affiliation">National
Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ajit Jadhav</span> - <span class="hlFld-Affiliation affiliation">National
Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">David J. Maloney</span> - <span class="hlFld-Affiliation affiliation">National
Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Matthew D. Hall</span> - <span class="hlFld-Affiliation affiliation">National
Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-5073-442X" title="Orcid link">http://orcid.org/0000-0002-5073-442X</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Anton Simeonov</span> - <span class="hlFld-Affiliation affiliation">National
Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Haian Fu</span> - <span class="hlFld-Affiliation affiliation">Departments of Pharmacology, 
    Hematology and Medical
Oncology, 
    Emory
Chemical Biology
Discovery Center, 
    The Winship Cancer Institute, 
    , , , , and , Emory University, Atlanta, Georgia 30322, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Paula M. Vertino</span> - <span class="hlFld-Affiliation affiliation">The Winship Cancer Institute, 
    Department of Radiation Oncology, 
    , , , , and , Emory University, Atlanta, Georgia 30322, United States</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>J.R.H. performed all crystallographic experiments. C.B.W. performed inhibition and steady state studies using bioluminescence. Q.C. performed Western blot analysis and cell killing assays. X.L. generated ITC and FDH data and chemical drawings. J.R.S. performed protein purification. X.Z. participated in experimental design and discussion throughout. G.R., B.T.M., D.J.J., and K.P. synthesized compounds. S.C.K. performed AlphaLISA experiments. M.C. and M.J.H. performed immunocytochemistry and IncuCyte experiments. X.H. performed data processing. P.S. and. X.X. performed culture media stability assays. A.S., A.J., and M.D.H. designed experiments and discussed the study. G.R., D.J.J., B.T.M., D.J.M. planned synthesis of compounds. H.F. participated in organization and discussion of the study. J.R.H., P.M.V., and X.C. wrote the initial manuscript, and all were involved in analyzing data and discussion. The X-ray crystallographic component of the study was initially carried out in Emory University (as dated in Table S1), the structural analyses and the manuscript were completed at the University of Texas MD Anderson Cancer Center.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i36">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_52861" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_52861" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This work was supported in part with Federal funds from the National Cancer Institute (NCI), National Institutes of Health (NIH), under NCI Chemical Biology Consortium Contract No. HHSN261200800001E (to H.F.), NIH grant GM114306 and Cancer Prevention Research Institute of Texas (CPRIT) grant RR160029 (to X.C.). The mass spectrometry experiments were performed by David Hawke of the Proteomics and Metabolomics Facility (at the University of Texas MD Anderson Cancer Center), supported in part by CPRIT (RP130397) and NIH (1S10OD012304-01 and P30CA016672). The content of this publication does not necessarily reflect the views or policies of the Department of Health and Human Services, nor does mention of trade names, commercial products, or organizations imply endorsement by the U.S. Government. The Department of Biochemistry of Emory University School of Medicine and CPRIT supported the use of the Southeast Regional Collaborative Access Team (SERCAT) synchrotron beamlines at the Advanced Photon Source of Argonne National Laboratory. We thank Qin Yan of Yale University, Anthony Welch and Dane Liston of NCI, and Andrew Flint of Leidos Biomedical Research, Inc. (which operates the Frederick National Laboratory for Cancer Research) for discussion throughout this project.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">αKG</td><td class="NLM_def"><p class="first last">α-ketoglutarate</p></td></tr><tr><td class="NLM_term">KDM5A</td><td class="NLM_def"><p class="first last">histone lysine demethylase 5A</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i38">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_41570" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_41570" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 53 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gasser, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, E.</span></span> <span> </span><span class="NLM_article-title">Epigenetics and disease: pharmaceutical opportunities</span>. <i>Appl. Clin. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">67</span>, <span class="refDoi"> DOI: 10.2147/TACG.S11639</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=10.2147%2FTACG.S11639" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=67&author=S.+M.+Gasserauthor=E.+Li&title=Epigenetics+and+disease%3A+pharmaceutical+opportunities&doi=10.2147%2FTACG.S11639"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.2147%2FTACG.S11639&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2147%252FTACG.S11639%26sid%3Dliteratum%253Aachs%26aulast%3DGasser%26aufirst%3DS.%2BM.%26aulast%3DLi%26aufirst%3DE.%26atitle%3DEpigenetics%2520and%2520disease%253A%2520pharmaceutical%2520opportunities%26jtitle%3DAppl.%2520Clin.%2520Genet.%26date%3D2011%26volume%3D67%26spage%3D67%26doi%3D10.2147%2FTACG.S11639" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Black, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Rechem, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whetstine, J. R.</span></span> <span> </span><span class="NLM_article-title">Histone lysine methylation dynamics: establishment, regulation, and biological impact</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">491</span>– <span class="NLM_lpage">507</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2012.11.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=10.1016%2Fj.molcel.2012.11.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=23200123" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=1%3ACAS%3A528%3ADC%252BC38XhslKqtLvM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2012&pages=491-507&author=J.+C.+Blackauthor=C.+Van+Rechemauthor=J.+R.+Whetstine&title=Histone+lysine+methylation+dynamics%3A+establishment%2C+regulation%2C+and+biological+impact&doi=10.1016%2Fj.molcel.2012.11.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Histone Lysine Methylation Dynamics: Establishment, Regulation, and Biological Impact</span></div><div class="casAuthors">Black, Joshua C.; Van Rechem, Capucine; Whetstine, Johnathan R.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">491-507</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">A review.  Histone lysine methylation has emerged as a crit. player in the regulation of gene expression, cell cycle, genome stability, and nuclear architecture.  Over the past decade, a tremendous amt. of progress has led to the characterization of Me modifications and the lysine methyltransferases (KMTs) and lysine demethylases (KDMs) that regulate them.  Here, we review the discovery and characterization of the KMTs and KDMs and the Me modifications they regulate.  We discuss the localization of the KMTs and KDMs as well as the distribution of lysine methylation throughout the genome.  We highlight how these data have shaped our view of lysine methylation as a key determinant of complex chromatin states.  Finally, we discuss the regulation of KMTs and KDMs by proteasomal degrdn., posttranscriptional mechanisms, and metabolic status.  We propose key questions for the field and highlight areas that we predict will yield exciting discoveries in the years to come.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbnBk0Al-Ce7Vg90H21EOLACvtfcHk0ljOGWMu2Ty5WA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhslKqtLvM&md5=b9a8e9c29c630592f5413f7e36974735</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2012.11.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2012.11.006%26sid%3Dliteratum%253Aachs%26aulast%3DBlack%26aufirst%3DJ.%2BC.%26aulast%3DVan%2BRechem%26aufirst%3DC.%26aulast%3DWhetstine%26aufirst%3DJ.%2BR.%26atitle%3DHistone%2520lysine%2520methylation%2520dynamics%253A%2520establishment%252C%2520regulation%252C%2520and%2520biological%2520impact%26jtitle%3DMol.%2520Cell%26date%3D2012%26volume%3D48%26spage%3D491%26epage%3D507%26doi%3D10.1016%2Fj.molcel.2012.11.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, K. N.</span></span> <span> </span><span class="NLM_article-title">Association between H3K4 methylation and cancer prognosis: A meta-analysis</span>. <i>Thorac. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">794</span>– <span class="NLM_lpage">799</span>, <span class="refDoi"> DOI: 10.1111/1759-7714.12647</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=10.1111%2F1759-7714.12647" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=29737623" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1ejsLjP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=794-799&author=S.+Liauthor=L.+Shenauthor=K.+N.+Chen&title=Association+between+H3K4+methylation+and+cancer+prognosis%3A+A+meta-analysis&doi=10.1111%2F1759-7714.12647"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Association between H3K4 methylation and cancer prognosis: A meta-analysis</span></div><div class="casAuthors">Li, Simin; Shen, Luyan; Chen, Ke-Neng</div><div class="citationInfo"><span class="NLM_cas:title">Thoracic Cancer</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">794-799</span>CODEN:
                <span class="NLM_cas:coden">TCHAEQ</span>;
        ISSN:<span class="NLM_cas:issn">1759-7706</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons Singapore Pte. Ltd.</span>)
        </div><div class="casAbstract">Background : Histone H3 lysine 4 methylation (H3K4 methylation), including mono-methylation (H3K4me1), di-methylation (H3K4me2), or tri-methylation (H3K4me3), is one of the epigenetic modifications to histone proteins, which are related to the transcriptional activation of genes.  H3K4 methylation has both tumor inhibiting and promoting effects, and the prognostic value of H3K4 methylation in cancer remains controversial.  Therefore, we performed a systematic review and meta-anal. to examine the assocn. between H3K4 methylation and cancer prognosis.  Methods : A comprehensive search of PubMed, Web of Science, ScienceDirect, Embase, and Ovid databases was conducted to identify studies investigating the assocn. between H3K4 methylation and prognosis of patients with malignant tumors.  The data and characteristics of each study were extd., and the hazard ratio (HR) at a 95% confidence interval (CI) was calcd. to est. the effect.  Results : A total of 1474 patients in 10 studies were enrolled in this meta-anal.  The pooled HR of 1.52 (95% CI 1.02-2.26) indicated that patients with a lower level of H3K4me2 expression were expected to have shorter overall survival, while the pooled HR of 0.45 (95% CI 0.27-0.74) indicated that patients with a lower level of H3K4me3 expression were expected to have longer overall survival.  Conclusion : This meta-anal. indicates that increased H3K4me3 expression and decreased H3K4me2 expression might be predictive factors of poor prognosis in cancer.  Further large cohort studies are needed to confirm these findings.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoTCdN4iOwtxbVg90H21EOLACvtfcHk0ljOGWMu2Ty5WA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1ejsLjP&md5=3c7ebe189d60416fb77d5dc436539c33</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1111%2F1759-7714.12647&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252F1759-7714.12647%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DS.%26aulast%3DShen%26aufirst%3DL.%26aulast%3DChen%26aufirst%3DK.%2BN.%26atitle%3DAssociation%2520between%2520H3K4%2520methylation%2520and%2520cancer%2520prognosis%253A%2520A%2520meta-analysis%26jtitle%3DThorac.%2520Cancer%26date%3D2018%26volume%3D9%26spage%3D794%26epage%3D799%26doi%3D10.1111%2F1759-7714.12647" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulligan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whetstine, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casero, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span> <span> </span><span class="NLM_article-title">Histone demethylation mediated by the nuclear amine oxidase homolog LSD1</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>119</i></span>,  <span class="NLM_fpage">941</span>– <span class="NLM_lpage">53</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2004.12.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=10.1016%2Fj.cell.2004.12.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=15620353" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=1%3ACAS%3A528%3ADC%252BD2MXltlSitg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=2004&pages=941-53&author=Y.+Shiauthor=F.+Lanauthor=C.+Matsonauthor=P.+Mulliganauthor=J.+R.+Whetstineauthor=P.+A.+Coleauthor=R.+A.+Caseroauthor=Y.+Shi&title=Histone+demethylation+mediated+by+the+nuclear+amine+oxidase+homolog+LSD1&doi=10.1016%2Fj.cell.2004.12.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Histone demethylation mediated by the nuclear amine oxidase homolog LSD1</span></div><div class="casAuthors">Shi, Yujiang; Lan, Fei; Matson, Caitlin; Mulligan, Peter; Whetstine, Johnathan R.; Cole, Philip A.; Casero, Robert A.; Shi, Yang</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">119</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">941-953</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Posttranslational modifications of histone N-terminal tails impact chromatin structure and gene transcription.  While the extent of histone acetylation is detd. by both acetyltransferases and deacetylases, it has been unclear whether histone methylation is also regulated by enzymes with opposing activities.  Here, we provide evidence that LSD1 (KIAA0601), a nuclear homolog of amine oxidases, functions as a histone demethylase and transcriptional corepressor.  LSD1 specifically demethylates histone H3 lysine 4, which is linked to active transcription.  Lysine demethylation occurs via an oxidn. reaction that generates formaldehyde.  Importantly, RNAi inhibition of LSD1 causes an increase in H3 lysine 4 methylation and concomitant derepression of target genes, suggesting that LSD1 represses transcription via histone demethylation.  The results thus identify a histone demethylase conserved from S. pombe to human and reveal dynamic regulation of histone methylation by both histone methylases and demethylases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXHr-8kwGdXrVg90H21EOLACvtfcHk0liqach88oJn8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXltlSitg%253D%253D&md5=3474569204255e90a832667ba654748b</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2004.12.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2004.12.012%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DY.%26aulast%3DLan%26aufirst%3DF.%26aulast%3DMatson%26aufirst%3DC.%26aulast%3DMulligan%26aufirst%3DP.%26aulast%3DWhetstine%26aufirst%3DJ.%2BR.%26aulast%3DCole%26aufirst%3DP.%2BA.%26aulast%3DCasero%26aufirst%3DR.%2BA.%26aulast%3DShi%26aufirst%3DY.%26atitle%3DHistone%2520demethylation%2520mediated%2520by%2520the%2520nuclear%2520amine%2520oxidase%2520homolog%2520LSD1%26jtitle%3DCell%26date%3D2004%26volume%3D119%26spage%3D941%26epage%3D53%26doi%3D10.1016%2Fj.cell.2004.12.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsukada, Y.</span></span> <span> </span><span class="NLM_article-title">The discovery of histone demethylases</span>. <i>Cold Spring Harbor Perspect. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">a022335</span>, <span class="refDoi"> DOI: 10.1101/cshperspect.a022335</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=10.1101%2Fcshperspect.a022335" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFCiurnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2013&pages=a022335&author=Y.+G.+Shiauthor=Y.+Tsukada&title=The+discovery+of+histone+demethylases&doi=10.1101%2Fcshperspect.a022335"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">The discovery of histone demethylases [Erratum to document cited in CA160:002399]</span></div><div class="casAuthors">Shi, Yujiang Geno; Tsukada, Yuichi</div><div class="citationInfo"><span class="NLM_cas:title">Cold Spring Harbor Perspectives in Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">a022335/1</span>CODEN:
                <span class="NLM_cas:coden">CSHPEU</span>;
        ISSN:<span class="NLM_cas:issn">1943-0264</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">A review.  The authors felt the published article did not provide a sufficiently accurate account of the discovery of histone demethylases; the description has been modified in the PDF and HTML versions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOorDbV4YtbrVg90H21EOLACvtfcHk0liqach88oJn8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFCiurnF&md5=c935849fd498b3a27ca906dec4d01721</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1101%2Fcshperspect.a022335&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fcshperspect.a022335%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DY.%2BG.%26aulast%3DTsukada%26aufirst%3DY.%26atitle%3DThe%2520discovery%2520of%2520histone%2520demethylases%26jtitle%3DCold%2520Spring%2520Harbor%2520Perspect.%2520Biol.%26date%3D2013%26volume%3D5%26spage%3Da022335%26doi%3D10.1101%2Fcshperspect.a022335" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jambhekar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anastas, J. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span> <span> </span><span class="NLM_article-title">Histone lysine demethylase inhibitors</span>. <i>Cold Spring Harbor Perspect. Med.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">a026484</span>, <span class="refDoi"> DOI: 10.1101/cshperspect.a026484</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=10.1101%2Fcshperspect.a026484" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=28049654" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtFKms7vF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=a026484&author=A.+Jambhekarauthor=J.+N.+Anastasauthor=Y.+Shi&title=Histone+lysine+demethylase+inhibitors&doi=10.1101%2Fcshperspect.a026484"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Histone lysine demethylase inhibitors</span></div><div class="casAuthors">Jambhekar, Ashwini; Anastas, Jamie N.; Shi, Yang</div><div class="citationInfo"><span class="NLM_cas:title">Cold Spring Harbor Perspectives in Medicine</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">a026484/1-a026484/26</span>CODEN:
                <span class="NLM_cas:coden">CSHPFV</span>;
        ISSN:<span class="NLM_cas:issn">2157-1422</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">The dynamic regulation of covalent modifications to histones is essential for maintaining genomic integrity and cell identity and is often compromised in cancer.  Aberrant expression of histone lysine demethylases has been documented in many types of blood and solid tumors, and thus demethylases represent promising therapeutic targets.  Recent advances in high-throughput chem. screening, structure-based drug design, and structure-activity relationship studies have improved both the specificity and the in vivo efficacy of demethylase inhibitors.  This review will briefly outline the connection between demethylases and cancer and will provide a comprehensive overview of the structure, specificity, and utility of currently available demethylase inhibitors.  To date, a select group of demethylase inhibitors is being evaluated in clin. trials, and addnl. compds. may soon follow from the bench to the bedside.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoqMUZw4KjjDLVg90H21EOLACvtfcHk0liqach88oJn8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtFKms7vF&md5=22ba67d04750ab8d410baee59a64d7e6</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1101%2Fcshperspect.a026484&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fcshperspect.a026484%26sid%3Dliteratum%253Aachs%26aulast%3DJambhekar%26aufirst%3DA.%26aulast%3DAnastas%26aufirst%3DJ.%2BN.%26aulast%3DShi%26aufirst%3DY.%26atitle%3DHistone%2520lysine%2520demethylase%2520inhibitors%26jtitle%3DCold%2520Spring%2520Harbor%2520Perspect.%2520Med.%26date%3D2017%26volume%3D7%26spage%3Da026484%26doi%3D10.1101%2Fcshperspect.a026484" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H. M.</span></span> <span> </span><span class="NLM_article-title">A systematic review of histone lysine-specific demethylase 1 and Its inhibitors</span>. <i>Med. Res. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">1032</span>– <span class="NLM_lpage">1071</span>, <span class="refDoi"> DOI: 10.1002/med.21350</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=10.1002%2Fmed.21350" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=25990136" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1yqtrfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2015&pages=1032-1071&author=Y.+C.+Zhengauthor=J.+Maauthor=Z.+Wangauthor=J.+Liauthor=B.+Jiangauthor=W.+Zhouauthor=X.+Shiauthor=X.+Wangauthor=W.+Zhaoauthor=H.+M.+Liu&title=A+systematic+review+of+histone+lysine-specific+demethylase+1+and+Its+inhibitors&doi=10.1002%2Fmed.21350"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">A Systematic Review of Histone Lysine-Specific Demethylase 1 and Its Inhibitors</span></div><div class="casAuthors">Zheng, Yi-Chao; Ma, Jinlian; Wang, Zhiru; Li, Jinfeng; Jiang, Bailing; Zhou, Wenjuan; Shi, Xiaojing; Wang, Xixin; Zhao, Wen; Liu, Hong-Min</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Research Reviews</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1032-1071</span>CODEN:
                <span class="NLM_cas:coden">MRREDD</span>;
        ISSN:<span class="NLM_cas:issn">0198-6325</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">Histone lysine-specific demethylase 1 (LSD1) is the first discovered and reported histone demethylase by Dr. Shi Yang's group in 2004.  It is classified as a member of amine oxidase superfamily, the common feature of which is using the FAD (FAD) as its cofactor.  Since it is located in cell nucleus and acts as a histone methylation eraser, LSD1 specifically removes mono- or dimethylated histone H3 lysine 4 (H3K4) and H3 lysine 9 (H3K9) through formaldehyde-generating oxidn.  It has been indicated that LSD1 and its downstream targets are involved in a wide range of biol. courses, including embryonic development and tumor-cell growth and metastasis.  LSD1 has been reported to be overexpressed in variety of tumors.  Inactivating LSD1 or downregulating its expression inhibits cancer-cell development.  LSD1 targeting inhibitors may represent a new insight in anticancer drug discovery.  This review summarizes recent studies about LSD1 and mainly focuses on the basic physiol. function of LSD1 and its involved mechanisms in pathophysiol. conditions, as well as the development of LSD1 inhibitors as potential anticancer therapeutic agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7JXHHRI-QAbVg90H21EOLACvtfcHk0lgCoIMY2Kf4sg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1yqtrfN&md5=43b875856c2fe9f7a02857e7d6de73e6</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1002%2Fmed.21350&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmed.21350%26sid%3Dliteratum%253Aachs%26aulast%3DZheng%26aufirst%3DY.%2BC.%26aulast%3DMa%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DJiang%26aufirst%3DB.%26aulast%3DZhou%26aufirst%3DW.%26aulast%3DShi%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DZhao%26aufirst%3DW.%26aulast%3DLiu%26aufirst%3DH.%2BM.%26atitle%3DA%2520systematic%2520review%2520of%2520histone%2520lysine-specific%2520demethylase%25201%2520and%2520Its%2520inhibitors%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2015%26volume%3D35%26spage%3D1032%26epage%3D1071%26doi%3D10.1002%2Fmed.21350" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Christensen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agger, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cloos, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pasini, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rose, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sennels, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rappsilber, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salcini, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helin, K.</span></span> <span> </span><span class="NLM_article-title">RBP2 belongs to a family of demethylases, specific for tri-and dimethylated lysine 4 on histone 3</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>128</i></span>,  <span class="NLM_fpage">1063</span>– <span class="NLM_lpage">1076</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2007.02.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=10.1016%2Fj.cell.2007.02.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=17320161" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=1%3ACAS%3A528%3ADC%252BD2sXkslGms78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2007&pages=1063-1076&author=J.+Christensenauthor=K.+Aggerauthor=P.+A.+Cloosauthor=D.+Pasiniauthor=S.+Roseauthor=L.+Sennelsauthor=J.+Rappsilberauthor=K.+H.+Hansenauthor=A.+E.+Salciniauthor=K.+Helin&title=RBP2+belongs+to+a+family+of+demethylases%2C+specific+for+tri-and+dimethylated+lysine+4+on+histone+3&doi=10.1016%2Fj.cell.2007.02.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">RBP2 belongs to a family of demethylases, specific for triand dimethylated lysine 4 on histone 3</span></div><div class="casAuthors">Christensen, Jesper; Agger, Karl; Cloos, Paul A. C.; Pasini, Diego; Rose, Simon; Sennels, Lau; Rappsilber, Juri; Hansen, Klaus H.; Salcini, Anna Elisabetta; Helin, Kristian</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">128</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1063-1076</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Methylation of histones has been regarded as a stable modification defining the epigenetic program of the cell, which regulates chromatin structure and transcription.  However, the recent discovery of histone demethylases has challenged the stable nature of histone methylation.  Here we demonstrate that the JARID1 proteins RBP2, PLU1, and SMCX are histone demethylases specific for di- and trimethylated histone 3 lysine 4 (H3K4).  Consistent with a role for the JARID1 Drosophila homolog Lid in regulating expression of homeotic genes during development, we show that RBP2 is displaced from Hox genes during embryonic stem (ES) cell differentiation correlating with an increase of their H3K4me3 levels and expression.  Furthermore, we show that mutation or RNAi depletion of the C. elegans JARID1 homolog rbr-2 leads to increased levels of H3K4me3 during larval development and defects in vulva formation.  Taken together, these results suggest that H3K4me3/me2 demethylation regulated by the JARID1 family plays an important role during development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqoiLQh_7D2g7Vg90H21EOLACvtfcHk0lgCoIMY2Kf4sg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXkslGms78%253D&md5=c1f04b597253ddc6d89b62bf0b99b13c</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2007.02.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2007.02.003%26sid%3Dliteratum%253Aachs%26aulast%3DChristensen%26aufirst%3DJ.%26aulast%3DAgger%26aufirst%3DK.%26aulast%3DCloos%26aufirst%3DP.%2BA.%26aulast%3DPasini%26aufirst%3DD.%26aulast%3DRose%26aufirst%3DS.%26aulast%3DSennels%26aufirst%3DL.%26aulast%3DRappsilber%26aufirst%3DJ.%26aulast%3DHansen%26aufirst%3DK.%2BH.%26aulast%3DSalcini%26aufirst%3DA.%2BE.%26aulast%3DHelin%26aufirst%3DK.%26atitle%3DRBP2%2520belongs%2520to%2520a%2520family%2520of%2520demethylases%252C%2520specific%2520for%2520tri-and%2520dimethylated%2520lysine%25204%2520on%2520histone%25203%26jtitle%3DCell%26date%3D2007%26volume%3D128%26spage%3D1063%26epage%3D1076%26doi%3D10.1016%2Fj.cell.2007.02.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Blair, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sayegh, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, Q.</span></span> <span> </span><span class="NLM_article-title">Epigenetic regulation by lysine demethylase 5 (KDM5) enzymes in cancer</span>. <i>Cancers</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">1383</span>– <span class="NLM_lpage">1404</span>, <span class="refDoi"> DOI: 10.3390/cancers3011383</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=10.3390%2Fcancers3011383" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=21544224" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktVWiurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2011&pages=1383-1404&author=L.+P.+Blairauthor=J.+Caoauthor=M.+R.+Zouauthor=J.+Sayeghauthor=Q.+Yan&title=Epigenetic+regulation+by+lysine+demethylase+5+%28KDM5%29+enzymes+in+cancer&doi=10.3390%2Fcancers3011383"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Epigenetic regulation by lysine demethylase 5 (KDM5) enzymes in cancer</span></div><div class="casAuthors">Blair, Lauren P.; Cao, Jian; Zou, Mike Ran; Sayegh, Joyce; Yan, Qin</div><div class="citationInfo"><span class="NLM_cas:title">Cancers</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1383-1404</span>CODEN:
                <span class="NLM_cas:coden">CANCCT</span>;
        ISSN:<span class="NLM_cas:issn">2072-6694</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">A review.  Similar to genetic alterations, epigenetic aberrations contribute significantly to tumor initiation and progression.  In many cases, these changes are caused by activation or inactivation of the regulators that maintain epigenetic states.  Here we review our current knowledge on the KDM5/JARID1 family of histone demethylases.  This family of enzymes contains a JmjC domain and is capable of removing tri- and di- Me marks from lysine 4 on histone H3.  Among these proteins, RBP2 mediates drug resistance while JARID1B is required for melanoma maintenance.  Preclin. studies suggest inhibition of these enzymes can suppress tumorigenesis and provide strong rationale for development of their inhibitors for use in cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8lTt6GjrOJrVg90H21EOLACvtfcHk0lgCoIMY2Kf4sg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktVWiurg%253D&md5=426fa61d08aa818b45781b57e1aba5f3</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.3390%2Fcancers3011383&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fcancers3011383%26sid%3Dliteratum%253Aachs%26aulast%3DBlair%26aufirst%3DL.%2BP.%26aulast%3DCao%26aufirst%3DJ.%26aulast%3DZou%26aufirst%3DM.%2BR.%26aulast%3DSayegh%26aufirst%3DJ.%26aulast%3DYan%26aufirst%3DQ.%26atitle%3DEpigenetic%2520regulation%2520by%2520lysine%2520demethylase%25205%2520%2528KDM5%2529%2520enzymes%2520in%2520cancer%26jtitle%3DCancers%26date%3D2011%26volume%3D3%26spage%3D1383%26epage%3D1404%26doi%3D10.3390%2Fcancers3011383" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rasmussen, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Staller, P.</span></span> <span> </span><span class="NLM_article-title">The KDM5 family of histone demethylases as targets in oncology drug discovery</span>. <i>Epigenomics</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">277</span>– <span class="NLM_lpage">286</span>, <span class="refDoi"> DOI: 10.2217/epi.14.14</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=10.2217%2Fepi.14.14" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=25111482" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtlans77K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2014&pages=277-286&author=P.+B.+Rasmussenauthor=P.+Staller&title=The+KDM5+family+of+histone+demethylases+as+targets+in+oncology+drug+discovery&doi=10.2217%2Fepi.14.14"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">The KDM5 family of histone demethylases as targets in oncology drug discovery</span></div><div class="casAuthors">Rasmussen, Peter Birk; Staller, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Epigenomics</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">277-286</span>CODEN:
                <span class="NLM_cas:coden">EPIGC7</span>;
        ISSN:<span class="NLM_cas:issn">1750-192X</span>.
    
            (<span class="NLM_cas:orgname">Future Medicine Ltd.</span>)
        </div><div class="casAbstract">A review.  There is growing evidence for a causal role of the KDM5 family of histone demethylases in human cancer.  In particular, KDM5A (JARID1A/RBP2) and KDM5B (JARID1B/PLU1) contribute to cancer cell proliferation, reduce the expression of tumor suppressor genes, promote the development of drug tolerance and maintain tumor-initiating cells.  KDM5 enzymes remove tri- and di-methylations of lysine 4 of histone H3 - modifications that occur at the start site of transcription in actively transcribed genes.  However, the importance of the histone demethylase activity of KDM5 proteins for cancer cells has not been resolved so far.  The currently available approaches suppress or remove the targeted proteins and thereby affect their putative functions as structural components and recruitment factors for other chromatin-assocd. proteins.  Therefore, the development of specific enzymic inhibitors for KDM5 will promote our understanding of the biol. role of their catalytic activity and yield potential novel anticancer therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPJyGCVkwViLVg90H21EOLACvtfcHk0lgLKFD_H76FuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtlans77K&md5=0b8dc67f4fde6ce24457dce092dbdd83</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.2217%2Fepi.14.14&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2217%252Fepi.14.14%26sid%3Dliteratum%253Aachs%26aulast%3DRasmussen%26aufirst%3DP.%2BB.%26aulast%3DStaller%26aufirst%3DP.%26atitle%3DThe%2520KDM5%2520family%2520of%2520histone%2520demethylases%2520as%2520targets%2520in%2520oncology%2520drug%2520discovery%26jtitle%3DEpigenomics%26date%3D2014%26volume%3D6%26spage%3D277%26epage%3D286%26doi%3D10.2217%2Fepi.14.14" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Harmeyer, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Facompre, N. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herlyn, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Basu, D.</span></span> <span> </span><span class="NLM_article-title">JARID1 histone demethylases: emerging targets in cancer</span>. <i>Trends Cancer</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">713</span>– <span class="NLM_lpage">725</span>, <span class="refDoi"> DOI: 10.1016/j.trecan.2017.08.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=10.1016%2Fj.trecan.2017.08.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=28958389" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=1%3ACAS%3A528%3ADC%252BC1MXksVyitg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2017&pages=713-725&author=K.+M.+Harmeyerauthor=N.+D.+Facompreauthor=M.+Herlynauthor=D.+Basu&title=JARID1+histone+demethylases%3A+emerging+targets+in+cancer&doi=10.1016%2Fj.trecan.2017.08.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">JARID1 Histone Demethylases: Emerging Targets in Cancer</span></div><div class="casAuthors">Harmeyer, Kayla M.; Facompre, Nicole D.; Herlyn, Meenhard; Basu, Devraj</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Cancer</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">713-725</span>CODEN:
                <span class="NLM_cas:coden">TCRAB5</span>;
        ISSN:<span class="NLM_cas:issn">2405-8033</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  JARID1 proteins are histone demethylases that both regulate normal cell fates during development and contribute to the epigenetic plasticity that underlies malignant transformation.  This H3K4 demethylase family participates in multiple repressive transcriptional complexes at promoters and has broader regulatory effects on chromatin that remain ill-defined.  There is growing understanding of the oncogenic and tumor suppressive functions of JARID1 proteins, which are contingent on cell context and the protein isoform.  Their contributions to stem cell-like dedifferentiation, tumor aggressiveness, and therapy resistance in cancer have sustained interest in the development of JARID1 inhibitors.  Here we review the diverse and context-specific functions of the JARID1 proteins that may impact the utilization of emerging targeted inhibitors of this histone demethylase family in cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJYLbdIKcjybVg90H21EOLACvtfcHk0lgLKFD_H76FuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXksVyitg%253D%253D&md5=e728e47a8d60d5e83f29ab6d153a2be4</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.trecan.2017.08.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.trecan.2017.08.004%26sid%3Dliteratum%253Aachs%26aulast%3DHarmeyer%26aufirst%3DK.%2BM.%26aulast%3DFacompre%26aufirst%3DN.%2BD.%26aulast%3DHerlyn%26aufirst%3DM.%26aulast%3DBasu%26aufirst%3DD.%26atitle%3DJARID1%2520histone%2520demethylases%253A%2520emerging%2520targets%2520in%2520cancer%26jtitle%3DTrends%2520Cancer%26date%3D2017%26volume%3D3%26spage%3D713%26epage%3D725%26doi%3D10.1016%2Fj.trecan.2017.08.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shokri, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doudi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fathi-Roudsari, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kouhkan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanati, M. H.</span></span> <span> </span><span class="NLM_article-title">Targeting histone demethylases KDM5A and KDM5B in AML cancer cells: A comparative view</span>. <i>Leuk. Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>68</i></span>,  <span class="NLM_fpage">105</span>– <span class="NLM_lpage">111</span>, <span class="refDoi"> DOI: 10.1016/j.leukres.2018.02.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=10.1016%2Fj.leukres.2018.02.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=29602065" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=1%3ACAS%3A528%3ADC%252BC1cXmsVaru7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2018&pages=105-111&author=G.+Shokriauthor=S.+Doudiauthor=M.+Fathi-Roudsariauthor=F.+Kouhkanauthor=M.+H.+Sanati&title=Targeting+histone+demethylases+KDM5A+and+KDM5B+in+AML+cancer+cells%3A+A+comparative+view&doi=10.1016%2Fj.leukres.2018.02.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting histone demethylases KDM5A and KDM5B in AML cancer cells: A comparative view</span></div><div class="casAuthors">Shokri, Gelareh; Doudi, Shaghayegh; Fathi-Roudsari, Mehrnoosh; Kouhkan, Fatemeh; Sanati, Mohammad-Hossein</div><div class="citationInfo"><span class="NLM_cas:title">Leukemia Research</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">105-111</span>CODEN:
                <span class="NLM_cas:coden">LEREDD</span>;
        ISSN:<span class="NLM_cas:issn">0145-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Epigenetic modifications play an important role in initiation and progression of cancers including acute myeloid leukemia.  Among different epigenetic modifiers, lysine specific demethylases have been noticed as potential therapeutic targets.  KDM5 family of histone demethylases which removes Me marks from lysine residues of H3, are frequently found in the promoter region of transcriptionally active genes resulting in repression of expression.  Here we have compared the effects of KDM5A and KDM5B downregulation on HL-60 cell line behavior.  KDM5A/5B knockdown resulted in lower viability of HL-60 cells in addn. to modified cell cycle distribution and sub-G1 accumulation.  Induction of apoptosis was obsd. in both knockdown cells.  But in spite of similarity in their role, downregulation of KDM5A showed more efficient anti-leukemic effects in comparison to KDM5B.  Cells showed higher accumulation in sub-G1 and apoptosis occurred significantly higher and also earlier after KDM5A redn.  Expression anal. confirmed almost 5 and 4 fold increased expression for bax and caspase-3 after downregulation of KDM5A in comparison to KDM5B.  Due to the present study we propose KDM5A as a potential target for therapeutic aspects of acute myeloid leukemia although further investigations are needed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcIUHbiIE13rVg90H21EOLACvtfcHk0lgLKFD_H76FuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXmsVaru7k%253D&md5=22a34070de67e600b8164273c1112405</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.leukres.2018.02.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.leukres.2018.02.003%26sid%3Dliteratum%253Aachs%26aulast%3DShokri%26aufirst%3DG.%26aulast%3DDoudi%26aufirst%3DS.%26aulast%3DFathi-Roudsari%26aufirst%3DM.%26aulast%3DKouhkan%26aufirst%3DF.%26aulast%3DSanati%26aufirst%3DM.%2BH.%26atitle%3DTargeting%2520histone%2520demethylases%2520KDM5A%2520and%2520KDM5B%2520in%2520AML%2520cancer%2520cells%253A%2520A%2520comparative%2520view%26jtitle%3DLeuk.%2520Res.%26date%3D2018%26volume%3D68%26spage%3D105%26epage%3D111%26doi%3D10.1016%2Fj.leukres.2018.02.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Labadie, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ortwine, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vinogradova, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiefer, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mauer, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gehling, V. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harmange, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cummings, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gustafson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van der Porten, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liederer, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deshmukh, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">An, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ran, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Classon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trojer, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dragovich, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, L.</span></span> <span> </span><span class="NLM_article-title">From a novel HTS hit to potent, selective, and orally bioavailable KDM5 inhibitors</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">2974</span>– <span class="NLM_lpage">2981</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2017.05.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=10.1016%2Fj.bmcl.2017.05.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=28512031" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=1%3ACAS%3A528%3ADC%252BC2sXnvFCqtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2017&pages=2974-2981&author=J.+Liangauthor=S.+Labadieauthor=B.+Zhangauthor=D.+F.+Ortwineauthor=S.+Patelauthor=M.+Vinogradovaauthor=J.+R.+Kieferauthor=T.+Mauerauthor=V.+S.+Gehlingauthor=J.+C.+Harmangeauthor=R.+Cummingsauthor=T.+Laiauthor=J.+Liaoauthor=X.+Zhengauthor=Y.+Liuauthor=A.+Gustafsonauthor=E.+Van+der+Portenauthor=W.+Maoauthor=B.+M.+Liedererauthor=G.+Deshmukhauthor=L.+Anauthor=Y.+Ranauthor=M.+Classonauthor=P.+Trojerauthor=P.+S.+Dragovichauthor=L.+Murray&title=From+a+novel+HTS+hit+to+potent%2C+selective%2C+and+orally+bioavailable+KDM5+inhibitors&doi=10.1016%2Fj.bmcl.2017.05.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">From a novel HTS hit to potent, selective, and orally bioavailable KDM5 inhibitors</span></div><div class="casAuthors">Liang, Jun; Labadie, Sharada; Zhang, Birong; Ortwine, Daniel F.; Patel, Snahel; Vinogradova, Maia; Kiefer, James R.; Mauer, Till; Gehling, Victor S.; Harmange, Jean-Christophe; Cummings, Richard; Lai, Tommy; Liao, Jiangpeng; Zheng, Xiaoping; Liu, Yichin; Gustafson, Amy; Van der Porten, Erica; Mao, Weifeng; Liederer, Bianca M.; Deshmukh, Gauri; An, Le; Ran, Yingqing; Classon, Marie; Trojer, Patrick; Dragovich, Peter S.; Murray, Lesley</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">2974-2981</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A high-throughput screening (HTS) of the Genentech/Roche library identified a novel, uncharged scaffold as a KDM5A inhibitor.  Lacking insight into the binding mode, initial attempts to improve inhibitor potency failed to improve potency, and synthesis of analogs was further hampered by the presence of a C-C bond between the pyrrolidine and pyridine.  Replacing this with a C-N bond significantly simplified synthesis, yielding pyrazole analog, [3-(4-bromo-1H-pyrazol-1-yl)-1-pyrrolidinyl][5-(1-methylethyl)-1H-pyrazol-3-yl]methanone (35), of which the authors obtained a co-crystal structure with KDM5A.  Using structure-based design approach, the authors identified, N-[(3R)-1-[[5-(1-methylethyl)-1H-pyrazol-3-yl]carbonyl]-3-pyrrolidinyl]cyclopropanecarboxamide (50), with improved biochem., cell potency and reduced MW and lower lipophilicity (Log D) compared with the original hit.  Furthermore, 50 showed lower clearance than [5-(1-methylethyl)-1H-pyrazol-3-yl][(3S)-3-[6-methyl-4-(1-methyl-1H-pyrazol-4-yl)-2-pyridinyl]-1-pyrrolidinyl]methanone (9) in mice.  In combination with its remarkably low plasma protein binding (PPB) in mice (40%), oral dosing of 50 at 5 mg/kg resulted in unbound Cmax ∼2-fold of its cell potency (PC9 H3K4Me3 0.96 μM), meeting the authors' criteria for an in vivo tool compd. from a new scaffold.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpksjXQBrVYsrVg90H21EOLACvtfcHk0lgLKFD_H76FuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXnvFCqtr4%253D&md5=04bee4e29775202fb1882adeccb8e722</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2017.05.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2017.05.016%26sid%3Dliteratum%253Aachs%26aulast%3DLiang%26aufirst%3DJ.%26aulast%3DLabadie%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DB.%26aulast%3DOrtwine%26aufirst%3DD.%2BF.%26aulast%3DPatel%26aufirst%3DS.%26aulast%3DVinogradova%26aufirst%3DM.%26aulast%3DKiefer%26aufirst%3DJ.%2BR.%26aulast%3DMauer%26aufirst%3DT.%26aulast%3DGehling%26aufirst%3DV.%2BS.%26aulast%3DHarmange%26aufirst%3DJ.%2BC.%26aulast%3DCummings%26aufirst%3DR.%26aulast%3DLai%26aufirst%3DT.%26aulast%3DLiao%26aufirst%3DJ.%26aulast%3DZheng%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DGustafson%26aufirst%3DA.%26aulast%3DVan%2Bder%2BPorten%26aufirst%3DE.%26aulast%3DMao%26aufirst%3DW.%26aulast%3DLiederer%26aufirst%3DB.%2BM.%26aulast%3DDeshmukh%26aufirst%3DG.%26aulast%3DAn%26aufirst%3DL.%26aulast%3DRan%26aufirst%3DY.%26aulast%3DClasson%26aufirst%3DM.%26aulast%3DTrojer%26aufirst%3DP.%26aulast%3DDragovich%26aufirst%3DP.%2BS.%26aulast%3DMurray%26aufirst%3DL.%26atitle%3DFrom%2520a%2520novel%2520HTS%2520hit%2520to%2520potent%252C%2520selective%252C%2520and%2520orally%2520bioavailable%2520KDM5%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2017%26volume%3D27%26spage%3D2974%26epage%3D2981%26doi%3D10.1016%2Fj.bmcl.2017.05.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Horton, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shanks, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rai, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mott, B. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jansen, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kales, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henderson, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pohida, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jadhav, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maloney, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simeonov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vertino, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, X.</span></span> <span> </span><span class="NLM_article-title">Insights into the action of inhibitor enantiomers against histone lysine demethylase 5A</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">3193</span>– <span class="NLM_lpage">3208</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00261</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00261" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=1%3ACAS%3A528%3ADC%252BC1cXkslWmtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=3193-3208&author=J.+R.+Hortonauthor=X.+Liuauthor=L.+Wuauthor=K.+Zhangauthor=J.+Shanksauthor=X.+Zhangauthor=G.+Raiauthor=B.+T.+Mottauthor=D.+J.+Jansenauthor=S.+C.+Kalesauthor=M.+J.+Hendersonauthor=K.+Pohidaauthor=Y.+Fangauthor=X.+Huauthor=A.+Jadhavauthor=D.+J.+Maloneyauthor=M.+D.+Hallauthor=A.+Simeonovauthor=H.+Fuauthor=P.+M.+Vertinoauthor=Q.+Yanauthor=X.+Cheng&title=Insights+into+the+action+of+inhibitor+enantiomers+against+histone+lysine+demethylase+5A&doi=10.1021%2Facs.jmedchem.8b00261"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Insights into the Action of Inhibitor Enantiomers against Histone Lysine Demethylase 5A</span></div><div class="casAuthors">Horton, John R.; Liu, Xu; Wu, Lizhen; Zhang, Kai; Shanks, John; Zhang, Xing; Rai, Ganesha; Mott, Bryan T.; Jansen, Daniel J.; Kales, Stephen C.; Henderson, Mark J.; Pohida, Katherine; Fang, Yuhong; Hu, Xin; Jadhav, Ajit; Maloney, David J.; Hall, Matthew D.; Simeonov, Anton; Fu, Haian; Vertino, Paula M.; Yan, Qin; Cheng, Xiaodong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3193-3208</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Isomers of chiral drugs can exhibit marked differences in biol. activities.  We studied the binding and inhibitory activities of 12 compds. against KDM5A.  Among them are two pairs of enantiomers representing two distinct inhibitor chemotypes, namely, (R)- and (S)-2-((2-chlorophenyl)(2-(piperidin-1-yl)ethoxy)methyl)-1H-pyrrolo[3,2-b]pyridine-7-carboxylic acid (compds. N51 and N52) and (R)- and (S)-N-(1-(3-isopropyl-1H-pyrazole-5-carbonyl)pyrrolidin-3-yl)cyclopropanecarboxamide (compds. N54 and N55).  In vitro, the S enantiomer of the N51/N52 pair (N52) and the R enantiomer of the N54/N55 pair (N54) exhibited about 4- to 5-fold greater binding affinity.  The more potent enzyme inhibition of KDM5A by the R-isoform for the cell-permeable N54/N55 pair translated to differences in growth inhibitory activity.  We detd. structures of the KDM5A catalytic domain in complex with all 12 inhibitors, which revealed the interactions (or lack thereof) responsible for the differences in binding affinity.  These results provide insights to guide improvements in binding potency and avenues for development of cell permeable inhibitors of the KDM5 family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovFfXyPcUszbVg90H21EOLACvtfcHk0liUhNeRMHs4DA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXkslWmtro%253D&md5=13ecac6902afd69d4ce7ad05803ca4d9</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00261%26sid%3Dliteratum%253Aachs%26aulast%3DHorton%26aufirst%3DJ.%2BR.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DWu%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DK.%26aulast%3DShanks%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DRai%26aufirst%3DG.%26aulast%3DMott%26aufirst%3DB.%2BT.%26aulast%3DJansen%26aufirst%3DD.%2BJ.%26aulast%3DKales%26aufirst%3DS.%2BC.%26aulast%3DHenderson%26aufirst%3DM.%2BJ.%26aulast%3DPohida%26aufirst%3DK.%26aulast%3DFang%26aufirst%3DY.%26aulast%3DHu%26aufirst%3DX.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DMaloney%26aufirst%3DD.%2BJ.%26aulast%3DHall%26aufirst%3DM.%2BD.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DFu%26aufirst%3DH.%26aulast%3DVertino%26aufirst%3DP.%2BM.%26aulast%3DYan%26aufirst%3DQ.%26aulast%3DCheng%26aufirst%3DX.%26atitle%3DInsights%2520into%2520the%2520action%2520of%2520inhibitor%2520enantiomers%2520against%2520histone%2520lysine%2520demethylase%25205A%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D3193%26epage%3D3208%26doi%3D10.1021%2Facs.jmedchem.8b00261" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vinogradova, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gehling, V. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gustafson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arora, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tindell, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williamson, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guler, G. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gangurde, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manieri, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Busby, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flynn, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Odate, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cochran, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wongchenko, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheung, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maile, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lau, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Costa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hegde, G. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jackson, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pitti, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnott, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bailey, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cummings, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albrecht, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harmange, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiefer, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trojer, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Classon, M.</span></span> <span> </span><span class="NLM_article-title">An inhibitor of KDM5 demethylases reduces survival of drug-tolerant cancer cells</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">531</span>– <span class="NLM_lpage">538</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2085</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=10.1038%2Fnchembio.2085" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=27214401" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=1%3ACAS%3A528%3ADC%252BC28XnvFOksLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=531-538&author=M.+Vinogradovaauthor=V.+S.+Gehlingauthor=A.+Gustafsonauthor=S.+Aroraauthor=C.+A.+Tindellauthor=C.+Wilsonauthor=K.+E.+Williamsonauthor=G.+D.+Gulerauthor=P.+Gangurdeauthor=W.+Manieriauthor=J.+Busbyauthor=E.+M.+Flynnauthor=F.+Lanauthor=H.+J.+Kimauthor=S.+Odateauthor=A.+G.+Cochranauthor=Y.+Liuauthor=M.+Wongchenkoauthor=Y.+Yangauthor=T.+K.+Cheungauthor=T.+M.+Maileauthor=T.+Lauauthor=M.+Costaauthor=G.+V.+Hegdeauthor=E.+Jacksonauthor=R.+Pittiauthor=D.+Arnottauthor=C.+Baileyauthor=S.+Bellonauthor=R.+T.+Cummingsauthor=B.+K.+Albrechtauthor=J.+C.+Harmangeauthor=J.+R.+Kieferauthor=P.+Trojerauthor=M.+Classon&title=An+inhibitor+of+KDM5+demethylases+reduces+survival+of+drug-tolerant+cancer+cells&doi=10.1038%2Fnchembio.2085"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">An inhibitor of KDM5 demethylases reduces survival of drug-tolerant cancer cells</span></div><div class="casAuthors">Vinogradova, Maia; Gehling, Victor S.; Gustafson, Amy; Arora, Shilpi; Tindell, Charles A.; Wilson, Catherine; Williamson, Kaylyn E.; Guler, Gulfem D.; Gangurde, Pranoti; Manieri, Wanda; Busby, Jennifer; Flynn, E. Megan; Lan, Fei; Kim, Hyo-jin; Odate, Shobu; Cochran, Andrea G.; Liu, Yichin; Wongchenko, Matthew; Yang, Yibin; Cheung, Tommy K.; Maile, Tobias M.; Lau, Ted; Costa, Michael; Hegde, Ganapati V.; Jackson, Erica; Pitti, Robert; Arnott, David; Bailey, Christopher; Bellon, Steve; Cummings, Richard T.; Albrecht, Brian K.; Harmange, Jean-Christophe; Kiefer, James R.; Trojer, Patrick; Classon, Marie</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">531-538</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The KDM5 family of histone demethylases catalyzes the demethylation of histone H3 on lysine 4 (H3K4) and is required for the survival of drug-tolerant persister cancer cells (DTPs).  Here the authors report the discovery and characterization of the specific KDM5 inhibitor CPI-455.  The crystal structure of KDM5A revealed the mechanism of inhibition of CPI-455 as well as the topol. arrangements of protein domains that influence substrate binding.  CPI-455 mediated KDM5 inhibition, elevated global levels of H3K4 trimethylation (H3K4me3) and decreased the no. of DTPs in multiple cancer cell line models treated with std. chemotherapy or targeted agents.  These findings show that pretreatment of cancer cells with a KDM5-specific inhibitor results in the ablation of a subpopulation of cancer cells that can serve as the founders for therapeutic relapse.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxyW7L2YsV1rVg90H21EOLACvtfcHk0liUhNeRMHs4DA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XnvFOksLg%253D&md5=41699f5c17c912b39cdda81f4dfaa31e</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2085&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2085%26sid%3Dliteratum%253Aachs%26aulast%3DVinogradova%26aufirst%3DM.%26aulast%3DGehling%26aufirst%3DV.%2BS.%26aulast%3DGustafson%26aufirst%3DA.%26aulast%3DArora%26aufirst%3DS.%26aulast%3DTindell%26aufirst%3DC.%2BA.%26aulast%3DWilson%26aufirst%3DC.%26aulast%3DWilliamson%26aufirst%3DK.%2BE.%26aulast%3DGuler%26aufirst%3DG.%2BD.%26aulast%3DGangurde%26aufirst%3DP.%26aulast%3DManieri%26aufirst%3DW.%26aulast%3DBusby%26aufirst%3DJ.%26aulast%3DFlynn%26aufirst%3DE.%2BM.%26aulast%3DLan%26aufirst%3DF.%26aulast%3DKim%26aufirst%3DH.%2BJ.%26aulast%3DOdate%26aufirst%3DS.%26aulast%3DCochran%26aufirst%3DA.%2BG.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DWongchenko%26aufirst%3DM.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DCheung%26aufirst%3DT.%2BK.%26aulast%3DMaile%26aufirst%3DT.%2BM.%26aulast%3DLau%26aufirst%3DT.%26aulast%3DCosta%26aufirst%3DM.%26aulast%3DHegde%26aufirst%3DG.%2BV.%26aulast%3DJackson%26aufirst%3DE.%26aulast%3DPitti%26aufirst%3DR.%26aulast%3DArnott%26aufirst%3DD.%26aulast%3DBailey%26aufirst%3DC.%26aulast%3DBellon%26aufirst%3DS.%26aulast%3DCummings%26aufirst%3DR.%2BT.%26aulast%3DAlbrecht%26aufirst%3DB.%2BK.%26aulast%3DHarmange%26aufirst%3DJ.%2BC.%26aulast%3DKiefer%26aufirst%3DJ.%2BR.%26aulast%3DTrojer%26aufirst%3DP.%26aulast%3DClasson%26aufirst%3DM.%26atitle%3DAn%2520inhibitor%2520of%2520KDM5%2520demethylases%2520reduces%2520survival%2520of%2520drug-tolerant%2520cancer%2520cells%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2016%26volume%3D12%26spage%3D531%26epage%3D538%26doi%3D10.1038%2Fnchembio.2085" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tumber, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nuzzi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hookway, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatch, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Velupillai, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johansson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawamura, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savitsky, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yapp, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szykowska, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bountra, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strain-Damerell, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burgess-Brown, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruda, G. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munro, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">England, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nowak, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schofield, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">La Thangue, N. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pawlyn, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morgan, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Athanasou, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oppermann, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span> <span> </span><span class="NLM_article-title">Potent and selective KDM5 inhibitor stops cellular demethylation of H3K4me3 at transcription start sites and proliferation of MM1S myeloma cells</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">371</span>– <span class="NLM_lpage">380</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2017.02.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=10.1016%2Fj.chembiol.2017.02.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=28262558" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjvVOktro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2017&pages=371-380&author=A.+Tumberauthor=A.+Nuzziauthor=E.+S.+Hookwayauthor=S.+B.+Hatchauthor=S.+Velupillaiauthor=C.+Johanssonauthor=A.+Kawamuraauthor=P.+Savitskyauthor=C.+Yappauthor=A.+Szykowskaauthor=N.+Wuauthor=C.+Bountraauthor=C.+Strain-Damerellauthor=N.+A.+Burgess-Brownauthor=G.+F.+Rudaauthor=O.+Fedorovauthor=S.+Munroauthor=K.+S.+Englandauthor=R.+P.+Nowakauthor=C.+J.+Schofieldauthor=N.+B.+La+Thangueauthor=C.+Pawlynauthor=F.+Daviesauthor=G.+Morganauthor=N.+Athanasouauthor=S.+Mullerauthor=U.+Oppermannauthor=P.+E.+Brennan&title=Potent+and+selective+KDM5+inhibitor+stops+cellular+demethylation+of+H3K4me3+at+transcription+start+sites+and+proliferation+of+MM1S+myeloma+cells&doi=10.1016%2Fj.chembiol.2017.02.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Potent and Selective KDM5 Inhibitor Stops Cellular Demethylation of H3K4me3 at Transcription Start Sites and Proliferation of MM1S Myeloma Cells</span></div><div class="casAuthors">Tumber, Anthony; Nuzzi, Andrea; Hookway, Edward S.; Hatch, Stephanie B.; Velupillai, Srikannathasan; Johansson, Catrine; Kawamura, Akane; Savitsky, Pavel; Yapp, Clarence; Szykowska, Aleksandra; Wu, Na; Bountra, Chas; Strain-Damerell, Claire; Burgess-Brown, Nicola A.; Ruda, Gian Filippo; Fedorov, Oleg; Munro, Shonagh; England, Katherine S.; Nowak, Radoslaw P.; Schofield, Christopher J.; La Thangue, Nicholas B.; Pawlyn, Charlotte; Davies, Faith; Morgan, Gareth; Athanasou, Nick; Muller, Susanne; Oppermann, Udo; Brennan, Paul E.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">371-380</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Methylation of lysine residues on histone tail is a dynamic epigenetic modification that plays a key role in chromatin structure and gene regulation.  Members of the KDM5 (also known as JARID1) sub-family are 2-oxoglutarate (2-OG) and Fe2+-dependent oxygenases acting as histone 3 lysine 4 tri-Me (H3K4me3) demethylases, regulating proliferation, stem cell self-renewal, and differentiation.  Here we present the characterization of KDOAM-25, an inhibitor of KDM5 enzymes.  KDOAM-25 shows biochem. half maximal inhibitory concn. values of <100 nM for KDM5A-D in vitro, high selectivity toward other 2-OG oxygenases sub-families, and no off-target activity on a panel of 55 receptors and enzymes.  In human cell assay systems, KDOAM-25 has a half maximal effective concn. of ∼50 μM and good selectivity toward other demethylases.  KDM5B is overexpressed in multiple myeloma and neg. correlated with the overall survival.  Multiple myeloma MM1S cells treated with KDOAM-25 show increased global H3K4 methylation at transcriptional start sites and impaired proliferation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-UzNQI-MiYbVg90H21EOLACvtfcHk0liMvuXcAuXrow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjvVOktro%253D&md5=0fc70179bda8b4fd38db1054c7bb3277</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2017.02.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2017.02.006%26sid%3Dliteratum%253Aachs%26aulast%3DTumber%26aufirst%3DA.%26aulast%3DNuzzi%26aufirst%3DA.%26aulast%3DHookway%26aufirst%3DE.%2BS.%26aulast%3DHatch%26aufirst%3DS.%2BB.%26aulast%3DVelupillai%26aufirst%3DS.%26aulast%3DJohansson%26aufirst%3DC.%26aulast%3DKawamura%26aufirst%3DA.%26aulast%3DSavitsky%26aufirst%3DP.%26aulast%3DYapp%26aufirst%3DC.%26aulast%3DSzykowska%26aufirst%3DA.%26aulast%3DWu%26aufirst%3DN.%26aulast%3DBountra%26aufirst%3DC.%26aulast%3DStrain-Damerell%26aufirst%3DC.%26aulast%3DBurgess-Brown%26aufirst%3DN.%2BA.%26aulast%3DRuda%26aufirst%3DG.%2BF.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DMunro%26aufirst%3DS.%26aulast%3DEngland%26aufirst%3DK.%2BS.%26aulast%3DNowak%26aufirst%3DR.%2BP.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26aulast%3DLa%2BThangue%26aufirst%3DN.%2BB.%26aulast%3DPawlyn%26aufirst%3DC.%26aulast%3DDavies%26aufirst%3DF.%26aulast%3DMorgan%26aufirst%3DG.%26aulast%3DAthanasou%26aufirst%3DN.%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DOppermann%26aufirst%3DU.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26atitle%3DPotent%2520and%2520selective%2520KDM5%2520inhibitor%2520stops%2520cellular%2520demethylation%2520of%2520H3K4me3%2520at%2520transcription%2520start%2520sites%2520and%2520proliferation%2520of%2520MM1S%2520myeloma%2520cells%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2017%26volume%3D24%26spage%3D371%26epage%3D380%26doi%3D10.1016%2Fj.chembiol.2017.02.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Horton, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gale, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shanks, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Webber, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bell, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kales, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mott, B. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rai, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jansen, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henderson, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urban, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simeonov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maloney, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johns, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jadhav, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vertino, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, X.</span></span> <span> </span><span class="NLM_article-title">Structural basis for KDM5A histone lysine demethylase inhibition by diverse compounds</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">769</span>– <span class="NLM_lpage">781</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2016.06.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=10.1016%2Fj.chembiol.2016.06.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=27427228" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFyiur3L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2016&pages=769-781&author=J.+R.+Hortonauthor=X.+Liuauthor=M.+Galeauthor=L.+Wuauthor=J.+R.+Shanksauthor=X.+Zhangauthor=P.+J.+Webberauthor=J.+S.+Bellauthor=S.+C.+Kalesauthor=B.+T.+Mottauthor=G.+Raiauthor=D.+J.+Jansenauthor=M.+J.+Hendersonauthor=D.+J.+Urbanauthor=M.+D.+Hallauthor=A.+Simeonovauthor=D.+J.+Maloneyauthor=M.+A.+Johnsauthor=H.+Fuauthor=A.+Jadhavauthor=P.+M.+Vertinoauthor=Q.+Yanauthor=X.+Cheng&title=Structural+basis+for+KDM5A+histone+lysine+demethylase+inhibition+by+diverse+compounds&doi=10.1016%2Fj.chembiol.2016.06.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Basis for KDM5A Histone Lysine Demethylase Inhibition by Diverse Compounds</span></div><div class="casAuthors">Horton, John R.; Liu, Xu; Gale, Molly; Wu, Lizhen; Shanks, John R.; Zhang, Xing; Webber, Philip J.; Bell, Joshua S. K.; Kales, Stephen C.; Mott, Bryan T.; Rai, Ganesha; Jansen, Daniel J.; Henderson, Mark J.; Urban, Daniel J.; Hall, Matthew D.; Simeonov, Anton; Maloney, David J.; Johns, Margaret A.; Fu, Haian; Jadhav, Ajit; Vertino, Paula M.; Yan, Qin; Cheng, Xiaodong</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">769-781</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The KDM5/JARID1 family of Fe(II)- and α-ketoglutarate-dependent demethylases removes Me groups from methylated lysine 4 of histone H3.  Accumulating evidence supports a role for KDM5 family members as oncogenic drivers.  We compare the in vitro inhibitory properties and binding affinity of ten diverse compds. with all four family members, and present the crystal structures of the KDM5A-linked Jumonji domain in complex with eight of these inhibitors in the presence of Mn(II).  All eight inhibitors structurally examd. occupy the binding site of α-ketoglutarate, but differ in their specific binding interactions, including the no. of ligands involved in metal coordination.  We also obsd. inhibitor-induced conformational changes in KDM5A, particularly those residues involved in the binding of α-ketoglutarate, the anticipated peptide substrate, and intramol. interactions.  We discuss how particular chem. moieties contribute to inhibitor potency and suggest strategies that might be utilized in the successful design of selective and potent epigenetic inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpstbuqHdktk7Vg90H21EOLACvtfcHk0lgAX7QOQl2mYQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFyiur3L&md5=0dc780604904ac406211a09577df5bf5</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2016.06.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2016.06.006%26sid%3Dliteratum%253Aachs%26aulast%3DHorton%26aufirst%3DJ.%2BR.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DGale%26aufirst%3DM.%26aulast%3DWu%26aufirst%3DL.%26aulast%3DShanks%26aufirst%3DJ.%2BR.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DWebber%26aufirst%3DP.%2BJ.%26aulast%3DBell%26aufirst%3DJ.%2BS.%26aulast%3DKales%26aufirst%3DS.%2BC.%26aulast%3DMott%26aufirst%3DB.%2BT.%26aulast%3DRai%26aufirst%3DG.%26aulast%3DJansen%26aufirst%3DD.%2BJ.%26aulast%3DHenderson%26aufirst%3DM.%2BJ.%26aulast%3DUrban%26aufirst%3DD.%2BJ.%26aulast%3DHall%26aufirst%3DM.%2BD.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DMaloney%26aufirst%3DD.%2BJ.%26aulast%3DJohns%26aufirst%3DM.%2BA.%26aulast%3DFu%26aufirst%3DH.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DVertino%26aufirst%3DP.%2BM.%26aulast%3DYan%26aufirst%3DQ.%26aulast%3DCheng%26aufirst%3DX.%26atitle%3DStructural%2520basis%2520for%2520KDM5A%2520histone%2520lysine%2520demethylase%2520inhibition%2520by%2520diverse%2520compounds%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2016%26volume%3D23%26spage%3D769%26epage%3D781%26doi%3D10.1016%2Fj.chembiol.2016.06.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Johansson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Velupillai, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tumber, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szykowska, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hookway, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nowak, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strain-Damerell, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gileadi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philpott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burgess-Brown, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kopec, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nuzzi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steuber, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Egner, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Badock, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munro, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LaThangue, N. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westaway, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Athanasou, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oppermann, U.</span></span> <span> </span><span class="NLM_article-title">Structural analysis of human KDM5B guides histone demethylase inhibitor development</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">539</span>– <span class="NLM_lpage">545</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2087</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=10.1038%2Fnchembio.2087" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=27214403" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=1%3ACAS%3A528%3ADC%252BC28XosVWisbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=539-545&author=C.+Johanssonauthor=S.+Velupillaiauthor=A.+Tumberauthor=A.+Szykowskaauthor=E.+S.+Hookwayauthor=R.+P.+Nowakauthor=C.+Strain-Damerellauthor=C.+Gileadiauthor=M.+Philpottauthor=N.+Burgess-Brownauthor=N.+Wuauthor=J.+Kopecauthor=A.+Nuzziauthor=H.+Steuberauthor=U.+Egnerauthor=V.+Badockauthor=S.+Munroauthor=N.+B.+LaThangueauthor=S.+Westawayauthor=J.+Brownauthor=N.+Athanasouauthor=R.+Prinjhaauthor=P.+E.+Brennanauthor=U.+Oppermann&title=Structural+analysis+of+human+KDM5B+guides+histone+demethylase+inhibitor+development&doi=10.1038%2Fnchembio.2087"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Structural analysis of human KDM5B guides histone demethylase inhibitor development</span></div><div class="casAuthors">Johansson, Catrine; Velupillai, Srikannathasan; Tumber, Anthony; Szykowska, Aleksandra; Hookway, Edward S.; Nowak, Radoslaw P.; Strain-Damerell, Claire; Gileadi, Carina; Philpott, Martin; Burgess-Brown, Nicola; Wu, Na; Kopec, Jola; Nuzzi, Andrea; Steuber, Holger; Egner, Ursula; Badock, Volker; Munro, Shonagh; LaThangue, Nicholas B.; Westaway, Sue; Brown, Jack; Athanasou, Nick; Prinjha, Rab; Brennan, Paul E.; Oppermann, Udo</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">539-545</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Members of the KDM5 (also known as JARID1) family are 2-oxoglutarate- and Fe2+-dependent oxygenases that act as histone H3K4 demethylases, thereby regulating cell proliferation and stem cell self-renewal and differentiation.  Here, the authors report crystal structures of the catalytic core of the human KDM5B enzyme in complex with 3 inhibitor chemotypes.  These scaffolds exploitet several aspects of the KDM5 active site, and their selectivity profiles reflected their hybrid features with respect to the KDM4 and KDM6 families.  Whereas GSK-J1, a previously identified KDM6 inhibitor, showed ∼7-fold less inhibitory activity toward KDM5B than toward KDM6 proteins, KDM5-C49 displayed 25-100-fold selectivity between KDM5B and KDM6B.  The cell-permeable deriv. KDM5-C70 had an antiproliferative effect in myeloma cells, leading to genome-wide elevation of H3K4me3 levels.  The selective inhibitor GSK467 exploited unique binding modes, but it lacked cellular potency in the myeloma system.  Taken together, these structural leads deliver multiple starting points for further rational and selective inhibitor design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqp-2vDMpTjHbVg90H21EOLACvtfcHk0lgAX7QOQl2mYQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XosVWisbs%253D&md5=2c4b6e5779ba43ed3584fc6f76b987cd</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2087&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2087%26sid%3Dliteratum%253Aachs%26aulast%3DJohansson%26aufirst%3DC.%26aulast%3DVelupillai%26aufirst%3DS.%26aulast%3DTumber%26aufirst%3DA.%26aulast%3DSzykowska%26aufirst%3DA.%26aulast%3DHookway%26aufirst%3DE.%2BS.%26aulast%3DNowak%26aufirst%3DR.%2BP.%26aulast%3DStrain-Damerell%26aufirst%3DC.%26aulast%3DGileadi%26aufirst%3DC.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DBurgess-Brown%26aufirst%3DN.%26aulast%3DWu%26aufirst%3DN.%26aulast%3DKopec%26aufirst%3DJ.%26aulast%3DNuzzi%26aufirst%3DA.%26aulast%3DSteuber%26aufirst%3DH.%26aulast%3DEgner%26aufirst%3DU.%26aulast%3DBadock%26aufirst%3DV.%26aulast%3DMunro%26aufirst%3DS.%26aulast%3DLaThangue%26aufirst%3DN.%2BB.%26aulast%3DWestaway%26aufirst%3DS.%26aulast%3DBrown%26aufirst%3DJ.%26aulast%3DAthanasou%26aufirst%3DN.%26aulast%3DPrinjha%26aufirst%3DR.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DOppermann%26aufirst%3DU.%26atitle%3DStructural%2520analysis%2520of%2520human%2520KDM5B%2520guides%2520histone%2520demethylase%2520inhibitor%2520development%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2016%26volume%3D12%26spage%3D539%26epage%3D545%26doi%3D10.1038%2Fnchembio.2087" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Herr, C. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hausinger, R. P.</span></span> <span> </span><span class="NLM_article-title">Amazing diversity in biochemical roles of Fe(II)/2-oxoglutarate oxygenases</span>. <i>Trends Biochem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>43</i></span>,  <span class="NLM_fpage">517</span>– <span class="NLM_lpage">532</span>, <span class="refDoi"> DOI: 10.1016/j.tibs.2018.04.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=10.1016%2Fj.tibs.2018.04.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=29709390" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=1%3ACAS%3A528%3ADC%252BC1cXnslKktL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2018&pages=517-532&author=C.+Q.+Herrauthor=R.+P.+Hausinger&title=Amazing+diversity+in+biochemical+roles+of+Fe%28II%29%2F2-oxoglutarate+oxygenases&doi=10.1016%2Fj.tibs.2018.04.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Amazing Diversity in Biochemical Roles of Fe(II)/2-Oxoglutarate Oxygenases</span></div><div class="casAuthors">Herr, Caitlyn Q.; Hausinger, Robert P.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Biochemical Sciences</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">517-532</span>CODEN:
                <span class="NLM_cas:coden">TBSCDB</span>;
        ISSN:<span class="NLM_cas:issn">0968-0004</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Since their discovery in the 1960s, the family of Fe(II)/2-oxoglutarate-dependent oxygenases has undergone a tremendous expansion to include enzymes catalyzing a vast diversity of biol. important reactions.  Recent examples highlight roles in controlling chromatin modification, transcription, mRNA demethylation, and mRNA splicing.  Others generate modifications in tRNA, translation factors, ribosomes, and other proteins.  Thus, oxygenases affect all components of mol. biol.'s central dogma, in which information flows from DNA to RNA to proteins.  These enzymes also function in biosynthesis and catabolism of cellular metabolites, including antibiotics and signaling mols.  Due to their crit. importance, ongoing efforts have targeted family members for the development of specific therapeutics.  This review provides a general overview of recently characterized oxygenase reactions and their key biol. roles.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWNCor1Ykj3LVg90H21EOLACvtfcHk0lhaf59pDgYCrA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXnslKktL4%253D&md5=fbdbc3a2ed5e19a0bbd9e2389471ad72</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.tibs.2018.04.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tibs.2018.04.002%26sid%3Dliteratum%253Aachs%26aulast%3DHerr%26aufirst%3DC.%2BQ.%26aulast%3DHausinger%26aufirst%3DR.%2BP.%26atitle%3DAmazing%2520diversity%2520in%2520biochemical%2520roles%2520of%2520Fe%2528II%2529%252F2-oxoglutarate%2520oxygenases%26jtitle%3DTrends%2520Biochem.%2520Sci.%26date%3D2018%26volume%3D43%26spage%3D517%26epage%3D532%26doi%3D10.1016%2Fj.tibs.2018.04.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kaniskan, H. U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martini, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Inhibitors of protein methyltransferases and demethylases</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>118</i></span>,  <span class="NLM_fpage">989</span>– <span class="NLM_lpage">1068</span>, <span class="refDoi"> DOI: 10.1021/acs.chemrev.6b00801</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.6b00801" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkslynsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=118&publication_year=2018&pages=989-1068&author=H.+U.+Kaniskanauthor=M.+L.+Martiniauthor=J.+Jin&title=Inhibitors+of+protein+methyltransferases+and+demethylases&doi=10.1021%2Facs.chemrev.6b00801"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibitors of Protein Methyltransferases and Demethylases</span></div><div class="casAuthors">Kaniskan, H. Umit; Martini, Michael L.; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">118</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">989-1068</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Post-translational modifications of histones by protein methyltransferases (PMTs) and histone demethylases (KDMs) play an important role in the regulation of gene expression and transcription and are implicated in cancer and many other diseases.  Many of these enzymes also target various nonhistone proteins impacting numerous crucial biol. pathways.  Given their key biol. functions and implications in human diseases, there has been a growing interest in assessing these enzymes as potential therapeutic targets.  Consequently, discovering and developing inhibitors of these enzymes has become a very active and fast-growing research area over the past decade.  In this review, the authors cover the discovery, characterization, and biol. application of inhibitors of PMTs and KDMs with emphasis on key advancements in the field.  The authors also discuss challenges, opportunities, and future directions in this emerging, exciting research field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJYjiYtw_n4rVg90H21EOLACvtfcHk0lhaf59pDgYCrA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkslynsL8%253D&md5=1ed520c6b0c42d2aa8a7f66b0a2d4e40</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.6b00801&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.6b00801%26sid%3Dliteratum%253Aachs%26aulast%3DKaniskan%26aufirst%3DH.%2BU.%26aulast%3DMartini%26aufirst%3DM.%2BL.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DInhibitors%2520of%2520protein%2520methyltransferases%2520and%2520demethylases%26jtitle%3DChem.%2520Rev.%26date%3D2018%26volume%3D118%26spage%3D989%26epage%3D1068%26doi%3D10.1021%2Facs.chemrev.6b00801" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Johansson, M. H.</span></span> <span> </span><span class="NLM_article-title">Reversible Michael additions: covalent inhibitors and prodrugs</span>. <i>Mini-Rev. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">1330</span>– <span class="NLM_lpage">1344</span>, <span class="refDoi"> DOI: 10.2174/13895575112091330</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=10.2174%2F13895575112091330" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=22625413" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs12hu7jJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2012&pages=1330-1344&author=M.+H.+Johansson&title=Reversible+Michael+additions%3A+covalent+inhibitors+and+prodrugs&doi=10.2174%2F13895575112091330"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Reversible Michael additions: covalent inhibitors and prodrugs</span></div><div class="casAuthors">Johansson, Martin H.</div><div class="citationInfo"><span class="NLM_cas:title">Mini-Reviews in Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">1330-1344</span>CODEN:
                <span class="NLM_cas:coden">MMCIAE</span>;
        ISSN:<span class="NLM_cas:issn">1389-5575</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Covalent inhibition is an efficient mol. mechanism for inhibiting enzymes or modulating the function of proteins and is found in the action of many drugs and biol. active natural products.  However, it is has been less appreciated that the formation of covalent bonds can be reversible or irreversible.  This review focuses on biol. active compds. that are Michael acceptors and how the reversible nature of the Michael addn. reaction influences biol. activity and how this can be exploited in designing prodrugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr9X1yh9k4q-7Vg90H21EOLACvtfcHk0lhaf59pDgYCrA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs12hu7jJ&md5=f90694162e39fe57915e888bbadbd2bb</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.2174%2F13895575112091330&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F13895575112091330%26sid%3Dliteratum%253Aachs%26aulast%3DJohansson%26aufirst%3DM.%2BH.%26atitle%3DReversible%2520Michael%2520additions%253A%2520covalent%2520inhibitors%2520and%2520prodrugs%26jtitle%3DMini-Rev.%2520Med.%2520Chem.%26date%3D2012%26volume%3D12%26spage%3D1330%26epage%3D1344%26doi%3D10.2174%2F13895575112091330" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Awoonor-Williams, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walsh, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rowley, C. N.</span></span> <span> </span><span class="NLM_article-title">Modeling covalent-modifier drugs</span>. <i>Biochim. Biophys. Acta, Proteins Proteomics</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>1865</i></span>,  <span class="NLM_fpage">1664</span>– <span class="NLM_lpage">1675</span>, <span class="refDoi"> DOI: 10.1016/j.bbapap.2017.05.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=10.1016%2Fj.bbapap.2017.05.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=28528876" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=1%3ACAS%3A528%3ADC%252BC2sXotVOrtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1865&publication_year=2017&pages=1664-1675&author=E.+Awoonor-Williamsauthor=A.+G.+Walshauthor=C.+N.+Rowley&title=Modeling+covalent-modifier+drugs&doi=10.1016%2Fj.bbapap.2017.05.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Modeling covalent-modifier drugs</span></div><div class="casAuthors">Awoonor-Williams, Ernest; Walsh, Andrew G.; Rowley, Christopher N.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Proteins and Proteomics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">1865</span>
        (<span class="NLM_cas:issue">11_Part_B</span>),
    <span class="NLM_cas:pages">1664-1675</span>CODEN:
                <span class="NLM_cas:coden">BBAPBW</span>;
        ISSN:<span class="NLM_cas:issn">1570-9639</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  In this review, we present a summary of how computer modeling has been used in the development of covalent-modifier drugs.  Covalent-modifier drugs bind by forming a chem. bond with their target.  This covalent binding can improve the selectivity of the drug for a target with complementary reactivity and result in increased binding affinities due to the strength of the covalent bond formed.  In some cases, this results in irreversible inhibition of the target, but some targeted covalent inhibitor (TCI) drugs bind covalently but reversibly.  Computer modeling is widely used in drug discovery, but different computational methods must be used to model covalent modifiers because of the chem. bonds formed.  Structural and bioinformatic anal. has identified sites of modification that could yield selectivity for a chosen target.  Docking methods, which are used to rank binding poses of large sets of inhibitors, have been augmented to support the formation of protein-ligand bonds and are now capable of predicting the binding pose of covalent modifiers accurately.  The pKa's of amino acids can be calcd. in order to assess their reactivity towards electrophiles.  QM/MM methods have been used to model the reaction mechanisms of covalent modification.  The continued development of these tools will allow computation to aid in the development of new covalent-modifier drugs.  This article is part of a Special Issue entitled: Biophysics in Canada, edited by Lewis Kay, John Baenziger, Albert Berghuis and Peter Tieleman.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpyPHPoiqcVk7Vg90H21EOLACvtfcHk0lhaf59pDgYCrA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXotVOrtbo%253D&md5=ac579900da99ea005d0d562b53a1bca2</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.bbapap.2017.05.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbapap.2017.05.009%26sid%3Dliteratum%253Aachs%26aulast%3DAwoonor-Williams%26aufirst%3DE.%26aulast%3DWalsh%26aufirst%3DA.%2BG.%26aulast%3DRowley%26aufirst%3DC.%2BN.%26atitle%3DModeling%2520covalent-modifier%2520drugs%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Proteins%2520Proteomics%26date%3D2017%26volume%3D1865%26spage%3D1664%26epage%3D1675%26doi%3D10.1016%2Fj.bbapap.2017.05.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bradshaw, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McFarland, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paavilainen, V. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bisconte, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tam, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phan, V. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romanov, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finkle, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ton, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loughhead, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nunn, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karr, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerritsen, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Funk, J. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Owens, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verner, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brameld, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldstein, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taunton, J.</span></span> <span> </span><span class="NLM_article-title">Prolonged and tunable residence time using reversible covalent kinase inhibitors</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">525</span>– <span class="NLM_lpage">531</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1817</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=10.1038%2Fnchembio.1817" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=26006010" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFeju7%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=525-531&author=J.+M.+Bradshawauthor=J.+M.+McFarlandauthor=V.+O.+Paavilainenauthor=A.+Bisconteauthor=D.+Tamauthor=V.+T.+Phanauthor=S.+Romanovauthor=D.+Finkleauthor=J.+Shuauthor=V.+Patelauthor=T.+Tonauthor=X.+Liauthor=D.+G.+Loughheadauthor=P.+A.+Nunnauthor=D.+E.+Karrauthor=M.+E.+Gerritsenauthor=J.+O.+Funkauthor=T.+D.+Owensauthor=E.+Vernerauthor=K.+A.+Brameldauthor=R.+J.+Hillauthor=D.+M.+Goldsteinauthor=J.+Taunton&title=Prolonged+and+tunable+residence+time+using+reversible+covalent+kinase+inhibitors&doi=10.1038%2Fnchembio.1817"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Prolonged and tunable residence time using reversible covalent kinase inhibitors</span></div><div class="casAuthors">Bradshaw, J. Michael; McFarland, Jesse M.; Paavilainen, Ville O.; Bisconte, Angelina; Tam, Danny; Phan, Vernon T.; Romanov, Sergei; Finkle, David; Shu, Jin; Patel, Vaishali; Ton, Tony; Li, Xiaoyan; Loughhead, David G.; Nunn, Philip A.; Karr, Dane E.; Gerritsen, Mary E.; Funk, Jens Oliver; Owens, Timothy D.; Verner, Erik; Brameld, Ken A.; Hill, Ronald J.; Goldstein, David M.; Taunton, Jack</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">525-531</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Drugs with prolonged on-target residence times often show superior efficacy, yet general strategies for optimizing drug-target residence time are lacking.  Here the authors made progress toward this elusive goal by targeting a noncatalytic cysteine in Bruton's tyrosine kinase (BTK) with reversible covalent inhibitors.  Using an inverted orientation of the cysteine-reactive cyanoacrylamide electrophile, the authors identified potent and selective BTK inhibitors that demonstrated biochem. residence times spanning from minutes to 7 d.  An inverted cyanoacrylamide with prolonged residence time in vivo remained bound to BTK for more than 18 h after clearance from the circulation.  The inverted cyanoacrylamide strategy was further used to discover fibroblast growth factor receptor (FGFR) kinase inhibitors with residence times of several days, demonstrating the generalizability of the approach.  Targeting of noncatalytic cysteines with inverted cyanoacrylamides may serve as a broadly applicable platform that facilitates 'residence time by design', the ability to modulate and improve the duration of target engagement in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXCpNprhn6kLVg90H21EOLACvtfcHk0lieMTMH9OJoWA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFeju7%252FE&md5=85a73d9ecd62695d03166f7009c68dd8</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1817&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1817%26sid%3Dliteratum%253Aachs%26aulast%3DBradshaw%26aufirst%3DJ.%2BM.%26aulast%3DMcFarland%26aufirst%3DJ.%2BM.%26aulast%3DPaavilainen%26aufirst%3DV.%2BO.%26aulast%3DBisconte%26aufirst%3DA.%26aulast%3DTam%26aufirst%3DD.%26aulast%3DPhan%26aufirst%3DV.%2BT.%26aulast%3DRomanov%26aufirst%3DS.%26aulast%3DFinkle%26aufirst%3DD.%26aulast%3DShu%26aufirst%3DJ.%26aulast%3DPatel%26aufirst%3DV.%26aulast%3DTon%26aufirst%3DT.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DLoughhead%26aufirst%3DD.%2BG.%26aulast%3DNunn%26aufirst%3DP.%2BA.%26aulast%3DKarr%26aufirst%3DD.%2BE.%26aulast%3DGerritsen%26aufirst%3DM.%2BE.%26aulast%3DFunk%26aufirst%3DJ.%2BO.%26aulast%3DOwens%26aufirst%3DT.%2BD.%26aulast%3DVerner%26aufirst%3DE.%26aulast%3DBrameld%26aufirst%3DK.%2BA.%26aulast%3DHill%26aufirst%3DR.%2BJ.%26aulast%3DGoldstein%26aufirst%3DD.%2BM.%26aulast%3DTaunton%26aufirst%3DJ.%26atitle%3DProlonged%2520and%2520tunable%2520residence%2520time%2520using%2520reversible%2520covalent%2520kinase%2520inhibitors%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26spage%3D525%26epage%3D531%26doi%3D10.1038%2Fnchembio.1817" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smith, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uchida, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taunton, J.</span></span> <span> </span><span class="NLM_article-title">Essential biphasic role for JAK3 catalytic activity in IL-2 receptor signaling</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">373</span>– <span class="NLM_lpage">379</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2056</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=10.1038%2Fnchembio.2056" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=27018889" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=1%3ACAS%3A528%3ADC%252BC28XksFCru74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=373-379&author=G.+A.+Smithauthor=K.+Uchidaauthor=A.+Weissauthor=J.+Taunton&title=Essential+biphasic+role+for+JAK3+catalytic+activity+in+IL-2+receptor+signaling&doi=10.1038%2Fnchembio.2056"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Essential biphasic role for JAK3 catalytic activity in IL-2 receptor signaling</span></div><div class="casAuthors">Smith, Geoffrey A.; Uchida, Kenji; Weiss, Arthur; Taunton, Jack</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">373-379</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">To drive lymphocyte proliferation and differentiation, common γ-chain (γc) cytokine receptors require hours to days of sustained stimulation.  JAK1 and JAK3 kinases are found together in all γc-receptor complexes, but how their resp. catalytic activities contribute to signaling over time is not known.  Here we dissect the temporal requirements for JAK3 kinase activity with a selective covalent inhibitor (JAK3i).  By monitoring phosphorylation of the transcription factor STAT5 over 20 h in CD4+ T cells stimulated with interleukin 2 (IL-2), we document a second wave of signaling that is much more sensitive to JAK3i than the first wave.  Selective inhibition of this second wave is sufficient to block cyclin expression and entry to S phase.  An inhibitor-resistant JAK3 mutant (C905S) rescued all effects of JAK3i in isolated T cells and in mice.  Our chem. genetic toolkit elucidates a biphasic requirement for JAK3 kinase activity in IL-2-driven T cell proliferation and will find broad utility in studies of γc-receptor signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonfXNNV1cRJbVg90H21EOLACvtfcHk0lieMTMH9OJoWA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XksFCru74%253D&md5=573d99767ce37cf58a8f71e9e01102b5</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2056&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2056%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DG.%2BA.%26aulast%3DUchida%26aufirst%3DK.%26aulast%3DWeiss%26aufirst%3DA.%26aulast%3DTaunton%26aufirst%3DJ.%26atitle%3DEssential%2520biphasic%2520role%2520for%2520JAK3%2520catalytic%2520activity%2520in%2520IL-2%2520receptor%2520signaling%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2016%26volume%3D12%26spage%3D373%26epage%3D379%26doi%3D10.1038%2Fnchembio.2056" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguez-Molina, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tseng, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simonett, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taunton, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ansari, A. Z.</span></span> <span> </span><span class="NLM_article-title">Engineered covalent inactivation of TFIIH-kinase reveals an elongation checkpoint and results in widespread mRNA stabilization</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">433</span>– <span class="NLM_lpage">444</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2016.06.036</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=10.1016%2Fj.molcel.2016.06.036" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2016&pages=433-444&author=J.+B.+Rodriguez-Molinaauthor=S.+C.+Tsengauthor=S.+P.+Simonettauthor=J.+Tauntonauthor=A.+Z.+Ansari&title=Engineered+covalent+inactivation+of+TFIIH-kinase+reveals+an+elongation+checkpoint+and+results+in+widespread+mRNA+stabilization&doi=10.1016%2Fj.molcel.2016.06.036"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2016.06.036&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2016.06.036%26sid%3Dliteratum%253Aachs%26aulast%3DRodriguez-Molina%26aufirst%3DJ.%2BB.%26aulast%3DTseng%26aufirst%3DS.%2BC.%26aulast%3DSimonett%26aufirst%3DS.%2BP.%26aulast%3DTaunton%26aufirst%3DJ.%26aulast%3DAnsari%26aufirst%3DA.%2BZ.%26atitle%3DEngineered%2520covalent%2520inactivation%2520of%2520TFIIH-kinase%2520reveals%2520an%2520elongation%2520checkpoint%2520and%2520results%2520in%2520widespread%2520mRNA%2520stabilization%26jtitle%3DMol.%2520Cell%26date%3D2016%26volume%3D63%26spage%3D433%26epage%3D444%26doi%3D10.1016%2Fj.molcel.2016.06.036" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Butler, K. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tempel, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Babault, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pittella-Silva, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Structure-based design of a covalent inhibitor of the SET domain-containing protein 8 (SETD8) lysine methyltransferase</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">9881</span>– <span class="NLM_lpage">9889</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01244</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01244" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=1%3ACAS%3A528%3ADC%252BC28XhslKjtLnE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=9881-9889&author=K.+V.+Butlerauthor=A.+Maauthor=W.+Yuauthor=F.+Liauthor=W.+Tempelauthor=N.+Babaultauthor=F.+Pittella-Silvaauthor=J.+Shaoauthor=J.+Wangauthor=M.+Luoauthor=M.+Vedadiauthor=P.+J.+Brownauthor=C.+H.+Arrowsmithauthor=J.+Jin&title=Structure-based+design+of+a+covalent+inhibitor+of+the+SET+domain-containing+protein+8+%28SETD8%29+lysine+methyltransferase&doi=10.1021%2Facs.jmedchem.6b01244"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Design of a Covalent Inhibitor of the SET Domain-Containing Protein 8 (SETD8) Lysine Methyltransferase</span></div><div class="casAuthors">Butler, Kyle V.; Ma, Anqi; Yu, Wenyu; Li, Fengling; Tempel, Wolfram; Babault, Nicolas; Pittella-Silva, Fabio; Shao, Jason; Wang, Junyi; Luo, Minkui; Vedadi, Masoud; Brown, Peter J.; Arrowsmith, Cheryl H.; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">9881-9889</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Selective inhibitors of protein lysine methyltransferases, including SET domain-contg. protein 8 (SETD8), are highly desired, as only a fraction of these enzymes are assocd. with high-quality inhibitors.  From our previously discovered SETD8 inhibitor, we developed a more potent analog and solved a cocrystal structure, which is the first crystal structure of SETD8 in complex with a small-mol. inhibitor.  This cocrystal structure allowed the design of a covalent inhibitor of SETD8 (MS453), which specifically modifies a cysteine residue near the inhibitor binding site, has an IC50 value of 804 nM, reacts with SETD8 with near-quant. yield, and is selective for SETD8 against 28 other methyltransferases.  We also solved the crystal structure of the covalent inhibitor in complex with SETD8.  This work provides at.-level perspective on the inhibition of SETD8 by small mols. and will help identify high-quality chem. probes of SETD8.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpjPurua1qCK7Vg90H21EOLACvtfcHk0lj9gqjddo_-Eg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhslKjtLnE&md5=b4d0b38dfe3a911633aad17642e78134</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01244&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01244%26sid%3Dliteratum%253Aachs%26aulast%3DButler%26aufirst%3DK.%2BV.%26aulast%3DMa%26aufirst%3DA.%26aulast%3DYu%26aufirst%3DW.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DTempel%26aufirst%3DW.%26aulast%3DBabault%26aufirst%3DN.%26aulast%3DPittella-Silva%26aufirst%3DF.%26aulast%3DShao%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DLuo%26aufirst%3DM.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DStructure-based%2520design%2520of%2520a%2520covalent%2520inhibitor%2520of%2520the%2520SET%2520domain-containing%2520protein%25208%2520%2528SETD8%2529%2520lysine%2520methyltransferase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D9881%26epage%3D9889%26doi%3D10.1021%2Facs.jmedchem.6b01244" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Horton, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engstrom, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zoeller, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shanks, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johns, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vertino, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, X.</span></span> <span> </span><span class="NLM_article-title">Characterization of a linked jumonji domain of the KDM5/JARID1 family of histone H3 lysine 4 demethylases</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>291</i></span>,  <span class="NLM_fpage">2631</span>– <span class="NLM_lpage">2646</span>, <span class="refDoi"> DOI: 10.1074/jbc.M115.698449</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=10.1074%2Fjbc.M115.698449" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=26645689" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVOqtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=291&publication_year=2016&pages=2631-2646&author=J.+R.+Hortonauthor=A.+Engstromauthor=E.+L.+Zoellerauthor=X.+Liuauthor=J.+R.+Shanksauthor=X.+Zhangauthor=M.+A.+Johnsauthor=P.+M.+Vertinoauthor=H.+Fuauthor=X.+Cheng&title=Characterization+of+a+linked+jumonji+domain+of+the+KDM5%2FJARID1+family+of+histone+H3+lysine+4+demethylases&doi=10.1074%2Fjbc.M115.698449"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of a Linked Jumonji Domain of the KDM5/JARID1 Family of Histone H3 Lysine 4 Demethylases</span></div><div class="casAuthors">Horton, John R.; Engstrom, Amanda; Zoeller, Elizabeth L.; Liu, Xu; Shanks, John R.; Zhang, Xing; Johns, Margaret A.; Vertino, Paula M.; Fu, Haian; Cheng, Xiaodong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">291</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2631-2646</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The KDM5/JARID1 family of Fe(II)- and α-ketoglutarate-dependent demethylases remove Me groups from tri- and dimethylated lysine 4 of histone H3.  Accumulating evidence from primary tumors and model systems supports a role for KDM5A (JARID1A/RBP2) and KDM5B (JARID1B/PLU1) as oncogenic drivers.  The KDM5 family is unique among the Jumonji domain-contg. histone demethylases in that there is an atypical insertion of a DNA-binding ARID domain and a histone-binding PHD domain into the Jumonji domain, which separates the catalytic domain into two fragments (JmjN and JmjC).  Here we demonstrate that internal deletion of the ARID and PHD1 domains has a negligible effect on in vitro enzymic kinetics of the KDM5 family of enzymes.  We present a crystal structure of the linked JmjN-JmjC domain from KDM5A, which reveals that the linked domain fully reconstitutes the cofactor (metal ion and α-ketoglutarate) binding characteristics of other structurally characterized Jumonji domain demethylases.  Docking studies with GSK-J1, a selective inhibitor of the KDM6/KDM5 subfamilies, identify crit. residues for binding of the inhibitor to the reconstituted KDM5 Jumonji domain.  Further, we found that GSK-J1 inhibited the demethylase activity of KDM5C with 8.5-fold increased potency compared with that of KDM5B at 1 mm α-ketoglutarate.  In contrast, JIB-04 (a pan-inhibitor of the Jumonji demethylase superfamily) had the opposite effect and was ∼8-fold more potent against KDM5B than against KDM5C.  Interestingly, the relative selectivity of JIB-04 toward KDM5B over KDM5C in vitro translates to a ∼10-50-fold greater growth-inhibitory activity against breast cancer cell lines.  These data define the minimal requirements for enzymic activity of the KDM5 family to be the linked JmjN-JmjC domain coupled with the immediate C-terminal helical zinc-binding domain and provide structural characterization of the linked JmjN-JmjC domain for the KDM5 family, which should prove useful in the design of KDM5 demethylase inhibitors with improved potency and selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpa4qqOk0zHF7Vg90H21EOLACvtfcHk0lj9gqjddo_-Eg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitVOqtr0%253D&md5=772a106c3c7f0a45c1b3d2ee81cc9487</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M115.698449&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M115.698449%26sid%3Dliteratum%253Aachs%26aulast%3DHorton%26aufirst%3DJ.%2BR.%26aulast%3DEngstrom%26aufirst%3DA.%26aulast%3DZoeller%26aufirst%3DE.%2BL.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DShanks%26aufirst%3DJ.%2BR.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DJohns%26aufirst%3DM.%2BA.%26aulast%3DVertino%26aufirst%3DP.%2BM.%26aulast%3DFu%26aufirst%3DH.%26aulast%3DCheng%26aufirst%3DX.%26atitle%3DCharacterization%2520of%2520a%2520linked%2520jumonji%2520domain%2520of%2520the%2520KDM5%252FJARID1%2520family%2520of%2520histone%2520H3%2520lysine%25204%2520demethylases%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2016%26volume%3D291%26spage%3D2631%26epage%3D2646%26doi%3D10.1074%2Fjbc.M115.698449" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kruidenier, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liddle, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Che, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joberty, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bantscheff, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bountra, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bridges, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diallo, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eberhard, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hutchinson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katso, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leveridge, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mander, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mosley, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramirez-Molina, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rowland, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schofield, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheppard, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swales, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tumber, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drewes, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oppermann, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, D. M.</span></span> <span> </span><span class="NLM_article-title">A selective jumonji H3K27 demethylase inhibitor modulates the proinflammatory macrophage response</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>488</i></span>,  <span class="NLM_fpage">404</span>– <span class="NLM_lpage">408</span>, <span class="refDoi"> DOI: 10.1038/nature11262</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=10.1038%2Fnature11262" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=22842901" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtFWjtbfI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=488&publication_year=2012&pages=404-408&author=L.+Kruidenierauthor=C.+W.+Chungauthor=Z.+Chengauthor=J.+Liddleauthor=K.+Cheauthor=G.+Jobertyauthor=M.+Bantscheffauthor=C.+Bountraauthor=A.+Bridgesauthor=H.+Dialloauthor=D.+Eberhardauthor=S.+Hutchinsonauthor=E.+Jonesauthor=R.+Katsoauthor=M.+Leveridgeauthor=P.+K.+Manderauthor=J.+Mosleyauthor=C.+Ramirez-Molinaauthor=P.+Rowlandauthor=C.+J.+Schofieldauthor=R.+J.+Sheppardauthor=J.+E.+Smithauthor=C.+Swalesauthor=R.+Tannerauthor=P.+Thomasauthor=A.+Tumberauthor=G.+Drewesauthor=U.+Oppermannauthor=D.+J.+Patelauthor=K.+Leeauthor=D.+M.+Wilson&title=A+selective+jumonji+H3K27+demethylase+inhibitor+modulates+the+proinflammatory+macrophage+response&doi=10.1038%2Fnature11262"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">A selective jumonji H3K27 demethylase inhibitor modulates the proinflammatory macrophage response</span></div><div class="casAuthors">Kruidenier, Laurens; Chung, Chun-wa; Cheng, Zhongjun; Liddle, John; Che, Ka Hing; Joberty, Gerard; Bantscheff, Marcus; Bountra, Chas; Bridges, Angela; Diallo, Hawa; Eberhard, Dirk; Hutchinson, Sue; Jones, Emma; Katso, Roy; Leveridge, Melanie; Mander, Palwinder K.; Mosley, Julie; Ramirez-Molina, Cesar; Rowland, Paul; Schofield, Christopher J.; Sheppard, Robert J.; Smith, Julia E.; Swales, Catherine; Tanner, Robert; Thomas, Pamela; Tumber, Anthony; Drewes, Gerard; Oppermann, Udo; Patel, Dinshaw J.; Lee, Kevin; Wilson, David M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">488</span>
        (<span class="NLM_cas:issue">7411</span>),
    <span class="NLM_cas:pages">404-408</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The jumonji (JMJ) family of histone demethylases are Fe2+- and α-ketoglutarate-dependent oxygenases that are essential components of regulatory transcriptional chromatin complexes.  These enzymes demethylate lysine residues in histones in a methylation-state and sequence-specific context.  Considerable effort has been devoted to gaining a mechanistic understanding of the roles of histone lysine demethylases in eukaryotic transcription, genome integrity and epigenetic inheritance, as well as in development, physiol. and disease.  However, because of the absence of any selective inhibitors, the relevance of the demethylase activity of JMJ enzymes in regulating cellular responses remains poorly understood.  Here we present a structure-guided small-mol. and chemoproteomics approach to elucidating the functional role of the H3K27me3-specific demethylase subfamily (KDM6 subfamily members JMJD3 and UTX).  The liganded structures of human and mouse JMJD3 provide novel insight into the specificity determinants for cofactor, substrate and inhibitor recognition by the KDM6 subfamily of demethylases.  We exploited these structural features to generate the first small-mol. catalytic site inhibitor that is selective for the H3K27me3-specific JMJ subfamily.  We demonstrate that this inhibitor binds in a novel manner and reduces lipopolysaccharide-induced proinflammatory cytokine prodn. by human primary macrophages, a process that depends on both JMJD3 and UTX.  Our results resolve the ambiguity assocd. with the catalytic function of H3K27-specific JMJs in regulating disease-relevant inflammatory responses and provide encouragement for designing small-mol. inhibitors to allow selective pharmacol. intervention across the JMJ family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-lxSNmxjfsbVg90H21EOLACvtfcHk0limMbbmOJaJhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtFWjtbfI&md5=a4fa470a86070b4777e16a71e51763c8</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1038%2Fnature11262&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature11262%26sid%3Dliteratum%253Aachs%26aulast%3DKruidenier%26aufirst%3DL.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DCheng%26aufirst%3DZ.%26aulast%3DLiddle%26aufirst%3DJ.%26aulast%3DChe%26aufirst%3DK.%26aulast%3DJoberty%26aufirst%3DG.%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DBountra%26aufirst%3DC.%26aulast%3DBridges%26aufirst%3DA.%26aulast%3DDiallo%26aufirst%3DH.%26aulast%3DEberhard%26aufirst%3DD.%26aulast%3DHutchinson%26aufirst%3DS.%26aulast%3DJones%26aufirst%3DE.%26aulast%3DKatso%26aufirst%3DR.%26aulast%3DLeveridge%26aufirst%3DM.%26aulast%3DMander%26aufirst%3DP.%2BK.%26aulast%3DMosley%26aufirst%3DJ.%26aulast%3DRamirez-Molina%26aufirst%3DC.%26aulast%3DRowland%26aufirst%3DP.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26aulast%3DSheppard%26aufirst%3DR.%2BJ.%26aulast%3DSmith%26aufirst%3DJ.%2BE.%26aulast%3DSwales%26aufirst%3DC.%26aulast%3DTanner%26aufirst%3DR.%26aulast%3DThomas%26aufirst%3DP.%26aulast%3DTumber%26aufirst%3DA.%26aulast%3DDrewes%26aufirst%3DG.%26aulast%3DOppermann%26aufirst%3DU.%26aulast%3DPatel%26aufirst%3DD.%2BJ.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DWilson%26aufirst%3DD.%2BM.%26atitle%3DA%2520selective%2520jumonji%2520H3K27%2520demethylase%2520inhibitor%2520modulates%2520the%2520proinflammatory%2520macrophage%2520response%26jtitle%3DNature%26date%3D2012%26volume%3D488%26spage%3D404%26epage%3D408%26doi%3D10.1038%2Fnature11262" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Heinemann, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nielsen, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hudlebusch, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lees, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larsen, D. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boesen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Labelle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerlach, L. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Birk, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helin, K.</span></span> <span> </span><span class="NLM_article-title">Inhibition of demethylases by GSK-J1/J4</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>514</i></span>,  <span class="NLM_fpage">E1</span>– <span class="NLM_lpage">E2</span>, <span class="refDoi"> DOI: 10.1038/nature13688</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=10.1038%2Fnature13688" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=514&publication_year=2014&pages=E1-E2&author=B.+Heinemannauthor=J.+M.+Nielsenauthor=H.+R.+Hudlebuschauthor=M.+J.+Leesauthor=D.+V.+Larsenauthor=T.+Boesenauthor=M.+Labelleauthor=L.+O.+Gerlachauthor=P.+Birkauthor=K.+Helin&title=Inhibition+of+demethylases+by+GSK-J1%2FJ4&doi=10.1038%2Fnature13688"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1038%2Fnature13688&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature13688%26sid%3Dliteratum%253Aachs%26aulast%3DHeinemann%26aufirst%3DB.%26aulast%3DNielsen%26aufirst%3DJ.%2BM.%26aulast%3DHudlebusch%26aufirst%3DH.%2BR.%26aulast%3DLees%26aufirst%3DM.%2BJ.%26aulast%3DLarsen%26aufirst%3DD.%2BV.%26aulast%3DBoesen%26aufirst%3DT.%26aulast%3DLabelle%26aufirst%3DM.%26aulast%3DGerlach%26aufirst%3DL.%2BO.%26aulast%3DBirk%26aufirst%3DP.%26aulast%3DHelin%26aufirst%3DK.%26atitle%3DInhibition%2520of%2520demethylases%2520by%2520GSK-J1%252FJ4%26jtitle%3DNature%26date%3D2014%26volume%3D514%26spage%3DE1%26epage%3DE2%26doi%3D10.1038%2Fnature13688" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cee, V. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Volak, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartberger, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tegley, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arvedson, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCarter, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tasker, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fotsch, C.</span></span> <span> </span><span class="NLM_article-title">Systematic study of the glutathione (GSH) reactivity of N-arylacrylamides: 1. Effects of aryl substitution</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">9171</span>– <span class="NLM_lpage">9178</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01018</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01018" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFGrsLfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=9171-9178&author=V.+J.+Ceeauthor=L.+P.+Volakauthor=Y.+Chenauthor=M.+D.+Bartbergerauthor=C.+Tegleyauthor=T.+Arvedsonauthor=J.+McCarterauthor=A.+S.+Taskerauthor=C.+Fotsch&title=Systematic+study+of+the+glutathione+%28GSH%29+reactivity+of+N-arylacrylamides%3A+1.+Effects+of+aryl+substitution&doi=10.1021%2Facs.jmedchem.5b01018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Systematic Study of the Glutathione (GSH) Reactivity of N-Arylacrylamides: 1. Effects of Aryl Substitution</span></div><div class="casAuthors">Cee, Victor J.; Volak, Laurie P.; Chen, Yuping; Bartberger, Michael D.; Tegley, Chris; Arvedson, Tara; McCarter, John; Tasker, Andrew S.; Fotsch, Christopher</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">9171-9178</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Success in the design of targeted covalent inhibitors depends in part on a knowledge of the factors influencing electrophile reactivity.  In an effort to further develop an understanding of structure-reactivity relations among N-arylacrylamides, the authors detd. glutathione (GSH) reaction rates for a family of N-arylacrylamides independently substituted at ortho-, meta-, and para-positions with 11 different groups common to inhibitor design.  Substituent effects on reaction rates show a linear Hammett correlation for ortho-, meta-, and para-substitution.  The authors note a correlation between 1H and 13C NMR chem. shifts of the acrylamide with GSH reaction rates, suggesting that NMR chem. shifts may be a convenient surrogate measure of relative acrylamide reactivity.  D. functional theory calcns. reveal a correlation between computed activation parameters and exptl. detd. reaction rates, validating the use of such methodol. for the screening of synthetic candidates in a prospective fashion.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoAoYRfo0lRj7Vg90H21EOLACvtfcHk0limMbbmOJaJhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFGrsLfN&md5=2f3a816446dd73d5be061ad8d2195e2d</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01018%26sid%3Dliteratum%253Aachs%26aulast%3DCee%26aufirst%3DV.%2BJ.%26aulast%3DVolak%26aufirst%3DL.%2BP.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DBartberger%26aufirst%3DM.%2BD.%26aulast%3DTegley%26aufirst%3DC.%26aulast%3DArvedson%26aufirst%3DT.%26aulast%3DMcCarter%26aufirst%3DJ.%26aulast%3DTasker%26aufirst%3DA.%2BS.%26aulast%3DFotsch%26aufirst%3DC.%26atitle%3DSystematic%2520study%2520of%2520the%2520glutathione%2520%2528GSH%2529%2520reactivity%2520of%2520N-arylacrylamides%253A%25201.%2520Effects%2520of%2520aryl%2520substitution%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D9171%26epage%3D9178%26doi%3D10.1021%2Facs.jmedchem.5b01018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nicolaou, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Estrada, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zak, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Safina, B. S.</span></span> <span> </span><span class="NLM_article-title">A mild and selective method for the hydrolysis of esters with trimethyltin hydroxide</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">1378</span>– <span class="NLM_lpage">1382</span>, <span class="refDoi"> DOI: 10.1002/anie.200462207</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=10.1002%2Fanie.200462207" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=1%3ACAS%3A528%3ADC%252BD2MXitVSis7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2005&pages=1378-1382&author=K.+C.+Nicolaouauthor=A.+A.+Estradaauthor=M.+Zakauthor=S.+H.+Leeauthor=B.+S.+Safina&title=A+mild+and+selective+method+for+the+hydrolysis+of+esters+with+trimethyltin+hydroxide&doi=10.1002%2Fanie.200462207"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">A mild and selective method for the hydrolysis of esters with trimethyltin hydroxide</span></div><div class="casAuthors">Nicolaou, K. C.; Estrada, Anthony A.; Zak, Mark; Lee, Sang Hyup; Safina, Brian S.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1378-1382</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Mild, selective, and efficient: A method that involved the use of trimethyltin hydroxide for the hydrolysis of specific ester groups allowed chemists to steer clear of unwanted elimination reactions and epimerizations.  The conversion of an ester into the corresponding carboxylic acid took place under mild conditions, with nearly complete retention of stereochem. integrity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrz07978j08rLVg90H21EOLACvtfcHk0limMbbmOJaJhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXitVSis7s%253D&md5=d0bd143616d9f2ab81c0dcbbb3abf9c7</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1002%2Fanie.200462207&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.200462207%26sid%3Dliteratum%253Aachs%26aulast%3DNicolaou%26aufirst%3DK.%2BC.%26aulast%3DEstrada%26aufirst%3DA.%2BA.%26aulast%3DZak%26aufirst%3DM.%26aulast%3DLee%26aufirst%3DS.%2BH.%26aulast%3DSafina%26aufirst%3DB.%2BS.%26atitle%3DA%2520mild%2520and%2520selective%2520method%2520for%2520the%2520hydrolysis%2520of%2520esters%2520with%2520trimethyltin%2520hydroxide%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2005%26volume%3D44%26spage%3D1378%26epage%3D1382%26doi%3D10.1002%2Fanie.200462207" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Horowitz, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trievel, R. C.</span></span> <span> </span><span class="NLM_article-title">Carbon-oxygen hydrogen bonding in biological structure and function</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>287</i></span>,  <span class="NLM_fpage">41576</span>– <span class="NLM_lpage">41582</span>, <span class="refDoi"> DOI: 10.1074/jbc.R112.418574</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=10.1074%2Fjbc.R112.418574" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=23048026" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=1%3ACAS%3A528%3ADC%252BC38XhslyksrnP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=287&publication_year=2012&pages=41576-41582&author=S.+Horowitzauthor=R.+C.+Trievel&title=Carbon-oxygen+hydrogen+bonding+in+biological+structure+and+function&doi=10.1074%2Fjbc.R112.418574"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Carbon-Oxygen Hydrogen Bonding in Biological Structure and Function</span></div><div class="casAuthors">Horowitz, Scott; Trievel, Raymond C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">287</span>
        (<span class="NLM_cas:issue">50</span>),
    <span class="NLM_cas:pages">41576-41582</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">A review.  Carbon-oxygen (CH···O) hydrogen bonding represents an unusual category of mol. interactions first documented in biol. structures over 4 decades ago.  Although CH···O hydrogen bonding has remained generally underappreciated in the biochem. literature, studies over the last 15 years have begun to yield direct evidence of these interactions in biol. systems.  In this minireview, we provide a historical context of biol. CH···O hydrogen bonding and summarize some major advancements from exptl. studies over the past several years that have elucidated the importance, prevalence, and functions of these interactions.  In particular, we examine the impact of CH···O bonds on protein and nucleic acid structure, mol. recognition, and enzyme catalysis and conclude by exploring overarching themes and unresolved questions regarding unconventional interactions in biomol. structure.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHj0osWPCzV7Vg90H21EOLACvtfcHk0lgrw1PhZo8G-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhslyksrnP&md5=e96e7e30d5f5e03b0733fea323ecd6a2</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1074%2Fjbc.R112.418574&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.R112.418574%26sid%3Dliteratum%253Aachs%26aulast%3DHorowitz%26aufirst%3DS.%26aulast%3DTrievel%26aufirst%3DR.%2BC.%26atitle%3DCarbon-oxygen%2520hydrogen%2520bonding%2520in%2520biological%2520structure%2520and%2520function%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2012%26volume%3D287%26spage%3D41576%26epage%3D41582%26doi%3D10.1074%2Fjbc.R112.418574" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Deeks, E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keating, G. M.</span></span> <span> </span><span class="NLM_article-title">Afatinib in advanced NSCLC: a profile of its use</span>. <i>Drugs Ther Perspect</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">89</span>– <span class="NLM_lpage">98</span>, <span class="refDoi"> DOI: 10.1007/s40267-018-0482-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=10.1007%2Fs40267-018-0482-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=29540977" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=1%3ACAS%3A280%3ADC%252BC1MnhtlSgsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2018&pages=89-98&author=E.+D.+Deeksauthor=G.+M.+Keating&title=Afatinib+in+advanced+NSCLC%3A+a+profile+of+its+use&doi=10.1007%2Fs40267-018-0482-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Afatinib in advanced NSCLC: a profile of its use</span></div><div class="casAuthors">Deeks Emma D; Keating Gillian M</div><div class="citationInfo"><span class="NLM_cas:title">Drugs & therapy perspectives : for rational drug selection and use</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">89-98</span>
        ISSN:<span class="NLM_cas:issn">1172-0360</span>.
    </div><div class="casAbstract">Afatinib [Giotrif(®) (EU); Gilotrif(®) (USA)] is an orally administered, irreversible inhibitor of the ErbB family of tyrosine kinases that provides an important first-line treatment option for advanced non-small cell lung cancer (NSCLC) with activating epidermal growth factor receptor (EGFR) mutations (i.e.  EGFR(actMUT+)), and an additional treatment option for squamous NSCLC that has progressed following first-line platinum-based chemotherapy.  Relative to gefitinib in the first-line treatment of EGFR(actMUT+) advanced lung adenocarcinoma, afatinib prolonged progression-free survival (PFS) and time to treatment failure (TTF), but not overall survival (OS).  Afatinib also prolonged PFS, but not OS, versus cisplatin-based chemotherapy in this setting; however, afatinib improved OS versus chemotherapy in the subgroup of patients with deletions in exon 19.  As a second-line treatment for advanced squamous NSCLC, afatinib prolonged PFS and OS compared with erlotinib, regardless of EGFR mutation status.  Afatinib had a predictable and manageable tolerability profile.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQMwcu0ZW2rPGZyxfu64m4ffW6udTcc2ebhvko1EAc2M7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1MnhtlSgsw%253D%253D&md5=a1ea42cd72849bc384d59c3491ddbf4f</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1007%2Fs40267-018-0482-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs40267-018-0482-6%26sid%3Dliteratum%253Aachs%26aulast%3DDeeks%26aufirst%3DE.%2BD.%26aulast%3DKeating%26aufirst%3DG.%2BM.%26atitle%3DAfatinib%2520in%2520advanced%2520NSCLC%253A%2520a%2520profile%2520of%2520its%2520use%26jtitle%3DDrugs%2520Ther%2520Perspect%26date%3D2018%26volume%3D34%26spage%3D89%26epage%3D98%26doi%3D10.1007%2Fs40267-018-0482-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Solca, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dahl, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zoephel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bader, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanderson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kraemer, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Himmelsbach, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haaksma, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adolf, G. R.</span></span> <span> </span><span class="NLM_article-title">Target binding properties and cellular activity of afatinib (BIBW 2992), an irreversible ErbB family blocker</span>. <i>J. Pharmacol. Exp. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>343</i></span>,  <span class="NLM_fpage">342</span>– <span class="NLM_lpage">350</span>, <span class="refDoi"> DOI: 10.1124/jpet.112.197756</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=10.1124%2Fjpet.112.197756" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=22888144" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFKktbzE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=343&publication_year=2012&pages=342-350&author=F.+Solcaauthor=G.+Dahlauthor=A.+Zoephelauthor=G.+Baderauthor=M.+Sandersonauthor=C.+Kleinauthor=O.+Kraemerauthor=F.+Himmelsbachauthor=E.+Haaksmaauthor=G.+R.+Adolf&title=Target+binding+properties+and+cellular+activity+of+afatinib+%28BIBW+2992%29%2C+an+irreversible+ErbB+family+blocker&doi=10.1124%2Fjpet.112.197756"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Target binding properties and cellular activity of afatinib (BIBW 2992), an irreversible ErbB family blocker</span></div><div class="casAuthors">Solca, Flavio; Dahl, Goeran; Zoephel, Andreas; Bader, Gerd; Sanderson, Michael; Klein, Christian; Kraemer, Oliver; Himmelsbach, Frank; Haaksma, Eric; Adolf, Guenther R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmacology and Experimental Therapeutics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">343</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">342-350</span>CODEN:
                <span class="NLM_cas:coden">JPETAB</span>;
        ISSN:<span class="NLM_cas:issn">1521-0103</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Deregulation of the ErbB (proto-oncogene B of the avian erythroblastosis virus AEV-H strain) receptor network is well recognized as an oncogenic driver in epithelial cancers.  Several targeted drugs have been developed, including antibodies and small-mol. kinase inhibitors, each of them characterized by distinct patterns of ErbB receptor interactions.  Understanding the precise pharmacol. properties of these compds. is important for optimal use in clin. practice.  Afatinib [BIBW 2992; N-[4-[(3-chloro-4-fluorophenyl)amino]-7-[[(3S)-tetrahydro-3-furanyl]oxy]-6-quinazolinyl]-4-(dimethylamino)-2-butenamide] is an ATP-competitive anilinoquinazoline deriv. harboring a reactive acrylamide group.  It was designed to covalently bind and irreversibly block enzymically active ErbB receptor family members.  Here, we show by X-ray crystallog. the covalent binding of afatinib to wild-type epidermal growth factor receptor (EGFR) and by mass spectrometry the covalent interaction with EGFR, EGFRL858R/T790M, human epidermal growth factor receptor 2 (HER2), and ErbB-4.  Afatinib potently inhibits the enzymatic activity of ErbB-4 (EC50 = 1 nM) and the proliferation of cancer cell lines driven by multiple ErbB receptor aberrations at concns. below 100 nM.  N-[4-[(3-chloro-4-fluorophenyl)amino]-7-[[(3S)-tetrahydro-3-furanyl]oxy]-6-quinazolinyl]-4-(dimethylamino)-2-butanamide (BI 37781), a close analog of afatinib lacking the acrylamide group and thus incapable of covalent bond formation, had similar potency on cells driven by EGFR or EGFRL858R, but less or no detectable activity on cells expressing EGFRL858R/T790M HER2 or ErbB-4.  These results stress the importance of the acrylamide group and show that afatinib differs from approved ErbB targeting agents by irreversibly inhibiting the kinase activity of all ErbB family members.  They provide a mechanistic rationale for the distinct pharmacol. features of this compd. and explain the clin. activity seen in some patients who are resistant to antibody or kinase inhibitor therapy because of secondary mutations or ErbB receptor "reprogramming.".</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGofXgQqaKXv-bVg90H21EOLACvtfcHk0lgrw1PhZo8G-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFKktbzE&md5=4a57df32abe8101f287fc6251f63f92d</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1124%2Fjpet.112.197756&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fjpet.112.197756%26sid%3Dliteratum%253Aachs%26aulast%3DSolca%26aufirst%3DF.%26aulast%3DDahl%26aufirst%3DG.%26aulast%3DZoephel%26aufirst%3DA.%26aulast%3DBader%26aufirst%3DG.%26aulast%3DSanderson%26aufirst%3DM.%26aulast%3DKlein%26aufirst%3DC.%26aulast%3DKraemer%26aufirst%3DO.%26aulast%3DHimmelsbach%26aufirst%3DF.%26aulast%3DHaaksma%26aufirst%3DE.%26aulast%3DAdolf%26aufirst%3DG.%2BR.%26atitle%3DTarget%2520binding%2520properties%2520and%2520cellular%2520activity%2520of%2520afatinib%2520%2528BIBW%25202992%2529%252C%2520an%2520irreversible%2520ErbB%2520family%2520blocker%26jtitle%3DJ.%2520Pharmacol.%2520Exp.%2520Ther.%26date%3D2012%26volume%3D343%26spage%3D342%26epage%3D350%26doi%3D10.1124%2Fjpet.112.197756" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Alves, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vidugiris, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goueli, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zegzouti, H.</span></span> <span> </span><span class="NLM_article-title">Bioluminescent high-throughput succinate detection method for monitoring the activity of JMJC histone demethylases and Fe(II)/2-oxoglutarate-dependent dioxygenases</span>. <i>SLAS Discov</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">242</span>– <span class="NLM_lpage">254</span>, <span class="refDoi"> DOI: 10.1177/2472555217745657</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=10.1177%2F2472555217745657" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=29239273" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=1%3ACAS%3A528%3ADC%252BC1cXjtFCqtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2018&pages=242-254&author=J.+Alvesauthor=G.+Vidugirisauthor=S.+A.+Goueliauthor=H.+Zegzouti&title=Bioluminescent+high-throughput+succinate+detection+method+for+monitoring+the+activity+of+JMJC+histone+demethylases+and+Fe%28II%29%2F2-oxoglutarate-dependent+dioxygenases&doi=10.1177%2F2472555217745657"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Bioluminescent High-Throughput Succinate Detection Method for Monitoring the Activity of JMJC Histone Demethylases and Fe(II)/2-Oxoglutarate-Dependent Dioxygenases</span></div><div class="casAuthors">Alves, Juliano; Vidugiris, Gediminas; Goueli, Said A.; Zegzouti, Hicham</div><div class="citationInfo"><span class="NLM_cas:title">SLAS Discovery</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">242-254</span>CODEN:
                <span class="NLM_cas:coden">SDLIBT</span>;
        ISSN:<span class="NLM_cas:issn">2472-5560</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">The modification of a diverse array of substrates by Fe(II)/2-oxoglutarate-dependent dioxygenases is central to the modulation of distinct biol. processes such as epigenetics, hypoxic signaling, and DNA/RNA repair.  Of these, JumonjiC domain-contg. histone lysine demethylases (JMJCs) and prolyl hydroxylases are potential drug targets due to their relevance to human diseases.  Thus, assays to interrogate this enzyme superfamily are needed to identify selective and potent inhibitors as leads for drug development and that could also be useful research tools.  Since succinate is a common product to all Fe(II)/2-oxoglutarate-dependent dioxygenase reactions, a method that detects succinate would be suitable to all members of this enzyme superfamily.  We therefore developed a bioluminescent and homogenous succinate detection assay and validated its use with diverse sets of enzyme classes.  We evaluated the substrate specificities of these enzymes, their apparent kinetic consts., and inhibition profiles and mode of action of reported and novel inhibitors.  Our results indicate that succinate detection is a useful readout for the monitoring of enzymic activities with distinct substrate entities, as well as for the discovery of novel inhibitors.  By investigating a large no. of Fe(II)/2-oxoglutarate-dependent enzymes, this method could have a significant impact on the field of dioxygenase research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpvvqSTuW_ZHbVg90H21EOLACvtfcHk0ljPtm6rITjHCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXjtFCqtr0%253D&md5=5501db23b33e5b42578cb043a9da8d9d</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1177%2F2472555217745657&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F2472555217745657%26sid%3Dliteratum%253Aachs%26aulast%3DAlves%26aufirst%3DJ.%26aulast%3DVidugiris%26aufirst%3DG.%26aulast%3DGoueli%26aufirst%3DS.%2BA.%26aulast%3DZegzouti%26aufirst%3DH.%26atitle%3DBioluminescent%2520high-throughput%2520succinate%2520detection%2520method%2520for%2520monitoring%2520the%2520activity%2520of%2520JMJC%2520histone%2520demethylases%2520and%2520Fe%2528II%2529%252F2-oxoglutarate-dependent%2520dioxygenases%26jtitle%3DSLAS%2520Discov%26date%3D2018%26volume%3D23%26spage%3D242%26epage%3D254%26doi%3D10.1177%2F2472555217745657" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Silverman, R. B.</span></span> <span> </span><span class="NLM_article-title">Mechanism-based enzyme inactivators</span>. <i>Methods Enzymol.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>249</i></span>,  <span class="NLM_fpage">240</span>– <span class="NLM_lpage">283</span>, <span class="refDoi"> DOI: 10.1016/0076-6879(95)49038-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=10.1016%2F0076-6879%2895%2949038-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=7791614" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=1%3ACAS%3A528%3ADyaK2MXmvFSlurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=249&publication_year=1995&pages=240-283&author=R.+B.+Silverman&title=Mechanism-based+enzyme+inactivators&doi=10.1016%2F0076-6879%2895%2949038-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanism-based enzyme inactivators</span></div><div class="casAuthors">Silverman, Richard B.</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Enzymology</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">249</span>
        (<span class="NLM_cas:issue">Enzyme Kinetics and Mechanism, Part D</span>),
    <span class="NLM_cas:pages">240-83</span>CODEN:
                <span class="NLM_cas:coden">MENZAU</span>;
        ISSN:<span class="NLM_cas:issn">0076-6879</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">A review, with 31 refs., on the terminol., basic kinetics, and uses of enzyme inactivators; criteria and exptl. protocols for mechanism-based enzyme inactivators; and applications of enzyme inactivators with examples from the author's lab.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrMhLcBvU-U4bVg90H21EOLACvtfcHk0ljPtm6rITjHCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXmvFSlurw%253D&md5=775c81e5ce7d0548d3468fc60b8243f5</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2F0076-6879%2895%2949038-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0076-6879%252895%252949038-8%26sid%3Dliteratum%253Aachs%26aulast%3DSilverman%26aufirst%3DR.%2BB.%26atitle%3DMechanism-based%2520enzyme%2520inactivators%26jtitle%3DMethods%2520Enzymol.%26date%3D1995%26volume%3D249%26spage%3D240%26epage%3D283%26doi%3D10.1016%2F0076-6879%2895%2949038-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerns, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jadhav, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, X.</span></span> <span> </span><span class="NLM_article-title">Highly predictive and interpretable models for PAMPA permeability</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1266</span>– <span class="NLM_lpage">1276</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2016.12.049</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=10.1016%2Fj.bmc.2016.12.049" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=28082071" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=1%3ACAS%3A528%3ADC%252BC2sXntlOhsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2017&pages=1266-1276&author=H.+Sunauthor=K.+Nguyenauthor=E.+Kernsauthor=Z.+Yanauthor=K.+R.+Yuauthor=P.+Shahauthor=A.+Jadhavauthor=X.+Xu&title=Highly+predictive+and+interpretable+models+for+PAMPA+permeability&doi=10.1016%2Fj.bmc.2016.12.049"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Highly predictive and interpretable models for PAMPA permeability</span></div><div class="casAuthors">Sun, Hongmao; Nguyen, Kimloan; Kerns, Edward; Yan, Zhengyin; Yu, Kyeong Ri; Shah, Pranav; Jadhav, Ajit; Xu, Xin</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1266-1276</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Cell membrane permeability is an important determinant for oral absorption and bioavailability of a drug mol.  An in silico model predicting drug permeability is described, which is built based on a large permeability dataset of 7488 compd. entries or 5435 structurally unique mols. measured by the same lab using parallel artificial membrane permeability assay (PAMPA).  On the basis of customized mol. descriptors, the support vector regression (SVR) model trained with 4071 compds. with quant. data is able to predict the remaining 1364 compds. with the qual. data with an area under the curve of receiver operating characteristic (AUC-ROC) of 0.90.  The support vector classification (SVC) model trained with half of the whole dataset comprised of both the quant. and the qual. data produced accurate predictions to the remaining data with the AUC-ROC of 0.88.  The results suggest that the developed SVR model is highly predictive and provides medicinal chemists a useful in silico tool to facilitate design and synthesis of novel compds. with optimal drug-like properties, and thus accelerate the lead optimization in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-y8FhTLLqbrVg90H21EOLACvtfcHk0ljPtm6rITjHCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXntlOhsA%253D%253D&md5=2f6b60dfcd168b5bf837405ca52457b2</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2016.12.049&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2016.12.049%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DH.%26aulast%3DNguyen%26aufirst%3DK.%26aulast%3DKerns%26aufirst%3DE.%26aulast%3DYan%26aufirst%3DZ.%26aulast%3DYu%26aufirst%3DK.%2BR.%26aulast%3DShah%26aufirst%3DP.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DXu%26aufirst%3DX.%26atitle%3DHighly%2520predictive%2520and%2520interpretable%2520models%2520for%2520PAMPA%2520permeability%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2017%26volume%3D25%26spage%3D1266%26epage%3D1276%26doi%3D10.1016%2Fj.bmc.2016.12.049" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lang, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horton, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jansen, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mott, B. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pohida, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rai, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kales, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henderson, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jadhav, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maloney, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simeonov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwasaki, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shadel, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, Q.</span></span> <span> </span><span class="NLM_article-title">KDM5 histone demethylases repress immune response via suppression of STING</span>. <i>PLoS Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">e2006134</span>, <span class="refDoi"> DOI: 10.1371/journal.pbio.2006134</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=10.1371%2Fjournal.pbio.2006134" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2018&pages=e2006134&author=L.+Wuauthor=J.+Caoauthor=W.+L.+Caiauthor=S.+M.+Langauthor=J.+R.+Hortonauthor=D.+J.+Jansenauthor=Z.+Z.+Liuauthor=J.+F.+Chenauthor=M.+Zhangauthor=B.+T.+Mottauthor=K.+Pohidaauthor=G.+Raiauthor=S.+C.+Kalesauthor=M.+J.+Hendersonauthor=X.+Huauthor=A.+Jadhavauthor=D.+J.+Maloneyauthor=A.+Simeonovauthor=S.+Zhuauthor=A.+Iwasakiauthor=M.+D.+Hallauthor=X.+Chengauthor=G.+S.+Shadelauthor=Q.+Yan&title=KDM5+histone+demethylases+repress+immune+response+via+suppression+of+STING&doi=10.1371%2Fjournal.pbio.2006134"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pbio.2006134&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pbio.2006134%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DL.%26aulast%3DCao%26aufirst%3DJ.%26aulast%3DCai%26aufirst%3DW.%2BL.%26aulast%3DLang%26aufirst%3DS.%2BM.%26aulast%3DHorton%26aufirst%3DJ.%2BR.%26aulast%3DJansen%26aufirst%3DD.%2BJ.%26aulast%3DLiu%26aufirst%3DZ.%2BZ.%26aulast%3DChen%26aufirst%3DJ.%2BF.%26aulast%3DZhang%26aufirst%3DM.%26aulast%3DMott%26aufirst%3DB.%2BT.%26aulast%3DPohida%26aufirst%3DK.%26aulast%3DRai%26aufirst%3DG.%26aulast%3DKales%26aufirst%3DS.%2BC.%26aulast%3DHenderson%26aufirst%3DM.%2BJ.%26aulast%3DHu%26aufirst%3DX.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DMaloney%26aufirst%3DD.%2BJ.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DZhu%26aufirst%3DS.%26aulast%3DIwasaki%26aufirst%3DA.%26aulast%3DHall%26aufirst%3DM.%2BD.%26aulast%3DCheng%26aufirst%3DX.%26aulast%3DShadel%26aufirst%3DG.%2BS.%26aulast%3DYan%26aufirst%3DQ.%26atitle%3DKDM5%2520histone%2520demethylases%2520repress%2520immune%2520response%2520via%2520suppression%2520of%2520STING%26jtitle%3DPLoS%2520Biol.%26date%3D2018%26volume%3D16%26spage%3De2006134%26doi%3D10.1371%2Fjournal.pbio.2006134" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wright, S. W.</span></span> <span> </span><span class="NLM_article-title">Preparation of 2-, 4-, 5-, and 6-aminonicotinic acid tert-butyl esters</span>. <i>J. Heterocyclic Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">442</span>– <span class="NLM_lpage">445</span>, <span class="refDoi"> DOI: 10.1002/jhet.799</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=10.1002%2Fjhet.799" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=1%3ACAS%3A528%3ADC%252BC38XmtFeis7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2012&pages=442-445&author=S.+W.+Wright&title=Preparation+of+2-%2C+4-%2C+5-%2C+and+6-aminonicotinic+acid+tert-butyl+esters&doi=10.1002%2Fjhet.799"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Preparation of 2-, 4-, 5-, and 6-aminonicotinic acid tert-butyl esters</span></div><div class="casAuthors">Wright, Stephen W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Heterocyclic Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">442-445</span>CODEN:
                <span class="NLM_cas:coden">JHTCAD</span>;
        ISSN:<span class="NLM_cas:issn">1943-5193</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Procedures are reported to prep. the tert-Bu esters of 2-aminonicotinic acid, 4-aminonicotinic acid, 5-aminonicotinic acid, and 6-aminonicotinic acid from 2-chloronicotinic acid, 4-chloronicotinic acid, 5-bromonicotinic acid, and 6-chloronicotinic acid, resp., without need for purifn. of intermediates.  J. Heterocyclic Chem., (2012).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgRevQmfJ-LrVg90H21EOLACvtfcHk0ljssJQdtDJzxg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XmtFeis7c%253D&md5=87557ea8fd125a1c42575da0472d79c6</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1002%2Fjhet.799&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjhet.799%26sid%3Dliteratum%253Aachs%26aulast%3DWright%26aufirst%3DS.%2BW.%26atitle%3DPreparation%2520of%25202-%252C%25204-%252C%25205-%252C%2520and%25206-aminonicotinic%2520acid%2520tert-butyl%2520esters%26jtitle%3DJ.%2520Heterocyclic%2520Chem.%26date%3D2012%26volume%3D49%26spage%3D442%26epage%3D445%26doi%3D10.1002%2Fjhet.799" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krishnamurthy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senanayake, C. H.</span></span> <span> </span><span class="NLM_article-title">Noncryogenic I/Br-Mg exchange of aromatic halides bearing sensitive functional groups using i-PrMgCl-bis[2-(N,N-dimethylamino)ethyl] ether complexes</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">305</span>– <span class="NLM_lpage">307</span>, <span class="refDoi"> DOI: 10.1021/ol052704p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ol052704p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2006&pages=305-307&author=X.+J.+Wangauthor=X.+Sunauthor=L.+Zhangauthor=Y.+Xuauthor=D.+Krishnamurthyauthor=C.+H.+Senanayake&title=Noncryogenic+I%2FBr-Mg+exchange+of+aromatic+halides+bearing+sensitive+functional+groups+using+i-PrMgCl-bis%5B2-%28N%2CN-dimethylamino%29ethyl%5D+ether+complexes&doi=10.1021%2Fol052704p"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Fol052704p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fol052704p%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DX.%2BJ.%26aulast%3DSun%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DKrishnamurthy%26aufirst%3DD.%26aulast%3DSenanayake%26aufirst%3DC.%2BH.%26atitle%3DNoncryogenic%2520I%252FBr-Mg%2520exchange%2520of%2520aromatic%2520halides%2520bearing%2520sensitive%2520functional%2520groups%2520using%2520i-PrMgCl-bis%255B2-%2528N%252CN-dimethylamino%2529ethyl%255D%2520ether%2520complexes%26jtitle%3DOrg.%2520Lett.%26date%3D2006%26volume%3D8%26spage%3D305%26epage%3D307%26doi%3D10.1021%2Fol052704p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Horton, J. R.</span>; <span class="NLM_string-name">Gale, M.</span>; <span class="NLM_string-name">Yan, Q.</span>; <span class="NLM_string-name">Cheng, X.</span></span> <span> </span><span class="NLM_article-title">The molecular basis of histone demethylation</span>. In  <i>DNA and Histone Methylation as Cancer Targets</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaneda, A.</span>, <span class="NLM_string-name">Tsukada, Y.-i.</span></span>, Eds.; <span class="NLM_publisher-name">Cancer Drug Discovery and Development</span>, <span class="NLM_year">2017</span>; pp  <span class="NLM_fpage">151</span>– <span class="NLM_lpage">219</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=10.1007%2F978-3-319-59786-7_7" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&pages=151-219&author=J.+R.+Horton&author=M.+Gale&author=Q.+Yan&author=X.+Chengauthor=A.+Kaneda&author=Y.-i.+Tsukada&title=DNA+and+Histone+Methylation+as+Cancer+Targets"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1007%2F978-3-319-59786-7_7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F978-3-319-59786-7_7%26sid%3Dliteratum%253Aachs%26aulast%3DHorton%26aufirst%3DJ.%2BR.%26atitle%3DThe%2520molecular%2520basis%2520of%2520histone%2520demethylation%26btitle%3DDNA%2520and%2520Histone%2520Methylation%2520as%2520Cancer%2520Targets%26aulast%3DKaneda%26aufirst%3DA.%26pub%3DCancer%2520Drug%2520Discovery%2520and%2520Development%26date%3D2017%26spage%3D151%26epage%3D219" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lin, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beshiri, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujiwara, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Francis, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cherniack, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geisen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blair, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawamori, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grisanzio, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watanabe, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minamishima, Y. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kulkarni, R. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Signoretti, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodig, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bronson, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orkin, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tuck, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benevolenskaya, E. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyerson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaelin, W. G.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, Q.</span></span> <span> </span><span class="NLM_article-title">Loss of the retinoblastoma binding protein 2 (RBP2) histone demethylase suppresses tumorigenesis in mice lacking Rb1 or Men1</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>108</i></span>,  <span class="NLM_fpage">13379</span>– <span class="NLM_lpage">13386</span>, <span class="refDoi"> DOI: 10.1073/pnas.1110104108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=10.1073%2Fpnas.1110104108" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=108&publication_year=2011&pages=13379-13386&author=W.+Linauthor=J.+Caoauthor=J.+Liuauthor=M.+L.+Beshiriauthor=Y.+Fujiwaraauthor=J.+Francisauthor=A.+D.+Cherniackauthor=C.+Geisenauthor=L.+P.+Blairauthor=M.+R.+Zouauthor=X.+Shenauthor=D.+Kawamoriauthor=Z.+Liuauthor=C.+Grisanzioauthor=H.+Watanabeauthor=Y.+A.+Minamishimaauthor=Q.+Zhangauthor=R.+N.+Kulkarniauthor=S.+Signorettiauthor=S.+J.+Rodigauthor=R.+T.+Bronsonauthor=S.+H.+Orkinauthor=D.+P.+Tuckauthor=E.+V.+Benevolenskayaauthor=M.+Meyersonauthor=W.+G.+Kaelinauthor=Q.+Yan&title=Loss+of+the+retinoblastoma+binding+protein+2+%28RBP2%29+histone+demethylase+suppresses+tumorigenesis+in+mice+lacking+Rb1+or+Men1&doi=10.1073%2Fpnas.1110104108"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1110104108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1110104108%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DW.%26aulast%3DCao%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DBeshiri%26aufirst%3DM.%2BL.%26aulast%3DFujiwara%26aufirst%3DY.%26aulast%3DFrancis%26aufirst%3DJ.%26aulast%3DCherniack%26aufirst%3DA.%2BD.%26aulast%3DGeisen%26aufirst%3DC.%26aulast%3DBlair%26aufirst%3DL.%2BP.%26aulast%3DZou%26aufirst%3DM.%2BR.%26aulast%3DShen%26aufirst%3DX.%26aulast%3DKawamori%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DGrisanzio%26aufirst%3DC.%26aulast%3DWatanabe%26aufirst%3DH.%26aulast%3DMinamishima%26aufirst%3DY.%2BA.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DKulkarni%26aufirst%3DR.%2BN.%26aulast%3DSignoretti%26aufirst%3DS.%26aulast%3DRodig%26aufirst%3DS.%2BJ.%26aulast%3DBronson%26aufirst%3DR.%2BT.%26aulast%3DOrkin%26aufirst%3DS.%2BH.%26aulast%3DTuck%26aufirst%3DD.%2BP.%26aulast%3DBenevolenskaya%26aufirst%3DE.%2BV.%26aulast%3DMeyerson%26aufirst%3DM.%26aulast%3DKaelin%26aufirst%3DW.%2BG.%26aulast%3DYan%26aufirst%3DQ.%26atitle%3DLoss%2520of%2520the%2520retinoblastoma%2520binding%2520protein%25202%2520%2528RBP2%2529%2520histone%2520demethylase%2520suppresses%2520tumorigenesis%2520in%2520mice%2520lacking%2520Rb1%2520or%2520Men1%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2011%26volume%3D108%26spage%3D13379%26epage%3D13386%26doi%3D10.1073%2Fpnas.1110104108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheung, W. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Booth, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, D. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, Q.</span></span> <span> </span><span class="NLM_article-title">Histone demethylase RBP2 is critical for breast cancer progression and metastasis</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">868</span>– <span class="NLM_lpage">877</span>, <span class="refDoi"> DOI: 10.1016/j.celrep.2014.02.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=10.1016%2Fj.celrep.2014.02.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=24582965" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjsVWgsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2014&pages=868-877&author=J.+Caoauthor=Z.+Liuauthor=W.+K.+Cheungauthor=M.+Zhaoauthor=S.+Y.+Chenauthor=S.+W.+Chanauthor=C.+J.+Boothauthor=D.+X.+Nguyenauthor=Q.+Yan&title=Histone+demethylase+RBP2+is+critical+for+breast+cancer+progression+and+metastasis&doi=10.1016%2Fj.celrep.2014.02.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Histone Demethylase RBP2 Is Critical for Breast Cancer Progression and Metastasis</span></div><div class="casAuthors">Cao, Jian; Liu, Zongzhi; Cheung, William K. C.; Zhao, Minghui; Chen, Sophia Y.; Chan, Siew Wee; Booth, Carmen J.; Nguyen, Don X.; Yan, Qin</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">868-877</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Metastasis is a major clin. challenge for cancer treatment.  Emerging evidence suggests that aberrant epigenetic modifications contribute significantly to tumor formation and progression.  However, the drivers and roles of such epigenetic changes in tumor metastasis are still poorly understood.  Using bioinformatic anal. of human breast cancer gene-expression data sets, we identified histone demethylase RBP2 as a putative mediator of metastatic progression.  By using both human breast cancer cells and genetically engineered mice, we demonstrated that RBP2 is crit. for breast cancer metastasis to the lung in multiple in vivo models.  Mechanistically, RBP2 promotes metastasis as a pleiotropic pos. regulator of many metastasis genes, including TNC.  In addn., RBP2 loss suppresses tumor formation in MMTV-neu transgenic mice.  These results suggest that therapeutic targeting of RBP2 is a potential strategy for inhibition of tumor progression and metastasis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXZwTcPjZTYLVg90H21EOLACvtfcHk0li_IkdPalTAhg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjsVWgsro%253D&md5=ecc62cbf9e2bcab60746534aa536b87d</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2014.02.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2014.02.004%26sid%3Dliteratum%253Aachs%26aulast%3DCao%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DCheung%26aufirst%3DW.%2BK.%26aulast%3DZhao%26aufirst%3DM.%26aulast%3DChen%26aufirst%3DS.%2BY.%26aulast%3DChan%26aufirst%3DS.%2BW.%26aulast%3DBooth%26aufirst%3DC.%2BJ.%26aulast%3DNguyen%26aufirst%3DD.%2BX.%26aulast%3DYan%26aufirst%3DQ.%26atitle%3DHistone%2520demethylase%2520RBP2%2520is%2520critical%2520for%2520breast%2520cancer%2520progression%2520and%2520metastasis%26jtitle%3DCell%2520Rep.%26date%3D2014%26volume%3D6%26spage%3D868%26epage%3D877%26doi%3D10.1016%2Fj.celrep.2014.02.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dalvi, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kollipara, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bayo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yenerall, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Timmons, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguez-Canales, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Behrens, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mino, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villalobos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parra, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suraokar, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pataer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swisher, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalhor, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhanu, N. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heymach, J. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coombes, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Girard, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gazdar, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kittler, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wistuba,  II</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minna, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinez, E. D.</span></span> <span> </span><span class="NLM_article-title">Taxane-platin-resistant lung cancers co-develop hypersensitivity to JumonjiC demethylase inhibitors</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">1669</span>– <span class="NLM_lpage">1684</span>, <span class="refDoi"> DOI: 10.1016/j.celrep.2017.04.077</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=10.1016%2Fj.celrep.2017.04.077" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=28538184" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=1%3ACAS%3A528%3ADC%252BC2sXotlygtrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2017&pages=1669-1684&author=M.+P.+Dalviauthor=L.+Wangauthor=R.+Zhongauthor=R.+K.+Kolliparaauthor=H.+Parkauthor=J.+Bayoauthor=P.+Yenerallauthor=Y.+Zhouauthor=B.+C.+Timmonsauthor=J.+Rodriguez-Canalesauthor=C.+Behrensauthor=B.+Minoauthor=P.+Villalobosauthor=E.+R.+Parraauthor=M.+Suraokarauthor=A.+Pataerauthor=S.+G.+Swisherauthor=N.+Kalhorauthor=N.+V.+Bhanuauthor=B.+A.+Garciaauthor=J.+V.+Heymachauthor=K.+Coombesauthor=Y.+Xieauthor=L.+Girardauthor=A.+F.+Gazdarauthor=R.+Kittlerauthor=+Wistubaauthor=J.+D.+Minnaauthor=E.+D.+Martinez&title=Taxane-platin-resistant+lung+cancers+co-develop+hypersensitivity+to+JumonjiC+demethylase+inhibitors&doi=10.1016%2Fj.celrep.2017.04.077"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Taxane-Platin-Resistant Lung Cancers Co-develop Hypersensitivity to JumonjiC Demethylase Inhibitors</span></div><div class="casAuthors">Dalvi, Maithili P.; Wang, Lei; Zhong, Rui; Kollipara, Rahul K.; Park, Hyunsil; Bayo, Juan; Yenerall, Paul; Zhou, Yunyun; Timmons, Brenda C.; Rodriguez-Canales, Jaime; Behrens, Carmen; Mino, Barbara; Villalobos, Pamela; Parra, Edwin R.; Suraokar, Milind; Pataer, Apar; Swisher, Stephen G.; Kalhor, Neda; Bhanu, Natarajan V.; Garcia, Benjamin A.; Heymach, John V.; Coombes, Kevin; Xie, Yang; Girard, Luc; Gazdar, Adi F.; Kittler, Ralf; Wistuba, Ignacio I.; Minna, John D.; Martinez, Elisabeth D.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1669-1684</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Although non-small cell lung cancer (NSCLC) patients benefit from std. taxane-platin chemotherapy, many relapse, developing drug resistance.  We established preclin. taxane-platin-chemoresistance models and identified a 35-gene resistance signature, which was assocd. with poor recurrence-free survival in neoadjuvant-treated NSCLC patients and included upregulation of the JumonjiC lysine demethylase KDM3B.  In fact, multi-drug-resistant cells progressively increased the expression of many JumonjiC demethylases, had altered histone methylation, and, importantly, showed hypersensitivity to JumonjiC inhibitors in vitro and in vivo.  Increasing taxane-platin resistance in progressive cell line series was accompanied by progressive sensitization to JIB-04 and GSK-J4.  These JumonjiC inhibitors partly reversed deregulated transcriptional programs, prevented the emergence of drug-tolerant colonies from chemo-naive cells, and synergized with std. chemotherapy in vitro and in vivo.  Our findings reveal JumonjiC inhibitors as promising therapies for targeting taxane-platin-chemoresistant NSCLCs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo23LDZmcc_1LVg90H21EOLACvtfcHk0li_IkdPalTAhg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXotlygtrc%253D&md5=d327d46e521fe85cab305a3bea1265f8</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2017.04.077&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2017.04.077%26sid%3Dliteratum%253Aachs%26aulast%3DDalvi%26aufirst%3DM.%2BP.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DZhong%26aufirst%3DR.%26aulast%3DKollipara%26aufirst%3DR.%2BK.%26aulast%3DPark%26aufirst%3DH.%26aulast%3DBayo%26aufirst%3DJ.%26aulast%3DYenerall%26aufirst%3DP.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DTimmons%26aufirst%3DB.%2BC.%26aulast%3DRodriguez-Canales%26aufirst%3DJ.%26aulast%3DBehrens%26aufirst%3DC.%26aulast%3DMino%26aufirst%3DB.%26aulast%3DVillalobos%26aufirst%3DP.%26aulast%3DParra%26aufirst%3DE.%2BR.%26aulast%3DSuraokar%26aufirst%3DM.%26aulast%3DPataer%26aufirst%3DA.%26aulast%3DSwisher%26aufirst%3DS.%2BG.%26aulast%3DKalhor%26aufirst%3DN.%26aulast%3DBhanu%26aufirst%3DN.%2BV.%26aulast%3DGarcia%26aufirst%3DB.%2BA.%26aulast%3DHeymach%26aufirst%3DJ.%2BV.%26aulast%3DCoombes%26aufirst%3DK.%26aulast%3DXie%26aufirst%3DY.%26aulast%3DGirard%26aufirst%3DL.%26aulast%3DGazdar%26aufirst%3DA.%2BF.%26aulast%3DKittler%26aufirst%3DR.%26aulast%3DWistuba%26aulast%3DMinna%26aufirst%3DJ.%2BD.%26aulast%3DMartinez%26aufirst%3DE.%2BD.%26atitle%3DTaxane-platin-resistant%2520lung%2520cancers%2520co-develop%2520hypersensitivity%2520to%2520JumonjiC%2520demethylase%2520inhibitors%26jtitle%3DCell%2520Rep.%26date%3D2017%26volume%3D19%26spage%3D1669%26epage%3D1684%26doi%3D10.1016%2Fj.celrep.2017.04.077" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roesch, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vultur, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bogeski, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimmermann, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Speicher, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korbel, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laschke, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gimotty, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philipp, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krause, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patzold, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villanueva, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krepler, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fukunaga-Kalabis, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoth, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bastian, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vogt, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herlyn, M.</span></span> <span> </span><span class="NLM_article-title">Overcoming intrinsic multidrug resistance in melanoma by blocking the mitochondrial respiratory chain of slow-cycling JARID1B(high) cells</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">811</span>– <span class="NLM_lpage">825</span>, <span class="refDoi"> DOI: 10.1016/j.ccr.2013.05.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=10.1016%2Fj.ccr.2013.05.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=23764003" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=1%3ACAS%3A528%3ADC%252BC3sXptFKku7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2013&pages=811-825&author=A.+Roeschauthor=A.+Vulturauthor=I.+Bogeskiauthor=H.+Wangauthor=K.+M.+Zimmermannauthor=D.+Speicherauthor=C.+Korbelauthor=M.+W.+Laschkeauthor=P.+A.+Gimottyauthor=S.+E.+Philippauthor=E.+Krauseauthor=S.+Patzoldauthor=J.+Villanuevaauthor=C.+Kreplerauthor=M.+Fukunaga-Kalabisauthor=M.+Hothauthor=B.+C.+Bastianauthor=T.+Vogtauthor=M.+Herlyn&title=Overcoming+intrinsic+multidrug+resistance+in+melanoma+by+blocking+the+mitochondrial+respiratory+chain+of+slow-cycling+JARID1B%28high%29+cells&doi=10.1016%2Fj.ccr.2013.05.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Overcoming Intrinsic Multidrug Resistance in Melanoma by Blocking the Mitochondrial Respiratory Chain of Slow-Cycling JARID1Bhigh Cells</span></div><div class="casAuthors">Roesch, Alexander; Vultur, Adina; Bogeski, Ivan; Wang, Huan; Zimmermann, Katharina M.; Speicher, David; Koerbel, Christina; Laschke, Matthias W.; Gimotty, Phyllis A.; Philipp, Stephan E.; Krause, Elmar; Paetzold, Sylvie; Villanueva, Jessie; Krepler, Clemens; Fukunaga-Kalabis, Mizuho; Hoth, Markus; Bastian, Boris C.; Vogt, Thomas; Herlyn, Meenhard</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">811-825</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Despite success with BRAFV600E inhibitors, therapeutic responses in patients with metastatic melanoma are short-lived because of the acquisition of drug resistance.  We identified a mechanism of intrinsic multidrug resistance based on the survival of a tumor cell subpopulation.  Treatment with various drugs, including cisplatin and vemurafenib, uniformly leads to enrichment of slow-cycling, long-term tumor-maintaining melanoma cells expressing the H3K4-demethylase JARID1B/KDM5B/PLU-1.  Proteome-profiling revealed an upregulation in enzymes of mitochondrial oxidative-ATP-synthesis (oxidative phosphorylation) in this subpopulation.  Inhibition of mitochondrial respiration blocked the emergence of the JARID1Bhigh subpopulation and sensitized melanoma cells to therapy, independent of their genotype.  Our findings support a two-tiered approach combining anticancer agents that eliminate rapidly proliferating melanoma cells with inhibitors of the drug-resistant slow-cycling subpopulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrLdv3RllMhCbVg90H21EOLACvtfcHk0li_IkdPalTAhg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXptFKku7s%253D&md5=ba9ce8134c1fc0f17c44aebae7e760f0</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2013.05.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2013.05.003%26sid%3Dliteratum%253Aachs%26aulast%3DRoesch%26aufirst%3DA.%26aulast%3DVultur%26aufirst%3DA.%26aulast%3DBogeski%26aufirst%3DI.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DZimmermann%26aufirst%3DK.%2BM.%26aulast%3DSpeicher%26aufirst%3DD.%26aulast%3DKorbel%26aufirst%3DC.%26aulast%3DLaschke%26aufirst%3DM.%2BW.%26aulast%3DGimotty%26aufirst%3DP.%2BA.%26aulast%3DPhilipp%26aufirst%3DS.%2BE.%26aulast%3DKrause%26aufirst%3DE.%26aulast%3DPatzold%26aufirst%3DS.%26aulast%3DVillanueva%26aufirst%3DJ.%26aulast%3DKrepler%26aufirst%3DC.%26aulast%3DFukunaga-Kalabis%26aufirst%3DM.%26aulast%3DHoth%26aufirst%3DM.%26aulast%3DBastian%26aufirst%3DB.%2BC.%26aulast%3DVogt%26aufirst%3DT.%26aulast%3DHerlyn%26aufirst%3DM.%26atitle%3DOvercoming%2520intrinsic%2520multidrug%2520resistance%2520in%2520melanoma%2520by%2520blocking%2520the%2520mitochondrial%2520respiratory%2520chain%2520of%2520slow-cycling%2520JARID1B%2528high%2529%2520cells%26jtitle%3DCancer%2520Cell%26date%3D2013%26volume%3D23%26spage%3D811%26epage%3D825%26doi%3D10.1016%2Fj.ccr.2013.05.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, D. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quinlan, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maheswaran, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDermott, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Azizian, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischbach, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brandstetter, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wittner, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramaswamy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Classon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Settleman, J.</span></span> <span> </span><span class="NLM_article-title">A chromatin-mediated reversible drug-tolerant state in cancer cell subpopulations</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>141</i></span>,  <span class="NLM_fpage">69</span>– <span class="NLM_lpage">80</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2010.02.027</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=10.1016%2Fj.cell.2010.02.027" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=20371346" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=1%3ACAS%3A528%3ADC%252BC3cXlsVSgtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=141&publication_year=2010&pages=69-80&author=S.+V.+Sharmaauthor=D.+Y.+Leeauthor=B.+Liauthor=M.+P.+Quinlanauthor=F.+Takahashiauthor=S.+Maheswaranauthor=U.+McDermottauthor=N.+Azizianauthor=L.+Zouauthor=M.+A.+Fischbachauthor=K.+K.+Wongauthor=K.+Brandstetterauthor=B.+Wittnerauthor=S.+Ramaswamyauthor=M.+Classonauthor=J.+Settleman&title=A+chromatin-mediated+reversible+drug-tolerant+state+in+cancer+cell+subpopulations&doi=10.1016%2Fj.cell.2010.02.027"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">A chromatin-mediated reversible drug-tolerant state in cancer cell subpopulations</span></div><div class="casAuthors">Sharma, Sreenath V.; Lee, Diana Y.; Li, Bihua; Quinlan, Margaret P.; Takahashi, Fumiyuki; Maheswaran, Shyamala; McDermott, Ultan; Azizian, Nancy; Zou, Lee; Fischbach, Michael A.; Wong, Kwok-Kin; Brandstetter, Kathleyn; Wittner, Ben; Ramaswamy, Sridhar; Classon, Marie; Settleman, Jeff</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">141</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">69-80</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Accumulating evidence implicates heterogeneity within cancer cell populations in the response to stressful exposures, including drug treatments.  While modeling the acute response to various anticancer agents in drug-sensitive human tumor cell lines, we consistently detected a small subpopulation of reversibly "drug-tolerant" cells.  These cells demonstrate >100-fold reduced drug sensitivity and maintain viability via engagement of IGF-1 receptor signaling and an altered chromatin state that requires the histone demethylase RBP2/KDM5A/Jarid1A.  This drug-tolerant phenotype is transiently acquired and relinquished at low frequency by individual cells within the population, implicating the dynamic regulation of phenotypic heterogeneity in drug tolerance.  The drug-tolerant subpopulation can be selectively ablated by treatment with IGF-1 receptor inhibitors or chromatin-modifying agents, potentially yielding a therapeutic opportunity.  Together, these findings suggest that cancer cell populations employ a dynamic survival strategy in which individual cells transiently assume a reversibly drug-tolerant state to protect the population from eradication by potentially lethal exposures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpK0DpI9mbR_bVg90H21EOLACvtfcHk0lhiv0SdWkqKrg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXlsVSgtb8%253D&md5=2070b7cc4385a4f98db15e5ebcd4589d</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2010.02.027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2010.02.027%26sid%3Dliteratum%253Aachs%26aulast%3DSharma%26aufirst%3DS.%2BV.%26aulast%3DLee%26aufirst%3DD.%2BY.%26aulast%3DLi%26aufirst%3DB.%26aulast%3DQuinlan%26aufirst%3DM.%2BP.%26aulast%3DTakahashi%26aufirst%3DF.%26aulast%3DMaheswaran%26aufirst%3DS.%26aulast%3DMcDermott%26aufirst%3DU.%26aulast%3DAzizian%26aufirst%3DN.%26aulast%3DZou%26aufirst%3DL.%26aulast%3DFischbach%26aufirst%3DM.%2BA.%26aulast%3DWong%26aufirst%3DK.%2BK.%26aulast%3DBrandstetter%26aufirst%3DK.%26aulast%3DWittner%26aufirst%3DB.%26aulast%3DRamaswamy%26aufirst%3DS.%26aulast%3DClasson%26aufirst%3DM.%26aulast%3DSettleman%26aufirst%3DJ.%26atitle%3DA%2520chromatin-mediated%2520reversible%2520drug-tolerant%2520state%2520in%2520cancer%2520cell%2520subpopulations%26jtitle%3DCell%26date%3D2010%26volume%3D141%26spage%3D69%26epage%3D80%26doi%3D10.1016%2Fj.cell.2010.02.027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Westaway, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Preston, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barker, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drewes, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eagle, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garton, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haslam, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayhow, T. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Humphreys, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joberty, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katso, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kruidenier, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leveridge, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pemberton, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioja, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seal, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shipley, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suckling, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span> <span> </span><span class="NLM_article-title">Cell penetrant inhibitors of the KDM4 and KDM5 families of histone lysine demethylases. 2. Pyrido[3,4-d]pyrimidin-4(3H)-one derivatives</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">1370</span>– <span class="NLM_lpage">1387</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01538</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01538" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtVKmsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=1370-1387&author=S.+M.+Westawayauthor=A.+G.+Prestonauthor=M.+D.+Barkerauthor=F.+Brownauthor=J.+A.+Brownauthor=M.+Campbellauthor=C.+W.+Chungauthor=G.+Drewesauthor=R.+Eagleauthor=N.+Gartonauthor=L.+Gordonauthor=C.+Haslamauthor=T.+G.+Hayhowauthor=P.+G.+Humphreysauthor=G.+Jobertyauthor=R.+Katsoauthor=L.+Kruidenierauthor=M.+Leveridgeauthor=M.+Pembertonauthor=I.+Riojaauthor=G.+A.+Sealauthor=T.+Shipleyauthor=O.+Singhauthor=C.+J.+Sucklingauthor=J.+Taylorauthor=P.+Thomasauthor=D.+M.+Wilsonauthor=K.+Leeauthor=R.+K.+Prinjha&title=Cell+penetrant+inhibitors+of+the+KDM4+and+KDM5+families+of+histone+lysine+demethylases.+2.+Pyrido%5B3%2C4-d%5Dpyrimidin-4%283H%29-one+derivatives&doi=10.1021%2Facs.jmedchem.5b01538"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Cell Penetrant Inhibitors of the KDM4 and KDM5 Families of Histone Lysine Demethylases. 2. Pyrido[3,4-d]pyrimidin-4(3H)-one Derivatives</span></div><div class="casAuthors">Westaway, Susan M.; Preston, Alex G. S.; Barker, Michael D.; Brown, Fiona; Brown, Jack A.; Campbell, Matthew; Chung, Chun-wa; Drewes, Gerard; Eagle, Robert; Garton, Neil; Gordon, Laurie; Haslam, Carl; Hayhow, Thomas G.; Humphreys, Philip G.; Joberty, Gerard; Katso, Roy; Kruidenier, Laurens; Leveridge, Melanie; Pemberton, Michelle; Rioja, Inma; Seal, Gail A.; Shipley, Tracy; Singh, Onkar; Suckling, Colin J.; Taylor, Joanna; Thomas, Pamela; Wilson, David M.; Lee, Kevin; Prinjha, Rab K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1370-1387</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Following the discovery of cell penetrant pyridine-4-carboxylate inhibitors of the KDM4 (JMJD2) and KDM5 (JARID1) families of histone lysine demethylases (e.g., 1), further optimization led to the identification of non-carboxylate inhibitors derived from pyrido[3,4-d]pyrimidin-4(3H)-one.  A no. of exemplars such as compd. 41 possess interesting activity profiles in KDM4C and KDM5C biochem. and target-specific, cellular mechanistic assays.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQbtUjkbJPebVg90H21EOLACvtfcHk0lhiv0SdWkqKrg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtVKmsbk%253D&md5=c17e46ecd9017fa9856d7cd39c6701bd</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01538&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01538%26sid%3Dliteratum%253Aachs%26aulast%3DWestaway%26aufirst%3DS.%2BM.%26aulast%3DPreston%26aufirst%3DA.%2BG.%26aulast%3DBarker%26aufirst%3DM.%2BD.%26aulast%3DBrown%26aufirst%3DF.%26aulast%3DBrown%26aufirst%3DJ.%2BA.%26aulast%3DCampbell%26aufirst%3DM.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DDrewes%26aufirst%3DG.%26aulast%3DEagle%26aufirst%3DR.%26aulast%3DGarton%26aufirst%3DN.%26aulast%3DGordon%26aufirst%3DL.%26aulast%3DHaslam%26aufirst%3DC.%26aulast%3DHayhow%26aufirst%3DT.%2BG.%26aulast%3DHumphreys%26aufirst%3DP.%2BG.%26aulast%3DJoberty%26aufirst%3DG.%26aulast%3DKatso%26aufirst%3DR.%26aulast%3DKruidenier%26aufirst%3DL.%26aulast%3DLeveridge%26aufirst%3DM.%26aulast%3DPemberton%26aufirst%3DM.%26aulast%3DRioja%26aufirst%3DI.%26aulast%3DSeal%26aufirst%3DG.%2BA.%26aulast%3DShipley%26aufirst%3DT.%26aulast%3DSingh%26aufirst%3DO.%26aulast%3DSuckling%26aufirst%3DC.%2BJ.%26aulast%3DTaylor%26aufirst%3DJ.%26aulast%3DThomas%26aufirst%3DP.%26aulast%3DWilson%26aufirst%3DD.%2BM.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26atitle%3DCell%2520penetrant%2520inhibitors%2520of%2520the%2520KDM4%2520and%2520KDM5%2520families%2520of%2520histone%2520lysine%2520demethylases.%25202.%2520Pyrido%255B3%252C4-d%255Dpyrimidin-4%25283H%2529-one%2520derivatives%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D1370%26epage%3D1387%26doi%3D10.1021%2Facs.jmedchem.5b01538" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Westaway, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Preston, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barker, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diallo, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Douault, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drewes, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eagle, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haslam, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayhow, T. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Humphreys, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joberty, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katso, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kruidenier, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leveridge, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liddle, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mosley, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muelbaier, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Randle, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioja, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rueger, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seal, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheppard, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span> <span> </span><span class="NLM_article-title">Cell penetrant inhibitors of the KDM4 and KDM5 families of histone lysine demethylases. 1. 3-Amino-4-pyridine carboxylate derivatives</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">1357</span>– <span class="NLM_lpage">1369</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01537</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01537" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=1357-1369&author=S.+M.+Westawayauthor=A.+G.+Prestonauthor=M.+D.+Barkerauthor=F.+Brownauthor=J.+A.+Brownauthor=M.+Campbellauthor=C.+W.+Chungauthor=H.+Dialloauthor=C.+Douaultauthor=G.+Drewesauthor=R.+Eagleauthor=L.+Gordonauthor=C.+Haslamauthor=T.+G.+Hayhowauthor=P.+G.+Humphreysauthor=G.+Jobertyauthor=R.+Katsoauthor=L.+Kruidenierauthor=M.+Leveridgeauthor=J.+Liddleauthor=J.+Mosleyauthor=M.+Muelbaierauthor=R.+Randleauthor=I.+Riojaauthor=A.+Ruegerauthor=G.+A.+Sealauthor=R.+J.+Sheppardauthor=O.+Singhauthor=J.+Taylorauthor=P.+Thomasauthor=D.+Thomsonauthor=D.+M.+Wilsonauthor=K.+Leeauthor=R.+K.+Prinjha&title=Cell+penetrant+inhibitors+of+the+KDM4+and+KDM5+families+of+histone+lysine+demethylases.+1.+3-Amino-4-pyridine+carboxylate+derivatives&doi=10.1021%2Facs.jmedchem.5b01537"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01537&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01537%26sid%3Dliteratum%253Aachs%26aulast%3DWestaway%26aufirst%3DS.%2BM.%26aulast%3DPreston%26aufirst%3DA.%2BG.%26aulast%3DBarker%26aufirst%3DM.%2BD.%26aulast%3DBrown%26aufirst%3DF.%26aulast%3DBrown%26aufirst%3DJ.%2BA.%26aulast%3DCampbell%26aufirst%3DM.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DDiallo%26aufirst%3DH.%26aulast%3DDouault%26aufirst%3DC.%26aulast%3DDrewes%26aufirst%3DG.%26aulast%3DEagle%26aufirst%3DR.%26aulast%3DGordon%26aufirst%3DL.%26aulast%3DHaslam%26aufirst%3DC.%26aulast%3DHayhow%26aufirst%3DT.%2BG.%26aulast%3DHumphreys%26aufirst%3DP.%2BG.%26aulast%3DJoberty%26aufirst%3DG.%26aulast%3DKatso%26aufirst%3DR.%26aulast%3DKruidenier%26aufirst%3DL.%26aulast%3DLeveridge%26aufirst%3DM.%26aulast%3DLiddle%26aufirst%3DJ.%26aulast%3DMosley%26aufirst%3DJ.%26aulast%3DMuelbaier%26aufirst%3DM.%26aulast%3DRandle%26aufirst%3DR.%26aulast%3DRioja%26aufirst%3DI.%26aulast%3DRueger%26aufirst%3DA.%26aulast%3DSeal%26aufirst%3DG.%2BA.%26aulast%3DSheppard%26aufirst%3DR.%2BJ.%26aulast%3DSingh%26aufirst%3DO.%26aulast%3DTaylor%26aufirst%3DJ.%26aulast%3DThomas%26aufirst%3DP.%26aulast%3DThomson%26aufirst%3DD.%26aulast%3DWilson%26aufirst%3DD.%2BM.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26atitle%3DCell%2520penetrant%2520inhibitors%2520of%2520the%2520KDM4%2520and%2520KDM5%2520families%2520of%2520histone%2520lysine%2520demethylases.%25201.%25203-Amino-4-pyridine%2520carboxylate%2520derivatives%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D1357%26epage%3D1369%26doi%3D10.1021%2Facs.jmedchem.5b01537" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Watanuki, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kosai, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osanai, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ogama, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mochizuki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tamai, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamaguchi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Satoh, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fukuhara, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maemondo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ichinose, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nukiwa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, N.</span></span> <span> </span><span class="NLM_article-title">Synergistic cytotoxicity of afatinib and cetuximab against EGFR T790M involves Rab11-dependent EGFR recycling</span>. <i>Biochem. Biophys. Res. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>455</i></span>,  <span class="NLM_fpage">269</span>– <span class="NLM_lpage">276</span>, <span class="refDoi"> DOI: 10.1016/j.bbrc.2014.11.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=10.1016%2Fj.bbrc.2014.11.003" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=455&publication_year=2014&pages=269-276&author=Z.+Watanukiauthor=H.+Kosaiauthor=N.+Osanaiauthor=N.+Ogamaauthor=M.+Mochizukiauthor=K.+Tamaiauthor=K.+Yamaguchiauthor=K.+Satohauthor=T.+Fukuharaauthor=M.+Maemondoauthor=M.+Ichinoseauthor=T.+Nukiwaauthor=N.+Tanaka&title=Synergistic+cytotoxicity+of+afatinib+and+cetuximab+against+EGFR+T790M+involves+Rab11-dependent+EGFR+recycling&doi=10.1016%2Fj.bbrc.2014.11.003"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1016%2Fj.bbrc.2014.11.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbrc.2014.11.003%26sid%3Dliteratum%253Aachs%26aulast%3DWatanuki%26aufirst%3DZ.%26aulast%3DKosai%26aufirst%3DH.%26aulast%3DOsanai%26aufirst%3DN.%26aulast%3DOgama%26aufirst%3DN.%26aulast%3DMochizuki%26aufirst%3DM.%26aulast%3DTamai%26aufirst%3DK.%26aulast%3DYamaguchi%26aufirst%3DK.%26aulast%3DSatoh%26aufirst%3DK.%26aulast%3DFukuhara%26aufirst%3DT.%26aulast%3DMaemondo%26aufirst%3DM.%26aulast%3DIchinose%26aufirst%3DM.%26aulast%3DNukiwa%26aufirst%3DT.%26aulast%3DTanaka%26aufirst%3DN.%26atitle%3DSynergistic%2520cytotoxicity%2520of%2520afatinib%2520and%2520cetuximab%2520against%2520EGFR%2520T790M%2520involves%2520Rab11-dependent%2520EGFR%2520recycling%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D2014%26volume%3D455%26spage%3D269%26epage%3D276%26doi%3D10.1016%2Fj.bbrc.2014.11.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Castillo-Aguilera, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Depreux, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halby, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arimondo, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goossens, L.</span></span> <span> </span><span class="NLM_article-title">DNA methylation targeting: the DNMT/HMT crosstalk challenge</span>. <i>Biomolecules</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">3</span>, <span class="refDoi"> DOI: 10.3390/biom7010003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=10.3390%2Fbiom7010003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=1%3ACAS%3A528%3ADC%252BC2sXksVSisrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=3&author=O.+Castillo-Aguileraauthor=P.+Depreuxauthor=L.+Halbyauthor=P.+B.+Arimondoauthor=L.+Goossens&title=DNA+methylation+targeting%3A+the+DNMT%2FHMT+crosstalk+challenge&doi=10.3390%2Fbiom7010003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">DNA methylation targeting: the DNMT/HMT crosstalk challenge</span></div><div class="casAuthors">Castillo-Aguilera, Omar; Depreux, Patrick; Halby, Ludovic; Arimondo, Paola B.; Goossens, Laurence</div><div class="citationInfo"><span class="NLM_cas:title">Biomolecules</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">3/1-3/21</span>CODEN:
                <span class="NLM_cas:coden">BIOMHC</span>;
        ISSN:<span class="NLM_cas:issn">2218-273X</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">A review.  Chromatin can adopt a decondensed state linked to gene transcription (euchromatin) and a condensed state linked to transcriptional repression (heterochromatin).  These states are controlled by epigenetic modulators that are active on either the DNA or the histones and are tightly assocd. to each other.  Methylation of both DNA and histones is involved in either the activation or silencing of genes and their crosstalk.  Since DNA/histone methylation patterns are altered in cancers, mols. that target these modifications are interesting therapeutic tools.  We present herein a vast panel of DNA methyltransferase inhibitors classified according to their mechanism, as well as selected histone methyltransferase inhibitors sharing a common mode of action.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGorl0eU7GvR7LVg90H21EOLACvtfcHk0lgD9NkJZvkjKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXksVSisrY%253D&md5=7445d305512673f988a04e9c46e82fd2</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.3390%2Fbiom7010003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fbiom7010003%26sid%3Dliteratum%253Aachs%26aulast%3DCastillo-Aguilera%26aufirst%3DO.%26aulast%3DDepreux%26aufirst%3DP.%26aulast%3DHalby%26aufirst%3DL.%26aulast%3DArimondo%26aufirst%3DP.%2BB.%26aulast%3DGoossens%26aufirst%3DL.%26atitle%3DDNA%2520methylation%2520targeting%253A%2520the%2520DNMT%252FHMT%2520crosstalk%2520challenge%26jtitle%3DBiomolecules%26date%3D2017%26volume%3D7%26spage%3D3%26doi%3D10.3390%2Fbiom7010003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Patel, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dickson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Din, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macleod, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jodrell, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramsahoye, B.</span></span> <span> </span><span class="NLM_article-title">Targeting of 5-aza-2’-deoxycytidine residues by chromatin-associated DNMT1 induces proteasomal degradation of the free enzyme</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">4313</span>– <span class="NLM_lpage">4324</span>, <span class="refDoi"> DOI: 10.1093/nar/gkq187</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=10.1093%2Fnar%2Fgkq187" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2010&pages=4313-4324&author=K.+Patelauthor=J.+Dicksonauthor=S.+Dinauthor=K.+Macleodauthor=D.+Jodrellauthor=B.+Ramsahoye&title=Targeting+of+5-aza-2%E2%80%99-deoxycytidine+residues+by+chromatin-associated+DNMT1+induces+proteasomal+degradation+of+the+free+enzyme&doi=10.1093%2Fnar%2Fgkq187"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkq187&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkq187%26sid%3Dliteratum%253Aachs%26aulast%3DPatel%26aufirst%3DK.%26aulast%3DDickson%26aufirst%3DJ.%26aulast%3DDin%26aufirst%3DS.%26aulast%3DMacleod%26aufirst%3DK.%26aulast%3DJodrell%26aufirst%3DD.%26aulast%3DRamsahoye%26aufirst%3DB.%26atitle%3DTargeting%2520of%25205-aza-2%25E2%2580%2599-deoxycytidine%2520residues%2520by%2520chromatin-associated%2520DNMT1%2520induces%2520proteasomal%2520degradation%2520of%2520the%2520free%2520enzyme%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2010%26volume%3D38%26spage%3D4313%26epage%3D4324%26doi%3D10.1093%2Fnar%2Fgkq187" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Das, B. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renaud, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doroshow, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takeda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pommier, Y.</span></span> <span> </span><span class="NLM_article-title">Trapping of PARP1 and PARP2 by clinical PARP inhibitors</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>72</i></span>,  <span class="NLM_fpage">5588</span>– <span class="NLM_lpage">5599</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-12-2753</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=10.1158%2F0008-5472.CAN-12-2753" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=23118055" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsF2jsLjE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=2012&pages=5588-5599&author=J.+Muraiauthor=S.+Y.+Huangauthor=B.+B.+Dasauthor=A.+Renaudauthor=Y.+Zhangauthor=J.+H.+Doroshowauthor=J.+Jiauthor=S.+Takedaauthor=Y.+Pommier&title=Trapping+of+PARP1+and+PARP2+by+clinical+PARP+inhibitors&doi=10.1158%2F0008-5472.CAN-12-2753"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Trapping of PARP1 and PARP2 by Clinical PARP Inhibitors</span></div><div class="casAuthors">Murai, Junko; Huang, Shar-yin N.; Das, Benu Brata; Renaud, Amelie; Zhang, Yiping; Doroshow, James H.; Ji, Jiuping; Takeda, Shunichi; Pommier, Yves</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">72</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">5588-5599</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">These findings indicate that PARP inhibitors in clin. trials not only block catalytic function but also trap PARP enzymes on DNA, illuminating the mechanistic basis for their cytotoxic activity in cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrW3m55HXn9ObVg90H21EOLACvtfcHk0lgQamo_tybxVg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsF2jsLjE&md5=81c590b6a4fbb9efc9c64f7de5e89524</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-12-2753&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-12-2753%26sid%3Dliteratum%253Aachs%26aulast%3DMurai%26aufirst%3DJ.%26aulast%3DHuang%26aufirst%3DS.%2BY.%26aulast%3DDas%26aufirst%3DB.%2BB.%26aulast%3DRenaud%26aufirst%3DA.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DDoroshow%26aufirst%3DJ.%2BH.%26aulast%3DJi%26aufirst%3DJ.%26aulast%3DTakeda%26aufirst%3DS.%26aulast%3DPommier%26aufirst%3DY.%26atitle%3DTrapping%2520of%2520PARP1%2520and%2520PARP2%2520by%2520clinical%2520PARP%2520inhibitors%26jtitle%3DCancer%2520Res.%26date%3D2012%26volume%3D72%26spage%3D5588%26epage%3D5599%26doi%3D10.1158%2F0008-5472.CAN-12-2753" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Muvarak, N. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chowdhury, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xia, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robert, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, E. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellani, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, Z. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duong, V. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rutherford, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagaria, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bentzen, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seidman, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baer, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lapidus, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baylin, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rassool, F. V.</span></span> <span> </span><span class="NLM_article-title">Enhancing the cytotoxic effects of PARP inhibitors with DNA demethylating agents - A potential therapy for cancer</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">637</span>– <span class="NLM_lpage">650</span>, <span class="refDoi"> DOI: 10.1016/j.ccell.2016.09.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=10.1016%2Fj.ccell.2016.09.002" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2016&pages=637-650&author=N.+E.+Muvarakauthor=K.+Chowdhuryauthor=L.+Xiaauthor=C.+Robertauthor=E.+Y.+Choiauthor=Y.+Caiauthor=M.+Bellaniauthor=Y.+Zouauthor=Z.+N.+Singhauthor=V.+H.+Duongauthor=T.+Rutherfordauthor=P.+Nagariaauthor=S.+M.+Bentzenauthor=M.+M.+Seidmanauthor=M.+R.+Baerauthor=R.+G.+Lapidusauthor=S.+B.+Baylinauthor=F.+V.+Rassool&title=Enhancing+the+cytotoxic+effects+of+PARP+inhibitors+with+DNA+demethylating+agents+-+A+potential+therapy+for+cancer&doi=10.1016%2Fj.ccell.2016.09.002"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2016.09.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2016.09.002%26sid%3Dliteratum%253Aachs%26aulast%3DMuvarak%26aufirst%3DN.%2BE.%26aulast%3DChowdhury%26aufirst%3DK.%26aulast%3DXia%26aufirst%3DL.%26aulast%3DRobert%26aufirst%3DC.%26aulast%3DChoi%26aufirst%3DE.%2BY.%26aulast%3DCai%26aufirst%3DY.%26aulast%3DBellani%26aufirst%3DM.%26aulast%3DZou%26aufirst%3DY.%26aulast%3DSingh%26aufirst%3DZ.%2BN.%26aulast%3DDuong%26aufirst%3DV.%2BH.%26aulast%3DRutherford%26aufirst%3DT.%26aulast%3DNagaria%26aufirst%3DP.%26aulast%3DBentzen%26aufirst%3DS.%2BM.%26aulast%3DSeidman%26aufirst%3DM.%2BM.%26aulast%3DBaer%26aufirst%3DM.%2BR.%26aulast%3DLapidus%26aufirst%3DR.%2BG.%26aulast%3DBaylin%26aufirst%3DS.%2BB.%26aulast%3DRassool%26aufirst%3DF.%2BV.%26atitle%3DEnhancing%2520the%2520cytotoxic%2520effects%2520of%2520PARP%2520inhibitors%2520with%2520DNA%2520demethylating%2520agents%2520-%2520A%2520potential%2520therapy%2520for%2520cancer%26jtitle%3DCancer%2520Cell%26date%3D2016%26volume%3D30%26spage%3D637%26epage%3D650%26doi%3D10.1016%2Fj.ccell.2016.09.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 13 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Yi Zhang, Deqin Rong, Bingbing Li, <span class="NLM_string-name hlFld-ContribAuthor">Yuanxiang Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting Epigenetic Regulators with Covalent Small-Molecule Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (12)
                                     , 7900-7925. <a href="https://doi.org/10.1021/acs.jmedchem.0c02055" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c02055</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c02055&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c02055%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DTargeting%252BEpigenetic%252BRegulators%252Bwith%252BCovalent%252BSmall-Molecule%252BInhibitors%26aulast%3DZhang%26aufirst%3DYi%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D26112020%26date%3D18022021%26volume%3D64%26issue%3D12%26spage%3D7900%26epage%3D7925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Yuka Miyake, Yukihiro Itoh, Yoshinori Suzuma, Hidehiko Kodama, Takashi Kurohara, Yasunobu Yamashita, Remy Narozny, Yutaro Hanatani, Shusaku Uchida, <span class="NLM_string-name hlFld-ContribAuthor">Takayoshi Suzuki</span>. </span><span class="cited-content_cbyCitation_article-title">Metalloprotein-Catalyzed Click Reaction for In Situ Generation of a Potent Inhibitor. </span><span class="cited-content_cbyCitation_journal-name">ACS Catalysis</span><span> <strong>2020,</strong> <em>10 </em>
                                    (10)
                                     , 5383-5392. <a href="https://doi.org/10.1021/acscatal.0c00369" title="DOI URL">https://doi.org/10.1021/acscatal.0c00369</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acscatal.0c00369&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facscatal.0c00369%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Catalysis%26atitle%3DMetalloprotein-Catalyzed%252BClick%252BReaction%252Bfor%252BIn%252BSitu%252BGeneration%252Bof%252Ba%252BPotent%252BInhibitor%26aulast%3DMiyake%26aufirst%3DYuka%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D22012020%26date%3D08042020%26date%3D23042020%26date%3D13042020%26volume%3D10%26issue%3D10%26spage%3D5383%26epage%3D5392" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Nektaria Petronikolou, James E. Longbotham, <span class="NLM_string-name hlFld-ContribAuthor">Danica Galonić Fujimori</span>. </span><span class="cited-content_cbyCitation_article-title">Extended Recognition of the Histone H3 Tail by Histone Demethylase KDM5A. </span><span class="cited-content_cbyCitation_journal-name">Biochemistry</span><span> <strong>2020,</strong> <em>59 </em>
                                    (5)
                                     , 647-651. <a href="https://doi.org/10.1021/acs.biochem.9b01036" title="DOI URL">https://doi.org/10.1021/acs.biochem.9b01036</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.biochem.9b01036&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.biochem.9b01036%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochemistry%26atitle%3DExtended%252BRecognition%252Bof%252Bthe%252BHistone%252BH3%252BTail%252Bby%252BHistone%252BDemethylase%252BKDM5A%26aulast%3DPetronikolou%26aufirst%3DNektaria%26date%3D2020%26date%3D2020%26date%3D2019%26date%3D2020%26date%3D25112019%26date%3D24012020%26date%3D30012020%26date%3D27012020%26volume%3D59%26issue%3D5%26spage%3D647%26epage%3D651" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Gaochan Wu, Tong Zhao, Dongwei Kang, Jian Zhang, Yuning Song, Vigneshwaran Namasivayam, Jacob Kongsted, Christophe Pannecouque, Erik De Clercq, Vasanthanathan Poongavanam, Xinyong Liu, <span class="NLM_string-name hlFld-ContribAuthor">Peng Zhan</span>. </span><span class="cited-content_cbyCitation_article-title">Overview of Recent Strategic Advances in Medicinal Chemistry. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (21)
                                     , 9375-9414. <a href="https://doi.org/10.1021/acs.jmedchem.9b00359" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b00359</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b00359&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b00359%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DOverview%252Bof%252BRecent%252BStrategic%252BAdvances%252Bin%252BMedicinal%252BChemistry%26aulast%3DWu%26aufirst%3DGaochan%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D26022019%26date%3D28052019%26date%3D03052019%26volume%3D62%26issue%3D21%26spage%3D9375%26epage%3D9414" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Martin Roatsch, Inga Hoffmann, Martine I. Abboud, Rebecca L. Hancock, Hanna Tarhonskaya, Kuo-Feng Hsu, Sarah E. Wilkins, Tzu-Lan Yeh, Kerstin Lippl, Kerstin Serrer, Isabelle Moneke, Theresa D. Ahrens, Dina Robaa, Sandra Wenzler, Nicolas P. F. Barthes, Henriette Franz, Wolfgang Sippl, Silke Lassmann, Sven Diederichs, Erik Schleicher, Christopher J. Schofield, Akane Kawamura, Roland Schüle, <span class="NLM_string-name hlFld-ContribAuthor">Manfred Jung</span>. </span><span class="cited-content_cbyCitation_article-title">The Clinically Used Iron Chelator Deferasirox Is an Inhibitor of Epigenetic JumonjiC Domain-Containing Histone Demethylases. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Biology</span><span> <strong>2019,</strong> <em>14 </em>
                                    (8)
                                     , 1737-1750. <a href="https://doi.org/10.1021/acschembio.9b00289" title="DOI URL">https://doi.org/10.1021/acschembio.9b00289</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acschembio.9b00289&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facschembio.9b00289%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Biology%26atitle%3DThe%252BClinically%252BUsed%252BIron%252BChelator%252BDeferasirox%252BIs%252Ban%252BInhibitor%252Bof%252BEpigenetic%252BJumonjiC%252BDomain-Containing%252BHistone%252BDemethylases%26aulast%3DRoatsch%26aufirst%3DMartin%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D12042019%26date%3D09072019%26date%3D19072019%26date%3D09072019%26volume%3D14%26issue%3D8%26spage%3D1737%26epage%3D1750" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Guan-Jun  Yang</span>, <span class="hlFld-ContribAuthor ">Ming-Hui  Zhu</span>, <span class="hlFld-ContribAuthor ">Xin-Jiang  Lu</span>, <span class="hlFld-ContribAuthor ">Yan-Jun  Liu</span>, <span class="hlFld-ContribAuthor ">Jian-Fei  Lu</span>, <span class="hlFld-ContribAuthor ">Chung-Hang  Leung</span>, <span class="hlFld-ContribAuthor ">Dik-Lung  Ma</span>, <span class="hlFld-ContribAuthor ">Jiong  Chen</span>. </span><span class="cited-content_cbyCitation_article-title">The emerging role of KDM5A in human cancer. </span><span class="cited-content_cbyCitation_journal-name">Journal of Hematology & Oncology</span><span> <strong>2021,</strong> <em>14 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s13045-021-01041-1" title="DOI URL">https://doi.org/10.1186/s13045-021-01041-1</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s13045-021-01041-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs13045-021-01041-1%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Hematology%2520%2526%2520Oncology%26atitle%3DThe%252Bemerging%252Brole%252Bof%252BKDM5A%252Bin%252Bhuman%252Bcancer%26aulast%3DYang%26aufirst%3DGuan-Jun%26date%3D2021%26date%3D2021%26volume%3D14%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Wenqing  Zhang</span>, <span class="hlFld-ContribAuthor ">Kan  Li</span>, <span class="hlFld-ContribAuthor ">Tianqi  Wang</span>, <span class="hlFld-ContribAuthor ">Ming  Wu</span>, <span class="hlFld-ContribAuthor ">Linli  Li</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of JMJD7 inhibitors with the aid of virtual screening and bioactivity evaluation. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2021,</strong> <em>45 </em>, 128139. <a href="https://doi.org/10.1016/j.bmcl.2021.128139" title="DOI URL">https://doi.org/10.1016/j.bmcl.2021.128139</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2021.128139&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2021.128139%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DDiscovery%252Bof%252BJMJD7%252Binhibitors%252Bwith%252Bthe%252Baid%252Bof%252Bvirtual%252Bscreening%252Band%252Bbioactivity%252Bevaluation%26aulast%3DZhang%26aufirst%3DWenqing%26date%3D2021%26volume%3D45%26spage%3D128139" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jenna N.  Beyer</span>, <span class="hlFld-ContribAuthor ">Nicole R.  Raniszewski</span>, <span class="hlFld-ContribAuthor ">George M.  Burslem</span>. </span><span class="cited-content_cbyCitation_article-title">Advances and Opportunities in Epigenetic Chemical Biology. </span><span class="cited-content_cbyCitation_journal-name">ChemBioChem</span><span> <strong>2021,</strong> <em>22 </em>
                                    (1)
                                     , 17-42. <a href="https://doi.org/10.1002/cbic.202000459" title="DOI URL">https://doi.org/10.1002/cbic.202000459</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cbic.202000459&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcbic.202000459%26sid%3Dliteratum%253Aachs%26jtitle%3DChemBioChem%26atitle%3DAdvances%252Band%252BOpportunities%252Bin%252BEpigenetic%252BChemical%252BBiology%26aulast%3DBeyer%26aufirst%3DJenna%2BN.%26date%3D2021%26date%3D2020%26volume%3D22%26issue%3D1%26spage%3D17%26epage%3D42" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yun-Dong  Fu</span>, <span class="hlFld-ContribAuthor ">Ming-Jie  Huang</span>, <span class="hlFld-ContribAuthor ">Jia-Wen  Guo</span>, <span class="hlFld-ContribAuthor ">Ya-Zhen  You</span>, <span class="hlFld-ContribAuthor ">Hong-Min  Liu</span>, <span class="hlFld-ContribAuthor ">Li-Hua  Huang</span>, <span class="hlFld-ContribAuthor ">Bin  Yu</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting histone demethylase KDM5B for cancer treatment. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>208 </em>, 112760. <a href="https://doi.org/10.1016/j.ejmech.2020.112760" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112760</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112760&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112760%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DTargeting%252Bhistone%252Bdemethylase%252BKDM5B%252Bfor%252Bcancer%252Btreatment%26aulast%3DFu%26aufirst%3DYun-Dong%26date%3D2020%26volume%3D208%26spage%3D112760" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sarder  Arifuzzaman</span>, <span class="hlFld-ContribAuthor ">Mst Reshma  Khatun</span>, <span class="hlFld-ContribAuthor ">Rabeya  Khatun</span>. </span><span class="cited-content_cbyCitation_article-title">Emerging of lysine demethylases (KDMs): From pathophysiological insights to novel therapeutic opportunities. </span><span class="cited-content_cbyCitation_journal-name">Biomedicine & Pharmacotherapy</span><span> <strong>2020,</strong> <em>129 </em>, 110392. <a href="https://doi.org/10.1016/j.biopha.2020.110392" title="DOI URL">https://doi.org/10.1016/j.biopha.2020.110392</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.biopha.2020.110392&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.biopha.2020.110392%26sid%3Dliteratum%253Aachs%26jtitle%3DBiomedicine%2520%2526%2520Pharmacotherapy%26atitle%3DEmerging%252Bof%252Blysine%252Bdemethylases%252B%252528KDMs%252529%25253A%252BFrom%252Bpathophysiological%252Binsights%252Bto%252Bnovel%252Btherapeutic%252Bopportunities%26aulast%3DArifuzzaman%26aufirst%3DSarder%26date%3D2020%26volume%3D129%26spage%3D110392" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Chongchen  Zhou</span>, <span class="hlFld-ContribAuthor ">Hai-liang  Zhu</span>, <span class="hlFld-ContribAuthor ">Yongtao  Duan</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting Histone Acetyltransferase MOZ/KAT6A as a New Avenue for Hematological Tumor Therapy. </span><span class="cited-content_cbyCitation_journal-name">Current Topics in Medicinal Chemistry</span><span> <strong>2020,</strong> <em>20 </em>
                                    (5)
                                     , 333-335. <a href="https://doi.org/10.2174/156802662005200304123442" title="DOI URL">https://doi.org/10.2174/156802662005200304123442</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.2174/156802662005200304123442&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.2174%2F156802662005200304123442%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Topics%2520in%2520Medicinal%2520Chemistry%26atitle%3DTargeting%252BHistone%252BAcetyltransferase%252BMOZ%25252FKAT6A%252Bas%252Ba%252BNew%252BAvenue%252Bfor%252BHematological%252BTumor%252BTherapy%26aulast%3DZhou%26aufirst%3DChongchen%26date%3D2020%26volume%3D20%26issue%3D5%26spage%3D333%26epage%3D335" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yann-Vaï  Le Bihan</span>, <span class="hlFld-ContribAuthor ">Rachel M.  Lanigan</span>, <span class="hlFld-ContribAuthor ">Butrus  Atrash</span>, <span class="hlFld-ContribAuthor ">Mark G.  McLaughlin</span>, <span class="hlFld-ContribAuthor ">Srikannathasan  Velupillai</span>, <span class="hlFld-ContribAuthor ">Andrew G.  Malcolm</span>, <span class="hlFld-ContribAuthor ">Katherine S.  England</span>, <span class="hlFld-ContribAuthor ">Gian Filippo  Ruda</span>, <span class="hlFld-ContribAuthor ">N. Yi  Mok</span>, <span class="hlFld-ContribAuthor ">Anthony  Tumber</span>, <span class="hlFld-ContribAuthor ">Kathy  Tomlin</span>, <span class="hlFld-ContribAuthor ">Harry  Saville</span>, <span class="hlFld-ContribAuthor ">Erald  Shehu</span>, <span class="hlFld-ContribAuthor ">Craig  McAndrew</span>, <span class="hlFld-ContribAuthor ">LeAnne  Carmichael</span>, <span class="hlFld-ContribAuthor ">James M.  Bennett</span>, <span class="hlFld-ContribAuthor ">Fiona  Jeganathan</span>, <span class="hlFld-ContribAuthor ">Paul  Eve</span>, <span class="hlFld-ContribAuthor ">Adam  Donovan</span>, <span class="hlFld-ContribAuthor ">Angela  Hayes</span>, <span class="hlFld-ContribAuthor ">Francesca  Wood</span>, <span class="hlFld-ContribAuthor ">Florence I.  Raynaud</span>, <span class="hlFld-ContribAuthor ">Oleg  Fedorov</span>, <span class="hlFld-ContribAuthor ">Paul E.  Brennan</span>, <span class="hlFld-ContribAuthor ">Rosemary  Burke</span>, <span class="hlFld-ContribAuthor ">Rob L.M.  van Montfort</span>, <span class="hlFld-ContribAuthor ">Olivia W.  Rossanese</span>, <span class="hlFld-ContribAuthor ">Julian  Blagg</span>, <span class="hlFld-ContribAuthor ">Vassilios  Bavetsias</span>. </span><span class="cited-content_cbyCitation_article-title">C8-substituted pyrido[3,4-d]pyrimidin-4(3H)-ones: Studies towards the identification of potent, cell penetrant Jumonji C domain containing histone lysine demethylase 4 subfamily (KDM4) inhibitors, compound profiling in cell-based target engagement assays. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>177 </em>, 316-337. <a href="https://doi.org/10.1016/j.ejmech.2019.05.041" title="DOI URL">https://doi.org/10.1016/j.ejmech.2019.05.041</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2019.05.041&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2019.05.041%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DC8-substituted%252Bpyrido%25255B3%25252C4-d%25255Dpyrimidin-4%2525283H%252529-ones%25253A%252BStudies%252Btowards%252Bthe%252Bidentification%252Bof%252Bpotent%25252C%252Bcell%252Bpenetrant%252BJumonji%252BC%252Bdomain%252Bcontaining%252Bhistone%252Blysine%252Bdemethylase%252B4%252Bsubfamily%252B%252528KDM4%252529%252Binhibitors%25252C%252Bcompound%252Bprofiling%252Bin%252Bcell-based%252Btarget%252Bengagement%252Bassays%26aulast%3DLe%2BBihan%26aufirst%3DYann-Va%25C3%25AF%26date%3D2019%26volume%3D177%26spage%3D316%26epage%3D337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yuka  Miyake</span>, <span class="hlFld-ContribAuthor ">Yukihiro  Itoh</span>, <span class="hlFld-ContribAuthor ">Atsushi  Hatanaka</span>, <span class="hlFld-ContribAuthor ">Yoshinori  Suzuma</span>, <span class="hlFld-ContribAuthor ">Miki  Suzuki</span>, <span class="hlFld-ContribAuthor ">Hidehiko  Kodama</span>, <span class="hlFld-ContribAuthor ">Yoshinobu  Arai</span>, <span class="hlFld-ContribAuthor ">Takayoshi  Suzuki</span>. </span><span class="cited-content_cbyCitation_article-title">Identification of novel lysine demethylase 5-selective inhibitors by inhibitor-based fragment merging strategy. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2019,</strong> <em>27 </em>
                                    (6)
                                     , 1119-1129. <a href="https://doi.org/10.1016/j.bmc.2019.02.006" title="DOI URL">https://doi.org/10.1016/j.bmc.2019.02.006</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2019.02.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2019.02.006%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DIdentification%252Bof%252Bnovel%252Blysine%252Bdemethylase%252B5-selective%252Binhibitors%252Bby%252Binhibitor-based%252Bfragment%252Bmerging%252Bstrategy%26aulast%3DMiyake%26aufirst%3DYuka%26date%3D2019%26volume%3D27%26issue%3D6%26spage%3D1119%26epage%3D1129" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01219/20181207/images/medium/jm-2018-01219j_0009.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01219/20181207/images/large/jm-2018-01219j_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01219&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01219/20181207/images/medium/jm-2018-01219j_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01219/20181207/images/large/jm-2018-01219j_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Binding of KDM5A by <b>GSK-J1</b> and its derivatives. (A) Sequence alignment of region of KDM5A containing Cys481 (in magenta) and corresponding regions in KDM4A (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6CG1">6CG1</a>), KDM6A (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3AVS">3AVS</a>), KDM2A (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4QXB">4QXB</a>), PHF8 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3KV4">3KV4</a>), and KDM3B (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4C8D">4C8D</a>). KDM5A and KDM4A share the highest sequence conservation (white letters against black background), followed by KDM6A. (B) Structure of ternary complex of GSK-J1-KDM5A-Mn(II) at 1.39 Å resolution (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6DQ4">6DQ4</a>). The omit electron density, contoured at 10σ above the mean, is shown for GSK-J1 (gray mesh). (C) Two alternative conformations are observed for Tyr409. The 2Fo–Fc electron density, contoured at 1.5σ above the mean, is shown for KDM5 protein residues (light gray mesh). (D) Cys481 is 4.2 Å away from the tip of GSK-J1. (E) Two orthogonal orientations of superimposed <b>GSK-J1</b> bound in KDM5A (yellow) and KDM6B (magenta). Cys481 of KDM5A replaces Pro-1388 of KDM6B. (F) Summary of the IC<sub>50</sub> values of inhibition of demethylation for catalytic domains of KDM4A, KDM5A, and KDM6A by FDH-coupled assay and isothermal titration calorimetry measurement of dissociation constants (<i>K</i><sub>D</sub>) of compounds to KDM5A catalytic domain (see <a class="ref internalNav" href="#notes1" aria-label="Figure S1">Figure S1</a> for original experimental data). (G) Chemical structures of <b>GSK-J1</b> and two related derivatives. Included are summary of inhibitor-bound X-ray structures of KDM5A (see <a class="ref internalNav" href="#notes1" aria-label="Table S1">Table S1</a> for detailed statistics of X-ray diffraction and refinement). (H) Structural details showing interactions between KDM5A and <b>N43</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6DQ5">6DQ5</a>). (I) Interactions displayed between KDM5A and <b>N44</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6DQ6">6DQ6</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01219/20181207/images/large/jm-2018-01219j_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01219&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01219/20181207/images/medium/jm-2018-01219j_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01219/20181207/images/large/jm-2018-01219j_0007.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Compounds <b>N43</b> and <b>N44</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01219/20181207/images/large/jm-2018-01219j_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01219&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01219/20181207/images/medium/jm-2018-01219j_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01219/20181207/images/large/jm-2018-01219j_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Comparison of <b>N46</b> and <b>N49</b>. (A,B) Chemical structures of <b>N46</b> and <b>N49</b> are identical except containing a 1<i>H</i>-pyrrolo[3,2-<i>b</i>]pyridine-7-carboxylate moiety (panel A) or thienopyridine-7-carboxylate (panel B). Included are a summary of ITC measurements of dissociation constants (<i>K</i><sub>D</sub>) (see <a class="ref internalNav" href="#notes1" aria-label="Figure S1B">Figure S1B</a> for original binding data) and a summary of inhibitor-bound X-ray structures of KDM5A (<a class="ref internalNav" href="#notes1" aria-label="Table S1">Table S1</a>). (C) In the racemic mixture of compound <b>N49</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6DQ8">6DQ8</a>), two possible conformations of 2-chlorophenyl moiety can be modeled into the electron density. (D) View rotated ∼180° from panel C.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01219/20181207/images/large/jm-2018-01219j_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01219&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01219/20181207/images/medium/jm-2018-01219j_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01219/20181207/images/large/jm-2018-01219j_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Structural snapshots of covalent modifiers bound into KDM5A active site. (A) Chemical structure of <b>N70</b>. (B) The amino group of 3-aminophenyl moiety of <b>N70</b> involves in a H-bond-like interaction with Cys481 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6DQA">6DQA</a>). The omit electron density, contoured at 5σ above the mean, is shown for the inhibitor (gray mesh). (C) Chemical structure of <b>N71</b>. (D) A covalent bond is formed between Cys481 and the C<sub>1</sub> carbon of <b>N71</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6DQB">6DQB</a>). (E) Two methyl groups of the capping dimethylamino-2-butenamide moiety formed weak H-bond contacts with main chain carbonyl oxygen atoms of Ser479 and Arg76, whereas the nitrogen was solvent exposed to water molecules (away from the viewer). (F) Superimposition of <b>N70</b> and <b>N71</b> bound with KDM5A. Note the rotation along the chiral carbon and pyrrolopyridine ring resulting in movement of 3-aminophenyl ring and alkoxyether piperidine moiety. (G) Chemical structure of Afatinib contains the same moiety of dimethylamino-2-butenamide. (H) Afatinib covalently inhibits EGFR kinase (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4G5J">4G5J</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01219/20181207/images/large/jm-2018-01219j_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01219&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01219/20181207/images/medium/jm-2018-01219j_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01219/20181207/images/large/jm-2018-01219j_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Irreversible inhibition of KDM5A in solution. (A) Summary of AlphaLISA derived IC<sub>50</sub> values from inhibition of demethylation by KDM5B and KDM4A (see <a class="ref internalNav" href="#notes1" aria-label="Figure S2">Figure S2</a> for original inhibition data). (B–G) IC<sub>50</sub> values of compounds <b>N70</b> and <b>N71</b> against KDM5A(1–739)ΔAP using the Promega Succinate-Glo assay under three αKG concentrations. The order involves preincubation (20 min) with the inhibitors before the addition of αKG to initiate the reaction. (B) Inhibition of KDM5A by <b>N70</b> is αKG dependent. (C) <b>N71</b> has no significant change to IC<sub>50</sub> values with variation in αKG concentration. (D) Inhibition of KDM5A by <b>N71</b> is enzyme concentration dependent. (E,F) Inhibition of KDM5A with <b>N71</b> (E) or <b>N70</b> (F) with or without incubation for 20 min. (G) Preformation of enzyme–inhibitor complex ([E] = 5 μM and initial [I] = 100 μM) followed by four times of buffer exchange removes the noncovalent modifying <b>N70</b> from solution but does not perturb <b>N71</b> complex with KDM5A. (H) KDM5A ([E] = 0.8 μM) was preincubated with five concentrations of <b>N71</b> ([I] = 0.01–5 μM). At each time point, the enzymatic reaction started by adding 5 μL of preincubated E+I complex to a solution containing H3 peptide ([S] = 60 μM) and αKG (100 μM), and the reaction was stopped after 7 min. (I) Inactivation plot, logarithm of the % remaining activity versus preincubation time. (J) Nonlinear regression analysis of the negative slopes against inhibitor concentration gives rise to <i>k</i><sub>inact</sub> and <i>K</i><sub>I</sub> values as indicated. (K) Summary of electrospray ionization mass spectrometry (ESI-tandem-MS) of KDM5A in the absence and presence of inhibitors (see <a class="ref internalNav" href="#notes1" aria-label="Figure S3">Figure S3</a> for raw spectrum). KDM5A (50 μM) was incubated with excess inhibitor (0.5 mM) for 30 min at ambient temperature.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01219/20181207/images/large/jm-2018-01219j_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01219&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01219/20181207/images/medium/jm-2018-01219j_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01219/20181207/images/large/jm-2018-01219j_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Concentration dependence of cytotoxicity of <b>N73</b>. (A) Chemical structures of ethyl (<b>GSK-J4</b> and <b>KDM5-C70</b>) or isopropyl ester (<b>N73</b> and <b>SI-5</b>) derivatives of indicated compounds. (B) PAMPA permeability. (C,D) Western blot analyses of MCF7 cells treated with the indicated concentration of compounds or DMSO (0.1%) control for 30 or 48 h. Total cells lysates were used for assays using indicated antibodies. (E) <b>N73</b> (isopropyl ester derivative of <b>N71</b>) does not increase H3K4me3 level in BT474 cells at 48 h as measured by immunocytochemistry. (F) Rapid cytotoxicity in BT474 cells treated with <b>N73</b> at concentrations >5 μM. Cell viability was measured using a CellTiter-Glo assay 24 h after treatment. (G,H) Proliferation of BT474 cells treated with <b>N73 or SI-5</b>. Percent confluence was tracked over 300 h by imaging the cells every 4 h using an IncuCyte live cell imaging system. (I) Real-time RT-PCR analysis of the indicated mRNAs in MCF7 cells treated with the indicated concentration of compounds or DMSO for 48 h. Data from biological triplicate experiments are shown.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01219/20181207/images/large/jm-2018-01219j_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01219&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01219/20181207/images/medium/jm-2018-01219j_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01219/20181207/images/large/jm-2018-01219j_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Images of bright-field microscopy of live cells after 24 h post-treatment with indicated concentrations of <b>KDM5-C70</b> (top), <b>N71</b> (middle), and <b>N73</b> (bottom) or 0.1% DMSO (upper left corner) in MCF7 (A), MDA-MB-231 (B), and HEK293T (C). Scale bar is 50 μM.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01219/20181207/images/large/jm-2018-01219j_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01219&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch2" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01219/20181207/images/medium/jm-2018-01219j_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01219/20181207/images/large/jm-2018-01219j_0008.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Compounds <b>N70</b>, <b>N71</b>, <b>N73</b>, and <b>SI-5</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-23/acs.jmedchem.8b01219/20181207/images/large/jm-2018-01219j_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01219&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i38">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_27482" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_27482" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 53 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gasser, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, E.</span></span> <span> </span><span class="NLM_article-title">Epigenetics and disease: pharmaceutical opportunities</span>. <i>Appl. Clin. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">67</span>, <span class="refDoi"> DOI: 10.2147/TACG.S11639</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=10.2147%2FTACG.S11639" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=67&author=S.+M.+Gasserauthor=E.+Li&title=Epigenetics+and+disease%3A+pharmaceutical+opportunities&doi=10.2147%2FTACG.S11639"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.2147%2FTACG.S11639&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2147%252FTACG.S11639%26sid%3Dliteratum%253Aachs%26aulast%3DGasser%26aufirst%3DS.%2BM.%26aulast%3DLi%26aufirst%3DE.%26atitle%3DEpigenetics%2520and%2520disease%253A%2520pharmaceutical%2520opportunities%26jtitle%3DAppl.%2520Clin.%2520Genet.%26date%3D2011%26volume%3D67%26spage%3D67%26doi%3D10.2147%2FTACG.S11639" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Black, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Rechem, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whetstine, J. R.</span></span> <span> </span><span class="NLM_article-title">Histone lysine methylation dynamics: establishment, regulation, and biological impact</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">491</span>– <span class="NLM_lpage">507</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2012.11.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=10.1016%2Fj.molcel.2012.11.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=23200123" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=1%3ACAS%3A528%3ADC%252BC38XhslKqtLvM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2012&pages=491-507&author=J.+C.+Blackauthor=C.+Van+Rechemauthor=J.+R.+Whetstine&title=Histone+lysine+methylation+dynamics%3A+establishment%2C+regulation%2C+and+biological+impact&doi=10.1016%2Fj.molcel.2012.11.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Histone Lysine Methylation Dynamics: Establishment, Regulation, and Biological Impact</span></div><div class="casAuthors">Black, Joshua C.; Van Rechem, Capucine; Whetstine, Johnathan R.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">491-507</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">A review.  Histone lysine methylation has emerged as a crit. player in the regulation of gene expression, cell cycle, genome stability, and nuclear architecture.  Over the past decade, a tremendous amt. of progress has led to the characterization of Me modifications and the lysine methyltransferases (KMTs) and lysine demethylases (KDMs) that regulate them.  Here, we review the discovery and characterization of the KMTs and KDMs and the Me modifications they regulate.  We discuss the localization of the KMTs and KDMs as well as the distribution of lysine methylation throughout the genome.  We highlight how these data have shaped our view of lysine methylation as a key determinant of complex chromatin states.  Finally, we discuss the regulation of KMTs and KDMs by proteasomal degrdn., posttranscriptional mechanisms, and metabolic status.  We propose key questions for the field and highlight areas that we predict will yield exciting discoveries in the years to come.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbnBk0Al-Ce7Vg90H21EOLACvtfcHk0ljZl_Xh-JXIqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhslKqtLvM&md5=b9a8e9c29c630592f5413f7e36974735</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2012.11.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2012.11.006%26sid%3Dliteratum%253Aachs%26aulast%3DBlack%26aufirst%3DJ.%2BC.%26aulast%3DVan%2BRechem%26aufirst%3DC.%26aulast%3DWhetstine%26aufirst%3DJ.%2BR.%26atitle%3DHistone%2520lysine%2520methylation%2520dynamics%253A%2520establishment%252C%2520regulation%252C%2520and%2520biological%2520impact%26jtitle%3DMol.%2520Cell%26date%3D2012%26volume%3D48%26spage%3D491%26epage%3D507%26doi%3D10.1016%2Fj.molcel.2012.11.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, K. N.</span></span> <span> </span><span class="NLM_article-title">Association between H3K4 methylation and cancer prognosis: A meta-analysis</span>. <i>Thorac. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">794</span>– <span class="NLM_lpage">799</span>, <span class="refDoi"> DOI: 10.1111/1759-7714.12647</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=10.1111%2F1759-7714.12647" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=29737623" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1ejsLjP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=794-799&author=S.+Liauthor=L.+Shenauthor=K.+N.+Chen&title=Association+between+H3K4+methylation+and+cancer+prognosis%3A+A+meta-analysis&doi=10.1111%2F1759-7714.12647"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Association between H3K4 methylation and cancer prognosis: A meta-analysis</span></div><div class="casAuthors">Li, Simin; Shen, Luyan; Chen, Ke-Neng</div><div class="citationInfo"><span class="NLM_cas:title">Thoracic Cancer</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">794-799</span>CODEN:
                <span class="NLM_cas:coden">TCHAEQ</span>;
        ISSN:<span class="NLM_cas:issn">1759-7706</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons Singapore Pte. Ltd.</span>)
        </div><div class="casAbstract">Background : Histone H3 lysine 4 methylation (H3K4 methylation), including mono-methylation (H3K4me1), di-methylation (H3K4me2), or tri-methylation (H3K4me3), is one of the epigenetic modifications to histone proteins, which are related to the transcriptional activation of genes.  H3K4 methylation has both tumor inhibiting and promoting effects, and the prognostic value of H3K4 methylation in cancer remains controversial.  Therefore, we performed a systematic review and meta-anal. to examine the assocn. between H3K4 methylation and cancer prognosis.  Methods : A comprehensive search of PubMed, Web of Science, ScienceDirect, Embase, and Ovid databases was conducted to identify studies investigating the assocn. between H3K4 methylation and prognosis of patients with malignant tumors.  The data and characteristics of each study were extd., and the hazard ratio (HR) at a 95% confidence interval (CI) was calcd. to est. the effect.  Results : A total of 1474 patients in 10 studies were enrolled in this meta-anal.  The pooled HR of 1.52 (95% CI 1.02-2.26) indicated that patients with a lower level of H3K4me2 expression were expected to have shorter overall survival, while the pooled HR of 0.45 (95% CI 0.27-0.74) indicated that patients with a lower level of H3K4me3 expression were expected to have longer overall survival.  Conclusion : This meta-anal. indicates that increased H3K4me3 expression and decreased H3K4me2 expression might be predictive factors of poor prognosis in cancer.  Further large cohort studies are needed to confirm these findings.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoTCdN4iOwtxbVg90H21EOLACvtfcHk0ljZl_Xh-JXIqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1ejsLjP&md5=3c7ebe189d60416fb77d5dc436539c33</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1111%2F1759-7714.12647&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252F1759-7714.12647%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DS.%26aulast%3DShen%26aufirst%3DL.%26aulast%3DChen%26aufirst%3DK.%2BN.%26atitle%3DAssociation%2520between%2520H3K4%2520methylation%2520and%2520cancer%2520prognosis%253A%2520A%2520meta-analysis%26jtitle%3DThorac.%2520Cancer%26date%3D2018%26volume%3D9%26spage%3D794%26epage%3D799%26doi%3D10.1111%2F1759-7714.12647" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulligan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whetstine, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casero, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span> <span> </span><span class="NLM_article-title">Histone demethylation mediated by the nuclear amine oxidase homolog LSD1</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>119</i></span>,  <span class="NLM_fpage">941</span>– <span class="NLM_lpage">53</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2004.12.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=10.1016%2Fj.cell.2004.12.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=15620353" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=1%3ACAS%3A528%3ADC%252BD2MXltlSitg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=2004&pages=941-53&author=Y.+Shiauthor=F.+Lanauthor=C.+Matsonauthor=P.+Mulliganauthor=J.+R.+Whetstineauthor=P.+A.+Coleauthor=R.+A.+Caseroauthor=Y.+Shi&title=Histone+demethylation+mediated+by+the+nuclear+amine+oxidase+homolog+LSD1&doi=10.1016%2Fj.cell.2004.12.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Histone demethylation mediated by the nuclear amine oxidase homolog LSD1</span></div><div class="casAuthors">Shi, Yujiang; Lan, Fei; Matson, Caitlin; Mulligan, Peter; Whetstine, Johnathan R.; Cole, Philip A.; Casero, Robert A.; Shi, Yang</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">119</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">941-953</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Posttranslational modifications of histone N-terminal tails impact chromatin structure and gene transcription.  While the extent of histone acetylation is detd. by both acetyltransferases and deacetylases, it has been unclear whether histone methylation is also regulated by enzymes with opposing activities.  Here, we provide evidence that LSD1 (KIAA0601), a nuclear homolog of amine oxidases, functions as a histone demethylase and transcriptional corepressor.  LSD1 specifically demethylates histone H3 lysine 4, which is linked to active transcription.  Lysine demethylation occurs via an oxidn. reaction that generates formaldehyde.  Importantly, RNAi inhibition of LSD1 causes an increase in H3 lysine 4 methylation and concomitant derepression of target genes, suggesting that LSD1 represses transcription via histone demethylation.  The results thus identify a histone demethylase conserved from S. pombe to human and reveal dynamic regulation of histone methylation by both histone methylases and demethylases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXHr-8kwGdXrVg90H21EOLACvtfcHk0ljZl_Xh-JXIqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXltlSitg%253D%253D&md5=3474569204255e90a832667ba654748b</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2004.12.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2004.12.012%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DY.%26aulast%3DLan%26aufirst%3DF.%26aulast%3DMatson%26aufirst%3DC.%26aulast%3DMulligan%26aufirst%3DP.%26aulast%3DWhetstine%26aufirst%3DJ.%2BR.%26aulast%3DCole%26aufirst%3DP.%2BA.%26aulast%3DCasero%26aufirst%3DR.%2BA.%26aulast%3DShi%26aufirst%3DY.%26atitle%3DHistone%2520demethylation%2520mediated%2520by%2520the%2520nuclear%2520amine%2520oxidase%2520homolog%2520LSD1%26jtitle%3DCell%26date%3D2004%26volume%3D119%26spage%3D941%26epage%3D53%26doi%3D10.1016%2Fj.cell.2004.12.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsukada, Y.</span></span> <span> </span><span class="NLM_article-title">The discovery of histone demethylases</span>. <i>Cold Spring Harbor Perspect. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">a022335</span>, <span class="refDoi"> DOI: 10.1101/cshperspect.a022335</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=10.1101%2Fcshperspect.a022335" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFCiurnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2013&pages=a022335&author=Y.+G.+Shiauthor=Y.+Tsukada&title=The+discovery+of+histone+demethylases&doi=10.1101%2Fcshperspect.a022335"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">The discovery of histone demethylases [Erratum to document cited in CA160:002399]</span></div><div class="casAuthors">Shi, Yujiang Geno; Tsukada, Yuichi</div><div class="citationInfo"><span class="NLM_cas:title">Cold Spring Harbor Perspectives in Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">a022335/1</span>CODEN:
                <span class="NLM_cas:coden">CSHPEU</span>;
        ISSN:<span class="NLM_cas:issn">1943-0264</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">A review.  The authors felt the published article did not provide a sufficiently accurate account of the discovery of histone demethylases; the description has been modified in the PDF and HTML versions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOorDbV4YtbrVg90H21EOLACvtfcHk0ljh6OuXRCKNqw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFCiurnF&md5=c935849fd498b3a27ca906dec4d01721</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1101%2Fcshperspect.a022335&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fcshperspect.a022335%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DY.%2BG.%26aulast%3DTsukada%26aufirst%3DY.%26atitle%3DThe%2520discovery%2520of%2520histone%2520demethylases%26jtitle%3DCold%2520Spring%2520Harbor%2520Perspect.%2520Biol.%26date%3D2013%26volume%3D5%26spage%3Da022335%26doi%3D10.1101%2Fcshperspect.a022335" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jambhekar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anastas, J. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span> <span> </span><span class="NLM_article-title">Histone lysine demethylase inhibitors</span>. <i>Cold Spring Harbor Perspect. Med.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">a026484</span>, <span class="refDoi"> DOI: 10.1101/cshperspect.a026484</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=10.1101%2Fcshperspect.a026484" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=28049654" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtFKms7vF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=a026484&author=A.+Jambhekarauthor=J.+N.+Anastasauthor=Y.+Shi&title=Histone+lysine+demethylase+inhibitors&doi=10.1101%2Fcshperspect.a026484"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Histone lysine demethylase inhibitors</span></div><div class="casAuthors">Jambhekar, Ashwini; Anastas, Jamie N.; Shi, Yang</div><div class="citationInfo"><span class="NLM_cas:title">Cold Spring Harbor Perspectives in Medicine</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">a026484/1-a026484/26</span>CODEN:
                <span class="NLM_cas:coden">CSHPFV</span>;
        ISSN:<span class="NLM_cas:issn">2157-1422</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">The dynamic regulation of covalent modifications to histones is essential for maintaining genomic integrity and cell identity and is often compromised in cancer.  Aberrant expression of histone lysine demethylases has been documented in many types of blood and solid tumors, and thus demethylases represent promising therapeutic targets.  Recent advances in high-throughput chem. screening, structure-based drug design, and structure-activity relationship studies have improved both the specificity and the in vivo efficacy of demethylase inhibitors.  This review will briefly outline the connection between demethylases and cancer and will provide a comprehensive overview of the structure, specificity, and utility of currently available demethylase inhibitors.  To date, a select group of demethylase inhibitors is being evaluated in clin. trials, and addnl. compds. may soon follow from the bench to the bedside.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoqMUZw4KjjDLVg90H21EOLACvtfcHk0ljh6OuXRCKNqw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtFKms7vF&md5=22ba67d04750ab8d410baee59a64d7e6</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1101%2Fcshperspect.a026484&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fcshperspect.a026484%26sid%3Dliteratum%253Aachs%26aulast%3DJambhekar%26aufirst%3DA.%26aulast%3DAnastas%26aufirst%3DJ.%2BN.%26aulast%3DShi%26aufirst%3DY.%26atitle%3DHistone%2520lysine%2520demethylase%2520inhibitors%26jtitle%3DCold%2520Spring%2520Harbor%2520Perspect.%2520Med.%26date%3D2017%26volume%3D7%26spage%3Da026484%26doi%3D10.1101%2Fcshperspect.a026484" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H. M.</span></span> <span> </span><span class="NLM_article-title">A systematic review of histone lysine-specific demethylase 1 and Its inhibitors</span>. <i>Med. Res. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">1032</span>– <span class="NLM_lpage">1071</span>, <span class="refDoi"> DOI: 10.1002/med.21350</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=10.1002%2Fmed.21350" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=25990136" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1yqtrfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2015&pages=1032-1071&author=Y.+C.+Zhengauthor=J.+Maauthor=Z.+Wangauthor=J.+Liauthor=B.+Jiangauthor=W.+Zhouauthor=X.+Shiauthor=X.+Wangauthor=W.+Zhaoauthor=H.+M.+Liu&title=A+systematic+review+of+histone+lysine-specific+demethylase+1+and+Its+inhibitors&doi=10.1002%2Fmed.21350"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">A Systematic Review of Histone Lysine-Specific Demethylase 1 and Its Inhibitors</span></div><div class="casAuthors">Zheng, Yi-Chao; Ma, Jinlian; Wang, Zhiru; Li, Jinfeng; Jiang, Bailing; Zhou, Wenjuan; Shi, Xiaojing; Wang, Xixin; Zhao, Wen; Liu, Hong-Min</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Research Reviews</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1032-1071</span>CODEN:
                <span class="NLM_cas:coden">MRREDD</span>;
        ISSN:<span class="NLM_cas:issn">0198-6325</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">Histone lysine-specific demethylase 1 (LSD1) is the first discovered and reported histone demethylase by Dr. Shi Yang's group in 2004.  It is classified as a member of amine oxidase superfamily, the common feature of which is using the FAD (FAD) as its cofactor.  Since it is located in cell nucleus and acts as a histone methylation eraser, LSD1 specifically removes mono- or dimethylated histone H3 lysine 4 (H3K4) and H3 lysine 9 (H3K9) through formaldehyde-generating oxidn.  It has been indicated that LSD1 and its downstream targets are involved in a wide range of biol. courses, including embryonic development and tumor-cell growth and metastasis.  LSD1 has been reported to be overexpressed in variety of tumors.  Inactivating LSD1 or downregulating its expression inhibits cancer-cell development.  LSD1 targeting inhibitors may represent a new insight in anticancer drug discovery.  This review summarizes recent studies about LSD1 and mainly focuses on the basic physiol. function of LSD1 and its involved mechanisms in pathophysiol. conditions, as well as the development of LSD1 inhibitors as potential anticancer therapeutic agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7JXHHRI-QAbVg90H21EOLACvtfcHk0lisTTcClDKebw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1yqtrfN&md5=43b875856c2fe9f7a02857e7d6de73e6</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1002%2Fmed.21350&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmed.21350%26sid%3Dliteratum%253Aachs%26aulast%3DZheng%26aufirst%3DY.%2BC.%26aulast%3DMa%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DJiang%26aufirst%3DB.%26aulast%3DZhou%26aufirst%3DW.%26aulast%3DShi%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DZhao%26aufirst%3DW.%26aulast%3DLiu%26aufirst%3DH.%2BM.%26atitle%3DA%2520systematic%2520review%2520of%2520histone%2520lysine-specific%2520demethylase%25201%2520and%2520Its%2520inhibitors%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2015%26volume%3D35%26spage%3D1032%26epage%3D1071%26doi%3D10.1002%2Fmed.21350" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Christensen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agger, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cloos, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pasini, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rose, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sennels, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rappsilber, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salcini, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helin, K.</span></span> <span> </span><span class="NLM_article-title">RBP2 belongs to a family of demethylases, specific for tri-and dimethylated lysine 4 on histone 3</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>128</i></span>,  <span class="NLM_fpage">1063</span>– <span class="NLM_lpage">1076</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2007.02.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=10.1016%2Fj.cell.2007.02.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=17320161" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=1%3ACAS%3A528%3ADC%252BD2sXkslGms78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2007&pages=1063-1076&author=J.+Christensenauthor=K.+Aggerauthor=P.+A.+Cloosauthor=D.+Pasiniauthor=S.+Roseauthor=L.+Sennelsauthor=J.+Rappsilberauthor=K.+H.+Hansenauthor=A.+E.+Salciniauthor=K.+Helin&title=RBP2+belongs+to+a+family+of+demethylases%2C+specific+for+tri-and+dimethylated+lysine+4+on+histone+3&doi=10.1016%2Fj.cell.2007.02.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">RBP2 belongs to a family of demethylases, specific for triand dimethylated lysine 4 on histone 3</span></div><div class="casAuthors">Christensen, Jesper; Agger, Karl; Cloos, Paul A. C.; Pasini, Diego; Rose, Simon; Sennels, Lau; Rappsilber, Juri; Hansen, Klaus H.; Salcini, Anna Elisabetta; Helin, Kristian</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">128</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1063-1076</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Methylation of histones has been regarded as a stable modification defining the epigenetic program of the cell, which regulates chromatin structure and transcription.  However, the recent discovery of histone demethylases has challenged the stable nature of histone methylation.  Here we demonstrate that the JARID1 proteins RBP2, PLU1, and SMCX are histone demethylases specific for di- and trimethylated histone 3 lysine 4 (H3K4).  Consistent with a role for the JARID1 Drosophila homolog Lid in regulating expression of homeotic genes during development, we show that RBP2 is displaced from Hox genes during embryonic stem (ES) cell differentiation correlating with an increase of their H3K4me3 levels and expression.  Furthermore, we show that mutation or RNAi depletion of the C. elegans JARID1 homolog rbr-2 leads to increased levels of H3K4me3 during larval development and defects in vulva formation.  Taken together, these results suggest that H3K4me3/me2 demethylation regulated by the JARID1 family plays an important role during development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqoiLQh_7D2g7Vg90H21EOLACvtfcHk0lisTTcClDKebw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXkslGms78%253D&md5=c1f04b597253ddc6d89b62bf0b99b13c</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2007.02.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2007.02.003%26sid%3Dliteratum%253Aachs%26aulast%3DChristensen%26aufirst%3DJ.%26aulast%3DAgger%26aufirst%3DK.%26aulast%3DCloos%26aufirst%3DP.%2BA.%26aulast%3DPasini%26aufirst%3DD.%26aulast%3DRose%26aufirst%3DS.%26aulast%3DSennels%26aufirst%3DL.%26aulast%3DRappsilber%26aufirst%3DJ.%26aulast%3DHansen%26aufirst%3DK.%2BH.%26aulast%3DSalcini%26aufirst%3DA.%2BE.%26aulast%3DHelin%26aufirst%3DK.%26atitle%3DRBP2%2520belongs%2520to%2520a%2520family%2520of%2520demethylases%252C%2520specific%2520for%2520tri-and%2520dimethylated%2520lysine%25204%2520on%2520histone%25203%26jtitle%3DCell%26date%3D2007%26volume%3D128%26spage%3D1063%26epage%3D1076%26doi%3D10.1016%2Fj.cell.2007.02.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Blair, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sayegh, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, Q.</span></span> <span> </span><span class="NLM_article-title">Epigenetic regulation by lysine demethylase 5 (KDM5) enzymes in cancer</span>. <i>Cancers</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">1383</span>– <span class="NLM_lpage">1404</span>, <span class="refDoi"> DOI: 10.3390/cancers3011383</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=10.3390%2Fcancers3011383" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=21544224" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktVWiurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2011&pages=1383-1404&author=L.+P.+Blairauthor=J.+Caoauthor=M.+R.+Zouauthor=J.+Sayeghauthor=Q.+Yan&title=Epigenetic+regulation+by+lysine+demethylase+5+%28KDM5%29+enzymes+in+cancer&doi=10.3390%2Fcancers3011383"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Epigenetic regulation by lysine demethylase 5 (KDM5) enzymes in cancer</span></div><div class="casAuthors">Blair, Lauren P.; Cao, Jian; Zou, Mike Ran; Sayegh, Joyce; Yan, Qin</div><div class="citationInfo"><span class="NLM_cas:title">Cancers</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1383-1404</span>CODEN:
                <span class="NLM_cas:coden">CANCCT</span>;
        ISSN:<span class="NLM_cas:issn">2072-6694</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">A review.  Similar to genetic alterations, epigenetic aberrations contribute significantly to tumor initiation and progression.  In many cases, these changes are caused by activation or inactivation of the regulators that maintain epigenetic states.  Here we review our current knowledge on the KDM5/JARID1 family of histone demethylases.  This family of enzymes contains a JmjC domain and is capable of removing tri- and di- Me marks from lysine 4 on histone H3.  Among these proteins, RBP2 mediates drug resistance while JARID1B is required for melanoma maintenance.  Preclin. studies suggest inhibition of these enzymes can suppress tumorigenesis and provide strong rationale for development of their inhibitors for use in cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8lTt6GjrOJrVg90H21EOLACvtfcHk0lisTTcClDKebw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktVWiurg%253D&md5=426fa61d08aa818b45781b57e1aba5f3</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.3390%2Fcancers3011383&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fcancers3011383%26sid%3Dliteratum%253Aachs%26aulast%3DBlair%26aufirst%3DL.%2BP.%26aulast%3DCao%26aufirst%3DJ.%26aulast%3DZou%26aufirst%3DM.%2BR.%26aulast%3DSayegh%26aufirst%3DJ.%26aulast%3DYan%26aufirst%3DQ.%26atitle%3DEpigenetic%2520regulation%2520by%2520lysine%2520demethylase%25205%2520%2528KDM5%2529%2520enzymes%2520in%2520cancer%26jtitle%3DCancers%26date%3D2011%26volume%3D3%26spage%3D1383%26epage%3D1404%26doi%3D10.3390%2Fcancers3011383" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rasmussen, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Staller, P.</span></span> <span> </span><span class="NLM_article-title">The KDM5 family of histone demethylases as targets in oncology drug discovery</span>. <i>Epigenomics</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">277</span>– <span class="NLM_lpage">286</span>, <span class="refDoi"> DOI: 10.2217/epi.14.14</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=10.2217%2Fepi.14.14" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=25111482" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtlans77K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2014&pages=277-286&author=P.+B.+Rasmussenauthor=P.+Staller&title=The+KDM5+family+of+histone+demethylases+as+targets+in+oncology+drug+discovery&doi=10.2217%2Fepi.14.14"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">The KDM5 family of histone demethylases as targets in oncology drug discovery</span></div><div class="casAuthors">Rasmussen, Peter Birk; Staller, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Epigenomics</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">277-286</span>CODEN:
                <span class="NLM_cas:coden">EPIGC7</span>;
        ISSN:<span class="NLM_cas:issn">1750-192X</span>.
    
            (<span class="NLM_cas:orgname">Future Medicine Ltd.</span>)
        </div><div class="casAbstract">A review.  There is growing evidence for a causal role of the KDM5 family of histone demethylases in human cancer.  In particular, KDM5A (JARID1A/RBP2) and KDM5B (JARID1B/PLU1) contribute to cancer cell proliferation, reduce the expression of tumor suppressor genes, promote the development of drug tolerance and maintain tumor-initiating cells.  KDM5 enzymes remove tri- and di-methylations of lysine 4 of histone H3 - modifications that occur at the start site of transcription in actively transcribed genes.  However, the importance of the histone demethylase activity of KDM5 proteins for cancer cells has not been resolved so far.  The currently available approaches suppress or remove the targeted proteins and thereby affect their putative functions as structural components and recruitment factors for other chromatin-assocd. proteins.  Therefore, the development of specific enzymic inhibitors for KDM5 will promote our understanding of the biol. role of their catalytic activity and yield potential novel anticancer therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPJyGCVkwViLVg90H21EOLACvtfcHk0ljDf0ZsjAX4XA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtlans77K&md5=0b8dc67f4fde6ce24457dce092dbdd83</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.2217%2Fepi.14.14&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2217%252Fepi.14.14%26sid%3Dliteratum%253Aachs%26aulast%3DRasmussen%26aufirst%3DP.%2BB.%26aulast%3DStaller%26aufirst%3DP.%26atitle%3DThe%2520KDM5%2520family%2520of%2520histone%2520demethylases%2520as%2520targets%2520in%2520oncology%2520drug%2520discovery%26jtitle%3DEpigenomics%26date%3D2014%26volume%3D6%26spage%3D277%26epage%3D286%26doi%3D10.2217%2Fepi.14.14" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Harmeyer, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Facompre, N. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herlyn, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Basu, D.</span></span> <span> </span><span class="NLM_article-title">JARID1 histone demethylases: emerging targets in cancer</span>. <i>Trends Cancer</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">713</span>– <span class="NLM_lpage">725</span>, <span class="refDoi"> DOI: 10.1016/j.trecan.2017.08.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=10.1016%2Fj.trecan.2017.08.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=28958389" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=1%3ACAS%3A528%3ADC%252BC1MXksVyitg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2017&pages=713-725&author=K.+M.+Harmeyerauthor=N.+D.+Facompreauthor=M.+Herlynauthor=D.+Basu&title=JARID1+histone+demethylases%3A+emerging+targets+in+cancer&doi=10.1016%2Fj.trecan.2017.08.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">JARID1 Histone Demethylases: Emerging Targets in Cancer</span></div><div class="casAuthors">Harmeyer, Kayla M.; Facompre, Nicole D.; Herlyn, Meenhard; Basu, Devraj</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Cancer</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">713-725</span>CODEN:
                <span class="NLM_cas:coden">TCRAB5</span>;
        ISSN:<span class="NLM_cas:issn">2405-8033</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  JARID1 proteins are histone demethylases that both regulate normal cell fates during development and contribute to the epigenetic plasticity that underlies malignant transformation.  This H3K4 demethylase family participates in multiple repressive transcriptional complexes at promoters and has broader regulatory effects on chromatin that remain ill-defined.  There is growing understanding of the oncogenic and tumor suppressive functions of JARID1 proteins, which are contingent on cell context and the protein isoform.  Their contributions to stem cell-like dedifferentiation, tumor aggressiveness, and therapy resistance in cancer have sustained interest in the development of JARID1 inhibitors.  Here we review the diverse and context-specific functions of the JARID1 proteins that may impact the utilization of emerging targeted inhibitors of this histone demethylase family in cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJYLbdIKcjybVg90H21EOLACvtfcHk0ljDf0ZsjAX4XA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXksVyitg%253D%253D&md5=e728e47a8d60d5e83f29ab6d153a2be4</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.trecan.2017.08.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.trecan.2017.08.004%26sid%3Dliteratum%253Aachs%26aulast%3DHarmeyer%26aufirst%3DK.%2BM.%26aulast%3DFacompre%26aufirst%3DN.%2BD.%26aulast%3DHerlyn%26aufirst%3DM.%26aulast%3DBasu%26aufirst%3DD.%26atitle%3DJARID1%2520histone%2520demethylases%253A%2520emerging%2520targets%2520in%2520cancer%26jtitle%3DTrends%2520Cancer%26date%3D2017%26volume%3D3%26spage%3D713%26epage%3D725%26doi%3D10.1016%2Fj.trecan.2017.08.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shokri, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doudi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fathi-Roudsari, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kouhkan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanati, M. H.</span></span> <span> </span><span class="NLM_article-title">Targeting histone demethylases KDM5A and KDM5B in AML cancer cells: A comparative view</span>. <i>Leuk. Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>68</i></span>,  <span class="NLM_fpage">105</span>– <span class="NLM_lpage">111</span>, <span class="refDoi"> DOI: 10.1016/j.leukres.2018.02.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=10.1016%2Fj.leukres.2018.02.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=29602065" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=1%3ACAS%3A528%3ADC%252BC1cXmsVaru7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2018&pages=105-111&author=G.+Shokriauthor=S.+Doudiauthor=M.+Fathi-Roudsariauthor=F.+Kouhkanauthor=M.+H.+Sanati&title=Targeting+histone+demethylases+KDM5A+and+KDM5B+in+AML+cancer+cells%3A+A+comparative+view&doi=10.1016%2Fj.leukres.2018.02.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting histone demethylases KDM5A and KDM5B in AML cancer cells: A comparative view</span></div><div class="casAuthors">Shokri, Gelareh; Doudi, Shaghayegh; Fathi-Roudsari, Mehrnoosh; Kouhkan, Fatemeh; Sanati, Mohammad-Hossein</div><div class="citationInfo"><span class="NLM_cas:title">Leukemia Research</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">105-111</span>CODEN:
                <span class="NLM_cas:coden">LEREDD</span>;
        ISSN:<span class="NLM_cas:issn">0145-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Epigenetic modifications play an important role in initiation and progression of cancers including acute myeloid leukemia.  Among different epigenetic modifiers, lysine specific demethylases have been noticed as potential therapeutic targets.  KDM5 family of histone demethylases which removes Me marks from lysine residues of H3, are frequently found in the promoter region of transcriptionally active genes resulting in repression of expression.  Here we have compared the effects of KDM5A and KDM5B downregulation on HL-60 cell line behavior.  KDM5A/5B knockdown resulted in lower viability of HL-60 cells in addn. to modified cell cycle distribution and sub-G1 accumulation.  Induction of apoptosis was obsd. in both knockdown cells.  But in spite of similarity in their role, downregulation of KDM5A showed more efficient anti-leukemic effects in comparison to KDM5B.  Cells showed higher accumulation in sub-G1 and apoptosis occurred significantly higher and also earlier after KDM5A redn.  Expression anal. confirmed almost 5 and 4 fold increased expression for bax and caspase-3 after downregulation of KDM5A in comparison to KDM5B.  Due to the present study we propose KDM5A as a potential target for therapeutic aspects of acute myeloid leukemia although further investigations are needed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcIUHbiIE13rVg90H21EOLACvtfcHk0ljDf0ZsjAX4XA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXmsVaru7k%253D&md5=22a34070de67e600b8164273c1112405</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.leukres.2018.02.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.leukres.2018.02.003%26sid%3Dliteratum%253Aachs%26aulast%3DShokri%26aufirst%3DG.%26aulast%3DDoudi%26aufirst%3DS.%26aulast%3DFathi-Roudsari%26aufirst%3DM.%26aulast%3DKouhkan%26aufirst%3DF.%26aulast%3DSanati%26aufirst%3DM.%2BH.%26atitle%3DTargeting%2520histone%2520demethylases%2520KDM5A%2520and%2520KDM5B%2520in%2520AML%2520cancer%2520cells%253A%2520A%2520comparative%2520view%26jtitle%3DLeuk.%2520Res.%26date%3D2018%26volume%3D68%26spage%3D105%26epage%3D111%26doi%3D10.1016%2Fj.leukres.2018.02.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Labadie, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ortwine, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vinogradova, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiefer, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mauer, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gehling, V. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harmange, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cummings, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gustafson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van der Porten, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liederer, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deshmukh, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">An, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ran, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Classon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trojer, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dragovich, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, L.</span></span> <span> </span><span class="NLM_article-title">From a novel HTS hit to potent, selective, and orally bioavailable KDM5 inhibitors</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">2974</span>– <span class="NLM_lpage">2981</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2017.05.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=10.1016%2Fj.bmcl.2017.05.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=28512031" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=1%3ACAS%3A528%3ADC%252BC2sXnvFCqtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2017&pages=2974-2981&author=J.+Liangauthor=S.+Labadieauthor=B.+Zhangauthor=D.+F.+Ortwineauthor=S.+Patelauthor=M.+Vinogradovaauthor=J.+R.+Kieferauthor=T.+Mauerauthor=V.+S.+Gehlingauthor=J.+C.+Harmangeauthor=R.+Cummingsauthor=T.+Laiauthor=J.+Liaoauthor=X.+Zhengauthor=Y.+Liuauthor=A.+Gustafsonauthor=E.+Van+der+Portenauthor=W.+Maoauthor=B.+M.+Liedererauthor=G.+Deshmukhauthor=L.+Anauthor=Y.+Ranauthor=M.+Classonauthor=P.+Trojerauthor=P.+S.+Dragovichauthor=L.+Murray&title=From+a+novel+HTS+hit+to+potent%2C+selective%2C+and+orally+bioavailable+KDM5+inhibitors&doi=10.1016%2Fj.bmcl.2017.05.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">From a novel HTS hit to potent, selective, and orally bioavailable KDM5 inhibitors</span></div><div class="casAuthors">Liang, Jun; Labadie, Sharada; Zhang, Birong; Ortwine, Daniel F.; Patel, Snahel; Vinogradova, Maia; Kiefer, James R.; Mauer, Till; Gehling, Victor S.; Harmange, Jean-Christophe; Cummings, Richard; Lai, Tommy; Liao, Jiangpeng; Zheng, Xiaoping; Liu, Yichin; Gustafson, Amy; Van der Porten, Erica; Mao, Weifeng; Liederer, Bianca M.; Deshmukh, Gauri; An, Le; Ran, Yingqing; Classon, Marie; Trojer, Patrick; Dragovich, Peter S.; Murray, Lesley</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">2974-2981</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A high-throughput screening (HTS) of the Genentech/Roche library identified a novel, uncharged scaffold as a KDM5A inhibitor.  Lacking insight into the binding mode, initial attempts to improve inhibitor potency failed to improve potency, and synthesis of analogs was further hampered by the presence of a C-C bond between the pyrrolidine and pyridine.  Replacing this with a C-N bond significantly simplified synthesis, yielding pyrazole analog, [3-(4-bromo-1H-pyrazol-1-yl)-1-pyrrolidinyl][5-(1-methylethyl)-1H-pyrazol-3-yl]methanone (35), of which the authors obtained a co-crystal structure with KDM5A.  Using structure-based design approach, the authors identified, N-[(3R)-1-[[5-(1-methylethyl)-1H-pyrazol-3-yl]carbonyl]-3-pyrrolidinyl]cyclopropanecarboxamide (50), with improved biochem., cell potency and reduced MW and lower lipophilicity (Log D) compared with the original hit.  Furthermore, 50 showed lower clearance than [5-(1-methylethyl)-1H-pyrazol-3-yl][(3S)-3-[6-methyl-4-(1-methyl-1H-pyrazol-4-yl)-2-pyridinyl]-1-pyrrolidinyl]methanone (9) in mice.  In combination with its remarkably low plasma protein binding (PPB) in mice (40%), oral dosing of 50 at 5 mg/kg resulted in unbound Cmax ∼2-fold of its cell potency (PC9 H3K4Me3 0.96 μM), meeting the authors' criteria for an in vivo tool compd. from a new scaffold.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpksjXQBrVYsrVg90H21EOLACvtfcHk0ljs_2SjZJQGYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXnvFCqtr4%253D&md5=04bee4e29775202fb1882adeccb8e722</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2017.05.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2017.05.016%26sid%3Dliteratum%253Aachs%26aulast%3DLiang%26aufirst%3DJ.%26aulast%3DLabadie%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DB.%26aulast%3DOrtwine%26aufirst%3DD.%2BF.%26aulast%3DPatel%26aufirst%3DS.%26aulast%3DVinogradova%26aufirst%3DM.%26aulast%3DKiefer%26aufirst%3DJ.%2BR.%26aulast%3DMauer%26aufirst%3DT.%26aulast%3DGehling%26aufirst%3DV.%2BS.%26aulast%3DHarmange%26aufirst%3DJ.%2BC.%26aulast%3DCummings%26aufirst%3DR.%26aulast%3DLai%26aufirst%3DT.%26aulast%3DLiao%26aufirst%3DJ.%26aulast%3DZheng%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DGustafson%26aufirst%3DA.%26aulast%3DVan%2Bder%2BPorten%26aufirst%3DE.%26aulast%3DMao%26aufirst%3DW.%26aulast%3DLiederer%26aufirst%3DB.%2BM.%26aulast%3DDeshmukh%26aufirst%3DG.%26aulast%3DAn%26aufirst%3DL.%26aulast%3DRan%26aufirst%3DY.%26aulast%3DClasson%26aufirst%3DM.%26aulast%3DTrojer%26aufirst%3DP.%26aulast%3DDragovich%26aufirst%3DP.%2BS.%26aulast%3DMurray%26aufirst%3DL.%26atitle%3DFrom%2520a%2520novel%2520HTS%2520hit%2520to%2520potent%252C%2520selective%252C%2520and%2520orally%2520bioavailable%2520KDM5%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2017%26volume%3D27%26spage%3D2974%26epage%3D2981%26doi%3D10.1016%2Fj.bmcl.2017.05.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Horton, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shanks, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rai, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mott, B. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jansen, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kales, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henderson, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pohida, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jadhav, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maloney, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simeonov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vertino, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, X.</span></span> <span> </span><span class="NLM_article-title">Insights into the action of inhibitor enantiomers against histone lysine demethylase 5A</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">3193</span>– <span class="NLM_lpage">3208</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00261</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00261" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=1%3ACAS%3A528%3ADC%252BC1cXkslWmtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=3193-3208&author=J.+R.+Hortonauthor=X.+Liuauthor=L.+Wuauthor=K.+Zhangauthor=J.+Shanksauthor=X.+Zhangauthor=G.+Raiauthor=B.+T.+Mottauthor=D.+J.+Jansenauthor=S.+C.+Kalesauthor=M.+J.+Hendersonauthor=K.+Pohidaauthor=Y.+Fangauthor=X.+Huauthor=A.+Jadhavauthor=D.+J.+Maloneyauthor=M.+D.+Hallauthor=A.+Simeonovauthor=H.+Fuauthor=P.+M.+Vertinoauthor=Q.+Yanauthor=X.+Cheng&title=Insights+into+the+action+of+inhibitor+enantiomers+against+histone+lysine+demethylase+5A&doi=10.1021%2Facs.jmedchem.8b00261"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Insights into the Action of Inhibitor Enantiomers against Histone Lysine Demethylase 5A</span></div><div class="casAuthors">Horton, John R.; Liu, Xu; Wu, Lizhen; Zhang, Kai; Shanks, John; Zhang, Xing; Rai, Ganesha; Mott, Bryan T.; Jansen, Daniel J.; Kales, Stephen C.; Henderson, Mark J.; Pohida, Katherine; Fang, Yuhong; Hu, Xin; Jadhav, Ajit; Maloney, David J.; Hall, Matthew D.; Simeonov, Anton; Fu, Haian; Vertino, Paula M.; Yan, Qin; Cheng, Xiaodong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3193-3208</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Isomers of chiral drugs can exhibit marked differences in biol. activities.  We studied the binding and inhibitory activities of 12 compds. against KDM5A.  Among them are two pairs of enantiomers representing two distinct inhibitor chemotypes, namely, (R)- and (S)-2-((2-chlorophenyl)(2-(piperidin-1-yl)ethoxy)methyl)-1H-pyrrolo[3,2-b]pyridine-7-carboxylic acid (compds. N51 and N52) and (R)- and (S)-N-(1-(3-isopropyl-1H-pyrazole-5-carbonyl)pyrrolidin-3-yl)cyclopropanecarboxamide (compds. N54 and N55).  In vitro, the S enantiomer of the N51/N52 pair (N52) and the R enantiomer of the N54/N55 pair (N54) exhibited about 4- to 5-fold greater binding affinity.  The more potent enzyme inhibition of KDM5A by the R-isoform for the cell-permeable N54/N55 pair translated to differences in growth inhibitory activity.  We detd. structures of the KDM5A catalytic domain in complex with all 12 inhibitors, which revealed the interactions (or lack thereof) responsible for the differences in binding affinity.  These results provide insights to guide improvements in binding potency and avenues for development of cell permeable inhibitors of the KDM5 family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovFfXyPcUszbVg90H21EOLACvtfcHk0lj7gyCUELFeYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXkslWmtro%253D&md5=13ecac6902afd69d4ce7ad05803ca4d9</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00261&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00261%26sid%3Dliteratum%253Aachs%26aulast%3DHorton%26aufirst%3DJ.%2BR.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DWu%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DK.%26aulast%3DShanks%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DRai%26aufirst%3DG.%26aulast%3DMott%26aufirst%3DB.%2BT.%26aulast%3DJansen%26aufirst%3DD.%2BJ.%26aulast%3DKales%26aufirst%3DS.%2BC.%26aulast%3DHenderson%26aufirst%3DM.%2BJ.%26aulast%3DPohida%26aufirst%3DK.%26aulast%3DFang%26aufirst%3DY.%26aulast%3DHu%26aufirst%3DX.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DMaloney%26aufirst%3DD.%2BJ.%26aulast%3DHall%26aufirst%3DM.%2BD.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DFu%26aufirst%3DH.%26aulast%3DVertino%26aufirst%3DP.%2BM.%26aulast%3DYan%26aufirst%3DQ.%26aulast%3DCheng%26aufirst%3DX.%26atitle%3DInsights%2520into%2520the%2520action%2520of%2520inhibitor%2520enantiomers%2520against%2520histone%2520lysine%2520demethylase%25205A%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D3193%26epage%3D3208%26doi%3D10.1021%2Facs.jmedchem.8b00261" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vinogradova, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gehling, V. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gustafson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arora, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tindell, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williamson, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guler, G. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gangurde, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manieri, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Busby, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flynn, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Odate, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cochran, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wongchenko, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheung, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maile, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lau, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Costa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hegde, G. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jackson, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pitti, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnott, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bailey, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cummings, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albrecht, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harmange, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiefer, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trojer, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Classon, M.</span></span> <span> </span><span class="NLM_article-title">An inhibitor of KDM5 demethylases reduces survival of drug-tolerant cancer cells</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">531</span>– <span class="NLM_lpage">538</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2085</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=10.1038%2Fnchembio.2085" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=27214401" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=1%3ACAS%3A528%3ADC%252BC28XnvFOksLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=531-538&author=M.+Vinogradovaauthor=V.+S.+Gehlingauthor=A.+Gustafsonauthor=S.+Aroraauthor=C.+A.+Tindellauthor=C.+Wilsonauthor=K.+E.+Williamsonauthor=G.+D.+Gulerauthor=P.+Gangurdeauthor=W.+Manieriauthor=J.+Busbyauthor=E.+M.+Flynnauthor=F.+Lanauthor=H.+J.+Kimauthor=S.+Odateauthor=A.+G.+Cochranauthor=Y.+Liuauthor=M.+Wongchenkoauthor=Y.+Yangauthor=T.+K.+Cheungauthor=T.+M.+Maileauthor=T.+Lauauthor=M.+Costaauthor=G.+V.+Hegdeauthor=E.+Jacksonauthor=R.+Pittiauthor=D.+Arnottauthor=C.+Baileyauthor=S.+Bellonauthor=R.+T.+Cummingsauthor=B.+K.+Albrechtauthor=J.+C.+Harmangeauthor=J.+R.+Kieferauthor=P.+Trojerauthor=M.+Classon&title=An+inhibitor+of+KDM5+demethylases+reduces+survival+of+drug-tolerant+cancer+cells&doi=10.1038%2Fnchembio.2085"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">An inhibitor of KDM5 demethylases reduces survival of drug-tolerant cancer cells</span></div><div class="casAuthors">Vinogradova, Maia; Gehling, Victor S.; Gustafson, Amy; Arora, Shilpi; Tindell, Charles A.; Wilson, Catherine; Williamson, Kaylyn E.; Guler, Gulfem D.; Gangurde, Pranoti; Manieri, Wanda; Busby, Jennifer; Flynn, E. Megan; Lan, Fei; Kim, Hyo-jin; Odate, Shobu; Cochran, Andrea G.; Liu, Yichin; Wongchenko, Matthew; Yang, Yibin; Cheung, Tommy K.; Maile, Tobias M.; Lau, Ted; Costa, Michael; Hegde, Ganapati V.; Jackson, Erica; Pitti, Robert; Arnott, David; Bailey, Christopher; Bellon, Steve; Cummings, Richard T.; Albrecht, Brian K.; Harmange, Jean-Christophe; Kiefer, James R.; Trojer, Patrick; Classon, Marie</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">531-538</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The KDM5 family of histone demethylases catalyzes the demethylation of histone H3 on lysine 4 (H3K4) and is required for the survival of drug-tolerant persister cancer cells (DTPs).  Here the authors report the discovery and characterization of the specific KDM5 inhibitor CPI-455.  The crystal structure of KDM5A revealed the mechanism of inhibition of CPI-455 as well as the topol. arrangements of protein domains that influence substrate binding.  CPI-455 mediated KDM5 inhibition, elevated global levels of H3K4 trimethylation (H3K4me3) and decreased the no. of DTPs in multiple cancer cell line models treated with std. chemotherapy or targeted agents.  These findings show that pretreatment of cancer cells with a KDM5-specific inhibitor results in the ablation of a subpopulation of cancer cells that can serve as the founders for therapeutic relapse.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxyW7L2YsV1rVg90H21EOLACvtfcHk0lj7gyCUELFeYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XnvFOksLg%253D&md5=41699f5c17c912b39cdda81f4dfaa31e</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2085&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2085%26sid%3Dliteratum%253Aachs%26aulast%3DVinogradova%26aufirst%3DM.%26aulast%3DGehling%26aufirst%3DV.%2BS.%26aulast%3DGustafson%26aufirst%3DA.%26aulast%3DArora%26aufirst%3DS.%26aulast%3DTindell%26aufirst%3DC.%2BA.%26aulast%3DWilson%26aufirst%3DC.%26aulast%3DWilliamson%26aufirst%3DK.%2BE.%26aulast%3DGuler%26aufirst%3DG.%2BD.%26aulast%3DGangurde%26aufirst%3DP.%26aulast%3DManieri%26aufirst%3DW.%26aulast%3DBusby%26aufirst%3DJ.%26aulast%3DFlynn%26aufirst%3DE.%2BM.%26aulast%3DLan%26aufirst%3DF.%26aulast%3DKim%26aufirst%3DH.%2BJ.%26aulast%3DOdate%26aufirst%3DS.%26aulast%3DCochran%26aufirst%3DA.%2BG.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DWongchenko%26aufirst%3DM.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DCheung%26aufirst%3DT.%2BK.%26aulast%3DMaile%26aufirst%3DT.%2BM.%26aulast%3DLau%26aufirst%3DT.%26aulast%3DCosta%26aufirst%3DM.%26aulast%3DHegde%26aufirst%3DG.%2BV.%26aulast%3DJackson%26aufirst%3DE.%26aulast%3DPitti%26aufirst%3DR.%26aulast%3DArnott%26aufirst%3DD.%26aulast%3DBailey%26aufirst%3DC.%26aulast%3DBellon%26aufirst%3DS.%26aulast%3DCummings%26aufirst%3DR.%2BT.%26aulast%3DAlbrecht%26aufirst%3DB.%2BK.%26aulast%3DHarmange%26aufirst%3DJ.%2BC.%26aulast%3DKiefer%26aufirst%3DJ.%2BR.%26aulast%3DTrojer%26aufirst%3DP.%26aulast%3DClasson%26aufirst%3DM.%26atitle%3DAn%2520inhibitor%2520of%2520KDM5%2520demethylases%2520reduces%2520survival%2520of%2520drug-tolerant%2520cancer%2520cells%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2016%26volume%3D12%26spage%3D531%26epage%3D538%26doi%3D10.1038%2Fnchembio.2085" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tumber, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nuzzi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hookway, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatch, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Velupillai, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johansson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawamura, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savitsky, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yapp, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szykowska, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bountra, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strain-Damerell, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burgess-Brown, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruda, G. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munro, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">England, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nowak, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schofield, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">La Thangue, N. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pawlyn, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morgan, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Athanasou, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oppermann, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span> <span> </span><span class="NLM_article-title">Potent and selective KDM5 inhibitor stops cellular demethylation of H3K4me3 at transcription start sites and proliferation of MM1S myeloma cells</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">371</span>– <span class="NLM_lpage">380</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2017.02.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=10.1016%2Fj.chembiol.2017.02.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=28262558" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjvVOktro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2017&pages=371-380&author=A.+Tumberauthor=A.+Nuzziauthor=E.+S.+Hookwayauthor=S.+B.+Hatchauthor=S.+Velupillaiauthor=C.+Johanssonauthor=A.+Kawamuraauthor=P.+Savitskyauthor=C.+Yappauthor=A.+Szykowskaauthor=N.+Wuauthor=C.+Bountraauthor=C.+Strain-Damerellauthor=N.+A.+Burgess-Brownauthor=G.+F.+Rudaauthor=O.+Fedorovauthor=S.+Munroauthor=K.+S.+Englandauthor=R.+P.+Nowakauthor=C.+J.+Schofieldauthor=N.+B.+La+Thangueauthor=C.+Pawlynauthor=F.+Daviesauthor=G.+Morganauthor=N.+Athanasouauthor=S.+Mullerauthor=U.+Oppermannauthor=P.+E.+Brennan&title=Potent+and+selective+KDM5+inhibitor+stops+cellular+demethylation+of+H3K4me3+at+transcription+start+sites+and+proliferation+of+MM1S+myeloma+cells&doi=10.1016%2Fj.chembiol.2017.02.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Potent and Selective KDM5 Inhibitor Stops Cellular Demethylation of H3K4me3 at Transcription Start Sites and Proliferation of MM1S Myeloma Cells</span></div><div class="casAuthors">Tumber, Anthony; Nuzzi, Andrea; Hookway, Edward S.; Hatch, Stephanie B.; Velupillai, Srikannathasan; Johansson, Catrine; Kawamura, Akane; Savitsky, Pavel; Yapp, Clarence; Szykowska, Aleksandra; Wu, Na; Bountra, Chas; Strain-Damerell, Claire; Burgess-Brown, Nicola A.; Ruda, Gian Filippo; Fedorov, Oleg; Munro, Shonagh; England, Katherine S.; Nowak, Radoslaw P.; Schofield, Christopher J.; La Thangue, Nicholas B.; Pawlyn, Charlotte; Davies, Faith; Morgan, Gareth; Athanasou, Nick; Muller, Susanne; Oppermann, Udo; Brennan, Paul E.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">371-380</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Methylation of lysine residues on histone tail is a dynamic epigenetic modification that plays a key role in chromatin structure and gene regulation.  Members of the KDM5 (also known as JARID1) sub-family are 2-oxoglutarate (2-OG) and Fe2+-dependent oxygenases acting as histone 3 lysine 4 tri-Me (H3K4me3) demethylases, regulating proliferation, stem cell self-renewal, and differentiation.  Here we present the characterization of KDOAM-25, an inhibitor of KDM5 enzymes.  KDOAM-25 shows biochem. half maximal inhibitory concn. values of <100 nM for KDM5A-D in vitro, high selectivity toward other 2-OG oxygenases sub-families, and no off-target activity on a panel of 55 receptors and enzymes.  In human cell assay systems, KDOAM-25 has a half maximal effective concn. of ∼50 μM and good selectivity toward other demethylases.  KDM5B is overexpressed in multiple myeloma and neg. correlated with the overall survival.  Multiple myeloma MM1S cells treated with KDOAM-25 show increased global H3K4 methylation at transcriptional start sites and impaired proliferation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-UzNQI-MiYbVg90H21EOLACvtfcHk0lizfJjEvi-pJA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjvVOktro%253D&md5=0fc70179bda8b4fd38db1054c7bb3277</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2017.02.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2017.02.006%26sid%3Dliteratum%253Aachs%26aulast%3DTumber%26aufirst%3DA.%26aulast%3DNuzzi%26aufirst%3DA.%26aulast%3DHookway%26aufirst%3DE.%2BS.%26aulast%3DHatch%26aufirst%3DS.%2BB.%26aulast%3DVelupillai%26aufirst%3DS.%26aulast%3DJohansson%26aufirst%3DC.%26aulast%3DKawamura%26aufirst%3DA.%26aulast%3DSavitsky%26aufirst%3DP.%26aulast%3DYapp%26aufirst%3DC.%26aulast%3DSzykowska%26aufirst%3DA.%26aulast%3DWu%26aufirst%3DN.%26aulast%3DBountra%26aufirst%3DC.%26aulast%3DStrain-Damerell%26aufirst%3DC.%26aulast%3DBurgess-Brown%26aufirst%3DN.%2BA.%26aulast%3DRuda%26aufirst%3DG.%2BF.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DMunro%26aufirst%3DS.%26aulast%3DEngland%26aufirst%3DK.%2BS.%26aulast%3DNowak%26aufirst%3DR.%2BP.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26aulast%3DLa%2BThangue%26aufirst%3DN.%2BB.%26aulast%3DPawlyn%26aufirst%3DC.%26aulast%3DDavies%26aufirst%3DF.%26aulast%3DMorgan%26aufirst%3DG.%26aulast%3DAthanasou%26aufirst%3DN.%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DOppermann%26aufirst%3DU.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26atitle%3DPotent%2520and%2520selective%2520KDM5%2520inhibitor%2520stops%2520cellular%2520demethylation%2520of%2520H3K4me3%2520at%2520transcription%2520start%2520sites%2520and%2520proliferation%2520of%2520MM1S%2520myeloma%2520cells%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2017%26volume%3D24%26spage%3D371%26epage%3D380%26doi%3D10.1016%2Fj.chembiol.2017.02.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Horton, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gale, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shanks, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Webber, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bell, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kales, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mott, B. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rai, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jansen, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henderson, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urban, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simeonov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maloney, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johns, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jadhav, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vertino, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, X.</span></span> <span> </span><span class="NLM_article-title">Structural basis for KDM5A histone lysine demethylase inhibition by diverse compounds</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">769</span>– <span class="NLM_lpage">781</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2016.06.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=10.1016%2Fj.chembiol.2016.06.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=27427228" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFyiur3L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2016&pages=769-781&author=J.+R.+Hortonauthor=X.+Liuauthor=M.+Galeauthor=L.+Wuauthor=J.+R.+Shanksauthor=X.+Zhangauthor=P.+J.+Webberauthor=J.+S.+Bellauthor=S.+C.+Kalesauthor=B.+T.+Mottauthor=G.+Raiauthor=D.+J.+Jansenauthor=M.+J.+Hendersonauthor=D.+J.+Urbanauthor=M.+D.+Hallauthor=A.+Simeonovauthor=D.+J.+Maloneyauthor=M.+A.+Johnsauthor=H.+Fuauthor=A.+Jadhavauthor=P.+M.+Vertinoauthor=Q.+Yanauthor=X.+Cheng&title=Structural+basis+for+KDM5A+histone+lysine+demethylase+inhibition+by+diverse+compounds&doi=10.1016%2Fj.chembiol.2016.06.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Basis for KDM5A Histone Lysine Demethylase Inhibition by Diverse Compounds</span></div><div class="casAuthors">Horton, John R.; Liu, Xu; Gale, Molly; Wu, Lizhen; Shanks, John R.; Zhang, Xing; Webber, Philip J.; Bell, Joshua S. K.; Kales, Stephen C.; Mott, Bryan T.; Rai, Ganesha; Jansen, Daniel J.; Henderson, Mark J.; Urban, Daniel J.; Hall, Matthew D.; Simeonov, Anton; Maloney, David J.; Johns, Margaret A.; Fu, Haian; Jadhav, Ajit; Vertino, Paula M.; Yan, Qin; Cheng, Xiaodong</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">769-781</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The KDM5/JARID1 family of Fe(II)- and α-ketoglutarate-dependent demethylases removes Me groups from methylated lysine 4 of histone H3.  Accumulating evidence supports a role for KDM5 family members as oncogenic drivers.  We compare the in vitro inhibitory properties and binding affinity of ten diverse compds. with all four family members, and present the crystal structures of the KDM5A-linked Jumonji domain in complex with eight of these inhibitors in the presence of Mn(II).  All eight inhibitors structurally examd. occupy the binding site of α-ketoglutarate, but differ in their specific binding interactions, including the no. of ligands involved in metal coordination.  We also obsd. inhibitor-induced conformational changes in KDM5A, particularly those residues involved in the binding of α-ketoglutarate, the anticipated peptide substrate, and intramol. interactions.  We discuss how particular chem. moieties contribute to inhibitor potency and suggest strategies that might be utilized in the successful design of selective and potent epigenetic inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpstbuqHdktk7Vg90H21EOLACvtfcHk0lizfJjEvi-pJA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFyiur3L&md5=0dc780604904ac406211a09577df5bf5</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2016.06.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2016.06.006%26sid%3Dliteratum%253Aachs%26aulast%3DHorton%26aufirst%3DJ.%2BR.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DGale%26aufirst%3DM.%26aulast%3DWu%26aufirst%3DL.%26aulast%3DShanks%26aufirst%3DJ.%2BR.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DWebber%26aufirst%3DP.%2BJ.%26aulast%3DBell%26aufirst%3DJ.%2BS.%26aulast%3DKales%26aufirst%3DS.%2BC.%26aulast%3DMott%26aufirst%3DB.%2BT.%26aulast%3DRai%26aufirst%3DG.%26aulast%3DJansen%26aufirst%3DD.%2BJ.%26aulast%3DHenderson%26aufirst%3DM.%2BJ.%26aulast%3DUrban%26aufirst%3DD.%2BJ.%26aulast%3DHall%26aufirst%3DM.%2BD.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DMaloney%26aufirst%3DD.%2BJ.%26aulast%3DJohns%26aufirst%3DM.%2BA.%26aulast%3DFu%26aufirst%3DH.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DVertino%26aufirst%3DP.%2BM.%26aulast%3DYan%26aufirst%3DQ.%26aulast%3DCheng%26aufirst%3DX.%26atitle%3DStructural%2520basis%2520for%2520KDM5A%2520histone%2520lysine%2520demethylase%2520inhibition%2520by%2520diverse%2520compounds%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2016%26volume%3D23%26spage%3D769%26epage%3D781%26doi%3D10.1016%2Fj.chembiol.2016.06.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Johansson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Velupillai, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tumber, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szykowska, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hookway, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nowak, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strain-Damerell, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gileadi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philpott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burgess-Brown, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kopec, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nuzzi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steuber, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Egner, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Badock, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munro, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LaThangue, N. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westaway, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Athanasou, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oppermann, U.</span></span> <span> </span><span class="NLM_article-title">Structural analysis of human KDM5B guides histone demethylase inhibitor development</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">539</span>– <span class="NLM_lpage">545</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2087</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=10.1038%2Fnchembio.2087" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=27214403" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=1%3ACAS%3A528%3ADC%252BC28XosVWisbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=539-545&author=C.+Johanssonauthor=S.+Velupillaiauthor=A.+Tumberauthor=A.+Szykowskaauthor=E.+S.+Hookwayauthor=R.+P.+Nowakauthor=C.+Strain-Damerellauthor=C.+Gileadiauthor=M.+Philpottauthor=N.+Burgess-Brownauthor=N.+Wuauthor=J.+Kopecauthor=A.+Nuzziauthor=H.+Steuberauthor=U.+Egnerauthor=V.+Badockauthor=S.+Munroauthor=N.+B.+LaThangueauthor=S.+Westawayauthor=J.+Brownauthor=N.+Athanasouauthor=R.+Prinjhaauthor=P.+E.+Brennanauthor=U.+Oppermann&title=Structural+analysis+of+human+KDM5B+guides+histone+demethylase+inhibitor+development&doi=10.1038%2Fnchembio.2087"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Structural analysis of human KDM5B guides histone demethylase inhibitor development</span></div><div class="casAuthors">Johansson, Catrine; Velupillai, Srikannathasan; Tumber, Anthony; Szykowska, Aleksandra; Hookway, Edward S.; Nowak, Radoslaw P.; Strain-Damerell, Claire; Gileadi, Carina; Philpott, Martin; Burgess-Brown, Nicola; Wu, Na; Kopec, Jola; Nuzzi, Andrea; Steuber, Holger; Egner, Ursula; Badock, Volker; Munro, Shonagh; LaThangue, Nicholas B.; Westaway, Sue; Brown, Jack; Athanasou, Nick; Prinjha, Rab; Brennan, Paul E.; Oppermann, Udo</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">539-545</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Members of the KDM5 (also known as JARID1) family are 2-oxoglutarate- and Fe2+-dependent oxygenases that act as histone H3K4 demethylases, thereby regulating cell proliferation and stem cell self-renewal and differentiation.  Here, the authors report crystal structures of the catalytic core of the human KDM5B enzyme in complex with 3 inhibitor chemotypes.  These scaffolds exploitet several aspects of the KDM5 active site, and their selectivity profiles reflected their hybrid features with respect to the KDM4 and KDM6 families.  Whereas GSK-J1, a previously identified KDM6 inhibitor, showed ∼7-fold less inhibitory activity toward KDM5B than toward KDM6 proteins, KDM5-C49 displayed 25-100-fold selectivity between KDM5B and KDM6B.  The cell-permeable deriv. KDM5-C70 had an antiproliferative effect in myeloma cells, leading to genome-wide elevation of H3K4me3 levels.  The selective inhibitor GSK467 exploited unique binding modes, but it lacked cellular potency in the myeloma system.  Taken together, these structural leads deliver multiple starting points for further rational and selective inhibitor design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqp-2vDMpTjHbVg90H21EOLACvtfcHk0ljjSayk86zPHw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XosVWisbs%253D&md5=2c4b6e5779ba43ed3584fc6f76b987cd</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2087&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2087%26sid%3Dliteratum%253Aachs%26aulast%3DJohansson%26aufirst%3DC.%26aulast%3DVelupillai%26aufirst%3DS.%26aulast%3DTumber%26aufirst%3DA.%26aulast%3DSzykowska%26aufirst%3DA.%26aulast%3DHookway%26aufirst%3DE.%2BS.%26aulast%3DNowak%26aufirst%3DR.%2BP.%26aulast%3DStrain-Damerell%26aufirst%3DC.%26aulast%3DGileadi%26aufirst%3DC.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DBurgess-Brown%26aufirst%3DN.%26aulast%3DWu%26aufirst%3DN.%26aulast%3DKopec%26aufirst%3DJ.%26aulast%3DNuzzi%26aufirst%3DA.%26aulast%3DSteuber%26aufirst%3DH.%26aulast%3DEgner%26aufirst%3DU.%26aulast%3DBadock%26aufirst%3DV.%26aulast%3DMunro%26aufirst%3DS.%26aulast%3DLaThangue%26aufirst%3DN.%2BB.%26aulast%3DWestaway%26aufirst%3DS.%26aulast%3DBrown%26aufirst%3DJ.%26aulast%3DAthanasou%26aufirst%3DN.%26aulast%3DPrinjha%26aufirst%3DR.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DOppermann%26aufirst%3DU.%26atitle%3DStructural%2520analysis%2520of%2520human%2520KDM5B%2520guides%2520histone%2520demethylase%2520inhibitor%2520development%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2016%26volume%3D12%26spage%3D539%26epage%3D545%26doi%3D10.1038%2Fnchembio.2087" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Herr, C. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hausinger, R. P.</span></span> <span> </span><span class="NLM_article-title">Amazing diversity in biochemical roles of Fe(II)/2-oxoglutarate oxygenases</span>. <i>Trends Biochem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>43</i></span>,  <span class="NLM_fpage">517</span>– <span class="NLM_lpage">532</span>, <span class="refDoi"> DOI: 10.1016/j.tibs.2018.04.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=10.1016%2Fj.tibs.2018.04.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=29709390" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=1%3ACAS%3A528%3ADC%252BC1cXnslKktL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2018&pages=517-532&author=C.+Q.+Herrauthor=R.+P.+Hausinger&title=Amazing+diversity+in+biochemical+roles+of+Fe%28II%29%2F2-oxoglutarate+oxygenases&doi=10.1016%2Fj.tibs.2018.04.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Amazing Diversity in Biochemical Roles of Fe(II)/2-Oxoglutarate Oxygenases</span></div><div class="casAuthors">Herr, Caitlyn Q.; Hausinger, Robert P.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Biochemical Sciences</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">517-532</span>CODEN:
                <span class="NLM_cas:coden">TBSCDB</span>;
        ISSN:<span class="NLM_cas:issn">0968-0004</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Since their discovery in the 1960s, the family of Fe(II)/2-oxoglutarate-dependent oxygenases has undergone a tremendous expansion to include enzymes catalyzing a vast diversity of biol. important reactions.  Recent examples highlight roles in controlling chromatin modification, transcription, mRNA demethylation, and mRNA splicing.  Others generate modifications in tRNA, translation factors, ribosomes, and other proteins.  Thus, oxygenases affect all components of mol. biol.'s central dogma, in which information flows from DNA to RNA to proteins.  These enzymes also function in biosynthesis and catabolism of cellular metabolites, including antibiotics and signaling mols.  Due to their crit. importance, ongoing efforts have targeted family members for the development of specific therapeutics.  This review provides a general overview of recently characterized oxygenase reactions and their key biol. roles.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWNCor1Ykj3LVg90H21EOLACvtfcHk0ljjSayk86zPHw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXnslKktL4%253D&md5=fbdbc3a2ed5e19a0bbd9e2389471ad72</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.tibs.2018.04.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tibs.2018.04.002%26sid%3Dliteratum%253Aachs%26aulast%3DHerr%26aufirst%3DC.%2BQ.%26aulast%3DHausinger%26aufirst%3DR.%2BP.%26atitle%3DAmazing%2520diversity%2520in%2520biochemical%2520roles%2520of%2520Fe%2528II%2529%252F2-oxoglutarate%2520oxygenases%26jtitle%3DTrends%2520Biochem.%2520Sci.%26date%3D2018%26volume%3D43%26spage%3D517%26epage%3D532%26doi%3D10.1016%2Fj.tibs.2018.04.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kaniskan, H. U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martini, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Inhibitors of protein methyltransferases and demethylases</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>118</i></span>,  <span class="NLM_fpage">989</span>– <span class="NLM_lpage">1068</span>, <span class="refDoi"> DOI: 10.1021/acs.chemrev.6b00801</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.6b00801" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkslynsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=118&publication_year=2018&pages=989-1068&author=H.+U.+Kaniskanauthor=M.+L.+Martiniauthor=J.+Jin&title=Inhibitors+of+protein+methyltransferases+and+demethylases&doi=10.1021%2Facs.chemrev.6b00801"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibitors of Protein Methyltransferases and Demethylases</span></div><div class="casAuthors">Kaniskan, H. Umit; Martini, Michael L.; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">118</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">989-1068</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Post-translational modifications of histones by protein methyltransferases (PMTs) and histone demethylases (KDMs) play an important role in the regulation of gene expression and transcription and are implicated in cancer and many other diseases.  Many of these enzymes also target various nonhistone proteins impacting numerous crucial biol. pathways.  Given their key biol. functions and implications in human diseases, there has been a growing interest in assessing these enzymes as potential therapeutic targets.  Consequently, discovering and developing inhibitors of these enzymes has become a very active and fast-growing research area over the past decade.  In this review, the authors cover the discovery, characterization, and biol. application of inhibitors of PMTs and KDMs with emphasis on key advancements in the field.  The authors also discuss challenges, opportunities, and future directions in this emerging, exciting research field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJYjiYtw_n4rVg90H21EOLACvtfcHk0ljjSayk86zPHw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkslynsL8%253D&md5=1ed520c6b0c42d2aa8a7f66b0a2d4e40</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.6b00801&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.6b00801%26sid%3Dliteratum%253Aachs%26aulast%3DKaniskan%26aufirst%3DH.%2BU.%26aulast%3DMartini%26aufirst%3DM.%2BL.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DInhibitors%2520of%2520protein%2520methyltransferases%2520and%2520demethylases%26jtitle%3DChem.%2520Rev.%26date%3D2018%26volume%3D118%26spage%3D989%26epage%3D1068%26doi%3D10.1021%2Facs.chemrev.6b00801" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Johansson, M. H.</span></span> <span> </span><span class="NLM_article-title">Reversible Michael additions: covalent inhibitors and prodrugs</span>. <i>Mini-Rev. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">1330</span>– <span class="NLM_lpage">1344</span>, <span class="refDoi"> DOI: 10.2174/13895575112091330</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=10.2174%2F13895575112091330" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=22625413" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs12hu7jJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2012&pages=1330-1344&author=M.+H.+Johansson&title=Reversible+Michael+additions%3A+covalent+inhibitors+and+prodrugs&doi=10.2174%2F13895575112091330"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Reversible Michael additions: covalent inhibitors and prodrugs</span></div><div class="casAuthors">Johansson, Martin H.</div><div class="citationInfo"><span class="NLM_cas:title">Mini-Reviews in Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">1330-1344</span>CODEN:
                <span class="NLM_cas:coden">MMCIAE</span>;
        ISSN:<span class="NLM_cas:issn">1389-5575</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Covalent inhibition is an efficient mol. mechanism for inhibiting enzymes or modulating the function of proteins and is found in the action of many drugs and biol. active natural products.  However, it is has been less appreciated that the formation of covalent bonds can be reversible or irreversible.  This review focuses on biol. active compds. that are Michael acceptors and how the reversible nature of the Michael addn. reaction influences biol. activity and how this can be exploited in designing prodrugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr9X1yh9k4q-7Vg90H21EOLACvtfcHk0lgRFvBmy4GAqw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs12hu7jJ&md5=f90694162e39fe57915e888bbadbd2bb</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.2174%2F13895575112091330&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F13895575112091330%26sid%3Dliteratum%253Aachs%26aulast%3DJohansson%26aufirst%3DM.%2BH.%26atitle%3DReversible%2520Michael%2520additions%253A%2520covalent%2520inhibitors%2520and%2520prodrugs%26jtitle%3DMini-Rev.%2520Med.%2520Chem.%26date%3D2012%26volume%3D12%26spage%3D1330%26epage%3D1344%26doi%3D10.2174%2F13895575112091330" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Awoonor-Williams, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walsh, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rowley, C. N.</span></span> <span> </span><span class="NLM_article-title">Modeling covalent-modifier drugs</span>. <i>Biochim. Biophys. Acta, Proteins Proteomics</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>1865</i></span>,  <span class="NLM_fpage">1664</span>– <span class="NLM_lpage">1675</span>, <span class="refDoi"> DOI: 10.1016/j.bbapap.2017.05.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=10.1016%2Fj.bbapap.2017.05.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=28528876" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=1%3ACAS%3A528%3ADC%252BC2sXotVOrtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1865&publication_year=2017&pages=1664-1675&author=E.+Awoonor-Williamsauthor=A.+G.+Walshauthor=C.+N.+Rowley&title=Modeling+covalent-modifier+drugs&doi=10.1016%2Fj.bbapap.2017.05.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Modeling covalent-modifier drugs</span></div><div class="casAuthors">Awoonor-Williams, Ernest; Walsh, Andrew G.; Rowley, Christopher N.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Proteins and Proteomics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">1865</span>
        (<span class="NLM_cas:issue">11_Part_B</span>),
    <span class="NLM_cas:pages">1664-1675</span>CODEN:
                <span class="NLM_cas:coden">BBAPBW</span>;
        ISSN:<span class="NLM_cas:issn">1570-9639</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  In this review, we present a summary of how computer modeling has been used in the development of covalent-modifier drugs.  Covalent-modifier drugs bind by forming a chem. bond with their target.  This covalent binding can improve the selectivity of the drug for a target with complementary reactivity and result in increased binding affinities due to the strength of the covalent bond formed.  In some cases, this results in irreversible inhibition of the target, but some targeted covalent inhibitor (TCI) drugs bind covalently but reversibly.  Computer modeling is widely used in drug discovery, but different computational methods must be used to model covalent modifiers because of the chem. bonds formed.  Structural and bioinformatic anal. has identified sites of modification that could yield selectivity for a chosen target.  Docking methods, which are used to rank binding poses of large sets of inhibitors, have been augmented to support the formation of protein-ligand bonds and are now capable of predicting the binding pose of covalent modifiers accurately.  The pKa's of amino acids can be calcd. in order to assess their reactivity towards electrophiles.  QM/MM methods have been used to model the reaction mechanisms of covalent modification.  The continued development of these tools will allow computation to aid in the development of new covalent-modifier drugs.  This article is part of a Special Issue entitled: Biophysics in Canada, edited by Lewis Kay, John Baenziger, Albert Berghuis and Peter Tieleman.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpyPHPoiqcVk7Vg90H21EOLACvtfcHk0lgRFvBmy4GAqw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXotVOrtbo%253D&md5=ac579900da99ea005d0d562b53a1bca2</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.bbapap.2017.05.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbapap.2017.05.009%26sid%3Dliteratum%253Aachs%26aulast%3DAwoonor-Williams%26aufirst%3DE.%26aulast%3DWalsh%26aufirst%3DA.%2BG.%26aulast%3DRowley%26aufirst%3DC.%2BN.%26atitle%3DModeling%2520covalent-modifier%2520drugs%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Proteins%2520Proteomics%26date%3D2017%26volume%3D1865%26spage%3D1664%26epage%3D1675%26doi%3D10.1016%2Fj.bbapap.2017.05.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bradshaw, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McFarland, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paavilainen, V. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bisconte, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tam, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phan, V. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romanov, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finkle, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ton, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loughhead, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nunn, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karr, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerritsen, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Funk, J. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Owens, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verner, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brameld, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldstein, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taunton, J.</span></span> <span> </span><span class="NLM_article-title">Prolonged and tunable residence time using reversible covalent kinase inhibitors</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">525</span>– <span class="NLM_lpage">531</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1817</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=10.1038%2Fnchembio.1817" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=26006010" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFeju7%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=525-531&author=J.+M.+Bradshawauthor=J.+M.+McFarlandauthor=V.+O.+Paavilainenauthor=A.+Bisconteauthor=D.+Tamauthor=V.+T.+Phanauthor=S.+Romanovauthor=D.+Finkleauthor=J.+Shuauthor=V.+Patelauthor=T.+Tonauthor=X.+Liauthor=D.+G.+Loughheadauthor=P.+A.+Nunnauthor=D.+E.+Karrauthor=M.+E.+Gerritsenauthor=J.+O.+Funkauthor=T.+D.+Owensauthor=E.+Vernerauthor=K.+A.+Brameldauthor=R.+J.+Hillauthor=D.+M.+Goldsteinauthor=J.+Taunton&title=Prolonged+and+tunable+residence+time+using+reversible+covalent+kinase+inhibitors&doi=10.1038%2Fnchembio.1817"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Prolonged and tunable residence time using reversible covalent kinase inhibitors</span></div><div class="casAuthors">Bradshaw, J. Michael; McFarland, Jesse M.; Paavilainen, Ville O.; Bisconte, Angelina; Tam, Danny; Phan, Vernon T.; Romanov, Sergei; Finkle, David; Shu, Jin; Patel, Vaishali; Ton, Tony; Li, Xiaoyan; Loughhead, David G.; Nunn, Philip A.; Karr, Dane E.; Gerritsen, Mary E.; Funk, Jens Oliver; Owens, Timothy D.; Verner, Erik; Brameld, Ken A.; Hill, Ronald J.; Goldstein, David M.; Taunton, Jack</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">525-531</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Drugs with prolonged on-target residence times often show superior efficacy, yet general strategies for optimizing drug-target residence time are lacking.  Here the authors made progress toward this elusive goal by targeting a noncatalytic cysteine in Bruton's tyrosine kinase (BTK) with reversible covalent inhibitors.  Using an inverted orientation of the cysteine-reactive cyanoacrylamide electrophile, the authors identified potent and selective BTK inhibitors that demonstrated biochem. residence times spanning from minutes to 7 d.  An inverted cyanoacrylamide with prolonged residence time in vivo remained bound to BTK for more than 18 h after clearance from the circulation.  The inverted cyanoacrylamide strategy was further used to discover fibroblast growth factor receptor (FGFR) kinase inhibitors with residence times of several days, demonstrating the generalizability of the approach.  Targeting of noncatalytic cysteines with inverted cyanoacrylamides may serve as a broadly applicable platform that facilitates 'residence time by design', the ability to modulate and improve the duration of target engagement in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXCpNprhn6kLVg90H21EOLACvtfcHk0lj-JH5PrIGVuA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFeju7%252FE&md5=85a73d9ecd62695d03166f7009c68dd8</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1817&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1817%26sid%3Dliteratum%253Aachs%26aulast%3DBradshaw%26aufirst%3DJ.%2BM.%26aulast%3DMcFarland%26aufirst%3DJ.%2BM.%26aulast%3DPaavilainen%26aufirst%3DV.%2BO.%26aulast%3DBisconte%26aufirst%3DA.%26aulast%3DTam%26aufirst%3DD.%26aulast%3DPhan%26aufirst%3DV.%2BT.%26aulast%3DRomanov%26aufirst%3DS.%26aulast%3DFinkle%26aufirst%3DD.%26aulast%3DShu%26aufirst%3DJ.%26aulast%3DPatel%26aufirst%3DV.%26aulast%3DTon%26aufirst%3DT.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DLoughhead%26aufirst%3DD.%2BG.%26aulast%3DNunn%26aufirst%3DP.%2BA.%26aulast%3DKarr%26aufirst%3DD.%2BE.%26aulast%3DGerritsen%26aufirst%3DM.%2BE.%26aulast%3DFunk%26aufirst%3DJ.%2BO.%26aulast%3DOwens%26aufirst%3DT.%2BD.%26aulast%3DVerner%26aufirst%3DE.%26aulast%3DBrameld%26aufirst%3DK.%2BA.%26aulast%3DHill%26aufirst%3DR.%2BJ.%26aulast%3DGoldstein%26aufirst%3DD.%2BM.%26aulast%3DTaunton%26aufirst%3DJ.%26atitle%3DProlonged%2520and%2520tunable%2520residence%2520time%2520using%2520reversible%2520covalent%2520kinase%2520inhibitors%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26spage%3D525%26epage%3D531%26doi%3D10.1038%2Fnchembio.1817" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smith, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uchida, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taunton, J.</span></span> <span> </span><span class="NLM_article-title">Essential biphasic role for JAK3 catalytic activity in IL-2 receptor signaling</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">373</span>– <span class="NLM_lpage">379</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2056</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=10.1038%2Fnchembio.2056" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=27018889" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=1%3ACAS%3A528%3ADC%252BC28XksFCru74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=373-379&author=G.+A.+Smithauthor=K.+Uchidaauthor=A.+Weissauthor=J.+Taunton&title=Essential+biphasic+role+for+JAK3+catalytic+activity+in+IL-2+receptor+signaling&doi=10.1038%2Fnchembio.2056"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Essential biphasic role for JAK3 catalytic activity in IL-2 receptor signaling</span></div><div class="casAuthors">Smith, Geoffrey A.; Uchida, Kenji; Weiss, Arthur; Taunton, Jack</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">373-379</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">To drive lymphocyte proliferation and differentiation, common γ-chain (γc) cytokine receptors require hours to days of sustained stimulation.  JAK1 and JAK3 kinases are found together in all γc-receptor complexes, but how their resp. catalytic activities contribute to signaling over time is not known.  Here we dissect the temporal requirements for JAK3 kinase activity with a selective covalent inhibitor (JAK3i).  By monitoring phosphorylation of the transcription factor STAT5 over 20 h in CD4+ T cells stimulated with interleukin 2 (IL-2), we document a second wave of signaling that is much more sensitive to JAK3i than the first wave.  Selective inhibition of this second wave is sufficient to block cyclin expression and entry to S phase.  An inhibitor-resistant JAK3 mutant (C905S) rescued all effects of JAK3i in isolated T cells and in mice.  Our chem. genetic toolkit elucidates a biphasic requirement for JAK3 kinase activity in IL-2-driven T cell proliferation and will find broad utility in studies of γc-receptor signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonfXNNV1cRJbVg90H21EOLACvtfcHk0lj-JH5PrIGVuA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XksFCru74%253D&md5=573d99767ce37cf58a8f71e9e01102b5</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2056&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2056%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DG.%2BA.%26aulast%3DUchida%26aufirst%3DK.%26aulast%3DWeiss%26aufirst%3DA.%26aulast%3DTaunton%26aufirst%3DJ.%26atitle%3DEssential%2520biphasic%2520role%2520for%2520JAK3%2520catalytic%2520activity%2520in%2520IL-2%2520receptor%2520signaling%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2016%26volume%3D12%26spage%3D373%26epage%3D379%26doi%3D10.1038%2Fnchembio.2056" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguez-Molina, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tseng, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simonett, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taunton, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ansari, A. Z.</span></span> <span> </span><span class="NLM_article-title">Engineered covalent inactivation of TFIIH-kinase reveals an elongation checkpoint and results in widespread mRNA stabilization</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">433</span>– <span class="NLM_lpage">444</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2016.06.036</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=10.1016%2Fj.molcel.2016.06.036" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2016&pages=433-444&author=J.+B.+Rodriguez-Molinaauthor=S.+C.+Tsengauthor=S.+P.+Simonettauthor=J.+Tauntonauthor=A.+Z.+Ansari&title=Engineered+covalent+inactivation+of+TFIIH-kinase+reveals+an+elongation+checkpoint+and+results+in+widespread+mRNA+stabilization&doi=10.1016%2Fj.molcel.2016.06.036"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2016.06.036&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2016.06.036%26sid%3Dliteratum%253Aachs%26aulast%3DRodriguez-Molina%26aufirst%3DJ.%2BB.%26aulast%3DTseng%26aufirst%3DS.%2BC.%26aulast%3DSimonett%26aufirst%3DS.%2BP.%26aulast%3DTaunton%26aufirst%3DJ.%26aulast%3DAnsari%26aufirst%3DA.%2BZ.%26atitle%3DEngineered%2520covalent%2520inactivation%2520of%2520TFIIH-kinase%2520reveals%2520an%2520elongation%2520checkpoint%2520and%2520results%2520in%2520widespread%2520mRNA%2520stabilization%26jtitle%3DMol.%2520Cell%26date%3D2016%26volume%3D63%26spage%3D433%26epage%3D444%26doi%3D10.1016%2Fj.molcel.2016.06.036" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Butler, K. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tempel, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Babault, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pittella-Silva, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Structure-based design of a covalent inhibitor of the SET domain-containing protein 8 (SETD8) lysine methyltransferase</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">9881</span>– <span class="NLM_lpage">9889</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01244</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01244" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=1%3ACAS%3A528%3ADC%252BC28XhslKjtLnE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=9881-9889&author=K.+V.+Butlerauthor=A.+Maauthor=W.+Yuauthor=F.+Liauthor=W.+Tempelauthor=N.+Babaultauthor=F.+Pittella-Silvaauthor=J.+Shaoauthor=J.+Wangauthor=M.+Luoauthor=M.+Vedadiauthor=P.+J.+Brownauthor=C.+H.+Arrowsmithauthor=J.+Jin&title=Structure-based+design+of+a+covalent+inhibitor+of+the+SET+domain-containing+protein+8+%28SETD8%29+lysine+methyltransferase&doi=10.1021%2Facs.jmedchem.6b01244"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Design of a Covalent Inhibitor of the SET Domain-Containing Protein 8 (SETD8) Lysine Methyltransferase</span></div><div class="casAuthors">Butler, Kyle V.; Ma, Anqi; Yu, Wenyu; Li, Fengling; Tempel, Wolfram; Babault, Nicolas; Pittella-Silva, Fabio; Shao, Jason; Wang, Junyi; Luo, Minkui; Vedadi, Masoud; Brown, Peter J.; Arrowsmith, Cheryl H.; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">9881-9889</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Selective inhibitors of protein lysine methyltransferases, including SET domain-contg. protein 8 (SETD8), are highly desired, as only a fraction of these enzymes are assocd. with high-quality inhibitors.  From our previously discovered SETD8 inhibitor, we developed a more potent analog and solved a cocrystal structure, which is the first crystal structure of SETD8 in complex with a small-mol. inhibitor.  This cocrystal structure allowed the design of a covalent inhibitor of SETD8 (MS453), which specifically modifies a cysteine residue near the inhibitor binding site, has an IC50 value of 804 nM, reacts with SETD8 with near-quant. yield, and is selective for SETD8 against 28 other methyltransferases.  We also solved the crystal structure of the covalent inhibitor in complex with SETD8.  This work provides at.-level perspective on the inhibition of SETD8 by small mols. and will help identify high-quality chem. probes of SETD8.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpjPurua1qCK7Vg90H21EOLACvtfcHk0lj-JH5PrIGVuA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhslKjtLnE&md5=b4d0b38dfe3a911633aad17642e78134</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01244&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01244%26sid%3Dliteratum%253Aachs%26aulast%3DButler%26aufirst%3DK.%2BV.%26aulast%3DMa%26aufirst%3DA.%26aulast%3DYu%26aufirst%3DW.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DTempel%26aufirst%3DW.%26aulast%3DBabault%26aufirst%3DN.%26aulast%3DPittella-Silva%26aufirst%3DF.%26aulast%3DShao%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DLuo%26aufirst%3DM.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DStructure-based%2520design%2520of%2520a%2520covalent%2520inhibitor%2520of%2520the%2520SET%2520domain-containing%2520protein%25208%2520%2528SETD8%2529%2520lysine%2520methyltransferase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D9881%26epage%3D9889%26doi%3D10.1021%2Facs.jmedchem.6b01244" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Horton, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engstrom, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zoeller, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shanks, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johns, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vertino, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, X.</span></span> <span> </span><span class="NLM_article-title">Characterization of a linked jumonji domain of the KDM5/JARID1 family of histone H3 lysine 4 demethylases</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>291</i></span>,  <span class="NLM_fpage">2631</span>– <span class="NLM_lpage">2646</span>, <span class="refDoi"> DOI: 10.1074/jbc.M115.698449</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=10.1074%2Fjbc.M115.698449" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=26645689" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVOqtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=291&publication_year=2016&pages=2631-2646&author=J.+R.+Hortonauthor=A.+Engstromauthor=E.+L.+Zoellerauthor=X.+Liuauthor=J.+R.+Shanksauthor=X.+Zhangauthor=M.+A.+Johnsauthor=P.+M.+Vertinoauthor=H.+Fuauthor=X.+Cheng&title=Characterization+of+a+linked+jumonji+domain+of+the+KDM5%2FJARID1+family+of+histone+H3+lysine+4+demethylases&doi=10.1074%2Fjbc.M115.698449"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of a Linked Jumonji Domain of the KDM5/JARID1 Family of Histone H3 Lysine 4 Demethylases</span></div><div class="casAuthors">Horton, John R.; Engstrom, Amanda; Zoeller, Elizabeth L.; Liu, Xu; Shanks, John R.; Zhang, Xing; Johns, Margaret A.; Vertino, Paula M.; Fu, Haian; Cheng, Xiaodong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">291</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2631-2646</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The KDM5/JARID1 family of Fe(II)- and α-ketoglutarate-dependent demethylases remove Me groups from tri- and dimethylated lysine 4 of histone H3.  Accumulating evidence from primary tumors and model systems supports a role for KDM5A (JARID1A/RBP2) and KDM5B (JARID1B/PLU1) as oncogenic drivers.  The KDM5 family is unique among the Jumonji domain-contg. histone demethylases in that there is an atypical insertion of a DNA-binding ARID domain and a histone-binding PHD domain into the Jumonji domain, which separates the catalytic domain into two fragments (JmjN and JmjC).  Here we demonstrate that internal deletion of the ARID and PHD1 domains has a negligible effect on in vitro enzymic kinetics of the KDM5 family of enzymes.  We present a crystal structure of the linked JmjN-JmjC domain from KDM5A, which reveals that the linked domain fully reconstitutes the cofactor (metal ion and α-ketoglutarate) binding characteristics of other structurally characterized Jumonji domain demethylases.  Docking studies with GSK-J1, a selective inhibitor of the KDM6/KDM5 subfamilies, identify crit. residues for binding of the inhibitor to the reconstituted KDM5 Jumonji domain.  Further, we found that GSK-J1 inhibited the demethylase activity of KDM5C with 8.5-fold increased potency compared with that of KDM5B at 1 mm α-ketoglutarate.  In contrast, JIB-04 (a pan-inhibitor of the Jumonji demethylase superfamily) had the opposite effect and was ∼8-fold more potent against KDM5B than against KDM5C.  Interestingly, the relative selectivity of JIB-04 toward KDM5B over KDM5C in vitro translates to a ∼10-50-fold greater growth-inhibitory activity against breast cancer cell lines.  These data define the minimal requirements for enzymic activity of the KDM5 family to be the linked JmjN-JmjC domain coupled with the immediate C-terminal helical zinc-binding domain and provide structural characterization of the linked JmjN-JmjC domain for the KDM5 family, which should prove useful in the design of KDM5 demethylase inhibitors with improved potency and selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpa4qqOk0zHF7Vg90H21EOLACvtfcHk0lg5wnE6K7tsLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitVOqtr0%253D&md5=772a106c3c7f0a45c1b3d2ee81cc9487</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M115.698449&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M115.698449%26sid%3Dliteratum%253Aachs%26aulast%3DHorton%26aufirst%3DJ.%2BR.%26aulast%3DEngstrom%26aufirst%3DA.%26aulast%3DZoeller%26aufirst%3DE.%2BL.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DShanks%26aufirst%3DJ.%2BR.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DJohns%26aufirst%3DM.%2BA.%26aulast%3DVertino%26aufirst%3DP.%2BM.%26aulast%3DFu%26aufirst%3DH.%26aulast%3DCheng%26aufirst%3DX.%26atitle%3DCharacterization%2520of%2520a%2520linked%2520jumonji%2520domain%2520of%2520the%2520KDM5%252FJARID1%2520family%2520of%2520histone%2520H3%2520lysine%25204%2520demethylases%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2016%26volume%3D291%26spage%3D2631%26epage%3D2646%26doi%3D10.1074%2Fjbc.M115.698449" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kruidenier, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liddle, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Che, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joberty, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bantscheff, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bountra, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bridges, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diallo, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eberhard, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hutchinson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katso, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leveridge, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mander, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mosley, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramirez-Molina, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rowland, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schofield, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheppard, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swales, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tumber, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drewes, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oppermann, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, D. M.</span></span> <span> </span><span class="NLM_article-title">A selective jumonji H3K27 demethylase inhibitor modulates the proinflammatory macrophage response</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>488</i></span>,  <span class="NLM_fpage">404</span>– <span class="NLM_lpage">408</span>, <span class="refDoi"> DOI: 10.1038/nature11262</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=10.1038%2Fnature11262" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=22842901" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtFWjtbfI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=488&publication_year=2012&pages=404-408&author=L.+Kruidenierauthor=C.+W.+Chungauthor=Z.+Chengauthor=J.+Liddleauthor=K.+Cheauthor=G.+Jobertyauthor=M.+Bantscheffauthor=C.+Bountraauthor=A.+Bridgesauthor=H.+Dialloauthor=D.+Eberhardauthor=S.+Hutchinsonauthor=E.+Jonesauthor=R.+Katsoauthor=M.+Leveridgeauthor=P.+K.+Manderauthor=J.+Mosleyauthor=C.+Ramirez-Molinaauthor=P.+Rowlandauthor=C.+J.+Schofieldauthor=R.+J.+Sheppardauthor=J.+E.+Smithauthor=C.+Swalesauthor=R.+Tannerauthor=P.+Thomasauthor=A.+Tumberauthor=G.+Drewesauthor=U.+Oppermannauthor=D.+J.+Patelauthor=K.+Leeauthor=D.+M.+Wilson&title=A+selective+jumonji+H3K27+demethylase+inhibitor+modulates+the+proinflammatory+macrophage+response&doi=10.1038%2Fnature11262"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">A selective jumonji H3K27 demethylase inhibitor modulates the proinflammatory macrophage response</span></div><div class="casAuthors">Kruidenier, Laurens; Chung, Chun-wa; Cheng, Zhongjun; Liddle, John; Che, Ka Hing; Joberty, Gerard; Bantscheff, Marcus; Bountra, Chas; Bridges, Angela; Diallo, Hawa; Eberhard, Dirk; Hutchinson, Sue; Jones, Emma; Katso, Roy; Leveridge, Melanie; Mander, Palwinder K.; Mosley, Julie; Ramirez-Molina, Cesar; Rowland, Paul; Schofield, Christopher J.; Sheppard, Robert J.; Smith, Julia E.; Swales, Catherine; Tanner, Robert; Thomas, Pamela; Tumber, Anthony; Drewes, Gerard; Oppermann, Udo; Patel, Dinshaw J.; Lee, Kevin; Wilson, David M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">488</span>
        (<span class="NLM_cas:issue">7411</span>),
    <span class="NLM_cas:pages">404-408</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The jumonji (JMJ) family of histone demethylases are Fe2+- and α-ketoglutarate-dependent oxygenases that are essential components of regulatory transcriptional chromatin complexes.  These enzymes demethylate lysine residues in histones in a methylation-state and sequence-specific context.  Considerable effort has been devoted to gaining a mechanistic understanding of the roles of histone lysine demethylases in eukaryotic transcription, genome integrity and epigenetic inheritance, as well as in development, physiol. and disease.  However, because of the absence of any selective inhibitors, the relevance of the demethylase activity of JMJ enzymes in regulating cellular responses remains poorly understood.  Here we present a structure-guided small-mol. and chemoproteomics approach to elucidating the functional role of the H3K27me3-specific demethylase subfamily (KDM6 subfamily members JMJD3 and UTX).  The liganded structures of human and mouse JMJD3 provide novel insight into the specificity determinants for cofactor, substrate and inhibitor recognition by the KDM6 subfamily of demethylases.  We exploited these structural features to generate the first small-mol. catalytic site inhibitor that is selective for the H3K27me3-specific JMJ subfamily.  We demonstrate that this inhibitor binds in a novel manner and reduces lipopolysaccharide-induced proinflammatory cytokine prodn. by human primary macrophages, a process that depends on both JMJD3 and UTX.  Our results resolve the ambiguity assocd. with the catalytic function of H3K27-specific JMJs in regulating disease-relevant inflammatory responses and provide encouragement for designing small-mol. inhibitors to allow selective pharmacol. intervention across the JMJ family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-lxSNmxjfsbVg90H21EOLACvtfcHk0lg5wnE6K7tsLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtFWjtbfI&md5=a4fa470a86070b4777e16a71e51763c8</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1038%2Fnature11262&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature11262%26sid%3Dliteratum%253Aachs%26aulast%3DKruidenier%26aufirst%3DL.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DCheng%26aufirst%3DZ.%26aulast%3DLiddle%26aufirst%3DJ.%26aulast%3DChe%26aufirst%3DK.%26aulast%3DJoberty%26aufirst%3DG.%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DBountra%26aufirst%3DC.%26aulast%3DBridges%26aufirst%3DA.%26aulast%3DDiallo%26aufirst%3DH.%26aulast%3DEberhard%26aufirst%3DD.%26aulast%3DHutchinson%26aufirst%3DS.%26aulast%3DJones%26aufirst%3DE.%26aulast%3DKatso%26aufirst%3DR.%26aulast%3DLeveridge%26aufirst%3DM.%26aulast%3DMander%26aufirst%3DP.%2BK.%26aulast%3DMosley%26aufirst%3DJ.%26aulast%3DRamirez-Molina%26aufirst%3DC.%26aulast%3DRowland%26aufirst%3DP.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26aulast%3DSheppard%26aufirst%3DR.%2BJ.%26aulast%3DSmith%26aufirst%3DJ.%2BE.%26aulast%3DSwales%26aufirst%3DC.%26aulast%3DTanner%26aufirst%3DR.%26aulast%3DThomas%26aufirst%3DP.%26aulast%3DTumber%26aufirst%3DA.%26aulast%3DDrewes%26aufirst%3DG.%26aulast%3DOppermann%26aufirst%3DU.%26aulast%3DPatel%26aufirst%3DD.%2BJ.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DWilson%26aufirst%3DD.%2BM.%26atitle%3DA%2520selective%2520jumonji%2520H3K27%2520demethylase%2520inhibitor%2520modulates%2520the%2520proinflammatory%2520macrophage%2520response%26jtitle%3DNature%26date%3D2012%26volume%3D488%26spage%3D404%26epage%3D408%26doi%3D10.1038%2Fnature11262" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Heinemann, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nielsen, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hudlebusch, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lees, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larsen, D. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boesen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Labelle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerlach, L. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Birk, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helin, K.</span></span> <span> </span><span class="NLM_article-title">Inhibition of demethylases by GSK-J1/J4</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>514</i></span>,  <span class="NLM_fpage">E1</span>– <span class="NLM_lpage">E2</span>, <span class="refDoi"> DOI: 10.1038/nature13688</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=10.1038%2Fnature13688" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=514&publication_year=2014&pages=E1-E2&author=B.+Heinemannauthor=J.+M.+Nielsenauthor=H.+R.+Hudlebuschauthor=M.+J.+Leesauthor=D.+V.+Larsenauthor=T.+Boesenauthor=M.+Labelleauthor=L.+O.+Gerlachauthor=P.+Birkauthor=K.+Helin&title=Inhibition+of+demethylases+by+GSK-J1%2FJ4&doi=10.1038%2Fnature13688"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1038%2Fnature13688&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature13688%26sid%3Dliteratum%253Aachs%26aulast%3DHeinemann%26aufirst%3DB.%26aulast%3DNielsen%26aufirst%3DJ.%2BM.%26aulast%3DHudlebusch%26aufirst%3DH.%2BR.%26aulast%3DLees%26aufirst%3DM.%2BJ.%26aulast%3DLarsen%26aufirst%3DD.%2BV.%26aulast%3DBoesen%26aufirst%3DT.%26aulast%3DLabelle%26aufirst%3DM.%26aulast%3DGerlach%26aufirst%3DL.%2BO.%26aulast%3DBirk%26aufirst%3DP.%26aulast%3DHelin%26aufirst%3DK.%26atitle%3DInhibition%2520of%2520demethylases%2520by%2520GSK-J1%252FJ4%26jtitle%3DNature%26date%3D2014%26volume%3D514%26spage%3DE1%26epage%3DE2%26doi%3D10.1038%2Fnature13688" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cee, V. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Volak, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartberger, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tegley, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arvedson, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCarter, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tasker, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fotsch, C.</span></span> <span> </span><span class="NLM_article-title">Systematic study of the glutathione (GSH) reactivity of N-arylacrylamides: 1. Effects of aryl substitution</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">9171</span>– <span class="NLM_lpage">9178</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01018</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01018" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFGrsLfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=9171-9178&author=V.+J.+Ceeauthor=L.+P.+Volakauthor=Y.+Chenauthor=M.+D.+Bartbergerauthor=C.+Tegleyauthor=T.+Arvedsonauthor=J.+McCarterauthor=A.+S.+Taskerauthor=C.+Fotsch&title=Systematic+study+of+the+glutathione+%28GSH%29+reactivity+of+N-arylacrylamides%3A+1.+Effects+of+aryl+substitution&doi=10.1021%2Facs.jmedchem.5b01018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Systematic Study of the Glutathione (GSH) Reactivity of N-Arylacrylamides: 1. Effects of Aryl Substitution</span></div><div class="casAuthors">Cee, Victor J.; Volak, Laurie P.; Chen, Yuping; Bartberger, Michael D.; Tegley, Chris; Arvedson, Tara; McCarter, John; Tasker, Andrew S.; Fotsch, Christopher</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">9171-9178</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Success in the design of targeted covalent inhibitors depends in part on a knowledge of the factors influencing electrophile reactivity.  In an effort to further develop an understanding of structure-reactivity relations among N-arylacrylamides, the authors detd. glutathione (GSH) reaction rates for a family of N-arylacrylamides independently substituted at ortho-, meta-, and para-positions with 11 different groups common to inhibitor design.  Substituent effects on reaction rates show a linear Hammett correlation for ortho-, meta-, and para-substitution.  The authors note a correlation between 1H and 13C NMR chem. shifts of the acrylamide with GSH reaction rates, suggesting that NMR chem. shifts may be a convenient surrogate measure of relative acrylamide reactivity.  D. functional theory calcns. reveal a correlation between computed activation parameters and exptl. detd. reaction rates, validating the use of such methodol. for the screening of synthetic candidates in a prospective fashion.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoAoYRfo0lRj7Vg90H21EOLACvtfcHk0lhv6KveredTFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFGrsLfN&md5=2f3a816446dd73d5be061ad8d2195e2d</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01018%26sid%3Dliteratum%253Aachs%26aulast%3DCee%26aufirst%3DV.%2BJ.%26aulast%3DVolak%26aufirst%3DL.%2BP.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DBartberger%26aufirst%3DM.%2BD.%26aulast%3DTegley%26aufirst%3DC.%26aulast%3DArvedson%26aufirst%3DT.%26aulast%3DMcCarter%26aufirst%3DJ.%26aulast%3DTasker%26aufirst%3DA.%2BS.%26aulast%3DFotsch%26aufirst%3DC.%26atitle%3DSystematic%2520study%2520of%2520the%2520glutathione%2520%2528GSH%2529%2520reactivity%2520of%2520N-arylacrylamides%253A%25201.%2520Effects%2520of%2520aryl%2520substitution%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D9171%26epage%3D9178%26doi%3D10.1021%2Facs.jmedchem.5b01018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nicolaou, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Estrada, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zak, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Safina, B. S.</span></span> <span> </span><span class="NLM_article-title">A mild and selective method for the hydrolysis of esters with trimethyltin hydroxide</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">1378</span>– <span class="NLM_lpage">1382</span>, <span class="refDoi"> DOI: 10.1002/anie.200462207</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=10.1002%2Fanie.200462207" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=1%3ACAS%3A528%3ADC%252BD2MXitVSis7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2005&pages=1378-1382&author=K.+C.+Nicolaouauthor=A.+A.+Estradaauthor=M.+Zakauthor=S.+H.+Leeauthor=B.+S.+Safina&title=A+mild+and+selective+method+for+the+hydrolysis+of+esters+with+trimethyltin+hydroxide&doi=10.1002%2Fanie.200462207"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">A mild and selective method for the hydrolysis of esters with trimethyltin hydroxide</span></div><div class="casAuthors">Nicolaou, K. C.; Estrada, Anthony A.; Zak, Mark; Lee, Sang Hyup; Safina, Brian S.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1378-1382</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Mild, selective, and efficient: A method that involved the use of trimethyltin hydroxide for the hydrolysis of specific ester groups allowed chemists to steer clear of unwanted elimination reactions and epimerizations.  The conversion of an ester into the corresponding carboxylic acid took place under mild conditions, with nearly complete retention of stereochem. integrity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrz07978j08rLVg90H21EOLACvtfcHk0lhv6KveredTFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXitVSis7s%253D&md5=d0bd143616d9f2ab81c0dcbbb3abf9c7</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1002%2Fanie.200462207&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.200462207%26sid%3Dliteratum%253Aachs%26aulast%3DNicolaou%26aufirst%3DK.%2BC.%26aulast%3DEstrada%26aufirst%3DA.%2BA.%26aulast%3DZak%26aufirst%3DM.%26aulast%3DLee%26aufirst%3DS.%2BH.%26aulast%3DSafina%26aufirst%3DB.%2BS.%26atitle%3DA%2520mild%2520and%2520selective%2520method%2520for%2520the%2520hydrolysis%2520of%2520esters%2520with%2520trimethyltin%2520hydroxide%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2005%26volume%3D44%26spage%3D1378%26epage%3D1382%26doi%3D10.1002%2Fanie.200462207" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Horowitz, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trievel, R. C.</span></span> <span> </span><span class="NLM_article-title">Carbon-oxygen hydrogen bonding in biological structure and function</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>287</i></span>,  <span class="NLM_fpage">41576</span>– <span class="NLM_lpage">41582</span>, <span class="refDoi"> DOI: 10.1074/jbc.R112.418574</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=10.1074%2Fjbc.R112.418574" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=23048026" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=1%3ACAS%3A528%3ADC%252BC38XhslyksrnP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=287&publication_year=2012&pages=41576-41582&author=S.+Horowitzauthor=R.+C.+Trievel&title=Carbon-oxygen+hydrogen+bonding+in+biological+structure+and+function&doi=10.1074%2Fjbc.R112.418574"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Carbon-Oxygen Hydrogen Bonding in Biological Structure and Function</span></div><div class="casAuthors">Horowitz, Scott; Trievel, Raymond C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">287</span>
        (<span class="NLM_cas:issue">50</span>),
    <span class="NLM_cas:pages">41576-41582</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">A review.  Carbon-oxygen (CH···O) hydrogen bonding represents an unusual category of mol. interactions first documented in biol. structures over 4 decades ago.  Although CH···O hydrogen bonding has remained generally underappreciated in the biochem. literature, studies over the last 15 years have begun to yield direct evidence of these interactions in biol. systems.  In this minireview, we provide a historical context of biol. CH···O hydrogen bonding and summarize some major advancements from exptl. studies over the past several years that have elucidated the importance, prevalence, and functions of these interactions.  In particular, we examine the impact of CH···O bonds on protein and nucleic acid structure, mol. recognition, and enzyme catalysis and conclude by exploring overarching themes and unresolved questions regarding unconventional interactions in biomol. structure.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHj0osWPCzV7Vg90H21EOLACvtfcHk0liWCoz1dTdR_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhslyksrnP&md5=e96e7e30d5f5e03b0733fea323ecd6a2</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1074%2Fjbc.R112.418574&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.R112.418574%26sid%3Dliteratum%253Aachs%26aulast%3DHorowitz%26aufirst%3DS.%26aulast%3DTrievel%26aufirst%3DR.%2BC.%26atitle%3DCarbon-oxygen%2520hydrogen%2520bonding%2520in%2520biological%2520structure%2520and%2520function%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2012%26volume%3D287%26spage%3D41576%26epage%3D41582%26doi%3D10.1074%2Fjbc.R112.418574" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Deeks, E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keating, G. M.</span></span> <span> </span><span class="NLM_article-title">Afatinib in advanced NSCLC: a profile of its use</span>. <i>Drugs Ther Perspect</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">89</span>– <span class="NLM_lpage">98</span>, <span class="refDoi"> DOI: 10.1007/s40267-018-0482-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=10.1007%2Fs40267-018-0482-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=29540977" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=1%3ACAS%3A280%3ADC%252BC1MnhtlSgsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2018&pages=89-98&author=E.+D.+Deeksauthor=G.+M.+Keating&title=Afatinib+in+advanced+NSCLC%3A+a+profile+of+its+use&doi=10.1007%2Fs40267-018-0482-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Afatinib in advanced NSCLC: a profile of its use</span></div><div class="casAuthors">Deeks Emma D; Keating Gillian M</div><div class="citationInfo"><span class="NLM_cas:title">Drugs & therapy perspectives : for rational drug selection and use</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">89-98</span>
        ISSN:<span class="NLM_cas:issn">1172-0360</span>.
    </div><div class="casAbstract">Afatinib [Giotrif(®) (EU); Gilotrif(®) (USA)] is an orally administered, irreversible inhibitor of the ErbB family of tyrosine kinases that provides an important first-line treatment option for advanced non-small cell lung cancer (NSCLC) with activating epidermal growth factor receptor (EGFR) mutations (i.e.  EGFR(actMUT+)), and an additional treatment option for squamous NSCLC that has progressed following first-line platinum-based chemotherapy.  Relative to gefitinib in the first-line treatment of EGFR(actMUT+) advanced lung adenocarcinoma, afatinib prolonged progression-free survival (PFS) and time to treatment failure (TTF), but not overall survival (OS).  Afatinib also prolonged PFS, but not OS, versus cisplatin-based chemotherapy in this setting; however, afatinib improved OS versus chemotherapy in the subgroup of patients with deletions in exon 19.  As a second-line treatment for advanced squamous NSCLC, afatinib prolonged PFS and OS compared with erlotinib, regardless of EGFR mutation status.  Afatinib had a predictable and manageable tolerability profile.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQMwcu0ZW2rPGZyxfu64m4ffW6udTcc2ebNLjvQGfK0Q7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1MnhtlSgsw%253D%253D&md5=a1ea42cd72849bc384d59c3491ddbf4f</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1007%2Fs40267-018-0482-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs40267-018-0482-6%26sid%3Dliteratum%253Aachs%26aulast%3DDeeks%26aufirst%3DE.%2BD.%26aulast%3DKeating%26aufirst%3DG.%2BM.%26atitle%3DAfatinib%2520in%2520advanced%2520NSCLC%253A%2520a%2520profile%2520of%2520its%2520use%26jtitle%3DDrugs%2520Ther%2520Perspect%26date%3D2018%26volume%3D34%26spage%3D89%26epage%3D98%26doi%3D10.1007%2Fs40267-018-0482-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Solca, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dahl, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zoephel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bader, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanderson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kraemer, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Himmelsbach, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haaksma, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adolf, G. R.</span></span> <span> </span><span class="NLM_article-title">Target binding properties and cellular activity of afatinib (BIBW 2992), an irreversible ErbB family blocker</span>. <i>J. Pharmacol. Exp. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>343</i></span>,  <span class="NLM_fpage">342</span>– <span class="NLM_lpage">350</span>, <span class="refDoi"> DOI: 10.1124/jpet.112.197756</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=10.1124%2Fjpet.112.197756" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=22888144" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFKktbzE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=343&publication_year=2012&pages=342-350&author=F.+Solcaauthor=G.+Dahlauthor=A.+Zoephelauthor=G.+Baderauthor=M.+Sandersonauthor=C.+Kleinauthor=O.+Kraemerauthor=F.+Himmelsbachauthor=E.+Haaksmaauthor=G.+R.+Adolf&title=Target+binding+properties+and+cellular+activity+of+afatinib+%28BIBW+2992%29%2C+an+irreversible+ErbB+family+blocker&doi=10.1124%2Fjpet.112.197756"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Target binding properties and cellular activity of afatinib (BIBW 2992), an irreversible ErbB family blocker</span></div><div class="casAuthors">Solca, Flavio; Dahl, Goeran; Zoephel, Andreas; Bader, Gerd; Sanderson, Michael; Klein, Christian; Kraemer, Oliver; Himmelsbach, Frank; Haaksma, Eric; Adolf, Guenther R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmacology and Experimental Therapeutics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">343</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">342-350</span>CODEN:
                <span class="NLM_cas:coden">JPETAB</span>;
        ISSN:<span class="NLM_cas:issn">1521-0103</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Deregulation of the ErbB (proto-oncogene B of the avian erythroblastosis virus AEV-H strain) receptor network is well recognized as an oncogenic driver in epithelial cancers.  Several targeted drugs have been developed, including antibodies and small-mol. kinase inhibitors, each of them characterized by distinct patterns of ErbB receptor interactions.  Understanding the precise pharmacol. properties of these compds. is important for optimal use in clin. practice.  Afatinib [BIBW 2992; N-[4-[(3-chloro-4-fluorophenyl)amino]-7-[[(3S)-tetrahydro-3-furanyl]oxy]-6-quinazolinyl]-4-(dimethylamino)-2-butenamide] is an ATP-competitive anilinoquinazoline deriv. harboring a reactive acrylamide group.  It was designed to covalently bind and irreversibly block enzymically active ErbB receptor family members.  Here, we show by X-ray crystallog. the covalent binding of afatinib to wild-type epidermal growth factor receptor (EGFR) and by mass spectrometry the covalent interaction with EGFR, EGFRL858R/T790M, human epidermal growth factor receptor 2 (HER2), and ErbB-4.  Afatinib potently inhibits the enzymatic activity of ErbB-4 (EC50 = 1 nM) and the proliferation of cancer cell lines driven by multiple ErbB receptor aberrations at concns. below 100 nM.  N-[4-[(3-chloro-4-fluorophenyl)amino]-7-[[(3S)-tetrahydro-3-furanyl]oxy]-6-quinazolinyl]-4-(dimethylamino)-2-butanamide (BI 37781), a close analog of afatinib lacking the acrylamide group and thus incapable of covalent bond formation, had similar potency on cells driven by EGFR or EGFRL858R, but less or no detectable activity on cells expressing EGFRL858R/T790M HER2 or ErbB-4.  These results stress the importance of the acrylamide group and show that afatinib differs from approved ErbB targeting agents by irreversibly inhibiting the kinase activity of all ErbB family members.  They provide a mechanistic rationale for the distinct pharmacol. features of this compd. and explain the clin. activity seen in some patients who are resistant to antibody or kinase inhibitor therapy because of secondary mutations or ErbB receptor "reprogramming.".</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGofXgQqaKXv-bVg90H21EOLACvtfcHk0ljGy7Ln8NdNhw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFKktbzE&md5=4a57df32abe8101f287fc6251f63f92d</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1124%2Fjpet.112.197756&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fjpet.112.197756%26sid%3Dliteratum%253Aachs%26aulast%3DSolca%26aufirst%3DF.%26aulast%3DDahl%26aufirst%3DG.%26aulast%3DZoephel%26aufirst%3DA.%26aulast%3DBader%26aufirst%3DG.%26aulast%3DSanderson%26aufirst%3DM.%26aulast%3DKlein%26aufirst%3DC.%26aulast%3DKraemer%26aufirst%3DO.%26aulast%3DHimmelsbach%26aufirst%3DF.%26aulast%3DHaaksma%26aufirst%3DE.%26aulast%3DAdolf%26aufirst%3DG.%2BR.%26atitle%3DTarget%2520binding%2520properties%2520and%2520cellular%2520activity%2520of%2520afatinib%2520%2528BIBW%25202992%2529%252C%2520an%2520irreversible%2520ErbB%2520family%2520blocker%26jtitle%3DJ.%2520Pharmacol.%2520Exp.%2520Ther.%26date%3D2012%26volume%3D343%26spage%3D342%26epage%3D350%26doi%3D10.1124%2Fjpet.112.197756" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Alves, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vidugiris, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goueli, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zegzouti, H.</span></span> <span> </span><span class="NLM_article-title">Bioluminescent high-throughput succinate detection method for monitoring the activity of JMJC histone demethylases and Fe(II)/2-oxoglutarate-dependent dioxygenases</span>. <i>SLAS Discov</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">242</span>– <span class="NLM_lpage">254</span>, <span class="refDoi"> DOI: 10.1177/2472555217745657</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=10.1177%2F2472555217745657" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=29239273" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=1%3ACAS%3A528%3ADC%252BC1cXjtFCqtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2018&pages=242-254&author=J.+Alvesauthor=G.+Vidugirisauthor=S.+A.+Goueliauthor=H.+Zegzouti&title=Bioluminescent+high-throughput+succinate+detection+method+for+monitoring+the+activity+of+JMJC+histone+demethylases+and+Fe%28II%29%2F2-oxoglutarate-dependent+dioxygenases&doi=10.1177%2F2472555217745657"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Bioluminescent High-Throughput Succinate Detection Method for Monitoring the Activity of JMJC Histone Demethylases and Fe(II)/2-Oxoglutarate-Dependent Dioxygenases</span></div><div class="casAuthors">Alves, Juliano; Vidugiris, Gediminas; Goueli, Said A.; Zegzouti, Hicham</div><div class="citationInfo"><span class="NLM_cas:title">SLAS Discovery</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">242-254</span>CODEN:
                <span class="NLM_cas:coden">SDLIBT</span>;
        ISSN:<span class="NLM_cas:issn">2472-5560</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">The modification of a diverse array of substrates by Fe(II)/2-oxoglutarate-dependent dioxygenases is central to the modulation of distinct biol. processes such as epigenetics, hypoxic signaling, and DNA/RNA repair.  Of these, JumonjiC domain-contg. histone lysine demethylases (JMJCs) and prolyl hydroxylases are potential drug targets due to their relevance to human diseases.  Thus, assays to interrogate this enzyme superfamily are needed to identify selective and potent inhibitors as leads for drug development and that could also be useful research tools.  Since succinate is a common product to all Fe(II)/2-oxoglutarate-dependent dioxygenase reactions, a method that detects succinate would be suitable to all members of this enzyme superfamily.  We therefore developed a bioluminescent and homogenous succinate detection assay and validated its use with diverse sets of enzyme classes.  We evaluated the substrate specificities of these enzymes, their apparent kinetic consts., and inhibition profiles and mode of action of reported and novel inhibitors.  Our results indicate that succinate detection is a useful readout for the monitoring of enzymic activities with distinct substrate entities, as well as for the discovery of novel inhibitors.  By investigating a large no. of Fe(II)/2-oxoglutarate-dependent enzymes, this method could have a significant impact on the field of dioxygenase research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpvvqSTuW_ZHbVg90H21EOLACvtfcHk0ljGy7Ln8NdNhw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXjtFCqtr0%253D&md5=5501db23b33e5b42578cb043a9da8d9d</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1177%2F2472555217745657&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F2472555217745657%26sid%3Dliteratum%253Aachs%26aulast%3DAlves%26aufirst%3DJ.%26aulast%3DVidugiris%26aufirst%3DG.%26aulast%3DGoueli%26aufirst%3DS.%2BA.%26aulast%3DZegzouti%26aufirst%3DH.%26atitle%3DBioluminescent%2520high-throughput%2520succinate%2520detection%2520method%2520for%2520monitoring%2520the%2520activity%2520of%2520JMJC%2520histone%2520demethylases%2520and%2520Fe%2528II%2529%252F2-oxoglutarate-dependent%2520dioxygenases%26jtitle%3DSLAS%2520Discov%26date%3D2018%26volume%3D23%26spage%3D242%26epage%3D254%26doi%3D10.1177%2F2472555217745657" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Silverman, R. B.</span></span> <span> </span><span class="NLM_article-title">Mechanism-based enzyme inactivators</span>. <i>Methods Enzymol.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>249</i></span>,  <span class="NLM_fpage">240</span>– <span class="NLM_lpage">283</span>, <span class="refDoi"> DOI: 10.1016/0076-6879(95)49038-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=10.1016%2F0076-6879%2895%2949038-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=7791614" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=1%3ACAS%3A528%3ADyaK2MXmvFSlurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=249&publication_year=1995&pages=240-283&author=R.+B.+Silverman&title=Mechanism-based+enzyme+inactivators&doi=10.1016%2F0076-6879%2895%2949038-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanism-based enzyme inactivators</span></div><div class="casAuthors">Silverman, Richard B.</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Enzymology</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">249</span>
        (<span class="NLM_cas:issue">Enzyme Kinetics and Mechanism, Part D</span>),
    <span class="NLM_cas:pages">240-83</span>CODEN:
                <span class="NLM_cas:coden">MENZAU</span>;
        ISSN:<span class="NLM_cas:issn">0076-6879</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">A review, with 31 refs., on the terminol., basic kinetics, and uses of enzyme inactivators; criteria and exptl. protocols for mechanism-based enzyme inactivators; and applications of enzyme inactivators with examples from the author's lab.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrMhLcBvU-U4bVg90H21EOLACvtfcHk0ljGy7Ln8NdNhw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXmvFSlurw%253D&md5=775c81e5ce7d0548d3468fc60b8243f5</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2F0076-6879%2895%2949038-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0076-6879%252895%252949038-8%26sid%3Dliteratum%253Aachs%26aulast%3DSilverman%26aufirst%3DR.%2BB.%26atitle%3DMechanism-based%2520enzyme%2520inactivators%26jtitle%3DMethods%2520Enzymol.%26date%3D1995%26volume%3D249%26spage%3D240%26epage%3D283%26doi%3D10.1016%2F0076-6879%2895%2949038-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerns, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jadhav, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, X.</span></span> <span> </span><span class="NLM_article-title">Highly predictive and interpretable models for PAMPA permeability</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1266</span>– <span class="NLM_lpage">1276</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2016.12.049</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=10.1016%2Fj.bmc.2016.12.049" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=28082071" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=1%3ACAS%3A528%3ADC%252BC2sXntlOhsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2017&pages=1266-1276&author=H.+Sunauthor=K.+Nguyenauthor=E.+Kernsauthor=Z.+Yanauthor=K.+R.+Yuauthor=P.+Shahauthor=A.+Jadhavauthor=X.+Xu&title=Highly+predictive+and+interpretable+models+for+PAMPA+permeability&doi=10.1016%2Fj.bmc.2016.12.049"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Highly predictive and interpretable models for PAMPA permeability</span></div><div class="casAuthors">Sun, Hongmao; Nguyen, Kimloan; Kerns, Edward; Yan, Zhengyin; Yu, Kyeong Ri; Shah, Pranav; Jadhav, Ajit; Xu, Xin</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1266-1276</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Cell membrane permeability is an important determinant for oral absorption and bioavailability of a drug mol.  An in silico model predicting drug permeability is described, which is built based on a large permeability dataset of 7488 compd. entries or 5435 structurally unique mols. measured by the same lab using parallel artificial membrane permeability assay (PAMPA).  On the basis of customized mol. descriptors, the support vector regression (SVR) model trained with 4071 compds. with quant. data is able to predict the remaining 1364 compds. with the qual. data with an area under the curve of receiver operating characteristic (AUC-ROC) of 0.90.  The support vector classification (SVC) model trained with half of the whole dataset comprised of both the quant. and the qual. data produced accurate predictions to the remaining data with the AUC-ROC of 0.88.  The results suggest that the developed SVR model is highly predictive and provides medicinal chemists a useful in silico tool to facilitate design and synthesis of novel compds. with optimal drug-like properties, and thus accelerate the lead optimization in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-y8FhTLLqbrVg90H21EOLACvtfcHk0lg5-W11x96Qaw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXntlOhsA%253D%253D&md5=2f6b60dfcd168b5bf837405ca52457b2</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2016.12.049&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2016.12.049%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DH.%26aulast%3DNguyen%26aufirst%3DK.%26aulast%3DKerns%26aufirst%3DE.%26aulast%3DYan%26aufirst%3DZ.%26aulast%3DYu%26aufirst%3DK.%2BR.%26aulast%3DShah%26aufirst%3DP.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DXu%26aufirst%3DX.%26atitle%3DHighly%2520predictive%2520and%2520interpretable%2520models%2520for%2520PAMPA%2520permeability%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2017%26volume%3D25%26spage%3D1266%26epage%3D1276%26doi%3D10.1016%2Fj.bmc.2016.12.049" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lang, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horton, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jansen, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mott, B. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pohida, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rai, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kales, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henderson, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jadhav, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maloney, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simeonov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwasaki, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shadel, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, Q.</span></span> <span> </span><span class="NLM_article-title">KDM5 histone demethylases repress immune response via suppression of STING</span>. <i>PLoS Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">e2006134</span>, <span class="refDoi"> DOI: 10.1371/journal.pbio.2006134</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=10.1371%2Fjournal.pbio.2006134" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2018&pages=e2006134&author=L.+Wuauthor=J.+Caoauthor=W.+L.+Caiauthor=S.+M.+Langauthor=J.+R.+Hortonauthor=D.+J.+Jansenauthor=Z.+Z.+Liuauthor=J.+F.+Chenauthor=M.+Zhangauthor=B.+T.+Mottauthor=K.+Pohidaauthor=G.+Raiauthor=S.+C.+Kalesauthor=M.+J.+Hendersonauthor=X.+Huauthor=A.+Jadhavauthor=D.+J.+Maloneyauthor=A.+Simeonovauthor=S.+Zhuauthor=A.+Iwasakiauthor=M.+D.+Hallauthor=X.+Chengauthor=G.+S.+Shadelauthor=Q.+Yan&title=KDM5+histone+demethylases+repress+immune+response+via+suppression+of+STING&doi=10.1371%2Fjournal.pbio.2006134"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pbio.2006134&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pbio.2006134%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DL.%26aulast%3DCao%26aufirst%3DJ.%26aulast%3DCai%26aufirst%3DW.%2BL.%26aulast%3DLang%26aufirst%3DS.%2BM.%26aulast%3DHorton%26aufirst%3DJ.%2BR.%26aulast%3DJansen%26aufirst%3DD.%2BJ.%26aulast%3DLiu%26aufirst%3DZ.%2BZ.%26aulast%3DChen%26aufirst%3DJ.%2BF.%26aulast%3DZhang%26aufirst%3DM.%26aulast%3DMott%26aufirst%3DB.%2BT.%26aulast%3DPohida%26aufirst%3DK.%26aulast%3DRai%26aufirst%3DG.%26aulast%3DKales%26aufirst%3DS.%2BC.%26aulast%3DHenderson%26aufirst%3DM.%2BJ.%26aulast%3DHu%26aufirst%3DX.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DMaloney%26aufirst%3DD.%2BJ.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DZhu%26aufirst%3DS.%26aulast%3DIwasaki%26aufirst%3DA.%26aulast%3DHall%26aufirst%3DM.%2BD.%26aulast%3DCheng%26aufirst%3DX.%26aulast%3DShadel%26aufirst%3DG.%2BS.%26aulast%3DYan%26aufirst%3DQ.%26atitle%3DKDM5%2520histone%2520demethylases%2520repress%2520immune%2520response%2520via%2520suppression%2520of%2520STING%26jtitle%3DPLoS%2520Biol.%26date%3D2018%26volume%3D16%26spage%3De2006134%26doi%3D10.1371%2Fjournal.pbio.2006134" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wright, S. W.</span></span> <span> </span><span class="NLM_article-title">Preparation of 2-, 4-, 5-, and 6-aminonicotinic acid tert-butyl esters</span>. <i>J. Heterocyclic Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">442</span>– <span class="NLM_lpage">445</span>, <span class="refDoi"> DOI: 10.1002/jhet.799</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=10.1002%2Fjhet.799" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=1%3ACAS%3A528%3ADC%252BC38XmtFeis7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2012&pages=442-445&author=S.+W.+Wright&title=Preparation+of+2-%2C+4-%2C+5-%2C+and+6-aminonicotinic+acid+tert-butyl+esters&doi=10.1002%2Fjhet.799"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Preparation of 2-, 4-, 5-, and 6-aminonicotinic acid tert-butyl esters</span></div><div class="casAuthors">Wright, Stephen W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Heterocyclic Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">442-445</span>CODEN:
                <span class="NLM_cas:coden">JHTCAD</span>;
        ISSN:<span class="NLM_cas:issn">1943-5193</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Procedures are reported to prep. the tert-Bu esters of 2-aminonicotinic acid, 4-aminonicotinic acid, 5-aminonicotinic acid, and 6-aminonicotinic acid from 2-chloronicotinic acid, 4-chloronicotinic acid, 5-bromonicotinic acid, and 6-chloronicotinic acid, resp., without need for purifn. of intermediates.  J. Heterocyclic Chem., (2012).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgRevQmfJ-LrVg90H21EOLACvtfcHk0lg5-W11x96Qaw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XmtFeis7c%253D&md5=87557ea8fd125a1c42575da0472d79c6</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1002%2Fjhet.799&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjhet.799%26sid%3Dliteratum%253Aachs%26aulast%3DWright%26aufirst%3DS.%2BW.%26atitle%3DPreparation%2520of%25202-%252C%25204-%252C%25205-%252C%2520and%25206-aminonicotinic%2520acid%2520tert-butyl%2520esters%26jtitle%3DJ.%2520Heterocyclic%2520Chem.%26date%3D2012%26volume%3D49%26spage%3D442%26epage%3D445%26doi%3D10.1002%2Fjhet.799" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krishnamurthy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senanayake, C. H.</span></span> <span> </span><span class="NLM_article-title">Noncryogenic I/Br-Mg exchange of aromatic halides bearing sensitive functional groups using i-PrMgCl-bis[2-(N,N-dimethylamino)ethyl] ether complexes</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">305</span>– <span class="NLM_lpage">307</span>, <span class="refDoi"> DOI: 10.1021/ol052704p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ol052704p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2006&pages=305-307&author=X.+J.+Wangauthor=X.+Sunauthor=L.+Zhangauthor=Y.+Xuauthor=D.+Krishnamurthyauthor=C.+H.+Senanayake&title=Noncryogenic+I%2FBr-Mg+exchange+of+aromatic+halides+bearing+sensitive+functional+groups+using+i-PrMgCl-bis%5B2-%28N%2CN-dimethylamino%29ethyl%5D+ether+complexes&doi=10.1021%2Fol052704p"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Fol052704p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fol052704p%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DX.%2BJ.%26aulast%3DSun%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DKrishnamurthy%26aufirst%3DD.%26aulast%3DSenanayake%26aufirst%3DC.%2BH.%26atitle%3DNoncryogenic%2520I%252FBr-Mg%2520exchange%2520of%2520aromatic%2520halides%2520bearing%2520sensitive%2520functional%2520groups%2520using%2520i-PrMgCl-bis%255B2-%2528N%252CN-dimethylamino%2529ethyl%255D%2520ether%2520complexes%26jtitle%3DOrg.%2520Lett.%26date%3D2006%26volume%3D8%26spage%3D305%26epage%3D307%26doi%3D10.1021%2Fol052704p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Horton, J. R.</span>; <span class="NLM_string-name">Gale, M.</span>; <span class="NLM_string-name">Yan, Q.</span>; <span class="NLM_string-name">Cheng, X.</span></span> <span> </span><span class="NLM_article-title">The molecular basis of histone demethylation</span>. In  <i>DNA and Histone Methylation as Cancer Targets</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaneda, A.</span>, <span class="NLM_string-name">Tsukada, Y.-i.</span></span>, Eds.; <span class="NLM_publisher-name">Cancer Drug Discovery and Development</span>, <span class="NLM_year">2017</span>; pp  <span class="NLM_fpage">151</span>– <span class="NLM_lpage">219</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=10.1007%2F978-3-319-59786-7_7" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&pages=151-219&author=J.+R.+Horton&author=M.+Gale&author=Q.+Yan&author=X.+Chengauthor=A.+Kaneda&author=Y.-i.+Tsukada&title=DNA+and+Histone+Methylation+as+Cancer+Targets"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1007%2F978-3-319-59786-7_7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F978-3-319-59786-7_7%26sid%3Dliteratum%253Aachs%26aulast%3DHorton%26aufirst%3DJ.%2BR.%26atitle%3DThe%2520molecular%2520basis%2520of%2520histone%2520demethylation%26btitle%3DDNA%2520and%2520Histone%2520Methylation%2520as%2520Cancer%2520Targets%26aulast%3DKaneda%26aufirst%3DA.%26pub%3DCancer%2520Drug%2520Discovery%2520and%2520Development%26date%3D2017%26spage%3D151%26epage%3D219" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lin, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beshiri, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujiwara, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Francis, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cherniack, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geisen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blair, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawamori, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grisanzio, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watanabe, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minamishima, Y. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kulkarni, R. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Signoretti, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodig, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bronson, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orkin, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tuck, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benevolenskaya, E. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyerson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaelin, W. G.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, Q.</span></span> <span> </span><span class="NLM_article-title">Loss of the retinoblastoma binding protein 2 (RBP2) histone demethylase suppresses tumorigenesis in mice lacking Rb1 or Men1</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>108</i></span>,  <span class="NLM_fpage">13379</span>– <span class="NLM_lpage">13386</span>, <span class="refDoi"> DOI: 10.1073/pnas.1110104108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=10.1073%2Fpnas.1110104108" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=108&publication_year=2011&pages=13379-13386&author=W.+Linauthor=J.+Caoauthor=J.+Liuauthor=M.+L.+Beshiriauthor=Y.+Fujiwaraauthor=J.+Francisauthor=A.+D.+Cherniackauthor=C.+Geisenauthor=L.+P.+Blairauthor=M.+R.+Zouauthor=X.+Shenauthor=D.+Kawamoriauthor=Z.+Liuauthor=C.+Grisanzioauthor=H.+Watanabeauthor=Y.+A.+Minamishimaauthor=Q.+Zhangauthor=R.+N.+Kulkarniauthor=S.+Signorettiauthor=S.+J.+Rodigauthor=R.+T.+Bronsonauthor=S.+H.+Orkinauthor=D.+P.+Tuckauthor=E.+V.+Benevolenskayaauthor=M.+Meyersonauthor=W.+G.+Kaelinauthor=Q.+Yan&title=Loss+of+the+retinoblastoma+binding+protein+2+%28RBP2%29+histone+demethylase+suppresses+tumorigenesis+in+mice+lacking+Rb1+or+Men1&doi=10.1073%2Fpnas.1110104108"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1110104108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1110104108%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DW.%26aulast%3DCao%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DBeshiri%26aufirst%3DM.%2BL.%26aulast%3DFujiwara%26aufirst%3DY.%26aulast%3DFrancis%26aufirst%3DJ.%26aulast%3DCherniack%26aufirst%3DA.%2BD.%26aulast%3DGeisen%26aufirst%3DC.%26aulast%3DBlair%26aufirst%3DL.%2BP.%26aulast%3DZou%26aufirst%3DM.%2BR.%26aulast%3DShen%26aufirst%3DX.%26aulast%3DKawamori%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DGrisanzio%26aufirst%3DC.%26aulast%3DWatanabe%26aufirst%3DH.%26aulast%3DMinamishima%26aufirst%3DY.%2BA.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DKulkarni%26aufirst%3DR.%2BN.%26aulast%3DSignoretti%26aufirst%3DS.%26aulast%3DRodig%26aufirst%3DS.%2BJ.%26aulast%3DBronson%26aufirst%3DR.%2BT.%26aulast%3DOrkin%26aufirst%3DS.%2BH.%26aulast%3DTuck%26aufirst%3DD.%2BP.%26aulast%3DBenevolenskaya%26aufirst%3DE.%2BV.%26aulast%3DMeyerson%26aufirst%3DM.%26aulast%3DKaelin%26aufirst%3DW.%2BG.%26aulast%3DYan%26aufirst%3DQ.%26atitle%3DLoss%2520of%2520the%2520retinoblastoma%2520binding%2520protein%25202%2520%2528RBP2%2529%2520histone%2520demethylase%2520suppresses%2520tumorigenesis%2520in%2520mice%2520lacking%2520Rb1%2520or%2520Men1%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2011%26volume%3D108%26spage%3D13379%26epage%3D13386%26doi%3D10.1073%2Fpnas.1110104108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheung, W. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Booth, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, D. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, Q.</span></span> <span> </span><span class="NLM_article-title">Histone demethylase RBP2 is critical for breast cancer progression and metastasis</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">868</span>– <span class="NLM_lpage">877</span>, <span class="refDoi"> DOI: 10.1016/j.celrep.2014.02.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=10.1016%2Fj.celrep.2014.02.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=24582965" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjsVWgsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2014&pages=868-877&author=J.+Caoauthor=Z.+Liuauthor=W.+K.+Cheungauthor=M.+Zhaoauthor=S.+Y.+Chenauthor=S.+W.+Chanauthor=C.+J.+Boothauthor=D.+X.+Nguyenauthor=Q.+Yan&title=Histone+demethylase+RBP2+is+critical+for+breast+cancer+progression+and+metastasis&doi=10.1016%2Fj.celrep.2014.02.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Histone Demethylase RBP2 Is Critical for Breast Cancer Progression and Metastasis</span></div><div class="casAuthors">Cao, Jian; Liu, Zongzhi; Cheung, William K. C.; Zhao, Minghui; Chen, Sophia Y.; Chan, Siew Wee; Booth, Carmen J.; Nguyen, Don X.; Yan, Qin</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">868-877</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Metastasis is a major clin. challenge for cancer treatment.  Emerging evidence suggests that aberrant epigenetic modifications contribute significantly to tumor formation and progression.  However, the drivers and roles of such epigenetic changes in tumor metastasis are still poorly understood.  Using bioinformatic anal. of human breast cancer gene-expression data sets, we identified histone demethylase RBP2 as a putative mediator of metastatic progression.  By using both human breast cancer cells and genetically engineered mice, we demonstrated that RBP2 is crit. for breast cancer metastasis to the lung in multiple in vivo models.  Mechanistically, RBP2 promotes metastasis as a pleiotropic pos. regulator of many metastasis genes, including TNC.  In addn., RBP2 loss suppresses tumor formation in MMTV-neu transgenic mice.  These results suggest that therapeutic targeting of RBP2 is a potential strategy for inhibition of tumor progression and metastasis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXZwTcPjZTYLVg90H21EOLACvtfcHk0ljVkN_xg_ttQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjsVWgsro%253D&md5=ecc62cbf9e2bcab60746534aa536b87d</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2014.02.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2014.02.004%26sid%3Dliteratum%253Aachs%26aulast%3DCao%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DCheung%26aufirst%3DW.%2BK.%26aulast%3DZhao%26aufirst%3DM.%26aulast%3DChen%26aufirst%3DS.%2BY.%26aulast%3DChan%26aufirst%3DS.%2BW.%26aulast%3DBooth%26aufirst%3DC.%2BJ.%26aulast%3DNguyen%26aufirst%3DD.%2BX.%26aulast%3DYan%26aufirst%3DQ.%26atitle%3DHistone%2520demethylase%2520RBP2%2520is%2520critical%2520for%2520breast%2520cancer%2520progression%2520and%2520metastasis%26jtitle%3DCell%2520Rep.%26date%3D2014%26volume%3D6%26spage%3D868%26epage%3D877%26doi%3D10.1016%2Fj.celrep.2014.02.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dalvi, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kollipara, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bayo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yenerall, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Timmons, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguez-Canales, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Behrens, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mino, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villalobos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parra, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suraokar, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pataer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swisher, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalhor, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhanu, N. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heymach, J. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coombes, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Girard, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gazdar, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kittler, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wistuba,  II</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minna, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinez, E. D.</span></span> <span> </span><span class="NLM_article-title">Taxane-platin-resistant lung cancers co-develop hypersensitivity to JumonjiC demethylase inhibitors</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">1669</span>– <span class="NLM_lpage">1684</span>, <span class="refDoi"> DOI: 10.1016/j.celrep.2017.04.077</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=10.1016%2Fj.celrep.2017.04.077" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=28538184" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=1%3ACAS%3A528%3ADC%252BC2sXotlygtrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2017&pages=1669-1684&author=M.+P.+Dalviauthor=L.+Wangauthor=R.+Zhongauthor=R.+K.+Kolliparaauthor=H.+Parkauthor=J.+Bayoauthor=P.+Yenerallauthor=Y.+Zhouauthor=B.+C.+Timmonsauthor=J.+Rodriguez-Canalesauthor=C.+Behrensauthor=B.+Minoauthor=P.+Villalobosauthor=E.+R.+Parraauthor=M.+Suraokarauthor=A.+Pataerauthor=S.+G.+Swisherauthor=N.+Kalhorauthor=N.+V.+Bhanuauthor=B.+A.+Garciaauthor=J.+V.+Heymachauthor=K.+Coombesauthor=Y.+Xieauthor=L.+Girardauthor=A.+F.+Gazdarauthor=R.+Kittlerauthor=+Wistubaauthor=J.+D.+Minnaauthor=E.+D.+Martinez&title=Taxane-platin-resistant+lung+cancers+co-develop+hypersensitivity+to+JumonjiC+demethylase+inhibitors&doi=10.1016%2Fj.celrep.2017.04.077"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Taxane-Platin-Resistant Lung Cancers Co-develop Hypersensitivity to JumonjiC Demethylase Inhibitors</span></div><div class="casAuthors">Dalvi, Maithili P.; Wang, Lei; Zhong, Rui; Kollipara, Rahul K.; Park, Hyunsil; Bayo, Juan; Yenerall, Paul; Zhou, Yunyun; Timmons, Brenda C.; Rodriguez-Canales, Jaime; Behrens, Carmen; Mino, Barbara; Villalobos, Pamela; Parra, Edwin R.; Suraokar, Milind; Pataer, Apar; Swisher, Stephen G.; Kalhor, Neda; Bhanu, Natarajan V.; Garcia, Benjamin A.; Heymach, John V.; Coombes, Kevin; Xie, Yang; Girard, Luc; Gazdar, Adi F.; Kittler, Ralf; Wistuba, Ignacio I.; Minna, John D.; Martinez, Elisabeth D.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1669-1684</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Although non-small cell lung cancer (NSCLC) patients benefit from std. taxane-platin chemotherapy, many relapse, developing drug resistance.  We established preclin. taxane-platin-chemoresistance models and identified a 35-gene resistance signature, which was assocd. with poor recurrence-free survival in neoadjuvant-treated NSCLC patients and included upregulation of the JumonjiC lysine demethylase KDM3B.  In fact, multi-drug-resistant cells progressively increased the expression of many JumonjiC demethylases, had altered histone methylation, and, importantly, showed hypersensitivity to JumonjiC inhibitors in vitro and in vivo.  Increasing taxane-platin resistance in progressive cell line series was accompanied by progressive sensitization to JIB-04 and GSK-J4.  These JumonjiC inhibitors partly reversed deregulated transcriptional programs, prevented the emergence of drug-tolerant colonies from chemo-naive cells, and synergized with std. chemotherapy in vitro and in vivo.  Our findings reveal JumonjiC inhibitors as promising therapies for targeting taxane-platin-chemoresistant NSCLCs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo23LDZmcc_1LVg90H21EOLACvtfcHk0lia8nCYpfn6Mg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXotlygtrc%253D&md5=d327d46e521fe85cab305a3bea1265f8</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2017.04.077&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2017.04.077%26sid%3Dliteratum%253Aachs%26aulast%3DDalvi%26aufirst%3DM.%2BP.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DZhong%26aufirst%3DR.%26aulast%3DKollipara%26aufirst%3DR.%2BK.%26aulast%3DPark%26aufirst%3DH.%26aulast%3DBayo%26aufirst%3DJ.%26aulast%3DYenerall%26aufirst%3DP.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DTimmons%26aufirst%3DB.%2BC.%26aulast%3DRodriguez-Canales%26aufirst%3DJ.%26aulast%3DBehrens%26aufirst%3DC.%26aulast%3DMino%26aufirst%3DB.%26aulast%3DVillalobos%26aufirst%3DP.%26aulast%3DParra%26aufirst%3DE.%2BR.%26aulast%3DSuraokar%26aufirst%3DM.%26aulast%3DPataer%26aufirst%3DA.%26aulast%3DSwisher%26aufirst%3DS.%2BG.%26aulast%3DKalhor%26aufirst%3DN.%26aulast%3DBhanu%26aufirst%3DN.%2BV.%26aulast%3DGarcia%26aufirst%3DB.%2BA.%26aulast%3DHeymach%26aufirst%3DJ.%2BV.%26aulast%3DCoombes%26aufirst%3DK.%26aulast%3DXie%26aufirst%3DY.%26aulast%3DGirard%26aufirst%3DL.%26aulast%3DGazdar%26aufirst%3DA.%2BF.%26aulast%3DKittler%26aufirst%3DR.%26aulast%3DWistuba%26aulast%3DMinna%26aufirst%3DJ.%2BD.%26aulast%3DMartinez%26aufirst%3DE.%2BD.%26atitle%3DTaxane-platin-resistant%2520lung%2520cancers%2520co-develop%2520hypersensitivity%2520to%2520JumonjiC%2520demethylase%2520inhibitors%26jtitle%3DCell%2520Rep.%26date%3D2017%26volume%3D19%26spage%3D1669%26epage%3D1684%26doi%3D10.1016%2Fj.celrep.2017.04.077" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roesch, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vultur, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bogeski, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimmermann, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Speicher, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korbel, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laschke, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gimotty, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philipp, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krause, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patzold, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villanueva, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krepler, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fukunaga-Kalabis, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoth, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bastian, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vogt, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herlyn, M.</span></span> <span> </span><span class="NLM_article-title">Overcoming intrinsic multidrug resistance in melanoma by blocking the mitochondrial respiratory chain of slow-cycling JARID1B(high) cells</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">811</span>– <span class="NLM_lpage">825</span>, <span class="refDoi"> DOI: 10.1016/j.ccr.2013.05.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=10.1016%2Fj.ccr.2013.05.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=23764003" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=1%3ACAS%3A528%3ADC%252BC3sXptFKku7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2013&pages=811-825&author=A.+Roeschauthor=A.+Vulturauthor=I.+Bogeskiauthor=H.+Wangauthor=K.+M.+Zimmermannauthor=D.+Speicherauthor=C.+Korbelauthor=M.+W.+Laschkeauthor=P.+A.+Gimottyauthor=S.+E.+Philippauthor=E.+Krauseauthor=S.+Patzoldauthor=J.+Villanuevaauthor=C.+Kreplerauthor=M.+Fukunaga-Kalabisauthor=M.+Hothauthor=B.+C.+Bastianauthor=T.+Vogtauthor=M.+Herlyn&title=Overcoming+intrinsic+multidrug+resistance+in+melanoma+by+blocking+the+mitochondrial+respiratory+chain+of+slow-cycling+JARID1B%28high%29+cells&doi=10.1016%2Fj.ccr.2013.05.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Overcoming Intrinsic Multidrug Resistance in Melanoma by Blocking the Mitochondrial Respiratory Chain of Slow-Cycling JARID1Bhigh Cells</span></div><div class="casAuthors">Roesch, Alexander; Vultur, Adina; Bogeski, Ivan; Wang, Huan; Zimmermann, Katharina M.; Speicher, David; Koerbel, Christina; Laschke, Matthias W.; Gimotty, Phyllis A.; Philipp, Stephan E.; Krause, Elmar; Paetzold, Sylvie; Villanueva, Jessie; Krepler, Clemens; Fukunaga-Kalabis, Mizuho; Hoth, Markus; Bastian, Boris C.; Vogt, Thomas; Herlyn, Meenhard</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">811-825</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Despite success with BRAFV600E inhibitors, therapeutic responses in patients with metastatic melanoma are short-lived because of the acquisition of drug resistance.  We identified a mechanism of intrinsic multidrug resistance based on the survival of a tumor cell subpopulation.  Treatment with various drugs, including cisplatin and vemurafenib, uniformly leads to enrichment of slow-cycling, long-term tumor-maintaining melanoma cells expressing the H3K4-demethylase JARID1B/KDM5B/PLU-1.  Proteome-profiling revealed an upregulation in enzymes of mitochondrial oxidative-ATP-synthesis (oxidative phosphorylation) in this subpopulation.  Inhibition of mitochondrial respiration blocked the emergence of the JARID1Bhigh subpopulation and sensitized melanoma cells to therapy, independent of their genotype.  Our findings support a two-tiered approach combining anticancer agents that eliminate rapidly proliferating melanoma cells with inhibitors of the drug-resistant slow-cycling subpopulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrLdv3RllMhCbVg90H21EOLACvtfcHk0lia8nCYpfn6Mg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXptFKku7s%253D&md5=ba9ce8134c1fc0f17c44aebae7e760f0</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2013.05.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2013.05.003%26sid%3Dliteratum%253Aachs%26aulast%3DRoesch%26aufirst%3DA.%26aulast%3DVultur%26aufirst%3DA.%26aulast%3DBogeski%26aufirst%3DI.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DZimmermann%26aufirst%3DK.%2BM.%26aulast%3DSpeicher%26aufirst%3DD.%26aulast%3DKorbel%26aufirst%3DC.%26aulast%3DLaschke%26aufirst%3DM.%2BW.%26aulast%3DGimotty%26aufirst%3DP.%2BA.%26aulast%3DPhilipp%26aufirst%3DS.%2BE.%26aulast%3DKrause%26aufirst%3DE.%26aulast%3DPatzold%26aufirst%3DS.%26aulast%3DVillanueva%26aufirst%3DJ.%26aulast%3DKrepler%26aufirst%3DC.%26aulast%3DFukunaga-Kalabis%26aufirst%3DM.%26aulast%3DHoth%26aufirst%3DM.%26aulast%3DBastian%26aufirst%3DB.%2BC.%26aulast%3DVogt%26aufirst%3DT.%26aulast%3DHerlyn%26aufirst%3DM.%26atitle%3DOvercoming%2520intrinsic%2520multidrug%2520resistance%2520in%2520melanoma%2520by%2520blocking%2520the%2520mitochondrial%2520respiratory%2520chain%2520of%2520slow-cycling%2520JARID1B%2528high%2529%2520cells%26jtitle%3DCancer%2520Cell%26date%3D2013%26volume%3D23%26spage%3D811%26epage%3D825%26doi%3D10.1016%2Fj.ccr.2013.05.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, D. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quinlan, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maheswaran, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDermott, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Azizian, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischbach, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brandstetter, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wittner, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramaswamy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Classon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Settleman, J.</span></span> <span> </span><span class="NLM_article-title">A chromatin-mediated reversible drug-tolerant state in cancer cell subpopulations</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>141</i></span>,  <span class="NLM_fpage">69</span>– <span class="NLM_lpage">80</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2010.02.027</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=10.1016%2Fj.cell.2010.02.027" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=20371346" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=1%3ACAS%3A528%3ADC%252BC3cXlsVSgtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=141&publication_year=2010&pages=69-80&author=S.+V.+Sharmaauthor=D.+Y.+Leeauthor=B.+Liauthor=M.+P.+Quinlanauthor=F.+Takahashiauthor=S.+Maheswaranauthor=U.+McDermottauthor=N.+Azizianauthor=L.+Zouauthor=M.+A.+Fischbachauthor=K.+K.+Wongauthor=K.+Brandstetterauthor=B.+Wittnerauthor=S.+Ramaswamyauthor=M.+Classonauthor=J.+Settleman&title=A+chromatin-mediated+reversible+drug-tolerant+state+in+cancer+cell+subpopulations&doi=10.1016%2Fj.cell.2010.02.027"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">A chromatin-mediated reversible drug-tolerant state in cancer cell subpopulations</span></div><div class="casAuthors">Sharma, Sreenath V.; Lee, Diana Y.; Li, Bihua; Quinlan, Margaret P.; Takahashi, Fumiyuki; Maheswaran, Shyamala; McDermott, Ultan; Azizian, Nancy; Zou, Lee; Fischbach, Michael A.; Wong, Kwok-Kin; Brandstetter, Kathleyn; Wittner, Ben; Ramaswamy, Sridhar; Classon, Marie; Settleman, Jeff</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">141</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">69-80</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Accumulating evidence implicates heterogeneity within cancer cell populations in the response to stressful exposures, including drug treatments.  While modeling the acute response to various anticancer agents in drug-sensitive human tumor cell lines, we consistently detected a small subpopulation of reversibly "drug-tolerant" cells.  These cells demonstrate >100-fold reduced drug sensitivity and maintain viability via engagement of IGF-1 receptor signaling and an altered chromatin state that requires the histone demethylase RBP2/KDM5A/Jarid1A.  This drug-tolerant phenotype is transiently acquired and relinquished at low frequency by individual cells within the population, implicating the dynamic regulation of phenotypic heterogeneity in drug tolerance.  The drug-tolerant subpopulation can be selectively ablated by treatment with IGF-1 receptor inhibitors or chromatin-modifying agents, potentially yielding a therapeutic opportunity.  Together, these findings suggest that cancer cell populations employ a dynamic survival strategy in which individual cells transiently assume a reversibly drug-tolerant state to protect the population from eradication by potentially lethal exposures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpK0DpI9mbR_bVg90H21EOLACvtfcHk0liPS5AE8_iVXQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXlsVSgtb8%253D&md5=2070b7cc4385a4f98db15e5ebcd4589d</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2010.02.027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2010.02.027%26sid%3Dliteratum%253Aachs%26aulast%3DSharma%26aufirst%3DS.%2BV.%26aulast%3DLee%26aufirst%3DD.%2BY.%26aulast%3DLi%26aufirst%3DB.%26aulast%3DQuinlan%26aufirst%3DM.%2BP.%26aulast%3DTakahashi%26aufirst%3DF.%26aulast%3DMaheswaran%26aufirst%3DS.%26aulast%3DMcDermott%26aufirst%3DU.%26aulast%3DAzizian%26aufirst%3DN.%26aulast%3DZou%26aufirst%3DL.%26aulast%3DFischbach%26aufirst%3DM.%2BA.%26aulast%3DWong%26aufirst%3DK.%2BK.%26aulast%3DBrandstetter%26aufirst%3DK.%26aulast%3DWittner%26aufirst%3DB.%26aulast%3DRamaswamy%26aufirst%3DS.%26aulast%3DClasson%26aufirst%3DM.%26aulast%3DSettleman%26aufirst%3DJ.%26atitle%3DA%2520chromatin-mediated%2520reversible%2520drug-tolerant%2520state%2520in%2520cancer%2520cell%2520subpopulations%26jtitle%3DCell%26date%3D2010%26volume%3D141%26spage%3D69%26epage%3D80%26doi%3D10.1016%2Fj.cell.2010.02.027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Westaway, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Preston, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barker, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drewes, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eagle, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garton, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haslam, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayhow, T. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Humphreys, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joberty, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katso, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kruidenier, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leveridge, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pemberton, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioja, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seal, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shipley, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suckling, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span> <span> </span><span class="NLM_article-title">Cell penetrant inhibitors of the KDM4 and KDM5 families of histone lysine demethylases. 2. Pyrido[3,4-d]pyrimidin-4(3H)-one derivatives</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">1370</span>– <span class="NLM_lpage">1387</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01538</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01538" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtVKmsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=1370-1387&author=S.+M.+Westawayauthor=A.+G.+Prestonauthor=M.+D.+Barkerauthor=F.+Brownauthor=J.+A.+Brownauthor=M.+Campbellauthor=C.+W.+Chungauthor=G.+Drewesauthor=R.+Eagleauthor=N.+Gartonauthor=L.+Gordonauthor=C.+Haslamauthor=T.+G.+Hayhowauthor=P.+G.+Humphreysauthor=G.+Jobertyauthor=R.+Katsoauthor=L.+Kruidenierauthor=M.+Leveridgeauthor=M.+Pembertonauthor=I.+Riojaauthor=G.+A.+Sealauthor=T.+Shipleyauthor=O.+Singhauthor=C.+J.+Sucklingauthor=J.+Taylorauthor=P.+Thomasauthor=D.+M.+Wilsonauthor=K.+Leeauthor=R.+K.+Prinjha&title=Cell+penetrant+inhibitors+of+the+KDM4+and+KDM5+families+of+histone+lysine+demethylases.+2.+Pyrido%5B3%2C4-d%5Dpyrimidin-4%283H%29-one+derivatives&doi=10.1021%2Facs.jmedchem.5b01538"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Cell Penetrant Inhibitors of the KDM4 and KDM5 Families of Histone Lysine Demethylases. 2. Pyrido[3,4-d]pyrimidin-4(3H)-one Derivatives</span></div><div class="casAuthors">Westaway, Susan M.; Preston, Alex G. S.; Barker, Michael D.; Brown, Fiona; Brown, Jack A.; Campbell, Matthew; Chung, Chun-wa; Drewes, Gerard; Eagle, Robert; Garton, Neil; Gordon, Laurie; Haslam, Carl; Hayhow, Thomas G.; Humphreys, Philip G.; Joberty, Gerard; Katso, Roy; Kruidenier, Laurens; Leveridge, Melanie; Pemberton, Michelle; Rioja, Inma; Seal, Gail A.; Shipley, Tracy; Singh, Onkar; Suckling, Colin J.; Taylor, Joanna; Thomas, Pamela; Wilson, David M.; Lee, Kevin; Prinjha, Rab K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1370-1387</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Following the discovery of cell penetrant pyridine-4-carboxylate inhibitors of the KDM4 (JMJD2) and KDM5 (JARID1) families of histone lysine demethylases (e.g., 1), further optimization led to the identification of non-carboxylate inhibitors derived from pyrido[3,4-d]pyrimidin-4(3H)-one.  A no. of exemplars such as compd. 41 possess interesting activity profiles in KDM4C and KDM5C biochem. and target-specific, cellular mechanistic assays.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQbtUjkbJPebVg90H21EOLACvtfcHk0liPS5AE8_iVXQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtVKmsbk%253D&md5=c17e46ecd9017fa9856d7cd39c6701bd</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01538&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01538%26sid%3Dliteratum%253Aachs%26aulast%3DWestaway%26aufirst%3DS.%2BM.%26aulast%3DPreston%26aufirst%3DA.%2BG.%26aulast%3DBarker%26aufirst%3DM.%2BD.%26aulast%3DBrown%26aufirst%3DF.%26aulast%3DBrown%26aufirst%3DJ.%2BA.%26aulast%3DCampbell%26aufirst%3DM.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DDrewes%26aufirst%3DG.%26aulast%3DEagle%26aufirst%3DR.%26aulast%3DGarton%26aufirst%3DN.%26aulast%3DGordon%26aufirst%3DL.%26aulast%3DHaslam%26aufirst%3DC.%26aulast%3DHayhow%26aufirst%3DT.%2BG.%26aulast%3DHumphreys%26aufirst%3DP.%2BG.%26aulast%3DJoberty%26aufirst%3DG.%26aulast%3DKatso%26aufirst%3DR.%26aulast%3DKruidenier%26aufirst%3DL.%26aulast%3DLeveridge%26aufirst%3DM.%26aulast%3DPemberton%26aufirst%3DM.%26aulast%3DRioja%26aufirst%3DI.%26aulast%3DSeal%26aufirst%3DG.%2BA.%26aulast%3DShipley%26aufirst%3DT.%26aulast%3DSingh%26aufirst%3DO.%26aulast%3DSuckling%26aufirst%3DC.%2BJ.%26aulast%3DTaylor%26aufirst%3DJ.%26aulast%3DThomas%26aufirst%3DP.%26aulast%3DWilson%26aufirst%3DD.%2BM.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26atitle%3DCell%2520penetrant%2520inhibitors%2520of%2520the%2520KDM4%2520and%2520KDM5%2520families%2520of%2520histone%2520lysine%2520demethylases.%25202.%2520Pyrido%255B3%252C4-d%255Dpyrimidin-4%25283H%2529-one%2520derivatives%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D1370%26epage%3D1387%26doi%3D10.1021%2Facs.jmedchem.5b01538" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Westaway, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Preston, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barker, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diallo, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Douault, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drewes, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eagle, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haslam, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayhow, T. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Humphreys, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joberty, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katso, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kruidenier, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leveridge, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liddle, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mosley, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muelbaier, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Randle, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioja, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rueger, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seal, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheppard, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span> <span> </span><span class="NLM_article-title">Cell penetrant inhibitors of the KDM4 and KDM5 families of histone lysine demethylases. 1. 3-Amino-4-pyridine carboxylate derivatives</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">1357</span>– <span class="NLM_lpage">1369</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01537</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01537" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=1357-1369&author=S.+M.+Westawayauthor=A.+G.+Prestonauthor=M.+D.+Barkerauthor=F.+Brownauthor=J.+A.+Brownauthor=M.+Campbellauthor=C.+W.+Chungauthor=H.+Dialloauthor=C.+Douaultauthor=G.+Drewesauthor=R.+Eagleauthor=L.+Gordonauthor=C.+Haslamauthor=T.+G.+Hayhowauthor=P.+G.+Humphreysauthor=G.+Jobertyauthor=R.+Katsoauthor=L.+Kruidenierauthor=M.+Leveridgeauthor=J.+Liddleauthor=J.+Mosleyauthor=M.+Muelbaierauthor=R.+Randleauthor=I.+Riojaauthor=A.+Ruegerauthor=G.+A.+Sealauthor=R.+J.+Sheppardauthor=O.+Singhauthor=J.+Taylorauthor=P.+Thomasauthor=D.+Thomsonauthor=D.+M.+Wilsonauthor=K.+Leeauthor=R.+K.+Prinjha&title=Cell+penetrant+inhibitors+of+the+KDM4+and+KDM5+families+of+histone+lysine+demethylases.+1.+3-Amino-4-pyridine+carboxylate+derivatives&doi=10.1021%2Facs.jmedchem.5b01537"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01537&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01537%26sid%3Dliteratum%253Aachs%26aulast%3DWestaway%26aufirst%3DS.%2BM.%26aulast%3DPreston%26aufirst%3DA.%2BG.%26aulast%3DBarker%26aufirst%3DM.%2BD.%26aulast%3DBrown%26aufirst%3DF.%26aulast%3DBrown%26aufirst%3DJ.%2BA.%26aulast%3DCampbell%26aufirst%3DM.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DDiallo%26aufirst%3DH.%26aulast%3DDouault%26aufirst%3DC.%26aulast%3DDrewes%26aufirst%3DG.%26aulast%3DEagle%26aufirst%3DR.%26aulast%3DGordon%26aufirst%3DL.%26aulast%3DHaslam%26aufirst%3DC.%26aulast%3DHayhow%26aufirst%3DT.%2BG.%26aulast%3DHumphreys%26aufirst%3DP.%2BG.%26aulast%3DJoberty%26aufirst%3DG.%26aulast%3DKatso%26aufirst%3DR.%26aulast%3DKruidenier%26aufirst%3DL.%26aulast%3DLeveridge%26aufirst%3DM.%26aulast%3DLiddle%26aufirst%3DJ.%26aulast%3DMosley%26aufirst%3DJ.%26aulast%3DMuelbaier%26aufirst%3DM.%26aulast%3DRandle%26aufirst%3DR.%26aulast%3DRioja%26aufirst%3DI.%26aulast%3DRueger%26aufirst%3DA.%26aulast%3DSeal%26aufirst%3DG.%2BA.%26aulast%3DSheppard%26aufirst%3DR.%2BJ.%26aulast%3DSingh%26aufirst%3DO.%26aulast%3DTaylor%26aufirst%3DJ.%26aulast%3DThomas%26aufirst%3DP.%26aulast%3DThomson%26aufirst%3DD.%26aulast%3DWilson%26aufirst%3DD.%2BM.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26atitle%3DCell%2520penetrant%2520inhibitors%2520of%2520the%2520KDM4%2520and%2520KDM5%2520families%2520of%2520histone%2520lysine%2520demethylases.%25201.%25203-Amino-4-pyridine%2520carboxylate%2520derivatives%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D1357%26epage%3D1369%26doi%3D10.1021%2Facs.jmedchem.5b01537" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Watanuki, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kosai, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osanai, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ogama, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mochizuki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tamai, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamaguchi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Satoh, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fukuhara, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maemondo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ichinose, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nukiwa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, N.</span></span> <span> </span><span class="NLM_article-title">Synergistic cytotoxicity of afatinib and cetuximab against EGFR T790M involves Rab11-dependent EGFR recycling</span>. <i>Biochem. Biophys. Res. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>455</i></span>,  <span class="NLM_fpage">269</span>– <span class="NLM_lpage">276</span>, <span class="refDoi"> DOI: 10.1016/j.bbrc.2014.11.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=10.1016%2Fj.bbrc.2014.11.003" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=455&publication_year=2014&pages=269-276&author=Z.+Watanukiauthor=H.+Kosaiauthor=N.+Osanaiauthor=N.+Ogamaauthor=M.+Mochizukiauthor=K.+Tamaiauthor=K.+Yamaguchiauthor=K.+Satohauthor=T.+Fukuharaauthor=M.+Maemondoauthor=M.+Ichinoseauthor=T.+Nukiwaauthor=N.+Tanaka&title=Synergistic+cytotoxicity+of+afatinib+and+cetuximab+against+EGFR+T790M+involves+Rab11-dependent+EGFR+recycling&doi=10.1016%2Fj.bbrc.2014.11.003"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1016%2Fj.bbrc.2014.11.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbrc.2014.11.003%26sid%3Dliteratum%253Aachs%26aulast%3DWatanuki%26aufirst%3DZ.%26aulast%3DKosai%26aufirst%3DH.%26aulast%3DOsanai%26aufirst%3DN.%26aulast%3DOgama%26aufirst%3DN.%26aulast%3DMochizuki%26aufirst%3DM.%26aulast%3DTamai%26aufirst%3DK.%26aulast%3DYamaguchi%26aufirst%3DK.%26aulast%3DSatoh%26aufirst%3DK.%26aulast%3DFukuhara%26aufirst%3DT.%26aulast%3DMaemondo%26aufirst%3DM.%26aulast%3DIchinose%26aufirst%3DM.%26aulast%3DNukiwa%26aufirst%3DT.%26aulast%3DTanaka%26aufirst%3DN.%26atitle%3DSynergistic%2520cytotoxicity%2520of%2520afatinib%2520and%2520cetuximab%2520against%2520EGFR%2520T790M%2520involves%2520Rab11-dependent%2520EGFR%2520recycling%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D2014%26volume%3D455%26spage%3D269%26epage%3D276%26doi%3D10.1016%2Fj.bbrc.2014.11.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Castillo-Aguilera, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Depreux, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halby, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arimondo, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goossens, L.</span></span> <span> </span><span class="NLM_article-title">DNA methylation targeting: the DNMT/HMT crosstalk challenge</span>. <i>Biomolecules</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">3</span>, <span class="refDoi"> DOI: 10.3390/biom7010003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=10.3390%2Fbiom7010003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=1%3ACAS%3A528%3ADC%252BC2sXksVSisrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=3&author=O.+Castillo-Aguileraauthor=P.+Depreuxauthor=L.+Halbyauthor=P.+B.+Arimondoauthor=L.+Goossens&title=DNA+methylation+targeting%3A+the+DNMT%2FHMT+crosstalk+challenge&doi=10.3390%2Fbiom7010003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">DNA methylation targeting: the DNMT/HMT crosstalk challenge</span></div><div class="casAuthors">Castillo-Aguilera, Omar; Depreux, Patrick; Halby, Ludovic; Arimondo, Paola B.; Goossens, Laurence</div><div class="citationInfo"><span class="NLM_cas:title">Biomolecules</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">3/1-3/21</span>CODEN:
                <span class="NLM_cas:coden">BIOMHC</span>;
        ISSN:<span class="NLM_cas:issn">2218-273X</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">A review.  Chromatin can adopt a decondensed state linked to gene transcription (euchromatin) and a condensed state linked to transcriptional repression (heterochromatin).  These states are controlled by epigenetic modulators that are active on either the DNA or the histones and are tightly assocd. to each other.  Methylation of both DNA and histones is involved in either the activation or silencing of genes and their crosstalk.  Since DNA/histone methylation patterns are altered in cancers, mols. that target these modifications are interesting therapeutic tools.  We present herein a vast panel of DNA methyltransferase inhibitors classified according to their mechanism, as well as selected histone methyltransferase inhibitors sharing a common mode of action.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGorl0eU7GvR7LVg90H21EOLACvtfcHk0ljHglKphWyr3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXksVSisrY%253D&md5=7445d305512673f988a04e9c46e82fd2</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.3390%2Fbiom7010003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fbiom7010003%26sid%3Dliteratum%253Aachs%26aulast%3DCastillo-Aguilera%26aufirst%3DO.%26aulast%3DDepreux%26aufirst%3DP.%26aulast%3DHalby%26aufirst%3DL.%26aulast%3DArimondo%26aufirst%3DP.%2BB.%26aulast%3DGoossens%26aufirst%3DL.%26atitle%3DDNA%2520methylation%2520targeting%253A%2520the%2520DNMT%252FHMT%2520crosstalk%2520challenge%26jtitle%3DBiomolecules%26date%3D2017%26volume%3D7%26spage%3D3%26doi%3D10.3390%2Fbiom7010003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Patel, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dickson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Din, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macleod, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jodrell, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramsahoye, B.</span></span> <span> </span><span class="NLM_article-title">Targeting of 5-aza-2’-deoxycytidine residues by chromatin-associated DNMT1 induces proteasomal degradation of the free enzyme</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">4313</span>– <span class="NLM_lpage">4324</span>, <span class="refDoi"> DOI: 10.1093/nar/gkq187</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=10.1093%2Fnar%2Fgkq187" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2010&pages=4313-4324&author=K.+Patelauthor=J.+Dicksonauthor=S.+Dinauthor=K.+Macleodauthor=D.+Jodrellauthor=B.+Ramsahoye&title=Targeting+of+5-aza-2%E2%80%99-deoxycytidine+residues+by+chromatin-associated+DNMT1+induces+proteasomal+degradation+of+the+free+enzyme&doi=10.1093%2Fnar%2Fgkq187"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkq187&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkq187%26sid%3Dliteratum%253Aachs%26aulast%3DPatel%26aufirst%3DK.%26aulast%3DDickson%26aufirst%3DJ.%26aulast%3DDin%26aufirst%3DS.%26aulast%3DMacleod%26aufirst%3DK.%26aulast%3DJodrell%26aufirst%3DD.%26aulast%3DRamsahoye%26aufirst%3DB.%26atitle%3DTargeting%2520of%25205-aza-2%25E2%2580%2599-deoxycytidine%2520residues%2520by%2520chromatin-associated%2520DNMT1%2520induces%2520proteasomal%2520degradation%2520of%2520the%2520free%2520enzyme%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2010%26volume%3D38%26spage%3D4313%26epage%3D4324%26doi%3D10.1093%2Fnar%2Fgkq187" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Das, B. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renaud, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doroshow, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takeda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pommier, Y.</span></span> <span> </span><span class="NLM_article-title">Trapping of PARP1 and PARP2 by clinical PARP inhibitors</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>72</i></span>,  <span class="NLM_fpage">5588</span>– <span class="NLM_lpage">5599</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-12-2753</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=10.1158%2F0008-5472.CAN-12-2753" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=23118055" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsF2jsLjE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=2012&pages=5588-5599&author=J.+Muraiauthor=S.+Y.+Huangauthor=B.+B.+Dasauthor=A.+Renaudauthor=Y.+Zhangauthor=J.+H.+Doroshowauthor=J.+Jiauthor=S.+Takedaauthor=Y.+Pommier&title=Trapping+of+PARP1+and+PARP2+by+clinical+PARP+inhibitors&doi=10.1158%2F0008-5472.CAN-12-2753"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Trapping of PARP1 and PARP2 by Clinical PARP Inhibitors</span></div><div class="casAuthors">Murai, Junko; Huang, Shar-yin N.; Das, Benu Brata; Renaud, Amelie; Zhang, Yiping; Doroshow, James H.; Ji, Jiuping; Takeda, Shunichi; Pommier, Yves</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">72</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">5588-5599</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">These findings indicate that PARP inhibitors in clin. trials not only block catalytic function but also trap PARP enzymes on DNA, illuminating the mechanistic basis for their cytotoxic activity in cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrW3m55HXn9ObVg90H21EOLACvtfcHk0ljHglKphWyr3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsF2jsLjE&md5=81c590b6a4fbb9efc9c64f7de5e89524</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-12-2753&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-12-2753%26sid%3Dliteratum%253Aachs%26aulast%3DMurai%26aufirst%3DJ.%26aulast%3DHuang%26aufirst%3DS.%2BY.%26aulast%3DDas%26aufirst%3DB.%2BB.%26aulast%3DRenaud%26aufirst%3DA.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DDoroshow%26aufirst%3DJ.%2BH.%26aulast%3DJi%26aufirst%3DJ.%26aulast%3DTakeda%26aufirst%3DS.%26aulast%3DPommier%26aufirst%3DY.%26atitle%3DTrapping%2520of%2520PARP1%2520and%2520PARP2%2520by%2520clinical%2520PARP%2520inhibitors%26jtitle%3DCancer%2520Res.%26date%3D2012%26volume%3D72%26spage%3D5588%26epage%3D5599%26doi%3D10.1158%2F0008-5472.CAN-12-2753" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Muvarak, N. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chowdhury, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xia, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robert, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, E. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellani, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, Z. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duong, V. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rutherford, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagaria, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bentzen, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seidman, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baer, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lapidus, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baylin, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rassool, F. V.</span></span> <span> </span><span class="NLM_article-title">Enhancing the cytotoxic effects of PARP inhibitors with DNA demethylating agents - A potential therapy for cancer</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">637</span>– <span class="NLM_lpage">650</span>, <span class="refDoi"> DOI: 10.1016/j.ccell.2016.09.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;key=10.1016%2Fj.ccell.2016.09.002" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2016&pages=637-650&author=N.+E.+Muvarakauthor=K.+Chowdhuryauthor=L.+Xiaauthor=C.+Robertauthor=E.+Y.+Choiauthor=Y.+Caiauthor=M.+Bellaniauthor=Y.+Zouauthor=Z.+N.+Singhauthor=V.+H.+Duongauthor=T.+Rutherfordauthor=P.+Nagariaauthor=S.+M.+Bentzenauthor=M.+M.+Seidmanauthor=M.+R.+Baerauthor=R.+G.+Lapidusauthor=S.+B.+Baylinauthor=F.+V.+Rassool&title=Enhancing+the+cytotoxic+effects+of+PARP+inhibitors+with+DNA+demethylating+agents+-+A+potential+therapy+for+cancer&doi=10.1016%2Fj.ccell.2016.09.002"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2016.09.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2016.09.002%26sid%3Dliteratum%253Aachs%26aulast%3DMuvarak%26aufirst%3DN.%2BE.%26aulast%3DChowdhury%26aufirst%3DK.%26aulast%3DXia%26aufirst%3DL.%26aulast%3DRobert%26aufirst%3DC.%26aulast%3DChoi%26aufirst%3DE.%2BY.%26aulast%3DCai%26aufirst%3DY.%26aulast%3DBellani%26aufirst%3DM.%26aulast%3DZou%26aufirst%3DY.%26aulast%3DSingh%26aufirst%3DZ.%2BN.%26aulast%3DDuong%26aufirst%3DV.%2BH.%26aulast%3DRutherford%26aufirst%3DT.%26aulast%3DNagaria%26aufirst%3DP.%26aulast%3DBentzen%26aufirst%3DS.%2BM.%26aulast%3DSeidman%26aufirst%3DM.%2BM.%26aulast%3DBaer%26aufirst%3DM.%2BR.%26aulast%3DLapidus%26aufirst%3DR.%2BG.%26aulast%3DBaylin%26aufirst%3DS.%2BB.%26aulast%3DRassool%26aufirst%3DF.%2BV.%26atitle%3DEnhancing%2520the%2520cytotoxic%2520effects%2520of%2520PARP%2520inhibitors%2520with%2520DNA%2520demethylating%2520agents%2520-%2520A%2520potential%2520therapy%2520for%2520cancer%26jtitle%3DCancer%2520Cell%26date%3D2016%26volume%3D30%26spage%3D637%26epage%3D650%26doi%3D10.1016%2Fj.ccell.2016.09.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6CG1" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6CG1','PDB','6CG1'); return false;">PDB: 6CG1</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3AVS" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3AVS','PDB','3AVS'); return false;">PDB: 3AVS</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4QXB" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4QXB','PDB','4QXB'); return false;">PDB: 4QXB</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3KV4" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3KV4','PDB','3KV4'); return false;">PDB: 3KV4</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4C8D" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4C8D','PDB','4C8D'); return false;">PDB: 4C8D</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6DQ4" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6DQ4','PDB','6DQ4'); return false;">PDB: 6DQ4</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6DQ5" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6DQ5','PDB','6DQ5'); return false;">PDB: 6DQ5</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6DQ6" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6DQ6','PDB','6DQ6'); return false;">PDB: 6DQ6</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4ASK" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4ASK','PDB','4ASK'); return false;">PDB: 4ASK</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ZPO" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ZPO','PDB','3ZPO'); return false;">PDB: 3ZPO</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6DQ8" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6DQ8','PDB','6DQ8'); return false;">PDB: 6DQ8</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6DQA" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6DQA','PDB','6DQA'); return false;">PDB: 6DQA</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6DQB" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6DQB','PDB','6DQB'); return false;">PDB: 6DQB</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4G5J" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4G5J','PDB','4G5J'); return false;">PDB: 4G5J</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i33"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b01219">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_07823"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.8b01219">10.1021/acs.jmedchem.8b01219</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b01219/suppl_file/jm8b01219_si_001.csv">CSV</a>)</p></li><li><p class="inline">Figures S1–S5 and Table S1 (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b01219/suppl_file/jm8b01219_si_002.pdf">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b01219/suppl_file/jm8b01219_si_001.csv">jm8b01219_si_001.csv (0.34 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b01219/suppl_file/jm8b01219_si_002.pdf">jm8b01219_si_002.pdf (4.18 MB)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The atomic coordinates and structure factors of KDM5A in complex with <b>GSK-J1</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6DQ4">6DQ4</a>), <b>N43</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6DQ5">6DQ5</a>), <b>N44</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6DQ6">6DQ6</a>), <b>N49</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6DQ8">6DQ8</a>), <b>N70</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6DQA">6DQA</a>), and <b>N71</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6DQB">6DQB</a>) have been deposited in the Protein Data Bank.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.8b01219&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2018.61.issue-23%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.8b01219%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.8b01219" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                7MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"67993ea20f45198b","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
