
<html lang="en"     class="pb-page"  data-request-id="12dec1d7-dec3-435c-81ca-c35aaa55a619"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.9b02121;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text;issue:issue:10.1021/jmcmar.2020.63.issue-6"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Discovery and SARs of 5-Chloro-N4-phenyl-N2-(pyridin-2-yl)pyrimidine-2,4-diamine Derivatives as Oral Available and Dual CDK 6 and 9 Inhibitors with Potent Antitumor Activity" /></meta><meta name="dc.Creator" content="Yang  Wang" /></meta><meta name="dc.Creator" content="Xing  Chen" /></meta><meta name="dc.Creator" content="Yaoyao  Yan" /></meta><meta name="dc.Creator" content="Xiaochen  Zhu" /></meta><meta name="dc.Creator" content="Mingming  Liu" /></meta><meta name="dc.Creator" content="Xinhua  Liu" /></meta><meta name="dc.Description" content="Cyclin-dependent kinases (CDKs) are promising therapeutic targets for cancer therapy. Herein, we describe our efforts toward the discovery of a series of 5-chloro-N4-phenyl-N2-(pyridin-2-yl)pyrimid..." /></meta><meta name="Description" content="Cyclin-dependent kinases (CDKs) are promising therapeutic targets for cancer therapy. Herein, we describe our efforts toward the discovery of a series of 5-chloro-N4-phenyl-N2-(pyridin-2-yl)pyrimid..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="March 4, 2020" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.9b02121" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright Â© 2020 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b02121" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.9b02121" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b02121" /></link>
        
    
    

<title>Discovery and SARs of 5-Chloro-N4-phenyl-N2-(pyridin-2-yl)pyrimidine-2,4-diamine Derivatives as Oral Available and Dual CDK 6 and 9 Inhibitors with Potent Antitumor Activity | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.9b02121" /></meta><meta property="og:title" content="Discovery and SARs of 5-Chloro-N4-phenyl-N2-(pyridin-2-yl)pyrimidine-2,4-diamine Derivatives as Oral Available and Dual CDK 6 and 9 Inhibitors with Potent Antitumor Activity" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02121/20200323/images/large/jm9b02121_0021.jpeg" /></meta><meta property="og:description" content="Cyclin-dependent kinases (CDKs) are promising therapeutic targets for cancer therapy. Herein, we describe our efforts toward the discovery of a series of 5-chloro-N4-phenyl-N2-(pyridin-2-yl)pyrimidine-2,4-diamine derivatives as dual CDK6 and 9 inhibitors. Intensive structural modifications lead to the identification of compound 66 as the most active dual CDK6/9 inhibitor with balancing potency against these two targets and good selectivity over CDK2. Further biological studies revealed that compound 66 was directly bound to CDK6/9, resulting in suppression of their downstream signaling pathway and inhibition of cell proliferation by blocking cell cycle progression and inducing cellular apoptosis. More importantly, compound 66 significantly inhibited tumor growth in a xenograft mouse model with no obvious toxicity, indicating the promising therapeutic potential of CDK6/9 dual inhibitors for cancer treatment. Therefore, the above results are of great importance in the development of dual CDK6/9 inhibitors for cancer therapy." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.9b02121"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b02121">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=âtrueâ></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=âtrueâ></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=âtrueâ></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=âtrueâ></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.9b02121&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.9b02121&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.9b02121&amp;href=/doi/10.1021/acs.jmedchem.9b02121" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2020</span><span class="cit-fg-volume">, 63</span><span class="cit-fg-issue">, 6</span><span class="cit-fg-pageRange">, 3327-3347</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/63/6" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.9b02090" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.9b02123" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Discovery and SARs of 5-Chloro-<i>N</i><sup>4</sup>-phenyl-<i>N</i><sup>2</sup>-(pyridin-2-yl)pyrimidine-2,4-diamine Derivatives as Oral Available and Dual CDK 6 and 9 Inhibitors with Potent Antitumor Activity</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Yang Wang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yang Wang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Pharmacy, Anhui Medical University, Hefei 230032, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yang++Wang">Yang Wang</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Xing Chen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xing Chen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Pharmacy, Anhui Medical University, Hefei 230032, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xing++Chen">Xing Chen</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Yaoyao Yan</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yaoyao Yan</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Pharmacy, Anhui Medical University, Hefei 230032, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yaoyao++Yan">Yaoyao Yan</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Xiaochen Zhu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xiaochen Zhu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Pharmacy, Anhui Medical University, Hefei 230032, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xiaochen++Zhu">Xiaochen Zhu</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Mingming Liu</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Mingming Liu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Pharmacy, Anhui Medical University, Hefei 230032, China</div><div class="loa-info-affiliations-info">Anhui Chem-Bright Bioengineering Co., Ltd., Huaibei 235025, China</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#83efeeeec3e2ebeef6ade6e7f6ade0ed"><span class="__cf_email__" data-cfemail="bbd7d6d6fbdad3d6ce95dedfce95d8d5">[emailÂ protected]</span></a>. Phone: +551-65172132 (M.L.).</span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Mingming++Liu">Mingming Liu</a></span></div></div></span></li><span class="comma-separator">,Â andÂ </span><li><span><span class="hlFld-ContribAuthor">Xinhua Liu</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xinhua Liu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Pharmacy, Anhui Medical University, Hefei 230032, China</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#18607074716d706058292e2b367b7775"><span class="__cf_email__" data-cfemail="69110105001c011129585f5a470a0604">[emailÂ protected]</span></a>. Phone/Fax: +86-551-65172353 (X.L.).</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xinhua++Liu">Xinhua Liu</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-7984-2977" title="Orcid link">http://orcid.org/0000-0002-7984-2977</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b02121&amp;href=/doi/10.1021%2Facs.jmedchem.9b02121" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2020</span></span><span class="cit-volume">, 63</span><span class="cit-issue">, 6</span><span class="cit-pageRange">, 3327â3347</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">March 4, 2020</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>20 December 2019</li><li><span class="item_label"><b>Published</b> online</span>4 March 2020</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 26 March 2020</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.9b02121" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b02121</a></div><div class="article_header-article-copyright"><strong>Copyright Â© 2020 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D3327%26pageCount%3D21%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DYang%2BWang%252C%2BXing%2BChen%252C%2BYaoyao%2BYan%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D63%26issueNum%3D6%26contentID%3Dacs.jmedchem.9b02121%26title%3DDiscovery%2Band%2BSARs%2Bof%2B5-Chloro-N4-phenyl-N2-%2528pyridin-2-yl%2529pyrimidine-2%252C4-diamine%2BDerivatives%2Bas%2BOral%2BAvailable%2Band%2BDual%2BCDK%2B6%2Band%2B9%2BInhibitors%2Bwith%2BPotent%2BAntitumor%2BActivity%26numPages%3D21%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D3347%26publicationDate%3DMarch%2B2020">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.9b02121"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">2056</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">3</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.9b02121" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Discovery and SARs of 5-Chloro-N4-phenyl-N2-(pyridin-2-yl)pyrimidine-2,4-diamine Derivatives as Oral Available and Dual CDK 6 and 9 Inhibitors with Potent Antitumor Activity&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Yang&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;Xing&quot;,&quot;last_name&quot;:&quot;Chen&quot;},{&quot;first_name&quot;:&quot;Yaoyao&quot;,&quot;last_name&quot;:&quot;Yan&quot;},{&quot;first_name&quot;:&quot;Xiaochen&quot;,&quot;last_name&quot;:&quot;Zhu&quot;},{&quot;first_name&quot;:&quot;Mingming&quot;,&quot;last_name&quot;:&quot;Liu&quot;},{&quot;first_name&quot;:&quot;Xinhua&quot;,&quot;last_name&quot;:&quot;Liu&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2020&quot;,&quot;month&quot;:&quot;03&quot;,&quot;day&quot;:&quot;04&quot;,&quot;issue&quot;:&quot;6&quot;,&quot;volume&quot;:&quot;63&quot;,&quot;pages&quot;:&quot;3327-3347&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.9b02121&quot;},&quot;abstract&quot;:&quot;Cyclin-dependent kinases (CDKs) are promising therapeutic targets for cancer therapy. Herein, we describe our efforts toward the discovery of a series of 5-chloro-N4-phenyl-N2-(pyridin-2-yl)pyrimidine-2,4-diamine derivatives as dual CDK6 and 9 inhibitors. Intensive structural modifications lead to the identification of compound 66 as the most active dual CDK6/9 inhibitor with balancing potency against these two targets and good selectivity over CDK2. Further biological studies revealed that compound 66 was directly bound to CDK6/9, resulting in suppression of their downstream signaling pathway and inhibition of cell proliferation by blocking cell cycle progression and inducing cellular apoptosis. More importantly, compound 66 significantly inhibited tumor growth in a xenograft mouse model with no obvious toxicity, indicating the promising therapeutic potential of CDK6/9 dual inhibitors for cancer treatment. Therefore, the above results are of great importance in the development of dual CDK6/9 inhibitors for c&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b02121&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b02121" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b02121&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b02121" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b02121&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b02121" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b02121&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b02121&amp;href=/doi/10.1021/acs.jmedchem.9b02121" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.9b02121" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.9b02121" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (8 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b02121&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.9b02121%26sid%3Dliteratum%253Aachs%26pmid%3D32129996%26genre%3Darticle%26aulast%3DWang%26date%3D2020%26atitle%3DDiscovery%2Band%2BSARs%2Bof%2B5-Chloro-N4-phenyl-N2-%2528pyridin-2-yl%2529pyrimidine-2%252C4-diamine%2BDerivatives%2Bas%2BOral%2BAvailable%2Band%2BDual%2BCDK%2B6%2Band%2B9%2BInhibitors%2Bwith%2BPotent%2BAntitumor%2BActivity%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D63%26issue%3D6%26spage%3D3327%26epage%3D3347%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">Â»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a>,</li><li><a href="/action/doSearch?ConceptID=291149" title="Apoptosis">Apoptosis</a>,</li><li><a href="/action/doSearch?ConceptID=290703" title="Pyridines">Pyridines</a>,</li><li><a href="/action/doSearch?ConceptID=291305" title="Cell physiology">Cell physiology</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/63/6" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/jmcmar.2020.63.issue-6/20200326/jmcmar.2020.63.issue-6.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02121/20200323/images/medium/jm9b02121_0021.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02121/20200323/images/large/jm9b02121_0021.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02121&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Cyclin-dependent kinases (CDKs) are promising therapeutic targets for cancer therapy. Herein, we describe our efforts toward the discovery of a series of 5-chloro-<i>N</i><sup>4</sup>-phenyl-<i>N</i><sup>2</sup>-(pyridin-2-yl)pyrimidine-2,4-diamine derivatives as dual CDK6 and 9 inhibitors. Intensive structural modifications lead to the identification of compound <b>66</b> as the most active dual CDK6/9 inhibitor with balancing potency against these two targets and good selectivity over CDK2. Further biological studies revealed that compound <b>66</b> was directly bound to CDK6/9, resulting in suppression of their downstream signaling pathway and inhibition of cell proliferation by blocking cell cycle progression and inducing cellular apoptosis. More importantly, compound <b>66</b> significantly inhibited tumor growth in a xenograft mouse model with no obvious toxicity, indicating the promising therapeutic potential of CDK6/9 dual inhibitors for cancer treatment. Therefore, the above results are of great importance in the development of dual CDK6/9 inhibitors for cancer therapy.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">1.  Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_61380" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_61380" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Cyclin-dependent kinases (CDKs) belong to a class of serine/threonine protein kinases that act synergistically with specific cyclins that control cell cycle progression, transcription, DNA repair, epigenetic regulation, and other important processes involved in proliferation, differentiation, and apoptosis.<a onclick="showRef(event, 'ref1 ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3">(1â3)</a> Deregulation of cell cycle progression is one of the hallmarks of cancer, and inappropriate activation and/or abnormal expression of CDKs were observed in a majority of human cancers. There is considerable evidence showing that CDK4/6 and cyclin D are hyperactivated or amplified in a wide spectrum of tumors, including breast carcinoma, melanomas, squamous cell carcinomas, and leukemias.<a onclick="showRef(event, 'ref4 ref5'); return false;" href="javascript:void(0);" class="ref ref4 ref5">(4,5)</a> Deregulation of the CDK4/6-cyclin D pathway promotes cell cycle progression by sequentially phosphorylating the retinoblastoma protein and facilitating the G1-S phase transition. Therefore, inhibition of CDK activity by synthetic small molecular inhibitors has emerged as a logical approach in the development of novel cancer therapies.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a></div><div class="NLM_p">A wide range of small-molecule CDK inhibitors have been synthesized and studied<a onclick="showRef(event, 'ref7 ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref7 ref8 ref9">(7â9)</a> (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>). Among them, palbociclib<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> (Pfizer), the highly selective CDK4/6 inhibitor, has been approved for the treatment of ER positive/HER2 negative breast cancer as the first-line medicine in February 2015, and it is currently prescribed as a combination therapy together with anti-estrogen agents. Followed closely, ribocilib<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> (Novartis) and abemaciclib<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> (Eli Lilly) as CDK4/6 inhibitors have also been approved by the FDA for breast cancer therapy. Although these have achieved good response,<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> drug resistance often occurs after a short period of medication, and severe adverse reactions such as aneutropenia, anemia, fatigue, and diarrhea constantly appear.<a onclick="showRef(event, 'ref14 ref15'); return false;" href="javascript:void(0);" class="ref ref14 ref15">(14,15)</a> Several studies have indicated that resistance of tumor cells to CDK4/6 inhibitors is not only due to the mutations at the binding site, resulting in the loss of inhibitor binding affinity, but also a response for mutations at downstream factors such as in the Rb-E2F signaling pathway.<a onclick="showRef(event, 'ref16 ref17'); return false;" href="javascript:void(0);" class="ref ref16 ref17">(16,17)</a> Therefore, development of resistance is normally unavoidable for CDK4/6 inhibitors, and this is also true for other therapeutic agents targeting one single target. Discovery of multitargeting small molecules has emerged as a promising strategy to overcome resistance to signal-targeting therapeutics.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02121/20200323/images/medium/jm9b02121_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02121/20200323/images/large/jm9b02121_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Structures of representative selective CDK inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02121/20200323/images/large/jm9b02121_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02121&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">With the elucidation of the mechanism of cycle-dependent kinases and the increasing number of co-crystal structures, the research of multitarget inhibitors has continued to expand.<a onclick="showRef(event, 'ref18 ref19 ref20 ref21 ref22'); return false;" href="javascript:void(0);" class="ref ref18 ref19 ref20 ref21 ref22">(18â22)</a> Among them, co-targeting cycle-related kinases and noncycle-related kinases have achieved encouraging results, including simultaneously targeting CDK1/5,<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> CDK2/5,<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> CDK1/2/5,<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> CDK2/9,<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> CDK1/2/9,<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> and CDK4/9.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> These studies tend to achieve better results than single-target inhibitors. By analyzing the characteristics of these and earlier discovered inhibitors, the researchers have gained more information for further development such as potent inhibitors of CDK1/2 often lead to increased toxicity, and some CDK2 inhibitors will act on unexpected targets such as CDK5 and GSK3, which will increase the complexity of inhibitor development and adverse drugability.<a onclick="showRef(event, 'ref10 ref29'); return false;" href="javascript:void(0);" class="ref ref10 ref29">(10,29)</a> Recently, Del Re et al. reported that CDK9 overexpression in patients with metastatic breast cancer is related to the clinical resistance of CDK4/6 inhibitors<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> and displays reduced sensitivities to anti-estrogen therapies.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> Lee and Zeidner also reported that CDK9 and CDK6 affect critical processes involved in tumorigenesis, maintaining immaturity, proliferation, and potential resistance pathways to currently available therapies in AML. Inhibiting CDK9 is now a novel approach to treat AML in target therapy combinations.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> Collectively, CDK9 has been widely recognized as a potential target for antitumors,<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> and its overexpression has been firmly confirmed to contribute the development of resistance to CDK4/6 inhibitors. Therefore, we reasoned that small-molecule inhibitors simultaneously targeting both CDK6 and 9 would not only result in synergistic/additive anticancer activity but also circumvent the resistance against CDK4/6 inhibitors. Aligned with this need, we attempted to design and synthesize selective CDK6/9 dual inhibitors, which can inhibit the cell cycle process and transcription activity at the same time, so as to improve the efficacy, reduce the side effects, and delay the appearance of drug resistance.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">2.  Results and Discussions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_20008" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_20008" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4">2.1.  Design</h3><div class="NLM_p">In order to discover potent dual CDK6 and 9 inhibitors, we screened our in-house compounds library by enzymatic activity assay. From the screen, a pyrimidine diamine derivative (compound <b>8</b>) was identified as a preliminary hit with moderate inhibitory effects on CDK6 and 9 but more effective potency on CDK2. Cell proliferation assay showed that compound <b>8</b> could suppress cell growth in a variety of tumor cell lines with IC<sub>50</sub> ranging from 38.6 to 56.2 Î¼M. Starting from compound <b>8</b>, with the aim to improve and balance potency on CDK6/9 and to elevate selectivity over CDK2, a small molecule library containing 59 compounds were designed and synthesized based on receptor structural-based drug design and in-depth structure activity relationships (SARs) studies.</div><div class="NLM_p">Based on the co-crystal complex structures of CDK6-palbociclib (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5L2I">5L2I</a>)<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A) and CDK9-A86 (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6GZH">6GZH</a>)<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B), the binding mode of compound <b>8</b> to CDK6 and 9 was established by ZDOCK module in Discovery Studio 2017 R2. From the docking models, key interactions between compound <b>8</b> and CDK6 (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>C) or CDK9 (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>D) were observed similar to palbociclib or <b>A86</b>, including two hydrogen bonds between 2-amino pyrimidine and key residues (Val101<sup>CDK6</sup>/Cys106<sup>CDK9</sup>) located at the hinge region of CDK6 or 9. The <i>o</i>-methyl phenyl substituent of compound <b>8</b>, similar to the cyclopentyl of palbociclib and the cyclopropyl of A89, occupied the hydrophobic pockets formed by Val27, Ile19, and Asn150 of CDK6 (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>E) or by Val33, Ile25, and Phe30 of CDK9 (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>F). However, when compared with palbociclib, compound <b>8</b> did not interact with the hydrophilic region of CDK6, in which hydrophilic residues Asp163 and Lys43 formed hydrogen bonds with the acetyl group of palbociclib. CDK9 also possesses the hydrophilic region formed by Asp167 and Lys48. However, to our knowledge, few CDK9 inhibitors have been found to occupy this region. In addition, the piperazine ring of palbociclib was located at the solvent exposure region but was identified to be critical for the inhibition of CDK4/6 (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>).</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02121/20200323/images/medium/jm9b02121_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02121/20200323/images/large/jm9b02121_0003.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (A) Interaction regions between CDK6 inhibitors represented by palbociclib and CDK6 (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5L2I">5L2I</a>); (B) CDK9 inhibitors represented by <b>A86</b> and CDK9 (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6GZH">6GZH</a>); docking model of compound <b>8</b> and CDK6 (C) or CDK9 (D); 2D histogram of docking model of compound <b>8</b> and CDK6 (E) or CDK9 (F) to indicate the active and potential sites of modifications of compound <b>8</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02121/20200323/images/large/jm9b02121_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02121&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Based on the information observed from the docking model, modifications and SAR studies on hit <b>8</b> were carried out in a step-by-step manner. First, the pyridine group (region B) was replaced by phenyl, benzyl, phenylethyl, or other heterocycles, resulting in compound <b>16</b> with improved activity (step 1), which indicates the essential role of pyridine for potency. To simulate the piperazine ring of palbociclib, we tried to introduce various aliphatic heterocyclic structures at the end of the pyridine ring in compound <b>16</b>, leading to the discovery of <i>N</i>-methylpiperazine derivatives <b>23</b> with much improvement on activity (step 2). Next, the substituents and length between phenyl and pyrimidine (region A) were modified, but no compound achieved improvement activity unfortunately. Any change in the distance between the pyrimidine and phenyl is not conducive. So, we identified the type of structure represented by compound <b>34</b> (step 3). To simulate cyclopentyl of palbociclib occupying the hydrophobic pocket of CDK6, we examined the position and volume of hydrophobic substituents on the phenyl ring. However, introduction of bulk hydrophobic groups can only increase the inhibitory effect on CDK6 but diminish the effect on CDK9, probably due to the small size hydrophobic pocket of CDK9. Therefore, considering the balancing effects between two targets, the acetylamino group was introduced to interact with the key residue located in the hydrophilic region, including Lys43<sup>CDK6</sup>/Lys48<sup>CDK9</sup> and Asp163<sup>CDK6</sup>/Asp167<sup>CDK9</sup>, resulting in the discovery of compound <b>66</b> with almost 100-fold (CDK6) and 170-fold (CDK9) increasing potency than that of hit <b>8</b> (step 4). More importantly, compared with palbociclib and abemaciclib, compound <b>66</b> exhibited balancing potency against CDK6 and 9 with almost equal IC<sub>50</sub> values and good selectively over CDK2 (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>).</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02121/20200323/images/medium/jm9b02121_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02121/20200323/images/large/jm9b02121_0004.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Step by step structural optimization on hit compound <b>8</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02121/20200323/images/large/jm9b02121_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02121&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5">2.2.  Synthesis</h3><div class="NLM_p">As depicted in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>, depending on the reactivity of 2-Cl and 4-Cl on the pyrimidine ring, intermediates <b>3aâ3ai</b> were obtained by the reaction of compound 2,4,5 trichloropyrimidine (<b>1</b>) with different amine <b>R</b><sup><b>1</b></sup>-NH<sub>2</sub> (compounds <b>2aâ2ai</b>) in the presence of TBAI and triethylamine. 1-Bromo-4-nitrobenzene (<b>4a</b>) or 5-bromo-2-nitropyridine (<b>4b</b>) is dissolved in dimethyl sulfoxide (DMSO) and reacted with heterocyclic amine (compounds <b>5aâ5f</b>) at 90 Â°C in the presence of potassium carbonate and TBAI to give nitro compounds (<b>6pâ6z</b>). Hydrogenation of the nitro compounds (<b>6pâ6z</b>) affords the amine (compounds <b>7pâ7z</b>). Finally, the targeted compound <b>8â33</b> and <b>34â67</b> were obtained by BuchwaldâHartwig reaction between compounds <b>3ai</b> and <b>7aâ7z</b> as well as between <b>3aâ3ai</b> and <b>7p</b>, catalyzed by Pd<sub>2</sub>(dba)<sub>3</sub> and phosphorous ligands (BINAP or XANTPHOS).</div><figure id="sch1" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02121/20200323/images/medium/jm9b02121_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02121/20200323/images/large/jm9b02121_0011.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Compounds <b>8â67</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02121/20200323/images/large/jm9b02121_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02121&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) 2,4,5-trichloropyrimidine, aubstituted aniline or substituted benzylamine or substituted phenethylamine, TBAI, triethylamine, DMSO, r.t. (b) 5-bromo-2-nitropyridine or 1-bromo-4-nitrobenzene, azaheterocyclic amine, potassium carbonate, DMSO, 70 Â°C. (c) Pd/C 10%, H<sub>2</sub>. (d) Compounds <b>3aâ3ai</b>, <b>7aâ7z</b>, Pd<sub>2</sub>(dba)<sub>3</sub>, BINAP, Na<i>O</i>tBu, dioxane, 110 Â°C, Ar<sub>2</sub> atmosphere. (e) Compounds <b>3aâ3ai</b>, <b>7aâ7z</b>, Pd<sub>2</sub>(dba)<sub>3</sub>, XANTPHOS, Cs<sub>2</sub>CO<sub>3</sub>, dioxane, 110 Â°C, Ar<sub>2</sub> atmosphere.</p></p></figure></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7">2.3.  SAR Studies</h3><div class="NLM_p">Biological evaluation of CDK2, 6, and 9 inhibitory activities was conducted using ADP-Glo kinase assay. The antiproliferative activities of targeted compounds were also determined in several human cancer cell lines, including MCF-7, MDA-MB-231, and Jurkat. Meanwhile, the inhibition on normal human hepatocytes L02 growth was measured in order to assess the selectivity for malignant over normal cells. Palbociclib, NVP-2, and abemaciclib served as the positive controls.</div><div class="NLM_p">In order to examine the effect of the pyridine ring of hit <b>8</b> on inhibitory activities, pyridine was replaced with the aliphatic ring, benzene, and furan, resulting in the synthesis of compounds <b>9â15</b>. As listed in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, these synthesized compounds exhibited much weaker inhibitory activities against both CDK6 and 9 than hit <b>8</b>. It is clear that the pyridine ring is critical for inhibitory activity on both enzyme and cell proliferation. Based on this finding, pyridine was retained in the following structural modification.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. SAR at Position 2 of the Pyrimidine Scaffold</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02121/20200323/images/medium/jm9b02121_0012.gif" alt="" id="GRAPHIC-d7e655-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02121/20200323/images/medium/jm9b02121_0013.gif" alt="" id="gr12" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">Inhibition of cell-free kinases activities.</p></div><div class="footnote" id="t1fn2"><sup><sup>b</sup></sup><p class="last">Inhibition of cell proliferation.</p></div><div class="footnote" id="t1fn3"><sup><sup>c</sup></sup><p class="last">Not determined.</p></div></div><div></div></div><div class="NLM_p">Next, as listed in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>, compounds <b>16â22</b> were synthesized to explore the effects of the N position, distance between the pyridine and pyrimidine, and substitution of the pyridine ring on inhibitory activities. Extension of the distance between the pyridine and pyrimidine significantly decreased the inhibitory activities on both enzyme and cell proliferation (compounds <b>20â22</b>). Introduction of substituents including Cl and methoxyl group (<b>17</b> and <b>19</b>) failed to increase activities. Comparing potencies among 2-, 3-, and 4-pyridyl derivatives (compounds <b>16</b>, <b>8</b>, and <b>18</b>), we found that 2-pyridyl derivative <b>16</b> possessed increased potency against CDK 6 but decreased inhibition against CDK2. Although CDK9 inhibition of compound <b>16</b> was slightly decreased, considering the improvement of activities on cell proliferative inhibition, 2-pyridyl substitution was fixed in the following modification.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. SAR at Position 2 of the Pyrimidine Scaffold</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02121/20200323/images/medium/jm9b02121_0014.gif" alt="" id="GRAPHIC-d7e713-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02121/20200323/images/medium/jm9b02121_0015.gif" alt="" id="gr13" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">Inhibition of cell-free kinases activities.</p></div><div class="footnote" id="t2fn2"><sup><sup>b</sup></sup><p class="last">Inhibition of cell proliferation.</p></div></div><div></div></div><div class="NLM_p">Previous studies have highlighted the critical role of piperazine groups for CDK6 inhibitory activities of palbociclib and abemaciclib. Considering the structural similarity between palbociclib and compound <b>16</b>, we tried to introduce an aliphatic heterocyclic ring onto the pyridine of compound <b>16</b> in order to improve CDK6 inhibitory activity. By introducing <i>N</i>-methylpiperazine, <i>N</i>-ethylpiperazine, piperazine, morpholine, and piperidine, we obtained compounds <b>23â27</b>. For comparison, we also replaced the pyridine ring with the phenyl ring to obtain corresponding diphenylprimidine diamine analogues <b>28â33</b>. In general, majority of the derivatives with aliphatic heterocyclic ring substituents (compounds <b>23</b>, <b>25â26</b>, and <b>28â31</b>) exhibited significantly increasing effects on both CDK6/9 and cell proliferative inhibition. However, phenyl-substituted pyridine analogues showed higher cytotoxicity against L02 normal cells and lost the selectivity for CDK6/9 over CDK2, indicating that pyridine might be critical for the selectivity. Although piperazine and morpholine-substituted pyridyl pyrimidine analogues (<b>25â26</b>) possessed potent inhibitory activities on CDK6/9, cytotoxicity of these two compounds were also obvious. In particular, <i>N</i>-Methylpiperazin-substituted pyridine analogue (<b>23</b>) exhibited the highest potency on CDK6 inhibition, good selectivity for CDK6/9 over CDK2, and moderate effects on cell proliferation; however, its inhibitory activities were almost 10-fold difference between CDK6 and 9. Therefore, we next kept <i>N</i>-methylpiperazin-substituted pyridine unchanged and focused on balancing the potency on CDK6 and 9 inhibitions in the following modification (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>).</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. SAR at Position 4 of the Benzene or Pyridine Ring Scaffold</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02121/20200323/images/medium/jm9b02121_0016.gif" alt="" id="GRAPHIC-d7e779-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02121/20200323/images/medium/jm9b02121_0017.gif" alt="" id="gr14" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">Inhibition of cell-free kinase activities.</p></div><div class="footnote" id="t3fn2"><sup><sup>b</sup></sup><p class="last">Inhibition of cell proliferation.</p></div></div><div></div></div><div class="NLM_p">Docking models of hit <b>8</b> with CDK6/9 revealed that 2-methyl phenyl occupied hydrophobic pockets formed by Val27, Ile19, and Asn150 of CDK6 as well as by Val33, Ile25, and Phe30 of CDK9. Based on the similarity between hydrophobic pockets of CDK6 and 9, we believed that modification of 2-methyl phenyl might be critical for balancing inhibitory potency on these two kinases. Therefore, we first examined the effects of distance between the phenyl ring and pyrimidine on the inhibitory activity. As listed in <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>, compared with compound <b>23</b>, deletion of the methyl group from the phenyl ring (compound <b>34</b>) or replacement of 2-methyl phenyl with substituted or nonsubstituted benzyl (compounds <b>35â41</b>) or phenethyl (compounds <b>42â45</b>) dramatically diminished the inhibitory effects on both enzyme activities and cell proliferation, especially on CDK9, possible because the hydrophobic pockets of CDK9 were much smaller than that of CDK6. Next, the favorite position and size of substituents on the phenyl ring for occupancy of the hydrophobic pockets of CDK6 and 9 were screened, resulting in the synthesis of methyl (compounds <b>47â48</b>)-, ethyl (compound <b>49</b>)-, <i>i</i>-propyl (compounds <b>50â52</b>)-, <i>t</i>-butyl (compound <b>53</b>)-, trimethyl (compound <b>54</b>)-, and methoxyl (compounds <b>55â57</b>)-substituted derivatives. Biological evaluation indicated that the inhibitory activities of these targeted compounds were decreased more or less, when compared with compound <b>23</b>. Especially para-bulk group-substituted derivatives (compounds <b>49</b>, <b>50</b>, and <b>53</b>) lost the potency on CDK9. Next, we introduced F (compounds <b>58â60</b>), trifluoromethyl (compounds <b>61</b> and <b>64</b>), or trifluoromethoxyl (compounds <b>62â63</b>) as hydrophobic groups to suit the hydrophobic pockets. Unfortunately, these derivatives containing F did not exhibit enough potency, especially on CDK9. Among them, the 2-F-substituted phenyl derivative (compound <b>60</b>) showed effective cell proliferative inhibition and good selectively for malignant cells over normal cells; however its inhibitory effects on CDK6/9 were much weaker than that of compound <b>23</b>, probably representing an alternative molecular mechanism. Moreover, the acetylamino group was also introduced at the <i>para</i>-(<b>65</b>), <i>meta</i>-(<b>66</b>), or <i>ortho</i>-(<b>67</b>) position of phenyl in order to form additional hydrogen bonds with key residues located at the hydrophilic region of CDK6/9. Biological evaluation showed that compared with compound <b>23</b>, although CDK6 inhibition was slightly decreased, the potency on CDK9 inhibition achieved nearly 4-fold elevation, leading to good balance between inhibition of CDK6 and 9 with nearly equal IC<sub>50</sub> as well as high selectivity for the inhibition of CDK6/9 over CDK2. More importantly, compound <b>66</b> exhibited considerable effects against cancer cells with IC<sub>50</sub> values ranging from 4.01 to 8.08 Î¼M but much lower cytotoxicity against L02 normal cells with an IC<sub>50</sub> value more than 100 Î¼M, representing excellent selectivity for malignant over normal cells.</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. SAR at Position 4 of the Pyrimidine Scaffold</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02121/20200323/images/medium/jm9b02121_0018.gif" alt="" id="GRAPHIC-d7e911-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02121/20200323/images/medium/jm9b02121_0019.gif" alt="" id="gr15" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup><sup>a</sup></sup><p class="last">Inhibition of cell-free kinases activities.</p></div><div class="footnote" id="t4fn2"><sup><sup>b</sup></sup><p class="last">Inhibition of cell proliferation.</p></div><div class="footnote" id="t4fn3"><sup><sup>c</sup></sup><p class="last">Not determined.</p></div></div><div></div></div><div class="NLM_p last">To investigate the selectivity among other CDKs, we tested inhibition ratios of compounds <b>66</b> against CDK3, 5, and 11 at 1 Î¼M. As listed in <a href="/doi/suppl/10.1021/acs.jmedchem.9b02121/suppl_file/jm9b02121_si_001.pdf" class="ext-link">Table S1</a>, the results showed that the inhibition ratios (at 1 Î¼M) of compound <b>66</b> against CDK3, 5, and 11 were less than 6.6%, indicating good selectivity of CDK6/9 (100%).</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12">2.4.  Cell Cycle Analysis</h3><div class="NLM_p">In order to detect the effect of compound <b>66</b> on cell cycle distribution, MDA-MB-231 cells were stained with PI probes and subjected to flow cytometry analysis. As shown in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>, treatment of MDA-MB-231 cells with lower concentrations (4 or 8 Î¼M) of compound <b>66</b> induced G0/G1 cell cycle arrests (71.85 and 77.51%, respectively), which is typical for CDK6 inhibitors, when compared with the control (57.65%). However, with the increasing concentration (12 Î¼M), the effect of compound <b>66</b> on cell cycle distribution was converted to G2/M arrest (34.27%) when compared with the control (15.90%), which is characteristic of CDK9 inhibitors. Meanwhile, the percentages of cells distributed at the S phase were decreased from 26.45 to 18.97, 14.36, and 15.23%. These data indicated that, different from that of palbociclib, compound <b>66</b> may exert dual CDK6 and CDK9 inhibitory potency.</div><figure id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02121/20200323/images/medium/jm9b02121_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02121/20200323/images/large/jm9b02121_0005.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Effects of compound <b>66</b> on cell cycle distribution. The experiment was performed independently three times, and representative results are presented.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02121/20200323/images/large/jm9b02121_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02121&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13">2.5.  Reactive Oxygen Species Assay</h3><div class="NLM_p">Previous studies have reported that inhibition of CDK6 can affect cellular glycolysis, thereby resulting in the loss of NADPH, increase of reactive oxygen species (ROS), and subsequent induction of cell death.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> Therefore, to detect the effects of compound <b>66</b> on cellular ROS, MDA-MB-231 cells were stained with 2,7-dichlorodi-hydrofluorescein diacetate (DCFH-DA) dye and analyzed by flow cytometry after treatment with indicated concentrations of compound <b>66</b>. Results from flow cytometry analysis showed that compound <b>66</b> increased the level of intracellular ROS in a dose-dependent manner (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>).</div><figure id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02121/20200323/images/medium/jm9b02121_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02121/20200323/images/large/jm9b02121_0006.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. ROS of MDA-MB-231 cells treated with compound <b>66</b> at different concentrations after 48 h. The results are given in the form of a superposition graph. The experiment was performed independently three times, and representative results are presented.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02121/20200323/images/large/jm9b02121_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02121&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14">2.6.  Apoptosis Analysis</h3><div class="NLM_p">Induction of cellular apoptosis is one of the main characteristics of antitumor agents. To determine the capacity of compound <b>66</b> in inducing cell apoptosis, MDA-MB-231 cells were double stained by annexin V-FITC and PI assay and subjected to flow cytometry analysis after treatment with indicated concentrations. As shown in <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>, the total proportion of apoptotic cells, including annexin V<sup>+</sup>/PI<sup>â</sup> (the right lower quadrant, representing early apoptosis) and annexin V<sup>+</sup>/PI<sup>+</sup> (the right upper quadrant, representing late apoptosis and necrosis) cells were dependently increased from 5.23 to 42.85%. Especially, treatment with compound <b>66</b> at 12 Î¼M dramatically induced apoptosis with the apoptotic ratio of 42.85%, which was much higher than that of palbociclib (8.69% at 20 Î¼M), indicating different mechanisms of compound <b>66</b> from typical CDK4/6 inhibitor. These results revealed that compound <b>66</b> could induce cellular apoptosis at relatively high condensations through an unique cellular mechanism,<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> probably due to the dual inhibition of CDK6 and 9.</div><figure id="fig6" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02121/20200323/images/medium/jm9b02121_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02121/20200323/images/large/jm9b02121_0007.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Effects of compound 66 on cellular apoptosis. Apoptosis of MDA-MB-231 cells treated with compound <b>66</b> at different concentrations after 48 h. Cells in Q1-UR and Q1-LR are considered to be apoptotic. The experiment was performed independently three times, and representative results are presented.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02121/20200323/images/large/jm9b02121_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02121&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15">2.7.  Cellular Thermal Shift Assay</h3><div class="NLM_p">Cellular thermal shift assay (CETSA) is based on ligand-induced thermal stabilization of target proteins.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> This experiment is likely to validate drug binding for a series of vital clinical targets, off-target effects or drug resistance in cancer cell lines. To avoid the endogenous cofactors such as ATP disturbance to the experiment, we used purified recombinant proteins in this assay. As shown in <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>, both CDK6 and 9 were stabilized in the presence of compound <b>66</b>. Especially at 57 Â°C, a majority of both CDK6 and 9 were stabilized to avoid degradation in the presence of compound <b>66</b>. In the contrast, at the same temperature, target protein was almost totally degraded in the absence of compound <b>66</b>. The results indicated that compound <b>66</b> was directly bound to CDK6 and 9.</div><figure id="fig7" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02121/20200323/images/medium/jm9b02121_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02121/20200323/images/large/jm9b02121_0008.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. CETSA assay to detection the thermal stability of recombinant CDK6 and CDK9 proteins treated with compound <b>66</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02121/20200323/images/large/jm9b02121_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02121&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16">2.8.  Microscale Thermophoresis Assay (MST)</h3><div class="NLM_p">In order to analyze the binding affinity (<i>K</i><sub>d</sub> value) between compound <b>66</b> and CDK6 or 9, GFP-fused CDK6 or 9 recombinant protein as well as their corresponding cyclin D3 or T1 was expressed in HEK293T cells and purified by affinity chromatography. A couple of CDK6 or 9 as well as their corresponding cyclin D3 or T1 was treated with indicated concentrations of compound <b>66,</b> and the binding affinity was then analyzed by microscale thermophoresis assay (MST).<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> As shown in <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>, results from MST assay showed that compound <b>66</b> exhibited high affinity to both CDK6 and 9, with a <i>K</i><sub>d</sub> value of 71 and 30 nM, respectively, which agrees well with the results of enzymatic inhibition assay.</div><figure id="fig8" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02121/20200323/images/medium/jm9b02121_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02121/20200323/images/large/jm9b02121_0009.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. MST assay to determine the affinity of compound <b>66</b> to CDK6 and CDK9 proteins. The experiment was performed independently three times, and representative results are presented.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02121/20200323/images/large/jm9b02121_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02121&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17">2.9.  Effects of Compound <b>66</b> on CDK6/9 Signaling Pathway</h3><div class="NLM_p">To further clarify effects of compound <b>66</b> on the signaling pathway of CDK6 and 9, Western blot analysis was used to detect the expression of CDK6/9 and their downstream-related proteins, after treatment with different concentrations. As shown in <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>, treatment with compound <b>66</b> did not affect the expression of CDK6 or 9 but significantly decreased CDK6 downstream targets p-Rb and E2F1, as well as CDK9 downstream effectors cyclin T1 and p-Pol II CTD (S2) expression, indicating the inhibition effects of compound <b>66</b> on CDK6 and 9 signaling pathways.</div><figure id="fig9" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02121/20200323/images/medium/jm9b02121_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02121/20200323/images/large/jm9b02121_0010.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Compound 66 inhibits the function of CDK6 and CDK9 and significantly downregulated the mRNA expression of c-MYC and Mcl-1. (A) Immunoblot analysis of the indicated proteins in MDA-MB-231 cells treated with compound <b>66</b>, and Î²-actin was used as a loading control. The experiment was performed independently three times, and representative results are presented. Phosphorylation level of Rb and RNA polymerase II CTD are presented as means Â± SEM. Significant difference is shown as *<i>p</i> < 0.05, **<i>p</i> < 0.01, ***<i>p</i> < 0.001 by one-way ANVOA. (B) qRT-RCR analysis of the mRNA expression of oncogene c-MYC and Mcl-1. The experiment was performed independent three times and presented as means Â± SEM. Significant difference is shown as *<i>p</i> < 0.05, **<i>p</i> < 0.01, ***<i>p</i> < 0.001 by one-way ANVOA.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02121/20200323/images/large/jm9b02121_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02121&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Since previous studies have indicated that CDK9 regulates transcriptional elongation of important genes involved in cell proliferation and survival, such as <i>MYC</i> and <i>MCL-1</i>.<a onclick="showRef(event, 'ref40 ref41'); return false;" href="javascript:void(0);" class="ref ref40 ref41">(40,41)</a> Thus, in our studies, the effects of compound <b>66</b> on c-Myc and Mcl-1 mRNA levels were analyzed by qualitative real-time polymerase chain reaction (qRT-PCR). As shown in <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>, treatment with various concentrations of compound <b>66</b> significantly decreased mRNA levels of both c-Myc and Mcl-1, which are well-known oncogenes with the functions for promoting proliferation and suppressing cellular apoptosis. Altogether, the results from western blotting and qRT-PCR clearly indicated that compound <b>66</b> acted as a dual CDK6/9 inhibitor and blocked both CDK6 and 9 signaling pathways.</div></div><div id="sec2_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18">2.10.  Aqueous Solubility, LipidâWater Partition Coefficient, Caco-2 Permeability Determination, and Metabolic Stability Assay in Hepatocyctes</h3><div class="NLM_p">In order to initially evaluate the drug ability of compound <b>66</b> before in vivo experiments. We tested the aqueous solubility, lipidâwater partition coefficient, and liver microsome stability of this compound. The thermodynamic equilibrium solubility of compound <b>66</b> in water and normal saline was determined as 0.340 and 0.214 mg/mL, respectively. However, the solubility of the hydrochloride of compound <b>66</b> was greatly increased to 7.70 and 3.96 mg/mL. This is very beneficial for oral administration. The experimental log <i>P</i> of compound <b>66</b> was found to be 1.23 which follows the Lipinski Rule of Five. In Caco-2 permeability experiments, compound <b>66</b> exhibited moderate permeability and no obvious efflux phenomena with the apparent permeability coefficient (<i>P</i><sub>app</sub>) value of 1.7 Ã 10 <sup>â6</sup> cm/s.</div><div class="NLM_p">We further evaluated the in vitro metabolic stability of compound <b>66</b> through both human and mouse hepatocyte metabolism. As listed in <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>, in general, compound <b>66</b> showed acceptable stability against metabolism of both human and mouse hepatocytes. It was more stable in human hepatocyte metabolism than in mouse, with half-life (<i>t</i><sub>1/2</sub>) of 91.2 and 31.2 min, respectively. In addition, we also detected the effects of compound <b>66</b> on the activity of CYPs. The results showed that compound <b>66</b> exhibited nonsignificant effects on 1A2, 2C9, 3C19, or 2D6 isoforms of CYP with IC<sub>50</sub> values more than 50 Î¼M and weak inhibition on 3A4 isoform with IC<sub>50</sub> values of 31.4 Î¼M, indicating low potential on drug interactions.</div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. Metabolic Stability in Hepatocytes</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">Â </th><th class="rowsep1 colsep0" colspan="3" align="center">human</th><th class="rowsep1 colsep0" colspan="3" align="center">mouse</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center"><i>t</i><sub>1/2</sub> (min)</th><th class="colsep0 rowsep0" align="center">CL (Î¼L/min/millionÂ cells)</th><th class="colsep0 rowsep0" align="center"><a href="/cdn-cgi/l/email-protection" class="__cf_email__" data-cfemail="8cfee9e1ede5e2e5e2ebccbdbebc">[emailÂ protected]</a>Â min (%)</th><th class="colsep0 rowsep0" align="center"><i>t</i><sub>1/2</sub> (min)</th><th class="colsep0 rowsep0" align="center">CL (Î¼L/min/millionÂ cells)</th><th class="colsep0 rowsep0" align="center"><a href="/cdn-cgi/l/email-protection" class="__cf_email__" data-cfemail="196b7c7478707770777e59282b29">[emailÂ protected]</a>Â min (%)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>66</b></td><td class="colsep0 rowsep0" align="left">91.9</td><td class="colsep0 rowsep0" align="left">7.5</td><td class="colsep0 rowsep0" align="left">40.4</td><td class="colsep0 rowsep0" align="left">39.2</td><td class="colsep0 rowsep0" align="left">17.7</td><td class="colsep0 rowsep0" align="left">12.0</td></tr></tbody></table></div></div></div><div id="sec2_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20">2.11.  In Vivo PK Study</h3><div class="NLM_p">On the basis of biochemical and cellular activities, compound <b>66</b> was further evaluated for in vivo pharmacokinetic (PK) properties via iv and po route. As summarized in <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a></a>, compound <b>66</b> showed moderate PK properties. After iv administration of 2 mg/kg compound <b>66</b>, the area under the concentrationâtime curve (AUC<sub>0ââ</sub>) was about 640.8 Î¼g/L h, and the half-life (<i>t</i><sub>1/2</sub>) was 1.76 h. In addition, via the oral administration (10 mg/kg), compound <b>66</b> showed an area under the concentrationâtime curve (AUC<sub>0ââ</sub>) of 925.7 Î¼g/L Ã <i>h</i>, a half-life (<i>t</i><sub>1/2</sub>) of 3.24 h, and a maximum plasma concentration (<i>C</i><sub>max</sub>) of 100.6 Î¼g/L. The bioavailability (<i>F</i>%) was about 28.9%, indicating orally available property of compound <b>66</b>.</div><div class="NLM_table-wrap" id="tbl6"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 6. In Vivo PK Properties of Compound <b>66</b></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">dose/routes</th><th class="colsep0 rowsep0" align="center" char="."><i>t</i><sub>1/2</sub> (h)</th><th class="colsep0 rowsep0" align="center" char="."><i>T</i><sub>max</sub> (h)</th><th class="colsep0 rowsep0" align="center" char="."><i>C</i><sub>max</sub> (Î¼g/L)</th><th class="colsep0 rowsep0" align="center" char=".">AUC<sub>0ââ</sub> (Î¼g/LÂ ÃÂ <i>h</i>)</th><th class="colsep0 rowsep0" align="center" char=".">CL (L/h/kg)</th><th class="colsep0 rowsep0" align="center" char="."><i>F</i>Â (%)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">2Â mg/kg (iv)</td><td class="colsep0 rowsep0" align="char" char=".">1.76</td><td class="colsep0 rowsep0" align="char" char=".">Â </td><td class="colsep0 rowsep0" align="char" char=".">994.7</td><td class="colsep0 rowsep0" align="char" char=".">640.8</td><td class="colsep0 rowsep0" align="char" char=".">3.13</td><td class="colsep0 rowsep0" align="char" char=".">Â </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">10Â mg/kg (po)</td><td class="colsep0 rowsep0" align="char" char=".">3.24</td><td class="colsep0 rowsep0" align="char" char=".">4.0</td><td class="colsep0 rowsep0" align="char" char=".">100.6</td><td class="colsep0 rowsep0" align="char" char=".">925.7</td><td class="colsep0 rowsep0" align="char" char=".">10.8</td><td class="colsep0 rowsep0" align="char" char=".">28.9</td></tr></tbody></table></div></div></div><div id="sec2_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22">2.12.  Safety and in Vivo Anticancer Activity</h3><div class="NLM_p">To evaluate the safety of compound <b>66</b>, ICR mice were orally administered with a single administration of this compound at 500 mg/kg. No significant weight loss and toxic reaction were observed for 7 days. Next, the in vivo antitumor activities of compound <b>66</b> was further evaluated in a human breast cancer xenograft mouse model. Approved clinical agent abemaciclib was selected as a positive control. As shown in <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>AâC, tumor growth was dose-dependently inhibited by oral administration of compound <b>66</b>, and treatment with 90 mg/kg compound displayed almost equivalent potency with 45 mg/kg abemaciclib. No significant weight loss was observed in animals treated with compound <b>66</b> (<a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>D). The pathological examination showed that this compound induced significant apoptosis in tumor tissues, which is presented as deep staining nucleoplasm, increasing intercellular space and shrinkage of the cytoplasm (<a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>E). Overall, results from in vivo studies indicated the potent antitumor and good safety of this compound.</div><figure id="fig10" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02121/20200323/images/medium/jm9b02121_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02121/20200323/images/large/jm9b02121_0002.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. In vivo antitumor activity of compound 66 in MDA-MB-231 tumor-bearing mice. (A) and (B) tumor growth curve and mice weight during the treatment. The results are presented as means Â± SEM (<i>n</i> = 6). (C,D) Final tumor size and weight after treatment with compound <b>66</b> in MDA-MB-231 tumor-bearing mice. The results are presented as mean Â± SEM (<i>n</i> = 6). Significant difference is shown as *<i>p</i> < 0.05, **<i>p</i> < 0.01, ***<i>p</i> < 0.001 by one-way ANVOA. (E) HE staining of tumor tissues after treatment. Representative results are given in the figure.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02121/20200323/images/large/jm9b02121_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02121&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i23">3.  Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_67373" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_67373" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">CDK 6 and 9 have emerged as a promising target for cancer therapy, and previous studies have indicated that suppression CDK9 might overcome the resistance to CDK6 inhibitors.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> With the aim to discover novel dual CDK6 and 9 inhibitors, a series of 5-chloro-pyrimidine-2,4-diamine derivatives were designed, synthesized, and evaluated for inhibitory potency on both kinase activity and cell proliferation. Starting from hit <b>8</b>, step-by-step lead optimization was carried out based on the structure of the ligandâreporter interaction and SAR studies, and compound <b>66</b> was discovered as the most active dual CDK6/9 inhibitor with balancing potency against two targets and selectivity over CDK2 compared with current clinical therapeutics palbociclib and abemaciclib. Further biological studies revealed that compound <b>66</b> was directly bound to CDK6/9, resulting in suppression of their downstream signaling pathway and inhibition of cell proliferation by blocking cell cycle progression and inducing cellular apoptosis. Importantly, compound <b>66</b> significantly inhibited tumor growth in xenograft mouse model with no obvious toxicity, indicating the promising therapeutic potential for cancer treatment. Therefore, the above results are of great importance in the development of balanced dual CDK6/9 inhibitors for cancer therapy.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i24">4.  Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_04781" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_04781" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25">4.1.  Chemistry Methods</h3><div class="NLM_p">All commercial reagents and solvents were purchased from commercial suppliers and used without further purification. Reactions were monitored by thin-layer chromatography (TLC) and visualized under UV light at 254 and 365 nm. The chromatograms were conducted on silica gel (300â400 mesh) using combiflash chromatography systems (Teledyne ISCO Rf200). <sup>1</sup>H and <sup>13</sup>C NMR spectral data were recorded with a Bruker 600 MHz spectrometer in CDCl<sub>3</sub> or DMSO-<i>d</i><sub>6</sub> using tetramethylsilane as the internal standard (IS). High-resolution mass spectrometry (HRMS) was recorded on an Agilent Technologies LC-TOF instrument. Targeted compounds (<b>8</b>, <b>16</b>, <b>23</b>, <b>34,</b> and <b>66</b>) were analyzed by a Shimadzu LC20A HPLC instrument with a UV/visible detector (5 Î¼m, 4.6 mm Ã 150 mm C<sub>18</sub> column). The purities of all targeted compounds were determined by monitoring at 254 nm and were confirmed to be more than 95%.</div><div id="sec4_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26">4.1.1.  General Procedure for the Synthesis of Compounds <b>3aâ3ag</b> Exemplified by 2,5-Dichloro-<i>N</i>-phenylpyrimidin-4-amine (<b>3a</b>)</h4><div class="NLM_p last">To a solution of 2,4,5-trichloropyrimidine (2.73 mmol, 0.5 g) in DMSO (5 mL), a catalytic amount of TBAI was added. After the solution changed from colorless to yellow, aniline (3.00 mmol, 0.28 g) and NEt<sub>3</sub> (3.00 mmol, 0.3 g) were added, and the resulting mixture was stirred at room temperature for 3 h. The reaction endpoint was detected by TLC (PE/EA = 20:1) and quenched by pouring into ice water (25 mL). After stirring for 30 min, the mixture was extracted with ethyl acetate (3 Ã 30 mL), and the combined ethyl acetate layer was washed twice with saturated sodium chloride solution. Combined organic phases were dried over Mg<sub>2</sub>SO<sub>4</sub> and evaporated in vacuo. The residue was purified with an automated chromatography system. Obtained as white solid; yield 89% (0.58 g). mp 83â84 Â°C; <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): Î´ 8.19 (s, 1H), 7.62 (d, <i>J</i> = 7.8 Hz, 2H), 7.41 (t, <i>J</i> = 7.9 Hz, 2H), 7.27 (s, 1H), 7.21 (t, <i>J</i> = 7.4 Hz, 1H). HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>10</sub>H<sub>8</sub>Cl<sub>2</sub>N<sub>3</sub>, 240.0090; found, 240.0086.</div></div><div id="sec4_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27">4.1.2.  <i>N</i>-Benzyl-2,5-dichloropyrimidin-4-amine (<b>3b</b>)</h4><div class="NLM_p last">White solid; yield 85%. mp 84â85 Â°C; <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): Î´ 8.06 (s, 1H), 7.40â7.32 (m, 5H), 5.79 (s, 1H), 4.72 (d, <i>J</i> = 5.6 Hz, 2H). HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>11</sub>H<sub>10</sub>Cl<sub>2</sub>N<sub>3</sub>, 254.0246; found, 240.0246.</div></div><div id="sec4_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28">4.1.3.  2,5-Dichloro-<i>N</i>-(4-(trifluoromethyl)benzyl)pyrimidin-4-amine (<b>3c</b>)</h4><div class="NLM_p last">White solid; yield 86%, mp 65â66 Â°C; <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): Î´ 8.08 (s, 1H), 7.63 (d, <i>J</i> = 8.0 Hz, 2H), 7.47 (d, <i>J</i> = 8.0 Hz, 2H), 5.92 (s, 1H), 4.80 (d, <i>J</i> = 5.9 Hz, 2H). HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>12</sub>H<sub>9</sub>Cl<sub>2</sub>F<sub>3</sub>N<sub>3</sub>, 322.0120; found, 322.0114.</div></div><div id="sec4_1_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29">4.1.4.  2,5-Dichloro-<i>N</i>-(4-fluorobenzyl)pyrimidin-4-amine (<b>3d</b>)</h4><div class="NLM_p last">White solid; yield 75%; mp 78â79 Â°C; <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): Î´ 8.06 (s, 1H), 7.37â7.29 (m, 2H), 7.10â7.01 (m, 2H), 5.78 (s, 1H), 4.68 (d, <i>J</i> = 5.7 Hz, 2H). HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>11</sub>H<sub>9</sub>Cl<sub>2</sub>FN<sub>3</sub>, 272.0152; found, 272.0149.</div></div><div id="sec4_1_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30">4.1.5.  2,5-Dichloro-<i>N</i>-(4-methylbenzyl)pyrimidin-4-amine (<b>3e</b>)</h4><div class="NLM_p last">White solid; yield 82%; mp 101â102 Â°C; <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): Î´ 8.04 (s, 1H), 7.24 (d, <i>J</i> = 8.0 Hz, 2H), 7.19 (d, <i>J</i> = 8.0 Hz, 2H), 5.84â5.58 (m, 1H), 4.66 (d, <i>J</i> = 5.5 Hz, 2H), 2.36 (s, 3H). HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>12</sub>H<sub>12</sub>Cl<sub>2</sub>N<sub>3</sub>, 268.0403; found, 268.0404.</div></div><div id="sec4_1_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31">4.1.6.  2,5-Dichloro-<i>N</i>-(4-methoxybenzyl)pyrimidin-4-amine (<b>3f</b>)</h4><div class="NLM_p last">White solid; yield 74%; mp 72â73 Â°C; <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): Î´ 8.02 (s, 1H), 7.28 (d, <i>J</i> = 8.6 Hz, 2H), 6.90 (d, <i>J</i> = 8.6 Hz, 2H), 5.74 (s, 1H), 4.63 (d, <i>J</i> = 5.5 Hz, 2H), 3.82 (s, 3H). HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>12</sub>H<sub>12</sub>Cl<sub>2</sub>N<sub>3</sub>O, 284.0352; found, 284.0345.</div></div><div id="sec4_1_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32">4.1.7.  2,5-Dichloro-<i>N</i>-(3-methoxybenzyl)pyrimidin-4-amine (<b>3g</b>)</h4><div class="NLM_p last">White solid; yield 89%; mp 61â62 Â°C; <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): Î´ 8.04 (s, 1H), 7.29 (t, <i>J</i> = 7.9 Hz, 1H), 6.92 (d, <i>J</i> = 7.6 Hz, 1H), 6.89 (d, <i>J</i> = 2.1 Hz, 1H), 6.86 (dd, <i>J</i> = 8.2, 2.2 Hz, 1H), 5.80 (s, 1H), 4.67 (d, <i>J</i> = 5.6 Hz, 1H), 3.81 (s, 1H). HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>12</sub>H<sub>12</sub>Cl<sub>2</sub>N<sub>3</sub>O, 284.0352; found, 284.0343.</div></div><div id="sec4_1_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33">4.1.8.  2,5-Dichloro-<i>N</i>-(2-methoxybenzyl)pyrimidin-4-amine (<b>3h</b>)</h4><div class="NLM_p last">White solid; yield 81%; mp 96â97 Â°C; mp 96â97 Â°C; <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): Î´ 7.98 (s, 1H), 7.34 (dd, <i>J</i> = 7.4, 1.5 Hz, 1H), 7.31 (td, <i>J</i> = 8.0, 1.7 Hz, 1H), 6.95 (td, <i>J</i> = 7.4, 0.8 Hz, 1H), 6.92 (d, <i>J</i> = 8.2 Hz, 1H), 6.17 (s, 1H), 4.70 (d, <i>J</i> = 5.9 Hz, 2H), 3.89 (s, 3H). HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>12</sub>H<sub>12</sub>Cl<sub>2</sub>N<sub>3</sub>O, 284.0352; found, 284.0344.</div></div><div id="sec4_1_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34">4.1.9.  2,5-Dichloro-<i>N</i>-phenethylpyrimidin-4-amine (<b>3i</b>)</h4><div class="NLM_p last">White solid; yield 74%; mp 91â92 Â°C; <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): Î´ 7.91 (s, 1H), 7.30 (t, <i>J</i> = 7.5 Hz, 2H), 7.23 (t, <i>J</i> = 7.4 Hz, 1H), 7.21â7.18 (m, 2H), 5.77 (t, <i>J</i> = 5.1 Hz, 1H), 3.74 (dd, <i>J</i> = 12.9, 7.0 Hz, 2H), 2.91 (t, <i>J</i> = 7.1 Hz, 2H). HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>12</sub>H<sub>12</sub>Cl<sub>2</sub>N<sub>3</sub>, 268.0403; found, 268.0402.</div></div><div id="sec4_1_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35">4.1.10.  2,5-Dichloro-<i>N</i>-(4-fluorophenethyl)pyrimidin-4-amine (<b>3j</b>)</h4><div class="NLM_p last">White solid; yield 88%; mp 101â102 Â°C; <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): Î´ 8.02 (s, 1H), 7.21â7.16 (m, 2H), 7.05â7.00 (m, 2H), 5.53 (s, 1H), 3.76 (dd, <i>J</i> = 13.0, 6.9 Hz, 2H), 2.92 (t, <i>J</i> = 7.0 Hz, 2H). HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>12</sub>H<sub>11</sub>Cl<sub>2</sub>FN<sub>3</sub>, 286.0309; found, 286.0301.</div></div><div id="sec4_1_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36">4.1.11.  2,5-Dichloro-<i>N</i>-(2-fluorophenethyl)pyrimidin-4-amine (<b>3k</b>)</h4><div class="NLM_p last">White solid; yield 86%; mp 85â86 Â°C; <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): Î´ 8.00 (s, 1H), 7.25â7.21 (m, 1H), 7.20 (dd, <i>J</i> = 7.5, 1.5 Hz, 1H), 7.10 (td, <i>J</i> = 7.5, 0.9 Hz, 1H), 7.08â7.04 (m, 1H), 5.63 (s, 1H), 3.78 (dd, <i>J</i> = 12.8, 6.7 Hz, 2H), 3.00 (t, <i>J</i> = 6.8 Hz, 2H). HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>12</sub>H<sub>11</sub>Cl<sub>2</sub>FN<sub>3</sub>, 286.0309; found, 286.0302.</div></div><div id="sec4_1_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37">4.1.12.  2,5-Dichloro-<i>N</i>-(4-methoxyphenethyl)pyrimidin-4-amine (<b>3l</b>)</h4><div class="NLM_p last">White solid; yield 75%; mp 75â76 Â°C; <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): Î´ 7.99 (s, 1H), 7.14 (d, <i>J</i> = 8.6 Hz, 2H), 6.87 (d, <i>J</i> = 8.6 Hz, 2H), 5.57 (s, 1H), 3.80 (s, 3H), 3.74 (dd, <i>J</i> = 12.7, 6.9 Hz, 2H), 2.88 (t, <i>J</i> = 6.9 Hz, 2H). HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>13</sub>H<sub>14</sub>Cl<sub>2</sub>N<sub>3</sub>O, 298.0508; found, 298.0507.</div></div><div id="sec4_1_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38">4.1.13.  2,5-Dichloro-<i>N</i>-(furan-2-ylmethyl)pyrimidin-4-amine (<b>3m</b>)</h4><div class="NLM_p last">White solid; yield 85%; mp 173â174 Â°C; <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): Î´ 8.05 (s, 1H), 7.40 (s, 1H), 6.35 (s, 1H), 6.34 (s, 1H), 5.79 (s, 1H), 4.71 (d, <i>J</i> = 5.5 Hz, 2H). HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>9</sub>H<sub>8</sub>Cl<sub>2</sub>N<sub>3</sub>O, 244.0039; found, 244.0036.</div></div><div id="sec4_1_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39">4.1.14.  2,5-Dichloro-<i>N</i>-(<i>p</i>-tolyl)pyrimidin-4-amine (<b>3n</b>)</h4><div class="NLM_p last">White solid; yield 89%; mp 93â94 Â°C; <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): Î´ 8.16 (s, 1H), 7.47 (d, <i>J</i> = 8.4 Hz, 2H), 7.22 (s, 1H), 7.19 (d, <i>J</i> = 8.4 Hz, 2H), 2.33 (s, 3H). HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>11</sub>H<sub>10</sub>Cl<sub>2</sub>N<sub>3</sub>, 254.0246; found, 254.0244.</div></div><div id="sec4_1_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40">4.1.15.  2,5-Dichloro-<i>N</i>-(<i>m</i>-tolyl)pyrimidin-4-amine (<b>3o</b>)</h4><div class="NLM_p last">White solid; yield 88%; mp 84â85 Â°C; <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): Î´ 8.16 (s, 1H), 7.48 (d, <i>J</i> = 8.0 Hz, 1H), 7.35 (s, 1H), 7.27 (t, <i>J</i> = 7.9 Hz, 1H), 7.22 (s, 1H), 6.99 (d, <i>J</i> = 7.5 Hz, 1H), 2.37 (s, 3H). HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>11</sub>H<sub>10</sub>Cl<sub>2</sub>N<sub>3</sub>, 254.0246; found, 254.0242.</div></div><div id="sec4_1_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41">4.1.16.  2,5-Dichloro-<i>N</i>-(4-ethylphenyl)pyrimidin-4-amine (<b>3p</b>)</h4><div class="NLM_p last">White solid; yield 90%; mp 91â92 Â°C; <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): Î´ 8.16 (s, 1H), 7.51 (d, <i>J</i> = 8.4 Hz, 2H), 7.22 (d, <i>J</i> = 8.3 Hz, 2H), 2.65 (q, <i>J</i> = 7.6 Hz, 2H), 1.24 (t, <i>J</i> = 7.6 Hz, 3H). HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>12</sub>H<sub>12</sub>Cl<sub>2</sub>N<sub>3</sub>, 268.0403; found, 268.0401.</div></div><div id="sec4_1_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42">4.1.17.  2,5-Dichloro-<i>N</i>-(4-isopropylphenyl)pyrimidin-4-amine (<b>3q</b>)</h4><div class="NLM_p last">White solid; yield 93%; mp 103â104 Â°C; <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): Î´ 8.16 (s, 1H), 7.52 (d, <i>J</i> = 8.5 Hz, 2H), 7.25 (d, <i>J</i> = 8.4 Hz, 2H), 7.22 (s, 1H), 2.98â2.85 (m, 1H), 1.26 (d, <i>J</i> = 6.9 Hz, 6H). HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>13</sub>H<sub>14</sub>Cl<sub>2</sub>N<sub>3</sub>, 282.0559; found, 282.0558.</div></div><div id="sec4_1_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43">4.1.18.  2,5-Dichloro-<i>N</i>-(2-isopropylphenyl)pyrimidin-4-amine (<b>3s</b>)</h4><div class="NLM_p last">White solid; yield 76%; mp 111â112 Â°C; <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): Î´ 8.19 (s, 1H), 7.65 (dd, <i>J</i> = 5.4, 3.9 Hz, 1H), 7.36 (dd, <i>J</i> = 5.4, 4.0 Hz, 1H), 7.30â7.24 (m, 2H), 7.17 (s, 1H), 3.03 (dt, <i>J</i> = 13.7, 6.9 Hz, 1H), 1.28 (d, <i>J</i> = 6.9 Hz, 6H). HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>13</sub>H<sub>14</sub>Cl<sub>2</sub>N<sub>3</sub>, 282.0559; found, 282.0560.</div></div><div id="sec4_1_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i44" class="anchor-spacer"></div><h4 class="article-section__title" id="_i44">4.1.19.  <i>N</i>-(4-(<i>tert</i>-Butyl)phenyl)-2,5-dichloropyrimidin-4-amine (<b>3t</b>)</h4><div class="NLM_p last">White solid; yield 79%; mp 153â154 Â°C; <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): Î´ 8.17 (s, 1H), 7.53 (d, <i>J</i> = 8.7 Hz, 2H), 7.42 (d, <i>J</i> = 8.7 Hz, 2H), 7.22 (s, 1H), 1.33 (s, 9H). HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>14</sub>H<sub>16</sub>Cl<sub>2</sub>N<sub>3</sub>, 296.0716; found, 296.0713.</div></div><div id="sec4_1_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i45" class="anchor-spacer"></div><h4 class="article-section__title" id="_i45">4.1.20.  2,5-Dichloro-<i>N</i>-mesitylpyrimidin-4-amine (<b>3u</b>)</h4><div class="NLM_p last">White solid; yield 69%; mp 119â120 Â°C; <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): Î´ 8.13 (s, 1H), 6.95 (s, 2H), 6.73 (s, 1H), 2.31 (s, 3H), 2.16 (s, 6H). HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>13</sub>H<sub>14</sub>Cl<sub>2</sub>N<sub>3</sub>, 282.0559; found, 282.0558.</div></div><div id="sec4_1_21" class="NLM_sec NLM_sec_level_3"><div id="ac_i46" class="anchor-spacer"></div><h4 class="article-section__title" id="_i46">4.1.21.  2,5-Dichloro-<i>N</i>-(4-methoxyphenyl)pyrimidin-4-amine (<b>3v</b>)</h4><div class="NLM_p last">White solid; yield 80%; mp 117â118 Â°C; <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): Î´ 8.15 (s, 1H), 7.48 (d, <i>J</i> = 9.0 Hz, 2H), 7.15 (s, 1H), 6.93 (d, <i>J</i> = 9.0 Hz, 2H), 3.82 (s, 3H). HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>11</sub>H<sub>10</sub>Cl<sub>2</sub>N<sub>3</sub>O, 270.0195; found, 270.0192.</div></div><div id="sec4_1_22" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47">4.1.22.  2,5-Dichloro-<i>N</i>-(3-methoxyphenyl)pyrimidin-4-amine (<b>3w</b>)</h4><div class="NLM_p last">White solid; yield 77%; mp 190â191 Â°C; <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): Î´ 8.19 (s, 1H), 7.39 (t, <i>J</i> = 2.1 Hz, 1H), 7.28 (t, <i>J</i> = 8.2 Hz, 1H), 7.25 (s, 1H), 7.10 (dd, <i>J</i> = 8.0, 1.6 Hz, 1H), 6.73 (dd, <i>J</i> = 8.3, 2.3 Hz, 1H), 3.84 (s, 3H). HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>11</sub>H<sub>10</sub>Cl<sub>2</sub>N<sub>3</sub>O, 270.0195; found, 270.0197.</div></div><div id="sec4_1_23" class="NLM_sec NLM_sec_level_3"><div id="ac_i48" class="anchor-spacer"></div><h4 class="article-section__title" id="_i48">4.1.23.  2,5-Dichloro-<i>N</i>-(2-methoxyphenyl)pyrimidin-4-amine (<b>3x</b>)</h4><div class="NLM_p last">White solid; yield 79%; mp 160â161 Â°C; <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): Î´ 8.50 (dd, <i>J</i> = 8.0, 1.5 Hz, 1H), 8.18 (s, 1H), 8.14 (s, 1H), 7.11 (td, <i>J</i> = 7.9, 1.6 Hz, 1H), 7.05 (td, <i>J</i> = 7.9, 1.2 Hz, 1H), 6.94 (dd, <i>J</i> = 8.1, 1.1 Hz, 1H), 3.95 (s, 3H). HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>11</sub>H<sub>10</sub>Cl<sub>2</sub>N<sub>3</sub>O, 270.0195; found, 270.0195.</div></div><div id="sec4_1_24" class="NLM_sec NLM_sec_level_3"><div id="ac_i49" class="anchor-spacer"></div><h4 class="article-section__title" id="_i49">4.1.24.  2,5-Dichloro-<i>N</i>-(4-fluorophenyl)pyrimidin-4-amineamine (<b>3y</b>)</h4><div class="NLM_p last">White solid; yield 65%; mp 102â103 Â°C; <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): Î´ 8.19 (s, 1H), 7.58â7.55 (m, 2H), 7.22 (s, 1H), 7.11â7.08 (m, 2H). HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>10</sub>H<sub>7</sub>Cl<sub>2</sub>FN<sub>3</sub>: 257.9996; found, 257.9992.</div></div><div id="sec4_1_25" class="NLM_sec NLM_sec_level_3"><div id="ac_i50" class="anchor-spacer"></div><h4 class="article-section__title" id="_i50">4.1.25.  2,5-Dichloro-<i>N</i>-(3-fluorophenyl)pyrimidin-4-amineamine (<b>3z</b>)</h4><div class="NLM_p last">White solid; yield 73%; mp 135â136 Â°C; <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): Î´ 8.23 (s, 1H), 7.61 (dt, <i>J</i> = 10.7, 2.2 Hz, 1H), 7.35â7.32 (m, 1H), 7.31â7.24 (m, 2H), 6.88 (td, <i>J</i> = 8.2, 2.3 Hz, 1H). HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>10</sub>H<sub>7</sub>Cl<sub>2</sub>FN<sub>3</sub>, 257.9996; found, 257.9991.</div></div><div id="sec4_1_26" class="NLM_sec NLM_sec_level_3"><div id="ac_i51" class="anchor-spacer"></div><h4 class="article-section__title" id="_i51">4.1.26.  2,5-Dichloro-<i>N</i>-(2-fluorophenyl)pyrimidin-4-amineamine (<b>3aa</b>)</h4><div class="NLM_p last">White solid; yield 56%; mp 129â130 Â°C; <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): Î´ 8.39 (td, <i>J</i> = 8.2, 1.4 Hz, 1H), 8.24 (s, 1H), 7.54 (s, 1H), 7.22 (t, <i>J</i> = 7.4 Hz, 1H), 7.19â7.11 (m, 2H). HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>10</sub>H<sub>7</sub>Cl<sub>2</sub>FN<sub>3</sub>, 257.9996; found, 257.9993.</div></div><div id="sec4_1_27" class="NLM_sec NLM_sec_level_3"><div id="ac_i52" class="anchor-spacer"></div><h4 class="article-section__title" id="_i52">4.1.27.  2,5-Dichloro-<i>N</i>-(4-(trifluoromethyl)phenyl)pyrimidin-4-amineamine (<b>3ab</b>)</h4><div class="NLM_p last">White solid; yield 69%; mp 135â136 Â°C; <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): Î´ 8.28 (s, 1H), 7.79 (d, <i>J</i> = 8.5 Hz, 2H), 7.67 (d, <i>J</i> = 8.6 Hz, 2H), 7.38 (s, 1H). HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>11</sub>H<sub>7</sub>Cl<sub>2</sub>F<sub>3</sub>N<sub>3</sub>, 307.9964; found, 307.9960.</div></div><div id="sec4_1_28" class="NLM_sec NLM_sec_level_3"><div id="ac_i53" class="anchor-spacer"></div><h4 class="article-section__title" id="_i53">4.1.28.  2,5-Dichloro-<i>N</i>-(4-(trifluoromethoxy)phenyl)pyrimidin-4-amineamine (<b>3ac</b>)</h4><div class="NLM_p last">White solid; yield 74%; mp 109â110 Â°C; <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): Î´ 8.21 (s, 1H), 7.65 (d, <i>J</i> = 9.0 Hz, 2H), 7.29 (s, 1H), 7.25 (d, <i>J</i> = 9.0 Hz, 2H). HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>11</sub>H<sub>7</sub>Cl<sub>2</sub>F<sub>3</sub>N<sub>3</sub>O, 323.9913; found, 323.9907.</div></div><div id="sec4_1_29" class="NLM_sec NLM_sec_level_3"><div id="ac_i54" class="anchor-spacer"></div><h4 class="article-section__title" id="_i54">4.1.29.  2,5-Dichloro-<i>N</i>-(3-(trifluoromethoxy)phenyl)pyrimidin-4-amine (<b>3ad</b>)</h4><div class="NLM_p last">White solid; yield 65%; mp 132â133 Â°C; <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): Î´ 8.24 (s, 1H), 7.67 (s, 1H), 7.52 (d, <i>J</i> = 8.2 Hz, 1H), 7.41 (t, <i>J</i> = 8.2 Hz, 1H), 7.30 (s, 1H), 7.04 (d, <i>J</i> = 8.2 Hz, 1H). HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>11</sub>H<sub>7</sub>Cl<sub>2</sub>F<sub>3</sub>N<sub>3</sub>O, 323.9913; found, 323.9906.</div></div><div id="sec4_1_30" class="NLM_sec NLM_sec_level_3"><div id="ac_i55" class="anchor-spacer"></div><h4 class="article-section__title" id="_i55">4.1.30.  2,5-Dichloro-<i>N</i>-(4-fluoro-3-(trifluoromethyl)phenyl)pyrimidin-4-amine (<b>3ae</b>)</h4><div class="NLM_p last">White solid; yield 72%; mp 159â160 Â°C; <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): Î´ 8.27 (s, 1H), 7.92â7.88 (m, 1H), 7.85 (dd, <i>J</i> = 6.0, 2.7 Hz, 1H), 7.29 (s, 1H), 7.28â7.27 (m, 1H). HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>11</sub>H<sub>6</sub>Cl<sub>2</sub>F<sub>4</sub>N<sub>3</sub>, 325.9869; found, 325.9862.</div></div><div id="sec4_1_31" class="NLM_sec NLM_sec_level_3"><div id="ac_i56" class="anchor-spacer"></div><h4 class="article-section__title" id="_i56">4.1.31.  <i>N</i>-(4-((2,5-Dichloropyrimidin-4-yl)amino)phenyl)acetamide (<b>3af</b>)</h4><div class="NLM_p last">White solid; yield 85%; mp 198â199 Â°C; <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ 9.97 (s, 1H), 9.47 (s, 1H), 8.34 (s, 1H), 7.57 (d, <i>J</i> = 8.8 Hz, 2H), 7.45 (d, <i>J</i> = 8.8 Hz, 2H), 2.05 (s, 3H). HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>12</sub>H<sub>11</sub>Cl<sub>2</sub>N<sub>4</sub>O, 297.0304; found, 297.0302.</div></div><div id="sec4_1_32" class="NLM_sec NLM_sec_level_3"><div id="ac_i57" class="anchor-spacer"></div><h4 class="article-section__title" id="_i57">4.1.32.  <i>N</i>-(3-((2,5-Dichloropyrimidin-4-yl)amino)phenyl)acetamide (<b>3ag</b>)</h4><div class="NLM_p last">White solid; yield 80%; mp 175â176 Â°C; <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ 9.99 (s, 1H), 9.53 (s, 1H), 8.36 (s, 1H), 7.83 (s, 1H), 7.36 (d, <i>J</i> = 7.9 Hz, 1H), 7.30 (t, <i>J</i> = 8.0 Hz, 1H), 7.23 (d, <i>J</i> = 7.9 Hz, 1H), 2.06 (s, 3H). HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>12</sub>H<sub>11</sub>Cl<sub>2</sub>N<sub>4</sub>O, 297.0304; found, 297.0301.</div></div><div id="sec4_1_33" class="NLM_sec NLM_sec_level_3"><div id="ac_i58" class="anchor-spacer"></div><h4 class="article-section__title" id="_i58">4.1.33.  <i>N</i>-(2-((2,5-Dichloropyrimidin-4-yl)amino)phenyl)acetamide (<b>3ah</b>)</h4><div class="NLM_p last">White solid; yield 72%; mp 183â184 Â°C; <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): Î´ 8.64 (s, 1H), 8.18 (s, 1H), 7.77 (d, <i>J</i> = 8.0 Hz, 1H), 7.69 (s, 1H), 7.34 (t, <i>J</i> = 7.7 Hz, 1H), 7.20 (t, <i>J</i> = 7.7 Hz, 1H), 7.08 (d, <i>J</i> = 7.9 Hz, 1H), 2.23 (s, 3H). HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>12</sub>H<sub>11</sub>Cl<sub>2</sub>N<sub>4</sub>O, 297.0304; found, 297.0329.</div></div><div id="sec4_1_34" class="NLM_sec NLM_sec_level_3"><div id="ac_i59" class="anchor-spacer"></div><h4 class="article-section__title" id="_i59">4.1.34.  2,5-Dichloro-<i>N</i>-(<i>o</i>-tolyl)pyrimidin-4-amine (<b>3ai</b>)</h4><div class="NLM_p last">White solid; yield 85%; mp 88â89 Â°C; <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): Î´ 8.19 (s, 1H), 7.85 (d, <i>J</i> = 8.0 Hz, 1H), 7.29 (t, <i>J</i> = 7.8 Hz, 1H), 7.27â7.25 (m, 1H), 7.17 (t, J = 7.5 Hz, 1H), 7.13 (s, 1H), 2.32 (s, 3H). HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>11</sub>H<sub>10</sub>Cl<sub>2</sub>N<sub>3</sub>, 254.0246; found, 254.0240.</div></div><div id="sec4_1_35" class="NLM_sec NLM_sec_level_3"><div id="ac_i60" class="anchor-spacer"></div><h4 class="article-section__title" id="_i60">4.1.35.  General Procedure for the Synthesis of Compounds <b>8â67</b> Exemplified by 5-Chloro-<i>N</i><sup>2</sup>-(5-(4-methylpiperazin-1-yl)pyridin-2-yl)-<i>N</i><sup>4</sup>-phenylpyrimidine-2,4-diamine (<b>34</b>)</h4><div class="NLM_p last">To a suspension of compound <b>3a</b> (0.83 mmol, 0.2 g) in 20 mL of 1,4-dioxane was added 5-(4-methylpiperazin-1-yl)pyridin-2-amine (1.00 mmol, 0.2 g), Pd<sub>2</sub>(dba)<sub>3</sub> (0.042 mmol, 38 mg), BINAP (0.083 mmol, 52 mg), and NaO<i>t</i>Bu (1.25 mmol, 1.2 g) with the flask was purged with Ar<sub>2</sub>. Then, the flask was sealed, and the mixture was heated for 12 h at 110 Â°C. The reaction was cooled to room temperature, and the solvent was removed under reduced pressure. The residue was purified with an automated chromatography system. Obtained as white powder; yield 59% (0.19 g). mp 197â198 Â°C; <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): Î´ 8.20 (s, 1H), 8.14 (s, 1H), 8.07 (d, <i>J</i> = 9.1 Hz, 1H), 8.01 (d, <i>J</i> = 2.9 Hz, 1H), 7.59 (d, <i>J</i> = 7.7 Hz, 2H), 7.38 (t, <i>J</i> = 7.9 Hz, 2H), 7.20 (d, <i>J</i> = 3.2 Hz, 1H), 7.18 (d, <i>J</i> = 7.3 Hz, 1H), 7.07 (s, 1H), 3.19â3.15 (m, 4H), 2.64â2.61 (m, 4H), 2.38 (s, 3H); <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): Î´ 156.92, 155.66, 154.27, 146.21, 142.70, 137.73, 136.56, 128.85 (2C), 126.35 (2C), 124.46, 121.99, 113.26, 105.32, 54.90 (2C), 49.64 (2C), 46.08; HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>20</sub>H<sub>23</sub>ClN<sub>7</sub>, 396.1698; found, 396.1697.</div></div><div id="sec4_1_36" class="NLM_sec NLM_sec_level_3"><div id="ac_i61" class="anchor-spacer"></div><h4 class="article-section__title" id="_i61">4.1.36.  5-Chloro-<i>N</i><sup>2</sup>-(pyridin-3-yl)-<i>N</i><sup>4</sup>-(<i>o</i>-tolyl)pyrimidine-2,4-diamine (<b>8</b>)</h4><div class="NLM_p last">White solid; yield 69%; mp 152â153 Â°C; <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): Î´ 8.47 (d, <i>J</i> = 2.5 Hz, 1H), 8.19 (d, <i>J</i> = 4.7 Hz, 1H), 8.08 (s, 1H), 8.02 (d, <i>J</i> = 8.3 Hz, 1H), 7.70 (d, <i>J</i> = 7.8 Hz, 1H), 7.34â7.27 (m, 2H), 7.20 (t, <i>J</i> = 7.4 Hz, 1H), 7.12 (s, 1H), 7.08 (dd, <i>J</i> = 8.4, 4.7 Hz, 1H), 6.92 (s, 1H), 2.30 (s, 3H); <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): Î´ 157.66, 156.53, 154.04, 143.17, 140.62, 136.29, 135.79, 131.79, 130.70, 126.55, 125.94, 125.74, 125.03, 123.19, 105.88, 17.91; HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>16</sub>H<sub>15</sub>ClN<sub>5</sub>, 312.1010; found, 312.1011.</div></div><div id="sec4_1_37" class="NLM_sec NLM_sec_level_3"><div id="ac_i62" class="anchor-spacer"></div><h4 class="article-section__title" id="_i62">4.1.37.  5-Chloro-<i>N</i><sup>2</sup>-cyclopropyl-<i>N</i><sup>4</sup>-(<i>o</i>-tolyl)pyrimidine-2,4-diamine (<b>9</b>)</h4><div class="NLM_p last">White solid; yield 85%; mp 131â132 Â°C; <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): Î´ 8.00 (s, 1H), 7.26 (s, 1H), 7.22 (t, <i>J</i> = 7.0 Hz, 2H), 7.06 (t, <i>J</i> = 7.4 Hz, 1H), 6.91 (s, 1H), 5.16 (s, 1H), 2.69 (qd, 1H), 2.33 (s, 3H), 0.77 (q, <i>J</i> = 6.7 Hz, 2H), 0.64â0.42 (m, 2H); <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): Î´ 161.41, 155.72, 154.06, 136.54, 130.40 (2C), 126.38 (2C), 124.18, 122.48, 24.07, 17.82, 7.24 (2C); HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>14</sub>H<sub>16</sub>ClN<sub>4</sub>, 275.1058; found, 275.1060.</div></div><div id="sec4_1_38" class="NLM_sec NLM_sec_level_3"><div id="ac_i63" class="anchor-spacer"></div><h4 class="article-section__title" id="_i63">4.1.38.  5-Chloro-<i>N</i><sup>2</sup>-cyclohexyl-<i>N</i><sup>4</sup>-(<i>o</i>-tolyl)pyrimidine-2,4-diamine (<b>10</b>)</h4><div class="NLM_p last">White solid; yield 88%; mp 128.3â129.5 Â°C; <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): Î´ 8.07 (s, 1H), 7.93 (s, 1H), 7.22 (d, <i>J</i> = 7.7 Hz, 2H), 7.06 (t, <i>J</i> = 7.4 Hz, 1H), 6.85 (s, 1H), 4.86 (s, 1H), 3.66 (s, 1H), 2.32 (s, 3H), 2.01 (d, <i>J</i> = 9.7 Hz, 2H), 1.73 (d, <i>J</i> = 13.6 Hz, 2H), 1.62 (d, <i>J</i> = 13.6 Hz, 1H), 1.35 (d, <i>J</i> = 12.4 Hz, 2H), 1.26â1.10 (m, 3H); <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): Î´ 159.84, 155.78, 154.18, 136.61, 130.39 (2C), 126.32 (2C), 124.15, 122.59, 50.30, 33.17, 25.75 (2C), 24.96 (2C), 17.79; HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>17</sub>H<sub>21</sub>ClN<sub>4</sub>, 317.1528; found, 317.1531.</div></div><div id="sec4_1_39" class="NLM_sec NLM_sec_level_3"><div id="ac_i64" class="anchor-spacer"></div><h4 class="article-section__title" id="_i64">4.1.39.  5-Chloro-<i>N</i><sup>2</sup>-phenyl-<i>N</i><sup>4</sup>-(<i>o</i>-tolyl)pyrimidine-2,4-diamine (<b>11</b>)</h4><div class="NLM_p last">White solid; yield 74%; mp 165â166 Â°C; <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): Î´ 8.06 (s, 1H), 7.81 (d, <i>J</i> = 7.9 Hz, 1H), 7.44 (d, <i>J</i> = 7.8 Hz, 2H), 7.29â7.25 (m, 2H), 7.21 (t, <i>J</i> = 7.9 Hz, 2H), 7.17 (t, <i>J</i> = 7.4 Hz, 1H), 6.99 (s, 1H), 6.97 (t, <i>J</i> = 7.4 Hz, 1H), 6.90 (s, 1H), 2.32 (s, 3H); <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): Î´ 157.91, 156.31, 154.06, 139.47, 136.08, 131.18, 130.57, 128.67 (2C), 126.48, 125.44, 124.66, 122.21, 119.11 (2C), 105.14, 17.90; HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>17</sub>H<sub>16</sub>ClN<sub>4</sub>, 311.1058; found, 311.1062.</div></div><div id="sec4_1_40" class="NLM_sec NLM_sec_level_3"><div id="ac_i65" class="anchor-spacer"></div><h4 class="article-section__title" id="_i65">4.1.40.  5-Chloro-<i>N</i><sup>4</sup>-(<i>o</i>-tolyl)-<i>N</i><sup>2</sup>-(<i>p</i>-tolyl)pyrimidine-2,4-diamine (<b>12</b>)</h4><div class="NLM_p last">White solid; yield 70%; mp 136â137 Â°C; <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): Î´ 8.04 (s, 1H), 7.83 (d, <i>J</i> = 7.9 Hz, 1H), 7.32 (d, <i>J</i> = 8.3 Hz, 2H), 7.26 (t, <i>J</i> = 8.3 Hz, 2H), 7.16 (t, <i>J</i> = 7.4 Hz, 1H), 7.02 (d, <i>J</i> = 8.2 Hz, 2H), 6.90 (d, <i>J</i> = 9.1 Hz, 2H), 2.32 (s, 3H), 2.29 (s, 3H); <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): Î´ 158.05, 156.23, 154.06, 136.88, 136.13, 131.80, 130.93, 130.55, 129.20 (2C), 126.48, 125.29, 124.47, 119.37 (2C), 104.86, 20.69, 17.91; HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>18</sub>H<sub>18</sub>ClN<sub>4</sub>, 325.1215; found, 325.1215.</div></div><div id="sec4_1_41" class="NLM_sec NLM_sec_level_3"><div id="ac_i66" class="anchor-spacer"></div><h4 class="article-section__title" id="_i66">4.1.41.  <i>N</i><sup>2</sup>-Benzyl-5-chloro-<i>N</i><sup>4</sup>-(<i>o</i>-tolyl)pyrimidine-2,4-diamine (<b>13</b>)</h4><div class="NLM_p last">White solid; yield 77%; mp 149â150 Â°C; <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): Î´ 7.94 (s, 1H), 7.85 (d, <i>J</i> = 7.9 Hz, 1H), 7.28 (m, 5H), 7.20 (d, <i>J</i> = 7.5 Hz, 1H), 7.15 (t, <i>J</i> = 6.9 Hz, 1H), 7.05 (t, <i>J</i> = 7.4 Hz, 1H), 6.85 (s, 1H), 4.51 (d, <i>J</i> = 5.3 Hz, 2H), 2.29 (s, 3H); <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): Î´ 160.36, 155.94, 154.15, 139.22, 136.36, 130.40, 128.45 (2C), 127.41 (2C), 127.07, 126.40, 124.48, 123.09, 45.75, 17.79; HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>18</sub>H<sub>18</sub>ClN<sub>4</sub>: 325.1215; found, 325.1215.</div></div><div id="sec4_1_42" class="NLM_sec NLM_sec_level_3"><div id="ac_i67" class="anchor-spacer"></div><h4 class="article-section__title" id="_i67">4.1.42.  5-Chloro-<i>N</i><sup>2</sup>-phenethyl-<i>N</i><sup>4</sup>-(<i>o</i>-tolyl)pyrimidine-2,4-diamine (<b>14</b>)</h4><div class="NLM_p last">White solid; yield 80%; mp 99â100 Â°C; <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): Î´ 7.96â7.91 (s, 1H), 7.28 (t, <i>J</i> = 7.5 Hz, 2H), 7.26 (s, 1H), 7.25â7.21 (m, 2H), 7.20 (d, <i>J</i> = 7.4 Hz, 1H), 7.14 (d, <i>J</i> = 5.8 Hz, 2H), 7.10 (t, <i>J</i> = 7.4 Hz, 1H), 6.87 (s, 1H), 5.02 (s, 2H), 3.56 (d, <i>J</i> = 6.3 Hz, 2H), 2.84 (t, <i>J</i> = 7.1 Hz, 2H), 2.33 (s, 3H); <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): Î´ 160.42, 155.95, 154.15, 139.23, 136.49, 130.49, 128.79 (2C), 128.46 (2C), 126.45, 126.25, 124.48, 123.06, 43.15, 35.94, 17.87; HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>19</sub>H<sub>20</sub>ClN<sub>4</sub>, 339.1371; found, 339.1373.</div></div><div id="sec4_1_43" class="NLM_sec NLM_sec_level_3"><div id="ac_i68" class="anchor-spacer"></div><h4 class="article-section__title" id="_i68">4.1.43.  5-Chloro-<i>N</i><sup>2</sup>-(furan-2-ylmethyl)-<i>N</i><sup>4</sup>-(<i>o</i>-tolyl)pyrimidine-2,4-diamine (<b>15</b>)</h4><div class="NLM_p last">White solid; yield 66%; mp 129.2â131.3 Â°C; <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): Î´ 7.96 (s, 1H), 7.94 (d, <i>J</i> = 7.8 Hz, 1H), 7.33 (s, 1H), 7.22 (t, <i>J</i> = 7.1 Hz, 2H), 7.08 (t, <i>J</i> = 7.4 Hz, 1H), 6.86 (s, 1H), 6.28 (dd, <i>J</i> = 2.8, 1.8 Hz, 1H), 6.13 (s, 1H), 5.26 (s, 1H), 4.49 (d, <i>J</i> = 5.4 Hz, 2H), 2.31 (s, 3H); <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): Î´ 160.06, 155.96, 154.10, 152.44, 141.83, 136.33 (2C), 130.44, 126.39 (2C), 124.62, 123.24, 110.23, 106.78, 38.87, 17.81; HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>16</sub>H<sub>15</sub>ClN<sub>4</sub>O, 315.1007; found, 315.1011.</div></div><div id="sec4_1_44" class="NLM_sec NLM_sec_level_3"><div id="ac_i69" class="anchor-spacer"></div><h4 class="article-section__title" id="_i69">4.1.44.  5-Chloro-<i>N</i><sup>2</sup>-(pyridin-2-yl)-<i>N</i><sup>4</sup>-(<i>o</i>-tolyl)pyrimidine-2,4-diamine (<b>16</b>)</h4><div class="NLM_p last">White solid; yield 55%; mp 144â145 Â°C; <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): Î´ 8.62 (s, 1H), 8.27 (d, <i>J</i> = 4.7 Hz, 1H), 8.19 (s, 1H), 7.98 (d, <i>J</i> = 8.5 Hz, 1H), 7.73 (d, <i>J</i> = 7.9 Hz, 1H), 7.42 (t, <i>J</i> = 7.8 Hz, 1H), 7.28 (t, <i>J</i> = 7.2 Hz, 2H), 7.20 (t, <i>J</i> = 7.4 Hz, 1H), 6.91 (s, 1H), 6.87â6.74 (m, 1H), 2.31 (s, 3H); <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): Î´ 157.00, 156.46, 154.24, 152.85, 147.87, 137.48, 135.99, 131.99, 130.63, 126.46, 125.82, 125.27, 117.23, 112.55, 105.76, 17.93; HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>16</sub>H<sub>15</sub>ClN<sub>5</sub>, 312.1010; found, 312.1010.</div></div><div id="sec4_1_45" class="NLM_sec NLM_sec_level_3"><div id="ac_i70" class="anchor-spacer"></div><h4 class="article-section__title" id="_i70">4.1.45.  5-Chloro-<i>N</i><sup>2</sup>-(4-chloropyridin-2-yl)-<i>N</i><sup>4</sup>-(<i>o</i>-tolyl)pyrimidine-2,4-diamine (<b>17</b>)</h4><div class="NLM_p last">White solid; yield 58%; mp 164â166 Â°C; <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): Î´ 9.02 (s, 1H), 8.23 (s, 1H), 8.18 (d, <i>J</i> = 5.3 Hz, 1H), 8.09 (s, 1H), 7.68 (d, <i>J</i> = 7.8 Hz, 1H), 7.37â7.29 (m, 2H), 7.21 (t, <i>J</i> = 7.5 Hz, 1H), 6.94 (s, 1H), 6.82 (dd, <i>J</i> = 5.3, 1.7 Hz, 1H), 2.32 (s, 3H); <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): Î´ 156.74, 156.51, 154.23, 153.83, 148.58, 145.20, 135.62, 131.84, 130.98, 127.05, 126.20, 124.95, 117.58, 112.19, 106.15, 17.98; HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>16</sub>H<sub>14</sub>Cl<sub>2</sub>N<sub>5</sub>, 346.062; found, 346.0621.</div></div><div id="sec4_1_46" class="NLM_sec NLM_sec_level_3"><div id="ac_i71" class="anchor-spacer"></div><h4 class="article-section__title" id="_i71">4.1.46.  5-Chloro-<i>N</i><sup>2</sup>-(pyridin-4-yl)-<i>N</i><sup>4</sup>-(<i>o</i>-tolyl)pyrimidine-2,4-diamine (<b>18</b>)</h4><div class="NLM_p last">White solid; yield 50%; mp 182â184 Â°C; <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): Î´ 8.28 (d, <i>J</i> = 6.3 Hz, 2H), 8.11 (s, 1H), 7.65 (d, <i>J</i> = 7.8 Hz, 1H), 7.32 (d, <i>J</i> = 6.3 Hz, 4H), 7.25 (s, 1H), 7.18 (s, 1H), 6.95 (s, 1H), 2.28 (s, 3H); <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): Î´ 157.13, 156.72, 154.01, 150.19 (2C), 146.27, 135.65, 132.52, 130.81, 126.61, 126.38, 125.60, 112.43 (2C), 106.54, 17.95; HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>16</sub>H<sub>15</sub>ClN<sub>5</sub>, 312.1010; found, 312.1011.</div></div><div id="sec4_1_47" class="NLM_sec NLM_sec_level_3"><div id="ac_i72" class="anchor-spacer"></div><h4 class="article-section__title" id="_i72">4.1.47.  5-Chloro-<i>N</i><sup>2</sup>-(2-methoxypyridin-4-yl)-<i>N</i><sup>4</sup>-(<i>o</i>-tolyl)pyrimidine-2,4-diamine (<b>19</b>)</h4><div class="NLM_p last">White solid; yield 47%; mp 163â164 Â°C; <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): Î´ 8.10 (s, 1H), 7.88 (d, <i>J</i> = 5.8 Hz, 1H), 7.71 (d, <i>J</i> = 7.9 Hz, 1H), 7.30 (d, <i>J</i> = 7.3 Hz, 2H), 7.21 (t, <i>J</i> = 7.4 Hz, 1H), 7.13 (s, 1H), 6.98 (s, 1H), 6.94 (s, 1H), 6.87 (d, <i>J</i> = 5.8 Hz, 1H), 3.87 (s, 3H), 2.31 (s, 3H); <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): Î´ 165.44, 157.21, 156.54, 153.90, 148.39, 146.93, 135.63, 131.79, 130.81, 126.73, 126.03, 125.03, 107.60, 106.57, 97.85, 53.30, 17.92; HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>17</sub>H<sub>16</sub>ClN<sub>5</sub>O, 341.5288; found, 341.5289.</div></div><div id="sec4_1_48" class="NLM_sec NLM_sec_level_3"><div id="ac_i73" class="anchor-spacer"></div><h4 class="article-section__title" id="_i73">4.1.48.  5-Chloro-<i>N</i><sup>2</sup>-(pyridin-4-ylmethyl)-<i>N</i><sup>4</sup>-(<i>o</i>-tolyl)pyrimidine-2,4-diamine (<b>20</b>)</h4><div class="NLM_p last">White solid; yield 80%; mp 137â138 Â°C; <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): Î´ 8.49 (d, <i>J</i> = 5.4 Hz, 2H), 7.94 (s, 1H), 7.20 (d, <i>J</i> = 7.6 Hz, 1H), 7.14 (s, 2H), 7.07 (t, <i>J</i> = 7.2 Hz, 2H), 6.84 (s, 1H), 5.53 (s, 1H), 4.48 (s, 2H), 2.24 (s, 3H), 1.80 (s, 1H); <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): Î´ 160.21, 156.12, 154.17, 149.78 (2C), 148.68, 136.09, 130.46, 130.24, 126.34, 124.88, 123.50, 121.98 (2C), 44.60, 17.75; HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>17</sub>H<sub>16</sub>ClN<sub>5</sub>, 325.5377; found, 325.5379.</div></div><div id="sec4_1_49" class="NLM_sec NLM_sec_level_3"><div id="ac_i74" class="anchor-spacer"></div><h4 class="article-section__title" id="_i74">4.1.49.  5-Chloro-<i>N</i><sup>2</sup>-(2-(pyridin-2-yl)ethyl)-<i>N</i><sup>4</sup>-(<i>o</i>-tolyl)pyrimidine-2,4-diamine (<b>21</b>)</h4><div class="NLM_p last">White solid; yield 74%; mp 116.2â117.2 Â°C; <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): Î´ 8.53 (d, <i>J</i> = 4.8 Hz, 1H), 8.04 (s, 1H), 7.93 (s, 1H), 7.56 (dd, <i>J</i> = 7.6, 6.3 Hz, 1H), 7.22 (t, <i>J</i> = 6.8 Hz, 2H), 7.11 (dd, <i>J</i> = 7.3, 5.1 Hz, 1H), 7.07 (t, <i>J</i> = 7.4 Hz, 2H), 6.86 (s, 1H), 5.40 (s, 1H), 3.72 (dd, <i>J</i> = 12.8, 6.3 Hz, 2H), 3.03 (t, <i>J</i> = 6.7 Hz, 2H), 2.32 (s, 3H); <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): Î´ 160.45, 159.59, 155.87, 154.14, 149.34 (2C), 136.56, 136.29, 130.43 (2C), 126.42, 124.31, 123.33, 122.87, 121.31, 41.34, 37.67, 17.83; HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>18</sub>H<sub>18</sub>ClN<sub>5</sub>, 340.1323; found, 340.1319.</div></div><div id="sec4_1_50" class="NLM_sec NLM_sec_level_3"><div id="ac_i75" class="anchor-spacer"></div><h4 class="article-section__title" id="_i75">4.1.50.  5-Chloro-<i>N</i><sup>2</sup>-(2-(pyridin-3-yl)ethyl)-<i>N</i><sup>4</sup>-(<i>o</i>-tolyl)pyrimidine-2,4-diamine (<b>22</b>)</h4><div class="NLM_p last">White solid; yield 79%; mp 107.1â108.3 Â°C; <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): Î´ 8.45 (d, <i>J</i> = 4.8 Hz, 1H), 8.40 (s, 1H), 7.94 (s, 1H), 7.92 (d, <i>J</i> = 7.3 Hz, 1H), 7.40 (s, 1H), 7.25â7.22 (m, 2H), 7.18 (dd, <i>J</i> = 7.5, 4.9 Hz, 1H), 7.11 (t, <i>J</i> = 7.4 Hz, 1H), 6.86 (s, 1H), 5.08 (s, 1H), 3.53 (d, <i>J</i> = 5.2 Hz, 2H), 2.83 (t, <i>J</i> = 7.0 Hz, 2H), 2.32 (s, 3H); <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): Î´ 160.32, 156.09, 154.15, 150.16, 147.80 (2C), 136.37, 136.19, 134.63, 130.53, 130.13, 126.41, 124.80, 123.44, 123.28, 42.84, 33.14, 17.85; HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>18</sub>H<sub>18</sub>ClN<sub>5</sub>, 340.1323; found, 340.1319.</div></div><div id="sec4_1_51" class="NLM_sec NLM_sec_level_3"><div id="ac_i76" class="anchor-spacer"></div><h4 class="article-section__title" id="_i76">4.1.51.  5-Chloro-<i>N</i><sup>2</sup>-(5-(4-methylpiperazin-1-yl)pyridin-2-yl)-<i>N</i><sup>4</sup>-(<i>o</i>-tolyl)pyrimidine-2,4-diamine (<b>23</b>)</h4><div class="NLM_p last">White solid; yield 42%; mp 172â173 Â°C; <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): Î´ 8.30 (s, 1H), 8.14 (s, 1H), 7.98 (d, <i>J</i> = 2.9 Hz, 1H), 7.87 (d, <i>J</i> = 9.1 Hz, 1H), 7.76 (d, <i>J</i> = 7.9 Hz, 1H), 7.31â7.26 (m, 2H), 7.18 (t, <i>J</i> = 7.5 Hz, 1H), 7.06 (dd, <i>J</i> = 9.1, 2.9 Hz, 1H), 6.90 (s, 1H), 3.15â3.11 (m, 4H), 2.62â2.57 (m, 4H), 2.36 (s, 3H), 2.30 (s, 3H); <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): Î´ 156.99, 156.25, 154.21, 146.37, 142.54, 136.47, 136.05, 131.62, 130.60, 126.44, 126.38, 125.59, 124.93, 112.93, 105.13, 54.92 (2C), 49.70 (2C), 46.09, 18.02; HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>21</sub>H<sub>25</sub>ClN<sub>7</sub>, 410.1854; found, 410.1853.</div></div><div id="sec4_1_52" class="NLM_sec NLM_sec_level_3"><div id="ac_i77" class="anchor-spacer"></div><h4 class="article-section__title" id="_i77">4.1.52.  5-Chloro-<i>N</i><sup>2</sup>-(5-(4-ethylpiperazin-1-yl)pyridin-2-yl)-<i>N</i><sup>4</sup>-(<i>o</i>-tolyl)pyrimidine-2,4-diamine (<b>24</b>)</h4><div class="NLM_p last">White solid; yield 45%; mp 226.3â227.6 Â°C; <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): Î´ 8.10 (s, 1H), 7.91 (s, 1H), 7.89 (s, 1H), 7.76 (d, <i>J</i> = 7.7 Hz, 1H), 7.67 (s, 1H), 7.33â7.27 (m, 2H), 7.19 (t, <i>J</i> = 7.4 Hz, 1H), 7.07 (d, <i>J</i> = 8.9 Hz, 1H), 6.90 (s, 1H), 3.93â3.79 (m, 4H), 3.18â2.97 (m, 4H), 2.31 (s, 3H), 1.65â1.53 (m, 3H), 1.31â1.25 (m, 2H); <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): Î´ 156.89, 156.33, 154.10, 146.39, 142.69, 136.04, 130.60 (2C), 126.42, 126.04 (2C), 125.62, 124.94, 120.83, 112.92, 66.77 (2C), 49.96 (2C), 31.47, 30.13, 17.92; HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>22</sub>H<sub>26</sub>ClN<sub>7</sub>, 423.6788; found, 423.6789.</div></div><div id="sec4_1_53" class="NLM_sec NLM_sec_level_3"><div id="ac_i78" class="anchor-spacer"></div><h4 class="article-section__title" id="_i78">4.1.53.  5-Chloro-<i>N</i><sup>2</sup>-(5-(piperazin-1-yl)pyridin-2-yl)-<i>N</i><sup>4</sup>-(<i>o</i>-tolyl)pyrimidine-2,4-diamine (<b>25</b>)</h4><div class="NLM_p last">White solid; yield 54%; mp 197.8â198.6 Â°C; <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): Î´ 8.11 (s, 1H), 7.94 (d, <i>J</i> = 2.5 Hz, 1H), 7.89 (d, <i>J</i> = 9.1 Hz, 1H), 7.86 (s, 1H), 7.75 (d, <i>J</i> = 7.8 Hz, 1H), 7.32â7.26 (m, 2H), 7.19 (t, <i>J</i> = 7.4 Hz, 1H), 7.07 (d, <i>J</i> = 9.0 Hz, 1H), 6.89 (s, 1H), 3.69â3.54 (m, 4H), 3.12â2.98 (m, 4H), 2.35â2.20 (m, 3H), 1.54â1.42 (m, 9H); <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): Î´ 156.91, 156.35, 154.61, 154.15, 146.70, 142.68, 137.12, 136.04, 131.70, 130.60, 126.96, 126.42, 125.66, 125.00, 112.84, 105.42, 79.93, 50.13 (4C), 28.40 (3C), 17.92; HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>25</sub>H<sub>30</sub>ClN<sub>7</sub>O<sub>2</sub>, 496.2222; found, 496.2228.</div></div><div id="sec4_1_54" class="NLM_sec NLM_sec_level_3"><div id="ac_i79" class="anchor-spacer"></div><h4 class="article-section__title" id="_i79">4.1.54.  5-Chloro-<i>N</i><sup>2</sup>-(5-morpholinopyridin-2-yl)-<i>N</i><sup>4</sup>-(<i>o</i>-tolyl)pyrimidine-2,4-diamine (<b>26</b>)</h4><div class="NLM_p last">White solid; yield 43%; mp 167.8â168.5 Â°C; <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): Î´ 8.23 (s, 1H), 8.11 (s, 1H), 7.94 (d, <i>J</i> = 2.5 Hz, 1H), 7.86 (d, <i>J</i> = 9.0 Hz, 1H), 7.76 (d, <i>J</i> = 7.8 Hz, 1H), 7.27 (t, <i>J</i> = 10.5 Hz, 2H), 7.18 (t, <i>J</i> = 7.4 Hz, 1H), 7.06 (dd, <i>J</i> = 9.1, 2.5 Hz, 1H), 6.89 (s, 1H), 3.32â3.00 (m, 4H), 2.77â2.58 (m, 4H), 2.30 (s, 3H); <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): Î´ 156.99, 156.33, 154.13, 146.25, 142.70, 136.28, 136.09, 131.62, 130.57, 126.42, 126.32, 125.58, 124.98, 113.01, 105.23, 52.51 (2C), 49.65 (2C), 17.91; HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>20</sub>H<sub>21</sub>ClN<sub>6</sub>O, 396.6088; found, 396.6089.</div></div><div id="sec4_1_55" class="NLM_sec NLM_sec_level_3"><div id="ac_i80" class="anchor-spacer"></div><h4 class="article-section__title" id="_i80">4.1.55.  5-Chloro-<i>N</i><sup>2</sup>-(5-(piperidin-1-yl)pyridin-2-yl)-<i>N</i><sup>4</sup>-(<i>o</i>-tolyl)pyrimidine-2,4-diamine (<b>27</b>)</h4><div class="NLM_p last">mp 175.1â176.4 Â°C; <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): Î´ 8.10 (s, 1H), 7.94 (d, <i>J</i> = 2.6 Hz, 1H), 7.85 (d, <i>J</i> = 9.1 Hz, 1H), 7.83 (s, 1H), 7.78 (d, <i>J</i> = 7.9 Hz, 1H), 7.29â7.27 (m, 2H), 7.18 (t, <i>J</i> = 7.4 Hz, 1H), 7.08 (dd, <i>J</i> = 9.2, 2.3 Hz, 1H), 6.88 (s, 1H), 3.13â2.86 (m, 4H), 2.31 (s, 3H), 1.76â1.68 (m, 4H), 1.56 (dt, <i>J</i> = 11.3, 5.8 Hz, 2H); <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): Î´ 157.00, 156.29, 154.15, 145.74, 143.82, 136.78, 136.10, 131.49, 130.56, 126.75, 126.42, 125.53, 124.87, 112.85, 105.22, 51.31 (2C), 25.79 (2C), 24.00, 17.91; HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>21</sub>H<sub>23</sub>ClN<sub>6</sub>, 395.1745; found, 395.1740.</div></div><div id="sec4_1_56" class="NLM_sec NLM_sec_level_3"><div id="ac_i81" class="anchor-spacer"></div><h4 class="article-section__title" id="_i81">4.1.56.  5-Chloro-<i>N</i><sup>2</sup>-(4-(4-methylpiperazin-1-yl)phenyl)-<i>N</i><sup>4</sup>-(<i>o</i>-tolyl)pyrimidine-2,4-diamine (<b>28</b>)</h4><div class="NLM_p last">White solid; yield 48%; mp 163.4â164.5 Â°C; <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): Î´ 8.01 (s, 1H), 7.87 (d, <i>J</i> = 8.0 Hz, 1H), 7.32 (d, <i>J</i> = 9.0 Hz, 2H), 7.25â7.20 (m, 2H), 7.13 (td, <i>J</i> = 7.4, 1.1 Hz, 1H), 6.91 (s, 1H), 6.89 (s, 1H), 6.83 (d, <i>J</i> = 8.9 Hz, 2H), 3.17â3.06 (m, 4H), 2.63â2.56 (m, 4H), 2.35 (s, 3H), 2.31 (s, 3H); <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): Î´ 158.32, 156.09, 154.07, 148.29, 136.24, 131.73, 130.50, 130.42, 126.47, 124.99, 124.04, 121.02 (2C), 117.31 (2C), 104.55, 51.49 (2C),25.95 (2C), 24.23, 17.89; HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>22</sub>H<sub>26</sub>ClN<sub>6</sub>, 409.1902; found, 409.1903.</div></div><div id="sec4_1_57" class="NLM_sec NLM_sec_level_3"><div id="ac_i82" class="anchor-spacer"></div><h4 class="article-section__title" id="_i82">4.1.57.  5-Chloro-<i>N</i><sup>2</sup>-(4-(4-ethylpiperazin-1-yl)phenyl)-<i>N</i><sup>4</sup>-(<i>o</i>-tolyl)pyrimidine-2,4-diamine (<b>29</b>)</h4><div class="NLM_p last">White solid; yield 51%; mp 171.3â172 Â°C; <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): Î´ 8.02 (s, 1H), 7.87 (d, <i>J</i> = 8.0 Hz, 1H), 7.33 (d, <i>J</i> = 8.9 Hz, 2H), 7.25â7.20 (m, 2H), 7.14 (t, <i>J</i> = 7.4 Hz, 1H), 6.89 (s, 1H), 6.85 (s, 1H), 6.84 (d, <i>J</i> = 8.9 Hz, 2H), 3.20â3.07 (m, 4H), 2.67â2.59 (m, 4H), 2.50 (q, <i>J</i> = 7.3 Hz, 2H), 2.32 (s, 3H), 1.14 (t, <i>J</i> = 7.2 Hz, 3H); <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): Î´ 158.30, 156.13, 154.06, 147.21, 136.22, 132.16, 130.53, 130.49 (2C), 126.45 (2C), 125.05, 124.13, 121.11 (2C), 116.83, 52.82 (2C), 52.32, 49.86 (2C), 17.87, 11.88; HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>23</sub>H<sub>28</sub>ClN<sub>6</sub>, 423.2058; found, 423.2059.</div></div><div id="sec4_1_58" class="NLM_sec NLM_sec_level_3"><div id="ac_i83" class="anchor-spacer"></div><h4 class="article-section__title" id="_i83">4.1.58.  <i>tert</i>-Butyl4-(4-((5-chloro-4-(<i>o</i>-tolylamino)pyrimidin-2-yl)amino)phenyl)piperazine-1-carboxylate (<b>30</b>)</h4><div class="NLM_p last">White solid; yield 68%; mp 162.8â163.3 Â°C; <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): Î´ 8.03 (s, 1H), 7.85 (d, <i>J</i> = 8.0 Hz, 1H), 7.34 (d, <i>J</i> = 8.9 Hz, 2H), 7.25 (dd, <i>J</i> = 17.8, 7.6 Hz, 2H), 7.15 (t, <i>J</i> = 7.4 Hz, 1H), 6.90 (s, 2H), 6.82 (d, <i>J</i> = 8.8 Hz, 2H), 3.65â3.54 (m, 4H), 3.05 (s, 4H), 2.32 (s, 3H), 1.49 (s, 9H); <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): Î´ 158.17, 156.19, 154.70, 154.01 147.03, 136.18, 132.75, 130.74 (2C), 130.53 (2C), 126.43, 125.17, 124.27, 120.91, 117.44, 104.73, 79.81, 50.22 (4C), 28.42 (3C), 17.88; HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>26</sub>H<sub>32</sub>ClN<sub>6</sub>O<sub>2</sub>, 495.2270; found, 495.2270.</div></div><div id="sec4_1_59" class="NLM_sec NLM_sec_level_3"><div id="ac_i84" class="anchor-spacer"></div><h4 class="article-section__title" id="_i84">4.1.59.  5-Chloro-<i>N</i><sup>2</sup>-(4-morpholinophenyl)-<i>N</i><sup>4</sup>-(<i>o</i>-tolyl)pyrimidine-2,4-diamine (<b>31</b>)</h4><div class="NLM_p last">White solid; yield 55%; mp 248.7â249.6 Â°C; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ 8.94 (s, 1H), 8.56 (s, 1H), 7.98 (s, 1H), 7.40â7.13 (m, 6H), 6.58 (d, <i>J</i> = 8.7 Hz, 2H), 3.78â3.54 (m, 4H), 3.05â2.77 (m, 4H), 2.13 (s, 3H); HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>21</sub>H<sub>23</sub>ClN<sub>5</sub>O, 396.1586; found, 396.1586.</div></div><div id="sec4_1_60" class="NLM_sec NLM_sec_level_3"><div id="ac_i85" class="anchor-spacer"></div><h4 class="article-section__title" id="_i85">4.1.60.  5-Chloro-<i>N</i><sup>2</sup>-(4-(piperidin-1-yl)phenyl)-<i>N</i><sup>4</sup>-(<i>o</i>-tolyl)pyrimidine-2,4-diamine</h4><div class="NLM_p last">White solid; yield 60%; (<b>32</b>) mp 147â148.6 Â°C; <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): Î´ 8.00 (s, 1H), 7.87 (d, <i>J</i> = 7.9 Hz, 1H), 7.30 (d, <i>J</i> = 8.9 Hz, 2H), 7.23 (dd, <i>J</i> = 12.9, 6.9 Hz, 2H), 7.12 (td, <i>J</i> = 7.5, 0.9 Hz, 1H), 6.88 (s, 1H), 6.87 (s, 1H), 6.83 (d, J = 8.9 Hz, 2H), 3.12â2.95 (m, 4H), 2.30 (s, 4H), 1.75â1.64 (m, 6H); <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): Î´ 158.28, 156.11, 154.06, 147.14, 136.21, 132.17, 130.51, 126.46, 125.05, 124.09, 121.08 (2C), 116.83 (2C), 104.65, 55.14 (2C), 49.85 (2C), 49.16, 46.07, 17.88; HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>22</sub>H<sub>25</sub>ClN<sub>5</sub>, 394.1793; found, 394.1794.</div></div><div id="sec4_1_61" class="NLM_sec NLM_sec_level_3"><div id="ac_i86" class="anchor-spacer"></div><h4 class="article-section__title" id="_i86">4.1.61.  5-Chloro-<i>N</i><sup>2</sup>-(4-(1-methylazepan-4-yl)phenyl)-<i>N</i><sup>4</sup>-(<i>o</i>-tolyl)pyrimidine-2,4-diamine (<b>33</b>)</h4><div class="NLM_p last">White solid; yield 32%; mp 119.1â120.2 Â°C; <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): Î´ 8.00 (s, 1H), 7.92 (d, <i>J</i> = 8.1 Hz, 1H), 7.28 (d, <i>J</i> = 9.0 Hz, 2H), 7.24â7.18 (m, 2H), 7.11 (td, <i>J</i> = 7.4, 1.2 Hz, 1H), 6.90 (s, 1H), 6.83 (s, 1H), 6.59 (d, <i>J</i> = 9.0 Hz, 2H), 3.66â3.60 (m, 2H), 3.47 (t, <i>J</i> = 6.5 Hz, 2H), 2.90â2.84 (m, 2H), 2.79â2.69 (m, 2H), 2.51 (s, 3H), 2.32 (s, 2H), 2.21â2.11 (m, 2H). <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): Î´ 158.58, 156.04, 154.01, 145.33, 136.31, 130.47, 130.15, 129.07, 126.41, 124.80, 123.79, 122.41 (2C), 112.00 (2C), 104.43, 58.01, 56.69, 48.05, 47.73, 45.90, 26.63, 17.85; HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>23</sub>H<sub>27</sub>C<sub>l</sub>N<sub>6</sub>, 423.2058; found, 423.2058.</div></div><div id="sec4_1_62" class="NLM_sec NLM_sec_level_3"><div id="ac_i87" class="anchor-spacer"></div><h4 class="article-section__title" id="_i87">4.1.62.  <i>N</i><sup>4</sup>-Benzyl-5-chloro-<i>N</i><sup>2</sup>-(5-(4-methylpiperazin-1-yl)pyridin-2-yl)pyrimidine-2,4-diamine (<b>35</b>)</h4><div class="NLM_p last">White solid; yield 55%; mp 177â178 Â°C; <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): Î´ 8.04 (d, <i>J</i> = 9.1 Hz, 1H), 8.00 (s, 1H), 7.99 (d, <i>J</i> = 2.3 Hz, 1H), 7.96 (s, 1H), 7.38â7.34 (m, 4H), 7.33â7.29 (m, 1H), 7.20 (dd, <i>J</i> = 9.1, 2.7 Hz, 1H), 5.60 (t, <i>J</i> = 4.9 Hz, 1H), 4.71 (d, <i>J</i> = 5.7 Hz, 2H), 3.16â3.12 (m, 4H), 2.60â2.56 (m, 4H), 2.35 (s, 3H); <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): Î´ 157.68, 157.21, 153.22, 146.38, 142.61, 138.13, 136.55, 128.74 (2C), 127.59, 127.52 (2C), 126.37, 112.81, 105.07, 54.93 (2C), 49.73 (2C), 46.10, 45.00; HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>20</sub>H<sub>23</sub>ClN<sub>7</sub>, 409.6588; found, 409.6589.</div></div><div id="sec4_1_63" class="NLM_sec NLM_sec_level_3"><div id="ac_i88" class="anchor-spacer"></div><h4 class="article-section__title" id="_i88">4.1.63.  5-Chloro-<i>N</i><sup>2</sup>-(5-(4-methylpiperazin-1-yl)pyridin-2-yl)-<i>N</i><sup>4</sup>-(4-(trifluoromethyl)benzyl)pyrimidine-2,4-diamine (<b>36</b>)</h4><div class="NLM_p last">White solid; yield 50%; mp 181â182 Â°C; <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): Î´ 8.91 (s, 1H), 8.09 (s, 1H), 8.05 (d, <i>J</i> = 2.9 Hz, 1H), 7.89 (d, <i>J</i> = 9.1 Hz, 1H), 7.60 (d, <i>J</i> = 8.2 Hz, 2H), 7.45 (d, <i>J</i> = 8.1 Hz, 2H), 7.10 (dd, <i>J</i> = 9.1, 2.9 Hz, 1H), 5.75 (t, <i>J</i> = 5.9 Hz, 1H), 4.76 (d, <i>J</i> = 5.9 Hz, 2H), 3.18â3.08 (m, 4H), 2.62â2.55 (m, 4H), 2.35 (s, 3H); <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): Î´ 157.68, 157.31, 153.55, 146.39, 142.58, 142.48, 136.57, 129.68 (q, <i>J</i> = 32.5 Hz, 2C), 127.47, 126.15, 125.58 (q, <i>J</i> = 3.7 Hz, 2C), 124.05 (q, <i>J</i> = 272.1 Hz), 112.78, 104.84, 54.89 (2C), 49.63 (2C), 46.05, 44.45; HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>20</sub>H<sub>23</sub>ClFN<sub>7</sub>, 477.6488; found, 477.6489.</div></div><div id="sec4_1_64" class="NLM_sec NLM_sec_level_3"><div id="ac_i89" class="anchor-spacer"></div><h4 class="article-section__title" id="_i89">4.1.64.  5-Chloro-<i>N</i><sup>4</sup>-(4-methylbenzyl)-<i>N</i><sup>2</sup>-(5-(4-methylpiperazin-1-yl)pyridin-2-yl)pyrimidine-2,4-diamine (<b>37</b>)</h4><div class="NLM_p last">White solid; yield 40%; mp 181â182 Â°C; <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): Î´ 8.09 (d, <i>J</i> = 9.0 Hz, 1H), 8.08 (s, 1H), 8.00 (s, 1H), 7.99 (s, 1H), 7.25 (d, <i>J</i> = 7.9 Hz, 2H), 7.22 (dd, <i>J</i> = 9.1, 3.0 Hz, 1H), 7.17 (d, <i>J</i> = 7.8 Hz, 2H), 5.56 (t, <i>J</i> = 5.5 Hz, 1H), 4.66 (d, <i>J</i> = 5.6 Hz, 2H), 3.27â3.06 (m, 4H), 2.74â2.51 (m, 4H), 2.36 (s, 3H), 2.35 (s, 3H); <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>) 157.62, 157.25, 153.33, 146.40, 142.63,136.60, 133.95 (d, <i>J</i> = 3.3 Hz, 2C), 129.17 (d, <i>J</i> = 8.1 Hz, 2C), 126.30, 115.65, 115.50, 112.81, 105.00, 54.92 (2C), 49.70 (2C), 46.07, 44.27; HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>22</sub>H<sub>26</sub>ClN<sub>7</sub>, 424.2010; found, 424.2010.</div></div><div id="sec4_1_65" class="NLM_sec NLM_sec_level_3"><div id="ac_i90" class="anchor-spacer"></div><h4 class="article-section__title" id="_i90">4.1.65.  5-Chloro-<i>N</i><sup>4</sup>-(4-fluorobenzyl)-<i>N</i><sup>2</sup>-(5-(4-methylpiperazin-1-yl)pyridin-2-yl)pyrimidine-2,4-diamine (<b>38</b>)</h4><div class="NLM_p last">White solid; yield 54%; mp 177â178 Â°C; <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): Î´ 8.25 (s, 1H), 8.04 (d, <i>J</i> = 9.1 Hz, 1H), 8.02 (s, 1H), 8.01 (d, <i>J</i> = 2.9 Hz, 1H), 7.36â7.30 (m, 2H), 7.20 (dd, <i>J</i> = 9.1, 3.0 Hz, 1H), 7.04 (t, <i>J</i> = 8.6 Hz, 2H), 5.59 (t, <i>J</i> = 5.6 Hz, 1H), 4.67 (d, <i>J</i> = 5.7 Hz, 2H), 3.39â3.02 (m, 4H), 2.83â2.55 (m, 4H), 2.36 (s, 3H); <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): Î´ 162.34, 157.62, 157.25, 153.33, 146.40, 142.63, 136.60, 133.96, 129.20, 129.14, 126.30, 115.65, 115.50, 112.81, 105.00, 54.92 (2C), 49.70 (2C), 46.07, 44.27; HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>21</sub>H<sub>24</sub>ClFN<sub>7</sub>, 428.1760; found, 428.1760.</div></div><div id="sec4_1_66" class="NLM_sec NLM_sec_level_3"><div id="ac_i91" class="anchor-spacer"></div><h4 class="article-section__title" id="_i91">4.1.66.  5-Chloro-<i>N</i><sup>4</sup>-(4-methoxybenzyl)-<i>N</i><sup>2</sup>-(5-(4-methylpiperazin-1-yl)pyridin-2-yl)pyrimidine-2,4-diamine (<b>39</b>)</h4><div class="NLM_p last">White solid; yield 55%; mp 146â147 Â°C; <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): Î´ 8.42 (s, 1H), 8.12 (d, <i>J</i> = 9.1 Hz, 1H), 8.04 (d, <i>J</i> = 2.9 Hz, 1H), 8.02 (s, 1H), 7.29 (d, <i>J</i> = 8.6 Hz, 2H), 7.23 (dd, <i>J</i> = 9.1, 3.0 Hz, 1H), 6.89 (d, <i>J</i> = 8.6 Hz, 2H), 5.53 (t, <i>J</i> = 5.5 Hz, 1H), 4.64 (d, <i>J</i> = 5.5 Hz, 2H), 3.80 (s, 3H), 3.24â3.02 (m, 4H), 2.68â2.49 (m, 4H), 2.36 (s, 3H); <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): Î´ 159.11, 157.61, 157.27, 153.17, 146.55, 142.54, 136.62, 130.14, 128.96 (2C), 126.40, 114.14 (2C), 112.84, 104.99, 55.30, 54.91 (2C), 49.71 (2C), 46.05, 44.54; HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>22</sub>H<sub>26</sub>ClN<sub>7</sub>O, 440.1960; found, 440.1960.</div></div><div id="sec4_1_67" class="NLM_sec NLM_sec_level_3"><div id="ac_i92" class="anchor-spacer"></div><h4 class="article-section__title" id="_i92">4.1.67.  5-Chloro-<i>N</i><sup>4</sup>-(3-methoxybenzyl)-<i>N</i><sup>2</sup>-(5-(4-methylpiperazin-1-yl)pyridin-2-yl)pyrimidine-2,4-diamine (<b>40</b>)</h4><div class="NLM_p last">White solid; yield 55%; mp 162â163 Â°C; <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): Î´ 8.24 (s, 1H), 8.02 (m, 3H), 7.28 (t, <i>J</i> = 7.9 Hz, 1H), 7.21 (dd, <i>J</i> = 9.1, 2.9 Hz, 1H), 6.95 (d, <i>J</i> = 7.6 Hz, 1H), 6.91 (s, 1H), 6.84 (d, <i>J</i> = 9.7 Hz, 1H), 5.63 (t, <i>J</i> = 5.6 Hz, 1H), 4.68 (d, <i>J</i> = 5.7 Hz, 2H), 3.79 (s, 3H), 3.20â3.10 (m, 4H), 2.65â2.56 (m, 4H), 2.37 (s, 3H); <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): Î´ 159.91, 157.67, 157.23, 153.28, 146.41, 142.59, 139.81, 136.56, 129.79, 126.31, 119.68, 113.32, 112.86, 112.75, 105.02, 55.22, 54.93 (2C), 49.70 (2C), 46.08, 44.97; HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>22</sub>H<sub>26</sub>ClN<sub>7</sub>O, 440.1960; found, 440.1960.</div></div><div id="sec4_1_68" class="NLM_sec NLM_sec_level_3"><div id="ac_i93" class="anchor-spacer"></div><h4 class="article-section__title" id="_i93">4.1.68.  5-Chloro-<i>N</i><sup>4</sup>-(2-methoxybenzyl)-<i>N</i><sup>2</sup>-(5-(4-methylpiperazin-1-yl)pyridin-2-yl)pyrimidine-2,4-diamine (<b>41</b>)</h4><div class="NLM_p last">White solid; yield 50%; mp 156â157 Â°C; <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): Î´ 8.16 (d, <i>J</i> = 9.1 Hz, 1H), 7.99 (d, <i>J</i> = 2.9 Hz, 1H), 7.94 (s, 1H), 7.82 (s, 1H), 7.33â7.22 (m, 3H), 6.95â6.87 (m, 2H), 5.83 (t, <i>J</i> = 5.8 Hz, 1H), 4.70 (d, <i>J</i> = 5.9 Hz, 2H), 3.90 (s, 3H), 3.32â3.10 (m, 4H), 2.91â2.55 (m, 4H), 2.36 (s, 3H); <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): Î´ 157.69, 157.63, 157.17, 152.90, 146.58, 142.57, 136.66, 129.48, 128.92, 126.38, 126.09, 120.61, 112.82, 110.40, 105.37, 55.36, 54.94 (2C), 49.78 (2C), 46.07, 40.82; HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>22</sub>H<sub>26</sub>ClN<sub>7</sub>O, 440.1960; found, 440.1960.</div></div><div id="sec4_1_69" class="NLM_sec NLM_sec_level_3"><div id="ac_i94" class="anchor-spacer"></div><h4 class="article-section__title" id="_i94">4.1.69.  5-Chloro-<i>N</i><sup>2</sup>-(5-(4-methylpiperazin-1-yl)pyridin-2-yl)-<i>N</i><sup>4</sup>-phenethylpyrimidine-2,4-diamine (<b>42</b>)</h4><div class="NLM_p last">White solid; yield 55%; mp 208â210 Â°C; <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): Î´ 8.19 (d, <i>J</i> = 9.1 Hz, 1H), 8.00 (d, <i>J</i> = 2.6 Hz, 1H), 7.95 (s, 1H), 7.87 (s, 1H), 7.33 (t, <i>J</i> = 7.5 Hz, 2H), 7.25 (s, 1H), 7.24 (d, <i>J</i> = 3.2 Hz, 1H), 7.22 (d, <i>J</i> = 7.3 Hz, 2H), 5.35 (t, <i>J</i> = 5.8 Hz, 1H), 3.76 (dd, <i>J</i> = 13.6, 6.8 Hz, 2H), 3.19â3.09 (m, 4H), 2.95 (t, <i>J</i> = 7.2 Hz, 2H), 2.64â2.54 (m, 4H), 2.36 (s, 3H); <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): Î´ 157.69, 157.23, 153.00, 146.49, 142.62, 138.72, 136.69, 128.80 (2C), 128.69 (2C), 126.61, 126.37, 112.76, 105.21, 54.96 (2C), 49.77 (2C), 46.14, 42.33, 35.53; HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>22</sub>H<sub>27</sub>ClN<sub>7</sub>, 424.2011; found, 424.2011.</div></div><div id="sec4_1_70" class="NLM_sec NLM_sec_level_3"><div id="ac_i95" class="anchor-spacer"></div><h4 class="article-section__title" id="_i95">4.1.70.  5-Chloro-<i>N</i><sup>4</sup>-(4-fluorophenethyl)-<i>N</i><sup>2</sup>-(5-(4-methylpiperazin-1-yl)pyridin-2-yl)pyrimidine-2,4-diamine (<b>43</b>)</h4><div class="NLM_p last">White solid; yield 50%; mp 240â241 Â°C; <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): Î´ 8.18 (d, <i>J</i> = 9.1 Hz, 1H), 7.98 (d, <i>J</i> = 2.9 Hz, 1H), 7.94 (s, 1H), 7.59 (s, 1H), 7.25 (d, <i>J</i> = 3.0 Hz, 1H), 7.18 (dd, <i>J</i> = 8.5, 5.5 Hz, 2H), 7.07â6.96 (m, 2H), 5.33 (t, <i>J</i> = 5.7 Hz, 1H), 3.79â3.67 (m, 2H), 3.20â3.13 (m, 4H), 2.93 (t, <i>J</i> = 7.2 Hz, 2H), 2.63â2.57 (m, 4H), 2.38 (s, 3H); <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): Î´ 161.67, 157.69, 157.18, 153.00, 146.34, 142.69, 136.65, 134.36, 130.19, 130.14, 126.28, 115.54, 115.40, 112.74, 105.37, 54.91 (2C), 49.68 (2C), 46.05, 42.31, 34.70; HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>22</sub>H<sub>26</sub>ClFN<sub>7</sub>, 424.1917; found, 424.1918.</div></div><div id="sec4_1_71" class="NLM_sec NLM_sec_level_3"><div id="ac_i96" class="anchor-spacer"></div><h4 class="article-section__title" id="_i96">4.1.71.  5-Chloro-<i>N</i><sup>4</sup>-(2-fluorophenethyl)-<i>N</i><sup>2</sup>-(5-(4-methylpiperazin-1-yl)pyridin-2-yl)pyrimidine-2,4-diamine (<b>44</b>)</h4><div class="NLM_p last">White solid; yield 56%; mp 198â200 Â°C; <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): Î´ 8.20 (d, <i>J</i> = 9.1 Hz, 1H), 8.06 (s, 1H), 8.02 (d, <i>J</i> = 2.8 Hz, 1H), 7.97 (s, 1H), 7.28 (dd, <i>J</i> = 9.1, 2.8 Hz, 1H), 7.25â7.17 (m, 2H), 7.11â7.02 (m, 2H), 5.41 (t, <i>J</i> = 5.8 Hz, 1H), 3.78 (dd, <i>J</i> = 13.3, 6.8 Hz, 2H), 3.19â3.13 (m, 4H), 3.01 (t, <i>J</i> = 7.1 Hz, 2H), 2.67â2.57 (m, 4H), 2.36 (s, 3H); <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): Î´ 161.24, 157.75, 157.24, 153.03, 146.52, 142.63, 136.70, 131.15, 128.46, 126.36, 125.70, 124.21, 115.43, 112.78, 105.15, 54.96 (2C), 49.78 (2C), 46.10, 41.10, 29.24; HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>22</sub>H<sub>26</sub>ClFN<sub>7</sub>, 442.1917; found, 442.1918.</div></div><div id="sec4_1_72" class="NLM_sec NLM_sec_level_3"><div id="ac_i97" class="anchor-spacer"></div><h4 class="article-section__title" id="_i97">4.1.72.  5-Chloro-<i>N</i><sup>4</sup>-(4-methoxyphenethyl)-<i>N</i><sup>2</sup>-(5-(4-methylpiperazin-1-yl)pyridin-2-yl)pyrimidine-2,4-diamine (<b>45</b>)</h4><div class="NLM_p last">White solid; yield 45%; mp 205â206 Â°C; <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): Î´ 8.20 (d, <i>J</i> = 9.1 Hz, 1H), 8.01 (d, <i>J</i> = 2.7 Hz, 2H), 7.95 (s, 1H), 7.26 (dd, <i>J</i> = 9.1, 2.9 Hz, 1H), 7.14 (d, <i>J</i> = 8.5 Hz, 2H), 6.87 (d, <i>J</i> = 8.6 Hz, 2H), 5.35 (t, <i>J</i> = 5.7 Hz, 1H), 3.80 (s, 3H), 3.72 (dd, <i>J</i> = 13.3, 6.8 Hz, 2H), 3.25â3.06 (m, 4H), 2.90 (t, <i>J</i> = 7.1 Hz, 2H), 2.66â2.55 (m, 4H), 2.37 (s, 3H); <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): Î´ 158.33, 157.72, 157.25, 152.95, 146.55, 142.56, 136.67, 130.69, 129.71 (2C), 126.38, 114.09 (2C), 112.77, 105.23, 55.26, 54.90 (2C), 49.69 (2C), 46.02, 42.47, 34.60; HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>23</sub>H<sub>29</sub>ClN<sub>7</sub>O, 454.2117; found, 454.2117.</div></div><div id="sec4_1_73" class="NLM_sec NLM_sec_level_3"><div id="ac_i98" class="anchor-spacer"></div><h4 class="article-section__title" id="_i98">4.1.73.  5-Chloro-<i>N</i><sup>4</sup>-(furan-2-ylmethyl)-<i>N</i><sup>2</sup>-(5-(4-methylpiperazin-1-yl)pyridin-2-yl)pyrimidine-2,4-diamine (<b>46</b>)</h4><div class="NLM_p last">White solid; yield 40%; mp 190â191 Â°C; <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): Î´ 8.21 (s, 1H), 8.15 (d, <i>J</i> = 9.1 Hz, 1H), 8.01 (s, 1H), 7.99 (s, 1H), 7.38 (s, 1H), 7.33â7.24 (m, 1H), 6.33 (s, 1H), 6.27 (s, 1H), 5.57 (s, 1H), 4.68 (d, <i>J</i> = 5.4 Hz, 2H), 3.16 (s, 4H), 2.60 (s, 4H), 2.35 (s, 3H); <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): Î´ 157.43, 157.18, 153.34, 151.24, 146.50, 142.60, 142.27, 136.59, 126.37, 112.91, 110.47, 107.59, 105.10, 54.89 (2C), 49.67 (2C), 46.01, 38.04; HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>19</sub>H<sub>22</sub>ClN<sub>7</sub>O, 400.1647; found, 400.1647.</div></div><div id="sec4_1_74" class="NLM_sec NLM_sec_level_3"><div id="ac_i99" class="anchor-spacer"></div><h4 class="article-section__title" id="_i99">4.1.74.  5-Chloro-<i>N</i><sup>2</sup>-(5-(4-methylpiperazin-1-yl)pyridin-2-yl)-<i>N</i><sup>4</sup>-(<i>p</i>-tolyl)pyrimidine-2,4-diamine (<b>47</b>)</h4><div class="NLM_p last">White solid; yield 38%; mp 171â172 Â°C; <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): Î´ 8.28 (s, 1H), 8.10 (s, 1H), 8.05 (d, <i>J</i> = 9.1 Hz, 1H), 8.00 (d, <i>J</i> = 2.9 Hz, 1H), 7.44 (d, <i>J</i> = 8.3 Hz, 2H), 7.19â7.14 (m, 3H), 7.01 (s, 1H), 3.21â3.08 (m, 4H), 2.75â2.55 (m, 5H), 2.36 (s, 3H), 2.35 (s, 3H); <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): Î´ 156.96, 155.75, 154.06, 146.34, 142.64, 136.61, 135.13, 134.14, 129.37 (2C), 126.41 (2C), 122.06, 113.23, 105.24, 54.92 (2C), 49.71 (2C), 46.09, 21.00; HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>21</sub>H<sub>25</sub>ClN<sub>7</sub>, 410.1854; found, 410.1853.</div></div><div id="sec4_1_75" class="NLM_sec NLM_sec_level_3"><div id="ac_i100" class="anchor-spacer"></div><h4 class="article-section__title" id="_i100">4.1.75.  5-Chloro-<i>N</i><sup>2</sup>-(5-(4-methylpiperazin-1-yl)pyridin-2-yl)-<i>N</i><sup>4</sup>-(<i>m</i>-tolyl)pyrimidine-2,4-diamine (<b>48</b>)</h4><div class="NLM_p last">White solid; yield 42%; mp 191â192 Â°C; <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): Î´ 8.12 (s, 2H), 8.09 (d, <i>J</i> = 9.1 Hz, 1H), 8.01 (d, <i>J</i> = 2.9 Hz, 1H), 7.43 (s, 1H), 7.38 (d, <i>J</i> = 8.0 Hz, 1H), 7.26 (t, <i>J</i> = 7.8 Hz, 1H), 7.19 (dd, <i>J</i> = 9.1, 3.0 Hz, 1H), 7.03 (s, 1H), 6.99 (d, <i>J</i> = 7.5 Hz, 1H), 3.23â3.02 (m, 4H), 2.65â2.56 (m, 4H), 2.38 (s, 3H), 2.37 (s, 3H); <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): Î´ 156.90, 155.65, 154.20, 146.23, 142.67, 138.78, 137.63, 136.68, 128.68, 126.38, 125.25, 122.44, 118.99, 113.16, 105.35, 54.88 (2C), 49.64 (2C), 46.04, 21.56; HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>21</sub>H<sub>25</sub>ClN<sub>7</sub>, 410.1854; found, 410.1853.</div></div><div id="sec4_1_76" class="NLM_sec NLM_sec_level_3"><div id="ac_i101" class="anchor-spacer"></div><h4 class="article-section__title" id="_i101">4.1.76.  5-Chloro-<i>N</i><sup>4</sup>-(4-ethylphenyl)-<i>N</i><sup>2</sup>-(5-(4-methylpiperazin-1-yl)pyridin-2-yl)pyrimidine-2,4-diamine (<b>49</b>)</h4><div class="NLM_p last">White solid; yield 46%; mp 169â170 Â°C; <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): Î´ 8.35 (s, 1H), 8.11 (s, 1H), 8.04 (d, <i>J</i> = 9.1 Hz, 1H), 8.01 (d, <i>J</i> = 2.8 Hz, 1H), 7.46 (d, <i>J</i> = 8.3 Hz, 2H), 7.19 (d, <i>J</i> = 8.3 Hz, 2H), 7.15 (dd, <i>J</i> = 9.1, 2.9 Hz, 1H), 7.01 (s, 1H), 3.19â3.07 (m, 4H), 2.67 (q, <i>J</i> = 7.6 Hz, 2H), 2.61â2.55 (m, 4H), 2.35 (s, 3H), 1.26 (t, <i>J</i> = 7.6 Hz, 3H); <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): Î´ 157.02, 155.86, 154.12, 146.35, 142.67, 140.64, 136.67, 135.35, 128.15 (2C), 126.21, 122.36 (2C), 113.24, 105.17, 54.94 (2C), 49.71 (2C), 46.07, 28.35, 15.73; HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>22</sub>H<sub>26</sub>ClN<sub>7</sub>, 424.2011; found, 424.2011.</div></div><div id="sec4_1_77" class="NLM_sec NLM_sec_level_3"><div id="ac_i102" class="anchor-spacer"></div><h4 class="article-section__title" id="_i102">4.1.77.  5-Chloro-<i>N</i><sup>4</sup>-(4-isopropylphenyl)-<i>N</i><sup>2</sup>-(5-(4-methylpiperazin-1-yl)pyridin-2-yl)pyrimidine-2,4-diamine (<b>50</b>)</h4><div class="NLM_p last">White solid; yield 45%; mp 165â166 Â°C; <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): Î´ 8.30 (s, 1H), 8.11 (s, 1H), 8.03 (d, <i>J</i> = 9.1 Hz, 1H), 8.00 (d, <i>J</i> = 2.7 Hz, 1H), 7.46 (d, <i>J</i> = 8.4 Hz, 2H), 7.22 (d, <i>J</i> = 8.4 Hz, 2H), 7.13 (dd, <i>J</i> = 9.1, 2.8 Hz, 1H), 7.01 (s, 1H), 3.17â3.09 (m, 4H), 2.93 (dt, <i>J</i> = 13.8, 6.9 Hz, 1H), 2.71â2.53 (m, 4H), 2.35 (s, 3H), 1.28 (d, <i>J</i> = 6.9 Hz, 6H); <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): Î´ 157.02, 155.93, 154.15, 146.34, 145.35, 142.66, 136.69, 135.38, 126.70 (2C), 126.13, 122.54 (2C), 113.25, 105.13, 54.93 (2C), 49.70 (2C), 46.07, 33.62, 24.07 (2C); HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>23</sub>H<sub>29</sub>ClN<sub>7</sub>: 438.2167; found, 438.2167.</div></div><div id="sec4_1_78" class="NLM_sec NLM_sec_level_3"><div id="ac_i103" class="anchor-spacer"></div><h4 class="article-section__title" id="_i103">4.1.78.  5-Chloro-<i>N</i><sup>4</sup>-(3-isopropylphenyl)-<i>N</i><sup>2</sup>-(5-(4-methylpiperazin-1-yl)pyridin-2-yl)pyrimidine-2,4-diamine (<b>51</b>)</h4><div class="NLM_p last">White solid; yield 58%; mp 159â160 Â°C; <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): Î´ 8.39 (s, 1H), 8.14 (s, 1H), 8.06 (d, <i>J</i> = 9.1 Hz, 1H), 8.02 (d, <i>J</i> = 2.9 Hz, 1H), 7.49 (d, <i>J</i> = 7.9 Hz, 1H), 7.32 (s, 1H), 7.30 (t, <i>J</i> = 7.8 Hz, 1H), 7.16 (dd, <i>J</i> = 9.1, 2.8 Hz, 1H), 7.05 (d, <i>J</i> = 7.6 Hz, 2H), 3.18â3.09 (m, 4H), 2.90 (dp, <i>J</i> = 13.8, 6.8 Hz, 1H), 2.63â2.53 (m, 4H), 2.35 (s, 3H), 1.26 (d, <i>J</i> = 6.9 Hz, 6H); <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): Î´ 157.05, 155.77, 154.23, 149.89, 146.29, 142.68, 137.72, 136.67, 128.75, 126.25, 122.49, 120.10, 119.67, 113.16, 105.24, 54.94 (2C), 49.68 (2C), 46.08, 34.07, 23.97 (2C); HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>23</sub>H<sub>29</sub>ClN<sub>7</sub>, 438.2167; found, 438.2167.</div></div><div id="sec4_1_79" class="NLM_sec NLM_sec_level_3"><div id="ac_i104" class="anchor-spacer"></div><h4 class="article-section__title" id="_i104">4.1.79.  5-Chloro-<i>N</i><sup>4</sup>-(2-isopropylphenyl)-<i>N</i><sup>2</sup>-(5-(4-methylpiperazin-1-yl)pyridin-2-yl)pyrimidine-2,4-diamine (<b>52</b>)</h4><div class="NLM_p last">White solid; yield 45%; mp 184â185 Â°C; <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): Î´ 8.14 (s, 1H), 8.12 (s, 1H), 7.95 (d, <i>J</i> = 2.9 Hz, 1H), 7.78 (d, <i>J</i> = 9.1 Hz, 1H), 7.63 (d, <i>J</i> = 7.6 Hz, 1H), 7.38 (d, <i>J</i> = 7.6 Hz, 1H), 7.33â7.22 (m, 2H), 7.00 (dd, <i>J</i> = 9.1, 2.7 Hz, 1H), 6.92 (s, 1H), 3.19â3.00 (m, 5H), 2.64â2.52 (m, 4H), 2.36 (s, 3H), 1.23 (d, <i>J</i> = 6.9 Hz, 6H); <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): Î´ 157.02, 156.96, 154.18, 146.39, 142.96, 142.50, 136.43, 134.68, 126.67, 126.47, 126.30, 126.09, 125.91, 112.93, 104.92, 54.92 (2C), 49.71 (2C), 46.04, 28.27, 23.06 (2C); HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>23</sub>H<sub>29</sub>ClN<sub>7</sub>, 438.2167; found, 438.2167.</div></div><div id="sec4_1_80" class="NLM_sec NLM_sec_level_3"><div id="ac_i105" class="anchor-spacer"></div><h4 class="article-section__title" id="_i105">4.1.80.  <i>N</i><sup>4</sup>-(4-(<i>tert</i>-Butyl)phenyl)-5-chloro-<i>N</i><sup>2</sup>-(5-(4-methylpiperazin-1-yl)pyridin-2-yl)pyrimidine-2,4-diamine (<b>53</b>)</h4><div class="NLM_p last">White solid; yield 43%; mp 172â173 Â°C; <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): Î´ 8.37 (s, 1H), 8.11 (s, 1H), 8.04 (d, <i>J</i> = 9.1 Hz, 1H), 8.00 (d, <i>J</i> = 2.8 Hz, 1H), 7.47 (d, <i>J</i> = 8.5 Hz, 2H), 7.38 (d, <i>J</i> = 8.5 Hz, 2H), 7.13 (dd, <i>J</i> = 9.1, 2.7 Hz, 1H), 7.01 (s, 1H), 3.22â3.04 (m, 4H), 2.63â2.51 (m, 4H), 2.35 (s, 3H), 1.35 (s, 9H); <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): Î´ 157.01, 155.94, 154.15, 147.61, 146.31, 142.65, 136.66, 135.06, 126.08 (2C), 125.62, 122.24 (2C), 113.29, 105.16, 54.90 (2C), 49.65 (2C), 46.04, 34.43, 31.41 (3C); HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>24</sub>H<sub>31</sub>ClN<sub>7</sub>, 452.2324; found, 452.2324.</div></div><div id="sec4_1_81" class="NLM_sec NLM_sec_level_3"><div id="ac_i106" class="anchor-spacer"></div><h4 class="article-section__title" id="_i106">4.1.81.  5-Chloro-<i>N</i><sup>4</sup>-mesityl-<i>N</i><sup>2</sup>-(5-(4-methylpiperazin-1-yl)pyridin-2-yl)pyrimidine-2,4-diamine (<b>54</b>)</h4><div class="NLM_p last">White solid; yield 46%; mp 209â210 Â°C; <sup>1</sup>H NMR (600 MHz, cdcl<sub>3</sub>): Î´ 8.24 (s, 1H), 8.10 (s, 1H), 7.92 (d, <i>J</i> = 2.9 Hz, 1H), 7.53 (d, <i>J</i> = 8.9 Hz, 1H), 6.98 (s, 2H), 6.87 (dd, <i>J</i> = 9.1, 2.4 Hz, 1H), 6.50 (s, 1H), 3.18â2.98 (m, 4H), 2.70â2.52 (m, 4H), 2.36 (s, 3H), 2.35 (s, 3H), 2.17 (s, 6H); <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): Î´ 157.19, 157.16, 154.03, 146.71, 142.26, 136.82, 136.46, 136.19, 132.34, 128.81 (2C), 126.21 (2C), 112.55, 104.43, 54.94 (2C), 49.76 (2C), 46.08, 21.05, 18.38 (2C); HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>23</sub>H<sub>28</sub>ClN<sub>7</sub>, 438.2167; found, 438.2166.</div></div><div id="sec4_1_82" class="NLM_sec NLM_sec_level_3"><div id="ac_i107" class="anchor-spacer"></div><h4 class="article-section__title" id="_i107">4.1.82.  5-Chloro-<i>N</i><sup>4</sup>-(4-methoxyphenyl)-<i>N</i><sup>2</sup>-(5-(4-methylpiperazin-1-yl)pyridin-2-yl)pyrimidine-2,4-diamine (<b>55</b>)</h4><div class="NLM_p last">White solid; yield 59%; mp 174â176 Â°C; <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): Î´ 8.48 (s, 1H), 8.12 (s, 1H), 8.03 (s, 1H), 8.02 (d, <i>J</i> = 2.9 Hz, 1H), 7.45 (d, <i>J</i> = 8.9 Hz, 2H), 7.15 (dd, <i>J</i> = 9.1, 2.9 Hz, 1H), 6.96 (s, 1H), 6.91 (d, <i>J</i> = 8.9 Hz, 2H), 3.84 (s, 3H), 3.28â3.08 (m, 4H), 2.73â2.54 (m, 4H), 2.36 (s, 3H); <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): Î´ 156.98, 156.74, 156.01, 154.04, 146.35, 142.60, 136.53, 130.62, 126.33, 124.31, 113.98 (2C), 113.11 (2C), 104.98, 55.54, 54.94 (2C), 49.72 (2C), 46.12; HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>21</sub>H<sub>25</sub>ClN<sub>7</sub>O, 426.1804; found, 426.1806.</div></div><div id="sec4_1_83" class="NLM_sec NLM_sec_level_3"><div id="ac_i108" class="anchor-spacer"></div><h4 class="article-section__title" id="_i108">4.1.83.  5-Chloro-<i>N</i><sup>4</sup>-(3-methoxyphenyl)-<i>N</i><sup>2</sup>-(5-(4-methylpiperazin-1-yl)pyridin-2-yl)pyrimidine-2,4-diamine (<b>56</b>)</h4><div class="NLM_p last">White solid; yield 49%; mp 162â163 Â°C; <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): Î´ 8.59 (s, 1H), 8.14 (s, 1H), 8.09 (d, <i>J</i> = 9.1 Hz, 1H), 8.03 (d, <i>J</i> = 2.9 Hz, 1H), 7.26 (m, 2H), 7.19 (dd, <i>J</i> = 9.1, 3.0 Hz, 1H), 7.08 (dd, <i>J</i> = 7.9, 1.4 Hz, 1H), 7.05 (s, 1H), 6.70 (dd, <i>J</i> = 8.2, 2.0 Hz, 1H), 3.76 (s, 3H), 3.27â3.04 (m, 4H), 2.74â2.47 (m, 4H), 2.35 (s, 3H); <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): Î´ 160.04, 156.94, 155.58, 154.33, 146.17, 142.73, 138.89, 136.59, 129.53, 126.38, 114.11, 113.34, 109.90, 107.84, 105.31, 55.34, 54.91 (2C), 49.65 (2C), 46.10; HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>21</sub>H<sub>25</sub>ClN<sub>7</sub>O, 426.1804; found, 426.1806.</div></div><div id="sec4_1_84" class="NLM_sec NLM_sec_level_3"><div id="ac_i109" class="anchor-spacer"></div><h4 class="article-section__title" id="_i109">4.1.84.  5-Chloro-<i>N</i><sup>4</sup>-(2-methoxyphenyl)-<i>N</i><sup>2</sup>-(5-(4-methylpiperazin-1-yl)pyridin-2-yl)pyrimidine-2,4-diamine (<b>57</b>)</h4><div class="NLM_p last">White solid; yield 47%; mp 199â200 Â°C; <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): Î´ 8.43 (dd, <i>J</i> = 8.0, 1.1 Hz, 1H), 8.30 (s, 1H), 8.15 (d, <i>J</i> = 9.0 Hz, 1H), 8.14 (s, 1H), 8.04 (d, <i>J</i> = 2.9 Hz, 1H), 7.81 (s, 1H), 7.31â7.23 (m, 1H), 7.07 (td, <i>J</i> = 7.9, 1.5 Hz, 1H), 7.00â6.95 (m, 1H), 6.94 (dd, <i>J</i> = 8.1, 0.9 Hz, 1H), 3.93 (s, 3H), 3.26â3.12 (m, 4H), 2.67â2.52 (m, 4H), 2.37 (s, 3H); <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): Î´ 156.95, 155.33, 153.86, 148.83, 146.22, 142.77, 136.55, 127.66, 126.35, 123.46, 120.81, 120.53, 113.43, 110.08, 106.33, 55.86, 54.92 (2C), 49.67 (2C), 46.11; HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>21</sub>H<sub>25</sub>ClN<sub>7</sub>O, 426.1804; found, 426.1806.</div></div><div id="sec4_1_85" class="NLM_sec NLM_sec_level_3"><div id="ac_i110" class="anchor-spacer"></div><h4 class="article-section__title" id="_i110">4.1.85.  5-Chloro-<i>N</i><sup>4</sup>-(4-fluorophenyl)-<i>N</i><sup>2</sup>-(5-(4-methylpiperazin-1-yl)pyridin-2-yl)pyrimidine-2,4-diamine (<b>58</b>)</h4><div class="NLM_p last">White solid; yield 38%; mp 222â223 Â°C; <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): Î´ 8.10 (s, 1H), 8.00 (d, <i>J</i> = 8.8 Hz, 1H), 7.97 (s, 1H), 7.76 (s, 1H), 7.53 (s, 2H), 7.18 (d, <i>J</i> = 8.7 Hz, 1H), 7.09 (t, <i>J</i> = 8.0 Hz, 2H), 7.00 (s, 1H), 3.19 (s, 4H), 2.65 (s, 4H), 2.40 (s, 3H); <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): Î´ 159.76, 156.78, 155.79, 154.26, 146.01, 142.71, 136.55, 133.59, 126.34, 124.18, 115.65 (2C), 113.06 (2C), 105.30, 54.81 (2C), 49.48 (2C), 45.94; HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>20</sub>H<sub>22</sub>ClFN<sub>7</sub>, 414.1604; found, 414.1605.</div></div><div id="sec4_1_86" class="NLM_sec NLM_sec_level_3"><div id="ac_i111" class="anchor-spacer"></div><h4 class="article-section__title" id="_i111">4.1.86.  5-Chloro-<i>N</i><sup>4</sup>-(3-fluorophenyl)-<i>N</i><sup>2</sup>-(5-(4-methylpiperazin-1-yl)pyridin-2-yl)pyrimidine-2,4-diamine (<b>59</b>)</h4><div class="NLM_p last">White solid; yield 43%; mp 204â206 Â°C; <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): Î´ 8.14 (s, 1H), 8.04 (d, <i>J</i> = 9.1 Hz, 1H), 8.01 (d, <i>J</i> = 2.7 Hz, 2H), 7.68 (dt, <i>J</i> = 11.1, 2.2 Hz, 1H), 7.30 (td, <i>J</i> = 8.1, 6.6 Hz, 1H), 7.27â7.24 (m, 1H), 7.18 (dd, <i>J</i> = 9.1, 2.7 Hz, 1H), 7.12 (s, 1H), 6.85 (td, <i>J</i> = 8.2, 2.0 Hz, 1H), 3.19â3.13 (m, 4H), 2.63â2.57 (m, 4H), 2.37 (s, 3H); <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): Î´ 156.95, 155.32, 154.52, 145.69, 143.07, 139.43 (d, <i>J</i> = 11.0 Hz), 136.48, 129.89, 129.83, 126.25, 116.51, 113.59, 110.86 (d, <i>J</i> = 21.4 Hz), 108.86 (d, <i>J</i> = 26.4 Hz), 105.47, 54.90 (2C), 49.55 (2C), 46.08; HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>20</sub>H<sub>22</sub>ClFN<sub>7</sub>, 414.1604; found, 414.1605.</div></div><div id="sec4_1_87" class="NLM_sec NLM_sec_level_3"><div id="ac_i112" class="anchor-spacer"></div><h4 class="article-section__title" id="_i112">4.1.87.  5-Chloro-<i>N</i><sup>4</sup>-(2-fluorophenyl)-<i>N</i><sup>2</sup>-(5-(4-methylpiperazin-1-yl)pyridin-2-yl)pyrimidine-2,4-diamine (<b>60</b>)</h4><div class="NLM_p last">White solid; yield 55%; mp 192â193 Â°C; <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): Î´ 8.63 (s, 1H), 8.29 (t, <i>J</i> = 8.1 Hz, 1H), 8.19 (s, 1H), 8.07â8.02 (m, 2H), 7.27 (s, 1H), 7.21 (dd, <i>J</i> = 9.1, 3.0 Hz, 1H), 7.18â7.07 (m, 3H), 3.21â3.11 (m, 4H), 2.68â2.56 (m, 4H), 2.37 (s, 3H); <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): Î´ 156.96,153.72 (d, <i>J</i> = 244.4 Hz), 155.45, 154.47, 146.04, 142.86, 136.54, 126.43 (d, <i>J</i> = 10.2 Hz), 126.17,124.43 (d, <i>J</i> = 7.6 Hz), 124.05 (d, <i>J</i> = 3.8 Hz), 123.39,115.11 (d, <i>J</i> = 19.3 Hz) 113.36, 105.86, 54.92 (2C), 49.63 (2C), 46.10; HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>20</sub>H<sub>22</sub>ClFN<sub>7</sub>, 414.1604; found, 414.1605.</div></div><div id="sec4_1_88" class="NLM_sec NLM_sec_level_3"><div id="ac_i113" class="anchor-spacer"></div><h4 class="article-section__title" id="_i113">4.1.88.  5-Chloro-<i>N</i><sup>2</sup>-(5-(4-methylpiperazin-1-yl)pyridin-2-yl)-<i>N</i><sup>4</sup>-(4-(trifluoromethyl)phenyl)pyrimidine-2,4-diamine (<b>61</b>)</h4><div class="NLM_p last">White solid; yield 43%; mp 210â211 Â°C; <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): Î´ 8.65â8.59 (m, 1H), 8.18 (s, 1H), 8.04 (d, <i>J</i> = 2.9 Hz, 1H), 8.00 (d, <i>J</i> = 9.0 Hz, 1H), 7.72 (d, <i>J</i> = 8.5 Hz, 2H), 7.57 (d, <i>J</i> = 8.5 Hz, 2H), 7.20 (dd, <i>J</i> = 9.2, 3.0 Hz, 1H), 7.19 (s, 1H), 3.18â3.14 (m, 4H), 2.62â2.57 (m, 4H), 2.36 (s, 3H); <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): Î´ 156.84, 155.23, 154.75, 145.71, 143.02, 141.03, 136.49, 126.09 (q, <i>J</i> = 3.8 Hz, 4C), 125.74 (d, <i>J</i> = 32.5 Hz), 124.12 (q, <i>J</i> = 271.4 Hz), 120.88, 113.37, 105.63, 54.89, 49.52, 46.12; HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>21</sub>H<sub>22</sub>ClF<sub>3</sub>N<sub>7</sub>, 464.1572; found, 464.1572.</div></div><div id="sec4_1_89" class="NLM_sec NLM_sec_level_3"><div id="ac_i114" class="anchor-spacer"></div><h4 class="article-section__title" id="_i114">4.1.89.  5-Chloro-<i>N</i><sup>2</sup>-(5-(4-methylpiperazin-1-yl)pyridin-2-yl)-<i>N</i><sup>4</sup>-(4-(trifluoromethoxy)phenyl)pyrimidine-2,4-diamine (<b>62</b>)</h4><div class="NLM_p last">White solid; yield 45%; mp 206â207 Â°C; <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): Î´ 8.21 (s, 1H), 8.16 (s, 1H), 8.02 (d, <i>J</i> = 2.9 Hz, 1H), 7.99 (d, <i>J</i> = 9.1 Hz, 1H), 7.61 (d, <i>J</i> = 8.9 Hz, 2H), 7.23 (d, <i>J</i> = 8.5 Hz, 2H), 7.17 (dd, <i>J</i> = 9.1, 2.9 Hz, 1H), 7.07 (s, 1H), 3.19â3.15 (m, 4H), 2.65â2.59 (m, 4H), 2.38 (s, 3H); <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): Î´ 156.84, 155.56, 154.56, 145.89, 145.44, 142.84,136.51, 136.42, 126.06, 123.34, 121.63 (2C), 120.49 (q, <i>J</i> = 256.8 Hz), 113.26 (2C), 105.24, 54.86 (2C), 49.49 (2C), 46.06; HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>21</sub>H<sub>22</sub>ClF<sub>3</sub>N<sub>7</sub>O, 480.1521; found, 480.1521.</div></div><div id="sec4_1_90" class="NLM_sec NLM_sec_level_3"><div id="ac_i115" class="anchor-spacer"></div><h4 class="article-section__title" id="_i115">4.1.90.  5-Chloro-<i>N</i><sup>2</sup>-(5-(4-methylpiperazin-1-yl)pyridin-2-yl)-<i>N</i><sup>4</sup>-(3-(trifluoromethoxy)phenyl)pyrimidine-2,4-diamine (<b>63</b>)</h4><div class="NLM_p last">White solid; yield 55%; mp 225â226 Â°C; <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): Î´ 8.23 (s, 1H), 8.16 (s, 1H), 8.09â7.94 (m, 2H), 7.69 (s, 1H), 7.43 (d, <i>J</i> = 8.2 Hz, 1H), 7.35 (t, <i>J</i> = 8.2 Hz, 1H), 7.23 (dd, <i>J</i> = 9.1, 2.8 Hz, 1H), 7.13 (s, 1H), 7.01 (d, <i>J</i> = 8.2 Hz, 1H), 3.25â3.07 (m, 4H), 2.65â2.53 (m, 4H), 2.36 (s, 3H); <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): Î´ 156.97, 155.31, 154.66, 149.55, 145.75, 143.00, 139.37, 136.54, 129.81, 126.14, 120.43 (d, <i>J</i> = 257.5 Hz), 119.36, 115.92, 114.12, 113.36, 105.36, 54.89 (2C), 49.49 (2C), 46.06; HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>21</sub>H<sub>22</sub>ClF<sub>3</sub>N<sub>7</sub>O, 480.1521; found, 480.1521.</div></div><div id="sec4_1_91" class="NLM_sec NLM_sec_level_3"><div id="ac_i116" class="anchor-spacer"></div><h4 class="article-section__title" id="_i116">4.1.91.  5-Chloro-<i>N</i><sup>4</sup>-(4-fluoro-3-(trifluoromethyl)phenyl)-<i>N</i><sup>2</sup>-(5-(4-methylpiperazin-1-yl)pyridin-2-yl)pyrimidine-2,4-diamine (<b>64</b>)</h4><div class="NLM_p last">White solid; yield 50%; mp 177â178 Â°C; <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): Î´ 8.61 (s, 1H), 8.19 (s, 1H), 8.05 (d, <i>J</i> = 2.8 Hz, 1H), 7.92 (d, <i>J</i> = 9.0 Hz, 1H), 7.86 (dd, <i>J</i> = 6.0, 2.6 Hz, 1H), 7.73â7.70 (m, 1H), 7.21 (d, <i>J</i> = 9.3 Hz, 1H), 7.19 (dd, <i>J</i> = 9.0, 2.8 Hz, 1H), 7.10 (s, 1H), 3.20â3.14 (m, 4H), 2.65â2.59 (m, 4H), 2.38 (s, 3H); <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): Î´ 156.88, 155.44, 154.86, 145.73, 142.89, 136.58, 133.90 (d, <i>J</i> = 3.2 Hz), 127.32 (d, <i>J</i> = 8.2 Hz), 126.18, 122.28 (d, <i>J</i> = 273.6 Hz), 120.88, 118.53 (dd, <i>J</i> = 33.3, 13.4 Hz), 117.31 (d, <i>J</i> = 21.7 Hz), 113.04 (2C), 104.96, 54.86 (2C), 49.46 (2C), 46.06; HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>21</sub>H<sub>21</sub>ClF<sub>4</sub>N<sub>7</sub>, 482.1478; found, 482.1478.</div></div><div id="sec4_1_92" class="NLM_sec NLM_sec_level_3"><div id="ac_i117" class="anchor-spacer"></div><h4 class="article-section__title" id="_i117">4.1.92.  <i>N</i>-(4-((5-Chloro-2-((5-(4-methylpiperazin-1-yl)pyridin-2-yl)amino)pyrimidin-4-yl)amino)phenyl)acetamide (<b>65</b>)</h4><div class="NLM_p last">White solid; yield 40%; mp 252â253 Â°C; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ 9.95 (s, 1H), 9.31 (s, 1H), 8.81 (s, 1H), 8.11 (s, 1H), 7.94 (d, <i>J</i> = 2.9 Hz, 1H), 7.79 (d, <i>J</i> = 9.1 Hz, 1H), 7.62 (d, <i>J</i> = 8.9 Hz, 2H), 7.55 (d, <i>J</i> = 9.0 Hz, 2H), 7.20 (dd, <i>J</i> = 9.1, 3.0 Hz, 1H), 3.11â2.99 (m, 4H), 2.48â2.43 (m, 4H), 2.22 (s, 3H), 2.05 (s, 3H); <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ 168.46, 157.40, 156.41, 154.71, 146.04, 142.86, 135.97, 135.59, 134.16, 125.31, 124.21, 119.54 (2C), 114.16 (2C), 104.53, 54.92 (2C), 48.96 (2C), 46.22, 24.39; HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>22</sub>H<sub>26</sub>ClN<sub>8</sub>O, 453.1913; found, 453.1913.</div></div><div id="sec4_1_93" class="NLM_sec NLM_sec_level_3"><div id="ac_i118" class="anchor-spacer"></div><h4 class="article-section__title" id="_i118">4.1.93.  <i>N</i>-(3-((5-Chloro-2-((5-(4-methylpiperazin-1-yl)pyridin-2-yl)amino)pyrimidin-4-yl)amino)phenyl)acetamide (<b>66</b>)</h4><div class="NLM_p last">White solid; yield 55%; mp 220â221 Â°C; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ 9.94 (s, 1H), 9.19 (s, 1H), 8.94 (s, 1H), 8.16 (s, 1H), 7.95 (d, <i>J</i> = 3.0 Hz, 1H), 7.94 (s, 1H), 7.87 (d, <i>J</i> = 9.1 Hz, 1H), 7.42 (d, <i>J</i> = 7.6 Hz, 1H), 7.32 (d, <i>J</i> = 8.2 Hz, 1H), 7.29 (d, <i>J</i> = 7.8 Hz, 1H), 7.14 (dd, <i>J</i> = 9.2, 2.8 Hz, 1H), 3.12â2.99 (m, 4H), 2.49â2.43 (m, 4H), 2.23 (s, 3H), 2.05 (s, 3H); <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ 167.96, 156.60,155.79, 154.42, 145.27, 142.25, 139.10, 138.58, 135.09, 128.26, 124.56, 118.02,114.82, 114.19, 113.15, 104.15, 54.28, 48.31, 45.56, 23.87; HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>22</sub>H<sub>26</sub>ClN<sub>8</sub>O, 453.1913; found, 453.1913.</div></div><div id="sec4_1_94" class="NLM_sec NLM_sec_level_3"><div id="ac_i119" class="anchor-spacer"></div><h4 class="article-section__title" id="_i119">4.1.94.  <i>N</i>-(2-((5-Chloro-2-((5-(4-methylpiperazin-1-yl)pyridin-2-yl)amino)pyrimidin-4-yl)amino)phenyl)acetamide (<b>67</b>)</h4><div class="NLM_p last">White solid; yield 43%; mp 245â246 Â°C; <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): Î´ 8.10 (s, 1H), 7.93 (d, <i>J</i> = 2.8 Hz, 1H), 7.84 (d, <i>J</i> = 9.3 Hz, 2H), 7.75 (s, 1H), 7.74 (s, 1H), 7.67â7.64 (m, 1H), 7.50â7.46 (m, 1H), 7.30â7.27 (m, 2H), 7.09 (dd, <i>J</i> = 9.3, 2.8 Hz, 1H), 3.17â3.10 (m, 4H), 2.61 (s, 4H), 2.38 (s, 3H), 2.17 (s, 3H); <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): Î´ 169.51, 156.78, 156.53, 154.55, 145.89, 142.78, 136.37, 131.41, 131.29, 127.15, 126.45, 126.33, 126.20, 124.73, 112.89, 105.74, 54.89 (2C), 49.63 (2C), 46.11, 23.94; HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>22</sub>H<sub>26</sub>ClN<sub>8</sub>O, 453.1913; found, 453.1913.</div></div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i120" class="anchor-spacer"></div><h3 class="article-section__title" id="_i120">4.2.  Cell Culture and Viability Assay</h3><div class="NLM_p last">The MCF-7 and jurkat cells were maintained on RPMI 1640 medium (Hyclone, US) while MDA-MB-231 and L02 cells were cultured in DMEM medium (Hyclone, US). Both kinds of medium were supplemented with 10% fetal bovine serum (Biological Industries, Israel) 100 U/mL penicillin and 100 Î¼g/mL streptomycin in 25 cm<sup>2</sup> culture flasks at 37 Â°C humidified atmosphere with 5% CO<sub>2</sub>. Compounds were dissolved in DMSO and filtered through 0.22 Î¼m pore-size filter units before addition to cell line appropriate media. Cell viability was assessed by CCK-8 assay. For this assay, various cells were seeded in 96-well plates at 5000 cells/well, and 12 h later, the cells were treated with different concentrations of compounds for 48 h. Treated cells were incubated with 10% CCK-8 solution for 2 h at 37 Â°C. The absorbance was measured at 450 nm. The relative amount of inhibition of cell growth was calculated as follows<span class="NLM_disp-formula" id="ueq1"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02121/20200323/images/medium/jm9b02121_m001.gif" alt="" id="_i121" /></img></span></div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i122" class="anchor-spacer"></div><h3 class="article-section__title" id="_i122">4.3.  Kinase Assays</h3><div class="NLM_p last">The active CDK2/CyclinA2, CDK6/CyclinD3, and CDK9/CyclinK kinases were purchased from Promega, and the kinase inhibition was detected by ADP-Glo Kinase Assay (Promega, Madison, Wisconsin, USA) according to the manufacturerâs instructions. For CDK2/Cyclin A2 and CDK6/CyclinD3, the assay was against the substrate histone H1 with a final concentration of 0.1 mg/mL, and for CDK9/CyclinK, the assay was against the PDKtide substrate (KTFCGTPEYLAPEVRREPRILSEEEQEMFRDFDYIADWC) with a final concentration of 0.1 mg/mL also. The final ATP concentrations of CDK2/CyclinA2, CDK6/CyclinD3, and CDK9/CyclinK reactions are the <i>K</i><sub>m</sub> of corresponding kinases, which were 50, 15, and 10 Î¼M, respectively. Maximum detection concentrations of tested compounds were 10 Î¼M, 10 times gradient dilution. The kinase reaction of CDK2, CDK6, and CDK9 lasted for 10, 60, and 120 min, respectively, in room temperature and finally detected by a microplate reader (PerkinElmer, Waltham, Ma, USA). The positive control palbociclib (MedChemExpress), abemaciclib (MedChemExpress), and NVP-2 (GlpBio Technology) were diluted to 50 mM in DMSO and stored at â20 Â°C before use.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i123" class="anchor-spacer"></div><h3 class="article-section__title" id="_i123">4.4.  Cell Cycle</h3><div class="NLM_p last">The distribution of cells at specific cell cycle stages was evaluated by flow cytometry. MDA-MB-231 cells were cultured in six-well plates (5 Ã 10<sup>5</sup> cells/well) and treated with the title compound in a humidified incubator with 5% CO<sub>2</sub>, 37 Â°C for 48 h. The untreated cells served as controls. After incubation, cells were harvested, washed with phosphate-buffered saline (PBS) twice, and fixed with precooled ethanol for 1 h at â20 Â°C. Then, the cells were treated with RNase for 30 min, stained with propidium iodide (Bestbio) for 30 min in the dark, and subsequently analyzed by a flow cytometer (CytoFLEX, Beckman).</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i124" class="anchor-spacer"></div><h3 class="article-section__title" id="_i124">4.5.  Apoptosis</h3><div class="NLM_p last">For measuring apoptosis, MDA-MB-231 cells were seeded in six-well plates (5 Ã 10<sup>5</sup> cells/well) and treated with the title compound for 48 h. The untreated cells served as controls. After incubation, the cells were collected, washed twice with PBS, and resuspended in V-FITC buffer. Thereafter, the cells were stained with annexin V-FITC for 15 min and incubated with PI for 5 min in the dark. Prepared samples need to be tested immediately by flow cytometry.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i125" class="anchor-spacer"></div><h3 class="article-section__title" id="_i125">4.6.  Reactive Oxygen Species</h3><div class="NLM_p last">Intracellular ROS of MDA-MB-231 cells was detected by DCFH-DA (Beyotime). MDA-MB-231 cells were cultured in six-well plates (5 Ã 10<sup>5</sup> cells/well) and treated with the title compound for 48 h. The untreated cells were served as controls. After incubation, cells were harvested, washed with PBS twice, and stained with DCFH-DA probe. Thereafter, the cells were incubated at 37 Â°C for 20 min in the dark and analyzed immediately by flow cytometry.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i126" class="anchor-spacer"></div><h3 class="article-section__title" id="_i126">4.7.  Protein Expression and Purification</h3><div class="NLM_p last">All recombinant proteins were expressed in HEK293T cells using pcDNA3.1 vector. Six plasmids were constructed including pcDNA3.1-CDK6-his, pcDNA3.1-CyclinD3-his, pcDNA3.1-CDK9-his, pcDNA3.1-CyclinT1-his, pcDNA3.1-CDK6-GFP, and pcDNA3.1-CDK9-GFP. In the case of the activity of CDK depends on the existence of cyclin proteins, we co-expressed four pairs of recombinant proteins in Hek293T cells using lip2000-mediated transfection. After 48 h of culture, the cells were collected and lysed with lysis buffer (50 mM HEPES pH 7.5, 300 mM NaCl, 2 mM MgCl<sub>2</sub>, 1 mM DTT, 1% NP-40, 10% glycerol, 20 mM imidazole, 1 mM PMSF). Purification with Ni-6FF agarose beads (Solarbio, Beijing, China) and desalting with HiTrap Desalting 5 mL (GE healthcare, Marlborough, USA), the purified recombinant proteins were finally maintained in HEPES buffer (50 mM HEPES pH 7.5, 150 mM NaCl, 10 mM MgCl<sub>2</sub>, 0.5 mM DTT, 10% glycerol). The prepared proteins were used for the experiment directly or stored at â80 Â°C.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i127" class="anchor-spacer"></div><h3 class="article-section__title" id="_i127">4.8.  Cellular Thermal Shift Assay</h3><div class="NLM_p last">The active CDK6-his/CyclinD3-his and CDK9-his/CyclinT1-his proteins were diluted with HEPES buffer (50 mM HEPES pH 7.5, 150 mM NaCl, 10 mM MgCl<sub>2</sub>, 0.5 mM DTT, 10% glycerol) and divided into two groups in PCR tubes. One group mixed with 10 Î¼M compound <b>66</b> and the other mixed with 0.5% DMSO as a negative control. Both sets of samples were heated under the same conditions at temperatures ranging from 45 to 63Â°. Each sample was heated at a single temperature for 3 min, placed on ice, and centrifuged at 15,000<i>g</i> for 20 min. The supernatant was collected for WB testing.</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i128" class="anchor-spacer"></div><h3 class="article-section__title" id="_i128">4.9.  Microscale Thermophoresis</h3><div class="NLM_p last">The active full-length CDK6-GFP/CyclinD3-his and CDK9-GFP/CyclinT1 proteins were overexpressed in Hek293T cells. After purification and desalting, the recombinant proteins were used for the experiment directly or stored at â80 Â°C. The tested compounds were 2Ã diluted to various doses, starting at 50 Î¼M (CDK6) or 100 Î¼M (CDK9) and incubated with equivalent proteins (including 0.1% Tween-20) for 30 min at room temperature. After that, adding the sample to the capillary for detection by NanoTemper Monolith NT.115 (NanoTepmer Technologies, Munich, Germany).</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i129" class="anchor-spacer"></div><h3 class="article-section__title" id="_i129">4.10.  Western Blot</h3><div class="NLM_p last">The cells were plated in 60 mm culture dishes (Corning Life Sciences) at a density of 5 Ã 10<sup>5</sup> cells/dish. After 48 h of incubation with the title compound in different concentrations, the cells were washed with PBS once and harvested in 1.5 mL EP tubes. RIPA lysis buffer with 1 mM PMSF were used for resuspending the cells. Following incubation on ice for 30 min, the homogenate was centrifuged at 15,000<i>g</i> for 20 min at 4 Â°C. Protein concentrations were determined using a bicinchoninic acid assay (Beyotime Institute of Biotechnology, Beijing, China). An appropriate volume of 5Ã SDS loading buffer was added to the prepared cell lysate and thermal denaturation for 10 min in 95 Â°C. Subsequently, 50 Î¼g of protein was separated using 12% sodium dodecyl sulfate-polyacrylamide gel electrophoresis (SDS-PAGE) and transferred onto polyvinylidene difluoride membranes. The blotted membranes were blocked with 5% skim milk for 2 h and probed with primary antibodies overnight at 4 Â°C. After at least 8 h of incubation, the membranes were washed and probed with secondary antibodies for 1 h. After that, the membrane was cleaned thoroughly and exposed to the ChemiDoc MP Imaging System (Bio-Rad, Hercules, CA, USA). The antibodies used in this study are presented in <a href="/doi/suppl/10.1021/acs.jmedchem.9b02121/suppl_file/jm9b02121_si_001.pdf" class="ext-link">Table S2</a>, and Î²-actin was used as the loading control.</div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i130" class="anchor-spacer"></div><h3 class="article-section__title" id="_i130">4.11.  qRT-PCR</h3><div class="NLM_p last">Total RNA was isolated using TRIzol reagent (Invitrogen Life Technologies, Carlsbad, CA, USA) and transcribed to cDNA using PrimeScript Master Mix Kit (Takara, Kyoto, Japan). After quantitation and diluting to 200 ng/Î¼L with DEPC treated water, the cDNA was used as a template for PCR using the TB-Green Premix Ex Taq II kit (Takara, Kyoto, Japan) in a CFX Connect Real-Time System (Bio-rad, California, USA). The primers used in this study are presented in <a href="/doi/suppl/10.1021/acs.jmedchem.9b02121/suppl_file/jm9b02121_si_001.pdf" class="ext-link">Table S3</a>, and Î²-actin was selected as an internal control for each experiment. The cycling conditions were as follows: predenaturation at 95 Â°C for 30 s, followed by 40 cycles at 95 Â°C for 30 s, at 60 Â°C for 5 s, and at 60 Â°C for 30 s. The specificity of the amplification products was confirmed by a melting curve analysis. All reactions were run in triplicate. As a measure of relative change in expression between the parental and resistant samples, ÎÎ<i>C</i><sub><i>t</i></sub> values were calculated and converted to approximate fold change values (2<sup>âÎÎ<i>C</i><sub><i>t</i></sub></sup>).</div></div><div id="sec4_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i131" class="anchor-spacer"></div><h3 class="article-section__title" id="_i131">4.12.  Aqueous Solubility, LipidâWater Partition Coefficient, Caco-2 Permeability Determination, and Metabolic Stability in Hepatocyctes</h3><div class="NLM_p">We determined aqueous solubility using a miniaturized shake-flask method wherein the supernatant was analyzed by HPLC.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> Lipid-water partition coefficient were determined using a shake-flask method, and the concentrations of the test compound in the two phases were determined by HPLC. log <i>P</i> = log (concentration of drug in octanol/concentration of drug in aqueous solution).</div><div class="NLM_p">For caco-2 cells permeability assay, the tested compound was added to intestinal side or basolateral side inserts precoated with caco-2 cells, respectively. After incubating with caco-2 cells at 37 Â°C for 90 min, the receiver-side buffer was removed for analysis by LCâMS/MS. <i>P</i><sub>app</sub> = (d<i>Q</i>/d<i>t</i>)/(<i>C</i><sub>0</sub><i>A</i>), where d<i>Q</i>/d<i>t</i> is the rate of permeation, <i>C</i><sub>0</sub> is the initial concentration of test agent, and <i>A</i> is the area of the monolayer.</div><div class="NLM_p last">In the hepatocyte metabolism test, the prepared hepatocytes were added to a 24-well plate and mixed with the test compound or controls, followed by incubation at 37 Â°C. The reaction mixture was transferred into a new EP tube at 0 and 120 min, and methanol/ACN (1:1, v/v) containing predetermined IS was added to stop the reaction. After being vortexed for 1 min, the mixture was centrifuged at 4000 rpm for 15 min to finally obtain the supernatant for LCâMS/MS analysis.</div></div><div id="sec4_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i132" class="anchor-spacer"></div><h3 class="article-section__title" id="_i132">4.13.  In Vivo PK Study</h3><div class="NLM_p last">SD rats were received 2 or 10 mg/kg via intravenously injection (iv) or oral administration (po), respectively. The time points for blood sample collection were 0.08 (only for iv), 0.25, 0.5, 1, 2, 4, 8, and 24 h post dose (<i>n</i> = 3). For each point, 300 Î¼L of collected blood was transferred into a new EP tube containing 10 Î¼L of heparin as an anticoagulant and centrifuged at 4000 rpm for 10 min. Plasma (50 Î¼L) was separated and transferred to another EP tube, and 200 Î¼L of methanol containing predetermined IS was added to the plasma to precipitate plasma proteins. After being vortexed for 1 min, the mixture was centrifuged at 12,000<i>g</i> for 10 min to finally obtain the supernatant for LCâMS/MS analysis.</div></div><div id="sec4_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i133" class="anchor-spacer"></div><h3 class="article-section__title" id="_i133">4.14.  In Vivo Antitumor Assay</h3><div class="NLM_p last">The experimental procedures of the animal study were proved by the Animal Care and Use Committee at Anhui Medical University (no. LLSC20190275). MDA-MB-231 cells (5 Ã 10<sup>6</sup>) were unilaterally injected into the armpit of 6 weeks old female BALB/c nuânu mice (Shanghai SLAC Laboratory Animal Co., Ltd). After 10 days observation, mice that successfully modeled with MDA-MB-231 cells were randomly divided into five groups (<i>n</i> = 6) and received an oral administration of 22.5, 45, or 90 mg/kg compound <b>66</b>, 45 mg/kg abemaciclib as a positive control, or solvent as a negative control once every day during 3 weeks. Body weights and tumor size were determined every 3 days. The tumor volume was measured by a vernier caliper and calculated using the following formula: 0.5 Ã short Ã short Ã long. At the end of the experiment, the mice were sacrificed, and the tumor tissues were weighed and HE-stained.</div></div><div id="sec4_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i134" class="anchor-spacer"></div><h3 class="article-section__title" id="_i134">4.15.  Statistical Analysis</h3><div class="NLM_p last">The IC<sub>50</sub> of cell viability and kinase activity are expressed as mean Â± SD. The other ones are shown as mean Â± SEM. One-way ANVOA was used to determine the statistical significance at *<i>p</i> < 0.05, **<i>p</i> < 0.01, and ***<i>p</i> < 0.001. SPSS 17.0 and Graphpad Prism 5 software were used for statistical analyses.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i135"><a href="/doi/suppl/10.1021/acs.jmedchem.9b02121" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_68278" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_68278" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.9b02121?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b02121</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline"><sup>1</sup>H NMR and <sup>13</sup>C NMR spectra and purity table of target compounds (<a href="/doi/suppl/10.1021/acs.jmedchem.9b02121/suppl_file/jm9b02121_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings and some data (<a href="/doi/suppl/10.1021/acs.jmedchem.9b02121/suppl_file/jm9b02121_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b02121/suppl_file/jm9b02121_si_001.pdf">jm9b02121_si_001.pdf (11.03 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b02121/suppl_file/jm9b02121_si_002.csv">jm9b02121_si_002.csv (6.8 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.9b02121" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_63466" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_63466" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mingming Liu</span> - <span class="hlFld-Affiliation affiliation">School
of Pharmacy, Anhui Medical University, Hefei 230032, China</span>;Â 
    <span class="hlFld-Affiliation affiliation">Anhui
Chem-Bright Bioengineering Co., Ltd., Huaibei 235025, China</span>;Â 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#2c4041416c4d44415902494859024f42"><span class="__cf_email__" data-cfemail="d9b5b4b499b8b1b4acf7bcbdacf7bab7">[emailÂ protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xinhua Liu</span> - <span class="hlFld-Affiliation affiliation">School
of Pharmacy, Anhui Medical University, Hefei 230032, China</span>;Â 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-7984-2977" title="Orcid link">http://orcid.org/0000-0002-7984-2977</a>;Â 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#027a6a6e6b776a7a423334312c616d6f"><span class="__cf_email__" data-cfemail="1169797d78647969512027223f727e7c">[emailÂ protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yang Wang</span> - <span class="hlFld-Affiliation affiliation">School
of Pharmacy, Anhui Medical University, Hefei 230032, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xing Chen</span> - <span class="hlFld-Affiliation affiliation">School
of Pharmacy, Anhui Medical University, Hefei 230032, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yaoyao Yan</span> - <span class="hlFld-Affiliation affiliation">School
of Pharmacy, Anhui Medical University, Hefei 230032, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xiaochen Zhu</span> - <span class="hlFld-Affiliation affiliation">School
of Pharmacy, Anhui Medical University, Hefei 230032, China</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>Y.W. and X.C. are co-first authors and contributed equally to this work.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i137">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_86714" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_86714" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This study was supported grants from the Natural Science Foundation of China (21977001, 21572003) and by Natural Science Research Program of Anhui Educational Committee (KJ2019A0232) and by National studentsâ innovation and entrepreneurship training program (201910366045). Thanks to the Center for Scientific Research of Anhui Medical University for valuable help in our experiment.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">AML</td><td class="NLM_def"><p class="first last">acute myeloid leukemia</p></td></tr><tr><td class="NLM_term">CCK-8</td><td class="NLM_def"><p class="first last">Cell Counting Kit</p></td></tr><tr><td class="NLM_term">BINAP</td><td class="NLM_def"><p class="first last">(Â±)-2,2â²-bis(diphenylphosphino)-1,1â²-BINAPhthyl</p></td></tr><tr><td class="NLM_term">CDK</td><td class="NLM_def"><p class="first last">cyclin dependent kinase</p></td></tr><tr><td class="NLM_term">CDKI</td><td class="NLM_def"><p class="first last">cyclin dependent kinase inhibitor</p></td></tr><tr><td class="NLM_term">CETSA</td><td class="NLM_def"><p class="first last">cellular thermal shift assay</p></td></tr><tr><td class="NLM_term">DCFH-DA</td><td class="NLM_def"><p class="first last">2,7-dichlorodi-hydrofluorescein diacetate</p></td></tr><tr><td class="NLM_term">DMSO</td><td class="NLM_def"><p class="first last">dimethyl sulfoxide</p></td></tr><tr><td class="NLM_term">FITC</td><td class="NLM_def"><p class="first last">fluorescein isothiocyanate</p></td></tr><tr><td class="NLM_term">HPLC</td><td class="NLM_def"><p class="first last">high performance liquid chromatography</p></td></tr><tr><td class="NLM_term">MS</td><td class="NLM_def"><p class="first last">mass spectrum</p></td></tr><tr><td class="NLM_term">MST</td><td class="NLM_def"><p class="first last">microscale thermophoresis</p></td></tr><tr><td class="NLM_term">NADPH</td><td class="NLM_def"><p class="first last">nicotinamide adenine dinucleotide phosphate</p></td></tr><tr><td class="NLM_term">Na<i>O</i>tBu</td><td class="NLM_def"><p class="first last">sodium <i>tert</i>-butoxide</p></td></tr><tr><td class="NLM_term">PI</td><td class="NLM_def"><p class="first last">propidium iodide</p></td></tr><tr><td class="NLM_term">Pol II CTD</td><td class="NLM_def"><p class="first last">RNA polymerase II C-terminal domain</p></td></tr><tr><td class="NLM_term">p-Pol II CTD S2</td><td class="NLM_def"><p class="first last">RNA polymerase II C-terminal domain (phospho S2)</p></td></tr><tr><td class="NLM_term">qRT-PCR</td><td class="NLM_def"><p class="first last">qualitative real-time polymerase chain reaction</p></td></tr><tr><td class="NLM_term">ROS</td><td class="NLM_def"><p class="first last">reactive oxygen species</p></td></tr><tr><td class="NLM_term">Rb</td><td class="NLM_def"><p class="first last">retinoblastoma</p></td></tr><tr><td class="NLM_term">p-Rb</td><td class="NLM_def"><p class="first last">retinoblastoma (phospho S780)</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure activity relationships</p></td></tr><tr><td class="NLM_term">TBAI</td><td class="NLM_def"><p class="first last">tetrabutylammonium iodide</p></td></tr><tr><td class="NLM_term">XANTPHOS</td><td class="NLM_def"><p class="first last">(9,9-dimethyl-4,5-bis(diphenylphosphino)xanthene)</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i139">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_67871" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_67871" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 42 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Deshpande, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sicinski, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hinds, P. W.</span></span> <span> </span><span class="NLM_article-title">Cyclins and CDKs in Development and Cancer: a Perspective</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">2909</span>â <span class="NLM_lpage">2915</span>, <span class="refDoi">Â DOI: 10.1038/sj.onc.1208618</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=10.1038%2Fsj.onc.1208618" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=15838524" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=1%3ACAS%3A528%3ADC%252BD2MXjtlehsLw%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2005&pages=2909-2915&author=A.+Deshpandeauthor=P.+Sicinskiauthor=P.+W.+Hinds&title=Cyclins+and+CDKs+in+Development+and+Cancer%3A+a+Perspective&doi=10.1038%2Fsj.onc.1208618"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Cyclins and cdks in development and cancer: a perspective</span></div><div class="casAuthors">Deshpande, Amit; Sicinski, Peter; Hinds, Philip W.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">2909-2915</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  A fundamental aspect of cancer is dysregulated cell cycle control.  Unlike normal cells that only proliferate when compelled to do so by developmental or other mitogenic signals in response to tissue growth needs, the proliferation of cancer cells proceeds essentially unchecked.  This does not mean that cancer cell cycles are necessarily different from those found in normal cycling cells, but rather implies that cancer cells proliferate because they are no longer subject to proliferation-inhibitory influences arising from the stroma or from gene expression pattern changes consequent to 'terminal' differentiation, nor do they necessarily require extrinsic growth factors to recruit them into or maintain their proliferative state.  Finally, cancer cells have also often avoided normal controls linked to cell cycle progression that halt proliferation in the presence of damaged DNA or other physiol. insults.  The result of these alterations is the inappropriate proliferation commonly assocd. with cancerous tumor formation.  This review will summarize the current understanding of dysregulation of the G0/G1-to-S-phase transition in cancer cells, with particular emphasis on recent in vivo studies that suggest a need to rethink existing models of cell cycle control in development and tumorigenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdx5230em1arVg90H21EOLACvtfcHk0ljErSHzivPE-g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXjtlehsLw%253D&md5=84de90f5864255f0505413f1bdebe3a2</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fsj.onc.1208618&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.onc.1208618%26sid%3Dliteratum%253Aachs%26aulast%3DDeshpande%26aufirst%3DA.%26aulast%3DSicinski%26aufirst%3DP.%26aulast%3DHinds%26aufirst%3DP.%2BW.%26atitle%3DCyclins%2520and%2520CDKs%2520in%2520Development%2520and%2520Cancer%253A%2520a%2520Perspective%26jtitle%3DOncogene%26date%3D2005%26volume%3D24%26spage%3D2909%26epage%3D2915%26doi%3D10.1038%2Fsj.onc.1208618" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lim, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaldis, P.</span></span> <span> </span><span class="NLM_article-title">Cdks, Cyclins and CKIs: Roles Beyond Cell Cycle Regulation</span>. <i>Development</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>140</i></span>,  <span class="NLM_fpage">3079</span>â <span class="NLM_lpage">3093</span>, <span class="refDoi">Â DOI: 10.1242/dev.091744</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=10.1242%2Fdev.091744" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=23861057" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsFGktL%252FF" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=140&publication_year=2013&pages=3079-3093&author=S.+Limauthor=P.+Kaldis&title=Cdks%2C+Cyclins+and+CKIs%3A+Roles+Beyond+Cell+Cycle+Regulation&doi=10.1242%2Fdev.091744"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Cdks, cyclins and CKIs: roles beyond cell cycle regulation</span></div><div class="casAuthors">Lim, Shuhui; Kaldis, Philipp</div><div class="citationInfo"><span class="NLM_cas:title">Development (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">140</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">3079-3093</span>CODEN:
                <span class="NLM_cas:coden">DEVPED</span>;
        ISSN:<span class="NLM_cas:issn">0950-1991</span>.
    
            (<span class="NLM_cas:orgname">Company of Biologists Ltd.</span>)
        </div><div class="casAbstract">A review.  Cyclin-dependent kinases (Cdks) are serine/threonine kinases and their catalytic activities are modulated by interactions with cyclins and Cdk inhibitors (CKIs).  Close cooperation between this trio is necessary for ensuring orderly progression through the cell cycle.  In addn. to their well-established function in cell cycle control, it is becoming increasingly apparent that mammalian Cdks, cyclins and CKIs play indispensable roles in processes such as transcription, epigenetic regulation, metab., stem cell self-renewal, neuronal functions and spermatogenesis.  Even more remarkably, they can accomplish some of these tasks individually, without the need for Cdk/cyclin complex formation or kinase activity.  In this Review, we discuss the latest revelations about Cdks, cyclins and CKIs with the goal of showcasing their functional diversity beyond cell cycle regulation and their impact on development and disease in mammals.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqp5LNcD6NywLVg90H21EOLACvtfcHk0ljleJAcLoZBLQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsFGktL%252FF&md5=8095fc798a0e7efeab60d36ee414596f</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1242%2Fdev.091744&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1242%252Fdev.091744%26sid%3Dliteratum%253Aachs%26aulast%3DLim%26aufirst%3DS.%26aulast%3DKaldis%26aufirst%3DP.%26atitle%3DCdks%252C%2520Cyclins%2520and%2520CKIs%253A%2520Roles%2520Beyond%2520Cell%2520Cycle%2520Regulation%26jtitle%3DDevelopment%26date%3D2013%26volume%3D140%26spage%3D3079%26epage%3D3093%26doi%3D10.1242%2Fdev.091744" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Malumbres, M.</span></span> <span> </span><span class="NLM_article-title">Cyclin-dependent Kinases</span>. <i>Genome Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">122</span>, <span class="refDoi">Â DOI: 10.1186/gb4184</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=10.1186%2Fgb4184" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=25180339" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=1%3ACAS%3A280%3ADC%252BC2M%252FnsVehsg%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2014&pages=122&author=M.+Malumbres&title=Cyclin-dependent+Kinases&doi=10.1186%2Fgb4184"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Cyclin-dependent kinases</span></div><div class="casAuthors">Malumbres Marcos</div><div class="citationInfo"><span class="NLM_cas:title">Genome biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">122</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Cyclin-dependent kinases (CDKs) are protein kinases characterized by needing a separate subunit - a cyclin - that provides domains essential for enzymatic activity.  CDKs play important roles in the control of cell division and modulate transcription in response to several extra- and intracellular cues.  The evolutionary expansion of the CDK family in mammals led to the division of CDKs into three cell-cycle-related subfamilies (Cdk1, Cdk4 and Cdk5) and five transcriptional subfamilies (Cdk7, Cdk8, Cdk9, Cdk11 and Cdk20).  Unlike the prototypical Cdc28 kinase of budding yeast, most of these CDKs bind one or a few cyclins, consistent with functional specialization during evolution.  This review summarizes how, although CDKs are traditionally separated into cell-cycle or transcriptional CDKs, these activities are frequently combined in many family members.  Not surprisingly, deregulation of this family of proteins is a hallmark of several diseases, including cancer, and drug-targeted inhibition of specific members has generated very encouraging results in clinical trials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQodaug4zJdjgHqQV89nF-2fW6udTcc2eYc_RY5OiSL9rntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2M%252FnsVehsg%253D%253D&md5=9fd79ec8f495f712bada681b0a0b5afe</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1186%2Fgb4184&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fgb4184%26sid%3Dliteratum%253Aachs%26aulast%3DMalumbres%26aufirst%3DM.%26atitle%3DCyclin-dependent%2520Kinases%26jtitle%3DGenome%2520Biol.%26date%3D2014%26volume%3D15%26spage%3D122%26doi%3D10.1186%2Fgb4184" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">Targeting Cancer with Small Molecule Kinase Inhibitors</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">28</span>â <span class="NLM_lpage">39</span>, <span class="refDoi">Â DOI: 10.1038/nrc2559</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=10.1038%2Fnrc2559" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=19104514" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=1%3ACAS%3A528%3ADC%252BD1MXpt1yg" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2009&pages=28-39&author=J.+Zhangauthor=P.+L.+Yangauthor=N.+S.+Gray&title=Targeting+Cancer+with+Small+Molecule+Kinase+Inhibitors&doi=10.1038%2Fnrc2559"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting cancer with small molecule kinase inhibitors</span></div><div class="casAuthors">Zhang, Jianming; Yang, Priscilla L.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">28-39</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Deregulation of kinase activity has emerged as a major mechanism by which cancer cells evade normal physiol. constraints on growth and survival.  To date, 11 kinase inhibitors have received US Food and Drug Administration approval as cancer treatments, and there are considerable efforts to develop selective small mol. inhibitors for a host of other kinases that are implicated in cancer and other diseases.  Herein we discuss the current challenges in the field, such as designing selective inhibitors and developing strategies to overcome resistance mutations.  This Review provides a broad overview of some of the approaches currently used to discover and characterize new kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGriSM3BN_uEz7Vg90H21EOLACvtfcHk0ljleJAcLoZBLQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXpt1yg&md5=35bc38f414cd21eef5e61ce181b580fb</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fnrc2559&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc2559%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DP.%2BL.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DTargeting%2520Cancer%2520with%2520Small%2520Molecule%2520Kinase%2520Inhibitors%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2009%26volume%3D9%26spage%3D28%26epage%3D39%26doi%3D10.1038%2Fnrc2559" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">MÃ¼ller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaikuad, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">The Ins and Outs of Selective Kinase Inhibitor Development</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">818</span>â <span class="NLM_lpage">821</span>, <span class="refDoi">Â DOI: 10.1038/nchembio.1938</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=10.1038%2Fnchembio.1938" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=26485069" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1ykt7fF" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=818-821&author=S.+M%C3%BCllerauthor=A.+Chaikuadauthor=N.+S.+Grayauthor=S.+Knapp&title=The+Ins+and+Outs+of+Selective+Kinase+Inhibitor+Development&doi=10.1038%2Fnchembio.1938"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">The ins and outs of selective kinase inhibitor development</span></div><div class="casAuthors">Muller, Susanne; Chaikuad, Apirat; Gray, Nathanael S.; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">818-821</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Protein kinases have emerged as one of the most successful families of drug targets.  To date, most selective kinase inhibitors have been discovered serendipitously either through broad selectivity screening or through the discovery of unique binding modes.  Here the authors discuss design strategies that could lead to a broader coverage of the kinome with selective inhibitors and to a more rational approach for developing them.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpQzaMrxSYx_rVg90H21EOLACvtfcHk0lg_U7WuQJA6ew"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1ykt7fF&md5=f5adf0bbaa388c87b26bae23cfad0bae</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1938&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1938%26sid%3Dliteratum%253Aachs%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DChaikuad%26aufirst%3DA.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DThe%2520Ins%2520and%2520Outs%2520of%2520Selective%2520Kinase%2520Inhibitor%2520Development%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26spage%3D818%26epage%3D821%26doi%3D10.1038%2Fnchembio.1938" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Asghar, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Witkiewicz, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turner, N. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knudsen, E. S.</span></span> <span> </span><span class="NLM_article-title">The History and Future of Targeting Cyclin-dependent Kinases in Cancer Therapy</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">130</span>â <span class="NLM_lpage">146</span>, <span class="refDoi">Â DOI: 10.1038/nrd4504</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=10.1038%2Fnrd4504" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=25633797" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjvVCrtLc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2015&pages=130-146&author=U.+Asgharauthor=A.+K.+Witkiewiczauthor=N.+C.+Turnerauthor=E.+S.+Knudsen&title=The+History+and+Future+of+Targeting+Cyclin-dependent+Kinases+in+Cancer+Therapy&doi=10.1038%2Fnrd4504"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">The history and future of targeting cyclin-dependent kinases in cancer therapy</span></div><div class="casAuthors">Asghar, Uzma; Witkiewicz, Agnieszka K.; Turner, Nicholas C.; Knudsen, Erik S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">130-146</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Cancer represents a pathol. manifestation of uncontrolled cell division; therefore, it has long been anticipated that our understanding of the basic principles of cell cycle control would result in effective cancer therapies.  In particular, cyclin-dependent kinases (CDKs) that promote transition through the cell cycle were expected to be key therapeutic targets because many tumorigenic events ultimately drive proliferation by impinging on CDK4 or CDK6 complexes in the G1 phase of the cell cycle.  Moreover, perturbations in chromosomal stability and aspects of S phase and G2/M control mediated by CDK2 and CDK1 are pivotal tumorigenic events.  Translating this knowledge into successful clin. development of CDK inhibitors has historically been challenging, and numerous CDK inhibitors have demonstrated disappointing results in clin. trials.  Here, we review the biol. of CDKs, the rationale for therapeutically targeting discrete kinase complexes and historical clin. results of CDK inhibitors.  We also discuss how CDK inhibitors with high selectivity (particularly for both CDK4 and CDK6), in combination with patient stratification, have resulted in more substantial clin. activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJ_utd-yiEZLVg90H21EOLACvtfcHk0lg_U7WuQJA6ew"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjvVCrtLc%253D&md5=113bcd2e99a8267119d180670d3c7761</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnrd4504&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4504%26sid%3Dliteratum%253Aachs%26aulast%3DAsghar%26aufirst%3DU.%26aulast%3DWitkiewicz%26aufirst%3DA.%2BK.%26aulast%3DTurner%26aufirst%3DN.%2BC.%26aulast%3DKnudsen%26aufirst%3DE.%2BS.%26atitle%3DThe%2520History%2520and%2520Future%2520of%2520Targeting%2520Cyclin-dependent%2520Kinases%2520in%2520Cancer%2520Therapy%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2015%26volume%3D14%26spage%3D130%26epage%3D146%26doi%3D10.1038%2Fnrd4504" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roskoski, R.,  Jr.</span></span> <span> </span><span class="NLM_article-title">A Historical Overview of Protein Kinases and Their Targeted Small Molecule Inhibitors</span>. <i>Pharmacol. Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>100</i></span>,  <span class="NLM_fpage">1</span>â <span class="NLM_lpage">23</span>, <span class="refDoi">Â DOI: 10.1016/j.phrs.2015.07.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=10.1016%2Fj.phrs.2015.07.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=26207888" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1amsL3M" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=100&publication_year=2015&pages=1-23&author=R.+Roskoski&title=A+Historical+Overview+of+Protein+Kinases+and+Their+Targeted+Small+Molecule+Inhibitors&doi=10.1016%2Fj.phrs.2015.07.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">A historical overview of protein kinases and their targeted small molecule inhibitors</span></div><div class="casAuthors">Roskoski, Robert, Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacological Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">100</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1-23</span>CODEN:
                <span class="NLM_cas:coden">PHMREP</span>;
        ISSN:<span class="NLM_cas:issn">1043-6618</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Protein kinases play a predominant regulatory role in nearly every aspect of cell biol. and they can modify the function of a protein in almost every conceivable way.  Protein phosphorylation can increase or decrease enzyme activity and it can alter other biol. activities such as transcription and translation.  Moreover, some phosphorylation sites on a given protein are stimulatory while others are inhibitory.  The human protein kinase gene family consists of 518 members along with 106 pseudogenes.  Furthermore, about 50 of the 518 gene products lack important catalytic residues and are called protein pseudokinases.  The non-catalytic allosteric interaction of protein kinases and pseudokinases with other proteins has added an important regulatory feature to the biochem. and cell biol. of the protein kinase superfamily.  With rare exceptions, a divalent cation such as Mg2+ is required for the reaction.  All protein kinases exist in a basal state and are activated only as necessary by divergent regulatory stimuli.  The mechanisms for switching between dormant and active protein kinases can be intricate.  Phosphorylase kinase was the first protein kinase to be characterized biochem. and the mechanism of its regulation led to the discovery of cAMP-dependent protein kinase (protein kinase A, or PKA), which catalyzes the phosphorylation and activation of phosphorylase kinase.  This was the first protein kinase cascade or signaling module to be elucidated.  The epidermal growth factor receptor-Ras-Raf-MEK-ERK signaling module contains protein-tyrosine, protein-serine/threonine, and dual specificity protein kinases.  PKA has served as a prototype of this enzyme family and more is known about this enzyme than any other protein kinase.  The inactive PKA holoenzyme consists of two regulatory and two catalytic subunits.  After binding four mols. of cAMP, the holoenzyme dissocs. into a regulatory subunit dimer (each monomer binds two cAMP) and two free and active catalytic subunits.  PKA and all other protein kinase domains have a small amino-terminal lobe and large carboxyterminal lobe as detd. by X-ray crystallog.  The N-lobe and C-lobe form a cleft that serves as a docking site for MgATP.  Nearly all active protein kinases contain a K/E/D/D signature sequence that plays important structural and catalytic roles.  Protein kinases contain hydrophobic catalytic and regulatory spines and collateral shell residues that are required to assemble the active enzyme.  There are two general kinds of conformational changes assocd. with most protein kinases.  The first conformational change involves the formation of an intact regulatory spine to form an active enzyme.  The second conformational change occurs in active kinases as they toggle between open and closed conformations during their catalytic cycles.  Because mutations and dysregulation of protein kinases play causal roles in human disease, this family of enzymes has become one of the most important drug targets over the past two decades.  Imatinib was approved by the United States FDA for the treatment of chronic myelogenous leukemia in 2001; this small mol. inhibits the BCR-Abl protein kinase oncoprotein that results from the formation of the Philadelphia chromosome.  More than two dozen other orally effective mechanism-based small mol. protein kinase inhibitors have been subsequently approved by the FDA.  These drugs bind to the ATP-binding site of their target enzymes and extend into nearby hydrophobic pockets.  Most of these protein kinase inhibitors prolong survival in cancer patients only weeks or months longer than std. cytotoxic therapies.  In contrast, the clin. effectiveness of imatinib against chronic myelogenous leukemia is vastly superior to that of any other targeted protein kinase inhibitor with overall survival lasting a decade or more.  However, the near universal and expected development of drug resistance in the treatment of neoplastic disorders requires new approaches to solve this therapeutic challenge.  Cancer is the predominant indication for these drugs, but disease targets are increasing.  For example, we can expect the approval of new drugs inhibiting other protein kinases in the treatment of illnesses such as hypertension, Parkinson's disease, and autoimmune diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrrSGGFPXk0aLVg90H21EOLACvtfcHk0lg_U7WuQJA6ew"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1amsL3M&md5=819cc1d8541b51603841011d3e5f6bdb</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.phrs.2015.07.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.phrs.2015.07.010%26sid%3Dliteratum%253Aachs%26aulast%3DRoskoski%26aufirst%3DR.%26atitle%3DA%2520Historical%2520Overview%2520of%2520Protein%2520Kinases%2520and%2520Their%2520Targeted%2520Small%2520Molecule%2520Inhibitors%26jtitle%3DPharmacol.%2520Res.%26date%3D2015%26volume%3D100%26spage%3D1%26epage%3D23%26doi%3D10.1016%2Fj.phrs.2015.07.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Olson, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erb, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doctor, Z. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwiatkowski, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boukhali, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haas, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nomanbhoy, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">Pharmacological Perturbation of CDK9 Using Selective CDK9 Inhibition or Degradation</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">163</span>â <span class="NLM_lpage">170</span>, <span class="refDoi">Â DOI: 10.1038/nchembio.2538</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=10.1038%2Fnchembio.2538" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=29251720" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFyrsr7O" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2018&pages=163-170&author=C.+M.+Olsonauthor=B.+Jiangauthor=M.+A.+Erbauthor=Y.+Liangauthor=Z.+M.+Doctorauthor=Z.+Zhangauthor=T.+Zhangauthor=N.+Kwiatkowskiauthor=M.+Boukhaliauthor=J.+L.+Greenauthor=W.+Haasauthor=T.+Nomanbhoyauthor=E.+S.+Fischerauthor=R.+A.+Youngauthor=J.+E.+Bradnerauthor=G.+E.+Winterauthor=N.+S.+Gray&title=Pharmacological+Perturbation+of+CDK9+Using+Selective+CDK9+Inhibition+or+Degradation&doi=10.1038%2Fnchembio.2538"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacological perturbation of CDK9 using selective CDK9 inhibition or degradation</span></div><div class="casAuthors">Olson, Calla M.; Jiang, Baishan; Erb, Michael A.; Liang, Yanke; Doctor, Zainab M.; Zhang, Zinan; Zhang, Tinghu; Kwiatkowski, Nicholas; Boukhali, Myriam; Green, Jennifer L.; Haas, Wilhelm; Nomanbhoy, Tyzoon; Fischer, Eric S.; Young, Richard A.; Bradner, James E.; Winter, Georg E.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">163-170</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Cyclin-dependent kinase 9 (CDK9), an important regulator of transcriptional elongation, is a promising target for cancer therapy, particularly for cancers driven by transcriptional dysregulation.  The authors characterized NVP-2, a selective ATP-competitive CDK9 inhibitor, and THAL-SNS-032, a selective CDK9 degrader consisting of a CDK-binding SNS-032 ligand linked to a thalidomide deriv. that binds the E3 ubiquitin ligase Cereblon (CRBN).  To the authors' surprise, THAL-SNS-032 induced rapid degrdn. of CDK9 without affecting the levels of other SNS-032 targets.  Moreover, the transcriptional changes elicited by THAL-SNS-032 were more like those caused by NVP-2 than those induced by SNS-032.  Notably, compd. washout did not significantly reduce levels of THAL-SNS-032-induced apoptosis, suggesting that CDK9 degrdn. had prolonged cytotoxic effects compared with CDK9 inhibition.  Thus, the authors' findings suggest that thalidomide conjugation represents a promising strategy for converting multi-targeted inhibitors into selective degraders and reveal that kinase degrdn. can induce distinct pharmacol. effects compared with inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWnT9r0GIlmbVg90H21EOLACvtfcHk0linHZogbdGjlg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFyrsr7O&md5=88aabb209b4cb7516a652d6c75a35667</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2538&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2538%26sid%3Dliteratum%253Aachs%26aulast%3DOlson%26aufirst%3DC.%2BM.%26aulast%3DJiang%26aufirst%3DB.%26aulast%3DErb%26aufirst%3DM.%2BA.%26aulast%3DLiang%26aufirst%3DY.%26aulast%3DDoctor%26aufirst%3DZ.%2BM.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DKwiatkowski%26aufirst%3DN.%26aulast%3DBoukhali%26aufirst%3DM.%26aulast%3DGreen%26aufirst%3DJ.%2BL.%26aulast%3DHaas%26aufirst%3DW.%26aulast%3DNomanbhoy%26aufirst%3DT.%26aulast%3DFischer%26aufirst%3DE.%2BS.%26aulast%3DYoung%26aufirst%3DR.%2BA.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DWinter%26aufirst%3DG.%2BE.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DPharmacological%2520Perturbation%2520of%2520CDK9%2520Using%2520Selective%2520CDK9%2520Inhibition%2520or%2520Degradation%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2018%26volume%3D14%26spage%3D163%26epage%3D170%26doi%3D10.1038%2Fnchembio.2538" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cicenas, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valius, M.</span></span> <span> </span><span class="NLM_article-title">The CDK Inhibitors in Cancer Research and Therapy</span>. <i>J. Cancer Res. Clin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>137</i></span>,  <span class="NLM_fpage">1409</span>â <span class="NLM_lpage">1418</span>, <span class="refDoi">Â DOI: 10.1007/s00432-011-1039-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=10.1007%2Fs00432-011-1039-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=21877198" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFynt7rK" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=137&publication_year=2011&pages=1409-1418&author=J.+Cicenasauthor=M.+Valius&title=The+CDK+Inhibitors+in+Cancer+Research+and+Therapy&doi=10.1007%2Fs00432-011-1039-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">The CDK inhibitors in cancer research and therapy</span></div><div class="casAuthors">Cicenas, Jonas; Valius, Mindaugas</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cancer Research and Clinical Oncology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">137</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1409-1418</span>CODEN:
                <span class="NLM_cas:coden">JCROD7</span>;
        ISSN:<span class="NLM_cas:issn">0171-5216</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  Chem. compds. that interfere with an enzymic function of kinases are useful for gaining insight into the complicated biochem. processes in mammalian cells.  Cyclin-dependent kinases (CDK) play an essential role in the control of the cell cycle and/or proliferation.  These kinases as well as their regulators are frequently deregulated in different human tumors.  Aberrations in CDK activity have also been obsd. in viral infections, Alzheimer's, Parkinson's diseases, ischemia and some proliferative disorders.  This led to an intensive search for small-mol. CDK inhibitors not only for research purposes, but also for therapeutic applications.  Here, we discuss seventeen CDK inhibitors and their use in cancer research or therapy.  This review should help researchers to decide which inhibitor is best suited for the specific purpose of their research.  For this purpose, the targets, com. availability and IC50 values are provided for each inhibitor.  The review will also provide an overview of the clin. studies performed with some of these inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrEjNnnzkVahrVg90H21EOLACvtfcHk0linHZogbdGjlg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFynt7rK&md5=85500301e1d113148cf7854125e542ab</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1007%2Fs00432-011-1039-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00432-011-1039-4%26sid%3Dliteratum%253Aachs%26aulast%3DCicenas%26aufirst%3DJ.%26aulast%3DValius%26aufirst%3DM.%26atitle%3DThe%2520CDK%2520Inhibitors%2520in%2520Cancer%2520Research%2520and%2520Therapy%26jtitle%3DJ.%2520Cancer%2520Res.%2520Clin.%2520Oncol.%26date%3D2011%26volume%3D137%26spage%3D1409%26epage%3D1418%26doi%3D10.1007%2Fs00432-011-1039-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Toogood, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harvey, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Repine, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheehan, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">VanderWel, S. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keller, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNamara, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sherry, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brodfuehrer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barvian, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fry, D. W.</span></span> <span> </span><span class="NLM_article-title">Discovery of a Potent and Selective Inhibitor of Cyclin-dependent Kinase 4/6</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">2388</span>â <span class="NLM_lpage">2406</span>, <span class="refDoi">Â DOI: 10.1021/jm049354h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm049354h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=1%3ACAS%3A528%3ADC%252BD2MXitVGisL0%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=2388-2406&author=P.+L.+Toogoodauthor=P.+J.+Harveyauthor=J.+T.+Repineauthor=D.+J.+Sheehanauthor=S.+N.+VanderWelauthor=H.+Zhouauthor=P.+R.+Kellerauthor=D.+J.+McNamaraauthor=D.+Sherryauthor=T.+Zhuauthor=J.+Brodfuehrerauthor=C.+Choiauthor=M.+R.+Barvianauthor=D.+W.+Fry&title=Discovery+of+a+Potent+and+Selective+Inhibitor+of+Cyclin-dependent+Kinase+4%2F6&doi=10.1021%2Fjm049354h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Potent and Selective Inhibitor of Cyclin-Dependent Kinase 4/6</span></div><div class="casAuthors">Toogood, Peter L.; Harvey, Patricia J.; Repine, Joseph T.; Sheehan, Derek J.; VanderWel, Scott N.; Zhou, Hairong; Keller, Paul R.; McNamara, Dennis J.; Sherry, Debra; Zhu, Tong; Brodfuehrer, Joanne; Choi, Chung; Barvian, Mark R.; Fry, David W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2388-2406</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A pharmacol. approach to inhibition of cyclin-dependent kinases 4 and 6 (Cdk4/6) using highly selective small mol. inhibitors has the potential to provide novel cancer therapies for clin. use.  Achieving high levels of selectivity for Cdk4/6, vs. other ATP-dependent kinases, presents a significant challenge.  The pyrido[2,3-d]pyrimidin-7-one template provides an effective platform for the inhibition of a broad cross-section of kinases, including Cdks.  It is now demonstrated that the modification of pyrido[2,3-d]pyrimidin-7-ones to include a 2-aminopyridine side chain at the C2-position provides inhibitors with exquisite selectivity for Cdk4/6 in vitro.  This selectivity profile is recapitulated in cells where the most selective inhibitors create a G1 block at concns. up to 100-fold the IC50 for cell proliferation.  On the basis of its selectivity profile and pharmacokinetic profile, compd. I (PD 0332991) was identified as a drug candidate for the treatment of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoX577J5U0r97Vg90H21EOLACvtfcHk0linHZogbdGjlg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXitVGisL0%253D&md5=23cbe58426ff1975695712db7c1b128a</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Fjm049354h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm049354h%26sid%3Dliteratum%253Aachs%26aulast%3DToogood%26aufirst%3DP.%2BL.%26aulast%3DHarvey%26aufirst%3DP.%2BJ.%26aulast%3DRepine%26aufirst%3DJ.%2BT.%26aulast%3DSheehan%26aufirst%3DD.%2BJ.%26aulast%3DVanderWel%26aufirst%3DS.%2BN.%26aulast%3DZhou%26aufirst%3DH.%26aulast%3DKeller%26aufirst%3DP.%2BR.%26aulast%3DMcNamara%26aufirst%3DD.%2BJ.%26aulast%3DSherry%26aufirst%3DD.%26aulast%3DZhu%26aufirst%3DT.%26aulast%3DBrodfuehrer%26aufirst%3DJ.%26aulast%3DChoi%26aufirst%3DC.%26aulast%3DBarvian%26aufirst%3DM.%2BR.%26aulast%3DFry%26aufirst%3DD.%2BW.%26atitle%3DDiscovery%2520of%2520a%2520Potent%2520and%2520Selective%2520Inhibitor%2520of%2520Cyclin-dependent%2520Kinase%25204%252F6%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D2388%26epage%3D2406%26doi%3D10.1021%2Fjm049354h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">VanArsdale, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boshoff, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arndt, K. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abraham, R. T.</span></span> <span> </span><span class="NLM_article-title">Molecular Pathways: Targeting the Cyclin D-CDK4/6 Axis for Cancer Treatment</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">2905</span>â <span class="NLM_lpage">2910</span>, <span class="refDoi">Â DOI: 10.1158/1078-0432.ccr-14-0816</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=10.1158%2F1078-0432.CCR-14-0816" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=25941111" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=1%3ACAS%3A280%3ADC%252BC2Mfgs1Wisg%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2015&pages=2905-2910&author=T.+VanArsdaleauthor=C.+Boshoffauthor=K.+T.+Arndtauthor=R.+T.+Abraham&title=Molecular+Pathways%3A+Targeting+the+Cyclin+D-CDK4%2F6+Axis+for+Cancer+Treatment&doi=10.1158%2F1078-0432.ccr-14-0816"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Pathways: Targeting the Cyclin D-CDK4/6 Axis for Cancer Treatment</span></div><div class="casAuthors">VanArsdale Todd; Abraham Robert T; Boshoff Chris; Arndt Kim T</div><div class="citationInfo"><span class="NLM_cas:title">Clinical cancer research : an official journal of the American Association for Cancer Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">2905-10</span>
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    </div><div class="casAbstract">Cancer cells bypass normal controls over mitotic cell-cycle progression to achieve a deregulated state of proliferation.  The retinoblastoma tumor suppressor protein (pRb) governs a key cell-cycle checkpoint that normally prevents G1-phase cells from entering S-phase in the absence of appropriate mitogenic signals.  Cancer cells frequently overcome pRb-dependent growth suppression via constitutive phosphorylation and inactivation of pRb function by cyclin-dependent kinase (CDK) 4 or CDK6 partnered with D-type cyclins.  Three selective CDK4/6 inhibitors, palbociclib (Ibrance; Pfizer), ribociclib (Novartis), and abemaciclib (Lilly), are in various stages of development in a variety of pRb-positive tumor types, including breast cancer, melanoma, liposarcoma, and non-small cell lung cancer.  The emerging, positive clinical data obtained to date finally validate the two decades-old hypothesis that the cyclin D-CDK4/6 pathway is a rational target for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR6Rstlwhprintlzo9xastyfW6udTcc2eYPf5p6IDLLw7ntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2Mfgs1Wisg%253D%253D&md5=5b3443d0ec58550d8ef4f1993726fb5e</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-14-0816&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-14-0816%26sid%3Dliteratum%253Aachs%26aulast%3DVanArsdale%26aufirst%3DT.%26aulast%3DBoshoff%26aufirst%3DC.%26aulast%3DArndt%26aufirst%3DK.%2BT.%26aulast%3DAbraham%26aufirst%3DR.%2BT.%26atitle%3DMolecular%2520Pathways%253A%2520Targeting%2520the%2520Cyclin%2520D-CDK4%252F6%2520Axis%2520for%2520Cancer%2520Treatment%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2015%26volume%3D21%26spage%3D2905%26epage%3D2910%26doi%3D10.1158%2F1078-0432.ccr-14-0816" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gelbert, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanchez-Martinez, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Del Prado, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lallena, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torres, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ajamie, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wishart, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flack, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neubauer, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iversen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cronier, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kreklau, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Dios, A.</span></span> <span> </span><span class="NLM_article-title">Preclinical Characterization of the CDK4/6 Inhibitor LY2835219: In-vivo Cell Cycle-Dependent/independent Anti-tumor Activities Alone/in Combination with Gemcitabine</span>. <i>Invest. New Drugs</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">825</span>â <span class="NLM_lpage">837</span>, <span class="refDoi">Â DOI: 10.1007/s10637-014-0120-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=10.1007%2Fs10637-014-0120-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=24919854" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtVWrt7%252FN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2014&pages=825-837&author=L.+M.+Gelbertauthor=S.+Caiauthor=X.+Linauthor=C.+Sanchez-Martinezauthor=M.+Del+Pradoauthor=M.+J.+Lallenaauthor=R.+Torresauthor=R.+T.+Ajamieauthor=G.+N.+Wishartauthor=R.+S.+Flackauthor=B.+L.+Neubauerauthor=J.+Youngauthor=E.+M.+Chanauthor=P.+Iversenauthor=D.+Cronierauthor=E.+Kreklauauthor=A.+de+Dios&title=Preclinical+Characterization+of+the+CDK4%2F6+Inhibitor+LY2835219%3A+In-vivo+Cell+Cycle-Dependent%2Findependent+Anti-tumor+Activities+Alone%2Fin+Combination+with+Gemcitabine&doi=10.1007%2Fs10637-014-0120-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Preclinical characterization of the CDK4/6 inhibitor LY2835219: in-vivo cell cycle-dependent/independent anti-tumor activities alone/in combination with gemcitabine</span></div><div class="casAuthors">Gelbert, Lawrence M.; Cai, Shufen; Lin, Xi; Sanchez-Martinez, Concepcion; del Prado, Miriam; Lallena, Maria Jose; Torres, Raquel; Ajamie, Rose T.; Wishart, Graham N.; Flack, Robert Steven; Neubauer, Blake Lee; Young, Jamie; Chan, Edward M.; Iversen, Philip; Cronier, Damien; Kreklau, Emiko; de Dios, Alfonso</div><div class="citationInfo"><span class="NLM_cas:title">Investigational New Drugs</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">825-837</span>CODEN:
                <span class="NLM_cas:coden">INNDDK</span>;
        ISSN:<span class="NLM_cas:issn">0167-6997</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">The G1 restriction point is crit. for regulating the cell cycle and is controlled by the Rb pathway (CDK4/6-cyclin D1-Rb-p16/ink4a).  This pathway is important because of its inactivation in a majority of human tumors.  Transition through the restriction point requires phosphorylation of retinoblastoma protein (Rb) by CDK4/6, which are highly validated cancer drug targets.  We present the identification and characterization of a potent CDK4/6 inhibitor, LY2835219.  LY2835219 inhibits CDK4 and CDK6 with low nanomolar potency, inhibits Rb phosphorylation resulting in a G1 arrest and inhibition of proliferation, and its activity is specific for Rb-proficient cells.  In vivo target inhibition studies show LY2835219 is a potent inhibitor of Rb phosphorylation, induces a complete cell cycle arrest and suppresses expression of several Rb-E2F-regulated proteins 24 h after a single dose.  Oral administration of LY2835219 inhibits tumor growth in human tumor xenografts representing different histologies in tumor-bearing mice.  LY2835219 is effective and well tolerated when administered up to 56 days in immunodeficient mice without significant loss of body wt. or tumor outgrowth.  In calu-6 xenografts, LY2835219 in combination with gemcitabine enhanced in vivo antitumor activity without a G1 cell cycle arrest, but was assocd. with a redn. of ribonucleotide reductase expression.  These results suggest LY2835219 may be used alone or in combination with std.-of-care cytotoxic therapy.  In summary, we have identified a potent, orally active small-mol. inhibitor of CDK4/6 that is active in xenograft tumors.  LY2835219 is currently in clin. development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoQrFS1mDbyCrVg90H21EOLACvtfcHk0lgTqHTxniieJg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtVWrt7%252FN&md5=a90fee83bfe8da7e4a7474ccf02fc59a</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1007%2Fs10637-014-0120-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10637-014-0120-7%26sid%3Dliteratum%253Aachs%26aulast%3DGelbert%26aufirst%3DL.%2BM.%26aulast%3DCai%26aufirst%3DS.%26aulast%3DLin%26aufirst%3DX.%26aulast%3DSanchez-Martinez%26aufirst%3DC.%26aulast%3DDel%2BPrado%26aufirst%3DM.%26aulast%3DLallena%26aufirst%3DM.%2BJ.%26aulast%3DTorres%26aufirst%3DR.%26aulast%3DAjamie%26aufirst%3DR.%2BT.%26aulast%3DWishart%26aufirst%3DG.%2BN.%26aulast%3DFlack%26aufirst%3DR.%2BS.%26aulast%3DNeubauer%26aufirst%3DB.%2BL.%26aulast%3DYoung%26aufirst%3DJ.%26aulast%3DChan%26aufirst%3DE.%2BM.%26aulast%3DIversen%26aufirst%3DP.%26aulast%3DCronier%26aufirst%3DD.%26aulast%3DKreklau%26aufirst%3DE.%26aulast%3Dde%2BDios%26aufirst%3DA.%26atitle%3DPreclinical%2520Characterization%2520of%2520the%2520CDK4%252F6%2520Inhibitor%2520LY2835219%253A%2520In-vivo%2520Cell%2520Cycle-Dependent%252Findependent%2520Anti-tumor%2520Activities%2520Alone%252Fin%2520Combination%2520with%2520Gemcitabine%26jtitle%3DInvest.%2520New%2520Drugs%26date%3D2014%26volume%3D32%26spage%3D825%26epage%3D837%26doi%3D10.1007%2Fs10637-014-0120-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">DiPippo, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barnett, C. M.</span></span> <span> </span><span class="NLM_article-title">Cyclin-Dependent Kinase Inhibitors for the Treatment of Breast Cancer: Past, Present, and Future</span>. <i>Pharmacotherapy</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">652</span>â <span class="NLM_lpage">667</span>, <span class="refDoi">Â DOI: 10.1002/phar.1756</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=10.1002%2Fphar.1756" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=27087139" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=1%3ACAS%3A280%3ADC%252BC28bgslCksA%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2016&pages=652-667&author=A.+J.+DiPippoauthor=N.+K.+Patelauthor=C.+M.+Barnett&title=Cyclin-Dependent+Kinase+Inhibitors+for+the+Treatment+of+Breast+Cancer%3A+Past%2C+Present%2C+and+Future&doi=10.1002%2Fphar.1756"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Cyclin-Dependent Kinase Inhibitors for the Treatment of Breast Cancer: Past, Present, and Future</span></div><div class="casAuthors">DiPippo Adam J; Patel Neelam K; Barnett Chad M</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacotherapy</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">652-67</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Treatment of metastatic breast cancer (MBC) that is resistant to endocrine therapy presents a significant clinical challenge.  The well-known role of cell cycle dysregulation in these patients is partly mediated by cyclin-dependent kinase (CDK) activity.  Specific cyclin and CDK complexes regulate cell cycle progression by managing the transition through the cell cycle, and inhibition of CDKs represents an important target for novel agents.  First-generation CDK inhibitors (e.g., flavopiridol) were relatively nonselective and had an unacceptable toxicity profile in early trials.  Second-generation CDK inhibitors were designed to target the CDK4 and CDK6 (CDK4/6) pathway and have shown promising clinical activity with an acceptable toxicity profile in patients with MBC.  Palbociclib is a first-in-class CDK4/6 inhibitor that was granted accelerated U.S.  Food and Drug Administration approval in combination with letrozole for the treatment of MBC in the first-line setting (February 2015) as well as in combination with fulvestrant for MBC that had progressed on previous endocrine therapy (February 2016).  Other CDK4/6 inhibitors, including ribociclib and abemaciclib, are under investigation as monotherapy and in combination with endocrine or anti-human epidermal growth receptor 2 therapy for the treatment of MBC.  Ongoing clinical trials should provide additional information to guide the appropriate use of these agents and identify patient populations that could derive the most benefit.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSYivEamzCCty86Js-BCLiifW6udTcc2eYPf5p6IDLLw7ntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28bgslCksA%253D%253D&md5=cc590c6fb21bbdc1e5384e4020a4123f</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1002%2Fphar.1756&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fphar.1756%26sid%3Dliteratum%253Aachs%26aulast%3DDiPippo%26aufirst%3DA.%2BJ.%26aulast%3DPatel%26aufirst%3DN.%2BK.%26aulast%3DBarnett%26aufirst%3DC.%2BM.%26atitle%3DCyclin-Dependent%2520Kinase%2520Inhibitors%2520for%2520the%2520Treatment%2520of%2520Breast%2520Cancer%253A%2520Past%252C%2520Present%252C%2520and%2520Future%26jtitle%3DPharmacotherapy%26date%3D2016%26volume%3D36%26spage%3D652%26epage%3D667%26doi%3D10.1002%2Fphar.1756" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roskoski, R.,  Jr.</span></span> <span> </span><span class="NLM_article-title">Cyclin-dependent Protein Kinase Inhibitors Including Palbociclib as Anticancer Drugs</span>. <i>Pharmacol. Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>107</i></span>,  <span class="NLM_fpage">249</span>â <span class="NLM_lpage">275</span>, <span class="refDoi">Â DOI: 10.1016/j.phrs.2016.03.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=10.1016%2Fj.phrs.2016.03.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=26995305" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=1%3ACAS%3A528%3ADC%252BC28XnvFOjt7o%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=107&publication_year=2016&pages=249-275&author=R.+Roskoski&title=Cyclin-dependent+Protein+Kinase+Inhibitors+Including+Palbociclib+as+Anticancer+Drugs&doi=10.1016%2Fj.phrs.2016.03.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Cyclin-dependent protein kinase inhibitors including palbociclib as anticancer drugs</span></div><div class="casAuthors">Roskoski, Robert, Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacological Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">107</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">249-275</span>CODEN:
                <span class="NLM_cas:coden">PHMREP</span>;
        ISSN:<span class="NLM_cas:issn">1043-6618</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Cyclins and cyclin-dependent protein kinases (CDKs) are important regulatory components that are required for cell cycle progression.  The levels of the cell cycle CDKs are generally const. and their activities are controlled by cyclins, proteins whose levels oscillate during each cell cycle.  Addnl. CDK family members were subsequently discovered that play significant roles in a wide range of activities including the control of gene transcription, metab., and neuronal function.  In response to mitogenic stimuli, cells in the G1 phase of the cell cycle produce cyclins of the D type that activate CDK4/6.  These activated enzymes catalyze the monophosphorylation of the retinoblastoma protein.  Then CDK2-cyclin E catalyzes the hyperphosphorylation of Rb that promotes the release and activation of the E2F transcription factors, which in turn lead to the generation of several proteins required for cell cycle progression.  As a result, cells pass through the G1-restriction point and are committed to complete cell division.  CDK2-cyclin A, CDK1-cyclin A, and CDK1-cyclin B are required for S, G2, and M-phase progression.  Increased cyclin or CDK expression or decreased levels of endogenous CDK inhibitors such as INK4 or CIP/KIP have been obsd. in various cancers.  In contrast to the mutational activation of EGFR, Kit, or B-Raf in the pathogenesis of malignancies, mutations in the CDKs that cause cancers are rare.  Owing to their role in cell proliferation, CDKs represent natural targets for anticancer therapies.  Abemaciclib (LY2835219), ribociclib (Lee011), and palbociclib (Ibrance or PD0332991) target CDK4/6 with IC50 values in the low nanomolar range.  Palbociclib and other CDK inhibitors bind in the cleft between the small and large lobes of the CDKs and inhibit the binding of ATP.  Like ATP, palbociclib forms hydrogen bonds with residues in the hinge segment of the cleft.  Like the adenine base of ATP, palbociclib interacts with catalytic spine residues CS6 and CS7.  CDK antagonists are in clin. trials for the treatment of a variety of malignancies.  Significantly, palbociclib has been approved by the FDA for the treatment of hormone-receptor pos./human epidermal growth factor receptor-2 neg. breast cancer in conjunction with letrozole as a first-line therapy and with fulvestrant as a second-line treatment.  As inhibitors of the cell cycle, it is not surprising that one of their most common toxicities is myelosuppression with decreased neutrophil prodn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGol0On7HAFtl7Vg90H21EOLACvtfcHk0li5XJEW7OACpQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XnvFOjt7o%253D&md5=0356807b16735e3903706eac10635bc7</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2Fj.phrs.2016.03.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.phrs.2016.03.012%26sid%3Dliteratum%253Aachs%26aulast%3DRoskoski%26aufirst%3DR.%26atitle%3DCyclin-dependent%2520Protein%2520Kinase%2520Inhibitors%2520Including%2520Palbociclib%2520as%2520Anticancer%2520Drugs%26jtitle%3DPharmacol.%2520Res.%26date%3D2016%26volume%3D107%26spage%3D249%26epage%3D275%26doi%3D10.1016%2Fj.phrs.2016.03.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCartney, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Migliaccio, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonechi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biagioni, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romagnoli, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Luca, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galardi, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Risi, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Santo, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benelli, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malorni, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Leo, A.</span></span> <span> </span><span class="NLM_article-title">Mechanisms of Resistance to CDK4/6 Inhibitors: Potential Implications and Biomarkers for Clinical Practice</span>. <i>Front. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">666</span>, <span class="refDoi">Â DOI: 10.3389/fonc.2019.00666</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=10.3389%2Ffonc.2019.00666" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=31396487" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=1%3ACAS%3A280%3ADC%252BB3MvmtlKrug%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2019&pages=666&author=A.+McCartneyauthor=I.+Migliaccioauthor=M.+Bonechiauthor=C.+Biagioniauthor=D.+Romagnoliauthor=F.+De+Lucaauthor=F.+Galardiauthor=E.+Risiauthor=I.+De+Santoauthor=M.+Benelliauthor=L.+Malorniauthor=A.+Di+Leo&title=Mechanisms+of+Resistance+to+CDK4%2F6+Inhibitors%3A+Potential+Implications+and+Biomarkers+for+Clinical+Practice&doi=10.3389%2Ffonc.2019.00666"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanisms of Resistance to CDK4/6 Inhibitors: Potential Implications and Biomarkers for Clinical Practice</span></div><div class="casAuthors">McCartney Amelia; Risi Emanuela; De Santo Irene; Malorni Luca; Di Leo Angelo; Migliaccio Ilenia; Bonechi Martina; De Luca Francesca; Galardi Francesca; Malorni Luca; Biagioni Chiara; Romagnoli Dario; Benelli Matteo</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in oncology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">666</span>
        ISSN:<span class="NLM_cas:issn">2234-943X</span>.
    </div><div class="casAbstract">The recent arrival of CDK4/6 inhibitor agents, with an approximate doubling of progression-free survival (PFS) associated with their use in hormone receptor-positive, HER2-negative advanced breast cancer (BC), has radically changed the approach to managing this disease.  However, resistance to CDK4/6 inhibitors is considered a near-inevitability in most patients.  Mechanisms of resistance to these agents are multifactorial, and research in this field is still evolving.  Biomarkers with the ability to identify early resistance, or to predict the likelihood of successful treatment using CDK4/6 inhibitors are yet to be identified, and represent an area of unmet clinical need.  Here we present selected mechanisms of resistance to CDK4/6 inhibitors, largely focussing on roles of Rb, cyclin E1, and the PIK3CA pathway, with discussion of associated biomarkers which have been investigated and applied in recent pre-clinical and clinical studies.  These biological drivers may furthermore influence clinical treatment strategies adopted beyond CDK4/6 resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRrJ96Nuqm037P3FVUi_gf8fW6udTcc2eaniXcmz5Hujrntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3MvmtlKrug%253D%253D&md5=cea69c3a374c8599fe94ba34f65e6eab</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.3389%2Ffonc.2019.00666&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffonc.2019.00666%26sid%3Dliteratum%253Aachs%26aulast%3DMcCartney%26aufirst%3DA.%26aulast%3DMigliaccio%26aufirst%3DI.%26aulast%3DBonechi%26aufirst%3DM.%26aulast%3DBiagioni%26aufirst%3DC.%26aulast%3DRomagnoli%26aufirst%3DD.%26aulast%3DDe%2BLuca%26aufirst%3DF.%26aulast%3DGalardi%26aufirst%3DF.%26aulast%3DRisi%26aufirst%3DE.%26aulast%3DDe%2BSanto%26aufirst%3DI.%26aulast%3DBenelli%26aufirst%3DM.%26aulast%3DMalorni%26aufirst%3DL.%26aulast%3DDi%2BLeo%26aufirst%3DA.%26atitle%3DMechanisms%2520of%2520Resistance%2520to%2520CDK4%252F6%2520Inhibitors%253A%2520Potential%2520Implications%2520and%2520Biomarkers%2520for%2520Clinical%2520Practice%26jtitle%3DFront.%2520Oncol.%26date%3D2019%26volume%3D9%26spage%3D666%26doi%3D10.3389%2Ffonc.2019.00666" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Condorelli, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spring, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">OâShaughnessy, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lacroix, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bailleux, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dubois, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagy, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lanman, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iafrate, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andre, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bardia, A.</span></span> <span> </span><span class="NLM_article-title">Polyclonal RB1 Mutations and Acquired Resistance to CDK 4/6 Inhibitors in Patients with Metastatic Breast Cancer</span>. <i>Ann. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">640</span>â <span class="NLM_lpage">645</span>, <span class="refDoi">Â DOI: 10.1093/annonc/mdx784</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=10.1093%2Fannonc%2Fmdx784" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=29236940" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=1%3ACAS%3A280%3ADC%252BC1MzhtFWisw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2018&pages=640-645&author=R.+Condorelliauthor=L.+Springauthor=J.+O%E2%80%99Shaughnessyauthor=L.+Lacroixauthor=C.+Bailleuxauthor=V.+Scottauthor=J.+Duboisauthor=R.+J.+Nagyauthor=R.+B.+Lanmanauthor=A.+J.+Iafrateauthor=F.+Andreauthor=A.+Bardia&title=Polyclonal+RB1+Mutations+and+Acquired+Resistance+to+CDK+4%2F6+Inhibitors+in+Patients+with+Metastatic+Breast+Cancer&doi=10.1093%2Fannonc%2Fmdx784"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Polyclonal RB1 mutations and acquired resistance to CDK 4/6 inhibitors in patients with metastatic breast cancer</span></div><div class="casAuthors">Condorelli R; Lacroix L; Bailleux C; Scott V; Andre F; Spring L; Dubois J; Iafrate A J; Bardia A; O'Shaughnessy J; O'Shaughnessy J; Nagy R J; Lanman R B</div><div class="citationInfo"><span class="NLM_cas:title">Annals of oncology : official journal of the European Society for Medical Oncology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">640-645</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Background:  While deregulation of the cyclin D1-CDK4/6-retinoblastoma pathway is common in hormone receptor positive (HR+) breast cancer, Rb is usually intact in HR+ breast cancer, and targeted CDK 4/6 inhibitors that act upstream of Rb, are routinely being utilized in clinical practice.  However, factors that can lead to clinical resistance to CDK 4/6 inhibitors are not known.  Patients and methods:  We identified patients who had pre- and post-genotyping in tissue and peripheral blood samples after receiving CDK 4/6 inhibitors.  Genotyping was carried out in tumor tissue or blood collected before start of CDK 4/6 inhibitor and after disease progression on CDK 4/6 inhibitor, covering more than 90% of the coding region in RB1.  Results:  We identified detectable acquired RB1 mutations in circulating tumor DNA (ctDNA) after exposure to CDK4/6 inhibitor (palbociclib, palbociclib, ribociclib) for 5, 8, and 13 months, respectively, in three patients.  The RB1 mutations included substitution in donor splicing site of exon 8 of the RB1 gene in patient #1; substitution in donor splicing site of exon 22 of RB1 gene, exon 19 deletion, exon 3 insertion in patient #2; and RB1 exon 16 H483Y mutation in patient #3.  None of these RB1 mutations were present in the pre-CDK 4/6 specimen highlighting these molecular alterations, which lead to functional loss of Rb1, likely emerged under selective pressure from the CDK4/6 inhibitor potentially confering therapeutic resistance.  Conclusion:  This is the first clinical report to describe the emergence of somatic RB1 mutations after exposure to palbociclib or ribociclib, in patients with metastatic breast cancer.  Further research is needed to validate these findings, identify how these mutations temporally emerge under selective pressure of CDK 4/6 inhibitor, and develop rational therapeutic strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT57Ei4We3h7fIW7hznt0zpfW6udTcc2ebTWtN5cXYMDLntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1MzhtFWisw%253D%253D&md5=0f659618552c56fdc7b21f50bbb7c0cf</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1093%2Fannonc%2Fmdx784&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fannonc%252Fmdx784%26sid%3Dliteratum%253Aachs%26aulast%3DCondorelli%26aufirst%3DR.%26aulast%3DSpring%26aufirst%3DL.%26aulast%3DO%25E2%2580%2599Shaughnessy%26aufirst%3DJ.%26aulast%3DLacroix%26aufirst%3DL.%26aulast%3DBailleux%26aufirst%3DC.%26aulast%3DScott%26aufirst%3DV.%26aulast%3DDubois%26aufirst%3DJ.%26aulast%3DNagy%26aufirst%3DR.%2BJ.%26aulast%3DLanman%26aufirst%3DR.%2BB.%26aulast%3DIafrate%26aufirst%3DA.%2BJ.%26aulast%3DAndre%26aufirst%3DF.%26aulast%3DBardia%26aufirst%3DA.%26atitle%3DPolyclonal%2520RB1%2520Mutations%2520and%2520Acquired%2520Resistance%2520to%2520CDK%25204%252F6%2520Inhibitors%2520in%2520Patients%2520with%2520Metastatic%2520Breast%2520Cancer%26jtitle%3DAnn.%2520Oncol.%26date%3D2018%26volume%3D29%26spage%3D640%26epage%3D645%26doi%3D10.1093%2Fannonc%2Fmdx784" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kent, L. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leone, G.</span></span> <span> </span><span class="NLM_article-title">The broken cycle: E2F Dysfunction in Cancer</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">326</span>â <span class="NLM_lpage">338</span>, <span class="refDoi">Â DOI: 10.1038/s41568-019-0143-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=10.1038%2Fs41568-019-0143-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=31053804" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=1%3ACAS%3A528%3ADC%252BC1MXptFSqt7k%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2019&pages=326-338&author=L.+N.+Kentauthor=G.+Leone&title=The+broken+cycle%3A+E2F+Dysfunction+in+Cancer&doi=10.1038%2Fs41568-019-0143-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">The broken cycle: E2F dysfunction in cancer</span></div><div class="casAuthors">Kent, Lindsey N.; Leone, Gustavo</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">326-338</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">The cyclin-dependent kinase (CDK)-RB-E2F axis forms the core transcriptional machinery driving cell cycle progression, dictating the timing and fidelity of genome replication and ensuring genetic material is accurately passed through each cell division cycle.  The ultimate effectors of this axis are members of a family of eight distinct E2F genes encoding transcriptional activators and repressors.  E2F transcriptional activity is tightly regulated throughout the cell cycle via transcriptional and translational regulation, post-translational modifications, protein degrdn., binding to cofactors and subcellular localization.  Alterations in one or more key components of this axis (CDKs, cyclins, CDK inhibitors and the RB family of proteins) occur in virtually all cancers and result in heightened oncogenic E2F activity, leading to uncontrolled proliferation.  In this Review, we discuss the activities of E2F proteins with an emphasis on the newest atypical E2F family members, the specific and redundant functions of E2F proteins, how misexpression of E2F transcriptional targets promotes cancer and both current and developing therapeutic strategies being used to target this oncogenic pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZOonvudzTx7Vg90H21EOLACvtfcHk0lg11yir28bL_g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXptFSqt7k%253D&md5=8f9b774c2dbdfee74823ab5a8f14b4cc</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1038%2Fs41568-019-0143-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41568-019-0143-7%26sid%3Dliteratum%253Aachs%26aulast%3DKent%26aufirst%3DL.%2BN.%26aulast%3DLeone%26aufirst%3DG.%26atitle%3DThe%2520broken%2520cycle%253A%2520E2F%2520Dysfunction%2520in%2520Cancer%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2019%26volume%3D19%26spage%3D326%26epage%3D338%26doi%3D10.1038%2Fs41568-019-0143-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Santo, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siu, K. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raje, N.</span></span> <span> </span><span class="NLM_article-title">Targeting Cyclin-Dependent Kinases and Cell Cycle Progression in Human Cancers</span>. <i>Semin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">788</span>â <span class="NLM_lpage">800</span>, <span class="refDoi">Â DOI: 10.1053/j.seminoncol.2015.09.024</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=10.1053%2Fj.seminoncol.2015.09.024" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=26615126" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFWksL7J" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2015&pages=788-800&author=L.+Santoauthor=K.+T.+Siuauthor=N.+Raje&title=Targeting+Cyclin-Dependent+Kinases+and+Cell+Cycle+Progression+in+Human+Cancers&doi=10.1053%2Fj.seminoncol.2015.09.024"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting Cyclin-Dependent Kinases and Cell Cycle Progression in Human Cancers</span></div><div class="casAuthors">Santo, Loredana; Siu, Ka Tat; Raje, Noopur</div><div class="citationInfo"><span class="NLM_cas:title">Seminars in Oncology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">788-800</span>CODEN:
                <span class="NLM_cas:coden">SOLGAV</span>;
        ISSN:<span class="NLM_cas:issn">0093-7754</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Uncontrolled cell division is a defining characteristic of cancer cells.  Cyclin-dependent kinases (Cdks) are crit. regulators of cell cycle progression.  Deregulated Cdk activities as a result of gene amplification, translocation, or point mutations of Cdks or cyclins, have been reported in a majority of human cancers.  These kinases, therefore, represent potential therapeutic targets for the treatment of cancer.  In this review, we offer an overview of Cdk functions in driving cell cycle progression and transcriptional regulation, a highlight of the DNA damage checkpoints, and an outline of the most relevant Cdk inhibitors currently in clin. trials with an emphasis on the Cdk inhibitors used for treatment of multiple myeloma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBLbVHqlZ2OrVg90H21EOLACvtfcHk0lg11yir28bL_g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFWksL7J&md5=ee57a7c9b080ae0ad3860a4d8b2dd691</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1053%2Fj.seminoncol.2015.09.024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1053%252Fj.seminoncol.2015.09.024%26sid%3Dliteratum%253Aachs%26aulast%3DSanto%26aufirst%3DL.%26aulast%3DSiu%26aufirst%3DK.%2BT.%26aulast%3DRaje%26aufirst%3DN.%26atitle%3DTargeting%2520Cyclin-Dependent%2520Kinases%2520and%2520Cell%2520Cycle%2520Progression%2520in%2520Human%2520Cancers%26jtitle%3DSemin.%2520Oncol.%26date%3D2015%26volume%3D42%26spage%3D788%26epage%3D800%26doi%3D10.1053%2Fj.seminoncol.2015.09.024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weng, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zuo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span> <span> </span><span class="NLM_article-title">Recent Progress of Cyclin-dependent Kinase Inhibitors as Potential Anticancer Agents</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">2047</span>â <span class="NLM_lpage">2076</span>, <span class="refDoi">Â DOI: 10.4155/fmc-2016-0129</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=10.4155%2Ffmc-2016-0129" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=27774795" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=1%3ACAS%3A528%3ADC%252BC28XhslGisbnJ" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2016&pages=2047-2076&author=T.+Liauthor=T.+Wengauthor=M.+Zuoauthor=Z.+Weiauthor=M.+Chenauthor=Z.+Li&title=Recent+Progress+of+Cyclin-dependent+Kinase+Inhibitors+as+Potential+Anticancer+Agents&doi=10.4155%2Ffmc-2016-0129"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Recent progress of cyclin-dependent kinase inhibitors as potential anticancer agents</span></div><div class="casAuthors">Li, Tinghan; Weng, Tianwei; Zuo, Minzan; Wei, Zhihui; Chen, Ming; Li, Zhiyu</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">2047-2076</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">Deregulation of the cell cycle is a common feature in human cancer.  The inhibition of cyclin-dependent kinases (CDKs), which play a crucial role in control of the cell cycle, has always been one of the most promising areas in cancer chemotherapy.  This review first summarizes the biol. of CDKs and then focuses on the recent advances in both broad-range and selective CDK inhibitors during the last 5 years.  The design rationale, structural optimization and structure-activity relationships anal. of these small mols. have been discussed in detail and the key interactions with the amino-acid residues of the most important compds. are highlighted.  Future perspectives for CDKs inhibitors will be defined in the development of highly selective CDK inhibitors, an accurate knowledge of gene control mechanism and further predictive biomarker research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrKW1A5UEVYeLVg90H21EOLACvtfcHk0lguCMP7B3qUKQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhslGisbnJ&md5=90331b2b80a383eaee6483a6e6e6e1e6</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.4155%2Ffmc-2016-0129&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc-2016-0129%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DT.%26aulast%3DWeng%26aufirst%3DT.%26aulast%3DZuo%26aufirst%3DM.%26aulast%3DWei%26aufirst%3DZ.%26aulast%3DChen%26aufirst%3DM.%26aulast%3DLi%26aufirst%3DZ.%26atitle%3DRecent%2520Progress%2520of%2520Cyclin-dependent%2520Kinase%2520Inhibitors%2520as%2520Potential%2520Anticancer%2520Agents%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2016%26volume%3D8%26spage%3D2047%26epage%3D2076%26doi%3D10.4155%2Ffmc-2016-0129" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ling, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, T.</span></span> <span> </span><span class="NLM_article-title">Discovery of 4-((7H-Pyrrolo[2,3-d]pyrimidin-4-yl)amino)-N-(4-((4-methylpiperazin-1-yl)methyl)phenyl)-1H-pyrazole-3-carboxamide (FN-1501), an FLT3- and CDK-Kinase Inhibitor with Potentially High Efficiency against Acute Myelocytic Leukemia</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">1499</span>â <span class="NLM_lpage">1518</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.7b01261</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01261" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtlyqurg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=1499-1518&author=Y.+Wangauthor=Y.+Zhiauthor=Q.+Jinauthor=S.+Luauthor=G.+Linauthor=H.+Yuanauthor=T.+Yangauthor=Z.+Wangauthor=C.+Yaoauthor=J.+Lingauthor=H.+Guoauthor=T.+Liauthor=J.+Jinauthor=B.+Liauthor=L.+Zhangauthor=Y.+Chenauthor=T.+Lu&title=Discovery+of+4-%28%287H-Pyrrolo%5B2%2C3-d%5Dpyrimidin-4-yl%29amino%29-N-%284-%28%284-methylpiperazin-1-yl%29methyl%29phenyl%29-1H-pyrazole-3-carboxamide+%28FN-1501%29%2C+an+FLT3-+and+CDK-Kinase+Inhibitor+with+Potentially+High+Efficiency+against+Acute+Myelocytic+Leukemia&doi=10.1021%2Facs.jmedchem.7b01261"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of 4-((7H-Pyrrolo[2,3-d]pyrimidin-4-yl)amino)-N-(4-((4-methylpiperazin-1-yl)methyl)phenyl)-1H-pyrazole-3-carboxamide (FN-1501), an FLT3- and CDK-Kinase Inhibitor with Potentially High Efficiency against Acute Myelocytic Leukemia</span></div><div class="casAuthors">Wang, Yue; Zhi, Yanle; Jin, Qiaomei; Lu, Shuai; Lin, Guowu; Yuan, Haoliang; Yang, Taotao; Wang, Zhanwei; Yao, Chao; Ling, Jun; Guo, Hao; Li, Tonghui; Jin, Jianlin; Li, Baoquan; Zhang, Li; Chen, Yadong; Lu, Tao</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1499-1518</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A series of 1-H-pyrazole-3-carboxamide derivs. have been designed and synthesized that exhibit excellent FLT3 and CDK inhibition and antiproliferative activities.  A structure-activity-relationship study illustrates that the incorporation of a pyrimidine-fused heterocycle at position 4 of the pyrazole is crit. for FLT3 and CDK inhibition.  Compd. 50 (FN-1501), which possesses potent inhibitory activities against FLT3, CDK2, CDK4, and CDK6 with IC50 values in the nanomolar range, shows antiproliferative activities against MV4-11 cells (IC50: 0.008 Î¼M), which correlates with the suppression of retinoblastoma phosphorylation, FLT3, ERK, AKT, and STAT5 and the onset of apoptosis.  Acute-toxicity studies in mice show that compd. I (LD50: 186 mg/kg) is safer than AT7519 (32 mg/kg).  In MV4-11 xenografts in a nude-mouse model, compd. I can induce tumor regression at the dose of 15 mg/kg, which is more efficient than cytarabine (50 mg/kg).  Taken together, these results demonstrate the potential of this unique compd. for further development into a drug applied in acute-myeloid-leukemia (AML) therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrfX9S4QkGccbVg90H21EOLACvtfcHk0lguCMP7B3qUKQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtlyqurg%253D&md5=2757ad7559e655c5d96f0a8af8c74aa5</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01261&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01261%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DY.%26aulast%3DZhi%26aufirst%3DY.%26aulast%3DJin%26aufirst%3DQ.%26aulast%3DLu%26aufirst%3DS.%26aulast%3DLin%26aufirst%3DG.%26aulast%3DYuan%26aufirst%3DH.%26aulast%3DYang%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DYao%26aufirst%3DC.%26aulast%3DLing%26aufirst%3DJ.%26aulast%3DGuo%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DT.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DB.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DLu%26aufirst%3DT.%26atitle%3DDiscovery%2520of%25204-%2528%25287H-Pyrrolo%255B2%252C3-d%255Dpyrimidin-4-yl%2529amino%2529-N-%25284-%2528%25284-methylpiperazin-1-yl%2529methyl%2529phenyl%2529-1H-pyrazole-3-carboxamide%2520%2528FN-1501%2529%252C%2520an%2520FLT3-%2520and%2520CDK-Kinase%2520Inhibitor%2520with%2520Potentially%2520High%2520Efficiency%2520against%2520Acute%2520Myelocytic%2520Leukemia%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D1499%26epage%3D1518%26doi%3D10.1021%2Facs.jmedchem.7b01261" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">PrÃ©vel, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurzawa, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van, T. N. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morris, M. C.</span></span> <span> </span><span class="NLM_article-title">Fluorescent Biosensors for Drug Discovery New Tools for Old Targets-screening for Inhibitors of Cyclin-Dependent Kinases</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>88</i></span>,  <span class="NLM_fpage">74</span>â <span class="NLM_lpage">88</span>, <span class="refDoi">Â DOI: 10.1016/j.ejmech.2014.10.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=10.1016%2Fj.ejmech.2014.10.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=25314935" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1yhurnK" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=88&publication_year=2014&pages=74-88&author=C.+Pr%C3%A9velauthor=L.+Kurzawaauthor=T.+N.+N.+Vanauthor=M.+C.+Morris&title=Fluorescent+Biosensors+for+Drug+Discovery+New+Tools+for+Old+Targets-screening+for+Inhibitors+of+Cyclin-Dependent+Kinases&doi=10.1016%2Fj.ejmech.2014.10.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Fluorescent biosensors for drug discovery new tools for old targets - Screening for inhibitors of cyclin-dependent kinases</span></div><div class="casAuthors">Prevel, Camille; Kurzawa, Laetitia; Van, Thi Nhu Ngoc; Morris, May C.</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">88</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">74-88</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">A review.  Cyclin-dependent kinases play central roles in regulation of cell cycle progression, transcriptional regulation and other major biol. processes such as neuronal differentiation and metab.  These kinases are hyperactivated in most human cancers and constitute attractive pharmacol. targets.  A large no. of ATP-competitive inhibitors of CDKs have been identified from natural substances, in high throughput screening assays, or through structure-guided approaches.  Alternative strategies have been explored to target essential protein/protein interfaces and screen for allosteric inhibitors that trap inactive intermediates or prevent conformational activation.  However this remains a major challenge given the highly conserved structural features of these kinases, and calls for new and alternative screening technologies.  Fluorescent biosensors constitute powerful tools for the detection of biomols. in complex biol. samples, and are well suited to study dynamic processes and highlight mol. alterations assocd. with pathol. disorders.  They further constitute sensitive and selective tools which can be readily implemented to high throughput and high content screens in drug discovery programs.  Our group has developed fluorescent biosensors to probe cyclin-dependent kinases and gain insight into their mol. behavior in vitro and in living cells.  These tools provide a means of monitoring subtle alterations in the abundance and activity of CDK/Cyclins and can respond to compds. that interfere with the conformational dynamics of these kinases.  In this review we discuss the different strategies which have been devised to target CDK/Cyclins, and describe the implementation of our CDK/Cyclin biosensors to develop HTS/HCS assays in view of identifying new classes of inhibitors for cancer therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXcNXQWPLI_bVg90H21EOLACvtfcHk0lguCMP7B3qUKQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1yhurnK&md5=93fecc37ae33d1a1b6370a89cb9d28d0</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2014.10.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2014.10.003%26sid%3Dliteratum%253Aachs%26aulast%3DPr%25C3%25A9vel%26aufirst%3DC.%26aulast%3DKurzawa%26aufirst%3DL.%26aulast%3DVan%26aufirst%3DT.%2BN.%2BN.%26aulast%3DMorris%26aufirst%3DM.%2BC.%26atitle%3DFluorescent%2520Biosensors%2520for%2520Drug%2520Discovery%2520New%2520Tools%2520for%2520Old%2520Targets-screening%2520for%2520Inhibitors%2520of%2520Cyclin-Dependent%2520Kinases%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2014%26volume%3D88%26spage%3D74%26epage%3D88%26doi%3D10.1016%2Fj.ejmech.2014.10.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Coxon, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anscombe, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harnor, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carbain, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golding, B. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hardcastle, I. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harlow, L. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korolchuk, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matheson, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Newell, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noble, M. E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sivaprakasam, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tudhope, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turner, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wedge, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffin, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Endicott, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cano, C.</span></span> <span> </span><span class="NLM_article-title">Cyclin-Dependent Kinase (CDK) Inhibitors: Structure-Activity Relationships and Insights into the CDK-2 Selectivity of 6-Substituted 2-Arylaminopurines</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">1746</span>â <span class="NLM_lpage">1767</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.6b01254</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01254" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=1%3ACAS%3A528%3ADC%252BC28XitFCgs7fK" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=1746-1767&author=C.+R.+Coxonauthor=E.+Anscombeauthor=S.+J.+Harnorauthor=M.+P.+Martinauthor=B.+Carbainauthor=B.+T.+Goldingauthor=I.+R.+Hardcastleauthor=L.+K.+Harlowauthor=S.+Korolchukauthor=C.+J.+Mathesonauthor=D.+R.+Newellauthor=M.+E.+M.+Nobleauthor=M.+Sivaprakasamauthor=S.+J.+Tudhopeauthor=D.+M.+Turnerauthor=L.+Z.+Wangauthor=S.+R.+Wedgeauthor=C.+Wongauthor=R.+J.+Griffinauthor=J.+A.+Endicottauthor=C.+Cano&title=Cyclin-Dependent+Kinase+%28CDK%29+Inhibitors%3A+Structure-Activity+Relationships+and+Insights+into+the+CDK-2+Selectivity+of+6-Substituted+2-Arylaminopurines&doi=10.1021%2Facs.jmedchem.6b01254"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Cyclin-Dependent Kinase (CDK) Inhibitors: Structure-Activity Relationships and Insights into the CDK-2 Selectivity of 6-Substituted 2-Arylaminopurines</span></div><div class="casAuthors">Coxon, Christopher R.; Anscombe, Elizabeth; Harnor, Suzannah J.; Martin, Mathew P.; Carbain, Benoit; Golding, Bernard T.; Hardcastle, Ian R.; Harlow, Lisa K.; Korolchuk, Svitlana; Matheson, Christopher J.; Newell, David R.; Noble, Martin E. M.; Sivaprakasam, Mangaleswaran; Tudhope, Susan J.; Turner, David M.; Wang, Lan Z.; Wedge, Stephen R.; Wong, Christopher; Griffin, Roger J.; Endicott, Jane A.; Cano, Celine</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1746-1767</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Purines and related heterocycles substituted at C-2 with 4'-sulfamoylanilino and at C-6 with a variety of groups have been synthesized with the aim of achieving selectivity of binding to CDK2 over CDK1.  6-Substituents that favor competitive inhibition at the ATP binding site of CDK2 were identified and typically exhibited 10-80-fold greater inhibition of CDK2 compared to CDK1.  Most impressive was 4-((6-([1,1'-biphenyl]-3-yl)-9H-purin-2-yl)amino) benzenesulfonamide (73) that exhibited high potency towards CDK2 (IC50 0.044 Î¼M), but was â¼ 2000-fold less active towards CDK1 (IC50 86 Î¼M).  This compd. is therefore a useful tool for studies of cell cycle regulation.  Crystal structures of inhibitor-kinase complexes showed that the inhibitor stabilizes a glycine-rich loop conformation that shapes the ATP ribose binding pocket, and that is preferred in CDK2, but has not been obsd. in CDK1.  This aspect of the active site may be exploited for the design of inhibitors that distinguish between CDK1 and CDK2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGocYzXYCKo7CrVg90H21EOLACvtfcHk0lilNahJocCUqA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitFCgs7fK&md5=520549b69fec2b2af9758bfcf8708c55</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01254&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01254%26sid%3Dliteratum%253Aachs%26aulast%3DCoxon%26aufirst%3DC.%2BR.%26aulast%3DAnscombe%26aufirst%3DE.%26aulast%3DHarnor%26aufirst%3DS.%2BJ.%26aulast%3DMartin%26aufirst%3DM.%2BP.%26aulast%3DCarbain%26aufirst%3DB.%26aulast%3DGolding%26aufirst%3DB.%2BT.%26aulast%3DHardcastle%26aufirst%3DI.%2BR.%26aulast%3DHarlow%26aufirst%3DL.%2BK.%26aulast%3DKorolchuk%26aufirst%3DS.%26aulast%3DMatheson%26aufirst%3DC.%2BJ.%26aulast%3DNewell%26aufirst%3DD.%2BR.%26aulast%3DNoble%26aufirst%3DM.%2BE.%2BM.%26aulast%3DSivaprakasam%26aufirst%3DM.%26aulast%3DTudhope%26aufirst%3DS.%2BJ.%26aulast%3DTurner%26aufirst%3DD.%2BM.%26aulast%3DWang%26aufirst%3DL.%2BZ.%26aulast%3DWedge%26aufirst%3DS.%2BR.%26aulast%3DWong%26aufirst%3DC.%26aulast%3DGriffin%26aufirst%3DR.%2BJ.%26aulast%3DEndicott%26aufirst%3DJ.%2BA.%26aulast%3DCano%26aufirst%3DC.%26atitle%3DCyclin-Dependent%2520Kinase%2520%2528CDK%2529%2520Inhibitors%253A%2520Structure-Activity%2520Relationships%2520and%2520Insights%2520into%2520the%2520CDK-2%2520Selectivity%2520of%25206-Substituted%25202-Arylaminopurines%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D1746%26epage%3D1767%26doi%3D10.1021%2Facs.jmedchem.6b01254" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, T. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lozach, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Surpateanu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Retailleau, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iorga, B. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meijer, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">GuÃ©ritte, F.</span></span> <span> </span><span class="NLM_article-title">Synthesis, Biological Evaluation, and Molecular Modeling of Natural and Unnatural Flavonoidal Alkaloids, Inhibitors of Kinases</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">2811</span>â <span class="NLM_lpage">2819</span>, <span class="refDoi">Â DOI: 10.1021/jm201727w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm201727w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=1%3ACAS%3A528%3ADC%252BC38XislSls70%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=2811-2819&author=T.+B.+Nguyenauthor=O.+Lozachauthor=G.+Surpateanuauthor=Q.+Wangauthor=P.+Retailleauauthor=B.+I.+Iorgaauthor=L.+Meijerauthor=F.+Gu%C3%A9ritte&title=Synthesis%2C+Biological+Evaluation%2C+and+Molecular+Modeling+of+Natural+and+Unnatural+Flavonoidal+Alkaloids%2C+Inhibitors+of+Kinases&doi=10.1021%2Fjm201727w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis, Biological Evaluation, and Molecular Modeling of Natural and Unnatural Flavonoidal Alkaloids, Inhibitors of Kinases</span></div><div class="casAuthors">Nguyen, Thanh Binh; Lozach, Olivier; Surpateanu, Georgiana; Wang, Qian; Retailleau, Pascal; Iorga, Bogdan I.; Meijer, Laurent; Gueritte, Francoise</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2811-2819</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The screening of the ICSN chem. library on various disease-relevant protein kinases led to the identification of natural flavonoidal alkaloids of unknown configuration as potent inhibitors of the CDK1 and CDK5 kinases.  We thus developed an efficient and modular synthetic strategy for their prepn. and that of analogs in order to det. the abs. configuration of the active natural flavonoidal alkaloids and to provide further insights on the structure-activity relationships in this series.  For example, reacting chrysin with hexanal and MeNH2 gave a mixt. of 5,7-dihydroxy-6-(1-methylamino)hexyl chromenone deriv. I [R = CH(NHMe)(CH2)4Me, R1 = H] and 5,7-dihydroxy-8-(1-methylamino)hexyl chromenone I [R = H, R1 = CH(NHMe)(CH2)4Me].  Sepn. of the 8-substituted deriv. into the R and S enantiomers was accomplished using SFC HPLC.  The structural determinants of the interaction between some flavonoidal alkaloids with specific kinases were also evaluated using mol. modeling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcblOGXLZewbVg90H21EOLACvtfcHk0lilNahJocCUqA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XislSls70%253D&md5=632d257992af13ace1aea830e9353bd0</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Fjm201727w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm201727w%26sid%3Dliteratum%253Aachs%26aulast%3DNguyen%26aufirst%3DT.%2BB.%26aulast%3DLozach%26aufirst%3DO.%26aulast%3DSurpateanu%26aufirst%3DG.%26aulast%3DWang%26aufirst%3DQ.%26aulast%3DRetailleau%26aufirst%3DP.%26aulast%3DIorga%26aufirst%3DB.%2BI.%26aulast%3DMeijer%26aufirst%3DL.%26aulast%3DGu%25C3%25A9ritte%26aufirst%3DF.%26atitle%3DSynthesis%252C%2520Biological%2520Evaluation%252C%2520and%2520Molecular%2520Modeling%2520of%2520Natural%2520and%2520Unnatural%2520Flavonoidal%2520Alkaloids%252C%2520Inhibitors%2520of%2520Kinases%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D2811%26epage%3D2819%26doi%3D10.1021%2Fjm201727w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schonbrunn, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Betzi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alam, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becker, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Francis, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chakrasali, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jakkaraj, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kazi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sebti, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cubitt, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gebhard, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hazlehurst, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tash, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Georg, G. I.</span></span> <span> </span><span class="NLM_article-title">Development of Highly Potent and Selective Diaminothiazole Inhibitors of Cyclin-dependent Kinases</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">3768</span>â <span class="NLM_lpage">3782</span>, <span class="refDoi">Â DOI: 10.1021/jm301234k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm301234k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=1%3ACAS%3A528%3ADC%252BC3sXmtF2jtb8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=3768-3782&author=E.+Schonbrunnauthor=S.+Betziauthor=R.+Alamauthor=M.+P.+Martinauthor=A.+Beckerauthor=H.+Hanauthor=R.+Francisauthor=R.+Chakrasaliauthor=S.+Jakkarajauthor=A.+Kaziauthor=S.+M.+Sebtiauthor=C.+L.+Cubittauthor=A.+W.+Gebhardauthor=L.+A.+Hazlehurstauthor=J.+S.+Tashauthor=G.+I.+Georg&title=Development+of+Highly+Potent+and+Selective+Diaminothiazole+Inhibitors+of+Cyclin-dependent+Kinases&doi=10.1021%2Fjm301234k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Development of Highly Potent and Selective Diaminothiazole Inhibitors of Cyclin-Dependent Kinases</span></div><div class="casAuthors">Schonbrunn, Ernst; Betzi, Stephane; Alam, Riazul; Martin, Mathew P.; Becker, Andreas; Han, Huijong; Francis, Rawle; Chakrasali, Ramappa; Jakkaraj, Sudhakar; Kazi, Aslamuzzaman; Sebti, Said M.; Cubitt, Christopher L.; Gebhard, Anthony W.; Hazlehurst, Lori A.; Tash, Joseph S.; Georg, Gunda I.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3768-3782</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Diaminothiazoles such as I were prepd. as inhibitors of cyclin-dependent kinase 2 (CDK2) for potential use as antitumor agents and as male contraceptive agents based on the optimization of a hit compd. with weak inhibitory activity (IC50 = 15 Î¼M) discovered by high-throughput screening.  Using cocrystal structures of 35 diaminothiazoles with CDK2, the structure-activity relations of diaminothiazoles for CDK2 inhibition were detd. and used to find I.  I was tested for inhibition against a panel of 339 kinases; cyclin-dependent kinases were inhibited selectively, particularly CDK2 and CDK5 over CDK9, CDK1, CDK4, and CDK6.  I inhibited the proliferation of 13 out of 15 cancer cell lines with IC50 values of 0.27-6.9 Î¼M; the antiproliferative activity correlated to complete suppression of retinoblastoma phosphorylation and the onset of apoptosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpeM-VFFhEenrVg90H21EOLACvtfcHk0lilNahJocCUqA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXmtF2jtb8%253D&md5=067bdf2349f9cd853b83d19131e0770f</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fjm301234k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm301234k%26sid%3Dliteratum%253Aachs%26aulast%3DSchonbrunn%26aufirst%3DE.%26aulast%3DBetzi%26aufirst%3DS.%26aulast%3DAlam%26aufirst%3DR.%26aulast%3DMartin%26aufirst%3DM.%2BP.%26aulast%3DBecker%26aufirst%3DA.%26aulast%3DHan%26aufirst%3DH.%26aulast%3DFrancis%26aufirst%3DR.%26aulast%3DChakrasali%26aufirst%3DR.%26aulast%3DJakkaraj%26aufirst%3DS.%26aulast%3DKazi%26aufirst%3DA.%26aulast%3DSebti%26aufirst%3DS.%2BM.%26aulast%3DCubitt%26aufirst%3DC.%2BL.%26aulast%3DGebhard%26aufirst%3DA.%2BW.%26aulast%3DHazlehurst%26aufirst%3DL.%2BA.%26aulast%3DTash%26aufirst%3DJ.%2BS.%26aulast%3DGeorg%26aufirst%3DG.%2BI.%26atitle%3DDevelopment%2520of%2520Highly%2520Potent%2520and%2520Selective%2520Diaminothiazole%2520Inhibitors%2520of%2520Cyclin-dependent%2520Kinases%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D3768%26epage%3D3782%26doi%3D10.1021%2Fjm301234k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kuo, G.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deangelis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emanuel, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Connolly, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gruninger, R. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rugg, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuentes-Pesquera, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Middleton, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jolliffe, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, W. V.</span></span> <span> </span><span class="NLM_article-title">Synthesis and Identification of [1,3,5]triazine-pyridine Biheteroaryl as a Novel Series of Potent Cyclin-dependent Kinase Inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">4535</span>â <span class="NLM_lpage">4546</span>, <span class="refDoi">Â DOI: 10.1021/jm040214h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm040214h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=1%3ACAS%3A528%3ADC%252BD2MXks1Sgt7w%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=4535-4546&author=G.-H.+Kuoauthor=A.+Deangelisauthor=S.+Emanuelauthor=A.+Wangauthor=Y.+Zhangauthor=P.+J.+Connollyauthor=X.+Chenauthor=R.+H.+Gruningerauthor=C.+Ruggauthor=A.+Fuentes-Pesqueraauthor=S.+A.+Middletonauthor=L.+Jolliffeauthor=W.+V.+Murray&title=Synthesis+and+Identification+of+%5B1%2C3%2C5%5Dtriazine-pyridine+Biheteroaryl+as+a+Novel+Series+of+Potent+Cyclin-dependent+Kinase+Inhibitors&doi=10.1021%2Fjm040214h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and Identification of [1,3,5]Triazine-pyridine Biheteroaryl as a Novel Series of Potent Cyclin-Dependent Kinase Inhibitors</span></div><div class="casAuthors">Kuo, Gee-Hong; DeAngelis, Alan; Emanuel, Stuart; Wang, Aihua; Zhang, Yan; Connolly, Peter J.; Chen, Xin; Gruninger, Robert H.; Rugg, Catherine; Fuentes-Pesquera, Angel; Middleton, Steven A.; Jolliffe, Linda; Murray, William V.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">4535-4546</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">On the basis of previous studies, a identified pyrazine-pyridine deriv. was identified as a potent vascular endothelial growth factor inhibitor and a pyrimidine-pyridine deriv. as a moderately potent cyclin dependent kinase (CDK) inhibitor.  A proposed combination of CGP-60474 and a pyrimidine-pyridine deriv. led to the discovery of [1,3,5]triazine-pyridine as a new series of potent CDK inhibitors.  Palladium-catalyzed C-C bond formation reactions, particularly the Negishi coupling reaction, were used to assemble various triazine-heteroaryl analogs effectively.  Among them, 3-[[4-[4-[(3-chlorophenyl)amino]-1,3,5-triazin-2-yl]-2-pyridinyl]amino]-1-propanol (I) displayed high inhibitory potency at CDK1 (IC50 = 0.021 Î¼M), CDK2, and CDK5 and sub-micromolar potency at CDK4, CDK6, and CDK7.  I also displayed high potency at GSK-3Î².  It demonstrated potent antiproliferative activity on various tumor cell lines, including HeLa, HCT-116, U937, and A375.  I was administered i.p. at 150 and 125 mg/kg to nude mice bearing human A375 xenografts, the compd. produced a significant survival increase.  Mol. docking studies were conducted in an attempt to enhance the understanding of the obsd. structure-activity relationship.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoj1MFdlX2V_LVg90H21EOLACvtfcHk0lh3GT_G4fxH4A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXks1Sgt7w%253D&md5=1901f66de5152ade248ad440a4a1cb7c</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fjm040214h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm040214h%26sid%3Dliteratum%253Aachs%26aulast%3DKuo%26aufirst%3DG.-H.%26aulast%3DDeangelis%26aufirst%3DA.%26aulast%3DEmanuel%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DA.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DConnolly%26aufirst%3DP.%2BJ.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DGruninger%26aufirst%3DR.%2BH.%26aulast%3DRugg%26aufirst%3DC.%26aulast%3DFuentes-Pesquera%26aufirst%3DA.%26aulast%3DMiddleton%26aufirst%3DS.%2BA.%26aulast%3DJolliffe%26aufirst%3DL.%26aulast%3DMurray%26aufirst%3DW.%2BV.%26atitle%3DSynthesis%2520and%2520Identification%2520of%2520%255B1%252C3%252C5%255Dtriazine-pyridine%2520Biheteroaryl%2520as%2520a%2520Novel%2520Series%2520of%2520Potent%2520Cyclin-dependent%2520Kinase%2520Inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D4535%26epage%3D4546%26doi%3D10.1021%2Fjm040214h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Singh, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chashoo, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khan, S. U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mahajan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nargotra, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mahajan, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mintoo, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guru, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aruri, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thatikonda, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sahu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chibber, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mir, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bharate, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madishetti, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nandi, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mondhe, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhushan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malik, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mignani, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vishwakarma, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, P. P.</span></span> <span> </span><span class="NLM_article-title">Design of Novel 3-Pyrimidinylazaindole CDK2/9 Inhibitors with Potent In Vitro and In Vivo Antitumor Efficacy in a Triple-Negative Breast Cancer Model</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">9470</span>â <span class="NLM_lpage">9489</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.7b00663</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00663" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvVWqtbnL" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=9470-9489&author=U.+Singhauthor=G.+Chashooauthor=S.+U.+Khanauthor=P.+Mahajanauthor=A.+Nargotraauthor=G.+Mahajanauthor=A.+Singhauthor=A.+Sharmaauthor=M.+J.+Mintooauthor=S.+K.+Guruauthor=H.+Aruriauthor=T.+Thatikondaauthor=P.+Sahuauthor=P.+Chibberauthor=V.+Kumarauthor=S.+A.+Mirauthor=S.+S.+Bharateauthor=S.+Madishettiauthor=U.+Nandiauthor=G.+Singhauthor=D.+M.+Mondheauthor=S.+Bhushanauthor=F.+Malikauthor=S.+Mignaniauthor=R.+A.+Vishwakarmaauthor=P.+P.+Singh&title=Design+of+Novel+3-Pyrimidinylazaindole+CDK2%2F9+Inhibitors+with+Potent+In+Vitro+and+In+Vivo+Antitumor+Efficacy+in+a+Triple-Negative+Breast+Cancer+Model&doi=10.1021%2Facs.jmedchem.7b00663"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Design of Novel 3-Pyrimidinylazaindole CDK2/9 Inhibitors with Potent In Vitro and In Vivo Antitumor Efficacy in a Triple-Negative Breast Cancer Model</span></div><div class="casAuthors">Singh, Umed; Chashoo, Gousia; Khan, Sameer U.; Mahajan, Priya; Nargotra, Amit; Mahajan, Girish; Singh, Amarinder; Sharma, Anjna; Mintoo, Mubashir J.; Guru, Santosh Kumar; Aruri, Hariprasad; Thatikonda, Thanusha; Sahu, Promod; Chibber, Pankaj; Kumar, Vikas; Mir, Sameer A.; Bharate, Sonali S.; Madishetti, Sreedhar; Nandi, Utpal; Singh, Gurdarshan; Mondhe, Dilip Manikrao; Bhushan, Shashi; Malik, Fayaz; Mignani, Serge; Vishwakarma, Ram A.; Singh, Parvinder Pal</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">9470-9489</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In the present study, novel series of 3-pyrimidinylazaindoles were designed and synthesized using bioinformatics strategy as cyclin-dependent kinases CDK2 and CDK9 inhibitors, which play crit. roles in cell-cycle control and regulation of cell transcription.  The present approach has given new dimensions to existing SAR and opens the new opportunity for lead optimization from comparatively inexpensive starting materials.  The study led to the identification of alternative lead candidate I with nanomolar potency against CDK2 and CDK9 and potent anti-proliferative activities against a panel of tested tumor cell lines along better safety ratio of â¼30 in comparison to reported leads.  In addn., the identified lead I has demonstrated good soly. and acceptable in vivo PK profile.  The identified lead I has also shown in vivo efficacy in mouse triple neg. breast cancer (TNBC) syngeneic model with TGI of 90% without any mortality growth inhibition in comparison to reported leads.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrvqLjfKAkGYbVg90H21EOLACvtfcHk0lh3GT_G4fxH4A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvVWqtbnL&md5=5f6edf3003a3a7b2e9b29d51c68846d1</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00663&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00663%26sid%3Dliteratum%253Aachs%26aulast%3DSingh%26aufirst%3DU.%26aulast%3DChashoo%26aufirst%3DG.%26aulast%3DKhan%26aufirst%3DS.%2BU.%26aulast%3DMahajan%26aufirst%3DP.%26aulast%3DNargotra%26aufirst%3DA.%26aulast%3DMahajan%26aufirst%3DG.%26aulast%3DSingh%26aufirst%3DA.%26aulast%3DSharma%26aufirst%3DA.%26aulast%3DMintoo%26aufirst%3DM.%2BJ.%26aulast%3DGuru%26aufirst%3DS.%2BK.%26aulast%3DAruri%26aufirst%3DH.%26aulast%3DThatikonda%26aufirst%3DT.%26aulast%3DSahu%26aufirst%3DP.%26aulast%3DChibber%26aufirst%3DP.%26aulast%3DKumar%26aufirst%3DV.%26aulast%3DMir%26aufirst%3DS.%2BA.%26aulast%3DBharate%26aufirst%3DS.%2BS.%26aulast%3DMadishetti%26aufirst%3DS.%26aulast%3DNandi%26aufirst%3DU.%26aulast%3DSingh%26aufirst%3DG.%26aulast%3DMondhe%26aufirst%3DD.%2BM.%26aulast%3DBhushan%26aufirst%3DS.%26aulast%3DMalik%26aufirst%3DF.%26aulast%3DMignani%26aufirst%3DS.%26aulast%3DVishwakarma%26aufirst%3DR.%2BA.%26aulast%3DSingh%26aufirst%3DP.%2BP.%26atitle%3DDesign%2520of%2520Novel%25203-Pyrimidinylazaindole%2520CDK2%252F9%2520Inhibitors%2520with%2520Potent%2520In%2520Vitro%2520and%2520In%2520Vivo%2520Antitumor%2520Efficacy%2520in%2520a%2520Triple-Negative%2520Breast%2520Cancer%2520Model%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D9470%26epage%3D9489%26doi%3D10.1021%2Facs.jmedchem.7b00663" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Heathcote, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kroll, S. H. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hazel, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Periyasamy, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alikian, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kanneganti, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jogalekar, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scheiper, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbazanges, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blum, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brackow, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siwicka, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pace, R. D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuchter, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snyder, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liotta, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freemont, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aboagye, E. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coombes, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrett, A. G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ali, S.</span></span> <span> </span><span class="NLM_article-title">A Novel Pyrazolo[1,5-a]pyrimidine Is a Potent Inhibitor of Cyclin-dependent Protein Kinases 1, 2, and 9, Which Demonstrates Antitumor Effects in Human Tumor Xenografts Following Oral Administration</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">8508</span>â <span class="NLM_lpage">8522</span>, <span class="refDoi">Â DOI: 10.1021/jm100732t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm100732t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVeksL%252FJ" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=8508-8522&author=D.+A.+Heathcoteauthor=H.+Patelauthor=S.+H.+B.+Krollauthor=P.+Hazelauthor=M.+Periyasamyauthor=M.+Alikianauthor=S.+K.+Kannegantiauthor=A.+S.+Jogalekarauthor=B.+Scheiperauthor=M.+Barbazangesauthor=A.+Blumauthor=J.+Brackowauthor=A.+Siwickaauthor=R.+D.+M.+Paceauthor=M.+J.+Fuchterauthor=J.+P.+Snyderauthor=D.+C.+Liottaauthor=P.+S.+Freemontauthor=E.+O.+Aboagyeauthor=R.+C.+Coombesauthor=A.+G.+M.+Barrettauthor=S.+Ali&title=A+Novel+Pyrazolo%5B1%2C5-a%5Dpyrimidine+Is+a+Potent+Inhibitor+of+Cyclin-dependent+Protein+Kinases+1%2C+2%2C+and+9%2C+Which+Demonstrates+Antitumor+Effects+in+Human+Tumor+Xenografts+Following+Oral+Administration&doi=10.1021%2Fjm100732t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">A Novel Pyrazolo[1,5-a]pyrimidine is a Potent Inhibitor of Cyclin-Dependent Protein Kinases 1, 2, and 9, Which Demonstrates Antitumor Effects in Human Tumor Xenografts Following Oral Administration</span></div><div class="casAuthors">Heathcote, Dean A.; Patel, Hetal; Kroll, Sebastian H. B.; Hazel, Pascale; Periyasamy, Manikandan; Alikian, Mary; Kanneganti, Seshu K.; Jogalekar, Ashutosh S.; Scheiper, Bodo; Barbazanges, Marion; Blum, Andreas; Brackow, Jan; Siwicka, Alekasandra; Pace, Robert D. M.; Fuchter, Matthew J.; Snyder, James P.; Liotta, Dennis C.; Freemont, Paul. S.; Aboagye, Eric O.; Coombes, R. Charles; Barrett, Anthony G. M.; Ali, Simak</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">8508-8522</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Cyclin-dependent protein kinases (CDKs) are central to the appropriate regulation of cell proliferation, apoptosis, and gene expression.  Abnormalities in CDK activity and regulation are common features of cancer, making CDK family members attractive targets for the development of anticancer drugs.  Here, we report the identification of a pyrazolo[1,5-a]pyrimidine derived compd., 4k (BS-194, I), as a selective and potent CDK inhibitor, which inhibits CDK2, CDK1, CDK5, CDK7, and CDK9 (IC50 = 3, 30, 30, 250, and 90 nmol/L, resp.).  Cell-based studies showed inhibition of the phosphorylation of CDK substrates, Rb and the RNA polymerase II C-terminal domain, down-regulation of cyclins A, E, and D1, and cell cycle block in the S and G2/M phases.  Consistent with these findings, 4k demonstrated potent antiproliferative activity in 60 cancer cell lines tested (mean GI50 = 280 nmol/L).  Pharmacokinetic studies showed that 4k is orally bioavailable, with an elimination half-life of 178 min following oral dosing in mice.  When administered at a concn. of 25 mg/kg orally, 4k inhibited human tumor xenografts and suppressed CDK substrate phosphorylation.  These findings identify 4k as a novel, potent CDK selective inhibitor with potential for oral delivery in cancer patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpUJPhLKEWwBLVg90H21EOLACvtfcHk0lgodFymyb1XVg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVeksL%252FJ&md5=b84f41ab0cbf298a9ac248bc0a9d0d73</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Fjm100732t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm100732t%26sid%3Dliteratum%253Aachs%26aulast%3DHeathcote%26aufirst%3DD.%2BA.%26aulast%3DPatel%26aufirst%3DH.%26aulast%3DKroll%26aufirst%3DS.%2BH.%2BB.%26aulast%3DHazel%26aufirst%3DP.%26aulast%3DPeriyasamy%26aufirst%3DM.%26aulast%3DAlikian%26aufirst%3DM.%26aulast%3DKanneganti%26aufirst%3DS.%2BK.%26aulast%3DJogalekar%26aufirst%3DA.%2BS.%26aulast%3DScheiper%26aufirst%3DB.%26aulast%3DBarbazanges%26aufirst%3DM.%26aulast%3DBlum%26aufirst%3DA.%26aulast%3DBrackow%26aufirst%3DJ.%26aulast%3DSiwicka%26aufirst%3DA.%26aulast%3DPace%26aufirst%3DR.%2BD.%2BM.%26aulast%3DFuchter%26aufirst%3DM.%2BJ.%26aulast%3DSnyder%26aufirst%3DJ.%2BP.%26aulast%3DLiotta%26aufirst%3DD.%2BC.%26aulast%3DFreemont%26aufirst%3DP.%2BS.%26aulast%3DAboagye%26aufirst%3DE.%2BO.%26aulast%3DCoombes%26aufirst%3DR.%2BC.%26aulast%3DBarrett%26aufirst%3DA.%2BG.%2BM.%26aulast%3DAli%26aufirst%3DS.%26atitle%3DA%2520Novel%2520Pyrazolo%255B1%252C5-a%255Dpyrimidine%2520Is%2520a%2520Potent%2520Inhibitor%2520of%2520Cyclin-dependent%2520Protein%2520Kinases%25201%252C%25202%252C%2520and%25209%252C%2520Which%2520Demonstrates%2520Antitumor%2520Effects%2520in%2520Human%2520Tumor%2520Xenografts%2520Following%2520Oral%2520Administration%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D8508%26epage%3D8522%26doi%3D10.1021%2Fjm100732t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nie, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, R.</span></span> <span> </span><span class="NLM_article-title">Discovery of N1-(4-((7-Cyclopentyl-6-(dimethylcarbamoyl)-7 H-pyrrolo[2,3-d]pyrimidin-2-yl)amino)phenyl)- N<sup>8</sup>-hydroxyoctanediamide as a Novel Inhibitor Targeting Cyclin-dependent Kinase 4/9 (CDK4/9) and Histone Deacetlyase1 (HDAC1) Against Malignant Cancer</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">3166</span>â <span class="NLM_lpage">3192</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.8b00209</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00209" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=1%3ACAS%3A528%3ADC%252BC1cXktFWgs7c%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=3166-3192&author=Y.+Liauthor=X.+Luoauthor=Q.+Guoauthor=Y.+Nieauthor=T.+Wangauthor=C.+Zhangauthor=Z.+Huangauthor=X.+Wangauthor=Y.+Liuauthor=Y.+Chenauthor=J.+Zhengauthor=S.+Yangauthor=Y.+Fanauthor=R.+Xiang&title=Discovery+of+N1-%284-%28%287-Cyclopentyl-6-%28dimethylcarbamoyl%29-7+H-pyrrolo%5B2%2C3-d%5Dpyrimidin-2-yl%29amino%29phenyl%29-+N8-hydroxyoctanediamide+as+a+Novel+Inhibitor+Targeting+Cyclin-dependent+Kinase+4%2F9+%28CDK4%2F9%29+and+Histone+Deacetlyase1+%28HDAC1%29+Against+Malignant+Cancer&doi=10.1021%2Facs.jmedchem.8b00209"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of N1-(4-((7-Cyclopentyl-6-(dimethylcarbamoyl)-7H-pyrrolo[2,3-d]pyrimidin-2-yl)amino)phenyl)-N8-hydroxyoctanediamide as a Novel Inhibitor Targeting Cyclin-dependent Kinase 4/9 (CDK4/9) and Histone Deacetylase 1 (HDAC1) against Malignant Cancer</span></div><div class="casAuthors">Li, Yongtao; Luo, Xiaohe; Guo, Qingxiang; Nie, Yongwei; Wang, Tianqi; Zhang, Chao; Huang, Zhi; Wang, Xin; Liu, Yanhua; Chen, Yanan; Zheng, Jianyu; Yang, Shengyong; Fan, Yan; Xiang, Rong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3166-3192</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A series of highly potent, selective inhibitors targeting both CDK4/9 and HDAC1 have been designed and synthesized.  N1-(4-((7-Cyclopentyl-6-(dimethylcarbamoyl)-7H-pyrrolo[2,3-d]pyrimidin-2-yl)amino)phenyl)-N8-hydroxyoctanediamide (I) was discovered.  The lead compd. I with excellent CDK4/9 and HDAC1 inhibitory activity of IC50 = 8.8, 12, and 2.2 nM, resp., can effectively induce apoptosis of cancer cell lines.  The kinase profiling of compd. I showed excellent selectivity and specificity.  Compd. I induces G2/M arrest in high concn. and G0/G1 arrest in low concn. to prevent the proliferation and differentiation of cancer cells.  Mice bared-breast cancer treated with I showed significant antitumor efficacy.  The insight into mechanisms of I indicated that it could induce cancer cell death via cell apoptosis based on CDK4/9 and HDAC1 repression and phosphorylation of p53.  The data demonstrated the compd. I could be a promising drug candidate for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4B7BpJ-foDbVg90H21EOLACvtfcHk0lgodFymyb1XVg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXktFWgs7c%253D&md5=b1ca389e67a3ccd3fe5c005cb547e369</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00209&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00209%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DY.%26aulast%3DLuo%26aufirst%3DX.%26aulast%3DGuo%26aufirst%3DQ.%26aulast%3DNie%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DT.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DHuang%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DZheng%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DS.%26aulast%3DFan%26aufirst%3DY.%26aulast%3DXiang%26aufirst%3DR.%26atitle%3DDiscovery%2520of%2520N1-%25284-%2528%25287-Cyclopentyl-6-%2528dimethylcarbamoyl%2529-7%2520H-pyrrolo%255B2%252C3-d%255Dpyrimidin-2-yl%2529amino%2529phenyl%2529-%2520N8-hydroxyoctanediamide%2520as%2520a%2520Novel%2520Inhibitor%2520Targeting%2520Cyclin-dependent%2520Kinase%25204%252F9%2520%2528CDK4%252F9%2529%2520and%2520Histone%2520Deacetlyase1%2520%2528HDAC1%2529%2520Against%2520Malignant%2520Cancer%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D3166%26epage%3D3192%26doi%3D10.1021%2Facs.jmedchem.8b00209" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tsou, H.-R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Otteng, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tran, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Floyd, M. B.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reich, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Birnberg, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kutterer, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ayral-Kaloustian, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ravi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nilakantan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grillo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGinnis, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rabindran, S. K.</span></span> <span> </span><span class="NLM_article-title">4-(phenylaminomethylene)isoquinoline-1,3(2H,4H)-diones as Potent and Selective Inhibitors of the Cyclin-dependent Kinase 4 (CDK4)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">3507</span>â <span class="NLM_lpage">3525</span>, <span class="refDoi">Â DOI: 10.1021/jm800072z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm800072z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=1%3ACAS%3A528%3ADC%252BD1cXmtF2rtrc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=3507-3525&author=H.-R.+Tsouauthor=M.+Ottengauthor=T.+Tranauthor=M.+B.+Floydauthor=M.+Reichauthor=G.+Birnbergauthor=K.+Kuttererauthor=S.+Ayral-Kaloustianauthor=M.+Raviauthor=R.+Nilakantanauthor=M.+Grilloauthor=J.+P.+McGinnisauthor=S.+K.+Rabindran&title=4-%28phenylaminomethylene%29isoquinoline-1%2C3%282H%2C4H%29-diones+as+Potent+and+Selective+Inhibitors+of+the+Cyclin-dependent+Kinase+4+%28CDK4%29&doi=10.1021%2Fjm800072z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">4-(Phenylaminomethylene)isoquinoline-1,3(2H,4H)-diones as Potent and Selective Inhibitors of the Cyclin-Dependent Kinase 4 (CDK4)</span></div><div class="casAuthors">Tsou, Hwei-Ru; Otteng, Mercy; Tran, Tritin; Floyd, M. Brawner, Jr.; Reich, Marvin; Birnberg, Gary; Kutterer, Kristina; Ayral-Kaloustian, Semiramis; Ravi, Malini; Nilakantan, Ramaswamy; Grillo, Mary; McGinnis, John P.; Rabindran, Sridhar K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3507-3525</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The cyclin-dependent kinases (CDKs), as complexes with their resp. partners, the cyclins, are crit. regulators of cell cycle progression.  Because aberrant regulations of CDK4/cyclin D1 lead to uncontrolled cell proliferation, a hallmark of cancer, small-mol. inhibitors of CDK4/cyclin D1 are attractive as prospective antitumor agents.  The series of 4-(phenylaminomethylene)isoquinoline-1,3(2H,4H)-dione derivs. reported here represents a novel class of potent inhibitors that selectively inhibit CDK4 over CDK2 and CDK1 activities.  In the headpiece of the 4-(phenylaminomethylene)isoquinoline-1,3(2H,4H)-dione, a basic amine substitutent is required on the aniline ring for the CDK4 inhibitory activity.  The inhibitory activity is further enhanced when an aryl or heteroaryl substituent is introduced at the C-6 position of the isoquinoline-1,3(2H,4H)-dione core.  We present here SAR data and a CDK4 mimic model that explains the binding, potency, and selectivity of our CDK4 selective inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq09yU47pubWbVg90H21EOLACvtfcHk0lgodFymyb1XVg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXmtF2rtrc%253D&md5=35083ce22416a584311a810ecf2b624d</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Fjm800072z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm800072z%26sid%3Dliteratum%253Aachs%26aulast%3DTsou%26aufirst%3DH.-R.%26aulast%3DOtteng%26aufirst%3DM.%26aulast%3DTran%26aufirst%3DT.%26aulast%3DFloyd%26aufirst%3DM.%2BB.%26aulast%3DReich%26aufirst%3DM.%26aulast%3DBirnberg%26aufirst%3DG.%26aulast%3DKutterer%26aufirst%3DK.%26aulast%3DAyral-Kaloustian%26aufirst%3DS.%26aulast%3DRavi%26aufirst%3DM.%26aulast%3DNilakantan%26aufirst%3DR.%26aulast%3DGrillo%26aufirst%3DM.%26aulast%3DMcGinnis%26aufirst%3DJ.%2BP.%26aulast%3DRabindran%26aufirst%3DS.%2BK.%26atitle%3D4-%2528phenylaminomethylene%2529isoquinoline-1%252C3%25282H%252C4H%2529-diones%2520as%2520Potent%2520and%2520Selective%2520Inhibitors%2520of%2520the%2520Cyclin-dependent%2520Kinase%25204%2520%2528CDK4%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D3507%26epage%3D3525%26doi%3D10.1021%2Fjm800072z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Del
Re, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertolini, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crucitta, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fontanelli, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rofi, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Angelis, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diodati, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cavallero, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gianfilippo, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salvadori, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fogli, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Falcone, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scatena, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naccarato, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roncella, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghilli, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morganti, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fontana, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Danesi, R.</span></span> <span> </span><span class="NLM_article-title">Overexpression of TK1 and CDK9 in Plasma-derived Exosomes is Associated with Clinical Resistance to CDK4/6 Inhibitors in Metastatic Breast Cancer Patients</span>. <i>Breast Canc. Res. Treat.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>178</i></span>,  <span class="NLM_fpage">57</span>â <span class="NLM_lpage">62</span>, <span class="refDoi">Â DOI: 10.1007/s10549-019-05365-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=10.1007%2Fs10549-019-05365-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=31346846" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsVGrtbzM" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=178&publication_year=2019&pages=57-62&author=M.+Del%0AReauthor=I.+Bertoliniauthor=S.+Crucittaauthor=L.+Fontanelliauthor=E.+Rofiauthor=C.+De+Angelisauthor=L.+Diodatiauthor=D.+Cavalleroauthor=G.+Gianfilippoauthor=B.+Salvadoriauthor=S.+Fogliauthor=A.+Falconeauthor=C.+Scatenaauthor=A.+G.+Naccaratoauthor=M.+Roncellaauthor=M.+Ghilliauthor=R.+Morgantiauthor=A.+Fontanaauthor=R.+Danesi&title=Overexpression+of+TK1+and+CDK9+in+Plasma-derived+Exosomes+is+Associated+with+Clinical+Resistance+to+CDK4%2F6+Inhibitors+in+Metastatic+Breast+Cancer+Patients&doi=10.1007%2Fs10549-019-05365-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Overexpression of TK1 and CDK9 in plasma-derived exosomes is associated with clinical resistance to CDK4/6 inhibitors in metastatic breast cancer patients</span></div><div class="casAuthors">Del Re, Marzia; Bertolini, Ilaria; Crucitta, Stefania; Fontanelli, Lorenzo; Rofi, Eleonora; De Angelis, Claudia; Diodati, Lucrezia; Cavallero, Diletta; Gianfilippo, Giulia; Salvadori, Barbara; Fogli, Stefano; Falcone, Alfredo; Scatena, Cristian; Naccarato, Antonio Giuseppe; Roncella, Manuela; Ghilli, Matteo; Morganti, Riccardo; Fontana, Andrea; Danesi, Romano</div><div class="citationInfo"><span class="NLM_cas:title">Breast Cancer Research and Treatment</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">178</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">57-62</span>CODEN:
                <span class="NLM_cas:coden">BCTRD6</span>;
        ISSN:<span class="NLM_cas:issn">0167-6806</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">The present article addresses the role of the mRNA expression of thymidine kinase 1 (TK1), CDK4, 6 and 9 in plasma-derived exosomes and their relevance in the pharmacol. activity of CDK4/6i.  Methods: Blood samples of 40 HR+/HER2- advanced breast cancer patients were collected before (T0) the administration of palbociclib plus hormonal therapy and after 3 mo (T1).  RNA was isolated from exosomes and analyzed for the expression of TK1, CDK 4, 6 and 9 by digital droplet PCR (ddPCR).  Results: A higher value of TK1 copies/mL at baseline (T0) was significantly assocd. with the no. of previous lines of chemotherapy (p = 0.009).  In patients with PD, a significant increase was obsd. in the no. of copies/mL of TK1 (p = 0.01) and CDK9 (p = 0.03) comparing T1 vs.  T0 values. No significant correlations between response to treatment and clin. parameters were found at univariate anal.  High baseline CDK4 expression was significantly correlated with longer PFS in patients treated with fulvestrant + palbociclib (low vs. high: 6.45 mo vs. not reached, p = 0.01).  Conclusions: The present study demonstrates that, in plasma-derived exosomes, high baseline CDK4 mRNA levels are assocd. with response to palbociclib plus hormonal therapy, while the increase in TK1 and CDK9 mRNA copies/mL is assocd. with clin. resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpAoJIlNIJ7NLVg90H21EOLACvtfcHk0lhQ2XbPKndqfw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsVGrtbzM&md5=e5684c7965c4527fe92c7c5c9869e428</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1007%2Fs10549-019-05365-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10549-019-05365-y%26sid%3Dliteratum%253Aachs%26aulast%3DDel%2BRe%26aufirst%3DM.%26aulast%3DBertolini%26aufirst%3DI.%26aulast%3DCrucitta%26aufirst%3DS.%26aulast%3DFontanelli%26aufirst%3DL.%26aulast%3DRofi%26aufirst%3DE.%26aulast%3DDe%2BAngelis%26aufirst%3DC.%26aulast%3DDiodati%26aufirst%3DL.%26aulast%3DCavallero%26aufirst%3DD.%26aulast%3DGianfilippo%26aufirst%3DG.%26aulast%3DSalvadori%26aufirst%3DB.%26aulast%3DFogli%26aufirst%3DS.%26aulast%3DFalcone%26aufirst%3DA.%26aulast%3DScatena%26aufirst%3DC.%26aulast%3DNaccarato%26aufirst%3DA.%2BG.%26aulast%3DRoncella%26aufirst%3DM.%26aulast%3DGhilli%26aufirst%3DM.%26aulast%3DMorganti%26aufirst%3DR.%26aulast%3DFontana%26aufirst%3DA.%26aulast%3DDanesi%26aufirst%3DR.%26atitle%3DOverexpression%2520of%2520TK1%2520and%2520CDK9%2520in%2520Plasma-derived%2520Exosomes%2520is%2520Associated%2520with%2520Clinical%2520Resistance%2520to%2520CDK4%252F6%2520Inhibitors%2520in%2520Metastatic%2520Breast%2520Cancer%2520Patients%26jtitle%3DBreast%2520Canc.%2520Res.%2520Treat.%26date%3D2019%26volume%3D178%26spage%3D57%26epage%3D62%26doi%3D10.1007%2Fs10549-019-05365-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sengupta, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biarnes, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jordan, V. C.</span></span> <span> </span><span class="NLM_article-title">Cyclin Dependent Kinase-9 Mediated Transcriptional De-regulation of cMYC as a Critical Determinant of Endocrine-therapy Resistance in Breast cancers</span>. <i>Breast Canc. Res. Treat.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>143</i></span>,  <span class="NLM_fpage">113</span>â <span class="NLM_lpage">124</span>, <span class="refDoi">Â DOI: 10.1007/s10549-013-2789-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=10.1007%2Fs10549-013-2789-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=24309997" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFKgu7jN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=143&publication_year=2014&pages=113-124&author=S.+Senguptaauthor=M.+C.+Biarnesauthor=V.+C.+Jordan&title=Cyclin+Dependent+Kinase-9+Mediated+Transcriptional+De-regulation+of+cMYC+as+a+Critical+Determinant+of+Endocrine-therapy+Resistance+in+Breast+cancers&doi=10.1007%2Fs10549-013-2789-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Cyclin dependent kinase-9 mediated transcriptional de-regulation of cMYC as a critical determinant of endocrine-therapy resistance in breast cancers</span></div><div class="casAuthors">Sengupta, Surojeet; Biarnes, Michael C.; Jordan, V. Craig</div><div class="citationInfo"><span class="NLM_cas:title">Breast Cancer Research and Treatment</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">143</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">113-124</span>CODEN:
                <span class="NLM_cas:coden">BCTRD6</span>;
        ISSN:<span class="NLM_cas:issn">0167-6806</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Endocrine therapy resistance in estrogen receptor alpha pos. (ERÎ±+) breast cancers remains a major obstacle for maintaining efficacy of targeted therapies.  We investigated the significance and the mechanisms involved in cMYC over-expression in a MCF7 derived panel of ERÎ±+ breast cancer cells which can proliferate in the absence of estrogen with different sensitivities to anti-hormone therapies.  We show that all the resistant cell lines tested over-express cMYC as compared to parental MCF7 cells and its inhibition lead to the differential blocking of estrogen-independent proliferation in resistant cells.  Further investigation of the resistant cell line, MCF7:5C, suggested transcriptional de-regulation of cMYC gene was responsible for its over-expression.  Chromatin immuno-pptn. assay revealed markedly higher recruitment of phosphorylated serine-2 carboxy-terminal domain (CTD) of RNA polymerase-II at the proximal promoter of cMYC gene, which is responsible for transcriptional elongation of the cMYC RNA.  The level of CDK9, a factor responsible for the phosphorylation of serine-2 of RNA polymerase II CTD, was found to be elevated in all the resistant cell lines.  Pharmacol. inhibition of CDK9 not only reduced the transcripts and the protein levels of cMYC in MCF7:5C cells but also selectively inhibited the estrogen-independent growth of all the resistant cell lines.  This study describes the up-stream mol. events involved in the transcriptional over-expression of cMYC gene in breast cancer cells proliferating estrogen-independently and identifies CDK9 as a potential novel drug target for therapeutic intervention in endocrine-resistant breast cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoY3q-BuILrXrVg90H21EOLACvtfcHk0lhQ2XbPKndqfw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFKgu7jN&md5=9d79fb5d2cab2f59dade67c2f6ac7747</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1007%2Fs10549-013-2789-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10549-013-2789-2%26sid%3Dliteratum%253Aachs%26aulast%3DSengupta%26aufirst%3DS.%26aulast%3DBiarnes%26aufirst%3DM.%2BC.%26aulast%3DJordan%26aufirst%3DV.%2BC.%26atitle%3DCyclin%2520Dependent%2520Kinase-9%2520Mediated%2520Transcriptional%2520De-regulation%2520of%2520cMYC%2520as%2520a%2520Critical%2520Determinant%2520of%2520Endocrine-therapy%2520Resistance%2520in%2520Breast%2520cancers%26jtitle%3DBreast%2520Canc.%2520Res.%2520Treat.%26date%3D2014%26volume%3D143%26spage%3D113%26epage%3D124%26doi%3D10.1007%2Fs10549-013-2789-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeidner, J. F.</span></span> <span> </span><span class="NLM_article-title">Cyclin-dependent Kinase (CDK) 9 and 4/6 Inhibitors in Acute Myeloid Leukemia (AML): a Promising Therapeutic Approach</span>. <i>Expet Opin. Invest. Drugs</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">989</span>â <span class="NLM_lpage">1001</span>, <span class="refDoi">Â DOI: 10.1080/13543784.2019.1678583</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=10.1080%2F13543784.2019.1678583" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=31612739" title="">PubMed</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2019&pages=989-1001&author=D.+J.+Leeauthor=J.+F.+Zeidner&title=Cyclin-dependent+Kinase+%28CDK%29+9+and+4%2F6+Inhibitors+in+Acute+Myeloid+Leukemia+%28AML%29%3A+a+Promising+Therapeutic+Approach&doi=10.1080%2F13543784.2019.1678583"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1080%2F13543784.2019.1678583&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F13543784.2019.1678583%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DD.%2BJ.%26aulast%3DZeidner%26aufirst%3DJ.%2BF.%26atitle%3DCyclin-dependent%2520Kinase%2520%2528CDK%2529%25209%2520and%25204%252F6%2520Inhibitors%2520in%2520Acute%2520Myeloid%2520Leukemia%2520%2528AML%2529%253A%2520a%2520Promising%2520Therapeutic%2520Approach%26jtitle%3DExpet%2520Opin.%2520Invest.%2520Drugs%26date%3D2019%26volume%3D28%26spage%3D989%26epage%3D1001%26doi%3D10.1080%2F13543784.2019.1678583" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dean, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hornicek, F. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duan, Z.</span></span> <span> </span><span class="NLM_article-title">Cyclin-dependent kinase 9 (CDK9) is a Novel Prognostic Marker and Therapeutic Target in Ovarian Cancer</span>. <i>FASEB J.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">5990</span>â <span class="NLM_lpage">6000</span>, <span class="refDoi">Â DOI: 10.1096/fj.201801789rr</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=10.1096%2Ffj.201801789RR" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=30726104" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtFChs7zO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2019&pages=5990-6000&author=J.+Wangauthor=D.+C.+Deanauthor=F.+J.+Hornicekauthor=H.+Shiauthor=Z.+Duan&title=Cyclin-dependent+kinase+9+%28CDK9%29+is+a+Novel+Prognostic+Marker+and+Therapeutic+Target+in+Ovarian+Cancer&doi=10.1096%2Ffj.201801789rr"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Cyclin-dependent kinase 9 (CDK9) is a novel prognostic marker and therapeutic target in ovarian cancer</span></div><div class="casAuthors">Wang, Jinglu; Dean, Dylan C.; Hornicek, Francis J.; Shi, Huirong; Duan, Zhenfeng</div><div class="citationInfo"><span class="NLM_cas:title">FASEB Journal</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">5990-6000</span>CODEN:
                <span class="NLM_cas:coden">FAJOEC</span>;
        ISSN:<span class="NLM_cas:issn">0892-6638</span>.
    
            (<span class="NLM_cas:orgname">Federation of American Societies for Experimental Biology</span>)
        </div><div class="casAbstract">Despite surgical and chemotherapeutic advances over the past few decades, the prognosis for ovarian cancer remains very poor.  Although cyclin-dependent kinase (CDK) 9 has an established pathogenic role in various cancers, its function in ovarian cancer remains poorly defined.  The purpose of this study was to evaluate the expression of CDK9 and its therapeutic potential in ovarian cancer.  CDK9 expression was detd. by immunohistochem. in a unique ovarian cancer tissue microarray constructed with paired primary, metastatic, and recurrent tumor tissues from 26 ovarian cancer patients.  CDK9 was highly expressed in human ovarian cancer cell lines and was also elevated in metastatic and recurrent ovarian tumor tissue compared with patient-matched primary ovarian tumor tissue.  In addn., increased CDK9 significantly correlated with poor patient prognosis.  Inhibition of CDK9 by small interfering RNA or CDK9 inhibitor functionally suppressed RNA transcription elongation, induced apoptosis, and reduced proliferation of ovarian cancer cells.  Inhibition of CDK9 also suppressed ovarian cancer cell spheroid growth, clonogenicity formation, and migration activity.  Our results reveal CDK9 as a novel prognostic biomarker and a promising therapeutic target for preventing metastasis and recurrence while also improving the overall clin. outcome for ovarian cancer patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTuSeIUOKO-LVg90H21EOLACvtfcHk0liuIstUgngsrg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtFChs7zO&md5=cc691e71c659a48477b30130ccf6cd24</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1096%2Ffj.201801789RR&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1096%252Ffj.201801789RR%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DDean%26aufirst%3DD.%2BC.%26aulast%3DHornicek%26aufirst%3DF.%2BJ.%26aulast%3DShi%26aufirst%3DH.%26aulast%3DDuan%26aufirst%3DZ.%26atitle%3DCyclin-dependent%2520kinase%25209%2520%2528CDK9%2529%2520is%2520a%2520Novel%2520Prognostic%2520Marker%2520and%2520Therapeutic%2520Target%2520in%2520Ovarian%2520Cancer%26jtitle%3DFASEB%2520J.%26date%3D2019%26volume%3D33%26spage%3D5990%26epage%3D6000%26doi%3D10.1096%2Ffj.201801789rr" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, N. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferre, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergqvist, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Solowiej, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diehl, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, Y.-A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagata, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">VanArsdale, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, B. W.</span></span> <span> </span><span class="NLM_article-title">Spectrum and Degree of CDK Drug Interactions Predicts Clinical Performance</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">2273</span>â <span class="NLM_lpage">2281</span>, <span class="refDoi">Â DOI: 10.1158/1535-7163.mct-16-0300</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=10.1158%2F1535-7163.MCT-16-0300" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=27496135" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=1%3ACAS%3A528%3ADC%252BC28XhslGisbjK" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=2273-2281&author=P.+Chenauthor=N.+V.+Leeauthor=W.+Huauthor=M.+Xuauthor=R.+A.+Ferreauthor=H.+Lamauthor=S.+Bergqvistauthor=J.+Solowiejauthor=W.+Diehlauthor=Y.-A.+Heauthor=X.+Yuauthor=A.+Nagataauthor=T.+VanArsdaleauthor=B.+W.+Murray&title=Spectrum+and+Degree+of+CDK+Drug+Interactions+Predicts+Clinical+Performance&doi=10.1158%2F1535-7163.mct-16-0300"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Spectrum and Degree of CDK Drug Interactions Predicts Clinical Performance</span></div><div class="casAuthors">Chen, Ping; Lee, Nathan V.; Hu, Wenyue; Xu, Meirong; Ferre, Rose Ann; Lam, Hieu; Bergqvist, Simon; Solowiej, James; Diehl, Wade; He, You-Ai; Yu, Xiu; Nagata, Asako; Van Arsdale, Todd; Murray, Brion W.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2273-2281</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Therapeutically targeting aberrant intracellular kinase signaling is attractive from a biol. perspective but drug development is often hindered by toxicities and inadequate efficacy.  Predicting drug behaviors using cellular and animal models is confounded by redundant kinase activities, a lack of unique substrates, and cell-specific signaling networks.  Cyclin-dependent kinase (CDK) drugs exemplify this phenomenon because they are reported to target common processes yet have distinct clin. activities.  Tumor cell studies of ATP-competitive CDK drugs (dinaciclib, AG-024322, abemaciclib, palbociclib, ribociclib) indicate similar pharmacol. while analyses in untransformed cells illuminates significant differences.  To resolve this apparent disconnect, drug behaviors are described at the mol. level.  Nonkinase binding studies and kinome interaction anal. (recombinant and endogenous kinases) reveal that proteins outside of the CDK family appear to have little role in dinaciclib/palbociclib/ribociclib pharmacol., may contribute for abemaciclib, and confounds AG-024322 anal.  CDK2 and CDK6 cocrystal structures with the drugs identify the mol. interactions responsible for potency and kinase selectivity.  Efficient drug binding to the unique hinge architecture of CDKs enables selectivity toward most of the human kinome.  Selectivity between CDK family members is achieved through interactions with nonconserved elements of the ATP-binding pocket.  Integrating clin. drug exposures into the anal. predicts that both palbociclib and ribociclib are CDK4/6 inhibitors, abemaciclib inhibits CDK4/6/9, and dinaciclib is a broad-spectrum CDK inhibitor (CDK2/3/4/6/9).  Understanding the mol. components of potency and selectivity also facilitates rational design of future generations of kinase-directed drugs.  Mol Cancer Ther; 15(10); 2273-81.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3TbbI98T8v7Vg90H21EOLACvtfcHk0liuIstUgngsrg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhslGisbjK&md5=112e9e46bb33393516fa94620e6383c6</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-16-0300&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-16-0300%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DP.%26aulast%3DLee%26aufirst%3DN.%2BV.%26aulast%3DHu%26aufirst%3DW.%26aulast%3DXu%26aufirst%3DM.%26aulast%3DFerre%26aufirst%3DR.%2BA.%26aulast%3DLam%26aufirst%3DH.%26aulast%3DBergqvist%26aufirst%3DS.%26aulast%3DSolowiej%26aufirst%3DJ.%26aulast%3DDiehl%26aufirst%3DW.%26aulast%3DHe%26aufirst%3DY.-A.%26aulast%3DYu%26aufirst%3DX.%26aulast%3DNagata%26aufirst%3DA.%26aulast%3DVanArsdale%26aufirst%3DT.%26aulast%3DMurray%26aufirst%3DB.%2BW.%26atitle%3DSpectrum%2520and%2520Degree%2520of%2520CDK%2520Drug%2520Interactions%2520Predicts%2520Clinical%2520Performance%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2016%26volume%3D15%26spage%3D2273%26epage%3D2281%26doi%3D10.1158%2F1535-7163.mct-16-0300" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Minzel, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venkatachalam, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fink, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brachya, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burstain, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaham, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rivlin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Omer, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zinger, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elias, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winter, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erdman, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sullivan, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fung, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mercurio, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vacca, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaushansky, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shlush, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oren, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levine, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pikarsky, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snir-Alkalay, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ben-Neriah, Y.</span></span> <span> </span><span class="NLM_article-title">Small Molecules Co-targeting CKIalpha and the Transcriptional Kinases CDK7/9 Control AML in Preclinical Models</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>175</i></span>,  <span class="NLM_fpage">171</span>â <span class="NLM_lpage">185</span>, <span class="refDoi">Â DOI: 10.1016/j.cell.2018.07.045</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=10.1016%2Fj.cell.2018.07.045" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=30146162" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsFyktbvF" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=175&publication_year=2018&pages=171-185&author=W.+Minzelauthor=A.+Venkatachalamauthor=A.+Finkauthor=E.+Hungauthor=G.+Brachyaauthor=I.+Burstainauthor=M.+Shahamauthor=A.+Rivlinauthor=I.+Omerauthor=A.+Zingerauthor=S.+Eliasauthor=E.+Winterauthor=P.+E.+Erdmanauthor=R.+W.+Sullivanauthor=L.+Fungauthor=F.+Mercurioauthor=D.+Liauthor=J.+Vaccaauthor=N.+Kaushanskyauthor=L.+Shlushauthor=M.+Orenauthor=R.+Levineauthor=E.+Pikarskyauthor=I.+Snir-Alkalayauthor=Y.+Ben-Neriah&title=Small+Molecules+Co-targeting+CKIalpha+and+the+Transcriptional+Kinases+CDK7%2F9+Control+AML+in+Preclinical+Models&doi=10.1016%2Fj.cell.2018.07.045"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Small Molecules Co-targeting CKIÎ± and the Transcriptional Kinases CDK7/9 Control AML in Preclinical Models</span></div><div class="casAuthors">Minzel, Waleed; Venkatachalam, Avanthika; Fink, Avner; Hung, Eric; Brachya, Guy; Burstain, Ido; Shaham, Maya; Rivlin, Amitai; Omer, Itay; Zinger, Adar; Elias, Shlomo; Winter, Eitan; Erdman, Paul E.; Sullivan, Robert W.; Fung, Leah; Mercurio, Frank; Li, Dansu; Vacca, Joseph; Kaushansky, Nathali; Shlush, Liran; Oren, Moshe; Levine, Ross; Pikarsky, Eli; Snir-Alkalay, Irit; Ben-Neriah, Yinon</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">175</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">171-185.e25</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">CKIÎ± ablation induces p53 activation, and CKIÎ± degrdn. underlies the therapeutic effect of lenalidomide in a pre-leukemia syndrome.  Here we describe the development of CKIÎ± inhibitors, which co-target the transcriptional kinases CDK7 and CDK9, thereby augmenting CKIÎ±-induced p53 activation and its anti-leukemic activity.  Oncogene-driving super-enhancers (SEs) are highly sensitive to CDK7/9 inhibition.  We identified multiple newly gained SEs in primary mouse acute myeloid leukemia (AML) cells and demonstrate that the inhibitors abolish many SEs and preferentially suppress the transcription elongation of SE-driven oncogenes.  We show that blocking CKIÎ± together with CDK7 and/or CDK9 synergistically stabilize p53, deprive leukemia cells of survival and proliferation-maintaining SE-driven oncogenes, and induce apoptosis.  Leukemia progenitors are selectively eliminated by the inhibitors, explaining their therapeutic efficacy with preserved hematopoiesis and leukemia cure potential; they eradicate leukemia in MLL-AF9 and Tet2-/-;Flt3ITD AML mouse models and in several patient-derived AML xenograft models, supporting their potential efficacy in curing human leukemia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqg4QEH6R_ll7Vg90H21EOLACvtfcHk0lgIPyQW-0MWIw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsFyktbvF&md5=b3e088a6b262dc8998ff334867102aa5</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2018.07.045&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2018.07.045%26sid%3Dliteratum%253Aachs%26aulast%3DMinzel%26aufirst%3DW.%26aulast%3DVenkatachalam%26aufirst%3DA.%26aulast%3DFink%26aufirst%3DA.%26aulast%3DHung%26aufirst%3DE.%26aulast%3DBrachya%26aufirst%3DG.%26aulast%3DBurstain%26aufirst%3DI.%26aulast%3DShaham%26aufirst%3DM.%26aulast%3DRivlin%26aufirst%3DA.%26aulast%3DOmer%26aufirst%3DI.%26aulast%3DZinger%26aufirst%3DA.%26aulast%3DElias%26aufirst%3DS.%26aulast%3DWinter%26aufirst%3DE.%26aulast%3DErdman%26aufirst%3DP.%2BE.%26aulast%3DSullivan%26aufirst%3DR.%2BW.%26aulast%3DFung%26aufirst%3DL.%26aulast%3DMercurio%26aufirst%3DF.%26aulast%3DLi%26aufirst%3DD.%26aulast%3DVacca%26aufirst%3DJ.%26aulast%3DKaushansky%26aufirst%3DN.%26aulast%3DShlush%26aufirst%3DL.%26aulast%3DOren%26aufirst%3DM.%26aulast%3DLevine%26aufirst%3DR.%26aulast%3DPikarsky%26aufirst%3DE.%26aulast%3DSnir-Alkalay%26aufirst%3DI.%26aulast%3DBen-Neriah%26aufirst%3DY.%26atitle%3DSmall%2520Molecules%2520Co-targeting%2520CKIalpha%2520and%2520the%2520Transcriptional%2520Kinases%2520CDK7%252F9%2520Control%2520AML%2520in%2520Preclinical%2520Models%26jtitle%3DCell%26date%3D2018%26volume%3D175%26spage%3D171%26epage%3D185%26doi%3D10.1016%2Fj.cell.2018.07.045" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicolay, B. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chick, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suski, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keibler, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sicinska, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerdemann, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haining, W. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polyak, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gygi, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dyson, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sicinski, P.</span></span> <span> </span><span class="NLM_article-title">The Metabolic Function of Cyclin D3-CDK6 Kinase in Cancer Cell Survival</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>546</i></span>,  <span class="NLM_fpage">426</span>â <span class="NLM_lpage">430</span>, <span class="refDoi">Â DOI: 10.1038/nature22797</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=10.1038%2Fnature22797" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=28607489" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=1%3ACAS%3A528%3ADC%252BC2sXpt1Oqtb0%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=546&publication_year=2017&pages=426-430&author=H.+Wangauthor=B.+N.+Nicolayauthor=J.+M.+Chickauthor=X.+Gaoauthor=Y.+Gengauthor=H.+Renauthor=H.+Gaoauthor=G.+Yangauthor=J.+A.+Williamsauthor=J.+M.+Suskiauthor=M.+A.+Keiblerauthor=E.+Sicinskaauthor=U.+Gerdemannauthor=W.+N.+Hainingauthor=T.+M.+Robertsauthor=K.+Polyakauthor=S.+P.+Gygiauthor=N.+J.+Dysonauthor=P.+Sicinski&title=The+Metabolic+Function+of+Cyclin+D3-CDK6+Kinase+in+Cancer+Cell+Survival&doi=10.1038%2Fnature22797"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">The metabolic function of cyclin D3-CDK6 kinase in cancer cell survival</span></div><div class="casAuthors">Wang, Haizhen; Nicolay, Brandon N.; Chick, Joel M.; Gao, Xueliang; Geng, Yan; Ren, Hong; Gao, Hui; Yang, Guizhi; Williams, Juliet A.; Suski, Jan M.; Keibler, Mark A.; Sicinska, Ewa; Gerdemann, Ulrike; Haining, W. Nicholas; Roberts, Thomas M.; Polyak, Kornelia; Gygi, Steven P.; Dyson, Nicholas J.; Sicinski, Piotr</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">546</span>
        (<span class="NLM_cas:issue">7658</span>),
    <span class="NLM_cas:pages">426-430</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">D-type cyclins (D1, D2 and D3) and their assocd. cyclin-dependent kinases (CDK4 and CDK6) are components of the core cell cycle machinery that drives cell proliferation.  Inhibitors of CDK4 and CDK6 are currently being tested in clin. trials for patients with several cancer types, with promising results.  Here, using human cancer cells and patient-derived xenografts in mice, we show that the cyclin D3-CDK6 kinase phosphorylates and inhibits the catalytic activity of two key enzymes in the glycolytic pathway, 6-phosphofructokinase and pyruvate kinase M2.  This re-directs the glycolytic intermediates into the pentose phosphate (PPP) and serine pathways.  Inhibition of cyclin D3-CDK6 in tumor cells reduces flow through the PPP and serine pathways, thereby depleting the antioxidants NADPH and glutathione.  This, in turn, increases the levels of reactive oxygen species and causes apoptosis of tumor cells.  The pro-survival function of cyclin D-assocd. kinase operates in tumors expressing high levels of cyclin D3-CDK6 complexes.  We propose that measuring the levels of cyclin D3-CDK6 in human cancers might help to identify tumor subsets that undergo cell death and tumor regression upon inhibition of CDK4 and CDK6.  Cyclin D3-CDK6, through its ability to link cell cycle and cell metab., represents a particularly powerful oncoprotein that affects cancer cells at several levels, and this property can be exploited for anti-cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqD4-8f9Q8KErVg90H21EOLACvtfcHk0lgIPyQW-0MWIw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXpt1Oqtb0%253D&md5=f2539399d1c4fb2163fb245d25c07843</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1038%2Fnature22797&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature22797%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DH.%26aulast%3DNicolay%26aufirst%3DB.%2BN.%26aulast%3DChick%26aufirst%3DJ.%2BM.%26aulast%3DGao%26aufirst%3DX.%26aulast%3DGeng%26aufirst%3DY.%26aulast%3DRen%26aufirst%3DH.%26aulast%3DGao%26aufirst%3DH.%26aulast%3DYang%26aufirst%3DG.%26aulast%3DWilliams%26aufirst%3DJ.%2BA.%26aulast%3DSuski%26aufirst%3DJ.%2BM.%26aulast%3DKeibler%26aufirst%3DM.%2BA.%26aulast%3DSicinska%26aufirst%3DE.%26aulast%3DGerdemann%26aufirst%3DU.%26aulast%3DHaining%26aufirst%3DW.%2BN.%26aulast%3DRoberts%26aufirst%3DT.%2BM.%26aulast%3DPolyak%26aufirst%3DK.%26aulast%3DGygi%26aufirst%3DS.%2BP.%26aulast%3DDyson%26aufirst%3DN.%2BJ.%26aulast%3DSicinski%26aufirst%3DP.%26atitle%3DThe%2520Metabolic%2520Function%2520of%2520Cyclin%2520D3-CDK6%2520Kinase%2520in%2520Cancer%2520Cell%2520Survival%26jtitle%3DNature%26date%3D2017%26volume%3D546%26spage%3D426%26epage%3D430%26doi%3D10.1038%2Fnature22797" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sathe, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koshy, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmid, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thalgott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">SchwarzenbÃ¶ck, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krause, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holm, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gschwend, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Retz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nawroth, R.</span></span> <span> </span><span class="NLM_article-title">CDK4/6 Inhibition Controls Proliferation of Bladder Cancer and Transcription of RB1</span>. <i>J. Urol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>195</i></span>,  <span class="NLM_fpage">771</span>â <span class="NLM_lpage">779</span>, <span class="refDoi">Â DOI: 10.1016/j.juro.2015.08.082</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=10.1016%2Fj.juro.2015.08.082" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=26318986" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVOqsr7O" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=195&publication_year=2016&pages=771-779&author=A.+Satheauthor=N.+Koshyauthor=S.+C.+Schmidauthor=M.+Thalgottauthor=S.+M.+Schwarzenb%C3%B6ckauthor=B.+J.+Krauseauthor=P.+S.+Holmauthor=J.+E.+Gschwendauthor=M.+Retzauthor=R.+Nawroth&title=CDK4%2F6+Inhibition+Controls+Proliferation+of+Bladder+Cancer+and+Transcription+of+RB1&doi=10.1016%2Fj.juro.2015.08.082"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">CDK4/6 Inhibition Controls Proliferation of Bladder Cancer and Transcription of RB1</span></div><div class="casAuthors">Sathe, Anuja; Koshy, Nicole; Schmid, Sebastian C.; Thalgott, Mark; Schwarzenboeck, Sarah M.; Krause, Bernd J.; Holm, Per S.; Gschwend, Juergen E.; Retz, Margitta; Nawroth, Roman</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Urology (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">195</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">771-779</span>CODEN:
                <span class="NLM_cas:coden">JOURAA</span>;
        ISSN:<span class="NLM_cas:issn">0022-5347</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">The retinoblastoma signaling network is frequently altered in advanced bladder cancer.  We investigated the potential of CDK4/6 as a therapeutic target and detd. biomarkers for patient stratification.  Genetic alterations were analyzed using public databases, including TCGA (The Cancer Genome Atlas), COSMIC (Catalog of Somatic Mutations in Cancer) and CCLE (Cancer Cell Line Encyclopedia).  Effects of the CDK4/6-inhibitor PD-0332991 or LY2835219 were examd. in 10 bladder cancer cell lines by immunoblot, cell viability, apoptosis and cell cycle progression.  Efficacy of the PD-0332991 and cisplatin combination was analyzed using the combination index.  Gene expression level was detd. by quant. polymerase chain reaction.  Cytomegalovirus promoter regulated recombinant retinoblastoma was used for reconstitution.  Three-dimensional xenografts were grown on chicken chorioallantoic membrane and analyzed by measuring tumor wt. and immunohistochem. expression of total retinoblastoma and Ki-67.  PD-0332991 treatment decreased the proliferation of retinoblastoma pos. bladder cancer cell lines and was synergistic in combination with cisplatin.  PD-0332991 or LY2835219 treatment decreased the phosphorylation, total protein and transcript level of retinoblastoma.  Treatment resulted in a decrease in E2F target gene expression (CCNA2 and CCNE2) and cell cycle progression from G0/G1 to the S-phase but did not affect apoptosis.  In retinoblastoma neg. cells reconstituted with recombinant retinoblastoma PD-0332991 affected only phosphorylation and not the total retinoblastoma level.  These cells remained resistant to treatment.  In 3-dimensional retinoblastoma xenografts, treatment resulted in reduced tumor wt. and decreased expression of total retinoblastoma and Ki-67.  We provide preclin. evidence that CDK4/6 inhibition is a potential therapeutic strategy for retinoblastoma pos. bladder cancer that probably acts by neg. regulating retinoblastoma transcription.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXXQ64fNhPSLVg90H21EOLACvtfcHk0lgIPyQW-0MWIw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVOqsr7O&md5=7b141310df974995067f1fc1a369f676</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2Fj.juro.2015.08.082&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.juro.2015.08.082%26sid%3Dliteratum%253Aachs%26aulast%3DSathe%26aufirst%3DA.%26aulast%3DKoshy%26aufirst%3DN.%26aulast%3DSchmid%26aufirst%3DS.%2BC.%26aulast%3DThalgott%26aufirst%3DM.%26aulast%3DSchwarzenb%25C3%25B6ck%26aufirst%3DS.%2BM.%26aulast%3DKrause%26aufirst%3DB.%2BJ.%26aulast%3DHolm%26aufirst%3DP.%2BS.%26aulast%3DGschwend%26aufirst%3DJ.%2BE.%26aulast%3DRetz%26aufirst%3DM.%26aulast%3DNawroth%26aufirst%3DR.%26atitle%3DCDK4%252F6%2520Inhibition%2520Controls%2520Proliferation%2520of%2520Bladder%2520Cancer%2520and%2520Transcription%2520of%2520RB1%26jtitle%3DJ.%2520Urol.%26date%3D2016%26volume%3D195%26spage%3D771%26epage%3D779%26doi%3D10.1016%2Fj.juro.2015.08.082" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Molina, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jafari, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ignatushchenko, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larsson, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sreekumar, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nordlund, P.</span></span> <span> </span><span class="NLM_article-title">Monitoring Drug Target Engagement in Cells and Tissues Using the Cellular Thermal Shift Assay</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>341</i></span>,  <span class="NLM_fpage">84</span>â <span class="NLM_lpage">87</span>, <span class="refDoi">Â DOI: 10.1126/science.1233606</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=10.1126%2Fscience.1233606" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=23828940" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtVCgs7rP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=341&publication_year=2013&pages=84-87&author=D.+M.+Molinaauthor=R.+Jafariauthor=M.+Ignatushchenkoauthor=T.+Sekiauthor=E.+A.+Larssonauthor=C.+Danauthor=L.+Sreekumarauthor=Y.+Caoauthor=P.+Nordlund&title=Monitoring+Drug+Target+Engagement+in+Cells+and+Tissues+Using+the+Cellular+Thermal+Shift+Assay&doi=10.1126%2Fscience.1233606"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Monitoring Drug Target Engagement in Cells and Tissues Using the Cellular Thermal Shift Assay</span></div><div class="casAuthors">Molina, Daniel Martinez; Jafari, Rozbeh; Ignatushchenko, Marina; Seki, Takahiro; Larsson, E. Andreas; Dan, Chen; Sreekumar, Lekshmy; Cao, Yihai; Nordlund, Paer</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">341</span>
        (<span class="NLM_cas:issue">6141</span>),
    <span class="NLM_cas:pages">84-87</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The efficacy of therapeutics is dependent on a drug binding to its cognate target.  Optimization of target engagement by drugs in cells is often challenging, because drug binding cannot be monitored inside cells.  We have developed a method for evaluating drug binding to target proteins in cells and tissue samples.  This cellular thermal shift assay (CETSA) is based on the biophys. principle of ligand-induced thermal stabilization of target proteins.  Using this assay, we validated drug binding for a set of important clin. targets and monitored processes of drug transport and activation, off-target effects and drug resistance in cancer cell lines, as well as drug distribution in tissues.  CETSA is likely to become a valuable tool for the validation and optimization of drug target engagement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoaaWSoozY7ObVg90H21EOLACvtfcHk0lge2KStlU-dFw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtVCgs7rP&md5=f8ba686ba38b67dbdf366c49eb7375b4</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1126%2Fscience.1233606&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1233606%26sid%3Dliteratum%253Aachs%26aulast%3DMolina%26aufirst%3DD.%2BM.%26aulast%3DJafari%26aufirst%3DR.%26aulast%3DIgnatushchenko%26aufirst%3DM.%26aulast%3DSeki%26aufirst%3DT.%26aulast%3DLarsson%26aufirst%3DE.%2BA.%26aulast%3DDan%26aufirst%3DC.%26aulast%3DSreekumar%26aufirst%3DL.%26aulast%3DCao%26aufirst%3DY.%26aulast%3DNordlund%26aufirst%3DP.%26atitle%3DMonitoring%2520Drug%2520Target%2520Engagement%2520in%2520Cells%2520and%2520Tissues%2520Using%2520the%2520Cellular%2520Thermal%2520Shift%2520Assay%26jtitle%3DScience%26date%3D2013%26volume%3D341%26spage%3D84%26epage%3D87%26doi%3D10.1126%2Fscience.1233606" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wienken, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baaske, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rothbauer, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Braun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duhr, S.</span></span> <span> </span><span class="NLM_article-title">Protein-binding Assays in Biological Liquids Using Microscale Thermophoresis</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">100</span>, <span class="refDoi">Â DOI: 10.1038/ncomms1093</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=10.1038%2Fncomms1093" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=20981028" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=1%3ACAS%3A280%3ADC%252BC3cbisleguw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2010&pages=100&author=C.+J.+Wienkenauthor=P.+Baaskeauthor=U.+Rothbauerauthor=D.+Braunauthor=S.+Duhr&title=Protein-binding+Assays+in+Biological+Liquids+Using+Microscale+Thermophoresis&doi=10.1038%2Fncomms1093"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Protein-binding assays in biological liquids using microscale thermophoresis</span></div><div class="casAuthors">Wienken Christoph J; Baaske Philipp; Rothbauer Ulrich; Braun Dieter; Duhr Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">100</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Protein interactions inside the human body are expected to differ from the situation in vitro.  This is crucial when investigating protein functions or developing new drugs.  In this study, we present a sample-efficient, free-solution method, termed microscale thermophoresis, that is capable of analysing interactions of proteins or small molecules in biological liquids such as blood serum or cell lysate.  The technique is based on the thermophoresis of molecules, which provides information about molecule size, charge and hydration shell.  We validated the method using immunologically relevant systems including human interferon gamma and the interaction of calmodulin with calcium.  The affinity of the small-molecule inhibitor quercetin to its kinase PKA was determined in buffer and human serum, revealing a 400-fold reduced affinity in serum.  This information about the influence of the biological matrix may allow to make more reliable conclusions on protein functionality, and may facilitate more efficient drug development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTOCqf6rsXbPv6BottYdIAWfW6udTcc2eZSDeiwCg5THrntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3cbisleguw%253D%253D&md5=5660d96475370e7f46ea6682c9398cb1</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1038%2Fncomms1093&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms1093%26sid%3Dliteratum%253Aachs%26aulast%3DWienken%26aufirst%3DC.%2BJ.%26aulast%3DBaaske%26aufirst%3DP.%26aulast%3DRothbauer%26aufirst%3DU.%26aulast%3DBraun%26aufirst%3DD.%26aulast%3DDuhr%26aufirst%3DS.%26atitle%3DProtein-binding%2520Assays%2520in%2520Biological%2520Liquids%2520Using%2520Microscale%2520Thermophoresis%26jtitle%3DNat.%2520Commun.%26date%3D2010%26volume%3D1%26spage%3D100%26doi%3D10.1038%2Fncomms1093" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Marshall, N. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Price, D. H.</span></span> <span> </span><span class="NLM_article-title">Control of RNA polymerase II Elongation Potential by a Novel Carboxyl-terminal Domain Kinase</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>271</i></span>,  <span class="NLM_fpage">27176</span>â <span class="NLM_lpage">27183</span>, <span class="refDoi">Â DOI: 10.1074/jbc.271.43.27176</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=10.1074%2Fjbc.271.43.27176" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=8900211" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=1%3ACAS%3A528%3ADyaK28XmsFSisb8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=271&publication_year=1996&pages=27176-27183&author=N.+F.+Marshallauthor=J.+Pengauthor=Z.+Xieauthor=D.+H.+Price&title=Control+of+RNA+polymerase+II+Elongation+Potential+by+a+Novel+Carboxyl-terminal+Domain+Kinase&doi=10.1074%2Fjbc.271.43.27176"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Control of RNA polymerase II elongation potential by a novel carboxyl-terminal domain kinase</span></div><div class="casAuthors">Marshall, Nick F.; Peng, Junmin; Xie, Zhi; Price, David H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">271</span>
        (<span class="NLM_cas:issue">43</span>),
    <span class="NLM_cas:pages">27176-27183</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The entry of RNA polymerase II into a productive mode of elongation is controlled, in part, by the postinitiation activity of pos. transcription elongation factor b (P-TEFb).  Removal of the C-terminal domain (CTD) of the large subunit of RNA polymerase II abolishes productive elongation.  Correspondingly, P-TEFb can phosphorylate the CTD of pure RNA polymerase II.  Furthermore, P-TEFb can phosphorylate the CTD of RNA polymerase II when the polymerase is in an early elongation complex.  Both the function and kinase activity of P-TEFb are blocked by the drugs 5,6-dichloro-1-Î²-D-ribofuranosylbenzimidazole (DRB) and H-8.  P-TEFb is distinct from transcription factor IIH (TFIIH) because the 2 factors have no subunits in common, P-TEFb is more sensitive to DRB than is TFIIH, and most importantly, TFIIH cannot substitute functionally for P-TEFb.  Phosphorylation of the CTD by P-TEFb may control the transition from abortive into productive elongation mode.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVKuChjQS6p7Vg90H21EOLACvtfcHk0lge2KStlU-dFw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XmsFSisb8%253D&md5=ed4cfba88e5096094114d212e5da372c</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1074%2Fjbc.271.43.27176&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.271.43.27176%26sid%3Dliteratum%253Aachs%26aulast%3DMarshall%26aufirst%3DN.%2BF.%26aulast%3DPeng%26aufirst%3DJ.%26aulast%3DXie%26aufirst%3DZ.%26aulast%3DPrice%26aufirst%3DD.%2BH.%26atitle%3DControl%2520of%2520RNA%2520polymerase%2520II%2520Elongation%2520Potential%2520by%2520a%2520Novel%2520Carboxyl-terminal%2520Domain%2520Kinase%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1996%26volume%3D271%26spage%3D27176%26epage%3D27183%26doi%3D10.1074%2Fjbc.271.43.27176" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hnisz, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abraham, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, T. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lau, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saint-AndrÃ©, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sigova, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoke, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. A.</span></span> <span> </span><span class="NLM_article-title">Super-enhancers in the Control of Cell Identity and Disease</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>155</i></span>,  <span class="NLM_fpage">934</span>â <span class="NLM_lpage">947</span>, <span class="refDoi">Â DOI: 10.1016/j.cell.2013.09.053</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=10.1016%2Fj.cell.2013.09.053" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=24119843" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1SrtLbL" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=155&publication_year=2013&pages=934-947&author=D.+Hniszauthor=B.+J.+Abrahamauthor=T.+I.+Leeauthor=A.+Lauauthor=V.+Saint-Andr%C3%A9author=A.+A.+Sigovaauthor=H.+A.+Hokeauthor=R.+A.+Young&title=Super-enhancers+in+the+Control+of+Cell+Identity+and+Disease&doi=10.1016%2Fj.cell.2013.09.053"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Super-Enhancers in the Control of Cell Identity and Disease</span></div><div class="casAuthors">Hnisz, Denes; Abraham, Brian J.; Lee, Tong Ihn; Lau, Ashley; Saint-Andre, Violaine; Sigova, Alla A.; Hoke, Heather A.; Young, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">155</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">934-947</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Super-enhancers are large clusters of transcriptional enhancers that drive expression of genes that define cell identity.  Improved understanding of the roles that super-enhancers play in biol. would be afforded by knowing the constellation of factors that constitute these domains and by identifying super-enhancers across the spectrum of human cell types.  We describe here the population of transcription factors, cofactors, chromatin regulators, and transcription app. occupying super-enhancers in embryonic stem cells and evidence that super-enhancers are highly transcribed.  We produce a catalog of super-enhancers in a broad range of human cell types and find that super-enhancers assoc. with genes that control and define the biol. of these cells.  Interestingly, disease-assocd. variation is esp. enriched in the super-enhancers of disease-relevant cell types.  Furthermore, we find that cancer cells generate super-enhancers at oncogenes and other genes important in tumor pathogenesis.  Thus, super-enhancers play key roles in human cell identity in health and in disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqOx0QNN2fg3bVg90H21EOLACvtfcHk0ljRmY0c4mTllw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1SrtLbL&md5=ca83fdac41e4f19387adb996bd0e37d8</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2013.09.053&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2013.09.053%26sid%3Dliteratum%253Aachs%26aulast%3DHnisz%26aufirst%3DD.%26aulast%3DAbraham%26aufirst%3DB.%2BJ.%26aulast%3DLee%26aufirst%3DT.%2BI.%26aulast%3DLau%26aufirst%3DA.%26aulast%3DSaint-Andr%25C3%25A9%26aufirst%3DV.%26aulast%3DSigova%26aufirst%3DA.%2BA.%26aulast%3DHoke%26aufirst%3DH.%2BA.%26aulast%3DYoung%26aufirst%3DR.%2BA.%26atitle%3DSuper-enhancers%2520in%2520the%2520Control%2520of%2520Cell%2520Identity%2520and%2520Disease%26jtitle%3DCell%26date%3D2013%26volume%3D155%26spage%3D934%26epage%3D947%26doi%3D10.1016%2Fj.cell.2013.09.053" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bharate, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vishwakarma, R. A.</span></span> <span> </span><span class="NLM_article-title">Thermodynamic Equilibrium Solubility Measurements in Simulated Fluids by 96-well Plate Method in Early Drug Discovery</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1561</span>â <span class="NLM_lpage">1567</span>, <span class="refDoi">Â DOI: 10.1016/j.bmcl.2015.02.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=10.1016%2Fj.bmcl.2015.02.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=25740159" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=1%3ACAS%3A528%3ADC%252BC2MXivVSrsbc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2015&pages=1561-1567&author=S.+S.+Bharateauthor=R.+A.+Vishwakarma&title=Thermodynamic+Equilibrium+Solubility+Measurements+in+Simulated+Fluids+by+96-well+Plate+Method+in+Early+Drug+Discovery&doi=10.1016%2Fj.bmcl.2015.02.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Thermodynamic equilibrium solubility measurements in simulated fluids by 96-well plate method in early drug discovery</span></div><div class="casAuthors">Bharate, Sonali S.; Vishwakarma, Ram A.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1561-1567</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">An early prediction of soly. in physiol. media (PBS, SGF and SIF) is useful to predict qual. bioavailability and absorption of lead candidates.  Despite of the availability of multiple soly. estn. methods, none of the reported method involves simplified fixed protocol for diverse set of compds.  Therefore, a simple and medium-throughput soly. estn. protocol is highly desirable during lead optimization stage.  The present work introduces a rapid method for assessment of thermodn. equil. soly. of compds. in aq. media using 96-well microplate.  The developed protocol is straightforward to set up and takes advantage of the sensitivity of UV spectroscopy.  The compd., in stock soln. in methanol, is introduced in microgram quantities into microplate wells followed by drying at an ambient temp.  Microplates were shaken upon addn. of test media and the supernatant was analyzed by UV method.  A plot of absorbance vs. concn. of a sample provides satn. point, which is thermodn. equil. soly. of a sample.  The established protocol was validated using a large panel of com. available drugs and with conventional miniaturized shake flask method (r2 >0.84).  Addnl., the statistically significant QSPR models were established using exptl. soly. values of 52 compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrEhIADhdQd3LVg90H21EOLACvtfcHk0ljRmY0c4mTllw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXivVSrsbc%253D&md5=e118b10c67ca311ea5da994793546c11</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2015.02.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2015.02.013%26sid%3Dliteratum%253Aachs%26aulast%3DBharate%26aufirst%3DS.%2BS.%26aulast%3DVishwakarma%26aufirst%3DR.%2BA.%26atitle%3DThermodynamic%2520Equilibrium%2520Solubility%2520Measurements%2520in%2520Simulated%2520Fluids%2520by%252096-well%2520Plate%2520Method%2520in%2520Early%2520Drug%2520Discovery%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D25%26spage%3D1561%26epage%3D1567%26doi%3D10.1016%2Fj.bmcl.2015.02.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 3 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Tizhi Wu, Zhen Qin, Yucheng Tian, Jubo Wang, Chenxi Xu, Zhiyu Li, <span class="NLM_string-name hlFld-ContribAuthor">Jinlei Bian</span>. </span><span class="cited-content_cbyCitation_article-title">Recent Developments in the Biology and Medicinal Chemistry of CDK9 Inhibitors: An Update. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (22)
                                     , 13228-13257. <a href="https://doi.org/10.1021/acs.jmedchem.0c00744" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00744</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00744&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00744%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DRecent%252BDevelopments%252Bin%252Bthe%252BBiology%252Band%252BMedicinal%252BChemistry%252Bof%252BCDK9%252BInhibitors%25253A%252BAn%252BUpdate%26aulast%3DWu%26aufirst%3DTizhi%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D03052020%26date%3D17092020%26date%3D31082020%26volume%3D63%26issue%3D22%26spage%3D13228%26epage%3D13257" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xin  Luan</span>, <span class="hlFld-ContribAuthor ">Ye  Wu</span>, <span class="hlFld-ContribAuthor ">Yi-Wen  Shen</span>, <span class="hlFld-ContribAuthor ">Hong  Zhang</span>, <span class="hlFld-ContribAuthor ">Yu-Dong  Zhou</span>, <span class="hlFld-ContribAuthor ">Hong-Zhuan  Chen</span>, <span class="hlFld-ContribAuthor ">Dale G.  Nagle</span>, <span class="hlFld-ContribAuthor ">Wei-Dong  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">Cytotoxic and antitumor peptides as novel chemotherapeutics. </span><span class="cited-content_cbyCitation_journal-name">Natural Product Reports</span><span> <strong>2021,</strong> <em>38 </em>
                                    (1)
                                     , 7-17. <a href="https://doi.org/10.1039/D0NP00019A" title="DOI URL">https://doi.org/10.1039/D0NP00019A</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D0NP00019A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD0NP00019A%26sid%3Dliteratum%253Aachs%26jtitle%3DNatural%2520Product%2520Reports%26atitle%3DCytotoxic%252Band%252Bantitumor%252Bpeptides%252Bas%252Bnovel%252Bchemotherapeutics%26aulast%3DLuan%26aufirst%3DXin%26date%3D2021%26date%3D2021%26volume%3D38%26issue%3D1%26spage%3D7%26epage%3D17" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Bharat  Goel</span>, <span class="hlFld-ContribAuthor ">Nancy  Tripathi</span>, <span class="hlFld-ContribAuthor ">Nivedita  Bhardwaj</span>, <span class="hlFld-ContribAuthor ">Shreyans K.  Jain</span>. </span><span class="cited-content_cbyCitation_article-title">Small Molecule CDK Inhibitors for the Therapeutic Management of Cancer. </span><span class="cited-content_cbyCitation_journal-name">Current Topics in Medicinal Chemistry</span><span> <strong>2020,</strong> <em>20 </em>
                                    (17)
                                     , 1535-1563. <a href="https://doi.org/10.2174/1568026620666200516152756" title="DOI URL">https://doi.org/10.2174/1568026620666200516152756</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.2174/1568026620666200516152756&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.2174%2F1568026620666200516152756%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Topics%2520in%2520Medicinal%2520Chemistry%26atitle%3DSmall%252BMolecule%252BCDK%252BInhibitors%252Bfor%252Bthe%252BTherapeutic%252BManagement%252Bof%252BCancer%26aulast%3DGoel%26aufirst%3DBharat%26date%3D2020%26volume%3D20%26issue%3D17%26spage%3D1535%26epage%3D1563" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02121/20200323/images/medium/jm9b02121_0021.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02121/20200323/images/large/jm9b02121_0021.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02121&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02121/20200323/images/medium/jm9b02121_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02121/20200323/images/large/jm9b02121_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Structures of representative selective CDK inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02121/20200323/images/large/jm9b02121_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02121&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02121/20200323/images/medium/jm9b02121_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02121/20200323/images/large/jm9b02121_0003.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (A) Interaction regions between CDK6 inhibitors represented by palbociclib and CDK6 (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5L2I">5L2I</a>); (B) CDK9 inhibitors represented by <b>A86</b> and CDK9 (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6GZH">6GZH</a>); docking model of compound <b>8</b> and CDK6 (C) or CDK9 (D); 2D histogram of docking model of compound <b>8</b> and CDK6 (E) or CDK9 (F) to indicate the active and potential sites of modifications of compound <b>8</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02121/20200323/images/large/jm9b02121_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02121&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02121/20200323/images/medium/jm9b02121_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02121/20200323/images/large/jm9b02121_0004.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Step by step structural optimization on hit compound <b>8</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02121/20200323/images/large/jm9b02121_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02121&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02121/20200323/images/medium/jm9b02121_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02121/20200323/images/large/jm9b02121_0011.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Compounds <b>8â67</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02121/20200323/images/large/jm9b02121_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02121&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) 2,4,5-trichloropyrimidine, aubstituted aniline or substituted benzylamine or substituted phenethylamine, TBAI, triethylamine, DMSO, r.t. (b) 5-bromo-2-nitropyridine or 1-bromo-4-nitrobenzene, azaheterocyclic amine, potassium carbonate, DMSO, 70 Â°C. (c) Pd/C 10%, H<sub>2</sub>. (d) Compounds <b>3aâ3ai</b>, <b>7aâ7z</b>, Pd<sub>2</sub>(dba)<sub>3</sub>, BINAP, Na<i>O</i>tBu, dioxane, 110 Â°C, Ar<sub>2</sub> atmosphere. (e) Compounds <b>3aâ3ai</b>, <b>7aâ7z</b>, Pd<sub>2</sub>(dba)<sub>3</sub>, XANTPHOS, Cs<sub>2</sub>CO<sub>3</sub>, dioxane, 110 Â°C, Ar<sub>2</sub> atmosphere.</p></p></figure><figure data-id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02121/20200323/images/medium/jm9b02121_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02121/20200323/images/large/jm9b02121_0005.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Effects of compound <b>66</b> on cell cycle distribution. The experiment was performed independently three times, and representative results are presented.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02121/20200323/images/large/jm9b02121_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02121&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02121/20200323/images/medium/jm9b02121_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02121/20200323/images/large/jm9b02121_0006.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. ROS of MDA-MB-231 cells treated with compound <b>66</b> at different concentrations after 48 h. The results are given in the form of a superposition graph. The experiment was performed independently three times, and representative results are presented.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02121/20200323/images/large/jm9b02121_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02121&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02121/20200323/images/medium/jm9b02121_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02121/20200323/images/large/jm9b02121_0007.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Effects of compound 66 on cellular apoptosis. Apoptosis of MDA-MB-231 cells treated with compound <b>66</b> at different concentrations after 48 h. Cells in Q1-UR and Q1-LR are considered to be apoptotic. The experiment was performed independently three times, and representative results are presented.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02121/20200323/images/large/jm9b02121_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02121&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02121/20200323/images/medium/jm9b02121_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02121/20200323/images/large/jm9b02121_0008.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. CETSA assay to detection the thermal stability of recombinant CDK6 and CDK9 proteins treated with compound <b>66</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02121/20200323/images/large/jm9b02121_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02121&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02121/20200323/images/medium/jm9b02121_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02121/20200323/images/large/jm9b02121_0009.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. MST assay to determine the affinity of compound <b>66</b> to CDK6 and CDK9 proteins. The experiment was performed independently three times, and representative results are presented.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02121/20200323/images/large/jm9b02121_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02121&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02121/20200323/images/medium/jm9b02121_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02121/20200323/images/large/jm9b02121_0010.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Compound 66 inhibits the function of CDK6 and CDK9 and significantly downregulated the mRNA expression of c-MYC and Mcl-1. (A) Immunoblot analysis of the indicated proteins in MDA-MB-231 cells treated with compound <b>66</b>, and Î²-actin was used as a loading control. The experiment was performed independently three times, and representative results are presented. Phosphorylation level of Rb and RNA polymerase II CTD are presented as means Â± SEM. Significant difference is shown as *<i>p</i> < 0.05, **<i>p</i> < 0.01, ***<i>p</i> < 0.001 by one-way ANVOA. (B) qRT-RCR analysis of the mRNA expression of oncogene c-MYC and Mcl-1. The experiment was performed independent three times and presented as means Â± SEM. Significant difference is shown as *<i>p</i> < 0.05, **<i>p</i> < 0.01, ***<i>p</i> < 0.001 by one-way ANVOA.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02121/20200323/images/large/jm9b02121_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02121&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig10" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02121/20200323/images/medium/jm9b02121_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02121/20200323/images/large/jm9b02121_0002.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. In vivo antitumor activity of compound 66 in MDA-MB-231 tumor-bearing mice. (A) and (B) tumor growth curve and mice weight during the treatment. The results are presented as means Â± SEM (<i>n</i> = 6). (C,D) Final tumor size and weight after treatment with compound <b>66</b> in MDA-MB-231 tumor-bearing mice. The results are presented as mean Â± SEM (<i>n</i> = 6). Significant difference is shown as *<i>p</i> < 0.05, **<i>p</i> < 0.01, ***<i>p</i> < 0.001 by one-way ANVOA. (E) HE staining of tumor tissues after treatment. Representative results are given in the figure.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02121/20200323/images/large/jm9b02121_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02121&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i139">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_24728" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_24728" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 42 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Deshpande, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sicinski, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hinds, P. W.</span></span> <span> </span><span class="NLM_article-title">Cyclins and CDKs in Development and Cancer: a Perspective</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">2909</span>â <span class="NLM_lpage">2915</span>, <span class="refDoi">Â DOI: 10.1038/sj.onc.1208618</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=10.1038%2Fsj.onc.1208618" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=15838524" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=1%3ACAS%3A528%3ADC%252BD2MXjtlehsLw%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2005&pages=2909-2915&author=A.+Deshpandeauthor=P.+Sicinskiauthor=P.+W.+Hinds&title=Cyclins+and+CDKs+in+Development+and+Cancer%3A+a+Perspective&doi=10.1038%2Fsj.onc.1208618"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Cyclins and cdks in development and cancer: a perspective</span></div><div class="casAuthors">Deshpande, Amit; Sicinski, Peter; Hinds, Philip W.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">2909-2915</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  A fundamental aspect of cancer is dysregulated cell cycle control.  Unlike normal cells that only proliferate when compelled to do so by developmental or other mitogenic signals in response to tissue growth needs, the proliferation of cancer cells proceeds essentially unchecked.  This does not mean that cancer cell cycles are necessarily different from those found in normal cycling cells, but rather implies that cancer cells proliferate because they are no longer subject to proliferation-inhibitory influences arising from the stroma or from gene expression pattern changes consequent to 'terminal' differentiation, nor do they necessarily require extrinsic growth factors to recruit them into or maintain their proliferative state.  Finally, cancer cells have also often avoided normal controls linked to cell cycle progression that halt proliferation in the presence of damaged DNA or other physiol. insults.  The result of these alterations is the inappropriate proliferation commonly assocd. with cancerous tumor formation.  This review will summarize the current understanding of dysregulation of the G0/G1-to-S-phase transition in cancer cells, with particular emphasis on recent in vivo studies that suggest a need to rethink existing models of cell cycle control in development and tumorigenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdx5230em1arVg90H21EOLACvtfcHk0li5R52KqQuMtA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXjtlehsLw%253D&md5=84de90f5864255f0505413f1bdebe3a2</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fsj.onc.1208618&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.onc.1208618%26sid%3Dliteratum%253Aachs%26aulast%3DDeshpande%26aufirst%3DA.%26aulast%3DSicinski%26aufirst%3DP.%26aulast%3DHinds%26aufirst%3DP.%2BW.%26atitle%3DCyclins%2520and%2520CDKs%2520in%2520Development%2520and%2520Cancer%253A%2520a%2520Perspective%26jtitle%3DOncogene%26date%3D2005%26volume%3D24%26spage%3D2909%26epage%3D2915%26doi%3D10.1038%2Fsj.onc.1208618" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lim, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaldis, P.</span></span> <span> </span><span class="NLM_article-title">Cdks, Cyclins and CKIs: Roles Beyond Cell Cycle Regulation</span>. <i>Development</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>140</i></span>,  <span class="NLM_fpage">3079</span>â <span class="NLM_lpage">3093</span>, <span class="refDoi">Â DOI: 10.1242/dev.091744</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=10.1242%2Fdev.091744" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=23861057" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsFGktL%252FF" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=140&publication_year=2013&pages=3079-3093&author=S.+Limauthor=P.+Kaldis&title=Cdks%2C+Cyclins+and+CKIs%3A+Roles+Beyond+Cell+Cycle+Regulation&doi=10.1242%2Fdev.091744"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Cdks, cyclins and CKIs: roles beyond cell cycle regulation</span></div><div class="casAuthors">Lim, Shuhui; Kaldis, Philipp</div><div class="citationInfo"><span class="NLM_cas:title">Development (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">140</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">3079-3093</span>CODEN:
                <span class="NLM_cas:coden">DEVPED</span>;
        ISSN:<span class="NLM_cas:issn">0950-1991</span>.
    
            (<span class="NLM_cas:orgname">Company of Biologists Ltd.</span>)
        </div><div class="casAbstract">A review.  Cyclin-dependent kinases (Cdks) are serine/threonine kinases and their catalytic activities are modulated by interactions with cyclins and Cdk inhibitors (CKIs).  Close cooperation between this trio is necessary for ensuring orderly progression through the cell cycle.  In addn. to their well-established function in cell cycle control, it is becoming increasingly apparent that mammalian Cdks, cyclins and CKIs play indispensable roles in processes such as transcription, epigenetic regulation, metab., stem cell self-renewal, neuronal functions and spermatogenesis.  Even more remarkably, they can accomplish some of these tasks individually, without the need for Cdk/cyclin complex formation or kinase activity.  In this Review, we discuss the latest revelations about Cdks, cyclins and CKIs with the goal of showcasing their functional diversity beyond cell cycle regulation and their impact on development and disease in mammals.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqp5LNcD6NywLVg90H21EOLACvtfcHk0li5R52KqQuMtA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsFGktL%252FF&md5=8095fc798a0e7efeab60d36ee414596f</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1242%2Fdev.091744&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1242%252Fdev.091744%26sid%3Dliteratum%253Aachs%26aulast%3DLim%26aufirst%3DS.%26aulast%3DKaldis%26aufirst%3DP.%26atitle%3DCdks%252C%2520Cyclins%2520and%2520CKIs%253A%2520Roles%2520Beyond%2520Cell%2520Cycle%2520Regulation%26jtitle%3DDevelopment%26date%3D2013%26volume%3D140%26spage%3D3079%26epage%3D3093%26doi%3D10.1242%2Fdev.091744" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Malumbres, M.</span></span> <span> </span><span class="NLM_article-title">Cyclin-dependent Kinases</span>. <i>Genome Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">122</span>, <span class="refDoi">Â DOI: 10.1186/gb4184</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=10.1186%2Fgb4184" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=25180339" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=1%3ACAS%3A280%3ADC%252BC2M%252FnsVehsg%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2014&pages=122&author=M.+Malumbres&title=Cyclin-dependent+Kinases&doi=10.1186%2Fgb4184"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Cyclin-dependent kinases</span></div><div class="casAuthors">Malumbres Marcos</div><div class="citationInfo"><span class="NLM_cas:title">Genome biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">122</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Cyclin-dependent kinases (CDKs) are protein kinases characterized by needing a separate subunit - a cyclin - that provides domains essential for enzymatic activity.  CDKs play important roles in the control of cell division and modulate transcription in response to several extra- and intracellular cues.  The evolutionary expansion of the CDK family in mammals led to the division of CDKs into three cell-cycle-related subfamilies (Cdk1, Cdk4 and Cdk5) and five transcriptional subfamilies (Cdk7, Cdk8, Cdk9, Cdk11 and Cdk20).  Unlike the prototypical Cdc28 kinase of budding yeast, most of these CDKs bind one or a few cyclins, consistent with functional specialization during evolution.  This review summarizes how, although CDKs are traditionally separated into cell-cycle or transcriptional CDKs, these activities are frequently combined in many family members.  Not surprisingly, deregulation of this family of proteins is a hallmark of several diseases, including cancer, and drug-targeted inhibition of specific members has generated very encouraging results in clinical trials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQodaug4zJdjgHqQV89nF-2fW6udTcc2eZ92mRV-kmcy7ntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2M%252FnsVehsg%253D%253D&md5=9fd79ec8f495f712bada681b0a0b5afe</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1186%2Fgb4184&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fgb4184%26sid%3Dliteratum%253Aachs%26aulast%3DMalumbres%26aufirst%3DM.%26atitle%3DCyclin-dependent%2520Kinases%26jtitle%3DGenome%2520Biol.%26date%3D2014%26volume%3D15%26spage%3D122%26doi%3D10.1186%2Fgb4184" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">Targeting Cancer with Small Molecule Kinase Inhibitors</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">28</span>â <span class="NLM_lpage">39</span>, <span class="refDoi">Â DOI: 10.1038/nrc2559</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=10.1038%2Fnrc2559" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=19104514" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=1%3ACAS%3A528%3ADC%252BD1MXpt1yg" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2009&pages=28-39&author=J.+Zhangauthor=P.+L.+Yangauthor=N.+S.+Gray&title=Targeting+Cancer+with+Small+Molecule+Kinase+Inhibitors&doi=10.1038%2Fnrc2559"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting cancer with small molecule kinase inhibitors</span></div><div class="casAuthors">Zhang, Jianming; Yang, Priscilla L.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">28-39</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Deregulation of kinase activity has emerged as a major mechanism by which cancer cells evade normal physiol. constraints on growth and survival.  To date, 11 kinase inhibitors have received US Food and Drug Administration approval as cancer treatments, and there are considerable efforts to develop selective small mol. inhibitors for a host of other kinases that are implicated in cancer and other diseases.  Herein we discuss the current challenges in the field, such as designing selective inhibitors and developing strategies to overcome resistance mutations.  This Review provides a broad overview of some of the approaches currently used to discover and characterize new kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGriSM3BN_uEz7Vg90H21EOLACvtfcHk0lgH6w9iWcMU0w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXpt1yg&md5=35bc38f414cd21eef5e61ce181b580fb</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fnrc2559&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc2559%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DP.%2BL.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DTargeting%2520Cancer%2520with%2520Small%2520Molecule%2520Kinase%2520Inhibitors%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2009%26volume%3D9%26spage%3D28%26epage%3D39%26doi%3D10.1038%2Fnrc2559" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">MÃ¼ller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaikuad, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">The Ins and Outs of Selective Kinase Inhibitor Development</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">818</span>â <span class="NLM_lpage">821</span>, <span class="refDoi">Â DOI: 10.1038/nchembio.1938</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=10.1038%2Fnchembio.1938" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=26485069" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1ykt7fF" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=818-821&author=S.+M%C3%BCllerauthor=A.+Chaikuadauthor=N.+S.+Grayauthor=S.+Knapp&title=The+Ins+and+Outs+of+Selective+Kinase+Inhibitor+Development&doi=10.1038%2Fnchembio.1938"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">The ins and outs of selective kinase inhibitor development</span></div><div class="casAuthors">Muller, Susanne; Chaikuad, Apirat; Gray, Nathanael S.; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">818-821</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Protein kinases have emerged as one of the most successful families of drug targets.  To date, most selective kinase inhibitors have been discovered serendipitously either through broad selectivity screening or through the discovery of unique binding modes.  Here the authors discuss design strategies that could lead to a broader coverage of the kinome with selective inhibitors and to a more rational approach for developing them.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpQzaMrxSYx_rVg90H21EOLACvtfcHk0lgH6w9iWcMU0w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1ykt7fF&md5=f5adf0bbaa388c87b26bae23cfad0bae</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1938&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1938%26sid%3Dliteratum%253Aachs%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DChaikuad%26aufirst%3DA.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DThe%2520Ins%2520and%2520Outs%2520of%2520Selective%2520Kinase%2520Inhibitor%2520Development%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26spage%3D818%26epage%3D821%26doi%3D10.1038%2Fnchembio.1938" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Asghar, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Witkiewicz, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turner, N. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knudsen, E. S.</span></span> <span> </span><span class="NLM_article-title">The History and Future of Targeting Cyclin-dependent Kinases in Cancer Therapy</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">130</span>â <span class="NLM_lpage">146</span>, <span class="refDoi">Â DOI: 10.1038/nrd4504</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=10.1038%2Fnrd4504" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=25633797" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjvVCrtLc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2015&pages=130-146&author=U.+Asgharauthor=A.+K.+Witkiewiczauthor=N.+C.+Turnerauthor=E.+S.+Knudsen&title=The+History+and+Future+of+Targeting+Cyclin-dependent+Kinases+in+Cancer+Therapy&doi=10.1038%2Fnrd4504"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">The history and future of targeting cyclin-dependent kinases in cancer therapy</span></div><div class="casAuthors">Asghar, Uzma; Witkiewicz, Agnieszka K.; Turner, Nicholas C.; Knudsen, Erik S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">130-146</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Cancer represents a pathol. manifestation of uncontrolled cell division; therefore, it has long been anticipated that our understanding of the basic principles of cell cycle control would result in effective cancer therapies.  In particular, cyclin-dependent kinases (CDKs) that promote transition through the cell cycle were expected to be key therapeutic targets because many tumorigenic events ultimately drive proliferation by impinging on CDK4 or CDK6 complexes in the G1 phase of the cell cycle.  Moreover, perturbations in chromosomal stability and aspects of S phase and G2/M control mediated by CDK2 and CDK1 are pivotal tumorigenic events.  Translating this knowledge into successful clin. development of CDK inhibitors has historically been challenging, and numerous CDK inhibitors have demonstrated disappointing results in clin. trials.  Here, we review the biol. of CDKs, the rationale for therapeutically targeting discrete kinase complexes and historical clin. results of CDK inhibitors.  We also discuss how CDK inhibitors with high selectivity (particularly for both CDK4 and CDK6), in combination with patient stratification, have resulted in more substantial clin. activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJ_utd-yiEZLVg90H21EOLACvtfcHk0lgH6w9iWcMU0w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjvVCrtLc%253D&md5=113bcd2e99a8267119d180670d3c7761</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnrd4504&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4504%26sid%3Dliteratum%253Aachs%26aulast%3DAsghar%26aufirst%3DU.%26aulast%3DWitkiewicz%26aufirst%3DA.%2BK.%26aulast%3DTurner%26aufirst%3DN.%2BC.%26aulast%3DKnudsen%26aufirst%3DE.%2BS.%26atitle%3DThe%2520History%2520and%2520Future%2520of%2520Targeting%2520Cyclin-dependent%2520Kinases%2520in%2520Cancer%2520Therapy%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2015%26volume%3D14%26spage%3D130%26epage%3D146%26doi%3D10.1038%2Fnrd4504" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roskoski, R.,  Jr.</span></span> <span> </span><span class="NLM_article-title">A Historical Overview of Protein Kinases and Their Targeted Small Molecule Inhibitors</span>. <i>Pharmacol. Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>100</i></span>,  <span class="NLM_fpage">1</span>â <span class="NLM_lpage">23</span>, <span class="refDoi">Â DOI: 10.1016/j.phrs.2015.07.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=10.1016%2Fj.phrs.2015.07.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=26207888" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1amsL3M" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=100&publication_year=2015&pages=1-23&author=R.+Roskoski&title=A+Historical+Overview+of+Protein+Kinases+and+Their+Targeted+Small+Molecule+Inhibitors&doi=10.1016%2Fj.phrs.2015.07.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">A historical overview of protein kinases and their targeted small molecule inhibitors</span></div><div class="casAuthors">Roskoski, Robert, Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacological Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">100</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1-23</span>CODEN:
                <span class="NLM_cas:coden">PHMREP</span>;
        ISSN:<span class="NLM_cas:issn">1043-6618</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Protein kinases play a predominant regulatory role in nearly every aspect of cell biol. and they can modify the function of a protein in almost every conceivable way.  Protein phosphorylation can increase or decrease enzyme activity and it can alter other biol. activities such as transcription and translation.  Moreover, some phosphorylation sites on a given protein are stimulatory while others are inhibitory.  The human protein kinase gene family consists of 518 members along with 106 pseudogenes.  Furthermore, about 50 of the 518 gene products lack important catalytic residues and are called protein pseudokinases.  The non-catalytic allosteric interaction of protein kinases and pseudokinases with other proteins has added an important regulatory feature to the biochem. and cell biol. of the protein kinase superfamily.  With rare exceptions, a divalent cation such as Mg2+ is required for the reaction.  All protein kinases exist in a basal state and are activated only as necessary by divergent regulatory stimuli.  The mechanisms for switching between dormant and active protein kinases can be intricate.  Phosphorylase kinase was the first protein kinase to be characterized biochem. and the mechanism of its regulation led to the discovery of cAMP-dependent protein kinase (protein kinase A, or PKA), which catalyzes the phosphorylation and activation of phosphorylase kinase.  This was the first protein kinase cascade or signaling module to be elucidated.  The epidermal growth factor receptor-Ras-Raf-MEK-ERK signaling module contains protein-tyrosine, protein-serine/threonine, and dual specificity protein kinases.  PKA has served as a prototype of this enzyme family and more is known about this enzyme than any other protein kinase.  The inactive PKA holoenzyme consists of two regulatory and two catalytic subunits.  After binding four mols. of cAMP, the holoenzyme dissocs. into a regulatory subunit dimer (each monomer binds two cAMP) and two free and active catalytic subunits.  PKA and all other protein kinase domains have a small amino-terminal lobe and large carboxyterminal lobe as detd. by X-ray crystallog.  The N-lobe and C-lobe form a cleft that serves as a docking site for MgATP.  Nearly all active protein kinases contain a K/E/D/D signature sequence that plays important structural and catalytic roles.  Protein kinases contain hydrophobic catalytic and regulatory spines and collateral shell residues that are required to assemble the active enzyme.  There are two general kinds of conformational changes assocd. with most protein kinases.  The first conformational change involves the formation of an intact regulatory spine to form an active enzyme.  The second conformational change occurs in active kinases as they toggle between open and closed conformations during their catalytic cycles.  Because mutations and dysregulation of protein kinases play causal roles in human disease, this family of enzymes has become one of the most important drug targets over the past two decades.  Imatinib was approved by the United States FDA for the treatment of chronic myelogenous leukemia in 2001; this small mol. inhibits the BCR-Abl protein kinase oncoprotein that results from the formation of the Philadelphia chromosome.  More than two dozen other orally effective mechanism-based small mol. protein kinase inhibitors have been subsequently approved by the FDA.  These drugs bind to the ATP-binding site of their target enzymes and extend into nearby hydrophobic pockets.  Most of these protein kinase inhibitors prolong survival in cancer patients only weeks or months longer than std. cytotoxic therapies.  In contrast, the clin. effectiveness of imatinib against chronic myelogenous leukemia is vastly superior to that of any other targeted protein kinase inhibitor with overall survival lasting a decade or more.  However, the near universal and expected development of drug resistance in the treatment of neoplastic disorders requires new approaches to solve this therapeutic challenge.  Cancer is the predominant indication for these drugs, but disease targets are increasing.  For example, we can expect the approval of new drugs inhibiting other protein kinases in the treatment of illnesses such as hypertension, Parkinson's disease, and autoimmune diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrrSGGFPXk0aLVg90H21EOLACvtfcHk0lg_CspGYq9qGg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1amsL3M&md5=819cc1d8541b51603841011d3e5f6bdb</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.phrs.2015.07.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.phrs.2015.07.010%26sid%3Dliteratum%253Aachs%26aulast%3DRoskoski%26aufirst%3DR.%26atitle%3DA%2520Historical%2520Overview%2520of%2520Protein%2520Kinases%2520and%2520Their%2520Targeted%2520Small%2520Molecule%2520Inhibitors%26jtitle%3DPharmacol.%2520Res.%26date%3D2015%26volume%3D100%26spage%3D1%26epage%3D23%26doi%3D10.1016%2Fj.phrs.2015.07.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Olson, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erb, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doctor, Z. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwiatkowski, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boukhali, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haas, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nomanbhoy, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">Pharmacological Perturbation of CDK9 Using Selective CDK9 Inhibition or Degradation</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">163</span>â <span class="NLM_lpage">170</span>, <span class="refDoi">Â DOI: 10.1038/nchembio.2538</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=10.1038%2Fnchembio.2538" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=29251720" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFyrsr7O" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2018&pages=163-170&author=C.+M.+Olsonauthor=B.+Jiangauthor=M.+A.+Erbauthor=Y.+Liangauthor=Z.+M.+Doctorauthor=Z.+Zhangauthor=T.+Zhangauthor=N.+Kwiatkowskiauthor=M.+Boukhaliauthor=J.+L.+Greenauthor=W.+Haasauthor=T.+Nomanbhoyauthor=E.+S.+Fischerauthor=R.+A.+Youngauthor=J.+E.+Bradnerauthor=G.+E.+Winterauthor=N.+S.+Gray&title=Pharmacological+Perturbation+of+CDK9+Using+Selective+CDK9+Inhibition+or+Degradation&doi=10.1038%2Fnchembio.2538"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacological perturbation of CDK9 using selective CDK9 inhibition or degradation</span></div><div class="casAuthors">Olson, Calla M.; Jiang, Baishan; Erb, Michael A.; Liang, Yanke; Doctor, Zainab M.; Zhang, Zinan; Zhang, Tinghu; Kwiatkowski, Nicholas; Boukhali, Myriam; Green, Jennifer L.; Haas, Wilhelm; Nomanbhoy, Tyzoon; Fischer, Eric S.; Young, Richard A.; Bradner, James E.; Winter, Georg E.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">163-170</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Cyclin-dependent kinase 9 (CDK9), an important regulator of transcriptional elongation, is a promising target for cancer therapy, particularly for cancers driven by transcriptional dysregulation.  The authors characterized NVP-2, a selective ATP-competitive CDK9 inhibitor, and THAL-SNS-032, a selective CDK9 degrader consisting of a CDK-binding SNS-032 ligand linked to a thalidomide deriv. that binds the E3 ubiquitin ligase Cereblon (CRBN).  To the authors' surprise, THAL-SNS-032 induced rapid degrdn. of CDK9 without affecting the levels of other SNS-032 targets.  Moreover, the transcriptional changes elicited by THAL-SNS-032 were more like those caused by NVP-2 than those induced by SNS-032.  Notably, compd. washout did not significantly reduce levels of THAL-SNS-032-induced apoptosis, suggesting that CDK9 degrdn. had prolonged cytotoxic effects compared with CDK9 inhibition.  Thus, the authors' findings suggest that thalidomide conjugation represents a promising strategy for converting multi-targeted inhibitors into selective degraders and reveal that kinase degrdn. can induce distinct pharmacol. effects compared with inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWnT9r0GIlmbVg90H21EOLACvtfcHk0lg_CspGYq9qGg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFyrsr7O&md5=88aabb209b4cb7516a652d6c75a35667</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2538&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2538%26sid%3Dliteratum%253Aachs%26aulast%3DOlson%26aufirst%3DC.%2BM.%26aulast%3DJiang%26aufirst%3DB.%26aulast%3DErb%26aufirst%3DM.%2BA.%26aulast%3DLiang%26aufirst%3DY.%26aulast%3DDoctor%26aufirst%3DZ.%2BM.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DKwiatkowski%26aufirst%3DN.%26aulast%3DBoukhali%26aufirst%3DM.%26aulast%3DGreen%26aufirst%3DJ.%2BL.%26aulast%3DHaas%26aufirst%3DW.%26aulast%3DNomanbhoy%26aufirst%3DT.%26aulast%3DFischer%26aufirst%3DE.%2BS.%26aulast%3DYoung%26aufirst%3DR.%2BA.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DWinter%26aufirst%3DG.%2BE.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DPharmacological%2520Perturbation%2520of%2520CDK9%2520Using%2520Selective%2520CDK9%2520Inhibition%2520or%2520Degradation%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2018%26volume%3D14%26spage%3D163%26epage%3D170%26doi%3D10.1038%2Fnchembio.2538" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cicenas, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valius, M.</span></span> <span> </span><span class="NLM_article-title">The CDK Inhibitors in Cancer Research and Therapy</span>. <i>J. Cancer Res. Clin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>137</i></span>,  <span class="NLM_fpage">1409</span>â <span class="NLM_lpage">1418</span>, <span class="refDoi">Â DOI: 10.1007/s00432-011-1039-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=10.1007%2Fs00432-011-1039-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=21877198" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFynt7rK" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=137&publication_year=2011&pages=1409-1418&author=J.+Cicenasauthor=M.+Valius&title=The+CDK+Inhibitors+in+Cancer+Research+and+Therapy&doi=10.1007%2Fs00432-011-1039-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">The CDK inhibitors in cancer research and therapy</span></div><div class="casAuthors">Cicenas, Jonas; Valius, Mindaugas</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cancer Research and Clinical Oncology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">137</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1409-1418</span>CODEN:
                <span class="NLM_cas:coden">JCROD7</span>;
        ISSN:<span class="NLM_cas:issn">0171-5216</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  Chem. compds. that interfere with an enzymic function of kinases are useful for gaining insight into the complicated biochem. processes in mammalian cells.  Cyclin-dependent kinases (CDK) play an essential role in the control of the cell cycle and/or proliferation.  These kinases as well as their regulators are frequently deregulated in different human tumors.  Aberrations in CDK activity have also been obsd. in viral infections, Alzheimer's, Parkinson's diseases, ischemia and some proliferative disorders.  This led to an intensive search for small-mol. CDK inhibitors not only for research purposes, but also for therapeutic applications.  Here, we discuss seventeen CDK inhibitors and their use in cancer research or therapy.  This review should help researchers to decide which inhibitor is best suited for the specific purpose of their research.  For this purpose, the targets, com. availability and IC50 values are provided for each inhibitor.  The review will also provide an overview of the clin. studies performed with some of these inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrEjNnnzkVahrVg90H21EOLACvtfcHk0lg_CspGYq9qGg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFynt7rK&md5=85500301e1d113148cf7854125e542ab</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1007%2Fs00432-011-1039-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00432-011-1039-4%26sid%3Dliteratum%253Aachs%26aulast%3DCicenas%26aufirst%3DJ.%26aulast%3DValius%26aufirst%3DM.%26atitle%3DThe%2520CDK%2520Inhibitors%2520in%2520Cancer%2520Research%2520and%2520Therapy%26jtitle%3DJ.%2520Cancer%2520Res.%2520Clin.%2520Oncol.%26date%3D2011%26volume%3D137%26spage%3D1409%26epage%3D1418%26doi%3D10.1007%2Fs00432-011-1039-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Toogood, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harvey, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Repine, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheehan, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">VanderWel, S. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keller, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNamara, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sherry, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brodfuehrer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barvian, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fry, D. W.</span></span> <span> </span><span class="NLM_article-title">Discovery of a Potent and Selective Inhibitor of Cyclin-dependent Kinase 4/6</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">2388</span>â <span class="NLM_lpage">2406</span>, <span class="refDoi">Â DOI: 10.1021/jm049354h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm049354h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=1%3ACAS%3A528%3ADC%252BD2MXitVGisL0%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=2388-2406&author=P.+L.+Toogoodauthor=P.+J.+Harveyauthor=J.+T.+Repineauthor=D.+J.+Sheehanauthor=S.+N.+VanderWelauthor=H.+Zhouauthor=P.+R.+Kellerauthor=D.+J.+McNamaraauthor=D.+Sherryauthor=T.+Zhuauthor=J.+Brodfuehrerauthor=C.+Choiauthor=M.+R.+Barvianauthor=D.+W.+Fry&title=Discovery+of+a+Potent+and+Selective+Inhibitor+of+Cyclin-dependent+Kinase+4%2F6&doi=10.1021%2Fjm049354h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Potent and Selective Inhibitor of Cyclin-Dependent Kinase 4/6</span></div><div class="casAuthors">Toogood, Peter L.; Harvey, Patricia J.; Repine, Joseph T.; Sheehan, Derek J.; VanderWel, Scott N.; Zhou, Hairong; Keller, Paul R.; McNamara, Dennis J.; Sherry, Debra; Zhu, Tong; Brodfuehrer, Joanne; Choi, Chung; Barvian, Mark R.; Fry, David W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2388-2406</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A pharmacol. approach to inhibition of cyclin-dependent kinases 4 and 6 (Cdk4/6) using highly selective small mol. inhibitors has the potential to provide novel cancer therapies for clin. use.  Achieving high levels of selectivity for Cdk4/6, vs. other ATP-dependent kinases, presents a significant challenge.  The pyrido[2,3-d]pyrimidin-7-one template provides an effective platform for the inhibition of a broad cross-section of kinases, including Cdks.  It is now demonstrated that the modification of pyrido[2,3-d]pyrimidin-7-ones to include a 2-aminopyridine side chain at the C2-position provides inhibitors with exquisite selectivity for Cdk4/6 in vitro.  This selectivity profile is recapitulated in cells where the most selective inhibitors create a G1 block at concns. up to 100-fold the IC50 for cell proliferation.  On the basis of its selectivity profile and pharmacokinetic profile, compd. I (PD 0332991) was identified as a drug candidate for the treatment of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoX577J5U0r97Vg90H21EOLACvtfcHk0lisx9ghCR1XAA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXitVGisL0%253D&md5=23cbe58426ff1975695712db7c1b128a</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Fjm049354h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm049354h%26sid%3Dliteratum%253Aachs%26aulast%3DToogood%26aufirst%3DP.%2BL.%26aulast%3DHarvey%26aufirst%3DP.%2BJ.%26aulast%3DRepine%26aufirst%3DJ.%2BT.%26aulast%3DSheehan%26aufirst%3DD.%2BJ.%26aulast%3DVanderWel%26aufirst%3DS.%2BN.%26aulast%3DZhou%26aufirst%3DH.%26aulast%3DKeller%26aufirst%3DP.%2BR.%26aulast%3DMcNamara%26aufirst%3DD.%2BJ.%26aulast%3DSherry%26aufirst%3DD.%26aulast%3DZhu%26aufirst%3DT.%26aulast%3DBrodfuehrer%26aufirst%3DJ.%26aulast%3DChoi%26aufirst%3DC.%26aulast%3DBarvian%26aufirst%3DM.%2BR.%26aulast%3DFry%26aufirst%3DD.%2BW.%26atitle%3DDiscovery%2520of%2520a%2520Potent%2520and%2520Selective%2520Inhibitor%2520of%2520Cyclin-dependent%2520Kinase%25204%252F6%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D2388%26epage%3D2406%26doi%3D10.1021%2Fjm049354h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">VanArsdale, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boshoff, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arndt, K. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abraham, R. T.</span></span> <span> </span><span class="NLM_article-title">Molecular Pathways: Targeting the Cyclin D-CDK4/6 Axis for Cancer Treatment</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">2905</span>â <span class="NLM_lpage">2910</span>, <span class="refDoi">Â DOI: 10.1158/1078-0432.ccr-14-0816</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=10.1158%2F1078-0432.CCR-14-0816" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=25941111" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=1%3ACAS%3A280%3ADC%252BC2Mfgs1Wisg%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2015&pages=2905-2910&author=T.+VanArsdaleauthor=C.+Boshoffauthor=K.+T.+Arndtauthor=R.+T.+Abraham&title=Molecular+Pathways%3A+Targeting+the+Cyclin+D-CDK4%2F6+Axis+for+Cancer+Treatment&doi=10.1158%2F1078-0432.ccr-14-0816"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Pathways: Targeting the Cyclin D-CDK4/6 Axis for Cancer Treatment</span></div><div class="casAuthors">VanArsdale Todd; Abraham Robert T; Boshoff Chris; Arndt Kim T</div><div class="citationInfo"><span class="NLM_cas:title">Clinical cancer research : an official journal of the American Association for Cancer Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">2905-10</span>
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    </div><div class="casAbstract">Cancer cells bypass normal controls over mitotic cell-cycle progression to achieve a deregulated state of proliferation.  The retinoblastoma tumor suppressor protein (pRb) governs a key cell-cycle checkpoint that normally prevents G1-phase cells from entering S-phase in the absence of appropriate mitogenic signals.  Cancer cells frequently overcome pRb-dependent growth suppression via constitutive phosphorylation and inactivation of pRb function by cyclin-dependent kinase (CDK) 4 or CDK6 partnered with D-type cyclins.  Three selective CDK4/6 inhibitors, palbociclib (Ibrance; Pfizer), ribociclib (Novartis), and abemaciclib (Lilly), are in various stages of development in a variety of pRb-positive tumor types, including breast cancer, melanoma, liposarcoma, and non-small cell lung cancer.  The emerging, positive clinical data obtained to date finally validate the two decades-old hypothesis that the cyclin D-CDK4/6 pathway is a rational target for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR6Rstlwhprintlzo9xastyfW6udTcc2ebVnsJzR2bMxLntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2Mfgs1Wisg%253D%253D&md5=5b3443d0ec58550d8ef4f1993726fb5e</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-14-0816&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-14-0816%26sid%3Dliteratum%253Aachs%26aulast%3DVanArsdale%26aufirst%3DT.%26aulast%3DBoshoff%26aufirst%3DC.%26aulast%3DArndt%26aufirst%3DK.%2BT.%26aulast%3DAbraham%26aufirst%3DR.%2BT.%26atitle%3DMolecular%2520Pathways%253A%2520Targeting%2520the%2520Cyclin%2520D-CDK4%252F6%2520Axis%2520for%2520Cancer%2520Treatment%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2015%26volume%3D21%26spage%3D2905%26epage%3D2910%26doi%3D10.1158%2F1078-0432.ccr-14-0816" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gelbert, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanchez-Martinez, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Del Prado, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lallena, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torres, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ajamie, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wishart, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flack, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neubauer, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iversen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cronier, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kreklau, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Dios, A.</span></span> <span> </span><span class="NLM_article-title">Preclinical Characterization of the CDK4/6 Inhibitor LY2835219: In-vivo Cell Cycle-Dependent/independent Anti-tumor Activities Alone/in Combination with Gemcitabine</span>. <i>Invest. New Drugs</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">825</span>â <span class="NLM_lpage">837</span>, <span class="refDoi">Â DOI: 10.1007/s10637-014-0120-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=10.1007%2Fs10637-014-0120-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=24919854" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtVWrt7%252FN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2014&pages=825-837&author=L.+M.+Gelbertauthor=S.+Caiauthor=X.+Linauthor=C.+Sanchez-Martinezauthor=M.+Del+Pradoauthor=M.+J.+Lallenaauthor=R.+Torresauthor=R.+T.+Ajamieauthor=G.+N.+Wishartauthor=R.+S.+Flackauthor=B.+L.+Neubauerauthor=J.+Youngauthor=E.+M.+Chanauthor=P.+Iversenauthor=D.+Cronierauthor=E.+Kreklauauthor=A.+de+Dios&title=Preclinical+Characterization+of+the+CDK4%2F6+Inhibitor+LY2835219%3A+In-vivo+Cell+Cycle-Dependent%2Findependent+Anti-tumor+Activities+Alone%2Fin+Combination+with+Gemcitabine&doi=10.1007%2Fs10637-014-0120-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Preclinical characterization of the CDK4/6 inhibitor LY2835219: in-vivo cell cycle-dependent/independent anti-tumor activities alone/in combination with gemcitabine</span></div><div class="casAuthors">Gelbert, Lawrence M.; Cai, Shufen; Lin, Xi; Sanchez-Martinez, Concepcion; del Prado, Miriam; Lallena, Maria Jose; Torres, Raquel; Ajamie, Rose T.; Wishart, Graham N.; Flack, Robert Steven; Neubauer, Blake Lee; Young, Jamie; Chan, Edward M.; Iversen, Philip; Cronier, Damien; Kreklau, Emiko; de Dios, Alfonso</div><div class="citationInfo"><span class="NLM_cas:title">Investigational New Drugs</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">825-837</span>CODEN:
                <span class="NLM_cas:coden">INNDDK</span>;
        ISSN:<span class="NLM_cas:issn">0167-6997</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">The G1 restriction point is crit. for regulating the cell cycle and is controlled by the Rb pathway (CDK4/6-cyclin D1-Rb-p16/ink4a).  This pathway is important because of its inactivation in a majority of human tumors.  Transition through the restriction point requires phosphorylation of retinoblastoma protein (Rb) by CDK4/6, which are highly validated cancer drug targets.  We present the identification and characterization of a potent CDK4/6 inhibitor, LY2835219.  LY2835219 inhibits CDK4 and CDK6 with low nanomolar potency, inhibits Rb phosphorylation resulting in a G1 arrest and inhibition of proliferation, and its activity is specific for Rb-proficient cells.  In vivo target inhibition studies show LY2835219 is a potent inhibitor of Rb phosphorylation, induces a complete cell cycle arrest and suppresses expression of several Rb-E2F-regulated proteins 24 h after a single dose.  Oral administration of LY2835219 inhibits tumor growth in human tumor xenografts representing different histologies in tumor-bearing mice.  LY2835219 is effective and well tolerated when administered up to 56 days in immunodeficient mice without significant loss of body wt. or tumor outgrowth.  In calu-6 xenografts, LY2835219 in combination with gemcitabine enhanced in vivo antitumor activity without a G1 cell cycle arrest, but was assocd. with a redn. of ribonucleotide reductase expression.  These results suggest LY2835219 may be used alone or in combination with std.-of-care cytotoxic therapy.  In summary, we have identified a potent, orally active small-mol. inhibitor of CDK4/6 that is active in xenograft tumors.  LY2835219 is currently in clin. development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoQrFS1mDbyCrVg90H21EOLACvtfcHk0lisx9ghCR1XAA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtVWrt7%252FN&md5=a90fee83bfe8da7e4a7474ccf02fc59a</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1007%2Fs10637-014-0120-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10637-014-0120-7%26sid%3Dliteratum%253Aachs%26aulast%3DGelbert%26aufirst%3DL.%2BM.%26aulast%3DCai%26aufirst%3DS.%26aulast%3DLin%26aufirst%3DX.%26aulast%3DSanchez-Martinez%26aufirst%3DC.%26aulast%3DDel%2BPrado%26aufirst%3DM.%26aulast%3DLallena%26aufirst%3DM.%2BJ.%26aulast%3DTorres%26aufirst%3DR.%26aulast%3DAjamie%26aufirst%3DR.%2BT.%26aulast%3DWishart%26aufirst%3DG.%2BN.%26aulast%3DFlack%26aufirst%3DR.%2BS.%26aulast%3DNeubauer%26aufirst%3DB.%2BL.%26aulast%3DYoung%26aufirst%3DJ.%26aulast%3DChan%26aufirst%3DE.%2BM.%26aulast%3DIversen%26aufirst%3DP.%26aulast%3DCronier%26aufirst%3DD.%26aulast%3DKreklau%26aufirst%3DE.%26aulast%3Dde%2BDios%26aufirst%3DA.%26atitle%3DPreclinical%2520Characterization%2520of%2520the%2520CDK4%252F6%2520Inhibitor%2520LY2835219%253A%2520In-vivo%2520Cell%2520Cycle-Dependent%252Findependent%2520Anti-tumor%2520Activities%2520Alone%252Fin%2520Combination%2520with%2520Gemcitabine%26jtitle%3DInvest.%2520New%2520Drugs%26date%3D2014%26volume%3D32%26spage%3D825%26epage%3D837%26doi%3D10.1007%2Fs10637-014-0120-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">DiPippo, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barnett, C. M.</span></span> <span> </span><span class="NLM_article-title">Cyclin-Dependent Kinase Inhibitors for the Treatment of Breast Cancer: Past, Present, and Future</span>. <i>Pharmacotherapy</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">652</span>â <span class="NLM_lpage">667</span>, <span class="refDoi">Â DOI: 10.1002/phar.1756</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=10.1002%2Fphar.1756" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=27087139" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=1%3ACAS%3A280%3ADC%252BC28bgslCksA%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2016&pages=652-667&author=A.+J.+DiPippoauthor=N.+K.+Patelauthor=C.+M.+Barnett&title=Cyclin-Dependent+Kinase+Inhibitors+for+the+Treatment+of+Breast+Cancer%3A+Past%2C+Present%2C+and+Future&doi=10.1002%2Fphar.1756"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Cyclin-Dependent Kinase Inhibitors for the Treatment of Breast Cancer: Past, Present, and Future</span></div><div class="casAuthors">DiPippo Adam J; Patel Neelam K; Barnett Chad M</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacotherapy</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">652-67</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Treatment of metastatic breast cancer (MBC) that is resistant to endocrine therapy presents a significant clinical challenge.  The well-known role of cell cycle dysregulation in these patients is partly mediated by cyclin-dependent kinase (CDK) activity.  Specific cyclin and CDK complexes regulate cell cycle progression by managing the transition through the cell cycle, and inhibition of CDKs represents an important target for novel agents.  First-generation CDK inhibitors (e.g., flavopiridol) were relatively nonselective and had an unacceptable toxicity profile in early trials.  Second-generation CDK inhibitors were designed to target the CDK4 and CDK6 (CDK4/6) pathway and have shown promising clinical activity with an acceptable toxicity profile in patients with MBC.  Palbociclib is a first-in-class CDK4/6 inhibitor that was granted accelerated U.S.  Food and Drug Administration approval in combination with letrozole for the treatment of MBC in the first-line setting (February 2015) as well as in combination with fulvestrant for MBC that had progressed on previous endocrine therapy (February 2016).  Other CDK4/6 inhibitors, including ribociclib and abemaciclib, are under investigation as monotherapy and in combination with endocrine or anti-human epidermal growth receptor 2 therapy for the treatment of MBC.  Ongoing clinical trials should provide additional information to guide the appropriate use of these agents and identify patient populations that could derive the most benefit.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSYivEamzCCty86Js-BCLiifW6udTcc2eak-S4biKgtPrntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28bgslCksA%253D%253D&md5=cc590c6fb21bbdc1e5384e4020a4123f</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1002%2Fphar.1756&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fphar.1756%26sid%3Dliteratum%253Aachs%26aulast%3DDiPippo%26aufirst%3DA.%2BJ.%26aulast%3DPatel%26aufirst%3DN.%2BK.%26aulast%3DBarnett%26aufirst%3DC.%2BM.%26atitle%3DCyclin-Dependent%2520Kinase%2520Inhibitors%2520for%2520the%2520Treatment%2520of%2520Breast%2520Cancer%253A%2520Past%252C%2520Present%252C%2520and%2520Future%26jtitle%3DPharmacotherapy%26date%3D2016%26volume%3D36%26spage%3D652%26epage%3D667%26doi%3D10.1002%2Fphar.1756" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roskoski, R.,  Jr.</span></span> <span> </span><span class="NLM_article-title">Cyclin-dependent Protein Kinase Inhibitors Including Palbociclib as Anticancer Drugs</span>. <i>Pharmacol. Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>107</i></span>,  <span class="NLM_fpage">249</span>â <span class="NLM_lpage">275</span>, <span class="refDoi">Â DOI: 10.1016/j.phrs.2016.03.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=10.1016%2Fj.phrs.2016.03.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=26995305" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=1%3ACAS%3A528%3ADC%252BC28XnvFOjt7o%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=107&publication_year=2016&pages=249-275&author=R.+Roskoski&title=Cyclin-dependent+Protein+Kinase+Inhibitors+Including+Palbociclib+as+Anticancer+Drugs&doi=10.1016%2Fj.phrs.2016.03.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Cyclin-dependent protein kinase inhibitors including palbociclib as anticancer drugs</span></div><div class="casAuthors">Roskoski, Robert, Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacological Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">107</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">249-275</span>CODEN:
                <span class="NLM_cas:coden">PHMREP</span>;
        ISSN:<span class="NLM_cas:issn">1043-6618</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Cyclins and cyclin-dependent protein kinases (CDKs) are important regulatory components that are required for cell cycle progression.  The levels of the cell cycle CDKs are generally const. and their activities are controlled by cyclins, proteins whose levels oscillate during each cell cycle.  Addnl. CDK family members were subsequently discovered that play significant roles in a wide range of activities including the control of gene transcription, metab., and neuronal function.  In response to mitogenic stimuli, cells in the G1 phase of the cell cycle produce cyclins of the D type that activate CDK4/6.  These activated enzymes catalyze the monophosphorylation of the retinoblastoma protein.  Then CDK2-cyclin E catalyzes the hyperphosphorylation of Rb that promotes the release and activation of the E2F transcription factors, which in turn lead to the generation of several proteins required for cell cycle progression.  As a result, cells pass through the G1-restriction point and are committed to complete cell division.  CDK2-cyclin A, CDK1-cyclin A, and CDK1-cyclin B are required for S, G2, and M-phase progression.  Increased cyclin or CDK expression or decreased levels of endogenous CDK inhibitors such as INK4 or CIP/KIP have been obsd. in various cancers.  In contrast to the mutational activation of EGFR, Kit, or B-Raf in the pathogenesis of malignancies, mutations in the CDKs that cause cancers are rare.  Owing to their role in cell proliferation, CDKs represent natural targets for anticancer therapies.  Abemaciclib (LY2835219), ribociclib (Lee011), and palbociclib (Ibrance or PD0332991) target CDK4/6 with IC50 values in the low nanomolar range.  Palbociclib and other CDK inhibitors bind in the cleft between the small and large lobes of the CDKs and inhibit the binding of ATP.  Like ATP, palbociclib forms hydrogen bonds with residues in the hinge segment of the cleft.  Like the adenine base of ATP, palbociclib interacts with catalytic spine residues CS6 and CS7.  CDK antagonists are in clin. trials for the treatment of a variety of malignancies.  Significantly, palbociclib has been approved by the FDA for the treatment of hormone-receptor pos./human epidermal growth factor receptor-2 neg. breast cancer in conjunction with letrozole as a first-line therapy and with fulvestrant as a second-line treatment.  As inhibitors of the cell cycle, it is not surprising that one of their most common toxicities is myelosuppression with decreased neutrophil prodn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGol0On7HAFtl7Vg90H21EOLACvtfcHk0lj9ptwhS8fpSw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XnvFOjt7o%253D&md5=0356807b16735e3903706eac10635bc7</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2Fj.phrs.2016.03.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.phrs.2016.03.012%26sid%3Dliteratum%253Aachs%26aulast%3DRoskoski%26aufirst%3DR.%26atitle%3DCyclin-dependent%2520Protein%2520Kinase%2520Inhibitors%2520Including%2520Palbociclib%2520as%2520Anticancer%2520Drugs%26jtitle%3DPharmacol.%2520Res.%26date%3D2016%26volume%3D107%26spage%3D249%26epage%3D275%26doi%3D10.1016%2Fj.phrs.2016.03.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCartney, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Migliaccio, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonechi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biagioni, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romagnoli, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Luca, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galardi, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Risi, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Santo, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benelli, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malorni, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Leo, A.</span></span> <span> </span><span class="NLM_article-title">Mechanisms of Resistance to CDK4/6 Inhibitors: Potential Implications and Biomarkers for Clinical Practice</span>. <i>Front. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">666</span>, <span class="refDoi">Â DOI: 10.3389/fonc.2019.00666</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=10.3389%2Ffonc.2019.00666" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=31396487" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=1%3ACAS%3A280%3ADC%252BB3MvmtlKrug%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2019&pages=666&author=A.+McCartneyauthor=I.+Migliaccioauthor=M.+Bonechiauthor=C.+Biagioniauthor=D.+Romagnoliauthor=F.+De+Lucaauthor=F.+Galardiauthor=E.+Risiauthor=I.+De+Santoauthor=M.+Benelliauthor=L.+Malorniauthor=A.+Di+Leo&title=Mechanisms+of+Resistance+to+CDK4%2F6+Inhibitors%3A+Potential+Implications+and+Biomarkers+for+Clinical+Practice&doi=10.3389%2Ffonc.2019.00666"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanisms of Resistance to CDK4/6 Inhibitors: Potential Implications and Biomarkers for Clinical Practice</span></div><div class="casAuthors">McCartney Amelia; Risi Emanuela; De Santo Irene; Malorni Luca; Di Leo Angelo; Migliaccio Ilenia; Bonechi Martina; De Luca Francesca; Galardi Francesca; Malorni Luca; Biagioni Chiara; Romagnoli Dario; Benelli Matteo</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in oncology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">666</span>
        ISSN:<span class="NLM_cas:issn">2234-943X</span>.
    </div><div class="casAbstract">The recent arrival of CDK4/6 inhibitor agents, with an approximate doubling of progression-free survival (PFS) associated with their use in hormone receptor-positive, HER2-negative advanced breast cancer (BC), has radically changed the approach to managing this disease.  However, resistance to CDK4/6 inhibitors is considered a near-inevitability in most patients.  Mechanisms of resistance to these agents are multifactorial, and research in this field is still evolving.  Biomarkers with the ability to identify early resistance, or to predict the likelihood of successful treatment using CDK4/6 inhibitors are yet to be identified, and represent an area of unmet clinical need.  Here we present selected mechanisms of resistance to CDK4/6 inhibitors, largely focussing on roles of Rb, cyclin E1, and the PIK3CA pathway, with discussion of associated biomarkers which have been investigated and applied in recent pre-clinical and clinical studies.  These biological drivers may furthermore influence clinical treatment strategies adopted beyond CDK4/6 resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRrJ96Nuqm037P3FVUi_gf8fW6udTcc2eZrzNLi3FgzPbntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3MvmtlKrug%253D%253D&md5=cea69c3a374c8599fe94ba34f65e6eab</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.3389%2Ffonc.2019.00666&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffonc.2019.00666%26sid%3Dliteratum%253Aachs%26aulast%3DMcCartney%26aufirst%3DA.%26aulast%3DMigliaccio%26aufirst%3DI.%26aulast%3DBonechi%26aufirst%3DM.%26aulast%3DBiagioni%26aufirst%3DC.%26aulast%3DRomagnoli%26aufirst%3DD.%26aulast%3DDe%2BLuca%26aufirst%3DF.%26aulast%3DGalardi%26aufirst%3DF.%26aulast%3DRisi%26aufirst%3DE.%26aulast%3DDe%2BSanto%26aufirst%3DI.%26aulast%3DBenelli%26aufirst%3DM.%26aulast%3DMalorni%26aufirst%3DL.%26aulast%3DDi%2BLeo%26aufirst%3DA.%26atitle%3DMechanisms%2520of%2520Resistance%2520to%2520CDK4%252F6%2520Inhibitors%253A%2520Potential%2520Implications%2520and%2520Biomarkers%2520for%2520Clinical%2520Practice%26jtitle%3DFront.%2520Oncol.%26date%3D2019%26volume%3D9%26spage%3D666%26doi%3D10.3389%2Ffonc.2019.00666" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Condorelli, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spring, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">OâShaughnessy, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lacroix, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bailleux, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dubois, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagy, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lanman, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iafrate, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andre, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bardia, A.</span></span> <span> </span><span class="NLM_article-title">Polyclonal RB1 Mutations and Acquired Resistance to CDK 4/6 Inhibitors in Patients with Metastatic Breast Cancer</span>. <i>Ann. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">640</span>â <span class="NLM_lpage">645</span>, <span class="refDoi">Â DOI: 10.1093/annonc/mdx784</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=10.1093%2Fannonc%2Fmdx784" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=29236940" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=1%3ACAS%3A280%3ADC%252BC1MzhtFWisw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2018&pages=640-645&author=R.+Condorelliauthor=L.+Springauthor=J.+O%E2%80%99Shaughnessyauthor=L.+Lacroixauthor=C.+Bailleuxauthor=V.+Scottauthor=J.+Duboisauthor=R.+J.+Nagyauthor=R.+B.+Lanmanauthor=A.+J.+Iafrateauthor=F.+Andreauthor=A.+Bardia&title=Polyclonal+RB1+Mutations+and+Acquired+Resistance+to+CDK+4%2F6+Inhibitors+in+Patients+with+Metastatic+Breast+Cancer&doi=10.1093%2Fannonc%2Fmdx784"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Polyclonal RB1 mutations and acquired resistance to CDK 4/6 inhibitors in patients with metastatic breast cancer</span></div><div class="casAuthors">Condorelli R; Lacroix L; Bailleux C; Scott V; Andre F; Spring L; Dubois J; Iafrate A J; Bardia A; O'Shaughnessy J; O'Shaughnessy J; Nagy R J; Lanman R B</div><div class="citationInfo"><span class="NLM_cas:title">Annals of oncology : official journal of the European Society for Medical Oncology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">640-645</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Background:  While deregulation of the cyclin D1-CDK4/6-retinoblastoma pathway is common in hormone receptor positive (HR+) breast cancer, Rb is usually intact in HR+ breast cancer, and targeted CDK 4/6 inhibitors that act upstream of Rb, are routinely being utilized in clinical practice.  However, factors that can lead to clinical resistance to CDK 4/6 inhibitors are not known.  Patients and methods:  We identified patients who had pre- and post-genotyping in tissue and peripheral blood samples after receiving CDK 4/6 inhibitors.  Genotyping was carried out in tumor tissue or blood collected before start of CDK 4/6 inhibitor and after disease progression on CDK 4/6 inhibitor, covering more than 90% of the coding region in RB1.  Results:  We identified detectable acquired RB1 mutations in circulating tumor DNA (ctDNA) after exposure to CDK4/6 inhibitor (palbociclib, palbociclib, ribociclib) for 5, 8, and 13 months, respectively, in three patients.  The RB1 mutations included substitution in donor splicing site of exon 8 of the RB1 gene in patient #1; substitution in donor splicing site of exon 22 of RB1 gene, exon 19 deletion, exon 3 insertion in patient #2; and RB1 exon 16 H483Y mutation in patient #3.  None of these RB1 mutations were present in the pre-CDK 4/6 specimen highlighting these molecular alterations, which lead to functional loss of Rb1, likely emerged under selective pressure from the CDK4/6 inhibitor potentially confering therapeutic resistance.  Conclusion:  This is the first clinical report to describe the emergence of somatic RB1 mutations after exposure to palbociclib or ribociclib, in patients with metastatic breast cancer.  Further research is needed to validate these findings, identify how these mutations temporally emerge under selective pressure of CDK 4/6 inhibitor, and develop rational therapeutic strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT57Ei4We3h7fIW7hznt0zpfW6udTcc2eapNBswQV4eRLntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1MzhtFWisw%253D%253D&md5=0f659618552c56fdc7b21f50bbb7c0cf</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1093%2Fannonc%2Fmdx784&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fannonc%252Fmdx784%26sid%3Dliteratum%253Aachs%26aulast%3DCondorelli%26aufirst%3DR.%26aulast%3DSpring%26aufirst%3DL.%26aulast%3DO%25E2%2580%2599Shaughnessy%26aufirst%3DJ.%26aulast%3DLacroix%26aufirst%3DL.%26aulast%3DBailleux%26aufirst%3DC.%26aulast%3DScott%26aufirst%3DV.%26aulast%3DDubois%26aufirst%3DJ.%26aulast%3DNagy%26aufirst%3DR.%2BJ.%26aulast%3DLanman%26aufirst%3DR.%2BB.%26aulast%3DIafrate%26aufirst%3DA.%2BJ.%26aulast%3DAndre%26aufirst%3DF.%26aulast%3DBardia%26aufirst%3DA.%26atitle%3DPolyclonal%2520RB1%2520Mutations%2520and%2520Acquired%2520Resistance%2520to%2520CDK%25204%252F6%2520Inhibitors%2520in%2520Patients%2520with%2520Metastatic%2520Breast%2520Cancer%26jtitle%3DAnn.%2520Oncol.%26date%3D2018%26volume%3D29%26spage%3D640%26epage%3D645%26doi%3D10.1093%2Fannonc%2Fmdx784" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kent, L. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leone, G.</span></span> <span> </span><span class="NLM_article-title">The broken cycle: E2F Dysfunction in Cancer</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">326</span>â <span class="NLM_lpage">338</span>, <span class="refDoi">Â DOI: 10.1038/s41568-019-0143-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=10.1038%2Fs41568-019-0143-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=31053804" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=1%3ACAS%3A528%3ADC%252BC1MXptFSqt7k%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2019&pages=326-338&author=L.+N.+Kentauthor=G.+Leone&title=The+broken+cycle%3A+E2F+Dysfunction+in+Cancer&doi=10.1038%2Fs41568-019-0143-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">The broken cycle: E2F dysfunction in cancer</span></div><div class="casAuthors">Kent, Lindsey N.; Leone, Gustavo</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">326-338</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">The cyclin-dependent kinase (CDK)-RB-E2F axis forms the core transcriptional machinery driving cell cycle progression, dictating the timing and fidelity of genome replication and ensuring genetic material is accurately passed through each cell division cycle.  The ultimate effectors of this axis are members of a family of eight distinct E2F genes encoding transcriptional activators and repressors.  E2F transcriptional activity is tightly regulated throughout the cell cycle via transcriptional and translational regulation, post-translational modifications, protein degrdn., binding to cofactors and subcellular localization.  Alterations in one or more key components of this axis (CDKs, cyclins, CDK inhibitors and the RB family of proteins) occur in virtually all cancers and result in heightened oncogenic E2F activity, leading to uncontrolled proliferation.  In this Review, we discuss the activities of E2F proteins with an emphasis on the newest atypical E2F family members, the specific and redundant functions of E2F proteins, how misexpression of E2F transcriptional targets promotes cancer and both current and developing therapeutic strategies being used to target this oncogenic pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZOonvudzTx7Vg90H21EOLACvtfcHk0liPlBP697J5Cw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXptFSqt7k%253D&md5=8f9b774c2dbdfee74823ab5a8f14b4cc</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1038%2Fs41568-019-0143-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41568-019-0143-7%26sid%3Dliteratum%253Aachs%26aulast%3DKent%26aufirst%3DL.%2BN.%26aulast%3DLeone%26aufirst%3DG.%26atitle%3DThe%2520broken%2520cycle%253A%2520E2F%2520Dysfunction%2520in%2520Cancer%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2019%26volume%3D19%26spage%3D326%26epage%3D338%26doi%3D10.1038%2Fs41568-019-0143-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Santo, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siu, K. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raje, N.</span></span> <span> </span><span class="NLM_article-title">Targeting Cyclin-Dependent Kinases and Cell Cycle Progression in Human Cancers</span>. <i>Semin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">788</span>â <span class="NLM_lpage">800</span>, <span class="refDoi">Â DOI: 10.1053/j.seminoncol.2015.09.024</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=10.1053%2Fj.seminoncol.2015.09.024" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=26615126" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFWksL7J" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2015&pages=788-800&author=L.+Santoauthor=K.+T.+Siuauthor=N.+Raje&title=Targeting+Cyclin-Dependent+Kinases+and+Cell+Cycle+Progression+in+Human+Cancers&doi=10.1053%2Fj.seminoncol.2015.09.024"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting Cyclin-Dependent Kinases and Cell Cycle Progression in Human Cancers</span></div><div class="casAuthors">Santo, Loredana; Siu, Ka Tat; Raje, Noopur</div><div class="citationInfo"><span class="NLM_cas:title">Seminars in Oncology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">788-800</span>CODEN:
                <span class="NLM_cas:coden">SOLGAV</span>;
        ISSN:<span class="NLM_cas:issn">0093-7754</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Uncontrolled cell division is a defining characteristic of cancer cells.  Cyclin-dependent kinases (Cdks) are crit. regulators of cell cycle progression.  Deregulated Cdk activities as a result of gene amplification, translocation, or point mutations of Cdks or cyclins, have been reported in a majority of human cancers.  These kinases, therefore, represent potential therapeutic targets for the treatment of cancer.  In this review, we offer an overview of Cdk functions in driving cell cycle progression and transcriptional regulation, a highlight of the DNA damage checkpoints, and an outline of the most relevant Cdk inhibitors currently in clin. trials with an emphasis on the Cdk inhibitors used for treatment of multiple myeloma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBLbVHqlZ2OrVg90H21EOLACvtfcHk0liPlBP697J5Cw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFWksL7J&md5=ee57a7c9b080ae0ad3860a4d8b2dd691</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1053%2Fj.seminoncol.2015.09.024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1053%252Fj.seminoncol.2015.09.024%26sid%3Dliteratum%253Aachs%26aulast%3DSanto%26aufirst%3DL.%26aulast%3DSiu%26aufirst%3DK.%2BT.%26aulast%3DRaje%26aufirst%3DN.%26atitle%3DTargeting%2520Cyclin-Dependent%2520Kinases%2520and%2520Cell%2520Cycle%2520Progression%2520in%2520Human%2520Cancers%26jtitle%3DSemin.%2520Oncol.%26date%3D2015%26volume%3D42%26spage%3D788%26epage%3D800%26doi%3D10.1053%2Fj.seminoncol.2015.09.024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weng, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zuo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span> <span> </span><span class="NLM_article-title">Recent Progress of Cyclin-dependent Kinase Inhibitors as Potential Anticancer Agents</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">2047</span>â <span class="NLM_lpage">2076</span>, <span class="refDoi">Â DOI: 10.4155/fmc-2016-0129</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=10.4155%2Ffmc-2016-0129" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=27774795" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=1%3ACAS%3A528%3ADC%252BC28XhslGisbnJ" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2016&pages=2047-2076&author=T.+Liauthor=T.+Wengauthor=M.+Zuoauthor=Z.+Weiauthor=M.+Chenauthor=Z.+Li&title=Recent+Progress+of+Cyclin-dependent+Kinase+Inhibitors+as+Potential+Anticancer+Agents&doi=10.4155%2Ffmc-2016-0129"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Recent progress of cyclin-dependent kinase inhibitors as potential anticancer agents</span></div><div class="casAuthors">Li, Tinghan; Weng, Tianwei; Zuo, Minzan; Wei, Zhihui; Chen, Ming; Li, Zhiyu</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">2047-2076</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">Deregulation of the cell cycle is a common feature in human cancer.  The inhibition of cyclin-dependent kinases (CDKs), which play a crucial role in control of the cell cycle, has always been one of the most promising areas in cancer chemotherapy.  This review first summarizes the biol. of CDKs and then focuses on the recent advances in both broad-range and selective CDK inhibitors during the last 5 years.  The design rationale, structural optimization and structure-activity relationships anal. of these small mols. have been discussed in detail and the key interactions with the amino-acid residues of the most important compds. are highlighted.  Future perspectives for CDKs inhibitors will be defined in the development of highly selective CDK inhibitors, an accurate knowledge of gene control mechanism and further predictive biomarker research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrKW1A5UEVYeLVg90H21EOLACvtfcHk0liPlBP697J5Cw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhslGisbnJ&md5=90331b2b80a383eaee6483a6e6e6e1e6</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.4155%2Ffmc-2016-0129&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc-2016-0129%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DT.%26aulast%3DWeng%26aufirst%3DT.%26aulast%3DZuo%26aufirst%3DM.%26aulast%3DWei%26aufirst%3DZ.%26aulast%3DChen%26aufirst%3DM.%26aulast%3DLi%26aufirst%3DZ.%26atitle%3DRecent%2520Progress%2520of%2520Cyclin-dependent%2520Kinase%2520Inhibitors%2520as%2520Potential%2520Anticancer%2520Agents%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2016%26volume%3D8%26spage%3D2047%26epage%3D2076%26doi%3D10.4155%2Ffmc-2016-0129" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ling, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, T.</span></span> <span> </span><span class="NLM_article-title">Discovery of 4-((7H-Pyrrolo[2,3-d]pyrimidin-4-yl)amino)-N-(4-((4-methylpiperazin-1-yl)methyl)phenyl)-1H-pyrazole-3-carboxamide (FN-1501), an FLT3- and CDK-Kinase Inhibitor with Potentially High Efficiency against Acute Myelocytic Leukemia</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">1499</span>â <span class="NLM_lpage">1518</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.7b01261</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01261" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtlyqurg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=1499-1518&author=Y.+Wangauthor=Y.+Zhiauthor=Q.+Jinauthor=S.+Luauthor=G.+Linauthor=H.+Yuanauthor=T.+Yangauthor=Z.+Wangauthor=C.+Yaoauthor=J.+Lingauthor=H.+Guoauthor=T.+Liauthor=J.+Jinauthor=B.+Liauthor=L.+Zhangauthor=Y.+Chenauthor=T.+Lu&title=Discovery+of+4-%28%287H-Pyrrolo%5B2%2C3-d%5Dpyrimidin-4-yl%29amino%29-N-%284-%28%284-methylpiperazin-1-yl%29methyl%29phenyl%29-1H-pyrazole-3-carboxamide+%28FN-1501%29%2C+an+FLT3-+and+CDK-Kinase+Inhibitor+with+Potentially+High+Efficiency+against+Acute+Myelocytic+Leukemia&doi=10.1021%2Facs.jmedchem.7b01261"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of 4-((7H-Pyrrolo[2,3-d]pyrimidin-4-yl)amino)-N-(4-((4-methylpiperazin-1-yl)methyl)phenyl)-1H-pyrazole-3-carboxamide (FN-1501), an FLT3- and CDK-Kinase Inhibitor with Potentially High Efficiency against Acute Myelocytic Leukemia</span></div><div class="casAuthors">Wang, Yue; Zhi, Yanle; Jin, Qiaomei; Lu, Shuai; Lin, Guowu; Yuan, Haoliang; Yang, Taotao; Wang, Zhanwei; Yao, Chao; Ling, Jun; Guo, Hao; Li, Tonghui; Jin, Jianlin; Li, Baoquan; Zhang, Li; Chen, Yadong; Lu, Tao</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1499-1518</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A series of 1-H-pyrazole-3-carboxamide derivs. have been designed and synthesized that exhibit excellent FLT3 and CDK inhibition and antiproliferative activities.  A structure-activity-relationship study illustrates that the incorporation of a pyrimidine-fused heterocycle at position 4 of the pyrazole is crit. for FLT3 and CDK inhibition.  Compd. 50 (FN-1501), which possesses potent inhibitory activities against FLT3, CDK2, CDK4, and CDK6 with IC50 values in the nanomolar range, shows antiproliferative activities against MV4-11 cells (IC50: 0.008 Î¼M), which correlates with the suppression of retinoblastoma phosphorylation, FLT3, ERK, AKT, and STAT5 and the onset of apoptosis.  Acute-toxicity studies in mice show that compd. I (LD50: 186 mg/kg) is safer than AT7519 (32 mg/kg).  In MV4-11 xenografts in a nude-mouse model, compd. I can induce tumor regression at the dose of 15 mg/kg, which is more efficient than cytarabine (50 mg/kg).  Taken together, these results demonstrate the potential of this unique compd. for further development into a drug applied in acute-myeloid-leukemia (AML) therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrfX9S4QkGccbVg90H21EOLACvtfcHk0lgRXFh99uBEOQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtlyqurg%253D&md5=2757ad7559e655c5d96f0a8af8c74aa5</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01261&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01261%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DY.%26aulast%3DZhi%26aufirst%3DY.%26aulast%3DJin%26aufirst%3DQ.%26aulast%3DLu%26aufirst%3DS.%26aulast%3DLin%26aufirst%3DG.%26aulast%3DYuan%26aufirst%3DH.%26aulast%3DYang%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DYao%26aufirst%3DC.%26aulast%3DLing%26aufirst%3DJ.%26aulast%3DGuo%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DT.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DB.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DLu%26aufirst%3DT.%26atitle%3DDiscovery%2520of%25204-%2528%25287H-Pyrrolo%255B2%252C3-d%255Dpyrimidin-4-yl%2529amino%2529-N-%25284-%2528%25284-methylpiperazin-1-yl%2529methyl%2529phenyl%2529-1H-pyrazole-3-carboxamide%2520%2528FN-1501%2529%252C%2520an%2520FLT3-%2520and%2520CDK-Kinase%2520Inhibitor%2520with%2520Potentially%2520High%2520Efficiency%2520against%2520Acute%2520Myelocytic%2520Leukemia%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D1499%26epage%3D1518%26doi%3D10.1021%2Facs.jmedchem.7b01261" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">PrÃ©vel, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurzawa, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van, T. N. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morris, M. C.</span></span> <span> </span><span class="NLM_article-title">Fluorescent Biosensors for Drug Discovery New Tools for Old Targets-screening for Inhibitors of Cyclin-Dependent Kinases</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>88</i></span>,  <span class="NLM_fpage">74</span>â <span class="NLM_lpage">88</span>, <span class="refDoi">Â DOI: 10.1016/j.ejmech.2014.10.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=10.1016%2Fj.ejmech.2014.10.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=25314935" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1yhurnK" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=88&publication_year=2014&pages=74-88&author=C.+Pr%C3%A9velauthor=L.+Kurzawaauthor=T.+N.+N.+Vanauthor=M.+C.+Morris&title=Fluorescent+Biosensors+for+Drug+Discovery+New+Tools+for+Old+Targets-screening+for+Inhibitors+of+Cyclin-Dependent+Kinases&doi=10.1016%2Fj.ejmech.2014.10.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Fluorescent biosensors for drug discovery new tools for old targets - Screening for inhibitors of cyclin-dependent kinases</span></div><div class="casAuthors">Prevel, Camille; Kurzawa, Laetitia; Van, Thi Nhu Ngoc; Morris, May C.</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">88</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">74-88</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">A review.  Cyclin-dependent kinases play central roles in regulation of cell cycle progression, transcriptional regulation and other major biol. processes such as neuronal differentiation and metab.  These kinases are hyperactivated in most human cancers and constitute attractive pharmacol. targets.  A large no. of ATP-competitive inhibitors of CDKs have been identified from natural substances, in high throughput screening assays, or through structure-guided approaches.  Alternative strategies have been explored to target essential protein/protein interfaces and screen for allosteric inhibitors that trap inactive intermediates or prevent conformational activation.  However this remains a major challenge given the highly conserved structural features of these kinases, and calls for new and alternative screening technologies.  Fluorescent biosensors constitute powerful tools for the detection of biomols. in complex biol. samples, and are well suited to study dynamic processes and highlight mol. alterations assocd. with pathol. disorders.  They further constitute sensitive and selective tools which can be readily implemented to high throughput and high content screens in drug discovery programs.  Our group has developed fluorescent biosensors to probe cyclin-dependent kinases and gain insight into their mol. behavior in vitro and in living cells.  These tools provide a means of monitoring subtle alterations in the abundance and activity of CDK/Cyclins and can respond to compds. that interfere with the conformational dynamics of these kinases.  In this review we discuss the different strategies which have been devised to target CDK/Cyclins, and describe the implementation of our CDK/Cyclin biosensors to develop HTS/HCS assays in view of identifying new classes of inhibitors for cancer therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXcNXQWPLI_bVg90H21EOLACvtfcHk0lgRXFh99uBEOQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1yhurnK&md5=93fecc37ae33d1a1b6370a89cb9d28d0</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2014.10.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2014.10.003%26sid%3Dliteratum%253Aachs%26aulast%3DPr%25C3%25A9vel%26aufirst%3DC.%26aulast%3DKurzawa%26aufirst%3DL.%26aulast%3DVan%26aufirst%3DT.%2BN.%2BN.%26aulast%3DMorris%26aufirst%3DM.%2BC.%26atitle%3DFluorescent%2520Biosensors%2520for%2520Drug%2520Discovery%2520New%2520Tools%2520for%2520Old%2520Targets-screening%2520for%2520Inhibitors%2520of%2520Cyclin-Dependent%2520Kinases%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2014%26volume%3D88%26spage%3D74%26epage%3D88%26doi%3D10.1016%2Fj.ejmech.2014.10.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Coxon, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anscombe, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harnor, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carbain, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golding, B. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hardcastle, I. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harlow, L. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korolchuk, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matheson, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Newell, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noble, M. E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sivaprakasam, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tudhope, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turner, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wedge, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffin, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Endicott, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cano, C.</span></span> <span> </span><span class="NLM_article-title">Cyclin-Dependent Kinase (CDK) Inhibitors: Structure-Activity Relationships and Insights into the CDK-2 Selectivity of 6-Substituted 2-Arylaminopurines</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">1746</span>â <span class="NLM_lpage">1767</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.6b01254</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01254" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=1%3ACAS%3A528%3ADC%252BC28XitFCgs7fK" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=1746-1767&author=C.+R.+Coxonauthor=E.+Anscombeauthor=S.+J.+Harnorauthor=M.+P.+Martinauthor=B.+Carbainauthor=B.+T.+Goldingauthor=I.+R.+Hardcastleauthor=L.+K.+Harlowauthor=S.+Korolchukauthor=C.+J.+Mathesonauthor=D.+R.+Newellauthor=M.+E.+M.+Nobleauthor=M.+Sivaprakasamauthor=S.+J.+Tudhopeauthor=D.+M.+Turnerauthor=L.+Z.+Wangauthor=S.+R.+Wedgeauthor=C.+Wongauthor=R.+J.+Griffinauthor=J.+A.+Endicottauthor=C.+Cano&title=Cyclin-Dependent+Kinase+%28CDK%29+Inhibitors%3A+Structure-Activity+Relationships+and+Insights+into+the+CDK-2+Selectivity+of+6-Substituted+2-Arylaminopurines&doi=10.1021%2Facs.jmedchem.6b01254"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Cyclin-Dependent Kinase (CDK) Inhibitors: Structure-Activity Relationships and Insights into the CDK-2 Selectivity of 6-Substituted 2-Arylaminopurines</span></div><div class="casAuthors">Coxon, Christopher R.; Anscombe, Elizabeth; Harnor, Suzannah J.; Martin, Mathew P.; Carbain, Benoit; Golding, Bernard T.; Hardcastle, Ian R.; Harlow, Lisa K.; Korolchuk, Svitlana; Matheson, Christopher J.; Newell, David R.; Noble, Martin E. M.; Sivaprakasam, Mangaleswaran; Tudhope, Susan J.; Turner, David M.; Wang, Lan Z.; Wedge, Stephen R.; Wong, Christopher; Griffin, Roger J.; Endicott, Jane A.; Cano, Celine</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1746-1767</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Purines and related heterocycles substituted at C-2 with 4'-sulfamoylanilino and at C-6 with a variety of groups have been synthesized with the aim of achieving selectivity of binding to CDK2 over CDK1.  6-Substituents that favor competitive inhibition at the ATP binding site of CDK2 were identified and typically exhibited 10-80-fold greater inhibition of CDK2 compared to CDK1.  Most impressive was 4-((6-([1,1'-biphenyl]-3-yl)-9H-purin-2-yl)amino) benzenesulfonamide (73) that exhibited high potency towards CDK2 (IC50 0.044 Î¼M), but was â¼ 2000-fold less active towards CDK1 (IC50 86 Î¼M).  This compd. is therefore a useful tool for studies of cell cycle regulation.  Crystal structures of inhibitor-kinase complexes showed that the inhibitor stabilizes a glycine-rich loop conformation that shapes the ATP ribose binding pocket, and that is preferred in CDK2, but has not been obsd. in CDK1.  This aspect of the active site may be exploited for the design of inhibitors that distinguish between CDK1 and CDK2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGocYzXYCKo7CrVg90H21EOLACvtfcHk0lik2eBTkJ-CaQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitFCgs7fK&md5=520549b69fec2b2af9758bfcf8708c55</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01254&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01254%26sid%3Dliteratum%253Aachs%26aulast%3DCoxon%26aufirst%3DC.%2BR.%26aulast%3DAnscombe%26aufirst%3DE.%26aulast%3DHarnor%26aufirst%3DS.%2BJ.%26aulast%3DMartin%26aufirst%3DM.%2BP.%26aulast%3DCarbain%26aufirst%3DB.%26aulast%3DGolding%26aufirst%3DB.%2BT.%26aulast%3DHardcastle%26aufirst%3DI.%2BR.%26aulast%3DHarlow%26aufirst%3DL.%2BK.%26aulast%3DKorolchuk%26aufirst%3DS.%26aulast%3DMatheson%26aufirst%3DC.%2BJ.%26aulast%3DNewell%26aufirst%3DD.%2BR.%26aulast%3DNoble%26aufirst%3DM.%2BE.%2BM.%26aulast%3DSivaprakasam%26aufirst%3DM.%26aulast%3DTudhope%26aufirst%3DS.%2BJ.%26aulast%3DTurner%26aufirst%3DD.%2BM.%26aulast%3DWang%26aufirst%3DL.%2BZ.%26aulast%3DWedge%26aufirst%3DS.%2BR.%26aulast%3DWong%26aufirst%3DC.%26aulast%3DGriffin%26aufirst%3DR.%2BJ.%26aulast%3DEndicott%26aufirst%3DJ.%2BA.%26aulast%3DCano%26aufirst%3DC.%26atitle%3DCyclin-Dependent%2520Kinase%2520%2528CDK%2529%2520Inhibitors%253A%2520Structure-Activity%2520Relationships%2520and%2520Insights%2520into%2520the%2520CDK-2%2520Selectivity%2520of%25206-Substituted%25202-Arylaminopurines%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D1746%26epage%3D1767%26doi%3D10.1021%2Facs.jmedchem.6b01254" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, T. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lozach, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Surpateanu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Retailleau, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iorga, B. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meijer, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">GuÃ©ritte, F.</span></span> <span> </span><span class="NLM_article-title">Synthesis, Biological Evaluation, and Molecular Modeling of Natural and Unnatural Flavonoidal Alkaloids, Inhibitors of Kinases</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">2811</span>â <span class="NLM_lpage">2819</span>, <span class="refDoi">Â DOI: 10.1021/jm201727w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm201727w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=1%3ACAS%3A528%3ADC%252BC38XislSls70%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=2811-2819&author=T.+B.+Nguyenauthor=O.+Lozachauthor=G.+Surpateanuauthor=Q.+Wangauthor=P.+Retailleauauthor=B.+I.+Iorgaauthor=L.+Meijerauthor=F.+Gu%C3%A9ritte&title=Synthesis%2C+Biological+Evaluation%2C+and+Molecular+Modeling+of+Natural+and+Unnatural+Flavonoidal+Alkaloids%2C+Inhibitors+of+Kinases&doi=10.1021%2Fjm201727w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis, Biological Evaluation, and Molecular Modeling of Natural and Unnatural Flavonoidal Alkaloids, Inhibitors of Kinases</span></div><div class="casAuthors">Nguyen, Thanh Binh; Lozach, Olivier; Surpateanu, Georgiana; Wang, Qian; Retailleau, Pascal; Iorga, Bogdan I.; Meijer, Laurent; Gueritte, Francoise</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2811-2819</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The screening of the ICSN chem. library on various disease-relevant protein kinases led to the identification of natural flavonoidal alkaloids of unknown configuration as potent inhibitors of the CDK1 and CDK5 kinases.  We thus developed an efficient and modular synthetic strategy for their prepn. and that of analogs in order to det. the abs. configuration of the active natural flavonoidal alkaloids and to provide further insights on the structure-activity relationships in this series.  For example, reacting chrysin with hexanal and MeNH2 gave a mixt. of 5,7-dihydroxy-6-(1-methylamino)hexyl chromenone deriv. I [R = CH(NHMe)(CH2)4Me, R1 = H] and 5,7-dihydroxy-8-(1-methylamino)hexyl chromenone I [R = H, R1 = CH(NHMe)(CH2)4Me].  Sepn. of the 8-substituted deriv. into the R and S enantiomers was accomplished using SFC HPLC.  The structural determinants of the interaction between some flavonoidal alkaloids with specific kinases were also evaluated using mol. modeling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcblOGXLZewbVg90H21EOLACvtfcHk0lik2eBTkJ-CaQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XislSls70%253D&md5=632d257992af13ace1aea830e9353bd0</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Fjm201727w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm201727w%26sid%3Dliteratum%253Aachs%26aulast%3DNguyen%26aufirst%3DT.%2BB.%26aulast%3DLozach%26aufirst%3DO.%26aulast%3DSurpateanu%26aufirst%3DG.%26aulast%3DWang%26aufirst%3DQ.%26aulast%3DRetailleau%26aufirst%3DP.%26aulast%3DIorga%26aufirst%3DB.%2BI.%26aulast%3DMeijer%26aufirst%3DL.%26aulast%3DGu%25C3%25A9ritte%26aufirst%3DF.%26atitle%3DSynthesis%252C%2520Biological%2520Evaluation%252C%2520and%2520Molecular%2520Modeling%2520of%2520Natural%2520and%2520Unnatural%2520Flavonoidal%2520Alkaloids%252C%2520Inhibitors%2520of%2520Kinases%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D2811%26epage%3D2819%26doi%3D10.1021%2Fjm201727w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schonbrunn, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Betzi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alam, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becker, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Francis, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chakrasali, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jakkaraj, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kazi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sebti, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cubitt, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gebhard, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hazlehurst, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tash, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Georg, G. I.</span></span> <span> </span><span class="NLM_article-title">Development of Highly Potent and Selective Diaminothiazole Inhibitors of Cyclin-dependent Kinases</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">3768</span>â <span class="NLM_lpage">3782</span>, <span class="refDoi">Â DOI: 10.1021/jm301234k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm301234k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=1%3ACAS%3A528%3ADC%252BC3sXmtF2jtb8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=3768-3782&author=E.+Schonbrunnauthor=S.+Betziauthor=R.+Alamauthor=M.+P.+Martinauthor=A.+Beckerauthor=H.+Hanauthor=R.+Francisauthor=R.+Chakrasaliauthor=S.+Jakkarajauthor=A.+Kaziauthor=S.+M.+Sebtiauthor=C.+L.+Cubittauthor=A.+W.+Gebhardauthor=L.+A.+Hazlehurstauthor=J.+S.+Tashauthor=G.+I.+Georg&title=Development+of+Highly+Potent+and+Selective+Diaminothiazole+Inhibitors+of+Cyclin-dependent+Kinases&doi=10.1021%2Fjm301234k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Development of Highly Potent and Selective Diaminothiazole Inhibitors of Cyclin-Dependent Kinases</span></div><div class="casAuthors">Schonbrunn, Ernst; Betzi, Stephane; Alam, Riazul; Martin, Mathew P.; Becker, Andreas; Han, Huijong; Francis, Rawle; Chakrasali, Ramappa; Jakkaraj, Sudhakar; Kazi, Aslamuzzaman; Sebti, Said M.; Cubitt, Christopher L.; Gebhard, Anthony W.; Hazlehurst, Lori A.; Tash, Joseph S.; Georg, Gunda I.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3768-3782</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Diaminothiazoles such as I were prepd. as inhibitors of cyclin-dependent kinase 2 (CDK2) for potential use as antitumor agents and as male contraceptive agents based on the optimization of a hit compd. with weak inhibitory activity (IC50 = 15 Î¼M) discovered by high-throughput screening.  Using cocrystal structures of 35 diaminothiazoles with CDK2, the structure-activity relations of diaminothiazoles for CDK2 inhibition were detd. and used to find I.  I was tested for inhibition against a panel of 339 kinases; cyclin-dependent kinases were inhibited selectively, particularly CDK2 and CDK5 over CDK9, CDK1, CDK4, and CDK6.  I inhibited the proliferation of 13 out of 15 cancer cell lines with IC50 values of 0.27-6.9 Î¼M; the antiproliferative activity correlated to complete suppression of retinoblastoma phosphorylation and the onset of apoptosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpeM-VFFhEenrVg90H21EOLACvtfcHk0lik2eBTkJ-CaQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXmtF2jtb8%253D&md5=067bdf2349f9cd853b83d19131e0770f</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fjm301234k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm301234k%26sid%3Dliteratum%253Aachs%26aulast%3DSchonbrunn%26aufirst%3DE.%26aulast%3DBetzi%26aufirst%3DS.%26aulast%3DAlam%26aufirst%3DR.%26aulast%3DMartin%26aufirst%3DM.%2BP.%26aulast%3DBecker%26aufirst%3DA.%26aulast%3DHan%26aufirst%3DH.%26aulast%3DFrancis%26aufirst%3DR.%26aulast%3DChakrasali%26aufirst%3DR.%26aulast%3DJakkaraj%26aufirst%3DS.%26aulast%3DKazi%26aufirst%3DA.%26aulast%3DSebti%26aufirst%3DS.%2BM.%26aulast%3DCubitt%26aufirst%3DC.%2BL.%26aulast%3DGebhard%26aufirst%3DA.%2BW.%26aulast%3DHazlehurst%26aufirst%3DL.%2BA.%26aulast%3DTash%26aufirst%3DJ.%2BS.%26aulast%3DGeorg%26aufirst%3DG.%2BI.%26atitle%3DDevelopment%2520of%2520Highly%2520Potent%2520and%2520Selective%2520Diaminothiazole%2520Inhibitors%2520of%2520Cyclin-dependent%2520Kinases%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D3768%26epage%3D3782%26doi%3D10.1021%2Fjm301234k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kuo, G.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deangelis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emanuel, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Connolly, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gruninger, R. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rugg, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuentes-Pesquera, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Middleton, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jolliffe, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, W. V.</span></span> <span> </span><span class="NLM_article-title">Synthesis and Identification of [1,3,5]triazine-pyridine Biheteroaryl as a Novel Series of Potent Cyclin-dependent Kinase Inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">4535</span>â <span class="NLM_lpage">4546</span>, <span class="refDoi">Â DOI: 10.1021/jm040214h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm040214h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=1%3ACAS%3A528%3ADC%252BD2MXks1Sgt7w%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=4535-4546&author=G.-H.+Kuoauthor=A.+Deangelisauthor=S.+Emanuelauthor=A.+Wangauthor=Y.+Zhangauthor=P.+J.+Connollyauthor=X.+Chenauthor=R.+H.+Gruningerauthor=C.+Ruggauthor=A.+Fuentes-Pesqueraauthor=S.+A.+Middletonauthor=L.+Jolliffeauthor=W.+V.+Murray&title=Synthesis+and+Identification+of+%5B1%2C3%2C5%5Dtriazine-pyridine+Biheteroaryl+as+a+Novel+Series+of+Potent+Cyclin-dependent+Kinase+Inhibitors&doi=10.1021%2Fjm040214h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and Identification of [1,3,5]Triazine-pyridine Biheteroaryl as a Novel Series of Potent Cyclin-Dependent Kinase Inhibitors</span></div><div class="casAuthors">Kuo, Gee-Hong; DeAngelis, Alan; Emanuel, Stuart; Wang, Aihua; Zhang, Yan; Connolly, Peter J.; Chen, Xin; Gruninger, Robert H.; Rugg, Catherine; Fuentes-Pesquera, Angel; Middleton, Steven A.; Jolliffe, Linda; Murray, William V.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">4535-4546</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">On the basis of previous studies, a identified pyrazine-pyridine deriv. was identified as a potent vascular endothelial growth factor inhibitor and a pyrimidine-pyridine deriv. as a moderately potent cyclin dependent kinase (CDK) inhibitor.  A proposed combination of CGP-60474 and a pyrimidine-pyridine deriv. led to the discovery of [1,3,5]triazine-pyridine as a new series of potent CDK inhibitors.  Palladium-catalyzed C-C bond formation reactions, particularly the Negishi coupling reaction, were used to assemble various triazine-heteroaryl analogs effectively.  Among them, 3-[[4-[4-[(3-chlorophenyl)amino]-1,3,5-triazin-2-yl]-2-pyridinyl]amino]-1-propanol (I) displayed high inhibitory potency at CDK1 (IC50 = 0.021 Î¼M), CDK2, and CDK5 and sub-micromolar potency at CDK4, CDK6, and CDK7.  I also displayed high potency at GSK-3Î².  It demonstrated potent antiproliferative activity on various tumor cell lines, including HeLa, HCT-116, U937, and A375.  I was administered i.p. at 150 and 125 mg/kg to nude mice bearing human A375 xenografts, the compd. produced a significant survival increase.  Mol. docking studies were conducted in an attempt to enhance the understanding of the obsd. structure-activity relationship.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoj1MFdlX2V_LVg90H21EOLACvtfcHk0lh24kjEZs-Xtg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXks1Sgt7w%253D&md5=1901f66de5152ade248ad440a4a1cb7c</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fjm040214h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm040214h%26sid%3Dliteratum%253Aachs%26aulast%3DKuo%26aufirst%3DG.-H.%26aulast%3DDeangelis%26aufirst%3DA.%26aulast%3DEmanuel%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DA.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DConnolly%26aufirst%3DP.%2BJ.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DGruninger%26aufirst%3DR.%2BH.%26aulast%3DRugg%26aufirst%3DC.%26aulast%3DFuentes-Pesquera%26aufirst%3DA.%26aulast%3DMiddleton%26aufirst%3DS.%2BA.%26aulast%3DJolliffe%26aufirst%3DL.%26aulast%3DMurray%26aufirst%3DW.%2BV.%26atitle%3DSynthesis%2520and%2520Identification%2520of%2520%255B1%252C3%252C5%255Dtriazine-pyridine%2520Biheteroaryl%2520as%2520a%2520Novel%2520Series%2520of%2520Potent%2520Cyclin-dependent%2520Kinase%2520Inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D4535%26epage%3D4546%26doi%3D10.1021%2Fjm040214h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Singh, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chashoo, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khan, S. U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mahajan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nargotra, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mahajan, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mintoo, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guru, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aruri, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thatikonda, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sahu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chibber, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mir, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bharate, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madishetti, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nandi, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mondhe, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhushan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malik, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mignani, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vishwakarma, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, P. P.</span></span> <span> </span><span class="NLM_article-title">Design of Novel 3-Pyrimidinylazaindole CDK2/9 Inhibitors with Potent In Vitro and In Vivo Antitumor Efficacy in a Triple-Negative Breast Cancer Model</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">9470</span>â <span class="NLM_lpage">9489</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.7b00663</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00663" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvVWqtbnL" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=9470-9489&author=U.+Singhauthor=G.+Chashooauthor=S.+U.+Khanauthor=P.+Mahajanauthor=A.+Nargotraauthor=G.+Mahajanauthor=A.+Singhauthor=A.+Sharmaauthor=M.+J.+Mintooauthor=S.+K.+Guruauthor=H.+Aruriauthor=T.+Thatikondaauthor=P.+Sahuauthor=P.+Chibberauthor=V.+Kumarauthor=S.+A.+Mirauthor=S.+S.+Bharateauthor=S.+Madishettiauthor=U.+Nandiauthor=G.+Singhauthor=D.+M.+Mondheauthor=S.+Bhushanauthor=F.+Malikauthor=S.+Mignaniauthor=R.+A.+Vishwakarmaauthor=P.+P.+Singh&title=Design+of+Novel+3-Pyrimidinylazaindole+CDK2%2F9+Inhibitors+with+Potent+In+Vitro+and+In+Vivo+Antitumor+Efficacy+in+a+Triple-Negative+Breast+Cancer+Model&doi=10.1021%2Facs.jmedchem.7b00663"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Design of Novel 3-Pyrimidinylazaindole CDK2/9 Inhibitors with Potent In Vitro and In Vivo Antitumor Efficacy in a Triple-Negative Breast Cancer Model</span></div><div class="casAuthors">Singh, Umed; Chashoo, Gousia; Khan, Sameer U.; Mahajan, Priya; Nargotra, Amit; Mahajan, Girish; Singh, Amarinder; Sharma, Anjna; Mintoo, Mubashir J.; Guru, Santosh Kumar; Aruri, Hariprasad; Thatikonda, Thanusha; Sahu, Promod; Chibber, Pankaj; Kumar, Vikas; Mir, Sameer A.; Bharate, Sonali S.; Madishetti, Sreedhar; Nandi, Utpal; Singh, Gurdarshan; Mondhe, Dilip Manikrao; Bhushan, Shashi; Malik, Fayaz; Mignani, Serge; Vishwakarma, Ram A.; Singh, Parvinder Pal</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">9470-9489</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In the present study, novel series of 3-pyrimidinylazaindoles were designed and synthesized using bioinformatics strategy as cyclin-dependent kinases CDK2 and CDK9 inhibitors, which play crit. roles in cell-cycle control and regulation of cell transcription.  The present approach has given new dimensions to existing SAR and opens the new opportunity for lead optimization from comparatively inexpensive starting materials.  The study led to the identification of alternative lead candidate I with nanomolar potency against CDK2 and CDK9 and potent anti-proliferative activities against a panel of tested tumor cell lines along better safety ratio of â¼30 in comparison to reported leads.  In addn., the identified lead I has demonstrated good soly. and acceptable in vivo PK profile.  The identified lead I has also shown in vivo efficacy in mouse triple neg. breast cancer (TNBC) syngeneic model with TGI of 90% without any mortality growth inhibition in comparison to reported leads.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrvqLjfKAkGYbVg90H21EOLACvtfcHk0lh24kjEZs-Xtg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvVWqtbnL&md5=5f6edf3003a3a7b2e9b29d51c68846d1</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00663&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00663%26sid%3Dliteratum%253Aachs%26aulast%3DSingh%26aufirst%3DU.%26aulast%3DChashoo%26aufirst%3DG.%26aulast%3DKhan%26aufirst%3DS.%2BU.%26aulast%3DMahajan%26aufirst%3DP.%26aulast%3DNargotra%26aufirst%3DA.%26aulast%3DMahajan%26aufirst%3DG.%26aulast%3DSingh%26aufirst%3DA.%26aulast%3DSharma%26aufirst%3DA.%26aulast%3DMintoo%26aufirst%3DM.%2BJ.%26aulast%3DGuru%26aufirst%3DS.%2BK.%26aulast%3DAruri%26aufirst%3DH.%26aulast%3DThatikonda%26aufirst%3DT.%26aulast%3DSahu%26aufirst%3DP.%26aulast%3DChibber%26aufirst%3DP.%26aulast%3DKumar%26aufirst%3DV.%26aulast%3DMir%26aufirst%3DS.%2BA.%26aulast%3DBharate%26aufirst%3DS.%2BS.%26aulast%3DMadishetti%26aufirst%3DS.%26aulast%3DNandi%26aufirst%3DU.%26aulast%3DSingh%26aufirst%3DG.%26aulast%3DMondhe%26aufirst%3DD.%2BM.%26aulast%3DBhushan%26aufirst%3DS.%26aulast%3DMalik%26aufirst%3DF.%26aulast%3DMignani%26aufirst%3DS.%26aulast%3DVishwakarma%26aufirst%3DR.%2BA.%26aulast%3DSingh%26aufirst%3DP.%2BP.%26atitle%3DDesign%2520of%2520Novel%25203-Pyrimidinylazaindole%2520CDK2%252F9%2520Inhibitors%2520with%2520Potent%2520In%2520Vitro%2520and%2520In%2520Vivo%2520Antitumor%2520Efficacy%2520in%2520a%2520Triple-Negative%2520Breast%2520Cancer%2520Model%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D9470%26epage%3D9489%26doi%3D10.1021%2Facs.jmedchem.7b00663" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Heathcote, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kroll, S. H. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hazel, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Periyasamy, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alikian, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kanneganti, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jogalekar, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scheiper, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbazanges, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blum, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brackow, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siwicka, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pace, R. D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuchter, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snyder, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liotta, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freemont, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aboagye, E. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coombes, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrett, A. G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ali, S.</span></span> <span> </span><span class="NLM_article-title">A Novel Pyrazolo[1,5-a]pyrimidine Is a Potent Inhibitor of Cyclin-dependent Protein Kinases 1, 2, and 9, Which Demonstrates Antitumor Effects in Human Tumor Xenografts Following Oral Administration</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">8508</span>â <span class="NLM_lpage">8522</span>, <span class="refDoi">Â DOI: 10.1021/jm100732t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm100732t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVeksL%252FJ" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=8508-8522&author=D.+A.+Heathcoteauthor=H.+Patelauthor=S.+H.+B.+Krollauthor=P.+Hazelauthor=M.+Periyasamyauthor=M.+Alikianauthor=S.+K.+Kannegantiauthor=A.+S.+Jogalekarauthor=B.+Scheiperauthor=M.+Barbazangesauthor=A.+Blumauthor=J.+Brackowauthor=A.+Siwickaauthor=R.+D.+M.+Paceauthor=M.+J.+Fuchterauthor=J.+P.+Snyderauthor=D.+C.+Liottaauthor=P.+S.+Freemontauthor=E.+O.+Aboagyeauthor=R.+C.+Coombesauthor=A.+G.+M.+Barrettauthor=S.+Ali&title=A+Novel+Pyrazolo%5B1%2C5-a%5Dpyrimidine+Is+a+Potent+Inhibitor+of+Cyclin-dependent+Protein+Kinases+1%2C+2%2C+and+9%2C+Which+Demonstrates+Antitumor+Effects+in+Human+Tumor+Xenografts+Following+Oral+Administration&doi=10.1021%2Fjm100732t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">A Novel Pyrazolo[1,5-a]pyrimidine is a Potent Inhibitor of Cyclin-Dependent Protein Kinases 1, 2, and 9, Which Demonstrates Antitumor Effects in Human Tumor Xenografts Following Oral Administration</span></div><div class="casAuthors">Heathcote, Dean A.; Patel, Hetal; Kroll, Sebastian H. B.; Hazel, Pascale; Periyasamy, Manikandan; Alikian, Mary; Kanneganti, Seshu K.; Jogalekar, Ashutosh S.; Scheiper, Bodo; Barbazanges, Marion; Blum, Andreas; Brackow, Jan; Siwicka, Alekasandra; Pace, Robert D. M.; Fuchter, Matthew J.; Snyder, James P.; Liotta, Dennis C.; Freemont, Paul. S.; Aboagye, Eric O.; Coombes, R. Charles; Barrett, Anthony G. M.; Ali, Simak</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">8508-8522</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Cyclin-dependent protein kinases (CDKs) are central to the appropriate regulation of cell proliferation, apoptosis, and gene expression.  Abnormalities in CDK activity and regulation are common features of cancer, making CDK family members attractive targets for the development of anticancer drugs.  Here, we report the identification of a pyrazolo[1,5-a]pyrimidine derived compd., 4k (BS-194, I), as a selective and potent CDK inhibitor, which inhibits CDK2, CDK1, CDK5, CDK7, and CDK9 (IC50 = 3, 30, 30, 250, and 90 nmol/L, resp.).  Cell-based studies showed inhibition of the phosphorylation of CDK substrates, Rb and the RNA polymerase II C-terminal domain, down-regulation of cyclins A, E, and D1, and cell cycle block in the S and G2/M phases.  Consistent with these findings, 4k demonstrated potent antiproliferative activity in 60 cancer cell lines tested (mean GI50 = 280 nmol/L).  Pharmacokinetic studies showed that 4k is orally bioavailable, with an elimination half-life of 178 min following oral dosing in mice.  When administered at a concn. of 25 mg/kg orally, 4k inhibited human tumor xenografts and suppressed CDK substrate phosphorylation.  These findings identify 4k as a novel, potent CDK selective inhibitor with potential for oral delivery in cancer patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpUJPhLKEWwBLVg90H21EOLACvtfcHk0li5aseck83oCQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVeksL%252FJ&md5=b84f41ab0cbf298a9ac248bc0a9d0d73</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Fjm100732t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm100732t%26sid%3Dliteratum%253Aachs%26aulast%3DHeathcote%26aufirst%3DD.%2BA.%26aulast%3DPatel%26aufirst%3DH.%26aulast%3DKroll%26aufirst%3DS.%2BH.%2BB.%26aulast%3DHazel%26aufirst%3DP.%26aulast%3DPeriyasamy%26aufirst%3DM.%26aulast%3DAlikian%26aufirst%3DM.%26aulast%3DKanneganti%26aufirst%3DS.%2BK.%26aulast%3DJogalekar%26aufirst%3DA.%2BS.%26aulast%3DScheiper%26aufirst%3DB.%26aulast%3DBarbazanges%26aufirst%3DM.%26aulast%3DBlum%26aufirst%3DA.%26aulast%3DBrackow%26aufirst%3DJ.%26aulast%3DSiwicka%26aufirst%3DA.%26aulast%3DPace%26aufirst%3DR.%2BD.%2BM.%26aulast%3DFuchter%26aufirst%3DM.%2BJ.%26aulast%3DSnyder%26aufirst%3DJ.%2BP.%26aulast%3DLiotta%26aufirst%3DD.%2BC.%26aulast%3DFreemont%26aufirst%3DP.%2BS.%26aulast%3DAboagye%26aufirst%3DE.%2BO.%26aulast%3DCoombes%26aufirst%3DR.%2BC.%26aulast%3DBarrett%26aufirst%3DA.%2BG.%2BM.%26aulast%3DAli%26aufirst%3DS.%26atitle%3DA%2520Novel%2520Pyrazolo%255B1%252C5-a%255Dpyrimidine%2520Is%2520a%2520Potent%2520Inhibitor%2520of%2520Cyclin-dependent%2520Protein%2520Kinases%25201%252C%25202%252C%2520and%25209%252C%2520Which%2520Demonstrates%2520Antitumor%2520Effects%2520in%2520Human%2520Tumor%2520Xenografts%2520Following%2520Oral%2520Administration%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D8508%26epage%3D8522%26doi%3D10.1021%2Fjm100732t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nie, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, R.</span></span> <span> </span><span class="NLM_article-title">Discovery of N1-(4-((7-Cyclopentyl-6-(dimethylcarbamoyl)-7 H-pyrrolo[2,3-d]pyrimidin-2-yl)amino)phenyl)- N<sup>8</sup>-hydroxyoctanediamide as a Novel Inhibitor Targeting Cyclin-dependent Kinase 4/9 (CDK4/9) and Histone Deacetlyase1 (HDAC1) Against Malignant Cancer</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">3166</span>â <span class="NLM_lpage">3192</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.8b00209</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00209" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=1%3ACAS%3A528%3ADC%252BC1cXktFWgs7c%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=3166-3192&author=Y.+Liauthor=X.+Luoauthor=Q.+Guoauthor=Y.+Nieauthor=T.+Wangauthor=C.+Zhangauthor=Z.+Huangauthor=X.+Wangauthor=Y.+Liuauthor=Y.+Chenauthor=J.+Zhengauthor=S.+Yangauthor=Y.+Fanauthor=R.+Xiang&title=Discovery+of+N1-%284-%28%287-Cyclopentyl-6-%28dimethylcarbamoyl%29-7+H-pyrrolo%5B2%2C3-d%5Dpyrimidin-2-yl%29amino%29phenyl%29-+N8-hydroxyoctanediamide+as+a+Novel+Inhibitor+Targeting+Cyclin-dependent+Kinase+4%2F9+%28CDK4%2F9%29+and+Histone+Deacetlyase1+%28HDAC1%29+Against+Malignant+Cancer&doi=10.1021%2Facs.jmedchem.8b00209"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of N1-(4-((7-Cyclopentyl-6-(dimethylcarbamoyl)-7H-pyrrolo[2,3-d]pyrimidin-2-yl)amino)phenyl)-N8-hydroxyoctanediamide as a Novel Inhibitor Targeting Cyclin-dependent Kinase 4/9 (CDK4/9) and Histone Deacetylase 1 (HDAC1) against Malignant Cancer</span></div><div class="casAuthors">Li, Yongtao; Luo, Xiaohe; Guo, Qingxiang; Nie, Yongwei; Wang, Tianqi; Zhang, Chao; Huang, Zhi; Wang, Xin; Liu, Yanhua; Chen, Yanan; Zheng, Jianyu; Yang, Shengyong; Fan, Yan; Xiang, Rong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3166-3192</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A series of highly potent, selective inhibitors targeting both CDK4/9 and HDAC1 have been designed and synthesized.  N1-(4-((7-Cyclopentyl-6-(dimethylcarbamoyl)-7H-pyrrolo[2,3-d]pyrimidin-2-yl)amino)phenyl)-N8-hydroxyoctanediamide (I) was discovered.  The lead compd. I with excellent CDK4/9 and HDAC1 inhibitory activity of IC50 = 8.8, 12, and 2.2 nM, resp., can effectively induce apoptosis of cancer cell lines.  The kinase profiling of compd. I showed excellent selectivity and specificity.  Compd. I induces G2/M arrest in high concn. and G0/G1 arrest in low concn. to prevent the proliferation and differentiation of cancer cells.  Mice bared-breast cancer treated with I showed significant antitumor efficacy.  The insight into mechanisms of I indicated that it could induce cancer cell death via cell apoptosis based on CDK4/9 and HDAC1 repression and phosphorylation of p53.  The data demonstrated the compd. I could be a promising drug candidate for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4B7BpJ-foDbVg90H21EOLACvtfcHk0li5aseck83oCQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXktFWgs7c%253D&md5=b1ca389e67a3ccd3fe5c005cb547e369</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00209&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00209%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DY.%26aulast%3DLuo%26aufirst%3DX.%26aulast%3DGuo%26aufirst%3DQ.%26aulast%3DNie%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DT.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DHuang%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DZheng%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DS.%26aulast%3DFan%26aufirst%3DY.%26aulast%3DXiang%26aufirst%3DR.%26atitle%3DDiscovery%2520of%2520N1-%25284-%2528%25287-Cyclopentyl-6-%2528dimethylcarbamoyl%2529-7%2520H-pyrrolo%255B2%252C3-d%255Dpyrimidin-2-yl%2529amino%2529phenyl%2529-%2520N8-hydroxyoctanediamide%2520as%2520a%2520Novel%2520Inhibitor%2520Targeting%2520Cyclin-dependent%2520Kinase%25204%252F9%2520%2528CDK4%252F9%2529%2520and%2520Histone%2520Deacetlyase1%2520%2528HDAC1%2529%2520Against%2520Malignant%2520Cancer%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D3166%26epage%3D3192%26doi%3D10.1021%2Facs.jmedchem.8b00209" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tsou, H.-R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Otteng, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tran, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Floyd, M. B.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reich, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Birnberg, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kutterer, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ayral-Kaloustian, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ravi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nilakantan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grillo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGinnis, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rabindran, S. K.</span></span> <span> </span><span class="NLM_article-title">4-(phenylaminomethylene)isoquinoline-1,3(2H,4H)-diones as Potent and Selective Inhibitors of the Cyclin-dependent Kinase 4 (CDK4)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">3507</span>â <span class="NLM_lpage">3525</span>, <span class="refDoi">Â DOI: 10.1021/jm800072z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm800072z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=1%3ACAS%3A528%3ADC%252BD1cXmtF2rtrc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=3507-3525&author=H.-R.+Tsouauthor=M.+Ottengauthor=T.+Tranauthor=M.+B.+Floydauthor=M.+Reichauthor=G.+Birnbergauthor=K.+Kuttererauthor=S.+Ayral-Kaloustianauthor=M.+Raviauthor=R.+Nilakantanauthor=M.+Grilloauthor=J.+P.+McGinnisauthor=S.+K.+Rabindran&title=4-%28phenylaminomethylene%29isoquinoline-1%2C3%282H%2C4H%29-diones+as+Potent+and+Selective+Inhibitors+of+the+Cyclin-dependent+Kinase+4+%28CDK4%29&doi=10.1021%2Fjm800072z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">4-(Phenylaminomethylene)isoquinoline-1,3(2H,4H)-diones as Potent and Selective Inhibitors of the Cyclin-Dependent Kinase 4 (CDK4)</span></div><div class="casAuthors">Tsou, Hwei-Ru; Otteng, Mercy; Tran, Tritin; Floyd, M. Brawner, Jr.; Reich, Marvin; Birnberg, Gary; Kutterer, Kristina; Ayral-Kaloustian, Semiramis; Ravi, Malini; Nilakantan, Ramaswamy; Grillo, Mary; McGinnis, John P.; Rabindran, Sridhar K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3507-3525</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The cyclin-dependent kinases (CDKs), as complexes with their resp. partners, the cyclins, are crit. regulators of cell cycle progression.  Because aberrant regulations of CDK4/cyclin D1 lead to uncontrolled cell proliferation, a hallmark of cancer, small-mol. inhibitors of CDK4/cyclin D1 are attractive as prospective antitumor agents.  The series of 4-(phenylaminomethylene)isoquinoline-1,3(2H,4H)-dione derivs. reported here represents a novel class of potent inhibitors that selectively inhibit CDK4 over CDK2 and CDK1 activities.  In the headpiece of the 4-(phenylaminomethylene)isoquinoline-1,3(2H,4H)-dione, a basic amine substitutent is required on the aniline ring for the CDK4 inhibitory activity.  The inhibitory activity is further enhanced when an aryl or heteroaryl substituent is introduced at the C-6 position of the isoquinoline-1,3(2H,4H)-dione core.  We present here SAR data and a CDK4 mimic model that explains the binding, potency, and selectivity of our CDK4 selective inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq09yU47pubWbVg90H21EOLACvtfcHk0lhXNma2VLdVVg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXmtF2rtrc%253D&md5=35083ce22416a584311a810ecf2b624d</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Fjm800072z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm800072z%26sid%3Dliteratum%253Aachs%26aulast%3DTsou%26aufirst%3DH.-R.%26aulast%3DOtteng%26aufirst%3DM.%26aulast%3DTran%26aufirst%3DT.%26aulast%3DFloyd%26aufirst%3DM.%2BB.%26aulast%3DReich%26aufirst%3DM.%26aulast%3DBirnberg%26aufirst%3DG.%26aulast%3DKutterer%26aufirst%3DK.%26aulast%3DAyral-Kaloustian%26aufirst%3DS.%26aulast%3DRavi%26aufirst%3DM.%26aulast%3DNilakantan%26aufirst%3DR.%26aulast%3DGrillo%26aufirst%3DM.%26aulast%3DMcGinnis%26aufirst%3DJ.%2BP.%26aulast%3DRabindran%26aufirst%3DS.%2BK.%26atitle%3D4-%2528phenylaminomethylene%2529isoquinoline-1%252C3%25282H%252C4H%2529-diones%2520as%2520Potent%2520and%2520Selective%2520Inhibitors%2520of%2520the%2520Cyclin-dependent%2520Kinase%25204%2520%2528CDK4%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D3507%26epage%3D3525%26doi%3D10.1021%2Fjm800072z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Del
Re, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertolini, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crucitta, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fontanelli, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rofi, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Angelis, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diodati, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cavallero, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gianfilippo, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salvadori, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fogli, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Falcone, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scatena, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naccarato, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roncella, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghilli, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morganti, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fontana, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Danesi, R.</span></span> <span> </span><span class="NLM_article-title">Overexpression of TK1 and CDK9 in Plasma-derived Exosomes is Associated with Clinical Resistance to CDK4/6 Inhibitors in Metastatic Breast Cancer Patients</span>. <i>Breast Canc. Res. Treat.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>178</i></span>,  <span class="NLM_fpage">57</span>â <span class="NLM_lpage">62</span>, <span class="refDoi">Â DOI: 10.1007/s10549-019-05365-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=10.1007%2Fs10549-019-05365-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=31346846" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsVGrtbzM" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=178&publication_year=2019&pages=57-62&author=M.+Del%0AReauthor=I.+Bertoliniauthor=S.+Crucittaauthor=L.+Fontanelliauthor=E.+Rofiauthor=C.+De+Angelisauthor=L.+Diodatiauthor=D.+Cavalleroauthor=G.+Gianfilippoauthor=B.+Salvadoriauthor=S.+Fogliauthor=A.+Falconeauthor=C.+Scatenaauthor=A.+G.+Naccaratoauthor=M.+Roncellaauthor=M.+Ghilliauthor=R.+Morgantiauthor=A.+Fontanaauthor=R.+Danesi&title=Overexpression+of+TK1+and+CDK9+in+Plasma-derived+Exosomes+is+Associated+with+Clinical+Resistance+to+CDK4%2F6+Inhibitors+in+Metastatic+Breast+Cancer+Patients&doi=10.1007%2Fs10549-019-05365-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Overexpression of TK1 and CDK9 in plasma-derived exosomes is associated with clinical resistance to CDK4/6 inhibitors in metastatic breast cancer patients</span></div><div class="casAuthors">Del Re, Marzia; Bertolini, Ilaria; Crucitta, Stefania; Fontanelli, Lorenzo; Rofi, Eleonora; De Angelis, Claudia; Diodati, Lucrezia; Cavallero, Diletta; Gianfilippo, Giulia; Salvadori, Barbara; Fogli, Stefano; Falcone, Alfredo; Scatena, Cristian; Naccarato, Antonio Giuseppe; Roncella, Manuela; Ghilli, Matteo; Morganti, Riccardo; Fontana, Andrea; Danesi, Romano</div><div class="citationInfo"><span class="NLM_cas:title">Breast Cancer Research and Treatment</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">178</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">57-62</span>CODEN:
                <span class="NLM_cas:coden">BCTRD6</span>;
        ISSN:<span class="NLM_cas:issn">0167-6806</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">The present article addresses the role of the mRNA expression of thymidine kinase 1 (TK1), CDK4, 6 and 9 in plasma-derived exosomes and their relevance in the pharmacol. activity of CDK4/6i.  Methods: Blood samples of 40 HR+/HER2- advanced breast cancer patients were collected before (T0) the administration of palbociclib plus hormonal therapy and after 3 mo (T1).  RNA was isolated from exosomes and analyzed for the expression of TK1, CDK 4, 6 and 9 by digital droplet PCR (ddPCR).  Results: A higher value of TK1 copies/mL at baseline (T0) was significantly assocd. with the no. of previous lines of chemotherapy (p = 0.009).  In patients with PD, a significant increase was obsd. in the no. of copies/mL of TK1 (p = 0.01) and CDK9 (p = 0.03) comparing T1 vs.  T0 values. No significant correlations between response to treatment and clin. parameters were found at univariate anal.  High baseline CDK4 expression was significantly correlated with longer PFS in patients treated with fulvestrant + palbociclib (low vs. high: 6.45 mo vs. not reached, p = 0.01).  Conclusions: The present study demonstrates that, in plasma-derived exosomes, high baseline CDK4 mRNA levels are assocd. with response to palbociclib plus hormonal therapy, while the increase in TK1 and CDK9 mRNA copies/mL is assocd. with clin. resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpAoJIlNIJ7NLVg90H21EOLACvtfcHk0lhXNma2VLdVVg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsVGrtbzM&md5=e5684c7965c4527fe92c7c5c9869e428</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1007%2Fs10549-019-05365-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10549-019-05365-y%26sid%3Dliteratum%253Aachs%26aulast%3DDel%2BRe%26aufirst%3DM.%26aulast%3DBertolini%26aufirst%3DI.%26aulast%3DCrucitta%26aufirst%3DS.%26aulast%3DFontanelli%26aufirst%3DL.%26aulast%3DRofi%26aufirst%3DE.%26aulast%3DDe%2BAngelis%26aufirst%3DC.%26aulast%3DDiodati%26aufirst%3DL.%26aulast%3DCavallero%26aufirst%3DD.%26aulast%3DGianfilippo%26aufirst%3DG.%26aulast%3DSalvadori%26aufirst%3DB.%26aulast%3DFogli%26aufirst%3DS.%26aulast%3DFalcone%26aufirst%3DA.%26aulast%3DScatena%26aufirst%3DC.%26aulast%3DNaccarato%26aufirst%3DA.%2BG.%26aulast%3DRoncella%26aufirst%3DM.%26aulast%3DGhilli%26aufirst%3DM.%26aulast%3DMorganti%26aufirst%3DR.%26aulast%3DFontana%26aufirst%3DA.%26aulast%3DDanesi%26aufirst%3DR.%26atitle%3DOverexpression%2520of%2520TK1%2520and%2520CDK9%2520in%2520Plasma-derived%2520Exosomes%2520is%2520Associated%2520with%2520Clinical%2520Resistance%2520to%2520CDK4%252F6%2520Inhibitors%2520in%2520Metastatic%2520Breast%2520Cancer%2520Patients%26jtitle%3DBreast%2520Canc.%2520Res.%2520Treat.%26date%3D2019%26volume%3D178%26spage%3D57%26epage%3D62%26doi%3D10.1007%2Fs10549-019-05365-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sengupta, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biarnes, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jordan, V. C.</span></span> <span> </span><span class="NLM_article-title">Cyclin Dependent Kinase-9 Mediated Transcriptional De-regulation of cMYC as a Critical Determinant of Endocrine-therapy Resistance in Breast cancers</span>. <i>Breast Canc. Res. Treat.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>143</i></span>,  <span class="NLM_fpage">113</span>â <span class="NLM_lpage">124</span>, <span class="refDoi">Â DOI: 10.1007/s10549-013-2789-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=10.1007%2Fs10549-013-2789-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=24309997" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFKgu7jN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=143&publication_year=2014&pages=113-124&author=S.+Senguptaauthor=M.+C.+Biarnesauthor=V.+C.+Jordan&title=Cyclin+Dependent+Kinase-9+Mediated+Transcriptional+De-regulation+of+cMYC+as+a+Critical+Determinant+of+Endocrine-therapy+Resistance+in+Breast+cancers&doi=10.1007%2Fs10549-013-2789-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Cyclin dependent kinase-9 mediated transcriptional de-regulation of cMYC as a critical determinant of endocrine-therapy resistance in breast cancers</span></div><div class="casAuthors">Sengupta, Surojeet; Biarnes, Michael C.; Jordan, V. Craig</div><div class="citationInfo"><span class="NLM_cas:title">Breast Cancer Research and Treatment</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">143</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">113-124</span>CODEN:
                <span class="NLM_cas:coden">BCTRD6</span>;
        ISSN:<span class="NLM_cas:issn">0167-6806</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Endocrine therapy resistance in estrogen receptor alpha pos. (ERÎ±+) breast cancers remains a major obstacle for maintaining efficacy of targeted therapies.  We investigated the significance and the mechanisms involved in cMYC over-expression in a MCF7 derived panel of ERÎ±+ breast cancer cells which can proliferate in the absence of estrogen with different sensitivities to anti-hormone therapies.  We show that all the resistant cell lines tested over-express cMYC as compared to parental MCF7 cells and its inhibition lead to the differential blocking of estrogen-independent proliferation in resistant cells.  Further investigation of the resistant cell line, MCF7:5C, suggested transcriptional de-regulation of cMYC gene was responsible for its over-expression.  Chromatin immuno-pptn. assay revealed markedly higher recruitment of phosphorylated serine-2 carboxy-terminal domain (CTD) of RNA polymerase-II at the proximal promoter of cMYC gene, which is responsible for transcriptional elongation of the cMYC RNA.  The level of CDK9, a factor responsible for the phosphorylation of serine-2 of RNA polymerase II CTD, was found to be elevated in all the resistant cell lines.  Pharmacol. inhibition of CDK9 not only reduced the transcripts and the protein levels of cMYC in MCF7:5C cells but also selectively inhibited the estrogen-independent growth of all the resistant cell lines.  This study describes the up-stream mol. events involved in the transcriptional over-expression of cMYC gene in breast cancer cells proliferating estrogen-independently and identifies CDK9 as a potential novel drug target for therapeutic intervention in endocrine-resistant breast cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoY3q-BuILrXrVg90H21EOLACvtfcHk0ljozq7IBaF3xg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFKgu7jN&md5=9d79fb5d2cab2f59dade67c2f6ac7747</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1007%2Fs10549-013-2789-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10549-013-2789-2%26sid%3Dliteratum%253Aachs%26aulast%3DSengupta%26aufirst%3DS.%26aulast%3DBiarnes%26aufirst%3DM.%2BC.%26aulast%3DJordan%26aufirst%3DV.%2BC.%26atitle%3DCyclin%2520Dependent%2520Kinase-9%2520Mediated%2520Transcriptional%2520De-regulation%2520of%2520cMYC%2520as%2520a%2520Critical%2520Determinant%2520of%2520Endocrine-therapy%2520Resistance%2520in%2520Breast%2520cancers%26jtitle%3DBreast%2520Canc.%2520Res.%2520Treat.%26date%3D2014%26volume%3D143%26spage%3D113%26epage%3D124%26doi%3D10.1007%2Fs10549-013-2789-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeidner, J. F.</span></span> <span> </span><span class="NLM_article-title">Cyclin-dependent Kinase (CDK) 9 and 4/6 Inhibitors in Acute Myeloid Leukemia (AML): a Promising Therapeutic Approach</span>. <i>Expet Opin. Invest. Drugs</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">989</span>â <span class="NLM_lpage">1001</span>, <span class="refDoi">Â DOI: 10.1080/13543784.2019.1678583</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=10.1080%2F13543784.2019.1678583" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=31612739" title="">PubMed</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2019&pages=989-1001&author=D.+J.+Leeauthor=J.+F.+Zeidner&title=Cyclin-dependent+Kinase+%28CDK%29+9+and+4%2F6+Inhibitors+in+Acute+Myeloid+Leukemia+%28AML%29%3A+a+Promising+Therapeutic+Approach&doi=10.1080%2F13543784.2019.1678583"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1080%2F13543784.2019.1678583&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F13543784.2019.1678583%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DD.%2BJ.%26aulast%3DZeidner%26aufirst%3DJ.%2BF.%26atitle%3DCyclin-dependent%2520Kinase%2520%2528CDK%2529%25209%2520and%25204%252F6%2520Inhibitors%2520in%2520Acute%2520Myeloid%2520Leukemia%2520%2528AML%2529%253A%2520a%2520Promising%2520Therapeutic%2520Approach%26jtitle%3DExpet%2520Opin.%2520Invest.%2520Drugs%26date%3D2019%26volume%3D28%26spage%3D989%26epage%3D1001%26doi%3D10.1080%2F13543784.2019.1678583" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dean, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hornicek, F. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duan, Z.</span></span> <span> </span><span class="NLM_article-title">Cyclin-dependent kinase 9 (CDK9) is a Novel Prognostic Marker and Therapeutic Target in Ovarian Cancer</span>. <i>FASEB J.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">5990</span>â <span class="NLM_lpage">6000</span>, <span class="refDoi">Â DOI: 10.1096/fj.201801789rr</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=10.1096%2Ffj.201801789RR" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=30726104" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtFChs7zO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2019&pages=5990-6000&author=J.+Wangauthor=D.+C.+Deanauthor=F.+J.+Hornicekauthor=H.+Shiauthor=Z.+Duan&title=Cyclin-dependent+kinase+9+%28CDK9%29+is+a+Novel+Prognostic+Marker+and+Therapeutic+Target+in+Ovarian+Cancer&doi=10.1096%2Ffj.201801789rr"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Cyclin-dependent kinase 9 (CDK9) is a novel prognostic marker and therapeutic target in ovarian cancer</span></div><div class="casAuthors">Wang, Jinglu; Dean, Dylan C.; Hornicek, Francis J.; Shi, Huirong; Duan, Zhenfeng</div><div class="citationInfo"><span class="NLM_cas:title">FASEB Journal</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">5990-6000</span>CODEN:
                <span class="NLM_cas:coden">FAJOEC</span>;
        ISSN:<span class="NLM_cas:issn">0892-6638</span>.
    
            (<span class="NLM_cas:orgname">Federation of American Societies for Experimental Biology</span>)
        </div><div class="casAbstract">Despite surgical and chemotherapeutic advances over the past few decades, the prognosis for ovarian cancer remains very poor.  Although cyclin-dependent kinase (CDK) 9 has an established pathogenic role in various cancers, its function in ovarian cancer remains poorly defined.  The purpose of this study was to evaluate the expression of CDK9 and its therapeutic potential in ovarian cancer.  CDK9 expression was detd. by immunohistochem. in a unique ovarian cancer tissue microarray constructed with paired primary, metastatic, and recurrent tumor tissues from 26 ovarian cancer patients.  CDK9 was highly expressed in human ovarian cancer cell lines and was also elevated in metastatic and recurrent ovarian tumor tissue compared with patient-matched primary ovarian tumor tissue.  In addn., increased CDK9 significantly correlated with poor patient prognosis.  Inhibition of CDK9 by small interfering RNA or CDK9 inhibitor functionally suppressed RNA transcription elongation, induced apoptosis, and reduced proliferation of ovarian cancer cells.  Inhibition of CDK9 also suppressed ovarian cancer cell spheroid growth, clonogenicity formation, and migration activity.  Our results reveal CDK9 as a novel prognostic biomarker and a promising therapeutic target for preventing metastasis and recurrence while also improving the overall clin. outcome for ovarian cancer patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTuSeIUOKO-LVg90H21EOLACvtfcHk0ljozq7IBaF3xg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtFChs7zO&md5=cc691e71c659a48477b30130ccf6cd24</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1096%2Ffj.201801789RR&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1096%252Ffj.201801789RR%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DDean%26aufirst%3DD.%2BC.%26aulast%3DHornicek%26aufirst%3DF.%2BJ.%26aulast%3DShi%26aufirst%3DH.%26aulast%3DDuan%26aufirst%3DZ.%26atitle%3DCyclin-dependent%2520kinase%25209%2520%2528CDK9%2529%2520is%2520a%2520Novel%2520Prognostic%2520Marker%2520and%2520Therapeutic%2520Target%2520in%2520Ovarian%2520Cancer%26jtitle%3DFASEB%2520J.%26date%3D2019%26volume%3D33%26spage%3D5990%26epage%3D6000%26doi%3D10.1096%2Ffj.201801789rr" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, N. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferre, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergqvist, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Solowiej, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diehl, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, Y.-A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagata, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">VanArsdale, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, B. W.</span></span> <span> </span><span class="NLM_article-title">Spectrum and Degree of CDK Drug Interactions Predicts Clinical Performance</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">2273</span>â <span class="NLM_lpage">2281</span>, <span class="refDoi">Â DOI: 10.1158/1535-7163.mct-16-0300</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=10.1158%2F1535-7163.MCT-16-0300" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=27496135" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=1%3ACAS%3A528%3ADC%252BC28XhslGisbjK" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=2273-2281&author=P.+Chenauthor=N.+V.+Leeauthor=W.+Huauthor=M.+Xuauthor=R.+A.+Ferreauthor=H.+Lamauthor=S.+Bergqvistauthor=J.+Solowiejauthor=W.+Diehlauthor=Y.-A.+Heauthor=X.+Yuauthor=A.+Nagataauthor=T.+VanArsdaleauthor=B.+W.+Murray&title=Spectrum+and+Degree+of+CDK+Drug+Interactions+Predicts+Clinical+Performance&doi=10.1158%2F1535-7163.mct-16-0300"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Spectrum and Degree of CDK Drug Interactions Predicts Clinical Performance</span></div><div class="casAuthors">Chen, Ping; Lee, Nathan V.; Hu, Wenyue; Xu, Meirong; Ferre, Rose Ann; Lam, Hieu; Bergqvist, Simon; Solowiej, James; Diehl, Wade; He, You-Ai; Yu, Xiu; Nagata, Asako; Van Arsdale, Todd; Murray, Brion W.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2273-2281</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Therapeutically targeting aberrant intracellular kinase signaling is attractive from a biol. perspective but drug development is often hindered by toxicities and inadequate efficacy.  Predicting drug behaviors using cellular and animal models is confounded by redundant kinase activities, a lack of unique substrates, and cell-specific signaling networks.  Cyclin-dependent kinase (CDK) drugs exemplify this phenomenon because they are reported to target common processes yet have distinct clin. activities.  Tumor cell studies of ATP-competitive CDK drugs (dinaciclib, AG-024322, abemaciclib, palbociclib, ribociclib) indicate similar pharmacol. while analyses in untransformed cells illuminates significant differences.  To resolve this apparent disconnect, drug behaviors are described at the mol. level.  Nonkinase binding studies and kinome interaction anal. (recombinant and endogenous kinases) reveal that proteins outside of the CDK family appear to have little role in dinaciclib/palbociclib/ribociclib pharmacol., may contribute for abemaciclib, and confounds AG-024322 anal.  CDK2 and CDK6 cocrystal structures with the drugs identify the mol. interactions responsible for potency and kinase selectivity.  Efficient drug binding to the unique hinge architecture of CDKs enables selectivity toward most of the human kinome.  Selectivity between CDK family members is achieved through interactions with nonconserved elements of the ATP-binding pocket.  Integrating clin. drug exposures into the anal. predicts that both palbociclib and ribociclib are CDK4/6 inhibitors, abemaciclib inhibits CDK4/6/9, and dinaciclib is a broad-spectrum CDK inhibitor (CDK2/3/4/6/9).  Understanding the mol. components of potency and selectivity also facilitates rational design of future generations of kinase-directed drugs.  Mol Cancer Ther; 15(10); 2273-81.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3TbbI98T8v7Vg90H21EOLACvtfcHk0lh0acIiBDI5iQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhslGisbjK&md5=112e9e46bb33393516fa94620e6383c6</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-16-0300&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-16-0300%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DP.%26aulast%3DLee%26aufirst%3DN.%2BV.%26aulast%3DHu%26aufirst%3DW.%26aulast%3DXu%26aufirst%3DM.%26aulast%3DFerre%26aufirst%3DR.%2BA.%26aulast%3DLam%26aufirst%3DH.%26aulast%3DBergqvist%26aufirst%3DS.%26aulast%3DSolowiej%26aufirst%3DJ.%26aulast%3DDiehl%26aufirst%3DW.%26aulast%3DHe%26aufirst%3DY.-A.%26aulast%3DYu%26aufirst%3DX.%26aulast%3DNagata%26aufirst%3DA.%26aulast%3DVanArsdale%26aufirst%3DT.%26aulast%3DMurray%26aufirst%3DB.%2BW.%26atitle%3DSpectrum%2520and%2520Degree%2520of%2520CDK%2520Drug%2520Interactions%2520Predicts%2520Clinical%2520Performance%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2016%26volume%3D15%26spage%3D2273%26epage%3D2281%26doi%3D10.1158%2F1535-7163.mct-16-0300" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Minzel, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venkatachalam, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fink, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brachya, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burstain, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaham, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rivlin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Omer, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zinger, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elias, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winter, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erdman, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sullivan, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fung, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mercurio, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vacca, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaushansky, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shlush, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oren, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levine, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pikarsky, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snir-Alkalay, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ben-Neriah, Y.</span></span> <span> </span><span class="NLM_article-title">Small Molecules Co-targeting CKIalpha and the Transcriptional Kinases CDK7/9 Control AML in Preclinical Models</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>175</i></span>,  <span class="NLM_fpage">171</span>â <span class="NLM_lpage">185</span>, <span class="refDoi">Â DOI: 10.1016/j.cell.2018.07.045</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=10.1016%2Fj.cell.2018.07.045" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=30146162" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsFyktbvF" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=175&publication_year=2018&pages=171-185&author=W.+Minzelauthor=A.+Venkatachalamauthor=A.+Finkauthor=E.+Hungauthor=G.+Brachyaauthor=I.+Burstainauthor=M.+Shahamauthor=A.+Rivlinauthor=I.+Omerauthor=A.+Zingerauthor=S.+Eliasauthor=E.+Winterauthor=P.+E.+Erdmanauthor=R.+W.+Sullivanauthor=L.+Fungauthor=F.+Mercurioauthor=D.+Liauthor=J.+Vaccaauthor=N.+Kaushanskyauthor=L.+Shlushauthor=M.+Orenauthor=R.+Levineauthor=E.+Pikarskyauthor=I.+Snir-Alkalayauthor=Y.+Ben-Neriah&title=Small+Molecules+Co-targeting+CKIalpha+and+the+Transcriptional+Kinases+CDK7%2F9+Control+AML+in+Preclinical+Models&doi=10.1016%2Fj.cell.2018.07.045"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Small Molecules Co-targeting CKIÎ± and the Transcriptional Kinases CDK7/9 Control AML in Preclinical Models</span></div><div class="casAuthors">Minzel, Waleed; Venkatachalam, Avanthika; Fink, Avner; Hung, Eric; Brachya, Guy; Burstain, Ido; Shaham, Maya; Rivlin, Amitai; Omer, Itay; Zinger, Adar; Elias, Shlomo; Winter, Eitan; Erdman, Paul E.; Sullivan, Robert W.; Fung, Leah; Mercurio, Frank; Li, Dansu; Vacca, Joseph; Kaushansky, Nathali; Shlush, Liran; Oren, Moshe; Levine, Ross; Pikarsky, Eli; Snir-Alkalay, Irit; Ben-Neriah, Yinon</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">175</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">171-185.e25</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">CKIÎ± ablation induces p53 activation, and CKIÎ± degrdn. underlies the therapeutic effect of lenalidomide in a pre-leukemia syndrome.  Here we describe the development of CKIÎ± inhibitors, which co-target the transcriptional kinases CDK7 and CDK9, thereby augmenting CKIÎ±-induced p53 activation and its anti-leukemic activity.  Oncogene-driving super-enhancers (SEs) are highly sensitive to CDK7/9 inhibition.  We identified multiple newly gained SEs in primary mouse acute myeloid leukemia (AML) cells and demonstrate that the inhibitors abolish many SEs and preferentially suppress the transcription elongation of SE-driven oncogenes.  We show that blocking CKIÎ± together with CDK7 and/or CDK9 synergistically stabilize p53, deprive leukemia cells of survival and proliferation-maintaining SE-driven oncogenes, and induce apoptosis.  Leukemia progenitors are selectively eliminated by the inhibitors, explaining their therapeutic efficacy with preserved hematopoiesis and leukemia cure potential; they eradicate leukemia in MLL-AF9 and Tet2-/-;Flt3ITD AML mouse models and in several patient-derived AML xenograft models, supporting their potential efficacy in curing human leukemia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqg4QEH6R_ll7Vg90H21EOLACvtfcHk0lh0acIiBDI5iQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsFyktbvF&md5=b3e088a6b262dc8998ff334867102aa5</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2018.07.045&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2018.07.045%26sid%3Dliteratum%253Aachs%26aulast%3DMinzel%26aufirst%3DW.%26aulast%3DVenkatachalam%26aufirst%3DA.%26aulast%3DFink%26aufirst%3DA.%26aulast%3DHung%26aufirst%3DE.%26aulast%3DBrachya%26aufirst%3DG.%26aulast%3DBurstain%26aufirst%3DI.%26aulast%3DShaham%26aufirst%3DM.%26aulast%3DRivlin%26aufirst%3DA.%26aulast%3DOmer%26aufirst%3DI.%26aulast%3DZinger%26aufirst%3DA.%26aulast%3DElias%26aufirst%3DS.%26aulast%3DWinter%26aufirst%3DE.%26aulast%3DErdman%26aufirst%3DP.%2BE.%26aulast%3DSullivan%26aufirst%3DR.%2BW.%26aulast%3DFung%26aufirst%3DL.%26aulast%3DMercurio%26aufirst%3DF.%26aulast%3DLi%26aufirst%3DD.%26aulast%3DVacca%26aufirst%3DJ.%26aulast%3DKaushansky%26aufirst%3DN.%26aulast%3DShlush%26aufirst%3DL.%26aulast%3DOren%26aufirst%3DM.%26aulast%3DLevine%26aufirst%3DR.%26aulast%3DPikarsky%26aufirst%3DE.%26aulast%3DSnir-Alkalay%26aufirst%3DI.%26aulast%3DBen-Neriah%26aufirst%3DY.%26atitle%3DSmall%2520Molecules%2520Co-targeting%2520CKIalpha%2520and%2520the%2520Transcriptional%2520Kinases%2520CDK7%252F9%2520Control%2520AML%2520in%2520Preclinical%2520Models%26jtitle%3DCell%26date%3D2018%26volume%3D175%26spage%3D171%26epage%3D185%26doi%3D10.1016%2Fj.cell.2018.07.045" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicolay, B. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chick, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suski, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keibler, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sicinska, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerdemann, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haining, W. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polyak, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gygi, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dyson, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sicinski, P.</span></span> <span> </span><span class="NLM_article-title">The Metabolic Function of Cyclin D3-CDK6 Kinase in Cancer Cell Survival</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>546</i></span>,  <span class="NLM_fpage">426</span>â <span class="NLM_lpage">430</span>, <span class="refDoi">Â DOI: 10.1038/nature22797</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=10.1038%2Fnature22797" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=28607489" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=1%3ACAS%3A528%3ADC%252BC2sXpt1Oqtb0%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=546&publication_year=2017&pages=426-430&author=H.+Wangauthor=B.+N.+Nicolayauthor=J.+M.+Chickauthor=X.+Gaoauthor=Y.+Gengauthor=H.+Renauthor=H.+Gaoauthor=G.+Yangauthor=J.+A.+Williamsauthor=J.+M.+Suskiauthor=M.+A.+Keiblerauthor=E.+Sicinskaauthor=U.+Gerdemannauthor=W.+N.+Hainingauthor=T.+M.+Robertsauthor=K.+Polyakauthor=S.+P.+Gygiauthor=N.+J.+Dysonauthor=P.+Sicinski&title=The+Metabolic+Function+of+Cyclin+D3-CDK6+Kinase+in+Cancer+Cell+Survival&doi=10.1038%2Fnature22797"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">The metabolic function of cyclin D3-CDK6 kinase in cancer cell survival</span></div><div class="casAuthors">Wang, Haizhen; Nicolay, Brandon N.; Chick, Joel M.; Gao, Xueliang; Geng, Yan; Ren, Hong; Gao, Hui; Yang, Guizhi; Williams, Juliet A.; Suski, Jan M.; Keibler, Mark A.; Sicinska, Ewa; Gerdemann, Ulrike; Haining, W. Nicholas; Roberts, Thomas M.; Polyak, Kornelia; Gygi, Steven P.; Dyson, Nicholas J.; Sicinski, Piotr</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">546</span>
        (<span class="NLM_cas:issue">7658</span>),
    <span class="NLM_cas:pages">426-430</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">D-type cyclins (D1, D2 and D3) and their assocd. cyclin-dependent kinases (CDK4 and CDK6) are components of the core cell cycle machinery that drives cell proliferation.  Inhibitors of CDK4 and CDK6 are currently being tested in clin. trials for patients with several cancer types, with promising results.  Here, using human cancer cells and patient-derived xenografts in mice, we show that the cyclin D3-CDK6 kinase phosphorylates and inhibits the catalytic activity of two key enzymes in the glycolytic pathway, 6-phosphofructokinase and pyruvate kinase M2.  This re-directs the glycolytic intermediates into the pentose phosphate (PPP) and serine pathways.  Inhibition of cyclin D3-CDK6 in tumor cells reduces flow through the PPP and serine pathways, thereby depleting the antioxidants NADPH and glutathione.  This, in turn, increases the levels of reactive oxygen species and causes apoptosis of tumor cells.  The pro-survival function of cyclin D-assocd. kinase operates in tumors expressing high levels of cyclin D3-CDK6 complexes.  We propose that measuring the levels of cyclin D3-CDK6 in human cancers might help to identify tumor subsets that undergo cell death and tumor regression upon inhibition of CDK4 and CDK6.  Cyclin D3-CDK6, through its ability to link cell cycle and cell metab., represents a particularly powerful oncoprotein that affects cancer cells at several levels, and this property can be exploited for anti-cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqD4-8f9Q8KErVg90H21EOLACvtfcHk0lh0acIiBDI5iQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXpt1Oqtb0%253D&md5=f2539399d1c4fb2163fb245d25c07843</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1038%2Fnature22797&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature22797%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DH.%26aulast%3DNicolay%26aufirst%3DB.%2BN.%26aulast%3DChick%26aufirst%3DJ.%2BM.%26aulast%3DGao%26aufirst%3DX.%26aulast%3DGeng%26aufirst%3DY.%26aulast%3DRen%26aufirst%3DH.%26aulast%3DGao%26aufirst%3DH.%26aulast%3DYang%26aufirst%3DG.%26aulast%3DWilliams%26aufirst%3DJ.%2BA.%26aulast%3DSuski%26aufirst%3DJ.%2BM.%26aulast%3DKeibler%26aufirst%3DM.%2BA.%26aulast%3DSicinska%26aufirst%3DE.%26aulast%3DGerdemann%26aufirst%3DU.%26aulast%3DHaining%26aufirst%3DW.%2BN.%26aulast%3DRoberts%26aufirst%3DT.%2BM.%26aulast%3DPolyak%26aufirst%3DK.%26aulast%3DGygi%26aufirst%3DS.%2BP.%26aulast%3DDyson%26aufirst%3DN.%2BJ.%26aulast%3DSicinski%26aufirst%3DP.%26atitle%3DThe%2520Metabolic%2520Function%2520of%2520Cyclin%2520D3-CDK6%2520Kinase%2520in%2520Cancer%2520Cell%2520Survival%26jtitle%3DNature%26date%3D2017%26volume%3D546%26spage%3D426%26epage%3D430%26doi%3D10.1038%2Fnature22797" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sathe, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koshy, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmid, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thalgott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">SchwarzenbÃ¶ck, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krause, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holm, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gschwend, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Retz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nawroth, R.</span></span> <span> </span><span class="NLM_article-title">CDK4/6 Inhibition Controls Proliferation of Bladder Cancer and Transcription of RB1</span>. <i>J. Urol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>195</i></span>,  <span class="NLM_fpage">771</span>â <span class="NLM_lpage">779</span>, <span class="refDoi">Â DOI: 10.1016/j.juro.2015.08.082</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=10.1016%2Fj.juro.2015.08.082" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=26318986" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVOqsr7O" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=195&publication_year=2016&pages=771-779&author=A.+Satheauthor=N.+Koshyauthor=S.+C.+Schmidauthor=M.+Thalgottauthor=S.+M.+Schwarzenb%C3%B6ckauthor=B.+J.+Krauseauthor=P.+S.+Holmauthor=J.+E.+Gschwendauthor=M.+Retzauthor=R.+Nawroth&title=CDK4%2F6+Inhibition+Controls+Proliferation+of+Bladder+Cancer+and+Transcription+of+RB1&doi=10.1016%2Fj.juro.2015.08.082"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">CDK4/6 Inhibition Controls Proliferation of Bladder Cancer and Transcription of RB1</span></div><div class="casAuthors">Sathe, Anuja; Koshy, Nicole; Schmid, Sebastian C.; Thalgott, Mark; Schwarzenboeck, Sarah M.; Krause, Bernd J.; Holm, Per S.; Gschwend, Juergen E.; Retz, Margitta; Nawroth, Roman</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Urology (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">195</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">771-779</span>CODEN:
                <span class="NLM_cas:coden">JOURAA</span>;
        ISSN:<span class="NLM_cas:issn">0022-5347</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">The retinoblastoma signaling network is frequently altered in advanced bladder cancer.  We investigated the potential of CDK4/6 as a therapeutic target and detd. biomarkers for patient stratification.  Genetic alterations were analyzed using public databases, including TCGA (The Cancer Genome Atlas), COSMIC (Catalog of Somatic Mutations in Cancer) and CCLE (Cancer Cell Line Encyclopedia).  Effects of the CDK4/6-inhibitor PD-0332991 or LY2835219 were examd. in 10 bladder cancer cell lines by immunoblot, cell viability, apoptosis and cell cycle progression.  Efficacy of the PD-0332991 and cisplatin combination was analyzed using the combination index.  Gene expression level was detd. by quant. polymerase chain reaction.  Cytomegalovirus promoter regulated recombinant retinoblastoma was used for reconstitution.  Three-dimensional xenografts were grown on chicken chorioallantoic membrane and analyzed by measuring tumor wt. and immunohistochem. expression of total retinoblastoma and Ki-67.  PD-0332991 treatment decreased the proliferation of retinoblastoma pos. bladder cancer cell lines and was synergistic in combination with cisplatin.  PD-0332991 or LY2835219 treatment decreased the phosphorylation, total protein and transcript level of retinoblastoma.  Treatment resulted in a decrease in E2F target gene expression (CCNA2 and CCNE2) and cell cycle progression from G0/G1 to the S-phase but did not affect apoptosis.  In retinoblastoma neg. cells reconstituted with recombinant retinoblastoma PD-0332991 affected only phosphorylation and not the total retinoblastoma level.  These cells remained resistant to treatment.  In 3-dimensional retinoblastoma xenografts, treatment resulted in reduced tumor wt. and decreased expression of total retinoblastoma and Ki-67.  We provide preclin. evidence that CDK4/6 inhibition is a potential therapeutic strategy for retinoblastoma pos. bladder cancer that probably acts by neg. regulating retinoblastoma transcription.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXXQ64fNhPSLVg90H21EOLACvtfcHk0lh3Egl3pCJCFw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVOqsr7O&md5=7b141310df974995067f1fc1a369f676</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2Fj.juro.2015.08.082&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.juro.2015.08.082%26sid%3Dliteratum%253Aachs%26aulast%3DSathe%26aufirst%3DA.%26aulast%3DKoshy%26aufirst%3DN.%26aulast%3DSchmid%26aufirst%3DS.%2BC.%26aulast%3DThalgott%26aufirst%3DM.%26aulast%3DSchwarzenb%25C3%25B6ck%26aufirst%3DS.%2BM.%26aulast%3DKrause%26aufirst%3DB.%2BJ.%26aulast%3DHolm%26aufirst%3DP.%2BS.%26aulast%3DGschwend%26aufirst%3DJ.%2BE.%26aulast%3DRetz%26aufirst%3DM.%26aulast%3DNawroth%26aufirst%3DR.%26atitle%3DCDK4%252F6%2520Inhibition%2520Controls%2520Proliferation%2520of%2520Bladder%2520Cancer%2520and%2520Transcription%2520of%2520RB1%26jtitle%3DJ.%2520Urol.%26date%3D2016%26volume%3D195%26spage%3D771%26epage%3D779%26doi%3D10.1016%2Fj.juro.2015.08.082" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Molina, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jafari, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ignatushchenko, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larsson, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sreekumar, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nordlund, P.</span></span> <span> </span><span class="NLM_article-title">Monitoring Drug Target Engagement in Cells and Tissues Using the Cellular Thermal Shift Assay</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>341</i></span>,  <span class="NLM_fpage">84</span>â <span class="NLM_lpage">87</span>, <span class="refDoi">Â DOI: 10.1126/science.1233606</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=10.1126%2Fscience.1233606" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=23828940" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtVCgs7rP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=341&publication_year=2013&pages=84-87&author=D.+M.+Molinaauthor=R.+Jafariauthor=M.+Ignatushchenkoauthor=T.+Sekiauthor=E.+A.+Larssonauthor=C.+Danauthor=L.+Sreekumarauthor=Y.+Caoauthor=P.+Nordlund&title=Monitoring+Drug+Target+Engagement+in+Cells+and+Tissues+Using+the+Cellular+Thermal+Shift+Assay&doi=10.1126%2Fscience.1233606"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Monitoring Drug Target Engagement in Cells and Tissues Using the Cellular Thermal Shift Assay</span></div><div class="casAuthors">Molina, Daniel Martinez; Jafari, Rozbeh; Ignatushchenko, Marina; Seki, Takahiro; Larsson, E. Andreas; Dan, Chen; Sreekumar, Lekshmy; Cao, Yihai; Nordlund, Paer</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">341</span>
        (<span class="NLM_cas:issue">6141</span>),
    <span class="NLM_cas:pages">84-87</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The efficacy of therapeutics is dependent on a drug binding to its cognate target.  Optimization of target engagement by drugs in cells is often challenging, because drug binding cannot be monitored inside cells.  We have developed a method for evaluating drug binding to target proteins in cells and tissue samples.  This cellular thermal shift assay (CETSA) is based on the biophys. principle of ligand-induced thermal stabilization of target proteins.  Using this assay, we validated drug binding for a set of important clin. targets and monitored processes of drug transport and activation, off-target effects and drug resistance in cancer cell lines, as well as drug distribution in tissues.  CETSA is likely to become a valuable tool for the validation and optimization of drug target engagement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoaaWSoozY7ObVg90H21EOLACvtfcHk0lh3Egl3pCJCFw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtVCgs7rP&md5=f8ba686ba38b67dbdf366c49eb7375b4</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1126%2Fscience.1233606&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1233606%26sid%3Dliteratum%253Aachs%26aulast%3DMolina%26aufirst%3DD.%2BM.%26aulast%3DJafari%26aufirst%3DR.%26aulast%3DIgnatushchenko%26aufirst%3DM.%26aulast%3DSeki%26aufirst%3DT.%26aulast%3DLarsson%26aufirst%3DE.%2BA.%26aulast%3DDan%26aufirst%3DC.%26aulast%3DSreekumar%26aufirst%3DL.%26aulast%3DCao%26aufirst%3DY.%26aulast%3DNordlund%26aufirst%3DP.%26atitle%3DMonitoring%2520Drug%2520Target%2520Engagement%2520in%2520Cells%2520and%2520Tissues%2520Using%2520the%2520Cellular%2520Thermal%2520Shift%2520Assay%26jtitle%3DScience%26date%3D2013%26volume%3D341%26spage%3D84%26epage%3D87%26doi%3D10.1126%2Fscience.1233606" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wienken, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baaske, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rothbauer, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Braun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duhr, S.</span></span> <span> </span><span class="NLM_article-title">Protein-binding Assays in Biological Liquids Using Microscale Thermophoresis</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">100</span>, <span class="refDoi">Â DOI: 10.1038/ncomms1093</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=10.1038%2Fncomms1093" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=20981028" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=1%3ACAS%3A280%3ADC%252BC3cbisleguw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2010&pages=100&author=C.+J.+Wienkenauthor=P.+Baaskeauthor=U.+Rothbauerauthor=D.+Braunauthor=S.+Duhr&title=Protein-binding+Assays+in+Biological+Liquids+Using+Microscale+Thermophoresis&doi=10.1038%2Fncomms1093"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Protein-binding assays in biological liquids using microscale thermophoresis</span></div><div class="casAuthors">Wienken Christoph J; Baaske Philipp; Rothbauer Ulrich; Braun Dieter; Duhr Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">100</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Protein interactions inside the human body are expected to differ from the situation in vitro.  This is crucial when investigating protein functions or developing new drugs.  In this study, we present a sample-efficient, free-solution method, termed microscale thermophoresis, that is capable of analysing interactions of proteins or small molecules in biological liquids such as blood serum or cell lysate.  The technique is based on the thermophoresis of molecules, which provides information about molecule size, charge and hydration shell.  We validated the method using immunologically relevant systems including human interferon gamma and the interaction of calmodulin with calcium.  The affinity of the small-molecule inhibitor quercetin to its kinase PKA was determined in buffer and human serum, revealing a 400-fold reduced affinity in serum.  This information about the influence of the biological matrix may allow to make more reliable conclusions on protein functionality, and may facilitate more efficient drug development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTOCqf6rsXbPv6BottYdIAWfW6udTcc2ebvEZj9CTOX_Lntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3cbisleguw%253D%253D&md5=5660d96475370e7f46ea6682c9398cb1</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1038%2Fncomms1093&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms1093%26sid%3Dliteratum%253Aachs%26aulast%3DWienken%26aufirst%3DC.%2BJ.%26aulast%3DBaaske%26aufirst%3DP.%26aulast%3DRothbauer%26aufirst%3DU.%26aulast%3DBraun%26aufirst%3DD.%26aulast%3DDuhr%26aufirst%3DS.%26atitle%3DProtein-binding%2520Assays%2520in%2520Biological%2520Liquids%2520Using%2520Microscale%2520Thermophoresis%26jtitle%3DNat.%2520Commun.%26date%3D2010%26volume%3D1%26spage%3D100%26doi%3D10.1038%2Fncomms1093" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Marshall, N. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Price, D. H.</span></span> <span> </span><span class="NLM_article-title">Control of RNA polymerase II Elongation Potential by a Novel Carboxyl-terminal Domain Kinase</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>271</i></span>,  <span class="NLM_fpage">27176</span>â <span class="NLM_lpage">27183</span>, <span class="refDoi">Â DOI: 10.1074/jbc.271.43.27176</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=10.1074%2Fjbc.271.43.27176" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=8900211" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=1%3ACAS%3A528%3ADyaK28XmsFSisb8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=271&publication_year=1996&pages=27176-27183&author=N.+F.+Marshallauthor=J.+Pengauthor=Z.+Xieauthor=D.+H.+Price&title=Control+of+RNA+polymerase+II+Elongation+Potential+by+a+Novel+Carboxyl-terminal+Domain+Kinase&doi=10.1074%2Fjbc.271.43.27176"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Control of RNA polymerase II elongation potential by a novel carboxyl-terminal domain kinase</span></div><div class="casAuthors">Marshall, Nick F.; Peng, Junmin; Xie, Zhi; Price, David H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">271</span>
        (<span class="NLM_cas:issue">43</span>),
    <span class="NLM_cas:pages">27176-27183</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The entry of RNA polymerase II into a productive mode of elongation is controlled, in part, by the postinitiation activity of pos. transcription elongation factor b (P-TEFb).  Removal of the C-terminal domain (CTD) of the large subunit of RNA polymerase II abolishes productive elongation.  Correspondingly, P-TEFb can phosphorylate the CTD of pure RNA polymerase II.  Furthermore, P-TEFb can phosphorylate the CTD of RNA polymerase II when the polymerase is in an early elongation complex.  Both the function and kinase activity of P-TEFb are blocked by the drugs 5,6-dichloro-1-Î²-D-ribofuranosylbenzimidazole (DRB) and H-8.  P-TEFb is distinct from transcription factor IIH (TFIIH) because the 2 factors have no subunits in common, P-TEFb is more sensitive to DRB than is TFIIH, and most importantly, TFIIH cannot substitute functionally for P-TEFb.  Phosphorylation of the CTD by P-TEFb may control the transition from abortive into productive elongation mode.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVKuChjQS6p7Vg90H21EOLACvtfcHk0lj3BO_Ua54_PA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XmsFSisb8%253D&md5=ed4cfba88e5096094114d212e5da372c</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1074%2Fjbc.271.43.27176&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.271.43.27176%26sid%3Dliteratum%253Aachs%26aulast%3DMarshall%26aufirst%3DN.%2BF.%26aulast%3DPeng%26aufirst%3DJ.%26aulast%3DXie%26aufirst%3DZ.%26aulast%3DPrice%26aufirst%3DD.%2BH.%26atitle%3DControl%2520of%2520RNA%2520polymerase%2520II%2520Elongation%2520Potential%2520by%2520a%2520Novel%2520Carboxyl-terminal%2520Domain%2520Kinase%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1996%26volume%3D271%26spage%3D27176%26epage%3D27183%26doi%3D10.1074%2Fjbc.271.43.27176" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hnisz, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abraham, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, T. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lau, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saint-AndrÃ©, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sigova, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoke, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. A.</span></span> <span> </span><span class="NLM_article-title">Super-enhancers in the Control of Cell Identity and Disease</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>155</i></span>,  <span class="NLM_fpage">934</span>â <span class="NLM_lpage">947</span>, <span class="refDoi">Â DOI: 10.1016/j.cell.2013.09.053</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=10.1016%2Fj.cell.2013.09.053" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=24119843" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1SrtLbL" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=155&publication_year=2013&pages=934-947&author=D.+Hniszauthor=B.+J.+Abrahamauthor=T.+I.+Leeauthor=A.+Lauauthor=V.+Saint-Andr%C3%A9author=A.+A.+Sigovaauthor=H.+A.+Hokeauthor=R.+A.+Young&title=Super-enhancers+in+the+Control+of+Cell+Identity+and+Disease&doi=10.1016%2Fj.cell.2013.09.053"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Super-Enhancers in the Control of Cell Identity and Disease</span></div><div class="casAuthors">Hnisz, Denes; Abraham, Brian J.; Lee, Tong Ihn; Lau, Ashley; Saint-Andre, Violaine; Sigova, Alla A.; Hoke, Heather A.; Young, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">155</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">934-947</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Super-enhancers are large clusters of transcriptional enhancers that drive expression of genes that define cell identity.  Improved understanding of the roles that super-enhancers play in biol. would be afforded by knowing the constellation of factors that constitute these domains and by identifying super-enhancers across the spectrum of human cell types.  We describe here the population of transcription factors, cofactors, chromatin regulators, and transcription app. occupying super-enhancers in embryonic stem cells and evidence that super-enhancers are highly transcribed.  We produce a catalog of super-enhancers in a broad range of human cell types and find that super-enhancers assoc. with genes that control and define the biol. of these cells.  Interestingly, disease-assocd. variation is esp. enriched in the super-enhancers of disease-relevant cell types.  Furthermore, we find that cancer cells generate super-enhancers at oncogenes and other genes important in tumor pathogenesis.  Thus, super-enhancers play key roles in human cell identity in health and in disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqOx0QNN2fg3bVg90H21EOLACvtfcHk0lj3BO_Ua54_PA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1SrtLbL&md5=ca83fdac41e4f19387adb996bd0e37d8</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2013.09.053&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2013.09.053%26sid%3Dliteratum%253Aachs%26aulast%3DHnisz%26aufirst%3DD.%26aulast%3DAbraham%26aufirst%3DB.%2BJ.%26aulast%3DLee%26aufirst%3DT.%2BI.%26aulast%3DLau%26aufirst%3DA.%26aulast%3DSaint-Andr%25C3%25A9%26aufirst%3DV.%26aulast%3DSigova%26aufirst%3DA.%2BA.%26aulast%3DHoke%26aufirst%3DH.%2BA.%26aulast%3DYoung%26aufirst%3DR.%2BA.%26atitle%3DSuper-enhancers%2520in%2520the%2520Control%2520of%2520Cell%2520Identity%2520and%2520Disease%26jtitle%3DCell%26date%3D2013%26volume%3D155%26spage%3D934%26epage%3D947%26doi%3D10.1016%2Fj.cell.2013.09.053" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bharate, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vishwakarma, R. A.</span></span> <span> </span><span class="NLM_article-title">Thermodynamic Equilibrium Solubility Measurements in Simulated Fluids by 96-well Plate Method in Early Drug Discovery</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1561</span>â <span class="NLM_lpage">1567</span>, <span class="refDoi">Â DOI: 10.1016/j.bmcl.2015.02.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=10.1016%2Fj.bmcl.2015.02.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=25740159" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;key=1%3ACAS%3A528%3ADC%252BC2MXivVSrsbc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2015&pages=1561-1567&author=S.+S.+Bharateauthor=R.+A.+Vishwakarma&title=Thermodynamic+Equilibrium+Solubility+Measurements+in+Simulated+Fluids+by+96-well+Plate+Method+in+Early+Drug+Discovery&doi=10.1016%2Fj.bmcl.2015.02.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Thermodynamic equilibrium solubility measurements in simulated fluids by 96-well plate method in early drug discovery</span></div><div class="casAuthors">Bharate, Sonali S.; Vishwakarma, Ram A.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1561-1567</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">An early prediction of soly. in physiol. media (PBS, SGF and SIF) is useful to predict qual. bioavailability and absorption of lead candidates.  Despite of the availability of multiple soly. estn. methods, none of the reported method involves simplified fixed protocol for diverse set of compds.  Therefore, a simple and medium-throughput soly. estn. protocol is highly desirable during lead optimization stage.  The present work introduces a rapid method for assessment of thermodn. equil. soly. of compds. in aq. media using 96-well microplate.  The developed protocol is straightforward to set up and takes advantage of the sensitivity of UV spectroscopy.  The compd., in stock soln. in methanol, is introduced in microgram quantities into microplate wells followed by drying at an ambient temp.  Microplates were shaken upon addn. of test media and the supernatant was analyzed by UV method.  A plot of absorbance vs. concn. of a sample provides satn. point, which is thermodn. equil. soly. of a sample.  The established protocol was validated using a large panel of com. available drugs and with conventional miniaturized shake flask method (r2 >0.84).  Addnl., the statistically significant QSPR models were established using exptl. soly. values of 52 compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrEhIADhdQd3LVg90H21EOLACvtfcHk0lhgA2NdV_Uz4Q"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXivVSrsbc%253D&md5=e118b10c67ca311ea5da994793546c11</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2015.02.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2015.02.013%26sid%3Dliteratum%253Aachs%26aulast%3DBharate%26aufirst%3DS.%2BS.%26aulast%3DVishwakarma%26aufirst%3DR.%2BA.%26atitle%3DThermodynamic%2520Equilibrium%2520Solubility%2520Measurements%2520in%2520Simulated%2520Fluids%2520by%252096-well%2520Plate%2520Method%2520in%2520Early%2520Drug%2520Discovery%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D25%26spage%3D1561%26epage%3D1567%26doi%3D10.1016%2Fj.bmcl.2015.02.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5L2I" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5L2I','PDB','5L2I'); return false;">PDB: 5L2I</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6GZH" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6GZH','PDB','6GZH'); return false;">PDB: 6GZH</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i135"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b02121">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_33440"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.9b02121?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b02121</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline"><sup>1</sup>H NMR and <sup>13</sup>C NMR spectra and purity table of target compounds (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b02121/suppl_file/jm9b02121_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings and some data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b02121/suppl_file/jm9b02121_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b02121/suppl_file/jm9b02121_si_001.pdf">jm9b02121_si_001.pdf (11.03 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b02121/suppl_file/jm9b02121_si_002.csv">jm9b02121_si_002.csv (6.8 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.9b02121&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.9b02121%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2020.63.issue-6" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.9b02121" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                8MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright Â© 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">Youâve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679970a76ef53d60","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
