
<html lang="en"     class="pb-page"  data-request-id="550bac6a-0ade-4de4-8b19-dd3e98451628"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.8b01978;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text;issue:issue:10.1021/jmcmar.2019.62.issue-9"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Identification of Three-Way DNA Junction Ligands through Screening of Chemical Libraries and Validation by Complementary in Vitro Assays" /></meta><meta name="dc.Creator" content="Katerina  Duskova" /></meta><meta name="dc.Creator" content="Jérémy  Lamarche" /></meta><meta name="dc.Creator" content="Souheila  Amor" /></meta><meta name="dc.Creator" content="Coralie  Caron" /></meta><meta name="dc.Creator" content="Nicolas  Queyriaux" /></meta><meta name="dc.Creator" content="Marie  Gaschard" /></meta><meta name="dc.Creator" content="Marie-José  Penouilh" /></meta><meta name="dc.Creator" content="Guillaume  de Robillard" /></meta><meta name="dc.Creator" content="Dominique  Delmas" /></meta><meta name="dc.Creator" content="Charles H.  Devillers" /></meta><meta name="dc.Creator" content="Anton  Granzhan" /></meta><meta name="dc.Creator" content="Marie-Paule  Teulade-Fichou" /></meta><meta name="dc.Creator" content="Murielle  Chavarot-Kerlidou" /></meta><meta name="dc.Creator" content="Bruno  Therrien" /></meta><meta name="dc.Creator" content="Sébastien  Britton" /></meta><meta name="dc.Creator" content="David  Monchaud" /></meta><meta name="dc.Description" content="The human genome is replete with repetitive DNA sequences that can fold into thermodynamically stable secondary structures such as hairpins and quadruplexes. Cellular enzymes exist to cope with the..." /></meta><meta name="Description" content="The human genome is replete with repetitive DNA sequences that can fold into thermodynamically stable secondary structures such as hairpins and quadruplexes. Cellular enzymes exist to cope with the..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="April 3, 2019" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.8b01978" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2019 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.8b01978" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.8b01978" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.8b01978" /></link>
        
    
    

<title>Identification of Three-Way DNA Junction Ligands through Screening of Chemical Libraries and Validation by Complementary in Vitro Assays | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.8b01978" /></meta><meta property="og:title" content="Identification of Three-Way DNA Junction Ligands through Screening of Chemical Libraries and Validation by Complementary in Vitro Assays" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.8b01978/20190502/images/large/jm-2018-019785_0003.jpeg" /></meta><meta property="og:description" content="The human genome is replete with repetitive DNA sequences that can fold into thermodynamically stable secondary structures such as hairpins and quadruplexes. Cellular enzymes exist to cope with these structures whose stable accumulation would result in DNA damage through interference with DNA transactions such as transcription and replication. Therefore, the chemical stabilization of secondary DNA structures offers an attractive way to foster DNA transaction-associated damages to trigger cell death in proliferating cancer cells. While much emphasis has been recently given to DNA quadruplexes, we focused here on three-way DNA junctions (TWJ) and report on a strategy to identify TWJ-targeting agents through a combination of in vitro techniques (TWJ-screen, polyacrylamide gel electrophoresis, fluorescence resonance energy transfer-melting, electrospray ionization mass spectrometry, dialysis equilibrium, and sulforhodamine B assays). We designed a complete workflow and screened 1200 compounds to identify promising TWJ ligands selected on stringent criteria in terms of TWJ-folding ability, affinity, and selectivity." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.8b01978"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.8b01978">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.8b01978&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.8b01978&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.8b01978&amp;href=/doi/10.1021/acs.jmedchem.8b01978" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2019</span><span class="cit-fg-volume">, 62</span><span class="cit-fg-issue">, 9</span><span class="cit-fg-pageRange">, 4456-4466</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/62/9" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.8b01943" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.8b01986" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Identification of Three-Way DNA Junction Ligands through Screening of Chemical Libraries and Validation by Complementary in Vitro Assays</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Katerina Duskova</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Katerina Duskova</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institut de Chimie Moléculaire (ICMUB), CNRS UMR6302, UBFC, 21078 Dijon, France</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Katerina++Duskova">Katerina Duskova</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jérémy Lamarche</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jérémy Lamarche</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institut de Chimie Moléculaire (ICMUB), CNRS UMR6302, UBFC, 21078 Dijon, France</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=J%C3%A9r%C3%A9my++Lamarche">Jérémy Lamarche</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Souheila Amor</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Souheila Amor</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Lipids, Nutrition & Cancers (LNC), INSERM U1321, UBFC, 21000 Dijon, France</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Souheila++Amor">Souheila Amor</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Coralie Caron</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Coralie Caron</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institut Curie, PSL Research University, CNRS UMR9187, INSERM U1196, 91405 Orsay, France</div><div class="loa-info-affiliations-info">Université Paris-Sud, Université Paris Saclay, 91405 Orsay, France</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Coralie++Caron">Coralie Caron</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Nicolas Queyriaux</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Nicolas Queyriaux</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Laboratoire de Chimie et Biologie des Métaux, Université Grenoble Alpes, CNRS UMR5249, CEA, 38054 Grenoble, France</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Nicolas++Queyriaux">Nicolas Queyriaux</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Marie Gaschard</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Marie Gaschard</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institut de Chimie, Université de Neuchâtel, CH-2000 Neuchatel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Marie++Gaschard">Marie Gaschard</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Marie-José Penouilh</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Marie-José Penouilh</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Pôle Chimie Moléculaire (PACSMUB), Faculté des Sciences Mirande, UBFC, 21078 Dijon, France</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Marie-Jos%C3%A9++Penouilh">Marie-José Penouilh</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Guillaume de Robillard</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Guillaume de Robillard</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institut de Chimie Moléculaire (ICMUB), CNRS UMR6302, UBFC, 21078 Dijon, France</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Guillaume++de+Robillard">Guillaume de Robillard</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Dominique Delmas</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Dominique Delmas</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Lipids, Nutrition & Cancers (LNC), INSERM U1321, UBFC, 21000 Dijon, France</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Dominique++Delmas">Dominique Delmas</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Charles H. Devillers</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Charles H. Devillers</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institut de Chimie Moléculaire (ICMUB), CNRS UMR6302, UBFC, 21078 Dijon, France</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Charles+H.++Devillers">Charles H. Devillers</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-9078-7035" title="Orcid link">http://orcid.org/0000-0001-9078-7035</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Anton Granzhan</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Anton Granzhan</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institut Curie, PSL Research University, CNRS UMR9187, INSERM U1196, 91405 Orsay, France</div><div class="loa-info-affiliations-info">Université Paris-Sud, Université Paris Saclay, 91405 Orsay, France</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Anton++Granzhan">Anton Granzhan</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-0424-0461" title="Orcid link">http://orcid.org/0000-0002-0424-0461</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Marie-Paule Teulade-Fichou</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Marie-Paule Teulade-Fichou</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institut Curie, PSL Research University, CNRS UMR9187, INSERM U1196, 91405 Orsay, France</div><div class="loa-info-affiliations-info">Université Paris-Sud, Université Paris Saclay, 91405 Orsay, France</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Marie-Paule++Teulade-Fichou">Marie-Paule Teulade-Fichou</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Murielle Chavarot-Kerlidou</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Murielle Chavarot-Kerlidou</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Laboratoire de Chimie et Biologie des Métaux, Université Grenoble Alpes, CNRS UMR5249, CEA, 38054 Grenoble, France</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Murielle++Chavarot-Kerlidou">Murielle Chavarot-Kerlidou</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-2709-3772" title="Orcid link">http://orcid.org/0000-0003-2709-3772</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Bruno Therrien</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Bruno Therrien</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institut de Chimie, Université de Neuchâtel, CH-2000 Neuchatel, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Bruno++Therrien">Bruno Therrien</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-0388-2745" title="Orcid link">http://orcid.org/0000-0002-0388-2745</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Sébastien Britton</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Sébastien Britton</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institut de Pharmacologie et de Biologie Structurale, IPBS, CNRS UMR5089, Université de Toulouse, UPS, équipe labellisée la Ligue Contre le Cancer, 31077 Toulouse, France</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=S%C3%A9bastien++Britton">Sébastien Britton</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">David Monchaud</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">David Monchaud</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institut de Chimie Moléculaire (ICMUB), CNRS UMR6302, UBFC, 21078 Dijon, France</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#187c796e717c367577767b70796d7c587b766a6b367e6a"><span class="__cf_email__" data-cfemail="dabebbacb3bef4b7b5b4b9b2bbafbe9ab9b4a8a9f4bca8">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=David++Monchaud">David Monchaud</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-3056-9295" title="Orcid link">http://orcid.org/0000-0002-3056-9295</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.8b01978&amp;href=/doi/10.1021%2Facs.jmedchem.8b01978" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2019</span></span><span class="cit-volume">, 62</span><span class="cit-issue">, 9</span><span class="cit-pageRange">, 4456–4466</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">April 3, 2019</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>18 December 2018</li><li><span class="item_label"><b>Published</b> online</span>3 April 2019</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 9 May 2019</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.8b01978" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b01978</a></div><div class="article_header-article-copyright"><strong>Copyright © 2019 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D4456%26pageCount%3D11%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DKaterina%2BDuskova%252C%2BJ%25C3%25A9r%25C3%25A9my%2BLamarche%252C%2BSouheila%2BAmor%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D62%26issueNum%3D9%26contentID%3Dacs.jmedchem.8b01978%26title%3DIdentification%2Bof%2BThree-Way%2BDNA%2BJunction%2BLigands%2Bthrough%2BScreening%2Bof%2BChemical%2BLibraries%2Band%2BValidation%2Bby%2BComplementary%2Bin%2BVitro%2BAssays%26numPages%3D11%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D4466%26publicationDate%3DMay%2B2019">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.8b01978"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1018</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">8</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.8b01978" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Identification of Three-Way DNA Junction Ligands through Screening of Chemical Libraries and Validation by Complementary in Vitro Assays&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Katerina&quot;,&quot;last_name&quot;:&quot;Duskova&quot;},{&quot;first_name&quot;:&quot;Jérémy&quot;,&quot;last_name&quot;:&quot;Lamarche&quot;},{&quot;first_name&quot;:&quot;Souheila&quot;,&quot;last_name&quot;:&quot;Amor&quot;},{&quot;first_name&quot;:&quot;Coralie&quot;,&quot;last_name&quot;:&quot;Caron&quot;},{&quot;first_name&quot;:&quot;Nicolas&quot;,&quot;last_name&quot;:&quot;Queyriaux&quot;},{&quot;first_name&quot;:&quot;Marie&quot;,&quot;last_name&quot;:&quot;Gaschard&quot;},{&quot;first_name&quot;:&quot;Marie-José&quot;,&quot;last_name&quot;:&quot;Penouilh&quot;},{&quot;first_name&quot;:&quot;Guillaume&quot;,&quot;last_name&quot;:&quot;de Robillard&quot;},{&quot;first_name&quot;:&quot;Dominique&quot;,&quot;last_name&quot;:&quot;Delmas&quot;},{&quot;first_name&quot;:&quot;Charles&quot;,&quot;last_name&quot;:&quot;H. Devillers&quot;},{&quot;first_name&quot;:&quot;Anton&quot;,&quot;last_name&quot;:&quot;Granzhan&quot;},{&quot;first_name&quot;:&quot;Marie-Paule&quot;,&quot;last_name&quot;:&quot;Teulade-Fichou&quot;},{&quot;first_name&quot;:&quot;Murielle&quot;,&quot;last_name&quot;:&quot;Chavarot-Kerlidou&quot;},{&quot;first_name&quot;:&quot;Bruno&quot;,&quot;last_name&quot;:&quot;Therrien&quot;},{&quot;first_name&quot;:&quot;Sébastien&quot;,&quot;last_name&quot;:&quot;Britton&quot;},{&quot;first_name&quot;:&quot;David&quot;,&quot;last_name&quot;:&quot;Monchaud&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2019&quot;,&quot;month&quot;:&quot;04&quot;,&quot;day&quot;:&quot;03&quot;,&quot;issue&quot;:&quot;9&quot;,&quot;volume&quot;:&quot;62&quot;,&quot;pages&quot;:&quot;4456-4466&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.8b01978&quot;},&quot;abstract&quot;:&quot;The human genome is replete with repetitive DNA sequences that can fold into thermodynamically stable secondary structures such as hairpins and quadruplexes. Cellular enzymes exist to cope with these structures whose stable accumulation would result in DNA damage through interference with DNA transactions such as transcription and replication. Therefore, the chemical stabilization of secondary DNA structures offers an attractive way to foster DNA transaction-associated damages to trigger cell death in proliferating cancer cells. While much emphasis has been recently given to DNA quadruplexes, we focused here on three-way DNA junctions (TWJ) and report on a strategy to identify TWJ-targeting agents through a combination of in vitro techniques (TWJ-screen, polyacrylamide gel electrophoresis, fluorescence resonance energy transfer-melting, electrospray ionization mass spectrometry, dialysis equilibrium, and sulforhodamine B assays). We designed a complete workflow and screened 1200 compounds to identify promisin&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b01978&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b01978" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b01978&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b01978" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b01978&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b01978" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.8b01978&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b01978&amp;href=/doi/10.1021/acs.jmedchem.8b01978" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.8b01978" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.8b01978" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (3 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b01978&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.8b01978%26sid%3Dliteratum%253Aachs%26pmid%3D30942581%26genre%3Darticle%26aulast%3DDuskova%26date%3D2019%26atitle%3DIdentification%2Bof%2BThree-Way%2BDNA%2BJunction%2BLigands%2Bthrough%2BScreening%2Bof%2BChemical%2BLibraries%2Band%2BValidation%2Bby%2BComplementary%2Bin%2BVitro%2BAssays%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D62%26issue%3D9%26spage%3D4456%26epage%3D4466%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (3)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292366" title="Ligands">Ligands</a>,</li><li><a href="/action/doSearch?ConceptID=291439" title="Genetics">Genetics</a>,</li><li><a href="/action/doSearch?ConceptID=291445" title="Chemical structure">Chemical structure</a>,</li><li><a href="/action/doSearch?ConceptID=291350" title="Assays">Assays</a>,</li><li><a href="/action/doSearch?ConceptID=291230" title="Polyacrylamide gel electrophoresis">Polyacrylamide gel electrophoresis</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/62/9" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/jmcmar.2019.62.issue-9/20190509/jmcmar.2019.62.issue-9.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.8b01978/20190502/images/medium/jm-2018-019785_0003.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.8b01978/20190502/images/large/jm-2018-019785_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01978&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">The human genome is replete with repetitive DNA sequences that can fold into thermodynamically stable secondary structures such as hairpins and quadruplexes. Cellular enzymes exist to cope with these structures whose stable accumulation would result in DNA damage through interference with DNA transactions such as transcription and replication. Therefore, the chemical stabilization of secondary DNA structures offers an attractive way to foster DNA transaction-associated damages to trigger cell death in proliferating cancer cells. While much emphasis has been recently given to DNA quadruplexes, we focused here on three-way DNA junctions (TWJ) and report on a strategy to identify TWJ-targeting agents through a combination of in vitro techniques (TWJ-screen, polyacrylamide gel electrophoresis, fluorescence resonance energy transfer-melting, electrospray ionization mass spectrometry, dialysis equilibrium, and sulforhodamine B assays). We designed a complete workflow and screened 1200 compounds to identify promising TWJ ligands selected on stringent criteria in terms of TWJ-folding ability, affinity, and selectivity.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_45342" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_45342" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">DNA transactions (replication and transcription) are finely orchestrated cellular processes that are frequently challenged by situations of crisis notably when dealing with damaged DNA or secondary structures or in situations of nucleotide pool shortage or imbalance, which can result in DNA and RNA polymerase stalling.<a onclick="showRef(event, 'ref1 ref2'); return false;" href="javascript:void(0);" class="ref ref1 ref2">(1,2)</a> If persistent, these polymerase arrests activate the DNA damage response that includes DNA repair mechanisms and that dictates cell tolerance and survival.<a onclick="showRef(event, 'ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref3 ref4">(3,4)</a> Crisis inducers can lead to genomic instability and eventually to cell death if DNA repair is unsuccessful or unfaithful or generates toxic products. Sources of such situations can be endogenous (e.g., stochastic nucleotide misincorporation, reactive oxygen species-promoted lesions) or exogenous (e.g., irradiation, genotoxic compounds).<a onclick="showRef(event, 'ref5 ref6'); return false;" href="javascript:void(0);" class="ref ref5 ref6">(5,6)</a></div><div class="NLM_p">Several anticancer therapies rely on genotoxic chemicals that trigger covalent DNA damages including cross-links (e.g., cisplatin), alkylated bases (e.g., temozolomide), or trapped topoisomerase I and II complexes (e.g., camptothecin analogues and etoposide).<a onclick="showRef(event, 'ref7 ref8'); return false;" href="javascript:void(0);" class="ref ref7 ref8">(7,8)</a> An alternative approach would be to consider chemicals capable of stabilizing DNA secondary structures that might arise during DNA transactions.<a onclick="showRef(event, 'ref9 ref10'); return false;" href="javascript:void(0);" class="ref ref9 ref10">(9,10)</a> Repeated DNA sequences are indeed prone to fold into higher-order structures in response to the torsional stress provoked by the progression of polymerases. These secondary structures (i.e., hairpin, G-quadruplex, R-loop, DNA junction), whose nature depends on the sequences involved (i.e., direct or inverted tandem repeats), represent topological obstacles the cellular machinery must cope with efficiently to proceed, mostly through the dedicated helicases.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> Stabilizing these structures by chemicals thus represents an innovative way to trigger cell crisis and death in cells defective for genome maintenance mechanisms.</div><div class="NLM_p">Over the past few years, most emphasis has been given to G-quadruplexes as roadblocks to DNA transactions<a onclick="showRef(event, 'ref12 ref13 ref14'); return false;" href="javascript:void(0);" class="ref ref12 ref13 ref14">(12−14)</a> but also as new and promising genetic switches to control gene expression.<a onclick="showRef(event, 'ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref15 ref16">(15,16)</a> Alternatively, we<a onclick="showRef(event, 'ref17 ref18 ref19'); return false;" href="javascript:void(0);" class="ref ref17 ref18 ref19">(17−19)</a> and others<a onclick="showRef(event, 'ref20 ref21 ref22 ref23 ref24 ref25 ref26 ref27'); return false;" href="javascript:void(0);" class="ref ref20 ref21 ref22 ref23 ref24 ref25 ref26 ref27">(20−27)</a> have invested efforts to study DNA junctions as druggable targets, chiefly three-way DNA junctions (TWJ). We recently developed the high-throughput screening (HTS) assay TWJ-screen that allows for in vitro identification of promising TWJ-interacting compounds (or TWJ ligands).<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> To be selective, ligands must interact with the structural peculiarity of the TWJ structure, that is, the branch point: this cavity, at the very heart of the structure, results from the convergence of the three duplex arms that constitute the TWJ (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A). Here, we kept on searching for new TWJ ligands evaluating 1200 candidates from three different chemical libraries. We then further investigated the TWJ-interacting properties of the best candidates via a series of complementary in vitro assays (polyacrylamide gel electrophoresis (PAGE), fluorescence resonance energy transfer (FRET)-melting, electrospray ionization mass spectrometry (ESI-MS), and equilibrium dialysis) to gain reliable insights into their affinity and selectivity for TWJ.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.8b01978/20190502/images/medium/jm-2018-019785_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.8b01978/20190502/images/large/jm-2018-019785_0004.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. (A) Schematic representation of the folding of a TWJ structure. (B) TWJ-screen results of a library of 1200 compounds, expressed as the % of quenched FAM fluorescence of the three strands’ mixture (M) in the presence of 5 mol equiv candidate (i.e., M–(M + ligand), <i>Y</i>-axis) as a function of the % of FAM-TWJ-S1 fluorescence measured in the presence of 5 mol equiv candidate (i.e., FAM-TWJ-S1 + ligand, <i>X</i>-axis). Experiments performed with 0.2 μM DNA, without or with 1.0 μM candidates, in 10 mM lithium cacodylate plus 10 mM KCl/90 mM LiCl, pH 7.2 at 37 °C for 1 h. (C) Chemical structures of the top 15 ligands identified through TWJ-screen evaluation.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.8b01978/20190502/images/large/jm-2018-019785_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01978&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_83573" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_83573" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Selection of Chemical Libraries for TWJ-Screen Evaluations</h3><div class="NLM_p last">To assess whether the TWJ-screen assay is suited for screening wide collections of small-molecule candidates, we used three different chemical libraries. (i) The Pathogen Box, from Medicines for Malaria Venture:<a onclick="showRef(event, 'ref28 ref29'); return false;" href="javascript:void(0);" class="ref ref28 ref29">(28,29)</a> this library, available free of charge, comprises 400 druglike molecules intended to be assayed against neglected tropical diseases including malaria, tuberculosis, toxoplasmosis, dengue, etc. Chemical structures can be found at <a href="https://www.pathogenbox.org/about-pathogen-box/composition" class="extLink">https://www.pathogenbox.org/about-pathogen-box/composition</a>. (ii) The Chimiothèque Nationale Essentielle (CNE, version 2017):<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> this library comprises 640 compounds, either synthetic or natural products, gathered from different French laboratories under the authority of the French Chemical Library. This library is commercially available; the full list of chemicals cannot be shown due to owner restriction. (iii) The so-called ICMUB library: it comprises 160 compounds that have been assembled through national and international collaborations over the years. The full list of compounds cannot be shown due to confidentiality issues; however, the chemical structures of all identified hits are shown in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>. Collectively, these three libraries represent a total of 1200 candidates, which display a very high level of chemical diversity.</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Improved Version of the TWJ-Screen Assay</h3><div class="NLM_p">The TWJ-screen assay allows for quantifying the TWJ-folding ability of chemical compounds in an HTS manner.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> This method is based on the ligand-promoted formation of a TWJ structure from a mixture of the three separated strands, two of them being labeled with fluorophores (one with fluorescein amidite, or FAM, the other one with tetramethylrhodamine, or TAMRA). The ligand-promoted TWJ formation results in the FAM and TAMRA fluorophore being in proximity, resulting in the quenching of FAM fluorescence that represents a quantitative measurement of TWJ formation. The TWJ-screen protocol was slightly adapted to screen wider chemical libraries: the initial protocol was performed at room temperature (i.e., in a 20–25 °C range), making the temperature the only parameter left uncontrolled. For improving its reproducibility, the TWJ-screen was here implemented at a fixed temperature of 37 °C. This higher temperature was selected since it allowed for improving the reproducibility of the assay—lessening the unassisted assembly of the three separated strands into a TWJ structure (vide infra)—and for selecting candidates in a more stringent manner, given that they have to fold and interact with TWJ in less-favorable thermal conditions. We screened approximately 60 candidates/day using the otherwise initially described conditions: in 100 μL (final volume) of buffered solution (10 mM lithium cacodylate buffer, pH 7.2, supplemented with 10 mM KCl/90 mM LiCl, or CacoK) was introduced 0.2 μM of the three strands initially described by Leontis et al.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> and further studied by Brabec et al. for their ability to assemble into a TWJ stable only upon interaction with ad hoc ligands,<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> that is, FAM-TWJ-S1 (FAM-d[<sup>5′</sup>CG<sub>2</sub>A<sub>2</sub>CG<sub>2</sub>CACTCG<sup>3′</sup>]), TWJ-S2 (d[<sup>5′</sup>CGAGTGCAGCGTG<sub>2</sub><sup>3′</sup>]), and TWJ-S3-TAMRA (d[<sup>5′</sup>C<sub>2</sub>ACGCTCGT<sub>2</sub>C<sub>2</sub>G<sup>3′</sup>]-TAMRA). Then, 5 mol equiv (i.e., 1.0 μM) of candidate were introduced (except in control wells, vide infra) as triplicates. The microplate preparation is completed with six control wells, that is, three wells with FAM-TWJ-S1 alone (to define the 100% FAM emission) and three wells with the three separated strands (a mixture termed M) to verify the lack of spontaneous TWJ folding. The microplate is centrifuged (30 s) and then gently stirred for 1 h at 37 °C prior to fluorescence reading. The proficiency of candidates to shift the equilibrium toward the folded TWJ is quantified by comparing the normalized FAM intensity (NFI) of FAM-TWJ-S1 alone (defined as 100%) versus [FAM-TWJ-S1 + ligand] (for discarding unwarranted compound’s interaction with S1) on the one hand and the NFI of the mixture M vs [M + ligand] (that quantifies the TWJ folding per se) on the other hand. Collected results are shown in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B and <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>. The vast majority of the 1200 compounds screened were found ineffective (NFI<sub>M–[M+ligand]</sub> → 0%, NFI<sub>[FAM-TWJ-S1+ligand]</sub> → 100%), but 15 candidates emerged, displaying NFI<sub>M–[M+ligand]</sub> values > 20% (red dashed line, <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B) (the complete TWJ-screen results are provided in the <a href="/doi/suppl/10.1021/acs.jmedchem.8b01978/suppl_file/jm8b01978_si_001.pdf" class="ext-link">Supporting Information</a>). These ligands, compounds <b>414</b>, <b>415</b>, <b>439</b>, <b>444</b>, <b>448</b>, <b>451</b>, <b>459</b>, <b>460</b>, <b>471</b>, <b>473</b>, <b>543</b>, <b>544</b>, <b>545</b>, <b>546</b>, and <b>562</b>, illustrated all possible scenarios, from candidates that display a good ability to fold TWJ without interacting with the FAM label (flagged as “++++” in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>; e.g., <b>451</b>, with NFI<sub>M–[M+ligand]</sub> = 42% and NFI<sub>[FAM-TWJ-S1+ligand]</sub> > 100%) to compounds with very high NFI<sub>M–[M+ligand]</sub> and very strong interaction with the FAM label (flagged as “-” in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>; e.g., <b>545</b>, with NFI<sub>M–[M+ligand]</sub> = 86% and NFI<sub>[FAM-TWJ-S1+ligand]</sub> = 8%). The TWJ-screen assay thus allowed for sorting molecules in a very stringent manner. The chemical structures of the 15 candidates are shown in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>C. It is unsurprising to find previously identified hits,<a onclick="showRef(event, 'ref18 ref19'); return false;" href="javascript:void(0);" class="ref ref18 ref19">(18,19)</a> including the azacyclophanes 1,5-BisNP-O (<b>451</b>), 2,7-BisA-O (<b>459</b>), 2,7-Bis-NP-NH (<b>460</b>), and 4,5-BisA-O (<b>473</b>) and the azacryptand 3,3′-TrisBP (<b>471</b>)<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> and the metallacages Cube-DONQ (<b>444</b>) and Prism-DONQ (<b>448</b>).<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> We also identified new candidates (all from the ICMUB library) that all display well-defined molecular shapes and volumes: the ruthenium trisdiimine complex NQ418 (<b>439</b>); two new triptycene derivatives XX-67 (<b>543</b>) and XX-32 (<b>544</b>), whose synthesis will be described elsewhere (Granzhan et al.), structurally close to the compounds previously studied by Chenoweth and co-workers;<a onclick="showRef(event, 'ref25 ref26'); return false;" href="javascript:void(0);" class="ref ref25 ref26">(25,26)</a> and a newly reported organometallic cage Rec-(AN)-DONQ (<b>562</b>).<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> Classical DNA intercalators are also identified, notably two porphyrins, TMPyP4 (<b>545</b>)<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> and TEGPy (<b>546</b>),<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> and two cationic pyrene derivatives, AZ03 (<b>414</b>) and AZ04 (<b>415</b>),<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> owing to their innate ability to interact with negatively charged nucleic acids.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Top TWJ Ligand Selection on the Basis of TWJ-Screen, PAGE, and FRET-Melting Assays</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center" char=".">TWJ-Screen<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></th><th class="rowsep1 colsep0" align="center">PAGE<a class="ref internalNav" href="#t1fn3" aria-label="c">c</a></th><th class="rowsep1 colsep0" colspan="4" align="center">FRET-melting assay</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">ID</th><th class="colsep0 rowsep0" align="center" char=".">NFI<sub>M–[M+ligand]</sub> (%)</th><th class="colsep0 rowsep0" align="center">NFI<sub>[S1+ligand]</sub><a class="ref internalNav" href="#t1fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center">cplx.</th><th class="colsep0 rowsep0" align="center">Δ<i>T</i><sub>1/2</sub> (°C)</th><th class="colsep0 rowsep0" align="center">Δ<i>T</i><sub>1/2,15eq., ds26</sub></th><th class="colsep0 rowsep0" align="center">Δ<i>T</i><sub>1/2,50eq., ds26</sub></th><th class="colsep0 rowsep0" align="center"><sup>FRET</sup><i>S</i><sub>50eq., ds26</sub><a class="ref internalNav" href="#t1fn4" aria-label="d">d</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>545</b></td><td class="colsep0 rowsep0" align="char" char=".">86</td><td class="colsep0 rowsep0" align="left">8% (-)</td><td class="colsep0 rowsep0" align="left">nc</td><td class="colsep0 rowsep0" align="left">22.5</td><td class="colsep0 rowsep0" align="left">12.0</td><td class="colsep0 rowsep0" align="left">8.0 °C</td><td class="colsep0 rowsep0" align="left">35% (-)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>546</b></td><td class="colsep0 rowsep0" align="char" char=".">72</td><td class="colsep0 rowsep0" align="left">27% (+)</td><td class="colsep0 rowsep0" align="left">nc</td><td class="colsep0 rowsep0" align="left">11.0</td><td class="colsep0 rowsep0" align="left">7.0 °C</td><td class="colsep0 rowsep0" align="left">4.0 °C</td><td class="colsep0 rowsep0" align="left">36% (-)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>543</b></td><td class="colsep0 rowsep0" align="char" char=".">63</td><td class="colsep0 rowsep0" align="left">40% (++)</td><td class="colsep0 rowsep0" align="left">-</td><td class="colsep0 rowsep0" align="left">0.8</td><td class="colsep0 rowsep0" align="left">nd</td><td class="colsep0 rowsep0" align="left">nd</td><td class="colsep0 rowsep0" align="left">nd</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>448</b></td><td class="colsep0 rowsep0" align="char" char=".">55</td><td class="colsep0 rowsep0" align="left">21% (+)</td><td class="colsep0 rowsep0" align="left">nc</td><td class="colsep0 rowsep0" align="left">15.8</td><td class="colsep0 rowsep0" align="left">8.7 °C</td><td class="colsep0 rowsep0" align="left">8.1 °C</td><td class="colsep0 rowsep0" align="left">51% (+)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>459</b></td><td class="colsep0 rowsep0" align="char" char=".">54</td><td class="colsep0 rowsep0" align="left">27% (+)</td><td class="colsep0 rowsep0" align="left">nc</td><td class="colsep0 rowsep0" align="left">23.0</td><td class="colsep0 rowsep0" align="left">20.4 °C</td><td class="colsep0 rowsep0" align="left">17.1 °C</td><td class="colsep0 rowsep0" align="left">84% (+++)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>460</b></td><td class="colsep0 rowsep0" align="char" char=".">46</td><td class="colsep0 rowsep0" align="left">37% (++)</td><td class="colsep0 rowsep0" align="left">+</td><td class="colsep0 rowsep0" align="left">4.2</td><td class="colsep0 rowsep0" align="left">nd<a class="ref internalNav" href="#t1fn5" aria-label="e">e</a></td><td class="colsep0 rowsep0" align="left">nd</td><td class="colsep0 rowsep0" align="left">nd</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>444</b></td><td class="colsep0 rowsep0" align="char" char=".">43</td><td class="colsep0 rowsep0" align="left">39% (++)</td><td class="colsep0 rowsep0" align="left">+</td><td class="colsep0 rowsep0" align="left">4.9</td><td class="colsep0 rowsep0" align="left">nd</td><td class="colsep0 rowsep0" align="left">nd</td><td class="colsep0 rowsep0" align="left">nd</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>451</b></td><td class="colsep0 rowsep0" align="char" char=".">42</td><td class="colsep0 rowsep0" align="left">118% (++++)</td><td class="colsep0 rowsep0" align="left">+</td><td class="colsep0 rowsep0" align="left">10.4</td><td class="colsep0 rowsep0" align="left">8.9 °C</td><td class="colsep0 rowsep0" align="left">8.5 °C</td><td class="colsep0 rowsep0" align="left">82% (+++)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>473</b></td><td class="colsep0 rowsep0" align="char" char=".">35</td><td class="colsep0 rowsep0" align="left">80% (+++)</td><td class="colsep0 rowsep0" align="left">nc</td><td class="colsep0 rowsep0" align="left">11.6</td><td class="colsep0 rowsep0" align="left">5.5 °C</td><td class="colsep0 rowsep0" align="left">4.6 °C</td><td class="colsep0 rowsep0" align="left">39% (-)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>471</b></td><td class="colsep0 rowsep0" align="char" char=".">33</td><td class="colsep0 rowsep0" align="left">109% (++++)</td><td class="colsep0 rowsep0" align="left">+</td><td class="colsep0 rowsep0" align="left">17.5</td><td class="colsep0 rowsep0" align="left">16.6 °C</td><td class="colsep0 rowsep0" align="left">15.9 °C</td><td class="colsep0 rowsep0" align="left">91% (+++)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>439</b></td><td class="colsep0 rowsep0" align="char" char=".">28</td><td class="colsep0 rowsep0" align="left">88% (+++)</td><td class="colsep0 rowsep0" align="left">-</td><td class="colsep0 rowsep0" align="left">1.8</td><td class="colsep0 rowsep0" align="left">nd</td><td class="colsep0 rowsep0" align="left">nd</td><td class="colsep0 rowsep0" align="left">nd</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>544</b></td><td class="colsep0 rowsep0" align="char" char=".">27</td><td class="colsep0 rowsep0" align="left">100% (++++)</td><td class="colsep0 rowsep0" align="left">-</td><td class="colsep0 rowsep0" align="left">0.6</td><td class="colsep0 rowsep0" align="left">nd</td><td class="colsep0 rowsep0" align="left">nd</td><td class="colsep0 rowsep0" align="left">nd</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>415</b></td><td class="colsep0 rowsep0" align="char" char=".">23</td><td class="colsep0 rowsep0" align="left">100% (++++)</td><td class="colsep0 rowsep0" align="left">-</td><td class="colsep0 rowsep0" align="left">0</td><td class="colsep0 rowsep0" align="left">nd</td><td class="colsep0 rowsep0" align="left">nd</td><td class="colsep0 rowsep0" align="left">nd</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>562</b></td><td class="colsep0 rowsep0" align="char" char=".">23</td><td class="colsep0 rowsep0" align="left">56% (+++)</td><td class="colsep0 rowsep0" align="left">-</td><td class="colsep0 rowsep0" align="left">2.0</td><td class="colsep0 rowsep0" align="left">nd</td><td class="colsep0 rowsep0" align="left">nd</td><td class="colsep0 rowsep0" align="left">nd</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>414</b></td><td class="colsep0 rowsep0" align="char" char=".">22</td><td class="colsep0 rowsep0" align="left">101% (+++)</td><td class="colsep0 rowsep0" align="left">-</td><td class="colsep0 rowsep0" align="left">1.5</td><td class="colsep0 rowsep0" align="left">nd</td><td class="colsep0 rowsep0" align="left">nd</td><td class="colsep0 rowsep0" align="left">nd</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">Selected candidates with NFI<sub>M–[M+ligand]</sub> > 20%.</p></div><div class="footnote" id="t1fn2"><sup><sup>b</sup></sup><p class="last">Interaction with the FAM label: “-” for NFI<sub>[S1+ligand]</sub> < 10%, “+” for NFI<sub>[S1+ligand]</sub> = 10–30%, “++” for NFI<sub>[S1+ligand]</sub> = 30–50%, “+++” for NFI<sub>[S1+ligand]</sub> = 50–100%, and “++++” for NFI<sub>[S1+ligand]</sub> > 100%.</p></div><div class="footnote" id="t1fn3"><sup><sup>c</sup></sup><p class="last">Qualitative PAGE analysis: “nc” for non conclusive, “-” for no detectable TWJ/ligand complex, and “+” for detectable TWJ/ligand complex.</p></div><div class="footnote" id="t1fn4"><sup><sup>d</sup></sup><p class="last">Selectivity determined by FRET: “-” for <sup>FRET</sup><i>S</i> < 50%, “+” for <sup>FRET</sup><i>S</i> = 50–80%, and “+++” for <sup>FRET</sup><i>S</i> > 80%.</p></div><div class="footnote" id="t1fn5"><sup><sup>e</sup></sup><p class="last">“nd” for not determined (Δ<i>T</i><sub>1/2</sub> < 5 °C).</p></div></div></div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Confirming the TWJ-Folding Capability via Nondenaturing PAGE Analyses</h3><div class="NLM_p">To further investigate the intrinsic quality of the 15 identified candidates, their TWJ-folding capabilities were subsequently investigated by nondenaturing PAGE analysis according to the methodology described by Brabec et al.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> We used the same TWJ-forming strands, TWJ-S1, TWJ-S2, and TWJ-S3 (without fluorescent labels). The obtained gels (15% polyacrylamide, 5 μM DNA loading/well), presented in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A, offered a straightforward—but qualitative—reading of the TWJ-folding ability of the candidates, visualized by the difference of migration between the TWJ-S1 alone and the mixture of strands M (lower dashed rectangle) and the folded TWJ/ligand complexes, which migrate significantly more slowly (upper dashed rectangle). Five compounds were found efficient in these conditions (red arrows), i.e., <b>444</b>, <b>448</b>, <b>451</b>, <b>460</b>, and <b>471</b>; four compounds led to a smeared band (green arrows), i.e., <b>459</b>, <b>473</b>, <b>545</b>, and <b>546</b>, thus making conclusions hard to draw solely on the basis of PAGE; and six compounds were found inactive (yellow arrows), i.e., <b>414</b>, <b>415</b>, <b>439</b>, <b>543</b>, <b>544</b>, and <b>562</b>. Given that PAGE conditions are known to be challenging for noncovalent assemblies, these results underline the good TWJ-interacting properties of the five most efficient compounds.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.8b01978/20190502/images/medium/jm-2018-019785_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.8b01978/20190502/images/large/jm-2018-019785_0001.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (A) PAGE analysis performed with TWJ-S1 (5.0 μM) and M (5.0 μM) in the absence and presence of the top 15 ligands (5 mol equiv) in TBE buffer + 100 mM NaCl, pH 8.3, at 4 °C for 1 h (gels were poststained with SybrGold; multiple gels assembled in a single image) (B–D) FRET-melting experiments performed with F-TWJ-T and the top 15 ligands in either a comparative (B, C) or a competitive manner (C, D, i.e., in the presence of increasing amounts of ds26 (up to 50 mol equiv)) expressed as Δ<i>T</i><sub>1/2</sub> values (in °C). Best candidates (Δ<i>T</i><sub>1/2</sub> > 10 °C, (B) and <sup>FRET</sup><i>S</i> > 80% in the presence of 50 mol equiv ds26, (D)) are highlighted in red. Experiments performed with 0.2 μM DNA, without or with 1.0 μM candidates, with or without 3 and 10 μM ds26, in 10 mM lithium cacodylate plus 10 mM KCl/90 mM LiCl, pH 7.2, between 25 and 90 °C. (E) PAGE analysis of competitive experiments performed with TWJ strands, <b>471</b> (2 mol equiv), and increasing amounts of ds12 (up to 18 mol equiv) in TBE buffer + 100 mM NaCl, pH 8.3, at 4 °C for 45 min (gels are poststained with SybrGold for visualizing ds12 and unfolded ds12, i.e., ss12).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.8b01978/20190502/images/large/jm-2018-019785_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01978&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Investigating the TWJ-Interacting Properties via the FRET-Melting Assay</h3><div class="NLM_p">We also assessed whether the 15 identified ligands interact with a prefolded TWJ structure. To this end, their ability to stabilize folded DNA junctions against thermal denaturation was evaluated by the FRET-melting assay using the doubly labeled FAM-d[<sup>5′</sup>A(CT)<sub>2</sub>(TC)<sub>2</sub>G-T<sub>6</sub>-C(GA)<sub>2</sub>GCGAC-T<sub>6</sub>-GTCGC(AG)<sub>2</sub>T<sup>3′</sup>]-TAMRA system (FAM-TWJ-TAMRA, or F-TWJ-T).<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> Experiments were performed as follows: in 100 μL (final volume) of buffered solution (CacoK), 0.2 μM of F-TWJ-T in the absence (control wells) or presence of 5 mol equiv (i.e., 1.0 μM) ligands (as triplicates) was heated from 25 to 75 °C (1 °C per step). The melting of the DNA structure was monitored through FAM emission (quenched at low temperature due to the close proximity with TAMRA in folded TWJ and restored at elevated temperature due to the thermal unfolding of the DNA junction): the thermal stability of F-TWJ-T, expressed as its melting temperature <i>T</i><sub>1/2</sub> (here, <i>T</i><sub>1/2</sub> = 43.4 °C in the absence of a ligand), increased up to 66.4 °C in the presence of the best TWJ stabilizer (compound <b>459</b>, Δ<i>T</i><sub>1/2</sub> = 23.0 °C, <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). The Δ<i>T</i><sub>1/2</sub> values for the 15 selected compounds are reported in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B: only seven of these candidates (<b>448</b>, <b>451</b>, <b>459</b>, <b>471</b>, <b>473</b>, <b>545</b>, and <b>546</b>) imparted a significant stability (Δ<i>T</i><sub>1/2</sub> > 10 °C, <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Compounds <b>444</b> and <b>460</b>, yet the most promising candidates by PAGE, would have been discarded on the FRET-melting result basis (Δ<i>T</i><sub>1/2</sub> = 4.9 and 4.2 °C, respectively); conversely, <b>459</b>, the most efficient TWJ stabilizer here, led to debatable PAGE results. This discrepancy illustrated that TWJ-screen and FRET-melting assays monitored two different processes (TWJ folding on the one hand and stabilization of folded TWJ on the other hand), thus making these two tests complementary.</div><div class="NLM_p">The TWJ-interacting properties of these seven candidates were further investigated via competitive FRET-melting experiments to assess their TWJ selectivity over duplex DNA. Experiments were performed with F-TWJ-T, 5 mol equiv of ligand, in the absence (control wells) or presence of 15 or 50 mol equiv of a duplex competitor, here ds26 (comprising two self-complementary strands ss26 d[<sup>5</sup>CA<sub>2</sub>TCG<sub>2</sub>ATCGA<sub>2</sub>T<sub>2</sub>CGATC<sub>2</sub>GAT<sub>2</sub>G<sup>3′</sup>]). The capacity of a ligand to withstand the excess of ds26 is expressed as <sup>FRET</sup><i>S</i> values (<sup>FRET</sup><i>S</i> = (Δ<i>T</i><sub>1/2</sub>[+ds26]/Δ<i>T</i><sub>1/2</sub>[no ds26]) × 100, in %). Results seen in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>C and <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> indicate that only three compounds are both good stabilizers (Δ<i>T</i><sub>1/2</sub> > 10 °C) and selective (<sup>FRET</sup><i>S</i> > 80%), that is, the azacyclophanes <b>451</b> and <b>459</b> and the azacryptand <b>471</b>, making them the most promising ligands identified through this three-step selection process. These results also discarded the two porphyrins (<b>545</b> and <b>546</b>) as false positives owing to their low selectivities (<sup>FRET</sup><i>S</i> = 35 and 36%, respectively), which makes them prone to interact with nucleic acids irrespective of their secondary structures.</div><div class="NLM_p">Of note, the exquisite selectivity of <b>471</b> (<sup>FRET</sup><i>S</i> = 91%, <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>) was further investigated via a competitive PAGE analysis performed with an excess (2, 9, and 18 mol equiv) of duplex DNA (here ds12, comprising two self-complementary strands ss12 d[<sup>5′</sup>CGCGA<sub>2</sub>T<sub>2</sub>CGCG<sup>3′</sup>]). Results seen in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>E confirmed that the TWJ-folding capability of <b>471</b> withstands the presence of competitive duplexes, thereby further demonstrating its selectivity for DNA junctions.</div><div class="NLM_p last">Collectively, these results highlight the necessity of performing TWJ-screen, PAGE, and FRET-melting evaluations in a stepwise manner to identify promising ligands on a reliable basis. In the present case, three compounds passed this series of tests successfully, namely, the macrocycles <b>451</b> and <b>459</b> (1,5-BisNP-O and 2,7-BisA-O, respectively) and the macrobicycle <b>471</b> (3,3′-TrisBP).</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Quantifying the Ligand–TWJ DNA Association by ESI-MS and Equilibrium Dialysis Analyses</h3><div class="NLM_p">We next investigated further the TWJ-interacting properties of <b>451</b>, <b>459</b>, and <b>471</b> via electrospray ionization mass spectrometry (ESI-MS). This technique provides a unique opportunity to directly access both the equilibrium affinity constants (<i>K</i>) and the stoichiometry of the ligand–DNA association as already documented with duplex- and quadruplex DNA<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> but not yet applied to DNA junctions. Investigations were here performed with both the prefolded TWJ used for FRET-melting investigations (without fluorescent labels) and the duplex DNA ds17 (d[<sup>5′</sup>C<sub>2</sub>AGT<sub>2</sub>CGTAGTA<sub>2</sub>C<sub>3</sub><sup>3′</sup>]/d[<sup>5′</sup>G<sub>3</sub>T<sub>2</sub>ACTACGA<sub>2</sub>CTG<sub>2</sub><sup>3′</sup>]). Measurements were performed in 80:20 (v/v) ammonium acetate buffer (100 mM, pH 7.0)/MeOH with 5 μM DNA and increasing amounts of ligands (from 0 to 2 mol equiv). Results obtained with <b>471</b> (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A; see the <a href="/doi/suppl/10.1021/acs.jmedchem.8b01978/suppl_file/jm8b01978_si_001.pdf" class="ext-link">Supporting Information</a> for compounds <b>451</b> and <b>459</b>) showed the exclusive formation of the 1:1 TWJ/<b>471</b> complex, regardless of the DNA/ligand ratio, and no interaction with ds17. These results highlighted a unique ligand binding site within the TWJ structure—most likely the branch point—and confirmed the exquisite TWJ selectivity of <b>471</b>. Similar experiments performed with <b>451</b> and <b>459</b> showed that the major species is the 1:1 TWJ/ligand complexes as well, along with traces of free DNA and 2:1 complexes, and also highlighted weak and random interactions with ds17 (since 1:1, 2:1, and 3:1 complexes could be detected). To further characterize and quantify these interactions, we calculated the equilibrium association constants (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>):<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a><i>K</i> values confirmed the high affinities of <b>451</b>, <b>459</b>, and <b>471</b> for TWJ (<i>K</i> between 3.9 × 10<sup>6</sup> and 1.9 × 10<sup>9</sup> M<sup>–1</sup>) along with their excellent selectivity over duplex DNA (>2-log difference).</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.8b01978/20190502/images/medium/jm-2018-019785_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.8b01978/20190502/images/large/jm-2018-019785_0002.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. (A) ESI-MS profiles of the association of TWJ (left panels) or ds17 (right panels) with increasing amounts (0, 1, and 2 mol equiv) of <b>471</b>. Experiments performed with 5.0 μM DNA, without or with 5.0 or 10 μM ligand, in 100 mM ammonium acetate, pH 7.2, after 1 h equilibration at 25 °C. (B) Equilibrium dialysis results for <b>451</b>, <b>459</b>, and <b>471</b> (black bars, free ligand; brown bar, ligand bound to TWJ; orange bar, ligand bound to ds17). Experiments performed with 75 μM DNA (base pairs), without or with 2 μM ligand, in sodium phosphate + 100 mM NaCl, pH 7.2, after 24 h equilibration at 25 °C. (C) Viability profiles obtained via the sulforhodamine B (SRB) assay with <b>451</b>, <b>459</b>, and <b>471</b> after 72 h incubation (37 °C, Dulbecco’s modified Eagle’s medium (DMEM)) with MCF7 (brow line) and MDA-MB-231 (orange line) cells.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.8b01978/20190502/images/large/jm-2018-019785_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01978&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Quantification of DNA Interactions (ESI-MS and Dialysis) and Toxicity (SRB) of the Top 3 TWJ Ligands</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="×" /></col><col align="char" char="×" /></col><col align="left" /></col><col align="char" char="×" /></col><col align="char" char="×" /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="3" align="center" char="×">ESI-MS (<i>K</i>, M<sup>–1</sup>)<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></th><th class="rowsep1 colsep0" colspan="3" align="center" char="×">Equilibrium dialysis (<i>K</i><sub>app</sub>, M<sup>–1</sup>)</th><th class="rowsep1 colsep0" colspan="3" align="center" char=".">SRB test (LD<sub>50</sub>, μM)</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">ID</th><th class="colsep0 rowsep0" align="center" char="×">TWJ</th><th class="colsep0 rowsep0" align="center" char="×">ds</th><th class="colsep0 rowsep0" align="center">TWJ/ds</th><th class="colsep0 rowsep0" align="center" char="×">TWJ</th><th class="colsep0 rowsep0" align="center" char="×">ds</th><th class="colsep0 rowsep0" align="center">TWJ/ds</th><th class="colsep0 rowsep0" align="center" char=".">MCF7</th><th class="colsep0 rowsep0" align="center" char=".">MDA</th><th class="colsep0 rowsep0" align="center" char=".">BJ-hTERT</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>451</b></td><td class="colsep0 rowsep0" align="char" char="×">4.7 × 10<sup>7</sup></td><td class="colsep0 rowsep0" align="char" char="×">7.5 × 10<sup>4</sup></td><td class="colsep0 rowsep0" align="left">>2-log</td><td class="colsep0 rowsep0" align="char" char="×">8.7 × 10<sup>6</sup></td><td class="colsep0 rowsep0" align="char" char="×">5.9 × 10<sup>5</sup></td><td class="colsep0 rowsep0" align="left">>1-log</td><td class="colsep0 rowsep0" align="char" char=".">0.10</td><td class="colsep0 rowsep0" align="char" char=".">0.16</td><td class="colsep0 rowsep0" align="char" char=".">0.04</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>459</b></td><td class="colsep0 rowsep0" align="char" char="×">1.9 × 10<sup>9</sup></td><td class="colsep0 rowsep0" align="char" char="×">3.1 × 10<sup>4</sup></td><td class="colsep0 rowsep0" align="left">>4-log</td><td class="colsep0 rowsep0" align="char" char="×">1.4 × 10<sup>7</sup></td><td class="colsep0 rowsep0" align="char" char="×">3.8 × 10<sup>5</sup></td><td class="colsep0 rowsep0" align="left">>1-log</td><td class="colsep0 rowsep0" align="char" char=".">4.80</td><td class="colsep0 rowsep0" align="char" char=".">2.80</td><td class="colsep0 rowsep0" align="char" char=".">1.00</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>471</b></td><td class="colsep0 rowsep0" align="char" char="×">3.9 × 10<sup>6</sup></td><td class="colsep0 rowsep0" align="char" char="×">7.5 × 10<sup>3 </sup><a class="ref internalNav" href="#t2fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="left">>2-log<a class="ref internalNav" href="#t2fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="char" char="×">6.0 × 10<sup>6</sup></td><td class="colsep0 rowsep0" align="char" char="×">4.7 × 10<sup>4</sup></td><td class="colsep0 rowsep0" align="left">>2-log</td><td class="colsep0 rowsep0" align="char" char=".">1.30</td><td class="colsep0 rowsep0" align="char" char=".">1.30</td><td class="colsep0 rowsep0" align="char" char=".">2.60</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last"><i>K</i> calculations on the basis of experiments performed with 10 μM DNA and 20 μM ligands.</p></div><div class="footnote" id="t2fn2"><sup><sup>b</sup></sup><p class="last">No ds17/<b>471</b> interactions were detected at the 1:2 ratio; <i>K</i><sub>a</sub> was calculated for experiments performed at a 1:3 DNA/ligand ratio.</p></div></div></div><div class="NLM_p last">We next decided to confirm these results by an alternative technique, performed in solution, and selected the equilibrium dialysis (or competition dialysis) that has been already applied to a wide variety of DNA structures (single-stranded and double-stranded DNA, triplex- and quadruplex DNA)<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> but not to DNA junctions. In this experiment, both TWJ and ds17, isolated from the main solution in semipermeable chambers (200 μL at 75 μM, expressed in base pairs), were dialyzed against a solution of ligands (200 mL at 2 μM). After equilibration (24 h, 25 °C), the amount of ligand bound to each structure is quantified by fluorescence titration of the content of the dialysis chambers after leaking the ligand by 1% sodium dodecyl sulfate (SDS) treatments (versus a quantification of the ligand remaining in the dialysis solution). Results presented in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B show the preferred fixation of the ligands onto TWJ but also emphasized the better selectivity of <b>471</b> as compared to <b>451</b> and <b>459</b>. Again, we quantified these interactions through the calculations of the apparent affinity constant <i>K</i><sub>app</sub> (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>):<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> they confirmed the high affinities of <b>451</b>, <b>459</b>, and <b>471</b> for TWJ (<i>K</i><sub>app</sub> between 6.0 × 10<sup>6</sup> and 1.4 × 10<sup>7</sup> M<sup>–1</sup>) along with their good (<b>451</b> and <b>459</b>) to excellent selectivity over duplex DNA (>2-log difference for <b>471</b>).</div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Preliminary Cell-Based Assays with <b>451</b>, <b>459</b>, and <b>471</b></h3><div class="NLM_p">After the thorough evaluation of their TWJ-interacting properties in vitro, we decided to evaluate the antiproliferative properties of <b>451</b>, <b>459</b>, and <b>471</b> against cancer cells. Their cellular activities were evaluated on two cancer cell lines, MCF7 and MDA-MB-231, which are extensively used in the study of experimental therapeutics since they represent two different classes of breast cancers: MCF7 cells are derived from a hormone-responsive cancer, expressing estrogen (ER), progesterone (PR), and glucocorticoid receptors (GRs); and MDA-MB-231 cells are representative of triple-negative breast cancers (ER<sup>–</sup>, PR<sup>–</sup>, and HER2<sup>–</sup> (human epidermal growth factor receptor 2)), which are typically more aggressive and challenging to cure. In addition, MCF7 and MDA-MB-231 are instrumental for the evaluation of DNA-damaging agents since they have functional and mutant p53, respectively, a critical protein controlling DNA damage checkpoints and mediating cell-cycle arrests and/or apoptosis in response to DNA damage. Viability analysis was performed here through the sulforhodamine B (SRB) test,<a onclick="showRef(event, 'ref41 ref42'); return false;" href="javascript:void(0);" class="ref ref41 ref42">(41,42)</a> and the collected results were compared to the firmly established anticancer (and DNA-damaging) agent camptothecin (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01978/suppl_file/jm8b01978_si_001.pdf" class="ext-link">Figure S4</a>).<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> To this end, cells were incubated for 72 h at 37 °C in DMEM without or with a ligand prior to fixation with a solution of trichloroacetic acid 10% (1 h at 4 °C). After supernatant removal, a solution of SRB (0.2% in 1% acetic acid) was added for 30 min before being washed (acetic acid 1%) and dried. A solution of Tris base (10 mM) was then added to allow for optical density (OD) reading (at 530 nm) to assess cell viability and determine the lethal dose (LD<sub>50</sub>, the concentration at which 50% of the cell growth inhibition is reached).</div><div class="NLM_p last">The results (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>C and <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>) showed that the cellular activity stood in the same range for both cells lines, with higher inhibition properties for <b>451</b> (with LD<sub>50</sub> = 0.10 and 0.16 μM for MCF7 and MDA-MB-231, respectively), then for <b>471</b> (LD<sub>50</sub> = 1.30 and 1.30 μM) and for <b>459</b> (LD<sub>50</sub> = 4.80 and 2.80 μM). In these conditions, <b>451</b> displayed a higher activity than camptothecin (with LD<sub>50</sub> = 1.70 and 1.40 μM), which is here found comparable to <b>471</b>. For the sake of comparison, we also performed SRB investigations with normal fibroblasts (BJ-hTERT) as models of healthy cells.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> As seen in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>, only <b>471</b> was found less toxic in BJ-hTERT than in cancer cells (2-fold difference, with LD<sub>50</sub> = 1.30 and 2.60 μM for cancer cells and BJ-hTERT, respectively), making it the most promising compound from that series of ligands. To gain insights into the capacity of <b>471</b> to trigger DNA damage, we performed preliminary immunodetection studies using an antibody raised against the phosphorylated histone H2AX (so-called γH2AX), an established marker of DNA damage.<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> The pilot study was carried out with MCF7 treated with <b>471</b> and camptothecin as the control. Collected images seen in <a href="/doi/suppl/10.1021/acs.jmedchem.8b01978/suppl_file/jm8b01978_si_001.pdf" class="ext-link">Figure S5</a> confirmed that the ligands triggered a marked accumulation of γH2AX foci. These preliminary data thus provide an encouraging sign for the future of this new class of noncanonical DNA structure-targeting ligands as DNA-damaging agents, and efforts are now invested to establish a firm link between cellular TWJ interaction and DNA damage.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i12">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_33336" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_33336" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Compared to normal cells, cancer cells harbor a flawed repertoire of DNA damage signaling and repair capabilities, collectively known as DNA damage response, or DDR.<a onclick="showRef(event, 'ref1 ref2 ref3 ref4 ref5 ref6'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3 ref4 ref5 ref6">(1−6)</a> Three main aspects of DDR make cancer cells different from healthy cells: (i) loss of one or more DDR pathways, (ii) increased level of replicative stress, and (iii) endogenous DNA damages.<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> These aspects thus provide a window of therapeutic opportunities for compounds that trigger or exacerbate cellular crisis states. Our approach is here focused on compounds that stabilize three-way DNA junctions (TWJ) given that these higher-order DNA structures represent topological hindrances to polymerase that can hamper proper DNA transactions (replication, transcription).<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> The TWJ ligands could thus represent a novel and promising class of crisis-inducing agents.</div><div class="NLM_p">To identify promising TWJ ligands, we developed and report here a validation framework that comprises six complementary in vitro assays: the TWJ-screen assay, which allowed for evaluating the ability to promote the TWJ folding of 1200 compounds; a PAGE analysis, which confirms the TWJ-folding ability of the 15 identified candidates in more stringent conditions; the FRET-melting assay, which provides insights into both the TWJ-stabilizing properties and selectivity of the identified candidates; ESI-MS investigations, which further studied TWJ selectivity and gives access to a quantification (affinity constants, <i>K</i><sub>a</sub>) of the ligand–DNA association; the equilibrium dialysis, which provides an alternative quantification of the ligand–DNA binding parameters; and the SRB test, which assesses the antiproliferative activity of identified TWJ ligands against human breast cancer cells (MCF7 and MDA-MB-231).</div><div class="NLM_p last">Collectively, the wealth of data collected in the course of this study highlights the strength of our methodology to uncover TWJ ligands, with the identification of three promising compounds here, namely, <b>451</b>, <b>459</b>, and <b>471</b>, whose properties make them ideally suited to be used as prototype ligands throughout the validation chain. Exploring their use as anticancer agents will require to decipher the mechanisms underlying their antiproliferative properties. Also, our results highlight the need for screening wider chemical libraries to uncover ever more efficient TWJ ligands, and we are convinced that the framework described here is an ideal toolbox to do so. Importantly, libraries to be tested must not be focused on druglike compounds or small molecules only (like the Pathogen Box or the CNE) but must include chemicals with defined shapes and volumes, given that it appears to be a major determinant of efficient TWJ ligands. Indeed, thanks to the pioneering investigations by Hannon and co-workers performed with iron supramolecular cylinders,<a onclick="showRef(event, 'ref20 ref21 ref22 ref23 ref24'); return false;" href="javascript:void(0);" class="ref ref20 ref21 ref22 ref23 ref24">(20−24)</a> we know that the privileged binding site for compounds within the TWJ structure is the branch point, a cavity that results from the convergence of three duplex arms. This cavity is hydrophobic owing to its side walls made of base pairs (here two T = A and one G≡C base pairs), thus being suited to accommodate sterically demanding ligands displaying large aromatic moieties. This was confirmed here through the unveiling of three promising candidates, the azacyclophanes <b>451</b> and <b>459</b> (1,5-BisNP-O and 2,7-BisA-O, respectively) and the azacryptand <b>471</b> (3,3′-TrisBP), all of which display well-defined three-dimensional structures and large aromatic moieties (naphthalene, acridine, and bipyridine) prone to interacting efficiently with the cavity walls. These results thus provide new perspectives on and further insights into the cellular activity of the azacyclophanes and azacryptands, which could originate in multiple dysregulated pathways (TWJ interaction as well as interactions with abasic sites and cleavage and inhibition of DNA repair).<a onclick="showRef(event, 'ref47 ref48'); return false;" href="javascript:void(0);" class="ref ref47 ref48">(47,48)</a> They also contribute to expand the portfolio of DNA binders with promising anticancer properties and the panoply of DNA structure targets with high therapeutic value.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i13">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_50426" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_50426" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Chemicals</h3><div class="NLM_p last">The open access Pathogen Box comprises 400 compounds with biological activity against specific pathogenic organisms. The box is provided by the Medicines for Malaria Venture as an open access compound collection (<a href="http://www.pathogenbox.org/" class="extLink">http://www.pathogenbox.org/</a>). The chimiothèque nationale essentielle is a collection of 640 compounds assembled by and purchased from the French National chemolibrary (<a href="http://chimiotheque-nationale.cn.cnrs.fr/" class="extLink">http://chimiotheque-nationale.cn.cnrs.fr/</a>). All other chemicals have been prepared according to methodologies described (with purity ≥95% as evaluated by elemental analyses or high-performance liquid chromatography–-mass spectrometry) in the corresponding references, except camptothecin (99.8% purity), purchased from Selleckchem.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Oligonucleotides</h3><div class="NLM_p">The lyophilized DNA sequences (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>), purchased from Eurogentec (Seraing, Belgium), were first diluted at 500 μM in deionized water (18.2 MΩ cm resistivity). The actual concentration of each DNA solution was determined after a dilution to 1 μM theoretical concentration through UV spectral analysis at 260 nm (after 5 min at 90 °C) with the molar extinction coefficient values provided by the manufacturer. Separated strands (FAM-TWJ-S1, TWJ-S1, TWJ-S2, TWJ-S3-TAMRA, and TWJ-S3) were subsequently diluted in a CacoK buffer (10 mM lithium cacodylate buffer plus 10 mM KCl/90 mM LiCl pH 7.2) at 2 μM for TWJ-screen and at 9 μM for PAGE experiments. For FRET-melting experiments, the higher-order DNA structures of both FAM-TWJ-TAMRA and ds26 were prepared by mixing 40 μL of the constitutive strand (500 μM) with 8 μL of a lithium cacodylate buffer solution (100 mM, pH 7.2), plus 8 μL of a KCl/LiCl solution (100/900 mM) and 24 μL of water. For ESI-MS analysis, the higher-order DNA structure was prepared by mixing 17 μL of TWJ (500 μM) with 17 μL of ammonium acetate buffer (1 M, pH 7.0) and 136 μL of water. The higher-order structures were folded according to two procedures: (a) for FAM-TWJ-TAMRA (FRET-melting) and TWJ (ESI-MS), solutions were heated (90 °C, 5 min), cooled on ice (FRET-melting) or to 25 °C gradually (ESI-MS), and then stored at least overnight (4 °C); and (b) for ds26, the solutions were heated (90 °C, 5 min), gradually cooled (65, 60, 55, 50, 40, and 30 °C (60 min per step), 25 °C (2 h)), and then stored overnight (4 °C).</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Oligonucleotides Used in this Study</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">status</th><th class="colsep0 rowsep0" align="center">nature</th><th class="colsep0 rowsep0" align="center">name</th><th class="colsep0 rowsep0" align="center">sequence</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">labeled</td><td class="colsep0 rowsep0" align="left">TWJ</td><td class="colsep0 rowsep0" align="left">FAM-TWJ-S1</td><td class="colsep0 rowsep0" align="left">FAM-d[<sup>5</sup><sup>′</sup>CG<sub>2</sub>A<sub>2</sub>CG<sub>2</sub>CACTCG<sup>3</sup><sup>′</sup>]</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">TWJ-S3-TAMRA</td><td class="colsep0 rowsep0" align="left">d[<sup>5</sup><sup>′</sup>C<sub>2</sub>ACGCTCGT<sub>2</sub>C<sub>2</sub>G<sup>3</sup><sup>′</sup>]-TAMRA</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">F-TWJ-T</td><td class="colsep0 rowsep0" align="left">FAM-d[<sup>5</sup><sup>′</sup>A(CT)<sub>2</sub>(TC)<sub>2</sub>G-T<sub>6</sub>-C(GA)<sub>2</sub>GCGAC-T<sub>6</sub>-GTCGC(AG)<sub>2</sub>T<sup>3</sup><sup>′</sup>]-TAMRA</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">unlabeled</td><td class="colsep0 rowsep0" align="left">TWJ</td><td class="colsep0 rowsep0" align="left">TWJ-S1</td><td class="colsep0 rowsep0" align="left">d[<sup>5</sup><sup>′</sup>CG<sub>2</sub>A<sub>2</sub>CG<sub>2</sub>CACTCG<sup>3</sup><sup>′</sup>]</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">TWJ-S2</td><td class="colsep0 rowsep0" align="left">d[<sup>5</sup><sup>′</sup>CGAGTGCAGCGTG<sub>2</sub><sup>3</sup><sup>′</sup>]</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">TWJ-S3</td><td class="colsep0 rowsep0" align="left">d[<sup>5</sup><sup>′</sup>C<sub>2</sub>ACGCTCGT<sub>2</sub>C<sub>2</sub>G<sup>3</sup><sup>′</sup>]</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">TWJ</td><td class="colsep0 rowsep0" align="left">d[<sup>5</sup><sup>′</sup>A(CT)<sub>2</sub>(TC)<sub>2</sub>G-T<sub>6</sub>-C(GA)<sub>2</sub>GCGAC-T<sub>6</sub>-GTCGC(AG)<sub>2</sub>T<sup>3</sup><sup>′</sup>]</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">duplex</td><td class="colsep0 rowsep0" align="left">ds12</td><td class="colsep0 rowsep0" align="left">strand 1: d[<sup>5</sup><sup>′</sup>CGCGA<sub>2</sub>T<sub>2</sub>CGCG<sup>3</sup><sup>′</sup>]</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">strand 2: d[<sup>5</sup><sup>′</sup>CGCGA<sub>2</sub>T<sub>2</sub>CGCG<sup>3</sup><sup>′</sup>]</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">ds17</td><td class="colsep0 rowsep0" align="left">strand 1: d[<sup>5</sup><sup>′</sup>CCAGTTCGTAGTAACCC<sup>3</sup><sup>′</sup>]</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">strand 2: d[<sup>5</sup><sup>′</sup>GGGTTACTACGAACTGG<sup>3</sup><sup>′</sup>]</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">ds26</td><td class="colsep0 rowsep0" align="left">strand 1: d[<sup>5</sup><sup>′</sup>CAATCGGATCGAATTCGATCCGATTG<sup>3</sup><sup>′</sup>]</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">strand 2: d[<sup>5</sup><sup>′</sup>GTTAGCCTAGCTTAAGCTAGGCTAAC<sup>3</sup><sup>′</sup>]</td></tr></tbody></table></div></div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> TWJ-Screen Assay</h3><div class="NLM_p last">Experiments are performed in a 96-well format plate (Greiner, F-bottom black) using a BMG Labtech ClarioStar equipped with FAM filters (λ<sub>ex</sub> = 492 nm; λ<sub>em</sub> = 516 nm) at 37 °C. Experiments are performed in CacoK buffer (10 mM lithium cacodylate plus 10 mM KCl/90 mM LiCl, pH 7.2, final volume: 100 μL per well) with 1 μM ligand (10 μL of 10 μM solution) and 0.2 μM DNA (stepwise addition of 10 μL of 2 μM solution of FAM-TWJ-S1, TWJ-S2, and TWJ-S3-TAMRA). The microplate is centrifuged quickly (30 s) and then placed into the ClarioStar. The FAM fluorescence is monitored upon gentle stirring at 37 °C every 5 min during 1 h. Final data are analyzed by using Excel (Microsoft Corp.) and OriginPro9 (OriginLab Corp.): the results are expressed as normalized fluorescence intensity (NFI) values collected at 60 min. The efficiency of ligands to fold TWJ is quantified by comparing the NFI of FAM-TWJ-S1 alone (defined as 100%) versus [FAM-TWJ-S1 + ligand] (for discarding unwarranted ligand’s interaction with FAM-TWJ-S1) on the one hand and the NFI of the mixture M ([FAM-TWJ-S1 + TWJ-S2 + TWJ-S3-TAMRA]) versus [M + ligand] (that quantifies the TWJ folding per se) on the other hand. Reported NFI values are means of three experiments.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> FRET-Melting Assay</h3><div class="NLM_p last">Experiments are performed in a 96-well format plate (Agilent) using Agilent Stratagene Mx3005P equipped with FAM filters (λ<sub>ex</sub> = 492 nm; λ<sub>em</sub> = 516 nm) from 25 to 90 °C. Experiments are performed in CacoK buffer (10 mM lithium cacodylate plus 10 mM KCl/90 mM LiCl, pH 7.2, final volume: 100 μL per well) with 0.2 μM DNA (the labeled sequence FAM-TWJ-TAMRA), and 1 μM ligand (1 μL of 100 μM solution). The microplate is centrifuged quickly (10 s), gently stirred for 30 min at 25 °C, centrifuged quickly (10 s) again, and then placed into Mx3005P. After a first equilibration step (25 °C, 30 s), a stepwise increase of 1 °C every 30 s for 65 cycles to reach 90 °C was performed, and measurements were made after each cycle. Final data were analyzed with Excel and OriginPro9. The emission of FAM was normalized (0 to 1), and <i>T</i><sub>1/2</sub> was defined as the temperature for which the normalized emission is 0.5; reported Δ<i>T</i><sub>1/2</sub> values are means of three experiments. Competitive experiments are performed similarly, that is, with labeled DNA (FAM-TWJ-TAMRA, 0.2 μM) in the presence of a ligand (1.0 μM, 5 mol equiv) and increasing amounts (3.0 and 10.0 μM, 15 and 50 mol equiv) of the unlabeled competitor ds26.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> Polyacrylamide Gel Electrophoresis</h3><div class="NLM_p last">Nondenaturing polyacrylamide gel electrophoresis (PAGE) was performed according to the protocol described by Malina et al.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> in 15% polyacrylamide gel (prepared by mixing 6.8 mL of acrylamide (40%), 11.2 mL of TBE buffer, 180 μL of APS (10% w/v), and 18 μL of TEMED; 15 min polymerization). Samples were prepared in 18 μL (volume) comprising 15 μL of DNA or DNA/ligand mixtures plus 3 μL of DNA loading dye (6×). Each solution was prepared separately: TWJ-S1 alone (6 μM), [TWJ-S1 + TWJ-S2 + TWJ-S3] (or M) (6 μM), [M (6 μM) + 1 mol equiv ligand (6 μM)], [M (6 μM) + 2 mol equiv ligand (12 μM)], [M (6 μM) + 3 mol equiv ligand (18 μM)], [M (6 μM) + 5 mol equiv ligand (30 μM)], and [TWJ-S1 + TWJ-S2] (6 μM); the solutions were stirred for 1 h at 25 °C prior to the addition of 3 μL of DNA loading dye (6×). These mixes were stirred for 15 min at 25 °C (a period during which the gel is stacked at 7 W (150–180 V, 43–38 mA) in TBE buffer enriched with 100 mM NaCl, pH 8.3) prior to the loading of 12 μL per well of each solution and 1 h migration at 7 W. After the migration, gels were analyzed after a poststaining step (SYBR Gold solution, 1:10 000, 10 min, 25 °C under gentle agitation) with a UVP MultiDoc-It imaging system (λ<sub>ex</sub> = 302 nm). Competitive PAGE experiments were performed according to a similar protocol but including FAM-TWJ-S1 (instead of the unlabeled TWJ-S1). Then, 15 μL of the following solutions was prepared: ds12 alone (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>) (252 μM-nt), M alone (252 μM-nt), [M (252 μM-nt) + 2 mol equiv ligand (12 μM)], [M (252 μM-nt) + 2 mol equiv ligand (12 μM) + 1 mol equiv ds12 (252 μM-nt)], [M (252 μM-nt) + 2 mol equiv ligand (12 μM) + 5 mol equiv ds12 (1260 μM-nt)], [M (252 μM-nt) + 2 mol equiv ligand (12 μM) + 10 mol equiv ds12 (2520 μM-nt)]. The solutions were stirred for 1 h at 25 °C prior to the addition of 3 μL of sucrose 48%. These mixes were stirred for 15 min at 25 °C prior to the loading of 12 μL per well of each solution and 50 min migration at 7 W. After the migration, gels were poststained (SYBRGold solution, 1:10 000) and imaged with the MultiDoc-It (λ<sub>ex</sub> = 302 nm).</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> ESI-MS Analysis</h3><div class="NLM_p last">Electrospray mass spectrometry experiments were performed on an LTQ Orbitrap XL (Thermo Scientific) spectrometer equipped with an Ion Max source and a HESI-II probe in the negative ion mode. TWJ and ds17 alone as well as the corresponding DNA/ligand mixtures (1:1 and 1:2 mol equiv) were prepared in 100 mM ammonium acetate buffer and equilibrated at 25 °C for 1 h. To obtain a stable electrospray signal, 20% methanol was added to the solution just before injection. The solutions were injected with a syringe pump at a flow rate of 5 μL min<sup>–1</sup>. The full scan mass was recorded in the 600–4000 <i>m</i>/<i>z</i> range. The following tuning parameters were used: heater temperature = 50 °C, spray voltage = 4.0 kV, capillary temperature = 275 °C, tube lens = −160.00 (negative ion mode), and capillary voltage between −35.00 and −60.00 V. Quantification of the equilibrium affinity constants (<i>K</i>) of ligands for either TWJ or ds17 has been done according to the procedure described by Rosu et al.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a></div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> Competition Equilibrium Dialysis</h3><div class="NLM_p">The oligonucleotide solutions of TWJ and ds17 were prepared as 75 μM (base pair) concentration in 111 mM NaCl solution. These solutions were then heated at 90 °C for 5 min, cooled down slowly to 25 °C, and left at 4 °C overnight. For each dialysis experiment, four dialysis units were used (Slide-A-Lyzer MINI Dialysis Units, molecular weight cutoff: 3.5 kDa) that were previously washed with milli-Q water. Two of them were filled with 200 μL of TWJ DNA solution and the other two bags with 200 μL of the solution of duplex DNA ds17. Dialysis units were then placed into a beaker containing 200 mL of 2–2.5 μM solution of the tested ligand enriched with 111 mM NaCl, covered with parafilm and aluminum foil, and allowed to equilibrate for 24 h at room temperature. The solutions from each dialysis unit were then transferred to Eppendorf tubes. The content of each bag was then mixed with sodium dodecyl sulfate solution (10%) to reach a 1% SDS concentration (v/v). The concentrations of the free product in the dialysate solution and product concentrations in the dialysis bags were determined by fluorescence measurements using the molar extinction coefficients of the ligand (determined in the presence and absence of the detergent; see <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Tables <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a>, respectively), and finally, the apparent association constants (<i>K</i><sub>app</sub>) were calculated according to the formula:<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a><i>K</i><sub>app</sub> = [L-NA]/[NA][L], expressed in M<sup>–1</sup>, in which [L-NA] is the concentration of the ligand bound to nucleic acid determined as a difference between the total ligand concentration inside the dialysis unit and the free ligand concentration (dialysate solution), [NA] is the concentration of free DNA calculated as a difference between the initial concentration of nucleic acid and [L-NA], and [L] is the free ligand concentration (dialysate solution).</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Extinction Coefficients of the Tested Ligands</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="×" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">ligand</th><th class="colsep0 rowsep0" align="center" char=".">excitation wavelength (nm)</th><th class="colsep0 rowsep0" align="center" char=".">emission wavelength (nm)</th><th class="colsep0 rowsep0" align="center" char="×">molar fluorescence extinction coefficient (mol<sup>–1</sup> L<sup>–1</sup> cm<sup>–1</sup>)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>451</b></td><td class="colsep0 rowsep0" align="char" char=".">285</td><td class="colsep0 rowsep0" align="char" char=".">330</td><td class="colsep0 rowsep0" align="char" char="×">2.33 × 10<sup>9</sup></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>459</b></td><td class="colsep0 rowsep0" align="char" char=".">300</td><td class="colsep0 rowsep0" align="char" char=".">400</td><td class="colsep0 rowsep0" align="char" char="×">4.74 × 10<sup>8</sup></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>471</b></td><td class="colsep0 rowsep0" align="char" char=".">200</td><td class="colsep0 rowsep0" align="char" char=".">319</td><td class="colsep0 rowsep0" align="char" char="×">9.38 × 10<sup>8</sup></td></tr></tbody></table></div></div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. Molar Fluorescent Extinction Coefficients of the Tested Ligands</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="×" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">ligand</th><th class="colsep0 rowsep0" align="center" char=".">excitation wavelength (nm)</th><th class="colsep0 rowsep0" align="center" char=".">emission wavelength (nm)</th><th class="colsep0 rowsep0" align="center" char="×">molar fluorescence extinction coefficient (mol<sup>–1</sup> L<sup>–1</sup> cm<sup>–1</sup>)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>451</b></td><td class="colsep0 rowsep0" align="char" char=".">285</td><td class="colsep0 rowsep0" align="char" char=".">335</td><td class="colsep0 rowsep0" align="char" char="×">3.91 × 10<sup>9</sup></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>459</b></td><td class="colsep0 rowsep0" align="char" char=".">300</td><td class="colsep0 rowsep0" align="char" char=".">400</td><td class="colsep0 rowsep0" align="char" char="×">1.07 × 10<sup>9</sup></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>471</b></td><td class="colsep0 rowsep0" align="char" char=".">200</td><td class="colsep0 rowsep0" align="char" char=".">318</td><td class="colsep0 rowsep0" align="char" char="×">6.58 × 10<sup>8</sup></td></tr></tbody></table></div></div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> Cell Culture</h3><div class="NLM_p last">MCF7 and MDA-MB-231 (breast adenocarcinoma) cells were routinely cultured in 75 cm<sup>2</sup> tissue culture flasks (Nunc) at 37 °C in a humidified, 5% CO<sub>2</sub> atmosphere in Dulbecco’s modified Eagle’s medium (DMEM) supplemented with 10% fetal bovine serum (FBS, Gibco) and a 1% penicillin–streptomycin (Pen–Strep, 5.0 units mL<sup>–1</sup> Pen/5.0 μg mL<sup>–1</sup> Strep, Gibco) mixture. The same culture conditions were used for BJ-hTERT, normal foreskin fibroblasts immortalized by hTERT transduction, except that the growth medium was a 4:1 mixture of DMEM and M199 medium supplemented with 15% FBS and Pen–Strep. Cells were subcultured twice a week using a standard protocol: the medium was first removed by aspiration; the cells were subsequently washed once with Dulbecco’s phosphate-buffered saline, and 1.5 mL of a trypsin solution (0.25%, Gibco) was added. After 5 min at 37 °C, cells were manually harvested, and 500 μL of the solution of cells in the suspension was dispensed into three cell culture flasks containing 10 mL of DMEM medium (with 10% FBS and 1% Pen–Strep).</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> Cell Proliferation SRB Assay</h3><div class="NLM_p last">The antiproliferative properties of the ligands were assessed through the sulforhodamine B (SRB) assay according to the protocols described by Vichai & Kirtikara and Skehan et al.<a onclick="showRef(event, 'ref41 ref42'); return false;" href="javascript:void(0);" class="ref ref41 ref42">(41,42)</a> Cells were seeded in a 96-well plate (6000 cells per well) in 160 μL of growth medium for 24 h at 37 °C. Then, 40 μL of the ligand solution was added to reach the final concentration of the ligands between 500 and 0.05 μM or 50 and 0.005 μM and incubated for 72 h at 37 °C. After 72 h, the media was removed and the cells were fixed with a solution of trichloroacetic acid 10% (150 μL, 1 h at 4 °C). The supernatant was removed, and the fixed cells were washed with water and then dried. A 100 μL solution of SRB (0.2% in 1% acetic acid) was added into each well (except control wells); after 30 min, the supernatant was removed, the wells were washed three times with 150 μL of acetic acid (1%), and dried. After that, 150 μL of Tris base (10 mM) was added in each well, and the microplate was gently stirred for 5 min at 25 °C. Optical density (OD) values were determined at 530 nm. Final data were analyzed with Excel (Microsoft Corp.) and GraphPad 7.0 (Prism software): the OD<sub>530nm</sub> was normalized (from 0 to 100; 0 for ligand untreated, SRB unstained cells and 100 for ligand untreated, SRB stained cells), and LD<sub>50</sub> (lethal dose, defined as the concentration at which 50% of the cell growth inhibition is reached) was determined for a normalized OD<sub>530nm</sub> of 50%; reported LD<sub>50</sub> values are means of three experiments.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i26"><a href="/doi/suppl/10.1021/acs.jmedchem.8b01978" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_38557" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_38557" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.8b01978" class="ext-link">10.1021/acs.jmedchem.8b01978</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">ESI-MS evaluations of <b>451</b>, <b>459</b>, and <b>471</b> with both TWJ-DNA and ds-DNA (Figures S1–S3), SRB assays performed with MCF7 and MDA-MB-231 cells and camptothecin, and with BJ-hTERT and <b>451</b>, <b>459</b>, <b>471</b>, and camptothecin (Figure S4) and immunodetection experiments performed with MCF7 cells and <b>471</b> and camptothecin (Figure S5) (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01978/suppl_file/jm8b01978_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Complete TWJ-screen results and the molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01978/suppl_file/jm8b01978_si_002.csv" class="ext-link">CSV</a>)</p></li><li><p class="inline">Top ligand selection (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01978/suppl_file/jm8b01978_si_003.xlsx" class="ext-link">XLSX</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b01978/suppl_file/jm8b01978_si_001.pdf">jm8b01978_si_001.pdf (1.6 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b01978/suppl_file/jm8b01978_si_002.csv">jm8b01978_si_002.csv (3.19 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b01978/suppl_file/jm8b01978_si_003.xlsx">jm8b01978_si_003.xlsx (321.41 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.8b01978" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_61856" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_61856" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">David Monchaud</span> - <span class="hlFld-Affiliation affiliation">Institut
de Chimie Moléculaire (ICMUB), CNRS UMR6302, UBFC, 21078 Dijon, France</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-3056-9295" title="Orcid link">http://orcid.org/0000-0002-3056-9295</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#4125203728256f2c2e2f222920342501222f33326f2733"><span class="__cf_email__" data-cfemail="224643544b460c4f4d4c414a43574662414c50510c4450">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Katerina Duskova</span> - <span class="hlFld-Affiliation affiliation">Institut
de Chimie Moléculaire (ICMUB), CNRS UMR6302, UBFC, 21078 Dijon, France</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jérémy Lamarche</span> - <span class="hlFld-Affiliation affiliation">Institut
de Chimie Moléculaire (ICMUB), CNRS UMR6302, UBFC, 21078 Dijon, France</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Souheila Amor</span> - <span class="hlFld-Affiliation affiliation">Lipids,
Nutrition & Cancers (LNC), INSERM U1321, UBFC, 21000 Dijon, France</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Coralie Caron</span> - <span class="hlFld-Affiliation affiliation">Institut
Curie, PSL Research University, CNRS UMR9187, INSERM U1196, 91405 Orsay, France</span>; 
    <span class="hlFld-Affiliation affiliation">Université
Paris-Sud, Université Paris Saclay, 91405 Orsay, France</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Nicolas Queyriaux</span> - <span class="hlFld-Affiliation affiliation">Laboratoire
de Chimie et Biologie des Métaux, Université Grenoble Alpes, CNRS UMR5249, CEA, 38054 Grenoble, France</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Marie Gaschard</span> - <span class="hlFld-Affiliation affiliation">Institut
de Chimie, Université de Neuchâtel, CH-2000 Neuchatel, Switzerland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Marie-José Penouilh</span> - <span class="hlFld-Affiliation affiliation">Pôle
Chimie Moléculaire (PACSMUB), Faculté
des Sciences Mirande, UBFC, 21078 Dijon, France</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Guillaume de Robillard</span> - <span class="hlFld-Affiliation affiliation">Institut
de Chimie Moléculaire (ICMUB), CNRS UMR6302, UBFC, 21078 Dijon, France</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Dominique Delmas</span> - <span class="hlFld-Affiliation affiliation">Lipids,
Nutrition & Cancers (LNC), INSERM U1321, UBFC, 21000 Dijon, France</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Charles H. Devillers</span> - <span class="hlFld-Affiliation affiliation">Institut
de Chimie Moléculaire (ICMUB), CNRS UMR6302, UBFC, 21078 Dijon, France</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-9078-7035" title="Orcid link">http://orcid.org/0000-0001-9078-7035</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Anton Granzhan</span> - <span class="hlFld-Affiliation affiliation">Institut
Curie, PSL Research University, CNRS UMR9187, INSERM U1196, 91405 Orsay, France</span>; 
    <span class="hlFld-Affiliation affiliation">Université
Paris-Sud, Université Paris Saclay, 91405 Orsay, France</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-0424-0461" title="Orcid link">http://orcid.org/0000-0002-0424-0461</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Marie-Paule Teulade-Fichou</span> - <span class="hlFld-Affiliation affiliation">Institut
Curie, PSL Research University, CNRS UMR9187, INSERM U1196, 91405 Orsay, France</span>; 
    <span class="hlFld-Affiliation affiliation">Université
Paris-Sud, Université Paris Saclay, 91405 Orsay, France</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Murielle Chavarot-Kerlidou</span> - <span class="hlFld-Affiliation affiliation">Laboratoire
de Chimie et Biologie des Métaux, Université Grenoble Alpes, CNRS UMR5249, CEA, 38054 Grenoble, France</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-2709-3772" title="Orcid link">http://orcid.org/0000-0003-2709-3772</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Bruno Therrien</span> - <span class="hlFld-Affiliation affiliation">Institut
de Chimie, Université de Neuchâtel, CH-2000 Neuchatel, Switzerland</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-0388-2745" title="Orcid link">http://orcid.org/0000-0002-0388-2745</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sébastien Britton</span> - <span class="hlFld-Affiliation affiliation">Institut
de Pharmacologie et de Biologie Structurale, IPBS, CNRS UMR5089, Université
de Toulouse, UPS, équipe labellisée la Ligue Contre
le Cancer, 31077 Toulouse, France</span></div></li></ul></li><li></li><li><span class="author-information-subsection-header">Funding</span><p>This project has received funding from the European Union’s Horizon 2020 research and innovation program (grant agreement H2020-MSCA-IF-2016-750368), from the Agence Nationale de la Recherche (grant agreement ANR-17-CE17-0010-01), and IDEX Paris Saclay (“IDI 2016”, ANR-11-IDEX-0003-02) and is also part of the project “Pharmacoimagerie & agents théranostiques” supported by the Université de Bourgogne and Conseil Régional de Bourgogne (PARI) and the European Union (PO FEDER-FSE Bourgogne 2014/2020 programs).</p></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d7e3086-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i27">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_18284" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_18284" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The authors thank Medicines for Malaria Venture for the Pathogen Box and the French National Chemical Library for the CNE, as well as R. A. Weinberg for the gift of the BJ-hTERT cell line.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">DNA</td><td class="NLM_def"><p class="first last">deoxyribonucleic acid</p></td></tr><tr><td class="NLM_term">TWJ</td><td class="NLM_def"><p class="first last">three-way DNA junction</p></td></tr><tr><td class="NLM_term">PAGE</td><td class="NLM_def"><p class="first last">polyacrylamide gel electrophoresis</p></td></tr><tr><td class="NLM_term">FRET</td><td class="NLM_def"><p class="first last">fluorescence resonance energy transfer</p></td></tr><tr><td class="NLM_term">ESI-MS</td><td class="NLM_def"><p class="first last">electrospray ionization mass spectrometry</p></td></tr><tr><td class="NLM_term">SRB</td><td class="NLM_def"><p class="first last">sulforhodamine B</p></td></tr><tr><td class="NLM_term">HTS</td><td class="NLM_def"><p class="first last">high-throughput screen</p></td></tr><tr><td class="NLM_term">CNE</td><td class="NLM_def"><p class="first last">chimiothèque nationale essentielle</p></td></tr><tr><td class="NLM_term">ICMUB</td><td class="NLM_def"><p class="first last">institut de chimie moléculaire de l’université de Bourgogne</p></td></tr><tr><td class="NLM_term">FAM</td><td class="NLM_def"><p class="first last">fluorescein amidite</p></td></tr><tr><td class="NLM_term">TAMRA</td><td class="NLM_def"><p class="first last">tetramethylrhodamine amidite</p></td></tr><tr><td class="NLM_term">NFI</td><td class="NLM_def"><p class="first last">normalized FAM intensity</p></td></tr><tr><td class="NLM_term">SDS</td><td class="NLM_def"><p class="first last">sodium dodecyl sulfate</p></td></tr><tr><td class="NLM_term">LD</td><td class="NLM_def"><p class="first last">lethal dose</p></td></tr><tr><td class="NLM_term">DDR</td><td class="NLM_def"><p class="first last">DNA damage response</p></td></tr><tr><td class="NLM_term">DMEM</td><td class="NLM_def"><p class="first last">Dulbecco’s modified Eagle medium</p></td></tr><tr><td class="NLM_term">FBS</td><td class="NLM_def"><p class="first last">fetal bovine serum</p></td></tr><tr><td class="NLM_term">OD</td><td class="NLM_def"><p class="first last">optical density</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i29">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_85470" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_85470" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 48 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zeman, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cimprich, K. A.</span></span> <span> </span><span class="NLM_article-title">Causes and consequences of replication stress</span>. <i>Nat. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">2</span>– <span class="NLM_lpage">9</span>, <span class="refDoi"> DOI: 10.1038/ncb2897</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=10.1038%2Fncb2897" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=24366029" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFyks7zI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2014&pages=2-9&author=M.+K.+Zemanauthor=K.+A.+Cimprich&title=Causes+and+consequences+of+replication+stress&doi=10.1038%2Fncb2897"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Causes and consequences of replication stress</span></div><div class="casAuthors">Zeman, Michelle K.; Cimprich, Karlene A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Cell Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">2-9</span>CODEN:
                <span class="NLM_cas:coden">NCBIFN</span>;
        ISSN:<span class="NLM_cas:issn">1465-7392</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Replication stress is a complex phenomenon that has serious implications for genome stability, cell survival and human disease.  Generation of aberrant replication fork structures contg. single-stranded DNA activates the replication stress response, primarily mediated by the kinase ATR (ATM- and Rad3-related).  Along with its downstream effectors, ATR stabilizes and helps to restart stalled replication forks, avoiding the generation of DNA damage and genome instability.  Understanding this response may be key to diagnosing and treating human diseases caused by defective responses to replication stress.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrO2sHtB4BRZLVg90H21EOLACvtfcHk0lhn-eYY1xYHgA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFyks7zI&md5=108310bae187037f311ca991fa0eed10</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fncb2897&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncb2897%26sid%3Dliteratum%253Aachs%26aulast%3DZeman%26aufirst%3DM.%2BK.%26aulast%3DCimprich%26aufirst%3DK.%2BA.%26atitle%3DCauses%2520and%2520consequences%2520of%2520replication%2520stress%26jtitle%3DNat.%2520Cell%2520Biol.%26date%3D2014%26volume%3D16%26spage%3D2%26epage%3D9%26doi%3D10.1038%2Fncb2897" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Técher, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koundrioukoff, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicolas, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Debatisse, M.</span></span> <span> </span><span class="NLM_article-title">The impact of replication stress on replication dynamics and DNA damage in vertebrate cells</span>. <i>Nat. Rev. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">535</span>– <span class="NLM_lpage">550</span>, <span class="refDoi"> DOI: 10.1038/nrg.2017.46</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=10.1038%2Fnrg.2017.46" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=28714480" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtFOrurzK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=535-550&author=H.+T%C3%A9cherauthor=S.+Koundrioukoffauthor=A.+Nicolasauthor=M.+Debatisse&title=The+impact+of+replication+stress+on+replication+dynamics+and+DNA+damage+in+vertebrate+cells&doi=10.1038%2Fnrg.2017.46"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">The impact of replication stress on replication dynamics and DNA damage in vertebrate cells</span></div><div class="casAuthors">Techer, Herve; Koundrioukoff, Stephane; Nicolas, Alain; Debatisse, Michelle</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Genetics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">535-550</span>CODEN:
                <span class="NLM_cas:coden">NRGAAM</span>;
        ISSN:<span class="NLM_cas:issn">1471-0056</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The interplay between replication stress and the S phase checkpoint is a key determinant of genome maintenance, and has a major impact on human diseases, notably, tumor initiation and progression.  Recent studies have yielded insights into sequence-dependent and sequence-independent sources of endogenous replication stress.  These stresses result in nuclease-induced DNA damage, checkpoint activation and genome-wide replication fork slowing.  Several hypotheses have been proposed to account for the mechanisms involved in this complex response.  Recent results have shown that the slowing of the replication forks most commonly results from DNA precursor starvation.  By concomitantly increasing the d. of replication initiation, the cell elicits an efficient compensatory strategy to avoid mitotic anomalies and the inheritance of damage over cell generations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovTOAyuyvPobVg90H21EOLACvtfcHk0lhbYLXLiBKgUA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtFOrurzK&md5=137bbadc349995265f02b390544817a1</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnrg.2017.46&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrg.2017.46%26sid%3Dliteratum%253Aachs%26aulast%3DT%25C3%25A9cher%26aufirst%3DH.%26aulast%3DKoundrioukoff%26aufirst%3DS.%26aulast%3DNicolas%26aufirst%3DA.%26aulast%3DDebatisse%26aufirst%3DM.%26atitle%3DThe%2520impact%2520of%2520replication%2520stress%2520on%2520replication%2520dynamics%2520and%2520DNA%2520damage%2520in%2520vertebrate%2520cells%26jtitle%3DNat.%2520Rev.%2520Genet.%26date%3D2017%26volume%3D18%26spage%3D535%26epage%3D550%26doi%3D10.1038%2Fnrg.2017.46" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gaillard, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">García-Muse, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aguilera, A.</span></span> <span> </span><span class="NLM_article-title">Replication stress and cancer</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">276</span>– <span class="NLM_lpage">289</span>, <span class="refDoi"> DOI: 10.1038/nrc3916</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=10.1038%2Fnrc3916" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=25907220" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1Wrur3K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2015&pages=276-289&author=H.+Gaillardauthor=T.+Garc%C3%ADa-Museauthor=A.+Aguilera&title=Replication+stress+and+cancer&doi=10.1038%2Fnrc3916"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Replication stress and cancer</span></div><div class="casAuthors">Gaillard, Helene; Garcia-Muse, Tatiana; Aguilera, Andres</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">276-289</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Genome instability is a hallmark of cancer, and DNA replication is the most vulnerable cellular process that can lead to it.  Any condition leading to high levels of DNA damage will result in replication stress, which is a source of genome instability and a feature of pre-cancerous and cancerous cells.  Therefore, understanding the mol. basis of replication stress is crucial to the understanding of tumorigenesis.  Although a neg. aspect of replication stress is its prominent role in tumorigenesis, a pos. aspect is that it provides a potential target for cancer therapy.  In this Review, we discuss the link between persistent replication stress and tumorigenesis, with the goal of shedding light on the mechanisms underlying the initiation of an oncogenic process, which should open up new possibilities for cancer diagnostics and treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxvJ6UZvAnXbVg90H21EOLACvtfcHk0lhbYLXLiBKgUA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1Wrur3K&md5=8244fca02b10d3356dbd915337203373</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnrc3916&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc3916%26sid%3Dliteratum%253Aachs%26aulast%3DGaillard%26aufirst%3DH.%26aulast%3DGarc%25C3%25ADa-Muse%26aufirst%3DT.%26aulast%3DAguilera%26aufirst%3DA.%26atitle%3DReplication%2520stress%2520and%2520cancer%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2015%26volume%3D15%26spage%3D276%26epage%3D289%26doi%3D10.1038%2Fnrc3916" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dobbelstein, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sørensen, C. S.</span></span> <span> </span><span class="NLM_article-title">Exploiting replicative stress to treat cancer</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">405</span>– <span class="NLM_lpage">423</span>, <span class="refDoi"> DOI: 10.1038/nrd4553</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=10.1038%2Fnrd4553" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2015&pages=405-423&author=M.+Dobbelsteinauthor=C.+S.+S%C3%B8rensen&title=Exploiting+replicative+stress+to+treat+cancer&doi=10.1038%2Fnrd4553"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fnrd4553&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4553%26sid%3Dliteratum%253Aachs%26aulast%3DDobbelstein%26aufirst%3DM.%26aulast%3DS%25C3%25B8rensen%26aufirst%3DC.%2BS.%26atitle%3DExploiting%2520replicative%2520stress%2520to%2520treat%2520cancer%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2015%26volume%3D14%26spage%3D405%26epage%3D423%26doi%3D10.1038%2Fnrd4553" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jackson, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartek, J.</span></span> <span> </span><span class="NLM_article-title">The DNA-damage response in human biology and disease</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>461</i></span>,  <span class="NLM_fpage">1071</span>– <span class="NLM_lpage">1078</span>, <span class="refDoi"> DOI: 10.1038/nature08467</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=10.1038%2Fnature08467" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=19847258" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtlGktLrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=461&publication_year=2009&pages=1071-1078&author=S.+P.+Jacksonauthor=J.+Bartek&title=The+DNA-damage+response+in+human+biology+and+disease&doi=10.1038%2Fnature08467"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">The DNA-damage response in human biology and disease</span></div><div class="casAuthors">Jackson, Stephen P.; Bartek, Jiri</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">461</span>
        (<span class="NLM_cas:issue">7267</span>),
    <span class="NLM_cas:pages">1071-1078</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The prime objective for every life form is to deliver its genetic material, intact and unchanged, to the next generation.  This must be achieved despite const. assaults by endogenous and environmental agents on the DNA.  To counter this threat, life has evolved several systems to detect DNA damage, signal its presence and mediate its repair.  Such responses, which have an impact on a wide range of cellular events, are biol. significant because they prevent diverse human diseases.  Our improving understanding of DNA-damage responses is providing new avenues for disease management.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpQ0Oyp6nX9V7Vg90H21EOLACvtfcHk0lhbYLXLiBKgUA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtlGktLrK&md5=cca7cfec66852124695ebeb515f255a5</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnature08467&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature08467%26sid%3Dliteratum%253Aachs%26aulast%3DJackson%26aufirst%3DS.%2BP.%26aulast%3DBartek%26aufirst%3DJ.%26atitle%3DThe%2520DNA-damage%2520response%2520in%2520human%2520biology%2520and%2520disease%26jtitle%3DNature%26date%3D2009%26volume%3D461%26spage%3D1071%26epage%3D1078%26doi%3D10.1038%2Fnature08467" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ciccia, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elledge, S. J.</span></span> <span> </span><span class="NLM_article-title">The DNA damage response: making it safe to play with knives</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">179</span>– <span class="NLM_lpage">204</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2010.09.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=10.1016%2Fj.molcel.2010.09.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=20965415" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtlCjtr7P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2010&pages=179-204&author=A.+Cicciaauthor=S.+J.+Elledge&title=The+DNA+damage+response%3A+making+it+safe+to+play+with+knives&doi=10.1016%2Fj.molcel.2010.09.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">The DNA Damage Response: Making It Safe to Play with Knives</span></div><div class="casAuthors">Ciccia, Alberto; Elledge, Stephen J.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">179-204</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Damage to our genetic material is an ongoing threat to both our ability to faithfully transmit genetic information to our offspring as well as our own survival.  To respond to these threats, eukaryotes have evolved the DNA damage response (DDR).  The DDR is a complex signal transduction pathway that has the ability to sense DNA damage and transduce this information to the cell to influence cellular responses to DNA damage.  Cells possess an arsenal of enzymic tools capable of remodeling and repairing DNA; however, their activities must be tightly regulated in a temporal, spatial, and DNA lesion-appropriate fashion to optimize repair and prevent unnecessary and potentially deleterious alterations in the structure of DNA during normal cellular processes.  This review will focus on how the DDR controls DNA repair and the phenotypic consequences of defects in these crit. regulatory functions in mammals.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGofclv-VUxTXLVg90H21EOLACvtfcHk0liiArHrGU7vnQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtlCjtr7P&md5=71d5ed7d3a82d9b395877510bbe4df03</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2010.09.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2010.09.019%26sid%3Dliteratum%253Aachs%26aulast%3DCiccia%26aufirst%3DA.%26aulast%3DElledge%26aufirst%3DS.%2BJ.%26atitle%3DThe%2520DNA%2520damage%2520response%253A%2520making%2520it%2520safe%2520to%2520play%2520with%2520knives%26jtitle%3DMol.%2520Cell%26date%3D2010%26volume%3D40%26spage%3D179%26epage%3D204%26doi%3D10.1016%2Fj.molcel.2010.09.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguez, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, K. M.</span></span> <span> </span><span class="NLM_article-title">Unravelling the genomic targets of small molecules using high-throughput sequencing</span>. <i>Nat. Rev. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">783</span>– <span class="NLM_lpage">796</span>, <span class="refDoi"> DOI: 10.1038/nrg3796</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=10.1038%2Fnrg3796" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=25311424" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslCitbzE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2014&pages=783-796&author=R.+Rodriguezauthor=K.+M.+Miller&title=Unravelling+the+genomic+targets+of+small+molecules+using+high-throughput+sequencing&doi=10.1038%2Fnrg3796"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Unravelling the genomic targets of small molecules using high-throughput sequencing</span></div><div class="casAuthors">Rodriguez, Raphael; Miller, Kyle M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Genetics</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">783-796</span>CODEN:
                <span class="NLM_cas:coden">NRGAAM</span>;
        ISSN:<span class="NLM_cas:issn">1471-0056</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Small mols. - including various approved and novel cancer therapeutics - can operate at the genomic level by targeting the DNA and protein components of chromatin.  Emerging evidence suggests that functional interactions between small mols. and the genome are non-stochastic and are influenced by a dynamic interplay between DNA sequences and chromatin states.  The establishment of genome-wide maps of small-mol. targets using unbiased methodologies can help to characterize and exploit drug responses.  In this Review, we discuss how high-throughput sequencing strategies, such as ChIP-seq (chromatin immunopptn. followed by sequencing) and Chem-seq (chem. affinity capture and massively parallel DNA sequencing), are enabling the comprehensive identification of small-mol. target sites throughout the genome, thereby providing insights into unanticipated drug effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9JdFmc3eSnrVg90H21EOLACvtfcHk0liiArHrGU7vnQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslCitbzE&md5=569d9a5be0cb270a030314d93b24c11f</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fnrg3796&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrg3796%26sid%3Dliteratum%253Aachs%26aulast%3DRodriguez%26aufirst%3DR.%26aulast%3DMiller%26aufirst%3DK.%2BM.%26atitle%3DUnravelling%2520the%2520genomic%2520targets%2520of%2520small%2520molecules%2520using%2520high-throughput%2520sequencing%26jtitle%3DNat.%2520Rev.%2520Genet.%26date%3D2014%26volume%3D15%26spage%3D783%26epage%3D796%26doi%3D10.1038%2Fnrg3796" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Helleday, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petermann, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lundin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hodgson, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, R. A.</span></span> <span> </span><span class="NLM_article-title">DNA repair pathways as targets for cancer therapy</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">193</span>– <span class="NLM_lpage">204</span>, <span class="refDoi"> DOI: 10.1038/nrc2342</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=10.1038%2Fnrc2342" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=18256616" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=1%3ACAS%3A528%3ADC%252BD1cXitlOisr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2008&pages=193-204&author=T.+Helledayauthor=E.+Petermannauthor=C.+Lundinauthor=B.+Hodgsonauthor=R.+A.+Sharma&title=DNA+repair+pathways+as+targets+for+cancer+therapy&doi=10.1038%2Fnrc2342"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">DNA repair pathways as targets for cancer therapy</span></div><div class="casAuthors">Helleday, Thomas; Petermann, Eva; Lundin, Cecilia; Hodgson, Ben; Sharma, Ricky A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">193-204</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  DNA repair pathways can enable tumor cells to survive DNA damage that is induced by chemotherapeutic treatments; therefore, inhibitors of specific DNA repair pathways might prove efficacious when used in combination with DNA-damaging chemotherapeutic drugs.  In addn., alterations in DNA repair pathways that arise during tumor development can make some cancer cells reliant on a reduced set of DNA repair pathways for survival.  There is evidence that drugs that inhibit one of these pathways in such tumors could prove useful as single-agent therapies, with the potential advantage that this approach could be selective for tumor cells and have fewer side effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3W53LJmdCPrVg90H21EOLACvtfcHk0liiArHrGU7vnQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXitlOisr8%253D&md5=8592af6d815e5ca9c80ee256791bafb7</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnrc2342&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc2342%26sid%3Dliteratum%253Aachs%26aulast%3DHelleday%26aufirst%3DT.%26aulast%3DPetermann%26aufirst%3DE.%26aulast%3DLundin%26aufirst%3DC.%26aulast%3DHodgson%26aufirst%3DB.%26aulast%3DSharma%26aufirst%3DR.%2BA.%26atitle%3DDNA%2520repair%2520pathways%2520as%2520targets%2520for%2520cancer%2520therapy%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2008%26volume%3D8%26spage%3D193%26epage%3D204%26doi%3D10.1038%2Fnrc2342" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mirkin, E. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mirkin, S. M.</span></span> <span> </span><span class="NLM_article-title">Replication fork stalling at natural impediments</span>. <i>Microbiol. Mol. Biol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>71</i></span>,  <span class="NLM_fpage">13</span>– <span class="NLM_lpage">35</span>, <span class="refDoi"> DOI: 10.1128/MMBR.00030-06</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=10.1128%2FMMBR.00030-06" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=17347517" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=1%3ACAS%3A528%3ADC%252BD2sXksleqsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2007&pages=13-35&author=E.+V.+Mirkinauthor=S.+M.+Mirkin&title=Replication+fork+stalling+at+natural+impediments&doi=10.1128%2FMMBR.00030-06"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Replication fork stalling at natural impediments</span></div><div class="casAuthors">Mirkin, Ekaterina V.; Mirkin, Sergei M.</div><div class="citationInfo"><span class="NLM_cas:title">Microbiology and Molecular Biology Reviews</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">71</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">13-35</span>CODEN:
                <span class="NLM_cas:coden">MMBRF7</span>;
        ISSN:<span class="NLM_cas:issn">1092-2172</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">A review.  Accurate and complete replication of the genome in every cell division is a prerequisite of genomic stability.  Thus, both prokaryotic and eukaryotic replication forks are extremely precise and robust mol. machines that have evolved to be up to the task.  However, it has recently become clear that the replication fork is more of a hurdler than a runner: it must overcome various obstacles present on its way.  Such obstacles can be called natural impediments to DNA replication, as opposed to external and genetic factors.  Natural impediments to DNA replication are particular DNA binding proteins, unusual secondary structures in DNA, and transcription complexes that occasionally (in eukaryotes) or constantly (in prokaryotes) operate on replicating templates.  This review describes the mechanisms and consequences of replication stalling at various natural impediments, with an emphasis on the role of replication stalling in genomic instability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcteIUmVZAYLVg90H21EOLACvtfcHk0liiArHrGU7vnQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXksleqsLg%253D&md5=f3e547ba124f2a93c5fb6cd3d8e64222</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1128%2FMMBR.00030-06&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMMBR.00030-06%26sid%3Dliteratum%253Aachs%26aulast%3DMirkin%26aufirst%3DE.%2BV.%26aulast%3DMirkin%26aufirst%3DS.%2BM.%26atitle%3DReplication%2520fork%2520stalling%2520at%2520natural%2520impediments%26jtitle%3DMicrobiol.%2520Mol.%2520Biol.%2520Rev.%26date%3D2007%26volume%3D71%26spage%3D13%26epage%3D35%26doi%3D10.1128%2FMMBR.00030-06" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bacolla, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vasquez, K. M.</span></span> <span> </span><span class="NLM_article-title">Non-B DNA structure-induced genetic instability and evolution</span>. <i>Cell. Mol. Life Sci.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">43</span>– <span class="NLM_lpage">62</span>, <span class="refDoi"> DOI: 10.1007/s00018-009-0131-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=10.1007%2Fs00018-009-0131-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=19727556" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=1%3ACAS%3A528%3ADC%252BC3cXis1ajsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2010&pages=43-62&author=J.+Zhaoauthor=A.+Bacollaauthor=G.+Wangauthor=K.+M.+Vasquez&title=Non-B+DNA+structure-induced+genetic+instability+and+evolution&doi=10.1007%2Fs00018-009-0131-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Non-B DNA structure-induced genetic instability and evolution</span></div><div class="casAuthors">Zhao, Junhua; Bacolla, Albino; Wang, Guliang; Vasquez, Karen M.</div><div class="citationInfo"><span class="NLM_cas:title">Cellular and Molecular Life Sciences</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">43-62</span>CODEN:
                <span class="NLM_cas:coden">CMLSFI</span>;
        ISSN:<span class="NLM_cas:issn">1420-682X</span>.
    
            (<span class="NLM_cas:orgname">Birkhaeuser Verlag</span>)
        </div><div class="casAbstract">A review.  Repetitive DNA motifs are abundant in the genomes of various species and have the capacity to adopt non-canonical (i.e., non-B) DNA structures.  Several non-B DNA structures, including cruciforms, slipped structures, triplexes, G-quadruplexes, and Z-DNA, have been shown to cause mutations, such as deletions, expansions, and translocations in both prokaryotes and eukaryotes.  Their distributions in genomes are not random and often co-localize with sites of chromosomal breakage assocd. with genetic diseases.  Current genome-wide sequence analyses suggest that the genomic instabilities induced by non-B DNA structure-forming sequences not only result in predisposition to disease, but also contribute to rapid evolutionary changes, particularly in genes assocd. with development and regulatory functions.  In this review, we describe the occurrence of non-B DNA-forming sequences in various species, the classes of genes enriched in non-B DNA-forming sequences, and recent mechanistic studies on DNA structure-induced genomic instability to highlight their importance in genomes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8JVuub9QLlrVg90H21EOLACvtfcHk0lg6EqzoLEGFFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXis1ajsA%253D%253D&md5=452d69f118747ee4f8b513f1c8d9723c</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1007%2Fs00018-009-0131-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00018-009-0131-2%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DJ.%26aulast%3DBacolla%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DG.%26aulast%3DVasquez%26aufirst%3DK.%2BM.%26atitle%3DNon-B%2520DNA%2520structure-induced%2520genetic%2520instability%2520and%2520evolution%26jtitle%3DCell.%2520Mol.%2520Life%2520Sci.%26date%3D2010%26volume%3D67%26spage%3D43%26epage%3D62%26doi%3D10.1007%2Fs00018-009-0131-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Caruthers, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKay, D. B.</span></span> <span> </span><span class="NLM_article-title">Helicase structure and mechanism</span>. <i>Curr. Opin. Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">123</span>– <span class="NLM_lpage">133</span>, <span class="refDoi"> DOI: 10.1016/S0959-440X(02)00298-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=10.1016%2FS0959-440X%2802%2900298-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=11839499" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=1%3ACAS%3A528%3ADC%252BD38XhtF2itLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2002&pages=123-133&author=J.+M.+Caruthersauthor=D.+B.+McKay&title=Helicase+structure+and+mechanism&doi=10.1016%2FS0959-440X%2802%2900298-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Helicase structure and mechanism</span></div><div class="casAuthors">Caruthers, Jonathan M.; McKay, David B.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Structural Biology</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">123-133</span>CODEN:
                <span class="NLM_cas:coden">COSBEF</span>;
        ISSN:<span class="NLM_cas:issn">0959-440X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">A review with 42 refs.  Structural information on helicase proteins has expanded recently beyond DNA helicases Rep and PcrA, and hepatitis C virus RNA helicase to include UvrB, members of the DEA(D/H)-box RNA helicase family, examples of DnaB-related helicases, and RuvB.  The expanding database of structures has clarified the structural 'theme and variations' that relate the different helicase families.  Furthermore, information is emerging on the functions of the conserved helicase motifs and their participation in the mechanisms by which these proteins catalyze the remodeling of DNA and RNA in ATP-dependent activities.  The expanding database of helicase structures, enzymes that catalyze the sepn. of duplex oligonucleotides into single strands, has clarified the structural themes and variations that relate the different families.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXfRg9Xe7zjbVg90H21EOLACvtfcHk0lg6EqzoLEGFFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XhtF2itLY%253D&md5=138b992adc3e001b305ba02cf1108268</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2FS0959-440X%2802%2900298-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0959-440X%252802%252900298-1%26sid%3Dliteratum%253Aachs%26aulast%3DCaruthers%26aufirst%3DJ.%2BM.%26aulast%3DMcKay%26aufirst%3DD.%2BB.%26atitle%3DHelicase%2520structure%2520and%2520mechanism%26jtitle%3DCurr.%2520Opin.%2520Struct.%2520Biol.%26date%3D2002%26volume%3D12%26spage%3D123%26epage%3D133%26doi%3D10.1016%2FS0959-440X%2802%2900298-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zimmer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tacconi, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Folio, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Badie, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porru, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klare, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tumiati, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Markkanen, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halder, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryan, A.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Targeting BRCA1 and BRCA2 deficiencies with G-quadruplex-interacting compounds</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">449</span>– <span class="NLM_lpage">460</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2015.12.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=10.1016%2Fj.molcel.2015.12.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=26748828" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtlKlug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2016&pages=449-460&author=J.+Zimmerauthor=E.+M.+Tacconiauthor=C.+Folioauthor=S.+Badieauthor=M.+Porruauthor=K.+Klareauthor=M.+Tumiatiauthor=E.+Markkanenauthor=S.+Halderauthor=A.+Ryan&title=Targeting+BRCA1+and+BRCA2+deficiencies+with+G-quadruplex-interacting+compounds&doi=10.1016%2Fj.molcel.2015.12.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting BRCA1 and BRCA2 Deficiencies with G-Quadruplex-Interacting Compounds</span></div><div class="casAuthors">Zimmer, Jutta; Tacconi, Eliana M. C.; Folio, Cecilia; Badie, Sophie; Porru, Manuela; Klare, Kerstin; Tumiati, Manuela; Markkanen, Enni; Halder, Swagata; Ryan, Anderson; Jackson, Stephen P.; Ramadan, Kristijan; Kuznetsov, Sergey G.; Biroccio, Annamaria; Sale, Julian E.; Tarsounas, Madalena</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">449-460</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">G-quadruplex (G4)-forming genomic sequences, including telomeres, represent natural replication fork barriers.  Stalled replication forks can be stabilized and restarted by homologous recombination (HR), which also repairs DNA double-strand breaks (DSBs) arising at collapsed forks.  We have previously shown that HR facilitates telomere replication.  Here, we demonstrate that the replication efficiency of guanine-rich (G-rich) telomeric repeats is decreased significantly in cells lacking HR.  Treatment with the G4-stabilizing compd. pyridostatin (PDS) increases telomere fragility in BRCA2-deficient cells, suggesting that G4 formation drives telomere instability.  Remarkably, PDS reduces proliferation of HR-defective cells by inducing DSB accumulation, checkpoint activation, and deregulated G2/M progression and by enhancing the replication defect intrinsic to HR deficiency.  PDS toxicity extends to HR-defective cells that have acquired olaparib resistance through loss of 53BP1 or REV7.  Altogether, these results highlight the therapeutic potential of G4-stabilizing drugs to selectively eliminate HR-compromised cells and tumors, including those resistant to PARP inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5EXONoaAQUrVg90H21EOLACvtfcHk0lg6EqzoLEGFFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtlKlug%253D%253D&md5=b2299dbbb76cc442c47a1889174caf5b</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2015.12.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2015.12.004%26sid%3Dliteratum%253Aachs%26aulast%3DZimmer%26aufirst%3DJ.%26aulast%3DTacconi%26aufirst%3DE.%2BM.%26aulast%3DFolio%26aufirst%3DC.%26aulast%3DBadie%26aufirst%3DS.%26aulast%3DPorru%26aufirst%3DM.%26aulast%3DKlare%26aufirst%3DK.%26aulast%3DTumiati%26aufirst%3DM.%26aulast%3DMarkkanen%26aufirst%3DE.%26aulast%3DHalder%26aufirst%3DS.%26aulast%3DRyan%26aufirst%3DA.%26atitle%3DTargeting%2520BRCA1%2520and%2520BRCA2%2520deficiencies%2520with%2520G-quadruplex-interacting%2520compounds%26jtitle%3DMol.%2520Cell%26date%3D2016%26volume%3D61%26spage%3D449%26epage%3D460%26doi%3D10.1016%2Fj.molcel.2015.12.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Antonio, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKinney, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mathew, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ho, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Neil, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dos Santos, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silvester, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia, J.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">CX-5461 is a DNA G-quadruplex stabilizer with selective lethality in BRCA1/2 deficient tumours</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">14432</span> <span class="refDoi"> DOI: 10.1038/ncomms14432</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=10.1038%2Fncomms14432" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=28211448" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjtVKms7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&author=H.+Xuauthor=M.+Di+Antonioauthor=S.+McKinneyauthor=V.+Mathewauthor=B.+Hoauthor=N.+J.+O%E2%80%99Neilauthor=N.+Dos+Santosauthor=J.+Silvesterauthor=V.+Weiauthor=J.+Garcia&title=CX-5461+is+a+DNA+G-quadruplex+stabilizer+with+selective+lethality+in+BRCA1%2F2+deficient+tumours&doi=10.1038%2Fncomms14432"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">CX-5461 is a DNA G-quadruplex stabilizer with selective lethality in BRCA1/2 deficient tumours</span></div><div class="casAuthors">Xu, Hong; Di Antonio, Marco; McKinney, Steven; Mathew, Veena; Ho, Brandon; O'Neil, Nigel J.; Dos Santos, Nancy; Silvester, Jennifer; Wei, Vivien; Garcia, Jessica; Kabeer, Farhia; Lai, Daniel; Soriano, Priscilla; Banath, Judit; Chiu, Derek S.; Yap, Damian; Le, Daniel D.; Ye, Frank B.; Zhang, Anni; Thu, Kelsie; Soong, John; Lin, Shu-chuan; Tsai, Angela Hsin Chin; Osako, Tomo; Algara, Teresa; Saunders, Darren N.; Wong, Jason; Xian, Jian; Bally, Marcel B.; Brenton, James D.; Brown, Grant W.; Shah, Sohrab P.; Cescon, David; Mak, Tak W.; Caldas, Carlos; Stirling, Peter C.; Hieter, Phil; Balasubramanian, Shankar; Aparicio, Samuel</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">14432</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">G-quadruplex DNAs form four-stranded helical structures and are proposed to play key roles in different cellular processes.  Targeting G-quadruplex DNAs for cancer treatment is a very promising prospect.  Here, we show that CX-5461 is a G-quadruplex stabilizer, with specific toxicity against BRCA deficiencies in cancer cells and polyclonal patient-derived xenograft models, including tumors resistant to PARP inhibition.  Exposure to CX-5461, and its related drug CX-3543, blocks replication forks and induces ssDNA gaps or breaks.  The BRCA and NHEJ pathways are required for the repair of CX-5461 and CX-3543-induced DNA damage and failure to do so leads to lethality.  These data strengthen the concept of G4 targeting as a therapeutic approach, specifically for targeting HR and NHEJ deficient cancers and other tumors deficient for DNA damage repair.  CX-5461 is now in advanced phase I clin. trial for patients with BRCA1/2 deficient tumors (Canadian trial, NCT02719977, opened May 2016).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGppKJRejuN5X7Vg90H21EOLACvtfcHk0lgVfCJ4G52l4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjtVKms7s%253D&md5=8e496952f88642ff05eaf136fe6856d6</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2Fncomms14432&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms14432%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DH.%26aulast%3DDi%2BAntonio%26aufirst%3DM.%26aulast%3DMcKinney%26aufirst%3DS.%26aulast%3DMathew%26aufirst%3DV.%26aulast%3DHo%26aufirst%3DB.%26aulast%3DO%25E2%2580%2599Neil%26aufirst%3DN.%2BJ.%26aulast%3DDos%2BSantos%26aufirst%3DN.%26aulast%3DSilvester%26aufirst%3DJ.%26aulast%3DWei%26aufirst%3DV.%26aulast%3DGarcia%26aufirst%3DJ.%26atitle%3DCX-5461%2520is%2520a%2520DNA%2520G-quadruplex%2520stabilizer%2520with%2520selective%2520lethality%2520in%2520BRCA1%252F2%2520deficient%2520tumours%26jtitle%3DNat.%2520Commun.%26date%3D2017%26volume%3D8%26doi%3D10.1038%2Fncomms14432" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lemmens, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Schendel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tijsterman, M.</span></span> <span> </span><span class="NLM_article-title">Mutagenic consequences of a single G-quadruplex demonstrate mitotic inheritance of DNA replication fork barriers</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>, <span class="NLM_elocation-id">8909</span> <span class="refDoi"> DOI: 10.1038/ncomms9909</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=10.1038%2Fncomms9909" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=26563448" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVOmsbbK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&author=B.+Lemmensauthor=R.+Van+Schendelauthor=M.+Tijsterman&title=Mutagenic+consequences+of+a+single+G-quadruplex+demonstrate+mitotic+inheritance+of+DNA+replication+fork+barriers&doi=10.1038%2Fncomms9909"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Mutagenic consequences of a single G-quadruplex demonstrate mitotic inheritance of DNA replication fork barriers</span></div><div class="casAuthors">Lemmens, Bennie; van Schendel, Robin; Tijsterman, Marcel</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">8909</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Faithful DNA replication is vital to prevent disease-causing mutations, chromosomal aberrations and malignant transformation.  However, accuracy conflicts with pace and flexibility and cells rely on specialized polymerases and helicases to ensure effective and timely replication of genomes that contain DNA lesions or secondary structures.  If and how cells can tolerate a permanent barrier to replication is, however, unknown.  Here we show that a single unresolved G-quadruplexed DNA structure can persist through multiple mitotic divisions without changing conformation.  Failed replication across a G-quadruplex causes single-strand DNA gaps that give rise to DNA double-strand breaks in subsequent cell divisions, which are processed by polymerase theta (POLQ)-mediated alternative end joining.  Lineage tracing expts. further reveal that persistent G-quadruplexes cause genetic heterogeneity during organ development.  Our data demonstrate that a single lesion can cause multiple unique genomic rearrangements, and that alternative end joining enables cells to proliferate in the presence of mitotically inherited replication blocks.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqp8h0JCrSGUrVg90H21EOLACvtfcHk0lgVfCJ4G52l4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVOmsbbK&md5=db43b84764420d505e3c6dba0bc0c70d</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1038%2Fncomms9909&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms9909%26sid%3Dliteratum%253Aachs%26aulast%3DLemmens%26aufirst%3DB.%26aulast%3DVan%2BSchendel%26aufirst%3DR.%26aulast%3DTijsterman%26aufirst%3DM.%26atitle%3DMutagenic%2520consequences%2520of%2520a%2520single%2520G-quadruplex%2520demonstrate%2520mitotic%2520inheritance%2520of%2520DNA%2520replication%2520fork%2520barriers%26jtitle%3DNat.%2520Commun.%26date%3D2015%26volume%3D6%26doi%3D10.1038%2Fncomms9909" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bochman, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paeschke, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zakian, V. A.</span></span> <span> </span><span class="NLM_article-title">DNA secondary structures: stability and function of G-quadruplex structures</span>. <i>Nat. Rev. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">770</span>– <span class="NLM_lpage">780</span>, <span class="refDoi"> DOI: 10.1038/nrg3296</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=10.1038%2Fnrg3296" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=23032257" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVCisb%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2012&pages=770-780&author=M.+L.+Bochmanauthor=K.+Paeschkeauthor=V.+A.+Zakian&title=DNA+secondary+structures%3A+stability+and+function+of+G-quadruplex+structures&doi=10.1038%2Fnrg3296"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">DNA secondary structures: stability and function of structures</span></div><div class="casAuthors">Bochman, Matthew L.; Paeschke, Katrin; Zakian, Virginia A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Genetics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">770-780</span>CODEN:
                <span class="NLM_cas:coden">NRGAAM</span>;
        ISSN:<span class="NLM_cas:issn">1471-0056</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  In addn. to the canonical double helix, DNA can fold into various other inter- and intramol. secondary structures.  Although many such structures were long thought to be in vitro artifacts, bioinformatics demonstrates that DNA sequences capable of forming these structures are conserved throughout evolution, suggesting the existence of non-B-form DNA in vivo.  In addn., genes whose products promote formation or resoln. of these structures are found in diverse organisms, and a growing body of work suggests that the resoln. of DNA secondary structures is crit. for genome integrity.  This Review focuses on emerging evidence relating to the characteristics of G-quadruplex structures and the possible influence of such structures on genomic stability and cellular processes, such as transcription.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpov7zDhgxxprVg90H21EOLACvtfcHk0lgVfCJ4G52l4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVCisb%252FE&md5=9d70afb3ef1bca023874bcede128223a</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fnrg3296&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrg3296%26sid%3Dliteratum%253Aachs%26aulast%3DBochman%26aufirst%3DM.%2BL.%26aulast%3DPaeschke%26aufirst%3DK.%26aulast%3DZakian%26aufirst%3DV.%2BA.%26atitle%3DDNA%2520secondary%2520structures%253A%2520stability%2520and%2520function%2520of%2520G-quadruplex%2520structures%26jtitle%3DNat.%2520Rev.%2520Genet.%26date%3D2012%26volume%3D13%26spage%3D770%26epage%3D780%26doi%3D10.1038%2Fnrg3296" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hänsel-Hertsch, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Antonio, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balasubramanian, S.</span></span> <span> </span><span class="NLM_article-title">DNA G-quadruplexes in the human genome: detection, functions and therapeutic potential</span>. <i>Nat. Rev. Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">279</span>, <span class="refDoi"> DOI: 10.1038/nrm.2017.3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=10.1038%2Fnrm.2017.3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=28225080" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjt1agsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=279&author=R.+H%C3%A4nsel-Hertschauthor=M.+Di+Antonioauthor=S.+Balasubramanian&title=DNA+G-quadruplexes+in+the+human+genome%3A+detection%2C+functions+and+therapeutic+potential&doi=10.1038%2Fnrm.2017.3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">DNA G-quadruplexes in the human genome: detection, functions and therapeutic potential</span></div><div class="casAuthors">Hansel-Hertsch, Robert; Di Antonio, Marco; Balasubramanian, Shankar</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">279-284</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Single-stranded guanine-rich DNA sequences can fold into four-stranded DNA structures called G-quadruplexes (G4s) that arise from the self-stacking of two or more guanine quartets.  There has been considerable recent progress in the detection and mapping of G4 structures in the human genome and in biol. relevant contexts.  These advancements, many of which align with predictions made previously in computational studies, provide important new insights into the functions of G4 structures in, for example, the regulation of transcription and genome stability, and uncover their potential relevance for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojysTqYKqZi7Vg90H21EOLACvtfcHk0lgVfCJ4G52l4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjt1agsr0%253D&md5=452d13815a3284a4ca51607d1522fb5f</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1038%2Fnrm.2017.3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm.2017.3%26sid%3Dliteratum%253Aachs%26aulast%3DH%25C3%25A4nsel-Hertsch%26aufirst%3DR.%26aulast%3DDi%2BAntonio%26aufirst%3DM.%26aulast%3DBalasubramanian%26aufirst%3DS.%26atitle%3DDNA%2520G-quadruplexes%2520in%2520the%2520human%2520genome%253A%2520detection%252C%2520functions%2520and%2520therapeutic%2520potential%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2017%26volume%3D18%26spage%3D279%26doi%3D10.1038%2Fnrm.2017.3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vuong, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stefan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lejault, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rousselin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denat, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monchaud, D.</span></span> <span> </span><span class="NLM_article-title">Identifying three-way DNA junction-specific small-molecules</span>. <i>Biochimie</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>94</i></span>,  <span class="NLM_fpage">442</span>– <span class="NLM_lpage">450</span>, <span class="refDoi"> DOI: 10.1016/j.biochi.2011.08.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=10.1016%2Fj.biochi.2011.08.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=21884749" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=1%3ACAS%3A528%3ADC%252BC38XovFCqsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=94&publication_year=2012&pages=442-450&author=S.+Vuongauthor=L.+Stefanauthor=P.+Lejaultauthor=Y.+Rousselinauthor=F.+Denatauthor=D.+Monchaud&title=Identifying+three-way+DNA+junction-specific+small-molecules&doi=10.1016%2Fj.biochi.2011.08.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Identifying three-way DNA junction-specific small-molecules</span></div><div class="casAuthors">Vuong, Sophie; Stefan, Loic; Lejault, Pauline; Rousselin, Yoann; Denat, Franck; Monchaud, David</div><div class="citationInfo"><span class="NLM_cas:title">Biochimie</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">94</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">442-450</span>CODEN:
                <span class="NLM_cas:coden">BICMBE</span>;
        ISSN:<span class="NLM_cas:issn">0300-9084</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Three-way junction DNA (TWJ-DNA, also known as 3WJ-DNA) is an alternative secondary DNA structure comprised of three duplex-DNAs that converge towards a single point, termed the branch point.  This point is characterized by unique geometrical properties that make its specific targeting by synthetic small-mols. possible.  Such a targeting has already been demonstrated in the solid state but not thoroughly biophys. investigated in soln.  Herein, a set of simple biophys. assays has been developed to identify TWJ-specific small-mol. ligands; these assays, inspired by the considerable body of work that has been reported to characterize the interactions between small-mols. and other higher-order DNA (notably quadruplex-DNA), have been calibrated with a known non-specific DNA binder (the porphyrin TMPyP4) and validated via the study of a small series of triazacyclononane (TACN) derivs. (metal-free or not) and the identification of a fairly-affinic and exquisitely TWJ-selective candidate (a TACN-quinoline construct named TACN-Q).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpQFURKgDIsMLVg90H21EOLACvtfcHk0ljs8mxK5Ri5gg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XovFCqsA%253D%253D&md5=22e86bc003f218a8e78097401a7afda0</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.biochi.2011.08.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.biochi.2011.08.012%26sid%3Dliteratum%253Aachs%26aulast%3DVuong%26aufirst%3DS.%26aulast%3DStefan%26aufirst%3DL.%26aulast%3DLejault%26aufirst%3DP.%26aulast%3DRousselin%26aufirst%3DY.%26aulast%3DDenat%26aufirst%3DF.%26aulast%3DMonchaud%26aufirst%3DD.%26atitle%3DIdentifying%2520three-way%2520DNA%2520junction-specific%2520small-molecules%26jtitle%3DBiochimie%26date%3D2012%26volume%3D94%26spage%3D442%26epage%3D450%26doi%3D10.1016%2Fj.biochi.2011.08.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Novotna, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laguerre, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Granzhan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pirrotta, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teulade-Fichou, M.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monchaud, D.</span></span> <span> </span><span class="NLM_article-title">Cationic azacryptands as selective three-way DNA junction binding agents</span>. <i>Org. Biomol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">215</span>– <span class="NLM_lpage">222</span>, <span class="refDoi"> DOI: 10.1039/C4OB01846J</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=10.1039%2FC4OB01846J" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2015&pages=215-222&author=J.+Novotnaauthor=A.+Laguerreauthor=A.+Granzhanauthor=M.+Pirrottaauthor=M.-P.+Teulade-Fichouauthor=D.+Monchaud&title=Cationic+azacryptands+as+selective+three-way+DNA+junction+binding+agents&doi=10.1039%2FC4OB01846J"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1039%2FC4OB01846J&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC4OB01846J%26sid%3Dliteratum%253Aachs%26aulast%3DNovotna%26aufirst%3DJ.%26aulast%3DLaguerre%26aufirst%3DA.%26aulast%3DGranzhan%26aufirst%3DA.%26aulast%3DPirrotta%26aufirst%3DM.%26aulast%3DTeulade-Fichou%26aufirst%3DM.-P.%26aulast%3DMonchaud%26aufirst%3DD.%26atitle%3DCationic%2520azacryptands%2520as%2520selective%2520three-way%2520DNA%2520junction%2520binding%2520agents%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2015%26volume%3D13%26spage%3D215%26epage%3D222%26doi%3D10.1039%2FC4OB01846J" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guyon, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pirrotta, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duskova, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Granzhan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teulade-Fichou, M.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monchaud, D.</span></span> <span> </span><span class="NLM_article-title">TWJ-Screen: an isothermal screening assay to assess ligand/DNA junction interactions in vitro</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>46</i></span>, <span class="NLM_elocation-id">e16</span> <span class="refDoi"> DOI: 10.1093/nar/gkx1118</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=10.1093%2Fnar%2Fgkx1118" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2018&author=L.+Guyonauthor=M.+Pirrottaauthor=K.+Duskovaauthor=A.+Granzhanauthor=M.-P.+Teulade-Fichouauthor=D.+Monchaud&title=TWJ-Screen%3A+an+isothermal+screening+assay+to+assess+ligand%2FDNA+junction+interactions+in+vitro&doi=10.1093%2Fnar%2Fgkx1118"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkx1118&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkx1118%26sid%3Dliteratum%253Aachs%26aulast%3DGuyon%26aufirst%3DL.%26aulast%3DPirrotta%26aufirst%3DM.%26aulast%3DDuskova%26aufirst%3DK.%26aulast%3DGranzhan%26aufirst%3DA.%26aulast%3DTeulade-Fichou%26aufirst%3DM.-P.%26aulast%3DMonchaud%26aufirst%3DD.%26atitle%3DTWJ-Screen%253A%2520an%2520isothermal%2520screening%2520assay%2520to%2520assess%2520ligand%252FDNA%2520junction%2520interactions%2520in%2520vitro%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2018%26volume%3D46%26doi%3D10.1093%2Fnar%2Fgkx1118" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Oleksi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blanco, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Usón, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aymamí, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodger, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hannon, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coll, M.</span></span> <span> </span><span class="NLM_article-title">Molecular recognition of a three-way DNA junction by a metallosupramolecular helicate</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">1227</span>– <span class="NLM_lpage">1231</span>, <span class="refDoi"> DOI: 10.1002/anie.200503822</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=10.1002%2Fanie.200503822" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=1%3ACAS%3A528%3ADC%252BD28XhvFyjsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2006&pages=1227-1231&author=A.+Oleksiauthor=A.+G.+Blancoauthor=R.+Boerauthor=I.+Us%C3%B3nauthor=J.+Aymam%C3%ADauthor=A.+Rodgerauthor=M.+J.+Hannonauthor=M.+Coll&title=Molecular+recognition+of+a+three-way+DNA+junction+by+a+metallosupramolecular+helicate&doi=10.1002%2Fanie.200503822"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular recognition of a three-way DNA junction by a metallosupramolecular helicate</span></div><div class="casAuthors">Oleksi, Aneta; Blanco, Alexandre G.; Boer, Roeland; Uson, Isabel; Aymami, Joan; Rodger, Alison; Hannon, Michael J.; Coll, Miquel</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1227-1231</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Three point turn: A metallosupramol. helicate with trigonal-antiprismatic geometry fits perfectly into the central hydrophobic cavity of a three-way DNA junction allowing a new mode of DNA recognition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWvdABofmYkLVg90H21EOLACvtfcHk0ljs8mxK5Ri5gg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhvFyjsr0%253D&md5=28e2147e5ab2cb533bc33aee6728ef40</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1002%2Fanie.200503822&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.200503822%26sid%3Dliteratum%253Aachs%26aulast%3DOleksi%26aufirst%3DA.%26aulast%3DBlanco%26aufirst%3DA.%2BG.%26aulast%3DBoer%26aufirst%3DR.%26aulast%3DUs%25C3%25B3n%26aufirst%3DI.%26aulast%3DAymam%25C3%25AD%26aufirst%3DJ.%26aulast%3DRodger%26aufirst%3DA.%26aulast%3DHannon%26aufirst%3DM.%2BJ.%26aulast%3DColl%26aufirst%3DM.%26atitle%3DMolecular%2520recognition%2520of%2520a%2520three-way%2520DNA%2520junction%2520by%2520a%2520metallosupramolecular%2520helicate%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2006%26volume%3D45%26spage%3D1227%26epage%3D1231%26doi%3D10.1002%2Fanie.200503822" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cerasino, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hannon, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sletten, E.</span></span> <span> </span><span class="NLM_article-title">DNA three-way junction with a dinuclear iron (II) supramolecular helicate at the center: A NMR structural study</span>. <i>Inorg. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">6245</span>– <span class="NLM_lpage">6251</span>, <span class="refDoi"> DOI: 10.1021/ic062415c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ic062415c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2007&pages=6245-6251&author=L.+Cerasinoauthor=M.+J.+Hannonauthor=E.+Sletten&title=DNA+three-way+junction+with+a+dinuclear+iron+%28II%29+supramolecular+helicate+at+the+center%3A+A+NMR+structural+study&doi=10.1021%2Fic062415c"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fic062415c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fic062415c%26sid%3Dliteratum%253Aachs%26aulast%3DCerasino%26aufirst%3DL.%26aulast%3DHannon%26aufirst%3DM.%2BJ.%26aulast%3DSletten%26aufirst%3DE.%26atitle%3DDNA%2520three-way%2520junction%2520with%2520a%2520dinuclear%2520iron%2520%2528II%2529%2520supramolecular%2520helicate%2520at%2520the%2520center%253A%2520A%2520NMR%2520structural%2520study%26jtitle%3DInorg.%2520Chem.%26date%3D2007%26volume%3D46%26spage%3D6245%26epage%3D6251%26doi%3D10.1021%2Fic062415c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Malina, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hannon, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brabec, V.</span></span> <span> </span><span class="NLM_article-title">Recognition of DNA three-way junctions by metallosupramolecular cylinders: gel electrophoresis studies</span>. <i>Chem. Eur. J.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">3871</span>– <span class="NLM_lpage">3877</span>, <span class="refDoi"> DOI: 10.1002/chem.200700159</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=10.1002%2Fchem.200700159" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=17397023" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=1%3ACAS%3A528%3ADC%252BD2sXlsl2it70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2007&pages=3871-3877&author=J.+Malinaauthor=M.+J.+Hannonauthor=V.+Brabec&title=Recognition+of+DNA+three-way+junctions+by+metallosupramolecular+cylinders%3A+gel+electrophoresis+studies&doi=10.1002%2Fchem.200700159"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Recognition of DNA three-way junctions by metallosupramolecular cylinders: gel electrophoresis studies</span></div><div class="casAuthors">Malina, Jaroslav; Hannon, Michael J.; Brabec, Viktor</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry - A European Journal</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">3871-3877</span>CODEN:
                <span class="NLM_cas:coden">CEUJED</span>;
        ISSN:<span class="NLM_cas:issn">0947-6539</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The interaction of metallosupramol. cylinders with DNA three-way junctions has been studied by gel electrophoresis.  A recent X-ray crystal structure of a palindromic oligonucleotide forming part of a complex with such a cylinder revealed binding at the heart of a three-way junction structure.  The studies reported herein confirm that this is not solely an artifact of crystn. and reveal that this is a potentially very powerful new mode of DNA recognition with wide scope.  The cylinders are much more effective at stabilizing three-way junctions than simple magnesium dications or org. or metallo-org. tetracations, with the M cylinder enantiomer being more effective than P.  The recognition is not restricted to three-way junctions formed from palindromic DNA with a central AT step at the junction; non-palindromic three-way junctions and those with GC steps are also stabilized.  The cylinder is also revealed to stabilize other Y-shaped junctions, such as that formed at a fraying point in duplex DNA (for example, a replication fork), and other DNA three-way junction structures, such as those contg. unpaired nucleotides, perhaps by opening up this structure to access the central cavity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqb4tvdsmAUMbVg90H21EOLACvtfcHk0lh7mI-I9RPNqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXlsl2it70%253D&md5=9c8795a6c23168be76227b551a5c6688</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1002%2Fchem.200700159&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fchem.200700159%26sid%3Dliteratum%253Aachs%26aulast%3DMalina%26aufirst%3DJ.%26aulast%3DHannon%26aufirst%3DM.%2BJ.%26aulast%3DBrabec%26aufirst%3DV.%26atitle%3DRecognition%2520of%2520DNA%2520three-way%2520junctions%2520by%2520metallosupramolecular%2520cylinders%253A%2520gel%2520electrophoresis%2520studies%26jtitle%3DChem.%2520Eur.%2520J.%26date%3D2007%26volume%3D13%26spage%3D3871%26epage%3D3877%26doi%3D10.1002%2Fchem.200700159" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hotze, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hodges, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayden, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanchez-Cano, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paines, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Male, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tse, M.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunce, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chipman, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hannon, M. J.</span></span> <span> </span><span class="NLM_article-title">Supramolecular iron cylinder with unprecedented DNA binding is a potent cytostatic and apoptotic agent without exhibiting genotoxicity</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">1258</span>– <span class="NLM_lpage">1267</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2008.10.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=10.1016%2Fj.chembiol.2008.10.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=19101470" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsFajt7rF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2008&pages=1258-1267&author=A.+C.+Hotzeauthor=N.+J.+Hodgesauthor=R.+E.+Haydenauthor=C.+Sanchez-Canoauthor=C.+Painesauthor=N.+Maleauthor=M.-K.+Tseauthor=C.+M.+Bunceauthor=J.+K.+Chipmanauthor=M.+J.+Hannon&title=Supramolecular+iron+cylinder+with+unprecedented+DNA+binding+is+a+potent+cytostatic+and+apoptotic+agent+without+exhibiting+genotoxicity&doi=10.1016%2Fj.chembiol.2008.10.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Supramolecular Iron Cylinder with Unprecedented DNA Binding Is a Potent Cytostatic and Apoptotic Agent without Exhibiting Genotoxicity</span></div><div class="casAuthors">Hotze, Anna C. G.; Hodges, Nikolas J.; Hayden, Rachel E.; Sanchez-Cano, Carlos; Paines, Christopher; Male, Natalia; Tse, Man-Kit; Bunce, Chris M.; Chipman, J. Kevin; Hannon, Michael J.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1258-1267</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The supramol. iron cylinder, [Fe2L3]Cl4 (L = C25H20N4), shows unprecedented DNA binding in vitro, inducing intramol. DNA coiling and also targeting Y-shaped DNA junctions.  Its effects on proliferation and survival in both tumor and normal cell lines were investigated.  Iron cylinder reduced mitochondrial activity of cultures with potency similar to cisplatin, inhibited the cell cycle, and increased cell death by apoptosis.  Assocd. with this, a lowering of the assocn. of propidium iodide (PI) with cellular DNA was obsd., consistent with an obsd. competitive displacement of PI from naked DNA by cylinders.  Importantly, and in contrast to existing anticancer drugs such as cisplatin, the iron cylinder [Fe2L3]4+ was not genotoxic.  In summary, the design of metal complexes such as [Fe2L3]4+ with potential anticancer properties in the absence of genotoxicity may represent a significant step toward therapeutic advancement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4a9ryNz_fp7Vg90H21EOLACvtfcHk0lh7mI-I9RPNqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsFajt7rF&md5=765d8fc5f9abce744a01374f10b84270</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2008.10.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2008.10.016%26sid%3Dliteratum%253Aachs%26aulast%3DHotze%26aufirst%3DA.%2BC.%26aulast%3DHodges%26aufirst%3DN.%2BJ.%26aulast%3DHayden%26aufirst%3DR.%2BE.%26aulast%3DSanchez-Cano%26aufirst%3DC.%26aulast%3DPaines%26aufirst%3DC.%26aulast%3DMale%26aufirst%3DN.%26aulast%3DTse%26aufirst%3DM.-K.%26aulast%3DBunce%26aufirst%3DC.%2BM.%26aulast%3DChipman%26aufirst%3DJ.%2BK.%26aulast%3DHannon%26aufirst%3DM.%2BJ.%26atitle%3DSupramolecular%2520iron%2520cylinder%2520with%2520unprecedented%2520DNA%2520binding%2520is%2520a%2520potent%2520cytostatic%2520and%2520apoptotic%2520agent%2520without%2520exhibiting%2520genotoxicity%26jtitle%3DChem.%2520Biol.%26date%3D2008%26volume%3D15%26spage%3D1258%26epage%3D1267%26doi%3D10.1016%2Fj.chembiol.2008.10.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boer, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerckhoffs, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parajo, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pascu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Usón, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lincoln, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hannon, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coll, M.</span></span> <span> </span><span class="NLM_article-title">Self-Assembly of functionalizable two-component 3D DNA arrays through the induced formation of DNA three-way-Junction branch points by supramolecular cylinders</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">2336</span>– <span class="NLM_lpage">2339</span>, <span class="refDoi"> DOI: 10.1002/anie.200906742</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=10.1002%2Fanie.200906742" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjslGlur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2010&pages=2336-2339&author=D.+R.+Boerauthor=J.+M.+Kerckhoffsauthor=Y.+Parajoauthor=M.+Pascuauthor=I.+Us%C3%B3nauthor=P.+Lincolnauthor=M.+J.+Hannonauthor=M.+Coll&title=Self-Assembly+of+functionalizable+two-component+3D+DNA+arrays+through+the+induced+formation+of+DNA+three-way-Junction+branch+points+by+supramolecular+cylinders&doi=10.1002%2Fanie.200906742"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Self-Assembly of Functionalizable Two-Component 3D DNA Arrays through the Induced Formation of DNA Three-Way-Junction Branch Points by Supramolecular Cylinders</span></div><div class="casAuthors">Boer, D. Roeland; Kerckhoffs, Jessica M. C. A.; Parajo, Yolanda; Pascu, Mirela; Uson, Isabel; Lincoln, Per; Hannon, Michael J.; Coll, Miquel</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">2336-2339, S2336/1-S2336/13</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">We report the design of 3D DNA assemblies composed of short palindromic oligonucleotides, based on the use of three-way junctions (3WJs) as branching points.  The advantage of using 3WJs lies in the fact that they are specifically recognized by a class of helicate coordination compds. called supramol. cylinders.  These mols. can thus be introduced noncovalently into the 3WJ branching points of fully base paired 3WJs and 3WJs that contain unpaired bases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkQy3UMK5QwLVg90H21EOLACvtfcHk0lh7mI-I9RPNqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjslGlur8%253D&md5=b910b76e2dd72f19265970f24f1490e5</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1002%2Fanie.200906742&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.200906742%26sid%3Dliteratum%253Aachs%26aulast%3DBoer%26aufirst%3DD.%2BR.%26aulast%3DKerckhoffs%26aufirst%3DJ.%2BM.%26aulast%3DParajo%26aufirst%3DY.%26aulast%3DPascu%26aufirst%3DM.%26aulast%3DUs%25C3%25B3n%26aufirst%3DI.%26aulast%3DLincoln%26aufirst%3DP.%26aulast%3DHannon%26aufirst%3DM.%2BJ.%26aulast%3DColl%26aufirst%3DM.%26atitle%3DSelf-Assembly%2520of%2520functionalizable%2520two-component%25203D%2520DNA%2520arrays%2520through%2520the%2520induced%2520formation%2520of%2520DNA%2520three-way-Junction%2520branch%2520points%2520by%2520supramolecular%2520cylinders%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2010%26volume%3D49%26spage%3D2336%26epage%3D2339%26doi%3D10.1002%2Fanie.200906742" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Barros, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chenoweth, D. M.</span></span> <span> </span><span class="NLM_article-title">Recognition of nucleic acid junctions using triptycene-based molecules</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">13746</span>– <span class="NLM_lpage">13750</span>, <span class="refDoi"> DOI: 10.1002/anie.201407061</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=10.1002%2Fanie.201407061" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitVCit73O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2014&pages=13746-13750&author=S.+A.+Barrosauthor=D.+M.+Chenoweth&title=Recognition+of+nucleic+acid+junctions+using+triptycene-based+molecules&doi=10.1002%2Fanie.201407061"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Recognition of Nucleic Acid Junctions Using Triptycene-Based Molecules</span></div><div class="casAuthors">Barros, Stephanie A.; Chenoweth, David M.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">50</span>),
    <span class="NLM_cas:pages">13746-13750</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The modulation of nucleic acids by small mols. is an essential process across the kingdoms of life.  Targeting nucleic acids with small mols. represents a significant challenge at the forefront of chem. biol.  Nucleic acid junctions are ubiquitous structural motifs in nature and in designed materials.  Herein, we describe a new class of structure-specific nucleic acid junction stabilizers based on a triptycene scaffold.  Triptycenes provide significant stabilization of DNA and RNA three-way junctions, providing a new scaffold for the development of nucleic acid junction binders with enhanced recognition properties.  Addnl., we report cytotoxicity and cell uptake data in two human ovarian carcinoma cell lines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrsalKqYAXH4LVg90H21EOLACvtfcHk0liFBpecgxPy_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitVCit73O&md5=b3d59010e6f130eab638869353164ca2</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1002%2Fanie.201407061&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201407061%26sid%3Dliteratum%253Aachs%26aulast%3DBarros%26aufirst%3DS.%2BA.%26aulast%3DChenoweth%26aufirst%3DD.%2BM.%26atitle%3DRecognition%2520of%2520nucleic%2520acid%2520junctions%2520using%2520triptycene-based%2520molecules%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2014%26volume%3D53%26spage%3D13746%26epage%3D13750%26doi%3D10.1002%2Fanie.201407061" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Barros, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chenoweth, D. M.</span></span> <span> </span><span class="NLM_article-title">Triptycene-based small molecules modulate (CAG)·(CTG) repeat junctions</span>. <i>Chem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">4752</span>– <span class="NLM_lpage">4755</span>, <span class="refDoi"> DOI: 10.1039/C5SC01595B</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=10.1039%2FC5SC01595B" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=26366282" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVSgs7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=4752-4755&author=S.+A.+Barrosauthor=D.+M.+Chenoweth&title=Triptycene-based+small+molecules+modulate+%28CAG%29%C2%B7%28CTG%29+repeat+junctions&doi=10.1039%2FC5SC01595B"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Triptycene-based small molecules modulate (CAG)·(CTG) repeat junctions</span></div><div class="casAuthors">Barros, Stephanie A.; Chenoweth, David M.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Science</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">4752-4755</span>CODEN:
                <span class="NLM_cas:coden">CSHCCN</span>;
        ISSN:<span class="NLM_cas:issn">2041-6520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Nucleic acid three-way junctions (3WJs) play key roles in biol. processes such as nucleic acid replication in addn. to being implicated as dynamic transient intermediates in trinucleotide repeat sequences.  Structural modulation of specific nucleic acid junctions could allow for control of biol. processes and disease states at the nucleic acid level.  Trinucleotide repeat expansions are assocd. with several neurodegenerative diseases where dynamic slippage is thought to occur during replication, forming transient 3WJ intermediates with the complementary strand.  Here, we report triptycene-based mols. that bind to a d(CAG)·(CTG) repeat using a gel shift assay, fluorescence-quenching and CD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrjGX8rbzmZEbVg90H21EOLACvtfcHk0liFBpecgxPy_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVSgs7rP&md5=140e24608fdb2513be74fc685ba0e3ac</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1039%2FC5SC01595B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC5SC01595B%26sid%3Dliteratum%253Aachs%26aulast%3DBarros%26aufirst%3DS.%2BA.%26aulast%3DChenoweth%26aufirst%3DD.%2BM.%26atitle%3DTriptycene-based%2520small%2520molecules%2520modulate%2520%2528CAG%2529%25C2%25B7%2528CTG%2529%2520repeat%2520junctions%26jtitle%3DChem.%2520Sci.%26date%3D2015%26volume%3D6%26spage%3D4752%26epage%3D4755%26doi%3D10.1039%2FC5SC01595B" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, P.</span></span> <span> </span><span class="NLM_article-title">Supramolecular recognition of three way junctions DNA by a cationic calix [3] carbazole</span>. <i>Chem. Eur. J.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">6087</span>– <span class="NLM_lpage">6093</span>, <span class="refDoi"> DOI: 10.1002/chem.201705564</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=10.1002%2Fchem.201705564" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=29315943" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=1%3ACAS%3A528%3ADC%252BC1cXis1Snsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2018&pages=6087-6093&author=Z.+Yangauthor=Y.+Chenauthor=G.+Liauthor=Z.+Tianauthor=L.+Zhaoauthor=X.+Wuauthor=Q.+Maauthor=M.+Liuauthor=P.+Yang&title=Supramolecular+recognition+of+three+way+junctions+DNA+by+a+cationic+calix+%5B3%5D+carbazole&doi=10.1002%2Fchem.201705564"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Supramolecular Recognition of Three Way Junction DNA by a Cationic Calix[3]carbazole</span></div><div class="casAuthors">Yang, Zhaozheng; Chen, Yan; Li, Gang; Tian, Zhangmin; Zhao, Liang; Wu, Xi; Ma, Qi; Liu, Mingzhe; Yang, Peng</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry - A European Journal</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">6087-6093</span>CODEN:
                <span class="NLM_cas:coden">CEUJED</span>;
        ISSN:<span class="NLM_cas:issn">0947-6539</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">DNA three-way junctions (TWJ-DNA) are intermediate structures in DNA replication and/or recombination.  They play very important roles in biol. processes, but more subtle functions are still unknown due partially to the lack of a fluorescent ligand.  In this study, a cationic calix[3]carbazole (I) has been synthesized and its properties of interacting with TWJ-DNA have been evaluated by UV/Vis and fluorescence spectroscopy, CD, gel electrophoresis, and 1H NMR studies.  The results show that I binds to the central hydrophobic cavity of TWJ-DNA.  Moreover, it could selectively bind to TWJ-DNA over duplex and quadruplex DNA.  Furthermore, I possesses the capability of serving as the TWJ-DNA probe as its trap-II excimer emission is turned on by TWJ-DNA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDqZgCDmxcBrVg90H21EOLACvtfcHk0liFBpecgxPy_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXis1Snsbs%253D&md5=19162337173f4eedba50a642b871ab3e</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1002%2Fchem.201705564&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fchem.201705564%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DZ.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DG.%26aulast%3DTian%26aufirst%3DZ.%26aulast%3DZhao%26aufirst%3DL.%26aulast%3DWu%26aufirst%3DX.%26aulast%3DMa%26aufirst%3DQ.%26aulast%3DLiu%26aufirst%3DM.%26aulast%3DYang%26aufirst%3DP.%26atitle%3DSupramolecular%2520recognition%2520of%2520three%2520way%2520junctions%2520DNA%2520by%2520a%2520cationic%2520calix%2520%255B3%255D%2520carbazole%26jtitle%3DChem.%2520Eur.%2520J.%26date%3D2018%26volume%3D24%26spage%3D6087%26epage%3D6093%26doi%3D10.1002%2Fchem.201705564" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Van
Voorhis, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, J. H.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Open source drug discovery with the malaria box compound collection for neglected diseases and beyond</span>. <i>PLoS Pathog.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>, <span class="NLM_elocation-id">e1005763</span> <span class="refDoi"> DOI: 10.1371/journal.ppat.1005763</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=10.1371%2Fjournal.ppat.1005763" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=27467575" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvVelu7rL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&author=W.+C.+Van%0AVoorhisauthor=J.+H.+Adams&title=Open+source+drug+discovery+with+the+malaria+box+compound+collection+for+neglected+diseases+and+beyond&doi=10.1371%2Fjournal.ppat.1005763"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Open source drug discovery with the malaria box compound collection for neglected diseases and beyond</span></div><div class="casAuthors">Van Voorhis, Wesley C.; Adams, John H.; Adelfio, Roberto; Ahyong, Vida; Akabas, Myles H.; Alano, Pietro; Alday, Aintzane; Aleman Resto, Yesmalie; Alsibaee, Aishah; Alzualde, Ainhoa; Andrews, Katherine T.; Avery, Simon V.; Avery, Vicky M.; Ayong, Lawrence; Baker, Mark; Baker, Stephen; Ben Mamoun, Choukri; Bhatia, Sangeeta; Bickle, Quentin; Bounaadja, Lotfi; Bowling, Tana; Bosch, Jurgen; Boucher, Lauren E.; Boyom, Fabrice F.; Brea, Jose; Brennan, Marian; Burton, Audrey; Caffrey, Conor R.; Camarda, Grazia; Carrasquilla, Manuela; Carter, Dee; Cassera, Maria Belen; Cheng, Ken Chih-Chien; Chindaudomsate, Worathad; Chubb, Anthony; Colon, Beatrice L.; Colon-Lopez, Daisy D.; Corbett, Yolanda; Crowther, Gregory J.; Cowan, Noemi; D'Alessandro, Sarah; Dang, Na Le; Delves, Michael; De Risi, Joseph L.; Du, Alan Y.; Duffy, Sandra; El-Sayed, Shimaa Abd El-Salam; Ferdig, Michael T.; Fernandez Robledo, Jose A.; Fidock, David A.; Florent, Isabelle; Fokou, Patrick V. T.; Galstian, Ani; Gamo, Francisco Javier; Gokool, Suzanne; Gold, Ben; Golub, Todd; Goldgof, Gregory M.; Guha, Rajarshi; Guiguemde, W. Armand; Gural, Nil; Guy, R. Kiplin; Hansen, Michael A. E.; Hanson, Kirsten K.; Hemphill, Andrew; Hooft van Huijsduijnen, Rob; Horii, Takaaki; Horrocks, Paul; Hughes, Tyler B.; Huston, Christopher; Igarashi, Ikuo; Ingram-Sieber, Katrin; Itoe, Maurice A.; Jadhav, Ajit; Jensen, Amornrat Naranuntarat; Jensen, Laran T.; Jiang, Rays H. Y.; Kaiser, Annette; Keiser, Jennifer; Ketas, Thomas; Kicka, Sebastien; Kim, Sunyoung; Kirk, Kiaran; Kumar, Vidya P.; Kyle, Dennis E.; Lafuente, Maria Jose; Landfear, Scott; Lee, Nathan; Lee, Sukjun; Lehane, Adele M.; Li, Fengwu; Little, David; Liu, Liqiong; Llinas, Manuel; Loza, Maria I.; Lubar, Aristea; Lucantoni, Leonardo; Lucet, Isabelle; Maes, Louis; Mancama, Dalu; Mansour, Nuha R.; March, Sandra; McGowan, Sheena; Vera, Iset Medina; Meister, Stephan; Mercer, Luke; Mestres, Jordi; Mfopa, Alvine N.; Misra, Raj N.; Moon, Seunghyun; Moore, John P.; Morais Rodrigues da Costa, Francielly; Muller, Joachim; Muriana, Arantza; Hewitt, Stephen Nakazawa; Nare, Bakela; Nathan, Carl; Narraidoo, Nathalie; Nawaratna, Sujeevi; Ojo, Kayode K.; Ortiz, Diana; Panic, Gordana; Papadatos, George; Parapini, Silvia; Patra, Kailash; Pham, Ngoc; Prats, Sarah; Plouffe, David M.; Poulsen, Sally-Ann; Pradhan, Anupam; Quevedo, Celia; Quinn, Ronald J.; Rice, Christopher A.; Rizk, Mohamed Abdo; Ruecker, Andrea; St. Onge, Robert; Ferreira, Rafaela Salgado; Samra, Jasmeet; Robinett, Natalie G.; Schlecht, Ulrich; Schmitt, Marjorie; Villela, Filipe Silva; Silvestrini, Francesco; Sinden, Robert; Smith, Dennis A.; Soldati, Thierry; Spitzmuller, Andreas; Stamm, Serge Maximilian; Sullivan, David J.; Sullivan, William; Suresh, Sundari; Suzuki, Brian M.; Suzuki, Yo; Swamidass, S. Joshua; Taramelli, Donatella; Tchokouaha, Lauve R. Y.; Theron, Anjo; Thomas, David; Tonissen, Kathryn F.; Townson, Simon; Tripathi, Abhai K.; Trofimov, Valentin; Udenze, Kenneth O.; Ullah, Imran; Vallieres, Cindy; Vigil, Edgar; Vinetz, Joseph M.; Vinh, Phat Voong; Vu, Hoan; Watanabe, Nao-aki; Weatherby, Kate; White, Pamela M.; Wilks, Andrew F.; Winzeler, Elizabeth A.; Wojcik, Edward; Wree, Melanie; Wu, Wesley; Yokoyama, Naoaki; Zollo, Paul H. A.; Abla, Nada; Blasco, Benjamin; Burrows, Jeremy; Laleu, Benoit; Leroy, Didier; Spangenberg, Thomas; Wells, Timothy; Willis, Paul A.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Pathogens</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">e1005763/1-e1005763/23</span>CODEN:
                <span class="NLM_cas:coden">PPLACN</span>;
        ISSN:<span class="NLM_cas:issn">1553-7374</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">A major cause of the paucity of new starting points for drug discovery is the lack of interaction between academia and industry.  Much of the global resource in biol. is present in universities, whereas the focus of medicinal chem. is still largely within industry.  Open source drug discovery, with sharing of information, is clearly a first step towards overcoming this gap.  But the interface could esp. be bridged through a scale-up of open sharing of phys. compds., which would accelerate the finding of new starting points for drug discovery.  The Medicines for Malaria Venture Malaria Box is a collection of over 400 compds. representing families of structures identified in phenotypic screens of pharmaceutical and academic libraries against the Plasmodium falciparum malaria parasite.  The set has now been distributed to almost 200 research groups globally in the last two years, with the only stipulation that information from the screens is deposited in the public domain.  This paper reports for the first time on 236 screens that have been carried out against the Malaria Box and compares these results with 55 assays that were previously published, in a format that allows a meta-anal. of the combined dataset.  The combined biochem. and cellular assays presented here suggest mechanisms of action for 135 (34%) of the compds. active in killing multiple life-cycle stages of the malaria parasite, including asexual blood, liver, gametocyte, gametes and insect ookinete stages.  In addn., many compds. demonstrated activity against other pathogens, showing hits in assays with 16 protozoa, 7 helminths, 9 bacterial and mycobacterial species, the dengue fever mosquito vector, and the NCI60 human cancer cell line panel of 60 human tumor cell lines.  Toxicol., pharmacokinetic and metabolic properties were collected on all the compds., assisting in the selection of the most promising candidates for murine proof-of-concept expts. and medicinal chem. programs.  The data for all of these assays are presented and analyzed to show how outstanding leads for many indications can be selected.  These results reveal the immense potential for translating the dispersed expertise in biol. assays involving human pathogens into drug discovery starting points, by providing open access to new families of mols., and emphasize how a small addnl. investment made to help acquire and distribute compds., and sharing the data, can catalyze drug discovery for dozens of different indications.  Another lesson is that when multiple screens from different groups are run on the same library, results can be integrated quickly to select the most valuable starting points for subsequent medicinal chem. efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrDKNb7zLhgFrVg90H21EOLACvtfcHk0lg69UZxkt9PLA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvVelu7rL&md5=0545e9836cf38a96b4ab000ca0a810bf</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1371%2Fjournal.ppat.1005763&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.ppat.1005763%26sid%3Dliteratum%253Aachs%26aulast%3DVan%2BVoorhis%26aufirst%3DW.%2BC.%26aulast%3DAdams%26aufirst%3DJ.%2BH.%26atitle%3DOpen%2520source%2520drug%2520discovery%2520with%2520the%2520malaria%2520box%2520compound%2520collection%2520for%2520neglected%2520diseases%2520and%2520beyond%26jtitle%3DPLoS%2520Pathog.%26date%3D2016%26volume%3D12%26doi%3D10.1371%2Fjournal.ppat.1005763" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wells, T. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willis, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burrows, J. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Huijsduijnen, R. H.</span></span> <span> </span><span class="NLM_article-title">Open data in drug discovery and development: lessons from malaria</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">661</span>– <span class="NLM_lpage">662</span>, <span class="refDoi"> DOI: 10.1038/nrd.2016.154</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=10.1038%2Fnrd.2016.154" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=27516171" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhtlehur7K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=661-662&author=T.+N.+Wellsauthor=P.+Willisauthor=J.+N.+Burrowsauthor=R.+H.+van+Huijsduijnen&title=Open+data+in+drug+discovery+and+development%3A+lessons+from+malaria&doi=10.1038%2Fnrd.2016.154"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Open data in drug discovery and development: lessons from malaria</span></div><div class="casAuthors">Wells, Timothy N. C.; Willis, Paul; Burrows, Jeremy N.; van Huijsduijnen, Rob Hooft</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">661-662</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">There is a growing consensus that drug discovery thrives in an open environment.  Here, we describe how the malaria community has embraced four levels of open data - open science, open innovation, open access and open source - to catalyze the development of new medicines, and consider principles that could enable open data approaches to be applied to other disease areas.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrYw2ky6KJ1CbVg90H21EOLACvtfcHk0lg69UZxkt9PLA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhtlehur7K&md5=52a8b28339e33e744cad544ace5ad513</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.154&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.154%26sid%3Dliteratum%253Aachs%26aulast%3DWells%26aufirst%3DT.%2BN.%26aulast%3DWillis%26aufirst%3DP.%26aulast%3DBurrows%26aufirst%3DJ.%2BN.%26aulast%3Dvan%2BHuijsduijnen%26aufirst%3DR.%2BH.%26atitle%3DOpen%2520data%2520in%2520drug%2520discovery%2520and%2520development%253A%2520lessons%2520from%2520malaria%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2016%26volume%3D15%26spage%3D661%26epage%3D662%26doi%3D10.1038%2Fnrd.2016.154" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mahuteau-Betzer, F.</span></span> <span> </span><span class="NLM_article-title">The french national compound library: advances and future prospects</span>. <i>Med. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">417</span>– <span class="NLM_lpage">422</span>, <span class="refDoi"> DOI: 10.1051/medsci/20153104016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=10.1051%2Fmedsci%2F20153104016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=1%3ACAS%3A280%3ADC%252BC2Mfis1Cjuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2015&pages=417-422&author=F.+Mahuteau-Betzer&title=The+french+national+compound+library%3A+advances+and+future+prospects&doi=10.1051%2Fmedsci%2F20153104016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">The French National Compound Library: advances and future prospects</span></div><div class="casAuthors">Mahuteau-Betzer Florence</div><div class="citationInfo"><span class="NLM_cas:title">Medecine sciences : M/S</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">417-22</span>
        ISSN:<span class="NLM_cas:issn">0767-0974</span>.
    </div><div class="casAbstract">The French National Compound Library (Chimiotheque Nationale) has been created in 2003 and is the federation of local collections.  It contains more than 56 000 small molecules and natural compounds synthesised or isolated in different laboratories over the past years.  This explains the diversity of the collection.  The strength of this initiative is the ability to connect chemists and biologists for the development of hits.  This development involves the synthesis of analogues or/and chemical tools to find new targets.  These collaborations lead to the identification of new chemical probes.  These probes able to modulate a biological function are essential to study biological pathways.  They can also be useful for therapeutic applications.  This article will describe the major achievements and perspectives of the French Chemical Library.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTjzt2Wqi1XWxgd2Wuil24tfW6udTcc2eaBzcNlTTv8jLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2Mfis1Cjuw%253D%253D&md5=db4eaa72173176a4393c72ad8316365e</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1051%2Fmedsci%2F20153104016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1051%252Fmedsci%252F20153104016%26sid%3Dliteratum%253Aachs%26aulast%3DMahuteau-Betzer%26aufirst%3DF.%26atitle%3DThe%2520french%2520national%2520compound%2520library%253A%2520advances%2520and%2520future%2520prospects%26jtitle%3DMed.%2520Sci.%26date%3D2015%26volume%3D31%26spage%3D417%26epage%3D422%26doi%3D10.1051%2Fmedsci%2F20153104016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kadrmas, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ravin, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leontis, N. B.</span></span> <span> </span><span class="NLM_article-title">Relative stabilities of DNA three-way, four-way and five-way junctions (multi-helix junction loops): unpaired nucleotides can be stabilizing or destabilizing</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">2212</span>– <span class="NLM_lpage">2222</span>, <span class="refDoi"> DOI: 10.1093/nar/23.12.2212</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=10.1093%2Fnar%2F23.12.2212" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=1995&pages=2212-2222&author=J.+L.+Kadrmasauthor=A.+J.+Ravinauthor=N.+B.+Leontis&title=Relative+stabilities+of+DNA+three-way%2C+four-way+and+five-way+junctions+%28multi-helix+junction+loops%29%3A+unpaired+nucleotides+can+be+stabilizing+or+destabilizing&doi=10.1093%2Fnar%2F23.12.2212"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1093%2Fnar%2F23.12.2212&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252F23.12.2212%26sid%3Dliteratum%253Aachs%26aulast%3DKadrmas%26aufirst%3DJ.%2BL.%26aulast%3DRavin%26aufirst%3DA.%2BJ.%26aulast%3DLeontis%26aufirst%3DN.%2BB.%26atitle%3DRelative%2520stabilities%2520of%2520DNA%2520three-way%252C%2520four-way%2520and%2520five-way%2520junctions%2520%2528multi-helix%2520junction%2520loops%2529%253A%2520unpaired%2520nucleotides%2520can%2520be%2520stabilizing%2520or%2520destabilizing%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D1995%26volume%3D23%26spage%3D2212%26epage%3D2222%26doi%3D10.1093%2Fnar%2F23.12.2212" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Granzhan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monchaud, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saettel, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guédin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mergny, J.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teulade-Fichou, M.-P.</span></span> <span> </span><span class="NLM_article-title">“One ring to bind them all”—Part II: Identification of promising G-quadruplex ligands by screening of cyclophane-type macrocycles</span>. <i>J. Nucleic Acids</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>2010</i></span>, <span class="NLM_elocation-id">460561</span> <span class="refDoi"> DOI: 10.4061/2010/460561</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=10.4061%2F2010%2F460561" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=20725622" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2010&publication_year=2010&author=A.+Granzhanauthor=D.+Monchaudauthor=N.+Saettelauthor=A.+Gu%C3%A9dinauthor=J.-L.+Mergnyauthor=M.-P.+Teulade-Fichou&title=%E2%80%9COne+ring+to+bind+them+all%E2%80%9D%E2%80%94Part+II%3A+Identification+of+promising+G-quadruplex+ligands+by+screening+of+cyclophane-type+macrocycles&doi=10.4061%2F2010%2F460561"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.4061%2F2010%2F460561&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4061%252F2010%252F460561%26sid%3Dliteratum%253Aachs%26aulast%3DGranzhan%26aufirst%3DA.%26aulast%3DMonchaud%26aufirst%3DD.%26aulast%3DSaettel%26aufirst%3DN.%26aulast%3DGu%25C3%25A9din%26aufirst%3DA.%26aulast%3DMergny%26aufirst%3DJ.-L.%26aulast%3DTeulade-Fichou%26aufirst%3DM.-P.%26atitle%3D%25E2%2580%259COne%2520ring%2520to%2520bind%2520them%2520all%25E2%2580%259D%25E2%2580%2594Part%2520II%253A%2520Identification%2520of%2520promising%2520G-quadruplex%2520ligands%2520by%2520screening%2520of%2520cyclophane-type%2520macrocycles%26jtitle%3DJ.%2520Nucleic%2520Acids%26date%3D2010%26volume%3D2010%26doi%3D10.4061%2F2010%2F460561" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Freudenreich, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barry, N. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Süss-Fink, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Therrien, B.</span></span> <span> </span><span class="NLM_article-title">Permanent encapsulation or host–guest behavior of aromatic molecules in hexanuclear arene ruthenium prisms</span>. <i>Eur. J. Inorg. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>2010</i></span>,  <span class="NLM_fpage">2400</span>– <span class="NLM_lpage">2405</span>, <span class="refDoi"> DOI: 10.1002/ejic.201000191</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=10.1002%2Fejic.201000191" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2010&publication_year=2010&pages=2400-2405&author=J.+Freudenreichauthor=N.+P.+Barryauthor=G.+S%C3%BCss-Finkauthor=B.+Therrien&title=Permanent+encapsulation+or+host%E2%80%93guest+behavior+of+aromatic+molecules+in+hexanuclear+arene+ruthenium+prisms&doi=10.1002%2Fejic.201000191"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1002%2Fejic.201000191&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fejic.201000191%26sid%3Dliteratum%253Aachs%26aulast%3DFreudenreich%26aufirst%3DJ.%26aulast%3DBarry%26aufirst%3DN.%2BP.%26aulast%3DS%25C3%25BCss-Fink%26aufirst%3DG.%26aulast%3DTherrien%26aufirst%3DB.%26atitle%3DPermanent%2520encapsulation%2520or%2520host%25E2%2580%2593guest%2520behavior%2520of%2520aromatic%2520molecules%2520in%2520hexanuclear%2520arene%2520ruthenium%2520prisms%26jtitle%3DEur.%2520J.%2520Inorg.%2520Chem.%26date%3D2010%26volume%3D2010%26spage%3D2400%26epage%3D2405%26doi%3D10.1002%2Fejic.201000191" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gaschard, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nehzat, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheminel, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Therrien, B.</span></span> <span> </span><span class="NLM_article-title">Arene ruthenium metalla-assemblies with anthracene moieties for PDT applications</span>. <i>Inorganics</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">97</span>, <span class="refDoi"> DOI: 10.3390/inorganics6030097</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=10.3390%2Finorganics6030097" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=1%3ACAS%3A528%3ADC%252BC1MXktVWrtb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2018&pages=97&author=M.+Gaschardauthor=F.+Nehzatauthor=T.+Cheminelauthor=B.+Therrien&title=Arene+ruthenium+metalla-assemblies+with+anthracene+moieties+for+PDT+applications&doi=10.3390%2Finorganics6030097"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Arene ruthenium metalla-assemblies with anthracene moieties for PDT applications</span></div><div class="casAuthors">Gaschard, Marie; Nehzat, Farzaneh; Cheminel, Thomas; Therrien, Bruno</div><div class="citationInfo"><span class="NLM_cas:title">Inorganics</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">97/1-97/16</span>CODEN:
                <span class="NLM_cas:coden">INORCW</span>;
        ISSN:<span class="NLM_cas:issn">2304-6740</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">The synthesis and characterization of three metalla-rectangles of the general formula [Ru4(η6-p-cymene)4(μ4-clip)2(μ2-Lanthr)2][CF3SO3]4 (Lanthr: 9,10-bis(3,3'-ethynylpyridyl) anthracene; clip = oxa: oxalato; dobq: 2,5-dioxido-1,4-benzoquinonato; donq: 5,8-dioxido-1, 4-naphthoquinonato) are presented.  The mol. structure of the metalla-rectangle [Ru4(η6-p-cymene)4(μ4-oxa)2(μ2-Lanthr)2]4+ has been confirmed by the single-crystal X-ray structure anal. of [Ru4(η6-p-cymene)4(μ4-oxa)2(μ2-Lanthr)2][CF3SO3]4 · 4 acetone (A2 · 4 acetone), thus showing the anthracene moieties to be available for reaction with oxygen.  While the formation of the endoperoxide form of Lanthr was obsd. in soln. upon white light irradn., the same reaction does not occur when Lanthr is part of the metalla-assemblies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpS9yKZXENK8rVg90H21EOLACvtfcHk0liVWYjgzDJRWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXktVWrtb4%253D&md5=a3fab8ef9adff67a50d571bf802f8a63</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.3390%2Finorganics6030097&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Finorganics6030097%26sid%3Dliteratum%253Aachs%26aulast%3DGaschard%26aufirst%3DM.%26aulast%3DNehzat%26aufirst%3DF.%26aulast%3DCheminel%26aufirst%3DT.%26aulast%3DTherrien%26aufirst%3DB.%26atitle%3DArene%2520ruthenium%2520metalla-assemblies%2520with%2520anthracene%2520moieties%2520for%2520PDT%2520applications%26jtitle%3DInorganics%26date%3D2018%26volume%3D6%26spage%3D97%26doi%3D10.3390%2Finorganics6030097" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stefan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertrand, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richard, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le Gendre, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denat, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picquet, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monchaud, D.</span></span> <span> </span><span class="NLM_article-title">Assessing the differential affinity of small molecules for noncanonical DNA structures</span>. <i>ChemBioChem</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">1905</span>– <span class="NLM_lpage">1912</span>, <span class="refDoi"> DOI: 10.1002/cbic.201200396</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=10.1002%2Fcbic.201200396" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2012&pages=1905-1912&author=L.+Stefanauthor=B.+Bertrandauthor=P.+Richardauthor=P.+Le+Gendreauthor=F.+Denatauthor=M.+Picquetauthor=D.+Monchaud&title=Assessing+the+differential+affinity+of+small+molecules+for+noncanonical+DNA+structures&doi=10.1002%2Fcbic.201200396"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1002%2Fcbic.201200396&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbic.201200396%26sid%3Dliteratum%253Aachs%26aulast%3DStefan%26aufirst%3DL.%26aulast%3DBertrand%26aufirst%3DB.%26aulast%3DRichard%26aufirst%3DP.%26aulast%3DLe%2BGendre%26aufirst%3DP.%26aulast%3DDenat%26aufirst%3DF.%26aulast%3DPicquet%26aufirst%3DM.%26aulast%3DMonchaud%26aufirst%3DD.%26atitle%3DAssessing%2520the%2520differential%2520affinity%2520of%2520small%2520molecules%2520for%2520noncanonical%2520DNA%2520structures%26jtitle%3DChemBioChem%26date%3D2012%26volume%3D13%26spage%3D1905%26epage%3D1912%26doi%3D10.1002%2Fcbic.201200396" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Laguerre, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pirrotta, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Desbois, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gros, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lesniewska, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monchaud, D.</span></span> <span> </span><span class="NLM_article-title">Surface-promoted aggregation of amphiphilic quadruplex ligands drives their selectivity for alternative DNA structures</span>. <i>Org. Biomol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">7034</span>– <span class="NLM_lpage">7039</span>, <span class="refDoi"> DOI: 10.1039/C5OB00692A</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=10.1039%2FC5OB00692A" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2015&pages=7034-7039&author=A.+Laguerreauthor=Y.+Changauthor=M.+Pirrottaauthor=N.+Desboisauthor=C.+P.+Grosauthor=E.+Lesniewskaauthor=D.+Monchaud&title=Surface-promoted+aggregation+of+amphiphilic+quadruplex+ligands+drives+their+selectivity+for+alternative+DNA+structures&doi=10.1039%2FC5OB00692A"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1039%2FC5OB00692A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC5OB00692A%26sid%3Dliteratum%253Aachs%26aulast%3DLaguerre%26aufirst%3DA.%26aulast%3DChang%26aufirst%3DY.%26aulast%3DPirrotta%26aufirst%3DM.%26aulast%3DDesbois%26aufirst%3DN.%26aulast%3DGros%26aufirst%3DC.%2BP.%26aulast%3DLesniewska%26aufirst%3DE.%26aulast%3DMonchaud%26aufirst%3DD.%26atitle%3DSurface-promoted%2520aggregation%2520of%2520amphiphilic%2520quadruplex%2520ligands%2520drives%2520their%2520selectivity%2520for%2520alternative%2520DNA%2520structures%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2015%26volume%3D13%26spage%3D7034%26epage%3D7039%26doi%3D10.1039%2FC5OB00692A" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">de
Robillard, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Makni, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cattey, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrieu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Devillers, C. H.</span></span> <span> </span><span class="NLM_article-title">Towards sustainable synthesis of pyren-1-yl azoliums via electrochemical oxidative C–N coupling</span>. <i>Green. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">4669</span>– <span class="NLM_lpage">4679</span>, <span class="refDoi"> DOI: 10.1039/C5GC01142F</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=10.1039%2FC5GC01142F" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFWjur3K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2015&pages=4669-4679&author=G.+de%0ARobillardauthor=O.+Makniauthor=H.+Catteyauthor=J.+Andrieuauthor=C.+H.+Devillers&title=Towards+sustainable+synthesis+of+pyren-1-yl+azoliums+via+electrochemical+oxidative+C%E2%80%93N+coupling&doi=10.1039%2FC5GC01142F"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Towards sustainable synthesis of pyren-1-yl azoliums via electrochemical oxidative C-N coupling</span></div><div class="casAuthors">de Robillard, Guillaume; Makni, Oumayma; Cattey, Helene; Andrieu, Jacques; Devillers, Charles H.</div><div class="citationInfo"><span class="NLM_cas:title">Green Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4669-4679</span>CODEN:
                <span class="NLM_cas:coden">GRCHFJ</span>;
        ISSN:<span class="NLM_cas:issn">1463-9262</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Electrosynthesis of 1-methyl-3-(pyren-1-yl)-1H-imidazol-3-ium tetrafluoroborate via oxidative C-N coupling of pyrene with methylimidazole is optimized with the aim to reduce waste and simplify the exptl. setup.  Several parameters are tested such as cell configuration (no. of compartments), pyrene concn., amt. of nucleophile, electrosynthetic method (potentiostatic/galvanostatic), amt. of electrons, atm. (Ar/air), solvent quality and presence/absence of a supporting electrolyte.  The optimized conditions are successfully applied to the synthesis of 1-methyl-3-(pyren-1-yl)-1H-benzimidazol-3-ium tetrafluoroborate, 1-methyl-4-(pyren-1-yl)-1H-1,2,4-triazol-4-ium tetrafluoroborate and 3-(pyren-1-yl)-benzothiazol-3-ium tetrafluoroborate.  These four pyren-1-yl-azolium salts are characterized by NMR, MS, elemental anal., UV-Vis absorption and emission spectroscopy.  The X-ray crystallog. structures of 1-methyl-3-(pyren-1-yl)-1H-imidazol-3-ium tetrafluoroborate and 1-methyl-3-(pyren-1-yl)-1H-benzimidazol-3-ium tetrafluoroborate are presented.  The synthesis of the target compds. was achieved by a reaction of pyrene (i.e., polycyclic arom. hydrocarbon, arom. hydrocarbon) with 1-methyl-1H-imidazole, 1-methyl-1H-benzimidazole, 1-methyl-1H-1,2,4-triazole, benzothiazole in the presence of hydrogen tetrafluoroborate(1-) compd. with 1,1'-oxybis[ethane] (i.e., hydrogen tetrafluoroborate di-Et ether complex).  The title compds. thus formed included (pyrenyl)imidazolium salt, (pyrenyl)triazolium salt, (pyrenyl)benzothiazolium tetrafluoroborate, (pyrenyl)benzimidazolium tetrafluoroborate.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHDOL3_teuDLVg90H21EOLACvtfcHk0ljkosKa6HsT2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFWjur3K&md5=3c0774db029418879f7570370b458705</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1039%2FC5GC01142F&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC5GC01142F%26sid%3Dliteratum%253Aachs%26aulast%3Dde%2BRobillard%26aufirst%3DG.%26aulast%3DMakni%26aufirst%3DO.%26aulast%3DCattey%26aufirst%3DH.%26aulast%3DAndrieu%26aufirst%3DJ.%26aulast%3DDevillers%26aufirst%3DC.%2BH.%26atitle%3DTowards%2520sustainable%2520synthesis%2520of%2520pyren-1-yl%2520azoliums%2520via%2520electrochemical%2520oxidative%2520C%25E2%2580%2593N%2520coupling%26jtitle%3DGreen.%2520Chem.%26date%3D2015%26volume%3D17%26spage%3D4669%26epage%3D4679%26doi%3D10.1039%2FC5GC01142F" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gabelica, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Pauw, E.</span></span> <span> </span><span class="NLM_article-title">A simple method to determine electrospray response factors of noncovalent complexes</span>. <i>Anal. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>81</i></span>,  <span class="NLM_fpage">6708</span>– <span class="NLM_lpage">6715</span>, <span class="refDoi"> DOI: 10.1021/ac900785m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ac900785m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=1%3ACAS%3A528%3ADC%252BD1MXosVWntLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=81&publication_year=2009&pages=6708-6715&author=V.+Gabelicaauthor=F.+Rosuauthor=E.+De+Pauw&title=A+simple+method+to+determine+electrospray+response+factors+of+noncovalent+complexes&doi=10.1021%2Fac900785m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">A Simple Method to Determine Electrospray Response Factors of Noncovalent Complexes</span></div><div class="casAuthors">Gabelica, Valerie; Rosu, Frederic; De Pauw, Edwin</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Chemistry (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">81</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">6708-6715</span>CODEN:
                <span class="NLM_cas:coden">ANCHAM</span>;
        ISSN:<span class="NLM_cas:issn">0003-2700</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The quant. study of noncovalent complexes by electrospray mass spectrometry requires the detn. of the relative response of each species.  The method proposed here to det. the electrospray response factors is based on the use of (1) an internal std. and (2) the mass balance equation applied to one binding partner M, for which different complexes MxLy are detected in the electrospray mass spectra.  A set of expts. providing various ratios between the complexes (e.g., different ligand concns. in a titrn. expt. or different time points in a kinetics expt.) is used to generate a set of independent linear equations that can be solved using simple matrix algebra to find the response factors of each MxLy complex relative to that of the internal std.  The response factors can then be used to det. equil. dissocn. consts. or for the quant. monitoring of reaction kinetics.  The first is illustrated with a study of DNA-ligand complexes, where the authors show that neither minor groove binding nor intercalation dramatically affects the DNA response factor.  The second is illustrated with a study of the assocn. kinetics of the telomeric G-quadruplex dGGG(TTAGGG)3 with its complementary strand, where the response factors allow correcting for the relative response of the quadruplex and the long duplex and obtaining reproducible assocn. rate consts. independently of the source tuning potentials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrS4BvCgVhd2LVg90H21EOLACvtfcHk0ljkosKa6HsT2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXosVWntLw%253D&md5=e66feaaa16b4086832ec960c3e8f7c8a</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fac900785m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fac900785m%26sid%3Dliteratum%253Aachs%26aulast%3DGabelica%26aufirst%3DV.%26aulast%3DRosu%26aufirst%3DF.%26aulast%3DDe%2BPauw%26aufirst%3DE.%26atitle%3DA%2520simple%2520method%2520to%2520determine%2520electrospray%2520response%2520factors%2520of%2520noncovalent%2520complexes%26jtitle%3DAnal.%2520Chem.%26date%3D2009%26volume%3D81%26spage%3D6708%26epage%3D6715%26doi%3D10.1021%2Fac900785m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rosu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gabelica, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Houssier, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Pauw, E.</span></span> <span> </span><span class="NLM_article-title">Determination of affinity, stoichiometry and sequence selectivity of minor groove binder complexes with double-stranded oligodeoxynucleotides by electrospray ionization mass spectrometry</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>30</i></span>, <span class="NLM_elocation-id">e82</span> <span class="refDoi"> DOI: 10.1093/nar/gnf081</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=10.1093%2Fnar%2Fgnf081" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2002&author=F.+Rosuauthor=V.+Gabelicaauthor=C.+Houssierauthor=E.+De+Pauw&title=Determination+of+affinity%2C+stoichiometry+and+sequence+selectivity+of+minor+groove+binder+complexes+with+double-stranded+oligodeoxynucleotides+by+electrospray+ionization+mass+spectrometry&doi=10.1093%2Fnar%2Fgnf081"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgnf081&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgnf081%26sid%3Dliteratum%253Aachs%26aulast%3DRosu%26aufirst%3DF.%26aulast%3DGabelica%26aufirst%3DV.%26aulast%3DHoussier%26aufirst%3DC.%26aulast%3DDe%2BPauw%26aufirst%3DE.%26atitle%3DDetermination%2520of%2520affinity%252C%2520stoichiometry%2520and%2520sequence%2520selectivity%2520of%2520minor%2520groove%2520binder%2520complexes%2520with%2520double-stranded%2520oligodeoxynucleotides%2520by%2520electrospray%2520ionization%2520mass%2520spectrometry%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2002%26volume%3D30%26doi%3D10.1093%2Fnar%2Fgnf081" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ragazzon, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garbett, N. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaires, J. B.</span></span> <span> </span><span class="NLM_article-title">Competition dialysis: a method for the study of structural selective nucleic acid binding</span>. <i>Methods</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">173</span>– <span class="NLM_lpage">182</span>, <span class="refDoi"> DOI: 10.1016/j.ymeth.2006.09.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=10.1016%2Fj.ymeth.2006.09.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=17472899" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=1%3ACAS%3A528%3ADC%252BD2sXkslCntrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2007&pages=173-182&author=P.+A.+Ragazzonauthor=N.+C.+Garbettauthor=J.+B.+Chaires&title=Competition+dialysis%3A+a+method+for+the+study+of+structural+selective+nucleic+acid+binding&doi=10.1016%2Fj.ymeth.2006.09.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Competition dialysis: a method for the study of structural selective nucleic acid binding</span></div><div class="casAuthors">Ragazzon, Patricia A.; Garbett, Nichola C.; Chaires, Jonathan B.</div><div class="citationInfo"><span class="NLM_cas:title">Methods (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">173-182</span>CODEN:
                <span class="NLM_cas:coden">MTHDE9</span>;
        ISSN:<span class="NLM_cas:issn">1046-2023</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Competition dialysis is a powerful new tool for the discovery of ligands that bind to nucleic acids with structural- or sequence-selectivity.  The method is based on firm thermodn. principles and is simple to implement.  In the competition dialysis expt., an array of nucleic acid structures and sequences is dialyzed against a common test ligand soln.  After equilibration, the amt. of ligand bound to each structure or sequence is detd. by absorbance or fluorescence measurements.  Since all structures and sequences are in equil. with the same free ligand concn., the amt. bound is directly proportional to the ligand binding affinity.  Competition dialysis thus provides a direct and quant. measure of selectivity, and unambiguously identifies which of the samples within the array are preferred by a particular ligand.  We describe here the third generation implementation of the method, in which competition dialysis was adapted for use in a 96-well plate format.  In this format, we have been able to greatly expand the array of nucleic acid structures studied, and now can routinely study the interactions of a ligand of interest with 46 different structures and sequences.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrrGii5sg50V7Vg90H21EOLACvtfcHk0ljkosKa6HsT2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXkslCntrk%253D&md5=4154e0e912fb778ca37ae85ceada94d9</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1016%2Fj.ymeth.2006.09.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ymeth.2006.09.010%26sid%3Dliteratum%253Aachs%26aulast%3DRagazzon%26aufirst%3DP.%2BA.%26aulast%3DGarbett%26aufirst%3DN.%2BC.%26aulast%3DChaires%26aufirst%3DJ.%2BB.%26atitle%3DCompetition%2520dialysis%253A%2520a%2520method%2520for%2520the%2520study%2520of%2520structural%2520selective%2520nucleic%2520acid%2520binding%26jtitle%3DMethods%26date%3D2007%26volume%3D42%26spage%3D173%26epage%3D182%26doi%3D10.1016%2Fj.ymeth.2006.09.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vichai, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirtikara, K.</span></span> <span> </span><span class="NLM_article-title">Sulforhodamine B colorimetric assay for cytotoxicity screening</span>. <i>Nat. Protoc.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">1112</span>– <span class="NLM_lpage">1116</span>, <span class="refDoi"> DOI: 10.1038/nprot.2006.179</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=10.1038%2Fnprot.2006.179" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=17406391" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtFOitL3O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2006&pages=1112-1116&author=V.+Vichaiauthor=K.+Kirtikara&title=Sulforhodamine+B+colorimetric+assay+for+cytotoxicity+screening&doi=10.1038%2Fnprot.2006.179"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Sulforhodamine B colorimetric assay for cytotoxicity screening</span></div><div class="casAuthors">Vichai, Vanicha; Kirtikara, Kanyawim</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1112-1116</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The sulforhodamine B (SRB) assay is used for cell d. detn., based on the measurement of cellular protein content.  The method described here was optimized for the toxicity screening of compds. to adherent cells in a 96-well format.  After an incubation period, cell monolayers are fixed with 10% (wt/vol) trichloroacetic acid and stained for 30 min, after which the excess dye is removed by washing repeatedly with 1% (vol/vol) acetic acid.  The protein-bound dye is dissolved in 10 mM Tris base soln. for OD detn. at 510 nm using a microplate reader.  The results are linear over a 20-fold range of cell nos. and the sensitivity is comparable to those of fluorometric methods.  The method not only allows a large no. of samples to be tested within a few days, but also requires only simple equipment and inexpensive reagents.  The SRB assay is therefore an efficient and highly cost-effective method for screening.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGZ8pao808l7Vg90H21EOLACvtfcHk0lggRuXbSN-yzw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtFOitL3O&md5=7173035a1f9eb3f02dd7f1a947bca659</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2006.179&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2006.179%26sid%3Dliteratum%253Aachs%26aulast%3DVichai%26aufirst%3DV.%26aulast%3DKirtikara%26aufirst%3DK.%26atitle%3DSulforhodamine%2520B%2520colorimetric%2520assay%2520for%2520cytotoxicity%2520screening%26jtitle%3DNat.%2520Protoc.%26date%3D2006%26volume%3D1%26spage%3D1112%26epage%3D1116%26doi%3D10.1038%2Fnprot.2006.179" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Skehan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Storeng, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scudiero, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monks, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McMahon, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vistica, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warren, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bokesch, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kenney, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boyd, M. R.</span></span> <span> </span><span class="NLM_article-title">New colorimetric cytotoxicity assay for anticancer-drug screening</span>. <i>JNCI, J. Natl. Cancer Inst.</i> <span class="NLM_year" style="font-weight: bold;">1990</span>,  <span class="NLM_volume"><i>82</i></span>,  <span class="NLM_fpage">1107</span>– <span class="NLM_lpage">1112</span>, <span class="refDoi"> DOI: 10.1093/jnci/82.13.1107</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=10.1093%2Fjnci%2F82.13.1107" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=2359136" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=1%3ACAS%3A528%3ADyaK3cXltVylsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=82&publication_year=1990&pages=1107-1112&author=P.+Skehanauthor=R.+Storengauthor=D.+Scudieroauthor=A.+Monksauthor=J.+McMahonauthor=D.+Visticaauthor=J.+T.+Warrenauthor=H.+Bokeschauthor=S.+Kenneyauthor=M.+R.+Boyd&title=New+colorimetric+cytotoxicity+assay+for+anticancer-drug+screening&doi=10.1093%2Fjnci%2F82.13.1107"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">New colorimetric cytotoxicity assay for anticancer-drug screening</span></div><div class="casAuthors">Skehan, Philip; Storeng, Ritsa; Scudiero, Dominic; Monks, Anne; McMahon, James; Vistica, David; Warren, Jonathan T.; Bokesch, Heidi; Kenney, Susan; Boyd, Michael R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the National Cancer Institute</span>
        (<span class="NLM_cas:date">1990</span>),
    <span class="NLM_cas:volume">82</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">1107-12</span>CODEN:
                <span class="NLM_cas:coden">JNCIEQ</span>;
        ISSN:<span class="NLM_cas:issn">0027-8874</span>.
    </div><div class="casAbstract">A rapid, sensitive, and inexpensive method was developed for measuring the cellular protein content of adherent and suspension cultures in 96-well microtiter plates.  The method is suitable for ordinary lab. purposes and for very large-scale applications, such as the National Cancer Institute's disease-oriented in vitro anticancer-drug discovery screen, which requires the use of several million culture wells per yr.  Cultures fixed with trichloroacetic acid were stained for 30 min with 0.4% (wt/vol) sulforhodamine B (SRB) dissolved in 1% acetic acid.  Unbound dye was removed by four washes with 1% acetic acid, and protein-bound dye was extd. with 10 mM unbuffered Tris base [tris(hydroxymethyl)aminomethane] for detn. of optical d. in a computer-interfaced, 96-well microtiter plate reader.  The SRB assay results were linear with the no. of cells and with values for cellular protein measured by both the Lowry and Bradford assays at densities ranging from sparse subconfluence to multilayered supraconfluence.  The signal-to-noise ratio at 564 nm was approx. 1.5 with 1,000 cells per well.  The sensitivity of the SRB assay compared favorably with sensitivities of several fluorescence assays and was superior to those of both the Lowry and Bradford assays and to those of 20 other visible dyes.  The SRB assay provides a colorimetric end point that is nondestructive, indefinitely stable, and visible to the naked eye.  It provides a sensitive measure of drug-induced cytotoxicity, is useful in quantitating clonogenicity, and is well suited to high-vol., automated drug screening.  SRB fluoresces strongly with laser excitation at 488 nm and can be measured quant. at the single-cell level by static fluorescence cytometry.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrA1MIZ3rkcqLVg90H21EOLACvtfcHk0lggRuXbSN-yzw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3cXltVylsL8%253D&md5=d7bc1784a1f181842d2999efa7fa870c</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1093%2Fjnci%2F82.13.1107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjnci%252F82.13.1107%26sid%3Dliteratum%253Aachs%26aulast%3DSkehan%26aufirst%3DP.%26aulast%3DStoreng%26aufirst%3DR.%26aulast%3DScudiero%26aufirst%3DD.%26aulast%3DMonks%26aufirst%3DA.%26aulast%3DMcMahon%26aufirst%3DJ.%26aulast%3DVistica%26aufirst%3DD.%26aulast%3DWarren%26aufirst%3DJ.%2BT.%26aulast%3DBokesch%26aufirst%3DH.%26aulast%3DKenney%26aufirst%3DS.%26aulast%3DBoyd%26aufirst%3DM.%2BR.%26atitle%3DNew%2520colorimetric%2520cytotoxicity%2520assay%2520for%2520anticancer-drug%2520screening%26jtitle%3DJNCI%252C%2520J.%2520Natl.%2520Cancer%2520Inst.%26date%3D1990%26volume%3D82%26spage%3D1107%26epage%3D1112%26doi%3D10.1093%2Fjnci%2F82.13.1107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pommier, Y.</span></span> <span> </span><span class="NLM_article-title">Topoisomerase I inhibitors: camptothecins and beyond</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">789</span>– <span class="NLM_lpage">802</span>, <span class="refDoi"> DOI: 10.1038/nrc1977</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=10.1038%2Fnrc1977" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=16990856" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=1%3ACAS%3A528%3ADC%252BD28XpvVClsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2006&pages=789-802&author=Y.+Pommier&title=Topoisomerase+I+inhibitors%3A+camptothecins+and+beyond&doi=10.1038%2Fnrc1977"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Topoisomerase I inhibitors: camptothecins and beyond</span></div><div class="casAuthors">Pommier, Yves</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">789-802</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Nuclear DNA topoisomerase I (TOP1) is an essential human enzyme.  It is the only known target of the alkaloid camptothecin, from which the potent anticancer agents irinotecan and topotecan are derived.  As camptothecins bind at the interface of the TOP1-DNA complex, they represent a paradigm for interfacial inhibitors that reversibly trap macromol. complexes.  Several camptothecin and noncamptothecin derivs. are being developed to further increase antitumor activity and reduce side effects.  The mechanisms and mol. determinants of tumor response to TOP1 inhibitors are reviewed, and rational combinations of TOP1 inhibitors with other drugs are considered based on current knowledge of repair and checkpoint pathways that are assocd. with TOP1-mediated DNA damage.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrl5lxd3qfkwbVg90H21EOLACvtfcHk0lggRuXbSN-yzw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XpvVClsr0%253D&md5=a437970077c0edbbab602b296fcd092a</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1038%2Fnrc1977&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc1977%26sid%3Dliteratum%253Aachs%26aulast%3DPommier%26aufirst%3DY.%26atitle%3DTopoisomerase%2520I%2520inhibitors%253A%2520camptothecins%2520and%2520beyond%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2006%26volume%3D6%26spage%3D789%26epage%3D802%26doi%3D10.1038%2Fnrc1977" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hahn, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Counter, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lundberg, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beijersbergen, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brooks, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinberg, R. A.</span></span> <span> </span><span class="NLM_article-title">Creation of human tumour cells with defined genetic elements</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>400</i></span>,  <span class="NLM_fpage">464</span>– <span class="NLM_lpage">468</span>, <span class="refDoi"> DOI: 10.1038/22780</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=10.1038%2F22780" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=10440377" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=1%3ACAS%3A528%3ADyaK1MXltVGmtLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=400&publication_year=1999&pages=464-468&author=W.+C.+Hahnauthor=C.+M.+Counterauthor=A.+S.+Lundbergauthor=R.+L.+Beijersbergenauthor=M.+W.+Brooksauthor=R.+A.+Weinberg&title=Creation+of+human+tumour+cells+with+defined+genetic+elements&doi=10.1038%2F22780"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Creation of human tumor cells with defined genetic elements</span></div><div class="casAuthors">Hahn, William C.; Counter, Christopher M.; Lundberg, Ante S.; Beijersbergen, Roderick L.; Brooks, Mary W.; Weinberg, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London)</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">400</span>
        (<span class="NLM_cas:issue">6743</span>),
    <span class="NLM_cas:pages">464-468</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Macmillan Magazines</span>)
        </div><div class="casAbstract">During malignant transformation, cancer cells acquire genetic mutations that override the normal mechanisms controlling cellular proliferation.  Primary rodent cells are efficiently converted into tumorigenic cells by the coexpression of cooperating oncogenes.  However, similar expts. with human cells have consistently failed to yield tumorigenic transformants, indicating a fundamental difference in the biol. of human and rodent cells.  The few reported successes in the creation of human tumor cells have depended on the use of chem. or phys. agents to achieve immortalization, the selection of rare, spontaneously arising immortalized cells, or the use of an entire viral genome.  The authors show here that the ectopic expression of the telomerase catalytic subunit (hTERT) in combination with two oncogenes (the simian virus 40 large-T oncoprotein and an oncogenic allele of H-ras) results in direct tumorigenic conversion of normal human epithelial and fibroblast cells.  These results demonstrate that disruption of the intracellular pathways regulated by large-T, oncogenic ras and telomerase suffices to create a human tumor cell.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrIWwnNvgmviLVg90H21EOLACvtfcHk0ljW0sNxRBqMIQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXltVGmtLc%253D&md5=09363db22f5dfe8fc4c52c3864491069</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1038%2F22780&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F22780%26sid%3Dliteratum%253Aachs%26aulast%3DHahn%26aufirst%3DW.%2BC.%26aulast%3DCounter%26aufirst%3DC.%2BM.%26aulast%3DLundberg%26aufirst%3DA.%2BS.%26aulast%3DBeijersbergen%26aufirst%3DR.%2BL.%26aulast%3DBrooks%26aufirst%3DM.%2BW.%26aulast%3DWeinberg%26aufirst%3DR.%2BA.%26atitle%3DCreation%2520of%2520human%2520tumour%2520cells%2520with%2520defined%2520genetic%2520elements%26jtitle%3DNature%26date%3D1999%26volume%3D400%26spage%3D464%26epage%3D468%26doi%3D10.1038%2F22780" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bonner, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Redon, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dickey, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakamura, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sedelnikova, O. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Solier, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pommier, Y.</span></span> <span> </span><span class="NLM_article-title">γH2AX and cancer</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">957</span>– <span class="NLM_lpage">967</span>, <span class="refDoi"> DOI: 10.1038/nrc2523</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=10.1038%2Fnrc2523" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=19005492" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsVWhu7rK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2008&pages=957-967&author=W.+M.+Bonnerauthor=C.+E.+Redonauthor=J.+S.+Dickeyauthor=A.+J.+Nakamuraauthor=O.+A.+Sedelnikovaauthor=S.+Solierauthor=Y.+Pommier&title=%CE%B3H2AX+and+cancer&doi=10.1038%2Fnrc2523"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">γH2AX and cancer</span></div><div class="casAuthors">Bonner, William M.; Redon, Christophe E.; Dickey, Jennifer S.; Nakamura, Asako J.; Sedelnikova, Olga A.; Solier, Stephanie; Pommier, Yves</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">957-967</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Histone H2AX phosphorylation on a serine four residues from the carboxyl terminus (producing γH2AX) is a sensitive marker for DNA double-strand breaks (DSBs).  DSBs may lead to cancer but, paradoxically, are also used to kill cancer cells.  Using γH2AX detection to det. the extent of DSB induction may help to detect precancerous cells, to stage cancers, to monitor the effectiveness of cancer therapies and to develop novel anticancer drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDvJ8vnkfC0bVg90H21EOLACvtfcHk0ljW0sNxRBqMIQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsVWhu7rK&md5=b0a83efe87f2823e79146ba8a2edf808</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1038%2Fnrc2523&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc2523%26sid%3Dliteratum%253Aachs%26aulast%3DBonner%26aufirst%3DW.%2BM.%26aulast%3DRedon%26aufirst%3DC.%2BE.%26aulast%3DDickey%26aufirst%3DJ.%2BS.%26aulast%3DNakamura%26aufirst%3DA.%2BJ.%26aulast%3DSedelnikova%26aufirst%3DO.%2BA.%26aulast%3DSolier%26aufirst%3DS.%26aulast%3DPommier%26aufirst%3DY.%26atitle%3D%25CE%25B3H2AX%2520and%2520cancer%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2008%26volume%3D8%26spage%3D957%26epage%3D967%26doi%3D10.1038%2Fnrc2523" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">O’Connor, M. J.</span></span> <span> </span><span class="NLM_article-title">Targeting the DNA damage response in cancer</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">547</span>– <span class="NLM_lpage">560</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2015.10.040</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=10.1016%2Fj.molcel.2015.10.040" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=26590714" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVyqtbvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2015&pages=547-560&author=M.+J.+O%E2%80%99Connor&title=Targeting+the+DNA+damage+response+in+cancer&doi=10.1016%2Fj.molcel.2015.10.040"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting the DNA Damage Response in Cancer</span></div><div class="casAuthors">O'Connor, Mark J.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">547-560</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">An underlying hallmark of cancers is their genomic instability, which is assocd. with a greater propensity to accumulate DNA damage.  Historical treatment of cancer by radiotherapy and DNA-damaging chemotherapy is based on this principle, yet it is accompanied by significant collateral damage to normal tissue and unwanted side effects.  Targeted therapy based on inhibiting the DNA damage response (DDR) in cancers offers the potential for a greater therapeutic window by tailoring treatment to patients with tumors lacking specific DDR functions.  The recent approval of olaparib (Lynparza), the poly (ADP-ribose) polymerase (PARP) inhibitor for treating tumors harboring BRCA1 or BRCA2 mutations, represents the first medicine based on this principle, exploiting an underlying cause of tumor formation that also represents an Achilles' heel.  This review highlights the different concepts behind targeting DDR in cancer and how this can provide significant opportunities for DDR-based therapies in the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWZ64MrshabrVg90H21EOLACvtfcHk0ljW0sNxRBqMIQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVyqtbvI&md5=b3f0d737d3e7dbf1f404b3f7658ab271</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2015.10.040&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2015.10.040%26sid%3Dliteratum%253Aachs%26aulast%3DO%25E2%2580%2599Connor%26aufirst%3DM.%2BJ.%26atitle%3DTargeting%2520the%2520DNA%2520damage%2520response%2520in%2520cancer%26jtitle%3DMol.%2520Cell%26date%3D2015%26volume%3D60%26spage%3D547%26epage%3D560%26doi%3D10.1016%2Fj.molcel.2015.10.040" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kotera, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poyer, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Granzhan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teulade-Fichou, M.-P.</span></span> <span> </span><span class="NLM_article-title">Efficient inhibition of human AP endonuclease 1 (APE1) via substrate masking by abasic site-binding macrocyclic ligands</span>. <i>Chem. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">15948</span>– <span class="NLM_lpage">15951</span>, <span class="refDoi"> DOI: 10.1039/C5CC06084B</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=10.1039%2FC5CC06084B" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2015&pages=15948-15951&author=N.+Koteraauthor=F.+Poyerauthor=A.+Granzhanauthor=M.-P.+Teulade-Fichou&title=Efficient+inhibition+of+human+AP+endonuclease+1+%28APE1%29+via+substrate+masking+by+abasic+site-binding+macrocyclic+ligands&doi=10.1039%2FC5CC06084B"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1039%2FC5CC06084B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC5CC06084B%26sid%3Dliteratum%253Aachs%26aulast%3DKotera%26aufirst%3DN.%26aulast%3DPoyer%26aufirst%3DF.%26aulast%3DGranzhan%26aufirst%3DA.%26aulast%3DTeulade-Fichou%26aufirst%3DM.-P.%26atitle%3DEfficient%2520inhibition%2520of%2520human%2520AP%2520endonuclease%25201%2520%2528APE1%2529%2520via%2520substrate%2520masking%2520by%2520abasic%2520site-binding%2520macrocyclic%2520ligands%26jtitle%3DChem.%2520Commun.%26date%3D2015%26volume%3D51%26spage%3D15948%26epage%3D15951%26doi%3D10.1039%2FC5CC06084B" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Caron, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duong, X. N. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guillot, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bombard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Granzhan, A.</span></span> <span> </span><span class="NLM_article-title">Interaction of functionalized naphthalenophanes with abasic sites in DNA: DNA cleavage, DNA cleavage inhibition, and formation of ligand–DNA adducts</span>. <i>Chem. Eur. J.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1949</span>– <span class="NLM_lpage">1962</span>, <span class="refDoi"> DOI: 10.1002/chem.201805555</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=10.1002%2Fchem.201805555" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=30508326" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsVKkurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2019&pages=1949-1962&author=C.+Caronauthor=X.+N.+T.+Duongauthor=R.+Guillotauthor=S.+Bombardauthor=A.+Granzhan&title=Interaction+of+functionalized+naphthalenophanes+with+abasic+sites+in+DNA%3A+DNA+cleavage%2C+DNA+cleavage+inhibition%2C+and+formation+of+ligand%E2%80%93DNA+adducts&doi=10.1002%2Fchem.201805555"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Interaction of Functionalized Naphthalenophanes with Abasic Sites in DNA: DNA Cleavage, DNA Cleavage Inhibition, and Formation of Ligand-DNA Adducts</span></div><div class="casAuthors">Caron, Coralie; Duong, Xuan N. T.; Guillot, Regis; Bombard, Sophie; Granzhan, Anton</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry - A European Journal</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1949-1962</span>CODEN:
                <span class="NLM_cas:coden">CEUJED</span>;
        ISSN:<span class="NLM_cas:issn">0947-6539</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Ligands interacting with abasic (AP) sites in DNA may generate roadblocks in base-excision DNA repair (BER) due to indirect inhibition of DNA repair enzymes (e.g., APE1) and/or formation of toxic byproducts, resulting from ligand-induced strand cleavage or covalent cross-links.  Herein, a series of 12 putative AP-site ligands, sharing the common naphthalenophane scaffold, but endowed with a variety of substituents, have been prepd. and systematically studied.  The results demonstrate that most naphthalenophanes bind to AP sites in DNA and inhibit the APE1-induced hydrolysis of the latter in vitro.  Remarkably, their APE1 inhibitory activity, as characterized by IC50 and KI values, can be directly related to their affinity and selectivity to AP sites, as assessed by means of fluorescence melting expts.  On the other hand, the mol. design of naphthalenophanes has a crucial influence on their intrinsic AP-site cleavage activity (i.e., ligand-catalyzed β- and β,δ-elimination reactions at the AP site), as illustrated by the compds. either having an exceptionally high AP-site cleavage activity (e.g., 2,7-BisNP-S, 125-fold more efficacious than spermine) or being totally devoid of this activity (four compds.).  Finally, the unprecedented formation of a stable covalent DNA adduct upon reaction of one ligand (2,7-BisNP-NH) with its own product of the AP-site cleavage is revealed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqsl52szWsAmLVg90H21EOLACvtfcHk0ljRHj552Qgqlw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsVKkurw%253D&md5=9d8b88350a376d92f8eac95e54ce598e</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1002%2Fchem.201805555&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fchem.201805555%26sid%3Dliteratum%253Aachs%26aulast%3DCaron%26aufirst%3DC.%26aulast%3DDuong%26aufirst%3DX.%2BN.%2BT.%26aulast%3DGuillot%26aufirst%3DR.%26aulast%3DBombard%26aufirst%3DS.%26aulast%3DGranzhan%26aufirst%3DA.%26atitle%3DInteraction%2520of%2520functionalized%2520naphthalenophanes%2520with%2520abasic%2520sites%2520in%2520DNA%253A%2520DNA%2520cleavage%252C%2520DNA%2520cleavage%2520inhibition%252C%2520and%2520formation%2520of%2520ligand%25E2%2580%2593DNA%2520adducts%26jtitle%3DChem.%2520Eur.%2520J.%26date%3D2019%26volume%3D25%26spage%3D1949%26epage%3D1962%26doi%3D10.1002%2Fchem.201805555" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 8 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Catherine A. J. Hooper, Lucia Cardo, James S. Craig, Lazaros Melidis, Aditya Garai, Ross T. Egan, Viktoriia Sadovnikova, Florian Burkert, Louise Male, Nikolas J. Hodges, Douglas F. Browning, Roselyne Rosas, Fengbo Liu, Fillipe V. Rocha, Mauro A. Lima, Simin Liu, David Bardelang, <span class="NLM_string-name hlFld-ContribAuthor">Michael J. Hannon</span>. </span><span class="cited-content_cbyCitation_article-title">Rotaxanating Metallo-supramolecular Nano-cylinder Helicates to Switch DNA Junction Binding. </span><span class="cited-content_cbyCitation_journal-name">Journal of the American Chemical Society</span><span> <strong>2020,</strong> <em>142 </em>
                                    (49)
                                     , 20651-20660. <a href="https://doi.org/10.1021/jacs.0c07750" title="DOI URL">https://doi.org/10.1021/jacs.0c07750</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jacs.0c07750&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjacs.0c07750%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520the%2520American%2520Chemical%2520Society%26atitle%3DRotaxanating%252BMetallo-supramolecular%252BNano-cylinder%252BHelicates%252Bto%252BSwitch%252BDNA%252BJunction%252BBinding%26aulast%3DHooper%26aufirst%3DCatherine%2BA.%2BJ.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D18072020%26date%3D20112020%26volume%3D142%26issue%3D49%26spage%3D20651%26epage%3D20660" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Katerina Duskova, Pauline Lejault, Élie Benchimol, Régis Guillot, Sébastien Britton, Anton Granzhan, <span class="NLM_string-name hlFld-ContribAuthor">David Monchaud</span>. </span><span class="cited-content_cbyCitation_article-title">DNA Junction Ligands Trigger DNA Damage and Are Synthetic Lethal with DNA Repair Inhibitors in Cancer Cells. </span><span class="cited-content_cbyCitation_journal-name">Journal of the American Chemical Society</span><span> <strong>2020,</strong> <em>142 </em>
                                    (1)
                                     , 424-435. <a href="https://doi.org/10.1021/jacs.9b11150" title="DOI URL">https://doi.org/10.1021/jacs.9b11150</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jacs.9b11150&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjacs.9b11150%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520the%2520American%2520Chemical%2520Society%26atitle%3DDNA%252BJunction%252BLigands%252BTrigger%252BDNA%252BDamage%252Band%252BAre%252BSynthetic%252BLethal%252Bwith%252BDNA%252BRepair%252BInhibitors%252Bin%252BCancer%252BCells%26aulast%3DDuskova%26aufirst%3DKaterina%26date%3D2020%26date%3D2019%26date%3D2019%26date%3D16102019%26date%3D20122019%26date%3D13122019%26volume%3D142%26issue%3D1%26spage%3D424%26epage%3D435" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Danielle  Westhuizen</span>, <span class="hlFld-ContribAuthor ">Cathryn A.  Slabber</span>, <span class="hlFld-ContribAuthor ">Manuel A.  Fernandes</span>, <span class="hlFld-ContribAuthor ">Daniël F.  Joubert</span>, <span class="hlFld-ContribAuthor ">George  Kleinhans</span>, <span class="hlFld-ContribAuthor ">C. Johan  Westhuizen</span>, <span class="hlFld-ContribAuthor ">André  Stander</span>, <span class="hlFld-ContribAuthor ">Orde Q.  Munro</span>, <span class="hlFld-ContribAuthor ">Daniela I.  Bezuidenhout</span>. </span><span class="cited-content_cbyCitation_article-title">A Cytotoxic Bis(1,2,3‐triazol‐5‐ylidene)carbazolide Gold(III) Complex Targets DNA by Partial Intercalation. </span><span class="cited-content_cbyCitation_journal-name">Chemistry – A European Journal</span><span> <strong>2021,</strong> <em>27 </em>
                                    (32)
                                     , 8295-8307. <a href="https://doi.org/10.1002/chem.202100598" title="DOI URL">https://doi.org/10.1002/chem.202100598</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/chem.202100598&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fchem.202100598%26sid%3Dliteratum%253Aachs%26jtitle%3DChemistry%2520%25E2%2580%2593%2520A%2520European%2520Journal%26atitle%3DA%252BCytotoxic%252BBis%2525281%25252C2%25252C3%2525E2%252580%252590triazol%2525E2%252580%2525905%2525E2%252580%252590ylidene%252529carbazolide%252BGold%252528III%252529%252BComplex%252BTargets%252BDNA%252Bby%252BPartial%252BIntercalation%26aulast%3DWesthuizen%26aufirst%3DDanielle%26date%3D2021%26date%3D2021%26volume%3D27%26issue%3D32%26spage%3D8295%26epage%3D8307" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Bruno  Therrien</span>. </span><span class="cited-content_cbyCitation_article-title">Unmasking Arene Ruthenium Building Blocks. </span><span class="cited-content_cbyCitation_journal-name">The Chemical Record</span><span> <strong>2021,</strong> <em>21 </em>
                                    (3)
                                     , 460-468. <a href="https://doi.org/10.1002/tcr.202000128" title="DOI URL">https://doi.org/10.1002/tcr.202000128</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/tcr.202000128&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Ftcr.202000128%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Chemical%2520Record%26atitle%3DUnmasking%252BArene%252BRuthenium%252BBuilding%252BBlocks%26aulast%3DTherrien%26aufirst%3DBruno%26date%3D2021%26date%3D2020%26volume%3D21%26issue%3D3%26spage%3D460%26epage%3D468" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Pavel  Štarha</span>. </span><span class="cited-content_cbyCitation_article-title">Multinuclear biologically active Ru, Rh, Os and Ir arene complexes. </span><span class="cited-content_cbyCitation_journal-name">Coordination Chemistry Reviews</span><span> <strong>2021,</strong> <em>431 </em>, 213690. <a href="https://doi.org/10.1016/j.ccr.2020.213690" title="DOI URL">https://doi.org/10.1016/j.ccr.2020.213690</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ccr.2020.213690&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ccr.2020.213690%26sid%3Dliteratum%253Aachs%26jtitle%3DCoordination%2520Chemistry%2520Reviews%26atitle%3DMultinuclear%252Bbiologically%252Bactive%252BRu%25252C%252BRh%25252C%252BOs%252Band%252BIr%252Barene%252Bcomplexes%26aulast%3D%25C5%25A0tarha%26aufirst%3DPavel%26date%3D2021%26volume%3D431%26spage%3D213690" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Joanna  Zell</span>, <span class="hlFld-ContribAuthor ">Francesco  Rota Sperti</span>, <span class="hlFld-ContribAuthor ">Sébastien  Britton</span>, <span class="hlFld-ContribAuthor ">David  Monchaud</span>. </span><span class="cited-content_cbyCitation_article-title">DNA folds threaten genetic stability and can be leveraged for chemotherapy. </span><span class="cited-content_cbyCitation_journal-name">RSC Chemical Biology</span><span> <strong>2021,</strong> <em>2 </em>
                                    (1)
                                     , 47-76. <a href="https://doi.org/10.1039/D0CB00151A" title="DOI URL">https://doi.org/10.1039/D0CB00151A</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D0CB00151A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD0CB00151A%26sid%3Dliteratum%253Aachs%26jtitle%3DRSC%2520Chemical%2520Biology%26atitle%3DDNA%252Bfolds%252Bthreaten%252Bgenetic%252Bstability%252Band%252Bcan%252Bbe%252Bleveraged%252Bfor%252Bchemotherapy%26aulast%3DZell%26aufirst%3DJoanna%26date%3D2021%26date%3D2021%26volume%3D2%26issue%3D1%26spage%3D47%26epage%3D76" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yoanes Maria  Vianney</span>, <span class="hlFld-ContribAuthor ">Pit  Preckwinkel</span>, <span class="hlFld-ContribAuthor ">Swantje  Mohr</span>, <span class="hlFld-ContribAuthor ">Klaus  Weisz</span>. </span><span class="cited-content_cbyCitation_article-title">Quadruplex–Duplex Junction: A High‐Affinity Binding Site for Indoloquinoline Ligands. </span><span class="cited-content_cbyCitation_journal-name">Chemistry – A European Journal</span><span> <strong>2020,</strong> <em>26 </em>
                                    (70)
                                     , 16910-16922. <a href="https://doi.org/10.1002/chem.202003540" title="DOI URL">https://doi.org/10.1002/chem.202003540</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/chem.202003540&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fchem.202003540%26sid%3Dliteratum%253Aachs%26jtitle%3DChemistry%2520%25E2%2580%2593%2520A%2520European%2520Journal%26atitle%3DQuadruplex%2525E2%252580%252593Duplex%252BJunction%25253A%252BA%252BHigh%2525E2%252580%252590Affinity%252BBinding%252BSite%252Bfor%252BIndoloquinoline%252BLigands%26aulast%3DVianney%26aufirst%3DYoanes%2BMaria%26date%3D2020%26date%3D2020%26volume%3D26%26issue%3D70%26spage%3D16910%26epage%3D16922" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Pauline  Lejault</span>, <span class="hlFld-ContribAuthor ">Katerina  Duskova</span>, <span class="hlFld-ContribAuthor ">Claire  Bernhard</span>, <span class="hlFld-ContribAuthor ">Ibai E.  Valverde</span>, <span class="hlFld-ContribAuthor ">Anthony  Romieu</span>, <span class="hlFld-ContribAuthor ">David  Monchaud</span>. </span><span class="cited-content_cbyCitation_article-title">The Scope of Application of Macrocyclic Polyamines Beyond Metal Chelation. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Organic Chemistry</span><span> <strong>2019,</strong> <em>2019 </em>
                                    (36)
                                     , 6146-6157. <a href="https://doi.org/10.1002/ejoc.201900870" title="DOI URL">https://doi.org/10.1002/ejoc.201900870</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/ejoc.201900870&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fejoc.201900870%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Organic%2520Chemistry%26atitle%3DThe%252BScope%252Bof%252BApplication%252Bof%252BMacrocyclic%252BPolyamines%252BBeyond%252BMetal%252BChelation%26aulast%3DLejault%26aufirst%3DPauline%26date%3D2019%26date%3D2019%26volume%3D2019%26issue%3D36%26spage%3D6146%26epage%3D6157" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.8b01978/20190502/images/medium/jm-2018-019785_0003.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.8b01978/20190502/images/large/jm-2018-019785_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01978&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.8b01978/20190502/images/medium/jm-2018-019785_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.8b01978/20190502/images/large/jm-2018-019785_0004.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. (A) Schematic representation of the folding of a TWJ structure. (B) TWJ-screen results of a library of 1200 compounds, expressed as the % of quenched FAM fluorescence of the three strands’ mixture (M) in the presence of 5 mol equiv candidate (i.e., M–(M + ligand), <i>Y</i>-axis) as a function of the % of FAM-TWJ-S1 fluorescence measured in the presence of 5 mol equiv candidate (i.e., FAM-TWJ-S1 + ligand, <i>X</i>-axis). Experiments performed with 0.2 μM DNA, without or with 1.0 μM candidates, in 10 mM lithium cacodylate plus 10 mM KCl/90 mM LiCl, pH 7.2 at 37 °C for 1 h. (C) Chemical structures of the top 15 ligands identified through TWJ-screen evaluation.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.8b01978/20190502/images/large/jm-2018-019785_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01978&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.8b01978/20190502/images/medium/jm-2018-019785_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.8b01978/20190502/images/large/jm-2018-019785_0001.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (A) PAGE analysis performed with TWJ-S1 (5.0 μM) and M (5.0 μM) in the absence and presence of the top 15 ligands (5 mol equiv) in TBE buffer + 100 mM NaCl, pH 8.3, at 4 °C for 1 h (gels were poststained with SybrGold; multiple gels assembled in a single image) (B–D) FRET-melting experiments performed with F-TWJ-T and the top 15 ligands in either a comparative (B, C) or a competitive manner (C, D, i.e., in the presence of increasing amounts of ds26 (up to 50 mol equiv)) expressed as Δ<i>T</i><sub>1/2</sub> values (in °C). Best candidates (Δ<i>T</i><sub>1/2</sub> > 10 °C, (B) and <sup>FRET</sup><i>S</i> > 80% in the presence of 50 mol equiv ds26, (D)) are highlighted in red. Experiments performed with 0.2 μM DNA, without or with 1.0 μM candidates, with or without 3 and 10 μM ds26, in 10 mM lithium cacodylate plus 10 mM KCl/90 mM LiCl, pH 7.2, between 25 and 90 °C. (E) PAGE analysis of competitive experiments performed with TWJ strands, <b>471</b> (2 mol equiv), and increasing amounts of ds12 (up to 18 mol equiv) in TBE buffer + 100 mM NaCl, pH 8.3, at 4 °C for 45 min (gels are poststained with SybrGold for visualizing ds12 and unfolded ds12, i.e., ss12).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.8b01978/20190502/images/large/jm-2018-019785_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01978&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.8b01978/20190502/images/medium/jm-2018-019785_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.8b01978/20190502/images/large/jm-2018-019785_0002.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. (A) ESI-MS profiles of the association of TWJ (left panels) or ds17 (right panels) with increasing amounts (0, 1, and 2 mol equiv) of <b>471</b>. Experiments performed with 5.0 μM DNA, without or with 5.0 or 10 μM ligand, in 100 mM ammonium acetate, pH 7.2, after 1 h equilibration at 25 °C. (B) Equilibrium dialysis results for <b>451</b>, <b>459</b>, and <b>471</b> (black bars, free ligand; brown bar, ligand bound to TWJ; orange bar, ligand bound to ds17). Experiments performed with 75 μM DNA (base pairs), without or with 2 μM ligand, in sodium phosphate + 100 mM NaCl, pH 7.2, after 24 h equilibration at 25 °C. (C) Viability profiles obtained via the sulforhodamine B (SRB) assay with <b>451</b>, <b>459</b>, and <b>471</b> after 72 h incubation (37 °C, Dulbecco’s modified Eagle’s medium (DMEM)) with MCF7 (brow line) and MDA-MB-231 (orange line) cells.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.8b01978/20190502/images/large/jm-2018-019785_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01978&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i29">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_78012" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_78012" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 48 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zeman, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cimprich, K. A.</span></span> <span> </span><span class="NLM_article-title">Causes and consequences of replication stress</span>. <i>Nat. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">2</span>– <span class="NLM_lpage">9</span>, <span class="refDoi"> DOI: 10.1038/ncb2897</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=10.1038%2Fncb2897" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=24366029" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFyks7zI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2014&pages=2-9&author=M.+K.+Zemanauthor=K.+A.+Cimprich&title=Causes+and+consequences+of+replication+stress&doi=10.1038%2Fncb2897"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Causes and consequences of replication stress</span></div><div class="casAuthors">Zeman, Michelle K.; Cimprich, Karlene A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Cell Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">2-9</span>CODEN:
                <span class="NLM_cas:coden">NCBIFN</span>;
        ISSN:<span class="NLM_cas:issn">1465-7392</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Replication stress is a complex phenomenon that has serious implications for genome stability, cell survival and human disease.  Generation of aberrant replication fork structures contg. single-stranded DNA activates the replication stress response, primarily mediated by the kinase ATR (ATM- and Rad3-related).  Along with its downstream effectors, ATR stabilizes and helps to restart stalled replication forks, avoiding the generation of DNA damage and genome instability.  Understanding this response may be key to diagnosing and treating human diseases caused by defective responses to replication stress.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrO2sHtB4BRZLVg90H21EOLACvtfcHk0liqG22ox5EfkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFyks7zI&md5=108310bae187037f311ca991fa0eed10</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fncb2897&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncb2897%26sid%3Dliteratum%253Aachs%26aulast%3DZeman%26aufirst%3DM.%2BK.%26aulast%3DCimprich%26aufirst%3DK.%2BA.%26atitle%3DCauses%2520and%2520consequences%2520of%2520replication%2520stress%26jtitle%3DNat.%2520Cell%2520Biol.%26date%3D2014%26volume%3D16%26spage%3D2%26epage%3D9%26doi%3D10.1038%2Fncb2897" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Técher, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koundrioukoff, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicolas, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Debatisse, M.</span></span> <span> </span><span class="NLM_article-title">The impact of replication stress on replication dynamics and DNA damage in vertebrate cells</span>. <i>Nat. Rev. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">535</span>– <span class="NLM_lpage">550</span>, <span class="refDoi"> DOI: 10.1038/nrg.2017.46</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=10.1038%2Fnrg.2017.46" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=28714480" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtFOrurzK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=535-550&author=H.+T%C3%A9cherauthor=S.+Koundrioukoffauthor=A.+Nicolasauthor=M.+Debatisse&title=The+impact+of+replication+stress+on+replication+dynamics+and+DNA+damage+in+vertebrate+cells&doi=10.1038%2Fnrg.2017.46"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">The impact of replication stress on replication dynamics and DNA damage in vertebrate cells</span></div><div class="casAuthors">Techer, Herve; Koundrioukoff, Stephane; Nicolas, Alain; Debatisse, Michelle</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Genetics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">535-550</span>CODEN:
                <span class="NLM_cas:coden">NRGAAM</span>;
        ISSN:<span class="NLM_cas:issn">1471-0056</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The interplay between replication stress and the S phase checkpoint is a key determinant of genome maintenance, and has a major impact on human diseases, notably, tumor initiation and progression.  Recent studies have yielded insights into sequence-dependent and sequence-independent sources of endogenous replication stress.  These stresses result in nuclease-induced DNA damage, checkpoint activation and genome-wide replication fork slowing.  Several hypotheses have been proposed to account for the mechanisms involved in this complex response.  Recent results have shown that the slowing of the replication forks most commonly results from DNA precursor starvation.  By concomitantly increasing the d. of replication initiation, the cell elicits an efficient compensatory strategy to avoid mitotic anomalies and the inheritance of damage over cell generations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovTOAyuyvPobVg90H21EOLACvtfcHk0li2utMv4POtOg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtFOrurzK&md5=137bbadc349995265f02b390544817a1</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnrg.2017.46&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrg.2017.46%26sid%3Dliteratum%253Aachs%26aulast%3DT%25C3%25A9cher%26aufirst%3DH.%26aulast%3DKoundrioukoff%26aufirst%3DS.%26aulast%3DNicolas%26aufirst%3DA.%26aulast%3DDebatisse%26aufirst%3DM.%26atitle%3DThe%2520impact%2520of%2520replication%2520stress%2520on%2520replication%2520dynamics%2520and%2520DNA%2520damage%2520in%2520vertebrate%2520cells%26jtitle%3DNat.%2520Rev.%2520Genet.%26date%3D2017%26volume%3D18%26spage%3D535%26epage%3D550%26doi%3D10.1038%2Fnrg.2017.46" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gaillard, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">García-Muse, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aguilera, A.</span></span> <span> </span><span class="NLM_article-title">Replication stress and cancer</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">276</span>– <span class="NLM_lpage">289</span>, <span class="refDoi"> DOI: 10.1038/nrc3916</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=10.1038%2Fnrc3916" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=25907220" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1Wrur3K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2015&pages=276-289&author=H.+Gaillardauthor=T.+Garc%C3%ADa-Museauthor=A.+Aguilera&title=Replication+stress+and+cancer&doi=10.1038%2Fnrc3916"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Replication stress and cancer</span></div><div class="casAuthors">Gaillard, Helene; Garcia-Muse, Tatiana; Aguilera, Andres</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">276-289</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Genome instability is a hallmark of cancer, and DNA replication is the most vulnerable cellular process that can lead to it.  Any condition leading to high levels of DNA damage will result in replication stress, which is a source of genome instability and a feature of pre-cancerous and cancerous cells.  Therefore, understanding the mol. basis of replication stress is crucial to the understanding of tumorigenesis.  Although a neg. aspect of replication stress is its prominent role in tumorigenesis, a pos. aspect is that it provides a potential target for cancer therapy.  In this Review, we discuss the link between persistent replication stress and tumorigenesis, with the goal of shedding light on the mechanisms underlying the initiation of an oncogenic process, which should open up new possibilities for cancer diagnostics and treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxvJ6UZvAnXbVg90H21EOLACvtfcHk0li2utMv4POtOg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1Wrur3K&md5=8244fca02b10d3356dbd915337203373</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnrc3916&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc3916%26sid%3Dliteratum%253Aachs%26aulast%3DGaillard%26aufirst%3DH.%26aulast%3DGarc%25C3%25ADa-Muse%26aufirst%3DT.%26aulast%3DAguilera%26aufirst%3DA.%26atitle%3DReplication%2520stress%2520and%2520cancer%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2015%26volume%3D15%26spage%3D276%26epage%3D289%26doi%3D10.1038%2Fnrc3916" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dobbelstein, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sørensen, C. S.</span></span> <span> </span><span class="NLM_article-title">Exploiting replicative stress to treat cancer</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">405</span>– <span class="NLM_lpage">423</span>, <span class="refDoi"> DOI: 10.1038/nrd4553</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=10.1038%2Fnrd4553" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2015&pages=405-423&author=M.+Dobbelsteinauthor=C.+S.+S%C3%B8rensen&title=Exploiting+replicative+stress+to+treat+cancer&doi=10.1038%2Fnrd4553"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fnrd4553&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4553%26sid%3Dliteratum%253Aachs%26aulast%3DDobbelstein%26aufirst%3DM.%26aulast%3DS%25C3%25B8rensen%26aufirst%3DC.%2BS.%26atitle%3DExploiting%2520replicative%2520stress%2520to%2520treat%2520cancer%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2015%26volume%3D14%26spage%3D405%26epage%3D423%26doi%3D10.1038%2Fnrd4553" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jackson, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartek, J.</span></span> <span> </span><span class="NLM_article-title">The DNA-damage response in human biology and disease</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>461</i></span>,  <span class="NLM_fpage">1071</span>– <span class="NLM_lpage">1078</span>, <span class="refDoi"> DOI: 10.1038/nature08467</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=10.1038%2Fnature08467" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=19847258" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtlGktLrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=461&publication_year=2009&pages=1071-1078&author=S.+P.+Jacksonauthor=J.+Bartek&title=The+DNA-damage+response+in+human+biology+and+disease&doi=10.1038%2Fnature08467"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">The DNA-damage response in human biology and disease</span></div><div class="casAuthors">Jackson, Stephen P.; Bartek, Jiri</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">461</span>
        (<span class="NLM_cas:issue">7267</span>),
    <span class="NLM_cas:pages">1071-1078</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The prime objective for every life form is to deliver its genetic material, intact and unchanged, to the next generation.  This must be achieved despite const. assaults by endogenous and environmental agents on the DNA.  To counter this threat, life has evolved several systems to detect DNA damage, signal its presence and mediate its repair.  Such responses, which have an impact on a wide range of cellular events, are biol. significant because they prevent diverse human diseases.  Our improving understanding of DNA-damage responses is providing new avenues for disease management.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpQ0Oyp6nX9V7Vg90H21EOLACvtfcHk0li2utMv4POtOg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtlGktLrK&md5=cca7cfec66852124695ebeb515f255a5</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnature08467&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature08467%26sid%3Dliteratum%253Aachs%26aulast%3DJackson%26aufirst%3DS.%2BP.%26aulast%3DBartek%26aufirst%3DJ.%26atitle%3DThe%2520DNA-damage%2520response%2520in%2520human%2520biology%2520and%2520disease%26jtitle%3DNature%26date%3D2009%26volume%3D461%26spage%3D1071%26epage%3D1078%26doi%3D10.1038%2Fnature08467" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ciccia, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elledge, S. J.</span></span> <span> </span><span class="NLM_article-title">The DNA damage response: making it safe to play with knives</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">179</span>– <span class="NLM_lpage">204</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2010.09.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=10.1016%2Fj.molcel.2010.09.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=20965415" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtlCjtr7P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2010&pages=179-204&author=A.+Cicciaauthor=S.+J.+Elledge&title=The+DNA+damage+response%3A+making+it+safe+to+play+with+knives&doi=10.1016%2Fj.molcel.2010.09.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">The DNA Damage Response: Making It Safe to Play with Knives</span></div><div class="casAuthors">Ciccia, Alberto; Elledge, Stephen J.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">179-204</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Damage to our genetic material is an ongoing threat to both our ability to faithfully transmit genetic information to our offspring as well as our own survival.  To respond to these threats, eukaryotes have evolved the DNA damage response (DDR).  The DDR is a complex signal transduction pathway that has the ability to sense DNA damage and transduce this information to the cell to influence cellular responses to DNA damage.  Cells possess an arsenal of enzymic tools capable of remodeling and repairing DNA; however, their activities must be tightly regulated in a temporal, spatial, and DNA lesion-appropriate fashion to optimize repair and prevent unnecessary and potentially deleterious alterations in the structure of DNA during normal cellular processes.  This review will focus on how the DDR controls DNA repair and the phenotypic consequences of defects in these crit. regulatory functions in mammals.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGofclv-VUxTXLVg90H21EOLACvtfcHk0lhc2p03-DsL6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtlCjtr7P&md5=71d5ed7d3a82d9b395877510bbe4df03</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2010.09.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2010.09.019%26sid%3Dliteratum%253Aachs%26aulast%3DCiccia%26aufirst%3DA.%26aulast%3DElledge%26aufirst%3DS.%2BJ.%26atitle%3DThe%2520DNA%2520damage%2520response%253A%2520making%2520it%2520safe%2520to%2520play%2520with%2520knives%26jtitle%3DMol.%2520Cell%26date%3D2010%26volume%3D40%26spage%3D179%26epage%3D204%26doi%3D10.1016%2Fj.molcel.2010.09.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguez, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, K. M.</span></span> <span> </span><span class="NLM_article-title">Unravelling the genomic targets of small molecules using high-throughput sequencing</span>. <i>Nat. Rev. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">783</span>– <span class="NLM_lpage">796</span>, <span class="refDoi"> DOI: 10.1038/nrg3796</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=10.1038%2Fnrg3796" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=25311424" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslCitbzE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2014&pages=783-796&author=R.+Rodriguezauthor=K.+M.+Miller&title=Unravelling+the+genomic+targets+of+small+molecules+using+high-throughput+sequencing&doi=10.1038%2Fnrg3796"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Unravelling the genomic targets of small molecules using high-throughput sequencing</span></div><div class="casAuthors">Rodriguez, Raphael; Miller, Kyle M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Genetics</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">783-796</span>CODEN:
                <span class="NLM_cas:coden">NRGAAM</span>;
        ISSN:<span class="NLM_cas:issn">1471-0056</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Small mols. - including various approved and novel cancer therapeutics - can operate at the genomic level by targeting the DNA and protein components of chromatin.  Emerging evidence suggests that functional interactions between small mols. and the genome are non-stochastic and are influenced by a dynamic interplay between DNA sequences and chromatin states.  The establishment of genome-wide maps of small-mol. targets using unbiased methodologies can help to characterize and exploit drug responses.  In this Review, we discuss how high-throughput sequencing strategies, such as ChIP-seq (chromatin immunopptn. followed by sequencing) and Chem-seq (chem. affinity capture and massively parallel DNA sequencing), are enabling the comprehensive identification of small-mol. target sites throughout the genome, thereby providing insights into unanticipated drug effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9JdFmc3eSnrVg90H21EOLACvtfcHk0lhc2p03-DsL6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslCitbzE&md5=569d9a5be0cb270a030314d93b24c11f</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fnrg3796&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrg3796%26sid%3Dliteratum%253Aachs%26aulast%3DRodriguez%26aufirst%3DR.%26aulast%3DMiller%26aufirst%3DK.%2BM.%26atitle%3DUnravelling%2520the%2520genomic%2520targets%2520of%2520small%2520molecules%2520using%2520high-throughput%2520sequencing%26jtitle%3DNat.%2520Rev.%2520Genet.%26date%3D2014%26volume%3D15%26spage%3D783%26epage%3D796%26doi%3D10.1038%2Fnrg3796" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Helleday, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petermann, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lundin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hodgson, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, R. A.</span></span> <span> </span><span class="NLM_article-title">DNA repair pathways as targets for cancer therapy</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">193</span>– <span class="NLM_lpage">204</span>, <span class="refDoi"> DOI: 10.1038/nrc2342</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=10.1038%2Fnrc2342" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=18256616" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=1%3ACAS%3A528%3ADC%252BD1cXitlOisr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2008&pages=193-204&author=T.+Helledayauthor=E.+Petermannauthor=C.+Lundinauthor=B.+Hodgsonauthor=R.+A.+Sharma&title=DNA+repair+pathways+as+targets+for+cancer+therapy&doi=10.1038%2Fnrc2342"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">DNA repair pathways as targets for cancer therapy</span></div><div class="casAuthors">Helleday, Thomas; Petermann, Eva; Lundin, Cecilia; Hodgson, Ben; Sharma, Ricky A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">193-204</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  DNA repair pathways can enable tumor cells to survive DNA damage that is induced by chemotherapeutic treatments; therefore, inhibitors of specific DNA repair pathways might prove efficacious when used in combination with DNA-damaging chemotherapeutic drugs.  In addn., alterations in DNA repair pathways that arise during tumor development can make some cancer cells reliant on a reduced set of DNA repair pathways for survival.  There is evidence that drugs that inhibit one of these pathways in such tumors could prove useful as single-agent therapies, with the potential advantage that this approach could be selective for tumor cells and have fewer side effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3W53LJmdCPrVg90H21EOLACvtfcHk0lhc2p03-DsL6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXitlOisr8%253D&md5=8592af6d815e5ca9c80ee256791bafb7</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnrc2342&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc2342%26sid%3Dliteratum%253Aachs%26aulast%3DHelleday%26aufirst%3DT.%26aulast%3DPetermann%26aufirst%3DE.%26aulast%3DLundin%26aufirst%3DC.%26aulast%3DHodgson%26aufirst%3DB.%26aulast%3DSharma%26aufirst%3DR.%2BA.%26atitle%3DDNA%2520repair%2520pathways%2520as%2520targets%2520for%2520cancer%2520therapy%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2008%26volume%3D8%26spage%3D193%26epage%3D204%26doi%3D10.1038%2Fnrc2342" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mirkin, E. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mirkin, S. M.</span></span> <span> </span><span class="NLM_article-title">Replication fork stalling at natural impediments</span>. <i>Microbiol. Mol. Biol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>71</i></span>,  <span class="NLM_fpage">13</span>– <span class="NLM_lpage">35</span>, <span class="refDoi"> DOI: 10.1128/MMBR.00030-06</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=10.1128%2FMMBR.00030-06" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=17347517" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=1%3ACAS%3A528%3ADC%252BD2sXksleqsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2007&pages=13-35&author=E.+V.+Mirkinauthor=S.+M.+Mirkin&title=Replication+fork+stalling+at+natural+impediments&doi=10.1128%2FMMBR.00030-06"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Replication fork stalling at natural impediments</span></div><div class="casAuthors">Mirkin, Ekaterina V.; Mirkin, Sergei M.</div><div class="citationInfo"><span class="NLM_cas:title">Microbiology and Molecular Biology Reviews</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">71</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">13-35</span>CODEN:
                <span class="NLM_cas:coden">MMBRF7</span>;
        ISSN:<span class="NLM_cas:issn">1092-2172</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">A review.  Accurate and complete replication of the genome in every cell division is a prerequisite of genomic stability.  Thus, both prokaryotic and eukaryotic replication forks are extremely precise and robust mol. machines that have evolved to be up to the task.  However, it has recently become clear that the replication fork is more of a hurdler than a runner: it must overcome various obstacles present on its way.  Such obstacles can be called natural impediments to DNA replication, as opposed to external and genetic factors.  Natural impediments to DNA replication are particular DNA binding proteins, unusual secondary structures in DNA, and transcription complexes that occasionally (in eukaryotes) or constantly (in prokaryotes) operate on replicating templates.  This review describes the mechanisms and consequences of replication stalling at various natural impediments, with an emphasis on the role of replication stalling in genomic instability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcteIUmVZAYLVg90H21EOLACvtfcHk0lhc2p03-DsL6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXksleqsLg%253D&md5=f3e547ba124f2a93c5fb6cd3d8e64222</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1128%2FMMBR.00030-06&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMMBR.00030-06%26sid%3Dliteratum%253Aachs%26aulast%3DMirkin%26aufirst%3DE.%2BV.%26aulast%3DMirkin%26aufirst%3DS.%2BM.%26atitle%3DReplication%2520fork%2520stalling%2520at%2520natural%2520impediments%26jtitle%3DMicrobiol.%2520Mol.%2520Biol.%2520Rev.%26date%3D2007%26volume%3D71%26spage%3D13%26epage%3D35%26doi%3D10.1128%2FMMBR.00030-06" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bacolla, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vasquez, K. M.</span></span> <span> </span><span class="NLM_article-title">Non-B DNA structure-induced genetic instability and evolution</span>. <i>Cell. Mol. Life Sci.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">43</span>– <span class="NLM_lpage">62</span>, <span class="refDoi"> DOI: 10.1007/s00018-009-0131-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=10.1007%2Fs00018-009-0131-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=19727556" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=1%3ACAS%3A528%3ADC%252BC3cXis1ajsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2010&pages=43-62&author=J.+Zhaoauthor=A.+Bacollaauthor=G.+Wangauthor=K.+M.+Vasquez&title=Non-B+DNA+structure-induced+genetic+instability+and+evolution&doi=10.1007%2Fs00018-009-0131-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Non-B DNA structure-induced genetic instability and evolution</span></div><div class="casAuthors">Zhao, Junhua; Bacolla, Albino; Wang, Guliang; Vasquez, Karen M.</div><div class="citationInfo"><span class="NLM_cas:title">Cellular and Molecular Life Sciences</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">43-62</span>CODEN:
                <span class="NLM_cas:coden">CMLSFI</span>;
        ISSN:<span class="NLM_cas:issn">1420-682X</span>.
    
            (<span class="NLM_cas:orgname">Birkhaeuser Verlag</span>)
        </div><div class="casAbstract">A review.  Repetitive DNA motifs are abundant in the genomes of various species and have the capacity to adopt non-canonical (i.e., non-B) DNA structures.  Several non-B DNA structures, including cruciforms, slipped structures, triplexes, G-quadruplexes, and Z-DNA, have been shown to cause mutations, such as deletions, expansions, and translocations in both prokaryotes and eukaryotes.  Their distributions in genomes are not random and often co-localize with sites of chromosomal breakage assocd. with genetic diseases.  Current genome-wide sequence analyses suggest that the genomic instabilities induced by non-B DNA structure-forming sequences not only result in predisposition to disease, but also contribute to rapid evolutionary changes, particularly in genes assocd. with development and regulatory functions.  In this review, we describe the occurrence of non-B DNA-forming sequences in various species, the classes of genes enriched in non-B DNA-forming sequences, and recent mechanistic studies on DNA structure-induced genomic instability to highlight their importance in genomes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8JVuub9QLlrVg90H21EOLACvtfcHk0ljvQSINGlhG6A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXis1ajsA%253D%253D&md5=452d69f118747ee4f8b513f1c8d9723c</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1007%2Fs00018-009-0131-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00018-009-0131-2%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DJ.%26aulast%3DBacolla%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DG.%26aulast%3DVasquez%26aufirst%3DK.%2BM.%26atitle%3DNon-B%2520DNA%2520structure-induced%2520genetic%2520instability%2520and%2520evolution%26jtitle%3DCell.%2520Mol.%2520Life%2520Sci.%26date%3D2010%26volume%3D67%26spage%3D43%26epage%3D62%26doi%3D10.1007%2Fs00018-009-0131-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Caruthers, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKay, D. B.</span></span> <span> </span><span class="NLM_article-title">Helicase structure and mechanism</span>. <i>Curr. Opin. Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">123</span>– <span class="NLM_lpage">133</span>, <span class="refDoi"> DOI: 10.1016/S0959-440X(02)00298-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=10.1016%2FS0959-440X%2802%2900298-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=11839499" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=1%3ACAS%3A528%3ADC%252BD38XhtF2itLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2002&pages=123-133&author=J.+M.+Caruthersauthor=D.+B.+McKay&title=Helicase+structure+and+mechanism&doi=10.1016%2FS0959-440X%2802%2900298-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Helicase structure and mechanism</span></div><div class="casAuthors">Caruthers, Jonathan M.; McKay, David B.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Structural Biology</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">123-133</span>CODEN:
                <span class="NLM_cas:coden">COSBEF</span>;
        ISSN:<span class="NLM_cas:issn">0959-440X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">A review with 42 refs.  Structural information on helicase proteins has expanded recently beyond DNA helicases Rep and PcrA, and hepatitis C virus RNA helicase to include UvrB, members of the DEA(D/H)-box RNA helicase family, examples of DnaB-related helicases, and RuvB.  The expanding database of structures has clarified the structural 'theme and variations' that relate the different helicase families.  Furthermore, information is emerging on the functions of the conserved helicase motifs and their participation in the mechanisms by which these proteins catalyze the remodeling of DNA and RNA in ATP-dependent activities.  The expanding database of helicase structures, enzymes that catalyze the sepn. of duplex oligonucleotides into single strands, has clarified the structural themes and variations that relate the different families.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXfRg9Xe7zjbVg90H21EOLACvtfcHk0ljvQSINGlhG6A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XhtF2itLY%253D&md5=138b992adc3e001b305ba02cf1108268</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2FS0959-440X%2802%2900298-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0959-440X%252802%252900298-1%26sid%3Dliteratum%253Aachs%26aulast%3DCaruthers%26aufirst%3DJ.%2BM.%26aulast%3DMcKay%26aufirst%3DD.%2BB.%26atitle%3DHelicase%2520structure%2520and%2520mechanism%26jtitle%3DCurr.%2520Opin.%2520Struct.%2520Biol.%26date%3D2002%26volume%3D12%26spage%3D123%26epage%3D133%26doi%3D10.1016%2FS0959-440X%2802%2900298-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zimmer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tacconi, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Folio, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Badie, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porru, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klare, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tumiati, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Markkanen, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halder, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryan, A.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Targeting BRCA1 and BRCA2 deficiencies with G-quadruplex-interacting compounds</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">449</span>– <span class="NLM_lpage">460</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2015.12.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=10.1016%2Fj.molcel.2015.12.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=26748828" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtlKlug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2016&pages=449-460&author=J.+Zimmerauthor=E.+M.+Tacconiauthor=C.+Folioauthor=S.+Badieauthor=M.+Porruauthor=K.+Klareauthor=M.+Tumiatiauthor=E.+Markkanenauthor=S.+Halderauthor=A.+Ryan&title=Targeting+BRCA1+and+BRCA2+deficiencies+with+G-quadruplex-interacting+compounds&doi=10.1016%2Fj.molcel.2015.12.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting BRCA1 and BRCA2 Deficiencies with G-Quadruplex-Interacting Compounds</span></div><div class="casAuthors">Zimmer, Jutta; Tacconi, Eliana M. C.; Folio, Cecilia; Badie, Sophie; Porru, Manuela; Klare, Kerstin; Tumiati, Manuela; Markkanen, Enni; Halder, Swagata; Ryan, Anderson; Jackson, Stephen P.; Ramadan, Kristijan; Kuznetsov, Sergey G.; Biroccio, Annamaria; Sale, Julian E.; Tarsounas, Madalena</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">449-460</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">G-quadruplex (G4)-forming genomic sequences, including telomeres, represent natural replication fork barriers.  Stalled replication forks can be stabilized and restarted by homologous recombination (HR), which also repairs DNA double-strand breaks (DSBs) arising at collapsed forks.  We have previously shown that HR facilitates telomere replication.  Here, we demonstrate that the replication efficiency of guanine-rich (G-rich) telomeric repeats is decreased significantly in cells lacking HR.  Treatment with the G4-stabilizing compd. pyridostatin (PDS) increases telomere fragility in BRCA2-deficient cells, suggesting that G4 formation drives telomere instability.  Remarkably, PDS reduces proliferation of HR-defective cells by inducing DSB accumulation, checkpoint activation, and deregulated G2/M progression and by enhancing the replication defect intrinsic to HR deficiency.  PDS toxicity extends to HR-defective cells that have acquired olaparib resistance through loss of 53BP1 or REV7.  Altogether, these results highlight the therapeutic potential of G4-stabilizing drugs to selectively eliminate HR-compromised cells and tumors, including those resistant to PARP inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5EXONoaAQUrVg90H21EOLACvtfcHk0ljvQSINGlhG6A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtlKlug%253D%253D&md5=b2299dbbb76cc442c47a1889174caf5b</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2015.12.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2015.12.004%26sid%3Dliteratum%253Aachs%26aulast%3DZimmer%26aufirst%3DJ.%26aulast%3DTacconi%26aufirst%3DE.%2BM.%26aulast%3DFolio%26aufirst%3DC.%26aulast%3DBadie%26aufirst%3DS.%26aulast%3DPorru%26aufirst%3DM.%26aulast%3DKlare%26aufirst%3DK.%26aulast%3DTumiati%26aufirst%3DM.%26aulast%3DMarkkanen%26aufirst%3DE.%26aulast%3DHalder%26aufirst%3DS.%26aulast%3DRyan%26aufirst%3DA.%26atitle%3DTargeting%2520BRCA1%2520and%2520BRCA2%2520deficiencies%2520with%2520G-quadruplex-interacting%2520compounds%26jtitle%3DMol.%2520Cell%26date%3D2016%26volume%3D61%26spage%3D449%26epage%3D460%26doi%3D10.1016%2Fj.molcel.2015.12.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Antonio, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKinney, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mathew, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ho, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Neil, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dos Santos, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silvester, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia, J.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">CX-5461 is a DNA G-quadruplex stabilizer with selective lethality in BRCA1/2 deficient tumours</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">14432</span> <span class="refDoi"> DOI: 10.1038/ncomms14432</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=10.1038%2Fncomms14432" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=28211448" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjtVKms7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&author=H.+Xuauthor=M.+Di+Antonioauthor=S.+McKinneyauthor=V.+Mathewauthor=B.+Hoauthor=N.+J.+O%E2%80%99Neilauthor=N.+Dos+Santosauthor=J.+Silvesterauthor=V.+Weiauthor=J.+Garcia&title=CX-5461+is+a+DNA+G-quadruplex+stabilizer+with+selective+lethality+in+BRCA1%2F2+deficient+tumours&doi=10.1038%2Fncomms14432"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">CX-5461 is a DNA G-quadruplex stabilizer with selective lethality in BRCA1/2 deficient tumours</span></div><div class="casAuthors">Xu, Hong; Di Antonio, Marco; McKinney, Steven; Mathew, Veena; Ho, Brandon; O'Neil, Nigel J.; Dos Santos, Nancy; Silvester, Jennifer; Wei, Vivien; Garcia, Jessica; Kabeer, Farhia; Lai, Daniel; Soriano, Priscilla; Banath, Judit; Chiu, Derek S.; Yap, Damian; Le, Daniel D.; Ye, Frank B.; Zhang, Anni; Thu, Kelsie; Soong, John; Lin, Shu-chuan; Tsai, Angela Hsin Chin; Osako, Tomo; Algara, Teresa; Saunders, Darren N.; Wong, Jason; Xian, Jian; Bally, Marcel B.; Brenton, James D.; Brown, Grant W.; Shah, Sohrab P.; Cescon, David; Mak, Tak W.; Caldas, Carlos; Stirling, Peter C.; Hieter, Phil; Balasubramanian, Shankar; Aparicio, Samuel</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">14432</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">G-quadruplex DNAs form four-stranded helical structures and are proposed to play key roles in different cellular processes.  Targeting G-quadruplex DNAs for cancer treatment is a very promising prospect.  Here, we show that CX-5461 is a G-quadruplex stabilizer, with specific toxicity against BRCA deficiencies in cancer cells and polyclonal patient-derived xenograft models, including tumors resistant to PARP inhibition.  Exposure to CX-5461, and its related drug CX-3543, blocks replication forks and induces ssDNA gaps or breaks.  The BRCA and NHEJ pathways are required for the repair of CX-5461 and CX-3543-induced DNA damage and failure to do so leads to lethality.  These data strengthen the concept of G4 targeting as a therapeutic approach, specifically for targeting HR and NHEJ deficient cancers and other tumors deficient for DNA damage repair.  CX-5461 is now in advanced phase I clin. trial for patients with BRCA1/2 deficient tumors (Canadian trial, NCT02719977, opened May 2016).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGppKJRejuN5X7Vg90H21EOLACvtfcHk0lhi3wog-XTE5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjtVKms7s%253D&md5=8e496952f88642ff05eaf136fe6856d6</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2Fncomms14432&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms14432%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DH.%26aulast%3DDi%2BAntonio%26aufirst%3DM.%26aulast%3DMcKinney%26aufirst%3DS.%26aulast%3DMathew%26aufirst%3DV.%26aulast%3DHo%26aufirst%3DB.%26aulast%3DO%25E2%2580%2599Neil%26aufirst%3DN.%2BJ.%26aulast%3DDos%2BSantos%26aufirst%3DN.%26aulast%3DSilvester%26aufirst%3DJ.%26aulast%3DWei%26aufirst%3DV.%26aulast%3DGarcia%26aufirst%3DJ.%26atitle%3DCX-5461%2520is%2520a%2520DNA%2520G-quadruplex%2520stabilizer%2520with%2520selective%2520lethality%2520in%2520BRCA1%252F2%2520deficient%2520tumours%26jtitle%3DNat.%2520Commun.%26date%3D2017%26volume%3D8%26doi%3D10.1038%2Fncomms14432" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lemmens, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Schendel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tijsterman, M.</span></span> <span> </span><span class="NLM_article-title">Mutagenic consequences of a single G-quadruplex demonstrate mitotic inheritance of DNA replication fork barriers</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>, <span class="NLM_elocation-id">8909</span> <span class="refDoi"> DOI: 10.1038/ncomms9909</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=10.1038%2Fncomms9909" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=26563448" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVOmsbbK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&author=B.+Lemmensauthor=R.+Van+Schendelauthor=M.+Tijsterman&title=Mutagenic+consequences+of+a+single+G-quadruplex+demonstrate+mitotic+inheritance+of+DNA+replication+fork+barriers&doi=10.1038%2Fncomms9909"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Mutagenic consequences of a single G-quadruplex demonstrate mitotic inheritance of DNA replication fork barriers</span></div><div class="casAuthors">Lemmens, Bennie; van Schendel, Robin; Tijsterman, Marcel</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">8909</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Faithful DNA replication is vital to prevent disease-causing mutations, chromosomal aberrations and malignant transformation.  However, accuracy conflicts with pace and flexibility and cells rely on specialized polymerases and helicases to ensure effective and timely replication of genomes that contain DNA lesions or secondary structures.  If and how cells can tolerate a permanent barrier to replication is, however, unknown.  Here we show that a single unresolved G-quadruplexed DNA structure can persist through multiple mitotic divisions without changing conformation.  Failed replication across a G-quadruplex causes single-strand DNA gaps that give rise to DNA double-strand breaks in subsequent cell divisions, which are processed by polymerase theta (POLQ)-mediated alternative end joining.  Lineage tracing expts. further reveal that persistent G-quadruplexes cause genetic heterogeneity during organ development.  Our data demonstrate that a single lesion can cause multiple unique genomic rearrangements, and that alternative end joining enables cells to proliferate in the presence of mitotically inherited replication blocks.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqp8h0JCrSGUrVg90H21EOLACvtfcHk0lhi3wog-XTE5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVOmsbbK&md5=db43b84764420d505e3c6dba0bc0c70d</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1038%2Fncomms9909&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms9909%26sid%3Dliteratum%253Aachs%26aulast%3DLemmens%26aufirst%3DB.%26aulast%3DVan%2BSchendel%26aufirst%3DR.%26aulast%3DTijsterman%26aufirst%3DM.%26atitle%3DMutagenic%2520consequences%2520of%2520a%2520single%2520G-quadruplex%2520demonstrate%2520mitotic%2520inheritance%2520of%2520DNA%2520replication%2520fork%2520barriers%26jtitle%3DNat.%2520Commun.%26date%3D2015%26volume%3D6%26doi%3D10.1038%2Fncomms9909" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bochman, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paeschke, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zakian, V. A.</span></span> <span> </span><span class="NLM_article-title">DNA secondary structures: stability and function of G-quadruplex structures</span>. <i>Nat. Rev. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">770</span>– <span class="NLM_lpage">780</span>, <span class="refDoi"> DOI: 10.1038/nrg3296</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=10.1038%2Fnrg3296" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=23032257" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVCisb%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2012&pages=770-780&author=M.+L.+Bochmanauthor=K.+Paeschkeauthor=V.+A.+Zakian&title=DNA+secondary+structures%3A+stability+and+function+of+G-quadruplex+structures&doi=10.1038%2Fnrg3296"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">DNA secondary structures: stability and function of structures</span></div><div class="casAuthors">Bochman, Matthew L.; Paeschke, Katrin; Zakian, Virginia A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Genetics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">770-780</span>CODEN:
                <span class="NLM_cas:coden">NRGAAM</span>;
        ISSN:<span class="NLM_cas:issn">1471-0056</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  In addn. to the canonical double helix, DNA can fold into various other inter- and intramol. secondary structures.  Although many such structures were long thought to be in vitro artifacts, bioinformatics demonstrates that DNA sequences capable of forming these structures are conserved throughout evolution, suggesting the existence of non-B-form DNA in vivo.  In addn., genes whose products promote formation or resoln. of these structures are found in diverse organisms, and a growing body of work suggests that the resoln. of DNA secondary structures is crit. for genome integrity.  This Review focuses on emerging evidence relating to the characteristics of G-quadruplex structures and the possible influence of such structures on genomic stability and cellular processes, such as transcription.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpov7zDhgxxprVg90H21EOLACvtfcHk0lhi3wog-XTE5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVCisb%252FE&md5=9d70afb3ef1bca023874bcede128223a</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fnrg3296&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrg3296%26sid%3Dliteratum%253Aachs%26aulast%3DBochman%26aufirst%3DM.%2BL.%26aulast%3DPaeschke%26aufirst%3DK.%26aulast%3DZakian%26aufirst%3DV.%2BA.%26atitle%3DDNA%2520secondary%2520structures%253A%2520stability%2520and%2520function%2520of%2520G-quadruplex%2520structures%26jtitle%3DNat.%2520Rev.%2520Genet.%26date%3D2012%26volume%3D13%26spage%3D770%26epage%3D780%26doi%3D10.1038%2Fnrg3296" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hänsel-Hertsch, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Antonio, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balasubramanian, S.</span></span> <span> </span><span class="NLM_article-title">DNA G-quadruplexes in the human genome: detection, functions and therapeutic potential</span>. <i>Nat. Rev. Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">279</span>, <span class="refDoi"> DOI: 10.1038/nrm.2017.3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=10.1038%2Fnrm.2017.3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=28225080" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjt1agsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=279&author=R.+H%C3%A4nsel-Hertschauthor=M.+Di+Antonioauthor=S.+Balasubramanian&title=DNA+G-quadruplexes+in+the+human+genome%3A+detection%2C+functions+and+therapeutic+potential&doi=10.1038%2Fnrm.2017.3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">DNA G-quadruplexes in the human genome: detection, functions and therapeutic potential</span></div><div class="casAuthors">Hansel-Hertsch, Robert; Di Antonio, Marco; Balasubramanian, Shankar</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">279-284</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Single-stranded guanine-rich DNA sequences can fold into four-stranded DNA structures called G-quadruplexes (G4s) that arise from the self-stacking of two or more guanine quartets.  There has been considerable recent progress in the detection and mapping of G4 structures in the human genome and in biol. relevant contexts.  These advancements, many of which align with predictions made previously in computational studies, provide important new insights into the functions of G4 structures in, for example, the regulation of transcription and genome stability, and uncover their potential relevance for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojysTqYKqZi7Vg90H21EOLACvtfcHk0lhi3wog-XTE5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjt1agsr0%253D&md5=452d13815a3284a4ca51607d1522fb5f</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1038%2Fnrm.2017.3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm.2017.3%26sid%3Dliteratum%253Aachs%26aulast%3DH%25C3%25A4nsel-Hertsch%26aufirst%3DR.%26aulast%3DDi%2BAntonio%26aufirst%3DM.%26aulast%3DBalasubramanian%26aufirst%3DS.%26atitle%3DDNA%2520G-quadruplexes%2520in%2520the%2520human%2520genome%253A%2520detection%252C%2520functions%2520and%2520therapeutic%2520potential%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2017%26volume%3D18%26spage%3D279%26doi%3D10.1038%2Fnrm.2017.3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vuong, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stefan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lejault, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rousselin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denat, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monchaud, D.</span></span> <span> </span><span class="NLM_article-title">Identifying three-way DNA junction-specific small-molecules</span>. <i>Biochimie</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>94</i></span>,  <span class="NLM_fpage">442</span>– <span class="NLM_lpage">450</span>, <span class="refDoi"> DOI: 10.1016/j.biochi.2011.08.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=10.1016%2Fj.biochi.2011.08.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=21884749" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=1%3ACAS%3A528%3ADC%252BC38XovFCqsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=94&publication_year=2012&pages=442-450&author=S.+Vuongauthor=L.+Stefanauthor=P.+Lejaultauthor=Y.+Rousselinauthor=F.+Denatauthor=D.+Monchaud&title=Identifying+three-way+DNA+junction-specific+small-molecules&doi=10.1016%2Fj.biochi.2011.08.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Identifying three-way DNA junction-specific small-molecules</span></div><div class="casAuthors">Vuong, Sophie; Stefan, Loic; Lejault, Pauline; Rousselin, Yoann; Denat, Franck; Monchaud, David</div><div class="citationInfo"><span class="NLM_cas:title">Biochimie</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">94</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">442-450</span>CODEN:
                <span class="NLM_cas:coden">BICMBE</span>;
        ISSN:<span class="NLM_cas:issn">0300-9084</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Three-way junction DNA (TWJ-DNA, also known as 3WJ-DNA) is an alternative secondary DNA structure comprised of three duplex-DNAs that converge towards a single point, termed the branch point.  This point is characterized by unique geometrical properties that make its specific targeting by synthetic small-mols. possible.  Such a targeting has already been demonstrated in the solid state but not thoroughly biophys. investigated in soln.  Herein, a set of simple biophys. assays has been developed to identify TWJ-specific small-mol. ligands; these assays, inspired by the considerable body of work that has been reported to characterize the interactions between small-mols. and other higher-order DNA (notably quadruplex-DNA), have been calibrated with a known non-specific DNA binder (the porphyrin TMPyP4) and validated via the study of a small series of triazacyclononane (TACN) derivs. (metal-free or not) and the identification of a fairly-affinic and exquisitely TWJ-selective candidate (a TACN-quinoline construct named TACN-Q).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpQFURKgDIsMLVg90H21EOLACvtfcHk0lh16jLnCRgBVQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XovFCqsA%253D%253D&md5=22e86bc003f218a8e78097401a7afda0</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.biochi.2011.08.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.biochi.2011.08.012%26sid%3Dliteratum%253Aachs%26aulast%3DVuong%26aufirst%3DS.%26aulast%3DStefan%26aufirst%3DL.%26aulast%3DLejault%26aufirst%3DP.%26aulast%3DRousselin%26aufirst%3DY.%26aulast%3DDenat%26aufirst%3DF.%26aulast%3DMonchaud%26aufirst%3DD.%26atitle%3DIdentifying%2520three-way%2520DNA%2520junction-specific%2520small-molecules%26jtitle%3DBiochimie%26date%3D2012%26volume%3D94%26spage%3D442%26epage%3D450%26doi%3D10.1016%2Fj.biochi.2011.08.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Novotna, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laguerre, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Granzhan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pirrotta, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teulade-Fichou, M.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monchaud, D.</span></span> <span> </span><span class="NLM_article-title">Cationic azacryptands as selective three-way DNA junction binding agents</span>. <i>Org. Biomol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">215</span>– <span class="NLM_lpage">222</span>, <span class="refDoi"> DOI: 10.1039/C4OB01846J</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=10.1039%2FC4OB01846J" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2015&pages=215-222&author=J.+Novotnaauthor=A.+Laguerreauthor=A.+Granzhanauthor=M.+Pirrottaauthor=M.-P.+Teulade-Fichouauthor=D.+Monchaud&title=Cationic+azacryptands+as+selective+three-way+DNA+junction+binding+agents&doi=10.1039%2FC4OB01846J"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1039%2FC4OB01846J&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC4OB01846J%26sid%3Dliteratum%253Aachs%26aulast%3DNovotna%26aufirst%3DJ.%26aulast%3DLaguerre%26aufirst%3DA.%26aulast%3DGranzhan%26aufirst%3DA.%26aulast%3DPirrotta%26aufirst%3DM.%26aulast%3DTeulade-Fichou%26aufirst%3DM.-P.%26aulast%3DMonchaud%26aufirst%3DD.%26atitle%3DCationic%2520azacryptands%2520as%2520selective%2520three-way%2520DNA%2520junction%2520binding%2520agents%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2015%26volume%3D13%26spage%3D215%26epage%3D222%26doi%3D10.1039%2FC4OB01846J" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guyon, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pirrotta, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duskova, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Granzhan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teulade-Fichou, M.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monchaud, D.</span></span> <span> </span><span class="NLM_article-title">TWJ-Screen: an isothermal screening assay to assess ligand/DNA junction interactions in vitro</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>46</i></span>, <span class="NLM_elocation-id">e16</span> <span class="refDoi"> DOI: 10.1093/nar/gkx1118</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=10.1093%2Fnar%2Fgkx1118" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2018&author=L.+Guyonauthor=M.+Pirrottaauthor=K.+Duskovaauthor=A.+Granzhanauthor=M.-P.+Teulade-Fichouauthor=D.+Monchaud&title=TWJ-Screen%3A+an+isothermal+screening+assay+to+assess+ligand%2FDNA+junction+interactions+in+vitro&doi=10.1093%2Fnar%2Fgkx1118"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkx1118&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkx1118%26sid%3Dliteratum%253Aachs%26aulast%3DGuyon%26aufirst%3DL.%26aulast%3DPirrotta%26aufirst%3DM.%26aulast%3DDuskova%26aufirst%3DK.%26aulast%3DGranzhan%26aufirst%3DA.%26aulast%3DTeulade-Fichou%26aufirst%3DM.-P.%26aulast%3DMonchaud%26aufirst%3DD.%26atitle%3DTWJ-Screen%253A%2520an%2520isothermal%2520screening%2520assay%2520to%2520assess%2520ligand%252FDNA%2520junction%2520interactions%2520in%2520vitro%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2018%26volume%3D46%26doi%3D10.1093%2Fnar%2Fgkx1118" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Oleksi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blanco, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Usón, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aymamí, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodger, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hannon, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coll, M.</span></span> <span> </span><span class="NLM_article-title">Molecular recognition of a three-way DNA junction by a metallosupramolecular helicate</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">1227</span>– <span class="NLM_lpage">1231</span>, <span class="refDoi"> DOI: 10.1002/anie.200503822</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=10.1002%2Fanie.200503822" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=1%3ACAS%3A528%3ADC%252BD28XhvFyjsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2006&pages=1227-1231&author=A.+Oleksiauthor=A.+G.+Blancoauthor=R.+Boerauthor=I.+Us%C3%B3nauthor=J.+Aymam%C3%ADauthor=A.+Rodgerauthor=M.+J.+Hannonauthor=M.+Coll&title=Molecular+recognition+of+a+three-way+DNA+junction+by+a+metallosupramolecular+helicate&doi=10.1002%2Fanie.200503822"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular recognition of a three-way DNA junction by a metallosupramolecular helicate</span></div><div class="casAuthors">Oleksi, Aneta; Blanco, Alexandre G.; Boer, Roeland; Uson, Isabel; Aymami, Joan; Rodger, Alison; Hannon, Michael J.; Coll, Miquel</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1227-1231</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Three point turn: A metallosupramol. helicate with trigonal-antiprismatic geometry fits perfectly into the central hydrophobic cavity of a three-way DNA junction allowing a new mode of DNA recognition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWvdABofmYkLVg90H21EOLACvtfcHk0lh16jLnCRgBVQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhvFyjsr0%253D&md5=28e2147e5ab2cb533bc33aee6728ef40</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1002%2Fanie.200503822&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.200503822%26sid%3Dliteratum%253Aachs%26aulast%3DOleksi%26aufirst%3DA.%26aulast%3DBlanco%26aufirst%3DA.%2BG.%26aulast%3DBoer%26aufirst%3DR.%26aulast%3DUs%25C3%25B3n%26aufirst%3DI.%26aulast%3DAymam%25C3%25AD%26aufirst%3DJ.%26aulast%3DRodger%26aufirst%3DA.%26aulast%3DHannon%26aufirst%3DM.%2BJ.%26aulast%3DColl%26aufirst%3DM.%26atitle%3DMolecular%2520recognition%2520of%2520a%2520three-way%2520DNA%2520junction%2520by%2520a%2520metallosupramolecular%2520helicate%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2006%26volume%3D45%26spage%3D1227%26epage%3D1231%26doi%3D10.1002%2Fanie.200503822" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cerasino, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hannon, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sletten, E.</span></span> <span> </span><span class="NLM_article-title">DNA three-way junction with a dinuclear iron (II) supramolecular helicate at the center: A NMR structural study</span>. <i>Inorg. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">6245</span>– <span class="NLM_lpage">6251</span>, <span class="refDoi"> DOI: 10.1021/ic062415c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ic062415c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2007&pages=6245-6251&author=L.+Cerasinoauthor=M.+J.+Hannonauthor=E.+Sletten&title=DNA+three-way+junction+with+a+dinuclear+iron+%28II%29+supramolecular+helicate+at+the+center%3A+A+NMR+structural+study&doi=10.1021%2Fic062415c"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fic062415c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fic062415c%26sid%3Dliteratum%253Aachs%26aulast%3DCerasino%26aufirst%3DL.%26aulast%3DHannon%26aufirst%3DM.%2BJ.%26aulast%3DSletten%26aufirst%3DE.%26atitle%3DDNA%2520three-way%2520junction%2520with%2520a%2520dinuclear%2520iron%2520%2528II%2529%2520supramolecular%2520helicate%2520at%2520the%2520center%253A%2520A%2520NMR%2520structural%2520study%26jtitle%3DInorg.%2520Chem.%26date%3D2007%26volume%3D46%26spage%3D6245%26epage%3D6251%26doi%3D10.1021%2Fic062415c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Malina, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hannon, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brabec, V.</span></span> <span> </span><span class="NLM_article-title">Recognition of DNA three-way junctions by metallosupramolecular cylinders: gel electrophoresis studies</span>. <i>Chem. Eur. J.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">3871</span>– <span class="NLM_lpage">3877</span>, <span class="refDoi"> DOI: 10.1002/chem.200700159</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=10.1002%2Fchem.200700159" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=17397023" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=1%3ACAS%3A528%3ADC%252BD2sXlsl2it70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2007&pages=3871-3877&author=J.+Malinaauthor=M.+J.+Hannonauthor=V.+Brabec&title=Recognition+of+DNA+three-way+junctions+by+metallosupramolecular+cylinders%3A+gel+electrophoresis+studies&doi=10.1002%2Fchem.200700159"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Recognition of DNA three-way junctions by metallosupramolecular cylinders: gel electrophoresis studies</span></div><div class="casAuthors">Malina, Jaroslav; Hannon, Michael J.; Brabec, Viktor</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry - A European Journal</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">3871-3877</span>CODEN:
                <span class="NLM_cas:coden">CEUJED</span>;
        ISSN:<span class="NLM_cas:issn">0947-6539</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The interaction of metallosupramol. cylinders with DNA three-way junctions has been studied by gel electrophoresis.  A recent X-ray crystal structure of a palindromic oligonucleotide forming part of a complex with such a cylinder revealed binding at the heart of a three-way junction structure.  The studies reported herein confirm that this is not solely an artifact of crystn. and reveal that this is a potentially very powerful new mode of DNA recognition with wide scope.  The cylinders are much more effective at stabilizing three-way junctions than simple magnesium dications or org. or metallo-org. tetracations, with the M cylinder enantiomer being more effective than P.  The recognition is not restricted to three-way junctions formed from palindromic DNA with a central AT step at the junction; non-palindromic three-way junctions and those with GC steps are also stabilized.  The cylinder is also revealed to stabilize other Y-shaped junctions, such as that formed at a fraying point in duplex DNA (for example, a replication fork), and other DNA three-way junction structures, such as those contg. unpaired nucleotides, perhaps by opening up this structure to access the central cavity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqb4tvdsmAUMbVg90H21EOLACvtfcHk0ljIxpzQRuVIfQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXlsl2it70%253D&md5=9c8795a6c23168be76227b551a5c6688</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1002%2Fchem.200700159&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fchem.200700159%26sid%3Dliteratum%253Aachs%26aulast%3DMalina%26aufirst%3DJ.%26aulast%3DHannon%26aufirst%3DM.%2BJ.%26aulast%3DBrabec%26aufirst%3DV.%26atitle%3DRecognition%2520of%2520DNA%2520three-way%2520junctions%2520by%2520metallosupramolecular%2520cylinders%253A%2520gel%2520electrophoresis%2520studies%26jtitle%3DChem.%2520Eur.%2520J.%26date%3D2007%26volume%3D13%26spage%3D3871%26epage%3D3877%26doi%3D10.1002%2Fchem.200700159" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hotze, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hodges, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayden, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanchez-Cano, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paines, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Male, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tse, M.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunce, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chipman, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hannon, M. J.</span></span> <span> </span><span class="NLM_article-title">Supramolecular iron cylinder with unprecedented DNA binding is a potent cytostatic and apoptotic agent without exhibiting genotoxicity</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">1258</span>– <span class="NLM_lpage">1267</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2008.10.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=10.1016%2Fj.chembiol.2008.10.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=19101470" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsFajt7rF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2008&pages=1258-1267&author=A.+C.+Hotzeauthor=N.+J.+Hodgesauthor=R.+E.+Haydenauthor=C.+Sanchez-Canoauthor=C.+Painesauthor=N.+Maleauthor=M.-K.+Tseauthor=C.+M.+Bunceauthor=J.+K.+Chipmanauthor=M.+J.+Hannon&title=Supramolecular+iron+cylinder+with+unprecedented+DNA+binding+is+a+potent+cytostatic+and+apoptotic+agent+without+exhibiting+genotoxicity&doi=10.1016%2Fj.chembiol.2008.10.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Supramolecular Iron Cylinder with Unprecedented DNA Binding Is a Potent Cytostatic and Apoptotic Agent without Exhibiting Genotoxicity</span></div><div class="casAuthors">Hotze, Anna C. G.; Hodges, Nikolas J.; Hayden, Rachel E.; Sanchez-Cano, Carlos; Paines, Christopher; Male, Natalia; Tse, Man-Kit; Bunce, Chris M.; Chipman, J. Kevin; Hannon, Michael J.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1258-1267</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The supramol. iron cylinder, [Fe2L3]Cl4 (L = C25H20N4), shows unprecedented DNA binding in vitro, inducing intramol. DNA coiling and also targeting Y-shaped DNA junctions.  Its effects on proliferation and survival in both tumor and normal cell lines were investigated.  Iron cylinder reduced mitochondrial activity of cultures with potency similar to cisplatin, inhibited the cell cycle, and increased cell death by apoptosis.  Assocd. with this, a lowering of the assocn. of propidium iodide (PI) with cellular DNA was obsd., consistent with an obsd. competitive displacement of PI from naked DNA by cylinders.  Importantly, and in contrast to existing anticancer drugs such as cisplatin, the iron cylinder [Fe2L3]4+ was not genotoxic.  In summary, the design of metal complexes such as [Fe2L3]4+ with potential anticancer properties in the absence of genotoxicity may represent a significant step toward therapeutic advancement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4a9ryNz_fp7Vg90H21EOLACvtfcHk0ljIxpzQRuVIfQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsFajt7rF&md5=765d8fc5f9abce744a01374f10b84270</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2008.10.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2008.10.016%26sid%3Dliteratum%253Aachs%26aulast%3DHotze%26aufirst%3DA.%2BC.%26aulast%3DHodges%26aufirst%3DN.%2BJ.%26aulast%3DHayden%26aufirst%3DR.%2BE.%26aulast%3DSanchez-Cano%26aufirst%3DC.%26aulast%3DPaines%26aufirst%3DC.%26aulast%3DMale%26aufirst%3DN.%26aulast%3DTse%26aufirst%3DM.-K.%26aulast%3DBunce%26aufirst%3DC.%2BM.%26aulast%3DChipman%26aufirst%3DJ.%2BK.%26aulast%3DHannon%26aufirst%3DM.%2BJ.%26atitle%3DSupramolecular%2520iron%2520cylinder%2520with%2520unprecedented%2520DNA%2520binding%2520is%2520a%2520potent%2520cytostatic%2520and%2520apoptotic%2520agent%2520without%2520exhibiting%2520genotoxicity%26jtitle%3DChem.%2520Biol.%26date%3D2008%26volume%3D15%26spage%3D1258%26epage%3D1267%26doi%3D10.1016%2Fj.chembiol.2008.10.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boer, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerckhoffs, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parajo, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pascu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Usón, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lincoln, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hannon, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coll, M.</span></span> <span> </span><span class="NLM_article-title">Self-Assembly of functionalizable two-component 3D DNA arrays through the induced formation of DNA three-way-Junction branch points by supramolecular cylinders</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">2336</span>– <span class="NLM_lpage">2339</span>, <span class="refDoi"> DOI: 10.1002/anie.200906742</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=10.1002%2Fanie.200906742" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjslGlur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2010&pages=2336-2339&author=D.+R.+Boerauthor=J.+M.+Kerckhoffsauthor=Y.+Parajoauthor=M.+Pascuauthor=I.+Us%C3%B3nauthor=P.+Lincolnauthor=M.+J.+Hannonauthor=M.+Coll&title=Self-Assembly+of+functionalizable+two-component+3D+DNA+arrays+through+the+induced+formation+of+DNA+three-way-Junction+branch+points+by+supramolecular+cylinders&doi=10.1002%2Fanie.200906742"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Self-Assembly of Functionalizable Two-Component 3D DNA Arrays through the Induced Formation of DNA Three-Way-Junction Branch Points by Supramolecular Cylinders</span></div><div class="casAuthors">Boer, D. Roeland; Kerckhoffs, Jessica M. C. A.; Parajo, Yolanda; Pascu, Mirela; Uson, Isabel; Lincoln, Per; Hannon, Michael J.; Coll, Miquel</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">2336-2339, S2336/1-S2336/13</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">We report the design of 3D DNA assemblies composed of short palindromic oligonucleotides, based on the use of three-way junctions (3WJs) as branching points.  The advantage of using 3WJs lies in the fact that they are specifically recognized by a class of helicate coordination compds. called supramol. cylinders.  These mols. can thus be introduced noncovalently into the 3WJ branching points of fully base paired 3WJs and 3WJs that contain unpaired bases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkQy3UMK5QwLVg90H21EOLACvtfcHk0ljIxpzQRuVIfQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjslGlur8%253D&md5=b910b76e2dd72f19265970f24f1490e5</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1002%2Fanie.200906742&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.200906742%26sid%3Dliteratum%253Aachs%26aulast%3DBoer%26aufirst%3DD.%2BR.%26aulast%3DKerckhoffs%26aufirst%3DJ.%2BM.%26aulast%3DParajo%26aufirst%3DY.%26aulast%3DPascu%26aufirst%3DM.%26aulast%3DUs%25C3%25B3n%26aufirst%3DI.%26aulast%3DLincoln%26aufirst%3DP.%26aulast%3DHannon%26aufirst%3DM.%2BJ.%26aulast%3DColl%26aufirst%3DM.%26atitle%3DSelf-Assembly%2520of%2520functionalizable%2520two-component%25203D%2520DNA%2520arrays%2520through%2520the%2520induced%2520formation%2520of%2520DNA%2520three-way-Junction%2520branch%2520points%2520by%2520supramolecular%2520cylinders%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2010%26volume%3D49%26spage%3D2336%26epage%3D2339%26doi%3D10.1002%2Fanie.200906742" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Barros, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chenoweth, D. M.</span></span> <span> </span><span class="NLM_article-title">Recognition of nucleic acid junctions using triptycene-based molecules</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">13746</span>– <span class="NLM_lpage">13750</span>, <span class="refDoi"> DOI: 10.1002/anie.201407061</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=10.1002%2Fanie.201407061" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitVCit73O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2014&pages=13746-13750&author=S.+A.+Barrosauthor=D.+M.+Chenoweth&title=Recognition+of+nucleic+acid+junctions+using+triptycene-based+molecules&doi=10.1002%2Fanie.201407061"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Recognition of Nucleic Acid Junctions Using Triptycene-Based Molecules</span></div><div class="casAuthors">Barros, Stephanie A.; Chenoweth, David M.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">50</span>),
    <span class="NLM_cas:pages">13746-13750</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The modulation of nucleic acids by small mols. is an essential process across the kingdoms of life.  Targeting nucleic acids with small mols. represents a significant challenge at the forefront of chem. biol.  Nucleic acid junctions are ubiquitous structural motifs in nature and in designed materials.  Herein, we describe a new class of structure-specific nucleic acid junction stabilizers based on a triptycene scaffold.  Triptycenes provide significant stabilization of DNA and RNA three-way junctions, providing a new scaffold for the development of nucleic acid junction binders with enhanced recognition properties.  Addnl., we report cytotoxicity and cell uptake data in two human ovarian carcinoma cell lines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrsalKqYAXH4LVg90H21EOLACvtfcHk0lixGigneh52QA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitVCit73O&md5=b3d59010e6f130eab638869353164ca2</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1002%2Fanie.201407061&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201407061%26sid%3Dliteratum%253Aachs%26aulast%3DBarros%26aufirst%3DS.%2BA.%26aulast%3DChenoweth%26aufirst%3DD.%2BM.%26atitle%3DRecognition%2520of%2520nucleic%2520acid%2520junctions%2520using%2520triptycene-based%2520molecules%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2014%26volume%3D53%26spage%3D13746%26epage%3D13750%26doi%3D10.1002%2Fanie.201407061" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Barros, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chenoweth, D. M.</span></span> <span> </span><span class="NLM_article-title">Triptycene-based small molecules modulate (CAG)·(CTG) repeat junctions</span>. <i>Chem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">4752</span>– <span class="NLM_lpage">4755</span>, <span class="refDoi"> DOI: 10.1039/C5SC01595B</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=10.1039%2FC5SC01595B" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=26366282" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVSgs7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=4752-4755&author=S.+A.+Barrosauthor=D.+M.+Chenoweth&title=Triptycene-based+small+molecules+modulate+%28CAG%29%C2%B7%28CTG%29+repeat+junctions&doi=10.1039%2FC5SC01595B"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Triptycene-based small molecules modulate (CAG)·(CTG) repeat junctions</span></div><div class="casAuthors">Barros, Stephanie A.; Chenoweth, David M.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Science</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">4752-4755</span>CODEN:
                <span class="NLM_cas:coden">CSHCCN</span>;
        ISSN:<span class="NLM_cas:issn">2041-6520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Nucleic acid three-way junctions (3WJs) play key roles in biol. processes such as nucleic acid replication in addn. to being implicated as dynamic transient intermediates in trinucleotide repeat sequences.  Structural modulation of specific nucleic acid junctions could allow for control of biol. processes and disease states at the nucleic acid level.  Trinucleotide repeat expansions are assocd. with several neurodegenerative diseases where dynamic slippage is thought to occur during replication, forming transient 3WJ intermediates with the complementary strand.  Here, we report triptycene-based mols. that bind to a d(CAG)·(CTG) repeat using a gel shift assay, fluorescence-quenching and CD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrjGX8rbzmZEbVg90H21EOLACvtfcHk0lixGigneh52QA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVSgs7rP&md5=140e24608fdb2513be74fc685ba0e3ac</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1039%2FC5SC01595B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC5SC01595B%26sid%3Dliteratum%253Aachs%26aulast%3DBarros%26aufirst%3DS.%2BA.%26aulast%3DChenoweth%26aufirst%3DD.%2BM.%26atitle%3DTriptycene-based%2520small%2520molecules%2520modulate%2520%2528CAG%2529%25C2%25B7%2528CTG%2529%2520repeat%2520junctions%26jtitle%3DChem.%2520Sci.%26date%3D2015%26volume%3D6%26spage%3D4752%26epage%3D4755%26doi%3D10.1039%2FC5SC01595B" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, P.</span></span> <span> </span><span class="NLM_article-title">Supramolecular recognition of three way junctions DNA by a cationic calix [3] carbazole</span>. <i>Chem. Eur. J.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">6087</span>– <span class="NLM_lpage">6093</span>, <span class="refDoi"> DOI: 10.1002/chem.201705564</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=10.1002%2Fchem.201705564" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=29315943" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=1%3ACAS%3A528%3ADC%252BC1cXis1Snsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2018&pages=6087-6093&author=Z.+Yangauthor=Y.+Chenauthor=G.+Liauthor=Z.+Tianauthor=L.+Zhaoauthor=X.+Wuauthor=Q.+Maauthor=M.+Liuauthor=P.+Yang&title=Supramolecular+recognition+of+three+way+junctions+DNA+by+a+cationic+calix+%5B3%5D+carbazole&doi=10.1002%2Fchem.201705564"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Supramolecular Recognition of Three Way Junction DNA by a Cationic Calix[3]carbazole</span></div><div class="casAuthors">Yang, Zhaozheng; Chen, Yan; Li, Gang; Tian, Zhangmin; Zhao, Liang; Wu, Xi; Ma, Qi; Liu, Mingzhe; Yang, Peng</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry - A European Journal</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">6087-6093</span>CODEN:
                <span class="NLM_cas:coden">CEUJED</span>;
        ISSN:<span class="NLM_cas:issn">0947-6539</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">DNA three-way junctions (TWJ-DNA) are intermediate structures in DNA replication and/or recombination.  They play very important roles in biol. processes, but more subtle functions are still unknown due partially to the lack of a fluorescent ligand.  In this study, a cationic calix[3]carbazole (I) has been synthesized and its properties of interacting with TWJ-DNA have been evaluated by UV/Vis and fluorescence spectroscopy, CD, gel electrophoresis, and 1H NMR studies.  The results show that I binds to the central hydrophobic cavity of TWJ-DNA.  Moreover, it could selectively bind to TWJ-DNA over duplex and quadruplex DNA.  Furthermore, I possesses the capability of serving as the TWJ-DNA probe as its trap-II excimer emission is turned on by TWJ-DNA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDqZgCDmxcBrVg90H21EOLACvtfcHk0lixGigneh52QA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXis1Snsbs%253D&md5=19162337173f4eedba50a642b871ab3e</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1002%2Fchem.201705564&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fchem.201705564%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DZ.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DG.%26aulast%3DTian%26aufirst%3DZ.%26aulast%3DZhao%26aufirst%3DL.%26aulast%3DWu%26aufirst%3DX.%26aulast%3DMa%26aufirst%3DQ.%26aulast%3DLiu%26aufirst%3DM.%26aulast%3DYang%26aufirst%3DP.%26atitle%3DSupramolecular%2520recognition%2520of%2520three%2520way%2520junctions%2520DNA%2520by%2520a%2520cationic%2520calix%2520%255B3%255D%2520carbazole%26jtitle%3DChem.%2520Eur.%2520J.%26date%3D2018%26volume%3D24%26spage%3D6087%26epage%3D6093%26doi%3D10.1002%2Fchem.201705564" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Van
Voorhis, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, J. H.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Open source drug discovery with the malaria box compound collection for neglected diseases and beyond</span>. <i>PLoS Pathog.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>, <span class="NLM_elocation-id">e1005763</span> <span class="refDoi"> DOI: 10.1371/journal.ppat.1005763</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=10.1371%2Fjournal.ppat.1005763" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=27467575" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvVelu7rL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&author=W.+C.+Van%0AVoorhisauthor=J.+H.+Adams&title=Open+source+drug+discovery+with+the+malaria+box+compound+collection+for+neglected+diseases+and+beyond&doi=10.1371%2Fjournal.ppat.1005763"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Open source drug discovery with the malaria box compound collection for neglected diseases and beyond</span></div><div class="casAuthors">Van Voorhis, Wesley C.; Adams, John H.; Adelfio, Roberto; Ahyong, Vida; Akabas, Myles H.; Alano, Pietro; Alday, Aintzane; Aleman Resto, Yesmalie; Alsibaee, Aishah; Alzualde, Ainhoa; Andrews, Katherine T.; Avery, Simon V.; Avery, Vicky M.; Ayong, Lawrence; Baker, Mark; Baker, Stephen; Ben Mamoun, Choukri; Bhatia, Sangeeta; Bickle, Quentin; Bounaadja, Lotfi; Bowling, Tana; Bosch, Jurgen; Boucher, Lauren E.; Boyom, Fabrice F.; Brea, Jose; Brennan, Marian; Burton, Audrey; Caffrey, Conor R.; Camarda, Grazia; Carrasquilla, Manuela; Carter, Dee; Cassera, Maria Belen; Cheng, Ken Chih-Chien; Chindaudomsate, Worathad; Chubb, Anthony; Colon, Beatrice L.; Colon-Lopez, Daisy D.; Corbett, Yolanda; Crowther, Gregory J.; Cowan, Noemi; D'Alessandro, Sarah; Dang, Na Le; Delves, Michael; De Risi, Joseph L.; Du, Alan Y.; Duffy, Sandra; El-Sayed, Shimaa Abd El-Salam; Ferdig, Michael T.; Fernandez Robledo, Jose A.; Fidock, David A.; Florent, Isabelle; Fokou, Patrick V. T.; Galstian, Ani; Gamo, Francisco Javier; Gokool, Suzanne; Gold, Ben; Golub, Todd; Goldgof, Gregory M.; Guha, Rajarshi; Guiguemde, W. Armand; Gural, Nil; Guy, R. Kiplin; Hansen, Michael A. E.; Hanson, Kirsten K.; Hemphill, Andrew; Hooft van Huijsduijnen, Rob; Horii, Takaaki; Horrocks, Paul; Hughes, Tyler B.; Huston, Christopher; Igarashi, Ikuo; Ingram-Sieber, Katrin; Itoe, Maurice A.; Jadhav, Ajit; Jensen, Amornrat Naranuntarat; Jensen, Laran T.; Jiang, Rays H. Y.; Kaiser, Annette; Keiser, Jennifer; Ketas, Thomas; Kicka, Sebastien; Kim, Sunyoung; Kirk, Kiaran; Kumar, Vidya P.; Kyle, Dennis E.; Lafuente, Maria Jose; Landfear, Scott; Lee, Nathan; Lee, Sukjun; Lehane, Adele M.; Li, Fengwu; Little, David; Liu, Liqiong; Llinas, Manuel; Loza, Maria I.; Lubar, Aristea; Lucantoni, Leonardo; Lucet, Isabelle; Maes, Louis; Mancama, Dalu; Mansour, Nuha R.; March, Sandra; McGowan, Sheena; Vera, Iset Medina; Meister, Stephan; Mercer, Luke; Mestres, Jordi; Mfopa, Alvine N.; Misra, Raj N.; Moon, Seunghyun; Moore, John P.; Morais Rodrigues da Costa, Francielly; Muller, Joachim; Muriana, Arantza; Hewitt, Stephen Nakazawa; Nare, Bakela; Nathan, Carl; Narraidoo, Nathalie; Nawaratna, Sujeevi; Ojo, Kayode K.; Ortiz, Diana; Panic, Gordana; Papadatos, George; Parapini, Silvia; Patra, Kailash; Pham, Ngoc; Prats, Sarah; Plouffe, David M.; Poulsen, Sally-Ann; Pradhan, Anupam; Quevedo, Celia; Quinn, Ronald J.; Rice, Christopher A.; Rizk, Mohamed Abdo; Ruecker, Andrea; St. Onge, Robert; Ferreira, Rafaela Salgado; Samra, Jasmeet; Robinett, Natalie G.; Schlecht, Ulrich; Schmitt, Marjorie; Villela, Filipe Silva; Silvestrini, Francesco; Sinden, Robert; Smith, Dennis A.; Soldati, Thierry; Spitzmuller, Andreas; Stamm, Serge Maximilian; Sullivan, David J.; Sullivan, William; Suresh, Sundari; Suzuki, Brian M.; Suzuki, Yo; Swamidass, S. Joshua; Taramelli, Donatella; Tchokouaha, Lauve R. Y.; Theron, Anjo; Thomas, David; Tonissen, Kathryn F.; Townson, Simon; Tripathi, Abhai K.; Trofimov, Valentin; Udenze, Kenneth O.; Ullah, Imran; Vallieres, Cindy; Vigil, Edgar; Vinetz, Joseph M.; Vinh, Phat Voong; Vu, Hoan; Watanabe, Nao-aki; Weatherby, Kate; White, Pamela M.; Wilks, Andrew F.; Winzeler, Elizabeth A.; Wojcik, Edward; Wree, Melanie; Wu, Wesley; Yokoyama, Naoaki; Zollo, Paul H. A.; Abla, Nada; Blasco, Benjamin; Burrows, Jeremy; Laleu, Benoit; Leroy, Didier; Spangenberg, Thomas; Wells, Timothy; Willis, Paul A.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Pathogens</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">e1005763/1-e1005763/23</span>CODEN:
                <span class="NLM_cas:coden">PPLACN</span>;
        ISSN:<span class="NLM_cas:issn">1553-7374</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">A major cause of the paucity of new starting points for drug discovery is the lack of interaction between academia and industry.  Much of the global resource in biol. is present in universities, whereas the focus of medicinal chem. is still largely within industry.  Open source drug discovery, with sharing of information, is clearly a first step towards overcoming this gap.  But the interface could esp. be bridged through a scale-up of open sharing of phys. compds., which would accelerate the finding of new starting points for drug discovery.  The Medicines for Malaria Venture Malaria Box is a collection of over 400 compds. representing families of structures identified in phenotypic screens of pharmaceutical and academic libraries against the Plasmodium falciparum malaria parasite.  The set has now been distributed to almost 200 research groups globally in the last two years, with the only stipulation that information from the screens is deposited in the public domain.  This paper reports for the first time on 236 screens that have been carried out against the Malaria Box and compares these results with 55 assays that were previously published, in a format that allows a meta-anal. of the combined dataset.  The combined biochem. and cellular assays presented here suggest mechanisms of action for 135 (34%) of the compds. active in killing multiple life-cycle stages of the malaria parasite, including asexual blood, liver, gametocyte, gametes and insect ookinete stages.  In addn., many compds. demonstrated activity against other pathogens, showing hits in assays with 16 protozoa, 7 helminths, 9 bacterial and mycobacterial species, the dengue fever mosquito vector, and the NCI60 human cancer cell line panel of 60 human tumor cell lines.  Toxicol., pharmacokinetic and metabolic properties were collected on all the compds., assisting in the selection of the most promising candidates for murine proof-of-concept expts. and medicinal chem. programs.  The data for all of these assays are presented and analyzed to show how outstanding leads for many indications can be selected.  These results reveal the immense potential for translating the dispersed expertise in biol. assays involving human pathogens into drug discovery starting points, by providing open access to new families of mols., and emphasize how a small addnl. investment made to help acquire and distribute compds., and sharing the data, can catalyze drug discovery for dozens of different indications.  Another lesson is that when multiple screens from different groups are run on the same library, results can be integrated quickly to select the most valuable starting points for subsequent medicinal chem. efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrDKNb7zLhgFrVg90H21EOLACvtfcHk0lhKUywM7udXRA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvVelu7rL&md5=0545e9836cf38a96b4ab000ca0a810bf</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1371%2Fjournal.ppat.1005763&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.ppat.1005763%26sid%3Dliteratum%253Aachs%26aulast%3DVan%2BVoorhis%26aufirst%3DW.%2BC.%26aulast%3DAdams%26aufirst%3DJ.%2BH.%26atitle%3DOpen%2520source%2520drug%2520discovery%2520with%2520the%2520malaria%2520box%2520compound%2520collection%2520for%2520neglected%2520diseases%2520and%2520beyond%26jtitle%3DPLoS%2520Pathog.%26date%3D2016%26volume%3D12%26doi%3D10.1371%2Fjournal.ppat.1005763" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wells, T. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willis, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burrows, J. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Huijsduijnen, R. H.</span></span> <span> </span><span class="NLM_article-title">Open data in drug discovery and development: lessons from malaria</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">661</span>– <span class="NLM_lpage">662</span>, <span class="refDoi"> DOI: 10.1038/nrd.2016.154</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=10.1038%2Fnrd.2016.154" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=27516171" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhtlehur7K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=661-662&author=T.+N.+Wellsauthor=P.+Willisauthor=J.+N.+Burrowsauthor=R.+H.+van+Huijsduijnen&title=Open+data+in+drug+discovery+and+development%3A+lessons+from+malaria&doi=10.1038%2Fnrd.2016.154"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Open data in drug discovery and development: lessons from malaria</span></div><div class="casAuthors">Wells, Timothy N. C.; Willis, Paul; Burrows, Jeremy N.; van Huijsduijnen, Rob Hooft</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">661-662</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">There is a growing consensus that drug discovery thrives in an open environment.  Here, we describe how the malaria community has embraced four levels of open data - open science, open innovation, open access and open source - to catalyze the development of new medicines, and consider principles that could enable open data approaches to be applied to other disease areas.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrYw2ky6KJ1CbVg90H21EOLACvtfcHk0lhKUywM7udXRA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhtlehur7K&md5=52a8b28339e33e744cad544ace5ad513</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.154&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.154%26sid%3Dliteratum%253Aachs%26aulast%3DWells%26aufirst%3DT.%2BN.%26aulast%3DWillis%26aufirst%3DP.%26aulast%3DBurrows%26aufirst%3DJ.%2BN.%26aulast%3Dvan%2BHuijsduijnen%26aufirst%3DR.%2BH.%26atitle%3DOpen%2520data%2520in%2520drug%2520discovery%2520and%2520development%253A%2520lessons%2520from%2520malaria%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2016%26volume%3D15%26spage%3D661%26epage%3D662%26doi%3D10.1038%2Fnrd.2016.154" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mahuteau-Betzer, F.</span></span> <span> </span><span class="NLM_article-title">The french national compound library: advances and future prospects</span>. <i>Med. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">417</span>– <span class="NLM_lpage">422</span>, <span class="refDoi"> DOI: 10.1051/medsci/20153104016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=10.1051%2Fmedsci%2F20153104016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=1%3ACAS%3A280%3ADC%252BC2Mfis1Cjuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2015&pages=417-422&author=F.+Mahuteau-Betzer&title=The+french+national+compound+library%3A+advances+and+future+prospects&doi=10.1051%2Fmedsci%2F20153104016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">The French National Compound Library: advances and future prospects</span></div><div class="casAuthors">Mahuteau-Betzer Florence</div><div class="citationInfo"><span class="NLM_cas:title">Medecine sciences : M/S</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">417-22</span>
        ISSN:<span class="NLM_cas:issn">0767-0974</span>.
    </div><div class="casAbstract">The French National Compound Library (Chimiotheque Nationale) has been created in 2003 and is the federation of local collections.  It contains more than 56 000 small molecules and natural compounds synthesised or isolated in different laboratories over the past years.  This explains the diversity of the collection.  The strength of this initiative is the ability to connect chemists and biologists for the development of hits.  This development involves the synthesis of analogues or/and chemical tools to find new targets.  These collaborations lead to the identification of new chemical probes.  These probes able to modulate a biological function are essential to study biological pathways.  They can also be useful for therapeutic applications.  This article will describe the major achievements and perspectives of the French Chemical Library.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTjzt2Wqi1XWxgd2Wuil24tfW6udTcc2ebHi0EH_cHGlrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2Mfis1Cjuw%253D%253D&md5=db4eaa72173176a4393c72ad8316365e</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1051%2Fmedsci%2F20153104016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1051%252Fmedsci%252F20153104016%26sid%3Dliteratum%253Aachs%26aulast%3DMahuteau-Betzer%26aufirst%3DF.%26atitle%3DThe%2520french%2520national%2520compound%2520library%253A%2520advances%2520and%2520future%2520prospects%26jtitle%3DMed.%2520Sci.%26date%3D2015%26volume%3D31%26spage%3D417%26epage%3D422%26doi%3D10.1051%2Fmedsci%2F20153104016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kadrmas, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ravin, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leontis, N. B.</span></span> <span> </span><span class="NLM_article-title">Relative stabilities of DNA three-way, four-way and five-way junctions (multi-helix junction loops): unpaired nucleotides can be stabilizing or destabilizing</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">2212</span>– <span class="NLM_lpage">2222</span>, <span class="refDoi"> DOI: 10.1093/nar/23.12.2212</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=10.1093%2Fnar%2F23.12.2212" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=1995&pages=2212-2222&author=J.+L.+Kadrmasauthor=A.+J.+Ravinauthor=N.+B.+Leontis&title=Relative+stabilities+of+DNA+three-way%2C+four-way+and+five-way+junctions+%28multi-helix+junction+loops%29%3A+unpaired+nucleotides+can+be+stabilizing+or+destabilizing&doi=10.1093%2Fnar%2F23.12.2212"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1093%2Fnar%2F23.12.2212&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252F23.12.2212%26sid%3Dliteratum%253Aachs%26aulast%3DKadrmas%26aufirst%3DJ.%2BL.%26aulast%3DRavin%26aufirst%3DA.%2BJ.%26aulast%3DLeontis%26aufirst%3DN.%2BB.%26atitle%3DRelative%2520stabilities%2520of%2520DNA%2520three-way%252C%2520four-way%2520and%2520five-way%2520junctions%2520%2528multi-helix%2520junction%2520loops%2529%253A%2520unpaired%2520nucleotides%2520can%2520be%2520stabilizing%2520or%2520destabilizing%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D1995%26volume%3D23%26spage%3D2212%26epage%3D2222%26doi%3D10.1093%2Fnar%2F23.12.2212" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Granzhan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monchaud, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saettel, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guédin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mergny, J.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teulade-Fichou, M.-P.</span></span> <span> </span><span class="NLM_article-title">“One ring to bind them all”—Part II: Identification of promising G-quadruplex ligands by screening of cyclophane-type macrocycles</span>. <i>J. Nucleic Acids</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>2010</i></span>, <span class="NLM_elocation-id">460561</span> <span class="refDoi"> DOI: 10.4061/2010/460561</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=10.4061%2F2010%2F460561" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=20725622" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2010&publication_year=2010&author=A.+Granzhanauthor=D.+Monchaudauthor=N.+Saettelauthor=A.+Gu%C3%A9dinauthor=J.-L.+Mergnyauthor=M.-P.+Teulade-Fichou&title=%E2%80%9COne+ring+to+bind+them+all%E2%80%9D%E2%80%94Part+II%3A+Identification+of+promising+G-quadruplex+ligands+by+screening+of+cyclophane-type+macrocycles&doi=10.4061%2F2010%2F460561"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.4061%2F2010%2F460561&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4061%252F2010%252F460561%26sid%3Dliteratum%253Aachs%26aulast%3DGranzhan%26aufirst%3DA.%26aulast%3DMonchaud%26aufirst%3DD.%26aulast%3DSaettel%26aufirst%3DN.%26aulast%3DGu%25C3%25A9din%26aufirst%3DA.%26aulast%3DMergny%26aufirst%3DJ.-L.%26aulast%3DTeulade-Fichou%26aufirst%3DM.-P.%26atitle%3D%25E2%2580%259COne%2520ring%2520to%2520bind%2520them%2520all%25E2%2580%259D%25E2%2580%2594Part%2520II%253A%2520Identification%2520of%2520promising%2520G-quadruplex%2520ligands%2520by%2520screening%2520of%2520cyclophane-type%2520macrocycles%26jtitle%3DJ.%2520Nucleic%2520Acids%26date%3D2010%26volume%3D2010%26doi%3D10.4061%2F2010%2F460561" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Freudenreich, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barry, N. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Süss-Fink, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Therrien, B.</span></span> <span> </span><span class="NLM_article-title">Permanent encapsulation or host–guest behavior of aromatic molecules in hexanuclear arene ruthenium prisms</span>. <i>Eur. J. Inorg. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>2010</i></span>,  <span class="NLM_fpage">2400</span>– <span class="NLM_lpage">2405</span>, <span class="refDoi"> DOI: 10.1002/ejic.201000191</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=10.1002%2Fejic.201000191" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2010&publication_year=2010&pages=2400-2405&author=J.+Freudenreichauthor=N.+P.+Barryauthor=G.+S%C3%BCss-Finkauthor=B.+Therrien&title=Permanent+encapsulation+or+host%E2%80%93guest+behavior+of+aromatic+molecules+in+hexanuclear+arene+ruthenium+prisms&doi=10.1002%2Fejic.201000191"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1002%2Fejic.201000191&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fejic.201000191%26sid%3Dliteratum%253Aachs%26aulast%3DFreudenreich%26aufirst%3DJ.%26aulast%3DBarry%26aufirst%3DN.%2BP.%26aulast%3DS%25C3%25BCss-Fink%26aufirst%3DG.%26aulast%3DTherrien%26aufirst%3DB.%26atitle%3DPermanent%2520encapsulation%2520or%2520host%25E2%2580%2593guest%2520behavior%2520of%2520aromatic%2520molecules%2520in%2520hexanuclear%2520arene%2520ruthenium%2520prisms%26jtitle%3DEur.%2520J.%2520Inorg.%2520Chem.%26date%3D2010%26volume%3D2010%26spage%3D2400%26epage%3D2405%26doi%3D10.1002%2Fejic.201000191" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gaschard, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nehzat, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheminel, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Therrien, B.</span></span> <span> </span><span class="NLM_article-title">Arene ruthenium metalla-assemblies with anthracene moieties for PDT applications</span>. <i>Inorganics</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">97</span>, <span class="refDoi"> DOI: 10.3390/inorganics6030097</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=10.3390%2Finorganics6030097" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=1%3ACAS%3A528%3ADC%252BC1MXktVWrtb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2018&pages=97&author=M.+Gaschardauthor=F.+Nehzatauthor=T.+Cheminelauthor=B.+Therrien&title=Arene+ruthenium+metalla-assemblies+with+anthracene+moieties+for+PDT+applications&doi=10.3390%2Finorganics6030097"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Arene ruthenium metalla-assemblies with anthracene moieties for PDT applications</span></div><div class="casAuthors">Gaschard, Marie; Nehzat, Farzaneh; Cheminel, Thomas; Therrien, Bruno</div><div class="citationInfo"><span class="NLM_cas:title">Inorganics</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">97/1-97/16</span>CODEN:
                <span class="NLM_cas:coden">INORCW</span>;
        ISSN:<span class="NLM_cas:issn">2304-6740</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">The synthesis and characterization of three metalla-rectangles of the general formula [Ru4(η6-p-cymene)4(μ4-clip)2(μ2-Lanthr)2][CF3SO3]4 (Lanthr: 9,10-bis(3,3'-ethynylpyridyl) anthracene; clip = oxa: oxalato; dobq: 2,5-dioxido-1,4-benzoquinonato; donq: 5,8-dioxido-1, 4-naphthoquinonato) are presented.  The mol. structure of the metalla-rectangle [Ru4(η6-p-cymene)4(μ4-oxa)2(μ2-Lanthr)2]4+ has been confirmed by the single-crystal X-ray structure anal. of [Ru4(η6-p-cymene)4(μ4-oxa)2(μ2-Lanthr)2][CF3SO3]4 · 4 acetone (A2 · 4 acetone), thus showing the anthracene moieties to be available for reaction with oxygen.  While the formation of the endoperoxide form of Lanthr was obsd. in soln. upon white light irradn., the same reaction does not occur when Lanthr is part of the metalla-assemblies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpS9yKZXENK8rVg90H21EOLACvtfcHk0ljxRr774fQyzQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXktVWrtb4%253D&md5=a3fab8ef9adff67a50d571bf802f8a63</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.3390%2Finorganics6030097&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Finorganics6030097%26sid%3Dliteratum%253Aachs%26aulast%3DGaschard%26aufirst%3DM.%26aulast%3DNehzat%26aufirst%3DF.%26aulast%3DCheminel%26aufirst%3DT.%26aulast%3DTherrien%26aufirst%3DB.%26atitle%3DArene%2520ruthenium%2520metalla-assemblies%2520with%2520anthracene%2520moieties%2520for%2520PDT%2520applications%26jtitle%3DInorganics%26date%3D2018%26volume%3D6%26spage%3D97%26doi%3D10.3390%2Finorganics6030097" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stefan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertrand, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richard, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le Gendre, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denat, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picquet, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monchaud, D.</span></span> <span> </span><span class="NLM_article-title">Assessing the differential affinity of small molecules for noncanonical DNA structures</span>. <i>ChemBioChem</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">1905</span>– <span class="NLM_lpage">1912</span>, <span class="refDoi"> DOI: 10.1002/cbic.201200396</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=10.1002%2Fcbic.201200396" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2012&pages=1905-1912&author=L.+Stefanauthor=B.+Bertrandauthor=P.+Richardauthor=P.+Le+Gendreauthor=F.+Denatauthor=M.+Picquetauthor=D.+Monchaud&title=Assessing+the+differential+affinity+of+small+molecules+for+noncanonical+DNA+structures&doi=10.1002%2Fcbic.201200396"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1002%2Fcbic.201200396&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbic.201200396%26sid%3Dliteratum%253Aachs%26aulast%3DStefan%26aufirst%3DL.%26aulast%3DBertrand%26aufirst%3DB.%26aulast%3DRichard%26aufirst%3DP.%26aulast%3DLe%2BGendre%26aufirst%3DP.%26aulast%3DDenat%26aufirst%3DF.%26aulast%3DPicquet%26aufirst%3DM.%26aulast%3DMonchaud%26aufirst%3DD.%26atitle%3DAssessing%2520the%2520differential%2520affinity%2520of%2520small%2520molecules%2520for%2520noncanonical%2520DNA%2520structures%26jtitle%3DChemBioChem%26date%3D2012%26volume%3D13%26spage%3D1905%26epage%3D1912%26doi%3D10.1002%2Fcbic.201200396" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Laguerre, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pirrotta, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Desbois, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gros, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lesniewska, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monchaud, D.</span></span> <span> </span><span class="NLM_article-title">Surface-promoted aggregation of amphiphilic quadruplex ligands drives their selectivity for alternative DNA structures</span>. <i>Org. Biomol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">7034</span>– <span class="NLM_lpage">7039</span>, <span class="refDoi"> DOI: 10.1039/C5OB00692A</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=10.1039%2FC5OB00692A" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2015&pages=7034-7039&author=A.+Laguerreauthor=Y.+Changauthor=M.+Pirrottaauthor=N.+Desboisauthor=C.+P.+Grosauthor=E.+Lesniewskaauthor=D.+Monchaud&title=Surface-promoted+aggregation+of+amphiphilic+quadruplex+ligands+drives+their+selectivity+for+alternative+DNA+structures&doi=10.1039%2FC5OB00692A"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1039%2FC5OB00692A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC5OB00692A%26sid%3Dliteratum%253Aachs%26aulast%3DLaguerre%26aufirst%3DA.%26aulast%3DChang%26aufirst%3DY.%26aulast%3DPirrotta%26aufirst%3DM.%26aulast%3DDesbois%26aufirst%3DN.%26aulast%3DGros%26aufirst%3DC.%2BP.%26aulast%3DLesniewska%26aufirst%3DE.%26aulast%3DMonchaud%26aufirst%3DD.%26atitle%3DSurface-promoted%2520aggregation%2520of%2520amphiphilic%2520quadruplex%2520ligands%2520drives%2520their%2520selectivity%2520for%2520alternative%2520DNA%2520structures%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2015%26volume%3D13%26spage%3D7034%26epage%3D7039%26doi%3D10.1039%2FC5OB00692A" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">de
Robillard, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Makni, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cattey, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrieu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Devillers, C. H.</span></span> <span> </span><span class="NLM_article-title">Towards sustainable synthesis of pyren-1-yl azoliums via electrochemical oxidative C–N coupling</span>. <i>Green. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">4669</span>– <span class="NLM_lpage">4679</span>, <span class="refDoi"> DOI: 10.1039/C5GC01142F</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=10.1039%2FC5GC01142F" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFWjur3K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2015&pages=4669-4679&author=G.+de%0ARobillardauthor=O.+Makniauthor=H.+Catteyauthor=J.+Andrieuauthor=C.+H.+Devillers&title=Towards+sustainable+synthesis+of+pyren-1-yl+azoliums+via+electrochemical+oxidative+C%E2%80%93N+coupling&doi=10.1039%2FC5GC01142F"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Towards sustainable synthesis of pyren-1-yl azoliums via electrochemical oxidative C-N coupling</span></div><div class="casAuthors">de Robillard, Guillaume; Makni, Oumayma; Cattey, Helene; Andrieu, Jacques; Devillers, Charles H.</div><div class="citationInfo"><span class="NLM_cas:title">Green Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4669-4679</span>CODEN:
                <span class="NLM_cas:coden">GRCHFJ</span>;
        ISSN:<span class="NLM_cas:issn">1463-9262</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Electrosynthesis of 1-methyl-3-(pyren-1-yl)-1H-imidazol-3-ium tetrafluoroborate via oxidative C-N coupling of pyrene with methylimidazole is optimized with the aim to reduce waste and simplify the exptl. setup.  Several parameters are tested such as cell configuration (no. of compartments), pyrene concn., amt. of nucleophile, electrosynthetic method (potentiostatic/galvanostatic), amt. of electrons, atm. (Ar/air), solvent quality and presence/absence of a supporting electrolyte.  The optimized conditions are successfully applied to the synthesis of 1-methyl-3-(pyren-1-yl)-1H-benzimidazol-3-ium tetrafluoroborate, 1-methyl-4-(pyren-1-yl)-1H-1,2,4-triazol-4-ium tetrafluoroborate and 3-(pyren-1-yl)-benzothiazol-3-ium tetrafluoroborate.  These four pyren-1-yl-azolium salts are characterized by NMR, MS, elemental anal., UV-Vis absorption and emission spectroscopy.  The X-ray crystallog. structures of 1-methyl-3-(pyren-1-yl)-1H-imidazol-3-ium tetrafluoroborate and 1-methyl-3-(pyren-1-yl)-1H-benzimidazol-3-ium tetrafluoroborate are presented.  The synthesis of the target compds. was achieved by a reaction of pyrene (i.e., polycyclic arom. hydrocarbon, arom. hydrocarbon) with 1-methyl-1H-imidazole, 1-methyl-1H-benzimidazole, 1-methyl-1H-1,2,4-triazole, benzothiazole in the presence of hydrogen tetrafluoroborate(1-) compd. with 1,1'-oxybis[ethane] (i.e., hydrogen tetrafluoroborate di-Et ether complex).  The title compds. thus formed included (pyrenyl)imidazolium salt, (pyrenyl)triazolium salt, (pyrenyl)benzothiazolium tetrafluoroborate, (pyrenyl)benzimidazolium tetrafluoroborate.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHDOL3_teuDLVg90H21EOLACvtfcHk0ljxRr774fQyzQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFWjur3K&md5=3c0774db029418879f7570370b458705</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1039%2FC5GC01142F&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC5GC01142F%26sid%3Dliteratum%253Aachs%26aulast%3Dde%2BRobillard%26aufirst%3DG.%26aulast%3DMakni%26aufirst%3DO.%26aulast%3DCattey%26aufirst%3DH.%26aulast%3DAndrieu%26aufirst%3DJ.%26aulast%3DDevillers%26aufirst%3DC.%2BH.%26atitle%3DTowards%2520sustainable%2520synthesis%2520of%2520pyren-1-yl%2520azoliums%2520via%2520electrochemical%2520oxidative%2520C%25E2%2580%2593N%2520coupling%26jtitle%3DGreen.%2520Chem.%26date%3D2015%26volume%3D17%26spage%3D4669%26epage%3D4679%26doi%3D10.1039%2FC5GC01142F" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gabelica, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Pauw, E.</span></span> <span> </span><span class="NLM_article-title">A simple method to determine electrospray response factors of noncovalent complexes</span>. <i>Anal. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>81</i></span>,  <span class="NLM_fpage">6708</span>– <span class="NLM_lpage">6715</span>, <span class="refDoi"> DOI: 10.1021/ac900785m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ac900785m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=1%3ACAS%3A528%3ADC%252BD1MXosVWntLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=81&publication_year=2009&pages=6708-6715&author=V.+Gabelicaauthor=F.+Rosuauthor=E.+De+Pauw&title=A+simple+method+to+determine+electrospray+response+factors+of+noncovalent+complexes&doi=10.1021%2Fac900785m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">A Simple Method to Determine Electrospray Response Factors of Noncovalent Complexes</span></div><div class="casAuthors">Gabelica, Valerie; Rosu, Frederic; De Pauw, Edwin</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Chemistry (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">81</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">6708-6715</span>CODEN:
                <span class="NLM_cas:coden">ANCHAM</span>;
        ISSN:<span class="NLM_cas:issn">0003-2700</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The quant. study of noncovalent complexes by electrospray mass spectrometry requires the detn. of the relative response of each species.  The method proposed here to det. the electrospray response factors is based on the use of (1) an internal std. and (2) the mass balance equation applied to one binding partner M, for which different complexes MxLy are detected in the electrospray mass spectra.  A set of expts. providing various ratios between the complexes (e.g., different ligand concns. in a titrn. expt. or different time points in a kinetics expt.) is used to generate a set of independent linear equations that can be solved using simple matrix algebra to find the response factors of each MxLy complex relative to that of the internal std.  The response factors can then be used to det. equil. dissocn. consts. or for the quant. monitoring of reaction kinetics.  The first is illustrated with a study of DNA-ligand complexes, where the authors show that neither minor groove binding nor intercalation dramatically affects the DNA response factor.  The second is illustrated with a study of the assocn. kinetics of the telomeric G-quadruplex dGGG(TTAGGG)3 with its complementary strand, where the response factors allow correcting for the relative response of the quadruplex and the long duplex and obtaining reproducible assocn. rate consts. independently of the source tuning potentials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrS4BvCgVhd2LVg90H21EOLACvtfcHk0lgr3F7jUMCdEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXosVWntLw%253D&md5=e66feaaa16b4086832ec960c3e8f7c8a</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fac900785m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fac900785m%26sid%3Dliteratum%253Aachs%26aulast%3DGabelica%26aufirst%3DV.%26aulast%3DRosu%26aufirst%3DF.%26aulast%3DDe%2BPauw%26aufirst%3DE.%26atitle%3DA%2520simple%2520method%2520to%2520determine%2520electrospray%2520response%2520factors%2520of%2520noncovalent%2520complexes%26jtitle%3DAnal.%2520Chem.%26date%3D2009%26volume%3D81%26spage%3D6708%26epage%3D6715%26doi%3D10.1021%2Fac900785m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rosu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gabelica, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Houssier, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Pauw, E.</span></span> <span> </span><span class="NLM_article-title">Determination of affinity, stoichiometry and sequence selectivity of minor groove binder complexes with double-stranded oligodeoxynucleotides by electrospray ionization mass spectrometry</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>30</i></span>, <span class="NLM_elocation-id">e82</span> <span class="refDoi"> DOI: 10.1093/nar/gnf081</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=10.1093%2Fnar%2Fgnf081" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2002&author=F.+Rosuauthor=V.+Gabelicaauthor=C.+Houssierauthor=E.+De+Pauw&title=Determination+of+affinity%2C+stoichiometry+and+sequence+selectivity+of+minor+groove+binder+complexes+with+double-stranded+oligodeoxynucleotides+by+electrospray+ionization+mass+spectrometry&doi=10.1093%2Fnar%2Fgnf081"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgnf081&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgnf081%26sid%3Dliteratum%253Aachs%26aulast%3DRosu%26aufirst%3DF.%26aulast%3DGabelica%26aufirst%3DV.%26aulast%3DHoussier%26aufirst%3DC.%26aulast%3DDe%2BPauw%26aufirst%3DE.%26atitle%3DDetermination%2520of%2520affinity%252C%2520stoichiometry%2520and%2520sequence%2520selectivity%2520of%2520minor%2520groove%2520binder%2520complexes%2520with%2520double-stranded%2520oligodeoxynucleotides%2520by%2520electrospray%2520ionization%2520mass%2520spectrometry%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2002%26volume%3D30%26doi%3D10.1093%2Fnar%2Fgnf081" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ragazzon, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garbett, N. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaires, J. B.</span></span> <span> </span><span class="NLM_article-title">Competition dialysis: a method for the study of structural selective nucleic acid binding</span>. <i>Methods</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">173</span>– <span class="NLM_lpage">182</span>, <span class="refDoi"> DOI: 10.1016/j.ymeth.2006.09.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=10.1016%2Fj.ymeth.2006.09.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=17472899" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=1%3ACAS%3A528%3ADC%252BD2sXkslCntrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2007&pages=173-182&author=P.+A.+Ragazzonauthor=N.+C.+Garbettauthor=J.+B.+Chaires&title=Competition+dialysis%3A+a+method+for+the+study+of+structural+selective+nucleic+acid+binding&doi=10.1016%2Fj.ymeth.2006.09.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Competition dialysis: a method for the study of structural selective nucleic acid binding</span></div><div class="casAuthors">Ragazzon, Patricia A.; Garbett, Nichola C.; Chaires, Jonathan B.</div><div class="citationInfo"><span class="NLM_cas:title">Methods (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">173-182</span>CODEN:
                <span class="NLM_cas:coden">MTHDE9</span>;
        ISSN:<span class="NLM_cas:issn">1046-2023</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Competition dialysis is a powerful new tool for the discovery of ligands that bind to nucleic acids with structural- or sequence-selectivity.  The method is based on firm thermodn. principles and is simple to implement.  In the competition dialysis expt., an array of nucleic acid structures and sequences is dialyzed against a common test ligand soln.  After equilibration, the amt. of ligand bound to each structure or sequence is detd. by absorbance or fluorescence measurements.  Since all structures and sequences are in equil. with the same free ligand concn., the amt. bound is directly proportional to the ligand binding affinity.  Competition dialysis thus provides a direct and quant. measure of selectivity, and unambiguously identifies which of the samples within the array are preferred by a particular ligand.  We describe here the third generation implementation of the method, in which competition dialysis was adapted for use in a 96-well plate format.  In this format, we have been able to greatly expand the array of nucleic acid structures studied, and now can routinely study the interactions of a ligand of interest with 46 different structures and sequences.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrrGii5sg50V7Vg90H21EOLACvtfcHk0lgr3F7jUMCdEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXkslCntrk%253D&md5=4154e0e912fb778ca37ae85ceada94d9</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1016%2Fj.ymeth.2006.09.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ymeth.2006.09.010%26sid%3Dliteratum%253Aachs%26aulast%3DRagazzon%26aufirst%3DP.%2BA.%26aulast%3DGarbett%26aufirst%3DN.%2BC.%26aulast%3DChaires%26aufirst%3DJ.%2BB.%26atitle%3DCompetition%2520dialysis%253A%2520a%2520method%2520for%2520the%2520study%2520of%2520structural%2520selective%2520nucleic%2520acid%2520binding%26jtitle%3DMethods%26date%3D2007%26volume%3D42%26spage%3D173%26epage%3D182%26doi%3D10.1016%2Fj.ymeth.2006.09.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vichai, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirtikara, K.</span></span> <span> </span><span class="NLM_article-title">Sulforhodamine B colorimetric assay for cytotoxicity screening</span>. <i>Nat. Protoc.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">1112</span>– <span class="NLM_lpage">1116</span>, <span class="refDoi"> DOI: 10.1038/nprot.2006.179</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=10.1038%2Fnprot.2006.179" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=17406391" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtFOitL3O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2006&pages=1112-1116&author=V.+Vichaiauthor=K.+Kirtikara&title=Sulforhodamine+B+colorimetric+assay+for+cytotoxicity+screening&doi=10.1038%2Fnprot.2006.179"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Sulforhodamine B colorimetric assay for cytotoxicity screening</span></div><div class="casAuthors">Vichai, Vanicha; Kirtikara, Kanyawim</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1112-1116</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The sulforhodamine B (SRB) assay is used for cell d. detn., based on the measurement of cellular protein content.  The method described here was optimized for the toxicity screening of compds. to adherent cells in a 96-well format.  After an incubation period, cell monolayers are fixed with 10% (wt/vol) trichloroacetic acid and stained for 30 min, after which the excess dye is removed by washing repeatedly with 1% (vol/vol) acetic acid.  The protein-bound dye is dissolved in 10 mM Tris base soln. for OD detn. at 510 nm using a microplate reader.  The results are linear over a 20-fold range of cell nos. and the sensitivity is comparable to those of fluorometric methods.  The method not only allows a large no. of samples to be tested within a few days, but also requires only simple equipment and inexpensive reagents.  The SRB assay is therefore an efficient and highly cost-effective method for screening.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGZ8pao808l7Vg90H21EOLACvtfcHk0lgr3F7jUMCdEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtFOitL3O&md5=7173035a1f9eb3f02dd7f1a947bca659</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2006.179&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2006.179%26sid%3Dliteratum%253Aachs%26aulast%3DVichai%26aufirst%3DV.%26aulast%3DKirtikara%26aufirst%3DK.%26atitle%3DSulforhodamine%2520B%2520colorimetric%2520assay%2520for%2520cytotoxicity%2520screening%26jtitle%3DNat.%2520Protoc.%26date%3D2006%26volume%3D1%26spage%3D1112%26epage%3D1116%26doi%3D10.1038%2Fnprot.2006.179" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Skehan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Storeng, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scudiero, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monks, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McMahon, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vistica, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warren, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bokesch, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kenney, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boyd, M. R.</span></span> <span> </span><span class="NLM_article-title">New colorimetric cytotoxicity assay for anticancer-drug screening</span>. <i>JNCI, J. Natl. Cancer Inst.</i> <span class="NLM_year" style="font-weight: bold;">1990</span>,  <span class="NLM_volume"><i>82</i></span>,  <span class="NLM_fpage">1107</span>– <span class="NLM_lpage">1112</span>, <span class="refDoi"> DOI: 10.1093/jnci/82.13.1107</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=10.1093%2Fjnci%2F82.13.1107" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=2359136" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=1%3ACAS%3A528%3ADyaK3cXltVylsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=82&publication_year=1990&pages=1107-1112&author=P.+Skehanauthor=R.+Storengauthor=D.+Scudieroauthor=A.+Monksauthor=J.+McMahonauthor=D.+Visticaauthor=J.+T.+Warrenauthor=H.+Bokeschauthor=S.+Kenneyauthor=M.+R.+Boyd&title=New+colorimetric+cytotoxicity+assay+for+anticancer-drug+screening&doi=10.1093%2Fjnci%2F82.13.1107"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">New colorimetric cytotoxicity assay for anticancer-drug screening</span></div><div class="casAuthors">Skehan, Philip; Storeng, Ritsa; Scudiero, Dominic; Monks, Anne; McMahon, James; Vistica, David; Warren, Jonathan T.; Bokesch, Heidi; Kenney, Susan; Boyd, Michael R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the National Cancer Institute</span>
        (<span class="NLM_cas:date">1990</span>),
    <span class="NLM_cas:volume">82</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">1107-12</span>CODEN:
                <span class="NLM_cas:coden">JNCIEQ</span>;
        ISSN:<span class="NLM_cas:issn">0027-8874</span>.
    </div><div class="casAbstract">A rapid, sensitive, and inexpensive method was developed for measuring the cellular protein content of adherent and suspension cultures in 96-well microtiter plates.  The method is suitable for ordinary lab. purposes and for very large-scale applications, such as the National Cancer Institute's disease-oriented in vitro anticancer-drug discovery screen, which requires the use of several million culture wells per yr.  Cultures fixed with trichloroacetic acid were stained for 30 min with 0.4% (wt/vol) sulforhodamine B (SRB) dissolved in 1% acetic acid.  Unbound dye was removed by four washes with 1% acetic acid, and protein-bound dye was extd. with 10 mM unbuffered Tris base [tris(hydroxymethyl)aminomethane] for detn. of optical d. in a computer-interfaced, 96-well microtiter plate reader.  The SRB assay results were linear with the no. of cells and with values for cellular protein measured by both the Lowry and Bradford assays at densities ranging from sparse subconfluence to multilayered supraconfluence.  The signal-to-noise ratio at 564 nm was approx. 1.5 with 1,000 cells per well.  The sensitivity of the SRB assay compared favorably with sensitivities of several fluorescence assays and was superior to those of both the Lowry and Bradford assays and to those of 20 other visible dyes.  The SRB assay provides a colorimetric end point that is nondestructive, indefinitely stable, and visible to the naked eye.  It provides a sensitive measure of drug-induced cytotoxicity, is useful in quantitating clonogenicity, and is well suited to high-vol., automated drug screening.  SRB fluoresces strongly with laser excitation at 488 nm and can be measured quant. at the single-cell level by static fluorescence cytometry.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrA1MIZ3rkcqLVg90H21EOLACvtfcHk0liC-Eadm_Ispg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3cXltVylsL8%253D&md5=d7bc1784a1f181842d2999efa7fa870c</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1093%2Fjnci%2F82.13.1107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjnci%252F82.13.1107%26sid%3Dliteratum%253Aachs%26aulast%3DSkehan%26aufirst%3DP.%26aulast%3DStoreng%26aufirst%3DR.%26aulast%3DScudiero%26aufirst%3DD.%26aulast%3DMonks%26aufirst%3DA.%26aulast%3DMcMahon%26aufirst%3DJ.%26aulast%3DVistica%26aufirst%3DD.%26aulast%3DWarren%26aufirst%3DJ.%2BT.%26aulast%3DBokesch%26aufirst%3DH.%26aulast%3DKenney%26aufirst%3DS.%26aulast%3DBoyd%26aufirst%3DM.%2BR.%26atitle%3DNew%2520colorimetric%2520cytotoxicity%2520assay%2520for%2520anticancer-drug%2520screening%26jtitle%3DJNCI%252C%2520J.%2520Natl.%2520Cancer%2520Inst.%26date%3D1990%26volume%3D82%26spage%3D1107%26epage%3D1112%26doi%3D10.1093%2Fjnci%2F82.13.1107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pommier, Y.</span></span> <span> </span><span class="NLM_article-title">Topoisomerase I inhibitors: camptothecins and beyond</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">789</span>– <span class="NLM_lpage">802</span>, <span class="refDoi"> DOI: 10.1038/nrc1977</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=10.1038%2Fnrc1977" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=16990856" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=1%3ACAS%3A528%3ADC%252BD28XpvVClsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2006&pages=789-802&author=Y.+Pommier&title=Topoisomerase+I+inhibitors%3A+camptothecins+and+beyond&doi=10.1038%2Fnrc1977"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Topoisomerase I inhibitors: camptothecins and beyond</span></div><div class="casAuthors">Pommier, Yves</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">789-802</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Nuclear DNA topoisomerase I (TOP1) is an essential human enzyme.  It is the only known target of the alkaloid camptothecin, from which the potent anticancer agents irinotecan and topotecan are derived.  As camptothecins bind at the interface of the TOP1-DNA complex, they represent a paradigm for interfacial inhibitors that reversibly trap macromol. complexes.  Several camptothecin and noncamptothecin derivs. are being developed to further increase antitumor activity and reduce side effects.  The mechanisms and mol. determinants of tumor response to TOP1 inhibitors are reviewed, and rational combinations of TOP1 inhibitors with other drugs are considered based on current knowledge of repair and checkpoint pathways that are assocd. with TOP1-mediated DNA damage.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrl5lxd3qfkwbVg90H21EOLACvtfcHk0liF10X2WztewA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XpvVClsr0%253D&md5=a437970077c0edbbab602b296fcd092a</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1038%2Fnrc1977&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc1977%26sid%3Dliteratum%253Aachs%26aulast%3DPommier%26aufirst%3DY.%26atitle%3DTopoisomerase%2520I%2520inhibitors%253A%2520camptothecins%2520and%2520beyond%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2006%26volume%3D6%26spage%3D789%26epage%3D802%26doi%3D10.1038%2Fnrc1977" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hahn, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Counter, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lundberg, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beijersbergen, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brooks, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinberg, R. A.</span></span> <span> </span><span class="NLM_article-title">Creation of human tumour cells with defined genetic elements</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>400</i></span>,  <span class="NLM_fpage">464</span>– <span class="NLM_lpage">468</span>, <span class="refDoi"> DOI: 10.1038/22780</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=10.1038%2F22780" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=10440377" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=1%3ACAS%3A528%3ADyaK1MXltVGmtLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=400&publication_year=1999&pages=464-468&author=W.+C.+Hahnauthor=C.+M.+Counterauthor=A.+S.+Lundbergauthor=R.+L.+Beijersbergenauthor=M.+W.+Brooksauthor=R.+A.+Weinberg&title=Creation+of+human+tumour+cells+with+defined+genetic+elements&doi=10.1038%2F22780"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Creation of human tumor cells with defined genetic elements</span></div><div class="casAuthors">Hahn, William C.; Counter, Christopher M.; Lundberg, Ante S.; Beijersbergen, Roderick L.; Brooks, Mary W.; Weinberg, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London)</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">400</span>
        (<span class="NLM_cas:issue">6743</span>),
    <span class="NLM_cas:pages">464-468</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Macmillan Magazines</span>)
        </div><div class="casAbstract">During malignant transformation, cancer cells acquire genetic mutations that override the normal mechanisms controlling cellular proliferation.  Primary rodent cells are efficiently converted into tumorigenic cells by the coexpression of cooperating oncogenes.  However, similar expts. with human cells have consistently failed to yield tumorigenic transformants, indicating a fundamental difference in the biol. of human and rodent cells.  The few reported successes in the creation of human tumor cells have depended on the use of chem. or phys. agents to achieve immortalization, the selection of rare, spontaneously arising immortalized cells, or the use of an entire viral genome.  The authors show here that the ectopic expression of the telomerase catalytic subunit (hTERT) in combination with two oncogenes (the simian virus 40 large-T oncoprotein and an oncogenic allele of H-ras) results in direct tumorigenic conversion of normal human epithelial and fibroblast cells.  These results demonstrate that disruption of the intracellular pathways regulated by large-T, oncogenic ras and telomerase suffices to create a human tumor cell.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrIWwnNvgmviLVg90H21EOLACvtfcHk0liF10X2WztewA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXltVGmtLc%253D&md5=09363db22f5dfe8fc4c52c3864491069</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1038%2F22780&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F22780%26sid%3Dliteratum%253Aachs%26aulast%3DHahn%26aufirst%3DW.%2BC.%26aulast%3DCounter%26aufirst%3DC.%2BM.%26aulast%3DLundberg%26aufirst%3DA.%2BS.%26aulast%3DBeijersbergen%26aufirst%3DR.%2BL.%26aulast%3DBrooks%26aufirst%3DM.%2BW.%26aulast%3DWeinberg%26aufirst%3DR.%2BA.%26atitle%3DCreation%2520of%2520human%2520tumour%2520cells%2520with%2520defined%2520genetic%2520elements%26jtitle%3DNature%26date%3D1999%26volume%3D400%26spage%3D464%26epage%3D468%26doi%3D10.1038%2F22780" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bonner, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Redon, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dickey, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakamura, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sedelnikova, O. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Solier, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pommier, Y.</span></span> <span> </span><span class="NLM_article-title">γH2AX and cancer</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">957</span>– <span class="NLM_lpage">967</span>, <span class="refDoi"> DOI: 10.1038/nrc2523</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=10.1038%2Fnrc2523" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=19005492" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsVWhu7rK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2008&pages=957-967&author=W.+M.+Bonnerauthor=C.+E.+Redonauthor=J.+S.+Dickeyauthor=A.+J.+Nakamuraauthor=O.+A.+Sedelnikovaauthor=S.+Solierauthor=Y.+Pommier&title=%CE%B3H2AX+and+cancer&doi=10.1038%2Fnrc2523"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">γH2AX and cancer</span></div><div class="casAuthors">Bonner, William M.; Redon, Christophe E.; Dickey, Jennifer S.; Nakamura, Asako J.; Sedelnikova, Olga A.; Solier, Stephanie; Pommier, Yves</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">957-967</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Histone H2AX phosphorylation on a serine four residues from the carboxyl terminus (producing γH2AX) is a sensitive marker for DNA double-strand breaks (DSBs).  DSBs may lead to cancer but, paradoxically, are also used to kill cancer cells.  Using γH2AX detection to det. the extent of DSB induction may help to detect precancerous cells, to stage cancers, to monitor the effectiveness of cancer therapies and to develop novel anticancer drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDvJ8vnkfC0bVg90H21EOLACvtfcHk0liF10X2WztewA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsVWhu7rK&md5=b0a83efe87f2823e79146ba8a2edf808</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1038%2Fnrc2523&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc2523%26sid%3Dliteratum%253Aachs%26aulast%3DBonner%26aufirst%3DW.%2BM.%26aulast%3DRedon%26aufirst%3DC.%2BE.%26aulast%3DDickey%26aufirst%3DJ.%2BS.%26aulast%3DNakamura%26aufirst%3DA.%2BJ.%26aulast%3DSedelnikova%26aufirst%3DO.%2BA.%26aulast%3DSolier%26aufirst%3DS.%26aulast%3DPommier%26aufirst%3DY.%26atitle%3D%25CE%25B3H2AX%2520and%2520cancer%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2008%26volume%3D8%26spage%3D957%26epage%3D967%26doi%3D10.1038%2Fnrc2523" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">O’Connor, M. J.</span></span> <span> </span><span class="NLM_article-title">Targeting the DNA damage response in cancer</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">547</span>– <span class="NLM_lpage">560</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2015.10.040</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=10.1016%2Fj.molcel.2015.10.040" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=26590714" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVyqtbvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2015&pages=547-560&author=M.+J.+O%E2%80%99Connor&title=Targeting+the+DNA+damage+response+in+cancer&doi=10.1016%2Fj.molcel.2015.10.040"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting the DNA Damage Response in Cancer</span></div><div class="casAuthors">O'Connor, Mark J.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">547-560</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">An underlying hallmark of cancers is their genomic instability, which is assocd. with a greater propensity to accumulate DNA damage.  Historical treatment of cancer by radiotherapy and DNA-damaging chemotherapy is based on this principle, yet it is accompanied by significant collateral damage to normal tissue and unwanted side effects.  Targeted therapy based on inhibiting the DNA damage response (DDR) in cancers offers the potential for a greater therapeutic window by tailoring treatment to patients with tumors lacking specific DDR functions.  The recent approval of olaparib (Lynparza), the poly (ADP-ribose) polymerase (PARP) inhibitor for treating tumors harboring BRCA1 or BRCA2 mutations, represents the first medicine based on this principle, exploiting an underlying cause of tumor formation that also represents an Achilles' heel.  This review highlights the different concepts behind targeting DDR in cancer and how this can provide significant opportunities for DDR-based therapies in the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWZ64MrshabrVg90H21EOLACvtfcHk0liF10X2WztewA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVyqtbvI&md5=b3f0d737d3e7dbf1f404b3f7658ab271</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2015.10.040&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2015.10.040%26sid%3Dliteratum%253Aachs%26aulast%3DO%25E2%2580%2599Connor%26aufirst%3DM.%2BJ.%26atitle%3DTargeting%2520the%2520DNA%2520damage%2520response%2520in%2520cancer%26jtitle%3DMol.%2520Cell%26date%3D2015%26volume%3D60%26spage%3D547%26epage%3D560%26doi%3D10.1016%2Fj.molcel.2015.10.040" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kotera, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poyer, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Granzhan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teulade-Fichou, M.-P.</span></span> <span> </span><span class="NLM_article-title">Efficient inhibition of human AP endonuclease 1 (APE1) via substrate masking by abasic site-binding macrocyclic ligands</span>. <i>Chem. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">15948</span>– <span class="NLM_lpage">15951</span>, <span class="refDoi"> DOI: 10.1039/C5CC06084B</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=10.1039%2FC5CC06084B" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2015&pages=15948-15951&author=N.+Koteraauthor=F.+Poyerauthor=A.+Granzhanauthor=M.-P.+Teulade-Fichou&title=Efficient+inhibition+of+human+AP+endonuclease+1+%28APE1%29+via+substrate+masking+by+abasic+site-binding+macrocyclic+ligands&doi=10.1039%2FC5CC06084B"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1039%2FC5CC06084B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC5CC06084B%26sid%3Dliteratum%253Aachs%26aulast%3DKotera%26aufirst%3DN.%26aulast%3DPoyer%26aufirst%3DF.%26aulast%3DGranzhan%26aufirst%3DA.%26aulast%3DTeulade-Fichou%26aufirst%3DM.-P.%26atitle%3DEfficient%2520inhibition%2520of%2520human%2520AP%2520endonuclease%25201%2520%2528APE1%2529%2520via%2520substrate%2520masking%2520by%2520abasic%2520site-binding%2520macrocyclic%2520ligands%26jtitle%3DChem.%2520Commun.%26date%3D2015%26volume%3D51%26spage%3D15948%26epage%3D15951%26doi%3D10.1039%2FC5CC06084B" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Caron, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duong, X. N. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guillot, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bombard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Granzhan, A.</span></span> <span> </span><span class="NLM_article-title">Interaction of functionalized naphthalenophanes with abasic sites in DNA: DNA cleavage, DNA cleavage inhibition, and formation of ligand–DNA adducts</span>. <i>Chem. Eur. J.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1949</span>– <span class="NLM_lpage">1962</span>, <span class="refDoi"> DOI: 10.1002/chem.201805555</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=10.1002%2Fchem.201805555" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=30508326" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsVKkurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2019&pages=1949-1962&author=C.+Caronauthor=X.+N.+T.+Duongauthor=R.+Guillotauthor=S.+Bombardauthor=A.+Granzhan&title=Interaction+of+functionalized+naphthalenophanes+with+abasic+sites+in+DNA%3A+DNA+cleavage%2C+DNA+cleavage+inhibition%2C+and+formation+of+ligand%E2%80%93DNA+adducts&doi=10.1002%2Fchem.201805555"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Interaction of Functionalized Naphthalenophanes with Abasic Sites in DNA: DNA Cleavage, DNA Cleavage Inhibition, and Formation of Ligand-DNA Adducts</span></div><div class="casAuthors">Caron, Coralie; Duong, Xuan N. T.; Guillot, Regis; Bombard, Sophie; Granzhan, Anton</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry - A European Journal</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1949-1962</span>CODEN:
                <span class="NLM_cas:coden">CEUJED</span>;
        ISSN:<span class="NLM_cas:issn">0947-6539</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Ligands interacting with abasic (AP) sites in DNA may generate roadblocks in base-excision DNA repair (BER) due to indirect inhibition of DNA repair enzymes (e.g., APE1) and/or formation of toxic byproducts, resulting from ligand-induced strand cleavage or covalent cross-links.  Herein, a series of 12 putative AP-site ligands, sharing the common naphthalenophane scaffold, but endowed with a variety of substituents, have been prepd. and systematically studied.  The results demonstrate that most naphthalenophanes bind to AP sites in DNA and inhibit the APE1-induced hydrolysis of the latter in vitro.  Remarkably, their APE1 inhibitory activity, as characterized by IC50 and KI values, can be directly related to their affinity and selectivity to AP sites, as assessed by means of fluorescence melting expts.  On the other hand, the mol. design of naphthalenophanes has a crucial influence on their intrinsic AP-site cleavage activity (i.e., ligand-catalyzed β- and β,δ-elimination reactions at the AP site), as illustrated by the compds. either having an exceptionally high AP-site cleavage activity (e.g., 2,7-BisNP-S, 125-fold more efficacious than spermine) or being totally devoid of this activity (four compds.).  Finally, the unprecedented formation of a stable covalent DNA adduct upon reaction of one ligand (2,7-BisNP-NH) with its own product of the AP-site cleavage is revealed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqsl52szWsAmLVg90H21EOLACvtfcHk0lihWY8Dut_IMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsVKkurw%253D&md5=9d8b88350a376d92f8eac95e54ce598e</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1002%2Fchem.201805555&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fchem.201805555%26sid%3Dliteratum%253Aachs%26aulast%3DCaron%26aufirst%3DC.%26aulast%3DDuong%26aufirst%3DX.%2BN.%2BT.%26aulast%3DGuillot%26aufirst%3DR.%26aulast%3DBombard%26aufirst%3DS.%26aulast%3DGranzhan%26aufirst%3DA.%26atitle%3DInteraction%2520of%2520functionalized%2520naphthalenophanes%2520with%2520abasic%2520sites%2520in%2520DNA%253A%2520DNA%2520cleavage%252C%2520DNA%2520cleavage%2520inhibition%252C%2520and%2520formation%2520of%2520ligand%25E2%2580%2593DNA%2520adducts%26jtitle%3DChem.%2520Eur.%2520J.%26date%3D2019%26volume%3D25%26spage%3D1949%26epage%3D1962%26doi%3D10.1002%2Fchem.201805555" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i26"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b01978">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_50714"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.8b01978">10.1021/acs.jmedchem.8b01978</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">ESI-MS evaluations of <b>451</b>, <b>459</b>, and <b>471</b> with both TWJ-DNA and ds-DNA (Figures S1–S3), SRB assays performed with MCF7 and MDA-MB-231 cells and camptothecin, and with BJ-hTERT and <b>451</b>, <b>459</b>, <b>471</b>, and camptothecin (Figure S4) and immunodetection experiments performed with MCF7 cells and <b>471</b> and camptothecin (Figure S5) (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b01978/suppl_file/jm8b01978_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Complete TWJ-screen results and the molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b01978/suppl_file/jm8b01978_si_002.csv">CSV</a>)</p></li><li><p class="inline">Top ligand selection (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b01978/suppl_file/jm8b01978_si_003.xlsx">XLSX</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b01978/suppl_file/jm8b01978_si_001.pdf">jm8b01978_si_001.pdf (1.6 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b01978/suppl_file/jm8b01978_si_002.csv">jm8b01978_si_002.csv (3.19 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b01978/suppl_file/jm8b01978_si_003.xlsx">jm8b01978_si_003.xlsx (321.41 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.8b01978&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.8b01978%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2019.62.issue-9" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.8b01978" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                3MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679953b768a93d8e","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
