
<html lang="en"     class="pb-page"  data-request-id="8ad65698-6a6b-4548-9cbe-5536c2a0ad5d"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";article:article:10.1021/jm300687e;requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;issue:issue:10.1021/jmcmar.2012.55.issue-14;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Directory of Useful Decoys, Enhanced (DUD-E): Better Ligands and Decoys for Better Benchmarking" /></meta><meta name="dc.Creator" content="Michael M.  Mysinger" /></meta><meta name="dc.Creator" content="Michael  Carchia" /></meta><meta name="dc.Creator" content="John. J.  Irwin" /></meta><meta name="dc.Creator" content="Brian K.  Shoichet" /></meta><meta name="dc.Description" content="A key metric to assess molecular docking remains ligand enrichment against challenging decoys. Whereas the directory of useful decoys (DUD) has been widely used, clear areas for optimization have e..." /></meta><meta name="Description" content="A key metric to assess molecular docking remains ligand enrichment against challenging decoys. Whereas the directory of useful decoys (DUD) has been widely used, clear areas for optimization have e..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="July 5, 2012" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/jm300687e" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2012 American Chemical Society" /></meta><meta name="citation_fulltext_world_readable" content=""/></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Fjm300687e" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Fjm300687e" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Fjm300687e" /></link>
        
    
    

<title>Directory of Useful Decoys, Enhanced (DUD-E): Better Ligands and Decoys for Better Benchmarking | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/jm300687e" /></meta><meta property="og:title" content="Directory of Useful Decoys, Enhanced (DUD-E): Better Ligands and Decoys for Better Benchmarking" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-14/jm300687e/production/images/large/jm-2012-00687e_0008.jpeg" /></meta><meta property="og:description" content="A key metric to assess molecular docking remains ligand enrichment against challenging decoys. Whereas the directory of useful decoys (DUD) has been widely used, clear areas for optimization have emerged. Here we describe an improved benchmarking set that includes more diverse targets such as GPCRs and ion channels, totaling 102 proteins with 22886 clustered ligands drawn from ChEMBL, each with 50 property-matched decoys drawn from ZINC. To ensure chemotype diversity, we cluster each target’s ligands by their Bemis–Murcko atomic frameworks. We add net charge to the matched physicochemical properties and include only the most dissimilar decoys, by topology, from the ligands. An online automated tool (http://decoys.docking.org) generates these improved matched decoys for user-supplied ligands. We test this data set by docking all 102 targets, using the results to improve the balance between ligand desolvation and electrostatics in DOCK 3.6. The complete DUD-E benchmarking set is freely available at http://dude.docking.org." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/jm300687e"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/jm300687e">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-8/abseba.2021.7.issue-8/20210809/abseba.2021.7.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-15/accacs.2021.11.issue-15/20210806/accacs.2021.11.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-31/acsodf.2021.6.issue-31/20210810/acsodf.2021.6.issue-31.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-31/ascecg.2021.9.issue-31/20210809/ascecg.2021.9.issue-31.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-31/ancham.2021.93.issue-31/20210810/ancham.2021.93.issue-31.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-31/bichaw.2021.60.issue-31/20210810/bichaw.2021.60.issue-31.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-8/bomaf6.2021.22.issue-8/20210809/bomaf6.2021.22.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-29/cgeabj.2021.99.issue-29/20210809-01/cgeabj.2021.99.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/cmatex/largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-15/enfuem.2021.35.issue-15/20210805/enfuem.2021.35.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-8/estlcu.2021.8.issue-8/20210810/estlcu.2021.8.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-8/jceda8.2021.98.issue-8/20210810/jceda8.2021.98.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/jctcce/largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-15/joceah.2021.86.issue-15/20210806/joceah.2021.86.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-30/jpcafh.2021.125.issue-30/20210805/jpcafh.2021.125.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-30/jpcbfk.2021.125.issue-30/20210805/jpcbfk.2021.125.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-30/jpccck.2021.125.issue-30/20210805/jpccck.2021.125.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-30/jpclcd.2021.12.issue-30/20210805/jpclcd.2021.12.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-8/jprobs.2021.20.issue-8/20210806/jprobs.2021.20.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-31/langd5.2021.37.issue-31/20210810/langd5.2021.37.issue-31.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-15/mamobx.2021.54.issue-15/20210810/mamobx.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-15/orlef7.2021.23.issue-15/20210806/orlef7.2021.23.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-15/orgnd7.2021.40.issue-15/20210809/orgnd7.2021.40.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Fjm300687e&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Fjm300687e&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/jm300687e&amp;href=/doi/10.1021/jm300687e" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2012</span><span class="cit-fg-volume">, 55</span><span class="cit-fg-issue">, 14</span><span class="cit-fg-pageRange">, 6582-6594</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/55/14" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/jm300658n" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/jm300714p" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Directory of Useful Decoys, Enhanced (DUD-E): Better Ligands and Decoys for Better Benchmarking</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Michael+M.++Mysinger">Michael M. Mysinger</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Michael++Carchia">Michael Carchia</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=John.+J.++Irwin">John. J. Irwin</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Brian+K.++Shoichet">Brian K. Shoichet</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-symbol">†</span> <span class="aff-text">Department of Pharmaceutical Chemistry, University of California San Francisco, San Francisco, California 94158-2330, United States</span></div><div class="corresp-info"><strong>*</strong>For J.J.I.: phone, (415) 514-4127; E-mail, <a href="/cdn-cgi/l/email-protection#6b0101022b080c07451e08180d450e0f1e"><span class="__cf_email__" data-cfemail="5f3535361f3c3833712a3c2c39713a3b2a">[email protected]</span></a>. For B.K.S.: phone, (415) 514-4126; E-mail, <a href="/cdn-cgi/l/email-protection#a8dbc0c7c1cbc0cddce8cbcfc486ddcbdbce86cdccdd"><span class="__cf_email__" data-cfemail="65160d0a0c060d0011250602094b100616034b000110">[email protected]</span></a>. Address: John J. Irwin or Brian K. Shoichet, Department of Pharmaceutical Chemistry, University of California San Francisco, 1700 Fourth Street, Box 2550, San Francisco, CA 94158-2330.</div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Fjm300687e&amp;href=/doi/10.1021%2Fjm300687e" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2012</span></span><span class="cit-volume">, 55</span><span class="cit-issue">, 14</span><span class="cit-pageRange">, 6582–6594</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">June 20, 2012</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>16 May 2012</li><li><span class="item_label"><b>Published</b> online</span>5 July 2012</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 26 July 2012</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/jm300687e" title="DOI URL">https://doi.org/10.1021/jm300687e</a></div><div class="article_header-article-copyright"><strong>Copyright © 2012 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/page/rightslinkno.jsp">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"><a href="/page/policy/authorchoice_termsofuse.html"><img src="/pb-assets/images/programsLogos/OA_Icons_Ac-1612971054130.svg" alt="ACS AuthorChoice" /></img><span>ACS AuthorChoice</span></a></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/jm300687e"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">16519</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">819</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/jm300687e" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Directory of Useful Decoys, Enhanced (DUD-E): Better Ligands and Decoys for Better Benchmarking&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Michael&quot;,&quot;last_name&quot;:&quot;M. Mysinger&quot;},{&quot;first_name&quot;:&quot;Michael&quot;,&quot;last_name&quot;:&quot;Carchia&quot;},{&quot;first_name&quot;:&quot;John.&quot;,&quot;last_name&quot;:&quot;J. Irwin&quot;},{&quot;first_name&quot;:&quot;Brian&quot;,&quot;last_name&quot;:&quot;K. Shoichet&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2012&quot;,&quot;month&quot;:&quot;07&quot;,&quot;day&quot;:&quot;05&quot;,&quot;issue&quot;:&quot;14&quot;,&quot;volume&quot;:&quot;55&quot;,&quot;pages&quot;:&quot;6582-6594&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/jm300687e&quot;},&quot;abstract&quot;:&quot;A key metric to assess molecular docking remains ligand enrichment against challenging decoys. Whereas the directory of useful decoys (DUD) has been widely used, clear areas for optimization have emerged. Here we describe an improved benchmarking set that includes more diverse targets such as GPCRs and ion channels, totaling 102 proteins with 22886 clustered ligands drawn from ChEMBL, each with 50 property-matched decoys drawn from ZINC. To ensure chemotype diversity, we cluster each target’s ligands by their Bemis–Murcko atomic frameworks. We add net charge to the matched physicochemical properties and include only the most dissimilar decoys, by topology, from the ligands. An online automated tool (http://decoys.docking.org) generates these improved matched decoys for user-supplied ligands. We test this data set by docking all 102 targets, using the results to improve the balance between ligand desolvation and electrostatics in DOCK 3.6. The complete DUD-E benchmarking set is freely available at http://dude.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Fjm300687e&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm300687e" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm300687e&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=jm300687e" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm300687e&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=jm300687e" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Fjm300687e&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm300687e&amp;href=/doi/10.1021/jm300687e" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/jm300687e" /></input><a href="/doi/pdf/10.1021/jm300687e" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (5 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/jm300687e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Fjm300687e%26sid%3Dliteratum%253Aachs%26pmid%3D22716043%26genre%3Darticle%26aulast%3DMysinger%26date%3D2012%26atitle%3DDirectory%2Bof%2BUseful%2BDecoys%252C%2BEnhanced%2B%2528DUD-E%2529%253A%2BBetter%2BLigands%2Band%2BDecoys%2Bfor%2BBetter%2BBenchmarking%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D55%26issue%3D14%26spage%3D6582%26epage%3D6594%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (4)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=292366" title="Ligands">Ligands</a>,</li><li><a href="/action/doSearch?ConceptID=292331" title="Benchmarking">Benchmarking</a>,</li><li><a href="/action/doSearch?ConceptID=291445" title="Chemical structure">Chemical structure</a>,</li><li><a href="/action/doSearch?ConceptID=292186" title="Receptors">Receptors</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/55/14" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-14/jmcmar.2012.55.issue-14/production/jmcmar.2012.55.issue-14.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-14/jm300687e/production/images/medium/jm-2012-00687e_0008.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-14/jm300687e/production/images/large/jm-2012-00687e_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm300687e&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">A key metric to assess molecular docking remains ligand enrichment against challenging decoys. Whereas the directory of useful decoys (DUD) has been widely used, clear areas for optimization have emerged. Here we describe an improved benchmarking set that includes more diverse targets such as GPCRs and ion channels, totaling 102 proteins with 22886 clustered ligands drawn from ChEMBL, each with 50 property-matched decoys drawn from ZINC. To ensure chemotype diversity, we cluster each target’s ligands by their Bemis–Murcko atomic frameworks. We add net charge to the matched physicochemical properties and include only the most dissimilar decoys, by topology, from the ligands. An online automated tool (<a href="http://decoys.docking.org" class="extLink">http://decoys.docking.org</a>) generates these improved matched decoys for user-supplied ligands. We test this data set by docking all 102 targets, using the results to improve the balance between ligand desolvation and electrostatics in DOCK 3.6. The complete DUD-E benchmarking set is freely available at <a href="http://dude.docking.org" class="extLink">http://dude.docking.org</a>.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_40151" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_40151" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">While molecular docking screens routinely leverage protein structure to discover new ligands,<a onclick="showRef(event, 'ref1 ref2 ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3 ref4">(1-4)</a> quantitative assessment of their performance remains problematic.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> Although prospective assessment of docking performance is irreplaceable,<a onclick="showRef(event, 'ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref6 ref7">(6, 7)</a> it is both time-consuming and expensive. Because a general correlation between docking scores and affinities is beyond current methods,<a onclick="showRef(event, 'ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref8 ref9">(8, 9)</a> the field relies on ligand enrichment in docking hit lists to evaluate retrospective performance.<a onclick="showRef(event, 'ref10 ref11 ref12 ref13 ref14'); return false;" href="javascript:void(0);" class="ref ref10 ref11 ref12 ref13 ref14">(10-14)</a> “Enrichment” measures how known ligands rank versus a background of decoy molecules and so depends not only on the nature of the ligands but also on the background decoys. Thus to compare docking enrichments, a benchmarking set of ligands and decoys is needed.</div><div class="NLM_p">The original Directory of Useful Decoys (DUD) was designed to meet this benchmarking need while controlling for decoy bias on enrichment.<a onclick="showRef(event, 'ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref15 ref16">(15, 16)</a> Given a random drug-like set of decoys, Verdonk et al. showed that targets which bind high molecular weight ligands naturally get higher enrichments due to correlation between larger molecules and better docking scores.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> In contrast, actual ligand binding affinities correlate with molecular size only for very small molecules.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> Unable to separate the true correlations of simple molecular properties that aid prospective ligand discovery from the artifical correlations that arise from biases, it is informative to ask what value molecular docking adds beyond these properties. To this end, DUD decoys are matched to the physical chemistry of ligands on a target-by-target basis: by the properties of molecular weight, calculated logP, number of rotatable bonds, and hydrogen bond donors and acceptors. To fulfill their role as negative controls, decoys should not actually bind, so DUD used 2-D similarity fingerprints to minimize the topological similarity between decoys and ligands. In short, DUD decoys were chosen to resemble ligands physically and so be challenging for docking but at the same time be topologically dissimilar to minimize the likelihood of actual binding.</div><div class="NLM_p">Through intense use,<a onclick="showRef(event, 'ref19 ref20 ref21 ref22 ref23 ref24 ref25 ref26'); return false;" href="javascript:void(0);" class="ref ref19 ref20 ref21 ref22 ref23 ref24 ref25 ref26">(19-26)</a> weaknesses in the original DUD set have appeared in both the ligands and decoys. Good and Oprea noted that a handful of chemotypes dominate many ligand sets, allowing high ranks for one scaffold to cause good overall enrichment.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> One way to circumvent this problem is using chemotype retrieval metrics,<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> but another is to remove the “analogue bias” from the database by clustering on ligand scaffolds. After clustering the 40 targets, Good’s subset of DUD contains only 13 targets with over 15 ligands, indicating a need for more targets with more ligands. Another important goal is to increase target diversity, for example, by adding membrane domain proteins, none of which are represented in DUD.</div><div class="NLM_p">As there were weaknesses in the DUD ligands, this was also true of the decoys. Several investigators<a onclick="showRef(event, 'ref29 ref30 ref31'); return false;" href="javascript:void(0);" class="ref ref29 ref30 ref31">(29-31)</a> observed that despite property matching on logP, net formal charge is still imbalanced in DUD; 42% of all ligands are charged versus only 15% of decoys. Property matching of decoys to ligands could also be tightened by choosing decoys more embedded in ligand property space.<a onclick="showRef(event, 'ref32 ref33'); return false;" href="javascript:void(0);" class="ref ref32 ref33">(32, 33)</a> Despite a 2-D chemical dissimilarity filter to prevent decoys from being active, some original DUD decoys still appear to bind, and these false decoys artificially reduce docking enrichment.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> Addressing both false decoys and decoy property embedding, Vogel et al. released DEKOIS for the original 40 DUD targets. Gatica and Cavasotto generated ligand and decoy sets for 147 G protein-coupled receptors (GPCRs) while adding net charge to property matching.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> Very recently, a python GUI application was announced to generate property-matched decoys.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> By ignoring synthetic feasibility, Wallach and Lilien generate virtual decoy sets for the original DUD targets with tighter property-matching.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> Instead of generating computational decoys, the MUV set selects decoys for 17 targets that were negative in public high-throughput screens.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> Instead of generating decoys at all, REPROVIS-DB assembles ligand and database data from earlier successful virtual screens which are deemed reproducible.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a></div><div class="NLM_p last">Here we describe a new version of DUD that addresses these liabilities and develops new functionality. By drawing on ChEMBL09,<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> each DUD-Enhanced (DUD-E) ligand has a measured affinity supported by a literature reference. Though ligands are now typically clustered by Bemis–Murcko atomic frameworks<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> to reduce chemotype bias, there are still on average 224 ligands per target. The target list is expanded from 40 to 102, favoring targets with many ligands and multiple<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> structures. The additions include several drug relevant membrane proteins: five GPCRs, two ion channels, and two cytochrome P450s. Meanwhile, false decoys are reduced by more stringent filtering of topological dissimilarity. Where possible, measured experimental decoys are included. Finally, we consider how DUD-E performs as a benchmark versus the original DUD and explore its use as a tool for evaluating and optimizing molecular docking.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_77274" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_77274" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The ideal target for a benchmarking set would be well studied, with many measured ligand affinities and multiple, diverse cocrystal ligand structures. To this end, the enhanced DUD database (DUD-E) is largely based on the intersection of ChEMBL,<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> for ligand annotations and affinities, and the RCSB PDB,<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> for structures. As we sought targets to enlarge the set, the 40 original DUD targets were first priority, 38 of which we included. Platelet-derived growth factor receptor β was dropped, as it was a homology model. Estrogen receptor α (ESR1) is a single target in DUD-E, whereas it was split into agonists and antagonists previously. To enlarge the benchmarking set, we used three main criteria. First, we favored new target classes with pharmacological precedence. Second, we sought targets with many ligands and crystal structures, as they likely reflect a combination of target relevance and ease of study. Third, we preferred targets that could modestly enrich known ligands using fully automated docking, as these may be both easy to prepare and amenable to docking. Conversely, targets with mostly covalent ligands were deprioritized.</div><div class="NLM_p">DUD-E targets are defined by their UniProt<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> gene prefix, with data from each species being combined into a single data set. While ChEMBL annotates ligands to a particular UniProt accession code, the ligand overlap between orthologous targets is surprisingly small. For example, among 1555 unique ligands with affinities below 1 μM for the human dopamine D3 receptor and 744 ligands for the rat orthologue, only 85 ligands are in both sets. These two orthologues share 97% trans-membrane sequence identity (79% overall), so this low overlap suggests to us that ChEMBL ligand annotations are sparse and do not typically reflect species specificity. Therefore, we pooled the data for all species, defining a DUD-E target as a UniProt gene prefix (such as DRD3), and not the full gene_species pair (such as DRD3_HUMAN or P35462).</div><div class="NLM_p">The 102 targets span diverse protein categories, including 26 kinases, 15 proteases, 11 nuclear receptors, five GPCRs, two ion channels, two cytochrome P450s, 36 other enzymes, and five miscellaneous proteins (Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>). Altogether 66695 raw ligands, defined as those with annotated affinities better than 1 μM to their target, molecular weights less than 600 and fewer than 20 rotatable bonds were extracted from ChEMBL09 (or the AmpC β-lactamase literature) (Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>). That is an average of 654 ligands per target with a minimum of 40 and a maximum of 3090. Though negative binding is rarely reported, we also found 9219 experimental decoys (i.e., no measurable affinity up to 30 μM), with a maximum of 1070 for cyclo-oxygenase-1 (PGH1).</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-14/jm300687e/production/images/medium/jm-2012-00687e_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-14/jm300687e/production/images/large/jm-2012-00687e_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. DUD-E target classification. Number of the 102 targets that belong to eight broad protein categories.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-14/jm300687e/production/images/large/jm-2012-00687e_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm300687e&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Characteristics of DUD-E</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="center" /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center">total</th><th class="colsep0 rowsep0" align="center" char=".">ChEMBL</th><th class="colsep0 rowsep0" align="center" char=".">manual</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">no. targets</td><td class="colsep0 rowsep0" align="char" char=".">102</td><td class="colsep0 rowsep0" align="char" char=".">101</td><td class="colsep0 rowsep0" align="char" char=".">1</td></tr></tbody></table><table class="table" border="0"><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center" char=".">total</th><th class="colsep0 rowsep0" align="center" char=".">average</th><th class="colsep0 rowsep0" align="center" char=".">minimum</th><th class="colsep0 rowsep0" align="center" char=".">maximum</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">no. raw ligands</td><td class="colsep0 rowsep0" align="char" char=".">66695</td><td class="colsep0 rowsep0" align="right" char=".">653.9</td><td class="colsep0 rowsep0" align="char" char=".">40</td><td class="colsep0 rowsep0" align="char" char=".">3090</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">no. clustered ligands</td><td class="colsep0 rowsep0" align="char" char=".">22886</td><td class="colsep0 rowsep0" align="right" char=".">224.4</td><td class="colsep0 rowsep0" align="char" char=".">40</td><td class="colsep0 rowsep0" align="char" char=".">592</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">no. experimental decoys</td><td class="colsep0 rowsep0" align="char" char=".">9219</td><td class="colsep0 rowsep0" align="right" char=".">90.4</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="char" char=".">1070</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">no. clustered ligands unique charge states</td><td class="colsep0 rowsep0" align="char" char=".">28377</td><td class="colsep0 rowsep0" align="right" char=".">278.2</td><td class="colsep0 rowsep0" align="char" char=".">46</td><td class="colsep0 rowsep0" align="char" char=".">1030</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">no. computational decoys</td><td class="colsep0 rowsep0" align="char" char=".">1411214</td><td class="colsep0 rowsep0" align="right" char=".">13835</td><td class="colsep0 rowsep0" align="char" char=".">2300</td><td class="colsep0 rowsep0" align="char" char=".">51500</td></tr></tbody></table></div></div><div class="NLM_p">With targets selected, we chose a single X-ray structure to represent each target in docking studies (Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>, <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information Table S1">Supporting Information Table S1</a>). To find the structure most amenable to docking, we used an automated docking campaign to screen 3690 PDB structures against their clustered ligands and property-matched decoys (see below). Preference was given to higher resolution, to higher automated enrichment, and to the human orthologue. We avoided mutant structures, unresolved active site loops, extraneous bound peptides, or structures too constrained for many of that target’s ligands. Where we had domain knowledge, the most representative structure was preferred, for example a DFG-in structure for kinases or an antagonist structure for estrogen receptor α (ESR1). For 57 out of 102 targets, a DOCK Blaster<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> prepared structure was used for DUD-E, directly from the automated tool chain. Another 45 targets required manual intervention, most due to simple errors in automated preparation (e.g., incomplete metal atom preparation, missing cofactors, or nonstandard amino acids). A select few needed expert intervention to arrive at modest enrichment, such as adding crystallographic waters, changing histidine protonation, flipping ambiguous side-chains such as asparagine, or increasing a local dipole moment on a specific residue (a technique we often use prospectively to improve polar complementarity<a onclick="showRef(event, 'ref43 ref44'); return false;" href="javascript:void(0);" class="ref ref43 ref44">(43, 44)</a>). In five targets, we incorporated prior docking preparations used for prospective ligand discovery: adenosine A<sub>2A</sub> receptor (AA2AR),<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> β<sub>1</sub> adrenergic receptor (ADRB1), AmpC β-lactamase (AMPC), C-X-C chemokine receptor type 4 (CXCR4),<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> and dopamine D<sub>3</sub> receptor (DRD3).<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a></div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Overview of Representative Targets</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">target class</th><th class="colsep0 rowsep0" align="center">gene ID</th><th class="colsep0 rowsep0" align="center">description</th><th class="colsep0 rowsep0" align="center" char=".">total ligands</th><th class="colsep0 rowsep0" align="center" char=".">clustered ligands</th><th class="colsep0 rowsep0" align="center" char=".">experimental decoys</th><th class="colsep0 rowsep0" align="center" char=".">matched decoys</th><th class="colsep0 rowsep0" align="center">PDB</th><th class="colsep0 rowsep0" align="center" char=".">LogAUC (%)</th><th class="colsep0 rowsep0" align="center" char=".">ROC EF<sub>1</sub></th><th class="colsep0 rowsep0" align="center" char=".">AUC (%)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">cytochrome P450</td><td class="colsep0 rowsep0" align="left">CP2C9</td><td class="colsep0 rowsep0" align="left">cytochrome P450 2C9</td><td class="colsep0 rowsep0" align="char" char=".">145</td><td class="colsep0 rowsep0" align="char" char=".">120</td><td class="colsep0 rowsep0" align="char" char=".">176</td><td class="colsep0 rowsep0" align="char" char=".">7450</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1R9O">1R9O</a></td><td class="colsep0 rowsep0" align="char" char=".">7</td><td class="colsep0 rowsep0" align="char" char=".">3</td><td class="colsep0 rowsep0" align="char" char=".">60</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">CP3A4</td><td class="colsep0 rowsep0" align="left">cytochrome P450 3A4</td><td class="colsep0 rowsep0" align="char" char=".">302</td><td class="colsep0 rowsep0" align="char" char=".">170</td><td class="colsep0 rowsep0" align="char" char=".">267</td><td class="colsep0 rowsep0" align="char" char=".">11800</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3NXU">3NXU</a></td><td class="colsep0 rowsep0" align="char" char=".">7</td><td class="colsep0 rowsep0" align="char" char=".">2</td><td class="colsep0 rowsep0" align="char" char=".">63</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="11" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">GPCR</td><td class="colsep0 rowsep0" align="left">AA2AR</td><td class="colsep0 rowsep0" align="left">adenosine A2a receptor</td><td class="colsep0 rowsep0" align="char" char=".">3057</td><td class="colsep0 rowsep0" align="char" char=".">482</td><td class="colsep0 rowsep0" align="char" char=".">192</td><td class="colsep0 rowsep0" align="char" char=".">31550</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3EML">3EML</a></td><td class="colsep0 rowsep0" align="char" char=".">28</td><td class="colsep0 rowsep0" align="char" char=".">22</td><td class="colsep0 rowsep0" align="char" char=".">83</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">ADRB1</td><td class="colsep0 rowsep0" align="left">β-1 adrenergic receptor</td><td class="colsep0 rowsep0" align="char" char=".">648</td><td class="colsep0 rowsep0" align="char" char=".">247</td><td class="colsep0 rowsep0" align="char" char=".">69</td><td class="colsep0 rowsep0" align="char" char=".">15850</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2VT4">2VT4</a></td><td class="colsep0 rowsep0" align="char" char=".">19</td><td class="colsep0 rowsep0" align="char" char=".">11</td><td class="colsep0 rowsep0" align="char" char=".">76</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">CXCR4</td><td class="colsep0 rowsep0" align="left">C-X-C chemokine receptor type 4</td><td class="colsep0 rowsep0" align="char" char=".">40</td><td class="colsep0 rowsep0" align="char" char=".">40</td><td class="colsep0 rowsep0" align="char" char=".">14</td><td class="colsep0 rowsep0" align="char" char=".">3406</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ODU">3ODU</a></td><td class="colsep0 rowsep0" align="char" char=".">36</td><td class="colsep0 rowsep0" align="char" char=".">18</td><td class="colsep0 rowsep0" align="char" char=".">90</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="11" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ion channel</td><td class="colsep0 rowsep0" align="left">GRIA2</td><td class="colsep0 rowsep0" align="left">glutamate receptor ionotropic AMPA 2</td><td class="colsep0 rowsep0" align="char" char=".">476</td><td class="colsep0 rowsep0" align="char" char=".">158</td><td class="colsep0 rowsep0" align="char" char=".">201</td><td class="colsep0 rowsep0" align="char" char=".">11845</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3KGC">3KGC</a></td><td class="colsep0 rowsep0" align="char" char=".">23</td><td class="colsep0 rowsep0" align="char" char=".">23</td><td class="colsep0 rowsep0" align="char" char=".">71</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">GRIK1</td><td class="colsep0 rowsep0" align="left">glutamate receptor ionotropic kainate 1</td><td class="colsep0 rowsep0" align="char" char=".">136</td><td class="colsep0 rowsep0" align="char" char=".">101</td><td class="colsep0 rowsep0" align="char" char=".">235</td><td class="colsep0 rowsep0" align="char" char=".">6550</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1VSO">1VSO</a></td><td class="colsep0 rowsep0" align="char" char=".">35</td><td class="colsep0 rowsep0" align="char" char=".">27</td><td class="colsep0 rowsep0" align="char" char=".">86</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="11" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">kinase</td><td class="colsep0 rowsep0" align="left">AKT1</td><td class="colsep0 rowsep0" align="left">serine/threonine-protein kinase AKT</td><td class="colsep0 rowsep0" align="char" char=".">585</td><td class="colsep0 rowsep0" align="char" char=".">293</td><td class="colsep0 rowsep0" align="char" char=".">53</td><td class="colsep0 rowsep0" align="char" char=".">16450</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3CQW">3CQW</a></td><td class="colsep0 rowsep0" align="char" char=".">27</td><td class="colsep0 rowsep0" align="char" char=".">29</td><td class="colsep0 rowsep0" align="char" char=".">72</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">MK10</td><td class="colsep0 rowsep0" align="left">c-Jun N-terminal kinase 3</td><td class="colsep0 rowsep0" align="char" char=".">199</td><td class="colsep0 rowsep0" align="char" char=".">104</td><td class="colsep0 rowsep0" align="char" char=".">23</td><td class="colsep0 rowsep0" align="char" char=".">6600</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2ZDT">2ZDT</a></td><td class="colsep0 rowsep0" align="char" char=".">24</td><td class="colsep0 rowsep0" align="char" char=".">11</td><td class="colsep0 rowsep0" align="char" char=".">82</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">MK14</td><td class="colsep0 rowsep0" align="left">MAP kinase p38 α</td><td class="colsep0 rowsep0" align="char" char=".">2205</td><td class="colsep0 rowsep0" align="char" char=".">578</td><td class="colsep0 rowsep0" align="char" char=".">73</td><td class="colsep0 rowsep0" align="char" char=".">35850</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2QD9">2QD9</a></td><td class="colsep0 rowsep0" align="char" char=".">17</td><td class="colsep0 rowsep0" align="char" char=".">10</td><td class="colsep0 rowsep0" align="char" char=".">74</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="11" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">miscellaneous</td><td class="colsep0 rowsep0" align="left">KIF11</td><td class="colsep0 rowsep0" align="left">kinesin-like protein 1</td><td class="colsep0 rowsep0" align="char" char=".">272</td><td class="colsep0 rowsep0" align="char" char=".">116</td><td class="colsep0 rowsep0" align="char" char=".">29</td><td class="colsep0 rowsep0" align="char" char=".">6850</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3CJO">3CJO</a></td><td class="colsep0 rowsep0" align="char" char=".">34</td><td class="colsep0 rowsep0" align="char" char=".">35</td><td class="colsep0 rowsep0" align="char" char=".">77</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">XIAP</td><td class="colsep0 rowsep0" align="left">inhibitor of apoptosis protein 3</td><td class="colsep0 rowsep0" align="char" char=".">100</td><td class="colsep0 rowsep0" align="char" char=".">100</td><td class="colsep0 rowsep0" align="char" char=".">7</td><td class="colsep0 rowsep0" align="char" char=".">5150</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3HL5">3HL5</a></td><td class="colsep0 rowsep0" align="char" char=".">52</td><td class="colsep0 rowsep0" align="char" char=".">55</td><td class="colsep0 rowsep0" align="char" char=".">88</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="11" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">nuclear receptor</td><td class="colsep0 rowsep0" align="left">ESR1</td><td class="colsep0 rowsep0" align="left">estrogen receptor α</td><td class="colsep0 rowsep0" align="char" char=".">1297</td><td class="colsep0 rowsep0" align="char" char=".">383</td><td class="colsep0 rowsep0" align="char" char=".">136</td><td class="colsep0 rowsep0" align="char" char=".">20685</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1SJ0">1SJ0</a></td><td class="colsep0 rowsep0" align="char" char=".">18</td><td class="colsep0 rowsep0" align="char" char=".">15</td><td class="colsep0 rowsep0" align="char" char=".">67</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">MCR</td><td class="colsep0 rowsep0" align="left">mineralocorticoid receptor</td><td class="colsep0 rowsep0" align="char" char=".">201</td><td class="colsep0 rowsep0" align="char" char=".">94</td><td class="colsep0 rowsep0" align="char" char=".">2</td><td class="colsep0 rowsep0" align="char" char=".">5150</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2AA2">2AA2</a></td><td class="colsep0 rowsep0" align="char" char=".">–4</td><td class="colsep0 rowsep0" align="char" char=".">2</td><td class="colsep0 rowsep0" align="char" char=".">36</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">THB</td><td class="colsep0 rowsep0" align="left">thyroid hormone receptor β-1</td><td class="colsep0 rowsep0" align="char" char=".">246</td><td class="colsep0 rowsep0" align="char" char=".">103</td><td class="colsep0 rowsep0" align="char" char=".">29</td><td class="colsep0 rowsep0" align="char" char=".">7450</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1Q4X">1Q4X</a></td><td class="colsep0 rowsep0" align="char" char=".">36</td><td class="colsep0 rowsep0" align="char" char=".">38</td><td class="colsep0 rowsep0" align="char" char=".">79</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">PPARD</td><td class="colsep0 rowsep0" align="left">peroxisome proliferator-activated receptor δ</td><td class="colsep0 rowsep0" align="char" char=".">699</td><td class="colsep0 rowsep0" align="char" char=".">240</td><td class="colsep0 rowsep0" align="char" char=".">79</td><td class="colsep0 rowsep0" align="char" char=".">12250</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2ZNP">2ZNP</a></td><td class="colsep0 rowsep0" align="char" char=".">32</td><td class="colsep0 rowsep0" align="char" char=".">20</td><td class="colsep0 rowsep0" align="char" char=".">89</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="11" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">other enzymes</td><td class="colsep0 rowsep0" align="left">FNTA</td><td class="colsep0 rowsep0" align="left">protein farnesyltransferase type I α</td><td class="colsep0 rowsep0" align="char" char=".">1430</td><td class="colsep0 rowsep0" align="char" char=".">592</td><td class="colsep0 rowsep0" align="char" char=".">132</td><td class="colsep0 rowsep0" align="char" char=".">51500</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3E37">3E37</a></td><td class="colsep0 rowsep0" align="char" char=".">16</td><td class="colsep0 rowsep0" align="char" char=".">7</td><td class="colsep0 rowsep0" align="char" char=".">76</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">HDAC8</td><td class="colsep0 rowsep0" align="left">histone deacetylase 8</td><td class="colsep0 rowsep0" align="char" char=".">309</td><td class="colsep0 rowsep0" align="char" char=".">170</td><td class="colsep0 rowsep0" align="char" char=".">73</td><td class="colsep0 rowsep0" align="char" char=".">10450</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3F07">3F07</a></td><td class="colsep0 rowsep0" align="char" char=".">29</td><td class="colsep0 rowsep0" align="char" char=".">24</td><td class="colsep0 rowsep0" align="char" char=".">80</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">HIVINT</td><td class="colsep0 rowsep0" align="left">HIV type 1 integrase</td><td class="colsep0 rowsep0" align="char" char=".">167</td><td class="colsep0 rowsep0" align="char" char=".">100</td><td class="colsep0 rowsep0" align="char" char=".">268</td><td class="colsep0 rowsep0" align="char" char=".">6650</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3NF7">3NF7</a></td><td class="colsep0 rowsep0" align="char" char=".">8</td><td class="colsep0 rowsep0" align="char" char=".">2</td><td class="colsep0 rowsep0" align="char" char=".">64</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">KITH</td><td class="colsep0 rowsep0" align="left">thymidine kinase</td><td class="colsep0 rowsep0" align="char" char=".">57</td><td class="colsep0 rowsep0" align="char" char=".">57</td><td class="colsep0 rowsep0" align="char" char=".">68</td><td class="colsep0 rowsep0" align="char" char=".">2850</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2B8T">2B8T</a></td><td class="colsep0 rowsep0" align="char" char=".">15</td><td class="colsep0 rowsep0" align="char" char=".">0</td><td class="colsep0 rowsep0" align="char" char=".">80</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">PARP1</td><td class="colsep0 rowsep0" align="left">poly (ADP-ribose)polymerase-1</td><td class="colsep0 rowsep0" align="char" char=".">1031</td><td class="colsep0 rowsep0" align="char" char=".">508</td><td class="colsep0 rowsep0" align="char" char=".">12</td><td class="colsep0 rowsep0" align="char" char=".">30050</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3L3M">3L3M</a></td><td class="colsep0 rowsep0" align="char" char=".">25</td><td class="colsep0 rowsep0" align="char" char=".">21</td><td class="colsep0 rowsep0" align="char" char=".">79</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">PUR2</td><td class="colsep0 rowsep0" align="left">GAR transformylase</td><td class="colsep0 rowsep0" align="char" char=".">50</td><td class="colsep0 rowsep0" align="char" char=".">50</td><td class="colsep0 rowsep0" align="char" char=".">12</td><td class="colsep0 rowsep0" align="char" char=".">2700</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1NJS">1NJS</a></td><td class="colsep0 rowsep0" align="char" char=".">51</td><td class="colsep0 rowsep0" align="char" char=".">50</td><td class="colsep0 rowsep0" align="char" char=".">92</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="11" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">protease</td><td class="colsep0 rowsep0" align="left">DPP4</td><td class="colsep0 rowsep0" align="left">dipeptidyl peptidase IV</td><td class="colsep0 rowsep0" align="char" char=".">1939</td><td class="colsep0 rowsep0" align="char" char=".">533</td><td class="colsep0 rowsep0" align="char" char=".">167</td><td class="colsep0 rowsep0" align="char" char=".">40950</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2I78">2I78</a></td><td class="colsep0 rowsep0" align="char" char=".">41</td><td class="colsep0 rowsep0" align="char" char=".">41</td><td class="colsep0 rowsep0" align="char" char=".">87</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">FA10</td><td class="colsep0 rowsep0" align="left">coagulation factor X</td><td class="colsep0 rowsep0" align="char" char=".">3090</td><td class="colsep0 rowsep0" align="char" char=".">537</td><td class="colsep0 rowsep0" align="char" char=".">176</td><td class="colsep0 rowsep0" align="char" char=".">28325</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3KL6">3KL6</a></td><td class="colsep0 rowsep0" align="char" char=".">39</td><td class="colsep0 rowsep0" align="char" char=".">36</td><td class="colsep0 rowsep0" align="char" char=".">87</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">LKHA4</td><td class="colsep0 rowsep0" align="left">leukotriene A4 hydrolase</td><td class="colsep0 rowsep0" align="char" char=".">343</td><td class="colsep0 rowsep0" align="char" char=".">171</td><td class="colsep0 rowsep0" align="char" char=".">21</td><td class="colsep0 rowsep0" align="char" char=".">9450</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3CHP">3CHP</a></td><td class="colsep0 rowsep0" align="char" char=".">18</td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="char" char=".">82</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">MMP13</td><td class="colsep0 rowsep0" align="left">matrix metallo-proteinase 13</td><td class="colsep0 rowsep0" align="char" char=".">1632</td><td class="colsep0 rowsep0" align="char" char=".">572</td><td class="colsep0 rowsep0" align="char" char=".">26</td><td class="colsep0 rowsep0" align="char" char=".">37200</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=830C">830C</a></td><td class="colsep0 rowsep0" align="char" char=".">12</td><td class="colsep0 rowsep0" align="char" char=".">5</td><td class="colsep0 rowsep0" align="char" char=".">71</td></tr></tbody></table></div></div><div class="NLM_p">To increase scaffold diversity and to make smaller, more manageable ligand sets, we clustered the raw ChEMBL ligands by their Bemis–Murcko atomic frameworks.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> These atom-type based frameworks include ring systems of the molecule and connecting linkers, minus any side fragments. For example, the seventh largest Murcko cluster in kinesin-like protein 1 (KIF11) has seven ligands, all close analogues (Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>A). If at least 100 frameworks were present, then we included only the highest affinity ligand from each framework. If fewer were available, we raised the number of ligands selected from each framework until we obtained more than 100 molecules, trading diversity for quantity. Returning to kinesin-like protein 1, we extracted only 70 Murcko frameworks (Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>B). Out of 276 raw ligands, the five largest Murcko clusters contained 146 ligands (53%). Selecting the two or three highest affinity ligands from each framework results in 98 and 118 ligands, respectively, so we stopped at three ligands per framework. In the process we still managed to remove 158 lower affinity compounds from highly redundant clusters. In a few targets, more than 600 ligands remained even after clustering, so we reduced the affinity threshold below 1 μM in the sequence (300, 100, 30, 10, and 3 nM), until fewer than 600 frameworks were found. For example, in adenosine A<sub>2A</sub> receptor, there are 3096 raw ligands resulting in 1099 frameworks at 1 μM, but we can reduce the number of frameworks to 483 using a 30 nM affinity threshold (Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>C).</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-14/jm300687e/production/images/medium/jm-2012-00687e_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-14/jm300687e/production/images/large/jm-2012-00687e_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Ligand clustering. (A) The seventh largest Murcko cluster of kinesin-like protein 1 (KIF11), showing both the scaffold (left) and all seven member ligands. (B) Number of ligands in each of the 70 KIF11 Bemis–Murcko atomic frameworks. We removed lower affinity compounds over-represented clusters (above the line), while retaining 100 ligands. (C) Number of adenosine A<sub>2A</sub> receptor (AA2AR) Murcko clusters is plotted against affinity threshold. Fewer than 600 clusters are present using a 30 nM affinity threshold.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-14/jm300687e/production/images/large/jm-2012-00687e_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm300687e&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To examine the effect of clustering on docking enrichments, we docked the three targets with the highest and lowest fraction of clustered to raw ligands from those with enough ligands to pick one ligand per Murcko cluster. To measure docking performance we used LogAUC, an aggregate metric that gives early enrichment more weight. As described previously,<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> LogAUC is completely analogous to AUC but in the transformed space after you have zoomed in on early enrichment by taking the semilog of the <i>x</i>-axis. In tryptase β1 (TRYB1), the target with the highest clustered fraction, clustering substantially decreases the LogAUC by 6%, whereas in the other five targets clustering increases the LogAUC (<a class="ref internalNav" href="#notes-1" aria-label="Supporting Information Table S2">Supporting Information Table S2</a>). The mean absolute deviation over the six targets is 3.7% LogAUC, but in all cases the raw and clustered ROC curves have similar shapes (data not shown). Overall, we believe the clustered sets provide a better measure of docking performance with lower docking effort and will be used in the remainder of this work.</div><div class="NLM_p">A key problem with the original DUD decoys was that they sometimes closely resembled the ligands, occasionally even being confirmed as binders. Enforcing 2-D topological dissimilarity between decoys and ligands should eliminate this problem in principle, but in practice critical ligand binding “warheads” often remain in the decoy set selected from ZINC,<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> e.g., amidine groups in factor Xa (FA10). By identifying these warheads in three targets (Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>A), we investigated how to eliminate false decoys. In the original DUD, CACTVS fingerprints were used to select decoys with Tanimoto coefficients (<i>T</i><sub>c</sub>) to ligands below 0.9, which is roughly similar to using Daylight fingerprints with <i>T</i><sub>c</sub> below 0.7.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> In recent work,<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> we used Daylight fingerprints with a more restrictive <i>T</i><sub>c</sub> < 0.5. Using this filter on the enhanced DUD ligand sets, we still saw 39%, 53%, and 96% of possible warhead bearing molecules passing through in factor Xa (FA10), glycinamide ribonucleotide transformylase (PUR2), and thymidine kinase (KITH), respectively (Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>B). Using Daylight with <i>T</i><sub>c</sub> < 0.325, we reduced FA10 warheads below 1% but still saw 14% and 34% in PUR2 and KITH. Clearly different targets and even different ligands require different absolute thresholds. To circumvent this, we removed a percentage of the most similar decoys for each ligand, sorted by maximum <i>T</i><sub>c</sub> to any ligand. This allowed the effective absolute threshold to vary. Removing 50% of the decoys with Daylight was better in KITH, while removing 50% with ECFP4 was better in FA10 and PUR2. The final procedure of using ECFP4 fingerprints and removing 75% of the decoys, resulted in 0.2%, 0%, and 5.8% of warheads remaining, substantially reducing the number of false decoys. Having refined the decoy dissimilarity procedure on three targets where we could define a warhead, we then applied it to all generated decoys. To help ensure that the resulting decoys were, in fact, substantially different, topologically, from the ligands, we compared the two by a metric partially orthogonal to topology, asking how many decoy molecules shared the same scaffold as a ligand. Of the 805136 decoy scaffolds over all of DUD-E, only 692 (0.086%) were found among the 25503 ligand scaffolds, consistent with substantial topological differences among the two sets despite their close physical property matching.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-14/jm300687e/production/images/medium/jm-2012-00687e_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-14/jm300687e/production/images/large/jm-2012-00687e_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Decoy generation. (A) Three key “warhead” groups from factor Xa (FA10), glycinamide ribonucleotide transformylase (PUR2), and thymidine kinase (KITH). (B) Fraction of warheads remaining is plotted against the dissimilarity method. The dissimilarity methods consist of a fingerprint (Daylight or ECFP4) and either a hard cutoff or a fraction of the most dissimilar decoys to be retained. (C) Property distributions of estrogen receptor α (ESR1) for both the 383 ligands (blue) and the 20685 property-matched decoys (red).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-14/jm300687e/production/images/large/jm-2012-00687e_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm300687e&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In addition to reducing false decoys, the DUD-E decoy generation procedure was extensively revised. Each decoy derived from a particular ligand, where decoy property ranges around the ligands properties adjusted to seven possible widths. This adapted to local chemical space around each ligand, allowing more closely matched decoys. Also, net charge was added to the property matching, as it is critical in electrostatics and desolvation. The improved property-matching can be seen in the property histograms for estrogen receptor α (ESR1) (Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>C) as well as the averages and standard deviations for all the targets (<a class="ref internalNav" href="#notes-1" aria-label="Supporting Information Table S3">Supporting Information Table S3</a>). Using ZINC<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> for the potential decoy pool made them purchasable, enabling experimental testing for actual binding to the target. As a result of this work, this enhanced decoy procedure has been fully automated and is available online to generate DUD-E style decoys for any user supplied list of input ligands at <a href="http://decoys.docking.org" class="extLink">http://decoys.docking.org</a>.</div><div class="NLM_p">The original DUD paper<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> showed that a property-matched decoy set is more challenging for docking than a random collection of molecules. Therefore, we compared enrichments using property-matched decoys to those using a random drug-like background, which consisted of all ChEMBL12 ligands with affinities better than 10 μM. Switching from a drug-like background to DUD-E property-matched decoys does reduce average enrichment over the 102 targets, from 26.8% to 24.4% LogAUC (<a class="ref internalNav" href="#notes-1" aria-label="Supporting Information Table S4">Supporting Information Table S4</a>). Yet for three targets, the property-matched sets unexpectedly led to much better enrichment, by more than 15% LogAUC. In both glutamate receptor ionotropic kainate 1 (GRIK1) and purine nucleoside phosphorylase (PNPH), the ligands have low molecular weights (<a class="ref internalNav" href="#notes-1" aria-label="Supporting Information Table S3">Supporting Information Table S3</a>) and thus scored poorly against the generally larger ChEMBL12 molecules, just as Verdonk<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> suggests. In urokinase-type plasminogen activator (UROK), the top of the drug-like docking hit list is dominated by decoys with amidine “warheads”. Because these are likely binders, the increased property-matched enrichment resulted from fewer false decoys in that set. Indeed, the 2.4% LogAUC reduction that occurs upon switching to property-matched decoys arises from these two competing factors: property matching the decoys reduces enrichment, and reduction of false decoys increases enrichment.</div><div class="NLM_p">Overall, enrichment as measured by average LogAUC is 1.5 fold higher in DUD-E compared to the original DUD. To understand this, we first isolated the change due to the revised decoy generation procedure. Using the original DUD ligands and target preparations, but switching from original decoys to these revised decoys substantially increased the average enrichment over the 37 directly comparable targets from 14.8% to 19.7% LogAUC (Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a>, <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information Table S5">Supporting Information Table S5</a>). With the new adaptive property-matching procedure incorporating net charge, the revised decoys might have been expected to lower enrichment, but instead we saw an overall increase. Inspecting the docking hit lists, we observed a dramatic decrease in high scoring decoys that resemble ligands to a degree that they might actually bind. Indeed, all three targets with identifiable warheads that we used to tune the dissimilarity procedure showed large increases in enrichment: FA10 increases from 13% to 28% LogAUC, PUR2 from 40% to 62% LogAUC, and KITH from 1% to 32% LogAUC. If we now isolate the switch from original ligands and revised decoys to both DUD-E ligands and decoys, we see a moderate decrease in average enrichment from 19.7% to 16.4% LogAUC. We attribute this decrease to the larger, more diverse clustered ligand lists in DUD-E. Lastly, switching the target preparation, and the choice of the particular PDB structure used to represent a target, substantially increases enrichment from 16.4% to 22.8% LogAUC between DUD and DUD-E (<a class="ref internalNav" href="#notes-1" aria-label="Supporting Information Table S5">Supporting Information Table S5</a>). The overall effect with SEV ligand desolvation in DOCK 3.6 is to increase average enrichment from 14.8% LogAUC against DUD to 22.8% LogAUC against the DUD-E benchmark.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Decomposition of Enrichment Changes between DUD and DUD-E</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">incremental change</th><th class="colsep0 rowsep0" align="center">all original</th><th class="colsep0 rowsep0" align="center">new style decoys</th><th class="colsep0 rowsep0" align="center">switch to new ligands</th><th class="colsep0 rowsep0" align="center">switch target preparation</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">decoys</td><td class="colsep0 rowsep0" align="center">DUD</td><td class="colsep0 rowsep0" align="center">DUD-E</td><td class="colsep0 rowsep0" align="center">DUD-E</td><td class="colsep0 rowsep0" align="center">DUD-E</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ligands</td><td class="colsep0 rowsep0" align="center">DUD</td><td class="colsep0 rowsep0" align="center">DUD</td><td class="colsep0 rowsep0" align="center">DUD-E</td><td class="colsep0 rowsep0" align="center">DUD-E</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">receptor preparation</td><td class="colsep0 rowsep0" align="center">DUD</td><td class="colsep0 rowsep0" align="center">DUD</td><td class="colsep0 rowsep0" align="center">DUD</td><td class="colsep0 rowsep0" align="center">DUD-E</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">average LogAUC<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="center">14.8</td><td class="colsep0 rowsep0" align="center">19.7</td><td class="colsep0 rowsep0" align="center">16.4</td><td class="colsep0 rowsep0" align="center">22.8</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup>a</sup><p class="last">Over the 37 common targets (target-by-target data in <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information Table S5">Supporting Information Table S5</a>).</p></div></div></div><div class="NLM_p">A central motivation for any benchmarking set is to test, at least retrospectively, new methods. We wanted to explore how our recent context-dependent ligand desolvation method<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> behaved against the DUD-E benchmark. We therefore used it to re-examine the utility of solvent-excluded volume (SEV) ligand desolvation versus using no desolvation term (None) or using the full transfer free energy from water to hexadecane (Full). In our initial study of these terms on the 40 original DUD targets, SEV improved upon None by just 0.7% average LogAUC. Conversely, over the 102 DUD-E targets, SEV substantially outperformed None by 3.8% LogAUC on average, with average LogAUC values of 20.6, 14.3, and 24.4% for None, Full, and SEV desolvation methods, respectively (Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>, <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information Table S4">Supporting Information Table S4</a>). Despite these average trends, ROC curves on individual targets can vary significantly among the various methods (Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>). As in the original desolvation analysis, some targets are more amenable to full desolvation, such as catechol <i>O</i>-methyltransferase (COMT) and purine nucleoside phosphorylase (PNPH), while others are more amenable to no desolvation, such as factor X (FA10) and glycinamide ribonucleotide transformylase (PUR2). Against the DUD-E benchmark, SEV desolvation not only outperforms the other methods, but performs well in both types of targets. This suggests that over a more comprehensive set of targets, and what we argue is a better set of ligands and decoys, the advantage of the more physically correct SEV ligand desolvation treatment becomes more pronounced.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-14/jm300687e/production/images/medium/jm-2012-00687e_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-14/jm300687e/production/images/large/jm-2012-00687e_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Retrospective enrichment comparing ligand desolvation and electrostatics methods. Docking results over DUD-E as measured by LogAUC. “None” has no ligand desolvation term, “SEV” uses solvent-excluded volume ligand desolvation, “Thin” employs a thin low-dielectric layer in the electrostatic calculations.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-14/jm300687e/production/images/large/jm-2012-00687e_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm300687e&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-14/jm300687e/production/images/medium/jm-2012-00687e_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-14/jm300687e/production/images/large/jm-2012-00687e_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Representative ROC plots. ROC plots using no desolvation (None), solvent-excluded volume ligand desolvation (SEV), the thin low-dielectric layer (Thin), or a drug-like background that consists of all ChEMBL12 ligands with affinities better than 10 μM (Drug-like). The black dotted line represents the results expected from docking ligands randomly. LogAUC percentages are reported in the legend text.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-14/jm300687e/production/images/large/jm-2012-00687e_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm300687e&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Electrostatic interaction with the protein is a large term that opposes ligand desolvation, with their relative balance being critical for binding. Because we do not know the binding pose of putative ligands prior to docking, we need to approximate the region of low dielectric the ligand might occupy to precompute electrostatic grids. Previously, we used the negative image of the receptor (computed by SPHGEN) to construct this low dielectric region, but manual tweaking was often required. In the large open binding pocket of CXCR4, we observed that using a thin layer of low-dielectric around just the edge of the protein allowed ligands to interact with it while reducing the bulk dielectric perturbation at the center of its large binding pocket.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> Here we explored using an automated thin dielectric layer strategy across the entire DUD-E set. Visually, these new automated thinner dielectric layers are more physically realistic, even in the rare case when they are effectively thicker than the previous layers (due to a water probe being able to penetrate that layer). With these thin low-dielectric layers (Thin), the average LogAUC over the 102 targets improved from 24.4% to 24.9% (Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>, <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information Table S4">Supporting Information Table S4</a>). Six targets used manually prepared dielectric layers (AA2A2, ADRB1, AMPC, CDK2, CXCR4, and DRD3) and thus do not directly reflect the difference between automated dielectric layers. Excluding those six enlarges the average difference from 0.5% to 1.0% LogAUC. Admittedly, these are moderate differences, but they exemplify how DUD-E may be used to test new docking methods and hint that as we progress docking models, enrichment will improve.</div><div class="NLM_p">Here we present three representative targets in greater detail to display a magnified view of DUD-E.</div><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Mineralocorticoid Receptor (MCR)</h3><div class="NLM_p">MCR has the lowest enrichment in DUD-E. Across all 11 automatically docked structures, enrichment of DUD-E ligands to its decoys was negligible. Thus we selected the same PDB structure as the original DUD, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2AA2">2AA2</a> at 1.95 Å resolution. While enrichment using the new DUD-E sets was worse than random at −4% LogAUC and 36% AUC (Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>), using the original DUD ligands and decoys gave 45% LogAUC and 76% AUC. Despite poor enrichment in DUD-E, building and docking the crystal ligand from scratch, ignoring crystallographic information, resulted in good pose agreement (Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a>A). Taken together, we can rationalize the enrichment differences, as 13 of 15 original ligands shared a polycyclic scaffold with the well-docked crystal ligand, while the 94 new ligands had much more scaffold diversity. Thus the reduced enrichment in DUD-E reflects increased chemotype diversity as a result of including more ligands and clustering them by Bemis–Murcko atomic frameworks. Of the four lowest enriching targets in DUD-E, three are nuclear hormone receptors, with glucocorticoid receptor (GCR) and androgen receptor (ANDR) joining MCR. These receptors all have hydrophobic pockets with flexible binding site residues such as methionine and leucine so that a single rigid receptor may be incapable of docking all of their ligands. Thus these targets may be good tests of flexible receptor docking methods.</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-14/jm300687e/production/images/medium/jm-2012-00687e_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-14/jm300687e/production/images/large/jm-2012-00687e_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Representative docking poses. The crystallographic ligand was rebuilt and docked from scratch. (A–F) The crystal pose (magenta) is compared to the resulting docked pose (green). In (C), more ligand conformations are generated and the redocked pose is also shown (tan). Key hydrogen bonds are shown by black dotted lines, and the partially transparent protein surface is colored by atom type.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-14/jm300687e/production/images/large/jm-2012-00687e_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm300687e&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Thyroid Hormone Receptor β1 (THB)</h3><div class="NLM_p last">THB produced good enrichment when a structure with an open subpocket was selected. Enrichment for the 16 automatically docked structures varies significantly, ranging from 13% (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1NQ0">1NQ0</a>) to 37% LogAUC (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1Q4X">1Q4X</a>). The lower enriching structures have larger cavities near Arg320 (right side of Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a>B), opening to solvent in <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1NQ0">1NQ0</a>; the higher enriching structures have larger cavities at the other end of the binding site near Met420 (left side), opening to solvent in <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1Q4X">1Q4X</a>. We selected the automated preparation of <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1Q4X">1Q4X</a> despite its modest 2.80 Å resolution because Thr273 is pushed away by the crystal ligand, making the left subpocket larger. Using SEV desolvation then yields enrichment statistics of 36% LogAUC, 79% AUC, and a receiver operating characteristic curve based enrichment factor at 1% (EF<sub>1</sub>) of 38 (Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>). The redocked crystal ligand has excellent pose agreement (Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a>B).</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Serine/Threonine-Protein Kinase AKT (AKT1)</h3><div class="NLM_p last">AKT1 is a newly added kinase that demonstrates several considerations during PDB structure selection. Whereas 10 PDB structures were automatically docked, four got worse than random enrichment. All four correspond to structures of the Pleckstrin homology (PH) domain instead of the kinase domain. The structure with the best normal AUC, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3O96">3O96</a>, corresponds to an allosteric site at the interface of the PH and kinase domains, not the traditional ATP binding pocket. While the best enriching structure by LogAUC, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3CQW">3CQW</a> at 2.00 Å, corresponds to the canonical site, its nonstandard phosphothreonine amino acid evades the automated protocol. Preparing that residue manually results in 27% LogAUC, 72% AUC, and 29 EF<sub>1</sub> (Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>). Nevertheless, the redocked ligand (green) fails to generate the crystal ligand pose (magenta) (Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a>C). The ligand, however, is quite small, with one central rotatable bond, and requires a specific rotation about that bond to fit in the binding site. Lowering the rmsd threshold for ligand conformation generation allows that rotation to be sampled, restoring the correct ligand binding pose (tan) (Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a>C).</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i9">Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_04507" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_04507" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Docking continues to be judged by hit rates in prospective studies, and by enrichment in retrospective recall studies, because it cannot now hope to calculate affinities or even monotonic rank order. Like protein structure prediction, docking thus remains an empirical, although we would argue also a pragmatic field. Its reliance on enrichment has driven the development of benchmarking sets, first explored by Rognan<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> and Jain,<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> recently investigated by Boeckler<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> and Cavasotto;<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> the most widely used and cited of these remains the Directory of Useful Decoys (DUD).<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> Despite its widespread adoption, DUD retains serious liabilities, including a lack of ligand diversity, lack of property-matching to net charge, and a substantial number of false decoys. The enhanced DUD (DUD-E) described here was developed to address these shortcomings and to expand the target list to be more reprentative of pharmacologically relevant space.</div><div id="sec3_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Balancing Ligands and Decoys for Enrichment</h3><div class="NLM_p">An important problem with DUD arose from the ligands and decoys originally chosen. The former sometimes over-represented in a few chemotypes, and the latter were sometimes not decoys but actually ligands. Moreover, the mapping of specific ligands to their matched decoys had been lost in the released set. In the 102 ligand and decoy sets that comprise DUD-E, ligand diversity in any given set is substantially increased, reducing the bias that can come from a single chemotype ranking well. With at least 40 ligands for every target and a preference to maximize chemotype diversity, DUD-E allows for more representative tests of docking screens. Correspondingly, property-matching decoys to each ligand individually, while more stringently removing false decoys (i.e., ligands), allows investigators to directly match specific ligands to their decoy molecules and reduces what had been artifactually low enrichment for some targets in DUD. Adding net charge as a property to match between ligands and decoys resolves a discrepancy between them in DUD, where the ligands had tended to be more charged, on average, than the decoys, which had the effect of skewing our evaluation of physical forces like desolvation.</div><div class="NLM_p last">The impact of these changes on docking performance is substantial and clarifying. In isolation from other effects, clustering the ligands for diversity reduces enrichment, as one might expect because high-performing, over-represented sets have been largely removed. Conversely, the new decoys <i>increase</i> enrichment compared to the DUD performance. At first this seemed counterintuitive, because one imagines that a better-balanced, more stringent decoy set will be a greater challenge for a docking program. However, this is more than balanced by the removal of what had been false decoys (ligands), which artifactually reduced enrichment in DUD because, as ligands, they had often ranked well but counted as decoys they diluted the annotated ligands. Finally, the new target preparations, carefully selected from a docking campaign to over 3500 structures, also increased enrichment. Overall, the increased enrichment in DUD-E should provide more sensitivity for benchmarking docking algorithms, giving it greater responsiveness to modifications that reduce enrichment as well as those that increase it.</div></div><div id="sec3_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Online Tools for Automated Generation of Further Ligand and Decoy Sets</h3><div class="NLM_p last">DUD-E is built to be a better platform for refinement and extension of ligand and decoy sets. Targets are independent of one another, both in ligands and decoys, allowing target addition, deletion, or replacement. The protocol to generate decoys for DUD-E is made available online to generate decoys for any target given only a list of ligand structures, which enables extension of DUD-E to new targets of interest by individual investigators. The decoy server pulls directly from a purchasable subset of the ZINC database, inheriting its improvements and purchasing updates.<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> The final decoy selection from the applicable pool of decoys is random where possible, allowing the generation of multiple decoys sets to test overfitting to the canonical DUD-E decoys. Each decoy belongs to one and only one ligand, so if one wants to filter a ligand, then the corresponding decoys can be easily removed. For example, we provide raw ligand and decoy sets before clustering by Bemis–Murcko atomic frameworks. If a different clustering method was desired, which selected a different subset of the raw ligands, then the corresponding decoys could be retained (furthermore, we provide the python script used to generate clustered subsets from raw sets). We also include extra data that allows some design decisions to be altered, for instance, we include the marginal ligands which are active above our 1 μM cutoff.</div></div><div id="sec3_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Applications to Docking Optimization and Testing</h3><div class="NLM_p">DUD-E should provide a more robust benchmarking set for exploring new docking methods, so we were keen to test it against new methods that we had been investigating. When tested against the older DUD set, we had found that a new solvent-excluded volume (SEV) ligand desolvation method had had a disappointingly small effect on enrichment despite what was clearly a better physical model. However, when measured against the DUD-E benchmark, the differential performance between the old and new method increased substantially in the latter’s favor. Similarly, against the DUD-E benchmark, a more physically realistic dielectric layer, used to calculated the electrostatic interaction term from static Poission–Boltzmann maps, also led to improved enrichments that had been largely masked in the DUD set, owing to the problems described above.</div><div class="NLM_p">Certain cautions merit airing. Most importantly, DUD-E is a large data set synthesized from several source databases, each of which is continuously evolving and improving. Thus individual errors are expected, though usually traceable to the source database at the time DUD-E was constructed. Although we only show docking results using DOCK 3.6 with solvent-excluded volume ligand desolvation, DUD-E was designed to be a general benchmarking set. Thus some arbitrary choices and simplifying assumptions were made in the effort to provide one canonical data set useful to compare docking algorithms. For instance, we assume a single PDB code can represent the target, but some targets are highly flexible or they contain both orthosteric and allosteric binding pockets. Fundamentally, DUD and DUD-E are designed to measure value-added screening performance of 3-D methods over simple 1-D molecular properties. Decoys that might bind are removed using 2-D ligand similarity, so DUD-E is inappropriate to test 2-D methods. Through its construction, ligands light up against DUD-E decoys using these 2-D similarity methods, which create an artificially favorable enrichment bias for them. A final caution is that to filter more false decoys in DUD-E, we keep only a quarter of the most highly dissimilar decoys. However, while we show that this increased dissimilarity removes false decoys, it could also contribute to artificial increases in docking enrichment.</div><div class="NLM_p last">Notwithstanding these caveats, DUD-E is substantially improved over the original DUD. It is a larger, more diverse data set with better matched decoys that resemble ligands less, correcting many flaws in its predecessor. Although we anticipate that it will be most widely used in the instantiation we describe here, it was developed with the idea that it could be flexibly extended and evolved; the tools to do so are even provided online (<a href="http://dude.docking.org" class="extLink">http://dude.docking.org</a>). We hope that it and its descendants will provide a useful tool for docking evaluation in the community until such time as a more fundamental measurement of docking performance is possible.</div></div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i13">Methods</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_12253" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_12253" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> ChEMBL and RCSB PDB Data Extraction</h3><div class="NLM_p last">This enhanced DUD database has been constructed by combining ligand data from ChEMBL<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> and structural data from RCSB PDB<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> (<a class="ref internalNav" href="#notes-1" aria-label="Supporting Information Figure S1A">Supporting Information Figure S1A</a>). Ligands assigned to protein targets (ChEMBL confidence score ≥4) with affinities (IC<sub>50</sub>, EC<sub>50</sub>, <i>K</i><sub>i</sub>, <i>K</i><sub>d</sub>, and log variants thereof) of 1 μM or better were extracted from the ChEMBL09 database.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> Similarly, we assigned experimental decoys as molecules with no measurable affinity at 30 μM or higher (greater than relation only). The remaining ligands with affinities above 1 μM, and decoys with no measurable affinity below 30 μM, are included for completeness and dubbed “marginal”. Via ChEMBL, ligands are associated with a particular target sequence by UniProt<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> accession code, and then mapped<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> from UniProt accession codes to protein data bank (PDB) structures (X-ray only) using <a href="http://www.uniprot.org/docs/pdbtosp.txt" class="extLink">http://www.uniprot.org/docs/pdbtosp.txt</a>, obtained on February 23, 2011.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Target Selection Docking</h3><div class="NLM_p last">Preliminary docking calculations were performed on each PDB structure that mapped to ChEMBL ligands and contained a single, unambiguous cocrystal ligand as prepared by DOCK Blaster.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> Property-matched computational decoys were generated by the automated decoy generation procedure below, using Daylight fingerprints with a Tanimoto coefficient (<i>T</i><sub>c</sub>) threshold below 0.5. These decoys were docked and compared to their cognate ligands using DOCK 3.6 with solvent-excluded volume (SEV) ligand desolvation.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> Balancing the parallel goals of diversity, drug relevance, many ligands and structures, and at least modest automated docking enrichment, we selected 119 tentative targets for the new DUD. This list was reduced to the final 102 targets by factors such as ligand and PDB duplication between targets (e.g., FNTB duplicates FNTA), low resolution structures (RAF1), sterically constrained binding sites (NR1H2, THA), or over-representation (MK08, MTOR).</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Target Preparation</h3><div class="NLM_p last">For each target, we assembled all UniProt accession codes (species) with any raw ChEMBL compounds (ligands, decoys, marginal ligands, or marginal decoys). For only those accession codes, structures were extracted using the ChEMBL to PDB mapping, except P07700 was manually added to ADRB1 to include six more rare structures for that GPCR. This procedure neglects those PDB structures that belong to an accession code having no ChEMBL compounds. For example, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1KIM">1KIM</a> is the PDB structure of thymidine kinase (KITH) in the original DUD. This KITH structure is from herpes virus (UniProt P03176), an accession code with no raw compounds extracted from ChEMBL, and is thus not included in the ChEMBL/PDB intersection used to construct the new DUD. Still, 5025 PDB codes were sent to an updated DOCK Blaster pipeline for automated docking preparation (<a class="ref internalNav" href="#notes-1" aria-label="Supporting Information Figure S1D">Supporting Information Figure S1D</a>). In some cases, an unambiguous ligand could not be found to indicate the binding site, but we were able to assign 565 additional ligands by manually inspecting over 1300 structures. Ultimately, 3692 structures completed input grid preparation, and all but two finished docking and enrichment analysis. Clustered ligands sets were docked to property-matched decoys (both described below) using ECFP4 fingerprints and removing the most similar 75% of queried decoys. DOCK 3.6 was run using SEV ligand desolvation (as below). For each target, enrichment, resolution, and organism were collected and sorted by enrichment in pdb_analyze.txt, available online at <a href="http://dude.docking.org" class="extLink">http://dude.docking.org</a>. Crude notes on the selection process are recorded in pdb_selection.txt, and the picked structure is listed in pdb_blessed.txt. AA2AR and DRD3 docking preparations were provided by Jens Carlson,<a onclick="showRef(event, 'ref44 ref45'); return false;" href="javascript:void(0);" class="ref ref44 ref45">(44, 45)</a> CXCR4 partially by Dahlia Weiss,<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> ADRB1 by Peter Kolb (personal communication), and AMPC by Sarah Barelier, Oliv Eidam, and Inbar Fish (unpublished results).</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Ligand Preparation</h3><div class="NLM_p last">To prepare ligand sets for each target, ChEMBL affinities and log variants were first normalized to nM units (<a class="ref internalNav" href="#notes-1" aria-label="Supporting Information Figure S1B">Supporting Information Figure S1B</a>). Salts were removed, charges were normalized, and properties were calculated using Molinspiration’s mib package (<a href="http://www.molinspiration.com" class="extLink">www.molinspiration.com</a>). Ligands with 600 Da or higher molecular weight or with 20 or more rotatable bonds were removed. Smiles were put in canonical form using OpenEye’s OEChem software.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> Ligand sets from each species were combined, sorted by ascending normalized affinity, and then made unique based on canonical smiles. The same procedure was used to collate the experimental decoys, marginal ligands, and marginal decoys. For AmpC β-lactamase (AMPC), an original DUD target, the ChEMBL09 ligands are covalent in nature. To identify noncovalent ligands, we manually compiled ligands<a onclick="showRef(event, 'ref6 ref43 ref49 ref50'); return false;" href="javascript:void(0);" class="ref ref6 ref43 ref49 ref50">(6, 43, 49, 50)</a> with affinities below 5 mM and experimental decoys<a onclick="showRef(event, 'ref43 ref51'); return false;" href="javascript:void(0);" class="ref ref43 ref51">(43, 51)</a> from the literature.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Ligand Clustering</h3><div class="NLM_p last">To reduce the sometimes large number of ChEMBL ligands down to a manageable size while also increasing scaffold diversity as suggested by Good and Oprea,<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> we clustered the ligands by their Bemis–Murcko atomic frameworks,<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> as generated by Molinspiration’s mib. If there were 100 or more frameworks, we chose only the highest affinity ligand from each. If there were fewer than 100 Murcko frameworks, we increased the number of highest affinity ligands taken from each until we achieved at least 100 ligands (or until all ligands were included). Conversely, if there were more than 600 Murcko frameworks, then we decreased the ligand affinity threshold in the sequence [1 μM, 300 nM, 100 nM, 30 nM, 10 nM, 3 nM] until fewer than 600 frameworks were present, where we then took the highest affinity ligand from each framework. While clustered ligand sets are the default, the full unclustered ligand sets and corresponding decoys are available. The script (subset_decoys.py) used to select the clustered subset given the ligand ids is provided with the full ligand set to enable other clustering algorithms or filtering methods to be substituted.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> Automated Decoy Generation</h3><div class="NLM_p last">As in the original DUD, we property-matched decoys to ligands using molecular weight, estimated water–octanol partition coefficient (miLogP), rotatable bonds, hydrogen bond acceptors, and hydrogen bond donors, plus we added net charge. We generated all ligand protonation states in pH range 6–8 using Schrödinger’s Epik with arguments “-ph 7.0 -pht 1.0 -tp 0.20” (<a class="ref internalNav" href="#notes-1" aria-label="Supporting Information Figure S1C">Supporting Information Figure S1C</a>). Molecular properties were then computed using Molinspiration’s mib. Over all the protonated forms of a given ligand, we kept only those with a unique set of the six physicochemical properties. For each of these unique property sets, we aimed to generate 50 matched decoys. For example, a single input ligand predicted to have two alternate charges would get 50 decoys property-matched to each charge. To accomplish this, a pool of decoys was selected from ZINC<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> using a dynamic protocol that adapted to local chemical space by narrowing or widening windows in seven steps around the six properties. The goal was to return 3000–9000 potential decoys that matched the decoy’s reference protonation state (predicted most prevalent form at pH 7.05). In the final decoy procedure, ECFP4 fingerprints were generated by Scitegic’s Pipeline Pilot for ligands and potential decoys. The decoys were sorted by their maximum <i>T</i><sub>c</sub> to any ligand, and the most dissimilar 25% were retained through this dissimilarity filter. We then remove duplicate decoys from the ligand set by sorting decoys from least to most duplicated and assigned each decoy to the protonated ligand which has the least number of decoys already assigned. This ensures unique decoys were spread across the ligands as evenly as possible. Finally, if available, 50 decoys were picked randomly from this deduplicated list.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> Original DUD Comparison</h3><div class="NLM_p last">For the original DUD comparison, we downloaded ligands and decoys from <a href="http://dud.docking.org" class="extLink">dud.docking.org</a> and prepared docking flexibases with our modern ZINC toolchain.<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> The original DUD target preparations were copies of the original, modified to perform SEV desolvation calculations as described previously.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> We also generated DUD-E style automated decoys and flexibases for the original DUD ligands. The analysis was performed on the 37 directly comparable targets, excluding the original targets PDGFrb, ER<sub>agonist</sub>, and ER<sub>antagonist</sub>.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> Docking Calculations</h3><div class="NLM_p last">Except as noted, docking calculations were performed with DOCK 3.6 and solvent-excluded volume (SEV) ligand desolvation as described previously.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> Ligand conformations were generated by OpenEye’s Omega.<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> For sampling, the minimum number of graph matching nodes was changed to 3, and ligand overlap was changed to 0.1. Ligands were limited to between 5 and 100 heavy atoms. The timeout for an individual ligand hierarchy was 180 s. We performed 200 steps of simplex minimization, with initial translations of 0.2 Å and initial rotations of 5°. The thin dielectric layer Delphi spheres were created by walking out each DMS (<a href="http://www.cgl.ucsf.edu/Overview/ftp/dms.zip" class="extLink">http://www.cgl.ucsf.edu/Overview/ftp/dms.zip</a>) surface normal by 1.8 Å and placing a sphere. This thin sphere layer is then used as input to makespheres1.pl in place of the usual SPHGEN spheres. The random background calculations were performed using SEV desolvation by seeding the DUD-E ligands into the entire ChEMBL12_10 subset of ZINC, which includes 273375 ligands with annotated affinities below 10 μM.</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> Docking Metrics</h3><div class="NLM_p last">The area under the curve (AUC) of the receiver operating characteristic (ROC) is one common metric to measure docking performance. However, ROC plots often use a semilog transformation of the <i>x</i>-axis to zoom in on early changes. As described previously,<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> LogAUC is completely analogous to AUC in this transformed space, measuring the percentage of the unit area under the curve. Formally, we use the adjusted LogAUC<sub>0.001</sub> here, which spans three decades of log space and subtracts the LogAUC of the random curve (14.462%) so that random enrichment is 0%. We typically refer to the adjusted LogAUC<sub>0.001</sub> as either adjusted LogAUC or simply LogAUC. The ROC-based enrichment factor at 1% (EF<sub>1</sub>) is the percent of ligands found when 1% of the decoys have been found and is preferred over traditional enrichment factors.<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i23"><a href="/doi/suppl/10.1021/jm300687e">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_30741" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_30741" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">Figure showing DUD-E workflows, while tables provide detailed target-by-target data and tab delimited text files provide the raw data. This material is available free of charge via the Internet at <a href="http://pubs.acs.org" class="extLink">http://pubs.acs.org</a>.</p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm300687e/suppl_file/jm300687e_si_001.pdf">jm300687e_si_001.pdf (298.46 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm300687e/suppl_file/jm300687e_si_002.pdf">jm300687e_si_002.pdf (289.74 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm300687e/suppl_file/jm300687e_si_003.txt">jm300687e_si_003.txt (9.45 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm300687e/suppl_file/jm300687e_si_004.txt">jm300687e_si_004.txt (23.06 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/jm300687e" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_26275" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_26275" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">John. J. Irwin</span> - <span class="hlFld-Affiliation affiliation">Department of Pharmaceutical
Chemistry, University of California San Francisco, San Francisco,
California 94158-2330, United States</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#9cf6f6f5dcfffbf0b2e9ffeffab2f9f8e9bceff4f3f5fff4f9e8dcfffbf0b2e9ffeffab2f9f8e9"><span class="__cf_email__" data-cfemail="d5bfbfbc95b6b2b9fba0b6a6b3fbb0b1a0">[email protected]</span> <span class="__cf_email__" data-cfemail="8ffce7e0e6ece7eafbcfece8e3a1faecfce9a1eaebfa">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Brian K. Shoichet</span> - <span class="hlFld-Affiliation affiliation">Department of Pharmaceutical
Chemistry, University of California San Francisco, San Francisco,
California 94158-2330, United States</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#e48e8e8da4878388ca91879782ca818091c4978c8b8d878c8190a4878388ca91879782ca818091"><span class="__cf_email__" data-cfemail="bdd7d7d4fddedad193c8decedb93d8d9c8">[email protected]</span> <span class="__cf_email__" data-cfemail="6b1803040208030e1f2b080c07451e08180d450e0f1e">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Michael M. Mysinger</span> - <span class="hlFld-Affiliation affiliation">Department of Pharmaceutical
Chemistry, University of California San Francisco, San Francisco,
California 94158-2330, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Michael Carchia</span> - <span class="hlFld-Affiliation affiliation">Department of Pharmaceutical
Chemistry, University of California San Francisco, San Francisco,
California 94158-2330, United States</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d1623e1640-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i24">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_18452" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_18452" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">Supported by NIH grant GM71896 (to J.J.I. and B.K.S.). We thank Andrew Good for discussions that initiated DUD-E. We thank Teague Sterling for website development and Sunil Koovakkat for DOCK bugfixes. We are grateful to the commercial software vendors who support ZINC and the decoy generation toolchain: Molinspiration (Bratislava, Slovakia) for mib, OpenEye Scientific Software (Santa Fe, NM) for OEChem, Omega, and QuacPac, Molecular Networks (Erlangen, Germany) for Corina, Accelrys (San Diego, CA) for Pipeline Pilot, and ChemAxon (Budapest, Hungary) for cxcalc. We thank Oliv Eidam, Matthew Merski, and Nir London for reading this manuscript.</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i25" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i25"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i26" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i26"> Abbreviations Used</h2><tr><td class="NLM_term">DUD</td><td class="NLM_def"><p class="first last">Directory of Useful Decoys</p></td></tr><tr><td class="NLM_term">DUD-E</td><td class="NLM_def"><p class="first last">Directory of Useful Decoys—Enhanced</p></td></tr><tr><td class="NLM_term">EF<sub>1</sub></td><td class="NLM_def"><p class="first last">enrichment factor at 1% of ROC curve</p></td></tr><tr><td class="NLM_term">PH</td><td class="NLM_def"><p class="first last">pleckstrin homology</p></td></tr><tr><td class="NLM_term">ROC</td><td class="NLM_def"><p class="first last">receiver operating characteristic</p></td></tr><tr><td class="NLM_term">SEV</td><td class="NLM_def"><p class="first last">solvent-excluded volume</p></td></tr><tr><td class="NLM_term"><i>T</i><sub>c</sub></td><td class="NLM_def"><p class="first last">Tanimoto coefficient</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i27">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_41374" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_41374" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 53 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Kitchen, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Decornez, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Furr, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bajorath, J.</span><span> </span><span class="NLM_article-title">Docking and scoring in virtual screening for drug discovery: methods and applications</span> <span class="citation_source-journal">Nature Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">935</span><span class="NLM_x">–</span> <span class="NLM_lpage">949</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2004&pages=935-949&author=D.+B.+Kitchenauthor=H.+Decornezauthor=J.+R.+Furrauthor=J.+Bajorath&title=Docking+and+scoring+in+virtual+screening+for+drug+discovery%3A+methods+and+applications"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKitchen%26aufirst%3DD.%2BB.%26aulast%3DDecornez%26aufirst%3DH.%26aulast%3DFurr%26aufirst%3DJ.%2BR.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DDocking%2520and%2520scoring%2520in%2520virtual%2520screening%2520for%2520drug%2520discovery%253A%2520methods%2520and%2520applications%26jtitle%3DNature%2520Rev.%2520Drug%2520Discovery%26date%3D2004%26volume%3D3%26spage%3D935%26epage%3D949" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Kolb, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosenbaum, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fung, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kobilka, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Structure-based discovery of beta(2)-adrenergic receptor ligands</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U.S.A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">106</span><span class="NLM_x">, </span> <span class="NLM_fpage">6843</span><span class="NLM_x">–</span> <span class="NLM_lpage">6848</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2009&pages=6843-6848&author=P.+Kolbauthor=D.+M.+Rosenbaumauthor=J.+J.+Irwinauthor=J.+J.+Fungauthor=B.+K.+Kobilkaauthor=B.+K.+Shoichet&title=Structure-based+discovery+of+beta%282%29-adrenergic+receptor+ligands"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKolb%26aufirst%3DP.%26aulast%3DRosenbaum%26aufirst%3DD.%2BM.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DFung%26aufirst%3DJ.%2BJ.%26aulast%3DKobilka%26aufirst%3DB.%2BK.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DStructure-based%2520discovery%2520of%2520beta%25282%2529-adrenergic%2520receptor%2520ligands%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2009%26volume%3D106%26spage%3D6843%26epage%3D6848" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Mysinger, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weiss, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ziarek, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gravel, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Doak, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karpiak, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heveker, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Volkman, B. F.</span><span> </span><span class="NLM_article-title">Structure-based ligand discovery for the protein–protein interface of chemokine receptor CXCR4</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U.S.A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">109</span><span class="NLM_x">, </span> <span class="NLM_fpage">5517</span><span class="NLM_x">–</span> <span class="NLM_lpage">5522</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2012&pages=5517-5522&author=M.+M.+Mysingerauthor=D.+R.+Weissauthor=J.+J.+Ziarekauthor=S.+Gravelauthor=A.+K.+Doakauthor=J.+Karpiakauthor=N.+Hevekerauthor=B.+K.+Shoichetauthor=B.+F.+Volkman&title=Structure-based+ligand+discovery+for+the+protein%E2%80%93protein+interface+of+chemokine+receptor+CXCR4"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMysinger%26aufirst%3DM.%2BM.%26aulast%3DWeiss%26aufirst%3DD.%2BR.%26aulast%3DZiarek%26aufirst%3DJ.%2BJ.%26aulast%3DGravel%26aufirst%3DS.%26aulast%3DDoak%26aufirst%3DA.%2BK.%26aulast%3DKarpiak%26aufirst%3DJ.%26aulast%3DHeveker%26aufirst%3DN.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DVolkman%26aufirst%3DB.%2BF.%26atitle%3DStructure-based%2520ligand%2520discovery%2520for%2520the%2520protein%25E2%2580%2593protein%2520interface%2520of%2520chemokine%2520receptor%2520CXCR4%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2012%26volume%3D109%26spage%3D5517%26epage%3D5522" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Gruneberg, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stubbs, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klebe, G.</span><span> </span><span class="NLM_article-title">Successful virtual screening for novel inhibitors of human carbonic anhydrase: strategy and experimental confirmation</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">3588</span><span class="NLM_x">–</span> <span class="NLM_lpage">3602</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm011112j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2002&pages=3588-3602&author=S.+Grunebergauthor=M.+T.+Stubbsauthor=G.+Klebe&title=Successful+virtual+screening+for+novel+inhibitors+of+human+carbonic+anhydrase%3A+strategy+and+experimental+confirmation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1021%2Fjm011112j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm011112j%26sid%3Dliteratum%253Aachs%26aulast%3DGruneberg%26aufirst%3DS.%26aulast%3DStubbs%26aufirst%3DM.%2BT.%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DSuccessful%2520virtual%2520screening%2520for%2520novel%2520inhibitors%2520of%2520human%2520carbonic%2520anhydrase%253A%2520strategy%2520and%2520experimental%2520confirmation%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2002%26volume%3D45%26spage%3D3588%26epage%3D3602" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Jain, A. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nicholls, A.</span><span> </span><span class="NLM_article-title">Recommendations for evaluation of computational methods</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">133</span><span class="NLM_x">–</span> <span class="NLM_lpage">139</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Fjm300687e&amp;key=10.1007%2Fs10822-008-9196-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Fjm300687e&amp;key=18338228" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Fjm300687e&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjsFOnsb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2008&pages=133-139&author=A.+N.+Jainauthor=A.+Nicholls&title=Recommendations+for+evaluation+of+computational+methods"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Recommendations for evaluation of computational methods</span></div><div class="casAuthors">Jain, Ajay N.; Nicholls, Anthony</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">3-4</span>),
    <span class="NLM_cas:pages">133-139</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  The field of computational chem., particularly as applied to drug design, has become increasingly important in terms of the practical application of predictive modeling to pharmaceutical research and development.  Tools for exploiting protein structures or sets of ligands known to bind particular targets can be used for binding-mode prediction, virtual screening, and prediction of activity.  A serious weakness within the field is a lack of stds. with respect to quant. evaluation of methods, data set prepn., and data set sharing.  Our goal should be to report new methods or comparative evaluations of methods in a manner that supports decision making for practical applications.  Here we propose a modest beginning, with recommendations for requirements on statistical reporting, requirements for data sharing, and best practices for benchmark prepn. and usage.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZyHVFOx399LVg90H21EOLACvtfcHk0lj1iSB3UsR-Jg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjsFOnsb0%253D&md5=9576e150079dcbf6e4bfea5070082016</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1007%2Fs10822-008-9196-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-008-9196-5%26sid%3Dliteratum%253Aachs%26aulast%3DJain%26aufirst%3DA.%2BN.%26aulast%3DNicholls%26aufirst%3DA.%26atitle%3DRecommendations%2520for%2520evaluation%2520of%2520computational%2520methods%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2008%26volume%3D22%26spage%3D133%26epage%3D139" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Babaoglu, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simeonov, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nelson, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thomas, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cancian, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Costi, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maltby, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jadhav, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Inglese, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Austin, C. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Comprehensive mechanistic analysis of hits from high-throughput and docking screens against beta-lactamase</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">2502</span><span class="NLM_x">–</span> <span class="NLM_lpage">2511</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm701500e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Fjm300687e&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjtFSksb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=2502-2511&author=K.+Babaogluauthor=A.+Simeonovauthor=J.+J.+Irwinauthor=M.+E.+Nelsonauthor=B.+Fengauthor=C.+J.+Thomasauthor=L.+Cancianauthor=M.+P.+Costiauthor=D.+A.+Maltbyauthor=A.+Jadhavauthor=J.+Ingleseauthor=C.+P.+Austinauthor=B.+K.+Shoichet&title=Comprehensive+mechanistic+analysis+of+hits+from+high-throughput+and+docking+screens+against+beta-lactamase"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Comprehensive Mechanistic Analysis of Hits from High-Throughput and Docking Screens against β-Lactamase</span></div><div class="casAuthors">Babaoglu, Kerim; Simeonov, Anton; Irwin, John J.; Nelson, Michael E.; Feng, Brian; Thomas, Craig J.; Cancian, Laura; Costi, M. Paola; Maltby, David A.; Jadhav, Ajit; Inglese, James; Austin, Christopher P.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2502-2511</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">High-throughput screening (HTS) is widely used in drug discovery.  Esp. for screens of unbiased libraries, false positives can dominate "hit lists"; their origins are much debated.  Here we det. the mechanism of every active hit from a screen of 70,563 unbiased mols. against β-lactamase using quant. HTS (qHTS).  Of the 1274 initial inhibitors, 95% were detergent-sensitive and were classified as aggregators.  Among the 70 remaining were 25 potent, covalent-acting β-lactams.  Mass spectra, counter-screens, and crystallog. identified 12 as promiscuous covalent inhibitors.  The remaining 33 were either aggregators or irreproducible.  No specific reversible inhibitors were found.  We turned to mol. docking to prioritize mols. from the same library for testing at higher concns.  Of 16 tested, 2 were modest inhibitors.  Subsequent X-ray structures corresponded to the docking prediction.  Analog synthesis improved affinity to 8 μM.  These results suggest that it may be the phys. behavior of org. mols., not their reactivity, that accounts for most screening artifacts.  Structure-based methods may prioritize weak-but-novel chemotypes in unbiased library screens.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpNtnaCegS8aLVg90H21EOLACvtfcHk0ljPMYN2Vj9dHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjtFSksb0%253D&md5=7736429aeafc6b17a5d0182e24df0dd4</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1021%2Fjm701500e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm701500e%26sid%3Dliteratum%253Aachs%26aulast%3DBabaoglu%26aufirst%3DK.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DNelson%26aufirst%3DM.%2BE.%26aulast%3DFeng%26aufirst%3DB.%26aulast%3DThomas%26aufirst%3DC.%2BJ.%26aulast%3DCancian%26aufirst%3DL.%26aulast%3DCosti%26aufirst%3DM.%2BP.%26aulast%3DMaltby%26aufirst%3DD.%2BA.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DInglese%26aufirst%3DJ.%26aulast%3DAustin%26aufirst%3DC.%2BP.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DComprehensive%2520mechanistic%2520analysis%2520of%2520hits%2520from%2520high-throughput%2520and%2520docking%2520screens%2520against%2520beta-lactamase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D2502%26epage%3D2511" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Ferreira, R. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simeonov, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jadhav, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eidam, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mott, B. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keiser, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McKerrow, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maloney, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Complementarity between a docking and a high-throughput screen in discovering new cruzain inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">4891</span><span class="NLM_x">–</span> <span class="NLM_lpage">4905</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm100488w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Fjm300687e&amp;key=1%3ACAS%3A528%3ADC%252BC3cXntlWrurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=4891-4905&author=R.+S.+Ferreiraauthor=A.+Simeonovauthor=A.+Jadhavauthor=O.+Eidamauthor=B.+T.+Mottauthor=M.+J.+Keiserauthor=J.+H.+McKerrowauthor=D.+J.+Maloneyauthor=J.+J.+Irwinauthor=B.+K.+Shoichet&title=Complementarity+between+a+docking+and+a+high-throughput+screen+in+discovering+new+cruzain+inhibitors"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Complementarity Between a Docking and a High-Throughput Screen in Discovering New Cruzain Inhibitors</span></div><div class="casAuthors">Ferreira, Rafaela S.; Simeonov, Anton; Jadhav, Ajit; Eidam, Oliv; Mott, Bryan T.; Keiser, Michael J.; McKerrow, James H.; Maloney, David J.; Irwin, John J.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">4891-4905</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Virtual and high-throughput screens (HTS) should have complementary strengths and weaknesses, but studies that prospectively and comprehensively compare them are rare.  We undertook a parallel docking and HTS screen of 197861 compds. against cruzain, a thiol protease target for Chagas disease, looking for reversible, competitive inhibitors.  On workup, 99% of the hits were eliminated as false positives, yielding 146 well-behaved, competitive ligands.  These fell into five chemotypes: two were prioritized by scoring among the top 0.1% of the docking-ranked library, two were prioritized by behavior in the HTS and by clustering, and one chemotype was prioritized by both approaches.  Detn. of an inhibitor/cruzain crystal structure and comparison of the high-scoring docking hits to expt. illuminated the origins of docking false-negatives and false-positives.  Prioritizing mols. that are both predicted by docking and are HTS-active yields well-behaved mols., relatively unobscured by the false-positives to which both techniques are individually prone.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmmoDO-1mm8bVg90H21EOLACvtfcHk0ljPMYN2Vj9dHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXntlWrurk%253D&md5=0661663ff0802bf981a1997eeca9fa05</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1021%2Fjm100488w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm100488w%26sid%3Dliteratum%253Aachs%26aulast%3DFerreira%26aufirst%3DR.%2BS.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DEidam%26aufirst%3DO.%26aulast%3DMott%26aufirst%3DB.%2BT.%26aulast%3DKeiser%26aufirst%3DM.%2BJ.%26aulast%3DMcKerrow%26aufirst%3DJ.%2BH.%26aulast%3DMaloney%26aufirst%3DD.%2BJ.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DComplementarity%2520between%2520a%2520docking%2520and%2520a%2520high-throughput%2520screen%2520in%2520discovering%2520new%2520cruzain%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D4891%26epage%3D4905" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Gohlke, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klebe, G.</span><span> </span><span class="NLM_article-title">Approaches to the description and prediction of the binding affinity of small-molecule ligands to macromolecular receptors</span> <span class="citation_source-journal">Angew. Chem., Int. Ed. Engl.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">41</span><span class="NLM_x">, </span> <span class="NLM_fpage">2644</span><span class="NLM_x">–</span> <span class="NLM_lpage">2676</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2002&pages=2644-2676&author=H.+Gohlkeauthor=G.+Klebe&title=Approaches+to+the+description+and+prediction+of+the+binding+affinity+of+small-molecule+ligands+to+macromolecular+receptors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGohlke%26aufirst%3DH.%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DApproaches%2520to%2520the%2520description%2520and%2520prediction%2520of%2520the%2520binding%2520affinity%2520of%2520small-molecule%2520ligands%2520to%2520macromolecular%2520receptors%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%2520Engl.%26date%3D2002%26volume%3D41%26spage%3D2644%26epage%3D2676" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Enyedy, I. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Egan, W. J.</span><span> </span><span class="NLM_article-title">Can we use docking and scoring for hit-to-lead optimization?</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">161</span><span class="NLM_x">–</span> <span class="NLM_lpage">168</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2008&pages=161-168&author=I.+J.+Enyedyauthor=W.+J.+Egan&title=Can+we+use+docking+and+scoring+for+hit-to-lead+optimization%3F"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DEnyedy%26aufirst%3DI.%2BJ.%26aulast%3DEgan%26aufirst%3DW.%2BJ.%26atitle%3DCan%2520we%2520use%2520docking%2520and%2520scoring%2520for%2520hit-to-lead%2520optimization%253F%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2008%26volume%3D22%26spage%3D161%26epage%3D168" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Stahl, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rarey, M.</span><span> </span><span class="NLM_article-title">Detailed analysis of scoring functions for virtual screening</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">44</span><span class="NLM_x">, </span> <span class="NLM_fpage">1035</span><span class="NLM_x">–</span> <span class="NLM_lpage">1042</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm0003992" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2001&pages=1035-1042&author=M.+Stahlauthor=M.+Rarey&title=Detailed+analysis+of+scoring+functions+for+virtual+screening"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Fjm0003992&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0003992%26sid%3Dliteratum%253Aachs%26aulast%3DStahl%26aufirst%3DM.%26aulast%3DRarey%26aufirst%3DM.%26atitle%3DDetailed%2520analysis%2520of%2520scoring%2520functions%2520for%2520virtual%2520screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2001%26volume%3D44%26spage%3D1035%26epage%3D1042" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Bissantz, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Folkers, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rognan, D.</span><span> </span><span class="NLM_article-title">Protein-based virtual screening of chemical databases. 1. Evaluation of different docking/scoring combinations</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">43</span><span class="NLM_x">, </span> <span class="NLM_fpage">4759</span><span class="NLM_x">–</span> <span class="NLM_lpage">4767</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm001044l" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2000&pages=4759-4767&author=C.+Bissantzauthor=G.+Folkersauthor=D.+Rognan&title=Protein-based+virtual+screening+of+chemical+databases.+1.+Evaluation+of+different+docking%2Fscoring+combinations"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Fjm001044l&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm001044l%26sid%3Dliteratum%253Aachs%26aulast%3DBissantz%26aufirst%3DC.%26aulast%3DFolkers%26aufirst%3DG.%26aulast%3DRognan%26aufirst%3DD.%26atitle%3DProtein-based%2520virtual%2520screening%2520of%2520chemical%2520databases.%25201.%2520Evaluation%2520of%2520different%2520docking%252Fscoring%2520combinations%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2000%26volume%3D43%26spage%3D4759%26epage%3D4767" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Pham, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jain, A. N.</span><span> </span><span class="NLM_article-title">Parameter estimation for scoring protein–ligand interactions using negative training data</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">5856</span><span class="NLM_x">–</span> <span class="NLM_lpage">5868</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm050040j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=5856-5868&author=T.+A.+Phamauthor=A.+N.+Jain&title=Parameter+estimation+for+scoring+protein%E2%80%93ligand+interactions+using+negative+training+data"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Fjm050040j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm050040j%26sid%3Dliteratum%253Aachs%26aulast%3DPham%26aufirst%3DT.%2BA.%26aulast%3DJain%26aufirst%3DA.%2BN.%26atitle%3DParameter%2520estimation%2520for%2520scoring%2520protein%25E2%2580%2593ligand%2520interactions%2520using%2520negative%2520training%2520data%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D5856%26epage%3D5868" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Kellenberger, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rodrigo, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muller, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rognan, D.</span><span> </span><span class="NLM_article-title">Comparative evaluation of eight docking tools for docking and virtual screening accuracy</span> <span class="citation_source-journal">Proteins</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">225</span><span class="NLM_x">–</span> <span class="NLM_lpage">242</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2004&pages=225-242&author=E.+Kellenbergerauthor=J.+Rodrigoauthor=P.+Mullerauthor=D.+Rognan&title=Comparative+evaluation+of+eight+docking+tools+for+docking+and+virtual+screening+accuracy"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKellenberger%26aufirst%3DE.%26aulast%3DRodrigo%26aufirst%3DJ.%26aulast%3DMuller%26aufirst%3DP.%26aulast%3DRognan%26aufirst%3DD.%26atitle%3DComparative%2520evaluation%2520of%2520eight%2520docking%2520tools%2520for%2520docking%2520and%2520virtual%2520screening%2520accuracy%26jtitle%3DProteins%26date%3D2004%26volume%3D57%26spage%3D225%26epage%3D242" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Ferrara, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gohlke, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Price, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klebe, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brooks, C. L.,  III.</span><span> </span><span class="NLM_article-title">Assessing scoring functions for protein–ligand interactions</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">3032</span><span class="NLM_x">–</span> <span class="NLM_lpage">3047</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm030489h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=3032-3047&author=P.+Ferraraauthor=H.+Gohlkeauthor=D.+J.+Priceauthor=G.+Klebeauthor=C.+L.+Brooks&title=Assessing+scoring+functions+for+protein%E2%80%93ligand+interactions"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Fjm030489h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm030489h%26sid%3Dliteratum%253Aachs%26aulast%3DFerrara%26aufirst%3DP.%26aulast%3DGohlke%26aufirst%3DH.%26aulast%3DPrice%26aufirst%3DD.%2BJ.%26aulast%3DKlebe%26aufirst%3DG.%26aulast%3DBrooks%26aufirst%3DC.%2BL.%26atitle%3DAssessing%2520scoring%2520functions%2520for%2520protein%25E2%2580%2593ligand%2520interactions%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D3032%26epage%3D3047" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Huang, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span> </span><span class="NLM_article-title">Benchmarking sets for molecular docking</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">6789</span><span class="NLM_x">–</span> <span class="NLM_lpage">6801</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm0608356" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Fjm300687e&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtFehurzI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=6789-6801&author=N.+Huangauthor=B.+K.+Shoichetauthor=J.+J.+Irwin&title=Benchmarking+sets+for+molecular+docking"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Benchmarking Sets for Molecular Docking</span></div><div class="casAuthors">Huang, Niu; Shoichet, Brian K.; Irwin, John J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">6789-6801</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Ligand enrichment among top-ranking hits is a key metric of mol. docking.  To avoid bias, decoys should resemble ligands phys., so that enrichment is not simply a sepn. of gross features, yet be chem. distinct from them, so that they are unlikely to be binders.  We have assembled a directory of useful decoys (DUD), with 2950 ligands for 40 different targets.  Every ligand has 36 decoy mols. that are phys. similar but topol. distinct, leading to a database of 98 266 compds.  For most targets, enrichment was at least half a log better with uncorrected databases such as the MDDR than with DUD, evidence of bias in the former.  These calcns. also allowed 40×40 cross-docking, where the enrichments of each ligand set could be compared for all 40 targets, enabling a specificity metric for the docking screens.  DUD is freely available online as a benchmarking set for docking at http://blaster.docking.org/dud/.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-8FVhVSsBm7Vg90H21EOLACvtfcHk0lgIq0sf4Di_XQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtFehurzI&md5=43af46968c2a6d66334922bf0caedbe1</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Fjm0608356&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0608356%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DN.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26atitle%3DBenchmarking%2520sets%2520for%2520molecular%2520docking%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D6789%26epage%3D6801" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Christofferson, A. J.; Huang, N.</span><span> </span><span class="NLM_article-title">How to benchmark methods for structure-based virtual screening of large compound libraries</span>. In  <span class="citation_source-book">Computational Drug Discovery and Design (Methods in Molecular Biology)</span>; <span class="NLM_edition">2011/12/21</span> ed.; <span class="NLM_contrib-group">Baron, R.</span>, Ed.; <span class="NLM_publisher-name">Springer Protocols</span>: <span class="NLM_publisher-loc">New York</span>,<span class="NLM_x"> </span><span class="NLM_year">2012</span><span class="NLM_x">; </span>Vol.  <span class="NLM_volume">819</span>, Chapter 13, pp  <span class="NLM_fpage">187</span><span class="NLM_x">–</span> <span class="NLM_lpage">195</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Fjm300687e&amp;key=10.1007%2F978-1-61779-465-0_13" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&pages=187-195&author=A.+J.+Christofferson&author=N.+Huangauthor=R.+Baron&title=Computational+Drug+Discovery+and+Design+%28Methods+in+Molecular+Biology%29"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1007%2F978-1-61779-465-0_13&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F978-1-61779-465-0_13%26sid%3Dliteratum%253Aachs%26aulast%3DChristofferson%26aufirst%3DA.%2BJ.%26atitle%3DHow%2520to%2520benchmark%2520methods%2520for%2520structure-based%2520virtual%2520screening%2520of%2520large%2520compound%2520libraries%26btitle%3DComputational%2520Drug%2520Discovery%2520and%2520Design%2520%2528Methods%2520in%2520Molecular%2520Biology%2529%26aulast%3DBaron%26aufirst%3DR.%26pub%3DSpringer%2520Protocols%26date%3D2012%26volume%3D819%26spage%3D187%26epage%3D195" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Verdonk, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berdini, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hartshorn, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mooij, W. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, R. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Watson, P.</span><span> </span><span class="NLM_article-title">Virtual screening using protein–ligand docking: avoiding artificial enrichment</span> <span class="citation_source-journal">J. Chem. Inf. Comput. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">44</span><span class="NLM_x">, </span> <span class="NLM_fpage">793</span><span class="NLM_x">–</span> <span class="NLM_lpage">806</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci034289q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Fjm300687e&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVGhs70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2004&pages=793-806&author=M.+L.+Verdonkauthor=V.+Berdiniauthor=M.+J.+Hartshornauthor=W.+T.+Mooijauthor=C.+W.+Murrayauthor=R.+D.+Taylorauthor=P.+Watson&title=Virtual+screening+using+protein%E2%80%93ligand+docking%3A+avoiding+artificial+enrichment"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Virtual screening using protein-ligand docking: Avoiding artificial enrichment</span></div><div class="casAuthors">Verdonk, Marcel L.; Berdini, Valerio; Hartshorn, Michael J.; Mooij, Wijnand T. M.; Murray, Christopher W.; Taylor, Richard D.; Watson, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Computer Sciences</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">793-806</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">0095-2338</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This study addresses a no. of topical issues around the use of protein-ligand docking in virtual screening.  We show that, for the validation of such methods, it is key to use focused libraries (contg. compds. with one-dimensional properties, similar to the actives), rather than "random" or "drug-like" libraries to test the actives against.  We also show that, to obtain good enrichments, the docking program needs to produce reliable binding modes.  We demonstrate how pharmacophores can be used to guide the dockings and improve enrichments, and we compare the performance of three consensus-ranking protocols against ranking based on individual scoring functions.  Finally, we show that protein-ligand docking can be an effective aid in the screening for weak, fragment-like binders, which has rapidly become a popular strategy for hit identification.  All results presented are based on carefully constructed virtual screening expts. against four targets, using the protein-ligand docking program GOLD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqRRkYiB4qsvbVg90H21EOLACvtfcHk0lgIq0sf4Di_XQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVGhs70%253D&md5=b0cd9e9a04ddebe401dee9744e2184cb</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fci034289q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci034289q%26sid%3Dliteratum%253Aachs%26aulast%3DVerdonk%26aufirst%3DM.%2BL.%26aulast%3DBerdini%26aufirst%3DV.%26aulast%3DHartshorn%26aufirst%3DM.%2BJ.%26aulast%3DMooij%26aufirst%3DW.%2BT.%26aulast%3DMurray%26aufirst%3DC.%2BW.%26aulast%3DTaylor%26aufirst%3DR.%2BD.%26aulast%3DWatson%26aufirst%3DP.%26atitle%3DVirtual%2520screening%2520using%2520protein%25E2%2580%2593ligand%2520docking%253A%2520avoiding%2520artificial%2520enrichment%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D2004%26volume%3D44%26spage%3D793%26epage%3D806" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Kuntz, I. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sharp, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kollman, P. A.</span><span> </span><span class="NLM_article-title">The maximal affinity of ligands</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U.S.A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">96</span><span class="NLM_x">, </span> <span class="NLM_fpage">9997</span><span class="NLM_x">–</span> <span class="NLM_lpage">10002</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Fjm300687e&amp;key=10.1073%2Fpnas.96.18.9997" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Fjm300687e&amp;key=10468550" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Fjm300687e&amp;key=1%3ACAS%3A528%3ADyaK1MXlvFehu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=96&publication_year=1999&pages=9997-10002&author=I.+D.+Kuntzauthor=K.+Chenauthor=K.+A.+Sharpauthor=P.+A.+Kollman&title=The+maximal+affinity+of+ligands"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">The maximal affinity of ligands</span></div><div class="casAuthors">Kuntz, I. D.; Chen, K.; Sharp, K. A.; Kollman, P. A.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">96</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">9997-10002</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">We explore the question of what are the best ligands for macromol. targets.  A survey of exptl. data on a large no. of the strongest-binding ligands indicates that the free energy of binding increases with the no. of nonhydrogen atoms with an initial slope of ≈-1.5 kcal/mol (1 cal = 4.18 J) per atom.  For ligands that contain more than 15 nonhydrogen atoms, the free energy of binding increases very little with relative mol. mass.  This nonlinearity is largely ascribed to nonthermodynamic factors.  An anal. of the dominant interactions suggests that van der Waals interactions and hydrophobic effects provide a reasonable basis for understanding binding affinities across the entire set of ligands.  Interesting outliers that bind unusually strongly on a per atom basis include metal ions, covalently attached ligands, and a few well known complexes such as biotin-avidin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0L6KZhR0TxbVg90H21EOLACvtfcHk0lhmworAD-yHow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXlvFehu7s%253D&md5=46fe894b016f3041831a2f0b71f812b1</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1073%2Fpnas.96.18.9997&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.96.18.9997%26sid%3Dliteratum%253Aachs%26aulast%3DKuntz%26aufirst%3DI.%2BD.%26aulast%3DChen%26aufirst%3DK.%26aulast%3DSharp%26aufirst%3DK.%2BA.%26aulast%3DKollman%26aufirst%3DP.%2BA.%26atitle%3DThe%2520maximal%2520affinity%2520of%2520ligands%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D1999%26volume%3D96%26spage%3D9997%26epage%3D10002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Fan, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Webb, B. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klebe, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sali, A.</span><span> </span><span class="NLM_article-title">Molecular Docking Screens Using Comparative Models of Proteins</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">2512</span><span class="NLM_x">–</span> <span class="NLM_lpage">2527</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci9003706" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Fjm300687e&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1yntL7M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2009&pages=2512-2527&author=H.+Fanauthor=J.+J.+Irwinauthor=B.+M.+Webbauthor=G.+Klebeauthor=B.+K.+Shoichetauthor=A.+Sali&title=Molecular+Docking+Screens+Using+Comparative+Models+of+Proteins"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Docking Screens Using Comparative Models of Proteins</span></div><div class="casAuthors">Fan, Hao; Irwin, John J.; Webb, Benjamin M.; Klebe, Gerhard; Shoichet, Brian K.; Sali, Andrej</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2512-2527</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Two orders of magnitude more protein sequences can be modeled by comparative modeling than have been detd. by X-ray crystallog. and NMR spectroscopy.  Investigators have nevertheless been cautious about using comparative models for ligand discovery because of concerns about model errors.  We suggest how to exploit comparative models for mol. screens, based on docking against a wide range of crystallog. structures and comparative models with known ligands.  To account for the variation in the ligand-binding pocket as it binds different ligands, we calc. "consensus" enrichment by ranking each library compd. by its best docking score against all available comparative models and/or modeling templates.  For the majority of the targets, the consensus enrichment for multiple models was better than or comparable to that of the holo and apo X-ray structures.  Even for single models, the models are significantly more enriching than the template structure if the template is paralogous and shares more than 25% sequence identity with the target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrrgXhylX-BJrVg90H21EOLACvtfcHk0lhmworAD-yHow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht1yntL7M&md5=11bd5284c6b43592906e45b6aea3019d</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Fci9003706&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci9003706%26sid%3Dliteratum%253Aachs%26aulast%3DFan%26aufirst%3DH.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DWebb%26aufirst%3DB.%2BM.%26aulast%3DKlebe%26aufirst%3DG.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DSali%26aufirst%3DA.%26atitle%3DMolecular%2520Docking%2520Screens%2520Using%2520Comparative%2520Models%2520of%2520Proteins%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2009%26volume%3D49%26spage%3D2512%26epage%3D2527" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Repasky, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murphy, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banks, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greenwood, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tubert-Brohman, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bhat, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friesner, R. A.</span><span> </span><span class="NLM_article-title">Docking performance of the glide program as evaluated on the Astex and DUD datasets: a complete set of glide SP results and selected results for a new scoring function integrating WaterMap and glide</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span><span class="NLM_elocation-id"></span><span class="refDoi"> DOI: 10.1007/s10822-012-9575-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Fjm300687e&amp;key=10.1007%2Fs10822-012-9575-9" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&author=M.+P.+Repaskyauthor=R.+B.+Murphyauthor=J.+L.+Banksauthor=J.+R.+Greenwoodauthor=I.+Tubert-Brohmanauthor=S.+Bhatauthor=R.+A.+Friesner&title=Docking+performance+of+the+glide+program+as+evaluated+on+the+Astex+and+DUD+datasets%3A+a+complete+set+of+glide+SP+results+and+selected+results+for+a+new+scoring+function+integrating+WaterMap+and+glide&doi=10.1007%2Fs10822-012-9575-9"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1007%2Fs10822-012-9575-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-012-9575-9%26sid%3Dliteratum%253Aachs%26aulast%3DRepasky%26aufirst%3DM.%2BP.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DBanks%26aufirst%3DJ.%2BL.%26aulast%3DGreenwood%26aufirst%3DJ.%2BR.%26aulast%3DTubert-Brohman%26aufirst%3DI.%26aulast%3DBhat%26aufirst%3DS.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26atitle%3DDocking%2520performance%2520of%2520the%2520glide%2520program%2520as%2520evaluated%2520on%2520the%2520Astex%2520and%2520DUD%2520datasets%253A%2520a%2520complete%2520set%2520of%2520glide%2520SP%2520results%2520and%2520selected%2520results%2520for%2520a%2520new%2520scoring%2520function%2520integrating%2520WaterMap%2520and%2520glide%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2012%26doi%3D10.1007%2Fs10822-012-9575-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Brozell, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mukherjee, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balius, T. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roe, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Case, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rizzo, R. C.</span><span> </span><span class="NLM_article-title">Evaluation of DOCK 6 as a pose generation and database enrichment tool</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span><span class="NLM_elocation-id"></span><span class="refDoi"> DOI: 10.1007/s10822-012-9565-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Fjm300687e&amp;key=10.1007%2Fs10822-012-9565-y" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&author=S.+R.+Brozellauthor=S.+Mukherjeeauthor=T.+E.+Baliusauthor=D.+R.+Roeauthor=D.+A.+Caseauthor=R.+C.+Rizzo&title=Evaluation+of+DOCK+6+as+a+pose+generation+and+database+enrichment+tool&doi=10.1007%2Fs10822-012-9565-y"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1007%2Fs10822-012-9565-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-012-9565-y%26sid%3Dliteratum%253Aachs%26aulast%3DBrozell%26aufirst%3DS.%2BR.%26aulast%3DMukherjee%26aufirst%3DS.%26aulast%3DBalius%26aufirst%3DT.%2BE.%26aulast%3DRoe%26aufirst%3DD.%2BR.%26aulast%3DCase%26aufirst%3DD.%2BA.%26aulast%3DRizzo%26aufirst%3DR.%2BC.%26atitle%3DEvaluation%2520of%2520DOCK%25206%2520as%2520a%2520pose%2520generation%2520and%2520database%2520enrichment%2520tool%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2012%26doi%3D10.1007%2Fs10822-012-9565-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Neves, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Totrov, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abagyan, R.</span><span> </span><span class="NLM_article-title">Docking and scoring with ICM: the benchmarking results and strategies for improvement</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span><span class="NLM_elocation-id"></span><span class="refDoi"> DOI: 10.1007/s10822-012-9547-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Fjm300687e&amp;key=10.1007%2Fs10822-012-9547-0" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&author=M.+A.+Nevesauthor=M.+Totrovauthor=R.+Abagyan&title=Docking+and+scoring+with+ICM%3A+the+benchmarking+results+and+strategies+for+improvement&doi=10.1007%2Fs10822-012-9547-0"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1007%2Fs10822-012-9547-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-012-9547-0%26sid%3Dliteratum%253Aachs%26aulast%3DNeves%26aufirst%3DM.%2BA.%26aulast%3DTotrov%26aufirst%3DM.%26aulast%3DAbagyan%26aufirst%3DR.%26atitle%3DDocking%2520and%2520scoring%2520with%2520ICM%253A%2520the%2520benchmarking%2520results%2520and%2520strategies%2520for%2520improvement%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2012%26doi%3D10.1007%2Fs10822-012-9547-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Spitzer, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jain, A. N.</span><span> </span><span class="NLM_article-title">Surflex-Dock: docking benchmarks and real-world application</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span><span class="NLM_elocation-id"></span><span class="refDoi"> DOI: 10.1007/s10822-011-9533-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Fjm300687e&amp;key=10.1007%2Fs10822-011-9533-y" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&author=R.+Spitzerauthor=A.+N.+Jain&title=Surflex-Dock%3A+docking+benchmarks+and+real-world+application&doi=10.1007%2Fs10822-011-9533-y"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1007%2Fs10822-011-9533-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-011-9533-y%26sid%3Dliteratum%253Aachs%26aulast%3DSpitzer%26aufirst%3DR.%26aulast%3DJain%26aufirst%3DA.%2BN.%26atitle%3DSurflex-Dock%253A%2520docking%2520benchmarks%2520and%2520real-world%2520application%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2012%26doi%3D10.1007%2Fs10822-011-9533-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Schneider, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hindle, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lange, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klein, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Albrecht, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Briem, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beyer, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Claussen, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gastreich, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lemmen, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rarey, M.</span><span> </span><span class="NLM_article-title">Substantial improvements in large-scale redocking and screening using the novel HYDE scoring function</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span><span class="NLM_elocation-id"></span><span class="refDoi"> DOI: 10.1007/s10822-011-9531-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Fjm300687e&amp;key=10.1007%2Fs10822-011-9531-0" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2011&author=N.+Schneiderauthor=S.+Hindleauthor=G.+Langeauthor=R.+Kleinauthor=J.+Albrechtauthor=H.+Briemauthor=K.+Beyerauthor=H.+Claussenauthor=M.+Gastreichauthor=C.+Lemmenauthor=M.+Rarey&title=Substantial+improvements+in+large-scale+redocking+and+screening+using+the+novel+HYDE+scoring+function&doi=10.1007%2Fs10822-011-9531-0"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1007%2Fs10822-011-9531-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-011-9531-0%26sid%3Dliteratum%253Aachs%26aulast%3DSchneider%26aufirst%3DN.%26aulast%3DHindle%26aufirst%3DS.%26aulast%3DLange%26aufirst%3DG.%26aulast%3DKlein%26aufirst%3DR.%26aulast%3DAlbrecht%26aufirst%3DJ.%26aulast%3DBriem%26aufirst%3DH.%26aulast%3DBeyer%26aufirst%3DK.%26aulast%3DClaussen%26aufirst%3DH.%26aulast%3DGastreich%26aufirst%3DM.%26aulast%3DLemmen%26aufirst%3DC.%26aulast%3DRarey%26aufirst%3DM.%26atitle%3DSubstantial%2520improvements%2520in%2520large-scale%2520redocking%2520and%2520screening%2520using%2520the%2520novel%2520HYDE%2520scoring%2520function%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2011%26doi%3D10.1007%2Fs10822-011-9531-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Liebeschuetz, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cole, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korb, O.</span><span> </span><span class="NLM_article-title">Pose prediction and virtual screening performance of GOLD scoring functions in a standardized test</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span><span class="NLM_elocation-id"></span><span class="refDoi"> DOI: 10.1007/s10822-012-9551-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Fjm300687e&amp;key=10.1007%2Fs10822-012-9551-4" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&author=J.+W.+Liebeschuetzauthor=J.+C.+Coleauthor=O.+Korb&title=Pose+prediction+and+virtual+screening+performance+of+GOLD+scoring+functions+in+a+standardized+test&doi=10.1007%2Fs10822-012-9551-4"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1007%2Fs10822-012-9551-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-012-9551-4%26sid%3Dliteratum%253Aachs%26aulast%3DLiebeschuetz%26aufirst%3DJ.%2BW.%26aulast%3DCole%26aufirst%3DJ.%2BC.%26aulast%3DKorb%26aufirst%3DO.%26atitle%3DPose%2520prediction%2520and%2520virtual%2520screening%2520performance%2520of%2520GOLD%2520scoring%2520functions%2520in%2520a%2520standardized%2520test%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2012%26doi%3D10.1007%2Fs10822-012-9551-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Novikov, F. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stroylov, V. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zeifman, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stroganov, O. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kulkov, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chilov, G. G.</span><span> </span><span class="NLM_article-title">Lead Finder docking and virtual screening evaluation with Astex and DUD test sets</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span><span class="NLM_elocation-id"></span><span class="refDoi"> DOI: 10.1007/s10822-012-9549-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Fjm300687e&amp;key=10.1007%2Fs10822-012-9549-y" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&author=F.+N.+Novikovauthor=V.+S.+Stroylovauthor=A.+A.+Zeifmanauthor=O.+V.+Stroganovauthor=V.+Kulkovauthor=G.+G.+Chilov&title=Lead+Finder+docking+and+virtual+screening+evaluation+with+Astex+and+DUD+test+sets&doi=10.1007%2Fs10822-012-9549-y"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1007%2Fs10822-012-9549-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-012-9549-y%26sid%3Dliteratum%253Aachs%26aulast%3DNovikov%26aufirst%3DF.%2BN.%26aulast%3DStroylov%26aufirst%3DV.%2BS.%26aulast%3DZeifman%26aufirst%3DA.%2BA.%26aulast%3DStroganov%26aufirst%3DO.%2BV.%26aulast%3DKulkov%26aufirst%3DV.%26aulast%3DChilov%26aufirst%3DG.%2BG.%26atitle%3DLead%2520Finder%2520docking%2520and%2520virtual%2520screening%2520evaluation%2520with%2520Astex%2520and%2520DUD%2520test%2520sets%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2012%26doi%3D10.1007%2Fs10822-012-9549-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Good, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oprea, T. I.</span><span> </span><span class="NLM_article-title">Optimization of CAMD techniques 3. Virtual screening enrichment studies: a help or hindrance in tool selection?</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">169</span><span class="NLM_x">–</span> <span class="NLM_lpage">178</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2008&pages=169-178&author=A.+C.+Goodauthor=T.+I.+Oprea&title=Optimization+of+CAMD+techniques+3.+Virtual+screening+enrichment+studies%3A+a+help+or+hindrance+in+tool+selection%3F"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGood%26aufirst%3DA.%2BC.%26aulast%3DOprea%26aufirst%3DT.%2BI.%26atitle%3DOptimization%2520of%2520CAMD%2520techniques%25203.%2520Virtual%2520screening%2520enrichment%2520studies%253A%2520a%2520help%2520or%2520hindrance%2520in%2520tool%2520selection%253F%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2008%26volume%3D22%26spage%3D169%26epage%3D178" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Mackey, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Melville, J. L.</span><span> </span><span class="NLM_article-title">Better than random? The chemotype enrichment problem</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">1154</span><span class="NLM_x">–</span> <span class="NLM_lpage">1162</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci8003978" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Fjm300687e&amp;key=1%3ACAS%3A528%3ADC%252BD1MXltVCgs78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2009&pages=1154-1162&author=M.+D.+Mackeyauthor=J.+L.+Melville&title=Better+than+random%3F+The+chemotype+enrichment+problem"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Better than Random? The Chemotype Enrichment Problem</span></div><div class="casAuthors">Mackey, Mark D.; Melville, James L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1154-1162</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Chemotype enrichment is increasingly recognized as an important measure of virtual screening performance.  However, little attention has been paid to producing metrics which can quantify chemotype retrieval.  Here, we examine two different protocols for analyzing chemotype retrieval: "cluster averaging", where the contribution of each active to the scoring metric is proportional to the no. of other actives with the same chemotype, and "first found", where only the first active for a given chemotype contributes to the score.  We demonstrate that this latter anal., common in the qual. anal. used in the current literature, has important drawbacks when combined with quant. metrics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrFRLJq6JcTO7Vg90H21EOLACvtfcHk0ljbwb4yVPZhhg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXltVCgs78%253D&md5=51b6efb624c3a383b7546997c11ab1e0</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fci8003978&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci8003978%26sid%3Dliteratum%253Aachs%26aulast%3DMackey%26aufirst%3DM.%2BD.%26aulast%3DMelville%26aufirst%3DJ.%2BL.%26atitle%3DBetter%2520than%2520random%253F%2520The%2520chemotype%2520enrichment%2520problem%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2009%26volume%3D49%26spage%3D1154%26epage%3D1162" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">Hawkins, P. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Warren, G. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Skillman, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nicholls, A.</span><span> </span><span class="NLM_article-title">How to do an evaluation: pitfalls and traps</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">179</span><span class="NLM_x">–</span> <span class="NLM_lpage">190</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Fjm300687e&amp;key=10.1007%2Fs10822-007-9166-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Fjm300687e&amp;key=18217218" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Fjm300687e&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjsFOnsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2008&pages=179-190&author=P.+C.+Hawkinsauthor=G.+L.+Warrenauthor=A.+G.+Skillmanauthor=A.+Nicholls&title=How+to+do+an+evaluation%3A+pitfalls+and+traps"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">How to do an evaluation: pitfalls and traps</span></div><div class="casAuthors">Hawkins, Paul C. D.; Warren, Gregory L.; Skillman, A. Geoffrey; Nicholls, Anthony</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">3-4</span>),
    <span class="NLM_cas:pages">179-190</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">The recent literature is replete with papers evaluating computational tools (often those operating on 3D structures) for their performance in a certain set of tasks.  Most commonly these papers compare a no. of docking tools for their performance in cognate re-docking (pose prediction) and/or virtual screening.  Related papers have been published on ligand-based tools: pose prediction by conformer generators and virtual screening using a variety of ligand-based approaches.  The reliability of these comparisons is critically affected by a no. of factors usually ignored by the authors, including bias in the datasets used in virtual screening, the metrics used to assess performance in virtual screening and pose prediction and errors in crystal structures used.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoyuwDKbJAiRLVg90H21EOLACvtfcHk0ljbwb4yVPZhhg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjsFOnsLk%253D&md5=3e176941c4a526f3a97519ce02007792</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1007%2Fs10822-007-9166-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-007-9166-3%26sid%3Dliteratum%253Aachs%26aulast%3DHawkins%26aufirst%3DP.%2BC.%26aulast%3DWarren%26aufirst%3DG.%2BL.%26aulast%3DSkillman%26aufirst%3DA.%2BG.%26aulast%3DNicholls%26aufirst%3DA.%26atitle%3DHow%2520to%2520do%2520an%2520evaluation%253A%2520pitfalls%2520and%2520traps%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2008%26volume%3D22%26spage%3D179%26epage%3D190" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Irwin, J. J.</span><span> </span><span class="NLM_article-title">Community benchmarks for virtual screening</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">193</span><span class="NLM_x">–</span> <span class="NLM_lpage">199</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Fjm300687e&amp;key=10.1007%2Fs10822-008-9189-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Fjm300687e&amp;key=18273555" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Fjm300687e&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjsFOnsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2008&pages=193-199&author=J.+J.+Irwin&title=Community+benchmarks+for+virtual+screening"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Community benchmarks for virtual screening</span></div><div class="casAuthors">Irwin, John J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">3-4</span>),
    <span class="NLM_cas:pages">193-199</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Ligand enrichment among top-ranking hits is a key metric of virtual screening.  To avoid bias, decoys should resemble ligands phys., so that enrichment is not attributable to simple differences of gross features.  We therefore created a directory of useful decoys (DUD) by selecting decoys that resembled annotated ligands phys. but not topol. to benchmark docking performance.  DUD has 2950 annotated ligands and 95,316 property-matched decoys for 40 targets.  It is by far the largest and most comprehensive public data set for benchmarking virtual screening programs that I am aware of.  This paper outlines several ways that DUD can be improved to provide better telemetry to investigators seeking to understand both the strengths and the weaknesses of current docking methods.  I also highlight several pitfalls for the unwary: a risk of over-optimization, questions about chem. space, and the proper scope for using DUD.  Careful attention to both the compn. of benchmarks and how they are used is essential to avoid being misled by overfitting and bias.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoAn7WJldC7AbVg90H21EOLACvtfcHk0ljRaSHSLk8Y6A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjsFOnsb8%253D&md5=4972b768c915bc3e7ee05210dcf1b7a8</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1007%2Fs10822-008-9189-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-008-9189-4%26sid%3Dliteratum%253Aachs%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26atitle%3DCommunity%2520benchmarks%2520for%2520virtual%2520screening%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2008%26volume%3D22%26spage%3D193%26epage%3D199" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group">Mysinger, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Rapid context-dependent ligand desolvation in molecular docking</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">1561</span><span class="NLM_x">–</span> <span class="NLM_lpage">1573</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci100214a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Fjm300687e&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVGrsLvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=1561-1573&author=M.+M.+Mysingerauthor=B.+K.+Shoichet&title=Rapid+context-dependent+ligand+desolvation+in+molecular+docking"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Rapid Context-Dependent Ligand Desolvation in Molecular Docking</span></div><div class="casAuthors">Mysinger, Michael M.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1561-1573</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In structure-based screens for new ligands, a mol. docking algorithm must rapidly score many mols. in multiple configurations, accounting for both the ligand's interactions with receptor and its competing interactions with solvent.  Here the authors explore a context-dependent ligand desolvation scoring term for mol. docking.  The authors relate the Generalized-Born effective Born radii for every ligand atom to a fractional desolvation and then use this fraction to scale an atom-by-atom decompn. of the full transfer free energy.  The fractional desolvation is precomputed on a scoring grid by numerically integrating over the vol. of receptor proximal to a ligand atom, weighted by distance.  To test this method's performance, the authors dock ligands vs. property-matched decoys over 40 DUD targets.  Context-dependent desolvation better enriches ligands compared to both the raw full transfer free energy penalty and compared to ignoring desolvation altogether, though the improvement is modest.  More compellingly, the new method improves docking performance across receptor types.  Thus, whereas entirely ignoring desolvation works best for charged sites and overpenalizing with full desolvation works well for neutral sites, the phys. more correct context-dependent ligand desolvation is competitive across both types of targets.  The method also reliably discriminates ligands from highly charged mols., where ignoring desolvation performs poorly.  Since this context-dependent ligand desolvation may be precalcd., it improves docking reliability with minimal cost to calcn. time and may be readily incorporated into any physics-based docking program.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrtPnS0XiKpYLVg90H21EOLACvtfcHk0ljRaSHSLk8Y6A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVGrsLvI&md5=5124d2c4618e07b4f1e41609c9e61362</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fci100214a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci100214a%26sid%3Dliteratum%253Aachs%26aulast%3DMysinger%26aufirst%3DM.%2BM.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DRapid%2520context-dependent%2520ligand%2520desolvation%2520in%2520molecular%2520docking%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D1561%26epage%3D1573" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group">Vogel, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bauer, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boeckler, F. M.</span><span> </span><span class="NLM_article-title">DEKOIS: demanding evaluation kits for objective in silico screening—a versatile tool for benchmarking docking programs and scoring functions</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">2650</span><span class="NLM_x">–</span> <span class="NLM_lpage">2665</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci2001549" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Fjm300687e&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtValsbnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2011&pages=2650-2665&author=S.+M.+Vogelauthor=M.+R.+Bauerauthor=F.+M.+Boeckler&title=DEKOIS%3A+demanding+evaluation+kits+for+objective+in+silico+screening%E2%80%94a+versatile+tool+for+benchmarking+docking+programs+and+scoring+functions"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">DEKOIS: Demanding Evaluation Kits for Objective in Silico Screening - A Versatile Tool for Benchmarking Docking Programs and Scoring Functions</span></div><div class="casAuthors">Vogel, Simon M.; Bauer, Matthias R.; Boeckler, Frank M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2650-2665</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">For widely applied in silico screening techniques success depends on the rational selection of an appropriate method.  We herein present a fast, versatile, and robust method to construct demanding evaluation kits for objective in silico screening (DEKOIS).  This automated process enables creating tailor-made decoy sets for any given sets of bioactives.  It facilitates a target-dependent validation of docking algorithms and scoring functions helping to save time and resources.  We have developed metrics for assessing and improving decoy set quality and employ them to investigate how decoy embedding affects docking.  We demonstrate that screening performance is target-dependent and can be impaired by latent actives in the decoy set (LADS) or enhanced by poor decoy embedding.  The presented method allows extending and complementing the collection of publicly available high quality decoy sets toward new target space.  All present and future DEKOIS data sets will be made accessible at www.dekois.com.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoK-1LphWadorVg90H21EOLACvtfcHk0lgNgA6BXiQsAw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtValsbnN&md5=e3f59bcd3f898dce1c116a325e914861</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fci2001549&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci2001549%26sid%3Dliteratum%253Aachs%26aulast%3DVogel%26aufirst%3DS.%2BM.%26aulast%3DBauer%26aufirst%3DM.%2BR.%26aulast%3DBoeckler%26aufirst%3DF.%2BM.%26atitle%3DDEKOIS%253A%2520demanding%2520evaluation%2520kits%2520for%2520objective%2520in%2520silico%2520screening%25E2%2580%2594a%2520versatile%2520tool%2520for%2520benchmarking%2520docking%2520programs%2520and%2520scoring%2520functions%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2011%26volume%3D51%26spage%3D2650%26epage%3D2665" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group">Wallach, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lilien, R.</span><span> </span><span class="NLM_article-title">Virtual decoy sets for molecular docking benchmarks</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">196</span><span class="NLM_x">–</span> <span class="NLM_lpage">202</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci100374f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Fjm300687e&amp;key=1%3ACAS%3A528%3ADC%252BC3MXoslel" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2011&pages=196-202&author=I.+Wallachauthor=R.+Lilien&title=Virtual+decoy+sets+for+molecular+docking+benchmarks"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Virtual Decoy Sets for Molecular Docking Benchmarks</span></div><div class="casAuthors">Wallach, Izhar; Lilien, Ryan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">196-202</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Virtual docking algorithms are often evaluated on their ability to sep. active ligands from decoy mols.  The current state-of-the-art benchmark, the Directory of Useful Decoys (DUD), minimizes bias by including decoys from a library of synthetically feasible mols. that are phys. similar yet chem. dissimilar to the active ligands.  We show that by ignoring synthetic feasibility, we can compile a benchmark that is comparable to the DUD and less biased with respect to phys. similarity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoTD04ZpF-QV7Vg90H21EOLACvtfcHk0lgNgA6BXiQsAw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXoslel&md5=3b58c7641cb2c665ce381081d8b1bf41</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Fci100374f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci100374f%26sid%3Dliteratum%253Aachs%26aulast%3DWallach%26aufirst%3DI.%26aulast%3DLilien%26aufirst%3DR.%26atitle%3DVirtual%2520decoy%2520sets%2520for%2520molecular%2520docking%2520benchmarks%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2011%26volume%3D51%26spage%3D196%26epage%3D202" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group">Gatica, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cavasotto, C. N.</span><span> </span><span class="NLM_article-title">Ligand and decoy sets for docking to G protein-coupled receptors</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">6</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci200412p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Fjm300687e&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs1ajsb3P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=1-6&author=E.+A.+Gaticaauthor=C.+N.+Cavasotto&title=Ligand+and+decoy+sets+for+docking+to+G+protein-coupled+receptors"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand and Decoy Sets for Docking to G Protein-Coupled Receptors</span></div><div class="casAuthors">Gatica, Edgar A.; Cavasotto, Claudio N.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-6</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We compiled a G protein-coupled receptor (GPCR) ligand library (GLL) for 147 targets, selecting for each ligand 39 decoy mols., collected in the GPCR Decoy Database (GDD).  Decoys were chosen ensuring a ligand-decoy similarity of six phys. properties, while enforcing ligand-decoy chem. dissimilarity.  The performance in docking of the GDD was evaluated on 19 GPCRs, showing a marked decrease in enrichment compared to bias-uncorrected decoy sets.  Both the GLL and GDD are freely available for the scientific community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrYHZXxtR-bB7Vg90H21EOLACvtfcHk0lgNgA6BXiQsAw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs1ajsb3P&md5=a51e1b60afda728614a3bc1ceed79cfd</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Fci200412p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci200412p%26sid%3Dliteratum%253Aachs%26aulast%3DGatica%26aufirst%3DE.%2BA.%26aulast%3DCavasotto%26aufirst%3DC.%2BN.%26atitle%3DLigand%2520and%2520decoy%2520sets%2520for%2520docking%2520to%2520G%2520protein-coupled%2520receptors%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26spage%3D1%26epage%3D6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group">Cereto-Massague, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guasch, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valls, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mulero, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pujadas, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garcia-Vallve, S.</span><span> </span><span class="NLM_article-title">DecoyFinder: an easy-to-use python GUI application for building target-specific decoy sets</span> <span class="citation_source-journal">Bioinformatics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">1661</span><span class="NLM_x">–</span> <span class="NLM_lpage">1662</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2012&pages=1661-1662&author=A.+Cereto-Massagueauthor=L.+Guaschauthor=C.+Vallsauthor=M.+Muleroauthor=G.+Pujadasauthor=S.+Garcia-Vallve&title=DecoyFinder%3A+an+easy-to-use+python+GUI+application+for+building+target-specific+decoy+sets"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DCereto-Massague%26aufirst%3DA.%26aulast%3DGuasch%26aufirst%3DL.%26aulast%3DValls%26aufirst%3DC.%26aulast%3DMulero%26aufirst%3DM.%26aulast%3DPujadas%26aufirst%3DG.%26aulast%3DGarcia-Vallve%26aufirst%3DS.%26atitle%3DDecoyFinder%253A%2520an%2520easy-to-use%2520python%2520GUI%2520application%2520for%2520building%2520target-specific%2520decoy%2520sets%26jtitle%3DBioinformatics%26date%3D2012%26volume%3D28%26spage%3D1661%26epage%3D1662" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group">Rohrer, S. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baumann, K.</span><span> </span><span class="NLM_article-title">Maximum unbiased validation (MUV) data sets for virtual screening based on PubChem bioactivity data</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">169</span><span class="NLM_x">–</span> <span class="NLM_lpage">184</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci8002649" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Fjm300687e&amp;key=1%3ACAS%3A528%3ADC%252BD1MXptlOhtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2009&pages=169-184&author=S.+G.+Rohrerauthor=K.+Baumann&title=Maximum+unbiased+validation+%28MUV%29+data+sets+for+virtual+screening+based+on+PubChem+bioactivity+data"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Maximum Unbiased Validation (MUV) Data Sets for Virtual Screening Based on PubChem Bioactivity Data</span></div><div class="casAuthors">Rohrer, Sebastian G.; Baumann, Knut</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">169-184</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Refined nearest neighbor anal. was recently introduced for the anal. of virtual screening benchmark data sets.  It constitutes a technique from the field of spatial statistics and provides a math. framework for the nonparametric anal. of mapped point patterns.  Here, refined nearest neighbor anal. is used to design benchmark data sets for virtual screening based on PubChem bioactivity data.  A workflow is devised that purges data sets of compds. active against pharmaceutically relevant targets from unselective hits.  Topol. optimization using exptl. design strategies monitored by refined nearest neighbor anal. functions is applied to generate corresponding data sets of actives and decoys that are unbiased with regard to analog bias and artificial enrichment.  These data sets provide a tool for Maximum Unbiased Validation (MUV) of virtual screening methods.  The data sets and a software package implementing the MUV design workflow are freely available.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGph4RHBumlGlLVg90H21EOLACvtfcHk0liN4VErwpC_Gg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXptlOhtQ%253D%253D&md5=17805f49d995c4c082c3d07be7145a2d</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Fci8002649&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci8002649%26sid%3Dliteratum%253Aachs%26aulast%3DRohrer%26aufirst%3DS.%2BG.%26aulast%3DBaumann%26aufirst%3DK.%26atitle%3DMaximum%2520unbiased%2520validation%2520%2528MUV%2529%2520data%2520sets%2520for%2520virtual%2520screening%2520based%2520on%2520PubChem%2520bioactivity%2520data%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2009%26volume%3D49%26spage%3D169%26epage%3D184" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group">Ripphausen, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wassermann, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bajorath, J.</span><span> </span><span class="NLM_article-title">REPROVIS-DB: a benchmark system for ligand-based virtual screening derived from reproducible prospective applications</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">2467</span><span class="NLM_x">–</span> <span class="NLM_lpage">2473</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci200309j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Fjm300687e&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1ait7fJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2011&pages=2467-2473&author=P.+Ripphausenauthor=A.+M.+Wassermannauthor=J.+Bajorath&title=REPROVIS-DB%3A+a+benchmark+system+for+ligand-based+virtual+screening+derived+from+reproducible+prospective+applications"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">REPROVIS-DB: A Benchmark System for Ligand-Based Virtual Screening Derived from Reproducible Prospective Applications</span></div><div class="casAuthors">Ripphausen, Peter; Wassermann, Anne Mai; Bajorath, Jurgen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2467-2473</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Benchmark calcns. are essential for the evaluation of virtual screening (VS) methods.  Typically, classes of known active compds. taken from the medicinal chem. literature are divided into ref. mols. (search templates) and potential hits that are added to background databases assumed to consist of compds. not sharing this activity.  Then VS calcns. are carried out, and the recall of known active compds. is detd.  However, conventional benchmarking is affected by a no. of problems that reduce its value for method evaluation.  In addn. to often insufficient statistical validation and the lack of generally accepted evaluation stds., the artificial nature of typical benchmark settings is often criticized.  Retrospective benchmark calcns. generally overestimate the potential of VS methods and do not scale with their performance in prospective applications.  In order to provide addnl. opportunities for benchmarking that more closely resemble practical VS conditions, we have designed a publicly available compd. database (DB) of reproducible virtual screens (REPROVIS-DB) that organizes information from successful ligand-based VS applications including ref. compds., screening databases, compd. selection criteria, and exptl. confirmed hits.  Using the currently available 25 hand-selected compd. data sets, one can attempt to reproduce successful virtual screens with other than the originally applied methods and assess their potential for practical applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDPwJwNXQc67Vg90H21EOLACvtfcHk0liN4VErwpC_Gg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1ait7fJ&md5=8439a3f6471f43afe159d08c28a5bc14</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Fci200309j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci200309j%26sid%3Dliteratum%253Aachs%26aulast%3DRipphausen%26aufirst%3DP.%26aulast%3DWassermann%26aufirst%3DA.%2BM.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DREPROVIS-DB%253A%2520a%2520benchmark%2520system%2520for%2520ligand-based%2520virtual%2520screening%2520derived%2520from%2520reproducible%2520prospective%2520applications%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2011%26volume%3D51%26spage%3D2467%26epage%3D2473" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group">Gaulton, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bellis, L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bento, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chambers, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davies, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hersey, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Light, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McGlinchey, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Michalovich, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Al-Lazikani, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Overington, J. P.</span><span> </span><span class="NLM_article-title">ChEMBL: a large-scale bioactivity database for drug discovery</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">D1100</span><span class="NLM_x">–</span> <span class="NLM_lpage">1107</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Fjm300687e&amp;key=10.1093%2Fnar%2Fgkr777" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Fjm300687e&amp;key=21948594" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Fjm300687e&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs12htbjN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2012&pages=D1100-1107&author=A.+Gaultonauthor=L.+J.+Bellisauthor=A.+P.+Bentoauthor=J.+Chambersauthor=M.+Daviesauthor=A.+Herseyauthor=Y.+Lightauthor=S.+McGlincheyauthor=D.+Michalovichauthor=B.+Al-Lazikaniauthor=J.+P.+Overington&title=ChEMBL%3A+a+large-scale+bioactivity+database+for+drug+discovery"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">ChEMBL: a large-scale bioactivity database for drug discovery</span></div><div class="casAuthors">Gaulton, Anna; Bellis, Louisa J.; Bento, A. Patricia; Chambers, Jon; Davies, Mark; Hersey, Anne; Light, Yvonne; McGlinchey, Shaun; Michalovich, David; Al-Lazikani, Bissan; Overington, John P.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D1100-D1107</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">ChEMBL is an Open Data database contg. binding, functional and ADMET information for a large no. of drug-like bioactive compds.  These data are manually abstracted from the primary published literature on a regular basis, then further curated and standardized to maximize their quality and utility across a wide range of chem. biol. and drug-discovery research problems.  Currently, the database contains 5.4 million bioactivity measurements for more than 1 million compds. and 5200 protein targets.  Access is available through a web-based interface, data downloads and web services at: https://www.ebi.ac.uk/chembldb.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9e9jRENTtubVg90H21EOLACvtfcHk0liDak3tCtlh_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs12htbjN&md5=aedf7793e1ca54b6a4fa272ea3ef7d0e</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkr777&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkr777%26sid%3Dliteratum%253Aachs%26aulast%3DGaulton%26aufirst%3DA.%26aulast%3DBellis%26aufirst%3DL.%2BJ.%26aulast%3DBento%26aufirst%3DA.%2BP.%26aulast%3DChambers%26aufirst%3DJ.%26aulast%3DDavies%26aufirst%3DM.%26aulast%3DHersey%26aufirst%3DA.%26aulast%3DLight%26aufirst%3DY.%26aulast%3DMcGlinchey%26aufirst%3DS.%26aulast%3DMichalovich%26aufirst%3DD.%26aulast%3DAl-Lazikani%26aufirst%3DB.%26aulast%3DOverington%26aufirst%3DJ.%2BP.%26atitle%3DChEMBL%253A%2520a%2520large-scale%2520bioactivity%2520database%2520for%2520drug%2520discovery%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2012%26volume%3D40%26spage%3DD1100%26epage%3D1107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group">Bemis, G. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murcko, M. A.</span><span> </span><span class="NLM_article-title">The properties of known drugs. 1. Molecular frameworks</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">39</span><span class="NLM_x">, </span> <span class="NLM_fpage">2887</span><span class="NLM_x">–</span> <span class="NLM_lpage">2893</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm9602928" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Fjm300687e&amp;key=1%3ACAS%3A528%3ADyaK28XjvVejtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=1996&pages=2887-2893&author=G.+W.+Bemisauthor=M.+A.+Murcko&title=The+properties+of+known+drugs.+1.+Molecular+frameworks"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">The Properties of Known Drugs. 1. Molecular Frameworks</span></div><div class="casAuthors">Bemis, Guy W.; Murcko, Mark A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">2887-2893</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">To better understand the common features present in drug mols., we use shape description methods to analyze a database of com. available drugs and prep. a list of common drug shapes.  A useful way of organizing this structural data is to group the atoms of each drug mol. into ring, linker, framework, and side chain atoms.  On the basis of the two-dimensional mol. structures (without regard to atom type, hybridization, and bond order), there are 1179 different frameworks among the 5120 compds. analyzed.  However, the shapes of half of the drugs in the database are described by the 32 most frequently occurring frameworks.  This suggests that the diversity of shapes in the set of known drugs is extremely low.  In our second method of anal., in which atom type, hybridization, and bond order are considered, more diversity is seen; there are 2506 different frameworks among the 5120 compds. in the database, and the most frequently occurring 42 frameworks account for only one-fourth of the drugs.  We discuss the possible interpretations of these findings and the way they may be used to guide future drug discovery research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoodEb3sfDSdrVg90H21EOLACvtfcHk0liDak3tCtlh_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XjvVejtro%253D&md5=5e2c4fdfea9434456a0cca83de4185b3</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Fjm9602928&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm9602928%26sid%3Dliteratum%253Aachs%26aulast%3DBemis%26aufirst%3DG.%2BW.%26aulast%3DMurcko%26aufirst%3DM.%2BA.%26atitle%3DThe%2520properties%2520of%2520known%2520drugs.%25201.%2520Molecular%2520frameworks%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1996%26volume%3D39%26spage%3D2887%26epage%3D2893" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group">Berman, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Westbrook, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilliland, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bhat, T. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weissig, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shindyalov, I. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourne, P. E.</span><span> </span><span class="NLM_article-title">The Protein Data Bank</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">235</span><span class="NLM_x">–</span> <span class="NLM_lpage">242</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Fjm300687e&amp;key=10.1093%2Fnar%2F28.1.235" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Fjm300687e&amp;key=10592235" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Fjm300687e&amp;key=1%3ACAS%3A528%3ADC%252BD3cXhvVKjt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2000&pages=235-242&author=H.+M.+Bermanauthor=J.+Westbrookauthor=Z.+Fengauthor=G.+Gillilandauthor=T.+N.+Bhatauthor=H.+Weissigauthor=I.+N.+Shindyalovauthor=P.+E.+Bourne&title=The+Protein+Data+Bank"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">The Protein Data Bank</span></div><div class="casAuthors">Berman, Helen M.; Westbrook, John; Feng, Zukang; Gilliland, Gary; Bhat, T. N.; Weissig, Helge; Shindyalov, Ilya N.; Bourne, Philip E.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The Protein Data Bank (PDB; http://www.rcsb.org/pdb/)is the single worldwide archive of structural data of biol. macromols.  This paper describes the goals of the PDB, the systems in place for data deposition and access, how to obtain further information, and near-term plans for the future development of the resource.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpStGCxT8QzirVg90H21EOLACvtfcHk0lhgFM9BPzq1gw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXhvVKjt7w%253D&md5=227fb393f754be2be375ab727bfd05dc</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1093%2Fnar%2F28.1.235&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252F28.1.235%26sid%3Dliteratum%253Aachs%26aulast%3DBerman%26aufirst%3DH.%2BM.%26aulast%3DWestbrook%26aufirst%3DJ.%26aulast%3DFeng%26aufirst%3DZ.%26aulast%3DGilliland%26aufirst%3DG.%26aulast%3DBhat%26aufirst%3DT.%2BN.%26aulast%3DWeissig%26aufirst%3DH.%26aulast%3DShindyalov%26aufirst%3DI.%2BN.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26atitle%3DThe%2520Protein%2520Data%2520Bank%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2000%26volume%3D28%26spage%3D235%26epage%3D242" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group">Apweiler, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bairoch, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barker, W. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boeckmann, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferro, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gasteiger, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lopez, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Magrane, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Natale, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Donovan, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Redaschi, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yeh, L. S.</span><span> </span><span class="NLM_article-title">UniProt: The Universal Protein Knowledgebase</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">32</span><span class="NLM_x">, </span> <span class="NLM_fpage">D115</span><span class="NLM_x">–</span> <span class="NLM_lpage">D119</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Fjm300687e&amp;key=10.1093%2Fnar%2Fgkh131" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Fjm300687e&amp;key=14681372" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Fjm300687e&amp;key=1%3ACAS%3A528%3ADC%252BD3sXhtVSru7vK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2004&pages=D115-D119&author=R.+Apweilerauthor=A.+Bairochauthor=C.+H.+Wuauthor=W.+C.+Barkerauthor=B.+Boeckmannauthor=S.+Ferroauthor=E.+Gasteigerauthor=H.+Huangauthor=R.+Lopezauthor=M.+Magraneauthor=M.+J.+Martinauthor=D.+A.+Nataleauthor=C.+O%E2%80%99Donovanauthor=N.+Redaschiauthor=L.+S.+Yeh&title=UniProt%3A+The+Universal+Protein+Knowledgebase"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">UniProt: the universal protein knowledgebase</span></div><div class="casAuthors">Apweiler, Rolf; Bairoch, Amos; Wu, Cathy H.; Barker, Winona C.; Boeckmann, Brigitte; Ferro, Serenella; Gasteiger, Elisabeth; Huang, Hongzhan; Lopez, Rodrigo; Magrane, Michele; Martin, Maria J.; Natale, Darren A.; O'Donovan, Claire; Redaschi, Nicole; Yeh, Lai-Su L.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">Database</span>),
    <span class="NLM_cas:pages">D115-D119</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">To provide the scientific community with a single, centralized, authoritative resource for protein sequences and functional information, the Swiss-Prot, TrEMBL and PIR protein database activities have united to form the Universal Protein Knowledgebase (UniProt) consortium.  Our mission is to provide a comprehensive, fully classified, richly and accurately annotated protein sequence knowledgebase, with extensive cross-refs. and query interfaces.  The central database will have two sections, corresponding to the familiar Swiss-Prot (fully manually curated entries) and TrEMBL (enriched with automated classification, annotation and extensive cross-refs.).  For convenient sequence searches, UniProt also provides several non-redundant sequence databases.  The UniProt NREF (UniRef) databases provide representative subsets of the knowledgebase suitable for efficient searching.  The comprehensive UniProt Archive (UniParc) is updated daily from many public source databases.  The UniProt databases can be accessed online (http://www.uniprot.org) or downloaded in several formats (ftp://ftp.uniprot.org/pub).  The scientific community is encouraged to submit data for inclusion in UniProt.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGos0pWEYnhzKLVg90H21EOLACvtfcHk0lhgFM9BPzq1gw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXhtVSru7vK&md5=4b692ffbb5c27bd9b4d2a8d0809e84e1</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkh131&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkh131%26sid%3Dliteratum%253Aachs%26aulast%3DApweiler%26aufirst%3DR.%26aulast%3DBairoch%26aufirst%3DA.%26aulast%3DWu%26aufirst%3DC.%2BH.%26aulast%3DBarker%26aufirst%3DW.%2BC.%26aulast%3DBoeckmann%26aufirst%3DB.%26aulast%3DFerro%26aufirst%3DS.%26aulast%3DGasteiger%26aufirst%3DE.%26aulast%3DHuang%26aufirst%3DH.%26aulast%3DLopez%26aufirst%3DR.%26aulast%3DMagrane%26aufirst%3DM.%26aulast%3DMartin%26aufirst%3DM.%2BJ.%26aulast%3DNatale%26aufirst%3DD.%2BA.%26aulast%3DO%25E2%2580%2599Donovan%26aufirst%3DC.%26aulast%3DRedaschi%26aufirst%3DN.%26aulast%3DYeh%26aufirst%3DL.%2BS.%26atitle%3DUniProt%253A%2520The%2520Universal%2520Protein%2520Knowledgebase%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2004%26volume%3D32%26spage%3DD115%26epage%3DD119" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mysinger, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Colizzi, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wassam, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cao, Y.</span><span> </span><span class="NLM_article-title">Automated docking screens: a feasibility study</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">5712</span><span class="NLM_x">–</span> <span class="NLM_lpage">5720</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm9006966" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Fjm300687e&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtVOjurjN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=5712-5720&author=J.+J.+Irwinauthor=B.+K.+Shoichetauthor=M.+M.+Mysingerauthor=N.+Huangauthor=F.+Colizziauthor=P.+Wassamauthor=Y.+Cao&title=Automated+docking+screens%3A+a+feasibility+study"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Automated Docking Screens: A Feasibility Study</span></div><div class="casAuthors">Irwin, John J.; Shoichet, Brian K.; Mysinger, Michael M.; Huang, Niu; Colizzi, Francesco; Wassam, Pascal; Cao, Yiqun</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">5712-5720</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mol. docking is the most practical approach to leverage protein structure for ligand discovery, but the technique retains important liabilities that make it challenging to deploy on a large scale.  We have therefore created an expert system, DOCK Blaster, to investigate the feasibility of full automation.  The method requires a PDB code, sometimes with a ligand structure, and from that alone can launch a full screen of large libraries.  A crit. feature is self-assessment, which ests. the anticipated reliability of the automated screening results using pose fidelity and enrichment.  Against common benchmarks, DOCK Blaster recapitulates the crystal ligand pose within 2 Å rmsd 50-60% of the time; inferior to an expert, but respectable.  Half the time the ligand also ranked among the top 5% of 100 phys. matched decoys chosen on the fly.  Further tests were undertaken culminating in a study of 7755 eligible PDB structures.  In 1398 cases, the redocked ligand ranked in the top 5% of 100 property-matched decoys while also posing within 2 Å rmsd, suggesting that unsupervised prospective docking is viable.  DOCK Blaster is available at http://blaster.docking.org.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPZAd6CrHGp7Vg90H21EOLACvtfcHk0ljU05Jzg8inLA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtVOjurjN&md5=f302db1f77b87f26d1fa2fece55bca96</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Fjm9006966&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm9006966%26sid%3Dliteratum%253Aachs%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DMysinger%26aufirst%3DM.%2BM.%26aulast%3DHuang%26aufirst%3DN.%26aulast%3DColizzi%26aufirst%3DF.%26aulast%3DWassam%26aufirst%3DP.%26aulast%3DCao%26aufirst%3DY.%26atitle%3DAutomated%2520docking%2520screens%253A%2520a%2520feasibility%2520study%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D5712%26epage%3D5720" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group">Powers, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morandi, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Structure-based discovery of a novel, noncovalent inhibitor of AmpC beta-lactamase</span> <span class="citation_source-journal">Structure</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">1013</span><span class="NLM_x">–</span> <span class="NLM_lpage">1023</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Fjm300687e&amp;key=10.1016%2FS0969-2126%2802%2900799-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Fjm300687e&amp;key=12121656" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Fjm300687e&amp;key=1%3ACAS%3A528%3ADC%252BD38XlsVyjsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2002&pages=1013-1023&author=R.+A.+Powersauthor=F.+Morandiauthor=B.+K.+Shoichet&title=Structure-based+discovery+of+a+novel%2C+noncovalent+inhibitor+of+AmpC+beta-lactamase"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Discovery of a Novel, Noncovalent Inhibitor of AmpC β-Lactamase</span></div><div class="casAuthors">Powers, Rachel A.; Morandi, Federica; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1013-1023</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">β-Lactamases are the most widespread resistance mechanisms to β-lactam antibiotics, and there is a pressing need for novel, non-β-lactam drugs.  A database of over 200,000 compds. was docked to the active site of AmpC β-lactamase to identify potential inhibitors.  Fifty-six compds. were tested, and three had Ki values of 650 μM or better.  The best of these, 3-[(4-chloroanilino)sulfonyl]thiophene-2-carboxylic acid, was a competitive noncovalent inhibitor (Ki = 26 μM), which also reversed resistance to β-lactams in bacteria expressing AmpC.  The structure of AmpC in complex with this compd. was detd. by x-ray crystallog. to 1.94 A and reveals that the inhibitor interacts with key active-site residues in sites targeted in the docking calcn.  Indeed, the exptl. detd. conformation of the inhibitor closely resembles the prediction.  The structure of the enzyme-inhibitor complex presents an opportunity to improve binding affinity in a novel series of inhibitors discovered by structure-based methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKX-gft1lXA7Vg90H21EOLACvtfcHk0ljU05Jzg8inLA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XlsVyjsrk%253D&md5=5f539484d08191f0c65626493fc1b262</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1016%2FS0969-2126%2802%2900799-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0969-2126%252802%252900799-2%26sid%3Dliteratum%253Aachs%26aulast%3DPowers%26aufirst%3DR.%2BA.%26aulast%3DMorandi%26aufirst%3DF.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DStructure-based%2520discovery%2520of%2520a%2520novel%252C%2520noncovalent%2520inhibitor%2520of%2520AmpC%2520beta-lactamase%26jtitle%3DStructure%26date%3D2002%26volume%3D10%26spage%3D1013%26epage%3D1023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group">Carlsson, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yoo, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gao, Z. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacobson, K. A.</span><span> </span><span class="NLM_article-title">Structure-based discovery of A2A adenosine receptor ligands</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">3748</span><span class="NLM_x">–</span> <span class="NLM_lpage">3755</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm100240h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Fjm300687e&amp;key=1%3ACAS%3A528%3ADC%252BC3cXkvFaqsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=3748-3755&author=J.+Carlssonauthor=L.+Yooauthor=Z.+G.+Gaoauthor=J.+J.+Irwinauthor=B.+K.+Shoichetauthor=K.+A.+Jacobson&title=Structure-based+discovery+of+A2A+adenosine+receptor+ligands"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Discovery of A2A Adenosine Receptor Ligands</span></div><div class="casAuthors">Carlsson, Jens; Yoo, Lena; Gao, Zhan-Guo; Irwin, John J.; Shoichet, Brian K.; Jacobson, Kenneth A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3748-3755</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The recent detn. of X-ray structures of pharmacol. relevant GPCRs has made these targets accessible to structure-based ligand discovery.  Here we explore whether novel chemotypes may be discovered for the A2A adenosine receptor, based on complementarity to its recently detd. structure.  The A2A adenosine receptor signals in the periphery and the CNS, with agonists explored as anti-inflammatory drugs and antagonists explored for neurodegenerative diseases.  We used mol. docking to screen a 1.4 million compd. database against the X-ray structure computationally and tested 20 high-ranking, previously unknown mols. exptl.  Of these 35% showed substantial activity with affinities between 200 nM and 9 μM.  For the most potent of these new inhibitors, over 50-fold specificity was obsd. for the A2A vs. the related A1 and A3 subtypes.  These high hit rates and affinities at least partly reflect the bias of com. libraries toward GPCR-like chemotypes, an issue that we attempt to investigate quant.  Despite this bias, many of the most potent new ligands were novel, dissimilar from known ligands, providing new lead structures for modulation of this medically important target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpzvS6-9T9bd7Vg90H21EOLACvtfcHk0ljU05Jzg8inLA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXkvFaqsL8%253D&md5=c36c941d52d2cec06387d79c4c423d46</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Fjm100240h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm100240h%26sid%3Dliteratum%253Aachs%26aulast%3DCarlsson%26aufirst%3DJ.%26aulast%3DYoo%26aufirst%3DL.%26aulast%3DGao%26aufirst%3DZ.%2BG.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DJacobson%26aufirst%3DK.%2BA.%26atitle%3DStructure-based%2520discovery%2520of%2520A2A%2520adenosine%2520receptor%2520ligands%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D3748%26epage%3D3755" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group">Carlsson, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coleman, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Setola, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fan, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schlessinger, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sali, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Ligand discovery from a dopamine D3 receptor homology model and crystal structure</span> <span class="citation_source-journal">Natre Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">769</span><span class="NLM_x">–</span> <span class="NLM_lpage">778</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Fjm300687e&amp;key=10.1038%2Fnchembio.662" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Fjm300687e&amp;key=21926995" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Fjm300687e&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFylsLvP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=769-778&author=J.+Carlssonauthor=R.+G.+Colemanauthor=V.+Setolaauthor=J.+J.+Irwinauthor=H.+Fanauthor=A.+Schlessingerauthor=A.+Saliauthor=B.+L.+Rothauthor=B.+K.+Shoichet&title=Ligand+discovery+from+a+dopamine+D3+receptor+homology+model+and+crystal+structure"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand discovery from a dopamine D3 receptor homology model and crystal structure</span></div><div class="casAuthors">Carlsson, Jens; Coleman, Ryan G.; Setola, Vincent; Irwin, John J.; Fan, Hao; Schlessinger, Avner; Sali, Andrej; Roth, Bryan L.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">769-778</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">G protein-coupled receptors (GPCRs) are intensely studied as drug targets and for their role in signaling.  With the detn. of the first crystal structures, interest in structure-based ligand discovery increased.  Unfortunately, for most GPCRs no exptl. structures are available.  The detn. of the D3 receptor structure and the challenge to the community to predict it enabled a fully prospective comparison of ligand discovery from a modeled structure vs. that of the subsequently released crystal structure.  Over 3.3 million mols. were docked against a homol. model, and 26 of the highest ranking were tested for binding.  Six had affinities ranging from 0.2 to 3.1 μM.  Subsequently, the crystal structure was released and the docking screen repeated.  Of the 25 compds. selected, five had affinities ranging from 0.3 to 3.0 μM.  One of the new ligands from the homol. model screen was optimized for affinity to 81 nM.  The feasibility of docking screens against modeled GPCRs more generally is considered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqz6gBx3Olbn7Vg90H21EOLACvtfcHk0lhb8p-D70NGVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFylsLvP&md5=7242aa512079a348678b309d352ecdb9</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.662&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.662%26sid%3Dliteratum%253Aachs%26aulast%3DCarlsson%26aufirst%3DJ.%26aulast%3DColeman%26aufirst%3DR.%2BG.%26aulast%3DSetola%26aufirst%3DV.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DFan%26aufirst%3DH.%26aulast%3DSchlessinger%26aufirst%3DA.%26aulast%3DSali%26aufirst%3DA.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DLigand%2520discovery%2520from%2520a%2520dopamine%2520D3%2520receptor%2520homology%2520model%2520and%2520crystal%2520structure%26jtitle%3DNatre%2520Chem.%2520Biol.%26date%3D2011%26volume%3D7%26spage%3D769%26epage%3D778" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">ZINC—a free database of commercially available compounds for virtual screening</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">177</span><span class="NLM_x">–</span> <span class="NLM_lpage">182</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci049714%2B" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Fjm300687e&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVOjt77J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2005&pages=177-182&author=J.+J.+Irwinauthor=B.+K.+Shoichet&title=ZINC%E2%80%94a+free+database+of+commercially+available+compounds+for+virtual+screening"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">ZINC - A Free Database of Commercially Available Compounds for Virtual Screening</span></div><div class="casAuthors">Irwin, John J.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Computer Sciences</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">177-182</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">0095-2338</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A crit. barrier to entry into structure-based virtual screening is the lack of a suitable, easy to access database of purchasable compds.  We have therefore prepd. a library of 727 842 mols., each with 3D structure, using catalogs of compds. from vendors (the size of this library continues to grow).  The mols. have been assigned biol. relevant protonation states and are annotated with properties such as mol. wt., calcd. LogP, and no. of rotatable bonds.  Each mol. in the library contains vendor and purchasing information and is ready for docking using a no. of popular docking programs.  Within certain limits, the mols. are prepd. in multiple protonation states and multiple tautomeric forms.  In one format, multiple conformations are available for the mols.  This database is available for free download (http://zinc.docking.org) in several common file formats including SMILES, mol2, 3D SDF, and DOCK flexibase format.  A Web-based query tool incorporating a mol. drawing interface enables the database to be searched and browsed and subsets to be created.  Users can process their own mols. by uploading them to a server.  Our hope is that this database will bring virtual screening libraries to a wide community of structural biologists and medicinal chemists.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPhFw5E2XACbVg90H21EOLACvtfcHk0lhb8p-D70NGVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVOjt77J&md5=e3892b7dc8608b17a3e63541a5ed60e6</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Fci049714%2B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci049714%252B%26sid%3Dliteratum%253Aachs%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DZINC%25E2%2580%2594a%2520free%2520database%2520of%2520commercially%2520available%2520compounds%2520for%2520virtual%2520screening%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2005%26volume%3D45%26spage%3D177%26epage%3D182" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group">Velankar, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McNeil, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mittard-Runte, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suarez, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barrell, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Apweiler, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Henrick, K.</span><span> </span><span class="NLM_article-title">E-MSD: an integrated data resource for bioinformatics</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">33</span><span class="NLM_x">, </span> <span class="NLM_fpage">D262</span><span class="NLM_x">–</span> <span class="NLM_lpage">265</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2005&pages=D262-265&author=S.+Velankarauthor=P.+McNeilauthor=V.+Mittard-Runteauthor=A.+Suarezauthor=D.+Barrellauthor=R.+Apweilerauthor=K.+Henrick&title=E-MSD%3A+an+integrated+data+resource+for+bioinformatics"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DVelankar%26aufirst%3DS.%26aulast%3DMcNeil%26aufirst%3DP.%26aulast%3DMittard-Runte%26aufirst%3DV.%26aulast%3DSuarez%26aufirst%3DA.%26aulast%3DBarrell%26aufirst%3DD.%26aulast%3DApweiler%26aufirst%3DR.%26aulast%3DHenrick%26aufirst%3DK.%26atitle%3DE-MSD%253A%2520an%2520integrated%2520data%2520resource%2520for%2520bioinformatics%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2005%26volume%3D33%26spage%3DD262%26epage%3D265" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group">Hawkins, P. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Skillman, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nicholls, A.</span><span> </span><span class="NLM_article-title">Comparison of shape-matching and docking as virtual screening tools</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">74</span><span class="NLM_x">–</span> <span class="NLM_lpage">82</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm0603365" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Fjm300687e&amp;key=1%3ACAS%3A528%3ADC%252BD28Xhtlansb%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2007&pages=74-82&author=P.+C.+Hawkinsauthor=A.+G.+Skillmanauthor=A.+Nicholls&title=Comparison+of+shape-matching+and+docking+as+virtual+screening+tools"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Comparison of Shape-Matching and Docking as Virtual Screening Tools</span></div><div class="casAuthors">Hawkins, Paul C. D.; Skillman, A. Geoffrey; Nicholls, Anthony</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">74-82</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Ligand docking is a widely used approach in virtual screening.  In recent years a large no. of publications have appeared in which docking tools are compared and evaluated for their effectiveness in virtual screening against a wide variety of protein targets.  These studies have shown that the effectiveness of docking in virtual screening is highly variable due to a large no. of possible confounding factors.  Another class of method that has shown promise in virtual screening is the shape-based, ligand-centric approach.  Several direct comparisons of docking with the shape-based tool ROCS have been conducted using data sets from some of these recent docking publications.  The results show that a shape-based, ligand-centric approach is more consistent than, and often superior to, the protein-centric approach taken by docking.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoe0IKOly7zibVg90H21EOLACvtfcHk0lhb8p-D70NGVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xhtlansb%252FF&md5=6f97f5c0cc092b4e225f7c2656c1bcf6</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1021%2Fjm0603365&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0603365%26sid%3Dliteratum%253Aachs%26aulast%3DHawkins%26aufirst%3DP.%2BC.%26aulast%3DSkillman%26aufirst%3DA.%2BG.%26aulast%3DNicholls%26aufirst%3DA.%26atitle%3DComparison%2520of%2520shape-matching%2520and%2520docking%2520as%2520virtual%2520screening%2520tools%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2007%26volume%3D50%26spage%3D74%26epage%3D82" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group">Teotico, D. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Babaoglu, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rocklin, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferreira, R. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giannetti, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Docking for fragment inhibitors of AmpC beta-lactamase</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U.S.A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">106</span><span class="NLM_x">, </span> <span class="NLM_fpage">7455</span><span class="NLM_x">–</span> <span class="NLM_lpage">7460</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Fjm300687e&amp;key=10.1073%2Fpnas.0813029106" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Fjm300687e&amp;key=19416920" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Fjm300687e&amp;key=1%3ACAS%3A528%3ADC%252BD1MXmt1Khtrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2009&pages=7455-7460&author=D.+G.+Teoticoauthor=K.+Babaogluauthor=G.+J.+Rocklinauthor=R.+S.+Ferreiraauthor=A.+M.+Giannettiauthor=B.+K.+Shoichet&title=Docking+for+fragment+inhibitors+of+AmpC+beta-lactamase"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Docking for fragment inhibitors of AmpC 83-lactamase</span></div><div class="casAuthors">Teotico, Denise G.; Babaoglu, Kerim; Rocklin, Gabriel J.; Ferreira, Rafaela S.; Giannetti, Anthony M.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">7455-7460</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Fragment screens for new ligands have had wide success, not withstanding their constraint to libraries of 1000-10,000 mols.  Larger libraries would be addressable were mol. docking reliable for fragment screens, but this has not been widely accepted.  To investigate docking's ability to prioritize fragments, a library of >137,000 such mols. were docked against the structure of β-lactamase.  Forty-eight fragments highly ranked by docking were acquired and tested; 23 had Ki values ranging from 0.7 to 9.2 mM.  X-ray crystal structures of the enzyme-bound complexes were detd. for 8 of the fragments.  For 4, the correspondence between the predicted and exptl. structures was high (RMSD between 1.2 and 1.4 Å), whereas for another 2, the fidelity was lower but retained most key interactions (RMSD 2.4-2.6 Å).  Two of the 8 fragments adopted very different poses in the active site owing to enzyme conformational changes.  The 48% hit rate of the fragment docking compares very favorably with "lead-like" docking and high-throughput screening against the same enzyme.  To understand this, we investigated the occurrence of the fragment scaffolds among larger, lead-like mols.  Approx. 1% of com. available fragments contain these inhibitors whereas only 10-7% of lead-like mols. do.  This suggests that many more chemotypes and combinations of chemotypes are present among fragments than are available among lead-like mols., contributing to the higher hit rates.  The ability of docking to prioritize these fragments suggests that the technique can be used to exploit the better chemotype coverage that exists at the fragment level.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprpafzXv4ydrVg90H21EOLACvtfcHk0lhb8p-D70NGVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXmt1Khtrg%253D&md5=d2df01432d5381c226e8aa7e008457b5</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0813029106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0813029106%26sid%3Dliteratum%253Aachs%26aulast%3DTeotico%26aufirst%3DD.%2BG.%26aulast%3DBabaoglu%26aufirst%3DK.%26aulast%3DRocklin%26aufirst%3DG.%2BJ.%26aulast%3DFerreira%26aufirst%3DR.%2BS.%26aulast%3DGiannetti%26aufirst%3DA.%2BM.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DDocking%2520for%2520fragment%2520inhibitors%2520of%2520AmpC%2520beta-lactamase%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2009%26volume%3D106%26spage%3D7455%26epage%3D7460" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group">Tondi, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morandi, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bonnet, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Costi, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Structure-based optimization of a non-beta-lactam lead results in inhibitors that do not up-regulate beta-lactamase expression in cell culture</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">127</span><span class="NLM_x">, </span> <span class="NLM_fpage">4632</span><span class="NLM_x">–</span> <span class="NLM_lpage">4639</span></span><div class="citationLinks">[<a href="/doi/10.1021/ja042984o" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=127&publication_year=2005&pages=4632-4639&author=D.+Tondiauthor=F.+Morandiauthor=R.+Bonnetauthor=M.+P.+Costiauthor=B.+K.+Shoichet&title=Structure-based+optimization+of+a+non-beta-lactam+lead+results+in+inhibitors+that+do+not+up-regulate+beta-lactamase+expression+in+cell+culture"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1021%2Fja042984o&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja042984o%26sid%3Dliteratum%253Aachs%26aulast%3DTondi%26aufirst%3DD.%26aulast%3DMorandi%26aufirst%3DF.%26aulast%3DBonnet%26aufirst%3DR.%26aulast%3DCosti%26aufirst%3DM.%2BP.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DStructure-based%2520optimization%2520of%2520a%2520non-beta-lactam%2520lead%2520results%2520in%2520inhibitors%2520that%2520do%2520not%2520up-regulate%2520beta-lactamase%2520expression%2520in%2520cell%2520culture%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2005%26volume%3D127%26spage%3D4632%26epage%3D4639" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group">Graves, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brenk, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Decoys for docking</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">3714</span><span class="NLM_x">–</span> <span class="NLM_lpage">3728</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm0491187" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Fjm300687e&amp;key=1%3ACAS%3A528%3ADC%252BD2MXjvFKjsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=3714-3728&author=A.+P.+Gravesauthor=R.+Brenkauthor=B.+K.+Shoichet&title=Decoys+for+docking"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Decoys for Docking</span></div><div class="casAuthors">Graves, Alan P.; Brenk, Ruth; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">3714-3728</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mol. docking is widely used to predict novel lead compds. for drug discovery.  Success depends on the quality of the docking scoring function, among other factors.  An imperfect scoring function can mislead by predicting incorrect ligand geometries or by selecting nonbinding mols. over true ligands.  These false-pos. hits may be considered "decoys".  Although these decoys are frustrating, they potentially provide important tests for a docking algorithm; the more subtle the decoy, the more rigorous the test.  Indeed, decoy databases have been used to improve protein structure prediction algorithms and protein-protein docking algorithms.  Here, we describe 20 geometric decoys in five enzymes and 166 "hit list" decoys-i.e., mols. predicted to bind by our docking program that were tested and found not to do so - for β-lactamase and two cavity sites in lysozyme.  Esp. in the cavity sites, which are very simple, these decoys highlight particular weaknesses in our scoring function.  We also consider the performance of five other widely used docking scoring functions against our geometric and hit list decoys.  Intriguingly, whereas many of these other scoring functions performed better on the geometric decoys, they typically performed worse on the hit list decoys, often highly ranking mols. that seemed to poorly complement the model sites.  Several of these "hits" from the other scoring functions were tested exptl. and found, in fact, to be decoys.  Collectively, these decoys provide a tool for the development and improvement of mol. docking scoring functions.  Such improvements may, in turn, be rapidly tested exptl. against these and related exptl. systems, which are well-behaved in assays and for structure detn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpklqU6W2ulh7Vg90H21EOLACvtfcHk0lj-XHHMo_q1JQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXjvFKjsbs%253D&md5=1797b501c1777dc94553701d45c822f0</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1021%2Fjm0491187&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0491187%26sid%3Dliteratum%253Aachs%26aulast%3DGraves%26aufirst%3DA.%2BP.%26aulast%3DBrenk%26aufirst%3DR.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DDecoys%2520for%2520docking%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D3714%26epage%3D3728" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group">Hawkins, P. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Skillman, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Warren, G. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ellingson, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stahl, M. T.</span><span> </span><span class="NLM_article-title">Conformer generation with OMEGA: algorithm and validation using high quality structures from the Protein Data Bank and Cambridge Structural Database</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">572</span><span class="NLM_x">–</span> <span class="NLM_lpage">584</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci100031x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Fjm300687e&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjtlaisrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=572-584&author=P.+C.+Hawkinsauthor=A.+G.+Skillmanauthor=G.+L.+Warrenauthor=B.+A.+Ellingsonauthor=M.+T.+Stahl&title=Conformer+generation+with+OMEGA%3A+algorithm+and+validation+using+high+quality+structures+from+the+Protein+Data+Bank+and+Cambridge+Structural+Database"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Conformer Generation with OMEGA: Algorithm and Validation Using High Quality Structures from the Protein Databank and Cambridge Structural Database</span></div><div class="casAuthors">Hawkins, Paul C. D.; Skillman, A. Geoffrey; Warren, Gregory L.; Ellingson, Benjamin A.; Stahl, Matthew T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">572-584</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Here, we present the algorithm and validation for OMEGA, a systematic, knowledge-based conformer generator.  The algorithm consists of three phases: assembly of an initial 3D structure from a library of fragments; exhaustive enumeration of all rotatable torsions using values drawn from a knowledge-based list of angles, thereby generating a large set of conformations; and sampling of this set by geometric and energy criteria.  Validation of conformer generators like OMEGA has often been undertaken by comparing computed conformer sets to exptl. mol. conformations from crystallog., usually from the Protein Databank (PDB).  Such an approach is fraught with difficulty due to the systematic problems with small mol. structures in the PDB.  Methods are presented to identify a diverse set of small mol. structures from cocomplexes in the PDB that has maximal reliability.  A challenging set of 197 high quality, carefully selected ligand structures from well-solved models was obtained using these methods.  This set will provide a sound basis for comparison and validation of conformer generators in the future.  Validation results from this set are compared to the results using structures of a set of druglike mols. extd. from the Cambridge Structural Database (CSD).  OMEGA is found to perform very well in reproducing the crystallog. conformations from both these data sets using two complementary metrics of success.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqq1SOwPH1NzrVg90H21EOLACvtfcHk0lj-XHHMo_q1JQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjtlaisrY%253D&md5=fb87ecc9c51eddef63b41fffcd9babee</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1021%2Fci100031x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci100031x%26sid%3Dliteratum%253Aachs%26aulast%3DHawkins%26aufirst%3DP.%2BC.%26aulast%3DSkillman%26aufirst%3DA.%2BG.%26aulast%3DWarren%26aufirst%3DG.%2BL.%26aulast%3DEllingson%26aufirst%3DB.%2BA.%26aulast%3DStahl%26aufirst%3DM.%2BT.%26atitle%3DConformer%2520generation%2520with%2520OMEGA%253A%2520algorithm%2520and%2520validation%2520using%2520high%2520quality%2520structures%2520from%2520the%2520Protein%2520Data%2520Bank%2520and%2520Cambridge%2520Structural%2520Database%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D572%26epage%3D584" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group">Jain, A. N.</span><span> </span><span class="NLM_article-title">Bias, reporting, and sharing: computational evaluations of docking methods</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">201</span><span class="NLM_x">–</span> <span class="NLM_lpage">212</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2008&pages=201-212&author=A.+N.+Jain&title=Bias%2C+reporting%2C+and+sharing%3A+computational+evaluations+of+docking+methods"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DJain%26aufirst%3DA.%2BN.%26atitle%3DBias%252C%2520reporting%252C%2520and%2520sharing%253A%2520computational%2520evaluations%2520of%2520docking%2520methods%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2008%26volume%3D22%26spage%3D201%26epage%3D212" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 819 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Biao Ma, Kei Terayama, Shigeyuki Matsumoto, Yuta Isaka, Yoko Sasakura, Hiroaki Iwata, Mitsugu Araki, <span class="NLM_string-name hlFld-ContribAuthor">Yasushi Okuno</span>. </span><span class="cited-content_cbyCitation_article-title">Structure-Based de Novo Molecular Generator Combined with Artificial Intelligence and Docking Simulations. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2021,</strong> <em>61 </em>
                                    (7)
                                     , 3304-3313. <a href="https://doi.org/10.1021/acs.jcim.1c00679" title="DOI URL">https://doi.org/10.1021/acs.jcim.1c00679</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.1c00679&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.1c00679%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DStructure-Based%252Bde%252BNovo%252BMolecular%252BGenerator%252BCombined%252Bwith%252BArtificial%252BIntelligence%252Band%252BDocking%252BSimulations%26aulast%3DMa%26aufirst%3DBiao%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D15062021%26date%3D09072021%26volume%3D61%26issue%3D7%26spage%3D3304%26epage%3D3313" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Felix Musil, Andrea Grisafi, Albert P. Bartók, Christoph Ortner, Gábor Csányi, <span class="NLM_string-name hlFld-ContribAuthor">Michele Ceriotti</span>. </span><span class="cited-content_cbyCitation_article-title">Physics-Inspired Structural Representations for Molecules and Materials. </span><span class="cited-content_cbyCitation_journal-name">Chemical Reviews</span><span> <strong>2021</strong>, Article ASAP.</span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.chemrev.1c00021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.chemrev.1c00021%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Reviews%26atitle%3DPhysics-Inspired%252BStructural%252BRepresentations%252Bfor%252BMolecules%252Band%252BMaterials%26aulast%3DMusil%26aufirst%3DFelix%26date%3D2021%26date%3D2021%26date%3D08012021%26volume%3D0%26issue%3D0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Hugo Guterres, Sang-Jun Park, Han Zhang, <span class="NLM_string-name hlFld-ContribAuthor">Wonpil Im</span>. </span><span class="cited-content_cbyCitation_article-title">CHARMM-GUI LBS Finder & Refiner for Ligand Binding Site Prediction and Refinement. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2021</strong>, Article ASAP.</span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.1c00561&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.1c00561%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DCHARMM-GUI%252BLBS%252BFinder%252B%252526%252BRefiner%252Bfor%252BLigand%252BBinding%252BSite%252BPrediction%252Band%252BRefinement%26aulast%3DGuterres%26aufirst%3DHugo%26date%3D2021%26date%3D2021%26date%3D17052021%26volume%3D0%26issue%3D0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Jerome Eberhardt, Diogo Santos-Martins, Andreas F. Tillack, <span class="NLM_string-name hlFld-ContribAuthor">Stefano Forli</span>. </span><span class="cited-content_cbyCitation_article-title">AutoDock Vina 1.2.0: New Docking Methods, Expanded Force Field, and Python Bindings. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2021</strong>, Article ASAP.</span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.1c00203&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.1c00203%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DAutoDock%252BVina%252B1.2.0%25253A%252BNew%252BDocking%252BMethods%25252C%252BExpanded%252BForce%252BField%25252C%252Band%252BPython%252BBindings%26aulast%3DEberhardt%26aufirst%3DJerome%26date%3D2021%26date%3D2021%26date%3D22022021%26volume%3D0%26issue%3D0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Wei Chen, Guanxing Chen, Lu Zhao, <span class="NLM_string-name hlFld-ContribAuthor">Calvin Yu-Chian Chen</span>. </span><span class="cited-content_cbyCitation_article-title">Predicting Drug–Target Interactions with Deep-Embedding Learning of Graphs and Sequences. </span><span class="cited-content_cbyCitation_journal-name">The Journal of Physical Chemistry A</span><span> <strong>2021,</strong> <em>125 </em>
                                    (25)
                                     , 5633-5642. <a href="https://doi.org/10.1021/acs.jpca.1c02419" title="DOI URL">https://doi.org/10.1021/acs.jpca.1c02419</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jpca.1c02419&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jpca.1c02419%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Journal%2520of%2520Physical%2520Chemistry%2520A%26atitle%3DPredicting%252BDrug%2525E2%252580%252593Target%252BInteractions%252Bwith%252BDeep-Embedding%252BLearning%252Bof%252BGraphs%252Band%252BSequences%26aulast%3DChen%26aufirst%3DWei%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D17032021%26date%3D05062021%26date%3D18062021%26volume%3D125%26issue%3D25%26spage%3D5633%26epage%3D5642" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">SahaIshika</span><span class="NLM_role">Graduate Student Researcher</span><a class="email" href="/cdn-cgi/l/email-protection#543d273c3d3f3527353c3514337a213738357a313021"><span class="nobrWithWbr"><span class="__cf_email__" data-cfemail="7f160c1716141e0c1e171e3f1851">[email protected]</span><wbr></wbr>ucla.<wbr></wbr>edu</span></a><span class="hlFld-ContribAuthor ">HarranPatrick G.</span><span class="NLM_role">D.J. & J.M. Cram Chair in Organic Chemistry</span><a class="email" href="/cdn-cgi/l/email-protection#3d555c4f4f5c537d5e55585013485e515c13585948"><span class="nobrWithWbr"><span class="__cf_email__" data-cfemail="93fbf2e1e1f2fdd3f0fbf6febd">[email protected]</span><wbr></wbr>ucla.<wbr></wbr>edu</span></a>Dr. Jonathan Bohmann, Department of Pharmaceuticals and Bioengineering, Southwest Research Institute, <div class="author-comment"><p class="first last">A great introduction to the topic of virtual screening, which is often seen as a black box.  It does a good job weaving a lot of material together. The molecular simulation community has needed a book like this for a long time. It is honest about the pitfalls of VS while pointing to reasons why VS is needed and the positive impacts on early drug discovery programs, where resources are generally scarce.</p></div><span class="NLM_string-name hlFld-ContribAuthor">Ryan Gumpper, Postdoctoral Researcher, University of North Carolina at Chapel Hill</span><div class="author-comment"><p class="first last">I would recommend this work to young graduate students starting in the field and anyone who is interested in expanding their knowledge in the field.</p></div>. </span><span class="cited-content_cbyCitation_article-title">Virtual Screening for Chemists. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,<a href="https://doi.org/10.1021/acsinfocus.7e5001" title="DOI URL">https://doi.org/10.1021/acsinfocus.7e5001</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsinfocus.7e5001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsinfocus.7e5001%26sid%3Dliteratum%253Aachs%26atitle%3DVirtual%252BScreening%252Bfor%252BChemists%26aulast%3DSaha%26aufirst%3DIshika%26date%3D2021%26date%3D2021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Alzbeta Tuerkova, Orsolya Ungvári, Réka Laczkó-Rigó, Erzsébet Mernyák, Gergely Szakács, Csilla Özvegy-Laczka, <span class="NLM_string-name hlFld-ContribAuthor">Barbara Zdrazil</span>. </span><span class="cited-content_cbyCitation_article-title">Data-Driven Ensemble Docking to Map Molecular Interactions of Steroid Analogs with Hepatic Organic Anion Transporting Polypeptides. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2021,</strong> <em>61 </em>
                                    (6)
                                     , 3109-3127. <a href="https://doi.org/10.1021/acs.jcim.1c00362" title="DOI URL">https://doi.org/10.1021/acs.jcim.1c00362</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.1c00362&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.1c00362%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DData-Driven%252BEnsemble%252BDocking%252Bto%252BMap%252BMolecular%252BInteractions%252Bof%252BSteroid%252BAnalogs%252Bwith%252BHepatic%252BOrganic%252BAnion%252BTransporting%252BPolypeptides%26aulast%3DTuerkova%26aufirst%3DAlzbeta%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D29032021%26date%3D09062021%26volume%3D61%26issue%3D6%26spage%3D3109%26epage%3D3127" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Lijuan Yang, Guanghui Yang, Xiaolong Chen, Qiong Yang, Xiaojun Yao, Zhitong Bing, Yuzhen Niu, Liang Huang, <span class="NLM_string-name hlFld-ContribAuthor">Lei Yang</span>. </span><span class="cited-content_cbyCitation_article-title">Deep Scoring Neural Network Replacing the Scoring Function Components to Improve the Performance of Structure-Based Molecular Docking. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Neuroscience</span><span> <strong>2021,</strong> <em>12 </em>
                                    (12)
                                     , 2133-2142. <a href="https://doi.org/10.1021/acschemneuro.1c00110" title="DOI URL">https://doi.org/10.1021/acschemneuro.1c00110</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acschemneuro.1c00110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facschemneuro.1c00110%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Neuroscience%26atitle%3DDeep%252BScoring%252BNeural%252BNetwork%252BReplacing%252Bthe%252BScoring%252BFunction%252BComponents%252Bto%252BImprove%252Bthe%252BPerformance%252Bof%252BStructure-Based%252BMolecular%252BDocking%26aulast%3DYang%26aufirst%3DLijuan%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D01032021%26date%3D26052021%26date%3D03062021%26volume%3D12%26issue%3D12%26spage%3D2133%26epage%3D2142" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Raghuram Srinivas, Niraj Verma, Elfi Kraka, <span class="NLM_string-name hlFld-ContribAuthor">Eric C. Larson</span>. </span><span class="cited-content_cbyCitation_article-title">Deep Learning-Based Ligand Design Using Shared Latent Implicit Fingerprints from Collaborative Filtering. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2021,</strong> <em>61 </em>
                                    (5)
                                     , 2159-2174. <a href="https://doi.org/10.1021/acs.jcim.0c01355" title="DOI URL">https://doi.org/10.1021/acs.jcim.0c01355</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.0c01355&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.0c01355%26sid%3Dliteratum%253Aachs%26jtitle%3DJ.%2520Cheminf.%26date%3D2018%26volume%3D10%26spage%3D56%26aulast%3DSrinivas%26aufirst%3DR.%26jtitle%3DJ.%2520Cheminf.%26date%3D2018%26volume%3D10%26spage%3D56%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DDeep%252BLearning-Based%252BLigand%252BDesign%252BUsing%252BShared%252BLatent%252BImplicit%252BFingerprints%252Bfrom%252BCollaborative%252BFiltering%26aulast%3DSrinivas%26aufirst%3DRaghuram%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D20112020%26date%3D26042021%26volume%3D61%26issue%3D5%26spage%3D2159%26epage%3D2174" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Francois Berenger, Ashutosh Kumar, Kam Y. J. Zhang, <span class="NLM_string-name hlFld-ContribAuthor">Yoshihiro Yamanishi</span>. </span><span class="cited-content_cbyCitation_article-title">Lean-Docking: Exploiting Ligands’ Predicted Docking Scores to Accelerate Molecular Docking. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2021,</strong> <em>61 </em>
                                    (5)
                                     , 2341-2352. <a href="https://doi.org/10.1021/acs.jcim.0c01452" title="DOI URL">https://doi.org/10.1021/acs.jcim.0c01452</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.0c01452&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.0c01452%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DLean-Docking%25253A%252BExploiting%252BLigands%2525E2%252580%252599%252BPredicted%252BDocking%252BScores%252Bto%252BAccelerate%252BMolecular%252BDocking%26aulast%3DBerenger%26aufirst%3DFrancois%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D17122020%26date%3D16042021%26volume%3D61%26issue%3D5%26spage%3D2341%26epage%3D2352" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Hongyi Zhou, Hongnan Cao, <span class="NLM_string-name hlFld-ContribAuthor">Jeffrey Skolnick</span>. </span><span class="cited-content_cbyCitation_article-title">FRAGSITE: A Fragment-Based Approach for Virtual Ligand Screening. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2021,</strong> <em>61 </em>
                                    (4)
                                     , 2074-2089. <a href="https://doi.org/10.1021/acs.jcim.0c01160" title="DOI URL">https://doi.org/10.1021/acs.jcim.0c01160</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.0c01160&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.0c01160%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DFRAGSITE%25253A%252BA%252BFragment-Based%252BApproach%252Bfor%252BVirtual%252BLigand%252BScreening%26aulast%3DZhou%26aufirst%3DHongyi%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D05102020%26date%3D16032021%26volume%3D61%26issue%3D4%26spage%3D2074%26epage%3D2089" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Joanna Zarnecka, Iva Lukac, Stephen J. Messham, Alhusein Hussin, Francesco Coppola, Steven J. Enoch, Alexander G. Dossetter, Edward J. Griffen, <span class="NLM_string-name hlFld-ContribAuthor">Andrew G. Leach</span>. </span><span class="cited-content_cbyCitation_article-title">Mapping Ligand-Shape Space for Protein–Ligand Systems: Distinguishing Key-in-Lock and Hand-in-Glove Proteins. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2021,</strong> <em>61 </em>
                                    (4)
                                     , 1859-1874. <a href="https://doi.org/10.1021/acs.jcim.1c00089" title="DOI URL">https://doi.org/10.1021/acs.jcim.1c00089</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.1c00089&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.1c00089%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DMapping%252BLigand-Shape%252BSpace%252Bfor%252BProtein%2525E2%252580%252593Ligand%252BSystems%25253A%252BDistinguishing%252BKey-in-Lock%252Band%252BHand-in-Glove%252BProteins%26aulast%3DZarnecka%26aufirst%3DJoanna%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D28012021%26date%3D23032021%26volume%3D61%26issue%3D4%26spage%3D1859%26epage%3D1874" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Chao Li, Jun Sun, <span class="NLM_string-name hlFld-ContribAuthor">Vasile Palade</span>. </span><span class="cited-content_cbyCitation_article-title">MSLDOCK: Multi-Swarm Optimization for Flexible Ligand Docking and Virtual Screening. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2021,</strong> <em>61 </em>
                                    (3)
                                     , 1500-1515. <a href="https://doi.org/10.1021/acs.jcim.0c01358" title="DOI URL">https://doi.org/10.1021/acs.jcim.0c01358</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.0c01358&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.0c01358%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DMSLDOCK%25253A%252BMulti-Swarm%252BOptimization%252Bfor%252BFlexible%252BLigand%252BDocking%252Band%252BVirtual%252BScreening%26aulast%3DLi%26aufirst%3DChao%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D21112020%26date%3D04032021%26volume%3D61%26issue%3D3%26spage%3D1500%26epage%3D1515" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Reed M. Stein, Ying Yang, Trent E. Balius, Matt J. O’Meara, Jiankun Lyu, Jennifer Young, Khanh Tang, Brian K. Shoichet, <span class="NLM_string-name hlFld-ContribAuthor">John J. Irwin</span>. </span><span class="cited-content_cbyCitation_article-title">Property-Unmatched Decoys in Docking Benchmarks. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2021,</strong> <em>61 </em>
                                    (2)
                                     , 699-714. <a href="https://doi.org/10.1021/acs.jcim.0c00598" title="DOI URL">https://doi.org/10.1021/acs.jcim.0c00598</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.0c00598&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.0c00598%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DProperty-Unmatched%252BDecoys%252Bin%252BDocking%252BBenchmarks%26aulast%3DStein%26aufirst%3DReed%2BM.%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D29052020%26date%3D25012021%26volume%3D61%26issue%3D2%26spage%3D699%26epage%3D714" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Robert Schmidt, Florian Krull, Anna Lina Heinzke, <span class="NLM_string-name hlFld-ContribAuthor">Matthias Rarey</span>. </span><span class="cited-content_cbyCitation_article-title">Disconnected Maximum Common Substructures under Constraints. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2021,</strong> <em>61 </em>
                                    (1)
                                     , 167-178. <a href="https://doi.org/10.1021/acs.jcim.0c00741" title="DOI URL">https://doi.org/10.1021/acs.jcim.0c00741</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.0c00741&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.0c00741%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DDisconnected%252BMaximum%252BCommon%252BSubstructures%252Bunder%252BConstraints%26aulast%3DSchmidt%26aufirst%3DRobert%26date%3D2021%26date%3D2020%26date%3D2020%26date%3D30062020%26date%3D16122020%26volume%3D61%26issue%3D1%26spage%3D167%26epage%3D178" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Katherine J. Schultz, Sean M. Colby, Vivian S. Lin, Aaron T. Wright, <span class="NLM_string-name hlFld-ContribAuthor">Ryan S. Renslow</span>. </span><span class="cited-content_cbyCitation_article-title">Ligand- and Structure-Based Analysis of Deep Learning-Generated Potential α2a Adrenoceptor Agonists. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2021,</strong> <em>61 </em>
                                    (1)
                                     , 481-492. <a href="https://doi.org/10.1021/acs.jcim.0c01019" title="DOI URL">https://doi.org/10.1021/acs.jcim.0c01019</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.0c01019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.0c01019%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DLigand-%252Band%252BStructure-Based%252BAnalysis%252Bof%252BDeep%252BLearning-Generated%252BPotential%252B%2525CE%2525B12a%252BAdrenoceptor%252BAgonists%26aulast%3DSchultz%26aufirst%3DKatherine%2BJ.%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D28082020%26date%3D06012021%26volume%3D61%26issue%3D1%26spage%3D481%26epage%3D492" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Hugo Guterres, Sang-Jun Park, Wei Jiang, <span class="NLM_string-name hlFld-ContribAuthor">Wonpil Im</span>. </span><span class="cited-content_cbyCitation_article-title">Ligand-Binding-Site Refinement to Generate Reliable Holo Protein Structure Conformations from Apo Structures. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2021,</strong> <em>61 </em>
                                    (1)
                                     , 535-546. <a href="https://doi.org/10.1021/acs.jcim.0c01354" title="DOI URL">https://doi.org/10.1021/acs.jcim.0c01354</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.0c01354&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.0c01354%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DLigand-Binding-Site%252BRefinement%252Bto%252BGenerate%252BReliable%252BHolo%252BProtein%252BStructure%252BConformations%252Bfrom%252BApo%252BStructures%26aulast%3DGuterres%26aufirst%3DHugo%26date%3D2021%26date%3D2020%26date%3D2020%26date%3D20112020%26date%3D18122020%26volume%3D61%26issue%3D1%26spage%3D535%26epage%3D546" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Laura Chachulski, <span class="NLM_string-name hlFld-ContribAuthor">Björn Windshügel</span>. </span><span class="cited-content_cbyCitation_article-title">LEADS-FRAG: A Benchmark Data Set for Assessment of Fragment Docking Performance. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (12)
                                     , 6544-6554. <a href="https://doi.org/10.1021/acs.jcim.0c00693" title="DOI URL">https://doi.org/10.1021/acs.jcim.0c00693</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.0c00693&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.0c00693%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DLEADS-FRAG%25253A%252BA%252BBenchmark%252BData%252BSet%252Bfor%252BAssessment%252Bof%252BFragment%252BDocking%252BPerformance%26aulast%3DChachulski%26aufirst%3DLaura%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D18062020%26date%3D08122020%26volume%3D60%26issue%3D12%26spage%3D6544%26epage%3D6554" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Isaac de Araújo Matos, Nivan Bezerra da Costa Júnior, <span class="NLM_string-name hlFld-ContribAuthor">Flavia Carla Meotti</span>. </span><span class="cited-content_cbyCitation_article-title">Integration of an Inhibitor-like Rule and Structure-based Virtual Screening for the Discovery of Novel Myeloperoxidase Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (12)
                                     , 6408-6418. <a href="https://doi.org/10.1021/acs.jcim.0c00813" title="DOI URL">https://doi.org/10.1021/acs.jcim.0c00813</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.0c00813&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.0c00813%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DIntegration%252Bof%252Ban%252BInhibitor-like%252BRule%252Band%252BStructure-based%252BVirtual%252BScreening%252Bfor%252Bthe%252BDiscovery%252Bof%252BNovel%252BMyeloperoxidase%252BInhibitors%26aulast%3DMatos%26aufirst%3DIsaac%2Bde%2BAra%25C3%25BAjo%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D17072020%26date%3D03122020%26volume%3D60%26issue%3D12%26spage%3D6408%26epage%3D6418" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Jacques Boitreaud, Vincent Mallet, Carlos Oliver, <span class="NLM_string-name hlFld-ContribAuthor">Jérôme Waldispühl</span>. </span><span class="cited-content_cbyCitation_article-title">OptiMol: Optimization of Binding Affinities in Chemical Space for Drug Discovery. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (12)
                                     , 5658-5666. <a href="https://doi.org/10.1021/acs.jcim.0c00833" title="DOI URL">https://doi.org/10.1021/acs.jcim.0c00833</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.0c00833&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.0c00833%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DOptiMol%25253A%252BOptimization%252Bof%252BBinding%252BAffinities%252Bin%252BChemical%252BSpace%252Bfor%252BDrug%252BDiscovery%26aulast%3DBoitreaud%26aufirst%3DJacques%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D22072020%26date%3D07102020%26date%3D28092020%26volume%3D60%26issue%3D12%26spage%3D5658%26epage%3D5666" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">A. Acharya, R. Agarwal, M. B. Baker, J. Baudry, D. Bhowmik, S. Boehm, K. G. Byler, S. Y. Chen, L. Coates, C. J. Cooper, O. Demerdash, I. Daidone, J. D. Eblen, S. Ellingson, S. Forli, J. Glaser, J. C. Gumbart, J. Gunnels, O. Hernandez, S. Irle, D. W. Kneller, A. Kovalevsky, J. Larkin, T. J. Lawrence, S. LeGrand, S.-H. Liu, J.C. Mitchell, G. Park, J.M. Parks, A. Pavlova, L. Petridis, D. Poole, L. Pouchard, A. Ramanathan, D. M. Rogers, D. Santos-Martins, A. Scheinberg, A. Sedova, Y. Shen, J. C. Smith, M. D. Smith, C. Soto, A. Tsaris, M. Thavappiragasam, A. F. Tillack, J. V. Vermaas, V. Q. Vuong, J. Yin, S. Yoo, M. Zahran, <span class="NLM_string-name hlFld-ContribAuthor">L. Zanetti-Polzi</span>. </span><span class="cited-content_cbyCitation_article-title">Supercomputer-Based Ensemble Docking Drug Discovery Pipeline with Application to Covid-19. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (12)
                                     , 5832-5852. <a href="https://doi.org/10.1021/acs.jcim.0c01010" title="DOI URL">https://doi.org/10.1021/acs.jcim.0c01010</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.0c01010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.0c01010%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DSupercomputer-Based%252BEnsemble%252BDocking%252BDrug%252BDiscovery%252BPipeline%252Bwith%252BApplication%252Bto%252BCovid-19%26aulast%3DAcharya%26aufirst%3DA.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D27082020%26date%3D16122020%26volume%3D60%26issue%3D12%26spage%3D5832%26epage%3D5852" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Maximilian Grimm, Yang Liu, Xiaocong Yang, Chunya Bu, Zhixiong Xiao, <span class="NLM_string-name hlFld-ContribAuthor">Yang Cao</span>. </span><span class="cited-content_cbyCitation_article-title">LigMate: A Multifeature Integration Algorithm for Ligand-Similarity-Based Virtual Screening. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (12)
                                     , 6044-6053. <a href="https://doi.org/10.1021/acs.jcim.9b01210" title="DOI URL">https://doi.org/10.1021/acs.jcim.9b01210</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.9b01210&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.9b01210%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DLigMate%25253A%252BA%252BMultifeature%252BIntegration%252BAlgorithm%252Bfor%252BLigand-Similarity-Based%252BVirtual%252BScreening%26aulast%3DGrimm%26aufirst%3DMaximilian%26date%3D2020%26date%3D2020%26date%3D2019%26date%3D31122019%26date%3D16112020%26volume%3D60%26issue%3D12%26spage%3D6044%26epage%3D6053" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Paul Maragakis, Hunter Nisonoff, Brian Cole, <span class="NLM_string-name hlFld-ContribAuthor">David E. Shaw</span>. </span><span class="cited-content_cbyCitation_article-title">A Deep-Learning View of Chemical Space Designed to Facilitate Drug Discovery. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (10)
                                     , 4487-4496. <a href="https://doi.org/10.1021/acs.jcim.0c00321" title="DOI URL">https://doi.org/10.1021/acs.jcim.0c00321</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.0c00321&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.0c00321%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DA%252BDeep-Learning%252BView%252Bof%252BChemical%252BSpace%252BDesigned%252Bto%252BFacilitate%252BDrug%252BDiscovery%26aulast%3DMaragakis%26aufirst%3DPaul%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D01042020%26date%3D11092020%26date%3D22072020%26volume%3D60%26issue%3D10%26spage%3D4487%26epage%3D4496" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Huaipan Jiang, Mengran Fan, Jian Wang, Anup Sarma, Shruti Mohanty, Nikolay V. Dokholyan, Mehrdad Mahdavi, <span class="NLM_string-name hlFld-ContribAuthor">Mahmut T. Kandemir</span>. </span><span class="cited-content_cbyCitation_article-title">Guiding Conventional Protein–Ligand Docking Software with Convolutional Neural Networks. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (10)
                                     , 4594-4602. <a href="https://doi.org/10.1021/acs.jcim.0c00542" title="DOI URL">https://doi.org/10.1021/acs.jcim.0c00542</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.0c00542&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.0c00542%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DGuiding%252BConventional%252BProtein%2525E2%252580%252593Ligand%252BDocking%252BSoftware%252Bwith%252BConvolutional%252BNeural%252BNetworks%26aulast%3DJiang%26aufirst%3DHuaipan%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D15052020%26date%3D14102020%26volume%3D60%26issue%3D10%26spage%3D4594%26epage%3D4602" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Jean-Rémy Marchand, Tim Knehans, Amedeo Caflisch, <span class="NLM_string-name hlFld-ContribAuthor">Andreas Vitalis</span>. </span><span class="cited-content_cbyCitation_article-title">An ABSINTH-Based Protocol for Predicting Binding Affinities between Proteins and Small Molecules. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (10)
                                     , 5188-5202. <a href="https://doi.org/10.1021/acs.jcim.0c00558" title="DOI URL">https://doi.org/10.1021/acs.jcim.0c00558</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.0c00558&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.0c00558%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DAn%252BABSINTH-Based%252BProtocol%252Bfor%252BPredicting%252BBinding%252BAffinities%252Bbetween%252BProteins%252Band%252BSmall%252BMolecules%26aulast%3DMarchand%26aufirst%3DJean-R%25C3%25A9my%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D19052020%26date%3D23092020%26date%3D08092020%26volume%3D60%26issue%3D10%26spage%3D5188%26epage%3D5202" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Dingyan Wang, Zeen Yang, Bingqing Zhu, Xuefeng Mei, <span class="NLM_string-name hlFld-ContribAuthor">Xiaomin Luo</span>. </span><span class="cited-content_cbyCitation_article-title">Machine-Learning-Guided Cocrystal Prediction Based on Large Data Base. </span><span class="cited-content_cbyCitation_journal-name">Crystal Growth & Design</span><span> <strong>2020,</strong> <em>20 </em>
                                    (10)
                                     , 6610-6621. <a href="https://doi.org/10.1021/acs.cgd.0c00767" title="DOI URL">https://doi.org/10.1021/acs.cgd.0c00767</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.cgd.0c00767&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.cgd.0c00767%26sid%3Dliteratum%253Aachs%26jtitle%3DCrystal%2520Growth%2520%2526%2520Design%26atitle%3DMachine-Learning-Guided%252BCocrystal%252BPrediction%252BBased%252Bon%252BLarge%252BData%252BBase%26aulast%3DWang%26aufirst%3DDingyan%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D05062020%26date%3D30082020%26date%3D15092020%26date%3D31082020%26volume%3D20%26issue%3D10%26spage%3D6610%26epage%3D6621" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Ann E. Cleves, <span class="NLM_string-name hlFld-ContribAuthor">Ajay N. Jain</span>. </span><span class="cited-content_cbyCitation_article-title">Structure- and Ligand-Based Virtual Screening on DUD-E+: Performance Dependence on Approximations to the Binding Pocket. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (9)
                                     , 4296-4310. <a href="https://doi.org/10.1021/acs.jcim.0c00115" title="DOI URL">https://doi.org/10.1021/acs.jcim.0c00115</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.0c00115&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.0c00115%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DStructure-%252Band%252BLigand-Based%252BVirtual%252BScreening%252Bon%252BDUD-E%25252B%25253A%252BPerformance%252BDependence%252Bon%252BApproximations%252Bto%252Bthe%252BBinding%252BPocket%26aulast%3DCleves%26aufirst%3DAnn%2BE.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D31012020%26date%3D22042020%26date%3D09042020%26volume%3D60%26issue%3D9%26spage%3D4296%26epage%3D4310" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Siduo Jiang, Miklos Feher, Chris Williams, Brian Cole, <span class="NLM_string-name hlFld-ContribAuthor">David E. Shaw</span>. </span><span class="cited-content_cbyCitation_article-title">AutoPH4: An Automated Method for Generating Pharmacophore Models from Protein Binding Pockets. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (9)
                                     , 4326-4338. <a href="https://doi.org/10.1021/acs.jcim.0c00121" title="DOI URL">https://doi.org/10.1021/acs.jcim.0c00121</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.0c00121&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.0c00121%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DAutoPH4%25253A%252BAn%252BAutomated%252BMethod%252Bfor%252BGenerating%252BPharmacophore%252BModels%252Bfrom%252BProtein%252BBinding%252BPockets%26aulast%3DJiang%26aufirst%3DSiduo%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D31012020%26date%3D04082020%26date%3D08072020%26volume%3D60%26issue%3D9%26spage%3D4326%26epage%3D4338" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Jonathan Fine, Matthew Muhoberac, Guillaume Fraux, <span class="NLM_string-name hlFld-ContribAuthor">Gaurav Chopra</span>. </span><span class="cited-content_cbyCitation_article-title">DUBS: A Framework for Developing Directory of Useful Benchmarking Sets for Virtual Screening. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (9)
                                     , 4137-4143. <a href="https://doi.org/10.1021/acs.jcim.0c00122" title="DOI URL">https://doi.org/10.1021/acs.jcim.0c00122</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.0c00122&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.0c00122%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DDUBS%25253A%252BA%252BFramework%252Bfor%252BDeveloping%252BDirectory%252Bof%252BUseful%252BBenchmarking%252BSets%252Bfor%252BVirtual%252BScreening%26aulast%3DFine%26aufirst%3DJonathan%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D03022020%26date%3D03082020%26date%3D08072020%26volume%3D60%26issue%3D9%26spage%3D4137%26epage%3D4143" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Viet-Khoa Tran-Nguyen, Célien Jacquemard, <span class="NLM_string-name hlFld-ContribAuthor">Didier Rognan</span>. </span><span class="cited-content_cbyCitation_article-title">LIT-PCBA: An Unbiased Data Set for Machine Learning and Virtual Screening. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (9)
                                     , 4263-4273. <a href="https://doi.org/10.1021/acs.jcim.0c00155" title="DOI URL">https://doi.org/10.1021/acs.jcim.0c00155</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.0c00155&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.0c00155%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DLIT-PCBA%25253A%252BAn%252BUnbiased%252BData%252BSet%252Bfor%252BMachine%252BLearning%252Band%252BVirtual%252BScreening%26aulast%3DTran-Nguyen%26aufirst%3DViet-Khoa%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D12022020%26date%3D23042020%26date%3D13042020%26volume%3D60%26issue%3D9%26spage%3D4263%26epage%3D4273" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Paul G. Francoeur, Tomohide Masuda, Jocelyn Sunseri, Andrew Jia, Richard B. Iovanisci, Ian Snyder, <span class="NLM_string-name hlFld-ContribAuthor">David R. Koes</span>. </span><span class="cited-content_cbyCitation_article-title">Three-Dimensional Convolutional Neural Networks and a Cross-Docked Data Set for Structure-Based Drug Design. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (9)
                                     , 4200-4215. <a href="https://doi.org/10.1021/acs.jcim.0c00411" title="DOI URL">https://doi.org/10.1021/acs.jcim.0c00411</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.0c00411&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.0c00411%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DThree-Dimensional%252BConvolutional%252BNeural%252BNetworks%252Band%252Ba%252BCross-Docked%252BData%252BSet%252Bfor%252BStructure-Based%252BDrug%252BDesign%26aulast%3DFrancoeur%26aufirst%3DPaul%2BG.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D21042020%26date%3D10092020%26date%3D31082020%26volume%3D60%26issue%3D9%26spage%3D4200%26epage%3D4215" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">W. Patrick Walters,  (<span class="NLM_role">Guest Editor</span>)<span class="NLM_string-name hlFld-ContribAuthor">Renxiao Wang</span> (<span class="NLM_role">Associate Editor</span>). </span><span class="cited-content_cbyCitation_article-title">New Trends in Virtual Screening. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (9)
                                     , 4109-4111. <a href="https://doi.org/10.1021/acs.jcim.0c01009" title="DOI URL">https://doi.org/10.1021/acs.jcim.0c01009</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.0c01009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.0c01009%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DNew%252BTrends%252Bin%252BVirtual%252BScreening%26aulast%3DWalters%26aufirst%3DW.%2BPatrick%26date%3D2020%26date%3D2020%26volume%3D60%26issue%3D9%26spage%3D4109%26epage%3D4111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Yan Li, Yingduo Gao, M. Katharine Holloway, <span class="NLM_string-name hlFld-ContribAuthor">Renxiao Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Prediction of the Favorable Hydration Sites in a Protein Binding Pocket and Its Application to Scoring Function Formulation. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (9)
                                     , 4359-4375. <a href="https://doi.org/10.1021/acs.jcim.9b00619" title="DOI URL">https://doi.org/10.1021/acs.jcim.9b00619</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.9b00619&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.9b00619%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DPrediction%252Bof%252Bthe%252BFavorable%252BHydration%252BSites%252Bin%252Ba%252BProtein%252BBinding%252BPocket%252Band%252BIts%252BApplication%252Bto%252BScoring%252BFunction%252BFormulation%26aulast%3DLi%26aufirst%3DYan%26date%3D2020%26date%3D2020%26date%3D2019%26date%3D26072019%26date%3D01062020%26date%3D13052020%26volume%3D60%26issue%3D9%26spage%3D4359%26epage%3D4375" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Joseph A. Morrone, Jeffrey K. Weber, Tien Huynh, Heng Luo, <span class="NLM_string-name hlFld-ContribAuthor">Wendy D. Cornell</span>. </span><span class="cited-content_cbyCitation_article-title">Combining Docking Pose Rank and Structure with Deep Learning Improves Protein–Ligand Binding Mode Prediction over a Baseline Docking Approach. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (9)
                                     , 4170-4179. <a href="https://doi.org/10.1021/acs.jcim.9b00927" title="DOI URL">https://doi.org/10.1021/acs.jcim.9b00927</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.9b00927&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.9b00927%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DCombining%252BDocking%252BPose%252BRank%252Band%252BStructure%252Bwith%252BDeep%252BLearning%252BImproves%252BProtein%2525E2%252580%252593Ligand%252BBinding%252BMode%252BPrediction%252Bover%252Ba%252BBaseline%252BDocking%252BApproach%26aulast%3DMorrone%26aufirst%3DJoseph%2BA.%26date%3D2020%26date%3D2020%26date%3D2019%26date%3D04102019%26date%3D03032020%26date%3D20022020%26volume%3D60%26issue%3D9%26spage%3D4170%26epage%3D4179" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Wen-Ling Ye, Chao Shen, Guo-Li Xiong, Jun-Jie Ding, Ai-Ping Lu, Ting-Jun Hou, <span class="NLM_string-name hlFld-ContribAuthor">Dong-Sheng Cao</span>. </span><span class="cited-content_cbyCitation_article-title">Improving Docking-Based Virtual Screening Ability by Integrating Multiple Energy Auxiliary Terms from Molecular Docking Scoring. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (9)
                                     , 4216-4230. <a href="https://doi.org/10.1021/acs.jcim.9b00977" title="DOI URL">https://doi.org/10.1021/acs.jcim.9b00977</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.9b00977&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.9b00977%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DImproving%252BDocking-Based%252BVirtual%252BScreening%252BAbility%252Bby%252BIntegrating%252BMultiple%252BEnergy%252BAuxiliary%252BTerms%252Bfrom%252BMolecular%252BDocking%252BScoring%26aulast%3DYe%26aufirst%3DWen-Ling%26date%3D2020%26date%3D2020%26date%3D2019%26date%3D21102019%26date%3D11052020%26date%3D30042020%26volume%3D60%26issue%3D9%26spage%3D4216%26epage%3D4230" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Lindsey Burggraaff, Eelke B. Lenselink, Willem Jespers, Jesper van Engelen, Brandon J. Bongers, Marina Gorostiola González, Rongfang Liu, Holger H. Hoos, Herman W. T. van Vlijmen, Adriaan P. IJzerman, <span class="NLM_string-name hlFld-ContribAuthor">Gerard J. P. van Westen</span>. </span><span class="cited-content_cbyCitation_article-title">Successive Statistical and Structure-Based Modeling to Identify Chemically Novel Kinase Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (9)
                                     , 4283-4295. <a href="https://doi.org/10.1021/acs.jcim.9b01204" title="DOI URL">https://doi.org/10.1021/acs.jcim.9b01204</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.9b01204&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.9b01204%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DSuccessive%252BStatistical%252Band%252BStructure-Based%252BModeling%252Bto%252BIdentify%252BChemically%252BNovel%252BKinase%252BInhibitors%26aulast%3DBurggraaff%26aufirst%3DLindsey%26date%3D2020%26date%3D2020%26date%3D2019%26date%3D30122019%26date%3D12052020%26date%3D28042020%26volume%3D60%26issue%3D9%26spage%3D4283%26epage%3D4295" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Adrian Stecula, Muhammad S. Hussain, <span class="NLM_string-name hlFld-ContribAuthor">Ronald E. Viola</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of Novel Inhibitors of a Critical Brain Enzyme Using a Homology Model and a Deep Convolutional Neural Network. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (16)
                                     , 8867-8875. <a href="https://doi.org/10.1021/acs.jmedchem.0c00473" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00473</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00473&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00473%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252BNovel%252BInhibitors%252Bof%252Ba%252BCritical%252BBrain%252BEnzyme%252BUsing%252Ba%252BHomology%252BModel%252Band%252Ba%252BDeep%252BConvolutional%252BNeural%252BNetwork%26aulast%3DStecula%26aufirst%3DAdrian%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D20032020%26date%3D30072020%26volume%3D63%26issue%3D16%26spage%3D8867%26epage%3D8875" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Jack Scantlebury, Nathan Brown, Frank Von Delft, <span class="NLM_string-name hlFld-ContribAuthor">Charlotte M. Deane</span>. </span><span class="cited-content_cbyCitation_article-title">Data Set Augmentation Allows Deep Learning-Based Virtual Screening to Better Generalize to Unseen Target Classes and Highlight Important Binding Interactions. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (8)
                                     , 3722-3730. <a href="https://doi.org/10.1021/acs.jcim.0c00263" title="DOI URL">https://doi.org/10.1021/acs.jcim.0c00263</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.0c00263&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.0c00263%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DData%252BSet%252BAugmentation%252BAllows%252BDeep%252BLearning-Based%252BVirtual%252BScreening%252Bto%252BBetter%252BGeneralize%252Bto%252BUnseen%252BTarget%252BClasses%252Band%252BHighlight%252BImportant%252BBinding%252BInteractions%26aulast%3DScantlebury%26aufirst%3DJack%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D15032020%26date%3D04082020%26date%3D23072020%26volume%3D60%26issue%3D8%26spage%3D3722%26epage%3D3730" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Enade P. Istyastono, Muhammad Radifar, Nunung Yuniarti, Vivitri D. Prasasty, <span class="NLM_string-name hlFld-ContribAuthor">Sudi Mungkasi</span>. </span><span class="cited-content_cbyCitation_article-title">PyPLIF HIPPOS: A Molecular Interaction Fingerprinting Tool for Docking Results of AutoDock Vina and PLANTS. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (8)
                                     , 3697-3702. <a href="https://doi.org/10.1021/acs.jcim.0c00305" title="DOI URL">https://doi.org/10.1021/acs.jcim.0c00305</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.0c00305&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.0c00305%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DPyPLIF%252BHIPPOS%25253A%252BA%252BMolecular%252BInteraction%252BFingerprinting%252BTool%252Bfor%252BDocking%252BResults%252Bof%252BAutoDock%252BVina%252Band%252BPLANTS%26aulast%3DIstyastono%26aufirst%3DEnade%2BP.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D27032020%26date%3D04082020%26date%3D20072020%26volume%3D60%26issue%3D8%26spage%3D3697%26epage%3D3702" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Natasha Kamerlin, Mickaël G. Delcey, Sergio Manzetti, <span class="NLM_string-name hlFld-ContribAuthor">David van der Spoel</span>. </span><span class="cited-content_cbyCitation_article-title">Toward a Computational Ecotoxicity Assay. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (8)
                                     , 3792-3803. <a href="https://doi.org/10.1021/acs.jcim.0c00574" title="DOI URL">https://doi.org/10.1021/acs.jcim.0c00574</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.0c00574&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.0c00574%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DToward%252Ba%252BComputational%252BEcotoxicity%252BAssay%26aulast%3DKamerlin%26aufirst%3DNatasha%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D24052020%26date%3D21072020%26date%3D10072020%26volume%3D60%26issue%3D8%26spage%3D3792%26epage%3D3803" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Francesco Gentile, Vibudh Agrawal, Michael Hsing, Anh-Tien Ton, Fuqiang Ban, Ulf Norinder, Martin E. Gleave, <span class="NLM_string-name hlFld-ContribAuthor">Artem Cherkasov</span>. </span><span class="cited-content_cbyCitation_article-title">Deep Docking: A Deep Learning Platform for Augmentation of Structure Based Drug Discovery. </span><span class="cited-content_cbyCitation_journal-name">ACS Central Science</span><span> <strong>2020,</strong> <em>6 </em>
                                    (6)
                                     , 939-949. <a href="https://doi.org/10.1021/acscentsci.0c00229" title="DOI URL">https://doi.org/10.1021/acscentsci.0c00229</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acscentsci.0c00229&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facscentsci.0c00229%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Central%2520Science%26atitle%3DDeep%252BDocking%25253A%252BA%252BDeep%252BLearning%252BPlatform%252Bfor%252BAugmentation%252Bof%252BStructure%252BBased%252BDrug%252BDiscovery%26aulast%3DGentile%26aufirst%3DFrancesco%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D24022020%26date%3D19052020%26volume%3D6%26issue%3D6%26spage%3D939%26epage%3D949" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Dimitris Gazgalis, Mehreen Zaka, Bilal Haider Abbasi, Diomedes E. Logothetis, Mihaly Mezei, <span class="NLM_string-name hlFld-ContribAuthor">Meng Cui</span>. </span><span class="cited-content_cbyCitation_article-title">Protein Binding Pocket Optimization for Virtual High-Throughput Screening (vHTS) Drug Discovery. </span><span class="cited-content_cbyCitation_journal-name">ACS Omega</span><span> <strong>2020,</strong> <em>5 </em>
                                    (24)
                                     , 14297-14307. <a href="https://doi.org/10.1021/acsomega.0c00522" title="DOI URL">https://doi.org/10.1021/acsomega.0c00522</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsomega.0c00522&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsomega.0c00522%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Omega%26atitle%3DProtein%252BBinding%252BPocket%252BOptimization%252Bfor%252BVirtual%252BHigh-Throughput%252BScreening%252B%252528vHTS%252529%252BDrug%252BDiscovery%26aulast%3DGazgalis%26aufirst%3DDimitris%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D05022020%26date%3D28052020%26date%3D10062020%26volume%3D5%26issue%3D24%26spage%3D14297%26epage%3D14307" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Hussein Hassan-Harrirou, Ce Zhang, <span class="NLM_string-name hlFld-ContribAuthor">Thomas Lemmin</span>. </span><span class="cited-content_cbyCitation_article-title">RosENet: Improving Binding Affinity Prediction by Leveraging Molecular Mechanics Energies with an Ensemble of 3D Convolutional Neural Networks. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (6)
                                     , 2791-2802. <a href="https://doi.org/10.1021/acs.jcim.0c00075" title="DOI URL">https://doi.org/10.1021/acs.jcim.0c00075</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.0c00075&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.0c00075%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DRosENet%25253A%252BImproving%252BBinding%252BAffinity%252BPrediction%252Bby%252BLeveraging%252BMolecular%252BMechanics%252BEnergies%252Bwith%252Ban%252BEnsemble%252Bof%252B3D%252BConvolutional%252BNeural%252BNetworks%26aulast%3DHassan-Harrirou%26aufirst%3DHussein%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D22012020%26date%3D26052020%26date%3D11052020%26volume%3D60%26issue%3D6%26spage%3D2791%26epage%3D2802" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Marco Tutone, Ivana Pibiri, Riccardo Perriera, Ambra Campofelice, Giulia Culletta, Raffaella Melfi, Andrea Pace, Anna Maria Almerico, <span class="NLM_string-name hlFld-ContribAuthor">Laura Lentini</span>. </span><span class="cited-content_cbyCitation_article-title">Pharmacophore-Based Design of New Chemical Scaffolds as Translational Readthrough-Inducing Drugs (TRIDs). </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2020,</strong> <em>11 </em>
                                    (5)
                                     , 747-753. <a href="https://doi.org/10.1021/acsmedchemlett.9b00609" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.9b00609</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.9b00609&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.9b00609%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DPharmacophore-Based%252BDesign%252Bof%252BNew%252BChemical%252BScaffolds%252Bas%252BTranslational%252BReadthrough-Inducing%252BDrugs%252B%252528TRIDs%252529%26aulast%3DTutone%26aufirst%3DMarco%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D2019%26date%3D16122019%26date%3D18022020%26date%3D20022020%26date%3D18022020%26volume%3D11%26issue%3D5%26spage%3D747%26epage%3D753" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Hugo Guterres, <span class="NLM_string-name hlFld-ContribAuthor">Wonpil Im</span>. </span><span class="cited-content_cbyCitation_article-title">Improving Protein-Ligand Docking Results with High-Throughput Molecular Dynamics Simulations. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (4)
                                     , 2189-2198. <a href="https://doi.org/10.1021/acs.jcim.0c00057" title="DOI URL">https://doi.org/10.1021/acs.jcim.0c00057</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.0c00057&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.0c00057%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DImproving%252BProtein-Ligand%252BDocking%252BResults%252Bwith%252BHigh-Throughput%252BMolecular%252BDynamics%252BSimulations%26aulast%3DGuterres%26aufirst%3DHugo%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D18012020%26date%3D10042020%26date%3D31032020%26volume%3D60%26issue%3D4%26spage%3D2189%26epage%3D2198" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Peter R. Curran, Chris J. Radoux, Mihaela D. Smilova, Richard A. Sykes, Alicia P. Higueruelo, Anthony R. Bradley, Brian D. Marsden, David R. Spring, Tom L. Blundell, Andrew R. Leach, William R. Pitt, <span class="NLM_string-name hlFld-ContribAuthor">Jason C. Cole</span>. </span><span class="cited-content_cbyCitation_article-title">Hotspots API: A Python Package for the Detection of Small Molecule Binding Hotspots and Application to Structure-Based Drug Design. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (4)
                                     , 1911-1916. <a href="https://doi.org/10.1021/acs.jcim.9b00996" title="DOI URL">https://doi.org/10.1021/acs.jcim.9b00996</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.9b00996&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.9b00996%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DHotspots%252BAPI%25253A%252BA%252BPython%252BPackage%252Bfor%252Bthe%252BDetection%252Bof%252BSmall%252BMolecule%252BBinding%252BHotspots%252Band%252BApplication%252Bto%252BStructure-Based%252BDrug%252BDesign%26aulast%3DCurran%26aufirst%3DPeter%2BR.%26date%3D2020%26date%3D2020%26date%3D2019%26date%3D28102019%26date%3D02042020%26date%3D24032020%26volume%3D60%26issue%3D4%26spage%3D1911%26epage%3D1916" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Szymon Pach, Tim M. Sarter, Rafe Yousef, David Schaller, Silke Bergemann, Christoph Arkona, Jörg Rademann, Christoph Nitsche, <span class="NLM_string-name hlFld-ContribAuthor">Gerhard Wolber</span>. </span><span class="cited-content_cbyCitation_article-title">Catching a Moving Target: Comparative Modeling of Flaviviral NS2B-NS3 Reveals Small Molecule Zika Protease Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2020,</strong> <em>11 </em>
                                    (4)
                                     , 514-520. <a href="https://doi.org/10.1021/acsmedchemlett.9b00629" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.9b00629</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.9b00629&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.9b00629%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DCatching%252Ba%252BMoving%252BTarget%25253A%252BComparative%252BModeling%252Bof%252BFlaviviral%252BNS2B-NS3%252BReveals%252BSmall%252BMolecule%252BZika%252BProtease%252BInhibitors%26aulast%3DPach%26aufirst%3DSzymon%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D2019%26date%3D19122019%26date%3D03032020%26date%3D09032020%26date%3D03032020%26volume%3D11%26issue%3D4%26spage%3D514%26epage%3D520" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Eduardo H. B. Maia, Lucas Rolim Medaglia, Alisson Marques da Silva, <span class="NLM_string-name hlFld-ContribAuthor">Alex G. Taranto</span>. </span><span class="cited-content_cbyCitation_article-title">Molecular Architect: A User-Friendly Workflow for Virtual Screening. </span><span class="cited-content_cbyCitation_journal-name">ACS Omega</span><span> <strong>2020,</strong> <em>5 </em>
                                    (12)
                                     , 6628-6640. <a href="https://doi.org/10.1021/acsomega.9b04403" title="DOI URL">https://doi.org/10.1021/acsomega.9b04403</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsomega.9b04403&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsomega.9b04403%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Omega%26atitle%3DMolecular%252BArchitect%25253A%252BA%252BUser-Friendly%252BWorkflow%252Bfor%252BVirtual%252BScreening%26aulast%3DMaia%26aufirst%3DEduardo%2BH.%2BB.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D2019%26date%3D21122019%26date%3D06032020%26date%3D20032020%26volume%3D5%26issue%3D12%26spage%3D6628%26epage%3D6640" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Jonathan Fine, Janez Konc, Ram Samudrala, <span class="NLM_string-name hlFld-ContribAuthor">Gaurav Chopra</span>. </span><span class="cited-content_cbyCitation_article-title">CANDOCK: Chemical Atomic Network-Based Hierarchical Flexible Docking Algorithm Using Generalized Statistical Potentials. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (3)
                                     , 1509-1527. <a href="https://doi.org/10.1021/acs.jcim.9b00686" title="DOI URL">https://doi.org/10.1021/acs.jcim.9b00686</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.9b00686&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.9b00686%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DCANDOCK%25253A%252BChemical%252BAtomic%252BNetwork-Based%252BHierarchical%252BFlexible%252BDocking%252BAlgorithm%252BUsing%252BGeneralized%252BStatistical%252BPotentials%26aulast%3DFine%26aufirst%3DJonathan%26date%3D2020%26date%3D2020%26date%3D2019%26date%3D18082019%26date%3D03032020%26date%3D18022020%26volume%3D60%26issue%3D3%26spage%3D1509%26epage%3D1527" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Felipe R. S. Santos, William G. Lima, Eduardo H. B. Maia, Letícia C. Assis, Danilo Davyt, Alex Gutterres Taranto, <span class="NLM_string-name hlFld-ContribAuthor">Jaqueline M. S. Ferreira</span>. </span><span class="cited-content_cbyCitation_article-title">Identification of a Potential Zika Virus Inhibitor Targeting NS5 Methyltransferase Using Virtual Screening and Molecular Dynamics Simulations. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (2)
                                     , 562-568. <a href="https://doi.org/10.1021/acs.jcim.9b00809" title="DOI URL">https://doi.org/10.1021/acs.jcim.9b00809</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.9b00809&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.9b00809%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DIdentification%252Bof%252Ba%252BPotential%252BZika%252BVirus%252BInhibitor%252BTargeting%252BNS5%252BMethyltransferase%252BUsing%252BVirtual%252BScreening%252Band%252BMolecular%252BDynamics%252BSimulations%26aulast%3DSantos%26aufirst%3DFelipe%2BR.%2BS.%26date%3D2020%26date%3D2020%26date%3D2019%26date%3D18092019%26date%3D07022020%26date%3D27012020%26volume%3D60%26issue%3D2%26spage%3D562%26epage%3D568" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Felipe R. S. Santos, Damiana A. F. Nunes, William G. Lima, Danilo Davyt, Luciana L. Santos, Alex G. Taranto, <span class="NLM_string-name hlFld-ContribAuthor">Jaqueline M. S. Ferreira</span>. </span><span class="cited-content_cbyCitation_article-title">Identification of Zika Virus NS2B-NS3 Protease Inhibitors by Structure-Based Virtual Screening and Drug Repurposing Approaches. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (2)
                                     , 731-737. <a href="https://doi.org/10.1021/acs.jcim.9b00933" title="DOI URL">https://doi.org/10.1021/acs.jcim.9b00933</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.9b00933&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.9b00933%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DIdentification%252Bof%252BZika%252BVirus%252BNS2B-NS3%252BProtease%252BInhibitors%252Bby%252BStructure-Based%252BVirtual%252BScreening%252Band%252BDrug%252BRepurposing%252BApproaches%26aulast%3DSantos%26aufirst%3DFelipe%2BR.%2BS.%26date%3D2020%26date%3D2019%26date%3D2019%26date%3D07102019%26date%3D31122019%26date%3D18122019%26volume%3D60%26issue%3D2%26spage%3D731%26epage%3D737" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Minjian Yang, Bingzhong Tao, Chengjuan Chen, Wenqiang Jia, Shaolei Sun, Tiantai Zhang, <span class="NLM_string-name hlFld-ContribAuthor">Xiaojian Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Machine Learning Models Based on Molecular Fingerprints and an Extreme Gradient Boosting Method Lead to the Discovery of JAK2 Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2019,</strong> <em>59 </em>
                                    (12)
                                     , 5002-5012. <a href="https://doi.org/10.1021/acs.jcim.9b00798" title="DOI URL">https://doi.org/10.1021/acs.jcim.9b00798</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.9b00798&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.9b00798%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DMachine%252BLearning%252BModels%252BBased%252Bon%252BMolecular%252BFingerprints%252Band%252Ban%252BExtreme%252BGradient%252BBoosting%252BMethod%252BLead%252Bto%252Bthe%252BDiscovery%252Bof%252BJAK2%252BInhibitors%26aulast%3DYang%26aufirst%3DMinjian%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D15092019%26date%3D04122019%26date%3D20112019%26volume%3D59%26issue%3D12%26spage%3D5002%26epage%3D5012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Marleen
J. Meyer, Viktoria E. Neumann, Hannah Rosa Friesacher, Barbara Zdrazil, Jürgen Brockmöller, <span class="NLM_string-name hlFld-ContribAuthor">Mladen V. Tzvetkov</span>. </span><span class="cited-content_cbyCitation_article-title">Opioids as Substrates and Inhibitors of the Genetically Highly Variable Organic Cation Transporter OCT1. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (21)
                                     , 9890-9905. <a href="https://doi.org/10.1021/acs.jmedchem.9b01301" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01301</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01301&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b01301%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DOpioids%252Bas%252BSubstrates%252Band%252BInhibitors%252Bof%252Bthe%252BGenetically%252BHighly%252BVariable%252BOrganic%252BCation%252BTransporter%252BOCT1%26aulast%3DMeyer%26aufirst%3DMarleen%2BJ.%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D09082019%26date%3D04112019%26date%3D09102019%26volume%3D62%26issue%3D21%26spage%3D9890%26epage%3D9905" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Maritza Hernandez, Guo Liang Gan, Kirby Linvill, Carl Dukatz, Jun Feng, <span class="NLM_string-name hlFld-ContribAuthor">Govinda Bhisetti</span>. </span><span class="cited-content_cbyCitation_article-title">A Quantum-Inspired Method for Three-Dimensional Ligand-Based Virtual Screening. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2019,</strong> <em>59 </em>
                                    (10)
                                     , 4475-4485. <a href="https://doi.org/10.1021/acs.jcim.9b00195" title="DOI URL">https://doi.org/10.1021/acs.jcim.9b00195</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.9b00195&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.9b00195%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DA%252BQuantum-Inspired%252BMethod%252Bfor%252BThree-Dimensional%252BLigand-Based%252BVirtual%252BScreening%26aulast%3DHernandez%26aufirst%3DMaritza%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D04032019%26date%3D18102019%26volume%3D59%26issue%3D10%26spage%3D4475%26epage%3D4485" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Wen Torng, <span class="NLM_string-name hlFld-ContribAuthor">Russ B. Altman</span>. </span><span class="cited-content_cbyCitation_article-title">Graph Convolutional Neural Networks for Predicting Drug-Target Interactions. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2019,</strong> <em>59 </em>
                                    (10)
                                     , 4131-4149. <a href="https://doi.org/10.1021/acs.jcim.9b00628" title="DOI URL">https://doi.org/10.1021/acs.jcim.9b00628</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.9b00628&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.9b00628%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DGraph%252BConvolutional%252BNeural%252BNetworks%252Bfor%252BPredicting%252BDrug-Target%252BInteractions%26aulast%3DTorng%26aufirst%3DWen%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D29072019%26date%3D16102019%26date%3D03102019%26volume%3D59%26issue%3D10%26spage%3D4131%26epage%3D4149" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Miha Skalic, Davide Sabbadin, Boris Sattarov, Simone Sciabola, <span class="NLM_string-name hlFld-ContribAuthor">Gianni De Fabritiis</span>. </span><span class="cited-content_cbyCitation_article-title">From Target to Drug: Generative Modeling for the Multimodal Structure-Based Ligand Design. </span><span class="cited-content_cbyCitation_journal-name">Molecular Pharmaceutics</span><span> <strong>2019,</strong> <em>16 </em>
                                    (10)
                                     , 4282-4291. <a href="https://doi.org/10.1021/acs.molpharmaceut.9b00634" title="DOI URL">https://doi.org/10.1021/acs.molpharmaceut.9b00634</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.molpharmaceut.9b00634&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.molpharmaceut.9b00634%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Pharmaceutics%26atitle%3DFrom%252BTarget%252Bto%252BDrug%25253A%252BGenerative%252BModeling%252Bfor%252Bthe%252BMultimodal%252BStructure-Based%252BLigand%252BDesign%26aulast%3DSkalic%26aufirst%3DMiha%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D07062019%26date%3D22082019%26date%3D04082019%26date%3D10092019%26date%3D22082019%26volume%3D16%26issue%3D10%26spage%3D4282%26epage%3D4291" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Yunierkis Perez-Castillo, Stellamaris Sotomayor-Burneo, Karina Jimenes-Vargas, Mario Gonzalez-Rodriguez, Maykel Cruz-Monteagudo, Vinicio Armijos-Jaramillo, M. Natália
D. S. Cordeiro, Fernanda Borges, Aminael Sánchez-Rodríguez, <span class="NLM_string-name hlFld-ContribAuthor">Eduardo Tejera</span>. </span><span class="cited-content_cbyCitation_article-title">CompScore: Boosting Structure-Based Virtual Screening Performance by Incorporating Docking Scoring Function Components into Consensus Scoring. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2019,</strong> <em>59 </em>
                                    (9)
                                     , 3655-3666. <a href="https://doi.org/10.1021/acs.jcim.9b00343" title="DOI URL">https://doi.org/10.1021/acs.jcim.9b00343</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.9b00343&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.9b00343%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DCompScore%25253A%252BBoosting%252BStructure-Based%252BVirtual%252BScreening%252BPerformance%252Bby%252BIncorporating%252BDocking%252BScoring%252BFunction%252BComponents%252Binto%252BConsensus%252BScoring%26aulast%3DPerez-Castillo%26aufirst%3DYunierkis%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D23042019%26date%3D06092019%26date%3D26082019%26volume%3D59%26issue%3D9%26spage%3D3655%26epage%3D3666" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Süleyman Selim Çınaroğlu, <span class="NLM_string-name hlFld-ContribAuthor">Emel Timuçin</span>. </span><span class="cited-content_cbyCitation_article-title">Comparative Assessment of Seven Docking Programs on a Nonredundant Metalloprotein Subset of the PDBbind Refined. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2019,</strong> <em>59 </em>
                                    (9)
                                     , 3846-3859. <a href="https://doi.org/10.1021/acs.jcim.9b00346" title="DOI URL">https://doi.org/10.1021/acs.jcim.9b00346</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.9b00346&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.9b00346%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DComparative%252BAssessment%252Bof%252BSeven%252BDocking%252BPrograms%252Bon%252Ba%252BNonredundant%252BMetalloprotein%252BSubset%252Bof%252Bthe%252BPDBbind%252BRefined%26aulast%3D%25C3%2587%25C4%25B1naro%25C4%259Flu%26aufirst%3DS%25C3%25BCleyman%2BSelim%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D24042019%26date%3D10092019%26date%3D28082019%26volume%3D59%26issue%3D9%26spage%3D3846%26epage%3D3859" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Jaechang Lim, Seongok Ryu, Kyubyong Park, Yo Joong Choe, Jiyeon Ham, <span class="NLM_string-name hlFld-ContribAuthor">Woo Youn Kim</span>. </span><span class="cited-content_cbyCitation_article-title">Predicting Drug–Target Interaction Using a Novel Graph Neural Network with 3D Structure-Embedded Graph Representation. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2019,</strong> <em>59 </em>
                                    (9)
                                     , 3981-3988. <a href="https://doi.org/10.1021/acs.jcim.9b00387" title="DOI URL">https://doi.org/10.1021/acs.jcim.9b00387</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.9b00387&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.9b00387%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DPredicting%252BDrug%2525E2%252580%252593Target%252BInteraction%252BUsing%252Ba%252BNovel%252BGraph%252BNeural%252BNetwork%252Bwith%252B3D%252BStructure-Embedded%252BGraph%252BRepresentation%26aulast%3DLim%26aufirst%3DJaechang%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D09052019%26date%3D06092019%26date%3D23082019%26volume%3D59%26issue%3D9%26spage%3D3981%26epage%3D3988" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Ruqaiya Khalil, Sajda Ashraf, Asaad Khalid, <span class="NLM_string-name hlFld-ContribAuthor">Zaheer Ul-Haq</span>. </span><span class="cited-content_cbyCitation_article-title">Exploring Novel N-Myristoyltransferase Inhibitors: A Molecular Dynamics Simulation Approach. </span><span class="cited-content_cbyCitation_journal-name">ACS Omega</span><span> <strong>2019,</strong> <em>4 </em>
                                    (9)
                                     , 13658-13670. <a href="https://doi.org/10.1021/acsomega.9b00843" title="DOI URL">https://doi.org/10.1021/acsomega.9b00843</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsomega.9b00843&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsomega.9b00843%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Omega%26atitle%3DExploring%252BNovel%252BN-Myristoyltransferase%252BInhibitors%25253A%252BA%252BMolecular%252BDynamics%252BSimulation%252BApproach%26aulast%3DKhalil%26aufirst%3DRuqaiya%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D27032019%26date%3D25072019%26date%3D15082019%26volume%3D4%26issue%3D9%26spage%3D13658%26epage%3D13670" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Damien Monet, Nathan Desdouits, Michael Nilges, <span class="NLM_string-name hlFld-ContribAuthor">Arnaud Blondel</span>. </span><span class="cited-content_cbyCitation_article-title">mkgridXf: Consistent Identification of Plausible Binding Sites Despite the Elusive Nature of Cavities and Grooves in Protein Dynamics. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2019,</strong> <em>59 </em>
                                    (8)
                                     , 3506-3518. <a href="https://doi.org/10.1021/acs.jcim.9b00103" title="DOI URL">https://doi.org/10.1021/acs.jcim.9b00103</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.9b00103&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.9b00103%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DmkgridXf%25253A%252BConsistent%252BIdentification%252Bof%252BPlausible%252BBinding%252BSites%252BDespite%252Bthe%252BElusive%252BNature%252Bof%252BCavities%252Band%252BGrooves%252Bin%252BProtein%252BDynamics%26aulast%3DMonet%26aufirst%3DDamien%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D01022019%26date%3D25072019%26date%3D09072019%26volume%3D59%26issue%3D8%26spage%3D3506%26epage%3D3518" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Juan Pablo Arcon, Lucas A. Defelipe, Elias D. Lopez, Osvaldo Burastero, Carlos P. Modenutti, Xavier Barril, Marcelo A. Marti, <span class="NLM_string-name hlFld-ContribAuthor">Adrian G. Turjanski</span>. </span><span class="cited-content_cbyCitation_article-title">Cosolvent-Based Protein Pharmacophore for Ligand Enrichment in Virtual Screening. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2019,</strong> <em>59 </em>
                                    (8)
                                     , 3572-3583. <a href="https://doi.org/10.1021/acs.jcim.9b00371" title="DOI URL">https://doi.org/10.1021/acs.jcim.9b00371</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.9b00371&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.9b00371%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DCosolvent-Based%252BProtein%252BPharmacophore%252Bfor%252BLigand%252BEnrichment%252Bin%252BVirtual%252BScreening%26aulast%3DArcon%26aufirst%3DJuan%2BPablo%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D02052019%26date%3D16082019%26date%3D02082019%26volume%3D59%26issue%3D8%26spage%3D3572%26epage%3D3583" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Sami T. Kurkinen, Sakari Lätti, Olli T. Pentikäinen, <span class="NLM_string-name hlFld-ContribAuthor">Pekka A. Postila</span>. </span><span class="cited-content_cbyCitation_article-title">Getting Docking into Shape Using Negative Image-Based Rescoring. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2019,</strong> <em>59 </em>
                                    (8)
                                     , 3584-3599. <a href="https://doi.org/10.1021/acs.jcim.9b00383" title="DOI URL">https://doi.org/10.1021/acs.jcim.9b00383</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.9b00383&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.9b00383%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DGetting%252BDocking%252Binto%252BShape%252BUsing%252BNegative%252BImage-Based%252BRescoring%26aulast%3DKurkinen%26aufirst%3DSami%2BT.%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D08052019%26date%3D24072019%26date%3D10072019%26volume%3D59%26issue%3D8%26spage%3D3584%26epage%3D3599" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Wilfredo Evangelista Falcon, Sally R. Ellingson, Jeremy C. Smith, <span class="NLM_string-name hlFld-ContribAuthor">Jerome Baudry</span>. </span><span class="cited-content_cbyCitation_article-title">Ensemble Docking in Drug Discovery: How Many Protein Configurations from Molecular Dynamics Simulations are Needed To Reproduce Known Ligand Binding?. </span><span class="cited-content_cbyCitation_journal-name">The Journal of Physical Chemistry B</span><span> <strong>2019,</strong> <em>123 </em>
                                    (25)
                                     , 5189-5195. <a href="https://doi.org/10.1021/acs.jpcb.8b11491" title="DOI URL">https://doi.org/10.1021/acs.jpcb.8b11491</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jpcb.8b11491&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jpcb.8b11491%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Journal%2520of%2520Physical%2520Chemistry%2520B%26atitle%3DEnsemble%252BDocking%252Bin%252BDrug%252BDiscovery%25253A%252BHow%252BMany%252BProtein%252BConfigurations%252Bfrom%252BMolecular%252BDynamics%252BSimulations%252Bare%252BNeeded%252BTo%252BReproduce%252BKnown%252BLigand%252BBinding%25253F%26aulast%3DEvangelista%2BFalcon%26aufirst%3DWilfredo%26date%3D2019%26date%3D2019%26date%3D2018%26date%3D2019%26date%3D28112018%26date%3D26012019%26date%3D12022019%26date%3D29012019%26volume%3D123%26issue%3D25%26spage%3D5189%26epage%3D5195" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Jean-Paul Ebejer, Paul W. Finn, Wing Ki Wong, Charlotte M. Deane, <span class="NLM_string-name hlFld-ContribAuthor">Garrett M. Morris</span>. </span><span class="cited-content_cbyCitation_article-title">Ligity: A Non-Superpositional, Knowledge-Based Approach to Virtual Screening. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2019,</strong> <em>59 </em>
                                    (6)
                                     , 2600-2616. <a href="https://doi.org/10.1021/acs.jcim.8b00779" title="DOI URL">https://doi.org/10.1021/acs.jcim.8b00779</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.8b00779&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.8b00779%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DLigity%25253A%252BA%252BNon-Superpositional%25252C%252BKnowledge-Based%252BApproach%252Bto%252BVirtual%252BScreening%26aulast%3DEbejer%26aufirst%3DJean-Paul%26date%3D2019%26date%3D2019%26date%3D2018%26date%3D02112018%26date%3D04062019%26date%3D22052019%26volume%3D59%26issue%3D6%26spage%3D2600%26epage%3D2616" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Xiao Hu, <span class="NLM_string-name hlFld-ContribAuthor">Alessandro Contini</span>. </span><span class="cited-content_cbyCitation_article-title">Rescoring Virtual Screening Results with the MM-PBSA Methods: Beware of Internal Dielectric Constants. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2019,</strong> <em>59 </em>
                                    (6)
                                     , 2714-2728. <a href="https://doi.org/10.1021/acs.jcim.9b00095" title="DOI URL">https://doi.org/10.1021/acs.jcim.9b00095</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.9b00095&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.9b00095%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DRescoring%252BVirtual%252BScreening%252BResults%252Bwith%252Bthe%252BMM-PBSA%252BMethods%25253A%252BBeware%252Bof%252BInternal%252BDielectric%252BConstants%26aulast%3DHu%26aufirst%3DXiao%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D30012019%26date%3D15052019%26date%3D07052019%26volume%3D59%26issue%3D6%26spage%3D2714%26epage%3D2728" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">David Schaller, Szymon Pach, <span class="NLM_string-name hlFld-ContribAuthor">Gerhard Wolber</span>. </span><span class="cited-content_cbyCitation_article-title">PyRod: Tracing Water Molecules in Molecular Dynamics Simulations. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2019,</strong> <em>59 </em>
                                    (6)
                                     , 2818-2829. <a href="https://doi.org/10.1021/acs.jcim.9b00281" title="DOI URL">https://doi.org/10.1021/acs.jcim.9b00281</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.9b00281&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.9b00281%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DPyRod%25253A%252BTracing%252BWater%252BMolecules%252Bin%252BMolecular%252BDynamics%252BSimulations%26aulast%3DSchaller%26aufirst%3DDavid%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D03042019%26date%3D30052019%26date%3D22052019%26volume%3D59%26issue%3D6%26spage%3D2818%26epage%3D2829" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Angela Lopez-del Rio, Alfons Nonell-Canals, David Vidal, <span class="NLM_string-name hlFld-ContribAuthor">Alexandre Perera-Lluna</span>. </span><span class="cited-content_cbyCitation_article-title">Evaluation of Cross-Validation Strategies in Sequence-Based Binding Prediction Using Deep Learning. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2019,</strong> <em>59 </em>
                                    (4)
                                     , 1645-1657. <a href="https://doi.org/10.1021/acs.jcim.8b00663" title="DOI URL">https://doi.org/10.1021/acs.jcim.8b00663</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.8b00663&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.8b00663%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DEvaluation%252Bof%252BCross-Validation%252BStrategies%252Bin%252BSequence-Based%252BBinding%252BPrediction%252BUsing%252BDeep%252BLearning%26aulast%3DLopez-del%2BRio%26aufirst%3DAngela%26date%3D2019%26date%3D2019%26date%3D2018%26date%3D27092018%26date%3D22022019%26date%3D07022019%26volume%3D59%26issue%3D4%26spage%3D1645%26epage%3D1657" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Carlos Garcia-Hernandez, Alberto Fernández, <span class="NLM_string-name hlFld-ContribAuthor">Francesc Serratosa</span>. </span><span class="cited-content_cbyCitation_article-title">Ligand-Based Virtual Screening Using Graph Edit Distance as Molecular Similarity Measure. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2019,</strong> <em>59 </em>
                                    (4)
                                     , 1410-1421. <a href="https://doi.org/10.1021/acs.jcim.8b00820" title="DOI URL">https://doi.org/10.1021/acs.jcim.8b00820</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.8b00820&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.8b00820%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DLigand-Based%252BVirtual%252BScreening%252BUsing%252BGraph%252BEdit%252BDistance%252Bas%252BMolecular%252BSimilarity%252BMeasure%26aulast%3DGarcia-Hernandez%26aufirst%3DCarlos%26date%3D2019%26date%3D2019%26date%3D2018%26date%3D20112018%26date%3D10042019%26date%3D28032019%26volume%3D59%26issue%3D4%26spage%3D1410%26epage%3D1421" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Nobuaki Yasuo, <span class="NLM_string-name hlFld-ContribAuthor">Masakazu Sekijima</span>. </span><span class="cited-content_cbyCitation_article-title">Improved Method of Structure-Based Virtual Screening via Interaction-Energy-Based Learning. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2019,</strong> <em>59 </em>
                                    (3)
                                     , 1050-1061. <a href="https://doi.org/10.1021/acs.jcim.8b00673" title="DOI URL">https://doi.org/10.1021/acs.jcim.8b00673</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.8b00673&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.8b00673%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DImproved%252BMethod%252Bof%252BStructure-Based%252BVirtual%252BScreening%252Bvia%252BInteraction-Energy-Based%252BLearning%26aulast%3DYasuo%26aufirst%3DNobuaki%26date%3D2019%26date%3D2019%26date%3D2018%26date%3D01102018%26date%3D18032019%26date%3D26022019%26volume%3D59%26issue%3D3%26spage%3D1050%26epage%3D1061" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Miha Skalic, José Jiménez, Davide Sabbadin, <span class="NLM_string-name hlFld-ContribAuthor">Gianni De Fabritiis</span>. </span><span class="cited-content_cbyCitation_article-title">Shape-Based Generative Modeling for de Novo Drug Design. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2019,</strong> <em>59 </em>
                                    (3)
                                     , 1205-1214. <a href="https://doi.org/10.1021/acs.jcim.8b00706" title="DOI URL">https://doi.org/10.1021/acs.jcim.8b00706</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.8b00706&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.8b00706%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DShape-Based%252BGenerative%252BModeling%252Bfor%252Bde%252BNovo%252BDrug%252BDesign%26aulast%3DSkalic%26aufirst%3DMiha%26date%3D2019%26date%3D2019%26date%3D2018%26date%3D10102018%26date%3D28022019%26date%3D14022019%26volume%3D59%26issue%3D3%26spage%3D1205%26epage%3D1214" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Jochen Sieg, Florian Flachsenberg, <span class="NLM_string-name hlFld-ContribAuthor">Matthias Rarey</span>. </span><span class="cited-content_cbyCitation_article-title">In Need of Bias Control: Evaluating Chemical Data for Machine Learning in Structure-Based Virtual Screening. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2019,</strong> <em>59 </em>
                                    (3)
                                     , 947-961. <a href="https://doi.org/10.1021/acs.jcim.8b00712" title="DOI URL">https://doi.org/10.1021/acs.jcim.8b00712</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.8b00712&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.8b00712%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DIn%252BNeed%252Bof%252BBias%252BControl%25253A%252BEvaluating%252BChemical%252BData%252Bfor%252BMachine%252BLearning%252Bin%252BStructure-Based%252BVirtual%252BScreening%26aulast%3DSieg%26aufirst%3DJochen%26date%3D2019%26date%3D2019%26date%3D2018%26date%3D12102018%26date%3D05032019%26volume%3D59%26issue%3D3%26spage%3D947%26epage%3D961" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Minyi Su, Qifan Yang, Yu Du, Guoqin Feng, Zhihai Liu, Yan Li, <span class="NLM_string-name hlFld-ContribAuthor">Renxiao Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Comparative Assessment of Scoring Functions: The CASF-2016 Update. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2019,</strong> <em>59 </em>
                                    (2)
                                     , 895-913. <a href="https://doi.org/10.1021/acs.jcim.8b00545" title="DOI URL">https://doi.org/10.1021/acs.jcim.8b00545</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.8b00545&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.8b00545%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DComparative%252BAssessment%252Bof%252BScoring%252BFunctions%25253A%252BThe%252BCASF-2016%252BUpdate%26aulast%3DSu%26aufirst%3DMinyi%26date%3D2019%26date%3D2018%26date%3D2018%26date%3D14082018%26date%3D11122018%26date%3D27112018%26volume%3D59%26issue%3D2%26spage%3D895%26epage%3D913" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Shuangjia Zheng, Xin Yan, Yuedong Yang, <span class="NLM_string-name hlFld-ContribAuthor">Jun Xu</span>. </span><span class="cited-content_cbyCitation_article-title">Identifying Structure–Property Relationships through SMILES Syntax Analysis with Self-Attention Mechanism. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2019,</strong> <em>59 </em>
                                    (2)
                                     , 914-923. <a href="https://doi.org/10.1021/acs.jcim.8b00803" title="DOI URL">https://doi.org/10.1021/acs.jcim.8b00803</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.8b00803&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.8b00803%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DIdentifying%252BStructure%2525E2%252580%252593Property%252BRelationships%252Bthrough%252BSMILES%252BSyntax%252BAnalysis%252Bwith%252BSelf-Attention%252BMechanism%26aulast%3DZheng%26aufirst%3DShuangjia%26date%3D2019%26date%3D2019%26date%3D2018%26date%3D13112018%26date%3D06022019%26date%3D22012019%26volume%3D59%26issue%3D2%26spage%3D914%26epage%3D923" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Benjamin
P. Brown, Jeffrey Mendenhall, <span class="NLM_string-name hlFld-ContribAuthor">Jens Meiler</span>. </span><span class="cited-content_cbyCitation_article-title">BCL::MolAlign: Three-Dimensional Small Molecule Alignment for Pharmacophore Mapping. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2019,</strong> <em>59 </em>
                                    (2)
                                     , 689-701. <a href="https://doi.org/10.1021/acs.jcim.9b00020" title="DOI URL">https://doi.org/10.1021/acs.jcim.9b00020</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.9b00020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.9b00020%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DBCL%25253A%25253AMolAlign%25253A%252BThree-Dimensional%252BSmall%252BMolecule%252BAlignment%252Bfor%252BPharmacophore%252BMapping%26aulast%3DBrown%26aufirst%3DBenjamin%2BP.%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D04012019%26date%3D12022019%26date%3D01022019%26volume%3D59%26issue%3D2%26spage%3D689%26epage%3D701" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Matthew
P. Seddon, David A. Cosgrove, Martin J. Packer, <span class="NLM_string-name hlFld-ContribAuthor">Valerie J. Gillet</span>. </span><span class="cited-content_cbyCitation_article-title">Alignment-Free Molecular Shape Comparison Using Spectral Geometry: The Framework. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2019,</strong> <em>59 </em>
                                    (1)
                                     , 98-116. <a href="https://doi.org/10.1021/acs.jcim.8b00676" title="DOI URL">https://doi.org/10.1021/acs.jcim.8b00676</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.8b00676&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.8b00676%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DAlignment-Free%252BMolecular%252BShape%252BComparison%252BUsing%252BSpectral%252BGeometry%25253A%252BThe%252BFramework%26aulast%3DSeddon%26aufirst%3DMatthew%2BP.%26date%3D2019%26date%3D2018%26date%3D2018%26date%3D01102018%26date%3D05122018%26date%3D21112018%26volume%3D59%26issue%3D1%26spage%3D98%26epage%3D116" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Viet-Khoa Tran-Nguyen, Franck Da Silva, Guillaume Bret, <span class="NLM_string-name hlFld-ContribAuthor">Didier Rognan</span>. </span><span class="cited-content_cbyCitation_article-title">All in One: Cavity Detection, Druggability Estimate, Cavity-Based Pharmacophore Perception, and Virtual Screening. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2019,</strong> <em>59 </em>
                                    (1)
                                     , 573-585. <a href="https://doi.org/10.1021/acs.jcim.8b00684" title="DOI URL">https://doi.org/10.1021/acs.jcim.8b00684</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.8b00684&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.8b00684%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DAll%252Bin%252BOne%25253A%252BCavity%252BDetection%25252C%252BDruggability%252BEstimate%25252C%252BCavity-Based%252BPharmacophore%252BPerception%25252C%252Band%252BVirtual%252BScreening%26aulast%3DTran-Nguyen%26aufirst%3DViet-Khoa%26date%3D2019%26date%3D2018%26date%3D2018%26date%3D02102018%26date%3D04012019%26date%3D18122018%26volume%3D59%26issue%3D1%26spage%3D573%26epage%3D585" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Haiyi Chen, Weitao Fu, Zhe Wang, Xuwen Wang, Tailong Lei, Feng Zhu, Dan Li, Shan Chang, Lei Xu, <span class="NLM_string-name hlFld-ContribAuthor">Tingjun Hou</span>. </span><span class="cited-content_cbyCitation_article-title">Reliability of Docking-Based Virtual Screening for GPCR Ligands with Homology Modeled Structures: A Case Study of the Angiotensin II Type I Receptor. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Neuroscience</span><span> <strong>2019,</strong> <em>10 </em>
                                    (1)
                                     , 677-689. <a href="https://doi.org/10.1021/acschemneuro.8b00489" title="DOI URL">https://doi.org/10.1021/acschemneuro.8b00489</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acschemneuro.8b00489&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facschemneuro.8b00489%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Neuroscience%26atitle%3DReliability%252Bof%252BDocking-Based%252BVirtual%252BScreening%252Bfor%252BGPCR%252BLigands%252Bwith%252BHomology%252BModeled%252BStructures%25253A%252BA%252BCase%252BStudy%252Bof%252Bthe%252BAngiotensin%252BII%252BType%252BI%252BReceptor%26aulast%3DChen%26aufirst%3DHaiyi%26date%3D2019%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D13092018%26date%3D28092018%26date%3D17102018%26date%3D28092018%26volume%3D10%26issue%3D1%26spage%3D677%26epage%3D689" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Hongyi Zhou, Hongnan Cao, <span class="NLM_string-name hlFld-ContribAuthor">Jeffrey Skolnick</span>. </span><span class="cited-content_cbyCitation_article-title">FINDSITEcomb2.0: A New Approach for Virtual Ligand Screening of Proteins and Virtual Target Screening of Biomolecules. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2018,</strong> <em>58 </em>
                                    (11)
                                     , 2343-2354. <a href="https://doi.org/10.1021/acs.jcim.8b00309" title="DOI URL">https://doi.org/10.1021/acs.jcim.8b00309</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.8b00309&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.8b00309%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DFINDSITEcomb2.0%25253A%252BA%252BNew%252BApproach%252Bfor%252BVirtual%252BLigand%252BScreening%252Bof%252BProteins%252Band%252BVirtual%252BTarget%252BScreening%252Bof%252BBiomolecules%26aulast%3DZhou%26aufirst%3DHongyi%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D21052018%26date%3D16102018%26date%3D02102018%26volume%3D58%26issue%3D11%26spage%3D2343%26epage%3D2354" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Fergus Imrie, Anthony R. Bradley, Mihaela van der Schaar, <span class="NLM_string-name hlFld-ContribAuthor">Charlotte M. Deane</span>. </span><span class="cited-content_cbyCitation_article-title">Protein Family-Specific Models Using Deep Neural Networks and Transfer Learning Improve Virtual Screening and Highlight the Need for More Data. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2018,</strong> <em>58 </em>
                                    (11)
                                     , 2319-2330. <a href="https://doi.org/10.1021/acs.jcim.8b00350" title="DOI URL">https://doi.org/10.1021/acs.jcim.8b00350</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.8b00350&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.8b00350%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DProtein%252BFamily-Specific%252BModels%252BUsing%252BDeep%252BNeural%252BNetworks%252Band%252BTransfer%252BLearning%252BImprove%252BVirtual%252BScreening%252Band%252BHighlight%252Bthe%252BNeed%252Bfor%252BMore%252BData%26aulast%3DImrie%26aufirst%3DFergus%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D04062018%26date%3D16102018%26date%3D01102018%26volume%3D58%26issue%3D11%26spage%3D2319%26epage%3D2330" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Roberta Rocca, Federica Moraca, Giosuè Costa, Carmine Talarico, Francesco Ortuso, Silvia Da Ros, Giulia Nicoletto, Claudia Sissi, Stefano Alcaro, <span class="NLM_string-name hlFld-ContribAuthor">Anna Artese</span>. </span><span class="cited-content_cbyCitation_article-title">In Silico Identification of Piperidinyl-amine Derivatives as Novel Dual Binders of Oncogene c-myc/c-Kit G-quadruplexes. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2018,</strong> <em>9 </em>
                                    (8)
                                     , 848-853. <a href="https://doi.org/10.1021/acsmedchemlett.8b00275" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.8b00275</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.8b00275&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.8b00275%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DIn%252BSilico%252BIdentification%252Bof%252BPiperidinyl-amine%252BDerivatives%252Bas%252BNovel%252BDual%252BBinders%252Bof%252BOncogene%252Bc-myc%25252Fc-Kit%252BG-quadruplexes%26aulast%3DRocca%26aufirst%3DRoberta%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D18062018%26date%3D10072018%26date%3D18072018%26date%3D10072018%26volume%3D9%26issue%3D8%26spage%3D848%26epage%3D853" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Mariama Jaiteh, Alexey Zeifman, Marcus Saarinen, Per Svenningsson, Jose Bréa, Maria Isabel Loza, <span class="NLM_string-name hlFld-ContribAuthor">Jens Carlsson</span>. </span><span class="cited-content_cbyCitation_article-title">Docking Screens for Dual Inhibitors of Disparate Drug Targets for Parkinson’s Disease. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>61 </em>
                                    (12)
                                     , 5269-5278. <a href="https://doi.org/10.1021/acs.jmedchem.8b00204" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b00204</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b00204&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.8b00204%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDocking%252BScreens%252Bfor%252BDual%252BInhibitors%252Bof%252BDisparate%252BDrug%252BTargets%252Bfor%252BParkinson%2525E2%252580%252599s%252BDisease%26aulast%3DJaiteh%26aufirst%3DMariama%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D06022018%26date%3D15062018%26date%3D24052018%26volume%3D61%26issue%3D12%26spage%3D5269%26epage%3D5278" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Alessandro Pedretti, Angelica Mazzolari, <span class="NLM_string-name hlFld-ContribAuthor">Giulio Vistoli</span>. </span><span class="cited-content_cbyCitation_article-title">WarpEngine, a Flexible Platform for Distributed Computing Implemented in the VEGA Program and Specially Targeted for Virtual Screening Studies. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2018,</strong> <em>58 </em>
                                    (6)
                                     , 1154-1160. <a href="https://doi.org/10.1021/acs.jcim.8b00086" title="DOI URL">https://doi.org/10.1021/acs.jcim.8b00086</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.8b00086&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.8b00086%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DWarpEngine%25252C%252Ba%252BFlexible%252BPlatform%252Bfor%252BDistributed%252BComputing%252BImplemented%252Bin%252Bthe%252BVEGA%252BProgram%252Band%252BSpecially%252BTargeted%252Bfor%252BVirtual%252BScreening%252BStudies%26aulast%3DPedretti%26aufirst%3DAlessandro%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D13022018%26date%3D21052018%26date%3D10052018%26volume%3D58%26issue%3D6%26spage%3D1154%26epage%3D1160" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Izhar Wallach, <span class="NLM_string-name hlFld-ContribAuthor">Abraham Heifets</span>. </span><span class="cited-content_cbyCitation_article-title">Most Ligand-Based Classification Benchmarks Reward Memorization Rather than Generalization. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2018,</strong> <em>58 </em>
                                    (5)
                                     , 916-932. <a href="https://doi.org/10.1021/acs.jcim.7b00403" title="DOI URL">https://doi.org/10.1021/acs.jcim.7b00403</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.7b00403&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.7b00403%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DMost%252BLigand-Based%252BClassification%252BBenchmarks%252BReward%252BMemorization%252BRather%252Bthan%252BGeneralization%26aulast%3DWallach%26aufirst%3DIzhar%26date%3D2018%26date%3D2018%26date%3D2017%26date%3D01072017%26date%3D08052018%26date%3D26042018%26volume%3D58%26issue%3D5%26spage%3D916%26epage%3D932" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Jie Xia, Terry-Elinor Reid, Song Wu, Liangren Zhang, <span class="NLM_string-name hlFld-ContribAuthor">Xiang Simon Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Maximal Unbiased Benchmarking Data Sets for Human Chemokine Receptors and Comparative Analysis. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2018,</strong> <em>58 </em>
                                    (5)
                                     , 1104-1120. <a href="https://doi.org/10.1021/acs.jcim.8b00004" title="DOI URL">https://doi.org/10.1021/acs.jcim.8b00004</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.8b00004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.8b00004%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DMaximal%252BUnbiased%252BBenchmarking%252BData%252BSets%252Bfor%252BHuman%252BChemokine%252BReceptors%252Band%252BComparative%252BAnalysis%26aulast%3DXia%26aufirst%3DJie%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D02012018%26date%3D08052018%26date%3D26042018%26volume%3D58%26issue%3D5%26spage%3D1104%26epage%3D1120" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Claas Strecker, <span class="NLM_string-name hlFld-ContribAuthor">Bernd Meyer</span>. </span><span class="cited-content_cbyCitation_article-title">Plasticity of the Binding Site of Renin: Optimized Selection of Protein Structures for Ensemble Docking. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2018,</strong> <em>58 </em>
                                    (5)
                                     , 1121-1131. <a href="https://doi.org/10.1021/acs.jcim.8b00010" title="DOI URL">https://doi.org/10.1021/acs.jcim.8b00010</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.8b00010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.8b00010%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DPlasticity%252Bof%252Bthe%252BBinding%252BSite%252Bof%252BRenin%25253A%252BOptimized%252BSelection%252Bof%252BProtein%252BStructures%252Bfor%252BEnsemble%252BDocking%26aulast%3DStrecker%26aufirst%3DClaas%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D09012018%26date%3D02052018%26date%3D23042018%26volume%3D58%26issue%3D5%26spage%3D1121%26epage%3D1131" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Raquel Rodríguez-Pérez, Tomoyuki Miyao, Swarit Jasial, Martin Vogt, <span class="NLM_string-name hlFld-ContribAuthor">Jürgen Bajorath</span>. </span><span class="cited-content_cbyCitation_article-title">Prediction of Compound Profiling Matrices Using Machine Learning. </span><span class="cited-content_cbyCitation_journal-name">ACS Omega</span><span> <strong>2018,</strong> <em>3 </em>
                                    (4)
                                     , 4713-4723. <a href="https://doi.org/10.1021/acsomega.8b00462" title="DOI URL">https://doi.org/10.1021/acsomega.8b00462</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsomega.8b00462&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsomega.8b00462%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Omega%26atitle%3DPrediction%252Bof%252BCompound%252BProfiling%252BMatrices%252BUsing%252BMachine%252BLearning%26aulast%3DRodr%25C3%25ADguez-P%25C3%25A9rez%26aufirst%3DRaquel%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D12032018%26date%3D20042018%26volume%3D3%26issue%3D4%26spage%3D4713%26epage%3D4723" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="NLM_string-name hlFld-ContribAuthor">Suman K. Chakravarti</span>. </span><span class="cited-content_cbyCitation_article-title">Distributed Representation of Chemical Fragments. </span><span class="cited-content_cbyCitation_journal-name">ACS Omega</span><span> <strong>2018,</strong> <em>3 </em>
                                    (3)
                                     , 2825-2836. <a href="https://doi.org/10.1021/acsomega.7b02045" title="DOI URL">https://doi.org/10.1021/acsomega.7b02045</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsomega.7b02045&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsomega.7b02045%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Omega%26atitle%3DDistributed%252BRepresentation%252Bof%252BChemical%252BFragments%26aulast%3DChakravarti%26aufirst%3DSuman%2BK.%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D2017%26date%3D21122017%26date%3D23022018%26volume%3D3%26issue%3D3%26spage%3D2825%26epage%3D2836" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Zhe Wang, Yu Kang, Dan Li, Huiyong Sun, Xiaowu Dong, Xiaojun Yao, Lei Xu, Shan Chang, Youyong Li, <span class="NLM_string-name hlFld-ContribAuthor">Tingjun Hou</span>. </span><span class="cited-content_cbyCitation_article-title">Benchmark Study Based on 2P2IDB to Gain Insights into the Discovery of Small-Molecule PPI Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">The Journal of Physical Chemistry B</span><span> <strong>2018,</strong> <em>122 </em>
                                    (9)
                                     , 2544-2555. <a href="https://doi.org/10.1021/acs.jpcb.7b12658" title="DOI URL">https://doi.org/10.1021/acs.jpcb.7b12658</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jpcb.7b12658&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jpcb.7b12658%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Journal%2520of%2520Physical%2520Chemistry%2520B%26atitle%3DBenchmark%252BStudy%252BBased%252Bon%252B2P2IDB%252Bto%252BGain%252BInsights%252Binto%252Bthe%252BDiscovery%252Bof%252BSmall-Molecule%252BPPI%252BInhibitors%26aulast%3DWang%26aufirst%3DZhe%26date%3D2018%26date%3D2018%26date%3D2017%26date%3D2018%26date%3D23122017%26date%3D03022018%26date%3D22022018%26date%3D08022018%26volume%3D122%26issue%3D9%26spage%3D2544%26epage%3D2555" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Stephen J.  Barigye</span>, <span class="hlFld-ContribAuthor ">Matheus P.  Freitas</span>, <span class="hlFld-ContribAuthor ">Priscila  Ausina</span>, <span class="hlFld-ContribAuthor ">Patricia  Zancan</span>, <span class="hlFld-ContribAuthor ">Mauro  Sola-Penna</span>, and <span class="hlFld-ContribAuthor ">Juan A.  Castillo-Garit</span>  . </span><span class="cited-content_cbyCitation_article-title">Discrete Fourier Transform-Based Multivariate Image Analysis: Application to Modeling of Aromatase Inhibitory Activity. </span><span class="cited-content_cbyCitation_journal-name">ACS Combinatorial Science</span><span> <strong>2018,</strong> <em>20 </em>
                                    (2)
                                     , 75-81. <a href="https://doi.org/10.1021/acscombsci.7b00155" title="DOI URL">https://doi.org/10.1021/acscombsci.7b00155</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acscombsci.7b00155&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facscombsci.7b00155%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Combinatorial%2520Science%26atitle%3DDiscrete%252BFourier%252BTransform-Based%252BMultivariate%252BImage%252BAnalysis%25253A%252BApplication%252Bto%252BModeling%252Bof%252BAromatase%252BInhibitory%252BActivity%26aulast%3DBarigye%26aufirst%3DStephen%2BJ.%26date%3D2018%26date%3D2018%26date%3D2017%26date%3D2017%26date%3D23102017%26date%3D17122017%26date%3D22012018%26date%3D12022018%26date%3D03012018%26volume%3D20%26issue%3D2%26spage%3D75%26epage%3D81" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Tao  Liang</span>, <span class="hlFld-ContribAuthor ">Yuan  Yuan</span>, <span class="hlFld-ContribAuthor ">Ran  Wang</span>, <span class="hlFld-ContribAuthor ">Yanzhi  Guo</span>, <span class="hlFld-ContribAuthor ">Menglong  Li</span>, <span class="hlFld-ContribAuthor ">Xuemei  Pu</span>, and <span class="hlFld-ContribAuthor ">Chuan  Li</span>  . </span><span class="cited-content_cbyCitation_article-title">Structural Features and Ligand Selectivity for 10 Intermediates in the Activation Process of β2-Adrenergic Receptor. </span><span class="cited-content_cbyCitation_journal-name">ACS Omega</span><span> <strong>2017,</strong> <em>2 </em>
                                    (12)
                                     , 8557-8567. <a href="https://doi.org/10.1021/acsomega.7b01031" title="DOI URL">https://doi.org/10.1021/acsomega.7b01031</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsomega.7b01031&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsomega.7b01031%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Omega%26atitle%3DStructural%252BFeatures%252Band%252BLigand%252BSelectivity%252Bfor%252B10%252BIntermediates%252Bin%252Bthe%252BActivation%252BProcess%252Bof%252B%2525CE%2525B22-Adrenergic%252BReceptor%26aulast%3DLiang%26aufirst%3DTao%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D19072017%26date%3D14092017%26date%3D31122017%26volume%3D2%26issue%3D12%26spage%3D8557%26epage%3D8567" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Daniela  Trisciuzzi</span>, <span class="hlFld-ContribAuthor ">Domenico  Alberga</span>, <span class="hlFld-ContribAuthor ">Kamel  Mansouri</span>, <span class="hlFld-ContribAuthor ">Richard  Judson</span>, <span class="hlFld-ContribAuthor ">Ettore  Novellino</span>, <span class="hlFld-ContribAuthor ">Giuseppe Felice  Mangiatordi</span>, and <span class="hlFld-ContribAuthor ">Orazio  Nicolotti</span>  . </span><span class="cited-content_cbyCitation_article-title">Predictive Structure-Based Toxicology Approaches To Assess the Androgenic Potential of Chemicals. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2017,</strong> <em>57 </em>
                                    (11)
                                     , 2874-2884. <a href="https://doi.org/10.1021/acs.jcim.7b00420" title="DOI URL">https://doi.org/10.1021/acs.jcim.7b00420</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.7b00420&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.7b00420%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DPredictive%252BStructure-Based%252BToxicology%252BApproaches%252BTo%252BAssess%252Bthe%252BAndrogenic%252BPotential%252Bof%252BChemicals%26aulast%3DTrisciuzzi%26aufirst%3DDaniela%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D13072017%26date%3D26102017%26date%3D27112017%26date%3D12102017%26volume%3D57%26issue%3D11%26spage%3D2874%26epage%3D2884" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Barbara  Männel</span>, <span class="hlFld-ContribAuthor ">Mariama  Jaiteh</span>, <span class="hlFld-ContribAuthor ">Alexey  Zeifman</span>, <span class="hlFld-ContribAuthor ">Alena  Randakova</span>, <span class="hlFld-ContribAuthor ">Dorothee  Möller</span>, <span class="hlFld-ContribAuthor ">Harald  Hübner</span>, <span class="hlFld-ContribAuthor ">Peter  Gmeiner</span>, and <span class="hlFld-ContribAuthor ">Jens  Carlsson</span>  . </span><span class="cited-content_cbyCitation_article-title">Structure-Guided Screening for Functionally Selective D2 Dopamine Receptor Ligands from a Virtual Chemical Library. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Biology</span><span> <strong>2017,</strong> <em>12 </em>
                                    (10)
                                     , 2652-2661. <a href="https://doi.org/10.1021/acschembio.7b00493" title="DOI URL">https://doi.org/10.1021/acschembio.7b00493</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acschembio.7b00493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facschembio.7b00493%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Biology%26atitle%3DStructure-Guided%252BScreening%252Bfor%252BFunctionally%252BSelective%252BD2%252BDopamine%252BReceptor%252BLigands%252Bfrom%252Ba%252BVirtual%252BChemical%252BLibrary%26aulast%3DM%25C3%25A4nnel%26aufirst%3DBarbara%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D15062017%26date%3D28082017%26date%3D19092017%26date%3D20102017%26date%3D28082017%26volume%3D12%26issue%3D10%26spage%3D2652%26epage%3D2661" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lizi  Xia</span>, <span class="hlFld-ContribAuthor ">Wessel A. C.  Burger</span>, <span class="hlFld-ContribAuthor ">Jacobus P. D.  van Veldhoven</span>, <span class="hlFld-ContribAuthor ">Boaz J.  Kuiper</span>, <span class="hlFld-ContribAuthor ">Tirsa T.  van Duijl</span>, <span class="hlFld-ContribAuthor ">Eelke B.  Lenselink</span>, <span class="hlFld-ContribAuthor ">Ellen  Paasman</span>, <span class="hlFld-ContribAuthor ">Laura H.  Heitman</span>, and <span class="hlFld-ContribAuthor ">Adriaan P.  IJzerman</span>  . </span><span class="cited-content_cbyCitation_article-title">Structure–Affinity Relationships and Structure–Kinetics Relationships of Pyrido[2,1-f]purine-2,4-dione Derivatives as Human Adenosine A3 Receptor Antagonists. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2017,</strong> <em>60 </em>
                                    (17)
                                     , 7555-7568. <a href="https://doi.org/10.1021/acs.jmedchem.7b00950" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b00950</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b00950&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.7b00950%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DStructure%2525E2%252580%252593Affinity%252BRelationships%252Band%252BStructure%2525E2%252580%252593Kinetics%252BRelationships%252Bof%252BPyrido%25255B2%25252C1-f%25255Dpurine-2%25252C4-dione%252BDerivatives%252Bas%252BHuman%252BAdenosine%252BA3%252BReceptor%252BAntagonists%26aulast%3DXia%26aufirst%3DLizi%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D28062017%26date%3D30082017%26date%3D14092017%26date%3D14082017%26volume%3D60%26issue%3D17%26spage%3D7555%26epage%3D7568" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Bryce K.  Allen</span>, <span class="hlFld-ContribAuthor ">Saurabh  Mehta</span>, <span class="hlFld-ContribAuthor ">Stuart W. J.  Ember</span>, <span class="hlFld-ContribAuthor ">Jin-Yi  Zhu</span>, <span class="hlFld-ContribAuthor ">Ernst  Schönbrunn</span>, <span class="hlFld-ContribAuthor ">Nagi G.  Ayad</span>, and <span class="hlFld-ContribAuthor ">Stephan C.  Schürer</span>  . </span><span class="cited-content_cbyCitation_article-title">Identification of a Novel Class of BRD4 Inhibitors by Computational Screening and Binding Simulations. </span><span class="cited-content_cbyCitation_journal-name">ACS Omega</span><span> <strong>2017,</strong> <em>2 </em>
                                    (8)
                                     , 4760-4771. <a href="https://doi.org/10.1021/acsomega.7b00553" title="DOI URL">https://doi.org/10.1021/acsomega.7b00553</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsomega.7b00553&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsomega.7b00553%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Omega%26atitle%3DIdentification%252Bof%252Ba%252BNovel%252BClass%252Bof%252BBRD4%252BInhibitors%252Bby%252BComputational%252BScreening%252Band%252BBinding%252BSimulations%26aulast%3DAllen%26aufirst%3DBryce%2BK.%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D04052017%26date%3D27072017%26date%3D31082017%26volume%3D2%26issue%3D8%26spage%3D4760%26epage%3D4771" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Melisa E.  Gantner</span>, <span class="hlFld-ContribAuthor ">Roxana N.  Peroni</span>, <span class="hlFld-ContribAuthor ">Juan F.  Morales</span>, <span class="hlFld-ContribAuthor ">María L.  Villalba</span>, <span class="hlFld-ContribAuthor ">María E.  Ruiz</span>, and <span class="hlFld-ContribAuthor ">Alan  Talevi</span>  . </span><span class="cited-content_cbyCitation_article-title">Development and Validation of a Computational Model Ensemble for the Early Detection of BCRP/ABCG2 Substrates during the Drug Design Stage. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2017,</strong> <em>57 </em>
                                    (8)
                                     , 1868-1880. <a href="https://doi.org/10.1021/acs.jcim.7b00016" title="DOI URL">https://doi.org/10.1021/acs.jcim.7b00016</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.7b00016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.7b00016%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DDevelopment%252Band%252BValidation%252Bof%252Ba%252BComputational%252BModel%252BEnsemble%252Bfor%252Bthe%252BEarly%252BDetection%252Bof%252BBCRP%25252FABCG2%252BSubstrates%252Bduring%252Bthe%252BDrug%252BDesign%252BStage%26aulast%3DGantner%26aufirst%3DMelisa%2BE.%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D09012017%26date%3D08082017%26date%3D28082017%26date%3D14072017%26volume%3D57%26issue%3D8%26spage%3D1868%26epage%3D1880" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jing  Xing</span>, <span class="hlFld-ContribAuthor ">Wenchao  Lu</span>, <span class="hlFld-ContribAuthor ">Rongfeng  Liu</span>, <span class="hlFld-ContribAuthor ">Yulan  Wang</span>, <span class="hlFld-ContribAuthor ">Yiqian  Xie</span>, <span class="hlFld-ContribAuthor ">Hao  Zhang</span>, <span class="hlFld-ContribAuthor ">Zhe  Shi</span>, <span class="hlFld-ContribAuthor ">Hao  Jiang</span>, <span class="hlFld-ContribAuthor ">Yu-Chih  Liu</span>, <span class="hlFld-ContribAuthor ">Kaixian  Chen</span>, <span class="hlFld-ContribAuthor ">Hualiang  Jiang</span>, <span class="hlFld-ContribAuthor ">Cheng  Luo</span>, and <span class="hlFld-ContribAuthor ">Mingyue  Zheng</span>  . </span><span class="cited-content_cbyCitation_article-title">Machine-Learning-Assisted Approach for Discovering Novel Inhibitors Targeting Bromodomain-Containing Protein 4. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2017,</strong> <em>57 </em>
                                    (7)
                                     , 1677-1690. <a href="https://doi.org/10.1021/acs.jcim.7b00098" title="DOI URL">https://doi.org/10.1021/acs.jcim.7b00098</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.7b00098&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.7b00098%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DMachine-Learning-Assisted%252BApproach%252Bfor%252BDiscovering%252BNovel%252BInhibitors%252BTargeting%252BBromodomain-Containing%252BProtein%252B4%26aulast%3DXing%26aufirst%3DJing%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D18022017%26date%3D10072017%26date%3D24072017%26date%3D21062017%26volume%3D57%26issue%3D7%26spage%3D1677%26epage%3D1690" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Spencer S.  Ericksen</span>, <span class="hlFld-ContribAuthor ">Haozhen  Wu</span>, <span class="hlFld-ContribAuthor ">Huikun  Zhang</span>, <span class="hlFld-ContribAuthor ">Lauren A.  Michael</span>, <span class="hlFld-ContribAuthor ">Michael A.  Newton</span>, <span class="hlFld-ContribAuthor ">F. Michael  Hoffmann</span>, and <span class="hlFld-ContribAuthor ">Scott A.  Wildman</span>  . </span><span class="cited-content_cbyCitation_article-title">Machine Learning Consensus Scoring Improves Performance Across Targets in Structure-Based Virtual Screening. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2017,</strong> <em>57 </em>
                                    (7)
                                     , 1579-1590. <a href="https://doi.org/10.1021/acs.jcim.7b00153" title="DOI URL">https://doi.org/10.1021/acs.jcim.7b00153</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.7b00153&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.7b00153%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DMachine%252BLearning%252BConsensus%252BScoring%252BImproves%252BPerformance%252BAcross%252BTargets%252Bin%252BStructure-Based%252BVirtual%252BScreening%26aulast%3DEricksen%26aufirst%3DSpencer%2BS.%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D16032017%26date%3D12072017%26date%3D24072017%26date%3D27062017%26volume%3D57%26issue%3D7%26spage%3D1579%26epage%3D1590" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Andrew  Anighoro</span> and <span class="hlFld-ContribAuthor ">Jürgen  Bajorath</span>  . </span><span class="cited-content_cbyCitation_article-title">Compound Ranking Based on Fuzzy Three-Dimensional Similarity Improves the Performance of Docking into Homology Models of G-Protein-Coupled Receptors. </span><span class="cited-content_cbyCitation_journal-name">ACS Omega</span><span> <strong>2017,</strong> <em>2 </em>
                                    (6)
                                     , 2583-2592. <a href="https://doi.org/10.1021/acsomega.7b00330" title="DOI URL">https://doi.org/10.1021/acsomega.7b00330</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsomega.7b00330&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsomega.7b00330%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Omega%26atitle%3DCompound%252BRanking%252BBased%252Bon%252BFuzzy%252BThree-Dimensional%252BSimilarity%252BImproves%252Bthe%252BPerformance%252Bof%252BDocking%252Binto%252BHomology%252BModels%252Bof%252BG-Protein-Coupled%252BReceptors%26aulast%3DAnighoro%26aufirst%3DAndrew%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D20032017%26date%3D31052017%26date%3D30062017%26volume%3D2%26issue%3D6%26spage%3D2583%26epage%3D2592" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jie  Xia</span>, <span class="hlFld-ContribAuthor ">Jui-Hua  Hsieh</span>, <span class="hlFld-ContribAuthor ">Huabin  Hu</span>, <span class="hlFld-ContribAuthor ">Song  Wu</span>, and <span class="hlFld-ContribAuthor ">Xiang Simon  Wang</span>  . </span><span class="cited-content_cbyCitation_article-title">The Development of Target-Specific Pose Filter Ensembles To Boost Ligand Enrichment for Structure-Based Virtual Screening. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2017,</strong> <em>57 </em>
                                    (6)
                                     , 1414-1425. <a href="https://doi.org/10.1021/acs.jcim.6b00749" title="DOI URL">https://doi.org/10.1021/acs.jcim.6b00749</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.6b00749&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.6b00749%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DThe%252BDevelopment%252Bof%252BTarget-Specific%252BPose%252BFilter%252BEnsembles%252BTo%252BBoost%252BLigand%252BEnrichment%252Bfor%252BStructure-Based%252BVirtual%252BScreening%26aulast%3DXia%26aufirst%3DJie%26date%3D2017%26date%3D2017%26date%3D2016%26date%3D12122016%26date%3D01062017%26date%3D26062017%26date%3D16052017%26volume%3D57%26issue%3D6%26spage%3D1414%26epage%3D1425" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div><a href="#" data-source="/pb/widgets/cited-by?doi=10.1021%2Fjm300687e&amp;widgetId=9cf770f4-a9a2-46e0-bd29-3da15bcf4ae0&amp;pbContext=%3Barticle%3Aarticle%3A10.1021%2Fjm300687e%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2012.55.issue-14%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text&amp;citedByCount=819&amp;pagesCount=6&amp;pageNumber=2" title="Load more citations" class="button_primary float-right cited-by__see-more mb-5"><i aria-hidden="”true”" class="icon-angle-double-down"></i><span>Load more citations</span></a></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-14/jm300687e/production/images/medium/jm-2012-00687e_0008.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-14/jm300687e/production/images/large/jm-2012-00687e_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm300687e&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-14/jm300687e/production/images/medium/jm-2012-00687e_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-14/jm300687e/production/images/large/jm-2012-00687e_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. DUD-E target classification. Number of the 102 targets that belong to eight broad protein categories.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-14/jm300687e/production/images/large/jm-2012-00687e_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm300687e&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-14/jm300687e/production/images/medium/jm-2012-00687e_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-14/jm300687e/production/images/large/jm-2012-00687e_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Ligand clustering. (A) The seventh largest Murcko cluster of kinesin-like protein 1 (KIF11), showing both the scaffold (left) and all seven member ligands. (B) Number of ligands in each of the 70 KIF11 Bemis–Murcko atomic frameworks. We removed lower affinity compounds over-represented clusters (above the line), while retaining 100 ligands. (C) Number of adenosine A<sub>2A</sub> receptor (AA2AR) Murcko clusters is plotted against affinity threshold. Fewer than 600 clusters are present using a 30 nM affinity threshold.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-14/jm300687e/production/images/large/jm-2012-00687e_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm300687e&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-14/jm300687e/production/images/medium/jm-2012-00687e_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-14/jm300687e/production/images/large/jm-2012-00687e_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Decoy generation. (A) Three key “warhead” groups from factor Xa (FA10), glycinamide ribonucleotide transformylase (PUR2), and thymidine kinase (KITH). (B) Fraction of warheads remaining is plotted against the dissimilarity method. The dissimilarity methods consist of a fingerprint (Daylight or ECFP4) and either a hard cutoff or a fraction of the most dissimilar decoys to be retained. (C) Property distributions of estrogen receptor α (ESR1) for both the 383 ligands (blue) and the 20685 property-matched decoys (red).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-14/jm300687e/production/images/large/jm-2012-00687e_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm300687e&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-14/jm300687e/production/images/medium/jm-2012-00687e_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-14/jm300687e/production/images/large/jm-2012-00687e_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Retrospective enrichment comparing ligand desolvation and electrostatics methods. Docking results over DUD-E as measured by LogAUC. “None” has no ligand desolvation term, “SEV” uses solvent-excluded volume ligand desolvation, “Thin” employs a thin low-dielectric layer in the electrostatic calculations.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-14/jm300687e/production/images/large/jm-2012-00687e_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm300687e&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-14/jm300687e/production/images/medium/jm-2012-00687e_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-14/jm300687e/production/images/large/jm-2012-00687e_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Representative ROC plots. ROC plots using no desolvation (None), solvent-excluded volume ligand desolvation (SEV), the thin low-dielectric layer (Thin), or a drug-like background that consists of all ChEMBL12 ligands with affinities better than 10 μM (Drug-like). The black dotted line represents the results expected from docking ligands randomly. LogAUC percentages are reported in the legend text.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-14/jm300687e/production/images/large/jm-2012-00687e_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm300687e&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-14/jm300687e/production/images/medium/jm-2012-00687e_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-14/jm300687e/production/images/large/jm-2012-00687e_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Representative docking poses. The crystallographic ligand was rebuilt and docked from scratch. (A–F) The crystal pose (magenta) is compared to the resulting docked pose (green). In (C), more ligand conformations are generated and the redocked pose is also shown (tan). Key hydrogen bonds are shown by black dotted lines, and the partially transparent protein surface is colored by atom type.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-14/jm300687e/production/images/large/jm-2012-00687e_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm300687e&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i27">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_83542" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_83542" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 53 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Kitchen, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Decornez, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Furr, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bajorath, J.</span><span> </span><span class="NLM_article-title">Docking and scoring in virtual screening for drug discovery: methods and applications</span> <span class="citation_source-journal">Nature Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">935</span><span class="NLM_x">–</span> <span class="NLM_lpage">949</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2004&pages=935-949&author=D.+B.+Kitchenauthor=H.+Decornezauthor=J.+R.+Furrauthor=J.+Bajorath&title=Docking+and+scoring+in+virtual+screening+for+drug+discovery%3A+methods+and+applications"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKitchen%26aufirst%3DD.%2BB.%26aulast%3DDecornez%26aufirst%3DH.%26aulast%3DFurr%26aufirst%3DJ.%2BR.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DDocking%2520and%2520scoring%2520in%2520virtual%2520screening%2520for%2520drug%2520discovery%253A%2520methods%2520and%2520applications%26jtitle%3DNature%2520Rev.%2520Drug%2520Discovery%26date%3D2004%26volume%3D3%26spage%3D935%26epage%3D949" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Kolb, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosenbaum, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fung, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kobilka, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Structure-based discovery of beta(2)-adrenergic receptor ligands</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U.S.A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">106</span><span class="NLM_x">, </span> <span class="NLM_fpage">6843</span><span class="NLM_x">–</span> <span class="NLM_lpage">6848</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2009&pages=6843-6848&author=P.+Kolbauthor=D.+M.+Rosenbaumauthor=J.+J.+Irwinauthor=J.+J.+Fungauthor=B.+K.+Kobilkaauthor=B.+K.+Shoichet&title=Structure-based+discovery+of+beta%282%29-adrenergic+receptor+ligands"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKolb%26aufirst%3DP.%26aulast%3DRosenbaum%26aufirst%3DD.%2BM.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DFung%26aufirst%3DJ.%2BJ.%26aulast%3DKobilka%26aufirst%3DB.%2BK.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DStructure-based%2520discovery%2520of%2520beta%25282%2529-adrenergic%2520receptor%2520ligands%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2009%26volume%3D106%26spage%3D6843%26epage%3D6848" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Mysinger, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weiss, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ziarek, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gravel, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Doak, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karpiak, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heveker, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Volkman, B. F.</span><span> </span><span class="NLM_article-title">Structure-based ligand discovery for the protein–protein interface of chemokine receptor CXCR4</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U.S.A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">109</span><span class="NLM_x">, </span> <span class="NLM_fpage">5517</span><span class="NLM_x">–</span> <span class="NLM_lpage">5522</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2012&pages=5517-5522&author=M.+M.+Mysingerauthor=D.+R.+Weissauthor=J.+J.+Ziarekauthor=S.+Gravelauthor=A.+K.+Doakauthor=J.+Karpiakauthor=N.+Hevekerauthor=B.+K.+Shoichetauthor=B.+F.+Volkman&title=Structure-based+ligand+discovery+for+the+protein%E2%80%93protein+interface+of+chemokine+receptor+CXCR4"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMysinger%26aufirst%3DM.%2BM.%26aulast%3DWeiss%26aufirst%3DD.%2BR.%26aulast%3DZiarek%26aufirst%3DJ.%2BJ.%26aulast%3DGravel%26aufirst%3DS.%26aulast%3DDoak%26aufirst%3DA.%2BK.%26aulast%3DKarpiak%26aufirst%3DJ.%26aulast%3DHeveker%26aufirst%3DN.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DVolkman%26aufirst%3DB.%2BF.%26atitle%3DStructure-based%2520ligand%2520discovery%2520for%2520the%2520protein%25E2%2580%2593protein%2520interface%2520of%2520chemokine%2520receptor%2520CXCR4%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2012%26volume%3D109%26spage%3D5517%26epage%3D5522" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Gruneberg, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stubbs, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klebe, G.</span><span> </span><span class="NLM_article-title">Successful virtual screening for novel inhibitors of human carbonic anhydrase: strategy and experimental confirmation</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">3588</span><span class="NLM_x">–</span> <span class="NLM_lpage">3602</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm011112j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2002&pages=3588-3602&author=S.+Grunebergauthor=M.+T.+Stubbsauthor=G.+Klebe&title=Successful+virtual+screening+for+novel+inhibitors+of+human+carbonic+anhydrase%3A+strategy+and+experimental+confirmation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1021%2Fjm011112j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm011112j%26sid%3Dliteratum%253Aachs%26aulast%3DGruneberg%26aufirst%3DS.%26aulast%3DStubbs%26aufirst%3DM.%2BT.%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DSuccessful%2520virtual%2520screening%2520for%2520novel%2520inhibitors%2520of%2520human%2520carbonic%2520anhydrase%253A%2520strategy%2520and%2520experimental%2520confirmation%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2002%26volume%3D45%26spage%3D3588%26epage%3D3602" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Jain, A. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nicholls, A.</span><span> </span><span class="NLM_article-title">Recommendations for evaluation of computational methods</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">133</span><span class="NLM_x">–</span> <span class="NLM_lpage">139</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Fjm300687e&amp;key=10.1007%2Fs10822-008-9196-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Fjm300687e&amp;key=18338228" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Fjm300687e&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjsFOnsb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2008&pages=133-139&author=A.+N.+Jainauthor=A.+Nicholls&title=Recommendations+for+evaluation+of+computational+methods"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Recommendations for evaluation of computational methods</span></div><div class="casAuthors">Jain, Ajay N.; Nicholls, Anthony</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">3-4</span>),
    <span class="NLM_cas:pages">133-139</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  The field of computational chem., particularly as applied to drug design, has become increasingly important in terms of the practical application of predictive modeling to pharmaceutical research and development.  Tools for exploiting protein structures or sets of ligands known to bind particular targets can be used for binding-mode prediction, virtual screening, and prediction of activity.  A serious weakness within the field is a lack of stds. with respect to quant. evaluation of methods, data set prepn., and data set sharing.  Our goal should be to report new methods or comparative evaluations of methods in a manner that supports decision making for practical applications.  Here we propose a modest beginning, with recommendations for requirements on statistical reporting, requirements for data sharing, and best practices for benchmark prepn. and usage.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZyHVFOx399LVg90H21EOLACvtfcHk0lgYkRHI-BpAeA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjsFOnsb0%253D&md5=9576e150079dcbf6e4bfea5070082016</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1007%2Fs10822-008-9196-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-008-9196-5%26sid%3Dliteratum%253Aachs%26aulast%3DJain%26aufirst%3DA.%2BN.%26aulast%3DNicholls%26aufirst%3DA.%26atitle%3DRecommendations%2520for%2520evaluation%2520of%2520computational%2520methods%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2008%26volume%3D22%26spage%3D133%26epage%3D139" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Babaoglu, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simeonov, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nelson, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thomas, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cancian, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Costi, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maltby, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jadhav, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Inglese, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Austin, C. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Comprehensive mechanistic analysis of hits from high-throughput and docking screens against beta-lactamase</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">2502</span><span class="NLM_x">–</span> <span class="NLM_lpage">2511</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm701500e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Fjm300687e&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjtFSksb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=2502-2511&author=K.+Babaogluauthor=A.+Simeonovauthor=J.+J.+Irwinauthor=M.+E.+Nelsonauthor=B.+Fengauthor=C.+J.+Thomasauthor=L.+Cancianauthor=M.+P.+Costiauthor=D.+A.+Maltbyauthor=A.+Jadhavauthor=J.+Ingleseauthor=C.+P.+Austinauthor=B.+K.+Shoichet&title=Comprehensive+mechanistic+analysis+of+hits+from+high-throughput+and+docking+screens+against+beta-lactamase"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Comprehensive Mechanistic Analysis of Hits from High-Throughput and Docking Screens against β-Lactamase</span></div><div class="casAuthors">Babaoglu, Kerim; Simeonov, Anton; Irwin, John J.; Nelson, Michael E.; Feng, Brian; Thomas, Craig J.; Cancian, Laura; Costi, M. Paola; Maltby, David A.; Jadhav, Ajit; Inglese, James; Austin, Christopher P.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2502-2511</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">High-throughput screening (HTS) is widely used in drug discovery.  Esp. for screens of unbiased libraries, false positives can dominate "hit lists"; their origins are much debated.  Here we det. the mechanism of every active hit from a screen of 70,563 unbiased mols. against β-lactamase using quant. HTS (qHTS).  Of the 1274 initial inhibitors, 95% were detergent-sensitive and were classified as aggregators.  Among the 70 remaining were 25 potent, covalent-acting β-lactams.  Mass spectra, counter-screens, and crystallog. identified 12 as promiscuous covalent inhibitors.  The remaining 33 were either aggregators or irreproducible.  No specific reversible inhibitors were found.  We turned to mol. docking to prioritize mols. from the same library for testing at higher concns.  Of 16 tested, 2 were modest inhibitors.  Subsequent X-ray structures corresponded to the docking prediction.  Analog synthesis improved affinity to 8 μM.  These results suggest that it may be the phys. behavior of org. mols., not their reactivity, that accounts for most screening artifacts.  Structure-based methods may prioritize weak-but-novel chemotypes in unbiased library screens.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpNtnaCegS8aLVg90H21EOLACvtfcHk0ljC9Za2MxQPTw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjtFSksb0%253D&md5=7736429aeafc6b17a5d0182e24df0dd4</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1021%2Fjm701500e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm701500e%26sid%3Dliteratum%253Aachs%26aulast%3DBabaoglu%26aufirst%3DK.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DNelson%26aufirst%3DM.%2BE.%26aulast%3DFeng%26aufirst%3DB.%26aulast%3DThomas%26aufirst%3DC.%2BJ.%26aulast%3DCancian%26aufirst%3DL.%26aulast%3DCosti%26aufirst%3DM.%2BP.%26aulast%3DMaltby%26aufirst%3DD.%2BA.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DInglese%26aufirst%3DJ.%26aulast%3DAustin%26aufirst%3DC.%2BP.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DComprehensive%2520mechanistic%2520analysis%2520of%2520hits%2520from%2520high-throughput%2520and%2520docking%2520screens%2520against%2520beta-lactamase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D2502%26epage%3D2511" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Ferreira, R. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simeonov, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jadhav, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eidam, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mott, B. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keiser, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McKerrow, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maloney, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Complementarity between a docking and a high-throughput screen in discovering new cruzain inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">4891</span><span class="NLM_x">–</span> <span class="NLM_lpage">4905</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm100488w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Fjm300687e&amp;key=1%3ACAS%3A528%3ADC%252BC3cXntlWrurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=4891-4905&author=R.+S.+Ferreiraauthor=A.+Simeonovauthor=A.+Jadhavauthor=O.+Eidamauthor=B.+T.+Mottauthor=M.+J.+Keiserauthor=J.+H.+McKerrowauthor=D.+J.+Maloneyauthor=J.+J.+Irwinauthor=B.+K.+Shoichet&title=Complementarity+between+a+docking+and+a+high-throughput+screen+in+discovering+new+cruzain+inhibitors"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Complementarity Between a Docking and a High-Throughput Screen in Discovering New Cruzain Inhibitors</span></div><div class="casAuthors">Ferreira, Rafaela S.; Simeonov, Anton; Jadhav, Ajit; Eidam, Oliv; Mott, Bryan T.; Keiser, Michael J.; McKerrow, James H.; Maloney, David J.; Irwin, John J.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">4891-4905</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Virtual and high-throughput screens (HTS) should have complementary strengths and weaknesses, but studies that prospectively and comprehensively compare them are rare.  We undertook a parallel docking and HTS screen of 197861 compds. against cruzain, a thiol protease target for Chagas disease, looking for reversible, competitive inhibitors.  On workup, 99% of the hits were eliminated as false positives, yielding 146 well-behaved, competitive ligands.  These fell into five chemotypes: two were prioritized by scoring among the top 0.1% of the docking-ranked library, two were prioritized by behavior in the HTS and by clustering, and one chemotype was prioritized by both approaches.  Detn. of an inhibitor/cruzain crystal structure and comparison of the high-scoring docking hits to expt. illuminated the origins of docking false-negatives and false-positives.  Prioritizing mols. that are both predicted by docking and are HTS-active yields well-behaved mols., relatively unobscured by the false-positives to which both techniques are individually prone.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmmoDO-1mm8bVg90H21EOLACvtfcHk0ljC9Za2MxQPTw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXntlWrurk%253D&md5=0661663ff0802bf981a1997eeca9fa05</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1021%2Fjm100488w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm100488w%26sid%3Dliteratum%253Aachs%26aulast%3DFerreira%26aufirst%3DR.%2BS.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DEidam%26aufirst%3DO.%26aulast%3DMott%26aufirst%3DB.%2BT.%26aulast%3DKeiser%26aufirst%3DM.%2BJ.%26aulast%3DMcKerrow%26aufirst%3DJ.%2BH.%26aulast%3DMaloney%26aufirst%3DD.%2BJ.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DComplementarity%2520between%2520a%2520docking%2520and%2520a%2520high-throughput%2520screen%2520in%2520discovering%2520new%2520cruzain%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D4891%26epage%3D4905" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Gohlke, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klebe, G.</span><span> </span><span class="NLM_article-title">Approaches to the description and prediction of the binding affinity of small-molecule ligands to macromolecular receptors</span> <span class="citation_source-journal">Angew. Chem., Int. Ed. Engl.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">41</span><span class="NLM_x">, </span> <span class="NLM_fpage">2644</span><span class="NLM_x">–</span> <span class="NLM_lpage">2676</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2002&pages=2644-2676&author=H.+Gohlkeauthor=G.+Klebe&title=Approaches+to+the+description+and+prediction+of+the+binding+affinity+of+small-molecule+ligands+to+macromolecular+receptors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGohlke%26aufirst%3DH.%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DApproaches%2520to%2520the%2520description%2520and%2520prediction%2520of%2520the%2520binding%2520affinity%2520of%2520small-molecule%2520ligands%2520to%2520macromolecular%2520receptors%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%2520Engl.%26date%3D2002%26volume%3D41%26spage%3D2644%26epage%3D2676" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Enyedy, I. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Egan, W. J.</span><span> </span><span class="NLM_article-title">Can we use docking and scoring for hit-to-lead optimization?</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">161</span><span class="NLM_x">–</span> <span class="NLM_lpage">168</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2008&pages=161-168&author=I.+J.+Enyedyauthor=W.+J.+Egan&title=Can+we+use+docking+and+scoring+for+hit-to-lead+optimization%3F"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DEnyedy%26aufirst%3DI.%2BJ.%26aulast%3DEgan%26aufirst%3DW.%2BJ.%26atitle%3DCan%2520we%2520use%2520docking%2520and%2520scoring%2520for%2520hit-to-lead%2520optimization%253F%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2008%26volume%3D22%26spage%3D161%26epage%3D168" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Stahl, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rarey, M.</span><span> </span><span class="NLM_article-title">Detailed analysis of scoring functions for virtual screening</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">44</span><span class="NLM_x">, </span> <span class="NLM_fpage">1035</span><span class="NLM_x">–</span> <span class="NLM_lpage">1042</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm0003992" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2001&pages=1035-1042&author=M.+Stahlauthor=M.+Rarey&title=Detailed+analysis+of+scoring+functions+for+virtual+screening"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Fjm0003992&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0003992%26sid%3Dliteratum%253Aachs%26aulast%3DStahl%26aufirst%3DM.%26aulast%3DRarey%26aufirst%3DM.%26atitle%3DDetailed%2520analysis%2520of%2520scoring%2520functions%2520for%2520virtual%2520screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2001%26volume%3D44%26spage%3D1035%26epage%3D1042" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Bissantz, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Folkers, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rognan, D.</span><span> </span><span class="NLM_article-title">Protein-based virtual screening of chemical databases. 1. Evaluation of different docking/scoring combinations</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">43</span><span class="NLM_x">, </span> <span class="NLM_fpage">4759</span><span class="NLM_x">–</span> <span class="NLM_lpage">4767</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm001044l" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2000&pages=4759-4767&author=C.+Bissantzauthor=G.+Folkersauthor=D.+Rognan&title=Protein-based+virtual+screening+of+chemical+databases.+1.+Evaluation+of+different+docking%2Fscoring+combinations"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Fjm001044l&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm001044l%26sid%3Dliteratum%253Aachs%26aulast%3DBissantz%26aufirst%3DC.%26aulast%3DFolkers%26aufirst%3DG.%26aulast%3DRognan%26aufirst%3DD.%26atitle%3DProtein-based%2520virtual%2520screening%2520of%2520chemical%2520databases.%25201.%2520Evaluation%2520of%2520different%2520docking%252Fscoring%2520combinations%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2000%26volume%3D43%26spage%3D4759%26epage%3D4767" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Pham, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jain, A. N.</span><span> </span><span class="NLM_article-title">Parameter estimation for scoring protein–ligand interactions using negative training data</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">5856</span><span class="NLM_x">–</span> <span class="NLM_lpage">5868</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm050040j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=5856-5868&author=T.+A.+Phamauthor=A.+N.+Jain&title=Parameter+estimation+for+scoring+protein%E2%80%93ligand+interactions+using+negative+training+data"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Fjm050040j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm050040j%26sid%3Dliteratum%253Aachs%26aulast%3DPham%26aufirst%3DT.%2BA.%26aulast%3DJain%26aufirst%3DA.%2BN.%26atitle%3DParameter%2520estimation%2520for%2520scoring%2520protein%25E2%2580%2593ligand%2520interactions%2520using%2520negative%2520training%2520data%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D5856%26epage%3D5868" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Kellenberger, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rodrigo, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muller, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rognan, D.</span><span> </span><span class="NLM_article-title">Comparative evaluation of eight docking tools for docking and virtual screening accuracy</span> <span class="citation_source-journal">Proteins</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">225</span><span class="NLM_x">–</span> <span class="NLM_lpage">242</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2004&pages=225-242&author=E.+Kellenbergerauthor=J.+Rodrigoauthor=P.+Mullerauthor=D.+Rognan&title=Comparative+evaluation+of+eight+docking+tools+for+docking+and+virtual+screening+accuracy"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKellenberger%26aufirst%3DE.%26aulast%3DRodrigo%26aufirst%3DJ.%26aulast%3DMuller%26aufirst%3DP.%26aulast%3DRognan%26aufirst%3DD.%26atitle%3DComparative%2520evaluation%2520of%2520eight%2520docking%2520tools%2520for%2520docking%2520and%2520virtual%2520screening%2520accuracy%26jtitle%3DProteins%26date%3D2004%26volume%3D57%26spage%3D225%26epage%3D242" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Ferrara, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gohlke, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Price, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klebe, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brooks, C. L.,  III.</span><span> </span><span class="NLM_article-title">Assessing scoring functions for protein–ligand interactions</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">3032</span><span class="NLM_x">–</span> <span class="NLM_lpage">3047</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm030489h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=3032-3047&author=P.+Ferraraauthor=H.+Gohlkeauthor=D.+J.+Priceauthor=G.+Klebeauthor=C.+L.+Brooks&title=Assessing+scoring+functions+for+protein%E2%80%93ligand+interactions"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Fjm030489h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm030489h%26sid%3Dliteratum%253Aachs%26aulast%3DFerrara%26aufirst%3DP.%26aulast%3DGohlke%26aufirst%3DH.%26aulast%3DPrice%26aufirst%3DD.%2BJ.%26aulast%3DKlebe%26aufirst%3DG.%26aulast%3DBrooks%26aufirst%3DC.%2BL.%26atitle%3DAssessing%2520scoring%2520functions%2520for%2520protein%25E2%2580%2593ligand%2520interactions%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D3032%26epage%3D3047" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Huang, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span> </span><span class="NLM_article-title">Benchmarking sets for molecular docking</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">6789</span><span class="NLM_x">–</span> <span class="NLM_lpage">6801</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm0608356" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Fjm300687e&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtFehurzI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=6789-6801&author=N.+Huangauthor=B.+K.+Shoichetauthor=J.+J.+Irwin&title=Benchmarking+sets+for+molecular+docking"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Benchmarking Sets for Molecular Docking</span></div><div class="casAuthors">Huang, Niu; Shoichet, Brian K.; Irwin, John J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">6789-6801</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Ligand enrichment among top-ranking hits is a key metric of mol. docking.  To avoid bias, decoys should resemble ligands phys., so that enrichment is not simply a sepn. of gross features, yet be chem. distinct from them, so that they are unlikely to be binders.  We have assembled a directory of useful decoys (DUD), with 2950 ligands for 40 different targets.  Every ligand has 36 decoy mols. that are phys. similar but topol. distinct, leading to a database of 98 266 compds.  For most targets, enrichment was at least half a log better with uncorrected databases such as the MDDR than with DUD, evidence of bias in the former.  These calcns. also allowed 40×40 cross-docking, where the enrichments of each ligand set could be compared for all 40 targets, enabling a specificity metric for the docking screens.  DUD is freely available online as a benchmarking set for docking at http://blaster.docking.org/dud/.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-8FVhVSsBm7Vg90H21EOLACvtfcHk0lh0swh07R6sUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtFehurzI&md5=43af46968c2a6d66334922bf0caedbe1</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Fjm0608356&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0608356%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DN.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26atitle%3DBenchmarking%2520sets%2520for%2520molecular%2520docking%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D6789%26epage%3D6801" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Christofferson, A. J.; Huang, N.</span><span> </span><span class="NLM_article-title">How to benchmark methods for structure-based virtual screening of large compound libraries</span>. In  <span class="citation_source-book">Computational Drug Discovery and Design (Methods in Molecular Biology)</span>; <span class="NLM_edition">2011/12/21</span> ed.; <span class="NLM_contrib-group">Baron, R.</span>, Ed.; <span class="NLM_publisher-name">Springer Protocols</span>: <span class="NLM_publisher-loc">New York</span>,<span class="NLM_x"> </span><span class="NLM_year">2012</span><span class="NLM_x">; </span>Vol.  <span class="NLM_volume">819</span>, Chapter 13, pp  <span class="NLM_fpage">187</span><span class="NLM_x">–</span> <span class="NLM_lpage">195</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Fjm300687e&amp;key=10.1007%2F978-1-61779-465-0_13" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&pages=187-195&author=A.+J.+Christofferson&author=N.+Huangauthor=R.+Baron&title=Computational+Drug+Discovery+and+Design+%28Methods+in+Molecular+Biology%29"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1007%2F978-1-61779-465-0_13&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F978-1-61779-465-0_13%26sid%3Dliteratum%253Aachs%26aulast%3DChristofferson%26aufirst%3DA.%2BJ.%26atitle%3DHow%2520to%2520benchmark%2520methods%2520for%2520structure-based%2520virtual%2520screening%2520of%2520large%2520compound%2520libraries%26btitle%3DComputational%2520Drug%2520Discovery%2520and%2520Design%2520%2528Methods%2520in%2520Molecular%2520Biology%2529%26aulast%3DBaron%26aufirst%3DR.%26pub%3DSpringer%2520Protocols%26date%3D2012%26volume%3D819%26spage%3D187%26epage%3D195" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Verdonk, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berdini, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hartshorn, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mooij, W. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, R. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Watson, P.</span><span> </span><span class="NLM_article-title">Virtual screening using protein–ligand docking: avoiding artificial enrichment</span> <span class="citation_source-journal">J. Chem. Inf. Comput. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">44</span><span class="NLM_x">, </span> <span class="NLM_fpage">793</span><span class="NLM_x">–</span> <span class="NLM_lpage">806</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci034289q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Fjm300687e&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVGhs70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2004&pages=793-806&author=M.+L.+Verdonkauthor=V.+Berdiniauthor=M.+J.+Hartshornauthor=W.+T.+Mooijauthor=C.+W.+Murrayauthor=R.+D.+Taylorauthor=P.+Watson&title=Virtual+screening+using+protein%E2%80%93ligand+docking%3A+avoiding+artificial+enrichment"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Virtual screening using protein-ligand docking: Avoiding artificial enrichment</span></div><div class="casAuthors">Verdonk, Marcel L.; Berdini, Valerio; Hartshorn, Michael J.; Mooij, Wijnand T. M.; Murray, Christopher W.; Taylor, Richard D.; Watson, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Computer Sciences</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">793-806</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">0095-2338</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This study addresses a no. of topical issues around the use of protein-ligand docking in virtual screening.  We show that, for the validation of such methods, it is key to use focused libraries (contg. compds. with one-dimensional properties, similar to the actives), rather than "random" or "drug-like" libraries to test the actives against.  We also show that, to obtain good enrichments, the docking program needs to produce reliable binding modes.  We demonstrate how pharmacophores can be used to guide the dockings and improve enrichments, and we compare the performance of three consensus-ranking protocols against ranking based on individual scoring functions.  Finally, we show that protein-ligand docking can be an effective aid in the screening for weak, fragment-like binders, which has rapidly become a popular strategy for hit identification.  All results presented are based on carefully constructed virtual screening expts. against four targets, using the protein-ligand docking program GOLD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqRRkYiB4qsvbVg90H21EOLACvtfcHk0lh0swh07R6sUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVGhs70%253D&md5=b0cd9e9a04ddebe401dee9744e2184cb</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fci034289q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci034289q%26sid%3Dliteratum%253Aachs%26aulast%3DVerdonk%26aufirst%3DM.%2BL.%26aulast%3DBerdini%26aufirst%3DV.%26aulast%3DHartshorn%26aufirst%3DM.%2BJ.%26aulast%3DMooij%26aufirst%3DW.%2BT.%26aulast%3DMurray%26aufirst%3DC.%2BW.%26aulast%3DTaylor%26aufirst%3DR.%2BD.%26aulast%3DWatson%26aufirst%3DP.%26atitle%3DVirtual%2520screening%2520using%2520protein%25E2%2580%2593ligand%2520docking%253A%2520avoiding%2520artificial%2520enrichment%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D2004%26volume%3D44%26spage%3D793%26epage%3D806" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Kuntz, I. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sharp, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kollman, P. A.</span><span> </span><span class="NLM_article-title">The maximal affinity of ligands</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U.S.A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">96</span><span class="NLM_x">, </span> <span class="NLM_fpage">9997</span><span class="NLM_x">–</span> <span class="NLM_lpage">10002</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Fjm300687e&amp;key=10.1073%2Fpnas.96.18.9997" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Fjm300687e&amp;key=10468550" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Fjm300687e&amp;key=1%3ACAS%3A528%3ADyaK1MXlvFehu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=96&publication_year=1999&pages=9997-10002&author=I.+D.+Kuntzauthor=K.+Chenauthor=K.+A.+Sharpauthor=P.+A.+Kollman&title=The+maximal+affinity+of+ligands"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">The maximal affinity of ligands</span></div><div class="casAuthors">Kuntz, I. D.; Chen, K.; Sharp, K. A.; Kollman, P. A.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">96</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">9997-10002</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">We explore the question of what are the best ligands for macromol. targets.  A survey of exptl. data on a large no. of the strongest-binding ligands indicates that the free energy of binding increases with the no. of nonhydrogen atoms with an initial slope of ≈-1.5 kcal/mol (1 cal = 4.18 J) per atom.  For ligands that contain more than 15 nonhydrogen atoms, the free energy of binding increases very little with relative mol. mass.  This nonlinearity is largely ascribed to nonthermodynamic factors.  An anal. of the dominant interactions suggests that van der Waals interactions and hydrophobic effects provide a reasonable basis for understanding binding affinities across the entire set of ligands.  Interesting outliers that bind unusually strongly on a per atom basis include metal ions, covalently attached ligands, and a few well known complexes such as biotin-avidin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0L6KZhR0TxbVg90H21EOLACvtfcHk0lh0swh07R6sUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXlvFehu7s%253D&md5=46fe894b016f3041831a2f0b71f812b1</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1073%2Fpnas.96.18.9997&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.96.18.9997%26sid%3Dliteratum%253Aachs%26aulast%3DKuntz%26aufirst%3DI.%2BD.%26aulast%3DChen%26aufirst%3DK.%26aulast%3DSharp%26aufirst%3DK.%2BA.%26aulast%3DKollman%26aufirst%3DP.%2BA.%26atitle%3DThe%2520maximal%2520affinity%2520of%2520ligands%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D1999%26volume%3D96%26spage%3D9997%26epage%3D10002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Fan, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Webb, B. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klebe, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sali, A.</span><span> </span><span class="NLM_article-title">Molecular Docking Screens Using Comparative Models of Proteins</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">2512</span><span class="NLM_x">–</span> <span class="NLM_lpage">2527</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci9003706" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Fjm300687e&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1yntL7M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2009&pages=2512-2527&author=H.+Fanauthor=J.+J.+Irwinauthor=B.+M.+Webbauthor=G.+Klebeauthor=B.+K.+Shoichetauthor=A.+Sali&title=Molecular+Docking+Screens+Using+Comparative+Models+of+Proteins"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Docking Screens Using Comparative Models of Proteins</span></div><div class="casAuthors">Fan, Hao; Irwin, John J.; Webb, Benjamin M.; Klebe, Gerhard; Shoichet, Brian K.; Sali, Andrej</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2512-2527</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Two orders of magnitude more protein sequences can be modeled by comparative modeling than have been detd. by X-ray crystallog. and NMR spectroscopy.  Investigators have nevertheless been cautious about using comparative models for ligand discovery because of concerns about model errors.  We suggest how to exploit comparative models for mol. screens, based on docking against a wide range of crystallog. structures and comparative models with known ligands.  To account for the variation in the ligand-binding pocket as it binds different ligands, we calc. "consensus" enrichment by ranking each library compd. by its best docking score against all available comparative models and/or modeling templates.  For the majority of the targets, the consensus enrichment for multiple models was better than or comparable to that of the holo and apo X-ray structures.  Even for single models, the models are significantly more enriching than the template structure if the template is paralogous and shares more than 25% sequence identity with the target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrrgXhylX-BJrVg90H21EOLACvtfcHk0lj3SaIwVbeCKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht1yntL7M&md5=11bd5284c6b43592906e45b6aea3019d</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Fci9003706&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci9003706%26sid%3Dliteratum%253Aachs%26aulast%3DFan%26aufirst%3DH.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DWebb%26aufirst%3DB.%2BM.%26aulast%3DKlebe%26aufirst%3DG.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DSali%26aufirst%3DA.%26atitle%3DMolecular%2520Docking%2520Screens%2520Using%2520Comparative%2520Models%2520of%2520Proteins%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2009%26volume%3D49%26spage%3D2512%26epage%3D2527" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Repasky, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murphy, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banks, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greenwood, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tubert-Brohman, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bhat, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friesner, R. A.</span><span> </span><span class="NLM_article-title">Docking performance of the glide program as evaluated on the Astex and DUD datasets: a complete set of glide SP results and selected results for a new scoring function integrating WaterMap and glide</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span><span class="NLM_elocation-id"></span><span class="refDoi"> DOI: 10.1007/s10822-012-9575-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Fjm300687e&amp;key=10.1007%2Fs10822-012-9575-9" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&author=M.+P.+Repaskyauthor=R.+B.+Murphyauthor=J.+L.+Banksauthor=J.+R.+Greenwoodauthor=I.+Tubert-Brohmanauthor=S.+Bhatauthor=R.+A.+Friesner&title=Docking+performance+of+the+glide+program+as+evaluated+on+the+Astex+and+DUD+datasets%3A+a+complete+set+of+glide+SP+results+and+selected+results+for+a+new+scoring+function+integrating+WaterMap+and+glide&doi=10.1007%2Fs10822-012-9575-9"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1007%2Fs10822-012-9575-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-012-9575-9%26sid%3Dliteratum%253Aachs%26aulast%3DRepasky%26aufirst%3DM.%2BP.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DBanks%26aufirst%3DJ.%2BL.%26aulast%3DGreenwood%26aufirst%3DJ.%2BR.%26aulast%3DTubert-Brohman%26aufirst%3DI.%26aulast%3DBhat%26aufirst%3DS.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26atitle%3DDocking%2520performance%2520of%2520the%2520glide%2520program%2520as%2520evaluated%2520on%2520the%2520Astex%2520and%2520DUD%2520datasets%253A%2520a%2520complete%2520set%2520of%2520glide%2520SP%2520results%2520and%2520selected%2520results%2520for%2520a%2520new%2520scoring%2520function%2520integrating%2520WaterMap%2520and%2520glide%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2012%26doi%3D10.1007%2Fs10822-012-9575-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Brozell, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mukherjee, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balius, T. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roe, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Case, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rizzo, R. C.</span><span> </span><span class="NLM_article-title">Evaluation of DOCK 6 as a pose generation and database enrichment tool</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span><span class="NLM_elocation-id"></span><span class="refDoi"> DOI: 10.1007/s10822-012-9565-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Fjm300687e&amp;key=10.1007%2Fs10822-012-9565-y" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&author=S.+R.+Brozellauthor=S.+Mukherjeeauthor=T.+E.+Baliusauthor=D.+R.+Roeauthor=D.+A.+Caseauthor=R.+C.+Rizzo&title=Evaluation+of+DOCK+6+as+a+pose+generation+and+database+enrichment+tool&doi=10.1007%2Fs10822-012-9565-y"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1007%2Fs10822-012-9565-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-012-9565-y%26sid%3Dliteratum%253Aachs%26aulast%3DBrozell%26aufirst%3DS.%2BR.%26aulast%3DMukherjee%26aufirst%3DS.%26aulast%3DBalius%26aufirst%3DT.%2BE.%26aulast%3DRoe%26aufirst%3DD.%2BR.%26aulast%3DCase%26aufirst%3DD.%2BA.%26aulast%3DRizzo%26aufirst%3DR.%2BC.%26atitle%3DEvaluation%2520of%2520DOCK%25206%2520as%2520a%2520pose%2520generation%2520and%2520database%2520enrichment%2520tool%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2012%26doi%3D10.1007%2Fs10822-012-9565-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Neves, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Totrov, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abagyan, R.</span><span> </span><span class="NLM_article-title">Docking and scoring with ICM: the benchmarking results and strategies for improvement</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span><span class="NLM_elocation-id"></span><span class="refDoi"> DOI: 10.1007/s10822-012-9547-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Fjm300687e&amp;key=10.1007%2Fs10822-012-9547-0" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&author=M.+A.+Nevesauthor=M.+Totrovauthor=R.+Abagyan&title=Docking+and+scoring+with+ICM%3A+the+benchmarking+results+and+strategies+for+improvement&doi=10.1007%2Fs10822-012-9547-0"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1007%2Fs10822-012-9547-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-012-9547-0%26sid%3Dliteratum%253Aachs%26aulast%3DNeves%26aufirst%3DM.%2BA.%26aulast%3DTotrov%26aufirst%3DM.%26aulast%3DAbagyan%26aufirst%3DR.%26atitle%3DDocking%2520and%2520scoring%2520with%2520ICM%253A%2520the%2520benchmarking%2520results%2520and%2520strategies%2520for%2520improvement%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2012%26doi%3D10.1007%2Fs10822-012-9547-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Spitzer, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jain, A. N.</span><span> </span><span class="NLM_article-title">Surflex-Dock: docking benchmarks and real-world application</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span><span class="NLM_elocation-id"></span><span class="refDoi"> DOI: 10.1007/s10822-011-9533-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Fjm300687e&amp;key=10.1007%2Fs10822-011-9533-y" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&author=R.+Spitzerauthor=A.+N.+Jain&title=Surflex-Dock%3A+docking+benchmarks+and+real-world+application&doi=10.1007%2Fs10822-011-9533-y"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1007%2Fs10822-011-9533-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-011-9533-y%26sid%3Dliteratum%253Aachs%26aulast%3DSpitzer%26aufirst%3DR.%26aulast%3DJain%26aufirst%3DA.%2BN.%26atitle%3DSurflex-Dock%253A%2520docking%2520benchmarks%2520and%2520real-world%2520application%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2012%26doi%3D10.1007%2Fs10822-011-9533-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Schneider, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hindle, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lange, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klein, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Albrecht, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Briem, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beyer, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Claussen, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gastreich, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lemmen, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rarey, M.</span><span> </span><span class="NLM_article-title">Substantial improvements in large-scale redocking and screening using the novel HYDE scoring function</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span><span class="NLM_elocation-id"></span><span class="refDoi"> DOI: 10.1007/s10822-011-9531-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Fjm300687e&amp;key=10.1007%2Fs10822-011-9531-0" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2011&author=N.+Schneiderauthor=S.+Hindleauthor=G.+Langeauthor=R.+Kleinauthor=J.+Albrechtauthor=H.+Briemauthor=K.+Beyerauthor=H.+Claussenauthor=M.+Gastreichauthor=C.+Lemmenauthor=M.+Rarey&title=Substantial+improvements+in+large-scale+redocking+and+screening+using+the+novel+HYDE+scoring+function&doi=10.1007%2Fs10822-011-9531-0"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1007%2Fs10822-011-9531-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-011-9531-0%26sid%3Dliteratum%253Aachs%26aulast%3DSchneider%26aufirst%3DN.%26aulast%3DHindle%26aufirst%3DS.%26aulast%3DLange%26aufirst%3DG.%26aulast%3DKlein%26aufirst%3DR.%26aulast%3DAlbrecht%26aufirst%3DJ.%26aulast%3DBriem%26aufirst%3DH.%26aulast%3DBeyer%26aufirst%3DK.%26aulast%3DClaussen%26aufirst%3DH.%26aulast%3DGastreich%26aufirst%3DM.%26aulast%3DLemmen%26aufirst%3DC.%26aulast%3DRarey%26aufirst%3DM.%26atitle%3DSubstantial%2520improvements%2520in%2520large-scale%2520redocking%2520and%2520screening%2520using%2520the%2520novel%2520HYDE%2520scoring%2520function%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2011%26doi%3D10.1007%2Fs10822-011-9531-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Liebeschuetz, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cole, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korb, O.</span><span> </span><span class="NLM_article-title">Pose prediction and virtual screening performance of GOLD scoring functions in a standardized test</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span><span class="NLM_elocation-id"></span><span class="refDoi"> DOI: 10.1007/s10822-012-9551-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Fjm300687e&amp;key=10.1007%2Fs10822-012-9551-4" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&author=J.+W.+Liebeschuetzauthor=J.+C.+Coleauthor=O.+Korb&title=Pose+prediction+and+virtual+screening+performance+of+GOLD+scoring+functions+in+a+standardized+test&doi=10.1007%2Fs10822-012-9551-4"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1007%2Fs10822-012-9551-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-012-9551-4%26sid%3Dliteratum%253Aachs%26aulast%3DLiebeschuetz%26aufirst%3DJ.%2BW.%26aulast%3DCole%26aufirst%3DJ.%2BC.%26aulast%3DKorb%26aufirst%3DO.%26atitle%3DPose%2520prediction%2520and%2520virtual%2520screening%2520performance%2520of%2520GOLD%2520scoring%2520functions%2520in%2520a%2520standardized%2520test%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2012%26doi%3D10.1007%2Fs10822-012-9551-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Novikov, F. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stroylov, V. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zeifman, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stroganov, O. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kulkov, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chilov, G. G.</span><span> </span><span class="NLM_article-title">Lead Finder docking and virtual screening evaluation with Astex and DUD test sets</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span><span class="NLM_elocation-id"></span><span class="refDoi"> DOI: 10.1007/s10822-012-9549-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Fjm300687e&amp;key=10.1007%2Fs10822-012-9549-y" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&author=F.+N.+Novikovauthor=V.+S.+Stroylovauthor=A.+A.+Zeifmanauthor=O.+V.+Stroganovauthor=V.+Kulkovauthor=G.+G.+Chilov&title=Lead+Finder+docking+and+virtual+screening+evaluation+with+Astex+and+DUD+test+sets&doi=10.1007%2Fs10822-012-9549-y"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1007%2Fs10822-012-9549-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-012-9549-y%26sid%3Dliteratum%253Aachs%26aulast%3DNovikov%26aufirst%3DF.%2BN.%26aulast%3DStroylov%26aufirst%3DV.%2BS.%26aulast%3DZeifman%26aufirst%3DA.%2BA.%26aulast%3DStroganov%26aufirst%3DO.%2BV.%26aulast%3DKulkov%26aufirst%3DV.%26aulast%3DChilov%26aufirst%3DG.%2BG.%26atitle%3DLead%2520Finder%2520docking%2520and%2520virtual%2520screening%2520evaluation%2520with%2520Astex%2520and%2520DUD%2520test%2520sets%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2012%26doi%3D10.1007%2Fs10822-012-9549-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Good, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oprea, T. I.</span><span> </span><span class="NLM_article-title">Optimization of CAMD techniques 3. Virtual screening enrichment studies: a help or hindrance in tool selection?</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">169</span><span class="NLM_x">–</span> <span class="NLM_lpage">178</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2008&pages=169-178&author=A.+C.+Goodauthor=T.+I.+Oprea&title=Optimization+of+CAMD+techniques+3.+Virtual+screening+enrichment+studies%3A+a+help+or+hindrance+in+tool+selection%3F"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGood%26aufirst%3DA.%2BC.%26aulast%3DOprea%26aufirst%3DT.%2BI.%26atitle%3DOptimization%2520of%2520CAMD%2520techniques%25203.%2520Virtual%2520screening%2520enrichment%2520studies%253A%2520a%2520help%2520or%2520hindrance%2520in%2520tool%2520selection%253F%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2008%26volume%3D22%26spage%3D169%26epage%3D178" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Mackey, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Melville, J. L.</span><span> </span><span class="NLM_article-title">Better than random? The chemotype enrichment problem</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">1154</span><span class="NLM_x">–</span> <span class="NLM_lpage">1162</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci8003978" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Fjm300687e&amp;key=1%3ACAS%3A528%3ADC%252BD1MXltVCgs78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2009&pages=1154-1162&author=M.+D.+Mackeyauthor=J.+L.+Melville&title=Better+than+random%3F+The+chemotype+enrichment+problem"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Better than Random? The Chemotype Enrichment Problem</span></div><div class="casAuthors">Mackey, Mark D.; Melville, James L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1154-1162</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Chemotype enrichment is increasingly recognized as an important measure of virtual screening performance.  However, little attention has been paid to producing metrics which can quantify chemotype retrieval.  Here, we examine two different protocols for analyzing chemotype retrieval: "cluster averaging", where the contribution of each active to the scoring metric is proportional to the no. of other actives with the same chemotype, and "first found", where only the first active for a given chemotype contributes to the score.  We demonstrate that this latter anal., common in the qual. anal. used in the current literature, has important drawbacks when combined with quant. metrics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrFRLJq6JcTO7Vg90H21EOLACvtfcHk0li2neQGViyydw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXltVCgs78%253D&md5=51b6efb624c3a383b7546997c11ab1e0</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fci8003978&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci8003978%26sid%3Dliteratum%253Aachs%26aulast%3DMackey%26aufirst%3DM.%2BD.%26aulast%3DMelville%26aufirst%3DJ.%2BL.%26atitle%3DBetter%2520than%2520random%253F%2520The%2520chemotype%2520enrichment%2520problem%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2009%26volume%3D49%26spage%3D1154%26epage%3D1162" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">Hawkins, P. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Warren, G. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Skillman, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nicholls, A.</span><span> </span><span class="NLM_article-title">How to do an evaluation: pitfalls and traps</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">179</span><span class="NLM_x">–</span> <span class="NLM_lpage">190</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Fjm300687e&amp;key=10.1007%2Fs10822-007-9166-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Fjm300687e&amp;key=18217218" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Fjm300687e&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjsFOnsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2008&pages=179-190&author=P.+C.+Hawkinsauthor=G.+L.+Warrenauthor=A.+G.+Skillmanauthor=A.+Nicholls&title=How+to+do+an+evaluation%3A+pitfalls+and+traps"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">How to do an evaluation: pitfalls and traps</span></div><div class="casAuthors">Hawkins, Paul C. D.; Warren, Gregory L.; Skillman, A. Geoffrey; Nicholls, Anthony</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">3-4</span>),
    <span class="NLM_cas:pages">179-190</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">The recent literature is replete with papers evaluating computational tools (often those operating on 3D structures) for their performance in a certain set of tasks.  Most commonly these papers compare a no. of docking tools for their performance in cognate re-docking (pose prediction) and/or virtual screening.  Related papers have been published on ligand-based tools: pose prediction by conformer generators and virtual screening using a variety of ligand-based approaches.  The reliability of these comparisons is critically affected by a no. of factors usually ignored by the authors, including bias in the datasets used in virtual screening, the metrics used to assess performance in virtual screening and pose prediction and errors in crystal structures used.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoyuwDKbJAiRLVg90H21EOLACvtfcHk0li2neQGViyydw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjsFOnsLk%253D&md5=3e176941c4a526f3a97519ce02007792</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1007%2Fs10822-007-9166-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-007-9166-3%26sid%3Dliteratum%253Aachs%26aulast%3DHawkins%26aufirst%3DP.%2BC.%26aulast%3DWarren%26aufirst%3DG.%2BL.%26aulast%3DSkillman%26aufirst%3DA.%2BG.%26aulast%3DNicholls%26aufirst%3DA.%26atitle%3DHow%2520to%2520do%2520an%2520evaluation%253A%2520pitfalls%2520and%2520traps%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2008%26volume%3D22%26spage%3D179%26epage%3D190" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Irwin, J. J.</span><span> </span><span class="NLM_article-title">Community benchmarks for virtual screening</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">193</span><span class="NLM_x">–</span> <span class="NLM_lpage">199</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Fjm300687e&amp;key=10.1007%2Fs10822-008-9189-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Fjm300687e&amp;key=18273555" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Fjm300687e&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjsFOnsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2008&pages=193-199&author=J.+J.+Irwin&title=Community+benchmarks+for+virtual+screening"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Community benchmarks for virtual screening</span></div><div class="casAuthors">Irwin, John J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">3-4</span>),
    <span class="NLM_cas:pages">193-199</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Ligand enrichment among top-ranking hits is a key metric of virtual screening.  To avoid bias, decoys should resemble ligands phys., so that enrichment is not attributable to simple differences of gross features.  We therefore created a directory of useful decoys (DUD) by selecting decoys that resembled annotated ligands phys. but not topol. to benchmark docking performance.  DUD has 2950 annotated ligands and 95,316 property-matched decoys for 40 targets.  It is by far the largest and most comprehensive public data set for benchmarking virtual screening programs that I am aware of.  This paper outlines several ways that DUD can be improved to provide better telemetry to investigators seeking to understand both the strengths and the weaknesses of current docking methods.  I also highlight several pitfalls for the unwary: a risk of over-optimization, questions about chem. space, and the proper scope for using DUD.  Careful attention to both the compn. of benchmarks and how they are used is essential to avoid being misled by overfitting and bias.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoAn7WJldC7AbVg90H21EOLACvtfcHk0li2neQGViyydw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjsFOnsb8%253D&md5=4972b768c915bc3e7ee05210dcf1b7a8</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1007%2Fs10822-008-9189-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-008-9189-4%26sid%3Dliteratum%253Aachs%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26atitle%3DCommunity%2520benchmarks%2520for%2520virtual%2520screening%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2008%26volume%3D22%26spage%3D193%26epage%3D199" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group">Mysinger, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Rapid context-dependent ligand desolvation in molecular docking</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">1561</span><span class="NLM_x">–</span> <span class="NLM_lpage">1573</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci100214a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Fjm300687e&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVGrsLvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=1561-1573&author=M.+M.+Mysingerauthor=B.+K.+Shoichet&title=Rapid+context-dependent+ligand+desolvation+in+molecular+docking"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Rapid Context-Dependent Ligand Desolvation in Molecular Docking</span></div><div class="casAuthors">Mysinger, Michael M.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1561-1573</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In structure-based screens for new ligands, a mol. docking algorithm must rapidly score many mols. in multiple configurations, accounting for both the ligand's interactions with receptor and its competing interactions with solvent.  Here the authors explore a context-dependent ligand desolvation scoring term for mol. docking.  The authors relate the Generalized-Born effective Born radii for every ligand atom to a fractional desolvation and then use this fraction to scale an atom-by-atom decompn. of the full transfer free energy.  The fractional desolvation is precomputed on a scoring grid by numerically integrating over the vol. of receptor proximal to a ligand atom, weighted by distance.  To test this method's performance, the authors dock ligands vs. property-matched decoys over 40 DUD targets.  Context-dependent desolvation better enriches ligands compared to both the raw full transfer free energy penalty and compared to ignoring desolvation altogether, though the improvement is modest.  More compellingly, the new method improves docking performance across receptor types.  Thus, whereas entirely ignoring desolvation works best for charged sites and overpenalizing with full desolvation works well for neutral sites, the phys. more correct context-dependent ligand desolvation is competitive across both types of targets.  The method also reliably discriminates ligands from highly charged mols., where ignoring desolvation performs poorly.  Since this context-dependent ligand desolvation may be precalcd., it improves docking reliability with minimal cost to calcn. time and may be readily incorporated into any physics-based docking program.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrtPnS0XiKpYLVg90H21EOLACvtfcHk0lhPVPDxELBOyg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVGrsLvI&md5=5124d2c4618e07b4f1e41609c9e61362</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fci100214a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci100214a%26sid%3Dliteratum%253Aachs%26aulast%3DMysinger%26aufirst%3DM.%2BM.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DRapid%2520context-dependent%2520ligand%2520desolvation%2520in%2520molecular%2520docking%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D1561%26epage%3D1573" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group">Vogel, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bauer, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boeckler, F. M.</span><span> </span><span class="NLM_article-title">DEKOIS: demanding evaluation kits for objective in silico screening—a versatile tool for benchmarking docking programs and scoring functions</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">2650</span><span class="NLM_x">–</span> <span class="NLM_lpage">2665</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci2001549" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Fjm300687e&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtValsbnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2011&pages=2650-2665&author=S.+M.+Vogelauthor=M.+R.+Bauerauthor=F.+M.+Boeckler&title=DEKOIS%3A+demanding+evaluation+kits+for+objective+in+silico+screening%E2%80%94a+versatile+tool+for+benchmarking+docking+programs+and+scoring+functions"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">DEKOIS: Demanding Evaluation Kits for Objective in Silico Screening - A Versatile Tool for Benchmarking Docking Programs and Scoring Functions</span></div><div class="casAuthors">Vogel, Simon M.; Bauer, Matthias R.; Boeckler, Frank M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2650-2665</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">For widely applied in silico screening techniques success depends on the rational selection of an appropriate method.  We herein present a fast, versatile, and robust method to construct demanding evaluation kits for objective in silico screening (DEKOIS).  This automated process enables creating tailor-made decoy sets for any given sets of bioactives.  It facilitates a target-dependent validation of docking algorithms and scoring functions helping to save time and resources.  We have developed metrics for assessing and improving decoy set quality and employ them to investigate how decoy embedding affects docking.  We demonstrate that screening performance is target-dependent and can be impaired by latent actives in the decoy set (LADS) or enhanced by poor decoy embedding.  The presented method allows extending and complementing the collection of publicly available high quality decoy sets toward new target space.  All present and future DEKOIS data sets will be made accessible at www.dekois.com.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoK-1LphWadorVg90H21EOLACvtfcHk0lhPVPDxELBOyg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtValsbnN&md5=e3f59bcd3f898dce1c116a325e914861</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fci2001549&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci2001549%26sid%3Dliteratum%253Aachs%26aulast%3DVogel%26aufirst%3DS.%2BM.%26aulast%3DBauer%26aufirst%3DM.%2BR.%26aulast%3DBoeckler%26aufirst%3DF.%2BM.%26atitle%3DDEKOIS%253A%2520demanding%2520evaluation%2520kits%2520for%2520objective%2520in%2520silico%2520screening%25E2%2580%2594a%2520versatile%2520tool%2520for%2520benchmarking%2520docking%2520programs%2520and%2520scoring%2520functions%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2011%26volume%3D51%26spage%3D2650%26epage%3D2665" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group">Wallach, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lilien, R.</span><span> </span><span class="NLM_article-title">Virtual decoy sets for molecular docking benchmarks</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">196</span><span class="NLM_x">–</span> <span class="NLM_lpage">202</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci100374f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Fjm300687e&amp;key=1%3ACAS%3A528%3ADC%252BC3MXoslel" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2011&pages=196-202&author=I.+Wallachauthor=R.+Lilien&title=Virtual+decoy+sets+for+molecular+docking+benchmarks"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Virtual Decoy Sets for Molecular Docking Benchmarks</span></div><div class="casAuthors">Wallach, Izhar; Lilien, Ryan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">196-202</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Virtual docking algorithms are often evaluated on their ability to sep. active ligands from decoy mols.  The current state-of-the-art benchmark, the Directory of Useful Decoys (DUD), minimizes bias by including decoys from a library of synthetically feasible mols. that are phys. similar yet chem. dissimilar to the active ligands.  We show that by ignoring synthetic feasibility, we can compile a benchmark that is comparable to the DUD and less biased with respect to phys. similarity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoTD04ZpF-QV7Vg90H21EOLACvtfcHk0lhPVPDxELBOyg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXoslel&md5=3b58c7641cb2c665ce381081d8b1bf41</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Fci100374f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci100374f%26sid%3Dliteratum%253Aachs%26aulast%3DWallach%26aufirst%3DI.%26aulast%3DLilien%26aufirst%3DR.%26atitle%3DVirtual%2520decoy%2520sets%2520for%2520molecular%2520docking%2520benchmarks%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2011%26volume%3D51%26spage%3D196%26epage%3D202" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group">Gatica, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cavasotto, C. N.</span><span> </span><span class="NLM_article-title">Ligand and decoy sets for docking to G protein-coupled receptors</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">6</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci200412p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Fjm300687e&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs1ajsb3P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=1-6&author=E.+A.+Gaticaauthor=C.+N.+Cavasotto&title=Ligand+and+decoy+sets+for+docking+to+G+protein-coupled+receptors"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand and Decoy Sets for Docking to G Protein-Coupled Receptors</span></div><div class="casAuthors">Gatica, Edgar A.; Cavasotto, Claudio N.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-6</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We compiled a G protein-coupled receptor (GPCR) ligand library (GLL) for 147 targets, selecting for each ligand 39 decoy mols., collected in the GPCR Decoy Database (GDD).  Decoys were chosen ensuring a ligand-decoy similarity of six phys. properties, while enforcing ligand-decoy chem. dissimilarity.  The performance in docking of the GDD was evaluated on 19 GPCRs, showing a marked decrease in enrichment compared to bias-uncorrected decoy sets.  Both the GLL and GDD are freely available for the scientific community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrYHZXxtR-bB7Vg90H21EOLACvtfcHk0lhPVPDxELBOyg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs1ajsb3P&md5=a51e1b60afda728614a3bc1ceed79cfd</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Fci200412p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci200412p%26sid%3Dliteratum%253Aachs%26aulast%3DGatica%26aufirst%3DE.%2BA.%26aulast%3DCavasotto%26aufirst%3DC.%2BN.%26atitle%3DLigand%2520and%2520decoy%2520sets%2520for%2520docking%2520to%2520G%2520protein-coupled%2520receptors%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26spage%3D1%26epage%3D6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group">Cereto-Massague, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guasch, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valls, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mulero, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pujadas, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garcia-Vallve, S.</span><span> </span><span class="NLM_article-title">DecoyFinder: an easy-to-use python GUI application for building target-specific decoy sets</span> <span class="citation_source-journal">Bioinformatics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">1661</span><span class="NLM_x">–</span> <span class="NLM_lpage">1662</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2012&pages=1661-1662&author=A.+Cereto-Massagueauthor=L.+Guaschauthor=C.+Vallsauthor=M.+Muleroauthor=G.+Pujadasauthor=S.+Garcia-Vallve&title=DecoyFinder%3A+an+easy-to-use+python+GUI+application+for+building+target-specific+decoy+sets"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DCereto-Massague%26aufirst%3DA.%26aulast%3DGuasch%26aufirst%3DL.%26aulast%3DValls%26aufirst%3DC.%26aulast%3DMulero%26aufirst%3DM.%26aulast%3DPujadas%26aufirst%3DG.%26aulast%3DGarcia-Vallve%26aufirst%3DS.%26atitle%3DDecoyFinder%253A%2520an%2520easy-to-use%2520python%2520GUI%2520application%2520for%2520building%2520target-specific%2520decoy%2520sets%26jtitle%3DBioinformatics%26date%3D2012%26volume%3D28%26spage%3D1661%26epage%3D1662" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group">Rohrer, S. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baumann, K.</span><span> </span><span class="NLM_article-title">Maximum unbiased validation (MUV) data sets for virtual screening based on PubChem bioactivity data</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">169</span><span class="NLM_x">–</span> <span class="NLM_lpage">184</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci8002649" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Fjm300687e&amp;key=1%3ACAS%3A528%3ADC%252BD1MXptlOhtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2009&pages=169-184&author=S.+G.+Rohrerauthor=K.+Baumann&title=Maximum+unbiased+validation+%28MUV%29+data+sets+for+virtual+screening+based+on+PubChem+bioactivity+data"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Maximum Unbiased Validation (MUV) Data Sets for Virtual Screening Based on PubChem Bioactivity Data</span></div><div class="casAuthors">Rohrer, Sebastian G.; Baumann, Knut</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">169-184</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Refined nearest neighbor anal. was recently introduced for the anal. of virtual screening benchmark data sets.  It constitutes a technique from the field of spatial statistics and provides a math. framework for the nonparametric anal. of mapped point patterns.  Here, refined nearest neighbor anal. is used to design benchmark data sets for virtual screening based on PubChem bioactivity data.  A workflow is devised that purges data sets of compds. active against pharmaceutically relevant targets from unselective hits.  Topol. optimization using exptl. design strategies monitored by refined nearest neighbor anal. functions is applied to generate corresponding data sets of actives and decoys that are unbiased with regard to analog bias and artificial enrichment.  These data sets provide a tool for Maximum Unbiased Validation (MUV) of virtual screening methods.  The data sets and a software package implementing the MUV design workflow are freely available.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGph4RHBumlGlLVg90H21EOLACvtfcHk0lh7Se2yxcHvfA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXptlOhtQ%253D%253D&md5=17805f49d995c4c082c3d07be7145a2d</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Fci8002649&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci8002649%26sid%3Dliteratum%253Aachs%26aulast%3DRohrer%26aufirst%3DS.%2BG.%26aulast%3DBaumann%26aufirst%3DK.%26atitle%3DMaximum%2520unbiased%2520validation%2520%2528MUV%2529%2520data%2520sets%2520for%2520virtual%2520screening%2520based%2520on%2520PubChem%2520bioactivity%2520data%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2009%26volume%3D49%26spage%3D169%26epage%3D184" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group">Ripphausen, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wassermann, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bajorath, J.</span><span> </span><span class="NLM_article-title">REPROVIS-DB: a benchmark system for ligand-based virtual screening derived from reproducible prospective applications</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">2467</span><span class="NLM_x">–</span> <span class="NLM_lpage">2473</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci200309j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Fjm300687e&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1ait7fJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2011&pages=2467-2473&author=P.+Ripphausenauthor=A.+M.+Wassermannauthor=J.+Bajorath&title=REPROVIS-DB%3A+a+benchmark+system+for+ligand-based+virtual+screening+derived+from+reproducible+prospective+applications"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">REPROVIS-DB: A Benchmark System for Ligand-Based Virtual Screening Derived from Reproducible Prospective Applications</span></div><div class="casAuthors">Ripphausen, Peter; Wassermann, Anne Mai; Bajorath, Jurgen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2467-2473</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Benchmark calcns. are essential for the evaluation of virtual screening (VS) methods.  Typically, classes of known active compds. taken from the medicinal chem. literature are divided into ref. mols. (search templates) and potential hits that are added to background databases assumed to consist of compds. not sharing this activity.  Then VS calcns. are carried out, and the recall of known active compds. is detd.  However, conventional benchmarking is affected by a no. of problems that reduce its value for method evaluation.  In addn. to often insufficient statistical validation and the lack of generally accepted evaluation stds., the artificial nature of typical benchmark settings is often criticized.  Retrospective benchmark calcns. generally overestimate the potential of VS methods and do not scale with their performance in prospective applications.  In order to provide addnl. opportunities for benchmarking that more closely resemble practical VS conditions, we have designed a publicly available compd. database (DB) of reproducible virtual screens (REPROVIS-DB) that organizes information from successful ligand-based VS applications including ref. compds., screening databases, compd. selection criteria, and exptl. confirmed hits.  Using the currently available 25 hand-selected compd. data sets, one can attempt to reproduce successful virtual screens with other than the originally applied methods and assess their potential for practical applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDPwJwNXQc67Vg90H21EOLACvtfcHk0lh7Se2yxcHvfA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1ait7fJ&md5=8439a3f6471f43afe159d08c28a5bc14</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Fci200309j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci200309j%26sid%3Dliteratum%253Aachs%26aulast%3DRipphausen%26aufirst%3DP.%26aulast%3DWassermann%26aufirst%3DA.%2BM.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DREPROVIS-DB%253A%2520a%2520benchmark%2520system%2520for%2520ligand-based%2520virtual%2520screening%2520derived%2520from%2520reproducible%2520prospective%2520applications%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2011%26volume%3D51%26spage%3D2467%26epage%3D2473" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group">Gaulton, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bellis, L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bento, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chambers, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davies, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hersey, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Light, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McGlinchey, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Michalovich, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Al-Lazikani, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Overington, J. P.</span><span> </span><span class="NLM_article-title">ChEMBL: a large-scale bioactivity database for drug discovery</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">D1100</span><span class="NLM_x">–</span> <span class="NLM_lpage">1107</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Fjm300687e&amp;key=10.1093%2Fnar%2Fgkr777" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Fjm300687e&amp;key=21948594" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Fjm300687e&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs12htbjN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2012&pages=D1100-1107&author=A.+Gaultonauthor=L.+J.+Bellisauthor=A.+P.+Bentoauthor=J.+Chambersauthor=M.+Daviesauthor=A.+Herseyauthor=Y.+Lightauthor=S.+McGlincheyauthor=D.+Michalovichauthor=B.+Al-Lazikaniauthor=J.+P.+Overington&title=ChEMBL%3A+a+large-scale+bioactivity+database+for+drug+discovery"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">ChEMBL: a large-scale bioactivity database for drug discovery</span></div><div class="casAuthors">Gaulton, Anna; Bellis, Louisa J.; Bento, A. Patricia; Chambers, Jon; Davies, Mark; Hersey, Anne; Light, Yvonne; McGlinchey, Shaun; Michalovich, David; Al-Lazikani, Bissan; Overington, John P.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D1100-D1107</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">ChEMBL is an Open Data database contg. binding, functional and ADMET information for a large no. of drug-like bioactive compds.  These data are manually abstracted from the primary published literature on a regular basis, then further curated and standardized to maximize their quality and utility across a wide range of chem. biol. and drug-discovery research problems.  Currently, the database contains 5.4 million bioactivity measurements for more than 1 million compds. and 5200 protein targets.  Access is available through a web-based interface, data downloads and web services at: https://www.ebi.ac.uk/chembldb.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9e9jRENTtubVg90H21EOLACvtfcHk0lh7Se2yxcHvfA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs12htbjN&md5=aedf7793e1ca54b6a4fa272ea3ef7d0e</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkr777&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkr777%26sid%3Dliteratum%253Aachs%26aulast%3DGaulton%26aufirst%3DA.%26aulast%3DBellis%26aufirst%3DL.%2BJ.%26aulast%3DBento%26aufirst%3DA.%2BP.%26aulast%3DChambers%26aufirst%3DJ.%26aulast%3DDavies%26aufirst%3DM.%26aulast%3DHersey%26aufirst%3DA.%26aulast%3DLight%26aufirst%3DY.%26aulast%3DMcGlinchey%26aufirst%3DS.%26aulast%3DMichalovich%26aufirst%3DD.%26aulast%3DAl-Lazikani%26aufirst%3DB.%26aulast%3DOverington%26aufirst%3DJ.%2BP.%26atitle%3DChEMBL%253A%2520a%2520large-scale%2520bioactivity%2520database%2520for%2520drug%2520discovery%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2012%26volume%3D40%26spage%3DD1100%26epage%3D1107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group">Bemis, G. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murcko, M. A.</span><span> </span><span class="NLM_article-title">The properties of known drugs. 1. Molecular frameworks</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">39</span><span class="NLM_x">, </span> <span class="NLM_fpage">2887</span><span class="NLM_x">–</span> <span class="NLM_lpage">2893</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm9602928" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Fjm300687e&amp;key=1%3ACAS%3A528%3ADyaK28XjvVejtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=1996&pages=2887-2893&author=G.+W.+Bemisauthor=M.+A.+Murcko&title=The+properties+of+known+drugs.+1.+Molecular+frameworks"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">The Properties of Known Drugs. 1. Molecular Frameworks</span></div><div class="casAuthors">Bemis, Guy W.; Murcko, Mark A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">2887-2893</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">To better understand the common features present in drug mols., we use shape description methods to analyze a database of com. available drugs and prep. a list of common drug shapes.  A useful way of organizing this structural data is to group the atoms of each drug mol. into ring, linker, framework, and side chain atoms.  On the basis of the two-dimensional mol. structures (without regard to atom type, hybridization, and bond order), there are 1179 different frameworks among the 5120 compds. analyzed.  However, the shapes of half of the drugs in the database are described by the 32 most frequently occurring frameworks.  This suggests that the diversity of shapes in the set of known drugs is extremely low.  In our second method of anal., in which atom type, hybridization, and bond order are considered, more diversity is seen; there are 2506 different frameworks among the 5120 compds. in the database, and the most frequently occurring 42 frameworks account for only one-fourth of the drugs.  We discuss the possible interpretations of these findings and the way they may be used to guide future drug discovery research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoodEb3sfDSdrVg90H21EOLACvtfcHk0ljNcweJCPTDQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XjvVejtro%253D&md5=5e2c4fdfea9434456a0cca83de4185b3</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Fjm9602928&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm9602928%26sid%3Dliteratum%253Aachs%26aulast%3DBemis%26aufirst%3DG.%2BW.%26aulast%3DMurcko%26aufirst%3DM.%2BA.%26atitle%3DThe%2520properties%2520of%2520known%2520drugs.%25201.%2520Molecular%2520frameworks%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1996%26volume%3D39%26spage%3D2887%26epage%3D2893" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group">Berman, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Westbrook, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilliland, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bhat, T. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weissig, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shindyalov, I. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourne, P. E.</span><span> </span><span class="NLM_article-title">The Protein Data Bank</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">235</span><span class="NLM_x">–</span> <span class="NLM_lpage">242</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Fjm300687e&amp;key=10.1093%2Fnar%2F28.1.235" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Fjm300687e&amp;key=10592235" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Fjm300687e&amp;key=1%3ACAS%3A528%3ADC%252BD3cXhvVKjt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2000&pages=235-242&author=H.+M.+Bermanauthor=J.+Westbrookauthor=Z.+Fengauthor=G.+Gillilandauthor=T.+N.+Bhatauthor=H.+Weissigauthor=I.+N.+Shindyalovauthor=P.+E.+Bourne&title=The+Protein+Data+Bank"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">The Protein Data Bank</span></div><div class="casAuthors">Berman, Helen M.; Westbrook, John; Feng, Zukang; Gilliland, Gary; Bhat, T. N.; Weissig, Helge; Shindyalov, Ilya N.; Bourne, Philip E.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The Protein Data Bank (PDB; http://www.rcsb.org/pdb/)is the single worldwide archive of structural data of biol. macromols.  This paper describes the goals of the PDB, the systems in place for data deposition and access, how to obtain further information, and near-term plans for the future development of the resource.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpStGCxT8QzirVg90H21EOLACvtfcHk0ljNcweJCPTDQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXhvVKjt7w%253D&md5=227fb393f754be2be375ab727bfd05dc</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1093%2Fnar%2F28.1.235&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252F28.1.235%26sid%3Dliteratum%253Aachs%26aulast%3DBerman%26aufirst%3DH.%2BM.%26aulast%3DWestbrook%26aufirst%3DJ.%26aulast%3DFeng%26aufirst%3DZ.%26aulast%3DGilliland%26aufirst%3DG.%26aulast%3DBhat%26aufirst%3DT.%2BN.%26aulast%3DWeissig%26aufirst%3DH.%26aulast%3DShindyalov%26aufirst%3DI.%2BN.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26atitle%3DThe%2520Protein%2520Data%2520Bank%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2000%26volume%3D28%26spage%3D235%26epage%3D242" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group">Apweiler, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bairoch, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barker, W. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boeckmann, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferro, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gasteiger, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lopez, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Magrane, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Natale, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Donovan, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Redaschi, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yeh, L. S.</span><span> </span><span class="NLM_article-title">UniProt: The Universal Protein Knowledgebase</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">32</span><span class="NLM_x">, </span> <span class="NLM_fpage">D115</span><span class="NLM_x">–</span> <span class="NLM_lpage">D119</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Fjm300687e&amp;key=10.1093%2Fnar%2Fgkh131" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Fjm300687e&amp;key=14681372" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Fjm300687e&amp;key=1%3ACAS%3A528%3ADC%252BD3sXhtVSru7vK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2004&pages=D115-D119&author=R.+Apweilerauthor=A.+Bairochauthor=C.+H.+Wuauthor=W.+C.+Barkerauthor=B.+Boeckmannauthor=S.+Ferroauthor=E.+Gasteigerauthor=H.+Huangauthor=R.+Lopezauthor=M.+Magraneauthor=M.+J.+Martinauthor=D.+A.+Nataleauthor=C.+O%E2%80%99Donovanauthor=N.+Redaschiauthor=L.+S.+Yeh&title=UniProt%3A+The+Universal+Protein+Knowledgebase"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">UniProt: the universal protein knowledgebase</span></div><div class="casAuthors">Apweiler, Rolf; Bairoch, Amos; Wu, Cathy H.; Barker, Winona C.; Boeckmann, Brigitte; Ferro, Serenella; Gasteiger, Elisabeth; Huang, Hongzhan; Lopez, Rodrigo; Magrane, Michele; Martin, Maria J.; Natale, Darren A.; O'Donovan, Claire; Redaschi, Nicole; Yeh, Lai-Su L.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">Database</span>),
    <span class="NLM_cas:pages">D115-D119</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">To provide the scientific community with a single, centralized, authoritative resource for protein sequences and functional information, the Swiss-Prot, TrEMBL and PIR protein database activities have united to form the Universal Protein Knowledgebase (UniProt) consortium.  Our mission is to provide a comprehensive, fully classified, richly and accurately annotated protein sequence knowledgebase, with extensive cross-refs. and query interfaces.  The central database will have two sections, corresponding to the familiar Swiss-Prot (fully manually curated entries) and TrEMBL (enriched with automated classification, annotation and extensive cross-refs.).  For convenient sequence searches, UniProt also provides several non-redundant sequence databases.  The UniProt NREF (UniRef) databases provide representative subsets of the knowledgebase suitable for efficient searching.  The comprehensive UniProt Archive (UniParc) is updated daily from many public source databases.  The UniProt databases can be accessed online (http://www.uniprot.org) or downloaded in several formats (ftp://ftp.uniprot.org/pub).  The scientific community is encouraged to submit data for inclusion in UniProt.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGos0pWEYnhzKLVg90H21EOLACvtfcHk0ljNcweJCPTDQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXhtVSru7vK&md5=4b692ffbb5c27bd9b4d2a8d0809e84e1</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkh131&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkh131%26sid%3Dliteratum%253Aachs%26aulast%3DApweiler%26aufirst%3DR.%26aulast%3DBairoch%26aufirst%3DA.%26aulast%3DWu%26aufirst%3DC.%2BH.%26aulast%3DBarker%26aufirst%3DW.%2BC.%26aulast%3DBoeckmann%26aufirst%3DB.%26aulast%3DFerro%26aufirst%3DS.%26aulast%3DGasteiger%26aufirst%3DE.%26aulast%3DHuang%26aufirst%3DH.%26aulast%3DLopez%26aufirst%3DR.%26aulast%3DMagrane%26aufirst%3DM.%26aulast%3DMartin%26aufirst%3DM.%2BJ.%26aulast%3DNatale%26aufirst%3DD.%2BA.%26aulast%3DO%25E2%2580%2599Donovan%26aufirst%3DC.%26aulast%3DRedaschi%26aufirst%3DN.%26aulast%3DYeh%26aufirst%3DL.%2BS.%26atitle%3DUniProt%253A%2520The%2520Universal%2520Protein%2520Knowledgebase%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2004%26volume%3D32%26spage%3DD115%26epage%3DD119" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mysinger, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Colizzi, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wassam, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cao, Y.</span><span> </span><span class="NLM_article-title">Automated docking screens: a feasibility study</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">5712</span><span class="NLM_x">–</span> <span class="NLM_lpage">5720</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm9006966" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Fjm300687e&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtVOjurjN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=5712-5720&author=J.+J.+Irwinauthor=B.+K.+Shoichetauthor=M.+M.+Mysingerauthor=N.+Huangauthor=F.+Colizziauthor=P.+Wassamauthor=Y.+Cao&title=Automated+docking+screens%3A+a+feasibility+study"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Automated Docking Screens: A Feasibility Study</span></div><div class="casAuthors">Irwin, John J.; Shoichet, Brian K.; Mysinger, Michael M.; Huang, Niu; Colizzi, Francesco; Wassam, Pascal; Cao, Yiqun</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">5712-5720</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mol. docking is the most practical approach to leverage protein structure for ligand discovery, but the technique retains important liabilities that make it challenging to deploy on a large scale.  We have therefore created an expert system, DOCK Blaster, to investigate the feasibility of full automation.  The method requires a PDB code, sometimes with a ligand structure, and from that alone can launch a full screen of large libraries.  A crit. feature is self-assessment, which ests. the anticipated reliability of the automated screening results using pose fidelity and enrichment.  Against common benchmarks, DOCK Blaster recapitulates the crystal ligand pose within 2 Å rmsd 50-60% of the time; inferior to an expert, but respectable.  Half the time the ligand also ranked among the top 5% of 100 phys. matched decoys chosen on the fly.  Further tests were undertaken culminating in a study of 7755 eligible PDB structures.  In 1398 cases, the redocked ligand ranked in the top 5% of 100 property-matched decoys while also posing within 2 Å rmsd, suggesting that unsupervised prospective docking is viable.  DOCK Blaster is available at http://blaster.docking.org.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPZAd6CrHGp7Vg90H21EOLACvtfcHk0lhGLwbnuUtqVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtVOjurjN&md5=f302db1f77b87f26d1fa2fece55bca96</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Fjm9006966&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm9006966%26sid%3Dliteratum%253Aachs%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DMysinger%26aufirst%3DM.%2BM.%26aulast%3DHuang%26aufirst%3DN.%26aulast%3DColizzi%26aufirst%3DF.%26aulast%3DWassam%26aufirst%3DP.%26aulast%3DCao%26aufirst%3DY.%26atitle%3DAutomated%2520docking%2520screens%253A%2520a%2520feasibility%2520study%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D5712%26epage%3D5720" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group">Powers, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morandi, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Structure-based discovery of a novel, noncovalent inhibitor of AmpC beta-lactamase</span> <span class="citation_source-journal">Structure</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">1013</span><span class="NLM_x">–</span> <span class="NLM_lpage">1023</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Fjm300687e&amp;key=10.1016%2FS0969-2126%2802%2900799-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Fjm300687e&amp;key=12121656" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Fjm300687e&amp;key=1%3ACAS%3A528%3ADC%252BD38XlsVyjsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2002&pages=1013-1023&author=R.+A.+Powersauthor=F.+Morandiauthor=B.+K.+Shoichet&title=Structure-based+discovery+of+a+novel%2C+noncovalent+inhibitor+of+AmpC+beta-lactamase"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Discovery of a Novel, Noncovalent Inhibitor of AmpC β-Lactamase</span></div><div class="casAuthors">Powers, Rachel A.; Morandi, Federica; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1013-1023</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">β-Lactamases are the most widespread resistance mechanisms to β-lactam antibiotics, and there is a pressing need for novel, non-β-lactam drugs.  A database of over 200,000 compds. was docked to the active site of AmpC β-lactamase to identify potential inhibitors.  Fifty-six compds. were tested, and three had Ki values of 650 μM or better.  The best of these, 3-[(4-chloroanilino)sulfonyl]thiophene-2-carboxylic acid, was a competitive noncovalent inhibitor (Ki = 26 μM), which also reversed resistance to β-lactams in bacteria expressing AmpC.  The structure of AmpC in complex with this compd. was detd. by x-ray crystallog. to 1.94 A and reveals that the inhibitor interacts with key active-site residues in sites targeted in the docking calcn.  Indeed, the exptl. detd. conformation of the inhibitor closely resembles the prediction.  The structure of the enzyme-inhibitor complex presents an opportunity to improve binding affinity in a novel series of inhibitors discovered by structure-based methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKX-gft1lXA7Vg90H21EOLACvtfcHk0lhGLwbnuUtqVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XlsVyjsrk%253D&md5=5f539484d08191f0c65626493fc1b262</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1016%2FS0969-2126%2802%2900799-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0969-2126%252802%252900799-2%26sid%3Dliteratum%253Aachs%26aulast%3DPowers%26aufirst%3DR.%2BA.%26aulast%3DMorandi%26aufirst%3DF.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DStructure-based%2520discovery%2520of%2520a%2520novel%252C%2520noncovalent%2520inhibitor%2520of%2520AmpC%2520beta-lactamase%26jtitle%3DStructure%26date%3D2002%26volume%3D10%26spage%3D1013%26epage%3D1023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group">Carlsson, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yoo, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gao, Z. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacobson, K. A.</span><span> </span><span class="NLM_article-title">Structure-based discovery of A2A adenosine receptor ligands</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">3748</span><span class="NLM_x">–</span> <span class="NLM_lpage">3755</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm100240h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Fjm300687e&amp;key=1%3ACAS%3A528%3ADC%252BC3cXkvFaqsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=3748-3755&author=J.+Carlssonauthor=L.+Yooauthor=Z.+G.+Gaoauthor=J.+J.+Irwinauthor=B.+K.+Shoichetauthor=K.+A.+Jacobson&title=Structure-based+discovery+of+A2A+adenosine+receptor+ligands"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Discovery of A2A Adenosine Receptor Ligands</span></div><div class="casAuthors">Carlsson, Jens; Yoo, Lena; Gao, Zhan-Guo; Irwin, John J.; Shoichet, Brian K.; Jacobson, Kenneth A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3748-3755</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The recent detn. of X-ray structures of pharmacol. relevant GPCRs has made these targets accessible to structure-based ligand discovery.  Here we explore whether novel chemotypes may be discovered for the A2A adenosine receptor, based on complementarity to its recently detd. structure.  The A2A adenosine receptor signals in the periphery and the CNS, with agonists explored as anti-inflammatory drugs and antagonists explored for neurodegenerative diseases.  We used mol. docking to screen a 1.4 million compd. database against the X-ray structure computationally and tested 20 high-ranking, previously unknown mols. exptl.  Of these 35% showed substantial activity with affinities between 200 nM and 9 μM.  For the most potent of these new inhibitors, over 50-fold specificity was obsd. for the A2A vs. the related A1 and A3 subtypes.  These high hit rates and affinities at least partly reflect the bias of com. libraries toward GPCR-like chemotypes, an issue that we attempt to investigate quant.  Despite this bias, many of the most potent new ligands were novel, dissimilar from known ligands, providing new lead structures for modulation of this medically important target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpzvS6-9T9bd7Vg90H21EOLACvtfcHk0lhGLwbnuUtqVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXkvFaqsL8%253D&md5=c36c941d52d2cec06387d79c4c423d46</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Fjm100240h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm100240h%26sid%3Dliteratum%253Aachs%26aulast%3DCarlsson%26aufirst%3DJ.%26aulast%3DYoo%26aufirst%3DL.%26aulast%3DGao%26aufirst%3DZ.%2BG.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DJacobson%26aufirst%3DK.%2BA.%26atitle%3DStructure-based%2520discovery%2520of%2520A2A%2520adenosine%2520receptor%2520ligands%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D3748%26epage%3D3755" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group">Carlsson, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coleman, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Setola, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fan, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schlessinger, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sali, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Ligand discovery from a dopamine D3 receptor homology model and crystal structure</span> <span class="citation_source-journal">Natre Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">769</span><span class="NLM_x">–</span> <span class="NLM_lpage">778</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Fjm300687e&amp;key=10.1038%2Fnchembio.662" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Fjm300687e&amp;key=21926995" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Fjm300687e&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFylsLvP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=769-778&author=J.+Carlssonauthor=R.+G.+Colemanauthor=V.+Setolaauthor=J.+J.+Irwinauthor=H.+Fanauthor=A.+Schlessingerauthor=A.+Saliauthor=B.+L.+Rothauthor=B.+K.+Shoichet&title=Ligand+discovery+from+a+dopamine+D3+receptor+homology+model+and+crystal+structure"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand discovery from a dopamine D3 receptor homology model and crystal structure</span></div><div class="casAuthors">Carlsson, Jens; Coleman, Ryan G.; Setola, Vincent; Irwin, John J.; Fan, Hao; Schlessinger, Avner; Sali, Andrej; Roth, Bryan L.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">769-778</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">G protein-coupled receptors (GPCRs) are intensely studied as drug targets and for their role in signaling.  With the detn. of the first crystal structures, interest in structure-based ligand discovery increased.  Unfortunately, for most GPCRs no exptl. structures are available.  The detn. of the D3 receptor structure and the challenge to the community to predict it enabled a fully prospective comparison of ligand discovery from a modeled structure vs. that of the subsequently released crystal structure.  Over 3.3 million mols. were docked against a homol. model, and 26 of the highest ranking were tested for binding.  Six had affinities ranging from 0.2 to 3.1 μM.  Subsequently, the crystal structure was released and the docking screen repeated.  Of the 25 compds. selected, five had affinities ranging from 0.3 to 3.0 μM.  One of the new ligands from the homol. model screen was optimized for affinity to 81 nM.  The feasibility of docking screens against modeled GPCRs more generally is considered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqz6gBx3Olbn7Vg90H21EOLACvtfcHk0lhGLwbnuUtqVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFylsLvP&md5=7242aa512079a348678b309d352ecdb9</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.662&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.662%26sid%3Dliteratum%253Aachs%26aulast%3DCarlsson%26aufirst%3DJ.%26aulast%3DColeman%26aufirst%3DR.%2BG.%26aulast%3DSetola%26aufirst%3DV.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DFan%26aufirst%3DH.%26aulast%3DSchlessinger%26aufirst%3DA.%26aulast%3DSali%26aufirst%3DA.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DLigand%2520discovery%2520from%2520a%2520dopamine%2520D3%2520receptor%2520homology%2520model%2520and%2520crystal%2520structure%26jtitle%3DNatre%2520Chem.%2520Biol.%26date%3D2011%26volume%3D7%26spage%3D769%26epage%3D778" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">ZINC—a free database of commercially available compounds for virtual screening</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">177</span><span class="NLM_x">–</span> <span class="NLM_lpage">182</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci049714%2B" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Fjm300687e&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVOjt77J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2005&pages=177-182&author=J.+J.+Irwinauthor=B.+K.+Shoichet&title=ZINC%E2%80%94a+free+database+of+commercially+available+compounds+for+virtual+screening"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">ZINC - A Free Database of Commercially Available Compounds for Virtual Screening</span></div><div class="casAuthors">Irwin, John J.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Computer Sciences</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">177-182</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">0095-2338</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A crit. barrier to entry into structure-based virtual screening is the lack of a suitable, easy to access database of purchasable compds.  We have therefore prepd. a library of 727 842 mols., each with 3D structure, using catalogs of compds. from vendors (the size of this library continues to grow).  The mols. have been assigned biol. relevant protonation states and are annotated with properties such as mol. wt., calcd. LogP, and no. of rotatable bonds.  Each mol. in the library contains vendor and purchasing information and is ready for docking using a no. of popular docking programs.  Within certain limits, the mols. are prepd. in multiple protonation states and multiple tautomeric forms.  In one format, multiple conformations are available for the mols.  This database is available for free download (http://zinc.docking.org) in several common file formats including SMILES, mol2, 3D SDF, and DOCK flexibase format.  A Web-based query tool incorporating a mol. drawing interface enables the database to be searched and browsed and subsets to be created.  Users can process their own mols. by uploading them to a server.  Our hope is that this database will bring virtual screening libraries to a wide community of structural biologists and medicinal chemists.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPhFw5E2XACbVg90H21EOLACvtfcHk0lhULCWXHWhn7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVOjt77J&md5=e3892b7dc8608b17a3e63541a5ed60e6</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Fci049714%2B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci049714%252B%26sid%3Dliteratum%253Aachs%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DZINC%25E2%2580%2594a%2520free%2520database%2520of%2520commercially%2520available%2520compounds%2520for%2520virtual%2520screening%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2005%26volume%3D45%26spage%3D177%26epage%3D182" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group">Velankar, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McNeil, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mittard-Runte, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suarez, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barrell, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Apweiler, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Henrick, K.</span><span> </span><span class="NLM_article-title">E-MSD: an integrated data resource for bioinformatics</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">33</span><span class="NLM_x">, </span> <span class="NLM_fpage">D262</span><span class="NLM_x">–</span> <span class="NLM_lpage">265</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2005&pages=D262-265&author=S.+Velankarauthor=P.+McNeilauthor=V.+Mittard-Runteauthor=A.+Suarezauthor=D.+Barrellauthor=R.+Apweilerauthor=K.+Henrick&title=E-MSD%3A+an+integrated+data+resource+for+bioinformatics"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DVelankar%26aufirst%3DS.%26aulast%3DMcNeil%26aufirst%3DP.%26aulast%3DMittard-Runte%26aufirst%3DV.%26aulast%3DSuarez%26aufirst%3DA.%26aulast%3DBarrell%26aufirst%3DD.%26aulast%3DApweiler%26aufirst%3DR.%26aulast%3DHenrick%26aufirst%3DK.%26atitle%3DE-MSD%253A%2520an%2520integrated%2520data%2520resource%2520for%2520bioinformatics%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2005%26volume%3D33%26spage%3DD262%26epage%3D265" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group">Hawkins, P. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Skillman, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nicholls, A.</span><span> </span><span class="NLM_article-title">Comparison of shape-matching and docking as virtual screening tools</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">74</span><span class="NLM_x">–</span> <span class="NLM_lpage">82</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm0603365" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Fjm300687e&amp;key=1%3ACAS%3A528%3ADC%252BD28Xhtlansb%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2007&pages=74-82&author=P.+C.+Hawkinsauthor=A.+G.+Skillmanauthor=A.+Nicholls&title=Comparison+of+shape-matching+and+docking+as+virtual+screening+tools"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Comparison of Shape-Matching and Docking as Virtual Screening Tools</span></div><div class="casAuthors">Hawkins, Paul C. D.; Skillman, A. Geoffrey; Nicholls, Anthony</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">74-82</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Ligand docking is a widely used approach in virtual screening.  In recent years a large no. of publications have appeared in which docking tools are compared and evaluated for their effectiveness in virtual screening against a wide variety of protein targets.  These studies have shown that the effectiveness of docking in virtual screening is highly variable due to a large no. of possible confounding factors.  Another class of method that has shown promise in virtual screening is the shape-based, ligand-centric approach.  Several direct comparisons of docking with the shape-based tool ROCS have been conducted using data sets from some of these recent docking publications.  The results show that a shape-based, ligand-centric approach is more consistent than, and often superior to, the protein-centric approach taken by docking.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoe0IKOly7zibVg90H21EOLACvtfcHk0lhULCWXHWhn7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xhtlansb%252FF&md5=6f97f5c0cc092b4e225f7c2656c1bcf6</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1021%2Fjm0603365&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0603365%26sid%3Dliteratum%253Aachs%26aulast%3DHawkins%26aufirst%3DP.%2BC.%26aulast%3DSkillman%26aufirst%3DA.%2BG.%26aulast%3DNicholls%26aufirst%3DA.%26atitle%3DComparison%2520of%2520shape-matching%2520and%2520docking%2520as%2520virtual%2520screening%2520tools%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2007%26volume%3D50%26spage%3D74%26epage%3D82" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group">Teotico, D. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Babaoglu, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rocklin, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferreira, R. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giannetti, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Docking for fragment inhibitors of AmpC beta-lactamase</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U.S.A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">106</span><span class="NLM_x">, </span> <span class="NLM_fpage">7455</span><span class="NLM_x">–</span> <span class="NLM_lpage">7460</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Fjm300687e&amp;key=10.1073%2Fpnas.0813029106" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Fjm300687e&amp;key=19416920" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Fjm300687e&amp;key=1%3ACAS%3A528%3ADC%252BD1MXmt1Khtrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2009&pages=7455-7460&author=D.+G.+Teoticoauthor=K.+Babaogluauthor=G.+J.+Rocklinauthor=R.+S.+Ferreiraauthor=A.+M.+Giannettiauthor=B.+K.+Shoichet&title=Docking+for+fragment+inhibitors+of+AmpC+beta-lactamase"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Docking for fragment inhibitors of AmpC 83-lactamase</span></div><div class="casAuthors">Teotico, Denise G.; Babaoglu, Kerim; Rocklin, Gabriel J.; Ferreira, Rafaela S.; Giannetti, Anthony M.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">7455-7460</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Fragment screens for new ligands have had wide success, not withstanding their constraint to libraries of 1000-10,000 mols.  Larger libraries would be addressable were mol. docking reliable for fragment screens, but this has not been widely accepted.  To investigate docking's ability to prioritize fragments, a library of >137,000 such mols. were docked against the structure of β-lactamase.  Forty-eight fragments highly ranked by docking were acquired and tested; 23 had Ki values ranging from 0.7 to 9.2 mM.  X-ray crystal structures of the enzyme-bound complexes were detd. for 8 of the fragments.  For 4, the correspondence between the predicted and exptl. structures was high (RMSD between 1.2 and 1.4 Å), whereas for another 2, the fidelity was lower but retained most key interactions (RMSD 2.4-2.6 Å).  Two of the 8 fragments adopted very different poses in the active site owing to enzyme conformational changes.  The 48% hit rate of the fragment docking compares very favorably with "lead-like" docking and high-throughput screening against the same enzyme.  To understand this, we investigated the occurrence of the fragment scaffolds among larger, lead-like mols.  Approx. 1% of com. available fragments contain these inhibitors whereas only 10-7% of lead-like mols. do.  This suggests that many more chemotypes and combinations of chemotypes are present among fragments than are available among lead-like mols., contributing to the higher hit rates.  The ability of docking to prioritize these fragments suggests that the technique can be used to exploit the better chemotype coverage that exists at the fragment level.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprpafzXv4ydrVg90H21EOLACvtfcHk0lhULCWXHWhn7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXmt1Khtrg%253D&md5=d2df01432d5381c226e8aa7e008457b5</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0813029106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0813029106%26sid%3Dliteratum%253Aachs%26aulast%3DTeotico%26aufirst%3DD.%2BG.%26aulast%3DBabaoglu%26aufirst%3DK.%26aulast%3DRocklin%26aufirst%3DG.%2BJ.%26aulast%3DFerreira%26aufirst%3DR.%2BS.%26aulast%3DGiannetti%26aufirst%3DA.%2BM.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DDocking%2520for%2520fragment%2520inhibitors%2520of%2520AmpC%2520beta-lactamase%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2009%26volume%3D106%26spage%3D7455%26epage%3D7460" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group">Tondi, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morandi, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bonnet, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Costi, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Structure-based optimization of a non-beta-lactam lead results in inhibitors that do not up-regulate beta-lactamase expression in cell culture</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">127</span><span class="NLM_x">, </span> <span class="NLM_fpage">4632</span><span class="NLM_x">–</span> <span class="NLM_lpage">4639</span></span><div class="citationLinks">[<a href="/doi/10.1021/ja042984o" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=127&publication_year=2005&pages=4632-4639&author=D.+Tondiauthor=F.+Morandiauthor=R.+Bonnetauthor=M.+P.+Costiauthor=B.+K.+Shoichet&title=Structure-based+optimization+of+a+non-beta-lactam+lead+results+in+inhibitors+that+do+not+up-regulate+beta-lactamase+expression+in+cell+culture"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1021%2Fja042984o&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja042984o%26sid%3Dliteratum%253Aachs%26aulast%3DTondi%26aufirst%3DD.%26aulast%3DMorandi%26aufirst%3DF.%26aulast%3DBonnet%26aufirst%3DR.%26aulast%3DCosti%26aufirst%3DM.%2BP.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DStructure-based%2520optimization%2520of%2520a%2520non-beta-lactam%2520lead%2520results%2520in%2520inhibitors%2520that%2520do%2520not%2520up-regulate%2520beta-lactamase%2520expression%2520in%2520cell%2520culture%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2005%26volume%3D127%26spage%3D4632%26epage%3D4639" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group">Graves, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brenk, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Decoys for docking</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">3714</span><span class="NLM_x">–</span> <span class="NLM_lpage">3728</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm0491187" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Fjm300687e&amp;key=1%3ACAS%3A528%3ADC%252BD2MXjvFKjsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=3714-3728&author=A.+P.+Gravesauthor=R.+Brenkauthor=B.+K.+Shoichet&title=Decoys+for+docking"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Decoys for Docking</span></div><div class="casAuthors">Graves, Alan P.; Brenk, Ruth; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">3714-3728</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mol. docking is widely used to predict novel lead compds. for drug discovery.  Success depends on the quality of the docking scoring function, among other factors.  An imperfect scoring function can mislead by predicting incorrect ligand geometries or by selecting nonbinding mols. over true ligands.  These false-pos. hits may be considered "decoys".  Although these decoys are frustrating, they potentially provide important tests for a docking algorithm; the more subtle the decoy, the more rigorous the test.  Indeed, decoy databases have been used to improve protein structure prediction algorithms and protein-protein docking algorithms.  Here, we describe 20 geometric decoys in five enzymes and 166 "hit list" decoys-i.e., mols. predicted to bind by our docking program that were tested and found not to do so - for β-lactamase and two cavity sites in lysozyme.  Esp. in the cavity sites, which are very simple, these decoys highlight particular weaknesses in our scoring function.  We also consider the performance of five other widely used docking scoring functions against our geometric and hit list decoys.  Intriguingly, whereas many of these other scoring functions performed better on the geometric decoys, they typically performed worse on the hit list decoys, often highly ranking mols. that seemed to poorly complement the model sites.  Several of these "hits" from the other scoring functions were tested exptl. and found, in fact, to be decoys.  Collectively, these decoys provide a tool for the development and improvement of mol. docking scoring functions.  Such improvements may, in turn, be rapidly tested exptl. against these and related exptl. systems, which are well-behaved in assays and for structure detn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpklqU6W2ulh7Vg90H21EOLACvtfcHk0ljbdxjkIYnSfw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXjvFKjsbs%253D&md5=1797b501c1777dc94553701d45c822f0</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1021%2Fjm0491187&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0491187%26sid%3Dliteratum%253Aachs%26aulast%3DGraves%26aufirst%3DA.%2BP.%26aulast%3DBrenk%26aufirst%3DR.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DDecoys%2520for%2520docking%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D3714%26epage%3D3728" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group">Hawkins, P. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Skillman, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Warren, G. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ellingson, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stahl, M. T.</span><span> </span><span class="NLM_article-title">Conformer generation with OMEGA: algorithm and validation using high quality structures from the Protein Data Bank and Cambridge Structural Database</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">572</span><span class="NLM_x">–</span> <span class="NLM_lpage">584</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci100031x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Fjm300687e&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjtlaisrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=572-584&author=P.+C.+Hawkinsauthor=A.+G.+Skillmanauthor=G.+L.+Warrenauthor=B.+A.+Ellingsonauthor=M.+T.+Stahl&title=Conformer+generation+with+OMEGA%3A+algorithm+and+validation+using+high+quality+structures+from+the+Protein+Data+Bank+and+Cambridge+Structural+Database"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Conformer Generation with OMEGA: Algorithm and Validation Using High Quality Structures from the Protein Databank and Cambridge Structural Database</span></div><div class="casAuthors">Hawkins, Paul C. D.; Skillman, A. Geoffrey; Warren, Gregory L.; Ellingson, Benjamin A.; Stahl, Matthew T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">572-584</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Here, we present the algorithm and validation for OMEGA, a systematic, knowledge-based conformer generator.  The algorithm consists of three phases: assembly of an initial 3D structure from a library of fragments; exhaustive enumeration of all rotatable torsions using values drawn from a knowledge-based list of angles, thereby generating a large set of conformations; and sampling of this set by geometric and energy criteria.  Validation of conformer generators like OMEGA has often been undertaken by comparing computed conformer sets to exptl. mol. conformations from crystallog., usually from the Protein Databank (PDB).  Such an approach is fraught with difficulty due to the systematic problems with small mol. structures in the PDB.  Methods are presented to identify a diverse set of small mol. structures from cocomplexes in the PDB that has maximal reliability.  A challenging set of 197 high quality, carefully selected ligand structures from well-solved models was obtained using these methods.  This set will provide a sound basis for comparison and validation of conformer generators in the future.  Validation results from this set are compared to the results using structures of a set of druglike mols. extd. from the Cambridge Structural Database (CSD).  OMEGA is found to perform very well in reproducing the crystallog. conformations from both these data sets using two complementary metrics of success.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqq1SOwPH1NzrVg90H21EOLACvtfcHk0ljbdxjkIYnSfw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjtlaisrY%253D&md5=fb87ecc9c51eddef63b41fffcd9babee</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1021%2Fci100031x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci100031x%26sid%3Dliteratum%253Aachs%26aulast%3DHawkins%26aufirst%3DP.%2BC.%26aulast%3DSkillman%26aufirst%3DA.%2BG.%26aulast%3DWarren%26aufirst%3DG.%2BL.%26aulast%3DEllingson%26aufirst%3DB.%2BA.%26aulast%3DStahl%26aufirst%3DM.%2BT.%26atitle%3DConformer%2520generation%2520with%2520OMEGA%253A%2520algorithm%2520and%2520validation%2520using%2520high%2520quality%2520structures%2520from%2520the%2520Protein%2520Data%2520Bank%2520and%2520Cambridge%2520Structural%2520Database%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D572%26epage%3D584" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group">Jain, A. N.</span><span> </span><span class="NLM_article-title">Bias, reporting, and sharing: computational evaluations of docking methods</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">201</span><span class="NLM_x">–</span> <span class="NLM_lpage">212</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2008&pages=201-212&author=A.+N.+Jain&title=Bias%2C+reporting%2C+and+sharing%3A+computational+evaluations+of+docking+methods"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DJain%26aufirst%3DA.%2BN.%26atitle%3DBias%252C%2520reporting%252C%2520and%2520sharing%253A%2520computational%2520evaluations%2520of%2520docking%2520methods%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2008%26volume%3D22%26spage%3D201%26epage%3D212" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1R9O" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1R9O','PDB','1R9O'); return false;">PDB: 1R9O</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3NXU" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3NXU','PDB','3NXU'); return false;">PDB: 3NXU</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3EML" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3EML','PDB','3EML'); return false;">PDB: 3EML</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2VT4" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2VT4','PDB','2VT4'); return false;">PDB: 2VT4</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ODU" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ODU','PDB','3ODU'); return false;">PDB: 3ODU</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3KGC" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3KGC','PDB','3KGC'); return false;">PDB: 3KGC</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1VSO" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1VSO','PDB','1VSO'); return false;">PDB: 1VSO</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3CQW" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3CQW','PDB','3CQW'); return false;">PDB: 3CQW</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2ZDT" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2ZDT','PDB','2ZDT'); return false;">PDB: 2ZDT</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2QD9" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2QD9','PDB','2QD9'); return false;">PDB: 2QD9</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3CJO" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3CJO','PDB','3CJO'); return false;">PDB: 3CJO</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3HL5" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3HL5','PDB','3HL5'); return false;">PDB: 3HL5</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1SJ0" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1SJ0','PDB','1SJ0'); return false;">PDB: 1SJ0</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2AA2" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2AA2','PDB','2AA2'); return false;">PDB: 2AA2</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1Q4X" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1Q4X','PDB','1Q4X'); return false;">PDB: 1Q4X</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2ZNP" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2ZNP','PDB','2ZNP'); return false;">PDB: 2ZNP</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3E37" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3E37','PDB','3E37'); return false;">PDB: 3E37</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3F07" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3F07','PDB','3F07'); return false;">PDB: 3F07</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3NF7" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3NF7','PDB','3NF7'); return false;">PDB: 3NF7</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2B8T" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2B8T','PDB','2B8T'); return false;">PDB: 2B8T</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3L3M" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3L3M','PDB','3L3M'); return false;">PDB: 3L3M</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1NJS" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1NJS','PDB','1NJS'); return false;">PDB: 1NJS</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2I78" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2I78','PDB','2I78'); return false;">PDB: 2I78</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3KL6" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3KL6','PDB','3KL6'); return false;">PDB: 3KL6</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3CHP" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3CHP','PDB','3CHP'); return false;">PDB: 3CHP</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=830C" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=830C','PDB','830C'); return false;">PDB: 830C</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1NQ0" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1NQ0','PDB','1NQ0'); return false;">PDB: 1NQ0</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3O96" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3O96','PDB','3O96'); return false;">PDB: 3O96</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1KIM" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1KIM','PDB','1KIM'); return false;">PDB: 1KIM</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i23"><a href="/doi/suppl/10.1021/jm300687e">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_78274"></div></div></div></div></div><hr /></hr><p class="last">Figure showing DUD-E workflows, while tables provide detailed target-by-target data and tab delimited text files provide the raw data. This material is available free of charge via the Internet at <a class="extLink" href="http://pubs.acs.org">http://pubs.acs.org</a>.</p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm300687e/suppl_file/jm300687e_si_001.pdf">jm300687e_si_001.pdf (298.46 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm300687e/suppl_file/jm300687e_si_002.pdf">jm300687e_si_002.pdf (289.74 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm300687e/suppl_file/jm300687e_si_003.txt">jm300687e_si_003.txt (9.45 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm300687e/suppl_file/jm300687e_si_004.txt">jm300687e_si_004.txt (23.06 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Fjm300687e&amp;pbContext=%3Barticle%3Aarticle%3A10.1021%2Fjm300687e%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2012.55.issue-14%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/jm300687e" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                5MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"67c75f42db4f3da6","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
