
<html lang="en"     class="pb-page"  data-request-id="c5024711-c14b-44ad-abfb-bc44967387f8"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;issue:issue:10.1021/jmcmar.2008.51.issue-16;website:website:acspubs;article:article:10.1021/jm8006239;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Exploiting Ordered Waters in Molecular Docking" /></meta><meta name="dc.Creator" content="Niu  Huang" /></meta><meta name="dc.Creator" content="Brian K.  Shoichet" /></meta><meta name="dc.Description" content="A current weakness in docking is the treatment of water-mediated protein−ligand interactions. We explore switching ordered water molecules “on” and “off” during docking screens of a large library. ..." /></meta><meta name="Description" content="A current weakness in docking is the treatment of water-mediated protein−ligand interactions. We explore switching ordered water molecules “on” and “off” during docking screens of a large library. ..." /></meta><meta name="dc.Publisher" content=" American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="August 5, 2008" /></meta><meta name="dc.Type" content="rapid-communication" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/jm8006239" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2008 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Fjm8006239" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Fjm8006239" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Fjm8006239" /></link>
        
    
    

<title>Exploiting Ordered Waters in Molecular Docking | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/jm8006239" /></meta><meta property="og:title" content="Exploiting Ordered Waters in Molecular Docking" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-16/jm8006239/production/images/large/jm-2008-006239_0004.jpeg" /></meta><meta property="og:description" content=" A current weakness in docking is the treatment of water-mediated protein−ligand interactions. We explore switching ordered water molecules “on” and “off” during docking screens of a large library. The method assumes additivity and scales linearly with the number of waters sampled despite the exponential growth in configurations. It is tested for ligand enrichment against 24 targets, exploring up to 256 water configurations. Water inclusion increased enrichment substantially for 12 targets, while most others were largely unaffected." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/jm8006239"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/jm8006239">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-8/abseba.2021.7.issue-8/20210809/abseba.2021.7.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-15/accacs.2021.11.issue-15/20210806/accacs.2021.11.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-31/ascecg.2021.9.issue-31/20210809/ascecg.2021.9.issue-31.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-8/bomaf6.2021.22.issue-8/20210809/bomaf6.2021.22.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-29/cgeabj.2021.99.issue-29/20210809-01/cgeabj.2021.99.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-15/enfuem.2021.35.issue-15/20210805/enfuem.2021.35.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-15/joceah.2021.86.issue-15/20210806/joceah.2021.86.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-30/jpcafh.2021.125.issue-30/20210805/jpcafh.2021.125.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-30/jpcbfk.2021.125.issue-30/20210805/jpcbfk.2021.125.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-30/jpccck.2021.125.issue-30/20210805/jpccck.2021.125.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-30/jpclcd.2021.12.issue-30/20210805/jpclcd.2021.12.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-8/jprobs.2021.20.issue-8/20210806/jprobs.2021.20.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-15/orlef7.2021.23.issue-15/20210806/orlef7.2021.23.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-15/orgnd7.2021.40.issue-15/20210809/orgnd7.2021.40.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Fjm8006239&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Fjm8006239&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/jm8006239&amp;href=/doi/10.1021/jm8006239" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2008</span><span class="cit-fg-volume">, 51</span><span class="cit-fg-issue">, 16</span><span class="cit-fg-pageRange">, 4862-4865</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/51/16" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/jm800546t" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Letter</span><a href="/doi/10.1021/jm800634k" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Exploiting Ordered Waters in Molecular Docking</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Niu++Huang">Niu Huang</a></span></span></li><span class="comma-separator"> and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Brian+K.++Shoichet">Brian K. Shoichet</a></span><span class="author-xref-symbol "><sup>*</sup></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info"><span class="aff-text">Department of Pharmaceutical Chemistry, University of California—San Francisco, 1700 Fourth Street, San Francisco, California 94158-2550</span></div><div class="corresp-info"><strong>*</strong> To whom correspondence should be addressed. Phone: 1-415-514-4126. Fax: 1-415-514-4260. E-mail: <a href="/cdn-cgi/l/email-protection#05766d6a6c666d6071456662692b706676632b606170"><span class="__cf_email__" data-cfemail="eb9883848288838e9fab888c87c59e88988dc58e8f9e">[email protected]</span></a></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Fjm8006239&amp;href=/doi/10.1021%2Fjm8006239" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2008</span></span><span class="cit-volume">, 51</span><span class="cit-issue">, 16</span><span class="cit-pageRange">, 4862–4865</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">August 5, 2008</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>26 May 2008</li><li><span class="item_label"><b>Published</b> online</span>5 August 2008</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 1 August 2008</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/jm8006239" title="DOI URL">https://doi.org/10.1021/jm8006239</a></div><div class="article_header-article-copyright"><strong>Copyright © 2008 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D4862%26pageCount%3D4%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DNiu%2BHuang%252C%2BBrian%2BK.%2BShoichet%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D51%26issueNum%3D16%26contentID%3Djm8006239%26title%3DExploiting%2BOrdered%2BWaters%2Bin%2BMolecular%2BDocking%26numPages%3D4%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D4865%26publicationDate%3DAugust%2B2008">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/jm8006239"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1790</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">92</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/jm8006239" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Exploiting Ordered Waters in Molecular Docking&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Niu&quot;,&quot;last_name&quot;:&quot;Huang&quot;},{&quot;first_name&quot;:&quot;Brian&quot;,&quot;last_name&quot;:&quot;K. Shoichet&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2008&quot;,&quot;month&quot;:&quot;08&quot;,&quot;day&quot;:&quot;05&quot;,&quot;issue&quot;:&quot;16&quot;,&quot;volume&quot;:&quot;51&quot;,&quot;pages&quot;:&quot;4862-4865&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/jm8006239&quot;},&quot;abstract&quot;:&quot;A current weakness in docking is the treatment of water-mediated protein−ligand interactions. We explore switching ordered water molecules “on” and “off” during docking screens of a large library. The method assumes additivity and scales linearly with the number of waters sampled despite the exponential growth in configurations. It is tested for ligand enrichment against 24 targets, exploring up to 256 water configurations. Water inclusion increased enrichment substantially for 12 targets, while most others were largely unaffected.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Fjm8006239&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm8006239" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm8006239&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=jm8006239" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm8006239&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=jm8006239" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Fjm8006239&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm8006239&amp;href=/doi/10.1021/jm8006239" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/jm8006239" /></input><a href="/doi/pdf/10.1021/jm8006239" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (1 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/jm8006239&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Fjm8006239%26sid%3Dliteratum%253Aachs%26pmid%3D18680357%26genre%3Darticle%26aulast%3DHuang%26date%3D2008%26atitle%3DExploiting%2BOrdered%2BWaters%2Bin%2BMolecular%2BDocking%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D51%26issue%3D16%26spage%3D4862%26epage%3D4865%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (1)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=292366" title="Ligands">Ligands</a>,</li><li><a href="/action/doSearch?ConceptID=291754" title="Molecules">Molecules</a>,</li><li><a href="/action/doSearch?ConceptID=290731" title="Biological databases">Biological databases</a>,</li><li><a href="/action/doSearch?ConceptID=291901" title="Chemical calculations">Chemical calculations</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/51/16" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-16/jm8006239/production/images/medium/jm-2008-006239_0004.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-16/jm8006239/production/images/large/jm-2008-006239_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm8006239&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">A current weakness in docking is the treatment of water-mediated protein−ligand interactions. We explore switching ordered water molecules “on” and “off” during docking screens of a large library. The method assumes additivity and scales linearly with the number of waters sampled despite the exponential growth in configurations. It is tested for ligand enrichment against 24 targets, exploring up to 256 water configurations. Water inclusion increased enrichment substantially for 12 targets, while most others were largely unaffected.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><p class="first">Ordered water molecules play an important role in protein−ligand recognition, either being displaced on ligand binding or bridging groups to stabilize the complex.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> In a recent survey of high-resolution structures, over 85% of complexes had one or more water molecules bridging protein and ligand, with an average of 3.5 per complex.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> Since the identity of the mediating waters can change from ligand to ligand and since many of the waters observed in an apo-structure are displaced by ligand binding, predicting the role of a particular ordered water molecule to ligand binding remains challenging.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a></p><p>The problem of treating ordered waters is acute in molecular docking, which relies on rapid evaluations of discrete states. These rise exponentially with the number of water molecules sampled. Also, it is rarely clear which waters should be treated as displaceable and which should be treated as fixed, despite efforts to categorize them based on environment or crystallographic observation (e.g., thermal factors).<a onclick="showRef(event, 'ref2 ref4 ref5'); return false;" href="javascript:void(0);" class="ref ref2 ref4 ref5">(2, 4, 5)</a> Rarey and colleagues used a “particle concept” to place implicit waters for docking pose prediction.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> Whereas this improved the fidelity of some ligand geometries, others deteriorated. Other approaches, including GOLD, AUTODOCK, and GLIDE, have also explored incorporating waters (implicitly or explicitly) to predict ligand−protein complexes.<a onclick="showRef(event, 'ref7 ref8 ref9 ref10 ref11 ref12 ref13'); return false;" href="javascript:void(0);" class="ref ref7 ref8 ref9 ref10 ref11 ref12 ref13">(7-13)</a></p><p>Here, we investigate the effect of sampling multiple water positions in docking database screens, which, as far as we are aware, has not been previously investigated. We use this procedure in screens of 24 targets from the DUD database, which contains annotated ligands and property-matched decoys for each target<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> (Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>). Water molecules within 5 Å of the ligand were obtained from the X-ray structures. All waters that bridged the protein and the ligand, as well as all that formed at least two hydrogen bonds with the protein−ligand complex and with the primary bridging waters, were selected; only those DUD targets that had such water-mediated contacts were included, reducing the targets from 40 to 24. The water hydrogen positions were optimized using the protein local optimization program (PLOP).<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> The resulting orientations were typically unaffected by the presence of the ligand except in two cases (HSP90 and DHFR), consistent with a recent study.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a></p><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Docking calculation times with (wat) and without (ref)<a class="ref internalNav" href="#tbl1-fn1" aria-label="a">a</a> displaceable waters</div></div></div><div class="scrollable-table-wrap"><table class="table frame_topbot"><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead valign="middle"><tr valign="top"><th align="center" valign="middle"> </th><th align="center" valign="middle" char="."> </th><th align="center" valign="middle" char="."> </th><th colspan="2" align="center" valign="middle" char=".">total time (h)</th><th align="center" valign="middle" char="."> </th></tr><tr valign="top"><th align="center" valign="middle">protein</th><th align="center" valign="middle" char=".">no. wat</th><th align="center" valign="middle" char=".">no. config</th><th align="center" valign="middle" char=".">ref</th><th align="center" valign="middle" char=".">wat</th><th align="center" valign="middle" char=".">performance factor</th></tr></thead><tbody><tr valign="top"><td align="left">CDK2</td><td align="char" char=".">7</td><td align="char" char=".">128</td><td align="char" char=".">571.7</td><td align="char" char=".">2078.9</td><td align="char" char=".">35.2</td></tr><tr valign="top"><td align="left">EGFr</td><td align="char" char=".">6</td><td align="char" char=".">64</td><td align="char" char=".">1681.4</td><td align="char" char=".">4716.2</td><td align="char" char=".">22.8</td></tr><tr valign="top"><td align="left">FGFr1</td><td align="char" char=".">3</td><td align="char" char=".">8</td><td align="char" char=".">2088.1</td><td align="char" char=".">5347.9</td><td align="char" char=".">4.8</td></tr><tr valign="top"><td align="left">HSP90</td><td align="char" char=".">6</td><td align="char" char=".">96</td><td align="char" char=".">177.4</td><td align="char" char=".">1479.3</td><td align="char" char=".">11.5</td></tr><tr valign="top"><td align="left">SRC</td><td align="char" char=".">6</td><td align="char" char=".">64</td><td align="char" char=".">3192.3</td><td align="char" char=".">9529.9</td><td align="char" char=".">21.4</td></tr><tr valign="top"><td align="left">TK</td><td align="char" char=".">5</td><td align="char" char=".">32</td><td align="char" char=".">56.5</td><td align="char" char=".">207.5</td><td align="char" char=".">8.7</td></tr><tr valign="top"><td align="left">VEGFr2</td><td align="char" char=".">6</td><td align="char" char=".">64</td><td align="char" char=".">2038.4</td><td align="char" char=".">11188.4</td><td align="char" char=".">11.7</td></tr><tr valign="top"><td align="left">Fxa</td><td align="char" char=".">5</td><td align="char" char=".">32</td><td align="char" char=".">315.0</td><td align="char" char=".">1794.5</td><td align="char" char=".">5.6</td></tr><tr valign="top"><td align="left">Thrombin</td><td align="char" char=".">5</td><td align="char" char=".">32</td><td align="char" char=".">231.5</td><td align="char" char=".">1495.4</td><td align="char" char=".">5.0</td></tr><tr valign="top"><td align="left">Trypsin</td><td align="char" char=".">5</td><td align="char" char=".">32</td><td align="char" char=".">710.1</td><td align="char" char=".">2488.5</td><td align="char" char=".">9.1</td></tr><tr valign="top"><td align="left">ACE</td><td align="char" char=".">6</td><td align="char" char=".">64</td><td align="char" char=".">119.8</td><td align="char" char=".">866.7</td><td align="char" char=".">8.8</td></tr><tr valign="top"><td align="left">ADA</td><td align="char" char=".">7</td><td align="char" char=".">128</td><td align="char" char=".">105.8</td><td align="char" char=".">885.6</td><td align="char" char=".">15.3</td></tr><tr valign="top"><td align="left">COMT</td><td align="char" char=".">2</td><td align="char" char=".">4</td><td align="char" char=".">606.7</td><td align="char" char=".">1535.9</td><td align="char" char=".">1.6</td></tr><tr valign="top"><td align="left">PDE5</td><td align="char" char=".">7</td><td align="char" char=".">128</td><td align="char" char=".">780.1</td><td align="char" char=".">3160.1</td><td align="char" char=".">31.6</td></tr><tr valign="top"><td align="left">DHFR</td><td align="char" char=".">2</td><td align="char" char=".">6</td><td align="char" char=".">960.6</td><td align="char" char=".">1750.0</td><td align="char" char=".">3.3</td></tr><tr valign="top"><td align="left">GART</td><td align="char" char=".">1</td><td align="char" char=".">2</td><td align="char" char=".">79.1</td><td align="char" char=".">140.9</td><td align="char" char=".">1.1</td></tr><tr valign="top"><td align="left">AChE</td><td align="char" char=".">8</td><td align="char" char=".">256</td><td align="char" char=".">227.0</td><td align="char" char=".">2009.4</td><td align="char" char=".">28.9</td></tr><tr valign="top"><td align="left">AmpC</td><td align="char" char=".">6</td><td align="char" char=".">216</td><td align="char" char=".">607.4</td><td align="char" char=".">4447.1</td><td align="char" char=".">29.5</td></tr><tr valign="top"><td align="left">GPB</td><td align="char" char=".">8</td><td align="char" char=".">256</td><td align="char" char=".">102.5</td><td align="char" char=".">1635.0</td><td align="char" char=".">16.1</td></tr><tr valign="top"><td align="left">HIVPR</td><td align="char" char=".">4</td><td align="char" char=".">16</td><td align="char" char=".">2049.4</td><td align="char" char=".">5001.1</td><td align="char" char=".">6.6</td></tr><tr valign="top"><td align="left">HMGR</td><td align="char" char=".">3</td><td align="char" char=".">8</td><td align="char" char=".">505.6</td><td align="char" char=".">1301.8</td><td align="char" char=".">3.1</td></tr><tr valign="top"><td align="left">NA</td><td align="char" char=".">4</td><td align="char" char=".">16</td><td align="char" char=".">275.5</td><td align="char" char=".">1302.1</td><td align="char" char=".">3.4</td></tr><tr valign="top"><td align="left">PNP</td><td align="char" char=".">3</td><td align="char" char=".">8</td><td align="char" char=".">51.3</td><td align="char" char=".">130.6</td><td align="char" char=".">3.1</td></tr><tr valign="top"><td align="left">SAHH</td><td align="char" char=".">1</td><td align="char" char=".">2</td><td align="char" char=".">28.7</td><td align="char" char=".">37.6</td><td align="char" char=".">1.5</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-tbl1-fn1"><div class="footnote" id="tbl1-fn1"><sup>a</sup><p class="last">Total time reported for 1 CPU in hours. For each target, the entire DUD set of over 97 000 molecules was docked.</p></div></div></div><p>To sample multiple waters positions, we adapted a flexible-receptor docking method that treats multiple flexible regions of the protein independently, recombining them to generate discrete conformations.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> Because the method assumes additivity among the regions, it scales linearly rather than exponentially with degrees of freedom. This is, of course, an approximation. We modified the approach to treat individual water molecules as the flexible receptor regions. Each water is represented in the “off” state (displaced) or one of several “on” states (retained, there can be more than one configuration for a retained water molecule), and all waters are treated as equally displaceable; we did not consider the differential energy of water binding. This is a second approximation.</p><p>For each water molecule a separate electrostatic and van der Waals potential map is calculated. Every docked molecule is scored against each individual water potential grid, as well as an overall grid representing the rest of the invariant protein. For every water molecule the “off” state or one of the “on” states is chosen on the basis of whether the overall interaction of the protein and waters with the ligand is improved. The optimal water configuration for any docked molecule is assembled from the best state for each water, and the score of the docked molecule is summed from the ligand−protein and ligand−water interactions. For instance, AChE has eight water molecules that were sampled in the docking calculation (Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>). In their best orientations, ligand 1 interacts with four waters but displaces four others; ligand 2 chooses five waters displacing three others, while ligand 3 keeps only three waters and displaces the other five.</p><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-16/jm8006239/production/images/medium/jm-2008-006239_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-16/jm8006239/production/images/large/jm-2008-006239_0001.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Docking poses of representative ligands (carbons in green) from the library screens showing the roles of different specific water configurations. Note that we combined waters from two different crystal structures for AmpC (waters from 1xgj colored in red and from 1l2s colored in blue). Images generated with Chimera.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-16/jm8006239/production/images/large/jm-2008-006239_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm8006239&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p>We assessed the quality of the docking screen by the enrichment of annotated ligands from the vast majority of DUD molecules that are decoys. Enrichment factors measure the prioritization of ligands among the top ranks of a docking-ordered library. For instance, if 10% of the ligands are found among the top 0.1% of a docking-ranked database, the enrichment factor at this point is 100, if 20% of the ligands are found among the top 1% of the ranked database, the enrichment factor is 20. For each of the 24 proteins targeted here there were typically dozens to hundreds of annotated ligands, each with 36 decoys that resembled them physically but differed from them topologically. The overall DUD database consists of 40 targets, 2950 annotated ligands, and 95 316 decoys (for some DUD targets bridging waters were not observed, excluding them from this study). We compared docking screens against the naked protein without ordered water molecules to exactly the same screen where up to eight ordered waters were modeled.</p><p>The screens were evaluated by four criteria: How often were the enrichment factors improved by including displaceable waters? Were the docked poses sensible? Did success hinge on the inclusion of particular waters? What was the calculation cost? Between one and eight water molecules, or 2−256 configurations, were modeled in the 24 targets. For 12 targets enrichment increased noticeably (Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>). For targets such as CDK2, the improvement was modest, with the enrichment factor at 1% and 20% of the ranked database (EF<sub>1</sub> and EF<sub>20</sub><a class="ref internalNav" href="#fn1" aria-label="a">a</a>) <div class="NLM_fn" id="fn1">a<p class="last">Abbreviations: CDK2, cyclin-dependent kinase 2; EGFr, epidermal growth factor receptor; FGFr1, fibroblast growth factor receptor kinase; HSP90, human heat shock protein 90; SRC, tyrosine kinase SRC; TK, thymidine kinase; VEGFr2, vascular endothelial growth factor receptor; FXa, factor Xa; ACE, angiotensin-converting enzyme; ADA, adenosine deaminase; COMT, catechol <i>O</i>-methyltransferase; PDE5, phosphodiesterase 5; DHFR, dihydrofolate reductase; GART, glycinamide ribonucleotide transformylase; AChE, acetylcholinesterase; AmpC, AmpC β-lactamase; GPB, glycogen phosphorylase β; HIVPR, HIV protease; HMGR, hydroxymethylglutaryl-CoA reductase; NA, neuraminidase; PNP, purine nucleoside phosphorylase; SAHH, <i>S</i>-adenosylhomocysteine; EF<sub>1</sub> and EF<sub>20</sub>, enrichment factors at 1% and 20%.</p></div> increasing from an EF<sub>1</sub> of 0 (no ligands found at all) without ordered waters to 2.0 when they were included and a corresponding increase at EF<sub>20</sub> of 0.4−2.2 (<a class="ref internalNav" href="#si1" aria-label="Table S1, Supporting Information">Table S1, Supporting Information</a>). For targets like COMT, the improvements were more substantial, from an EF<sub>1</sub> of 8.2 without ordered water molecules to one of 41.2 when waters were modeled. For 11 of the 24 targets enrichment was largely unaffected; for 7 of these, however, enrichment factors were already high even without the waters, leaving little room for improvement. For one target, VEGFr, enrichment was slightly diminished by the inclusion of displaceable water molecules. Ligand enrichment was matched by docked ligand geometries that corresponded well to the crystallographic result (Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>).</p><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-16/jm8006239/production/images/medium/jm-2008-006239_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-16/jm8006239/production/images/large/jm-2008-006239_0002.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Docking enrichment plots for 24 protein targets. The docking ranked database (<i>x</i>-axis) vs the percentage of ligands found (<i>y</i>-axis) at any percentage of the ranking. The gray line is random selection; the black line is docking enrichment neglecting the binding site waters, and the red line is the enrichment with displaceable waters. Targets are listed in same order as in Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-16/jm8006239/production/images/large/jm-2008-006239_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm8006239&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p>Are the increased enrichments due to biased choice of waters? Although most were picked from those mediating specific ligand−protein interactions, several considerations moderate this bias. First, the enrichment calculations consider tens to hundreds of ligand molecules, not simply the one observed crystallographically. Second, for four targets, factor Xa, trypsin, AmpC, and neuraminidase, we purposely included apo structure water molecules that are displaced on ligand binding (<a class="ref internalNav" href="#si1" aria-label="Figure S1, Supporting Information">Figure S1, Supporting Information</a>). The enrichment curves were undiminished relative to using waters only from the holo complex. Third, we repeated the docking screens with the formerly displaceable waters fixed in the site. This diminished enrichment substantially for 15 targets, left it unaffected for 8, and improved for only 1 (<a class="ref internalNav" href="#si1" aria-label="Figure S2, Supporting Information">Figure S2, Supporting Information</a>). Finally, for AmpC water molecules from two different structures were combined, still resulting in improved enrichment.</p><p>The improved enrichment factors were accompanied by a large increase in the degrees of freedom of the system: from a single naked site to one with up to 256 water configurations. If calculation time increased correspondingly, the approach would be impractical. However, treating each water molecule independently led to run times that scaled linearly with the number of water molecules, not as their exponent. For instance, for HIV protease and neuraminidase (4 water molecules, 16 configurations) the database screen took 2.5- and 4.7-fold longer than the screen against the naked receptor alone (2049 and 275 CPU hours vs 5001 and 1302 with the waters included). For AChE and GPB (8 water molecules, 256 configurations) the calculation time was 8.9- and 15.9-fold longer than screening the naked receptor (Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>). Admittedly, neither the linear increase in run time nor the memory allotted to the scoring grids can be extended indefinitely, but for now both are within reason for modern computational systems.</p><p>In principle, one can imagine how the approximations in this approach might doom it. The assumption that each water behaves independently can never be wholly true. Still, if the waters are sufficiently far from each other, their cooperativity may be modest.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> More dubious is the assumption that the displacement energy of every water is identical. Quantifying this differential displacement cost should further improve the screening results.</p><p>More generally, increasing degrees of freedom in docking, for instance by sampling alternative protein conformations or relaxing the protein, can diminish docking quality unless they are carefully balanced with internal energy considerations. Here, although sampling ordered waters improves the docking scores for ligands and decoys, the improvement is substantially better for the ligands (Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>). Since the waters are displaceable and different waters are exploited by different ligands (Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>), this cannot be explained by the trivial exclusion of all but the true ligands by the mere presence of the waters. Intriguingly, whereas several ligand water configurations are exploited, they are dominated by a few (10 of a possible 216 in the case of AmpC β-lactamase, for instance, Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>). For decoys the water configurations are more evenly distributed among the possible configurations (Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>).</p><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-16/jm8006239/production/images/medium/jm-2008-006239_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-16/jm8006239/production/images/large/jm-2008-006239_0003.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Docking energies for ligands and decoys with and without displaceable waters for four targets. (Left) The distribution of docking energies (<i>x</i>-axis) and ligand and decoy ranks expressed as a normalized probability (<i>y</i>-axis): ligands (solid line) and decoys (dotted line) with (red) and without (black) displaceable waters. (Right) The presence of particular water configurations for high-scoring ligand (red) and decoy (blue) geometries, numbered sequentially.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-16/jm8006239/production/images/large/jm-2008-006239_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm8006239&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="last">If there is no first principle reason why this method <i>should</i> work, given that it ignores the internal energy differences among the different ordered water structures, the observation that it does work for many targets, and rarely degrades performance for others, may be widely exploited. There is little in the approach that is specific to the docking program used here (DOCK3.5.54), and it may be straightforward to implement in other platforms. Modeling ordered, displaceable water molecules may amplify the specificity for ligands implicit in the structure of the protein without greatly affecting decoy molecules.</p><div class="NLM_back"><div class="article_supporting-info" id="si1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3"><a href="/doi/suppl/10.1021/jm8006239">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_03123" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_03123" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The affect of modeling waters displaced on ligand binding, the dependence of run time, and enrichment on the number of waters modeled. This material is available free of charge via the Internet at <a href="http://pubs.acs.org" class="extLink">http://pubs.acs.org</a>. Protein structures and water positions are available from <a href="http://dud.docking.org/jmc-water" class="extLink">http://dud.docking.org/jmc-water</a>.</p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm8006239/suppl_file/jm8006239-file002.pdf">jm8006239-file002.pdf (538.82 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/jm8006239" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_66880" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_66880" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Brian K. Shoichet</span> - <span class="hlFld-Affiliation affiliation">Department of Pharmaceutical Chemistry, University of California—San Francisco, 1700 Fourth Street, San Francisco, California 94158-2550</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#95e6fdfafcf6fdf0e1d5f6f2f9bbe0f6e6f3bbf0f1e0"><span class="__cf_email__" data-cfemail="4c3f2423252f2429380c2f2b2062392f3f2a62292839">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Niu Huang</span> - <span class="hlFld-Affiliation affiliation">Department of Pharmaceutical Chemistry, University of California—San Francisco, 1700 Fourth Street, San Francisco, California 94158-2550</span></div></li></ul></li><li></li><li></li><li></li></ul></div><div class="ack"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_12431" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_12431" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This work was supported by NIH Grant GM59957. We thank Binqing Wei for algorithmic advice and Sarah Boyce and Michael Mysinger for reading this manuscript.</p></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i4">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_12302" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_12302" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 17 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Barillari, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Viner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Essex, J. W.</span><span> </span><span class="NLM_article-title">Classification of water molecules in protein binding sites</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_year" style="font-weight: bold;">2007</span> <span class="NLM_volume">129</span> <span class="NLM_fpage">2577</span> <span class="NLM_lpage">2587</span></span><div class="citationLinks">[<a href="/doi/10.1021/ja066980q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Fjm8006239&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhsFGjt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=129&publication_year=2007&pages=2577-2587&author=C.+Barillariauthor=J.+Taylorauthor=R.+Vinerauthor=J.+W.+Essex&title=Classification+of+water+molecules+in+protein+binding+sites"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Classification of Water Molecules in Protein Binding Sites</span></div><div class="casAuthors">Barillari, Caterina; Taylor, Justine; Viner, Russell; Essex, Jonathan W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">129</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2577-2587</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Water mols. play a crucial role in mediating the interaction between a ligand and a macromol. receptor.  An understanding of the nature and role of each water mol. in the active site of a protein could greatly increase the efficiency of rational drug design approaches: if the propensity of a water mol. for displacement can be detd., then synthetic effort may be most profitably applied to the design of specific ligands with the displacement of this water mol. in mind.  In this paper, a thermodn. anal. of water mols. in the binding sites of six proteins, each complexed with a no. of inhibitors, is presented.  Two classes of water mols. were identified: those conserved and not displaced by any of the ligands, and those that are displaced by some ligands.  The abs. binding free energies of 54 water mols. were calcd. using the double decoupling method, with replica exchange thermodn. integration in Monte Carlo simulations.  It was found that conserved water mols. are on av. more tightly bound than displaced water mols.  In addn., Bayesian statistics is used to calc. the probability that a particular water mol. may be displaced by an appropriately designed ligand, given the calcd. binding free energy of the water mol.  This approach therefore allows the numerical assessment of whether or not a given water mol. should be targeted for displacement as part of a rational drug design strategy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr38ETxUkD-NrVg90H21EOLACvtfcHk0lh2qwBo2CB2ew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhsFGjt7o%253D&md5=e74664f19993a9e9c087dbb9c1d54c3e</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1021%2Fja066980q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja066980q%26sid%3Dliteratum%253Aachs%26aulast%3DBarillari%26aufirst%3DC.%26aulast%3DTaylor%26aufirst%3DJ.%26aulast%3DViner%26aufirst%3DR.%26aulast%3DEssex%26aufirst%3DJ.%2BW.%26atitle%3DClassification%2520of%2520water%2520molecules%2520in%2520protein%2520binding%2520sites%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2007%26volume%3D129%26spage%3D2577%26epage%3D2587" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Lu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, C. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, S.</span><span> </span><span class="NLM_article-title">Analysis of ligand-bound water molecules in high-resolution crystal structures of protein−ligand complexes</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_year" style="font-weight: bold;">2007</span> <span class="NLM_volume">47</span> <span class="NLM_fpage">668</span> <span class="NLM_lpage">675</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci6003527" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Fjm8006239&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFalsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2007&pages=668-675&author=Y.+Luauthor=R.+Wangauthor=C.+Y.+Yangauthor=S.+Wang&title=Analysis+of+ligand-bound+water+molecules+in+high-resolution+crystal+structures+of+protein%E2%88%92ligand+complexes"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Analysis of Ligand-Bound Water Molecules in High-Resolution Crystal Structures of Protein-Ligand Complexes</span></div><div class="casAuthors">Lu, Yipin; Wang, Renxiao; Yang, Chao-Yie; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">668-675</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We have performed a comprehensive anal. of water mols. at the protein-ligand interfaces obsd. in 392 high-resoln. crystal structures.  There are a total of 1829 ligand-bound water mols. in these 392 complexes; 18% are surface water mols., and 72% are interfacial water mols.  The no. of ligand-bound water mols. in each complex structure ranges from 0 to 21 and has an av. of 4.6.  Of these interfacial water mols., 76% are considered to be bridging water mols., characterized by having polar interactions with both ligand and protein atoms.  Among a no. of factors that may influence the no. of ligand-bound water mols., the polar van der Waals (vdw) surface area of ligands has the highest Pearson linear correlation coeff. of 0.63.  Our regression anal. predicted that one more ligand-bound water mol. is expected for every addnl. 24 Å2 in the polar vdw surface area of the ligand.  In contrast to the observation that the resoln. is the primary factor influencing the no. of water mols. in crystallog. models of proteins, we found that there is only a weak relationship between the no. of ligand-bound water mols. and the resoln. of the crystal structures.  An anal. of the isotropic B factors of buried ligand-bound water mols. suggested that, when water mols. have fewer than two polar interactions with the protein-ligand complex, they are more mobile than protein atoms in the crystal structures; when they have more than three polar interactions, they are significantly less mobile than protein atoms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2Ryn_w53w0rVg90H21EOLACvtfcHk0lh2qwBo2CB2ew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFalsbg%253D&md5=4d46f344e82d0f4fce26cb20b6e28010</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1021%2Fci6003527&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci6003527%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DR.%26aulast%3DYang%26aufirst%3DC.%2BY.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DAnalysis%2520of%2520ligand-bound%2520water%2520molecules%2520in%2520high-resolution%2520crystal%2520structures%2520of%2520protein%25E2%2588%2592ligand%2520complexes%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2007%26volume%3D47%26spage%3D668%26epage%3D675" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Mancera, R. L.</span><span> </span><span class="NLM_article-title">Molecular modeling of hydration in drug design</span> <span class="citation_source-journal">Curr. Opin. Drug Discovery Dev.</span><span class="NLM_year" style="font-weight: bold;">2007</span> <span class="NLM_volume">10</span> <span class="NLM_fpage">275</span> <span class="NLM_lpage">280</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Fjm8006239&amp;key=17554853" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Fjm8006239&amp;key=1%3ACAS%3A528%3ADC%252BD2sXmtVWlsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2007&pages=275-280&author=R.+L.+Mancera&title=Molecular+modeling+of+hydration+in+drug+design"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular modeling of hydration in drug design</span></div><div class="casAuthors">Mancera, Ricardo L.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Drug Discovery & Development</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">275-280</span>CODEN:
                <span class="NLM_cas:coden">CODDFF</span>;
        ISSN:<span class="NLM_cas:issn">1367-6733</span>.
    
            (<span class="NLM_cas:orgname">Thomson Scientific</span>)
        </div><div class="casAbstract">A review.  The accurate prediction of the binding mode and affinity of a drug mol. to its putative receptor is crucial for successful drug design and optimization.  Modeling the role of the aq. solvent in ligand-protein interactions is one of the key components.  Three main computational techniques have been developed to model hydration: free energy methods, ligand-protein docking and scoring, and the explicit inclusion of tightly bound water mols. in modeling.  This review provides a summary of the most recent developments in these approaches used for the mol. modeling of hydration effects in the context of computational structure-based drug design methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5oaWakeqXu7Vg90H21EOLACvtfcHk0lhxWg8N3y67Uw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXmtVWlsrg%253D&md5=00f8343ef349b19271d7199dd048d215</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMancera%26aufirst%3DR.%2BL.%26atitle%3DMolecular%2520modeling%2520of%2520hydration%2520in%2520drug%2520design%26jtitle%3DCurr.%2520Opin.%2520Drug%2520Discovery%2520Dev.%26date%3D2007%26volume%3D10%26spage%3D275%26epage%3D280" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Fitzpatrick, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Steinmetz, A. C. U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ringe, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klibanov, A. M.</span><span> </span><span class="NLM_article-title">Enzyme crystal structure in a neat organic solvent</span> <span class="citation_source-journal">Proc. Nat. Acad. Sci. U.S.A.</span><span class="NLM_year" style="font-weight: bold;">1993</span> <span class="NLM_volume">90</span> <span class="NLM_fpage">8653</span> <span class="NLM_lpage">8657</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Fjm8006239&amp;key=10.1073%2Fpnas.90.18.8653" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Fjm8006239&amp;key=8378343" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Fjm8006239&amp;key=1%3ACAS%3A528%3ADyaK2cXjsV2m" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=90&publication_year=1993&pages=8653-8657&author=P.+A.+Fitzpatrickauthor=A.+C.+U.+Steinmetzauthor=D.+Ringeauthor=A.+M.+Klibanov&title=Enzyme+crystal+structure+in+a+neat+organic+solvent"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Enzyme crystal structure in a neat organic solvent</span></div><div class="casAuthors">Fitzpatrick, Paul A.; Steinmetz, Anke C. U.; Ringe, Dagmar; Klibanov, Alexander M.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">90</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">8653-7</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    </div><div class="casAbstract">The crystal structure of the serine protease subtilisin Carlsberg in anhyd. acetonitrile was detd. at 2.3 Å resoln.  It was found to be essentially identical to the three-dimensional structure of the enzyme is water; the differences obsd. were smaller than those between two independently detd. structures in aq. soln.  The hydrogen bond system of the catalytic triad is intact in acetonitrile.  The majority (99 of 119) of enzyme-bound, structural water mols. have such a great affinity to subtilisin that they are not displaced even in anhyd. acetonitrile.  Of the 12 enzyme-bound acetonitrile mols., 4 displace water mols. and 8 bind where no water had been obsd. before.  One-third of all subtilisin-bound acetonitrile mols. reside in the active center, occupying the same region (P1, P2, and P3 binding sites) as the specific protein inhibitor eglin c.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbO1HQQt_shbVg90H21EOLACvtfcHk0lhxWg8N3y67Uw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXjsV2m&md5=80c75820f758722bd231cab982b63a1f</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1073%2Fpnas.90.18.8653&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.90.18.8653%26sid%3Dliteratum%253Aachs%26aulast%3DFitzpatrick%26aufirst%3DP.%2BA.%26aulast%3DSteinmetz%26aufirst%3DA.%2BC.%2BU.%26aulast%3DRinge%26aufirst%3DD.%26aulast%3DKlibanov%26aufirst%3DA.%2BM.%26atitle%3DEnzyme%2520crystal%2520structure%2520in%2520a%2520neat%2520organic%2520solvent%26jtitle%3DProc.%2520Nat.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D1993%26volume%3D90%26spage%3D8653%26epage%3D8657" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Raymer, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sanschagrin, P. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Punch, W. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Venkataraman, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goodman, E. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuhn, L. A.</span><span> </span><span class="NLM_article-title">Predicting conserved water-mediated and polar ligand interactions in proteins using a K-nearest-neighbors genetic algorithm</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_year" style="font-weight: bold;">1997</span> <span class="NLM_volume">265</span> <span class="NLM_fpage">445</span> <span class="NLM_lpage">464</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Fjm8006239&amp;key=10.1006%2Fjmbi.1996.0746" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Fjm8006239&amp;key=9034363" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Fjm8006239&amp;key=1%3ACAS%3A528%3ADyaK2sXht1aktrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=265&publication_year=1997&pages=445-464&author=M.+L.+Raymerauthor=P.+C.+Sanschagrinauthor=W.+F.+Punchauthor=S.+Venkataramanauthor=E.+D.+Goodmanauthor=L.+A.+Kuhn&title=Predicting+conserved+water-mediated+and+polar+ligand+interactions+in+proteins+using+a+K-nearest-neighbors+genetic+algorithm"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Predicting conserved water-mediated and polar ligand interactions in protein using a K-nearest-neighbors genetic algorithm</span></div><div class="casAuthors">Raymer, Michael L.; Sanschagrin, Paul C.; Punch, William F.; Venkataraman, Sridhar; Goodman, Erik D.; Kuhn, Leslie A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">265</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">445-464</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">Water-mediated ligand interactions are essential to biol. processes, from product displacement in thymidylate synthase to DNA recognition by Trp repressor, yet the structural chem. influencing whether bound water is displaced or participates in ligand binding is not well characterized.  Consolv, employing a hybrid k-nearest-neighbors classifier/genetic algorithm, predicts bound water mols. conserved between free and ligand-bound protein structures by examg. the environment of each water mol. in the free structure.  Four environmental features are used: the water mol.'s crystallog. temp. factor, the no. of hydrogen bonds between the water mol. and protein, and the d. and hydrophilicity of neighboring protein atoms.  After training on 13 non-homologous proteins, Consolv predicted the conservation of active-site water mols. upon ligand binding with 75% accuracy (Matthews coeff. Cm = 0.41) for seven new proteins.  Mispredictions typically involved water mols. predicted to be conserved that were displaced by a polar ligand atom, indicating that Consolv correctly assesses polar binding sites; 90% accuracy (Cm = 0.78) was achieved for predicting conserved active-site water or polar ligand atom binding.  Consolv thus provides an accurate means for optimizing ligand design by identifying sites favored to be occupied by either a mediating water mol. or a polar ligand atom, as well as water mols. likely to be displaced by the ligand.  Accuracy for predicting first-shell water conservation between independently detd. structures was 61% (Cm = 0.23).  The ability to predict water-mediated and polar interactions from the free protein structure indicates the surprising extent to which the conservation or displacement of active-site bound water is independent of the ligand, and shows that the protein micro-environment of each water mol. is the dominant influence.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpaqbnU8qfRV7Vg90H21EOLACvtfcHk0lhTdEzq7Ee-DA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXht1aktrw%253D&md5=1dfd393d67b240857c928943abb7c6b3</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1006%2Fjmbi.1996.0746&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fjmbi.1996.0746%26sid%3Dliteratum%253Aachs%26aulast%3DRaymer%26aufirst%3DM.%2BL.%26aulast%3DSanschagrin%26aufirst%3DP.%2BC.%26aulast%3DPunch%26aufirst%3DW.%2BF.%26aulast%3DVenkataraman%26aufirst%3DS.%26aulast%3DGoodman%26aufirst%3DE.%2BD.%26aulast%3DKuhn%26aufirst%3DL.%2BA.%26atitle%3DPredicting%2520conserved%2520water-mediated%2520and%2520polar%2520ligand%2520interactions%2520in%2520proteins%2520using%2520a%2520K-nearest-neighbors%2520genetic%2520algorithm%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1997%26volume%3D265%26spage%3D445%26epage%3D464" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Rarey, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kramer, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lengauer, T.</span><span> </span><span class="NLM_article-title">The particle concept: placing discrete water molecules during protein−ligand docking predictions</span> <span class="citation_source-journal">Proteins</span><span class="NLM_year" style="font-weight: bold;">1999</span> <span class="NLM_volume">34</span> <span class="NLM_fpage">17</span> <span class="NLM_lpage">28</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=1999&pages=17-28&author=M.+Rareyauthor=B.+Kramerauthor=T.+Lengauer&title=The+particle+concept%3A+placing+discrete+water+molecules+during+protein%E2%88%92ligand+docking+predictions"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DRarey%26aufirst%3DM.%26aulast%3DKramer%26aufirst%3DB.%26aulast%3DLengauer%26aufirst%3DT.%26atitle%3DThe%2520particle%2520concept%253A%2520placing%2520discrete%2520water%2520molecules%2520during%2520protein%25E2%2588%2592ligand%2520docking%2520predictions%26jtitle%3DProteins%26date%3D1999%26volume%3D34%26spage%3D17%26epage%3D28" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Minke, W. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diller, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hol, W. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verlinde, C. L.</span><span> </span><span class="NLM_article-title">The role of waters in docking strategies with incremental flexibility for carbohydrate derivatives: heat-labile enterotoxin, a multivalent test case</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_year" style="font-weight: bold;">1999</span> <span class="NLM_volume">42</span> <span class="NLM_fpage">1778</span> <span class="NLM_lpage">1788</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm980472c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=1999&pages=1778-1788&author=W.+E.+Minkeauthor=D.+J.+Dillerauthor=W.+G.+Holauthor=C.+L.+Verlinde&title=The+role+of+waters+in+docking+strategies+with+incremental+flexibility+for+carbohydrate+derivatives%3A+heat-labile+enterotoxin%2C+a+multivalent+test+case"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1021%2Fjm980472c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm980472c%26sid%3Dliteratum%253Aachs%26aulast%3DMinke%26aufirst%3DW.%2BE.%26aulast%3DDiller%26aufirst%3DD.%2BJ.%26aulast%3DHol%26aufirst%3DW.%2BG.%26aulast%3DVerlinde%26aufirst%3DC.%2BL.%26atitle%3DThe%2520role%2520of%2520waters%2520in%2520docking%2520strategies%2520with%2520incremental%2520flexibility%2520for%2520carbohydrate%2520derivatives%253A%2520heat-labile%2520enterotoxin%252C%2520a%2520multivalent%2520test%2520case%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1999%26volume%3D42%26spage%3D1778%26epage%3D1788" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Osterberg, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morris, G. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sanner, M. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olson, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goodsell, D. S.</span><span> </span><span class="NLM_article-title">Automated docking to multiple target structures: incorporation of protein mobility and structural water heterogeneity in AutoDock</span> <span class="citation_source-journal">Proteins</span><span class="NLM_year" style="font-weight: bold;">2002</span> <span class="NLM_volume">46</span> <span class="NLM_fpage">34</span> <span class="NLM_lpage">40</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2002&pages=34-40&author=F.+Osterbergauthor=G.+M.+Morrisauthor=M.+F.+Sannerauthor=A.+J.+Olsonauthor=D.+S.+Goodsell&title=Automated+docking+to+multiple+target+structures%3A+incorporation+of+protein+mobility+and+structural+water+heterogeneity+in+AutoDock"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DOsterberg%26aufirst%3DF.%26aulast%3DMorris%26aufirst%3DG.%2BM.%26aulast%3DSanner%26aufirst%3DM.%2BF.%26aulast%3DOlson%26aufirst%3DA.%2BJ.%26aulast%3DGoodsell%26aufirst%3DD.%2BS.%26atitle%3DAutomated%2520docking%2520to%2520multiple%2520target%2520structures%253A%2520incorporation%2520of%2520protein%2520mobility%2520and%2520structural%2520water%2520heterogeneity%2520in%2520AutoDock%26jtitle%3DProteins%26date%3D2002%26volume%3D46%26spage%3D34%26epage%3D40" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">de Graaf, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pospisil, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pos, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Folkers, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vermeulen, N. P.</span><span> </span><span class="NLM_article-title">Binding mode prediction of cytochrome p450 and thymidine kinase protein−ligand complexes by consideration of water and rescoring in automated docking</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_year" style="font-weight: bold;">2005</span> <span class="NLM_volume">48</span> <span class="NLM_fpage">2308</span> <span class="NLM_lpage">2318</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm049650u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=2308-2318&author=C.+de+Graafauthor=P.+Pospisilauthor=W.+Posauthor=G.+Folkersauthor=N.+P.+Vermeulen&title=Binding+mode+prediction+of+cytochrome+p450+and+thymidine+kinase+protein%E2%88%92ligand+complexes+by+consideration+of+water+and+rescoring+in+automated+docking"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Fjm049650u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm049650u%26sid%3Dliteratum%253Aachs%26aulast%3Dde%2BGraaf%26aufirst%3DC.%26aulast%3DPospisil%26aufirst%3DP.%26aulast%3DPos%26aufirst%3DW.%26aulast%3DFolkers%26aufirst%3DG.%26aulast%3DVermeulen%26aufirst%3DN.%2BP.%26atitle%3DBinding%2520mode%2520prediction%2520of%2520cytochrome%2520p450%2520and%2520thymidine%2520kinase%2520protein%25E2%2588%2592ligand%2520complexes%2520by%2520consideration%2520of%2520water%2520and%2520rescoring%2520in%2520automated%2520docking%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D2308%26epage%3D2318" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Verdonk, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chessari, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cole, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hartshorn, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nissink, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, R. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, R.</span><span> </span><span class="NLM_article-title">Modeling water molecules in protein−ligand docking using GOLD</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_year" style="font-weight: bold;">2005</span> <span class="NLM_volume">48</span> <span class="NLM_fpage">6504</span> <span class="NLM_lpage">6515</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm050543p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=6504-6515&author=M.+L.+Verdonkauthor=G.+Chessariauthor=J.+C.+Coleauthor=M.+J.+Hartshornauthor=C.+W.+Murrayauthor=J.+W.+Nissinkauthor=R.+D.+Taylorauthor=R.+Taylor&title=Modeling+water+molecules+in+protein%E2%88%92ligand+docking+using+GOLD"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Fjm050543p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm050543p%26sid%3Dliteratum%253Aachs%26aulast%3DVerdonk%26aufirst%3DM.%2BL.%26aulast%3DChessari%26aufirst%3DG.%26aulast%3DCole%26aufirst%3DJ.%2BC.%26aulast%3DHartshorn%26aufirst%3DM.%2BJ.%26aulast%3DMurray%26aufirst%3DC.%2BW.%26aulast%3DNissink%26aufirst%3DJ.%2BW.%26aulast%3DTaylor%26aufirst%3DR.%2BD.%26aulast%3DTaylor%26aufirst%3DR.%26atitle%3DModeling%2520water%2520molecules%2520in%2520protein%25E2%2588%2592ligand%2520docking%2520using%2520GOLD%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D6504%26epage%3D6515" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Corbeil, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Englebienne, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moitessier, N.</span><span> </span><span class="NLM_article-title">Docking ligands into flexible and solvated macromolecules. 1. Development and validation of FITTED 1.0</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_year" style="font-weight: bold;">2007</span> <span class="NLM_volume">47</span> <span class="NLM_fpage">435</span> <span class="NLM_lpage">449</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci6002637" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Fjm8006239&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhvVWjtbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2007&pages=435-449&author=C.+R.+Corbeilauthor=P.+Englebienneauthor=N.+Moitessier&title=Docking+ligands+into+flexible+and+solvated+macromolecules.+1.+Development+and+validation+of+FITTED+1.0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Docking Ligands into Flexible and Solvated Macromolecules. 1. Development and Validation of FITTED 1.0</span></div><div class="casAuthors">Corbeil, Christopher R.; Englebienne, Pablo; Moitessier, Nicolas</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">435-449</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The authors report the development and validation of a novel suite of programs, FITTED 1.0, for the docking of flexible ligands into flexible proteins.  This docking tool is unique in that it can deal with both the flexibility of macromols. (side chains and main chains) and the presence of bridging water mols. while treating protein/ligand complexes as realistically dynamic systems.  This software relies on a genetic algorithm to account for the flexibility of the two mols. as well as the location of bridging water mols.  In addn., FITTED 1.0 features a novel application of a switching function to retain or displace key water mols. from the protein-ligand complexes.  Two independent modules, ProCESS and SMART, were developed to set up the proteins and the ligands prior to the docking stage.  Validation of the accuracy of the software was achieved via the application of FITTED 1.0 to the docking of inhibitors of HIV-1 protease, thymidine kinase, trypsin, factor Xa, and MMP to their resp. proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqH9ypbBV5-k7Vg90H21EOLACvtfcHk0liq1EtW17H97w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhvVWjtbY%253D&md5=f1b5be0eb418ecc158f5f057c1f9567c</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Fci6002637&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci6002637%26sid%3Dliteratum%253Aachs%26aulast%3DCorbeil%26aufirst%3DC.%2BR.%26aulast%3DEnglebienne%26aufirst%3DP.%26aulast%3DMoitessier%26aufirst%3DN.%26atitle%3DDocking%2520ligands%2520into%2520flexible%2520and%2520solvated%2520macromolecules.%25201.%2520Development%2520and%2520validation%2520of%2520FITTED%25201.0%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2007%26volume%3D47%26spage%3D435%26epage%3D449" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Friesner, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murphy, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Repasky, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, L. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greenwood, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Halgren, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sanschagrin, P. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mainz, D. T.</span><span> </span><span class="NLM_article-title">Extra precision glide: docking and scoring incorporating a model of hydrophobic enclosure for protein−ligand complexes</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_year" style="font-weight: bold;">2006</span> <span class="NLM_volume">49</span> <span class="NLM_fpage">6177</span> <span class="NLM_lpage">6196</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm051256o" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=6177-6196&author=R.+A.+Friesnerauthor=R.+B.+Murphyauthor=M.+P.+Repaskyauthor=L.+L.+Fryeauthor=J.+R.+Greenwoodauthor=T.+A.+Halgrenauthor=P.+C.+Sanschagrinauthor=D.+T.+Mainz&title=Extra+precision+glide%3A+docking+and+scoring+incorporating+a+model+of+hydrophobic+enclosure+for+protein%E2%88%92ligand+complexes"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Fjm051256o&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm051256o%26sid%3Dliteratum%253Aachs%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DRepasky%26aufirst%3DM.%2BP.%26aulast%3DFrye%26aufirst%3DL.%2BL.%26aulast%3DGreenwood%26aufirst%3DJ.%2BR.%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DSanschagrin%26aufirst%3DP.%2BC.%26aulast%3DMainz%26aufirst%3DD.%2BT.%26atitle%3DExtra%2520precision%2520glide%253A%2520docking%2520and%2520scoring%2520incorporating%2520a%2520model%2520of%2520hydrophobic%2520enclosure%2520for%2520protein%25E2%2588%2592ligand%2520complexes%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D6177%26epage%3D6196" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Roberts, B. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mancera, R. L.</span><span> </span><span class="NLM_article-title">Ligand−protein docking with water molecules</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_year" style="font-weight: bold;">2008</span> <span class="NLM_volume">48</span> <span class="NLM_fpage">397</span> <span class="NLM_lpage">408</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci700285e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Fjm8006239&amp;key=1%3ACAS%3A528%3ADC%252BD1cXptl2itQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2008&pages=397-408&author=B.+C.+Robertsauthor=R.+L.+Mancera&title=Ligand%E2%88%92protein+docking+with+water+molecules"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand-Protein Docking with Water Molecules</span></div><div class="casAuthors">Roberts, Benjamin C.; Mancera, Ricardo L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">397-408</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The presence of water mols. plays an important role in the accuracy of ligand-protein docking predictions.  Comprehensive docking simulations have been performed on a large set of ligand-protein complexes whose crystal structures contain water mols. in their binding sites.  Only those water mols. found in the immediate vicinity of both the ligand and the protein were considered.  The authors have investigated whether prior optimization of the orientation of water mols. in either the presence or absence of the bound ligand has any effect on the accuracy of docking predictions.  The authors have obsd. a statistically significant overall increase in accuracy when water mols. are included during docking simulations and have found this to be independent of the method of optimization of the orientation of water mols.  These results confirm the importance of including water mols. whenever possible in a ligand-protein docking simulation.  The authors' findings also reveal that prior optimization of the orientation of water mols., in the absence of any bound ligand, does not have a detrimental effect on the improved accuracy of ligand-protein docking.  This is important, given the use of docking simulations to predict the binding modes of new ligands or drug mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpRgAZwwN3kBLVg90H21EOLACvtfcHk0liH5uNPe9tkxg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXptl2itQ%253D%253D&md5=9559fcddc8fed156eb0b5a7ba982cc4a</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Fci700285e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci700285e%26sid%3Dliteratum%253Aachs%26aulast%3DRoberts%26aufirst%3DB.%2BC.%26aulast%3DMancera%26aufirst%3DR.%2BL.%26atitle%3DLigand%25E2%2588%2592protein%2520docking%2520with%2520water%2520molecules%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2008%26volume%3D48%26spage%3D397%26epage%3D408" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Huang, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span> </span><span class="NLM_article-title">Benchmarking sets for molecular docking</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_year" style="font-weight: bold;">2006</span> <span class="NLM_volume">49</span> <span class="NLM_fpage">6789</span> <span class="NLM_lpage">6801</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm0608356" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Fjm8006239&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtFehurzI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=6789-6801&author=N.+Huangauthor=B.+K.+Shoichetauthor=J.+J.+Irwin&title=Benchmarking+sets+for+molecular+docking"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Benchmarking Sets for Molecular Docking</span></div><div class="casAuthors">Huang, Niu; Shoichet, Brian K.; Irwin, John J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">6789-6801</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Ligand enrichment among top-ranking hits is a key metric of mol. docking.  To avoid bias, decoys should resemble ligands phys., so that enrichment is not simply a sepn. of gross features, yet be chem. distinct from them, so that they are unlikely to be binders.  We have assembled a directory of useful decoys (DUD), with 2950 ligands for 40 different targets.  Every ligand has 36 decoy mols. that are phys. similar but topol. distinct, leading to a database of 98 266 compds.  For most targets, enrichment was at least half a log better with uncorrected databases such as the MDDR than with DUD, evidence of bias in the former.  These calcns. also allowed 40×40 cross-docking, where the enrichments of each ligand set could be compared for all 40 targets, enabling a specificity metric for the docking screens.  DUD is freely available online as a benchmarking set for docking at http://blaster.docking.org/dud/.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-8FVhVSsBm7Vg90H21EOLACvtfcHk0liH5uNPe9tkxg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtFehurzI&md5=43af46968c2a6d66334922bf0caedbe1</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Fjm0608356&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0608356%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DN.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26atitle%3DBenchmarking%2520sets%2520for%2520molecular%2520docking%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D6789%26epage%3D6801" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Huang, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kalyanaraman, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacobson, M. P.</span><span> </span><span class="NLM_article-title">Physics-based scoring of protein−ligand complexes: enrichment of known inhibitors in large-scale virtual screening</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_year" style="font-weight: bold;">2006</span> <span class="NLM_volume">46</span> <span class="NLM_fpage">243</span> <span class="NLM_lpage">253</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci0502855" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Fjm8006239&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtF2ku77N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2006&pages=243-253&author=N.+Huangauthor=C.+Kalyanaramanauthor=J.+J.+Irwinauthor=M.+P.+Jacobson&title=Physics-based+scoring+of+protein%E2%88%92ligand+complexes%3A+enrichment+of+known+inhibitors+in+large-scale+virtual+screening"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Physics-Based Scoring of Protein-Ligand Complexes: Enrichment of Known Inhibitors in Large-Scale Virtual Screening</span></div><div class="casAuthors">Huang, Niu; Kalyanaraman, Chakrapani; Irwin, John J.; Jacobson, Matthew P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">243-253</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The authors demonstrate that using an all-atom mol. mechanics force field combined with an implicit solvent model for scoring protein-ligand complexes is a promising approach for improving inhibitor enrichment in the virtual screening of large compd. databases.  The rescoring method is evaluated by the extent to which known binders for nine diverse, therapeutically relevant enzymes are enriched against a background of ∼100 000 drug-like decoys.  The improvement in enrichment is most robust and dramatic within the top 1% of the ranked database, i.e., the first thousand compds.; below the first few percent of the ranked database, there is little overall improvement.  The improved early enrichment is likely due to the more realistic treatment of ligand and receptor desolvation in the rescoring procedure.  We also present anecdotal but encouraging results assessing the ability of the rescoring method to predict specificity of inhibitors for structurally related proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrUqkz9N6IErLVg90H21EOLACvtfcHk0liH5uNPe9tkxg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtF2ku77N&md5=cf636695aca21a87e050e0f39286ea1d</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Fci0502855&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci0502855%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DN.%26aulast%3DKalyanaraman%26aufirst%3DC.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DJacobson%26aufirst%3DM.%2BP.%26atitle%3DPhysics-based%2520scoring%2520of%2520protein%25E2%2588%2592ligand%2520complexes%253A%2520enrichment%2520of%2520known%2520inhibitors%2520in%2520large-scale%2520virtual%2520screening%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2006%26volume%3D46%26spage%3D243%26epage%3D253" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Wei, B. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weaver, L. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferrari, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matthews, B. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Testing a flexible-receptor docking algorithm in a model binding site</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_year" style="font-weight: bold;">2004</span> <span class="NLM_volume">337</span> <span class="NLM_fpage">1161</span> <span class="NLM_lpage">1182</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Fjm8006239&amp;key=10.1016%2Fj.jmb.2004.02.015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Fjm8006239&amp;key=15046985" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Fjm8006239&amp;key=1%3ACAS%3A528%3ADC%252BD2cXisVSmsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=337&publication_year=2004&pages=1161-1182&author=B.+Q.+Weiauthor=L.+H.+Weaverauthor=A.+M.+Ferrariauthor=B.+W.+Matthewsauthor=B.+K.+Shoichet&title=Testing+a+flexible-receptor+docking+algorithm+in+a+model+binding+site"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Testing a Flexible-receptor Docking Algorithm in a Model Binding Site</span></div><div class="casAuthors">Wei, Binqing Q.; Weaver, Larry H.; Ferrari, Anna M.; Matthews, Brian W.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">337</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1161-1182</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Sampling receptor flexibility is challenging for database docking.  We consider a method that treats multiple flexible regions of the binding site independently, recombining them to generate different discrete conformations.  This algorithm scales linearly rather than exponentially with the receptor's degrees of freedom.  The method was first evaluated for its ability to identify known ligands of a hydrophobic cavity mutant of T4 lysozyme (L99A).  Some 200,000 mols. of the Available Chem. Directory (ACD) were docked against an ensemble of cavity conformations.  Surprisingly, the enrichment of known ligands from among a much larger no. of decoys in the ACD was worse than simply docking to the apo conformation alone.  Large decoys, accommodated in the larger cavity conformations sampled in the ensemble, were ranked better than known small ligands.  The calcn. was redone with an energy correction term that considered the cost of forming the larger cavity conformations.  Enrichment improved, as did the balance between high-ranking large and small ligands.  In a second retrospective test, the ACD was docked against a conformational ensemble of thymidylate synthase.  Compared to docking against individual enzyme conformations, the flexible receptor docking approach improved enrichment of known ligands.  Including a receptor conformational energy weighting term improved enrichment further.  To test the method prospectively, the ACD database was docked against another cavity mutant of lysozyme (L99A/M102Q).  A total of 18 new compds. predicted to bind this polar cavity and to change its conformation were tested exptl.; 14 were found to bind.  The bound structures for seven ligands were detd. by x-ray crystallog.  The predicted geometries of these ligands all corresponded to the obsd. geometries to within 0.7 Å RMSD or better.  Significant conformational changes of the cavity were obsd. in all seven complexes.  In five structures, part of the obsd. accommodations were correctly predicted; in two structures, the receptor conformational changes were unanticipated and thus never sampled.  These results suggest that although sampling receptor flexibility can lead to novel ligands that would have been missed when docking a rigid structure, it is also important to consider receptor conformational energy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3S7C0vkfqQrVg90H21EOLACvtfcHk0lgFqogYFjCKWg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXisVSmsbg%253D&md5=6343c78da4cd0b0171dedb660ec9cacd</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2004.02.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2004.02.015%26sid%3Dliteratum%253Aachs%26aulast%3DWei%26aufirst%3DB.%2BQ.%26aulast%3DWeaver%26aufirst%3DL.%2BH.%26aulast%3DFerrari%26aufirst%3DA.%2BM.%26aulast%3DMatthews%26aufirst%3DB.%2BW.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DTesting%2520a%2520flexible-receptor%2520docking%2520algorithm%2520in%2520a%2520model%2520binding%2520site%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2004%26volume%3D337%26spage%3D1161%26epage%3D1182" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Pettersen, E. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goddard, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Couch, G. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greenblatt, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meng, E. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferrin, T. E.</span><span> </span><span class="NLM_article-title">UCSF Chimera—a visualization system for exploratory research and analysis</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_year" style="font-weight: bold;">2004</span> <span class="NLM_volume">25</span> <span class="NLM_fpage">1605</span> <span class="NLM_lpage">1612</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2004&pages=1605-1612&author=E.+F.+Pettersenauthor=T.+D.+Goddardauthor=C.+C.+Huangauthor=G.+S.+Couchauthor=D.+M.+Greenblattauthor=E.+C.+Mengauthor=T.+E.+Ferrin&title=UCSF+Chimera%E2%80%94a+visualization+system+for+exploratory+research+and+analysis"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DPettersen%26aufirst%3DE.%2BF.%26aulast%3DGoddard%26aufirst%3DT.%2BD.%26aulast%3DHuang%26aufirst%3DC.%2BC.%26aulast%3DCouch%26aufirst%3DG.%2BS.%26aulast%3DGreenblatt%26aufirst%3DD.%2BM.%26aulast%3DMeng%26aufirst%3DE.%2BC.%26aulast%3DFerrin%26aufirst%3DT.%2BE.%26atitle%3DUCSF%2520Chimera%25E2%2580%2594a%2520visualization%2520system%2520for%2520exploratory%2520research%2520and%2520analysis%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2004%26volume%3D25%26spage%3D1605%26epage%3D1612" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 92 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">SahaIshika</span><span class="NLM_role">Graduate Student Researcher</span><a class="email" href="/cdn-cgi/l/email-protection#0a63796263616b796b626b4a6d247f69666b246f6e7f"><span class="nobrWithWbr"><span class="__cf_email__" data-cfemail="61081209080a001200090021064f">[email protected]</span><wbr></wbr>ucla.<wbr></wbr>edu</span></a><span class="hlFld-ContribAuthor ">HarranPatrick G.</span><span class="NLM_role">D.J. & J.M. Cram Chair in Organic Chemistry</span><a class="email" href="/cdn-cgi/l/email-protection#cca4adbebeada28cafa4a9a1e2b9afa0ade2a9a8b9"><span class="nobrWithWbr"><span class="__cf_email__" data-cfemail="92faf3e0e0f3fcd2f1faf7ffbc">[email protected]</span><wbr></wbr>ucla.<wbr></wbr>edu</span></a>Dr. Jonathan Bohmann, Department of Pharmaceuticals and Bioengineering, Southwest Research Institute, <div class="author-comment"><p class="first last">A great introduction to the topic of virtual screening, which is often seen as a black box.  It does a good job weaving a lot of material together. The molecular simulation community has needed a book like this for a long time. It is honest about the pitfalls of VS while pointing to reasons why VS is needed and the positive impacts on early drug discovery programs, where resources are generally scarce.</p></div><span class="NLM_string-name hlFld-ContribAuthor">Ryan Gumpper, Postdoctoral Researcher, University of North Carolina at Chapel Hill</span><div class="author-comment"><p class="first last">I would recommend this work to young graduate students starting in the field and anyone who is interested in expanding their knowledge in the field.</p></div>. </span><span class="cited-content_cbyCitation_article-title">Virtual Screening for Chemists. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,<a href="https://doi.org/10.1021/acsinfocus.7e5001" title="DOI URL">https://doi.org/10.1021/acsinfocus.7e5001</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsinfocus.7e5001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsinfocus.7e5001%26sid%3Dliteratum%253Aachs%26atitle%3DVirtual%252BScreening%252Bfor%252BChemists%26aulast%3DSaha%26aufirst%3DIshika%26date%3D2021%26date%3D2021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Jorge Enrique Hernández
González, Lilian Hernández Alvarez, Vitor B. P. Leite, <span class="NLM_string-name hlFld-ContribAuthor">Pedro Geraldo Pascutti</span>. </span><span class="cited-content_cbyCitation_article-title">Water Bridges Play a Key Role in Affinity and Selectivity for Malarial Protease Falcipain-2. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (11)
                                     , 5499-5512. <a href="https://doi.org/10.1021/acs.jcim.0c00294" title="DOI URL">https://doi.org/10.1021/acs.jcim.0c00294</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.0c00294&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.0c00294%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DWater%252BBridges%252BPlay%252Ba%252BKey%252BRole%252Bin%252BAffinity%252Band%252BSelectivity%252Bfor%252BMalarial%252BProtease%252BFalcipain-2%26aulast%3DHern%25C3%25A1ndez%2BGonz%25C3%25A1lez%26aufirst%3DJorge%2BEnrique%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D25032020%26date%3D22072020%26date%3D07072020%26volume%3D60%26issue%3D11%26spage%3D5499%26epage%3D5512" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Gudong Li, Wei Li, Yuting Xie, Xiaobo Wan, Guojun Zheng, Niu Huang, <span class="NLM_string-name hlFld-ContribAuthor">Yu Zhou</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of Novel Pim-1 Kinase Inhibitors with a Flexible-Receptor Docking Protocol. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2019,</strong> <em>59 </em>
                                    (10)
                                     , 4116-4119. <a href="https://doi.org/10.1021/acs.jcim.9b00494" title="DOI URL">https://doi.org/10.1021/acs.jcim.9b00494</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.9b00494&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.9b00494%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DDiscovery%252Bof%252BNovel%252BPim-1%252BKinase%252BInhibitors%252Bwith%252Ba%252BFlexible-Receptor%252BDocking%252BProtocol%26aulast%3DLi%26aufirst%3DGudong%26date%3D2019%26date%3D2019%26date%3D17102019%26date%3D14102019%26volume%3D59%26issue%3D10%26spage%3D4116%26epage%3D4119" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Benedict
W. J. Irwin, Sinisa Vukovic, Michael C. Payne, <span class="NLM_string-name hlFld-ContribAuthor">David J. Huggins</span>. </span><span class="cited-content_cbyCitation_article-title">Large-Scale Study of Hydration Environments through Hydration Sites. </span><span class="cited-content_cbyCitation_journal-name">The Journal of Physical Chemistry B</span><span> <strong>2019,</strong> <em>123 </em>
                                    (19)
                                     , 4220-4229. <a href="https://doi.org/10.1021/acs.jpcb.9b02490" title="DOI URL">https://doi.org/10.1021/acs.jpcb.9b02490</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jpcb.9b02490&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jpcb.9b02490%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Journal%2520of%2520Physical%2520Chemistry%2520B%26atitle%3DLarge-Scale%252BStudy%252Bof%252BHydration%252BEnvironments%252Bthrough%252BHydration%252BSites%26aulast%3DIrwin%26aufirst%3DBenedict%2BW.%2BJ.%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D15032019%26date%3D22042019%26date%3D07052019%26date%3D26042019%26volume%3D123%26issue%3D19%26spage%3D4220%26epage%3D4229" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Axel Rudling, Adolfo Orro, <span class="NLM_string-name hlFld-ContribAuthor">Jens Carlsson</span>. </span><span class="cited-content_cbyCitation_article-title">Prediction of Ordered Water Molecules in Protein Binding Sites from Molecular Dynamics Simulations: The Impact of Ligand Binding on Hydration Networks. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2018,</strong> <em>58 </em>
                                    (2)
                                     , 350-361. <a href="https://doi.org/10.1021/acs.jcim.7b00520" title="DOI URL">https://doi.org/10.1021/acs.jcim.7b00520</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.7b00520&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.7b00520%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DPrediction%252Bof%252BOrdered%252BWater%252BMolecules%252Bin%252BProtein%252BBinding%252BSites%252Bfrom%252BMolecular%252BDynamics%252BSimulations%25253A%252BThe%252BImpact%252Bof%252BLigand%252BBinding%252Bon%252BHydration%252BNetworks%26aulast%3DRudling%26aufirst%3DAxel%26date%3D2018%26date%3D2018%26date%3D2017%26date%3D31082017%26date%3D05022018%26date%3D08012018%26volume%3D58%26issue%3D2%26spage%3D350%26epage%3D361" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Francesca  Spyrakis</span>, <span class="hlFld-ContribAuthor ">Mostafa H.  Ahmed</span>, <span class="hlFld-ContribAuthor ">Alexander S.  Bayden</span>, <span class="hlFld-ContribAuthor ">Pietro  Cozzini</span>, <span class="hlFld-ContribAuthor ">Andrea  Mozzarelli</span>, and <span class="hlFld-ContribAuthor ">Glen E.  Kellogg</span>  . </span><span class="cited-content_cbyCitation_article-title">The Roles of Water in the Protein Matrix: A Largely Untapped Resource for Drug Discovery. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2017,</strong> <em>60 </em>
                                    (16)
                                     , 6781-6827. <a href="https://doi.org/10.1021/acs.jmedchem.7b00057" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b00057</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b00057&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.7b00057%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DThe%252BRoles%252Bof%252BWater%252Bin%252Bthe%252BProtein%252BMatrix%25253A%252BA%252BLargely%252BUntapped%252BResource%252Bfor%252BDrug%252BDiscovery%26aulast%3DSpyrakis%26aufirst%3DFrancesca%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D13012017%26date%3D05052017%26date%3D24082017%26volume%3D60%26issue%3D16%26spage%3D6781%26epage%3D6827" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Wei  Xiao</span>, <span class="hlFld-ContribAuthor ">Zenghui  He</span>, <span class="hlFld-ContribAuthor ">Meijian  Sun</span>, <span class="hlFld-ContribAuthor ">Shiliang  Li</span>, and <span class="hlFld-ContribAuthor ">Honglin  Li</span>  . </span><span class="cited-content_cbyCitation_article-title">Statistical Analysis, Investigation, and Prediction of the Water Positions in the Binding Sites of Proteins. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2017,</strong> <em>57 </em>
                                    (7)
                                     , 1517-1528. <a href="https://doi.org/10.1021/acs.jcim.6b00620" title="DOI URL">https://doi.org/10.1021/acs.jcim.6b00620</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.6b00620&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.6b00620%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DStatistical%252BAnalysis%25252C%252BInvestigation%25252C%252Band%252BPrediction%252Bof%252Bthe%252BWater%252BPositions%252Bin%252Bthe%252BBinding%252BSites%252Bof%252BProteins%26aulast%3DXiao%26aufirst%3DWei%26date%3D2017%26date%3D2017%26date%3D2016%26date%3D12102016%26date%3D27062017%26date%3D24072017%26date%3D12062017%26volume%3D57%26issue%3D7%26spage%3D1517%26epage%3D1528" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">John J.  Irwin</span> and <span class="hlFld-ContribAuthor ">Brian K.  Shoichet</span>  . </span><span class="cited-content_cbyCitation_article-title">Docking Screens for Novel Ligands Conferring New Biology. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2016,</strong> <em>59 </em>
                                    (9)
                                     , 4103-4120. <a href="https://doi.org/10.1021/acs.jmedchem.5b02008" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.5b02008</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.5b02008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.5b02008%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDocking%252BScreens%252Bfor%252BNovel%252BLigands%252BConferring%252BNew%252BBiology%26aulast%3DIrwin%26aufirst%3DJohn%2BJ.%26date%3D2016%26date%3D2016%26date%3D2015%26date%3D25122015%26date%3D15032016%26date%3D12052016%26date%3D25022016%26volume%3D59%26issue%3D9%26spage%3D4103%26epage%3D4120" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">WenJuan  Huang</span>, <span class="hlFld-ContribAuthor ">Nikolay  Blinov</span>, <span class="hlFld-ContribAuthor ">David S.  Wishart</span>, and <span class="hlFld-ContribAuthor ">Andriy  Kovalenko</span>  . </span><span class="cited-content_cbyCitation_article-title">Role of Water in Ligand Binding to Maltose-Binding Protein: Insight from a New Docking Protocol Based on the 3D-RISM-KH Molecular Theory of Solvation. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2015,</strong> <em>55 </em>
                                    (2)
                                     , 317-328. <a href="https://doi.org/10.1021/ci500520q" title="DOI URL">https://doi.org/10.1021/ci500520q</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci500520q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci500520q%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DRole%252Bof%252BWater%252Bin%252BLigand%252BBinding%252Bto%252BMaltose-Binding%252BProtein%25253A%252BInsight%252Bfrom%252Ba%252BNew%252BDocking%252BProtocol%252BBased%252Bon%252Bthe%252B3D-RISM-KH%252BMolecular%252BTheory%252Bof%252BSolvation%26aulast%3DHuang%26aufirst%3DWenJuan%26date%3D2015%26date%3D2015%26date%3D2014%26date%3D27082014%26date%3D26012015%26date%3D23022015%26date%3D29122014%26volume%3D55%26issue%3D2%26spage%3D317%26epage%3D328" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Eric  Therrien</span>, <span class="hlFld-ContribAuthor ">Nathanael  Weill</span>, <span class="hlFld-ContribAuthor ">Anna  Tomberg</span>, <span class="hlFld-ContribAuthor ">Christopher R.  Corbeil</span>, <span class="hlFld-ContribAuthor ">Devin  Lee</span>, and <span class="hlFld-ContribAuthor ">Nicolas  Moitessier</span>  . </span><span class="cited-content_cbyCitation_article-title">Docking Ligands into Flexible and Solvated Macromolecules. 7. Impact of Protein Flexibility and Water Molecules on Docking-Based Virtual Screening Accuracy. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2014,</strong> <em>54 </em>
                                    (11)
                                     , 3198-3210. <a href="https://doi.org/10.1021/ci500299h" title="DOI URL">https://doi.org/10.1021/ci500299h</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci500299h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci500299h%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DDocking%252BLigands%252Binto%252BFlexible%252Band%252BSolvated%252BMacromolecules.%252B7.%252BImpact%252Bof%252BProtein%252BFlexibility%252Band%252BWater%252BMolecules%252Bon%252BDocking-Based%252BVirtual%252BScreening%252BAccuracy%26aulast%3DTherrien%26aufirst%3DEric%26date%3D2014%26date%3D2014%26date%3D2014%26date%3D16052014%26date%3D27102014%26date%3D24112014%26date%3D03102014%26volume%3D54%26issue%3D11%26spage%3D3198%26epage%3D3210" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Maria Letizia  Barreca</span>, <span class="hlFld-ContribAuthor ">Nunzio  Iraci</span>, <span class="hlFld-ContribAuthor ">Giuseppe  Manfroni</span>, <span class="hlFld-ContribAuthor ">Rosy  Gaetani</span>, <span class="hlFld-ContribAuthor ">Chiara  Guercini</span>, <span class="hlFld-ContribAuthor ">Stefano  Sabatini</span>, <span class="hlFld-ContribAuthor ">Oriana  Tabarrini</span>, and <span class="hlFld-ContribAuthor ">Violetta  Cecchetti</span>  . </span><span class="cited-content_cbyCitation_article-title">Accounting for Target Flexibility and Water Molecules by Docking to Ensembles of Target Structures: The HCV NS5B Palm Site I Inhibitors Case Study. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2014,</strong> <em>54 </em>
                                    (2)
                                     , 481-497. <a href="https://doi.org/10.1021/ci400367m" title="DOI URL">https://doi.org/10.1021/ci400367m</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci400367m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci400367m%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DAccounting%252Bfor%252BTarget%252BFlexibility%252Band%252BWater%252BMolecules%252Bby%252BDocking%252Bto%252BEnsembles%252Bof%252BTarget%252BStructures%25253A%252BThe%252BHCV%252BNS5B%252BPalm%252BSite%252BI%252BInhibitors%252BCase%252BStudy%26aulast%3DBarreca%26aufirst%3DMaria%2BLetizia%26date%3D2014%26date%3D2013%26date%3D2013%26date%3D25062013%26date%3D05092013%26date%3D24022014%26date%3D18082013%26volume%3D54%26issue%3D2%26spage%3D481%26epage%3D497" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kamran  Haider</span> and <span class="hlFld-ContribAuthor ">David J.  Huggins</span>  . </span><span class="cited-content_cbyCitation_article-title">Combining Solvent Thermodynamic Profiles with Functionality Maps of the Hsp90 Binding Site to Predict the Displacement of Water Molecules. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2013,</strong> <em>53 </em>
                                    (10)
                                     , 2571-2586. <a href="https://doi.org/10.1021/ci4003409" title="DOI URL">https://doi.org/10.1021/ci4003409</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci4003409&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci4003409%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DCombining%252BSolvent%252BThermodynamic%252BProfiles%252Bwith%252BFunctionality%252BMaps%252Bof%252Bthe%252BHsp90%252BBinding%252BSite%252Bto%252BPredict%252Bthe%252BDisplacement%252Bof%252BWater%252BMolecules%26aulast%3DHaider%26aufirst%3DKamran%26date%3D2013%26date%3D2013%26date%3D2013%26date%3D12062013%26date%3D15102013%26date%3D28102013%26date%3D26092013%26volume%3D53%26issue%3D10%26spage%3D2571%26epage%3D2586" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ashutosh  Kumar</span> and <span class="hlFld-ContribAuthor ">Kam Y. J.  Zhang</span>  . </span><span class="cited-content_cbyCitation_article-title">Investigation on the Effect of Key Water Molecules on Docking Performance in CSARdock Exercise. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2013,</strong> <em>53 </em>
                                    (8)
                                     , 1880-1892. <a href="https://doi.org/10.1021/ci400052w" title="DOI URL">https://doi.org/10.1021/ci400052w</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci400052w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci400052w%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DInvestigation%252Bon%252Bthe%252BEffect%252Bof%252BKey%252BWater%252BMolecules%252Bon%252BDocking%252BPerformance%252Bin%252BCSARdock%252BExercise%26aulast%3DKumar%26aufirst%3DAshutosh%26date%3D2013%26date%3D2013%26date%3D2012%26date%3D30112012%26date%3D08052013%26date%3D26082013%26date%3D25042013%26volume%3D53%26issue%3D8%26spage%3D1880%26epage%3D1892" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Alfonso T.  García-Sosa</span>  . </span><span class="cited-content_cbyCitation_article-title">Hydration Properties of Ligands and Drugs in Protein Binding Sites: Tightly-Bound, Bridging Water Molecules and Their Effects and Consequences on Molecular Design Strategies. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2013,</strong> <em>53 </em>
                                    (6)
                                     , 1388-1405. <a href="https://doi.org/10.1021/ci3005786" title="DOI URL">https://doi.org/10.1021/ci3005786</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci3005786&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci3005786%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DHydration%252BProperties%252Bof%252BLigands%252Band%252BDrugs%252Bin%252BProtein%252BBinding%252BSites%25253A%252BTightly-Bound%25252C%252BBridging%252BWater%252BMolecules%252Band%252BTheir%252BEffects%252Band%252BConsequences%252Bon%252BMolecular%252BDesign%252BStrategies%26aulast%3DGarc%25C3%25ADa-Sosa%26aufirst%3DAlfonso%2BT.%26date%3D2013%26date%3D2013%26date%3D2012%26date%3D05122012%26date%3D23052013%26date%3D24062013%26date%3D13052013%26volume%3D53%26issue%3D6%26spage%3D1388%26epage%3D1405" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Laurent  Hoffer</span> and <span class="hlFld-ContribAuthor ">Dragos  Horvath</span>  . </span><span class="cited-content_cbyCitation_article-title">S4MPLE – Sampler For Multiple Protein–Ligand Entities: Simultaneous Docking of Several Entities. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2013,</strong> <em>53 </em>
                                    (1)
                                     , 88-102. <a href="https://doi.org/10.1021/ci300495r" title="DOI URL">https://doi.org/10.1021/ci300495r</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci300495r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci300495r%26sid%3Dliteratum%253Aachs%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1995%26volume%3D117%26spage%3D5179%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2004%26volume%3D25%26spage%3D1157%26jtitle%3DJ.%2520Mol.%2520Graphics%2520Modell.%26date%3D2012%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2007%26volume%3D50%26spage%3D726%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DS4MPLE%252B%2525E2%252580%252593%252BSampler%252BFor%252BMultiple%252BProtein%2525E2%252580%252593Ligand%252BEntities%25253A%252BSimultaneous%252BDocking%252Bof%252BSeveral%252BEntities%26aulast%3DHoffer%26aufirst%3DLaurent%26date%3D2013%26date%3D2012%26date%3D2012%26date%3D15102012%26date%3D24122012%26date%3D28012013%26date%3D10122012%26volume%3D53%26issue%3D1%26spage%3D88%26epage%3D102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Fung-Yi  Chan</span>, <span class="hlFld-ContribAuthor ">Marco A. C.  Neves</span>, <span class="hlFld-ContribAuthor ">Ning  Sun</span>, <span class="hlFld-ContribAuthor ">Man-Wah  Tsang</span>, <span class="hlFld-ContribAuthor ">Yun-Chung  Leung</span>, <span class="hlFld-ContribAuthor ">Tak-Hang  Chan</span>, <span class="hlFld-ContribAuthor ">Ruben  Abagyan</span>, and <span class="hlFld-ContribAuthor ">Kwok-Yin  Wong</span>  . </span><span class="cited-content_cbyCitation_article-title">Validation of the AmpC β-Lactamase Binding Site and Identification of Inhibitors with Novel Scaffolds. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2012,</strong> <em>52 </em>
                                    (5)
                                     , 1367-1375. <a href="https://doi.org/10.1021/ci300068m" title="DOI URL">https://doi.org/10.1021/ci300068m</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci300068m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci300068m%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DValidation%252Bof%252Bthe%252BAmpC%252B%2525CE%2525B2-Lactamase%252BBinding%252BSite%252Band%252BIdentification%252Bof%252BInhibitors%252Bwith%252BNovel%252BScaffolds%26aulast%3DChan%26aufirst%3DFung-Yi%26date%3D2012%26date%3D2012%26date%3D2012%26date%3D05022012%26date%3D04052012%26date%3D25052012%26volume%3D52%26issue%3D5%26spage%3D1367%26epage%3D1375" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Stefano  Forli</span> and <span class="hlFld-ContribAuthor ">Arthur J.  Olson</span>  . </span><span class="cited-content_cbyCitation_article-title">A Force Field with Discrete Displaceable Waters and Desolvation Entropy for Hydrated Ligand Docking. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2012,</strong> <em>55 </em>
                                    (2)
                                     , 623-638. <a href="https://doi.org/10.1021/jm2005145" title="DOI URL">https://doi.org/10.1021/jm2005145</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jm2005145&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjm2005145%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DA%252BForce%252BField%252Bwith%252BDiscrete%252BDisplaceable%252BWaters%252Band%252BDesolvation%252BEntropy%252Bfor%252BHydrated%252BLigand%252BDocking%26aulast%3DForli%26aufirst%3DStefano%26date%3D2012%26date%3D2012%26date%3D2011%26date%3D27042011%26date%3D13012012%26date%3D26012012%26date%3D09122011%26volume%3D55%26issue%3D2%26spage%3D623%26epage%3D638" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Gianluca  Rossato</span>, <span class="hlFld-ContribAuthor ">Beat  Ernst</span>, <span class="hlFld-ContribAuthor ">Angelo  Vedani</span>, and <span class="hlFld-ContribAuthor ">Martin  Smieško</span>  . </span><span class="cited-content_cbyCitation_article-title">AcquaAlta: A Directional Approach to the Solvation of Ligand–Protein Complexes. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2011,</strong> <em>51 </em>
                                    (8)
                                     , 1867-1881. <a href="https://doi.org/10.1021/ci200150p" title="DOI URL">https://doi.org/10.1021/ci200150p</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci200150p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci200150p%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DAcquaAlta%25253A%252BA%252BDirectional%252BApproach%252Bto%252Bthe%252BSolvation%252Bof%252BLigand%2525E2%252580%252593Protein%252BComplexes%26aulast%3DRossato%26aufirst%3DGianluca%26date%3D2011%26date%3D2011%26date%3D2011%26date%3D29032011%26date%3D18072011%26date%3D22082011%26date%3D29062011%26volume%3D51%26issue%3D8%26spage%3D1867%26epage%3D1881" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Rafaela S.  Ferreira</span>, <span class="hlFld-ContribAuthor ">Anton  Simeonov</span>, <span class="hlFld-ContribAuthor ">Ajit  Jadhav</span>, <span class="hlFld-ContribAuthor ">Oliv  Eidam</span>, <span class="hlFld-ContribAuthor ">Bryan T.  Mott</span>, <span class="hlFld-ContribAuthor ">Michael J.  Keiser</span>, <span class="hlFld-ContribAuthor ">James H.  McKerrow</span>, <span class="hlFld-ContribAuthor ">David J.  Maloney</span>, <span class="hlFld-ContribAuthor ">John J.  Irwin</span> and <span class="hlFld-ContribAuthor ">Brian K.  Shoichet</span>  . </span><span class="cited-content_cbyCitation_article-title">Complementarity Between a Docking and a High-Throughput Screen in Discovering New Cruzain Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2010,</strong> <em>53 </em>
                                    (13)
                                     , 4891-4905. <a href="https://doi.org/10.1021/jm100488w" title="DOI URL">https://doi.org/10.1021/jm100488w</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jm100488w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjm100488w%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DComplementarity%252BBetween%252Ba%252BDocking%252Band%252Ba%252BHigh-Throughput%252BScreen%252Bin%252BDiscovering%252BNew%252BCruzain%252BInhibitors%26aulast%3DFerreira%26aufirst%3DRafaela%2BS.%26date%3D2010%26date%3D2010%26date%3D2010%26date%3D03022010%26date%3D11062010%26date%3D08072010%26volume%3D53%26issue%3D13%26spage%3D4891%26epage%3D4905" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Francesca  Milletti</span> and <span class="hlFld-ContribAuthor ">Anna  Vulpetti</span>  . </span><span class="cited-content_cbyCitation_article-title">Tautomer Preference in PDB Complexes and its Impact on Structure-Based Drug Discovery. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2010,</strong> <em>50 </em>
                                    (6)
                                     , 1062-1074. <a href="https://doi.org/10.1021/ci900501c" title="DOI URL">https://doi.org/10.1021/ci900501c</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci900501c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci900501c%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DTautomer%252BPreference%252Bin%252BPDB%252BComplexes%252Band%252Bits%252BImpact%252Bon%252BStructure-Based%252BDrug%252BDiscovery%26aulast%3DMilletti%26aufirst%3DFrancesca%26date%3D2010%26date%3D2010%26date%3D2009%26date%3D23%252012%25202009%26date%3D01062010%26date%3D28062010%26volume%3D50%26issue%3D6%26spage%3D1062%26epage%3D1074" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Fiona M.  McRobb</span>, <span class="hlFld-ContribAuthor ">Ben  Capuano</span>, <span class="hlFld-ContribAuthor ">Ian T.  Crosby</span>, <span class="hlFld-ContribAuthor ">David K.  Chalmers</span> and <span class="hlFld-ContribAuthor ">Elizabeth  Yuriev</span>. </span><span class="cited-content_cbyCitation_article-title">Homology Modeling and Docking Evaluation of Aminergic G Protein-Coupled Receptors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2010,</strong> <em>50 </em>
                                    (4)
                                     , 626-637. <a href="https://doi.org/10.1021/ci900444q" title="DOI URL">https://doi.org/10.1021/ci900444q</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci900444q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci900444q%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DHomology%252BModeling%252Band%252BDocking%252BEvaluation%252Bof%252BAminergic%252BG%252BProtein-Coupled%252BReceptors%26aulast%3DMcRobb%26aufirst%3DFiona%2BM.%26date%3D2010%26date%3D2010%26date%3D2009%26date%3D15%252011%25202009%26date%3D01032010%26date%3D26042010%26volume%3D50%26issue%3D4%26spage%3D626%26epage%3D637" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ramasamy  Thilagavathi</span> and <span class="hlFld-ContribAuthor ">Ricardo L.  Mancera</span>. </span><span class="cited-content_cbyCitation_article-title">Ligand−Protein Cross-Docking with Water Molecules. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2010,</strong> <em>50 </em>
                                    (3)
                                     , 415-421. <a href="https://doi.org/10.1021/ci900345h" title="DOI URL">https://doi.org/10.1021/ci900345h</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci900345h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci900345h%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DLigand%2525E2%252588%252592Protein%252BCross-Docking%252Bwith%252BWater%252BMolecules%26aulast%3DThilagavathi%26aufirst%3DRamasamy%26date%3D2010%26date%3D2010%26date%3D2009%26date%3D12%252009%25202009%26date%3D17022010%26date%3D22032010%26volume%3D50%26issue%3D3%26spage%3D415%26epage%3D421" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Rita  Santos</span>, <span class="hlFld-ContribAuthor ">Jozef  Hritz</span> and <span class="hlFld-ContribAuthor ">Chris  Oostenbrink</span>. </span><span class="cited-content_cbyCitation_article-title">Role of Water in Molecular Docking Simulations of Cytochrome P450 2D6. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2010,</strong> <em>50 </em>
                                    (1)
                                     , 146-154. <a href="https://doi.org/10.1021/ci900293e" title="DOI URL">https://doi.org/10.1021/ci900293e</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci900293e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci900293e%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DRole%252Bof%252BWater%252Bin%252BMolecular%252BDocking%252BSimulations%252Bof%252BCytochrome%252BP450%252B2D6%26aulast%3DSantos%26aufirst%3DRita%26date%3D2010%26date%3D2009%26date%3D2009%26date%3D7%252008%25202009%26date%3D09112009%26date%3D25012010%26volume%3D50%26issue%3D1%26spage%3D146%26epage%3D154" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Debananda  Das</span>, <span class="hlFld-ContribAuthor ">Yasuhiro  Koh</span>, <span class="hlFld-ContribAuthor ">Yasushi  Tojo</span>, <span class="hlFld-ContribAuthor ">Arun K.  Ghosh</span> and <span class="hlFld-ContribAuthor ">Hiroaki  Mitsuya</span>  . </span><span class="cited-content_cbyCitation_article-title">Prediction of Potency of Protease Inhibitors Using Free Energy Simulations with Polarizable Quantum Mechanics-Based Ligand Charges and a Hybrid Water Model. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2009,</strong> <em>49 </em>
                                    (12)
                                     , 2851-2862. <a href="https://doi.org/10.1021/ci900320p" title="DOI URL">https://doi.org/10.1021/ci900320p</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci900320p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci900320p%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DPrediction%252Bof%252BPotency%252Bof%252BProtease%252BInhibitors%252BUsing%252BFree%252BEnergy%252BSimulations%252Bwith%252BPolarizable%252BQuantum%252BMechanics-Based%252BLigand%252BCharges%252Band%252Ba%252BHybrid%252BWater%252BModel%26aulast%3DDas%26aufirst%3DDebananda%26date%3D2009%26date%3D2009%26date%3D2009%26date%3D26%252008%25202009%26date%3D24112009%26date%3D28122009%26volume%3D49%26issue%3D12%26spage%3D2851%26epage%3D2862" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Leonardo  Pisani</span>, <span class="hlFld-ContribAuthor ">Giovanni  Muncipinto</span>, <span class="hlFld-ContribAuthor ">Teresa Fabiola  Miscioscia</span>, <span class="hlFld-ContribAuthor ">Orazio  Nicolotti</span>, <span class="hlFld-ContribAuthor ">Francesco  Leonetti</span>, <span class="hlFld-ContribAuthor ">Marco  Catto</span>, <span class="hlFld-ContribAuthor ">Carla  Caccia</span>, <span class="hlFld-ContribAuthor ">Patricia  Salvati</span>, <span class="hlFld-ContribAuthor ">Ramon  Soto-Otero</span>, <span class="hlFld-ContribAuthor ">Estefania  Mendez-Alvarez</span>, <span class="hlFld-ContribAuthor ">Celine  Passeleu</span> and <span class="hlFld-ContribAuthor ">Angelo  Carotti</span>  . </span><span class="cited-content_cbyCitation_article-title">Discovery of a Novel Class of Potent Coumarin Monoamine Oxidase B Inhibitors: Development and Biopharmacological Profiling of 7-[(3-Chlorobenzyl)oxy]-4-[(methylamino)methyl]-2H-chromen-2-one Methanesulfonate (NW-1772) as a Highly Potent, Selective, Reversible, and Orally Active Monoamine Oxidase B Inhibitor. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2009,</strong> <em>52 </em>
                                    (21)
                                     , 6685-6706. <a href="https://doi.org/10.1021/jm9010127" title="DOI URL">https://doi.org/10.1021/jm9010127</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jm9010127&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjm9010127%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252Ba%252BNovel%252BClass%252Bof%252BPotent%252BCoumarin%252BMonoamine%252BOxidase%252BB%252BInhibitors%25253A%252BDevelopment%252Band%252BBiopharmacological%252BProfiling%252Bof%252B7-%25255B%2525283-Chlorobenzyl%252529oxy%25255D-4-%25255B%252528methylamino%252529methyl%25255D-2H-chromen-2-one%252BMethanesulfonate%252B%252528NW-1772%252529%252Bas%252Ba%252BHighly%252BPotent%25252C%252BSelective%25252C%252BReversible%25252C%252Band%252BOrally%252BActive%252BMonoamine%252BOxidase%252BB%252BInhibitor%26aulast%3DPisani%26aufirst%3DLeonardo%26date%3D2009%26date%3D2009%26date%3D2009%26date%3D09072009%26date%3D07102009%26date%3D12112009%26volume%3D52%26issue%3D21%26spage%3D6685%26epage%3D6706" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Julien  Michel</span>, <span class="hlFld-ContribAuthor ">Julian  Tirado-Rives</span> and <span class="hlFld-ContribAuthor ">William L.  Jorgensen</span>. </span><span class="cited-content_cbyCitation_article-title">Prediction of the Water Content in Protein Binding Sites. </span><span class="cited-content_cbyCitation_journal-name">The Journal of Physical Chemistry B</span><span> <strong>2009,</strong> <em>113 </em>
                                    (40)
                                     , 13337-13346. <a href="https://doi.org/10.1021/jp9047456" title="DOI URL">https://doi.org/10.1021/jp9047456</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jp9047456&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjp9047456%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Journal%2520of%2520Physical%2520Chemistry%2520B%26atitle%3DPrediction%252Bof%252Bthe%252BWater%252BContent%252Bin%252BProtein%252BBinding%252BSites%26aulast%3DMichel%26aufirst%3DJulien%26date%3D2009%26date%3D2009%26date%3D2009%26date%3D2009%26date%3D20%252005%25202009%26date%3D9%252007%25202009%26date%3D15092009%26date%3D08102009%26volume%3D113%26issue%3D40%26spage%3D13337%26epage%3D13346" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">John J.  Irwin</span>, <span class="hlFld-ContribAuthor ">Brian K.  Shoichet</span>, <span class="hlFld-ContribAuthor ">Michael M.  Mysinger</span>, <span class="hlFld-ContribAuthor ">Niu  Huang</span>, <span class="hlFld-ContribAuthor ">Francesco  Colizzi</span>, <span class="hlFld-ContribAuthor ">Pascal  Wassam</span> and <span class="hlFld-ContribAuthor ">Yiqun  Cao</span>  . </span><span class="cited-content_cbyCitation_article-title">Automated Docking Screens: A Feasibility Study. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2009,</strong> <em>52 </em>
                                    (18)
                                     , 5712-5720. <a href="https://doi.org/10.1021/jm9006966" title="DOI URL">https://doi.org/10.1021/jm9006966</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jm9006966&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjm9006966%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DAutomated%252BDocking%252BScreens%25253A%252BA%252BFeasibility%252BStudy%26aulast%3DIrwin%26aufirst%3DJohn%2BJ.%26date%3D2009%26date%3D2009%26date%3D2009%26date%3D22052009%26date%3D31082009%26date%3D24092009%26volume%3D52%26issue%3D18%26spage%3D5712%26epage%3D5720" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Christoph  Gorgulla</span>, <span class="hlFld-ContribAuthor ">Süleyman Selim  Çınaroğlu</span>, <span class="hlFld-ContribAuthor ">Patrick D.  Fischer</span>, <span class="hlFld-ContribAuthor ">Konstantin  Fackeldey</span>, <span class="hlFld-ContribAuthor ">Gerhard  Wagner</span>, <span class="hlFld-ContribAuthor ">Haribabu  Arthanari</span>. </span><span class="cited-content_cbyCitation_article-title">VirtualFlow Ants—Ultra-Large Virtual Screenings with Artificial Intelligence Driven Docking Algorithm Based on Ant Colony Optimization. </span><span class="cited-content_cbyCitation_journal-name">International Journal of Molecular Sciences</span><span> <strong>2021,</strong> <em>22 </em>
                                    (11)
                                     , 5807. <a href="https://doi.org/10.3390/ijms22115807" title="DOI URL">https://doi.org/10.3390/ijms22115807</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/ijms22115807&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fijms22115807%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Molecular%2520Sciences%26atitle%3DVirtualFlow%252BAnts%2525E2%252580%252594Ultra-Large%252BVirtual%252BScreenings%252Bwith%252BArtificial%252BIntelligence%252BDriven%252BDocking%252BAlgorithm%252BBased%252Bon%252BAnt%252BColony%252BOptimization%26aulast%3DGorgulla%26aufirst%3DChristoph%26date%3D2021%26date%3D2021%26volume%3D22%26issue%3D11%26spage%3D5807" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Pierre  Matricon</span>, <span class="hlFld-ContribAuthor ">R. Rama  Suresh</span>, <span class="hlFld-ContribAuthor ">Zhan-Guo  Gao</span>, <span class="hlFld-ContribAuthor ">Nicolas  Panel</span>, <span class="hlFld-ContribAuthor ">Kenneth A.  Jacobson</span>, <span class="hlFld-ContribAuthor ">Jens  Carlsson</span>. </span><span class="cited-content_cbyCitation_article-title">Ligand design by targeting a binding site water. </span><span class="cited-content_cbyCitation_journal-name">Chemical Science</span><span> <strong>2021,</strong> <em>12 </em>
                                    (3)
                                     , 960-968. <a href="https://doi.org/10.1039/D0SC04938G" title="DOI URL">https://doi.org/10.1039/D0SC04938G</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D0SC04938G&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD0SC04938G%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Science%26atitle%3DLigand%252Bdesign%252Bby%252Btargeting%252Ba%252Bbinding%252Bsite%252Bwater%26aulast%3DMatricon%26aufirst%3DPierre%26date%3D2021%26date%3D2021%26volume%3D12%26issue%3D3%26spage%3D960%26epage%3D968" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Om  Silakari</span>, <span class="hlFld-ContribAuthor ">Pankaj Kumar  Singh</span>. </span><span class="cited-content_cbyCitation_article-title">Water mapping: Analysis of binding site spaces to enhance binding. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,, 179-201. <a href="https://doi.org/10.1016/B978-0-12-820546-4.00008-8" title="DOI URL">https://doi.org/10.1016/B978-0-12-820546-4.00008-8</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-820546-4.00008-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-820546-4.00008-8%26sid%3Dliteratum%253Aachs%26atitle%3DWater%252Bmapping%25253A%252BAnalysis%252Bof%252Bbinding%252Bsite%252Bspaces%252Bto%252Benhance%252Bbinding%26aulast%3DSilakari%26aufirst%3DOm%26date%3D2021%26spage%3D179%26epage%3D201%26pub%3DElsevier%26atitle%3DConcepts%252Band%252BExperimental%252BProtocols%252Bof%252BModelling%252Band%252BInformatics%252Bin%252BDrug%252BDesign%26date%3D2021%26volume%3D108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Bruno Junior  Neves</span>, <span class="hlFld-ContribAuthor ">Melina  Mottin</span>, <span class="hlFld-ContribAuthor ">José Teofilo  Moreira-Filho</span>, <span class="hlFld-ContribAuthor ">Bruna Katiele de Paula  Sousa</span>, <span class="hlFld-ContribAuthor ">Sabrina Silva  Mendonca</span>, <span class="hlFld-ContribAuthor ">Carolina Horta  Andrade</span>. </span><span class="cited-content_cbyCitation_article-title">Best Practices for Docking-Based Virtual Screening. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,, 75-98. <a href="https://doi.org/10.1016/B978-0-12-822312-3.00001-1" title="DOI URL">https://doi.org/10.1016/B978-0-12-822312-3.00001-1</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-822312-3.00001-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-822312-3.00001-1%26sid%3Dliteratum%253Aachs%26atitle%3DBest%252BPractices%252Bfor%252BDocking-Based%252BVirtual%252BScreening%26aulast%3DNeves%26aufirst%3DBruno%2BJunior%26date%3D2021%26spage%3D75%26epage%3D98%26pub%3DElsevier%26atitle%3DMolecular%252BDocking%252Bfor%252BComputer-Aided%252BDrug%252BDesign%26date%3D2021%26volume%3D50" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Veronika  Temml</span>, <span class="hlFld-ContribAuthor ">Daniela  Schuster</span>. </span><span class="cited-content_cbyCitation_article-title">Molecular Docking for Natural Product Investigations: Pitfalls and Ways to Overcome Them. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,, 391-405. <a href="https://doi.org/10.1016/B978-0-12-822312-3.00027-8" title="DOI URL">https://doi.org/10.1016/B978-0-12-822312-3.00027-8</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-822312-3.00027-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-822312-3.00027-8%26sid%3Dliteratum%253Aachs%26atitle%3DMolecular%252BDocking%252Bfor%252BNatural%252BProduct%252BInvestigations%25253A%252BPitfalls%252Band%252BWays%252Bto%252BOvercome%252BThem%26aulast%3DTemml%26aufirst%3DVeronika%26date%3D2021%26spage%3D391%26epage%3D405%26pub%3DElsevier%26atitle%3DMolecular%252BDocking%252Bfor%252BComputer-Aided%252BDrug%252BDesign%26date%3D2021%26volume%3D6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Marley L.  Samways</span>, <span class="hlFld-ContribAuthor ">Richard D.  Taylor</span>, <span class="hlFld-ContribAuthor ">Hannah E.  Bruce Macdonald</span>, <span class="hlFld-ContribAuthor ">Jonathan W.  Essex</span>. </span><span class="cited-content_cbyCitation_article-title">Water molecules at protein–drug interfaces: computational prediction and analysis methods. </span><span class="cited-content_cbyCitation_journal-name">Chemical Society Reviews</span><span> <strong>2021,</strong> <em>326 </em><a href="https://doi.org/10.1039/D0CS00151A" title="DOI URL">https://doi.org/10.1039/D0CS00151A</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D0CS00151A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD0CS00151A%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Society%2520Reviews%26atitle%3DWater%252Bmolecules%252Bat%252Bprotein%2525E2%252580%252593drug%252Binterfaces%25253A%252Bcomputational%252Bprediction%252Band%252Banalysis%252Bmethods%26aulast%3DSamways%26aufirst%3DMarley%2BL.%26date%3D2021%26date%3D2021%26volume%3D326" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Antonio  Zandona</span>, <span class="hlFld-ContribAuthor ">Gabriela  Lihtar</span>, <span class="hlFld-ContribAuthor ">Nikola  Maraković</span>, <span class="hlFld-ContribAuthor ">Katarina  Miš</span>, <span class="hlFld-ContribAuthor ">Valentina  Bušić</span>, <span class="hlFld-ContribAuthor ">Dajana  Gašo-Sokač</span>, <span class="hlFld-ContribAuthor ">Sergej  Pirkmajer</span>, <span class="hlFld-ContribAuthor ">Maja  Katalinić</span>. </span><span class="cited-content_cbyCitation_article-title">Vitamin B3-Based Biologically Active Compounds as Inhibitors of Human Cholinesterases. </span><span class="cited-content_cbyCitation_journal-name">International Journal of Molecular Sciences</span><span> <strong>2020,</strong> <em>21 </em>
                                    (21)
                                     , 8088. <a href="https://doi.org/10.3390/ijms21218088" title="DOI URL">https://doi.org/10.3390/ijms21218088</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/ijms21218088&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fijms21218088%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Molecular%2520Sciences%26atitle%3DVitamin%252BB3-Based%252BBiologically%252BActive%252BCompounds%252Bas%252BInhibitors%252Bof%252BHuman%252BCholinesterases%26aulast%3DZandona%26aufirst%3DAntonio%26date%3D2020%26date%3D2020%26volume%3D21%26issue%3D21%26spage%3D8088" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Qiangna  Lu</span>, <span class="hlFld-ContribAuthor ">Lian-Wen  Qi</span>, <span class="hlFld-ContribAuthor ">Jinfeng  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Improving protein–ligand binding prediction by considering the bridging water molecules in Autodock. </span><span class="cited-content_cbyCitation_journal-name">Journal of Theoretical and Computational Chemistry</span><span> <strong>2019,</strong> <em>18 </em>
                                    (05)
                                     , 1950027. <a href="https://doi.org/10.1142/S0219633619500275" title="DOI URL">https://doi.org/10.1142/S0219633619500275</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1142/S0219633619500275&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1142%2FS0219633619500275%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Theoretical%2520and%2520Computational%2520Chemistry%26atitle%3DImproving%252Bprotein%2525E2%252580%252593ligand%252Bbinding%252Bprediction%252Bby%252Bconsidering%252Bthe%252Bbridging%252Bwater%252Bmolecules%252Bin%252BAutodock%26aulast%3DLu%26aufirst%3DQiangna%26date%3D2019%26date%3D2019%26volume%3D18%26issue%3D05%26spage%3D1950027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Olivia  Slater</span>, <span class="hlFld-ContribAuthor ">Maria  Kontoyianni</span>. </span><span class="cited-content_cbyCitation_article-title">The compromise of virtual screening and its impact on drug discovery. </span><span class="cited-content_cbyCitation_journal-name">Expert Opinion on Drug Discovery</span><span> <strong>2019,</strong> <em>14 </em>
                                    (7)
                                     , 619-637. <a href="https://doi.org/10.1080/17460441.2019.1604677" title="DOI URL">https://doi.org/10.1080/17460441.2019.1604677</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/17460441.2019.1604677&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F17460441.2019.1604677%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Opinion%2520on%2520Drug%2520Discovery%26atitle%3DThe%252Bcompromise%252Bof%252Bvirtual%252Bscreening%252Band%252Bits%252Bimpact%252Bon%252Bdrug%252Bdiscovery%26aulast%3DSlater%26aufirst%3DOlivia%26date%3D2019%26date%3D2019%26volume%3D14%26issue%3D7%26spage%3D619%26epage%3D637" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Aditi  Gangopadhyay</span>, <span class="hlFld-ContribAuthor ">Hirak Jyoti  Chakraborty</span>, <span class="hlFld-ContribAuthor ">Abhijit  Datta</span>. </span><span class="cited-content_cbyCitation_article-title">Protein Docking and Drug Design. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 889-922. <a href="https://doi.org/10.4018/978-1-5225-8903-7.ch035" title="DOI URL">https://doi.org/10.4018/978-1-5225-8903-7.ch035</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4018/978-1-5225-8903-7.ch035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4018%2F978-1-5225-8903-7.ch035%26sid%3Dliteratum%253Aachs%26atitle%3DProtein%252BDocking%252Band%252BDrug%252BDesign%26aulast%3DGangopadhyay%26aufirst%3DAditi%26date%3D2019%26spage%3D889%26epage%3D922%26pub%3DIGI%2520Global%26atitle%3DBiotechnology%26aulast%3DManagement%2BAssociation%26aufirst%3DInformation%2BResources%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yuanxun  Wang</span>, <span class="hlFld-ContribAuthor ">Yunfei  Du</span>, <span class="hlFld-ContribAuthor ">Niu  Huang</span>. </span><span class="cited-content_cbyCitation_article-title">A survey of the role of nitrile groups in protein–ligand interactions. </span><span class="cited-content_cbyCitation_journal-name">Future Medicinal Chemistry</span><span> <strong>2018,</strong> <em>10 </em>
                                    (23)
                                     , 2713-2728. <a href="https://doi.org/10.4155/fmc-2018-0252" title="DOI URL">https://doi.org/10.4155/fmc-2018-0252</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fmc-2018-0252&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffmc-2018-0252%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Medicinal%2520Chemistry%26atitle%3DA%252Bsurvey%252Bof%252Bthe%252Brole%252Bof%252Bnitrile%252Bgroups%252Bin%252Bprotein%2525E2%252580%252593ligand%252Binteractions%26aulast%3DWang%26aufirst%3DYuanxun%26date%3D2018%26volume%3D10%26issue%3D23%26spage%3D2713%26epage%3D2728" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Wei  Xiao</span>, <span class="hlFld-ContribAuthor ">Disha  Wang</span>, <span class="hlFld-ContribAuthor ">Zihao  Shen</span>, <span class="hlFld-ContribAuthor ">Shiliang  Li</span>, <span class="hlFld-ContribAuthor ">Honglin  Li</span>. </span><span class="cited-content_cbyCitation_article-title">Multi-Body Interactions in Molecular Docking Program Devised with Key Water Molecules in Protein Binding Sites. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2018,</strong> <em>23 </em>
                                    (9)
                                     , 2321. <a href="https://doi.org/10.3390/molecules23092321" title="DOI URL">https://doi.org/10.3390/molecules23092321</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules23092321&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules23092321%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DMulti-Body%252BInteractions%252Bin%252BMolecular%252BDocking%252BProgram%252BDevised%252Bwith%252BKey%252BWater%252BMolecules%252Bin%252BProtein%252BBinding%252BSites%26aulast%3DXiao%26aufirst%3DWei%26date%3D2018%26date%3D2018%26volume%3D23%26issue%3D9%26spage%3D2321" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Prabu  Manoharan</span>, <span class="hlFld-ContribAuthor ">Kiranmai  Chennoju</span>, <span class="hlFld-ContribAuthor ">Nanda  Ghoshal</span>. </span><span class="cited-content_cbyCitation_article-title">Computational analysis of BACE1-ligand complex crystal structures and linear discriminant analysis for identification of BACE1 inhibitors with anti P-glycoprotein binding property. </span><span class="cited-content_cbyCitation_journal-name">Journal of Biomolecular Structure and Dynamics</span><span> <strong>2018,</strong> <em>36 </em>
                                    (1)
                                     , 262-276. <a href="https://doi.org/10.1080/07391102.2016.1276477" title="DOI URL">https://doi.org/10.1080/07391102.2016.1276477</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/07391102.2016.1276477&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F07391102.2016.1276477%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Biomolecular%2520Structure%2520and%2520Dynamics%26atitle%3DComputational%252Banalysis%252Bof%252BBACE1-ligand%252Bcomplex%252Bcrystal%252Bstructures%252Band%252Blinear%252Bdiscriminant%252Banalysis%252Bfor%252Bidentification%252Bof%252BBACE1%252Binhibitors%252Bwith%252Banti%252BP-glycoprotein%252Bbinding%252Bproperty%26aulast%3DManoharan%26aufirst%3DPrabu%26date%3D2018%26date%3D2017%26volume%3D36%26issue%3D1%26spage%3D262%26epage%3D276" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Aditi  Gangopadhyay</span>, <span class="hlFld-ContribAuthor ">Hirak Jyoti  Chakraborty</span>, <span class="hlFld-ContribAuthor ">Abhijit  Datta</span>. </span><span class="cited-content_cbyCitation_article-title">Protein Docking and Drug Design. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2018,</strong>,, 207-241. <a href="https://doi.org/10.4018/978-1-5225-2607-0.ch009" title="DOI URL">https://doi.org/10.4018/978-1-5225-2607-0.ch009</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4018/978-1-5225-2607-0.ch009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4018%2F978-1-5225-2607-0.ch009%26sid%3Dliteratum%253Aachs%26atitle%3DProtein%252BDocking%252Band%252BDrug%252BDesign%26aulast%3DGangopadhyay%26aufirst%3DAditi%26date%3D2018%26spage%3D207%26epage%3D241%26pub%3DIGI%2520Global%26atitle%3DApplying%252BBig%252BData%252BAnalytics%252Bin%252BBioinformatics%252Band%252BMedicine%26aulast%3DLytras%26aufirst%3DMiltiadis%2BD.%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Trent E.  Balius</span>, <span class="hlFld-ContribAuthor ">Marcus  Fischer</span>, <span class="hlFld-ContribAuthor ">Reed M.  Stein</span>, <span class="hlFld-ContribAuthor ">Thomas B.  Adler</span>, <span class="hlFld-ContribAuthor ">Crystal N.  Nguyen</span>, <span class="hlFld-ContribAuthor ">Anthony  Cruz</span>, <span class="hlFld-ContribAuthor ">Michael K.  Gilson</span>, <span class="hlFld-ContribAuthor ">Tom  Kurtzman</span>, <span class="hlFld-ContribAuthor ">Brian K.  Shoichet</span>. </span><span class="cited-content_cbyCitation_article-title">Testing inhomogeneous solvation theory in structure-based ligand discovery. </span><span class="cited-content_cbyCitation_journal-name">Proceedings of the National Academy of Sciences</span><span> <strong>2017,</strong> <em>114 </em>
                                    (33)
                                     , E6839-E6846. <a href="https://doi.org/10.1073/pnas.1703287114" title="DOI URL">https://doi.org/10.1073/pnas.1703287114</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1073/pnas.1703287114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1073%2Fpnas.1703287114%26sid%3Dliteratum%253Aachs%26jtitle%3DProceedings%2520of%2520the%2520National%2520Academy%2520of%2520Sciences%26atitle%3DTesting%252Binhomogeneous%252Bsolvation%252Btheory%252Bin%252Bstructure-based%252Bligand%252Bdiscovery%26aulast%3DBalius%26aufirst%3DTrent%2BE.%26date%3D2017%26date%3D2017%26volume%3D114%26issue%3D33%26spage%3DE6839%26epage%3DE6846" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Akshay  Sridhar</span>, <span class="hlFld-ContribAuthor ">Gregory A.  Ross</span>, <span class="hlFld-ContribAuthor ">Philip C.  Biggin</span>, . </span><span class="cited-content_cbyCitation_article-title">Waterdock 2.0: Water placement prediction for Holo-structures with a pymol plugin. </span><span class="cited-content_cbyCitation_journal-name">PLOS ONE</span><span> <strong>2017,</strong> <em>12 </em>
                                    (2)
                                     , e0172743. <a href="https://doi.org/10.1371/journal.pone.0172743" title="DOI URL">https://doi.org/10.1371/journal.pone.0172743</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1371/journal.pone.0172743&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1371%2Fjournal.pone.0172743%26sid%3Dliteratum%253Aachs%26jtitle%3DPLOS%2520ONE%26atitle%3DWaterdock%252B2.0%25253A%252BWater%252Bplacement%252Bprediction%252Bfor%252BHolo-structures%252Bwith%252Ba%252Bpymol%252Bplugin%26aulast%3DSridhar%26aufirst%3DAkshay%26date%3D2017%26date%3D2017%26volume%3D12%26issue%3D2%26spage%3De0172743" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Damian  Bartuzi</span>, <span class="hlFld-ContribAuthor ">Agnieszka  Kaczor</span>, <span class="hlFld-ContribAuthor ">Katarzyna  Targowska-Duda</span>, <span class="hlFld-ContribAuthor ">Dariusz  Matosiuk</span>. </span><span class="cited-content_cbyCitation_article-title">Recent Advances and Applications of Molecular Docking to G Protein-Coupled Receptors. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2017,</strong> <em>22 </em>
                                    (2)
                                     , 340. <a href="https://doi.org/10.3390/molecules22020340" title="DOI URL">https://doi.org/10.3390/molecules22020340</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules22020340&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules22020340%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DRecent%252BAdvances%252Band%252BApplications%252Bof%252BMolecular%252BDocking%252Bto%252BG%252BProtein-Coupled%252BReceptors%26aulast%3DBartuzi%26aufirst%3DDamian%26date%3D2017%26date%3D2017%26volume%3D22%26issue%3D2%26spage%3D340" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jahan B.  Ghasemi</span>, <span class="hlFld-ContribAuthor ">Azizeh  Abdolmaleki</span>, <span class="hlFld-ContribAuthor ">Fereshteh  Shiri</span>. </span><span class="cited-content_cbyCitation_article-title">Molecular Docking Challenges and Limitations. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2017,</strong>,, 770-794. <a href="https://doi.org/10.4018/978-1-5225-1762-7.ch030" title="DOI URL">https://doi.org/10.4018/978-1-5225-1762-7.ch030</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4018/978-1-5225-1762-7.ch030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4018%2F978-1-5225-1762-7.ch030%26sid%3Dliteratum%253Aachs%26atitle%3DMolecular%252BDocking%252BChallenges%252Band%252BLimitations%26aulast%3DGhasemi%26aufirst%3DJahan%2BB.%26date%3D2017%26spage%3D770%26epage%3D794%26pub%3DIGI%2520Global%26atitle%3DPharmaceutical%252BSciences%26aulast%3DManagement%2BAssociation%26aufirst%3DInformation%2BResources%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shota  Uehara</span>, <span class="hlFld-ContribAuthor ">Shigenori  Tanaka</span>. </span><span class="cited-content_cbyCitation_article-title">AutoDock-GIST: Incorporating Thermodynamics of Active-Site Water into Scoring Function for Accurate Protein-Ligand Docking. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2016,</strong> <em>21 </em>
                                    (11)
                                     , 1604. <a href="https://doi.org/10.3390/molecules21111604" title="DOI URL">https://doi.org/10.3390/molecules21111604</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules21111604&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules21111604%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DAutoDock-GIST%25253A%252BIncorporating%252BThermodynamics%252Bof%252BActive-Site%252BWater%252Binto%252BScoring%252BFunction%252Bfor%252BAccurate%252BProtein-Ligand%252BDocking%26aulast%3DUehara%26aufirst%3DShota%26date%3D2016%26date%3D2016%26volume%3D21%26issue%3D11%26spage%3D1604" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ganesh Prasad  Mishra</span>, <span class="hlFld-ContribAuthor ">Rajesh  Sharma</span>. </span><span class="cited-content_cbyCitation_article-title">Identification of Potential PPAR γ Agonists as Hypoglycemic Agents: Molecular Docking Approach. </span><span class="cited-content_cbyCitation_journal-name">Interdisciplinary Sciences: Computational Life Sciences</span><span> <strong>2016,</strong> <em>8 </em>
                                    (3)
                                     , 220-228. <a href="https://doi.org/10.1007/s12539-015-0126-7" title="DOI URL">https://doi.org/10.1007/s12539-015-0126-7</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s12539-015-0126-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs12539-015-0126-7%26sid%3Dliteratum%253Aachs%26jtitle%3DInterdisciplinary%2520Sciences%253A%2520Computational%2520Life%2520Sciences%26atitle%3DIdentification%252Bof%252BPotential%252BPPAR%252B%2525CE%2525B3%252BAgonists%252Bas%252BHypoglycemic%252BAgents%25253A%252BMolecular%252BDocking%252BApproach%26aulast%3DMishra%26aufirst%3DGanesh%2BPrasad%26date%3D2016%26date%3D2015%26volume%3D8%26issue%3D3%26spage%3D220%26epage%3D228" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ran  Cao</span>. </span><span class="cited-content_cbyCitation_article-title">Free energy calculation provides insight into the action mechanism of selective PARP-1 inhibitor. </span><span class="cited-content_cbyCitation_journal-name">Journal of Molecular Modeling</span><span> <strong>2016,</strong> <em>22 </em>
                                    (4)
                                     <a href="https://doi.org/10.1007/s00894-016-2952-x" title="DOI URL">https://doi.org/10.1007/s00894-016-2952-x</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s00894-016-2952-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs00894-016-2952-x%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Molecular%2520Modeling%26atitle%3DFree%252Benergy%252Bcalculation%252Bprovides%252Binsight%252Binto%252Bthe%252Baction%252Bmechanism%252Bof%252Bselective%252BPARP-1%252Binhibitor%26aulast%3DCao%26aufirst%3DRan%26date%3D2016%26date%3D2016%26volume%3D22%26issue%3D4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jahan B.  Ghasemi</span>, <span class="hlFld-ContribAuthor ">Azizeh  Abdolmaleki</span>, <span class="hlFld-ContribAuthor ">Fereshteh  Shiri</span>. </span><span class="cited-content_cbyCitation_article-title">Molecular Docking Challenges and Limitations. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2016,</strong>,, 56-80. <a href="https://doi.org/10.4018/978-1-5225-0362-0.ch003" title="DOI URL">https://doi.org/10.4018/978-1-5225-0362-0.ch003</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4018/978-1-5225-0362-0.ch003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4018%2F978-1-5225-0362-0.ch003%26sid%3Dliteratum%253Aachs%26atitle%3DMolecular%252BDocking%252BChallenges%252Band%252BLimitations%26aulast%3DGhasemi%26aufirst%3DJahan%2BB.%26date%3D2016%26spage%3D56%26epage%3D80%26pub%3DIGI%2520Global%26atitle%3DApplied%252BCase%252BStudies%252Band%252BSolutions%252Bin%252BMolecular%252BDocking-Based%252BDrug%252BDesign%26aulast%3DDastmalchi%26aufirst%3DSiavoush%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Francesca  Spyrakis</span>, <span class="hlFld-ContribAuthor ">Claudio N.  Cavasotto</span>. </span><span class="cited-content_cbyCitation_article-title">Open challenges in structure-based virtual screening: Receptor modeling, target flexibility consideration and active site water molecules description. </span><span class="cited-content_cbyCitation_journal-name">Archives of Biochemistry and Biophysics</span><span> <strong>2015,</strong> <em>583 </em>, 105-119. <a href="https://doi.org/10.1016/j.abb.2015.08.002" title="DOI URL">https://doi.org/10.1016/j.abb.2015.08.002</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.abb.2015.08.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.abb.2015.08.002%26sid%3Dliteratum%253Aachs%26jtitle%3DArchives%2520of%2520Biochemistry%2520and%2520Biophysics%26atitle%3DOpen%252Bchallenges%252Bin%252Bstructure-based%252Bvirtual%252Bscreening%25253A%252BReceptor%252Bmodeling%25252C%252Btarget%252Bflexibility%252Bconsideration%252Band%252Bactive%252Bsite%252Bwater%252Bmolecules%252Bdescription%26aulast%3DSpyrakis%26aufirst%3DFrancesca%26date%3D2015%26volume%3D583%26spage%3D105%26epage%3D119" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Stefano  Forli</span>. </span><span class="cited-content_cbyCitation_article-title">Charting a Path to Success in Virtual Screening. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2015,</strong> <em>20 </em>
                                    (10)
                                     , 18732-18758. <a href="https://doi.org/10.3390/molecules201018732" title="DOI URL">https://doi.org/10.3390/molecules201018732</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules201018732&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules201018732%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DCharting%252Ba%252BPath%252Bto%252BSuccess%252Bin%252BVirtual%252BScreening%26aulast%3DForli%26aufirst%3DStefano%26date%3D2015%26date%3D2015%26volume%3D20%26issue%3D10%26spage%3D18732%26epage%3D18758" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">William J.  Allen</span>, <span class="hlFld-ContribAuthor ">Trent E.  Balius</span>, <span class="hlFld-ContribAuthor ">Sudipto  Mukherjee</span>, <span class="hlFld-ContribAuthor ">Scott R.  Brozell</span>, <span class="hlFld-ContribAuthor ">Demetri T.  Moustakas</span>, <span class="hlFld-ContribAuthor ">P. Therese  Lang</span>, <span class="hlFld-ContribAuthor ">David A.  Case</span>, <span class="hlFld-ContribAuthor ">Irwin D.  Kuntz</span>, <span class="hlFld-ContribAuthor ">Robert C.  Rizzo</span>. </span><span class="cited-content_cbyCitation_article-title">DOCK 6: Impact of new features and current docking performance. </span><span class="cited-content_cbyCitation_journal-name">Journal of Computational Chemistry</span><span> <strong>2015,</strong> <em>36 </em>
                                    (15)
                                     , 1132-1156. <a href="https://doi.org/10.1002/jcc.23905" title="DOI URL">https://doi.org/10.1002/jcc.23905</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/jcc.23905&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fjcc.23905%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Computational%2520Chemistry%26atitle%3DDOCK%252B6%25253A%252BImpact%252Bof%252Bnew%252Bfeatures%252Band%252Bcurrent%252Bdocking%252Bperformance%26aulast%3DAllen%26aufirst%3DWilliam%2BJ.%26date%3D2015%26date%3D2015%26volume%3D36%26issue%3D15%26spage%3D1132%26epage%3D1156" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Elke  Persch</span>, <span class="hlFld-ContribAuthor ">Oliver  Dumele</span>, <span class="hlFld-ContribAuthor ">François  Diederich</span>. </span><span class="cited-content_cbyCitation_article-title">Molekulare Erkennung in chemischen und biologischen Systemen. </span><span class="cited-content_cbyCitation_journal-name">Angewandte Chemie</span><span> <strong>2015,</strong> <em>127 </em>
                                    (11)
                                     , 3341-3382. <a href="https://doi.org/10.1002/ange.201408487" title="DOI URL">https://doi.org/10.1002/ange.201408487</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/ange.201408487&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fange.201408487%26sid%3Dliteratum%253Aachs%26jtitle%3DAngewandte%2520Chemie%26atitle%3DMolekulare%252BErkennung%252Bin%252Bchemischen%252Bund%252Bbiologischen%252BSystemen%26aulast%3DPersch%26aufirst%3DElke%26date%3D2015%26date%3D2015%26volume%3D127%26issue%3D11%26spage%3D3341%26epage%3D3382" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Elke  Persch</span>, <span class="hlFld-ContribAuthor ">Oliver  Dumele</span>, <span class="hlFld-ContribAuthor ">François  Diederich</span>. </span><span class="cited-content_cbyCitation_article-title">Molecular Recognition in Chemical and Biological Systems. </span><span class="cited-content_cbyCitation_journal-name">Angewandte Chemie International Edition</span><span> <strong>2015,</strong> <em>54 </em>
                                    (11)
                                     , 3290-3327. <a href="https://doi.org/10.1002/anie.201408487" title="DOI URL">https://doi.org/10.1002/anie.201408487</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/anie.201408487&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fanie.201408487%26sid%3Dliteratum%253Aachs%26jtitle%3DAngewandte%2520Chemie%2520International%2520Edition%26atitle%3DMolecular%252BRecognition%252Bin%252BChemical%252Band%252BBiological%252BSystems%26aulast%3DPersch%26aufirst%3DElke%26date%3D2015%26date%3D2015%26volume%3D54%26issue%3D11%26spage%3D3290%26epage%3D3327" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Michael  Berry</span>, <span class="hlFld-ContribAuthor ">Burtram  Fielding</span>, <span class="hlFld-ContribAuthor ">Junaid  Gamieldien</span>. </span><span class="cited-content_cbyCitation_article-title">Practical Considerations in Virtual Screening and Molecular Docking. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2015,</strong>,, 487-502. <a href="https://doi.org/10.1016/B978-0-12-802508-6.00027-2" title="DOI URL">https://doi.org/10.1016/B978-0-12-802508-6.00027-2</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-802508-6.00027-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-802508-6.00027-2%26sid%3Dliteratum%253Aachs%26atitle%3DPractical%252BConsiderations%252Bin%252BVirtual%252BScreening%252Band%252BMolecular%252BDocking%26aulast%3DBerry%26aufirst%3DMichael%26date%3D2015%26spage%3D487%26epage%3D502%26pub%3DElsevier%26atitle%3DEmerging%252BTrends%252Bin%252BComputational%252BBiology%25252C%252BBioinformatics%25252C%252Band%252BSystems%252BBiology%26date%3D2015%26volume%3D26" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">G.  Gerogiokas</span>, <span class="hlFld-ContribAuthor ">M. W. Y.  Southey</span>, <span class="hlFld-ContribAuthor ">M. P.  Mazanetz</span>, <span class="hlFld-ContribAuthor ">A.  Hefeitz</span>, <span class="hlFld-ContribAuthor ">M.  Bodkin</span>, <span class="hlFld-ContribAuthor ">R. J.  Law</span>, <span class="hlFld-ContribAuthor ">J.  Michel</span>. </span><span class="cited-content_cbyCitation_article-title">Evaluation of water displacement energetics in protein binding sites with grid cell theory. </span><span class="cited-content_cbyCitation_journal-name">Physical Chemistry Chemical Physics</span><span> <strong>2015,</strong> <em>17 </em>
                                    (13)
                                     , 8416-8426. <a href="https://doi.org/10.1039/C4CP05572A" title="DOI URL">https://doi.org/10.1039/C4CP05572A</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C4CP05572A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC4CP05572A%26sid%3Dliteratum%253Aachs%26jtitle%3DPhysical%2520Chemistry%2520Chemical%2520Physics%26atitle%3DEvaluation%252Bof%252Bwater%252Bdisplacement%252Benergetics%252Bin%252Bprotein%252Bbinding%252Bsites%252Bwith%252Bgrid%252Bcell%252Btheory%26aulast%3DGerogiokas%26aufirst%3DG.%26date%3D2015%26date%3D2015%26volume%3D17%26issue%3D13%26spage%3D8416%26epage%3D8426" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Prabu  Manoharan</span>, <span class="hlFld-ContribAuthor ">Kiranmai  Chennoju</span>, <span class="hlFld-ContribAuthor ">Nanda  Ghoshal</span>. </span><span class="cited-content_cbyCitation_article-title">Target specific proteochemometric model development for BACE1 – protein flexibility and structural water are critical in virtual screening. </span><span class="cited-content_cbyCitation_journal-name">Molecular BioSystems</span><span> <strong>2015,</strong> <em>11 </em>
                                    (7)
                                     , 1955-1972. <a href="https://doi.org/10.1039/C5MB00088B" title="DOI URL">https://doi.org/10.1039/C5MB00088B</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C5MB00088B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC5MB00088B%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520BioSystems%26atitle%3DTarget%252Bspecific%252Bproteochemometric%252Bmodel%252Bdevelopment%252Bfor%252BBACE1%252B%2525E2%252580%252593%252Bprotein%252Bflexibility%252Band%252Bstructural%252Bwater%252Bare%252Bcritical%252Bin%252Bvirtual%252Bscreening%26aulast%3DManoharan%26aufirst%3DPrabu%26date%3D2015%26date%3D2015%26volume%3D11%26issue%3D7%26spage%3D1955%26epage%3D1972" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hanzi  Sun</span>, <span class="hlFld-ContribAuthor ">Lifeng  Zhao</span>, <span class="hlFld-ContribAuthor ">Shiming  Peng</span>, <span class="hlFld-ContribAuthor ">Niu  Huang</span>. </span><span class="cited-content_cbyCitation_article-title">Incorporating replacement free energy of binding-site waters in molecular docking. </span><span class="cited-content_cbyCitation_journal-name">Proteins: Structure, Function, and Bioinformatics</span><span> <strong>2014,</strong> <em>82 </em>
                                    (9)
                                     , 1765-1776. <a href="https://doi.org/10.1002/prot.24530" title="DOI URL">https://doi.org/10.1002/prot.24530</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/prot.24530&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fprot.24530%26sid%3Dliteratum%253Aachs%26jtitle%3DProteins%253A%2520Structure%252C%2520Function%252C%2520and%2520Bioinformatics%26atitle%3DIncorporating%252Breplacement%252Bfree%252Benergy%252Bof%252Bbinding-site%252Bwaters%252Bin%252Bmolecular%252Bdocking%26aulast%3DSun%26aufirst%3DHanzi%26date%3D2014%26date%3D2014%26volume%3D82%26issue%3D9%26spage%3D1765%26epage%3D1776" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xian-qiang  Sun</span>, <span class="hlFld-ContribAuthor ">Lei  Chen</span>, <span class="hlFld-ContribAuthor ">Yao-zong  Li</span>, <span class="hlFld-ContribAuthor ">Wei-hua  Li</span>, <span class="hlFld-ContribAuthor ">Gui-xia  Liu</span>, <span class="hlFld-ContribAuthor ">Yao-quan  Tu</span>, <span class="hlFld-ContribAuthor ">Yun  Tang</span>. </span><span class="cited-content_cbyCitation_article-title">Structure-based ensemble-QSAR model: a novel approach to the study of the EGFR tyrosine kinase and its inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Acta Pharmacologica Sinica</span><span> <strong>2014,</strong> <em>35 </em>
                                    (2)
                                     , 301-310. <a href="https://doi.org/10.1038/aps.2013.148" title="DOI URL">https://doi.org/10.1038/aps.2013.148</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/aps.2013.148&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Faps.2013.148%26sid%3Dliteratum%253Aachs%26jtitle%3DActa%2520Pharmacologica%2520Sinica%26atitle%3DStructure-based%252Bensemble-QSAR%252Bmodel%25253A%252Ba%252Bnovel%252Bapproach%252Bto%252Bthe%252Bstudy%252Bof%252Bthe%252BEGFR%252Btyrosine%252Bkinase%252Band%252Bits%252Binhibitors%26aulast%3DSun%26aufirst%3DXian-qiang%26date%3D2014%26date%3D2013%26volume%3D35%26issue%3D2%26spage%3D301%26epage%3D310" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Martin  Frank</span>. </span><span class="cited-content_cbyCitation_article-title">Computational Docking as a Tool for the Rational Design of Carbohydrate-Based Drugs. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2014,</strong>,, 53-72. <a href="https://doi.org/10.1007/7355_2014_42" title="DOI URL">https://doi.org/10.1007/7355_2014_42</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/7355_2014_42&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F7355_2014_42%26sid%3Dliteratum%253Aachs%26atitle%3DComputational%252BDocking%252Bas%252Ba%252BTool%252Bfor%252Bthe%252BRational%252BDesign%252Bof%252BCarbohydrate-Based%252BDrugs%26aulast%3DFrank%26aufirst%3DMartin%26date%3D2014%26date%3D2014%26spage%3D53%26epage%3D72%26pub%3DSpringer%2520International%2520Publishing%26atitle%3DCarbohydrates%252Bas%252BDrugs%26aulast%3DSeeberger%26aufirst%3DPeter%2BH.%26date%3D2014%26volume%3D12" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">N. F.  Brás</span>, <span class="hlFld-ContribAuthor ">N. M. F. S. A.  Cerqueira</span>, <span class="hlFld-ContribAuthor ">S. F.  Sousa</span>, <span class="hlFld-ContribAuthor ">P. A.  Fernandes</span>, <span class="hlFld-ContribAuthor ">M. J.  Ramos</span>. </span><span class="cited-content_cbyCitation_article-title">Protein Ligand DockingDocking in Drug DiscoveryDrug Discovery. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2014,</strong>,, 249-286. <a href="https://doi.org/10.1007/978-3-319-09976-7_11" title="DOI URL">https://doi.org/10.1007/978-3-319-09976-7_11</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-3-319-09976-7_11&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-3-319-09976-7_11%26sid%3Dliteratum%253Aachs%26atitle%3DProtein%252BLigand%252BDockingDocking%252Bin%252BDrug%252BDiscoveryDrug%252BDiscovery%26aulast%3DBr%25C3%25A1s%26aufirst%3DN.%2BF.%26date%3D2014%26date%3D2014%26spage%3D249%26epage%3D286%26pub%3DSpringer%2520International%2520Publishing%26atitle%3DProtein%252BModelling%26aulast%3DN%25C3%25A1ray-Szab%25C3%25B3%26aufirst%3DG%25C3%25A1bor%26date%3D2014%26volume%3D13" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Darren  Fayne</span>. </span><span class="cited-content_cbyCitation_article-title">De-peptidising protein–protein interactions – big jobs for small molecules. </span><span class="cited-content_cbyCitation_journal-name">Drug Discovery Today: Technologies</span><span> <strong>2013,</strong> <em>10 </em>
                                    (4)
                                     , e467-e474. <a href="https://doi.org/10.1016/j.ddtec.2013.08.002" title="DOI URL">https://doi.org/10.1016/j.ddtec.2013.08.002</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ddtec.2013.08.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ddtec.2013.08.002%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Technologies%26atitle%3DDe-peptidising%252Bprotein%2525E2%252580%252593protein%252Binteractions%252B%2525E2%252580%252593%252Bbig%252Bjobs%252Bfor%252Bsmall%252Bmolecules%26aulast%3DFayne%26aufirst%3DDarren%26date%3D2013%26volume%3D10%26issue%3D4%26spage%3De467%26epage%3De474" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Thomas  Fischer</span>, <span class="hlFld-ContribAuthor ">Rainer  Riedl</span>. </span><span class="cited-content_cbyCitation_article-title">Strategic Targeting of Multiple Water‐Mediated Interactions: A Concise and Rational Structure‐Based Design Approach to Potent and Selective MMP‐13 Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2013,</strong> <em>8 </em>
                                    (9)
                                     , 1457-1461. <a href="https://doi.org/10.1002/cmdc.201300278" title="DOI URL">https://doi.org/10.1002/cmdc.201300278</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.201300278&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.201300278%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DStrategic%252BTargeting%252Bof%252BMultiple%252BWater%2525E2%252580%252590Mediated%252BInteractions%25253A%252BA%252BConcise%252Band%252BRational%252BStructure%2525E2%252580%252590Based%252BDesign%252BApproach%252Bto%252BPotent%252Band%252BSelective%252BMMP%2525E2%252580%25259013%252BInhibitors%26aulast%3DFischer%26aufirst%3DThomas%26date%3D2013%26date%3D2013%26volume%3D8%26issue%3D9%26spage%3D1457%26epage%3D1461" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Marc  van Dijk</span>, <span class="hlFld-ContribAuthor ">Koen M.  Visscher</span>, <span class="hlFld-ContribAuthor ">Panagiotis L.  Kastritis</span>, <span class="hlFld-ContribAuthor ">Alexandre M. J. J.  Bonvin</span>. </span><span class="cited-content_cbyCitation_article-title">Solvated protein–DNA docking using HADDOCK. </span><span class="cited-content_cbyCitation_journal-name">Journal of Biomolecular NMR</span><span> <strong>2013,</strong> <em>56 </em>
                                    (1)
                                     , 51-63. <a href="https://doi.org/10.1007/s10858-013-9734-x" title="DOI URL">https://doi.org/10.1007/s10858-013-9734-x</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s10858-013-9734-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs10858-013-9734-x%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Biomolecular%2520NMR%26atitle%3DSolvated%252Bprotein%2525E2%252580%252593DNA%252Bdocking%252Busing%252BHADDOCK%26aulast%3Dvan%2BDijk%26aufirst%3DMarc%26date%3D2013%26date%3D2013%26volume%3D56%26issue%3D1%26spage%3D51%26epage%3D63" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">G.  Madhavi Sastry</span>, <span class="hlFld-ContribAuthor ">Matvey  Adzhigirey</span>, <span class="hlFld-ContribAuthor ">Tyler  Day</span>, <span class="hlFld-ContribAuthor ">Ramakrishna  Annabhimoju</span>, <span class="hlFld-ContribAuthor ">Woody  Sherman</span>. </span><span class="cited-content_cbyCitation_article-title">Protein and ligand preparation: parameters, protocols, and influence on virtual screening enrichments. </span><span class="cited-content_cbyCitation_journal-name">Journal of Computer-Aided Molecular Design</span><span> <strong>2013,</strong> <em>27 </em>
                                    (3)
                                     , 221-234. <a href="https://doi.org/10.1007/s10822-013-9644-8" title="DOI URL">https://doi.org/10.1007/s10822-013-9644-8</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s10822-013-9644-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs10822-013-9644-8%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Computer-Aided%2520Molecular%2520Design%26atitle%3DProtein%252Band%252Bligand%252Bpreparation%25253A%252Bparameters%25252C%252Bprotocols%25252C%252Band%252Binfluence%252Bon%252Bvirtual%252Bscreening%252Benrichments%26aulast%3DMadhavi%2BSastry%26aufirst%3DG.%26date%3D2013%26date%3D2013%26volume%3D27%26issue%3D3%26spage%3D221%26epage%3D234" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hemant V.  Chavan</span>, <span class="hlFld-ContribAuthor ">Babasaheb P.  Bandgar</span>, <span class="hlFld-ContribAuthor ">Laxman K.  Adsul</span>, <span class="hlFld-ContribAuthor ">Valmik D.  Dhakane</span>, <span class="hlFld-ContribAuthor ">Pravin S.  Bhale</span>, <span class="hlFld-ContribAuthor ">Vishnu N.  Thakare</span>, <span class="hlFld-ContribAuthor ">Vijay  Masand</span>. </span><span class="cited-content_cbyCitation_article-title">Design, synthesis, characterization and anti-inflammatory evaluation of novel pyrazole amalgamated flavones. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2013,</strong> <em>23 </em>
                                    (5)
                                     , 1315-1321. <a href="https://doi.org/10.1016/j.bmcl.2012.12.094" title="DOI URL">https://doi.org/10.1016/j.bmcl.2012.12.094</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2012.12.094&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2012.12.094%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DDesign%25252C%252Bsynthesis%25252C%252Bcharacterization%252Band%252Banti-inflammatory%252Bevaluation%252Bof%252Bnovel%252Bpyrazole%252Bamalgamated%252Bflavones%26aulast%3DChavan%26aufirst%3DHemant%2BV.%26date%3D2013%26volume%3D23%26issue%3D5%26spage%3D1315%26epage%3D1321" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Napat  Songtawee</span>, <span class="hlFld-ContribAuthor ">M. Paul  Gleeson</span>, <span class="hlFld-ContribAuthor ">Kiattawee  Choowongkomon</span>. </span><span class="cited-content_cbyCitation_article-title">Computational study of EGFR inhibition: molecular dynamics studies on the active and inactive protein conformations. </span><span class="cited-content_cbyCitation_journal-name">Journal of Molecular Modeling</span><span> <strong>2013,</strong> <em>19 </em>
                                    (2)
                                     , 497-509. <a href="https://doi.org/10.1007/s00894-012-1559-0" title="DOI URL">https://doi.org/10.1007/s00894-012-1559-0</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s00894-012-1559-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs00894-012-1559-0%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Molecular%2520Modeling%26atitle%3DComputational%252Bstudy%252Bof%252BEGFR%252Binhibition%25253A%252Bmolecular%252Bdynamics%252Bstudies%252Bon%252Bthe%252Bactive%252Band%252Binactive%252Bprotein%252Bconformations%26aulast%3DSongtawee%26aufirst%3DNapat%26date%3D2013%26date%3D2012%26volume%3D19%26issue%3D2%26spage%3D497%26epage%3D509" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Cornelis  Matijssen</span>, <span class="hlFld-ContribAuthor ">M. Cris  Silva-Santisteban</span>, <span class="hlFld-ContribAuthor ">Isaac M.  Westwood</span>, <span class="hlFld-ContribAuthor ">Samerene  Siddique</span>, <span class="hlFld-ContribAuthor ">Vanessa  Choi</span>, <span class="hlFld-ContribAuthor ">Peter  Sheldrake</span>, <span class="hlFld-ContribAuthor ">Rob L.M.  van Montfort</span>, <span class="hlFld-ContribAuthor ">Julian  Blagg</span>. </span><span class="cited-content_cbyCitation_article-title">Benzimidazole inhibitors of the protein kinase CHK2: Clarification of the binding mode by flexible side chain docking and protein–ligand crystallography. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2012,</strong> <em>20 </em>
                                    (22)
                                     , 6630-6639. <a href="https://doi.org/10.1016/j.bmc.2012.09.024" title="DOI URL">https://doi.org/10.1016/j.bmc.2012.09.024</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2012.09.024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2012.09.024%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DBenzimidazole%252Binhibitors%252Bof%252Bthe%252Bprotein%252Bkinase%252BCHK2%25253A%252BClarification%252Bof%252Bthe%252Bbinding%252Bmode%252Bby%252Bflexible%252Bside%252Bchain%252Bdocking%252Band%252Bprotein%2525E2%252580%252593ligand%252Bcrystallography%26aulast%3DMatijssen%26aufirst%3DCornelis%26date%3D2012%26volume%3D20%26issue%3D22%26spage%3D6630%26epage%3D6639" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Michael  Brunsteiner</span>, <span class="hlFld-ContribAuthor ">Pavel A.  Petukhov</span>. </span><span class="cited-content_cbyCitation_article-title">Insights from comprehensive multiple receptor docking to HDAC8. </span><span class="cited-content_cbyCitation_journal-name">Journal of Molecular Modeling</span><span> <strong>2012,</strong> <em>18 </em>
                                    (8)
                                     , 3927-3939. <a href="https://doi.org/10.1007/s00894-011-1297-8" title="DOI URL">https://doi.org/10.1007/s00894-011-1297-8</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s00894-011-1297-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs00894-011-1297-8%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Molecular%2520Modeling%26atitle%3DInsights%252Bfrom%252Bcomprehensive%252Bmultiple%252Breceptor%252Bdocking%252Bto%252BHDAC8%26aulast%3DBrunsteiner%26aufirst%3DMichael%26date%3D2012%26date%3D2012%26volume%3D18%26issue%3D8%26spage%3D3927%26epage%3D3939" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Christopher R.  Corbeil</span>, <span class="hlFld-ContribAuthor ">Christopher I.  Williams</span>, <span class="hlFld-ContribAuthor ">Paul  Labute</span>. </span><span class="cited-content_cbyCitation_article-title">Variability in docking success rates due to dataset preparation. </span><span class="cited-content_cbyCitation_journal-name">Journal of Computer-Aided Molecular Design</span><span> <strong>2012,</strong> <em>26 </em>
                                    (6)
                                     , 775-786. <a href="https://doi.org/10.1007/s10822-012-9570-1" title="DOI URL">https://doi.org/10.1007/s10822-012-9570-1</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s10822-012-9570-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs10822-012-9570-1%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Computer-Aided%2520Molecular%2520Design%26atitle%3DVariability%252Bin%252Bdocking%252Bsuccess%252Brates%252Bdue%252Bto%252Bdataset%252Bpreparation%26aulast%3DCorbeil%26aufirst%3DChristopher%2BR.%26date%3D2012%26date%3D2012%26volume%3D26%26issue%3D6%26spage%3D775%26epage%3D786" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Christoph  Sotriffer</span>. </span><span class="cited-content_cbyCitation_article-title">Scoring Functions for Protein-Ligand Interactions. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2012,</strong>,, 237-263. <a href="https://doi.org/10.1002/9783527645947.ch12" title="DOI URL">https://doi.org/10.1002/9783527645947.ch12</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/9783527645947.ch12&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F9783527645947.ch12%26sid%3Dliteratum%253Aachs%26atitle%3DScoring%252BFunctions%252Bfor%252BProtein-Ligand%252BInteractions%26aulast%3DSotriffer%26aufirst%3DChristoph%26date%3D2012%26date%3D2012%26spage%3D237%26epage%3D263%26pub%3DWiley-VCH%2520Verlag%2520GmbH%2520%2526%2520Co.%2520KGaA%26atitle%3DProtein-Ligand%252BInteractions%26aulast%3DGohlke%26aufirst%3DHolger%26date%3D2012%26volume%3D9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Riccardo  Baron</span>, <span class="hlFld-ContribAuthor ">Piotr  Setny</span>, <span class="hlFld-ContribAuthor ">J. Andrew  McCammon</span>. </span><span class="cited-content_cbyCitation_article-title">Hydrophobic Association and Volume-Confined Water Molecules. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2012,</strong>,, 145-170. <a href="https://doi.org/10.1002/9783527645947.ch8" title="DOI URL">https://doi.org/10.1002/9783527645947.ch8</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/9783527645947.ch8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F9783527645947.ch8%26sid%3Dliteratum%253Aachs%26atitle%3DHydrophobic%252BAssociation%252Band%252BVolume-Confined%252BWater%252BMolecules%26aulast%3DBaron%26aufirst%3DRiccardo%26date%3D2012%26date%3D2012%26spage%3D145%26epage%3D170%26pub%3DWiley-VCH%2520Verlag%2520GmbH%2520%2526%2520Co.%2520KGaA%26atitle%3DProtein-Ligand%252BInteractions%26aulast%3DGohlke%26aufirst%3DHolger%26date%3D2012%26volume%3D108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Gregory A.  Ross</span>, <span class="hlFld-ContribAuthor ">Garrett M.  Morris</span>, <span class="hlFld-ContribAuthor ">Philip C.  Biggin</span>, . </span><span class="cited-content_cbyCitation_article-title">Rapid and Accurate Prediction and Scoring of Water Molecules in Protein Binding Sites. </span><span class="cited-content_cbyCitation_journal-name">PLoS ONE</span><span> <strong>2012,</strong> <em>7 </em>
                                    (3)
                                     , e32036. <a href="https://doi.org/10.1371/journal.pone.0032036" title="DOI URL">https://doi.org/10.1371/journal.pone.0032036</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1371/journal.pone.0032036&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1371%2Fjournal.pone.0032036%26sid%3Dliteratum%253Aachs%26jtitle%3DPLoS%2520ONE%26atitle%3DRapid%252Band%252BAccurate%252BPrediction%252Band%252BScoring%252Bof%252BWater%252BMolecules%252Bin%252BProtein%252BBinding%252BSites%26aulast%3DRoss%26aufirst%3DGregory%2BA.%26date%3D2012%26date%3D2012%26volume%3D7%26issue%3D3%26spage%3De32036" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Simona  Distinto</span>, <span class="hlFld-ContribAuthor ">Matilde  Yáñez</span>, <span class="hlFld-ContribAuthor ">Stefano  Alcaro</span>, <span class="hlFld-ContribAuthor ">M. Cristina  Cardia</span>, <span class="hlFld-ContribAuthor ">Marco  Gaspari</span>, <span class="hlFld-ContribAuthor ">M. Luisa  Sanna</span>, <span class="hlFld-ContribAuthor ">Rita  Meleddu</span>, <span class="hlFld-ContribAuthor ">Francesco  Ortuso</span>, <span class="hlFld-ContribAuthor ">Johannes  Kirchmair</span>, <span class="hlFld-ContribAuthor ">Patrick  Markt</span>, <span class="hlFld-ContribAuthor ">Adriana  Bolasco</span>, <span class="hlFld-ContribAuthor ">Gerhard  Wolber</span>, <span class="hlFld-ContribAuthor ">Daniela  Secci</span>, <span class="hlFld-ContribAuthor ">Elias  Maccioni</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis and biological assessment of novel 2-thiazolylhydrazones and computational analysis of their recognition by monoamine oxidase B. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2012,</strong> <em>48 </em>, 284-295. <a href="https://doi.org/10.1016/j.ejmech.2011.12.027" title="DOI URL">https://doi.org/10.1016/j.ejmech.2011.12.027</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2011.12.027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2011.12.027%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DSynthesis%252Band%252Bbiological%252Bassessment%252Bof%252Bnovel%252B2-thiazolylhydrazones%252Band%252Bcomputational%252Banalysis%252Bof%252Btheir%252Brecognition%252Bby%252Bmonoamine%252Boxidase%252BB%26aulast%3DDistinto%26aufirst%3DSimona%26date%3D2012%26volume%3D48%26spage%3D284%26epage%3D295" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Laura M.  Salonen</span>, <span class="hlFld-ContribAuthor ">Mareike C.  Holland</span>, <span class="hlFld-ContribAuthor ">Philip S. J.  Kaib</span>, <span class="hlFld-ContribAuthor ">Wolfgang  Haap</span>, <span class="hlFld-ContribAuthor ">Jörg  Benz</span>, <span class="hlFld-ContribAuthor ">Jean-Luc  Mary</span>, <span class="hlFld-ContribAuthor ">Olivier  Kuster</span>, <span class="hlFld-ContribAuthor ">W. Bernd  Schweizer</span>, <span class="hlFld-ContribAuthor ">David W.  Banner</span>, <span class="hlFld-ContribAuthor ">François  Diederich</span>. </span><span class="cited-content_cbyCitation_article-title">Molecular Recognition at the Active Site of Factor Xa: Cation-π Interactions, Stacking on Planar Peptide Surfaces, and Replacement of Structural Water. </span><span class="cited-content_cbyCitation_journal-name">Chemistry - A European Journal</span><span> <strong>2012,</strong> <em>18 </em>
                                    (1)
                                     , 213-222. <a href="https://doi.org/10.1002/chem.201102571" title="DOI URL">https://doi.org/10.1002/chem.201102571</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/chem.201102571&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fchem.201102571%26sid%3Dliteratum%253Aachs%26jtitle%3DChemistry%2520-%2520A%2520European%2520Journal%26atitle%3DMolecular%252BRecognition%252Bat%252Bthe%252BActive%252BSite%252Bof%252BFactor%252BXa%25253A%252BCation-%2525CF%252580%252BInteractions%25252C%252BStacking%252Bon%252BPlanar%252BPeptide%252BSurfaces%25252C%252Band%252BReplacement%252Bof%252BStructural%252BWater%26aulast%3DSalonen%26aufirst%3DLaura%2BM.%26date%3D2012%26date%3D2011%26volume%3D18%26issue%3D1%26spage%3D213%26epage%3D222" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Toan B.  Nguyen</span>, <span class="hlFld-ContribAuthor ">Sergio E.  Wong</span>, <span class="hlFld-ContribAuthor ">Felice C.  Lightstone</span>. </span><span class="cited-content_cbyCitation_article-title">Leveraging Structural Information for the Discovery of New Drugs: Computational Methods. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2012,</strong>,, 209-234. <a href="https://doi.org/10.1007/978-1-61779-520-6_9" title="DOI URL">https://doi.org/10.1007/978-1-61779-520-6_9</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-1-61779-520-6_9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-1-61779-520-6_9%26sid%3Dliteratum%253Aachs%26atitle%3DLeveraging%252BStructural%252BInformation%252Bfor%252Bthe%252BDiscovery%252Bof%252BNew%252BDrugs%25253A%252BComputational%252BMethods%26aulast%3DNguyen%26aufirst%3DToan%2BB.%26date%3D2012%26date%3D2011%26spage%3D209%26epage%3D234%26pub%3DHumana%2520Press%26atitle%3DStructure-Based%252BDrug%252BDiscovery%26aulast%3DTari%26aufirst%3DLeslie%2BW.%26date%3D2012%26volume%3D841" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Olivier  Taboureau</span>, <span class="hlFld-ContribAuthor ">Jonathan B.  Baell</span>, <span class="hlFld-ContribAuthor ">Juan  Fernández-Recio</span>, <span class="hlFld-ContribAuthor ">Bruno O.  Villoutreix</span>. </span><span class="cited-content_cbyCitation_article-title">Established and Emerging Trends in Computational Drug Discovery in the Structural Genomics Era. </span><span class="cited-content_cbyCitation_journal-name">Chemistry & Biology</span><span> <strong>2012,</strong> <em>19 </em>
                                    (1)
                                     , 29-41. <a href="https://doi.org/10.1016/j.chembiol.2011.12.007" title="DOI URL">https://doi.org/10.1016/j.chembiol.2011.12.007</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.chembiol.2011.12.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.chembiol.2011.12.007%26sid%3Dliteratum%253Aachs%26jtitle%3DChemistry%2520%2526%2520Biology%26atitle%3DEstablished%252Band%252BEmerging%252BTrends%252Bin%252BComputational%252BDrug%252BDiscovery%252Bin%252Bthe%252BStructural%252BGenomics%252BEra%26aulast%3DTaboureau%26aufirst%3DOlivier%26date%3D2012%26volume%3D19%26issue%3D1%26spage%3D29%26epage%3D41" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Gregorio  Fernández-Ballester</span>, <span class="hlFld-ContribAuthor ">Asia  Fernández-Carvajal</span>, <span class="hlFld-ContribAuthor ">José Manuel  González-Ros</span>, <span class="hlFld-ContribAuthor ">Antonio  Ferrer-Montiel</span>. </span><span class="cited-content_cbyCitation_article-title">Ionic Channels as Targets for Drug Design: A Review on Computational Methods. </span><span class="cited-content_cbyCitation_journal-name">Pharmaceutics</span><span> <strong>2011,</strong> <em>3 </em>
                                    (4)
                                     , 932-953. <a href="https://doi.org/10.3390/pharmaceutics3040932" title="DOI URL">https://doi.org/10.3390/pharmaceutics3040932</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/pharmaceutics3040932&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fpharmaceutics3040932%26sid%3Dliteratum%253Aachs%26jtitle%3DPharmaceutics%26atitle%3DIonic%252BChannels%252Bas%252BTargets%252Bfor%252BDrug%252BDesign%25253A%252BA%252BReview%252Bon%252BComputational%252BMethods%26aulast%3DFern%25C3%25A1ndez-Ballester%26aufirst%3DGregorio%26date%3D2011%26date%3D2011%26volume%3D3%26issue%3D4%26spage%3D932%26epage%3D953" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">D. J.  Huggins</span>, <span class="hlFld-ContribAuthor ">B.  Tidor</span>. </span><span class="cited-content_cbyCitation_article-title">Systematic placement of structural water molecules for improved scoring of protein-ligand interactions. </span><span class="cited-content_cbyCitation_journal-name">Protein Engineering Design and Selection</span><span> <strong>2011,</strong> <em>24 </em>
                                    (10)
                                     , 777-789. <a href="https://doi.org/10.1093/protein/gzr036" title="DOI URL">https://doi.org/10.1093/protein/gzr036</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1093/protein/gzr036&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1093%2Fprotein%2Fgzr036%26sid%3Dliteratum%253Aachs%26jtitle%3DProtein%2520Engineering%2520Design%2520and%2520Selection%26atitle%3DSystematic%252Bplacement%252Bof%252Bstructural%252Bwater%252Bmolecules%252Bfor%252Bimproved%252Bscoring%252Bof%252Bprotein-ligand%252Binteractions%26aulast%3DHuggins%26aufirst%3DD.%2BJ.%26date%3D2011%26date%3D2011%26volume%3D24%26issue%3D10%26spage%3D777%26epage%3D789" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shao-Yong  Lu</span>, <span class="hlFld-ContribAuthor ">Yong-Jun  Jiang</span>, <span class="hlFld-ContribAuthor ">Jing  Lv</span>, <span class="hlFld-ContribAuthor ">Jian-Wei  Zou</span>, <span class="hlFld-ContribAuthor ">Tian-Xing  Wu</span>. </span><span class="cited-content_cbyCitation_article-title">Role of bridging water molecules in GSK3β-inhibitor complexes: Insights from QM/MM, MD, and molecular docking studies. </span><span class="cited-content_cbyCitation_journal-name">Journal of Computational Chemistry</span><span> <strong>2011,</strong> <em>32 </em>
                                    (9)
                                     , 1907-1918. <a href="https://doi.org/10.1002/jcc.21775" title="DOI URL">https://doi.org/10.1002/jcc.21775</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/jcc.21775&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fjcc.21775%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Computational%2520Chemistry%26atitle%3DRole%252Bof%252Bbridging%252Bwater%252Bmolecules%252Bin%252BGSK3%2525CE%2525B2-inhibitor%252Bcomplexes%25253A%252BInsights%252Bfrom%252BQM%25252FMM%25252C%252BMD%25252C%252Band%252Bmolecular%252Bdocking%252Bstudies%26aulast%3DLu%26aufirst%3DShao-Yong%26date%3D2011%26date%3D2011%26volume%3D32%26issue%3D9%26spage%3D1907%26epage%3D1918" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Johannes  Kirchmair</span>, <span class="hlFld-ContribAuthor ">Gudrun M.  Spitzer</span>, <span class="hlFld-ContribAuthor ">Klaus R.  Liedl</span>. </span><span class="cited-content_cbyCitation_article-title">Consideration of Water and Solvation Effects in Virtual Screening. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2011,</strong>,, 263-289. <a href="https://doi.org/10.1002/9783527633326.ch10" title="DOI URL">https://doi.org/10.1002/9783527633326.ch10</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/9783527633326.ch10&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F9783527633326.ch10%26sid%3Dliteratum%253Aachs%26atitle%3DConsideration%252Bof%252BWater%252Band%252BSolvation%252BEffects%252Bin%252BVirtual%252BScreening%26aulast%3DKirchmair%26aufirst%3DJohannes%26date%3D2011%26date%3D2011%26spage%3D263%26epage%3D289%26pub%3DWiley-VCH%2520Verlag%2520GmbH%2520%2526%2520Co.%2520KGaA%26atitle%3DVirtual%252BScreening%26aulast%3DSotriffer%26aufirst%3DChristoph%26date%3D2011%26volume%3D12" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Didier  Rognan</span>. </span><span class="cited-content_cbyCitation_article-title">Docking Methods for Virtual Screening: Principles and Recent Advances. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2011,</strong>,, 153-176. <a href="https://doi.org/10.1002/9783527633326.ch6" title="DOI URL">https://doi.org/10.1002/9783527633326.ch6</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/9783527633326.ch6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F9783527633326.ch6%26sid%3Dliteratum%253Aachs%26atitle%3DDocking%252BMethods%252Bfor%252BVirtual%252BScreening%25253A%252BPrinciples%252Band%252BRecent%252BAdvances%26aulast%3DRognan%26aufirst%3DDidier%26date%3D2011%26date%3D2011%26spage%3D153%26epage%3D176%26pub%3DWiley-VCH%2520Verlag%2520GmbH%2520%2526%2520Co.%2520KGaA%26atitle%3DVirtual%252BScreening%26aulast%3DSotriffer%26aufirst%3DChristoph%26date%3D2011%26volume%3D12" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Bohdan  Waszkowycz</span>, <span class="hlFld-ContribAuthor ">David E.  Clark</span>, <span class="hlFld-ContribAuthor ">Emanuela  Gancia</span>. </span><span class="cited-content_cbyCitation_article-title">Outstanding challenges in protein–ligand docking and structure‐based virtual screening. </span><span class="cited-content_cbyCitation_journal-name">WIREs Computational Molecular Science</span><span> <strong>2011,</strong> <em>1 </em>
                                    (2)
                                     , 229-259. <a href="https://doi.org/10.1002/wcms.18" title="DOI URL">https://doi.org/10.1002/wcms.18</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/wcms.18&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fwcms.18%26sid%3Dliteratum%253Aachs%26jtitle%3DWIREs%2520Computational%2520Molecular%2520Science%26atitle%3DOutstanding%252Bchallenges%252Bin%252Bprotein%2525E2%252580%252593ligand%252Bdocking%252Band%252Bstructure%2525E2%252580%252590based%252Bvirtual%252Bscreening%26aulast%3DWaszkowycz%26aufirst%3DBohdan%26date%3D2011%26date%3D2011%26volume%3D1%26issue%3D2%26spage%3D229%26epage%3D259" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sergio E  Wong</span>, <span class="hlFld-ContribAuthor ">Felice C  Lightstone</span>. </span><span class="cited-content_cbyCitation_article-title">Accounting for water molecules in drug design. </span><span class="cited-content_cbyCitation_journal-name">Expert Opinion on Drug Discovery</span><span> <strong>2011,</strong> <em>6 </em>
                                    (1)
                                     , 65-74. <a href="https://doi.org/10.1517/17460441.2011.534452" title="DOI URL">https://doi.org/10.1517/17460441.2011.534452</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1517/17460441.2011.534452&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1517%2F17460441.2011.534452%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Opinion%2520on%2520Drug%2520Discovery%26atitle%3DAccounting%252Bfor%252Bwater%252Bmolecules%252Bin%252Bdrug%252Bdesign%26aulast%3DWong%26aufirst%3DSergio%2BE%26date%3D2011%26date%3D2010%26volume%3D6%26issue%3D1%26spage%3D65%26epage%3D74" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Alfonso T.  García-Sosa</span>, <span class="hlFld-ContribAuthor ">Ricardo L.  Mancera</span>. </span><span class="cited-content_cbyCitation_article-title">Free Energy Calculations of Mutations Involving a Tightly Bound Water Molecule and Ligand Substitutions in a Ligand-Protein Complex. </span><span class="cited-content_cbyCitation_journal-name">Molecular Informatics</span><span> <strong>2010,</strong> <em>29 </em>
                                    (8-9)
                                     , 589-600. <a href="https://doi.org/10.1002/minf.201000007" title="DOI URL">https://doi.org/10.1002/minf.201000007</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/minf.201000007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fminf.201000007%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Informatics%26atitle%3DFree%252BEnergy%252BCalculations%252Bof%252BMutations%252BInvolving%252Ba%252BTightly%252BBound%252BWater%252BMolecule%252Band%252BLigand%252BSubstitutions%252Bin%252Ba%252BLigand-Protein%252BComplex%26aulast%3DGarc%25C3%25ADa-Sosa%26aufirst%3DAlfonso%2BT.%26date%3D2010%26date%3D2010%26volume%3D29%26issue%3D8-9%26spage%3D589%26epage%3D600" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yu  Chen</span>, <span class="hlFld-ContribAuthor ">Denise Teotico  Pohlhaus</span>. </span><span class="cited-content_cbyCitation_article-title">In silico docking and scoring of fragments. </span><span class="cited-content_cbyCitation_journal-name">Drug Discovery Today: Technologies</span><span> <strong>2010,</strong> <em>7 </em>
                                    (3)
                                     , e149-e156. <a href="https://doi.org/10.1016/j.ddtec.2010.11.002" title="DOI URL">https://doi.org/10.1016/j.ddtec.2010.11.002</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ddtec.2010.11.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ddtec.2010.11.002%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Technologies%26atitle%3DIn%252Bsilico%252Bdocking%252Band%252Bscoring%252Bof%252Bfragments%26aulast%3DChen%26aufirst%3DYu%26date%3D2010%26volume%3D7%26issue%3D3%26spage%3De149%26epage%3De156" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ranjit  Vijayan</span>, <span class="hlFld-ContribAuthor ">Michelle A.  Sahai</span>, <span class="hlFld-ContribAuthor ">Thomas  Czajkowski</span>, <span class="hlFld-ContribAuthor ">Philip C.  Biggin</span>. </span><span class="cited-content_cbyCitation_article-title">A comparative analysis of the role of water in the binding pockets of ionotropic glutamate receptors. </span><span class="cited-content_cbyCitation_journal-name">Physical Chemistry Chemical Physics</span><span> <strong>2010,</strong> <em>12 </em>
                                    (42)
                                     , 14057. <a href="https://doi.org/10.1039/c004336b" title="DOI URL">https://doi.org/10.1039/c004336b</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/c004336b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2Fc004336b%26sid%3Dliteratum%253Aachs%26jtitle%3DPhysical%2520Chemistry%2520Chemical%2520Physics%26atitle%3DA%252Bcomparative%252Banalysis%252Bof%252Bthe%252Brole%252Bof%252Bwater%252Bin%252Bthe%252Bbinding%252Bpockets%252Bof%252Bionotropic%252Bglutamate%252Breceptors%26aulast%3DVijayan%26aufirst%3DRanjit%26date%3D2010%26volume%3D12%26issue%3D42%26spage%3D14057" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Manuel  Ellermann</span>, <span class="hlFld-ContribAuthor ">Roland  Jakob-Roetne</span>, <span class="hlFld-ContribAuthor ">Christian  Lerner</span>, <span class="hlFld-ContribAuthor ">Edilio  Borroni</span>, <span class="hlFld-ContribAuthor ">Daniel  Schlatter</span>, <span class="hlFld-ContribAuthor ">Doris  Roth</span>, <span class="hlFld-ContribAuthor ">Andreas  Ehler</span>, <span class="hlFld-ContribAuthor ">Markus Georg  Rudolph</span>, <span class="hlFld-ContribAuthor ">François  Diederich</span>. </span><span class="cited-content_cbyCitation_article-title">Molekulare Erkennung in der aktiven Tasche der Catechol-
              O
              - Methyltransferase: energetisch günstige Verdrängung eines von einem Bisubstratinhibitor importierten Wassermoleküls. </span><span class="cited-content_cbyCitation_journal-name">Angewandte Chemie</span><span> <strong>2009,</strong> <em>121 </em>
                                    (48)
                                     , 9256-9260. <a href="https://doi.org/10.1002/ange.200904410" title="DOI URL">https://doi.org/10.1002/ange.200904410</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/ange.200904410&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fange.200904410%26sid%3Dliteratum%253Aachs%26jtitle%3DAngewandte%2520Chemie%26atitle%3DMolekulare%252BErkennung%252Bin%252Bder%252Baktiven%252BTasche%252Bder%252BCatechol-%252BO%252B-%252BMethyltransferase%25253A%252Benergetisch%252Bg%2525C3%2525BCnstige%252BVerdr%2525C3%2525A4ngung%252Beines%252Bvon%252Beinem%252BBisubstratinhibitor%252Bimportierten%252BWassermolek%2525C3%2525BCls%26aulast%3DEllermann%26aufirst%3DManuel%26date%3D2009%26date%3D2009%26volume%3D121%26issue%3D48%26spage%3D9256%26epage%3D9260" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Manuel  Ellermann</span>, <span class="hlFld-ContribAuthor ">Roland  Jakob-Roetne</span>, <span class="hlFld-ContribAuthor ">Christian  Lerner</span>, <span class="hlFld-ContribAuthor ">Edilio  Borroni</span>, <span class="hlFld-ContribAuthor ">Daniel  Schlatter</span>, <span class="hlFld-ContribAuthor ">Doris  Roth</span>, <span class="hlFld-ContribAuthor ">Andreas  Ehler</span>, <span class="hlFld-ContribAuthor ">Markus���Georg  Rudolph</span>, <span class="hlFld-ContribAuthor ">Fran��ois  Diederich</span>. </span><span class="cited-content_cbyCitation_article-title">Molecular Recognition at the Active Site of Catechol-
              O
              -Methyltransferase: Energetically Favorable Replacement of a Water Molecule Imported by a Bisubstrate Inhibitor. </span><span class="cited-content_cbyCitation_journal-name">Angewandte Chemie International Edition</span><span> <strong>2009,</strong> <em>48 </em>
                                    (48)
                                     , 9092-9096. <a href="https://doi.org/10.1002/anie.200904410" title="DOI URL">https://doi.org/10.1002/anie.200904410</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/anie.200904410&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fanie.200904410%26sid%3Dliteratum%253Aachs%26jtitle%3DAngewandte%2520Chemie%2520International%2520Edition%26atitle%3DMolecular%252BRecognition%252Bat%252Bthe%252BActive%252BSite%252Bof%252BCatechol-%252BO%252B-Methyltransferase%25253A%252BEnergetically%252BFavorable%252BReplacement%252Bof%252Ba%252BWater%252BMolecule%252BImported%252Bby%252Ba%252BBisubstrate%252BInhibitor%26aulast%3DEllermann%26aufirst%3DManuel%26date%3D2009%26volume%3D48%26issue%3D48%26spage%3D9092%26epage%3D9096" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Fedor N.  Novikov</span>, <span class="hlFld-ContribAuthor ">Ghermes G.  Chilov</span>. </span><span class="cited-content_cbyCitation_article-title">Molecular docking: theoretical background, practical applications and perspectives. </span><span class="cited-content_cbyCitation_journal-name">Mendeleev Communications</span><span> <strong>2009,</strong> <em>19 </em>
                                    (5)
                                     , 237-242. <a href="https://doi.org/10.1016/j.mencom.2009.09.001" title="DOI URL">https://doi.org/10.1016/j.mencom.2009.09.001</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.mencom.2009.09.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.mencom.2009.09.001%26sid%3Dliteratum%253Aachs%26jtitle%3DMendeleev%2520Communications%26atitle%3DMolecular%252Bdocking%25253A%252Btheoretical%252Bbackground%25252C%252Bpractical%252Bapplications%252Band%252Bperspectives%26aulast%3DNovikov%26aufirst%3DFedor%2BN.%26date%3D2009%26volume%3D19%26issue%3D5%26spage%3D237%26epage%3D242" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mor  Rubinstein</span>, <span class="hlFld-ContribAuthor ">Masha Y.  Niv</span>. </span><span class="cited-content_cbyCitation_article-title">Peptidic modulators of protein-protein interactions: Progress and challenges in computational design. </span><span class="cited-content_cbyCitation_journal-name">Biopolymers</span><span> <strong>2009,</strong> <em>91 </em>
                                    (7)
                                     , 505-513. <a href="https://doi.org/10.1002/bip.21164" title="DOI URL">https://doi.org/10.1002/bip.21164</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/bip.21164&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fbip.21164%26sid%3Dliteratum%253Aachs%26jtitle%3DBiopolymers%26atitle%3DPeptidic%252Bmodulators%252Bof%252Bprotein-protein%252Binteractions%25253A%252BProgress%252Band%252Bchallenges%252Bin%252Bcomputational%252Bdesign%26aulast%3DRubinstein%26aufirst%3DMor%26date%3D2009%26volume%3D91%26issue%3D7%26spage%3D505%26epage%3D513" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ute F.  Röhrig</span>, <span class="hlFld-ContribAuthor ">Aurélien  Grosdidier</span>, <span class="hlFld-ContribAuthor ">Vincent  Zoete</span>, <span class="hlFld-ContribAuthor ">Olivier  Michielin</span>. </span><span class="cited-content_cbyCitation_article-title">Docking to heme proteins. </span><span class="cited-content_cbyCitation_journal-name">Journal of Computational Chemistry</span><span> <strong>2009,</strong> <em>3 </em>, NA-NA. <a href="https://doi.org/10.1002/jcc.21244" title="DOI URL">https://doi.org/10.1002/jcc.21244</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/jcc.21244&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fjcc.21244%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Computational%2520Chemistry%26atitle%3DDocking%252Bto%252Bheme%252Bproteins%26aulast%3DR%25C3%25B6hrig%26aufirst%3DUte%2BF.%26date%3D2009%26volume%3D3%26spage%3DNA%26epage%3DNA" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-16/jm8006239/production/images/medium/jm-2008-006239_0004.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-16/jm8006239/production/images/large/jm-2008-006239_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm8006239&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-16/jm8006239/production/images/medium/jm-2008-006239_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-16/jm8006239/production/images/large/jm-2008-006239_0001.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Docking poses of representative ligands (carbons in green) from the library screens showing the roles of different specific water configurations. Note that we combined waters from two different crystal structures for AmpC (waters from 1xgj colored in red and from 1l2s colored in blue). Images generated with Chimera.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-16/jm8006239/production/images/large/jm-2008-006239_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm8006239&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-16/jm8006239/production/images/medium/jm-2008-006239_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-16/jm8006239/production/images/large/jm-2008-006239_0002.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Docking enrichment plots for 24 protein targets. The docking ranked database (<i>x</i>-axis) vs the percentage of ligands found (<i>y</i>-axis) at any percentage of the ranking. The gray line is random selection; the black line is docking enrichment neglecting the binding site waters, and the red line is the enrichment with displaceable waters. Targets are listed in same order as in Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-16/jm8006239/production/images/large/jm-2008-006239_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm8006239&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-16/jm8006239/production/images/medium/jm-2008-006239_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-16/jm8006239/production/images/large/jm-2008-006239_0003.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Docking energies for ligands and decoys with and without displaceable waters for four targets. (Left) The distribution of docking energies (<i>x</i>-axis) and ligand and decoy ranks expressed as a normalized probability (<i>y</i>-axis): ligands (solid line) and decoys (dotted line) with (red) and without (black) displaceable waters. (Right) The presence of particular water configurations for high-scoring ligand (red) and decoy (blue) geometries, numbered sequentially.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2008/jmcmar.2008.51.issue-16/jm8006239/production/images/large/jm-2008-006239_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm8006239&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i4">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_04740" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_04740" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 17 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Barillari, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Viner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Essex, J. W.</span><span> </span><span class="NLM_article-title">Classification of water molecules in protein binding sites</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_year" style="font-weight: bold;">2007</span> <span class="NLM_volume">129</span> <span class="NLM_fpage">2577</span> <span class="NLM_lpage">2587</span></span><div class="citationLinks">[<a href="/doi/10.1021/ja066980q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Fjm8006239&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhsFGjt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=129&publication_year=2007&pages=2577-2587&author=C.+Barillariauthor=J.+Taylorauthor=R.+Vinerauthor=J.+W.+Essex&title=Classification+of+water+molecules+in+protein+binding+sites"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Classification of Water Molecules in Protein Binding Sites</span></div><div class="casAuthors">Barillari, Caterina; Taylor, Justine; Viner, Russell; Essex, Jonathan W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">129</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2577-2587</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Water mols. play a crucial role in mediating the interaction between a ligand and a macromol. receptor.  An understanding of the nature and role of each water mol. in the active site of a protein could greatly increase the efficiency of rational drug design approaches: if the propensity of a water mol. for displacement can be detd., then synthetic effort may be most profitably applied to the design of specific ligands with the displacement of this water mol. in mind.  In this paper, a thermodn. anal. of water mols. in the binding sites of six proteins, each complexed with a no. of inhibitors, is presented.  Two classes of water mols. were identified: those conserved and not displaced by any of the ligands, and those that are displaced by some ligands.  The abs. binding free energies of 54 water mols. were calcd. using the double decoupling method, with replica exchange thermodn. integration in Monte Carlo simulations.  It was found that conserved water mols. are on av. more tightly bound than displaced water mols.  In addn., Bayesian statistics is used to calc. the probability that a particular water mol. may be displaced by an appropriately designed ligand, given the calcd. binding free energy of the water mol.  This approach therefore allows the numerical assessment of whether or not a given water mol. should be targeted for displacement as part of a rational drug design strategy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr38ETxUkD-NrVg90H21EOLACvtfcHk0lg8yWgVeO78nA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhsFGjt7o%253D&md5=e74664f19993a9e9c087dbb9c1d54c3e</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1021%2Fja066980q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja066980q%26sid%3Dliteratum%253Aachs%26aulast%3DBarillari%26aufirst%3DC.%26aulast%3DTaylor%26aufirst%3DJ.%26aulast%3DViner%26aufirst%3DR.%26aulast%3DEssex%26aufirst%3DJ.%2BW.%26atitle%3DClassification%2520of%2520water%2520molecules%2520in%2520protein%2520binding%2520sites%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2007%26volume%3D129%26spage%3D2577%26epage%3D2587" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Lu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, C. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, S.</span><span> </span><span class="NLM_article-title">Analysis of ligand-bound water molecules in high-resolution crystal structures of protein−ligand complexes</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_year" style="font-weight: bold;">2007</span> <span class="NLM_volume">47</span> <span class="NLM_fpage">668</span> <span class="NLM_lpage">675</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci6003527" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Fjm8006239&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFalsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2007&pages=668-675&author=Y.+Luauthor=R.+Wangauthor=C.+Y.+Yangauthor=S.+Wang&title=Analysis+of+ligand-bound+water+molecules+in+high-resolution+crystal+structures+of+protein%E2%88%92ligand+complexes"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Analysis of Ligand-Bound Water Molecules in High-Resolution Crystal Structures of Protein-Ligand Complexes</span></div><div class="casAuthors">Lu, Yipin; Wang, Renxiao; Yang, Chao-Yie; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">668-675</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We have performed a comprehensive anal. of water mols. at the protein-ligand interfaces obsd. in 392 high-resoln. crystal structures.  There are a total of 1829 ligand-bound water mols. in these 392 complexes; 18% are surface water mols., and 72% are interfacial water mols.  The no. of ligand-bound water mols. in each complex structure ranges from 0 to 21 and has an av. of 4.6.  Of these interfacial water mols., 76% are considered to be bridging water mols., characterized by having polar interactions with both ligand and protein atoms.  Among a no. of factors that may influence the no. of ligand-bound water mols., the polar van der Waals (vdw) surface area of ligands has the highest Pearson linear correlation coeff. of 0.63.  Our regression anal. predicted that one more ligand-bound water mol. is expected for every addnl. 24 Å2 in the polar vdw surface area of the ligand.  In contrast to the observation that the resoln. is the primary factor influencing the no. of water mols. in crystallog. models of proteins, we found that there is only a weak relationship between the no. of ligand-bound water mols. and the resoln. of the crystal structures.  An anal. of the isotropic B factors of buried ligand-bound water mols. suggested that, when water mols. have fewer than two polar interactions with the protein-ligand complex, they are more mobile than protein atoms in the crystal structures; when they have more than three polar interactions, they are significantly less mobile than protein atoms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2Ryn_w53w0rVg90H21EOLACvtfcHk0lg8yWgVeO78nA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFalsbg%253D&md5=4d46f344e82d0f4fce26cb20b6e28010</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1021%2Fci6003527&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci6003527%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DR.%26aulast%3DYang%26aufirst%3DC.%2BY.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DAnalysis%2520of%2520ligand-bound%2520water%2520molecules%2520in%2520high-resolution%2520crystal%2520structures%2520of%2520protein%25E2%2588%2592ligand%2520complexes%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2007%26volume%3D47%26spage%3D668%26epage%3D675" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Mancera, R. L.</span><span> </span><span class="NLM_article-title">Molecular modeling of hydration in drug design</span> <span class="citation_source-journal">Curr. Opin. Drug Discovery Dev.</span><span class="NLM_year" style="font-weight: bold;">2007</span> <span class="NLM_volume">10</span> <span class="NLM_fpage">275</span> <span class="NLM_lpage">280</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Fjm8006239&amp;key=17554853" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Fjm8006239&amp;key=1%3ACAS%3A528%3ADC%252BD2sXmtVWlsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2007&pages=275-280&author=R.+L.+Mancera&title=Molecular+modeling+of+hydration+in+drug+design"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular modeling of hydration in drug design</span></div><div class="casAuthors">Mancera, Ricardo L.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Drug Discovery & Development</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">275-280</span>CODEN:
                <span class="NLM_cas:coden">CODDFF</span>;
        ISSN:<span class="NLM_cas:issn">1367-6733</span>.
    
            (<span class="NLM_cas:orgname">Thomson Scientific</span>)
        </div><div class="casAbstract">A review.  The accurate prediction of the binding mode and affinity of a drug mol. to its putative receptor is crucial for successful drug design and optimization.  Modeling the role of the aq. solvent in ligand-protein interactions is one of the key components.  Three main computational techniques have been developed to model hydration: free energy methods, ligand-protein docking and scoring, and the explicit inclusion of tightly bound water mols. in modeling.  This review provides a summary of the most recent developments in these approaches used for the mol. modeling of hydration effects in the context of computational structure-based drug design methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5oaWakeqXu7Vg90H21EOLACvtfcHk0lg8yWgVeO78nA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXmtVWlsrg%253D&md5=00f8343ef349b19271d7199dd048d215</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMancera%26aufirst%3DR.%2BL.%26atitle%3DMolecular%2520modeling%2520of%2520hydration%2520in%2520drug%2520design%26jtitle%3DCurr.%2520Opin.%2520Drug%2520Discovery%2520Dev.%26date%3D2007%26volume%3D10%26spage%3D275%26epage%3D280" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Fitzpatrick, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Steinmetz, A. C. U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ringe, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klibanov, A. M.</span><span> </span><span class="NLM_article-title">Enzyme crystal structure in a neat organic solvent</span> <span class="citation_source-journal">Proc. Nat. Acad. Sci. U.S.A.</span><span class="NLM_year" style="font-weight: bold;">1993</span> <span class="NLM_volume">90</span> <span class="NLM_fpage">8653</span> <span class="NLM_lpage">8657</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Fjm8006239&amp;key=10.1073%2Fpnas.90.18.8653" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Fjm8006239&amp;key=8378343" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Fjm8006239&amp;key=1%3ACAS%3A528%3ADyaK2cXjsV2m" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=90&publication_year=1993&pages=8653-8657&author=P.+A.+Fitzpatrickauthor=A.+C.+U.+Steinmetzauthor=D.+Ringeauthor=A.+M.+Klibanov&title=Enzyme+crystal+structure+in+a+neat+organic+solvent"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Enzyme crystal structure in a neat organic solvent</span></div><div class="casAuthors">Fitzpatrick, Paul A.; Steinmetz, Anke C. U.; Ringe, Dagmar; Klibanov, Alexander M.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">90</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">8653-7</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    </div><div class="casAbstract">The crystal structure of the serine protease subtilisin Carlsberg in anhyd. acetonitrile was detd. at 2.3 Å resoln.  It was found to be essentially identical to the three-dimensional structure of the enzyme is water; the differences obsd. were smaller than those between two independently detd. structures in aq. soln.  The hydrogen bond system of the catalytic triad is intact in acetonitrile.  The majority (99 of 119) of enzyme-bound, structural water mols. have such a great affinity to subtilisin that they are not displaced even in anhyd. acetonitrile.  Of the 12 enzyme-bound acetonitrile mols., 4 displace water mols. and 8 bind where no water had been obsd. before.  One-third of all subtilisin-bound acetonitrile mols. reside in the active center, occupying the same region (P1, P2, and P3 binding sites) as the specific protein inhibitor eglin c.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbO1HQQt_shbVg90H21EOLACvtfcHk0liL89eNdwNvmQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXjsV2m&md5=80c75820f758722bd231cab982b63a1f</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1073%2Fpnas.90.18.8653&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.90.18.8653%26sid%3Dliteratum%253Aachs%26aulast%3DFitzpatrick%26aufirst%3DP.%2BA.%26aulast%3DSteinmetz%26aufirst%3DA.%2BC.%2BU.%26aulast%3DRinge%26aufirst%3DD.%26aulast%3DKlibanov%26aufirst%3DA.%2BM.%26atitle%3DEnzyme%2520crystal%2520structure%2520in%2520a%2520neat%2520organic%2520solvent%26jtitle%3DProc.%2520Nat.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D1993%26volume%3D90%26spage%3D8653%26epage%3D8657" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Raymer, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sanschagrin, P. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Punch, W. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Venkataraman, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goodman, E. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuhn, L. A.</span><span> </span><span class="NLM_article-title">Predicting conserved water-mediated and polar ligand interactions in proteins using a K-nearest-neighbors genetic algorithm</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_year" style="font-weight: bold;">1997</span> <span class="NLM_volume">265</span> <span class="NLM_fpage">445</span> <span class="NLM_lpage">464</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Fjm8006239&amp;key=10.1006%2Fjmbi.1996.0746" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Fjm8006239&amp;key=9034363" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Fjm8006239&amp;key=1%3ACAS%3A528%3ADyaK2sXht1aktrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=265&publication_year=1997&pages=445-464&author=M.+L.+Raymerauthor=P.+C.+Sanschagrinauthor=W.+F.+Punchauthor=S.+Venkataramanauthor=E.+D.+Goodmanauthor=L.+A.+Kuhn&title=Predicting+conserved+water-mediated+and+polar+ligand+interactions+in+proteins+using+a+K-nearest-neighbors+genetic+algorithm"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Predicting conserved water-mediated and polar ligand interactions in protein using a K-nearest-neighbors genetic algorithm</span></div><div class="casAuthors">Raymer, Michael L.; Sanschagrin, Paul C.; Punch, William F.; Venkataraman, Sridhar; Goodman, Erik D.; Kuhn, Leslie A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">265</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">445-464</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">Water-mediated ligand interactions are essential to biol. processes, from product displacement in thymidylate synthase to DNA recognition by Trp repressor, yet the structural chem. influencing whether bound water is displaced or participates in ligand binding is not well characterized.  Consolv, employing a hybrid k-nearest-neighbors classifier/genetic algorithm, predicts bound water mols. conserved between free and ligand-bound protein structures by examg. the environment of each water mol. in the free structure.  Four environmental features are used: the water mol.'s crystallog. temp. factor, the no. of hydrogen bonds between the water mol. and protein, and the d. and hydrophilicity of neighboring protein atoms.  After training on 13 non-homologous proteins, Consolv predicted the conservation of active-site water mols. upon ligand binding with 75% accuracy (Matthews coeff. Cm = 0.41) for seven new proteins.  Mispredictions typically involved water mols. predicted to be conserved that were displaced by a polar ligand atom, indicating that Consolv correctly assesses polar binding sites; 90% accuracy (Cm = 0.78) was achieved for predicting conserved active-site water or polar ligand atom binding.  Consolv thus provides an accurate means for optimizing ligand design by identifying sites favored to be occupied by either a mediating water mol. or a polar ligand atom, as well as water mols. likely to be displaced by the ligand.  Accuracy for predicting first-shell water conservation between independently detd. structures was 61% (Cm = 0.23).  The ability to predict water-mediated and polar interactions from the free protein structure indicates the surprising extent to which the conservation or displacement of active-site bound water is independent of the ligand, and shows that the protein micro-environment of each water mol. is the dominant influence.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpaqbnU8qfRV7Vg90H21EOLACvtfcHk0liL89eNdwNvmQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXht1aktrw%253D&md5=1dfd393d67b240857c928943abb7c6b3</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1006%2Fjmbi.1996.0746&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fjmbi.1996.0746%26sid%3Dliteratum%253Aachs%26aulast%3DRaymer%26aufirst%3DM.%2BL.%26aulast%3DSanschagrin%26aufirst%3DP.%2BC.%26aulast%3DPunch%26aufirst%3DW.%2BF.%26aulast%3DVenkataraman%26aufirst%3DS.%26aulast%3DGoodman%26aufirst%3DE.%2BD.%26aulast%3DKuhn%26aufirst%3DL.%2BA.%26atitle%3DPredicting%2520conserved%2520water-mediated%2520and%2520polar%2520ligand%2520interactions%2520in%2520proteins%2520using%2520a%2520K-nearest-neighbors%2520genetic%2520algorithm%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1997%26volume%3D265%26spage%3D445%26epage%3D464" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Rarey, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kramer, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lengauer, T.</span><span> </span><span class="NLM_article-title">The particle concept: placing discrete water molecules during protein−ligand docking predictions</span> <span class="citation_source-journal">Proteins</span><span class="NLM_year" style="font-weight: bold;">1999</span> <span class="NLM_volume">34</span> <span class="NLM_fpage">17</span> <span class="NLM_lpage">28</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=1999&pages=17-28&author=M.+Rareyauthor=B.+Kramerauthor=T.+Lengauer&title=The+particle+concept%3A+placing+discrete+water+molecules+during+protein%E2%88%92ligand+docking+predictions"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DRarey%26aufirst%3DM.%26aulast%3DKramer%26aufirst%3DB.%26aulast%3DLengauer%26aufirst%3DT.%26atitle%3DThe%2520particle%2520concept%253A%2520placing%2520discrete%2520water%2520molecules%2520during%2520protein%25E2%2588%2592ligand%2520docking%2520predictions%26jtitle%3DProteins%26date%3D1999%26volume%3D34%26spage%3D17%26epage%3D28" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Minke, W. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diller, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hol, W. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verlinde, C. L.</span><span> </span><span class="NLM_article-title">The role of waters in docking strategies with incremental flexibility for carbohydrate derivatives: heat-labile enterotoxin, a multivalent test case</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_year" style="font-weight: bold;">1999</span> <span class="NLM_volume">42</span> <span class="NLM_fpage">1778</span> <span class="NLM_lpage">1788</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm980472c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=1999&pages=1778-1788&author=W.+E.+Minkeauthor=D.+J.+Dillerauthor=W.+G.+Holauthor=C.+L.+Verlinde&title=The+role+of+waters+in+docking+strategies+with+incremental+flexibility+for+carbohydrate+derivatives%3A+heat-labile+enterotoxin%2C+a+multivalent+test+case"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1021%2Fjm980472c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm980472c%26sid%3Dliteratum%253Aachs%26aulast%3DMinke%26aufirst%3DW.%2BE.%26aulast%3DDiller%26aufirst%3DD.%2BJ.%26aulast%3DHol%26aufirst%3DW.%2BG.%26aulast%3DVerlinde%26aufirst%3DC.%2BL.%26atitle%3DThe%2520role%2520of%2520waters%2520in%2520docking%2520strategies%2520with%2520incremental%2520flexibility%2520for%2520carbohydrate%2520derivatives%253A%2520heat-labile%2520enterotoxin%252C%2520a%2520multivalent%2520test%2520case%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1999%26volume%3D42%26spage%3D1778%26epage%3D1788" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Osterberg, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morris, G. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sanner, M. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olson, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goodsell, D. S.</span><span> </span><span class="NLM_article-title">Automated docking to multiple target structures: incorporation of protein mobility and structural water heterogeneity in AutoDock</span> <span class="citation_source-journal">Proteins</span><span class="NLM_year" style="font-weight: bold;">2002</span> <span class="NLM_volume">46</span> <span class="NLM_fpage">34</span> <span class="NLM_lpage">40</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2002&pages=34-40&author=F.+Osterbergauthor=G.+M.+Morrisauthor=M.+F.+Sannerauthor=A.+J.+Olsonauthor=D.+S.+Goodsell&title=Automated+docking+to+multiple+target+structures%3A+incorporation+of+protein+mobility+and+structural+water+heterogeneity+in+AutoDock"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DOsterberg%26aufirst%3DF.%26aulast%3DMorris%26aufirst%3DG.%2BM.%26aulast%3DSanner%26aufirst%3DM.%2BF.%26aulast%3DOlson%26aufirst%3DA.%2BJ.%26aulast%3DGoodsell%26aufirst%3DD.%2BS.%26atitle%3DAutomated%2520docking%2520to%2520multiple%2520target%2520structures%253A%2520incorporation%2520of%2520protein%2520mobility%2520and%2520structural%2520water%2520heterogeneity%2520in%2520AutoDock%26jtitle%3DProteins%26date%3D2002%26volume%3D46%26spage%3D34%26epage%3D40" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">de Graaf, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pospisil, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pos, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Folkers, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vermeulen, N. P.</span><span> </span><span class="NLM_article-title">Binding mode prediction of cytochrome p450 and thymidine kinase protein−ligand complexes by consideration of water and rescoring in automated docking</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_year" style="font-weight: bold;">2005</span> <span class="NLM_volume">48</span> <span class="NLM_fpage">2308</span> <span class="NLM_lpage">2318</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm049650u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=2308-2318&author=C.+de+Graafauthor=P.+Pospisilauthor=W.+Posauthor=G.+Folkersauthor=N.+P.+Vermeulen&title=Binding+mode+prediction+of+cytochrome+p450+and+thymidine+kinase+protein%E2%88%92ligand+complexes+by+consideration+of+water+and+rescoring+in+automated+docking"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Fjm049650u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm049650u%26sid%3Dliteratum%253Aachs%26aulast%3Dde%2BGraaf%26aufirst%3DC.%26aulast%3DPospisil%26aufirst%3DP.%26aulast%3DPos%26aufirst%3DW.%26aulast%3DFolkers%26aufirst%3DG.%26aulast%3DVermeulen%26aufirst%3DN.%2BP.%26atitle%3DBinding%2520mode%2520prediction%2520of%2520cytochrome%2520p450%2520and%2520thymidine%2520kinase%2520protein%25E2%2588%2592ligand%2520complexes%2520by%2520consideration%2520of%2520water%2520and%2520rescoring%2520in%2520automated%2520docking%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D2308%26epage%3D2318" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Verdonk, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chessari, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cole, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hartshorn, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nissink, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, R. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, R.</span><span> </span><span class="NLM_article-title">Modeling water molecules in protein−ligand docking using GOLD</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_year" style="font-weight: bold;">2005</span> <span class="NLM_volume">48</span> <span class="NLM_fpage">6504</span> <span class="NLM_lpage">6515</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm050543p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=6504-6515&author=M.+L.+Verdonkauthor=G.+Chessariauthor=J.+C.+Coleauthor=M.+J.+Hartshornauthor=C.+W.+Murrayauthor=J.+W.+Nissinkauthor=R.+D.+Taylorauthor=R.+Taylor&title=Modeling+water+molecules+in+protein%E2%88%92ligand+docking+using+GOLD"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Fjm050543p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm050543p%26sid%3Dliteratum%253Aachs%26aulast%3DVerdonk%26aufirst%3DM.%2BL.%26aulast%3DChessari%26aufirst%3DG.%26aulast%3DCole%26aufirst%3DJ.%2BC.%26aulast%3DHartshorn%26aufirst%3DM.%2BJ.%26aulast%3DMurray%26aufirst%3DC.%2BW.%26aulast%3DNissink%26aufirst%3DJ.%2BW.%26aulast%3DTaylor%26aufirst%3DR.%2BD.%26aulast%3DTaylor%26aufirst%3DR.%26atitle%3DModeling%2520water%2520molecules%2520in%2520protein%25E2%2588%2592ligand%2520docking%2520using%2520GOLD%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D6504%26epage%3D6515" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Corbeil, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Englebienne, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moitessier, N.</span><span> </span><span class="NLM_article-title">Docking ligands into flexible and solvated macromolecules. 1. Development and validation of FITTED 1.0</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_year" style="font-weight: bold;">2007</span> <span class="NLM_volume">47</span> <span class="NLM_fpage">435</span> <span class="NLM_lpage">449</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci6002637" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Fjm8006239&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhvVWjtbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2007&pages=435-449&author=C.+R.+Corbeilauthor=P.+Englebienneauthor=N.+Moitessier&title=Docking+ligands+into+flexible+and+solvated+macromolecules.+1.+Development+and+validation+of+FITTED+1.0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Docking Ligands into Flexible and Solvated Macromolecules. 1. Development and Validation of FITTED 1.0</span></div><div class="casAuthors">Corbeil, Christopher R.; Englebienne, Pablo; Moitessier, Nicolas</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">435-449</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The authors report the development and validation of a novel suite of programs, FITTED 1.0, for the docking of flexible ligands into flexible proteins.  This docking tool is unique in that it can deal with both the flexibility of macromols. (side chains and main chains) and the presence of bridging water mols. while treating protein/ligand complexes as realistically dynamic systems.  This software relies on a genetic algorithm to account for the flexibility of the two mols. as well as the location of bridging water mols.  In addn., FITTED 1.0 features a novel application of a switching function to retain or displace key water mols. from the protein-ligand complexes.  Two independent modules, ProCESS and SMART, were developed to set up the proteins and the ligands prior to the docking stage.  Validation of the accuracy of the software was achieved via the application of FITTED 1.0 to the docking of inhibitors of HIV-1 protease, thymidine kinase, trypsin, factor Xa, and MMP to their resp. proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqH9ypbBV5-k7Vg90H21EOLACvtfcHk0lgej8GG2Viujg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhvVWjtbY%253D&md5=f1b5be0eb418ecc158f5f057c1f9567c</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Fci6002637&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci6002637%26sid%3Dliteratum%253Aachs%26aulast%3DCorbeil%26aufirst%3DC.%2BR.%26aulast%3DEnglebienne%26aufirst%3DP.%26aulast%3DMoitessier%26aufirst%3DN.%26atitle%3DDocking%2520ligands%2520into%2520flexible%2520and%2520solvated%2520macromolecules.%25201.%2520Development%2520and%2520validation%2520of%2520FITTED%25201.0%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2007%26volume%3D47%26spage%3D435%26epage%3D449" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Friesner, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murphy, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Repasky, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, L. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greenwood, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Halgren, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sanschagrin, P. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mainz, D. T.</span><span> </span><span class="NLM_article-title">Extra precision glide: docking and scoring incorporating a model of hydrophobic enclosure for protein−ligand complexes</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_year" style="font-weight: bold;">2006</span> <span class="NLM_volume">49</span> <span class="NLM_fpage">6177</span> <span class="NLM_lpage">6196</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm051256o" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=6177-6196&author=R.+A.+Friesnerauthor=R.+B.+Murphyauthor=M.+P.+Repaskyauthor=L.+L.+Fryeauthor=J.+R.+Greenwoodauthor=T.+A.+Halgrenauthor=P.+C.+Sanschagrinauthor=D.+T.+Mainz&title=Extra+precision+glide%3A+docking+and+scoring+incorporating+a+model+of+hydrophobic+enclosure+for+protein%E2%88%92ligand+complexes"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Fjm051256o&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm051256o%26sid%3Dliteratum%253Aachs%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DRepasky%26aufirst%3DM.%2BP.%26aulast%3DFrye%26aufirst%3DL.%2BL.%26aulast%3DGreenwood%26aufirst%3DJ.%2BR.%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DSanschagrin%26aufirst%3DP.%2BC.%26aulast%3DMainz%26aufirst%3DD.%2BT.%26atitle%3DExtra%2520precision%2520glide%253A%2520docking%2520and%2520scoring%2520incorporating%2520a%2520model%2520of%2520hydrophobic%2520enclosure%2520for%2520protein%25E2%2588%2592ligand%2520complexes%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D6177%26epage%3D6196" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Roberts, B. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mancera, R. L.</span><span> </span><span class="NLM_article-title">Ligand−protein docking with water molecules</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_year" style="font-weight: bold;">2008</span> <span class="NLM_volume">48</span> <span class="NLM_fpage">397</span> <span class="NLM_lpage">408</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci700285e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Fjm8006239&amp;key=1%3ACAS%3A528%3ADC%252BD1cXptl2itQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2008&pages=397-408&author=B.+C.+Robertsauthor=R.+L.+Mancera&title=Ligand%E2%88%92protein+docking+with+water+molecules"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand-Protein Docking with Water Molecules</span></div><div class="casAuthors">Roberts, Benjamin C.; Mancera, Ricardo L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">397-408</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The presence of water mols. plays an important role in the accuracy of ligand-protein docking predictions.  Comprehensive docking simulations have been performed on a large set of ligand-protein complexes whose crystal structures contain water mols. in their binding sites.  Only those water mols. found in the immediate vicinity of both the ligand and the protein were considered.  The authors have investigated whether prior optimization of the orientation of water mols. in either the presence or absence of the bound ligand has any effect on the accuracy of docking predictions.  The authors have obsd. a statistically significant overall increase in accuracy when water mols. are included during docking simulations and have found this to be independent of the method of optimization of the orientation of water mols.  These results confirm the importance of including water mols. whenever possible in a ligand-protein docking simulation.  The authors' findings also reveal that prior optimization of the orientation of water mols., in the absence of any bound ligand, does not have a detrimental effect on the improved accuracy of ligand-protein docking.  This is important, given the use of docking simulations to predict the binding modes of new ligands or drug mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpRgAZwwN3kBLVg90H21EOLACvtfcHk0libKEP9mHf43g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXptl2itQ%253D%253D&md5=9559fcddc8fed156eb0b5a7ba982cc4a</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Fci700285e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci700285e%26sid%3Dliteratum%253Aachs%26aulast%3DRoberts%26aufirst%3DB.%2BC.%26aulast%3DMancera%26aufirst%3DR.%2BL.%26atitle%3DLigand%25E2%2588%2592protein%2520docking%2520with%2520water%2520molecules%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2008%26volume%3D48%26spage%3D397%26epage%3D408" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Huang, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span> </span><span class="NLM_article-title">Benchmarking sets for molecular docking</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_year" style="font-weight: bold;">2006</span> <span class="NLM_volume">49</span> <span class="NLM_fpage">6789</span> <span class="NLM_lpage">6801</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm0608356" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Fjm8006239&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtFehurzI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=6789-6801&author=N.+Huangauthor=B.+K.+Shoichetauthor=J.+J.+Irwin&title=Benchmarking+sets+for+molecular+docking"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Benchmarking Sets for Molecular Docking</span></div><div class="casAuthors">Huang, Niu; Shoichet, Brian K.; Irwin, John J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">6789-6801</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Ligand enrichment among top-ranking hits is a key metric of mol. docking.  To avoid bias, decoys should resemble ligands phys., so that enrichment is not simply a sepn. of gross features, yet be chem. distinct from them, so that they are unlikely to be binders.  We have assembled a directory of useful decoys (DUD), with 2950 ligands for 40 different targets.  Every ligand has 36 decoy mols. that are phys. similar but topol. distinct, leading to a database of 98 266 compds.  For most targets, enrichment was at least half a log better with uncorrected databases such as the MDDR than with DUD, evidence of bias in the former.  These calcns. also allowed 40×40 cross-docking, where the enrichments of each ligand set could be compared for all 40 targets, enabling a specificity metric for the docking screens.  DUD is freely available online as a benchmarking set for docking at http://blaster.docking.org/dud/.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-8FVhVSsBm7Vg90H21EOLACvtfcHk0libKEP9mHf43g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtFehurzI&md5=43af46968c2a6d66334922bf0caedbe1</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Fjm0608356&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0608356%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DN.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26atitle%3DBenchmarking%2520sets%2520for%2520molecular%2520docking%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D6789%26epage%3D6801" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Huang, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kalyanaraman, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacobson, M. P.</span><span> </span><span class="NLM_article-title">Physics-based scoring of protein−ligand complexes: enrichment of known inhibitors in large-scale virtual screening</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_year" style="font-weight: bold;">2006</span> <span class="NLM_volume">46</span> <span class="NLM_fpage">243</span> <span class="NLM_lpage">253</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci0502855" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Fjm8006239&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtF2ku77N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2006&pages=243-253&author=N.+Huangauthor=C.+Kalyanaramanauthor=J.+J.+Irwinauthor=M.+P.+Jacobson&title=Physics-based+scoring+of+protein%E2%88%92ligand+complexes%3A+enrichment+of+known+inhibitors+in+large-scale+virtual+screening"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Physics-Based Scoring of Protein-Ligand Complexes: Enrichment of Known Inhibitors in Large-Scale Virtual Screening</span></div><div class="casAuthors">Huang, Niu; Kalyanaraman, Chakrapani; Irwin, John J.; Jacobson, Matthew P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">243-253</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The authors demonstrate that using an all-atom mol. mechanics force field combined with an implicit solvent model for scoring protein-ligand complexes is a promising approach for improving inhibitor enrichment in the virtual screening of large compd. databases.  The rescoring method is evaluated by the extent to which known binders for nine diverse, therapeutically relevant enzymes are enriched against a background of ∼100 000 drug-like decoys.  The improvement in enrichment is most robust and dramatic within the top 1% of the ranked database, i.e., the first thousand compds.; below the first few percent of the ranked database, there is little overall improvement.  The improved early enrichment is likely due to the more realistic treatment of ligand and receptor desolvation in the rescoring procedure.  We also present anecdotal but encouraging results assessing the ability of the rescoring method to predict specificity of inhibitors for structurally related proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrUqkz9N6IErLVg90H21EOLACvtfcHk0libKEP9mHf43g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtF2ku77N&md5=cf636695aca21a87e050e0f39286ea1d</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Fci0502855&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci0502855%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DN.%26aulast%3DKalyanaraman%26aufirst%3DC.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DJacobson%26aufirst%3DM.%2BP.%26atitle%3DPhysics-based%2520scoring%2520of%2520protein%25E2%2588%2592ligand%2520complexes%253A%2520enrichment%2520of%2520known%2520inhibitors%2520in%2520large-scale%2520virtual%2520screening%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2006%26volume%3D46%26spage%3D243%26epage%3D253" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Wei, B. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weaver, L. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferrari, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matthews, B. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Testing a flexible-receptor docking algorithm in a model binding site</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_year" style="font-weight: bold;">2004</span> <span class="NLM_volume">337</span> <span class="NLM_fpage">1161</span> <span class="NLM_lpage">1182</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Fjm8006239&amp;key=10.1016%2Fj.jmb.2004.02.015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Fjm8006239&amp;key=15046985" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Fjm8006239&amp;key=1%3ACAS%3A528%3ADC%252BD2cXisVSmsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=337&publication_year=2004&pages=1161-1182&author=B.+Q.+Weiauthor=L.+H.+Weaverauthor=A.+M.+Ferrariauthor=B.+W.+Matthewsauthor=B.+K.+Shoichet&title=Testing+a+flexible-receptor+docking+algorithm+in+a+model+binding+site"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Testing a Flexible-receptor Docking Algorithm in a Model Binding Site</span></div><div class="casAuthors">Wei, Binqing Q.; Weaver, Larry H.; Ferrari, Anna M.; Matthews, Brian W.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">337</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1161-1182</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Sampling receptor flexibility is challenging for database docking.  We consider a method that treats multiple flexible regions of the binding site independently, recombining them to generate different discrete conformations.  This algorithm scales linearly rather than exponentially with the receptor's degrees of freedom.  The method was first evaluated for its ability to identify known ligands of a hydrophobic cavity mutant of T4 lysozyme (L99A).  Some 200,000 mols. of the Available Chem. Directory (ACD) were docked against an ensemble of cavity conformations.  Surprisingly, the enrichment of known ligands from among a much larger no. of decoys in the ACD was worse than simply docking to the apo conformation alone.  Large decoys, accommodated in the larger cavity conformations sampled in the ensemble, were ranked better than known small ligands.  The calcn. was redone with an energy correction term that considered the cost of forming the larger cavity conformations.  Enrichment improved, as did the balance between high-ranking large and small ligands.  In a second retrospective test, the ACD was docked against a conformational ensemble of thymidylate synthase.  Compared to docking against individual enzyme conformations, the flexible receptor docking approach improved enrichment of known ligands.  Including a receptor conformational energy weighting term improved enrichment further.  To test the method prospectively, the ACD database was docked against another cavity mutant of lysozyme (L99A/M102Q).  A total of 18 new compds. predicted to bind this polar cavity and to change its conformation were tested exptl.; 14 were found to bind.  The bound structures for seven ligands were detd. by x-ray crystallog.  The predicted geometries of these ligands all corresponded to the obsd. geometries to within 0.7 Å RMSD or better.  Significant conformational changes of the cavity were obsd. in all seven complexes.  In five structures, part of the obsd. accommodations were correctly predicted; in two structures, the receptor conformational changes were unanticipated and thus never sampled.  These results suggest that although sampling receptor flexibility can lead to novel ligands that would have been missed when docking a rigid structure, it is also important to consider receptor conformational energy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3S7C0vkfqQrVg90H21EOLACvtfcHk0lgdhOC6P4H7IA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXisVSmsbg%253D&md5=6343c78da4cd0b0171dedb660ec9cacd</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2004.02.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2004.02.015%26sid%3Dliteratum%253Aachs%26aulast%3DWei%26aufirst%3DB.%2BQ.%26aulast%3DWeaver%26aufirst%3DL.%2BH.%26aulast%3DFerrari%26aufirst%3DA.%2BM.%26aulast%3DMatthews%26aufirst%3DB.%2BW.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DTesting%2520a%2520flexible-receptor%2520docking%2520algorithm%2520in%2520a%2520model%2520binding%2520site%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2004%26volume%3D337%26spage%3D1161%26epage%3D1182" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Pettersen, E. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goddard, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Couch, G. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greenblatt, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meng, E. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferrin, T. E.</span><span> </span><span class="NLM_article-title">UCSF Chimera—a visualization system for exploratory research and analysis</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_year" style="font-weight: bold;">2004</span> <span class="NLM_volume">25</span> <span class="NLM_fpage">1605</span> <span class="NLM_lpage">1612</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2004&pages=1605-1612&author=E.+F.+Pettersenauthor=T.+D.+Goddardauthor=C.+C.+Huangauthor=G.+S.+Couchauthor=D.+M.+Greenblattauthor=E.+C.+Mengauthor=T.+E.+Ferrin&title=UCSF+Chimera%E2%80%94a+visualization+system+for+exploratory+research+and+analysis"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DPettersen%26aufirst%3DE.%2BF.%26aulast%3DGoddard%26aufirst%3DT.%2BD.%26aulast%3DHuang%26aufirst%3DC.%2BC.%26aulast%3DCouch%26aufirst%3DG.%2BS.%26aulast%3DGreenblatt%26aufirst%3DD.%2BM.%26aulast%3DMeng%26aufirst%3DE.%2BC.%26aulast%3DFerrin%26aufirst%3DT.%2BE.%26atitle%3DUCSF%2520Chimera%25E2%2580%2594a%2520visualization%2520system%2520for%2520exploratory%2520research%2520and%2520analysis%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2004%26volume%3D25%26spage%3D1605%26epage%3D1612" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-si1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i3"><a href="/doi/suppl/10.1021/jm8006239">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_33228"></div></div></div></div></div><hr /></hr><p class="last">The affect of modeling waters displaced on ligand binding, the dependence of run time, and enrichment on the number of waters modeled. This material is available free of charge via the Internet at <a class="extLink" href="http://pubs.acs.org">http://pubs.acs.org</a>. Protein structures and water positions are available from <a class="extLink" href="http://dud.docking.org/jmc-water">http://dud.docking.org/jmc-water</a>.</p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm8006239/suppl_file/jm8006239-file002.pdf">jm8006239-file002.pdf (538.82 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Fjm8006239&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2008.51.issue-16%3Bwebsite%3Awebsite%3Aacspubs%3Barticle%3Aarticle%3A10.1021%2Fjm8006239%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/jm8006239" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                1MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"67c745080cbb3bec","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
