
<html lang="en"     class="pb-page"  data-request-id="18a602dd-517b-40c0-947b-626e25f26824"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;issue:issue:10.1021/jmcmar.2019.62.issue-21;pageGroup:string:Publication Pages;article:article:10.1021/acs.jmedchem.9b01238;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="High-Affinity Alkynyl Bisubstrate Inhibitors of Nicotinamide N-Methyltransferase (NNMT)" /></meta><meta name="dc.Creator" content="Rocco&#xA;L.  Policarpo" /></meta><meta name="dc.Creator" content="Ludovic  Decultot" /></meta><meta name="dc.Creator" content="Elizabeth  May" /></meta><meta name="dc.Creator" content="Petr  Kuzmič" /></meta><meta name="dc.Creator" content="Samuel  Carlson" /></meta><meta name="dc.Creator" content="Danny  Huang" /></meta><meta name="dc.Creator" content="Vincent  Chu" /></meta><meta name="dc.Creator" content="Brandon A.  Wright" /></meta><meta name="dc.Creator" content="Saravanakumar  Dhakshinamoorthy" /></meta><meta name="dc.Creator" content="Aimo  Kannt" /></meta><meta name="dc.Creator" content="Shilpa  Rani" /></meta><meta name="dc.Creator" content="Sreekanth  Dittakavi" /></meta><meta name="dc.Creator" content="Joseph D.  Panarese" /></meta><meta name="dc.Creator" content="Rachelle  Gaudet" /></meta><meta name="dc.Creator" content="Matthew D.  Shair" /></meta><meta name="dc.Description" content="Nicotinamide N-methyltransferase (NNMT) is a metabolic enzyme that methylates nicotinamide (NAM) using cofactor S-adenosylmethionine (SAM). NNMT overexpression has been linked to diabetes, obesity,..." /></meta><meta name="Description" content="Nicotinamide N-methyltransferase (NNMT) is a metabolic enzyme that methylates nicotinamide (NAM) using cofactor S-adenosylmethionine (SAM). NNMT overexpression has been linked to diabetes, obesity,..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="October 7, 2019" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.9b01238" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2019 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b01238" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.9b01238" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b01238" /></link>
        
    
    

<title>High-Affinity Alkynyl Bisubstrate Inhibitors of Nicotinamide N-Methyltransferase (NNMT) | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.9b01238" /></meta><meta property="og:title" content="High-Affinity Alkynyl Bisubstrate Inhibitors of Nicotinamide N-Methyltransferase (NNMT)" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01238/20191108/images/large/jm9b01238_0008.jpeg" /></meta><meta property="og:description" content="Nicotinamide N-methyltransferase (NNMT) is a metabolic enzyme that methylates nicotinamide (NAM) using cofactor S-adenosylmethionine (SAM). NNMT overexpression has been linked to diabetes, obesity, and various cancers. In this work, structure-based rational design led to the development of potent and selective alkynyl bisubstrate inhibitors of NNMT. The reported nicotinamide–SAM conjugate (named NS1) features an alkyne as a key design element that closely mimics the linear, 180° transition state geometry found in the NNMT-catalyzed SAM → NAM methyl transfer reaction. NS1 was synthesized in 14 steps and found to be a high-affinity, subnanomolar NNMT inhibitor. An X-ray cocrystal structure and SAR study revealed the ability of an alkynyl linker to span the methyl transfer tunnel of NNMT with ideal shape complementarity. The compounds reported in this work represent the most potent and selective NNMT inhibitors reported to date. The rational design principle described herein could potentially be extended to other methyltransferase enzymes." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.9b01238"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b01238">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-8/abseba.2021.7.issue-8/20210809/abseba.2021.7.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-15/accacs.2021.11.issue-15/20210806/accacs.2021.11.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-31/acsodf.2021.6.issue-31/20210810/acsodf.2021.6.issue-31.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-31/ascecg.2021.9.issue-31/20210809/ascecg.2021.9.issue-31.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-31/ancham.2021.93.issue-31/20210810/ancham.2021.93.issue-31.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-31/bichaw.2021.60.issue-31/20210810/bichaw.2021.60.issue-31.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-8/bomaf6.2021.22.issue-8/20210809/bomaf6.2021.22.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-29/cgeabj.2021.99.issue-29/20210809-01/cgeabj.2021.99.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-15/cmatex.2021.33.issue-15/20210810/cmatex.2021.33.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-15/enfuem.2021.35.issue-15/20210805/enfuem.2021.35.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-8/estlcu.2021.8.issue-8/20210810/estlcu.2021.8.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-8/jceda8.2021.98.issue-8/20210810/jceda8.2021.98.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-8/jctcce.2021.17.issue-8/20210810/jctcce.2021.17.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-15/joceah.2021.86.issue-15/20210806/joceah.2021.86.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-30/jpcafh.2021.125.issue-30/20210805/jpcafh.2021.125.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-30/jpcbfk.2021.125.issue-30/20210805/jpcbfk.2021.125.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-30/jpccck.2021.125.issue-30/20210805/jpccck.2021.125.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-30/jpclcd.2021.12.issue-30/20210805/jpclcd.2021.12.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-8/jprobs.2021.20.issue-8/20210806/jprobs.2021.20.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-31/langd5.2021.37.issue-31/20210810/langd5.2021.37.issue-31.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-15/mamobx.2021.54.issue-15/20210810/mamobx.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-15/orlef7.2021.23.issue-15/20210806/orlef7.2021.23.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-15/orgnd7.2021.40.issue-15/20210809/orgnd7.2021.40.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.9b01238&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.9b01238&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.9b01238&amp;href=/doi/10.1021/acs.jmedchem.9b01238" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2019</span><span class="cit-fg-volume">, 62</span><span class="cit-fg-issue">, 21</span><span class="cit-fg-pageRange">, 9837-9873</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/62/21" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.9b01220" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.9b01283" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">High-Affinity Alkynyl Bisubstrate Inhibitors of Nicotinamide <i>N</i>-Methyltransferase (NNMT)</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Rocco L. Policarpo</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Rocco L. Policarpo</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry and Chemical Biology, Harvard University, Cambridge, Massachusetts 02138, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Rocco%0AL.++Policarpo">Rocco L. Policarpo</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-9819-2394" title="Orcid link">http://orcid.org/0000-0001-9819-2394</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ludovic Decultot</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ludovic Decultot</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry and Chemical Biology, Harvard University, Cambridge, Massachusetts 02138, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ludovic++Decultot">Ludovic Decultot</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Elizabeth May</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Elizabeth May</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Molecular and Cellular Biology, Harvard University, Cambridge, Massachusetts 02138, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Elizabeth++May">Elizabeth May</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Petr Kuzmič</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Petr Kuzmič</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">BioKin Ltd., Watertown, Massachusetts 02472, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Petr++Kuzmi%C4%8D">Petr Kuzmič</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Samuel Carlson</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Samuel Carlson</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Molecular and Cellular Biology, Harvard University, Cambridge, Massachusetts 02138, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Samuel++Carlson">Samuel Carlson</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Danny Huang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Danny Huang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry and Chemical Biology, Harvard University, Cambridge, Massachusetts 02138, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Danny++Huang">Danny Huang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Vincent Chu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Vincent Chu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry and Chemical Biology, Harvard University, Cambridge, Massachusetts 02138, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Vincent++Chu">Vincent Chu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Brandon A. Wright</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Brandon A. Wright</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry and Chemical Biology, Harvard University, Cambridge, Massachusetts 02138, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Brandon+A.++Wright">Brandon A. Wright</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-4149-5292" title="Orcid link">http://orcid.org/0000-0003-4149-5292</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Saravanakumar Dhakshinamoorthy</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Saravanakumar Dhakshinamoorthy</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Jubilant Biosys Ltd., Yeshwantpur, Bangalore, 560 022 Karnataka, India</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Saravanakumar++Dhakshinamoorthy">Saravanakumar Dhakshinamoorthy</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Aimo Kannt</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Aimo Kannt</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Sanofi Research and Development, Industriepark Hoechst, H823, D-65926 Frankfurt am Main, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Aimo++Kannt">Aimo Kannt</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Shilpa Rani</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Shilpa Rani</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Jubilant Biosys Ltd., Yeshwantpur, Bangalore, 560 022 Karnataka, India</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Shilpa++Rani">Shilpa Rani</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Sreekanth Dittakavi</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Sreekanth Dittakavi</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Jubilant Biosys Ltd., Yeshwantpur, Bangalore, 560 022 Karnataka, India</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Sreekanth++Dittakavi">Sreekanth Dittakavi</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Joseph D. Panarese</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Joseph D. Panarese</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry and Chemical Biology, Harvard University, Cambridge, Massachusetts 02138, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Joseph+D.++Panarese">Joseph D. Panarese</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Rachelle Gaudet</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Rachelle Gaudet</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Molecular and Cellular Biology, Harvard University, Cambridge, Massachusetts 02138, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Rachelle++Gaudet">Rachelle Gaudet</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Matthew D. Shair</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Matthew D. Shair</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry and Chemical Biology, Harvard University, Cambridge, Massachusetts 02138, United States</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#72011a131b0032111a171f1b0106000b5c1a1300041300165c171607"><span class="__cf_email__" data-cfemail="25564d444c5765464d40484c5651575c0b4d4457534457410b404150">[email protected]</span></a>. Phone: 617-495-5008.</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Matthew+D.++Shair">Matthew D. Shair</a></span></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b01238&amp;href=/doi/10.1021%2Facs.jmedchem.9b01238" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2019</span></span><span class="cit-volume">, 62</span><span class="cit-issue">, 21</span><span class="cit-pageRange">, 9837–9873</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">October 7, 2019</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>5 August 2019</li><li><span class="item_label"><b>Published</b> online</span>7 October 2019</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 14 November 2019</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.9b01238" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01238</a></div><div class="article_header-article-copyright"><strong>Copyright © 2019 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D9837%26pageCount%3D37%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DRocco%250AL.%2BPolicarpo%252C%2BLudovic%2BDecultot%252C%2BElizabeth%2BMay%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D62%26issueNum%3D21%26contentID%3Dacs.jmedchem.9b01238%26title%3DHigh-Affinity%2BAlkynyl%2BBisubstrate%2BInhibitors%2Bof%2BNicotinamide%2BN-Methyltransferase%2B%2528NNMT%2529%26numPages%3D37%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D9873%26publicationDate%3DNovember%2B2019">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.9b01238"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1919</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">15</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.9b01238" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;High-Affinity Alkynyl Bisubstrate Inhibitors of Nicotinamide N-Methyltransferase (NNMT)&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Rocco\nL.&quot;,&quot;last_name&quot;:&quot;Policarpo&quot;},{&quot;first_name&quot;:&quot;Ludovic&quot;,&quot;last_name&quot;:&quot;Decultot&quot;},{&quot;first_name&quot;:&quot;Elizabeth&quot;,&quot;last_name&quot;:&quot;May&quot;},{&quot;first_name&quot;:&quot;Petr&quot;,&quot;last_name&quot;:&quot;Kuzmič&quot;},{&quot;first_name&quot;:&quot;Samuel&quot;,&quot;last_name&quot;:&quot;Carlson&quot;},{&quot;first_name&quot;:&quot;Danny&quot;,&quot;last_name&quot;:&quot;Huang&quot;},{&quot;first_name&quot;:&quot;Vincent&quot;,&quot;last_name&quot;:&quot;Chu&quot;},{&quot;first_name&quot;:&quot;Brandon&quot;,&quot;last_name&quot;:&quot;A. Wright&quot;},{&quot;first_name&quot;:&quot;Saravanakumar&quot;,&quot;last_name&quot;:&quot;Dhakshinamoorthy&quot;},{&quot;first_name&quot;:&quot;Aimo&quot;,&quot;last_name&quot;:&quot;Kannt&quot;},{&quot;first_name&quot;:&quot;Shilpa&quot;,&quot;last_name&quot;:&quot;Rani&quot;},{&quot;first_name&quot;:&quot;Sreekanth&quot;,&quot;last_name&quot;:&quot;Dittakavi&quot;},{&quot;first_name&quot;:&quot;Joseph&quot;,&quot;last_name&quot;:&quot;D. Panarese&quot;},{&quot;first_name&quot;:&quot;Rachelle&quot;,&quot;last_name&quot;:&quot;Gaudet&quot;},{&quot;first_name&quot;:&quot;Matthew&quot;,&quot;last_name&quot;:&quot;D. Shair&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2019&quot;,&quot;month&quot;:&quot;10&quot;,&quot;day&quot;:&quot;07&quot;,&quot;issue&quot;:&quot;21&quot;,&quot;volume&quot;:&quot;62&quot;,&quot;pages&quot;:&quot;9837-9873&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.9b01238&quot;},&quot;abstract&quot;:&quot;Nicotinamide N-methyltransferase (NNMT) is a metabolic enzyme that methylates nicotinamide (NAM) using cofactor S-adenosylmethionine (SAM). NNMT overexpression has been linked to diabetes, obesity, and various cancers. In this work, structure-based rational design led to the development of potent and selective alkynyl bisubstrate inhibitors of NNMT. The reported nicotinamide–SAM conjugate (named NS1) features an alkyne as a key design element that closely mimics the linear, 180° transition state geometry found in the NNMT-catalyzed SAM → NAM methyl transfer reaction. NS1 was synthesized in 14 steps and found to be a high-affinity, subnanomolar NNMT inhibitor. An X-ray cocrystal structure and SAR study revealed the ability of an alkynyl linker to span the methyl transfer tunnel of NNMT with ideal shape complementarity. The compounds reported in this work represent the most potent and selective NNMT inhibitors reported to date. The rational design principle described herein could potentially be extended to othe&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01238&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01238" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01238&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01238" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01238&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01238" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b01238&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01238&amp;href=/doi/10.1021/acs.jmedchem.9b01238" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.9b01238" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.9b01238" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (5 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01238&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.9b01238%26sid%3Dliteratum%253Aachs%26pmid%3D31589440%26genre%3Darticle%26aulast%3DPolicarpo%26date%3D2019%26atitle%3DHigh-Affinity%2BAlkynyl%2BBisubstrate%2BInhibitors%2Bof%2BNicotinamide%2BN-Methyltransferase%2B%2528NNMT%2529%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D62%26issue%3D21%26spage%3D9837%26epage%3D9873%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (8)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291456" title="Hydrocarbons">Hydrocarbons</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=292349" title="Alkyls">Alkyls</a>,</li><li><a href="/action/doSearch?ConceptID=291071" title="Nicotinamide">Nicotinamide</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/62/21" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/jmcmar.2019.62.issue-21/20191114/jmcmar.2019.62.issue-21.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01238/20191108/images/medium/jm9b01238_0008.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01238/20191108/images/large/jm9b01238_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01238&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Nicotinamide <i>N</i>-methyltransferase (NNMT) is a metabolic enzyme that methylates nicotinamide (NAM) using cofactor <i>S</i>-adenosylmethionine (SAM). NNMT overexpression has been linked to diabetes, obesity, and various cancers. In this work, structure-based rational design led to the development of potent and selective alkynyl bisubstrate inhibitors of NNMT. The reported nicotinamide–SAM conjugate (named NS1) features an alkyne as a key design element that closely mimics the linear, 180° transition state geometry found in the NNMT-catalyzed SAM → NAM methyl transfer reaction. NS1 was synthesized in 14 steps and found to be a high-affinity, subnanomolar NNMT inhibitor. An X-ray cocrystal structure and SAR study revealed the ability of an alkynyl linker to span the methyl transfer tunnel of NNMT with ideal shape complementarity. The compounds reported in this work represent the most potent and selective NNMT inhibitors reported to date. The rational design principle described herein could potentially be extended to other methyltransferase enzymes.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_78860" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_78860" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Nicotinamide <i>N</i>-methyltransferase (NNMT) is a metabolic enzyme responsible for the methylation of nicotinamide (NAM) using the cofactor <i>S</i>-adenosylmethionine (SAM), resulting in the production of 1-methylnicotinamide (MNAM) and <i>S</i>-adenosylhomocysteine (SAH).<a onclick="showRef(event, 'ref1 ref2'); return false;" href="javascript:void(0);" class="ref ref1 ref2">(1,2)</a> While initially established as a vitamin B3 clearance enzyme, recent work has shown NNMT to be an important regulator of several intracellular pathways in fat, liver, and cancer cells.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> NNMT can regulate methyl donor metabolism by direct interaction with proteins of the methionine cycle<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> and can regulate hepatic nutrient metabolism through Sirt1 stabilization.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> By decreasing cellular SAM levels, NNMT can modulate the epigenetic landscape of cancer cells<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> and embryonic stem cells.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a></div><div class="NLM_p">Accordingly, NNMT has emerged as a disease-relevant enzyme and a possible point of therapeutic intervention. Abnormal NNMT expression is implicated in several diseases, including diabetes,<a onclick="showRef(event, 'ref7 ref8'); return false;" href="javascript:void(0);" class="ref ref7 ref8">(7,8)</a> obesity,<a onclick="showRef(event, 'ref7 ref9'); return false;" href="javascript:void(0);" class="ref ref7 ref9">(7,9)</a> and a variety of cancers.<a onclick="showRef(event, 'ref2 ref5 ref10'); return false;" href="javascript:void(0);" class="ref ref2 ref5 ref10">(2,5,10)</a> Increased NNMT protein expression was observed in adipose and liver tissue of obese and diabetic mice,<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> while increased NNMT mRNA expression was observed in humans.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> NNMT is overexpressed in glioblastoma, leading to cellular SAM depletion, DNA hypomethylation, and accelerated tumor growth.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> Recently, NNMT was shown to be a master metabolic regulator of cancer-associated fibroblasts, with NNMT expression supporting ovarian cancer migration, proliferation, and metastasis.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> The successful development of potent and selective NNMT inhibitors would aid efforts to elucidate the role of NNMT in disease, potentially enabling new strategies to treat a variety of metabolic disorders, cancers, and other pathologies.</div><div class="NLM_p">Several NNMT inhibitors have been reported, including methylated quinolines,<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> nicotinamide analogues,<a onclick="showRef(event, 'ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref15 ref16">(15,16)</a> covalent inhibitors,<a onclick="showRef(event, 'ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref17 ref18">(17,18)</a> and amino-adenosine derived bisubstrate inhibitors.<a onclick="showRef(event, 'ref19 ref20 ref21'); return false;" href="javascript:void(0);" class="ref ref19 ref20 ref21">(19−21)</a> Bisubstrate inhibitors are compounds that bind both the substrate and cofactor binding pockets within an enzyme active site. The design of bisubstrate inhibitors relates closely to two well-established ligand design strategies: transition-state mimicry and fragment-based design.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> Importantly, if two weak inhibitors that bind a protein at distinct sites are linked in an optimal manner, the resulting bisubstrate inhibitors can obtain several orders of magnitude increase in binding affinity.<a onclick="showRef(event, 'ref23 ref24'); return false;" href="javascript:void(0);" class="ref ref23 ref24">(23,24)</a> As a result, linker identity is crucial to properly position the key interacting groups of the two component molecules and minimize unfavorable interactions between the linker and the protein.</div><div class="NLM_p">In this work, we used a combination of structure-based design and molecular docking to develop high-affinity alkynyl bisubstrate inhibitors of NNMT. To begin, we examined the substrate-bound NNMT cocrystal structure (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ROD">3ROD</a>) and noted that the NAM nitrogen and SAH sulfur atoms are positioned ∼4 Å apart and reside in a small tunnel that facilitates the S<sub>N</sub>2-type methyl transfer reaction (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>a,b).<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> We envisioned an alkyne as the optimal linker between a NAM-like and a SAM-like fragment, as an alkyne linker resembles the linear, 180° transition state geometry found in the SAM → NAM methyl transfer reaction (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>c,d).</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01238/20191108/images/medium/jm9b01238_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01238/20191108/images/large/jm9b01238_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Overview of the alkynyl bisubstrate strategy. (a) Binding pose of ligands SAH and NAM (teal) rendered from previously published cocrystal structure (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ROD">3ROD</a>). (b) Graphical representation of the SAM → NAM methyl transfer reaction catalyzed by NNMT. (c) Graphical representation of alkynyl bisubstrate inhibitor NS1 posed to mimic the binding geometry of SAM/NAM. (d) Molecular docking output pose of alkynyl bisubstrate inhibitor NS1 (black) overlaid with SAH and NAM from structure <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ROD">3ROD</a> (teal).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01238/20191108/images/large/jm9b01238_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01238&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Thus, we rationally designed alkynyl nicotinamide–SAM conjugate 1 (named NS1, <b>10</b>). Molecular docking with Glide<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> predicted NS1 to be a better binder than SAM by >3 kcal/mol (see Supporting Information, <a class="ref internalNav" href="#notes1" aria-label="Part 1">Part 1</a>). The docked pose of NS1 (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>d) overlays well with NAM and SAH in the previously published substrate-bound crystal structure; the alkyne linker fits well into the methyl transfer tunnel. With molecular docking suggesting NS1 as a potential NNMT inhibitor, we aimed to synthesize and test NS1 in an NNMT inhibition assay.</div><div class="NLM_p last">6′-Alkynyl nucleosides were first proposed and synthesized in 1990 as mixtures of alkynyl and amino acid diastereomers.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> Unfortunately, the designed alkynyl nucleosides were poor inhibitors when tested against catechol <i>O</i>-methyltransferase (COMT).<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> In this work, we demonstrate that the alkynyl bisubstrate strategy toward SAM-dependent methyltransferase inhibition is a valuable approach. We synthesized the rationally designed alkynyl bisubstrate compound NS1 as a single enantiomer and diastereomer in 14 steps and showed it to be a high-affinity, selective NNMT inhibitor with a <i>K</i><sub>i</sub> of 500 pM.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_66126" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_66126" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> NS1 Synthesis</h3><div class="NLM_p">The synthesis of NS1 (<b>10</b>, <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>) began with commercially available acetonide <b>1</b>. Alkylation of the corresponding triflate with lithium TMS-acetylide followed by silyl deprotection generated alkyne <b>2</b>. Copper-catalyzed semireduction<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> of <b>2</b> rendered terminal alkene <b>3</b>, which was converted to enal <b>4</b> by cross-metathesis with crotonaldehyde. Solvent and catalyst screening (Supporting Information, <a class="ref internalNav" href="#notes1" aria-label="Table S1">Table S1</a>, <a class="ref internalNav" href="#notes1" aria-label="Figure S1">Figure S1</a>) revealed methyl <i>tert</i>-butyl ether (MTBE) and the commercially available Nitro-Grela olefin metathesis catalyst<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> as an optimal solvent/catalyst system to effect this transformation. The key C6′ stereocenter was installed by a rhodium-catalyzed asymmetric alkynylation<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> to afford aldehyde <b>5</b> as a single diastereomer (see Supporting Information, <a class="ref internalNav" href="#notes1" aria-label="Part 1, Section 3">Part 1, Section 3</a> for numbering scheme). A Horner–Wadsworth–Emmons olefination with a phosphonoglycine derivative yielded enamide <b>6</b>. The C17′ stereocenter was set via rhodium-catalyzed asymmetric hydrogenation with a Burk-type 1,2-bis(phospholano)ethane (BPE) ligand,<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> providing (<i>S</i>)-trifluoroacetamide <b>7</b> as a single diastereomer. Subsequent acylation and Vorbrüggen glycosylation promoted by a 2,6-di-<i>tert</i>-butyl-4-methylpyridinium triflate salt<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> delivered nucleoside <b>8</b>. Introduction of the benzamide moiety by a Sonogashira cross-coupling and a global deprotection sequence completed the synthesis of NS1 (<b>10</b>) in a total of 14 steps with an overall yield of 9.1%. The small-molecule X-ray crystal structure of NS1·TFA was obtained, confirming the regiochemistry of the adenine nucleobase (N9-linkage) as well as the configuration of the C6′ and C17′ stereocenters.</div><figure id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01238/20191108/images/medium/jm9b01238_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01238/20191108/images/large/jm9b01238_0007.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis and X-ray Crystal Structure of NS1 (<b>10</b>)</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01238/20191108/images/large/jm9b01238_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01238&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Biochemical Evaluation of NS1 and Analogues</h3><div class="NLM_p">A fluorescence-based NNMT inhibition assay<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> revealed NS1 to be a highly potent, subnanomolar (500 pM) NNMT inhibitor. In the assay, NNMT catalyzes the SAM-dependent methylation of quinoline, forming 1-methylquinolinium (1MQ) and SAH. Quinoline as opposed to NAM as methyl group acceptor has been previously employed by Loring and Thompson,<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> who determined the detailed kinetic mechanism of NNMT (rapid equilibrium random Bi Bi in Cleland’s nomenclature<a onclick="showRef(event, 'ref36 ref37 ref38'); return false;" href="javascript:void(0);" class="ref ref36 ref37 ref38">(36−38)</a>) and thus indirectly established that NAM and quinoline are mechanistically equivalent as mutually interchangeable methyl group acceptors.</div><div class="NLM_p">More importantly, for the purposes of our inhibition study, quinoline enables continuous real-time monitoring of the reaction progress, using fluorescence detection to monitor the appearance of 1MQ. In contrast, previously described NNMT assays using NAM as native substrate were forced to rely on discontinuous (end-point) detection<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> and are thus unsuitable for real-time recording of continuous reaction progress, as was done in our study. Utilization of a quinoline-based continuous real-time assay was especially important because many enzymatic progress curves were prominently nonlinear, both in the presence and in the absence of inhibitors (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). In the specific case of NS1, the reaction progress curves were strongly nonlinear even within the first 2 minutes of the inhibition assay and therefore no “initial linear region” could be identified. Thus, as a matter of principle, the inhibitors reported in this study could not be studied by conventional initial rate kinetic methods.<a onclick="showRef(event, 'ref40 ref41'); return false;" href="javascript:void(0);" class="ref ref40 ref41">(40,41)</a></div><figure id="fig2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01238/20191108/images/medium/jm9b01238_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01238/20191108/images/large/jm9b01238_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Kinetic studies of NNMT. A time course of the NNMT kinetic assay in the absence (red circles) or in the presence (remaining symbols) of NS1. Inhibitor concentrations are shown at top right. Δ<i>F</i> are scaled fluorescence changes in relative fluorescence units (“rfu”). The smooth curves represent the best least-squares fit to a system of first-order differential equations automatically generated by the DynaFit software package<a onclick="showRef(event, 'ref42 ref43'); return false;" href="javascript:void(0);" class="ref ref42 ref43">(42,43)</a> according to the reaction mechanism shown in the Supporting Information, <a class="ref internalNav" href="#notes1" aria-label="Part 3, Enzyme Kinetics)">Part 3, Enzyme Kinetics)</a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01238/20191108/images/large/jm9b01238_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01238&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Therefore, we analyzed the full reaction time-course using the software package DynaFit.<a onclick="showRef(event, 'ref42 ref43'); return false;" href="javascript:void(0);" class="ref ref42 ref43">(42,43)</a> The mathematical models for each of several postulated kinetic mechanisms were systems of first-order ordinary differential equations (ODEs) automatically derived by the DynaFit software. Inhibition constants were computed as the ratio <i>K</i><sub>i</sub> = <i>k</i><sub>off</sub>/<i>k</i><sub>on</sub>, where <i>k</i><sub>off</sub> is the microscopic rate constant for the dissociation of the enzyme–inhibitor complex and <i>k</i><sub>on</sub> is the corresponding microscopic association rate constant (for details, see Supporting Information, <a class="ref internalNav" href="#notes1" aria-label="Part 3, Section 2, pp. 13–15">Part 3, Section 2, pp. 13–15</a>). Because the <i>K</i><sub>i</sub> values reported in this work span 6 orders of magnitude, they are presented and discussed after logarithmic scaling, as p<i>K</i><sub>i</sub> (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>, <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>).</div><div class="NLM_p last">Note that the inhibition constants, <i>K</i><sub>i</sub>, reported in this study are true dissociation constants of the enzyme—inhibitor complex, as opposed to apparent inhibition constants, <i>K</i><sub>i</sub><sup>app</sup>, and therefore are entirely independent of the nature of the particular substrate (quinoline vs NAM).<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> This interpretation is based on the assumption, supported by the X-ray structural studies reported herein, that bisubstrate inhibitors in principle bind to the target enzyme in such a way that the binding of a bisubstrate inhibitor precludes any simultaneous binding of either of the two cosubstrates.</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> X-ray Cocrystal Structure of NS1 Bound to hNNMT</h3><div class="NLM_p">We determined the cocrystal structure of NS1-bound NNMT (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>, PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ORR">6ORR</a>, Supporting Information, <a class="ref internalNav" href="#notes1" aria-label="Table S2">Table S2</a>) and observed an NS1-shaped density in the NAM/SAM binding pocket in the electron density map (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>a). Fitting NS1 into the density decreased the <i>R</i><sub>free</sub> value and improved the map quality, revealing that NS1 occupies both the NAM and SAM binding sites in the same orientation as the native substrates, with the alkynyl linker occupying the methyl transfer tunnel. The NS1 alkyne is an optimal surrogate for the SAM-dependent methyl transfer reaction of NNMT and effectively positions both structural components of NS1 properly in space, allowing NS1 to capture similar binding interactions as the native substrates (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>b).</div><figure id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01238/20191108/images/medium/jm9b01238_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01238/20191108/images/large/jm9b01238_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Cocrystal structure of NS1-bound NNMT (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ORR">6ORR</a>). (a) A 2<i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> omit map contoured at 1σ reveals NS1-shaped electron density in the binding pocket of NNMT. (b) NS1 (black) binds NNMT in an orientation that closely resembles a previously published cocrystal structure containing ligands SAH and NAM (cyan; PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ROD">3ROD</a>). (c–e) NS1 also exploits the same protein interactions that NNMT uses to bind its native substrates. Ligand-interacting residues from NNMT are shown in yellow; black dotted lines represent hydrogen bonds. (f) MS2756-bound NNMT (brown; PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6B1A">6B1A</a>) superimposed on NS1-bound NNMT.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01238/20191108/images/large/jm9b01238_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01238&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">NS1 exploits the same amino acids that contact the native substrates to bind NNMT. To stabilize the adenosine portion of NS1, D142 forms a hydrogen bond with the primary amine on the adenine nucleobase, while the hydroxyl groups of the ribose hydrogen-bond extensively with the backbone nitrogen atom of S87 and the side chains of D85 and N90 (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>c). The carboxylic acid of the NS1 amino acid accepts four hydrogen bonds from Y20, Y25, Y69, and T163, while the amino group hydrogen-bonds with the backbone carbonyl of G63 (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>d). A hydrophobic clamp positions the aromatic ring of the benzamide moiety between Y204 and L164, with S201 and S213 stabilizing the amide group through hydrogen bonds (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>e).</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Stucture–Activity Relationships (SAR)</h3><div class="NLM_p">The synthesis of NS1 was designed to be highly modular to facilitate a systematic structure–activity analysis of our bisubstrate approach (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>,<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>). SAR analysis began with the minimal motif desthia-SAH (<b>11</b>) and built up to the full NS1 structure incrementally (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>a). Desthia-SAH, completely lacking an alkynyl-benzamide side chain, was a poor NNMT inhibitor. Addition of an acetylene (<b>12</b>) or a phenyl-acetylene (<b>13</b>) side chain yielded equally poor NNMT inhibitors. The potency of the parent compound NS1 (<b>10</b>) highlights the importance of the benzamide functionality in driving compound potency.</div><figure id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01238/20191108/images/medium/jm9b01238_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01238/20191108/images/large/jm9b01238_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Structure–activity relationship (SAR) studies of bisubstrate NNMT inhibitors. p<i>K</i><sub>i</sub> values (−log<sub>10</sub><i>K</i><sub>i</sub>) are shown in red and are used to facilitate comparison between inhibitors of widely varying potency. (a) Exploration of the alkynyl bisubstrate approach to NNMT inhibition. (b) Amino acid modifications.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01238/20191108/images/large/jm9b01238_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01238&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01238/20191108/images/medium/jm9b01238_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01238/20191108/images/large/jm9b01238_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. SAR studies of bisubstrate inhibitors. p<i>K</i><sub>i</sub> values (−log<sub>10</sub><i>K</i><sub><i>i</i></sub>) are shown in red and are used to facilitate comparison between inhibitors of widely varying potency. (a) Modification of the benzamide moiety of NS1. (b) Previously published amino-bisubstrate NNMT inhibitors were synthesized and tested alongside NS1 and analogues.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01238/20191108/images/large/jm9b01238_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01238&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">A fundamental aspect of our design strategy relied on the C6′ stereocenter to properly direct the NS1 alkyne toward the NAM binding pocket. Analysis of the NNMT crystal structure (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ROD">3ROD</a>) suggested that the <i>S</i> stereochemistry would best capture the transition state geometry modeled in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>. Accordingly, NS1 was a better inhibitor than the NS1-C6′ epimer (<b>14</b>) by >200-fold in terms of the inhibition constant <i>K</i><sub>i</sub>. The linear vector maintained by the alkynyl linker of NS1 proved optimal; NS1-alkane (<b>15</b>) was almost 10-fold less potent than NS1 itself (in terms of <i>K</i><sub>i</sub>). The NS1-alkane 6′ epimer (<b>16</b>) and alkynyl epimer <b>14</b> had similar p<i>K</i><sub>i</sub> values (6.97 vs 6.91, respectively). These results may be explained by the shape of the NNMT binding pocket; the narrow width of the methyl transfer tunnel may accommodate linear alkynyl geometry better than a staggered aliphatic chain. Furthermore, the rigidity of the alkynyl linker could lower the entropic cost of binding relative to alkane-containing analogues.</div><div class="NLM_p last">More drastic structural modifications to the NS1 scaffold included excision of the entire amino acid side chain (<b>17</b>) or the entire adenine nucleobase (<b>19</b>). Compound <b>17</b>, a simplified bisubstrate inhibitor designed to target the adenosine and NAM binding pockets of NNMT, was a weaker inhibitor than NS1 by ∼500-fold in terms of <i>K</i><sub>i</sub>. Attempts to improve the inhibitory activity of <b>17</b> by conformational restriction (via the installation of a cyclopropyl group; <b>18</b>) proved detrimental. Removal of the adenine nucleobase (<b>19</b>) completely abrogated NNMT inhibition.</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> SAR: Amino Acid Modification</h3><div class="NLM_p">Alteration of the NS1 amino acid (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>b) revealed the C17′ amino group of NS1 to be critical for inhibition, as compound <b>20</b> (lacking the C17′ amine) was a poor NNMT inhibitor. Removal of the carboxylic acid was better tolerated, as compound <b>21</b> (which retains the C17′ amine) had a p<i>K</i><sub>i</sub> of 7.24 ± 0.02. Amide- (<b>22</b>) and methyl ester-containing (<b>23</b>) analogues were poor NNMT inhibitors (5.98 ± 0.01 and 6.36 ± 0.08, respectively), while amino-amide- (<b>24</b>) or urea-containing (<b>25</b>) analogues were slightly better (7.77 ± 0.02 and 7.12 ± 0.05, respectively). These results are consistent with the structural features of <b>24</b> and <b>25</b>, as both contain the same number of potential hydrogen-bonding groups as NS1 and native substrate SAM (although differing slightly in their donor/acceptor capacities and positions). Compound <b>22</b> has one fewer hydrogen-bonding atom than NS1. In <b>23</b>, the methyl ester likely adds too much bulk to fit in the amino acid pocket while also reducing the hydrogen bonding ability of the amino acid carboxylate.</div><div class="NLM_p last">Only one side chain modification yielded a more potent compound than NS1: the addition of a single methylene unit in the NS1 backbone (<b>26</b>, homo-NS1). Homo-NS1 had a p<i>K</i><sub>i</sub> of 9.62 ± 0.05 (compared to NS1 with p<i>K</i><sub>i</sub> = 9.30 ± 0.07), indicating a ∼2-fold increase in inhibitory activity in terms of <i>K</i><sub>i</sub>. This finding is consistent with prior reports showing that the addition of a methylene unit in amino-bisubstrate NNMT inhibitors can improve potency.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> We also synthesized and tested previously reported compounds MS2734 and MS2756 (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>b). In our hands, MS2734, a one-carbon chain-extended homologue of MS2756, was a 10-fold better inhibitor than MS2756 in terms of <i>K</i><sub>i</sub><b>(</b>p<i>K</i><sub>i</sub> of 7.05 ± 0.01 vs 6.04 ± 0.06). The benefit of an extra methylene group in bisubstrate inhibitors could arise because an added methylene better mimics the longer C–S bonds of the native substrates. The trimethylene linker of homo-NS1 would also be more flexible than the dimethylene linker of NS1, which may allow the molecule more room to optimally satisfy hydrogen bonds between the amino acid, benzamide, ribose, and adenine moieties to the NNMT binding pocket.</div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> SAR: Aryl Modification</h3><div class="NLM_p">After examining the impact of side chain modifications on inhibitor potency, we tested the effects of aryl group modification. We purposely designed the NS1 synthesis to accommodate a late-stage Sonogashira cross-coupling reaction to allow for easy introduction of a desired aryl moiety late in our synthesis. Leveraging this capability, we prepared gram quantities of intermediate <b>8</b> (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>) and synthesized a variety of aryl substituted NS1 analogues (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>a).</div><div class="NLM_p">These studies revealed that proper positioning of an amide group on NS1 is essential for NNMT binding, as <i>para</i>-benzamide (<b>27</b>), <i>ortho</i>-benzamide (<b>28</b>), and sulfonamide (<b>29</b>) substituted NS1 analogues were much less potent than NS1. We designed fluorine (<b>30</b>), methyl (<b>31</b>), trifluoromethyl (<b>32</b>), and chlorine (<b>33</b>) containing analogues to (1) bind a small hydrophobic pocket in the nicotinamide binding site, (2) modulate the p<i>K</i><sub>a</sub> of the NS1 benzamide group through inductive effects, and (3) potentially alter the rotation of the NS1 amide relative to the NS1 benzamide phenyl ring. While <b>30</b>, <b>31</b>, and <b>32</b> were potent NNMT inhibitors (p<i>K</i><sub>i</sub> > 8.0), only chloro-substitution (<b>33</b>) yielded a more potent NNMT inhibitor than NS1 (p<i>K</i><sub>i</sub> = 9.72 ± 0.15). As shown in <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>a, there is a small hydrophobic region in the NAM binding pocket <i>para</i> to the alkyne of the NS1 scaffold. The small, hydrophobic chlorine atom of <b>33</b> likely makes van der Waals contacts with this region, providing additional binding affinity.</div><figure id="fig6" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01238/20191108/images/medium/jm9b01238_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01238/20191108/images/large/jm9b01238_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Chemical properties and geometry of NS1 closely match the hydrophobic profile and shape of the NNMT binding pocket. (a) The binding pocket is buried in the hydrophobic core of the protein (gray). Inset: A solvent-accessible surface representation of the binding pocket colored by the Eisenberg hydrophobicity scale<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> reveals a hydrophobic patch (dark red) that contacts the C6′ carbon atom of NS1. (b) A side-on view shows that the binding pocket has shape complementarity specific to the <i>S</i> diastereomer of NS1 at the C6′ carbon.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01238/20191108/images/large/jm9b01238_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01238&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">After interrogating ring substitution at the <i>para</i>-position, we attempted to restrict rotation of the NS1 amide by converting the benzamide moiety to a benzolactam (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>a, <b>34</b> and <b>35</b>). Benzolactams <b>34</b> and <b>35</b> were moderate NNMT inhibitors, with a six-membered ring (<b>34</b>) better tolerated than a five-membered ring (<b>35</b>). We also examined C13′/C14′ substitution with methylenedioxy analogue <b>36</b> (p<i>K</i><sub>i</sub> = 7.49 ± 0.05) or introduced a heterocyclic nitrogen atom (to better mimic NAM, <b>37</b>, <b>38</b>, <b>39</b>, and <b>40</b>). Compounds <b>36</b>–<b>40</b> all had p<i>K</i><sub>i</sub> values below that of NS1, demonstrating that C13′/C14′ substitution or inclusion of a nitrogen atom are not beneficial in these cases. Lastly, we designed and synthesized amino naphthalene derivative <b>41</b> (a bisubstrate analogue of the previously reported NNMT inhibitor 5-amino-1-methylquinolinium<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a>), but <b>41</b> was also a poor inhibitor (p<i>K</i><sub>i</sub> = 5.43 ± 0.07).</div></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> NS1-Alkane (<b>15</b>) vs Amino-Bisubstrate Inhibitor MS2756</h3><div class="NLM_p">In the final stage of our SAR analysis, we compared our inhibitors with previously reported amino bisubstrate NNMT inhibitors (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>b). Previously reported compounds VH45,<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> MS2756,<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> and MS2734<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> use a tertiary amine core and an alkyl tether to link a SAM-like fragment to a NAM-like fragment. An alignment of NS1-bound NNMT with MS2756-bound NNMT<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> shows the similar binding modes of the two compounds (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>f). We synthesized and tested all three of these compounds alongside NS1-alkane (<b>15</b>, <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>a) using the same assay protocols and kinetic analyses to determine <i>K</i><sub>i</sub> values. We compared MS2756 and NS1-alkane <i>directly</i> because they are close structural analogues in which the tertiary amine core of MS2756 has been replaced with an sp<sup>3</sup> hybridized carbon stereocenter. These studies revealed that NS1-alkane was approximately 200-fold more potent than MS2756 in terms of <i>K</i><sub>i</sub>.</div><div class="NLM_p">The contrast between the observed activity of MS2756 and NS1-alkane is noteworthy; changing a <i>single</i> non-hydrogen atom (tertiary nitrogen to sp<sup>3</sup> carbon stereocenter) substantially alters compound potency. We hypothesize that a variety of factors could be responsible for this observation. It is evident that the stereochemical definition at the C6′ stereocenter of NS1-alkane is beneficial, as evidenced by the ∼20-fold difference in <i>K</i><sub>i</sub> between NS1-alkane and its C6′ epimer (<b>16</b>). A much larger difference (∼250-fold) in <i>K</i><sub>i</sub> between NS1 and its C6′ epimer (<b>14</b>) was also observed, demonstrating that as linker rigidity increases, C6′ stereodefinition becomes more important.</div><div class="NLM_p">While NS1-alkane and NS1 both show large differences in <i>K</i><sub>i</sub> values between their C6′ stereoisomers, the tertiary nitrogen atom in MS2756 maintains sp<sup>3</sup> tetrahedral geometry but is free to invert rapidly at room temperature. In solution, we imagine that NS1-alkane and NS1 assume the optimal tetrahedral geometry at C6′ necessary for binding, whereas MS2756 is free to occupy conformations that are unproductive for binding because of nitrogen inversion. It is also likely that the hydrophobic, carbon-based scaffolds of NS1-alkane and NS1 are preferred over the polar, tertiary amine core of MS2756. NS1-alkane and NS1 likely form favorable van der Waals contacts with the NNMT active site, as the closest residue to the NS1-alkane/NS1 C6′ carbon stereocenter is F15 (∼4 Å away). This phenylalanine residue presents a large hydrophobic<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> surface in the NNMT ligand binding pocket (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>a), near the NS1-alkane/NS1 C6′ stereocenter.</div><div class="NLM_p last">Finally, the transition state structure of PNMT (phenylethanolamine <i>N</i>-methyltransferase<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a>), the second closest homologue of NNMT, involves even distribution of positive charge along the <i>entire</i> axis of methyl transfer.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> If NNMT has a transition state structure similar to that of PNMT, analogues like MS2756, which localize positive charge on a single nitrogen atom, do not accurately capture the transient electrostatics of the methyl transfer transition state. While not positively charged, carbon-based linkers avoid the localized positive charge of MS2756 and other amino-bisubstrate inhibitors.</div></div><div id="sec2_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Selectivity Evaluation</h3><div class="NLM_p">While several previously reported NNMT inhibitors were described as selective for NNMT, none of them have been tested against the closest homologues<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> of NNMT or other small-molecule methyltransferases.<a onclick="showRef(event, 'ref49 ref50'); return false;" href="javascript:void(0);" class="ref ref49 ref50">(49,50)</a> We aimed to critically evaluate the selectivity profile of NS1 by testing it against the methyltransferases most like NNMT (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Sequence similarity analysis and generation of a sequence similarity network<a onclick="showRef(event, 'ref51 ref52'); return false;" href="javascript:void(0);" class="ref ref51 ref52">(51,52)</a> (Supporting Information, <a class="ref internalNav" href="#notes1" aria-label="Figure S2, Table S3">Figure S2, Table S3</a>) showed that NNMT, INMT (indolethylamine <i>N</i>-methyltransferase<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a>), and PNMT cluster tightly, with INMT and PNMT having 53% and 39% sequence identity to NNMT, respectively. A complementary approach based on structural homology analysis via the DALI<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> server demonstrated that INMT and PNMT are also the closest structural homologues of NNMT (Supporting Information, <a class="ref internalNav" href="#notes1" aria-label="Figure S3, S4, Table S4">Figure S3, S4, Table S4</a>).<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> In addition to the selectivity of NS1 among small-molecule methyltransferases, we tested NS1 against a few representative DNA, histone, and protein–arginine methyltransferases (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, Supporting Information, <a class="ref internalNav" href="#notes1" aria-label="Table S5">Table S5</a>). These studies revealed thiopurine <i>S</i>-methyltransferase (TPMT) to be the closest off-target of NS1, with an IC<sub>50</sub> of ∼1 μM. This result might be explained by the flexibility of the TPMT active site<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a> and the ability of TPMT to bind a variety of small-molecule substrates and inhibitors.<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a> INMT was the second closest off-target of NS1, with an IC<sub>50</sub> of 3.4 μM (Supporting Information, <a class="ref internalNav" href="#notes1" aria-label="Figure S5">Figure S5</a>).</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Methyltransferase Selectivity Profile of NS1</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="center" /></col><col align="center" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">small-molecule methyltransferase</th><th class="colsep0 rowsep0" align="center">inhibition at 10 μM (%)</th><th class="colsep0 rowsep0" align="center">inhibition at 1 μM (%)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">TPMT<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="center">76</td><td class="colsep0 rowsep0" align="center">51</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">INMT<a class="ref internalNav" href="#t1fn2" aria-label="b">b</a><sup>,</sup><a class="ref internalNav" href="#t1fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="center">53</td><td class="colsep0 rowsep0" align="center">14</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">COMT<a class="ref internalNav" href="#t1fn4" aria-label="d">d</a></td><td class="colsep0 rowsep0" align="center">43</td><td class="colsep0 rowsep0" align="center">0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PNMT<a class="ref internalNav" href="#t1fn5" aria-label="e">e</a></td><td class="colsep0 rowsep0" align="center">26</td><td class="colsep0 rowsep0" align="center">0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">GAMT<a class="ref internalNav" href="#t1fn6" aria-label="f">f</a></td><td class="colsep0 rowsep0" align="center">26</td><td class="colsep0 rowsep0" align="center">0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">GNMT<a class="ref internalNav" href="#t1fn7" aria-label="g">g</a></td><td class="colsep0 rowsep0" align="center">23</td><td class="colsep0 rowsep0" align="center">4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">HNMT<a class="ref internalNav" href="#t1fn8" aria-label="h">h</a></td><td class="colsep0 rowsep0" align="center">2</td><td class="colsep0 rowsep0" align="center">6</td></tr></tbody></table><table class="table "><colgroup><col align="left" /></col><col align="center" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">protein or DNA methyltransferase</th><th class="colsep0 rowsep0" align="center">IC<sub>50</sub> (μM)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">hDNMT3a<a class="ref internalNav" href="#t1fn9" aria-label="i">i</a></td><td class="colsep0 rowsep0" align="center">7.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PRMT1<a class="ref internalNav" href="#t1fn10" aria-label="j">j</a></td><td class="colsep0 rowsep0" align="center">>10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ASH1L<a class="ref internalNav" href="#t1fn11" aria-label="k">k</a></td><td class="colsep0 rowsep0" align="center">>10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">DOT1L<a class="ref internalNav" href="#t1fn12" aria-label="l">l</a></td><td class="colsep0 rowsep0" align="center">>10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">EHMT1<a class="ref internalNav" href="#t1fn13" aria-label="m">m</a></td><td class="colsep0 rowsep0" align="center">>10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">G9a<a class="ref internalNav" href="#t1fn14" aria-label="n">n</a></td><td class="colsep0 rowsep0" align="center">>10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SETDB1<a class="ref internalNav" href="#t1fn15" aria-label="o">o</a></td><td class="colsep0 rowsep0" align="center">>10</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">Thiopurine <i>S</i>-methyltransferase.</p></div><div class="footnote" id="t1fn2"><sup><sup>b</sup></sup><p class="last">Indolethylamine <i>N</i>-methyltransferase.</p></div><div class="footnote" id="t1fn3"><sup><sup>c</sup></sup><p class="last">Values interpolated from the full IC<sub>50</sub> curve shown in Supporting Information, <a class="ref internalNav" href="#notes1" aria-label="Figure S5">Figure S5</a>.</p></div><div class="footnote" id="t1fn4"><sup><sup>d</sup></sup><p class="last">Catechol <i>O</i>-methyltransferase.</p></div><div class="footnote" id="t1fn5"><sup><sup>e</sup></sup><p class="last">Phenylethanolamine <i>N</i>-methyltransferase.</p></div><div class="footnote" id="t1fn6"><sup><sup>f</sup></sup><p class="last">Guanidinoacetate <i>N</i>-methyltransferase.</p></div><div class="footnote" id="t1fn7"><sup><sup>g</sup></sup><p class="last">Glycine <i>N</i>-methyltransferase.</p></div><div class="footnote" id="t1fn8"><sup><sup>h</sup></sup><p class="last">Histamine <i>N</i>-methyltransferase.</p></div><div class="footnote" id="t1fn9"><sup><sup>i</sup></sup><p class="last">DNA (cytosine-5)-methyltransferase 3A.</p></div><div class="footnote" id="t1fn10"><sup><sup>j</sup></sup><p class="last">Protein arginine <i>N</i>-methyltransferase 1.</p></div><div class="footnote" id="t1fn11"><sup><sup>k</sup></sup><p class="last">Histone-lysine <i>N</i>-methyltransferase ASH1L.</p></div><div class="footnote" id="t1fn12"><sup><sup>l</sup></sup><p class="last">Histone-lysine <i>N</i>-methyltransferase, H3 lysine-79 specific.</p></div><div class="footnote" id="t1fn13"><sup><sup>m</sup></sup><p class="last">Histone-lysine <i>N</i>-methyltransferase EHMT1.</p></div><div class="footnote" id="t1fn14"><sup><sup>n</sup></sup><p class="last">Histone-lysine <i>N</i>-methyltransferase EHMT2.</p></div><div class="footnote" id="t1fn15"><sup><sup>o</sup></sup><p class="last">Histone-lysine <i>N</i>-methyltransferase SETDB1.</p></div></div></div><div class="NLM_p last">While in this case NS1 proved selective for NNMT, the introduction of different aryl groups on the NS1 scaffold could likely be used to target other methyltransferases selectively. In this work, cross coupling of compound <b>8</b> (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>) with 3-iodobenzamide generated NS1; cross coupling of <b>8</b> with indole or thiopurine fragments could provide inhibitors of INMT and TPMT, respectively. In cases where off-target binding or inhibition is observed, aryl groups could be modified until a desired selectivity profile is achieved.</div></div><div id="sec2_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Thermal Stabilization and Cell-Based Evaluation</h3><div class="NLM_p">Along with biochemical profiling of NS1 and analogues, we found that NS1 binds and stabilizes NNMT in lysate of K562 cells (a chronic myelogenous leukemia line known to express NNMT<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a>). The cellular thermal shift assay<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a> revealed that NS1 and <b>25</b> shift the melting curve of NNMT by 8 and 5 °C, respectively (Supporting Information, <a class="ref internalNav" href="#notes1" aria-label="Figures S6, S8">Figures S6, S8</a>) and in a dose-dependent manner (Supporting Information, <a class="ref internalNav" href="#notes1" aria-label="Figures S7, S9">Figures S7, S9</a>). We also evaluated several compounds for cytotoxicity via the CellTiter-Glo luminescent cell viability assay in U2OS cells (a human bone osteosarcoma line known to express NNMT<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a>). None of the compounds tested (NS1, <b>21</b>, <b>23</b>, <b>24</b>, and <b>25</b>) were toxic to U2OS cells after 48 h at a 100 μM dose (Supporting Information, <a class="ref internalNav" href="#notes1" aria-label="Table S7">Table S7</a>).</div><div class="NLM_p last">We tested the ability of NS1, <b>21</b>, <b>23</b>, <b>24</b>, and <b>25</b> to decrease MNAM levels in U2OS cells, but the effects were modest (Supporting Information, <a class="ref internalNav" href="#notes1" aria-label="Table S8">Table S8</a>). NS1-methyl ester (<b>23</b>), designed to be more cell permeable than NS1, performed best in this assay, decreasing MNAM levels by 30% at a 31.6 μM dose. A–B permeability testing (Caco-2, pH 6.5/7.4) showed that NS1 and analogues have limited permeability (most permeable out of those tested was NS1-amine <b>21</b>, 1.3 × 10<sup>–6</sup> cm/s, Supporting Information, <a class="ref internalNav" href="#notes1" aria-label="Table S9">Table S9</a>). Ongoing work is aimed at developing more cell-penetrant alkynyl NNMT inhibitors that would be useful probes in cell-based studies.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i15">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_35066" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_35066" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">In this work, we designed and synthesized the most potent and selective NNMT inhibitors reported to date. A modular, scalable synthesis of alkynyl nucleosides enabled us to make and test a variety of alkynyl bisubstrate inhibitors, leading to the discovery of NS1 as a selective, picomolar NNMT inhibitor. Alkynyl bisubstrate inhibition could be a generalizable strategy; covalent tethering of alkynyl nucleosides to substrate mimics of other small-molecule methyltransferases could provide a method for selective inhibition of these enzymes. The stereodefined alkyne present in the NS1 scaffold could be used for click chemistry or other functionalization, enabling the development of bisubstrate probes or inhibitors of protein arginine/lysine methyltransferases.<a onclick="showRef(event, 'ref58 ref59'); return false;" href="javascript:void(0);" class="ref ref58 ref59">(58,59)</a></div><div class="NLM_p last">Finally, this work demonstrates the value of alkynes in small-molecule ligand design in the context of methyltransferase inhibition. Alkynes provide unique rigidity, geometry, and spatial occupancy not captured by common fragment linkers. Current state-of-the-art bisubstrate inhibitor designs often rely upon reductive amination, alkylation, and amide-bond forming reactions to link fragments together. These reactions are generally fast and easy to implement but often produce highly flexible structures characterized by relatively weak overall binding affinity. This report demonstrates that C6′ alkynyl nucleosides, while more synthetically complex, can be used to generate potent bisubstrate methyltransferase inhibitors.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i16">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_58852" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_58852" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> General Chemistry Procedures</h3><div class="NLM_p last">All reactions were performed under an atmosphere of nitrogen in flame- or oven-dried glassware unless otherwise noted. Reactions were monitored by thin layer chromatography (TLC) using Merck silica gel 60 covered glass backed plates (0.25 mm, F<sub>254</sub>). TLC plates were visualized under UV light and/or exposure to an acidic solution of <i>p</i>-anisaldehyde, an aqueous solution of ceric ammonium molybdate (CAM), or an aqueous solution of potassium permanganate, followed by heating. Reactions were also monitored by analytical LCMS using an Agilent 6120 Quadrupole MS and an Agilent 1260 Infinity II LC system equipped with (1) a Phenomenex Kinetex F5 column (2.6 μm, 100 Å, 100 mm × 3.0 mm), (2) a Phenomenex Kinetex Biphenyl column (2.6 μm, 100 Å, 100 mm × 3.0 mm), or (3) a Thermo Scientific Accucore aQ C18 column (2.6 μm, 100 mm × 2.1 mm). For analytical LCMS, solvent A consisted of 0.1% (v/v) FA/water and solvent B consisted of 0.1% (v/v) FA/CH<sub>3</sub>CN. Two methods were used: method A for columns 1, 2, and 3: 5% B for 1 min, 5% → 95% B over 8 min, 95% B for 2 min, and reequilibration for 2 min; flow rate, 0.55 mL/min (1), 0.95 mL/min (2), 0.70 mL/min (3), column temperature 50 °C; UV<sub>254</sub>, and method B for columns 2 and 3: 0% B for 1 min, 0% → 40% B over 8 min, 40% → 95% B over 2 min, 95% B for 2 min, and reequilibration for 2 min; flow rate 0.95 mL/min (2), 0.70 mL/min (3), column temperature: 50 °C; UV<sub>254</sub>. Column 2 (method B) was used for the final analysis of very polar compounds, including NS1 (<b>10</b>) and analogues. Flash column chromatography was performed on an Interchim PuriFlash 215 system with prepacked silica gel cartridges (Büchi FlashPure (35–45 μm), Büchi Reveleris HP (16–24 μm), Teledyne Isco RediSep Rf (35–70 μm) and Teledyne Isco RediSep Rf Gold (20–40 μm)). Preparative HPLC was performed on an Agilent 1260 Infinity II preparative LC system equipped with a Kromasil C18 column (10 μm, 100 Å, 21.2 mm × 250 mm) using solvents A (0.1% (v/v) FA/water) and B (0.1% (v/v) FA/CH<sub>3</sub>CN) or A′ (0.1% (v/v) TFA/water) and B′ (0.1% (v/v) TFA/CH<sub>3</sub>CN) at flow rate 10 mL/min delivering gradients specified in individual experimental protocols. Final compounds were prepared by lyophilization after HPLC purification. All compounds reported in this manuscript and <a class="ref internalNav" href="#notes1" aria-label="Supporting Information">Supporting Information</a> have a purity of ≥95% as determined by analytical LCMS and NMR (<sup>1</sup>H and <sup>13</sup>C) unless otherwise noted.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Instrumentation</h3><div class="NLM_p last"><sup>1</sup>H NMR spectra were recorded on Varian INOVA-600 or Varian INOVA-500 spectrometers at ambient temperature. Chemical shifts are reported in ppm (δ scale) relative to residual undeuterated solvent (CDCl<sub>3</sub>, 7.26 (CHCl<sub>3</sub>); CD<sub>3</sub>OD, 3.31 (CD<sub>2</sub>HOD); CD<sub>3</sub>CN, 1.94 (CD<sub>2</sub>HCN); D<sub>2</sub>O, 4.79 (HOD)). Data are reported as follows: chemical shift (δ ppm) (integration, multiplicity (s = singlet, d = doublet, t = triplet, q = quartet, m = multiplet, br = broad, app = apparent, or a combination of these), coupling constant(s) <i>J</i> (Hz)). When a mixture of deuterated solvents is employed, the residual undeuterated solvent used for reference is indicated in bold on the NMR spectra presented in the <a href="/doi/suppl/10.1021/acs.jmedchem.9b01238/suppl_file/jm9b01238_si_002.pdf" class="ext-link">Supporting Information</a>, Part 2. <sup>13</sup>C NMR spectra were recorded on Varian INOVA-500, Varian MERCURY-400, and JEOL J400 spectrometers at ambient temperature. Data are reported as follows: chemical shifts (δ scale, ppm) relative to the carbon resonances of the solvent (CDCl<sub>3</sub>, 77.16; CD<sub>3</sub>OD, 49.00; CD<sub>3</sub>CN, 1.32). When a mixture of deuterated solvents is employed, the solvent used for reference is indicated in bold. <sup>19</sup>F NMR spectra were recorded on Varian INOVA-500 and Varian MERCURY-400 spectrometers at ambient temperature. Data are reported as follows: chemical shifts (δ scale, ppm) relative to CFCl<sub>3</sub> and referenced to benzotrifluoride (BTF), hexafluorobenzene (HFB), or trifluoroacetic acid (TFA) as internal standards (BTF IStd, δ −63.7; HFB IStd, δ −164.9; TFA IStd, δ −76.6). <sup>31</sup>P NMR spectra were recorded on a Varian MERCURY-400 spectrometer at ambient temperature. Data are reported as follows: chemical shifts (δ scale, ppm), calibrated to an external standard of triphenyl phosphate in CDCl<sub>3</sub> (0.0485 M, δ −17.6 relative to H<sub>3</sub>PO<sub>4</sub> at δ 0). Infrared (FTIR) spectra were recorded on a Bruker Alpha FT-IR spectrophotometer. Data is reported in frequency of absorption (cm<sup>–1</sup>). The IR spectrum of each compound (solid or liquid) was acquired directly neat or on a thin layer at ambient temperature. High resolution mass spectra (HRMS) were recorded on a Bruker micrOTOF-Q II or an Agilent 6220 LC-TOF equipped with an electrospray ionization source (ESI+).</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> Materials</h3><div class="NLM_p last">Commercial reagents and solvents were used as received unless otherwise noted. All solvents were of commercially available anhydrous grade (DriSolv or equivalent) unless otherwise noted. The molarity of <i>n</i>-butyllithium solutions was determined by titration using <i>N</i>-benzylbenzamide<a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a> or <i>N</i>-pivaloyl-<i>o</i>-toluidine<a onclick="showRef(event, 'ref61'); return false;" href="javascript:void(0);" class="ref ref61">(61)</a> with similar results (average of three determinations). [Rh(OAc)(C<sub>2</sub>H<sub>4</sub>)<sub>2</sub>]<sub>2</sub> was prepared according to a known procedure<a onclick="showRef(event, 'ref62'); return false;" href="javascript:void(0);" class="ref ref62">(62)</a> using [RhCl(C<sub>2</sub>H<sub>4</sub>)<sub>2</sub>]<sub>2</sub> (CAS 12081-16-2) obtained from Strem Chemicals, Inc. (catalogue no. 45-0270). The following catalysts were all obtained from Strem Chemicals, Inc.: Nitro-Grela catalyst ([1,3-bis(2,4,6-trimethylphenylimidazolidin-2-ylidene)]-(2-<i>i</i>-propoxy-5-nitrobenzylidene)ruthenium(II) dichloride), CAS 502964-52-5, catalogue no. 44-0758; (<i>S</i>,<i>S</i>)-MeBPE-Rh catalyst<a onclick="showRef(event, 'ref63'); return false;" href="javascript:void(0);" class="ref ref63">(63)</a> ((−)-1,2-bis((2<i>S</i>,5<i>S</i>)-2,5-dimethylphospholano)ethane(1,5-cyclooctadiene)rhodium(I) tetrafluoroborate), CAS 213343-65-8, catalog no. 45-0169; (<i>S</i>)-DTBM-SegPhos ((<i>S</i>)-(+)-5,5′-bis[di(3,5-di-<i>t</i>-butyl-4-methoxyphenyl)phosphino]-4,4′-bi-1,3-benzodioxole), CAS 210169-40-7, catalogue no. 15-0067; [Rh(nbd)<sub>2</sub>]BF<sub>4</sub>, CAS 36620-11-8, catalogue no. 45-0230; (<i>S</i>,<i>S</i>)-MeDUPHOS ((+)-1,2-bis((2<i>S</i>,5<i>S</i>)-2,5-dimethylphospholano)benzene), CAS 136735-95-0, catalogue no. 15-0092. (±)-<i>Z</i>-α-Phosphonoglycine trimethyl ester, CAS 88568-95-0, was obtained from Chem-Impex International (catalogue no. 14125).</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> Synthesis of NS1 (<b>10</b>)</h3><div id="sec4_4_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> (3a<i>R</i>,4<i>R</i>,6<i>R</i>,6a<i>R</i>)-4-Methoxy-2,2-dimethyl-6-(prop-2-yn-1-yl)tetrahydrofuro[3,4-<i>d</i>][1,3]dioxole (<b>2</b>)</h4><div class="NLM_p last">The following is an adaption of a published procedure.<a onclick="showRef(event, 'ref64'); return false;" href="javascript:void(0);" class="ref ref64">(64)</a> To a stirred solution of 2,6-lutidine (9.85 mL, 84.6 mmol, 1.1 equiv) in CH<sub>2</sub>Cl<sub>2</sub> (130 mL) at −78 °C was added trifluoromethanesulfonic anhydride (13.7 mL, 81.5 mmol, 1.06 equiv) over 5 min (flask A, 1 L rbf), followed by the addition of a solution of alcohol <b>1</b> (15.7 g, 76.9 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (33 mL) over 10 min. The mixture was stirred at −78 °C for 2 h and warmed to 0 °C. Meanwhile, to a separate flask (flask B, 500 mL rbf) containing a solution of TMS-acetylene (32.0 mL, 231 mmol, 3.00 equiv) in dry THF (130 mL) at −78 °C was added <i>n</i>-butyllithium (2.56 M in hexanes, 93.7 mL, 240 mmol, 3.12 equiv) at a rate of 5 mL/min via syringe. The resulting mixture was stirred at −78 °C for 1.5 h and warmed to 0 °C. While the contents of flask B remained at 0 °C, flask A was removed from the ice/water bath, volatiles were removed in vacuo (bath temperature at 20 °C), and a brief subjection to high vacuum yielded a red-orange paste. The crude triflate in flask A was resuspended in dry THF (65 mL), and DMPU (35 mL) was added. The contents were stirred vigorously for 10 min until complete solubilization was noted and cooled to −78 °C. The contents of flask B were cooled to −78 °C and transferred to flask A via cannula. The resulting mixture was stirred at −78 °C for 1 h and at −12 °C for 3 h (NaCl/ice bath), before the slow addition of TBAF (<b>Caution, gas evolution!</b>, 1.0 M in THF, 250 mL, 250 mmol, 3.25 equiv) via cannula. The mixture was allowed to warm to RT and was stirred for another 30 min before volatiles were removed in vacuo. The reddish-brown residue was resuspended in MTBE (350 mL), and a satd aq NH<sub>4</sub>Cl solution (300 mL) was added. The layers were roughly separated, and the organic phase, largely emulsified, was filtered through a pad of CaCO<sub>3</sub>/Celite to give two clear layers in the filtrate. The layers were separated, and the combined aqueous phases were extracted with MTBE (2 × 200 mL) and EtOAc (200 mL). The combined organic extracts were washed with a 5% aq LiCl solution (800 mL) and then with brine (800 mL), dried over MgSO<sub>4</sub>, filtered, and concentrated. The crude residue was purified by dry column vacuum chromatography<a onclick="showRef(event, 'ref65'); return false;" href="javascript:void(0);" class="ref ref65">(65)</a> (cyclohexane/EtOAc, 0 → 30%) to yield pure <b>2</b> as well as semipure <b>2</b>, which was repurified by silica gel chromatography (hexanes/EtOAc, 0 <i>→</i> 30%). Compound <b>2</b> (11.1 g, 52.3 mmol, 68%) was obtained as a light-yellow oil. <i>R</i><sub><i>f</i></sub> = 0.60 (hexanes/EtOAc, 67:33). FTIR (neat), ν<sub>max</sub> (cm<sup>–1</sup>): 3286, 2989, 2938, 2835, 1438, 1209, 1090, 1053, 1041. <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 4.95 (s, 1H), 4.69 (d, <i>J</i> = 5.9 Hz, 1H), 4.60 (d, <i>J</i> = 5.9 Hz, 1H), 4.30 (dd, <i>J</i> = 9.3, 6.6 Hz, 1H), 3.32 (s, 3H), 2.52 (ddd, <i>J</i> = 16.7, 6.6, 2.7 Hz, 1H), 2.42 (ddd, <i>J</i> = 16.6, 9.4, 2.7 Hz, 1H), 2.04 (t, <i>J</i> = 2.7 Hz, 1H), 1.46 (s, 3H), 1.31 (s, 3H). <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): δ 112.3, 109.6, 85.2, 85.1, 83.3, 80.2, 70.4, 54.7, 26.4, 24.9, 24.6. HRMS (ESI+): [C<sub>11</sub>H<sub>16</sub>O<sub>4</sub>Na]<sup>+</sup> calcd 235.0941, measured 235.0938, Δ 1.0 ppm.</div></div><div id="sec4_4_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> (3a<i>R</i>,4<i>R</i>,6<i>R</i>,6a<i>R</i>)-4-Allyl-6-methoxy-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxole (<b>3</b>)</h4><div class="NLM_p last">Alkyne <b>2</b> (10.6 g, 49.9 mmol) was reduced according to protocols adapted from Cox et al.<a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(66)</a> Crude material was purified by silica gel chromatography (hexanes/EtOAc, 0 → 30%) to afford alkene <b>3</b> (9.97 g, 46.5 mmol, 93%) as a clear, viscous oil. The flash column chromatography removed ca. 98% of PHMS, yielding material that was suitable for use in subsequent reactions. An analytically pure sample was prepared by subjecting 2.0 g of the postcolumn material to short path distillation (Kugelrohr) at 130 °C and 200 mTorr. <i>R</i><sub><i>f</i></sub> = 0.50 (hexanes/EtOAc, 80:20). FTIR (neat), ν<sub>max</sub> (cm<sup>–1</sup>): 3078, 2987, 2938, 2833, 1643, 1381, 1372, 1209, 1193, 1105, 1090, 1059, 1031. <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 5.82 (dddd, <i>J</i> = 16.7, 10.3, 7.4, 6.2 Hz, 1H), 5.15–5.09 (m, 2H), 4.96 (s, 1H), 4.61 (d, <i>J</i> = 5.9 Hz, 1H), 4.57 (dd, <i>J</i> = 6.0, 1.0 Hz, 1H), 4.23 (td, <i>J</i> = 7.8, 1.0 Hz, 1H), 3.34 (s, 3H), 2.46–2.38 (m, 1H), 2.32–2.24 (m, 1H), 1.48 (s, 3H), 1.31 (s, 3H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 134.5, 117.7, 112.4, 109.6, 86.6, 85.7, 83.6, 55.0, 39.6, 26.6, 25.2. HRMS (ESI+): [C<sub>11</sub>H<sub>18</sub>O<sub>4</sub>Na]<sup>+</sup> calcd 237.1095, measured 237.1097, Δ 1.0 ppm.</div></div><div id="sec4_4_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> (<i>E</i>)-4-((3a<i>R</i>,4<i>R</i>,6<i>R</i>,6a<i>R</i>)-6-Methoxy-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)but-2-enal (<b>4</b>)</h4><div class="NLM_p last">To a solution of alkene <b>3</b> (2.0 g, 9.3 mmol) in MTBE (degassed prior to addition by sparging with nitrogen for 15 min, 28 mL) at RT, were added <i>E</i>-crotonaldehyde (3.9 mL, 47 mmol, 5.0 equiv) and nitro-Grela metathesis catalyst (63 mg, 93 μmol, 1 mol %) sequentially. The flask headspace was purged with nitrogen for 10 min and the mixture was stirred at 40 °C for 48 h, at which point TLC analysis indicated ca. 50% conversion. More nitro-Grela catalyst (63 mg, 93 μmol, 1 mol %) and <i>E</i>-crotonaldehyde (3.9 mL, 47 mmol, 5.0 equiv) were added, and the reaction mixture was stirred at 40 °C for another 24 h, at which point complete conversion was indicated by TLC analysis. Volatiles were removed in vacuo, and the crude material was purified by silica gel chromatography (hexanes/EtOAc, 2 → 50%) to afford enal <b>4</b> (1.60 g, 6.60 mmol, 71%) as a yellow oil. <i>R</i><sub><i>f</i></sub> = 0.41 (hexanes/EtOAc, 50:50). FTIR (neat), ν<sub>max</sub> (cm<sup>–1</sup>): 2990, 2936, 2832, 2735, 1688, 1641, 1453, 1382, 1377, 1298, 1269, 1263, 1242, 1210, 1194, 1161, 1140, 1104, 1089, 1058, 1026. <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 9.54 (d, <i>J</i> = 7.8 Hz, 1H), 6.84 (dt, <i>J</i> = 15.7, 6.8 Hz, 1H), 6.21 (ddt, <i>J</i> = 15.7, 7.8, 1.5 Hz, 1H), 4.98 (s, 1H), 4.64 (d, <i>J</i> = 5.9 Hz, 1H), 4.56 (dd, <i>J</i> = 6.0, 1.1 Hz, 1H), 4.35 (ddd, <i>J</i> = 8.8, 6.4, 1.0 Hz, 1H), 3.34 (s, 3H), 2.68 (dddd, <i>J</i> = 15.4, 8.5, 6.6, 1.6 Hz, 1H), 2.58 (dtd, <i>J</i> = 15.2, 6.7,1.5 Hz, 1H), 1.48 (s, 3H), 1.32 (s, 3H). <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): δ 193.6, 153.5, 134.8, 112.6, 109.8, 85.4, 85.3, 83.7, 54.7, 38.4, 26.5, 25.0. HRMS (ESI+): calcd for [C<sub>12</sub>H<sub>18</sub>O<sub>5</sub>Na]<sup>+</sup> 265.1046, measured 265.1062, Δ 6.0 ppm.</div></div><div id="sec4_4_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> (<i>R</i>)-3-(((3a<i>R</i>,4<i>R</i>,6<i>R</i>,6a<i>R</i>)-6-Methoxy-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)methyl)-5-(triisopropylsilyl)pent-4-ynal (<b>5</b>)</h4><div class="NLM_p last">Aldehyde <b>5</b> was prepared using protocols adapted from Nishimura et al.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> A 250 mL flask was charged with [Rh(OAc)(C<sub>2</sub>H<sub>4</sub>)<sub>2</sub>]<sub>2</sub> catalyst (126 mg, 0.289 mmol, 2.5 mol %), (<i>S</i>)-DTBM-SegPhos (818 mg, 0.693 mmol, 6.0 mol %), and a large stir bar. The flask headspace was evacuated and backfilled with nitrogen (3×), and methanol was added (22 mL). The resulting orange suspension was stirred vigorously at RT until a red-orange solution was obtained (30 min). At this time, a solution of enal <b>4</b> (2.80 g, 11.6 mmol) in MeOH (10 mL) and TIPS-acetylene (5.19 mL, 23.1 mmol, 2.0 equiv) were added sequentially to the reaction mixture. The resulting red mixture was stirred at 40 °C for 15 h, and volatiles were removed in vacuo. Crude material was purified by silica gel chromatography (hexanes/EtOAc, 0 → 50%) to afford aldehyde <b>5</b> (4.60 g, 10.8 mmol, 94%) as a single diastereomer, as a light-yellow oil. <i>R</i><sub><i>f</i></sub> = 0.42 (cyclohexane/EtOAc, 75:25). FTIR (neat), ν<sub>max</sub> (cm<sup>–1</sup>): 2941, 2865, 2722, 2167, 1728, 1463, 1381, 1371, 1271, 1241, 1210, 1193, 1160, 1093, 1058, 1018. <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 9.82 (t, <i>J</i> = 2.1 Hz, 1H), 4.95 (s, 1H), 4.61 (dd, <i>J</i> = 5.9, 1.2 Hz, 1H), 4.58 (d, <i>J</i> = 5.9 Hz, 1H), 4.49 (ddd, <i>J</i> = 10.6, 4.2, 1.2 Hz, 1H), 3.35 (s, 3H), 3.16 (dtd, <i>J</i> = 11.2, 6.9, 3.9 Hz, 1H), 2.60 (ddd, <i>J</i> = 16.5, 7.5, 2.2 Hz, 1H), 2.55 (ddd, <i>J</i> = 16.5, 6.4, 2.1 Hz, 1H), 1.74 (ddd, <i>J</i> = 13.3, 10.6, 4.0 Hz, 1H), 1.64 (ddd, <i>J</i> = 13.2, 11.1, 4.2 Hz, 1H), 1.47 (s, 3H), 1.31 (s, 3H), 1.11–0.97 (m, 21H). <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): δ 200.7, 112.6, 109.9, 108.6, 85.5, 85.3, 84.5, 84.2, 55.4, 49.0, 40.7, 26.7, 25.4, 25.1, 18.7, 11.3. HRMS (ESI+): calcd for [C<sub>23</sub>H<sub>40</sub>O<sub>5</sub>Si<sub>1</sub>Na]<sup>+</sup> 447.2537, measured 447.2527, Δ 2.3 ppm.</div></div><div id="sec4_4_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> Methyl (<i>S</i>,<i>Z</i>)-5-(((3a<i>R</i>,4<i>R</i>,6<i>R</i>,6a<i>R</i>)-6-Methoxy-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)methyl)-2-(2,2,2-trifluoroacetamido)-7-(triisopropylsilyl)hept-2-en-6-ynoate (<b>6</b>)</h4><div class="NLM_p last">A solution of methyl 2-(dimethoxyphosphoryl)-2-(2,2,2-trifluoroacetamido)acetate (<b>S2</b>, 3.11 g, 10.6 mmol, 1.5 equiv) in THF (21 mL) was added dropwise over 20 min to a stirred suspension of NaH (95%, 254 mg, 10.6 mmol, 1.5 equiv) in THF (10 mL) at 0 °C. The resulting cloudy mixture was stirred at 0 °C for 30 min, and a solution of aldehyde <b>5</b> (3.00 g, 7.06 mmol) in THF (7.1 mL) was added over 5 min. The resulting mixture was allowed to warm gradually to RT and was stirred for 2 h, at which point the reaction media was heated to 40 °C and stirred for another 6 h. The mixture was allowed to cool to RT, and volatiles were removed in vacuo. The residue was partitioned between MTBE (25 mL) and 1× PBS (15 mL). The mixture was stirred vigorously for 10 min, and the layers were separated. The aqueous phase was extracted with MTBE (3 × 25 mL). The combined organic extracts were washed with brine (80 mL), dried over MgSO<sub>4</sub>, filtered, and concentrated. Crude material was purified by silica gel chromatography (hexanes/EtOAc, 0 → 30%) to yield enamide <b>6</b> (3.74 g, 6.32 mmol, 89%) as a 6:1 mixture of <i>Z</i> and <i>E</i> isomers as a light-yellow oil. *Note: <i>Z</i> and <i>E</i> isomers are not separated for the following step, as they both react to form the desired product. <i>R</i><sub><i>f</i></sub> = 0.48 (hexanes/EtOAc, 67:33). FTIR (neat), ν<sub>max</sub> (cm<sup>–1</sup>): 3297, 2943, 2866, 1720, 1664, 1528, 1463, 1439, 1382, 1373, 1335, 1283, 1210, 1160, 1106, 1092, 1062, 1043, 1016. <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ (<i>Z</i>-isomer) 7.73 (s, 1H), 7.07 (t, <i>J</i> = 7.2 Hz, 1H), 4.94 (s, 1H), 4.61 (dd, <i>J</i> = 6.0, 1.1 Hz, 1H), 4.58 (d, <i>J</i> = 5.9 Hz, 1H), 4.44 (ddd, <i>J</i> = 10.4, 4.4, 1.1 Hz, 1H), 3.81 (s, 3H), 3.32 (s, 3H), 2.87 (ddd, <i>J</i> = 8.3, 5.5, 3.7 Hz, 1H), 2.40 (ddd, <i>J</i> = 15.8, 7.2, 5.1 Hz, 1H), 2.26 (ddd, <i>J</i> = 15.8, 8.7, 7.2 Hz, 1H), 1.71 (ddd, <i>J</i> = 13.3, 10.4, 4.2 Hz, 1H), 1.65 (ddd, <i>J</i> = 13.4, 10.8, 4.5 Hz, 1H), 1.46 (s, 3H), 1.30 (s, 3H), 1.06–1.02 (m, 21H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ (<i>Z</i>-isomer) 163.3, 155.1 (q, <sup>2</sup><i>J</i><sub>C–F</sub> = 38 Hz), 138.3, 133.3, 123.8, 115.6 (q, <sup>1</sup><i>J</i><sub>C–F</sub> = 288 Hz), 112.0, 109.4, 109.1, 85.2, 84.2, 83.7, 54.5, 52.2, 40.6, 34.4, 29.2, 26.2, 24.8, 18.3, 11.0. <sup>19</sup>F NMR (471 MHz, CDCl<sub>3</sub>, BTF IStd): δ (<i>Z</i>-isomer) −76.4. HRMS (ESI+): calcd for [C<sub>28</sub>H<sub>44</sub>F<sub>3</sub>N<sub>1</sub>O<sub>7</sub>Si<sub>1</sub>K]<sup>+</sup> 630.2471, measured 630.2462, Δ 1.4 ppm.</div></div><div id="sec4_4_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> Methyl (2<i>S</i>,5<i>S</i>)-5-(((3a<i>R</i>,4<i>R</i>,6<i>R</i>,6a<i>R</i>)-6-Methoxy-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)methyl)-2-(2,2,2-trifluoroacetamido)-7-(triisopropylsilyl)hept-6-ynoate (<b>7</b>)</h4><div class="NLM_p last">To a flask charged with olefin <b>6</b> (300 mg, 0.507 mmol), as a 6:1 mixture of <i>Z</i> and <i>E</i> isomers, was added MeOH (20 mL) at RT under a nitrogen atmosphere. The contents were stirred until full dissolution was noted. Separately, (<i>S</i>,<i>S</i>)-MeBPE-Rh (28 mg, 0.051 mmol, 10 mol %) was weighed out into a vial in a glovebox. The vial was removed from the glovebox, placed under a nitrogen atmosphere, and MeOH (10 mL) was added. The methanolic solution of (<i>S</i>,<i>S</i>)-MeBPE-Rh was transferred via syringe to the methanolic solution of olefin <b>6</b>. The flask headspace was purged with hydrogen (balloon), the outlet needle was removed, and the balloon was refilled with hydrogen. The contents were stirred vigorously at RT under a hydrogen atmosphere (balloon) for 4 h, and volatiles were removed in vacuo. Crude material was purified by silica gel chromatography (hexanes/EtOAc, 0 → 50%) to afford protected amino acid <b>7</b> (276 mg, 0.465 mmol, 92%) as a single diastereomer, as a clear viscous oil. <i>R</i><sub><i>f</i></sub> = 0.45 (hexanes/EtOAc, 67:33). FTIR (neat), ν<sub>max</sub> (cm<sup>–1</sup>): 3321, 2942, 2865, 2165, 1750, 1719, 1549, 1462, 1440, 1382, 1372, 1208, 1160, 1095, 1060, 1017. <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 6.84 (d, <i>J</i> = 7.8 Hz, 1H), 4.93 (s, 1H), 4.66 (td, <i>J</i> = 7.7, 4.6 Hz, 1H), 4.61 (dd, <i>J</i> = 5.9, 1.2 Hz, 1H), 4.57 (d, <i>J</i> = 5.9 Hz, 1H), 4.43 (ddd, <i>J</i> = 10.1, 4.8, 1.2 Hz, 1H), 3.79 (s, 3H), 3.30 (s, 3H), 2.70–2.62 (m, 1H), 2.14 (dddd, <i>J</i> = 14.0, 10.8, 5.8, 4.6 Hz, 1H), 2.08–1.99 (m, 1H), 1.68–1.57 (m, 2H), 1.54 (dddd, <i>J</i> = 13.1, 10.7, 5.9, 4.5 Hz, 1H), 1.46 (s, 3H), 1.36 (dddd, <i>J</i> = 13.1, 11.2, 9.7, 4.4 Hz, 1H), (s, 3H), 1.08–1.03 (m, 21H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 171.4, 156.9 (q, <sup>2</sup><i>J</i><sub>C–F</sub> = 38 Hz), 115.7 (q, <sup>1</sup><i>J</i><sub>C–F</sub> = 288 Hz), 112.5, 109.8, 109.5, 85.6, 85.5, 84.5, 83.7, 55.1, 53.0, 52.3, 41.1, 31.1, 30.1, 29.8, 26.7, 25.4, 18.7, 11.3. <sup>19</sup>F NMR (471 MHz, CDCl<sub>3</sub>, BTF IStd): δ −76.8. HRMS (ESI+): calcd for [C<sub>28</sub>H<sub>46</sub>F<sub>3</sub>N<sub>1</sub>O<sub>7</sub>Si<sub>1</sub>Na]<sup>+</sup> 616.2888, measured 616.2890, Δ 0.4 ppm.</div></div><div id="sec4_4_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> (2<i>R</i>,3<i>R</i>,4<i>R</i>,5<i>R</i>)-2-(6-Benzamido-9<i>H</i>-purin-9-yl)-5-((2<i>S</i>,5<i>S</i>)-2-ethynyl-6-methoxy-6-oxo-5-(2,2,2-trifluoroacetamido)hexyl)tetrahydrofuran-3,4-diyl Diacetate (<b>8</b>)</h4><div class="NLM_p last">To a suspension of <i>N</i><sup>6</sup>-benzoyladenine (742 mg, 3.10 mmol, 2.15 equiv) in propionitrile (dried over 4 Å molecular sieves, 10 mL) at RT was added <i>N</i>,<i>O</i>-bis(trimethylsilyl)acetamide (0.97 mL, 4.0 mmol, 2.75 equiv). The resulting mixture was stirred at 80 °C for 10 min, upon which complete solubilization of <i>N</i><sup>6</sup>-benzoyladenine was observed. To a separate flask charged with triacetate <b>S4</b> (dried by azeotropic distillation with benzene (4×), 735 mg, 1.44 mmol) and 2,6-di-<i>tert</i>-butyl-4-methylpyridinium triflate (51 mg, 0.14 mmol, 10 mol %) was added propionitrile (10 mL). The resulting mixture was stirred at RT for 5 min until a clear solution was obtained and the solution of silylated <i>N</i><sup>6</sup>-benzoyladenine was added. The reaction mixture was stirred at 95 °C for 3 h, cooled to RT, and volatiles were removed in vacuo. Crude material was purified by silica gel chromatography (toluene/MeCN, 0 → 50%) to afford nucleoside <b>8</b> (663 mg, 0.963 mmol, 67%) as a white amorphous solid. <i>R</i><sub><i>f</i></sub> = 0.48 (toluene/MeCN, 50:50). FTIR (thin-film), ν<sub>max</sub> (cm<sup>–1</sup>): 3273, 3086, 2928, 1748, 1720, 1611, 1582, 1511, 1487, 1456, 1374, 1329, 1243, 1217, 1182, 1160, 1099, 1074, 1047, 1029. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): δ 9.09 (s, 1H), 8.77 (s, 1H), 8.09 (s, 1H), 8.03–7.99 (m, 2H), 7.62–7.57 (m, 1H), 7.53–7.48 (m, 2H), 7.06 (d, <i>J</i> = 7.9 Hz, 1H), 6.12 (d, <i>J</i> = 5.4 Hz, 1H), 6.07 (t, <i>J</i> = 5.4 Hz, 1H), 5.55 (dd, <i>J</i> = 5.5, 4.5 Hz, 1H), 4.61 (td, <i>J</i> = 7.8, 5.0 Hz, 1H), 4.48 (ddd, <i>J</i> = 10.9, 4.5, 2.9 Hz, 1H), 3.77 (s, 3H), 2.59 (ddtd, <i>J</i> = 11.4, 9.1, 4.4, 2.5 Hz, 1H), 2.16 (d, <i>J</i> = 2.4 Hz, 1H), 2.15 (s, 3H), 2.11–2.00 (m, 2H), 2.06 (s, 3H), 1.95 (dddd, <i>J</i> = 13.9, 10.9, 7.7, 4.6 Hz, 1H), 1.86 (ddd, <i>J</i> = 13.8, 11.2, 2.9 Hz, 1H), 1.61–1.54 (m, 1H), 1.45 (dddd, <i>J</i> = 13.6, 10.9, 9.3, 4.6 Hz, 1H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 171.2, 169.8, 169.6, 165.2, 157.0 (q, <sup>2</sup><i>J</i><sub>C–F</sub> = 37.6 Hz), 152.6, 151.6, 149.9, 142.4, 133.5, 132.8, 128.8, 128.1, 123.9, 115.7 (q, <sup>1</sup><i>J</i><sub>C–F</sub> = 288 Hz), 87.0, 84.7, 80.7, 73.5, 72.9, 71.7, 53.0, 52.3, 38.2, 30.8, 29.6, 27.9, 20.6, 20.4. <sup>19</sup>F NMR (471 MHz, CDCl<sub>3</sub>, BTF IStd): δ −76.7. HRMS (ESI+): calcd for [C<sub>31</sub>H<sub>31</sub>F<sub>3</sub>N<sub>6</sub>O<sub>9</sub>Na]<sup>+</sup> 711.1988, measured 711.1997, Δ 1.2 ppm.</div></div><div id="sec4_4_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> (2<i>R</i>,3<i>R</i>,4<i>R</i>,5<i>R</i>)-2-(6-Benzamido-9<i>H</i>-purin-9-yl)-5-((2<i>S</i>,5<i>S</i>)-2-((3-carbamoylphenyl)ethynyl)-6-methoxy-6-oxo-5-(2,2,2-trifluoroacetamido)hexyl)tetrahydrofuran-3,4-diyl Diacetate (<b>9</b>)</h4><div class="NLM_p last">To a vial charged with alkyne <b>8</b> (79 mg, 0.11 mmol) were added 3-iodobenzamide (57 mg, 0.23 mmol, 2.0 equiv), CuI (4 mg, 0.02 mmol, 20 mol %), and Pd(PPh<sub>3</sub>)<sub>4</sub> (7 mg, 6 μmol, 5 mol %). The vial headspace was purged with nitrogen for 5 min and a 5:1:1 mixture of toluene, DMF, and <i>i</i>-Pr<sub>2</sub>NEt (degassed by sparging with nitrogen for 15 min, 3.5 mL) was added at RT. The resulting mixture was stirred at 70 °C for 3 h. The solution was allowed to cool to RT, and volatiles were removed in vacuo. Crude material was purified by silica gel chromatography (toluene/MeCN, 0 → 100%) to afford protected NS1 <b>9</b> (83 mg, 0.10 mmol, 90%) as a white amorphous solid. <i>R</i><sub><i>f</i></sub> = 0.17 (EtOAc/<i>i</i>-PrOH, 50:50). FTIR (neat), ν<sub>max</sub> (cm<sup>–1</sup>): 3255, 3064, 2929, 1744, 1717, 1666, 1610, 1578, 1511, 1486, 1455, 1374, 1329, 1240, 1212, 1180, 1158, 1096, 1073, 1046, 1029. <sup>1</sup>H NMR (600 MHz, CD<sub>2</sub>Cl<sub>2</sub>): δ 9.03 (br s, 1H), 8.75 (s, 1H), 8.14 (s, 1H), 8.00 (t, <i>J</i> = 1.3 Hz, 1H), 7.99 (d, <i>J</i> = 1.5 Hz, 1H), 7.87 (td, <i>J</i> = 1.8, 0.6 Hz, 1H), 7.77 (ddd, <i>J</i> = 7.8, 1.9, 1.1 Hz, 1H), 7.64 (ddt, <i>J</i> = 8.0, 7.0, 1.3 Hz, 1H), 7.57–7.54 (m, 2H), 7.53 (dd, <i>J</i> = 1.6, 1.0 Hz, 1H), 7.40 (td, <i>J</i> = 7.8, 0.6 Hz, 1H), 7.15 (d, <i>J</i> = 7.7 Hz, 1H), 6.53 (br s, 1H), 6.18 (d, <i>J</i> = 5.8 Hz, 1H), 6.15 (t, <i>J</i> = 5.5 Hz, 1H), 5.70 (br s, 1H), 5.65 (dd, <i>J</i> = 5.3, 4.0 Hz, 1H), 4.65 (td, <i>J</i> = 7.5, 5.5 Hz, 1H), 4.54 (dt, <i>J</i> = 10.1, 3.9 Hz, 1H), 3.77 (s, 3H), 2.83 (ddt, <i>J</i> = 11.0, 9.0, 4.4 Hz, 1H), 2.19 (ddd, <i>J</i> = 13.8, 10.1, 4.0 Hz, 1H), 2.15 (s, 3H), 2.11 (dt, <i>J</i> = 10.5, 5.1 Hz, 1H), 2.11–1.99 (m, 2H), 2.04 (s, 3H), 1.68 (ddt, <i>J</i> = 13.0, 10.7, 5.6 Hz, 1H), 1.57 (dddd, <i>J</i> = 13.5, 10.8, 9.3, 4.8 Hz, 1H). <sup>13</sup>C NMR (101 MHz, CD<sub>2</sub>Cl<sub>2</sub>): δ 171.6, 170.4, 170.1, 168.8, 165.4, 157.4 (q, <sup>2</sup><i>J</i><sub>C–F</sub> = 37 Hz), 152.8, 152.1, 150.3, 142.9, 134.8, 134.1, 134.0, 133.1, 131.1, 129.1, 128.9, 128.3, 127.4, 124.6, 123.9, 116.1 (q, <sup>1</sup><i>J</i><sub>C–F</sub> = 287 Hz), 91.9, 87.3, 83.1, 81.5, 74.1, 73.0, 53.3, 53.0, 38.8, 31.4, 29.8, 29.1, 20.8, 20.6. <sup>19</sup>F NMR (471 MHz, CDCl<sub>3</sub>, BTF IStd): δ −76.7. HRMS (ESI+): calcd for [C<sub>38</sub>H<sub>37</sub>F<sub>3</sub>N<sub>7</sub>O<sub>10</sub>]<sup>+</sup> 808.2549, measured 808.2538, Δ 1.3 ppm.</div></div><div id="sec4_4_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> (2<i>S</i>,5<i>S</i>)-2-Amino-5-(((2<i>R</i>,3<i>S</i>,4<i>R</i>,5<i>R</i>)-5-(6-amino-9<i>H</i>-purin-9-yl)-3,4-dihydroxytetrahydrofuran-2-yl)methyl)-7-(3-carbamoylphenyl)hept-6-ynoic Acid (NS1, <b>10</b>)</h4><div class="NLM_p last">To a solution of <b>9</b> (158 mg, 196 μmol) in THF (2.0 mL) at RT was added a 0.5 M aq LiOH solution (2.0 mL). The resulting mixture was stirred for 2.5 h, cooled to 0 °C, and a 10% aq AcOH solution was added dropwise until pH 7 was reached. Volatiles were removed in vacuo, and the residue was dissolved in a solution of ammonia in methanol (7 N, 8 mL) at RT. The resulting mixture was stirred for 18 h, and volatiles were removed in vacuo. Crude material was purified by preparative HPLC (gradient elution: 5% → 55% B over 45 min) to afford NS1 <b>10</b> (107 mg, 172 μmol, 88% over 2 steps) as the TFA salt. Single crystals suitable for X-ray diffraction studies were grown from a 4:1:1 volumetric mixture of 10 mM aq NS1-TFA:<i>n</i>-BuOH:<i>i</i>-PrOH via slow evaporation at 4 °C over 2 weeks. Full small-molecule X-ray crystallographic data for NS1 (<b>10</b>) are reported in the <a class="ref internalNav" href="#notes1" aria-label="Supporting Information">Supporting Information</a>. <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>CN (350 μL)/D<sub>2</sub>O (350 μL)/<i>d</i>-TFA (40 μL)): δ 8.37 (s, 1H), 8.34 (s, 1H), 7.78 (td, <i>J</i> = 1.8, 0.6 Hz, 1H), 7.72 (ddd, <i>J</i> = 7.8, 1.9, 1.2 Hz, 1H), 7.52 (dt, <i>J</i> = 7.8, 1.4 Hz, 1H), 7.40 (td, <i>J</i> = 7.8, 0.6 Hz, 1H), 6.01 (d, <i>J</i> = 5.1 Hz, 1H), 4.74 (t, <i>J</i> = 5.1 Hz, 1H), 4.38 (ddd, <i>J</i> = 10.3, 4.4, 3.2 Hz, 1H), 4.23 (t, <i>J</i> = 4.8 Hz, 1H), 4.02 (dd, <i>J</i> = 6.6, 5.8 Hz, 1H), 2.81 (ddt, <i>J</i> = 10.6, 9.1, 4.5 Hz, 1H), 2.17 (dddd, <i>J</i> = 14.4, 11.5, 6.7, 4.5 Hz, 1H), 2.10–1.97 (m, 2H), 1.95–1.89 (m, 1H), 1.81–1.72 (m, 1H), 1.61–1.52 (m, 1H). <sup>13</sup>C NMR (101 MHz, CD<sub>3</sub>CN (350 μL)/D<sub>2</sub>O (350 μL)/<i>d</i>-TFA (40 μL)): δ 172.1, 171.3, 151.0, 149.4, 145.2, 144.1, 135.8, 134.4, 131.6, 129.9, 128.1, 124.5, 120.1, 92.8, 90.0, 83.9, 83.4, 74.8, 74.5, 53.5, 38.9, 30.9, 29.6, 28.7. HRMS (ESI+): calcd for [C<sub>24</sub>H<sub>28</sub>N<sub>7</sub>O<sub>6</sub>]<sup>+</sup> 510.2096, measured 510.2082, Δ 2.6 ppm.</div></div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> Synthesis of Compounds <b>11</b>–<b>41</b></h3><div id="sec4_5_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> (<i>S</i>)-2-Amino-6-((2<i>R</i>,3<i>S</i>,4<i>R</i>,5<i>R</i>)-5-(6-amino-9<i>H</i>-purin-9-yl)-3,4-dihydroxytetrahydrofuran-2-yl)hexanoic Acid (<b>11</b>)</h4><div class="NLM_p last">To a solution of nucleoside <b>S11</b> (73 mg, 0.10 mmol) in a 4:2:1 mixture of MeOH, THF, and water (35 mL) at RT was added CsOH·<i>x</i>H<sub>2</sub>O (100 mg). The resulting mixture was stirred for 16 h, and a 10% AcOH aq solution was added dropwise until neutral pH was reached. Volatiles were removed in vacuo. Crude material was purified by preparative HPLC (5% → 95% B′ over 80 min) to afford the partially deprotected carbamate. To a solution of the carbamate intermediate in a 10:1 mixture of <i>i</i>-PrOAc and <i>t</i>-BuOH (25 mL) at RT was added Pd/C (10%, 65 mg, 61 μmol). The reaction mixture was purged with hydrogen (3×) and stirred at RT under a hydrogen atmosphere (balloon) for 2 h. The reaction rate appeared sluggish so Pd(OH)<sub>2</sub>/C (20%, 105 mg, 150 μmol) was added as a suspension in a 1:1 mixture of <i>i</i>-PrOAc and AcOH (10 mL), and the resulting mixture was stirred at 50 °C for 2 h. The reaction remained slow, so EtOH (20 mL) was added. The temperature was lowered to 40 °C, and the reaction media was stirred for 3 h, upon which complete cleavage of the Cbz protecting group was observed.<a onclick="showRef(event, 'ref70'); return false;" href="javascript:void(0);" class="ref ref70">(70)</a> The flask headspace was purged with nitrogen under vigorous stirring. The contents were filtered over a pad of Celite, and the volatiles were removed in vacuo. Crude material was purified by preparative HPLC (2.5% → 27.5% B′ over 80 min) to afford <b>11</b> (33 mg, 69 μmol, 66% over 2 steps) as the TFA salt as a fluffy white powder. <sup>1</sup>H NMR (600 MHz, D<sub>2</sub>O): δ 8.48 (s, 1H), 8.45 (s, 1H), 6.12 (d, <i>J</i> = 5.1 Hz, 1H), 4.82 (t, <i>J</i> = 5.2 Hz, 1H), 4.29 (t, <i>J</i> = 5.0 Hz, 1H), 4.21–4.16 (m, 1H), 4.00 (t, <i>J</i> = 6.2 Hz, 1H), 1.98 (app ddt, <i>J</i> = 14.2, 10.9, 5.5 Hz, 1H), 1.94–1.86 (m, 1H), 1.88–1.77 (m, 2H), 1.59–1.39 (m, 4H). <sup>13</sup>C NMR (101 MHz, D<sub>2</sub>O): δ 173.2, 150.3, 148.6, 144.9, 143.0, 119.2, 88.4, 85.1, 74.1, 73.4, 53.6, 32.5, 30.0, 24.7, 24.2. HRMS (ESI+): calcd for [C<sub>15</sub>H<sub>23</sub>N<sub>6</sub>O<sub>5</sub>]<sup>+</sup> 367.1724, measured 367.1728, Δ 1.1 ppm.</div></div><div id="sec4_5_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> (2<i>S</i>,5<i>S</i>)-2-Amino-5-(((2<i>R</i>,3<i>S</i>,4<i>R</i>,5<i>R</i>)-5-(6-amino-9<i>H</i>-purin-9-yl)-3,4-dihydroxytetrahydrofuran-2-yl)methyl)hept-6-ynoic Acid (<b>12</b>)</h4><div class="NLM_p last">To a solution of <b>8</b> (42 mg, 61 μmol) in THF (1.0 mL) at RT was added a 0.5 M LiOH aq solution (1.0 mL). The resulting mixture was stirred for 1 h, cooled to 0 °C, and a 10% AcOH aq solution was added dropwise until pH 7 was reached. Volatiles were removed in vacuo. The residue was dissolved in a solution of ammonia in methanol (7 N, 3 mL) at RT. The resulting mixture was stirred for 16 h, and the volatiles were removed in vacuo. Crude material was purified by preparative HPLC (2% → 42% B over 30 min followed by 42% → 95% B over 5 min) to afford <b>12</b> (22 mg, 44 μmol, 72% over 2 steps) as the TFA salt upon treatment with <i>d</i>-TFA as part of sample preparation for NMR analysis. <sup>1</sup>H NMR (500 MHz, CD<sub>3</sub>CN (350 μL)/D<sub>2</sub>O (350 μL)/<i>d</i>-TFA (40 μL)): δ 8.87 (s, 2H), 6.50 (d, <i>J</i> = 5.0 Hz, 1H), 5.19 (t, <i>J</i> = 5.1 Hz, 1H), 4.68 (t, <i>J</i> = 4.8 Hz, 1H), 4.49 (dd, <i>J</i> = 6.7, 5.8 Hz, 1H), 3.08 (dddd, <i>J</i> = 13.7, 7.2, 5.8, 4.3 Hz, 1H), 3.02 (d, <i>J</i> = 2.4 Hz, 1H), 2.59 (dddd, <i>J</i> = 14.2, 11.6, 6.8, 4.7 Hz, 1H), 2.53–2.45 (m, 1H), 2.41 (td, <i>J</i> = 10.0, 5.2 Hz, 1H), 2.33 (ddd, <i>J</i> = 14.0, 10.9, 3.4 Hz, 1H), 2.19 (tt, <i>J</i> = 12.3, 5.0 Hz, 1H), 1.99 (dddd, <i>J</i> = 13.4, 11.9, 9.1, 4.6 Hz, 1H). <sup>13</sup>C NMR (101 MHz, CD<sub>3</sub>CN (350 μL)/D<sub>2</sub>O (350 μL)/<i>d</i>-TFA (40 μL)): δ 172.0, 150.9, 149.3, 145.2, 144.0, 120.0, 89.8, 86.1, 83.7, 74.8, 74.4, 73.1, 53.4, 38.8, 30.7, 28.8, 28.4. HRMS (ESI+): calcd for [C<sub>17</sub>H<sub>23</sub>N<sub>6</sub>O<sub>5</sub>]<sup>+</sup> 391.1724, measured 391.1722, Δ 0.7 ppm.</div></div><div id="sec4_5_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> (2<i>S</i>,5<i>S</i>)-2-Amino-5-(((2<i>R</i>,3<i>S</i>,4<i>R</i>,5<i>R</i>)-5-(6-amino-9<i>H</i>-purin-9-yl)-3,4-dihydroxytetrahydrofuran-2-yl)methyl)-7-phenylhept-6-ynoic Acid (<b>13</b>)</h4><div class="NLM_p last">To a solution of <b>S12</b> (74 mg, 97 μmol) in THF (7.0 mL) at RT, was added a 0.5 M LiOH aq solution (1.7 mL). The resulting mixture was stirred for 3 h, cooled to 0 °C, and a 10% AcOH aq solution was added dropwise until pH 7 was reached. Volatiles were removed in vacuo. The residue was dissolved in a solution of ammonia in methanol (7 N, 10 mL) at RT. The resulting mixture was stirred for 18 h, and the volatiles were removed in vacuo. Crude material was purified by preparative HPLC (5% → 45% B over 45 min followed by 45% → 95% B over 5 min) to afford <b>13</b> (35 mg, 60 μmol, 62% over 2 steps) as the TFA salt upon treatment with <i>d</i>-TFA as part of the sample preparation for NMR analysis. <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>CN (350 μL)/D<sub>2</sub>O (350 μL)/<i>d</i>-TFA (40 μL)): δ 8.84 (s, 1H), 8.82 (s, 1H), 7.84–7.80 (m, 2H), 7.78–7.74 (m, 3H), 6.47 (d, <i>J</i> = 5.1 Hz, 1H), 5.19 (t, <i>J</i> = 5.1 Hz, 1H), 4.83 (ddd, <i>J</i> = 10.5, 4.5, 3.2 Hz, 1H), 4.68 (t, <i>J</i> = 4.8 Hz, 1H), 4.48 (dd, <i>J</i> = 6.7, 5.7 Hz, 1H), 3.27 (ddt, <i>J</i> = 10.7, 9.2, 4.5 Hz, 1H), 2.63 (dddd, <i>J</i> = 14.3, 11.5, 6.8, 4.5 Hz, 1H), 2.56–2.44 (m, 2H), 2.38 (ddd, <i>J</i> = 13.9, 10.8, 3.3 Hz, 1H), 2.23 (tt, <i>J</i> = 12.6, 5.0 Hz, 1H), 2.03 (dddd, <i>J</i> = 13.4, 11.9, 9.2, 4.5 Hz, 1H). <sup>13</sup>C NMR (126 MHz, CD<sub>3</sub>CN (350 μL)/D<sub>2</sub>O (350 μL)/<i>d</i>-TFA (40 μL)): δ 172.0, 151.0, 149.4, 145.2, 144.0, 132.4, 129.5, 129.2, 123.9, 120.1, 91.7, 89.9, 84.2, 83.9, 74.8, 74.5, 53.5, 39.0, 30.9, 29.5, 28.7. HRMS (ESI+): calcd for [C<sub>23</sub>H<sub>26</sub>N<sub>6</sub>O<sub>5</sub>Na]<sup>+</sup> 489.1857, measured 489.1841, Δ 3.2 ppm.</div></div><div id="sec4_5_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> (2<i>S</i>,5<i>R</i>)-2-Amino-5-(((2<i>R</i>,3<i>S</i>,4<i>R</i>,5<i>R</i>)-5-(6-amino-9<i>H</i>-purin-9-yl)-3,4-dihydroxytetrahydrofuran-2-yl)methyl)-7-(3-carbamoylphenyl)hept-6-ynoic Acid (<b>14</b>)</h4><div class="NLM_p last">To a solution of <b>S19</b> (50 mg, 62 μmol) in THF (1.2 mL) at RT was added a 0.5 M LiOH aq solution (1.2 mL). The resulting mixture was stirred for 1 h, cooled to 0 °C, and a 1 M HCl aq solution (0.22 mL) was added until pH 7 was reached. Volatiles were removed in vacuo. The residue was dissolved in a solution of ammonia in methanol (7 N, 5 mL) at RT. The resulting mixture was stirred for 18 h, and the volatiles were removed in vacuo. Crude material was purified by preparative HPLC (5% → 45% B over 30 min followed by 45% → 95% B over 5 min) to afford <b>14</b> (38 mg, 62 μmol, quant over 2 steps) as the TFA salt upon treatment with <i>d</i>-TFA (50 μL), as part of the sample preparation for NMR analysis. <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>CN (350 μL)/D<sub>2</sub>O (350 μL)/<i>d</i>-TFA (50 μL)): δ 8.37 (s, 1H), 8.31 (s, 1H), 7.70 (ddd, <i>J</i> = 7.7, 1.7, 1.3 Hz, 1H), 7.67 (ddd, <i>J</i> = 1.7, 1.5, 0.4 Hz, 1H), 7.41 (ddd, <i>J</i> = 7.7, 1.5, 1.3 Hz, 1H), 7.36 (app td, <i>J</i> = 7.7, 0.4 Hz, 1H), 6.00 (d, <i>J</i> = 5.2 Hz, 1H), 4.79 (app t, <i>J</i> = 5.2 Hz, 1H), 4.28 (ddd, <i>J</i> = 8.6, 5.1, 4.0 Hz, 1H), 4.24 (dd, <i>J</i> = 5.2, 4.0 Hz, 1H), 4.01 (app t, <i>J</i> = 6.4 Hz, 1H), 2.81 (app tt, <i>J</i> = 8.6, 5.3 Hz, 1H), 2.21 (dddd, <i>J</i> = 14.2, 11.5, 6.4, 5.4 Hz, 1H), 2.08 (app dt, <i>J</i> = 13.9, 8.6 Hz, 1H), 2.03–1.94 (m, 2H), 1.76–1.63 (m, 2H). <sup>13</sup>C NMR (126 MHz, CD<sub>3</sub>CN (350 μL)/D<sub>2</sub>O (350 μL)/<i>d</i>-TFA (50 μL)): δ 172.2, 171.6, 150.9, 149.4, 145.2, 144.1, 135.7, 134.2, 131.3, 129.9, 128.1, 124.5, 120.0, 93.8, 90.0, 84.6, 82.9, 74.7, 74.7, 53.6, 38.9, 30.7, 29.8, 28.6. HRMS (ESI+): calcd for [C<sub>24</sub>H<sub>28</sub>N<sub>7</sub>O<sub>6</sub>]<sup>+</sup> 510.2096, measured 510.2112, Δ 3.2 ppm.</div></div><div id="sec4_5_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> (2<i>S</i>,5<i>S</i>)-2-Amino-5-(((2<i>R</i>,3<i>S</i>,4<i>R</i>,5<i>R</i>)-5-(6-amino-9<i>H</i>-purin-9-yl)-3,4-dihydroxytetrahydrofuran-2-yl)methyl)-7-(3-carbamoylphenyl)heptanoic Acid (<b>15</b>)</h4><div class="NLM_p last">To a solution of <b>9</b> (20 mg, 25 μmol) in MeOH (1.3 mL) at RT was added Pd/C (10%, 8 mg, 7 μmol, 30 mol %). The vial headspace was evacuated and refilled with hydrogen (3×), and the resulting mixture was stirred vigorously at 50 °C under a hydrogen atmosphere (balloon) for 17 h and filtered through a nylon syringe filter (13 mm, 0.22 μm). The solids were washed with MeOH (5 × 1 mL), and the filtrates were concentrated. Crude alkane was used directly in the next step without further purification. To a solution of crude alkane in THF (0.5 mL) at RT was added a 0.5 M LiOH aq solution (0.46 mL). The resulting mixture was stirred for 1 h, cooled to 0 °C, and a 1 M HCl aq solution (0.11 mL) was added to reach pH 7. Volatiles were removed in vacuo. The residue was dissolved in a solution of ammonia in methanol (7 N, 4.5 mL) at RT. The resulting mixture was stirred for 18 h, and the volatiles were removed in vacuo. Crude material was purified by preparative HPLC (5% → 45% B over 30 min followed by 45% → 95% B over 5 min) to afford NS1-alkane (<b>15</b>) (16 mg, 25 μmol, quant over 3 steps) as the TFA salt upon treatment with <i>d</i>-TFA as part of the sample preparation for NMR analysis. <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>CN (350 μL)/D<sub>2</sub>O (350 μL)/<i>d</i>-TFA (40 μL)): δ 8.32 (s, 1H), 8.31 (s, 1H), 7.57–7.56 (m, 1H), 7.54 (app dt, <i>J</i> = 7.3, 1.7 Hz, 1H), 7.32 (app dt, <i>J</i> = 7.7, 1.7 Hz, 1H), 7.30 (ddd, <i>J</i> = 7.7, 7.3, 0.5 Hz, 1H), 5.93 (d, <i>J</i> = 4.7 Hz, 1H), 4.64 (app t, <i>J</i> = 4.7 Hz, 1H), 4.13–4.10 (m, 1H), 4.10 (dd, <i>J</i> = 5.3, 4.7 Hz, 1H), 3.94 (app t, <i>J</i> = 6.1 Hz, 1H), 2.65–2.53 (m, 2H), 1.93–1.87 (m, 1H), 1.86–1.78 (m, 1H), 1.76–1.70 (m, 2H), 1.68–1.60 (m, 1H), 1.60–1.53 (m, 2H), 1.52–1.45 (m, 1H), 1.45–1.38 (m, 1H). <sup>13</sup>C NMR (126 MHz, CD<sub>3</sub>CN (350 μL)/D<sub>2</sub>O (350 μL)/<i>d</i>-TFA (40 μL)): δ 172.5, 172.3, 151.0, 149.4, 145.3, 144.2, 144.0, 133.8, 133.4, 129.7, 128.4, 125.9, 120.0, 89.9, 83.4, 74.8, 74.7, 53.9, 37.4, 34.8, 34.3, 32.8, 29.0, 27.9. HRMS (ESI+): calcd for [C<sub>24</sub>H<sub>32</sub>N<sub>7</sub>O<sub>6</sub>]<sup>+</sup> 514.2414, measured 514.2437, Δ 4.5 ppm.</div></div><div id="sec4_5_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> (2<i>S</i>,5<i>R</i>)-2-Amino-5-(((2<i>R</i>,3<i>S</i>,4<i>R</i>,5<i>R</i>)-5-(6-amino-9<i>H</i>-purin-9-yl)-3,4-dihydroxytetrahydrofuran-2-yl)methyl)-7-(3-carbamoylphenyl)heptanoic Acid (<b>16</b>)</h4><div class="NLM_p last">To a solution of <b>14</b> (10 mg, 16 μmol) in a 9:1 mixture of EtOH and water (2.0 mL) at RT was added Pd/C (10%, 5 mg, 5 μmol, 30 mol %). The vial headspace was evacuated and refilled with nitrogen (3×) followed by hydrogen (3×), and the resulting mixture was stirred vigorously at 50 °C under a hydrogen atmosphere (balloon) for 2.5 h and filtered through a nylon syringe filter (13 mm, 0.22 μm). The solids were washed with EtOH (3 × 1 mL), and the filtrates were concentrated. Crude material was purified by preparative HPLC (5% → 45% B′ over 30 min followed by 45% → 95% B′ over 5 min) to afford <b>16</b> (9 mg, 14 μmol, 89%) as the TFA salt. <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>CN (350 μL)/D<sub>2</sub>O (350 μL)): δ 8.26 (s, 1H), 8.24 (s, 1H), 7.56–7.52 (m, 2H), 7.29–7.25 (m, 2H), 5.92 (d, <i>J</i> = 4.6 Hz, 1H), 4.64 (dd, <i>J</i> = 5.0, 4.6 Hz, 1H), 4.09 (app t, <i>J</i> = 5.0 Hz, 1H), 4.09–4.05 (m, 1H), 3.80 (app t, <i>J</i> = 6.1 Hz, 1H), 2.59–2.52 (m, 2H), 1.90–1.83 (m, 1H), 1.80–1.70 (m, 2H), 1.67 (ddd, <i>J</i> = 14.3, 7.6, 3.5 Hz, 1H), 1.64–1.51 (m, 3H), 1.48–1.37 (m, 2H). <sup>13</sup>C NMR (126 MHz, CD<sub>3</sub>CN (350 μL)/D<sub>2</sub>O (350 μL)): δ 173.1, 172.3, 152.4, 149.5, 147.5, 144.2, 143.0, 133.8, 133.2, 129.6, 128.3, 125.8, 119.9, 89.5, 83.6, 74.8, 74.6, 54.6, 37.7, 35.6, 34.3, 32.7, 28.1, 27.5. HRMS (ESI+): calcd for [C<sub>24</sub>H<sub>32</sub>N<sub>7</sub>O<sub>6</sub>]<sup>+</sup> 514.2409, measured 514.2423, Δ 2.9 ppm.</div></div><div id="sec4_5_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> 3-(4-((2<i>R</i>,3<i>S</i>,4<i>R</i>,5<i>R</i>)-5-(6-Amino-9<i>H</i>-purin-9-yl)-3,4-dihydroxytetrahydrofuran-2-yl)but-1-yn-1-yl)benzamide (<b>17</b>)</h4><div class="NLM_p last">To a vial charged with <b>S26</b> (58 mg, 97 μmol) was added a solution of ammonia in methanol (7 N, 8.0 mL) at RT. The resulting mixture was stirred for 17 h, and volatiles were removed in vacuo. Crude material was purified by preparative HPLC (10% → 45% B′ over 35 min followed by 45% → 95% B′ over 5 min) to afford <b>17</b> (35 mg, 86 μmol, 88%) as a white fluffy solid. <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>CN (400 μL)/D<sub>2</sub>O (400 μL)): δ 8.36 (s, 1H), 8.31 (s, 1H), 7.70–7.69 (m, 1H), 7.71–7.67 (m, 1H), 7.46–7.43 (m, 1H), 7.38–7.35 (m, 1H), 5.98 (d, <i>J</i> = 5.1 Hz, 1H), 4.70 (dd, <i>J</i> = 5.1, 4.7 Hz, 1H), 4.22–4.18 (m, 2H), 2.53 (app dt, <i>J</i> = 17.3, 6.9 Hz, 1H), 2.49 (app dt, <i>J</i> = 17.3, 7.2 Hz, 1H), 2.03–1.96 (m, 2H). <sup>13</sup>C NMR (126 MHz, CD<sub>3</sub>CN (400 μL)/D<sub>2</sub>O (400 μL)): δ 171.3, 150.9, 149.4, 145.3, 143.8, 135.6, 134.2, 131.2, 129.9, 127.9, 124.6, 119.9, 91.6, 89.6, 84.8, 80.9, 74.8, 74.1, 32.6, 16.3. HRMS (ESI+): calcd for [C<sub>20</sub>H<sub>20</sub>N<sub>6</sub>O<sub>4</sub>Na]<sup>+</sup> 431.1438, measured 431.1453, Δ 3.4 ppm.</div></div><div id="sec4_5_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> 3-(((1<i>R</i>,2<i>R</i>)-2-((2<i>R</i>,3<i>S</i>,4<i>R</i>,5<i>R</i>)-5-(6-Amino-9<i>H</i>-purin-9-yl)-3,4-dihydroxytetrahydrofuran-2-yl)cyclopropyl)ethynyl)benzamide (<b>18</b>)</h4><div class="NLM_p last">To a vial charged with <b>S33</b> (50 mg, 82 μmol) was added a solution of ammonia in methanol (7 N, 6.0 mL) at RT. The resulting mixture was stirred for 17 h, and volatiles were removed in vacuo. Crude material was purified by preparative HPLC (10% → 45% B’ over 30 min followed by 45% → 95% B′ over 5 min) to afford <b>18</b> (32 mg, 76 μmol, 93%) as a white fluffy solid. <sup>1</sup>H NMR (500 MHz, CD<sub>3</sub>CN (350 μL)/D<sub>2</sub>O (350 μL)): δ 8.39 (s, 1H), 8.32 (s, 1H), 7.71 (td, <i>J</i> = 1.8, 0.6 Hz, 1H), 7.67 (ddd, <i>J</i> = 7.8, 1.8, 1.2 Hz, 1H), 7.44 (ddd, <i>J</i> = 7.8, 1.8, 1.2 Hz, 1H), 7.35 (td, <i>J</i> = 7.8, 0.6 Hz, 1H), 5.95 (d, <i>J</i> = 4.7 Hz, 1H), 4.73 (dd, <i>J</i> = 5.2, 4.7 Hz, 1H), 4.36 (dd, <i>J</i> = 5.2, 4.8 Hz, 1H), 3.45 (dd, <i>J</i> = 9.1, 4.8 Hz, 1H), 1.62 (dddd, <i>J</i> = 9.1, 8.8, 5.9, 4.4 Hz, 1H), 1.53 (ddd, <i>J</i> = 8.6, 5.4, 4.4 Hz, 1H), 1.05 (ddd, <i>J</i> = 8.8, 5.4, 4.8 Hz, 1H), 1.02 (ddd, <i>J</i> = 8.6, 5.9, 4.8 Hz, 1H). <sup>13</sup>C NMR (126 MHz, CD<sub>3</sub>CN (350 μL)/D<sub>2</sub>O (350 μL)): δ 171.2, 151.1, 149.3, 145.4, 143.8, 135.6, 134.3, 131.3, 129.8, 127.8, 124.5, 120.0, 93.6, 89.7, 88.5, 76.7, 74.9, 74.4, 25.2, 12.9, 6.4. HRMS (ESI+): calcd for [C<sub>21</sub>H<sub>21</sub>N<sub>6</sub>O<sub>4</sub>]<sup>+</sup> 421.1619, measured 421.1627, Δ 1.9 ppm.</div></div><div id="sec4_5_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> (2<i>S</i>,5<i>S</i>)-2-Amino-7-(3-carbamoylphenyl)-5-(((2<i>R</i>,3<i>S</i>,4<i>R</i>,5<i>R</i>)-3,4-dihydroxy-5-methoxytetrahydrofuran-2-yl)methyl)hept-6-ynoic Acid (<b>19</b>)</h4><div class="NLM_p last">To a solution of <b>S35</b> (23 mg, 44 μmol) in THF (2.0 mL) at RT was added a 0.5 M LiOH aq solution (2.0 mL). The reaction mixture was stirred at RT for 3 h, and a 10% AcOH aq solution was added slowly until pH 5 was obtained. Volatiles were removed in vacuo, and the residue was purified by preparative HPLC (5% → 55% B′ over 60 min followed by 55% → 95% B′ over 5 min) to afford <b>19</b> (11 mg, 28 μmol, 63%) as the TFA salt. <sup>1</sup>H NMR (600 MHz, D<sub>2</sub>O): δ 7.84 (t, <i>J</i> = 1.8 Hz, 1H), 7.74 (ddd, <i>J</i> = 7.9, 1.9, 1.0 Hz, 1H), 7.65 (ddt, <i>J</i> = 7.7, 1.6, 0.8 Hz, 1H), 7.47 (tt, <i>J</i> = 7.9, 0.6 Hz, 1H), 4.90 (d, <i>J</i> = 1.4 Hz, 1H), 4.31 (ddd, <i>J</i> = 10.2, 6.3, 3.0 Hz, 1H), 4.13 (dd, <i>J</i> = 6.3, 4.7 Hz, 1H), 4.11–4.06 (m, 2H), 3.40 (s, 3H), 2.97–2.89 (m, 1H), 2.25 (dddd, <i>J</i> = 14.2, 11.5, 6.7, 4.6 Hz, 1H), 2.16 (ddt, <i>J</i> = 14.4, 11.2, 5.5 Hz, 1H), 1.93 (ddd, <i>J</i> = 13.8, 10.9, 3.1 Hz, 1H), 1.85–1.80 (m, 1H), 1.77 (ddd, <i>J</i> = 13.7, 10.6, 4.5 Hz, 1H), 1.70–1.59 (m, 1H). <sup>13</sup>C NMR (101 MHz, D<sub>2</sub>O): δ 172.6, 172.3, 135.3, 133.1, 130.5, 129.0, 127.2, 123.2, 108.2, 92.1, 82.7, 81.0, 74.6, 74.4, 55.2, 53.2, 39.5, 29.9, 29.1, 28.0. HRMS (ESI+): calcd for [C<sub>20</sub>H<sub>27</sub>N<sub>2</sub>O<sub>7</sub>]<sup>+</sup> 407.1813, measured 407.1809, Δ 1.0 ppm.</div></div><div id="sec4_5_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> (<i>S</i>)-5-(((2<i>R</i>,3<i>S</i>,4<i>R</i>,5<i>R</i>)-5-(6-Amino-9<i>H</i>-purin-9-yl)-3,4-dihydroxytetrahydrofuran-2-yl)methyl)-7-(3-carbamoylphenyl)hept-6-ynoic Acid (<b>20</b>)</h4><div class="NLM_p last">To a solution of <b>S41</b> (105 mg, 0.151 mmol) in a 2:1 mixture of MeOH and THF (24 mL) at 0 °C was added a 0.2 M LiOH aq solution (7.5 mL, 3.01 mmol, 20 equiv). The resulting mixture was stirred at RT for 24 h and organics were removed in vacuo. A 1 M HCl aq solution (1.7 mL) was added, and volatiles were removed in vacuo. Crude material was purified by preparative HPLC (30% → 95% B over 35 min followed by 95% B over 5 min) to afford <b>20</b> (45 mg, 91 μmol, 60%) as a fluffy white solid. <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>CN (400 μL)/D<sub>2</sub>O (400 μL)/<i>d</i>-TFA (100 μL)): δ 8.36 (s, 1H), 8.34 (s, 1H), 7.76 (dd, <i>J</i> = 1.8, 1.2 Hz, 1H), 7.70 (ddd, <i>J</i> = 7.8, 1.8, 1.2 Hz, 1H), 7.49 (app dt, <i>J</i> = 7.8, 1.2 Hz, 1H), 7.37 (app t, <i>J</i> = 7.8 Hz, 1H), 6.01 (d, <i>J</i> = 5.3 Hz, 1H), 4.81–4.77 (dd, <i>J</i> = 5.3, 5.0 Hz, 1H), 4.41 (ddd, <i>J</i> = 10.6, 4.0, 3.1 Hz, 1H), 4.25–4.21 (dd, <i>J</i> = 5.0, 4.0 Hz, 1H), 2.78–2.71 (m, 1H), 2.36–2.26 (m, 2H), 2.01 (ddd, <i>J</i> = 13.9, 10.6, 4.1 Hz, 1H), 1.87 (ddd, <i>J</i> = 13.9, 11.0, 3.1 Hz, 1H), 1.83–1.75 (m, 1H), 1.73–1.64 (m, 1H), 1.61–1.49 (m, 2H). <sup>13</sup>C NMR (126 MHz, CD<sub>3</sub>CN (400 μL)/D<sub>2</sub>O (400 μL)/<i>d</i>-TFA (100 μL)): δ 178.3, 171.9, 151.1, 149.5, 145.3, 144.4, 136.0, 134.2, 131.5, 130.0, 128.1, 124.9, 120.2, 94.0, 90.3, 84.5, 82.9, 74.9, 74.8, 39.2, 35.2, 34.5, 29.8, 23.5. HRMS (ESI+): calcd for [C<sub>24</sub>H<sub>27</sub>N<sub>6</sub>O<sub>6</sub>]<sup>+</sup> 495.1987, measured 495.1984, Δ 0.4 ppm.</div></div><div id="sec4_5_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> 3-((<i>S</i>)-6-Amino-3-(((2<i>R</i>,3<i>S</i>,4<i>R</i>,5<i>R</i>)-5-(6-amino-9<i>H</i>-purin-9-yl)-3,4-dihydroxytetrahydrofuran-2-yl)methyl)hex-1-yn-1-yl)benzamide (<b>21</b>)</h4><div class="NLM_p last">To a vial charged with <b>S47</b> (79 mg, 0.11 mmol) was added a solution of ammonia in methanol (7 N, 15 mL) at RT. The resulting mixture was stirred for 24 h, and volatiles were removed in vacuo. Crude material was purified by preparative HPLC (5% → 45% B over 30 min followed by 45% → 95% B over 5 min) to afford <b>21</b> (33 mg, 57 μmol, 54%) as the TFA salt upon treatment with <i>d</i>-TFA as part of the sample preparation for NMR analysis. <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>CN (350 μL)/D<sub>2</sub>O (350 μL)/<i>d</i>-TFA (40 μL)): δ 8.36 (s, 1H), 8.33 (s, 1H), 7.76 (ddd, <i>J</i> = 1.8, 1.5, 0.4 Hz, 1H), 7.71 (ddd, <i>J</i> = 7.8, 1.8, 1.2 Hz, 1H), 7.50 (ddd, <i>J</i> = 7.8, 1.5, 1.2 Hz, 1H), 7.39 (app td, <i>J</i> = 7.8, 0.4 Hz, 1H), 6.00 (d, <i>J</i> = 5.0 Hz, 1H), 4.71 (dd, <i>J</i> = 5.2, 5.0 Hz, 1H), 4.36 (ddd, <i>J</i> = 10.3, 4.5, 3.4 Hz, 1H), 4.21 (dd, <i>J</i> = 5.2, 4.5 Hz, 1H), 2.97–2.89 (m, 2H), 2.80–2.74 (m, 1H), 1.98 (ddd, <i>J</i> = 14.1, 10.3, 4.3 Hz, 1H), 1.90 (ddd, <i>J</i> = 14.1, 10.7, 3.4 Hz, 1H), 1.89–1.82 (m, 1H), 1.78–1.70 (m, 1H), 1.61 (app ddt, <i>J</i> = 13.2, 10.7, 5.4 Hz, 1H), 1.54 (dddd, <i>J</i> = 13.2, 10.6, 9.1, 5.1 Hz, 1H). <sup>13</sup>C NMR (126 MHz, CD<sub>3</sub>CN (350 μL)/D<sub>2</sub>O (350 μL)/<i>d</i>-TFA (40 μL)): δ 171.4, 150.9, 149.4, 145.2, 144.0, 135.7, 134.2, 131.4, 129.9, 128.0, 124.5, 120.0, 93.3, 89.8, 83.9, 83.1, 74.8, 74.4, 40.2, 38.9, 32.3, 29.5, 25.8. HRMS (ESI+): calcd for [C<sub>23</sub>H<sub>28</sub>N<sub>7</sub>O<sub>4</sub>]<sup>+</sup> 466.2197, measured 466.2219, Δ 4.7 ppm.</div></div><div id="sec4_5_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> 3-((<i>S</i>)-7-Amino-3-(((2<i>R</i>,3<i>S</i>,4<i>R</i>,5<i>R</i>)-5-(6-amino-9<i>H</i>-purin-9-yl)-3,4-dihydroxytetrahydrofuran-2-yl)methyl)-7-oxohept-1-yn-1-yl)benzamide (<b>22</b>)</h4><div class="NLM_p last">To a vial charged with <b>S41</b> (105 mg, 0.151 mmol) was added a solution of ammonia in methanol (7 N, 18 mL) at RT. The resulting mixture was stirred at 40 °C for 5 days, and volatiles were removed in vacuo. Crude material was purified by preparative HPLC (5% → 45% B over 30 min followed by 45% → 95% B over 5 min) to afford <b>22</b> (57 mg, 0.12 mmol, 77%) as a white fluffy solid. <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>CN (400 μL)/D<sub>2</sub>O (400 μL)/<i>d</i>-TFA (50 μL)): δ 8.35 (s, 1H), 8.32 (s, 1H), 7.75 (app t, <i>J</i> = 1.5 Hz, 1H), 7.69 (app dt, <i>J</i> = 7.8, 1.5 Hz, 1H), 7.48 (app dt, <i>J</i> = 7.8, 1.5 Hz, 1H), 7.36 (app t, <i>J</i> = 7.8 Hz, 1H), 5.99 (d, <i>J</i> = 5.2 Hz, 1H), 4.75 (app t, <i>J</i> = 5.2 Hz, 1H), 4.38 (ddd, <i>J</i> = 10.6, 4.1, 3.1 Hz, 1H), 4.21 (dd, <i>J</i> = 5.2, 4.1 Hz, 1H), 2.73 (dddd, <i>J</i> = 10.9, 9.2, 5.4, 4.0 Hz, 1H), 2.23 (ddd, <i>J</i> = 14.6, 8.0, 7.2 Hz, 1H), 2.20 (ddd, <i>J</i> = 14.6, 7.8, 7.1 Hz, 1H), 1.99 (ddd, <i>J</i> = 13.8, 10.6, 4.1 Hz, 1H), 1.85 (ddd, <i>J</i> = 13.8, 11.0, 3.1 Hz, 1H), 1.82–1.73 (m, 1H), 1.71–1.62 (m, 1H), 1.59–1.47 (m, 2H). <sup>13</sup>C NMR (126 MHz, CD<sub>3</sub>CN (400 μL)/D<sub>2</sub>O (400 μL)/<i>d</i>-TFA (50 μL)): δ 179.5, 171.4, 150.9, 149.4, 145.2, 144.2, 135.8, 134.2, 131.4, 129.9, 128.0, 124.7, 120.1, 93.9, 90.1, 84.3, 82.8, 74.8, 74.6, 39.1, 35.6, 35.1, 29.7, 24.2. HRMS (ESI+): calcd for [C<sub>24</sub>H<sub>27</sub>N<sub>7</sub>O<sub>5</sub>Na]<sup>+</sup> 516.1966, measured 516.1986, Δ 3.9 ppm.</div></div><div id="sec4_5_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43"> Methyl (2<i>S</i>,5<i>S</i>)-2-Amino-5-(((2<i>R</i>,3<i>S</i>,4<i>R</i>,5<i>R</i>)-5-(6-amino-9<i>H</i>-purin-9-yl)-3,4-dihydroxytetrahydrofuran-2-yl)methyl)-7-(3-carbamoylphenyl)hept-6-ynoate (<b>23</b>)</h4><div class="NLM_p last">To a solution of <b>S51</b> (32 mg, 34 μmol) in a 1:1 mixture of MeCN and CH<sub>2</sub>Cl<sub>2</sub> (2.0 mL) at RT was added piperidine (0.50 mL). The resulting mixture was stirred for 20 min, and volatiles were removed in vacuo. The residue was dissolved in a solution of ammonia in methanol (7 N, 4.5 mL) at RT. The resulting mixture was stirred for 18 h, and the volatiles were removed in vacuo. Crude material was purified by preparative HPLC (5% → 45% B′ over 35 min followed by 45% → 95% B′ over 5 min) to afford <b>23</b> (12 mg, 19 μmol, 27% over 2 steps) as the TFA salt. <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>CN (350 μL)/D<sub>2</sub>O (350 μL)): δ 8.33 (s, 1H), 8.30 (s, 1H), 7.77 (ddd, <i>J</i> = 1.9, 1.5, 0.5 Hz, 1H), 7.72 (ddd, <i>J</i> = 7.8, 1.9, 1.2 Hz, 1H), 7.51 (ddd, <i>J</i> = 7.8, 1.5, 1.2 Hz, 1H), 7.40 (app td, <i>J</i> = 7.8, 0.5 Hz, 1H), 5.98 (d, <i>J</i> = 5.0 Hz, 1H), 4.68 (dd, <i>J</i> = 5.2, 5.0 Hz, 1H), 4.34 (ddd, <i>J</i> = 10.0, 4.7, 3.4 Hz, 1H), 4.19 (dd, <i>J</i> = 5.2, 4.7 Hz, 1H), 4.04 (dd, <i>J</i> = 6.6, 6.1 Hz, 1H), 3.73 (s, 3H), 2.82–2.75 (m, 1H), 2.13 (dddd, <i>J</i> = 14.3, 11.6, 6.6, 4.6 Hz, 1H), 2.06–1.95 (m, 2H), 1.91 (ddd, <i>J</i> = 13.9, 10.5, 3.4 Hz, 1H), 1.71 (app ddt, <i>J</i> = 12.8, 11.6, 5.1 Hz, 1H), 1.52 (dddd, <i>J</i> = 12.8, 11.7, 9.7, 4.6 Hz, 1H). <sup>13</sup>C NMR (126 MHz, CD<sub>3</sub>CN (350 μL)/D<sub>2</sub>O (350 μL)): δ 171.2, 170.9, 151.5, 149.4, 146.2, 143.6, 135.7, 134.3, 131.4, 129.9, 128.1, 124.3, 119.9, 92.7, 89.7, 83.6, 83.4, 74.7, 74.3, 54.3, 53.5, 38.7, 30.7, 29.5, 28.7. HRMS (ESI+): calcd for [C<sub>25</sub>H<sub>30</sub>N<sub>7</sub>O<sub>6</sub>]<sup>+</sup> 524.2252, measured 524.2271, Δ 3.6 ppm.</div></div><div id="sec4_5_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i44" class="anchor-spacer"></div><h4 class="article-section__title" id="_i44"> 3-((3<i>S</i>,6<i>S</i>)-6,7-Diamino-3-(((2<i>R</i>,3<i>S</i>,4<i>R</i>,5<i>R</i>)-5-(6-amino-9<i>H</i>-purin-9-yl)-3,4-dihydroxytetrahydrofuran-2-yl)methyl)-7-oxohept-1-yn-1-yl)benzamide (<b>24</b>)</h4><div class="NLM_p last">To a vial charged with <b>9</b> (44 mg, 54 μmol) was added a solution of ammonia in methanol (7 N, 15 mL) at RT. The resulting mixture was stirred at 40 °C for 36 h, and the volatiles were removed in vacuo. Crude material was purified by preparative HPLC (5% → 45% B over 30 min followed by 45% → 95% B over 5 min) to afford <b>24</b> (31 mg, 50 μmol, 91%) as the TFA salt upon treatment with <i>d</i>-TFA as part of the sample preparation for NMR analysis. <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>CN (350 μL)/D<sub>2</sub>O (350 μL)/<i>d</i>-TFA (50 μL)): δ 8.36 (s, 1H), 8.33 (s, 1H), 7.78–7.76 (m, 1H), 7.71 (ddd, <i>J</i> = 7.8, 1.9, 1.2 Hz, 1H), 7.51 (ddd, <i>J</i> = 7.8, 1.5, 1.2 Hz, 1H), 7.39 (app td, <i>J</i> = 7.8, 0.6 Hz, 1H), 6.00 (d, <i>J</i> = 5.1 Hz, 1H), 4.73 (app t, <i>J</i> = 5.1 Hz, 1H), 4.37 (ddd, <i>J</i> = 10.3, 4.5, 3.4 Hz, 1H), 4.22 (dd, <i>J</i> = 5.1, 4.5 Hz, 1H), 3.96 (app t, <i>J</i> = 6.4 Hz, 1H), 2.79 (dddd, <i>J</i> = 10.6, 9.2, 4.5, 4.3 Hz, 1H), 2.13 (dddd, <i>J</i> = 14.2, 12.4, 6.4, 4.4 Hz, 1H), 2.00 (ddd, <i>J</i> = 14.3, 10.3, 4.3 Hz, 1H), 1.98–1.89 (m, 2H), 1.67 (dddd, <i>J</i> = 13.1, 12.4, 4.8, 4.5 Hz, 1H), 1.56 (dddd, <i>J</i> = 13.1, 12.7, 9.2, 4.4 Hz, 1H). <sup>13</sup>C NMR (126 MHz, CD<sub>3</sub>CN (350 μL)/D<sub>2</sub>O (350 μL)/<i>d</i>-TFA (50 μL)): δ 172.4, 171.6, 151.0, 149.4, 145.2, 144.1, 135.9, 134.2, 131.6, 130.0, 128.1, 124.5, 120.1, 92.9, 90.0, 84.0, 83.4, 74.9, 74.5, 53.8, 38.8, 30.7, 29.8, 29.7. HRMS (ESI+): calcd for [C<sub>24</sub>H<sub>28</sub>N<sub>8</sub>O<sub>5</sub>Na]<sup>+</sup> 531.2075, measured 531.2089, Δ 2.7 ppm.</div></div><div id="sec4_5_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i45" class="anchor-spacer"></div><h4 class="article-section__title" id="_i45"> 3-((<i>S</i>)-3-(((2<i>R</i>,3<i>S</i>,4<i>R</i>,5<i>R</i>)-5-(6-Amino-9<i>H</i>-purin-9-yl)-3,4-dihydroxytetrahydrofuran-2-yl)methyl)-5-ureidopent-1-yn-1-yl)benzamide (<b>25</b>)</h4><div class="NLM_p last">To a vial charged with <b>S58</b> (31 mg, 45 μmol) was added a solution of ammonia in methanol (7 N, 5.0 mL) at RT. The resulting mixture was stirred for 17 h, and the volatiles were removed in vacuo. Crude material was purified by preparative HPLC (5% → 45% B over 35 min followed by 45% → 95% B over 5 min) to afford <b>25</b> (18 mg, 36 μmol, 80%) as a fluffy white solid. <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>CN (350 μL)/D<sub>2</sub>O (350 μL)/<i>d</i>-TFA (40 μL)): δ 8.54 (s, 1H), 8.51 (s, 1H), 7.96 (t, <i>J</i> = 1.7 Hz, 1H), 7.90 (ddd, <i>J</i> = 7.8, 1.9, 1.1 Hz, 1H), 7.70 (dt, <i>J</i> = 7.7, 1.3 Hz, 1H), 7.58 (dd, <i>J</i> = 7.8, 0.6 Hz, 1H), 6.18 (d, <i>J</i> = 5.2 Hz, 1H), 4.93 (t, <i>J</i> = 5.2 Hz, 1H), 4.56 (dt, <i>J</i> = 10.6, 3.5 Hz, 1H), 4.40 (t, <i>J</i> = 4.7 Hz, 1H), 3.50 (ddd, <i>J</i> = 13.3, 7.7, 5.4 Hz, 1H), 3.43 (dt, <i>J</i> = 13.7, 7.5 Hz, 1H), 2.98 (tt, <i>J</i> = 9.7, 4.5 Hz, 1H), 2.19 (ddd, <i>J</i> = 14.4, 10.5, 4.2 Hz, 1H), 2.08 (ddd, <i>J</i> = 13.9, 10.9, 3.2 Hz, 1H), 2.00–1.91 (m, 1H), 1.86 (dddd, <i>J</i> = 13.2, 9.7, 7.4, 5.4 Hz, 1H). <sup>13</sup>C NMR (126 MHz, CD<sub>3</sub>CN (350 μL)/D<sub>2</sub>O (350 μL)/<i>d</i>-TFA (40 μL)): δ 171.6, 161.7, 151.0, 149.5, 145.2, 144.3, 135.9, 134.3, 131.5, 130.0, 128.2, 124.6, 120.1, 93.0, 90.1, 84.1, 83.2, 74.8, 74.6, 39.5, 39.0, 35.3, 27.6. HRMS (ESI+): calcd for [C<sub>23</sub>H<sub>26</sub>N<sub>8</sub>O<sub>5</sub>Na]<sup>+</sup> 517.1918, measured 517.1912, Δ 1.2 ppm.</div></div><div id="sec4_5_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i46" class="anchor-spacer"></div><h4 class="article-section__title" id="_i46"> (2<i>S</i>,6<i>S</i>)-2-Amino-6-(((2<i>R</i>,3<i>S</i>,4<i>R</i>,5<i>R</i>)-5-(6-amino-9<i>H</i>-purin-9-yl)-3,4-dihydroxytetrahydrofuran-2-yl)methyl)-8-(3-carbamoylphenyl)oct-7-ynoic Acid (<b>26</b>)</h4><div class="NLM_p last">To a solution of <b>S66</b> (69 mg, 84 μmol) in THF (1.5 mL) at RT was added a 0.5 M LiOH aq solution (1.5 mL). The resulting mixture was stirred for 1 h, cooled to 0 °C, and a 1 M HCl aq solution (0.20 mL) was added. Volatiles were removed in vacuo. The residue was dissolved in a solution of ammonia in methanol (7 N, 4.5 mL) at RT. The resulting mixture was stirred for 18 h, and volatiles were removed in vacuo. Crude material was purified by preparative HPLC (5% → 25% B′ over 30 min followed by 25% → 95% B′ over 5 min) to afford <b>26</b> (9 mg, 14 μmol, 17% over 2 steps) as the TFA salt. <sup>1</sup>H NMR (600 MHz, D<sub>2</sub>O (700 μL)): δ 8.42 (s, 1H), 8.30 (s, 1H), 7.70–7.65 (m, 1H), 7.51–7.48 (m, 1H), 7.37–7.33 (m, 2H), 6.08 (d, <i>J</i> = 5.0 Hz, 1H), 4.95 (dd, <i>J</i> = 5.1, 5.0 Hz, 1H), 4.47 (ddd, <i>J</i> = 9.2, 4.6, 3.2 Hz, 1H), 4.40 (dd, <i>J</i> = 5.1, 4.6 Hz, 1H), 3.95 (app t, <i>J</i> = 6.1 Hz, 1H), 2.89–2.84 (m, 1H), 2.08 (ddd, <i>J</i> = 14.4, 9.2, 3.7 Hz, 1H), 2.00 (ddd, <i>J</i> = 14.4, 9.8, 3.2 Hz, 1H), 1.96–1.91 (m, 2H), 1.72–1.60 (m, 3H), 1.56–1.49 (m, 1H). <sup>13</sup>C NMR (126 MHz, CD<sub>3</sub>CN (350 μL)/D<sub>2</sub>O (350 μL)): δ 172.5, 171.3, 151.2, 149.4, 145.7, 143.8, 135.7, 134.2, 131.4, 129.9, 127.9, 124.5, 120.0, 93.7, 89.7, 83.9, 82.9, 74.7, 74.4, 53.7, 38.9, 34.9, 30.5, 29.5, 23.2. HRMS (ESI+): calcd for [C<sub>25</sub>H<sub>30</sub>N<sub>7</sub>O<sub>6</sub>]<sup>+</sup> 524.2252, measured 524.2277, Δ 4.7 ppm.</div></div><div id="sec4_5_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> (2<i>S</i>,5<i>S</i>)-2-Amino-5-(((2<i>R</i>,3<i>S</i>,4<i>R</i>,5<i>R</i>)-5-(6-amino-9<i>H</i>-purin-9-yl)-3,4-dihydroxytetrahydrofuran-2-yl)methyl)-7-(4-carbamoylphenyl)hept-6-ynoic Acid (<b>27</b>)</h4><div class="NLM_p last">To a solution of <b>S67</b> (18 mg, 22 μmol) in THF (0.41 mL) at RT was added a 0.5 M LiOH aq solution (0.41 mL). The resulting mixture was stirred for 1 h, cooled to 0 °C, and a 1 M HCl aq solution (0.10 mL) was added. Volatiles were removed in vacuo. The residue was dissolved in a solution of ammonia in methanol (7 N, 4.5 mL) at RT. The resulting mixture was stirred for 18 h, and volatiles were removed in vacuo. Crude material was purified by preparative HPLC (5% → 45% B′ over 30 min followed by 45% → 95% B′ over 5 min) to afford <b>27</b> (7 mg, 11 μmol, 50% over 2 steps) as the TFA salt. <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>CN (350 μL)/D<sub>2</sub>O (350 μL)): δ 8.32 (s, 1H), 8.30 (s, 1H), 7.73–7.70 (m, 2H), 7.43–7.39 (m, 2H), 5.97 (d, <i>J</i> = 5.0 Hz, 1H), 4.71 (dd, <i>J</i> = 5.3, 5.0 Hz, 1H), 4.33 (ddd, <i>J</i> = 10.3, 4.5, 3.3 Hz, 1H), 4.19 (dd, <i>J</i> = 5.3, 4.5 Hz, 1H), 3.90 (dd, <i>J</i> = 6.8, 5.7 Hz, 1H), 2.83–2.76 (m, 1H), 2.10 (dddd, <i>J</i> = 14.2, 11.5, 6.8, 4.6 Hz, 1H), 2.03–1.96 (m, 2H), 1.94–1.88 (m, 1H), 1.73 (dddd, <i>J</i> = 12.8, 12.0, 5.4, 4.6 Hz, 1H), 1.55 (m, 1H). <sup>13</sup>C NMR (126 MHz, CD<sub>3</sub>CN (350 μL)/D<sub>2</sub>O (350 μL)): δ 172.6, 171.4, 151.7, 149.4, 146.5, 143.6, 133.2, 132.5, 128.6, 127.7, 120.0, 94.7, 89.7, 83.7, 83.4, 74.7, 74.4, 53.9, 38.6, 30.9, 29.6, 28.8. HRMS (ESI+): calcd for [C<sub>24</sub>H<sub>28</sub>N<sub>7</sub>O<sub>6</sub>]<sup>+</sup> 510.2096, measured 510.2097, Δ 0.2 ppm.</div></div><div id="sec4_5_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i48" class="anchor-spacer"></div><h4 class="article-section__title" id="_i48"> (2<i>S</i>,5<i>S</i>)-2-Amino-5-(((2<i>R</i>,3<i>S</i>,4<i>R</i>,5<i>R</i>)-5-(6-amino-9<i>H</i>-purin-9-yl)-3,4-dihydroxytetrahydrofuran-2-yl)methyl)-7-(2-carbamoylphenyl)hept-6-ynoic Acid (<b>28</b>)</h4><div class="NLM_p last">To a solution of <b>S68</b> (18 mg, 22 μmol) in THF (0.41 mL) at RT was added a 0.5 M LiOH aq solution (0.41 mL). The resulting mixture was stirred for 1 h, cooled to 0 °C, and a 1 M HCl aq solution (0.10 mL) was added. Volatiles were removed in vacuo. The residue was dissolved in a solution of ammonia in methanol (7 N, 4.5 mL) at RT. The resulting mixture was stirred for 18 h, and the volatiles were removed in vacuo. Crude material was purified by preparative HPLC (5% → 45% B′ over 30 min followed by 45% → 95% B′ over 5 min) to afford <b>28</b> (8 mg, 13 μmol, 58% over 2 steps) as the TFA salt. <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>CN (350 μL)/D<sub>2</sub>O (350 μL)): δ 8.32 (s, 1H), 8.29 (s, 1H), 7.56–7.52 (m, 1H), 7.44–7.41 (m, 1H), 7.40 (app td, <i>J</i> = 7.6, 1.5 Hz, 1H), 7.37 (ddd, <i>J</i> = 7.6, 7.1, 2.0 Hz, 1H), 5.97 (d, <i>J</i> = 5.1 Hz, 1H), 4.71 (app t, <i>J</i> = 5.1 Hz, 1H), 4.34 (ddd, <i>J</i> = 10.1, 4.5, 3.4 Hz, 1H), 4.20–4.18 (m, 1H), 3.88 (dd, <i>J</i> = 6.7, 5.9 Hz, 1H), 2.84–2.76 (m, 1H), 2.09 (dddd, <i>J</i> = 14.2, 11.7, 6.7, 4.6 Hz, 1H), 2.01–1.95 (m, 2H), 1.95–1.89 (m, 1H), 1.72 (app ddt, <i>J</i> = 13.4, 11.7, 4.9 Hz, 1H), 1.60–1.52 (m, 1H). <sup>13</sup>C NMR (126 MHz, CD<sub>3</sub>CN (350 μL)/D<sub>2</sub>O (350 μL)): δ 172.8, 151.7, 149.4, 146.5, 143.6, 137.3, 134.1, 131.7, 129.4, 128.8, 121.3, 120.0, 97.1, 89.7, 83.7, 81.8, 74.6, 74.4, 54.1, 38.5, 30.9, 29.7, 28.9. HRMS (ESI+): calcd for [C<sub>24</sub>H<sub>28</sub>N<sub>7</sub>O<sub>6</sub>]<sup>+</sup> 510.2096, measured 510.2095, Δ 0.2 ppm.</div></div><div id="sec4_5_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i49" class="anchor-spacer"></div><h4 class="article-section__title" id="_i49"> (2<i>S</i>,5<i>S</i>)-2-Amino-5-(((2<i>R</i>,3<i>S</i>,4<i>R</i>,5<i>R</i>)-5-(6-amino-9<i>H</i>-purin-9-yl)-3,4-dihydroxytetrahydrofuran-2-yl)methyl)-7-(3-sulfamoylphenyl)hept-6-ynoic Acid (<b>29</b>)</h4><div class="NLM_p last">To a solution of <b>S69</b> (41 mg, 49 μmol) in THF (1.3 mL) at RT was added a 0.5 M LiOH aq solution (1.1 mL). The resulting mixture was stirred for 1 h, cooled to 0 °C, and a 1 M HCl aq solution (0.20 mL) was added. Volatiles were removed in vacuo. The residue was dissolved in a solution of ammonia in methanol (7 N, 5 mL) at RT. The resulting mixture was stirred for 18 h, and volatiles were removed in vacuo. Crude material was purified by preparative HPLC (5% → 45% B over 30 min followed by 45% → 95% B over 5 min) to afford <b>29</b> (26 mg, 39 μmol, 81% over 2 steps) as the TFA salt upon treatment with <i>d</i>-TFA as part of the sample preparation for NMR analysis. <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>CN (350 μL)/D<sub>2</sub>O (350 μL)/<i>d</i>-TFA (40 μL)): δ 8.36 (s, 1H), 8.34 (s, 1H), 7.82 (dd, <i>J</i> = 1.9, 1.5 Hz, 1H), 7.77 (ddd, <i>J</i> = 7.8, 1.9, 1.2 Hz, 1H), 7.57 (ddd, <i>J</i> = 7.8, 1.5, 1.2 Hz, 1H), 7.48 (app t, <i>J</i> = 7.8 Hz, 1H), 6.00 (d, <i>J</i> = 5.1 Hz, 1H), 4.73 (dd, <i>J</i> = 5.2, 5.1 Hz, 1H), 4.36 (ddd, <i>J</i> = 10.3, 4.5, 3.3 Hz, 1H), 4.22 (dd, <i>J</i> = 5.2, 4.5 Hz, 1H), 4.01 (dd, <i>J</i> = 6.7, 5.8 Hz, 1H), 2.86–2.78 (m, 1H), 2.16 (dddd, <i>J</i> = 14.4, 11.7, 6.7, 4.5 Hz, 1H), 2.10–2.02 (m, 1H), 2.01 (ddd, <i>J</i> = 14.3, 10.3, 4.1 Hz, 1H), 1.96–1.90 (m, 1H), 1.77 (app ddt, <i>J</i> = 13.0, 11.7, 5.1 Hz, 1H), 1.58 (dddd, <i>J</i> = 13.0, 11.7, 9.6, 4.5 Hz, 1H). <sup>13</sup>C NMR (126 MHz, CD<sub>3</sub>CN (350 μL)/D<sub>2</sub>O (350 μL)/<i>d</i>-TFA (40 μL)): δ 172.1, 151.0, 149.4, 145.3, 144.1, 143.7, 136.4, 130.6, 129.6, 126.4, 125.2, 120.1, 93.8, 90.1, 83.9, 82.8, 74.9, 74.5, 53.6, 38.8, 30.9, 29.6, 28.7. HRMS (ESI+): calcd for [C<sub>23</sub>H<sub>28</sub>N<sub>7</sub>O<sub>7</sub>S<sub>1</sub>]<sup>+</sup> 546.1765, measured 546.1764, Δ 0.3 ppm.</div></div><div id="sec4_5_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i50" class="anchor-spacer"></div><h4 class="article-section__title" id="_i50"> (2<i>S</i>,5<i>S</i>)-2-Amino-5-(((2<i>R</i>,3<i>S</i>,4<i>R</i>,5<i>R</i>)-5-(6-amino-9<i>H</i>-purin-9-yl)-3,4-dihydroxytetrahydrofuran-2-yl)methyl)-7-(3-carbamoyl-4-fluorophenyl)hept-6-ynoic Acid (<b>30</b>)</h4><div class="NLM_p last">To a solution of <b>S71</b> (44 mg, 53 μmol) in THF (1 mL) at RT was added a 0.5 M LiOH aq solution (1 mL). The resulting mixture was stirred for 1 h, cooled to 0 °C, and a 1 M HCl aq solution (0.20 mL) was added. Volatiles were removed in vacuo. The residue was dissolved in a solution of ammonia in methanol (7 N, 5 mL) at RT. The resulting mixture was stirred for 18 h, and the volatiles were removed in vacuo. Crude material was purified by preparative HPLC (5% → 45% B over 30 min followed by 45% → 95% B over 5 min) to afford <b>30</b> (17 mg, 26 μmol, 50% over 2 steps) as the TFA salt upon treatment with <i>d</i>-TFA as part of the sample preparation for NMR analysis. <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>CN (350 μL)/D<sub>2</sub>O (350 μL)/<i>d</i>-TFA (40 μL)): δ 8.36 (s, 1H), 8.34 (s, 1H), 7.76 (dd, <i>J</i> = 7.1, 2.3 Hz, 1H), 7.51 (ddd, <i>J</i> = 8.6, 4.8, 2.3 Hz, 1H), 7.15 (dd, <i>J</i> = 11.1, 8.6 Hz, 1H), 6.00 (d, <i>J</i> = 5.1 Hz, 1H), 4.72 (app t, <i>J</i> = 5.1 Hz, 1H), 4.35 (ddd, <i>J</i> = 10.3, 4.5, 3.3 Hz, 1H), 4.21 (dd, <i>J</i> = 5.1, 4.5 Hz, 1H), 4.01 (dd, <i>J</i> = 6.7, 5.8 Hz, 1H), 2.82–2.76 (m, 1H), 2.15 (dddd, <i>J</i> = 14.4, 11.7, 6.7, 4.5 Hz, 1H), 2.07–2.02 (m, 1H), 1.99 (ddd, <i>J</i> = 13.9, 10.3, 4.5 Hz, 1H), 1.91 (ddd, <i>J</i> = 13.9, 10.8, 3.3 Hz, 1H), 1.75 (app ddt, <i>J</i> = 12.9, 11.7, 5.0 Hz, 1H), 1.56 (dddd, <i>J</i> = 12.9, 11.9, 9.5, 4.5 Hz, 1H). <sup>13</sup>C NMR (126 MHz, CD<sub>3</sub>CN (350 μL)/D<sub>2</sub>O (350 μL)/<i>d</i>-TFA (40 μL)): δ 172.2, 167.5 (d, <i>J</i><sub>C–F</sub> = 2.0 Hz), 160.7 (d, <i>J</i><sub>C–F</sub> = 253 Hz), 151.0, 149.4, 145.3, 144.1, 137.7 (d, <i>J</i><sub>C–F</sub> = 9.5 Hz), 134.8 (d, <i>J</i><sub>C–F</sub> = 2.7 Hz), 122.4 (d, <i>J</i><sub>C–F</sub> = 14 Hz), 120.9 (d, <i>J</i><sub>C–F</sub> = 3.7 Hz), 120.1, 117.9 (d, <i>J</i><sub>C–F</sub> = 25 Hz), 92.5, 90.0, 84.0, 82.4, 74.9, 74.5, 53.6, 38.9, 30.9, 29.6, 28.8. <sup>19</sup>F NMR (471 MHz, CD<sub>3</sub>CN (350 μL)/D<sub>2</sub>O (350 μL)/<i>d</i>-TFA (40 μL), HFB IStd): δ −113.6. HRMS (ESI+): calcd for [C<sub>24</sub>H<sub>26</sub>F<sub>1</sub>N<sub>7</sub>O<sub>6</sub>Na]<sup>+</sup> 550.1821, measured 550.1821, Δ 0.0 ppm.</div></div><div id="sec4_5_21" class="NLM_sec NLM_sec_level_3"><div id="ac_i51" class="anchor-spacer"></div><h4 class="article-section__title" id="_i51"> (2<i>S</i>,5<i>S</i>)-2-Amino-5-(((2<i>R</i>,3<i>S</i>,4<i>R</i>,5<i>R</i>)-5-(6-amino-9<i>H</i>-purin-9-yl)-3,4-dihydroxytetrahydrofuran-2-yl)methyl)-7-(3-carbamoyl-4-methylphenyl)hept-6-ynoic Acid (<b>31</b>)</h4><div class="NLM_p last">To a solution of <b>S73</b> (26 mg, 32 μmol) in THF (1.5 mL) at RT was added a 0.5 M LiOH aq solution (0.9 mL). The resulting mixture was stirred for 1 h, cooled to 0 °C, and a 1 M HCl aq solution (0.17 mL) was added. Volatiles were removed in vacuo. The residue was dissolved in a solution of ammonia in methanol (7 N, 5 mL) at RT. The resulting mixture was stirred for 18 h, and volatiles were removed in vacuo. Crude material was purified by preparative HPLC (5% → 45% B over 30 min followed by 45% → 95% B over 5 min) to afford <b>31</b> (11 mg, 17 μmol, 55% over 2 steps) as the TFA salt upon treatment with <i>d</i>-TFA as part of the sample preparation for NMR analysis. <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>CN (350 μL)/D<sub>2</sub>O (350 μL)/<i>d</i>-TFA (40 μL)): δ 8.36 (s, 1H), 8.33 (s, 1H), 7.37 (br d, <i>J</i> = 1.8 Hz, 1H), 7.29 (dd, <i>J</i> = 7.9, 1.8 Hz, 1H), 7.17 (br d, <i>J</i> = 7.9 Hz, 1H), 5.99 (d, <i>J</i> = 5.1 Hz, 1H), 4.73 (app t, <i>J</i> = 5.1 Hz, 1H), 4.35 (ddd, <i>J</i> = 10.3, 4.5, 3.2 Hz, 1H), 4.21 (dd, <i>J</i> = 5.1, 4.5 Hz, 1H), 4.00 (dd, <i>J</i> = 6.7, 5.8 Hz, 1H), 2.82–2.75 (m, 1H), 2.32 (s, 3H), 2.14 (dddd, <i>J</i> = 14.3, 11.7, 6.7, 4.5 Hz, 1H), 2.07–2.00 (m, 1H), 1.99 (ddd, <i>J</i> = 13.9, 10.3, 4.3 Hz, 1H), 1.90 (ddd, <i>J</i> = 13.9, 10.7, 3.2 Hz, 1H), 1.78–1.70 (app ddt, <i>J</i> = 13.0, 11.7, 5.1 Hz, 1H), 1.55 (dddd, <i>J</i> = 13.0, 11.8, 9.3, 4.5 Hz, 1H). <sup>13</sup>C NMR (126 MHz, CD<sub>3</sub>CN (350 μL)/D<sub>2</sub>O (350 μL)/<i>d</i>-TFA (40 μL)): δ 174.5, 172.2, 151.0, 149.4, 145.2, 144.1, 136.9, 136.5, 133.9, 132.1, 130.9, 121.4, 120.1, 92.1, 90.0, 83.9, 83.5, 74.8, 74.5, 53.6, 38.9, 30.9, 29.6, 28.8, 19.8. HRMS (ESI+): calcd for [C<sub>25</sub>H<sub>29</sub>N<sub>7</sub>O<sub>6</sub>Na]<sup>+</sup> 546.2072, measured 546.2074, Δ 0.5 ppm.</div></div><div id="sec4_5_22" class="NLM_sec NLM_sec_level_3"><div id="ac_i52" class="anchor-spacer"></div><h4 class="article-section__title" id="_i52"> (2<i>S</i>,5<i>S</i>)-2-Amino-5-(((2<i>R</i>,3<i>S</i>,4<i>R</i>,5<i>R</i>)-5-(6-amino-9<i>H</i>-purin-9-yl)-3,4-dihydroxytetrahydrofuran-2-yl)methyl)-7-(3-carbamoyl-4-(trifluoromethyl)phenyl)hept-6-ynoic Acid (<b>32</b>)</h4><div class="NLM_p last">To a solution of <b>S75</b> (51 mg, 58 μmol) in THF (5.0 mL) at RT was added a 0.5 M LiOH aq solution (1.5 mL). The resulting mixture was stirred for 3 h, cooled to 0 °C, and a 10% AcOH aq solution was added dropwise until pH 7 was reached. Volatiles were removed in vacuo. The residue was dissolved in a solution of ammonia in methanol (7 N, 8 mL) at RT. The resulting mixture was stirred for 18 h, and the volatiles were removed in vacuo. Crude material was purified by preparative HPLC (5% → 35% B over 30 min followed by 35% → 95% B over 5 min) to afford <b>32</b> (22 mg, 32 μmol, 55% over 2 steps) as the TFA salt upon treatment with <i>d</i>-TFA as part of the sample preparation for NMR analysis. <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>CN (350 μL)/D<sub>2</sub>O (350 μL)/<i>d</i>-TFA (40 μL)): δ 8.82 (s, 1H), 8.80 (s, 1H), 8.12 (d, <i>J</i> = 8.2 Hz, 1H), 8.01 (ddt, <i>J</i> = 8.2, 1.8, 1.0 Hz, 1H), 8.00–7.97 (m, 1H), 6.45 (d, <i>J</i> = 5.0 Hz, 1H), 5.16 (t, <i>J</i> = 5.1 Hz, 1H), 4.82–4.79 (m, 1H), 4.65 (t, <i>J</i> = 4.8 Hz, 1H), 4.46 (dd, <i>J</i> = 6.7, 5.8 Hz, 1H), 3.32–3.25 (m, 1H), 2.60 (dddd, <i>J</i> = 14.3, 11.5, 6.7, 4.5 Hz, 1H), 2.54–2.43 (m, 2H), 2.41–2.35 (m, 1H), 2.27–2.18 (m, 1H), 2.03 (dddd, <i>J</i> = 13.4, 11.8, 9.3, 4.5 Hz, 1H). <sup>13</sup>C NMR (126 MHz, CD<sub>3</sub>CN (350 μL)/D<sub>2</sub>O (350 μL)/<i>d</i>-TFA (40 μL)): δ 172.0, 171.9, 151.0, 149.4, 145.2, 144.0, 136.0, 133.8, 132.0, 128.3, 127.8, 127.8, 127.7, 126.6, 126.3, 125.5, 123.4, 120.0, 117.9, 115.6, 113.3, 95.7, 89.9, 83.8, 82.3, 74.8, 74.4, 53.5, 38.7, 30.7, 29.6, 28.7. HRMS (ESI+): calcd for [C<sub>25</sub>H<sub>26</sub>F<sub>3</sub>N<sub>7</sub>O<sub>6</sub>Na]<sup>+</sup> 600.1789, measured 600.1776, Δ 2.2 ppm.</div></div><div id="sec4_5_23" class="NLM_sec NLM_sec_level_3"><div id="ac_i53" class="anchor-spacer"></div><h4 class="article-section__title" id="_i53"> (2<i>S</i>,5<i>S</i>)-2-Amino-5-(((2<i>R</i>,3<i>S</i>,4<i>R</i>,5<i>R</i>)-5-(6-amino-9<i>H</i>-purin-9-yl)-3,4-dihydroxytetrahydrofuran-2-yl)methyl)-7-(3-carbamoyl-4-chlorophenyl)hept-6-ynoic Acid (<b>33</b>)</h4><div class="NLM_p last">To a solution of <b>S77</b> (45 mg, 53 μmol) in THF (1 mL) at RT was added a 0.5 M LiOH aq solution (1 mL). The resulting mixture was stirred for 1 h, cooled to 0 °C, and a 1 M HCl aq solution (0.20 mL) was added. Volatiles were removed in vacuo. The residue was dissolved in a solution of ammonia in methanol (7 N, 5 mL) at RT. The resulting mixture was stirred for 18 h, and the volatiles were removed in vacuo. Crude material was purified by preparative HPLC (5% → 45% B over 30 min followed by 45% → 95% B over 5 min) to afford <b>33</b> (35 mg, 53 μmol, quant over 2 steps) as the TFA salt upon treatment with <i>d</i>-TFA as part of the sample preparation for NMR analysis. <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>CN (350 μL)/D<sub>2</sub>O (350 μL)/<i>d</i>-TFA (40 μL)): δ 8.36 (s, 1H), 8.34 (s, 1H), 7.48–7.47 (m, 1H), 7.38–7.36 (m, 2H), 5.99 (d, <i>J</i> = 5.1 Hz, 1H), 4.72 (app t, <i>J</i> = 5.1 Hz, 1H), 4.34 (ddd, <i>J</i> = 10.4, 4.5, 3.2 Hz, 1H), 4.20 (dd, <i>J</i> = 5.1, 4.5 Hz, 1H), 4.00 (dd, <i>J</i> = 6.7, 5.8 Hz, 1H), 2.79 (app ddt, <i>J</i> = 10.7, 9.2, 4.5 Hz, 1H), 2.14 (dddd, <i>J</i> = 14.3, 11.6, 6.7, 4.5 Hz, 1H), 2.07–2.01 (m, 1H), 2.00 (ddd, <i>J</i> = 14.1, 10.4, 4.5 Hz, 1H), 1.91 (ddd, <i>J</i> = 14.1, 10.7, 3.2 Hz, 1H), 1.75 (dddd, <i>J</i> = 13.1, 11.6, 5.0, 4.5 Hz, 1H), 1.55 (dddd, <i>J</i> = 13.1, 11.8, 9.2, 4.5 Hz, 1H). <sup>13</sup>C NMR (126 MHz, CD<sub>3</sub>CN (350 μL)/D<sub>2</sub>O (350 μL)/<i>d</i>-TFA (40 μL)): δ 172.1, 171.1, 151.0, 149.4, 145.2, 144.1, 136.0, 135.1, 132.7, 131.3, 131.0, 123.1, 120.1, 93.9, 90.0, 83.9, 82.4, 74.8, 74.5, 53.5, 38.8, 30.8, 29.6, 28.7. HRMS (ESI+): calcd for [C<sub>24</sub>H<sub>26</sub>Cl<sub>1</sub>N<sub>7</sub>O<sub>6</sub>Na]<sup>+</sup> 566.1525, measured 566.1528, Δ 0.4 ppm.</div></div><div id="sec4_5_24" class="NLM_sec NLM_sec_level_3"><div id="ac_i54" class="anchor-spacer"></div><h4 class="article-section__title" id="_i54"> (2<i>S</i>,5<i>S</i>)-2-Amino-5-(((2<i>R</i>,3<i>S</i>,4<i>R</i>,5<i>R</i>)-5-(6-amino-9<i>H</i>-purin-9-yl)-3,4-dihydroxytetrahydrofuran-2-yl)methyl)-7-(1-oxo-1,2,3,4-tetrahydroisoquinolin-7-yl)hept-6-ynoic Acid (<b>34</b>)</h4><div class="NLM_p last">To a solution of <b>S78</b> (29 mg, 35 μmol) in THF (0.65 mL) at RT was added a 0.5 M LiOH aq solution (0.65 mL). The resulting mixture was stirred for 1 h, cooled to 0 °C, and a 1 M HCl aq solution (0.14 mL) was added. Volatiles were removed in vacuo. The residue was dissolved in a solution of ammonia in methanol (7 N, 5 mL) at RT. The resulting mixture was stirred for 18 h, and volatiles were removed in vacuo. Crude material was purified by preparative HPLC (5% → 45% B over 30 min followed by 45% → 95% B over 5 min) to afford <b>34</b> (23 mg, 35 μmol, quant over 2 steps) as the TFA salt upon treatment with <i>d</i>-TFA as part of the sample preparation for NMR analysis. <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>CN (350 μL)/D<sub>2</sub>O (350 μL)/<i>d</i>-TFA (40 μL)): δ 8.36 (s, 1H), 8.33 (s, 1H), 7.08 (br d, <i>J</i> = 7.7 Hz, 1H), 6.96 (dd, <i>J</i> = 7.7, 1.6 Hz, 1H), 6.82 (d, <i>J</i> = 1.6 Hz, 1H), 5.99 (d, <i>J</i> = 5.1 Hz, 1H), 4.73 (app t, <i>J</i> = 5.1 Hz, 1H), 4.35 (ddd, <i>J</i> = 10.4, 4.5, 3.2 Hz, 1H), 4.21 (dd, <i>J</i> = 5.1, 4.5 Hz, 1H), 4.00 (dd, <i>J</i> = 6.7, 5.6 Hz, 1H), 2.89–2.85 (m, 2H), 2.78 (dddd, <i>J</i> = 10.8, 9.3, 4.7, 4.3 Hz, 1H), 2.53–2.48 (m, 2H), 2.13 (dddd, <i>J</i> = 14.3, 11.8, 6.7, 4.6 Hz, 1H), 2.03 (dddd, <i>J</i> = 14.3, 11.7, 5.6, 5.2 Hz, 1H), 1.99 (ddd, <i>J</i> = 14.0, 10.4, 4.3 Hz, 1H), 1.89 (ddd, <i>J</i> = 14.0, 10.8, 3.2 Hz, 1H), 1.74 (dddd, <i>J</i> = 13.0, 11.8, 5.2, 4.7 Hz, 1H), 1.54 (dddd, <i>J</i> = 13.0, 11.7, 9.3, 4.6 Hz, 1H). <sup>13</sup>C NMR (126 MHz, CD<sub>3</sub>CN (350 μL)/D<sub>2</sub>O (350 μL)/<i>d</i>-TFA (40 μL)): δ 174.6, 172.2, 151.0, 149.4, 145.2, 144.1, 138.1, 129.2, 127.5, 125.6, 122.8, 120.1, 119.1, 91.7, 90.0, 83.9, 83.8, 74.8, 74.5, 53.5, 38.9, 30.9, 30.7, 29.5, 28.7, 25.4. HRMS (ESI+): calcd for [C<sub>26</sub>H<sub>30</sub>N<sub>7</sub>O<sub>6</sub>]<sup>+</sup> 536.2252, measured 536.2268, Δ 3.0 ppm.</div></div><div id="sec4_5_25" class="NLM_sec NLM_sec_level_3"><div id="ac_i55" class="anchor-spacer"></div><h4 class="article-section__title" id="_i55"> (2<i>S</i>,5<i>S</i>)-2-Amino-5-(((2<i>R</i>,3<i>S</i>,4<i>R</i>,5<i>R</i>)-5-(6-amino-9<i>H</i>-purin-9-yl)-3,4-dihydroxytetrahydrofuran-2-yl)methyl)-7-(3-oxoisoindolin-5-yl)hept-6-ynoic Acid (<b>35</b>)</h4><div class="NLM_p last">To a solution of <b>8</b> (107 mg, 0.155 mmol) and 6-iodoisoindolin-1-one (101 mg, 0.388 mmol, 2.5 equiv) in a 5:3:1 mixture of toluene, DMF, and <i>i</i>-Pr<sub>2</sub>NEt (degassed by sparging with nitrogen for 15 min, 4.7 mL) at RT were added CuI (7 mg, 0.04 mmol, 25 mol %) and Pd(PPh<sub>3</sub>)<sub>4</sub> (9 mg, 8 μmol, 5 mol %) rapidly. The resulting mixture was stirred at 70 °C for 3 h. The solution was allowed to cool to RT, and volatiles were removed in vacuo. Crude material was filtered through a short pad of silica gel (EtOAc/<i>i</i>-PrOH, 85:15) to afford the desired crude reaction product which was used directly in the next step without further purification. To a solution of the crude reaction product in THF (3 mL) at RT was added a 0.5 M LiOH aq solution (3 mL). The resulting mixture was stirred for 1 h, cooled to 0 °C, and a 1 M HCl aq solution (0.60 mL) was added. Volatiles were removed in vacuo. The residue was dissolved in a solution of ammonia in methanol (7 N, 18 mL) at RT. The resulting mixture was stirred for 18 h, and volatiles were removed in vacuo. Crude material was purified by preparative HPLC (5% → 45% B over 30 min followed by 45% → 95% B over 5 min) to afford <b>35</b> (35 mg, 55 μmol, 36% over 3 steps) as the TFA salt upon treatment with <i>d</i>-TFA as part of the sample preparation for NMR analysis. <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>CN (350 μL)/D<sub>2</sub>O (350 μL)/<i>d</i>-TFA (40 μL)): δ 8.36 (s, 1H), 8.33 (s, 1H), 7.65 (br s, 1H), 7.54 (dd, <i>J</i> = 7.9, 1.5 Hz, 1H), 7.48–7.45 (m, 1H), 6.01 (d, <i>J</i> = 5.1 Hz, 1H), 4.74 (app t, <i>J</i> = 5.1 Hz, 1H), 4.40 (br s, 2H), 4.38 (ddd, <i>J</i> = 10.2, 4.4, 3.3 Hz, 1H), 4.23 (app t, <i>J</i> = 4.8 Hz, 1H), 4.03 (app t, <i>J</i> = 6.2 Hz, 1H), 2.82 (app ddt, <i>J</i> = 10.7, 9.2, 4.5 Hz, 1H), 2.18 (dddd, <i>J</i> = 14.4, 11.4, 6.7, 4.5 Hz, 1H), 2.06 (app ddt, <i>J</i> = 14.5, 11.5, 5.6 Hz, 1H), 2.01 (ddd, <i>J</i> = 14.1, 10.2, 4.2 Hz, 1H), 1.96–1.90 (m, 1H), 1.77 (app ddt, <i>J</i> = 12.9, 11.5, 5.0 Hz, 1H), 1.57 (dddd, <i>J</i> = 13.4, 11.7, 9.4, 4.4 Hz, 1H). <sup>13</sup>C NMR (126 MHz, CD<sub>3</sub>CN (350 μL)/D<sub>2</sub>O (350 μL)/<i>d</i>-TFA (40 μL)): δ 172.8, 172.2, 150.9, 149.4, 145.2, 145.1, 144.1, 136.1, 132.7, 127.0, 124.9, 123.8, 120.0, 92.7, 90.0, 84.0, 83.4, 74.8, 74.5, 53.6, 47.0, 38.8, 30.9, 29.6, 28.8. HRMS (ESI+): calcd for [C<sub>25</sub>H<sub>28</sub>N<sub>7</sub>O<sub>6</sub>]<sup>+</sup> 522.2096, measured 522.2103, Δ 1.4 ppm.</div></div><div id="sec4_5_26" class="NLM_sec NLM_sec_level_3"><div id="ac_i56" class="anchor-spacer"></div><h4 class="article-section__title" id="_i56"> (2<i>S</i>,5<i>S</i>)-2-Amino-5-(((2<i>R</i>,3<i>S</i>,4<i>R</i>,5<i>R</i>)-5-(6-amino-9<i>H</i>-purin-9-yl)-3,4-dihydroxytetrahydrofuran-2-yl)methyl)-7-(6-carbamoylbenzo[<i>d</i>][1,3]dioxol-4-yl)hept-6-ynoic Acid (<b>36</b>)</h4><div class="NLM_p last">To a solution of <b>S81</b> (27 mg, 32 μmol) in THF (0.60 mL) at RT was added a 0.5 M LiOH aq solution (0.60 mL). The resulting mixture was stirred for 1 h, cooled to 0 °C, and a 1 M HCl aq solution (90 μL) was added. Volatiles were removed in vacuo. The residue was dissolved in a solution of ammonia in methanol (7 N, 4.5 mL) at RT. The resulting mixture was stirred for 18 h, and volatiles were removed in vacuo. Crude material was purified by preparative HPLC (5% → 40% B′ over 40 min followed by 40% → 95% B′ over 5 min) to afford <b>36</b> (16 mg, 24 μmol, 76% over 2 steps) as the TFA salt. <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>CN (350 μL)/D<sub>2</sub>O (350 μL)): δ 8.31 (s, 1H), 8.29 (s, 1H), 7.32 (d, <i>J</i> = 1.8 Hz, 1H), 7.19 (d, <i>J</i> = 1.8 Hz, 1H), 6.05 (d, <i>J</i> = 1.0 Hz, 1H), 6.04 (d, <i>J</i> = 1.0 Hz, 1H), 5.96 (d, <i>J</i> = 5.1 Hz, 1H), 4.71 (app t, <i>J</i> = 5.1 Hz, 1H), 4.32 (ddd, <i>J</i> = 10.4, 4.4, 3.2 Hz, 1H), 4.19 (dd, <i>J</i> = 5.1, 4.4 Hz, 1H), 3.88 (dd, <i>J</i> = 6.7, 5.8 Hz, 1H), 2.84–2.78 (m, 1H), 2.08 (dddd, <i>J</i> = 14.2, 11.7, 6.7, 4.7 Hz, 1H), 2.03–1.97 (m, 2H), 1.90 (ddd, <i>J</i> = 13.9, 10.7, 3.2 Hz, 1H), 1.72 (app ddt, <i>J</i> = 13.1, 11.7, 5.0 Hz, 1H), 1.55 (dddd, <i>J</i> = 13.1, 11.7, 9.2, 4.7 Hz, 1H). <sup>13</sup>C NMR (101 MHz, CD<sub>3</sub>CN (350 μL)/D<sub>2</sub>O (350 μL)/<i>d</i>-TFA (40 μL)): δ 172.4, 171.4, 152.8, 151.1, 149.5, 149.1, 145.4, 144.4, 127.8, 126.9, 120.1, 108.6, 105.4, 104.0, 97.2, 90.3, 84.1, 77.5, 74.9, 74.6, 53.8, 38.7, 30.9, 29.9, 28.9. HRMS (ESI+): calcd for [C<sub>25</sub>H<sub>28</sub>N<sub>7</sub>O<sub>8</sub>]<sup>+</sup> 554.1994, measured 554.2004, Δ 1.8 ppm.</div></div><div id="sec4_5_27" class="NLM_sec NLM_sec_level_3"><div id="ac_i57" class="anchor-spacer"></div><h4 class="article-section__title" id="_i57"> (2<i>S</i>,5<i>S</i>)-2-Amino-5-(((2<i>R</i>,3<i>S</i>,4<i>R</i>,5<i>R</i>)-5-(6-amino-9<i>H</i>-purin-9-yl)-3,4-dihydroxytetrahydrofuran-2-yl)methyl)-7-(6-carbamoylpyridin-2-yl)hept-6-ynoic Acid (<b>37</b>)</h4><div class="NLM_p last">To a solution of <b>S82</b> (28 mg, 35 μmol) in THF (0.66 mL) at RT was added a 0.5 M LiOH aq solution (0.66 mL). The resulting mixture was stirred for 1 h, cooled to 0 °C, and a 1 M HCl aq solution (0.13 mL) was added. Volatiles were removed in vacuo. The residue was dissolved in a solution of ammonia in methanol (7 N, 4 mL) at RT. The resulting mixture was stirred for 18 h, and the volatiles were removed in vacuo. Crude material was purified by preparative HPLC (5% → 20% B over 30 min followed by 20% → 95% B over 5 min) to afford <b>37</b> (19 mg, 30 μmol, 88% over 2 steps) as the TFA salt upon treatment with <i>d</i>-TFA as part of the sample preparation for NMR analysis. <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>CN (350 μL)/D<sub>2</sub>O (350 μL)/<i>d</i>-TFA (40 μL)): δ 8.37 (s, 1H), 8.34 (s, 1H), 7.99 (dd, <i>J</i> = 7.9, 1.4 Hz, 1H), 7.96 (dd, <i>J</i> = 7.9, 7.5 Hz, 1H), 7.63 (dd, <i>J</i> = 7.5, 1.4 Hz, 1H), 6.00 (d, <i>J</i> = 5.0 Hz, 1H), 4.74 (dd, <i>J</i> = 5.2, 5.0 Hz, 1H), 4.37 (ddd, <i>J</i> = 10.1, 4.5, 3.4 Hz, 1H), 4.23 (dd, <i>J</i> = 5.2, 4.5 Hz, 1H), 4.02 (dd, <i>J</i> = 6.7, 5.8 Hz, 1H), 2.91–2.85 (m, 1H), 2.15 (dddd, <i>J</i> = 14.3, 11.6, 6.7, 4.6 Hz, 1H), 2.10–2.02 (m, 2H), 1.98 (ddd, <i>J</i> = 13.9, 10.6, 3.4 Hz, 1H), 1.80 (app ddt, <i>J</i> = 13.1, 11.6, 5.0 Hz, 1H), 1.64 (dddd, <i>J</i> = 13.1, 11.8, 9.2, 4.6 Hz, 1H). <sup>13</sup>C NMR (126 MHz, CD<sub>3</sub>CN (350 μL)/D<sub>2</sub>O (350 μL)/<i>d</i>-TFA (40 μL)): δ 172.1, 167.9, 151.0, 149.6, 149.4, 145.2, 144.2, 142.3, 140.6, 131.6, 123.1, 120.1, 95.1, 90.0, 83.8, 82.2, 74.8, 74.4, 53.5, 38.3, 30.6, 29.6, 28.7. HRMS (ESI+): calcd for [C<sub>23</sub>H<sub>27</sub>N<sub>8</sub>O<sub>6</sub>]<sup>+</sup> 511.2048, measured 511.2059, Δ 2.1 ppm.</div></div><div id="sec4_5_28" class="NLM_sec NLM_sec_level_3"><div id="ac_i58" class="anchor-spacer"></div><h4 class="article-section__title" id="_i58"> (2<i>S</i>,5<i>S</i>)-2-Amino-5-(((2<i>R</i>,3<i>S</i>,4<i>R</i>,5<i>R</i>)-5-(6-amino-9<i>H</i>-purin-9-yl)-3,4-dihydroxytetrahydrofuran-2-yl)methyl)-7-(2-carbamoylpyridin-4-yl)hept-6-ynoic Acid (<b>38</b>)</h4><div class="NLM_p last">To a solution of <b>S83</b> (35 mg, 43 μmol) in THF (0.8 mL) at RT was added a 0.5 M LiOH aq solution (0.8 mL). The resulting mixture was stirred for 1 h, cooled to 0 °C, and a 1 M HCl aq solution (0.16 mL) was added. Volatiles were removed in vacuo. The residue was dissolved in a solution of ammonia in methanol (7 N, 4 mL) at RT. The resulting mixture was stirred for 18 h, and volatiles were removed in vacuo. Crude material was purified by preparative HPLC (5% → 20% B over 30 min followed by 20% → 95% B over 5 min) to afford <b>38</b> (19 mg, 30 μmol, 70% over 2 steps) as the TFA salt upon treatment with <i>d</i>-TFA as part of the sample preparation for NMR analysis. <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>CN (350 μL)/D<sub>2</sub>O (350 μL)/<i>d</i>-TFA (40 μL)): δ 8.68 (dd, <i>J</i> = 5.8, 0.6 Hz, 1H), 8.37 (s, 1H), 8.35 (s, 1H), 8.27 (dd, <i>J</i> = 1.6, 0.6 Hz, 1H), 7.90 (dd, <i>J</i> = 5.8, 1.6 Hz, 1H), 6.01 (d, <i>J</i> = 5.0 Hz, 1H), 4.73 (dd, <i>J</i> = 5.2, 5.0 Hz, 1H), 4.36 (ddd, <i>J</i> = 10.5, 4.4, 3.3 Hz, 1H), 4.22 (dd, <i>J</i> = 5.2, 4.4 Hz, 1H), 4.01 (dd, <i>J</i> = 6.6, 5.9 Hz, 1H), 2.95 (app ddt, <i>J</i> = 10.9, 9.3, 4.8 Hz, 1H), 2.14 (dddd, <i>J</i> = 14.3, 11.6, 6.6, 4.6 Hz, 1H), 2.09–2.02 (m, 2H), 1.99 (ddd, <i>J</i> = 14.0, 10.9, 3.3 Hz, 1H), 1.82 (app ddt, <i>J</i> = 13.3, 11.6, 4.8 Hz, 1H), 1.64 (dddd, <i>J</i> = 13.3, 11.9, 9.3, 4.6 Hz, 1H). <sup>13</sup>C NMR (126 MHz, CD<sub>3</sub>CN (350 μL)/D<sub>2</sub>O (350 μL)/<i>d</i>-TFA (40 μL)): δ 172.1, 165.6, 151.0, 149.4, 145.5, 145.3, 145.2, 144.2, 141.1, 131.6, 127.6, 120.1, 105.9, 90.1, 83.6, 81.0, 74.8, 74.5, 53.5, 38.3, 30.5, 30.2, 28.7. HRMS (ESI+): calcd for [C<sub>23</sub>H<sub>27</sub>N<sub>8</sub>O<sub>6</sub>]<sup>+</sup> 511.2048, measured 511.2050, Δ 0.4 ppm.</div></div><div id="sec4_5_29" class="NLM_sec NLM_sec_level_3"><div id="ac_i59" class="anchor-spacer"></div><h4 class="article-section__title" id="_i59"> (2<i>S</i>,5<i>S</i>)-2-Amino-5-(((2<i>R</i>,3<i>S</i>,4<i>R</i>,5<i>R</i>)-5-(6-amino-9<i>H</i>-purin-9-yl)-3,4-dihydroxytetrahydrofuran-2-yl)methyl)-7-(5-carbamoylpyridin-3-yl)hept-6-ynoic Acid (<b>39</b>)</h4><div class="NLM_p last">To a solution of <b>S84</b> (26 mg, 32 μmol) in THF (0.6 mL) at RT was added a 0.5 M LiOH aq solution (0.6 mL). The resulting mixture was stirred for 1 h, cooled to 0 °C, and a 1 M HCl aq solution (0.13 mL) was added. Volatiles were removed in vacuo. The residue was dissolved in a solution of ammonia in methanol (7 N, 4.5 mL) at RT. The resulting mixture was stirred for 18 h, and the volatiles were removed in vacuo. Crude material was purified by preparative HPLC (5% → 20% B over 30 min followed by 20% → 95% B over 5 min) to afford <b>39</b> (16 mg, 26 μmol, 80% over 2 steps) as the TFA salt upon treatment with <i>d</i>-TFA as part of the sample preparation for NMR analysis. <sup>1</sup>H NMR (500 MHz, CD<sub>3</sub>CN (350 μL)/D<sub>2</sub>O (350 μL)/<i>d</i>-TFA (40 μL)): δ 9.04 (d, <i>J</i> = 1.8 Hz, 1H), 8.89 (d, <i>J</i> = 1.8 Hz, 1H), 8.81 (app t, <i>J</i> = 1.8 Hz, 1H), 8.37 (s, 1H), 8.36 (s, 1H), 6.01 (d, <i>J</i> = 4.9 Hz, 1H), 4.72 (app t, <i>J</i> = 4.9 Hz, 1H), 4.37 (ddd, <i>J</i> = 10.3, 4.9, 3.1 Hz, 1H), 4.22 (app t, <i>J</i> = 4.9 Hz, 1H), 4.01 (app t, <i>J</i> = 6.3 Hz, 1H), 2.95–2.86 (m, 1H), 2.15 (dddd, <i>J</i> = 14.3, 11.6, 6.3, 4.8 Hz, 1H), 2.10–2.01 (m, 2H), 1.98 (ddd, <i>J</i> = 13.7, 10.7, 3.1 Hz, 1H), 1.85–1.77 (m, 1H), 1.68–1.59 (m, 1H). <sup>13</sup>C NMR (126 MHz, CD<sub>3</sub>CN (350 μL)/D<sub>2</sub>O (350 μL)/<i>d</i>-TFA (40 μL)): δ 172.1, 165.8, 151.1, 149.4, 147.7, 147.0, 145.3, 144.1, 141.1, 133.9, 125.1, 120.1, 101.4, 90.1, 83.7, 77.4, 74.9, 74.5, 53.5, 38.4, 30.6, 30.0, 28.7. HRMS (ESI+): calcd for [C<sub>23</sub>H<sub>27</sub>N<sub>8</sub>O<sub>6</sub>]<sup>+</sup> 511.2048, measured 511.2069, Δ 4.0 ppm.</div></div><div id="sec4_5_30" class="NLM_sec NLM_sec_level_3"><div id="ac_i60" class="anchor-spacer"></div><h4 class="article-section__title" id="_i60"> (2<i>S</i>,5<i>S</i>)-2-Amino-5-(((2<i>R</i>,3<i>S</i>,4<i>R</i>,5<i>R</i>)-5-(6-amino-9<i>H</i>-purin-9-yl)-3,4-dihydroxytetrahydrofuran-2-yl)methyl)-7-(4-carbamoylpyridin-2-yl)hept-6-ynoic Acid (<b>40</b>)</h4><div class="NLM_p last">To a solution of <b>S85</b> (22 mg, 27 μmol) in THF (1.0 mL) at RT was added a 0.5 M LiOH aq solution (1.0 mL). The resulting mixture was stirred for 1 h, cooled to 0 °C, and a 1 M HCl aq solution (0.20 mL) was added. Volatiles were removed in vacuo. The residue was dissolved in a solution of ammonia in methanol (7 N, 4 mL) at RT. The resulting mixture was stirred for 18 h, and the volatiles were removed in vacuo. Crude material was purified by preparative HPLC (5% → 20% B over 30 min followed by 20% → 95% B over 5 min) to afford <b>40</b> (14 mg, 22 μmol, 82% over 2 steps) as the TFA salt upon treatment with <i>d</i>-TFA as part of the sample preparation for NMR analysis. <sup>1</sup>H NMR (500 MHz, CD<sub>3</sub>CN (350 μL)/D<sub>2</sub>O (350 μL)/<i>d</i>-TFA (40 μL)): δ 8.73 (dd, <i>J</i> = 6.1, 0.7 Hz, 1H), 8.37 (s, 1H), 8.35 (s, 1H), 8.22 (dd, <i>J</i> = 1.9, 0.7 Hz, 1H), 8.14 (dd, <i>J</i> = 6.1, 1.9 Hz, 1H), 6.01 (d, <i>J</i> = 4.9 Hz, 1H), 4.73 (dd, <i>J</i> = 5.1, 4.9 Hz, 1H), 4.35 (ddd, <i>J</i> = 10.2, 4.8, 3.4 Hz, 1H), 4.23 (dd, <i>J</i> = 5.1, 4.8 Hz, 1H), 4.01 (app t, <i>J</i> = 6.3 Hz, 1H), 3.03–2.96 (m, 1H), 2.18–1.99 (m, 4H), 1.89–1.80 (m, 1H), 1.74–1.64 (m, 1H). <sup>13</sup>C NMR (126 MHz, CD<sub>3</sub>CN (350 μL)/D<sub>2</sub>O (350 μL)/<i>d</i>-TFA (40 μL)): δ 172.1, 166.8, 151.0, 150.0, 149.4, 145.3, 144.8, 144.2, 137.5, 129.7, 124.8, 120.1, 106.3, 90.1, 83.4, 76.1, 74.8, 74.4, 53.5, 38.0, 30.3, 30.2, 28.6. HRMS (ESI+): calcd for [C<sub>23</sub>H<sub>27</sub>N<sub>8</sub>O<sub>6</sub>]<sup>+</sup> 511.2048, measured 511.2055, Δ 1.3 ppm.</div></div><div id="sec4_5_31" class="NLM_sec NLM_sec_level_3"><div id="ac_i61" class="anchor-spacer"></div><h4 class="article-section__title" id="_i61"> (2<i>S</i>,5<i>S</i>)-2-Amino-5-(((2<i>R</i>,3<i>S</i>,4<i>R</i>,5<i>R</i>)-5-(6-amino-9<i>H</i>-purin-9-yl)-3,4-dihydroxytetrahydrofuran-2-yl)methyl)-7-(5-aminonaphthalen-1-yl)hept-6-ynoic Acid (<b>41</b>)</h4><div class="NLM_p last">To a solution of <b>S86</b> (34 mg, 41 μmol) in THF (0.76 mL) at RT was added a 0.5 M LiOH aq solution (0.76 mL). The resulting mixture was stirred for 1 h, cooled to 0 °C, and a 1 M HCl aq solution (0.17 mL) was added. Volatiles were removed in vacuo. The residue was dissolved in a solution of ammonia in methanol (7 N, 4.5 mL) at RT. The resulting mixture was stirred for 18 h, and the volatiles were removed in vacuo. Crude material was purified by preparative HPLC (5% → 45% B over 30 min followed by 45% → 95% B over 5 min) to afford <b>41</b> (20 mg, 26 μmol, 64% over 2 steps) as the TFA salt upon treatment with <i>d</i>-TFA as part of sample preparation for NMR analysis. <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>CN (350 μL)/D<sub>2</sub>O (350 μL)/<i>d</i>-TFA (40 μL)): δ 8.33 (s, 1H), 8.29 (app dt, <i>J</i> = 8.4, 1.0 Hz, 1H), 8.29 (s, 1H), 7.89 (app dt, <i>J</i> = 8.5, 1.0 Hz, 1H), 7.70 (dd, <i>J</i> = 7.2, 1.0 Hz, 1H), 7.63 (dd, <i>J</i> = 7.5, 1.0 Hz, 1H), 7.61 (dd, <i>J</i> = 8.5, 7.2 Hz, 1H), 7.57 (dd, <i>J</i> = 8.4, 7.5 Hz, 1H), 6.00 (d, <i>J</i> = 5.0 Hz, 1H), 4.77 (dd, <i>J</i> = 5.2, 5.0 Hz, 1H), 4.44 (ddd, <i>J</i> = 10.0, 4.6, 3.3 Hz, 1H), 4.28 (dd, <i>J</i> = 5.2, 4.6 Hz, 1H), 4.06 (dd, <i>J</i> = 6.7, 5.6 Hz, 1H), 3.01–2.95 (m, 1H), 2.23 (dddd, <i>J</i> = 14.4, 11.6, 6.7, 4.6 Hz, 1H), 2.17–2.10 (m, 1H), 2.11 (ddd, <i>J</i> = 14.0, 10.0, 4.5 Hz, 1H), 2.05 (ddd, <i>J</i> = 14.0, 10.3, 3.3 Hz, 1H), 1.84 (app ddt, <i>J</i> = 13.1, 11.6, 5.1 Hz, 1H), 1.72–1.64 (m, 1H). <sup>13</sup>C NMR (126 MHz, CD<sub>3</sub>CN (350 μL)/D<sub>2</sub>O (350 μL)/<i>d</i>-TFA (40 μL)): δ 172.1, 150.8, 149.4, 145.1, 144.2, 134.6, 132.5, 128.5, 128.4, 127.5, 127.5, 127.4, 123.0, 122.7, 122.0, 119.9, 98.5, 90.1, 83.9, 81.3, 74.7, 74.4, 53.5, 38.6, 30.8, 29.7, 28.8. HRMS (ESI+): calcd for [C<sub>27</sub>H<sub>30</sub>N<sub>7</sub>O<sub>5</sub>]<sup>+</sup> 532.2303, measured 532.2325, Δ 4.2 ppm.</div></div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i62" class="anchor-spacer"></div><h3 class="article-section__title" id="_i62"> Synthesis of Compounds <b>S2</b>–<b>S86</b></h3><div id="sec4_6_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i63" class="anchor-spacer"></div><h4 class="article-section__title" id="_i63"> Methyl 2-(Dimethoxyphosphoryl)-2-(2,2,2-trifluoroacetamido)acetate (<b>S2</b>).<a onclick="showRef(event, 'ref67'); return false;" href="javascript:void(0);" class="ref ref67">(67)</a></h4><div class="NLM_p last">To a solution of (±)-<i>Z</i>-α-Phosphonoglycine trimethyl ester (<b>S1</b>) (3.31 g, 9.99 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (50 mL) at RT, was added Pd/C (10%, 0.53 g, 0.50 mmol, 5 mol %). The flask headspace was evacuated and refilled with hydrogen (3×), and the reaction mixture was stirred vigorously under a hydrogen atmosphere (balloon) for 24 h. The flask headspace was evacuated and refilled with nitrogen, and the contents were filtered through a pad of Celite. The filtrate was cooled to 0 °C, and TFAA (3.47 mL, 25.0 mmol, 2.5 equiv) was added dropwise. The reaction mixture was stirred at RT for 12 h, and the solution was concentrated to yield an orange residue. Crude material was purified by silica gel chromatography (hexanes/EtOAc, 0 → 100%) to afford TFA protected phosphonate <b>S2</b> (2.05 g, 6.99 mmol, 70% over 2 steps). <i>R</i><sub><i>f</i></sub> = 0.41 (EtOAc). FTIR (thin film), ν<sub>max</sub> (cm<sup>−1</sup>): 3197, 3046, 2964, 2861, 1757, 1722, 1559, 1433, 1360, 1307, 1257, 1212, 1180, 1150, 1035. <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 7.23 (br s, 1H), 5.14 (dd, <sup>2</sup><i>J</i><sub>H–P</sub> = 21.3, <i>J</i> = 8.7 Hz, 1H), 3.89 (s, 3H), 3.87 (d, <sup>3</sup><i>J</i><sub>H–P</sub>, <i>J</i> = 11.2 Hz, 3H), 3.83 (d, <sup>3</sup><i>J</i><sub>H–P</sub>, <i>J</i> = 11.1 Hz, 3H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 165.5, 157.2 (qd, <sup>2</sup><i>J</i><sub>C–F</sub> = 38 Hz, <sup>3</sup><i>J</i><sub>C–P</sub> = 5.9 Hz), 115.7 (q, <sup>1</sup><i>J</i><sub>C–F</sub> = 287 Hz), 54.5 (d, <sup>2</sup><i>J</i><sub>C–P</sub> = 6.6 Hz), 54.1 (d, <sup>2</sup><i>J</i><sub>C–P</sub> = 6.9 Hz), 53.3, 50.3 (d, <sup>1</sup><i>J</i><sub>C–P</sub> = 152 Hz). <sup>19</sup>F NMR (471 MHz, CDCl<sub>3</sub>, BTF IStd): δ −76.5. <sup>31</sup>P NMR (162 MHz, CDCl<sub>3</sub>): δ 16.1. HRMS (ESI+): calcd for [C<sub>7</sub>H<sub>12</sub>F<sub>3</sub>N<sub>1</sub>O<sub>6</sub>P<sub>1</sub>]<sup>+</sup> 294.0360, measured 294.0349, Δ 3.7 ppm.</div></div><div id="sec4_6_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i64" class="anchor-spacer"></div><h4 class="article-section__title" id="_i64"> Methyl (2<i>S</i>,5<i>S</i>)-5-(((3a<i>R</i>,4<i>R</i>,6<i>R</i>,6a<i>R</i>)-6-Methoxy-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)methyl)-2-(2,2,2-trifluoroacetamido)hept-6-ynoate (<b>S3</b>)</h4><div class="NLM_p last">To a vial charged with <b>7</b> (182 mg, 0.307 mmol) was added DMF (3 mL) at RT under a nitrogen atmosphere. A separate vial was charged with TASF (253 mg, 0.920 mmol, 3.0 equiv) in a glovebox, put under a nitrogen atmosphere, and DMF (2 mL) was added at RT. The resulting solution was added to the vial containing alkyne <b>7</b>, and the reaction mixture was stirred at 60 °C for 2 h. Volatiles were removed in vacuo. The residue was partitioned between Et<sub>2</sub>O (40 mL) and a 1:1 mixture of brine and 5% LiCl aq solution (40 mL). The layers were separated, and the aqueous phase was extracted with Et<sub>2</sub>O (3 × 30 mL). The combined organic extracts were dried over MgSO<sub>4</sub>, filtered, and concentrated. Crude material was purified by silica gel chromatography (hexanes/EtOAc, 5 → 50%) to yield <b>S3</b> (128 mg, 0.293 mmol, 95%) as a colorless oil. <i>R</i><sub><i>f</i></sub> = 0.35 (hexanes/EtOAc, 60:40). FTIR (neat), ν<sub>max</sub> (cm<sup>–1</sup>): 3292, 3089, 2990, 2954, 2938, 2836, 1747, 1719, 1553, 1455, 1440, 1383, 1275, 1209, 1160, 1104, 1060. <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 6.94 (d, <i>J</i> = 7.9 Hz, 1H), 4.93 (s, 1H), 4.64 (td, <i>J</i> = 7.8, 4.9 Hz, 1H), 4.59 (d, <i>J</i> = 6.0 Hz, 1H), 4.55 (dd, <i>J</i> = 6.0, 1.0 Hz, 1H), 4.45 (ddd, <i>J</i> = 11.2, 3.9, 0.9 Hz, 1H), 3.80 (s, 3H), 3.31 (s, 3H), 2.66–2.58 (m, 1H), 2.14 (d, <i>J</i> = 2.4 Hz, 1H), 2.10 (ddq, <i>J</i> = 14.0, 10.8, 5.2 Hz, 1H), 2.05–1.94 (m, 1H), 1.67 (ddd, <i>J</i> = 13.3, 11.2, 4.0 Hz, 1H), 1.55 (dddd, <i>J</i> = 21.7, 13.2, 10.9, 4.6 Hz, 2H), 1.47 (s, 3H), 1.40 (dddd, <i>J</i> = 16.5, 14.4, 8.7, 3.9 Hz, 1H), 1.30 (s, 3H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 171.3, 157.0 (q, <sup>2</sup><i>J</i><sub>C–F</sub> = 38 Hz), 115.1 (q, <sup>1</sup><i>J</i><sub>C–F</sub> = 287 Hz), 112.5, 110.1, 85.6, 85.1, 85.0, 84.5, 71.6, 55.3, 53.1, 52.6, 52.5, 40.4, 30.7, 29.8, 28.6, 26.6, 25.1. HRMS (ESI+): calcd for [C<sub>19</sub>H<sub>26</sub>F<sub>3</sub>N<sub>1</sub>O<sub>7</sub>Na]<sup>+</sup> 460.1554, measured 460.1569, Δ 3.3 ppm.</div></div><div id="sec4_6_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i65" class="anchor-spacer"></div><h4 class="article-section__title" id="_i65"> (3<i>R</i>,4<i>R</i>,5<i>R</i>)-5-((2<i>S</i>,5<i>S</i>)-2-Ethynyl-6-methoxy-6-oxo-5-(2,2,2-trifluoroacetamido)hexyl)tetrahydrofuran-2,3,4-triyl Triacetate (<b>S4</b>)</h4><div class="NLM_p last">To a solution of acetonide <b>S3</b> (1.21 g, 2.77 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (16.5 mL) at RT was added a 2:1 mixture of TFA and water (51 mL). The reaction mixture was stirred at RT for 21 h, and volatiles were removed in vacuo. The residue was resuspended in Ac<sub>2</sub>O (6.0 mL, 64 mmol, 23 equiv) and pyridine (12.0 mL, 149 mmol, 54 equiv) was added. The resulting mixture was stirred at RT for 22 h, and volatiles were removed in vacuo. Crude material was purified by silica gel chromatography (hexanes/EtOAc, 0 → 40%) to afford triacetate <b>S4</b> (735 mg, 1.44 mmol, 52% over 2 steps) as a 1:1.7 mixture of α and β anomers, as a light-yellow oil. <i>R</i><sub><i>f</i></sub> = 0.37 (α-anomer)/0.46 (β-anomer) (hexanes/EtOAc, 50:50). FTIR (thin-film), ν<sub>max</sub> (cm<sup>–1</sup>): 3290, 2956, 1748, 1724, 1553, 1438, 1373, 1219, 1178, 1112, 1049, 1013. <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>) δ α-anomer: 6.92 (d, <i>J</i> = 7.8 Hz, 1H), 6.36 (d, <i>J</i> = 4.6 Hz, 1H), 5.22 (dd, <i>J</i> = 6.8, 4.6 Hz, 1H), 5.05 (dd, <i>J</i> = 6.8, 3.7 Hz, 1H), 4.63 (td, <i>J</i> = 7.6, 5.1 Hz, 1H), 4.43 (dt, <i>J</i> = 10.2, 3.6 Hz, 1H), 3.80 (s, 3H), 2.62 (dqd, <i>J</i> = 13.8, 4.6, 2.4 Hz, 1H), 2.13 (d, <i>J</i> = 2.4 Hz, 1H), 2.12 (s, 3H), 2.08 (s, 3H), 2.08–2.04 (m, 1H), 2.07 (s, 3H), 2.01 (dddd, <i>J</i> = 18.5, 9.1, 5.3, 2.9 Hz, 1H), 1.84 (ddd, <i>J</i> = 14.1, 10.8, 3.5 Hz, 1H), 1.65–1.59 (m, 1H), 1.59–1.53 (m, 1H), 1.40 (dddd, <i>J</i> = 13.1, 11.0, 9.5, 4.8 Hz, 1H). β-anomer: 7.02 (d, <i>J</i> = 8.1 Hz, 1H), 6.10 (s, 1H), 5.31 (dd, <i>J</i> = 4.9, 0.9 Hz, 1H), 5.16 (dd, <i>J</i> = 7.2, 4.7 Hz, 1H), 4.64 (td, <i>J</i> = 8.2, 4.6 Hz, 1H), 4.33 (ddd, <i>J</i> = 10.1, 7.1, 2.7 Hz, 1H), 3.81 (s, 3H), 2.66 (ttd, <i>J</i> = 9.6, 4.9, 2.4 Hz, 1H), 2.13 (d, <i>J</i> = 2.4 Hz, 1H), 2.12 (s, 3H), 2.10 (s, 3H), 2.08–2.04 (m, 1H), 2.06 (s, 3H), 2.05–1.97 (m, 1H), 1.84 (ddd, <i>J</i> = 13.1, 9.9, 2.8 Hz, 1H), 1.69–1.61 (m, 1H), 1.60–1.55 (m, 1H), 1.47 (dddd, <i>J</i> = 13.2, 11.3, 8.9, 4.5 Hz, 1H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) δ α-anomer: 171.1, 169.9, 169.5, 169.3, 156.9 (q, <sup>2</sup><i>J</i><sub>C–F</sub> = 38 Hz), 115.7 (q, <sup>1</sup><i>J</i><sub>C–F</sub> = 288 Hz), 93.7, 84.6, 81.2, 72.5, 71.5, 69.7, 52.9, 52.3, 38.9, 30.4, 29.1, 28.4, 21.0, 20.5, 20.2. β-Anomer: 171.1, 170.1, 169.7, 169.4, 156.9 (q, <sup>2</sup><i>J</i><sub>C–F</sub> = 38 Hz), 115.6 (q, <sup>1</sup><i>J</i><sub>C–F</sub> = 288 Hz), 98.3, 84.9, 79.8, 74.3, 73.8, 71.3, 52.9, 52.3, 39.6, 30.6, 29.5, 28.0, 21.0, 20.6, 20.2. <sup>19</sup>F NMR (471 MHz, CDCl<sub>3</sub>, BTF IStd): δ −75.8. HRMS (ESI+): calcd for [C<sub>21</sub>H<sub>26</sub>F<sub>3</sub>N<sub>1</sub>O<sub>10</sub>Na]<sup>+</sup> 532.1401, measured 532.1407, Δ 1.2 ppm.</div></div><div id="sec4_6_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i66" class="anchor-spacer"></div><h4 class="article-section__title" id="_i66"> Methyl 6-((3a<i>R</i>,4<i>R</i>,6<i>R</i>,6a<i>R</i>)-6-Methoxy-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)hexanoate (<b>S6</b>).<a onclick="showRef(event, 'ref68'); return false;" href="javascript:void(0);" class="ref ref68">(68)</a></h4><div class="NLM_p last">To a 3-necked flask charged with pyridine (10 mL) were added NaI (1.49 g, 10.0 mmol, 1.0 equiv) and FeCl<sub>3</sub>·6H<sub>2</sub>O (1.35 g, 5.00 mmol, 50 mol %) sequentially, both in one portion. Zinc powder (3.92 g, 60.0 mmol, 6.0 equiv) was added portionwise to the reaction mixture (<b>Caution: exothermic</b>), monitoring the temperature so that it remained below 40 °C. Upon complete addition of the zinc powder, the resulting mixture was allowed to cool gradually to RT and a solution of riboside <b>S5</b><a onclick="showRef(event, 'ref69'); return false;" href="javascript:void(0);" class="ref ref69">(69)</a> (3.14 g, 10.0 mmol) in methyl acrylate (6.0 mL) was added over 10 min via syringe. The reaction mixture was stirred at RT for 2 h, before the addition of toluene (100 mL) and filtration over Celite. Volatiles were removed in vacuo to give methyl ester <b>S6</b> (2.52 g, 9.25 mmol, 93%), which was of suitable purity to be used in the next step without further purification. <i>R</i><sub><i>f</i></sub> = 0.42 (cyclohexane/EtOAc, 67:33). FTIR (thin film), ν<sub>max</sub> (cm<sup>–1</sup>): 2989, 2940, 1735, 1437, 1372, 1270, 1239, 1208, 1194, 1159, 1090, 1057, 1011. <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 4.94 (s, 1H), 4.59 (d, <i>J</i> = 5.9 Hz, 1H), 4.52 (dd, <i>J</i> = 5.9, 1.1 Hz, 1H), 4.13 (ddd, <i>J</i> = 8.9, 6.4, 1.1 Hz, 1H), 3.67 (s, 3H), 3.34 (s, 2H), 2.35 (td, <i>J</i> = 7.4, 1.4 Hz, 2H), 1.86–1.76 (m, 1H), 1.71 (ddtd, <i>J</i> = 13.2, 10.1, 7.3, 5.7 Hz, 1H), 1.62 (dddd, <i>J</i> = 14.1, 10.1, 8.9, 5.1 Hz, 1H), 1.53 (ddt, <i>J</i> = 13.4, 10.4, 6.0 Hz, 1H), 1.47 (s, 3H), 1.31 (s, 3H). <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): δ 173.5, 112.1, 109.5, 86.7, 85.5, 84.1, 54.9, 51.4, 34.4, 33.5, 26.4, 24.9, 21.7. HRMS (ESI+): calcd for [C<sub>13</sub>H<sub>22</sub>O<sub>6</sub>Na]<sup>+</sup> 297.1309, measured 297.1307, Δ 0.7 ppm.</div></div><div id="sec4_6_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i67" class="anchor-spacer"></div><h4 class="article-section__title" id="_i67"> 4-((3a<i>R</i>,4<i>R</i>,6<i>R</i>,6a<i>R</i>)-6-Methoxy-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)butanal (<b>S7</b>)</h4><div class="NLM_p last">To an oversized flask (250 mL) containing a solution of methyl ester <b>S6</b> (2.15 g, 7.83 mmol) in toluene (20 mL) at −85 °C (<i>i</i>-PrOH/liquid nitrogen bath) was added <i>i</i>-Bu<sub>2</sub>AlH (1.0 M in hexanes, 8.0 mL, 8.0 mmol, 1.02 equiv) over 40 min via syringe pump, down the side-wall of the flask. Upon complete addition of <i>i</i>-Bu<sub>2</sub>AlH, the reaction mixture was stirred at −85 °C for 1 h, before the slow addition of MeOH (4 mL) over 20 min down the side-wall of the flask, followed by a potassium sodium tartrate satd aq solution (40 mL) and EtOAc (50 mL). The reaction mixture was removed from the cooling bath and allowed to warm gradually to RT. Water (10 mL) and brine (10 mL) were added, and the biphasic mixture was stirred for another 10 min. The layers were separated, and the aqueous phase was extracted with EtOAc (3 × 50 mL). The combined organic extracts were washed with brine (200 mL), dried over MgSO<sub>4</sub>, filtered, and concentrated to give aldehyde <b>S7</b> (1.80 g, 7.37 mmol, 94%) as a light-yellow oil, which was of suitable purity to be used in the next step without further purification. <i>R</i><sub><i>f</i></sub> = 0.31 (cyclohexane/EtOAc, 67:33). FTIR (thin film), ν<sub>max</sub> (cm<sup>–1</sup>): 2988, 2938, 2832, 2723, 1724, 1457, 1412, 1381, 1372, 1273, 1240, 1209, 1193, 1160, 1087, 1058, 1018. <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 9.73 (t, <i>J</i> = 1.6 Hz, 1H), 4.89 (s, 1H), 4.55 (d, <i>J</i> = 6.0 Hz, 1H), 4.48 (dd, <i>J</i> = 6.0, 1.1 Hz, 1H), 4.09 (ddd, <i>J</i> = 9.0, 6.3, 1.1 Hz, 1H), 3.30 (s, 3H), 2.45 (dddd, <i>J</i> = 8.7, 7.1, 3.4, 1.6 Hz, 2H), 1.81–1.71 (m, 1H), 1.72–1.63 (m, 1H), 1.59 (dtd, <i>J</i> = 13.6, 9.2, 5.2 Hz, 1H), 1.49 (ddt, <i>J</i> = 13.5, 10.1, 6.1 Hz, 1H), 1.43 (s, 3H), 1.27 (s, 3H). <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): δ 202.0, 112.3, 109.6, 86.8, 85.5, 84.1, 55.1, 43.4, 34.4, 26.6, 25.0, 18.9.</div></div><div id="sec4_6_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i68" class="anchor-spacer"></div><h4 class="article-section__title" id="_i68"> Methyl (<i>Z</i>)-2-(((Benzyloxy)carbonyl)amino)-6-((3a<i>R</i>,4<i>R</i>,6<i>R</i>,6a<i>R</i>)-6-methoxy-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)hex-2-enoate (<b>S8</b>)</h4><div class="NLM_p last">To a stirred suspension of (±)-Z-α-Phosphonoglycine trimethyl ester (<b>S1</b>) (343 mg, 1.04 mmol, 1.2 equiv) in THF (3.0 mL) at −78 °C was added 1,1,3,3-tetramethylguanidine (0.12 mL, 0.95 mmol, 1.1 equiv). The resulting mixture was stirred at −78 °C for 15 min, before the addition of a solution of aldehyde <b>S7</b> (211 mg, 0.864 mmol) in THF (2.0 mL). The reaction mixture was stirred at −78 °C for 30 min and then placed in an ice/water bath and stirred for 1 h. The reaction was quenched by the addition of a 10% AcOH aq solution (2 mL), and water (10 mL) was added. The layers were separated, and the aqueous phase was extracted with CH<sub>2</sub>Cl<sub>2</sub> (3 × 15 mL). The combined organic extracts were washed with water (10 mL) and brine (10 mL), dried over MgSO<sub>4</sub>, filtered, and concentrated. <sup>1</sup>H NMR analysis of the crude revealed a 15:1 mixture of <i>Z</i> and <i>E</i> isomers. Crude material was purified by silica gel chromatography (2,2,4-trimethylpentane/EtOAc, 0 → 50%). <i>Z</i> and <i>E</i> isomers separated, and only fractions containing <i>Z</i> isomers were collected to afford enamide <b>S8</b> (353 mg, 0.785 mmol, 91%) as a clear viscous oil. <i>R</i><sub><i>f</i></sub> = 0.44 (cyclohexane/EtOAc, 50:50). FTIR (thin film), ν<sub>max</sub> (cm<sup>–1</sup>): 3316, 2988, 2942, 1716, 1657, 1500, 1455, 1437, 1381, 1373, 1270, 1213, 1160, 1105, 1089, 1047. <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 7.41–7.30 (m, 5H), 6.61 (t, <i>J</i> = 7.2 Hz, 1H), 5.14 (s, 2H), 4.93 (s, 1H), 4.58 (d, <i>J</i> = 5.9 Hz, 1H), 4.49 (d, <i>J</i> = 5.9 Hz, 1H), 4.12 (dd, <i>J</i> = 8.3, 6.3 Hz, 1H), 3.75 (s, 3H), 3.33 (s, 3H), 2.31–2.18 (m, 2H), 1.68–1.48 (m, 4H), 1.47 (s, 3H), 1.30 (s, 3H).; <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): δ 165.1, 137.5, 136.1, 128.6, 128.3, 128.2, 112.3, 109.6, 86.9, 85.6, 84.2, 67.4, 55.0, 52.5, 34.7, 28.1, 26.6, 25.1. HRMS (ESI+): calcd for [C<sub>23</sub>H<sub>32</sub>NO<sub>8</sub>]<sup>+</sup> 450.2122, measured 450.2124, Δ 0.3 ppm.</div></div><div id="sec4_6_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i69" class="anchor-spacer"></div><h4 class="article-section__title" id="_i69"> Methyl (<i>S</i>)-2-(((Benzyloxy)carbonyl)amino)-6-((3a<i>R</i>,4<i>R</i>,6<i>R</i>,6a<i>R</i>)-6-methoxy-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)hexanoate (<b>S9</b>)</h4><div class="NLM_p last">To a flask charged with enamide <b>S8</b> (179 mg, 0.398 mmol, exclusively <i>Z</i> isomer) was added MeOH (5 mL) at RT under a nitrogen atmosphere. The contents were stirred until full dissolution was noted. Separately, [Rh(nbd)<sub>2</sub>]BF<sub>4</sub> (49 mg, 0.12 mmol, 30 mol %) and (<i>S</i>,<i>S</i>)-Me-DUPHOS (43 mg, 0.14 mmol, 35 mol %) were weighed out into the same vial in a glovebox. The vial was removed from the glovebox, placed under a nitrogen atmosphere, and MeOH (10 mL) was added. The flask headspace was evacuated and backfilled with hydrogen (3×). The methanolic solution of enamide <b>S8</b> was transferred to the vial containing the catalyst (2 mL of MeOH used for vial rinse), and the contents were stirred vigorously at RT under a hydrogen atmosphere (balloon) for 1 h. Volatiles were removed in vacuo, and crude material was purified by silica gel chromatography (2,2,4-trimethylpentane/EtOAc, 0 → 50%) to afford protected amino acid <b>S9</b> (172 mg, 0.381 mmol, 96%) as a clear viscous oil and as a single diastereomer. <i>R</i><sub><i>f</i></sub> = 0.32 (cyclohexane/EtOAc, 67:33). FTIR (thin film), ν<sub>max</sub> (cm<sup>–1</sup>): 3338, 2988, 2939, 2862, 1720, 1523, 1455, 1439, 1381, 1373, 1348, 1259, 1239, 1209, 1161, 1106, 1087, 1056, 1026. <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 7.38–7.28 (m, 5H), 5.31 (d, <i>J</i> = 8.4 Hz, 1H), 5.09 (s, 2H), 4.92 (s, 1H), 4.57 (d, <i>J</i> = 5.9 Hz, 1H), 4.48 (d, <i>J</i> = 6.0 Hz, 1H), 4.37 (td, <i>J</i> = 8.0, 5.1 Hz, 1H), 4.10 (dd, <i>J</i> = 8.9, 5.8 Hz, 1H), 3.73 (s, 3H), 3.31 (s, 3H), 1.88–1.79 (m, 1H), 1.64 (app h, <i>J</i> = 8.1 Hz, 1H), 1.58 (app q, <i>J</i> = 8.8 Hz, 1H), 1.49–1.43 (m, 2H), 1.46 (s, 3H), 1.40–1.32 (m, 3H), 1.30 (s, 3H). <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): δ 173.0, 155.9, 136.3, 128.5, 128.2, 128.1, 112.2, 109.4, 87.0, 85.6, 84.1, 67.0, 54.9, 53.8, 52.3, 34.8, 32.6, 26.5, 25.8, 25.0. HRMS (ESI+): calcd for [C<sub>23</sub>H<sub>34</sub>NO<sub>8</sub>]<sup>+</sup> 452.2280, measured 452.2285, Δ 1.1 ppm.</div></div><div id="sec4_6_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i70" class="anchor-spacer"></div><h4 class="article-section__title" id="_i70"> (3<i>R</i>,4<i>R</i>,5<i>R</i>)-5-((<i>S</i>)-5-(((Benzyloxy)carbonyl)amino)-6-methoxy-6-oxohexyl)tetrahydrofuran-2,3,4-triyl Triacetate (<b>S10</b>)</h4><div class="NLM_p last">To a vial charged with acetonide <b>S9</b> (171 mg, 0.378 mmol) was added a 4:1 mixture of acetic acid and water (7.5 mL) at RT. The resulting mixture was stirred at 85 °C for 8 h. Volatiles were removed in vacuo, and the residue was dried by azeotropic distillation with benzene (1 × 10 mL). Crude material was used directly in the next step without further purification. To a solution of the crude triol in CH<sub>2</sub>Cl<sub>2</sub> (4.0 mL) at RT were added pyridine (0.50 mL, 6.2 mmol, 16 equiv), Ac<sub>2</sub>O (0.50 mL, 5.3 mmol, 14 equiv), and DMAP (5 mg, 0.04 mmol, 0.11 equiv) sequentially. The resulting mixture was stirred at RT for 2 h, and the volatiles were removed in vacuo. Crude material was purified by silica gel chromatography (2,2,4-trimethylpentane/EtOAc, 0 → 100%) to afford triacetate <b>S10</b> (152 mg, 0.290 mmol, 77% over 2 steps) as a ca. 1:1 mixture of α and β anomers as a colorless oil. <i>R</i><sub><i>f</i></sub> = 0.28 (α-anomer)/0.37 (β-anomer) (EtOAc). FTIR (thin film), ν<sub>max</sub> (cm<sup>–1</sup>): 3359, 2950, 1743, 1721, 1523, 1455, 1437, 1371, 1216, 1110, 1047, 1010. <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>) δ α-anomer: 7.38–7.27 (m, 5H), 6.34 (d, <i>J</i> = 4.6 Hz, 1H), 5.29 (d, <i>J</i> = 8.2 Hz, 1H), 5.18 (dd, <i>J</i> = 6.8, 4.6 Hz, 1H), 5.09 (s, 2H), 5.01 (dd, <i>J</i> = 6.8, 3.6 Hz, 1H), 4.35 (td, <i>J</i> = 7.8, 5.1 Hz, 1H), 4.20–4.15 (m, 1H), 3.73 (s, 3H), 2.10 (s, 3H), 2.09 (s, 3H), 2.05 (s, 3H), 1.83–1.77 (m, 1H), 1.69–1.54 (m, 3H), 1.47–1.42 (m, 1H), 1.39–1.28 (m, 3H). β-Anomer: 7.38–7.28 (m, 5H), 6.10 (d, <i>J</i> = 1.1 Hz, 1H), 5.31 (d, <i>J</i> = 8.1 Hz, 1H), 5.29 (dd, <i>J</i> = 4.8, 1.2 Hz, 1H), 5.14 (dd, <i>J</i> = 6.9, 4.8 Hz, 1H), 5.10 (s, 2H), 4.36 (td, <i>J</i> = 7.8, 5.0 Hz, 1H), 4.12 (td, <i>J</i> = 7.4, 5.2 Hz, 1H), 3.73 (s, 3H), 2.10 (s, 3H), 2.07 (s, 3H), 2.05 (s, 3H), 1.85–1.78 (m, 1H), 1.68–1.55 (m, 2H), 1.51–1.42 (m, 1H), 1.40–1.29 (m, 4H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) δ mixture of α and β-anomers: 172.9, 170.2, 169.8, 169.4, 155.9, 136.3, 128.6, 128.2, 128.2, 98.3, 94.0, 83.3, 81.4, 74.6, 73.6, 72.3, 69.9, 67.0, 53.8, 52.4, 33.8, 33.1, 32.5, 25.0, 25.0, 24.9, 24.8, 21.1, 21.1, 20.7, 20.6, 20.6, 20.4. HRMS (ESI+): calcd for [C<sub>25</sub>H<sub>34</sub>NO<sub>11</sub>]<sup>+</sup> 524.2126, measured 524.2134, Δ 1.5 ppm.</div></div><div id="sec4_6_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i71" class="anchor-spacer"></div><h4 class="article-section__title" id="_i71"> (2<i>R</i>,3<i>R</i>,4<i>R</i>,5<i>R</i>)-2-(6-Benzamido-9<i>H</i>-purin-9-yl)-5-((<i>S</i>)-5-(((benzyloxy)carbonyl)amino)-6-methoxy-6-oxohexyl)tetrahydrofuran-3,4-diyl Diacetate (<b>S11</b>)</h4><div class="NLM_p last">To a suspension of <i>N</i><sup>6</sup>-benzoyladenine (149 mg, 0.624 mmol, 2.15 equiv) in propionitrile (dried over 4 Å MS, 6.0 mL) at RT, was added <i>N</i>,<i>O</i>-bis(trimethylsilyl)acetamide (0.21 mL, 0.87 mmol, 3.00 equiv). The resulting mixture was stirred at 95 °C for 10 min, upon which complete solubilization of <i>N</i><sup>6</sup>-benzoyladenine was observed. To a separate flask charged with triacetate <b>S10</b> (152 mg, 0.290 mmol) and 2,6-di-<i>tert</i>-butyl-4-methylpyridinium triflate (21 mg, 58 μmol, 20 mol %) was added propionitrile (6.0 mL). The resulting mixture was stirred at RT for 5 min until a clear solution was obtained. This clear solution was added to the solution of silylated <i>N</i><sup>6</sup>-benzoyladenine. The reaction mixture was stirred at 95 °C for 2.5 h, cooled to RT, and the volatiles were removed in vacuo. Crude material was purified by silica gel chromatography (2,2,4-trimethylpentane/EtOAc, 0 → 100% then EtOAc/<i>i</i>-PrOH, 0 → 15%) to afford nucleoside <b>S11</b> (148 mg, 0.211 mmol, 73%) as a white amorphous solid. <i>R</i><sub><i>f</i></sub> = 0.28 (EtOAc). FTIR (thin film), ν<sub>max</sub> (cm<sup>–1</sup>): 3330, 2950, 1744, 1703, 1609, 1581, 1510, 1487, 1454, 1408, 1373, 1330, 1239, 1214, 1177, 1095, 1047, 1028. <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 9.28 (br s, 1H), 8.74 (s, 1H), 8.11 (s, 1H), 8.01–7.97 (m, 2H), 7.59–7.53 (m, 1H), 7.49–7.44 (m, 2H), 7.33–7.24 (m, 5H), 6.12 (d, <i>J</i> = 5.3 Hz, 1H), 5.94 (t, <i>J</i> = 5.5 Hz, 1H), 5.45 (d, <i>J</i> = 3.5 Hz, 1H), 5.44 (t, <i>J</i> = 5.3 Hz, 1H), 5.07 (s, 2H), 4.34 (td, <i>J</i> = 8.0, 5.1 Hz, 1H), 4.15 (dt, <i>J</i> = 8.8, 4.9 Hz, 1H), 3.69 (s, 3H), 2.11 (s, 3H), 2.04 (s, 3H), 1.85–1.70 (m, 3H), 1.65–1.57 (m, 1H), 1.51–1.42 (m, 1H), 1.41–1.30 (m, 3H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 172.9, 169.7, 169.5, 165.0, 155.9, 152.7, 151.8, 149.9, 141.9, 136.3, 133.6, 132.8, 128.8, 128.5, 128.2, 128.1, 128.0, 123.9, 86.5, 82.4, 73.3, 73.1, 67.0, 53.7, 52.4, 32.9, 32.5, 24.9, 24.9, 20.6, 20.4. HRMS (ESI+): calcd for [C<sub>35</sub>H<sub>38</sub>N<sub>6</sub>O<sub>10</sub>Na]<sup>+</sup> 725.2542, measured 725.2573, Δ 4.3 ppm.</div></div><div id="sec4_6_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i72" class="anchor-spacer"></div><h4 class="article-section__title" id="_i72"> (2<i>R</i>,3<i>R</i>,4<i>R</i>,5<i>R</i>)-2-(6-Benzamido-9<i>H</i>-purin-9-yl)-5-((2<i>S</i>,5<i>S</i>)-6-methoxy-6-oxo-2-(phenylethynyl)-5-(2,2,2-trifluoroacetamido)hexyl)tetrahydrofuran-3,4-diyl Diacetate (<b>S12</b>)</h4><div class="NLM_p last">To a vial charged with nucleoside <b>8</b> (82 mg, 0.12 mmol) were added iodobenzene (49 mg, 0.24 mmol, 2.0 equiv), CuI (5 mg, 0.02 mmol, 20 mol %), and Pd(PPh<sub>3</sub>)<sub>4</sub> (7 mg, 6 μmol, 5 mol %). The vial headspace was purged with nitrogen for 5 min and a 5:1:1 mixture of toluene, DMF, and <i>i</i>-Pr<sub>2</sub>NEt (degassed by sparging with nitrogen for 15 min, 3.5 mL) was added at RT. The resulting mixture was stirred at 70 °C for 3 h. The solution was allowed to cool to RT, and volatiles were removed in vacuo. Crude material was purified by silica gel chromatography (EtOAc/<i>i</i>-PrOH, 0 → 5%) to afford <b>S12</b> (76 mg, 99 μmol, 83%) as a white amorphous solid. <i>R</i><sub><i>f</i></sub> = 0.48 (EtOAc). FTIR (thin-film), ν<sub>max</sub> (cm<sup>–1</sup>): 3297, 3082, 2954, 1745, 1717, 1610, 1582, 1509, 1489, 1455, 1373, 1328, 1239, 1212, 1176, 1159, 1096, 1072, 1047, 1029. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): δ 8.95 (s, 1H), 8.81 (s, 1H), 8.13 (s, 1H), 8.01 (d, <i>J</i> = 7.6 Hz, 2H), 7.61 (t, <i>J</i> = 7.2 Hz, 1H), 7.52 (t, <i>J</i> = 7.5 Hz, 2H), 7.40 (ddt, <i>J</i> = 5.3, 2.7, 1.4 Hz, 2H), 7.31 (tt, <i>J</i> = 3.8, 2.2 Hz, 3H), 6.92 (d, <i>J</i> = 5.8 Hz, 1H), 6.14 (d, <i>J</i> = 5.4 Hz, 1H), 6.09 (t, <i>J</i> = 5.4 Hz, 1H), 5.59 (t, <i>J</i> = 4.9 Hz, 1H) 4.66 (td, <i>J</i> = 7.7, 5.0 Hz, 1H), 4.53 (ddd, <i>J</i> = 10.8, 4.5, 2.9 Hz, 1H), 3.78 (s, 3H), 2.82 (ddt, <i>J</i> = 10.9, 9.1, 4.5 Hz, 1H), 2.16 (s, 3H), 2.19–2.09 (m, 1H), 2.07 (s, 3H), 2.06–1.92 (m, 2H), 1.70–1.60 (m, 2H), 1.58–1.46 (m, 1H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 171.2, 169.8, 169.6, 164.8, 157.0 (q, <sup>2</sup><i>J</i><sub>C–F</sub> = 38 Hz), 152.9, 151.7, 149.8, 142.4, 133.5, 132.9, 131.8, 128.4, 128.2, 128.0, 124.0, 123.2, 115.7 (q, <sup>1</sup><i>J</i><sub>C–F</sub> = 288 Hz), 90.0, 87.1, 83.8, 81.0, 77.3, 73.7, 73.0, 53.1, 52.4, 38.8, 31.1, 30.0, 28.8, 20.7, 20.5. <sup>19</sup>F NMR (471 MHz, CDCl<sub>3</sub>, BTF IStd): δ −76.7. HRMS (ESI+): calcd for [C<sub>37</sub>H<sub>36</sub>F<sub>3</sub>N<sub>6</sub>O<sub>9</sub>]<sup>+</sup> 765.2490, measured 765.2504, Δ 1.8 ppm.</div></div><div id="sec4_6_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i73" class="anchor-spacer"></div><h4 class="article-section__title" id="_i73"> (<i>S</i>)-3-(((3a<i>R</i>,4<i>R</i>,6<i>R</i>,6a<i>R</i>)-6-Methoxy-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)methyl)-5-(triisopropylsilyl)pent-4-ynal (<b>S13</b>)</h4><div class="NLM_p last">A 50 mL flask was charged with [Rh(OAc)(C<sub>2</sub>H<sub>4</sub>)<sub>2</sub>]<sub>2</sub> (60 mg, 0.14 mmol, 2.5 mol %) and (<i>R</i>)-DTBMSegPhos (389 mg, 0.329 mmol, 6.0 mol %). The flask headspace was purged with nitrogen for 10 min followed by the addition of MeOH (14 mL). The resulting orange suspension was stirred vigorously at RT until a clear solution was obtained (15 min), followed by the sequential addition of enal <b>4</b> (1.33 g, 5.49 mmol) in MeOH (4 mL) and TIPS-acetylene (2.46 mL, 11.0 mmol, 2.0 equiv). The resulting red mixture was stirred at 40 °C for 15 h, and volatiles were removed in vacuo. Crude material was purified by silica gel chromatography (hexanes/acetone, 0 → 30%) to afford aldehyde <b>S13</b> (2.00 g, 4.71 mmol, 86%) as a single diastereomer, as a light-yellow oil. <i>R</i><sub><i>f</i></sub> = 0.31 (hexanes/acetone, 85:15). FTIR (thin film), ν<sub>max</sub> (cm<sup>–1</sup>): 2942, 2866, 2169, 1729, 1464, 1382, 1211, 1160, 1108. <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 9.83 (dd, <i>J</i> = 2.1, 1.7 Hz, 1H), 4.95 (s, 1H), 4.60 (s, 2H), 4.39 (app t, <i>J</i> = 7.7 Hz, 1H), 3.34 (s, 3H), 3.09 (app tdd, <i>J</i> = 7.7, 6.5, 6.0 Hz, 1H), 2.65 (ddd, <i>J</i> = 16.6, 6.0, 1.7 Hz, 1H), 2.60 (ddd, <i>J</i> = 16.6, 7.7, 2.1 Hz, 1H), 1.89 (app dt, <i>J</i> = 13.6, 7.7 Hz, 1H), 1.73 (ddd, <i>J</i> = 13.6, 7.7, 6.5 Hz, 1H), 1.47 (s, 3H), 1.31 (s, 3H), 1.06–1.03 (m, 21H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 200.8, 112.6, 109.6, 108.8, 85.6, 84.8, 83.9, 83.8, 55.2, 47.8, 39.9, 26.7, 25.3, 24.2, 18.7, 11.3. HRMS (ESI+): calcd for [C<sub>23</sub>H<sub>40</sub>O<sub>5</sub>Si<sub>1</sub>Na]<sup>+</sup> 447.2537, measured 447.2523, Δ 3.2 ppm.</div></div><div id="sec4_6_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i74" class="anchor-spacer"></div><h4 class="article-section__title" id="_i74"> Methyl (<i>R</i>,<i>Z</i>)-5-(((3a<i>R</i>,4<i>R</i>,6<i>R</i>,6a<i>R</i>)-6-Methoxy-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)methyl)-2-(2,2,2-trifluoroacetamido)-7-(triisopropylsilyl)hept-2-en-6-ynoate (<b>S14</b>)</h4><div class="NLM_p last">To a suspension of NaH (95%, 45 mg, 1.8 mmol, 1.5 equiv) in THF (2.5 mL) at 0 °C was added a solution of phosphonate <b>S2</b> (517 mg, 1.76 mmol, 1.5 equiv) in THF (2.5 mL) dropwise. The resulting cloudy mixture was stirred at 0 °C for 30 min, and a solution of aldehyde <b>S13</b> (499 mg, 1.18 mmol) in THF (2.5 mL) was added in one portion. The yellow reaction mixture was stirred at 40 °C for 1.5 h and cooled to 0 °C, before the addition of an aqueous pH 7 buffer (6 mL) and Et<sub>2</sub>O (6 mL). The layers were separated, and the aqueous phase was extracted with Et<sub>2</sub>O (3 × 6 mL). The combined organic extracts were dried over MgSO<sub>4</sub>, filtered, and concentrated. Crude material was purified by silica gel chromatography (hexanes/EtOAc, 1 → 35%) to afford olefin <b>S14</b> (651 mg, 1.10 mmol, 94%) as a 6:1 mixture of <i>Z</i> and <i>E</i> isomers as a colorless oil. <i>R</i><sub><i>f</i></sub> = 0.28 (hexanes/EtOAc, 80:20). FTIR (thin film), ν<sub>max</sub> (cm<sup>–1</sup>): 3305, 2943, 2865, 2168, 1721, 1665, 1536, 1463, 1439, 1382, 1274, 1210, 1160, 1107, 1061. <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 7.78 (s, 1H), 7.09 (dd, <i>J</i> = 7.5, 7.2 Hz, 1H), 4.95 (s, 1H), 4.60 (d, <i>J</i> = 6.0 Hz, 1H), 4.55 (dd, <i>J</i> = 6.0, 0.9 Hz, 1H), 4.37 (app td, <i>J</i> = 7.5, 0.9 Hz, 1H), 3.81 (s, 3H), 3.34 (s, 3H), 2.81 (dddd, <i>J</i> = 9.0, 7.5, 7.0, 4.6 Hz, 1H), 2.45 (ddd, <i>J</i> = 15.7, 7.2, 4.6 Hz, 1H), 2.29 (ddd, <i>J</i> = 15.7, 9.0, 7.5 Hz, 1H), 1.89 (app dt, <i>J</i> = 13.7, 7.5 Hz, 1H), 1.70 (ddd, <i>J</i> = 13.7, 7.5, 7.0 Hz, 1H), 1.46 (s, 3H), 1.30 (s, 3H), 1.06–1.04 (m, 21H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>): δ 163.8, 155.2 (q, <sup>2</sup><i>J</i><sub>C–F</sub> = 38 Hz), 137.7, 123.7, 115.8 (q, <sup>1</sup><i>J</i><sub>C–F</sub> = 288 Hz), 112.6, 109.5, 109.3, 85.6, 84.8, 83.9, 55.1, 52.9, 40.0, 33.8, 28.8, 26.6, 25.2, 18.7, 11.3. <sup>19</sup>F NMR (471 MHz, CDCl<sub>3</sub>, BTF IStd): δ −76.3, −77.2. HRMS (ESI+): calcd for [C<sub>28</sub>H<sub>44</sub>F<sub>3</sub>N<sub>1</sub>O<sub>7</sub>Si<sub>1</sub>Na]<sup>+</sup> 614.2731, measured 614.2735, Δ 0.7 ppm.</div></div><div id="sec4_6_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i75" class="anchor-spacer"></div><h4 class="article-section__title" id="_i75"> Methyl (2<i>S</i>,5<i>R</i>)-5-(((3a<i>R</i>,4<i>R</i>,6<i>R</i>,6a<i>R</i>)-6-Methoxy-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)methyl)-2-(2,2,2-trifluoroacetamido)-7-(triisopropylsilyl)hept-6-ynoate (<b>S15</b>)</h4><div class="NLM_p last">To a flask charged with enamide <b>S14</b> (6:1 mixture of <i>Z</i> and <i>E</i> isomers, 651 mg, 1.10 mmol) was added MeOH (32 mL) under a nitrogen atmosphere. The contents were stirred until full solubilization was observed. In a glovebox, a separate vial was charged with (<i>S</i>,<i>S</i>)-MeBPE-Rh (31 mg, 55 μmol, 5 mol %). The vial was removed from the glovebox, placed under a nitrogen atmosphere, and MeOH (4 mL) was added. The resulting orange solution was transferred to the flask containing the methanolic solution of enamide <b>S14</b> via syringe. The flask headspace was evacuated and refilled with hydrogen (3×), and the reaction mixture was stirred vigorously under a hydrogen atmosphere (balloon) for 1 h. Volatiles were removed in vacuo. Crude material was purified by silica gel chromatography (2,2,4-trimethylpentane/EtOAc, 1 → 50%) to afford protected amino acid <b>S15</b> (627 mg, 1.06 mmol, 96%) as a single diastereomer, as a colorless oil. <i>R</i><sub><i>f</i></sub> = 0.30 (hexanes/EtOAc, 85:15). FTIR (thin film), ν<sub>max</sub> (cm<sup>–1</sup>): 3319, 2942, 2865, 2166, 1749, 1720, 1549, 1461, 1382, 1208, 1159, 1106, 1092, 1061. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): δ 6.90 (d, <i>J</i> = 7.5 Hz, 1H), 4.93 (s, 1H), 4.64 (app td, <i>J</i> = 7.5, 5.2 Hz, 1H), 4.59 (d, <i>J</i> = 5.9 Hz, 1H), 4.53 (dd, <i>J</i> = 5.9, 0.8 Hz, 1H), 4.36 (app td, <i>J</i> = 7.8, 0.8 Hz, 1H), 3.79 (s, 3H), 3.33 (s, 3H), 2.58–2.51 (m, 1H), 2.29 (dddd, <i>J</i> = 13.9, 11.4, 5.2, 4.7 Hz, 1H), 1.90–1.81 (m, 2H), 1.63 (ddd, <i>J</i> = 13.6, 7.8, 6.7 Hz, 1H), 1.59–1.44 (m, 2H), 1.46 (s, 3H), 1.30 (s, 3H), 1.06–1.02 (m, 21H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 171.4, 156.9 (q, <sup>2</sup><i>J</i><sub>C–F</sub> = 38 Hz), 115.7 (q, <sup>1</sup><i>J</i><sub>C–F</sub> = 288 Hz), 112.6, 109.6, 109.5, 85.6, 84.9, 83.9, 83.5, 55.1, 53.1, 52.7, 40.3, 30.4, 30.1, 29.3, 26.6, 25.2, 18.7, 11.3. <sup>19</sup>F NMR (471 MHz, CDCl<sub>3</sub>, BTF IStd): δ −76.8. HRMS (ESI+): calcd for [C<sub>28</sub>H<sub>46</sub>F<sub>3</sub>N<sub>1</sub>O<sub>7</sub>Si<sub>1</sub>Na]<sup>+</sup> 616.2888, measured 616.2890, Δ 0.3 ppm.</div></div><div id="sec4_6_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i76" class="anchor-spacer"></div><h4 class="article-section__title" id="_i76"> Methyl (2<i>S</i>,5<i>R</i>)-5-(((3a<i>R</i>,4<i>R</i>,6<i>R</i>,6a<i>R</i>)-6-Methoxy-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)methyl)-2-(2,2,2-trifluoroacetamido)hept-6-ynoate (<b>S16</b>)</h4><div class="NLM_p last">To a solution of TIPS alkyne <b>S15</b> (108 mg, 0.182 mmol) in DMF (4 mL) at RT was added a solution of TASF (0.15 g, 0.55 mmol, 3.0 equiv) in DMF (3.3 mL). The resulting mixture was stirred at 60 °C for 2.5 h, and volatiles were removed in vacuo. The residue was partitioned between Et<sub>2</sub>O (7 mL) and a 1:1 mixture of brine and 5% LiCl aq solution (7 mL). The layers were separated, and the aqueous phase was extracted with Et<sub>2</sub>O (3 × 7 mL). The combined organic extracts were dried over MgSO<sub>4</sub>, filtered, and concentrated. Crude material was purified by silica gel chromatography (2,2,4-trimethylpentane/EtOAc, 5 → 50%) to afford alkyne <b>S16</b> (80 mg, 0.18 mmol, quant) as a colorless oil. <i>R</i><sub><i>f</i></sub> = 0.26 (hexanes/EtOAc, 70:30). FTIR (thin film), ν<sub>max</sub> (cm<sup>–1</sup>): 3288, 2936, 1747, 1719, 1552, 1456, 1440, 1383, 1374, 1209, 1160, 1106, 1091, 1059. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): δ 6.96 (d, <i>J</i> = 7.5 Hz, 1H), 4.93 (s, 1H), 4.63 (ddd, <i>J</i> = 7.5, 7.0, 5.8 Hz, 1H), 4.59 (d, <i>J</i> = 5.9 Hz, 1H), 4.51 (dd, <i>J</i> = 5.9, 1.0 Hz, 1H), 4.33 (app td, <i>J</i> = 7.5, 1.0 Hz, 1H), 3.80 (s, 3H), 3.33 (s, 3H), 2.50 (ddddd, <i>J</i> = 9.5, 7.3, 6.8, 4.6, 2.4 Hz, 1H), 2.22 (dddd, <i>J</i> = 14.0, 11.2, 5.8, 4.8 Hz, 1H), 2.13 (d, <i>J</i> = 2.4 Hz, 1H), 1.90–1.82 (m, 2H), 1.64 (ddd, <i>J</i> = 13.7, 7.5, 6.8 Hz, 1H), 1.60–1.44 (m, 2H), 1.47 (s, 3H), 1.30 (s, 3H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 171.3, 156.9 (q, <sup>2</sup><i>J</i><sub>C–F</sub> = 38 Hz), 115.7 (q, <sup>1</sup><i>J</i><sub>C–F</sub> = 288 Hz), 112.6, 109.8, 85.6, 85.4, 84.6, 84.0, 71.2, 55.2, 53.1, 52.6, 39.8, 29.8, 29.6, 28.2, 26.6, 25.1. <sup>19</sup>F NMR (471 MHz, CDCl<sub>3</sub>, BTF IStd): δ −76.8. HRMS (ESI+): calcd for [C<sub>19</sub>H<sub>26</sub>F<sub>3</sub>N<sub>1</sub>O<sub>7</sub>Na]<sup>+</sup> 460.1554, measured 460.1562, Δ 1.9 ppm.</div></div><div id="sec4_6_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i77" class="anchor-spacer"></div><h4 class="article-section__title" id="_i77"> (3<i>R</i>,4<i>R</i>,5<i>R</i>)-5-((2<i>R</i>,5<i>S</i>)-2-Ethynyl-6-methoxy-6-oxo-5-(2,2,2-trifluoroacetamido)hexyl)tetrahydrofuran-2,3,4-triyl triacetate (<b>S17</b>)</h4><div class="NLM_p last">To a solution of acetonide <b>S16</b> (80 mg, 0.18 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (0.8 mL) at 0 °C was added a 4:1 mixture of TFA and water (3.6 mL). The resulting mixture was stirred at RT for 5.5 h. Volatiles were removed in vacuo, and the residue was dried by azeotropic distillation with benzene (2 × 0.8 mL). Crude material was used directly in the next step without further purification. To a solution of the crude triol in CH<sub>2</sub>Cl<sub>2</sub> (3.6 mL) at RT were added pyridine (0.15 mL, 1.8 mmol, 10 equiv), Ac<sub>2</sub>O (0.17 mL, 1.8 mmol, 10 equiv), and DMAP (16 mg, 0.13 mmol, 0.70 equiv) sequentially. The resulting mixture was stirred at RT for 15 h and volatiles were removed in vacuo. Crude material was purified by silica gel chromatography (hexanes/EtOAc, 5 → 70%) to afford triacetate <b>S17</b> (60 mg, 0.12 mmol, 64% over 2 steps) as a 2:1 mixture of diastereomers, as a colorless oil. <i>R</i><sub><i>f</i></sub> = 0.35/0.43 (hexanes/EtOAc, 50:50). FTIR (thin film), ν<sub>max</sub> (cm<sup>–1</sup>): 3290, 2930, 1746, 1722, 1554, 1438, 1372, 1214, 1111, 1011. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): δ 7.12 (d, <i>J</i> = 8.1 Hz, 2H), 6.98 (d, <i>J</i> = 8.0 Hz, 1H), 6.35 (d, <i>J</i> = 4.5 Hz, 1H), 6.06 (d, <i>J</i> = 1.0 Hz, 2H), 5.30 (dd, <i>J</i> = 4.8, 1.0 Hz, 2H), 5.25–5.20 (m, 3H), 5.17 (dd, <i>J</i> = 6.9, 3.5 Hz, 1H), 4.66–4.59 (m, 3H), 4.38–4.32 (m, 3H), 3.80 (s, 3H), 3.79 (s, 6H), 2.60–2.50 (m, 3H), 2.25–2.15 (m, 2H), 2.14 (d, <i>J</i> = 2.5 Hz, 1H), 2.13 (d, <i>J</i> = 2.5 Hz, 2H), 2.12 (s, 6H), 2.11 (s, 3H), 2.11 (s, 3H), 2.09 (s, 6H), 2.07 (s, 6H), 2.06 (s, 3H), 1.93–1.83 (m, 4H), 1.83–1.74 (m, 6H), 1.61–1.47 (m, 6H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 171.2, 170.2, 170.0, 169.9, 169.7, 169.6, 169.5, 157.1 (q, <sup>2</sup><i>J</i><sub>C–F</sub> = 38 Hz), 156.9 (q, <sup>2</sup><i>J</i><sub>C–F</sub> = 37 Hz), 115.8 (q, <sup>1</sup><i>J</i><sub>C–F</sub> = 287 Hz), 115.7 (q, <sup>1</sup><i>J</i><sub>C–F</sub> = 288 Hz), 98.5, 93.9, 85.2, 85.0, 81.3, 79.2, 74.4, 73.7, 72.0, 71.6, 71.2, 69.9, 53.1, 53.1, 52.6, 52.5, 38.4, 37.9, 29.7, 29.6, 29.5, 27.8, 27.3, 21.2, 21.1, 20.8, 20.6, 20.6, 20.4. <sup>19</sup>F NMR (471 MHz, CDCl<sub>3</sub>, BTF IStd): δ −76.75, −76.76. HRMS (ESI+): calcd for [C<sub>21</sub>H<sub>26</sub>F<sub>3</sub>N<sub>1</sub>O<sub>10</sub>Na]<sup>+</sup> 532.1401, measured 532.1416, Δ 2.8 ppm.</div></div><div id="sec4_6_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i78" class="anchor-spacer"></div><h4 class="article-section__title" id="_i78"> (3<i>R</i>,4<i>R</i>,5<i>R</i>)-2-(6-Benzamido-9<i>H</i>-purin-9-yl)-5-((2<i>R</i>,5<i>S</i>)-2-ethynyl-6-methoxy-6-oxo-5-(2,2,2-trifluoroacetamido)hexyl)tetrahydrofuran-3,4-diyl Diacetate (<b>S18</b>)</h4><div class="NLM_p last">To a suspension of <i>N</i><sup>6</sup>-benzoyladenine (113 mg, 0.471 mmol, 4.0 equiv) in propionitrile (dried over 4 Å MS, 2.4 mL) at RT was added <i>N</i>,<i>O</i>-bis(trimethylsilyl)acetamide (0.14 mL, 0.59 mmol, 5.0 equiv). The resulting mixture was stirred at 80 °C for 10 min, upon which complete solubilization of <i>N</i><sup>6</sup>-benzoyladenine was observed. To a separate flask charged with triacetate <b>S17</b> (dried by azeotropic distillation with benzene (4×), 60 mg, 0.12 mmol) and 2,6-di-<i>tert</i>-butyl-4-methylpyridinium triflate (21 mg, 59 μmol, 50 mol %) was added propionitrile (2.4 mL). The resulting mixture was stirred at RT for 5 min until a clear solution was obtained, and the solution of silylated <i>N</i><sup>6</sup>-benzoyladenine was added. The reaction mixture was stirred at 95 °C for 5.5 h, cooled to RT, and the volatiles were removed in vacuo. Crude material was purified by silica gel chromatography (2,2,4-trimethylpentane/EtOAc, 30 → 100%) to afford nucleoside <b>S18</b> (53 mg, 77 μmol, 65%) as a colorless oil. <i>R</i><sub><i>f</i></sub> = 0.33 (hexanes/EtOAc, 20:80). FTIR (thin film), ν<sub>max</sub> (cm<sup>–1</sup>): 3283, 3084, 2929, 1745, 1720, 1612, 1583, 1511, 1487, 1456, 1374, 1244, 1218, 1075, 1029. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): δ 9.07 (s, 1H), 8.79 (s, 1H), 8.11 (s, 1H), 8.04–7.98 (m, 2H), 7.63–7.57 (m, 1H), 7.54–7.49 (m, 2H), 7.04 (br d, <i>J</i> = 7.6 Hz, 1H), 6.12 (d, <i>J</i> = 5.5 Hz, 1H), 6.04 (app t, <i>J</i> = 5.5 Hz, 1H), 5.58 (dd, <i>J</i> = 5.5, 4.5 Hz, 1H), 4.60 (ddd, <i>J</i> = 7.6, 7.0, 5.4 Hz, 1H), 4.38 (ddd, <i>J</i> = 7.5, 5.8, 4.5 Hz, 1H), 3.76 (s, 3H), 2.57 (dddd, <i>J</i> = 14.0, 8.2, 6.4, 2.4 Hz, 1H), 2.23–2.12 (m, 2H), 2.15 (s, 3H), 2.15 (d, <i>J</i> = 2.4 Hz, 1H), 2.06 (s, 3H), 2.00–1.93 (m, 1H), 1.86–1.77 (m, 1H), 1.54–1.48 (m, 2H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): δ 171.2, 169.8, 169.6, 164.8, 156.9 (q, <sup>2</sup><i>J</i><sub>C–F</sub> = 38 Hz), 153.0, 151.8, 149.9, 142.0, 133.5, 133.0, 129.0, 128.0, 124.0, 115.7 (q, <sup>1</sup><i>J</i><sub>C–F</sub> = 288 Hz), 86.7, 85.1, 80.6, 73.3, 72.8, 71.5, 53.2, 52.5, 37.7, 29.6, 29.6, 27.5, 20.7, 20.5. <sup>19</sup>F NMR (471 MHz, CDCl<sub>3</sub>, BTF IStd): δ −76.7. HRMS (ESI+): calcd for [C<sub>31</sub>H<sub>32</sub>F<sub>3</sub>N<sub>6</sub>O<sub>9</sub>]<sup>+</sup> 689.2177, measured 689.2178, Δ 0.1 ppm.</div></div><div id="sec4_6_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i79" class="anchor-spacer"></div><h4 class="article-section__title" id="_i79"> (3<i>R</i>,4<i>R</i>,5<i>R</i>)-2-(6-Benzamido-9<i>H</i>-purin-9-yl)-5-((2<i>R</i>,5<i>S</i>)-2-((3-carbamoylphenyl)ethynyl)-6-methoxy-6-oxo-5-(2,2,2-trifluoroacetamido)hexyl)tetrahydrofuran-3,4-diyl diacetate (<b>S19</b>)</h4><div class="NLM_p last">To a solution of nucleoside <b>S18</b> (53 mg, 77 μmol) and 3-iodobenzamide (48 mg, 0.19 mmol, 2.5 equiv) in a 5:1:1 mixture of toluene, DMF, and <i>i</i>-Pr<sub>2</sub>NEt (degassed by sparging with nitrogen for 15 min, 3.9 mL) at RT were introduced CuI (4 mg, 0.02 mmol, 25 mol %) and Pd(PPh<sub>3</sub>)<sub>4</sub> (4 mg, 4 μmol, 5 mol %) rapidly. The resulting mixture was stirred at 60 °C for 2 h. The solution was allowed to cool to RT, and volatiles were removed in vacuo. Crude material was purified by silica gel chromatography (EtOAc/<i>i</i>-PrOH, 0 → 5%) to afford <b>S19</b> (50 mg, 62 μmol, 80%) as a colorless oil. <i>R</i><sub><i>f</i></sub> = 0.43 (EtOAc/<i>i</i>-PrOH, 95:5). FTIR (thin film), ν<sub>max</sub> (cm<sup>–1</sup>): 3284, 2932, 1746, 1716, 1668, 1611, 1580, 1455, 1375, 1242, 1215, 1183, 1160, 1075. <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 9.23 (s, 1H), 8.75 (s, 1H), 8.18 (s, 1H), 8.04–7.98 (m, 2H), 7.84–7.82 (m, 1H), 7.80 (ddd, <i>J</i> = 7.7, 1.9, 1.3 Hz, 1H), 7.61–7.56 (m, 1H), 7.52–7.47 (m, 2H), 7.46 (app dt, <i>J</i> = 7.7, 1.3 Hz, 1H), 7.36 (br d, <i>J</i> = 7.3 Hz, 1H), 7.34 (app t, <i>J</i> = 7.7 Hz, 1H), 6.91 (br s, 1H), 6.16 (d, <i>J</i> = 5.7 Hz, 1H), 6.13 (dd, <i>J</i> = 5.7, 5.4 Hz, 1H), 5.99 (br s, 1H), 5.76 (dd, <i>J</i> = 5.4, 4.3 Hz, 1H), 4.68–4.63 (m, 1H), 4.43 (ddd, <i>J</i> = 7.0, 5.5, 4.3 Hz, 1H), 3.76 (s, 3H), 2.87–2.81 (m, 1H), 2.33–2.26 (m, 1H), 2.26 (ddd, <i>J</i> = 14.2, 8.4, 7.0 Hz, 1H), 2.14 (s, 3H), 2.06 (s, 3H), 2.05–2.01 (m, 1H), 1.97–1.89 (m, 1H), 1.61–1.55 (m, 2H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 171.3, 170.0, 169.6, 168.8, 165.0, 157.1 (q, <sup>2</sup><i>J</i><sub>C–F</sub> = 38 Hz), 152.9, 151.9, 149.9, 142.2, 134.5, 133.6, 133.5, 133.0, 130.6, 129.0, 128.8, 128.1, 127.7, 124.1, 123.3, 115.7 (q, <sup>1</sup><i>J</i><sub>C–F</sub> = 288 Hz), 91.7, 86.7, 82.9, 81.1, 73.2, 72.7, 532, 52.7, 37.6, 30.2, 29.6, 28.0, 20.8, 20.5. <sup>19</sup>F NMR (471 MHz, CDCl<sub>3</sub>, BTF IStd): δ −76.7. HRMS (ESI+): calcd for [C<sub>38</sub>H<sub>37</sub>F<sub>3</sub>N<sub>7</sub>O<sub>10</sub>]<sup>+</sup> 808.2549, measured 808.2541, Δ 0.9 ppm.</div></div><div id="sec4_6_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i80" class="anchor-spacer"></div><h4 class="article-section__title" id="_i80"> 3-((3a<i>R</i>,4<i>R</i>,6<i>R</i>,6a<i>R</i>)-6-Methoxy-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)propan-1-ol (<b>S21</b>).<a onclick="showRef(event, 'ref71'); return false;" href="javascript:void(0);" class="ref ref71">(71)</a></h4><div class="NLM_p last">To a solution of known α,β-unsaturated methyl ester <b>S20</b><a onclick="showRef(event, 'ref72'); return false;" href="javascript:void(0);" class="ref ref72">(72)</a> (1.22 g, 4.72 mmol) in a 10:1 mixture of Et<sub>2</sub>O and MeOH (22 mL) at 0 °C, was added a solution of LiBH<sub>4</sub> (2.0 M in THF, 9.45 mL, 18.9 mmol, 4.0 equiv) dropwise. The resulting mixture was allowed to warm gradually to RT and stirred for 17 h before the addition of a 1:1 mixture of a NH<sub>4</sub>Cl satd aq solution and water (35 mL). The layers were separated, and the aqueous phase was extracted with Et<sub>2</sub>O (3 × 35 mL). The combined organic extracts were washed with brine (100 mL), dried over MgSO<sub>4</sub>, filtered, and concentrated to give alcohol <b>S21</b> (1.10 g, 4.72 mmol, quant) as a colorless oil. <i>R</i><sub><i>f</i></sub> = 0.24 (hexanes/EtOAc, 50:50). FTIR (thin film), ν<sub>max</sub> (cm<sup>–1</sup>): 3422, 2989, 2937, 1449, 1373, 1210, 1160, 1088, 1054, 1015. <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 4.95 (s, 1H), 4.61 (d, <i>J</i> = 5.9 Hz, 1H), 4.54 (dd, <i>J</i> = 5.9, 1.0 Hz, 1H), 4.20–4.16 (m, 1H), 3.70–3.67 (m, 2H), 3.36 (s, 3H), 1.76–1.59 (m, 4H), 1.48 (s, 3H), 1.32 (s, 3H). <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): δ 112.4, 109.7, 87.2, 85.5, 84.3, 62.3, 55.1, 31.6, 29.6, 26.6, 25.1. HRMS (ESI+): calcd for [C<sub>11</sub>H<sub>20</sub>O<sub>5</sub>Na]<sup>+</sup> 255.1203, measured 255.1196, Δ 2.6 ppm.</div></div><div id="sec4_6_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i81" class="anchor-spacer"></div><h4 class="article-section__title" id="_i81"> 3-((3a<i>R</i>,4<i>R</i>,6<i>R</i>,6a<i>R</i>)-6-Methoxy-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)propanal (<b>S22</b>).<a onclick="showRef(event, 'ref73'); return false;" href="javascript:void(0);" class="ref ref73">(73)</a></h4><div class="NLM_p last">To a solution of alcohol <b>S21</b> (400 mg, 1.72 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (11.5 mL) at RT was added DMP (876 mg, 2.07 mmol, 1.2 equiv) in one portion. The resulting mixture was stirred at RT for 1 h before the addition of a 1:1 mixture of a Na<sub>2</sub>S<sub>2</sub>O<sub>3</sub> satd aq solution and a NaHCO<sub>3</sub> satd aq solution (10 mL). The biphasic mixture was stirred vigorously at RT for 10 min, and the layers were separated. The aqueous phase was extracted with Et<sub>2</sub>O (3 × 10 mL), and the combined organic extracts were washed with brine (40 mL), dried over MgSO<sub>4</sub>, filtered, and concentrated. Crude material was purified by silica gel chromatography (pentane/EtOAc, 2 → 50%) to afford aldehyde <b>S22</b> (324 mg, 1.41 mmol, 82%) as a colorless oil. <i>R</i><sub><i>f</i></sub> = 0.32 (hexanes/EtOAc, 75:25). FTIR (thin film), ν<sub>max</sub> (cm<sup>–1</sup>): 2988, 2936, 2834, 2724, 1721, 1373, 1209, 1160, 1090, 1058. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): δ 9.79 (t, <i>J</i> = 1.3 Hz, 1H), 4.93 (s, 1H), 4.59 (d, <i>J</i> = 5.9 Hz, 1H), 4.52 (dd, <i>J</i> = 5.9, 1.1 Hz, 1H), 4.14 (ddd, <i>J</i> = 9.2, 6.5, 1.1 Hz, 1H), 3.32 (s, 3H), 2.62 (dddd, <i>J</i> = 18.1, 7.9, 6.6, 1.3 Hz, 1H), 2.57 (dddd, <i>J</i> = 18.1, 8.2, 6.9, 1.3 Hz, 1H), 1.89–1.82 (m, 2H), 1.45 (s, 3H), 1.30 (s, 3H). <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): δ 201.4, 112.5, 109.9, 86.3, 85.6, 84.1, 55.3, 40.8, 27.5, 26.6, 25.1. HRMS (ESI+): calcd for [C<sub>11</sub>H<sub>18</sub>O<sub>5</sub>Na]<sup>+</sup> 253.1046, measured 253.1036, Δ 4.0 ppm.</div></div><div id="sec4_6_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i82" class="anchor-spacer"></div><h4 class="article-section__title" id="_i82"> (3a<i>R</i>,4<i>R</i>,6<i>R</i>,6a<i>R</i>)-4-(But-3-yn-1-yl)-6-methoxy-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxole (<b>S23</b>).<a onclick="showRef(event, 'ref74'); return false;" href="javascript:void(0);" class="ref ref74">(74)</a></h4><div class="NLM_p last">To a solution of aldehyde <b>S22</b> (324 mg, 1.41 mmol) in MeOH (6.0 mL) at RT was added K<sub>2</sub>CO<sub>3</sub> (583 mg, 4.22 mmol, 3.0 equiv), followed by a solution of dimethyl-1-diazo-2-oxopropyl phosphonate (338 mg, 1.76 mmol, 1.25 equiv) in MeOH (3.0 mL) under vigorous stirring. The resulting yellow suspension was stirred at RT for 3 h. The reaction was quenched by the addition of a NH<sub>4</sub>Cl satd aq solution (15 mL), and Et<sub>2</sub>O (25 mL) was added. The layers were separated and the aqueous phase was extracted with Et<sub>2</sub>O (3 × 25 mL). The combined organic extracts were dried over MgSO<sub>4</sub>, filtered, and concentrated. Crude material was purified by silica gel chromatography (hexane/Et<sub>2</sub>O, 2 → 50%) to afford alkyne <b>S23</b> (254 mg, 1.12 mmol, 80%) as a colorless oil. <i>R</i><sub><i>f</i></sub> = 0.50 (hexanes/EtOAc, 75:25). <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 4.95 (s, 1H), 4.60 (d, <i>J</i> = 5.9 Hz, 1H), 4.56 (dd, <i>J</i> = 5.9, 1.0 Hz, 1H), 4.30 (ddd, <i>J</i> = 9.8, 5.8, 1.0 Hz, 1H), 3.34 (s, 3H), 2.36 (dddd, <i>J</i> = 16.8, 7.8, 5.9, 2.6 Hz, 1H), 2.31 (dddd, <i>J</i> = 16.8, 7.8, 7.3, 2.6 Hz, 1H), 1.98 (t, <i>J</i> = 2.6 Hz, 1H), 1.80 (dddd, <i>J</i> = 13.4, 9.8, 7.3, 5.9 Hz, 1H), 1.74 (app dtd, <i>J</i> = 13.4, 7.8, 5.8 Hz, 1H), 1.48 (s, 3H), 1.31 (s, 3H). <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): δ 112.5, 109.8, 85.9, 85.7, 84.1, 83.3, 69.1, 55.3, 33.9, 26.6, 25.1, 15.6. HRMS (ESI+): calcd for [C<sub>12</sub>H<sub>18</sub>O<sub>4</sub>Na]<sup>+</sup> 249.1097, measured 249.1118, Δ 8.4 ppm.</div></div><div id="sec4_6_21" class="NLM_sec NLM_sec_level_3"><div id="ac_i83" class="anchor-spacer"></div><h4 class="article-section__title" id="_i83"> (3<i>R</i>,4<i>R</i>,5<i>R</i>)-5-(But-3-yn-1-yl)tetrahydrofuran-2,3,4-triyl Triacetate (<b>S24</b>)</h4><div class="NLM_p last">To a vial charged with acetonide <b>S23</b> (50 mg, 0.22 mmol) was added a 4:1 mixture of acetic acid and water (2.5 mL) at RT. The resulting mixture was stirred at 80 °C for 5 h. Volatiles were removed in vacuo, and the residue was dried by azeotropic distillation with benzene (2 × 1.0 mL). Crude material was used directly in the next step without further purification. To a solution of the crude triol in CH<sub>2</sub>Cl<sub>2</sub> (2.5 mL) at RT were added pyridine (0.18 mL, 2.2 mmol, 10 equiv), Ac<sub>2</sub>O (0.21 mL, 2.2 mmol, 10 equiv), and DMAP (19 mg, 0.15 mmol, 0.70 equiv) sequentially. The resulting mixture was stirred at RT for 2 h, and volatiles were removed in vacuo. Crude material was purified by silica gel chromatography (2,2,4-trimethylpentane/EtOAc, 2 → 60%) to afford triacetate <b>S24</b> (58 mg, 0.19 mmol, 88% over 2 steps) as a 1:1 mixture of diastereomers as a colorless oil; <i>R</i><sub><i>f</i></sub> = 0.42/0.50 (hexanes/EtOAc, 50:50). FTIR (thin film), ν<sub>max</sub> (cm<sup>–1</sup>): 3285, 2958, 2920, 2850, 2117, 1742, 1433, 1371, 1214, 1107, 1010. <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 6.36 (d, <i>J</i> = 4.6 Hz, 1H), 6.12 (d, <i>J</i> = 1.0 Hz, 1H), 5.30 (dd, <i>J</i> = 4.8, 1.0 Hz, 1H), 5.21 (dd, <i>J</i> = 6.8, 4.6 Hz, 1H), 5.21 (dd, <i>J</i> = 7.0, 4.8 Hz, 1H), 5.10 (dd, <i>J</i> = 6.8, 3.6 Hz, 1H), 4.33 (ddd, <i>J</i> = 8.5, 4.9, 3.6 Hz, 1H), 4.27 (ddd, <i>J</i> = 8.5, 7.0, 4.6 Hz, 1H), 2.39–2.25 (m, 4H), 2.11 (s, 6H), 2.10 (s, 3H), 2.07 (s, 3H), 2.06 (s, 3H), 2.05 (s, 3H), 1.97 (t, <i>J</i> = 2.8 Hz, 1H), 1.96 (t, <i>J</i> = 2.6 Hz, 1H), 1.95–1.86 (m, 2H), 1.86–1.78 (m, 2H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 170.2, 170.0, 169.8, 169.6, 169.5, 169.2, 98.3, 93.9, 83.0, 82.8, 82.1, 80.3, 74.7, 73.5, 72.1, 70.0, 69.4, 69.2, 33.2, 32.3, 21.2, 21.2, 20.8, 20.7, 20.6, 20.4, 14.9, 14.8. HRMS (ESI+): calcd for [C<sub>14</sub>H<sub>18</sub>O<sub>7</sub>Na]<sup>+</sup> 321.0945, measured 321.0952, Δ 2.3 ppm.</div></div><div id="sec4_6_22" class="NLM_sec NLM_sec_level_3"><div id="ac_i84" class="anchor-spacer"></div><h4 class="article-section__title" id="_i84"> (2<i>R</i>,3<i>R</i>,4<i>R</i>,5<i>R</i>)-2-(6-Benzamido-9<i>H</i>-purin-9-yl)-5-(but-3-yn-1-yl)tetrahydrofuran-3,4-diyl Diacetate (<b>S25</b>)</h4><div class="NLM_p last">To a suspension of <i>N</i><sup>6</sup>-benzoyladenine (160 mg, 0.670 mmol, 4.0 equiv) in propionitrile (dried over 4 Å MS, 5.0 mL) at RT was added <i>N</i>,<i>O</i>-bis(trimethylsilyl)acetamide (0.20 mL, 0.84 mmol, 5.0 equiv). The resulting mixture was stirred at 80 °C for 10 min, upon which complete solubilization of <i>N</i><sup>6</sup>-benzoyladenine was observed. To a separate flask charged with triacetate <b>S24</b> (dried by azeotropic distillation with benzene (4×), 50 mg, 0.17 mmol) and 2,6-di-<i>tert</i>-butyl-4-methylpyridinium triflate (30 mg, 84 μmol, 50 mol %) was added propionitrile (5.0 mL). The resulting mixture was stirred at RT for 5 min, until a clear solution was obtained, and the solution of silylated <i>N</i><sup>6</sup>-benzoyladenine was added. The reaction mixture was stirred at 95 °C for 17 h, cooled to RT, and the volatiles were removed in vacuo. Crude material was purified by silica gel chromatography (2,2,4-trimethylpentane/EtOAc, 0 → 100%) to afford nucleoside <b>S25</b> (73 mg, 0.15 mmol, 91%) as a colorless oil. <i>R</i><sub><i>f</i></sub> = 0.31 (EtOAc). FTIR (thin film), ν<sub>max</sub> (cm<sup>–1</sup>): 3291, 3091, 2934, 1747, 1699, 1609, 1582, 1510, 1486, 1455, 1373, 1239, 1217, 1099, 1073, 1053. <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 9.12 (s, 1H), 8.78 (s, 1H), 8.09 (s, 1H), 8.04–7.98 (m, 2H), 7.62–7.57 (m, 1H), 7.54–7.48 (m, 2H), 6.13 (d, <i>J</i> = 5.4 Hz, 1H), 6.04 (dd, <i>J</i> = 5.6, 5.4 Hz, 1H), 5.57 (dd, <i>J</i> = 5.6, 4.7 Hz, 1H), 4.37 (ddd, <i>J</i> = 9.2, 4.7, 4.3 Hz, 1H), 2.38 (dddd, <i>J</i> = 17.0, 7.0, 5.6, 2.6 Hz, 1H), 2.31 (dddd, <i>J</i> = 17.0, 8.5, 6.8, 2.6 Hz, 1H), 2.15 (s, 3H), 2.16–2.09 (m, 1H), 2.06 (s, 3H), 2.06–2.00 (m, 1H), 1.99 (t, <i>J</i> = 2.6 Hz, 1H). <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): δ 169.8, 169.6, 164.8, 153.0, 151.7, 149.9, 142.1, 133.6, 133.0, 129.0, 128.0, 124.0, 86.9, 82.8, 81.3, 73.3, 73.1, 69.6, 31.8, 20.7, 20.5, 14.8. HRMS (ESI+): calcd for [C<sub>24</sub>H<sub>24</sub>N<sub>5</sub>O<sub>6</sub>]<sup>+</sup> 478.1721, measured 478.1728, Δ 1.5 ppm.</div></div><div id="sec4_6_23" class="NLM_sec NLM_sec_level_3"><div id="ac_i85" class="anchor-spacer"></div><h4 class="article-section__title" id="_i85"> (2<i>R</i>,3<i>R</i>,4<i>R</i>,5<i>R</i>)-2-(6-Benzamido-9<i>H</i>-purin-9-yl)-5-(4-(3-carbamoylphenyl)but-3-yn-1-yl)tetrahydrofuran-3,4-diyl Diacetate (<b>S26</b>)</h4><div class="NLM_p last">To a vial charged with alkyne <b>S25</b> (72 mg, 0.15 mmol) were added 3-iodobenzamide (75 mg, 0.30 mmol, 2.0 equiv), CuI (7 mg, 38 μmol, 25 mol %), and Pd(PPh<sub>3</sub>)<sub>4</sub> (9 mg, 8 μmol, 5 mol %). The vial headspace was purged with nitrogen for 5 min and a 5:1:1 mixture of toluene, DMF, and <i>i</i>-Pr<sub>2</sub>NEt (degassed by sparging with nitrogen for 15 min, 6.0 mL) was added at RT. The resulting mixture was stirred at 80 °C for 1 h. The solution was allowed to cool to RT, and volatiles were removed in vacuo. Crude material was purified by silica gel chromatography (EtOAc/<i>i</i>-PrOH, 0 → 20%) to afford <b>S26</b> (58 mg, 97 μmol, 64%) as a colorless oil. <i>R</i><sub><i>f</i></sub> = 0.24 (EtOAc/<i>i</i>-PrOH, 90:10). FTIR (thin film), ν<sub>max</sub> (cm<sup>–1</sup>): 3187, 2931, 1747, 1664, 1609, 1578, 1511, 1486, 1454, 1373, 1239, 1215, 1097, 1049. <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 9.30 (s, 1H), 8.76 (s, 1H), 8.17 (s, 1H), 8.04–7.97 (m, 2H), 7.83 (app t, <i>J</i> = 1.5 Hz, 1H), 7.74 (app dt, <i>J</i> = 7.8, 1.5 Hz, 1H), 7.59–7.55 (m, 1H), 7.50–7.45 (m, 3H), 7.33 (app t, <i>J</i> = 7.8 Hz, 1H), 6.64 (br s, 1H), 6.17 (d, <i>J</i> = 5.6 Hz, 1H), 6.13 (br s, 1H), 6.11 (app t, <i>J</i> = 5.6 Hz, 1H), 5.71 (dd, <i>J</i> = 5.6, 4.3 Hz, 1H), 4.40 (ddd, <i>J</i> = 8.1, 4.9, 4.3 Hz, 1H), 2.61 (app dt, <i>J</i> = 17.2, 6.3 Hz, 1H), 2.53 (ddd, <i>J</i> = 17.2, 8.4, 6.1 Hz, 1H), 2.23–2.17 (m, 1H), 2.14 (s, 3H), 2.16–2.09 (m, 1H), 2.05 (s, 3H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>): δ 170.0, 169.7, 168.9, 165.0, 152.9, 151.9, 149.9, 142.2, 134.7, 133.7, 133.5, 133.0, 130.6, 128.9, 128.7, 128.1, 127.1, 124.1, 124.0, 89.7, 86.7, 81.8, 80.8, 73.3, 73.0, 31.7, 20.8, 20.5, 15.6. HRMS (ESI+): calcd for [C<sub>31</sub>H<sub>29</sub>N<sub>6</sub>O<sub>7</sub>]<sup>+</sup> 597.2092, measured 597.2097, Δ 0.8 ppm.</div></div><div id="sec4_6_24" class="NLM_sec NLM_sec_level_3"><div id="ac_i86" class="anchor-spacer"></div><h4 class="article-section__title" id="_i86"> ((1<i>R</i>,2<i>R</i>)-2-((3a<i>R</i>,4<i>R</i>,6<i>R</i>,6a<i>R</i>)-6-Methoxy-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)cyclopropyl)methanol (<b>S28</b>)</h4><div class="NLM_p last">The following protocol was adapted from an early publication reporting the asymmetric cyclopropanation of allylic alcohols with dioxaborolane ligands.<a onclick="showRef(event, 'ref75'); return false;" href="javascript:void(0);" class="ref ref75">(75)</a> To a stirred solution of diethylzinc in hexanes (1.0 M in hexanes, 4.46 mL, 4.46 mmol, 2.23 equiv) was added CH<sub>2</sub>Cl<sub>2</sub> (8.0 mL). The mixture was cooled to 0 °C, and diiodomethane (0.72 mL, 8.9 mmol, 4.45 equiv) was added dropwise over 5 min. The reaction mixture was stirred at 0 °C for 15 min (white precipitate was formed), and a precooled (0 °C) solution of butylboronic acid <i>N</i>,<i>N</i>,<i>N</i>′,<i>N</i>′-tetramethyl-<span class="smallcaps smallerCapital">d</span>-tartaric acid diamide ester (360 mg, 2.28 mmol, 1.14 equiv) and allylic alcohol <b>S27</b><a onclick="showRef(event, 'ref72'); return false;" href="javascript:void(0);" class="ref ref72">(72)</a> (457 mg, 2.00 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (12 mL) was rapidly added via syringe. The resulting mixture was stirred at RT for 1 h, cooled to 0 °C, and a NH<sub>4</sub>Cl satd aq solution (15 mL) was added slowly. The layers were separated, and the aqueous phase was extracted with CH<sub>2</sub>Cl<sub>2</sub> (3 × 30 mL). The combined organic extracts were passed through a phase separator (Biotage Isolute), dried over MgSO<sub>4</sub>, filtered, and concentrated. Crude material was purified by silica gel chromatography (2,2,4-trimethylpentane/EtOAc, 0 → 100%) to afford cyclopropyl alcohol <b>S28</b> (382 mg, 1.56 mmol, 78%) as a 12:1 mixture of cyclopropyl diastereomers (major = desired), as a colorless oil. <i>R</i><sub><i>f</i></sub> = 0.24 (cyclohexane/EtOAc, 50:50). FTIR (thin film), ν<sub>max</sub> (cm<sup>–1</sup>): 3446, 2991, 2935, 2251, 2041, 1458, 1382, 1373, 1274, 1210, 1194, 1159, 1105, 1089, 1053, 1030. <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 4.95 (s, 1H), 4.74 (d, <i>J</i> = 5.9 Hz, 1H), 4.66 (d, <i>J</i> = 5.9 Hz, 1H), 3.51–3.43 (m, 3H), 3.38 (s, 3H), 1.45 (s, 3H), 1.31 (s, 3H), 1.21 (dddd, <i>J</i> = 12.0, 8.4, 6.9, 5.1 Hz, 1H), 0.94 (ddt, <i>J</i> = 10.4, 9.0, 4.7 Hz, 1H), 0.53 (ddt, <i>J</i> = 17.2, 8.4, 5.1 Hz, 2H). <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): δ 112.1, 109.1, 91.1, 85.3, 84.0, 65.5, 54.4, 26.3, 24.8, 20.8, 20.4, 8.1. HRMS (ESI+): calcd for [C<sub>12</sub>H<sub>20</sub>O<sub>5</sub>Na]<sup>+</sup> 267.1203, measured 267.1202, Δ 0.2 ppm.</div></div><div id="sec4_6_25" class="NLM_sec NLM_sec_level_3"><div id="ac_i87" class="anchor-spacer"></div><h4 class="article-section__title" id="_i87"> (1<i>R</i>,2<i>R</i>)-2-((3a<i>R</i>,4<i>R</i>,6<i>R</i>,6a<i>R</i>)-6-Methoxy-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)cyclopropane-1-carbaldehyde (<b>S29</b>)</h4><div class="NLM_p last">To a solution of cyclopropyl alcohol <b>S28</b> (245 mg, 1.00 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (20 mL) open to atmosphere at RT was added DMP (510 mg, 1.20 mmol, 1.2 equiv) portionwise over 5 min. The reaction mixture was stirred for 2 h and was diluted with Et<sub>2</sub>O (10 mL). The suspension was filtered, and the volatiles were removed in vacuo. The crude residue was purified by silica gel chromatography (2,2,4-trimethylpentane/EtOAc, 0 → 50%) to yield aldehyde <b>S29</b> (182 mg, 0.751 mmol, 75%) as a clear oil. In our hands, aldehyde <b>S29</b> was best prepared, purified, and used immediately in the next reaction. <i>R</i><sub><i>f</i></sub> = 0.39 (cyclohexane/EtOAc, 67:33). <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 9.16 (d, <i>J</i> = 4.9 Hz, 1H), 4.96 (s, 1H), 4.75 (d, <i>J</i> = 5.9 Hz, 1H), 4.66 (d, <i>J</i> = 5.8 Hz, 1H), 3.55 (d, <i>J</i> = 10.0 Hz, 1H), 3.28 (s, 3H), 2.02 (app dtd, <i>J</i> = 7.8, 4.9, 3.9 Hz, 1H), 1.79 (app tdd, <i>J</i> = 10.1, 8.9, 6.4, 4.0 Hz, 1H), 1.45 (s, 3H), 1.33 (dd, <i>J</i> = 8.9, 4.9 Hz, 1H), 1.31 (s, 3H), 1.07 (ddd, <i>J</i> = 8.3, 6.4, 4.9 Hz, 1H). <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): δ 199.9, 112.5, 109.2, 89.3, 85.4, 84.0, 54.6, 28.9, 26.4, 26.1, 25.0, 12.7.</div></div><div id="sec4_6_26" class="NLM_sec NLM_sec_level_3"><div id="ac_i88" class="anchor-spacer"></div><h4 class="article-section__title" id="_i88"> (3a<i>R</i>,4<i>R</i>,6<i>R</i>,6a<i>R</i>)-4-((1<i>R</i>,2<i>R</i>)-2-Ethynylcyclopropyl)-6-methoxy-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxole (<b>S30</b>)</h4><div class="NLM_p last">To a solution of aldehyde <b>S29</b> (182 mg, 0.751 mmol) in MeOH (4.0 mL) at RT was added K<sub>2</sub>CO<sub>3</sub> (312 mg, 2.26 mmol, 3.0 equiv), followed by a solution of dimethyl-1-diazo-2-oxopropyl phosphonate (180 mg, 0.939 mmol, 1.25 equiv) in MeOH (3.0 mL) under vigorous stirring. The resulting yellow suspension was stirred at RT for 3 h. The reaction was quenched by the addition of a NH<sub>4</sub>Cl satd aq solution (10 mL), and CH<sub>2</sub>Cl<sub>2</sub> (15 mL) was added. The layers were separated, and the aqueous phase was extracted with CH<sub>2</sub>Cl<sub>2</sub> (3 × 15 mL). The combined organic extracts were dried over MgSO<sub>4</sub>, filtered, and concentrated. Crude material was purified by silica gel chromatography (cyclohexane/EtOAc, 0 → 50%) to afford alkyne <b>S30</b> (98 mg, 0.41 mmol, 55%) as a colorless oil. Alkyne <b>S30</b> crystallized when stored as a neat oil at −20 °C for 1 week. A small-molecule X-ray structure of <b>S30</b> was obtained, confirming its absolute configuration as shown in the <a class="ref internalNav" href="#notes1" aria-label="Supporting Information">Supporting Information</a>. <i>R</i><sub><i>f</i></sub> = 0.68 (cyclohexane/EtOAc, 67:33). FTIR (thin film), ν<sub>max</sub> (cm<sup>–1</sup>): 3283, 2989, 2935, 2834, 2116, 1458, 1407, 1382, 1373, 1307, 1273, 1233, 1209, 1194, 1159, 1104, 1090, 1056, 1034, 1023. <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 4.99 (s, 1H), 4.73 (d, <i>J</i> = 5.9 Hz, 1H), 4.66 (d, <i>J</i> = 5.9 Hz, 1H), 3.42 (s, 3H), 3.39–3.34 (m, 1H), 1.80 (d, <i>J</i> = 2.0 Hz, 1H), 1.45 (s, 3H), 1.44–1.38 (m, 1H), 1.38–1.32 (m, 1H), 1.3 (s, 3H), 0.94 (app dt, <i>J</i> = 8.5, 5.0 Hz, 1H), 0.79–0.74 (m, 1H). <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): δ 112.4, 109.1, 90.5, 85.9, 85.6, 84.1, 64.8, 54.6, 26.5, 26.1, 25.0, 13.2, 6.1. HRMS (ESI+): calcd for [C<sub>13</sub>H<sub>19</sub>O<sub>4</sub>]<sup>+</sup> 239.1278, measured 239.1288, Δ 4.4 ppm.</div></div><div id="sec4_6_27" class="NLM_sec NLM_sec_level_3"><div id="ac_i89" class="anchor-spacer"></div><h4 class="article-section__title" id="_i89"> (3<i>R</i>,4<i>R</i>,5<i>R</i>)-5-((1<i>R</i>,2<i>R</i>)-2-Ethynylcyclopropyl)tetrahydrofuran-2,3,4-triyl Triacetate (<b>S31</b>)</h4><div class="NLM_p last">To a vial charged with acetonide <b>S30</b> (97 mg, 0.41 mmol) was added a 4:1 mixture of acetic acid and water (5.0 mL) at RT. The resulting mixture was stirred at 80 °C for 6 h. Volatiles were removed in vacuo, and the residue was dried by azeotropic distillation with benzene (2 × 1.0 mL). Crude material was used directly in the next step without further purification. To a solution of the crude triol in CH<sub>2</sub>Cl<sub>2</sub> (5.0 mL) at RT were added pyridine (0.99 mL, 12 mmol, 30 equiv), Ac<sub>2</sub>O (0.50 mL, 5.3 mmol, 13 equiv), and DMAP (10 mg, 0.082 mmol, 0.20 equiv) sequentially. The resulting mixture was stirred at RT for 2 h, and volatiles were removed in vacuo. Crude material was purified by silica gel chromatography (2,2,4-trimethylpentane/EtOAc, 0 → 100%) to afford triacetate <b>S31</b> (63 mg, 0.20 mmol, 50% over 2 steps) as a 4:5 mixture of α and β anomers, as a colorless oil. <i>R</i><sub><i>f</i></sub> = 0.60 (α-anomer)/0.74 (β-anomer) (cyclohexane/EtOAc, 50:50). FTIR (thin film), ν<sub>max</sub> (cm<sup>–1</sup>): 3284, 2580, 1742, 1431, 1370, 1210, 1103, 1044, 1009. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ α-anomer: 6.38 (dd, <i>J</i> = 3.9, 0.8 Hz, 1H), 5.26–5.19 (m, 2H), 3.85 (dd, <i>J</i> = 6.8, 2.9 Hz, 1H), 2.12 (s, 3H), 2.09 (s, 3H), 2.07 (s, 3H), 1.82 (d, <i>J</i> = 2.1 Hz, 1H), 1.42 (dddd, <i>J</i> = 8.8, 6.7, 6.0, 4.5 Hz, 1H), 1.31 (app dtd, <i>J</i> = 9.5, 5.0, 2.1 Hz, 1H), 0.97 (app dt, <i>J</i> = 8.7, 5.1 Hz, 1H), 0.91 (ddd, <i>J</i> = 8.9, 6.1, 4.8 Hz, 1H). β-Anomer: 6.12 (d, <i>J</i> = 1.1 Hz, 1H), 5.33 (dd, <i>J</i> = 4.8, 1.1 Hz, 1H), 5.30 (dd, <i>J</i> = 6.6, 4.8 Hz, 1H), 3.75 (t, <i>J</i> = 6.8 Hz, 1H), 2.12 (s, 3H), 2.10 (s, 3H), 2.07 (s, 3H), 1.82 (d, <i>J</i> = 2.1 Hz, 1H), 1.39 (dddd, <i>J</i> = 8.8, 7.0, 6.0, 4.4 Hz, 1H), 1.37–1.26 (m, 1H), 0.95 (app dt, <i>J</i> = 8.7, 5.0 Hz, 1H), 0.85 (ddd, <i>J</i> = 8.8, 6.0, 4.8 Hz, 1H). <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>) δ α-anomer: 170.1, 169.7, 169.4, 93.9, 85.2, 84.5, 72.6, 70.0, 65.3, 23.7, 21.1, 20.8, 20.4, 11.7, 4.6. β-Anomer: 169.8, 169.5, 169.3, 98.1, 82.8, 74.6, 73.8, 65.1, 55.5, 24.0, 21.2, 20.7, 20.6, 11.7, 4.4. HRMS (ESI+): calcd for [C<sub>15</sub>H<sub>18</sub>O<sub>7</sub>Na]<sup>+</sup> 333.0945, measured 333.0970, Δ 7.5 ppm.</div></div><div id="sec4_6_28" class="NLM_sec NLM_sec_level_3"><div id="ac_i90" class="anchor-spacer"></div><h4 class="article-section__title" id="_i90"> (2<i>R</i>,3<i>R</i>,4<i>R</i>,5<i>R</i>)-2-(6-Benzamido-9<i>H</i>-purin-9-yl)-5-((1<i>R</i>,2<i>R</i>)-2-ethynylcyclopropyl)tetrahydrofuran-3,4-diyl Diacetate (<b>S32</b>)</h4><div class="NLM_p last">To a suspension of <i>N</i><sup>6</sup>-benzoyladenine (114 mg, 0.478 mmol, 2.15 equiv) in propionitrile (dried over 4 Å MS, 6.0 mL) at RT was added <i>N</i>,<i>O</i>-bis(trimethylsilyl)acetamide (0.16 mL, 0.67 mmol, 3.00 equiv). The resulting mixture was stirred at 80 °C for 10 min, upon which complete solubilization of <i>N</i><sup>6</sup>-benzoyladenine was observed. To a separate flask charged with triacetate <b>S31</b> (dried by azeotropic distillation with benzene (4×), 69 mg, 0.22 mmol) and 2,6-di-<i>tert</i>-butyl-4-methylpyridinium triflate (16 mg, 44 μmol, 20 mol %) was added propionitrile (6.0 mL). The resulting mixture was stirred at RT for 5 min until a clear solution was obtained, and the solution of silylated <i>N</i><sup>6</sup>-benzoyladenine was added. The reaction mixture was stirred at 95 °C for 2.5 h, cooled to RT, and the volatiles were removed in vacuo. Crude material was filtered through a short pad of silica gel (EtOAc/<i>i</i>-PrOH, 90:10) to afford <b>S32</b>, which was further purified by preparative HPLC (5% → 95% B′ over 40 min) to afford <b>S32</b> (63 mg, 0.13 mmol, 58%) as a white amorphous solid. <i>R</i><sub><i>f</i></sub> = 0.19 (EtOAc). FTIR (thin film), ν<sub>max</sub> (cm<sup>–1</sup>): 3295, 3020, 2931, 1748, 1696, 1650, 1612, 1583, 1512, 1488, 1458, 1375, 1240, 1215, 1072, 1050. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): δ 8.80 (s, 1H), 8.24 (s, 1H), 8.07–8.02 (m, 2H), 7.65–7.59 (m, 1H), 7.56–7.50 (m, 2H), 6.13 (d, <i>J</i> = 5.4 Hz, 1H), 6.07 (app t, <i>J</i> = 5.4 Hz, 1H), 5.66 (dd, <i>J</i> = 5.4, 4.3 Hz, 1H), 3.68 (dd, <i>J</i> = 8.3, 4.3 Hz, 1H), 2.15 (s, 3H), 2.08 (s, 3H), 1.83 (d, <i>J</i> = 2.1 Hz, 1H), 1.78–1.71 (m, 1H), 1.45–1.39 (m, 1H), 1.06 (app dt, <i>J</i> = 8.8, 5.2 Hz, 1H), 0.92 (ddd, <i>J</i> = 8.8, 5.8, 5.0 Hz, 1H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 169.7, 169.5, 165.1, 152.0, 151.9, 149.7, 142.5, 133.2, 133.1, 129.0, 128.3, 123.7, 87.1, 85.2, 84.9, 73.6, 73.2, 65.5, 23.6, 20.7, 20.5, 12.1, 5.3. HRMS (ESI+): calcd for [C<sub>25</sub>H<sub>24</sub>N<sub>5</sub>O<sub>6</sub>]<sup>+</sup> 490.1721, measured 490.1725, Δ 0.8 ppm.</div></div><div id="sec4_6_29" class="NLM_sec NLM_sec_level_3"><div id="ac_i91" class="anchor-spacer"></div><h4 class="article-section__title" id="_i91"> (2<i>R</i>,3<i>R</i>,4<i>R</i>,5<i>R</i>)-2-(6-Benzamido-9<i>H</i>-purin-9-yl)-5-((1<i>R</i>,2<i>R</i>)-2-((3-carbamoylphenyl)ethynyl)cyclopropyl)tetrahydrofuran-3,4-diyl Diacetate (<b>S33</b>)</h4><div class="NLM_p last">To a vial charged with alkyne <b>S32</b> (48 mg, 98 μmol) were added 3-iodobenzamide (61 mg, 0.25 mmol, 2.5 equiv), CuI (5 mg, 0.03 mmol, 25 mol %), and Pd(PPh<sub>3</sub>)<sub>4</sub> (6 mg, 5 μmol, 5 mol %). The vial headspace was purged with nitrogen for 10 min, and a 5:1:1 mixture of toluene, DMF, and <i>i</i>-Pr<sub>2</sub>NEt (degassed by sparging with nitrogen for 15 min, 5.0 mL) was added at RT. The resulting mixture was stirred at 60 °C for 1.5 h. The solution was allowed to cool to RT, and volatiles were removed in vacuo. Crude material was filtered through a short pad of silica gel (EtOAc/<i>i</i>-PrOH, 75:25) to afford benzamide <b>S33</b>, which was further purified by preparative HPLC (15% → 95% B′ over 30 min) to afford benzamide <b>S33</b> (50 mg, 82 μmol, 84%) as a white amorphous solid. <i>R</i><sub><i>f</i></sub> = 0.35 (EtOAc/<i>i</i>-PrOH, 95:5). FTIR (thin film), ν<sub>max</sub> (cm<sup>–1</sup>): 3351, 3199, 3067, 2934, 1748, 1664, 1611, 1583, 1514, 1457, 1375, 1241, 1215, 1072. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): δ 8.80 (s, 1H), 8.35 (s, 1H), 8.03–7.99 (m, 2H), 7.76–7.74 (m, 1H), 7.74–7.71 (m, 1H), 7.62–7.57 (m, 1H), 7.52–7.46 (m, 3H), 7.33 (t, <i>J</i> = 7.7 Hz, 1H), 6.88 (br s, 1H), 6.80 (br s, 1H), 6.28 (dd, <i>J</i> = 5.8, 5.3 Hz, 1H), 6.17 (d, <i>J</i> = 5.8 Hz, 1H), 5.70 (dd, <i>J</i> = 5.3, 3.5 Hz, 1H), 3.69 (dd, <i>J</i> = 9.0, 3.5 Hz, 1H), 2.17 (s, 3H), 2.08 (s, 3H), 2.00–1.94 (m, 1H), 1.61–1.56 (m, 1H), 1.17 (app dt, <i>J</i> = 8.7, 5.2 Hz, 1H), 1.02 (app dt, <i>J</i> = 8.7, 5.4 Hz, 1H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 170.2, 169.8, 169.6, 165.4, 151.9, 151.8, 149.6, 143.2, 135.4, 133.4, 132.8, 132.7, 130.7, 129.0, 128.9, 128.3, 127.2, 124.1, 123.6, 92.1, 87.5, 86.3, 76.3, 74.1, 72.9, 24.3, 20.8, 20.5, 13.0, 6.4. HRMS (ESI+): calcd for [C<sub>32</sub>H<sub>29</sub>N<sub>6</sub>O<sub>7</sub>]<sup>+</sup> 609.2092, measured 609.2114, Δ 3.6 ppm.</div></div><div id="sec4_6_30" class="NLM_sec NLM_sec_level_3"><div id="ac_i92" class="anchor-spacer"></div><h4 class="article-section__title" id="_i92"> Methyl (2<i>S</i>,5<i>S</i>)-7-(3-Carbamoylphenyl)-5-(((3a<i>R</i>,4<i>R</i>,6<i>R</i>,6a<i>R</i>)-6-methoxy-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)methyl)-2-(2,2,2-trifluoroacetamido)hept-6-ynoate (<b>S34</b>)</h4><div class="NLM_p last">To a vial charged with alkyne <b>S3</b> (140 mg, 0.320 mmol) were added 3-iodobenzamide (197 mg, 0.799 mmol, 2.5 equiv), CuI (15 mg, 80 μmol, 25 mol %), and Pd(PPh<sub>3</sub>)<sub>4</sub> (19 mg, 16 μmol, 5 mol %). The vial headspace was purged with nitrogen for 5 min, and a 5:1:1 mixture of toluene, DMF, and <i>i</i>-Pr<sub>2</sub>NEt (degassed by sparging with nitrogen for 15 min, 7.1 mL) was added at RT. The resulting mixture was stirred at 60 °C for 1 h. The solution was allowed to cool to RT, and volatiles were removed in vacuo. Crude material was purified by silica gel chromatography (2,2,4-trimethylpentane/EtOAc, 0 → 100%) to afford <b>S34</b> (72 mg, 0.13 mmol, 40%) as a colorless solid. <i>R</i><sub><i>f</i></sub> = 0.43 (EtOAc). FTIR (thin-film), ν<sub>max</sub> (cm<sup>–1</sup>): 3307, 3066, 2935, 2253, 1719, 1665, 1612, 1575, 1438, 1382, 1274, 1209, 1158. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): δ 7.83 (td, <i>J</i> = 1.8, 0.6 Hz, 1H), 7.75 (ddd, <i>J</i> = 7.8, 1.9, 1.2 Hz, 1H), 7.51 (dt, <i>J</i> = 7.7, 1.4 Hz, 1H), 7.41–7.30 (m, 2H), 6.47 (br s, 1H), 5.97 (br s, 1H), 4.95 (s, 1H), 4.69 (td, <i>J</i> = 7.6, 5.2 Hz, 1H), 4.61 (s, 2H), 4.50 (dd, <i>J</i> = 10.8, 4.2 Hz, 1H), 3.80 (s, 3H), 3.34 (s, 3H), 2.87 (ddt, <i>J</i> = 10.6, 9.0, 4.4 Hz, 1H), 2.26–2.04 (m, 2H), 1.81–1.50 (m, 4H), 1.48 (s, 3H), 1.31 (s, 3H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 171.5, 168.8, 157.2 (q, <sup>2</sup><i>J</i><sub>C–F</sub> = 38 Hz), 134.9, 133.5, 130.8, 128.7, 127.2, 123.9, 115.8 (q, <sup>1</sup><i>J</i><sub>C–F</sub> = 288 Hz), 112.5, 110.2, 91.7, 85.6, 85.3, 84.5, 83.0, 55.4, 53.2, 52.7, 40.5, 30.9, 29.7, 29.4, 26.6, 25.0. <sup>19</sup>F NMR (376 MHz, CDCl<sub>3</sub>, BTF IStd): δ −76.8. HRMS (ESI+): calcd for [C<sub>26</sub>H<sub>31</sub>F<sub>3</sub>N<sub>2</sub>O<sub>8</sub>Na]<sup>+</sup> 579.1925, measured 579.1928, Δ 0.5 ppm.</div></div><div id="sec4_6_31" class="NLM_sec NLM_sec_level_3"><div id="ac_i93" class="anchor-spacer"></div><h4 class="article-section__title" id="_i93"> Methyl (2<i>S</i>,5<i>S</i>)-7-(3-Carbamoylphenyl)-5-(((2<i>R</i>,3<i>S</i>,4<i>R</i>,5<i>R</i>)-3,4-dihydroxy-5-methoxytetrahydrofuran-2-yl)methyl)-2-(2,2,2-trifluoroacetamido)hept-6-ynoate (<b>S35</b>)</h4><div class="NLM_p last">To a vial charged with acetonide <b>S34</b> (38 mg, 68 μmol) was added a 2:1 mixture of MeOH and TFA (6.0 mL) at RT. The vial was sealed tightly, and the reaction mixture was stirred at 85 °C for 6 h. Volatiles were removed in vacuo, and the residue was purified by silica gel chromatography (EtOAc/<i>i</i>-PrOH, 0 → 15%) to afford, in order of elution, β-anomer <b>S35</b> (15 mg, 29 μmol, 42%) and the α-anomer counterpart (14 mg, 27 μmol, 40%) as off-white solids. Only β-anomer <b>S35</b> was used in the subsequent step. <i>R</i><sub><i>f</i></sub> = 0.37 (β-anomer) (EtOAc/MeOH, 91:9). FTIR (thin-film), ν<sub>max</sub> (cm<sup>–1</sup>): 3349, 3076, 2921, 1719, 1665, 1613, 1598, 1573, 1438, 1383, 1250, 1211, 1180, 1161, 1127, 1103, 1063, 1029. <sup>1</sup>H NMR (600 MHz, (CD<sub>3</sub>)<sub>2</sub>CO) δ β-anomer: 8.82 (d, <i>J</i> = 8.1 Hz, 1H), 7.96 (t, <i>J</i> = 1.8 Hz, 1H), 7.88 (dt, <i>J</i> = 7.9, 1.4 Hz, 1H), 7.56 (dt, <i>J</i> = 7.7, 1.4 Hz, 1H), 7.54 (br s, 1H), 7.44 (t, <i>J</i> = 7.7 Hz, 1H), 6.70 (br s, 1H), 4.74 (s, 1H), 4.62 (ddd, <i>J</i> = 9.6, 8.0, 4.9 Hz, 1H), 4.23 (d, <i>J</i> = 4.1 Hz, 1H), 4.18 (ddd, <i>J</i> = 10.5, 6.5, 2.8 Hz, 1H), 4.12 (d, <i>J</i> = 7.2 Hz, 1H), 3.98 (td, <i>J</i> = 6.7, 4.8 Hz, 1H), 3.91 (t, <i>J</i> = 4.4 Hz, 1H), 3.74 (s, 3H), 3.29 (s, 3H), 2.98 (ddd, <i>J</i> = 15.3, 9.5, 4.7 Hz, 1H), 2.23 (dddd, <i>J</i> = 13.7, 10.9, 6.2, 4.8 Hz, 1H), 2.15 (ddt, <i>J</i> = 13.8, 9.6, 5.0 Hz, 1H), 1.91 (ddd, <i>J</i> = 13.4, 10.6, 2.9 Hz, 1H), 1.80 (dddd, <i>J</i> = 13.1, 10.0, 6.2, 4.9 Hz, 1H), 1.75–1.67 (m, 1H), 1.65 (ddd, <i>J</i> = 13.3, 10.6, 4.4 Hz, 1H). <sup>13</sup>C NMR (101 MHz, (CD<sub>3</sub>)<sub>2</sub>CO) δ β-anomer: 171.9, 168.3, 135.7, 135.1, 131.5, 129.4, 127.9, 124.9, 109.6, 93.3, 83.1, 82.0, 76.4, 76.2, 54.9, 53.6, 52.9, 42.1, 32.3. <sup>19</sup>F NMR (471 MHz, CDCl<sub>3</sub> (500 μL)/(CD<sub>3</sub>)<sub>2</sub>CO (50 μL), BTF IStd): δ β-anomer −76.6. HRMS (ESI+): calcd for [C<sub>23</sub>H<sub>28</sub>F<sub>3</sub>N<sub>2</sub>O<sub>8</sub>]<sup>+</sup> 517.1792, measured 517.1788, Δ 0.8 ppm.</div></div><div id="sec4_6_32" class="NLM_sec NLM_sec_level_3"><div id="ac_i94" class="anchor-spacer"></div><h4 class="article-section__title" id="_i94"> Methyl (<i>S</i>,<i>E</i>)-5-(((3a<i>R</i>,4<i>R</i>,6<i>R</i>,6a<i>R</i>)-6-Methoxy-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)methyl)-7-(triisopropylsilyl)hept-2-en-6-ynoate (<b>S36</b>)</h4><div class="NLM_p last">To a solution of <b>5</b> (728 mg, 1.71 mmol) in toluene (16 mL) at RT was added methyl (triphenylphosphoranylidene)acetate (1.06 g, 3.18 mmol, 1.85 equiv) in one portion. The resulting mixture was stirred at 50 °C for 2 h, and volatiles were removed in vacuo. Crude material was purified by silica gel chromatography (hexanes/EtOAc, 2 → 40%) to afford <b>S36</b> (747 mg, 1.55 mmol, 91%) as a 13:1 mixture of <i>E</i> and <i>Z</i> isomers, as a colorless oil. <i>R</i><sub><i>f</i></sub> = 0.35 (hexanes/Et<sub>2</sub>O, 70:30). FTIR (thin film), ν<sub>max</sub> (cm<sup>–1</sup>): 2942, 2865, 2169, 1726, 1660, 1463, 1436, 1381, 1371, 1270, 1210, 1159, 1093, 1060. <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 7.00 (app dt, <i>J</i> = 15.6, 7.3 Hz, 1H), 5.88 (app dt, <i>J</i> = 15.6, 1.4 Hz, 1H), 4.93 (s, 1H), 4.61 (dd, <i>J</i> = 5.9, 1.1 Hz, 1H), 4.57 (d, <i>J</i> = 5.9 Hz, 1H), 4.45 (ddd, <i>J</i> = 10.6, 4.3, 1.1 Hz, 1H), 3.71 (s, 3H), 3.31 (s, 3H), 2.80 (dddd, <i>J</i> = 11.2, 7.5, 6.0, 3.7 Hz, 1H), 2.43–2.34 (m, 2H), 1.69 (ddd, <i>J</i> = 13.3, 10.6, 3.7 Hz, 1H), 1.60 (ddd, <i>J</i> = 13.3, 11.2, 4.3 Hz, 1H), 1.46 (s, 3H), 1.30 (s, 3H), 1.06–1.02 (m, 21H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 166.7, 145.9, 123.3, 112.5, 109.8, 109.3, 85.6, 85.5, 84.5, 83.7, 55.2, 51.5, 40.6, 38.4, 29.7, 26.7, 25.4, 18.8, 11.3. HRMS (ESI+): calcd for [C<sub>26</sub>H<sub>44</sub>O<sub>6</sub>Si<sub>1</sub>Na]<sup>+</sup> 503.2799, measured 503.2787, Δ 2.5 ppm.</div></div><div id="sec4_6_33" class="NLM_sec NLM_sec_level_3"><div id="ac_i95" class="anchor-spacer"></div><h4 class="article-section__title" id="_i95"> Methyl (<i>S</i>)-5-(((3a<i>R</i>,4<i>R</i>,6<i>R</i>,6a<i>R</i>)-6-Methoxy-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)methyl)-7-(triisopropylsilyl)hept-6-ynoate (<b>S37</b>)</h4><div class="NLM_p last">To a vial charged with <b>S36</b> (747 mg, 1.55 mmol) were added toluene (degassed by sparging with nitrogen for 15 min, 16 mL) and <i>t</i>-BuOH (degassed by sparging with nitrogen for 15 min, 0.45 mL, 4.7 mmol, 3.0 equiv) at RT, followed by the addition of (BDP)CuH (1.0 M in toluene, 3.1 mL, 3.1 mmol, 2.0 equiv). The resulting mixture was stirred for 4.5 h, and volatiles were removed in vacuo. Crude material was purified by silica gel chromatography (hexanes/EtOAc, 1 → 25%) to afford <b>S37</b> (750 mg, 1.55 mmol, quant) as a colorless oil. <i>R</i><sub><i>f</i></sub> = 0.37 (hexanes/EtOAc, 80:20). FTIR (thin film), ν<sub>max</sub> (cm<sup>–1</sup>): 2942, 2865, 2165, 1741, 1462, 1381, 1371, 1209, 1160, 1092, 1059. <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 4.93 (s, 1H), 4.63 (dd, <i>J</i> = 5.9, 1.1 Hz, 1H), 4.57 (d, <i>J</i> = 5.9 Hz, 1H), 4.46 (ddd, <i>J</i> = 10.4, 4.6, 1.1 Hz, 1H), 3.65 (s, 3H), 3.31 (s, 3H), 2.63 (app dtd, <i>J</i> = 11.0, 7.1, 3.7 Hz, 1H), 2.38–2.29 (m, 2H), 1.90–1.82 (m, 1H), 1.82–1.73 (m, 1H), 1.67 (ddd, <i>J</i> = 13.3, 10.4, 3.7 Hz, 1H), 1.59 (ddd, <i>J</i> = 13.3, 11.0, 4.6 Hz, 1H), 1.51–1.46 (m, 2H), 1.46 (s, 3H), 1.30 (s, 3H), 1.07–1.03 (m, 21H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 173.9, 112.4, 110.5, 109.8, 85.8, 85.6, 84.6, 82.8, 55.2, 51.6, 41.1, 35.2, 33.9, 30.0, 26.7, 25.4, 22.8, 18.8, 11.4. HRMS (ESI+): calcd for [C<sub>26</sub>H<sub>46</sub>O<sub>6</sub>Si<sub>1</sub>Na]<sup>+</sup> 505.2956, measured 505.2961, Δ 0.9 ppm.</div></div><div id="sec4_6_34" class="NLM_sec NLM_sec_level_3"><div id="ac_i96" class="anchor-spacer"></div><h4 class="article-section__title" id="_i96"> Methyl (<i>S</i>)-5-(((3a<i>R</i>,4<i>R</i>,6<i>R</i>,6a<i>R</i>)-6-Methoxy-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)methyl)hept-6-ynoate (<b>S38</b>)</h4><div class="NLM_p last">To a solution of TIPS alkyne <b>S37</b> (750 mg, 1.55 mmol) in THF (16 mL) at 0 °C was added TBAF (1.0 M in THF, 1.94 mL, 1.94 mmol, 1.25 equiv) slowly. The resulting mixture was stirred at RT for 1 h, followed by the addition of water (15 mL). The layers were separated, and the aqueous phase was extracted with Et<sub>2</sub>O (3 × 15 mL). The combined organic extracts were dried over MgSO<sub>4</sub>, filtered, and concentrated. Crude material was purified by silica gel chromatography (hexanes/EtOAc, 1 → 30%) to afford alkyne <b>S38</b> (468 mg, 1.43 mmol, 92%) as a colorless oil. <i>R</i><sub><i>f</i></sub> = 0.35 (hexanes/EtOAc, 75:25). FTIR (thin film), ν<sub>max</sub> (cm<sup>–1</sup>): 3273, 2959, 2926, 1738, 1438, 1373, 1258, 1089, 1014. <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 4.93 (s, 1H), 4.59 (d, <i>J</i> = 6.0 Hz, 1H), 4.57 (dd, <i>J</i> = 6.0, 0.8 Hz, 1H), 4.48 (ddd, <i>J</i> = 11.1, 3.9, 0.8 Hz, 1H), 3.66 (s, 3H), 3.32 (s, 3H), 2.64–2.57 (m, 1H), 2.37–2.29 (m, 2H), 2.11 (d, <i>J</i> = 2.4 Hz, 1H), 1.88–1.80 (m, 1H), 1.79–1.72 (m, 1H), 1.69 (ddd, <i>J</i> = 13.3, 11.2, 3.9 Hz, 1H), 1.56 (ddd, <i>J</i> = 13.3, 11.1, 4.0 Hz, 1H), 1.52–1.48 (m, 2H), 1.48 (s, 3H), 1.30 (s, 3H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 173.9, 112.4, 110.1, 85.9, 85.7, 85.2, 84.6, 70.9, 55.3, 51.6, 40.4, 34.8, 33.8, 28.7, 26.6, 25.2, 22.6. HRMS (ESI+): calcd for [C<sub>17</sub>H<sub>26</sub>O<sub>6</sub>Na]<sup>+</sup> 349.1622, measured 349.1621, Δ 0.3 ppm.</div></div><div id="sec4_6_35" class="NLM_sec NLM_sec_level_3"><div id="ac_i97" class="anchor-spacer"></div><h4 class="article-section__title" id="_i97"> (3<i>R</i>,4<i>R</i>,5<i>R</i>)-5-((<i>S</i>)-2-Ethynyl-6-methoxy-6-oxohexyl)tetrahydrofuran-2,3,4-triyl Triacetate (<b>S39</b>)</h4><div class="NLM_p last">To a vial charged with acetonide <b>S38</b> (468 mg, 1.43 mmol) at 0 °C was added a 4:1 mixture of TFA and water (10 mL). The resulting mixture was stirred at RT for 5.5 h. Volatiles were removed in vacuo, and the residue was dried by azeotropic distillation with benzene (2 × 2 mL). Crude material was used directly in the next step without further purification. To a solution of the crude triol in CH<sub>2</sub>Cl<sub>2</sub> (14 mL) at RT were added pyridine (1.16 mL, 14.3 mmol, 10 equiv), Ac<sub>2</sub>O (1.36 mL, 14.3 mmol, 10 equiv), and DMAP (123 mg, 1.00 mmol, 0.70 equiv) sequentially. The resulting mixture was stirred at RT for 1 h, and volatiles were removed in vacuo. Crude material was purified by silica gel chromatography (hexanes/EtOAc, 2 → 60%) to afford triacetate <b>S39</b> (200 mg, 0.500 mmol, 35% over 2 steps) as a 2:1 mixture of diastereomers, as a colorless oil; <i>R</i><sub><i>f</i></sub> = 0.27/0.32 (hexanes/EtOAc, 60:40). FTIR (thin film), ν<sub>max</sub> (cm<sup>–1</sup>): 3280, 2952, 2923, 1739, 1436, 1371, 1216, 1110, 1011. <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 6.35 (d, <i>J</i> = 4.6 Hz, 1H), 6.13 (d, <i>J</i> = 1.1 Hz, 2H), 5.31 (dd, <i>J</i> = 4.8, 1.1 Hz, 2H), 5.22 (dd, <i>J</i> = 6.8, 4.6 Hz, 1H), 5.18 (dd, <i>J</i> = 6.9, 4.8 Hz, 2H), 5.06 (dd, <i>J</i> = 6.8, 3.7 Hz, 1H), 4.46 (app dt, <i>J</i> = 10.3, 3.7 Hz, 1H), 4.40 (ddd, <i>J</i> = 10.3, 6.9, 3.1 Hz, 2H), 3.66 (s, 9H), 2.63–2.57 (m, 3H), 2.37–2.28 (m, 6H), 2.11 (s, 6H), 2.11 (s, 3H), 2.10 (s, 3H), 2.07 (s, 6H), 2.06 (s, 6H), 2.05 (s, 3H), 1.89–1.80 (m, 3H), 1.80–1.69 (m, 6H), 1.65 (ddd, <i>J</i> = 13.9, 10.3, 4.2 Hz, 2H), 1.62 (ddd, <i>J</i> = 13.4, 10.3, 4.0 Hz, 1H), 1.55–1.45 (m, 6H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 173.9, 173.9, 170.2, 170.0, 169.8, 169.6, 169.5, 169.2, 98.5, 93.9, 85.7, 85.4, 81.5, 79.9, 74.7, 74.0, 72.7, 71.1, 70.9, 69.9, 51.7, 40.2, 39.2, 34.7, 34.7, 33.8, 33.8, 28.5, 28.2, 22.5, 22.5, 21.2, 21.2, 20.8, 20.7, 20.7, 20.4. HRMS (ESI+): calcd for [C<sub>19</sub>H<sub>26</sub>O<sub>9</sub>Na]<sup>+</sup> 421.1469, measured 421.1490, Δ 4.9 ppm.</div></div><div id="sec4_6_36" class="NLM_sec NLM_sec_level_3"><div id="ac_i98" class="anchor-spacer"></div><h4 class="article-section__title" id="_i98"> (2<i>R</i>,3<i>R</i>,4<i>R</i>,5<i>R</i>)-2-(6-Benzamido-9<i>H</i>-purin-9-yl)-5-((<i>S</i>)-2-ethynyl-6-methoxy-6-oxohexyl)tetrahydrofuran-3,4-diyl Diacetate (<b>S40</b>)</h4><div class="NLM_p last">To a suspension of <i>N</i><sup>6</sup>-benzoyladenine (240 mg, 1.00 mmol, 4.0 equiv) in propionitrile (dried over 4 Å MS, 4.2 mL) at RT was added <i>N</i>,<i>O</i>-bis(trimethylsilyl)acetamide (0.31 mL, 1.3 mmol, 5.0 equiv). The resulting mixture was stirred at 80 °C for 10 min, upon which complete solubilization of <i>N</i><sup>6</sup>-benzoyladenine was observed. To a separate flask charged with triacetate <b>S39</b> (dried by azeotropic distillation with benzene (4 × ), 100 mg, 0.251 mmol), and 2,6-di-<i>tert</i>-butyl-4-methylpyridinium triflate (45 mg, 0.13 mmol, 50 mol %) was added propionitrile (4.2 mL). The resulting mixture was stirred at RT for 5 min until a clear solution was obtained, and the solution of silylated <i>N</i><sup>6</sup>-benzoyladenine was added. The reaction mixture was stirred at 95 °C for 2.5 h, cooled to RT, and the volatiles were removed in vacuo. Crude material was purified by silica gel chromatography (toluene/MeCN, 2 → 60%) to afford nucleoside <b>S40</b> (102 mg, 0.177 mmol, 70%) as a colorless oil. <i>R</i><sub><i>f</i></sub> = 0.44 (EtOAc). FTIR (thin film), ν<sub>max</sub> (cm<sup>–1</sup>): 3285, 2952, 2925, 2855, 1744, 1703, 1610, 1582, 1510, 1486, 1455, 1241, 1219, 1096, 1074, 1047. <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 8.98 (br s, 1H), 8.80 (s, 1H), 8.07 (s, 1H), 8.04–8.00 (m, 2H), 7.63–7.59 (m, 1H), 7.55–7.50 (m, 2H), 6.14–6.11 (m, 2H), 5.59–5.54 (m, 1H), 4.53 (ddd, <i>J</i> = 11.0, 4.4, 2.8 Hz, 1H), 3.66 (s, 3H), 2.59–2.51 (m, 1H), 2.35–2.25 (m, 2H), 2.16 (s, 3H), 2.16–2.10 (m, 1H), 2.13 (d, <i>J</i> = 2.4 Hz, 1H), 2.07 (s, 3H), 1.88–1.80 (m, 2H), 1.75–1.67 (m, 1H), 1.55–1.48 (m, 2H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 173.8, 169.8, 169.5, 164.7, 152.9, 151.7, 149.9, 142.4, 133.6, 133.0, 129.0, 128.0, 124.1, 87.2, 85.5, 81.0, 73.8, 72.9, 71.2, 51.7, 38.4, 34.7, 33.8, 28.1, 22.5, 20.8, 20.5. HRMS (ESI+): calcd for [C<sub>29</sub>H<sub>32</sub>N<sub>5</sub>O<sub>8</sub>]<sup>+</sup> 578.2245, measured 578.2242, Δ 0.5 ppm.</div></div><div id="sec4_6_37" class="NLM_sec NLM_sec_level_3"><div id="ac_i99" class="anchor-spacer"></div><h4 class="article-section__title" id="_i99"> (2<i>R</i>,3<i>R</i>,4<i>R</i>,5<i>R</i>)-2-(6-Benzamido-9<i>H</i>-purin-9-yl)-5-((<i>S</i>)-2-((3-carbamoylphenyl)ethynyl)-6-methoxy-6-oxohexyl)tetrahydrofuran-3,4-diyl Diacetate (<b>S41</b>)</h4><div class="NLM_p last">To a solution of alkyne <b>S40</b> (102 mg, 0.177 mmol) and 3-iodobenzamide (131 mg, 0.530 mmol, 3.0 equiv) in a 5:1:1 mixture of toluene, DMF, and <i>i</i>-Pr<sub>2</sub>NEt (degassed by sparging with nitrogen for 15 min, 9.0 mL) at RT were introduced CuI (8 mg, 0.04 mmol, 25 mol %) and Pd(PPh<sub>3</sub>)<sub>4</sub> (10 mg, 8.8 μmol, 5 mol %) rapidly. The resulting mixture was stirred at 60 °C for 2.5 h. The solution was allowed to cool to RT, and volatiles were removed in vacuo. Crude material was purified by silica gel chromatography (EtOAc/<i>i</i>-PrOH, 0 → 10%) to afford benzamide <b>S41</b> (105 mg, 0.151 mmol, 85%) as a white amorphous solid. <i>R</i><sub><i>f</i></sub> = 0.39 (EtOAc/<i>i</i>-PrOH, 95:5). FTIR (thin film), ν<sub>max</sub> (cm<sup>–1</sup>): 3340, 3198, 2925, 2855, 1747, 1669, 1611, 1581, 1513, 1487, 1456, 1375, 1244, 1221, 1097, 1047. <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 9.12 (s, 1H), 8.79 (s, 1H), 8.11 (s, 1H), 8.03–8.00 (m, 2H), 7.87–7.86 (m, 1H), 7.75 (ddd, <i>J</i> = 7.8, 1.8, 1.2 Hz, 1H), 7.62–7.57 (m, 1H), 7.54–7.47 (m, 3H), 7.36 (app t, <i>J</i> = 7.8 Hz, 1H), 6.51 (br s, 1H), 6.18 (dd, <i>J</i> = 5.9, 5.2 Hz, 1H), 6.15 (d, <i>J</i> = 5.9 Hz, 1H), 5.98 (br s, 1H), 5.66 (dd, <i>J</i> = 5.2, 3.8 Hz, 1H), 4.53 (app dt, <i>J</i> = 10.0, 3.8 Hz, 1H), 3.65 (s, 3H), 2.76 (app dtd, <i>J</i> = 11.1, 7.0, 3.7 Hz, 1H), 2.39–2.29 (m, 2H), 2.20 (ddd, <i>J</i> = 13.6, 10.0, 3.7 Hz, 1H), 2.15 (s, 3H), 2.05 (s, 3H), 1.98 (ddd, <i>J</i> = 13.6, 11.1, 3.8 Hz, 1H), 1.92–1.84 (m, 1H), 1.81–1.73 (m, 1H), 1.63–1.57 (m, 2H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 173.9, 170.0, 169.7, 168.8, 164.8, 152.9, 151.8, 149.9, 142.4, 134.9, 133.6, 133.6, 133.0, 130.8, 129.0, 128.7, 128.0, 127.0, 124.3, 124.0, 92.3, 86.9, 82.5, 81.5, 74.0, 72.8, 51.7, 38.7, 35.0, 33.8, 29.0, 22.7, 20.8, 20.6. HRMS (ESI+): calcd for [C<sub>36</sub>H<sub>36</sub>N<sub>6</sub>O<sub>9</sub>Na]<sup>+</sup> 719.2436, measured 719.2452, Δ 2.3 ppm.</div></div><div id="sec4_6_38" class="NLM_sec NLM_sec_level_3"><div id="ac_i100" class="anchor-spacer"></div><h4 class="article-section__title" id="_i100"> Triisopropyl ((<i>S</i>,<i>E</i>)-3-(((3a<i>R</i>,4<i>R</i>,6<i>R</i>,6a<i>R</i>)-6-Methoxy-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)methyl)-6-nitrohex-5-en-1-yn-1-yl)silane (<b>S42</b>)</h4><div class="NLM_p last">To a solution of aldehyde <b>5</b> (385 mg, 0.907 mmol) in MeNO<sub>2</sub> (9 mL) at 0 °C was added Et<sub>3</sub>N (0.13 mL, 0.91 mmol, 1.00 equiv). The resulting mixture was stirred at RT for 17 h, and volatiles were removed in vacuo. To a solution of the residue in CH<sub>2</sub>Cl<sub>2</sub> (9 mL) at −78 °C were added <i>i</i>-Pr<sub>2</sub>NEt (0.40 mL, 2.3 mmol, 2.50 equiv) and MsCl (90 μL, 1.1 mmol, 1.25 equiv) sequentially. The resulting mixture was stirred at −78 °C for 2 h before the addition of a NH<sub>4</sub>Cl satd aq solution (8 mL). The biphasic mixture was allowed to warm to RT, and the layers were separated. The aqueous phase was extracted with CH<sub>2</sub>Cl<sub>2</sub> (3 × 8 mL). The combined organic extracts were dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated. Crude material was purified by silica gel chromatography (hexanes/EtOAc, 2 → 25%) to afford <b>S42</b> (319 mg, 0.682 mmol, 75% over 2 steps) as a colorless oil. <i>R</i><sub><i>f</i></sub> = 0.46 (hexanes/EtOAc, 75:25). FTIR (thin film), ν<sub>max</sub> (cm<sup>–1</sup>): 2941, 2865, 2168, 1655, 1528, 1463, 1351, 1210, 1092. <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 7.33 (ddd, <i>J</i> = 13.4, 7.9, 7.5 Hz, 1H), 7.06 (app dt, <i>J</i> = 13.4, 1.4 Hz, 1H), 4.95 (s, 1H), 4.62 (dd, <i>J</i> = 5.9, 1.1 Hz, 1H), 4.59 (d, <i>J</i> = 5.9 Hz, 1H), 4.46 (ddd, <i>J</i> = 10.6, 4.3, 1.1 Hz, 1H), 3.34 (s, 3H), 2.89 (dddd, <i>J</i> = 10.9, 7.9, 5.5, 4.0 Hz, 1H), 2.49 (dddd, <i>J</i> = 14.6, 7.5, 5.5, 1.4 Hz, 1H), 2.41 (app dtd, <i>J</i> = 14.6, 7.9, 1.4 Hz, 1H), 1.72 (ddd, <i>J</i> = 13.3, 10.6, 4.0 Hz, 1H), 1.66 (ddd, <i>J</i> = 13.3, 10.9, 4.3 Hz, 1H), 1.47 (s, 3H), 1.31 (s, 3H), 1.07–1.03 (m, 21H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 141.0, 139.2, 112.7, 110.0, 108.0, 85.5, 85.3, 85.1, 84.4, 55.4, 40.8, 34.4, 29.6, 26.7, 25.4, 18.7, 11.3. HRMS (ESI+): calcd for [C<sub>24</sub>H<sub>41</sub>N<sub>1</sub>O<sub>6</sub>Si<sub>1</sub>Na]<sup>+</sup> 490.2595, measured 490.2580, Δ 3.2 ppm.</div></div><div id="sec4_6_39" class="NLM_sec NLM_sec_level_3"><div id="ac_i101" class="anchor-spacer"></div><h4 class="article-section__title" id="_i101"> 2,2,2-Trifluoro-<i>N</i>-((<i>S</i>)-4-(((3a<i>R</i>,4<i>R</i>,6<i>R</i>,6a<i>R</i>)-6-methoxy-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)methyl)-6-(triisopropylsilyl)hex-5-yn-1-yl)acetamide (<b>S43</b>)</h4><div class="NLM_p last">To a solution of <b>S42</b> (319 mg, 0.682 mmol) in THF (6.8 mL) at 0 °C was added LiAlH<sub>4</sub> (1.0 M in Et<sub>2</sub>O, 4.8 mL, 4.8 mmol, 7.0 equiv) dropwise. The resulting mixture was stirred at RT for 4 h. The reaction was quenched by the dropwise addition of water (0.18 mL), a 15% NaOH aq solution (0.18 mL), and water (0.54 mL), sequentially, at 0 °C, and Na<sub>2</sub>SO<sub>4</sub> was added. The mixture was stirred vigorously for 20 min and filtered through a cotton plug to remove the white solids. The filtrate was concentrated, and the crude material was used directly in the next step without further purification. To a solution of the crude amine in THF (6.8 mL) at 0 °C were added Et<sub>3</sub>N (5.7 mL, 41 mmol, 60 equiv) and ethyl trifluoroacetate (2.4 mL, 21 mmol, 30 equiv) sequentially. The resulting mixture was stirred at RT for 17 h, and volatiles were removed in vacuo. Crude material was purified by silica gel chromatography (hexanes/EtOAc, 2 → 50%) to afford trifluoroacetamide <b>S43</b> (252 mg, 0.470 mmol, 69% over 2 steps) as a colorless oil. <i>R</i><sub><i>f</i></sub> = 0.47 (hexanes/EtOAc, 65:35). FTIR (thin film), ν<sub>max</sub> (cm<sup>–1</sup>): 3306, 2941, 2865, 2166, 1705, 1559, 1463, 1381, 1209, 1161, 1106, 1061. <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 6.27 (br s, 1H), 4.94 (s, 1H), 4.62 (dd, <i>J</i> = 5.9, 0.7 Hz, 1H), 4.58 (d, <i>J</i> = 5.9 Hz, 1H), 4.45 (ddd, <i>J</i> = 10.3, 4.7, 0.7 Hz, 1H), 3.44–3.39 (m, 2H), 3.32 (s, 3H), 2.69–2.63 (m, 1H), 1.88–1.80 (m, 1H), 1.76 (app ddq, <i>J</i> = 13.4, 9.9, 6.7 Hz, 1H), 1.68 (ddd, <i>J</i> = 13.4, 10.3, 4.1 Hz, 1H), 1.62 (ddd, <i>J</i> = 13.4, 10.7, 4.7 Hz, 1H), 1.56–1.48 (m, 2H), 1.47 (s, 3H), 1.31 (s, 3H), 1.08–1.03 (m, 21H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 157.3 (q, <sup>2</sup><i>J</i><sub>C–F</sub> = 37 Hz), 116.0 (q, <sup>1</sup><i>J</i><sub>C–F</sub> = 288 Hz), 112.5, 110.1, 109.8, 85.7, 85.6, 84.5, 83.4, 55.2, 41.2, 39.8, 32.8, 30.0, 26.9, 26.7, 25.4, 18.8, 11.3. <sup>19</sup>F NMR (471 MHz, CDCl<sub>3</sub>, BTF IStd): δ −76.9. HRMS (ESI+): calcd for [C<sub>26</sub>H<sub>44</sub>F<sub>3</sub>N<sub>1</sub>O<sub>5</sub>Si<sub>1</sub>Na]<sup>+</sup> 558.2833, measured 558.2858, Δ 4.4 ppm.</div></div><div id="sec4_6_40" class="NLM_sec NLM_sec_level_3"><div id="ac_i102" class="anchor-spacer"></div><h4 class="article-section__title" id="_i102"> 2,2,2-Trifluoro-<i>N</i>-((<i>S</i>)-4-(((3a<i>R</i>,4<i>R</i>,6<i>R</i>,6a<i>R</i>)-6-methoxy-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)methyl)hex-5-yn-1-yl)acetamide (<b>S44</b>)</h4><div class="NLM_p last">To a solution of TIPS alkyne <b>S43</b> (196 mg, 0.366 mmol) in DMF (3 mL) at RT was added a solution of TASF (0.30 g, 1.2 mmol, 3.0 equiv) in DMF (4 mL). The resulting mixture was stirred at 60 °C for 2.5 h, and volatiles were removed in vacuo. The residue was partitioned between Et<sub>2</sub>O (7 mL) and a 1:1 mixture of brine and 5% LiCl aq solution (7 mL). The layers were separated, and the aqueous phase was extracted with Et<sub>2</sub>O (3 × 7 mL). The combined organic extracts were dried over MgSO<sub>4</sub>, filtered, and concentrated. Crude material was purified by silica gel chromatography (hexanes/EtOAc, 5 → 50%) to afford alkyne <b>S44</b> (117 g, 0.308 mmol, 84%) as an amorphous white solid. <i>R</i><sub><i>f</i></sub> = 0.32 (hexanes/EtOAc, 65:35). FTIR (thin film), ν<sub>max</sub> (cm<sup>–1</sup>): 3298, 2937, 1706, 1558, 1454, 1382, 1209, 1159, 1104, 1059. <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 6.43 (br s, 1H), 4.94 (s, 1H), 4.59 (d, <i>J</i> = 6.0 Hz, 1H), 4.57 (dd, <i>J</i> = 6.0, 0.9 Hz, 1H), 4.47 (ddd, <i>J</i> = 11.1, 4.0, 0.9 Hz, 1H), 3.42–3.37 (m, 2H), 3.32 (s, 3H), 2.66–2.60 (m, 1H), 2.14 (d, <i>J</i> = 2.4 Hz, 1H), 1.86–1.78 (m, 1H), 1.74 (app ddt, <i>J</i> = 14.0, 10.3, 7.0 Hz, 1H), 1.69 (ddd, <i>J</i> = 13.3, 11.1, 4.0 Hz, 1H), 1.59 (ddd, <i>J</i> = 13.3, 10.9, 4.0 Hz, 1H), 1.53–1.49 (m, 2H), 1.48 (s, 3H), 1.32–1.30 (s, 3H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 157.4 (q, <sup>2</sup><i>J</i><sub>C–F</sub> = 37 Hz), 116.0 (q, <sup>1</sup><i>J</i><sub>C–F</sub> = 288 Hz), 112.5, 110.1, 85.6, 85.6, 85.1, 84.5, 71.3, 55.3, 40.5, 39.8, 32.3, 28.6, 26.8, 26.6, 25.1. <sup>19</sup>F NMR (471 MHz, CDCl<sub>3</sub>, BTF IStd): δ −76.9. HRMS (ESI+): calcd for [C<sub>17</sub>H<sub>24</sub>F<sub>3</sub>N<sub>1</sub>O<sub>5</sub>Na]<sup>+</sup> 402.1499, measured 402.1510, Δ 2.7 ppm.</div></div><div id="sec4_6_41" class="NLM_sec NLM_sec_level_3"><div id="ac_i103" class="anchor-spacer"></div><h4 class="article-section__title" id="_i103"> (3<i>R</i>,4<i>R</i>,5<i>R</i>)-5-((<i>S</i>)-2-Ethynyl-5-(2,2,2-trifluoroacetamido)pentyl)tetrahydrofuran-2,3,4-triyl Triacetate (<b>S45</b>)</h4><div class="NLM_p last">To a solution of <b>S44</b> (117 mg, 0.308 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (1.5 mL) at 0 °C was added a 4:1 mixture of TFA and water (6.0 mL). The resulting mixture was stirred at RT for 15 h. Volatiles were removed in vacuo, and the residue was dried by azeotropic distillation with benzene (2 × 2 mL). Crude material was used directly in the next step without further purification. To a solution of the crude triol in CH<sub>2</sub>Cl<sub>2</sub> (6.0 mL) at RT were added pyridine (0.37 mL, 4.6 mmol, 15.0 equiv), Ac<sub>2</sub>O (0.44 mL, 4.6 mmol, 15.0 equiv), and DMAP (39 mg, 0.32 mmol, 1.05 equiv) sequentially. The resulting mixture was stirred at RT for 2.5 h before the addition of water (6 mL). The layers were separated, and the aqueous phase was extracted with CH<sub>2</sub>Cl<sub>2</sub> (3 × 6 mL). The combined organic extracts were dried over MgSO<sub>4</sub>, filtered, and concentrated. Crude material was purified by silica gel chromatography (hexanes/EtOAc, 5 → 70%) to afford <b>S45</b> (74 mg, 0.16 mmol, 53% over 2 steps) as a 1:1 mixture of diastereomers as a colorless oil. <i>R</i><sub><i>f</i></sub> = 0.30/0.41 (hexanes/EtOAc, 50:50). FTIR (thin film), ν<sub>max</sub> (cm<sup>–1</sup>): 3288, 2933, 1749, 1720, 1558, 1435, 1373, 1218, 1180, 1158, 1012. <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 6.70–6.62 (m, 2H), 6.34 (d, <i>J</i> = 4.5 Hz, 1H), 6.10 (d, <i>J</i> = 0.9 Hz, 1H), 5.30 (dd, <i>J</i> = 4.8, 0.9 Hz, 1H), 5.21 (dd, <i>J</i> = 6.7, 4.5 Hz, 1H), 5.16 (dd, <i>J</i> = 7.0, 4.8 Hz, 1H), 5.04 (dd, <i>J</i> = 6.7, 3.7 Hz, 1H), 4.44 (app dt, <i>J</i> = 10.1, 3.7 Hz, 1H), 4.36 (ddd, <i>J</i> = 10.1, 7.0, 3.0 Hz, 1H), 3.42–3.33 (m, 4H), 2.66–2.59 (m, 2H), 2.13 (d, <i>J</i> = 2.4 Hz, 2H), 2.11 (s, 3H), 2.11 (s, 3H), 2.10 (s, 3H), 2.07 (s, 3H), 2.06 (s, 3H), 2.05 (s, 3H), 1.87–1.76 (m, 4H), 1.75–1.68 (m, 2H), 1.68–1.60 (m, 2H), 1.57–1.43 (m, 4H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 170.3, 170.1, 169.9, 169.7, 169.5, 169.4, 157.4 (q, <sup>2</sup><i>J</i><sub>C–F</sub> = 37 Hz), 157.4 (q, <sup>2</sup><i>J</i><sub>C–F</sub> = 37 Hz), 116.0 (q, <sup>1</sup><i>J</i><sub>C–F</sub> = 288 Hz), 116.0 (q, <sup>1</sup><i>J</i><sub>C–F</sub> = 288 Hz), 98.5, 93.8, 85.5, 85.2, 81.5, 79.9, 74.5, 73.9, 72.7, 71.4, 71.2, 69.9, 40.1, 39.7, 39.6, 39.3, 32.1, 32.0, 28.5, 28.1, 26.6, 26.3, 21.2, 21.2, 20.8, 20.7, 20.6, 20.4. <sup>19</sup>F NMR (471 MHz, CDCl<sub>3</sub>, BTF IStd): δ −76.88, −76.90. HRMS (ESI+): calcd for [C<sub>19</sub>H<sub>24</sub>F<sub>3</sub>N<sub>1</sub>O<sub>8</sub>Na]<sup>+</sup> 474.1346, measured 474.1358, Δ 2.5 ppm.</div></div><div id="sec4_6_42" class="NLM_sec NLM_sec_level_3"><div id="ac_i104" class="anchor-spacer"></div><h4 class="article-section__title" id="_i104"> (2<i>R</i>,3<i>R</i>,4<i>R</i>,5<i>R</i>)-2-(6-Benzamido-9<i>H</i>-purin-9-yl)-5-((<i>S</i>)-2-ethynyl-5-(2,2,2-trifluoroacetamido)pentyl)tetrahydrofuran-3,4-diyl Diacetate (<b>S46</b>)</h4><div class="NLM_p last">To a suspension of <i>N</i><sup>6</sup>-benzoyladenine (72 mg, 0.30 mmol, 4.0 equiv) in propionitrile (dried over 4 Å MS, 1.5 mL) at RT was added <i>N</i>,<i>O</i>-bis(trimethylsilyl)acetamide (92 μL, 0.38 mmol, 5.0 equiv). The resulting mixture was stirred at 80 °C for 10 min, upon which complete solubilization of <i>N</i><sup>6</sup>-benzoyladenine was observed. To a separate flask charged with triacetate <b>S45</b> (dried by azeotropic distillation with benzene (4 × ), 34 mg, 75 μmol) and 2,6-di-<i>tert</i>-butyl-4-methylpyridinium triflate (13 mg, 38 μmol, 50 mol %) was added propionitrile (1.5 mL). The resulting mixture was stirred at RT for 5 min until a clear solution was obtained, and the solution of silylated <i>N</i><sup>6</sup>-benzoyladenine was added. The reaction mixture was stirred at 95 °C for 4 h, cooled to RT, and the volatiles were removed in vacuo. Crude material was purified by silica gel chromatography (toluene/MeCN, 0 → 50%) to afford nucleoside <b>S46</b> (35 mg, 56 μmol, 74%) as a colorless oil. <i>R</i><sub><i>f</i></sub> = 0.31 (hexanes/EtOAc, 25:75). FTIR (thin film), ν<sub>max</sub> (cm<sup>–1</sup>): 3289, 2928, 1749, 1708, 1612, 1583, 1457, 1245, 1218, 1159, 1076, 1049. <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 9.15 (s, 1H), 8.75 (s, 1H), 8.07 (s, 1H), 8.04–7.98 (m, 2H), 7.62–7.57 (m, 1H), 7.53–7.48 (m, 2H), 6.71–6.62 (m, 1H), 6.13–6.09 (m, 2H), 5.55 (app t, <i>J</i> = 4.3 Hz, 1H), 4.49 (ddd, <i>J</i> = 10.8, 4.3, 3.1 Hz, 1H), 3.34 (m, 2H), 2.59–2.52 (m, 1H), 2.16 (s, 3H), 2.15 (d, <i>J</i> = 2.4 Hz, 1H), 2.12–2.06 (ddd, <i>J</i> = 13.5, 10.8, 4.2 Hz, 1H), 2.06 (s, 3H), 1.87 (ddd, <i>J</i> = 13.5, 11.2, 3.1 Hz, 1H), 1.82–1.74 (m, 1H), 1.67 (app tdd, <i>J</i> = 13.4, 8.5, 6.5 Hz, 1H), 1.56–1.45 (m, 2H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 169.9, 169.6, 164.9, 157.4 (q, <sup>2</sup><i>J</i><sub>C–F</sub> = 37 Hz), 152.8, 151.7, 149.9, 142.4, 133.5, 133.0, 129.0, 128.0, 124.1, 116.0 (q, <sup>1</sup><i>J</i><sub>C–F</sub> = 288 Hz), 87.2, 85.2, 80.9, 73.7, 72.9, 71.5, 39.6, 38.5, 32.2, 28.0, 26.7, 20.8, 20.5. <sup>19</sup>F NMR (471 MHz, CDCl<sub>3</sub>, BTF IStd): δ −76.9. HRMS (ESI+): calcd for [C<sub>29</sub>H<sub>30</sub>F<sub>3</sub>N<sub>6</sub>O<sub>7</sub>]<sup>+</sup> 631.2123, measured 631.2142, Δ 3.1 ppm.</div></div><div id="sec4_6_43" class="NLM_sec NLM_sec_level_3"><div id="ac_i105" class="anchor-spacer"></div><h4 class="article-section__title" id="_i105"> (2<i>R</i>,3<i>R</i>,4<i>R</i>,5<i>R</i>)-2-(6-Benzamido-9<i>H</i>-purin-9-yl)-5-((<i>S</i>)-2-((3-carbamoylphenyl)ethynyl)-5-(2,2,2-trifluoroacetamido)pentyl)tetrahydrofuran-3,4-diyl Diacetate (<b>S47</b>)</h4><div class="NLM_p last">To a solution of <b>S46</b> (33 mg, 52 μmol) and 3-iodobenzamide (39 mg, 0.16 mmol, 3.0 equiv) in a 5:1:1 mixture of toluene, DMF, and <i>i</i>-Pr<sub>2</sub>NEt (degassed by sparging with nitrogen for 15 min, 2.6 mL) at RT were introduced CuI (3 mg, 0.01 mmol, 25 mol %) and Pd(PPh<sub>3</sub>)<sub>4</sub> (3 mg, 3 μmol, 5 mol %) rapidly. The resulting mixture was stirred at 60 °C for 2 h. The solution was allowed to cool to RT, and volatiles were removed in vacuo. Crude material was purified by silica gel chromatography (EtOAc/<i>i</i>-PrOH, 0 → 10%) to afford <b>S47</b> (32 mg, 43 μmol, 82%) as a white amorphous solid. <i>R</i><sub><i>f</i></sub> = 0.28 (EtOAc). FTIR (thin film), ν<sub>max</sub> (cm<sup>–1</sup>): 3306, 2928, 1749, 1709, 1667, 1611, 1581, 1456, 1376, 1245, 1217, 1157, 1098, 1049. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): δ 9.32 (s, 1H), 8.75 (s, 1H), 8.12 (s, 1H), 8.04–7.99 (m, 2H), 7.89–7.87 (m, 1H), 7.76 (ddd, <i>J</i> = 7.7, 1.9, 1.2 Hz, 1H), 7.61–7.56 (m, 1H), 7.52–7.47 (m, 3H), 7.34 (app td, <i>J</i> = 7.7, 0.5 Hz, 1H), 7.19–7.16 (m, 1H), 6.80 (br s, 1H), 6.18 (dd, <i>J</i> = 5.8, 5.2 Hz, 1H), 6.14 (d, <i>J</i> = 5.8 Hz, 1H), 6.08 (br s, 1H), 5.63 (dd, <i>J</i> = 5.2, 3.8 Hz, 1H), 4.51 (ddd, <i>J</i> = 10.2, 3.8, 3.6 Hz, 1H), 3.43–3.31 (m, 2H), 2.76 (m, 1H), 2.15 (s, 3H), 2.19–2.12 (m, 1H), 2.06 (s, 3H), 1.96 (ddd, <i>J</i> = 13.3, 11.0, 3.6 Hz, 1H), 1.88–1.79 (m, 1H), 1.79–1.69 (m, 1H), 1.64–1.52 (m, 2H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 170.1, 169.7, 168.9, 165.1, 157.6 (q, <sup>2</sup><i>J</i><sub>C–F</sub> = 37 Hz), 152.7, 151.8, 150.0, 142.6, 134.7, 133.6, 133.5, 133.0, 131.0, 129.0, 128.7, 128.1, 127.3, 124.3, 123.7, 116.0 (q, <sup>1</sup><i>J</i><sub>C–F</sub> = 288 Hz), 91.8, 87.0, 82.9, 81.3, 73.9, 72.7, 39.9, 38.7, 32.5, 28.8, 26.7, 20.8, 20.6. <sup>19</sup>F NMR (471 MHz, CDCl<sub>3</sub>, BTF IStd): δ −76.8. HRMS (ESI+): calcd for [C<sub>36</sub>H<sub>34</sub>F<sub>3</sub>N<sub>7</sub>O<sub>8</sub>Na]<sup>+</sup> 772.2313, measured 772.2306, Δ 0.9 ppm.</div></div><div id="sec4_6_44" class="NLM_sec NLM_sec_level_3"><div id="ac_i106" class="anchor-spacer"></div><h4 class="article-section__title" id="_i106"> Methyl (2<i>S</i>,5<i>S</i>)-2-((((9<i>H</i>-Fluoren-9-yl)methoxy)carbonyl)amino)-5-(((3a<i>R</i>,4<i>R</i>,6<i>R</i>,6a<i>R</i>)-6-methoxy-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)methyl)hept-6-ynoate (<b>S48</b>)</h4><div class="NLM_p last">To a solution of acetonide <b>S3</b> (136 mg, 311 μmol) in THF (3.0 mL) was added a 0.5 M LiOH aq solution (1.3 mL) at 0 °C. After 75 min, more 0.5 M LiOH aq solution (1.5 mL) was added. The reaction mixture was allowed to warm gradually to RT and was stirred for 8 h. More 0.5 M LiOH aq solution (1.5 mL) was added, and the resulting mixture was stirred for 1 h. The reaction mixture was cooled to 0 °C, and a 1 M HCl aq solution was added dropwise until pH 6 was obtained. A NaHCO<sub>3</sub> satd aq solution (5.0 mL) was added slowly followed by 1,4-dioxane (3.0 mL). A solution of Fmoc-Cl in 1,4-dioxane (0.39 M, 1.0 mL, 0.39 mmol, 1.25 equiv) was added dropwise over 5 min under vigorous stirring. After 1 h, more Fmoc-Cl in 1,4-dioxane (0.39 M, 2.0 mL, 0.78 mmol, 2.50 equiv) was added, and the reaction mixture was stirred for 2 h. The reaction was quenched by the slow addition of a 1 M HCl aq solution (8 mL) at 0 °C followed by CH<sub>2</sub>Cl<sub>2</sub> (8 mL). The layers were separated using a phase separator (Biotage Isolute), and the aqueous phase was extracted with CH<sub>2</sub>Cl<sub>2</sub> (2 × 8 mL). The combined organic extracts were concentrated, and the residue was dissolved in MeOH (20 mL). (Trimethylsilyl)diazomethane (2.0 M in hexanes, 5.0 mL) was added to the methanolic solution over 2 min, and the reaction mixture was stirred for 1 h. More (trimethylsilyl)diazomethane (2.0 M in hexanes, 3.0 mL) was added, and the resulting mixture was stirred for 1 h. The reaction was quenched by the dropwise addition of glacial AcOH (4 mL) over 10 min, and the volatiles were removed in vacuo. Crude material was purified by silica gel chromatography (2,2,4-trimethylpentane/EtOAc, 0 → 100%) to afford carbamate <b>S48</b> (83 mg, 0.15 mmol, 47% over 3 steps) as an amorphous white solid. <i>R</i><sub><i>f</i></sub> = 0.56 (cyclohexane/EtOAc, 50:50). FTIR (thin film), ν<sub>max</sub> (cm<sup>–1</sup>): 3300, 2987, 2947, 1723, 1526, 1447, 1377, 1341, 1212, 1163, 1099, 1061. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): δ 7.77 (dq, <i>J</i> = 7.5, 1.0 Hz, 2H), 7.60 (t, <i>J</i> = 6.8 Hz, 2H), 7.40 (tq, <i>J</i> = 7.4, 1.0 Hz, 2H), 7.32 (tt, <i>J</i> = 7.5, 1.4 Hz, 2H), 5.31 (d, <i>J</i> = 8.5 Hz, 1H), 4.94 (s, 1H), 4.58 (q, <i>J</i> = 6.0 Hz, 2H), 4.48 (dd, <i>J</i> = 11.1, 4.0 Hz, 1H), 4.46–4.33 (m, 3H), 4.23 (t, <i>J</i> = 7.1 Hz, 1H), 3.77 (s, 3H), 3.31 (s, 3H), 2.66–2.60 (m, 1H), 2.14 (d, <i>J</i> = 2.3 Hz, 1H), 2.06–1.95 (m, 1H), 1.95–1.85 (m, 1H), 1.69 (ddd, <i>J</i> = 14.8, 11.2, 4.0 Hz, 1H), 1.58 (ddd, <i>J</i> = 16.3, 12.3, 4.7 Hz, 2H), 1.52fz (m, 1H), 1.49 (s, 3H), 1.31 (s, 3H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 172.9, 156.1, 144.0, 143.8, 141.4, 127.8, 127.2, 125.2, 125.1, 120.1, 112.4, 110.1, 110.1, 85.6, 85.1, 84.5, 71.4, 67.2, 55.3, 53.8, 52.6, 47.3, 40.4, 31.1, 30.5, 28.7, 26.6, 25.1. HRMS (ESI+): calcd for [C<sub>32</sub>H<sub>37</sub>N<sub>1</sub>O<sub>8</sub>Na]<sup>+</sup> 586.2411, measured 586.2407, Δ 0.8 ppm.</div></div><div id="sec4_6_45" class="NLM_sec NLM_sec_level_3"><div id="ac_i107" class="anchor-spacer"></div><h4 class="article-section__title" id="_i107"> (3<i>R</i>,4<i>R</i>,5<i>R</i>)-5-((2<i>S</i>,5<i>S</i>)-5-((((9<i>H</i>-Fluoren-9-yl)methoxy)carbonyl)amino)-2-ethynyl-6-methoxy-6-oxohexyl)tetrahydrofuran-2,3,4-triyl Triacetate (<b>S49</b>)</h4><div class="NLM_p last">To a vial charged with acetonide <b>S48</b> (43 mg, 76 μmol) was added a 4:1 mixture of acetic acid and water (3.5 mL) at RT. The resulting mixture was stirred at 80 °C for 10 h. Volatiles were removed in vacuo, and the residue was dried by azeotropic distillation with benzene (2 × 1.0 mL). Crude material was used directly in the next step without further purification. To a solution of the crude triol in CH<sub>2</sub>Cl<sub>2</sub> (3.5 mL) at RT were added pyridine (60 μL, 0.76 mmol, 10 equiv), Ac<sub>2</sub>O (70 μL, 0.76 mmol, 10 equiv), and DMAP (7 mg, 53 μmol, 0.70 equiv) sequentially. The resulting mixture was stirred at RT for 2 h, and volatiles were removed in vacuo. Crude material was purified by silica gel chromatography (2,2,4-trimethylpentane/EtOAc, 5 → 60%) to afford <b>S49</b> (42 mg, 66 μmol, 87% over 2 steps) as a 1:1 mixture of diastereomers as a colorless oil. <i>R</i><sub><i>f</i></sub> = 0.30/0.38 (hexanes/EtOAc, 50:50). FTIR (thin film), ν<sub>max</sub> (cm<sup>–1</sup>): 3355, 3294, 2954, 2924, 2853, 1744, 1722, 1526, 1451, 1372, 1218, 1107, 1050, 1012. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): δ 7.78–7.75 (m, 4H), 7.63–7.58 (m, 4H), 7.42–7.38 (m, 4H), 7.34–7.29 (m, 4H), 6.36 (d, <i>J</i> = 4.6 Hz, 1H), 6.14 (d, <i>J</i> = 1.0 Hz, 1H), 5.38 (br d, <i>J</i> = 8.5 Hz, 1H), 5.34 (br d, <i>J</i> = 8.4 Hz, 1H), 5.32 (dd, <i>J</i> = 4.8, 1.0 Hz, 1H), 5.23 (dd, <i>J</i> = 6.8, 4.6 Hz, 1H), 5.18 (dd, <i>J</i> = 7.1, 4.8 Hz, 1H), 5.07 (dd, <i>J</i> = 6.8, 3.6 Hz, 1H), 4.46 (app dt, <i>J</i> = 10.2, 3.6 Hz, 1H), 4.42–4.36 (m, 7H), 4.23 (app t, <i>J</i> = 7.0 Hz, 2H), 3.76 (s, 6H), 2.69–2.57 (m, 2H), 2.13 (d, <i>J</i> = 2.4 Hz, 1H), 2.12 (d, <i>J</i> = 2.4 Hz, 1H), 2.12 (s, 3H), 2.11 (s, 3H), 2.10 (s, 3H), 2.07 (s, 6H), 2.06 (s, 3H), 2.00–1.75 (m, 6H), 1.70–1.43 (m, 6H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 172.9, 172.8, 170.2, 170.0, 169.8, 169.6, 169.5, 169.3, 156.1, 156.0, 144.0, 144.0, 143.9, 143.8, 141.4, 141.4, 127.8, 127.2, 125.2, 120.1, 120.1, 98.5, 93.9, 85.3, 85.1, 81.4, 79.9, 74.6, 73.9, 72.6, 71.5, 71.3, 69.9, 67.1, 53.8, 53.7, 52.6, 47.3, 40.1, 39.2, 31.0, 30.4, 30.3, 28.6, 28.2, 21.2, 21.2, 20.8, 20.7, 20.7, 20.4. HRMS (ESI+): calcd for [C<sub>34</sub>H<sub>37</sub>N<sub>1</sub>O<sub>11</sub>Na]<sup>+</sup> 658.2259, measured 658.2248, Δ 1.7 ppm.</div></div><div id="sec4_6_46" class="NLM_sec NLM_sec_level_3"><div id="ac_i108" class="anchor-spacer"></div><h4 class="article-section__title" id="_i108"> (2<i>R</i>,3<i>R</i>,4<i>R</i>,5<i>R</i>)-2-((2<i>S</i>,5<i>S</i>)-5-((((9<i>H</i>-Fluoren-9-yl)methoxy)carbonyl)amino)-2-ethynyl-6-methoxy-6-oxohexyl)-5-(6-benzamido-9<i>H</i>-purin-9-yl)tetrahydrofuran-3,4-diyl Diacetate (<b>S50</b>)</h4><div class="NLM_p last">To a suspension of <i>N</i><sup>6</sup>-benzoyladenine (63 mg, 0.26 mmol, 4.0 equiv) in propionitrile (dried over 4 Å MS, 1.5 mL) at RT was added <i>N</i>,<i>O</i>-bis(trimethylsilyl)acetamide (80 μL, 0.33 mmol, 5.0 equiv). The resulting mixture was stirred at 80 °C for 10 min, upon which complete solubilization of <i>N</i><sup>6</sup>-benzoyladenine was observed. To a separate flask charged with triacetate <b>S49</b> (dried by azeotropic distillation with benzene (4×), 42 mg, 66 μmol) and 2,6-di-<i>tert</i>-butyl-4-methylpyridinium triflate (12 mg, 33 μmol, 50 mol %) was added propionitrile (1.5 mL). The resulting mixture was stirred at RT for 5 min until a clear solution was obtained, and the solution of silylated <i>N</i><sup>6</sup>-benzoyladenine was added. The reaction mixture was stirred at 90 °C for 4 h, cooled to RT, and the volatiles were removed in vacuo. Crude material was purified by silica gel chromatography (2,2,4-trimethylpentane/EtOAc, 30 → 100%) to afford nucleoside <b>S50</b> (44 mg, 54 μmol, 82%) as a colorless oil; <i>R</i><sub><i>f</i></sub> = 0.49 (EtOAc). FTIR (thin film), ν<sub>max</sub> (cm<sup>–1</sup>): 3300, 2953, 2926, 2854, 1745, 1704, 1610, 1582, 1510, 1452, 1241, 1217, 1098, 1073, 1047. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): δ 8.79 (s, 1H), 8.13 (s, 1H), 8.02–7.97 (m, 2H), 7.76–7.72 (m, 2H), 7.62–7.57 (m, 3H), 7.53–7.49 (m, 2H), 7.40–7.35 (m, 2H), 7.31–7.27 (m, 2H), 6.13 (br d, <i>J</i> = 5.5 Hz, 1H), 6.05 (br t, <i>J</i> = 5.5 Hz, 1H), 5.55 (br t, <i>J</i> = 5.0 Hz, 1H), 5.35 (br d, <i>J</i> = 8.4 Hz, 1H), 4.54–4.48 (m, 1H), 4.46–4.35 (m, 3H), 4.23 (br t, <i>J</i> = 7.1 Hz, 1H), 3.74 (s, 3H), 2.67–2.58 (m, 1H), 2.16 (s, 3H), 2.15 (d, <i>J</i> = 2.4 Hz, 1H), 2.12–2.05 (m, 1H), 2.07 (s, 3H), 2.00–1.92 (m, 1H), 1.92–1.80 (m, 2H), 1.64–1.45 (m, 2H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 172.8, 169.8, 169.6, 164.8, 156.1, 152.8, 151.7, 149.8, 144.0, 143.8, 142.3, 141.4, 133.5, 133.0, 129.0, 128.0, 127.8, 127.2, 125.2, 123.9, 120.1, 87.1, 85.1, 80.9, 73.7, 73.0, 71.6, 67.1, 53.6, 52.6, 47.3, 38.5, 31.0, 30.5, 28.0, 20.8, 20.5. HRMS (ESI+): calcd for [C<sub>44</sub>H<sub>43</sub>N<sub>6</sub>O<sub>10</sub>]<sup>+</sup> 815.3035, measured 815.3028, Δ 0.9 ppm.</div></div><div id="sec4_6_47" class="NLM_sec NLM_sec_level_3"><div id="ac_i109" class="anchor-spacer"></div><h4 class="article-section__title" id="_i109"> (2<i>R</i>,3<i>R</i>,4<i>R</i>,5<i>R</i>)-2-((2<i>S</i>,5<i>S</i>)-5-((((9<i>H</i>-Fluoren-9-yl)methoxy)carbonyl)amino)-2-((3-carbamoylphenyl)ethynyl)-6-methoxy-6-oxohexyl)-5-(6-benzamido-9<i>H</i>-purin-9-yl)tetrahydrofuran-3,4-diyl Diacetate (<b>S51</b>)</h4><div class="NLM_p last">To a vial charged with <b>S50</b> (61 mg, 75 μmol) were added 2-iodobenzamide (46 mg, 0.19 mmol, 2.5 equiv), CuI (4 mg, 0.02 mmol, 25 mol %), and Pd(PPh<sub>3</sub>)<sub>4</sub> (4 mg, 4 μmol, 5 mol %). The vial headspace was purged with nitrogen for 10 min and a 5:1:1 mixture of toluene, DMF, and <i>i</i>-Pr<sub>2</sub>NEt (degassed by sparging with nitrogen for 15 min, 2.5 mL) was added at RT. The resulting mixture was stirred at 60 °C for 2 h. The solution was allowed to cool to RT, and volatiles were removed in vacuo. Crude material was purified by silica gel chromatography (EtOAc/<i>i</i>-PrOH, 0 → 25%) to afford <b>S51</b> (68 mg, 73 μmol, 97%) as a light-yellow oil. FTIR (thin film), ν<sub>max</sub> (cm<sup>–1</sup>): 3357, 3070, 2951, 2928, 1746, 1703, 1669, 1613, 1582, 1520, 1452, 1376, 1245, 1219, 1103, 1080, 1050. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): δ 8.78 (s, 1H), 8.36 (s, 1H), 8.01 (br d, <i>J</i> = 7.7 Hz, 2H), 7.90–7.86 (m, 1H), 7.77 (dt, <i>J</i> = 7.9, 1.5 Hz, 1H), 7.74–7.69 (m, 2H), 7.63–7.58 (m, 1H), 7.57–7.52 (m, 3H), 7.52–7.48 (m, 2H), 7.39–7.33 (m, 3H), 7.28–7.24 (m, 2H), 6.92 (br s, 1H), 6.82 (br s, 1H), 6.17 (br d, <i>J</i> = 5.8 Hz, 1H), 6.10 (br t, <i>J</i> = 5.8 Hz, 1H), 5.71–5.65 (m, 1H), 5.51 (br d, <i>J</i> = 8.3 Hz, 1H), 4.56–4.50 (m, 1H), 4.49–4.41 (m, 1H), 4.41–4.32 (m, 2H), 4.18 (br t, <i>J</i> = 7.0 Hz, 1H), 3.75 (s, 3H), 2.89–2.80 (m, 1H), 2.16 (s, 3H), 2.19–2.12 (m, 1H), 2.06 (s, 3H), 2.12–2.02 (m, 2H), 1.99–1.89 (m, 1H), 1.74–1.64 (m, 1H), 1.64–1.54 (m, 1H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 172.8, 170.3, 170.2, 169.8, 165.6, 156.2, 152.0, 151.8, 149.4, 143.9, 143.7, 143.3, 141.4, 135.2, 133.5, 132.6, 131.1, 129.0, 128.9, 128.4, 127.9, 127.9, 127.4, 127.2, 127.2, 125.1, 123.9, 123.3, 120.1, 92.0, 87.1, 82.9, 81.7, 73.8, 72.9, 67.2, 53.6, 52.8, 47.2, 38.6, 31.1, 30.5, 28.8, 20.8, 20.5. HRMS (ESI+): calcd for [C<sub>51</sub>H<sub>47</sub>N<sub>7</sub>O<sub>11</sub>Na]<sup>+</sup> 956.3226, measured 956.3214, Δ 1.2 ppm.</div></div><div id="sec4_6_48" class="NLM_sec NLM_sec_level_3"><div id="ac_i110" class="anchor-spacer"></div><h4 class="article-section__title" id="_i110"> (<i>S</i>)-3-(((3a<i>R</i>,4<i>R</i>,6<i>R</i>,6a<i>R</i>)-6-Methoxy-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)methyl)-5-(triisopropylsilyl)pent-4-yn-1-ol (<b>S52</b>)</h4><div class="NLM_p last">To a solution of <b>5</b> (5.50 g, 12.9 mmol) in MeOH (100 mL) at −78 °C was added NaBH<sub>4</sub> (588 mg, 15.5 mmol, 1.2 equiv) portionwise. The reaction mixture was allowed to warm to 0 °C over 1 h then was warmed to RT and stirred for 1 h. Volatiles were removed in vacuo. The residue was partitioned between CH<sub>2</sub>Cl<sub>2</sub> (30 mL) and a NH<sub>4</sub>Cl satd aq solution (25 mL). The layers were separated, and the aqueous phase was extracted with CH<sub>2</sub>Cl<sub>2</sub> (3 × 25 mL). The combined organic extracts were dried over MgSO<sub>4</sub> and filtered over a silica plug to yield <b>S52</b> as a yellow oil (5.29 g, 12.4 mmol, 95%), which was of suitable purity to be used in the next step without further purification. While the protocol reported here was performed on purified starting material, it was found that direct treatment of the rhodium-catalyzed conjugate alkynylation reaction mixture (<b>5</b>) with NaBH<sub>4</sub> at 0 °C reliably generated alcohol <b>S52</b> in >90% yield over two steps (alkynylation/reduction). <i>R</i><sub><i>f</i></sub> = 0.28 (hexanes/EtOAc, 67:33). FTIR (thin-film), ν<sub>max</sub> (cm<sup>–1</sup>): 3424, 2936, 2891, 2864, 2163, 1463, 1377, 1372, 1347, 1272, 1242, 1211, 1193, 1161, 1104, 1092, 1057, 1019. <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 4.95 (s, 1H), 4.63 (dd, <i>J</i> = 6.0, 1.2 Hz, 1H), 4.59 (d, <i>J</i> = 5.9 Hz, 1H), 4.48 (ddd, <i>J</i> = 10.3, 4.6, 1.2 Hz, 1H), 3.88–3.80 (m, 2H), 3.33 (s, 3H), 2.83 (dddd, <i>J</i> = 10.9, 9.3, 5.2, 4.0 Hz, 1H), 1.80–1.62 (m, 5H), 1.47 (s, 3H), 1.31 (s, 3H), 1.10–1.00 (m, 21H). <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): δ 112.5, 110.6, 109.7, 85.6, 84.5, 83.3, 61.2, 55.2, 41.2, 38.5, 27.4, 26.7, 25.4, 18.8, 11.3. HRMS (ESI+): calcd for [C<sub>23</sub>H<sub>43</sub>O<sub>5</sub>Si<sub>1</sub>]<sup>+</sup> 427.2874, measured 427.2870, Δ 0.9 ppm.</div></div><div id="sec4_6_49" class="NLM_sec NLM_sec_level_3"><div id="ac_i111" class="anchor-spacer"></div><h4 class="article-section__title" id="_i111"> (<i>S</i>)-3-(((3a<i>R</i>,4<i>R</i>,6<i>R</i>,6a<i>R</i>)-6-Methoxy-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)methyl)pent-4-yn-1-ol (<b>S53</b>)</h4><div class="NLM_p last">To a solution of TIPS-alkyne <b>S52</b> (3.81 g, 8.92 mmol) in THF (15 mL) was added TBAF (1.0 M in THF, 15.0 mL, 15.0 mmol, 1.7 equiv) at RT. The reaction mixture was stirred for 3 h, and the volatiles were removed in vacuo. The residue was partitioned between Et<sub>2</sub>O (40 mL) and a NH<sub>4</sub>Cl satd aq solution (40 mL). The layers were separated, and the aqueous phase was extracted with Et<sub>2</sub>O (4 × 40 mL). The combined organic extracts were washed with brine (120 mL), dried over MgSO<sub>4</sub>, filtered, and concentrated. Crude material was purified by silica gel chromatography (hexanes/EtOAc, 0 → 100%) to afford alkyne <b>S53</b> (2.22 g, 8.21 mmol, 92%) as a colorless oil. The pure material crystallized upon storage at −20 °C over 3 weeks. The crystals were of suitable quality to obtain X-ray crystallographic data, which is reported in the <a class="ref internalNav" href="#notes1" aria-label="Supporting Information">Supporting Information</a>. <i>R</i><sub><i>f</i></sub> = 0.31 (cyclohexane/EtOAc, 50:50). FTIR (thin-film), ν<sub>max</sub> (cm<sup>–1</sup>): 3446, 3281, 2988, 2937, 2836, 1142, 1382, 1374, 1274, 1241, 1210, 1161, 1090, 1056. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): δ 4.94 (s, 1H), 4.60 (d, <i>J</i> = 6.0 Hz, 1H), 4.58 (dd, <i>J</i> = 6.0, 0.8 Hz, 1H), 4.50 (ddd, <i>J</i> = 11.2, 4.0, 0.8 Hz, 1H), 3.87–3.76 (m, 2H), 3.33 (s, 3H), 2.86–2.76 (m, 1H), 2.15 (d, <i>J</i> = 2.4 Hz, 1H), 1.79–1.66 (m, 3H), 1.62 (ddd, <i>J</i> = 13.3, 11.1, 4.0 Hz, 1H), 1.48 (s, 3H), 1.31 (s, 3H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 112.4, 110.1, 85.9, 85.7, 85.1, 84.5, 71.2, 60.8, 55.4, 40.5, 38.0, 26.6, 25.8, 25.1. HRMS (ESI+): calcd for [C<sub>14</sub>H<sub>23</sub>O<sub>5</sub>]<sup>+</sup> 271.1540, measured 271.1547, Δ 2.6 ppm.</div></div><div id="sec4_6_50" class="NLM_sec NLM_sec_level_3"><div id="ac_i112" class="anchor-spacer"></div><h4 class="article-section__title" id="_i112"> <i>N</i>-((<i>S</i>)-3-(((3a<i>R</i>,4<i>R</i>,6<i>R</i>,6a<i>R</i>)-6-Methoxy-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)methyl)pent-4-yn-1-yl)-2-nitrobenzenesulfonamide (<b>S54</b>)</h4><div class="NLM_p last">To a solution of alcohol <b>S53</b> (550 mg, 2.04 mmol), DMEAD<a onclick="showRef(event, 'ref76'); return false;" href="javascript:void(0);" class="ref ref76">(76)</a> (1.19 g, 5.09 mmol, 2.5 equiv), and PPh<sub>3</sub> (1.39 g, 5.29 mmol, 2.6 equiv) in a 4:1 mixture of toluene and THF (110 mL) stirred at 0 °C over activated 4 Å MS (1.65 g) was added a solution of 2-NsNH<sub>2</sub> (1.44 g, 7.12 mmol, 3.5 equiv) in THF (22 mL, prepared over 1.65 g of activated 4 Å MS) over 5 min. The reaction mixture was allowed to warm gradually to RT and was stirred for 72 h. The media was filtered, and the volatiles were removed in vacuo. Crude material was purified by silica gel chromatography (hexanes/EtOAc, 0 → 100%) to afford <b>S54</b> (496 mg, 1.09 mmol, 54%) as a white amorphous solid and unreacted <b>S53</b> (81 mg, 0.30 mmol, 15%). <i>R</i><sub><i>f</i></sub> = 0.27 (hexanes/EtOAc, 50:50). FTIR (thin-film), ν<sub>max</sub> (cm<sup>–1</sup>): 3289, 2989, 2936, 2836, 1540, 1442, 1415, 1363, 1343, 1300, 1274, 1241, 1210, 1192, 1162, 1087, 1056. <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 8.16–8.11 (m, 1H), 7.89–7.83 (m, 1H), 7.78–7.70 (m, 2H), 5.50 (t, <i>J</i> = 6.1 Hz, 1H), 4.93 (s, 1H), 4.59 (d, <i>J</i> = 5.9 Hz, 1H), 4.54 (dd, <i>J</i> = 6.0, 0.9 Hz, 1H), 4.43 (ddd, <i>J</i> = 11.1, 3.9, 0.9 Hz, 1H), 3.32 (s, 3H), 3.30–3.26 (m, 2H), 2.71 (dddd, <i>J</i> = 14.6, 8.9, 4.5, 2.9 Hz, 1H), 2.13 (d, <i>J</i> = 2.4 Hz, 1H), 1.77 (dtd, <i>J</i> = 13.2, 7.5, 4.8 Hz, 1H), 1.71–1.63 (m, 2H), 1.58 (ddd, <i>J</i> = 13.3, 11.0, 4.0 Hz, 1H), 1.47 (s, 3H), 1.30 (s, 3H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 148.1, 133.7, 133.6, 132.9, 131.1, 125.5, 112.4, 110.1, 85.5, 84.8, 84.7, 84.4, 71.9, 55.4, 41.9, 40.3, 35.2, 26.5, 26.5, 25.0. HRMS (ESI+): calcd for [C<sub>20</sub>H<sub>26</sub>N<sub>2</sub>O<sub>8</sub>S<sub>1</sub>Na]<sup>+</sup> 477.1302, measured 477.1287, Δ 3.2 ppm.</div></div><div id="sec4_6_51" class="NLM_sec NLM_sec_level_3"><div id="ac_i113" class="anchor-spacer"></div><h4 class="article-section__title" id="_i113"> (3<i>R</i>,4<i>R</i>,5<i>R</i>)-5-((<i>S</i>)-2-(2-((4-Nitrophenyl)sulfonamido)ethyl)but-3-yn-1-yl)tetrahydrofuran-2,3,4-triyl Triacetate (<b>S55</b>)</h4><div class="NLM_p last">To a solution of acetonide <b>S54</b> (210 mg, 0.462 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (0.80 mL) at 0 °C was added a 4:1 mixture of TFA and water (4.0 mL). The resulting mixture was stirred at RT for 15 h. Volatiles were removed in vacuo, and the residue was dried by azeotropic distillation with benzene (2 × 1.0 mL). Crude material was used directly in the next step without further purification. To a solution of the crude triol in CH<sub>2</sub>Cl<sub>2</sub> (4.8 mL) at RT were added pyridine (0.37 mL, 4.6 mmol, 10 equiv), Ac<sub>2</sub>O (0.44 mL, 4.6 mmol, 10 equiv), and DMAP (11 mg, 92 μmol, 20 mol %) sequentially. The resulting mixture was stirred at RT for 1 h, and volatiles were removed in vacuo. Crude material was purified by silica gel chromatography (2,2,4-trimethylpentane/EtOAc, 15 → 75%) to afford <b>S55</b> (70 mg, 0.13 mmol, 29% over 2 steps) as a 1.8:1 mixture of diastereomers as a colorless oil. <i>R</i><sub><i>f</i></sub> = 0.31/0.40 (hexanes/EtOAc, 40:60). FTIR (thin film), ν<sub>max</sub> (cm<sup>–1</sup>): 3288, 2925, 2853, 1747, 1541, 1424, 1368, 1222, 1167, 1124, 1082, 1058, 1012. <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 8.16–8.11 (m, 2.8H), 7.89–7.84 (m, 2.8H), 7.79–7.71 (m, 5.6H), 6.32 (d, <i>J</i> = 4.6 Hz, 1.0H), 6.11 (d, <i>J</i> = 1.0 Hz, 1.8H), 5.55 (app t, <i>J</i> = 6.1 Hz, 1.0H), 5.52 (app t, <i>J</i> = 6.1 Hz, 1.8H), 5.29 (dd, <i>J</i> = 4.9, 1.0 Hz, 1.8H), 5.19 (dd, <i>J</i> = 6.7, 4.6 Hz, 1.0H), 5.16 (dd, <i>J</i> = 6.9, 4.9 Hz, 1.8H), 5.02 (dd, <i>J</i> = 6.7, 3.7 Hz, 1.0H), 4.40 (ddd, <i>J</i> = 10.3, 3.7, 3.3 Hz, 1.0H), 4.34 (ddd, <i>J</i> = 10.2, 6.9, 3.1 Hz, 1.8H), 3.33–3.22 (m, 5.6H), 2.70–2.61 (m, 2.8H), 2.14 (d, <i>J</i> = 2.4 Hz, 2.8H), 2.12 (s, 5.4H), 2.11 (s, 3.0H), 2.10 (s, 3.0H), 2.08 (s, 5.4H), 2.06 (s, 5.4H), 2.06 (s, 3.0H), 1.86–1.73 (m, 5.6H), 1.71–1.60 (m, 4.6H), 1.56 (ddd, <i>J</i> = 14.0, 10.3, 4.2 Hz, 1.0H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 170.2, 170.0, 169.8, 169.6, 169.5, 169.2, 148.2, 133.8, 133.7, 133.7, 132.9, 132.9, 131.2, 131.2, 125.6, 125.6, 98.4, 93.8, 84.5, 84.3, 81.2, 79.6, 74.6, 73.8, 72.5, 72.1, 72.0, 69.9, 41.9, 40.1, 39.0, 35.2, 35.0, 26.6, 26.2, 21.2, 21.2, 20.8, 20.7, 20.7, 20.4. HRMS (ESI+): calcd for [C<sub>22</sub>H<sub>26</sub>N<sub>2</sub>O<sub>11</sub>S<sub>1</sub>Na]<sup>+</sup> 549.1150, measured 549.1138, Δ 2.1 ppm.</div></div><div id="sec4_6_52" class="NLM_sec NLM_sec_level_3"><div id="ac_i114" class="anchor-spacer"></div><h4 class="article-section__title" id="_i114"> (2<i>R</i>,3<i>R</i>,4<i>R</i>,5<i>R</i>)-2-(6-Benzamido-9<i>H</i>-purin-9-yl)-5-((<i>S</i>)-2-(2-((4-nitrophenyl)sulfonamido)ethyl)but-3-yn-1-yl)tetrahydrofuran-3,4-diyl Diacetate (<b>S56</b>)</h4><div class="NLM_p last">To a suspension of <i>N</i><sup>6</sup>-benzoyladenine (93 mg, 0.39 mmol, 2.15 equiv) in propionitrile (dried over 4 Å MS, 4.5 mL) at RT was added <i>N</i>,<i>O</i>-bis(trimethylsilyl)acetamide (121 μL, 0.497 mmol, 2.75 equiv). The resulting mixture was stirred at 80 °C for 10 min, upon which complete solubilization of <i>N</i><sup>6</sup>-benzoyladenine was observed. To a separate flask charged with triacetate <b>S55</b> (dried by azeotropic distillation with benzene (4×), 95 mg, 0.18 mmol) and 2,6-di-<i>tert</i>-butyl-4-methylpyridinium triflate (13 mg, 36 μmol, 20 mol %) was added propionitrile (4.5 mL). The resulting mixture was stirred at RT for 5 min until a clear solution was obtained, and the solution of silylated <i>N</i><sup>6</sup>-benzoyladenine was added. The reaction mixture was stirred at 95 °C for 30 h, cooled to RT, and the volatiles were removed in vacuo. Crude material was purified by silica gel chromatography (toluene/MeCN, 0 → 50%) to afford nucleoside <b>S56</b> (120 mg, 0.170 mmol, 94%) as a white solid. <i>R</i><sub><i>f</i></sub> = 0.36 (EtOAc). FTIR (thin-film), ν<sub>max</sub> (cm<sup>–1</sup>): 3295, 3097, 2945, 1746, 1699, 1610, 1582, 1540, 1509, 1485, 1455, 1423, 1365, 1338, 1298, 1240, 1218, 1164, 1074, 1047. <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 9.09 (s, 1H), 8.76 (s, 1H), 8.10 (dd, <i>J</i> = 7.7, 1.6 Hz, 1H), 8.07 (s, 1H), 8.02 (dt, <i>J</i> = 7.1, 1.4 Hz, 2H), 7.80 (dd, <i>J</i> = 7.6, 1.6 Hz, 1H), 7.72 (td, <i>J</i> = 7.6, 1.6 Hz, 1H), 7.68 (td, <i>J</i> = 7.6, 1.6 Hz, 1H), 7.63–7.59 (m, 1H), 7.55–7.49 (m, 2H), 6.10 (d, <i>J</i> = 4.7 Hz, 2H), 5.55 (q, <i>J</i> = 6.2, 5.4 Hz, 2H), 4.46 (ddd, <i>J</i> = 11.0, 4.5, 2.8 Hz, 1H), 3.25 (dt, <i>J</i> = 7.5, 6.2 Hz, 2H), 2.62 (dddq, <i>J</i> = 11.2, 9.0, 4.6, 2.5 Hz, 1H), 2.17 (d, <i>J</i> = 2.4 Hz, 1H), 2.16 (s, 3H), 2.07–2.12 (m, 1H), 2.07 (s, 3H), 1.85 (ddd, <i>J</i> = 13.8, 11.2, 2.9 Hz, 1H), 1.82–1.73 (m, 2H), 1.69 (ddt, <i>J</i> = 13.5, 9.9, 6.3 Hz, 1H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 169.9, 169.6, 164.8, 152.9, 151.6, 150.0, 148.1, 142.5, 133.8, 133.7, 133.6, 133.0, 133.0, 131.2, 129.0, 128.1, 125.5, 124.1, 87.2, 84.4, 80.8, 77.4, 73.7, 72.9, 72.2, 41.9, 38.3, 35.1, 26.1, 20.8, 20.6. HRMS (ESI+): calcd for [C<sub>32</sub>H<sub>32</sub>N<sub>7</sub>O<sub>10</sub>S<sub>1</sub>]<sup>+</sup> 706.1926, measured 706.1922, Δ 0.5 ppm.</div></div><div id="sec4_6_53" class="NLM_sec NLM_sec_level_3"><div id="ac_i115" class="anchor-spacer"></div><h4 class="article-section__title" id="_i115"> (2<i>R</i>,3<i>R</i>,4<i>R</i>,5<i>R</i>)-2-(6-Benzamido-9<i>H</i>-purin-9-yl)-5-((<i>S</i>)-4-(3-carbamoylphenyl)-2-(2-((4-nitrophenyl)sulfonamido)ethyl)but-3-yn-1-yl)tetrahydrofuran-3,4-diyl Diacetate (<b>S57</b>)</h4><div class="NLM_p last">To a vial charged with <b>S56</b> (54 mg, 77 μmol) were added 3-iodobenzamide (29 mg, 0.12 mmol, 1.5 equiv), CuI (3 mg, 0.02 mmol, 20 mol %), and Pd(PPh<sub>3</sub>)<sub>4</sub> (4 mg, 4 μmol, 5 mol %). The vial headspace was purged with nitrogen for 10 min, and a 5:1:1 mixture of toluene, DMF, and <i>i</i>-Pr<sub>2</sub>NEt (degassed by sparging with nitrogen for 15 min, 3.0 mL) was added at RT. The resulting mixture was stirred at 70 °C for 4 h. The solution was allowed to cool to RT, and volatiles were removed in vacuo. Crude material was purified by silica gel chromatography (EtOAc/<i>i</i>-PrOH, 0 → 15%) to afford benzamide <b>S57</b> (50 mg, 60 μmol, 78%) as a white amorphous solid. <i>R</i><sub><i>f</i></sub> = 0.16 (EtOAc/<i>i</i>-PrOH, 91:9). FTIR (thin-film), ν<sub>max</sub> (cm<sup>–1</sup>): 2926, 1746, 1667, 1610, 1579, 1540, 1511, 1484, 1455, 1366, 1339, 1298, 1241, 1215, 1163, 1073, 1047. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): δ 9.13 (s, 1H), 8.76 (s, 1H), 8.11 (s, 1H), 8.10–8.07 (m, 1H), 8.05–8.00 (m, 2H), 7.87–7.86 (m, 1H), 7.80 (ddd, <i>J</i> = 7.8, 1.9, 1.2 Hz, 1H), 7.79–7.76 (m, 1H), 7.71–7.64 (m, 2H), 7.63–7.58 (m, 1H), 7.55–7.47 (m, 3H), 7.38 (td, <i>J</i> = 7.7, 0.6 Hz, 1H), 6.56 (br s, 1H), 6.17 (t, <i>J</i> = 5.5 Hz, 1H), 6.13 (d, <i>J</i> = 5.8 Hz, 1H), 5.84 (br s, 1H), 5.79 (t, <i>J</i> = 6.1 Hz, 1H), 5.64 (dd, <i>J</i> = 5.3, 4.0 Hz, 1H), 4.50 (dt, <i>J</i> = 10.3, 3.7 Hz, 1H), 3.38–3.29 (m, 1H), 3.32–3.23 (m, 1H), 2.87 (tt, <i>J</i> = 10.2, 4.2 Hz, 1H), 2.23–2.10 (m, 1H), 2.16 (s, 3H), 2.06 (s, 3H), 1.98 (ddd, <i>J</i> = 13.5, 11.1, 3.6 Hz, 1H), 1.92–1.73 (m, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>): δ 170.1, 169.7, 168.6, 164.9, 152.8, 151.7, 150.0, 148.1, 142.6, 134.7, 133.8, 133.6, 133.6, 133.5, 133.0, 133.0, 131.2, 130.9, 129.0, 128.8, 128.1, 127.5, 125.5, 124.3, 123.4, 91.1, 87.1, 83.5, 81.2, 77.4, 77.2, 76.9, 73.8, 72.7, 42.1, 38.5, 35.3, 27.0, 20.8, 20.6. HRMS (ESI+): calcd for [C<sub>39</sub>H<sub>37</sub>N<sub>8</sub>O<sub>11</sub>S<sub>1</sub>]<sup>+</sup> 825.2297, measured 825.2300, Δ 0.4 ppm.</div></div><div id="sec4_6_54" class="NLM_sec NLM_sec_level_3"><div id="ac_i116" class="anchor-spacer"></div><h4 class="article-section__title" id="_i116"> (2<i>R</i>,3<i>R</i>,4<i>R</i>,5<i>R</i>)-2-(6-Benzamido-9<i>H</i>-purin-9-yl)-5-((<i>S</i>)-4-(3-carbamoylphenyl)-2-(2-ureidoethyl)but-3-yn-1-yl)tetrahydrofuran-3,4-diyl Diacetate (<b>S58</b>)</h4><div class="NLM_p last">To a solution of <b>S57</b> (112 mg, 0.136 mmol) in a 2:1 mixture of MeCN and DMF (10.5 mL) at 0 °C were added thiophenol (56 μL, 0.54 mmol, 4.0 equiv) and Cs<sub>2</sub>CO<sub>3</sub> (354 mg, 1.09 mmol, 8.0 equiv) sequentially. The resulting mixture was stirred at RT for 1 h and was filtered twice through a nylon syringe filter (13 mm, 0.22 μm). Volatiles were removed in vacuo. The crude amine was used directly in the next step without further purification. To a solution of the crude amine in a 1:1 mixture of CH<sub>2</sub>Cl<sub>2</sub> and <i>i</i>-PrOH (13.5 mL) at RT, was added (trimethylsilyl)isocyanate (1.29 mL, 9.51 mmol, 70 equiv). The resulting mixture was stirred at RT for 17 h, and the volatiles were removed in vacuo. Crude material was purified by preparative HPLC (5% → 95% B over 40 min) to afford <b>S58</b> (24 mg, 35 μmol, 26% over two steps) as a white fluffy solid. <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD): δ 8.78 (s, 1H), 8.58 (s, 1H), 8.13–8.08 (m, 2H), 7.94 (t, <i>J</i> = 1.8 Hz, 1H), 7.83 (dt, <i>J</i> = 7.9, 1.6 Hz, 1H), 7.71–7.64 (m, 1H), 7.61–7.54 (m, 3H), 7.43 (t, <i>J</i> = 7.8 Hz, 1H), 6.35 (d, <i>J</i> = 5.3 Hz, 1H), 6.20 (t, <i>J</i> = 5.5 Hz, 1H), 5.68 (t, <i>J</i> = 5.2 Hz, 1H), 4.60 (ddd, <i>J</i> = 10.5, 4.8, 3.1 Hz, 1H), 3.40 (ddd, <i>J</i> = 13.1, 7.4, 5.3 Hz, 1H), 3.28 (dt, <i>J</i> = 13.5, 7.4 Hz, 1H), 2.91 (ddt, <i>J</i> = 11.0, 9.3, 4.5 Hz, 1H), 2.28 (ddd, <i>J</i> = 14.1, 10.5, 4.0 Hz, 1H), 2.18 (s, 3H), 2.07 (s, 3H), 2.03 (ddd, <i>J</i> = 14.0, 11.2, 3.2 Hz, 1H), 1.82 (dtd, <i>J</i> = 12.7, 7.6, 5.1 Hz, 1H), 1.74 (dddd, <i>J</i> = 13.0, 9.5, 7.3, 5.4 Hz, 1H). <sup>13</sup>C NMR (126 MHz, CD<sub>3</sub>OD): δ 171.6, 171.4, 171.3, 168.2, 162.1, 153.5, 153.1, 151.4, 145.2, 135.7, 135.3, 134.9, 133.9, 131.8, 129.8, 129.7, 129.5, 128.1, 125.7, 125.2, 92.8, 88.5, 83.5, 82.1, 75.2, 74.2, 39.6, 36.9, 28.0, 20.5, 20.3. HRMS (ESI+): calcd for [C<sub>34</sub>H<sub>35</sub>N<sub>8</sub>O<sub>8</sub>]<sup>+</sup> 683.2572, measured 683.2598, Δ 3.8 ppm.</div></div><div id="sec4_6_55" class="NLM_sec NLM_sec_level_3"><div id="ac_i117" class="anchor-spacer"></div><h4 class="article-section__title" id="_i117"> (<i>S</i>)-4-(((3a<i>R</i>,4<i>R</i>,6<i>R</i>,6a<i>R</i>)-6-Methoxy-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)methyl)-6-(triisopropylsilyl)hex-5-ynenitrile (<b>S59</b>)</h4><div class="NLM_p last">To a solution of <b>S52</b> (0.54 g, 1.3 mmol) in benzene (44 mL) at RT were added acetone cyanohydrin (0.17 mL, 1.9 mmol, 1.5 equiv), P<i>n</i>-Bu<sub>3</sub> (0.47 mL, 1.9 mmol, 1.5 equiv), and a solution of TMAD (0.33 g, 1.9 mmol, 1.5 equiv) in benzene (20 mL). The resulting mixture, which quickly became thick and cloudy, was stirred at RT for 2 h, and volatiles were removed in vacuo. Crude material was purified by silica gel chromatography (hexanes/EtOAc, 2 → 40%) to afford <b>S59</b> (0.49 g, 1.1 mmol, 89%) as a colorless oil. <i>R</i><sub><i>f</i></sub> = 0.44 (hexanes/EtOAc, 70:30). FTIR (thin film), ν<sub>max</sub> (cm<sup>–1</sup>): 2941, 2865, 2247, 2166, 1463, 1381, 1371, 1241, 1210, 1192, 1160, 1092, 1057, 1018. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): δ 4.94 (s, 1H), 4.61 (dd, <i>J</i> = 6.0, 1.0 Hz, 1H), 4.58 (d, <i>J</i> = 6.0 Hz, 1H), 4.45 (ddd, <i>J</i> = 10.6, 4.5, 1.0 Hz, 1H), 3.34 (s, 3H), 2.81 (dddd, <i>J</i> = 10.6, 10.2, 4.6, 4.2 Hz, 1H), 2.59 (ddd, <i>J</i> = 16.9, 8.0, 5.6 Hz, 1H), 2.54 (ddd, <i>J</i> = 16.9, 8.0, 7.8 Hz, 1H), 1.86 (app dtd, <i>J</i> = 13.2, 8.0, 4.6 Hz, 1H), 1.74 (dddd, <i>J</i> = 13.2, 10.2, 7.8, 5.6 Hz, 1H), 1.69 (ddd, <i>J</i> = 13.2, 10.6, 4.2 Hz, 1H), 1.64 (ddd, <i>J</i> = 13.2, 10.6, 4.5 Hz, 1H), 1.46 (s, 3H), 1.30 (s, 3H), 1.07–1.03 (m, 21H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): δ 119.4, 112.6, 109.9, 108.0, 85.5, 85.3, 84.8, 84.4, 55.4, 40.8, 31.6, 29.8, 26.7, 25.4, 18.8, 15.4, 11.3. HRMS (ESI+): calcd for [C<sub>24</sub>H<sub>41</sub>N<sub>1</sub>O<sub>4</sub>Si<sub>1</sub>Na]<sup>+</sup> 458.2697, measured 458.2691, Δ 1.2 ppm.</div></div><div id="sec4_6_56" class="NLM_sec NLM_sec_level_3"><div id="ac_i118" class="anchor-spacer"></div><h4 class="article-section__title" id="_i118"> Methyl (<i>S</i>)-6-(((3a<i>R</i>,4<i>R</i>,6<i>R</i>,6a<i>R</i>)-6-Methoxy-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)methyl)-2-(2,2,2-trifluoroacetamido)-8-(triisopropylsilyl)oct-2-en-7-ynoate (<b>S61</b>)</h4><div class="NLM_p last">To a solution of nitrile <b>S59</b> (0.49 g, 1.1 mmol) in toluene (22 mL) at −78 °C was added <i>i</i>-Bu<sub>2</sub>AlH (1.0 M in hexanes, 1.68 mL, 1.68 mmol, 1.5 equiv) dropwise. The resulting mixture was stirred at −78 °C for 1.3 h. The reaction was quenched by the addition of a potassium sodium tartrate satd aq solution (22 mL), followed by the addition of Et<sub>2</sub>O (15 mL) and brine (15 mL). The biphasic mixture was stirred vigorously at RT for 45 min. The layers were separated, and the aqueous phase was extracted with Et<sub>2</sub>O (3 × 25 mL). The combined organic extracts were dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated. Crude material was filtered through a short pad of silica gel (hexanes/EtOAc, 75:25) to afford crude aldehyde <b>S60</b>, which was used directly in the next step without further purification. To a suspension of NaH (95%, 33 mg, 1.3 mmol, 1.5 equiv) in THF (1.7 mL) at 0 °C was added a solution of phosphonate <b>S2</b> (0.39 g, 1.3 mmol, 1.5 equiv) in THF (2.0 mL) dropwise. The resulting cloudy mixture was stirred at 0 °C for 30 min, and a solution of aldehyde <b>S60</b> (0.38 g, 0.88 mmol) in THF (3.7 mL) was added in one portion. The yellow reaction mixture was stirred at 45 °C for 20 h and cooled to 0 °C, before the addition of an aqueous pH 7 buffer (5 mL) and Et<sub>2</sub>O (5 mL). The layers were separated, and the aqueous phase was extracted with Et<sub>2</sub>O (3 × 10 mL). The combined organic extracts were dried over MgSO<sub>4</sub>, filtered, and concentrated. Crude material was purified by silica gel chromatography (hexanes/EtOAc, 1 → 40%) to afford olefin <b>S61</b> (0.44 g, 0.72 mmol, 82%) as a 1:1 mixture of <i>Z</i> and <i>E</i> isomers as a colorless oil. <i>R</i><sub><i>f</i></sub> = 0.17/0.31 (hexanes/EtOAc, 80:20). FTIR (thin film), ν<sub>max</sub> (cm<sup>–1</sup>): 3305, 2942, 2865, 2164, 1729, 1661, 1541, 1463, 1439, 1382, 1372, 1209, 1159, 1104, 1092, 1060. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): δ 8.30 (s, 1H), 7.60 (s, 1H), 7.37 (app t, <i>J</i> = 7.6 Hz, 1H), 6.86 (app t, <i>J</i> = 7.3 Hz, 1H), 4.94 (s, 1H), 4.94 (s, 1H), 4.63 (dd, <i>J</i> = 6.0, 1.2 Hz, 1H), 4.62 (dd, <i>J</i> = 6.0, 1.2 Hz, 1H), 4.58 (d, <i>J</i> = 6.0 Hz, 1H), 4.58 (d, <i>J</i> = 6.0 Hz, 1H), 4.49–4.43 (m, 2H), 3.87 (s, 3H), 3.80 (s, 3H), 3.32 (s, 3H), 3.32 (s, 3H), 2.89–2.75 (m, 2H), 2.74–2.61 (m, 2H), 2.45–2.36 (m, 1H), 2.36–2.26 (m, 1H), 1.74–1.57 (m, 8H), 1.46 (s, 6H), 1.31 (s, 6H), 1.08–1.00 (m, 42H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 163.9, 163.8, 155.1 (q, <sup>2</sup><i>J</i><sub>C–F</sub> = 38 Hz), 155.0 (q, <sup>2</sup><i>J</i><sub>C–F</sub> = 38 Hz), 140.7, 135.5, 123.6, 122.9, 115.8 (q, <sup>1</sup><i>J</i><sub>C–F</sub> = 288 Hz), 115.6 (q, <sup>1</sup><i>J</i><sub>C–F</sub> = 288 Hz), 112.5, 112.5, 110.1, 109.8, 109.8, 109.8, 85.7, 85.6, 85.6, 84.5, 84.5, 83.4, 83.2, 55.2, 55.2, 53.1, 53.0, 41.2, 41.1, 35.6, 34.1, 30.3, 30.2, 27.2, 26.7, 25.4, 25.4, 18.8, 18.8, 11.4, 11.3. <sup>19</sup>F NMR (471 MHz, CDCl<sub>3</sub>, BTF IStd): δ −76.4, −77.2. HRMS (ESI+): calcd for [C<sub>29</sub>H<sub>46</sub>F<sub>3</sub>N<sub>1</sub>O<sub>7</sub>Si<sub>1</sub>Na]<sup>+</sup> 628.2888, measured 628.2894, Δ 1.0 ppm.</div></div><div id="sec4_6_57" class="NLM_sec NLM_sec_level_3"><div id="ac_i119" class="anchor-spacer"></div><h4 class="article-section__title" id="_i119"> Methyl (2<i>S</i>,6<i>S</i>)-6-(((3a<i>R</i>,4<i>R</i>,6<i>R</i>,6a<i>R</i>)-6-Methoxy-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)methyl)-2-(2,2,2-trifluoroacetamido)-8-(triisopropylsilyl)oct-7-ynoate (<b>S62</b>)</h4><div class="NLM_p last">In a glovebox, [Rh(nbd)<sub>2</sub>]BF<sub>4</sub> (47 mg, 0.13 mmol, 20 mol %) and (<i>S</i>,<i>S</i>)-MeBPE (36 mg, 0.14 mmol, 22 mol %) were placed in two separate vials, capped with a septum, and placed under an atmosphere of nitrogen out of the glovebox. MeOH (3 mL) was added to the vial containing (<i>S</i>,<i>S</i>)-MeBPE, and the resulting colorless solution was transferred to the vial charged with [Rh(nbd)<sub>2</sub>]BF<sub>4</sub>. More MeOH (2 mL) was used for rinsing. The red solution of (<i>S</i>,<i>S</i>)-MeBPE-Rh was added to a solution of olefin <b>S61</b> (0.38 g, 0.63 mmol) in MeOH (16 mL). The flask headspace was evacuated and refilled with hydrogen (3×), and the reaction mixture was stirred vigorously at RT under an atmosphere of hydrogen (balloon) for 2 h. Volatiles were removed in vacuo. Crude material was purified by silica gel chromatography (hexanes/EtOAc, 1 → 60%) to afford protected amino acid <b>S62</b> (0.35 g, 0.57 mmol, 90%) as a colorless oil. <i>R</i><sub><i>f</i></sub> = 0.40 (hexanes/EtOAc, 70:30). FTIR (thin film), ν<sub>max</sub> (cm<sup>–1</sup>): 3317, 2941, 2865, 2165, 1749, 1724, 1549, 1462, 1382, 1208, 1160, 1105, 1094, 1060. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): δ 6.84 (d, <i>J</i> = 7.8 Hz, 1H), 4.94 (s, 1H), 4.65–4.60 (m, 2H), 4.58 (d, <i>J</i> = 5.9 Hz, 1H), 4.44 (ddd, <i>J</i> = 10.1, 4.7, 1.1 Hz, 1H), 3.79 (s, 3H), 3.31 (s, 3H), 2.65–2.58 (m, 1H), 1.98–1.89 (m, 1H), 1.88–1.78 (m, 1H), 1.69–1.54 (m, 3H), 1.51–1.42 (m, 3H), 1.46 (s, 3H), 1.31 (s, 3H), 1.07–1.01 (m, 21H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 171.4, 156.9 (q, <sup>2</sup><i>J</i><sub>C–F</sub> = 38 Hz), 115.7 (q, <sup>1</sup><i>J</i><sub>C–F</sub> = 288 Hz), 112.5, 110.3, 109.8, 85.7, 85.6, 84.5, 82.9, 55.2, 53.1, 52.7, 41.2, 35.3, 31.9, 30.1, 26.7, 25.4, 22.7, 18.8, 11.4. <sup>19</sup>F NMR (376 MHz, CDCl<sub>3</sub>, BTF IStd): δ −76.8. HRMS (ESI+): calcd for [C<sub>29</sub>H<sub>48</sub>F<sub>3</sub>N<sub>1</sub>O<sub>7</sub>Si<sub>1</sub>Na]<sup>+</sup> 630.3044, measured 630.3046, Δ 0.3 ppm.</div></div><div id="sec4_6_58" class="NLM_sec NLM_sec_level_3"><div id="ac_i120" class="anchor-spacer"></div><h4 class="article-section__title" id="_i120"> Methyl (2<i>S</i>,6<i>S</i>)-6-(((3a<i>R</i>,4<i>R</i>,6<i>R</i>,6a<i>R</i>)-6-Methoxy-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)methyl)-2-(2,2,2-trifluoroacetamido)oct-7-ynoate (<b>S63</b>)</h4><div class="NLM_p last">To a solution of TIPS alkyne <b>S62</b> (381 mg, 0.627 mmol) in DMF (3.2 mL) at RT was added a solution of TASF (518 mg, 1.88 mmol, 3.0 equiv) in DMF (3.2 mL). The resulting mixture was stirred at 60 °C for 2.5 h, and volatiles were removed in vacuo. The residue was partitioned between Et<sub>2</sub>O (15 mL) and a 1:1 mixture of brine and 5% LiCl aq solution (15 mL). The layers were separated, and the aqueous phase was extracted with Et<sub>2</sub>O (3 × 15 mL). The combined organic extracts were dried over MgSO<sub>4</sub>, filtered, and concentrated. Crude material was purified by silica gel chromatography (hexanes/EtOAc, 2 → 50%) to afford alkyne <b>S63</b> (250 mg, 0.554 mmol, 88%) as a colorless oil. <i>R</i><sub><i>f</i></sub> = 0.48 (hexanes/EtOAc, 60:40). FTIR (thin film), ν<sub>max</sub> (cm<sup>–1</sup>): 3309, 2937, 2867, 1747, 1717, 1550, 1440, 1374, 1208, 1158, 1089, 1058. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): δ 6.90 (d, <i>J</i> = 7.5 Hz, 1H), 4.93 (s, 1H), 4.63 (ddd, <i>J</i> = 7.5, 7.1, 5.6 Hz, 1H), 4.59 (d, <i>J</i> = 6.0 Hz, 1H), 4.56 (dd, <i>J</i> = 6.0, 0.9 Hz, 1H), 4.46 (ddd, <i>J</i> = 11.0, 4.1, 0.9 Hz, 1H), 3.79 (s, 3H), 3.31 (s, 3H), 2.63–2.55 (m, 1H), 2.10 (d, <i>J</i> = 2.4 Hz, 1H), 1.97–1.89 (m, 1H), 1.80 (dddd, <i>J</i> = 13.8, 10.4, 7.1, 4.6 Hz, 1H), 1.66 (ddd, <i>J</i> = 13.4, 11.0, 4.1 Hz, 1H), 1.60–1.52 (m, 2H), 1.47 (s, 3H), 1.51–1.39 (m, 3H), 1.30 (s, 3H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 171.4, 156.9 (q, <sup>2</sup><i>J</i><sub>C–F</sub> = 38 Hz), 115.7 (q, <sup>1</sup><i>J</i><sub>C–F</sub> = 288 Hz), 112.4, 110.1, 85.6, 85.6, 85.2, 84.5, 71.1, 55.3, 53.1, 52.6, 40.5, 34.4, 31.6, 28.5, 26.6, 25.1, 22.5. <sup>19</sup>F NMR (471 MHz, CDCl<sub>3</sub>, BTF IStd): δ −76.8. HRMS (ESI+): calcd for [C<sub>20</sub>H<sub>28</sub>F<sub>3</sub>N<sub>1</sub>O<sub>7</sub>Na]<sup>+</sup> 474.1710, measured 474.1721, Δ 2.3 ppm.</div></div><div id="sec4_6_59" class="NLM_sec NLM_sec_level_3"><div id="ac_i121" class="anchor-spacer"></div><h4 class="article-section__title" id="_i121"> (3<i>R</i>,4<i>R</i>,5<i>R</i>)-5-((2<i>S</i>,6<i>S</i>)-2-Ethynyl-7-methoxy-7-oxo-6-(2,2,2-trifluoroacetamido)heptyl)tetrahydrofuran-2,3,4-triyl Triacetate (<b>S64</b>)</h4><div class="NLM_p last">To a solution of acetonide <b>S63</b> (125 mg, 0.277 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (1.1 mL) at 0 °C was added a 4:1 mixture of TFA and water (5.5 mL). The resulting mixture was stirred at RT for 5.5 h. Volatiles were removed in vacuo, and the residue was dried by azeotropic distillation with benzene (2 × 1.5 mL). Crude material was used directly in the next step without further purification. To a solution of the crude triol in CH<sub>2</sub>Cl<sub>2</sub> (5.5 mL) at RT were added pyridine (0.22 mL, 2.8 mmol, 10 equiv), Ac<sub>2</sub>O (0.26 mL, 2.8 mmol, 10 equiv), and DMAP (24 mg, 0.19 mmol, 0.70 equiv) sequentially. The resulting mixture was stirred at RT for 18 h, and volatiles were removed in vacuo. Crude material was purified by silica gel chromatography (2,2,4-trimethylpentane/EtOAc, 5 → 70%) to afford triacetate <b>S64</b> (93 mg, 0.18 mmol, 64% over 2 steps) as a 2:1 mixture of diastereomers as a colorless oil. <i>R</i><sub><i>f</i></sub> = 0.23/0.29 (hexanes/EtOAc, 50:50). FTIR (thin film), ν<sub>max</sub> (cm<sup>–1</sup>): 3293, 2954, 2867, 1747, 1723, 1553, 1438, 1372, 1216, 1180, 1162, 1113, 1051, 1012. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): δ 6.96–6.89 (m, 3H), 6.35 (d, <i>J</i> = 4.6 Hz, 1H), 6.12 (d, <i>J</i> = 1.0 Hz, 2H), 5.30 (dd, <i>J</i> = 4.8, 1.0 Hz, 2H), 5.22 (dd, <i>J</i> = 6.8, 4.6 Hz, 1H), 5.17 (dd, <i>J</i> = 7.1, 4.8 Hz, 2H), 5.05 (dd, <i>J</i> = 6.8, 3.8 Hz, 1H), 4.65–4.60 (m, 3H), 4.45 (ddd, <i>J</i> = 10.3, 3.8, 3.3 Hz, 1H), 4.38 (ddd, <i>J</i> = 10.2, 7.1, 3.0 Hz, 2H), 3.79 (s, 6H), 3.79 (s, 3H), 2.63–2.55 (m, 3H), 2.12 (s, 6H), 2.11 (s, 3H), 2.10 (s, 3H), 2.09 (d, <i>J</i> = 2.4 Hz, 2H), 2.09 (d, <i>J</i> = 2.4 Hz, 1H), 2.07 (s, 6H), 2.07 (s, 6H), 2.05 (s, 3H), 1.99–1.89 (m, 3H), 1.85–1.73 (m, 6H), 1.66–1.53 (m, 6H), 1.53–1.37 (m, 9H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 171.4, 170.2, 170.1, 169.9, 169.7, 169.5, 169.3, 156.9 (q, <sup>2</sup><i>J</i><sub>C–F</sub> = 38 Hz), 156.9 (q, <sup>2</sup><i>J</i><sub>C–F</sub> = 38 Hz), 115.7 (q, <sup>1</sup><i>J</i><sub>C–F</sub> = 288 Hz), 98.5, 93.8, 85.3, 85.1, 81.4, 79.8, 74.6, 73.9, 72.6, 71.2, 71.1, 69.9, 53.1, 52.6, 40.3, 39.3, 34.4, 34.4, 31.6, 31.6, 28.5, 28.1, 22.5, 22.4, 21.2, 21.2, 20.8, 20.7, 20.7, 20.4; <sup>19</sup>F NMR (471 MHz, CDCl<sub>3</sub>, BTF IStd): δ −76.8. HRMS (ESI+): calcd for [C<sub>22</sub>H<sub>28</sub>F<sub>3</sub>N<sub>1</sub>O<sub>10</sub>Na]<sup>+</sup> 546.1558, measured 546.1565, Δ 1.3 ppm.</div></div><div id="sec4_6_60" class="NLM_sec NLM_sec_level_3"><div id="ac_i122" class="anchor-spacer"></div><h4 class="article-section__title" id="_i122"> (2<i>R</i>,3<i>R</i>,4<i>R</i>,5<i>R</i>)-2-(6-Benzamido-9<i>H</i>-purin-9-yl)-5-((2<i>S</i>,6<i>S</i>)-2-ethynyl-7-methoxy-7-oxo-6-(2,2,2-trifluoroacetamido)heptyl)tetrahydrofuran-3,4-diyl Diacetate (<b>S65</b>)</h4><div class="NLM_p last">To a suspension of <i>N</i><sup>6</sup>-benzoyladenine (302 mg, 1.26 mmol, 4.0 equiv) in propionitrile (dried over 4 Å MS, 6.0 mL) at RT was added <i>N</i>,<i>O</i>-bis(trimethylsilyl)acetamide (0.39 mL, 1.6 mmol, 5.0 equiv). The resulting mixture was stirred at 80 °C for 10 min, upon which complete solubilization of <i>N</i><sup>6</sup>-benzoyladenine was observed. To a separate flask charged with triacetate <b>S64</b> (dried by azeotropic distillation with benzene (4×), 165 mg, 0.315 mmol), and 2,6-di-<i>tert</i>-butyl-4-methylpyridinium triflate (56 mg, 0.16 mmol, 50 mol %) was added propionitrile (6.0 mL). The resulting mixture was stirred at RT for 5 min until a clear solution was obtained, and the solution of silylated <i>N</i><sup>6</sup>-benzoyladenine was added. The reaction mixture was stirred at 95 °C for 5.5 h, cooled to RT, and the volatiles were removed in vacuo. Crude material was purified by silica gel chromatography (2,2,4-trimethylpentane/EtOAc, 30 → 100%) to afford nucleoside <b>S65</b> (209 mg, 0.297 mmol, 94%) as a colorless oil. <i>R</i><sub><i>f</i></sub> = 0.43 (hexanes/EtOAc, 20:80). FTIR (thin film), ν<sub>max</sub> (cm<sup>–1</sup>): 3287, 2955, 2925, 2868, 1748, 1721, 1612, 1583, 1511, 1488, 1457, 1245, 1219, 1187, 1161, 1049. <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 9.13 (br s, 1H), 8.76 (s, 1H), 8.07 (s, 1H), 8.04–7.98 (m, 2H), 7.61–7.57 (m, 1H), 7.53–7.48 (m, 2H), 7.05 (br d, <i>J</i> = 7.9 Hz, 1H), 6.12–6.10 (m, 2H), 5.57–5.54 (m, 1H), 4.60 (ddd, <i>J</i> = 7.9, 7.4, 5.4 Hz, 1H), 4.50 (ddd, <i>J</i> = 11.0, 4.5, 2.8 Hz, 1H), 3.75 (s, 3H), 2.55–2.48 (m, 1H), 2.15 (s, 3H), 2.12 (d, <i>J</i> = 2.4 Hz, 1H), 2.10–2.03 (m, 1H), 2.06 (s, 3H), 1.89 (app ddt, <i>J</i> = 13.9, 10.8, 5.4 Hz, 1H), 1.83 (ddd, <i>J</i> = 13.8, 11.4, 2.8 Hz, 1H), 1.74 (dddd, <i>J</i> = 13.9, 10.7, 7.4, 4.8 Hz, 1H), 1.61–1.52 (m, 1H), 1.52–1.42 (m, 2H), 1.42–1.33 (m, 1H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 171.4, 169.8, 169.6, 164.8, 156.9 (q, <sup>2</sup><i>J</i><sub>C–F</sub> = 38 Hz), 152.8, 151.7, 149.9, 142.4, 133.6, 133.0, 129.0, 128.0, 124.2, 115.7 (q, <sup>1</sup><i>J</i><sub>C–F</sub> = 288 Hz), 87.2, 85.1, 80.9, 73.7, 72.9, 71.4, 53.0, 52.5, 38.6, 34.4, 31.6, 28.1, 22.6, 20.7, 20.5. <sup>19</sup>F NMR (376 MHz, CDCl<sub>3</sub>, BTF IStd): δ −76.8. HRMS (ESI+): calcd for [C<sub>32</sub>H<sub>34</sub>F<sub>3</sub>N<sub>6</sub>O<sub>9</sub>]<sup>+</sup> 703.2334, measured 703.2326, Δ 1.1 ppm.</div></div><div id="sec4_6_61" class="NLM_sec NLM_sec_level_3"><div id="ac_i123" class="anchor-spacer"></div><h4 class="article-section__title" id="_i123"> (2<i>R</i>,3<i>R</i>,4<i>R</i>,5<i>R</i>)-2-(6-Benzamido-9<i>H</i>-purin-9-yl)-5-((2<i>S</i>,6<i>S</i>)-2-((3-carbamoylphenyl)ethynyl)-7-methoxy-7-oxo-6-(2,2,2-trifluoroacetamido)heptyl)tetrahydrofuran-3,4-diyl Diacetate (<b>S66</b>)</h4><div class="NLM_p last">To a vial charged with nucleoside <b>S65</b> (89 mg, 0.13 mmol) were added 3-iodobenzamide (78 mg, 0.32 mmol, 2.5 equiv), CuI (6 mg, 0.03 mmol, 25 mol %), and Pd(PPh<sub>3</sub>)<sub>4</sub> (7 mg, 6 μmol, 5 mol %). The vial headspace was purged with nitrogen for 10 min, and a 5:1:1 mixture of toluene, DMF, and <i>i</i>-Pr<sub>2</sub>NEt (degassed by sparging with nitrogen for 15 min, 4.2 mL) was added at RT. The resulting mixture was stirred at 60 °C for 2 h. The solution was allowed to cool to RT, and volatiles were removed in vacuo. Crude material was filtered through a short pad of silica gel (EtOAc/<i>i</i>-PrOH, 95:5) to afford benzamide <b>S66</b>, which was further purified by preparative HPLC (15% → 95% B′ over 30 min) to afford benzamide <b>S66</b> (69 mg, 84 μmol, 66%) as a white amorphous solid. <i>R</i><sub><i>f</i></sub> = 0.51 (EtOAc/<i>i</i>-PrOH, 95:5). FTIR (thin film), ν<sub>max</sub> (cm<sup>–1</sup>): 3279, 3071, 2928, 1748, 1719, 1667, 1612, 1580, 1512, 1487, 1457, 1438, 1377, 1245, 1219, 1159, 1097. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): δ 8.79 (s, 1H), 8.46 (s, 1H), 8.07–8.01 (m, 2H), 7.87 (app t, <i>J</i> = 1.8 Hz, 1H), 7.80 (ddd, <i>J</i> = 7.8, 1.8, 1.3 Hz, 1H), 7.64–7.60 (m, 1H), 7.55–7.49 (m, 3H), 7.39 (app t, <i>J</i> = 7.8 Hz, 1H), 7.18 (br d, <i>J</i> = 7.8 Hz, 1H), 7.12 (br s, 2H), 6.20 (d, <i>J</i> = 5.8 Hz, 1H), 6.12 (dd, <i>J</i> = 5.8, 5.4 Hz, 1H), 5.65 (dd, <i>J</i> = 5.4, 4.0 Hz, 1H), 4.63 (app td, <i>J</i> = 7.8, 5.1 Hz, 1H), 4.54 (app dt, <i>J</i> = 9.6, 4.0 Hz, 1H), 3.74 (s, 3H), 2.80–2.72 (m, 1H), 2.17 (s, 3H), 2.13 (ddd, <i>J</i> = 13.8, 9.6, 3.6 Hz, 1H), 2.06 (s, 3H), 2.05–1.92 (m, 2H), 1.85–1.76 (m, 1H), 1.71–1.45 (m, 4H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 171.5, 170.7, 170.2, 169.9, 165.9, 157.2 (q, <sup>2</sup><i>J</i><sub>C–F</sub> = 38 Hz), 152.0, 151.2, 149.2, 143.5, 135.3, 133.7, 132.3, 132.2, 130.9, 129.1, 128.9, 128.5, 127.5, 123.9, 122.8, 115.7 (q, <sup>1</sup><i>J</i><sub>C–F</sub> = 288 Hz), 92.0, 87.4, 82.7, 81.8, 73.8, 72.9, 53.1, 52.7, 38.8, 34.6, 31.8, 29.0, 23.1, 20.8, 20.5. <sup>19</sup>F NMR (471 MHz, CDCl<sub>3</sub>, BTF IStd): δ −76.8. HRMS (ESI+): calcd for [C<sub>39</sub>H<sub>38</sub>F<sub>3</sub>N<sub>7</sub>O<sub>10</sub>Na]<sup>+</sup> 844.2524, measured 844.2532, Δ 0.8 ppm.</div></div><div id="sec4_6_62" class="NLM_sec NLM_sec_level_3"><div id="ac_i124" class="anchor-spacer"></div><h4 class="article-section__title" id="_i124"> (2<i>R</i>,3<i>R</i>,4<i>R</i>,5<i>R</i>)-2-(6-Benzamido-9<i>H</i>-purin-9-yl)-5-((2<i>S</i>,5<i>S</i>)-2-((4-carbamoylphenyl)ethynyl)-6-methoxy-6-oxo-5-(2,2,2-trifluoroacetamido)hexyl)tetrahydrofuran-3,4-diyl Diacetate (<b>S67</b>)</h4><div class="NLM_p last">To a vial charged with alkyne <b>8</b> (20 mg, 29 μmol) were added 4-iodobenzamide (18 mg, 73 μmol, 2.5 equiv), CuI (1 mg, 7 μmol, 25 mol %), and Pd(PPh<sub>3</sub>)<sub>4</sub> (2 mg, 2 μmol, 5 mol %). The vial headspace was purged with nitrogen for 5 min and a 5:1:1 mixture of toluene, DMF, and <i>i</i>-Pr<sub>2</sub>NEt (degassed by sparging with nitrogen for 15 min, 1.5 mL) was added at RT. The resulting mixture was stirred at 60 °C for 3 h. The solution was allowed to cool to RT, and volatiles were removed in vacuo. Crude material was filtered through a short pad of silica gel (EtOAc/<i>i</i>-PrOH, 95:5) to afford alkyne <b>S67</b>, which was further purified by preparative HPLC (15% → 95% B’ over 30 min) to afford <b>S67</b> (19 mg, 24 μmol, 81%) as a white amorphous solid. <sup>1</sup>H NMR (500 MHz, CD<sub>3</sub>OD): δ 8.79 (s, 1H), 8.63 (s, 1H), 8.11–8.07 (m, 2H), 7.86–7.80 (m, 2H), 7.70–7.65 (m, 1H), 7.60–7.55 (m, 2H), 7.48–7.44 (m, 2H), 6.34 (d, <i>J</i> = 5.0 Hz, 1H), 6.16 (dd, <i>J</i> = 5.7, 5.0 Hz, 1H), 5.66 (dd, <i>J</i> = 5.7, 5.0 Hz, 1H), 4.56 (ddd, <i>J</i> = 10.3, 5.0, 3.6 Hz, 1H), 4.53 (dd, <i>J</i> = 9.6, 4.9 Hz, 1H), 3.73 (s, 3H), 2.92–2.84 (m, 1H), 2.23 (ddd, <i>J</i> = 14.0, 10.3, 3.9 Hz, 1H), 2.15 (s, 3H), 2.18–2.10 (m, 1H), 2.06 (s, 3H), 2.05–2.00 (m, 2H), 1.74–1.60 (m, 2H). <sup>13</sup>C NMR (126 MHz, CD<sub>3</sub>OD): δ 172.5, 171.6, 171.5, 171.3, 168.7, 159.1 (q, <sup>2</sup><i>J</i><sub>C–F</sub> = 37 Hz), 153.1, 153.0, 151.0, 145.5, 134.6, 134.2, 134.2, 132.6, 129.8, 129.6, 128.7, 128.2, 125.0, 117.4 (q, <sup>1</sup><i>J</i><sub>C–F</sub> = 287 Hz), 94.3, 88.8, 83.9, 82.2, 75.0, 74.2, 53.8, 53.1, 39.4, 32.6, 29.9, 29.7, 20.5, 20.3. <sup>19</sup>F NMR (376 MHz, CD<sub>3</sub>OD, BTF IStd): δ −76.5. HRMS (ESI+): calcd for [C<sub>38</sub>H<sub>37</sub>F<sub>3</sub>N<sub>7</sub>O<sub>10</sub>]<sup>+</sup> 808.2549, measured 808.2556, Δ 0.9 ppm.</div></div><div id="sec4_6_63" class="NLM_sec NLM_sec_level_3"><div id="ac_i125" class="anchor-spacer"></div><h4 class="article-section__title" id="_i125"> (2<i>R</i>,3<i>R</i>,4<i>R</i>,5<i>R</i>)-2-(6-Benzamido-9<i>H</i>-purin-9-yl)-5-((2<i>S</i>,5<i>S</i>)-2-((2-carbamoylphenyl)ethynyl)-6-methoxy-6-oxo-5-(2,2,2-trifluoroacetamido)hexyl)tetrahydrofuran-3,4-diyl Diacetate (<b>S68</b>)</h4><div class="NLM_p last">To a vial charged with alkyne <b>8</b> (20 mg, 29 μmol) were added 2-iodobenzamide (18 mg, 73 μmol, 2.5 equiv), CuI (1 mg, 7 μmol, 25 mol %), and Pd(PPh<sub>3</sub>)<sub>4</sub> (2 mg, 2 μmol, 5 mol %). The vial headspace was purged with nitrogen for 5 min, and a 5:1:1 mixture of toluene, DMF, and <i>i</i>-Pr<sub>2</sub>NEt (degassed by sparging with nitrogen for 15 min, 1.5 mL) was added at RT. The resulting mixture was stirred at 60 °C for 3 h. The solution was allowed to cool to RT, and volatiles were removed in vacuo. Crude material was filtered through a short pad of silica gel (EtOAc/<i>i</i>-PrOH, 75:25) to afford <b>S68</b>, which was further purified by preparative HPLC (15% → 95% B’ over 30 min) to afford <b>S68</b> (18 mg, 22 μmol, 77%) as a white amorphous solid. <sup>1</sup>H NMR (600 MHz, CD<sub>3</sub>OD): δ 8.80 (s, 1H), 8.68 (s, 1H), 8.13–8.07 (m, 2H), 7.70–7.67 (m, 1H), 7.61–7.57 (m, 3H), 7.48–7.45 (m, 1H), 7.42 (app td, <i>J</i> = 7.5, 1.6 Hz, 1H), 7.39 (app td, <i>J</i> = 7.5, 1.6 Hz, 1H), 6.36 (d, <i>J</i> = 5.2 Hz, 1H), 6.15 (dd, <i>J</i> = 5.7, 5.2 Hz, 1H), 5.62 (dd, <i>J</i> = 5.7, 5.0 Hz, 1H), 4.62 (ddd, <i>J</i> = 10.4, 5.0, 3.0 Hz, 1H), 4.52 (dd, <i>J</i> = 9.8, 4.9 Hz, 1H), 3.73 (s, 3H), 2.92–2.86 (m, 1H), 2.22 (ddd, <i>J</i> = 13.7, 10.4, 4.2 Hz, 1H), 2.16 (s, 3H), 2.16–2.09 (m, 1H), 2.07 (s, 3H), 2.08–2.01 (m, 2H), 1.72–1.61 (m, 2H). <sup>13</sup>C NMR (126 MHz, CD<sub>3</sub>OD): δ 173.2, 172.6, 171.7, 171.3, 168.6, 159.1 (q, <sup>2</sup><i>J</i><sub>C–F</sub> = 38 Hz), 153.1, 152.4, 150.6, 145.5, 139.2, 134.6, 134.2, 134.1, 131.2, 129.9, 129.8, 129.6, 129.2, 128.8, 122.1, 117.4 (q, <sup>1</sup><i>J</i><sub>C–F</sub> = 287 Hz), 96.7, 88.8, 82.3, 82.2, 74.9, 74.2, 53.8, 53.1, 39.2, 32.5, 30.0, 29.7, 20.6, 20.3. <sup>19</sup>F NMR (471 MHz, CD<sub>3</sub>OD, BTF IStd): δ −76.5. HRMS (ESI+): calcd for [C<sub>38</sub>H<sub>37</sub>F<sub>3</sub>N<sub>7</sub>O<sub>10</sub>]<sup>+</sup> 808.2549, measured 808.2547, Δ 0.1 ppm.</div></div><div id="sec4_6_64" class="NLM_sec NLM_sec_level_3"><div id="ac_i126" class="anchor-spacer"></div><h4 class="article-section__title" id="_i126"> (2<i>R</i>,3<i>R</i>,4<i>R</i>,5<i>R</i>)-2-(6-Benzamido-9<i>H</i>-purin-9-yl)-5-((2<i>S</i>,5<i>S</i>)-6-methoxy-6-oxo-2-((3-sulfamoylphenyl)ethynyl)-5-(2,2,2-trifluoroacetamido)hexyl)tetrahydrofuran-3,4-diyl Diacetate (<b>S69</b>)</h4><div class="NLM_p last">To a solution of alkyne <b>8</b> (41 mg, 59 μmol) and 3-iodobenzenesulfonamide (25 mg, 89 μmol, 1.5 equiv) in a 5:1:1 mixture of toluene, DMF, and <i>i</i>-Pr<sub>2</sub>NEt (degassed by sparging with nitrogen for 15 min, 2.0 mL) at RT were introduced CuI (3 mg, 0.01 mmol, 25 mol %) and Pd(PPh<sub>3</sub>)<sub>4</sub> (3 mg, 3 μmol, 5 mol %) rapidly. The resulting mixture was stirred at 50 °C for 17 h. The solution was allowed to cool to RT, and volatiles were removed in vacuo. Crude material was purified by silica gel chromatography (EtOAc/<i>i</i>-PrOH, 0 → 10%) to afford <b>S69</b> (41 mg, 49 μmol, 82%) as a colorless oil. <i>R</i><sub><i>f</i></sub> = 0.29 (EtOAc/<i>i</i>-PrOH, 95:5). FTIR (thin film), ν<sub>max</sub> (cm<sup>–1</sup>): 3301, 3074, 2924, 1748, 1721, 1612, 1583, 1456, 1336, 1245, 1219, 1162, 1097. <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 9.11 (s, 1H), 8.75 (s, 1H), 8.16 (s, 1H), 8.02–7.96 (m, 2H), 7.85 (app t, <i>J</i> = 1.2 Hz, 1H), 7.77 (app dt, <i>J</i> = 7.8, 1.2 Hz, 1H), 7.62–7.57 (m, 1H), 7.53–7.48 (m, 2H), 7.47 (app dt, <i>J</i> = 7.8, 1.2 Hz, 1H), 7.37 (app t, <i>J</i> = 7.8 Hz, 1H), 7.27 (br d, <i>J</i> = 7.7 Hz, 1H), 6.15 (d, <i>J</i> = 5.7 Hz, 1H), 6.13 (dd, <i>J</i> = 5.7, 5.3 Hz, 1H), 5.63 (br s, 2H), 5.62 (dd, <i>J</i> = 5.3, 4.2 Hz, 1H), 4.65 (app td, <i>J</i> = 7.7, 5.5 Hz, 1H), 4.47 (ddd, <i>J</i> = 9.7, 4.2, 3.5 Hz, 1H), 3.77 (s, 3H), 2.86–2.81 (m, 1H), 2.21–2.14 (m, 1H), 2.16 (s, 3H), 2.10 (app ddt, <i>J</i> = 14.0, 11.1, 5.5 Hz, 1H), 2.06 (s, 3H), 2.05–1.98 (m, 2H), 1.69–1.60 (m, 1H), 1.60–1.52 (m, 1H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 171.3, 170.1, 169.8, 165.0, 157.2 (q, <sup>2</sup><i>J</i><sub>C–F</sub> = 38 Hz), 152.8, 151.8, 149.8, 142.7, 142.6, 135.2, 133.4, 133.1, 129.8, 129.1, 129.0, 128.1, 125.5, 124.4, 124.1, 115.7 (q, <sup>1</sup><i>J</i><sub>C–F</sub> = 288 Hz), 92.7, 87.0, 82.2, 81.1, 73.6, 72.8, 53.3, 52.5, 38.1, 30.8, 29.8, 28.6, 20.8, 20.6. <sup>19</sup>F NMR (471 MHz, CDCl<sub>3</sub>, BTF IStd): δ −76.6. HRMS (ESI+): calcd for [C<sub>37</sub>H<sub>37</sub>F<sub>3</sub>N<sub>7</sub>O<sub>11</sub>S<sub>1</sub>]<sup>+</sup> 844.2218, measured 844.2198, Δ 2.4 ppm.</div></div><div id="sec4_6_65" class="NLM_sec NLM_sec_level_3"><div id="ac_i127" class="anchor-spacer"></div><h4 class="article-section__title" id="_i127"> 2-Fluoro-5-Iodobenzamide (<b>S70</b>)</h4><div class="NLM_p last">To a vial charged with 2-fluoro-5-iodobenzonitrile (0.99 g, 4.0 mmol) was added H<sub>2</sub>SO<sub>4</sub> (98%, 4.0 mL). The resulting mixture was stirred at 70 °C for 2 h, cooled to 0 °C, before the addition of a 1 N NaOH aq solution (15 mL) and a NaHCO<sub>3</sub> satd aq solution (5 mL). The resulting white precipitate was filtered to yield 2-fluoro-5-iodobenzamide (<b>S70</b>) as a pale-orange, off-white powder (1.1 g, 4.0 mmol, quant). <i>R</i><sub><i>f</i></sub> = 0.43 (hexanes/EtOAc, 50:50). FTIR (thin film), ν<sub>max</sub> (cm<sup>–1</sup>): 3361, 3305, 3165, 1710, 1656, 1625, 1598, 1564, 1475, 1427, 1363, 1254, 1228. <sup>1</sup>H NMR (500 MHz, CD<sub>3</sub>OD): δ 8.09 (1H, dd, <i>J</i> = 6.8, 2.4 Hz), 7.85 (1H, ddd, <i>J</i> = 8.7, 4.7, 2.4 Hz), 7.03 (1H, dd, <i>J</i> = 10.7, 8.7 Hz). <sup>13</sup>C NMR (101 MHz, CD<sub>3</sub>OD): δ 166.9, 161.5 (d, <i>J</i><sub>C–F</sub> = 251 Hz), 143.3 (d, <i>J</i><sub>C–F</sub> = 8.7 Hz), 140.5 (d, <i>J</i><sub>C–F</sub> = 2.6 Hz), 125.7 (d, <i>J</i><sub>C–F</sub> = 15 Hz), 119.6 (d, <i>J</i><sub>C–F</sub> = 25 Hz), 88.0 (d, <i>J</i><sub>C–F</sub> = 3.7 Hz). <sup>19</sup>F NMR (471 MHz, CD<sub>3</sub>OD, BTF IStd): δ −116.3. HRMS (ESI+): calcd for [C<sub>7</sub>H<sub>5</sub>F<sub>1</sub>I<sub>1</sub>N<sub>1</sub>O<sub>1</sub>Na]<sup>+</sup> 287.9292, measured 287.9283, Δ 3.1 ppm.</div></div><div id="sec4_6_66" class="NLM_sec NLM_sec_level_3"><div id="ac_i128" class="anchor-spacer"></div><h4 class="article-section__title" id="_i128"> (2<i>R</i>,3<i>R</i>,4<i>R</i>,5<i>R</i>)-2-(6-Benzamido-9<i>H</i>-purin-9-yl)-5-((2<i>S</i>,5<i>S</i>)-2-((3-carbamoyl-4-fluorophenyl)ethynyl)-6-methoxy-6-oxo-5-(2,2,2-trifluoroacetamido)hexyl)tetrahydrofuran-3,4-diyl Diacetate (<b>S71</b>)</h4><div class="NLM_p last">To a vial charged with nucleoside <b>8</b> (31 mg, 46 μmol) were added 2-fluoro-5-iodobenzamide (<b>S70</b>) (20 mg, 75 μmol, 1.65 equiv), CuI (2 mg, 9 μmol, 20 mol %), and Pd(PPh<sub>3</sub>)<sub>4</sub> (3 mg, 2 μmol, 5 mol %). The vial headspace was purged with nitrogen for 5 min and a 5:1:1 mixture of toluene, DMF, and <i>i</i>-Pr<sub>2</sub>NEt (degassed by sparging with nitrogen for 15 min, 3.5 mL) was added at RT. The resulting mixture was stirred at 60 °C for 1 h. More 2-fluoro-5-iodobenzamide (10 mg, 38 μmol, 0.83 equiv) in a 5:1:1 mixture of toluene, DMF, and <i>i</i>-Pr<sub>2</sub>NEt (2.0 mL) was added, and the reaction mixture was stirred at 60 °C for another 3 h. The solution was allowed to cool to RT, and volatiles were removed in vacuo. Crude material was purified by silica gel chromatography (EtOAc/<i>i</i>-PrOH, 0 → 10%) to afford <b>S71</b> (34 mg, 41 μmol, 91%) as a white amorphous solid. FTIR (thin-film), ν<sub>max</sub> (cm<sup>–1</sup>): 3256, 3082, 3060, 3026, 2924, 2853, 1748, 1719, 1672, 1611, 1583, 1513, 1491, 1453, 1366, 1245, 1216, 1183, 1159, 1110, 1074, 1048, 1029. <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 9.02 (s, 1H), 8.80 (s, 1H), 8.14 (dd, <i>J</i> = 7.5, 2.3 Hz, 1H), 8.12 (s, 1H), 8.03 (d, <i>J</i> = 7.4 Hz, 2H), 7.69–7.59 (m, 1H), 7.55–7.44 (m, 3H), 7.09 (dd, <i>J</i> = 11.5, 8.5 Hz, 1H), 7.05 (br d, <i>J</i> = 7.8 Hz, 1H), 6.68 (br d, <i>J</i> = 10.9 Hz, 1H), 6.14 (d, <i>J</i> = 5.4 Hz, 1H), 6.11 (t, <i>J</i> = 5.4 Hz, 1H), 5.95 (br s, 1H), 5.60 (t, <i>J</i> = 5.0 Hz, 1H), 4.65 (td, <i>J</i> = 7.7, 5.0 Hz, 1H), 4.50 (ddd, <i>J</i> = 10.7, 4.6, 3.0 Hz, 1H), 3.79 (s, 3H), 2.82 (ddt, <i>J</i> = 10.9, 9.1, 4.5 Hz, 1H), 2.17 (s, 3H), 2.18–2.08 (m, 2H), 2.08 (s, 3H), 2.04–1.92 (m, 2H), 1.64 (ddd, <i>J</i> = 18.6, 9.3, 5.4 Hz, 1H), 1.52 (dddd, <i>J</i> = 13.5, 11.0, 9.4, 4.6 Hz, 1H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>): δ 171.3, 169.9, 169.7, 164.7, 164.1, 164.1, 161.6, 159.1, 157.3, 156.9, 156.5, 152.9, 151.7, 149.9, 142.4, 137.1, 137.0, 135.8, 135.8, 133.6, 133.0, 132.3, 132.2, 132.1, 132.1, 129.0, 128.7, 128.6, 128.0, 124.1, 120.5, 120.5, 120.4, 120.3, 117.2, 116.7, 116.4, 114.3, 91.1, 91.0, 87.2, 81.9, 80.9, 77.4, 73.7, 73.0, 53.3, 52.5, 38.6, 31.0, 30.0, 28.7, 23.6, 20.8, 20.6. <sup>19</sup>F NMR (471 MHz, CDCl<sub>3</sub>, BTF IStd): δ −76.7, −113.0. HRMS (ESI+): calcd for [C<sub>38</sub>H<sub>35</sub>F<sub>4</sub>N<sub>7</sub>O<sub>10</sub>Na]<sup>+</sup> 848.2272, measured 848.2274, Δ 0.2 ppm.</div></div><div id="sec4_6_67" class="NLM_sec NLM_sec_level_3"><div id="ac_i129" class="anchor-spacer"></div><h4 class="article-section__title" id="_i129"> 5-Iodo-2-methylbenzamide (<b>S72</b>).<a onclick="showRef(event, 'ref77'); return false;" href="javascript:void(0);" class="ref ref77">(77)</a></h4><div class="NLM_p last">To a vial charged with 5-iodo-2-methylbenzonitrile (0.12 g, 0.50 mmol) and <i>t</i>-BuOK (0.17 g, 1.5 mmol, 3.0 equiv) was added <i>t</i>-BuOH (4.5 mL) under a nitrogen atmosphere. The resulting mixture was stirred at 60 °C for 24 h before the addition of water (10 mL) and EtOAc (20 mL). The layers were separated, and the aqueous phase was extracted with EtOAc (3 × 20 mL). The combined organic extracts were washed with brine (80 mL), dried over MgSO<sub>4</sub>, filtered, and concentrated. Crude material was dissolved in hot toluene, recrystallized, and washed with hexanes to yield 5-iodo-2-methylbenzamide (<b>S72</b>) (85 mg, 0.33 mmol, 65% yield) as pale-yellow crystals. <i>R</i><sub><i>f</i></sub> = 0.20 (hexanes/EtOAc, 50:50). FTIR (thin film), ν<sub>max</sub> (cm<sup>–1</sup>): 3371, 3187, 1649, 1582, 1559, 1401, 1373. <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 7.77 (d, <i>J</i> = 1.8 Hz, 1H), 7.65 (dd, <i>J</i> = 8.2, 1.8 Hz, 1H), 6.99 (d, <i>J</i> = 8.2 Hz, 1H), 5.68 (br s, 1H), 5.62 (br s, 1H), 2.43 (s, 3H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 170.2, 139.3, 137.3, 136.2, 135.7, 133.2, 90.2, 19.7. HRMS (ESI+): calcd for [C<sub>8</sub>H<sub>9</sub>I<sub>1</sub>N<sub>1</sub>O<sub>1</sub>]<sup>+</sup> 261.9723, measured 261.9721, Δ 0.9 ppm.</div></div><div id="sec4_6_68" class="NLM_sec NLM_sec_level_3"><div id="ac_i130" class="anchor-spacer"></div><h4 class="article-section__title" id="_i130"> (2<i>R</i>,3<i>R</i>,4<i>R</i>,5<i>R</i>)-2-(6-Benzamido-9<i>H</i>-purin-9-yl)-5-((2<i>S</i>,5<i>S</i>)-2-((3-carbamoyl-4-methylphenyl)ethynyl)-6-methoxy-6-oxo-5-(2,2,2-trifluoroacetamido)hexyl)tetrahydrofuran-3,4-diyl Diacetate (<b>S73</b>)</h4><div class="NLM_p last">To a solution of alkyne <b>8</b> (50 mg, 73 μmol) and 5-iodo-2-methylbenzamide (<b>S72</b>) (47 mg, 0.18 mmol, 2.5 equiv) in a 5:1:1 mixture of toluene, DMF, and <i>i</i>-Pr<sub>2</sub>NEt (degassed by sparging with nitrogen for 15 min, 3.6 mL) at RT were introduced CuI (3 mg, 0.02 mmol, 25 mol %) and Pd(PPh<sub>3</sub>)<sub>4</sub> (4 mg, 4 μmol, 5 mol %) rapidly. The resulting mixture was stirred at 60 °C for 2 h. The solution was allowed to cool to RT, and volatiles were removed in vacuo. Crude material was filtered through a short pad of silica gel (EtOAc/<i>I</i>-PrOH, 95:5) to afford <b>S73</b>, which was further purified by preparative HPLC (15% → 95% B over 30 min) to afford <b>S73</b> (40 mg, 49 μmol, 67%) as a white amorphous solid. <i>R</i><sub><i>f</i></sub> = 0.34 (EtOAc). FTIR (thin film), ν<sub>max</sub> (cm<sup>–1</sup>): 3341, 1747, 1721, 1668, 1612, 1584, 1457, 1376, 1246, 1220, 1099, 1075, 1052. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): δ 9.28 (s, 1H), 8.74 (s, 1H), 8.13 (s, 1H), 8.04–7.98 (m, 2H), 7.61–7.56 (m, 1H), 7.52–7.47 (m, 2H), 7.46 (d, <i>J</i> = 1.8 Hz, 1H), 7.42 (br d, <i>J</i> = 7.7 Hz, 1H), 7.28 (dd, <i>J</i> = 7.8, 1.8 Hz, 1H), 7.11 (br d, <i>J</i> = 7.8 Hz, 1H), 6.32 (br s, 1H), 6.16 (br s, 1H), 6.13 (d, <i>J</i> = 5.6 Hz, 1H), 6.11 (dd, <i>J</i> = 5.6, 5.1 Hz, 1H), 5.61 (dd, <i>J</i> = 5.1, 4.2 Hz, 1H), 4.64 (app td, <i>J</i> = 7.7, 5.2 Hz, 1H), 4.48 (ddd, <i>J</i> = 10.0, 4.2, 3.6 Hz, 1H), 3.75 (s, 3H), 2.79 (dddd, <i>J</i> = 10.9, 9.3, 4.8, 4.0 Hz, 1H), 2.42 (s, 3H), 2.13 (s, 3H), 2.17–2.07 (m, 2H), 2.05 (s, 3H), 2.07–1.99 (m, 1H), 1.96 (ddd, <i>J</i> = 13.9, 10.9, 3.6 Hz, 1H), 1.63 (dddd, <i>J</i> = 13.4, 10.8, 5.8, 4.8 Hz, 1H), 1.53 (dddd, <i>J</i> = 13.4, 10.6, 9.3, 4.8 Hz, 1H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 171.4, 171.4, 170.0, 169.6, 165.0, 157.1 (q, <sup>2</sup><i>J</i><sub>C–F</sub> = 38 Hz), 152.8, 151.8, 149.9, 142.4, 136.8, 135.2, 133.5, 133.1, 133.0, 131.3, 130.6, 128.9, 128.1, 124.2, 120.6, 115.7 (q, <sup>1</sup><i>J</i><sub>C–F</sub> = 288 Hz), 90.5, 86.9, 83.0, 81.2, 73.7, 72.8, 53.2, 52.5, 38.5, 31.0, 29.8, 28.7, 20.8, 20.5, 20.1. <sup>19</sup>F NMR (471 MHz, CDCl<sub>3</sub>, BTF IStd): δ −76.6. HRMS (ESI+): calcd for [C<sub>39</sub>H<sub>39</sub>F<sub>3</sub>N<sub>7</sub>O<sub>10</sub>]<sup>+</sup> 822.2705, measured 822.2718, Δ 1.6 ppm.</div></div><div id="sec4_6_69" class="NLM_sec NLM_sec_level_3"><div id="ac_i131" class="anchor-spacer"></div><h4 class="article-section__title" id="_i131"> 5-Iodo-2-(trifluoromethyl)benzamide (<b>S74</b>)</h4><div class="NLM_p last">To a vial charged with 5-iodo-2-(trifluoromethyl)benzonitrile (0.30 g, 1.0 mmol) and <i>t</i>-BuOK (0.67 g, 6.0 mmol, 6.0 equiv) was added <i>t</i>-BuOH (5.0 mL) under a nitrogen atmosphere. The resulting mixture was stirred at 60 °C for 5 min, cooled to RT, and stirred for 48 h. The reaction was quenched by the addition of water (10 mL). The resulting suspension was filtered to produce an orange solid. Crude material was purified by silica gel chromatography (hexanes/EtOAc, 0 → 100%) to yield 5-iodo-2-(trifluoromethyl)benzamide (<b>S74</b>) as a white solid (0.13 g, 0.41 mmol, 41%). <i>R</i><sub><i>f</i></sub> = 0.46 (hexanes/EtOAc, 50:50). FTIR (thin film), ν<sub>max</sub> (cm<sup>–1</sup>): 3369, 3183, 1649, 1588, 1569, 1400, 1378, 1311, 1287. <sup>1</sup>H NMR (500 MHz, CD<sub>3</sub>OD): δ 8.01 (1H, ddq, <i>J</i> = 8.3, 1.7, 0.9 Hz), 7.95–7.92 (1H, m), 7.50 (1H, br d, <i>J</i> = 8.3 Hz). <sup>13</sup>C NMR (126 MHz, CD<sub>3</sub>OD): δ 171.3, 140.3, 138.6 (q, <i>J</i><sub>C–F</sub> = 2.1 Hz), 138.3, 129.0 (q, <i>J</i><sub>C–F</sub> = 4.8 Hz), 127.7 (q, <i>J</i><sub>C–F</sub> = 32.6 Hz), 125.1 (q, <i>J</i><sub>C–F</sub> = 273 Hz), 99.4 (q, <i>J</i><sub>C–F</sub> = 1.4 Hz). <sup>19</sup>F NMR (471 MHz, CD<sub>3</sub>OD, BTF IStd): δ −60.3. HRMS (ESI+): calcd for [C<sub>8</sub>H<sub>5</sub>F<sub>3</sub>I<sub>1</sub>N<sub>1</sub>O<sub>1</sub>Na]<sup>+</sup> 337.9260, measured 337.9260, Δ 0.0 ppm.</div></div><div id="sec4_6_70" class="NLM_sec NLM_sec_level_3"><div id="ac_i132" class="anchor-spacer"></div><h4 class="article-section__title" id="_i132"> (2<i>R</i>,3<i>R</i>,4<i>R</i>,5<i>R</i>)-2-(6-Benzamido-9<i>H</i>-purin-9-yl)-5-((2<i>S</i>,5<i>S</i>)-2-((3-carbamoyl-4-(trifluoromethyl)phenyl)ethynyl)-6-methoxy-6-oxo-5-(2,2,2-trifluoroacetamido)hexyl)tetrahydrofuran-3,4-diyl Diacetate (<b>S75</b>)</h4><div class="NLM_p last">To a vial charged with alkyne <b>8</b> (82 mg, 0.12 mmol) were added 5-iodo-2-(trifluoromethyl)benzamide (<b>S74</b>) (62 mg, 0.20 mmol, 1.65 equiv), CuI (5 mg, 24 μmol, 20 mol %), and Pd(PPh<sub>3</sub>)<sub>4</sub> (7 mg, 6 μmol, 5 mol %). The vial headspace was purged with nitrogen for 5 min, and a 5:1:1 mixture of toluene, DMF, and <i>i</i>-Pr<sub>2</sub>NEt (degassed by sparging with nitrogen for 15 min, 3.5 mL) was added at RT. The resulting mixture was stirred at 70 °C for 3 h. The solution was allowed to cool to RT, and volatiles were removed in vacuo. Crude material was purified by silica gel chromatography (EtOAc/<i>i</i>-PrOH, 0 → 5%) to afford <b>S75</b> (90 mg, 0.10 mmol, 86%) as a white amorphous solid. <i>R</i><sub><i>f</i></sub> = 0.48 (toluene/MeCN, 50:50). FTIR (thin-film), ν<sub>max</sub> (cm<sup>–1</sup>): 3306, 2926, 2165, 1747, 1719, 1672, 1609, 1582, 1511, 1487, 1456, 1410, 1373, 1313, 1289, 1242, 1216, 1175, 1133, 1114, 1074, 1039. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): δ 8.98 (s, 1H), 8.78 (s, 1H), 8.11 (s, 1H), 8.04–7.98 (m, 2H), 7.63–7.59 (m, 2H), 7.58 (dd, <i>J</i> = 1.6, 0.8 Hz, 1H), 7.57–7.49 (m, 2H), 7.48 (ddt, <i>J</i> = 8.2, 1.8, 0.9 Hz, 1H), 7.13 (d, <i>J</i> = 7.7 Hz, 1H), 6.20 (br s, 1H), 6.16–6.12 (m, 2H), 6.05 (br s, <i>J</i> = 2.7 Hz, 1H), 5.62 (td, <i>J</i> = 4.2, 0.8 Hz, 1H), 4.65 (td, <i>J</i> = 7.6, 5.2 Hz, 1H), 4.46 (dt, <i>J</i> = 10.3, 3.5 Hz, 1H), 2.86 (ddt, <i>J</i> = 10.6, 9.0, 4.5 Hz, 1H), 2.23–2.17 (m, 1H), 2.16 (s, 3H), 2.12 (ddd, <i>J</i> = 10.7, 8.4, 5.4 Hz, 1H), 2.07 (s, 3H), 2.04–1.96 (m, 2H), 1.66 (ddt, <i>J</i> = 13.1, 10.7, 5.4 Hz, 1H), 1.56 (ddd, <i>J</i> = 8.9, 6.3, 4.6 Hz, 1H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>): δ 171.3, 170.0, 169.6, 169.3, 165.0, 157.2 (q, <sup>2</sup><i>J</i><sub>C–F</sub> = 38 Hz), 152.7, 151.7, 149.9, 142.5, 135.2, 133.4, 133.0, 132.8, 132.2, 132.1, 131.7, 128.9, 128.6, 128.5, 128.1, 127.4, 126.6, 126.5, 126.3 (q, <sup>2</sup><i>J</i><sub>C–F</sub> = 32 Hz), 124.7, 123.4 (q, <sup>1</sup><i>J</i><sub>C–F</sub> = 274 Hz), 115.7 (q, <sup>1</sup><i>J</i><sub>C–F</sub> = 287 Hz), 94.4, 87.0, 81.7, 81.0, 77.3, 73.6, 72.7, 53.1, 52.4, 38.1, 30.8, 29.7, 28.6, 20.7, 20.5. <sup>19</sup>F NMR (471 MHz, CDCl<sub>3</sub>, BTF IStd): δ −60.0, −76.7. HRMS (ESI+): calcd for [C<sub>39</sub>H<sub>36</sub>F<sub>6</sub>N<sub>7</sub>O<sub>10</sub>]<sup>+</sup> 876.2422, measured 876.2426, Δ 0.4 ppm.</div></div><div id="sec4_6_71" class="NLM_sec NLM_sec_level_3"><div id="ac_i133" class="anchor-spacer"></div><h4 class="article-section__title" id="_i133"> 2-Chloro-5-iodobenzamide (<b>S76</b>)</h4><div class="NLM_p last">To a vial charged with 2-chloro-5-iodobenzonitrile (1.32 g, 5.01 mmol) and <i>t</i>-BuOK (1.69 g, 15.0 mmol, 3.0 equiv) was added <i>t</i>-BuOH (10.0 mL) under a nitrogen atmosphere. The resulting mixture was stirred at 60 °C for 5 min, cooled to RT, and stirred for 1 h. The reaction was quenched by the addition of water (10 mL). The solid amide was filtered and recrystallized from absolute ethanol to give 2-chloro-5-iodobenzamide (<b>S76</b>) as a white solid (896 mg, 3.18 mmol, 64%). <i>R</i><sub><i>f</i></sub> = 0.35 (hexanes/EtOAc, 50:50). FTIR (thin film), ν<sub>max</sub> (cm<sup>–1</sup>): 3373, 3180, 1655, 1618, 1462, 1397, 1362. <sup>1</sup>H NMR (500 MHz, CD<sub>3</sub>OD): δ 7.82 (1H, d, <i>J</i> = 2.2 Hz), 7.76 (1H, dd, <i>J</i> = 8.5, 2.2 Hz), 7.24 (1H, d, <i>J</i> = 8.5 Hz). <sup>13</sup>C NMR (151 MHz, CD<sub>3</sub>OD): δ 170.5, 141.1, 139.2, 138.6, 132.9, 131.8, 92.0. HRMS (ESI+): calcd for [C<sub>7</sub>H<sub>6</sub>Cl<sub>1</sub>I<sub>1</sub>N<sub>1</sub>O<sub>1</sub>]<sup>+</sup> 281.9177, measured 281.9188, Δ 3.9 ppm.</div></div><div id="sec4_6_72" class="NLM_sec NLM_sec_level_3"><div id="ac_i134" class="anchor-spacer"></div><h4 class="article-section__title" id="_i134"> (2<i>R</i>,3<i>R</i>,4<i>R</i>,5<i>R</i>)-2-(6-Benzamido-9<i>H</i>-purin-9-yl)-5-((2<i>S</i>,5<i>S</i>)-2-((3-carbamoyl-4-chlorophenyl)ethynyl)-6-methoxy-6-oxo-5-(2,2,2-trifluoroacetamido)hexyl)tetrahydrofuran-3,4-diyl diacetate (<b>S77</b>)</h4><div class="NLM_p last">To a solution of alkyne <b>8</b> (50 mg, 73 μmol) and 2-chloro-5-iodobenzamide (<b>S76</b>) (51 mg, 0.18 mmol, 2.5 equiv) in a 5:1:1 mixture of toluene, DMF, and <i>i</i>-Pr<sub>2</sub>NEt (degassed by sparging with nitrogen for 15 min, 3.6 mL) at RT were introduced CuI (3 mg, 0.02 mmol, 25 mol %) and Pd(PPh<sub>3</sub>)<sub>4</sub> (4 mg, 4 μmol, 5 mol %) rapidly. The resulting mixture was stirred at 70 °C for 3 h. The solution was allowed to cool to RT, and volatiles were removed in vacuo. Crude material was filtered through a short pad of silica gel (EtOAc/<i>i</i>-PrOH, 90:10) to afford <b>S77</b>, which was further purified by preparative HPLC (15% → 95% B over 30 min) to afford <b>S77</b> (45 mg, 53 μmol, 74%) as a white amorphous solid. <i>R</i><sub><i>f</i></sub> = 0.48 (EtOAc/<i>i</i>-PrOH, 95:5). FTIR (thin film), ν<sub>max</sub> (cm<sup>–1</sup>): 3320, 3085, 1748, 1719, 1673, 1613, 1583, 1456, 1374, 1246, 1219, 1047. <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 9.29 (s, 1H), 8.74 (s, 1H), 8.12 (s, 1H), 8.04–7.97 (m, 2H), 7.69 (d, <i>J</i> = 2.0 Hz, 1H), 7.60–7.56 (m, 1H), 7.50–7.46 (m, 3H), 7.31 (dd, <i>J</i> = 8.3, 2.0 Hz, 1H), 7.28 (d, <i>J</i> = 8.3 Hz, 1H), 6.65 (s, 1H), 6.48 (s, 1H), 6.16–6.08 (m, 2H), 5.58 (app t, <i>J</i> = 4.5 Hz, 1H), 4.63 (app td, <i>J</i> = 7.8, 5.1 Hz, 1H), 4.46 (ddd, <i>J</i> = 10.5, 4.5, 3.1 Hz, 1H), 3.75 (s, 3H), 2.80 (dddd, <i>J</i> = 11.1, 9.1, 5.3, 4.6 Hz, 1H), 2.14 (s, 3H), 2.19–2.11 (m, 1H), 2.12–2.06 (m, 1H), 2.05 (s, 3H), 2.03–1.96 (m, 1H), 1.94 (ddd, <i>J</i> = 13.8, 11.1, 3.1 Hz, 1H), 1.63 (app ddt, <i>J</i> = 13.0, 10.8, 5.3 Hz, 1H), 1.58–1.49 (m, 1H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 171.4, 170.0, 169.7, 167.7, 165.0, 157.2 (q, <sup>2</sup><i>J</i><sub>C–F</sub> = 38 Hz), 152.8, 151.7, 149.9, 142.5, 134.4, 134.1, 133.6, 133.5, 133.0, 130.5, 130.5, 128.9, 128.1, 124.1, 122.5, 115.7 (q, <sup>1</sup><i>J</i><sub>C–F</sub> = 288 Hz), 92.5, 87.1, 81.9, 80.9, 73.6, 72.8, 53.2, 52.5, 38.3, 30.9, 29.8, 28.7, 20.8, 20.5. <sup>19</sup>F NMR (471 MHz, CDCl<sub>3</sub>, BTF IStd): δ −76.6. HRMS (ESI+): calcd for [C<sub>38</sub>H<sub>36</sub>Cl<sub>1</sub>F<sub>3</sub>N<sub>7</sub>O<sub>10</sub>]<sup>+</sup> 842.2159, measured 842.2168, Δ 1.0 ppm.</div></div><div id="sec4_6_73" class="NLM_sec NLM_sec_level_3"><div id="ac_i135" class="anchor-spacer"></div><h4 class="article-section__title" id="_i135"> (2<i>R</i>,3<i>R</i>,4<i>R</i>,5<i>R</i>)-2-(6-Benzamido-9<i>H</i>-purin-9-yl)-5-((2<i>S</i>,5<i>S</i>)-6-methoxy-6-oxo-2-((1-oxo-1,2,3,4-tetrahydroisoquinolin-7-yl)ethynyl)-5-(2,2,2-trifluoroacetamido)hexyl)tetrahydrofuran-3,4-diyl Diacetate (<b>S78</b>)</h4><div class="NLM_p last">A vial was charged with alkyne <b>8</b> (40 mg, 58 μmol), 7-bromo-3,4-dihydroisoquinolin-1(2<i>H</i>)-one (33 mg, 0.15 mmol, 2.5 equiv), and CuI (3 mg, 0.02 mmol, 25 mol %). The vial headspace was purged with nitrogen for 5 min, and a 5:1:1.5 mixture of toluene, DMF, and <i>i</i>-Pr<sub>2</sub>NH (degassed by sparging with nitrogen for 15 min, 1.5 mL) was added at RT. The resulting mixture was further degassed by sparging with nitrogen for 5 min, and a solution of Pd(P<i>t</i>-Bu<sub>3</sub>)<sub>2</sub> (3 mg, 6 μmol, 10 mol %) in a 5:1:1.5 mixture of toluene, DMF, and <i>i</i>-Pr<sub>2</sub>NH (1.5 mL) was added. The resulting mixture was stirred at 55 °C for 6 h. The solution was allowed to cool to RT, and volatiles were removed in vacuo. Crude material was filtered through a short pad of silica gel (EtOAc/<i>i</i>-PrOH, 70:30) to afford <b>S78</b>, which was further purified by preparative HPLC (15% → 95% B over 30 min) to afford <b>S78</b> (29 mg, 35 μmol, 60%) as a white amorphous solid. FTIR (thin film), ν<sub>max</sub> (cm<sup>–1</sup>): 3262, 3082, 2938, 1749, 1720, 1681, 1612, 1579, 1513, 1482, 1456, 1373, 1243, 1218, 1097, 1048. <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 9.16 (s, 1H), 8.78 (s, 1H), 8.78 (s, 1H), 8.13 (s, 1H), 8.05–8.00 (m, 2H), 7.62–7.58 (m, 1H), 7.54–7.49 (m, 2H), 7.24 (br d, <i>J</i> = 7.7 Hz, 1H), 7.08 (d, <i>J</i> = 7.8 Hz, 1H), 7.01 (dd, <i>J</i> = 7.8, 1.2 Hz, 1H), 6.83 (d, <i>J</i> = 1.2 Hz, 1H), 6.14 (d, <i>J</i> = 5.6 Hz, 1H), 6.11 (dd, <i>J</i> = 5.6, 5.3 Hz, 1H), 5.59 (dd, <i>J</i> = 5.3, 4.3 Hz, 1H), 4.64 (app td, <i>J</i> = 7.7, 5.6 Hz, 1H), 4.49 (ddd, <i>J</i> = 10.4, 4.3, 3.1 Hz, 1H), 3.76 (s, 3H), 2.93 (app dd, <i>J</i> = 8.5, 6.6 Hz, 2H), 2.82–2.76 (m, 1H), 2.61 (app dd, <i>J</i> = 8.5, 6.6 Hz, 2H), 2.17–2.10 (m, 1H), 2.15 (s, 3H), 2.10–2.05 (m, 1H), 2.06 (s, 3H), 2.04–1.97 (m, 1H), 1.96 (ddd, <i>J</i> = 13.8, 11.0, 3.1 Hz, 1H), 1.64 (app ddt, <i>J</i> = 12.9, 10.6, 5.3 Hz, 1H), 1.57–1.49 (m, 1H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 171.8, 171.4, 169.9, 169.6, 164.9, 157.1 (q, <sup>2</sup><i>J</i><sub>C–F</sub> = 38 Hz), 152.8, 151.8, 150.0, 142.4, 137.4, 133.5, 133.0, 129.0, 128.1, 128.0, 126.7, 124.2, 124.0, 122.5, 118.4, 115.7 (q, <sup>1</sup><i>J</i><sub>C–F</sub> = 288 Hz), 90.4, 87.0, 83.1, 81.1, 73.8, 72.9, 53.2, 52.5, 38.7, 31.1, 30.6, 30.0, 28.8, 25.4, 20.8, 20.6. <sup>19</sup>F NMR (471 MHz, CDCl<sub>3</sub>, BTF IStd): δ −76.6. HRMS (ESI+): calcd for [C<sub>40</sub>H<sub>39</sub>F<sub>3</sub>N<sub>7</sub>O<sub>10</sub>]<sup>+</sup> 834.2705, measured 834.2702, Δ 0.3 ppm.</div></div><div id="sec4_6_74" class="NLM_sec NLM_sec_level_3"><div id="ac_i136" class="anchor-spacer"></div><h4 class="article-section__title" id="_i136"> 7-Iodobenzo[<i>d</i>][1,3]dioxole-5-carboxamide (<b>S80</b>)</h4><div class="NLM_p last">To a solution of known aldehyde 7-iodobenzo[<i>d</i>][1,3]dioxole-5-carbaldehyde (<b>S79</b>)<a onclick="showRef(event, 'ref78 ref79'); return false;" href="javascript:void(0);" class="ref ref78 ref79">(78,79)</a> (500 mg, 1.81 mmol) in <i>t</i>-BuOH (20 mL) at RT was added a solution of NaClO<sub>2</sub> (328 mg, 3.62 mmol, 2.0 equiv) and NaH<sub>2</sub>PO<sub>4</sub> (0.11 g, 0.91 mmol, 0.50 equiv) in water (4 mL). The resulting pale-yellow solution was cooled to 0 °C, and 2-methyl-2-butene (4.0 mL, 38 mmol, 21 equiv) was added dropwise. The reaction mixture was stirred at RT for 20 h, upon which more NaClO<sub>2</sub> (164 mg, 1.81 mmol, 1.0 equiv) and NaH<sub>2</sub>PO<sub>4</sub> (54 mg, 0.45 mmol, 0.25 equiv) were added as a solution in water (2 mL). The resulting mixture was stirred at RT for 72 h and diluted with a 10% H<sub>3</sub>PO<sub>4</sub> aq solution (10 mL). Water (20 mL) and EtOAc (40 mL) were added. The layers were separated, and the aqueous phase was extracted with EtOAc (2 × 40 mL). The combined organic extracts were washed with brine (80 mL), dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated to give a white solid. Crude material was used directly in the next step without further purification. To a flask charged with the crude carboxylic acid and HATU (758 mg, 1.99 mmol, 1.1 equiv) were added DMF (20 mL) and <i>i</i>-Pr<sub>2</sub>NEt (0.63 mL, 3.6 mmol, 2.0 equiv) at RT. The resulting brown mixture was stirred for 30 min, and NH<sub>4</sub>Cl (116 mg, 2.18 mmol, 1.2 equiv) was added. The flask headspace was flushed with nitrogen, and the reaction mixture was stirred for 24 h, before the addition of EtOAc (30 mL) and water (20 mL). The layers were separated and the aqueous phase was extracted with EtOAc (3 × 30 mL). The combined organic extracts were washed with a 5% LiCl aq solution (100 mL) and brine (2 × 100 mL), dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated. Crude material was purified by silica gel chromatography (hexanes/EtOAc, 5 → 100%) to afford iodobenzamide <b>S80</b> (348 mg, 1.20 mmol, 66% over 2 steps) as a white solid. <i>R</i><sub><i>f</i></sub> = 0.15 (hexanes/EtOAc, 50:50). FTIR (thin film), ν<sub>max</sub> (cm<sup>–1</sup>): 3355, 3187, 2924, 2854, 1658, 1592, 1479, 1424, 1254, 1044. <sup>1</sup>H NMR (500 MHz, CD<sub>3</sub>OD): δ 7.77 (d, <i>J</i> = 1.6 Hz, 1H), 7.31 (d, <i>J</i> = 1.6 Hz, 1H), 6.11 (s, 2H). <sup>13</sup>C NMR (101 MHz, CD<sub>3</sub>OD): δ 170.1, 153.9, 148.1, 132.3, 130.7, 108.7, 103.0, 70.6. HRMS (ESI+): calcd for [C<sub>8</sub>H<sub>7</sub>I<sub>1</sub>N<sub>1</sub>O<sub>3</sub>]<sup>+</sup> 291.9465, measured 291.9461, Δ 1.4 ppm.</div></div><div id="sec4_6_75" class="NLM_sec NLM_sec_level_3"><div id="ac_i137" class="anchor-spacer"></div><h4 class="article-section__title" id="_i137"> (2<i>R</i>,3<i>R</i>,4<i>R</i>,5<i>R</i>)-2-(6-Benzamido-9<i>H</i>-purin-9-yl)-5-((2<i>S</i>,5<i>S</i>)-2-((6-carbamoylbenzo[<i>d</i>][1,3]dioxol-4-yl)ethynyl)-6-methoxy-6-oxo-5-(2,2,2-trifluoroacetamido)hexyl)tetrahydrofuran-3,4-diyl Diacetate (<b>S81</b>)</h4><div class="NLM_p last">To a vial charged with alkyne <b>8</b> (30 mg, 44 μmol) were added iodobenzamide <b>S80</b> (32 mg, 0.11 mmol, 2.5 equiv), CuI (2 mg, 0.01 mmol, 25 mol %), and Pd(PPh<sub>3</sub>)<sub>4</sub> (3 mg, 2 μmol, 5 mol %). The vial headspace was purged with nitrogen for 5 min, and a 5:1:1 mixture of toluene, DMF, and <i>i</i>-Pr<sub>2</sub>NEt (degassed by sparging with nitrogen for 15 min, 2.3 mL) was added at RT. The resulting mixture was stirred at 60 °C for 3 h. The solution was allowed to cool to RT, and volatiles were removed in vacuo. Crude material was filtered through a short pad of silica gel (EtOAc/<i>i</i>-PrOH, 80:20) to afford <b>S81</b>, which was further purified by preparative HPLC (15% → 95% B′ over 30 min) to afford <b>S81</b> (27 mg, 32 μmol, 73%) as a white amorphous solid. FTIR (thin film), ν<sub>max</sub> (cm<sup>–1</sup>): 3351, 3222, 3077, 2937, 1746, 1717, 1659, 1593, 1424, 1243, 1203, 1045. <sup>1</sup>H NMR (500 MHz, CD<sub>3</sub>OD): δ 8.80 (s, 1H), 8.70 (s, 1H), 8.12–8.06 (m, 2H), 7.70–7.65 (m, 1H), 7.60–7.55 (m, 2H), 7.45 (d, <i>J</i> = 1.7 Hz, 1H), 7.28 (d, <i>J</i> = 1.7 Hz, 1H), 6.34 (d, <i>J</i> = 5.0 Hz, 1H), 6.14 (dd, <i>J</i> = 5.7, 5.0 Hz, 1H), 6.10 (d, <i>J</i> = 1.1 Hz, 1H), 6.10 (d, <i>J</i> = 1.1 Hz, 1H), 5.65 (dd, <i>J</i> = 5.7, 5.1 Hz, 1H), 4.58 (ddd, <i>J</i> = 10.2, 5.1, 3.2 Hz, 1H), 4.52 (dd, <i>J</i> = 9.6, 5.0 Hz, 1H), 3.73 (s, 3H), 2.93–2.86 (m, 1H), 2.22 (ddd, <i>J</i> = 14.1, 10.2, 4.1 Hz, 1H), 2.15 (s, 3H), 2.17–2.10 (m, 1H), 2.06 (s, 3H), 2.07–1.98 (m, 2H), 1.72–1.58 (m, 2H). <sup>13</sup>C NMR (126 MHz, CD<sub>3</sub>OD): δ 172.5, 171.6, 171.3, 170.7, 168.7, 159.1 (q, <sup>2</sup><i>J</i><sub>C–F</sub> = 38 Hz), 153.0, 152.9, 152.2, 150.4, 149.4, 145.6, 134.4, 134.3, 129.8, 129.6, 128.9, 126.7, 124.3, 117.0 (q, <sup>1</sup><i>J</i><sub>C–F</sub> = 288 Hz), 108.4, 105.6, 103.9, 96.7, 88.9, 82.2, 77.8, 74.9, 74.3, 53.7, 53.1, 39.3, 32.5, 29.9, 29.6, 20.5, 20.3. <sup>19</sup>F NMR (471 MHz, CDCl<sub>3</sub>, BTF IStd): δ −76.7. HRMS (ESI+): calcd for [C<sub>39</sub>H<sub>37</sub>F<sub>3</sub>N<sub>7</sub>O<sub>12</sub>]<sup>+</sup> 852.2447, measured 852.2483, Δ 4.3 ppm.</div></div><div id="sec4_6_76" class="NLM_sec NLM_sec_level_3"><div id="ac_i138" class="anchor-spacer"></div><h4 class="article-section__title" id="_i138"> (2<i>R</i>,3<i>R</i>,4<i>R</i>,5<i>R</i>)-2-(6-Benzamido-9<i>H</i>-purin-9-yl)-5-((2<i>S</i>,5<i>S</i>)-2-((6-carbamoylpyridin-2-yl)ethynyl)-6-methoxy-6-oxo-5-(2,2,2-trifluoroacetamido)hexyl)tetrahydrofuran-3,4-diyl Diacetate (<b>S82</b>)</h4><div class="NLM_p last">A vial was charged with alkyne <b>8</b> (35 mg, 51 μmol), 6-bromopyridine-2-carboxamide (26 mg, 0.13 mmol, 2.5 equiv), and CuI (2 mg, 0.01 mmol, 25 mol %). The vial headspace was purged with nitrogen for 5 min, and a 5:1:1.5 mixture of toluene, DMF, and <i>i</i>-Pr<sub>2</sub>NH (degassed by sparging with nitrogen for 15 min, 1.3 mL) was added at RT. The resulting mixture was further degassed by sparging with nitrogen for 5 min, and a solution of Pd(P<i>t</i>-Bu<sub>3</sub>)<sub>2</sub> (3 mg, 5 μmol, 10 mol %) in a 5:1:1.5 mixture of toluene, DMF, and <i>i</i>-Pr<sub>2</sub>NH (1.3 mL) was added. The resulting mixture was stirred at 50 °C for 5 h. The solution was allowed to cool to RT, and volatiles were removed in vacuo. Crude material was filtered through a short pad of silica gel (EtOAc/<i>i</i>-PrOH, 95:5) to afford <b>S82</b>, which was further purified by preparative HPLC (15% → 95% B over 30 min) to afford <b>S82</b> (28 mg, 35 μmol, 68%) as a white amorphous solid. <i>R</i><sub><i>f</i></sub> = 0.43 (EtOAc/<i>i</i>-PrOH, 95:5). FTIR (thin film), ν<sub>max</sub> (cm<sup>–1</sup>): 3292, 2928, 1748, 1718, 1690, 1611, 1584, 1511, 1487, 1457, 1377, 1245, 1220, 1161, 1075. <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 9.16 (s, 1H), 8.78 (s, 1H), 8.14 (s, 1H), 8.10 (dd, <i>J</i> = 7.8, 1.1 Hz, 1H), 8.05–7.99 (m, 2H), 7.95 (d, <i>J</i> = 4.0 Hz, 1H), 7.78 (app t, <i>J</i> = 7.8 Hz, 1H), 7.62–7.57 (m, 1H), 7.54–7.48 (m, 3H), 7.33 (br d, <i>J</i> = 7.7 Hz, 1H), 6.15 (d, <i>J</i> = 5.9 Hz, 1H), 6.14 (dd, <i>J</i> = 5.9, 4.8 Hz, 1H), 5.92 (d, <i>J</i> = 4.0 Hz, 1H), 5.60 (dd, <i>J</i> = 4.8, 3.9 Hz, 1H), 4.66 (app td, <i>J</i> = 7.7, 5.1 Hz, 1H), 4.51 (ddd, <i>J</i> = 10.2, 3.9, 3.4 Hz, 1H), 3.77 (s, 3H), 2.90–2.83 (m, 1H), 2.22 (ddd, <i>J</i> = 14.0, 10.2, 4.1 Hz, 1H), 2.15 (s, 3H), 2.18–2.11 (m, 1H), 2.09–2.02 (m, 2H), 2.06 (s, 3H), 1.71 (app ddt, <i>J</i> = 13.1, 10.7, 5.4 Hz, 1H), 1.65–1.57 (m, 1H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 171.3, 169.9, 169.6, 166.3, 164.9, 157.1 (q, <sup>2</sup><i>J</i><sub>C–F</sub> = 38 Hz), 152.8, 151.8, 150.0, 149.9, 142.4, 141.9, 137.6, 133.5, 133.0, 129.8, 129.0, 128.1, 124.2, 121.7, 115.7 (q, <sup>1</sup><i>J</i><sub>C–F</sub> = 288 Hz), 91.3, 87.0, 82.9, 81.2, 73.8, 72.7, 53.2, 52.5, 38.3, 30.8, 29.8, 28.7, 20.8, 20.5. <sup>19</sup>F NMR (471 MHz, CDCl<sub>3</sub>, BTF IStd): δ −76.6. HRMS (ESI+): calcd for [C<sub>37</sub>H<sub>36</sub>F<sub>3</sub>N<sub>8</sub>O<sub>10</sub>]<sup>+</sup> 809.2501, measured 809.2504, Δ 0.4 ppm.</div></div><div id="sec4_6_77" class="NLM_sec NLM_sec_level_3"><div id="ac_i139" class="anchor-spacer"></div><h4 class="article-section__title" id="_i139"> (2<i>R</i>,3<i>R</i>,4<i>R</i>,5<i>R</i>)-2-(6-Benzamido-9<i>H</i>-purin-9-yl)-5-((2<i>S</i>,5<i>S</i>)-2-((2-carbamoylpyridin-4-yl)ethynyl)-6-methoxy-6-oxo-5-(2,2,2-trifluoroacetamido)hexyl)tetrahydrofuran-3,4-diyl Diacetate (<b>S83</b>)</h4><div class="NLM_p last">A vial was charged with alkyne <b>8</b> (40 mg, 58 μmol), 4-bromopicolinamide (29 mg, 0.15 mmol, 2.5 equiv), and CuI (3 mg, 0.02 mmol, 25 mol %). The vial headspace was purged with nitrogen for 5 min, and a 5:1:1.5 mixture of toluene, DMF, and <i>i</i>-Pr<sub>2</sub>NH (degassed by sparging with nitrogen for 15 min, 1.5 mL) was added at RT. The resulting mixture was further degassed by sparging with nitrogen for 5 min, and a solution of Pd(P<i>t</i>-Bu<sub>3</sub>)<sub>2</sub> (3 mg, 6 μmol, 10 mol %) in a 5:1:1.5 mixture of toluene, DMF, and <i>i</i>-Pr<sub>2</sub>NH (1.5 mL) was added. The resulting mixture was stirred at 50 °C for 5 h. The solution was allowed to cool to RT, and volatiles were removed in vacuo. Crude material was filtered through a short pad of silica gel (EtOAc/<i>i</i>-PrOH, 70:30) to afford <b>S83</b>, which was further purified by preparative HPLC (15% → 95% B over 30 min) to afford <b>S83</b> (35 mg, 43 μmol, 75%) as a white amorphous solid. FTIR (thin film), ν<sub>max</sub> (cm<sup>–1</sup>): 3283, 3079, 2931, 1748, 1721, 1690, 1611, 1599, 1583, 1456, 1356, 1243, 1218, 1074. <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 9.07 (s, 1H), 8.78 (s, 1H), 8.48 (dd, <i>J</i> = 5.0, 0.8 Hz, 1H), 8.12 (dd, <i>J</i> = 1.6, 0.8 Hz, 1H), 8.11 (s, 1H), 8.04–8.00 (m, 2H), 7.82 (d, <i>J</i> = 4.2 Hz, 1H), 7.62–7.58 (m, 1H), 7.54–7.49 (m, 2H), 7.38 (dd, <i>J</i> = 5.0, 1.6 Hz, 1H), 7.31 (d, <i>J</i> = 7.8 Hz, 1H), 6.15–6.11 (m, 2H), 5.93 (d, <i>J</i> = 4.2 Hz, 1H), 5.62–5.58 (m, 1H), 4.65 (app td, <i>J</i> = 7.8, 5.4 Hz, 1H), 4.47 (ddd, <i>J</i> = 10.8, 4.3, 3.0 Hz, 1H), 3.78 (s, 3H), 2.88 (app ddt, <i>J</i> = 10.8, 9.2, 4.6 Hz, 1H), 2.22–2.15 (m, 1H), 2.17 (s, 3H), 2.11 (ddt, <i>J</i> = 14.2, 11.0, 5.4 Hz, 1H), 2.08 (s, 3H), 2.03–1.95 (m, 2H), 1.67 (app ddt, <i>J</i> = 13.4, 10.8, 5.4 Hz, 1H), 1.57 (dddd, <i>J</i> = 13.4, 11.0, 9.2, 4.6 Hz, 1H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 171.3, 169.9, 169.7, 166.4, 164.8, 157.1 (q, <sup>2</sup><i>J</i><sub>C–F</sub> = 38 Hz), 152.9, 151.7, 149.9, 149.6, 148.5, 142.5, 133.6, 133.0, 133.0, 129.0, 128.4, 128.1, 124.8, 124.2, 115.7 (q, <sup>1</sup><i>J</i><sub>C–F</sub> = 288 Hz), 96.7, 87.3, 81.1, 80.8, 73.7, 72.9, 53.3, 52.4, 38.3, 30.8, 29.9, 28.9, 20.8, 20.6. <sup>19</sup>F NMR (471 MHz, CDCl<sub>3</sub>, BTF IStd): δ −76.6. HRMS (ESI+): calcd for [C<sub>37</sub>H<sub>36</sub>F<sub>3</sub>N<sub>8</sub>O<sub>10</sub>]<sup>+</sup> 809.2501, measured 809.2491, Δ 1.2 ppm.</div></div><div id="sec4_6_78" class="NLM_sec NLM_sec_level_3"><div id="ac_i140" class="anchor-spacer"></div><h4 class="article-section__title" id="_i140"> (2<i>R</i>,3<i>R</i>,4<i>R</i>,5<i>R</i>)-2-(6-Benzamido-9<i>H</i>-purin-9-yl)-5-((2<i>S</i>,5<i>S</i>)-2-((5-carbamoylpyridin-3-yl)ethynyl)-6-methoxy-6-oxo-5-(2,2,2-trifluoroacetamido)hexyl)tetrahydrofuran-3,4-diyl Diacetate (<b>S84</b>)</h4><div class="NLM_p last">A vial was charged with alkyne <b>8</b> (40 mg, 58 μmol), 5-bromonicotinamide (29 mg, 0.15 mmol, 2.5 equiv), and CuI (3 mg, 0.02 mmol, 25 mol %). The vial headspace was purged with nitrogen for 5 min, and a 5:1:1.5 mixture of toluene, DMF, and <i>i</i>-Pr<sub>2</sub>NH (degassed by sparging with nitrogen for 15 min, 1.5 mL) was added at RT. The resulting mixture was further degassed by sparging with nitrogen for 5 min, and a solution of Pd(P<i>t</i>-Bu<sub>3</sub>)<sub>2</sub> (3 mg, 6 μmol, 10 mol %) in a 5:1:1.5 mixture of toluene, DMF, and <i>i</i>-Pr<sub>2</sub>NH (1.5 mL) was added. The resulting mixture was stirred at 50 °C for 5 h. The solution was allowed to cool to RT, and volatiles were removed in vacuo. Crude material was filtered through a short pad of silica gel (EtOAc/<i>i</i>-PrOH, 70:30) to afford <b>S84</b>, which was further purified by preparative HPLC (15% → 95% B over 30 min) to afford <b>S84</b> (26 mg, 32 μmol, 55%) as a white amorphous solid. FTIR (thin film), ν<sub>max</sub> (cm<sup>–1</sup>): 3263, 3085, 2954, 2928, 1748, 1720, 1677, 1612, 1583, 1456, 1376, 1245, 1219, 1075. <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 9.11 (s, 1H), 8.94 (d, <i>J</i> = 1.4 Hz, 1H), 8.77 (s, 1H), 8.67 (d, <i>J</i> = 2.0 Hz, 1H), 8.16–8.11 (m, 2H), 8.01 (m, 2H), 7.63–7.58 (m, 1H), 7.55–7.48 (m, 2H), 7.46 (br d, <i>J</i> = 7.5 Hz, 1H), 6.83 (br s, 1H), 6.17 (dd, <i>J</i> = 5.7, 5.1 Hz, 1H), 6.15 (d, <i>J</i> = 5.7 Hz, 1H), 6.05 (br s, 1H), 5.66 (dd, <i>J</i> = 5.1, 4.0 Hz, 1H), 4.65 (app td, <i>J</i> = 7.5, 5.4 Hz, 1H), 4.48 (ddd, <i>J</i> = 9.8, 4.0, 3.8 Hz, 1H), 3.79 (s, 3H), 2.88–2.82 (m, 1H), 2.21–2.15 (m, 1H), 2.16 (s, 3H), 2.15–2.08 (m, 1H), 2.07 (s, 3H), 2.08–1.99 (m, 2H), 1.72–1.64 (m, 1H), 1.63–1.54 (m, 1H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 171.3, 170.1, 169.8, 166.9, 165.0, 157.3 (q, <sup>2</sup><i>J</i><sub>C–F</sub> = 38 Hz), 154.5, 152.8, 151.8, 149.9, 147.6, 142.6, 138.1, 133.5, 133.1, 129.0, 128.5, 128.1, 124.3, 120.2, 115.7 (q, <sup>1</sup><i>J</i><sub>C–F</sub> = 288 Hz), 95.3, 87.1, 81.2, 79.9, 73.7, 72.7, 53.3, 52.6, 38.3, 30.9, 29.7, 28.9, 20.8, 20.6. <sup>19</sup>F NMR (471 MHz, CDCl<sub>3</sub>, BTF IStd): δ −76.6. HRMS (ESI+): calcd for [C<sub>37</sub>H<sub>36</sub>F<sub>3</sub>N<sub>8</sub>O<sub>10</sub>]<sup>+</sup> 809.2501, measured 809.2511, Δ 1.2 ppm.</div></div><div id="sec4_6_79" class="NLM_sec NLM_sec_level_3"><div id="ac_i141" class="anchor-spacer"></div><h4 class="article-section__title" id="_i141"> (2<i>R</i>,3<i>R</i>,4<i>R</i>,5<i>R</i>)-2-(6-Benzamido-9<i>H</i>-purin-9-yl)-5-((2<i>S</i>,5<i>S</i>)-2-((4-carbamoylpyridin-2-yl)ethynyl)-6-methoxy-6-oxo-5-(2,2,2-trifluoroacetamido)hexyl)tetrahydrofuran-3,4-diyl Diacetate (<b>S85</b>)</h4><div class="NLM_p last">A vial was charged with alkyne <b>8</b> (35 mg, 51 μmol), 2-bromo-isonicotinamide (26 mg, 0.13 mmol, 2.5 equiv), and CuI (2 mg, 0.01 mmol, 25 mol %). The vial headspace was purged with nitrogen for 5 min, and a 5:1:1.5 mixture of toluene, DMF, and <i>i</i>-Pr<sub>2</sub>NH (degassed by sparging with nitrogen for 15 min, 1.3 mL) was added at RT. The resulting mixture was further degassed by sparging with nitrogen for 5 min, and a solution of Pd(P<i>t</i>-Bu<sub>3</sub>)<sub>2</sub> (3 mg, 5 μmol, 10 mol %) in a 5:1:1.5 mixture of toluene, DMF, and <i>i</i>-Pr<sub>2</sub>NH (1.3 mL) was added. The resulting mixture was stirred at 50 °C for 5 h. The solution was allowed to cool to RT, and volatiles were removed in vacuo. Crude material was filtered through a short pad of silica gel (EtOAc/<i>I</i>-PrOH, 95:5) to afford <b>S85</b>, which was further purified by preparative HPLC (15% → 95% B over 30 min) to afford <b>S85</b> (22 mg, 27 μmol, 54%) as a light-yellow amorphous solid. <i>R</i><sub><i>f</i></sub> = 0.31 (EtOAc/<i>i</i>-PrOH, 95:5). FTIR (thin film), ν<sub>max</sub> (cm<sup>–1</sup>): 3289, 2925, 1747, 1719, 1681, 1612, 1583, 1549, 1457, 1414, 1376, 1245, 1220, 1162, 1047. <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 9.18 (s, 1H), 8.77 (s, 1H), 8.64 (dd, <i>J</i> = 5.2, 0.9 Hz, 1H), 8.13 (s, 1H), 8.05–7.98 (m, 2H), 7.79 (dd, <i>J</i> = 1.7, 0.9 Hz, 1H), 7.65 (dd, <i>J</i> = 5.2, 1.7 Hz, 1H), 7.62–7.58 (m, 1H), 7.59–7.56 (m, 1H), 7.55–7.48 (m, 2H), 7.11 (br s, 1H), 6.14 (br s, 1H), 6.14 (dd, <i>J</i> = 5.4, 5.0 Hz, 1H), 6.12 (d, <i>J</i> = 5.4 Hz, 1H), 5.66 (dd, <i>J</i> = 5.0, 4.4 Hz, 1H), 4.62 (app q, <i>J</i> = 6.9 Hz, 1H), 4.53 (ddd, <i>J</i> = 10.1, 4.4, 3.6 Hz, 1H), 3.77 (s, 3H), 2.85 (app ddt, <i>J</i> = 10.9, 9.0, 4.3 Hz, 1H), 2.15 (s, 3H), 2.19–2.09 (m, 3H), 2.06 (s, 3H), 2.07–2.01 (m, 1H), 1.73–1.66 (m, 1H), 1.65–1.57 (m, 1H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 171.2, 170.1, 169.8, 166.8, 165.0, 157.4 (q, <sup>2</sup><i>J</i><sub>C–F</sub> = 38 Hz), 152.8, 151.7, 150.7, 149.9, 143.9, 142.6, 141.0, 133.5, 133.1, 129.0, 128.1, 125.2, 124.2, 120.9, 115.7 (q, <sup>1</sup><i>J</i><sub>C–F</sub> = 288 Hz), 92.3, 87.2, 83.1, 80.9, 73.7, 72.9, 53.3, 52.8, 38.2, 31.0, 29.6, 28.8, 20.8, 20.6. <sup>19</sup>F NMR (471 MHz, CDCl<sub>3</sub>, BTF IStd): δ −76.6. HRMS (ESI+): calcd for [C<sub>37</sub>H<sub>36</sub>F<sub>3</sub>N<sub>8</sub>O<sub>10</sub>]<sup>+</sup> 809.2501, measured 809.2502, Δ 0.1 ppm.</div></div><div id="sec4_6_80" class="NLM_sec NLM_sec_level_3"><div id="ac_i142" class="anchor-spacer"></div><h4 class="article-section__title" id="_i142"> (2<i>R</i>,3<i>R</i>,4<i>R</i>,5<i>R</i>)-2-((2<i>S</i>,5<i>S</i>)-2-((5-Aminonaphthalen-1-yl)ethynyl)-6-methoxy-6-oxo-5-(2,2,2-trifluoroacetamido)hexyl)-5-(6-benzamido-9<i>H</i>-purin-9-yl)tetrahydrofuran-3,4-diyl Diacetate (<b>S86</b>)</h4><div class="NLM_p last">A vial was charged with alkyne <b>8</b> (42 mg, 61 μmol), 5-bromo-naphthalen-1-ylamine (34 mg, 0.15 mmol, 2.5 equiv), and CuI (3 mg, 0.02 mmol, 25 mol %). The vial headspace was purged with nitrogen for 5 min, and a 5:1:1.5 mixture of toluene, DMF, and <i>i</i>-Pr<sub>2</sub>NH (degassed by sparging with nitrogen for 15 min, 1.5 mL) was added at RT. The resulting mixture was further degassed by sparging with nitrogen for 5 min, and a solution of Pd(P<i>t</i>-Bu<sub>3</sub>)<sub>2</sub> (3 mg, 6 μmol, 10 mol %) in a 5:1:1.5 mixture of toluene, DMF, and <i>i</i>-Pr<sub>2</sub>NH (1.5 mL) was added. The resulting mixture was stirred at 50 °C for 5 h. The solution was allowed to cool to RT, and volatiles were removed in vacuo. Crude material was filtered through a short pad of silica gel (EtOAc/<i>i</i>PrOH, 70:30) to afford <b>S86</b>, which was further purified by preparative HPLC (20% → 95% B over 30 min) to afford <b>S86</b> (34 mg, 41 μmol, 67%) as a light-yellow amorphous solid. <i>R</i><sub><i>f</i></sub> = 0.42 (EtOAc). FTIR (thin film), ν<sub>max</sub> (cm<sup>–1</sup>): 3315, 3066, 2924, 1748, 1721, 1611, 1582, 1514, 1456, 1369, 1243, 1218, 1179, 1074. <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 9.02 (s, 1H), 8.80 (s, 1H), 8.12 (s, 1H), 8.03–7.99 (m, 2H), 7.80 (app dt, <i>J</i> = 8.5, 1.0 Hz, 1H), 7.70 (app dt, <i>J</i> = 8.3, 1.0 Hz, 1H), 7.62–7.58 (m, 2H), 7.53–7.50 (m, 2H), 7.37 (dd, <i>J</i> = 8.5, 7.1 Hz, 1H), 7.34 (dd, <i>J</i> = 8.3, 7.4 Hz, 1H), 7.04 (br d, <i>J</i> = 7.7 Hz, 1H), 6.80 (dd, <i>J</i> = 7.4, 1.0 Hz, 1H), 6.15 (d, <i>J</i> = 5.3 Hz, 1H), 6.09 (dd, <i>J</i> = 5.5, 5.3 Hz, 1H), 5.61 (dd, <i>J</i> = 5.5, 4.8 Hz, 1H), 4.69 (app td, <i>J</i> = 7.7, 4.9 Hz, 1H), 4.65 (ddd, <i>J</i> = 10.8, 4.8, 2.8 Hz, 1H), 4.32 (br s, 2H), 3.74 (s, 3H), 2.97 (app ddt, <i>J</i> = 10.8, 9.1, 4.5 Hz, 1H), 2.24–2.16 (m, 2H), 2.13 (s, 3H), 2.14–2.04 (m, 2H), 2.06 (s, 3H), 1.72 (dddd, <i>J</i> = 13.3, 10.8, 6.0, 4.8 Hz, 1H), 1.61 (dddd, <i>J</i> = 13.3, 10.9, 9.4, 4.5 Hz, 1H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 171.3, 169.9, 169.6, 164.7, 157.0 (q, <sup>2</sup><i>J</i><sub>C–F</sub> = 38 Hz), 152.9, 151.7, 149.9, 142.7, 142.3, 134.5, 133.6, 133.0, 130.7, 129.0, 128.0, 127.3, 124.1, 124.1, 123.5, 121.5, 121.3, 117.0, 115.7 (q, <sup>1</sup><i>J</i><sub>C–F</sub> = 288 Hz), 110.4, 94.9, 87.2, 82.4, 81.0, 73.7, 73.1, 53.2, 52.5, 38.9, 31.2, 30.3, 29.3, 20.8, 20.6. <sup>19</sup>F NMR (471 MHz, CDCl<sub>3</sub>, BTF IStd): δ −76.7. HRMS (ESI+): calcd for [C<sub>41</sub>H<sub>39</sub>F<sub>3</sub>N<sub>7</sub>O<sub>9</sub>]<sup>+</sup> 830.2756, measured 830.2772, Δ 1.9 ppm.</div></div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i143" class="anchor-spacer"></div><h3 class="article-section__title" id="_i143"> Cellular Thermal Shift Assay (CETSA)</h3><div id="sec4_7_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i144" class="anchor-spacer"></div><h4 class="article-section__title" id="_i144"> K562 Cell Culture</h4><div class="NLM_p last">The human chronic myeloid leukemia cell line K562 (ATCC CCL-243) was cultured in RPMI 1640 with Glutamax (Gibco) supplemented with 10% fetal bovine serum (Atlanta Biologicals) and 100 IU penicillin and 100 mg/mL streptomycin (Corning). Cells were cultured in T175 flasks with 50 mL of media per flask (Corning) in a 37 °C incubator with 5% CO<sub>2</sub> until the cells reached a density of 2 × 106 cells/mL.</div></div><div id="sec4_7_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i145" class="anchor-spacer"></div><h4 class="article-section__title" id="_i145"> Preparation of K562 Cell Lysate</h4><div class="NLM_p last">K562 cells were cultured as described above. Cells were washed with PBS and then resuspended in an appropriate volume of PBS supplemented with cOmplete protease inhibitor cocktail (Roche) to obtain a final cell count of 3.0 × 10<sup>7</sup> to 3.6 × 10<sup>7</sup> cells/mL. The suspension was transferred to microcentrifuge tubes and subjected to three rounds of freeze–thaw using liquid nitrogen and a room temperature water bath. The cell lysate was clarified by centrifuging at 17000<i>g</i> for 20 min at 4 °C, and the supernatant was transferred to a new microcentrifuge tube. Lysate protein concentration was determined by BCA assay, and the lysate was diluted to a protein concentration of 3 mg/mL with PBS supplemented with protease inhibitor cocktail. Aliquots were snap frozen in liquid nitrogen and stored at −80 °C.</div></div><div id="sec4_7_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i146" class="anchor-spacer"></div><h4 class="article-section__title" id="_i146"> CETSA with K562 Cell Lysate</h4><div class="NLM_p last">The CETSA assay was adapted from previously published protocols.<a onclick="showRef(event, 'ref80 ref81'); return false;" href="javascript:void(0);" class="ref ref80 ref81">(80,81)</a> K562 lysate prepared as described above was incubated at room temperature with DMSO, 100 μM of NS1 (<b>10</b>), or 100 μM of compound <b>25</b> for 30 min. A Bio-Rad C1000 Touch thermal cycler was preheated to the following temperatures using the gradient setting: 37, 38.8, 41.6, 45.2, 50.1, 53.7, 56, 57.9, 60.8, 64.3, 69.1, and 72.9 °C. DMSO-treated and compound-treated lysate were aliquotted into 12 PCR tubes, and each tube was heated at one of the temperatures listed above for 3 min. The samples were cooled at room temperature for 3 min then snap frozen in liquid nitrogen. When thawed, the samples were centrifuged at 17000<i>g</i> for 20 min at 4 °C. The supernatant was transferred to a new tube, mixed with NuPAGE LDS sample buffer (Novex) and β-mercaptoethanol (2.5%). The samples were heated at 70 °C for 10 min and analyzed by SDS-PAGE and Western blot using an anti-NNMT mouse antibody (Abcam, ab119758, 1:1000 dilution), an antiactin HRP-conjugated mouse antibody (Abcam, ab49900, 1:250000 dilution), and an antimouse HRP-conjugated IgG (Promega, W402B, 1:5000 dilution).</div></div><div id="sec4_7_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i147" class="anchor-spacer"></div><h4 class="article-section__title" id="_i147"> Isothermal Dose Response (IDTR) CETSA with K562 Cell Lysate</h4><div class="NLM_p last">K562 lysate prepared as described above was incubated at room temperature with DMSO or various concentrations of NS1 (<b>10</b>) or compound <b>25</b> for 30 min. A Bio-Rad C1000 Touch thermal cycler was preheated to 57 °C. Samples were aliquotted into PCR tubes, and each tube was heated at 57 °C in the thermal cycler for 3 min. The samples were cooled at room temperature for 3 min and then snap frozen in liquid nitrogen. When thawed, the samples were centrifuged at 17000<i>g</i> for 20 min at 4 °C. The supernatant was prepared for Western blot analysis as described in the CETSA procedure above.</div></div><div id="sec4_7_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i148" class="anchor-spacer"></div><h4 class="article-section__title" id="_i148"> Cytotoxicity Assay in U2OS Cells</h4><div class="NLM_p last">Cytotoxicity evaluation was performed with the Promega CellTiter-Glo luminescent cell viability assay. U2OS cells were maintained in DMEM-F12 glutamax complete growth media supplemented with 10% heat-inactivated fetal bovine serum (HI FBS). CellTiter-Glo Buffer, and CellTiter-Glo Substrate were thawed at room temperature prior to use. An appropriate volume (10 mL) of CellTiter-Glo Buffer was transferred into the amber bottle containing CellTiter-Glo Substrate to reconstitute the lyophilized enzyme/substrate mixture. The contents were gently vortexed to obtain a homogeneous solution. U2OS cells were seeded at 10000 cells/150 μL/well in 96 well cell culture plate and incubated overnight at 37 °C and 5% CO<sub>2</sub>. The next day, media was changed (150 μL), various concentrations of compounds (50 μL from 4× stock) were tested in duplicates, at eight concentrations, in a semi log dilution pattern, with starting concentration of 100 μM. Final concentration of DMSO in the assay was 1%. Doxorubicin hydrochloride was used as a positive control in the assay (10, 5, and 3 μM). The plates were incubated for 24 and 48 h at 37 °C with 5% CO<sub>2</sub>. After 24 and 48 h of incubation, CellTiter-Glo reagent (Promega) preparation was added to the cell culture medium present in each well in a 1:1 ratio by volume (e.g., add 50 μL of reagent to 50 μL of medium containing cells). The plate was gently vortexed for 10 min on an orbital shaker to induce cell lysis. The contents of the plate were transferred to an opaque half area 96 well plate. The plate was incubated at RT for 10 min to stabilize the luminescence signal. Luminescence output was read on a Tecan Spark plate reader. The % cell viability was calculated relative to DMSO control.</div></div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i149" class="anchor-spacer"></div><h3 class="article-section__title" id="_i149"> MNAM Measurement in U2OS Cells by LC-MS/MS</h3><div id="sec4_8_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i150" class="anchor-spacer"></div><h4 class="article-section__title" id="_i150"> Materials</h4><div class="NLM_p last">Acetonitrile (Millipore Sigma 1000292500), D4-MNAM (BioOrganics SRK(I)-257–3226), MNAM (BioOrganics BST(I)-256-3271), DPBS (Invitrogen 14190-136), Top Seal-A Plus (PerkinElmer 6050185), 96-well plate sterile TC treated (Eppendorf 0030730119), 96 well plate transfer plate (Costar 3364), and reference compound 6-methoxy nicotinamide/(<i>alternate name</i>: JBSNF-0088) (Arbor Chemical Corporation Limited, P180810). U2OS cells were obtained from ATCC and maintained in DMEM-F12 growth media containing 10% HI FBS and 1% Pen-Strep (filter sterilized) in 37 °C incubator with 5% CO<sub>2</sub>. Extraction buffer was prepared by adding internal standard (D4-MNAM, 20 ng/mL) to 100% acetonitrile.</div></div><div id="sec4_8_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i151" class="anchor-spacer"></div><h4 class="article-section__title" id="_i151"> Protocol</h4><div class="NLM_p last">Cells were maintained in cell culture flasks until seeding into microwell plates. Media was removed, and cells were washed with DPBS (−/−) and then detached with TrypLE Express (4 mL) into a 150 cm<sup>2</sup> flask (incubate at 37 °C for 5 min). The reaction was stopped by the addition of 5 mL of media, and cells were counted with a Vi-CELL cell counter. Cells were seeded into a 96 well microplate (100 μL/well = 10000 cells/well) and incubated for 24 h at 37 °C, 5% CO<sub>2</sub>, and 95% humidity, with the lid on. Media was replaced by addition of 100 μL of media/compound mixture and cells were incubated for 24 h at 37 °C, 5% CO<sub>2</sub>, and 95% humidity with the lid on. The media/compound mixture was removed, and cells were washed two times with 150 μL of DPBS(−/−). The DPBS was removed, and 100 μL of extraction buffer was added. The plate was incubated for 20 min at RT with mild shaking. Theb 100 μL autoclaved water was added and the plate was mixed gently. The plate was centrifuged at 5000<i>g</i> for 10 min, and then 150 μL of the extraction buffer/water mixture (final D4MNAM concentration of 10 ng/mL) was transferred into a 96 well microplate. MNAM levels were quantified by LC-MS/MS according to methods previously reported by Kannt et al.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a></div></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i152"><a href="/doi/suppl/10.1021/acs.jmedchem.9b01238" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_73314" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_73314" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.9b01238" class="ext-link">10.1021/acs.jmedchem.9b01238</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Supplementary figures and tables, synthetic schemes, and experimental protocols not listed in the main manuscript, including: molecular docking, protein expression and purification, biochemical assays, bioinformatic analyses, and protein crystallography (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01238/suppl_file/jm9b01238_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">NMR spectra (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01238/suppl_file/jm9b01238_si_002.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Enzyme kinetics (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01238/suppl_file/jm9b01238_si_003.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings for compounds <b>10</b>–<b>41</b>  (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01238/suppl_file/jm9b01238_si_004.csv" class="ext-link">CSV</a>)</p></li><li><p class="inline">NS1 <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ROD">3ROD</a> docking (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01238/suppl_file/jm9b01238_si_005.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">Small-molecule X-ray crystallography data for NS1 (<b>10</b>) (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01238/suppl_file/jm9b01238_si_006.cif" class="ext-link">CIF</a>)</p></li><li><p class="inline">Small-molecule X-ray crystallography data for <b>S30</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01238/suppl_file/jm9b01238_si_007.cif" class="ext-link">CIF</a>)</p></li><li><p class="inline">Small-molecule X-ray crystallography data for <b>S53</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01238/suppl_file/jm9b01238_si_008.cif" class="ext-link">CIF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01238/suppl_file/jm9b01238_si_001.pdf">jm9b01238_si_001.pdf (14.15 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01238/suppl_file/jm9b01238_si_002.pdf">jm9b01238_si_002.pdf (34.35 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01238/suppl_file/jm9b01238_si_003.pdf">jm9b01238_si_003.pdf (587.85 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01238/suppl_file/jm9b01238_si_004.csv">jm9b01238_si_004.csv (4.02 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01238/suppl_file/jm9b01238_si_005.pdb">jm9b01238_si_005.pdb (208.92 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01238/suppl_file/jm9b01238_si_006.cif">jm9b01238_si_006.cif (556.52 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01238/suppl_file/jm9b01238_si_007.cif">jm9b01238_si_007.cif (808.9 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01238/suppl_file/jm9b01238_si_008.cif">jm9b01238_si_008.cif (954.93 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The coordinates and structure factors of the hNNMT-NS1 (<b>10</b>) cocrystal structure have been deposited with the RCSB Protein Data Bank under accession code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ORR">6ORR</a>. The authors will release the atomic coordinates upon article publication.</p><div class="testing" data-doi="10.1021/acs.jmedchem.9b01238" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_11100" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_11100" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Matthew D. Shair</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry and Chemical Biology and , Harvard University, Cambridge, Massachusetts 02138, United States</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#53203b323a2113303b363e3a2027212a7d3b3221253221377d363726"><span class="__cf_email__" data-cfemail="45362d242c3705262d20282c3631373c6b2d2437332437216b202130">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Rocco
L. Policarpo</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry and Chemical Biology and , Harvard University, Cambridge, Massachusetts 02138, United States</span>; 
    <span>Present Address:
                        For R.L.P.: Incyte Corporation, 1801 Augustine Cut-Off, Wilmington, Delaware 19803, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-9819-2394" title="Orcid link">http://orcid.org/0000-0001-9819-2394</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ludovic Decultot</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry and Chemical Biology and , Harvard University, Cambridge, Massachusetts 02138, United States</span>; 
    <span>Present Address:
                        For L.D.: The Broad Institute of MIT and Harvard, Center for the Development of Therapeutics, 415 Main Street, Cambridge, Massachusetts 02142, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Elizabeth May</span> - <span class="hlFld-Affiliation affiliation">Department of Molecular and Cellular
Biology and , Harvard University, Cambridge, Massachusetts 02138, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Petr Kuzmič</span> - <span class="hlFld-Affiliation affiliation">BioKin
Ltd., Watertown, Massachusetts 02472, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Samuel Carlson</span> - <span class="hlFld-Affiliation affiliation">Department of Molecular and Cellular
Biology and , Harvard University, Cambridge, Massachusetts 02138, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Danny Huang</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry and Chemical Biology and , Harvard University, Cambridge, Massachusetts 02138, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Vincent Chu</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry and Chemical Biology and , Harvard University, Cambridge, Massachusetts 02138, United States</span>; 
    <span>Present Address:
                        For V.C.: Department of Cell Biology, Harvard Medical School, 240 Longwood Avenue, Boston, Massachusetts 02115, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Brandon A. Wright</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry and Chemical Biology and , Harvard University, Cambridge, Massachusetts 02138, United States</span>; 
    <span>Present Address:
                        For B.W.: Department of Chemistry, University of California, Berkeley, California 94720, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-4149-5292" title="Orcid link">http://orcid.org/0000-0003-4149-5292</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Saravanakumar Dhakshinamoorthy</span> - <span class="hlFld-Affiliation affiliation">Jubilant
Biosys Ltd., Yeshwantpur, Bangalore, 560 022 Karnataka, India</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Aimo Kannt</span> - <span class="hlFld-Affiliation affiliation">Sanofi
Research and Development, Industriepark Hoechst, H823, D-65926 Frankfurt am Main, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Shilpa Rani</span> - <span class="hlFld-Affiliation affiliation">Jubilant
Biosys Ltd., Yeshwantpur, Bangalore, 560 022 Karnataka, India</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sreekanth Dittakavi</span> - <span class="hlFld-Affiliation affiliation">Jubilant
Biosys Ltd., Yeshwantpur, Bangalore, 560 022 Karnataka, India</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Joseph D. Panarese</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry and Chemical Biology and , Harvard University, Cambridge, Massachusetts 02138, United States</span>; 
    <span>Present Address:
                        For J.P.: Enanta Pharmaceuticals, 500 Arsenal Street, Watertown, Massachusetts 02472, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Rachelle Gaudet</span> - <span class="hlFld-Affiliation affiliation">Department of Molecular and Cellular
Biology and , Harvard University, Cambridge, Massachusetts 02138, United States</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br><p class="inlineNote">The unprocessed diffraction images have been deposited in the SBGrid<a onclick="showRef(event, 'ref82'); return false;" href="javascript:void(0);" class="ref ref82">(82)</a> Data Bank with SBGDB ID 731.<a onclick="showRef(event, 'ref83'); return false;" href="javascript:void(0);" class="ref ref83">(83)</a></p><br /></br><p class="inlineNote">All authors have given approval to the final version of the manuscript.</p></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i161">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_06816" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_06816" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We Thank Dr. Shao-Liang Zheng for assistance with small-molecule X-ray crystallography and Cheryl Arrowsmith for supplying the NNMT and INMT plasmids used in this study (via Addgene). We thank the National Science Foundation for funding Rocco L. Policarpo, as the material presented in this manuscript is based upon work supported by the NSF Graduate Research Fellowship under grant no. DGE1144152. We thank the Harvard Blavatnik Biomedical Accelerator for funding portions of this project and the Harvard College Research Program for providing funding to Danny Huang and Brandon A. Wright. We thank Grace Kenney for assistance with sequence similarity network generation, James Tucker for helpful discussion and manuscript feedback, and Kevin Dalton for assistance with crystallographic data processing and refinement. We thank Liz Hedstrom (Brandeis University) for helpful comments on the manuscript and Paul Thompson (UMass Medical School) for providing raw kinetic data from a previously published report. We gratefully acknowledge the beamline staff of NE-CAT at the Advanced Photon Source (Argonne, IL, USA) for help with data collection. NE-CAT is funded by NIH (P41 GM103403 and S10 RR029205) and the Advanced Photon Source by the U.S. Department of Energy (DE-AC02-06CH11357).</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">NNMT</td><td class="NLM_def"><p class="first last">nicotinamide <i>N</i>-methyltransferase</p></td></tr><tr><td class="NLM_term">SAM</td><td class="NLM_def"><p class="first last"><i>S</i>-adenosylmethionine</p></td></tr><tr><td class="NLM_term">NAM</td><td class="NLM_def"><p class="first last">nicotinamide</p></td></tr><tr><td class="NLM_term">MNAM</td><td class="NLM_def"><p class="first last">1-methylnicotinamide</p></td></tr><tr><td class="NLM_term">SAH</td><td class="NLM_def"><p class="first last"><i>S</i>-adenosylhomocysteine</p></td></tr><tr><td class="NLM_term">COMT</td><td class="NLM_def"><p class="first last">catechol <i>O</i>-methyltransferase</p></td></tr><tr><td class="NLM_term">BPE</td><td class="NLM_def"><p class="first last">1,2-bis(phospholano)ethane</p></td></tr><tr><td class="NLM_term">INMT</td><td class="NLM_def"><p class="first last">indolethylamine <i>N</i>-methyltransferase</p></td></tr><tr><td class="NLM_term">PNMT</td><td class="NLM_def"><p class="first last">phenylethanolamine <i>N</i>-methyltransferase</p></td></tr><tr><td class="NLM_term">TPMT</td><td class="NLM_def"><p class="first last">thiopurine <i>S</i>-methyltransferase</p></td></tr><tr><td class="NLM_term">IStd</td><td class="NLM_def"><p class="first last">internal standard</p></td></tr><tr><td class="NLM_term">rbf</td><td class="NLM_def"><p class="first last">round-bottom flask</p></td></tr><tr><td class="NLM_term">PMHS</td><td class="NLM_def"><p class="first last">polymethylhydrosiloxane.</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i163">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_87301" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_87301" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 83 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="note"><p class="first last">Currently, there are many abbreviations used for nicotinamide and 1-methylnicotinamide in the literature, leading to inconsistency and confusion. We use the standardized nomenclature proposed by Pissios in a 2017 review article on NNMT (see following citation).</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pissios, P.</span></span> <span> </span><span class="NLM_article-title">Nicotinamide N-methyltransferase: more than a vitamin B3 clearance enzyme</span>. <i>Trends Endocrinol. Metab.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>28</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">340</span>– <span class="NLM_lpage">353</span>, <span class="refDoi"> DOI: 10.1016/j.tem.2017.02.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.1016%2Fj.tem.2017.02.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=28291578" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BC2sXivFehsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2017&pages=340-353&issue=5&author=P.+Pissios&title=Nicotinamide+N-methyltransferase%3A+more+than+a+vitamin+B3+clearance+enzyme&doi=10.1016%2Fj.tem.2017.02.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Nicotinamide N-Methyltransferase: More Than a Vitamin B3 Clearance Enzyme</span></div><div class="casAuthors">Pissios, Pavlos</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Endocrinology and Metabolism</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">340-353</span>CODEN:
                <span class="NLM_cas:coden">TENME4</span>;
        ISSN:<span class="NLM_cas:issn">1043-2760</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Nicotinamide (NAM) N-methyltransferase (NNMT) was originally identified as the enzyme responsible for the methylation of NAM, one of the forms of vitamin B3.  Methylated NAM is eventually excreted from the body.  Recent evidence has expanded the role of NNMT beyond clearance of excess vitamin B3.  NNMT has been implicated in the regulation of multiple metabolic pathways in tissues such as adipose and liver as well as cancer cells through the consumption of Me donors and generation of active metabolites.  This review examines recent findings regarding the function of NNMT in physiol. and disease and highlights potential new avenues for therapeutic intervention.  Finally, key gaps in our knowledge about this enzymic system and future areas of investigation are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHRx9vhXz5LbVg90H21EOLACvtfcHk0lhRzx9INBf_Uw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXivFehsrk%253D&md5=cf088bcc9434e006062d72de0c47c62a</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.tem.2017.02.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tem.2017.02.004%26sid%3Dliteratum%253Aachs%26aulast%3DPissios%26aufirst%3DP.%26atitle%3DNicotinamide%2520N-methyltransferase%253A%2520more%2520than%2520a%2520vitamin%2520B3%2520clearance%2520enzyme%26jtitle%3DTrends%2520Endocrinol.%2520Metab.%26date%3D2017%26volume%3D28%26issue%3D5%26spage%3D340%26epage%3D353%26doi%3D10.1016%2Fj.tem.2017.02.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hong, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhai, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pissios, P.</span></span> <span> </span><span class="NLM_article-title">Nicotinamide N-methyltransferase interacts with enzymes of the methionine cycle and regulates methyl donor metabolism</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>57</i></span> (<span class="NLM_issue">40</span>),  <span class="NLM_fpage">5775</span>– <span class="NLM_lpage">5779</span>, <span class="refDoi"> DOI: 10.1021/acs.biochem.8b00561</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.biochem.8b00561" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslaqtr7O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2018&pages=5775-5779&issue=40&author=S.+Hongauthor=B.+Zhaiauthor=P.+Pissios&title=Nicotinamide+N-methyltransferase+interacts+with+enzymes+of+the+methionine+cycle+and+regulates+methyl+donor+metabolism&doi=10.1021%2Facs.biochem.8b00561"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Nicotinamide N-methyltransferase interacts with enzymes of the methionine cycle and regulates methyl donor metabolism</span></div><div class="casAuthors">Hong, Shangyu; Zhai, Bo; Pissios, Pavlos</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">40</span>),
    <span class="NLM_cas:pages">5775-5779</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Me donor balance is crit. for epigenetic regulation in cells and is maintained by the so-called methionine cycle proteins that regenerate S-adenosylmethionine (SAM), the universal Me donor, from homocysteine formed by the activity of methyltransferases.  Nnmt is a liver enzyme that methylates nicotinamide, but its role in regulating Me donor balance in the liver is unclear.  In this study, we assessed the effect of altered Nnmt expression on various aspects of Me donor metab. in the liver.  We found that Nnmt overexpression decreased SAM levels and the SAM/ S-adenosylhomocysteine (SAH) ratio both in vivo and in vitro.  Nnmt knockdown did not change Me donor balance in mouse primary hepatocytes but increased SAM levels and the SAM/SAH ratio when Gnmt, the dominantly expressed methyltransferase in liver, was simultaneously knocked down.  Paradoxically, expression of enzymically deficient Nnmt increased the SAM/SAH ratio, suggesting that Nnmt can regulate Me donor balance independent of its methyltransferase activity.  Proteomics anal. of Nnmt-interacting proteins in the liver identified Bhmt, Mat1a, and Ahcy, all components of the methionine cycle, and functional expts. showed that mutant Nnmt increased the level of remethylation of homocysteine to SAM.  In summary, we show that the function of Nnmt in hepatic Me donor balance is multifactorial.  On one hand, Nnmt decreases Me donor balance, consistent with its activity as a methyltransferase consuming Me donors.  On the other hand, by co-opting the enzymes of the methionine cycle, Nnmt aids the recycling of homocysteine to SAM for another round of methylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqD5fiz3fTxsrVg90H21EOLACvtfcHk0lh3cTLzdwWUSg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslaqtr7O&md5=2689739ce6ec8ef59b09af7a20cb06ce</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1021%2Facs.biochem.8b00561&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.biochem.8b00561%26sid%3Dliteratum%253Aachs%26aulast%3DHong%26aufirst%3DS.%26aulast%3DZhai%26aufirst%3DB.%26aulast%3DPissios%26aufirst%3DP.%26atitle%3DNicotinamide%2520N-methyltransferase%2520interacts%2520with%2520enzymes%2520of%2520the%2520methionine%2520cycle%2520and%2520regulates%2520methyl%2520donor%2520metabolism%26jtitle%3DBiochemistry%26date%3D2018%26volume%3D57%26issue%3D40%26spage%3D5775%26epage%3D5779%26doi%3D10.1021%2Facs.biochem.8b00561" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hong, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moreno-Navarrete, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kikukawa, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tzameli, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prasad, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asara, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandez-Real, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maratos-Flier, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pissios, P.</span></span> <span> </span><span class="NLM_article-title">Nicotinamide N-methyltransferase regulates hepatic nutrient metabolism through Sirt1 protein stabilization</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>21</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">887</span>– <span class="NLM_lpage">894</span>, <span class="refDoi"> DOI: 10.1038/nm.3882</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.1038%2Fnm.3882" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=26168293" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1WltLjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2015&pages=887-894&issue=8&author=S.+Hongauthor=J.+M.+Moreno-Navarreteauthor=X.+Weiauthor=Y.+Kikukawaauthor=I.+Tzameliauthor=D.+Prasadauthor=Y.+Leeauthor=J.+M.+Asaraauthor=J.+M.+Fernandez-Realauthor=E.+Maratos-Flierauthor=P.+Pissios&title=Nicotinamide+N-methyltransferase+regulates+hepatic+nutrient+metabolism+through+Sirt1+protein+stabilization&doi=10.1038%2Fnm.3882"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Nicotinamide N-methyltransferase regulates hepatic nutrient metabolism through Sirt1 protein stabilization</span></div><div class="casAuthors">Hong, Shangyu; Moreno-Navarrete, Jose M.; Wei, Xiaojing; Kikukawa, Yusuke; Tzameli, Iphigenia; Prasad, Deepthi; Lee, Yoonjin; Asara, John M.; Fernandez-Real, Jose Manuel; Maratos-Flier, Eleftheria; Pissios, Pavlos</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">887-894</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Nicotinamide N-methyltransferase (Nnmt) methylates nicotinamide, a form of vitamin B3, to produce N1-methylnicotinamide (MNAM).  Nnmt has emerged as a metabolic regulator in adipocytes, but its role in the liver, the tissue with the strongest Nnmt expression, is not known.  In spite of its overall high expression, here we find that hepatic expression of Nnmt is highly variable and correlates with multiple metabolic parameters in mice and humans.  Further, we find that suppression of hepatic Nnmt expression in vivo alters glucose and cholesterol metab. and that the metabolic effects of Nnmt in the liver are mediated by its product MNAM.  Supplementation of high-fat diet with MNAM decreases serum and liver cholesterol and liver triglycerides levels in mice.  Mechanistically, increasing Nnmt expression or MNAM levels stabilizes sirtuin 1 protein, an effect that is required for their metabolic benefits.  In summary, we describe here a novel regulatory pathway for vitamin B3 that could provide a new opportunity for metabolic disease therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpaiKWlS3XZlrVg90H21EOLACvtfcHk0lh3cTLzdwWUSg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1WltLjO&md5=d3f54ad774f4ea0bf07ce9bb389c0139</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fnm.3882&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.3882%26sid%3Dliteratum%253Aachs%26aulast%3DHong%26aufirst%3DS.%26aulast%3DMoreno-Navarrete%26aufirst%3DJ.%2BM.%26aulast%3DWei%26aufirst%3DX.%26aulast%3DKikukawa%26aufirst%3DY.%26aulast%3DTzameli%26aufirst%3DI.%26aulast%3DPrasad%26aufirst%3DD.%26aulast%3DLee%26aufirst%3DY.%26aulast%3DAsara%26aufirst%3DJ.%2BM.%26aulast%3DFernandez-Real%26aufirst%3DJ.%2BM.%26aulast%3DMaratos-Flier%26aufirst%3DE.%26aulast%3DPissios%26aufirst%3DP.%26atitle%3DNicotinamide%2520N-methyltransferase%2520regulates%2520hepatic%2520nutrient%2520metabolism%2520through%2520Sirt1%2520protein%2520stabilization%26jtitle%3DNat.%2520Med.%26date%3D2015%26volume%3D21%26issue%3D8%26spage%3D887%26epage%3D894%26doi%3D10.1038%2Fnm.3882" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ulanovskaya, O. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zuhl, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B. F.</span></span> <span> </span><span class="NLM_article-title">NNMT promotes epigenetic remodeling in cancer by creating a metabolic methylation sink</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>9</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">300</span>– <span class="NLM_lpage">306</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1204</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.1038%2Fnchembio.1204" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=23455543" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjtlKnsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=300-306&issue=5&author=O.+A.+Ulanovskayaauthor=A.+M.+Zuhlauthor=B.+F.+Cravatt&title=NNMT+promotes+epigenetic+remodeling+in+cancer+by+creating+a+metabolic+methylation+sink&doi=10.1038%2Fnchembio.1204"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">NNMT promotes epigenetic remodeling in cancer by creating a metabolic methylation sink</span></div><div class="casAuthors">Ulanovskaya, Olesya A.; Zuhl, Andrea M.; Cravatt, Benjamin F.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">300-306</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Nicotinamide N-methyltransferase (NNMT) is overexpressed in a variety of human cancers, where it contributes to tumorigenesis by a mechanism that is still poorly understood.  Here we show using metabolomics that NNMT impairs the methylation potential of cancer cells by consuming Me units from S-adenosyl methionine to create the stable metabolic product 1-methylnicotinamide.  As a result, NNMT-expressing cancer cells have an altered epigenetic state that includes hypomethylated histones and other cancer-related proteins combined with heightened expression of protumorigenic gene products.  Our findings thus point to a direct mechanistic link between the deregulation of a metabolic enzyme and widespread changes in the methylation landscape of cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrsOQPNFxw5kbVg90H21EOLACvtfcHk0lh3cTLzdwWUSg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjtlKnsbs%253D&md5=8775d8b27fba0d039b0f6da7244d28a4</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1204&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1204%26sid%3Dliteratum%253Aachs%26aulast%3DUlanovskaya%26aufirst%3DO.%2BA.%26aulast%3DZuhl%26aufirst%3DA.%2BM.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26atitle%3DNNMT%2520promotes%2520epigenetic%2520remodeling%2520in%2520cancer%2520by%2520creating%2520a%2520metabolic%2520methylation%2520sink%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2013%26volume%3D9%26issue%3D5%26spage%3D300%26epage%3D306%26doi%3D10.1038%2Fnchembio.1204" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sperber, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mathieu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferreccio, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hesson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Devi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Detraux, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Battle, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Showalter, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valensisi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bielas, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ericson, N. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Margaretha, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robitaille, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Margineantu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fiehn, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hockenbery, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blau, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raftery, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Margolin, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hawkins, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moon, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ware, C. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruohola-Baker, H.</span></span> <span> </span><span class="NLM_article-title">The metabolome regulates the epigenetic landscape during naive-to-primed human embryonic stem cell transition</span>. <i>Nat. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>17</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">1523</span>– <span class="NLM_lpage">1535</span>, <span class="refDoi"> DOI: 10.1038/ncb3264</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.1038%2Fncb3264" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=26571212" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVOgtLfL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2015&pages=1523-1535&issue=12&author=H.+Sperberauthor=J.+Mathieuauthor=Y.+Wangauthor=A.+Ferreccioauthor=J.+Hessonauthor=Z.+Xuauthor=K.+A.+Fischerauthor=A.+Deviauthor=D.+Detrauxauthor=H.+Guauthor=S.+L.+Battleauthor=M.+Showalterauthor=C.+Valensisiauthor=J.+H.+Bielasauthor=N.+G.+Ericsonauthor=L.+Margarethaauthor=A.+M.+Robitailleauthor=D.+Margineantuauthor=O.+Fiehnauthor=D.+Hockenberyauthor=C.+A.+Blauauthor=D.+Rafteryauthor=A.+A.+Margolinauthor=R.+D.+Hawkinsauthor=R.+T.+Moonauthor=C.+B.+Wareauthor=H.+Ruohola-Baker&title=The+metabolome+regulates+the+epigenetic+landscape+during+naive-to-primed+human+embryonic+stem+cell+transition&doi=10.1038%2Fncb3264"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">The metabolome regulates the epigenetic landscape during naive-to-primed human embryonic stem cell transition</span></div><div class="casAuthors">Sperber, Henrik; Mathieu, Julie; Wang, Yuliang; Ferreccio, Amy; Hesson, Jennifer; Xu, Zhuojin; Fischer, Karin A.; Devi, Arikketh; Detraux, Damien; Gu, Haiwei; Battle, Stephanie L.; Showalter, Megan; Valensisi, Cristina; Bielas, Jason H.; Ericson, Nolan G.; Margaretha, Lilyana; Robitaille, Aaron M.; Margineantu, Daciana; Fiehn, Oliver; Hockenbery, David; Blau, C. Anthony; Raftery, Daniel; Margolin, Adam A.; Hawkins, R. David; Moon, Randall T.; Ware, Carol B.; Ruohola-Baker, Hannele</div><div class="citationInfo"><span class="NLM_cas:title">Nature Cell Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1523-1535</span>CODEN:
                <span class="NLM_cas:coden">NCBIFN</span>;
        ISSN:<span class="NLM_cas:issn">1465-7392</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">For nearly a century developmental biologists have recognized that cells from embryos can differ in their potential to differentiate into distinct cell types.  Recently, it has been recognized that embryonic stem cells derived from both mice and humans exhibit two stable yet epigenetically distinct states of pluripotency: naive and primed.  We now show that nicotinamide N-methyltransferase (NNMT) and the metabolic state regulate pluripotency in human embryonic stem cells (hESCs).  Specifically, in naive hESCs, NNMT and its enzymic product 1-methylnicotinamide are highly upregulated, and NNMT is required for low S-adenosyl methionine (SAM) levels and the H3K27me3 repressive state.  NNMT consumes SAM in naive cells, making it unavailable for histone methylation that represses Wnt and activates the HIF pathway in primed hESCs.  These data support the hypothesis that the metabolome regulates the epigenetic landscape of the earliest steps in human development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqaKx0pxyCNf7Vg90H21EOLACvtfcHk0lhSVcJymL_oSg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVOgtLfL&md5=2a9a212ef237b2790832ae423f47b4aa</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fncb3264&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncb3264%26sid%3Dliteratum%253Aachs%26aulast%3DSperber%26aufirst%3DH.%26aulast%3DMathieu%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DFerreccio%26aufirst%3DA.%26aulast%3DHesson%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DZ.%26aulast%3DFischer%26aufirst%3DK.%2BA.%26aulast%3DDevi%26aufirst%3DA.%26aulast%3DDetraux%26aufirst%3DD.%26aulast%3DGu%26aufirst%3DH.%26aulast%3DBattle%26aufirst%3DS.%2BL.%26aulast%3DShowalter%26aufirst%3DM.%26aulast%3DValensisi%26aufirst%3DC.%26aulast%3DBielas%26aufirst%3DJ.%2BH.%26aulast%3DEricson%26aufirst%3DN.%2BG.%26aulast%3DMargaretha%26aufirst%3DL.%26aulast%3DRobitaille%26aufirst%3DA.%2BM.%26aulast%3DMargineantu%26aufirst%3DD.%26aulast%3DFiehn%26aufirst%3DO.%26aulast%3DHockenbery%26aufirst%3DD.%26aulast%3DBlau%26aufirst%3DC.%2BA.%26aulast%3DRaftery%26aufirst%3DD.%26aulast%3DMargolin%26aufirst%3DA.%2BA.%26aulast%3DHawkins%26aufirst%3DR.%2BD.%26aulast%3DMoon%26aufirst%3DR.%2BT.%26aulast%3DWare%26aufirst%3DC.%2BB.%26aulast%3DRuohola-Baker%26aufirst%3DH.%26atitle%3DThe%2520metabolome%2520regulates%2520the%2520epigenetic%2520landscape%2520during%2520naive-to-primed%2520human%2520embryonic%2520stem%2520cell%2520transition%26jtitle%3DNat.%2520Cell%2520Biol.%26date%3D2015%26volume%3D17%26issue%3D12%26spage%3D1523%26epage%3D1535%26doi%3D10.1038%2Fncb3264" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kraus, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banks, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodgers, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pirinen, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pulinilkunnil, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gong, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sauve, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asara, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peroni, O. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monia, B. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhanot, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alhonen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Puigserver, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kahn, B. B.</span></span> <span> </span><span class="NLM_article-title">Nicotinamide N-methyltransferase knockdown protects against diet-induced obesity</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>508</i></span> (<span class="NLM_issue">7495</span>),  <span class="NLM_fpage">258</span>– <span class="NLM_lpage">262</span>, <span class="refDoi"> DOI: 10.1038/nature13198</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.1038%2Fnature13198" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=24717514" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmtlWhsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=508&publication_year=2014&pages=258-262&issue=7495&author=D.+Krausauthor=Q.+Yangauthor=D.+Kongauthor=A.+S.+Banksauthor=L.+Zhangauthor=J.+T.+Rodgersauthor=E.+Pirinenauthor=T.+C.+Pulinilkunnilauthor=F.+Gongauthor=Y.+C.+Wangauthor=Y.+Cenauthor=A.+A.+Sauveauthor=J.+M.+Asaraauthor=O.+D.+Peroniauthor=B.+P.+Moniaauthor=S.+Bhanotauthor=L.+Alhonenauthor=P.+Puigserverauthor=B.+B.+Kahn&title=Nicotinamide+N-methyltransferase+knockdown+protects+against+diet-induced+obesity&doi=10.1038%2Fnature13198"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Nicotinamide N-methyltransferase knockdown protects against diet-induced obesity</span></div><div class="casAuthors">Kraus, Daniel; Yang, Qin; Kong, Dong; Banks, Alexander S.; Zhang, Lin; Rodgers, Joseph T.; Pirinen, Eija; Pulinilkunnil, Thomas C.; Gong, Fengying; Wang, Ya-chin; Cen, Yana; Sauve, Anthony A.; Asara, John M.; Peroni, Odile D.; Monia, Brett P.; Bhanot, Sanjay; Alhonen, Leena; Puigserver, Pere; Kahn, Barbara B.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">508</span>
        (<span class="NLM_cas:issue">7495</span>),
    <span class="NLM_cas:pages">258-262</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">In obesity and type 2 diabetes, Glut4 glucose transporter expression is decreased selectively in adipocytes.  Adipose-specific knockout or overexpression of Glut4 alters systemic insulin sensitivity.  Here we show, using DNA array analyses, that nicotinamide N-methyltransferase (Nnmt) is the most strongly reciprocally regulated gene when comparing gene expression in white adipose tissue (WAT) from adipose-specific Glut4-knockout or adipose-specific Glut4-overexpressing mice with their resp. controls.  NNMT methylates nicotinamide (vitamin B3) using S-adenosylmethionine (SAM) as a Me donor.  Nicotinamide is a precursor of NAD+, an important cofactor linking cellular redox states with energy metab.  SAM provides propylamine for polyamine biosynthesis and donates a Me group for histone methylation.  Polyamine flux including synthesis, catabolism and excretion, is controlled by the rate-limiting enzymes ornithine decarboxylase (ODC) and spermidine-spermine N1-acetyltransferase (SSAT; encoded by Sat1) and by polyamine oxidase (PAO), and has a major role in energy metab.  We report that NNMT expression is increased in WAT and liver of obese and diabetic mice.  Nnmt knockdown in WAT and liver protects against diet-induced obesity by augmenting cellular energy expenditure.  NNMT inhibition increases adipose SAM and NAD+ levels and upregulates ODC and SSAT activity as well as expression, owing to the effects of NNMT on histone H3 lysine 4 methylation in adipose tissue.  Direct evidence for increased polyamine flux resulting from NNMT inhibition includes elevated urinary excretion and adipocyte secretion of diacetylspermine, a product of polyamine metab.  NNMT inhibition in adipocytes increases oxygen consumption in an ODC-, SSAT- and PAO-dependent manner.  Thus, NNMT is a novel regulator of histone methylation, polyamine flux and NAD+-dependent SIRT1 signaling, and is a unique and attractive target for treating obesity and type 2 diabetes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKuWPUvPnIqLVg90H21EOLACvtfcHk0lhSVcJymL_oSg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmtlWhsL8%253D&md5=c37b6137cd4537a0f6f1701a5a212dd6</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fnature13198&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature13198%26sid%3Dliteratum%253Aachs%26aulast%3DKraus%26aufirst%3DD.%26aulast%3DYang%26aufirst%3DQ.%26aulast%3DKong%26aufirst%3DD.%26aulast%3DBanks%26aufirst%3DA.%2BS.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DRodgers%26aufirst%3DJ.%2BT.%26aulast%3DPirinen%26aufirst%3DE.%26aulast%3DPulinilkunnil%26aufirst%3DT.%2BC.%26aulast%3DGong%26aufirst%3DF.%26aulast%3DWang%26aufirst%3DY.%2BC.%26aulast%3DCen%26aufirst%3DY.%26aulast%3DSauve%26aufirst%3DA.%2BA.%26aulast%3DAsara%26aufirst%3DJ.%2BM.%26aulast%3DPeroni%26aufirst%3DO.%2BD.%26aulast%3DMonia%26aufirst%3DB.%2BP.%26aulast%3DBhanot%26aufirst%3DS.%26aulast%3DAlhonen%26aufirst%3DL.%26aulast%3DPuigserver%26aufirst%3DP.%26aulast%3DKahn%26aufirst%3DB.%2BB.%26atitle%3DNicotinamide%2520N-methyltransferase%2520knockdown%2520protects%2520against%2520diet-induced%2520obesity%26jtitle%3DNature%26date%3D2014%26volume%3D508%26issue%3D7495%26spage%3D258%26epage%3D262%26doi%3D10.1038%2Fnature13198" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brachs, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polack, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brachs, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jahn-Hofmann, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elvert, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pfenninger, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barenz, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Margerie, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mai, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spranger, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kannt, A.</span></span> <span> </span><span class="NLM_article-title">Genetic nicotinamide N-methyltransferase (Nnmt) deficiency in male mice improves insulin sensitivity in diet-induced obesity but does not affect glucose tolerance</span>. <i>Diabetes</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>68</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">527</span>– <span class="NLM_lpage">542</span>, <span class="refDoi"> DOI: 10.2337/db18-0780</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.2337%2Fdb18-0780" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=30552109" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BC1MXptlSmtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2019&pages=527-542&issue=3&author=S.+Brachsauthor=J.+Polackauthor=M.+Brachsauthor=K.+Jahn-Hofmannauthor=R.+Elvertauthor=A.+Pfenningerauthor=F.+Barenzauthor=D.+Margerieauthor=K.+Maiauthor=J.+Sprangerauthor=A.+Kannt&title=Genetic+nicotinamide+N-methyltransferase+%28Nnmt%29+deficiency+in+male+mice+improves+insulin+sensitivity+in+diet-induced+obesity+but+does+not+affect+glucose+tolerance&doi=10.2337%2Fdb18-0780"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Genetic nicotinamide N-methyltransferase (Nnmt) deficiency in male mice improves insulin sensitivity in diet-induced obesity but does not affect glucose tolerance</span></div><div class="casAuthors">Brachs, Sebastian; Polack, James; Brachs, Maria; Jahn-Hofrnann, Kerstin; Evert, Ralf; Pfenninger, Anja; Barenz, Felix; Margerie, Daniel; Mai, Knut; Spranger, Joachim; Kannt, Aimo</div><div class="citationInfo"><span class="NLM_cas:title">Diabetes</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">527-542</span>CODEN:
                <span class="NLM_cas:coden">DIAEAZ</span>;
        ISSN:<span class="NLM_cas:issn">0012-1797</span>.
    
            (<span class="NLM_cas:orgname">American Diabetes Association</span>)
        </div><div class="casAbstract">Antisense oligonucleotide knockdown (ASO-KD) of nicotinamide N-methyltransferase (NNMT) in high-fat diet (HFD)-fed mice has been reported to reduce wt. gain and plasma insulin levels and to improve glucose toler- ance.  Using NNMT-ASO-KD or NNMT knockout mice (NNMT-/-), we tested the hypothesis that Nnmt deletion protects against diet-induced obesity and its metabolic consequences in males and females on obesity-inducing diets.  We also examd. samples from a human wt. redn. (WR) study for adipose NNMT (aNNMT) expression and plasma 1 -methylnicotinarnide (MNAM) levels.  In Western diet (WD)-fed female mice, NNMT- ASO-KD reduced body wt., fat mass, and insulin level and improved glucose tolerance.  Although NNMT-/- mice fed a std. diet had no obvious phenotype, NNMT-/- males fed an HFD showed strongly improved insulin sen- sitivity (IS), Furthermore, NNMT-/- females fed a WD showed reduced wt. gain, less fat, and lower insulin levels.  However, no improved glucose tolerance was obsd. in NNMT--/- mice.  Although NNMT expression in human fat biopsy samples increased during WR, cor- responding plasma MNAM levels significantly declined, suggesting that other mechanisms besides aNNMT ex- pression modulate circulating MNAM levels during WR.  In summary, upon NNMT deletion or knockdown in males and females fed different obesity-inducing diets, we obsd. sex- and diet-specific differences in body compn., wt., and glucose tolerance and esti- mates of IS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOvfmG9xy5crVg90H21EOLACvtfcHk0lhSVcJymL_oSg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXptlSmtr0%253D&md5=e7ceb8a20692f47b92f5f97e1edd38fc</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.2337%2Fdb18-0780&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2337%252Fdb18-0780%26sid%3Dliteratum%253Aachs%26aulast%3DBrachs%26aufirst%3DS.%26aulast%3DPolack%26aufirst%3DJ.%26aulast%3DBrachs%26aufirst%3DM.%26aulast%3DJahn-Hofmann%26aufirst%3DK.%26aulast%3DElvert%26aufirst%3DR.%26aulast%3DPfenninger%26aufirst%3DA.%26aulast%3DBarenz%26aufirst%3DF.%26aulast%3DMargerie%26aufirst%3DD.%26aulast%3DMai%26aufirst%3DK.%26aulast%3DSpranger%26aufirst%3DJ.%26aulast%3DKannt%26aufirst%3DA.%26atitle%3DGenetic%2520nicotinamide%2520N-methyltransferase%2520%2528Nnmt%2529%2520deficiency%2520in%2520male%2520mice%2520improves%2520insulin%2520sensitivity%2520in%2520diet-induced%2520obesity%2520but%2520does%2520not%2520affect%2520glucose%2520tolerance%26jtitle%3DDiabetes%26date%3D2019%26volume%3D68%26issue%3D3%26spage%3D527%26epage%3D542%26doi%3D10.2337%2Fdb18-0780" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ju, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, Z.</span></span> <span> </span><span class="NLM_article-title">Serum N(1)-methylnicotinamide is associated with obesity and diabetes in chinese</span>. <i>J. Clin. Endocrinol. Metab.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>100</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">3112</span>– <span class="NLM_lpage">3117</span>, <span class="refDoi"> DOI: 10.1210/jc.2015-1732</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.1210%2Fjc.2015-1732" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=26066674" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVGks7bO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=100&publication_year=2015&pages=3112-3117&issue=8&author=M.+Liuauthor=L.+Liauthor=J.+Chuauthor=B.+Zhuauthor=Q.+Zhangauthor=X.+Yinauthor=W.+Jiangauthor=G.+Daiauthor=W.+Juauthor=Z.+Wangauthor=Q.+Yangauthor=Z.+Fang&title=Serum+N%281%29-methylnicotinamide+is+associated+with+obesity+and+diabetes+in+chinese&doi=10.1210%2Fjc.2015-1732"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Serum N1-methylnicotinamide is associated with obesity and diabetes in Chinese</span></div><div class="casAuthors">Liu, Ming; Li, Lihua; Chu, Jihong; Zhu, Boyu; Zhang, Qingtao; Yin, Xueyan; Jiang, Weimin; Dai, Guoliang; Ju, Wenzheng; Wang, Zhenxing; Yang, Qin; Fang, Zhuyuan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Clinical Endocrinology and Metabolism</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">100</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3112-3117</span>CODEN:
                <span class="NLM_cas:coden">JCEMAZ</span>;
        ISSN:<span class="NLM_cas:issn">0021-972X</span>.
    
            (<span class="NLM_cas:orgname">Endocrine Society</span>)
        </div><div class="casAbstract">Context: Nicotinamide N-methyltransferase (NNMT) is a novel histone methylation modulator that regulates energy metab., and NNMT knockdown prevents diet-induced obesity in mice.  However, whether NNMT plays a role in human obesity and type 2 diabetes (T2DM) remains to be elucidated.  Objective: NNMT catalyzes methylation of nicotinamide to generate N1-methylnicotinamide (me-NAM).  We aimed to investigate the assocns. of serum me-NAM with obesity and T2DM in Chinese.  Design, Setting, and Participants: The study subjects (n = 1160) were recruited from Dali, a city of Yunnan Province, in southwest China.  Anthropometric phenotypes, fasting glucose, and serum lipids were measured.  Serum me-NAM was measured by liq. chromatog.-mass spectrometry.  Results: Serum me-NAM was pos. correlated with body mass index and waist circumference and neg. with high-d. lipoprotein (P ≤ .03).  The correlations remained highly significant in the multivariate adjusted correlation analyses.  In men (n = 691), pos. correlations between me-NAM and fasting glucose, low-d. lipoprotein, liver function, and serum creatinine levels were also obsd. in both simple and multivariate adjusted correlation analyses.  In multiple logistic regression analyses, elevated serum me-NAM was assocd. with higher risks for overweight/obesity (odds ratios, 2.36 and 5.78; 95% confidence intervals, 1.10-5.08 and 1.78-18.76 for men and women, resp.; P ≤ .03) and diabetes (odds ratios, 1.56 and 1.86; 95% confidence intervals, 1.10-2.22 and 1.05-3.31 for men and women, resp.; P ≤ .03).  Conclusions: This first large-scale population study shows that me-NAM, as an indicator of NNMT activity, is strongly assocd. with obesity and diabetes, supporting NNMT as a potential target for treating obesity and diabetes in humans.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKUX5ibbM74bVg90H21EOLACvtfcHk0litltKCLfeKfQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVGks7bO&md5=552cea7f01cd71291d71d0fbccf7386d</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1210%2Fjc.2015-1732&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1210%252Fjc.2015-1732%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DM.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DChu%26aufirst%3DJ.%26aulast%3DZhu%26aufirst%3DB.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DYin%26aufirst%3DX.%26aulast%3DJiang%26aufirst%3DW.%26aulast%3DDai%26aufirst%3DG.%26aulast%3DJu%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DYang%26aufirst%3DQ.%26aulast%3DFang%26aufirst%3DZ.%26atitle%3DSerum%2520N%25281%2529-methylnicotinamide%2520is%2520associated%2520with%2520obesity%2520and%2520diabetes%2520in%2520chinese%26jtitle%3DJ.%2520Clin.%2520Endocrinol.%2520Metab.%26date%3D2015%26volume%3D100%26issue%3D8%26spage%3D3112%26epage%3D3117%26doi%3D10.1210%2Fjc.2015-1732" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">You, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span> <span> </span><span class="NLM_article-title">Nicotinamide N-methyltransferase enhances the progression of prostate cancer by stabilizing sirtuin 1</span>. <i>Oncol. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>15</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">9195</span>– <span class="NLM_lpage">9201</span>, <span class="refDoi"> DOI: 10.3892/ol.2018.8474</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.3892%2Fol.2018.8474" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=29805651" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A280%3ADC%252BC1Mfns1Gqtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2018&pages=9195-9201&issue=6&author=Z.+Youauthor=Y.+Liuauthor=X.+Liu&title=Nicotinamide+N-methyltransferase+enhances+the+progression+of+prostate+cancer+by+stabilizing+sirtuin+1&doi=10.3892%2Fol.2018.8474"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Nicotinamide N-methyltransferase enhances the progression of prostate cancer by stabilizing sirtuin 1</span></div><div class="casAuthors">You Zhenyu; Liu Yang; Liu Xuefei</div><div class="citationInfo"><span class="NLM_cas:title">Oncology letters</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">9195-9201</span>
        ISSN:<span class="NLM_cas:issn">1792-1074</span>.
    </div><div class="casAbstract">A previous study demonstrated that nicotinamide N-methyltransferase (NNMT) is upregulated in the tissues of patients with prostate cancer (PCa); however, the specific underlying mechanism of this remains unclear.  To begin with, the expression of NNMT was investigated in the peripheral blood of patients with PCa and of healthy control subjects.  The results indicated that the expression level of NNMT was elevated in the peripheral blood and tissues of patients with PCa.  Furthermore, the overexpression of NNMT enhanced PC-3 cell viability, invasion and migration capacity.  Additionally, the overexpression of NNMT significantly increased the mRNA level of sirtuin 1 (SIRT1) in PC-3 cells.  In addition, nicotinamide treatment significantly suppressed the expression of SIRT1 even in PC-3 cells transfected with adeno-associated virus-NNMT.  Furthermore, the PC-3 cell invasion capacity was notably decreased by the nicotinamide treatment; however, such effects were largely abolished by the overexpression of NNMT in PC-3 cells.  These data indicated that NNMT enhanced PC-3 cell migration and invasion mainly by regulating SIRT1 expression.  In summary, the present study indicated that NNMT is an important regulator of SIRT1 expression in PC-3 cells and may be a potential therapeutic target for PCa.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS-eb_XPp1DSe-MdeMhXAHkfW6udTcc2ebtUnfFf6TaQ7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1Mfns1Gqtg%253D%253D&md5=d3f7f507b8828920608b3ec7a5a0c9ec</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.3892%2Fol.2018.8474&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3892%252Fol.2018.8474%26sid%3Dliteratum%253Aachs%26aulast%3DYou%26aufirst%3DZ.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DX.%26atitle%3DNicotinamide%2520N-methyltransferase%2520enhances%2520the%2520progression%2520of%2520prostate%2520cancer%2520by%2520stabilizing%2520sirtuin%25201%26jtitle%3DOncol.%2520Lett.%26date%3D2018%26volume%3D15%26issue%3D6%26spage%3D9195%26epage%3D9201%26doi%3D10.3892%2Fol.2018.8474" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kannt, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pfenninger, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teichert, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tonjes, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dietrich, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schon, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kloting, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bluher, M.</span></span> <span> </span><span class="NLM_article-title">Association of nicotinamide-N-methyltransferase mRNA expression in human adipose tissue and the plasma concentration of its product, 1-methylnicotinamide, with insulin resistance</span>. <i>Diabetologia</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">799</span>– <span class="NLM_lpage">808</span>, <span class="refDoi"> DOI: 10.1007/s00125-014-3490-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.1007%2Fs00125-014-3490-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=25596852" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFGqtbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=799-808&issue=4&author=A.+Kanntauthor=A.+Pfenningerauthor=L.+Teichertauthor=A.+Tonjesauthor=A.+Dietrichauthor=M.+R.+Schonauthor=N.+Klotingauthor=M.+Bluher&title=Association+of+nicotinamide-N-methyltransferase+mRNA+expression+in+human+adipose+tissue+and+the+plasma+concentration+of+its+product%2C+1-methylnicotinamide%2C+with+insulin+resistance&doi=10.1007%2Fs00125-014-3490-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Association of nicotinamide-N-methyltransferase mRNA expression in human adipose tissue and the plasma concentration of its product, 1-methylnicotinamide, with insulin resistance</span></div><div class="casAuthors">Kannt, Aimo; Pfenninger, Anja; Teichert, Lenore; Toenjes, Anke; Dietrich, Arne; Schoen, Michael R.; Kloeting, Nora; Blueher, Matthias</div><div class="citationInfo"><span class="NLM_cas:title">Diabetologia</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">799-808</span>CODEN:
                <span class="NLM_cas:coden">DBTGAJ</span>;
        ISSN:<span class="NLM_cas:issn">0012-186X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Aims/hypothesis: Nicotinamide-N-methyltransferase (NNMT) was recently shown to be upregulated in mouse models of insulin resistance and obesity.  So far, it is unknown whether NNMT is regulated in human disease.  We have explored the hypothesis that white adipose tissue (WAT) NNMT expression and plasma 1-methylnicotinamide (MNA) concn. are increased in human insulin resistance and type 2 diabetes.  Methods: NNMT expression and plasma MNA were analyzed in three groups of individuals: (1) 199 patients undergoing abdominal surgery; (2) 60 individuals on a 12-wk exercise program and (3) 55 patients on a two-step bariatric surgery program.  Results: Patients with manifest type 2 diabetes have a significantly (approx. twofold) higher NNMT expression both in omental and s.c. WAT compared with controls.  Notably, plasma MNA correlated significantly with WAT NNMT expression in patients with type 2 diabetes (women, r = 0.59, p < 0.001; men, r = 0.61, p < 0.001) but not in healthy control individuals.  In insulin-resistant individuals, there was an inverse correlation between insulin sensitivity and plasma MNA (r = 0.44, p = 0.01) or adipose tissue NNMT mRNA (r = 0.64, p < 0.001).  The latter assocn. was confirmed in a second cohort (n = 60, r = 0.78, p < 0.001).  Interventions improving insulin sensitivity-exercise and bariatric surgery-were assocd. with a significant (p < 0.001) redn. in WAT NNMT expression.  Bariatric surgery was also assocd. with a significant decrease in plasma MNA.  Conclusions/interpretation: We demonstrate that WAT NNMT expression is regulated in human insulin resistance and type 2 diabetes and that plasma MNA correlates with increased tissue NNMT expression and the degree of insulin resistance, making it a potential biomarker for loss of insulin sensitivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqd9fuoxaxMlLVg90H21EOLACvtfcHk0litltKCLfeKfQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFGqtbY%253D&md5=0ad5f86b100383e2769904f8a86482b9</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1007%2Fs00125-014-3490-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00125-014-3490-7%26sid%3Dliteratum%253Aachs%26aulast%3DKannt%26aufirst%3DA.%26aulast%3DPfenninger%26aufirst%3DA.%26aulast%3DTeichert%26aufirst%3DL.%26aulast%3DTonjes%26aufirst%3DA.%26aulast%3DDietrich%26aufirst%3DA.%26aulast%3DSchon%26aufirst%3DM.%2BR.%26aulast%3DKloting%26aufirst%3DN.%26aulast%3DBluher%26aufirst%3DM.%26atitle%3DAssociation%2520of%2520nicotinamide-N-methyltransferase%2520mRNA%2520expression%2520in%2520human%2520adipose%2520tissue%2520and%2520the%2520plasma%2520concentration%2520of%2520its%2520product%252C%25201-methylnicotinamide%252C%2520with%2520insulin%2520resistance%26jtitle%3DDiabetologia%26date%3D2015%26volume%3D58%26issue%3D4%26spage%3D799%26epage%3D808%26doi%3D10.1007%2Fs00125-014-3490-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jung, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, L. J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanvoranart, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubert, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prager, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wallace, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mack, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rich, J. N.</span></span> <span> </span><span class="NLM_article-title">Nicotinamide metabolism regulates glioblastoma stem cell maintenance</span>. <i>JCI Insight</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>2</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">90019</span>, <span class="refDoi"> DOI: 10.1172/jci.insight.90019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.1172%2Fjci.insight.90019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=28515364" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2017&pages=90019&issue=10&author=J.+Jungauthor=L.+J.+Y.+Kimauthor=X.+Wangauthor=Q.+Wuauthor=T.+Sanvoranartauthor=C.+G.+Hubertauthor=B.+C.+Pragerauthor=L.+C.+Wallaceauthor=X.+Jinauthor=S.+C.+Mackauthor=J.+N.+Rich&title=Nicotinamide+metabolism+regulates+glioblastoma+stem+cell+maintenance&doi=10.1172%2Fjci.insight.90019"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1172%2Fjci.insight.90019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1172%252Fjci.insight.90019%26sid%3Dliteratum%253Aachs%26aulast%3DJung%26aufirst%3DJ.%26aulast%3DKim%26aufirst%3DL.%2BJ.%2BY.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DWu%26aufirst%3DQ.%26aulast%3DSanvoranart%26aufirst%3DT.%26aulast%3DHubert%26aufirst%3DC.%2BG.%26aulast%3DPrager%26aufirst%3DB.%2BC.%26aulast%3DWallace%26aufirst%3DL.%2BC.%26aulast%3DJin%26aufirst%3DX.%26aulast%3DMack%26aufirst%3DS.%2BC.%26aulast%3DRich%26aufirst%3DJ.%2BN.%26atitle%3DNicotinamide%2520metabolism%2520regulates%2520glioblastoma%2520stem%2520cell%2520maintenance%26jtitle%3DJCI%2520Insight%26date%3D2017%26volume%3D2%26issue%3D10%26spage%3D90019%26doi%3D10.1172%2Fjci.insight.90019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Eckert, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coscia, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chryplewicz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hernandez, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tienda, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nahotko, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blaženović, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lastra, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curtis, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamada, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perets, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGregor, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrade, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fiehn, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moellering, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lengyel, E.</span></span> <span> </span><span class="NLM_article-title">Proteomics reveals NNMT as a master metabolic regulator of cancer-associated fibroblasts</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>569</i></span> (<span class="NLM_issue">7758</span>),  <span class="NLM_fpage">723</span>– <span class="NLM_lpage">728</span>, <span class="refDoi"> DOI: 10.1038/s41586-019-1173-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.1038%2Fs41586-019-1173-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=31043742" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BC1MXovFeis7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=569&publication_year=2019&pages=723-728&issue=7758&author=M.+A.+Eckertauthor=F.+Cosciaauthor=A.+Chryplewiczauthor=J.+W.+Changauthor=K.+M.+Hernandezauthor=S.+Panauthor=S.+M.+Tiendaauthor=D.+A.+Nahotkoauthor=G.+Liauthor=I.+Bla%C5%BEenovi%C4%87author=R.+R.+Lastraauthor=M.+Curtisauthor=S.+D.+Yamadaauthor=R.+Peretsauthor=S.+M.+McGregorauthor=J.+Andradeauthor=O.+Fiehnauthor=R.+E.+Moelleringauthor=M.+Mannauthor=E.+Lengyel&title=Proteomics+reveals+NNMT+as+a+master+metabolic+regulator+of+cancer-associated+fibroblasts&doi=10.1038%2Fs41586-019-1173-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Proteomics reveals NNMT as a master metabolic regulator of cancer-associated fibroblasts</span></div><div class="casAuthors">Eckert, Mark A.; Coscia, Fabian; Chryplewicz, Agnieszka; Chang, Jae Won; Hernandez, Kyle M.; Pan, Shawn; Tienda, Samantha M.; Nahotko, Dominik A.; Li, Gang; Blazenovic, Ivana; Lastra, Ricardo R.; Curtis, Marion; Yamada, S. Diane; Perets, Ruth; McGregor, Stephanie M.; Andrade, Jorge; Fiehn, Oliver; Moellering, Raymond E.; Mann, Matthias; Lengyel, Ernst</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">569</span>
        (<span class="NLM_cas:issue">7758</span>),
    <span class="NLM_cas:pages">723-728</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">High-grade serous carcinoma has a poor prognosis, owing primarily to its early dissemination throughout the abdominal cavity.  Genomic and proteomic approaches have provided snapshots of the proteogenomics of ovarian cancer1,2, but a systematic examn. of both the tumor and stromal compartments is crit. in understanding ovarian cancer metastasis.  Here we develop a label-free proteomic workflow to analyze as few as 5000 formalin-fixed, paraffin-embedded cells microdissected from each compartment.  The tumor proteome was stable during progression from in situ lesions to metastatic disease; however, the metastasis-assocd. stroma was characterized by a highly conserved proteomic signature, prominently including the methyltransferase nicotinamide N-methyltransferase (NNMT) and several of the proteins that it regulates.  Stromal NNMT expression was necessary and sufficient for functional aspects of the cancer-assocd. fibroblast (CAF) phenotype, including the expression of CAF markers and the secretion of cytokines and oncogenic extracellular matrix.  Stromal NNMT expression supported ovarian cancer migration, proliferation and in vivo growth and metastasis.  Expression of NNMT in CAFs led to depletion of S-adenosyl methionine and redn. in histone methylation assocd. with widespread gene expression changes in the tumor stroma.  This work supports the use of ultra-low-input proteomics to identify candidate drivers of disease phenotypes.  NNMT is a central, metabolic regulator of CAF differentiation and cancer progression in the stroma that may be therapeutically targeted.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlL_GCBb1LSbVg90H21EOLACvtfcHk0lgbUkhvvWhtQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXovFeis7s%253D&md5=43dbbcee11b5d21c5fb77f5b8ff02701</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2Fs41586-019-1173-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-019-1173-8%26sid%3Dliteratum%253Aachs%26aulast%3DEckert%26aufirst%3DM.%2BA.%26aulast%3DCoscia%26aufirst%3DF.%26aulast%3DChryplewicz%26aufirst%3DA.%26aulast%3DChang%26aufirst%3DJ.%2BW.%26aulast%3DHernandez%26aufirst%3DK.%2BM.%26aulast%3DPan%26aufirst%3DS.%26aulast%3DTienda%26aufirst%3DS.%2BM.%26aulast%3DNahotko%26aufirst%3DD.%2BA.%26aulast%3DLi%26aufirst%3DG.%26aulast%3DBla%25C5%25BEenovi%25C4%2587%26aufirst%3DI.%26aulast%3DLastra%26aufirst%3DR.%2BR.%26aulast%3DCurtis%26aufirst%3DM.%26aulast%3DYamada%26aufirst%3DS.%2BD.%26aulast%3DPerets%26aufirst%3DR.%26aulast%3DMcGregor%26aufirst%3DS.%2BM.%26aulast%3DAndrade%26aufirst%3DJ.%26aulast%3DFiehn%26aufirst%3DO.%26aulast%3DMoellering%26aufirst%3DR.%2BE.%26aulast%3DMann%26aufirst%3DM.%26aulast%3DLengyel%26aufirst%3DE.%26atitle%3DProteomics%2520reveals%2520NNMT%2520as%2520a%2520master%2520metabolic%2520regulator%2520of%2520cancer-associated%2520fibroblasts%26jtitle%3DNature%26date%3D2019%26volume%3D569%26issue%3D7758%26spage%3D723%26epage%3D728%26doi%3D10.1038%2Fs41586-019-1173-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Neelakantan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vance, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hommel, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McHardy, S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watowich, S. J.</span></span> <span> </span><span class="NLM_article-title">Structure-activity relationship for small molecule inhibitors of nicotinamide N-methyltransferase</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">5015</span>– <span class="NLM_lpage">5028</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00389</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00389" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BC2sXos1Sgtrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=5015-5028&issue=12&author=H.+Neelakantanauthor=H.+Y.+Wangauthor=V.+Vanceauthor=J.+D.+Hommelauthor=S.+F.+McHardyauthor=S.+J.+Watowich&title=Structure-activity+relationship+for+small+molecule+inhibitors+of+nicotinamide+N-methyltransferase&doi=10.1021%2Facs.jmedchem.7b00389"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Activity Relationship for Small Molecule Inhibitors of Nicotinamide N-Methyltransferase</span></div><div class="casAuthors">Neelakantan, Harshini; Wang, Hua-Yu; Vance, Virginia; Hommel, Jonathan D.; McHardy, Stanton F.; Watowich, Stanley J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">5015-5028</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Nicotinamide N-methyltransferase (NNMT) is a fundamental cytosolic biotransforming enzyme that catalyzes the N-methylation of endogenous and exogenous xenobiotics.  We have identified small mol. inhibitors of NNMT with >1000-fold range of activity and developed comprehensive structure-activity relationships (SARs) for NNMT inhibitors.  Screening of N-methylated quinolinium, isoquinolinium, pyrididium, and benzimidazolium/benzothiazolium analogs resulted in the identification of quinoliniums as a promising scaffold with very low micromolar (IC50 ∼ 1 μM) NNMT inhibition.  Computer-based docking of inhibitors to the NNMT substrate (nicotinamide)-binding site produced a robust correlation between ligand-enzyme interaction docking scores and exptl. calcd. IC50 values.  Predicted binding orientation of the quinolinium analogs revealed selective binding to the NNMT substrate-binding site residues and essential chem. features driving protein-ligand intermol. interactions and NNMT inhibition.  The development of this new series of small mol. NNMT inhibitors direct the future design of lead drug-like inhibitors to treat several metabolic and chronic disease conditions characterized by abnormal NNMT activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMxYy3poogIbVg90H21EOLACvtfcHk0lgbUkhvvWhtQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXos1Sgtrc%253D&md5=2cd7159b9e15569221849168e49e39cd</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00389&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00389%26sid%3Dliteratum%253Aachs%26aulast%3DNeelakantan%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DH.%2BY.%26aulast%3DVance%26aufirst%3DV.%26aulast%3DHommel%26aufirst%3DJ.%2BD.%26aulast%3DMcHardy%26aufirst%3DS.%2BF.%26aulast%3DWatowich%26aufirst%3DS.%2BJ.%26atitle%3DStructure-activity%2520relationship%2520for%2520small%2520molecule%2520inhibitors%2520of%2520nicotinamide%2520N-methyltransferase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26issue%3D12%26spage%3D5015%26epage%3D5028%26doi%3D10.1021%2Facs.jmedchem.7b00389" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kannt, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rajagopal, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kadnur, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suresh, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhamidipati, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swaminathan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hallur, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kristam, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elvert, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Czech, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pfenninger, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rudolph, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schreuder, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chandrasekar, D. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mane, V. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Birudukota, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaik, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zope, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burri, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anand, N. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thakur, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parveen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kandan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mullangi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gosu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruf, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhakshinamoorthy, S.</span></span> <span> </span><span class="NLM_article-title">A small molecule inhibitor of nicotinamide N-methyltransferase for the treatment of metabolic disorders</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">3660</span>, <span class="refDoi"> DOI: 10.1038/s41598-018-22081-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.1038%2Fs41598-018-22081-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=29483571" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A280%3ADC%252BC1Mrls1ahtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2018&pages=3660&author=A.+Kanntauthor=S.+Rajagopalauthor=S.+V.+Kadnurauthor=J.+Sureshauthor=R.+K.+Bhamidipatiauthor=S.+Swaminathanauthor=M.+S.+Hallurauthor=R.+Kristamauthor=R.+Elvertauthor=J.+Czechauthor=A.+Pfenningerauthor=C.+Rudolphauthor=H.+Schreuderauthor=D.+V.+Chandrasekarauthor=V.+S.+Maneauthor=S.+Birudukotaauthor=S.+Shaikauthor=B.+R.+Zopeauthor=R.+R.+Burriauthor=N.+N.+Anandauthor=M.+K.+Thakurauthor=M.+Singhauthor=R.+Parveenauthor=S.+Kandanauthor=R.+Mullangiauthor=T.+Yuraauthor=R.+Gosuauthor=S.+Rufauthor=S.+Dhakshinamoorthy&title=A+small+molecule+inhibitor+of+nicotinamide+N-methyltransferase+for+the+treatment+of+metabolic+disorders&doi=10.1038%2Fs41598-018-22081-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">A small molecule inhibitor of Nicotinamide N-methyltransferase for the treatment of metabolic disorders</span></div><div class="casAuthors">Kannt Aimo; Elvert Ralf; Czech Jorg; Pfenninger Anja; Rudolph Christine; Schreuder Herman; Ruf Sven; Kannt Aimo; Rajagopal Sridharan; Kadnur Sanjay Venkatachalapathi; Suresh Juluri; Bhamidipati Ravi Kanth; Swaminathan Srinivasan; Hallur Mahanandeesha Siddappa; Kristam Rajendra; Chandrasekar Devaraj Venkatapura; Mane Vishal Subhash; Birudukota Swarnakumari; Shaik Shama; Zope Bharat Ravindra; Burri Raghunadha Reddy; Anand Niranjan Naranapura; Thakur Manish Kumar; Singh Manvi; Parveen Reejuana; Kandan Saravanan; Mullangi Ramesh; Yura Takeshi; Gosu Ramachandraiah; Dhakshinamoorthy Saravanakumar</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">3660</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Nicotinamide N-methyltransferase (NNMT) is a cytosolic enzyme that catalyzes the transfer of a methyl group from the co-factor S-adenosyl-L-methionine (SAM) onto the substrate, nicotinamide (NA) to form 1-methyl-nicotinamide (MNA).  Higher NNMT expression and MNA concentrations have been associated with obesity and type-2 diabetes.  Here we report a small molecule analog of NA, JBSNF-000088, that inhibits NNMT activity, reduces MNA levels and drives insulin sensitization, glucose modulation and body weight reduction in animal models of metabolic disease.  In mice with high fat diet (HFD)-induced obesity, JBSNF-000088 treatment caused a reduction in body weight, improved insulin sensitivity and normalized glucose tolerance to the level of lean control mice.  These effects were not seen in NNMT knockout mice on HFD, confirming specificity of JBSNF-000088.  The compound also improved glucose handling in ob/ob and db/db mice albeit to a lesser extent and in the absence of weight loss.  Co-crystal structure analysis revealed the presence of the N-methylated product of JBSNF-000088 bound to the NNMT protein.  The N-methylated product was also detected in the plasma of mice treated with JBSNF-000088.  Hence, JBSNF-000088 may act as a slow-turnover substrate analog, driving the observed metabolic benefits.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRdHZ3bviJkhgaK-_ZudimnfW6udTcc2eZh34BCOc8z3bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1Mrls1ahtw%253D%253D&md5=3f2306da18e0ab2a41a57f08ea9384fb</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fs41598-018-22081-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-018-22081-7%26sid%3Dliteratum%253Aachs%26aulast%3DKannt%26aufirst%3DA.%26aulast%3DRajagopal%26aufirst%3DS.%26aulast%3DKadnur%26aufirst%3DS.%2BV.%26aulast%3DSuresh%26aufirst%3DJ.%26aulast%3DBhamidipati%26aufirst%3DR.%2BK.%26aulast%3DSwaminathan%26aufirst%3DS.%26aulast%3DHallur%26aufirst%3DM.%2BS.%26aulast%3DKristam%26aufirst%3DR.%26aulast%3DElvert%26aufirst%3DR.%26aulast%3DCzech%26aufirst%3DJ.%26aulast%3DPfenninger%26aufirst%3DA.%26aulast%3DRudolph%26aufirst%3DC.%26aulast%3DSchreuder%26aufirst%3DH.%26aulast%3DChandrasekar%26aufirst%3DD.%2BV.%26aulast%3DMane%26aufirst%3DV.%2BS.%26aulast%3DBirudukota%26aufirst%3DS.%26aulast%3DShaik%26aufirst%3DS.%26aulast%3DZope%26aufirst%3DB.%2BR.%26aulast%3DBurri%26aufirst%3DR.%2BR.%26aulast%3DAnand%26aufirst%3DN.%2BN.%26aulast%3DThakur%26aufirst%3DM.%2BK.%26aulast%3DSingh%26aufirst%3DM.%26aulast%3DParveen%26aufirst%3DR.%26aulast%3DKandan%26aufirst%3DS.%26aulast%3DMullangi%26aufirst%3DR.%26aulast%3DYura%26aufirst%3DT.%26aulast%3DGosu%26aufirst%3DR.%26aulast%3DRuf%26aufirst%3DS.%26aulast%3DDhakshinamoorthy%26aufirst%3DS.%26atitle%3DA%2520small%2520molecule%2520inhibitor%2520of%2520nicotinamide%2520N-methyltransferase%2520for%2520the%2520treatment%2520of%2520metabolic%2520disorders%26jtitle%3DSci.%2520Rep.%26date%3D2018%26volume%3D8%26spage%3D3660%26doi%3D10.1038%2Fs41598-018-22081-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ruf, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hallur, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anchan, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swamy, I. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murugesan, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sarkar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narasimhulu, L. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Putta, V. P. R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaik, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chandrasekar, D. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mane, V. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kadnur, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suresh, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhamidipati, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burri, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kristam, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schreuder, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Czech, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rudolph, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marker, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langer, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mullangi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gosu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kannt, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhakshinamoorthy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rajagopal, S.</span></span> <span> </span><span class="NLM_article-title">Novel nicotinamide analog as inhibitor of nicotinamide N-methyltransferase</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>28</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">922</span>– <span class="NLM_lpage">925</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2018.01.058</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.1016%2Fj.bmcl.2018.01.058" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=29433927" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisFCmsb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2018&pages=922-925&issue=5&author=S.+Rufauthor=M.+S.+Hallurauthor=N.+K.+Anchanauthor=I.+N.+Swamyauthor=K.+R.+Murugesanauthor=S.+Sarkarauthor=L.+K.+Narasimhuluauthor=V.+P.+R.+K.+Puttaauthor=S.+Shaikauthor=D.+V.+Chandrasekarauthor=V.+S.+Maneauthor=S.+V.+Kadnurauthor=J.+Sureshauthor=R.+K.+Bhamidipatiauthor=M.+Singhauthor=R.+R.+Burriauthor=R.+Kristamauthor=H.+Schreuderauthor=J.+Czechauthor=C.+Rudolphauthor=A.+Markerauthor=T.+Langerauthor=R.+Mullangiauthor=T.+Yuraauthor=R.+Gosuauthor=A.+Kanntauthor=S.+Dhakshinamoorthyauthor=S.+Rajagopal&title=Novel+nicotinamide+analog+as+inhibitor+of+nicotinamide+N-methyltransferase&doi=10.1016%2Fj.bmcl.2018.01.058"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Novel nicotinamide analog as inhibitor of nicotinamide N-methyltransferase</span></div><div class="casAuthors">Ruf, Sven; Hallur, Mahanandeesha Siddappa; Anchan, Nisha K.; Swamy, Indu N.; Murugesan, Karthikai Raj; Sarkar, Sayantani; Narasimhulu, Lokesh Kananti; Putta, V. P. Rama Kishore; Shaik, Shama; Chandrasekar, Devaraj Venkatapura; Mane, Vishal Subhash; Kadnur, Sanjay Venkatachalapathi; Suresh, Juluri; Bhamidipati, Ravi Kanth; Singh, Manvi; Burri, Raghunadha Reddy; Kristam, Rajendra; Schreuder, Herman; Czech, Joerg; Rudolph, Christine; Marker, Alexander; Langer, Thomas; Mullangi, Ramesh; Yura, Takeshi; Gosu, Ramachandraiah; Kannt, Aimo; Dhakshinamoorthy, Saravanakumar; Rajagopal, Sridharan</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">922-925</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Nicotinamide N-methyltransferase (NNMT) has been linked to obesity and diabetes.  We have identified a novel nicotinamide (NA) analog, compd. I that inhibited NNMT enzymic activity and reduced the formation of 1-methyl-nicotinamide (MNA), the primary metabolite of NA by ∼80% at 2 h when dosed in mice orally at 50 mg/kg.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxeiU30o2tybVg90H21EOLACvtfcHk0ljD0cu6orKqnQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisFCmsb0%253D&md5=06345ee6c292c482711d62639fac9a16</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2018.01.058&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2018.01.058%26sid%3Dliteratum%253Aachs%26aulast%3DRuf%26aufirst%3DS.%26aulast%3DHallur%26aufirst%3DM.%2BS.%26aulast%3DAnchan%26aufirst%3DN.%2BK.%26aulast%3DSwamy%26aufirst%3DI.%2BN.%26aulast%3DMurugesan%26aufirst%3DK.%2BR.%26aulast%3DSarkar%26aufirst%3DS.%26aulast%3DNarasimhulu%26aufirst%3DL.%2BK.%26aulast%3DPutta%26aufirst%3DV.%2BP.%2BR.%2BK.%26aulast%3DShaik%26aufirst%3DS.%26aulast%3DChandrasekar%26aufirst%3DD.%2BV.%26aulast%3DMane%26aufirst%3DV.%2BS.%26aulast%3DKadnur%26aufirst%3DS.%2BV.%26aulast%3DSuresh%26aufirst%3DJ.%26aulast%3DBhamidipati%26aufirst%3DR.%2BK.%26aulast%3DSingh%26aufirst%3DM.%26aulast%3DBurri%26aufirst%3DR.%2BR.%26aulast%3DKristam%26aufirst%3DR.%26aulast%3DSchreuder%26aufirst%3DH.%26aulast%3DCzech%26aufirst%3DJ.%26aulast%3DRudolph%26aufirst%3DC.%26aulast%3DMarker%26aufirst%3DA.%26aulast%3DLanger%26aufirst%3DT.%26aulast%3DMullangi%26aufirst%3DR.%26aulast%3DYura%26aufirst%3DT.%26aulast%3DGosu%26aufirst%3DR.%26aulast%3DKannt%26aufirst%3DA.%26aulast%3DDhakshinamoorthy%26aufirst%3DS.%26aulast%3DRajagopal%26aufirst%3DS.%26atitle%3DNovel%2520nicotinamide%2520analog%2520as%2520inhibitor%2520of%2520nicotinamide%2520N-methyltransferase%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2018%26volume%3D28%26issue%3D5%26spage%3D922%26epage%3D925%26doi%3D10.1016%2Fj.bmcl.2018.01.058" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Horning, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suciu, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghadiri, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ulanovskaya, O. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matthews, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lum, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Backus, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B. F.</span></span> <span> </span><span class="NLM_article-title">Chemical proteomic profiling of human methyltransferases</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>138</i></span> (<span class="NLM_issue">40</span>),  <span class="NLM_fpage">13335</span>– <span class="NLM_lpage">13343</span>, <span class="refDoi"> DOI: 10.1021/jacs.6b07830</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.6b07830" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFymu7bP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=138&publication_year=2016&pages=13335-13343&issue=40&author=B.+D.+Horningauthor=R.+M.+Suciuauthor=D.+A.+Ghadiriauthor=O.+A.+Ulanovskayaauthor=M.+L.+Matthewsauthor=K.+M.+Lumauthor=K.+M.+Backusauthor=S.+J.+Brownauthor=H.+Rosenauthor=B.+F.+Cravatt&title=Chemical+proteomic+profiling+of+human+methyltransferases&doi=10.1021%2Fjacs.6b07830"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical Proteomic Profiling of Human Methyltransferases</span></div><div class="casAuthors">Horning, Benjamin D.; Suciu, Radu M.; Ghadiri, Darian A.; Ulanovskaya, Olesya A.; Matthews, Megan L.; Lum, Kenneth M.; Backus, Keriann M.; Brown, Steven J.; Rosen, Hugh; Cravatt, Benjamin F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">138</span>
        (<span class="NLM_cas:issue">40</span>),
    <span class="NLM_cas:pages">13335-13343</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Methylation is a fundamental mechanism used in Nature to modify the structure and function of biomols., including proteins, DNA, RNA, and metabolites.  Me groups are predominantly installed into biomols. by a large and diverse class of S-adenosyl methionine (SAM)-dependent methyltransferases (MTs), of which there are ∼200 known or putative members in the human proteome.  Deregulated MT activity contributes to numerous diseases, including cancer, and several MT inhibitors are in clin. development.  Nonetheless, a large fraction of the human MT family remains poorly characterized, underscoring the need for new technologies to characterize MTs and their inhibitors in native biol. systems.  Here, the authors describe a suite of S-adenosyl homocysteine (SAH) photoreactive probes and their application in chem. proteomic expts. to profile and enrich a large no. of MTs (>50) from human cancer cell lysates with remarkable specificity over other classes of proteins.  The authors further demonstrate that the SAH probes can enrich MT-assocd. proteins and be used to screen for and assess the selectivity of MT inhibitors, leading to the discovery of a covalent inhibitor of nicotinamide N-methyltransferase (NNMT), an enzyme implicated in cancer and metabolic disorders.  The chem. proteomics probes and methods for their utilization reported herein should prove of value for the functional characterization of MTs, MT complexes, and MT inhibitors in mammalian biol. and disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplJsKzvFF-e7Vg90H21EOLACvtfcHk0lilwwUFkb3BEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFymu7bP&md5=0e07845e59e33381f3149beae7500e18</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fjacs.6b07830&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.6b07830%26sid%3Dliteratum%253Aachs%26aulast%3DHorning%26aufirst%3DB.%2BD.%26aulast%3DSuciu%26aufirst%3DR.%2BM.%26aulast%3DGhadiri%26aufirst%3DD.%2BA.%26aulast%3DUlanovskaya%26aufirst%3DO.%2BA.%26aulast%3DMatthews%26aufirst%3DM.%2BL.%26aulast%3DLum%26aufirst%3DK.%2BM.%26aulast%3DBackus%26aufirst%3DK.%2BM.%26aulast%3DBrown%26aufirst%3DS.%2BJ.%26aulast%3DRosen%26aufirst%3DH.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26atitle%3DChemical%2520proteomic%2520profiling%2520of%2520human%2520methyltransferases%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2016%26volume%3D138%26issue%3D40%26spage%3D13335%26epage%3D13343%26doi%3D10.1021%2Fjacs.6b07830" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, H. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suciu, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horning, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vinogradova, E. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ulanovskaya, O. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B. F.</span></span> <span> </span><span class="NLM_article-title">Covalent inhibitors of nicotinamide N-methyltransferase (NNMT) provide evidence for target engagement challenges in situ</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>28</i></span> (<span class="NLM_issue">16</span>),  <span class="NLM_fpage">2682</span>– <span class="NLM_lpage">2687</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2018.04.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.1016%2Fj.bmcl.2018.04.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=29731364" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BC1cXptV2qtbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2018&pages=2682-2687&issue=16&author=H.+Y.+Leeauthor=R.+M.+Suciuauthor=B.+D.+Horningauthor=E.+V.+Vinogradovaauthor=O.+A.+Ulanovskayaauthor=B.+F.+Cravatt&title=Covalent+inhibitors+of+nicotinamide+N-methyltransferase+%28NNMT%29+provide+evidence+for+target+engagement+challenges+in+situ&doi=10.1016%2Fj.bmcl.2018.04.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Covalent inhibitors of nicotinamide N-methyltransferase (NNMT) provide evidence for target engagement challenges in situ</span></div><div class="casAuthors">Lee, Hsin-Yu; Suciu, Radu M.; Horning, Benjamin D.; Vinogradova, Ekaterina V.; Ulanovskaya, Olesya A.; Cravatt, Benjamin F.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">2682-2687</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Nicotinamide N-methyltransferase (NNMT) catalyzes the N-methylation of nicotinamide using S-adenosyl-L-methionine (SAM) as a Me donor and, through doing so, can modulate cellular methylation potential to impact diverse epigenetic processes.  NNMT has been implicated in a range of diseases, including cancer and metabolic disorders.  Potent, selective, and cell-active inhibitors would constitute valuable probes to study the biol. functions and therapeutic potential of NNMT.  The authors previously reported the discovery of electrophilic small mols. that inhibit NNMT by reacting with an active-site cysteine residue in the SAM-binding pocket.  Here, the authors have used activity-based protein profiling (ABPP)-guided medicinal chem. to optimize the potency and selectivity of NNMT inhibitors, culminating in the discovery of multiple alpha-chloroacetamide (αCA) compds. with sub-μM IC50 values in vitro and excellent proteomic selectivity in cell lysates.  However, these compds. showed much weaker inhibition of NNMT in cells, a feature that was not shared by off-targets of the αCAs.  The authors' results show the potential for developing potent and selective covalent inhibitors of NNMT, but also highlight challenges that may be faced in targeting this enzyme in cellular systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoFjmALOSTRFbVg90H21EOLACvtfcHk0lilwwUFkb3BEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXptV2qtbY%253D&md5=9194d6ac5d1fd5a70683fb4af9cae74f</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2018.04.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2018.04.017%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DH.%2BY.%26aulast%3DSuciu%26aufirst%3DR.%2BM.%26aulast%3DHorning%26aufirst%3DB.%2BD.%26aulast%3DVinogradova%26aufirst%3DE.%2BV.%26aulast%3DUlanovskaya%26aufirst%3DO.%2BA.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26atitle%3DCovalent%2520inhibitors%2520of%2520nicotinamide%2520N-methyltransferase%2520%2528NNMT%2529%2520provide%2520evidence%2520for%2520target%2520engagement%2520challenges%2520in%2520situ%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2018%26volume%3D28%26issue%3D16%26spage%3D2682%26epage%3D2687%26doi%3D10.1016%2Fj.bmcl.2018.04.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">van
Haren, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taig, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuppens, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sastre Toraño, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moret, E. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parsons, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sartini, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emanuelli, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, N. I.</span></span> <span> </span><span class="NLM_article-title">Inhibitors of nicotinamide N-methyltransferase designed to mimic the methylation reaction transition state</span>. <i>Org. Biomol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>15</i></span> (<span class="NLM_issue">31</span>),  <span class="NLM_fpage">6656</span>– <span class="NLM_lpage">6667</span>, <span class="refDoi"> DOI: 10.1039/C7OB01357D</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.1039%2FC7OB01357D" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=28758655" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1aktLfI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2017&pages=6656-6667&issue=31&author=M.+J.+van%0AHarenauthor=R.+Taigauthor=J.+Kuppensauthor=J.+Sastre+Tora%C3%B1oauthor=E.+E.+Moretauthor=R.+B.+Parsonsauthor=D.+Sartiniauthor=M.+Emanuelliauthor=N.+I.+Martin&title=Inhibitors+of+nicotinamide+N-methyltransferase+designed+to+mimic+the+methylation+reaction+transition+state&doi=10.1039%2FC7OB01357D"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibitors of nicotinamide N-methyltransferase designed to mimic the methylation reaction transition state</span></div><div class="casAuthors">van Haren, Matthijs J.; Taig, Rebecca; Kuppens, Jilles; Sastre Torano, Javier; Moret, Ed E.; Parsons, Richard B.; Sartini, Davide; Emanuelli, Monica; Martin, Nathaniel I.</div><div class="citationInfo"><span class="NLM_cas:title">Organic & Biomolecular Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">31</span>),
    <span class="NLM_cas:pages">6656-6667</span>CODEN:
                <span class="NLM_cas:coden">OBCRAK</span>;
        ISSN:<span class="NLM_cas:issn">1477-0520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Nicotinamide N-methyltransferase (NNMT) is an enzyme that catalyzes the methylation of nicotinamide to form N'-methylnicotinamide.  Both NNMT and its methylated product have recently been linked to a variety of diseases, suggesting a role for the enzyme as a therapeutic target beyond its previously ascribed metabolic function in detoxification.  We here describe the systematic development of NNMT inhibitors derived from the structures of the substrates involved in the methylation reaction.  By covalently linking fragments of the NNMT substrates a diverse library of bisubstrate-like compds. was prepd.  The ability of these compds. to inhibit NNMT was evaluated providing valuable insights into the structural tolerances of the enzyme active site.  These studies led to the identification of new NNMT inhibitors that mimic the transition state of the methylation reaction and inhibit the enzyme with activity on par with established methyltransferase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxwrd9fP8Bf7Vg90H21EOLACvtfcHk0lilwwUFkb3BEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1aktLfI&md5=74280c50135611d361ab81b8fc55fa89</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1039%2FC7OB01357D&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC7OB01357D%26sid%3Dliteratum%253Aachs%26aulast%3Dvan%2BHaren%26aufirst%3DM.%2BJ.%26aulast%3DTaig%26aufirst%3DR.%26aulast%3DKuppens%26aufirst%3DJ.%26aulast%3DSastre%2BTora%25C3%25B1o%26aufirst%3DJ.%26aulast%3DMoret%26aufirst%3DE.%2BE.%26aulast%3DParsons%26aufirst%3DR.%2BB.%26aulast%3DSartini%26aufirst%3DD.%26aulast%3DEmanuelli%26aufirst%3DM.%26aulast%3DMartin%26aufirst%3DN.%2BI.%26atitle%3DInhibitors%2520of%2520nicotinamide%2520N-methyltransferase%2520designed%2520to%2520mimic%2520the%2520methylation%2520reaction%2520transition%2520state%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2017%26volume%3D15%26issue%3D31%26spage%3D6656%26epage%3D6667%26doi%3D10.1039%2FC7OB01357D" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Babault, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allali-Hassani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yue, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ju, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Discovery of bisubstrate inhibitors of nicotinamide N-methyltransferase (NNMT)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">1541</span>– <span class="NLM_lpage">1551</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01422</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01422" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BC1cXotVOruw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=1541-1551&issue=4&author=N.+Babaultauthor=A.+Allali-Hassaniauthor=F.+Liauthor=J.+Fanauthor=A.+Yueauthor=K.+Juauthor=F.+Liuauthor=M.+Vedadiauthor=J.+Liuauthor=J.+Jin&title=Discovery+of+bisubstrate+inhibitors+of+nicotinamide+N-methyltransferase+%28NNMT%29&doi=10.1021%2Facs.jmedchem.7b01422"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Bisubstrate Inhibitors of Nicotinamide N-Methyltransferase (NNMT)</span></div><div class="casAuthors">Babault, Nicolas; Allali-Hassani, Abdellah; Li, Fengling; Fan, Jie; Yue, Alex; Ju, Kevin; Liu, Feng; Vedadi, Masoud; Liu, Jing; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1541-1551</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Nicotinamide N-methyltransferase (NNMT) catalyzes the N-methylation of pyridine-contg. compds. using the cofactor S-5'-adenosyl-L-methionine (SAM) as the Me group donor.  Through the regulation of the levels of its substrates, cofactor, and products, NNMT plays important role in physiol. and pathophysiol.  Overexpression of NNMT has been implicated in various human diseases.  Potent and selective small-mol. NNMT inhibitors are valuable chem. tools for testing biol. and therapeutic hypotheses.  However, very few NNMT inhibitors have been reported.  Here, we describe the discovery of a bisubstrate NNMT inhibitor MS2734 (6), and characterization of this inhibitor in biochem., biophys., kinetic, and structural studies.  Importantly, we obtained the first crystal structure of human NNMT in complex with a small-mol. inhibitor.  The structure of the NNMT-6 complex has unambiguously demonstrated that 6 occupied both substrate and cofactor binding sites.  The findings paved the way for developing more potent and selective NNMT inhibitors in the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9ThNiA1GTwLVg90H21EOLACvtfcHk0lh-63K_-BxLCw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXotVOruw%253D%253D&md5=50e23e5f60beb64c184f6eadc6a75883</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01422%26sid%3Dliteratum%253Aachs%26aulast%3DBabault%26aufirst%3DN.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DFan%26aufirst%3DJ.%26aulast%3DYue%26aufirst%3DA.%26aulast%3DJu%26aufirst%3DK.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520bisubstrate%2520inhibitors%2520of%2520nicotinamide%2520N-methyltransferase%2520%2528NNMT%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26issue%3D4%26spage%3D1541%26epage%3D1551%26doi%3D10.1021%2Facs.jmedchem.7b01422" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Haren, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moret, E. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rood, J. J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sartini, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salvucci, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emanuelli, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craveur, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Babault, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, N. I.</span></span> <span> </span><span class="NLM_article-title">Bisubstrate inhibitors of nicotinamide N-methyltransferase (NNMT) with enhanced activity</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span> (<span class="NLM_issue">14</span>),  <span class="NLM_fpage">6597</span>– <span class="NLM_lpage">6614</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b00413</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b00413" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtlams7%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=6597-6614&issue=14&author=Y.+Gaoauthor=M.+J.+van+Harenauthor=E.+E.+Moretauthor=J.+J.+M.+Roodauthor=D.+Sartiniauthor=A.+Salvucciauthor=M.+Emanuelliauthor=P.+Craveurauthor=N.+Babaultauthor=J.+Jinauthor=N.+I.+Martin&title=Bisubstrate+inhibitors+of+nicotinamide+N-methyltransferase+%28NNMT%29+with+enhanced+activity&doi=10.1021%2Facs.jmedchem.9b00413"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Bisubstrate Inhibitors of Nicotinamide N-Methyltransferase (NNMT) with Enhanced Activity</span></div><div class="casAuthors">Gao, Yongzhi; van Haren, Matthijs J.; Moret, Ed E.; Rood, Johannes J. M.; Sartini, Davide; Salvucci, Alessia; Emanuelli, Monica; Craveur, Pierrick; Babault, Nicolas; Jin, Jian; Martin, Nathaniel I.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">6597-6614</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Nicotinamide N-methyltransferase (NNMT) catalyzes the methylation of nicotinamide to form N-methylnicotinamide.  Overexpression of NNMT is assocd. with a variety of diseases, including a no. of cancers and metabolic disorders, suggesting a role for NNMT as a potential therapeutic target.  By structural modification of a lead NNMT inhibitor previously developed in our group, we prepd. a diverse library of inhibitors to probe the different regions of the enzyme's active site.  This investigation revealed that incorporation of a naphthalene moiety, intended to bind the hydrophobic nicotinamide binding pocket via π-π stacking interactions, significantly increases the activity of bisubstrate-like NNMT inhibitors (half-maximal inhibitory concn. 1.41 μM).  These findings are further supported by isothermal titrn. calorimetry binding assays as well as modeling studies.  The most active NNMT inhibitor identified in the present study demonstrated a dose-dependent inhibitory effect on the cell proliferation of the HSC-2 human oral cancer cell line.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7zznNASlV9LVg90H21EOLACvtfcHk0lh-63K_-BxLCw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtlams7%252FO&md5=7384150bf22537f835de298dc7889de3</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b00413&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b00413%26sid%3Dliteratum%253Aachs%26aulast%3DGao%26aufirst%3DY.%26aulast%3Dvan%2BHaren%26aufirst%3DM.%2BJ.%26aulast%3DMoret%26aufirst%3DE.%2BE.%26aulast%3DRood%26aufirst%3DJ.%2BJ.%2BM.%26aulast%3DSartini%26aufirst%3DD.%26aulast%3DSalvucci%26aufirst%3DA.%26aulast%3DEmanuelli%26aufirst%3DM.%26aulast%3DCraveur%26aufirst%3DP.%26aulast%3DBabault%26aufirst%3DN.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DMartin%26aufirst%3DN.%2BI.%26atitle%3DBisubstrate%2520inhibitors%2520of%2520nicotinamide%2520N-methyltransferase%2520%2528NNMT%2529%2520with%2520enhanced%2520activity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26issue%3D14%26spage%3D6597%26epage%3D6614%26doi%3D10.1021%2Facs.jmedchem.9b00413" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lavogina, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Enkvist, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uri, A.</span></span> <span> </span><span class="NLM_article-title">Bisubstrate inhibitors of protein kinases: from principle to practical applications</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>5</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">23</span>– <span class="NLM_lpage">34</span>, <span class="refDoi"> DOI: 10.1002/cmdc.200900252</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.1002%2Fcmdc.200900252" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=19774589" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhs1SqsbzP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2010&pages=23-34&issue=1&author=D.+Lavoginaauthor=E.+Enkvistauthor=A.+Uri&title=Bisubstrate+inhibitors+of+protein+kinases%3A+from+principle+to+practical+applications&doi=10.1002%2Fcmdc.200900252"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Bisubstrate Inhibitors of Protein Kinases: from Principle to Practical Applications</span></div><div class="casAuthors">Lavogina, Darja; Enkvist, Erki; Uri, Asko</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">23-34</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  Bisubstrate inhibitors consist of two conjugated fragments, each targeted to a different binding site of a bisubstrate enzyme.  The design of bisubstrate inhibitors presupposes the formation of the ternary complex in the course of the catalyzed reaction.  The principle advantage of bisubstrate inhibitors is their ability to generate more interactions with the target enzyme that could result in improved affinity and selectivity of the conjugates, when compared with single-site inhibitors.  Among phosphotransferases, the approach was first successfully used for adenylate kinase in 1973.  Since then, several types of bisubstrate inhibitors have been developed for protein kinases, including conjugates of peptides with nucleotides, adenosine derivs. and potent ATP-competitive inhibitors.  Earlier bisubstrate inhibitors had pharmacokinetic qualities that were unsuitable for cellular expts. and hence were mostly used for in vitro studies.  The recently constructed conjugates of adenosine derivs. and D-arginine-rich peptides (ARCs) possess high kinase affinity, high biol. and chem. stability and good cell plasma membrane penetrative properties that enable their application in the regulation of cellular protein phosphorylation balances in cell and tissue expts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrSsmmKerEiPrVg90H21EOLACvtfcHk0lh-63K_-BxLCw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhs1SqsbzP&md5=7efb7b5aa44347f5fa8fc40f98065337</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.200900252&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.200900252%26sid%3Dliteratum%253Aachs%26aulast%3DLavogina%26aufirst%3DD.%26aulast%3DEnkvist%26aufirst%3DE.%26aulast%3DUri%26aufirst%3DA.%26atitle%3DBisubstrate%2520inhibitors%2520of%2520protein%2520kinases%253A%2520from%2520principle%2520to%2520practical%2520applications%26jtitle%3DChemMedChem%26date%3D2010%26volume%3D5%26issue%3D1%26spage%3D23%26epage%3D34%26doi%3D10.1002%2Fcmdc.200900252" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lechtenberg, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mace, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sessions, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williamson, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stalder, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wallez, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, G. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riedl, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pasquale, E. B.</span></span> <span> </span><span class="NLM_article-title">Structure-guided strategy for the development of potent bivalent ERK inhibitors</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">726</span>– <span class="NLM_lpage">731</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.7b00127</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.7b00127" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BC2sXpslSksrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=726-731&issue=7&author=B.+C.+Lechtenbergauthor=P.+D.+Maceauthor=E.+H.+Sessionsauthor=R.+Williamsonauthor=R.+Stalderauthor=Y.+Wallezauthor=G.+P.+Rothauthor=S.+J.+Riedlauthor=E.+B.+Pasquale&title=Structure-guided+strategy+for+the+development+of+potent+bivalent+ERK+inhibitors&doi=10.1021%2Facsmedchemlett.7b00127"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Guided Strategy for the Development of Potent Bivalent ERK Inhibitors</span></div><div class="casAuthors">Lechtenberg, Bernhard C.; Mace, Peter D.; Sessions, E. Hampton; Williamson, Robert; Stalder, Romain; Wallez, Yann; Roth, Gregory P.; Riedl, Stefan J.; Pasquale, Elena B.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">726-731</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">ERK is the effector kinase of the RAS-RAF-MEK-ERK signaling cascade, which promotes cell transformation and malignancy in many cancers and is thus a major drug target in oncol.  Kinase inhibitors targeting RAF or MEK are already used for the treatment of certain cancers, such as melanoma.  Although the initial response to these drugs can be dramatic, development of drug resistance is a major challenge, even with combination therapies targeting both RAF and MEK.  Importantly, most resistance mechanisms still rely on activation of the downstream effector kinase ERK, making it a promising target for drug development efforts.  Here, the authors report the design and structural/functional characterization of a set of bivalent ERK inhibitors that combine a small mol. inhibitor that binds to the ATP-binding pocket with a peptide that selectively binds to an ERK protein interaction surface, the D-site recruitment site (DRS).  The authors' studies show that the lead bivalent inhibitor, SBP3, has markedly improved potency compared to the small mol. inhibitor alone.  Unexpectedly, the authors found that SBP3 also binds to several ERK-related kinases that contain a DRS, highlighting the importance of exptl. verifying the predicted specificity of bivalent inhibitors.  However, SBP3 does not target any other kinases belonging to the same CMGC branch of the kinome.  Addnl., the authors' modular click chem. inhibitor design facilitates the generation of different combinations of small mol. inhibitors with ERK-targeting peptides.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTPtcu1FfITrVg90H21EOLACvtfcHk0lh-63K_-BxLCw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXpslSksrk%253D&md5=6f3855f67f0e4002f1b90218c2e8fd89</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.7b00127&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.7b00127%26sid%3Dliteratum%253Aachs%26aulast%3DLechtenberg%26aufirst%3DB.%2BC.%26aulast%3DMace%26aufirst%3DP.%2BD.%26aulast%3DSessions%26aufirst%3DE.%2BH.%26aulast%3DWilliamson%26aufirst%3DR.%26aulast%3DStalder%26aufirst%3DR.%26aulast%3DWallez%26aufirst%3DY.%26aulast%3DRoth%26aufirst%3DG.%2BP.%26aulast%3DRiedl%26aufirst%3DS.%2BJ.%26aulast%3DPasquale%26aufirst%3DE.%2BB.%26atitle%3DStructure-guided%2520strategy%2520for%2520the%2520development%2520of%2520potent%2520bivalent%2520ERK%2520inhibitors%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2017%26volume%3D8%26issue%3D7%26spage%3D726%26epage%3D731%26doi%3D10.1021%2Facsmedchemlett.7b00127" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R. A.</span></span> <i>Evaluation of Enzyme Inhibitors in Drug Discovery: A Guide for Medicinal Chemists and Pharmacologists</i>, <span class="NLM_edition">2nd</span> ed.; <span class="NLM_publisher-name">John Wiley & Sons, Inc.</span>: <span class="NLM_publisher-loc">Hoboken, NJ</span>, <span class="NLM_year">2005</span>; pp  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">538</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2005&pages=1-538&author=R.+A.+Copeland&title=Evaluation+of+Enzyme+Inhibitors+in+Drug+Discovery%3A+A+Guide+for+Medicinal+Chemists+and+Pharmacologists"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26btitle%3DEvaluation%2520of%2520Enzyme%2520Inhibitors%2520in%2520Drug%2520Discovery%253A%2520A%2520Guide%2520for%2520Medicinal%2520Chemists%2520and%2520Pharmacologists%26pub%3DJohn%2520Wiley%2520%2526%2520Sons%252C%2520Inc%26date%3D2005%26spage%3D1%26epage%3D538" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Peng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sartini, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pozzi, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilk, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emanuelli, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yee, V. C.</span></span> <span> </span><span class="NLM_article-title">Structural basis of substrate recognition in human nicotinamide N-methyltransferase</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>50</i></span> (<span class="NLM_issue">36</span>),  <span class="NLM_fpage">7800</span>– <span class="NLM_lpage">7808</span>, <span class="refDoi"> DOI: 10.1021/bi2007614</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi2007614" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtValsbnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2011&pages=7800-7808&issue=36&author=Y.+Pengauthor=D.+Sartiniauthor=V.+Pozziauthor=D.+Wilkauthor=M.+Emanuelliauthor=V.+C.+Yee&title=Structural+basis+of+substrate+recognition+in+human+nicotinamide+N-methyltransferase&doi=10.1021%2Fbi2007614"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis of substrate recognition in human nicotinamide N-methyltransferase</span></div><div class="casAuthors">Peng, Yi; Sartini, Davide; Pozzi, Valentina; Wilk, Dennis; Emanuelli, Monica; Yee, Vivien C.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">36</span>),
    <span class="NLM_cas:pages">7800-7808</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Nicotinamide N-methyltransferase (NNMT) catalyzes the N-methylation of nicotinamide, pyridines, and other analogs using S-adenosyl-L-methionine as donor.  NNMT plays a significant role in the regulation of metabolic pathways and is expressed at markedly high levels in several kinds of cancers, presenting it as a potential mol. target for cancer therapy.  Here, the authors detd. the crystal structure of human NNMT as a ternary complex bound to both the demethylated donor, S-adenosyl-L-homocysteine, and the acceptor substrate, nicotinamide, to 2.7 Å resoln.  These studies reveal the structural basis for nicotinamide binding and highlight several residues in the active site which may play roles in nicotinamide recognition and NNMT catalysis.  The functional importance of these residues was probed by mutagenesis.  Of three residues near the nicotinamide's amide group, substitution of S201 and S213 had no effect on enzyme activity while replacement of D197 dramatically decreased activity.  Substitutions of Y20, whose side chain hydroxyl interacts with both the nicotinamide arom. ring and AdoHcy carboxylate, also compromised activity.  Enzyme kinetics anal. revealed kcat/Km decreases of 2-3 orders of magnitude for the D197A and Y20A mutants, confirming the functional importance of these active site residues.  The mutants exhibited substantially increased Km for both NCA and AdoMet and modestly decreased kcat.  MD simulations revealed long-range conformational effects which provide an explanation for the large increase in Km(AdoMet) for the D197A mutant, which interacts directly only with nicotinamide in the ternary complex crystal structure.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqh1yxD6WIJFLVg90H21EOLACvtfcHk0ljvOFQqoX8v8w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtValsbnF&md5=926af50d2464e21094ffb17205793d19</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fbi2007614&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi2007614%26sid%3Dliteratum%253Aachs%26aulast%3DPeng%26aufirst%3DY.%26aulast%3DSartini%26aufirst%3DD.%26aulast%3DPozzi%26aufirst%3DV.%26aulast%3DWilk%26aufirst%3DD.%26aulast%3DEmanuelli%26aufirst%3DM.%26aulast%3DYee%26aufirst%3DV.%2BC.%26atitle%3DStructural%2520basis%2520of%2520substrate%2520recognition%2520in%2520human%2520nicotinamide%2520N-methyltransferase%26jtitle%3DBiochemistry%26date%3D2011%26volume%3D50%26issue%3D36%26spage%3D7800%26epage%3D7808%26doi%3D10.1021%2Fbi2007614" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R.
A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banks, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klicic, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mainz, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Repasky, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knoll, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shelley, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perry, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Francis, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shenkin, P. S.</span></span> <span> </span><span class="NLM_article-title">Glide: a new approach for rapid, accurate docking and scoring. 1. method and assessment of docking accuracy</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>47</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">1739</span>– <span class="NLM_lpage">1749</span>, <span class="refDoi"> DOI: 10.1021/jm0306430</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0306430" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFyit74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1739-1749&issue=7&author=R.%0AA.+Friesnerauthor=J.+L.+Banksauthor=R.+B.+Murphyauthor=T.+A.+Halgrenauthor=J.+J.+Klicicauthor=D.+T.+Mainzauthor=M.+P.+Repaskyauthor=E.+H.+Knollauthor=M.+Shelleyauthor=J.+K.+Perryauthor=D.+E.+Shawauthor=P.+Francisauthor=P.+S.+Shenkin&title=Glide%3A+a+new+approach+for+rapid%2C+accurate+docking+and+scoring.+1.+method+and+assessment+of+docking+accuracy&doi=10.1021%2Fjm0306430"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Glide: A new approach for rapid, accurate docking and scoring. 1. method and assessment of docking accuracy</span></div><div class="casAuthors">Friesner, Richard A.; Banks, Jay L.; Murphy, Robert B.; Halgren, Thomas A.; Klicic, Jasna J.; Mainz, Daniel T.; Repasky, Matthew P.; Knoll, Eric H.; Shelley, Mee; Perry, Jason K.; Shaw, David E.; Francis, Perry; Shenkin, Peter S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1739-1749</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Unlike other methods for docking ligands to the rigid 3D structure of a known protein receptor, Glide approximates a complete systematic search of the conformational, orientational, and positional space of the docked ligand.  In this search, an initial rough positioning and scoring phase that dramatically narrows the search space is followed by torsionally flexible energy optimization on an OPLS-AA nonbonded potential grid for a few hundred surviving candidate poses.  The very best candidates are further refined via a Monte Carlo sampling of pose conformation; in some cases, this is crucial to obtaining an accurate docked pose.  Selection of the best docked pose uses a model energy function that combines empirical and force-field-based terms.  Docking accuracy is assessed by redocking ligands from 282 cocrystd. PDB complexes starting from conformationally optimized ligand geometries that bear no memory of the correctly docked pose.  Errors in geometry for the top-ranked pose are less than 1 Å in nearly half of the cases and are greater than 2 Å in only about one-third of them.  Comparisons to published data on rms deviations show that Glide is nearly twice as accurate as GOLD and more than twice as accurate as FlexX for ligands having up to 20 rotatable bonds.  Glide is also found to be more accurate than the recently described Surflex method.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPuigu31qMMLVg90H21EOLACvtfcHk0lhkLjgAEWWdCA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFyit74%253D&md5=8cc2f0022318b12dd972e9c493375bf9</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Fjm0306430&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0306430%26sid%3Dliteratum%253Aachs%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DBanks%26aufirst%3DJ.%2BL.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DKlicic%26aufirst%3DJ.%2BJ.%26aulast%3DMainz%26aufirst%3DD.%2BT.%26aulast%3DRepasky%26aufirst%3DM.%2BP.%26aulast%3DKnoll%26aufirst%3DE.%2BH.%26aulast%3DShelley%26aufirst%3DM.%26aulast%3DPerry%26aufirst%3DJ.%2BK.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26aulast%3DFrancis%26aufirst%3DP.%26aulast%3DShenkin%26aufirst%3DP.%2BS.%26atitle%3DGlide%253A%2520a%2520new%2520approach%2520for%2520rapid%252C%2520accurate%2520docking%2520and%2520scoring.%25201.%2520method%2520and%2520assessment%2520of%2520docking%2520accuracy%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26issue%3D7%26spage%3D1739%26epage%3D1749%26doi%3D10.1021%2Fjm0306430" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yau, E. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coward, J. K.</span></span> <span> </span><span class="NLM_article-title">Synthesis of complex 6’-alkynyl-6’-dethia nucleoside analogs of S-adenosylhomocysteine as potential inhibitors of methyltransferases</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1990</span>,  <span class="NLM_volume"><i>55</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">3147</span>– <span class="NLM_lpage">3158</span>, <span class="refDoi"> DOI: 10.1021/jo00297a034</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo00297a034" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADyaK3cXitFOmsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=1990&pages=3147-3158&issue=10&author=E.+K.+Yauauthor=J.+K.+Coward&title=Synthesis+of+complex+6%E2%80%99-alkynyl-6%E2%80%99-dethia+nucleoside+analogs+of+S-adenosylhomocysteine+as+potential+inhibitors+of+methyltransferases&doi=10.1021%2Fjo00297a034"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis of complex 6'-alkynyl-6'-dethia nucleoside analogs of S-adenosylhomocysteine as potential inhibitors of methyltransferases</span></div><div class="casAuthors">Yau, Eric K.; Coward, James K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">1990</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3147-58</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    </div><div class="casAbstract">The synthesis of complex acetylenic nucleosides contg. a 5',6' carbon-carbon bond has been investigated.  Two synthetic routes were studied, both starting with an α-substituted butyrolactone.  The lactone could be converted to an β-keto sulfone and then to the corresponding enol phosphate di-Ph ester.  Reductive elimination to yield the desired aryl acetylene and could not be effected without some overredn. to the substituted olefin.  Alternatively, the lactone could be converted to an appropriately protected γ-hydroxy aldehyde which was then homologated to a terminal acetylene.  The successful synthesis of a prototypic member of the target class of compds. involved in a Pd(II)-mediated coupling of an aryl iodide to the appropriately functionalized terminal acetylene.  The resulting aryl acetylene was then further elaborated to the target amino acid deriv. I.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqnyIyG3Ot5E7Vg90H21EOLACvtfcHk0lhkLjgAEWWdCA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3cXitFOmsrY%253D&md5=a5419abf2f05077d8c839714a1a59d01</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Fjo00297a034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo00297a034%26sid%3Dliteratum%253Aachs%26aulast%3DYau%26aufirst%3DE.%2BK.%26aulast%3DCoward%26aufirst%3DJ.%2BK.%26atitle%3DSynthesis%2520of%2520complex%25206%25E2%2580%2599-alkynyl-6%25E2%2580%2599-dethia%2520nucleoside%2520analogs%2520of%2520S-adenosylhomocysteine%2520as%2520potential%2520inhibitors%2520of%2520methyltransferases%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D1990%26volume%3D55%26issue%3D10%26spage%3D3147%26epage%3D3158%26doi%3D10.1021%2Fjo00297a034" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Burns, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coward, J. K.</span></span> <span> </span><span class="NLM_article-title">Synthesis of complex δ-acetylenic amino acids as potential multisubstrate adduct inhibitors of methyltransferases</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>4</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">1455</span>– <span class="NLM_lpage">1470</span>, <span class="refDoi"> DOI: 10.1016/0968-0896(96)00139-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.1016%2F0968-0896%2896%2900139-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=8894103" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADyaK28Xmt1Sgsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=1996&pages=1455-1470&issue=9&author=M.+R.+Burnsauthor=J.+K.+Coward&title=Synthesis+of+complex+%CE%B4-acetylenic+amino+acids+as+potential+multisubstrate+adduct+inhibitors+of+methyltransferases&doi=10.1016%2F0968-0896%2896%2900139-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis of complex δ-acetylenic amino acids as potential multisubstrate adduct inhibitors of methyltransferases</span></div><div class="casAuthors">Burns, Mark R.; Coward, James K.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1455-1470</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">The synthesis of two types of δ-acetylenic amino acids is described.  Key intermediates were derived from terminal acetylenes via two different routes: (1) palladium-mediated, Heck-type arylation and (2) Simmons-Smith homologation followed by reaction of the resulting propargylic organometallic with a benzoyltrimethylsilane.  Further elaboration to the desired amino acids involved the coupling of carbanions derived from N-benzylidene glycine esters to complex alkyl halides.  The synthesis of nonnucleoside δ-acetylenic amino acids was successfully effected using this chem.  In the case of the nucleoside-contg. amino acids, a potential multisubstrate adduct inhibitor of catechol O-methyltransferase was synthesized via this route.  Unfortunately, the sensitivity to acid of 5'-deoxy, 5'-carbanucleosides prevented successful completion of the synthesis of a second nucleoside-contg. δ-amino acid as a possible inhibitor of phenethanolamine N-methyltransferase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZMqf8MG5ZPrVg90H21EOLACvtfcHk0lhkLjgAEWWdCA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28Xmt1Sgsbk%253D&md5=da68540557c8312b0db2ab501250bae5</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2F0968-0896%2896%2900139-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0968-0896%252896%252900139-3%26sid%3Dliteratum%253Aachs%26aulast%3DBurns%26aufirst%3DM.%2BR.%26aulast%3DCoward%26aufirst%3DJ.%2BK.%26atitle%3DSynthesis%2520of%2520complex%2520%25CE%25B4-acetylenic%2520amino%2520acids%2520as%2520potential%2520multisubstrate%2520adduct%2520inhibitors%2520of%2520methyltransferases%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D1996%26volume%3D4%26issue%3D9%26spage%3D1455%26epage%3D1470%26doi%3D10.1016%2F0968-0896%2896%2900139-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Whittaker, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lalic, G.</span></span> <span> </span><span class="NLM_article-title">Monophasic catalytic system for the selective semireduction of alkynes</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>15</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">1112</span>– <span class="NLM_lpage">1115</span>, <span class="refDoi"> DOI: 10.1021/ol4001679</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ol4001679" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BC3sXivVyju7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2013&pages=1112-1115&issue=5&author=A.+M.+Whittakerauthor=G.+Lalic&title=Monophasic+catalytic+system+for+the+selective+semireduction+of+alkynes&doi=10.1021%2Fol4001679"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Monophasic Catalytic System for the Selective Semireduction of Alkynes</span></div><div class="casAuthors">Whittaker, Aaron M.; Lalic, Gojko</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1112-1115</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7052</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A highly efficient semiredn. of alkynes has been developed.  Using 0.5-2 mol % of a copper catalyst, semiredn. can be accomplished with a wide range of substrates, including both internal and terminal alkynes without over-redn.  The new method has excellent chemoselectivity, and the semiredn. can be accomplished even in the presence of nitro and aryl iodo groups.  Semiredn. of internal alkynes produces Z-alkenes selectively.  Finally, com. availability of a catalyst precursor adds to the appeal of the new catalytic system.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohR-uMPDtKvrVg90H21EOLACvtfcHk0lj_iruiTkf34g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXivVyju7o%253D&md5=403c014320ca7da5fcb4e0c4c5c88200</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Fol4001679&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fol4001679%26sid%3Dliteratum%253Aachs%26aulast%3DWhittaker%26aufirst%3DA.%2BM.%26aulast%3DLalic%26aufirst%3DG.%26atitle%3DMonophasic%2520catalytic%2520system%2520for%2520the%2520selective%2520semireduction%2520of%2520alkynes%26jtitle%3DOrg.%2520Lett.%26date%3D2013%26volume%3D15%26issue%3D5%26spage%3D1112%26epage%3D1115%26doi%3D10.1021%2Fol4001679" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Michrowska, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bujok, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harutyunyan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sashuk, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dolgonos, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grela, K.</span></span> <span> </span><span class="NLM_article-title">Nitro-substituted Hoveyda-Grubbs ruthenium carbenes: enhancement of catalyst activity through electronic activation</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>126</i></span> (<span class="NLM_issue">30</span>),  <span class="NLM_fpage">9318</span>– <span class="NLM_lpage">9325</span>, <span class="refDoi"> DOI: 10.1021/ja048794v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja048794v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BD2cXlsFKrsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=126&publication_year=2004&pages=9318-9325&issue=30&author=A.+Michrowskaauthor=R.+Bujokauthor=S.+Harutyunyanauthor=V.+Sashukauthor=G.+Dolgonosauthor=K.+Grela&title=Nitro-substituted+Hoveyda-Grubbs+ruthenium+carbenes%3A+enhancement+of+catalyst+activity+through+electronic+activation&doi=10.1021%2Fja048794v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Nitro-Substituted Hoveyda-Grubbs Ruthenium Carbenes: Enhancement of Catalyst Activity through Electronic Activation</span></div><div class="casAuthors">Michrowska, Anna; Bujok, Robert; Harutyunyan, Syuzanna; Sashuk, Volodymyr; Dolgonos, Grigory; Grela, Karol</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">126</span>
        (<span class="NLM_cas:issue">30</span>),
    <span class="NLM_cas:pages">9318-9325</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The design, synthesis, stability, and catalytic activity of nitro-substituted Hoveyda-Grubbs metathesis catalysts [e.g. Cl2(1,3-bis(mesityl)imidazolidin-2-ylidene)Ru(:CHC6H3OiPr-2-NO2-5) with alkoxy-O also coordinating to Ru] are described.  The highly active and stable meta- and para-substituted complexes are attractive from a practical point of view.  These catalysts operate in very mild conditions and can be successfully applied in various types of metathesis [ring-closing metathesis, cross-metathesis (CM), and enyne metathesis].  Although the presence of a NO2 group leads to catalysts that are dramatically more active than both the 2nd-generation Grubbs's catalyst and the phosphine-free Hoveyda's carbene, enhancement of reactivity is somewhat lower than that obsd. for a sterically activated Hoveyda-Grubbs catalyst.  Attempts to combine two modes of activation, steric and electronic, result in severely decreasing a catalyst's stability.  Different Ru catalysts turned out to be optimal for different applications.  Whereas phosphine-free carbenes are catalysts of choice for CM of various electron-deficient substrates, they exhibit lower reactivity in the formation of tetrasubstituted double bonds.  No single catalyst outperforms all others in all possible applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqyxhE8BsxuL7Vg90H21EOLACvtfcHk0lj_iruiTkf34g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXlsFKrsro%253D&md5=9179e5f461bbe679eb05e51c042f37b5</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fja048794v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja048794v%26sid%3Dliteratum%253Aachs%26aulast%3DMichrowska%26aufirst%3DA.%26aulast%3DBujok%26aufirst%3DR.%26aulast%3DHarutyunyan%26aufirst%3DS.%26aulast%3DSashuk%26aufirst%3DV.%26aulast%3DDolgonos%26aufirst%3DG.%26aulast%3DGrela%26aufirst%3DK.%26atitle%3DNitro-substituted%2520Hoveyda-Grubbs%2520ruthenium%2520carbenes%253A%2520enhancement%2520of%2520catalyst%2520activity%2520through%2520electronic%2520activation%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2004%26volume%3D126%26issue%3D30%26spage%3D9318%26epage%3D9325%26doi%3D10.1021%2Fja048794v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nishimura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sawano, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayashi, T.</span></span> <span> </span><span class="NLM_article-title">Asymmetric synthesis of β-alkynyl aldehydes by rhodium-catalyzed conjugate alkynylation</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>48</i></span> (<span class="NLM_issue">43</span>),  <span class="NLM_fpage">8057</span>– <span class="NLM_lpage">8059</span>, <span class="refDoi"> DOI: 10.1002/anie.200904486</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.1002%2Fanie.200904486" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1Kju73I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2009&pages=8057-8059&issue=43&author=T.+Nishimuraauthor=T.+Sawanoauthor=T.+Hayashi&title=Asymmetric+synthesis+of+%CE%B2-alkynyl+aldehydes+by+rhodium-catalyzed+conjugate+alkynylation&doi=10.1002%2Fanie.200904486"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Asymmetric Synthesis of β-Alkynyl Aldehydes by Rhodium-Catalyzed Conjugate Alkynylation</span></div><div class="casAuthors">Nishimura, Takahiro; Sawano, Takahiro; Hayashi, Tamio</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">43</span>),
    <span class="NLM_cas:pages">8057-8059, S8057/1-S8057/61</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A rhodium-catalyzed conjugate alkynylation of α,β-unsatd. aldehydes giving enantio-enriched β-alkynyl aldehydes is reported.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTmsiMX6A4WLVg90H21EOLACvtfcHk0lj_iruiTkf34g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht1Kju73I&md5=7c662585f6b725253107794ecb73767d</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1002%2Fanie.200904486&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.200904486%26sid%3Dliteratum%253Aachs%26aulast%3DNishimura%26aufirst%3DT.%26aulast%3DSawano%26aufirst%3DT.%26aulast%3DHayashi%26aufirst%3DT.%26atitle%3DAsymmetric%2520synthesis%2520of%2520%25CE%25B2-alkynyl%2520aldehydes%2520by%2520rhodium-catalyzed%2520conjugate%2520alkynylation%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2009%26volume%3D48%26issue%3D43%26spage%3D8057%26epage%3D8059%26doi%3D10.1002%2Fanie.200904486" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Burk, M. J.</span></span> <span> </span><span class="NLM_article-title">Modular phospholane ligands in asymmetric catalysis</span>. <i>Acc. Chem. Res.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>33</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">363</span>– <span class="NLM_lpage">372</span>, <span class="refDoi"> DOI: 10.1021/ar990085c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ar990085c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BD3cXhsl2hu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2000&pages=363-372&issue=6&author=M.+J.+Burk&title=Modular+phospholane+ligands+in+asymmetric+catalysis&doi=10.1021%2Far990085c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Modular Phospholane Ligands in Asymmetric Catalysis</span></div><div class="casAuthors">Burk, Mark J.</div><div class="citationInfo"><span class="NLM_cas:title">Accounts of Chemical Research</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">363-372</span>CODEN:
                <span class="NLM_cas:coden">ACHRE4</span>;
        ISSN:<span class="NLM_cas:issn">0001-4842</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review contg. 51 refs. outlines the prepn. and application of a class of phosphine ligands based upon the trans-2,5-disubstituted phospholane moiety.  The modular nature of these ligands has allowed facile variation of both phospholane substituent and backbone structure, thus providing access to ligands.  Bidentate bis(phospholane) ligands are very useful in asym. catalytic hydrogenation reactions.  In particular, the authors highlight the versatility of highly efficient bis(phospholane)rhodium catalysts that allow enantioselective hydrogenation to produce a diverse range of compds. contg. C-N, C-O, and C-C stereogenic centers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpvHWVq09cZe7Vg90H21EOLACvtfcHk0lj_iruiTkf34g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXhsl2hu7s%253D&md5=b592ee7c317233753e93254fceb58d55</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Far990085c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Far990085c%26sid%3Dliteratum%253Aachs%26aulast%3DBurk%26aufirst%3DM.%2BJ.%26atitle%3DModular%2520phospholane%2520ligands%2520in%2520asymmetric%2520catalysis%26jtitle%3DAcc.%2520Chem.%2520Res.%26date%3D2000%26volume%3D33%26issue%3D6%26spage%3D363%26epage%3D372%26doi%3D10.1021%2Far990085c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sniady, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bedore, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jamison, T. F.</span></span> <span> </span><span class="NLM_article-title">One-flow, multistep synthesis of nucleosides by Bronsted acid-catalyzed glycosylation</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>50</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">2155</span>– <span class="NLM_lpage">2158</span>, <span class="refDoi"> DOI: 10.1002/anie.201006440</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.1002%2Fanie.201006440" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BC3MXisVWmtLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2011&pages=2155-2158&issue=9&author=A.+Sniadyauthor=M.+W.+Bedoreauthor=T.+F.+Jamison&title=One-flow%2C+multistep+synthesis+of+nucleosides+by+Bronsted+acid-catalyzed+glycosylation&doi=10.1002%2Fanie.201006440"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">One-Flow, Multistep Synthesis of Nucleosides by Bronsted Acid-Catalyzed Glycosylation</span></div><div class="casAuthors">Sniady, Adam; Bedore, Matthew W.; Jamison, Timothy F.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2155-2158, S2155/1-S2155/50</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Synthesis of nucleosides and nucleoside analogs using pyridinium triflate salts as efficient glycosylation catalysts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomJzJ1cWZ6JbVg90H21EOLACvtfcHk0lipxqfBYVqzkQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXisVWmtLY%253D&md5=58995c77e36a344b951499abc7502873</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1002%2Fanie.201006440&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201006440%26sid%3Dliteratum%253Aachs%26aulast%3DSniady%26aufirst%3DA.%26aulast%3DBedore%26aufirst%3DM.%2BW.%26aulast%3DJamison%26aufirst%3DT.%2BF.%26atitle%3DOne-flow%252C%2520multistep%2520synthesis%2520of%2520nucleosides%2520by%2520Bronsted%2520acid-catalyzed%2520glycosylation%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2011%26volume%3D50%26issue%3D9%26spage%3D2155%26epage%3D2158%26doi%3D10.1002%2Fanie.201006440" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Neelakantan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vance, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McHardy, S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watowich, S. J.</span></span> <span> </span><span class="NLM_article-title">Noncoupled fluorescent assay for direct real-time monitoring of nicotinamide N-methyltransferase activity</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>56</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">824</span>– <span class="NLM_lpage">832</span>, <span class="refDoi"> DOI: 10.1021/acs.biochem.6b01215</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.biochem.6b01215" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFajs74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2017&pages=824-832&issue=6&author=H.+Neelakantanauthor=V.+Vanceauthor=H.+L.+Wangauthor=S.+F.+McHardyauthor=S.+J.+Watowich&title=Noncoupled+fluorescent+assay+for+direct+real-time+monitoring+of+nicotinamide+N-methyltransferase+activity&doi=10.1021%2Facs.biochem.6b01215"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Noncoupled Fluorescent Assay for Direct Real-Time Monitoring of Nicotinamide N-Methyltransferase Activity</span></div><div class="casAuthors">Neelakantan, Harshini; Vance, Virginia; Wang, Hua-Yu Leo; McHardy, Stanton F.; Watowich, Stanley J.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">824-832</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Nicotinamide N-methyltransferase (NNMT) is an important biotransforming enzyme that catalyzes the transfer of a labile Me group from the ubiquitous cofactor, S-adenosyl-L-methionine (SAM), to endogenous and exogenous small mols. such as quinoline to form methylated end-products.  NNMT has been implicated in a no. of chronic disease conditions, including metabolic disorders, cardiovascular disease, cancer, osteoarthritis, kidney disease, and Parkinson's disease.  Here, the authors developed a novel noncoupled fluorescence-based methyltransferase assay that enabled direct ultrasensitive real-time detection of the NNMT reaction product, 1-methylquinolinium.  This is the 1st assay reported to date to utilize fluorescence spectroscopy to directly monitor NNMT product formation and activity in real-time.  This assay provided accurate kinetic data that enabled detailed comparative anal. of the NNMT reaction mechanism and kinetic parameters.  A reaction model based on a random bi-reactant mechanism produced global curve fits that were most consistent with steady-state initial velocity data collected across an array of substrate concns.  Based on the reaction mechanism, each substrate could independently bind to the NNMT apoenzyme; however, both substrates bound to the complementary binary complexes with ∼20-fold higher affinity relative to independent binding.  This reaction mechanism implies either substrate-induced conformational changes or bi-reactant intermol. interactions may stabilize substrate binding to the binary complex and formation of the ternary complex.  Importantly, this assay could rapidly generate concn. response curves for known NNMT inhibitors, suggesting its applicability for high-throughput screening (HTS) of chem. libraries to identify novel NNMT inhibitors.  Furthermore, this novel assay potentially offers a robust detection technol. for use in SAM substrate competition assays for the discovery and development of SAM-dependent methyltransferase inhibitors such as S-adenosyl-L-homocysteine and 1-methylnicotinamide.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6X5AP5dGb77Vg90H21EOLACvtfcHk0lipxqfBYVqzkQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFajs74%253D&md5=bf243bd7362322b094c31b1e247b7240</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Facs.biochem.6b01215&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.biochem.6b01215%26sid%3Dliteratum%253Aachs%26aulast%3DNeelakantan%26aufirst%3DH.%26aulast%3DVance%26aufirst%3DV.%26aulast%3DWang%26aufirst%3DH.%2BL.%26aulast%3DMcHardy%26aufirst%3DS.%2BF.%26aulast%3DWatowich%26aufirst%3DS.%2BJ.%26atitle%3DNoncoupled%2520fluorescent%2520assay%2520for%2520direct%2520real-time%2520monitoring%2520of%2520nicotinamide%2520N-methyltransferase%2520activity%26jtitle%3DBiochemistry%26date%3D2017%26volume%3D56%26issue%3D6%26spage%3D824%26epage%3D832%26doi%3D10.1021%2Facs.biochem.6b01215" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Loring, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, P. R.</span></span> <span> </span><span class="NLM_article-title">Kinetic mechanism of nicotinamide N-methyltransferase</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>57</i></span> (<span class="NLM_issue">38</span>),  <span class="NLM_fpage">5524</span>– <span class="NLM_lpage">5532</span>, <span class="refDoi"> DOI: 10.1021/acs.biochem.8b00775</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.biochem.8b00775" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsFyksLnM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2018&pages=5524-5532&issue=38&author=H.+S.+Loringauthor=P.+R.+Thompson&title=Kinetic+mechanism+of+nicotinamide+N-methyltransferase&doi=10.1021%2Facs.biochem.8b00775"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Kinetic Mechanism of Nicotinamide N-Methyltransferase</span></div><div class="casAuthors">Loring, Heather S.; Thompson, Paul R.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">38</span>),
    <span class="NLM_cas:pages">5524-5532</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Nicotinamide N-methyltransferase (NNMT) catalyzes the transfer of a Me group from S-adenosyl methionine to nicotinamide, pyridine and other structural analogs.  Aberrantly increased NNMT activity results in the depletion of S-adenosyl methionine (SAM), nicotinamide (NAM), and NAD+; NAM is required for NAD+ biosynthesis.  SAM depletion impairs the methylation potential of the cell, resulting in hypomethylated histones and an altered epigenetic profile.  In addn., decreased NAD+ levels neg. affect energy metab. by disrupting oxidative phosphorylation.  Due to its impact on epigenetic states and NAD+ levels, NNMT is implicated in cancer, neurodegenerative and metabolic diseases, making it an appealing target for therapeutic intervention.  To gain insights that would guide the design of inhibitors and activity-based probes, we performed detailed kinetic studies on human NNMT.  Herein, we report the kinetic mechanism of NNMT.  Our initial velocity, product inhibition, and dead-end analog inhibition studies collectively indicate that NNMT uses a rapid equil. ordered mechanism, where NNMT first binds SAM, which is followed by NAM.  Me transfer occurs and methylated NAM (MeNAM) and S-adenosyl homocysteine (SAH) are released consecutively.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBKv-k7WIajbVg90H21EOLACvtfcHk0lipxqfBYVqzkQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsFyksLnM&md5=8c67d54bd73a7fe4ad3dc51411d12d1e</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Facs.biochem.8b00775&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.biochem.8b00775%26sid%3Dliteratum%253Aachs%26aulast%3DLoring%26aufirst%3DH.%2BS.%26aulast%3DThompson%26aufirst%3DP.%2BR.%26atitle%3DKinetic%2520mechanism%2520of%2520nicotinamide%2520N-methyltransferase%26jtitle%3DBiochemistry%26date%3D2018%26volume%3D57%26issue%3D38%26spage%3D5524%26epage%3D5532%26doi%3D10.1021%2Facs.biochem.8b00775" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cleland, W. W.</span></span> <span> </span><span class="NLM_article-title">The kinetics of enzyme-catalyzed reactions with two or more substrates or products: I. nomenclature and rate equations</span>. <i>Biochim. Biophys. Acta, Spec. Sect. Enzymol. Subj.</i> <span class="NLM_year" style="font-weight: bold;">1963</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">104</span>– <span class="NLM_lpage">137</span>, <span class="refDoi"> DOI: 10.1016/0926-6569(63)90211-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.1016%2F0926-6569%2863%2990211-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADyaF3sXksl2gtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=1963&pages=104-137&author=W.+W.+Cleland&title=The+kinetics+of+enzyme-catalyzed+reactions+with+two+or+more+substrates+or+products%3A+I.+nomenclature+and+rate+equations&doi=10.1016%2F0926-6569%2863%2990211-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">The kinetics of enzyme-catalyzed reactions with two or more substrates or products. I. Nomenclature and rate equations</span></div><div class="casAuthors">Cleland, W. W.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Specialized Section on Enzymological Subjects</span>
        (<span class="NLM_cas:date">1963</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">104-37</span>CODEN:
                <span class="NLM_cas:coden">BBASD9</span>;
        ISSN:<span class="NLM_cas:issn">0926-6569</span>.
    </div><div class="casAbstract">A nomenclature is proposed to facilitate discussion of possible mechanisms for enzyme-catalyzed reactions with more than one substrate or product.  A shorthand notation for writing such mechanisms is also described.  A general method for expressing the full steady-state rate equations for these mechanisms in terms of measurable kinetic consts. is explained, and the resulting rate equations are given for a no. of mechanisms with 2 or 3 substrates or products.  The manner in which kinetic studies can be used to det. enzymic mechanisms is discussed, and examples are given to illustrate the types of information about reaction mechanisms that can be obtained.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQjkgumn5qvrVg90H21EOLACvtfcHk0liliqCrF4COkg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaF3sXksl2gtA%253D%253D&md5=50de4ab301560c410f6348e096e2db8d</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2F0926-6569%2863%2990211-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0926-6569%252863%252990211-6%26sid%3Dliteratum%253Aachs%26aulast%3DCleland%26aufirst%3DW.%2BW.%26atitle%3DThe%2520kinetics%2520of%2520enzyme-catalyzed%2520reactions%2520with%2520two%2520or%2520more%2520substrates%2520or%2520products%253A%2520I.%2520nomenclature%2520and%2520rate%2520equations%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Spec.%2520Sect.%2520Enzymol.%2520Subj.%26date%3D1963%26volume%3D67%26spage%3D104%26epage%3D137%26doi%3D10.1016%2F0926-6569%2863%2990211-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cleland, W. W.</span></span> <span> </span><span class="NLM_article-title">The kinetics of enzyme-catalyzed reactions with two or more substrates or products: II. inhibition: nomenclature and theory</span>. <i>Biochim. Biophys. Acta, Spec. Sect. Enzymol. Subj.</i> <span class="NLM_year" style="font-weight: bold;">1963</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">173</span>– <span class="NLM_lpage">187</span>, <span class="refDoi"> DOI: 10.1016/0926-6569(63)90226-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.1016%2F0926-6569%2863%2990226-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADyaF3sXmslOjsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=1963&pages=173-187&author=W.+W.+Cleland&title=The+kinetics+of+enzyme-catalyzed+reactions+with+two+or+more+substrates+or+products%3A+II.+inhibition%3A+nomenclature+and+theory&doi=10.1016%2F0926-6569%2863%2990226-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">The kinetics of enzyme-catalyzed reactions with two or more substrates or products. II. Inhibition--nomenclature and theory</span></div><div class="casAuthors">Cleland, W. W.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Specialized Section on Enzymological Subjects</span>
        (<span class="NLM_cas:date">1963</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">173-87</span>CODEN:
                <span class="NLM_cas:coden">BBASD9</span>;
        ISSN:<span class="NLM_cas:issn">0926-6569</span>.
    </div><div class="casAbstract">cf. CA 58, 8192c.  A nomenclature is proposed to describe different types of inhibitions of enzyme-catalyzed reactions, particularly for reactions with more than one substrate and product.  The rate equations for such inhibitions are discussed, as are methods for distinguishing between the various types of inhibition and obtaining inhibition and kinetic consts. from exptl. data.  Several examples are given of the type of information that can be obtained from inhibition studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGppRMzEG8P0qrVg90H21EOLACvtfcHk0liliqCrF4COkg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaF3sXmslOjsQ%253D%253D&md5=72a7b057094e41748e51e66a309f30bf</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2F0926-6569%2863%2990226-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0926-6569%252863%252990226-8%26sid%3Dliteratum%253Aachs%26aulast%3DCleland%26aufirst%3DW.%2BW.%26atitle%3DThe%2520kinetics%2520of%2520enzyme-catalyzed%2520reactions%2520with%2520two%2520or%2520more%2520substrates%2520or%2520products%253A%2520II.%2520inhibition%253A%2520nomenclature%2520and%2520theory%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Spec.%2520Sect.%2520Enzymol.%2520Subj.%26date%3D1963%26volume%3D67%26spage%3D173%26epage%3D187%26doi%3D10.1016%2F0926-6569%2863%2990226-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cleland, W. W.</span></span> <span> </span><span class="NLM_article-title">The kinetics of enzyme-catalyzed reactions with two or more substrates or products: III. prediction of initial velocity and inhibition patterns by inspection</span>. <i>Biochim. Biophys. Acta, Spec. Sect. Enzymol. Subj.</i> <span class="NLM_year" style="font-weight: bold;">1963</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">188</span>– <span class="NLM_lpage">196</span>, <span class="refDoi"> DOI: 10.1016/0926-6569(63)90227-X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.1016%2F0926-6569%2863%2990227-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADyaF3sXmslOjtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=1963&pages=188-196&author=W.+W.+Cleland&title=The+kinetics+of+enzyme-catalyzed+reactions+with+two+or+more+substrates+or+products%3A+III.+prediction+of+initial+velocity+and+inhibition+patterns+by+inspection&doi=10.1016%2F0926-6569%2863%2990227-X"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">The kinetics of enzyme-catalyzed reactions with two or more substrates or products. III. Prediction of initial velocity and inhibition patterns by inspection</span></div><div class="casAuthors">Cleland, W. W.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Specialized Section on Enzymological Subjects</span>
        (<span class="NLM_cas:date">1963</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">188-96</span>CODEN:
                <span class="NLM_cas:coden">BBASD9</span>;
        ISSN:<span class="NLM_cas:issn">0926-6569</span>.
    </div><div class="casAbstract">A general method for predicting initial velocity and dead end and product inhibition patterns by inspection of the mechanism is presented.  This method is applicable to any nonrandom mechanism without alternate reaction sequences.  Several examples of the application of the method are given.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3BXiUFNAX1rVg90H21EOLACvtfcHk0liliqCrF4COkg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaF3sXmslOjtg%253D%253D&md5=f63ac2b9ba05bef8de8d3768752a6de7</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2F0926-6569%2863%2990227-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0926-6569%252863%252990227-X%26sid%3Dliteratum%253Aachs%26aulast%3DCleland%26aufirst%3DW.%2BW.%26atitle%3DThe%2520kinetics%2520of%2520enzyme-catalyzed%2520reactions%2520with%2520two%2520or%2520more%2520substrates%2520or%2520products%253A%2520III.%2520prediction%2520of%2520initial%2520velocity%2520and%2520inhibition%2520patterns%2520by%2520inspection%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Spec.%2520Sect.%2520Enzymol.%2520Subj.%26date%3D1963%26volume%3D67%26spage%3D188%26epage%3D196%26doi%3D10.1016%2F0926-6569%2863%2990227-X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">van
Haren, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sastre Torano, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sartini, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emanuelli, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parsons, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, N. I.</span></span> <span> </span><span class="NLM_article-title">A rapid and efficient assay for the characterization of substrates and inhibitors of nicotinamide N-methyltransferase</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">5307</span>– <span class="NLM_lpage">5315</span>, <span class="refDoi"> DOI: 10.1021/acs.biochem.6b00733</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.biochem.6b00733" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVWntL7O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2016&pages=5307-5315&author=M.+J.+van%0AHarenauthor=J.+Sastre+Toranoauthor=D.+Sartiniauthor=M.+Emanuelliauthor=R.+B.+Parsonsauthor=N.+I.+Martin&title=A+rapid+and+efficient+assay+for+the+characterization+of+substrates+and+inhibitors+of+nicotinamide+N-methyltransferase&doi=10.1021%2Facs.biochem.6b00733"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">A rapid and efficient assay for the characterization of substrates and inhibitors of nicotinamide N-methyltransferase</span></div><div class="casAuthors">van Haren, Matthijs J.; Sastre Torano, Javier; Sartini, Davide; Emanuelli, Monica; Parsons, Richard B.; Martin, Nathaniel I.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">37</span>),
    <span class="NLM_cas:pages">5307-5315</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Nicotinamide N-methyltransferase (NNMT) is one of the most abundant small mol. methyltransferases in the human body and is primarily responsible for the N-methylation of the nicotinamide (vitamin B3).  Employing the cofactor, S-adenosyl-L-methionine, NNMT transfers a Me group to the pyridine N atom of nicotinamide to generate N-methylnicotinamide.  Interestingly, NNMT is also able to N-methylate a variety of other pyridine-contg. small mols., suggesting a secondary role for the enzyme in the detoxification of xenobiotics.  A no. of recent studies have also revealed links between NNMT overexpression and a variety of diseases, including multiple cancers, Parkinson's disease, diabetes, and obesity.  Here, to facilitate further study of both the substrate scope and potential for inhibitor development, the authors describe the development of a new activity assay for human NNMT.  The assay makes use of ultra-high-performance hydrophilic interaction chromatog., allowing for the rapid sepn. of the reaction products, coupled with quadrupole time-of-flight mass spectrometric detection, providing for enhanced sensitivity and enabling high-throughput sample anal.  The authors successfully demonstrate the general applicability of the method by performing kinetic analyses of NNMT-mediated methylation for a range of pyridine-based substrates.  These findings also provide new insight into the diversity of substrate recognition by NNMT in a quant. manner.  In addn., the authors further established the suitability of the assay for the identification and characterization of small mol. inhibitors of NNMT.  To do so, the authors investigated the inhibition of NNMT by the nonspecific methyltransferase inhibitors, sinefungin and S-adenosyl-L-homocysteine, revealing IC50 values in the low micromolar range.  The results of these inhibition studies are particularly noteworthy as they will permit future efforts toward the development of new NNMT-specific inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqBC8U-yKb1brVg90H21EOLACvtfcHk0liliqCrF4COkg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVWntL7O&md5=0ebb58b665b91dd5983b2bef91886c10</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Facs.biochem.6b00733&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.biochem.6b00733%26sid%3Dliteratum%253Aachs%26aulast%3Dvan%2BHaren%26aufirst%3DM.%2BJ.%26aulast%3DSastre%2BTorano%26aufirst%3DJ.%26aulast%3DSartini%26aufirst%3DD.%26aulast%3DEmanuelli%26aufirst%3DM.%26aulast%3DParsons%26aufirst%3DR.%2BB.%26aulast%3DMartin%26aufirst%3DN.%2BI.%26atitle%3DA%2520rapid%2520and%2520efficient%2520assay%2520for%2520the%2520characterization%2520of%2520substrates%2520and%2520inhibitors%2520of%2520nicotinamide%2520N-methyltransferase%26jtitle%3DBiochemistry%26date%3D2016%26volume%3D55%26spage%3D5307%26epage%3D5315%26doi%3D10.1021%2Facs.biochem.6b00733" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Segel, I. H.</span></span> <i>Enzyme Kinetics</i>; <span class="NLM_publisher-name">Wiley</span>: <span class="NLM_publisher-loc">New York</span>, <span class="NLM_year">1975</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1975&author=I.+H.+Segel&title=Enzyme+Kinetics"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DSegel%26aufirst%3DI.%2BH.%26btitle%3DEnzyme%2520Kinetics%26pub%3DWiley%26date%3D1975" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R. A.</span></span> <i>Evaluation of Enzyme Inhibitors in Drug Discovery</i>, <span class="NLM_edition">2nd</span> ed.; <span class="NLM_publisher-name">John Wiley</span>: <span class="NLM_publisher-loc">New York</span>, <span class="NLM_year">2013</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.1002%2F9781118540398" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&author=R.+A.+Copeland&title=Evaluation+of+Enzyme+Inhibitors+in+Drug+Discovery"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1002%2F9781118540398&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F9781118540398%26sid%3Dliteratum%253Aachs%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26btitle%3DEvaluation%2520of%2520Enzyme%2520Inhibitors%2520in%2520Drug%2520Discovery%26pub%3DJohn%2520Wiley%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kuzmic, P.</span></span> <span> </span><span class="NLM_article-title">Program DynaFit for the analysis of enzyme kinetic data: application to HIV proteinase</span>. <i>Anal. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>237</i></span>,  <span class="NLM_fpage">260</span>– <span class="NLM_lpage">273</span>, <span class="refDoi"> DOI: 10.1006/abio.1996.0238</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.1006%2Fabio.1996.0238" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=8660575" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADyaK28XjtlOktL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=237&publication_year=1996&pages=260-273&author=P.+Kuzmic&title=Program+DynaFit+for+the+analysis+of+enzyme+kinetic+data%3A+application+to+HIV+proteinase&doi=10.1006%2Fabio.1996.0238"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Program DYNAFIT for the analysis of enzyme kinetic data: application to HIV proteinase</span></div><div class="casAuthors">Kuzmic, Petr</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Biochemistry</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">237</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">260-273</span>CODEN:
                <span class="NLM_cas:coden">ANBCA2</span>;
        ISSN:<span class="NLM_cas:issn">0003-2697</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">A computer program with the code name DYNAFIT was developed for fitting either the initial velocities or the time course of enzyme reactions to an arbitrary mol. mechanism represented symbolically by a set of chem. equations.  Seven numerical tests and five graphical tests are applied to judge the goodness of fit.  Exptl. data on the inhibition of the dissociative dimeric proteinase from HIV were used in four test examples.  A set of initial velocities was analyzed to see if a tight-binding inhibitor could bind to the HIV proteinase monomer.  Three different sets of progress curves were analyzed (i) to det. the kinetic properties of an irreversible inhibitor, (ii) to investigate the dissocn. and denaturation mechanism for the protease dimer, and (iii) to investigate the inhibition mechanism for a transient inhibitor.  The program is available by anonymous ftp via uwmml.pharmacy.wisc.edu and on the World Wide Web via http://uwmml.pharmacy.wisc. edu.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGa_hR-OebWLVg90H21EOLACvtfcHk0liCMwqw9CcZmQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XjtlOktL4%253D&md5=71b4b01a4d81d3e38490b1124fe836b4</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1006%2Fabio.1996.0238&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fabio.1996.0238%26sid%3Dliteratum%253Aachs%26aulast%3DKuzmic%26aufirst%3DP.%26atitle%3DProgram%2520DynaFit%2520for%2520the%2520analysis%2520of%2520enzyme%2520kinetic%2520data%253A%2520application%2520to%2520HIV%2520proteinase%26jtitle%3DAnal.%2520Biochem.%26date%3D1996%26volume%3D237%26spage%3D260%26epage%3D273%26doi%3D10.1006%2Fabio.1996.0238" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kuzmic, P.</span></span> <span> </span><span class="NLM_article-title">DynaFit - a software package for enzymology</span>. <i>Methods Enzymol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>467</i></span>,  <span class="NLM_fpage">247</span>– <span class="NLM_lpage">280</span>, <span class="refDoi"> DOI: 10.1016/S0076-6879(09)67010-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.1016%2FS0076-6879%2809%2967010-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=19897096" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmt12rsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=467&publication_year=2009&pages=247-280&author=P.+Kuzmic&title=DynaFit+-+a+software+package+for+enzymology&doi=10.1016%2FS0076-6879%2809%2967010-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">DynaFit - a software package for enzymology</span></div><div class="casAuthors">Kuzmic, Petr</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Enzymology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">467</span>
        (<span class="NLM_cas:issue">Computer Methods, Part B</span>),
    <span class="NLM_cas:pages">247-280</span>CODEN:
                <span class="NLM_cas:coden">MENZAU</span>;
        ISSN:<span class="NLM_cas:issn">0076-6879</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">A review.  Since its original publication, the DynaFit software package has been used in more than 500 published studies.  Most applications have been in biochem., esp. in enzyme kinetics.  This paper describes a no. of recently added features and capabilities, in the hope that the tool will continue to be useful to the enzymol. community.  Fully functional DynaFit continues to be freely available to all academic researchers from http://www.biokin.com.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGogZuk_IwpvgrVg90H21EOLACvtfcHk0liCMwqw9CcZmQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXmt12rsA%253D%253D&md5=36f49ffbcd28396eb6884631687cb592</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1016%2FS0076-6879%2809%2967010-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0076-6879%252809%252967010-5%26sid%3Dliteratum%253Aachs%26aulast%3DKuzmic%26aufirst%3DP.%26atitle%3DDynaFit%2520-%2520a%2520software%2520package%2520for%2520enzymology%26jtitle%3DMethods%2520Enzymol.%26date%3D2009%26volume%3D467%26spage%3D247%26epage%3D280%26doi%3D10.1016%2FS0076-6879%2809%2967010-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Williams, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morrison, J. F.</span></span> <span> </span><span class="NLM_article-title">The kinetics of reversible tight-binding inhibition</span>. <i>Methods Enzymol.</i> <span class="NLM_year" style="font-weight: bold;">1979</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">437</span>– <span class="NLM_lpage">467</span>, <span class="refDoi"> DOI: 10.1016/0076-6879(79)63019-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.1016%2F0076-6879%2879%2963019-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=502865" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADyaL3cXovF2ntw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=1979&pages=437-467&author=J.+W.+Williamsauthor=J.+F.+Morrison&title=The+kinetics+of+reversible+tight-binding+inhibition&doi=10.1016%2F0076-6879%2879%2963019-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">The kinetics of reversible tight-binding inhibition</span></div><div class="casAuthors">Williams, Jeffrey W.; Morrison, John F.</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Enzymology</span>
        (<span class="NLM_cas:date">1979</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">Enzyme Kinet. Mech., Part A</span>),
    <span class="NLM_cas:pages">437-67</span>CODEN:
                <span class="NLM_cas:coden">MENZAU</span>;
        ISSN:<span class="NLM_cas:issn">0076-6879</span>.
    </div><div class="casAbstract">In addn. to a general discussion of the kinetics of reversible tight-binding inhibitors, which exert their effect at a concn. comparable to that of the enzyme, and for which steady-state theory is used, exptl. results are given of the tight-binding inhibition of creatine kinase by Eu3+ (actually Eu-ADP) under steady-state conditions.  The inhibition of creatine kinase by fixed concns. of Eu3+ in the presence of varying concns. of Mg-ADP indicated that the inhibition is competitive, suggesting that Eu-ADP acts as a structural analog of Mg-ADP.  The inhibition was noncompetitive with respect to creatine phosphate.  The dissocn. consts. for binding of Eu-ADP to the free enzyme and the enzyme-phosphocreatine complex were 1.5 and 2.2 nM, resp., as detd. from Ki values.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2CwV3voXt27Vg90H21EOLACvtfcHk0liCMwqw9CcZmQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3cXovF2ntw%253D%253D&md5=a770f91b51661c9502d0aca44a24e1c7</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1016%2F0076-6879%2879%2963019-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0076-6879%252879%252963019-7%26sid%3Dliteratum%253Aachs%26aulast%3DWilliams%26aufirst%3DJ.%2BW.%26aulast%3DMorrison%26aufirst%3DJ.%2BF.%26atitle%3DThe%2520kinetics%2520of%2520reversible%2520tight-binding%2520inhibition%26jtitle%3DMethods%2520Enzymol.%26date%3D1979%26volume%3D63%26spage%3D437%26epage%3D467%26doi%3D10.1016%2F0076-6879%2879%2963019-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Neelakantan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vance, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wetzel, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McHardy, S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finnerty, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hommel, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watowich, S. J.</span></span> <span> </span><span class="NLM_article-title">Selective and membrane-permeable small molecule inhibitors of nicotinamide N-methyltransferase reverse high fat diet-induced obesity in mice</span>. <i>Biochem. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>147</i></span>,  <span class="NLM_fpage">141</span>– <span class="NLM_lpage">152</span>, <span class="refDoi"> DOI: 10.1016/j.bcp.2017.11.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.1016%2Fj.bcp.2017.11.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=29155147" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvVegu7vL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=147&publication_year=2018&pages=141-152&author=H.+Neelakantanauthor=V.+Vanceauthor=M.+D.+Wetzelauthor=H.+L.+Wangauthor=S.+F.+McHardyauthor=C.+C.+Finnertyauthor=J.+D.+Hommelauthor=S.+J.+Watowich&title=Selective+and+membrane-permeable+small+molecule+inhibitors+of+nicotinamide+N-methyltransferase+reverse+high+fat+diet-induced+obesity+in+mice&doi=10.1016%2Fj.bcp.2017.11.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Selective and membrane-permeable small molecule inhibitors of nicotinamide N-methyltransferase reverse high fat diet-induced obesity in mice</span></div><div class="casAuthors">Neelakantan, Harshini; Vance, Virginia; Wetzel, Michael D.; Wang, Hua-Yu Leo; McHardy, Stanton F.; Finnerty, Celeste C.; Hommel, Jonathan D.; Watowich, Stanley J.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Pharmacology (Amsterdam, Netherlands)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">147</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">141-152</span>CODEN:
                <span class="NLM_cas:coden">BCPCA6</span>;
        ISSN:<span class="NLM_cas:issn">0006-2952</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">There is a crit. need for new mechanism-of-action drugs that reduce the burden of obesity and assocd. chronic metabolic comorbidities.  A potentially novel target to treat obesity and type 2 diabetes is nicotinamide-N-methyltransferase (NNMT), a cytosolic enzyme with newly identified roles in cellular metab. and energy homeostasis.  To validate NNMT as an anti-obesity drug target, the authors investigated the permeability, selectivity, mechanistic, and physiol. properties of a series of small mol. NNMT inhibitors.  Membrane permeability of NNMT inhibitors was characterized using parallel artificial membrane permeability and Caco-2 cell assays.  Selectivity was tested against structurally-related methyltransferases and NAD (NAD+) salvage pathway enzymes.  Effects of NNMT inhibitors on lipogenesis and intracellular levels of metabolites, including NNMT reaction product 1-methylnicotianamide (1-MNA) were evaluated in cultured adipocytes.  Effects of a potent NNMT inhibitor on obesity measures and plasma lipid were assessed in diet-induced obese mice fed a high-fat diet.  Methylquinolinium scaffolds with primary amine substitutions displayed high permeability from passive and active transport across membranes.  Importantly, methylquinolinium analogs displayed high selectivity, not inhibiting related SAM-dependent methyltransferases or enzymes in the NAD+ salvage pathway.  NNMT inhibitors reduced intracellular 1-MNA, increased intracellular NAD+ and S-(5'-adenosyl)-L-methionine (SAM), and suppressed lipogenesis in adipocytes.  Treatment of diet-induced obese mice systemically with a potent NNMT inhibitor significantly reduced body wt. and white adipose mass, decreased adipocyte size, and lowered plasma total cholesterol levels.  Notably, administration of NNMT inhibitors did not impact total food intake nor produce any observable adverse effects.  These results support development of small mol. NNMT inhibitors as therapeutics to reverse diet-induced obesity and validate NNMT as a viable target to treat obesity and related metabolic conditions.  Increased flux of key cellular energy regulators, including NAD+ and SAM, may potentially define the therapeutic mechanism-of-action of NNMT inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0K0OY6S9HhbVg90H21EOLACvtfcHk0li-dbOpmG_DhQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvVegu7vL&md5=4a798e5f69cf9989b8d7a9d0584f872d</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1016%2Fj.bcp.2017.11.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bcp.2017.11.007%26sid%3Dliteratum%253Aachs%26aulast%3DNeelakantan%26aufirst%3DH.%26aulast%3DVance%26aufirst%3DV.%26aulast%3DWetzel%26aufirst%3DM.%2BD.%26aulast%3DWang%26aufirst%3DH.%2BL.%26aulast%3DMcHardy%26aufirst%3DS.%2BF.%26aulast%3DFinnerty%26aufirst%3DC.%2BC.%26aulast%3DHommel%26aufirst%3DJ.%2BD.%26aulast%3DWatowich%26aufirst%3DS.%2BJ.%26atitle%3DSelective%2520and%2520membrane-permeable%2520small%2520molecule%2520inhibitors%2520of%2520nicotinamide%2520N-methyltransferase%2520reverse%2520high%2520fat%2520diet-induced%2520obesity%2520in%2520mice%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D2018%26volume%3D147%26spage%3D141%26epage%3D152%26doi%3D10.1016%2Fj.bcp.2017.11.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Eisenberg, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwarz, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Komaromy, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wall, R.</span></span> <span> </span><span class="NLM_article-title">Analysis of membrane and surface protein sequences with the hydrophobic moment plot</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1984</span>,  <span class="NLM_volume"><i>179</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">125</span>– <span class="NLM_lpage">142</span>, <span class="refDoi"> DOI: 10.1016/0022-2836(84)90309-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.1016%2F0022-2836%2884%2990309-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=6502707" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADyaL2MXnsFajuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=179&publication_year=1984&pages=125-142&issue=1&author=D.+Eisenbergauthor=E.+Schwarzauthor=M.+Komaromyauthor=R.+Wall&title=Analysis+of+membrane+and+surface+protein+sequences+with+the+hydrophobic+moment+plot&doi=10.1016%2F0022-2836%2884%2990309-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Analysis of membrane and surface protein sequences with the hydrophobic moment plot</span></div><div class="casAuthors">Eisenberg, D.; Schwarz, E.; Komaromy, M.; Wall, R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1984</span>),
    <span class="NLM_cas:volume">179</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">125-42</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    </div><div class="casAbstract">An algorithm was developed which identifies α-helices involved in the interactions of membrane proteins with lipid bilayers and which distinguishes them from helices in sol. proteins.  The membrane-assocd. helices are then classified with the aid of the hydrophobic moment plot, on which the hydrophobic moment of each helix is ploted as a function of its hydrophobicity.  The magnitude of hydrophobic moment measures the amphiphilicity of the helix (and hence its tendency to seek a surface between hydrophobic and hydrophilic phases), and the hydrophobicity measures its affinity for the membrane interior.  Segments of membrane proteins in α-helices tend to fall in 1 of 3 regions of a hydrophobic moment plot: (1) monomeric transmembrane anchors (class I HLA transmembrane sequences) lie in the region of highest hydrophobicity and smallest hydrophobic moment; (2) helices presumed to be paired (such as the transmembrane M segments of surface Igs) and helices which are bundled together in membranes (such as bacteriorhodopsin) fall in the adjacent region with higher hydrophobic moment and smaller hydrophobicity; and (3) helices from surface-seeking proteins (such as melittin) fall in the region with still higher hydrophobic moment.  α-Helices from globular proteins mainly fall in a region of lower mean hydrophobicity and hydrophobic moment.  Application of these methods to the sequence of diphtheria toxin suggests 4 transmembrane helices and a surface-seeking helix in fragment B, the moiety known to have transmembrane function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkBWsePnQWqbVg90H21EOLACvtfcHk0li-dbOpmG_DhQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL2MXnsFajuw%253D%253D&md5=a2b8d36bbe0e1454bd6a0917fc0e7089</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1016%2F0022-2836%2884%2990309-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0022-2836%252884%252990309-7%26sid%3Dliteratum%253Aachs%26aulast%3DEisenberg%26aufirst%3DD.%26aulast%3DSchwarz%26aufirst%3DE.%26aulast%3DKomaromy%26aufirst%3DM.%26aulast%3DWall%26aufirst%3DR.%26atitle%3DAnalysis%2520of%2520membrane%2520and%2520surface%2520protein%2520sequences%2520with%2520the%2520hydrophobic%2520moment%2520plot%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1984%26volume%3D179%26issue%3D1%26spage%3D125%26epage%3D142%26doi%3D10.1016%2F0022-2836%2884%2990309-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Axelrod, J.</span></span> <span> </span><span class="NLM_article-title">Purification and properties of phenylethanolamine-N-methyl transferase</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1962</span>,  <span class="NLM_volume"><i>237</i></span>,  <span class="NLM_fpage">1657</span>– <span class="NLM_lpage">1660</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=13863458" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADyaF3sXkt1Sr" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=237&publication_year=1962&pages=1657-1660&author=J.+Axelrod&title=Purification+and+properties+of+phenylethanolamine-N-methyl+transferase"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Purification and properties of phenylethanolamine-N-methyl transferase</span></div><div class="casAuthors">Axelrod, Julius</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">1962</span>),
    <span class="NLM_cas:volume">237</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1657-60</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    </div><div class="casAbstract">The purification and properties of phenylethanolamine-N-methyl transferase are described.  This enzyme can N-methylate a variety of normally occurring and foreign phenylethanolamine derivs., but not phenethylamines.  It is highly localized in the adrenal medulla.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqILjUyyOtQq7Vg90H21EOLACvtfcHk0li-dbOpmG_DhQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaF3sXkt1Sr&md5=242704163c1ebda03d9ebe3738fdef3f</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DAxelrod%26aufirst%3DJ.%26atitle%3DPurification%2520and%2520properties%2520of%2520phenylethanolamine-N-methyl%2520transferase%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1962%26volume%3D237%26spage%3D1657%26epage%3D1660" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stratton, C. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poulin, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schramm, V. L.</span></span> <span> </span><span class="NLM_article-title">Kinetic isotope effects and transition state structure for human phenylethanolamine N-methyltransferase</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">342</span>– <span class="NLM_lpage">346</span>, <span class="refDoi"> DOI: 10.1021/acschembio.6b00922</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.6b00922" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BC28XitFWgt7fL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=342-346&issue=2&author=C.+F.+Strattonauthor=M.+B.+Poulinauthor=Q.+Duauthor=V.+L.+Schramm&title=Kinetic+isotope+effects+and+transition+state+structure+for+human+phenylethanolamine+N-methyltransferase&doi=10.1021%2Facschembio.6b00922"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Kinetic Isotope Effects and Transition State Structure for Human Phenylethanolamine N-Methyltransferase</span></div><div class="casAuthors">Stratton, Christopher F.; Poulin, Myles B.; Du, Quan; Schramm, Vern L.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">342-346</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Phenylethanolamine N-methyltransferase (PNMT) catalyzes the S-adenosyl-L-methionine (SAM)-dependent conversion of norepinephrine to epinephrine.  Epinephrine has been assocd. with crit. processes in humans including the control of respiration and blood pressure.  Addnl., PNMT activity has been suggested to play a role in hypertension and Alzheimer's disease.  Here, labeled SAM substrates were used to measure primary methyl-14C and 36S, and secondary methyl-3H, 5'-3H, 5'-14C intrinsic kinetic isotope effects for human PNMT.  The transition state of human PNMT was modeled by matching kinetic isotope effects predicted via quantum chem. calcns. to intrinsic values.  The model showed that human PNMT catalyzes the formation of epinephrine through an early SN2 transition state in which Me transfer is rate-limiting.  This anal. provides information on the geometry and electrostatics of the human PNMT transition state structure, details that enable the design of transition state analogs as potential PNMT inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrSBDuQMfV317Vg90H21EOLACvtfcHk0li-dbOpmG_DhQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitFWgt7fL&md5=cadf5bbe757ceba13081bd8e63a2f1d6</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1021%2Facschembio.6b00922&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.6b00922%26sid%3Dliteratum%253Aachs%26aulast%3DStratton%26aufirst%3DC.%2BF.%26aulast%3DPoulin%26aufirst%3DM.%2BB.%26aulast%3DDu%26aufirst%3DQ.%26aulast%3DSchramm%26aufirst%3DV.%2BL.%26atitle%3DKinetic%2520isotope%2520effects%2520and%2520transition%2520state%2520structure%2520for%2520human%2520phenylethanolamine%2520N-methyltransferase%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2017%26volume%3D12%26issue%3D2%26spage%3D342%26epage%3D346%26doi%3D10.1021%2Facschembio.6b00922" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fujioka, M.</span></span> <span> </span><span class="NLM_article-title">Mammalian small molecule methyltransferases: their structural and functional features</span>. <i>Int. J. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">1992</span>,  <span class="NLM_volume"><i>24</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">1917</span>– <span class="NLM_lpage">1924</span>, <span class="refDoi"> DOI: 10.1016/0020-711X(92)90287-B</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.1016%2F0020-711X%2892%2990287-B" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1473604" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADyaK3sXhsVais7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=1992&pages=1917-1924&issue=12&author=M.+Fujioka&title=Mammalian+small+molecule+methyltransferases%3A+their+structural+and+functional+features&doi=10.1016%2F0020-711X%2892%2990287-B"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Mammalian small molecule methyltransferases:  their structural and functional features</span></div><div class="casAuthors">Fujioka, Motoji</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Biochemistry</span>
        (<span class="NLM_cas:date">1992</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1917-24</span>CODEN:
                <span class="NLM_cas:coden">IJBOBV</span>;
        ISSN:<span class="NLM_cas:issn">0020-711X</span>.
    </div><div class="casAbstract">A review with 63 refs. on structural and functional features of mammalian S-adenosyl-methionine-dependent small mol. methyltransferases.  The methyltransferases have similar protomer mol. wts. in the range of 25,000-35,000.  Two common sequence motifs are found in all enzymes of known sequence.  Whereas the kinetic mechanisms may be different, the methyltransferases in the free form bind S-adenosylmethionine.  Most, if not all, of mammalian small mol. methyltransferases appear to have vicinal thiols in a catalytically important area of the enzyme.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqY96I4ugaO97Vg90H21EOLACvtfcHk0liiWyHKcY8S_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXhsVais7k%253D&md5=92c43009a5f71452dffde48cbd07530f</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1016%2F0020-711X%2892%2990287-B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0020-711X%252892%252990287-B%26sid%3Dliteratum%253Aachs%26aulast%3DFujioka%26aufirst%3DM.%26atitle%3DMammalian%2520small%2520molecule%2520methyltransferases%253A%2520their%2520structural%2520and%2520functional%2520features%26jtitle%3DInt.%2520J.%2520Biochem.%26date%3D1992%26volume%3D24%26issue%3D12%26spage%3D1917%26epage%3D1924%26doi%3D10.1016%2F0020-711X%2892%2990287-B" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Petrossian, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clarke, S. G.</span></span> <span> </span><span class="NLM_article-title">Uncovering the human methyltransferasome</span>. <i>Mol. Cell. Proteomics</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>10</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">M110.000976</span>, <span class="refDoi"> DOI: 10.1074/mcp.M110.000976</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.1074%2Fmcp.M110.000976" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=20930037" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2011&pages=M110.000976&issue=1&author=T.+C.+Petrossianauthor=S.+G.+Clarke&title=Uncovering+the+human+methyltransferasome&doi=10.1074%2Fmcp.M110.000976"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1074%2Fmcp.M110.000976&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fmcp.M110.000976%26sid%3Dliteratum%253Aachs%26aulast%3DPetrossian%26aufirst%3DT.%2BC.%26aulast%3DClarke%26aufirst%3DS.%2BG.%26atitle%3DUncovering%2520the%2520human%2520methyltransferasome%26jtitle%3DMol.%2520Cell.%2520Proteomics%26date%3D2011%26volume%3D10%26issue%3D1%26spage%3DM110.000976%26doi%3D10.1074%2Fmcp.M110.000976" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gerlt, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bouvier, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davidson, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imker, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sadkhin, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slater, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whalen, K. L.</span></span> <span> </span><span class="NLM_article-title">Enzyme function initiative-enzyme similarity tool (EFI-EST): a web tool for generating protein sequence similarity networks</span>. <i>Biochim. Biophys. Acta, Proteins Proteomics</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>1854</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">1019</span>– <span class="NLM_lpage">1037</span>, <span class="refDoi"> DOI: 10.1016/j.bbapap.2015.04.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.1016%2Fj.bbapap.2015.04.015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=25900361" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BC2MXnvFSisb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1854&publication_year=2015&pages=1019-1037&issue=8&author=J.+A.+Gerltauthor=J.+T.+Bouvierauthor=D.+B.+Davidsonauthor=H.+J.+Imkerauthor=B.+Sadkhinauthor=D.+R.+Slaterauthor=K.+L.+Whalen&title=Enzyme+function+initiative-enzyme+similarity+tool+%28EFI-EST%29%3A+a+web+tool+for+generating+protein+sequence+similarity+networks&doi=10.1016%2Fj.bbapap.2015.04.015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Enzyme function initiative-enzyme similarity tool (EFI-EST): A web tool for generating protein sequence similarity networks</span></div><div class="casAuthors">Gerlt, John A.; Bouvier, Jason T.; Davidson, Daniel B.; Imker, Heidi J.; Sadkhin, Boris; Slater, David R.; Whalen, Katie L.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Proteins and Proteomics</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">1854</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1019-1037</span>CODEN:
                <span class="NLM_cas:coden">BBAPBW</span>;
        ISSN:<span class="NLM_cas:issn">1570-9639</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B. V.</span>)
        </div><div class="casAbstract">A review.  The Enzyme Function Initiative, an NIH/NIGMS-supported Large-Scale Collaborative Project (EFI; U54GM093342; http://enzymefunction.org/), is focused on devising and disseminating bioinformatics and computational tools as well as exptl. strategies for the prediction and assignment of functions (in vitro activities and in vivo physiol./metabolic roles) to uncharacterized enzymes discovered in genome projects.  Protein sequence similarity networks (SSNs) are visually powerful tools for analyzing sequence relationships in protein families (H.J. Atkinson, J.H. Morris, T.E. Ferrin, and P.C. Babbitt, PLoS One 2009, 4, e4345).  However, the members of the biol./biomedical community have not had access to the capability to generate SSNs for their "favorite" protein families.  In this article we announce the EFI-EST (Enzyme Function Initiative-Enzyme Similarity Tool) web tool (http://efi.igb.illinois.edu/efi-est/) that is available without cost for the automated generation of SSNs by the community.  The tool can create SSNs for the "closest neighbors" of a user-supplied protein sequence from the UniProt database (Option A) or of members of any user-supplied Pfam and/or InterPro family (Option B).  We provide an introduction to SSNs, a description of EFI-EST, and a demonstration of the use of EFI-EST to explore sequence-function space in the OMP decarboxylase superfamily (PF00215).  This article is designed as a tutorial that will allow members of the community to use the EFI-EST web tool for exploring sequence/function space in protein families.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqfzrew87TmDLVg90H21EOLACvtfcHk0liiWyHKcY8S_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXnvFSisb0%253D&md5=4897a23daaf5c2437acb0bed815095f8</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1016%2Fj.bbapap.2015.04.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbapap.2015.04.015%26sid%3Dliteratum%253Aachs%26aulast%3DGerlt%26aufirst%3DJ.%2BA.%26aulast%3DBouvier%26aufirst%3DJ.%2BT.%26aulast%3DDavidson%26aufirst%3DD.%2BB.%26aulast%3DImker%26aufirst%3DH.%2BJ.%26aulast%3DSadkhin%26aufirst%3DB.%26aulast%3DSlater%26aufirst%3DD.%2BR.%26aulast%3DWhalen%26aufirst%3DK.%2BL.%26atitle%3DEnzyme%2520function%2520initiative-enzyme%2520similarity%2520tool%2520%2528EFI-EST%2529%253A%2520a%2520web%2520tool%2520for%2520generating%2520protein%2520sequence%2520similarity%2520networks%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Proteins%2520Proteomics%26date%3D2015%26volume%3D1854%26issue%3D8%26spage%3D1019%26epage%3D1037%26doi%3D10.1016%2Fj.bbapap.2015.04.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zallot, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oberg, N. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerlt, J. A.</span></span> <span> </span><span class="NLM_article-title">’Democratized’ genomic enzymology web tools for functional assignment</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">77</span>– <span class="NLM_lpage">85</span>, <span class="refDoi"> DOI: 10.1016/j.cbpa.2018.09.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.1016%2Fj.cbpa.2018.09.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=30268904" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvVaht7zK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2018&pages=77-85&author=R.+Zallotauthor=N.+O.+Obergauthor=J.+A.+Gerlt&title=%E2%80%99Democratized%E2%80%99+genomic+enzymology+web+tools+for+functional+assignment&doi=10.1016%2Fj.cbpa.2018.09.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">'Democratized' genomic enzymology web tools for functional assignment</span></div><div class="casAuthors">Zallot, Remi; Oberg, Nils O.; Gerlt, John A.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">77-85</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The protein databases contain an exponentially growing no. of sequences as a result of the recent increase in ease and decrease in cost of genome sequencing.  The rate of data accumulation far exceeds the rate of functional studies, producing an increase in genomic 'dark matter', sequences for which no precise and validated function is defined.  Publicly accessible, that is 'democratized,' genomic enzymol. web tools are essential to leverage the protein and genome databases for discovery of the in vitro activities and in vivo functions of novel enzymes and proteins belonging to the dark matter.  In this review, we discuss the use of web tools that have proven successful for functional assignment.  We also describe a mechanism for ensuring the capture of published functional data so that the quality of both curated and automated annotations transfer can be improved.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojSOg1ygeEjLVg90H21EOLACvtfcHk0liiWyHKcY8S_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvVaht7zK&md5=67f7e0230fb75906c97a782db04e4398</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2018.09.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2018.09.009%26sid%3Dliteratum%253Aachs%26aulast%3DZallot%26aufirst%3DR.%26aulast%3DOberg%26aufirst%3DN.%2BO.%26aulast%3DGerlt%26aufirst%3DJ.%2BA.%26atitle%3D%25E2%2580%2599Democratized%25E2%2580%2599%2520genomic%2520enzymology%2520web%2520tools%2520for%2520functional%2520assignment%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2018%26volume%3D47%26spage%3D77%26epage%3D85%26doi%3D10.1016%2Fj.cbpa.2018.09.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinshilboum, R. M.</span></span> <span> </span><span class="NLM_article-title">Rabbit lung indolethylamine N-methyltransferase. cDNA and gene cloning and characterization</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>273</i></span> (<span class="NLM_issue">51</span>),  <span class="NLM_fpage">34502</span>– <span class="NLM_lpage">34510</span>, <span class="refDoi"> DOI: 10.1074/jbc.273.51.34502</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.1074%2Fjbc.273.51.34502" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=9852119" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADyaK1MXntleq" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=273&publication_year=1998&pages=34502-34510&issue=51&author=M.+A.+Thompsonauthor=R.+M.+Weinshilboum&title=Rabbit+lung+indolethylamine+N-methyltransferase.+cDNA+and+gene+cloning+and+characterization&doi=10.1074%2Fjbc.273.51.34502"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Rabbit lung indolethylamine N-methyltransferase cDNA and gene cloning and characterization</span></div><div class="casAuthors">Thompson, Michael A.; Weinshilboum, Richard M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">273</span>
        (<span class="NLM_cas:issue">51</span>),
    <span class="NLM_cas:pages">34502-34510</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Indolethylamine N-methyltransferase (INMT) catalyzes the N-methylation of tryptamine and structurally related compds.  This reaction has been studied because of its possible role in the in vivo synthesis of psychoactive compds. or neurotoxins and has been characterized biochem. in prepns. of rabbit lung.  Therefore, the authors set out to purify rabbit lung INMT, to clone and express its cDNA, and to clone and structurally characterize its gene as steps toward understanding the function and regulation of this enzyme.  Rabbit lung INMT was purified and partial amino acid sequence was obtained.  A polymerase chain reaction-based approach was then used to clone a rabbit lung INMT cDNA with a 792-base pair open reading frame that encoded a 263-amino acid protein with a predicted mol. mass of 29 kDa.  When the cDNA was expressed in COS-1 cells, the encoded protein catalyzed the methylation of tryptamine and structurally related compds., and was inhibited by two products of the reaction, S-adenosyl-L-homocysteine (AdoHcy) and N,N-dimethyltryptamine, as well as antimigraine drugs that are structurally related to N,N-dimethyltryptamine.  Northern blot anal. demonstrated the presence of 2.0-kilobase mRNA species in rabbit lung, liver and, at lower levels, in brain.  The cDNA was then used to clone the rabbit INMT gene.  That gene had three exons and was structurally similar to the genes for nicotinamide N-methyltransferase and phenylethanolamine N-methyltransferase in several species.  Cloning and expression of a rabbit lung INMT cDNA and cloning of the rabbit INMT gene represent important steps toward detn. of the function and regulation of this mammalian methyltransferase enzyme.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXgOW945aeZ7Vg90H21EOLACvtfcHk0lhkz04RA1Z7NQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXntleq&md5=fa4393a632c0fbc1c52d6f300cb5b463</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1074%2Fjbc.273.51.34502&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.273.51.34502%26sid%3Dliteratum%253Aachs%26aulast%3DThompson%26aufirst%3DM.%2BA.%26aulast%3DWeinshilboum%26aufirst%3DR.%2BM.%26atitle%3DRabbit%2520lung%2520indolethylamine%2520N-methyltransferase.%2520cDNA%2520and%2520gene%2520cloning%2520and%2520characterization%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1998%26volume%3D273%26issue%3D51%26spage%3D34502%26epage%3D34510%26doi%3D10.1074%2Fjbc.273.51.34502" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Holm, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laakso, L. M.</span></span> <span> </span><span class="NLM_article-title">Dali server update</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>44</i></span> (<span class="NLM_issue">W1</span>),  <span class="NLM_fpage">W351</span>– <span class="NLM_lpage">W355</span>, <span class="refDoi"> DOI: 10.1093/nar/gkw357</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.1093%2Fnar%2Fgkw357" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=27131377" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtV2itrrO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=W351-W355&issue=W1&author=L.+Holmauthor=L.+M.+Laakso&title=Dali+server+update&doi=10.1093%2Fnar%2Fgkw357"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Dali server update</span></div><div class="casAuthors">Holm, Liisa; Laakso, Laura M.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">W1</span>),
    <span class="NLM_cas:pages">W351-W355</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The Dali server is a network service for comparing protein structures in 3D.  In favorable cases, comparing 3D structures may reveal biol. interesting similarities that are not detectable by comparing sequences.  The Dali server has been running in various places for over 20 years and is used routinely by crystallographers on newly solved structures.  The latest update of the server provides enhanced analytics for the study of sequence and structure conservation.  The server performs three types of structure comparisons: (i) Protein Data Bank (PDB) search compares one query structure against those in the PDB and returns a list of similar structures; (ii) pairwise comparison compares one query structure against a list of structures specified by the user; and (iii) all against all structure comparison returns a structural similarity matrix, a dendrogram and a multidimensional scaling projection of a set of structures specified by the user.  Structural superimpositions are visualized using the Java-free WebGL viewer PV.  The structural alignment view is enhanced by sequence similarity searches against Uniprot.  The combined structure-sequence alignment information is compressed to a stack of aligned sequence logos.  In the stack, each structure is structurally aligned to the query protein and represented by a sequence logo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo9jpe1HGTYrLVg90H21EOLACvtfcHk0lhkz04RA1Z7NQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtV2itrrO&md5=cd2b5fabf6a01a4b35f350342ae7ddc2</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkw357&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkw357%26sid%3Dliteratum%253Aachs%26aulast%3DHolm%26aufirst%3DL.%26aulast%3DLaakso%26aufirst%3DL.%2BM.%26atitle%3DDali%2520server%2520update%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2016%26volume%3D44%26issue%3DW1%26spage%3DW351%26epage%3DW355%26doi%3D10.1093%2Fnar%2Fgkw357" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Peng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilk, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adjei, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salavaggione, O. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinshilboum, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yee, V. C.</span></span> <span> </span><span class="NLM_article-title">Structural basis of substrate recognition in thiopurine S-methyltransferase</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>47</i></span> (<span class="NLM_issue">23</span>),  <span class="NLM_fpage">6216</span>– <span class="NLM_lpage">6225</span>, <span class="refDoi"> DOI: 10.1021/bi800102x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi800102x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BD1cXlvFOgsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2008&pages=6216-6225&issue=23&author=Y.+Pengauthor=Q.+Fengauthor=D.+Wilkauthor=A.+A.+Adjeiauthor=O.+E.+Salavaggioneauthor=R.+M.+Weinshilboumauthor=V.+C.+Yee&title=Structural+basis+of+substrate+recognition+in+thiopurine+S-methyltransferase&doi=10.1021%2Fbi800102x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis of substrate recognition in thiopurine S-methyltransferase</span></div><div class="casAuthors">Peng, Yi; Feng, Qiping; Wilk, Dennis; Adjei, Araba A.; Salavaggione, Oreste E.; Weinshilboum, Richard M.; Yee, Vivien C.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">6216-6225</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Thiopurine S-methyltransferase (I) modulates the cytotoxic effects of thiopurine prodrugs such as 6-mercaptopurine by methylating them in a reaction using S-adenosyl-L-methionine as donor.  Patients with I variant allozymes exhibit diminished levels of protein and/or enzyme activity and are at risk for thiopurine drug-induced toxicity.  Here, the authors detd. 2 crystal structures of murine I, one as a binary complex with the product, S-adenosyl-L-homocysteine (II), and the other as a ternary complex with II and the substrate, 6-mercaptopurine (III), to 1.8 and 2.0 Å resoln., resp.  Comparison of the structures revealed that an active site loop became ordered upon III binding.  The positions of the 2 ligands were consistent with the expected SN2 reaction mechanism.  Amino acid residues Arg-147 and Arg-221, the only polar amino acids near III, were highlighted as possible participants in substrate deprotonation.  To probe whether these residues were important for catalysis, point mutants were prepd. in human I.  Substitution of Arg-152 (Arg-147 in murine I) with Glu decreased the Vmax and increased the Km for III but not the Km for S-adenosyl-L-methionine.  Substitution at this position with Ala or His residues and similar substitutions of Arg-226 (Arg-221 in murine I) resulted in no effect on enzyme activity.  The double mutant R152A/R226A exhibited a decreased Vmax and increased Km for III.  These observations suggested that either Arg-152 or Arg-226 may participate in some fashion in the I reaction, with one residue compensating when the other is altered, and that Arg-152 may interact with the substrate more directly than Arg-226, consistent with observations in the murine I crystal structure.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqzr1Kpnl7sV7Vg90H21EOLACvtfcHk0lhkz04RA1Z7NQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXlvFOgsro%253D&md5=f1fc23f59a78a0a7b0ac79e611dc6df4</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1021%2Fbi800102x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi800102x%26sid%3Dliteratum%253Aachs%26aulast%3DPeng%26aufirst%3DY.%26aulast%3DFeng%26aufirst%3DQ.%26aulast%3DWilk%26aufirst%3DD.%26aulast%3DAdjei%26aufirst%3DA.%2BA.%26aulast%3DSalavaggione%26aufirst%3DO.%2BE.%26aulast%3DWeinshilboum%26aufirst%3DR.%2BM.%26aulast%3DYee%26aufirst%3DV.%2BC.%26atitle%3DStructural%2520basis%2520of%2520substrate%2520recognition%2520in%2520thiopurine%2520S-methyltransferase%26jtitle%3DBiochemistry%26date%3D2008%26volume%3D47%26issue%3D23%26spage%3D6216%26epage%3D6225%26doi%3D10.1021%2Fbi800102x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ames, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Selassie, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woodson, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Loon, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansch, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinshilboum, R. M.</span></span> <span> </span><span class="NLM_article-title">Thiopurine methyltransferase: structure-activity relationships for benzoic acid inhibitors and thiophenol substrates</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1986</span>,  <span class="NLM_volume"><i>29</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">354</span>– <span class="NLM_lpage">358</span>, <span class="refDoi"> DOI: 10.1021/jm00153a009</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm00153a009" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADyaL28XosVamug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=1986&pages=354-358&issue=3&author=M.+M.+Amesauthor=C.+D.+Selassieauthor=L.+C.+Woodsonauthor=J.+A.+Van+Loonauthor=C.+Hanschauthor=R.+M.+Weinshilboum&title=Thiopurine+methyltransferase%3A+structure-activity+relationships+for+benzoic+acid+inhibitors+and+thiophenol+substrates&doi=10.1021%2Fjm00153a009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Thiopurine methyltransferase:  structure-activity relationships for benzoic acid inhibitors and thiophenol substrates</span></div><div class="casAuthors">Ames, Matthew M.; Selassie, Cynthia Dias; Woodson, Lee C.; Van Loon, Jon A.; Hansch, Corwin; Weinshilboum, Richard M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1986</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">354-8</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    </div><div class="casAbstract">Twenty-seven substituted benzoic acids were studied as inhibitors of partially purified human renal thiopurine methyltransferase (TPMT).  QSAR anal. resulted in the following equation: pI50 = 1.25π'3 + 0.73MR3,4 + 2.92 (where pI50 is the -log of the concn. of compd. that inhibits TPMT activity by 50% (IC50), π'3 is the relative hydrophobicity of the more hydrophobic of the 2 meta substituents, and MR3,4 is the molar refractivity of the more hydrophobic of the 2 meta substituents and of the para substituent on the Ph ring).  In addn., 14 substituted thiophenols were tested as substrates for the enzyme.  All 14 thiophenols tested were excellent substrates with Km values (0.8-7.8 μM) that were ≥2 orders of magnitude lower than those of any known thiopurine substrate for TPMT.  However, there was no discernible relation between the activities of thiophenol substrate and their physicochem. parameters.  Apparently, benzoic acid inhibitors of and thiophenol substrates for TPMT may interact with different sites on the enzyme.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhvqO1u1HSFLVg90H21EOLACvtfcHk0lissre8CgozkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL28XosVamug%253D%253D&md5=1f89c18b92d03a2696c3393bb91224da</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1021%2Fjm00153a009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm00153a009%26sid%3Dliteratum%253Aachs%26aulast%3DAmes%26aufirst%3DM.%2BM.%26aulast%3DSelassie%26aufirst%3DC.%2BD.%26aulast%3DWoodson%26aufirst%3DL.%2BC.%26aulast%3DVan%2BLoon%26aufirst%3DJ.%2BA.%26aulast%3DHansch%26aufirst%3DC.%26aulast%3DWeinshilboum%26aufirst%3DR.%2BM.%26atitle%3DThiopurine%2520methyltransferase%253A%2520structure-activity%2520relationships%2520for%2520benzoic%2520acid%2520inhibitors%2520and%2520thiophenol%2520substrates%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1986%26volume%3D29%26issue%3D3%26spage%3D354%26epage%3D358%26doi%3D10.1021%2Fjm00153a009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Martinez
Molina, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nordlund, P.</span></span> <span> </span><span class="NLM_article-title">The cellular thermal shift assay: a novel biophysical assay for in situ drug target engagement and mechanistic biomarker studies</span>. <i>Annu. Rev. Pharmacol. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>56</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">141</span>– <span class="NLM_lpage">161</span>, <span class="refDoi"> DOI: 10.1146/annurev-pharmtox-010715-103715</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.1146%2Fannurev-pharmtox-010715-103715" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=26566155" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVCqs7nI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2016&pages=141-161&issue=1&author=D.+Martinez%0AMolinaauthor=P.+Nordlund&title=The+cellular+thermal+shift+assay%3A+a+novel+biophysical+assay+for+in+situ+drug+target+engagement+and+mechanistic+biomarker+studies&doi=10.1146%2Fannurev-pharmtox-010715-103715"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">The Cellular Thermal Shift Assay: A Novel Biophysical Assay for In Situ Drug Target Engagement and Mechanistic Biomarker Studies</span></div><div class="casAuthors">Martinez Molina, Daniel; Nordlund, Paer</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Pharmacology and Toxicology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">141-161</span>CODEN:
                <span class="NLM_cas:coden">ARPTDI</span>;
        ISSN:<span class="NLM_cas:issn">0362-1642</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews</span>)
        </div><div class="casAbstract">A drug must engage its intended target to achieve its therapeutic effect.  However, conclusively measuring target engagement (TE) in situ is challenging.  This complicates preclin. development and is considered a key factor in the high rate of attrition in clin. trials.  Here, we discuss a recently developed, label-free, biophys. assay, the cellular thermal shift assay (CETSA), which facilitates the direct assessment of TE in cells and tissues at various stages of drug development.  CETSA also reveals biochem. events downstream of drug binding and therefore provides a promising means of establishing mechanistic biomarkers.  The implementation of proteome-wide CETSA using quant. mass spectrometry represents a novel strategy for defining off-target toxicity and polypharmacol. and for identifying downstream mechanistic biomarkers.  The first year of CETSA applications in the literature has focused on TE studies in cell culture systems and has confirmed the broad applicability of CETSA to many different target families.  The next phase of CETSA applications will likely encompass comprehensive animal and patient studies, and CETSA will likely serve as a very valuable tool in many stages of preclin. and clin. drug development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCOdmURGPcTbVg90H21EOLACvtfcHk0lissre8CgozkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVCqs7nI&md5=092ac50d54d92ef9746a90af8c634e94</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1146%2Fannurev-pharmtox-010715-103715&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-pharmtox-010715-103715%26sid%3Dliteratum%253Aachs%26aulast%3DMartinez%2BMolina%26aufirst%3DD.%26aulast%3DNordlund%26aufirst%3DP.%26atitle%3DThe%2520cellular%2520thermal%2520shift%2520assay%253A%2520a%2520novel%2520biophysical%2520assay%2520for%2520in%2520situ%2520drug%2520target%2520engagement%2520and%2520mechanistic%2520biomarker%2520studies%26jtitle%3DAnnu.%2520Rev.%2520Pharmacol.%2520Toxicol.%26date%3D2016%26volume%3D56%26issue%3D1%26spage%3D141%26epage%3D161%26doi%3D10.1146%2Fannurev-pharmtox-010715-103715" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kung, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malinoski, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prakash, G. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span> <span> </span><span class="NLM_article-title">Development of alkyne-containing pyrazolopyrimidines to overcome drug resistance of Bcr-Abl kinase</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span> (<span class="NLM_issue">23</span>),  <span class="NLM_fpage">9228</span>– <span class="NLM_lpage">9237</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01125</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01125" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVaht7vL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=9228-9237&issue=23&author=X.+Liuauthor=A.+Kungauthor=B.+Malinoskiauthor=G.+K.+Prakashauthor=C.+Zhang&title=Development+of+alkyne-containing+pyrazolopyrimidines+to+overcome+drug+resistance+of+Bcr-Abl+kinase&doi=10.1021%2Facs.jmedchem.5b01125"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Development of Alkyne-Containing Pyrazolopyrimidines To Overcome Drug Resistance of Bcr-Abl Kinase</span></div><div class="casAuthors">Liu, Xu; Kung, Alvin; Malinoski, Brock; Prakash, G. K. Surya; Zhang, Chao</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">9228-9237</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Despite the success of imatinib at inhibiting Bcr-Abl and treating chronic myelogenous leukemia (CML), resistance to the therapy occurs over time in patients.  In particular, the resistance to imatinib caused by the gatekeeper mutation T315I in Bcr-Abl remains a challenge in the clinic.  Inspired by the successful development of ponatinib to curb drug resistance, we hypothesize that the incorporation of an alkyne linker in other heterocyclic scaffolds can also achieve potent inhibition of Bcr-AblT315I by allowing for simultaneous occupancy of both the active site and the allosteric pocket in the Abl kinase domain.  Herein, we describe the design, synthesis, and characterization of a series of alkyne-contg. pyrazolopyrimidines as Bcr-Abl inhibitors.  Our results demonstrate that some alkyne-contg. pyrazolopyrimidines potently inhibit not only AblT315I in vitro but also Bcr-AblT315I in cells.  These pyrazolopyrimidines can serve as lead compds. for future development of novel targeted therapy to overcome drug resistance of CML.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvCwLSMJhHxLVg90H21EOLACvtfcHk0lissre8CgozkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVaht7vL&md5=adfdddc095b76175f04f8ace9f51b847</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01125&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01125%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DKung%26aufirst%3DA.%26aulast%3DMalinoski%26aufirst%3DB.%26aulast%3DPrakash%26aufirst%3DG.%2BK.%26aulast%3DZhang%26aufirst%3DC.%26atitle%3DDevelopment%2520of%2520alkyne-containing%2520pyrazolopyrimidines%2520to%2520overcome%2520drug%2520resistance%2520of%2520Bcr-Abl%2520kinase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26issue%3D23%26spage%3D9228%26epage%3D9237%26doi%3D10.1021%2Facs.jmedchem.5b01125" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Srinivasan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ng, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalesh, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, S. Q.</span></span> <span> </span><span class="NLM_article-title">Methods of using click chemistry in the discovery of enzyme inhibitors</span>. <i>Nat. Protoc.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>2</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">2655</span>– <span class="NLM_lpage">2664</span>, <span class="refDoi"> DOI: 10.1038/nprot.2007.323</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.1038%2Fnprot.2007.323" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=18007601" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlSlsrrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2007&pages=2655-2664&issue=11&author=R.+Srinivasanauthor=J.+Liauthor=S.+L.+Ngauthor=K.+A.+Kaleshauthor=S.+Q.+Yao&title=Methods+of+using+click+chemistry+in+the+discovery+of+enzyme+inhibitors&doi=10.1038%2Fnprot.2007.323"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Methods of using click chemistry in the discovery of enzyme inhibitors</span></div><div class="casAuthors">Srinivasan, Rajavel; Li, Junqi; Ng, Su Ling; Kalesh, Karunakaran A.; Yao, Shao Q.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2655-2664</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">This protocol describes the step-by-step procedures for the efficient assembly of bidentate inhibitor libraries of a target enzyme, using the so-called 'click chem.' between an alkyne-bearing core group and an azide-modified peripheral group, followed by direct biol. screening for the identification of potential 'hits'.  The reaction is highlighted by its modularity, high efficiency (∼100% yield in most cases) and tolerance toward many functional groups present in the fragments, as well as biocompatibility (typically carried out in aq. conditions with small amts. of biocompatible catalysts).  The approach consists of three steps: (i) chem. synthesis of alkyne-bearing protein tyrosine phosphatase or matrix metalloprotease core groups and diverse azide-modified peripheral groups; (ii) click chem. to assemble the bidentate inhibitor libraries; and (iii) direct screening of the libraries with target enzymes using 384-well microplate assays.  Following the chem. synthesis of the core and peripheral groups and optimization of the click chem. conditions (∼1 wk), steps (ii) and (iii) take 3 d to complete (∼1-2 d for library assembly and 1 d for inhibitor screening).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvVU_E7kmG4rVg90H21EOLACvtfcHk0lissre8CgozkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlSlsrrJ&md5=21306eca61966c0d8d1bbb504a17fdf4</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2007.323&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2007.323%26sid%3Dliteratum%253Aachs%26aulast%3DSrinivasan%26aufirst%3DR.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DNg%26aufirst%3DS.%2BL.%26aulast%3DKalesh%26aufirst%3DK.%2BA.%26aulast%3DYao%26aufirst%3DS.%2BQ.%26atitle%3DMethods%2520of%2520using%2520click%2520chemistry%2520in%2520the%2520discovery%2520of%2520enzyme%2520inhibitors%26jtitle%3DNat.%2520Protoc.%26date%3D2007%26volume%3D2%26issue%3D11%26spage%3D2655%26epage%3D2664%26doi%3D10.1038%2Fnprot.2007.323" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Burchat, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chong, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nielsen, N.</span></span> <span> </span><span class="NLM_article-title">Titration of alkyllithiums with a simple reagent to a blue endpoint</span>. <i>J. Organomet. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>542</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">281</span>– <span class="NLM_lpage">283</span>, <span class="refDoi"> DOI: 10.1016/S0022-328X(97)00143-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.1016%2FS0022-328X%2897%2900143-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADyaK2sXmt1Glurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=542&publication_year=1997&pages=281-283&issue=2&author=A.+F.+Burchatauthor=J.+M.+Chongauthor=N.+Nielsen&title=Titration+of+alkyllithiums+with+a+simple+reagent+to+a+blue+endpoint&doi=10.1016%2FS0022-328X%2897%2900143-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Titration of alkyllithiums with a simple reagent to a blue endpoint</span></div><div class="casAuthors">Burchat, Andrew F.; Chong, J. Michael; Nielsen, Nathan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organometallic Chemistry</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">542</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">281-283</span>CODEN:
                <span class="NLM_cas:coden">JORCAI</span>;
        ISSN:<span class="NLM_cas:issn">0022-328X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Reaction of readily available N-benzylbenzamide (1) with a slight excess of BuLi produces a deep blue dianion.  Under suitable conditions (THF, -40°), 1 may be used as a stoichiometric reagent to titrate BuLi, s-BuLi, and t-BuLi with good precision.  Less reactive alkyllithiums (MeLi and MeLi·LiBr), aryllithiums, and LDA may be titrated at higher temps. (-20 to 0°).  Titers are comparable to those obtained using the Gilman double titrn. method, and there appears to be no interference from alkoxides.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrY29vXPn-z0LVg90H21EOLACvtfcHk0ljedSjtCSzgKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXmt1Glurg%253D&md5=424c3be5372d4dea12d9d25d55fc778e</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1016%2FS0022-328X%2897%2900143-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0022-328X%252897%252900143-5%26sid%3Dliteratum%253Aachs%26aulast%3DBurchat%26aufirst%3DA.%2BF.%26aulast%3DChong%26aufirst%3DJ.%2BM.%26aulast%3DNielsen%26aufirst%3DN.%26atitle%3DTitration%2520of%2520alkyllithiums%2520with%2520a%2520simple%2520reagent%2520to%2520a%2520blue%2520endpoint%26jtitle%3DJ.%2520Organomet.%2520Chem.%26date%3D1997%26volume%3D542%26issue%3D2%26spage%3D281%26epage%3D283%26doi%3D10.1016%2FS0022-328X%2897%2900143-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Suffert, J.</span></span> <span> </span><span class="NLM_article-title">Simple direct titration of organolithium reagents using N-pivaloyl-<i>o</i>-toluidine and/or N-pivaloyl-<i>o</i>-benzylaniline</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1989</span>,  <span class="NLM_volume"><i>54</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">509</span>– <span class="NLM_lpage">510</span>, <span class="refDoi"> DOI: 10.1021/jo00263a052</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo00263a052" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADyaL1MXhtFCns74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=1989&pages=509-510&issue=2&author=J.+Suffert&title=Simple+direct+titration+of+organolithium+reagents+using+N-pivaloyl-o-toluidine+and%2For+N-pivaloyl-o-benzylaniline&doi=10.1021%2Fjo00263a052"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Simple direct titration of organolithium reagents using N-pivaloyl-o-toluidine and/or N-pivaloyl-o-benzylaniline</span></div><div class="casAuthors">Suffert, Jean</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">1989</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">509-10</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    </div><div class="casAbstract">Titrn. of organolithium reagents using N-pivaloyl o-toluidine (I) and/or N-pivaloyl o-benzylaniline (II) is described.  Compds. I and II are easily accessible, cryst., non hygroscopic, stable and may be stored without any precautions.  Comparative titrn. of organolithium reagents with four different methods of titrn. showed good agreement with these commonly used anal. procedures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_1HIyqf6jSLVg90H21EOLACvtfcHk0ljedSjtCSzgKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1MXhtFCns74%253D&md5=2c5f9c9286086e2bdfa9f6dbd5e03bc0</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1021%2Fjo00263a052&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo00263a052%26sid%3Dliteratum%253Aachs%26aulast%3DSuffert%26aufirst%3DJ.%26atitle%3DSimple%2520direct%2520titration%2520of%2520organolithium%2520reagents%2520using%2520N-pivaloyl-o-toluidine%2520and%252For%2520N-pivaloyl-o-benzylaniline%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D1989%26volume%3D54%26issue%3D2%26spage%3D509%26epage%3D510%26doi%3D10.1021%2Fjo00263a052" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Werner, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poelsma, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Windmüller, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barth, D.</span></span> <span> </span><span class="NLM_article-title">Synthesis and reactivity of bis(ethene) rhodium(I) and iridium(I) carboxylato complexes</span>. <i>Chem. Ber.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>129</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">647</span>– <span class="NLM_lpage">652</span>, <span class="refDoi"> DOI: 10.1002/cber.19961290609</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.1002%2Fcber.19961290609" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADyaK28XjtlWntLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=129&publication_year=1996&pages=647-652&issue=6&author=H.+Wernerauthor=S.+Poelsmaauthor=M.+E.+Schneiderauthor=B.+Windm%C3%BCllerauthor=D.+Barth&title=Synthesis+and+reactivity+of+bis%28ethene%29+rhodium%28I%29+and+iridium%28I%29+carboxylato+complexes&doi=10.1002%2Fcber.19961290609"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and reactivity of bis(ethene) rhodium(I) and iridium(I) carboxylato complexes</span></div><div class="casAuthors">Werner, Helmut; Poelsma, Sybo; Schneider, Michael E.; Windmueller, Bettina; Barth, Dietmar</div><div class="citationInfo"><span class="NLM_cas:title">Chemische Berichte</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">129</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">647-652</span>CODEN:
                <span class="NLM_cas:coden">CHBEAM</span>;
        ISSN:<span class="NLM_cas:issn">0009-2940</span>.
    
            (<span class="NLM_cas:orgname">VCH</span>)
        </div><div class="casAbstract">[Rh(μ-OAc)(C2H4)2]2 (I) and analogous carboxylatobis(olefin)rhodium(I) complexes [Rh(μ-O2CR)(olefin)2]2 (R = Me, CF3, Ph) were almost quant. prepd. under heterogeneous conditions from [RhCl(C8H14)2]2 or [RhCl(C2H4)2]2 and AcONa.3H2O or RCO2H/NaOH, resp.  The x-ray crystal structure anal. of I confirmed the bridging position of the acetato ligands.  The synthesis of [Ir(μ-OAc)(C2H4)2]2 (II) was carried out similarly by using [IrCl(C2H4)2]2 and AcONa.3H2O as starting materials.  The monoethene complex trans-[Ir(η1-OAc)(C2H4)(PiPr3)2] was obtained from II and PiPr3.  It reacted with H2 and HC≡CPh by oxidative addn. to give [IrH2(η2-OAc)(PiPr3)2] (III) and [IrH(C≡CPh)(η2-OAc)(PiPr3)2], resp.  Compd. III was also prepd. from [IrH2Cl(PiPr3)2] and AcONa.3H2O.  The new bis(ethene) complexes I, [Rh(μ-O2CPh)(C2H4)2]2, and II are catalytically less active than [RhCl(C2H4)2]2 in the reaction of C2H4 and Ph2CN2 and gave as C-C coupling product not only 1,1-diphenylpropene but also mixts. of it and 1,1-diphenylcyclopropane.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqA1T69GpaZPLVg90H21EOLACvtfcHk0ljedSjtCSzgKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XjtlWntLc%253D&md5=f6ce07d24bfb42960d19ce7fd738bfa0</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1002%2Fcber.19961290609&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcber.19961290609%26sid%3Dliteratum%253Aachs%26aulast%3DWerner%26aufirst%3DH.%26aulast%3DPoelsma%26aufirst%3DS.%26aulast%3DSchneider%26aufirst%3DM.%2BE.%26aulast%3DWindm%25C3%25BCller%26aufirst%3DB.%26aulast%3DBarth%26aufirst%3DD.%26atitle%3DSynthesis%2520and%2520reactivity%2520of%2520bis%2528ethene%2529%2520rhodium%2528I%2529%2520and%2520iridium%2528I%2529%2520carboxylato%2520complexes%26jtitle%3DChem.%2520Ber.%26date%3D1996%26volume%3D129%26issue%3D6%26spage%3D647%26epage%3D652%26doi%3D10.1002%2Fcber.19961290609" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="note"><p class="first last">The authors note that at times better results were obtained when the asymmetric hydrogenation catalyst was prepared freshly before use from [Rh(nbd)<sub>2</sub>]BF<sub>4</sub> (CAS: 36620-11-8, obtained from Strem Chemicals, Inc. (catalogue no. 45-0230)) and (<i>S</i>,<i>S</i>)-MeBPE ligand (CAS: 136779-26-5, obtained from Strem Chemicals, Inc. (catalogue no. 15-0105)).</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="ref64"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref64'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ellis, F.</span>; <span class="NLM_string-name">Fulton, H. E.</span>; <span class="NLM_string-name">Hall, A.</span>; <span class="NLM_string-name">Jaxa-Chamiec, A. A.</span>; <span class="NLM_string-name">Swanson, S.</span>; <span class="NLM_string-name">Vile, S.</span></span> <i>Acetylene Compounds</i>. <span class="NLM_patent">WO2004026890 (A1)</span>, <span class="NLM_year">2004</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2004&author=F.+Ellis&author=H.+E.+Fulton&author=A.+Hall&author=A.+A.+Jaxa-Chamiec&author=S.+Swanson&author=S.+Vile&title=Acetylene+Compounds"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DEllis%26aufirst%3DF.%26jtitle%3DAcetylene%2520Compounds%26date%3D2004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref65"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref65'); return false;" data-citation="" class="refNumLink">65</a></strong><div class="NLM_citation" id="cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pedersen, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenbohm, C.</span></span> <span> </span><span class="NLM_article-title">Dry column vacuum chromatography</span>. <i>Synthesis</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>2001</i></span> (<span class="NLM_issue">16</span>),  <span class="NLM_fpage">2431</span>– <span class="NLM_lpage">2434</span>, <span class="refDoi"> DOI: 10.1055/s-2001-18722</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.1055%2Fs-2001-18722" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2001&publication_year=2001&pages=2431-2434&issue=16&author=D.+S.+Pedersenauthor=C.+Rosenbohm&title=Dry+column+vacuum+chromatography&doi=10.1055%2Fs-2001-18722"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1055%2Fs-2001-18722&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1055%252Fs-2001-18722%26sid%3Dliteratum%253Aachs%26aulast%3DPedersen%26aufirst%3DD.%2BS.%26aulast%3DRosenbohm%26aufirst%3DC.%26atitle%3DDry%2520column%2520vacuum%2520chromatography%26jtitle%3DSynthesis%26date%3D2001%26volume%3D2001%26issue%3D16%26spage%3D2431%26epage%3D2434%26doi%3D10.1055%2Fs-2001-18722" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref66"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref66'); return false;" data-citation="" class="refNumLink">66</a></strong><div class="NLM_citation" id="cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cox, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whittaker, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lalic, G.</span></span> <span> </span><span class="NLM_article-title">Copper-catalyzed semi-reduction of alkynes</span>. <i>Org. Synth.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>93</i></span>,  <span class="NLM_fpage">385</span>– <span class="NLM_lpage">398</span>, <span class="refDoi"> DOI: 10.1002/0471264229.os093.28</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.1002%2F0471264229.os093.28" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=93&publication_year=2017&pages=385-398&author=N.+Coxauthor=H.+Dangauthor=A.+M.+Whittakerauthor=G.+Lalic&title=Copper-catalyzed+semi-reduction+of+alkynes&doi=10.1002%2F0471264229.os093.28"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1002%2F0471264229.os093.28&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F0471264229.os093.28%26sid%3Dliteratum%253Aachs%26aulast%3DCox%26aufirst%3DN.%26aulast%3DDang%26aufirst%3DH.%26aulast%3DWhittaker%26aufirst%3DA.%2BM.%26aulast%3DLalic%26aufirst%3DG.%26atitle%3DCopper-catalyzed%2520semi-reduction%2520of%2520alkynes%26jtitle%3DOrg.%2520Synth.%26date%3D2017%26volume%3D93%26spage%3D385%26epage%3D398%26doi%3D10.1002%2F0471264229.os093.28" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref67"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref67'); return false;" data-citation="" class="refNumLink">67</a></strong><div class="NLM_citation" id="cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Becerra-Figueroa, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Movilla, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prunet, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miscione, G. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gamba-Sánchez, D.</span></span> <span> </span><span class="NLM_article-title">An intramolecular oxa-Michael reaction on <i>α,β</i>-unsaturated α-amino-δ-hydroxycarboxylic acid esters. synthesis of functionalized 1,3-dioxanes</span>. <i>Org. Biomol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>16</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">1277</span>– <span class="NLM_lpage">1286</span>, <span class="refDoi"> DOI: 10.1039/C7OB03066E</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.1039%2FC7OB03066E" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=29384169" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhs12hs7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2018&pages=1277-1286&issue=8&author=L.+Becerra-Figueroaauthor=S.+Movillaauthor=J.+Prunetauthor=G.+P.+Miscioneauthor=D.+Gamba-S%C3%A1nchez&title=An+intramolecular+oxa-Michael+reaction+on+%CE%B1%2C%CE%B2-unsaturated+%CE%B1-amino-%CE%B4-hydroxycarboxylic+acid+esters.+synthesis+of+functionalized+1%2C3-dioxanes&doi=10.1039%2FC7OB03066E"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">An intramolecular oxa-Michael reaction on α,β-unsaturated α-amino-δ-hydroxycarboxylic acid esters. Synthesis of functionalized 1,3-dioxanes</span></div><div class="casAuthors">Becerra-Figueroa, L.; Movilla, S.; Prunet, J.; Miscione, G. P.; Gamba-Sanchez, D.</div><div class="citationInfo"><span class="NLM_cas:title">Organic & Biomolecular Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1277-1286</span>CODEN:
                <span class="NLM_cas:coden">OBCRAK</span>;
        ISSN:<span class="NLM_cas:issn">1477-0520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A highly diastereoselective intramol. oxa-Michael reaction on α,β-unsatd. α-amino-δ-hydroxycarboxylic acid esters is presented; 1,3-dioxanes functionalized in positions 2, 4 and 6 were obtained in good yields and with excellent selectivities; an exptl. and computational study was carried out to understand the reaction course in terms of yields and selectivities.  This reaction proceeds under mild reaction conditions using highly electrophilic aldehydes and ketones.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqsWQcHSB-YhLVg90H21EOLACvtfcHk0li7Z9gLQQUacg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhs12hs7Y%253D&md5=1cde60dd8de85cbf4d2a7699dc1a9d0d</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1039%2FC7OB03066E&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC7OB03066E%26sid%3Dliteratum%253Aachs%26aulast%3DBecerra-Figueroa%26aufirst%3DL.%26aulast%3DMovilla%26aufirst%3DS.%26aulast%3DPrunet%26aufirst%3DJ.%26aulast%3DMiscione%26aufirst%3DG.%2BP.%26aulast%3DGamba-S%25C3%25A1nchez%26aufirst%3DD.%26atitle%3DAn%2520intramolecular%2520oxa-Michael%2520reaction%2520on%2520%25CE%25B1%252C%25CE%25B2-unsaturated%2520%25CE%25B1-amino-%25CE%25B4-hydroxycarboxylic%2520acid%2520esters.%2520synthesis%2520of%2520functionalized%25201%252C3-dioxanes%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2018%26volume%3D16%26issue%3D8%26spage%3D1277%26epage%3D1286%26doi%3D10.1039%2FC7OB03066E" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref68"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref68'); return false;" data-citation="" class="refNumLink">68</a></strong><div class="NLM_citation" id="cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Blanchard, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Da Silva, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">El Kortbi, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fourrey, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robert-Gero, M.</span></span> <span> </span><span class="NLM_article-title">Zinc-iron couple induced conjugate addition of alkyl halide derived radicals to activated olefins</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>58</i></span> (<span class="NLM_issue">23</span>),  <span class="NLM_fpage">6517</span>– <span class="NLM_lpage">6519</span>, <span class="refDoi"> DOI: 10.1021/jo00075a066</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo00075a066" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADyaK3sXmsFShu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=1993&pages=6517-6519&issue=23&author=P.+Blanchardauthor=A.+D.+Da+Silvaauthor=M.+S.+El+Kortbiauthor=J.+L.+Fourreyauthor=M.+Robert-Gero&title=Zinc-iron+couple+induced+conjugate+addition+of+alkyl+halide+derived+radicals+to+activated+olefins&doi=10.1021%2Fjo00075a066"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">Zinc-iron couple induced conjugate addition of alkyl halide derived radicals to activated olefins</span></div><div class="casAuthors">Blanchard, Pierre; Da Silva, Adilson D.; El Kortbi, Mohammed S.; Fourrey, Jean Louis; Robert-Gero, Malka</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">6517-19</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    </div><div class="casAbstract">Addn. of halo glycosides, e.g. I (R = iodo), with olefins, e.g. CH2:CHCO2Me, in the presence of Zn-Fe gave the corresponding glycosides, e.g. I (R = CH2CH2CO2Me),.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrteqaxoL3037Vg90H21EOLACvtfcHk0li7Z9gLQQUacg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXmsFShu7o%253D&md5=af75a9e2c79b11dc3e7fe7bca356e831</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1021%2Fjo00075a066&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo00075a066%26sid%3Dliteratum%253Aachs%26aulast%3DBlanchard%26aufirst%3DP.%26aulast%3DDa%2BSilva%26aufirst%3DA.%2BD.%26aulast%3DEl%2BKortbi%26aufirst%3DM.%2BS.%26aulast%3DFourrey%26aufirst%3DJ.%2BL.%26aulast%3DRobert-Gero%26aufirst%3DM.%26atitle%3DZinc-iron%2520couple%2520induced%2520conjugate%2520addition%2520of%2520alkyl%2520halide%2520derived%2520radicals%2520to%2520activated%2520olefins%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D1993%26volume%3D58%26issue%3D23%26spage%3D6517%26epage%3D6519%26doi%3D10.1021%2Fjo00075a066" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref69"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref69'); return false;" data-citation="" class="refNumLink">69</a></strong><div class="NLM_citation" id="cit69"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, D.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, X.-S.</span></span> <span> </span><span class="NLM_article-title">Highly substituted cyclopentane–CMP conjugates as potent sialyltransferase inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span> (<span class="NLM_issue">20</span>),  <span class="NLM_fpage">7972</span>– <span class="NLM_lpage">7990</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01181</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01181" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFKiu7jO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=7972-7990&issue=20&author=W.+Liauthor=Y.+Niuauthor=D.-C.+Xiongauthor=X.+Caoauthor=X.-S.+Ye&title=Highly+substituted+cyclopentane%E2%80%93CMP+conjugates+as+potent+sialyltransferase+inhibitors&doi=10.1021%2Facs.jmedchem.5b01181"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">Highly Substituted Cyclopentane-CMP Conjugates as Potent Sialyltransferase Inhibitors</span></div><div class="casAuthors">Li, Wenming; Niu, Youhong; Xiong, De-Cai; Cao, Xiaoping; Ye, Xin-Shan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">7972-7990</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Sialylconjugates on cell surfaces are involved in many biol. events such as cellular recognition, signal transduction, and immune response.  It has been reported that aberrant sialylation at the nonreducing end of glycoconjugates and overexpression of sialyltransferases (STs) in cells are correlated with the malignance, invasion, and metastasis of tumors.  Therefore, inhibitors of STs would provide valuable leads for the discovery of antitumor drugs.  On the basis of the transition state of the enzyme-catalyzed sialylation reaction, we proposed that the cyclopentane skeleton in its two puckered conformations might mimic the planar structure of the donor (CMP-Neu5Ac) in the transition state.  A series of cyclopentane-contg. compds. were designed and synthesized by coupling different cyclopentane α-hydroxyphosphonates with cytidine phosphoramidite.  Their inhibitory activities against recombinant human ST6Gal-I were assayed, and a potent inhibitor (I) with a Ki of 0.028±0.006 μM was identified.  The results show that the cyclopentanoid-type compds. could become a new type of sialyltransferase inhibitors as biol. probes or drug leads.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDnUM0ETSyjrVg90H21EOLACvtfcHk0li7Z9gLQQUacg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFKiu7jO&md5=3c05cf23900d96ee3930cd305a1eef24</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01181&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01181%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DW.%26aulast%3DNiu%26aufirst%3DY.%26aulast%3DXiong%26aufirst%3DD.-C.%26aulast%3DCao%26aufirst%3DX.%26aulast%3DYe%26aufirst%3DX.-S.%26atitle%3DHighly%2520substituted%2520cyclopentane%25E2%2580%2593CMP%2520conjugates%2520as%2520potent%2520sialyltransferase%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26issue%3D20%26spage%3D7972%26epage%3D7990%26doi%3D10.1021%2Facs.jmedchem.5b01181" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref70"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref70'); return false;" data-citation="" class="refNumLink">70</a></strong><div class="note"><p class="first last">The authors note that the N-Cbz cleavage reaction rate increased markedly upon addition of EtOH.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="ref71"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref71'); return false;" data-citation="" class="refNumLink">71</a></strong><div class="NLM_citation" id="cit71"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Blanchard, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">El Kortbi, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fourrey, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawrence, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robert-Gero, M.</span></span> <span> </span><span class="NLM_article-title">Relationship between chemical structure and antileishmanial effect of sinefungine analogues</span>. <i>Nucleosides Nucleotides</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>15</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">1121</span>– <span class="NLM_lpage">1135</span>, <span class="refDoi"> DOI: 10.1080/07328319608007381</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.1080%2F07328319608007381" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADyaK28XjvVWktb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=1996&pages=1121-1135&issue=6&author=P.+Blanchardauthor=M.+S.+El+Kortbiauthor=J.+L.+Fourreyauthor=F.+Lawrenceauthor=M.+Robert-Gero&title=Relationship+between+chemical+structure+and+antileishmanial+effect+of+sinefungine+analogues&doi=10.1080%2F07328319608007381"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">Relationship between chemical structure and antileishmanial effect of sinefungin analogs</span></div><div class="casAuthors">Blanchard, P.; El Kortbi, M. S.; Fourrey, J.-L.; Lawrence, F.; Robert-Gero, M.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleosides & Nucleotides</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1121-1135</span>CODEN:
                <span class="NLM_cas:coden">NUNUD5</span>;
        ISSN:<span class="NLM_cas:issn">0732-8311</span>.
    
            (<span class="NLM_cas:orgname">Dekker</span>)
        </div><div class="casAbstract">The synthesis of various analogs of sinefungin, e.g. I (R = H, NH2), has been developed by means of an original approach which uses radical chem.  The study of their biol. activities revealed that for the antileishmanial effect of sinefungin, the presence of the amino group at C-6' in the (S)-configuration and the presence of the carboxyl group at C-9' are necessary.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_Ho5g0xhJybVg90H21EOLACvtfcHk0liXeujUl3w_Gw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XjvVWktb4%253D&md5=fb4457d5a3a6e706c1f9fff041ad7580</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1080%2F07328319608007381&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F07328319608007381%26sid%3Dliteratum%253Aachs%26aulast%3DBlanchard%26aufirst%3DP.%26aulast%3DEl%2BKortbi%26aufirst%3DM.%2BS.%26aulast%3DFourrey%26aufirst%3DJ.%2BL.%26aulast%3DLawrence%26aufirst%3DF.%26aulast%3DRobert-Gero%26aufirst%3DM.%26atitle%3DRelationship%2520between%2520chemical%2520structure%2520and%2520antileishmanial%2520effect%2520of%2520sinefungine%2520analogues%26jtitle%3DNucleosides%2520Nucleotides%26date%3D1996%26volume%3D15%26issue%3D6%26spage%3D1121%26epage%3D1135%26doi%3D10.1080%2F07328319608007381" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref72"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref72'); return false;" data-citation="" class="refNumLink">72</a></strong><div class="NLM_citation" id="cit72"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sarabia, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martín-Ortiz, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">López-Herrera, F. J.</span></span> <span> </span><span class="NLM_article-title">A convergent synthetic approach to the nucleoside-type liposidomycin antibiotics</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>5</i></span> (<span class="NLM_issue">21</span>),  <span class="NLM_fpage">3927</span>– <span class="NLM_lpage">3930</span>, <span class="refDoi"> DOI: 10.1021/ol0355074</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ol0355074" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BD3sXnt1Kksbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2003&pages=3927-3930&issue=21&author=F.+Sarabiaauthor=L.+Mart%C3%ADn-Ortizauthor=F.+J.+L%C3%B3pez-Herrera&title=A+convergent+synthetic+approach+to+the+nucleoside-type+liposidomycin+antibiotics&doi=10.1021%2Fol0355074"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">A Convergent Synthetic Approach to the Nucleoside-Type Liposidomycin Antibiotics</span></div><div class="casAuthors">Sarabia, Francisco; Martin-Ortiz, Laura; Lopez-Herrera, F. Jorge</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">3927-3930</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7060</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A synthetic approach toward the liposidomycins, a family of complex nucleoside-type antibiotics, is reported based on the synthesis of epoxy-amides derived from the reaction of sulfur ylides with the uridyl aldehyde deriv. I.  To this end, the epoxy-amide deriv. of indoline II was stereoselectively prepd. and, after treatment with DDQ, transformed into the corresponding N-indole epoxy-amide.  The N-indole epoxy-amide provides ready access to a variety of structures related to the diazepanone core present in the liposidomycins by reaction with a variety of amines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGruWQHXSPDo97Vg90H21EOLACvtfcHk0liXeujUl3w_Gw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXnt1Kksbs%253D&md5=25357efc8909b301084d57a08d2b6455</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1021%2Fol0355074&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fol0355074%26sid%3Dliteratum%253Aachs%26aulast%3DSarabia%26aufirst%3DF.%26aulast%3DMart%25C3%25ADn-Ortiz%26aufirst%3DL.%26aulast%3DL%25C3%25B3pez-Herrera%26aufirst%3DF.%2BJ.%26atitle%3DA%2520convergent%2520synthetic%2520approach%2520to%2520the%2520nucleoside-type%2520liposidomycin%2520antibiotics%26jtitle%3DOrg.%2520Lett.%26date%3D2003%26volume%3D5%26issue%3D21%26spage%3D3927%26epage%3D3930%26doi%3D10.1021%2Fol0355074" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref73"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref73'); return false;" data-citation="" class="refNumLink">73</a></strong><div class="NLM_citation" id="cit73"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Petakamsetty, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ansari, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramapanicker, R.</span></span> <span> </span><span class="NLM_article-title">Diastereoselective synthesis of furanose and pyranose substituted glycine and alanine derivatives via proline-catalyzed asymmetric α-amination of aldehydes</span>. <i>Carbohydr. Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>435</i></span>,  <span class="NLM_fpage">37</span>– <span class="NLM_lpage">49</span>, <span class="refDoi"> DOI: 10.1016/j.carres.2016.09.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.1016%2Fj.carres.2016.09.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=27693912" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhs1Srs7zI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=435&publication_year=2016&pages=37-49&author=R.+Petakamsettyauthor=A.+Ansariauthor=R.+Ramapanicker&title=Diastereoselective+synthesis+of+furanose+and+pyranose+substituted+glycine+and+alanine+derivatives+via+proline-catalyzed+asymmetric+%CE%B1-amination+of+aldehydes&doi=10.1016%2Fj.carres.2016.09.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">Diastereoselective synthesis of furanose and pyranose substituted glycine and alanine derivatives via proline-catalyzed asymmetric α-amination of aldehydes</span></div><div class="casAuthors">Petakamsetty, Ramu; Ansari, Anas; Ramapanicker, Ramesh</div><div class="citationInfo"><span class="NLM_cas:title">Carbohydrate Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">435</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">37-49</span>CODEN:
                <span class="NLM_cas:coden">CRBRAT</span>;
        ISSN:<span class="NLM_cas:issn">0008-6215</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A concise organocatalytic route toward the synthesis of furanose and pyranose substituted glycine and alanine derivs. is reported.  These compds. are core structural units of some of the naturally available antibiotics and antifungal agents.  Proline-catalyzed asym. α-amination of aldehydes derived from sugars is used as the key reaction to synthesize twelve sugar amino acid derivs.  The asym. transformations proceeded in good yields and with good to excellent diastereoselectivity.  The application of the synthesized amino acids is demonstrated by synthesizing a tripeptide contg. one of them.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrq4HLkO9P14LVg90H21EOLACvtfcHk0liXeujUl3w_Gw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhs1Srs7zI&md5=8752fc493afc01b6e01924db66f7b9da</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1016%2Fj.carres.2016.09.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.carres.2016.09.011%26sid%3Dliteratum%253Aachs%26aulast%3DPetakamsetty%26aufirst%3DR.%26aulast%3DAnsari%26aufirst%3DA.%26aulast%3DRamapanicker%26aufirst%3DR.%26atitle%3DDiastereoselective%2520synthesis%2520of%2520furanose%2520and%2520pyranose%2520substituted%2520glycine%2520and%2520alanine%2520derivatives%2520via%2520proline-catalyzed%2520asymmetric%2520%25CE%25B1-amination%2520of%2520aldehydes%26jtitle%3DCarbohydr.%2520Res.%26date%3D2016%26volume%3D435%26spage%3D37%26epage%3D49%26doi%3D10.1016%2Fj.carres.2016.09.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref74"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref74'); return false;" data-citation="" class="refNumLink">74</a></strong><div class="NLM_citation" id="cit74"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Herrera, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rondón, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suárez, E.</span></span> <span> </span><span class="NLM_article-title">Stereocontrolled photocyclization of 1,2-diketones: application of a 1,3-acetyl group transfer methodology to carbohydrates</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>73</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">3384</span>– <span class="NLM_lpage">3391</span>, <span class="refDoi"> DOI: 10.1021/jo702663w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo702663w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjvFejurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2008&pages=3384-3391&issue=9&author=A.+J.+Herreraauthor=M.+Rond%C3%B3nauthor=E.+Su%C3%A1rez&title=Stereocontrolled+photocyclization+of+1%2C2-diketones%3A+application+of+a+1%2C3-acetyl+group+transfer+methodology+to+carbohydrates&doi=10.1021%2Fjo702663w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">Stereocontrolled Photocyclization of 1,2-Diketones: Application of a 1,3-Acetyl Group Transfer Methodology to Carbohydrates</span></div><div class="casAuthors">Herrera, Antonio J.; Rondon, Maria; Suarez, Ernesto</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3384-3391</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Photolysis of 1-glycosyl-2,3-butanedione derivs. using visible light is a mild and selective procedure for the synthesis of chiral 1-hydroxy-1-methyl-5-oxaspiro[3.5]nonan-2-one carbohydrate derivs., e.g., I.  The results strongly suggest that stereocontrol of the cyclization is dependent on conformational and stereoelectronic factors.  Further oxidative opening of the 1-hydroxy-1-methyl-2-cyclobutanone moiety affords new C-ketoside derivs. either in C- and O-glycoside series, e.g., II.  This tandem two-step process could be considered to be a stereocontrolled 1,3-transference of an acetyl group, and it can be applied either to pyranose and furanose models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqUWYTOKhEDP7Vg90H21EOLACvtfcHk0liOns7DaZdRxA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjvFejurw%253D&md5=b096ed26cb720cd516e4f607fc9a1c25</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1021%2Fjo702663w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo702663w%26sid%3Dliteratum%253Aachs%26aulast%3DHerrera%26aufirst%3DA.%2BJ.%26aulast%3DRond%25C3%25B3n%26aufirst%3DM.%26aulast%3DSu%25C3%25A1rez%26aufirst%3DE.%26atitle%3DStereocontrolled%2520photocyclization%2520of%25201%252C2-diketones%253A%2520application%2520of%2520a%25201%252C3-acetyl%2520group%2520transfer%2520methodology%2520to%2520carbohydrates%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2008%26volume%3D73%26issue%3D9%26spage%3D3384%26epage%3D3391%26doi%3D10.1021%2Fjo702663w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref75"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref75'); return false;" data-citation="" class="refNumLink">75</a></strong><div class="NLM_citation" id="cit75"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Charette, A.
B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Juteau, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lebel, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Molinaro, C.</span></span> <span> </span><span class="NLM_article-title">Enantioselective cyclopropanation of allylic alcohols with dioxaborolane ligands:  scope and synthetic applications</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>120</i></span> (<span class="NLM_issue">46</span>),  <span class="NLM_fpage">11943</span>– <span class="NLM_lpage">11952</span>, <span class="refDoi"> DOI: 10.1021/ja982055v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja982055v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADyaK1cXntFegs74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=120&publication_year=1998&pages=11943-11952&issue=46&author=A.%0AB.+Charetteauthor=H.+Juteauauthor=H.+Lebelauthor=C.+Molinaro&title=Enantioselective+cyclopropanation+of+allylic+alcohols+with+dioxaborolane+ligands%3A%E2%80%89+scope+and+synthetic+applications&doi=10.1021%2Fja982055v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">Enantioselective Cyclopropanation of Allylic Alcohols with Dioxaborolane Ligands: Scope and Synthetic Applications</span></div><div class="casAuthors">Charette, Andre B.; Juteau, Helene; Lebel, Helene; Molinaro, Carmela</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">120</span>
        (<span class="NLM_cas:issue">46</span>),
    <span class="NLM_cas:pages">11943-11952</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A very effective chiral controller was found for the conversion of allylic alcs. into the corresponding enantiomerically enriched cyclopropanes using bis(iodomethyl)zinc.  A variety of chiral, nonracemic cyclopropylmethanols could be obtained according to this method.  This methodol. was extended with success to the cyclopropanation of unconjugated and conjugated polyenes and homoallylic alcs.  The cyclopropanation of allylic carbamates was investigated with this system, but it was found that enantioenriched cyclopropylmethylamines are best prepd. from enantioenriched cyclopropylmethanols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxyjTCaz_r3LVg90H21EOLACvtfcHk0liOns7DaZdRxA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXntFegs74%253D&md5=cdbfeb346b742e9ecb7023e250fd73b0</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1021%2Fja982055v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja982055v%26sid%3Dliteratum%253Aachs%26aulast%3DCharette%26aufirst%3DA.%2BB.%26aulast%3DJuteau%26aufirst%3DH.%26aulast%3DLebel%26aufirst%3DH.%26aulast%3DMolinaro%26aufirst%3DC.%26atitle%3DEnantioselective%2520cyclopropanation%2520of%2520allylic%2520alcohols%2520with%2520dioxaborolane%2520ligands%253A%25E2%2580%2589%2520scope%2520and%2520synthetic%2520applications%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1998%26volume%3D120%26issue%3D46%26spage%3D11943%26epage%3D11952%26doi%3D10.1021%2Fja982055v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref76"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref76'); return false;" data-citation="" class="refNumLink">76</a></strong><div class="NLM_citation" id="cit76"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hagiya, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muramoto, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Misaki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sugimura, T.</span></span> <span> </span><span class="NLM_article-title">DMEAD: a new dialkyl azodicarboxylate for the Mitsunobu reaction</span>. <i>Tetrahedron</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>65</i></span> (<span class="NLM_issue">31</span>),  <span class="NLM_fpage">6109</span>– <span class="NLM_lpage">6114</span>, <span class="refDoi"> DOI: 10.1016/j.tet.2009.05.048</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.1016%2Fj.tet.2009.05.048" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BD1MXotFCksbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2009&pages=6109-6114&issue=31&author=K.+Hagiyaauthor=N.+Muramotoauthor=T.+Misakiauthor=T.+Sugimura&title=DMEAD%3A+a+new+dialkyl+azodicarboxylate+for+the+Mitsunobu+reaction&doi=10.1016%2Fj.tet.2009.05.048"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit76R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">DMEAD: a new dialkyl azodicarboxylate for the Mitsunobu reaction</span></div><div class="casAuthors">Hagiya, Kazutake; Muramoto, Natsuko; Misaki, Tomonori; Sugimura, Takashi</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">65</span>
        (<span class="NLM_cas:issue">31</span>),
    <span class="NLM_cas:pages">6109-6114</span>CODEN:
                <span class="NLM_cas:coden">TETRAB</span>;
        ISSN:<span class="NLM_cas:issn">0040-4020</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Di-2-methoxyethyl azodicarboxylate (DMEAD) is prepd. in 65% yield in two steps as a cryst. solid.  Use of DMEAD in the Mitsunobu reaction of a variety of alcs. with pronucleophiles results in good yields of the products under sufficient stereospecificity of inversion, as conventional diisopropyl azodicarboxylate (DIAD) does.  Isolation of the product is, however, much easier with DMEAD than that with DIAD, because the hydrazine produced from DMEAD is highly hydrophilic and is completely separable by a simple extn. into neutral water.  Purifn. of the org. layer, after sepn. of the other byproduct, triphenylphosphine oxide, by filtration, easily provides high purity of the product in a good yield.  Concn. of the water layer yields the hydrazine, which can be reused for the prepn. of DMEAD.  One-step removal of the two byproducts by the aq. extn. was also possible when trimethylphosphine and DMEAD were employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8942g2yxAwbVg90H21EOLACvtfcHk0liOns7DaZdRxA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXotFCksbc%253D&md5=1465a85e39f7dd03b9c7f74d9f599c20</span></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1016%2Fj.tet.2009.05.048&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tet.2009.05.048%26sid%3Dliteratum%253Aachs%26aulast%3DHagiya%26aufirst%3DK.%26aulast%3DMuramoto%26aufirst%3DN.%26aulast%3DMisaki%26aufirst%3DT.%26aulast%3DSugimura%26aufirst%3DT.%26atitle%3DDMEAD%253A%2520a%2520new%2520dialkyl%2520azodicarboxylate%2520for%2520the%2520Mitsunobu%2520reaction%26jtitle%3DTetrahedron%26date%3D2009%26volume%3D65%26issue%3D31%26spage%3D6109%26epage%3D6114%26doi%3D10.1016%2Fj.tet.2009.05.048" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref77"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref77'); return false;" data-citation="" class="refNumLink">77</a></strong><div class="NLM_citation" id="cit77"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bergström, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suresh, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naidu, V. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Unelius, C. R.</span></span> <span> </span><span class="NLM_article-title">N-iodosuccinimide (NIS) in direct aromatic iodination</span>. <i>Eur. J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>2017</i></span> (<span class="NLM_issue">22</span>),  <span class="NLM_fpage">3234</span>– <span class="NLM_lpage">3239</span>, <span class="refDoi"> DOI: 10.1002/ejoc.201700173</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.1002%2Fejoc.201700173" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2017&publication_year=2017&pages=3234-3239&issue=22&author=M.+Bergstr%C3%B6mauthor=G.+Sureshauthor=V.+R.+Naiduauthor=C.+R.+Unelius&title=N-iodosuccinimide+%28NIS%29+in+direct+aromatic+iodination&doi=10.1002%2Fejoc.201700173"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=10.1002%2Fejoc.201700173&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fejoc.201700173%26sid%3Dliteratum%253Aachs%26aulast%3DBergstr%25C3%25B6m%26aufirst%3DM.%26aulast%3DSuresh%26aufirst%3DG.%26aulast%3DNaidu%26aufirst%3DV.%2BR.%26aulast%3DUnelius%26aufirst%3DC.%2BR.%26atitle%3DN-iodosuccinimide%2520%2528NIS%2529%2520in%2520direct%2520aromatic%2520iodination%26jtitle%3DEur.%2520J.%2520Org.%2520Chem.%26date%3D2017%26volume%3D2017%26issue%3D22%26spage%3D3234%26epage%3D3239%26doi%3D10.1002%2Fejoc.201700173" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref78"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref78'); return false;" data-citation="" class="refNumLink">78</a></strong><div class="NLM_citation" id="cit78"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nammalwar, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunce, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berlin, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bourne, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bourne, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrow, E. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrow, W. W.</span></span> <span> </span><span class="NLM_article-title">Approaches to iodinated derivatives of vanillin and isovanillin</span>. <i>Org. Prep. Proced. Int.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>44</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">146</span>– <span class="NLM_lpage">152</span>, <span class="refDoi"> DOI: 10.1080/00304948.2012.643700</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.1080%2F00304948.2012.643700" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=23805008" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BC38XltVKrsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2012&pages=146-152&issue=2&author=B.+Nammalwarauthor=R.+A.+Bunceauthor=K.+D.+Berlinauthor=C.+R.+Bourneauthor=P.+C.+Bourneauthor=E.+W.+Barrowauthor=W.+W.+Barrow&title=Approaches+to+iodinated+derivatives+of+vanillin+and+isovanillin&doi=10.1080%2F00304948.2012.643700"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit78R"><div class="casContent"><span class="casTitleNuber">78</span><div class="casTitle"><span class="NLM_cas:atitle">Approaches to Iodinated Derivatives of Vanillin and Isovanillin</span></div><div class="casAuthors">Nammalwar, Baskar; Bunce, Richard A.; Berlin, K. Darrell; Bourne, Christina R.; Bourne, Philip C.; Barrow, Esther W.; Barrow, William W.</div><div class="citationInfo"><span class="NLM_cas:title">Organic Preparations and Procedures International</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">146-152</span>CODEN:
                <span class="NLM_cas:coden">OPPIAK</span>;
        ISSN:<span class="NLM_cas:issn">0030-4948</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">A straightforward approach was developed by which derivs. of 5-iodovanillin and 5-iodoisovanillin can be prepd.  The use of aluminum chloride in pyridine proved most convenient and cost effective for the cleavage of the Me ether in 5-iodovanillin to generate 3,4-dihydroxy-5-iodobenzaldehyde (I, R = H).  Regiospecific O-alkylation of I (R = H) at the C-4 hydroxy group with Me iodide in DBU/DMF then provided an intermediate, which could be further alkylated at the C-5 hydroxy group using the same conditions to afford the ether series I (R = Me, Et, etc).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGric2cEP9xEVbVg90H21EOLACvtfcHk0liQ7RRKglc5gQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XltVKrsr0%253D&md5=dbfbe86baa645d5641d69413af5b1330</span></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=10.1080%2F00304948.2012.643700&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F00304948.2012.643700%26sid%3Dliteratum%253Aachs%26aulast%3DNammalwar%26aufirst%3DB.%26aulast%3DBunce%26aufirst%3DR.%2BA.%26aulast%3DBerlin%26aufirst%3DK.%2BD.%26aulast%3DBourne%26aufirst%3DC.%2BR.%26aulast%3DBourne%26aufirst%3DP.%2BC.%26aulast%3DBarrow%26aufirst%3DE.%2BW.%26aulast%3DBarrow%26aufirst%3DW.%2BW.%26atitle%3DApproaches%2520to%2520iodinated%2520derivatives%2520of%2520vanillin%2520and%2520isovanillin%26jtitle%3DOrg.%2520Prep.%2520Proced.%2520Int.%26date%3D2012%26volume%3D44%26issue%3D2%26spage%3D146%26epage%3D152%26doi%3D10.1080%2F00304948.2012.643700" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref79"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref79'); return false;" data-citation="" class="refNumLink">79</a></strong><div class="NLM_citation" id="cit79"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tae, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Min, S.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rhim, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choo, H.</span></span> <span> </span><span class="NLM_article-title">5-HT7 receptor modulators: amino groups attached to biphenyl scaffold determine functional activity</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>123</i></span>,  <span class="NLM_fpage">180</span>– <span class="NLM_lpage">190</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2016.07.029</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.1016%2Fj.ejmech.2016.07.029" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=27475109" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1OntrvP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=123&publication_year=2016&pages=180-190&author=Y.+Kimauthor=H.+Parkauthor=J.+Leeauthor=J.+Taeauthor=H.+J.+Kimauthor=S.-J.+Minauthor=H.+Rhimauthor=H.+Choo&title=5-HT7+receptor+modulators%3A+amino+groups+attached+to+biphenyl+scaffold+determine+functional+activity&doi=10.1016%2Fj.ejmech.2016.07.029"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit79R"><div class="casContent"><span class="casTitleNuber">79</span><div class="casTitle"><span class="NLM_cas:atitle">5-HT7 receptor modulators: Amino groups attached to biphenyl scaffold determine functional activity</span></div><div class="casAuthors">Kim, Youngjae; Park, Hyeri; Lee, Jeongeun; Tae, Jinsung; Kim, Hak Joong; Min, Sun-Joon; Rhim, Hyewhon; Choo, Hyunah</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">123</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">180-190</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">A novel scaffold for 5-HT7R modulators, a series of biphenyl-3-yl-methanamine derivs. with various amino groups I (R1 = H, Cl, Me, OMe; NR2R3 = 4-(2-methoxyphenyl)piperazine, 4-methyl-piperidine, NMe2, NMe), II (NR2R3 = 4-(2-methoxyphenyl)piperazine, 4-methyl-piperidine, NMe2, NMe) were designed and prepd.  Evaluation of functional activities as well as binding affinities of the title compds. identified partial agonists (EC50 = 0.55-3.2 μM) and full antagonists (IC50 = 5.57-23.1 μM) depending on the amino substituents.  Mol. docking study suggested that the ligand-based switch in functional activity from agonist to antagonist results from the size of the amino groups and thereby different binding modes to 5-HT7R.  In particular, interaction of the ligand with Arg367 of 5-HT7R is shown to differentiate agonists and antagonists.  In the pharmacophore model study, two distinct pharmacophore models can tell whether a ligand is an agonist or an antagonist.  Taken together, this study provides valuable information for designing novel compds. with selective agonistic or antagonistic properties against 5-HT7R.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_XDczBDq_vrVg90H21EOLACvtfcHk0liQ7RRKglc5gQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1OntrvP&md5=1655d984a53c5fd9292330f24e7510d8</span></div><a href="/servlet/linkout?suffix=cit79&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2016.07.029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2016.07.029%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DY.%26aulast%3DPark%26aufirst%3DH.%26aulast%3DLee%26aufirst%3DJ.%26aulast%3DTae%26aufirst%3DJ.%26aulast%3DKim%26aufirst%3DH.%2BJ.%26aulast%3DMin%26aufirst%3DS.-J.%26aulast%3DRhim%26aufirst%3DH.%26aulast%3DChoo%26aufirst%3DH.%26atitle%3D5-HT7%2520receptor%2520modulators%253A%2520amino%2520groups%2520attached%2520to%2520biphenyl%2520scaffold%2520determine%2520functional%2520activity%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2016%26volume%3D123%26spage%3D180%26epage%3D190%26doi%3D10.1016%2Fj.ejmech.2016.07.029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref80"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref80'); return false;" data-citation="" class="refNumLink">80</a></strong><div class="NLM_citation" id="cit80"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Martinez
Molina, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jafari, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ignatushchenko, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larsson, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sreekumar, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nordlund, P.</span></span> <span> </span><span class="NLM_article-title">Monitoring drug target engagement in cells and tissues using the cellular thermal shift assay</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>341</i></span> (<span class="NLM_issue">6141</span>),  <span class="NLM_fpage">84</span>– <span class="NLM_lpage">87</span>, <span class="refDoi"> DOI: 10.1126/science.1233606</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.1126%2Fscience.1233606" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=23828940" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A280%3ADC%252BC3sjotlOqtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=341&publication_year=2013&pages=84-87&issue=6141&author=D.+Martinez%0AMolinaauthor=R.+Jafariauthor=M.+Ignatushchenkoauthor=T.+Sekiauthor=E.+A.+Larssonauthor=C.+Danauthor=L.+Sreekumarauthor=Y.+Caoauthor=P.+Nordlund&title=Monitoring+drug+target+engagement+in+cells+and+tissues+using+the+cellular+thermal+shift+assay&doi=10.1126%2Fscience.1233606"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit80R"><div class="casContent"><span class="casTitleNuber">80</span><div class="casTitle"><span class="NLM_cas:atitle">Monitoring drug target engagement in cells and tissues using the cellular thermal shift assay</span></div><div class="casAuthors">Martinez Molina Daniel; Jafari Rozbeh; Ignatushchenko Marina; Seki Takahiro; Larsson E Andreas; Dan Chen; Sreekumar Lekshmy; Cao Yihai; Nordlund Par</div><div class="citationInfo"><span class="NLM_cas:title">Science (New York, N.Y.)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">341</span>
        (<span class="NLM_cas:issue">6141</span>),
    <span class="NLM_cas:pages">84-7</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The efficacy of therapeutics is dependent on a drug binding to its cognate target.  Optimization of target engagement by drugs in cells is often challenging, because drug binding cannot be monitored inside cells.  We have developed a method for evaluating drug binding to target proteins in cells and tissue samples.  This cellular thermal shift assay (CETSA) is based on the biophysical principle of ligand-induced thermal stabilization of target proteins.  Using this assay, we validated drug binding for a set of important clinical targets and monitored processes of drug transport and activation, off-target effects and drug resistance in cancer cell lines, as well as drug distribution in tissues.  CETSA is likely to become a valuable tool for the validation and optimization of drug target engagement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRRn5K_mF1rcrHHuJhbyicZfW6udTcc2eYAfpvrFgBZ-bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3sjotlOqtA%253D%253D&md5=a73bb49adb0d4f445223ea6f74fe3c10</span></div><a href="/servlet/linkout?suffix=cit80&amp;dbid=16384&amp;doi=10.1126%2Fscience.1233606&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1233606%26sid%3Dliteratum%253Aachs%26aulast%3DMartinez%2BMolina%26aufirst%3DD.%26aulast%3DJafari%26aufirst%3DR.%26aulast%3DIgnatushchenko%26aufirst%3DM.%26aulast%3DSeki%26aufirst%3DT.%26aulast%3DLarsson%26aufirst%3DE.%2BA.%26aulast%3DDan%26aufirst%3DC.%26aulast%3DSreekumar%26aufirst%3DL.%26aulast%3DCao%26aufirst%3DY.%26aulast%3DNordlund%26aufirst%3DP.%26atitle%3DMonitoring%2520drug%2520target%2520engagement%2520in%2520cells%2520and%2520tissues%2520using%2520the%2520cellular%2520thermal%2520shift%2520assay%26jtitle%3DScience%26date%3D2013%26volume%3D341%26issue%3D6141%26spage%3D84%26epage%3D87%26doi%3D10.1126%2Fscience.1233606" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref81"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref81'); return false;" data-citation="" class="refNumLink">81</a></strong><div class="NLM_citation" id="cit81"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jafari, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Almqvist, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Axelsson, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ignatushchenko, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lundbäck, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nordlund, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinez Molina, D.</span></span> <span> </span><span class="NLM_article-title">The cellular thermal shift assay for evaluating drug target interactions in cells</span>. <i>Nat. Protoc.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>9</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">2100</span>– <span class="NLM_lpage">2122</span>, <span class="refDoi"> DOI: 10.1038/nprot.2014.138</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.1038%2Fnprot.2014.138" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=25101824" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht12msrfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=2100-2122&issue=9&author=R.+Jafariauthor=H.+Almqvistauthor=H.+Axelssonauthor=M.+Ignatushchenkoauthor=T.+Lundb%C3%A4ckauthor=P.+Nordlundauthor=D.+Martinez+Molina&title=The+cellular+thermal+shift+assay+for+evaluating+drug+target+interactions+in+cells&doi=10.1038%2Fnprot.2014.138"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit81R"><div class="casContent"><span class="casTitleNuber">81</span><div class="casTitle"><span class="NLM_cas:atitle">The cellular thermal shift assay for evaluating drug target interactions in cells</span></div><div class="casAuthors">Jafari, Rozbeh; Almqvist, Helena; Axelsson, Hanna; Ignatushchenko, Marina; Lundbaeck, Thomas; Nordlund, Paer; Molina, Daniel Martinez</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2100-2122</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Thermal shift assays are used to study thermal stabilization of proteins upon ligand binding.  Such assays have been used extensively on purified proteins in the drug discovery industry and in academia to detect interactions.  Recently, we published a proof-of-principle study describing the implementation of thermal shift assays in a cellular format, which we call the cellular thermal shift assay (CETSA).  The method allows studies of target engagement of drug candidates in a cellular context, herein exemplified with exptl. data on the human kinases p38α and ERK1/2.  The assay involves treatment of cells with a compd. of interest, heating to denature and ppt. proteins, cell lysis, and the sepn. of cell debris and aggregates from the sol. protein fraction.  Whereas unbound proteins denature and ppt. at elevated temps., ligand-bound proteins remain in soln.  We describe two procedures for detecting the stabilized protein in the sol. fraction of the samples.  One approach involves sample workup and detection using quant. western blotting, whereas the second is performed directly in soln. and relies on the induced proximity of two target-directed antibodies upon binding to sol. protein.  The latter protocol has been optimized to allow an increased throughput, as potential applications require large nos. of samples.  Both approaches can be completed in a day.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzarqomAK6eLVg90H21EOLACvtfcHk0lhwbpsyeBF8Hg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht12msrfJ&md5=e7852e1a306bca1698a8e8463ae566b1</span></div><a href="/servlet/linkout?suffix=cit81&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2014.138&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2014.138%26sid%3Dliteratum%253Aachs%26aulast%3DJafari%26aufirst%3DR.%26aulast%3DAlmqvist%26aufirst%3DH.%26aulast%3DAxelsson%26aufirst%3DH.%26aulast%3DIgnatushchenko%26aufirst%3DM.%26aulast%3DLundb%25C3%25A4ck%26aufirst%3DT.%26aulast%3DNordlund%26aufirst%3DP.%26aulast%3DMartinez%2BMolina%26aufirst%3DD.%26atitle%3DThe%2520cellular%2520thermal%2520shift%2520assay%2520for%2520evaluating%2520drug%2520target%2520interactions%2520in%2520cells%26jtitle%3DNat.%2520Protoc.%26date%3D2014%26volume%3D9%26issue%3D9%26spage%3D2100%26epage%3D2122%26doi%3D10.1038%2Fnprot.2014.138" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref82"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref82'); return false;" data-citation="" class="refNumLink">82</a></strong><div class="NLM_citation" id="cit82"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Morin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eisenbraun, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Key, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanschagrin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Timony, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ottaviano, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sliz, P.</span></span> <span> </span><span class="NLM_article-title">Cutting edge: Collaboration gets the most out of software</span>. <i>eLife</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>2</i></span>, <span class="NLM_elocation-id">e01456</span> <span class="refDoi"> DOI: 10.7554/eLife.01456</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.7554%2FeLife.01456" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=24040512" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2013&author=A.+Morinauthor=B.+Eisenbraunauthor=J.+Keyauthor=C.+Sanschagrinauthor=M.+A.+Timonyauthor=M.+Ottavianoauthor=P.+Sliz&title=Cutting+edge%3A+Collaboration+gets+the+most+out+of+software&doi=10.7554%2FeLife.01456"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit82&amp;dbid=16384&amp;doi=10.7554%2FeLife.01456&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7554%252FeLife.01456%26sid%3Dliteratum%253Aachs%26aulast%3DMorin%26aufirst%3DA.%26aulast%3DEisenbraun%26aufirst%3DB.%26aulast%3DKey%26aufirst%3DJ.%26aulast%3DSanschagrin%26aufirst%3DC.%26aulast%3DTimony%26aufirst%3DM.%2BA.%26aulast%3DOttaviano%26aufirst%3DM.%26aulast%3DSliz%26aufirst%3DP.%26atitle%3DCutting%2520edge%253A%2520Collaboration%2520gets%2520the%2520most%2520out%2520of%2520software%26jtitle%3DeLife%26date%3D2013%26volume%3D2%26doi%3D10.7554%2FeLife.01456" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref83"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref83'); return false;" data-citation="" class="refNumLink">83</a></strong><div class="NLM_citation" id="ref83_1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">May, E. M.</span>; <span class="NLM_string-name">Policarpo, R. L.</span>; <span class="NLM_string-name">Gaudet, R.</span></span> <span> </span><span class="NLM_article-title">X-Ray Diffraction data for: Co-crystal structure of human Nicotinamide N-Methyltransferase (NNMT) in complex with High-Affinity Alkynyl Bisubstrate Inhibitor NS1</span>. PDB Code 6ORR,  <i>SBGrid Data Bank</i>, V1, <span class="refDoi"> DOI: 10.15785/SBGRID/731</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref83/ref83_1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.15785%2FSBGRID%2F731" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=May%2C+E.+M.%3B+Policarpo%2C+R.+L.%3B+Gaudet%2C+R.+X-Ray+Diffraction+data+for%3A+Co-crystal+structure+of+human+Nicotinamide+N-Methyltransferase+%28NNMT%29+in+complex+with+High-Affinity+Alkynyl+Bisubstrate+Inhibitor+NS1.+PDB+Code+6ORR%2C+SBGrid+Data+Bank%2C+V1%2C+10.15785%2FSBGRID%2F731."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=ref83_1&amp;dbid=16384&amp;doi=10.15785%2FSBGRID%2F731&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26id%3Ddoi%3A10.15785%252FSBGRID%252F731%26sid%3Dliteratum%253Aachs%26aulast%3DMay%26aufirst%3DE.%2BM.%26atitle%3DX-Ray%2520Diffraction%2520data%2520for%253A%2520Co-crystal%2520structure%2520of%2520human%2520Nicotinamide%2520N-Methyltransferase%2520%2528NNMT%2529%2520in%2520complex%2520with%2520High-Affinity%2520Alkynyl%2520Bisubstrate%2520Inhibitor%2520NS1%26jtitle%3DSBGrid%2520Data%2520Bank%26doi%3D10.15785%2FSBGRID%2F731" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':[],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':[],'ref64':['cit64'],'ref65':['cit65'],'ref66':['cit66'],'ref67':['cit67'],'ref68':['cit68'],'ref69':['cit69'],'ref70':[],'ref71':['cit71'],'ref72':['cit72'],'ref73':['cit73'],'ref74':['cit74'],'ref75':['cit75'],'ref76':['cit76'],'ref77':['cit77'],'ref78':['cit78'],'ref79':['cit79'],'ref80':['cit80'],'ref81':['cit81'],'ref82':['cit82'],'ref83':['ref83_1']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 15 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Charles A. Lewis, Jr., <span class="NLM_string-name hlFld-ContribAuthor">Richard Wolfenden</span>. </span><span class="cited-content_cbyCitation_article-title">The Burden Borne by Protein Methyltransferases: Rates and Equilibria of Non-enzymatic Methylation of Amino Acid Side Chains by SAM in Water. </span><span class="cited-content_cbyCitation_journal-name">Biochemistry</span><span> <strong>2021,</strong> <em>60 </em>
                                    (11)
                                     , 854-858. <a href="https://doi.org/10.1021/acs.biochem.1c00028" title="DOI URL">https://doi.org/10.1021/acs.biochem.1c00028</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.biochem.1c00028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.biochem.1c00028%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochemistry%26atitle%3DThe%252BBurden%252BBorne%252Bby%252BProtein%252BMethyltransferases%25253A%252BRates%252Band%252BEquilibria%252Bof%252BNon-enzymatic%252BMethylation%252Bof%252BAmino%252BAcid%252BSide%252BChains%252Bby%252BSAM%252Bin%252BWater%26aulast%3DLewis%26aufirst%3DCharles%2BA.%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D08012021%26date%3D27022021%26date%3D05032021%26volume%3D60%26issue%3D11%26spage%3D854%26epage%3D858" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Jian Lu, Aaron G. Bart, Qian Wu, Kevin R. Criscione, Michael J. McLeish, Emily E. Scott, <span class="NLM_string-name hlFld-ContribAuthor">Gary L. Grunewald</span>. </span><span class="cited-content_cbyCitation_article-title">Structure-Based Drug Design of Bisubstrate Inhibitors of Phenylethanolamine N-Methyltransferase Possessing Low Nanomolar Affinity at Both Substrate Binding Domains1. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (22)
                                     , 13878-13898. <a href="https://doi.org/10.1021/acs.jmedchem.0c01475" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01475</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01475&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01475%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DStructure-Based%252BDrug%252BDesign%252Bof%252BBisubstrate%252BInhibitors%252Bof%252BPhenylethanolamine%252BN-Methyltransferase%252BPossessing%252BLow%252BNanomolar%252BAffinity%252Bat%252BBoth%252BSubstrate%252BBinding%252BDomains1%26aulast%3DLu%26aufirst%3DJian%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D24082020%26date%3D04112020%26volume%3D63%26issue%3D22%26spage%3D13878%26epage%3D13898" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Niusha Mahmoodi, Rajesh K. Harijan, <span class="NLM_string-name hlFld-ContribAuthor">Vern L. Schramm</span>. </span><span class="cited-content_cbyCitation_article-title">Transition-State Analogues of Phenylethanolamine N-Methyltransferase. </span><span class="cited-content_cbyCitation_journal-name">Journal of the American Chemical Society</span><span> <strong>2020,</strong> <em>142 </em>
                                    (33)
                                     , 14222-14233. <a href="https://doi.org/10.1021/jacs.0c05446" title="DOI URL">https://doi.org/10.1021/jacs.0c05446</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jacs.0c05446&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjacs.0c05446%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520the%2520American%2520Chemical%2520Society%26atitle%3DTransition-State%252BAnalogues%252Bof%252BPhenylethanolamine%252BN-Methyltransferase%26aulast%3DMahmoodi%26aufirst%3DNiusha%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D18052020%26date%3D07082020%26date%3D24072020%26volume%3D142%26issue%3D33%26spage%3D14222%26epage%3D14233" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Dongxing Chen, Cheng Dong, Guangping Dong, Karthik Srinivasan, Jinrong Min, Nicholas Noinaj, <span class="NLM_string-name hlFld-ContribAuthor">Rong Huang</span>. </span><span class="cited-content_cbyCitation_article-title">Probing the Plasticity in the Active Site of Protein N-terminal Methyltransferase 1 Using Bisubstrate Analogues. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (15)
                                     , 8419-8431. <a href="https://doi.org/10.1021/acs.jmedchem.0c00770" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00770</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00770&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00770%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DProbing%252Bthe%252BPlasticity%252Bin%252Bthe%252BActive%252BSite%252Bof%252BProtein%252BN-terminal%252BMethyltransferase%252B1%252BUsing%252BBisubstrate%252BAnalogues%26aulast%3DChen%26aufirst%3DDongxing%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D06052020%26date%3D16072020%26date%3D01072020%26volume%3D63%26issue%3D15%26spage%3D8419%26epage%3D8431" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Ludovic Decultot, Rocco L. Policarpo, Brandon A. Wright, Danny Huang, <span class="NLM_string-name hlFld-ContribAuthor">Matthew D. Shair</span>. </span><span class="cited-content_cbyCitation_article-title">Asymmetric Total Synthesis of C9′-epi-Sinefungin. </span><span class="cited-content_cbyCitation_journal-name">Organic Letters</span><span> <strong>2020,</strong> <em>22 </em>
                                    (14)
                                     , 5594-5599. <a href="https://doi.org/10.1021/acs.orglett.0c01956" title="DOI URL">https://doi.org/10.1021/acs.orglett.0c01956</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.orglett.0c01956&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.orglett.0c01956%26sid%3Dliteratum%253Aachs%26jtitle%3DOrganic%2520Letters%26atitle%3DAsymmetric%252BTotal%252BSynthesis%252Bof%252BC9%2525E2%252580%2525B2-epi-Sinefungin%26aulast%3DDecultot%26aufirst%3DLudovic%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D10062020%26date%3D06072020%26volume%3D22%26issue%3D14%26spage%3D5594%26epage%3D5599" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Dongxing Chen, Linjie Li, Krystal Diaz, Iredia D. Iyamu, Ravi Yadav, Nicholas Noinaj, <span class="NLM_string-name hlFld-ContribAuthor">Rong Huang</span>. </span><span class="cited-content_cbyCitation_article-title">Novel Propargyl-Linked Bisubstrate Analogues as Tight-Binding Inhibitors for Nicotinamide N-Methyltransferase. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (23)
                                     , 10783-10797. <a href="https://doi.org/10.1021/acs.jmedchem.9b01255" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01255</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b01255%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DNovel%252BPropargyl-Linked%252BBisubstrate%252BAnalogues%252Bas%252BTight-Binding%252BInhibitors%252Bfor%252BNicotinamide%252BN-Methyltransferase%26aulast%3DChen%26aufirst%3DDongxing%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D31072019%26date%3D03122019%26date%3D14112019%26volume%3D62%26issue%3D23%26spage%3D10783%26epage%3D10797" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Juan C.  García-Cañaveras</span>, <span class="hlFld-ContribAuthor ">Agustín  Lahoz</span>. </span><span class="cited-content_cbyCitation_article-title">Tumor Microenvironment-Derived Metabolites: A Guide to Find New Metabolic Therapeutic Targets and Biomarkers. </span><span class="cited-content_cbyCitation_journal-name">Cancers</span><span> <strong>2021,</strong> <em>13 </em>
                                    (13)
                                     , 3230. <a href="https://doi.org/10.3390/cancers13133230" title="DOI URL">https://doi.org/10.3390/cancers13133230</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/cancers13133230&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fcancers13133230%26sid%3Dliteratum%253Aachs%26jtitle%3DCancers%26atitle%3DTumor%252BMicroenvironment-Derived%252BMetabolites%25253A%252BA%252BGuide%252Bto%252BFind%252BNew%252BMetabolic%252BTherapeutic%252BTargets%252Band%252BBiomarkers%26aulast%3DGarc%25C3%25ADa-Ca%25C3%25B1averas%26aufirst%3DJuan%2BC.%26date%3D2021%26date%3D2021%26volume%3D13%26issue%3D13%26spage%3D3230" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yongzhi  Gao</span>, <span class="hlFld-ContribAuthor ">Nathaniel I.  Martin</span>, <span class="hlFld-ContribAuthor ">Matthijs J.  van Haren</span>. </span><span class="cited-content_cbyCitation_article-title">Nicotinamide N-methyl transferase (NNMT): An emerging therapeutic target. </span><span class="cited-content_cbyCitation_journal-name">Drug Discovery Today</span><span> <strong>2021,</strong> <em>12 </em><a href="https://doi.org/10.1016/j.drudis.2021.05.011" title="DOI URL">https://doi.org/10.1016/j.drudis.2021.05.011</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.drudis.2021.05.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.drudis.2021.05.011%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Discovery%2520Today%26atitle%3DNicotinamide%252BN-methyl%252Btransferase%252B%252528NNMT%252529%25253A%252BAn%252Bemerging%252Btherapeutic%252Btarget%26aulast%3DGao%26aufirst%3DYongzhi%26date%3D2021%26volume%3D12" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Annalisa  Roberti</span>, <span class="hlFld-ContribAuthor ">Agustín F.  Fernández</span>, <span class="hlFld-ContribAuthor ">Mario F.  Fraga</span>. </span><span class="cited-content_cbyCitation_article-title">Nicotinamide N-methyltransferase: At the crossroads between cellular metabolism and epigenetic regulation. </span><span class="cited-content_cbyCitation_journal-name">Molecular Metabolism</span><span> <strong>2021,</strong> <em>45 </em>, 101165. <a href="https://doi.org/10.1016/j.molmet.2021.101165" title="DOI URL">https://doi.org/10.1016/j.molmet.2021.101165</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.molmet.2021.101165&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.molmet.2021.101165%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Metabolism%26atitle%3DNicotinamide%252BN-methyltransferase%25253A%252BAt%252Bthe%252Bcrossroads%252Bbetween%252Bcellular%252Bmetabolism%252Band%252Bepigenetic%252Bregulation%26aulast%3DRoberti%26aufirst%3DAnnalisa%26date%3D2021%26volume%3D45%26spage%3D101165" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Aimo  Kannt</span>, <span class="hlFld-ContribAuthor ">Sridharan  Rajagopal</span>, <span class="hlFld-ContribAuthor ">Mahanandeesha S.  Hallur</span>, <span class="hlFld-ContribAuthor ">Indu  Swamy</span>, <span class="hlFld-ContribAuthor ">Rajendra  Kristam</span>, <span class="hlFld-ContribAuthor ">Saravanakumar  Dhakshinamoorthy</span>, <span class="hlFld-ContribAuthor ">Joerg  Czech</span>, <span class="hlFld-ContribAuthor ">Gernot  Zech</span>, <span class="hlFld-ContribAuthor ">Herman  Schreuder</span>, <span class="hlFld-ContribAuthor ">Sven  Ruf</span>. </span><span class="cited-content_cbyCitation_article-title">Novel Inhibitors of Nicotinamide-N-Methyltransferase for the Treatment of Metabolic Disorders. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2021,</strong> <em>26 </em>
                                    (4)
                                     , 991. <a href="https://doi.org/10.3390/molecules26040991" title="DOI URL">https://doi.org/10.3390/molecules26040991</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules26040991&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules26040991%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DNovel%252BInhibitors%252Bof%252BNicotinamide-N-Methyltransferase%252Bfor%252Bthe%252BTreatment%252Bof%252BMetabolic%252BDisorders%26aulast%3DKannt%26aufirst%3DAimo%26date%3D2021%26date%3D2021%26volume%3D26%26issue%3D4%26spage%3D991" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Iredia D.  Iyamu</span>, <span class="hlFld-ContribAuthor ">Rong  Huang</span>. </span><span class="cited-content_cbyCitation_article-title">Mechanisms and inhibitors of nicotinamide
              N
              -methyltransferase. </span><span class="cited-content_cbyCitation_journal-name">RSC Medicinal Chemistry</span><span> <strong>2021,</strong> <em>28 </em><a href="https://doi.org/10.1039/D1MD00016K" title="DOI URL">https://doi.org/10.1039/D1MD00016K</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D1MD00016K&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD1MD00016K%26sid%3Dliteratum%253Aachs%26jtitle%3DRSC%2520Medicinal%2520Chemistry%26atitle%3DMechanisms%252Band%252Binhibitors%252Bof%252Bnicotinamide%252BN%252B-methyltransferase%26aulast%3DIyamu%26aufirst%3DIredia%2BD.%26date%3D2021%26date%3D2021%26volume%3D28" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Petra  Krafcikova</span>, <span class="hlFld-ContribAuthor ">Jan  Silhan</span>, <span class="hlFld-ContribAuthor ">Radim  Nencka</span>, <span class="hlFld-ContribAuthor ">Evzen  Boura</span>. </span><span class="cited-content_cbyCitation_article-title">Structural analysis of the SARS-CoV-2 methyltransferase complex involved in RNA cap creation bound to sinefungin. </span><span class="cited-content_cbyCitation_journal-name">Nature Communications</span><span> <strong>2020,</strong> <em>11 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s41467-020-17495-9" title="DOI URL">https://doi.org/10.1038/s41467-020-17495-9</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41467-020-17495-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41467-020-17495-9%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Communications%26atitle%3DStructural%252Banalysis%252Bof%252Bthe%252BSARS-CoV-2%252Bmethyltransferase%252Bcomplex%252Binvolved%252Bin%252BRNA%252Bcap%252Bcreation%252Bbound%252Bto%252Bsinefungin%26aulast%3DKrafcikova%26aufirst%3DPetra%26date%3D2020%26date%3D2020%26volume%3D11%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Iredia D.  Iyamu</span>, <span class="hlFld-ContribAuthor ">Rong  Huang</span>. </span><span class="cited-content_cbyCitation_article-title">Development of fluorescence polarization-based competition assay for nicotinamide N-methyltransferase. </span><span class="cited-content_cbyCitation_journal-name">Analytical Biochemistry</span><span> <strong>2020,</strong> <em>604 </em>, 113833. <a href="https://doi.org/10.1016/j.ab.2020.113833" title="DOI URL">https://doi.org/10.1016/j.ab.2020.113833</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ab.2020.113833&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ab.2020.113833%26sid%3Dliteratum%253Aachs%26jtitle%3DAnalytical%2520Biochemistry%26atitle%3DDevelopment%252Bof%252Bfluorescence%252Bpolarization-based%252Bcompetition%252Bassay%252Bfor%252Bnicotinamide%252BN-methyltransferase%26aulast%3DIyamu%26aufirst%3DIredia%2BD.%26date%3D2020%26volume%3D604%26spage%3D113833" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yanyan  Cui</span>, <span class="hlFld-ContribAuthor ">Dawei  Yang</span>, <span class="hlFld-ContribAuthor ">Wenjie  Wang</span>, <span class="hlFld-ContribAuthor ">Luyu  Zhang</span>, <span class="hlFld-ContribAuthor ">Hongtao  Liu</span>, <span class="hlFld-ContribAuthor ">Shanshan  Ma</span>, <span class="hlFld-ContribAuthor ">Wenna  Guo</span>, <span class="hlFld-ContribAuthor ">Minghao  Yao</span>, <span class="hlFld-ContribAuthor ">Kun  Zhang</span>, <span class="hlFld-ContribAuthor ">Wencai  Li</span>, <span class="hlFld-ContribAuthor ">Yanting  Zhang</span>, <span class="hlFld-ContribAuthor ">Fangxia  Guan</span>. </span><span class="cited-content_cbyCitation_article-title">Nicotinamide N‐methyltransferase decreases 5‐fluorouracil sensitivity in human esophageal squamous cell carcinoma through metabolic reprogramming and promoting the Warburg effect. </span><span class="cited-content_cbyCitation_journal-name">Molecular Carcinogenesis</span><span> <strong>2020,</strong> <em>59 </em>
                                    (8)
                                     , 940-954. <a href="https://doi.org/10.1002/mc.23209" title="DOI URL">https://doi.org/10.1002/mc.23209</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/mc.23209&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fmc.23209%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Carcinogenesis%26atitle%3DNicotinamide%252BN%2525E2%252580%252590methyltransferase%252Bdecreases%252B5%2525E2%252580%252590fluorouracil%252Bsensitivity%252Bin%252Bhuman%252Besophageal%252Bsquamous%252Bcell%252Bcarcinoma%252Bthrough%252Bmetabolic%252Breprogramming%252Band%252Bpromoting%252Bthe%252BWarburg%252Beffect%26aulast%3DCui%26aufirst%3DYanyan%26date%3D2020%26date%3D2020%26volume%3D59%26issue%3D8%26spage%3D940%26epage%3D954" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Panupong  Mahalapbutr</span>, <span class="hlFld-ContribAuthor ">Napat  Kongtaworn</span>, <span class="hlFld-ContribAuthor ">Thanyada  Rungrotmongkol</span>. </span><span class="cited-content_cbyCitation_article-title">Structural insight into the recognition of S-adenosyl-L-homocysteine and sinefungin in SARS-CoV-2 Nsp16/Nsp10 RNA cap 2′-O-Methyltransferase. </span><span class="cited-content_cbyCitation_journal-name">Computational and Structural Biotechnology Journal</span><span> <strong>2020,</strong> <em>18 </em>, 2757-2765. <a href="https://doi.org/10.1016/j.csbj.2020.09.032" title="DOI URL">https://doi.org/10.1016/j.csbj.2020.09.032</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.csbj.2020.09.032&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.csbj.2020.09.032%26sid%3Dliteratum%253Aachs%26jtitle%3DComputational%2520and%2520Structural%2520Biotechnology%2520Journal%26atitle%3DStructural%252Binsight%252Binto%252Bthe%252Brecognition%252Bof%252BS-adenosyl-L-homocysteine%252Band%252Bsinefungin%252Bin%252BSARS-CoV-2%252BNsp16%25252FNsp10%252BRNA%252Bcap%252B2%2525E2%252580%2525B2-O-Methyltransferase%26aulast%3DMahalapbutr%26aufirst%3DPanupong%26date%3D2020%26volume%3D18%26spage%3D2757%26epage%3D2765" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01238/20191108/images/medium/jm9b01238_0008.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01238/20191108/images/large/jm9b01238_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01238&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01238/20191108/images/medium/jm9b01238_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01238/20191108/images/large/jm9b01238_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Overview of the alkynyl bisubstrate strategy. (a) Binding pose of ligands SAH and NAM (teal) rendered from previously published cocrystal structure (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ROD">3ROD</a>). (b) Graphical representation of the SAM → NAM methyl transfer reaction catalyzed by NNMT. (c) Graphical representation of alkynyl bisubstrate inhibitor NS1 posed to mimic the binding geometry of SAM/NAM. (d) Molecular docking output pose of alkynyl bisubstrate inhibitor NS1 (black) overlaid with SAH and NAM from structure <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ROD">3ROD</a> (teal).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01238/20191108/images/large/jm9b01238_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01238&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01238/20191108/images/medium/jm9b01238_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01238/20191108/images/large/jm9b01238_0007.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis and X-ray Crystal Structure of NS1 (<b>10</b>)</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01238/20191108/images/large/jm9b01238_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01238&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01238/20191108/images/medium/jm9b01238_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01238/20191108/images/large/jm9b01238_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Kinetic studies of NNMT. A time course of the NNMT kinetic assay in the absence (red circles) or in the presence (remaining symbols) of NS1. Inhibitor concentrations are shown at top right. Δ<i>F</i> are scaled fluorescence changes in relative fluorescence units (“rfu”). The smooth curves represent the best least-squares fit to a system of first-order differential equations automatically generated by the DynaFit software package<a onclick="showRef(event, 'ref42 ref43'); return false;" href="javascript:void(0);" class="ref ref42 ref43">(42,43)</a> according to the reaction mechanism shown in the Supporting Information, <a class="ref internalNav" href="#notes1" aria-label="Part 3, Enzyme Kinetics)">Part 3, Enzyme Kinetics)</a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01238/20191108/images/large/jm9b01238_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01238&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01238/20191108/images/medium/jm9b01238_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01238/20191108/images/large/jm9b01238_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Cocrystal structure of NS1-bound NNMT (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ORR">6ORR</a>). (a) A 2<i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> omit map contoured at 1σ reveals NS1-shaped electron density in the binding pocket of NNMT. (b) NS1 (black) binds NNMT in an orientation that closely resembles a previously published cocrystal structure containing ligands SAH and NAM (cyan; PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ROD">3ROD</a>). (c–e) NS1 also exploits the same protein interactions that NNMT uses to bind its native substrates. Ligand-interacting residues from NNMT are shown in yellow; black dotted lines represent hydrogen bonds. (f) MS2756-bound NNMT (brown; PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6B1A">6B1A</a>) superimposed on NS1-bound NNMT.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01238/20191108/images/large/jm9b01238_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01238&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01238/20191108/images/medium/jm9b01238_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01238/20191108/images/large/jm9b01238_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Structure–activity relationship (SAR) studies of bisubstrate NNMT inhibitors. p<i>K</i><sub>i</sub> values (−log<sub>10</sub><i>K</i><sub>i</sub>) are shown in red and are used to facilitate comparison between inhibitors of widely varying potency. (a) Exploration of the alkynyl bisubstrate approach to NNMT inhibition. (b) Amino acid modifications.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01238/20191108/images/large/jm9b01238_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01238&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01238/20191108/images/medium/jm9b01238_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01238/20191108/images/large/jm9b01238_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. SAR studies of bisubstrate inhibitors. p<i>K</i><sub>i</sub> values (−log<sub>10</sub><i>K</i><sub><i>i</i></sub>) are shown in red and are used to facilitate comparison between inhibitors of widely varying potency. (a) Modification of the benzamide moiety of NS1. (b) Previously published amino-bisubstrate NNMT inhibitors were synthesized and tested alongside NS1 and analogues.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01238/20191108/images/large/jm9b01238_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01238&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01238/20191108/images/medium/jm9b01238_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01238/20191108/images/large/jm9b01238_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Chemical properties and geometry of NS1 closely match the hydrophobic profile and shape of the NNMT binding pocket. (a) The binding pocket is buried in the hydrophobic core of the protein (gray). Inset: A solvent-accessible surface representation of the binding pocket colored by the Eisenberg hydrophobicity scale<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> reveals a hydrophobic patch (dark red) that contacts the C6′ carbon atom of NS1. (b) A side-on view shows that the binding pocket has shape complementarity specific to the <i>S</i> diastereomer of NS1 at the C6′ carbon.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01238/20191108/images/large/jm9b01238_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01238&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i163">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_34321" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_34321" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 83 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="note"><p class="first last">Currently, there are many abbreviations used for nicotinamide and 1-methylnicotinamide in the literature, leading to inconsistency and confusion. We use the standardized nomenclature proposed by Pissios in a 2017 review article on NNMT (see following citation).</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pissios, P.</span></span> <span> </span><span class="NLM_article-title">Nicotinamide N-methyltransferase: more than a vitamin B3 clearance enzyme</span>. <i>Trends Endocrinol. Metab.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>28</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">340</span>– <span class="NLM_lpage">353</span>, <span class="refDoi"> DOI: 10.1016/j.tem.2017.02.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.1016%2Fj.tem.2017.02.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=28291578" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BC2sXivFehsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2017&pages=340-353&issue=5&author=P.+Pissios&title=Nicotinamide+N-methyltransferase%3A+more+than+a+vitamin+B3+clearance+enzyme&doi=10.1016%2Fj.tem.2017.02.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Nicotinamide N-Methyltransferase: More Than a Vitamin B3 Clearance Enzyme</span></div><div class="casAuthors">Pissios, Pavlos</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Endocrinology and Metabolism</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">340-353</span>CODEN:
                <span class="NLM_cas:coden">TENME4</span>;
        ISSN:<span class="NLM_cas:issn">1043-2760</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Nicotinamide (NAM) N-methyltransferase (NNMT) was originally identified as the enzyme responsible for the methylation of NAM, one of the forms of vitamin B3.  Methylated NAM is eventually excreted from the body.  Recent evidence has expanded the role of NNMT beyond clearance of excess vitamin B3.  NNMT has been implicated in the regulation of multiple metabolic pathways in tissues such as adipose and liver as well as cancer cells through the consumption of Me donors and generation of active metabolites.  This review examines recent findings regarding the function of NNMT in physiol. and disease and highlights potential new avenues for therapeutic intervention.  Finally, key gaps in our knowledge about this enzymic system and future areas of investigation are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHRx9vhXz5LbVg90H21EOLACvtfcHk0lhi32QN0HbBww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXivFehsrk%253D&md5=cf088bcc9434e006062d72de0c47c62a</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.tem.2017.02.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tem.2017.02.004%26sid%3Dliteratum%253Aachs%26aulast%3DPissios%26aufirst%3DP.%26atitle%3DNicotinamide%2520N-methyltransferase%253A%2520more%2520than%2520a%2520vitamin%2520B3%2520clearance%2520enzyme%26jtitle%3DTrends%2520Endocrinol.%2520Metab.%26date%3D2017%26volume%3D28%26issue%3D5%26spage%3D340%26epage%3D353%26doi%3D10.1016%2Fj.tem.2017.02.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hong, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhai, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pissios, P.</span></span> <span> </span><span class="NLM_article-title">Nicotinamide N-methyltransferase interacts with enzymes of the methionine cycle and regulates methyl donor metabolism</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>57</i></span> (<span class="NLM_issue">40</span>),  <span class="NLM_fpage">5775</span>– <span class="NLM_lpage">5779</span>, <span class="refDoi"> DOI: 10.1021/acs.biochem.8b00561</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.biochem.8b00561" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslaqtr7O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2018&pages=5775-5779&issue=40&author=S.+Hongauthor=B.+Zhaiauthor=P.+Pissios&title=Nicotinamide+N-methyltransferase+interacts+with+enzymes+of+the+methionine+cycle+and+regulates+methyl+donor+metabolism&doi=10.1021%2Facs.biochem.8b00561"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Nicotinamide N-methyltransferase interacts with enzymes of the methionine cycle and regulates methyl donor metabolism</span></div><div class="casAuthors">Hong, Shangyu; Zhai, Bo; Pissios, Pavlos</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">40</span>),
    <span class="NLM_cas:pages">5775-5779</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Me donor balance is crit. for epigenetic regulation in cells and is maintained by the so-called methionine cycle proteins that regenerate S-adenosylmethionine (SAM), the universal Me donor, from homocysteine formed by the activity of methyltransferases.  Nnmt is a liver enzyme that methylates nicotinamide, but its role in regulating Me donor balance in the liver is unclear.  In this study, we assessed the effect of altered Nnmt expression on various aspects of Me donor metab. in the liver.  We found that Nnmt overexpression decreased SAM levels and the SAM/ S-adenosylhomocysteine (SAH) ratio both in vivo and in vitro.  Nnmt knockdown did not change Me donor balance in mouse primary hepatocytes but increased SAM levels and the SAM/SAH ratio when Gnmt, the dominantly expressed methyltransferase in liver, was simultaneously knocked down.  Paradoxically, expression of enzymically deficient Nnmt increased the SAM/SAH ratio, suggesting that Nnmt can regulate Me donor balance independent of its methyltransferase activity.  Proteomics anal. of Nnmt-interacting proteins in the liver identified Bhmt, Mat1a, and Ahcy, all components of the methionine cycle, and functional expts. showed that mutant Nnmt increased the level of remethylation of homocysteine to SAM.  In summary, we show that the function of Nnmt in hepatic Me donor balance is multifactorial.  On one hand, Nnmt decreases Me donor balance, consistent with its activity as a methyltransferase consuming Me donors.  On the other hand, by co-opting the enzymes of the methionine cycle, Nnmt aids the recycling of homocysteine to SAM for another round of methylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqD5fiz3fTxsrVg90H21EOLACvtfcHk0lhi32QN0HbBww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslaqtr7O&md5=2689739ce6ec8ef59b09af7a20cb06ce</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1021%2Facs.biochem.8b00561&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.biochem.8b00561%26sid%3Dliteratum%253Aachs%26aulast%3DHong%26aufirst%3DS.%26aulast%3DZhai%26aufirst%3DB.%26aulast%3DPissios%26aufirst%3DP.%26atitle%3DNicotinamide%2520N-methyltransferase%2520interacts%2520with%2520enzymes%2520of%2520the%2520methionine%2520cycle%2520and%2520regulates%2520methyl%2520donor%2520metabolism%26jtitle%3DBiochemistry%26date%3D2018%26volume%3D57%26issue%3D40%26spage%3D5775%26epage%3D5779%26doi%3D10.1021%2Facs.biochem.8b00561" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hong, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moreno-Navarrete, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kikukawa, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tzameli, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prasad, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asara, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandez-Real, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maratos-Flier, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pissios, P.</span></span> <span> </span><span class="NLM_article-title">Nicotinamide N-methyltransferase regulates hepatic nutrient metabolism through Sirt1 protein stabilization</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>21</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">887</span>– <span class="NLM_lpage">894</span>, <span class="refDoi"> DOI: 10.1038/nm.3882</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.1038%2Fnm.3882" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=26168293" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1WltLjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2015&pages=887-894&issue=8&author=S.+Hongauthor=J.+M.+Moreno-Navarreteauthor=X.+Weiauthor=Y.+Kikukawaauthor=I.+Tzameliauthor=D.+Prasadauthor=Y.+Leeauthor=J.+M.+Asaraauthor=J.+M.+Fernandez-Realauthor=E.+Maratos-Flierauthor=P.+Pissios&title=Nicotinamide+N-methyltransferase+regulates+hepatic+nutrient+metabolism+through+Sirt1+protein+stabilization&doi=10.1038%2Fnm.3882"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Nicotinamide N-methyltransferase regulates hepatic nutrient metabolism through Sirt1 protein stabilization</span></div><div class="casAuthors">Hong, Shangyu; Moreno-Navarrete, Jose M.; Wei, Xiaojing; Kikukawa, Yusuke; Tzameli, Iphigenia; Prasad, Deepthi; Lee, Yoonjin; Asara, John M.; Fernandez-Real, Jose Manuel; Maratos-Flier, Eleftheria; Pissios, Pavlos</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">887-894</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Nicotinamide N-methyltransferase (Nnmt) methylates nicotinamide, a form of vitamin B3, to produce N1-methylnicotinamide (MNAM).  Nnmt has emerged as a metabolic regulator in adipocytes, but its role in the liver, the tissue with the strongest Nnmt expression, is not known.  In spite of its overall high expression, here we find that hepatic expression of Nnmt is highly variable and correlates with multiple metabolic parameters in mice and humans.  Further, we find that suppression of hepatic Nnmt expression in vivo alters glucose and cholesterol metab. and that the metabolic effects of Nnmt in the liver are mediated by its product MNAM.  Supplementation of high-fat diet with MNAM decreases serum and liver cholesterol and liver triglycerides levels in mice.  Mechanistically, increasing Nnmt expression or MNAM levels stabilizes sirtuin 1 protein, an effect that is required for their metabolic benefits.  In summary, we describe here a novel regulatory pathway for vitamin B3 that could provide a new opportunity for metabolic disease therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpaiKWlS3XZlrVg90H21EOLACvtfcHk0lhi32QN0HbBww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1WltLjO&md5=d3f54ad774f4ea0bf07ce9bb389c0139</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fnm.3882&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.3882%26sid%3Dliteratum%253Aachs%26aulast%3DHong%26aufirst%3DS.%26aulast%3DMoreno-Navarrete%26aufirst%3DJ.%2BM.%26aulast%3DWei%26aufirst%3DX.%26aulast%3DKikukawa%26aufirst%3DY.%26aulast%3DTzameli%26aufirst%3DI.%26aulast%3DPrasad%26aufirst%3DD.%26aulast%3DLee%26aufirst%3DY.%26aulast%3DAsara%26aufirst%3DJ.%2BM.%26aulast%3DFernandez-Real%26aufirst%3DJ.%2BM.%26aulast%3DMaratos-Flier%26aufirst%3DE.%26aulast%3DPissios%26aufirst%3DP.%26atitle%3DNicotinamide%2520N-methyltransferase%2520regulates%2520hepatic%2520nutrient%2520metabolism%2520through%2520Sirt1%2520protein%2520stabilization%26jtitle%3DNat.%2520Med.%26date%3D2015%26volume%3D21%26issue%3D8%26spage%3D887%26epage%3D894%26doi%3D10.1038%2Fnm.3882" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ulanovskaya, O. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zuhl, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B. F.</span></span> <span> </span><span class="NLM_article-title">NNMT promotes epigenetic remodeling in cancer by creating a metabolic methylation sink</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>9</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">300</span>– <span class="NLM_lpage">306</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1204</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.1038%2Fnchembio.1204" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=23455543" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjtlKnsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=300-306&issue=5&author=O.+A.+Ulanovskayaauthor=A.+M.+Zuhlauthor=B.+F.+Cravatt&title=NNMT+promotes+epigenetic+remodeling+in+cancer+by+creating+a+metabolic+methylation+sink&doi=10.1038%2Fnchembio.1204"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">NNMT promotes epigenetic remodeling in cancer by creating a metabolic methylation sink</span></div><div class="casAuthors">Ulanovskaya, Olesya A.; Zuhl, Andrea M.; Cravatt, Benjamin F.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">300-306</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Nicotinamide N-methyltransferase (NNMT) is overexpressed in a variety of human cancers, where it contributes to tumorigenesis by a mechanism that is still poorly understood.  Here we show using metabolomics that NNMT impairs the methylation potential of cancer cells by consuming Me units from S-adenosyl methionine to create the stable metabolic product 1-methylnicotinamide.  As a result, NNMT-expressing cancer cells have an altered epigenetic state that includes hypomethylated histones and other cancer-related proteins combined with heightened expression of protumorigenic gene products.  Our findings thus point to a direct mechanistic link between the deregulation of a metabolic enzyme and widespread changes in the methylation landscape of cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrsOQPNFxw5kbVg90H21EOLACvtfcHk0lgboGxRGTbZog"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjtlKnsbs%253D&md5=8775d8b27fba0d039b0f6da7244d28a4</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1204&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1204%26sid%3Dliteratum%253Aachs%26aulast%3DUlanovskaya%26aufirst%3DO.%2BA.%26aulast%3DZuhl%26aufirst%3DA.%2BM.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26atitle%3DNNMT%2520promotes%2520epigenetic%2520remodeling%2520in%2520cancer%2520by%2520creating%2520a%2520metabolic%2520methylation%2520sink%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2013%26volume%3D9%26issue%3D5%26spage%3D300%26epage%3D306%26doi%3D10.1038%2Fnchembio.1204" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sperber, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mathieu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferreccio, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hesson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Devi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Detraux, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Battle, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Showalter, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valensisi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bielas, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ericson, N. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Margaretha, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robitaille, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Margineantu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fiehn, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hockenbery, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blau, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raftery, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Margolin, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hawkins, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moon, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ware, C. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruohola-Baker, H.</span></span> <span> </span><span class="NLM_article-title">The metabolome regulates the epigenetic landscape during naive-to-primed human embryonic stem cell transition</span>. <i>Nat. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>17</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">1523</span>– <span class="NLM_lpage">1535</span>, <span class="refDoi"> DOI: 10.1038/ncb3264</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.1038%2Fncb3264" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=26571212" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVOgtLfL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2015&pages=1523-1535&issue=12&author=H.+Sperberauthor=J.+Mathieuauthor=Y.+Wangauthor=A.+Ferreccioauthor=J.+Hessonauthor=Z.+Xuauthor=K.+A.+Fischerauthor=A.+Deviauthor=D.+Detrauxauthor=H.+Guauthor=S.+L.+Battleauthor=M.+Showalterauthor=C.+Valensisiauthor=J.+H.+Bielasauthor=N.+G.+Ericsonauthor=L.+Margarethaauthor=A.+M.+Robitailleauthor=D.+Margineantuauthor=O.+Fiehnauthor=D.+Hockenberyauthor=C.+A.+Blauauthor=D.+Rafteryauthor=A.+A.+Margolinauthor=R.+D.+Hawkinsauthor=R.+T.+Moonauthor=C.+B.+Wareauthor=H.+Ruohola-Baker&title=The+metabolome+regulates+the+epigenetic+landscape+during+naive-to-primed+human+embryonic+stem+cell+transition&doi=10.1038%2Fncb3264"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">The metabolome regulates the epigenetic landscape during naive-to-primed human embryonic stem cell transition</span></div><div class="casAuthors">Sperber, Henrik; Mathieu, Julie; Wang, Yuliang; Ferreccio, Amy; Hesson, Jennifer; Xu, Zhuojin; Fischer, Karin A.; Devi, Arikketh; Detraux, Damien; Gu, Haiwei; Battle, Stephanie L.; Showalter, Megan; Valensisi, Cristina; Bielas, Jason H.; Ericson, Nolan G.; Margaretha, Lilyana; Robitaille, Aaron M.; Margineantu, Daciana; Fiehn, Oliver; Hockenbery, David; Blau, C. Anthony; Raftery, Daniel; Margolin, Adam A.; Hawkins, R. David; Moon, Randall T.; Ware, Carol B.; Ruohola-Baker, Hannele</div><div class="citationInfo"><span class="NLM_cas:title">Nature Cell Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1523-1535</span>CODEN:
                <span class="NLM_cas:coden">NCBIFN</span>;
        ISSN:<span class="NLM_cas:issn">1465-7392</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">For nearly a century developmental biologists have recognized that cells from embryos can differ in their potential to differentiate into distinct cell types.  Recently, it has been recognized that embryonic stem cells derived from both mice and humans exhibit two stable yet epigenetically distinct states of pluripotency: naive and primed.  We now show that nicotinamide N-methyltransferase (NNMT) and the metabolic state regulate pluripotency in human embryonic stem cells (hESCs).  Specifically, in naive hESCs, NNMT and its enzymic product 1-methylnicotinamide are highly upregulated, and NNMT is required for low S-adenosyl methionine (SAM) levels and the H3K27me3 repressive state.  NNMT consumes SAM in naive cells, making it unavailable for histone methylation that represses Wnt and activates the HIF pathway in primed hESCs.  These data support the hypothesis that the metabolome regulates the epigenetic landscape of the earliest steps in human development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqaKx0pxyCNf7Vg90H21EOLACvtfcHk0lgboGxRGTbZog"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVOgtLfL&md5=2a9a212ef237b2790832ae423f47b4aa</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fncb3264&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncb3264%26sid%3Dliteratum%253Aachs%26aulast%3DSperber%26aufirst%3DH.%26aulast%3DMathieu%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DFerreccio%26aufirst%3DA.%26aulast%3DHesson%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DZ.%26aulast%3DFischer%26aufirst%3DK.%2BA.%26aulast%3DDevi%26aufirst%3DA.%26aulast%3DDetraux%26aufirst%3DD.%26aulast%3DGu%26aufirst%3DH.%26aulast%3DBattle%26aufirst%3DS.%2BL.%26aulast%3DShowalter%26aufirst%3DM.%26aulast%3DValensisi%26aufirst%3DC.%26aulast%3DBielas%26aufirst%3DJ.%2BH.%26aulast%3DEricson%26aufirst%3DN.%2BG.%26aulast%3DMargaretha%26aufirst%3DL.%26aulast%3DRobitaille%26aufirst%3DA.%2BM.%26aulast%3DMargineantu%26aufirst%3DD.%26aulast%3DFiehn%26aufirst%3DO.%26aulast%3DHockenbery%26aufirst%3DD.%26aulast%3DBlau%26aufirst%3DC.%2BA.%26aulast%3DRaftery%26aufirst%3DD.%26aulast%3DMargolin%26aufirst%3DA.%2BA.%26aulast%3DHawkins%26aufirst%3DR.%2BD.%26aulast%3DMoon%26aufirst%3DR.%2BT.%26aulast%3DWare%26aufirst%3DC.%2BB.%26aulast%3DRuohola-Baker%26aufirst%3DH.%26atitle%3DThe%2520metabolome%2520regulates%2520the%2520epigenetic%2520landscape%2520during%2520naive-to-primed%2520human%2520embryonic%2520stem%2520cell%2520transition%26jtitle%3DNat.%2520Cell%2520Biol.%26date%3D2015%26volume%3D17%26issue%3D12%26spage%3D1523%26epage%3D1535%26doi%3D10.1038%2Fncb3264" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kraus, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banks, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodgers, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pirinen, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pulinilkunnil, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gong, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sauve, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asara, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peroni, O. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monia, B. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhanot, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alhonen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Puigserver, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kahn, B. B.</span></span> <span> </span><span class="NLM_article-title">Nicotinamide N-methyltransferase knockdown protects against diet-induced obesity</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>508</i></span> (<span class="NLM_issue">7495</span>),  <span class="NLM_fpage">258</span>– <span class="NLM_lpage">262</span>, <span class="refDoi"> DOI: 10.1038/nature13198</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.1038%2Fnature13198" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=24717514" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmtlWhsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=508&publication_year=2014&pages=258-262&issue=7495&author=D.+Krausauthor=Q.+Yangauthor=D.+Kongauthor=A.+S.+Banksauthor=L.+Zhangauthor=J.+T.+Rodgersauthor=E.+Pirinenauthor=T.+C.+Pulinilkunnilauthor=F.+Gongauthor=Y.+C.+Wangauthor=Y.+Cenauthor=A.+A.+Sauveauthor=J.+M.+Asaraauthor=O.+D.+Peroniauthor=B.+P.+Moniaauthor=S.+Bhanotauthor=L.+Alhonenauthor=P.+Puigserverauthor=B.+B.+Kahn&title=Nicotinamide+N-methyltransferase+knockdown+protects+against+diet-induced+obesity&doi=10.1038%2Fnature13198"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Nicotinamide N-methyltransferase knockdown protects against diet-induced obesity</span></div><div class="casAuthors">Kraus, Daniel; Yang, Qin; Kong, Dong; Banks, Alexander S.; Zhang, Lin; Rodgers, Joseph T.; Pirinen, Eija; Pulinilkunnil, Thomas C.; Gong, Fengying; Wang, Ya-chin; Cen, Yana; Sauve, Anthony A.; Asara, John M.; Peroni, Odile D.; Monia, Brett P.; Bhanot, Sanjay; Alhonen, Leena; Puigserver, Pere; Kahn, Barbara B.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">508</span>
        (<span class="NLM_cas:issue">7495</span>),
    <span class="NLM_cas:pages">258-262</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">In obesity and type 2 diabetes, Glut4 glucose transporter expression is decreased selectively in adipocytes.  Adipose-specific knockout or overexpression of Glut4 alters systemic insulin sensitivity.  Here we show, using DNA array analyses, that nicotinamide N-methyltransferase (Nnmt) is the most strongly reciprocally regulated gene when comparing gene expression in white adipose tissue (WAT) from adipose-specific Glut4-knockout or adipose-specific Glut4-overexpressing mice with their resp. controls.  NNMT methylates nicotinamide (vitamin B3) using S-adenosylmethionine (SAM) as a Me donor.  Nicotinamide is a precursor of NAD+, an important cofactor linking cellular redox states with energy metab.  SAM provides propylamine for polyamine biosynthesis and donates a Me group for histone methylation.  Polyamine flux including synthesis, catabolism and excretion, is controlled by the rate-limiting enzymes ornithine decarboxylase (ODC) and spermidine-spermine N1-acetyltransferase (SSAT; encoded by Sat1) and by polyamine oxidase (PAO), and has a major role in energy metab.  We report that NNMT expression is increased in WAT and liver of obese and diabetic mice.  Nnmt knockdown in WAT and liver protects against diet-induced obesity by augmenting cellular energy expenditure.  NNMT inhibition increases adipose SAM and NAD+ levels and upregulates ODC and SSAT activity as well as expression, owing to the effects of NNMT on histone H3 lysine 4 methylation in adipose tissue.  Direct evidence for increased polyamine flux resulting from NNMT inhibition includes elevated urinary excretion and adipocyte secretion of diacetylspermine, a product of polyamine metab.  NNMT inhibition in adipocytes increases oxygen consumption in an ODC-, SSAT- and PAO-dependent manner.  Thus, NNMT is a novel regulator of histone methylation, polyamine flux and NAD+-dependent SIRT1 signaling, and is a unique and attractive target for treating obesity and type 2 diabetes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKuWPUvPnIqLVg90H21EOLACvtfcHk0lgboGxRGTbZog"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmtlWhsL8%253D&md5=c37b6137cd4537a0f6f1701a5a212dd6</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fnature13198&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature13198%26sid%3Dliteratum%253Aachs%26aulast%3DKraus%26aufirst%3DD.%26aulast%3DYang%26aufirst%3DQ.%26aulast%3DKong%26aufirst%3DD.%26aulast%3DBanks%26aufirst%3DA.%2BS.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DRodgers%26aufirst%3DJ.%2BT.%26aulast%3DPirinen%26aufirst%3DE.%26aulast%3DPulinilkunnil%26aufirst%3DT.%2BC.%26aulast%3DGong%26aufirst%3DF.%26aulast%3DWang%26aufirst%3DY.%2BC.%26aulast%3DCen%26aufirst%3DY.%26aulast%3DSauve%26aufirst%3DA.%2BA.%26aulast%3DAsara%26aufirst%3DJ.%2BM.%26aulast%3DPeroni%26aufirst%3DO.%2BD.%26aulast%3DMonia%26aufirst%3DB.%2BP.%26aulast%3DBhanot%26aufirst%3DS.%26aulast%3DAlhonen%26aufirst%3DL.%26aulast%3DPuigserver%26aufirst%3DP.%26aulast%3DKahn%26aufirst%3DB.%2BB.%26atitle%3DNicotinamide%2520N-methyltransferase%2520knockdown%2520protects%2520against%2520diet-induced%2520obesity%26jtitle%3DNature%26date%3D2014%26volume%3D508%26issue%3D7495%26spage%3D258%26epage%3D262%26doi%3D10.1038%2Fnature13198" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brachs, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polack, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brachs, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jahn-Hofmann, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elvert, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pfenninger, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barenz, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Margerie, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mai, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spranger, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kannt, A.</span></span> <span> </span><span class="NLM_article-title">Genetic nicotinamide N-methyltransferase (Nnmt) deficiency in male mice improves insulin sensitivity in diet-induced obesity but does not affect glucose tolerance</span>. <i>Diabetes</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>68</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">527</span>– <span class="NLM_lpage">542</span>, <span class="refDoi"> DOI: 10.2337/db18-0780</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.2337%2Fdb18-0780" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=30552109" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BC1MXptlSmtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2019&pages=527-542&issue=3&author=S.+Brachsauthor=J.+Polackauthor=M.+Brachsauthor=K.+Jahn-Hofmannauthor=R.+Elvertauthor=A.+Pfenningerauthor=F.+Barenzauthor=D.+Margerieauthor=K.+Maiauthor=J.+Sprangerauthor=A.+Kannt&title=Genetic+nicotinamide+N-methyltransferase+%28Nnmt%29+deficiency+in+male+mice+improves+insulin+sensitivity+in+diet-induced+obesity+but+does+not+affect+glucose+tolerance&doi=10.2337%2Fdb18-0780"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Genetic nicotinamide N-methyltransferase (Nnmt) deficiency in male mice improves insulin sensitivity in diet-induced obesity but does not affect glucose tolerance</span></div><div class="casAuthors">Brachs, Sebastian; Polack, James; Brachs, Maria; Jahn-Hofrnann, Kerstin; Evert, Ralf; Pfenninger, Anja; Barenz, Felix; Margerie, Daniel; Mai, Knut; Spranger, Joachim; Kannt, Aimo</div><div class="citationInfo"><span class="NLM_cas:title">Diabetes</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">527-542</span>CODEN:
                <span class="NLM_cas:coden">DIAEAZ</span>;
        ISSN:<span class="NLM_cas:issn">0012-1797</span>.
    
            (<span class="NLM_cas:orgname">American Diabetes Association</span>)
        </div><div class="casAbstract">Antisense oligonucleotide knockdown (ASO-KD) of nicotinamide N-methyltransferase (NNMT) in high-fat diet (HFD)-fed mice has been reported to reduce wt. gain and plasma insulin levels and to improve glucose toler- ance.  Using NNMT-ASO-KD or NNMT knockout mice (NNMT-/-), we tested the hypothesis that Nnmt deletion protects against diet-induced obesity and its metabolic consequences in males and females on obesity-inducing diets.  We also examd. samples from a human wt. redn. (WR) study for adipose NNMT (aNNMT) expression and plasma 1 -methylnicotinarnide (MNAM) levels.  In Western diet (WD)-fed female mice, NNMT- ASO-KD reduced body wt., fat mass, and insulin level and improved glucose tolerance.  Although NNMT-/- mice fed a std. diet had no obvious phenotype, NNMT-/- males fed an HFD showed strongly improved insulin sen- sitivity (IS), Furthermore, NNMT-/- females fed a WD showed reduced wt. gain, less fat, and lower insulin levels.  However, no improved glucose tolerance was obsd. in NNMT--/- mice.  Although NNMT expression in human fat biopsy samples increased during WR, cor- responding plasma MNAM levels significantly declined, suggesting that other mechanisms besides aNNMT ex- pression modulate circulating MNAM levels during WR.  In summary, upon NNMT deletion or knockdown in males and females fed different obesity-inducing diets, we obsd. sex- and diet-specific differences in body compn., wt., and glucose tolerance and esti- mates of IS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOvfmG9xy5crVg90H21EOLACvtfcHk0lgVkvUN41kwSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXptlSmtr0%253D&md5=e7ceb8a20692f47b92f5f97e1edd38fc</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.2337%2Fdb18-0780&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2337%252Fdb18-0780%26sid%3Dliteratum%253Aachs%26aulast%3DBrachs%26aufirst%3DS.%26aulast%3DPolack%26aufirst%3DJ.%26aulast%3DBrachs%26aufirst%3DM.%26aulast%3DJahn-Hofmann%26aufirst%3DK.%26aulast%3DElvert%26aufirst%3DR.%26aulast%3DPfenninger%26aufirst%3DA.%26aulast%3DBarenz%26aufirst%3DF.%26aulast%3DMargerie%26aufirst%3DD.%26aulast%3DMai%26aufirst%3DK.%26aulast%3DSpranger%26aufirst%3DJ.%26aulast%3DKannt%26aufirst%3DA.%26atitle%3DGenetic%2520nicotinamide%2520N-methyltransferase%2520%2528Nnmt%2529%2520deficiency%2520in%2520male%2520mice%2520improves%2520insulin%2520sensitivity%2520in%2520diet-induced%2520obesity%2520but%2520does%2520not%2520affect%2520glucose%2520tolerance%26jtitle%3DDiabetes%26date%3D2019%26volume%3D68%26issue%3D3%26spage%3D527%26epage%3D542%26doi%3D10.2337%2Fdb18-0780" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ju, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, Z.</span></span> <span> </span><span class="NLM_article-title">Serum N(1)-methylnicotinamide is associated with obesity and diabetes in chinese</span>. <i>J. Clin. Endocrinol. Metab.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>100</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">3112</span>– <span class="NLM_lpage">3117</span>, <span class="refDoi"> DOI: 10.1210/jc.2015-1732</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.1210%2Fjc.2015-1732" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=26066674" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVGks7bO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=100&publication_year=2015&pages=3112-3117&issue=8&author=M.+Liuauthor=L.+Liauthor=J.+Chuauthor=B.+Zhuauthor=Q.+Zhangauthor=X.+Yinauthor=W.+Jiangauthor=G.+Daiauthor=W.+Juauthor=Z.+Wangauthor=Q.+Yangauthor=Z.+Fang&title=Serum+N%281%29-methylnicotinamide+is+associated+with+obesity+and+diabetes+in+chinese&doi=10.1210%2Fjc.2015-1732"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Serum N1-methylnicotinamide is associated with obesity and diabetes in Chinese</span></div><div class="casAuthors">Liu, Ming; Li, Lihua; Chu, Jihong; Zhu, Boyu; Zhang, Qingtao; Yin, Xueyan; Jiang, Weimin; Dai, Guoliang; Ju, Wenzheng; Wang, Zhenxing; Yang, Qin; Fang, Zhuyuan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Clinical Endocrinology and Metabolism</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">100</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3112-3117</span>CODEN:
                <span class="NLM_cas:coden">JCEMAZ</span>;
        ISSN:<span class="NLM_cas:issn">0021-972X</span>.
    
            (<span class="NLM_cas:orgname">Endocrine Society</span>)
        </div><div class="casAbstract">Context: Nicotinamide N-methyltransferase (NNMT) is a novel histone methylation modulator that regulates energy metab., and NNMT knockdown prevents diet-induced obesity in mice.  However, whether NNMT plays a role in human obesity and type 2 diabetes (T2DM) remains to be elucidated.  Objective: NNMT catalyzes methylation of nicotinamide to generate N1-methylnicotinamide (me-NAM).  We aimed to investigate the assocns. of serum me-NAM with obesity and T2DM in Chinese.  Design, Setting, and Participants: The study subjects (n = 1160) were recruited from Dali, a city of Yunnan Province, in southwest China.  Anthropometric phenotypes, fasting glucose, and serum lipids were measured.  Serum me-NAM was measured by liq. chromatog.-mass spectrometry.  Results: Serum me-NAM was pos. correlated with body mass index and waist circumference and neg. with high-d. lipoprotein (P ≤ .03).  The correlations remained highly significant in the multivariate adjusted correlation analyses.  In men (n = 691), pos. correlations between me-NAM and fasting glucose, low-d. lipoprotein, liver function, and serum creatinine levels were also obsd. in both simple and multivariate adjusted correlation analyses.  In multiple logistic regression analyses, elevated serum me-NAM was assocd. with higher risks for overweight/obesity (odds ratios, 2.36 and 5.78; 95% confidence intervals, 1.10-5.08 and 1.78-18.76 for men and women, resp.; P ≤ .03) and diabetes (odds ratios, 1.56 and 1.86; 95% confidence intervals, 1.10-2.22 and 1.05-3.31 for men and women, resp.; P ≤ .03).  Conclusions: This first large-scale population study shows that me-NAM, as an indicator of NNMT activity, is strongly assocd. with obesity and diabetes, supporting NNMT as a potential target for treating obesity and diabetes in humans.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKUX5ibbM74bVg90H21EOLACvtfcHk0lgVkvUN41kwSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVGks7bO&md5=552cea7f01cd71291d71d0fbccf7386d</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1210%2Fjc.2015-1732&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1210%252Fjc.2015-1732%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DM.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DChu%26aufirst%3DJ.%26aulast%3DZhu%26aufirst%3DB.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DYin%26aufirst%3DX.%26aulast%3DJiang%26aufirst%3DW.%26aulast%3DDai%26aufirst%3DG.%26aulast%3DJu%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DYang%26aufirst%3DQ.%26aulast%3DFang%26aufirst%3DZ.%26atitle%3DSerum%2520N%25281%2529-methylnicotinamide%2520is%2520associated%2520with%2520obesity%2520and%2520diabetes%2520in%2520chinese%26jtitle%3DJ.%2520Clin.%2520Endocrinol.%2520Metab.%26date%3D2015%26volume%3D100%26issue%3D8%26spage%3D3112%26epage%3D3117%26doi%3D10.1210%2Fjc.2015-1732" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">You, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span> <span> </span><span class="NLM_article-title">Nicotinamide N-methyltransferase enhances the progression of prostate cancer by stabilizing sirtuin 1</span>. <i>Oncol. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>15</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">9195</span>– <span class="NLM_lpage">9201</span>, <span class="refDoi"> DOI: 10.3892/ol.2018.8474</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.3892%2Fol.2018.8474" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=29805651" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A280%3ADC%252BC1Mfns1Gqtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2018&pages=9195-9201&issue=6&author=Z.+Youauthor=Y.+Liuauthor=X.+Liu&title=Nicotinamide+N-methyltransferase+enhances+the+progression+of+prostate+cancer+by+stabilizing+sirtuin+1&doi=10.3892%2Fol.2018.8474"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Nicotinamide N-methyltransferase enhances the progression of prostate cancer by stabilizing sirtuin 1</span></div><div class="casAuthors">You Zhenyu; Liu Yang; Liu Xuefei</div><div class="citationInfo"><span class="NLM_cas:title">Oncology letters</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">9195-9201</span>
        ISSN:<span class="NLM_cas:issn">1792-1074</span>.
    </div><div class="casAbstract">A previous study demonstrated that nicotinamide N-methyltransferase (NNMT) is upregulated in the tissues of patients with prostate cancer (PCa); however, the specific underlying mechanism of this remains unclear.  To begin with, the expression of NNMT was investigated in the peripheral blood of patients with PCa and of healthy control subjects.  The results indicated that the expression level of NNMT was elevated in the peripheral blood and tissues of patients with PCa.  Furthermore, the overexpression of NNMT enhanced PC-3 cell viability, invasion and migration capacity.  Additionally, the overexpression of NNMT significantly increased the mRNA level of sirtuin 1 (SIRT1) in PC-3 cells.  In addition, nicotinamide treatment significantly suppressed the expression of SIRT1 even in PC-3 cells transfected with adeno-associated virus-NNMT.  Furthermore, the PC-3 cell invasion capacity was notably decreased by the nicotinamide treatment; however, such effects were largely abolished by the overexpression of NNMT in PC-3 cells.  These data indicated that NNMT enhanced PC-3 cell migration and invasion mainly by regulating SIRT1 expression.  In summary, the present study indicated that NNMT is an important regulator of SIRT1 expression in PC-3 cells and may be a potential therapeutic target for PCa.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS-eb_XPp1DSe-MdeMhXAHkfW6udTcc2ebPhanY5O8lFrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1Mfns1Gqtg%253D%253D&md5=d3f7f507b8828920608b3ec7a5a0c9ec</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.3892%2Fol.2018.8474&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3892%252Fol.2018.8474%26sid%3Dliteratum%253Aachs%26aulast%3DYou%26aufirst%3DZ.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DX.%26atitle%3DNicotinamide%2520N-methyltransferase%2520enhances%2520the%2520progression%2520of%2520prostate%2520cancer%2520by%2520stabilizing%2520sirtuin%25201%26jtitle%3DOncol.%2520Lett.%26date%3D2018%26volume%3D15%26issue%3D6%26spage%3D9195%26epage%3D9201%26doi%3D10.3892%2Fol.2018.8474" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kannt, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pfenninger, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teichert, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tonjes, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dietrich, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schon, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kloting, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bluher, M.</span></span> <span> </span><span class="NLM_article-title">Association of nicotinamide-N-methyltransferase mRNA expression in human adipose tissue and the plasma concentration of its product, 1-methylnicotinamide, with insulin resistance</span>. <i>Diabetologia</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">799</span>– <span class="NLM_lpage">808</span>, <span class="refDoi"> DOI: 10.1007/s00125-014-3490-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.1007%2Fs00125-014-3490-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=25596852" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFGqtbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=799-808&issue=4&author=A.+Kanntauthor=A.+Pfenningerauthor=L.+Teichertauthor=A.+Tonjesauthor=A.+Dietrichauthor=M.+R.+Schonauthor=N.+Klotingauthor=M.+Bluher&title=Association+of+nicotinamide-N-methyltransferase+mRNA+expression+in+human+adipose+tissue+and+the+plasma+concentration+of+its+product%2C+1-methylnicotinamide%2C+with+insulin+resistance&doi=10.1007%2Fs00125-014-3490-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Association of nicotinamide-N-methyltransferase mRNA expression in human adipose tissue and the plasma concentration of its product, 1-methylnicotinamide, with insulin resistance</span></div><div class="casAuthors">Kannt, Aimo; Pfenninger, Anja; Teichert, Lenore; Toenjes, Anke; Dietrich, Arne; Schoen, Michael R.; Kloeting, Nora; Blueher, Matthias</div><div class="citationInfo"><span class="NLM_cas:title">Diabetologia</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">799-808</span>CODEN:
                <span class="NLM_cas:coden">DBTGAJ</span>;
        ISSN:<span class="NLM_cas:issn">0012-186X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Aims/hypothesis: Nicotinamide-N-methyltransferase (NNMT) was recently shown to be upregulated in mouse models of insulin resistance and obesity.  So far, it is unknown whether NNMT is regulated in human disease.  We have explored the hypothesis that white adipose tissue (WAT) NNMT expression and plasma 1-methylnicotinamide (MNA) concn. are increased in human insulin resistance and type 2 diabetes.  Methods: NNMT expression and plasma MNA were analyzed in three groups of individuals: (1) 199 patients undergoing abdominal surgery; (2) 60 individuals on a 12-wk exercise program and (3) 55 patients on a two-step bariatric surgery program.  Results: Patients with manifest type 2 diabetes have a significantly (approx. twofold) higher NNMT expression both in omental and s.c. WAT compared with controls.  Notably, plasma MNA correlated significantly with WAT NNMT expression in patients with type 2 diabetes (women, r = 0.59, p < 0.001; men, r = 0.61, p < 0.001) but not in healthy control individuals.  In insulin-resistant individuals, there was an inverse correlation between insulin sensitivity and plasma MNA (r = 0.44, p = 0.01) or adipose tissue NNMT mRNA (r = 0.64, p < 0.001).  The latter assocn. was confirmed in a second cohort (n = 60, r = 0.78, p < 0.001).  Interventions improving insulin sensitivity-exercise and bariatric surgery-were assocd. with a significant (p < 0.001) redn. in WAT NNMT expression.  Bariatric surgery was also assocd. with a significant decrease in plasma MNA.  Conclusions/interpretation: We demonstrate that WAT NNMT expression is regulated in human insulin resistance and type 2 diabetes and that plasma MNA correlates with increased tissue NNMT expression and the degree of insulin resistance, making it a potential biomarker for loss of insulin sensitivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqd9fuoxaxMlLVg90H21EOLACvtfcHk0lj18dNDga1Rgw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFGqtbY%253D&md5=0ad5f86b100383e2769904f8a86482b9</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1007%2Fs00125-014-3490-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00125-014-3490-7%26sid%3Dliteratum%253Aachs%26aulast%3DKannt%26aufirst%3DA.%26aulast%3DPfenninger%26aufirst%3DA.%26aulast%3DTeichert%26aufirst%3DL.%26aulast%3DTonjes%26aufirst%3DA.%26aulast%3DDietrich%26aufirst%3DA.%26aulast%3DSchon%26aufirst%3DM.%2BR.%26aulast%3DKloting%26aufirst%3DN.%26aulast%3DBluher%26aufirst%3DM.%26atitle%3DAssociation%2520of%2520nicotinamide-N-methyltransferase%2520mRNA%2520expression%2520in%2520human%2520adipose%2520tissue%2520and%2520the%2520plasma%2520concentration%2520of%2520its%2520product%252C%25201-methylnicotinamide%252C%2520with%2520insulin%2520resistance%26jtitle%3DDiabetologia%26date%3D2015%26volume%3D58%26issue%3D4%26spage%3D799%26epage%3D808%26doi%3D10.1007%2Fs00125-014-3490-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jung, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, L. J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanvoranart, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubert, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prager, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wallace, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mack, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rich, J. N.</span></span> <span> </span><span class="NLM_article-title">Nicotinamide metabolism regulates glioblastoma stem cell maintenance</span>. <i>JCI Insight</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>2</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">90019</span>, <span class="refDoi"> DOI: 10.1172/jci.insight.90019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.1172%2Fjci.insight.90019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=28515364" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2017&pages=90019&issue=10&author=J.+Jungauthor=L.+J.+Y.+Kimauthor=X.+Wangauthor=Q.+Wuauthor=T.+Sanvoranartauthor=C.+G.+Hubertauthor=B.+C.+Pragerauthor=L.+C.+Wallaceauthor=X.+Jinauthor=S.+C.+Mackauthor=J.+N.+Rich&title=Nicotinamide+metabolism+regulates+glioblastoma+stem+cell+maintenance&doi=10.1172%2Fjci.insight.90019"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1172%2Fjci.insight.90019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1172%252Fjci.insight.90019%26sid%3Dliteratum%253Aachs%26aulast%3DJung%26aufirst%3DJ.%26aulast%3DKim%26aufirst%3DL.%2BJ.%2BY.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DWu%26aufirst%3DQ.%26aulast%3DSanvoranart%26aufirst%3DT.%26aulast%3DHubert%26aufirst%3DC.%2BG.%26aulast%3DPrager%26aufirst%3DB.%2BC.%26aulast%3DWallace%26aufirst%3DL.%2BC.%26aulast%3DJin%26aufirst%3DX.%26aulast%3DMack%26aufirst%3DS.%2BC.%26aulast%3DRich%26aufirst%3DJ.%2BN.%26atitle%3DNicotinamide%2520metabolism%2520regulates%2520glioblastoma%2520stem%2520cell%2520maintenance%26jtitle%3DJCI%2520Insight%26date%3D2017%26volume%3D2%26issue%3D10%26spage%3D90019%26doi%3D10.1172%2Fjci.insight.90019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Eckert, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coscia, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chryplewicz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hernandez, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tienda, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nahotko, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blaženović, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lastra, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curtis, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamada, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perets, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGregor, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrade, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fiehn, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moellering, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lengyel, E.</span></span> <span> </span><span class="NLM_article-title">Proteomics reveals NNMT as a master metabolic regulator of cancer-associated fibroblasts</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>569</i></span> (<span class="NLM_issue">7758</span>),  <span class="NLM_fpage">723</span>– <span class="NLM_lpage">728</span>, <span class="refDoi"> DOI: 10.1038/s41586-019-1173-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.1038%2Fs41586-019-1173-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=31043742" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BC1MXovFeis7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=569&publication_year=2019&pages=723-728&issue=7758&author=M.+A.+Eckertauthor=F.+Cosciaauthor=A.+Chryplewiczauthor=J.+W.+Changauthor=K.+M.+Hernandezauthor=S.+Panauthor=S.+M.+Tiendaauthor=D.+A.+Nahotkoauthor=G.+Liauthor=I.+Bla%C5%BEenovi%C4%87author=R.+R.+Lastraauthor=M.+Curtisauthor=S.+D.+Yamadaauthor=R.+Peretsauthor=S.+M.+McGregorauthor=J.+Andradeauthor=O.+Fiehnauthor=R.+E.+Moelleringauthor=M.+Mannauthor=E.+Lengyel&title=Proteomics+reveals+NNMT+as+a+master+metabolic+regulator+of+cancer-associated+fibroblasts&doi=10.1038%2Fs41586-019-1173-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Proteomics reveals NNMT as a master metabolic regulator of cancer-associated fibroblasts</span></div><div class="casAuthors">Eckert, Mark A.; Coscia, Fabian; Chryplewicz, Agnieszka; Chang, Jae Won; Hernandez, Kyle M.; Pan, Shawn; Tienda, Samantha M.; Nahotko, Dominik A.; Li, Gang; Blazenovic, Ivana; Lastra, Ricardo R.; Curtis, Marion; Yamada, S. Diane; Perets, Ruth; McGregor, Stephanie M.; Andrade, Jorge; Fiehn, Oliver; Moellering, Raymond E.; Mann, Matthias; Lengyel, Ernst</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">569</span>
        (<span class="NLM_cas:issue">7758</span>),
    <span class="NLM_cas:pages">723-728</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">High-grade serous carcinoma has a poor prognosis, owing primarily to its early dissemination throughout the abdominal cavity.  Genomic and proteomic approaches have provided snapshots of the proteogenomics of ovarian cancer1,2, but a systematic examn. of both the tumor and stromal compartments is crit. in understanding ovarian cancer metastasis.  Here we develop a label-free proteomic workflow to analyze as few as 5000 formalin-fixed, paraffin-embedded cells microdissected from each compartment.  The tumor proteome was stable during progression from in situ lesions to metastatic disease; however, the metastasis-assocd. stroma was characterized by a highly conserved proteomic signature, prominently including the methyltransferase nicotinamide N-methyltransferase (NNMT) and several of the proteins that it regulates.  Stromal NNMT expression was necessary and sufficient for functional aspects of the cancer-assocd. fibroblast (CAF) phenotype, including the expression of CAF markers and the secretion of cytokines and oncogenic extracellular matrix.  Stromal NNMT expression supported ovarian cancer migration, proliferation and in vivo growth and metastasis.  Expression of NNMT in CAFs led to depletion of S-adenosyl methionine and redn. in histone methylation assocd. with widespread gene expression changes in the tumor stroma.  This work supports the use of ultra-low-input proteomics to identify candidate drivers of disease phenotypes.  NNMT is a central, metabolic regulator of CAF differentiation and cancer progression in the stroma that may be therapeutically targeted.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlL_GCBb1LSbVg90H21EOLACvtfcHk0liQUZfH7HmdLw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXovFeis7s%253D&md5=43dbbcee11b5d21c5fb77f5b8ff02701</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2Fs41586-019-1173-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-019-1173-8%26sid%3Dliteratum%253Aachs%26aulast%3DEckert%26aufirst%3DM.%2BA.%26aulast%3DCoscia%26aufirst%3DF.%26aulast%3DChryplewicz%26aufirst%3DA.%26aulast%3DChang%26aufirst%3DJ.%2BW.%26aulast%3DHernandez%26aufirst%3DK.%2BM.%26aulast%3DPan%26aufirst%3DS.%26aulast%3DTienda%26aufirst%3DS.%2BM.%26aulast%3DNahotko%26aufirst%3DD.%2BA.%26aulast%3DLi%26aufirst%3DG.%26aulast%3DBla%25C5%25BEenovi%25C4%2587%26aufirst%3DI.%26aulast%3DLastra%26aufirst%3DR.%2BR.%26aulast%3DCurtis%26aufirst%3DM.%26aulast%3DYamada%26aufirst%3DS.%2BD.%26aulast%3DPerets%26aufirst%3DR.%26aulast%3DMcGregor%26aufirst%3DS.%2BM.%26aulast%3DAndrade%26aufirst%3DJ.%26aulast%3DFiehn%26aufirst%3DO.%26aulast%3DMoellering%26aufirst%3DR.%2BE.%26aulast%3DMann%26aufirst%3DM.%26aulast%3DLengyel%26aufirst%3DE.%26atitle%3DProteomics%2520reveals%2520NNMT%2520as%2520a%2520master%2520metabolic%2520regulator%2520of%2520cancer-associated%2520fibroblasts%26jtitle%3DNature%26date%3D2019%26volume%3D569%26issue%3D7758%26spage%3D723%26epage%3D728%26doi%3D10.1038%2Fs41586-019-1173-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Neelakantan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vance, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hommel, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McHardy, S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watowich, S. J.</span></span> <span> </span><span class="NLM_article-title">Structure-activity relationship for small molecule inhibitors of nicotinamide N-methyltransferase</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">5015</span>– <span class="NLM_lpage">5028</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00389</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00389" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BC2sXos1Sgtrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=5015-5028&issue=12&author=H.+Neelakantanauthor=H.+Y.+Wangauthor=V.+Vanceauthor=J.+D.+Hommelauthor=S.+F.+McHardyauthor=S.+J.+Watowich&title=Structure-activity+relationship+for+small+molecule+inhibitors+of+nicotinamide+N-methyltransferase&doi=10.1021%2Facs.jmedchem.7b00389"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Activity Relationship for Small Molecule Inhibitors of Nicotinamide N-Methyltransferase</span></div><div class="casAuthors">Neelakantan, Harshini; Wang, Hua-Yu; Vance, Virginia; Hommel, Jonathan D.; McHardy, Stanton F.; Watowich, Stanley J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">5015-5028</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Nicotinamide N-methyltransferase (NNMT) is a fundamental cytosolic biotransforming enzyme that catalyzes the N-methylation of endogenous and exogenous xenobiotics.  We have identified small mol. inhibitors of NNMT with >1000-fold range of activity and developed comprehensive structure-activity relationships (SARs) for NNMT inhibitors.  Screening of N-methylated quinolinium, isoquinolinium, pyrididium, and benzimidazolium/benzothiazolium analogs resulted in the identification of quinoliniums as a promising scaffold with very low micromolar (IC50 ∼ 1 μM) NNMT inhibition.  Computer-based docking of inhibitors to the NNMT substrate (nicotinamide)-binding site produced a robust correlation between ligand-enzyme interaction docking scores and exptl. calcd. IC50 values.  Predicted binding orientation of the quinolinium analogs revealed selective binding to the NNMT substrate-binding site residues and essential chem. features driving protein-ligand intermol. interactions and NNMT inhibition.  The development of this new series of small mol. NNMT inhibitors direct the future design of lead drug-like inhibitors to treat several metabolic and chronic disease conditions characterized by abnormal NNMT activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMxYy3poogIbVg90H21EOLACvtfcHk0liQUZfH7HmdLw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXos1Sgtrc%253D&md5=2cd7159b9e15569221849168e49e39cd</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00389&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00389%26sid%3Dliteratum%253Aachs%26aulast%3DNeelakantan%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DH.%2BY.%26aulast%3DVance%26aufirst%3DV.%26aulast%3DHommel%26aufirst%3DJ.%2BD.%26aulast%3DMcHardy%26aufirst%3DS.%2BF.%26aulast%3DWatowich%26aufirst%3DS.%2BJ.%26atitle%3DStructure-activity%2520relationship%2520for%2520small%2520molecule%2520inhibitors%2520of%2520nicotinamide%2520N-methyltransferase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26issue%3D12%26spage%3D5015%26epage%3D5028%26doi%3D10.1021%2Facs.jmedchem.7b00389" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kannt, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rajagopal, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kadnur, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suresh, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhamidipati, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swaminathan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hallur, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kristam, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elvert, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Czech, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pfenninger, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rudolph, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schreuder, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chandrasekar, D. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mane, V. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Birudukota, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaik, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zope, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burri, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anand, N. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thakur, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parveen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kandan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mullangi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gosu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruf, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhakshinamoorthy, S.</span></span> <span> </span><span class="NLM_article-title">A small molecule inhibitor of nicotinamide N-methyltransferase for the treatment of metabolic disorders</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">3660</span>, <span class="refDoi"> DOI: 10.1038/s41598-018-22081-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.1038%2Fs41598-018-22081-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=29483571" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A280%3ADC%252BC1Mrls1ahtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2018&pages=3660&author=A.+Kanntauthor=S.+Rajagopalauthor=S.+V.+Kadnurauthor=J.+Sureshauthor=R.+K.+Bhamidipatiauthor=S.+Swaminathanauthor=M.+S.+Hallurauthor=R.+Kristamauthor=R.+Elvertauthor=J.+Czechauthor=A.+Pfenningerauthor=C.+Rudolphauthor=H.+Schreuderauthor=D.+V.+Chandrasekarauthor=V.+S.+Maneauthor=S.+Birudukotaauthor=S.+Shaikauthor=B.+R.+Zopeauthor=R.+R.+Burriauthor=N.+N.+Anandauthor=M.+K.+Thakurauthor=M.+Singhauthor=R.+Parveenauthor=S.+Kandanauthor=R.+Mullangiauthor=T.+Yuraauthor=R.+Gosuauthor=S.+Rufauthor=S.+Dhakshinamoorthy&title=A+small+molecule+inhibitor+of+nicotinamide+N-methyltransferase+for+the+treatment+of+metabolic+disorders&doi=10.1038%2Fs41598-018-22081-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">A small molecule inhibitor of Nicotinamide N-methyltransferase for the treatment of metabolic disorders</span></div><div class="casAuthors">Kannt Aimo; Elvert Ralf; Czech Jorg; Pfenninger Anja; Rudolph Christine; Schreuder Herman; Ruf Sven; Kannt Aimo; Rajagopal Sridharan; Kadnur Sanjay Venkatachalapathi; Suresh Juluri; Bhamidipati Ravi Kanth; Swaminathan Srinivasan; Hallur Mahanandeesha Siddappa; Kristam Rajendra; Chandrasekar Devaraj Venkatapura; Mane Vishal Subhash; Birudukota Swarnakumari; Shaik Shama; Zope Bharat Ravindra; Burri Raghunadha Reddy; Anand Niranjan Naranapura; Thakur Manish Kumar; Singh Manvi; Parveen Reejuana; Kandan Saravanan; Mullangi Ramesh; Yura Takeshi; Gosu Ramachandraiah; Dhakshinamoorthy Saravanakumar</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">3660</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Nicotinamide N-methyltransferase (NNMT) is a cytosolic enzyme that catalyzes the transfer of a methyl group from the co-factor S-adenosyl-L-methionine (SAM) onto the substrate, nicotinamide (NA) to form 1-methyl-nicotinamide (MNA).  Higher NNMT expression and MNA concentrations have been associated with obesity and type-2 diabetes.  Here we report a small molecule analog of NA, JBSNF-000088, that inhibits NNMT activity, reduces MNA levels and drives insulin sensitization, glucose modulation and body weight reduction in animal models of metabolic disease.  In mice with high fat diet (HFD)-induced obesity, JBSNF-000088 treatment caused a reduction in body weight, improved insulin sensitivity and normalized glucose tolerance to the level of lean control mice.  These effects were not seen in NNMT knockout mice on HFD, confirming specificity of JBSNF-000088.  The compound also improved glucose handling in ob/ob and db/db mice albeit to a lesser extent and in the absence of weight loss.  Co-crystal structure analysis revealed the presence of the N-methylated product of JBSNF-000088 bound to the NNMT protein.  The N-methylated product was also detected in the plasma of mice treated with JBSNF-000088.  Hence, JBSNF-000088 may act as a slow-turnover substrate analog, driving the observed metabolic benefits.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRdHZ3bviJkhgaK-_ZudimnfW6udTcc2eYkz5Gpnhu9qLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1Mrls1ahtw%253D%253D&md5=3f2306da18e0ab2a41a57f08ea9384fb</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fs41598-018-22081-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-018-22081-7%26sid%3Dliteratum%253Aachs%26aulast%3DKannt%26aufirst%3DA.%26aulast%3DRajagopal%26aufirst%3DS.%26aulast%3DKadnur%26aufirst%3DS.%2BV.%26aulast%3DSuresh%26aufirst%3DJ.%26aulast%3DBhamidipati%26aufirst%3DR.%2BK.%26aulast%3DSwaminathan%26aufirst%3DS.%26aulast%3DHallur%26aufirst%3DM.%2BS.%26aulast%3DKristam%26aufirst%3DR.%26aulast%3DElvert%26aufirst%3DR.%26aulast%3DCzech%26aufirst%3DJ.%26aulast%3DPfenninger%26aufirst%3DA.%26aulast%3DRudolph%26aufirst%3DC.%26aulast%3DSchreuder%26aufirst%3DH.%26aulast%3DChandrasekar%26aufirst%3DD.%2BV.%26aulast%3DMane%26aufirst%3DV.%2BS.%26aulast%3DBirudukota%26aufirst%3DS.%26aulast%3DShaik%26aufirst%3DS.%26aulast%3DZope%26aufirst%3DB.%2BR.%26aulast%3DBurri%26aufirst%3DR.%2BR.%26aulast%3DAnand%26aufirst%3DN.%2BN.%26aulast%3DThakur%26aufirst%3DM.%2BK.%26aulast%3DSingh%26aufirst%3DM.%26aulast%3DParveen%26aufirst%3DR.%26aulast%3DKandan%26aufirst%3DS.%26aulast%3DMullangi%26aufirst%3DR.%26aulast%3DYura%26aufirst%3DT.%26aulast%3DGosu%26aufirst%3DR.%26aulast%3DRuf%26aufirst%3DS.%26aulast%3DDhakshinamoorthy%26aufirst%3DS.%26atitle%3DA%2520small%2520molecule%2520inhibitor%2520of%2520nicotinamide%2520N-methyltransferase%2520for%2520the%2520treatment%2520of%2520metabolic%2520disorders%26jtitle%3DSci.%2520Rep.%26date%3D2018%26volume%3D8%26spage%3D3660%26doi%3D10.1038%2Fs41598-018-22081-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ruf, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hallur, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anchan, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swamy, I. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murugesan, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sarkar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narasimhulu, L. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Putta, V. P. R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaik, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chandrasekar, D. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mane, V. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kadnur, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suresh, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhamidipati, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burri, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kristam, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schreuder, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Czech, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rudolph, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marker, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langer, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mullangi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gosu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kannt, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhakshinamoorthy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rajagopal, S.</span></span> <span> </span><span class="NLM_article-title">Novel nicotinamide analog as inhibitor of nicotinamide N-methyltransferase</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>28</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">922</span>– <span class="NLM_lpage">925</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2018.01.058</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.1016%2Fj.bmcl.2018.01.058" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=29433927" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisFCmsb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2018&pages=922-925&issue=5&author=S.+Rufauthor=M.+S.+Hallurauthor=N.+K.+Anchanauthor=I.+N.+Swamyauthor=K.+R.+Murugesanauthor=S.+Sarkarauthor=L.+K.+Narasimhuluauthor=V.+P.+R.+K.+Puttaauthor=S.+Shaikauthor=D.+V.+Chandrasekarauthor=V.+S.+Maneauthor=S.+V.+Kadnurauthor=J.+Sureshauthor=R.+K.+Bhamidipatiauthor=M.+Singhauthor=R.+R.+Burriauthor=R.+Kristamauthor=H.+Schreuderauthor=J.+Czechauthor=C.+Rudolphauthor=A.+Markerauthor=T.+Langerauthor=R.+Mullangiauthor=T.+Yuraauthor=R.+Gosuauthor=A.+Kanntauthor=S.+Dhakshinamoorthyauthor=S.+Rajagopal&title=Novel+nicotinamide+analog+as+inhibitor+of+nicotinamide+N-methyltransferase&doi=10.1016%2Fj.bmcl.2018.01.058"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Novel nicotinamide analog as inhibitor of nicotinamide N-methyltransferase</span></div><div class="casAuthors">Ruf, Sven; Hallur, Mahanandeesha Siddappa; Anchan, Nisha K.; Swamy, Indu N.; Murugesan, Karthikai Raj; Sarkar, Sayantani; Narasimhulu, Lokesh Kananti; Putta, V. P. Rama Kishore; Shaik, Shama; Chandrasekar, Devaraj Venkatapura; Mane, Vishal Subhash; Kadnur, Sanjay Venkatachalapathi; Suresh, Juluri; Bhamidipati, Ravi Kanth; Singh, Manvi; Burri, Raghunadha Reddy; Kristam, Rajendra; Schreuder, Herman; Czech, Joerg; Rudolph, Christine; Marker, Alexander; Langer, Thomas; Mullangi, Ramesh; Yura, Takeshi; Gosu, Ramachandraiah; Kannt, Aimo; Dhakshinamoorthy, Saravanakumar; Rajagopal, Sridharan</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">922-925</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Nicotinamide N-methyltransferase (NNMT) has been linked to obesity and diabetes.  We have identified a novel nicotinamide (NA) analog, compd. I that inhibited NNMT enzymic activity and reduced the formation of 1-methyl-nicotinamide (MNA), the primary metabolite of NA by ∼80% at 2 h when dosed in mice orally at 50 mg/kg.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxeiU30o2tybVg90H21EOLACvtfcHk0ljVGmEJ18fS0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisFCmsb0%253D&md5=06345ee6c292c482711d62639fac9a16</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2018.01.058&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2018.01.058%26sid%3Dliteratum%253Aachs%26aulast%3DRuf%26aufirst%3DS.%26aulast%3DHallur%26aufirst%3DM.%2BS.%26aulast%3DAnchan%26aufirst%3DN.%2BK.%26aulast%3DSwamy%26aufirst%3DI.%2BN.%26aulast%3DMurugesan%26aufirst%3DK.%2BR.%26aulast%3DSarkar%26aufirst%3DS.%26aulast%3DNarasimhulu%26aufirst%3DL.%2BK.%26aulast%3DPutta%26aufirst%3DV.%2BP.%2BR.%2BK.%26aulast%3DShaik%26aufirst%3DS.%26aulast%3DChandrasekar%26aufirst%3DD.%2BV.%26aulast%3DMane%26aufirst%3DV.%2BS.%26aulast%3DKadnur%26aufirst%3DS.%2BV.%26aulast%3DSuresh%26aufirst%3DJ.%26aulast%3DBhamidipati%26aufirst%3DR.%2BK.%26aulast%3DSingh%26aufirst%3DM.%26aulast%3DBurri%26aufirst%3DR.%2BR.%26aulast%3DKristam%26aufirst%3DR.%26aulast%3DSchreuder%26aufirst%3DH.%26aulast%3DCzech%26aufirst%3DJ.%26aulast%3DRudolph%26aufirst%3DC.%26aulast%3DMarker%26aufirst%3DA.%26aulast%3DLanger%26aufirst%3DT.%26aulast%3DMullangi%26aufirst%3DR.%26aulast%3DYura%26aufirst%3DT.%26aulast%3DGosu%26aufirst%3DR.%26aulast%3DKannt%26aufirst%3DA.%26aulast%3DDhakshinamoorthy%26aufirst%3DS.%26aulast%3DRajagopal%26aufirst%3DS.%26atitle%3DNovel%2520nicotinamide%2520analog%2520as%2520inhibitor%2520of%2520nicotinamide%2520N-methyltransferase%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2018%26volume%3D28%26issue%3D5%26spage%3D922%26epage%3D925%26doi%3D10.1016%2Fj.bmcl.2018.01.058" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Horning, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suciu, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghadiri, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ulanovskaya, O. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matthews, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lum, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Backus, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B. F.</span></span> <span> </span><span class="NLM_article-title">Chemical proteomic profiling of human methyltransferases</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>138</i></span> (<span class="NLM_issue">40</span>),  <span class="NLM_fpage">13335</span>– <span class="NLM_lpage">13343</span>, <span class="refDoi"> DOI: 10.1021/jacs.6b07830</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.6b07830" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFymu7bP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=138&publication_year=2016&pages=13335-13343&issue=40&author=B.+D.+Horningauthor=R.+M.+Suciuauthor=D.+A.+Ghadiriauthor=O.+A.+Ulanovskayaauthor=M.+L.+Matthewsauthor=K.+M.+Lumauthor=K.+M.+Backusauthor=S.+J.+Brownauthor=H.+Rosenauthor=B.+F.+Cravatt&title=Chemical+proteomic+profiling+of+human+methyltransferases&doi=10.1021%2Fjacs.6b07830"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical Proteomic Profiling of Human Methyltransferases</span></div><div class="casAuthors">Horning, Benjamin D.; Suciu, Radu M.; Ghadiri, Darian A.; Ulanovskaya, Olesya A.; Matthews, Megan L.; Lum, Kenneth M.; Backus, Keriann M.; Brown, Steven J.; Rosen, Hugh; Cravatt, Benjamin F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">138</span>
        (<span class="NLM_cas:issue">40</span>),
    <span class="NLM_cas:pages">13335-13343</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Methylation is a fundamental mechanism used in Nature to modify the structure and function of biomols., including proteins, DNA, RNA, and metabolites.  Me groups are predominantly installed into biomols. by a large and diverse class of S-adenosyl methionine (SAM)-dependent methyltransferases (MTs), of which there are ∼200 known or putative members in the human proteome.  Deregulated MT activity contributes to numerous diseases, including cancer, and several MT inhibitors are in clin. development.  Nonetheless, a large fraction of the human MT family remains poorly characterized, underscoring the need for new technologies to characterize MTs and their inhibitors in native biol. systems.  Here, the authors describe a suite of S-adenosyl homocysteine (SAH) photoreactive probes and their application in chem. proteomic expts. to profile and enrich a large no. of MTs (>50) from human cancer cell lysates with remarkable specificity over other classes of proteins.  The authors further demonstrate that the SAH probes can enrich MT-assocd. proteins and be used to screen for and assess the selectivity of MT inhibitors, leading to the discovery of a covalent inhibitor of nicotinamide N-methyltransferase (NNMT), an enzyme implicated in cancer and metabolic disorders.  The chem. proteomics probes and methods for their utilization reported herein should prove of value for the functional characterization of MTs, MT complexes, and MT inhibitors in mammalian biol. and disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplJsKzvFF-e7Vg90H21EOLACvtfcHk0ljVGmEJ18fS0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFymu7bP&md5=0e07845e59e33381f3149beae7500e18</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fjacs.6b07830&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.6b07830%26sid%3Dliteratum%253Aachs%26aulast%3DHorning%26aufirst%3DB.%2BD.%26aulast%3DSuciu%26aufirst%3DR.%2BM.%26aulast%3DGhadiri%26aufirst%3DD.%2BA.%26aulast%3DUlanovskaya%26aufirst%3DO.%2BA.%26aulast%3DMatthews%26aufirst%3DM.%2BL.%26aulast%3DLum%26aufirst%3DK.%2BM.%26aulast%3DBackus%26aufirst%3DK.%2BM.%26aulast%3DBrown%26aufirst%3DS.%2BJ.%26aulast%3DRosen%26aufirst%3DH.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26atitle%3DChemical%2520proteomic%2520profiling%2520of%2520human%2520methyltransferases%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2016%26volume%3D138%26issue%3D40%26spage%3D13335%26epage%3D13343%26doi%3D10.1021%2Fjacs.6b07830" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, H. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suciu, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horning, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vinogradova, E. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ulanovskaya, O. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B. F.</span></span> <span> </span><span class="NLM_article-title">Covalent inhibitors of nicotinamide N-methyltransferase (NNMT) provide evidence for target engagement challenges in situ</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>28</i></span> (<span class="NLM_issue">16</span>),  <span class="NLM_fpage">2682</span>– <span class="NLM_lpage">2687</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2018.04.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.1016%2Fj.bmcl.2018.04.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=29731364" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BC1cXptV2qtbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2018&pages=2682-2687&issue=16&author=H.+Y.+Leeauthor=R.+M.+Suciuauthor=B.+D.+Horningauthor=E.+V.+Vinogradovaauthor=O.+A.+Ulanovskayaauthor=B.+F.+Cravatt&title=Covalent+inhibitors+of+nicotinamide+N-methyltransferase+%28NNMT%29+provide+evidence+for+target+engagement+challenges+in+situ&doi=10.1016%2Fj.bmcl.2018.04.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Covalent inhibitors of nicotinamide N-methyltransferase (NNMT) provide evidence for target engagement challenges in situ</span></div><div class="casAuthors">Lee, Hsin-Yu; Suciu, Radu M.; Horning, Benjamin D.; Vinogradova, Ekaterina V.; Ulanovskaya, Olesya A.; Cravatt, Benjamin F.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">2682-2687</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Nicotinamide N-methyltransferase (NNMT) catalyzes the N-methylation of nicotinamide using S-adenosyl-L-methionine (SAM) as a Me donor and, through doing so, can modulate cellular methylation potential to impact diverse epigenetic processes.  NNMT has been implicated in a range of diseases, including cancer and metabolic disorders.  Potent, selective, and cell-active inhibitors would constitute valuable probes to study the biol. functions and therapeutic potential of NNMT.  The authors previously reported the discovery of electrophilic small mols. that inhibit NNMT by reacting with an active-site cysteine residue in the SAM-binding pocket.  Here, the authors have used activity-based protein profiling (ABPP)-guided medicinal chem. to optimize the potency and selectivity of NNMT inhibitors, culminating in the discovery of multiple alpha-chloroacetamide (αCA) compds. with sub-μM IC50 values in vitro and excellent proteomic selectivity in cell lysates.  However, these compds. showed much weaker inhibition of NNMT in cells, a feature that was not shared by off-targets of the αCAs.  The authors' results show the potential for developing potent and selective covalent inhibitors of NNMT, but also highlight challenges that may be faced in targeting this enzyme in cellular systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoFjmALOSTRFbVg90H21EOLACvtfcHk0lhFXD552PLFRA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXptV2qtbY%253D&md5=9194d6ac5d1fd5a70683fb4af9cae74f</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2018.04.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2018.04.017%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DH.%2BY.%26aulast%3DSuciu%26aufirst%3DR.%2BM.%26aulast%3DHorning%26aufirst%3DB.%2BD.%26aulast%3DVinogradova%26aufirst%3DE.%2BV.%26aulast%3DUlanovskaya%26aufirst%3DO.%2BA.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26atitle%3DCovalent%2520inhibitors%2520of%2520nicotinamide%2520N-methyltransferase%2520%2528NNMT%2529%2520provide%2520evidence%2520for%2520target%2520engagement%2520challenges%2520in%2520situ%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2018%26volume%3D28%26issue%3D16%26spage%3D2682%26epage%3D2687%26doi%3D10.1016%2Fj.bmcl.2018.04.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">van
Haren, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taig, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuppens, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sastre Toraño, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moret, E. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parsons, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sartini, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emanuelli, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, N. I.</span></span> <span> </span><span class="NLM_article-title">Inhibitors of nicotinamide N-methyltransferase designed to mimic the methylation reaction transition state</span>. <i>Org. Biomol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>15</i></span> (<span class="NLM_issue">31</span>),  <span class="NLM_fpage">6656</span>– <span class="NLM_lpage">6667</span>, <span class="refDoi"> DOI: 10.1039/C7OB01357D</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.1039%2FC7OB01357D" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=28758655" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1aktLfI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2017&pages=6656-6667&issue=31&author=M.+J.+van%0AHarenauthor=R.+Taigauthor=J.+Kuppensauthor=J.+Sastre+Tora%C3%B1oauthor=E.+E.+Moretauthor=R.+B.+Parsonsauthor=D.+Sartiniauthor=M.+Emanuelliauthor=N.+I.+Martin&title=Inhibitors+of+nicotinamide+N-methyltransferase+designed+to+mimic+the+methylation+reaction+transition+state&doi=10.1039%2FC7OB01357D"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibitors of nicotinamide N-methyltransferase designed to mimic the methylation reaction transition state</span></div><div class="casAuthors">van Haren, Matthijs J.; Taig, Rebecca; Kuppens, Jilles; Sastre Torano, Javier; Moret, Ed E.; Parsons, Richard B.; Sartini, Davide; Emanuelli, Monica; Martin, Nathaniel I.</div><div class="citationInfo"><span class="NLM_cas:title">Organic & Biomolecular Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">31</span>),
    <span class="NLM_cas:pages">6656-6667</span>CODEN:
                <span class="NLM_cas:coden">OBCRAK</span>;
        ISSN:<span class="NLM_cas:issn">1477-0520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Nicotinamide N-methyltransferase (NNMT) is an enzyme that catalyzes the methylation of nicotinamide to form N'-methylnicotinamide.  Both NNMT and its methylated product have recently been linked to a variety of diseases, suggesting a role for the enzyme as a therapeutic target beyond its previously ascribed metabolic function in detoxification.  We here describe the systematic development of NNMT inhibitors derived from the structures of the substrates involved in the methylation reaction.  By covalently linking fragments of the NNMT substrates a diverse library of bisubstrate-like compds. was prepd.  The ability of these compds. to inhibit NNMT was evaluated providing valuable insights into the structural tolerances of the enzyme active site.  These studies led to the identification of new NNMT inhibitors that mimic the transition state of the methylation reaction and inhibit the enzyme with activity on par with established methyltransferase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxwrd9fP8Bf7Vg90H21EOLACvtfcHk0lhFXD552PLFRA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1aktLfI&md5=74280c50135611d361ab81b8fc55fa89</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1039%2FC7OB01357D&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC7OB01357D%26sid%3Dliteratum%253Aachs%26aulast%3Dvan%2BHaren%26aufirst%3DM.%2BJ.%26aulast%3DTaig%26aufirst%3DR.%26aulast%3DKuppens%26aufirst%3DJ.%26aulast%3DSastre%2BTora%25C3%25B1o%26aufirst%3DJ.%26aulast%3DMoret%26aufirst%3DE.%2BE.%26aulast%3DParsons%26aufirst%3DR.%2BB.%26aulast%3DSartini%26aufirst%3DD.%26aulast%3DEmanuelli%26aufirst%3DM.%26aulast%3DMartin%26aufirst%3DN.%2BI.%26atitle%3DInhibitors%2520of%2520nicotinamide%2520N-methyltransferase%2520designed%2520to%2520mimic%2520the%2520methylation%2520reaction%2520transition%2520state%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2017%26volume%3D15%26issue%3D31%26spage%3D6656%26epage%3D6667%26doi%3D10.1039%2FC7OB01357D" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Babault, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allali-Hassani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yue, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ju, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Discovery of bisubstrate inhibitors of nicotinamide N-methyltransferase (NNMT)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">1541</span>– <span class="NLM_lpage">1551</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01422</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01422" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BC1cXotVOruw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=1541-1551&issue=4&author=N.+Babaultauthor=A.+Allali-Hassaniauthor=F.+Liauthor=J.+Fanauthor=A.+Yueauthor=K.+Juauthor=F.+Liuauthor=M.+Vedadiauthor=J.+Liuauthor=J.+Jin&title=Discovery+of+bisubstrate+inhibitors+of+nicotinamide+N-methyltransferase+%28NNMT%29&doi=10.1021%2Facs.jmedchem.7b01422"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Bisubstrate Inhibitors of Nicotinamide N-Methyltransferase (NNMT)</span></div><div class="casAuthors">Babault, Nicolas; Allali-Hassani, Abdellah; Li, Fengling; Fan, Jie; Yue, Alex; Ju, Kevin; Liu, Feng; Vedadi, Masoud; Liu, Jing; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1541-1551</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Nicotinamide N-methyltransferase (NNMT) catalyzes the N-methylation of pyridine-contg. compds. using the cofactor S-5'-adenosyl-L-methionine (SAM) as the Me group donor.  Through the regulation of the levels of its substrates, cofactor, and products, NNMT plays important role in physiol. and pathophysiol.  Overexpression of NNMT has been implicated in various human diseases.  Potent and selective small-mol. NNMT inhibitors are valuable chem. tools for testing biol. and therapeutic hypotheses.  However, very few NNMT inhibitors have been reported.  Here, we describe the discovery of a bisubstrate NNMT inhibitor MS2734 (6), and characterization of this inhibitor in biochem., biophys., kinetic, and structural studies.  Importantly, we obtained the first crystal structure of human NNMT in complex with a small-mol. inhibitor.  The structure of the NNMT-6 complex has unambiguously demonstrated that 6 occupied both substrate and cofactor binding sites.  The findings paved the way for developing more potent and selective NNMT inhibitors in the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9ThNiA1GTwLVg90H21EOLACvtfcHk0lhFXD552PLFRA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXotVOruw%253D%253D&md5=50e23e5f60beb64c184f6eadc6a75883</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01422%26sid%3Dliteratum%253Aachs%26aulast%3DBabault%26aufirst%3DN.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DFan%26aufirst%3DJ.%26aulast%3DYue%26aufirst%3DA.%26aulast%3DJu%26aufirst%3DK.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520bisubstrate%2520inhibitors%2520of%2520nicotinamide%2520N-methyltransferase%2520%2528NNMT%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26issue%3D4%26spage%3D1541%26epage%3D1551%26doi%3D10.1021%2Facs.jmedchem.7b01422" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Haren, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moret, E. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rood, J. J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sartini, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salvucci, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emanuelli, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craveur, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Babault, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, N. I.</span></span> <span> </span><span class="NLM_article-title">Bisubstrate inhibitors of nicotinamide N-methyltransferase (NNMT) with enhanced activity</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span> (<span class="NLM_issue">14</span>),  <span class="NLM_fpage">6597</span>– <span class="NLM_lpage">6614</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b00413</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b00413" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtlams7%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=6597-6614&issue=14&author=Y.+Gaoauthor=M.+J.+van+Harenauthor=E.+E.+Moretauthor=J.+J.+M.+Roodauthor=D.+Sartiniauthor=A.+Salvucciauthor=M.+Emanuelliauthor=P.+Craveurauthor=N.+Babaultauthor=J.+Jinauthor=N.+I.+Martin&title=Bisubstrate+inhibitors+of+nicotinamide+N-methyltransferase+%28NNMT%29+with+enhanced+activity&doi=10.1021%2Facs.jmedchem.9b00413"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Bisubstrate Inhibitors of Nicotinamide N-Methyltransferase (NNMT) with Enhanced Activity</span></div><div class="casAuthors">Gao, Yongzhi; van Haren, Matthijs J.; Moret, Ed E.; Rood, Johannes J. M.; Sartini, Davide; Salvucci, Alessia; Emanuelli, Monica; Craveur, Pierrick; Babault, Nicolas; Jin, Jian; Martin, Nathaniel I.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">6597-6614</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Nicotinamide N-methyltransferase (NNMT) catalyzes the methylation of nicotinamide to form N-methylnicotinamide.  Overexpression of NNMT is assocd. with a variety of diseases, including a no. of cancers and metabolic disorders, suggesting a role for NNMT as a potential therapeutic target.  By structural modification of a lead NNMT inhibitor previously developed in our group, we prepd. a diverse library of inhibitors to probe the different regions of the enzyme's active site.  This investigation revealed that incorporation of a naphthalene moiety, intended to bind the hydrophobic nicotinamide binding pocket via π-π stacking interactions, significantly increases the activity of bisubstrate-like NNMT inhibitors (half-maximal inhibitory concn. 1.41 μM).  These findings are further supported by isothermal titrn. calorimetry binding assays as well as modeling studies.  The most active NNMT inhibitor identified in the present study demonstrated a dose-dependent inhibitory effect on the cell proliferation of the HSC-2 human oral cancer cell line.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7zznNASlV9LVg90H21EOLACvtfcHk0ljFKZgMbLiEZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtlams7%252FO&md5=7384150bf22537f835de298dc7889de3</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b00413&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b00413%26sid%3Dliteratum%253Aachs%26aulast%3DGao%26aufirst%3DY.%26aulast%3Dvan%2BHaren%26aufirst%3DM.%2BJ.%26aulast%3DMoret%26aufirst%3DE.%2BE.%26aulast%3DRood%26aufirst%3DJ.%2BJ.%2BM.%26aulast%3DSartini%26aufirst%3DD.%26aulast%3DSalvucci%26aufirst%3DA.%26aulast%3DEmanuelli%26aufirst%3DM.%26aulast%3DCraveur%26aufirst%3DP.%26aulast%3DBabault%26aufirst%3DN.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DMartin%26aufirst%3DN.%2BI.%26atitle%3DBisubstrate%2520inhibitors%2520of%2520nicotinamide%2520N-methyltransferase%2520%2528NNMT%2529%2520with%2520enhanced%2520activity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26issue%3D14%26spage%3D6597%26epage%3D6614%26doi%3D10.1021%2Facs.jmedchem.9b00413" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lavogina, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Enkvist, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uri, A.</span></span> <span> </span><span class="NLM_article-title">Bisubstrate inhibitors of protein kinases: from principle to practical applications</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>5</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">23</span>– <span class="NLM_lpage">34</span>, <span class="refDoi"> DOI: 10.1002/cmdc.200900252</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.1002%2Fcmdc.200900252" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=19774589" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhs1SqsbzP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2010&pages=23-34&issue=1&author=D.+Lavoginaauthor=E.+Enkvistauthor=A.+Uri&title=Bisubstrate+inhibitors+of+protein+kinases%3A+from+principle+to+practical+applications&doi=10.1002%2Fcmdc.200900252"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Bisubstrate Inhibitors of Protein Kinases: from Principle to Practical Applications</span></div><div class="casAuthors">Lavogina, Darja; Enkvist, Erki; Uri, Asko</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">23-34</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  Bisubstrate inhibitors consist of two conjugated fragments, each targeted to a different binding site of a bisubstrate enzyme.  The design of bisubstrate inhibitors presupposes the formation of the ternary complex in the course of the catalyzed reaction.  The principle advantage of bisubstrate inhibitors is their ability to generate more interactions with the target enzyme that could result in improved affinity and selectivity of the conjugates, when compared with single-site inhibitors.  Among phosphotransferases, the approach was first successfully used for adenylate kinase in 1973.  Since then, several types of bisubstrate inhibitors have been developed for protein kinases, including conjugates of peptides with nucleotides, adenosine derivs. and potent ATP-competitive inhibitors.  Earlier bisubstrate inhibitors had pharmacokinetic qualities that were unsuitable for cellular expts. and hence were mostly used for in vitro studies.  The recently constructed conjugates of adenosine derivs. and D-arginine-rich peptides (ARCs) possess high kinase affinity, high biol. and chem. stability and good cell plasma membrane penetrative properties that enable their application in the regulation of cellular protein phosphorylation balances in cell and tissue expts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrSsmmKerEiPrVg90H21EOLACvtfcHk0ljFKZgMbLiEZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhs1SqsbzP&md5=7efb7b5aa44347f5fa8fc40f98065337</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.200900252&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.200900252%26sid%3Dliteratum%253Aachs%26aulast%3DLavogina%26aufirst%3DD.%26aulast%3DEnkvist%26aufirst%3DE.%26aulast%3DUri%26aufirst%3DA.%26atitle%3DBisubstrate%2520inhibitors%2520of%2520protein%2520kinases%253A%2520from%2520principle%2520to%2520practical%2520applications%26jtitle%3DChemMedChem%26date%3D2010%26volume%3D5%26issue%3D1%26spage%3D23%26epage%3D34%26doi%3D10.1002%2Fcmdc.200900252" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lechtenberg, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mace, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sessions, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williamson, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stalder, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wallez, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, G. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riedl, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pasquale, E. B.</span></span> <span> </span><span class="NLM_article-title">Structure-guided strategy for the development of potent bivalent ERK inhibitors</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">726</span>– <span class="NLM_lpage">731</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.7b00127</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.7b00127" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BC2sXpslSksrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=726-731&issue=7&author=B.+C.+Lechtenbergauthor=P.+D.+Maceauthor=E.+H.+Sessionsauthor=R.+Williamsonauthor=R.+Stalderauthor=Y.+Wallezauthor=G.+P.+Rothauthor=S.+J.+Riedlauthor=E.+B.+Pasquale&title=Structure-guided+strategy+for+the+development+of+potent+bivalent+ERK+inhibitors&doi=10.1021%2Facsmedchemlett.7b00127"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Guided Strategy for the Development of Potent Bivalent ERK Inhibitors</span></div><div class="casAuthors">Lechtenberg, Bernhard C.; Mace, Peter D.; Sessions, E. Hampton; Williamson, Robert; Stalder, Romain; Wallez, Yann; Roth, Gregory P.; Riedl, Stefan J.; Pasquale, Elena B.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">726-731</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">ERK is the effector kinase of the RAS-RAF-MEK-ERK signaling cascade, which promotes cell transformation and malignancy in many cancers and is thus a major drug target in oncol.  Kinase inhibitors targeting RAF or MEK are already used for the treatment of certain cancers, such as melanoma.  Although the initial response to these drugs can be dramatic, development of drug resistance is a major challenge, even with combination therapies targeting both RAF and MEK.  Importantly, most resistance mechanisms still rely on activation of the downstream effector kinase ERK, making it a promising target for drug development efforts.  Here, the authors report the design and structural/functional characterization of a set of bivalent ERK inhibitors that combine a small mol. inhibitor that binds to the ATP-binding pocket with a peptide that selectively binds to an ERK protein interaction surface, the D-site recruitment site (DRS).  The authors' studies show that the lead bivalent inhibitor, SBP3, has markedly improved potency compared to the small mol. inhibitor alone.  Unexpectedly, the authors found that SBP3 also binds to several ERK-related kinases that contain a DRS, highlighting the importance of exptl. verifying the predicted specificity of bivalent inhibitors.  However, SBP3 does not target any other kinases belonging to the same CMGC branch of the kinome.  Addnl., the authors' modular click chem. inhibitor design facilitates the generation of different combinations of small mol. inhibitors with ERK-targeting peptides.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTPtcu1FfITrVg90H21EOLACvtfcHk0ljFKZgMbLiEZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXpslSksrk%253D&md5=6f3855f67f0e4002f1b90218c2e8fd89</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.7b00127&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.7b00127%26sid%3Dliteratum%253Aachs%26aulast%3DLechtenberg%26aufirst%3DB.%2BC.%26aulast%3DMace%26aufirst%3DP.%2BD.%26aulast%3DSessions%26aufirst%3DE.%2BH.%26aulast%3DWilliamson%26aufirst%3DR.%26aulast%3DStalder%26aufirst%3DR.%26aulast%3DWallez%26aufirst%3DY.%26aulast%3DRoth%26aufirst%3DG.%2BP.%26aulast%3DRiedl%26aufirst%3DS.%2BJ.%26aulast%3DPasquale%26aufirst%3DE.%2BB.%26atitle%3DStructure-guided%2520strategy%2520for%2520the%2520development%2520of%2520potent%2520bivalent%2520ERK%2520inhibitors%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2017%26volume%3D8%26issue%3D7%26spage%3D726%26epage%3D731%26doi%3D10.1021%2Facsmedchemlett.7b00127" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R. A.</span></span> <i>Evaluation of Enzyme Inhibitors in Drug Discovery: A Guide for Medicinal Chemists and Pharmacologists</i>, <span class="NLM_edition">2nd</span> ed.; <span class="NLM_publisher-name">John Wiley & Sons, Inc.</span>: <span class="NLM_publisher-loc">Hoboken, NJ</span>, <span class="NLM_year">2005</span>; pp  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">538</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2005&pages=1-538&author=R.+A.+Copeland&title=Evaluation+of+Enzyme+Inhibitors+in+Drug+Discovery%3A+A+Guide+for+Medicinal+Chemists+and+Pharmacologists"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26btitle%3DEvaluation%2520of%2520Enzyme%2520Inhibitors%2520in%2520Drug%2520Discovery%253A%2520A%2520Guide%2520for%2520Medicinal%2520Chemists%2520and%2520Pharmacologists%26pub%3DJohn%2520Wiley%2520%2526%2520Sons%252C%2520Inc%26date%3D2005%26spage%3D1%26epage%3D538" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Peng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sartini, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pozzi, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilk, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emanuelli, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yee, V. C.</span></span> <span> </span><span class="NLM_article-title">Structural basis of substrate recognition in human nicotinamide N-methyltransferase</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>50</i></span> (<span class="NLM_issue">36</span>),  <span class="NLM_fpage">7800</span>– <span class="NLM_lpage">7808</span>, <span class="refDoi"> DOI: 10.1021/bi2007614</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi2007614" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtValsbnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2011&pages=7800-7808&issue=36&author=Y.+Pengauthor=D.+Sartiniauthor=V.+Pozziauthor=D.+Wilkauthor=M.+Emanuelliauthor=V.+C.+Yee&title=Structural+basis+of+substrate+recognition+in+human+nicotinamide+N-methyltransferase&doi=10.1021%2Fbi2007614"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis of substrate recognition in human nicotinamide N-methyltransferase</span></div><div class="casAuthors">Peng, Yi; Sartini, Davide; Pozzi, Valentina; Wilk, Dennis; Emanuelli, Monica; Yee, Vivien C.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">36</span>),
    <span class="NLM_cas:pages">7800-7808</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Nicotinamide N-methyltransferase (NNMT) catalyzes the N-methylation of nicotinamide, pyridines, and other analogs using S-adenosyl-L-methionine as donor.  NNMT plays a significant role in the regulation of metabolic pathways and is expressed at markedly high levels in several kinds of cancers, presenting it as a potential mol. target for cancer therapy.  Here, the authors detd. the crystal structure of human NNMT as a ternary complex bound to both the demethylated donor, S-adenosyl-L-homocysteine, and the acceptor substrate, nicotinamide, to 2.7 Å resoln.  These studies reveal the structural basis for nicotinamide binding and highlight several residues in the active site which may play roles in nicotinamide recognition and NNMT catalysis.  The functional importance of these residues was probed by mutagenesis.  Of three residues near the nicotinamide's amide group, substitution of S201 and S213 had no effect on enzyme activity while replacement of D197 dramatically decreased activity.  Substitutions of Y20, whose side chain hydroxyl interacts with both the nicotinamide arom. ring and AdoHcy carboxylate, also compromised activity.  Enzyme kinetics anal. revealed kcat/Km decreases of 2-3 orders of magnitude for the D197A and Y20A mutants, confirming the functional importance of these active site residues.  The mutants exhibited substantially increased Km for both NCA and AdoMet and modestly decreased kcat.  MD simulations revealed long-range conformational effects which provide an explanation for the large increase in Km(AdoMet) for the D197A mutant, which interacts directly only with nicotinamide in the ternary complex crystal structure.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqh1yxD6WIJFLVg90H21EOLACvtfcHk0li8f-_9y1Mrxw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtValsbnF&md5=926af50d2464e21094ffb17205793d19</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fbi2007614&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi2007614%26sid%3Dliteratum%253Aachs%26aulast%3DPeng%26aufirst%3DY.%26aulast%3DSartini%26aufirst%3DD.%26aulast%3DPozzi%26aufirst%3DV.%26aulast%3DWilk%26aufirst%3DD.%26aulast%3DEmanuelli%26aufirst%3DM.%26aulast%3DYee%26aufirst%3DV.%2BC.%26atitle%3DStructural%2520basis%2520of%2520substrate%2520recognition%2520in%2520human%2520nicotinamide%2520N-methyltransferase%26jtitle%3DBiochemistry%26date%3D2011%26volume%3D50%26issue%3D36%26spage%3D7800%26epage%3D7808%26doi%3D10.1021%2Fbi2007614" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R.
A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banks, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klicic, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mainz, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Repasky, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knoll, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shelley, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perry, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Francis, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shenkin, P. S.</span></span> <span> </span><span class="NLM_article-title">Glide: a new approach for rapid, accurate docking and scoring. 1. method and assessment of docking accuracy</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>47</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">1739</span>– <span class="NLM_lpage">1749</span>, <span class="refDoi"> DOI: 10.1021/jm0306430</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0306430" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFyit74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1739-1749&issue=7&author=R.%0AA.+Friesnerauthor=J.+L.+Banksauthor=R.+B.+Murphyauthor=T.+A.+Halgrenauthor=J.+J.+Klicicauthor=D.+T.+Mainzauthor=M.+P.+Repaskyauthor=E.+H.+Knollauthor=M.+Shelleyauthor=J.+K.+Perryauthor=D.+E.+Shawauthor=P.+Francisauthor=P.+S.+Shenkin&title=Glide%3A+a+new+approach+for+rapid%2C+accurate+docking+and+scoring.+1.+method+and+assessment+of+docking+accuracy&doi=10.1021%2Fjm0306430"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Glide: A new approach for rapid, accurate docking and scoring. 1. method and assessment of docking accuracy</span></div><div class="casAuthors">Friesner, Richard A.; Banks, Jay L.; Murphy, Robert B.; Halgren, Thomas A.; Klicic, Jasna J.; Mainz, Daniel T.; Repasky, Matthew P.; Knoll, Eric H.; Shelley, Mee; Perry, Jason K.; Shaw, David E.; Francis, Perry; Shenkin, Peter S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1739-1749</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Unlike other methods for docking ligands to the rigid 3D structure of a known protein receptor, Glide approximates a complete systematic search of the conformational, orientational, and positional space of the docked ligand.  In this search, an initial rough positioning and scoring phase that dramatically narrows the search space is followed by torsionally flexible energy optimization on an OPLS-AA nonbonded potential grid for a few hundred surviving candidate poses.  The very best candidates are further refined via a Monte Carlo sampling of pose conformation; in some cases, this is crucial to obtaining an accurate docked pose.  Selection of the best docked pose uses a model energy function that combines empirical and force-field-based terms.  Docking accuracy is assessed by redocking ligands from 282 cocrystd. PDB complexes starting from conformationally optimized ligand geometries that bear no memory of the correctly docked pose.  Errors in geometry for the top-ranked pose are less than 1 Å in nearly half of the cases and are greater than 2 Å in only about one-third of them.  Comparisons to published data on rms deviations show that Glide is nearly twice as accurate as GOLD and more than twice as accurate as FlexX for ligands having up to 20 rotatable bonds.  Glide is also found to be more accurate than the recently described Surflex method.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPuigu31qMMLVg90H21EOLACvtfcHk0li8f-_9y1Mrxw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFyit74%253D&md5=8cc2f0022318b12dd972e9c493375bf9</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Fjm0306430&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0306430%26sid%3Dliteratum%253Aachs%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DBanks%26aufirst%3DJ.%2BL.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DKlicic%26aufirst%3DJ.%2BJ.%26aulast%3DMainz%26aufirst%3DD.%2BT.%26aulast%3DRepasky%26aufirst%3DM.%2BP.%26aulast%3DKnoll%26aufirst%3DE.%2BH.%26aulast%3DShelley%26aufirst%3DM.%26aulast%3DPerry%26aufirst%3DJ.%2BK.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26aulast%3DFrancis%26aufirst%3DP.%26aulast%3DShenkin%26aufirst%3DP.%2BS.%26atitle%3DGlide%253A%2520a%2520new%2520approach%2520for%2520rapid%252C%2520accurate%2520docking%2520and%2520scoring.%25201.%2520method%2520and%2520assessment%2520of%2520docking%2520accuracy%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26issue%3D7%26spage%3D1739%26epage%3D1749%26doi%3D10.1021%2Fjm0306430" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yau, E. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coward, J. K.</span></span> <span> </span><span class="NLM_article-title">Synthesis of complex 6’-alkynyl-6’-dethia nucleoside analogs of S-adenosylhomocysteine as potential inhibitors of methyltransferases</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1990</span>,  <span class="NLM_volume"><i>55</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">3147</span>– <span class="NLM_lpage">3158</span>, <span class="refDoi"> DOI: 10.1021/jo00297a034</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo00297a034" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADyaK3cXitFOmsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=1990&pages=3147-3158&issue=10&author=E.+K.+Yauauthor=J.+K.+Coward&title=Synthesis+of+complex+6%E2%80%99-alkynyl-6%E2%80%99-dethia+nucleoside+analogs+of+S-adenosylhomocysteine+as+potential+inhibitors+of+methyltransferases&doi=10.1021%2Fjo00297a034"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis of complex 6'-alkynyl-6'-dethia nucleoside analogs of S-adenosylhomocysteine as potential inhibitors of methyltransferases</span></div><div class="casAuthors">Yau, Eric K.; Coward, James K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">1990</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3147-58</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    </div><div class="casAbstract">The synthesis of complex acetylenic nucleosides contg. a 5',6' carbon-carbon bond has been investigated.  Two synthetic routes were studied, both starting with an α-substituted butyrolactone.  The lactone could be converted to an β-keto sulfone and then to the corresponding enol phosphate di-Ph ester.  Reductive elimination to yield the desired aryl acetylene and could not be effected without some overredn. to the substituted olefin.  Alternatively, the lactone could be converted to an appropriately protected γ-hydroxy aldehyde which was then homologated to a terminal acetylene.  The successful synthesis of a prototypic member of the target class of compds. involved in a Pd(II)-mediated coupling of an aryl iodide to the appropriately functionalized terminal acetylene.  The resulting aryl acetylene was then further elaborated to the target amino acid deriv. I.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqnyIyG3Ot5E7Vg90H21EOLACvtfcHk0li8f-_9y1Mrxw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3cXitFOmsrY%253D&md5=a5419abf2f05077d8c839714a1a59d01</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Fjo00297a034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo00297a034%26sid%3Dliteratum%253Aachs%26aulast%3DYau%26aufirst%3DE.%2BK.%26aulast%3DCoward%26aufirst%3DJ.%2BK.%26atitle%3DSynthesis%2520of%2520complex%25206%25E2%2580%2599-alkynyl-6%25E2%2580%2599-dethia%2520nucleoside%2520analogs%2520of%2520S-adenosylhomocysteine%2520as%2520potential%2520inhibitors%2520of%2520methyltransferases%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D1990%26volume%3D55%26issue%3D10%26spage%3D3147%26epage%3D3158%26doi%3D10.1021%2Fjo00297a034" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Burns, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coward, J. K.</span></span> <span> </span><span class="NLM_article-title">Synthesis of complex δ-acetylenic amino acids as potential multisubstrate adduct inhibitors of methyltransferases</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>4</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">1455</span>– <span class="NLM_lpage">1470</span>, <span class="refDoi"> DOI: 10.1016/0968-0896(96)00139-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.1016%2F0968-0896%2896%2900139-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=8894103" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADyaK28Xmt1Sgsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=1996&pages=1455-1470&issue=9&author=M.+R.+Burnsauthor=J.+K.+Coward&title=Synthesis+of+complex+%CE%B4-acetylenic+amino+acids+as+potential+multisubstrate+adduct+inhibitors+of+methyltransferases&doi=10.1016%2F0968-0896%2896%2900139-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis of complex δ-acetylenic amino acids as potential multisubstrate adduct inhibitors of methyltransferases</span></div><div class="casAuthors">Burns, Mark R.; Coward, James K.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1455-1470</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">The synthesis of two types of δ-acetylenic amino acids is described.  Key intermediates were derived from terminal acetylenes via two different routes: (1) palladium-mediated, Heck-type arylation and (2) Simmons-Smith homologation followed by reaction of the resulting propargylic organometallic with a benzoyltrimethylsilane.  Further elaboration to the desired amino acids involved the coupling of carbanions derived from N-benzylidene glycine esters to complex alkyl halides.  The synthesis of nonnucleoside δ-acetylenic amino acids was successfully effected using this chem.  In the case of the nucleoside-contg. amino acids, a potential multisubstrate adduct inhibitor of catechol O-methyltransferase was synthesized via this route.  Unfortunately, the sensitivity to acid of 5'-deoxy, 5'-carbanucleosides prevented successful completion of the synthesis of a second nucleoside-contg. δ-amino acid as a possible inhibitor of phenethanolamine N-methyltransferase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZMqf8MG5ZPrVg90H21EOLACvtfcHk0lhyUgs9PLGUNw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28Xmt1Sgsbk%253D&md5=da68540557c8312b0db2ab501250bae5</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2F0968-0896%2896%2900139-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0968-0896%252896%252900139-3%26sid%3Dliteratum%253Aachs%26aulast%3DBurns%26aufirst%3DM.%2BR.%26aulast%3DCoward%26aufirst%3DJ.%2BK.%26atitle%3DSynthesis%2520of%2520complex%2520%25CE%25B4-acetylenic%2520amino%2520acids%2520as%2520potential%2520multisubstrate%2520adduct%2520inhibitors%2520of%2520methyltransferases%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D1996%26volume%3D4%26issue%3D9%26spage%3D1455%26epage%3D1470%26doi%3D10.1016%2F0968-0896%2896%2900139-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Whittaker, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lalic, G.</span></span> <span> </span><span class="NLM_article-title">Monophasic catalytic system for the selective semireduction of alkynes</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>15</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">1112</span>– <span class="NLM_lpage">1115</span>, <span class="refDoi"> DOI: 10.1021/ol4001679</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ol4001679" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BC3sXivVyju7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2013&pages=1112-1115&issue=5&author=A.+M.+Whittakerauthor=G.+Lalic&title=Monophasic+catalytic+system+for+the+selective+semireduction+of+alkynes&doi=10.1021%2Fol4001679"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Monophasic Catalytic System for the Selective Semireduction of Alkynes</span></div><div class="casAuthors">Whittaker, Aaron M.; Lalic, Gojko</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1112-1115</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7052</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A highly efficient semiredn. of alkynes has been developed.  Using 0.5-2 mol % of a copper catalyst, semiredn. can be accomplished with a wide range of substrates, including both internal and terminal alkynes without over-redn.  The new method has excellent chemoselectivity, and the semiredn. can be accomplished even in the presence of nitro and aryl iodo groups.  Semiredn. of internal alkynes produces Z-alkenes selectively.  Finally, com. availability of a catalyst precursor adds to the appeal of the new catalytic system.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohR-uMPDtKvrVg90H21EOLACvtfcHk0lhyUgs9PLGUNw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXivVyju7o%253D&md5=403c014320ca7da5fcb4e0c4c5c88200</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Fol4001679&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fol4001679%26sid%3Dliteratum%253Aachs%26aulast%3DWhittaker%26aufirst%3DA.%2BM.%26aulast%3DLalic%26aufirst%3DG.%26atitle%3DMonophasic%2520catalytic%2520system%2520for%2520the%2520selective%2520semireduction%2520of%2520alkynes%26jtitle%3DOrg.%2520Lett.%26date%3D2013%26volume%3D15%26issue%3D5%26spage%3D1112%26epage%3D1115%26doi%3D10.1021%2Fol4001679" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Michrowska, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bujok, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harutyunyan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sashuk, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dolgonos, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grela, K.</span></span> <span> </span><span class="NLM_article-title">Nitro-substituted Hoveyda-Grubbs ruthenium carbenes: enhancement of catalyst activity through electronic activation</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>126</i></span> (<span class="NLM_issue">30</span>),  <span class="NLM_fpage">9318</span>– <span class="NLM_lpage">9325</span>, <span class="refDoi"> DOI: 10.1021/ja048794v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja048794v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BD2cXlsFKrsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=126&publication_year=2004&pages=9318-9325&issue=30&author=A.+Michrowskaauthor=R.+Bujokauthor=S.+Harutyunyanauthor=V.+Sashukauthor=G.+Dolgonosauthor=K.+Grela&title=Nitro-substituted+Hoveyda-Grubbs+ruthenium+carbenes%3A+enhancement+of+catalyst+activity+through+electronic+activation&doi=10.1021%2Fja048794v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Nitro-Substituted Hoveyda-Grubbs Ruthenium Carbenes: Enhancement of Catalyst Activity through Electronic Activation</span></div><div class="casAuthors">Michrowska, Anna; Bujok, Robert; Harutyunyan, Syuzanna; Sashuk, Volodymyr; Dolgonos, Grigory; Grela, Karol</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">126</span>
        (<span class="NLM_cas:issue">30</span>),
    <span class="NLM_cas:pages">9318-9325</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The design, synthesis, stability, and catalytic activity of nitro-substituted Hoveyda-Grubbs metathesis catalysts [e.g. Cl2(1,3-bis(mesityl)imidazolidin-2-ylidene)Ru(:CHC6H3OiPr-2-NO2-5) with alkoxy-O also coordinating to Ru] are described.  The highly active and stable meta- and para-substituted complexes are attractive from a practical point of view.  These catalysts operate in very mild conditions and can be successfully applied in various types of metathesis [ring-closing metathesis, cross-metathesis (CM), and enyne metathesis].  Although the presence of a NO2 group leads to catalysts that are dramatically more active than both the 2nd-generation Grubbs's catalyst and the phosphine-free Hoveyda's carbene, enhancement of reactivity is somewhat lower than that obsd. for a sterically activated Hoveyda-Grubbs catalyst.  Attempts to combine two modes of activation, steric and electronic, result in severely decreasing a catalyst's stability.  Different Ru catalysts turned out to be optimal for different applications.  Whereas phosphine-free carbenes are catalysts of choice for CM of various electron-deficient substrates, they exhibit lower reactivity in the formation of tetrasubstituted double bonds.  No single catalyst outperforms all others in all possible applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqyxhE8BsxuL7Vg90H21EOLACvtfcHk0lhyUgs9PLGUNw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXlsFKrsro%253D&md5=9179e5f461bbe679eb05e51c042f37b5</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fja048794v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja048794v%26sid%3Dliteratum%253Aachs%26aulast%3DMichrowska%26aufirst%3DA.%26aulast%3DBujok%26aufirst%3DR.%26aulast%3DHarutyunyan%26aufirst%3DS.%26aulast%3DSashuk%26aufirst%3DV.%26aulast%3DDolgonos%26aufirst%3DG.%26aulast%3DGrela%26aufirst%3DK.%26atitle%3DNitro-substituted%2520Hoveyda-Grubbs%2520ruthenium%2520carbenes%253A%2520enhancement%2520of%2520catalyst%2520activity%2520through%2520electronic%2520activation%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2004%26volume%3D126%26issue%3D30%26spage%3D9318%26epage%3D9325%26doi%3D10.1021%2Fja048794v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nishimura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sawano, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayashi, T.</span></span> <span> </span><span class="NLM_article-title">Asymmetric synthesis of β-alkynyl aldehydes by rhodium-catalyzed conjugate alkynylation</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>48</i></span> (<span class="NLM_issue">43</span>),  <span class="NLM_fpage">8057</span>– <span class="NLM_lpage">8059</span>, <span class="refDoi"> DOI: 10.1002/anie.200904486</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.1002%2Fanie.200904486" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1Kju73I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2009&pages=8057-8059&issue=43&author=T.+Nishimuraauthor=T.+Sawanoauthor=T.+Hayashi&title=Asymmetric+synthesis+of+%CE%B2-alkynyl+aldehydes+by+rhodium-catalyzed+conjugate+alkynylation&doi=10.1002%2Fanie.200904486"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Asymmetric Synthesis of β-Alkynyl Aldehydes by Rhodium-Catalyzed Conjugate Alkynylation</span></div><div class="casAuthors">Nishimura, Takahiro; Sawano, Takahiro; Hayashi, Tamio</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">43</span>),
    <span class="NLM_cas:pages">8057-8059, S8057/1-S8057/61</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A rhodium-catalyzed conjugate alkynylation of α,β-unsatd. aldehydes giving enantio-enriched β-alkynyl aldehydes is reported.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTmsiMX6A4WLVg90H21EOLACvtfcHk0lgsGpLRQET1Uw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht1Kju73I&md5=7c662585f6b725253107794ecb73767d</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1002%2Fanie.200904486&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.200904486%26sid%3Dliteratum%253Aachs%26aulast%3DNishimura%26aufirst%3DT.%26aulast%3DSawano%26aufirst%3DT.%26aulast%3DHayashi%26aufirst%3DT.%26atitle%3DAsymmetric%2520synthesis%2520of%2520%25CE%25B2-alkynyl%2520aldehydes%2520by%2520rhodium-catalyzed%2520conjugate%2520alkynylation%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2009%26volume%3D48%26issue%3D43%26spage%3D8057%26epage%3D8059%26doi%3D10.1002%2Fanie.200904486" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Burk, M. J.</span></span> <span> </span><span class="NLM_article-title">Modular phospholane ligands in asymmetric catalysis</span>. <i>Acc. Chem. Res.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>33</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">363</span>– <span class="NLM_lpage">372</span>, <span class="refDoi"> DOI: 10.1021/ar990085c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ar990085c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BD3cXhsl2hu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2000&pages=363-372&issue=6&author=M.+J.+Burk&title=Modular+phospholane+ligands+in+asymmetric+catalysis&doi=10.1021%2Far990085c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Modular Phospholane Ligands in Asymmetric Catalysis</span></div><div class="casAuthors">Burk, Mark J.</div><div class="citationInfo"><span class="NLM_cas:title">Accounts of Chemical Research</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">363-372</span>CODEN:
                <span class="NLM_cas:coden">ACHRE4</span>;
        ISSN:<span class="NLM_cas:issn">0001-4842</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review contg. 51 refs. outlines the prepn. and application of a class of phosphine ligands based upon the trans-2,5-disubstituted phospholane moiety.  The modular nature of these ligands has allowed facile variation of both phospholane substituent and backbone structure, thus providing access to ligands.  Bidentate bis(phospholane) ligands are very useful in asym. catalytic hydrogenation reactions.  In particular, the authors highlight the versatility of highly efficient bis(phospholane)rhodium catalysts that allow enantioselective hydrogenation to produce a diverse range of compds. contg. C-N, C-O, and C-C stereogenic centers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpvHWVq09cZe7Vg90H21EOLACvtfcHk0lgsGpLRQET1Uw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXhsl2hu7s%253D&md5=b592ee7c317233753e93254fceb58d55</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Far990085c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Far990085c%26sid%3Dliteratum%253Aachs%26aulast%3DBurk%26aufirst%3DM.%2BJ.%26atitle%3DModular%2520phospholane%2520ligands%2520in%2520asymmetric%2520catalysis%26jtitle%3DAcc.%2520Chem.%2520Res.%26date%3D2000%26volume%3D33%26issue%3D6%26spage%3D363%26epage%3D372%26doi%3D10.1021%2Far990085c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sniady, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bedore, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jamison, T. F.</span></span> <span> </span><span class="NLM_article-title">One-flow, multistep synthesis of nucleosides by Bronsted acid-catalyzed glycosylation</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>50</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">2155</span>– <span class="NLM_lpage">2158</span>, <span class="refDoi"> DOI: 10.1002/anie.201006440</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.1002%2Fanie.201006440" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BC3MXisVWmtLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2011&pages=2155-2158&issue=9&author=A.+Sniadyauthor=M.+W.+Bedoreauthor=T.+F.+Jamison&title=One-flow%2C+multistep+synthesis+of+nucleosides+by+Bronsted+acid-catalyzed+glycosylation&doi=10.1002%2Fanie.201006440"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">One-Flow, Multistep Synthesis of Nucleosides by Bronsted Acid-Catalyzed Glycosylation</span></div><div class="casAuthors">Sniady, Adam; Bedore, Matthew W.; Jamison, Timothy F.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2155-2158, S2155/1-S2155/50</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Synthesis of nucleosides and nucleoside analogs using pyridinium triflate salts as efficient glycosylation catalysts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomJzJ1cWZ6JbVg90H21EOLACvtfcHk0lgsGpLRQET1Uw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXisVWmtLY%253D&md5=58995c77e36a344b951499abc7502873</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1002%2Fanie.201006440&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201006440%26sid%3Dliteratum%253Aachs%26aulast%3DSniady%26aufirst%3DA.%26aulast%3DBedore%26aufirst%3DM.%2BW.%26aulast%3DJamison%26aufirst%3DT.%2BF.%26atitle%3DOne-flow%252C%2520multistep%2520synthesis%2520of%2520nucleosides%2520by%2520Bronsted%2520acid-catalyzed%2520glycosylation%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2011%26volume%3D50%26issue%3D9%26spage%3D2155%26epage%3D2158%26doi%3D10.1002%2Fanie.201006440" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Neelakantan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vance, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McHardy, S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watowich, S. J.</span></span> <span> </span><span class="NLM_article-title">Noncoupled fluorescent assay for direct real-time monitoring of nicotinamide N-methyltransferase activity</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>56</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">824</span>– <span class="NLM_lpage">832</span>, <span class="refDoi"> DOI: 10.1021/acs.biochem.6b01215</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.biochem.6b01215" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFajs74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2017&pages=824-832&issue=6&author=H.+Neelakantanauthor=V.+Vanceauthor=H.+L.+Wangauthor=S.+F.+McHardyauthor=S.+J.+Watowich&title=Noncoupled+fluorescent+assay+for+direct+real-time+monitoring+of+nicotinamide+N-methyltransferase+activity&doi=10.1021%2Facs.biochem.6b01215"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Noncoupled Fluorescent Assay for Direct Real-Time Monitoring of Nicotinamide N-Methyltransferase Activity</span></div><div class="casAuthors">Neelakantan, Harshini; Vance, Virginia; Wang, Hua-Yu Leo; McHardy, Stanton F.; Watowich, Stanley J.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">824-832</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Nicotinamide N-methyltransferase (NNMT) is an important biotransforming enzyme that catalyzes the transfer of a labile Me group from the ubiquitous cofactor, S-adenosyl-L-methionine (SAM), to endogenous and exogenous small mols. such as quinoline to form methylated end-products.  NNMT has been implicated in a no. of chronic disease conditions, including metabolic disorders, cardiovascular disease, cancer, osteoarthritis, kidney disease, and Parkinson's disease.  Here, the authors developed a novel noncoupled fluorescence-based methyltransferase assay that enabled direct ultrasensitive real-time detection of the NNMT reaction product, 1-methylquinolinium.  This is the 1st assay reported to date to utilize fluorescence spectroscopy to directly monitor NNMT product formation and activity in real-time.  This assay provided accurate kinetic data that enabled detailed comparative anal. of the NNMT reaction mechanism and kinetic parameters.  A reaction model based on a random bi-reactant mechanism produced global curve fits that were most consistent with steady-state initial velocity data collected across an array of substrate concns.  Based on the reaction mechanism, each substrate could independently bind to the NNMT apoenzyme; however, both substrates bound to the complementary binary complexes with ∼20-fold higher affinity relative to independent binding.  This reaction mechanism implies either substrate-induced conformational changes or bi-reactant intermol. interactions may stabilize substrate binding to the binary complex and formation of the ternary complex.  Importantly, this assay could rapidly generate concn. response curves for known NNMT inhibitors, suggesting its applicability for high-throughput screening (HTS) of chem. libraries to identify novel NNMT inhibitors.  Furthermore, this novel assay potentially offers a robust detection technol. for use in SAM substrate competition assays for the discovery and development of SAM-dependent methyltransferase inhibitors such as S-adenosyl-L-homocysteine and 1-methylnicotinamide.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6X5AP5dGb77Vg90H21EOLACvtfcHk0lgsGpLRQET1Uw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFajs74%253D&md5=bf243bd7362322b094c31b1e247b7240</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Facs.biochem.6b01215&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.biochem.6b01215%26sid%3Dliteratum%253Aachs%26aulast%3DNeelakantan%26aufirst%3DH.%26aulast%3DVance%26aufirst%3DV.%26aulast%3DWang%26aufirst%3DH.%2BL.%26aulast%3DMcHardy%26aufirst%3DS.%2BF.%26aulast%3DWatowich%26aufirst%3DS.%2BJ.%26atitle%3DNoncoupled%2520fluorescent%2520assay%2520for%2520direct%2520real-time%2520monitoring%2520of%2520nicotinamide%2520N-methyltransferase%2520activity%26jtitle%3DBiochemistry%26date%3D2017%26volume%3D56%26issue%3D6%26spage%3D824%26epage%3D832%26doi%3D10.1021%2Facs.biochem.6b01215" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Loring, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, P. R.</span></span> <span> </span><span class="NLM_article-title">Kinetic mechanism of nicotinamide N-methyltransferase</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>57</i></span> (<span class="NLM_issue">38</span>),  <span class="NLM_fpage">5524</span>– <span class="NLM_lpage">5532</span>, <span class="refDoi"> DOI: 10.1021/acs.biochem.8b00775</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.biochem.8b00775" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsFyksLnM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2018&pages=5524-5532&issue=38&author=H.+S.+Loringauthor=P.+R.+Thompson&title=Kinetic+mechanism+of+nicotinamide+N-methyltransferase&doi=10.1021%2Facs.biochem.8b00775"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Kinetic Mechanism of Nicotinamide N-Methyltransferase</span></div><div class="casAuthors">Loring, Heather S.; Thompson, Paul R.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">38</span>),
    <span class="NLM_cas:pages">5524-5532</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Nicotinamide N-methyltransferase (NNMT) catalyzes the transfer of a Me group from S-adenosyl methionine to nicotinamide, pyridine and other structural analogs.  Aberrantly increased NNMT activity results in the depletion of S-adenosyl methionine (SAM), nicotinamide (NAM), and NAD+; NAM is required for NAD+ biosynthesis.  SAM depletion impairs the methylation potential of the cell, resulting in hypomethylated histones and an altered epigenetic profile.  In addn., decreased NAD+ levels neg. affect energy metab. by disrupting oxidative phosphorylation.  Due to its impact on epigenetic states and NAD+ levels, NNMT is implicated in cancer, neurodegenerative and metabolic diseases, making it an appealing target for therapeutic intervention.  To gain insights that would guide the design of inhibitors and activity-based probes, we performed detailed kinetic studies on human NNMT.  Herein, we report the kinetic mechanism of NNMT.  Our initial velocity, product inhibition, and dead-end analog inhibition studies collectively indicate that NNMT uses a rapid equil. ordered mechanism, where NNMT first binds SAM, which is followed by NAM.  Me transfer occurs and methylated NAM (MeNAM) and S-adenosyl homocysteine (SAH) are released consecutively.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBKv-k7WIajbVg90H21EOLACvtfcHk0lhvTCfESMPqjA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsFyksLnM&md5=8c67d54bd73a7fe4ad3dc51411d12d1e</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Facs.biochem.8b00775&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.biochem.8b00775%26sid%3Dliteratum%253Aachs%26aulast%3DLoring%26aufirst%3DH.%2BS.%26aulast%3DThompson%26aufirst%3DP.%2BR.%26atitle%3DKinetic%2520mechanism%2520of%2520nicotinamide%2520N-methyltransferase%26jtitle%3DBiochemistry%26date%3D2018%26volume%3D57%26issue%3D38%26spage%3D5524%26epage%3D5532%26doi%3D10.1021%2Facs.biochem.8b00775" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cleland, W. W.</span></span> <span> </span><span class="NLM_article-title">The kinetics of enzyme-catalyzed reactions with two or more substrates or products: I. nomenclature and rate equations</span>. <i>Biochim. Biophys. Acta, Spec. Sect. Enzymol. Subj.</i> <span class="NLM_year" style="font-weight: bold;">1963</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">104</span>– <span class="NLM_lpage">137</span>, <span class="refDoi"> DOI: 10.1016/0926-6569(63)90211-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.1016%2F0926-6569%2863%2990211-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADyaF3sXksl2gtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=1963&pages=104-137&author=W.+W.+Cleland&title=The+kinetics+of+enzyme-catalyzed+reactions+with+two+or+more+substrates+or+products%3A+I.+nomenclature+and+rate+equations&doi=10.1016%2F0926-6569%2863%2990211-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">The kinetics of enzyme-catalyzed reactions with two or more substrates or products. I. Nomenclature and rate equations</span></div><div class="casAuthors">Cleland, W. W.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Specialized Section on Enzymological Subjects</span>
        (<span class="NLM_cas:date">1963</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">104-37</span>CODEN:
                <span class="NLM_cas:coden">BBASD9</span>;
        ISSN:<span class="NLM_cas:issn">0926-6569</span>.
    </div><div class="casAbstract">A nomenclature is proposed to facilitate discussion of possible mechanisms for enzyme-catalyzed reactions with more than one substrate or product.  A shorthand notation for writing such mechanisms is also described.  A general method for expressing the full steady-state rate equations for these mechanisms in terms of measurable kinetic consts. is explained, and the resulting rate equations are given for a no. of mechanisms with 2 or 3 substrates or products.  The manner in which kinetic studies can be used to det. enzymic mechanisms is discussed, and examples are given to illustrate the types of information about reaction mechanisms that can be obtained.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQjkgumn5qvrVg90H21EOLACvtfcHk0lhvTCfESMPqjA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaF3sXksl2gtA%253D%253D&md5=50de4ab301560c410f6348e096e2db8d</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2F0926-6569%2863%2990211-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0926-6569%252863%252990211-6%26sid%3Dliteratum%253Aachs%26aulast%3DCleland%26aufirst%3DW.%2BW.%26atitle%3DThe%2520kinetics%2520of%2520enzyme-catalyzed%2520reactions%2520with%2520two%2520or%2520more%2520substrates%2520or%2520products%253A%2520I.%2520nomenclature%2520and%2520rate%2520equations%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Spec.%2520Sect.%2520Enzymol.%2520Subj.%26date%3D1963%26volume%3D67%26spage%3D104%26epage%3D137%26doi%3D10.1016%2F0926-6569%2863%2990211-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cleland, W. W.</span></span> <span> </span><span class="NLM_article-title">The kinetics of enzyme-catalyzed reactions with two or more substrates or products: II. inhibition: nomenclature and theory</span>. <i>Biochim. Biophys. Acta, Spec. Sect. Enzymol. Subj.</i> <span class="NLM_year" style="font-weight: bold;">1963</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">173</span>– <span class="NLM_lpage">187</span>, <span class="refDoi"> DOI: 10.1016/0926-6569(63)90226-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.1016%2F0926-6569%2863%2990226-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADyaF3sXmslOjsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=1963&pages=173-187&author=W.+W.+Cleland&title=The+kinetics+of+enzyme-catalyzed+reactions+with+two+or+more+substrates+or+products%3A+II.+inhibition%3A+nomenclature+and+theory&doi=10.1016%2F0926-6569%2863%2990226-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">The kinetics of enzyme-catalyzed reactions with two or more substrates or products. II. Inhibition--nomenclature and theory</span></div><div class="casAuthors">Cleland, W. W.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Specialized Section on Enzymological Subjects</span>
        (<span class="NLM_cas:date">1963</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">173-87</span>CODEN:
                <span class="NLM_cas:coden">BBASD9</span>;
        ISSN:<span class="NLM_cas:issn">0926-6569</span>.
    </div><div class="casAbstract">cf. CA 58, 8192c.  A nomenclature is proposed to describe different types of inhibitions of enzyme-catalyzed reactions, particularly for reactions with more than one substrate and product.  The rate equations for such inhibitions are discussed, as are methods for distinguishing between the various types of inhibition and obtaining inhibition and kinetic consts. from exptl. data.  Several examples are given of the type of information that can be obtained from inhibition studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGppRMzEG8P0qrVg90H21EOLACvtfcHk0lhvTCfESMPqjA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaF3sXmslOjsQ%253D%253D&md5=72a7b057094e41748e51e66a309f30bf</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2F0926-6569%2863%2990226-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0926-6569%252863%252990226-8%26sid%3Dliteratum%253Aachs%26aulast%3DCleland%26aufirst%3DW.%2BW.%26atitle%3DThe%2520kinetics%2520of%2520enzyme-catalyzed%2520reactions%2520with%2520two%2520or%2520more%2520substrates%2520or%2520products%253A%2520II.%2520inhibition%253A%2520nomenclature%2520and%2520theory%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Spec.%2520Sect.%2520Enzymol.%2520Subj.%26date%3D1963%26volume%3D67%26spage%3D173%26epage%3D187%26doi%3D10.1016%2F0926-6569%2863%2990226-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cleland, W. W.</span></span> <span> </span><span class="NLM_article-title">The kinetics of enzyme-catalyzed reactions with two or more substrates or products: III. prediction of initial velocity and inhibition patterns by inspection</span>. <i>Biochim. Biophys. Acta, Spec. Sect. Enzymol. Subj.</i> <span class="NLM_year" style="font-weight: bold;">1963</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">188</span>– <span class="NLM_lpage">196</span>, <span class="refDoi"> DOI: 10.1016/0926-6569(63)90227-X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.1016%2F0926-6569%2863%2990227-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADyaF3sXmslOjtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=1963&pages=188-196&author=W.+W.+Cleland&title=The+kinetics+of+enzyme-catalyzed+reactions+with+two+or+more+substrates+or+products%3A+III.+prediction+of+initial+velocity+and+inhibition+patterns+by+inspection&doi=10.1016%2F0926-6569%2863%2990227-X"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">The kinetics of enzyme-catalyzed reactions with two or more substrates or products. III. Prediction of initial velocity and inhibition patterns by inspection</span></div><div class="casAuthors">Cleland, W. W.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Specialized Section on Enzymological Subjects</span>
        (<span class="NLM_cas:date">1963</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">188-96</span>CODEN:
                <span class="NLM_cas:coden">BBASD9</span>;
        ISSN:<span class="NLM_cas:issn">0926-6569</span>.
    </div><div class="casAbstract">A general method for predicting initial velocity and dead end and product inhibition patterns by inspection of the mechanism is presented.  This method is applicable to any nonrandom mechanism without alternate reaction sequences.  Several examples of the application of the method are given.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3BXiUFNAX1rVg90H21EOLACvtfcHk0lhvTCfESMPqjA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaF3sXmslOjtg%253D%253D&md5=f63ac2b9ba05bef8de8d3768752a6de7</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2F0926-6569%2863%2990227-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0926-6569%252863%252990227-X%26sid%3Dliteratum%253Aachs%26aulast%3DCleland%26aufirst%3DW.%2BW.%26atitle%3DThe%2520kinetics%2520of%2520enzyme-catalyzed%2520reactions%2520with%2520two%2520or%2520more%2520substrates%2520or%2520products%253A%2520III.%2520prediction%2520of%2520initial%2520velocity%2520and%2520inhibition%2520patterns%2520by%2520inspection%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Spec.%2520Sect.%2520Enzymol.%2520Subj.%26date%3D1963%26volume%3D67%26spage%3D188%26epage%3D196%26doi%3D10.1016%2F0926-6569%2863%2990227-X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">van
Haren, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sastre Torano, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sartini, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emanuelli, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parsons, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, N. I.</span></span> <span> </span><span class="NLM_article-title">A rapid and efficient assay for the characterization of substrates and inhibitors of nicotinamide N-methyltransferase</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">5307</span>– <span class="NLM_lpage">5315</span>, <span class="refDoi"> DOI: 10.1021/acs.biochem.6b00733</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.biochem.6b00733" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVWntL7O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2016&pages=5307-5315&author=M.+J.+van%0AHarenauthor=J.+Sastre+Toranoauthor=D.+Sartiniauthor=M.+Emanuelliauthor=R.+B.+Parsonsauthor=N.+I.+Martin&title=A+rapid+and+efficient+assay+for+the+characterization+of+substrates+and+inhibitors+of+nicotinamide+N-methyltransferase&doi=10.1021%2Facs.biochem.6b00733"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">A rapid and efficient assay for the characterization of substrates and inhibitors of nicotinamide N-methyltransferase</span></div><div class="casAuthors">van Haren, Matthijs J.; Sastre Torano, Javier; Sartini, Davide; Emanuelli, Monica; Parsons, Richard B.; Martin, Nathaniel I.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">37</span>),
    <span class="NLM_cas:pages">5307-5315</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Nicotinamide N-methyltransferase (NNMT) is one of the most abundant small mol. methyltransferases in the human body and is primarily responsible for the N-methylation of the nicotinamide (vitamin B3).  Employing the cofactor, S-adenosyl-L-methionine, NNMT transfers a Me group to the pyridine N atom of nicotinamide to generate N-methylnicotinamide.  Interestingly, NNMT is also able to N-methylate a variety of other pyridine-contg. small mols., suggesting a secondary role for the enzyme in the detoxification of xenobiotics.  A no. of recent studies have also revealed links between NNMT overexpression and a variety of diseases, including multiple cancers, Parkinson's disease, diabetes, and obesity.  Here, to facilitate further study of both the substrate scope and potential for inhibitor development, the authors describe the development of a new activity assay for human NNMT.  The assay makes use of ultra-high-performance hydrophilic interaction chromatog., allowing for the rapid sepn. of the reaction products, coupled with quadrupole time-of-flight mass spectrometric detection, providing for enhanced sensitivity and enabling high-throughput sample anal.  The authors successfully demonstrate the general applicability of the method by performing kinetic analyses of NNMT-mediated methylation for a range of pyridine-based substrates.  These findings also provide new insight into the diversity of substrate recognition by NNMT in a quant. manner.  In addn., the authors further established the suitability of the assay for the identification and characterization of small mol. inhibitors of NNMT.  To do so, the authors investigated the inhibition of NNMT by the nonspecific methyltransferase inhibitors, sinefungin and S-adenosyl-L-homocysteine, revealing IC50 values in the low micromolar range.  The results of these inhibition studies are particularly noteworthy as they will permit future efforts toward the development of new NNMT-specific inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqBC8U-yKb1brVg90H21EOLACvtfcHk0lhw_xdo2ORtiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVWntL7O&md5=0ebb58b665b91dd5983b2bef91886c10</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Facs.biochem.6b00733&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.biochem.6b00733%26sid%3Dliteratum%253Aachs%26aulast%3Dvan%2BHaren%26aufirst%3DM.%2BJ.%26aulast%3DSastre%2BTorano%26aufirst%3DJ.%26aulast%3DSartini%26aufirst%3DD.%26aulast%3DEmanuelli%26aufirst%3DM.%26aulast%3DParsons%26aufirst%3DR.%2BB.%26aulast%3DMartin%26aufirst%3DN.%2BI.%26atitle%3DA%2520rapid%2520and%2520efficient%2520assay%2520for%2520the%2520characterization%2520of%2520substrates%2520and%2520inhibitors%2520of%2520nicotinamide%2520N-methyltransferase%26jtitle%3DBiochemistry%26date%3D2016%26volume%3D55%26spage%3D5307%26epage%3D5315%26doi%3D10.1021%2Facs.biochem.6b00733" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Segel, I. H.</span></span> <i>Enzyme Kinetics</i>; <span class="NLM_publisher-name">Wiley</span>: <span class="NLM_publisher-loc">New York</span>, <span class="NLM_year">1975</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1975&author=I.+H.+Segel&title=Enzyme+Kinetics"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DSegel%26aufirst%3DI.%2BH.%26btitle%3DEnzyme%2520Kinetics%26pub%3DWiley%26date%3D1975" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R. A.</span></span> <i>Evaluation of Enzyme Inhibitors in Drug Discovery</i>, <span class="NLM_edition">2nd</span> ed.; <span class="NLM_publisher-name">John Wiley</span>: <span class="NLM_publisher-loc">New York</span>, <span class="NLM_year">2013</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.1002%2F9781118540398" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&author=R.+A.+Copeland&title=Evaluation+of+Enzyme+Inhibitors+in+Drug+Discovery"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1002%2F9781118540398&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F9781118540398%26sid%3Dliteratum%253Aachs%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26btitle%3DEvaluation%2520of%2520Enzyme%2520Inhibitors%2520in%2520Drug%2520Discovery%26pub%3DJohn%2520Wiley%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kuzmic, P.</span></span> <span> </span><span class="NLM_article-title">Program DynaFit for the analysis of enzyme kinetic data: application to HIV proteinase</span>. <i>Anal. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>237</i></span>,  <span class="NLM_fpage">260</span>– <span class="NLM_lpage">273</span>, <span class="refDoi"> DOI: 10.1006/abio.1996.0238</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.1006%2Fabio.1996.0238" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=8660575" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADyaK28XjtlOktL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=237&publication_year=1996&pages=260-273&author=P.+Kuzmic&title=Program+DynaFit+for+the+analysis+of+enzyme+kinetic+data%3A+application+to+HIV+proteinase&doi=10.1006%2Fabio.1996.0238"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Program DYNAFIT for the analysis of enzyme kinetic data: application to HIV proteinase</span></div><div class="casAuthors">Kuzmic, Petr</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Biochemistry</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">237</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">260-273</span>CODEN:
                <span class="NLM_cas:coden">ANBCA2</span>;
        ISSN:<span class="NLM_cas:issn">0003-2697</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">A computer program with the code name DYNAFIT was developed for fitting either the initial velocities or the time course of enzyme reactions to an arbitrary mol. mechanism represented symbolically by a set of chem. equations.  Seven numerical tests and five graphical tests are applied to judge the goodness of fit.  Exptl. data on the inhibition of the dissociative dimeric proteinase from HIV were used in four test examples.  A set of initial velocities was analyzed to see if a tight-binding inhibitor could bind to the HIV proteinase monomer.  Three different sets of progress curves were analyzed (i) to det. the kinetic properties of an irreversible inhibitor, (ii) to investigate the dissocn. and denaturation mechanism for the protease dimer, and (iii) to investigate the inhibition mechanism for a transient inhibitor.  The program is available by anonymous ftp via uwmml.pharmacy.wisc.edu and on the World Wide Web via http://uwmml.pharmacy.wisc. edu.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGa_hR-OebWLVg90H21EOLACvtfcHk0lhw_xdo2ORtiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XjtlOktL4%253D&md5=71b4b01a4d81d3e38490b1124fe836b4</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1006%2Fabio.1996.0238&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fabio.1996.0238%26sid%3Dliteratum%253Aachs%26aulast%3DKuzmic%26aufirst%3DP.%26atitle%3DProgram%2520DynaFit%2520for%2520the%2520analysis%2520of%2520enzyme%2520kinetic%2520data%253A%2520application%2520to%2520HIV%2520proteinase%26jtitle%3DAnal.%2520Biochem.%26date%3D1996%26volume%3D237%26spage%3D260%26epage%3D273%26doi%3D10.1006%2Fabio.1996.0238" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kuzmic, P.</span></span> <span> </span><span class="NLM_article-title">DynaFit - a software package for enzymology</span>. <i>Methods Enzymol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>467</i></span>,  <span class="NLM_fpage">247</span>– <span class="NLM_lpage">280</span>, <span class="refDoi"> DOI: 10.1016/S0076-6879(09)67010-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.1016%2FS0076-6879%2809%2967010-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=19897096" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmt12rsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=467&publication_year=2009&pages=247-280&author=P.+Kuzmic&title=DynaFit+-+a+software+package+for+enzymology&doi=10.1016%2FS0076-6879%2809%2967010-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">DynaFit - a software package for enzymology</span></div><div class="casAuthors">Kuzmic, Petr</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Enzymology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">467</span>
        (<span class="NLM_cas:issue">Computer Methods, Part B</span>),
    <span class="NLM_cas:pages">247-280</span>CODEN:
                <span class="NLM_cas:coden">MENZAU</span>;
        ISSN:<span class="NLM_cas:issn">0076-6879</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">A review.  Since its original publication, the DynaFit software package has been used in more than 500 published studies.  Most applications have been in biochem., esp. in enzyme kinetics.  This paper describes a no. of recently added features and capabilities, in the hope that the tool will continue to be useful to the enzymol. community.  Fully functional DynaFit continues to be freely available to all academic researchers from http://www.biokin.com.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGogZuk_IwpvgrVg90H21EOLACvtfcHk0lhw_xdo2ORtiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXmt12rsA%253D%253D&md5=36f49ffbcd28396eb6884631687cb592</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1016%2FS0076-6879%2809%2967010-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0076-6879%252809%252967010-5%26sid%3Dliteratum%253Aachs%26aulast%3DKuzmic%26aufirst%3DP.%26atitle%3DDynaFit%2520-%2520a%2520software%2520package%2520for%2520enzymology%26jtitle%3DMethods%2520Enzymol.%26date%3D2009%26volume%3D467%26spage%3D247%26epage%3D280%26doi%3D10.1016%2FS0076-6879%2809%2967010-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Williams, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morrison, J. F.</span></span> <span> </span><span class="NLM_article-title">The kinetics of reversible tight-binding inhibition</span>. <i>Methods Enzymol.</i> <span class="NLM_year" style="font-weight: bold;">1979</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">437</span>– <span class="NLM_lpage">467</span>, <span class="refDoi"> DOI: 10.1016/0076-6879(79)63019-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.1016%2F0076-6879%2879%2963019-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=502865" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADyaL3cXovF2ntw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=1979&pages=437-467&author=J.+W.+Williamsauthor=J.+F.+Morrison&title=The+kinetics+of+reversible+tight-binding+inhibition&doi=10.1016%2F0076-6879%2879%2963019-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">The kinetics of reversible tight-binding inhibition</span></div><div class="casAuthors">Williams, Jeffrey W.; Morrison, John F.</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Enzymology</span>
        (<span class="NLM_cas:date">1979</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">Enzyme Kinet. Mech., Part A</span>),
    <span class="NLM_cas:pages">437-67</span>CODEN:
                <span class="NLM_cas:coden">MENZAU</span>;
        ISSN:<span class="NLM_cas:issn">0076-6879</span>.
    </div><div class="casAbstract">In addn. to a general discussion of the kinetics of reversible tight-binding inhibitors, which exert their effect at a concn. comparable to that of the enzyme, and for which steady-state theory is used, exptl. results are given of the tight-binding inhibition of creatine kinase by Eu3+ (actually Eu-ADP) under steady-state conditions.  The inhibition of creatine kinase by fixed concns. of Eu3+ in the presence of varying concns. of Mg-ADP indicated that the inhibition is competitive, suggesting that Eu-ADP acts as a structural analog of Mg-ADP.  The inhibition was noncompetitive with respect to creatine phosphate.  The dissocn. consts. for binding of Eu-ADP to the free enzyme and the enzyme-phosphocreatine complex were 1.5 and 2.2 nM, resp., as detd. from Ki values.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2CwV3voXt27Vg90H21EOLACvtfcHk0ljOg4HgIovk4Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3cXovF2ntw%253D%253D&md5=a770f91b51661c9502d0aca44a24e1c7</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1016%2F0076-6879%2879%2963019-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0076-6879%252879%252963019-7%26sid%3Dliteratum%253Aachs%26aulast%3DWilliams%26aufirst%3DJ.%2BW.%26aulast%3DMorrison%26aufirst%3DJ.%2BF.%26atitle%3DThe%2520kinetics%2520of%2520reversible%2520tight-binding%2520inhibition%26jtitle%3DMethods%2520Enzymol.%26date%3D1979%26volume%3D63%26spage%3D437%26epage%3D467%26doi%3D10.1016%2F0076-6879%2879%2963019-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Neelakantan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vance, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wetzel, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McHardy, S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finnerty, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hommel, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watowich, S. J.</span></span> <span> </span><span class="NLM_article-title">Selective and membrane-permeable small molecule inhibitors of nicotinamide N-methyltransferase reverse high fat diet-induced obesity in mice</span>. <i>Biochem. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>147</i></span>,  <span class="NLM_fpage">141</span>– <span class="NLM_lpage">152</span>, <span class="refDoi"> DOI: 10.1016/j.bcp.2017.11.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.1016%2Fj.bcp.2017.11.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=29155147" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvVegu7vL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=147&publication_year=2018&pages=141-152&author=H.+Neelakantanauthor=V.+Vanceauthor=M.+D.+Wetzelauthor=H.+L.+Wangauthor=S.+F.+McHardyauthor=C.+C.+Finnertyauthor=J.+D.+Hommelauthor=S.+J.+Watowich&title=Selective+and+membrane-permeable+small+molecule+inhibitors+of+nicotinamide+N-methyltransferase+reverse+high+fat+diet-induced+obesity+in+mice&doi=10.1016%2Fj.bcp.2017.11.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Selective and membrane-permeable small molecule inhibitors of nicotinamide N-methyltransferase reverse high fat diet-induced obesity in mice</span></div><div class="casAuthors">Neelakantan, Harshini; Vance, Virginia; Wetzel, Michael D.; Wang, Hua-Yu Leo; McHardy, Stanton F.; Finnerty, Celeste C.; Hommel, Jonathan D.; Watowich, Stanley J.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Pharmacology (Amsterdam, Netherlands)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">147</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">141-152</span>CODEN:
                <span class="NLM_cas:coden">BCPCA6</span>;
        ISSN:<span class="NLM_cas:issn">0006-2952</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">There is a crit. need for new mechanism-of-action drugs that reduce the burden of obesity and assocd. chronic metabolic comorbidities.  A potentially novel target to treat obesity and type 2 diabetes is nicotinamide-N-methyltransferase (NNMT), a cytosolic enzyme with newly identified roles in cellular metab. and energy homeostasis.  To validate NNMT as an anti-obesity drug target, the authors investigated the permeability, selectivity, mechanistic, and physiol. properties of a series of small mol. NNMT inhibitors.  Membrane permeability of NNMT inhibitors was characterized using parallel artificial membrane permeability and Caco-2 cell assays.  Selectivity was tested against structurally-related methyltransferases and NAD (NAD+) salvage pathway enzymes.  Effects of NNMT inhibitors on lipogenesis and intracellular levels of metabolites, including NNMT reaction product 1-methylnicotianamide (1-MNA) were evaluated in cultured adipocytes.  Effects of a potent NNMT inhibitor on obesity measures and plasma lipid were assessed in diet-induced obese mice fed a high-fat diet.  Methylquinolinium scaffolds with primary amine substitutions displayed high permeability from passive and active transport across membranes.  Importantly, methylquinolinium analogs displayed high selectivity, not inhibiting related SAM-dependent methyltransferases or enzymes in the NAD+ salvage pathway.  NNMT inhibitors reduced intracellular 1-MNA, increased intracellular NAD+ and S-(5'-adenosyl)-L-methionine (SAM), and suppressed lipogenesis in adipocytes.  Treatment of diet-induced obese mice systemically with a potent NNMT inhibitor significantly reduced body wt. and white adipose mass, decreased adipocyte size, and lowered plasma total cholesterol levels.  Notably, administration of NNMT inhibitors did not impact total food intake nor produce any observable adverse effects.  These results support development of small mol. NNMT inhibitors as therapeutics to reverse diet-induced obesity and validate NNMT as a viable target to treat obesity and related metabolic conditions.  Increased flux of key cellular energy regulators, including NAD+ and SAM, may potentially define the therapeutic mechanism-of-action of NNMT inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0K0OY6S9HhbVg90H21EOLACvtfcHk0ljOg4HgIovk4Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvVegu7vL&md5=4a798e5f69cf9989b8d7a9d0584f872d</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1016%2Fj.bcp.2017.11.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bcp.2017.11.007%26sid%3Dliteratum%253Aachs%26aulast%3DNeelakantan%26aufirst%3DH.%26aulast%3DVance%26aufirst%3DV.%26aulast%3DWetzel%26aufirst%3DM.%2BD.%26aulast%3DWang%26aufirst%3DH.%2BL.%26aulast%3DMcHardy%26aufirst%3DS.%2BF.%26aulast%3DFinnerty%26aufirst%3DC.%2BC.%26aulast%3DHommel%26aufirst%3DJ.%2BD.%26aulast%3DWatowich%26aufirst%3DS.%2BJ.%26atitle%3DSelective%2520and%2520membrane-permeable%2520small%2520molecule%2520inhibitors%2520of%2520nicotinamide%2520N-methyltransferase%2520reverse%2520high%2520fat%2520diet-induced%2520obesity%2520in%2520mice%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D2018%26volume%3D147%26spage%3D141%26epage%3D152%26doi%3D10.1016%2Fj.bcp.2017.11.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Eisenberg, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwarz, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Komaromy, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wall, R.</span></span> <span> </span><span class="NLM_article-title">Analysis of membrane and surface protein sequences with the hydrophobic moment plot</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1984</span>,  <span class="NLM_volume"><i>179</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">125</span>– <span class="NLM_lpage">142</span>, <span class="refDoi"> DOI: 10.1016/0022-2836(84)90309-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.1016%2F0022-2836%2884%2990309-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=6502707" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADyaL2MXnsFajuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=179&publication_year=1984&pages=125-142&issue=1&author=D.+Eisenbergauthor=E.+Schwarzauthor=M.+Komaromyauthor=R.+Wall&title=Analysis+of+membrane+and+surface+protein+sequences+with+the+hydrophobic+moment+plot&doi=10.1016%2F0022-2836%2884%2990309-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Analysis of membrane and surface protein sequences with the hydrophobic moment plot</span></div><div class="casAuthors">Eisenberg, D.; Schwarz, E.; Komaromy, M.; Wall, R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1984</span>),
    <span class="NLM_cas:volume">179</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">125-42</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    </div><div class="casAbstract">An algorithm was developed which identifies α-helices involved in the interactions of membrane proteins with lipid bilayers and which distinguishes them from helices in sol. proteins.  The membrane-assocd. helices are then classified with the aid of the hydrophobic moment plot, on which the hydrophobic moment of each helix is ploted as a function of its hydrophobicity.  The magnitude of hydrophobic moment measures the amphiphilicity of the helix (and hence its tendency to seek a surface between hydrophobic and hydrophilic phases), and the hydrophobicity measures its affinity for the membrane interior.  Segments of membrane proteins in α-helices tend to fall in 1 of 3 regions of a hydrophobic moment plot: (1) monomeric transmembrane anchors (class I HLA transmembrane sequences) lie in the region of highest hydrophobicity and smallest hydrophobic moment; (2) helices presumed to be paired (such as the transmembrane M segments of surface Igs) and helices which are bundled together in membranes (such as bacteriorhodopsin) fall in the adjacent region with higher hydrophobic moment and smaller hydrophobicity; and (3) helices from surface-seeking proteins (such as melittin) fall in the region with still higher hydrophobic moment.  α-Helices from globular proteins mainly fall in a region of lower mean hydrophobicity and hydrophobic moment.  Application of these methods to the sequence of diphtheria toxin suggests 4 transmembrane helices and a surface-seeking helix in fragment B, the moiety known to have transmembrane function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkBWsePnQWqbVg90H21EOLACvtfcHk0ljOg4HgIovk4Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL2MXnsFajuw%253D%253D&md5=a2b8d36bbe0e1454bd6a0917fc0e7089</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1016%2F0022-2836%2884%2990309-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0022-2836%252884%252990309-7%26sid%3Dliteratum%253Aachs%26aulast%3DEisenberg%26aufirst%3DD.%26aulast%3DSchwarz%26aufirst%3DE.%26aulast%3DKomaromy%26aufirst%3DM.%26aulast%3DWall%26aufirst%3DR.%26atitle%3DAnalysis%2520of%2520membrane%2520and%2520surface%2520protein%2520sequences%2520with%2520the%2520hydrophobic%2520moment%2520plot%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1984%26volume%3D179%26issue%3D1%26spage%3D125%26epage%3D142%26doi%3D10.1016%2F0022-2836%2884%2990309-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Axelrod, J.</span></span> <span> </span><span class="NLM_article-title">Purification and properties of phenylethanolamine-N-methyl transferase</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1962</span>,  <span class="NLM_volume"><i>237</i></span>,  <span class="NLM_fpage">1657</span>– <span class="NLM_lpage">1660</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=13863458" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADyaF3sXkt1Sr" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=237&publication_year=1962&pages=1657-1660&author=J.+Axelrod&title=Purification+and+properties+of+phenylethanolamine-N-methyl+transferase"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Purification and properties of phenylethanolamine-N-methyl transferase</span></div><div class="casAuthors">Axelrod, Julius</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">1962</span>),
    <span class="NLM_cas:volume">237</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1657-60</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    </div><div class="casAbstract">The purification and properties of phenylethanolamine-N-methyl transferase are described.  This enzyme can N-methylate a variety of normally occurring and foreign phenylethanolamine derivs., but not phenethylamines.  It is highly localized in the adrenal medulla.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqILjUyyOtQq7Vg90H21EOLACvtfcHk0ljTNeTQ2BtBTw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaF3sXkt1Sr&md5=242704163c1ebda03d9ebe3738fdef3f</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DAxelrod%26aufirst%3DJ.%26atitle%3DPurification%2520and%2520properties%2520of%2520phenylethanolamine-N-methyl%2520transferase%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1962%26volume%3D237%26spage%3D1657%26epage%3D1660" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stratton, C. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poulin, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schramm, V. L.</span></span> <span> </span><span class="NLM_article-title">Kinetic isotope effects and transition state structure for human phenylethanolamine N-methyltransferase</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">342</span>– <span class="NLM_lpage">346</span>, <span class="refDoi"> DOI: 10.1021/acschembio.6b00922</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.6b00922" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BC28XitFWgt7fL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=342-346&issue=2&author=C.+F.+Strattonauthor=M.+B.+Poulinauthor=Q.+Duauthor=V.+L.+Schramm&title=Kinetic+isotope+effects+and+transition+state+structure+for+human+phenylethanolamine+N-methyltransferase&doi=10.1021%2Facschembio.6b00922"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Kinetic Isotope Effects and Transition State Structure for Human Phenylethanolamine N-Methyltransferase</span></div><div class="casAuthors">Stratton, Christopher F.; Poulin, Myles B.; Du, Quan; Schramm, Vern L.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">342-346</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Phenylethanolamine N-methyltransferase (PNMT) catalyzes the S-adenosyl-L-methionine (SAM)-dependent conversion of norepinephrine to epinephrine.  Epinephrine has been assocd. with crit. processes in humans including the control of respiration and blood pressure.  Addnl., PNMT activity has been suggested to play a role in hypertension and Alzheimer's disease.  Here, labeled SAM substrates were used to measure primary methyl-14C and 36S, and secondary methyl-3H, 5'-3H, 5'-14C intrinsic kinetic isotope effects for human PNMT.  The transition state of human PNMT was modeled by matching kinetic isotope effects predicted via quantum chem. calcns. to intrinsic values.  The model showed that human PNMT catalyzes the formation of epinephrine through an early SN2 transition state in which Me transfer is rate-limiting.  This anal. provides information on the geometry and electrostatics of the human PNMT transition state structure, details that enable the design of transition state analogs as potential PNMT inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrSBDuQMfV317Vg90H21EOLACvtfcHk0lhbvdcQ_gOL6w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitFWgt7fL&md5=cadf5bbe757ceba13081bd8e63a2f1d6</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1021%2Facschembio.6b00922&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.6b00922%26sid%3Dliteratum%253Aachs%26aulast%3DStratton%26aufirst%3DC.%2BF.%26aulast%3DPoulin%26aufirst%3DM.%2BB.%26aulast%3DDu%26aufirst%3DQ.%26aulast%3DSchramm%26aufirst%3DV.%2BL.%26atitle%3DKinetic%2520isotope%2520effects%2520and%2520transition%2520state%2520structure%2520for%2520human%2520phenylethanolamine%2520N-methyltransferase%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2017%26volume%3D12%26issue%3D2%26spage%3D342%26epage%3D346%26doi%3D10.1021%2Facschembio.6b00922" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fujioka, M.</span></span> <span> </span><span class="NLM_article-title">Mammalian small molecule methyltransferases: their structural and functional features</span>. <i>Int. J. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">1992</span>,  <span class="NLM_volume"><i>24</i></span> (<span class="NLM_issue">12</span>),  <span class="NLM_fpage">1917</span>– <span class="NLM_lpage">1924</span>, <span class="refDoi"> DOI: 10.1016/0020-711X(92)90287-B</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.1016%2F0020-711X%2892%2990287-B" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1473604" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADyaK3sXhsVais7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=1992&pages=1917-1924&issue=12&author=M.+Fujioka&title=Mammalian+small+molecule+methyltransferases%3A+their+structural+and+functional+features&doi=10.1016%2F0020-711X%2892%2990287-B"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Mammalian small molecule methyltransferases:  their structural and functional features</span></div><div class="casAuthors">Fujioka, Motoji</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Biochemistry</span>
        (<span class="NLM_cas:date">1992</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1917-24</span>CODEN:
                <span class="NLM_cas:coden">IJBOBV</span>;
        ISSN:<span class="NLM_cas:issn">0020-711X</span>.
    </div><div class="casAbstract">A review with 63 refs. on structural and functional features of mammalian S-adenosyl-methionine-dependent small mol. methyltransferases.  The methyltransferases have similar protomer mol. wts. in the range of 25,000-35,000.  Two common sequence motifs are found in all enzymes of known sequence.  Whereas the kinetic mechanisms may be different, the methyltransferases in the free form bind S-adenosylmethionine.  Most, if not all, of mammalian small mol. methyltransferases appear to have vicinal thiols in a catalytically important area of the enzyme.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqY96I4ugaO97Vg90H21EOLACvtfcHk0lhbvdcQ_gOL6w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXhsVais7k%253D&md5=92c43009a5f71452dffde48cbd07530f</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1016%2F0020-711X%2892%2990287-B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0020-711X%252892%252990287-B%26sid%3Dliteratum%253Aachs%26aulast%3DFujioka%26aufirst%3DM.%26atitle%3DMammalian%2520small%2520molecule%2520methyltransferases%253A%2520their%2520structural%2520and%2520functional%2520features%26jtitle%3DInt.%2520J.%2520Biochem.%26date%3D1992%26volume%3D24%26issue%3D12%26spage%3D1917%26epage%3D1924%26doi%3D10.1016%2F0020-711X%2892%2990287-B" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Petrossian, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clarke, S. G.</span></span> <span> </span><span class="NLM_article-title">Uncovering the human methyltransferasome</span>. <i>Mol. Cell. Proteomics</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>10</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">M110.000976</span>, <span class="refDoi"> DOI: 10.1074/mcp.M110.000976</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.1074%2Fmcp.M110.000976" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=20930037" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2011&pages=M110.000976&issue=1&author=T.+C.+Petrossianauthor=S.+G.+Clarke&title=Uncovering+the+human+methyltransferasome&doi=10.1074%2Fmcp.M110.000976"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1074%2Fmcp.M110.000976&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fmcp.M110.000976%26sid%3Dliteratum%253Aachs%26aulast%3DPetrossian%26aufirst%3DT.%2BC.%26aulast%3DClarke%26aufirst%3DS.%2BG.%26atitle%3DUncovering%2520the%2520human%2520methyltransferasome%26jtitle%3DMol.%2520Cell.%2520Proteomics%26date%3D2011%26volume%3D10%26issue%3D1%26spage%3DM110.000976%26doi%3D10.1074%2Fmcp.M110.000976" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gerlt, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bouvier, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davidson, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imker, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sadkhin, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slater, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whalen, K. L.</span></span> <span> </span><span class="NLM_article-title">Enzyme function initiative-enzyme similarity tool (EFI-EST): a web tool for generating protein sequence similarity networks</span>. <i>Biochim. Biophys. Acta, Proteins Proteomics</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>1854</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">1019</span>– <span class="NLM_lpage">1037</span>, <span class="refDoi"> DOI: 10.1016/j.bbapap.2015.04.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.1016%2Fj.bbapap.2015.04.015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=25900361" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BC2MXnvFSisb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1854&publication_year=2015&pages=1019-1037&issue=8&author=J.+A.+Gerltauthor=J.+T.+Bouvierauthor=D.+B.+Davidsonauthor=H.+J.+Imkerauthor=B.+Sadkhinauthor=D.+R.+Slaterauthor=K.+L.+Whalen&title=Enzyme+function+initiative-enzyme+similarity+tool+%28EFI-EST%29%3A+a+web+tool+for+generating+protein+sequence+similarity+networks&doi=10.1016%2Fj.bbapap.2015.04.015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Enzyme function initiative-enzyme similarity tool (EFI-EST): A web tool for generating protein sequence similarity networks</span></div><div class="casAuthors">Gerlt, John A.; Bouvier, Jason T.; Davidson, Daniel B.; Imker, Heidi J.; Sadkhin, Boris; Slater, David R.; Whalen, Katie L.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Proteins and Proteomics</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">1854</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1019-1037</span>CODEN:
                <span class="NLM_cas:coden">BBAPBW</span>;
        ISSN:<span class="NLM_cas:issn">1570-9639</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B. V.</span>)
        </div><div class="casAbstract">A review.  The Enzyme Function Initiative, an NIH/NIGMS-supported Large-Scale Collaborative Project (EFI; U54GM093342; http://enzymefunction.org/), is focused on devising and disseminating bioinformatics and computational tools as well as exptl. strategies for the prediction and assignment of functions (in vitro activities and in vivo physiol./metabolic roles) to uncharacterized enzymes discovered in genome projects.  Protein sequence similarity networks (SSNs) are visually powerful tools for analyzing sequence relationships in protein families (H.J. Atkinson, J.H. Morris, T.E. Ferrin, and P.C. Babbitt, PLoS One 2009, 4, e4345).  However, the members of the biol./biomedical community have not had access to the capability to generate SSNs for their "favorite" protein families.  In this article we announce the EFI-EST (Enzyme Function Initiative-Enzyme Similarity Tool) web tool (http://efi.igb.illinois.edu/efi-est/) that is available without cost for the automated generation of SSNs by the community.  The tool can create SSNs for the "closest neighbors" of a user-supplied protein sequence from the UniProt database (Option A) or of members of any user-supplied Pfam and/or InterPro family (Option B).  We provide an introduction to SSNs, a description of EFI-EST, and a demonstration of the use of EFI-EST to explore sequence-function space in the OMP decarboxylase superfamily (PF00215).  This article is designed as a tutorial that will allow members of the community to use the EFI-EST web tool for exploring sequence/function space in protein families.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqfzrew87TmDLVg90H21EOLACvtfcHk0lhPu9RW24wztw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXnvFSisb0%253D&md5=4897a23daaf5c2437acb0bed815095f8</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1016%2Fj.bbapap.2015.04.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbapap.2015.04.015%26sid%3Dliteratum%253Aachs%26aulast%3DGerlt%26aufirst%3DJ.%2BA.%26aulast%3DBouvier%26aufirst%3DJ.%2BT.%26aulast%3DDavidson%26aufirst%3DD.%2BB.%26aulast%3DImker%26aufirst%3DH.%2BJ.%26aulast%3DSadkhin%26aufirst%3DB.%26aulast%3DSlater%26aufirst%3DD.%2BR.%26aulast%3DWhalen%26aufirst%3DK.%2BL.%26atitle%3DEnzyme%2520function%2520initiative-enzyme%2520similarity%2520tool%2520%2528EFI-EST%2529%253A%2520a%2520web%2520tool%2520for%2520generating%2520protein%2520sequence%2520similarity%2520networks%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Proteins%2520Proteomics%26date%3D2015%26volume%3D1854%26issue%3D8%26spage%3D1019%26epage%3D1037%26doi%3D10.1016%2Fj.bbapap.2015.04.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zallot, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oberg, N. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerlt, J. A.</span></span> <span> </span><span class="NLM_article-title">’Democratized’ genomic enzymology web tools for functional assignment</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">77</span>– <span class="NLM_lpage">85</span>, <span class="refDoi"> DOI: 10.1016/j.cbpa.2018.09.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.1016%2Fj.cbpa.2018.09.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=30268904" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvVaht7zK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2018&pages=77-85&author=R.+Zallotauthor=N.+O.+Obergauthor=J.+A.+Gerlt&title=%E2%80%99Democratized%E2%80%99+genomic+enzymology+web+tools+for+functional+assignment&doi=10.1016%2Fj.cbpa.2018.09.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">'Democratized' genomic enzymology web tools for functional assignment</span></div><div class="casAuthors">Zallot, Remi; Oberg, Nils O.; Gerlt, John A.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">77-85</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The protein databases contain an exponentially growing no. of sequences as a result of the recent increase in ease and decrease in cost of genome sequencing.  The rate of data accumulation far exceeds the rate of functional studies, producing an increase in genomic 'dark matter', sequences for which no precise and validated function is defined.  Publicly accessible, that is 'democratized,' genomic enzymol. web tools are essential to leverage the protein and genome databases for discovery of the in vitro activities and in vivo functions of novel enzymes and proteins belonging to the dark matter.  In this review, we discuss the use of web tools that have proven successful for functional assignment.  We also describe a mechanism for ensuring the capture of published functional data so that the quality of both curated and automated annotations transfer can be improved.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojSOg1ygeEjLVg90H21EOLACvtfcHk0lhPu9RW24wztw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvVaht7zK&md5=67f7e0230fb75906c97a782db04e4398</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2018.09.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2018.09.009%26sid%3Dliteratum%253Aachs%26aulast%3DZallot%26aufirst%3DR.%26aulast%3DOberg%26aufirst%3DN.%2BO.%26aulast%3DGerlt%26aufirst%3DJ.%2BA.%26atitle%3D%25E2%2580%2599Democratized%25E2%2580%2599%2520genomic%2520enzymology%2520web%2520tools%2520for%2520functional%2520assignment%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2018%26volume%3D47%26spage%3D77%26epage%3D85%26doi%3D10.1016%2Fj.cbpa.2018.09.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinshilboum, R. M.</span></span> <span> </span><span class="NLM_article-title">Rabbit lung indolethylamine N-methyltransferase. cDNA and gene cloning and characterization</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>273</i></span> (<span class="NLM_issue">51</span>),  <span class="NLM_fpage">34502</span>– <span class="NLM_lpage">34510</span>, <span class="refDoi"> DOI: 10.1074/jbc.273.51.34502</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.1074%2Fjbc.273.51.34502" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=9852119" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADyaK1MXntleq" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=273&publication_year=1998&pages=34502-34510&issue=51&author=M.+A.+Thompsonauthor=R.+M.+Weinshilboum&title=Rabbit+lung+indolethylamine+N-methyltransferase.+cDNA+and+gene+cloning+and+characterization&doi=10.1074%2Fjbc.273.51.34502"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Rabbit lung indolethylamine N-methyltransferase cDNA and gene cloning and characterization</span></div><div class="casAuthors">Thompson, Michael A.; Weinshilboum, Richard M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">273</span>
        (<span class="NLM_cas:issue">51</span>),
    <span class="NLM_cas:pages">34502-34510</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Indolethylamine N-methyltransferase (INMT) catalyzes the N-methylation of tryptamine and structurally related compds.  This reaction has been studied because of its possible role in the in vivo synthesis of psychoactive compds. or neurotoxins and has been characterized biochem. in prepns. of rabbit lung.  Therefore, the authors set out to purify rabbit lung INMT, to clone and express its cDNA, and to clone and structurally characterize its gene as steps toward understanding the function and regulation of this enzyme.  Rabbit lung INMT was purified and partial amino acid sequence was obtained.  A polymerase chain reaction-based approach was then used to clone a rabbit lung INMT cDNA with a 792-base pair open reading frame that encoded a 263-amino acid protein with a predicted mol. mass of 29 kDa.  When the cDNA was expressed in COS-1 cells, the encoded protein catalyzed the methylation of tryptamine and structurally related compds., and was inhibited by two products of the reaction, S-adenosyl-L-homocysteine (AdoHcy) and N,N-dimethyltryptamine, as well as antimigraine drugs that are structurally related to N,N-dimethyltryptamine.  Northern blot anal. demonstrated the presence of 2.0-kilobase mRNA species in rabbit lung, liver and, at lower levels, in brain.  The cDNA was then used to clone the rabbit INMT gene.  That gene had three exons and was structurally similar to the genes for nicotinamide N-methyltransferase and phenylethanolamine N-methyltransferase in several species.  Cloning and expression of a rabbit lung INMT cDNA and cloning of the rabbit INMT gene represent important steps toward detn. of the function and regulation of this mammalian methyltransferase enzyme.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXgOW945aeZ7Vg90H21EOLACvtfcHk0lhPu9RW24wztw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXntleq&md5=fa4393a632c0fbc1c52d6f300cb5b463</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1074%2Fjbc.273.51.34502&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.273.51.34502%26sid%3Dliteratum%253Aachs%26aulast%3DThompson%26aufirst%3DM.%2BA.%26aulast%3DWeinshilboum%26aufirst%3DR.%2BM.%26atitle%3DRabbit%2520lung%2520indolethylamine%2520N-methyltransferase.%2520cDNA%2520and%2520gene%2520cloning%2520and%2520characterization%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1998%26volume%3D273%26issue%3D51%26spage%3D34502%26epage%3D34510%26doi%3D10.1074%2Fjbc.273.51.34502" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Holm, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laakso, L. M.</span></span> <span> </span><span class="NLM_article-title">Dali server update</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>44</i></span> (<span class="NLM_issue">W1</span>),  <span class="NLM_fpage">W351</span>– <span class="NLM_lpage">W355</span>, <span class="refDoi"> DOI: 10.1093/nar/gkw357</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.1093%2Fnar%2Fgkw357" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=27131377" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtV2itrrO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=W351-W355&issue=W1&author=L.+Holmauthor=L.+M.+Laakso&title=Dali+server+update&doi=10.1093%2Fnar%2Fgkw357"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Dali server update</span></div><div class="casAuthors">Holm, Liisa; Laakso, Laura M.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">W1</span>),
    <span class="NLM_cas:pages">W351-W355</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The Dali server is a network service for comparing protein structures in 3D.  In favorable cases, comparing 3D structures may reveal biol. interesting similarities that are not detectable by comparing sequences.  The Dali server has been running in various places for over 20 years and is used routinely by crystallographers on newly solved structures.  The latest update of the server provides enhanced analytics for the study of sequence and structure conservation.  The server performs three types of structure comparisons: (i) Protein Data Bank (PDB) search compares one query structure against those in the PDB and returns a list of similar structures; (ii) pairwise comparison compares one query structure against a list of structures specified by the user; and (iii) all against all structure comparison returns a structural similarity matrix, a dendrogram and a multidimensional scaling projection of a set of structures specified by the user.  Structural superimpositions are visualized using the Java-free WebGL viewer PV.  The structural alignment view is enhanced by sequence similarity searches against Uniprot.  The combined structure-sequence alignment information is compressed to a stack of aligned sequence logos.  In the stack, each structure is structurally aligned to the query protein and represented by a sequence logo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo9jpe1HGTYrLVg90H21EOLACvtfcHk0lhDYZwb5vqvHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtV2itrrO&md5=cd2b5fabf6a01a4b35f350342ae7ddc2</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkw357&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkw357%26sid%3Dliteratum%253Aachs%26aulast%3DHolm%26aufirst%3DL.%26aulast%3DLaakso%26aufirst%3DL.%2BM.%26atitle%3DDali%2520server%2520update%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2016%26volume%3D44%26issue%3DW1%26spage%3DW351%26epage%3DW355%26doi%3D10.1093%2Fnar%2Fgkw357" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Peng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilk, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adjei, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salavaggione, O. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinshilboum, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yee, V. C.</span></span> <span> </span><span class="NLM_article-title">Structural basis of substrate recognition in thiopurine S-methyltransferase</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>47</i></span> (<span class="NLM_issue">23</span>),  <span class="NLM_fpage">6216</span>– <span class="NLM_lpage">6225</span>, <span class="refDoi"> DOI: 10.1021/bi800102x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi800102x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BD1cXlvFOgsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2008&pages=6216-6225&issue=23&author=Y.+Pengauthor=Q.+Fengauthor=D.+Wilkauthor=A.+A.+Adjeiauthor=O.+E.+Salavaggioneauthor=R.+M.+Weinshilboumauthor=V.+C.+Yee&title=Structural+basis+of+substrate+recognition+in+thiopurine+S-methyltransferase&doi=10.1021%2Fbi800102x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis of substrate recognition in thiopurine S-methyltransferase</span></div><div class="casAuthors">Peng, Yi; Feng, Qiping; Wilk, Dennis; Adjei, Araba A.; Salavaggione, Oreste E.; Weinshilboum, Richard M.; Yee, Vivien C.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">6216-6225</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Thiopurine S-methyltransferase (I) modulates the cytotoxic effects of thiopurine prodrugs such as 6-mercaptopurine by methylating them in a reaction using S-adenosyl-L-methionine as donor.  Patients with I variant allozymes exhibit diminished levels of protein and/or enzyme activity and are at risk for thiopurine drug-induced toxicity.  Here, the authors detd. 2 crystal structures of murine I, one as a binary complex with the product, S-adenosyl-L-homocysteine (II), and the other as a ternary complex with II and the substrate, 6-mercaptopurine (III), to 1.8 and 2.0 Å resoln., resp.  Comparison of the structures revealed that an active site loop became ordered upon III binding.  The positions of the 2 ligands were consistent with the expected SN2 reaction mechanism.  Amino acid residues Arg-147 and Arg-221, the only polar amino acids near III, were highlighted as possible participants in substrate deprotonation.  To probe whether these residues were important for catalysis, point mutants were prepd. in human I.  Substitution of Arg-152 (Arg-147 in murine I) with Glu decreased the Vmax and increased the Km for III but not the Km for S-adenosyl-L-methionine.  Substitution at this position with Ala or His residues and similar substitutions of Arg-226 (Arg-221 in murine I) resulted in no effect on enzyme activity.  The double mutant R152A/R226A exhibited a decreased Vmax and increased Km for III.  These observations suggested that either Arg-152 or Arg-226 may participate in some fashion in the I reaction, with one residue compensating when the other is altered, and that Arg-152 may interact with the substrate more directly than Arg-226, consistent with observations in the murine I crystal structure.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqzr1Kpnl7sV7Vg90H21EOLACvtfcHk0lhDYZwb5vqvHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXlvFOgsro%253D&md5=f1fc23f59a78a0a7b0ac79e611dc6df4</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1021%2Fbi800102x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi800102x%26sid%3Dliteratum%253Aachs%26aulast%3DPeng%26aufirst%3DY.%26aulast%3DFeng%26aufirst%3DQ.%26aulast%3DWilk%26aufirst%3DD.%26aulast%3DAdjei%26aufirst%3DA.%2BA.%26aulast%3DSalavaggione%26aufirst%3DO.%2BE.%26aulast%3DWeinshilboum%26aufirst%3DR.%2BM.%26aulast%3DYee%26aufirst%3DV.%2BC.%26atitle%3DStructural%2520basis%2520of%2520substrate%2520recognition%2520in%2520thiopurine%2520S-methyltransferase%26jtitle%3DBiochemistry%26date%3D2008%26volume%3D47%26issue%3D23%26spage%3D6216%26epage%3D6225%26doi%3D10.1021%2Fbi800102x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ames, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Selassie, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woodson, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Loon, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansch, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinshilboum, R. M.</span></span> <span> </span><span class="NLM_article-title">Thiopurine methyltransferase: structure-activity relationships for benzoic acid inhibitors and thiophenol substrates</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1986</span>,  <span class="NLM_volume"><i>29</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">354</span>– <span class="NLM_lpage">358</span>, <span class="refDoi"> DOI: 10.1021/jm00153a009</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm00153a009" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADyaL28XosVamug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=1986&pages=354-358&issue=3&author=M.+M.+Amesauthor=C.+D.+Selassieauthor=L.+C.+Woodsonauthor=J.+A.+Van+Loonauthor=C.+Hanschauthor=R.+M.+Weinshilboum&title=Thiopurine+methyltransferase%3A+structure-activity+relationships+for+benzoic+acid+inhibitors+and+thiophenol+substrates&doi=10.1021%2Fjm00153a009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Thiopurine methyltransferase:  structure-activity relationships for benzoic acid inhibitors and thiophenol substrates</span></div><div class="casAuthors">Ames, Matthew M.; Selassie, Cynthia Dias; Woodson, Lee C.; Van Loon, Jon A.; Hansch, Corwin; Weinshilboum, Richard M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1986</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">354-8</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    </div><div class="casAbstract">Twenty-seven substituted benzoic acids were studied as inhibitors of partially purified human renal thiopurine methyltransferase (TPMT).  QSAR anal. resulted in the following equation: pI50 = 1.25π'3 + 0.73MR3,4 + 2.92 (where pI50 is the -log of the concn. of compd. that inhibits TPMT activity by 50% (IC50), π'3 is the relative hydrophobicity of the more hydrophobic of the 2 meta substituents, and MR3,4 is the molar refractivity of the more hydrophobic of the 2 meta substituents and of the para substituent on the Ph ring).  In addn., 14 substituted thiophenols were tested as substrates for the enzyme.  All 14 thiophenols tested were excellent substrates with Km values (0.8-7.8 μM) that were ≥2 orders of magnitude lower than those of any known thiopurine substrate for TPMT.  However, there was no discernible relation between the activities of thiophenol substrate and their physicochem. parameters.  Apparently, benzoic acid inhibitors of and thiophenol substrates for TPMT may interact with different sites on the enzyme.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhvqO1u1HSFLVg90H21EOLACvtfcHk0lhDYZwb5vqvHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL28XosVamug%253D%253D&md5=1f89c18b92d03a2696c3393bb91224da</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1021%2Fjm00153a009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm00153a009%26sid%3Dliteratum%253Aachs%26aulast%3DAmes%26aufirst%3DM.%2BM.%26aulast%3DSelassie%26aufirst%3DC.%2BD.%26aulast%3DWoodson%26aufirst%3DL.%2BC.%26aulast%3DVan%2BLoon%26aufirst%3DJ.%2BA.%26aulast%3DHansch%26aufirst%3DC.%26aulast%3DWeinshilboum%26aufirst%3DR.%2BM.%26atitle%3DThiopurine%2520methyltransferase%253A%2520structure-activity%2520relationships%2520for%2520benzoic%2520acid%2520inhibitors%2520and%2520thiophenol%2520substrates%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1986%26volume%3D29%26issue%3D3%26spage%3D354%26epage%3D358%26doi%3D10.1021%2Fjm00153a009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Martinez
Molina, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nordlund, P.</span></span> <span> </span><span class="NLM_article-title">The cellular thermal shift assay: a novel biophysical assay for in situ drug target engagement and mechanistic biomarker studies</span>. <i>Annu. Rev. Pharmacol. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>56</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">141</span>– <span class="NLM_lpage">161</span>, <span class="refDoi"> DOI: 10.1146/annurev-pharmtox-010715-103715</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.1146%2Fannurev-pharmtox-010715-103715" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=26566155" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVCqs7nI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2016&pages=141-161&issue=1&author=D.+Martinez%0AMolinaauthor=P.+Nordlund&title=The+cellular+thermal+shift+assay%3A+a+novel+biophysical+assay+for+in+situ+drug+target+engagement+and+mechanistic+biomarker+studies&doi=10.1146%2Fannurev-pharmtox-010715-103715"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">The Cellular Thermal Shift Assay: A Novel Biophysical Assay for In Situ Drug Target Engagement and Mechanistic Biomarker Studies</span></div><div class="casAuthors">Martinez Molina, Daniel; Nordlund, Paer</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Pharmacology and Toxicology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">141-161</span>CODEN:
                <span class="NLM_cas:coden">ARPTDI</span>;
        ISSN:<span class="NLM_cas:issn">0362-1642</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews</span>)
        </div><div class="casAbstract">A drug must engage its intended target to achieve its therapeutic effect.  However, conclusively measuring target engagement (TE) in situ is challenging.  This complicates preclin. development and is considered a key factor in the high rate of attrition in clin. trials.  Here, we discuss a recently developed, label-free, biophys. assay, the cellular thermal shift assay (CETSA), which facilitates the direct assessment of TE in cells and tissues at various stages of drug development.  CETSA also reveals biochem. events downstream of drug binding and therefore provides a promising means of establishing mechanistic biomarkers.  The implementation of proteome-wide CETSA using quant. mass spectrometry represents a novel strategy for defining off-target toxicity and polypharmacol. and for identifying downstream mechanistic biomarkers.  The first year of CETSA applications in the literature has focused on TE studies in cell culture systems and has confirmed the broad applicability of CETSA to many different target families.  The next phase of CETSA applications will likely encompass comprehensive animal and patient studies, and CETSA will likely serve as a very valuable tool in many stages of preclin. and clin. drug development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCOdmURGPcTbVg90H21EOLACvtfcHk0lhDYZwb5vqvHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVCqs7nI&md5=092ac50d54d92ef9746a90af8c634e94</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1146%2Fannurev-pharmtox-010715-103715&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-pharmtox-010715-103715%26sid%3Dliteratum%253Aachs%26aulast%3DMartinez%2BMolina%26aufirst%3DD.%26aulast%3DNordlund%26aufirst%3DP.%26atitle%3DThe%2520cellular%2520thermal%2520shift%2520assay%253A%2520a%2520novel%2520biophysical%2520assay%2520for%2520in%2520situ%2520drug%2520target%2520engagement%2520and%2520mechanistic%2520biomarker%2520studies%26jtitle%3DAnnu.%2520Rev.%2520Pharmacol.%2520Toxicol.%26date%3D2016%26volume%3D56%26issue%3D1%26spage%3D141%26epage%3D161%26doi%3D10.1146%2Fannurev-pharmtox-010715-103715" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kung, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malinoski, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prakash, G. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span> <span> </span><span class="NLM_article-title">Development of alkyne-containing pyrazolopyrimidines to overcome drug resistance of Bcr-Abl kinase</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span> (<span class="NLM_issue">23</span>),  <span class="NLM_fpage">9228</span>– <span class="NLM_lpage">9237</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01125</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01125" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVaht7vL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=9228-9237&issue=23&author=X.+Liuauthor=A.+Kungauthor=B.+Malinoskiauthor=G.+K.+Prakashauthor=C.+Zhang&title=Development+of+alkyne-containing+pyrazolopyrimidines+to+overcome+drug+resistance+of+Bcr-Abl+kinase&doi=10.1021%2Facs.jmedchem.5b01125"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Development of Alkyne-Containing Pyrazolopyrimidines To Overcome Drug Resistance of Bcr-Abl Kinase</span></div><div class="casAuthors">Liu, Xu; Kung, Alvin; Malinoski, Brock; Prakash, G. K. Surya; Zhang, Chao</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">9228-9237</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Despite the success of imatinib at inhibiting Bcr-Abl and treating chronic myelogenous leukemia (CML), resistance to the therapy occurs over time in patients.  In particular, the resistance to imatinib caused by the gatekeeper mutation T315I in Bcr-Abl remains a challenge in the clinic.  Inspired by the successful development of ponatinib to curb drug resistance, we hypothesize that the incorporation of an alkyne linker in other heterocyclic scaffolds can also achieve potent inhibition of Bcr-AblT315I by allowing for simultaneous occupancy of both the active site and the allosteric pocket in the Abl kinase domain.  Herein, we describe the design, synthesis, and characterization of a series of alkyne-contg. pyrazolopyrimidines as Bcr-Abl inhibitors.  Our results demonstrate that some alkyne-contg. pyrazolopyrimidines potently inhibit not only AblT315I in vitro but also Bcr-AblT315I in cells.  These pyrazolopyrimidines can serve as lead compds. for future development of novel targeted therapy to overcome drug resistance of CML.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvCwLSMJhHxLVg90H21EOLACvtfcHk0ljJcWrwh-rVCw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVaht7vL&md5=adfdddc095b76175f04f8ace9f51b847</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01125&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01125%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DKung%26aufirst%3DA.%26aulast%3DMalinoski%26aufirst%3DB.%26aulast%3DPrakash%26aufirst%3DG.%2BK.%26aulast%3DZhang%26aufirst%3DC.%26atitle%3DDevelopment%2520of%2520alkyne-containing%2520pyrazolopyrimidines%2520to%2520overcome%2520drug%2520resistance%2520of%2520Bcr-Abl%2520kinase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26issue%3D23%26spage%3D9228%26epage%3D9237%26doi%3D10.1021%2Facs.jmedchem.5b01125" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Srinivasan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ng, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalesh, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, S. Q.</span></span> <span> </span><span class="NLM_article-title">Methods of using click chemistry in the discovery of enzyme inhibitors</span>. <i>Nat. Protoc.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>2</i></span> (<span class="NLM_issue">11</span>),  <span class="NLM_fpage">2655</span>– <span class="NLM_lpage">2664</span>, <span class="refDoi"> DOI: 10.1038/nprot.2007.323</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.1038%2Fnprot.2007.323" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=18007601" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlSlsrrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2007&pages=2655-2664&issue=11&author=R.+Srinivasanauthor=J.+Liauthor=S.+L.+Ngauthor=K.+A.+Kaleshauthor=S.+Q.+Yao&title=Methods+of+using+click+chemistry+in+the+discovery+of+enzyme+inhibitors&doi=10.1038%2Fnprot.2007.323"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Methods of using click chemistry in the discovery of enzyme inhibitors</span></div><div class="casAuthors">Srinivasan, Rajavel; Li, Junqi; Ng, Su Ling; Kalesh, Karunakaran A.; Yao, Shao Q.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2655-2664</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">This protocol describes the step-by-step procedures for the efficient assembly of bidentate inhibitor libraries of a target enzyme, using the so-called 'click chem.' between an alkyne-bearing core group and an azide-modified peripheral group, followed by direct biol. screening for the identification of potential 'hits'.  The reaction is highlighted by its modularity, high efficiency (∼100% yield in most cases) and tolerance toward many functional groups present in the fragments, as well as biocompatibility (typically carried out in aq. conditions with small amts. of biocompatible catalysts).  The approach consists of three steps: (i) chem. synthesis of alkyne-bearing protein tyrosine phosphatase or matrix metalloprotease core groups and diverse azide-modified peripheral groups; (ii) click chem. to assemble the bidentate inhibitor libraries; and (iii) direct screening of the libraries with target enzymes using 384-well microplate assays.  Following the chem. synthesis of the core and peripheral groups and optimization of the click chem. conditions (∼1 wk), steps (ii) and (iii) take 3 d to complete (∼1-2 d for library assembly and 1 d for inhibitor screening).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvVU_E7kmG4rVg90H21EOLACvtfcHk0ljJcWrwh-rVCw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlSlsrrJ&md5=21306eca61966c0d8d1bbb504a17fdf4</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2007.323&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2007.323%26sid%3Dliteratum%253Aachs%26aulast%3DSrinivasan%26aufirst%3DR.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DNg%26aufirst%3DS.%2BL.%26aulast%3DKalesh%26aufirst%3DK.%2BA.%26aulast%3DYao%26aufirst%3DS.%2BQ.%26atitle%3DMethods%2520of%2520using%2520click%2520chemistry%2520in%2520the%2520discovery%2520of%2520enzyme%2520inhibitors%26jtitle%3DNat.%2520Protoc.%26date%3D2007%26volume%3D2%26issue%3D11%26spage%3D2655%26epage%3D2664%26doi%3D10.1038%2Fnprot.2007.323" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Burchat, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chong, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nielsen, N.</span></span> <span> </span><span class="NLM_article-title">Titration of alkyllithiums with a simple reagent to a blue endpoint</span>. <i>J. Organomet. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>542</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">281</span>– <span class="NLM_lpage">283</span>, <span class="refDoi"> DOI: 10.1016/S0022-328X(97)00143-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.1016%2FS0022-328X%2897%2900143-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADyaK2sXmt1Glurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=542&publication_year=1997&pages=281-283&issue=2&author=A.+F.+Burchatauthor=J.+M.+Chongauthor=N.+Nielsen&title=Titration+of+alkyllithiums+with+a+simple+reagent+to+a+blue+endpoint&doi=10.1016%2FS0022-328X%2897%2900143-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Titration of alkyllithiums with a simple reagent to a blue endpoint</span></div><div class="casAuthors">Burchat, Andrew F.; Chong, J. Michael; Nielsen, Nathan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organometallic Chemistry</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">542</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">281-283</span>CODEN:
                <span class="NLM_cas:coden">JORCAI</span>;
        ISSN:<span class="NLM_cas:issn">0022-328X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Reaction of readily available N-benzylbenzamide (1) with a slight excess of BuLi produces a deep blue dianion.  Under suitable conditions (THF, -40°), 1 may be used as a stoichiometric reagent to titrate BuLi, s-BuLi, and t-BuLi with good precision.  Less reactive alkyllithiums (MeLi and MeLi·LiBr), aryllithiums, and LDA may be titrated at higher temps. (-20 to 0°).  Titers are comparable to those obtained using the Gilman double titrn. method, and there appears to be no interference from alkoxides.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrY29vXPn-z0LVg90H21EOLACvtfcHk0ljJcWrwh-rVCw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXmt1Glurg%253D&md5=424c3be5372d4dea12d9d25d55fc778e</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1016%2FS0022-328X%2897%2900143-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0022-328X%252897%252900143-5%26sid%3Dliteratum%253Aachs%26aulast%3DBurchat%26aufirst%3DA.%2BF.%26aulast%3DChong%26aufirst%3DJ.%2BM.%26aulast%3DNielsen%26aufirst%3DN.%26atitle%3DTitration%2520of%2520alkyllithiums%2520with%2520a%2520simple%2520reagent%2520to%2520a%2520blue%2520endpoint%26jtitle%3DJ.%2520Organomet.%2520Chem.%26date%3D1997%26volume%3D542%26issue%3D2%26spage%3D281%26epage%3D283%26doi%3D10.1016%2FS0022-328X%2897%2900143-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Suffert, J.</span></span> <span> </span><span class="NLM_article-title">Simple direct titration of organolithium reagents using N-pivaloyl-<i>o</i>-toluidine and/or N-pivaloyl-<i>o</i>-benzylaniline</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1989</span>,  <span class="NLM_volume"><i>54</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">509</span>– <span class="NLM_lpage">510</span>, <span class="refDoi"> DOI: 10.1021/jo00263a052</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo00263a052" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADyaL1MXhtFCns74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=1989&pages=509-510&issue=2&author=J.+Suffert&title=Simple+direct+titration+of+organolithium+reagents+using+N-pivaloyl-o-toluidine+and%2For+N-pivaloyl-o-benzylaniline&doi=10.1021%2Fjo00263a052"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Simple direct titration of organolithium reagents using N-pivaloyl-o-toluidine and/or N-pivaloyl-o-benzylaniline</span></div><div class="casAuthors">Suffert, Jean</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">1989</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">509-10</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    </div><div class="casAbstract">Titrn. of organolithium reagents using N-pivaloyl o-toluidine (I) and/or N-pivaloyl o-benzylaniline (II) is described.  Compds. I and II are easily accessible, cryst., non hygroscopic, stable and may be stored without any precautions.  Comparative titrn. of organolithium reagents with four different methods of titrn. showed good agreement with these commonly used anal. procedures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_1HIyqf6jSLVg90H21EOLACvtfcHk0lg8zHhfjx3AKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1MXhtFCns74%253D&md5=2c5f9c9286086e2bdfa9f6dbd5e03bc0</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1021%2Fjo00263a052&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo00263a052%26sid%3Dliteratum%253Aachs%26aulast%3DSuffert%26aufirst%3DJ.%26atitle%3DSimple%2520direct%2520titration%2520of%2520organolithium%2520reagents%2520using%2520N-pivaloyl-o-toluidine%2520and%252For%2520N-pivaloyl-o-benzylaniline%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D1989%26volume%3D54%26issue%3D2%26spage%3D509%26epage%3D510%26doi%3D10.1021%2Fjo00263a052" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Werner, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poelsma, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Windmüller, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barth, D.</span></span> <span> </span><span class="NLM_article-title">Synthesis and reactivity of bis(ethene) rhodium(I) and iridium(I) carboxylato complexes</span>. <i>Chem. Ber.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>129</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">647</span>– <span class="NLM_lpage">652</span>, <span class="refDoi"> DOI: 10.1002/cber.19961290609</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.1002%2Fcber.19961290609" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADyaK28XjtlWntLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=129&publication_year=1996&pages=647-652&issue=6&author=H.+Wernerauthor=S.+Poelsmaauthor=M.+E.+Schneiderauthor=B.+Windm%C3%BCllerauthor=D.+Barth&title=Synthesis+and+reactivity+of+bis%28ethene%29+rhodium%28I%29+and+iridium%28I%29+carboxylato+complexes&doi=10.1002%2Fcber.19961290609"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and reactivity of bis(ethene) rhodium(I) and iridium(I) carboxylato complexes</span></div><div class="casAuthors">Werner, Helmut; Poelsma, Sybo; Schneider, Michael E.; Windmueller, Bettina; Barth, Dietmar</div><div class="citationInfo"><span class="NLM_cas:title">Chemische Berichte</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">129</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">647-652</span>CODEN:
                <span class="NLM_cas:coden">CHBEAM</span>;
        ISSN:<span class="NLM_cas:issn">0009-2940</span>.
    
            (<span class="NLM_cas:orgname">VCH</span>)
        </div><div class="casAbstract">[Rh(μ-OAc)(C2H4)2]2 (I) and analogous carboxylatobis(olefin)rhodium(I) complexes [Rh(μ-O2CR)(olefin)2]2 (R = Me, CF3, Ph) were almost quant. prepd. under heterogeneous conditions from [RhCl(C8H14)2]2 or [RhCl(C2H4)2]2 and AcONa.3H2O or RCO2H/NaOH, resp.  The x-ray crystal structure anal. of I confirmed the bridging position of the acetato ligands.  The synthesis of [Ir(μ-OAc)(C2H4)2]2 (II) was carried out similarly by using [IrCl(C2H4)2]2 and AcONa.3H2O as starting materials.  The monoethene complex trans-[Ir(η1-OAc)(C2H4)(PiPr3)2] was obtained from II and PiPr3.  It reacted with H2 and HC≡CPh by oxidative addn. to give [IrH2(η2-OAc)(PiPr3)2] (III) and [IrH(C≡CPh)(η2-OAc)(PiPr3)2], resp.  Compd. III was also prepd. from [IrH2Cl(PiPr3)2] and AcONa.3H2O.  The new bis(ethene) complexes I, [Rh(μ-O2CPh)(C2H4)2]2, and II are catalytically less active than [RhCl(C2H4)2]2 in the reaction of C2H4 and Ph2CN2 and gave as C-C coupling product not only 1,1-diphenylpropene but also mixts. of it and 1,1-diphenylcyclopropane.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqA1T69GpaZPLVg90H21EOLACvtfcHk0lg8zHhfjx3AKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XjtlWntLc%253D&md5=f6ce07d24bfb42960d19ce7fd738bfa0</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1002%2Fcber.19961290609&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcber.19961290609%26sid%3Dliteratum%253Aachs%26aulast%3DWerner%26aufirst%3DH.%26aulast%3DPoelsma%26aufirst%3DS.%26aulast%3DSchneider%26aufirst%3DM.%2BE.%26aulast%3DWindm%25C3%25BCller%26aufirst%3DB.%26aulast%3DBarth%26aufirst%3DD.%26atitle%3DSynthesis%2520and%2520reactivity%2520of%2520bis%2528ethene%2529%2520rhodium%2528I%2529%2520and%2520iridium%2528I%2529%2520carboxylato%2520complexes%26jtitle%3DChem.%2520Ber.%26date%3D1996%26volume%3D129%26issue%3D6%26spage%3D647%26epage%3D652%26doi%3D10.1002%2Fcber.19961290609" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="note"><p class="first last">The authors note that at times better results were obtained when the asymmetric hydrogenation catalyst was prepared freshly before use from [Rh(nbd)<sub>2</sub>]BF<sub>4</sub> (CAS: 36620-11-8, obtained from Strem Chemicals, Inc. (catalogue no. 45-0230)) and (<i>S</i>,<i>S</i>)-MeBPE ligand (CAS: 136779-26-5, obtained from Strem Chemicals, Inc. (catalogue no. 15-0105)).</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="rightTab-ref64"><div class="reference"><strong class="refLabel"><a href="#ref64" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ellis, F.</span>; <span class="NLM_string-name">Fulton, H. E.</span>; <span class="NLM_string-name">Hall, A.</span>; <span class="NLM_string-name">Jaxa-Chamiec, A. A.</span>; <span class="NLM_string-name">Swanson, S.</span>; <span class="NLM_string-name">Vile, S.</span></span> <i>Acetylene Compounds</i>. <span class="NLM_patent">WO2004026890 (A1)</span>, <span class="NLM_year">2004</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2004&author=F.+Ellis&author=H.+E.+Fulton&author=A.+Hall&author=A.+A.+Jaxa-Chamiec&author=S.+Swanson&author=S.+Vile&title=Acetylene+Compounds"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DEllis%26aufirst%3DF.%26jtitle%3DAcetylene%2520Compounds%26date%3D2004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref65"><div class="reference"><strong class="refLabel"><a href="#ref65" class="rightTabRefNumLink">65</a></strong><div class="NLM_citation" id="rightTab-cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pedersen, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenbohm, C.</span></span> <span> </span><span class="NLM_article-title">Dry column vacuum chromatography</span>. <i>Synthesis</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>2001</i></span> (<span class="NLM_issue">16</span>),  <span class="NLM_fpage">2431</span>– <span class="NLM_lpage">2434</span>, <span class="refDoi"> DOI: 10.1055/s-2001-18722</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.1055%2Fs-2001-18722" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2001&publication_year=2001&pages=2431-2434&issue=16&author=D.+S.+Pedersenauthor=C.+Rosenbohm&title=Dry+column+vacuum+chromatography&doi=10.1055%2Fs-2001-18722"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1055%2Fs-2001-18722&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1055%252Fs-2001-18722%26sid%3Dliteratum%253Aachs%26aulast%3DPedersen%26aufirst%3DD.%2BS.%26aulast%3DRosenbohm%26aufirst%3DC.%26atitle%3DDry%2520column%2520vacuum%2520chromatography%26jtitle%3DSynthesis%26date%3D2001%26volume%3D2001%26issue%3D16%26spage%3D2431%26epage%3D2434%26doi%3D10.1055%2Fs-2001-18722" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref66"><div class="reference"><strong class="refLabel"><a href="#ref66" class="rightTabRefNumLink">66</a></strong><div class="NLM_citation" id="rightTab-cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cox, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whittaker, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lalic, G.</span></span> <span> </span><span class="NLM_article-title">Copper-catalyzed semi-reduction of alkynes</span>. <i>Org. Synth.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>93</i></span>,  <span class="NLM_fpage">385</span>– <span class="NLM_lpage">398</span>, <span class="refDoi"> DOI: 10.1002/0471264229.os093.28</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.1002%2F0471264229.os093.28" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=93&publication_year=2017&pages=385-398&author=N.+Coxauthor=H.+Dangauthor=A.+M.+Whittakerauthor=G.+Lalic&title=Copper-catalyzed+semi-reduction+of+alkynes&doi=10.1002%2F0471264229.os093.28"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1002%2F0471264229.os093.28&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F0471264229.os093.28%26sid%3Dliteratum%253Aachs%26aulast%3DCox%26aufirst%3DN.%26aulast%3DDang%26aufirst%3DH.%26aulast%3DWhittaker%26aufirst%3DA.%2BM.%26aulast%3DLalic%26aufirst%3DG.%26atitle%3DCopper-catalyzed%2520semi-reduction%2520of%2520alkynes%26jtitle%3DOrg.%2520Synth.%26date%3D2017%26volume%3D93%26spage%3D385%26epage%3D398%26doi%3D10.1002%2F0471264229.os093.28" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref67"><div class="reference"><strong class="refLabel"><a href="#ref67" class="rightTabRefNumLink">67</a></strong><div class="NLM_citation" id="rightTab-cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Becerra-Figueroa, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Movilla, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prunet, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miscione, G. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gamba-Sánchez, D.</span></span> <span> </span><span class="NLM_article-title">An intramolecular oxa-Michael reaction on <i>α,β</i>-unsaturated α-amino-δ-hydroxycarboxylic acid esters. synthesis of functionalized 1,3-dioxanes</span>. <i>Org. Biomol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>16</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">1277</span>– <span class="NLM_lpage">1286</span>, <span class="refDoi"> DOI: 10.1039/C7OB03066E</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.1039%2FC7OB03066E" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=29384169" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhs12hs7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2018&pages=1277-1286&issue=8&author=L.+Becerra-Figueroaauthor=S.+Movillaauthor=J.+Prunetauthor=G.+P.+Miscioneauthor=D.+Gamba-S%C3%A1nchez&title=An+intramolecular+oxa-Michael+reaction+on+%CE%B1%2C%CE%B2-unsaturated+%CE%B1-amino-%CE%B4-hydroxycarboxylic+acid+esters.+synthesis+of+functionalized+1%2C3-dioxanes&doi=10.1039%2FC7OB03066E"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">An intramolecular oxa-Michael reaction on α,β-unsaturated α-amino-δ-hydroxycarboxylic acid esters. Synthesis of functionalized 1,3-dioxanes</span></div><div class="casAuthors">Becerra-Figueroa, L.; Movilla, S.; Prunet, J.; Miscione, G. P.; Gamba-Sanchez, D.</div><div class="citationInfo"><span class="NLM_cas:title">Organic & Biomolecular Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1277-1286</span>CODEN:
                <span class="NLM_cas:coden">OBCRAK</span>;
        ISSN:<span class="NLM_cas:issn">1477-0520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A highly diastereoselective intramol. oxa-Michael reaction on α,β-unsatd. α-amino-δ-hydroxycarboxylic acid esters is presented; 1,3-dioxanes functionalized in positions 2, 4 and 6 were obtained in good yields and with excellent selectivities; an exptl. and computational study was carried out to understand the reaction course in terms of yields and selectivities.  This reaction proceeds under mild reaction conditions using highly electrophilic aldehydes and ketones.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqsWQcHSB-YhLVg90H21EOLACvtfcHk0lg8zHhfjx3AKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhs12hs7Y%253D&md5=1cde60dd8de85cbf4d2a7699dc1a9d0d</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1039%2FC7OB03066E&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC7OB03066E%26sid%3Dliteratum%253Aachs%26aulast%3DBecerra-Figueroa%26aufirst%3DL.%26aulast%3DMovilla%26aufirst%3DS.%26aulast%3DPrunet%26aufirst%3DJ.%26aulast%3DMiscione%26aufirst%3DG.%2BP.%26aulast%3DGamba-S%25C3%25A1nchez%26aufirst%3DD.%26atitle%3DAn%2520intramolecular%2520oxa-Michael%2520reaction%2520on%2520%25CE%25B1%252C%25CE%25B2-unsaturated%2520%25CE%25B1-amino-%25CE%25B4-hydroxycarboxylic%2520acid%2520esters.%2520synthesis%2520of%2520functionalized%25201%252C3-dioxanes%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2018%26volume%3D16%26issue%3D8%26spage%3D1277%26epage%3D1286%26doi%3D10.1039%2FC7OB03066E" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref68"><div class="reference"><strong class="refLabel"><a href="#ref68" class="rightTabRefNumLink">68</a></strong><div class="NLM_citation" id="rightTab-cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Blanchard, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Da Silva, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">El Kortbi, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fourrey, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robert-Gero, M.</span></span> <span> </span><span class="NLM_article-title">Zinc-iron couple induced conjugate addition of alkyl halide derived radicals to activated olefins</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>58</i></span> (<span class="NLM_issue">23</span>),  <span class="NLM_fpage">6517</span>– <span class="NLM_lpage">6519</span>, <span class="refDoi"> DOI: 10.1021/jo00075a066</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo00075a066" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADyaK3sXmsFShu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=1993&pages=6517-6519&issue=23&author=P.+Blanchardauthor=A.+D.+Da+Silvaauthor=M.+S.+El+Kortbiauthor=J.+L.+Fourreyauthor=M.+Robert-Gero&title=Zinc-iron+couple+induced+conjugate+addition+of+alkyl+halide+derived+radicals+to+activated+olefins&doi=10.1021%2Fjo00075a066"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">Zinc-iron couple induced conjugate addition of alkyl halide derived radicals to activated olefins</span></div><div class="casAuthors">Blanchard, Pierre; Da Silva, Adilson D.; El Kortbi, Mohammed S.; Fourrey, Jean Louis; Robert-Gero, Malka</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">6517-19</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    </div><div class="casAbstract">Addn. of halo glycosides, e.g. I (R = iodo), with olefins, e.g. CH2:CHCO2Me, in the presence of Zn-Fe gave the corresponding glycosides, e.g. I (R = CH2CH2CO2Me),.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrteqaxoL3037Vg90H21EOLACvtfcHk0lg9BlYD1mNjYQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXmsFShu7o%253D&md5=af75a9e2c79b11dc3e7fe7bca356e831</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1021%2Fjo00075a066&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo00075a066%26sid%3Dliteratum%253Aachs%26aulast%3DBlanchard%26aufirst%3DP.%26aulast%3DDa%2BSilva%26aufirst%3DA.%2BD.%26aulast%3DEl%2BKortbi%26aufirst%3DM.%2BS.%26aulast%3DFourrey%26aufirst%3DJ.%2BL.%26aulast%3DRobert-Gero%26aufirst%3DM.%26atitle%3DZinc-iron%2520couple%2520induced%2520conjugate%2520addition%2520of%2520alkyl%2520halide%2520derived%2520radicals%2520to%2520activated%2520olefins%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D1993%26volume%3D58%26issue%3D23%26spage%3D6517%26epage%3D6519%26doi%3D10.1021%2Fjo00075a066" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref69"><div class="reference"><strong class="refLabel"><a href="#ref69" class="rightTabRefNumLink">69</a></strong><div class="NLM_citation" id="rightTab-cit69"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, D.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, X.-S.</span></span> <span> </span><span class="NLM_article-title">Highly substituted cyclopentane–CMP conjugates as potent sialyltransferase inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span> (<span class="NLM_issue">20</span>),  <span class="NLM_fpage">7972</span>– <span class="NLM_lpage">7990</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01181</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01181" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFKiu7jO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=7972-7990&issue=20&author=W.+Liauthor=Y.+Niuauthor=D.-C.+Xiongauthor=X.+Caoauthor=X.-S.+Ye&title=Highly+substituted+cyclopentane%E2%80%93CMP+conjugates+as+potent+sialyltransferase+inhibitors&doi=10.1021%2Facs.jmedchem.5b01181"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">Highly Substituted Cyclopentane-CMP Conjugates as Potent Sialyltransferase Inhibitors</span></div><div class="casAuthors">Li, Wenming; Niu, Youhong; Xiong, De-Cai; Cao, Xiaoping; Ye, Xin-Shan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">7972-7990</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Sialylconjugates on cell surfaces are involved in many biol. events such as cellular recognition, signal transduction, and immune response.  It has been reported that aberrant sialylation at the nonreducing end of glycoconjugates and overexpression of sialyltransferases (STs) in cells are correlated with the malignance, invasion, and metastasis of tumors.  Therefore, inhibitors of STs would provide valuable leads for the discovery of antitumor drugs.  On the basis of the transition state of the enzyme-catalyzed sialylation reaction, we proposed that the cyclopentane skeleton in its two puckered conformations might mimic the planar structure of the donor (CMP-Neu5Ac) in the transition state.  A series of cyclopentane-contg. compds. were designed and synthesized by coupling different cyclopentane α-hydroxyphosphonates with cytidine phosphoramidite.  Their inhibitory activities against recombinant human ST6Gal-I were assayed, and a potent inhibitor (I) with a Ki of 0.028±0.006 μM was identified.  The results show that the cyclopentanoid-type compds. could become a new type of sialyltransferase inhibitors as biol. probes or drug leads.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDnUM0ETSyjrVg90H21EOLACvtfcHk0lg9BlYD1mNjYQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFKiu7jO&md5=3c05cf23900d96ee3930cd305a1eef24</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01181&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01181%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DW.%26aulast%3DNiu%26aufirst%3DY.%26aulast%3DXiong%26aufirst%3DD.-C.%26aulast%3DCao%26aufirst%3DX.%26aulast%3DYe%26aufirst%3DX.-S.%26atitle%3DHighly%2520substituted%2520cyclopentane%25E2%2580%2593CMP%2520conjugates%2520as%2520potent%2520sialyltransferase%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26issue%3D20%26spage%3D7972%26epage%3D7990%26doi%3D10.1021%2Facs.jmedchem.5b01181" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref70"><div class="reference"><strong class="refLabel"><a href="#ref70" class="rightTabRefNumLink">70</a></strong><div class="note"><p class="first last">The authors note that the N-Cbz cleavage reaction rate increased markedly upon addition of EtOH.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="rightTab-ref71"><div class="reference"><strong class="refLabel"><a href="#ref71" class="rightTabRefNumLink">71</a></strong><div class="NLM_citation" id="rightTab-cit71"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Blanchard, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">El Kortbi, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fourrey, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawrence, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robert-Gero, M.</span></span> <span> </span><span class="NLM_article-title">Relationship between chemical structure and antileishmanial effect of sinefungine analogues</span>. <i>Nucleosides Nucleotides</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>15</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">1121</span>– <span class="NLM_lpage">1135</span>, <span class="refDoi"> DOI: 10.1080/07328319608007381</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.1080%2F07328319608007381" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADyaK28XjvVWktb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=1996&pages=1121-1135&issue=6&author=P.+Blanchardauthor=M.+S.+El+Kortbiauthor=J.+L.+Fourreyauthor=F.+Lawrenceauthor=M.+Robert-Gero&title=Relationship+between+chemical+structure+and+antileishmanial+effect+of+sinefungine+analogues&doi=10.1080%2F07328319608007381"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">Relationship between chemical structure and antileishmanial effect of sinefungin analogs</span></div><div class="casAuthors">Blanchard, P.; El Kortbi, M. S.; Fourrey, J.-L.; Lawrence, F.; Robert-Gero, M.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleosides & Nucleotides</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1121-1135</span>CODEN:
                <span class="NLM_cas:coden">NUNUD5</span>;
        ISSN:<span class="NLM_cas:issn">0732-8311</span>.
    
            (<span class="NLM_cas:orgname">Dekker</span>)
        </div><div class="casAbstract">The synthesis of various analogs of sinefungin, e.g. I (R = H, NH2), has been developed by means of an original approach which uses radical chem.  The study of their biol. activities revealed that for the antileishmanial effect of sinefungin, the presence of the amino group at C-6' in the (S)-configuration and the presence of the carboxyl group at C-9' are necessary.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_Ho5g0xhJybVg90H21EOLACvtfcHk0lg9BlYD1mNjYQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XjvVWktb4%253D&md5=fb4457d5a3a6e706c1f9fff041ad7580</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1080%2F07328319608007381&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F07328319608007381%26sid%3Dliteratum%253Aachs%26aulast%3DBlanchard%26aufirst%3DP.%26aulast%3DEl%2BKortbi%26aufirst%3DM.%2BS.%26aulast%3DFourrey%26aufirst%3DJ.%2BL.%26aulast%3DLawrence%26aufirst%3DF.%26aulast%3DRobert-Gero%26aufirst%3DM.%26atitle%3DRelationship%2520between%2520chemical%2520structure%2520and%2520antileishmanial%2520effect%2520of%2520sinefungine%2520analogues%26jtitle%3DNucleosides%2520Nucleotides%26date%3D1996%26volume%3D15%26issue%3D6%26spage%3D1121%26epage%3D1135%26doi%3D10.1080%2F07328319608007381" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref72"><div class="reference"><strong class="refLabel"><a href="#ref72" class="rightTabRefNumLink">72</a></strong><div class="NLM_citation" id="rightTab-cit72"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sarabia, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martín-Ortiz, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">López-Herrera, F. J.</span></span> <span> </span><span class="NLM_article-title">A convergent synthetic approach to the nucleoside-type liposidomycin antibiotics</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>5</i></span> (<span class="NLM_issue">21</span>),  <span class="NLM_fpage">3927</span>– <span class="NLM_lpage">3930</span>, <span class="refDoi"> DOI: 10.1021/ol0355074</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ol0355074" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BD3sXnt1Kksbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2003&pages=3927-3930&issue=21&author=F.+Sarabiaauthor=L.+Mart%C3%ADn-Ortizauthor=F.+J.+L%C3%B3pez-Herrera&title=A+convergent+synthetic+approach+to+the+nucleoside-type+liposidomycin+antibiotics&doi=10.1021%2Fol0355074"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">A Convergent Synthetic Approach to the Nucleoside-Type Liposidomycin Antibiotics</span></div><div class="casAuthors">Sarabia, Francisco; Martin-Ortiz, Laura; Lopez-Herrera, F. Jorge</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">3927-3930</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7060</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A synthetic approach toward the liposidomycins, a family of complex nucleoside-type antibiotics, is reported based on the synthesis of epoxy-amides derived from the reaction of sulfur ylides with the uridyl aldehyde deriv. I.  To this end, the epoxy-amide deriv. of indoline II was stereoselectively prepd. and, after treatment with DDQ, transformed into the corresponding N-indole epoxy-amide.  The N-indole epoxy-amide provides ready access to a variety of structures related to the diazepanone core present in the liposidomycins by reaction with a variety of amines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGruWQHXSPDo97Vg90H21EOLACvtfcHk0lgeKYgAz1FOug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXnt1Kksbs%253D&md5=25357efc8909b301084d57a08d2b6455</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1021%2Fol0355074&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fol0355074%26sid%3Dliteratum%253Aachs%26aulast%3DSarabia%26aufirst%3DF.%26aulast%3DMart%25C3%25ADn-Ortiz%26aufirst%3DL.%26aulast%3DL%25C3%25B3pez-Herrera%26aufirst%3DF.%2BJ.%26atitle%3DA%2520convergent%2520synthetic%2520approach%2520to%2520the%2520nucleoside-type%2520liposidomycin%2520antibiotics%26jtitle%3DOrg.%2520Lett.%26date%3D2003%26volume%3D5%26issue%3D21%26spage%3D3927%26epage%3D3930%26doi%3D10.1021%2Fol0355074" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref73"><div class="reference"><strong class="refLabel"><a href="#ref73" class="rightTabRefNumLink">73</a></strong><div class="NLM_citation" id="rightTab-cit73"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Petakamsetty, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ansari, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramapanicker, R.</span></span> <span> </span><span class="NLM_article-title">Diastereoselective synthesis of furanose and pyranose substituted glycine and alanine derivatives via proline-catalyzed asymmetric α-amination of aldehydes</span>. <i>Carbohydr. Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>435</i></span>,  <span class="NLM_fpage">37</span>– <span class="NLM_lpage">49</span>, <span class="refDoi"> DOI: 10.1016/j.carres.2016.09.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.1016%2Fj.carres.2016.09.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=27693912" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhs1Srs7zI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=435&publication_year=2016&pages=37-49&author=R.+Petakamsettyauthor=A.+Ansariauthor=R.+Ramapanicker&title=Diastereoselective+synthesis+of+furanose+and+pyranose+substituted+glycine+and+alanine+derivatives+via+proline-catalyzed+asymmetric+%CE%B1-amination+of+aldehydes&doi=10.1016%2Fj.carres.2016.09.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">Diastereoselective synthesis of furanose and pyranose substituted glycine and alanine derivatives via proline-catalyzed asymmetric α-amination of aldehydes</span></div><div class="casAuthors">Petakamsetty, Ramu; Ansari, Anas; Ramapanicker, Ramesh</div><div class="citationInfo"><span class="NLM_cas:title">Carbohydrate Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">435</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">37-49</span>CODEN:
                <span class="NLM_cas:coden">CRBRAT</span>;
        ISSN:<span class="NLM_cas:issn">0008-6215</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A concise organocatalytic route toward the synthesis of furanose and pyranose substituted glycine and alanine derivs. is reported.  These compds. are core structural units of some of the naturally available antibiotics and antifungal agents.  Proline-catalyzed asym. α-amination of aldehydes derived from sugars is used as the key reaction to synthesize twelve sugar amino acid derivs.  The asym. transformations proceeded in good yields and with good to excellent diastereoselectivity.  The application of the synthesized amino acids is demonstrated by synthesizing a tripeptide contg. one of them.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrq4HLkO9P14LVg90H21EOLACvtfcHk0lgeKYgAz1FOug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhs1Srs7zI&md5=8752fc493afc01b6e01924db66f7b9da</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1016%2Fj.carres.2016.09.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.carres.2016.09.011%26sid%3Dliteratum%253Aachs%26aulast%3DPetakamsetty%26aufirst%3DR.%26aulast%3DAnsari%26aufirst%3DA.%26aulast%3DRamapanicker%26aufirst%3DR.%26atitle%3DDiastereoselective%2520synthesis%2520of%2520furanose%2520and%2520pyranose%2520substituted%2520glycine%2520and%2520alanine%2520derivatives%2520via%2520proline-catalyzed%2520asymmetric%2520%25CE%25B1-amination%2520of%2520aldehydes%26jtitle%3DCarbohydr.%2520Res.%26date%3D2016%26volume%3D435%26spage%3D37%26epage%3D49%26doi%3D10.1016%2Fj.carres.2016.09.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref74"><div class="reference"><strong class="refLabel"><a href="#ref74" class="rightTabRefNumLink">74</a></strong><div class="NLM_citation" id="rightTab-cit74"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Herrera, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rondón, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suárez, E.</span></span> <span> </span><span class="NLM_article-title">Stereocontrolled photocyclization of 1,2-diketones: application of a 1,3-acetyl group transfer methodology to carbohydrates</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>73</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">3384</span>– <span class="NLM_lpage">3391</span>, <span class="refDoi"> DOI: 10.1021/jo702663w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo702663w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjvFejurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2008&pages=3384-3391&issue=9&author=A.+J.+Herreraauthor=M.+Rond%C3%B3nauthor=E.+Su%C3%A1rez&title=Stereocontrolled+photocyclization+of+1%2C2-diketones%3A+application+of+a+1%2C3-acetyl+group+transfer+methodology+to+carbohydrates&doi=10.1021%2Fjo702663w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">Stereocontrolled Photocyclization of 1,2-Diketones: Application of a 1,3-Acetyl Group Transfer Methodology to Carbohydrates</span></div><div class="casAuthors">Herrera, Antonio J.; Rondon, Maria; Suarez, Ernesto</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3384-3391</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Photolysis of 1-glycosyl-2,3-butanedione derivs. using visible light is a mild and selective procedure for the synthesis of chiral 1-hydroxy-1-methyl-5-oxaspiro[3.5]nonan-2-one carbohydrate derivs., e.g., I.  The results strongly suggest that stereocontrol of the cyclization is dependent on conformational and stereoelectronic factors.  Further oxidative opening of the 1-hydroxy-1-methyl-2-cyclobutanone moiety affords new C-ketoside derivs. either in C- and O-glycoside series, e.g., II.  This tandem two-step process could be considered to be a stereocontrolled 1,3-transference of an acetyl group, and it can be applied either to pyranose and furanose models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqUWYTOKhEDP7Vg90H21EOLACvtfcHk0lgeKYgAz1FOug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjvFejurw%253D&md5=b096ed26cb720cd516e4f607fc9a1c25</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1021%2Fjo702663w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo702663w%26sid%3Dliteratum%253Aachs%26aulast%3DHerrera%26aufirst%3DA.%2BJ.%26aulast%3DRond%25C3%25B3n%26aufirst%3DM.%26aulast%3DSu%25C3%25A1rez%26aufirst%3DE.%26atitle%3DStereocontrolled%2520photocyclization%2520of%25201%252C2-diketones%253A%2520application%2520of%2520a%25201%252C3-acetyl%2520group%2520transfer%2520methodology%2520to%2520carbohydrates%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2008%26volume%3D73%26issue%3D9%26spage%3D3384%26epage%3D3391%26doi%3D10.1021%2Fjo702663w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref75"><div class="reference"><strong class="refLabel"><a href="#ref75" class="rightTabRefNumLink">75</a></strong><div class="NLM_citation" id="rightTab-cit75"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Charette, A.
B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Juteau, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lebel, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Molinaro, C.</span></span> <span> </span><span class="NLM_article-title">Enantioselective cyclopropanation of allylic alcohols with dioxaborolane ligands:  scope and synthetic applications</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>120</i></span> (<span class="NLM_issue">46</span>),  <span class="NLM_fpage">11943</span>– <span class="NLM_lpage">11952</span>, <span class="refDoi"> DOI: 10.1021/ja982055v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja982055v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADyaK1cXntFegs74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=120&publication_year=1998&pages=11943-11952&issue=46&author=A.%0AB.+Charetteauthor=H.+Juteauauthor=H.+Lebelauthor=C.+Molinaro&title=Enantioselective+cyclopropanation+of+allylic+alcohols+with+dioxaborolane+ligands%3A%E2%80%89+scope+and+synthetic+applications&doi=10.1021%2Fja982055v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">Enantioselective Cyclopropanation of Allylic Alcohols with Dioxaborolane Ligands: Scope and Synthetic Applications</span></div><div class="casAuthors">Charette, Andre B.; Juteau, Helene; Lebel, Helene; Molinaro, Carmela</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">120</span>
        (<span class="NLM_cas:issue">46</span>),
    <span class="NLM_cas:pages">11943-11952</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A very effective chiral controller was found for the conversion of allylic alcs. into the corresponding enantiomerically enriched cyclopropanes using bis(iodomethyl)zinc.  A variety of chiral, nonracemic cyclopropylmethanols could be obtained according to this method.  This methodol. was extended with success to the cyclopropanation of unconjugated and conjugated polyenes and homoallylic alcs.  The cyclopropanation of allylic carbamates was investigated with this system, but it was found that enantioenriched cyclopropylmethylamines are best prepd. from enantioenriched cyclopropylmethanols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxyjTCaz_r3LVg90H21EOLACvtfcHk0lgeKYgAz1FOug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXntFegs74%253D&md5=cdbfeb346b742e9ecb7023e250fd73b0</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1021%2Fja982055v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja982055v%26sid%3Dliteratum%253Aachs%26aulast%3DCharette%26aufirst%3DA.%2BB.%26aulast%3DJuteau%26aufirst%3DH.%26aulast%3DLebel%26aufirst%3DH.%26aulast%3DMolinaro%26aufirst%3DC.%26atitle%3DEnantioselective%2520cyclopropanation%2520of%2520allylic%2520alcohols%2520with%2520dioxaborolane%2520ligands%253A%25E2%2580%2589%2520scope%2520and%2520synthetic%2520applications%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1998%26volume%3D120%26issue%3D46%26spage%3D11943%26epage%3D11952%26doi%3D10.1021%2Fja982055v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref76"><div class="reference"><strong class="refLabel"><a href="#ref76" class="rightTabRefNumLink">76</a></strong><div class="NLM_citation" id="rightTab-cit76"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hagiya, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muramoto, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Misaki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sugimura, T.</span></span> <span> </span><span class="NLM_article-title">DMEAD: a new dialkyl azodicarboxylate for the Mitsunobu reaction</span>. <i>Tetrahedron</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>65</i></span> (<span class="NLM_issue">31</span>),  <span class="NLM_fpage">6109</span>– <span class="NLM_lpage">6114</span>, <span class="refDoi"> DOI: 10.1016/j.tet.2009.05.048</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.1016%2Fj.tet.2009.05.048" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BD1MXotFCksbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2009&pages=6109-6114&issue=31&author=K.+Hagiyaauthor=N.+Muramotoauthor=T.+Misakiauthor=T.+Sugimura&title=DMEAD%3A+a+new+dialkyl+azodicarboxylate+for+the+Mitsunobu+reaction&doi=10.1016%2Fj.tet.2009.05.048"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit76R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">DMEAD: a new dialkyl azodicarboxylate for the Mitsunobu reaction</span></div><div class="casAuthors">Hagiya, Kazutake; Muramoto, Natsuko; Misaki, Tomonori; Sugimura, Takashi</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">65</span>
        (<span class="NLM_cas:issue">31</span>),
    <span class="NLM_cas:pages">6109-6114</span>CODEN:
                <span class="NLM_cas:coden">TETRAB</span>;
        ISSN:<span class="NLM_cas:issn">0040-4020</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Di-2-methoxyethyl azodicarboxylate (DMEAD) is prepd. in 65% yield in two steps as a cryst. solid.  Use of DMEAD in the Mitsunobu reaction of a variety of alcs. with pronucleophiles results in good yields of the products under sufficient stereospecificity of inversion, as conventional diisopropyl azodicarboxylate (DIAD) does.  Isolation of the product is, however, much easier with DMEAD than that with DIAD, because the hydrazine produced from DMEAD is highly hydrophilic and is completely separable by a simple extn. into neutral water.  Purifn. of the org. layer, after sepn. of the other byproduct, triphenylphosphine oxide, by filtration, easily provides high purity of the product in a good yield.  Concn. of the water layer yields the hydrazine, which can be reused for the prepn. of DMEAD.  One-step removal of the two byproducts by the aq. extn. was also possible when trimethylphosphine and DMEAD were employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8942g2yxAwbVg90H21EOLACvtfcHk0lhK9DpTjvvY3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXotFCksbc%253D&md5=1465a85e39f7dd03b9c7f74d9f599c20</span></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1016%2Fj.tet.2009.05.048&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tet.2009.05.048%26sid%3Dliteratum%253Aachs%26aulast%3DHagiya%26aufirst%3DK.%26aulast%3DMuramoto%26aufirst%3DN.%26aulast%3DMisaki%26aufirst%3DT.%26aulast%3DSugimura%26aufirst%3DT.%26atitle%3DDMEAD%253A%2520a%2520new%2520dialkyl%2520azodicarboxylate%2520for%2520the%2520Mitsunobu%2520reaction%26jtitle%3DTetrahedron%26date%3D2009%26volume%3D65%26issue%3D31%26spage%3D6109%26epage%3D6114%26doi%3D10.1016%2Fj.tet.2009.05.048" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref77"><div class="reference"><strong class="refLabel"><a href="#ref77" class="rightTabRefNumLink">77</a></strong><div class="NLM_citation" id="rightTab-cit77"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bergström, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suresh, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naidu, V. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Unelius, C. R.</span></span> <span> </span><span class="NLM_article-title">N-iodosuccinimide (NIS) in direct aromatic iodination</span>. <i>Eur. J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>2017</i></span> (<span class="NLM_issue">22</span>),  <span class="NLM_fpage">3234</span>– <span class="NLM_lpage">3239</span>, <span class="refDoi"> DOI: 10.1002/ejoc.201700173</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.1002%2Fejoc.201700173" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2017&publication_year=2017&pages=3234-3239&issue=22&author=M.+Bergstr%C3%B6mauthor=G.+Sureshauthor=V.+R.+Naiduauthor=C.+R.+Unelius&title=N-iodosuccinimide+%28NIS%29+in+direct+aromatic+iodination&doi=10.1002%2Fejoc.201700173"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=10.1002%2Fejoc.201700173&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fejoc.201700173%26sid%3Dliteratum%253Aachs%26aulast%3DBergstr%25C3%25B6m%26aufirst%3DM.%26aulast%3DSuresh%26aufirst%3DG.%26aulast%3DNaidu%26aufirst%3DV.%2BR.%26aulast%3DUnelius%26aufirst%3DC.%2BR.%26atitle%3DN-iodosuccinimide%2520%2528NIS%2529%2520in%2520direct%2520aromatic%2520iodination%26jtitle%3DEur.%2520J.%2520Org.%2520Chem.%26date%3D2017%26volume%3D2017%26issue%3D22%26spage%3D3234%26epage%3D3239%26doi%3D10.1002%2Fejoc.201700173" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref78"><div class="reference"><strong class="refLabel"><a href="#ref78" class="rightTabRefNumLink">78</a></strong><div class="NLM_citation" id="rightTab-cit78"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nammalwar, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunce, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berlin, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bourne, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bourne, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrow, E. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrow, W. W.</span></span> <span> </span><span class="NLM_article-title">Approaches to iodinated derivatives of vanillin and isovanillin</span>. <i>Org. Prep. Proced. Int.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>44</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">146</span>– <span class="NLM_lpage">152</span>, <span class="refDoi"> DOI: 10.1080/00304948.2012.643700</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.1080%2F00304948.2012.643700" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=23805008" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BC38XltVKrsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2012&pages=146-152&issue=2&author=B.+Nammalwarauthor=R.+A.+Bunceauthor=K.+D.+Berlinauthor=C.+R.+Bourneauthor=P.+C.+Bourneauthor=E.+W.+Barrowauthor=W.+W.+Barrow&title=Approaches+to+iodinated+derivatives+of+vanillin+and+isovanillin&doi=10.1080%2F00304948.2012.643700"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit78R"><div class="casContent"><span class="casTitleNuber">78</span><div class="casTitle"><span class="NLM_cas:atitle">Approaches to Iodinated Derivatives of Vanillin and Isovanillin</span></div><div class="casAuthors">Nammalwar, Baskar; Bunce, Richard A.; Berlin, K. Darrell; Bourne, Christina R.; Bourne, Philip C.; Barrow, Esther W.; Barrow, William W.</div><div class="citationInfo"><span class="NLM_cas:title">Organic Preparations and Procedures International</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">146-152</span>CODEN:
                <span class="NLM_cas:coden">OPPIAK</span>;
        ISSN:<span class="NLM_cas:issn">0030-4948</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">A straightforward approach was developed by which derivs. of 5-iodovanillin and 5-iodoisovanillin can be prepd.  The use of aluminum chloride in pyridine proved most convenient and cost effective for the cleavage of the Me ether in 5-iodovanillin to generate 3,4-dihydroxy-5-iodobenzaldehyde (I, R = H).  Regiospecific O-alkylation of I (R = H) at the C-4 hydroxy group with Me iodide in DBU/DMF then provided an intermediate, which could be further alkylated at the C-5 hydroxy group using the same conditions to afford the ether series I (R = Me, Et, etc).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGric2cEP9xEVbVg90H21EOLACvtfcHk0lhK9DpTjvvY3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XltVKrsr0%253D&md5=dbfbe86baa645d5641d69413af5b1330</span></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=10.1080%2F00304948.2012.643700&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F00304948.2012.643700%26sid%3Dliteratum%253Aachs%26aulast%3DNammalwar%26aufirst%3DB.%26aulast%3DBunce%26aufirst%3DR.%2BA.%26aulast%3DBerlin%26aufirst%3DK.%2BD.%26aulast%3DBourne%26aufirst%3DC.%2BR.%26aulast%3DBourne%26aufirst%3DP.%2BC.%26aulast%3DBarrow%26aufirst%3DE.%2BW.%26aulast%3DBarrow%26aufirst%3DW.%2BW.%26atitle%3DApproaches%2520to%2520iodinated%2520derivatives%2520of%2520vanillin%2520and%2520isovanillin%26jtitle%3DOrg.%2520Prep.%2520Proced.%2520Int.%26date%3D2012%26volume%3D44%26issue%3D2%26spage%3D146%26epage%3D152%26doi%3D10.1080%2F00304948.2012.643700" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref79"><div class="reference"><strong class="refLabel"><a href="#ref79" class="rightTabRefNumLink">79</a></strong><div class="NLM_citation" id="rightTab-cit79"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tae, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Min, S.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rhim, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choo, H.</span></span> <span> </span><span class="NLM_article-title">5-HT7 receptor modulators: amino groups attached to biphenyl scaffold determine functional activity</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>123</i></span>,  <span class="NLM_fpage">180</span>– <span class="NLM_lpage">190</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2016.07.029</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.1016%2Fj.ejmech.2016.07.029" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=27475109" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1OntrvP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=123&publication_year=2016&pages=180-190&author=Y.+Kimauthor=H.+Parkauthor=J.+Leeauthor=J.+Taeauthor=H.+J.+Kimauthor=S.-J.+Minauthor=H.+Rhimauthor=H.+Choo&title=5-HT7+receptor+modulators%3A+amino+groups+attached+to+biphenyl+scaffold+determine+functional+activity&doi=10.1016%2Fj.ejmech.2016.07.029"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit79R"><div class="casContent"><span class="casTitleNuber">79</span><div class="casTitle"><span class="NLM_cas:atitle">5-HT7 receptor modulators: Amino groups attached to biphenyl scaffold determine functional activity</span></div><div class="casAuthors">Kim, Youngjae; Park, Hyeri; Lee, Jeongeun; Tae, Jinsung; Kim, Hak Joong; Min, Sun-Joon; Rhim, Hyewhon; Choo, Hyunah</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">123</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">180-190</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">A novel scaffold for 5-HT7R modulators, a series of biphenyl-3-yl-methanamine derivs. with various amino groups I (R1 = H, Cl, Me, OMe; NR2R3 = 4-(2-methoxyphenyl)piperazine, 4-methyl-piperidine, NMe2, NMe), II (NR2R3 = 4-(2-methoxyphenyl)piperazine, 4-methyl-piperidine, NMe2, NMe) were designed and prepd.  Evaluation of functional activities as well as binding affinities of the title compds. identified partial agonists (EC50 = 0.55-3.2 μM) and full antagonists (IC50 = 5.57-23.1 μM) depending on the amino substituents.  Mol. docking study suggested that the ligand-based switch in functional activity from agonist to antagonist results from the size of the amino groups and thereby different binding modes to 5-HT7R.  In particular, interaction of the ligand with Arg367 of 5-HT7R is shown to differentiate agonists and antagonists.  In the pharmacophore model study, two distinct pharmacophore models can tell whether a ligand is an agonist or an antagonist.  Taken together, this study provides valuable information for designing novel compds. with selective agonistic or antagonistic properties against 5-HT7R.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_XDczBDq_vrVg90H21EOLACvtfcHk0lhZxWFVNAszpQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1OntrvP&md5=1655d984a53c5fd9292330f24e7510d8</span></div><a href="/servlet/linkout?suffix=cit79&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2016.07.029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2016.07.029%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DY.%26aulast%3DPark%26aufirst%3DH.%26aulast%3DLee%26aufirst%3DJ.%26aulast%3DTae%26aufirst%3DJ.%26aulast%3DKim%26aufirst%3DH.%2BJ.%26aulast%3DMin%26aufirst%3DS.-J.%26aulast%3DRhim%26aufirst%3DH.%26aulast%3DChoo%26aufirst%3DH.%26atitle%3D5-HT7%2520receptor%2520modulators%253A%2520amino%2520groups%2520attached%2520to%2520biphenyl%2520scaffold%2520determine%2520functional%2520activity%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2016%26volume%3D123%26spage%3D180%26epage%3D190%26doi%3D10.1016%2Fj.ejmech.2016.07.029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref80"><div class="reference"><strong class="refLabel"><a href="#ref80" class="rightTabRefNumLink">80</a></strong><div class="NLM_citation" id="rightTab-cit80"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Martinez
Molina, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jafari, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ignatushchenko, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larsson, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sreekumar, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nordlund, P.</span></span> <span> </span><span class="NLM_article-title">Monitoring drug target engagement in cells and tissues using the cellular thermal shift assay</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>341</i></span> (<span class="NLM_issue">6141</span>),  <span class="NLM_fpage">84</span>– <span class="NLM_lpage">87</span>, <span class="refDoi"> DOI: 10.1126/science.1233606</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.1126%2Fscience.1233606" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=23828940" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A280%3ADC%252BC3sjotlOqtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=341&publication_year=2013&pages=84-87&issue=6141&author=D.+Martinez%0AMolinaauthor=R.+Jafariauthor=M.+Ignatushchenkoauthor=T.+Sekiauthor=E.+A.+Larssonauthor=C.+Danauthor=L.+Sreekumarauthor=Y.+Caoauthor=P.+Nordlund&title=Monitoring+drug+target+engagement+in+cells+and+tissues+using+the+cellular+thermal+shift+assay&doi=10.1126%2Fscience.1233606"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit80R"><div class="casContent"><span class="casTitleNuber">80</span><div class="casTitle"><span class="NLM_cas:atitle">Monitoring drug target engagement in cells and tissues using the cellular thermal shift assay</span></div><div class="casAuthors">Martinez Molina Daniel; Jafari Rozbeh; Ignatushchenko Marina; Seki Takahiro; Larsson E Andreas; Dan Chen; Sreekumar Lekshmy; Cao Yihai; Nordlund Par</div><div class="citationInfo"><span class="NLM_cas:title">Science (New York, N.Y.)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">341</span>
        (<span class="NLM_cas:issue">6141</span>),
    <span class="NLM_cas:pages">84-7</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The efficacy of therapeutics is dependent on a drug binding to its cognate target.  Optimization of target engagement by drugs in cells is often challenging, because drug binding cannot be monitored inside cells.  We have developed a method for evaluating drug binding to target proteins in cells and tissue samples.  This cellular thermal shift assay (CETSA) is based on the biophysical principle of ligand-induced thermal stabilization of target proteins.  Using this assay, we validated drug binding for a set of important clinical targets and monitored processes of drug transport and activation, off-target effects and drug resistance in cancer cell lines, as well as drug distribution in tissues.  CETSA is likely to become a valuable tool for the validation and optimization of drug target engagement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRRn5K_mF1rcrHHuJhbyicZfW6udTcc2eZ6xthpkoT1v7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3sjotlOqtA%253D%253D&md5=a73bb49adb0d4f445223ea6f74fe3c10</span></div><a href="/servlet/linkout?suffix=cit80&amp;dbid=16384&amp;doi=10.1126%2Fscience.1233606&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1233606%26sid%3Dliteratum%253Aachs%26aulast%3DMartinez%2BMolina%26aufirst%3DD.%26aulast%3DJafari%26aufirst%3DR.%26aulast%3DIgnatushchenko%26aufirst%3DM.%26aulast%3DSeki%26aufirst%3DT.%26aulast%3DLarsson%26aufirst%3DE.%2BA.%26aulast%3DDan%26aufirst%3DC.%26aulast%3DSreekumar%26aufirst%3DL.%26aulast%3DCao%26aufirst%3DY.%26aulast%3DNordlund%26aufirst%3DP.%26atitle%3DMonitoring%2520drug%2520target%2520engagement%2520in%2520cells%2520and%2520tissues%2520using%2520the%2520cellular%2520thermal%2520shift%2520assay%26jtitle%3DScience%26date%3D2013%26volume%3D341%26issue%3D6141%26spage%3D84%26epage%3D87%26doi%3D10.1126%2Fscience.1233606" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref81"><div class="reference"><strong class="refLabel"><a href="#ref81" class="rightTabRefNumLink">81</a></strong><div class="NLM_citation" id="rightTab-cit81"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jafari, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Almqvist, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Axelsson, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ignatushchenko, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lundbäck, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nordlund, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinez Molina, D.</span></span> <span> </span><span class="NLM_article-title">The cellular thermal shift assay for evaluating drug target interactions in cells</span>. <i>Nat. Protoc.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>9</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">2100</span>– <span class="NLM_lpage">2122</span>, <span class="refDoi"> DOI: 10.1038/nprot.2014.138</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.1038%2Fnprot.2014.138" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=25101824" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht12msrfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=2100-2122&issue=9&author=R.+Jafariauthor=H.+Almqvistauthor=H.+Axelssonauthor=M.+Ignatushchenkoauthor=T.+Lundb%C3%A4ckauthor=P.+Nordlundauthor=D.+Martinez+Molina&title=The+cellular+thermal+shift+assay+for+evaluating+drug+target+interactions+in+cells&doi=10.1038%2Fnprot.2014.138"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit81R"><div class="casContent"><span class="casTitleNuber">81</span><div class="casTitle"><span class="NLM_cas:atitle">The cellular thermal shift assay for evaluating drug target interactions in cells</span></div><div class="casAuthors">Jafari, Rozbeh; Almqvist, Helena; Axelsson, Hanna; Ignatushchenko, Marina; Lundbaeck, Thomas; Nordlund, Paer; Molina, Daniel Martinez</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2100-2122</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Thermal shift assays are used to study thermal stabilization of proteins upon ligand binding.  Such assays have been used extensively on purified proteins in the drug discovery industry and in academia to detect interactions.  Recently, we published a proof-of-principle study describing the implementation of thermal shift assays in a cellular format, which we call the cellular thermal shift assay (CETSA).  The method allows studies of target engagement of drug candidates in a cellular context, herein exemplified with exptl. data on the human kinases p38α and ERK1/2.  The assay involves treatment of cells with a compd. of interest, heating to denature and ppt. proteins, cell lysis, and the sepn. of cell debris and aggregates from the sol. protein fraction.  Whereas unbound proteins denature and ppt. at elevated temps., ligand-bound proteins remain in soln.  We describe two procedures for detecting the stabilized protein in the sol. fraction of the samples.  One approach involves sample workup and detection using quant. western blotting, whereas the second is performed directly in soln. and relies on the induced proximity of two target-directed antibodies upon binding to sol. protein.  The latter protocol has been optimized to allow an increased throughput, as potential applications require large nos. of samples.  Both approaches can be completed in a day.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzarqomAK6eLVg90H21EOLACvtfcHk0lhZxWFVNAszpQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht12msrfJ&md5=e7852e1a306bca1698a8e8463ae566b1</span></div><a href="/servlet/linkout?suffix=cit81&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2014.138&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2014.138%26sid%3Dliteratum%253Aachs%26aulast%3DJafari%26aufirst%3DR.%26aulast%3DAlmqvist%26aufirst%3DH.%26aulast%3DAxelsson%26aufirst%3DH.%26aulast%3DIgnatushchenko%26aufirst%3DM.%26aulast%3DLundb%25C3%25A4ck%26aufirst%3DT.%26aulast%3DNordlund%26aufirst%3DP.%26aulast%3DMartinez%2BMolina%26aufirst%3DD.%26atitle%3DThe%2520cellular%2520thermal%2520shift%2520assay%2520for%2520evaluating%2520drug%2520target%2520interactions%2520in%2520cells%26jtitle%3DNat.%2520Protoc.%26date%3D2014%26volume%3D9%26issue%3D9%26spage%3D2100%26epage%3D2122%26doi%3D10.1038%2Fnprot.2014.138" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref82"><div class="reference"><strong class="refLabel"><a href="#ref82" class="rightTabRefNumLink">82</a></strong><div class="NLM_citation" id="rightTab-cit82"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Morin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eisenbraun, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Key, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanschagrin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Timony, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ottaviano, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sliz, P.</span></span> <span> </span><span class="NLM_article-title">Cutting edge: Collaboration gets the most out of software</span>. <i>eLife</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>2</i></span>, <span class="NLM_elocation-id">e01456</span> <span class="refDoi"> DOI: 10.7554/eLife.01456</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.7554%2FeLife.01456" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=24040512" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2013&author=A.+Morinauthor=B.+Eisenbraunauthor=J.+Keyauthor=C.+Sanschagrinauthor=M.+A.+Timonyauthor=M.+Ottavianoauthor=P.+Sliz&title=Cutting+edge%3A+Collaboration+gets+the+most+out+of+software&doi=10.7554%2FeLife.01456"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit82&amp;dbid=16384&amp;doi=10.7554%2FeLife.01456&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7554%252FeLife.01456%26sid%3Dliteratum%253Aachs%26aulast%3DMorin%26aufirst%3DA.%26aulast%3DEisenbraun%26aufirst%3DB.%26aulast%3DKey%26aufirst%3DJ.%26aulast%3DSanschagrin%26aufirst%3DC.%26aulast%3DTimony%26aufirst%3DM.%2BA.%26aulast%3DOttaviano%26aufirst%3DM.%26aulast%3DSliz%26aufirst%3DP.%26atitle%3DCutting%2520edge%253A%2520Collaboration%2520gets%2520the%2520most%2520out%2520of%2520software%26jtitle%3DeLife%26date%3D2013%26volume%3D2%26doi%3D10.7554%2FeLife.01456" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref83"><div class="reference"><strong class="refLabel"><a href="#ref83" class="rightTabRefNumLink">83</a></strong><div class="NLM_citation" id="rightTab-ref83_1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">May, E. M.</span>; <span class="NLM_string-name">Policarpo, R. L.</span>; <span class="NLM_string-name">Gaudet, R.</span></span> <span> </span><span class="NLM_article-title">X-Ray Diffraction data for: Co-crystal structure of human Nicotinamide N-Methyltransferase (NNMT) in complex with High-Affinity Alkynyl Bisubstrate Inhibitor NS1</span>. PDB Code 6ORR,  <i>SBGrid Data Bank</i>, V1, <span class="refDoi"> DOI: 10.15785/SBGRID/731</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref83/ref83_1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;key=10.15785%2FSBGRID%2F731" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=May%2C+E.+M.%3B+Policarpo%2C+R.+L.%3B+Gaudet%2C+R.+X-Ray+Diffraction+data+for%3A+Co-crystal+structure+of+human+Nicotinamide+N-Methyltransferase+%28NNMT%29+in+complex+with+High-Affinity+Alkynyl+Bisubstrate+Inhibitor+NS1.+PDB+Code+6ORR%2C+SBGrid+Data+Bank%2C+V1%2C+10.15785%2FSBGRID%2F731."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=ref83_1&amp;dbid=16384&amp;doi=10.15785%2FSBGRID%2F731&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26id%3Ddoi%3A10.15785%252FSBGRID%252F731%26sid%3Dliteratum%253Aachs%26aulast%3DMay%26aufirst%3DE.%2BM.%26atitle%3DX-Ray%2520Diffraction%2520data%2520for%253A%2520Co-crystal%2520structure%2520of%2520human%2520Nicotinamide%2520N-Methyltransferase%2520%2528NNMT%2529%2520in%2520complex%2520with%2520High-Affinity%2520Alkynyl%2520Bisubstrate%2520Inhibitor%2520NS1%26jtitle%3DSBGrid%2520Data%2520Bank%26doi%3D10.15785%2FSBGRID%2F731" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ROD" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ROD','PDB','3ROD'); return false;">PDB: 3ROD</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ORR" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ORR','PDB','6ORR'); return false;">PDB: 6ORR</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6B1A" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6B1A','PDB','6B1A'); return false;">PDB: 6B1A</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i152"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01238">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_56678"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.9b01238">10.1021/acs.jmedchem.9b01238</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Supplementary figures and tables, synthetic schemes, and experimental protocols not listed in the main manuscript, including: molecular docking, protein expression and purification, biochemical assays, bioinformatic analyses, and protein crystallography (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01238/suppl_file/jm9b01238_si_001.pdf">PDF</a>)</p></li><li><p class="inline">NMR spectra (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01238/suppl_file/jm9b01238_si_002.pdf">PDF</a>)</p></li><li><p class="inline">Enzyme kinetics (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01238/suppl_file/jm9b01238_si_003.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings for compounds <b>10</b>–<b>41</b>  (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01238/suppl_file/jm9b01238_si_004.csv">CSV</a>)</p></li><li><p class="inline">NS1 <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ROD">3ROD</a> docking (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01238/suppl_file/jm9b01238_si_005.pdb">PDB</a>)</p></li><li><p class="inline">Small-molecule X-ray crystallography data for NS1 (<b>10</b>) (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01238/suppl_file/jm9b01238_si_006.cif">CIF</a>)</p></li><li><p class="inline">Small-molecule X-ray crystallography data for <b>S30</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01238/suppl_file/jm9b01238_si_007.cif">CIF</a>)</p></li><li><p class="inline">Small-molecule X-ray crystallography data for <b>S53</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01238/suppl_file/jm9b01238_si_008.cif">CIF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01238/suppl_file/jm9b01238_si_001.pdf">jm9b01238_si_001.pdf (14.15 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01238/suppl_file/jm9b01238_si_002.pdf">jm9b01238_si_002.pdf (34.35 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01238/suppl_file/jm9b01238_si_003.pdf">jm9b01238_si_003.pdf (587.85 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01238/suppl_file/jm9b01238_si_004.csv">jm9b01238_si_004.csv (4.02 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01238/suppl_file/jm9b01238_si_005.pdb">jm9b01238_si_005.pdb (208.92 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01238/suppl_file/jm9b01238_si_006.cif">jm9b01238_si_006.cif (556.52 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01238/suppl_file/jm9b01238_si_007.cif">jm9b01238_si_007.cif (808.9 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01238/suppl_file/jm9b01238_si_008.cif">jm9b01238_si_008.cif (954.93 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The coordinates and structure factors of the hNNMT-NS1 (<b>10</b>) cocrystal structure have been deposited with the RCSB Protein Data Bank under accession code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6ORR">6ORR</a>. The authors will release the atomic coordinates upon article publication.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.9b01238&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2019.62.issue-21%3BpageGroup%3Astring%3APublication+Pages%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.9b01238%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.9b01238" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                5MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"67c796a18d482521","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
