
<html lang="en"     class="pb-page"  data-request-id="6a99aeea-3f0c-471f-9f0d-002440abc81c"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;issue:issue:10.1021/jmcmar.2018.61.issue-3;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.7b01473;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="MetaQSAR: An Integrated Database Engine to Manage and Analyze Metabolic Data" /></meta><meta name="dc.Creator" content="Alessandro  Pedretti" /></meta><meta name="dc.Creator" content="Angelica  Mazzolari" /></meta><meta name="dc.Creator" content="Giulio  Vistoli" /></meta><meta name="dc.Creator" content="Bernard  Testa" /></meta><meta name="dc.Description" content="The study describes the MetaQSAR tool, a new database engine specifically tailored to collect and analyze metabolic data. This is a plug-in embedded in the VEGA suite of programs (freely downloadab..." /></meta><meta name="Description" content="The study describes the MetaQSAR tool, a new database engine specifically tailored to collect and analyze metabolic data. This is a plug-in embedded in the VEGA suite of programs (freely downloadab..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="January 17, 2018" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.7b01473" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2017 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.7b01473" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.7b01473" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.7b01473" /></link>
        
    
    

<title>MetaQSAR: An Integrated Database Engine to Manage and Analyze Metabolic Data | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.7b01473" /></meta><meta property="og:title" content="MetaQSAR: An Integrated Database Engine to Manage and Analyze Metabolic Data" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01473/20180201/images/large/jm-2017-01473n_0006.jpeg" /></meta><meta property="og:description" content="The study describes the MetaQSAR tool, a new database engine specifically tailored to collect and analyze metabolic data. This is a plug-in embedded in the VEGA suite of programs (freely downloadable at www.vegazz.net) and takes advantage from all cheminformatics features implemented in the software with additional tools aimed to perform statistical analyses, similarity searches, and physicochemical profiling of the stored molecules. MetaQSAR also implements a novel metabolism classification, which groups the metabolic reactions in 101 classes and can find numerous applications in metabolic analyses. The potentials of MetaQSAR are here assessed by using it to store and analyze an extended database focused on metabolism of xenobiotics, which was collected by manually curated meta-analysis of the recent literature. The database includes 1890 substrates taken from about 1500 original papers in the years 2004–2015. The database was utilized in both physicochemical analyses and similarity searches, so evidencing the notable potentialities of MetaQSAR, which can find particularly fruitful applications in developing targeted predictive approaches." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.7b01473"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.7b01473">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-8/abseba.2021.7.issue-8/20210809/abseba.2021.7.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-15/accacs.2021.11.issue-15/20210806/accacs.2021.11.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-31/acsodf.2021.6.issue-31/20210810/acsodf.2021.6.issue-31.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-31/ascecg.2021.9.issue-31/20210809/ascecg.2021.9.issue-31.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-31/ancham.2021.93.issue-31/20210810/ancham.2021.93.issue-31.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-31/bichaw.2021.60.issue-31/20210810/bichaw.2021.60.issue-31.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-8/bomaf6.2021.22.issue-8/20210809/bomaf6.2021.22.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-29/cgeabj.2021.99.issue-29/20210809-01/cgeabj.2021.99.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-15/cmatex.2021.33.issue-15/20210810/cmatex.2021.33.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-15/enfuem.2021.35.issue-15/20210805/enfuem.2021.35.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-8/estlcu.2021.8.issue-8/20210810/estlcu.2021.8.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-8/jceda8.2021.98.issue-8/20210810/jceda8.2021.98.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-8/jctcce.2021.17.issue-8/20210810/jctcce.2021.17.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-15/joceah.2021.86.issue-15/20210806/joceah.2021.86.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-30/jpcafh.2021.125.issue-30/20210805/jpcafh.2021.125.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-30/jpcbfk.2021.125.issue-30/20210805/jpcbfk.2021.125.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-30/jpccck.2021.125.issue-30/20210805/jpccck.2021.125.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-30/jpclcd.2021.12.issue-30/20210805/jpclcd.2021.12.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-8/jprobs.2021.20.issue-8/20210806/jprobs.2021.20.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-31/langd5.2021.37.issue-31/20210810/langd5.2021.37.issue-31.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-15/mamobx.2021.54.issue-15/20210810/mamobx.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-15/orlef7.2021.23.issue-15/20210806/orlef7.2021.23.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-15/orgnd7.2021.40.issue-15/20210809/orgnd7.2021.40.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.7b01473&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.7b01473&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.7b01473&amp;href=/doi/10.1021/acs.jmedchem.7b01473" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2018</span><span class="cit-fg-volume">, 61</span><span class="cit-fg-issue">, 3</span><span class="cit-fg-pageRange">, 1019-1030</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/61/3" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.7b01466" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.7b01474" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">MetaQSAR: An Integrated Database Engine to Manage and Analyze Metabolic Data</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Alessandro++Pedretti">Alessandro Pedretti</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="loa-info-orcid"><a href="http://orcid.org/0000-0001-5916-2029" title="Orcid link"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img></a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Angelica++Mazzolari">Angelica Mazzolari</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Giulio++Vistoli">Giulio Vistoli</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="loa-info-orcid"><a href="http://orcid.org/0000-0002-3939-5172" title="Orcid link"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img></a></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Bernard++Testa">Bernard Testa</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span><span class="author-xref-symbol "><sup>§</sup></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-symbol">†</span> <span class="aff-text">Dipartimento di Scienze Farmaceutiche “Pietro Pratesi”, Facoltà di Farmacia, Università degli Studi di Milano, Via Luigi Mangiagalli 25, I-20133 Milano, Italy</span></div><div class="aff-info" id="aff2"><span class="aff-symbol">‡</span> <span class="aff-text">University of Lausanne, 1015 Lausanne, Switzerland</span></div><div class="corresp-info"><strong>*</strong>Phone: +390250319332. Fax: +390250319359. E-mail: <a href="/cdn-cgi/l/email-protection#42232e273131232c26302d6c322726302736362b02372c2b2f2b6c2b36"><span class="__cf_email__" data-cfemail="3d5c51584e4e5c53594f52134d58594f584949547d4853545054135449">[email protected]</span></a></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.7b01473&amp;href=/doi/10.1021%2Facs.jmedchem.7b01473" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2018</span></span><span class="cit-volume">, 61</span><span class="cit-issue">, 3</span><span class="cit-pageRange">, 1019–1030</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">December 15, 2017</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>4 October 2017</li><li><span class="item_label"><b>Published</b> online</span>17 January 2018</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 8 February 2018</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.7b01473" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b01473</a></div><div class="article_header-article-copyright"><strong>Copyright © 2017 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D1019%26pageCount%3D12%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DAlessandro%2BPedretti%252C%2BAngelica%2BMazzolari%252C%2BGiulio%2BVistoli%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D61%26issueNum%3D3%26contentID%3Dacs.jmedchem.7b01473%26title%3DMetaQSAR%253A%2BAn%2BIntegrated%2BDatabase%2BEngine%2Bto%2BManage%2Band%2BAnalyze%2BMetabolic%2BData%26numPages%3D12%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D1030%26publicationDate%3DFebruary%2B2018">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.7b01473"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1106</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">13</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.7b01473" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;MetaQSAR: An Integrated Database Engine to Manage and Analyze Metabolic Data&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Alessandro&quot;,&quot;last_name&quot;:&quot;Pedretti&quot;},{&quot;first_name&quot;:&quot;Angelica&quot;,&quot;last_name&quot;:&quot;Mazzolari&quot;},{&quot;first_name&quot;:&quot;Giulio&quot;,&quot;last_name&quot;:&quot;Vistoli&quot;},{&quot;first_name&quot;:&quot;Bernard&quot;,&quot;last_name&quot;:&quot;Testa&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2018&quot;,&quot;month&quot;:&quot;01&quot;,&quot;day&quot;:&quot;17&quot;,&quot;issue&quot;:&quot;3&quot;,&quot;volume&quot;:&quot;61&quot;,&quot;pages&quot;:&quot;1019-1030&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.7b01473&quot;},&quot;abstract&quot;:&quot;The study describes the MetaQSAR tool, a new database engine specifically tailored to collect and analyze metabolic data. This is a plug-in embedded in the VEGA suite of programs (freely downloadable at www.vegazz.net) and takes advantage from all cheminformatics features implemented in the software with additional tools aimed to perform statistical analyses, similarity searches, and physicochemical profiling of the stored molecules. MetaQSAR also implements a novel metabolism classification, which groups the metabolic reactions in 101 classes and can find numerous applications in metabolic analyses. The potentials of MetaQSAR are here assessed by using it to store and analyze an extended database focused on metabolism of xenobiotics, which was collected by manually curated meta-analysis of the recent literature. The database includes 1890 substrates taken from about 1500 original papers in the years 2004–2015. The database was utilized in both physicochemical analyses and similarity searches, so evidencing t&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b01473&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b01473" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b01473&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b01473" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b01473&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b01473" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.7b01473&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b01473&amp;href=/doi/10.1021/acs.jmedchem.7b01473" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.7b01473" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.7b01473" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (3 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b01473&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.7b01473%26sid%3Dliteratum%253Aachs%26pmid%3D29244953%26genre%3Darticle%26aulast%3DPedretti%26date%3D2018%26atitle%3DMetaQSAR%253A%2BAn%2BIntegrated%2BDatabase%2BEngine%2Bto%2BManage%2Band%2BAnalyze%2BMetabolic%2BData%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D61%26issue%3D3%26spage%3D1019%26epage%3D1030%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291719" title="Redox reactions">Redox reactions</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=292143" title="Metabolism">Metabolism</a>,</li><li><a href="/action/doSearch?ConceptID=290731" title="Biological databases">Biological databases</a>,</li><li><a href="/action/doSearch?ConceptID=290644" title="Molecular properties">Molecular properties</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/61/3" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/jmcmar.2018.61.issue-3/20180208/jmcmar.2018.61.issue-3.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01473/20180201/images/medium/jm-2017-01473n_0006.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01473/20180201/images/large/jm-2017-01473n_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01473&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">The study describes the MetaQSAR tool, a new database engine specifically tailored to collect and analyze metabolic data. This is a plug-in embedded in the VEGA suite of programs (freely downloadable at <a href="http://www.vegazz.net" class="extLink">www.vegazz.net</a>) and takes advantage from all cheminformatics features implemented in the software with additional tools aimed to perform statistical analyses, similarity searches, and physicochemical profiling of the stored molecules. MetaQSAR also implements a novel metabolism classification, which groups the metabolic reactions in 101 classes and can find numerous applications in metabolic analyses. The potentials of MetaQSAR are here assessed by using it to store and analyze an extended database focused on metabolism of xenobiotics, which was collected by manually curated meta-analysis of the recent literature. The database includes 1890 substrates taken from about 1500 original papers in the years 2004–2015. The database was utilized in both physicochemical analyses and similarity searches, so evidencing the notable potentialities of MetaQSAR, which can find particularly fruitful applications in developing targeted predictive approaches.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">1 Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_08668" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_08668" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The recent and steady progress in omics disciplines, and in related analytical techniques, has strongly increased the amount of scientific data routinely available to researchers.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> This growth involved both biological data (coming for example from HTS campaigns, genomic, or proteomic studies) and chemical data (coming from combinatorial synthesis, compounds’ collection and physicochemical profiling).<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> This growth fostered the development of computational methods able to store and to analyze a tremendous amount of scientific data, thus laying the ground for the emerging bioinformatics and cheminformatics fields.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> While involving different objects and pursuing different objectives, these two fields similarly imply the development of in silico tools able to collect and to analyze scientific databases often combined with efficient procedures to automatically retrieve scientific data from on line resources or scientific publications. Moreover, cheminformatics and bioinformatics approaches can be synergistically related because they investigate the same processes from chemical or biological standpoints, respectively.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a></div><div class="NLM_p">The study of drug metabolism offers a particularly fertile arena where bioinformatics and cheminformatics approaches can feed each other because metabolism combines the cheminformatics profiling of substrates and metabolites with the bioinformatics analyses of the involved biochemical pathways and proteins.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> Furthermore, databases for metabolic data can also take advantage of the approaches adopted to manage the databases of synthetic reactions because all these databases primarily involve an efficient collection of chemical reactions.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a></div><div class="NLM_p">The aforementioned progress has resulted in a very rich arsenal of databases and web-based resources focused on metabolic data and biochemical pathways, as recently reviewed.<a onclick="showRef(event, 'ref7 ref8'); return false;" href="javascript:void(0);" class="ref ref7 ref8">(7, 8)</a> However, the robotized acquisition of scientific data from various available resources may limit the critical curation and the accuracy of the collected data, resulting in databases which are suitable for qualitative analyses and comparisons but may prove ineffective in quantitative predictions where the inaccuracy of only a few data can impair the overall predictive power of the resulting models.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> This concern is particularly felt in metabolism prediction, which while not necessarily requiring huge databases is strongly influenced by the accuracy of the experimental metabolic data on which it is based. Finally, the existing resources focused on human metabolism are primarily bioinformatics-oriented and not tailored to drug metabolism because they are aimed mainly at predicting biochemical pathways, a task usually performed in metabolomics analyses. In contrast, a limited number of cheminformatics-oriented applications have been proposed even though they could find fruitful applications in metabolism prediction.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a></div><div class="NLM_p">On these grounds, the present study describes a novel application (MetaQSAR) embedded in the VEGA suite of programs<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> and specifically tailored to generate and manage metabolic databases. MetaQSAR has been developed as a plug-in of VEGA ZZ and can be used to collect, classify, and analyze metabolic reactions in an effective way while representing a versatile engine to develop in-house targeted applications for metabolism analysis and prediction. In detail, MetaQSAR is a relational database that comprises a management system which connects the database to the graphic user interface (GUI) for input, data analysis, and delivery. Notably, MetaQSAR includes a novel classification for the metabolic reactions which subdivides them into 28 classes and 101 subclasses, thus allowing an efficient clustering of the stored data.</div><div class="NLM_p last">To assess its potential, MetaQSAR was utilized to store and to analyze an extended version of the metabolic database already collected for our previous study.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> Besides an updated evaluation of the relative importance of biotransformation reactions in the metabolism of xenobiotics, the analysis reported here will exploit the cheminformatics features implemented in VEGA ZZ to profile the physicochemical properties of the included molecules as well as to perform similarity analyses. The results emphasize that cheminformatics methods applied to metabolism databases can provide meaningful results with noteworthy potential in developing similarity-based predictive approaches.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">2 Implementation</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_46424" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_46424" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i3" class="anchor-spacer"></div><h3 class="article-section__title" id="_i3">2.1 MetaQSAR Database</h3><div id="sec2_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i4" class="anchor-spacer"></div><h4 class="article-section__title" id="_i4">2.1.1 Database Structure</h4><div class="NLM_p">The database structure is shown in <a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a> in which the different data types are organized in tables (here shown as boxes) colored in different ways according to the type of information they contain. The relations between the tables, depicted by red arrows, indicate the logical links between rows of different tables. The pivotal table is reactions (sandy colored), in which each metabolic reaction is characterized by the following data: the substrate, the substrate atom(s) involved in the reaction, the metabolite formed, the reaction class, the enzyme(s) catalyzing the reaction, and the generation of the metabolite along with its toxicity and/or reactivity (these last two features encoded by simple Boolean descriptors). It should be remembered that a metabolite can be classified of first generation if directly formed by the xenobiotic, of second generation when coming from a first generation metabolite, and so on. The substrates and products of the reactions are saved in the molecules table (light-blue box), which includes the 1D and 2D structures as well as several molecular descriptors and fingerprints that can be used for similarity searches and physicochemical analyses. The corresponding 3D structures are stored into a different table (structures) to enhance their analysis. Notice that the molecules and structures tables do not discriminate between substrates and metabolites because a given molecule can be the product of an enzymatic reaction and the substrate of subsequent reaction and thus a unique classification would be impossible.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01473/20180201/images/medium/jm-2017-01473n_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01473/20180201/images/large/jm-2017-01473n_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Structure of the implemented metabolism database with the logical links between the different data types which are organized in tables here represented as boxes colored according to the data type.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01473/20180201/images/large/jm-2017-01473n_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01473&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Each object in the molecules table is linked to its bibliographic data (gray boxes). Moreover, a compact and human-readable <i>paper code</i> was implemented to simplify the management of the reviewed papers, and the user can define its own paper code provided it does not exceed the 16 characters.</div><div class="NLM_p">Sixteen enzyme classes (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>) have been defined in the enzymes table (yellow box) to be linked to the reactions table to specify the class of the major and possibly alternative enzymes involved in the reaction. The reaction class is probably the most informative property, which can be included when a new metabolic reaction is entered into the database. Indeed, MetaQSAR includes an enhanced classification, the scientific rationale of which is explained in the results and which subdivides the metabolic reactions into 101 classes. As compiled in <a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>, these 101 reaction classes are organized in a framework composed of three levels corresponding to (1) main classes (compiled in the ReaMain table), (2) classes (in the ReaClasses table), and (3) subclasses (in the ReaSubClasses table). This classification method allows searches at different levels of detail. For example, one may retrieve all redox reactions (main class level), only the Csp3 oxidations (class level), or even only the hydroxylation reactions of an isolated Csp3 atom (subclass level).</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Enzyme Classes Stored in the Enzyme Table Used for the Reaction Entries with the Corresponding Relative Abundance of Class As Monitored in the Here Collected Database</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">ID</th><th class="colsep0 rowsep0" align="center">enzyme class</th><th class="colsep0 rowsep0" align="center" char=".">%</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">cytochromes P450</td><td class="colsep0 rowsep0" align="char" char=".">55.38</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left">dehydrogenases</td><td class="colsep0 rowsep0" align="char" char=".">2.29</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left">FMO</td><td class="colsep0 rowsep0" align="char" char=".">4.19</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="left">XO, AO</td><td class="colsep0 rowsep0" align="char" char=".">2.17</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">5</td><td class="colsep0 rowsep0" align="left">peroxidases</td><td class="colsep0 rowsep0" align="char" char=".">2.39</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">6</td><td class="colsep0 rowsep0" align="left">other reductases</td><td class="colsep0 rowsep0" align="char" char=".">2.78</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">7</td><td class="colsep0 rowsep0" align="left">other oxidoreductases or autoxidations</td><td class="colsep0 rowsep0" align="char" char=".">0.59</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">8</td><td class="colsep0 rowsep0" align="left">hydrolases</td><td class="colsep0 rowsep0" align="char" char=".">9.16</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">9</td><td class="colsep0 rowsep0" align="left">UDP-glucuronosyltransferases</td><td class="colsep0 rowsep0" align="char" char=".">10.14</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">10</td><td class="colsep0 rowsep0" align="left">sulfotransferases</td><td class="colsep0 rowsep0" align="char" char=".">2.13</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">11</td><td class="colsep0 rowsep0" align="left">glutathione S-transferases and subsequent enzymes/reactions</td><td class="colsep0 rowsep0" align="char" char=".">2.37</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">12</td><td class="colsep0 rowsep0" align="left">acetyltransferases</td><td class="colsep0 rowsep0" align="char" char=".">0.94</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">13</td><td class="colsep0 rowsep0" align="left">acyl-CoA ligases and subsequent enzymes</td><td class="colsep0 rowsep0" align="char" char=".">1.23</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">14</td><td class="colsep0 rowsep0" align="left">methyltransferases</td><td class="colsep0 rowsep0" align="char" char=".">1.41</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">15</td><td class="colsep0 rowsep0" align="left">other transferases or nonenzymatic conjugations</td><td class="colsep0 rowsep0" align="char" char=".">1.82</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">16</td><td class="colsep0 rowsep0" align="left">nonenzymatic hydrolyses or (de)hydrations</td><td class="colsep0 rowsep0" align="char" char=".">1.02</td></tr></tbody></table></div></div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Classification of the Metabolic Reactions According to Main Class/Class/Subclass Scheme with the Corresponding Relative Abundance of Each Subclass As Monitored in the Here Collected Database</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="center" /></col><col align="left" /></col><col align="center" /></col><col align="left" /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" align="center">main classes</th><th class="rowsep1 colsep0" colspan="2" align="center">classes</th><th class="rowsep1 colsep0" colspan="2" align="center">subclasses</th><th class="colsep0 rowsep0" align="center" char="."> </th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">count</th><th class="colsep0 rowsep0" align="center">description</th><th class="colsep0 rowsep0" align="center">code</th><th class="colsep0 rowsep0" align="center">description</th><th class="colsep0 rowsep0" align="center">code</th><th class="colsep0 rowsep0" align="center">description</th><th class="colsep0 rowsep0" align="center" char=".">%</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">redox</td><td class="colsep0 rowsep0" align="center">01</td><td class="colsep0 rowsep0" align="left">oxidation of Csp<sup>3</sup></td><td class="colsep0 rowsep0" align="center">01</td><td class="colsep0 rowsep0" align="left">hydroxylation (or other oxidations) of isolated Csp<sup>3</sup></td><td class="colsep0 rowsep0" align="char" char=".">7.04</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">02</td><td class="colsep0 rowsep0" align="left">hydroxylation (or other oxidations) of C α to an unsaturated system (>C═C, >C═O, –C≡N, aryl)</td><td class="colsep0 rowsep0" align="char" char=".">6.89</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">03</td><td class="colsep0 rowsep0" align="left">hydroxylation (or other oxidations) of Csp<sup>3</sup> carrying an heteroatom (N, O, S, halo) (including subsequent dealkylation, deamination, or dehalogenation)</td><td class="colsep0 rowsep0" align="char" char=".">16.35</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">04</td><td class="colsep0 rowsep0" align="left">dehydrogenation of  >CH–CH< to >C═C<, >CH-N< to>C═N– (incl >C═N+<)</td><td class="colsep0 rowsep0" align="char" char=".">1.97</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">5</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">05</td><td class="colsep0 rowsep0" align="left">other Csp<sup>3</sup> oxidations (organometallic dealkylation, C–C cleavage, etc.)</td><td class="colsep0 rowsep0" align="char" char=".">0.21</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="7" align="left">    </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">6</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">02</td><td class="colsep0 rowsep0" align="left">oxidation of Csp<sup>2</sup> and Csp</td><td class="colsep0 rowsep0" align="center">01</td><td class="colsep0 rowsep0" align="left">oxidation of aryl compounds to epoxides, phenols, or other metabolites</td><td class="colsep0 rowsep0" align="char" char=".">16.54</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">7</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">02</td><td class="colsep0 rowsep0" align="left">oxidation of azarenes to lactams or other metabolites</td><td class="colsep0 rowsep0" align="char" char=".">2.37</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">8</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">03</td><td class="colsep0 rowsep0" align="left">oxygenation of >C═C< bonds to epoxides or other metabolites</td><td class="colsep0 rowsep0" align="char" char=".">1.24</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">9</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">04</td><td class="colsep0 rowsep0" align="left">oxygenation of –C-C≡C–H and –C-C≡C–C– bonds</td><td class="colsep0 rowsep0" align="char" char=".">0.17</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="7" align="left">     </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">10</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">03</td><td class="colsep0 rowsep0" align="left">–CHOH ↔ >C═O → –COOH</td><td class="colsep0 rowsep0" align="center">01</td><td class="colsep0 rowsep0" align="left">dehydrogenation of –CH<sub>2</sub>OH groups to −CHO and of >CHOH to >C═O</td><td class="colsep0 rowsep0" align="char" char=".">0.94</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">11</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">02</td><td class="colsep0 rowsep0" align="left">hydrogenation of −CHO to −CH<sub>2</sub>OH and of >C═O to >CHOH</td><td class="colsep0 rowsep0" align="char" char=".">1.86</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">12</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">03</td><td class="colsep0 rowsep0" align="left">oxidation of −CHO to –COOH</td><td class="colsep0 rowsep0" align="char" char=".">0.15</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">13</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">04</td><td class="colsep0 rowsep0" align="left">dehydrogenation of dihydrodiols to <i>ortho</i>-diphenols</td><td class="colsep0 rowsep0" align="char" char=".">0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="7" align="left">     </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">14</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">04</td><td class="colsep0 rowsep0" align="left">various redox reactions of carbon atoms</td><td class="colsep0 rowsep0" align="center">01</td><td class="colsep0 rowsep0" align="left">oxidative decarboxylation</td><td class="colsep0 rowsep0" align="char" char=".">0.04</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">15</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">02</td><td class="colsep0 rowsep0" align="left">reductive dehalogenations</td><td class="colsep0 rowsep0" align="char" char=".">0.13</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">16</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">03</td><td class="colsep0 rowsep0" align="left">reduction of arene and alkene epoxides</td><td class="colsep0 rowsep0" align="char" char=".">0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">17</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">04</td><td class="colsep0 rowsep0" align="left">other C reductions, e.g., of >C═C< to –CH<sub>2</sub>–CH<sub>2</sub>–</td><td class="colsep0 rowsep0" align="char" char=".">0.75</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="7" align="left">     </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">18</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">05</td><td class="colsep0 rowsep0" align="left">redox reactions of R<sub>3</sub>N</td><td class="colsep0 rowsep0" align="center">01</td><td class="colsep0 rowsep0" align="left">oxidation of tertiary alkylamines and heterocyclic amines to N-oxides or other metabolites</td><td class="colsep0 rowsep0" align="char" char=".">1.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">19</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">02</td><td class="colsep0 rowsep0" align="left">oxidation of tertiary arylamines, azarenes and azo compounds to N-oxides or other metabolites</td><td class="colsep0 rowsep0" align="char" char=".">1.05</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">20</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">03</td><td class="colsep0 rowsep0" align="left">reduction of N-oxides</td><td class="colsep0 rowsep0" align="char" char=".">0.21</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="7" align="left">     </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">21</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">06</td><td class="colsep0 rowsep0" align="left">oxidation of >NH, >NOH, and –N═O/reduction of –NO<sub>2</sub>, –N═O, >NOH, etc.</td><td class="colsep0 rowsep0" align="center">01</td><td class="colsep0 rowsep0" align="left">hydroxylation of amines to hydroxylamines or intermediates</td><td class="colsep0 rowsep0" align="char" char=".">1.42</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">22</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">02</td><td class="colsep0 rowsep0" align="left">hydroxylation of amides to hydroxylamides</td><td class="colsep0 rowsep0" align="char" char=".">0.21</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">23</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">03</td><td class="colsep0 rowsep0" align="left">oxidation of primary hydroxylamines to nitroso compounds or oximes (incl spontaneous dismutation), then to nitro compounds</td><td class="colsep0 rowsep0" align="char" char=".">0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">24</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">04</td><td class="colsep0 rowsep0" align="left">reduction of hydroxylamines and hydroxylamides (incl spontaneous dismutation)</td><td class="colsep0 rowsep0" align="char" char=".">0.27</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">25</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">05</td><td class="colsep0 rowsep0" align="left">reduction of nitroso compounds and oximes to hydroxylamines</td><td class="colsep0 rowsep0" align="char" char=".">0.29</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">26</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">06</td><td class="colsep0 rowsep0" align="left">reduction of nitro compounds to nitroso compounds</td><td class="colsep0 rowsep0" align="char" char=".">0.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">27</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">07</td><td class="colsep0 rowsep0" align="left">other N-oxidations (1,4-dihydropyridines, etc.)</td><td class="colsep0 rowsep0" align="char" char=".">0.19</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">28</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">08</td><td class="colsep0 rowsep0" align="left">other N-reductions (e.g., azo compounds to hydrazines, hydrazines to amines, reductive N-rings opening)</td><td class="colsep0 rowsep0" align="char" char=".">0.27</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="7" align="left">     </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">29</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">07</td><td class="colsep0 rowsep0" align="left">oxidation to quinones or analogues/reduction of quinones and analogues</td><td class="colsep0 rowsep0" align="center">01</td><td class="colsep0 rowsep0" align="left">oxidation of diphenols to quinones</td><td class="colsep0 rowsep0" align="char" char=".">0.31</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">30</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">02</td><td class="colsep0 rowsep0" align="left">oxidation of amino- and amido-phenols to quinoneimines or quinoneimides, respectively</td><td class="colsep0 rowsep0" align="char" char=".">0.23</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">31</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">03</td><td class="colsep0 rowsep0" align="left">oxidation of cresols and analogues to quinonemethides</td><td class="colsep0 rowsep0" align="char" char=".">0.46</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">32</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">04</td><td class="colsep0 rowsep0" align="left">other oxidations of phenols and amines (dimerization, quinone-like metabolites, etc.)</td><td class="colsep0 rowsep0" align="char" char=".">1.19</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">33</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">05</td><td class="colsep0 rowsep0" align="left">reduction of quinones and analogues</td><td class="colsep0 rowsep0" align="char" char=".">0.25</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="7" align="left">     </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">34</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">08</td><td class="colsep0 rowsep0" align="left">oxidation and reduction of S atoms</td><td class="colsep0 rowsep0" align="center">01</td><td class="colsep0 rowsep0" align="left">oxidation of thiols to sulfenic acids or disulfides</td><td class="colsep0 rowsep0" align="char" char=".">0.19</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">35</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">02</td><td class="colsep0 rowsep0" align="left">oxygenation of sulfenic acids to sulfinic acids, and of sulfinic acids to sulfonic acids</td><td class="colsep0 rowsep0" align="char" char=".">0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">36</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">03</td><td class="colsep0 rowsep0" align="left">oxygenation of sulfides to sulfoxides, and of sulfoxides to sulfones</td><td class="colsep0 rowsep0" align="char" char=".">1.97</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">37</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">04</td><td class="colsep0 rowsep0" align="left">oxygenation of thiones (>C═S) or thioamides to sulfines, and of sulfines to sulfenes</td><td class="colsep0 rowsep0" align="char" char=".">0.34</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">38</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">05</td><td class="colsep0 rowsep0" align="left">oxidative desulfurations of >C═S to ketones, and of –P═S to –P═O groups</td><td class="colsep0 rowsep0" align="char" char=".">0.23</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">39</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">06</td><td class="colsep0 rowsep0" align="left">S-oxygenations of disulfides, thiosulfinates (−SO-S−), α-disulfoxides (−SO-SO−), and thiosulfonates (−SO<sub>2</sub>-S−)</td><td class="colsep0 rowsep0" align="char" char=".">0.04</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">40</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">07</td><td class="colsep0 rowsep0" align="left">reduction of disulfides to thiols</td><td class="colsep0 rowsep0" align="char" char=".">0.02</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">41</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">08</td><td class="colsep0 rowsep0" align="left">reduction of sulfoxides to sulfides</td><td class="colsep0 rowsep0" align="char" char=".">0.08</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">42</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">09</td><td class="colsep0 rowsep0" align="left">other S-reductions</td><td class="colsep0 rowsep0" align="char" char=".">0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="7" align="left">     </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">43</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">09</td><td class="colsep0 rowsep0" align="left">redox reactions of other atoms</td><td class="colsep0 rowsep0" align="center">01</td><td class="colsep0 rowsep0" align="left">oxidation of silicon, phosphorus, arsenic and other elements</td><td class="colsep0 rowsep0" align="char" char=".">0.15</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">44</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">02</td><td class="colsep0 rowsep0" align="left">reduction of Se, P, Hg, As, and other elements</td><td class="colsep0 rowsep0" align="char" char=".">0.06</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="7" align="left">     </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">45</td><td class="colsep0 rowsep0" align="left">hydrolysis and other</td><td class="colsep0 rowsep0" align="center">11</td><td class="colsep0 rowsep0" align="left">hydrolysis of esters, lactones, and inorganic esters</td><td class="colsep0 rowsep0" align="center">01</td><td class="colsep0 rowsep0" align="left">hydrolysis of alkyl esters</td><td class="colsep0 rowsep0" align="char" char=".">2.05</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">46</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">02</td><td class="colsep0 rowsep0" align="left">hydrolysis of aryl esters</td><td class="colsep0 rowsep0" align="char" char=".">0.25</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">47</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">03</td><td class="colsep0 rowsep0" align="left">hydrolysis of anionic and cationic esters</td><td class="colsep0 rowsep0" align="char" char=".">2.01</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">48</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">04</td><td class="colsep0 rowsep0" align="left">hydrolysis of linear and cyclic carbamates (>N-CO-OR′) and carbonates (RO-CO-OR′)</td><td class="colsep0 rowsep0" align="char" char=".">0.54</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">49</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">05</td><td class="colsep0 rowsep0" align="left">hydrolysis of acyl β-glucuronides or other acylglycosides</td><td class="colsep0 rowsep0" align="char" char=".">0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">50</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">06</td><td class="colsep0 rowsep0" align="left">reversible hydrolytic opening of lactone rings</td><td class="colsep0 rowsep0" align="char" char=".">0.19</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">51</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">07</td><td class="colsep0 rowsep0" align="left">hydrolysis of thioesters (RCO-SR′ and RCS-SR′) and thiolactones</td><td class="colsep0 rowsep0" align="char" char=".">0.17</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">52</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">08</td><td class="colsep0 rowsep0" align="left">hydrolysis of esters of inorganic acids (nitrates, nitrites, sulfates, sulfamates, phosphates, phosphonates, etc.)</td><td class="colsep0 rowsep0" align="char" char=".">0.92</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="7" align="left">     </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">53</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">12</td><td class="colsep0 rowsep0" align="left">hydrolysis of amides, lactams and peptides</td><td class="colsep0 rowsep0" align="center">01</td><td class="colsep0 rowsep0" align="left">hydrolysis of alkyl and aryl amides [alkyl-CO-N< and aryl-CO-N<]</td><td class="colsep0 rowsep0" align="char" char=".">0.94</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">54</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">02</td><td class="colsep0 rowsep0" align="left">hydrolysis of anilides [aryl-N-CO-C∼], hydrazides [−CO-NHNN-CO-N<]</td><td class="colsep0 rowsep0" align="char" char=".">1.11</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">55</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">03</td><td class="colsep0 rowsep0" align="left">hydrolysis of lactams, cyclic imides and cyclic ureides</td><td class="colsep0 rowsep0" align="char" char=".">0.31</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">56</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">04</td><td class="colsep0 rowsep0" align="left">hydrolysis of peptide bonds</td><td class="colsep0 rowsep0" align="char" char=".">0.44</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="7" align="left">     </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">57</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">13</td><td class="colsep0 rowsep0" align="left">epoxide hydration</td><td class="colsep0 rowsep0" align="center">01</td><td class="colsep0 rowsep0" align="left">hydration of arene and alkene oxides</td><td class="colsep0 rowsep0" align="char" char=".">0.04</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="7" align="left">     </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">58</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">14</td><td class="colsep0 rowsep0" align="left">other hydrolyses/hydration reactions/nonenzymatic eliminations and rearrangements</td><td class="colsep0 rowsep0" align="center">01</td><td class="colsep0 rowsep0" align="left">hydrolysis of glucuronides and other glycosides (including N- and S-glycosides)</td><td class="colsep0 rowsep0" align="char" char=".">0.29</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">59</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">02</td><td class="colsep0 rowsep0" align="left">other ether hydrolyses (benzhydryl ethers, acetals, etc.)</td><td class="colsep0 rowsep0" align="char" char=".">0.04</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">60</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">03</td><td class="colsep0 rowsep0" align="left">hydrolytic cleavage of C═N bonds (in imines, hydrazones, imidates, amidines, oximes, oximines, isocyanates, etc.), and of C≡N bonds (nitriles). Double bond hydration</td><td class="colsep0 rowsep0" align="char" char=".">0.54</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">61</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">04</td><td class="colsep0 rowsep0" align="left">hydrolysis of linear Mannich bases (N-, O-, and S-Mannich bases) and cyclic Mannich bases (imidazolidines, oxazolidines, etc.)</td><td class="colsep0 rowsep0" align="char" char=".">0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">62</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">05</td><td class="colsep0 rowsep0" align="left">hydrolytic opening of other ring systems (1,2-oxazoles, etc.)</td><td class="colsep0 rowsep0" align="char" char=".">0.08</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">63</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">06</td><td class="colsep0 rowsep0" align="left">hydrolytic dehalogenations</td><td class="colsep0 rowsep0" align="char" char=".">0.21</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">64</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">07</td><td class="colsep0 rowsep0" align="left">substitution reactions (by H<sub>2</sub>O, halide, etc.) in complexes of Pt or other metals (with elimination of halide or other ligands)</td><td class="colsep0 rowsep0" align="char" char=".">0.02</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">65</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">08</td><td class="colsep0 rowsep0" align="left">bhond order increases by elimination of H<sub>2</sub>O or RSH or other Nu-H</td><td class="colsep0 rowsep0" align="char" char=".">0.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">66</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">09</td><td class="colsep0 rowsep0" align="left">cyclizations by intramolecular nucleophilic substitution (with elimination of amine, phenol, halide, or H<sub>2</sub>O)</td><td class="colsep0 rowsep0" align="char" char=".">0.08</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">67</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">10</td><td class="colsep0 rowsep0" align="left">any other nonredox, nonconjugation reaction</td><td class="colsep0 rowsep0" align="char" char=".">0.25</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="7" align="left">     </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">68</td><td class="colsep0 rowsep0" align="left">conjugations</td><td class="colsep0 rowsep0" align="center">21</td><td class="colsep0 rowsep0" align="left">O-glucuronidations and glycosylations</td><td class="colsep0 rowsep0" align="center">01</td><td class="colsep0 rowsep0" align="left">O-glucuronidation of alcohols</td><td class="colsep0 rowsep0" align="char" char=".">2.01</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">69</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">02</td><td class="colsep0 rowsep0" align="left">O-glucuronidation of phenols</td><td class="colsep0 rowsep0" align="char" char=".">3.94</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">70</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">03</td><td class="colsep0 rowsep0" align="left">O-glucuronidation of carboxylic acids (including subsequent rearrangements)</td><td class="colsep0 rowsep0" align="char" char=".">1.99</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">71</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">04</td><td class="colsep0 rowsep0" align="left">O-glucuronidation of hydroxylamines and hydroxylamides</td><td class="colsep0 rowsep0" align="char" char=".">0.15</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="7" align="left">     </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">72</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">22</td><td class="colsep0 rowsep0" align="left">N- and S-glucuronidations/all other glycosilations</td><td class="colsep0 rowsep0" align="center">01</td><td class="colsep0 rowsep0" align="left">N-glucuronidation of linear and cyclic amines (including ═N– in azarenes)</td><td class="colsep0 rowsep0" align="char" char=".">2.07</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">73</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">02</td><td class="colsep0 rowsep0" align="left">N-glucuronidation of amides</td><td class="colsep0 rowsep0" align="char" char=".">0.29</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">74</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">03</td><td class="colsep0 rowsep0" align="left">S-glucuronidation of thiols and thioacids, C-glucuronidation of acidic enols</td><td class="colsep0 rowsep0" align="char" char=".">0.04</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">75</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">04</td><td class="colsep0 rowsep0" align="left">any conjugations with glucose or other sugars</td><td class="colsep0 rowsep0" align="char" char=".">0.36</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="7" align="left">     </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">76</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">23</td><td class="colsep0 rowsep0" align="left">sulfonations (O–, N–, ...)</td><td class="colsep0 rowsep0" align="center">01</td><td class="colsep0 rowsep0" align="left">O-sulfonation of phenols</td><td class="colsep0 rowsep0" align="char" char=".">1.51</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">77</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">02</td><td class="colsep0 rowsep0" align="left">O-sulfonation of alcohols</td><td class="colsep0 rowsep0" align="char" char=".">0.44</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">78</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">03</td><td class="colsep0 rowsep0" align="left">other reactions (O-sulfonation of hydroxylamines, N-sulfonation of amines, etc.)</td><td class="colsep0 rowsep0" align="char" char=".">0.31</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="7" align="left">     </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">79</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">24</td><td class="colsep0 rowsep0" align="left">GSH and RSH conjugations + sequels/GSH-mediated reductions</td><td class="colsep0 rowsep0" align="center">01</td><td class="colsep0 rowsep0" align="left">nucleophilic additions of glutathione (to α,β-unsaturated carbonyls, quinones and analogues, isocyanates and isothiocyanates, epoxides, etc.)</td><td class="colsep0 rowsep0" align="char" char=".">1.72</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">90</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">02</td><td class="colsep0 rowsep0" align="left">reactions of glutathione addition–elimination (at C-X groups, acyl halides, halogenated olefins, etc.)</td><td class="colsep0 rowsep0" align="char" char=".">1.38</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">81</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">03</td><td class="colsep0 rowsep0" align="left">metabolic processing of glutathione conjugates up to thiols</td><td class="colsep0 rowsep0" align="char" char=".">0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">82</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">04</td><td class="colsep0 rowsep0" align="left">conjugation of glutathione with Hg, As, Pt, etc., compounds</td><td class="colsep0 rowsep0" align="char" char=".">0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">83</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">05</td><td class="colsep0 rowsep0" align="left">conjugations with other thiols (Cys, N-Ac-Cys, etc.)</td><td class="colsep0 rowsep0" align="char" char=".">0.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">84</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">06</td><td class="colsep0 rowsep0" align="left">reductions following glutathione conjugations</td><td class="colsep0 rowsep0" align="char" char=".">0.04</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">85</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">07</td><td class="colsep0 rowsep0" align="left">radical scavenging by glutathione and other thiols</td><td class="colsep0 rowsep0" align="char" char=".">0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="7" align="left">     </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">86</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">25</td><td class="colsep0 rowsep0" align="left">acetylations and acylations</td><td class="colsep0 rowsep0" align="center">01</td><td class="colsep0 rowsep0" align="left">N-acetylation of aromatic amines</td><td class="colsep0 rowsep0" align="char" char=".">0.67</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">87</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">02</td><td class="colsep0 rowsep0" align="left">N-acetylation of hydrazines and hydrazides</td><td class="colsep0 rowsep0" align="char" char=".">0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">88</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">03</td><td class="colsep0 rowsep0" align="left">other acetylations (N-acetylation of alkylamines, O-acetylation, etc.)</td><td class="colsep0 rowsep0" align="char" char=".">0.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">89</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">04</td><td class="colsep0 rowsep0" align="left">reactions of acylation (formylation, formation of fatty acyl esters, etc.)</td><td class="colsep0 rowsep0" align="char" char=".">0.19</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="7" align="left">     </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">90</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">26</td><td class="colsep0 rowsep0" align="left">CoASH-ligation followed by amino acid conjugations or other sequels</td><td class="colsep0 rowsep0" align="center">01</td><td class="colsep0 rowsep0" align="left">conjugation with glycine, glutamic acid, taurine, and other amino acids or short peptides</td><td class="colsep0 rowsep0" align="char" char=".">0.71</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">91</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">02</td><td class="colsep0 rowsep0" align="left">conjugation with carnitine</td><td class="colsep0 rowsep0" align="char" char=".">0.19</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">92</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">03</td><td class="colsep0 rowsep0" align="left">formation of hybrid glycerides, conjugation with cholesterol or other sterols</td><td class="colsep0 rowsep0" align="char" char=".">0.21</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">93</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">04</td><td class="colsep0 rowsep0" align="left">unidirectional chiral inversion of profens and analogues</td><td class="colsep0 rowsep0" align="char" char=".">0.06</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">94</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">05</td><td class="colsep0 rowsep0" align="left">chain elongation by 2C, β- or α-oxidation (loss of 2C or 1C, respectively), other sequels</td><td class="colsep0 rowsep0" align="char" char=".">0.13</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="7" align="left">     </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">95</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">27</td><td class="colsep0 rowsep0" align="left">methylations (O–, N–, S−)</td><td class="colsep0 rowsep0" align="center">01</td><td class="colsep0 rowsep0" align="left">O-methylation of catechols and other hydroxy groups</td><td class="colsep0 rowsep0" align="char" char=".">1.26</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">96</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">02</td><td class="colsep0 rowsep0" align="left">N-methylation of exocyclic and endocyclic amino groups (including ═N– in azarenes)</td><td class="colsep0 rowsep0" align="char" char=".">0.13</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">97</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">03</td><td class="colsep0 rowsep0" align="left">S-methylation of thiols</td><td class="colsep0 rowsep0" align="char" char=".">0.04</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">98</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">04</td><td class="colsep0 rowsep0" align="left">methylation of metals and metalloids (Hg, As, etc.)</td><td class="colsep0 rowsep0" align="char" char=".">0.02</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="7" align="left">     </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">99</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">28</td><td class="colsep0 rowsep0" align="left">other conjugations (PO<sub>4</sub>, CO<sub>2</sub>, ...)/transaminations</td><td class="colsep0 rowsep0" align="center">01</td><td class="colsep0 rowsep0" align="left">reactions of phosphorylation</td><td class="colsep0 rowsep0" align="char" char=".">0.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">100</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">02</td><td class="colsep0 rowsep0" align="left">nonenzymatic formation of hydrazones, binding of CO<sub>2</sub> to form carbamates</td><td class="colsep0 rowsep0" align="char" char=".">0.36</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">101</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="center">03</td><td class="colsep0 rowsep0" align="left">other reactions of conjugation; transaminations</td><td class="colsep0 rowsep0" align="char" char=".">0.29</td></tr></tbody></table></div></div></div><div id="sec2_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i7" class="anchor-spacer"></div><h4 class="article-section__title" id="_i7">2.1.2 MetaQSAR Graphical User Interface</h4><div class="NLM_p">The graphic user interface (GUI) of MetaQSAR was developed in C++ language as a plug-in of the VEGA ZZ program and is completely integrated in its graphic environment (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>). The interface is organized in five tabs in which the user can edit the metabolic reactions (reactions tab), manage the bibliographic data (papers and journals tabs), search for reactions by similarity or molecular properties (search tab), and analyze the data (statistics tab). For the interested readers, a detailed user’s guide can be found at <a href="http://nova.disfarm.unimi.it/manual/plugins/metaqsar.htm" class="extLink">http://nova.disfarm.unimi.it/manual/plugins/metaqsar.htm</a>.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01473/20180201/images/medium/jm-2017-01473n_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01473/20180201/images/large/jm-2017-01473n_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Graphical user interface of MetaQSAR (top right window) which is organized in tabs by which the user can perform the main tasks. When a substrate is selected, its 3D structure is displayed in the main VEGA ZZ window (top left window), allowing an easy selection of the atoms involved in a given reaction. Notice that these reactive atoms remain highlighted by a blue sphere. Examples of molecular searches which can be carried out using the MetaQSAR interface by structure (bottom left window) or by properties (bottom right window).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01473/20180201/images/large/jm-2017-01473n_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01473&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In the reactions tab, data entry is substrate-oriented because, when a new reaction is added, the substrate selection is the first step, and thus a new substrate has to be added before into the molecules table. Additionally, more advanced search modes are also available to find substrates with missing or incomplete metabolic data. Next, the metabolic reaction has to be defined using a graphical tree in which the reactions are grouped into different classes and which permits an easy selection. When a substrate is selected, the corresponding 3D structure is displayed in the VEGA ZZ main window, and for each reaction, the user can indicate the target atom(s) by clicking them in the 3D view. When the reactive atoms are not well-defined or uncertain, the reactive atoms can be better characterized by adding the confidence level of the data. For each added reaction, the user can finally define the generation to which a given metabolite belongs; enzymes involved and toxicity or reactivity can also be specified. In the papers tab, the user can manage the bibliographic data by adding new papers, editing them and linking substrates and products to their source in which they are cited.</div><div class="NLM_p">As exemplified in <a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>, MetaQSAR GUI includes some tools to retrieve and analyze the metabolic data stored in the database. In particular, in the search tab, substrates and products (i.e., molecules) can be searched by structural similarity with a query molecule or by molecular properties. For the searches by structure, the SMILES string of the query molecule is required. The Bingo similarity fingerprints of the substrates, as implemented by the Indigo toolkit (<a href="http://lifescience.opensource.epam.com/indigo/" class="extLink">http://lifescience.opensource.epam.com/indigo/</a>), are compared to that of the query molecule and evaluated by a Tanimoto similarity index.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> If the index so obtained is greater than a cutoff threshold specified by the user, the substrate and its metabolic reactions are shown as a tree report in the bottom box of the window and a preview of the corresponding 2D structure is shown by selecting a row of the tree. The search by molecular properties can be performed by specifying a set of criteria which are related to each other by logical operators (AND, OR). In detail, each criterion consists of a molecular descriptor, a mathematic operator, and a value (e.g., all substrates with log <i>P</i> > 5; all substrates with log <i>P</i> > 5 AND PSA > 80 Å<sup>2</sup> and so on). In such case also, the search results are shown in the window as a tree without the similarity values.</div><div class="NLM_p last">Several comprehensive statistics can be shown in the statistics tab, which includes four sections: main data, metabolic reactions, enzymes involved, and reaction counts. The main data section includes generic statistics such as the number of substrates, metabolic reactions, or enzyme classes. The metabolic reactions section shows the counts of the reactions according to metabolic generation (1, 2, and 3+) and of the reactions which yield toxic and/or reactive products. The enzyme and reaction sections report the statistics for each class (or subclass) of enzymes or metabolic reactions, reporting the number and the percentage of metabolic reactions belonging to each group (or subgroup). Similarly, bibliographic sections count the reviewed papers as classified by journal, publisher, or country.</div></div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8">2.2 Database Collection</h3><div class="NLM_p last">As mentioned in the <a class="ref internalNav" href="#sec1" aria-label="Introduction">Introduction</a>, the MetaQSAR engine was used to store an extended version of the database already described in a previous study. The main objective was the collection of a rather large number (>10<sup>3</sup>) of substrates and their metabolic reactions to be used in statistical analysis and as QSAR learning sets. The metabolic data were collected in a systematic paper-by-paper search of metabolic studies carried out in the primary literature (i.e., <i>Chemical Research in Toxicology</i>, <i>Xenobiotica</i>, and <i>Drug Metabolism and Disposition</i>) during the years 2004–2012 for the first two journals, and the years 2004–2015 for <i>Drug Metababolism and Disposition</i>. Great care was used in collecting, selecting, and screening literature metabolic data for entry into the database. Thus, the metabolites reported in each paper were examined critically for quality and adequacy according to a number of criteria which resemble those already exploited in the previous study and can be briefly summarized as follows:<table class="listgroup" border="0" width="95%" list-type="label"><tr class="li1"><td valign="top">1.</td><td colspan="5" valign="top" class="paragraph"><p class="last">The focus was on drugs and other xenobiotics, excluding endogenous compounds except when these are used as drugs (e.g., estradiol).</p></td></tr><tr class="li1"><td valign="top">2.</td><td colspan="5" valign="top" class="paragraph"><p class="last">The database was restricted to metabolic studies in humans and other mammals, carried out either in vivo, in cellular systems, or at a subcellular or enzymatic level.</p></td></tr><tr class="li1"><td valign="top">3.</td><td colspan="5" valign="top" class="paragraph"><p class="last">All papers were screened for biochemical and analytical quality.</p></td></tr><tr class="li1"><td valign="top">4.</td><td colspan="5" valign="top" class="paragraph"><p class="last">All structurally characterized metabolites were taken into account irrespective of their quantitative importance.</p></td></tr><tr class="li1"><td valign="top">5.</td><td colspan="5" valign="top" class="paragraph"><p class="last">The structure of a number of metabolites was not fully elucidated in the sense that the regioselectivity of the reaction was left undetermined. Such metabolites were deemed unfit for inclusion in relevant learning sets.</p></td></tr><tr class="li1"><td valign="top">6.</td><td colspan="5" valign="top" class="paragraph"><p class="last">The duplicate count of substrates and metabolites was avoided. For example, a given metabolite of a given substrate was counted only once when reported in several papers.</p></td></tr><tr class="li1"><td valign="top">7.</td><td colspan="5" valign="top" class="paragraph"><p class="last">Regio- and stereoisomers were considered as distinct molecules.</p></td></tr><tr class="li1"><td valign="top">8.</td><td colspan="5" valign="top" class="paragraph"><p class="last">When shown in the paper, or compatible with reported experimental conditions, the formation of some metabolites was assigned to two enzymatic (super)families or groups of enzymes. This was the case, e.g., for the CYP- and FMO-catalyzed oxygenation of some types of N- and S-containing functional groups or the CYP- and peroxidases-catalyzed formation of quinones from polyphenols.</p></td></tr><tr class="li1"><td valign="top">9.</td><td colspan="5" valign="top" class="paragraph"><p class="last">Two-electron reactions such as [−CHOH → >C═O], [>C═O → −CHOH] and [−CHO → −COOH] were each counted as a single step. The same applied, e.g., to the reactions [>NH → >NOH], {−NHOH → −N═O], [−NO<sub>2</sub> → −N═O], [N═O → −NHOH] and [>NOH → >NH].</p></td></tr></table></div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i9">3 Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_55400" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_55400" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec3_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10">3.1 Classification into Reaction Types and Enzymes</h3><div class="NLM_p">The first and major issue when setting up this database engine was to define a list of metabolic reactions that was as comprehensive and realistic as possible without being exaggeratedly long. Even though several classifications have been proposed for the enzymatic reactions mainly based on the involved enzymes (e.g., the EC system, <a href="http://www.sbcs.qmul.ac.uk/iubmb/enzyme/" class="extLink">http://www.sbcs.qmul.ac.uk/iubmb/enzyme/</a>), the list proposed here is instead based on the catalyzed reactions to reach a higher degree of classification in the field of metabolism of xenobiotics.</div><div class="NLM_p">This list appears in <a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a> and is based on extensive works (books and chapters) authored or coauthored by one of the present authors.<a onclick="showRef(event, 'ref14 ref15 ref16 ref17'); return false;" href="javascript:void(0);" class="ref ref14 ref15 ref16 ref17">(14-17)</a> As is customary, metabolic reactions were classified into redox reactions catalyzed by oxidoreductases, hydration/dehydration reactions catalyzed by hydrolases or occurring spontaneously, and conjugations catalyzed by transferases or in a few cases by ligases. These three major categories were divided in turn into second-level categories to take into account the diversity of target sites in substrate molecules. Carbon oxidations, being the most frequent reactions, were divided into Csp<sup>3</sup>- and Csp<sup>2</sup>/Csp-oxidations. These reactions are catalyzed mainly by cytochromes P450 [CYPs], while the molybdo-flavoenzymes (xanthine oxidoreductase [XOR] and aldehyde oxidoreductase [AOR]) play a comparatively modest yet non-negligible role. Redox reactions involving carbonyl compounds as products or substrates (i.e., oxidation of alcohols and aldehydes, reduction of aldehydes and ketones) are catalyzed by dehydrogenases such as alcohol dehydrogenases [ADHs], aldehyde dehydrogenases [ALDHs], and aldo-keto reductases [AKRs]. A variety of less frequently encountered redox reactions, such as reductions at C═C moieties and reductive dehalogenations, are catalyzed by CYPs or various reductases. The oxidation of N-containing functional groups such as tertiary, secondary, and primary amines are oxidized by CYPs and/or by flavin-containing monooxygenases [FMOs]. In contrast, the reduction of these groups involves CYPs or other reductases. Like for N-containing functional groups, oxidation reactions at sulfur-containing groups are catalyzed by CYPs, and to a lesser extent also by FMOs. The role of peroxidases such as myeloperoxidase [MPO] in catalyzing the formation of reactive quinones and quinoneimines is of particular toxicological significance, given the role played by NADPH quinone reductases [NQOs] and glutathione (GSH, see below) in their detoxication.</div><div class="NLM_p">The classification of hydrolytic reactions needs little explanation and involves, e.g., carboxylesterases and amidases. Note however that spontaneous reactions of hydration or dehydration were considered as a metabolic step.</div><div class="NLM_p">The classification of transferases was as follows: Methyltransferases such as catechol <i>O</i>-methyltransferase [COMTs], <i>N</i>-methyltransferases [NMTs], and the thiol methyltransferases [TMTs], sulfotransferases [SULTs], UDP-glucuronosyltransferases [UGTs], glutathione <i>S</i>-transferases [GSTs] plus all enzymes involved in the sequence of reactions leading to the corresponding mercapturic acids: peptidases, <i>N</i>-acetyltransferases [NATs], and C–S cleavage (β-lyase). Other conjugating enzymes are the fatty acyl–coenzyme A ligases [ACSs] and the enzymes catalyzing subsequent reactions (glycine <i>N</i>-acyltransferases [GLYAT] and glutamine <i>N</i>-acyltransferases) which form the corresponding amino acid conjugates, and the various enzymes involved in β-oxidation or 2C-elongation or chiral inversion, and finally other transferases such as phosphotransferases.</div><div class="NLM_p">Some conjugation reactions deserve an additional comment. Thus, the sequence from GSH conjugation to the formation of mercapturic acids and even to thiols (due to β-lyase catalyzed C–S cleavage) was considered as a single pathway because it proved futile to do otherwise. Some reactive metabolites or intermediates may react nonenzymatically with glutathione, a well-known nucleophile. Coupling with endogenous carbonyls to form hydrazones, and CO<sub>2</sub> addition to primary and secondary amines to form carbamic acids, are also known metabolic reactions occurring nonenzymatically.</div><div class="NLM_p last">In the vast majority of reactions, there was no ambiguity in assigning the formation of a given metabolite to a given enzyme (super)family or category (e.g., to CYP, FMO, or dehydrogenases/reductases) because either the enzyme (super)family was determined or no realistic alternative existed. However, a few cases (200 metabolites, i.e., <3% of total metabolites) led to double enzymatic assignment based on experimental evidence.</div></div><div id="sec3_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11">3.2 Content of the MetaQSAR-Based Database</h3><div id="sec3_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i12" class="anchor-spacer"></div><h4 class="article-section__title" id="_i12">3.2.1 Overall Metabolic Statistics</h4><div class="NLM_p">As a preamble, it should be emphasized that the previous study<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> considered all the metabolic reactions regardless of their generation, while the preliminary data reported here are focused only on reactions forming first-generation metabolites. As mentioned above, the screening process involved three primary journals as published in the 2004–2015 years. In this way, 1455 papers were analyzed; they featured a total of 1890 substrates, yielding a total of 4776 metabolic reactions, namely 2.53 first-generation metabolites per substrate. The generation of the 2D/3D structure for the corresponding metabolites is still a work-in-progress aimed at allowing the second-generation metabolites to be compiled. Currently, the database includes the structure of about 50% of the first-generation metabolites. When this work will be completed, the analysis of the second-generation metabolites could begin.</div><div class="NLM_p">When considering the distribution of first-Gen metabolites among the three main classes, the analyzed database has been found to include 3261 (68.3%) products of redox reactions, 507 (10.6%) products of hydrolyses, and 994 (20.8%) conjugates. These relative abundances are in line with those previously reported for the first generation, thus confirming that functionalization reactions play a largely predominant role, which progressively decreases when considering metabolic reactions of the second and third generations as revealed in the previous study.</div><div class="NLM_p"><a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a> schematizes the relative abundance of the 21 reaction classes, revealing that C-oxidations represent more than 50% of all monitored reactions, while redox reactions involving heteroatoms show a markedly minor role. Among the conjugation reactions, glucoronidations represent about the 10% of all collected reactions and almost the 40% when considering the conjugations only, thus emphasizing their very important role, which further increases when considering second- and third-generation metabolites. Finally, the hydrolyses of esters are conceivably more abundant than those of amides, while spontaneous hydrolytic processes represent a modest minority.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01473/20180201/images/medium/jm-2017-01473n_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01473/20180201/images/large/jm-2017-01473n_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Distribution of metabolites according to reaction classes. The percentages shown in the figure refer to 4776 first-generation metabolic reactions = 100%. The color code is as follows: redox reactions, blue; hydrolyses, yellow; conjugations, red. Alternating dark and light fields are used simply for graphical clarity. The reaction classes are reported in numerical order starting from the upper-right quadrant. For clarity, the following very poorly populated classes are not shown: (9) redox reactions of other atoms (0.21%); (13) epoxide hydration (0.04%); (28) other conjugations (1.5%).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01473/20180201/images/large/jm-2017-01473n_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01473&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p"><a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a> compiles the relative abundance as computed for the 101 reaction subclasses and reveals that the most abundant subclass is the oxidation of aryl compounds to epoxides, phenols, or other metabolites followed to a very short distance by hydroxylation (or other oxidations) of Csp<sup>3</sup> carrying an heteroatom. Hydrolyses and conjugations show a more homogeneous distribution among the monitored subclasses, the O-glucuronidation of phenols being the most populated group.</div><div class="NLM_p last">The relative abundance of the reaction classes (and subclasses) is reflected in that of the involved enzymes as reported in <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>, confirming the markedly prevalent role of cytochromes P450 followed by UDP-glucuronosyltransferases.</div></div><div id="sec3_2_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i13" class="anchor-spacer"></div><h4 class="article-section__title" id="_i13">3.2.2 Physicochemical Profiling of the Collected Substrates and Metabolites: An Overview</h4><div class="NLM_p">As mentioned under Methods, a very relevant feature of the MetaQSAR tool is the possibility of combining the database engine with all cheminformatics features implemented in the VEGA ZZ program. With a view to offering a meaningful example of this possibility, all collected substrates and the up-to-now inserted metabolites were analyzed by calculating an extended set of conformer-dependent and conformer-independent structural and physicochemical descriptors such as number of rotors, number of H-bonding groups, PSA, SAS, dipole moment, and log <i>P</i> as computed by the MLP approach.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a><a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a> compares the average values of four representative descriptors as computed for substrates and metabolites and clustered into the three main reaction classes. The analysis was focused on two structural descriptors, namely molecular weight and number of rotors which code for molecular size and flexibility, as well as two physicochemical descriptors, namely log <i>P</i> and PSA (polar surface area), which are related to polarity and H-bonding capacity. The results allow for some major considerations. First, the reported averages show clear differences between the three main reaction classes which, while considering the extent of standard deviations, suggest that, for example, (a) hydrolytic enzymes prefer larger and more flexible substrates, while (b) redox reactions preferentially involve more lipophilic molecules, and (c) conjugation reactions tend to metabolize less lipophilic and more rigid compounds. Second, the differences between substrates and metabolites show contrasting trends, the sole common trend being the lower lipophilicity of metabolites compared to substrates. Third, the standard deviations can be interpreted in two ways: (a) when compared between main reaction classes, they are suggestive of enzyme promiscuity, (b) while when compared between substrates and metabolites they encode the structural changes induced by a given reaction class. Hence, the reported standard deviations reveal that hydrolytic enzymes accommodate substrates with the widest structural heterogeneity while redox and conjugations reactions show a comparably narrower promiscuity. Again, (a) redox reactions do not show clear differences between substrates and metabolites, (b) hydrolytic reactions reduce the structural variability of their substrates, (c) while conjugations enlarge it. This last observation can be explained by considering that redox reactions usually induce small modifications on their substrates, hydrolytic reactions break the substrates releasing two fragments (such as a drug and a promoiety in prodrugs), and conjugations add a novel portion (the endocon) to the substrate structures. While being clearly preliminary, these results reveal meaningful differences between reaction (and consequently enzyme) classes, which invite further investigations and might even have a predictive role.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01473/20180201/images/medium/jm-2017-01473n_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01473/20180201/images/large/jm-2017-01473n_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Physicochemical profiling of substrates and metabolites as subdivided into the three main reaction classes and assessed by four representative descriptors such as molecular weight (MW), number of rotors, polar surface area (PSA), and virtual log <i>P</i> as computed by the MLP approach.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01473/20180201/images/large/jm-2017-01473n_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01473&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec3_2_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i14" class="anchor-spacer"></div><h4 class="article-section__title" id="_i14">3.2.3 Similarity Searches: A Exploratory Study</h4><div class="NLM_p">As explained under Methods, the cheminformatics tools implemented in VEGA ZZ allow targeted searches by structure (and substructure) as well as by molecular properties. To offer a relevant example of this, a set of very recent papers dealing with the metabolism of molecules which are not included into the database (see <a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a> and <a class="ref internalNav" href="#tbl3" aria-label="Table 3">Table 3</a>) were used in similarity analyses to investigate whether the results of such analyses can be used to indirectly predict the metabolism of these novel input molecules, in fact, an external test set. Such an analysis can reveal which role the simple molecular similarity can have in metabolism prediction at least as a starting point for further analyses. Usually, similarity-based predictions have the known problem of providing a non-negligible proportion of false positive results and must be combined with other predictive approaches to filter and prioritize the retrieved results.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> While such filtering procedures may well remain, one may hope that searches based on highly curated databases might limit the number of false positives thus reducing the relevance of postsearch filtering approaches. Clearly, the analysis is based on the assumption that the training set of metabolic data is rich enough to covers a fair proportion of the chemical space of the substrates.</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01473/20180201/images/medium/jm-2017-01473n_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01473/20180201/images/large/jm-2017-01473n_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Structure of the xenobiotics not included in the database and utilized in the exploratory predictive study with the involved atoms and the corresponding classes of the reported metabolic reactions (the blue numbers indicate the correctly predicted metabolic reactions, while the red numbers indicate the unpredicted reactions).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01473/20180201/images/large/jm-2017-01473n_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01473&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Major Results for the Similarity Searches As Performed on 10 Xenobiotics Not Included in the Database, The Metabolic Profile of Which Was Reported in Recent Papers<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="center" /></col><col align="center" /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="center" /></col><col align="center" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">substrate</th><th class="colsep0 rowsep0" align="center">retrieved hits<a class="ref internalNav" href="#tbl3-fn1" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center">retrieved reactions</th><th class="colsep0 rowsep0" align="center">three most similar molecules</th><th class="colsep0 rowsep0" align="center" char=".">reported metabolites</th><th class="colsep0 rowsep0" align="center" char=".">found metabolites</th><th class="colsep0 rowsep0" align="center" char=".">missed metabolites</th><th class="colsep0 rowsep0" align="center">false positives</th><th class="colsep0 rowsep0" align="center">ref</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">cobimetinib</td><td class="colsep0 rowsep0" align="center">7 (0.54)</td><td class="colsep0 rowsep0" align="center">13</td><td class="colsep0 rowsep0" align="left">ezetimibe, ref <a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">20</a> compd <b>3</b>, ref <a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">20</a> compd <b>7</b></td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="char" char=".">2</td><td class="colsep0 rowsep0" align="char" char=".">2</td><td class="colsep0 rowsep0" align="center">0</td><td class="colsep0 rowsep0" align="center"><a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">21</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">AZD1</td><td class="colsep0 rowsep0" align="center">7 (0.59)</td><td class="colsep0 rowsep0" align="center">39</td><td class="colsep0 rowsep0" align="left">PF-04937319 2-(3-chlorobenzyloxy)-6-(piperazinyl)pyrazine, BILR 355</td><td class="colsep0 rowsep0" align="char" char=".">5</td><td class="colsep0 rowsep0" align="char" char=".">5</td><td class="colsep0 rowsep0" align="char" char=".">0</td><td class="colsep0 rowsep0" align="center">3(2.01 + 2.02 + 6.1)</td><td class="colsep0 rowsep0" align="center"><a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">22</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">atomoxetine</td><td class="colsep0 rowsep0" align="center">164 (0.77)</td><td class="colsep0 rowsep0" align="center">497</td><td class="colsep0 rowsep0" align="left">fluoxetine, ref <a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">23</a> compd <b>2A</b>, ref <a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">23</a> compd <b>2L</b></td><td class="colsep0 rowsep0" align="char" char=".">3</td><td class="colsep0 rowsep0" align="char" char=".">2</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="center">1 (6.01)</td><td class="colsep0 rowsep0" align="center"><a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">24</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">osimertinib</td><td class="colsep0 rowsep0" align="center">13 (0.64)</td><td class="colsep0 rowsep0" align="center">42</td><td class="colsep0 rowsep0" align="left">imatinib, afatinib, flumatinib</td><td class="colsep0 rowsep0" align="char" char=".">10</td><td class="colsep0 rowsep0" align="char" char=".">9</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="center">3(2.3 + 4.4 + 12.2)</td><td class="colsep0 rowsep0" align="center"><a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">25</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">fimasartan</td><td class="colsep0 rowsep0" align="center">4 (0.55)</td><td class="colsep0 rowsep0" align="center">12</td><td class="colsep0 rowsep0" align="left">TBZ-30878, AMG-657417, palisperidone</td><td class="colsep0 rowsep0" align="char" char=".">9</td><td class="colsep0 rowsep0" align="char" char=".">7</td><td class="colsep0 rowsep0" align="char" char=".">2</td><td class="colsep0 rowsep0" align="center">2 (6.8 + 6.5)</td><td class="colsep0 rowsep0" align="center"><a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">26</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">EPZ015666</td><td class="colsep0 rowsep0" align="center">10 (0.56)</td><td class="colsep0 rowsep0" align="center">40</td><td class="colsep0 rowsep0" align="left">saquinavir, methotrexate, PH-302</td><td class="colsep0 rowsep0" align="char" char=".">2</td><td class="colsep0 rowsep0" align="char" char=".">2</td><td class="colsep0 rowsep0" align="char" char=".">0</td><td class="colsep0 rowsep0" align="center">2 (2.1 + 12.1)</td><td class="colsep0 rowsep0" align="center"><a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">27</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">bavachinin</td><td class="colsep0 rowsep0" align="center">154 (1)</td><td class="colsep0 rowsep0" align="center">414</td><td class="colsep0 rowsep0" align="left">8-prenylnaringenin, isoxanthohumol, liquiritigenin</td><td class="colsep0 rowsep0" align="char" char=".">8</td><td class="colsep0 rowsep0" align="char" char=".">8</td><td class="colsep0 rowsep0" align="char" char=".">0</td><td class="colsep0 rowsep0" align="center">1 (7.4)</td><td class="colsep0 rowsep0" align="center"><a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">28</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">nefopam</td><td class="colsep0 rowsep0" align="center">22 (0.86)</td><td class="colsep0 rowsep0" align="center">64</td><td class="colsep0 rowsep0" align="left">diphenhydramine, deramciclane, citalopram</td><td class="colsep0 rowsep0" align="char" char=".">6</td><td class="colsep0 rowsep0" align="char" char=".">6</td><td class="colsep0 rowsep0" align="char" char=".">0</td><td class="colsep0 rowsep0" align="center">1 (1.1)</td><td class="colsep0 rowsep0" align="center"><a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">29</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">omacetaxine</td><td class="colsep0 rowsep0" align="center">53 (0.65)</td><td class="colsep0 rowsep0" align="center">140</td><td class="colsep0 rowsep0" align="left">heroin, codeine, morphine</td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="char" char=".">3</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="center">1 (21.1)</td><td class="colsep0 rowsep0" align="center"><a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">30</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">total</td><td class="colsep0 rowsep0" align="center">436</td><td class="colsep0 rowsep0" align="center">1226</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="char" char=".">51</td><td class="colsep0 rowsep0" align="char" char=".">44</td><td class="colsep0 rowsep0" align="char" char=".">7</td><td class="colsep0 rowsep0" align="center">14</td><td class="colsep0 rowsep0" align="center"> </td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup>a</sup><p class="last">Retrieved hits and retrieved reactions refer to the substrates and corresponding metabolic reactions obtained by similarity searches on the collected database. The similarity searches were focused on compounds with Tanimoto distance greater than 0.5, and the predictions were performed by considering the three most similar retrieved molecules. Reported metabolites includes the number of the first generation metabolites reported for the novel input molecules as described in the corresponding reference papers as indicated in the last column. Found metabolites indicate the number of correctly predicted metabolites. Missed metabolites report the number of the unpredicted metabolites, while false positives include the number and the classes of the metabolites suggested by the reactions retrieved from the database but not reported in the corresponding experimental papers.</p></div><div class="footnote" id="tbl3-fn1"><sup>b</sup><p class="last">In parentheses, the maximum Tanimoto distance is reported.</p></div></div></div><div class="NLM_p">While considering its tentative nature, this analysis was performed by following a set of well-defined rules which can be summarized as follows: (a) the analysis was limited to the first-generation metabolites, (b) only compounds having a Tanimoto distance based on the Bingo fingerprints greater than 0.5 were considered, (c) the analysis was extended to the reactions reported for the three most similar compounds. <a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a> and <a class="ref internalNav" href="#tbl3" aria-label="Table 3">Table 3</a> summarizes the results and allows for some meaningful considerations. Of the 10 considered molecules, only for one very complex molecule (dasabuvir), the search was unable to find a similar molecule within the database because the maximum Tanimoto distance obtained (0.47) was below the assumed threshold. This underlines that the database conveniently covers the chemical space of xenobiotics and suggests that the search criteria should be tuned according to the chemical complexity of the input molecule. Nevertheless, such a calibration analysis goes beyond the scope of this exploratory study and thus dasabuvir was discarded here. Remarkably, the retrieved reactions are in agreement with 44 out of the 51 first-generation metabolites as reported for the nine input molecules. Only seven metabolites were not predictable; the false positives are very few and only in two cases they are more than two. This encouraging result emphasizes the crucial role of the number of retrieved compounds on which the analysis is focused which should be carefully chosen to limit the number of false positives. While considering the exploratory nature of this analysis, its results reveal that metabolism prediction based solely on molecular similarity and based on the present database possesses an encouraging predictive power showing a sensitivity equal to 0.86, a precision of 0.76, and a F1 score equal to 0.80.</div><div class="NLM_p">When considering the classes of the unpredicted reactions, one may note that the higher number of wrong predictions involves metabolic redox reactions and, in detail, the aryl hydroxylations appear to be the most problematic reactions with three unpredicted metabolites. If the relative abundance of missed redox metabolites might simply reflect their greater frequency in metabolism, the high number of unpredicted aryl hydroxylations can be due to the greater complexity of the aromatic systems which are captured with difficulty by similarity searches.</div><div class="NLM_p last">It should be emphasized that such a similarity analysis predicts the reaction classes an input substrate can undergo rather than structurally fully determined metabolites. Because several structurally compatible metabolites (e.g., regioisomers) may usually be generated by a predicted reaction class, such a similarity search should be combined with other computational approaches able to predict target sites of metabolism (soft spots) so prioritizing the putative metabolites derived from similarity analyses.</div></div></div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i16">4 Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_88188" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_88188" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">This study describes a database engine specifically designed to collect and manage metabolism data, which can be freely downloaded at <a href="http://www.vegazz.net" class="extLink">www.vegazz.net</a>. The tool is embedded in the VEGA ZZ suite of programs and thus can take advantage of all cheminformatics features implemented in the software. Along with an updated evaluation of the statistical relevance of the monitored reaction classes, the study includes two promising examples of the cheminformatics potentials of the database engine described here. In the first example, the physicochemical profile of the collected molecules was investigated, revealing significant differences between the substrates and metabolites as grouped into the three main classes. The second example emphasizes the encouraging role of similarity-based searches which prove successful in predicting the metabolic reactions of a set of recently published molecules not yet included in the database. Finally, an exhaustive classification of the metabolic reactions is presented which may find countless applications in metabolism analysis.</div></div><div class="NLM_back"><div class="article_supporting-info" id=""></div><div class="testing" data-doi="10.1021/acs.jmedchem.7b01473" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_30401" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_30401" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Alessandro Pedretti</span> - <span class="hlFld-Affiliation affiliation">Dipartimento
di Scienze Farmaceutiche “Pietro Pratesi”, Facoltà
di Farmacia, Università degli Studi
di Milano, Via Luigi Mangiagalli 25, I-20133 Milano, Italy</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-5916-2029" title="Orcid link">http://orcid.org/0000-0001-5916-2029</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#e889848d9b9b89868c9a87c6988d8c9a8d9c9c81a89d86818581c6819c"><span class="__cf_email__" data-cfemail="e7868b8294948689839588c9978283958293938ea792898e8a8ec98e93">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Angelica Mazzolari</span> - <span class="hlFld-Affiliation affiliation">Dipartimento
di Scienze Farmaceutiche “Pietro Pratesi”, Facoltà
di Farmacia, Università degli Studi
di Milano, Via Luigi Mangiagalli 25, I-20133 Milano, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Giulio Vistoli</span> - <span class="hlFld-Affiliation affiliation">Dipartimento
di Scienze Farmaceutiche “Pietro Pratesi”, Facoltà
di Farmacia, Università degli Studi
di Milano, Via Luigi Mangiagalli 25, I-20133 Milano, Italy</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-3939-5172" title="Orcid link">http://orcid.org/0000-0002-3939-5172</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Bernard Testa</span> - <span class="hlFld-Affiliation affiliation">University
of Lausanne, 1015 Lausanne, Switzerland</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>The manuscript was written through contributions of all authors. All authors have given approval to the final version of the manuscript.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br><sup>§</sup>B.T.: Emeritus Professor.<br /></br></div></li></ul></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i19" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i19"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i20" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i20"> Abbreviations Used</h2><tr><td class="NLM_term">ACS</td><td class="NLM_def"><p class="first last">acyl-coenzyme A ligase</p></td></tr><tr><td class="NLM_term">ADH</td><td class="NLM_def"><p class="first last">alcohol dehydrogenase</p></td></tr><tr><td class="NLM_term">AKR</td><td class="NLM_def"><p class="first last">aldo-keto reductase</p></td></tr><tr><td class="NLM_term">ALDH</td><td class="NLM_def"><p class="first last">aldehyde dehydrogenase</p></td></tr><tr><td class="NLM_term">AOR</td><td class="NLM_def"><p class="first last">aldehyde oxidoreductase</p></td></tr><tr><td class="NLM_term">COMT</td><td class="NLM_def"><p class="first last">catechol <i>O</i>-methyltransferase</p></td></tr><tr><td class="NLM_term">CYP</td><td class="NLM_def"><p class="first last">cytochrome P450</p></td></tr><tr><td class="NLM_term">FMO</td><td class="NLM_def"><p class="first last">flavin-containing monooxygenase</p></td></tr><tr><td class="NLM_term">GLYAT</td><td class="NLM_def"><p class="first last">glycine <i>N</i>-acyltransferase</p></td></tr><tr><td class="NLM_term">GSH</td><td class="NLM_def"><p class="first last">glutathione</p></td></tr><tr><td class="NLM_term">GUI</td><td class="NLM_def"><p class="first last">graphic user interface</p></td></tr><tr><td class="NLM_term">MLP</td><td class="NLM_def"><p class="first last">molecular lipophilicity potential</p></td></tr><tr><td class="NLM_term">MPO</td><td class="NLM_def"><p class="first last">myeloperoxidase</p></td></tr><tr><td class="NLM_term">NAT</td><td class="NLM_def"><p class="first last"><i>N</i>-acetyltransferase</p></td></tr><tr><td class="NLM_term">NMT</td><td class="NLM_def"><p class="first last"><i>N</i>-methyltransferase</p></td></tr><tr><td class="NLM_term">NQO</td><td class="NLM_def"><p class="first last">NADPH quinone reductase</p></td></tr><tr><td class="NLM_term">PSA</td><td class="NLM_def"><p class="first last">polar surface area</p></td></tr><tr><td class="NLM_term">SAS</td><td class="NLM_def"><p class="first last">solvent accessible surface</p></td></tr><tr><td class="NLM_term">SULT</td><td class="NLM_def"><p class="first last">sulfotransferase</p></td></tr><tr><td class="NLM_term">TMT</td><td class="NLM_def"><p class="first last">thiol methyltransferase</p></td></tr><tr><td class="NLM_term">UGT</td><td class="NLM_def"><p class="first last">UDP-glucuronosyltransferase</p></td></tr><tr><td class="NLM_term">XOR</td><td class="NLM_def"><p class="first last">xanthine oxidoreductase</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i21">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_35762" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_35762" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 30 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Aslam, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Basit, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nisar, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Khurshid, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rasool, M. H.</span><span> </span><span class="NLM_article-title">Proteomics: Technologies and Their Applications</span> <span class="citation_source-journal">J. Chromatogr. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">182</span><span class="NLM_x">–</span> <span class="NLM_lpage">196</span><span class="refDoi"> DOI: 10.1093/chromsci/bmw167</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01473&amp;key=10.1093%2Fchromsci%2Fbmw167" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01473&amp;key=28087761" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01473&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsFaktbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2017&pages=182-196&author=B.+Aslamauthor=M.+Basitauthor=M.+A.+Nisarauthor=M.+Khurshidauthor=M.+H.+Rasool&title=Proteomics%3A+Technologies+and+Their+Applications&doi=10.1093%2Fchromsci%2Fbmw167"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Proteomics: technologies and their applications</span></div><div class="casAuthors">Aslam, Bilal; Basit, Madiha; Nisar, Muhammad Atif; Khurshid, Mohsin; Rasool, Muhammad Hidayat</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chromatographic Science</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">182-196</span>CODEN:
                <span class="NLM_cas:coden">JCHSBZ</span>;
        ISSN:<span class="NLM_cas:issn">1945-239X</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Proteomics involves the applications of technologies for the identification and quantification of overall proteins present content of a cell, tissue or an organism.  It supplements the other "omics" technologies such as genomic and transcriptomics to expound the identity of proteins of an organism, and to cognize the structure and functions of a particular protein.  Proteomics-based technologies are utilized in various capacities for different research settings such as detection of various diagnostic markers, candidates for vaccine prodn., understanding pathogenicity mechanisms, alteration of expression patterns in response to different signals and interpretation of functional protein pathways in different diseases.  Proteomics is practically intricate because it includes the anal. and categorization of overall protein signatures of a genome.  Mass spectrometry with LC-MS-MS and MALDI-TOF/TOF being widely used equipment is the central among current proteomics.  However, utilization of proteomics facilities including the software for equipment, databases and the requirement of skilled personnel substantially increase the costs, therefore limit their wider use esp. in the developing world.  Furthermore, the proteome is highly dynamic because of complex regulatory systems that control the expression levels of proteins.  This review efforts to describe the various proteomics approaches, the recent developments and their application in research and anal.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoL--Tuk4G697Vg90H21EOLACvtfcHk0lh_HGgi_V0jMQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsFaktbk%253D&md5=0d69fa2515dfd694c147631d1c10c346</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1093%2Fchromsci%2Fbmw167&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fchromsci%252Fbmw167%26sid%3Dliteratum%253Aachs%26aulast%3DAslam%26aufirst%3DB.%26aulast%3DBasit%26aufirst%3DM.%26aulast%3DNisar%26aufirst%3DM.%2BA.%26aulast%3DKhurshid%26aufirst%3DM.%26aulast%3DRasool%26aufirst%3DM.%2BH.%26atitle%3DProteomics%253A%2520Technologies%2520and%2520Their%2520Applications%26jtitle%3DJ.%2520Chromatogr.%2520Sci.%26date%3D2017%26volume%3D55%26spage%3D182%26epage%3D196%26doi%3D10.1093%2Fchromsci%2Fbmw167" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Bourne, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lorsch, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Green, E. D.</span><span> </span><span class="NLM_article-title">Perspective: Sustaining the Big-Data Ecosystem</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">527</span><span class="NLM_x">, </span> <span class="NLM_fpage">S16</span><span class="NLM_x">–</span> <span class="NLM_lpage">S17</span><span class="refDoi"> DOI: 10.1038/527S16a</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01473&amp;key=10.1038%2F527S16a" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=527&publication_year=2015&pages=S16-S17&author=P.+E.+Bourneauthor=J.+R.+Lorschauthor=E.+D.+Green&title=Perspective%3A+Sustaining+the+Big-Data+Ecosystem&doi=10.1038%2F527S16a"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2F527S16a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F527S16a%26sid%3Dliteratum%253Aachs%26aulast%3DBourne%26aufirst%3DP.%2BE.%26aulast%3DLorsch%26aufirst%3DJ.%2BR.%26aulast%3DGreen%26aufirst%3DE.%2BD.%26atitle%3DPerspective%253A%2520Sustaining%2520the%2520Big-Data%2520Ecosystem%26jtitle%3DNature%26date%3D2015%26volume%3D527%26spage%3DS16%26epage%3DS17%26doi%3D10.1038%2F527S16a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Coveney, P. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dougherty, E. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Highfield, R. R.</span><span> </span><span class="NLM_article-title">Big Data Need Big Theory Too</span> <span class="citation_source-journal">Philos. Trans. R. Soc., A</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">374</span><span class="NLM_x"> (</span><span class="NLM_issue">2080</span><span class="NLM_x">) </span> <span class="NLM_fpage">20160153</span><span class="refDoi"> DOI: 10.1098/rsta.2016.0153</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01473&amp;key=10.1098%2Frsta.2016.0153" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=374&publication_year=2016&pages=20160153&issue=2080&author=P.+V.+Coveneyauthor=E.+R.+Doughertyauthor=R.+R.+Highfield&title=Big+Data+Need+Big+Theory+Too&doi=10.1098%2Frsta.2016.0153"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1098%2Frsta.2016.0153&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1098%252Frsta.2016.0153%26sid%3Dliteratum%253Aachs%26aulast%3DCoveney%26aufirst%3DP.%2BV.%26aulast%3DDougherty%26aufirst%3DE.%2BR.%26aulast%3DHighfield%26aufirst%3DR.%2BR.%26atitle%3DBig%2520Data%2520Need%2520Big%2520Theory%2520Too%26jtitle%3DPhilos.%2520Trans.%2520R.%2520Soc.%252C%2520A%26date%3D2016%26volume%3D374%26issue%3D2080%26spage%3D20160153%26doi%3D10.1098%2Frsta.2016.0153" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Bowling, F. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thomas, M.</span><span> </span><span class="NLM_article-title">Analyzing the Metabolome</span> <span class="citation_source-journal">Methods Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">1168</span><span class="NLM_x">, </span> <span class="NLM_fpage">31</span><span class="NLM_x">–</span> <span class="NLM_lpage">45</span><span class="refDoi"> DOI: 10.1007/978-1-4939-0847-9_3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01473&amp;key=10.1007%2F978-1-4939-0847-9_3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01473&amp;key=24870129" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01473&amp;key=1%3ACAS%3A528%3ADC%252BC2MXotVWku7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1168&publication_year=2014&pages=31-45&author=F.+G.+Bowlingauthor=M.+Thomas&title=Analyzing+the+Metabolome&doi=10.1007%2F978-1-4939-0847-9_3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Analyzing the Metabolome</span></div><div class="casAuthors">Bowling, Francis G.; Thomas, Mervyn</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Molecular Biology (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">1168</span>
        (<span class="NLM_cas:issue">Clinical Bioinformatics (2nd Edition)</span>),
    <span class="NLM_cas:pages">31-45</span>CODEN:
                <span class="NLM_cas:coden">MMBIED</span>;
        ISSN:<span class="NLM_cas:issn">1940-6029</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Metabolites, the chem. entities that are transformed during metab., provide a functional readout of cellular biochem. that offers the best prediction of the phenotype and the nature of a disease.  Mass spectrometry now allows thousands of metabolites to be quantitated.  The targeted or untargeted data from metabolic profiling can be combined with either supervised or unsupervised approaches to improve interpretation.  These sophisticated statistical techniques are computationally intensive.  This chapter reviews techniques applicable to metabolomics approaches to disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmzek7a2knh7Vg90H21EOLACvtfcHk0lh_HGgi_V0jMQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXotVWku7k%253D&md5=580348e236d0d39a35e20065d39b6028</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1007%2F978-1-4939-0847-9_3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-1-4939-0847-9_3%26sid%3Dliteratum%253Aachs%26aulast%3DBowling%26aufirst%3DF.%2BG.%26aulast%3DThomas%26aufirst%3DM.%26atitle%3DAnalyzing%2520the%2520Metabolome%26jtitle%3DMethods%2520Mol.%2520Biol.%26date%3D2014%26volume%3D1168%26spage%3D31%26epage%3D45%26doi%3D10.1007%2F978-1-4939-0847-9_3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Minkiewicz, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Darewicz, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iwaniak, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bucholska, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Starowicz, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Czyrko, E.</span><span> </span><span class="NLM_article-title">Internet Databases of the Properties, Enzymatic Reactions, and Metabolism of Small Molecules-Search Options and Applications in Food Science</span> <span class="citation_source-journal">Int. J. Mol. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x"> (</span><span class="NLM_issue">12</span><span class="NLM_x">) </span> <span class="NLM_fpage">2039</span><span class="NLM_x">–</span> <span class="NLM_lpage">2063</span><span class="refDoi"> DOI: 10.3390/ijms17122039</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01473&amp;key=10.3390%2Fijms17122039" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01473&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsFWntbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2016&pages=2039-2063&issue=12&author=P.+Minkiewiczauthor=M.+Darewiczauthor=A.+Iwaniakauthor=J.+Bucholskaauthor=P.+Starowiczauthor=E.+Czyrko&title=Internet+Databases+of+the+Properties%2C+Enzymatic+Reactions%2C+and+Metabolism+of+Small+Molecules-Search+Options+and+Applications+in+Food+Science&doi=10.3390%2Fijms17122039"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Internet databases of the properties, enzymatic reactions, and metabolism of small molecules-search options and applications in food science</span></div><div class="casAuthors">Minkiewicz, Piotr; Darewicz, Malgorzata; Iwaniak, Anna; Bucholska, Justyna; Starowicz, Piotr; Czyrko, Emilia</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Molecular Sciences</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2039/1-2039/24</span>CODEN:
                <span class="NLM_cas:coden">IJMCFK</span>;
        ISSN:<span class="NLM_cas:issn">1422-0067</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Internet databases of small mols., their enzymic reactions, and metab. have emerged as useful tools in food science.  Database searching is also introduced as part of chem. or enzymol. courses for food technol. students.  Such resources support the search for information about single compds. and facilitate the introduction of secondary analyses of large datasets.  Information can be retrieved from databases by searching for the compd. name or structure, annotating with the help of chem. codes or drawn using mol. editing software.  Data mining options may be enhanced by navigating through a network of links and cross-links between databases.  Exemplary databases reviewed in this article belong to two classes: tools concerning small mols. (including general and specialized databases annotating food components) and tools annotating enzymes and metab.  Some problems assocd. with database application are also discussed.  Data summarized in computer databases may be used for calcn. of daily intake of bioactive compds., prediction of metab. of food components, and their biol. activity as well as for prediction of interactions between food component and drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3vL4ql-iOArVg90H21EOLACvtfcHk0lh_HGgi_V0jMQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsFWntbc%253D&md5=d1423d8e85bb9f1dd5cd1a10982c5f7d</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.3390%2Fijms17122039&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fijms17122039%26sid%3Dliteratum%253Aachs%26aulast%3DMinkiewicz%26aufirst%3DP.%26aulast%3DDarewicz%26aufirst%3DM.%26aulast%3DIwaniak%26aufirst%3DA.%26aulast%3DBucholska%26aufirst%3DJ.%26aulast%3DStarowicz%26aufirst%3DP.%26aulast%3DCzyrko%26aufirst%3DE.%26atitle%3DInternet%2520Databases%2520of%2520the%2520Properties%252C%2520Enzymatic%2520Reactions%252C%2520and%2520Metabolism%2520of%2520Small%2520Molecules-Search%2520Options%2520and%2520Applications%2520in%2520Food%2520Science%26jtitle%3DInt.%2520J.%2520Mol.%2520Sci.%26date%3D2016%26volume%3D17%26issue%3D12%26spage%3D2039%26epage%3D2063%26doi%3D10.3390%2Fijms17122039" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Warr, W. A.</span><span> </span><span class="NLM_article-title">A Short Review of Chemical Reaction Database Systems, Computer-Aided Synthesis Design, Reaction Prediction and Synthetic Feasibility</span> <span class="citation_source-journal">Mol. Inf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">33</span><span class="NLM_x">, </span> <span class="NLM_fpage">469</span><span class="NLM_x">–</span> <span class="NLM_lpage">476</span><span class="refDoi"> DOI: 10.1002/minf.201400052</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01473&amp;key=10.1002%2Fminf.201400052" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01473&amp;key=27485985" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01473&amp;key=1%3ACAS%3A528%3ADC%252BC2cXovFCmur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2014&pages=469-476&author=W.+A.+Warr&title=A+Short+Review+of+Chemical+Reaction+Database+Systems%2C+Computer-Aided+Synthesis+Design%2C+Reaction+Prediction+and+Synthetic+Feasibility&doi=10.1002%2Fminf.201400052"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">A Short Review of Chemical Reaction Database Systems, Computer-Aided Synthesis Design, Reaction Prediction and Synthetic Feasibility</span></div><div class="casAuthors">Warr, Wendy A.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Informatics</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">6-7</span>),
    <span class="NLM_cas:pages">469-476</span>CODEN:
                <span class="NLM_cas:coden">MIONBS</span>;
        ISSN:<span class="NLM_cas:issn">1868-1743</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">This article is the text for a pedagogical lecture to be given at the Strasbourg Summer School in Chemoinformatics in June 2104.  It covers a very wide range of reaction topics including structure and reaction representation, reaction centers, atom-to-atom mapping, reaction retrieval systems, computer-aided synthesis design, retrosynthesis, reaction prediction and synthetic feasibility.  In the time available, the coverage of each topic can only be cursory; the main usefulness of this article to the research community is the extensive bibliog.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQWeDBafb6nrVg90H21EOLACvtfcHk0liV1nF3MYWJ0g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXovFCmur8%253D&md5=392354d1d71f1ad31f1087d56b9afd57</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1002%2Fminf.201400052&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fminf.201400052%26sid%3Dliteratum%253Aachs%26aulast%3DWarr%26aufirst%3DW.%2BA.%26atitle%3DA%2520Short%2520Review%2520of%2520Chemical%2520Reaction%2520Database%2520Systems%252C%2520Computer-Aided%2520Synthesis%2520Design%252C%2520Reaction%2520Prediction%2520and%2520Synthetic%2520Feasibility%26jtitle%3DMol.%2520Inf.%26date%3D2014%26volume%3D33%26spage%3D469%26epage%3D476%26doi%3D10.1002%2Fminf.201400052" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Caspi, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dreher, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karp, P. D.</span><span> </span><span class="NLM_article-title">The Challenge of Constructing, Classifying, and Representing Metabolic Pathways</span> <span class="citation_source-journal">FEMS Microbiol. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">345</span><span class="NLM_x">, </span> <span class="NLM_fpage">85</span><span class="NLM_x">–</span> <span class="NLM_lpage">93</span><span class="refDoi"> DOI: 10.1111/1574-6968.12194</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01473&amp;key=10.1111%2F1574-6968.12194" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=345&publication_year=2013&pages=85-93&author=R.+Caspiauthor=K.+Dreherauthor=P.+D.+Karp&title=The+Challenge+of+Constructing%2C+Classifying%2C+and+Representing+Metabolic+Pathways&doi=10.1111%2F1574-6968.12194"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1111%2F1574-6968.12194&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252F1574-6968.12194%26sid%3Dliteratum%253Aachs%26aulast%3DCaspi%26aufirst%3DR.%26aulast%3DDreher%26aufirst%3DK.%26aulast%3DKarp%26aufirst%3DP.%2BD.%26atitle%3DThe%2520Challenge%2520of%2520Constructing%252C%2520Classifying%252C%2520and%2520Representing%2520Metabolic%2520Pathways%26jtitle%3DFEMS%2520Microbiol.%2520Lett.%26date%3D2013%26volume%3D345%26spage%3D85%26epage%3D93%26doi%3D10.1111%2F1574-6968.12194" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Gasteiger, J.</span><span> </span><span class="NLM_article-title">Explorations into Chemical Reactions and Biochemical Pathways</span> <span class="citation_source-journal">Mol. Inf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">35</span><span class="NLM_x">, </span> <span class="NLM_fpage">588</span><span class="NLM_x">–</span> <span class="NLM_lpage">592</span><span class="refDoi"> DOI: 10.1002/minf.201600038</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01473&amp;key=10.1002%2Fminf.201600038" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01473&amp;key=27870242" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01473&amp;key=1%3ACAS%3A528%3ADC%252BC28Xns1ejtb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2016&pages=588-592&author=J.+Gasteiger&title=Explorations+into+Chemical+Reactions+and+Biochemical+Pathways&doi=10.1002%2Fminf.201600038"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Explorations into Chemical Reactions and Biochemical Pathways</span></div><div class="casAuthors">Gasteiger, Johann</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Informatics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">11-12</span>),
    <span class="NLM_cas:pages">588-592</span>CODEN:
                <span class="NLM_cas:coden">MIONBS</span>;
        ISSN:<span class="NLM_cas:issn">1868-1743</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A brief overview of the work in the research group of the present author on extg. knowledge from chem. reaction data is presented.  Methods have been developed to calc. physicochem. effects at the reaction site.  It is shown that these physicochem. effects can quite favorably be used to derive equations for the calcn. of data on gas phase reactions and on reactions in soln. such as aq. acidity of alcs. or carboxylic acids or the hydrolysis of amides.  Furthermore, it is shown that these physicochem. effects are quite effective for assigning reactions into reaction classes that correspond to chem. knowledge.  Biochem. reactions constitute a particularly interesting and challenging task for increasing our understanding of living species.  The BioPath. Database is a rich source of information on biochem. reactions and has been used for a variety of applications of chem., biol., or medicinal interests.  Thus, it was shown that biochem. reactions can be assigned by the physicochem. effects into classes that correspond to the classification of enzymes by the EC nos.  Furthermore, 3D models of reaction intermediates can be used for searching for novel enzyme inhibitors.  It was shown in a combined application of chemoinformatics and bioinformatics that essential pathways of diseases can be uncovered.  Furthermore, a study showed that bacterial flavor-forming pathways can be discovered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoezOUVwDEas7Vg90H21EOLACvtfcHk0liV1nF3MYWJ0g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xns1ejtb0%253D&md5=fcecba91b22a5a120f63ad70ab3b15c8</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1002%2Fminf.201600038&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fminf.201600038%26sid%3Dliteratum%253Aachs%26aulast%3DGasteiger%26aufirst%3DJ.%26atitle%3DExplorations%2520into%2520Chemical%2520Reactions%2520and%2520Biochemical%2520Pathways%26jtitle%3DMol.%2520Inf.%26date%3D2016%26volume%3D35%26spage%3D588%26epage%3D592%26doi%3D10.1002%2Fminf.201600038" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Karp, P. D.</span><span> </span><span class="NLM_article-title">Can We Replace Curation with Information Extraction Software?</span> <span class="citation_source-journal">Database</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">2016</span><span class="NLM_x">, </span> <span class="NLM_fpage">baw150</span><span class="refDoi"> DOI: 10.1093/database/baw150</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01473&amp;key=10.1093%2Fdatabase%2Fbaw150" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01473&amp;key=28025341" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2016&publication_year=2016&pages=baw150&author=P.+D.+Karp&title=Can+We+Replace+Curation+with+Information+Extraction+Software%3F&doi=10.1093%2Fdatabase%2Fbaw150"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1093%2Fdatabase%2Fbaw150&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fdatabase%252Fbaw150%26sid%3Dliteratum%253Aachs%26aulast%3DKarp%26aufirst%3DP.%2BD.%26atitle%3DCan%2520We%2520Replace%2520Curation%2520with%2520Information%2520Extraction%2520Software%253F%26jtitle%3DDatabase%26date%3D2016%26volume%3D2016%26spage%3Dbaw150%26doi%3D10.1093%2Fdatabase%2Fbaw150" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Hadadi, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hafner, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shajkofci, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zisaki, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hatzimanikatis, V.</span><span> </span><span class="NLM_article-title">ATLAS of Biochemistry: A Repository of All Possible Biochemical Reactions for Synthetic Biology and Metabolic Engineering Studies</span> <span class="citation_source-journal">ACS Synth. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">1155</span><span class="NLM_x">–</span> <span class="NLM_lpage">1166</span><span class="refDoi"> DOI: 10.1021/acssynbio.6b00054</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acssynbio.6b00054" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01473&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFGjs7vL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2016&pages=1155-1166&author=N.+Hadadiauthor=J.+Hafnerauthor=A.+Shajkofciauthor=A.+Zisakiauthor=V.+Hatzimanikatis&title=ATLAS+of+Biochemistry%3A+A+Repository+of+All+Possible+Biochemical+Reactions+for+Synthetic+Biology+and+Metabolic+Engineering+Studies&doi=10.1021%2Facssynbio.6b00054"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">ATLAS of Biochemistry: A Repository of All Possible Biochemical Reactions for Synthetic Biology and Metabolic Engineering Studies</span></div><div class="casAuthors">Hadadi, Noushin; Hafner, Jasmin; Shajkofci, Adrian; Zisaki, Aikaterini; Hatzimanikatis, Vassily</div><div class="citationInfo"><span class="NLM_cas:title">ACS Synthetic Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1155-1166</span>CODEN:
                <span class="NLM_cas:coden">ASBCD6</span>;
        ISSN:<span class="NLM_cas:issn">2161-5063</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Because the complexity of metab. cannot be intuitively understood or analyzed, computational methods are indispensable for studying biochem. and deepening our understanding of cellular metab. to promote new discoveries.  We used the computational framework BNICE.ch along with cheminformatic tools to assemble the whole theor. reactome from the known metabolome through expansion of the known biochem. presented in the Kyoto Encyclopedia of Genes and Genomes (KEGG) database.  We constructed the ATLAS of Biochem., a database of all theor. biochem. reactions based on known biochem. principles and compds.  ATLAS includes more than 130,000 hypothetical enzymic reactions that connect two or more KEGG metabolites through novel enzymic reactions that have never been reported to occur in living organisms.  Moreover, ATLAS reactions integrate 42% of KEGG metabolites that are not currently present in any KEGG reaction into one or more novel enzymic reactions.  The generated repository of information is organized in a Web-based database (http://lcsb-databases.epfl.ch/atlas/) that allows the user to search for all possible routes from any substrate compd. to any product.  The resulting pathways involve known and novel enzymic steps that may indicate unidentified enzymic activities and provide potential targets for protein engineering.  Our approach of introducing novel biochem. into pathway design and assocd. databases will be important for synthetic biol. and metabolic engineering.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxN_YUbnQ_9rVg90H21EOLACvtfcHk0lgO_o7QsPqPqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFGjs7vL&md5=faf6d7855e475d2d30bf3544cac87420</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Facssynbio.6b00054&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facssynbio.6b00054%26sid%3Dliteratum%253Aachs%26aulast%3DHadadi%26aufirst%3DN.%26aulast%3DHafner%26aufirst%3DJ.%26aulast%3DShajkofci%26aufirst%3DA.%26aulast%3DZisaki%26aufirst%3DA.%26aulast%3DHatzimanikatis%26aufirst%3DV.%26atitle%3DATLAS%2520of%2520Biochemistry%253A%2520A%2520Repository%2520of%2520All%2520Possible%2520Biochemical%2520Reactions%2520for%2520Synthetic%2520Biology%2520and%2520Metabolic%2520Engineering%2520Studies%26jtitle%3DACS%2520Synth.%2520Biol.%26date%3D2016%26volume%3D5%26spage%3D1155%26epage%3D1166%26doi%3D10.1021%2Facssynbio.6b00054" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Pedretti, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Villa, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vistoli, G.</span><span> </span><span class="NLM_article-title">VEGA--An Open Platform to Develop Chemo-bio-informatics Applications, Using Plug-in Architecture and Script Programming</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">167</span><span class="NLM_x">–</span> <span class="NLM_lpage">173</span><span class="refDoi"> DOI: 10.1023/B:JCAM.0000035186.90683.f2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01473&amp;key=10.1023%2FB%3AJCAM.0000035186.90683.f2" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2004&pages=167-173&author=A.+Pedrettiauthor=L.+Villaauthor=G.+Vistoli&title=VEGA%2D%2DAn+Open+Platform+to+Develop+Chemo-bio-informatics+Applications%2C+Using+Plug-in+Architecture+and+Script+Programming&doi=10.1023%2FB%3AJCAM.0000035186.90683.f2"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1023%2FB%3AJCAM.0000035186.90683.f2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FB%253AJCAM.0000035186.90683.f2%26sid%3Dliteratum%253Aachs%26aulast%3DPedretti%26aufirst%3DA.%26aulast%3DVilla%26aufirst%3DL.%26aulast%3DVistoli%26aufirst%3DG.%26atitle%3DVEGA--An%2520Open%2520Platform%2520to%2520Develop%2520Chemo-bio-informatics%2520Applications%252C%2520Using%2520Plug-in%2520Architecture%2520and%2520Script%2520Programming%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2004%26volume%3D18%26spage%3D167%26epage%3D173%26doi%3D10.1023%2FB%3AJCAM.0000035186.90683.f2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Testa, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pedretti, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vistoli, G.</span><span> </span><span class="NLM_article-title">Reactions and Enzymes in the Metabolism of Drugs and Other Xenobiotics</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x"> (</span><span class="NLM_issue">11–12</span><span class="NLM_x">) </span> <span class="NLM_fpage">549</span><span class="NLM_x">–</span> <span class="NLM_lpage">560</span><span class="refDoi"> DOI: 10.1016/j.drudis.2012.01.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01473&amp;key=10.1016%2Fj.drudis.2012.01.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01473&amp;key=22305937" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01473&amp;key=1%3ACAS%3A528%3ADC%252BC38XnslOiur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2012&pages=549-560&issue=11%E2%80%9312&author=B.+Testaauthor=A.+Pedrettiauthor=G.+Vistoli&title=Reactions+and+Enzymes+in+the+Metabolism+of+Drugs+and+Other+Xenobiotics&doi=10.1016%2Fj.drudis.2012.01.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Reactions and enzymes in the metabolism of drugs and other xenobiotics</span></div><div class="casAuthors">Testa, Bernard; Pedretti, Alessandro; Vistoli, Giulio</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">11-12</span>),
    <span class="NLM_cas:pages">549-560</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">In this article, we offer an overview of the compared quant. importance of biotransformation reactions in the metab. of drugs and other xenobiotics, based on a meta-anal. of current research interests.  Also, we assess the relative significance the enzyme (super)families or categories catalyzing these reactions.  We put the facts unveiled by the anal. into a drug discovery context and draw some implications.  The results confirm the primary role of cytochrome P 450-catalyzed oxidns. and UDP-glucuronosyl-catalyzed glucuronidations, but they also document the marked significance of several other reactions.  Thus, there is a need for several drug discovery scientists to better grasp the variety of drug metab. reactions and enzymes and their consequences.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbRR72SBUfcLVg90H21EOLACvtfcHk0lgO_o7QsPqPqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XnslOiur8%253D&md5=e11b5b1b7ea18e8bc817bff83efad456</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2012.01.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2012.01.017%26sid%3Dliteratum%253Aachs%26aulast%3DTesta%26aufirst%3DB.%26aulast%3DPedretti%26aufirst%3DA.%26aulast%3DVistoli%26aufirst%3DG.%26atitle%3DReactions%2520and%2520Enzymes%2520in%2520the%2520Metabolism%2520of%2520Drugs%2520and%2520Other%2520Xenobiotics%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2012%26volume%3D17%26issue%3D11%25E2%2580%259312%26spage%3D549%26epage%3D560%26doi%3D10.1016%2Fj.drudis.2012.01.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Bajusz, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rácz, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Héberger, K.</span><span> </span><span class="NLM_article-title">Why is Tanimoto Index an Appropriate Choice for Fingerprint-based Similarity Calculations?</span> <span class="citation_source-journal">J. Cheminf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">20</span><span class="refDoi"> DOI: 10.1186/s13321-015-0069-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01473&amp;key=10.1186%2Fs13321-015-0069-3" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2015&pages=20&author=D.+Bajuszauthor=A.+R%C3%A1czauthor=K.+H%C3%A9berger&title=Why+is+Tanimoto+Index+an+Appropriate+Choice+for+Fingerprint-based+Similarity+Calculations%3F&doi=10.1186%2Fs13321-015-0069-3"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1186%2Fs13321-015-0069-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs13321-015-0069-3%26sid%3Dliteratum%253Aachs%26aulast%3DBajusz%26aufirst%3DD.%26aulast%3DR%25C3%25A1cz%26aufirst%3DA.%26aulast%3DH%25C3%25A9berger%26aufirst%3DK.%26atitle%3DWhy%2520is%2520Tanimoto%2520Index%2520an%2520Appropriate%2520Choice%2520for%2520Fingerprint-based%2520Similarity%2520Calculations%253F%26jtitle%3DJ.%2520Cheminf.%26date%3D2015%26volume%3D7%26spage%3D20%26doi%3D10.1186%2Fs13321-015-0069-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Testa, B.; Krämer, S. D.</span> <span class="citation_source-book">The Biochemistry of Drug Metabolism, Vol. 1: Principles, Redox Reactions, Hydrolyses</span>; <span class="NLM_publisher-name">Wiley-VHCA and Wiley-VCH</span>: <span class="NLM_publisher-loc">Zurich and Weinheim</span>,<span class="NLM_x"> </span><span class="NLM_year">2008</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2008&author=B.+Testa&author=S.+D.+Kr%C3%A4mer&title=The+Biochemistry+of+Drug+Metabolism%2C+Vol.+1%3A+Principles%2C+Redox+Reactions%2C+Hydrolyses"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DTesta%26aufirst%3DB.%26btitle%3DThe%2520Biochemistry%2520of%2520Drug%2520Metabolism%252C%2520Vol.%25201%253A%2520Principles%252C%2520Redox%2520Reactions%252C%2520Hydrolyses%26pub%3DWiley-VHCA%2520and%2520Wiley-VCH%26date%3D2008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Testa, B.; Krämer, S. D.</span> <span class="citation_source-book">The Biochemistry of Drug Metabolism, Vol. 2: Conjugations, Consequences of Metabolism, Influencing Factors</span>; <span class="NLM_publisher-name">Wiley-VHCA and Wiley-VCH</span>: <span class="NLM_publisher-loc">Zurich and Weinheim</span>,<span class="NLM_x"> </span><span class="NLM_year">2010</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2010&author=B.+Testa&author=S.+D.+Kr%C3%A4mer&title=The+Biochemistry+of+Drug+Metabolism%2C+Vol.+2%3A+Conjugations%2C+Consequences+of+Metabolism%2C+Influencing+Factors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DTesta%26aufirst%3DB.%26btitle%3DThe%2520Biochemistry%2520of%2520Drug%2520Metabolism%252C%2520Vol.%25202%253A%2520Conjugations%252C%2520Consequences%2520of%2520Metabolism%252C%2520Influencing%2520Factors%26pub%3DWiley-VHCA%2520and%2520Wiley-VCH%26date%3D2010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Testa, B.</span><span> </span><span class="NLM_article-title">Principles of Drug Metabolism</span>. In  <span class="citation_source-book">Burger’s Medicinal Chemistry, Drug Discovery and Development</span>, <span class="NLM_edition">7th</span> ed.; <span class="NLM_contrib-group">Abraham, D. J.; Rotella, D. P.</span>, Eds.; <span class="NLM_publisher-name">Wiley</span>: <span class="NLM_publisher-loc">Hoboken, NJ</span>,<span class="NLM_x"> </span><span class="NLM_year">2010</span><span class="NLM_x">; </span>pp  <span class="NLM_fpage">127</span><span class="NLM_x">–</span> <span class="NLM_lpage">166</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01473&amp;key=10.1002%2F0471266949.bmc033.pub2" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2010&pages=127-166&author=B.+Testaauthor=D.+J.+Abraham&author=D.+P.+Rotella&title=Burger%E2%80%99s+Medicinal+Chemistry%2C+Drug+Discovery+and+Development"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1002%2F0471266949.bmc033.pub2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F0471266949.bmc033.pub2%26sid%3Dliteratum%253Aachs%26aulast%3DTesta%26aufirst%3DB.%26atitle%3DPrinciples%2520of%2520Drug%2520Metabolism%26btitle%3DBurger%25E2%2580%2599s%2520Medicinal%2520Chemistry%252C%2520Drug%2520Discovery%2520and%2520Development%26aulast%3DAbraham%26aufirst%3DD.%2BJ.%26pub%3DWiley%26date%3D2010%26spage%3D127%26epage%3D166" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Testa, B.; Clement, B.</span><span> </span><span class="NLM_article-title">Biotransformation Reactions and their Enzymes</span>. In  <span class="citation_source-book">The Practice of Medicinal Chemistry</span>, <span class="NLM_edition">4th</span> ed.; <span class="NLM_contrib-group">Wermuth, C. G.; Aldous, D.; Raboisson, P.; Rognan, D.</span>, Eds.; <span class="NLM_publisher-name">Academic Press</span>: <span class="NLM_publisher-loc">London</span>,<span class="NLM_x"> </span><span class="NLM_year">2015</span><span class="NLM_x">; </span>pp  <span class="NLM_fpage">561</span><span class="NLM_x">–</span> <span class="NLM_lpage">584</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01473&amp;key=10.1016%2FB978-0-12-417205-0.00024-9" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&pages=561-584&author=B.+Testa&author=B.+Clementauthor=C.+G.+Wermuth&author=D.+Aldous&author=P.+Raboisson&author=D.+Rognan&title=The+Practice+of+Medicinal+Chemistry"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2FB978-0-12-417205-0.00024-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1016%252FB978-0-12-417205-0.00024-9%26sid%3Dliteratum%253Aachs%26aulast%3DTesta%26aufirst%3DB.%26atitle%3DBiotransformation%2520Reactions%2520and%2520their%2520Enzymes%26btitle%3DThe%2520Practice%2520of%2520Medicinal%2520Chemistry%26aulast%3DWermuth%26aufirst%3DC.%2BG.%26pub%3DAcademic%2520Press%26date%3D2015%26spage%3D561%26epage%3D584" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Gaillard, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carrupt, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Testa, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boudon, A.</span><span> </span><span class="NLM_article-title">Molecular Lipophilicity Potential, a Tool in 3D-QSAR: Method and Applications</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">83</span><span class="NLM_x">–</span> <span class="NLM_lpage">96</span><span class="refDoi"> DOI: 10.1007/BF00119860</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01473&amp;key=10.1007%2FBF00119860" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=1994&pages=83-96&author=P.+Gaillardauthor=P.+A.+Carruptauthor=B.+Testaauthor=A.+Boudon&title=Molecular+Lipophilicity+Potential%2C+a+Tool+in+3D-QSAR%3A+Method+and+Applications&doi=10.1007%2FBF00119860"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1007%2FBF00119860&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FBF00119860%26sid%3Dliteratum%253Aachs%26aulast%3DGaillard%26aufirst%3DP.%26aulast%3DCarrupt%26aufirst%3DP.%2BA.%26aulast%3DTesta%26aufirst%3DB.%26aulast%3DBoudon%26aufirst%3DA.%26atitle%3DMolecular%2520Lipophilicity%2520Potential%252C%2520a%2520Tool%2520in%25203D-QSAR%253A%2520Method%2520and%2520Applications%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D1994%26volume%3D8%26spage%3D83%26epage%3D96%26doi%3D10.1007%2FBF00119860" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Marchant, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosser, E. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vessey, J. D.</span><span> </span><span class="NLM_article-title">A k-Nearest Neighbours Approach Using Metabolism-related Fingerprints to Improve In Silico Metabolite Ranking</span> <span class="citation_source-journal">Mol. Inf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">36</span><span class="NLM_x"> (</span><span class="NLM_issue">3</span><span class="NLM_x">) </span> <span class="NLM_fpage">1600105</span><span class="refDoi"> DOI: 10.1002/minf.201600105</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01473&amp;key=10.1002%2Fminf.201600105" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2017&pages=1600105&issue=3&author=C.+A.+Marchantauthor=E.+M.+Rosserauthor=J.+D.+Vessey&title=A+k-Nearest+Neighbours+Approach+Using+Metabolism-related+Fingerprints+to+Improve+In+Silico+Metabolite+Ranking&doi=10.1002%2Fminf.201600105"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1002%2Fminf.201600105&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fminf.201600105%26sid%3Dliteratum%253Aachs%26aulast%3DMarchant%26aufirst%3DC.%2BA.%26aulast%3DRosser%26aufirst%3DE.%2BM.%26aulast%3DVessey%26aufirst%3DJ.%2BD.%26atitle%3DA%2520k-Nearest%2520Neighbours%2520Approach%2520Using%2520Metabolism-related%2520Fingerprints%2520to%2520Improve%2520In%2520Silico%2520Metabolite%2520Ranking%26jtitle%3DMol.%2520Inf.%26date%3D2017%26volume%3D36%26issue%3D3%26spage%3D1600105%26doi%3D10.1002%2Fminf.201600105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Teffera, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berry, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brake, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lewis, R. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saffran, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moore, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, Z.</span><span> </span><span class="NLM_article-title">Impact of Hydrolysis-mediated Clearance on the Pharmacokinetics of Novel Anaplastic Lymphoma Kinase Inhibitors</span> <span class="citation_source-journal">Drug Metab. Dispos.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">41</span><span class="NLM_x">, </span> <span class="NLM_fpage">238</span><span class="NLM_x">–</span> <span class="NLM_lpage">247</span><span class="refDoi"> DOI: 10.1124/dmd.112.047993</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01473&amp;key=10.1124%2Fdmd.112.047993" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01473&amp;key=23118327" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01473&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlt1OntA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2013&pages=238-247&author=Y.+Tefferaauthor=L.+M.+Berryauthor=R.+L.+Brakeauthor=R.+T.+Lewisauthor=D.+C.+Saffranauthor=E.+Mooreauthor=J.+Liuauthor=Z.+Zhao&title=Impact+of+Hydrolysis-mediated+Clearance+on+the+Pharmacokinetics+of+Novel+Anaplastic+Lymphoma+Kinase+Inhibitors&doi=10.1124%2Fdmd.112.047993"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Impact of hydrolysis-mediated clearance on the pharmacokinetics of novel anaplastic lymphoma kinase inhibitors</span></div><div class="casAuthors">Teffera, Yohannes; Berry, Loren M.; Brake, Rachael L.; Lewis, Richard T.; Saffran, Douglas C.; Moore, Earl; Liu, Jingzhou; Zhao, Zhiyang</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism & Disposition</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">238-247</span>CODEN:
                <span class="NLM_cas:coden">DMDSAI</span>;
        ISSN:<span class="NLM_cas:issn">1521-009X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Compd. 1 [(E)-4-fluoro-N-(6-((4-(2-hydroxypropan-2-yl)piperidin-1-yl)methyl)-1-((1S,4S)-4-(isopropylcarbamoyl)cyclohexyl)-1H-benzo[d]imidazol-2(3H)-ylidene)benzamide], a new, potent, selective anaplastic lymphoma kinase (ALK) inhibitor with potential application for the treatment of cancer, was selected as candidate to advance into efficacy studies in mice.  However, the compd. underwent mouse-specific enzymic hydrolysis in plasma to a primary amine product (M1).  Subsequent i.v. pharmacokinetics studies in mice showed that compd. 1 had high clearance (CL) and a short half-life.  Oral dose escalation studies in mice indicated that elimination of compd. 1 was saturable, with higher doses achieving sufficient exposures above in vitro IC50.  Chem. efforts to minimize hydrolysis resulted in the discovery of several analogs that were stable in mouse plasma.  Three were taken in vivo into mice and showed decreased CL corresponding to increased in vitro stability in plasma.  However, the more stable compds. also showed reduced potency against ALK.  Kinetic studies in NADPH-fortified and unfortified microsomes and plasma produced submicromolar Km values and could help explain the satn. of elimination obsd. in vivo.  Predictions of CL based on kinetics from hydrolysis and NADPH-dependent pathways produced predicted hepatic CL values of 3.8, 3.0, 1.6, and 1.2 l/h·kg for compd. 1, compd. 2 [(E)-3,5-difluoro-N-(6-((4-(2-hydroxypropan-2-yl)piperidin-1-yl)methyl)-1-((1s,4s)-4-(isopropylcarbamoyl)cyclohexyl)-1H-benzo[d]imidazo-2(3H)-ylidene)benzamide], compd. 3 [(E)-3-chloro-5-fluoro-N-(6-((4-(2-hydroxypropan-2-yl)piperidin-1-yl)methyl)-1-((1s,4s)-4-(isopropylcarbamoyl)cyclohexyl)-1H-benzo[d]imidazol-2(3H)-ylidene)benzamide], and compd. 4 [(E)-N-(6-((4-(2-hydroxypropan-2-yl)piperidin-1-yl)methyl)-1-((1s,4s)-4-(isopropylcarbamoyl)cyclohexyl)-1H-benzo[d]imidazol-2(3H)-ylidene)-3-(trifluoromethyl)benzamide], resp.  The in vivo obsd. CLs for compds. 1, 2, 3, and 4 were 5.52, 3.51, 2.14, and 2.66 l/h kg, resp.  These results indicate that in vitro metab. kinetic data, incorporating contributions from both hydrolysis and NADPH-dependent metab., could be used to predict the systemic CL of compds. cleared via hydrolytic pathways provided that the in vitro assays thoroughly investigate the processes, including the contribution of other metabolic pathways and the possibility of satn. kinetics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokHx1C0KZVi7Vg90H21EOLACvtfcHk0li9SDNKWXLRlg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXlt1OntA%253D%253D&md5=a46b21704e0ecab631a6048ca947fe01</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1124%2Fdmd.112.047993&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.112.047993%26sid%3Dliteratum%253Aachs%26aulast%3DTeffera%26aufirst%3DY.%26aulast%3DBerry%26aufirst%3DL.%2BM.%26aulast%3DBrake%26aufirst%3DR.%2BL.%26aulast%3DLewis%26aufirst%3DR.%2BT.%26aulast%3DSaffran%26aufirst%3DD.%2BC.%26aulast%3DMoore%26aufirst%3DE.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DZhao%26aufirst%3DZ.%26atitle%3DImpact%2520of%2520Hydrolysis-mediated%2520Clearance%2520on%2520the%2520Pharmacokinetics%2520of%2520Novel%2520Anaplastic%2520Lymphoma%2520Kinase%2520Inhibitors%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2013%26volume%3D41%26spage%3D238%26epage%3D247%26doi%3D10.1124%2Fdmd.112.047993" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Takahashi, R. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Choo, E. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Halladay, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deng, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rooney, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gates, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hop, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Khojasteh, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dresser, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Musib, L.</span><span> </span><span class="NLM_article-title">Absorption, Metabolism, Excretion, and the Contribution of Intestinal Metabolism to the Oral Disposition of [14C]Cobimetinib, a MEK Inhibitor, in Humans</span> <span class="citation_source-journal">Drug Metab. Dispos.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">44</span><span class="NLM_x">, </span> <span class="NLM_fpage">28</span><span class="NLM_x">–</span> <span class="NLM_lpage">39</span><span class="refDoi"> DOI: 10.1124/dmd.115.066282</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01473&amp;key=10.1124%2Fdmd.115.066282" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=28-39&author=R.+H.+Takahashiauthor=E.+F.+Chooauthor=S.+Maauthor=S.+Wongauthor=J.+Halladayauthor=Y.+Dengauthor=I.+Rooneyauthor=M.+Gatesauthor=C.+E.+Hopauthor=S.+C.+Khojastehauthor=M.+J.+Dresserauthor=L.+Musib&title=Absorption%2C+Metabolism%2C+Excretion%2C+and+the+Contribution+of+Intestinal+Metabolism+to+the+Oral+Disposition+of+%5B14C%5DCobimetinib%2C+a+MEK+Inhibitor%2C+in+Humans&doi=10.1124%2Fdmd.115.066282"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1124%2Fdmd.115.066282&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.115.066282%26sid%3Dliteratum%253Aachs%26aulast%3DTakahashi%26aufirst%3DR.%2BH.%26aulast%3DChoo%26aufirst%3DE.%2BF.%26aulast%3DMa%26aufirst%3DS.%26aulast%3DWong%26aufirst%3DS.%26aulast%3DHalladay%26aufirst%3DJ.%26aulast%3DDeng%26aufirst%3DY.%26aulast%3DRooney%26aufirst%3DI.%26aulast%3DGates%26aufirst%3DM.%26aulast%3DHop%26aufirst%3DC.%2BE.%26aulast%3DKhojasteh%26aufirst%3DS.%2BC.%26aulast%3DDresser%26aufirst%3DM.%2BJ.%26aulast%3DMusib%26aufirst%3DL.%26atitle%3DAbsorption%252C%2520Metabolism%252C%2520Excretion%252C%2520and%2520the%2520Contribution%2520of%2520Intestinal%2520Metabolism%2520to%2520the%2520Oral%2520Disposition%2520of%2520%255B14C%255DCobimetinib%252C%2520a%2520MEK%2520Inhibitor%252C%2520in%2520Humans%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2016%26volume%3D44%26spage%3D28%26epage%3D39%26doi%3D10.1124%2Fdmd.115.066282" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Iegre, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hayes, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thompson, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weidolf, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Isin, E. M.</span><span> </span><span class="NLM_article-title">Database Extraction of Metabolite Information of Drug Candidates: Analysis of 27 AstraZeneca Compounds with Human Absorption, Distribution, Metabolism, and Excretion Data</span> <span class="citation_source-journal">Drug Metab. Dispos.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">44</span><span class="NLM_x">, </span> <span class="NLM_fpage">732</span><span class="NLM_x">–</span> <span class="NLM_lpage">740</span><span class="refDoi"> DOI: 10.1124/dmd.115.067850</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01473&amp;key=10.1124%2Fdmd.115.067850" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=732-740&author=J.+Iegreauthor=M.+A.+Hayesauthor=R.+A.+Thompsonauthor=L.+Weidolfauthor=E.+M.+Isin&title=Database+Extraction+of+Metabolite+Information+of+Drug+Candidates%3A+Analysis+of+27+AstraZeneca+Compounds+with+Human+Absorption%2C+Distribution%2C+Metabolism%2C+and+Excretion+Data&doi=10.1124%2Fdmd.115.067850"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1124%2Fdmd.115.067850&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.115.067850%26sid%3Dliteratum%253Aachs%26aulast%3DIegre%26aufirst%3DJ.%26aulast%3DHayes%26aufirst%3DM.%2BA.%26aulast%3DThompson%26aufirst%3DR.%2BA.%26aulast%3DWeidolf%26aufirst%3DL.%26aulast%3DIsin%26aufirst%3DE.%2BM.%26atitle%3DDatabase%2520Extraction%2520of%2520Metabolite%2520Information%2520of%2520Drug%2520Candidates%253A%2520Analysis%2520of%252027%2520AstraZeneca%2520Compounds%2520with%2520Human%2520Absorption%252C%2520Distribution%252C%2520Metabolism%252C%2520and%2520Excretion%2520Data%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2016%26volume%3D44%26spage%3D732%26epage%3D740%26doi%3D10.1124%2Fdmd.115.067850" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Zhu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ho, M. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong, Y. H.</span><span> </span><span class="NLM_article-title">3-Methoxylphenylpropyl Amides as Novel Receptor Subtype-selective Melatoninergic Ligands: Characterization of Physicochemical and Pharmacokinetic Properties</span> <span class="citation_source-journal">Xenobiotica</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">41</span><span class="NLM_x"> (</span><span class="NLM_issue">1</span><span class="NLM_x">) </span> <span class="NLM_fpage">35</span><span class="NLM_x">–</span> <span class="NLM_lpage">45</span><span class="refDoi"> DOI: 10.3109/00498254.2010.524264</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01473&amp;key=10.3109%2F00498254.2010.524264" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2011&pages=35-45&issue=1&author=J.+Zhuauthor=Y.+Huauthor=M.+K.+Hoauthor=Y.+H.+Wong&title=3-Methoxylphenylpropyl+Amides+as+Novel+Receptor+Subtype-selective+Melatoninergic+Ligands%3A+Characterization+of+Physicochemical+and+Pharmacokinetic+Properties&doi=10.3109%2F00498254.2010.524264"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.3109%2F00498254.2010.524264&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3109%252F00498254.2010.524264%26sid%3Dliteratum%253Aachs%26aulast%3DZhu%26aufirst%3DJ.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DHo%26aufirst%3DM.%2BK.%26aulast%3DWong%26aufirst%3DY.%2BH.%26atitle%3D3-Methoxylphenylpropyl%2520Amides%2520as%2520Novel%2520Receptor%2520Subtype-selective%2520Melatoninergic%2520Ligands%253A%2520Characterization%2520of%2520Physicochemical%2520and%2520Pharmacokinetic%2520Properties%26jtitle%3DXenobiotica%26date%3D2011%26volume%3D41%26issue%3D1%26spage%3D35%26epage%3D45%26doi%3D10.3109%2F00498254.2010.524264" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Dinh, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pearce, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Van Haandel, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gaedigk, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leeder, J. S.</span><span> </span><span class="NLM_article-title">Characterization of Atomoxetine Biotransformation and Implications for Development of PBPK Models for Dose Individualization in Children</span> <span class="citation_source-journal">Drug Metab. Dispos.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">44</span><span class="NLM_x">, </span> <span class="NLM_fpage">1070</span><span class="NLM_x">–</span> <span class="NLM_lpage">1079</span><span class="refDoi"> DOI: 10.1124/dmd.116.069518</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01473&amp;key=10.1124%2Fdmd.116.069518" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=1070-1079&author=J.+C.+Dinhauthor=R.+E.+Pearceauthor=L.+Van+Haandelauthor=A.+Gaedigkauthor=J.+S.+Leeder&title=Characterization+of+Atomoxetine+Biotransformation+and+Implications+for+Development+of+PBPK+Models+for+Dose+Individualization+in+Children&doi=10.1124%2Fdmd.116.069518"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1124%2Fdmd.116.069518&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.116.069518%26sid%3Dliteratum%253Aachs%26aulast%3DDinh%26aufirst%3DJ.%2BC.%26aulast%3DPearce%26aufirst%3DR.%2BE.%26aulast%3DVan%2BHaandel%26aufirst%3DL.%26aulast%3DGaedigk%26aufirst%3DA.%26aulast%3DLeeder%26aufirst%3DJ.%2BS.%26atitle%3DCharacterization%2520of%2520Atomoxetine%2520Biotransformation%2520and%2520Implications%2520for%2520Development%2520of%2520PBPK%2520Models%2520for%2520Dose%2520Individualization%2520in%2520Children%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2016%26volume%3D44%26spage%3D1070%26epage%3D1079%26doi%3D10.1124%2Fdmd.116.069518" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Dickinson, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cantarini, M. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Collier, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frewer, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pickup, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ballard, P.</span><span> </span><span class="NLM_article-title">Metabolic Disposition of Osimertinib in Rats, Dogs, and Humans: Insights into a Drug Designed to Bind Covalently to a Cysteine Residue of Epidermal Growth Factor Receptor</span> <span class="citation_source-journal">Drug Metab. Dispos.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">44</span><span class="NLM_x">, </span> <span class="NLM_fpage">1201</span><span class="NLM_x">–</span> <span class="NLM_lpage">1212</span><span class="refDoi"> DOI: 10.1124/dmd.115.069203</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01473&amp;key=10.1124%2Fdmd.115.069203" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01473&amp;key=27226351" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01473&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVajtb%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=1201-1212&author=P.+A.+Dickinsonauthor=M.+V.+Cantariniauthor=J.+Collierauthor=P.+Frewerauthor=S.+Martinauthor=K.+Pickupauthor=P.+Ballard&title=Metabolic+Disposition+of+Osimertinib+in+Rats%2C+Dogs%2C+and+Humans%3A+Insights+into+a+Drug+Designed+to+Bind+Covalently+to+a+Cysteine+Residue+of+Epidermal+Growth+Factor+Receptor&doi=10.1124%2Fdmd.115.069203"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Metabolic disposition of osimertinib in rats, dogs, and humans: insights into a drug designed to bind covalently to a cysteine residue of epidermal growth factor receptor</span></div><div class="casAuthors">Dickinson, Paul A.; Cantarini, Mireille V.; Collier, Jo; Frewer, Paul; Martin, Scott; Pickup, Kathryn; Ballard, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism & Disposition</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1201-1212</span>CODEN:
                <span class="NLM_cas:coden">DMDSAI</span>;
        ISSN:<span class="NLM_cas:issn">1521-009X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Preclin. and clin. studies were conducted to det. the metab. and pharmacokinetics of osimertinib and key metabolites AZ5104 and AZ7550.  Osimertinib was designed to covalently bind to epidermal growth factor receptors, allowing it to achieve nanomolar cellular potency.  Covalent binding was obsd. in incubations of radiolabeled osimertinib with human and rat hepatocytes, human and rat plasma, and human serum albumin.  Osimertinib, AZ5104, and AZ7550 were predominantly metabolized by CYP3A.  Seven metabolites were detected in human hepatocytes, also obsd. in rat or dog hepatocytes at similar or higher levels.  After oral administration of radiolabeled osimertinib to rats, drug-related material was widely distributed, with the highest radioactivity concns. measured at 6 h postdose in most tissues; radioactivity was detectable in 42% of tissues 60 days postdose.  Concns. of [14C]-radioactivity in blood were lower than in most tissues.  After the administration of a single oral dose of 20 mg of radiolabeled osimertinib to healthy male volunteers, ∼19% of the dose was recovered by 3 days postdose.  At 84 days postdose, mean total radioactivity recovery was 14.2% and 67.8% of the dose in urine and feces.  The most abundant metabolite identified in feces was AZ5104 (∼6% of dose).  Osimertinib accounted for ∼1% of total radioactivity in the plasma of non-small cell lung cancer patients after 22 days of 80-mg osimertinib once-daily treatment; the most abundant circulatory metabolites were AZ7550 and AZ5104 (<10% of total osimertinib-related material).  Osimertinib is extensively distributed and metabolized in humans and is eliminated primarily via the fecal route.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqslCW8h-RI9bVg90H21EOLACvtfcHk0lgo4d6Bp3lzDA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitVajtb%252FO&md5=4115de7691a3b39eec091660b5e5b1d6</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1124%2Fdmd.115.069203&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.115.069203%26sid%3Dliteratum%253Aachs%26aulast%3DDickinson%26aufirst%3DP.%2BA.%26aulast%3DCantarini%26aufirst%3DM.%2BV.%26aulast%3DCollier%26aufirst%3DJ.%26aulast%3DFrewer%26aufirst%3DP.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DPickup%26aufirst%3DK.%26aulast%3DBallard%26aufirst%3DP.%26atitle%3DMetabolic%2520Disposition%2520of%2520Osimertinib%2520in%2520Rats%252C%2520Dogs%252C%2520and%2520Humans%253A%2520Insights%2520into%2520a%2520Drug%2520Designed%2520to%2520Bind%2520Covalently%2520to%2520a%2520Cysteine%2520Residue%2520of%2520Epidermal%2520Growth%2520Factor%2520Receptor%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2016%26volume%3D44%26spage%3D1201%26epage%3D1212%26doi%3D10.1124%2Fdmd.115.069203" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Lee, J. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Choi, Y. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oh, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chi, Y. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paik, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, K. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jung, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ryu, C. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, K. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, D. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yoon, Y. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, S. K.</span><span> </span><span class="NLM_article-title">Characterization of Fimasartan Metabolites in Human Liver Microsomes and Human Plasma</span> <span class="citation_source-journal">Xenobiotica</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">40</span><span class="NLM_x">–</span> <span class="NLM_lpage">51</span><span class="refDoi"> DOI: 10.3109/00498254.2015.1047429</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01473&amp;key=10.3109%2F00498254.2015.1047429" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2016&pages=40-51&author=J.+Y.+Leeauthor=Y.+J.+Choiauthor=S.+J.+Ohauthor=Y.+H.+Chiauthor=S.+H.+Paikauthor=K.+H.+Leeauthor=J.+K.+Jungauthor=C.+S.+Ryuauthor=K.+B.+Kimauthor=D.+H.+Kimauthor=Y.+R.+Yoonauthor=S.+K.+Kim&title=Characterization+of+Fimasartan+Metabolites+in+Human+Liver+Microsomes+and+Human+Plasma&doi=10.3109%2F00498254.2015.1047429"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.3109%2F00498254.2015.1047429&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3109%252F00498254.2015.1047429%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DJ.%2BY.%26aulast%3DChoi%26aufirst%3DY.%2BJ.%26aulast%3DOh%26aufirst%3DS.%2BJ.%26aulast%3DChi%26aufirst%3DY.%2BH.%26aulast%3DPaik%26aufirst%3DS.%2BH.%26aulast%3DLee%26aufirst%3DK.%2BH.%26aulast%3DJung%26aufirst%3DJ.%2BK.%26aulast%3DRyu%26aufirst%3DC.%2BS.%26aulast%3DKim%26aufirst%3DK.%2BB.%26aulast%3DKim%26aufirst%3DD.%2BH.%26aulast%3DYoon%26aufirst%3DY.%2BR.%26aulast%3DKim%26aufirst%3DS.%2BK.%26atitle%3DCharacterization%2520of%2520Fimasartan%2520Metabolites%2520in%2520Human%2520Liver%2520Microsomes%2520and%2520Human%2520Plasma%26jtitle%3DXenobiotica%26date%3D2016%26volume%3D46%26spage%3D40%26epage%3D51%26doi%3D10.3109%2F00498254.2015.1047429" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Rioux, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duncan, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lantz, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miao, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chan-Penebre, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moyer, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Munchhof, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Copeland, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chesworth, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waters, N. J.</span><span> </span><span class="NLM_article-title">Species Differences in Metabolism of EPZ015666, an Oxetane-containing Protein Arginine Methyltransferase-5 (PRMT5) Inhibitor</span> <span class="citation_source-journal">Xenobiotica</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">268</span><span class="NLM_x">–</span> <span class="NLM_lpage">277</span><span class="refDoi"> DOI: 10.3109/00498254.2015.1072253</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01473&amp;key=10.3109%2F00498254.2015.1072253" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01473&amp;key=26294260" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01473&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFKlu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2016&pages=268-277&author=N.+Riouxauthor=K.+W.+Duncanauthor=R.+J.+Lantzauthor=X.+Miaoauthor=E.+Chan-Penebreauthor=M.+P.+Moyerauthor=M.+J.+Munchhofauthor=R.+A.+Copelandauthor=R.+Chesworthauthor=N.+J.+Waters&title=Species+Differences+in+Metabolism+of+EPZ015666%2C+an+Oxetane-containing+Protein+Arginine+Methyltransferase-5+%28PRMT5%29+Inhibitor&doi=10.3109%2F00498254.2015.1072253"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Species differences in metabolism of EPZ015666, an oxetane-containing protein arginine methyltransferase-5 (PRMT5) inhibitor</span></div><div class="casAuthors">Rioux, Nathalie; Duncan, Kenneth W.; Lantz, Ronald J.; Miao, Xiusheng; Chan-Penebre, Elayne; Moyer, Mikel P.; Munchhof, Michael J.; Copeland, Robert A.; Chesworth, Richard; Waters, Nigel J.</div><div class="citationInfo"><span class="NLM_cas:title">Xenobiotica</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">268-277</span>CODEN:
                <span class="NLM_cas:coden">XENOBH</span>;
        ISSN:<span class="NLM_cas:issn">0049-8254</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">1. Metabolite profiling and identification studies were conducted to understand the cross-species differences in the metabolic clearance of EPZ015666, a first-in-class protein arginine methyltransferase-5 (PRMT5) inhibitor, with anti-proliferative effects in preclin. models of Mantle Cell Lymphoma.  EPZ015666 exhibited low clearance in human, mouse and rat liver microsomes, in part by introduction of a 3-substituted oxetane ring on the mol.  In contrast, a higher clearance was obsd. in dog liver microsomes (DLM) that translated to a higher in vivo clearance in dog compared with rodent.2.  Structure elucidation via high resoln., accurate mass LC-MSn revealed that the prominent metabolites of EPZ015666 were present in hepatocytes from all species, with the highest turnover rate in dogs.  M1 and M2 resulted from oxidative oxetane ring scission, whereas M3 resulted from loss of the oxetane ring via an N-dealkylation reaction.3.  The formation of M1 and M2 in DLM was significantly abrogated in the presence of the specific CYP2D inhibitor, quinidine, and to a lesser extent by the CYP3A inhibitor, ketoconazole, corroborating data from human recombinant isoenzymes.4.  Our data indicate a marked species difference in the metab. of the PRMT5 inhibitor EPZ015666, with oxetane ring scission the predominant metabolic pathway in dog mediated largely by CYP2D.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq1gD_mblcnKbVg90H21EOLACvtfcHk0lhbvzaU7_eWLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFKlu7w%253D&md5=489924f6c228fbb431d6b7257e5b9d9a</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.3109%2F00498254.2015.1072253&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3109%252F00498254.2015.1072253%26sid%3Dliteratum%253Aachs%26aulast%3DRioux%26aufirst%3DN.%26aulast%3DDuncan%26aufirst%3DK.%2BW.%26aulast%3DLantz%26aufirst%3DR.%2BJ.%26aulast%3DMiao%26aufirst%3DX.%26aulast%3DChan-Penebre%26aufirst%3DE.%26aulast%3DMoyer%26aufirst%3DM.%2BP.%26aulast%3DMunchhof%26aufirst%3DM.%2BJ.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DChesworth%26aufirst%3DR.%26aulast%3DWaters%26aufirst%3DN.%2BJ.%26atitle%3DSpecies%2520Differences%2520in%2520Metabolism%2520of%2520EPZ015666%252C%2520an%2520Oxetane-containing%2520Protein%2520Arginine%2520Methyltransferase-5%2520%2528PRMT5%2529%2520Inhibitor%26jtitle%3DXenobiotica%26date%3D2016%26volume%3D46%26spage%3D268%26epage%3D277%26doi%3D10.3109%2F00498254.2015.1072253" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Xie, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Du, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, F.</span><span> </span><span class="NLM_article-title">Structural Elucidation of in Vitro Metabolites of Bavachinin in Rat Liver Microsomes by LC-ESI-MSn and Chemical Synthesis</span> <span class="citation_source-journal">Xenobiotica</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">296</span><span class="NLM_x">–</span> <span class="NLM_lpage">306</span><span class="refDoi"> DOI: 10.3109/00498254.2015.1074763</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01473&amp;key=10.3109%2F00498254.2015.1074763" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2016&pages=296-306&author=F.+Xieauthor=G.+Duauthor=S.+Maauthor=Y.+Liauthor=R.+Wangauthor=F.+Guo&title=Structural+Elucidation+of+in+Vitro+Metabolites+of+Bavachinin+in+Rat+Liver+Microsomes+by+LC-ESI-MSn+and+Chemical+Synthesis&doi=10.3109%2F00498254.2015.1074763"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.3109%2F00498254.2015.1074763&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3109%252F00498254.2015.1074763%26sid%3Dliteratum%253Aachs%26aulast%3DXie%26aufirst%3DF.%26aulast%3DDu%26aufirst%3DG.%26aulast%3DMa%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DR.%26aulast%3DGuo%26aufirst%3DF.%26atitle%3DStructural%2520Elucidation%2520of%2520in%2520Vitro%2520Metabolites%2520of%2520Bavachinin%2520in%2520Rat%2520Liver%2520Microsomes%2520by%2520LC-ESI-MSn%2520and%2520Chemical%2520Synthesis%26jtitle%3DXenobiotica%26date%3D2016%26volume%3D46%26spage%3D296%26epage%3D306%26doi%3D10.3109%2F00498254.2015.1074763" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">Sanga, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banach, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ledvina, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Modi, N. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mittur, A.</span><span> </span><span class="NLM_article-title">Pharmacokinetics, Metabolism, and Excretion of Nefopam, a Dual Reuptake Inhibitor in Healthy Male Volunteers</span> <span class="citation_source-journal">Xenobiotica</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">1001</span><span class="NLM_x">–</span> <span class="NLM_lpage">1016</span><span class="refDoi"> DOI: 10.3109/00498254.2015.1136989</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01473&amp;key=10.3109%2F00498254.2015.1136989" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2016&pages=1001-1016&author=M.+Sangaauthor=J.+Banachauthor=A.+Ledvinaauthor=N.+B.+Modiauthor=A.+Mittur&title=Pharmacokinetics%2C+Metabolism%2C+and+Excretion+of+Nefopam%2C+a+Dual+Reuptake+Inhibitor+in+Healthy+Male+Volunteers&doi=10.3109%2F00498254.2015.1136989"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.3109%2F00498254.2015.1136989&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3109%252F00498254.2015.1136989%26sid%3Dliteratum%253Aachs%26aulast%3DSanga%26aufirst%3DM.%26aulast%3DBanach%26aufirst%3DJ.%26aulast%3DLedvina%26aufirst%3DA.%26aulast%3DModi%26aufirst%3DN.%2BB.%26aulast%3DMittur%26aufirst%3DA.%26atitle%3DPharmacokinetics%252C%2520Metabolism%252C%2520and%2520Excretion%2520of%2520Nefopam%252C%2520a%2520Dual%2520Reuptake%2520Inhibitor%2520in%2520Healthy%2520Male%2520Volunteers%26jtitle%3DXenobiotica%26date%3D2016%26volume%3D46%26spage%3D1001%26epage%3D1016%26doi%3D10.3109%2F00498254.2015.1136989" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Nijenhuis, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lucas, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosing, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robertson, P.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hellriegel, E. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schellens, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beijnen, A. J.</span><span> </span><span class="NLM_article-title">Metabolite Profiling of 14C-Omacetaxine Mepesuccinate in Plasma and Excreta of Cancer Patients</span> <span class="citation_source-journal">Xenobiotica</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">1122</span><span class="NLM_x">–</span> <span class="NLM_lpage">1132</span><span class="refDoi"> DOI: 10.3109/00498254.2016.1152418</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01473&amp;key=10.3109%2F00498254.2016.1152418" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2016&pages=1122-1132&author=C.+M.+Nijenhuisauthor=L.+Lucasauthor=H.+Rosingauthor=P.+Robertsonauthor=E.+T.+Hellriegelauthor=J.+H.+Schellensauthor=A.+J.+Beijnen&title=Metabolite+Profiling+of+14C-Omacetaxine+Mepesuccinate+in+Plasma+and+Excreta+of+Cancer+Patients&doi=10.3109%2F00498254.2016.1152418"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.3109%2F00498254.2016.1152418&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3109%252F00498254.2016.1152418%26sid%3Dliteratum%253Aachs%26aulast%3DNijenhuis%26aufirst%3DC.%2BM.%26aulast%3DLucas%26aufirst%3DL.%26aulast%3DRosing%26aufirst%3DH.%26aulast%3DRobertson%26aufirst%3DP.%26aulast%3DHellriegel%26aufirst%3DE.%2BT.%26aulast%3DSchellens%26aufirst%3DJ.%2BH.%26aulast%3DBeijnen%26aufirst%3DA.%2BJ.%26atitle%3DMetabolite%2520Profiling%2520of%252014C-Omacetaxine%2520Mepesuccinate%2520in%2520Plasma%2520and%2520Excreta%2520of%2520Cancer%2520Patients%26jtitle%3DXenobiotica%26date%3D2016%26volume%3D46%26spage%3D1122%26epage%3D1132%26doi%3D10.3109%2F00498254.2016.1152418" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 13 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Christina de Bruyn
Kops, Martin Šícho, Angelica Mazzolari, <span class="NLM_string-name hlFld-ContribAuthor">Johannes Kirchmair</span>. </span><span class="cited-content_cbyCitation_article-title">GLORYx: Prediction of the Metabolites Resulting from Phase 1 and Phase 2 Biotransformations of Xenobiotics. </span><span class="cited-content_cbyCitation_journal-name">Chemical Research in Toxicology</span><span> <strong>2021,</strong> <em>34 </em>
                                    (2)
                                     , 286-299. <a href="https://doi.org/10.1021/acs.chemrestox.0c00224" title="DOI URL">https://doi.org/10.1021/acs.chemrestox.0c00224</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.chemrestox.0c00224&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.chemrestox.0c00224%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Research%2520in%2520Toxicology%26atitle%3DGLORYx%25253A%252BPrediction%252Bof%252Bthe%252BMetabolites%252BResulting%252Bfrom%252BPhase%252B1%252Band%252BPhase%252B2%252BBiotransformations%252Bof%252BXenobiotics%26aulast%3Dde%2BBruyn%2BKops%26aufirst%3DChristina%26date%3D2021%26date%3D2020%26date%3D2020%26date%3D03062020%26date%3D26082020%26date%3D10082020%26volume%3D34%26issue%3D2%26spage%3D286%26epage%3D299" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Fabio Del Bello, Alessandro Bonifazi, Gianfabio Giorgioni, Alessandro Piergentili, Maria Giovanna Sabbieti, Dimitrios Agas, Marzia Dell’Aera, Rosanna Matucci, Marcin Górecki, Gennaro Pescitelli, Giulio Vistoli, <span class="NLM_string-name hlFld-ContribAuthor">Wilma Quaglia</span>. </span><span class="cited-content_cbyCitation_article-title">Novel Potent Muscarinic Receptor Antagonists: Investigation on the Nature of Lipophilic Substituents in the 5- and/or 6-Positions of the 1,4-Dioxane Nucleus. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (11)
                                     , 5763-5782. <a href="https://doi.org/10.1021/acs.jmedchem.9b02100" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b02100</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b02100&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b02100%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DNovel%252BPotent%252BMuscarinic%252BReceptor%252BAntagonists%25253A%252BInvestigation%252Bon%252Bthe%252BNature%252Bof%252BLipophilic%252BSubstituents%252Bin%252Bthe%252B5-%252Band%25252For%252B6-Positions%252Bof%252Bthe%252B1%25252C4-Dioxane%252BNucleus%26aulast%3DDel%2BBello%26aufirst%3DFabio%26date%3D2020%26date%3D2020%26date%3D2019%26date%3D17122019%26date%3D19052020%26date%3D06052020%26volume%3D63%26issue%3D11%26spage%3D5763%26epage%3D5782" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Martin Šícho, Conrad Stork, Angelica Mazzolari, Christina de Bruyn Kops, Alessandro Pedretti, Bernard Testa, Giulio Vistoli, Daniel Svozil, <span class="NLM_string-name hlFld-ContribAuthor">Johannes Kirchmair</span>. </span><span class="cited-content_cbyCitation_article-title">FAME 3: Predicting the Sites of Metabolism in Synthetic Compounds and Natural Products for Phase 1 and Phase 2 Metabolic Enzymes. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2019,</strong> <em>59 </em>
                                    (8)
                                     , 3400-3412. <a href="https://doi.org/10.1021/acs.jcim.9b00376" title="DOI URL">https://doi.org/10.1021/acs.jcim.9b00376</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.9b00376&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.9b00376%26sid%3Dliteratum%253Aachs%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D1019%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DFAME%252B3%25253A%252BPredicting%252Bthe%252BSites%252Bof%252BMetabolism%252Bin%252BSynthetic%252BCompounds%252Band%252BNatural%252BProducts%252Bfor%252BPhase%252B1%252Band%252BPhase%252B2%252BMetabolic%252BEnzymes%26aulast%3D%25C5%25A0%25C3%25ADcho%26aufirst%3DMartin%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D04052019%26date%3D13082019%26date%3D30072019%26volume%3D59%26issue%3D8%26spage%3D3400%26epage%3D3412" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Fabio Del Bello, Maria Vittoria Micioni Di Bonaventura, Alessandro Bonifazi, Bernhard Wünsch, Dirk Schepmann, JoLynn B. Giancola, Emanuela Micioni Di Bonaventura, Giulio Vistoli, Gianfabio Giorgioni, Wilma Quaglia, Alessandro Piergentili, <span class="NLM_string-name hlFld-ContribAuthor">Carlo Cifani</span>. </span><span class="cited-content_cbyCitation_article-title">Investigation of the Role of Chirality in the Interaction with σ Receptors and Effect on Binge Eating Episode of a Potent σ1 Antagonist Analogue of Spipethiane. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Neuroscience</span><span> <strong>2019,</strong> <em>10 </em>
                                    (8)
                                     , 3391-3397. <a href="https://doi.org/10.1021/acschemneuro.9b00261" title="DOI URL">https://doi.org/10.1021/acschemneuro.9b00261</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acschemneuro.9b00261&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facschemneuro.9b00261%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Neuroscience%26atitle%3DInvestigation%252Bof%252Bthe%252BRole%252Bof%252BChirality%252Bin%252Bthe%252BInteraction%252Bwith%252B%2525CF%252583%252BReceptors%252Band%252BEffect%252Bon%252BBinge%252BEating%252BEpisode%252Bof%252Ba%252BPotent%252B%2525CF%2525831%252BAntagonist%252BAnalogue%252Bof%252BSpipethiane%26aulast%3DDel%2BBello%26aufirst%3DFabio%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D02052019%26date%3D12072019%26date%3D23072019%26date%3D12072019%26volume%3D10%26issue%3D8%26spage%3D3391%26epage%3D3397" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Angelica Mazzolari, Avid M. Afzal, Alessandro Pedretti, Bernard Testa, Giulio Vistoli, <span class="NLM_string-name hlFld-ContribAuthor">Andreas Bender</span>. </span><span class="cited-content_cbyCitation_article-title">Prediction of UGT-mediated Metabolism Using the Manually Curated MetaQSAR Database. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2019,</strong> <em>10 </em>
                                    (4)
                                     , 633-638. <a href="https://doi.org/10.1021/acsmedchemlett.8b00603" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.8b00603</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.8b00603&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.8b00603%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DPrediction%252Bof%252BUGT-mediated%252BMetabolism%252BUsing%252Bthe%252BManually%252BCurated%252BMetaQSAR%252BDatabase%26aulast%3DMazzolari%26aufirst%3DAngelica%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D2018%26date%3D30112018%26date%3D12022019%26date%3D18022019%26date%3D12022019%26volume%3D10%26issue%3D4%26spage%3D633%26epage%3D638" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Fabio Del Bello, Alessandro Bonifazi, Gianfabio Giorgioni, Carlo Cifani, Maria Vittoria Micioni Di Bonaventura, Riccardo Petrelli, Alessandro Piergentili, Stefano Fontana, Valerio Mammoli, Hideaki Yano, Rosanna Matucci, Giulio Vistoli, <span class="NLM_string-name hlFld-ContribAuthor">Wilma Quaglia</span>. </span><span class="cited-content_cbyCitation_article-title">1-[3-(4-Butylpiperidin-1-yl)propyl]-1,2,3,4-tetrahydroquinolin-2-one (77-LH-28-1) as a Model for the Rational Design of a Novel Class of Brain Penetrant Ligands with High Affinity and Selectivity for Dopamine D4 Receptor. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>61 </em>
                                    (8)
                                     , 3712-3725. <a href="https://doi.org/10.1021/acs.jmedchem.8b00265" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b00265</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b00265&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.8b00265%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3D1-%25255B3-%2525284-Butylpiperidin-1-yl%252529propyl%25255D-1%25252C2%25252C3%25252C4-tetrahydroquinolin-2-one%252B%25252877-LH-28-1%252529%252Bas%252Ba%252BModel%252Bfor%252Bthe%252BRational%252BDesign%252Bof%252Ba%252BNovel%252BClass%252Bof%252BBrain%252BPenetrant%252BLigands%252Bwith%252BHigh%252BAffinity%252Band%252BSelectivity%252Bfor%252BDopamine%252BD4%252BReceptor%26aulast%3DDel%2BBello%26aufirst%3DFabio%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D16022018%26date%3D06042018%26date%3D28032018%26volume%3D61%26issue%3D8%26spage%3D3712%26epage%3D3725" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">S. Cyrus  Khojasteh</span>, <span class="hlFld-ContribAuthor ">Upendra A.  Argikar</span>, <span class="hlFld-ContribAuthor ">James P.  Driscoll</span>, <span class="hlFld-ContribAuthor ">Carley J. S.  Heck</span>, <span class="hlFld-ContribAuthor ">Lloyd  King</span>, <span class="hlFld-ContribAuthor ">Klarissa D.  Jackson</span>, <span class="hlFld-ContribAuthor ">Wenying  Jian</span>, <span class="hlFld-ContribAuthor ">Amit S.  Kalgutkar</span>, <span class="hlFld-ContribAuthor ">Grover P.  Miller</span>, <span class="hlFld-ContribAuthor ">Valerie  Kramlinger</span>, <span class="hlFld-ContribAuthor ">Ivonne M. C. M.  Rietjens</span>, <span class="hlFld-ContribAuthor ">Aaron M.  Teitelbaum</span>, <span class="hlFld-ContribAuthor ">Kai  Wang</span>, <span class="hlFld-ContribAuthor ">Cong  Wei</span>. </span><span class="cited-content_cbyCitation_article-title">Novel advances in biotransformation and bioactivation research – 2020 year in review. </span><span class="cited-content_cbyCitation_journal-name">Drug Metabolism Reviews</span><span> <strong>2021,</strong> <em>48 </em>, 1-50. <a href="https://doi.org/10.1080/03602532.2021.1916028" title="DOI URL">https://doi.org/10.1080/03602532.2021.1916028</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/03602532.2021.1916028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F03602532.2021.1916028%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Metabolism%2520Reviews%26atitle%3DNovel%252Badvances%252Bin%252Bbiotransformation%252Band%252Bbioactivation%252Bresearch%252B%2525E2%252580%252593%252B2020%252Byear%252Bin%252Breview%26aulast%3DKhojasteh%26aufirst%3DS.%2BCyrus%26date%3D2021%26date%3D2021%26volume%3D48%26spage%3D1%26epage%3D50" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Alessandro  Pedretti</span>, <span class="hlFld-ContribAuthor ">Angelica  Mazzolari</span>, <span class="hlFld-ContribAuthor ">Silvia  Gervasoni</span>, <span class="hlFld-ContribAuthor ">Laura  Fumagalli</span>, <span class="hlFld-ContribAuthor ">Giulio  Vistoli</span>, . </span><span class="cited-content_cbyCitation_article-title">The VEGA suite of programs: an versatile platform for cheminformatics and drug design projects. </span><span class="cited-content_cbyCitation_journal-name">Bioinformatics</span><span> <strong>2021,</strong> <em>37 </em>
                                    (8)
                                     , 1174-1175. <a href="https://doi.org/10.1093/bioinformatics/btaa774" title="DOI URL">https://doi.org/10.1093/bioinformatics/btaa774</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1093/bioinformatics/btaa774&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1093%2Fbioinformatics%2Fbtaa774%26sid%3Dliteratum%253Aachs%26jtitle%3DBioinformatics%26atitle%3DThe%252BVEGA%252Bsuite%252Bof%252Bprograms%25253A%252Ban%252Bversatile%252Bplatform%252Bfor%252Bcheminformatics%252Band%252Bdrug%252Bdesign%252Bprojects%26aulast%3DPedretti%26aufirst%3DAlessandro%26date%3D2021%26date%3D2020%26volume%3D37%26issue%3D8%26spage%3D1174%26epage%3D1175" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Angelica  Mazzolari</span>, <span class="hlFld-ContribAuthor ">Luca  Sommaruga</span>, <span class="hlFld-ContribAuthor ">Alessandro  Pedretti</span>, <span class="hlFld-ContribAuthor ">Giulio  Vistoli</span>. </span><span class="cited-content_cbyCitation_article-title">MetaTREE, a Novel Database Focused on Metabolic Trees, Predicts an Important Detoxification Mechanism: The Glutathione Conjugation. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2021,</strong> <em>26 </em>
                                    (7)
                                     , 2098. <a href="https://doi.org/10.3390/molecules26072098" title="DOI URL">https://doi.org/10.3390/molecules26072098</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules26072098&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules26072098%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DMetaTREE%25252C%252Ba%252BNovel%252BDatabase%252BFocused%252Bon%252BMetabolic%252BTrees%25252C%252BPredicts%252Ban%252BImportant%252BDetoxification%252BMechanism%25253A%252BThe%252BGlutathione%252BConjugation%26aulast%3DMazzolari%26aufirst%3DAngelica%26date%3D2021%26date%3D2021%26volume%3D26%26issue%3D7%26spage%3D2098" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jonathan D  Tyzack</span>. </span><span class="cited-content_cbyCitation_article-title">Prediction of Drug Metabolism: Use of Structural Biology and In Silico Tools. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,<a href="https://doi.org/10.1016/B978-0-12-820472-6.00067-0" title="DOI URL">https://doi.org/10.1016/B978-0-12-820472-6.00067-0</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-820472-6.00067-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-820472-6.00067-0%26sid%3Dliteratum%253Aachs%26atitle%3DPrediction%252Bof%252BDrug%252BMetabolism%25253A%252BUse%252Bof%252BStructural%252BBiology%252Band%252BIn%252BSilico%252BTools%26aulast%3DTyzack%26aufirst%3DJonathan%2BD%26date%3D2021%26pub%3DElsevier%26atitle%3DReference%252BModule%252Bin%252BBiomedical%252BSciences%26date%3D2021%26volume%3D39" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">P. M.  Pieczywek</span>, <span class="hlFld-ContribAuthor ">W.  Płaziński</span>, <span class="hlFld-ContribAuthor ">A.  Zdunek</span>. </span><span class="cited-content_cbyCitation_article-title">Dissipative particle dynamics model of homogalacturonan based on molecular dynamics simulations. </span><span class="cited-content_cbyCitation_journal-name">Scientific Reports</span><span> <strong>2020,</strong> <em>10 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s41598-020-71820-2" title="DOI URL">https://doi.org/10.1038/s41598-020-71820-2</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41598-020-71820-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41598-020-71820-2%26sid%3Dliteratum%253Aachs%26jtitle%3DScientific%2520Reports%26atitle%3DDissipative%252Bparticle%252Bdynamics%252Bmodel%252Bof%252Bhomogalacturonan%252Bbased%252Bon%252Bmolecular%252Bdynamics%252Bsimulations%26aulast%3DPieczywek%26aufirst%3DP.%2BM.%26date%3D2020%26date%3D2020%26volume%3D10%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Conrad  Stork</span>, <span class="hlFld-ContribAuthor ">Gerd  Embruch</span>, <span class="hlFld-ContribAuthor ">Martin  Šícho</span>, <span class="hlFld-ContribAuthor ">Christina  de Bruyn Kops</span>, <span class="hlFld-ContribAuthor ">Ya  Chen</span>, <span class="hlFld-ContribAuthor ">Daniel  Svozil</span>, <span class="hlFld-ContribAuthor ">Johannes  Kirchmair</span>, . </span><span class="cited-content_cbyCitation_article-title">NERDD: a web portal providing access to in silico tools for drug discovery. </span><span class="cited-content_cbyCitation_journal-name">Bioinformatics</span><span> <strong>2019,</strong> <em>53 </em><a href="https://doi.org/10.1093/bioinformatics/btz695" title="DOI URL">https://doi.org/10.1093/bioinformatics/btz695</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1093/bioinformatics/btz695&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1093%2Fbioinformatics%2Fbtz695%26sid%3Dliteratum%253Aachs%26jtitle%3DBioinformatics%26atitle%3DNERDD%25253A%252Ba%252Bweb%252Bportal%252Bproviding%252Baccess%252Bto%252Bin%252Bsilico%252Btools%252Bfor%252Bdrug%252Bdiscovery%26aulast%3DStork%26aufirst%3DConrad%26date%3D2019%26date%3D2019%26volume%3D53" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Angelica  Mazzolari</span>, <span class="hlFld-ContribAuthor ">Giulio  Vistoli</span>, <span class="hlFld-ContribAuthor ">Bernard  Testa</span>, <span class="hlFld-ContribAuthor ">Alessandro  Pedretti</span>. </span><span class="cited-content_cbyCitation_article-title">Prediction of the Formation of Reactive Metabolites by A Novel Classifier Approach Based on Enrichment Factor Optimization (EFO) as Implemented in the VEGA Program. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2018,</strong> <em>23 </em>
                                    (11)
                                     , 2955. <a href="https://doi.org/10.3390/molecules23112955" title="DOI URL">https://doi.org/10.3390/molecules23112955</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules23112955&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules23112955%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DPrediction%252Bof%252Bthe%252BFormation%252Bof%252BReactive%252BMetabolites%252Bby%252BA%252BNovel%252BClassifier%252BApproach%252BBased%252Bon%252BEnrichment%252BFactor%252BOptimization%252B%252528EFO%252529%252Bas%252BImplemented%252Bin%252Bthe%252BVEGA%252BProgram%26aulast%3DMazzolari%26aufirst%3DAngelica%26date%3D2018%26date%3D2018%26volume%3D23%26issue%3D11%26spage%3D2955" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01473/20180201/images/medium/jm-2017-01473n_0006.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01473/20180201/images/large/jm-2017-01473n_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01473&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01473/20180201/images/medium/jm-2017-01473n_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01473/20180201/images/large/jm-2017-01473n_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Structure of the implemented metabolism database with the logical links between the different data types which are organized in tables here represented as boxes colored according to the data type.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01473/20180201/images/large/jm-2017-01473n_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01473&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01473/20180201/images/medium/jm-2017-01473n_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01473/20180201/images/large/jm-2017-01473n_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Graphical user interface of MetaQSAR (top right window) which is organized in tabs by which the user can perform the main tasks. When a substrate is selected, its 3D structure is displayed in the main VEGA ZZ window (top left window), allowing an easy selection of the atoms involved in a given reaction. Notice that these reactive atoms remain highlighted by a blue sphere. Examples of molecular searches which can be carried out using the MetaQSAR interface by structure (bottom left window) or by properties (bottom right window).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01473/20180201/images/large/jm-2017-01473n_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01473&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01473/20180201/images/medium/jm-2017-01473n_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01473/20180201/images/large/jm-2017-01473n_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Distribution of metabolites according to reaction classes. The percentages shown in the figure refer to 4776 first-generation metabolic reactions = 100%. The color code is as follows: redox reactions, blue; hydrolyses, yellow; conjugations, red. Alternating dark and light fields are used simply for graphical clarity. The reaction classes are reported in numerical order starting from the upper-right quadrant. For clarity, the following very poorly populated classes are not shown: (9) redox reactions of other atoms (0.21%); (13) epoxide hydration (0.04%); (28) other conjugations (1.5%).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01473/20180201/images/large/jm-2017-01473n_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01473&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01473/20180201/images/medium/jm-2017-01473n_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01473/20180201/images/large/jm-2017-01473n_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Physicochemical profiling of substrates and metabolites as subdivided into the three main reaction classes and assessed by four representative descriptors such as molecular weight (MW), number of rotors, polar surface area (PSA), and virtual log <i>P</i> as computed by the MLP approach.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01473/20180201/images/large/jm-2017-01473n_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01473&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01473/20180201/images/medium/jm-2017-01473n_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01473/20180201/images/large/jm-2017-01473n_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Structure of the xenobiotics not included in the database and utilized in the exploratory predictive study with the involved atoms and the corresponding classes of the reported metabolic reactions (the blue numbers indicate the correctly predicted metabolic reactions, while the red numbers indicate the unpredicted reactions).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01473/20180201/images/large/jm-2017-01473n_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01473&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i21">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_81006" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_81006" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 30 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Aslam, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Basit, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nisar, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Khurshid, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rasool, M. H.</span><span> </span><span class="NLM_article-title">Proteomics: Technologies and Their Applications</span> <span class="citation_source-journal">J. Chromatogr. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">182</span><span class="NLM_x">–</span> <span class="NLM_lpage">196</span><span class="refDoi"> DOI: 10.1093/chromsci/bmw167</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01473&amp;key=10.1093%2Fchromsci%2Fbmw167" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01473&amp;key=28087761" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01473&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsFaktbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2017&pages=182-196&author=B.+Aslamauthor=M.+Basitauthor=M.+A.+Nisarauthor=M.+Khurshidauthor=M.+H.+Rasool&title=Proteomics%3A+Technologies+and+Their+Applications&doi=10.1093%2Fchromsci%2Fbmw167"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Proteomics: technologies and their applications</span></div><div class="casAuthors">Aslam, Bilal; Basit, Madiha; Nisar, Muhammad Atif; Khurshid, Mohsin; Rasool, Muhammad Hidayat</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chromatographic Science</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">182-196</span>CODEN:
                <span class="NLM_cas:coden">JCHSBZ</span>;
        ISSN:<span class="NLM_cas:issn">1945-239X</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Proteomics involves the applications of technologies for the identification and quantification of overall proteins present content of a cell, tissue or an organism.  It supplements the other "omics" technologies such as genomic and transcriptomics to expound the identity of proteins of an organism, and to cognize the structure and functions of a particular protein.  Proteomics-based technologies are utilized in various capacities for different research settings such as detection of various diagnostic markers, candidates for vaccine prodn., understanding pathogenicity mechanisms, alteration of expression patterns in response to different signals and interpretation of functional protein pathways in different diseases.  Proteomics is practically intricate because it includes the anal. and categorization of overall protein signatures of a genome.  Mass spectrometry with LC-MS-MS and MALDI-TOF/TOF being widely used equipment is the central among current proteomics.  However, utilization of proteomics facilities including the software for equipment, databases and the requirement of skilled personnel substantially increase the costs, therefore limit their wider use esp. in the developing world.  Furthermore, the proteome is highly dynamic because of complex regulatory systems that control the expression levels of proteins.  This review efforts to describe the various proteomics approaches, the recent developments and their application in research and anal.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoL--Tuk4G697Vg90H21EOLACvtfcHk0lhQXVAwYUCb9A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsFaktbk%253D&md5=0d69fa2515dfd694c147631d1c10c346</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1093%2Fchromsci%2Fbmw167&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fchromsci%252Fbmw167%26sid%3Dliteratum%253Aachs%26aulast%3DAslam%26aufirst%3DB.%26aulast%3DBasit%26aufirst%3DM.%26aulast%3DNisar%26aufirst%3DM.%2BA.%26aulast%3DKhurshid%26aufirst%3DM.%26aulast%3DRasool%26aufirst%3DM.%2BH.%26atitle%3DProteomics%253A%2520Technologies%2520and%2520Their%2520Applications%26jtitle%3DJ.%2520Chromatogr.%2520Sci.%26date%3D2017%26volume%3D55%26spage%3D182%26epage%3D196%26doi%3D10.1093%2Fchromsci%2Fbmw167" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Bourne, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lorsch, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Green, E. D.</span><span> </span><span class="NLM_article-title">Perspective: Sustaining the Big-Data Ecosystem</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">527</span><span class="NLM_x">, </span> <span class="NLM_fpage">S16</span><span class="NLM_x">–</span> <span class="NLM_lpage">S17</span><span class="refDoi"> DOI: 10.1038/527S16a</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01473&amp;key=10.1038%2F527S16a" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=527&publication_year=2015&pages=S16-S17&author=P.+E.+Bourneauthor=J.+R.+Lorschauthor=E.+D.+Green&title=Perspective%3A+Sustaining+the+Big-Data+Ecosystem&doi=10.1038%2F527S16a"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2F527S16a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F527S16a%26sid%3Dliteratum%253Aachs%26aulast%3DBourne%26aufirst%3DP.%2BE.%26aulast%3DLorsch%26aufirst%3DJ.%2BR.%26aulast%3DGreen%26aufirst%3DE.%2BD.%26atitle%3DPerspective%253A%2520Sustaining%2520the%2520Big-Data%2520Ecosystem%26jtitle%3DNature%26date%3D2015%26volume%3D527%26spage%3DS16%26epage%3DS17%26doi%3D10.1038%2F527S16a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Coveney, P. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dougherty, E. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Highfield, R. R.</span><span> </span><span class="NLM_article-title">Big Data Need Big Theory Too</span> <span class="citation_source-journal">Philos. Trans. R. Soc., A</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">374</span><span class="NLM_x"> (</span><span class="NLM_issue">2080</span><span class="NLM_x">) </span> <span class="NLM_fpage">20160153</span><span class="refDoi"> DOI: 10.1098/rsta.2016.0153</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01473&amp;key=10.1098%2Frsta.2016.0153" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=374&publication_year=2016&pages=20160153&issue=2080&author=P.+V.+Coveneyauthor=E.+R.+Doughertyauthor=R.+R.+Highfield&title=Big+Data+Need+Big+Theory+Too&doi=10.1098%2Frsta.2016.0153"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1098%2Frsta.2016.0153&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1098%252Frsta.2016.0153%26sid%3Dliteratum%253Aachs%26aulast%3DCoveney%26aufirst%3DP.%2BV.%26aulast%3DDougherty%26aufirst%3DE.%2BR.%26aulast%3DHighfield%26aufirst%3DR.%2BR.%26atitle%3DBig%2520Data%2520Need%2520Big%2520Theory%2520Too%26jtitle%3DPhilos.%2520Trans.%2520R.%2520Soc.%252C%2520A%26date%3D2016%26volume%3D374%26issue%3D2080%26spage%3D20160153%26doi%3D10.1098%2Frsta.2016.0153" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Bowling, F. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thomas, M.</span><span> </span><span class="NLM_article-title">Analyzing the Metabolome</span> <span class="citation_source-journal">Methods Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">1168</span><span class="NLM_x">, </span> <span class="NLM_fpage">31</span><span class="NLM_x">–</span> <span class="NLM_lpage">45</span><span class="refDoi"> DOI: 10.1007/978-1-4939-0847-9_3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01473&amp;key=10.1007%2F978-1-4939-0847-9_3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01473&amp;key=24870129" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01473&amp;key=1%3ACAS%3A528%3ADC%252BC2MXotVWku7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1168&publication_year=2014&pages=31-45&author=F.+G.+Bowlingauthor=M.+Thomas&title=Analyzing+the+Metabolome&doi=10.1007%2F978-1-4939-0847-9_3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Analyzing the Metabolome</span></div><div class="casAuthors">Bowling, Francis G.; Thomas, Mervyn</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Molecular Biology (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">1168</span>
        (<span class="NLM_cas:issue">Clinical Bioinformatics (2nd Edition)</span>),
    <span class="NLM_cas:pages">31-45</span>CODEN:
                <span class="NLM_cas:coden">MMBIED</span>;
        ISSN:<span class="NLM_cas:issn">1940-6029</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Metabolites, the chem. entities that are transformed during metab., provide a functional readout of cellular biochem. that offers the best prediction of the phenotype and the nature of a disease.  Mass spectrometry now allows thousands of metabolites to be quantitated.  The targeted or untargeted data from metabolic profiling can be combined with either supervised or unsupervised approaches to improve interpretation.  These sophisticated statistical techniques are computationally intensive.  This chapter reviews techniques applicable to metabolomics approaches to disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmzek7a2knh7Vg90H21EOLACvtfcHk0lhQXVAwYUCb9A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXotVWku7k%253D&md5=580348e236d0d39a35e20065d39b6028</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1007%2F978-1-4939-0847-9_3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-1-4939-0847-9_3%26sid%3Dliteratum%253Aachs%26aulast%3DBowling%26aufirst%3DF.%2BG.%26aulast%3DThomas%26aufirst%3DM.%26atitle%3DAnalyzing%2520the%2520Metabolome%26jtitle%3DMethods%2520Mol.%2520Biol.%26date%3D2014%26volume%3D1168%26spage%3D31%26epage%3D45%26doi%3D10.1007%2F978-1-4939-0847-9_3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Minkiewicz, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Darewicz, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iwaniak, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bucholska, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Starowicz, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Czyrko, E.</span><span> </span><span class="NLM_article-title">Internet Databases of the Properties, Enzymatic Reactions, and Metabolism of Small Molecules-Search Options and Applications in Food Science</span> <span class="citation_source-journal">Int. J. Mol. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x"> (</span><span class="NLM_issue">12</span><span class="NLM_x">) </span> <span class="NLM_fpage">2039</span><span class="NLM_x">–</span> <span class="NLM_lpage">2063</span><span class="refDoi"> DOI: 10.3390/ijms17122039</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01473&amp;key=10.3390%2Fijms17122039" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01473&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsFWntbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2016&pages=2039-2063&issue=12&author=P.+Minkiewiczauthor=M.+Darewiczauthor=A.+Iwaniakauthor=J.+Bucholskaauthor=P.+Starowiczauthor=E.+Czyrko&title=Internet+Databases+of+the+Properties%2C+Enzymatic+Reactions%2C+and+Metabolism+of+Small+Molecules-Search+Options+and+Applications+in+Food+Science&doi=10.3390%2Fijms17122039"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Internet databases of the properties, enzymatic reactions, and metabolism of small molecules-search options and applications in food science</span></div><div class="casAuthors">Minkiewicz, Piotr; Darewicz, Malgorzata; Iwaniak, Anna; Bucholska, Justyna; Starowicz, Piotr; Czyrko, Emilia</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Molecular Sciences</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2039/1-2039/24</span>CODEN:
                <span class="NLM_cas:coden">IJMCFK</span>;
        ISSN:<span class="NLM_cas:issn">1422-0067</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Internet databases of small mols., their enzymic reactions, and metab. have emerged as useful tools in food science.  Database searching is also introduced as part of chem. or enzymol. courses for food technol. students.  Such resources support the search for information about single compds. and facilitate the introduction of secondary analyses of large datasets.  Information can be retrieved from databases by searching for the compd. name or structure, annotating with the help of chem. codes or drawn using mol. editing software.  Data mining options may be enhanced by navigating through a network of links and cross-links between databases.  Exemplary databases reviewed in this article belong to two classes: tools concerning small mols. (including general and specialized databases annotating food components) and tools annotating enzymes and metab.  Some problems assocd. with database application are also discussed.  Data summarized in computer databases may be used for calcn. of daily intake of bioactive compds., prediction of metab. of food components, and their biol. activity as well as for prediction of interactions between food component and drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3vL4ql-iOArVg90H21EOLACvtfcHk0lhQXVAwYUCb9A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsFWntbc%253D&md5=d1423d8e85bb9f1dd5cd1a10982c5f7d</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.3390%2Fijms17122039&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fijms17122039%26sid%3Dliteratum%253Aachs%26aulast%3DMinkiewicz%26aufirst%3DP.%26aulast%3DDarewicz%26aufirst%3DM.%26aulast%3DIwaniak%26aufirst%3DA.%26aulast%3DBucholska%26aufirst%3DJ.%26aulast%3DStarowicz%26aufirst%3DP.%26aulast%3DCzyrko%26aufirst%3DE.%26atitle%3DInternet%2520Databases%2520of%2520the%2520Properties%252C%2520Enzymatic%2520Reactions%252C%2520and%2520Metabolism%2520of%2520Small%2520Molecules-Search%2520Options%2520and%2520Applications%2520in%2520Food%2520Science%26jtitle%3DInt.%2520J.%2520Mol.%2520Sci.%26date%3D2016%26volume%3D17%26issue%3D12%26spage%3D2039%26epage%3D2063%26doi%3D10.3390%2Fijms17122039" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Warr, W. A.</span><span> </span><span class="NLM_article-title">A Short Review of Chemical Reaction Database Systems, Computer-Aided Synthesis Design, Reaction Prediction and Synthetic Feasibility</span> <span class="citation_source-journal">Mol. Inf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">33</span><span class="NLM_x">, </span> <span class="NLM_fpage">469</span><span class="NLM_x">–</span> <span class="NLM_lpage">476</span><span class="refDoi"> DOI: 10.1002/minf.201400052</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01473&amp;key=10.1002%2Fminf.201400052" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01473&amp;key=27485985" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01473&amp;key=1%3ACAS%3A528%3ADC%252BC2cXovFCmur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2014&pages=469-476&author=W.+A.+Warr&title=A+Short+Review+of+Chemical+Reaction+Database+Systems%2C+Computer-Aided+Synthesis+Design%2C+Reaction+Prediction+and+Synthetic+Feasibility&doi=10.1002%2Fminf.201400052"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">A Short Review of Chemical Reaction Database Systems, Computer-Aided Synthesis Design, Reaction Prediction and Synthetic Feasibility</span></div><div class="casAuthors">Warr, Wendy A.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Informatics</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">6-7</span>),
    <span class="NLM_cas:pages">469-476</span>CODEN:
                <span class="NLM_cas:coden">MIONBS</span>;
        ISSN:<span class="NLM_cas:issn">1868-1743</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">This article is the text for a pedagogical lecture to be given at the Strasbourg Summer School in Chemoinformatics in June 2104.  It covers a very wide range of reaction topics including structure and reaction representation, reaction centers, atom-to-atom mapping, reaction retrieval systems, computer-aided synthesis design, retrosynthesis, reaction prediction and synthetic feasibility.  In the time available, the coverage of each topic can only be cursory; the main usefulness of this article to the research community is the extensive bibliog.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQWeDBafb6nrVg90H21EOLACvtfcHk0lhlXFL6owvxcA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXovFCmur8%253D&md5=392354d1d71f1ad31f1087d56b9afd57</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1002%2Fminf.201400052&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fminf.201400052%26sid%3Dliteratum%253Aachs%26aulast%3DWarr%26aufirst%3DW.%2BA.%26atitle%3DA%2520Short%2520Review%2520of%2520Chemical%2520Reaction%2520Database%2520Systems%252C%2520Computer-Aided%2520Synthesis%2520Design%252C%2520Reaction%2520Prediction%2520and%2520Synthetic%2520Feasibility%26jtitle%3DMol.%2520Inf.%26date%3D2014%26volume%3D33%26spage%3D469%26epage%3D476%26doi%3D10.1002%2Fminf.201400052" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Caspi, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dreher, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karp, P. D.</span><span> </span><span class="NLM_article-title">The Challenge of Constructing, Classifying, and Representing Metabolic Pathways</span> <span class="citation_source-journal">FEMS Microbiol. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">345</span><span class="NLM_x">, </span> <span class="NLM_fpage">85</span><span class="NLM_x">–</span> <span class="NLM_lpage">93</span><span class="refDoi"> DOI: 10.1111/1574-6968.12194</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01473&amp;key=10.1111%2F1574-6968.12194" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=345&publication_year=2013&pages=85-93&author=R.+Caspiauthor=K.+Dreherauthor=P.+D.+Karp&title=The+Challenge+of+Constructing%2C+Classifying%2C+and+Representing+Metabolic+Pathways&doi=10.1111%2F1574-6968.12194"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1111%2F1574-6968.12194&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252F1574-6968.12194%26sid%3Dliteratum%253Aachs%26aulast%3DCaspi%26aufirst%3DR.%26aulast%3DDreher%26aufirst%3DK.%26aulast%3DKarp%26aufirst%3DP.%2BD.%26atitle%3DThe%2520Challenge%2520of%2520Constructing%252C%2520Classifying%252C%2520and%2520Representing%2520Metabolic%2520Pathways%26jtitle%3DFEMS%2520Microbiol.%2520Lett.%26date%3D2013%26volume%3D345%26spage%3D85%26epage%3D93%26doi%3D10.1111%2F1574-6968.12194" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Gasteiger, J.</span><span> </span><span class="NLM_article-title">Explorations into Chemical Reactions and Biochemical Pathways</span> <span class="citation_source-journal">Mol. Inf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">35</span><span class="NLM_x">, </span> <span class="NLM_fpage">588</span><span class="NLM_x">–</span> <span class="NLM_lpage">592</span><span class="refDoi"> DOI: 10.1002/minf.201600038</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01473&amp;key=10.1002%2Fminf.201600038" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01473&amp;key=27870242" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01473&amp;key=1%3ACAS%3A528%3ADC%252BC28Xns1ejtb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2016&pages=588-592&author=J.+Gasteiger&title=Explorations+into+Chemical+Reactions+and+Biochemical+Pathways&doi=10.1002%2Fminf.201600038"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Explorations into Chemical Reactions and Biochemical Pathways</span></div><div class="casAuthors">Gasteiger, Johann</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Informatics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">11-12</span>),
    <span class="NLM_cas:pages">588-592</span>CODEN:
                <span class="NLM_cas:coden">MIONBS</span>;
        ISSN:<span class="NLM_cas:issn">1868-1743</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A brief overview of the work in the research group of the present author on extg. knowledge from chem. reaction data is presented.  Methods have been developed to calc. physicochem. effects at the reaction site.  It is shown that these physicochem. effects can quite favorably be used to derive equations for the calcn. of data on gas phase reactions and on reactions in soln. such as aq. acidity of alcs. or carboxylic acids or the hydrolysis of amides.  Furthermore, it is shown that these physicochem. effects are quite effective for assigning reactions into reaction classes that correspond to chem. knowledge.  Biochem. reactions constitute a particularly interesting and challenging task for increasing our understanding of living species.  The BioPath. Database is a rich source of information on biochem. reactions and has been used for a variety of applications of chem., biol., or medicinal interests.  Thus, it was shown that biochem. reactions can be assigned by the physicochem. effects into classes that correspond to the classification of enzymes by the EC nos.  Furthermore, 3D models of reaction intermediates can be used for searching for novel enzyme inhibitors.  It was shown in a combined application of chemoinformatics and bioinformatics that essential pathways of diseases can be uncovered.  Furthermore, a study showed that bacterial flavor-forming pathways can be discovered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoezOUVwDEas7Vg90H21EOLACvtfcHk0lhlXFL6owvxcA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xns1ejtb0%253D&md5=fcecba91b22a5a120f63ad70ab3b15c8</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1002%2Fminf.201600038&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fminf.201600038%26sid%3Dliteratum%253Aachs%26aulast%3DGasteiger%26aufirst%3DJ.%26atitle%3DExplorations%2520into%2520Chemical%2520Reactions%2520and%2520Biochemical%2520Pathways%26jtitle%3DMol.%2520Inf.%26date%3D2016%26volume%3D35%26spage%3D588%26epage%3D592%26doi%3D10.1002%2Fminf.201600038" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Karp, P. D.</span><span> </span><span class="NLM_article-title">Can We Replace Curation with Information Extraction Software?</span> <span class="citation_source-journal">Database</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">2016</span><span class="NLM_x">, </span> <span class="NLM_fpage">baw150</span><span class="refDoi"> DOI: 10.1093/database/baw150</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01473&amp;key=10.1093%2Fdatabase%2Fbaw150" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01473&amp;key=28025341" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2016&publication_year=2016&pages=baw150&author=P.+D.+Karp&title=Can+We+Replace+Curation+with+Information+Extraction+Software%3F&doi=10.1093%2Fdatabase%2Fbaw150"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1093%2Fdatabase%2Fbaw150&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fdatabase%252Fbaw150%26sid%3Dliteratum%253Aachs%26aulast%3DKarp%26aufirst%3DP.%2BD.%26atitle%3DCan%2520We%2520Replace%2520Curation%2520with%2520Information%2520Extraction%2520Software%253F%26jtitle%3DDatabase%26date%3D2016%26volume%3D2016%26spage%3Dbaw150%26doi%3D10.1093%2Fdatabase%2Fbaw150" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Hadadi, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hafner, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shajkofci, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zisaki, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hatzimanikatis, V.</span><span> </span><span class="NLM_article-title">ATLAS of Biochemistry: A Repository of All Possible Biochemical Reactions for Synthetic Biology and Metabolic Engineering Studies</span> <span class="citation_source-journal">ACS Synth. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">1155</span><span class="NLM_x">–</span> <span class="NLM_lpage">1166</span><span class="refDoi"> DOI: 10.1021/acssynbio.6b00054</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acssynbio.6b00054" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01473&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFGjs7vL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2016&pages=1155-1166&author=N.+Hadadiauthor=J.+Hafnerauthor=A.+Shajkofciauthor=A.+Zisakiauthor=V.+Hatzimanikatis&title=ATLAS+of+Biochemistry%3A+A+Repository+of+All+Possible+Biochemical+Reactions+for+Synthetic+Biology+and+Metabolic+Engineering+Studies&doi=10.1021%2Facssynbio.6b00054"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">ATLAS of Biochemistry: A Repository of All Possible Biochemical Reactions for Synthetic Biology and Metabolic Engineering Studies</span></div><div class="casAuthors">Hadadi, Noushin; Hafner, Jasmin; Shajkofci, Adrian; Zisaki, Aikaterini; Hatzimanikatis, Vassily</div><div class="citationInfo"><span class="NLM_cas:title">ACS Synthetic Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1155-1166</span>CODEN:
                <span class="NLM_cas:coden">ASBCD6</span>;
        ISSN:<span class="NLM_cas:issn">2161-5063</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Because the complexity of metab. cannot be intuitively understood or analyzed, computational methods are indispensable for studying biochem. and deepening our understanding of cellular metab. to promote new discoveries.  We used the computational framework BNICE.ch along with cheminformatic tools to assemble the whole theor. reactome from the known metabolome through expansion of the known biochem. presented in the Kyoto Encyclopedia of Genes and Genomes (KEGG) database.  We constructed the ATLAS of Biochem., a database of all theor. biochem. reactions based on known biochem. principles and compds.  ATLAS includes more than 130,000 hypothetical enzymic reactions that connect two or more KEGG metabolites through novel enzymic reactions that have never been reported to occur in living organisms.  Moreover, ATLAS reactions integrate 42% of KEGG metabolites that are not currently present in any KEGG reaction into one or more novel enzymic reactions.  The generated repository of information is organized in a Web-based database (http://lcsb-databases.epfl.ch/atlas/) that allows the user to search for all possible routes from any substrate compd. to any product.  The resulting pathways involve known and novel enzymic steps that may indicate unidentified enzymic activities and provide potential targets for protein engineering.  Our approach of introducing novel biochem. into pathway design and assocd. databases will be important for synthetic biol. and metabolic engineering.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxN_YUbnQ_9rVg90H21EOLACvtfcHk0lgYD9Ui7o_H5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFGjs7vL&md5=faf6d7855e475d2d30bf3544cac87420</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Facssynbio.6b00054&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facssynbio.6b00054%26sid%3Dliteratum%253Aachs%26aulast%3DHadadi%26aufirst%3DN.%26aulast%3DHafner%26aufirst%3DJ.%26aulast%3DShajkofci%26aufirst%3DA.%26aulast%3DZisaki%26aufirst%3DA.%26aulast%3DHatzimanikatis%26aufirst%3DV.%26atitle%3DATLAS%2520of%2520Biochemistry%253A%2520A%2520Repository%2520of%2520All%2520Possible%2520Biochemical%2520Reactions%2520for%2520Synthetic%2520Biology%2520and%2520Metabolic%2520Engineering%2520Studies%26jtitle%3DACS%2520Synth.%2520Biol.%26date%3D2016%26volume%3D5%26spage%3D1155%26epage%3D1166%26doi%3D10.1021%2Facssynbio.6b00054" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Pedretti, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Villa, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vistoli, G.</span><span> </span><span class="NLM_article-title">VEGA--An Open Platform to Develop Chemo-bio-informatics Applications, Using Plug-in Architecture and Script Programming</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">167</span><span class="NLM_x">–</span> <span class="NLM_lpage">173</span><span class="refDoi"> DOI: 10.1023/B:JCAM.0000035186.90683.f2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01473&amp;key=10.1023%2FB%3AJCAM.0000035186.90683.f2" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2004&pages=167-173&author=A.+Pedrettiauthor=L.+Villaauthor=G.+Vistoli&title=VEGA%2D%2DAn+Open+Platform+to+Develop+Chemo-bio-informatics+Applications%2C+Using+Plug-in+Architecture+and+Script+Programming&doi=10.1023%2FB%3AJCAM.0000035186.90683.f2"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1023%2FB%3AJCAM.0000035186.90683.f2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FB%253AJCAM.0000035186.90683.f2%26sid%3Dliteratum%253Aachs%26aulast%3DPedretti%26aufirst%3DA.%26aulast%3DVilla%26aufirst%3DL.%26aulast%3DVistoli%26aufirst%3DG.%26atitle%3DVEGA--An%2520Open%2520Platform%2520to%2520Develop%2520Chemo-bio-informatics%2520Applications%252C%2520Using%2520Plug-in%2520Architecture%2520and%2520Script%2520Programming%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2004%26volume%3D18%26spage%3D167%26epage%3D173%26doi%3D10.1023%2FB%3AJCAM.0000035186.90683.f2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Testa, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pedretti, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vistoli, G.</span><span> </span><span class="NLM_article-title">Reactions and Enzymes in the Metabolism of Drugs and Other Xenobiotics</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x"> (</span><span class="NLM_issue">11–12</span><span class="NLM_x">) </span> <span class="NLM_fpage">549</span><span class="NLM_x">–</span> <span class="NLM_lpage">560</span><span class="refDoi"> DOI: 10.1016/j.drudis.2012.01.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01473&amp;key=10.1016%2Fj.drudis.2012.01.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01473&amp;key=22305937" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01473&amp;key=1%3ACAS%3A528%3ADC%252BC38XnslOiur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2012&pages=549-560&issue=11%E2%80%9312&author=B.+Testaauthor=A.+Pedrettiauthor=G.+Vistoli&title=Reactions+and+Enzymes+in+the+Metabolism+of+Drugs+and+Other+Xenobiotics&doi=10.1016%2Fj.drudis.2012.01.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Reactions and enzymes in the metabolism of drugs and other xenobiotics</span></div><div class="casAuthors">Testa, Bernard; Pedretti, Alessandro; Vistoli, Giulio</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">11-12</span>),
    <span class="NLM_cas:pages">549-560</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">In this article, we offer an overview of the compared quant. importance of biotransformation reactions in the metab. of drugs and other xenobiotics, based on a meta-anal. of current research interests.  Also, we assess the relative significance the enzyme (super)families or categories catalyzing these reactions.  We put the facts unveiled by the anal. into a drug discovery context and draw some implications.  The results confirm the primary role of cytochrome P 450-catalyzed oxidns. and UDP-glucuronosyl-catalyzed glucuronidations, but they also document the marked significance of several other reactions.  Thus, there is a need for several drug discovery scientists to better grasp the variety of drug metab. reactions and enzymes and their consequences.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbRR72SBUfcLVg90H21EOLACvtfcHk0lgYD9Ui7o_H5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XnslOiur8%253D&md5=e11b5b1b7ea18e8bc817bff83efad456</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2012.01.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2012.01.017%26sid%3Dliteratum%253Aachs%26aulast%3DTesta%26aufirst%3DB.%26aulast%3DPedretti%26aufirst%3DA.%26aulast%3DVistoli%26aufirst%3DG.%26atitle%3DReactions%2520and%2520Enzymes%2520in%2520the%2520Metabolism%2520of%2520Drugs%2520and%2520Other%2520Xenobiotics%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2012%26volume%3D17%26issue%3D11%25E2%2580%259312%26spage%3D549%26epage%3D560%26doi%3D10.1016%2Fj.drudis.2012.01.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Bajusz, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rácz, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Héberger, K.</span><span> </span><span class="NLM_article-title">Why is Tanimoto Index an Appropriate Choice for Fingerprint-based Similarity Calculations?</span> <span class="citation_source-journal">J. Cheminf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">20</span><span class="refDoi"> DOI: 10.1186/s13321-015-0069-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01473&amp;key=10.1186%2Fs13321-015-0069-3" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2015&pages=20&author=D.+Bajuszauthor=A.+R%C3%A1czauthor=K.+H%C3%A9berger&title=Why+is+Tanimoto+Index+an+Appropriate+Choice+for+Fingerprint-based+Similarity+Calculations%3F&doi=10.1186%2Fs13321-015-0069-3"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1186%2Fs13321-015-0069-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs13321-015-0069-3%26sid%3Dliteratum%253Aachs%26aulast%3DBajusz%26aufirst%3DD.%26aulast%3DR%25C3%25A1cz%26aufirst%3DA.%26aulast%3DH%25C3%25A9berger%26aufirst%3DK.%26atitle%3DWhy%2520is%2520Tanimoto%2520Index%2520an%2520Appropriate%2520Choice%2520for%2520Fingerprint-based%2520Similarity%2520Calculations%253F%26jtitle%3DJ.%2520Cheminf.%26date%3D2015%26volume%3D7%26spage%3D20%26doi%3D10.1186%2Fs13321-015-0069-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Testa, B.; Krämer, S. D.</span> <span class="citation_source-book">The Biochemistry of Drug Metabolism, Vol. 1: Principles, Redox Reactions, Hydrolyses</span>; <span class="NLM_publisher-name">Wiley-VHCA and Wiley-VCH</span>: <span class="NLM_publisher-loc">Zurich and Weinheim</span>,<span class="NLM_x"> </span><span class="NLM_year">2008</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2008&author=B.+Testa&author=S.+D.+Kr%C3%A4mer&title=The+Biochemistry+of+Drug+Metabolism%2C+Vol.+1%3A+Principles%2C+Redox+Reactions%2C+Hydrolyses"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DTesta%26aufirst%3DB.%26btitle%3DThe%2520Biochemistry%2520of%2520Drug%2520Metabolism%252C%2520Vol.%25201%253A%2520Principles%252C%2520Redox%2520Reactions%252C%2520Hydrolyses%26pub%3DWiley-VHCA%2520and%2520Wiley-VCH%26date%3D2008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Testa, B.; Krämer, S. D.</span> <span class="citation_source-book">The Biochemistry of Drug Metabolism, Vol. 2: Conjugations, Consequences of Metabolism, Influencing Factors</span>; <span class="NLM_publisher-name">Wiley-VHCA and Wiley-VCH</span>: <span class="NLM_publisher-loc">Zurich and Weinheim</span>,<span class="NLM_x"> </span><span class="NLM_year">2010</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2010&author=B.+Testa&author=S.+D.+Kr%C3%A4mer&title=The+Biochemistry+of+Drug+Metabolism%2C+Vol.+2%3A+Conjugations%2C+Consequences+of+Metabolism%2C+Influencing+Factors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DTesta%26aufirst%3DB.%26btitle%3DThe%2520Biochemistry%2520of%2520Drug%2520Metabolism%252C%2520Vol.%25202%253A%2520Conjugations%252C%2520Consequences%2520of%2520Metabolism%252C%2520Influencing%2520Factors%26pub%3DWiley-VHCA%2520and%2520Wiley-VCH%26date%3D2010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Testa, B.</span><span> </span><span class="NLM_article-title">Principles of Drug Metabolism</span>. In  <span class="citation_source-book">Burger’s Medicinal Chemistry, Drug Discovery and Development</span>, <span class="NLM_edition">7th</span> ed.; <span class="NLM_contrib-group">Abraham, D. J.; Rotella, D. P.</span>, Eds.; <span class="NLM_publisher-name">Wiley</span>: <span class="NLM_publisher-loc">Hoboken, NJ</span>,<span class="NLM_x"> </span><span class="NLM_year">2010</span><span class="NLM_x">; </span>pp  <span class="NLM_fpage">127</span><span class="NLM_x">–</span> <span class="NLM_lpage">166</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01473&amp;key=10.1002%2F0471266949.bmc033.pub2" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2010&pages=127-166&author=B.+Testaauthor=D.+J.+Abraham&author=D.+P.+Rotella&title=Burger%E2%80%99s+Medicinal+Chemistry%2C+Drug+Discovery+and+Development"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1002%2F0471266949.bmc033.pub2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F0471266949.bmc033.pub2%26sid%3Dliteratum%253Aachs%26aulast%3DTesta%26aufirst%3DB.%26atitle%3DPrinciples%2520of%2520Drug%2520Metabolism%26btitle%3DBurger%25E2%2580%2599s%2520Medicinal%2520Chemistry%252C%2520Drug%2520Discovery%2520and%2520Development%26aulast%3DAbraham%26aufirst%3DD.%2BJ.%26pub%3DWiley%26date%3D2010%26spage%3D127%26epage%3D166" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Testa, B.; Clement, B.</span><span> </span><span class="NLM_article-title">Biotransformation Reactions and their Enzymes</span>. In  <span class="citation_source-book">The Practice of Medicinal Chemistry</span>, <span class="NLM_edition">4th</span> ed.; <span class="NLM_contrib-group">Wermuth, C. G.; Aldous, D.; Raboisson, P.; Rognan, D.</span>, Eds.; <span class="NLM_publisher-name">Academic Press</span>: <span class="NLM_publisher-loc">London</span>,<span class="NLM_x"> </span><span class="NLM_year">2015</span><span class="NLM_x">; </span>pp  <span class="NLM_fpage">561</span><span class="NLM_x">–</span> <span class="NLM_lpage">584</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01473&amp;key=10.1016%2FB978-0-12-417205-0.00024-9" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&pages=561-584&author=B.+Testa&author=B.+Clementauthor=C.+G.+Wermuth&author=D.+Aldous&author=P.+Raboisson&author=D.+Rognan&title=The+Practice+of+Medicinal+Chemistry"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2FB978-0-12-417205-0.00024-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1016%252FB978-0-12-417205-0.00024-9%26sid%3Dliteratum%253Aachs%26aulast%3DTesta%26aufirst%3DB.%26atitle%3DBiotransformation%2520Reactions%2520and%2520their%2520Enzymes%26btitle%3DThe%2520Practice%2520of%2520Medicinal%2520Chemistry%26aulast%3DWermuth%26aufirst%3DC.%2BG.%26pub%3DAcademic%2520Press%26date%3D2015%26spage%3D561%26epage%3D584" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Gaillard, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carrupt, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Testa, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boudon, A.</span><span> </span><span class="NLM_article-title">Molecular Lipophilicity Potential, a Tool in 3D-QSAR: Method and Applications</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">83</span><span class="NLM_x">–</span> <span class="NLM_lpage">96</span><span class="refDoi"> DOI: 10.1007/BF00119860</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01473&amp;key=10.1007%2FBF00119860" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=1994&pages=83-96&author=P.+Gaillardauthor=P.+A.+Carruptauthor=B.+Testaauthor=A.+Boudon&title=Molecular+Lipophilicity+Potential%2C+a+Tool+in+3D-QSAR%3A+Method+and+Applications&doi=10.1007%2FBF00119860"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1007%2FBF00119860&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FBF00119860%26sid%3Dliteratum%253Aachs%26aulast%3DGaillard%26aufirst%3DP.%26aulast%3DCarrupt%26aufirst%3DP.%2BA.%26aulast%3DTesta%26aufirst%3DB.%26aulast%3DBoudon%26aufirst%3DA.%26atitle%3DMolecular%2520Lipophilicity%2520Potential%252C%2520a%2520Tool%2520in%25203D-QSAR%253A%2520Method%2520and%2520Applications%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D1994%26volume%3D8%26spage%3D83%26epage%3D96%26doi%3D10.1007%2FBF00119860" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Marchant, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosser, E. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vessey, J. D.</span><span> </span><span class="NLM_article-title">A k-Nearest Neighbours Approach Using Metabolism-related Fingerprints to Improve In Silico Metabolite Ranking</span> <span class="citation_source-journal">Mol. Inf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">36</span><span class="NLM_x"> (</span><span class="NLM_issue">3</span><span class="NLM_x">) </span> <span class="NLM_fpage">1600105</span><span class="refDoi"> DOI: 10.1002/minf.201600105</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01473&amp;key=10.1002%2Fminf.201600105" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2017&pages=1600105&issue=3&author=C.+A.+Marchantauthor=E.+M.+Rosserauthor=J.+D.+Vessey&title=A+k-Nearest+Neighbours+Approach+Using+Metabolism-related+Fingerprints+to+Improve+In+Silico+Metabolite+Ranking&doi=10.1002%2Fminf.201600105"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1002%2Fminf.201600105&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fminf.201600105%26sid%3Dliteratum%253Aachs%26aulast%3DMarchant%26aufirst%3DC.%2BA.%26aulast%3DRosser%26aufirst%3DE.%2BM.%26aulast%3DVessey%26aufirst%3DJ.%2BD.%26atitle%3DA%2520k-Nearest%2520Neighbours%2520Approach%2520Using%2520Metabolism-related%2520Fingerprints%2520to%2520Improve%2520In%2520Silico%2520Metabolite%2520Ranking%26jtitle%3DMol.%2520Inf.%26date%3D2017%26volume%3D36%26issue%3D3%26spage%3D1600105%26doi%3D10.1002%2Fminf.201600105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Teffera, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berry, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brake, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lewis, R. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saffran, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moore, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, Z.</span><span> </span><span class="NLM_article-title">Impact of Hydrolysis-mediated Clearance on the Pharmacokinetics of Novel Anaplastic Lymphoma Kinase Inhibitors</span> <span class="citation_source-journal">Drug Metab. Dispos.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">41</span><span class="NLM_x">, </span> <span class="NLM_fpage">238</span><span class="NLM_x">–</span> <span class="NLM_lpage">247</span><span class="refDoi"> DOI: 10.1124/dmd.112.047993</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01473&amp;key=10.1124%2Fdmd.112.047993" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01473&amp;key=23118327" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01473&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlt1OntA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2013&pages=238-247&author=Y.+Tefferaauthor=L.+M.+Berryauthor=R.+L.+Brakeauthor=R.+T.+Lewisauthor=D.+C.+Saffranauthor=E.+Mooreauthor=J.+Liuauthor=Z.+Zhao&title=Impact+of+Hydrolysis-mediated+Clearance+on+the+Pharmacokinetics+of+Novel+Anaplastic+Lymphoma+Kinase+Inhibitors&doi=10.1124%2Fdmd.112.047993"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Impact of hydrolysis-mediated clearance on the pharmacokinetics of novel anaplastic lymphoma kinase inhibitors</span></div><div class="casAuthors">Teffera, Yohannes; Berry, Loren M.; Brake, Rachael L.; Lewis, Richard T.; Saffran, Douglas C.; Moore, Earl; Liu, Jingzhou; Zhao, Zhiyang</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism & Disposition</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">238-247</span>CODEN:
                <span class="NLM_cas:coden">DMDSAI</span>;
        ISSN:<span class="NLM_cas:issn">1521-009X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Compd. 1 [(E)-4-fluoro-N-(6-((4-(2-hydroxypropan-2-yl)piperidin-1-yl)methyl)-1-((1S,4S)-4-(isopropylcarbamoyl)cyclohexyl)-1H-benzo[d]imidazol-2(3H)-ylidene)benzamide], a new, potent, selective anaplastic lymphoma kinase (ALK) inhibitor with potential application for the treatment of cancer, was selected as candidate to advance into efficacy studies in mice.  However, the compd. underwent mouse-specific enzymic hydrolysis in plasma to a primary amine product (M1).  Subsequent i.v. pharmacokinetics studies in mice showed that compd. 1 had high clearance (CL) and a short half-life.  Oral dose escalation studies in mice indicated that elimination of compd. 1 was saturable, with higher doses achieving sufficient exposures above in vitro IC50.  Chem. efforts to minimize hydrolysis resulted in the discovery of several analogs that were stable in mouse plasma.  Three were taken in vivo into mice and showed decreased CL corresponding to increased in vitro stability in plasma.  However, the more stable compds. also showed reduced potency against ALK.  Kinetic studies in NADPH-fortified and unfortified microsomes and plasma produced submicromolar Km values and could help explain the satn. of elimination obsd. in vivo.  Predictions of CL based on kinetics from hydrolysis and NADPH-dependent pathways produced predicted hepatic CL values of 3.8, 3.0, 1.6, and 1.2 l/h·kg for compd. 1, compd. 2 [(E)-3,5-difluoro-N-(6-((4-(2-hydroxypropan-2-yl)piperidin-1-yl)methyl)-1-((1s,4s)-4-(isopropylcarbamoyl)cyclohexyl)-1H-benzo[d]imidazo-2(3H)-ylidene)benzamide], compd. 3 [(E)-3-chloro-5-fluoro-N-(6-((4-(2-hydroxypropan-2-yl)piperidin-1-yl)methyl)-1-((1s,4s)-4-(isopropylcarbamoyl)cyclohexyl)-1H-benzo[d]imidazol-2(3H)-ylidene)benzamide], and compd. 4 [(E)-N-(6-((4-(2-hydroxypropan-2-yl)piperidin-1-yl)methyl)-1-((1s,4s)-4-(isopropylcarbamoyl)cyclohexyl)-1H-benzo[d]imidazol-2(3H)-ylidene)-3-(trifluoromethyl)benzamide], resp.  The in vivo obsd. CLs for compds. 1, 2, 3, and 4 were 5.52, 3.51, 2.14, and 2.66 l/h kg, resp.  These results indicate that in vitro metab. kinetic data, incorporating contributions from both hydrolysis and NADPH-dependent metab., could be used to predict the systemic CL of compds. cleared via hydrolytic pathways provided that the in vitro assays thoroughly investigate the processes, including the contribution of other metabolic pathways and the possibility of satn. kinetics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokHx1C0KZVi7Vg90H21EOLACvtfcHk0ljV0wRvBOFfrA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXlt1OntA%253D%253D&md5=a46b21704e0ecab631a6048ca947fe01</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1124%2Fdmd.112.047993&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.112.047993%26sid%3Dliteratum%253Aachs%26aulast%3DTeffera%26aufirst%3DY.%26aulast%3DBerry%26aufirst%3DL.%2BM.%26aulast%3DBrake%26aufirst%3DR.%2BL.%26aulast%3DLewis%26aufirst%3DR.%2BT.%26aulast%3DSaffran%26aufirst%3DD.%2BC.%26aulast%3DMoore%26aufirst%3DE.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DZhao%26aufirst%3DZ.%26atitle%3DImpact%2520of%2520Hydrolysis-mediated%2520Clearance%2520on%2520the%2520Pharmacokinetics%2520of%2520Novel%2520Anaplastic%2520Lymphoma%2520Kinase%2520Inhibitors%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2013%26volume%3D41%26spage%3D238%26epage%3D247%26doi%3D10.1124%2Fdmd.112.047993" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Takahashi, R. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Choo, E. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Halladay, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deng, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rooney, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gates, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hop, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Khojasteh, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dresser, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Musib, L.</span><span> </span><span class="NLM_article-title">Absorption, Metabolism, Excretion, and the Contribution of Intestinal Metabolism to the Oral Disposition of [14C]Cobimetinib, a MEK Inhibitor, in Humans</span> <span class="citation_source-journal">Drug Metab. Dispos.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">44</span><span class="NLM_x">, </span> <span class="NLM_fpage">28</span><span class="NLM_x">–</span> <span class="NLM_lpage">39</span><span class="refDoi"> DOI: 10.1124/dmd.115.066282</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01473&amp;key=10.1124%2Fdmd.115.066282" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=28-39&author=R.+H.+Takahashiauthor=E.+F.+Chooauthor=S.+Maauthor=S.+Wongauthor=J.+Halladayauthor=Y.+Dengauthor=I.+Rooneyauthor=M.+Gatesauthor=C.+E.+Hopauthor=S.+C.+Khojastehauthor=M.+J.+Dresserauthor=L.+Musib&title=Absorption%2C+Metabolism%2C+Excretion%2C+and+the+Contribution+of+Intestinal+Metabolism+to+the+Oral+Disposition+of+%5B14C%5DCobimetinib%2C+a+MEK+Inhibitor%2C+in+Humans&doi=10.1124%2Fdmd.115.066282"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1124%2Fdmd.115.066282&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.115.066282%26sid%3Dliteratum%253Aachs%26aulast%3DTakahashi%26aufirst%3DR.%2BH.%26aulast%3DChoo%26aufirst%3DE.%2BF.%26aulast%3DMa%26aufirst%3DS.%26aulast%3DWong%26aufirst%3DS.%26aulast%3DHalladay%26aufirst%3DJ.%26aulast%3DDeng%26aufirst%3DY.%26aulast%3DRooney%26aufirst%3DI.%26aulast%3DGates%26aufirst%3DM.%26aulast%3DHop%26aufirst%3DC.%2BE.%26aulast%3DKhojasteh%26aufirst%3DS.%2BC.%26aulast%3DDresser%26aufirst%3DM.%2BJ.%26aulast%3DMusib%26aufirst%3DL.%26atitle%3DAbsorption%252C%2520Metabolism%252C%2520Excretion%252C%2520and%2520the%2520Contribution%2520of%2520Intestinal%2520Metabolism%2520to%2520the%2520Oral%2520Disposition%2520of%2520%255B14C%255DCobimetinib%252C%2520a%2520MEK%2520Inhibitor%252C%2520in%2520Humans%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2016%26volume%3D44%26spage%3D28%26epage%3D39%26doi%3D10.1124%2Fdmd.115.066282" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Iegre, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hayes, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thompson, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weidolf, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Isin, E. M.</span><span> </span><span class="NLM_article-title">Database Extraction of Metabolite Information of Drug Candidates: Analysis of 27 AstraZeneca Compounds with Human Absorption, Distribution, Metabolism, and Excretion Data</span> <span class="citation_source-journal">Drug Metab. Dispos.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">44</span><span class="NLM_x">, </span> <span class="NLM_fpage">732</span><span class="NLM_x">–</span> <span class="NLM_lpage">740</span><span class="refDoi"> DOI: 10.1124/dmd.115.067850</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01473&amp;key=10.1124%2Fdmd.115.067850" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=732-740&author=J.+Iegreauthor=M.+A.+Hayesauthor=R.+A.+Thompsonauthor=L.+Weidolfauthor=E.+M.+Isin&title=Database+Extraction+of+Metabolite+Information+of+Drug+Candidates%3A+Analysis+of+27+AstraZeneca+Compounds+with+Human+Absorption%2C+Distribution%2C+Metabolism%2C+and+Excretion+Data&doi=10.1124%2Fdmd.115.067850"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1124%2Fdmd.115.067850&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.115.067850%26sid%3Dliteratum%253Aachs%26aulast%3DIegre%26aufirst%3DJ.%26aulast%3DHayes%26aufirst%3DM.%2BA.%26aulast%3DThompson%26aufirst%3DR.%2BA.%26aulast%3DWeidolf%26aufirst%3DL.%26aulast%3DIsin%26aufirst%3DE.%2BM.%26atitle%3DDatabase%2520Extraction%2520of%2520Metabolite%2520Information%2520of%2520Drug%2520Candidates%253A%2520Analysis%2520of%252027%2520AstraZeneca%2520Compounds%2520with%2520Human%2520Absorption%252C%2520Distribution%252C%2520Metabolism%252C%2520and%2520Excretion%2520Data%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2016%26volume%3D44%26spage%3D732%26epage%3D740%26doi%3D10.1124%2Fdmd.115.067850" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Zhu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ho, M. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong, Y. H.</span><span> </span><span class="NLM_article-title">3-Methoxylphenylpropyl Amides as Novel Receptor Subtype-selective Melatoninergic Ligands: Characterization of Physicochemical and Pharmacokinetic Properties</span> <span class="citation_source-journal">Xenobiotica</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">41</span><span class="NLM_x"> (</span><span class="NLM_issue">1</span><span class="NLM_x">) </span> <span class="NLM_fpage">35</span><span class="NLM_x">–</span> <span class="NLM_lpage">45</span><span class="refDoi"> DOI: 10.3109/00498254.2010.524264</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01473&amp;key=10.3109%2F00498254.2010.524264" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2011&pages=35-45&issue=1&author=J.+Zhuauthor=Y.+Huauthor=M.+K.+Hoauthor=Y.+H.+Wong&title=3-Methoxylphenylpropyl+Amides+as+Novel+Receptor+Subtype-selective+Melatoninergic+Ligands%3A+Characterization+of+Physicochemical+and+Pharmacokinetic+Properties&doi=10.3109%2F00498254.2010.524264"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.3109%2F00498254.2010.524264&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3109%252F00498254.2010.524264%26sid%3Dliteratum%253Aachs%26aulast%3DZhu%26aufirst%3DJ.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DHo%26aufirst%3DM.%2BK.%26aulast%3DWong%26aufirst%3DY.%2BH.%26atitle%3D3-Methoxylphenylpropyl%2520Amides%2520as%2520Novel%2520Receptor%2520Subtype-selective%2520Melatoninergic%2520Ligands%253A%2520Characterization%2520of%2520Physicochemical%2520and%2520Pharmacokinetic%2520Properties%26jtitle%3DXenobiotica%26date%3D2011%26volume%3D41%26issue%3D1%26spage%3D35%26epage%3D45%26doi%3D10.3109%2F00498254.2010.524264" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Dinh, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pearce, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Van Haandel, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gaedigk, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leeder, J. S.</span><span> </span><span class="NLM_article-title">Characterization of Atomoxetine Biotransformation and Implications for Development of PBPK Models for Dose Individualization in Children</span> <span class="citation_source-journal">Drug Metab. Dispos.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">44</span><span class="NLM_x">, </span> <span class="NLM_fpage">1070</span><span class="NLM_x">–</span> <span class="NLM_lpage">1079</span><span class="refDoi"> DOI: 10.1124/dmd.116.069518</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01473&amp;key=10.1124%2Fdmd.116.069518" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=1070-1079&author=J.+C.+Dinhauthor=R.+E.+Pearceauthor=L.+Van+Haandelauthor=A.+Gaedigkauthor=J.+S.+Leeder&title=Characterization+of+Atomoxetine+Biotransformation+and+Implications+for+Development+of+PBPK+Models+for+Dose+Individualization+in+Children&doi=10.1124%2Fdmd.116.069518"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1124%2Fdmd.116.069518&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.116.069518%26sid%3Dliteratum%253Aachs%26aulast%3DDinh%26aufirst%3DJ.%2BC.%26aulast%3DPearce%26aufirst%3DR.%2BE.%26aulast%3DVan%2BHaandel%26aufirst%3DL.%26aulast%3DGaedigk%26aufirst%3DA.%26aulast%3DLeeder%26aufirst%3DJ.%2BS.%26atitle%3DCharacterization%2520of%2520Atomoxetine%2520Biotransformation%2520and%2520Implications%2520for%2520Development%2520of%2520PBPK%2520Models%2520for%2520Dose%2520Individualization%2520in%2520Children%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2016%26volume%3D44%26spage%3D1070%26epage%3D1079%26doi%3D10.1124%2Fdmd.116.069518" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Dickinson, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cantarini, M. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Collier, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frewer, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pickup, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ballard, P.</span><span> </span><span class="NLM_article-title">Metabolic Disposition of Osimertinib in Rats, Dogs, and Humans: Insights into a Drug Designed to Bind Covalently to a Cysteine Residue of Epidermal Growth Factor Receptor</span> <span class="citation_source-journal">Drug Metab. Dispos.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">44</span><span class="NLM_x">, </span> <span class="NLM_fpage">1201</span><span class="NLM_x">–</span> <span class="NLM_lpage">1212</span><span class="refDoi"> DOI: 10.1124/dmd.115.069203</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01473&amp;key=10.1124%2Fdmd.115.069203" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01473&amp;key=27226351" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01473&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVajtb%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=1201-1212&author=P.+A.+Dickinsonauthor=M.+V.+Cantariniauthor=J.+Collierauthor=P.+Frewerauthor=S.+Martinauthor=K.+Pickupauthor=P.+Ballard&title=Metabolic+Disposition+of+Osimertinib+in+Rats%2C+Dogs%2C+and+Humans%3A+Insights+into+a+Drug+Designed+to+Bind+Covalently+to+a+Cysteine+Residue+of+Epidermal+Growth+Factor+Receptor&doi=10.1124%2Fdmd.115.069203"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Metabolic disposition of osimertinib in rats, dogs, and humans: insights into a drug designed to bind covalently to a cysteine residue of epidermal growth factor receptor</span></div><div class="casAuthors">Dickinson, Paul A.; Cantarini, Mireille V.; Collier, Jo; Frewer, Paul; Martin, Scott; Pickup, Kathryn; Ballard, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism & Disposition</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1201-1212</span>CODEN:
                <span class="NLM_cas:coden">DMDSAI</span>;
        ISSN:<span class="NLM_cas:issn">1521-009X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Preclin. and clin. studies were conducted to det. the metab. and pharmacokinetics of osimertinib and key metabolites AZ5104 and AZ7550.  Osimertinib was designed to covalently bind to epidermal growth factor receptors, allowing it to achieve nanomolar cellular potency.  Covalent binding was obsd. in incubations of radiolabeled osimertinib with human and rat hepatocytes, human and rat plasma, and human serum albumin.  Osimertinib, AZ5104, and AZ7550 were predominantly metabolized by CYP3A.  Seven metabolites were detected in human hepatocytes, also obsd. in rat or dog hepatocytes at similar or higher levels.  After oral administration of radiolabeled osimertinib to rats, drug-related material was widely distributed, with the highest radioactivity concns. measured at 6 h postdose in most tissues; radioactivity was detectable in 42% of tissues 60 days postdose.  Concns. of [14C]-radioactivity in blood were lower than in most tissues.  After the administration of a single oral dose of 20 mg of radiolabeled osimertinib to healthy male volunteers, ∼19% of the dose was recovered by 3 days postdose.  At 84 days postdose, mean total radioactivity recovery was 14.2% and 67.8% of the dose in urine and feces.  The most abundant metabolite identified in feces was AZ5104 (∼6% of dose).  Osimertinib accounted for ∼1% of total radioactivity in the plasma of non-small cell lung cancer patients after 22 days of 80-mg osimertinib once-daily treatment; the most abundant circulatory metabolites were AZ7550 and AZ5104 (<10% of total osimertinib-related material).  Osimertinib is extensively distributed and metabolized in humans and is eliminated primarily via the fecal route.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqslCW8h-RI9bVg90H21EOLACvtfcHk0lhPFw59CPGkjg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitVajtb%252FO&md5=4115de7691a3b39eec091660b5e5b1d6</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1124%2Fdmd.115.069203&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.115.069203%26sid%3Dliteratum%253Aachs%26aulast%3DDickinson%26aufirst%3DP.%2BA.%26aulast%3DCantarini%26aufirst%3DM.%2BV.%26aulast%3DCollier%26aufirst%3DJ.%26aulast%3DFrewer%26aufirst%3DP.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DPickup%26aufirst%3DK.%26aulast%3DBallard%26aufirst%3DP.%26atitle%3DMetabolic%2520Disposition%2520of%2520Osimertinib%2520in%2520Rats%252C%2520Dogs%252C%2520and%2520Humans%253A%2520Insights%2520into%2520a%2520Drug%2520Designed%2520to%2520Bind%2520Covalently%2520to%2520a%2520Cysteine%2520Residue%2520of%2520Epidermal%2520Growth%2520Factor%2520Receptor%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2016%26volume%3D44%26spage%3D1201%26epage%3D1212%26doi%3D10.1124%2Fdmd.115.069203" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Lee, J. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Choi, Y. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oh, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chi, Y. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paik, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, K. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jung, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ryu, C. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, K. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, D. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yoon, Y. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, S. K.</span><span> </span><span class="NLM_article-title">Characterization of Fimasartan Metabolites in Human Liver Microsomes and Human Plasma</span> <span class="citation_source-journal">Xenobiotica</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">40</span><span class="NLM_x">–</span> <span class="NLM_lpage">51</span><span class="refDoi"> DOI: 10.3109/00498254.2015.1047429</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01473&amp;key=10.3109%2F00498254.2015.1047429" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2016&pages=40-51&author=J.+Y.+Leeauthor=Y.+J.+Choiauthor=S.+J.+Ohauthor=Y.+H.+Chiauthor=S.+H.+Paikauthor=K.+H.+Leeauthor=J.+K.+Jungauthor=C.+S.+Ryuauthor=K.+B.+Kimauthor=D.+H.+Kimauthor=Y.+R.+Yoonauthor=S.+K.+Kim&title=Characterization+of+Fimasartan+Metabolites+in+Human+Liver+Microsomes+and+Human+Plasma&doi=10.3109%2F00498254.2015.1047429"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.3109%2F00498254.2015.1047429&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3109%252F00498254.2015.1047429%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DJ.%2BY.%26aulast%3DChoi%26aufirst%3DY.%2BJ.%26aulast%3DOh%26aufirst%3DS.%2BJ.%26aulast%3DChi%26aufirst%3DY.%2BH.%26aulast%3DPaik%26aufirst%3DS.%2BH.%26aulast%3DLee%26aufirst%3DK.%2BH.%26aulast%3DJung%26aufirst%3DJ.%2BK.%26aulast%3DRyu%26aufirst%3DC.%2BS.%26aulast%3DKim%26aufirst%3DK.%2BB.%26aulast%3DKim%26aufirst%3DD.%2BH.%26aulast%3DYoon%26aufirst%3DY.%2BR.%26aulast%3DKim%26aufirst%3DS.%2BK.%26atitle%3DCharacterization%2520of%2520Fimasartan%2520Metabolites%2520in%2520Human%2520Liver%2520Microsomes%2520and%2520Human%2520Plasma%26jtitle%3DXenobiotica%26date%3D2016%26volume%3D46%26spage%3D40%26epage%3D51%26doi%3D10.3109%2F00498254.2015.1047429" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Rioux, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duncan, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lantz, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miao, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chan-Penebre, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moyer, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Munchhof, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Copeland, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chesworth, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waters, N. J.</span><span> </span><span class="NLM_article-title">Species Differences in Metabolism of EPZ015666, an Oxetane-containing Protein Arginine Methyltransferase-5 (PRMT5) Inhibitor</span> <span class="citation_source-journal">Xenobiotica</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">268</span><span class="NLM_x">–</span> <span class="NLM_lpage">277</span><span class="refDoi"> DOI: 10.3109/00498254.2015.1072253</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01473&amp;key=10.3109%2F00498254.2015.1072253" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01473&amp;key=26294260" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01473&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFKlu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2016&pages=268-277&author=N.+Riouxauthor=K.+W.+Duncanauthor=R.+J.+Lantzauthor=X.+Miaoauthor=E.+Chan-Penebreauthor=M.+P.+Moyerauthor=M.+J.+Munchhofauthor=R.+A.+Copelandauthor=R.+Chesworthauthor=N.+J.+Waters&title=Species+Differences+in+Metabolism+of+EPZ015666%2C+an+Oxetane-containing+Protein+Arginine+Methyltransferase-5+%28PRMT5%29+Inhibitor&doi=10.3109%2F00498254.2015.1072253"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Species differences in metabolism of EPZ015666, an oxetane-containing protein arginine methyltransferase-5 (PRMT5) inhibitor</span></div><div class="casAuthors">Rioux, Nathalie; Duncan, Kenneth W.; Lantz, Ronald J.; Miao, Xiusheng; Chan-Penebre, Elayne; Moyer, Mikel P.; Munchhof, Michael J.; Copeland, Robert A.; Chesworth, Richard; Waters, Nigel J.</div><div class="citationInfo"><span class="NLM_cas:title">Xenobiotica</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">268-277</span>CODEN:
                <span class="NLM_cas:coden">XENOBH</span>;
        ISSN:<span class="NLM_cas:issn">0049-8254</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">1. Metabolite profiling and identification studies were conducted to understand the cross-species differences in the metabolic clearance of EPZ015666, a first-in-class protein arginine methyltransferase-5 (PRMT5) inhibitor, with anti-proliferative effects in preclin. models of Mantle Cell Lymphoma.  EPZ015666 exhibited low clearance in human, mouse and rat liver microsomes, in part by introduction of a 3-substituted oxetane ring on the mol.  In contrast, a higher clearance was obsd. in dog liver microsomes (DLM) that translated to a higher in vivo clearance in dog compared with rodent.2.  Structure elucidation via high resoln., accurate mass LC-MSn revealed that the prominent metabolites of EPZ015666 were present in hepatocytes from all species, with the highest turnover rate in dogs.  M1 and M2 resulted from oxidative oxetane ring scission, whereas M3 resulted from loss of the oxetane ring via an N-dealkylation reaction.3.  The formation of M1 and M2 in DLM was significantly abrogated in the presence of the specific CYP2D inhibitor, quinidine, and to a lesser extent by the CYP3A inhibitor, ketoconazole, corroborating data from human recombinant isoenzymes.4.  Our data indicate a marked species difference in the metab. of the PRMT5 inhibitor EPZ015666, with oxetane ring scission the predominant metabolic pathway in dog mediated largely by CYP2D.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq1gD_mblcnKbVg90H21EOLACvtfcHk0lhPFw59CPGkjg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFKlu7w%253D&md5=489924f6c228fbb431d6b7257e5b9d9a</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.3109%2F00498254.2015.1072253&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3109%252F00498254.2015.1072253%26sid%3Dliteratum%253Aachs%26aulast%3DRioux%26aufirst%3DN.%26aulast%3DDuncan%26aufirst%3DK.%2BW.%26aulast%3DLantz%26aufirst%3DR.%2BJ.%26aulast%3DMiao%26aufirst%3DX.%26aulast%3DChan-Penebre%26aufirst%3DE.%26aulast%3DMoyer%26aufirst%3DM.%2BP.%26aulast%3DMunchhof%26aufirst%3DM.%2BJ.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DChesworth%26aufirst%3DR.%26aulast%3DWaters%26aufirst%3DN.%2BJ.%26atitle%3DSpecies%2520Differences%2520in%2520Metabolism%2520of%2520EPZ015666%252C%2520an%2520Oxetane-containing%2520Protein%2520Arginine%2520Methyltransferase-5%2520%2528PRMT5%2529%2520Inhibitor%26jtitle%3DXenobiotica%26date%3D2016%26volume%3D46%26spage%3D268%26epage%3D277%26doi%3D10.3109%2F00498254.2015.1072253" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Xie, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Du, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, F.</span><span> </span><span class="NLM_article-title">Structural Elucidation of in Vitro Metabolites of Bavachinin in Rat Liver Microsomes by LC-ESI-MSn and Chemical Synthesis</span> <span class="citation_source-journal">Xenobiotica</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">296</span><span class="NLM_x">–</span> <span class="NLM_lpage">306</span><span class="refDoi"> DOI: 10.3109/00498254.2015.1074763</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01473&amp;key=10.3109%2F00498254.2015.1074763" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2016&pages=296-306&author=F.+Xieauthor=G.+Duauthor=S.+Maauthor=Y.+Liauthor=R.+Wangauthor=F.+Guo&title=Structural+Elucidation+of+in+Vitro+Metabolites+of+Bavachinin+in+Rat+Liver+Microsomes+by+LC-ESI-MSn+and+Chemical+Synthesis&doi=10.3109%2F00498254.2015.1074763"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.3109%2F00498254.2015.1074763&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3109%252F00498254.2015.1074763%26sid%3Dliteratum%253Aachs%26aulast%3DXie%26aufirst%3DF.%26aulast%3DDu%26aufirst%3DG.%26aulast%3DMa%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DR.%26aulast%3DGuo%26aufirst%3DF.%26atitle%3DStructural%2520Elucidation%2520of%2520in%2520Vitro%2520Metabolites%2520of%2520Bavachinin%2520in%2520Rat%2520Liver%2520Microsomes%2520by%2520LC-ESI-MSn%2520and%2520Chemical%2520Synthesis%26jtitle%3DXenobiotica%26date%3D2016%26volume%3D46%26spage%3D296%26epage%3D306%26doi%3D10.3109%2F00498254.2015.1074763" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">Sanga, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banach, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ledvina, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Modi, N. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mittur, A.</span><span> </span><span class="NLM_article-title">Pharmacokinetics, Metabolism, and Excretion of Nefopam, a Dual Reuptake Inhibitor in Healthy Male Volunteers</span> <span class="citation_source-journal">Xenobiotica</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">1001</span><span class="NLM_x">–</span> <span class="NLM_lpage">1016</span><span class="refDoi"> DOI: 10.3109/00498254.2015.1136989</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01473&amp;key=10.3109%2F00498254.2015.1136989" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2016&pages=1001-1016&author=M.+Sangaauthor=J.+Banachauthor=A.+Ledvinaauthor=N.+B.+Modiauthor=A.+Mittur&title=Pharmacokinetics%2C+Metabolism%2C+and+Excretion+of+Nefopam%2C+a+Dual+Reuptake+Inhibitor+in+Healthy+Male+Volunteers&doi=10.3109%2F00498254.2015.1136989"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.3109%2F00498254.2015.1136989&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3109%252F00498254.2015.1136989%26sid%3Dliteratum%253Aachs%26aulast%3DSanga%26aufirst%3DM.%26aulast%3DBanach%26aufirst%3DJ.%26aulast%3DLedvina%26aufirst%3DA.%26aulast%3DModi%26aufirst%3DN.%2BB.%26aulast%3DMittur%26aufirst%3DA.%26atitle%3DPharmacokinetics%252C%2520Metabolism%252C%2520and%2520Excretion%2520of%2520Nefopam%252C%2520a%2520Dual%2520Reuptake%2520Inhibitor%2520in%2520Healthy%2520Male%2520Volunteers%26jtitle%3DXenobiotica%26date%3D2016%26volume%3D46%26spage%3D1001%26epage%3D1016%26doi%3D10.3109%2F00498254.2015.1136989" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Nijenhuis, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lucas, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosing, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robertson, P.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hellriegel, E. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schellens, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beijnen, A. J.</span><span> </span><span class="NLM_article-title">Metabolite Profiling of 14C-Omacetaxine Mepesuccinate in Plasma and Excreta of Cancer Patients</span> <span class="citation_source-journal">Xenobiotica</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">1122</span><span class="NLM_x">–</span> <span class="NLM_lpage">1132</span><span class="refDoi"> DOI: 10.3109/00498254.2016.1152418</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01473&amp;key=10.3109%2F00498254.2016.1152418" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2016&pages=1122-1132&author=C.+M.+Nijenhuisauthor=L.+Lucasauthor=H.+Rosingauthor=P.+Robertsonauthor=E.+T.+Hellriegelauthor=J.+H.+Schellensauthor=A.+J.+Beijnen&title=Metabolite+Profiling+of+14C-Omacetaxine+Mepesuccinate+in+Plasma+and+Excreta+of+Cancer+Patients&doi=10.3109%2F00498254.2016.1152418"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.3109%2F00498254.2016.1152418&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3109%252F00498254.2016.1152418%26sid%3Dliteratum%253Aachs%26aulast%3DNijenhuis%26aufirst%3DC.%2BM.%26aulast%3DLucas%26aufirst%3DL.%26aulast%3DRosing%26aufirst%3DH.%26aulast%3DRobertson%26aufirst%3DP.%26aulast%3DHellriegel%26aufirst%3DE.%2BT.%26aulast%3DSchellens%26aufirst%3DJ.%2BH.%26aulast%3DBeijnen%26aufirst%3DA.%2BJ.%26atitle%3DMetabolite%2520Profiling%2520of%252014C-Omacetaxine%2520Mepesuccinate%2520in%2520Plasma%2520and%2520Excreta%2520of%2520Cancer%2520Patients%26jtitle%3DXenobiotica%26date%3D2016%26volume%3D46%26spage%3D1122%26epage%3D1132%26doi%3D10.3109%2F00498254.2016.1152418" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.7b01473&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2018.61.issue-3%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.7b01473%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.7b01473" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                3MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"67c7975abb4e3c88","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
