
<html lang="en"     class="pb-page"  data-request-id="eb147553-9c8f-40f0-b8f2-ee915178acc9"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;issue:issue:10.1021/jmcmar.2020.63.issue-20;pageGroup:string:Publication Pages;article:article:10.1021/acs.jmedchem.0c01500;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Automated Design of Macrocycles for Therapeutic Applications: From Small Molecules to Peptides and Proteins" /></meta><meta name="dc.Creator" content="Dan  Sindhikara" /></meta><meta name="dc.Creator" content="Michael  Wagner" /></meta><meta name="dc.Creator" content="Paraskevi  Gkeka" /></meta><meta name="dc.Creator" content="Stefan  Güssregen" /></meta><meta name="dc.Creator" content="Garima  Tiwari" /></meta><meta name="dc.Creator" content="Gerhard  Hessler" /></meta><meta name="dc.Creator" content="Engin  Yapici" /></meta><meta name="dc.Creator" content="Ziyu  Li" /></meta><meta name="dc.Creator" content="Andreas  Evers" /></meta><meta name="dc.Description" content="Macrocycles and cyclic peptides are increasingly attractive therapeutic modalities as they often have improved affinity, are able to bind to extended protein surfaces, and otherwise have favorable ..." /></meta><meta name="Description" content="Macrocycles and cyclic peptides are increasingly attractive therapeutic modalities as they often have improved affinity, are able to bind to extended protein surfaces, and otherwise have favorable ..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="October 5, 2020" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c01500" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2020 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c01500" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c01500" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c01500" /></link>
        
    
    

<title>Automated Design of Macrocycles for Therapeutic Applications: From Small Molecules to Peptides and Proteins | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c01500" /></meta><meta property="og:title" content="Automated Design of Macrocycles for Therapeutic Applications: From Small Molecules to Peptides and Proteins" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01500/20201019/images/large/jm0c01500_0014.jpeg" /></meta><meta property="og:description" content="Macrocycles and cyclic peptides are increasingly attractive therapeutic modalities as they often have improved affinity, are able to bind to extended protein surfaces, and otherwise have favorable properties. Macrocyclization of a known binder may stabilize its bioactive conformation and improve its metabolic stability, cell permeability, and in certain cases oral bioavailability. Herein, we present implementation and application of an approach that automatically generates, evaluates, and proposes cyclizations utilizing a library of well-established chemical reactions and reagents. Using the three-dimensional (3D) conformation of the linear molecule in complex with a target protein as the starting point, this approach identifies attachment points, generates linkers, evaluates their geometric compatibility, and ranks the resulting molecules with respect to their predicted conformational stability and interactions with the target protein. As we show here with prospective and retrospective case studies, this procedure can be applied for the macrocyclization of small molecules and peptides and even PROteolysis TArgeting Chimeras (PROTACs) and proteins." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c01500"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01500">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c01500&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c01500&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c01500&amp;href=/doi/10.1021/acs.jmedchem.0c01500" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2020</span><span class="cit-fg-volume">, 63</span><span class="cit-fg-issue">, 20</span><span class="cit-fg-pageRange">, 12100-12115</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/63/20" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.0c01454" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.0c01541" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Automated Design of Macrocycles for Therapeutic Applications: From Small Molecules to Peptides and Proteins</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Dan Sindhikara</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Dan Sindhikara</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Schrodinger, Inc., 120 West 45th Street, New York, New York 10036, United States</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#d094b1befe83b9beb4b8b9bbb1a2b190a3b3b8a2bfb4b9beb7b5a2feb3bfbd"><span class="__cf_email__" data-cfemail="d094b1befe83b9beb4b8b9bbb1a2b190a3b3b8a2bfb4b9beb7b5a2feb3bfbd">[email protected]</span></a>. Tel: +1 212 548 2371.</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Dan++Sindhikara">Dan Sindhikara</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Michael Wagner</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Michael Wagner</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Integrated Drug Discovery, Sanofi-Aventis Deutschland GmbH, Industriepark Hoechst, 65926 Frankfurt am Main, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Michael++Wagner">Michael Wagner</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Paraskevi Gkeka</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Paraskevi Gkeka</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Integrated Drug Discovery, Sanofi R&D, 1 Avenue Pierre Brossolette, 91385 Chilly-Mazarin, France</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Paraskevi++Gkeka">Paraskevi Gkeka</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-0752-3539" title="Orcid link">http://orcid.org/0000-0002-0752-3539</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Stefan Güssregen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Stefan Güssregen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Integrated Drug Discovery, Sanofi-Aventis Deutschland GmbH, Industriepark Hoechst, 65926 Frankfurt am Main, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Stefan++G%C3%BCssregen">Stefan Güssregen</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-1868-9614" title="Orcid link">http://orcid.org/0000-0002-1868-9614</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Garima Tiwari</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Garima Tiwari</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Integrated Drug Discovery, Sanofi-Aventis Deutschland GmbH, Industriepark Hoechst, 65926 Frankfurt am Main, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Garima++Tiwari">Garima Tiwari</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Gerhard Hessler</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Gerhard Hessler</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Integrated Drug Discovery, Sanofi-Aventis Deutschland GmbH, Industriepark Hoechst, 65926 Frankfurt am Main, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Gerhard++Hessler">Gerhard Hessler</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Engin Yapici</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Engin Yapici</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Schrodinger, Inc., 120 West 45th Street, New York, New York 10036, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Engin++Yapici">Engin Yapici</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ziyu Li</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ziyu Li</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Integrated Drug Discovery, Sanofi-Aventis Deutschland GmbH, Industriepark Hoechst, 65926 Frankfurt am Main, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ziyu++Li">Ziyu Li</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Andreas Evers</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Andreas Evers</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Integrated Drug Discovery, Sanofi-Aventis Deutschland GmbH, Industriepark Hoechst, 65926 Frankfurt am Main, Germany</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#a9e8c7cddbccc8da87ecdfccdbdae9c4ccdbcac2cedbc6dcd987cac6c4"><span class="__cf_email__" data-cfemail="25644b41574044560b605340575665484057464e42574a50550b464a48">[email protected]</span></a></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Andreas++Evers">Andreas Evers</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-4643-1941" title="Orcid link">http://orcid.org/0000-0003-4643-1941</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c01500&amp;href=/doi/10.1021%2Facs.jmedchem.0c01500" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2020</span></span><span class="cit-volume">, 63</span><span class="cit-issue">, 20</span><span class="cit-pageRange">, 12100–12115</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">October 5, 2020</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>27 August 2020</li><li><span class="item_label"><b>Published</b> online</span>5 October 2020</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 22 October 2020</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c01500" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01500</a></div><div class="article_header-article-copyright"><strong>Copyright © 2020 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D12100%26pageCount%3D16%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DDan%2BSindhikara%252C%2BMichael%2BWagner%252C%2BParaskevi%2BGkeka%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D63%26issueNum%3D20%26contentID%3Dacs.jmedchem.0c01500%26title%3DAutomated%2BDesign%2Bof%2BMacrocycles%2Bfor%2BTherapeutic%2BApplications%253A%2BFrom%2BSmall%2BMolecules%2Bto%2BPeptides%2Band%2BProteins%26numPages%3D16%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D12115%26publicationDate%3DOctober%2B2020">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c01500"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">2347</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">2</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c01500" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Automated Design of Macrocycles for Therapeutic Applications: From Small Molecules to Peptides and Proteins&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Dan&quot;,&quot;last_name&quot;:&quot;Sindhikara&quot;},{&quot;first_name&quot;:&quot;Michael&quot;,&quot;last_name&quot;:&quot;Wagner&quot;},{&quot;first_name&quot;:&quot;Paraskevi&quot;,&quot;last_name&quot;:&quot;Gkeka&quot;},{&quot;first_name&quot;:&quot;Stefan&quot;,&quot;last_name&quot;:&quot;Güssregen&quot;},{&quot;first_name&quot;:&quot;Garima&quot;,&quot;last_name&quot;:&quot;Tiwari&quot;},{&quot;first_name&quot;:&quot;Gerhard&quot;,&quot;last_name&quot;:&quot;Hessler&quot;},{&quot;first_name&quot;:&quot;Engin&quot;,&quot;last_name&quot;:&quot;Yapici&quot;},{&quot;first_name&quot;:&quot;Ziyu&quot;,&quot;last_name&quot;:&quot;Li&quot;},{&quot;first_name&quot;:&quot;Andreas&quot;,&quot;last_name&quot;:&quot;Evers&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2020&quot;,&quot;month&quot;:&quot;10&quot;,&quot;day&quot;:&quot;05&quot;,&quot;issue&quot;:&quot;20&quot;,&quot;volume&quot;:&quot;63&quot;,&quot;pages&quot;:&quot;12100-12115&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c01500&quot;},&quot;abstract&quot;:&quot;Macrocycles and cyclic peptides are increasingly attractive therapeutic modalities as they often have improved affinity, are able to bind to extended protein surfaces, and otherwise have favorable properties. Macrocyclization of a known binder may stabilize its bioactive conformation and improve its metabolic stability, cell permeability, and in certain cases oral bioavailability. Herein, we present implementation and application of an approach that automatically generates, evaluates, and proposes cyclizations utilizing a library of well-established chemical reactions and reagents. Using the three-dimensional (3D) conformation of the linear molecule in complex with a target protein as the starting point, this approach identifies attachment points, generates linkers, evaluates their geometric compatibility, and ranks the resulting molecules with respect to their predicted conformational stability and interactions with the target protein. As we show here with prospective and retrospective case studies, this pro&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01500&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01500" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01500&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01500" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01500&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01500" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c01500&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01500&amp;href=/doi/10.1021/acs.jmedchem.0c01500" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c01500" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c01500" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (6 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01500&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c01500%26sid%3Dliteratum%253Aachs%26pmid%3D33017535%26genre%3Darticle%26aulast%3DSindhikara%26date%3D2020%26atitle%3DAutomated%2BDesign%2Bof%2BMacrocycles%2Bfor%2BTherapeutic%2BApplications%253A%2BFrom%2BSmall%2BMolecules%2Bto%2BPeptides%2Band%2BProteins%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D63%26issue%3D20%26spage%3D12100%26epage%3D12115%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291240" title="Cyclization">Cyclization</a>,</li><li><a href="/action/doSearch?ConceptID=291445" title="Chemical structure">Chemical structure</a>,</li><li><a href="/action/doSearch?ConceptID=292186" title="Receptors">Receptors</a>,</li><li><a href="/action/doSearch?ConceptID=291163" title="Macrocycles">Macrocycles</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/63/20" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/jmcmar.2020.63.issue-20/20201022/jmcmar.2020.63.issue-20.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01500/20201019/images/medium/jm0c01500_0014.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01500/20201019/images/large/jm0c01500_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01500&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Macrocycles and cyclic peptides are increasingly attractive therapeutic modalities as they often have improved affinity, are able to bind to extended protein surfaces, and otherwise have favorable properties. Macrocyclization of a known binder may stabilize its bioactive conformation and improve its metabolic stability, cell permeability, and in certain cases oral bioavailability. Herein, we present implementation and application of an approach that automatically generates, evaluates, and proposes cyclizations utilizing a library of well-established chemical reactions and reagents. Using the three-dimensional (3D) conformation of the linear molecule in complex with a target protein as the starting point, this approach identifies attachment points, generates linkers, evaluates their geometric compatibility, and ranks the resulting molecules with respect to their predicted conformational stability and interactions with the target protein. As we show here with prospective and retrospective case studies, this procedure can be applied for the macrocyclization of small molecules and peptides and even PROteolysis TArgeting Chimeras (PROTACs) and proteins.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_32022" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_32022" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">During the past decade, macrocycles, defined as cyclic small molecules or peptides typically with a molecular weight (MW) of 500–2000 Da, have gained significant interest as therapeutic agents, due to the introduction of new approaches for their synthesis and screening.<a onclick="showRef(event, 'ref1 ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3">(1−3)</a> Moreover, there is an unmet need for therapeutic agents that will be able to target a not yet well-addressed type of disease-relevant surfaces and interfaces that include, among others, extra- and intracellular protein–protein interactions (PPIs) with large and shallow (featureless) binding sites typically less amenable to small-molecule drugs.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> Because of their size and complexity, macrocycles have demonstrated that they can bind with antibody-like affinity and specificity and successfully target PPIs.<a onclick="showRef(event, 'ref5 ref6 ref7 ref8 ref9 ref10 ref11 ref12 ref13 ref14 ref15 ref16 ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref5 ref6 ref7 ref8 ref9 ref10 ref11 ref12 ref13 ref14 ref15 ref16 ref17 ref18">(5−18)</a> Hence, macrocyclic molecules fill an important gap in the world of drugs between small molecules and larger biologics.<a onclick="showRef(event, 'ref19 ref20'); return false;" href="javascript:void(0);" class="ref ref19 ref20">(19,20)</a></div><div class="NLM_p">Significant progress has been made for peptides in terms of synthesis, formulation, and delivery,<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> but their therapeutic value is partially limited due to their short in vivo half-life and lack of oral bioavailability. Moreover, while in solution, peptides span a large conformational space that increases the entropic cost of binding. Cyclization is one of the ways to limit this entropic cost of binding and thus improve its potency. Importantly, it has been shown that constraining a peptide via cyclization can also result in improved selectivity.<a onclick="showRef(event, 'ref22 ref23 ref24'); return false;" href="javascript:void(0);" class="ref ref22 ref23 ref24">(22−24)</a> Furthermore, cyclization is known to enhance the folding of peptides into conformations that might allow the formation of intramolecular hydrogen bonds, which in turn could improve passive permeability by reducing polar surface area. In addition, cyclization generally reduces degradation in the gut, blood, and tissues by shielding residues from metabolic proteases. Finally, covalent cyclization often enhances stability against thermal and chemical stress (e.g., caused by elevated temperature or presence of denaturants), which is often an important goal in engineering studies for peptides or proteins/enzymes for biotherapeutic or biotechnological applications.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a></div><div class="NLM_p">Medicinal chemistry allows further optimization of cyclic peptides toward drug-likeness and enables access to the world of nonpeptidic macrocycles in drug discovery. Thereby, despite violating some or all of the “rule of five” (Ro5) parameters, some naturally occurring, as well as synthetic, peptidic, and nonpeptidic macrocycles have demonstrated the ability for cell permeability and oral bioavailability.<a onclick="showRef(event, 'ref26 ref27 ref28 ref29 ref30 ref31 ref32 ref33 ref34'); return false;" href="javascript:void(0);" class="ref ref26 ref27 ref28 ref29 ref30 ref31 ref32 ref33 ref34">(26−34)</a> Indeed, a number of macrocycles with a molecular weight (MW) ranging from 500 to 1500 Da have become successful drugs, with multiple examples that can be administered orally.<a onclick="showRef(event, 'ref7 ref35 ref36 ref37'); return false;" href="javascript:void(0);" class="ref ref7 ref35 ref36 ref37">(7,35−37)</a></div><div class="NLM_p">Meanwhile, a variety of chemical approaches for the cyclization of peptides and small molecules have been described in the literature (see refs <a onclick="showRef(event, 'ref3 ref6 ref29'); return false;" href="javascript:void(0);" class="ref ref3 ref6 ref29">(3,6,29)</a>, and <a onclick="showRef(event, 'ref38 ref39 ref40 ref41 ref42 ref43'); return false;" href="javascript:void(0);" class="ref ref38 ref39 ref40 ref41 ref42 ref43">(38−43)</a>). One of the most popular strategies for cyclization is lactamization. The first studies on intramolecular amide-bond formation via side-chain stapling were reported by Felix et al. for helix stabilization between <i>i</i>, <i>i</i> + 4 spaced amino acids.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> Since then, this approach has been further elaborated by many others, exploring linkages with different chain lengths and positioning, as described in detail in a number of reviews<a onclick="showRef(event, 'ref39 ref45'); return false;" href="javascript:void(0);" class="ref ref39 ref45">(39,45)</a> and references therein. The formation of carbon–carbon bonds may be facilitated using a ring-closing metathesis (RCM) reaction.<a onclick="showRef(event, 'ref46 ref47 ref48'); return false;" href="javascript:void(0);" class="ref ref46 ref47 ref48">(46−48)</a> This approach has been successfully applied to several peptides binding to different biological targets.<a onclick="showRef(event, 'ref49 ref50 ref51 ref52 ref53 ref54 ref55'); return false;" href="javascript:void(0);" class="ref ref49 ref50 ref51 ref52 ref53 ref54 ref55">(49−55)</a> Another strategy is the Cu(I)-catalyzed azide–alkyne cycloaddition<a onclick="showRef(event, 'ref56 ref57'); return false;" href="javascript:void(0);" class="ref ref56 ref57">(56,57)</a> as a prototype of the popular “click reaction”,<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(58)</a> which has been reported by Chorev et al. for an analogue of the parathyroid hormone-related peptide<a onclick="showRef(event, 'ref59 ref60'); return false;" href="javascript:void(0);" class="ref ref59 ref60">(59,60)</a> and by Wang et al. for B-cell CLL/lymphoma 9 (BCL9) α-helical peptides.<a onclick="showRef(event, 'ref61'); return false;" href="javascript:void(0);" class="ref ref61">(61)</a> An alternative approach, photoinduced 1,3-dipolar cycloaddition, the UV-induced reaction between tetrazoles and alkenes, has been applied by Madden et al. to staple dual peptide inhibitors of the p53-Mdm2/Mdmx interactions.<a onclick="showRef(event, 'ref62'); return false;" href="javascript:void(0);" class="ref ref62">(62)</a> Staples may also be introduced with thioethers by linking of two cysteines via thiol–ene/–yne reactions<a onclick="showRef(event, 'ref63 ref64 ref65 ref66'); return false;" href="javascript:void(0);" class="ref ref63 ref64 ref65 ref66">(63−66)</a> or cysteine arylation and alkylation.<a onclick="showRef(event, 'ref67 ref68 ref69 ref70 ref71 ref72 ref73'); return false;" href="javascript:void(0);" class="ref ref67 ref68 ref69 ref70 ref71 ref72 ref73">(67−73)</a> Cysteine alkylation with reagents, containing three thiol-reactive groups, can also be used for synthesis of bicyclic structures, when the precursor peptide carries three cysteines.<a onclick="showRef(event, 'ref74 ref75'); return false;" href="javascript:void(0);" class="ref ref74 ref75">(74,75)</a> These approaches have been used together with combinatorial chemistry or phage display to screen structurally diverse monocyclic<a onclick="showRef(event, 'ref76 ref77 ref78'); return false;" href="javascript:void(0);" class="ref ref76 ref77 ref78">(76−78)</a> or bicyclic peptides.<a onclick="showRef(event, 'ref79 ref80 ref81'); return false;" href="javascript:void(0);" class="ref ref79 ref80 ref81">(79−81)</a> Recently, it was demonstrated that this chemistry can also be applied to the in situ cyclization of proteins.<a onclick="showRef(event, 'ref82 ref83 ref84'); return false;" href="javascript:void(0);" class="ref ref82 ref83 ref84">(82−84)</a> For two proteins, <i>Staphylococcus aureus</i> sortase A (SrtA) and the KIX domain from the human cAMP response element-binding (CREB) protein, cysteine residues were introduced in three surface-exposed positions and incubated with a triselectrophilic cross-linker to generate bicyclic enzymes with high tolerance toward thermal and chemical stress.<a onclick="showRef(event, 'ref84'); return false;" href="javascript:void(0);" class="ref ref84">(84)</a></div><div class="NLM_p">Given the three-dimensional (3D) structure of a linear peptidic or nonpeptidic molecule as the starting point, the first goal of cyclization is to maintain the conformation of the bioactive region that is relevant for biological recognition and affinity, and to introduce linkers where interactions are either not affected or even further improved. Several studies investigated the optimal positioning and linker lengths for the stabilization of α-helical structures.<a onclick="showRef(event, 'ref46 ref59 ref61 ref68 ref85 ref86 ref87 ref88 ref89 ref90 ref91'); return false;" href="javascript:void(0);" class="ref ref46 ref59 ref61 ref68 ref85 ref86 ref87 ref88 ref89 ref90 ref91">(46,59,61,68,85−91)</a> However, peptide–protein recognition is frequently mediated via nonhelical structures, such as loops, turns, or β-sheets,<a onclick="showRef(event, 'ref92'); return false;" href="javascript:void(0);" class="ref ref92">(92)</a> and might therefore require specific optimization of the linker and its positioning in the linear molecule for conformational stabilization. Considering each residue in a peptide as potential cyclization site and taking into account the diversity of chemical strategies for cyclization might lead to a vast number of molecules that could be synthesized. Implementation of reliable in silico approaches for an accurate design is therefore needed to guide experimental efforts toward the most promising compounds.<a onclick="showRef(event, 'ref82 ref93 ref94 ref95 ref96'); return false;" href="javascript:void(0);" class="ref ref82 ref93 ref94 ref95 ref96">(82,93−96)</a> To perform rational design, a reliable description of macrocycles’ conformational landscape is essential. Unfortunately, the rugged conformational landscape limits the effectiveness of standard approaches for small molecules to reliably predict macrocycle conformations.<a onclick="showRef(event, 'ref10 ref97 ref98'); return false;" href="javascript:void(0);" class="ref ref10 ref97 ref98">(10,97,98)</a> Consequently, significant work has been made into enhanced sampling algorithms toward macrocycle conformation prediction.<a onclick="showRef(event, 'ref97 ref98 ref99 ref100 ref101 ref102 ref103 ref104 ref105 ref106 ref107 ref108 ref109'); return false;" href="javascript:void(0);" class="ref ref97 ref98 ref99 ref100 ref101 ref102 ref103 ref104 ref105 ref106 ref107 ref108 ref109">(97−109)</a> Additional work has been put into docking or otherwise calculating macrocycle binding free energies.<a onclick="showRef(event, 'ref110 ref111 ref112 ref113 ref114 ref115 ref116'); return false;" href="javascript:void(0);" class="ref ref110 ref111 ref112 ref113 ref114 ref115 ref116">(110−116)</a> More recently, a structure-based design of peptide macrocycles targeting the interaction site of human adaptor protein 14-3-3 using free-energy perturbation (FEP) calculations was proposed.<a onclick="showRef(event, 'ref117'); return false;" href="javascript:void(0);" class="ref ref117">(117)</a> Despite the above-mentioned efforts, to the best of our knowledge, there is no computational method available that combines automatic enumeration, evaluation, and ranking of the macrocyclizations based on a list of chemical linkers (derived from available reagents) and the 3D structure of a linear reference.</div><div class="NLM_p last">The aim of the present study is to implement an approach that, based on a given ligand 3D structure (ideally in complex with a receptor), will provide synthesizable in silico designed macrocyclic small molecules, peptides, or proteins targeting a receptor–ligand or protein–protein interface (antagonists or agonists). The presented approach automatically generates proposals for synthesis by enumerating and screening cyclizations. The screening is based on geometric constraints, conformational rigidity, and optionally the predicted interactions of the macrocycles to a target protein. The enumeration uses linkers or breeding structures for cyclization that can be synthesized with well-established chemical reactions utilizing (commercially) available chemical reagents. This list of chemical linkers can be modified by the user to consider specific cyclization chemistries and reagents.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_87473" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_87473" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Here, we present an overview of the general workflow for the recognition and evaluation of sites for macrocyclization, the identification of compatible chemical linkers, and their conformational and enthalpic scoring. To demonstrate the scope of this approach, prospective applications for a peptide (dual glucagon-like peptide-1 (GLP-1)/glucagon receptor agonists) and a small molecule (activated thrombin-activatable fibrinolysis inhibitor (TAFIa)) project and retrospective applications for a PROteolysis TArgeting Chimera (PROTAC) and a protein will be shown. In addition, a further retrospective peptide (β-catenin inhibitors) and small molecule (factor XIa inhibitors) example can be found in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c01500/suppl_file/jm0c01500_si_001.pdf" class="ext-link">Supporting Information</a> (SI).</div><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> General Workflow</h3><div class="NLM_p">Our strategy involves several integrated steps summarized in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>. For technical details, see the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>. Briefly, we start with the 3D structure of the linear ligand in complex with the target protein. Synthetic capabilities may suggest a set of cyclization linkers and attachment points (“limbs”). The approach will generate a full conformational ensemble of linkers, permute pairs of attachment points within the rigid ligand, and then geometrically filter cyclizations to eliminate highly strained or otherwise unfavorable ones. The output are 3D structures of complexed cyclized ligands. Additional scoring can reduce the number of synthesis proposals. Since the goal of cyclization is often stabilization of the bioactive conformation, here, we run conformational sampling and rank the cyclized ligands according to their calculated conformational propensity. Conformational scoring for small molecules and small peptides is performed using Prime Macrocycle Conformational Sampling (Prime-MCS)<a onclick="showRef(event, 'ref106 ref107'); return false;" href="javascript:void(0);" class="ref ref106 ref107">(106,107)</a> and for larger peptides and proteins with molecular dynamics (MD) simulations. An “enthalpic” scoring of the protein–ligand complex can be done using either the empirical scoring function GlideScore<a onclick="showRef(event, 'ref118 ref119'); return false;" href="javascript:void(0);" class="ref ref118 ref119">(118,119)</a> or the molecular mechanics-generalized Born surface area (MM/GBSA) method.<a onclick="showRef(event, 'ref120 ref121'); return false;" href="javascript:void(0);" class="ref ref120 ref121">(120,121)</a> In a standard project setting, it is generally difficult to accurately dissect the enthalpic and entropic contributions to the free energy of binding, in particular if experimental data are obtained from cellular in vitro assays. In terms of in silico prediction, rigorous free-energy calculations, encapsulating both entropic and enthalpic components, may be applied.<a onclick="showRef(event, 'ref116 ref122'); return false;" href="javascript:void(0);" class="ref ref116 ref122">(116,122)</a> However, since these calculations are time-consuming and their predictivity depends on the accuracy of the generated binding poses,<a onclick="showRef(event, 'ref123'); return false;" href="javascript:void(0);" class="ref ref123">(123)</a> here, for high-speed enthalpic screening, we applied GlideScore for small-molecule cyclizations and MM/GBSA otherwise. For the selection of proposals for experimental synthesis in the presented prospective case studies, we therefore ranked the virtually designed macrocycles based on their predicted (1) enthalpic scores and (2) subjected the top-ranked molecules to conformational scoring. These scores do usually not provide an accurate correlation with experimental potencies but are generally sufficient to provide an enrichment of active molecules among the top scorers.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01500/20201019/images/medium/jm0c01500_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01500/20201019/images/large/jm0c01500_0002.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Procedure for the automated design of macrocycles. (Step 1) The user prepares a linear starting molecule. The attachment points may be either identified automatically or specified by the user. (Step 2) Generation of chemical linkers. These might either be provided as explicit lists, e.g., based on available chemical reagents, or can be grown automatically by combining and enumerating small breeding fragments. (Step 3) A full conformational ensemble of each linker is generated and (step 4) mapped with all attachment points to eliminate unfavorable cyclizations applying different geometric filter criteria. (Step 5) Finally, the cyclized ligands are ranked applying conformational and/or enthalpic scoring.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01500/20201019/images/large/jm0c01500_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01500&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Identification of Attachment Points for Cyclization</h3><div class="NLM_p last">For the identification of attachment points (see <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>a), our approach differentiates between the application for peptides/proteins and small molecules. (i) For peptides, the default attachment “vectors” for the introduction of cyclizations are typically the Cα–Cβ bonds within the peptide sequence. Hence, this approach allows us to consider any (commercially or internally) available α-amino acid with side-chain groups suitably reactive for cyclization, and the resulting design proposals can be synthesized by classical stepwise (solid-phase) peptide synthesis with a subsequent cyclization step. As an additional option, it is possible to consider the Cα–Hα vector to introduce <span class="smallcaps smallerCapital">d</span>-amino acids for cyclization and conformational stabilization. (ii) For small molecules, on the other hand, the positioning of the attachment points for cyclization generally depends on the specific synthetic route of the target molecule. Therefore, our approach allows us to define the most suited attachment vectors by specifying atomic numbers or a user-defined substructure (defined as simplified molecular-input line-entry system (SMILES) arbitrary target specification (SMARTS)<a onclick="showRef(event, 'ref124'); return false;" href="javascript:void(0);" class="ref ref124">(124)</a>) within the starting molecule. With this procedure, it is also possible to introduce “nonstandard” cyclizations into a peptide, such as side chain-to-tail, head-to-side chain, head-to-tail, or other cyclizations.</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Chemical Linkers for Cyclization</h3><div class="NLM_p">There are two options to obtain chemical linkers for cyclization (see <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>b), which can be either (i) provided as an explicit list derived from reagents that allow cyclization along a known chemical route or (ii) grown automatically by fusing a set of small spacer fragments. <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> provides a list of example linkers which are particularly suited for the side-chain cyclization of peptides. These linkers are derived from common modified amino acids that allow for cyclization via lactamization, click chemistry, ring-closing metathesis, or cysteine alkylation. With respect to our in silico method, these linkers are attached to the Cα–Cβ or Cα–Hα vectors of the peptide to design cyclizations via <span class="smallcaps smallerCapital">l</span>- or <span class="smallcaps smallerCapital">d</span>-amino acids. This list can be reduced or extended, for example, based on other cyclization chemistry and/or the availability of corresponding reagents, to assure the cyclization proposals can be straightforwardly synthesized. Providing an explicit list of linkers might also be used in a small-molecule scenario where the synthetic route is already planned to identify the most promising linkers from a list of available chemical reagents.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Exemplary Chemical Linkers for Peptide Cyclization<a class="ref internalNav" href="#t1fn2" aria-label="b">b</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01500/20201019/images/medium/jm0c01500_0009.gif" alt="" id="fx1" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">Ring-closing metathesis (RCM).</p></div><div class="footnote" id="t1fn2"><sup><sup>b</sup></sup><p class="last">Two-armed linkers for the generation of monocycles based on common chemical reagents applying cysteine alkylation, lactamization, ring-closing metathesis, or click chemistry. This list can be modified by the user based on specific reagents and chemical cyclization routes.</p></div></div><div></div></div><div class="NLM_p">Alternatively, rather than specifying predefined cyclization linkers, the linkers can be constructed in silico from a small set of “breeding fragments” (see <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>, for example, fragments), combined and enumerated to fit within the geometrical constraints of the ligand–receptor complex. As a result, the algorithm will suggest a diverse set of linker chains that are synthetically tractable. While the default spacers shown in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a> are amide, poly(ethylene glycol) (PEG), or alkyl moieties, this list of fragments can be expanded, for example, to include aromatic rings, heterocycles, or other reactive groups.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Example Breeding Fragments for the In Silico Construction and Growth of Linkers for Small-Molecule Cyclization<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01500/20201019/images/medium/jm0c01500_0010.gif" alt="" id="fx2" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">This list can be modified by the user.</p></div></div><div></div></div></div><div id="sec2_3_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Generation of Bicyclic Structures</h3><div class="NLM_p">As mentioned above, it is often advantageous to transform linear peptides or proteins to bicyclic structures, for example, via cysteine alkylation using symmetric cyclization reagents that contain three thiol-reactive groups (see <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>).<a onclick="showRef(event, 'ref74 ref75 ref79 ref80 ref81'); return false;" href="javascript:void(0);" class="ref ref74 ref75 ref79 ref80 ref81">(74,75,79−81)</a> With our approach, it is possible to generate such bicyclic structures in a stepwise manner, as illustrated in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>. First, the trisymmetrical linker is converted to a two-armed linker with the third limb considered as “nonreactive”. With this two-armed linker, all geometrically reasonable (mono-)cyclized structures are generated as outlined above. In the next step, these cyclized structures are used as the starting points for a second cyclization routine with an appropriate linker, where the previously nonreactive limb is now considered as first attachment point and each Cα atom of the peptide backbone as second attachment point.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01500/20201019/images/medium/jm0c01500_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01500/20201019/images/large/jm0c01500_0003.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Approach for modeling bicyclic peptide structures using a trisymmetrical linker. In step 1, all geometrically reasonable monocyclic structures are generated. In the second cyclization routine (step 2), the unlinked (third) attachment point of the linker is evaluated for attachment to the peptide backbone for construction of the second cycle.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01500/20201019/images/large/jm0c01500_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01500&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Example of Three-Armed Linkers for the Generation of Bicyclic Structures Based on Common Chemical Reagents Applying Cysteine Alkylation<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01500/20201019/images/medium/jm0c01500_0011.gif" alt="" id="fx3" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">This list can be modified by the user.</p></div></div><div></div></div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Application Examples</h3><div id="sec2_4_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i12" class="anchor-spacer"></div><h4 class="article-section__title" id="_i12"> Peptide Example: Cyclization of Dual Glucagon-like Peptide-1 (GLP-1)/Glucagon Receptor Agonists (Prospective Case Study)</h4><div class="NLM_p">We previously described the discovery of dual agonists, which were identified by rational design.<a onclick="showRef(event, 'ref125 ref126'); return false;" href="javascript:void(0);" class="ref ref125 ref126">(125,126)</a> Structural elements of glucagon were engineered into the selective GLP-1 receptor agonist exendin-4, resulting in hybrid peptides with potent dual GLP-1/glucagon receptor activity.<a onclick="showRef(event, 'ref125'); return false;" href="javascript:void(0);" class="ref ref125">(125)</a> To probe the applicability of our cyclization approach in a prospective manner, here, we used the previously reported linear dual agonist <b>4a</b> as the starting point for the introduction of molecular cyclizations to optimize the potency on the GLP-1 receptor. <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>a shows the predicted binding mode of <b>4a</b> in the full-length structure of the GLP-1 receptor (pdb code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5VAI">5VAI</a>).<a onclick="showRef(event, 'ref127'); return false;" href="javascript:void(0);" class="ref ref127">(127)</a> In the first step, we automatically generated cyclization proposals with all chemical linkers shown in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, allowing all residues to be considered as a cyclization site. As a result, >2000 linkages were proposed by the cyclization algorithm (see <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>b). <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>b illustrates that the algorithm automatically identifies and excludes all residues of the linear peptide from cyclization that establish contacts with the GLP-1 receptor, for example, in the N-terminal peptide tail where several residues are known to establish interactions that are important for agonistic activity. To reduce the number of synthesis proposals, we next (i) focused the list of linkers to <b>1b</b>, <b>1f</b>, and <b>1k</b> (see <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>) for cyclization via cysteine alkylation, lactamization, or RCM, and (ii) based on structure–activity relationship (SAR) knowledge, specified residues 10, 12, 14, 16, 18, 20, 21, 28, 33, 35, and 39 as possible attachment sites for cyclization. With these parameters, 29 cyclization proposals were obtained (see <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>c). These proposals were subjected to molecular mechanical minimization, ranked according to their calculated MM/GBSA binding energy (see minimized poses in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>d) and subjected to conformational scoring with MD simulations (see the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a> for details).<a onclick="showRef(event, 'ref128'); return false;" href="javascript:void(0);" class="ref ref128">(128)</a> To assure that these cyclizations do not induce conformations that are too distant from the putative bioactive conformation, we considered only variants with a favorable conformational score (root-mean-square deviation (RMSD) <2.0) for further analysis. Finally, we assessed the remaining design proposals with respect to their predicted scores, diversity, and novelty. Using the linear peptides <b>4a</b> or <b>4g</b>, which carries a <span class="smallcaps smallerCapital">d</span>-serine (dSer) in position 2 for stabilization against dipeptidyl peptidase IV (DPP IV)-mediated cleavage, as structural references, we synthesized 10 stapled peptides, applying RCM, lactamization, or cysteine alkylation (see details in the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a> and analytical data in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c01500/suppl_file/jm0c01500_si_001.pdf" class="ext-link">Supporting Information</a>). Subsequently, we tested their potency on the GLP-1 and glucagon receptor in a cAMP assay in receptor overexpressing human embryonic kidney (HEK)-293 cells (see computed scores and experimental potency data in <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>).</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01500/20201019/images/medium/jm0c01500_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01500/20201019/images/large/jm0c01500_0004.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Cyclization of dual agonists of the GLP-1 and glucagon receptor. (a) Predicted binding mode of the dual agonist <b>4a</b> at the GLP-1R (pdb code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5VAI">5VAI</a>). (b) Automatically generated cyclization proposals with all chemical linkers shown in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>. (c) Cyclization proposals using a focused list of reagents and attachment positions. (d) Predicted binding modes of synthesized peptides obtained from MM/GBSA energy minimization.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01500/20201019/images/large/jm0c01500_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01500&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Sequences, Computed Properties, and EC<sub>50</sub> Values (Plus SEM Values; <i>n</i> = 2; Measured in pM in a cAMP Assay in Overexpressing HEK-293 Cell Lines) of Stapled Peptidic Dual GLP-1/Glucagon Receptor Agonists and Their Linear References<a class="ref internalNav" href="#t4fn7" aria-label="g">g</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" colspan="2" align="center">(a)</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">no</th><th class="colsep0 rowsep0" align="center">sequence</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4a</b></td><td class="colsep0 rowsep0" align="left">HSQGTFTSDLSKQMDSRRAQDFIEWLKNGGPSSGAPPPS–NH2</td></tr></tbody></table><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="±" /></col><col align="char" char="±" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" colspan="6" align="center">(b)</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center" char=".">computed properties</th><th class="rowsep1 colsep0" colspan="2" align="center" char="±">EC<sub>50</sub> [pM] (±SEM, <i>n</i> = 2)</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">no</th><th class="colsep0 rowsep0" align="center">sequence modification vs <b>4a</b></th><th class="colsep0 rowsep0" align="center" char=".">conformational score<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center" char=".">enthalpic score<a class="ref internalNav" href="#t4fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center" char="±">GLP-1 receptor</th><th class="colsep0 rowsep0" align="center" char="±">glucagon receptor<a class="ref internalNav" href="#t4fn3" aria-label="c">c</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4a</b></td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="char" char=".">1.69</td><td class="colsep0 rowsep0" align="char" char=".">–193.1</td><td class="colsep0 rowsep0" align="char" char="±">18.3 ± 1.0</td><td class="colsep0 rowsep0" align="char" char="±">28.6 ± 1.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4b</b></td><td class="colsep0 rowsep0" align="left">(K10,E14)-lactam</td><td class="colsep0 rowsep0" align="char" char=".">1.44</td><td class="colsep0 rowsep0" align="char" char=".">–187.4</td><td class="colsep0 rowsep0" align="char" char="±">47.5 ± 3.6</td><td class="colsep0 rowsep0" align="char" char="±">119 ± 5.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4c</b></td><td class="colsep0 rowsep0" align="left">(Pal<a class="ref internalNav" href="#t4fn4" aria-label="d">d</a>10,Pal14)-RCM</td><td class="colsep0 rowsep0" align="char" char=".">1.62</td><td class="colsep0 rowsep0" align="char" char=".">–192.9</td><td class="colsep0 rowsep0" align="char" char="±">7.4 ± 0.5</td><td class="colsep0 rowsep0" align="char" char="±">2.3 ± 0.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4d</b></td><td class="colsep0 rowsep0" align="left">(Pal14,Pal18)-RCM</td><td class="colsep0 rowsep0" align="char" char=".">1.66</td><td class="colsep0 rowsep0" align="char" char=".">–186.5</td><td class="colsep0 rowsep0" align="char" char="±">0.4 ± 0.0</td><td class="colsep0 rowsep0" align="char" char="±">369 ± 20.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4e</b></td><td class="colsep0 rowsep0" align="left">(C21,C35,mXyl<a class="ref internalNav" href="#t4fn5" aria-label="e">e</a>)-Cys-alk</td><td class="colsep0 rowsep0" align="char" char=".">1.51</td><td class="colsep0 rowsep0" align="char" char=".">–207.3</td><td class="colsep0 rowsep0" align="char" char="±">24.9 ± 1.2</td><td class="colsep0 rowsep0" align="char" char="±">12.9 ± 0.9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4f</b></td><td class="colsep0 rowsep0" align="left">(C21,C28,C35,Mes<a class="ref internalNav" href="#t4fn6" aria-label="f">f</a>)-Cys-alk</td><td class="colsep0 rowsep0" align="char" char=".">2.71</td><td class="colsep0 rowsep0" align="char" char=".">–217.9</td><td class="colsep0 rowsep0" align="char" char="±">194 ± 14.5</td><td class="colsep0 rowsep0" align="char" char="±">578 ± 32.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4g</b></td><td class="colsep0 rowsep0" align="left">dSer2</td><td class="colsep0 rowsep0" align="char" char=".">1.69</td><td class="colsep0 rowsep0" align="char" char=".">–217.6</td><td class="colsep0 rowsep0" align="char" char="±">75.4 ± 3.7</td><td class="colsep0 rowsep0" align="char" char="±">122 ± 5.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4h</b></td><td class="colsep0 rowsep0" align="left">dSer2;(Pal14,Pal18)-RCM</td><td class="colsep0 rowsep0" align="char" char=".">1.66</td><td class="colsep0 rowsep0" align="char" char=".">–210.6</td><td class="colsep0 rowsep0" align="char" char="±">1.2 ± 0.1</td><td class="colsep0 rowsep0" align="char" char="±">8430 ± 610</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4i</b></td><td class="colsep0 rowsep0" align="left">dSer2;(K16,E20)-lactam</td><td class="colsep0 rowsep0" align="char" char=".">1.35</td><td class="colsep0 rowsep0" align="char" char=".">–219.0</td><td class="colsep0 rowsep0" align="char" char="±">6.4 ± 0.2</td><td class="colsep0 rowsep0" align="char" char="±">7.8 ± 0.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4j</b></td><td class="colsep0 rowsep0" align="left">dSer2;(E28,K33)-lactam</td><td class="colsep0 rowsep0" align="char" char=".">1.48</td><td class="colsep0 rowsep0" align="char" char=".">–218.4</td><td class="colsep0 rowsep0" align="char" char="±">14.7 ± 0.8</td><td class="colsep0 rowsep0" align="char" char="±">54.2 ± 2.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4k</b></td><td class="colsep0 rowsep0" align="left">dSer2;(K28,E33)-lactam</td><td class="colsep0 rowsep0" align="char" char=".">1.40</td><td class="colsep0 rowsep0" align="char" char=".">–247.9</td><td class="colsep0 rowsep0" align="char" char="±">8.13 ± 0.3</td><td class="colsep0 rowsep0" align="char" char="±">59.6 ± 3.9</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup><sup>a</sup></sup><p class="last">The conformational score was calculated as RMSD over the Cα residues.</p></div><div class="footnote" id="t4fn2"><sup><sup>b</sup></sup><p class="last">The enthalpic score was obtained from Prime MM/GBSA based on the predicted peptide binding mode in the GLP-1 receptor.</p></div><div class="footnote" id="t4fn3"><sup><sup>c</sup></sup><p class="last">The enthalpic score was not calculated in the presence of the glucagon receptor. EC<sub>50</sub> values are provided here as additional SAR information.</p></div><div class="footnote" id="t4fn4"><sup><sup>d</sup></sup><p class="last">Pal = 4-pentenyl-alanine.</p></div><div class="footnote" id="t4fn5"><sup><sup>e</sup></sup><p class="last"><i>m</i>Xyl = <i>meta</i>-xylyl.</p></div><div class="footnote" id="t4fn6"><sup><sup>f</sup></sup><p class="last">Mes = mesitylene.</p></div><div class="footnote" id="t4fn7"><sup><sup>g</sup></sup><p class="last">The IUPAC names of all molecules are provided in <a href="/doi/suppl/10.1021/acs.jmedchem.0c01500/suppl_file/jm0c01500_si_001.pdf" class="ext-link">Table S4</a>.</p></div></div></div><div class="NLM_p">As shown in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>d, several <i>i</i>, <i>i</i> + 4 staples were identified along the peptide helix, as in peptides <b>4b</b>–<b>d</b> and <b>4h</b>–<b>i</b>. Such staples, obtained by lactamization or RCM, have been described for other GLP-1 receptor agonists before.<a onclick="showRef(event, 'ref129 ref130 ref131 ref132'); return false;" href="javascript:void(0);" class="ref ref129 ref130 ref131 ref132">(129−132)</a> Here, introduction of these staples into the linear peptides <b>4a</b> and <b>4g</b> resulted in improved potency, presumably due to conformational stabilization and/or establishment of additional interactions with the GLP-1 receptor. In detail, RCM from positions 14–18 increased potency vs the linear reference molecule by a factor of 45 (<b>4d</b> vs <b>4a</b>) or even 63 (<b>4h</b> vs <b>4g</b>), whereas cyclization from positions 10–14 increased potency by a factor of >2 (<b>4c</b> vs <b>4a</b>). For the (K10, E14) lactamizations, potency was slightly worsened (<b>4b</b> vs <b>4a</b>), while the (K16, E20) cyclization improved GLP-1R potency by a factor of 12 (<b>4i</b> vs <b>4g</b>). These data validate that our approach is capable of identifying classical cyclizations in helical peptide regions.</div><div class="NLM_p">In addition to these <i>i</i>, <i>i</i> + 4 staples, we were interested to see how far the method is able to design cyclizations within the nonhelical peptide region. Therefore, we additionally selected automatically generated cyclization proposals with attachment points outside the peptide helix (i.e., residues 28–39) for chemical synthesis. Applying lactamization, positions 28 and 33 were covalently linked via the side chains of Glu and Lys residues, providing peptides <b>4j</b> and <b>4k</b> (see <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>d). We were delighted to see that these modifications improved potency vs the linear reference peptide either by a factor of 5 (<b>4j</b> vs <b>4g</b>) or even 9 (<b>4k</b> vs <b>4g</b>). Finally, a covalent linkage introduced between residues 21 and 35 by cysteine alkylation via a <i>meta</i>-xylyl linker (peptide <b>4e</b>, see <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>d) maintained activity at the GLP-1 receptor. Such cyclizations as introduced in these positions for peptides <b>4j</b>, <b>4k</b>, and <b>4e</b> have, to our knowledge, not been described for exendin-4 analogues in the literature before and demonstrate that our approach is capable of identifying novel cyclizations not only in helical but also in looped regions of a peptide.</div><div class="NLM_p">Based on these findings, we also applied our algorithm for the identification of bicyclic structures (see above) using the three-armed linker <b>3a</b> as analogue of the two-armed linker <b>1b</b>. Due to the geometric constraints of the peptide’s 3D structure, only one proposal was obtained (peptide <b>4f</b>). Although it revealed an unfavorable conformational score (RMSD = 2.71), we synthesized <b>4f</b> using 1,3,5-tris(bromomethyl)benzene (TBMB) as cysteine cross-linker, with Cys residues introduced into positions 21, 28, and 35. EC<sub>50</sub> determination revealed, however, a significant drop in potency (>factor 10 at both receptors).</div><div class="NLM_p">Comparison of the computed enthalpic scores with the experimental GLP-1 receptor potency data (see data in <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>) does not reveal a linear correlation, possibly due to the fact that the modeled binding modes are not based on accurate experimental structures (the resolution of the GLP-1 receptor structure pdb <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5VAI">5VAI</a> is 4.1 Å). With respect to their conformational scores, the monocyclic peptides do not significantly differ. Interestingly, the bicyclic peptide <b>4f</b> reveals the lowest potency and has the worst conformational score, suggesting that the introduced bicycle forces the peptide backbone too far away from the bioactive conformation.</div><div class="NLM_p">In terms of SAR, it is interesting that the introduced cyclizations do not always result in proportional potency shifts at the GLP-1 and glucagon receptor. Most remarkably, introduction of cyclizations at positions 14–18 by RCM results in a significant potency increase at the GLP-1 receptor compared to the linear reference (factor 46 for <b>4d</b> vs <b>4a</b> and factor 63 for <b>4h</b> vs <b>4g</b>) but a significant potency decrease at the glucagon receptor (factor 29 for <b>4a</b> vs <b>4d</b> and factor 69 for <b>4g</b> vs <b>4h</b>). These results confirm that cyclization of peptides might not only be useful for the optimization of potency or metabolic stability but also to modulate selectivity between similar targets.<a onclick="showRef(event, 'ref22 ref23 ref24'); return false;" href="javascript:void(0);" class="ref ref22 ref23 ref24">(22−24)</a></div><div class="NLM_p last">In summary, this prospective case study illustrates that our approach identifies classical motifs for the stabilization of α-helices, but in addition is able to identify favorable cyclizations within noncanonical peptide regions based on the 3D structure of the linear reference peptide. Although we did not observe a perfect correlation between computed scores and experimental potency data, the approach was able to suggest highly active compounds for synthesis. Indeed, all synthesized design proposals (except for <b>4f</b>) were able to activate the GLP-1 receptor in the one- or two-digit picomolar range. Six out of these 10 molecules are even more potent than the linear reference peptides. Since the in silico approach was designed to consider only chemical linkers that are derived from available chemical reagents, all molecules could be straightforwardly synthesized.</div></div><div id="sec2_4_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i14" class="anchor-spacer"></div><h4 class="article-section__title" id="_i14"> Small Molecule Example 1: Macrocyclic Inhibitors of Activated Thrombin-Activatable Fibrinolysis Inhibitor (TAFIa) from Natural Product Anabaenopeptin (Prospective Case Study)</h4><div class="NLM_p">We previously described the identification of anabaenopeptins, cyclic peptides produced by cyanobacteria, as potent inhibitors of TAFIa.<a onclick="showRef(event, 'ref133'); return false;" href="javascript:void(0);" class="ref ref133">(133)</a> Starting from the X-ray co-crystal structure of anabaenopeptin C bound to the surrogate protease carboxypeptidase B (pdb code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LRJ">5LRJ</a>; see <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>a), drug design efforts led to simplified noncyclic small molecules with increased ligand efficiency.<a onclick="showRef(event, 'ref134'); return false;" href="javascript:void(0);" class="ref ref134">(134)</a> In addition, we started a structure-based rational approach toward macrocycles to reduce the size and polar peptidic character of the peptidic ring. Here, we used our automated approach for the prospective design of macrocyclic synthesis proposals. Starting from the bioactive conformation of anabaenopeptin C (pdb code 5lrj), we sketched a precursor structure with the attachment points for cyclization at the exit vectors of the macrocyclic ring (see <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>b and <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>). To identify suited linkers for cyclization, a set of five reactive “breeding fragments” (see <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>c) was provided to the in silico algorithm. Combinatorial enumeration of these fragments would theoretically result in >300 000 different linker chains. By automatic consideration of the geometrical constraints imposed by the linker attachment vectors and the protein environment, this number was automatically reduced to only 68 and further filtered out for unwanted structural motifs (see the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a> for technical details), resulting in 45 cyclization proposals. These were docked into the protease binding pocket, ranked according to their enthalpic scores (GlideScore; see <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>d), filtered by their conformational scores (RMSD < 2.0) and inspected with respect to their diversity, novelty, and synthetic accessibility. Here, we synthesized four macrocyclic analogues and subsequently tested their potency in an in vitro assay (see the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a> and the <a href="/doi/suppl/10.1021/acs.jmedchem.0c01500/suppl_file/jm0c01500_si_001.pdf" class="ext-link">Supporting Information</a> for all experimental details).</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01500/20201019/images/medium/jm0c01500_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01500/20201019/images/large/jm0c01500_0005.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Macrocylization of a TAFIa inhibitor. (a) X-ray structure of anabaenopeptin C in complex with the surrogate protease carboxypeptidase B (pdb code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LRJ">5LRJ</a>). (b) Anabaenopepin C was modified to obtain the starting structure for rescaffolding of the macrocycle. The attachment vectors are indicated in magenta. (c) Breeding fragments that were combined and enumerated for cyclization. Predicted binding modes of (d) automatically generated cyclization proposals and (e) four cyclization proposals that were synthesized and experimentally tested.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01500/20201019/images/large/jm0c01500_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01500&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. Macrocyclic Derivatives with Different Linkers and Their Computed Conformational and Enthalpic Scores<a class="ref internalNav" href="#t5fn3" aria-label="c">c</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01500/20201019/images/medium/jm0c01500_0012.gif" alt="" id="fx4" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01500/20201019/images/medium/jm0c01500_0013.gif" alt="" id="fx5" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t5fn1"><div class="footnote" id="t5fn1"><sup><sup>a</sup></sup><p class="last">The conformational score was calculated by Prime-MCS.</p></div><div class="footnote" id="t5fn2"><sup><sup>b</sup></sup><p class="last">The enthalpic score was obtained from GlideScore.</p></div><div class="footnote" id="t5fn3"><sup><sup>c</sup></sup><p class="last"><b>5a</b>–<b>c</b> were synthesized and subjected to IC<sub>50</sub> determination.</p></div></div><div></div></div><div class="NLM_p">As shown in <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>, these four macrocycles cover three different chemical classes that were synthesized via different chemical routes (see <a href="/doi/suppl/10.1021/acs.jmedchem.0c01500/suppl_file/jm0c01500_si_001.pdf" class="ext-link">SI</a>). Although the docking modes suggest similar 3D conformations, the macrocycles are considerably different with respect to their two-dimensional (2D) structures (except for <b>5b</b> and <b>5d</b>). IC<sub>50</sub> determination (see <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>) revealed inhibition for all four macrocycles in the two-digit nanomolar range, with the most potent compound (<b>5d</b>) showing an IC<sub>50</sub> value of 10 nM, which is in a similar range compared to other potent small-molecule inhibitors of TAFIa that have been reported before.<a onclick="showRef(event, 'ref134 ref135'); return false;" href="javascript:void(0);" class="ref ref134 ref135">(134,135)</a> In comparison to the reference molecule anabaenopeptin C, this represents a potency loss of factor 5, but therefore a significant improvement in terms of molecular weight (808 vs 483 Da). Thus, the obtained small-molecule macrocycles represent interesting starting points for further multiparameter compound optimization toward improved lead likeness, for example, by replacing the basic lysine residue by other less polar amino acids.<a onclick="showRef(event, 'ref134 ref135'); return false;" href="javascript:void(0);" class="ref ref134 ref135">(134,135)</a></div><div class="NLM_p">As for the previous case study, there is no linear correlation between the computed scores and the experimental affinities (see <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>). Interestingly, the least potent macrocycle also reveals the weakest conformational score. But since this dataset is small and does not significantly differ in terms of scores and pIC<sub>50</sub> data, it is not finally possible to conclude in how far the scoring scheme was able to efficiently enrich active molecules among the top scorers. Nevertheless, we would like to emphasize that the approach was able to suggest a small set of chemically feasible synthesis proposals from a large theoretical chemical space, with all synthesized molecules representing novel, potent, and interesting starting points for further optimization.</div><div class="NLM_p last">In conclusion, this prospective study demonstrates how molecules with larger peptidic motifs can be generally converted into leadlike small macrocycles with favorable ligand efficiency by shrinking and rescaffolding those regions that do not establish key interactions with the target protein.</div></div><div id="sec2_4_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i16" class="anchor-spacer"></div><h4 class="article-section__title" id="_i16"> Cyclization of a PROTAC (Retrospective Case Study)</h4><div class="NLM_p">PROteolysis TArgeting Chimeras (PROTACs) have become an appealing technology for modulating a protein of interest by degradation.<a onclick="showRef(event, 'ref136'); return false;" href="javascript:void(0);" class="ref ref136">(136)</a> PROTACs are bifunctional molecules where one ligand binds a protein of interest and the other one a ubiquitin ligase (E3). Both ligands are connected with an optimal linker. Degradation is initiated when PROTACs bind the protein of interest and E3 to form a ternary complex. Recently, Testa et al. demonstrated that macrocyclization can also be introduced into PROTACs.<a onclick="showRef(event, 'ref137'); return false;" href="javascript:void(0);" class="ref ref137">(137)</a> A crystal structure of PROTAC MZ1 (<b>6a</b>) in complex with the E3 ligase von Hippel–Lindau protein (VHL) and its target, the second bromodomain (BD2) of the bromodomain and extra-terminal (BET) motif protein (see <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>a), was used as a starting point for macrocyclization by introducing a PEG linker at attachment sites that are pointing away from both proteins (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>b). Despite a loss of binding affinity for BET family member bromodomain-containing protein 4 (Brd4), the macro-PROTAC <b>6b</b>, comprising three PEG units, exhibited cellular activity comparable to the linear PROTAC <b>6a</b>. Here, we investigated whether this cyclization can be reproduced with our automated approach, using the precursor structure shown in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>b as the starting point. For the in silico construction and growth of linkers, the PEG fragment <b>2a</b> (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>) was provided as breeding fragment. This fragment was combined and enumerated to fit within the geometrical constraints of the ligand–protein binding sites. Six linkers comprising three to eight PEG units were identified, including the three-PEG linker, which was synthesized (<b>6b</b>) and successfully tested by Testa et al. All macrocyclic PROTAC proposals were subjected to MM/GBSA binding energy calculations with the Prime software (see minimized poses in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>c). Comparison of <b>6b</b> with the X-ray structure (pdb code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6SIS">6SIS</a>) reveals that the experimental binding pose could be well reproduced (see <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>d).</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01500/20201019/images/medium/jm0c01500_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01500/20201019/images/large/jm0c01500_0006.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. (a) X-ray structure of the linear PROTAC MZ1 (<b>8a</b>) in complex with Brd4BD2 and VHL-EloB-EloC (pdb code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5T35">5T35</a>). (b) The PROTAC structure was modified to obtain the starting structure for cyclization. The attachment vectors are indicated in magenta. (c) Cyclized conformations comprising PEG units of different length, minimized in the protein environment. (d) Comparison of the predicted binding mode of <b>8b</b> with the experimental binding mode (pdb <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6SIS">6SIS</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01500/20201019/images/large/jm0c01500_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01500&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Due to their high molecular weight, a major optimization parameter of PROTACs is often their cellular activity.<a onclick="showRef(event, 'ref136 ref138'); return false;" href="javascript:void(0);" class="ref ref136 ref138">(136,138)</a> This case study demonstrates the applicability of our in silico macrocyclization approach for PROTACs and might therefore represent an interesting strategy to enhance their drug-likeness by improving cell permeability and cellular stability.</div></div><div id="sec2_4_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i17" class="anchor-spacer"></div><h4 class="article-section__title" id="_i17"> Protein Example: Generation of Bicyclic Structures of the KIX Domain from the Human CREB Binding Protein for Thermal Stabilization (Retrospective Case Study)</h4><div class="NLM_p">The KIX domain (kinase-inducible domain (KID) interaction domain) is an adaptor domain of transcriptional coactivator cAMP response element-binding (CREB) protein with two different protein binding partners and is composed of a central three α-helix bundle (α1, α2, α3). The junction between this bundle and the C-terminal 3<sub>10</sub> helix (G<sub>1</sub>) is crucial for structural integrity (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>a).<a onclick="showRef(event, 'ref139'); return false;" href="javascript:void(0);" class="ref ref139">(139)</a> Recently, Pelay-Gimeno et al. presented an in situ protein stabilization procedure (INCYPRO) that involves the introduction of three surface-exposed cysteine residues placed at different secondary structure elements while still being in spatial proximity.<a onclick="showRef(event, 'ref84'); return false;" href="javascript:void(0);" class="ref ref84">(84)</a> The recombinantly produced construct with cysteine residues in three different positions is then reacted with a triselectrophile for covalent cyclization. With this approach, a bicyclic variant of the KIX domain was designed that was cyclized via positions 594, 599, and 646 and exhibited strongly increased thermal stability while still showing functional binding activity.</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01500/20201019/images/medium/jm0c01500_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01500/20201019/images/large/jm0c01500_0007.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Generation of bicyclic variants of KIX domain from the human CREB binding protein. (a) NMR structure of KIX (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2AGH">2AGH</a>). The junction between the helical bundle (α1, α2, α3) and the C-terminal 310 helix (G1) that was subjected to covalent stabilization is indicated. (b) Cyclized construct that links residues 594, 599, and 646. (c, d) Three-dimensional alignment of snapshots from 100 ns simulations of the linear and cyclized proteins.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01500/20201019/images/large/jm0c01500_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01500&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In this case study, we investigated whether our in silico cyclization approach is able to automatically generate this bicyclic construct. In analogy to the procedure described by Pelay-Gimeno et al., residues around the C-terminal G<sub>1</sub> helix were selected as attachment sites for the generation of trivalently linked structures (see <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>a) using linker <b>3c</b> (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). Applying our procedure for the automatic generation of bicyclic structures (see above), 122 proposals were obtained, including the construct designed by Pelay-Gimeno et al. Both the cyclized construct and the natural protein were subjected to a 100 ns unrestrained MD simulation at an elevated temperature (340 K) in explicit water to rationalize the experimentally observed improvement of the cyclized variant in thermal stability. Although the conformational fluctuation in the lower unconstrained region of the helix bundle is similar between the cyclized and noncyclized variants (see <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>c,d), the region where helix G<sub>1</sub> is covalently linked to α<sub>1</sub> and α<sub>2</sub> is—in agreement with the improved thermal stability data reported by Pelay-Gimeno et al.—conformationally more stable (RMSD<sub>Cα</sub> 1.04 vs 1.72 Å).</div><div class="NLM_p last">This case demonstrates the applicability of in silico cyclization as a suitable method for the engineering of proteins for increased tolerance toward thermal and chemical denaturation.<a onclick="showRef(event, 'ref84'); return false;" href="javascript:void(0);" class="ref ref84">(84)</a></div></div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i18">Summary and Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_55044" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_55044" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Peptidic and nonpeptidic macrocycles represent an underexplored class of drugs with properties that could address current deficiencies of peptide and small-molecule drugs.<a onclick="showRef(event, 'ref96'); return false;" href="javascript:void(0);" class="ref ref96">(96)</a> Therefore, cyclization of molecules from small molecules to peptides and proteins has become an important approach in pharmaceutical design. Here, we have designed, implemented, and applied a computational algorithm to automatically generate cyclized molecules and screen them using conformational and enthalpic scoring. Through case studies on both retrospective and prospective data, we have shown that the method is able to generate numerous reasonable cyclizations, including the ones generated from simple breeding units. The method incorporates user-dictated synthetic capabilities and constraints to ensure proposals that can be straightforwardly synthesized.</div><div class="NLM_p">As we have demonstrated in two prospective case studies (peptidic dual GLP-1/glucagon receptor agonists and small-molecule TAFI inhibitors), the method was able to generate synthesis proposals that resulted in novel molecules that provided new SAR insights. Although all synthesis proposals resulted in highly potent molecules, we observed that there was no linear correlation between the predicted scores and the experimentally determined in vitro potencies, probably due to still existing inaccuracies of scoring functions and docking tools, as, for example, recently reviewed by Guedes et al.<a onclick="showRef(event, 'ref140'); return false;" href="javascript:void(0);" class="ref ref140">(140)</a> Nevertheless, since several studies have demonstrated the general efficiency of scoring functions in enriching active compounds among top scorers in virtual screening applications, we recommend to use enthalpic scoring in a realistic project scenario to reduce the number of virtual hits to a tractable number (e.g., ≤100). These hits should then be subjected to conformational scoring and/or free-energy calculations or the application of other (e.g., tailor-made) scoring functions. Finally, we recommend to inspect the resulting hits with respect to novelty, diversity, and availability of reagents to select the most promising candidates for synthesis.</div><div class="NLM_p">The presented cyclization approach can be applied to different modalities and consider peptidic and nonpeptidic structural motifs. This aspect might, for example, be particularly interesting in a project where an inhibitor of a PPI is searched. After identification of protein residues interacting with the target, the approach can first be used for the design of smaller cyclic peptides. In the next step, this cyclic peptide can be converted to a smaller macrocycle with less peptidic character to optimize cell permeability and oral bioavailability. Potential future advances in predicting permeability<a onclick="showRef(event, 'ref141'); return false;" href="javascript:void(0);" class="ref ref141">(141)</a> and pharmacokinetic properties of macrocycles might further strengthen the applicability of the presented method for the multiparameter optimization of macrocycles.</div><div class="NLM_p last">Taken together, we think that this approach marks a large step forward in accelerated pharmaceutical design via efficient molecular cyclization.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i19">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_21061" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_21061" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> Synthesis of Peptides</h3><div class="NLM_p last">Solid-phase synthesis was carried out on a typical Rink resin (e.g., from Agilent Technologies with a loading of 0.38 mmol/g, 75–150 μm). The Fmoc synthesis strategy was applied with <i>O</i>-(benzotriazole-1-yl)-1,1,3,3-tetramethyluronium hexafluorophosphate (HBTU)/<i>N</i>,<i>N</i>-diisopropylethylamine (DIPEA) activation. The peptide was cleaved from the resin with King’s cocktail.<a onclick="showRef(event, 'ref142'); return false;" href="javascript:void(0);" class="ref ref142">(142)</a> For the synthesis of cyclic lactam containing peptides, orthogonal protecting groups were used (allyl/alloc for the cyclization residues). For the synthesis of hydrocarbon stapled peptides via ring-closing metathesis (RCM), the corresponding amino acids with alkene side chains were used. On resin, cyclization was achieved by treatment of the resin with the Hoveyda–Grubbs second-generation catalyst from Aldrich. Finally, for the synthesis of cysteine alkylated peptides, cross-linking was achieved by addition of the cross-linker (1,3-bis(bromomethyl)benzene for <b>4e</b> and 1,3,5-tris(bromomethyl)benzene for <b>4f</b>) to the Cys deprotected peptides. The crude product was purified via preparative high-performance liquid chromatography (HPLC) on a Waters column (e.g., XBridge, BEH130, Prep C18, 5 μM) using an acetonitrile/water gradient (both buffers with 0.1% trifluoroacetic acid (TFA)). Purity and chemical identity of the product were assessed by ultra performance liquid chromatography (UPLC) and liquid chromatography-mass spectrometry (LC-MS) (for details, see <a href="/doi/suppl/10.1021/acs.jmedchem.0c01500/suppl_file/jm0c01500_si_001.pdf" class="ext-link">Table S1</a>) and confirmed to have ≥95% purity for all key compounds.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> In Vitro Cellular Assays for GLP-1 and Glucagon Receptor Potency</h3><div class="NLM_p last">Agonism of peptides for the two receptors was determined by functional assays measuring cAMP response of HEK-293 cell lines stably expressing human GLP-1 or glucagon receptor. The cells were grown in a T-175 culture flask placed at 37 °C to near confluence in medium (Dulbecco’s modified Eagle’s medium (DMEM)/10% fetal bovine serum (FBS)) and collected in 2 mL vials in cell culture medium containing 10% dimethyl sulfoxide (DMSO) in concentration of 10–50 million cells/mL. Each vial contained 1.8 mL of cells. The vials were slowly frozen to −80 °C in isopropanol and then transferred in liquid nitrogen for storage. Prior to their use, frozen cells were thawed quickly at 37 °C and washed (5 min at 900 rpm) with 20 mL of cell buffer (1× Hank’s balanced salt solution (HBSS); 20 mM 4-(2-hydroxyethyl)-1-piperazineethanesulfonic acid (HEPES), plus 0.1% human serum albumin (HSA)). The cells were resuspended in assay buffer (cell buffer plus 2 mM isobutylmethylxanthine (IBMX)) and adjusted to a cell density of 1 million cells/mL. For measurement of cAMP generation, 5 μL of cells (final 5000 cells/well) and 5 μL of test compound were added to a 384-well plate, followed by incubation for 30 min at room temperature. The cAMP generated was determined using a kit from Cisbio Corp. based on homogeneous time-resolved fluorescence (HTRF). The cAMP assay was performed according to manufacturer’s instructions (Cisbio). After addition of HTRF reagents diluted in lysis buffer (kit components), the plates were incubated for 1 h, followed by measurement of the fluorescence ratio at 665/620 nm. In vitro potency of agonists was quantified by determining the concentrations that caused 50% activation of the maximal response (EC<sub>50</sub>). Reported values are mean (±SEM) EC<sub>50</sub> (pM) from <i>n</i> = 2 values within a single experiment.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> Synthesis of TAFIa Inhibitors</h3><div class="NLM_p last">All solvents used were commercially available and were used without further purification. Reactions were typically run using anhydrous solvents (unless otherwise noted) under an inert atmosphere of argon. Starting materials used were available from commercial sources. <sup>1</sup>H NMR spectra were recorded in the indicated deuterated solvent at 500 MHz. Purity of all compounds tested in biological assays was determined to be ≥95% by LC-MS. Detailed synthesis procedures and analytical data are provided in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c01500/suppl_file/jm0c01500_si_001.pdf" class="ext-link">Supporting Information</a>.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> In Vitro Cellular Assays for TAFIa Inhibition</h3><div class="NLM_p last">The prepared substance was tested for TAFIa inhibition using the Actichrome plasma TAFI Activity Kit from American Diagnostica (Pr. No. 874). This entailed adding 28 μL of assay buffer (20 mM HEPES, 150 mM NaCl, pH 7.4) and 10 μL of TAFIa (American Diagnostica Pr. No. 874TAFIA; 2.5 μg/mL) to 2 μL of 2.5 mM DMSO solution of the substance and incubating in a 96 half-well microtiter plate at room temperature for 15 min. The enzyme reaction was started by adding 10 μL of TAFIa developer (prediluted 1:2 with assay buffer). The time course of the reaction was followed at 420 nm in a microtiter plate reader (SpectraMax plus 384; Molecular Devices) for 15 min. The IC<sub>50</sub> value was calculated from the averaged values (duplicate determination) of serial dilutions of the substance with the aid of the Softmax Pro software (version 4.8; Molecular Devices). Reported values are mean (±SEM) IC<sub>50</sub> from <i>n</i> = 2 values within a single experiment.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> Algorithmic Details of the Cyclizer</h3><div class="NLM_p">The cyclization method proceeds from a complexed ligand to a series of cyclized ligands, which have been geometrically screened. There are two “modes” for the workflow: “small molecule” and “peptide”. Though there are many similarities, some key differences made splitting of the two modes more efficient. Either mode will attempt to construct geometrically stable and chemically reasonable linkers to span the cyclization length in a manner that does not negatively affect the complex structure (e.g., by introducing strain, clashes, or breaking good interactions). For small molecules, there are typically fewer potential attachment points, and a series of “spacers” to span the cyclization length, which can be pieced together to fill the space. For peptides, the attachment points are fixed but numerous (e.g., all side chains), and the linkers tend to be of fixed length (e.g., known side-chain bridges). <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a> shows a breakdown of the two modes.</div><figure id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01500/20201019/images/medium/jm0c01500_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01500/20201019/images/large/jm0c01500_0008.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Workflow for (a) peptides and (b) small molecules.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01500/20201019/images/large/jm0c01500_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01500&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">For specific cases, it is possible to “mix” the peptide and small-molecule approach. In a small-molecule scenario, for example, the user can specify an explicit list of linkers (e.g., based on a list of available chemical reagents) that will not be enumerated and pieced together, but will be directly mapped onto the user-specified attachment points of the ligand. For a peptide, on the other hand, it is possible to evaluate other attachment points than the Cα–Cβ or Cα–Hα vector, for example, for the introduction of head-to-side chain, side chain-to-tail, and head-to-tail cyclizations or for a rescaffolding of the backbone of a peptide within a macrocycle.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> Detailed Description of the Workflow</h3><div id="sec4_6_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> Input</h4><div class="NLM_p last">The user must specify a complexed structure for cyclization, and it may be required to identify the ligand (this is sometimes nontrivial for peptide complexes as peptide ligands are chemically similar to proteins). The user must specify a mode; for the small molecule, mode must specify chemical linkers using SMILES patterns, and for the peptide, mode must specify side-chain bridges either by name or by SMILES. All of these inputs are sanity-checked to confirm no obvious problems.</div></div><div id="sec4_6_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> Limb Recognition</h4><div class="NLM_p last">For peptide cyclizations, typically all side-chain Cα–Cβ and Cα–Hα vectors are considered as potential limbs. This can be further controlled by specifying a subregion to target using Schrodinger’s atom selection language (ASL). This may be especially useful if there is a larger peptide system and there is some knowledge of the system that would preclude some regions from cyclization. For small molecules, users specify criteria for identifying potential branch points (such as ASL, SMARTS, or manually selecting limbs for cyclization).</div></div><div id="sec4_6_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> Small-Molecule Linker Construction</h4><div class="NLM_p last">To limit the combinatorics of permuting linkers, a minimum and maximum linker length is calculated using the recognized limbs. Then, linkers are enumerated using combinations of the specified spacers that could potentially span the distance within the range. These constructed linkers are then filtered using simple chemical rules to avoid unchemical linkers (e.g., avoid −OO– or −NN– linkages).</div></div><div id="sec4_6_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> Limb Pair Filtering</h4><div class="NLM_p last">All combinations of limb pairs are enumerated and filtered hierarchically to eliminate unphysical constructions. The pairs are filtered if they either have too few bonds between them or too few bonds per distance<a onclick="showRef(event, 'ref143'); return false;" href="javascript:void(0);" class="ref ref143">(143)</a> spanned to eliminate inefficient cyclization. Then, the linker is conformationally sampled. Subsequently, using the conformations, the remaining limbs are filtered whether they match the limb distance and orientations seen in the conformational ensemble.</div></div><div id="sec4_6_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> Attach Linker Precursors</h4><div class="NLM_p last">For small molecules, the entire constructed linker is attached to one of the limbs. For peptides, each limb side chain is mutated to either an <span class="smallcaps smallerCapital">l</span>- or <span class="smallcaps smallerCapital">d</span>-alanine (according to the specified chirality), then portions of the linker are attached to either side chain, leaving “dangling” side chains.</div></div><div id="sec4_6_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> Close Linker</h4><div class="NLM_p last">The dangling linkers are then closed by slowly bringing the closing bond atoms closer to each other and finally forming the bond. This process can be time-consuming as the bond closing often runs into steric or strain issues.</div></div><div id="sec4_6_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> Post Minimize and Filter</h4><div class="NLM_p last">Finally, the system is minimized and strain filters are applied to assure no highly strained linkers have been built. The cyclized molecules are then output for further analysis.</div></div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33"> Implementation</h3><div class="NLM_p last">The macrocyclization script has been implemented via command line in Schrodinger software as of release 2020-1. To run, use $SCHRODINGER/run -FROM psp macrocyclize -h.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> Molecular Dynamics Simulations</h3><div class="NLM_p last">MD simulations were carried out using an explicit solvent MD package, Desmond program (version 4.7, Desmond Molecular Dynamics System; D. E. Shaw Research, New York, NY and version 3.1, Maestro-Desmond Interoperability Tools; Schrödinger) with inbuilt optimized potentials for liquid simulation (OPLS 2.1) force field.<a onclick="showRef(event, 'ref144 ref145 ref146'); return false;" href="javascript:void(0);" class="ref ref144 ref145 ref146">(144−146)</a> The proteins were prepared for simulation by first checking their correctness using the protein preparation wizard tool and Epik module was used for deriving the protonation states of the proteins at neutral pH. The system was prepared by placing the proteins in a cubic box with periodic boundary conditions specifying the shape and size of box as 10 Å × 10 Å × 10 Å distance. Predefined TIP3P water model was used as a solvent, and the systems were neutralized by adding an appropriate number of ions. The solvated systems were relaxed by implementing steepest descent and the limited-memory Broyden–Fletcher–Goldfarb–Shanno algorithms in a hybrid manner. The simulation was performed under the NPT ensemble for 100 ns implementing the Berendsen thermostat and barostat methods. The Nose–Hoover thermostat algorithm<a onclick="showRef(event, 'ref147 ref148'); return false;" href="javascript:void(0);" class="ref ref147 ref148">(147,148)</a> was used to maintain a constant temperature, and the Martyna–Tobias–Klein barostat algorithm<a onclick="showRef(event, 'ref149'); return false;" href="javascript:void(0);" class="ref ref149">(149)</a> was employed for maintaining 1 atm of pressure throughout the simulation, respectively. The short-range coulombic interactions were analyzed using the short-range method with a cutoff value of 9.0 Å. The particle mesh Ewald (PME) method<a onclick="showRef(event, 'ref150'); return false;" href="javascript:void(0);" class="ref ref150">(150)</a> was used for treating the long-range electrostatic interactions. All bonds involving hydrogen atoms were constrained using the SHAKE algorithm.<a onclick="showRef(event, 'ref151'); return false;" href="javascript:void(0);" class="ref ref151">(151)</a></div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i35" class="anchor-spacer"></div><h3 class="article-section__title" id="_i35"> Enthalpic Scoring</h3><div class="NLM_p last">For small molecules, binding poses of cyclized molecules were predicted with Glide using integrating macrocycle ring sampling.<a onclick="showRef(event, 'ref111'); return false;" href="javascript:void(0);" class="ref ref111">(111)</a> Enthalpic scoring of the protein–ligand complex was done using the empirical scoring function GlideScore.<a onclick="showRef(event, 'ref118 ref119'); return false;" href="javascript:void(0);" class="ref ref118 ref119">(118,119)</a> For larger systems, the cyclized molecules were minimized in the protein environment, and the molecular mechanics-generalized Born surface area (MM/GBSA) method<a onclick="showRef(event, 'ref120 ref121'); return false;" href="javascript:void(0);" class="ref ref120 ref121">(120,121)</a> was used as enthalpic score.</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i36" class="anchor-spacer"></div><h3 class="article-section__title" id="_i36"> Conformational Scoring</h3><div class="NLM_p last">Conformational scoring was performed using Schrödinger’s macrocycle conformational stability script.<a onclick="showRef(event, 'ref107'); return false;" href="javascript:void(0);" class="ref ref107">(107)</a> Briefly, the script scores compounds on their propensity to maintain the specified bioactive substructure given a sampled ensemble. Here, macrocyclizations of small molecules were scored as described in the paper, using Prime-MCS to generate a conformational ensemble, then using the stability script to estimate the expected RMSD in the bioactive substructure using Boltzmann weighting at 298.13 K. For peptide and protein systems, simple NPT MD was run for 100 ns. The stability script here does not need to Boltzmann weight since the conformers already obey the NPT distribution. The bioactive substructure was defined uniquely for each system as is recommended and the conformational score is provided as average RMSD over the Cα atoms of the bioactive region.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-2"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i37"><a href="/doi/suppl/10.1021/acs.jmedchem.0c01500" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_40415" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_40415" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.0c01500?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01500</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Analytical data with RP-HPLC retention times and molecular masses of the synthesized peptides; IUPAC names of the synthesized peptides; detailed synthesis procedures and analytical data for TAFIa inhibitors; two additional retrospective case studies: a peptide example (hydrocarbon-stapled peptides targeting β-catenin) and a small molecule example (macrocyclic factor XIa inhibitors) (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01500/suppl_file/jm0c01500_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01500/suppl_file/jm0c01500_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01500/suppl_file/jm0c01500_si_001.pdf">jm0c01500_si_001.pdf (525.83 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01500/suppl_file/jm0c01500_si_002.csv">jm0c01500_si_002.csv (10.6 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.0c01500" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_56820" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_56820" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Dan Sindhikara</span> - <span class="hlFld-Affiliation affiliation">Schrodinger,
Inc., 120 West 45th Street, New York, New York 10036, United
States</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#ade9ccc383fec4c3c9c5c4c6ccdfcceddecec5dfc2c9c4c3cac8df83cec2c0"><span class="__cf_email__" data-cfemail="c682a7a8e895afa8a2aeafada7b4a786b5a5aeb4a9a2afa8a1a3b4e8a5a9ab">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Andreas Evers</span> - <span class="hlFld-Affiliation affiliation">Integrated
Drug Discovery, Sanofi-Aventis Deutschland
GmbH, Industriepark Hoechst, 65926 Frankfurt am Main, Germany</span>; 
    <span>Present Address:
                        Discovery & Development Technologies, Merck KGaA, Frankfurter Straße 250, 64293 Darmstadt, Germany</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-4643-1941" title="Orcid link">http://orcid.org/0000-0003-4643-1941</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#25644b41574044560b605340575665484057464e42574a50550b464a48"><span class="__cf_email__" data-cfemail="b9f8d7ddcbdcd8ca97fccfdccbcaf9d4dccbdad2decbd6ccc997dad6d4">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Michael Wagner</span> - <span class="hlFld-Affiliation affiliation">Integrated
Drug Discovery, Sanofi-Aventis Deutschland
GmbH, Industriepark Hoechst, 65926 Frankfurt am Main, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Paraskevi Gkeka</span> - <span class="hlFld-Affiliation affiliation">Integrated
Drug Discovery, Sanofi R&D, 1 Avenue Pierre Brossolette, 91385 Chilly-Mazarin, France</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-0752-3539" title="Orcid link">http://orcid.org/0000-0002-0752-3539</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Stefan Güssregen</span> - <span class="hlFld-Affiliation affiliation">Integrated
Drug Discovery, Sanofi-Aventis Deutschland
GmbH, Industriepark Hoechst, 65926 Frankfurt am Main, Germany</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-1868-9614" title="Orcid link">http://orcid.org/0000-0002-1868-9614</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Garima Tiwari</span> - <span class="hlFld-Affiliation affiliation">Integrated
Drug Discovery, Sanofi-Aventis Deutschland
GmbH, Industriepark Hoechst, 65926 Frankfurt am Main, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Gerhard Hessler</span> - <span class="hlFld-Affiliation affiliation">Integrated
Drug Discovery, Sanofi-Aventis Deutschland
GmbH, Industriepark Hoechst, 65926 Frankfurt am Main, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Engin Yapici</span> - <span class="hlFld-Affiliation affiliation">Schrodinger,
Inc., 120 West 45th Street, New York, New York 10036, United
States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ziyu Li</span> - <span class="hlFld-Affiliation affiliation">Integrated
Drug Discovery, Sanofi-Aventis Deutschland
GmbH, Industriepark Hoechst, 65926 Frankfurt am Main, Germany</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d7e2211-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i39">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_11454" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_11454" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The authors thank R. Loder and R. Dharanipragada for peptide synthesis; H. Köhler and J. Czech for in vitro potency characterization; and C. Poeverlein, E. Feyfant, K. Borrelli, T. Day, and J. Knight for helpful discussions.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">3D</td><td class="NLM_def"><p class="first last">three-dimensional</p></td></tr><tr><td class="NLM_term">alk</td><td class="NLM_def"><p class="first last">alkyl</p></td></tr><tr><td class="NLM_term">ASL</td><td class="NLM_def"><p class="first last">atom selection lanuage</p></td></tr><tr><td class="NLM_term">BCL9</td><td class="NLM_def"><p class="first last">B-cell CLL/lymphoma 9</p></td></tr><tr><td class="NLM_term">BD</td><td class="NLM_def"><p class="first last">bromodomain</p></td></tr><tr><td class="NLM_term">BET</td><td class="NLM_def"><p class="first last">bromodomain and extra-terminal domain</p></td></tr><tr><td class="NLM_term">Brd4</td><td class="NLM_def"><p class="first last">bromodomain 4</p></td></tr><tr><td class="NLM_term">cAMP</td><td class="NLM_def"><p class="first last">cyclic adenosine monophosphate</p></td></tr><tr><td class="NLM_term">CREB</td><td class="NLM_def"><p class="first last">cAMP response element-binding protein</p></td></tr><tr><td class="NLM_term">DCM</td><td class="NLM_def"><p class="first last">dichloromethane</p></td></tr><tr><td class="NLM_term">DIPEA</td><td class="NLM_def"><p class="first last"><i>N</i>,<i>N</i>-diisopropylethylamine</p></td></tr><tr><td class="NLM_term">DMEM</td><td class="NLM_def"><p class="first last">Dulbecco’s modified Eagle’s medium</p></td></tr><tr><td class="NLM_term">DMSO</td><td class="NLM_def"><p class="first last">dimethyl sulfoxide</p></td></tr><tr><td class="NLM_term">dOal</td><td class="NLM_def"><p class="first last"><span class="smallcaps smallerCapital">d</span>-7-octenyl-alanine</p></td></tr><tr><td class="NLM_term">DPP IV</td><td class="NLM_def"><p class="first last">dipeptidyl peptidase IV</p></td></tr><tr><td class="NLM_term">E3</td><td class="NLM_def"><p class="first last">E3 ubiquitin ligase</p></td></tr><tr><td class="NLM_term">Elo</td><td class="NLM_def"><p class="first last">elongin</p></td></tr><tr><td class="NLM_term">FBS</td><td class="NLM_def"><p class="first last">fetal bovine serum</p></td></tr><tr><td class="NLM_term">FEP</td><td class="NLM_def"><p class="first last">free-energy perturbation</p></td></tr><tr><td class="NLM_term">Fmoc</td><td class="NLM_def"><p class="first last">fluorenylmethoxycarbonyl</p></td></tr><tr><td class="NLM_term">GLP-1</td><td class="NLM_def"><p class="first last">glucagon-like peptide-1</p></td></tr><tr><td class="NLM_term">GLP-1R</td><td class="NLM_def"><p class="first last">glucagon-like peptide-1 receptor</p></td></tr><tr><td class="NLM_term">HBSS</td><td class="NLM_def"><p class="first last">Hank’s balanced salt solution</p></td></tr><tr><td class="NLM_term">HBTU</td><td class="NLM_def"><p class="first last"><i>O</i>-(benzotriazole-1-yl)-1,1,3,3-tetramethyluronium hexafluorophosphate</p></td></tr><tr><td class="NLM_term">HEK</td><td class="NLM_def"><p class="first last">human embryonic kidney</p></td></tr><tr><td class="NLM_term">HEPES</td><td class="NLM_def"><p class="first last">4-(2-hydroxyethyl)-1-piperazineethanesulfonic acid</p></td></tr><tr><td class="NLM_term">HPLC</td><td class="NLM_def"><p class="first last">high-performance liquid chromatography</p></td></tr><tr><td class="NLM_term">HSA</td><td class="NLM_def"><p class="first last">human serum albumin</p></td></tr><tr><td class="NLM_term">HTRF</td><td class="NLM_def"><p class="first last">homogeneous time-resolved fluorescence</p></td></tr><tr><td class="NLM_term">IBMX</td><td class="NLM_def"><p class="first last">isobutylmethylxanthine</p></td></tr><tr><td class="NLM_term">LC-MS</td><td class="NLM_def"><p class="first last">liquid chromatography–mass spectrometry</p></td></tr><tr><td class="NLM_term">KIX domain</td><td class="NLM_def"><p class="first last">kinase-inducible domain (KID) interacting domain</p></td></tr><tr><td class="NLM_term">MD</td><td class="NLM_def"><p class="first last">molecular dynamics</p></td></tr><tr><td class="NLM_term">Mes</td><td class="NLM_def"><p class="first last">mesitylene</p></td></tr><tr><td class="NLM_term">MM/GBSA</td><td class="NLM_def"><p class="first last">molecular mechanics-generalized Born surface area</p></td></tr><tr><td class="NLM_term">MW</td><td class="NLM_def"><p class="first last">molecular weight</p></td></tr><tr><td class="NLM_term"><i>m</i>Xyl</td><td class="NLM_def"><p class="first last"><i>meta</i>-Xylyl</p></td></tr><tr><td class="NLM_term">NMR</td><td class="NLM_def"><p class="first last">nuclear magnetic resonance</p></td></tr><tr><td class="NLM_term">PEG</td><td class="NLM_def"><p class="first last">poly(ethylene glycol)</p></td></tr><tr><td class="NLM_term">Prime-MCS</td><td class="NLM_def"><p class="first last">Prime macrocycle conformational sampling</p></td></tr><tr><td class="NLM_term">PROTACs</td><td class="NLM_def"><p class="first last">proteolysis targeting chimeras</p></td></tr><tr><td class="NLM_term">RCM</td><td class="NLM_def"><p class="first last">ring-closing metathesis</p></td></tr><tr><td class="NLM_term">Ro5</td><td class="NLM_def"><p class="first last">rule-of-five</p></td></tr><tr><td class="NLM_term">RMSD</td><td class="NLM_def"><p class="first last">root-mean-square deviation</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure–activity relationship</p></td></tr><tr><td class="NLM_term">SEM</td><td class="NLM_def"><p class="first last">standard error of the mean</p></td></tr><tr><td class="NLM_term">SMARTS</td><td class="NLM_def"><p class="first last">SMILES arbitrary target specification</p></td></tr><tr><td class="NLM_term">SMILES</td><td class="NLM_def"><p class="first last">simplified molecular-input line-entry system</p></td></tr><tr><td class="NLM_term">SrtA</td><td class="NLM_def"><p class="first last"><i>Staphylococcus aureus</i> sortase A</p></td></tr><tr><td class="NLM_term">TAFI</td><td class="NLM_def"><p class="first last">thrombin-activatable fibrinolysis inhibitor</p></td></tr><tr><td class="NLM_term">TBMB</td><td class="NLM_def"><p class="first last">1,3,5-tris(bromomethyl)benzene</p></td></tr><tr><td class="NLM_term">TFA</td><td class="NLM_def"><p class="first last">trifluoroacetic acid</p></td></tr><tr><td class="NLM_term">THF</td><td class="NLM_def"><p class="first last">tetrahydrofuran</p></td></tr><tr><td class="NLM_term">UPLC</td><td class="NLM_def"><p class="first last">ultraperformance liquid chromatography</p></td></tr><tr><td class="NLM_term">VHL</td><td class="NLM_def"><p class="first last">von Hippel–Lindau protein</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i41">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_37668" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_37668" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 151 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dougherty, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pei, D.</span></span> <span> </span><span class="NLM_article-title">Macrocycles as Protein-Protein Interaction Inhibitors</span>. <i>Biochem. J.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>474</i></span>,  <span class="NLM_fpage">1109</span>– <span class="NLM_lpage">1125</span>, <span class="refDoi"> DOI: 10.1042/BCJ20160619</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1042%2FBCJ20160619" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=28298556" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkvVygt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=474&publication_year=2017&pages=1109-1125&author=P.+G.+Doughertyauthor=Z.+Qianauthor=D.+Pei&title=Macrocycles+as+Protein-Protein+Interaction+Inhibitors&doi=10.1042%2FBCJ20160619"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Macrocycles as protein-protein interaction inhibitors</span></div><div class="casAuthors">Dougherty, Patrick G.; Qian, Ziqing; Pei, Dehua</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Journal</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">474</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1109-1125</span>CODEN:
                <span class="NLM_cas:coden">BIJOAK</span>;
        ISSN:<span class="NLM_cas:issn">0264-6021</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">Macrocyclic compds. such as cyclic peptides have emerged as a new and exciting class of drug candidates for inhibition of intracellular protein-protein interactions, which are challenging targets for conventional drug modalities (i.e. small mols. and proteins).  Over the past decade, several complementary technologies have been developed to synthesize macrocycle libraries and screen them for binding to therapeutically relevant targets.  Two different approaches have also been explored to increase the membrane permeability of cyclic peptides.  In this review, we discuss these methods and their applications in the discovery of macrocyclic compds. against protein-protein interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpyxFYrVReYebVg90H21EOLACvtfcHk0liKr70_LqeHyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkvVygt7g%253D&md5=abd5d7af8383fbe5b553d9501c27ce9c</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1042%2FBCJ20160619&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBCJ20160619%26sid%3Dliteratum%253Aachs%26aulast%3DDougherty%26aufirst%3DP.%2BG.%26aulast%3DQian%26aufirst%3DZ.%26aulast%3DPei%26aufirst%3DD.%26atitle%3DMacrocycles%2520as%2520Protein-Protein%2520Interaction%2520Inhibitors%26jtitle%3DBiochem.%2520J.%26date%3D2017%26volume%3D474%26spage%3D1109%26epage%3D1125%26doi%3D10.1042%2FBCJ20160619" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yudin, A. K.</span></span> <span> </span><span class="NLM_article-title">Macrocycles: Lessons from the Distant Past, Recent Developments, and Future Directions</span>. <i>Chem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">30</span>– <span class="NLM_lpage">49</span>, <span class="refDoi"> DOI: 10.1039/C4SC03089C</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1039%2FC4SC03089C" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=28553456" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVGhsLbE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=30-49&author=A.+K.+Yudin&title=Macrocycles%3A+Lessons+from+the+Distant+Past%2C+Recent+Developments%2C+and+Future+Directions&doi=10.1039%2FC4SC03089C"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Macrocycles: lessons from the distant past, recent developments, and future directions</span></div><div class="casAuthors">Yudin, Andrei K.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Science</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">30-49</span>CODEN:
                <span class="NLM_cas:coden">CSHCCN</span>;
        ISSN:<span class="NLM_cas:issn">2041-6520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  A noticeable increase in mol. complexity of drug targets has created an unmet need in the therapeutic agents that are larger than traditional small mols.  Macrocycles, which are cyclic compds. comprising 12 atoms or more, are now recognized as mols. that "are up to the task" to interrogate extended protein interfaces.  However, because macrocycles (particularly the ones based on peptides) are equipped with large polar surface areas, achieving cellular permeability and bioavailability is anything but straightforward.  While one might consider this to be the Achilles' heel of this class of compds., the synthetic community continues to develop creative approaches toward the synthesis of macrocycles and their site-selective modification.  This perspective provides an overview of both mechanistic and structural issues that bear on macrocycles as a unique class of mols.  The reader is offered a historical foray into some of the classic studies that have resulted in the current renaissance of macrocycles.  In addn., an attempt is made to overview the more recent developments that give hope that macrocycles might indeed turn into a useful therapeutic modality.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrCRWplVAQ417Vg90H21EOLACvtfcHk0lie5VhIW-oPuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVGhsLbE&md5=48262b4650417f04063c0180b652a886</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1039%2FC4SC03089C&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC4SC03089C%26sid%3Dliteratum%253Aachs%26aulast%3DYudin%26aufirst%3DA.%2BK.%26atitle%3DMacrocycles%253A%2520Lessons%2520from%2520the%2520Distant%2520Past%252C%2520Recent%2520Developments%252C%2520and%2520Future%2520Directions%26jtitle%3DChem.%2520Sci.%26date%3D2015%26volume%3D6%26spage%3D30%26epage%3D49%26doi%3D10.1039%2FC4SC03089C" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zaretsky, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yudin, A. K.</span></span> <span> </span><span class="NLM_article-title">Recent Advances in the Synthesis of Cyclic Pseudopeptides</span>. <i>Drug Discovery Today Technol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">3</span>– <span class="NLM_lpage">10</span>, <span class="refDoi"> DOI: 10.1016/j.ddtec.2017.11.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1016%2Fj.ddtec.2017.11.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=29249240" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A280%3ADC%252BC1MzitlCjsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2017&pages=3-10&author=S.+Zaretskyauthor=A.+K.+Yudin&title=Recent+Advances+in+the+Synthesis+of+Cyclic+Pseudopeptides&doi=10.1016%2Fj.ddtec.2017.11.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Recent advances in the synthesis of cyclic pseudopeptides</span></div><div class="casAuthors">Zaretsky Serge; Yudin Andrei K</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today. Technologies</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">3-10</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Constrained peptides pose tremendous value in drug discovery.  For example, owing to their large surface areas, they offer novel ways at inhibiting protein-protein interactions.  As this field has grown, it has become desirable to introduce non-peptidic functionality into these rings to enable differentiated structure activity relationships and improved pharmacokinetic properties.  Recent advances in the synthesis of cyclic pseudopeptides include macrocyclization through cysteine alkylation, multicomponent reactions, decarboxylative couplings, and novel stapling chemistry.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRGLPLs76yr96-9TkgmBqTqfW6udTcc2ebir__EUkgvAbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1MzitlCjsA%253D%253D&md5=c46d4e0edfd6f46413e0118e5187900c</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.ddtec.2017.11.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ddtec.2017.11.004%26sid%3Dliteratum%253Aachs%26aulast%3DZaretsky%26aufirst%3DS.%26aulast%3DYudin%26aufirst%3DA.%2BK.%26atitle%3DRecent%2520Advances%2520in%2520the%2520Synthesis%2520of%2520Cyclic%2520Pseudopeptides%26jtitle%3DDrug%2520Discovery%2520Today%2520Technol.%26date%3D2017%26volume%3D26%26spage%3D3%26epage%3D10%26doi%3D10.1016%2Fj.ddtec.2017.11.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mabonga, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kappo, A. P.</span></span> <span> </span><span class="NLM_article-title">Protein-Protein Interaction Modulators: Advances, Successes and Remaining Challenges</span>. <i>Biophys. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">559</span>– <span class="NLM_lpage">581</span>, <span class="refDoi"> DOI: 10.1007/s12551-019-00570-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1007%2Fs12551-019-00570-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=31301019" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtl2rsb3J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2019&pages=559-581&author=L.+Mabongaauthor=A.+P.+Kappo&title=Protein-Protein+Interaction+Modulators%3A+Advances%2C+Successes+and+Remaining+Challenges&doi=10.1007%2Fs12551-019-00570-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Protein-protein interaction modulators: advances, successes and remaining challenges</span></div><div class="casAuthors">Mabonga, Lloyd; Kappo, Abidemi Paul</div><div class="citationInfo"><span class="NLM_cas:title">Biophysical Reviews</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">559-581</span>CODEN:
                <span class="NLM_cas:coden">BRIECG</span>;
        ISSN:<span class="NLM_cas:issn">1867-2450</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  Modulating disease-relevant protein-protein interactions (PPIs) using small-mol. inhibitors is a quite indispensable diagnostic and therapeutic strategy in averting pathophysiol. cues and disease progression.  Over the years, targeting intracellular PPIs as drug design targets has been a challenging task owing to their highly dynamic and expansive interfacial areas (flat, featureless and relatively large).  However, advances in PPI-focused drug discovery technol. have been reported and a few drugs are already on the market, with some potential drug-like candidates already in clin. trials.  In this article, we review the advances, successes and remaining challenges in the application of small mols. as valuable PPI modulators in disease diagnosis and therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKCN2e4SrXuLVg90H21EOLACvtfcHk0lie5VhIW-oPuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtl2rsb3J&md5=02c2c9b4b9d1ccb279c76c21776bb2ec</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1007%2Fs12551-019-00570-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs12551-019-00570-x%26sid%3Dliteratum%253Aachs%26aulast%3DMabonga%26aufirst%3DL.%26aulast%3DKappo%26aufirst%3DA.%2BP.%26atitle%3DProtein-Protein%2520Interaction%2520Modulators%253A%2520Advances%252C%2520Successes%2520and%2520Remaining%2520Challenges%26jtitle%3DBiophys.%2520Rev.%26date%3D2019%26volume%3D11%26spage%3D559%26epage%3D581%26doi%3D10.1007%2Fs12551-019-00570-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abdalla, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGaw, L. J.</span></span> <span> </span><span class="NLM_article-title">Natural Cyclic Peptides as an Attractive Modality for Therapeutics: A Mini Review</span>. <i>Molecules</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>23</i></span>, <span class="NLM_elocation-id">2080</span> <span class="refDoi"> DOI: 10.3390/molecules23082080</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.3390%2Fmolecules23082080" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitVaktrvJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2018&author=M.+A.+Abdallaauthor=L.+J.+McGaw&title=Natural+Cyclic+Peptides+as+an+Attractive+Modality+for+Therapeutics%3A+A+Mini+Review&doi=10.3390%2Fmolecules23082080"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Natural cyclic peptides as an attractive modality for therapeutics: a mini review</span></div><div class="casAuthors">Abdalla, Muna Ali; McGaw, Lyndy J.</div><div class="citationInfo"><span class="NLM_cas:title">Molecules</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2080/1-2080/19</span>CODEN:
                <span class="NLM_cas:coden">MOLEFW</span>;
        ISSN:<span class="NLM_cas:issn">1420-3049</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Peptides are important biomols. which facilitate the understanding of complex biol. processes, which in turn could be serendipitous biol. targets for future drugs.  They are classified as a unique therapeutic niche and will play an important role as fascinating agents in the pharmaceutical landscape.  Until now, more than 40 cyclic peptide drugs are currently in the market, and approx. one new cyclopeptide drug enters the market annually on av.  Interestingly, the majority of clin. approved cyclic peptides are derived from natural sources, such as peptide antibiotics and human peptide hormones.  In this report, the importance of cyclic peptides is discussed, and their role in drug discovery as interesting therapeutic biomols. will be highlighted.  Recently isolated naturally occurring cyclic peptides from microorganisms, sponges, and other sources with a wide range of pharmacol. properties are reviewed herein.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGopsXVTy7L4WrVg90H21EOLACvtfcHk0li-P7WFxfb9tA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitVaktrvJ&md5=4b71e28d331f6131595ac933b6a2f71a</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.3390%2Fmolecules23082080&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fmolecules23082080%26sid%3Dliteratum%253Aachs%26aulast%3DAbdalla%26aufirst%3DM.%2BA.%26aulast%3DMcGaw%26aufirst%3DL.%2BJ.%26atitle%3DNatural%2520Cyclic%2520Peptides%2520as%2520an%2520Attractive%2520Modality%2520for%2520Therapeutics%253A%2520A%2520Mini%2520Review%26jtitle%3DMolecules%26date%3D2018%26volume%3D23%26doi%3D10.3390%2Fmolecules23082080" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ali, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atmaj, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Oosterwijk, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groves, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dömling, A.</span></span> <span> </span><span class="NLM_article-title">Stapled Peptides Inhibitors: A New Window for Target Drug Discovery</span>. <i>Comput. Struct. Biotechnol. J.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">263</span>– <span class="NLM_lpage">281</span>, <span class="refDoi"> DOI: 10.1016/j.csbj.2019.01.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1016%2Fj.csbj.2019.01.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=30867891" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC1MXjvFWgsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2019&pages=263-281&author=A.+M.+Aliauthor=J.+Atmajauthor=N.+Van+Oosterwijkauthor=M.+R.+Grovesauthor=A.+D%C3%B6mling&title=Stapled+Peptides+Inhibitors%3A+A+New+Window+for+Target+Drug+Discovery&doi=10.1016%2Fj.csbj.2019.01.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Stapled Peptides Inhibitors: A New Window for Target Drug Discovery</span></div><div class="casAuthors">Ali, Ameena M.; Atmaj, Jack; Van Oosterwijk, Niels; Groves, Matthew R.; Domling, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Computational and Structural Biotechnology Journal</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">263-281</span>CODEN:
                <span class="NLM_cas:coden">CSBJAC</span>;
        ISSN:<span class="NLM_cas:issn">2001-0370</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Protein-protein interaction (PPI) is a hot topic in clin. research as protein networking has a major impact in human disease.  Such PPIs are potential drugs targets, leading to the need to inhibit/block specific PPIs.  While small mol. inhibitors have had some success and reached clin. trials, they have generally failed to address the flat and large nature of PPI surfaces.  As a result, larger biologics were developed for PPI surfaces and they have successfully targeted PPIs located outside the cell.  However, biologics have low bioavailability and cannot reach intracellular targets.  A novel class -hydrocarbon-stapled α-helical peptides that are synthetic mini-proteins locked into their bioactive structure through site-specific introduction of a chem. linker- has shown promise.  Stapled peptides show an ability to inhibit intracellular PPIs that previously have been intractable with traditional small mol. or biologics, suggesting that they offer a novel therapeutic modality.  In this review, we highlight what stapling adds to natural-mimicking peptides, describe the revolution of synthetic chem. techniques and how current drug discovery approaches have been adapted to stabilize active peptide conformations, including ring-closing metathesis (RCM), lactamisation, cycloaddns. and reversible reactions.  We provide an overview on the available stapled peptide high-resoln. structures in the protein data bank, with four selected structures discussed in details due to remarkable interactions of their staple with the target surface.  We believe that stapled peptides are promising drug candidates and open the doors for peptide therapeutics to reach currently "undruggable" space.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8MjXifoHwd7Vg90H21EOLACvtfcHk0li-P7WFxfb9tA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXjvFWgsrg%253D&md5=e2db5b05352df8c74231579cd0f05ad9</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.csbj.2019.01.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.csbj.2019.01.012%26sid%3Dliteratum%253Aachs%26aulast%3DAli%26aufirst%3DA.%2BM.%26aulast%3DAtmaj%26aufirst%3DJ.%26aulast%3DVan%2BOosterwijk%26aufirst%3DN.%26aulast%3DGroves%26aufirst%3DM.%2BR.%26aulast%3DD%25C3%25B6mling%26aufirst%3DA.%26atitle%3DStapled%2520Peptides%2520Inhibitors%253A%2520A%2520New%2520Window%2520for%2520Target%2520Drug%2520Discovery%26jtitle%3DComput.%2520Struct.%2520Biotechnol.%2520J.%26date%3D2019%26volume%3D17%26spage%3D263%26epage%3D281%26doi%3D10.1016%2Fj.csbj.2019.01.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Giordanetto, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kihlberg, J.</span></span> <span> </span><span class="NLM_article-title">Macrocyclic Drugs and Clinical Candidates: What Can Medicinal Chemists Learn from Their Properties?</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">278</span>– <span class="NLM_lpage">295</span>, <span class="refDoi"> DOI: 10.1021/jm400887j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm400887j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtlGnurjI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=278-295&author=F.+Giordanettoauthor=J.+Kihlberg&title=Macrocyclic+Drugs+and+Clinical+Candidates%3A+What+Can+Medicinal+Chemists+Learn+from+Their+Properties%3F&doi=10.1021%2Fjm400887j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Macrocyclic Drugs and Clinical Candidates: What Can Medicinal Chemists Learn from Their Properties?</span></div><div class="casAuthors">Giordanetto, Fabrizio; Kihlberg, Jan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">278-295</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Macrocycles are ideal in efforts to tackle difficult targets, but our understanding of what makes them cell permeable and orally bioavailable is limited.  Anal. of approx. 100 macrocyclic drugs and clin. candidates revealed that macrocycles are predominantly used for infectious disease and in oncol. and that most belong to the macrolide or cyclic peptide class.  A significant no. (N = 34) of these macrocycles are administered orally, revealing that oral bioavailability can be obtained at mol. wts. up to and above 1 kDa and polar surface areas ranging toward 250 Å2.  Moreover, insight from a group of de novo designed oral macrocycles in clin. studies and understanding of how cyclosporin A and model cyclic hexapeptides cross cell membranes may unlock wider opportunities in drug discovery.  However, the no. of oral macrocycles is still low and it remains to be seen if they are outliers or if macrocycles will open up novel oral druggable space.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpaeM2DXwvmiLVg90H21EOLACvtfcHk0li-P7WFxfb9tA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtlGnurjI&md5=a3e1743bd5ba34f73d8e78f8cb1bee5f</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1021%2Fjm400887j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm400887j%26sid%3Dliteratum%253Aachs%26aulast%3DGiordanetto%26aufirst%3DF.%26aulast%3DKihlberg%26aufirst%3DJ.%26atitle%3DMacrocyclic%2520Drugs%2520and%2520Clinical%2520Candidates%253A%2520What%2520Can%2520Medicinal%2520Chemists%2520Learn%2520from%2520Their%2520Properties%253F%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D278%26epage%3D295%26doi%3D10.1021%2Fjm400887j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, A. C.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khanna, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, J.-H.</span></span> <span> </span><span class="NLM_article-title">A Comprehensive Review on Current Advances in Peptide Drug Development and Design</span>. <i>Int. J. Mol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>20</i></span>, <span class="NLM_elocation-id">2383</span> <span class="refDoi"> DOI: 10.3390/ijms20102383</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.3390%2Fijms20102383" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BB3cXosVGiuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2019&author=A.+C.-L.+Leeauthor=J.+L.+Harrisauthor=K.+K.+Khannaauthor=J.-H.+Hong&title=A+Comprehensive+Review+on+Current+Advances+in+Peptide+Drug+Development+and+Design&doi=10.3390%2Fijms20102383"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">A comprehensive review on current advances in peptide drug development and design</span></div><div class="casAuthors">Lee, Andy Ch-Lung; Harris, Janelle Louise; Khanna, Kum Kum; Hong, Ji-Hong</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Molecular Sciences</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2383/1-2383/21</span>CODEN:
                <span class="NLM_cas:coden">IJMCFK</span>;
        ISSN:<span class="NLM_cas:issn">1422-0067</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Protein-protein interactions (PPIs) execute many fundamental cellular functions and have served as prime drug targets over the last two decades.  Interfering intracellular PPIs with small mols. has been extremely difficult for larger or flat binding sites, as antibodies cannot cross the cell membrane to reach such target sites.  In recent years, peptides smaller size and balance of conformational rigidity and flexibility have made them promising candidates for targeting challenging binding interfaces with satisfactory binding affinity and specificity.  Deciphering and characterizing peptide-protein recognition mechanisms is thus central for the invention of peptide-based strategies to interfere with endogenous protein interactions, or improvement of the binding affinity and specificity of existing approaches.  Importantly, a variety of computation-aided rational designs for peptide therapeutics have been developed, which aim to deliver comprehensive docking for peptide-protein interaction interfaces.  Over 60 peptides have been approved and administrated globally in clinics.  Despite this, advances in various docking models are only on the merge of making their contribution to peptide drug development.  In this review, we provide (i) a holistic overview of peptide drug development and the fundamental technologies utilized to date, and (ii) an updated review on key developments of computational modeling of peptide-protein interactions (PepPIs) with an aim to assist exptl. biologists exploit suitable docking methods to advance peptide interfering strategies against PPIs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbSlnJ71PNHbVg90H21EOLACvtfcHk0lhiByjoV876kQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXosVGiuw%253D%253D&md5=344174f95cb22357d0b92d797874d81b</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.3390%2Fijms20102383&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fijms20102383%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DA.%2BC.-L.%26aulast%3DHarris%26aufirst%3DJ.%2BL.%26aulast%3DKhanna%26aufirst%3DK.%2BK.%26aulast%3DHong%26aufirst%3DJ.-H.%26atitle%3DA%2520Comprehensive%2520Review%2520on%2520Current%2520Advances%2520in%2520Peptide%2520Drug%2520Development%2520and%2520Design%26jtitle%3DInt.%2520J.%2520Mol.%2520Sci.%26date%3D2019%26volume%3D20%26doi%3D10.3390%2Fijms20102383" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Naylor, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bockus, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blanco, M.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lokey, R. S.</span></span> <span> </span><span class="NLM_article-title">Cyclic Peptide Natural Products Chart the Frontier of Oral Bioavailability in the Pursuit of Undruggable Targets</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">141</span>– <span class="NLM_lpage">147</span>, <span class="refDoi"> DOI: 10.1016/j.cbpa.2017.04.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1016%2Fj.cbpa.2017.04.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=28570865" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC2sXovVSgsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2017&pages=141-147&author=M.+R.+Naylorauthor=A.+T.+Bockusauthor=M.-J.+Blancoauthor=R.+S.+Lokey&title=Cyclic+Peptide+Natural+Products+Chart+the+Frontier+of+Oral+Bioavailability+in+the+Pursuit+of+Undruggable+Targets&doi=10.1016%2Fj.cbpa.2017.04.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Cyclic peptide natural products chart the frontier of oral bioavailability in the pursuit of undruggable targets</span></div><div class="casAuthors">Naylor, Matthew R.; Bockus, Andrew T.; Blanco, Maria-Jesus; Lokey, R. Scott</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">141-147</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">As interest in protein-protein interactions and other previously-undruggable targets increases, medicinal chemists are returning to natural products for design inspiration toward mols. that transcend the paradigm of small mol. drugs.  These compds., esp. peptides, often have poor ADME properties and thus require a more nuanced understanding of structure-property relationships to achieve desirable oral bioavailability.  Although there have been few clin. successes in this chem. space to date, recent work has identified opportunities to introduce favorable physicochem. properties to peptidic macrocycles that maintain activity and oral bioavailability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoanPrD1D-ePrVg90H21EOLACvtfcHk0lhiByjoV876kQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXovVSgsLw%253D&md5=175671ba338608979d38d340866df885</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2017.04.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2017.04.012%26sid%3Dliteratum%253Aachs%26aulast%3DNaylor%26aufirst%3DM.%2BR.%26aulast%3DBockus%26aufirst%3DA.%2BT.%26aulast%3DBlanco%26aufirst%3DM.-J.%26aulast%3DLokey%26aufirst%3DR.%2BS.%26atitle%3DCyclic%2520Peptide%2520Natural%2520Products%2520Chart%2520the%2520Frontier%2520of%2520Oral%2520Bioavailability%2520in%2520the%2520Pursuit%2520of%2520Undruggable%2520Targets%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2017%26volume%3D38%26spage%3D141%26epage%3D147%26doi%3D10.1016%2Fj.cbpa.2017.04.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Poongavanam, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Danelius, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peintner, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alcaraz, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caron, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cummings, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wlodek, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erdelyi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hawkins, P. C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ermondi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kihlberg, J.</span></span> <span> </span><span class="NLM_article-title">Conformational Sampling of Macrocyclic Drugs in Different Environments: Can We Find the Relevant Conformations?</span>. <i>ACS Omega</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">11742</span>– <span class="NLM_lpage">11757</span>, <span class="refDoi"> DOI: 10.1021/acsomega.8b01379</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsomega.8b01379" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslKhtrnI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2018&pages=11742-11757&author=V.+Poongavanamauthor=E.+Daneliusauthor=S.+Peintnerauthor=L.+Alcarazauthor=G.+Caronauthor=M.+D.+Cummingsauthor=S.+Wlodekauthor=M.+Erdelyiauthor=P.+C.+D.+Hawkinsauthor=G.+Ermondiauthor=J.+Kihlberg&title=Conformational+Sampling+of+Macrocyclic+Drugs+in+Different+Environments%3A+Can+We+Find+the+Relevant+Conformations%3F&doi=10.1021%2Facsomega.8b01379"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Conformational Sampling of Macrocyclic Drugs in Different Environments: Can We Find the Relevant Conformations?</span></div><div class="casAuthors">Poongavanam, Vasanthanathan; Danelius, Emma; Peintner, Stefan; Alcaraz, Lilian; Caron, Giulia; Cummings, Maxwell D.; Wlodek, Stanislaw; Erdelyi, Mate; Hawkins, Paul C. D.; Ermondi, Giuseppe; Kihlberg, Jan</div><div class="citationInfo"><span class="NLM_cas:title">ACS Omega</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">11742-11757</span>CODEN:
                <span class="NLM_cas:coden">ACSODF</span>;
        ISSN:<span class="NLM_cas:issn">2470-1343</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Conformational flexibility is a major determinant of the properties of macrocycles and other drugs in beyond rule of 5 (bRo5) space.  Prediction of conformations is essential for design of drugs in this space and the authors have evaluated three tools for conformational sampling of a set of ten bRo5 drugs and clin. candidates in polar and apolar environments.  The distance-geometry based OMEGA was found to yield ensembles spanning larger structure and property spaces than the two MD based methods, MOE-LowMode MD (MOE) and MacroModel-Macrocycle Conformational Sampling (MC).  Both MC and OMEGA, but not MOE, generated different ensembles for polar and apolar environments.  All three conformational search methods generated conformers similar to the crystal structure conformers for nine of the ten compds., with OMEGA performing somewhat better than MOE and MC.  MOE and OMEGA found all six conformers of roxithromycin that were identified by NMR in aq. solns., while only OMEGA sampled the three conformers obsd. in chloroform.  The authors suggest that characterization of conformers using mol. descriptors, e.g. the radius of gyration (Rgyr) and polar surface area (PSA), is preferred to energy- or RMSD-based methods for selection of biol. relevant conformers in drug discovery in bRo5 space.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoetOG-yDeciLVg90H21EOLACvtfcHk0lhiByjoV876kQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslKhtrnI&md5=a8722551b161e907daf55146f22b6d57</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Facsomega.8b01379&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsomega.8b01379%26sid%3Dliteratum%253Aachs%26aulast%3DPoongavanam%26aufirst%3DV.%26aulast%3DDanelius%26aufirst%3DE.%26aulast%3DPeintner%26aufirst%3DS.%26aulast%3DAlcaraz%26aufirst%3DL.%26aulast%3DCaron%26aufirst%3DG.%26aulast%3DCummings%26aufirst%3DM.%2BD.%26aulast%3DWlodek%26aufirst%3DS.%26aulast%3DErdelyi%26aufirst%3DM.%26aulast%3DHawkins%26aufirst%3DP.%2BC.%2BD.%26aulast%3DErmondi%26aufirst%3DG.%26aulast%3DKihlberg%26aufirst%3DJ.%26atitle%3DConformational%2520Sampling%2520of%2520Macrocyclic%2520Drugs%2520in%2520Different%2520Environments%253A%2520Can%2520We%2520Find%2520the%2520Relevant%2520Conformations%253F%26jtitle%3DACS%2520Omega%26date%3D2018%26volume%3D3%26spage%3D11742%26epage%3D11757%26doi%3D10.1021%2Facsomega.8b01379" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dougherty, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pei, D.</span></span> <span> </span><span class="NLM_article-title">Targeting Intracellular Protein-Protein Interactions with Cell-Permeable Cyclic Peptides</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">80</span>– <span class="NLM_lpage">86</span>, <span class="refDoi"> DOI: 10.1016/j.cbpa.2017.03.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1016%2Fj.cbpa.2017.03.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=28388463" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC2sXls1KlsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2017&pages=80-86&author=Z.+Qianauthor=P.+G.+Doughertyauthor=D.+Pei&title=Targeting+Intracellular+Protein-Protein+Interactions+with+Cell-Permeable+Cyclic+Peptides&doi=10.1016%2Fj.cbpa.2017.03.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting intracellular protein-protein interactions with cell-permeable cyclic peptides</span></div><div class="casAuthors">Qian, Ziqing; Dougherty, Patrick G.; Pei, Dehua</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">80-86</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Intracellular protein-protein interactions (PPIs) are challenging targets for conventional drug modalities, because small mols. generally do not bind to their large, flat binding sites with high affinity, whereas monoclonal antibodies cannot cross the cell membrane to reach the targets.  Cyclic peptides in the 700-2000 mol.-wt. range have the sufficient size and a balanced conformational flexibility/rigidity for binding to flat PPI interfaces with antibody-like affinity and specificity.  Several powerful cyclic peptide library technologies were developed over the past decade to rapidly discover potent, specific cyclic peptide ligands against proteins of interest including those involved in PPIs.  Methods are also being developed to enhance the membrane permeability of cyclic peptides through both passive diffusion and active transport mechanisms.  Integration of the permeability-enhancing elements into cyclic peptide design has led to an increasing no. of cell-permeable and biol. active cyclic peptides against intracellular PPIs.  In this account, we review the recent developments in the design and synthesis of cell-permeable cyclic peptides.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_IPrrU8rcMbVg90H21EOLACvtfcHk0lhiByjoV876kQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXls1KlsLg%253D&md5=2ea4ac695ee1f1651e5da4ed0bb7cf0b</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2017.03.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2017.03.011%26sid%3Dliteratum%253Aachs%26aulast%3DQian%26aufirst%3DZ.%26aulast%3DDougherty%26aufirst%3DP.%2BG.%26aulast%3DPei%26aufirst%3DD.%26atitle%3DTargeting%2520Intracellular%2520Protein-Protein%2520Interactions%2520with%2520Cell-Permeable%2520Cyclic%2520Peptides%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2017%26volume%3D38%26spage%3D80%26epage%3D86%26doi%3D10.1016%2Fj.cbpa.2017.03.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Robertson, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spring, D. R.</span></span> <span> </span><span class="NLM_article-title">Using Peptidomimetics and Constrained Peptides as Valuable Tools for Inhibiting Protein–Protein Interactions</span>. <i>Molecules</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>23</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">959</span>– <span class="NLM_lpage">976</span>, <span class="refDoi"> DOI: 10.3390/molecules23040959</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.3390%2Fmolecules23040959" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslGmsbrE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2018&pages=959-976&issue=4&author=N.+S.+Robertsonauthor=D.+R.+Spring&title=Using+Peptidomimetics+and+Constrained+Peptides+as+Valuable+Tools+for+Inhibiting+Protein%E2%80%93Protein+Interactions&doi=10.3390%2Fmolecules23040959"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Using peptidomimetics and constrained peptides as valuable tools for inhibiting protein-protein interactions</span></div><div class="casAuthors">Robertson, Naomi S.; Spring, David R.</div><div class="citationInfo"><span class="NLM_cas:title">Molecules</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">959/1-959/18</span>CODEN:
                <span class="NLM_cas:coden">MOLEFW</span>;
        ISSN:<span class="NLM_cas:issn">1420-3049</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Protein-protein interactions (PPIs) are tremendously important for the function of many biol. processes.  However, because of the structure of many protein-protein interfaces (flat, featureless and relatively large), they have largely been overlooked as potential drug targets.  In this review, we highlight the current tools used to study the mol. recognition of PPIs through the use of different peptidomimetics, from small mols. and scaffolds to peptides.  Then, we focus on constrained peptides, and in particular, ways to constrain -helixes through stapling using both one- and two-component techniques.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrwxPEMuLK6abVg90H21EOLACvtfcHk0lioXrqbRR0HHQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslGmsbrE&md5=480718d55ee9814eba533868e249faf1</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.3390%2Fmolecules23040959&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fmolecules23040959%26sid%3Dliteratum%253Aachs%26aulast%3DRobertson%26aufirst%3DN.%2BS.%26aulast%3DSpring%26aufirst%3DD.%2BR.%26atitle%3DUsing%2520Peptidomimetics%2520and%2520Constrained%2520Peptides%2520as%2520Valuable%2520Tools%2520for%2520Inhibiting%2520Protein%25E2%2580%2593Protein%2520Interactions%26jtitle%3DMolecules%26date%3D2018%26volume%3D23%26issue%3D4%26spage%3D959%26epage%3D976%26doi%3D10.3390%2Fmolecules23040959" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rubin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qvit, N.</span></span> <span> </span><span class="NLM_article-title">Cyclic Peptides for Protein-Protein Interaction Targets: Applications to Human Disease</span>. <i>Crit. Rev. Eukaryotic Gene Expression</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">199</span>– <span class="NLM_lpage">221</span>, <span class="refDoi"> DOI: 10.1615/CritRevEukaryotGeneExpr.2016016525</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1615%2FCritRevEukaryotGeneExpr.2016016525" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=27650985" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A280%3ADC%252BC2svis1ensg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2016&pages=199-221&author=S.+Rubinauthor=N.+Qvit&title=Cyclic+Peptides+for+Protein-Protein+Interaction+Targets%3A+Applications+to+Human+Disease&doi=10.1615%2FCritRevEukaryotGeneExpr.2016016525"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Cyclic Peptides for Protein-Protein Interaction Targets: Applications to Human Disease</span></div><div class="casAuthors">Rubin Samuel; Qvit Nir</div><div class="citationInfo"><span class="NLM_cas:title">Critical reviews in eukaryotic gene expression</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">199-221</span>
        ISSN:<span class="NLM_cas:issn">1045-4403</span>.
    </div><div class="casAbstract">Protein-protein interactions (PPIs) represent a significant portion of functionally relevant biological interactions, and therefore potential therapeutic targets.  Small molecules were traditionally used to target PPIs.  However, many PPI surfaces lack binding pockets due to their large and flat structures.  Antibodies can also be used to modulate PPIs, but they are expensive and not cell permeable.  Linear peptides are less expensive to produce than antibodies and are generally more selective than small molecules, but they are limited by decreased stability and poor permeability.  Modified peptides (peptidomimetics, e.g., cyclic peptides) can overcome these obstacles.  Advantages of using cyclic peptidomimetics to modulate PPIs derive from their conformational constraint, which supports target specificity, cell permeability, and metabolic stability.  Methods for rational design coupled with high-throughput techniques continue to support advances in the field.  Further development of cyclic peptidomimetics to modulate PPIs will improve treatment of human diseases, such as cancer, infection, neurodegeneration, and autoimmunity.  Here we describe several cyclic peptidomimetics that are currently used as drugs and many potential cyclic peptides PPI inhibitors in different stages of pre-clinical and clinical development.  Further development of cyclic peptidomimetics to modulate PPIs will continue to improve treatment of human diseases, such as cancer, infection, neurodegeneration, and autoimmunity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRv6Xi9HeQ4puQ4Pl9hzjQhfW6udTcc2eb2HjAh5zl3J7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2svis1ensg%253D%253D&md5=69772e8646e86a3e209a4328b88a83a1</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1615%2FCritRevEukaryotGeneExpr.2016016525&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1615%252FCritRevEukaryotGeneExpr.2016016525%26sid%3Dliteratum%253Aachs%26aulast%3DRubin%26aufirst%3DS.%26aulast%3DQvit%26aufirst%3DN.%26atitle%3DCyclic%2520Peptides%2520for%2520Protein-Protein%2520Interaction%2520Targets%253A%2520Applications%2520to%2520Human%2520Disease%26jtitle%3DCrit.%2520Rev.%2520Eukaryotic%2520Gene%2520Expression%26date%3D2016%26volume%3D26%26spage%3D199%26epage%3D221%26doi%3D10.1615%2FCritRevEukaryotGeneExpr.2016016525" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Russo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aiello, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grieco, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marasco, D.</span></span> <span> </span><span class="NLM_article-title">Targeting “Undruggable” Proteins: Design of Synthetic Cyclopeptides</span>. <i>Curr. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">748</span>– <span class="NLM_lpage">762</span>, <span class="refDoi"> DOI: 10.2174/0929867323666160112122540</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.2174%2F0929867323666160112122540" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=26758797" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC28Xlt1KjtL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2016&pages=748-762&author=A.+Russoauthor=C.+Aielloauthor=P.+Griecoauthor=D.+Marasco&title=Targeting+%E2%80%9CUndruggable%E2%80%9D+Proteins%3A+Design+of+Synthetic+Cyclopeptides&doi=10.2174%2F0929867323666160112122540"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting "Undruggable" Proteins: Design of Synthetic Cyclopeptides</span></div><div class="casAuthors">Russo, Anna; Aiello, Carmela; Grieco, Paolo; Marasco, Daniela</div><div class="citationInfo"><span class="NLM_cas:title">Current Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">748-762</span>CODEN:
                <span class="NLM_cas:coden">CMCHE7</span>;
        ISSN:<span class="NLM_cas:issn">0929-8673</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">The development of synthetic macrocycles represents a powerful approach toward the identification of new protein binders or inhibitors of Protein-Protein Interactions (PPI) which are known to play key biol. roles in cancer signaling as well as in the regulation of cell division cycle.  Structural investigations led to identify "hot loops" sharing common motifs that are mainly involved in PPIs.  Most PPIs occur through large and flat surfaces; currently these protein complexes are defined as "undruggable" by conventional drug-discovery approaches, since the identification of small mols. to inhibit these targets is often unreachable.  Typically macrocycles are 500-2000 Da in size, having 12-membered, or more, ring architecture: they do not obey the Lipinski's rule but, for them nature offers many examples as therapeutic agents such as erythromycin (antibiotic), cyclosporin (immunosuppressant) and somatostatin (hormone).  Peptide-based macrocycles offer the advantages of directly mimicking secondary structures involved in PPIs and their pharmacol. application is related to the potential improvement of lead peptides in terms of potency, selectivity, stability and cell permeation.  The promising relevance of cyclopeptides prompted to develop new synthetic methods for cyclization: often biotechnol. approaches as well as regioselective reactions have been employed to cyclize peptides rapidly and nearly quant.  Moreover, different synthetic strategies in peptidomimetics' macrocyclization are actually available based on surrogate peptide bonds or NCL (Native Chem. Ligation) methods.  In this review we focus on the most common methods for the prepn. of cyclopeptides and interesting applications of the last decade.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZqHzNKiqC0bVg90H21EOLACvtfcHk0lioXrqbRR0HHQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xlt1KjtL0%253D&md5=c066116e3227a90037c0e3c77af6e50c</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.2174%2F0929867323666160112122540&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F0929867323666160112122540%26sid%3Dliteratum%253Aachs%26aulast%3DRusso%26aufirst%3DA.%26aulast%3DAiello%26aufirst%3DC.%26aulast%3DGrieco%26aufirst%3DP.%26aulast%3DMarasco%26aufirst%3DD.%26atitle%3DTargeting%2520%25E2%2580%259CUndruggable%25E2%2580%259D%2520Proteins%253A%2520Design%2520of%2520Synthetic%2520Cyclopeptides%26jtitle%3DCurr.%2520Med.%2520Chem.%26date%3D2016%26volume%3D23%26spage%3D748%26epage%3D762%26doi%3D10.2174%2F0929867323666160112122540" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stone, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deber, C. M.</span></span> <span> </span><span class="NLM_article-title">Therapeutic Design of Peptide Modulators of Protein-Protein Interactions in Membranes</span>. <i>Biochim. Biophys. Acta, Biomembr.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>1859</i></span>,  <span class="NLM_fpage">577</span>– <span class="NLM_lpage">585</span>, <span class="refDoi"> DOI: 10.1016/j.bbamem.2016.08.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1016%2Fj.bbamem.2016.08.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=27580024" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVegsrbE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1859&publication_year=2017&pages=577-585&author=T.+A.+Stoneauthor=C.+M.+Deber&title=Therapeutic+Design+of+Peptide+Modulators+of+Protein-Protein+Interactions+in+Membranes&doi=10.1016%2Fj.bbamem.2016.08.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Therapeutic design of peptide modulators of protein-protein interactions in membranes</span></div><div class="casAuthors">Stone, Tracy A.; Deber, Charles M.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Biomembranes</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">1859</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">577-585</span>CODEN:
                <span class="NLM_cas:coden">BBBMBS</span>;
        ISSN:<span class="NLM_cas:issn">0005-2736</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Membrane proteins play the central roles in a variety of cellular processes, ranging from nutrient uptake and signalling, to cell-cell communication.  Their biol. functions are directly related to how they fold and assemble; defects often lead to disease.  Protein-protein interactions (PPIs) within the membrane are therefore of great interest as therapeutic targets.  Here we review the progress in the application of membrane-insertable peptides for the disruption or stabilization of membrane-based PPIs.  We describe the design and prepn. of transmembrane peptide mimics; and of several categories of peptidomimetics used for study, including D-enantiomers, non-natural amino acids, peptoids, and β-peptides.  Further aspects of the review describe modifications to membrane-insertable peptides, including lipidation and cyclization via hydrocarbon stapling.  These approaches provide a pathway toward the development of metabolically stable, non-toxic, and efficacious peptide modulators of membrane-based PPIs.  This article is part of a Special Issue entitled: Lipid order/lipid defects and lipid-control of protein activity edited by Dirk Schneider.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoIYM33lXE3IbVg90H21EOLACvtfcHk0ljm55kYU--O3A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVegsrbE&md5=c89943990f8a9f222e7cf6089f8b3f42</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.bbamem.2016.08.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbamem.2016.08.013%26sid%3Dliteratum%253Aachs%26aulast%3DStone%26aufirst%3DT.%2BA.%26aulast%3DDeber%26aufirst%3DC.%2BM.%26atitle%3DTherapeutic%2520Design%2520of%2520Peptide%2520Modulators%2520of%2520Protein-Protein%2520Interactions%2520in%2520Membranes%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Biomembr.%26date%3D2017%26volume%3D1859%26spage%3D577%26epage%3D585%26doi%3D10.1016%2Fj.bbamem.2016.08.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wójcik, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berlicki, Ł.</span></span> <span> </span><span class="NLM_article-title">Peptide-Based Inhibitors of Protein-Protein Interactions</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">707</span>– <span class="NLM_lpage">713</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2015.12.084</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1016%2Fj.bmcl.2015.12.084" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=26764190" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC28XitFKlsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2016&pages=707-713&author=P.+W%C3%B3jcikauthor=%C5%81.+Berlicki&title=Peptide-Based+Inhibitors+of+Protein-Protein+Interactions&doi=10.1016%2Fj.bmcl.2015.12.084"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Peptide-based inhibitors of protein-protein interactions</span></div><div class="casAuthors">Wojcik, Paulina; Berlicki, Lukasz</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">707-713</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Protein-protein interactions (PPIs) are key elements of several important biol. processes and have emerged as valuable targets in medicinal chem.  Importantly, numerous specific protein-protein interactions (e.g., p53-HDM2 and Bcl-2-BH3 domains) were found to be involved in the development of several diseases, including various types of cancer.  In general, the discovery of new synthetic PPI inhibitors is a challenging task because protein surfaces have not evolved in a manner that allows for specific binding of low mol. wt. compds.  Here, we review the discovery strategies for peptide-based PPI inhibitors.  Although peptide-based drug candidates exhibit significant drawbacks (in particular, low proteolytic stability), modifications of either the side chains or backbone could provide mols. of interest.  Moreover, due to the large mol. size of peptide-based compds., the discovery of mols. that specifically interact with extended protein surfaces is possible.  Two major strategies for constructing peptide-based PPI inhibitors are as follows: (a) cyclization (e.g., stapled peptides) and (b) modification of the backbone structure (e.g., β-peptides and peptoids).  These approaches for constructing PPI inhibitors enhance both the inhibitory activity and pharmacokinetic properties compared with non-modified α-peptides.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzQ6u5WIbqyrVg90H21EOLACvtfcHk0ljm55kYU--O3A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitFKlsw%253D%253D&md5=e4c33a3743f9a6d6829ac3c2e5904f01</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2015.12.084&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2015.12.084%26sid%3Dliteratum%253Aachs%26aulast%3DW%25C3%25B3jcik%26aufirst%3DP.%26aulast%3DBerlicki%26aufirst%3D%25C5%2581.%26atitle%3DPeptide-Based%2520Inhibitors%2520of%2520Protein-Protein%2520Interactions%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2016%26volume%3D26%26spage%3D707%26epage%3D713%26doi%3D10.1016%2Fj.bmcl.2015.12.084" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yamagishi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoji, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miyagawa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawakami, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katoh, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goto, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suga, H.</span></span> <span> </span><span class="NLM_article-title">Natural Product-like Macrocyclic N-Methyl-Peptide Inhibitors against a Ubiquitin Ligase Uncovered from a Ribosome-Expressed de Novo Library</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">1562</span>– <span class="NLM_lpage">1570</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2011.09.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1016%2Fj.chembiol.2011.09.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=22195558" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs12htrnI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2011&pages=1562-1570&author=Y.+Yamagishiauthor=I.+Shojiauthor=S.+Miyagawaauthor=T.+Kawakamiauthor=T.+Katohauthor=Y.+Gotoauthor=H.+Suga&title=Natural+Product-like+Macrocyclic+N-Methyl-Peptide+Inhibitors+against+a+Ubiquitin+Ligase+Uncovered+from+a+Ribosome-Expressed+de+Novo+Library&doi=10.1016%2Fj.chembiol.2011.09.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Natural Product-Like Macrocyclic N-Methyl-Peptide Inhibitors against a Ubiquitin Ligase Uncovered from a Ribosome-Expressed De Novo Library</span></div><div class="casAuthors">Yamagishi, Yusuke; Shoji, Ikuo; Miyagawa, Shoji; Kawakami, Takashi; Katoh, Takayuki; Goto, Yuki; Suga, Hiroaki</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1562-1570</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Summary: Naturally occurring peptides often possess macrocyclic and N-methylated backbone.  These features grant them structural rigidity, high affinity to targets, proteolytic resistance, and occasionally membrane permeability.  Because such peptides are produced by either nonribosomal peptide synthetases or enzymic posttranslational modifications, it is yet a formidable challenge in degenerating sequence or length and prepg. libraries for screening bioactive mols.  Here, we report a new means of synthesizing a de novo library of "natural product-like" macrocyclic N-methyl-peptides using translation machinery under the reprogrammed genetic code, which is coupled with an in vitro display technique, referred to as RaPID (random nonstandard peptides integrated discovery) system.  This system allows for rapid selection of strong binders against an arbitrarily chosen therapeutic target.  Here, we have demonstrated the selection of anti-E6AP macrocyclic N-methyl-peptides, one of which strongly inhibits polyubiquitination of proteins such as p53.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5j7qHsUITV7Vg90H21EOLACvtfcHk0ljm55kYU--O3A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs12htrnI&md5=14dd193429cd72bfb8dc87a9be5a7cf9</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2011.09.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2011.09.013%26sid%3Dliteratum%253Aachs%26aulast%3DYamagishi%26aufirst%3DY.%26aulast%3DShoji%26aufirst%3DI.%26aulast%3DMiyagawa%26aufirst%3DS.%26aulast%3DKawakami%26aufirst%3DT.%26aulast%3DKatoh%26aufirst%3DT.%26aulast%3DGoto%26aufirst%3DY.%26aulast%3DSuga%26aufirst%3DH.%26atitle%3DNatural%2520Product-like%2520Macrocyclic%2520N-Methyl-Peptide%2520Inhibitors%2520against%2520a%2520Ubiquitin%2520Ligase%2520Uncovered%2520from%2520a%2520Ribosome-Expressed%2520de%2520Novo%2520Library%26jtitle%3DChem.%2520Biol.%26date%3D2011%26volume%3D18%26spage%3D1562%26epage%3D1570%26doi%3D10.1016%2Fj.chembiol.2011.09.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zorzi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deyle, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heinis, C.</span></span> <span> </span><span class="NLM_article-title">Cyclic Peptide Therapeutics: Past, Present and Future</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">24</span>– <span class="NLM_lpage">29</span>, <span class="refDoi"> DOI: 10.1016/j.cbpa.2017.02.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1016%2Fj.cbpa.2017.02.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=28249193" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjsFGksLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2017&pages=24-29&author=A.+Zorziauthor=K.+Deyleauthor=C.+Heinis&title=Cyclic+Peptide+Therapeutics%3A+Past%2C+Present+and+Future&doi=10.1016%2Fj.cbpa.2017.02.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Cyclic peptide therapeutics: past, present and future</span></div><div class="casAuthors">Zorzi, Alessandro; Deyle, Kaycie; Heinis, Christian</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">24-29</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Cyclic peptides combine several favorable properties such as good binding affinity, target selectivity and low toxicity that make them an attractive modality for the development of therapeutics.  Over 40 cyclic peptide drugs are currently in clin. use and around one new cyclic peptide drug enters the market every year on av.  The vast majority of clin. approved cyclic peptides are derived from natural products, such as antimicrobials or human peptide hormones.  New powerful techniques based on rational design and in vitro evolution have enabled the de novo development of cyclic peptide ligands to targets for which nature does not offer solns.  A look at the cyclic peptides currently under clin. evaluation shows that several have been developed using such techniques.  This new source for cyclic peptide ligands introduces a freshness to the field, and it is likely that de novo developed cyclic peptides will be in clin. use in the near future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwchcMOj1bSbVg90H21EOLACvtfcHk0ljm55kYU--O3A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjsFGksLg%253D&md5=4b9581365b92b649dad8c7b22c5bd9c3</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2017.02.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2017.02.006%26sid%3Dliteratum%253Aachs%26aulast%3DZorzi%26aufirst%3DA.%26aulast%3DDeyle%26aufirst%3DK.%26aulast%3DHeinis%26aufirst%3DC.%26atitle%3DCyclic%2520Peptide%2520Therapeutics%253A%2520Past%252C%2520Present%2520and%2520Future%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2017%26volume%3D38%26spage%3D24%26epage%3D29%26doi%3D10.1016%2Fj.cbpa.2017.02.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ermert, P.</span></span> <span> </span><span class="NLM_article-title">Design, Properties and Recent Application of Macrocycles in Medicinal Chemistry</span>. <i>Chimia</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>71</i></span>,  <span class="NLM_fpage">678</span>– <span class="NLM_lpage">702</span>, <span class="refDoi"> DOI: 10.2533/chimia.2017.678</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.2533%2Fchimia.2017.678" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=29070413" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsFeitbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2017&pages=678-702&author=P.+Ermert&title=Design%2C+Properties+and+Recent+Application+of+Macrocycles+in+Medicinal+Chemistry&doi=10.2533%2Fchimia.2017.678"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Design, properties and recent application of macrocycles in medicinal chemistry</span></div><div class="casAuthors">Ermert, Philipp</div><div class="citationInfo"><span class="NLM_cas:title">Chimia</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">71</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">678-702</span>CODEN:
                <span class="NLM_cas:coden">CHIMAD</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Swiss Chemical Society</span>)
        </div><div class="casAbstract">Macrocyclic compds. have recently received increasing attention in drug discovery as these compds. offer the potential to modulate difficult targets and to access novel chemotypes.  Approaches towards libraries of macrocyclic compds. based on modular org. synthesis and applications of these technol. platforms to find and improve biol. active compds. are introduced in this minireview.  Alternatively, lead compds. may be obtained by truncation and modification of macrocyclic natural products.  Selected medicinal chem. programs are discussed, illustrating a macrocyclization approach toward ligands with improved properties.  The design of such ligands is often informed by X-ray crystal structures of protein-ligand complexes.  Efforts to understand cellular permeability and oral bioavailability of cyclic peptides and non-peptidic macrocycles are summarized.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqo_L_ef2BYLrVg90H21EOLACvtfcHk0lhJwdcLemNkDw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsFeitbg%253D&md5=aca0ab87a30863de362d9002c3d9f6cc</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.2533%2Fchimia.2017.678&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2533%252Fchimia.2017.678%26sid%3Dliteratum%253Aachs%26aulast%3DErmert%26aufirst%3DP.%26atitle%3DDesign%252C%2520Properties%2520and%2520Recent%2520Application%2520of%2520Macrocycles%2520in%2520Medicinal%2520Chemistry%26jtitle%3DChimia%26date%3D2017%26volume%3D71%26spage%3D678%26epage%3D702%26doi%3D10.2533%2Fchimia.2017.678" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ermert, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luther, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zbinden, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Obrecht, D.</span></span> <span> </span><span class="NLM_article-title">Frontier Between Cyclic Peptides and Macrocycles</span>. <i>Methods Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>2001</i></span>,  <span class="NLM_fpage">147</span>– <span class="NLM_lpage">202</span>, <span class="refDoi"> DOI: 10.1007/978-1-4939-9504-2_9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1007%2F978-1-4939-9504-2_9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=31134572" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC1MXit1eis7zE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2001&publication_year=2019&pages=147-202&author=P.+Ermertauthor=A.+Lutherauthor=P.+Zbindenauthor=D.+Obrecht&title=Frontier+Between+Cyclic+Peptides+and+Macrocycles&doi=10.1007%2F978-1-4939-9504-2_9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Frontier between cyclic peptides and macrocycles</span></div><div class="casAuthors">Ermert, Philipp; Luther, Anatol; Zbinden, Peter; Obrecht, Daniel</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Molecular Biology (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">2001</span>
        (<span class="NLM_cas:issue">Cyclic Peptide Design</span>),
    <span class="NLM_cas:pages">147-202</span>CODEN:
                <span class="NLM_cas:coden">MMBIED</span>;
        ISSN:<span class="NLM_cas:issn">1940-6029</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">This review describes a selection of macrocyclic natural products and structurally modified analogs contg. peptidic and non-peptidic elements as structural features that potentially modulate cellular permeability.  Examples range from exclusively peptidic structures like cyclosporin A or phepropeptins to compds. with mostly non-peptidic character, such as telomestatin or largazole.  Furthermore, semisynthetic approaches and synthesis platforms to generate general and focused libraries of compds. at the interface of cyclic peptides and non-peptidic macrocycles are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomwEAzfYUWmbVg90H21EOLACvtfcHk0lhJwdcLemNkDw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXit1eis7zE&md5=6526545a636d7b569524afab9a13b517</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1007%2F978-1-4939-9504-2_9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-1-4939-9504-2_9%26sid%3Dliteratum%253Aachs%26aulast%3DErmert%26aufirst%3DP.%26aulast%3DLuther%26aufirst%3DA.%26aulast%3DZbinden%26aufirst%3DP.%26aulast%3DObrecht%26aufirst%3DD.%26atitle%3DFrontier%2520Between%2520Cyclic%2520Peptides%2520and%2520Macrocycles%26jtitle%3DMethods%2520Mol.%2520Biol.%26date%3D2019%26volume%3D2001%26spage%3D147%26epage%3D202%26doi%3D10.1007%2F978-1-4939-9504-2_9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lau, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dunn, M. K.</span></span> <span> </span><span class="NLM_article-title">Therapeutic Peptides: Historical Perspectives, Current Development Trends, and Future Directions</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">2700</span>– <span class="NLM_lpage">2707</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2017.06.052</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1016%2Fj.bmc.2017.06.052" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=28720325" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtFygt7jP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2018&pages=2700-2707&author=J.+L.+Lauauthor=M.+K.+Dunn&title=Therapeutic+Peptides%3A+Historical+Perspectives%2C+Current+Development+Trends%2C+and+Future+Directions&doi=10.1016%2Fj.bmc.2017.06.052"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Therapeutic peptides: Historical perspectives, current development trends, and future directions</span></div><div class="casAuthors">Lau, Jolene L.; Dunn, Michael K.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2700-2707</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Peptide therapeutics have played a notable role in medical practice since the advent of insulin therapy in the 1920s.  Over 60 peptide drugs are approved in the United States and other major markets, and peptides continue to enter clin. development at a steady pace.  Peptide drug discovery has diversified beyond its traditional focus on endogenous human peptides to include a broader range of structures identified from other natural sources or through medicinal chem. efforts.  The authors maintain a comprehensive dataset on peptides that have entered human clin. studies that includes over 150 peptides in active development today.  Here the authors provide an overview of the peptide therapeutic landscape, including historical perspectives, mol. characteristics, regulatory benchmarks, and a therapeutic area breakdown.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqtmwxU3yil6LVg90H21EOLACvtfcHk0lhJwdcLemNkDw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtFygt7jP&md5=c4dc775ee302737cfc5080ea3f1c8c74</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2017.06.052&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2017.06.052%26sid%3Dliteratum%253Aachs%26aulast%3DLau%26aufirst%3DJ.%2BL.%26aulast%3DDunn%26aufirst%3DM.%2BK.%26atitle%3DTherapeutic%2520Peptides%253A%2520Historical%2520Perspectives%252C%2520Current%2520Development%2520Trends%252C%2520and%2520Future%2520Directions%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2018%26volume%3D26%26spage%3D2700%26epage%3D2707%26doi%3D10.1016%2Fj.bmc.2017.06.052" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hruby, V. J.</span></span> <span> </span><span class="NLM_article-title">Peptide Science: Exploring the Use of Chemical Principles and Interdisciplinary Collaboration for Understanding Life Processes</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">4215</span>– <span class="NLM_lpage">4231</span>, <span class="refDoi"> DOI: 10.1021/jm0303103</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0303103" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BD3sXmvFegur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2003&pages=4215-4231&author=V.+J.+Hruby&title=Peptide+Science%3A+Exploring+the+Use+of+Chemical+Principles+and+Interdisciplinary+Collaboration+for+Understanding+Life+Processes&doi=10.1021%2Fjm0303103"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Peptide Science: Exploring the Use of Chemical Principles and Interdisciplinary Collaboration for Understanding Life Processes</span></div><div class="casAuthors">Hruby, Victor J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">4215-4231</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  We applied concepts of conformational constraint to all of the peptide hormone and neurotransmitter ligands we were investigating, in addn. to our work on oxytocin and vasopressin design.  A major focus of our work to study was peptide conformation and design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoQnFyHQ6L68rVg90H21EOLACvtfcHk0lhJwdcLemNkDw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXmvFegur0%253D&md5=18c8eff8f94b7fbe6b6b29abe57d18f0</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Fjm0303103&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0303103%26sid%3Dliteratum%253Aachs%26aulast%3DHruby%26aufirst%3DV.%2BJ.%26atitle%3DPeptide%2520Science%253A%2520Exploring%2520the%2520Use%2520of%2520Chemical%2520Principles%2520and%2520Interdisciplinary%2520Collaboration%2520for%2520Understanding%2520Life%2520Processes%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2003%26volume%3D46%26spage%3D4215%26epage%3D4231%26doi%3D10.1021%2Fjm0303103" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hruby, V. J.</span></span> <span> </span><span class="NLM_article-title">Designing Peptide Receptor Agonists and Antagonists</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">847</span>– <span class="NLM_lpage">858</span>, <span class="refDoi"> DOI: 10.1038/nrd939</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1038%2Fnrd939" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=12415245" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BD38XotleqtLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2002&pages=847-858&author=V.+J.+Hruby&title=Designing+Peptide+Receptor+Agonists+and+Antagonists&doi=10.1038%2Fnrd939"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Designing peptide receptor agonists and antagonists</span></div><div class="casAuthors">Hruby, Victor J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">847-858</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The most ubiquitous mode for controlling and modulating cellular function, intercellular communication, immune response and information-transduction pathways is through peptide-protein non-covalent interactions.  Hormones, neurotransmitters, antigens, cytokines and growth factors represent key classes of such peptide ligands.  These ligands might either be processed fragments of larger precursor proteins or surface segments of larger proteins.  Although there are numerous exceptions, such as insulin, oxytocin and calcitonin, most ligands are not used directly as drugs, and often the most useful ligands for therapy would be analogs that act as antagonists of the native ligands.  A search for systematic structure-based or ligand-based approaches to designing such ligands has been an important concern.  Today, a robust strategy has been developed for the design of peptides as drugs, drug candidates and biol. tools.  This strategy includes structural, conformational, dynamic and topog. considerations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6bGXGVM4TrrVg90H21EOLACvtfcHk0ljYpNdyaUPKcg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XotleqtLg%253D&md5=a21f2c38a42e51059b4060c3f7c8966b</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1038%2Fnrd939&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd939%26sid%3Dliteratum%253Aachs%26aulast%3DHruby%26aufirst%3DV.%2BJ.%26atitle%3DDesigning%2520Peptide%2520Receptor%2520Agonists%2520and%2520Antagonists%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2002%26volume%3D1%26spage%3D847%26epage%3D858%26doi%3D10.1038%2Fnrd939" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kessler, H.</span></span> <span> </span><span class="NLM_article-title">Conformation and Biological Activity of Cyclic Peptides</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">1982</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">512</span>– <span class="NLM_lpage">523</span>, <span class="refDoi"> DOI: 10.1002/anie.198205121</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1002%2Fanie.198205121" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=1982&pages=512-523&author=H.+Kessler&title=Conformation+and+Biological+Activity+of+Cyclic+Peptides&doi=10.1002%2Fanie.198205121"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1002%2Fanie.198205121&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.198205121%26sid%3Dliteratum%253Aachs%26aulast%3DKessler%26aufirst%3DH.%26atitle%3DConformation%2520and%2520Biological%2520Activity%2520of%2520Cyclic%2520Peptides%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D1982%26volume%3D21%26spage%3D512%26epage%3D523%26doi%3D10.1002%2Fanie.198205121" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Purkayastha, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, T. J.</span></span> <span> </span><span class="NLM_article-title">Stabilization of Proteins by Covalent Cyclization</span>. <i>Biotechnol. Bioprocess Eng.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">702</span>– <span class="NLM_lpage">712</span>, <span class="refDoi"> DOI: 10.1007/s12257-019-0363-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1007%2Fs12257-019-0363-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitV2ht7vJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2019&pages=702-712&author=A.+Purkayasthaauthor=T.+J.+Kang&title=Stabilization+of+Proteins+by+Covalent+Cyclization&doi=10.1007%2Fs12257-019-0363-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Stabilization of Proteins by Covalent Cyclization</span></div><div class="casAuthors">Purkayastha, Anwesha; Kang, Taek Jin</div><div class="citationInfo"><span class="NLM_cas:title">Biotechnology and Bioprocess Engineering</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">702-712</span>CODEN:
                <span class="NLM_cas:coden">BBEIAU</span>;
        ISSN:<span class="NLM_cas:issn">1226-8372</span>.
    
            (<span class="NLM_cas:orgname">Korean Society for Biotechnology and Bioengineering</span>)
        </div><div class="casAbstract">Stabilizing proteins against the thermal stress or proteolytic attacks is an important goal in many protein engineering studies.  As the simplest approach to the rational engineering of proteins, cyclizing proteins covalently has attracted a great deal of attention.  The idea of stabilizing the folded state of a protein by connecting its loose ends came from the polymer theory and was evidenced by many super-stable cyclic peptides/proteins present in nature.  Lab. methodologies utilizing various tools such as inteins, transpeptidases, transglutaminases and split cellular anchoring proteins have been developed, engineered, and successfully adopted to protein cyclization reactions.  Depending on the method used, N- and C-termini could be joined together to yield backbone cyclized proteins, or side chains located near the ends might be crosslinked to yield side chain cyclized ones.  As each of the methods has its own pros and cons in its reaction scheme, the outcomes such as an increase in melting temp. of a given protein are different when different methods are applied.  In this review, we highlight the stabilizing effects exerted by protein cyclization focusing on not a specific cyclizing method but product proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNzPdVh_t6b7Vg90H21EOLACvtfcHk0ljYpNdyaUPKcg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitV2ht7vJ&md5=0847e36cc193c1cbcfb83815a8045fb9</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1007%2Fs12257-019-0363-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs12257-019-0363-4%26sid%3Dliteratum%253Aachs%26aulast%3DPurkayastha%26aufirst%3DA.%26aulast%3DKang%26aufirst%3DT.%2BJ.%26atitle%3DStabilization%2520of%2520Proteins%2520by%2520Covalent%2520Cyclization%26jtitle%3DBiotechnol.%2520Bioprocess%2520Eng.%26date%3D2019%26volume%3D24%26spage%3D702%26epage%3D712%26doi%3D10.1007%2Fs12257-019-0363-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moellering, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilinski, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grossmann, T. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeh, J. T.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verdine, G. L.</span></span> <span> </span><span class="NLM_article-title">Towards Understanding Cell Penetration by Stapled Peptides</span>. <i>Med. Chem. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">111</span>– <span class="NLM_lpage">119</span>, <span class="refDoi"> DOI: 10.1039/C4MD00131A</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1039%2FC4MD00131A" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=111-119&author=Q.+Chuauthor=R.+E.+Moelleringauthor=G.+J.+Hilinskiauthor=Y.-W.+Kimauthor=T.+N.+Grossmannauthor=J.+T.-H.+Yehauthor=G.+L.+Verdine&title=Towards+Understanding+Cell+Penetration+by+Stapled+Peptides&doi=10.1039%2FC4MD00131A"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1039%2FC4MD00131A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC4MD00131A%26sid%3Dliteratum%253Aachs%26aulast%3DChu%26aufirst%3DQ.%26aulast%3DMoellering%26aufirst%3DR.%2BE.%26aulast%3DHilinski%26aufirst%3DG.%2BJ.%26aulast%3DKim%26aufirst%3DY.-W.%26aulast%3DGrossmann%26aufirst%3DT.%2BN.%26aulast%3DYeh%26aufirst%3DJ.%2BT.-H.%26aulast%3DVerdine%26aufirst%3DG.%2BL.%26atitle%3DTowards%2520Understanding%2520Cell%2520Penetration%2520by%2520Stapled%2520Peptides%26jtitle%3DMed.%2520Chem.%2520Commun.%26date%3D2015%26volume%3D6%26spage%3D111%26epage%3D119%26doi%3D10.1039%2FC4MD00131A" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dougherty, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sahni, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pei, D.</span></span> <span> </span><span class="NLM_article-title">Understanding Cell Penetration of Cyclic Peptides</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>119</i></span>,  <span class="NLM_fpage">10241</span>– <span class="NLM_lpage">10287</span>, <span class="refDoi"> DOI: 10.1021/acs.chemrev.9b00008</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.9b00008" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC1MXptlCksLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=2019&pages=10241-10287&author=P.+G.+Doughertyauthor=A.+Sahniauthor=D.+Pei&title=Understanding+Cell+Penetration+of+Cyclic+Peptides&doi=10.1021%2Facs.chemrev.9b00008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Understanding Cell Penetration of Cyclic Peptides</span></div><div class="casAuthors">Dougherty, Patrick G.; Sahni, Ashweta; Pei, Dehua</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">119</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">10241-10287</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Approx. 75% of all disease-relevant human proteins, including those involved in intracellular protein-protein interactions (PPIs), are undruggable with the current drug modalities (i.e., small mols. and biologics).  Macrocyclic peptides provide a potential soln. to these undruggable targets because their larger sizes (relative to conventional small mols.) endow them the capability of binding to flat PPI interfaces with antibody-like affinity and specificity.  Powerful combinatorial library technologies have been developed to routinely identify cyclic peptides as potent, specific inhibitors against proteins including PPI targets.  However, with the exception of a very small set of sequences, the vast majority of cyclic peptides are impermeable to the cell membrane, preventing their application against intracellular targets.  This Review examines common structural features that render most cyclic peptides membrane impermeable, as well as the unique features that allow the minority of sequences to enter the cell interior by passive diffusion, endocytosis/endosomal escape, or other mechanisms.  We also present the current state of knowledge about the mol. mechanisms of cell penetration, the various strategies for designing cell-permeable, biol. active cyclic peptides against intracellular targets, and the assay methods available to quantify their cell-permeability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp0NDA9U7-1YrVg90H21EOLACvtfcHk0ljYpNdyaUPKcg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXptlCksLs%253D&md5=65f68a9b69f3f7f33641b6a91835b1dd</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.9b00008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.9b00008%26sid%3Dliteratum%253Aachs%26aulast%3DDougherty%26aufirst%3DP.%2BG.%26aulast%3DSahni%26aufirst%3DA.%26aulast%3DPei%26aufirst%3DD.%26atitle%3DUnderstanding%2520Cell%2520Penetration%2520of%2520Cyclic%2520Peptides%26jtitle%3DChem.%2520Rev.%26date%3D2019%26volume%3D119%26spage%3D10241%26epage%3D10287%26doi%3D10.1021%2Facs.chemrev.9b00008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Drucker, D. J.</span></span> <span> </span><span class="NLM_article-title">Advances in Oral Peptide Therapeutics</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">277</span>– <span class="NLM_lpage">289</span>, <span class="refDoi"> DOI: 10.1038/s41573-019-0053-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1038%2Fs41573-019-0053-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=31848464" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2019&pages=277-289&author=D.+J.+Drucker&title=Advances+in+Oral+Peptide+Therapeutics&doi=10.1038%2Fs41573-019-0053-0"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1038%2Fs41573-019-0053-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41573-019-0053-0%26sid%3Dliteratum%253Aachs%26aulast%3DDrucker%26aufirst%3DD.%2BJ.%26atitle%3DAdvances%2520in%2520Oral%2520Peptide%2520Therapeutics%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2019%26volume%3D19%26spage%3D277%26epage%3D289%26doi%3D10.1038%2Fs41573-019-0053-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jing, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, K.</span></span> <span> </span><span class="NLM_article-title">A Gold Mine for Drug Discovery: Strategies to Develop Cyclic Peptides into Therapies</span>. <i>Med. Res. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">753</span>– <span class="NLM_lpage">810</span>, <span class="refDoi"> DOI: 10.1002/med.21639</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1002%2Fmed.21639" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=31599007" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2019&pages=753-810&author=X.+Jingauthor=K.+Jin&title=A+Gold+Mine+for+Drug+Discovery%3A+Strategies+to+Develop+Cyclic+Peptides+into+Therapies&doi=10.1002%2Fmed.21639"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1002%2Fmed.21639&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmed.21639%26sid%3Dliteratum%253Aachs%26aulast%3DJing%26aufirst%3DX.%26aulast%3DJin%26aufirst%3DK.%26atitle%3DA%2520Gold%2520Mine%2520for%2520Drug%2520Discovery%253A%2520Strategies%2520to%2520Develop%2520Cyclic%2520Peptides%2520into%2520Therapies%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2019%26volume%3D40%26spage%3D753%26epage%3D810%26doi%3D10.1002%2Fmed.21639" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nielsen, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shepherd, N. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucke, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stoermer, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fairlie, D. P.</span></span> <span> </span><span class="NLM_article-title">Orally Absorbed Cyclic Peptides</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>117</i></span>,  <span class="NLM_fpage">8094</span>– <span class="NLM_lpage">8128</span>, <span class="refDoi"> DOI: 10.1021/acs.chemrev.6b00838</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.6b00838" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC2sXot12lurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=117&publication_year=2017&pages=8094-8128&author=D.+S.+Nielsenauthor=N.+E.+Shepherdauthor=W.+Xuauthor=A.+J.+Luckeauthor=M.+J.+Stoermerauthor=D.+P.+Fairlie&title=Orally+Absorbed+Cyclic+Peptides&doi=10.1021%2Facs.chemrev.6b00838"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Orally Absorbed Cyclic Peptides</span></div><div class="casAuthors">Nielsen, Daniel S.; Shepherd, Nicholas E.; Xu, Weijun; Lucke, Andrew J.; Stoermer, Martin J.; Fairlie, David P.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">117</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">8094-8128</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Peptides and proteins are not orally bioavailable in mammals, although a few peptides are intestinally absorbed in small amts.  Polypeptides are generally too large and polar to passively diffuse through lipid membranes, while most known active transport mechanisms facilitate cell uptake of only very small peptides.  Systematic evaluations of peptides with mol. wts. above 500 Da are needed to identify parameters that influence oral bioavailability.  Here we describe 125 cyclic peptides contg. four to thirty-seven amino acids that are orally absorbed by mammals.  Cyclization minimizes degrdn. in the gut, blood and tissues by removing cleavable N- and C-termini and by shielding components from metabolic enzymes.  Cyclization also folds peptides into bioactive conformations that det. exposure of polar atoms to solvation by water and lipids, and therefore should influence oral bioavailability.  Key chem. properties thought to influence oral absorption and bioavailability are analyzed, including mol. wt., octanol-water partitioning, hydrogen bond donors/acceptors, rotatable bonds and polar surface area.  The cyclic peptides violated to different degrees all of the limits traditionally considered to be important for oral bioavailability of drug-like small mols., although fewer hydrogen bond donors and reduced flexibility generally favored oral absorption.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprHf1YVkf7ErVg90H21EOLACvtfcHk0ljCrRTXQrqYIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXot12lurY%253D&md5=38af1945471e5ddffd3ae630ebbe129b</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.6b00838&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.6b00838%26sid%3Dliteratum%253Aachs%26aulast%3DNielsen%26aufirst%3DD.%2BS.%26aulast%3DShepherd%26aufirst%3DN.%2BE.%26aulast%3DXu%26aufirst%3DW.%26aulast%3DLucke%26aufirst%3DA.%2BJ.%26aulast%3DStoermer%26aufirst%3DM.%2BJ.%26aulast%3DFairlie%26aufirst%3DD.%2BP.%26atitle%3DOrally%2520Absorbed%2520Cyclic%2520Peptides%26jtitle%3DChem.%2520Rev.%26date%3D2017%26volume%3D117%26spage%3D8094%26epage%3D8128%26doi%3D10.1021%2Facs.chemrev.6b00838" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nielsen, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohman, R.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoang, H. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucke, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fairlie, D. P.</span></span> <span> </span><span class="NLM_article-title">Flexibility versus Rigidity for Orally Bioavailable Cyclic Hexapeptides</span>. <i>ChemBioChem</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">2289</span>– <span class="NLM_lpage">2293</span>, <span class="refDoi"> DOI: 10.1002/cbic.201500441</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1002%2Fcbic.201500441" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=26336864" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1Wjt73I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2015&pages=2289-2293&author=D.+S.+Nielsenauthor=R.-J.+Lohmanauthor=H.+N.+Hoangauthor=T.+A.+Hillauthor=A.+Jonesauthor=A.+J.+Luckeauthor=D.+P.+Fairlie&title=Flexibility+versus+Rigidity+for+Orally+Bioavailable+Cyclic+Hexapeptides&doi=10.1002%2Fcbic.201500441"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Flexibility versus Rigidity for Orally Bioavailable Cyclic Hexapeptides</span></div><div class="casAuthors">Nielsen, Daniel S.; Lohman, Rink-Jan; Hoang, Huy N.; Hill, Timothy A.; Jones, Alun; Lucke, Andrew J.; Fairlie, David P.</div><div class="citationInfo"><span class="NLM_cas:title">ChemBioChem</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">2289-2293</span>CODEN:
                <span class="NLM_cas:coden">CBCHFX</span>;
        ISSN:<span class="NLM_cas:issn">1439-4227</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Cyclic peptides and macrocycles have the potential to be membrane permeable and orally bioavailable, despite often not complying with the "rule of five" used in medicinal chem. to guide the discovery of oral drugs.  Here we compare solvent-dependent three-dimensional structures of three cyclic hexapeptides contg. D-amino acids, prolines, and intramol. hydrogen bonds.  Conformational rigidity rather than flexibility resulted in higher membrane permeability, metabolic stability and oral bioavailability, consistent with less polar surface exposure to solvent and a reduced entropy penalty for transition between polar and nonpolar environments.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpArFfvUbUBA7Vg90H21EOLACvtfcHk0ljCrRTXQrqYIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1Wjt73I&md5=b0cf19a1ee875648d518dcc0befd9c28</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1002%2Fcbic.201500441&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbic.201500441%26sid%3Dliteratum%253Aachs%26aulast%3DNielsen%26aufirst%3DD.%2BS.%26aulast%3DLohman%26aufirst%3DR.-J.%26aulast%3DHoang%26aufirst%3DH.%2BN.%26aulast%3DHill%26aufirst%3DT.%2BA.%26aulast%3DJones%26aufirst%3DA.%26aulast%3DLucke%26aufirst%3DA.%2BJ.%26aulast%3DFairlie%26aufirst%3DD.%2BP.%26atitle%3DFlexibility%2520versus%2520Rigidity%2520for%2520Orally%2520Bioavailable%2520Cyclic%2520Hexapeptides%26jtitle%3DChemBioChem%26date%3D2015%26volume%3D16%26spage%3D2289%26epage%3D2293%26doi%3D10.1002%2Fcbic.201500441" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Räder, A. F. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinmüller, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reichart, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schumacher-Klinger, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merzbach, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilon, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffman, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kessler, H.</span></span> <span> </span><span class="NLM_article-title">Orally Active Peptides: Is There a Magic Bullet?</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">14414</span>– <span class="NLM_lpage">14438</span>, <span class="refDoi"> DOI: 10.1002/anie.201807298</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1002%2Fanie.201807298" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A280%3ADC%252BB3c3hslegsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2018&pages=14414-14438&author=A.+F.+B.+R%C3%A4derauthor=M.+Weinm%C3%BCllerauthor=F.+Reichartauthor=A.+Schumacher-Klingerauthor=S.+Merzbachauthor=C.+Gilonauthor=A.+Hoffmanauthor=H.+Kessler&title=Orally+Active+Peptides%3A+Is+There+a+Magic+Bullet%3F&doi=10.1002%2Fanie.201807298"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Orally Active Peptides: Is There a Magic Bullet?</span></div><div class="casAuthors">Rader Andreas F B; Weinmuller Michael; Reichart Florian; Kessler Horst; Schumacher-Klinger Adi; Merzbach Shira; Gilon Chaim; Hoffman Amnon</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie (International ed. in English)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">44</span>),
    <span class="NLM_cas:pages">14414-14438</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">For decades, the development of peptides as potential drugs was aimed solely at peptides with the highest affinity, receptor selectivity, or stability against enzymatic degradation.  However, optimization of their oral availability is highly desirable to establish orally active peptides as potential drug candidates for everyday use.  A twofold optimization process is necessary to produce orally active peptides: 1) optimization of the affinity and selectivity and 2) optimization of the oral availability.  These two steps must be performed sequentially for the rational design of orally active peptides.  Nevertheless, additional knowledge is required to understand which structural changes increase oral availability, followed by incorporation of these elements into a peptide without changing its other biological properties.  Considerable efforts have been made to understand the influence of these modifications on oral availability.  One approach is to improve the oral availability of a peptide that has been previously optimized for biological activity, as described in (1) above.  The second approach is to first identify an intestinally permeable, metabolically stable peptide scaffold and then introduce the functional groups necessary for the desired biological function.  Previous approaches to achieving peptide oral availability have been claimed to have general applicability but, thus far, most of these solutions have not been successful in other cases.  This Review discusses diverse chemical modifications, model peptides optimized for bioavailability, and orally active peptides to summarize the state of the research on the oral activity of peptides.  We explain why no simple and straightforward strategy (i.e. a "magic bullet") exists for the design of an orally active peptide with a druglike biological function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSwS0O4JvOgT--MdeMhXAHkfW6udTcc2ebEJzh42w0s3bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3c3hslegsg%253D%253D&md5=d5d266bb2a6a26ec27d413ab3afdc311</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1002%2Fanie.201807298&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201807298%26sid%3Dliteratum%253Aachs%26aulast%3DR%25C3%25A4der%26aufirst%3DA.%2BF.%2BB.%26aulast%3DWeinm%25C3%25BCller%26aufirst%3DM.%26aulast%3DReichart%26aufirst%3DF.%26aulast%3DSchumacher-Klinger%26aufirst%3DA.%26aulast%3DMerzbach%26aufirst%3DS.%26aulast%3DGilon%26aufirst%3DC.%26aulast%3DHoffman%26aufirst%3DA.%26aulast%3DKessler%26aufirst%3DH.%26atitle%3DOrally%2520Active%2520Peptides%253A%2520Is%2520There%2520a%2520Magic%2520Bullet%253F%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2018%26volume%3D57%26spage%3D14414%26epage%3D14438%26doi%3D10.1002%2Fanie.201807298" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Villar, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beglov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chennamadhavuni, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porco, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozakov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vajda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitty, A.</span></span> <span> </span><span class="NLM_article-title">How Proteins Bind Macrocycles</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">723</span>– <span class="NLM_lpage">731</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1584</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1038%2Fnchembio.1584" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=25038790" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFygs7jF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=723-731&author=E.+A.+Villarauthor=D.+Beglovauthor=S.+Chennamadhavuniauthor=J.+A.+Porcoauthor=D.+Kozakovauthor=S.+Vajdaauthor=A.+Whitty&title=How+Proteins+Bind+Macrocycles&doi=10.1038%2Fnchembio.1584"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">How proteins bind macrocycles</span></div><div class="casAuthors">Villar, Elizabeth A.; Beglov, Dmitri; Chennamadhavuni, Spandan; Porco, John A. Jr; Kozakov, Dima; Vajda, Sandor; Whitty, Adrian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">723-731</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The potential utility of synthetic macrocycles (MCs) as drugs, particularly against low-druggability targets such as protein-protein interactions, has been widely discussed.  There is little information, however, to guide the design of MCs for good target protein-binding activity or bioavailability.  To address this knowledge gap, we analyze the binding modes of a representative set of MC-protein complexes.  The results, combined with consideration of the physicochem. properties of approved macrocyclic drugs, allow us to propose specific guidelines for the design of synthetic MC libraries with structural and physicochem. features likely to favor strong binding to protein targets as well as good bioavailability.  We addnl. provide evidence that large, natural product-derived MCs can bind targets that are not druggable by conventional, drug-like compds., supporting the notion that natural product-inspired synthetic MCs can expand the no. of proteins that are druggable by synthetic small mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqa3yuNCHU5d7Vg90H21EOLACvtfcHk0ljD0exySS1a6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFygs7jF&md5=0d41fba9267abb3a3d0065487937075a</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1584&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1584%26sid%3Dliteratum%253Aachs%26aulast%3DVillar%26aufirst%3DE.%2BA.%26aulast%3DBeglov%26aufirst%3DD.%26aulast%3DChennamadhavuni%26aufirst%3DS.%26aulast%3DPorco%26aufirst%3DJ.%2BA.%26aulast%3DKozakov%26aufirst%3DD.%26aulast%3DVajda%26aufirst%3DS.%26aulast%3DWhitty%26aufirst%3DA.%26atitle%3DHow%2520Proteins%2520Bind%2520Macrocycles%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2014%26volume%3D10%26spage%3D723%26epage%3D731%26doi%3D10.1038%2Fnchembio.1584" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vinogradov, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suga, H.</span></span> <span> </span><span class="NLM_article-title">Macrocyclic Peptides as Drug Candidates: Recent Progress and Remaining Challenges</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>141</i></span>,  <span class="NLM_fpage">4167</span>– <span class="NLM_lpage">4181</span>, <span class="refDoi"> DOI: 10.1021/jacs.8b13178</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.8b13178" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC1MXjtFCqtb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=141&publication_year=2019&pages=4167-4181&author=A.+A.+Vinogradovauthor=Y.+Yinauthor=H.+Suga&title=Macrocyclic+Peptides+as+Drug+Candidates%3A+Recent+Progress+and+Remaining+Challenges&doi=10.1021%2Fjacs.8b13178"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Macrocyclic Peptides as Drug Candidates: Recent Progress and Remaining Challenges</span></div><div class="casAuthors">Vinogradov, Alexander A.; Yin, Yizhen; Suga, Hiroaki</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">141</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4167-4181</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Peptides as a therapeutic modality attract much attention due to their synthetic accessibility, high degree of specific binding, and the ability to target protein surfaces traditionally considered "undruggable".  Unfortunately, at the same time, other pharmacol. properties of a generic peptide, such as metabolic stability and cell permeability, are quite poor, which limits the success of de novo discovered biol. active peptides as drug candidates.  Here, we review how macrocyclization as well as the incorporation of nonproteogenic amino acids and various conjugation strategies may be utilized to improve on these characteristics to create better drug candidates.  We analyze recent progress and remaining challenges in improving individual pharmacol. properties of bioactive peptides, and offer our opinion on interfacing these, often conflicting, considerations, to create balanced drug candidates as a potential way to make further progress in this area.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXRPkCAGk8-rVg90H21EOLACvtfcHk0ljD0exySS1a6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXjtFCqtb4%253D&md5=59b6197874d4709a74089367a09119cc</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Fjacs.8b13178&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.8b13178%26sid%3Dliteratum%253Aachs%26aulast%3DVinogradov%26aufirst%3DA.%2BA.%26aulast%3DYin%26aufirst%3DY.%26aulast%3DSuga%26aufirst%3DH.%26atitle%3DMacrocyclic%2520Peptides%2520as%2520Drug%2520Candidates%253A%2520Recent%2520Progress%2520and%2520Remaining%2520Challenges%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2019%26volume%3D141%26spage%3D4167%26epage%3D4181%26doi%3D10.1021%2Fjacs.8b13178" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Driggers, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hale, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terrett, N. K.</span></span> <span> </span><span class="NLM_article-title">The Exploration of Macrocycles for Drug Discovery--an Underexploited Structural Class</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">608</span>– <span class="NLM_lpage">624</span>, <span class="refDoi"> DOI: 10.1038/nrd2590</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1038%2Fnrd2590" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=18591981" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BD1cXnvFShsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2008&pages=608-624&author=E.+M.+Driggersauthor=S.+P.+Haleauthor=J.+Leeauthor=N.+K.+Terrett&title=The+Exploration+of+Macrocycles+for+Drug+Discovery%2D%2Dan+Underexploited+Structural+Class&doi=10.1038%2Fnrd2590"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">The exploration of macrocycles for drug discovery - an underexploited structural class</span></div><div class="casAuthors">Driggers, Edward M.; Hale, Stephen P.; Lee, Jinbo; Terrett, Nicholas K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">608-624</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Natural products comprised of a macrocycle ring structure have proven their therapeutic applications as antibiotics, immunosuppressants as well as anticancer agents.  Despite this, macrocyclic compds. remain under-explored.  Terrett and colleagues review the properties and features of current macrocycle drugs, emphasizing the vast potential of synthetic macrocycles in drug discovery.  Macrocyclic natural products have evolved to fulfil numerous biochem. functions, and their profound pharmacol. properties have led to their development as drugs.  A macrocycle provides diverse functionality and stereochem. complexity in a conformationally pre-organized ring structure.  This can result in high affinity and selectivity for protein targets, while preserving sufficient bioavailability to reach intracellular locations.  Despite these valuable characteristics, and the proven success of more than 100 marketed macrocycle drugs derived from natural products, this structural class has been poorly explored within drug discovery.  This is in part due to concerns about synthetic intractability and non-drug-like properties.  This Review describes the growing body of data in favor of macrocyclic therapeutics, and demonstrates that this class of compds. can be both fully drug-like in its properties and readily prepd. owing to recent advances in synthetic medicinal chem.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDtW8LpE4ru7Vg90H21EOLACvtfcHk0lhGhRM7FBb58A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXnvFShsbc%253D&md5=16e091818053e5b163b81078a8d03094</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1038%2Fnrd2590&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2590%26sid%3Dliteratum%253Aachs%26aulast%3DDriggers%26aufirst%3DE.%2BM.%26aulast%3DHale%26aufirst%3DS.%2BP.%26aulast%3DLee%26aufirst%3DJ.%26aulast%3DTerrett%26aufirst%3DN.%2BK.%26atitle%3DThe%2520Exploration%2520of%2520Macrocycles%2520for%2520Drug%2520Discovery--an%2520Underexploited%2520Structural%2520Class%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2008%26volume%3D7%26spage%3D608%26epage%3D624%26doi%3D10.1038%2Fnrd2590" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mallinson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collins, I.</span></span> <span> </span><span class="NLM_article-title">Macrocycles in New Drug Discovery</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">1409</span>– <span class="NLM_lpage">1438</span>, <span class="refDoi"> DOI: 10.4155/fmc.12.93</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.4155%2Ffmc.12.93" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=22857532" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtFCht7rO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2012&pages=1409-1438&author=J.+Mallinsonauthor=I.+Collins&title=Macrocycles+in+New+Drug+Discovery&doi=10.4155%2Ffmc.12.93"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Macrocycles in new drug discovery</span></div><div class="casAuthors">Mallinson, Jamie; Collins, Ian</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1409-1438</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">A review.  The use of drug-like macrocycles is emerging as an exciting area of medicinal chem., with several recent examples highlighting the favorable changes in biol. and physicochem. properties that macrocyclization can afford.  Natural product macrocycles and their synthetic derivs. have long been clin. useful and attention is now being focused on the wider use of macrocyclic scaffolds in medicinal chem. in the search for new drugs for increasingly challenging targets.  With the increasing awareness of concepts of drug-likeness and the dangers of mol. obesity', functionalized macrocyclic scaffolds could provide a way to generate ligand-efficient mols. with enhanced properties.  In this review we will sep. discuss the effects of macrocyclization upon potency, selectivity and physicochem. properties, concg. on recent case histories in oncol. drug discovery.  Addnl., we will highlight selected advances in the synthesis of macrocycles and provide an outlook on the future use of macrocyclic scaffolds in medicinal chem.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqSrzcbKW7Tc7Vg90H21EOLACvtfcHk0lhGhRM7FBb58A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtFCht7rO&md5=63ad275af92ac89ae6b36c5071b578f8</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.4155%2Ffmc.12.93&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.12.93%26sid%3Dliteratum%253Aachs%26aulast%3DMallinson%26aufirst%3DJ.%26aulast%3DCollins%26aufirst%3DI.%26atitle%3DMacrocycles%2520in%2520New%2520Drug%2520Discovery%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2012%26volume%3D4%26spage%3D1409%26epage%3D1438%26doi%3D10.4155%2Ffmc.12.93" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Whitty, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Viarengo, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, M.</span></span> <span> </span><span class="NLM_article-title">Progress towards the Broad Use of Non-Peptide Synthetic Macrocycles in Drug Discovery</span>. <i>Org. Biomol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">7729</span>– <span class="NLM_lpage">7735</span>, <span class="refDoi"> DOI: 10.1039/C7OB00056A</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1039%2FC7OB00056A" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=28876025" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsVKgsLrO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2017&pages=7729-7735&author=A.+Whittyauthor=L.+A.+Viarengoauthor=M.+Zhong&title=Progress+towards+the+Broad+Use+of+Non-Peptide+Synthetic+Macrocycles+in+Drug+Discovery&doi=10.1039%2FC7OB00056A"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Progress towards the broad use of non-peptide synthetic macrocycles in drug discovery</span></div><div class="casAuthors">Whitty, Adrian; Viarengo, Lauren A.; Zhong, Mengqi</div><div class="citationInfo"><span class="NLM_cas:title">Organic & Biomolecular Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">37</span>),
    <span class="NLM_cas:pages">7729-7735</span>CODEN:
                <span class="NLM_cas:coden">OBCRAK</span>;
        ISSN:<span class="NLM_cas:issn">1477-0520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  The authors discuss progress towards addressing three key questions pertaining to the design of screening libraries of synthetic nonpeptidic macrocycles (MCs) for drug discovery: What structural and physicochem. properties of MCs maximize the likelihood of achieving strong and specific binding to protein targets.  What features render a protein target suitable for binding MCs, and can this information be used to identify suitable targets for inhibition by MCs.  What properties of synthetic MCs confer good pharmaceutical properties, and particularly good aq. soly. coupled with passive membrane permeability.  The authors addnl. discuss how the criteria that define a meaningful MC screening hit are linked to the size of the screening library and the synthetic methodol. employed in its prepn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpeNdUQVdF6FrVg90H21EOLACvtfcHk0lhGhRM7FBb58A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsVKgsLrO&md5=451ddddc4457d5a2ee84b384aa15508e</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1039%2FC7OB00056A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC7OB00056A%26sid%3Dliteratum%253Aachs%26aulast%3DWhitty%26aufirst%3DA.%26aulast%3DViarengo%26aufirst%3DL.%2BA.%26aulast%3DZhong%26aufirst%3DM.%26atitle%3DProgress%2520towards%2520the%2520Broad%2520Use%2520of%2520Non-Peptide%2520Synthetic%2520Macrocycles%2520in%2520Drug%2520Discovery%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2017%26volume%3D15%26spage%3D7729%26epage%3D7735%26doi%3D10.1039%2FC7OB00056A" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dharanipragada, R.</span></span> <span> </span><span class="NLM_article-title">New Modalities in Conformationally Constrained Peptides for Potency, Selectivity and Cell Permeation</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">831</span>– <span class="NLM_lpage">849</span>, <span class="refDoi"> DOI: 10.4155/fmc.13.25</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.4155%2Ffmc.13.25" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=23651095" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC3sXntFyrsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2013&pages=831-849&author=R.+Dharanipragada&title=New+Modalities+in+Conformationally+Constrained+Peptides+for+Potency%2C+Selectivity+and+Cell+Permeation&doi=10.4155%2Ffmc.13.25"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">New modalities in conformationally constrained peptides for potency, selectivity and cell permeation</span></div><div class="casAuthors">Dharanipragada, Ram</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">831-849</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">A review.  There has been a resurgence of interest in peptide pharmaceuticals as they have an advantage of potency, selectivity and less toxicity compared with small-mol. therapeutics.  The main draw back of peptides is lack of stability to biol. media.  Constraining a peptide has been one of the approaches to improving in vivo stability of the peptides.  Several new modalities in constraining peptides have been developed over recent years and this review highlights some of the new developments.  The newer cyclization strategies have rendered, in some cases, oral activity, cell permeability, improved potency at the target receptor, selectivity against receptor subtypes and improved stability to enzymes.  As chemists further understand the rules governing cell permeability, oral absorption and enhancing stability of peptides, we can expect to see more peptides entering clinic for many unmet medical needs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoULvQN8YpaBbVg90H21EOLACvtfcHk0lhGhRM7FBb58A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXntFyrsrk%253D&md5=829f070989c1e8e9e729595ab92d30b5</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.4155%2Ffmc.13.25&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.13.25%26sid%3Dliteratum%253Aachs%26aulast%3DDharanipragada%26aufirst%3DR.%26atitle%3DNew%2520Modalities%2520in%2520Conformationally%2520Constrained%2520Peptides%2520for%2520Potency%252C%2520Selectivity%2520and%2520Cell%2520Permeation%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2013%26volume%3D5%26spage%3D831%26epage%3D849%26doi%3D10.4155%2Ffmc.13.25" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lau, Y. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Andrade, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spring, D. R.</span></span> <span> </span><span class="NLM_article-title">Peptide Stapling Techniques Based on Different Macrocyclisation Chemistries</span>. <i>Chem. Soc. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">91</span>– <span class="NLM_lpage">102</span>, <span class="refDoi"> DOI: 10.1039/C4CS00246F</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1039%2FC4CS00246F" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=25199043" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsV2mtb%252FL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2015&pages=91-102&author=Y.+H.+Lauauthor=P.+de+Andradeauthor=Y.+Wuauthor=D.+R.+Spring&title=Peptide+Stapling+Techniques+Based+on+Different+Macrocyclisation+Chemistries&doi=10.1039%2FC4CS00246F"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Peptide stapling techniques based on different macrocyclization chemistries</span></div><div class="casAuthors">Lau, Yu Heng; de Andrade, Peterson; Wu, Yuteng; Spring, David R.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Society Reviews</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">91-102</span>CODEN:
                <span class="NLM_cas:coden">CSRVBR</span>;
        ISSN:<span class="NLM_cas:issn">0306-0012</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  Peptide stapling is a strategy for constraining short peptides typically in an alpha-helical conformation.  Stapling is carried out by covalently linking the side-chains of two amino acids, thereby forming a peptide macrocycle.  There is an expanding repertoire of stapling techniques based on different macrocyclization chemistries.  In this tutorial review, the authors categorize and analyze key examples of peptide stapling in terms of their synthesis and applicability to biol. systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkqQ6yGsikIbVg90H21EOLACvtfcHk0lgs5qfyZAG-CA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsV2mtb%252FL&md5=0efb31241f9809b61a81474131a54de3</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1039%2FC4CS00246F&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC4CS00246F%26sid%3Dliteratum%253Aachs%26aulast%3DLau%26aufirst%3DY.%2BH.%26aulast%3Dde%2BAndrade%26aufirst%3DP.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DSpring%26aufirst%3DD.%2BR.%26atitle%3DPeptide%2520Stapling%2520Techniques%2520Based%2520on%2520Different%2520Macrocyclisation%2520Chemistries%26jtitle%3DChem.%2520Soc.%2520Rev.%26date%3D2015%26volume%3D44%26spage%3D91%26epage%3D102%26doi%3D10.1039%2FC4CS00246F" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ngambenjawong, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pineda, J. M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pun, S. H.</span></span> <span> </span><span class="NLM_article-title">Engineering an Affinity-Enhanced Peptide through Optimization of Cyclization Chemistry</span>. <i>Bioconjugate Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">2854</span>– <span class="NLM_lpage">2862</span>, <span class="refDoi"> DOI: 10.1021/acs.bioconjchem.6b00502</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.bioconjchem.6b00502" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhslamt7fO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2016&pages=2854-2862&author=C.+Ngambenjawongauthor=J.+M.+B.+Pinedaauthor=S.+H.+Pun&title=Engineering+an+Affinity-Enhanced+Peptide+through+Optimization+of+Cyclization+Chemistry&doi=10.1021%2Facs.bioconjchem.6b00502"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Engineering an Affinity-Enhanced Peptide through Optimization of Cyclization Chemistry</span></div><div class="casAuthors">Ngambenjawong, Chayanon; Pineda, Julio Marco B.; Pun, Suzie H.</div><div class="citationInfo"><span class="NLM_cas:title">Bioconjugate Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2854-2862</span>CODEN:
                <span class="NLM_cas:coden">BCCHES</span>;
        ISSN:<span class="NLM_cas:issn">1043-1802</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Peptide cyclization is a strategy used to improve stability and/or activity of peptides.  The most commonly used cyclization method is disulfide bridge formation of cysteine-contg. peptides as typically found in nature.  Over the years, an increasing no. of alternative chemistries for peptide cyclization with improved efficiency, kinetics, orthogonality, and stability have been reported.  However, there has been less appreciation for the opportunity to fine-tune peptide activity via the diverse chem. entities introduced at the site of linkage by different cyclization strategies.  Here, the authors demonstrate how cyclization optimization of an M2 "anti-inflammatory" macrophage-binding peptide (M2pep) resulted in a significant increase in binding affinity of the optimized analog to M2 macrophages while maintaining binding selectivity compared to M1 "pro-inflammatory" macrophages.  In this study, the authors report synthesis and evaluation of four cyclic M2pep(RY) analogs with diverse cyclization strategies; (1) Asp-[Amide]-Lys, (2) Azido-Lys-[Triazole (Copper(I)-catalyzed alkyne-azide cycloaddn. (CuAAC))]-Propargyl-Gly, (3) Cys-[Decafluorobiphenyl (DFBP)]-Cys, and (4) Cys-[Decafluorobiphenyl sulfone (DFS)]-Cys, whereby the chem. entity/linker at the linkage site is shown in the square bracket and is between the residues involved in cyclization.  These peptides are compared to a disulfide-cyclized M2pep(RY) that the authors previously reported as a serum-stable, affinity enhanced analog to the original linear M2pep.  DFBP-cyclized M2pep(RY) exhibits the highest binding activity to M2 macrophages with apparent dissocn. const. (KD) about 2.03 μM compared to 36.3 μM for the original disulfide-cyclized M2pep(RY) and 220 μM for the original linear peptide.  DFS-cyclized M2pep(RY) also binds more strongly than the original cyclized analog whereas Amide- and Triazole-cyclized M2pep(RY) analogs bind less strongly.  The authors verified that DFBP alone has negligible binding to M2 macrophages and the incorporation of diphenylalanine to the original sequence improves binding activity at the expense of soly. and increased toxicity.  In conclusion, the authors report development of cyclic M2pep(RY) analogs with diverse cyclization strategies leading to the discovery of DFBP-cyclized M2pep(RY) with super-enhanced M2 macrophage-binding activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoY-UeaKwT_xLVg90H21EOLACvtfcHk0lgs5qfyZAG-CA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhslamt7fO&md5=f0ed679dcb06b4bc8a29fe4c35a9500a</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Facs.bioconjchem.6b00502&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.bioconjchem.6b00502%26sid%3Dliteratum%253Aachs%26aulast%3DNgambenjawong%26aufirst%3DC.%26aulast%3DPineda%26aufirst%3DJ.%2BM.%2BB.%26aulast%3DPun%26aufirst%3DS.%2BH.%26atitle%3DEngineering%2520an%2520Affinity-Enhanced%2520Peptide%2520through%2520Optimization%2520of%2520Cyclization%2520Chemistry%26jtitle%3DBioconjugate%2520Chem.%26date%3D2016%26volume%3D27%26spage%3D2854%26epage%3D2862%26doi%3D10.1021%2Facs.bioconjchem.6b00502" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Skowron, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Speltz, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moore, T. W.</span></span> <span> </span><span class="NLM_article-title">Recent Structural Advances in Constrained Helical Peptides</span>. <i>Med. Res. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">749</span>– <span class="NLM_lpage">770</span>, <span class="refDoi"> DOI: 10.1002/med.21540</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1002%2Fmed.21540" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=30307621" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A280%3ADC%252BB3cznvF2itQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2019&pages=749-770&author=K.+J.+Skowronauthor=T.+E.+Speltzauthor=T.+W.+Moore&title=Recent+Structural+Advances+in+Constrained+Helical+Peptides&doi=10.1002%2Fmed.21540"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Recent structural advances in constrained helical peptides</span></div><div class="casAuthors">Skowron Kornelia J; Speltz Thomas E; Moore Terry W; Moore Terry W</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal research reviews</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">749-770</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Given the ubiquity of the  -helix in the proteome, there has been much research in developing mimics of  -helices, and most of this study has been toward developing protein-protein interaction inhibitors.  A common strategy for mimicking  -helices has been through the use of constrained, helical peptides.  The addition of a constraint typically provides for conformational and proteolytic stability and, in some cases, cell permeability.  Some of the most well-known strategies included are lactam formation and hydrocarbon "stapling." Beyond those strategies, there have been many recent advances in developing constrained peptides.  The purpose of this review is to highlight recent advances in the development of new helix-stabilizing technologies, constraint diversification strategies, tether diversification strategies, and combination strategies that create new bicyclic helical peptides.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSdKbhDWiU7TdLIORRPd5lbfW6udTcc2eZ8zEMtYv1PF7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3cznvF2itQ%253D%253D&md5=9277ab4a2572b4d0d2b500e82c267936</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1002%2Fmed.21540&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmed.21540%26sid%3Dliteratum%253Aachs%26aulast%3DSkowron%26aufirst%3DK.%2BJ.%26aulast%3DSpeltz%26aufirst%3DT.%2BE.%26aulast%3DMoore%26aufirst%3DT.%2BW.%26atitle%3DRecent%2520Structural%2520Advances%2520in%2520Constrained%2520Helical%2520Peptides%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2019%26volume%3D39%26spage%3D749%26epage%3D770%26doi%3D10.1002%2Fmed.21540" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheng, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span> <span> </span><span class="NLM_article-title">Synthesis of Bioactive and Stabilized Cyclic Peptides by Macrocyclization Using C(Sp3)-H Activation</span>. <i>Chem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">4565</span>– <span class="NLM_lpage">4570</span>, <span class="refDoi"> DOI: 10.1039/C6SC05530C</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1039%2FC6SC05530C" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=28936334" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmt1ektbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=4565-4570&author=J.+Tangauthor=Y.+Heauthor=H.+Chenauthor=W.+Shengauthor=H.+Wang&title=Synthesis+of+Bioactive+and+Stabilized+Cyclic+Peptides+by+Macrocyclization+Using+C%28Sp3%29-H+Activation&doi=10.1039%2FC6SC05530C"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis of bioactive and stabilized cyclic peptides by macrocyclization using C(sp3)-H activation</span></div><div class="casAuthors">Tang, Jian; He, Yadong; Chen, Hongfei; Sheng, Wangjian; Wang, Huan</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Science</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">4565-4570</span>CODEN:
                <span class="NLM_cas:coden">CSHCCN</span>;
        ISSN:<span class="NLM_cas:issn">2041-6520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Cyclic peptides have attracted increasing attention in recent years due to their ability to inhibit protein-protein interactions.  Current strategies to prep. cyclic peptides often rely on functional amino acid side chains or the incorporation of unnatural amino acids, thus limiting their structural diversity.  Here, we describe the development of a highly versatile peptide macrocyclization strategy through a palladium-catalyzed C(sp3)-H activation and the synthesis of cyclic peptides featuring unique hydrocarbon linkages between the β-carbon of amino acids and the arom. side chains of Phe and Trp.  We demonstrate that such peptides exhibit improved biol. properties compared to their acyclic counterparts.  Finally, we applied this method in the synthesis of the natural product celogentin C.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqE3dkZ92oXRbVg90H21EOLACvtfcHk0lgs5qfyZAG-CA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmt1ektbk%253D&md5=47dfb53e3dff4be724b0480c248e280c</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1039%2FC6SC05530C&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC6SC05530C%26sid%3Dliteratum%253Aachs%26aulast%3DTang%26aufirst%3DJ.%26aulast%3DHe%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DSheng%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DH.%26atitle%3DSynthesis%2520of%2520Bioactive%2520and%2520Stabilized%2520Cyclic%2520Peptides%2520by%2520Macrocyclization%2520Using%2520C%2528Sp3%2529-H%2520Activation%26jtitle%3DChem.%2520Sci.%26date%3D2017%26volume%3D8%26spage%3D4565%26epage%3D4570%26doi%3D10.1039%2FC6SC05530C" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">White, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yudin, A. K.</span></span> <span> </span><span class="NLM_article-title">Contemporary Strategies for Peptide Macrocyclization</span>. <i>Nat. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">509</span>– <span class="NLM_lpage">524</span>, <span class="refDoi"> DOI: 10.1038/nchem.1062</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1038%2Fnchem.1062" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=21697871" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC3MXnvFWjtrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2011&pages=509-524&author=C.+J.+Whiteauthor=A.+K.+Yudin&title=Contemporary+Strategies+for+Peptide+Macrocyclization&doi=10.1038%2Fnchem.1062"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Contemporary strategies for peptide macrocyclization</span></div><div class="casAuthors">White, Christopher J.; Yudin, Andrei K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">509-524</span>CODEN:
                <span class="NLM_cas:coden">NCAHBB</span>;
        ISSN:<span class="NLM_cas:issn">1755-4330</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Peptide macrocycles have found applications that range from drug discovery to nanomaterials.  These ring-shaped mols. have shown remarkable capacity for functional fine-tuning.  Such capacity is enabled by the possibility of adjusting the peptide conformation using the techniques of chem. synthesis.  Cyclic peptides have been difficult, and often impossible, to prep. using traditional synthetic methods.  For macrocyclization to occur, the activated peptide must adopt an entropically disfavored pre-cyclization conformation before forming the desired product.  Here, recent solns. to some of the major challenges in this important area of contemporary synthesis were reviewed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqUTda6OkfQprVg90H21EOLACvtfcHk0lh0AE2XTB5S6w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXnvFWjtrc%253D&md5=32dfd94692ca20901dc8828bcd27ae8c</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1038%2Fnchem.1062&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchem.1062%26sid%3Dliteratum%253Aachs%26aulast%3DWhite%26aufirst%3DC.%2BJ.%26aulast%3DYudin%26aufirst%3DA.%2BK.%26atitle%3DContemporary%2520Strategies%2520for%2520Peptide%2520Macrocyclization%26jtitle%3DNat.%2520Chem.%26date%3D2011%26volume%3D3%26spage%3D509%26epage%3D524%26doi%3D10.1038%2Fnchem.1062" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Felix, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heimer, E. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lambros, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fournier, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mowles, T. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maines, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wegrzynski, B. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toome, V.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Synthesis, Biological Activity and Conformational Analysis of Cyclic GRF Analogs</span>. <i>Int. J. Pept. Protein Res.</i> <span class="NLM_year" style="font-weight: bold;">1988</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">441</span>– <span class="NLM_lpage">454</span>, <span class="refDoi"> DOI: 10.1111/j.1399-3011.1988.tb01375.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1111%2Fj.1399-3011.1988.tb01375.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=3149952" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADyaL1MXmtlemtLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=1988&pages=441-454&author=A.+M.+Felixauthor=E.+P.+Heimerauthor=C.+T.+Wangauthor=T.+J.+Lambrosauthor=A.+Fournierauthor=T.+F.+Mowlesauthor=S.+Mainesauthor=R.+M.+Campbellauthor=B.+B.+Wegrzynskiauthor=V.+Toome&title=Synthesis%2C+Biological+Activity+and+Conformational+Analysis+of+Cyclic+GRF+Analogs&doi=10.1111%2Fj.1399-3011.1988.tb01375.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis, biological activity and conformational analysis of cyclic GRF analogs</span></div><div class="casAuthors">Felix, Arthur M.; Heimer, Edgar P.; Wang, Ching Tso; Lambros, Theodore J.; Fournier, Alain; Mowles, Thomas F.; Maines, Sarah; Campbell, Robert M.; Wegrzynski, Bogda B.; et al.</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Peptide & Protein Research</span>
        (<span class="NLM_cas:date">1988</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">441-54</span>CODEN:
                <span class="NLM_cas:coden">IJPPC3</span>;
        ISSN:<span class="NLM_cas:issn">0367-8377</span>.
    </div><div class="casAbstract">A novel cyclic growth hormone-releasing factor (GRF) analog, cyclo(Asp8-Lys12)-[Asp8,Ala15]-GRF(1-29)-NH2 (I), was synthesized by the solid-phase procedure and found to retain significant biol. activity.  Solid-phase cyclization of Asp8 to Lys12 proceeded rapidly (∼2 h) using the BOP reagent.  Substitution of Ala2 with D-Ala2 and/or NH2-terminal replacement (desNH2-Tyr1 or N-MeTyr1) in I resulted in highly potent analogs that were also active in vivo.  Conformational anal. (CD and mol. dynamics calcns. based on NOE-derived distance constraints) demonstrated that I contains a long α-helical segment even in aq. soln.  A series of cyclo8,12 stereoisomers contg. D-Asp2 and/or D-Lys12 were prepd. and also found to be highly potent and to retain significant α-helical conformation.  The high biol. activity of cyclo8,12[N-MeTyr1,D-Ala2,Asp8,Ala15]-GRF(1-29)-NH2 may be explained on the basis of retention of a preferred bioactive conformation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrDIM5X6XSbp7Vg90H21EOLACvtfcHk0lh0AE2XTB5S6w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1MXmtlemtLg%253D&md5=a9471ff69ff82b7250e51992b4e0d409</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1111%2Fj.1399-3011.1988.tb01375.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1399-3011.1988.tb01375.x%26sid%3Dliteratum%253Aachs%26aulast%3DFelix%26aufirst%3DA.%2BM.%26aulast%3DHeimer%26aufirst%3DE.%2BP.%26aulast%3DWang%26aufirst%3DC.%2BT.%26aulast%3DLambros%26aufirst%3DT.%2BJ.%26aulast%3DFournier%26aufirst%3DA.%26aulast%3DMowles%26aufirst%3DT.%2BF.%26aulast%3DMaines%26aufirst%3DS.%26aulast%3DCampbell%26aufirst%3DR.%2BM.%26aulast%3DWegrzynski%26aufirst%3DB.%2BB.%26aulast%3DToome%26aufirst%3DV.%26atitle%3DSynthesis%252C%2520Biological%2520Activity%2520and%2520Conformational%2520Analysis%2520of%2520Cyclic%2520GRF%2520Analogs%26jtitle%3DInt.%2520J.%2520Pept.%2520Protein%2520Res.%26date%3D1988%26volume%3D32%26spage%3D441%26epage%3D454%26doi%3D10.1111%2Fj.1399-3011.1988.tb01375.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, J. W.</span></span> <span> </span><span class="NLM_article-title">The Synthesis and Study of Side-Chain Lactam-Bridged Peptides</span>. <i>Biopolymers</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">49</span>– <span class="NLM_lpage">75</span>, <span class="refDoi"> DOI: 10.1002/bip.10203</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1002%2Fbip.10203" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=12228920" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BD38Xnsl2rsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2002&pages=49-75&author=J.+W.+Taylor&title=The+Synthesis+and+Study+of+Side-Chain+Lactam-Bridged+Peptides&doi=10.1002%2Fbip.10203"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">The synthesis and study of side-chain lactam-bridged peptides</span></div><div class="casAuthors">Taylor, John W.</div><div class="citationInfo"><span class="NLM_cas:title">Biopolymers</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">49-75</span>CODEN:
                <span class="NLM_cas:coden">BIPMAA</span>;
        ISSN:<span class="NLM_cas:issn">0006-3525</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A review.  This article reviews some of the synthetic approaches that have been used to prep. peptides with side-chain lactam bridges, conformational effects of these bridges and their applications to studies of bioactive peptides.  For example, some lactam-bridged peptides are designed to have increased receptor binding affinity.  Peptides with side-chain lactam bridges may also serve as models for protein folding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqMgDLaY7mzdrVg90H21EOLACvtfcHk0lh0AE2XTB5S6w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xnsl2rsb4%253D&md5=9f0e8da81c533ac03a9f1f1bfca7200b</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1002%2Fbip.10203&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fbip.10203%26sid%3Dliteratum%253Aachs%26aulast%3DTaylor%26aufirst%3DJ.%2BW.%26atitle%3DThe%2520Synthesis%2520and%2520Study%2520of%2520Side-Chain%2520Lactam-Bridged%2520Peptides%26jtitle%3DBiopolymers%26date%3D2002%26volume%3D66%26spage%3D49%26epage%3D75%26doi%3D10.1002%2Fbip.10203" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Blackwell, H. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grubbs, R. H.</span></span> <span> </span><span class="NLM_article-title">Highly Efficient Synthesis of Covalently Cross-Linked Peptide Helices by Ring-Closing Metathesis</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>37</i></span>,  <span class="NLM_fpage">3281</span>– <span class="NLM_lpage">3284</span>, <span class="refDoi"> DOI: 10.1002/(SICI)1521-3773(19981217)37:23<3281::AID-ANIE3281>3.0.CO;2-V</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1002%2F%28SICI%291521-3773%2819981217%2937%3A23%3C3281%3A%3AAID-ANIE3281%3E3.0.CO%3B2-V" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADyaK1MXksVehtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=1998&pages=3281-3284&author=H.+E.+Blackwellauthor=R.+H.+Grubbs&title=Highly+Efficient+Synthesis+of+Covalently+Cross-Linked+Peptide+Helices+by+Ring-Closing+Metathesis&doi=10.1002%2F%28SICI%291521-3773%2819981217%2937%3A23%3C3281%3A%3AAID-ANIE3281%3E3.0.CO%3B2-V"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Highly efficient synthesis of covalently cross-linked peptide helices by ring-closing metathesis</span></div><div class="casAuthors">Blackwell, Helen E.; Grubbs, Robert H.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">3281-3284</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH</span>)
        </div><div class="casAbstract">Ring closing metathesis of acyclic peptide dienes Boc-Val-X-Leu-Aib-Val-X-Leu-OMe [I; Boc = Me3CO2C; X = Ser(CH2CH:CH2), Hse(CH2CH:CH2); Aib = NHCMe2CO], prepd. by std. solid-phase peptide chem., using olefin metathesis catalyst (PCy3)2Cl2Ru:CHPh gave 21- and 23-membered macrocyclic alkenes I in 85 and 90% yields, resp.  Hydrogenation of II gave the corresponding alkane-bridged cyclopeptides III in 98% yield.  Significant conformational change did not occur on cyclization of I to III, as evidenced by far-UV CD.  A crystal structure of III (X = Hse) is reported.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqFCptOTOzERbVg90H21EOLACvtfcHk0liP6G3oK6u4NA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXksVehtA%253D%253D&md5=db2cc48a5a345762d90dc83f8287e67a</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291521-3773%2819981217%2937%3A23%3C3281%3A%3AAID-ANIE3281%3E3.0.CO%3B2-V&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291521-3773%252819981217%252937%253A23%253C3281%253A%253AAID-ANIE3281%253E3.0.CO%253B2-V%26sid%3Dliteratum%253Aachs%26aulast%3DBlackwell%26aufirst%3DH.%2BE.%26aulast%3DGrubbs%26aufirst%3DR.%2BH.%26atitle%3DHighly%2520Efficient%2520Synthesis%2520of%2520Covalently%2520Cross-Linked%2520Peptide%2520Helices%2520by%2520Ring-Closing%2520Metathesis%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D1998%26volume%3D37%26spage%3D3281%26epage%3D3284%26doi%3D10.1002%2F%28SICI%291521-3773%2819981217%2937%3A23%3C3281%3A%3AAID-ANIE3281%3E3.0.CO%3B2-V" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grossmann, T. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verdine, G. L.</span></span> <span> </span><span class="NLM_article-title">Synthesis of All-Hydrocarbon Stapled α-Helical Peptides by Ring-Closing Olefin Metathesis</span>. <i>Nat. Protoc.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">761</span>– <span class="NLM_lpage">771</span>, <span class="refDoi"> DOI: 10.1038/nprot.2011.324</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1038%2Fnprot.2011.324" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=21637196" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmvFyqurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&pages=761-771&author=Y.-W.+Kimauthor=T.+N.+Grossmannauthor=G.+L.+Verdine&title=Synthesis+of+All-Hydrocarbon+Stapled+%CE%B1-Helical+Peptides+by+Ring-Closing+Olefin+Metathesis&doi=10.1038%2Fnprot.2011.324"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis of all-hydrocarbon stapled α-helical peptides by ring-closing olefin metathesis</span></div><div class="casAuthors">Kim, Young-Woo; Grossmann, Tom N.; Verdine, Gregory L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">761-771</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">This protocol provides a detailed procedure for the prepn. of stapled α-helical peptides, which have proven their potential as useful mol. probes and as next-generation therapeutics.  Two crucial features of this protocol are the construction of peptide substrates contg. hindered α-Me, α-alkenyl amino acids and the ring-closing olefin metathesis (RCM) of the resulting resin-bound peptide substrates.  The stapling systems described in this protocol, namely bridging one or two turns of an α-helix, are highly adaptable to most peptide sequences, resulting in favorable RCM kinetics, helix stabilization and promotion of cellular uptake.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr2DCeMO4NgHLVg90H21EOLACvtfcHk0liP6G3oK6u4NA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmvFyqurg%253D&md5=02d269995eb1488632382c878df9fde4</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2011.324&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2011.324%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DY.-W.%26aulast%3DGrossmann%26aufirst%3DT.%2BN.%26aulast%3DVerdine%26aufirst%3DG.%2BL.%26atitle%3DSynthesis%2520of%2520All-Hydrocarbon%2520Stapled%2520%25CE%25B1-Helical%2520Peptides%2520by%2520Ring-Closing%2520Olefin%2520Metathesis%26jtitle%3DNat.%2520Protoc.%26date%3D2011%26volume%3D6%26spage%3D761%26epage%3D771%26doi%3D10.1038%2Fnprot.2011.324" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Verdine, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilinski, G. J.</span></span> <span> </span><span class="NLM_article-title">All-Hydrocarbon Stapled Peptides as Synthetic Cell-Accessible Mini-Proteins</span>. <i>Drug Discovery Today Technol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">e41</span>– <span class="NLM_lpage">e47</span>, <span class="refDoi"> DOI: 10.1016/j.ddtec.2012.01.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1016%2Fj.ddtec.2012.01.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=23264802" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC38XlvFaqsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2012&pages=e41-e47&author=G.+L.+Verdineauthor=G.+J.+Hilinski&title=All-Hydrocarbon+Stapled+Peptides+as+Synthetic+Cell-Accessible+Mini-Proteins&doi=10.1016%2Fj.ddtec.2012.01.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">All-hydrocarbon stapled peptides as Synthetic Cell-Accessible Mini-Proteins</span></div><div class="casAuthors">Verdine, Gregory L.; Hilinski, Gerard J.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today: Technologies</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">e41-e47</span>CODEN:
                <span class="NLM_cas:coden">DDTTB5</span>;
        ISSN:<span class="NLM_cas:issn">1740-6749</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  A majority of proteins responsible for the establishment and maintenance of human disease states are unable to be targeted therapeutically by mols. belonging to either of the two established classes of drugs, namely small mols. and protein therapeutics.  Recent efforts toward drugging these undruggable' proteins have led to greatly increased focus on cell-penetrating mini-proteins as a new class of agents for targeting intractable intracellular proteins.  These mols. are designed to combine the advantages intrinsic to each conventional therapeutic modality while overcoming their individual limitations.  One class of cell-penetrating mini-proteins, all-hydrocarbon stapled peptides, has recently demonstrated the ability to potently and specifically target previously intractable proteins such as transcription factors, vaulting this class to the forefront of a new wave of next-generation drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbIYOAoNrhCbVg90H21EOLACvtfcHk0liP6G3oK6u4NA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XlvFaqsLs%253D&md5=3d4b9e7c03349eab584c830e4252415f</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1016%2Fj.ddtec.2012.01.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ddtec.2012.01.004%26sid%3Dliteratum%253Aachs%26aulast%3DVerdine%26aufirst%3DG.%2BL.%26aulast%3DHilinski%26aufirst%3DG.%2BJ.%26atitle%3DAll-Hydrocarbon%2520Stapled%2520Peptides%2520as%2520Synthetic%2520Cell-Accessible%2520Mini-Proteins%26jtitle%3DDrug%2520Discovery%2520Today%2520Technol.%26date%3D2012%26volume%3D9%26spage%3De41%26epage%3De47%26doi%3D10.1016%2Fj.ddtec.2012.01.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bernal, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tyler, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korsmeyer, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walensky, L. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verdine, G. L.</span></span> <span> </span><span class="NLM_article-title">Reactivation of the P53 Tumor Suppressor Pathway by a Stapled P53 Peptide</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>129</i></span>,  <span class="NLM_fpage">2456</span>– <span class="NLM_lpage">2457</span>, <span class="refDoi"> DOI: 10.1021/ja0693587</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja0693587" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlGltrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=129&publication_year=2007&pages=2456-2457&author=F.+Bernalauthor=A.+F.+Tylerauthor=S.+J.+Korsmeyerauthor=L.+D.+Walenskyauthor=G.+L.+Verdine&title=Reactivation+of+the+P53+Tumor+Suppressor+Pathway+by+a+Stapled+P53+Peptide&doi=10.1021%2Fja0693587"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Reactivation of the p53 Tumor Suppressor Pathway by a Stapled p53 Peptide</span></div><div class="casAuthors">Bernal, Federico; Tyler, Andrew F.; Korsmeyer, Stanley J.; Walensky, Loren D.; Verdine, Gregory L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">129</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2456-2457</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The p53-hDM2 protein interaction is a validated therapeutic target in cancer.  We report the synthesis of stabilized alpha-helix of p53 (SAH-p53) compds. that antagonize the p53-hDM2 interaction.  We demonstrate that hydrocarbon stapling confers cellular permeability to a p53 peptide that is then capable of modulating transcriptional activity.  The lead SAH-p53 compd. triggers apoptosis in hDM2-overexpressing cancer cells by reactivating the native p53 signaling pathway.  SAH-p53 is the first example of an all-hydrocarbon i, i+7 stabilized peptide that subverts cancer through direct modulation of a transcriptional pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrU06ywOSa4CLVg90H21EOLACvtfcHk0liP6G3oK6u4NA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlGltrw%253D&md5=802905c5e6aaed4a224092f5603731fd</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Fja0693587&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja0693587%26sid%3Dliteratum%253Aachs%26aulast%3DBernal%26aufirst%3DF.%26aulast%3DTyler%26aufirst%3DA.%2BF.%26aulast%3DKorsmeyer%26aufirst%3DS.%2BJ.%26aulast%3DWalensky%26aufirst%3DL.%2BD.%26aulast%3DVerdine%26aufirst%3DG.%2BL.%26atitle%3DReactivation%2520of%2520the%2520P53%2520Tumor%2520Suppressor%2520Pathway%2520by%2520a%2520Stapled%2520P53%2520Peptide%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2007%26volume%3D129%26spage%3D2456%26epage%3D2457%26doi%3D10.1021%2Fja0693587" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bird, G. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madani, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perry, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Princiotto, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Supko, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gavathiotis, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sodroski, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walensky, L. D.</span></span> <span> </span><span class="NLM_article-title">Hydrocarbon Double-Stapling Remedies the Proteolytic Instability of a Lengthy Peptide Therapeutic</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>107</i></span>,  <span class="NLM_fpage">14093</span>– <span class="NLM_lpage">14098</span>, <span class="refDoi"> DOI: 10.1073/pnas.1002713107</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1073%2Fpnas.1002713107" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=20660316" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVCktrfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=107&publication_year=2010&pages=14093-14098&author=G.+H.+Birdauthor=N.+Madaniauthor=A.+F.+Perryauthor=A.+M.+Princiottoauthor=J.+G.+Supkoauthor=X.+Heauthor=E.+Gavathiotisauthor=J.+G.+Sodroskiauthor=L.+D.+Walensky&title=Hydrocarbon+Double-Stapling+Remedies+the+Proteolytic+Instability+of+a+Lengthy+Peptide+Therapeutic&doi=10.1073%2Fpnas.1002713107"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Hydrocarbon double-stapling remedies the proteolytic instability of a lengthy peptide therapeutic</span></div><div class="casAuthors">Bird, Gregory H.; Madani, Navid; Perry, Alisa F.; Princiotto, Amy M.; Supko, Jeffrey G.; He, Xiaoying; Gavathiotis, Evripidis; Sodroski, Joseph G.; Walensky, Loren D.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">107</span>
        (<span class="NLM_cas:issue">32</span>),
    <span class="NLM_cas:pages">14093-14098, S14093/1-S14093/8</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The pharmacol. utility of lengthy peptides can be hindered by loss of bioactive structure and rapid proteolysis, which limits bioavailability.  For example, enfuvirtide (Fuzeon, T20, DP178), a 36-amino acid peptide that inhibits human immunodeficiency virus type 1 (HIV-1) infection by effectively targeting the viral fusion app., has been relegated to a salvage treatment option mostly due to poor in vivo stability and lack of oral bioavailability.  To overcome the proteolytic shortcomings of long peptides as therapeutics, the authors examd. the biophys., biol., and pharmacol. impact of inserting all-hydrocarbon staples into an HIV-1 fusion inhibitor.  The authors find that peptide double-stapling confers striking protease resistance that translates into markedly improved pharmacokinetic properties, including oral absorption.  The authors detd. that the hydrocarbon staples create a proteolytic shield by combining reinforcement of overall α-helical structure, which slows the kinetics of proteolysis, with complete blockade of peptide cleavage at constrained sites in the immediate vicinity of the staple.  Importantly, double-stapling also optimizes the antiviral activity of HIV-1 fusion peptides and the antiproteolytic feature extends to other therapeutic peptide templates, such as the diabetes drug exenatide (Byetta).  Thus, hydrocarbon double-stapling may unlock the therapeutic potential of natural bioactive polypeptides by transforming them into structurally fortified agents with enhanced bioavailability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqme3pVM-fEB7Vg90H21EOLACvtfcHk0ljtMXl3NfuKuA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVCktrfJ&md5=95edd2112b56b564864e462eeb276e11</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1002713107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1002713107%26sid%3Dliteratum%253Aachs%26aulast%3DBird%26aufirst%3DG.%2BH.%26aulast%3DMadani%26aufirst%3DN.%26aulast%3DPerry%26aufirst%3DA.%2BF.%26aulast%3DPrinciotto%26aufirst%3DA.%2BM.%26aulast%3DSupko%26aufirst%3DJ.%2BG.%26aulast%3DHe%26aufirst%3DX.%26aulast%3DGavathiotis%26aufirst%3DE.%26aulast%3DSodroski%26aufirst%3DJ.%2BG.%26aulast%3DWalensky%26aufirst%3DL.%2BD.%26atitle%3DHydrocarbon%2520Double-Stapling%2520Remedies%2520the%2520Proteolytic%2520Instability%2520of%2520a%2520Lengthy%2520Peptide%2520Therapeutic%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2010%26volume%3D107%26spage%3D14093%26epage%3D14098%26doi%3D10.1073%2Fpnas.1002713107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Danial, N. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walensky, L. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fisher, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Molina, A. J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Datta, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pitter, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bird, G. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wikstrom, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deeney, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robertson, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morash, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kulkarni, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neschen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenberg, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corkey, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shirihai, O. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shulman, G. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lowell, B. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korsmeyer, S. J.</span></span> <span> </span><span class="NLM_article-title">Dual Role of Proapoptotic BAD in Insulin Secretion and Beta Cell Survival</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">144</span>– <span class="NLM_lpage">153</span>, <span class="refDoi"> DOI: 10.1038/nm1717</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1038%2Fnm1717" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=18223655" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhs1KhsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2008&pages=144-153&author=N.+N.+Danialauthor=L.+D.+Walenskyauthor=C.-Y.+Zhangauthor=C.+S.+Choiauthor=J.+K.+Fisherauthor=A.+J.+A.+Molinaauthor=S.+R.+Dattaauthor=K.+L.+Pitterauthor=G.+H.+Birdauthor=J.+D.+Wikstromauthor=J.+T.+Deeneyauthor=K.+Robertsonauthor=J.+Morashauthor=A.+Kulkarniauthor=S.+Neschenauthor=S.+Kimauthor=M.+E.+Greenbergauthor=B.+E.+Corkeyauthor=O.+S.+Shirihaiauthor=G.+I.+Shulmanauthor=B.+B.+Lowellauthor=S.+J.+Korsmeyer&title=Dual+Role+of+Proapoptotic+BAD+in+Insulin+Secretion+and+Beta+Cell+Survival&doi=10.1038%2Fnm1717"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Dual role of proapoptotic BAD in insulin secretion and beta cell survival</span></div><div class="casAuthors">Danial, Nika N.; Walensky, Loren D.; Zhang, Chen-Yu; Choi, Cheol Soo; Fisher, Jill K.; Molina, Anthony J. A.; Datta, Sandeep Robert; Pitter, Kenneth L.; Bird, Gregory H.; Wikstrom, Jakob D.; Deeney, Jude T.; Robertson, Kirsten; Morash, Joel; Kulkarni, Ameya; Neschen, Susanne; Kim, Sheene; Greenberg, Michael E.; Corkey, Barbara E.; Shirihai, Orian S.; Shulman, Gerald I.; Lowell, Bradford B.; Korsmeyer, Stanley J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">144-153</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The proapoptotic BCL-2 family member BAD resides in a glucokinase-contg. complex that regulates glucose-driven mitochondrial respiration.  Here, the authors present genetic evidence of a physiol. role for BAD in glucose-stimulated insulin secretion by beta cells.  This novel function of BAD is specifically dependent upon the phosphorylation of its BH3 sequence, previously defined as an essential death domain.  The authors highlight the pharmacol. relevance of phosphorylated BAD BH3 by using cell-permeable, hydrocarbon-stapled BAD BH3 helixes that target glucokinase, restore glucose-driven mitochondrial respiration and correct the insulin secretory response in Bad-deficient islets.  The authors' studies uncover an alternative target and function for the BAD BH3 domain and emphasize the therapeutic potential of phosphorylated BAD BH3 mimetics in selectively restoring beta cell function.  Furthermore, the authors show that BAD regulates the physiol. adaptation of beta cell mass during high-fat feeding.  The authors' findings provide genetic proof of the bifunctional activities of BAD in both beta cell survival and insulin secretion.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpQFhSE2I8jdLVg90H21EOLACvtfcHk0ljtMXl3NfuKuA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhs1KhsLo%253D&md5=204579cd0cabd3a8d372dbd23efdd869</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1038%2Fnm1717&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm1717%26sid%3Dliteratum%253Aachs%26aulast%3DDanial%26aufirst%3DN.%2BN.%26aulast%3DWalensky%26aufirst%3DL.%2BD.%26aulast%3DZhang%26aufirst%3DC.-Y.%26aulast%3DChoi%26aufirst%3DC.%2BS.%26aulast%3DFisher%26aufirst%3DJ.%2BK.%26aulast%3DMolina%26aufirst%3DA.%2BJ.%2BA.%26aulast%3DDatta%26aufirst%3DS.%2BR.%26aulast%3DPitter%26aufirst%3DK.%2BL.%26aulast%3DBird%26aufirst%3DG.%2BH.%26aulast%3DWikstrom%26aufirst%3DJ.%2BD.%26aulast%3DDeeney%26aufirst%3DJ.%2BT.%26aulast%3DRobertson%26aufirst%3DK.%26aulast%3DMorash%26aufirst%3DJ.%26aulast%3DKulkarni%26aufirst%3DA.%26aulast%3DNeschen%26aufirst%3DS.%26aulast%3DKim%26aufirst%3DS.%26aulast%3DGreenberg%26aufirst%3DM.%2BE.%26aulast%3DCorkey%26aufirst%3DB.%2BE.%26aulast%3DShirihai%26aufirst%3DO.%2BS.%26aulast%3DShulman%26aufirst%3DG.%2BI.%26aulast%3DLowell%26aufirst%3DB.%2BB.%26aulast%3DKorsmeyer%26aufirst%3DS.%2BJ.%26atitle%3DDual%2520Role%2520of%2520Proapoptotic%2520BAD%2520in%2520Insulin%2520Secretion%2520and%2520Beta%2520Cell%2520Survival%26jtitle%3DNat.%2520Med.%26date%3D2008%26volume%3D14%26spage%3D144%26epage%3D153%26doi%3D10.1038%2Fnm1717" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lawrence, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jourdan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vallée, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blandin, V.</span></span> <span> </span><span class="NLM_article-title">Peptide Cyclization via Ring-Closing Metathesis: The N-Alkenoxy Peptide Approach</span>. <i>Org. Biomol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">4575</span>– <span class="NLM_lpage">4581</span>, <span class="refDoi"> DOI: 10.1039/b812611a</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1039%2Fb812611a" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=19039366" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsVChtb7L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2008&pages=4575-4581&author=J.+Lawrenceauthor=M.+Jourdanauthor=Y.+Vall%C3%A9eauthor=V.+Blandin&title=Peptide+Cyclization+via+Ring-Closing+Metathesis%3A+The+N-Alkenoxy+Peptide+Approach&doi=10.1039%2Fb812611a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Peptide cyclization via ring-closing metathesis: the N-alkenoxy peptide approach</span></div><div class="casAuthors">Lawrence, James; Jourdan, Muriel; Vallee, Yannick; Blandin, Veronique</div><div class="citationInfo"><span class="NLM_cas:title">Organic & Biomolecular Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">4575-4581</span>CODEN:
                <span class="NLM_cas:coden">OBCRAK</span>;
        ISSN:<span class="NLM_cas:issn">1477-0520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">The prepn. of 17- and 18-membered cyclic hexapeptides from N-hydroxy tripeptides building blocks is described.  Introduction of an unsatd. chain on the hydroxamate oxygen followed by fragment coupling leads to N,N'-dialkenoxy hexapeptides that are efficiently cyclized through a ring-closing metathesis reaction.  The length of the alkene chains allows the modulation of the ring size.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8raTcvXy91rVg90H21EOLACvtfcHk0ljtMXl3NfuKuA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsVChtb7L&md5=d45a64eb1bc38d471c11e27a34357ab7</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1039%2Fb812611a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fb812611a%26sid%3Dliteratum%253Aachs%26aulast%3DLawrence%26aufirst%3DJ.%26aulast%3DJourdan%26aufirst%3DM.%26aulast%3DVall%25C3%25A9e%26aufirst%3DY.%26aulast%3DBlandin%26aufirst%3DV.%26atitle%3DPeptide%2520Cyclization%2520via%2520Ring-Closing%2520Metathesis%253A%2520The%2520N-Alkenoxy%2520Peptide%2520Approach%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2008%26volume%3D6%26spage%3D4575%26epage%3D4581%26doi%3D10.1039%2Fb812611a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sviridov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ikpot, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stonik, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drake, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amar, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osei-Hwedieh, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piszczek, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turner, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Remaley, A.</span></span> <span> </span><span class="NLM_article-title">Helix Stabilization of Amphipathic Peptides by Hydrocarbon Stapling Increases Cholesterol Efflux by the ABCA1 Transporter</span>. <i>Biochem. Biophys. Res. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>410</i></span>,  <span class="NLM_fpage">446</span>– <span class="NLM_lpage">451</span>, <span class="refDoi"> DOI: 10.1016/j.bbrc.2011.05.154</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1016%2Fj.bbrc.2011.05.154" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=21672528" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC3MXoslCns7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=410&publication_year=2011&pages=446-451&author=D.+Sviridovauthor=I.+Ikpotauthor=J.+Stonikauthor=S.+Drakeauthor=M.+Amarauthor=D.+Osei-Hwediehauthor=G.+Piszczekauthor=S.+Turnerauthor=A.+Remaley&title=Helix+Stabilization+of+Amphipathic+Peptides+by+Hydrocarbon+Stapling+Increases+Cholesterol+Efflux+by+the+ABCA1+Transporter&doi=10.1016%2Fj.bbrc.2011.05.154"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Helix stabilization of amphipathic peptides by hydrocarbon stapling increases cholesterol efflux by the ABCA1 transporter</span></div><div class="casAuthors">Sviridov, D. O.; Ikpot, I. Z.; Stonik, J.; Drake, S. K.; Amar, M.; Osei-Hwedieh, D. O.; Piszczek, G.; Turner, S.; Remaley, A. T.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical and Biophysical Research Communications</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">410</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">446-451</span>CODEN:
                <span class="NLM_cas:coden">BBRCA9</span>;
        ISSN:<span class="NLM_cas:issn">0006-291X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Apolipoprotein mimetic peptides are short amphipathic peptides that efflux cholesterol from cells by the ABCA1 transporter and are being investigated as therapeutic agents for cardiovascular disease.  We examd. the role of helix stabilization of these peptides in cholesterol efflux.  A 23-amino acid long peptide (Ac-VLEDSFKVSFLSALEEYTKKLNTQ-NH2) based on the last helix of apoA-I (A10) was synthesized, as well as two variants, S1A10 and S2A10, in which the third and fourth and third and fifth turn of each peptide, resp., were covalently joined by hydrocarbon staples.  By CD spectroscopy, the stapled variants at 24° were more helical in aq. buffer than A10 (A10 17%, S1A10 62%, S2A10 97%).  S1A10 and S2A10 unlike A10 were resistant to proteolysis by pepsin and chymotrypsin.  S1A10 and S2A10 showed more than a 10-fold increase in cholesterol efflux by the ABCA1 transporter compared to A10.  In summary, hydrocarbon stapling of amphipathic peptides increases their helicity, makes them resistant to proteolysis and enhances their ability to promote cholesterol efflux by the ABCA1 transporter, indicating that this peptide modification may be useful in the development of apolipoprotein mimetic peptides.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrK6lZ57TEU2bVg90H21EOLACvtfcHk0lhk1sXCpHKJvg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXoslCns7s%253D&md5=4285715e1d9ba3410a36f2273b82d199</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1016%2Fj.bbrc.2011.05.154&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbrc.2011.05.154%26sid%3Dliteratum%253Aachs%26aulast%3DSviridov%26aufirst%3DD.%26aulast%3DIkpot%26aufirst%3DI.%26aulast%3DStonik%26aufirst%3DJ.%26aulast%3DDrake%26aufirst%3DS.%26aulast%3DAmar%26aufirst%3DM.%26aulast%3DOsei-Hwedieh%26aufirst%3DD.%26aulast%3DPiszczek%26aufirst%3DG.%26aulast%3DTurner%26aufirst%3DS.%26aulast%3DRemaley%26aufirst%3DA.%26atitle%3DHelix%2520Stabilization%2520of%2520Amphipathic%2520Peptides%2520by%2520Hydrocarbon%2520Stapling%2520Increases%2520Cholesterol%2520Efflux%2520by%2520the%2520ABCA1%2520Transporter%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D2011%26volume%3D410%26spage%3D446%26epage%3D451%26doi%3D10.1016%2Fj.bbrc.2011.05.154" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Walensky, L. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kung, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Escher, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malia, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbuto, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wright, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verdine, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korsmeyer, S. J.</span></span> <span> </span><span class="NLM_article-title">Activation of Apoptosis in Vivo by a Hydrocarbon-Stapled BH3 Helix</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>305</i></span>,  <span class="NLM_fpage">1466</span>– <span class="NLM_lpage">1470</span>, <span class="refDoi"> DOI: 10.1126/science.1099191</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1126%2Fscience.1099191" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=15353804" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BD2cXntFart7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=305&publication_year=2004&pages=1466-1470&author=L.+D.+Walenskyauthor=A.+L.+Kungauthor=I.+Escherauthor=T.+J.+Maliaauthor=S.+Barbutoauthor=R.+D.+Wrightauthor=G.+Wagnerauthor=G.+L.+Verdineauthor=S.+J.+Korsmeyer&title=Activation+of+Apoptosis+in+Vivo+by+a+Hydrocarbon-Stapled+BH3+Helix&doi=10.1126%2Fscience.1099191"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Activation of Apoptosis in Vivo by a Hydrocarbon-Stapled BH3 Helix</span></div><div class="casAuthors">Walensky, Loren D.; Kung, Andrew L.; Escher, Iris; Malia, Thomas J.; Barbuto, Scott; Wright, Renee D.; Wagner, Gerhard; Verdine, Gregory L.; Korsmeyer, Stanley J.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">305</span>
        (<span class="NLM_cas:issue">5689</span>),
    <span class="NLM_cas:pages">1466-1470</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">BCL-2 family proteins constitute a crit. control point for the regulation of apoptosis.  Protein interaction between BCL-2 members is a prominent mechanism of control and is mediated through the amphipathic α-helical BH3 segment, an essential death domain.  We used a chem. strategy, termed hydrocarbon stapling, to generate BH3 peptides with improved pharmacol. properties.  The stapled peptides, called "stabilized alpha-helix of BCL-2 domains" (SAHBs), proved to be helical, protease-resistant, and cell-permeable mols. that bound with increased affinity to multidomain BCL-2 member pockets.  A SAHB of the BH3 domain from the BID protein specifically activated the apoptotic pathway to kill leukemia cells.  In addn., SAHB effectively inhibited the growth of human leukemia xenografts in vivo.  Hydrocarbon stapling of native peptides may provide a useful strategy for exptl. and therapeutic modulation of protein-protein interactions in many signaling pathways.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrpfOpXrd8FVbVg90H21EOLACvtfcHk0lhk1sXCpHKJvg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXntFart7c%253D&md5=53fca0524df7a584ccf9d0991bc1bb64</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1126%2Fscience.1099191&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1099191%26sid%3Dliteratum%253Aachs%26aulast%3DWalensky%26aufirst%3DL.%2BD.%26aulast%3DKung%26aufirst%3DA.%2BL.%26aulast%3DEscher%26aufirst%3DI.%26aulast%3DMalia%26aufirst%3DT.%2BJ.%26aulast%3DBarbuto%26aufirst%3DS.%26aulast%3DWright%26aufirst%3DR.%2BD.%26aulast%3DWagner%26aufirst%3DG.%26aulast%3DVerdine%26aufirst%3DG.%2BL.%26aulast%3DKorsmeyer%26aufirst%3DS.%2BJ.%26atitle%3DActivation%2520of%2520Apoptosis%2520in%2520Vivo%2520by%2520a%2520Hydrocarbon-Stapled%2520BH3%2520Helix%26jtitle%3DScience%26date%3D2004%26volume%3D305%26spage%3D1466%26epage%3D1470%26doi%3D10.1126%2Fscience.1099191" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhattacharya, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waheed, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tong, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heck, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curreli, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goger, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowburn, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freed, E. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Debnath, A. K.</span></span> <span> </span><span class="NLM_article-title">A Cell-Penetrating Helical Peptide as a Potential HIV-1 Inhibitor</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>378</i></span>,  <span class="NLM_fpage">565</span>– <span class="NLM_lpage">580</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2008.02.066</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1016%2Fj.jmb.2008.02.066" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=18374356" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BD1cXlslWks7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=378&publication_year=2008&pages=565-580&author=H.+Zhangauthor=Q.+Zhaoauthor=S.+Bhattacharyaauthor=A.+A.+Waheedauthor=X.+Tongauthor=A.+Hongauthor=S.+Heckauthor=F.+Curreliauthor=M.+Gogerauthor=D.+Cowburnauthor=E.+O.+Freedauthor=A.+K.+Debnath&title=A+Cell-Penetrating+Helical+Peptide+as+a+Potential+HIV-1+Inhibitor&doi=10.1016%2Fj.jmb.2008.02.066"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">A Cell-penetrating Helical Peptide as a Potential HIV-1 Inhibitor</span></div><div class="casAuthors">Zhang, Hongtao; Zhao, Qian; Bhattacharya, Shibani; Waheed, Abdul A.; Tong, Xiaohe; Hong, Anita; Heck, Susanne; Curreli, Francesca; Goger, Michael; Cowburn, David; Freed, Eric O.; Debnath, Asim K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">378</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">565-580</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The capsid domain of the human immunodeficiency virus type 1 (HIV-1) Gag polyprotein is a crit. determinant of virus assembly, and is therefore a potential target for developing drugs for AIDS therapy.  Recently, a 12-mer α-helical peptide (CAI) was reported to disrupt immature- and mature-like capsid particle assembly in vitro; however, it failed to inhibit HIV-1 in cell culture due to its inability to penetrate cells.  The same group reported the x-ray crystal structure of CAI in complex with the C-terminal domain of capsid (C-CA) at a resoln. of 1.7 Å.  Using this structural information, the authors have utilized a structure-based rational design approach to stabilize the α-helical structure of CAI and convert it to a cell-penetrating peptide (CPP).  The modified peptide (NYAD-1) showed enhanced α-helicity.  Expts. with laser scanning confocal microscopy indicated that NYAD-1 penetrated cells and colocalized with the Gag polyprotein during its trafficking to the plasma membrane where virus assembly takes place.  NYAD-1 disrupted the assembly of both immature- and mature-like virus particles in cell-free and cell-based in vitro systems.  NMR chem. shift perturbation anal. mapped the binding site of NYAD-1 to residues 169-191 of the C-terminal domain of HIV-1 capsid encompassing the hydrophobic cavity and the crit. dimerization domain with an improved binding affinity over CAI.  Furthermore, exptl. data indicate that NYAD-1 most likely targets capsid at a post-entry stage.  Most significantly, NYAD-1 inhibited a large panel of HIV-1 isolates in cell culture at low micromolar potency.  The study demonstrates how a structure-based rational design strategy can be used to convert a cell-impermeable peptide to a cell-permeable peptide that displays activity in cell-based assays without compromising its mechanism of action.  This proof-of-concept cell-penetrating peptide may aid validation of capsid as an anti-HIV-1 drug target and may help in designing peptidomimetics and small mol. drugs targeted to this protein.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGql5DKXGiSgWLVg90H21EOLACvtfcHk0lhk1sXCpHKJvg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXlslWks7w%253D&md5=9f22fccdd6dc38c9e5c8841c63216436</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2008.02.066&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2008.02.066%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DZhao%26aufirst%3DQ.%26aulast%3DBhattacharya%26aufirst%3DS.%26aulast%3DWaheed%26aufirst%3DA.%2BA.%26aulast%3DTong%26aufirst%3DX.%26aulast%3DHong%26aufirst%3DA.%26aulast%3DHeck%26aufirst%3DS.%26aulast%3DCurreli%26aufirst%3DF.%26aulast%3DGoger%26aufirst%3DM.%26aulast%3DCowburn%26aufirst%3DD.%26aulast%3DFreed%26aufirst%3DE.%2BO.%26aulast%3DDebnath%26aufirst%3DA.%2BK.%26atitle%3DA%2520Cell-Penetrating%2520Helical%2520Peptide%2520as%2520a%2520Potential%2520HIV-1%2520Inhibitor%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2008%26volume%3D378%26spage%3D565%26epage%3D580%26doi%3D10.1016%2Fj.jmb.2008.02.066" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rostovtsev, V. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, L. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fokin, V. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharpless, K. B.</span></span> <span> </span><span class="NLM_article-title">A Stepwise Huisgen Cycloaddition Process: Copper(I)-Catalyzed Regioselective “Ligation” of Azides and Terminal Alkynes</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">2596</span>– <span class="NLM_lpage">2599</span>, <span class="refDoi"> DOI: 10.1002/1521-3773(20020715)41:14<2596::AID-ANIE2596>3.0.CO;2-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1002%2F1521-3773%2820020715%2941%3A14%3C2596%3A%3AAID-ANIE2596%3E3.0.CO%3B2-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=12203546" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BD38Xls1Ohsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2002&pages=2596-2599&author=V.+V.+Rostovtsevauthor=L.+G.+Greenauthor=V.+V.+Fokinauthor=K.+B.+Sharpless&title=A+Stepwise+Huisgen+Cycloaddition+Process%3A+Copper%28I%29-Catalyzed+Regioselective+%E2%80%9CLigation%E2%80%9D+of+Azides+and+Terminal+Alkynes&doi=10.1002%2F1521-3773%2820020715%2941%3A14%3C2596%3A%3AAID-ANIE2596%3E3.0.CO%3B2-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">A stepwise Huisgen cycloaddition process: copper(I)-catalyzed regioselective "ligation" of azides and terminal alkynes</span></div><div class="casAuthors">Rostovtsev, Vsevolod V.; Green, Luke G.; Fokin, Valery V.; Sharpless, K. Barry</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">2596-2599</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH</span>)
        </div><div class="casAbstract">1,4-Disubstituted 1,2,3-triazoles I (R1 = PhCH2, PhCH2OCH2, 1-adamantyl, etc.; R2 = HO2C, Ph, PhOCH2, Et2NCH2, etc.) were readily and cleanly prepd. via highly efficient and regioselective copper(I)-catalyzed cycloaddn. of azides R1N3 with terminal alkynes R2C≡CH in 82-93% yields.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptcW2JUNM1r7Vg90H21EOLACvtfcHk0liPR98InKr5Cw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xls1Ohsr4%253D&md5=4603664be6639353b5e70f19b9f8d59f</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1002%2F1521-3773%2820020715%2941%3A14%3C2596%3A%3AAID-ANIE2596%3E3.0.CO%3B2-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F1521-3773%252820020715%252941%253A14%253C2596%253A%253AAID-ANIE2596%253E3.0.CO%253B2-4%26sid%3Dliteratum%253Aachs%26aulast%3DRostovtsev%26aufirst%3DV.%2BV.%26aulast%3DGreen%26aufirst%3DL.%2BG.%26aulast%3DFokin%26aufirst%3DV.%2BV.%26aulast%3DSharpless%26aufirst%3DK.%2BB.%26atitle%3DA%2520Stepwise%2520Huisgen%2520Cycloaddition%2520Process%253A%2520Copper%2528I%2529-Catalyzed%2520Regioselective%2520%25E2%2580%259CLigation%25E2%2580%259D%2520of%2520Azides%2520and%2520Terminal%2520Alkynes%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2002%26volume%3D41%26spage%3D2596%26epage%3D2599%26doi%3D10.1002%2F1521-3773%2820020715%2941%3A14%3C2596%3A%3AAID-ANIE2596%3E3.0.CO%3B2-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tornøe, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christensen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meldal, M.</span></span> <span> </span><span class="NLM_article-title">Peptidotriazoles on Solid Phase: [1,2,3]-Triazoles by Regiospecific Copper(I)-Catalyzed 1,3-Dipolar Cycloadditions of Terminal Alkynes to Azides</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">3057</span>– <span class="NLM_lpage">3064</span>, <span class="refDoi"> DOI: 10.1021/jo011148j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo011148j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BD38XisVeks7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2002&pages=3057-3064&author=C.+W.+Torn%C3%B8eauthor=C.+Christensenauthor=M.+Meldal&title=Peptidotriazoles+on+Solid+Phase%3A+%5B1%2C2%2C3%5D-Triazoles+by+Regiospecific+Copper%28I%29-Catalyzed+1%2C3-Dipolar+Cycloadditions+of+Terminal+Alkynes+to+Azides&doi=10.1021%2Fjo011148j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Peptidotriazoles on Solid Phase: [1,2,3]-Triazoles by Regiospecific Copper(I)-Catalyzed 1,3-Dipolar Cycloadditions of Terminal Alkynes to Azides</span></div><div class="casAuthors">Tornoe, Christian W.; Christensen, Caspar; Meldal, Morten</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3057-3064</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The cycloaddn. of azides to alkynes is one of the most important synthetic routes to 1H-[1,2,3]-triazoles.  This work reports a novel regiospecific copper(I)-catalyzed 1,3-dipolar cycloaddn. of terminal alkynes to azides on solid-phase.  Primary, secondary, and tertiary alkyl azides, aryl azides, and an azido sugar were used successfully in the copper(I)-catalyzed cycloaddn. producing diversely 1,4-substituted [1,2,3]-triazoles in peptide backbones or side chains.  The reaction conditions were fully compatible with solid-phase peptide synthesis on polar supports.  The copper(I) catalysis is mild and efficient (>95% conversion and purity in most cases) and furthermore, the x-ray structure of 2-azido-2-methylpropanoic acid has been solved, to yield structural information on the 1,3-dipoles entering the reaction.  Novel Fmoc-protected amino azides were prepd. from Fmoc-amino alcs. by Mitsunobu reaction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJXSF5l0g_lrVg90H21EOLACvtfcHk0liPR98InKr5Cw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XisVeks7w%253D&md5=6b3b805572783873952871f1b69f46dd</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1021%2Fjo011148j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo011148j%26sid%3Dliteratum%253Aachs%26aulast%3DTorn%25C3%25B8e%26aufirst%3DC.%2BW.%26aulast%3DChristensen%26aufirst%3DC.%26aulast%3DMeldal%26aufirst%3DM.%26atitle%3DPeptidotriazoles%2520on%2520Solid%2520Phase%253A%2520%255B1%252C2%252C3%255D-Triazoles%2520by%2520Regiospecific%2520Copper%2528I%2529-Catalyzed%25201%252C3-Dipolar%2520Cycloadditions%2520of%2520Terminal%2520Alkynes%2520to%2520Azides%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2002%26volume%3D67%26spage%3D3057%26epage%3D3064%26doi%3D10.1021%2Fjo011148j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kolb, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finn, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharpless, K. B.</span></span> <span> </span><span class="NLM_article-title">Click Chemistry: Diverse Chemical Function from a Few Good Reactions</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">2004</span>– <span class="NLM_lpage">2021</span>, <span class="refDoi"> DOI: 10.1002/1521-3773(20010601)40:11<2004::AID-ANIE2004>3.0.CO;2-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1002%2F1521-3773%2820010601%2940%3A11%3C2004%3A%3AAID-ANIE2004%3E3.0.CO%3B2-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=11433435" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BD3MXksVOis78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2001&pages=2004-2021&author=H.+C.+Kolbauthor=M.+G.+Finnauthor=K.+B.+Sharpless&title=Click+Chemistry%3A+Diverse+Chemical+Function+from+a+Few+Good+Reactions&doi=10.1002%2F1521-3773%2820010601%2940%3A11%3C2004%3A%3AAID-ANIE2004%3E3.0.CO%3B2-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Click chemistry: diverse chemical function from a few good reactions</span></div><div class="casAuthors">Kolb, Hartmuth C.; Finn, M. G.; Sharpless, K. Barry</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2004-2021</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH</span>)
        </div><div class="casAbstract">Review with > 88 refs.  Examn. of nature's favorite mols. reveals a striking preference for making carbon-heteroatom bonds over carbon-carbon bonds - surely no surprise given that carbon dioxide is nature's starting material and that most reactions are performed in water.  Nucleic acids, proteins, and polysaccharides are condensation polymers of small subunits stitched together by carbon-heteroatom bonds.  Even the 35 or so building blocks from which these crucial mols. are made each contain, at most, six contiguous C-C bonds, except for the three arom. amino acids.  Taking a cue from nature's approach, the development of a set of powerful, highly reliable, and selective reactions for the rapid synthesis of useful new compds. and combinatorial libraries through heteroatom links (C-X-C), an approach called "click chem." is addressed.  Click chem. is at once defined, enabled, and constrained by a handful of nearly perfect "spring-loaded" reactions.  The stringent criteria for a process to earn click chem. status are described along with examples of the mol. frameworks that are easily made using this spartan, but powerful, synthetic strategy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptrjo9nxXNRrVg90H21EOLACvtfcHk0liPR98InKr5Cw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXksVOis78%253D&md5=61c28b416c42c87821bc8a5bc2fdec36</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1002%2F1521-3773%2820010601%2940%3A11%3C2004%3A%3AAID-ANIE2004%3E3.0.CO%3B2-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F1521-3773%252820010601%252940%253A11%253C2004%253A%253AAID-ANIE2004%253E3.0.CO%253B2-5%26sid%3Dliteratum%253Aachs%26aulast%3DKolb%26aufirst%3DH.%2BC.%26aulast%3DFinn%26aufirst%3DM.%2BG.%26aulast%3DSharpless%26aufirst%3DK.%2BB.%26atitle%3DClick%2520Chemistry%253A%2520Diverse%2520Chemical%2520Function%2520from%2520a%2520Few%2520Good%2520Reactions%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2001%26volume%3D40%26spage%3D2004%26epage%3D2021%26doi%3D10.1002%2F1521-3773%2820010601%2940%3A11%3C2004%3A%3AAID-ANIE2004%3E3.0.CO%3B2-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cantel, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Isaad, A. L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scrima, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levy, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DiMarchi, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rovero, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halperin, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">D’Ursi, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Papini, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chorev, M.</span></span> <span> </span><span class="NLM_article-title">Synthesis and Conformational Analysis of a Cyclic Peptide Obtained via i to I+4 Intramolecular Side-Chain to Side-Chain Azide-Alkyne 1,3-Dipolar Cycloaddition</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>73</i></span>,  <span class="NLM_fpage">5663</span>– <span class="NLM_lpage">5674</span>, <span class="refDoi"> DOI: 10.1021/jo800142s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo800142s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BD1cXmtVGhs78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2008&pages=5663-5674&author=S.+Cantelauthor=A.+L.+C.+Isaadauthor=M.+Scrimaauthor=J.+J.+Levyauthor=R.+D.+DiMarchiauthor=P.+Roveroauthor=J.+A.+Halperinauthor=A.+M.+D%E2%80%99Ursiauthor=A.+M.+Papiniauthor=M.+Chorev&title=Synthesis+and+Conformational+Analysis+of+a+Cyclic+Peptide+Obtained+via+i+to+I%2B4+Intramolecular+Side-Chain+to+Side-Chain+Azide-Alkyne+1%2C3-Dipolar+Cycloaddition&doi=10.1021%2Fjo800142s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and conformational analysis of a cyclic peptide obtained via i to i+4 intramolecular side-chain to side-chain azide-alkyne 1,3-dipolar cycloaddition</span></div><div class="casAuthors">Cantel, Sonia; Le Chevalier Isaad, Alexandra; Scrima, Mario; Levy, Jay J.; DiMarchi, Richard D.; Rovero, Paolo; Halperin, Jose A.; D'Ursi, Anna Maria; Papini, Anna Maria; Chorev, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">5663-5674</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Intramol. side-chain to side-chain cyclization is an established approach to achieve stabilization of specific conformations and a recognized strategy to improve resistance toward proteolytic degrdn.  To this end, cyclizations, which are bioisosteric to the lactam-type side-chain to side-chain modification and do not require orthogonal protection schemes, are of great interest.  Herein, we report the employment of CuI-catalyzed 1,3-dipolar cycloaddn. of side chains modified with azido and alkynyl functions and explore alternative synthetic routes to efficiently generate 1,4-disubstituted [1,2,3]triazolyl-contg. cyclopeptides.  The solid-phase assembly of the linear precursor including ε-azido norleucine and the propargylglycine (Pra) in positions i and i+4, resp., was accomplished by either subjecting the resin-bound peptide to selective on-resin diazo transformation of a Lys into the Nle(ε-N3) or the incorporation of Fmoc-Nle(ε-N3)-OH during the stepwise build-up of the resin-bound peptide.  Soln.-phase CuI-catalyzed 1,3-dipolar cycloaddn. converts the linear precursor Ac-Lys-Gly-Nle(ε-N3)-Ser-Ile-Gln-Pra-Leu-Arg-NH2 into the 1,4-disubstituted [1,2,3]triazolyl-contg. cyclopeptide [Ac-Lys-Gly-Xaa(&1)-Ser-Ile-Gln-Yaa(&2)-Leu-Arg-NH2][(&1(CH2)4-1,4-[1,2,3]triazolyl-CH2&2)].  The conformational preferences of the latter model cyclopeptide, which is derived from the sequence of a highly helical and potent i to i+4 side-chain to side-chain lactam-contg. antagonist of parathyroid hormone-related peptide (PTHrP), are compared to the corresponding lactam analog Ac[Lys13(&1),Asp17(&2)]hPTHrP(11-19)NH2.  CD and NMR studies of [Ac-Lys-Gly-Xaa(&1)-Ser-Ile-Gln-Yaa(&2)-Leu-Arg-NH2][(&1(CH2)4-1,4-[1,2,3]triazolyl-CH2&2)] and Ac[Lys13(&1),Asp17(&2)]hPTHrP(11-19)NH2 in water/hexafluoroacetone (HFA) (50:50, vol./vol.) revealed a high prevalence of turn-helical structures involving in particular the cyclic regions of the mol.  Despite a slight difference of the backbone arrangement, the side-chains of Ser, Gln, and Ile located at the i+1 to i+3 of the ring-forming sequences share the same spatial orientation.  Both cyclopeptides differ regarding the location of the turn-helical segment, which in Ac[Lys13(&1),Asp17(&2)]hPTHrP(11-19)NH2 involves noncyclized residues while in [Ac-Lys-Gly-Xaa(&1)-Ser-Ile-Gln-Yaa(&2)-Leu-Arg-NH2][(&1(CH2)4-1,4-[1,2,3]triazolyl-CH2&2)] it overlaps with residues involved in the cyclic structure.  Therefore, the synthetic accessibility and conformational similarity of i to i+4 side-chain to side-chain cyclopeptide contg. the 1,4-disubstituted [1,2,3]triazolyl moiety to the lactam-type one may result in similar bioactivities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWO7f-VCjhQrVg90H21EOLACvtfcHk0liPR98InKr5Cw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXmtVGhs78%253D&md5=6f761bb8554306cf71d3facc87bb0683</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1021%2Fjo800142s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo800142s%26sid%3Dliteratum%253Aachs%26aulast%3DCantel%26aufirst%3DS.%26aulast%3DIsaad%26aufirst%3DA.%2BL.%2BC.%26aulast%3DScrima%26aufirst%3DM.%26aulast%3DLevy%26aufirst%3DJ.%2BJ.%26aulast%3DDiMarchi%26aufirst%3DR.%2BD.%26aulast%3DRovero%26aufirst%3DP.%26aulast%3DHalperin%26aufirst%3DJ.%2BA.%26aulast%3DD%25E2%2580%2599Ursi%26aufirst%3DA.%2BM.%26aulast%3DPapini%26aufirst%3DA.%2BM.%26aulast%3DChorev%26aufirst%3DM.%26atitle%3DSynthesis%2520and%2520Conformational%2520Analysis%2520of%2520a%2520Cyclic%2520Peptide%2520Obtained%2520via%2520i%2520to%2520I%252B4%2520Intramolecular%2520Side-Chain%2520to%2520Side-Chain%2520Azide-Alkyne%25201%252C3-Dipolar%2520Cycloaddition%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2008%26volume%3D73%26spage%3D5663%26epage%3D5674%26doi%3D10.1021%2Fjo800142s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Scrima, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chevalier-Isaad, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rovero, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Papini, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chorev, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">D’Ursi, A. M.</span></span> <span> </span><span class="NLM_article-title">CuI-Catalyzed Azide–Alkyne Intramolecular i-to-(I+4) Side-Chain-to-Side-Chain Cyclization Promotes the Formation of Helix-Like Secondary Structures</span>. <i>Eur. J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>2010</i></span>,  <span class="NLM_fpage">446</span>– <span class="NLM_lpage">457</span>, <span class="refDoi"> DOI: 10.1002/ejoc.200901157</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1002%2Fejoc.200901157" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2010&publication_year=2010&pages=446-457&author=M.+Scrimaauthor=A.+L.+Chevalier-Isaadauthor=P.+Roveroauthor=A.+M.+Papiniauthor=M.+Chorevauthor=A.+M.+D%E2%80%99Ursi&title=CuI-Catalyzed+Azide%E2%80%93Alkyne+Intramolecular+i-to-%28I%2B4%29+Side-Chain-to-Side-Chain+Cyclization+Promotes+the+Formation+of+Helix-Like+Secondary+Structures&doi=10.1002%2Fejoc.200901157"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1002%2Fejoc.200901157&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fejoc.200901157%26sid%3Dliteratum%253Aachs%26aulast%3DScrima%26aufirst%3DM.%26aulast%3DChevalier-Isaad%26aufirst%3DA.%2BL.%26aulast%3DRovero%26aufirst%3DP.%26aulast%3DPapini%26aufirst%3DA.%2BM.%26aulast%3DChorev%26aufirst%3DM.%26aulast%3DD%25E2%2580%2599Ursi%26aufirst%3DA.%2BM.%26atitle%3DCuI-Catalyzed%2520Azide%25E2%2580%2593Alkyne%2520Intramolecular%2520i-to-%2528I%252B4%2529%2520Side-Chain-to-Side-Chain%2520Cyclization%2520Promotes%2520the%2520Formation%2520of%2520Helix-Like%2520Secondary%2520Structures%26jtitle%3DEur.%2520J.%2520Org.%2520Chem.%26date%3D2010%26volume%3D2010%26spage%3D446%26epage%3D457%26doi%3D10.1002%2Fejoc.200901157" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kawamoto, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleska, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ran, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Design of Triazole-Stapled BCL9 α-Helical Peptides to Target the β-Catenin/B-Cell CLL/Lymphoma 9 (BCL9) Protein-Protein Interaction</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">1137</span>– <span class="NLM_lpage">1146</span>, <span class="refDoi"> DOI: 10.1021/jm201125d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm201125d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs1OjtLfL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=1137-1146&author=S.+A.+Kawamotoauthor=A.+Coleskaauthor=X.+Ranauthor=H.+Yiauthor=C.-Y.+Yangauthor=S.+Wang&title=Design+of+Triazole-Stapled+BCL9+%CE%B1-Helical+Peptides+to+Target+the+%CE%B2-Catenin%2FB-Cell+CLL%2FLymphoma+9+%28BCL9%29+Protein-Protein+Interaction&doi=10.1021%2Fjm201125d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Design of Triazole-Stapled BCL9 α-Helical Peptides to Target the β-Catenin/B-Cell CLL/lymphoma 9 (BCL9) Protein-Protein Interaction</span></div><div class="casAuthors">Kawamoto, Steven A.; Coleska, Adriana; Ran, Xu; Yi, Han; Yang, Chao-Yie; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1137-1146</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The interaction between β-catenin and B-cell CLL/lymphoma 9 (BCL9), crit. for the transcriptional activity of β-catenin, is mediated by a helical segment from BCL9 and a large binding groove in β-catenin.  Design of potent, metabolically stable BCL9 peptides represents an attractive approach to inhibit the activity of β-catenin.  In this study, we report the use of the Huisgen 1,3-dipolar cycloaddn. reaction to generate triazole-stapled BCL9 α-helical peptides.  The high efficiency and mild conditions of this 'click' reaction combined with the ease of synthesis of the necessary unnatural amino acids allows for facile synthesis of triazole-stapled peptides.  We have performed extensive optimization of this approach and identified the optimal combinations of azido and alkynyl linkers necessary for stapling BCL9 helixes.  The unsym. nature of the triazole staple also allowed the synthesis of double-stapled BCL9 peptides, which show a marked increase in helical character and an improvement in binding affinity and metabolic stability relative to wild-type and linear BCL9 peptides.  This study lays the foundation for further optimization of these triazole-stapled BCL9 peptides as potent, metabolically stable, and cell-permeable inhibitors to target the β-catenin and BCL9 interaction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqf88hHn2HxybVg90H21EOLACvtfcHk0lgBrDhaHOmGfw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs1OjtLfL&md5=d0ab7aaf4074398402ae5f8b004a481b</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1021%2Fjm201125d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm201125d%26sid%3Dliteratum%253Aachs%26aulast%3DKawamoto%26aufirst%3DS.%2BA.%26aulast%3DColeska%26aufirst%3DA.%26aulast%3DRan%26aufirst%3DX.%26aulast%3DYi%26aufirst%3DH.%26aulast%3DYang%26aufirst%3DC.-Y.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DDesign%2520of%2520Triazole-Stapled%2520BCL9%2520%25CE%25B1-Helical%2520Peptides%2520to%2520Target%2520the%2520%25CE%25B2-Catenin%252FB-Cell%2520CLL%252FLymphoma%25209%2520%2528BCL9%2529%2520Protein-Protein%2520Interaction%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D1137%26epage%3D1146%26doi%3D10.1021%2Fjm201125d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Madden, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muppidi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Q.</span></span> <span> </span><span class="NLM_article-title">Synthesis of Cell-Permeable Stapled Peptide Dual Inhibitors of the P53-Mdm2/Mdmx Interactions via Photoinduced Cycloaddition</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">1472</span>– <span class="NLM_lpage">1475</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2011.01.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1016%2Fj.bmcl.2011.01.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=21277201" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC3MXit1Gmu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2011&pages=1472-1475&author=M.+M.+Maddenauthor=A.+Muppidiauthor=Z.+Liauthor=X.+Liauthor=J.+Chenauthor=Q.+Lin&title=Synthesis+of+Cell-Permeable+Stapled+Peptide+Dual+Inhibitors+of+the+P53-Mdm2%2FMdmx+Interactions+via+Photoinduced+Cycloaddition&doi=10.1016%2Fj.bmcl.2011.01.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis of cell-permeable stapled peptide dual inhibitors of the p53-Mdm2/Mdmx interactions via photoinduced cycloaddition</span></div><div class="casAuthors">Madden, Michael M.; Muppidi, Avinash; Li, Zhenyu; Li, Xiaolong; Chen, Jiandong; Lin, Qing</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1472-1475</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The authors report the application of photoinduced 1,3-dipolar cycloaddn. reaction to "staple" a peptide dual inhibitor of the p53-Mdm2/Mdmx interactions.  A series of "stapled" peptide inhibitors was efficiently synthesized and showed excellent dual inhibitory activity in ELISA assay.  Furthermore, the pos. charged, "stapled" peptides showed enhanced cellular uptake along with modest in vivo activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIUVyYPsFaS7Vg90H21EOLACvtfcHk0lgBrDhaHOmGfw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXit1Gmu7w%253D&md5=8640cbbb835293bc90c551c083405d29</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2011.01.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2011.01.004%26sid%3Dliteratum%253Aachs%26aulast%3DMadden%26aufirst%3DM.%2BM.%26aulast%3DMuppidi%26aufirst%3DA.%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DLin%26aufirst%3DQ.%26atitle%3DSynthesis%2520of%2520Cell-Permeable%2520Stapled%2520Peptide%2520Dual%2520Inhibitors%2520of%2520the%2520P53-Mdm2%252FMdmx%2520Interactions%2520via%2520Photoinduced%2520Cycloaddition%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2011%26volume%3D21%26spage%3D1472%26epage%3D1475%26doi%3D10.1016%2Fj.bmcl.2011.01.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span> <span> </span><span class="NLM_article-title">Stapling of Unprotected Helical Peptides via Photo-Induced Intramolecular Thiol-Yne Hydrothiolation</span>. <i>Chem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">3325</span>– <span class="NLM_lpage">3330</span>, <span class="refDoi"> DOI: 10.1039/C6SC00106H</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1039%2FC6SC00106H" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=29997825" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC28XitFGjurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=3325-3330&author=Y.+Tianauthor=J.+Liauthor=H.+Zhaoauthor=X.+Zengauthor=D.+Wangauthor=Q.+Liuauthor=X.+Niuauthor=X.+Huangauthor=N.+Xuauthor=Z.+Li&title=Stapling+of+Unprotected+Helical+Peptides+via+Photo-Induced+Intramolecular+Thiol-Yne+Hydrothiolation&doi=10.1039%2FC6SC00106H"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Stapling of unprotected helical peptides via photo-induced intramolecular thiol-yne hydrothiolation</span></div><div class="casAuthors">Tian, Yuan; Li, Jingxu; Zhao, Hui; Zeng, Xiangze; Wang, Dongyuan; Liu, Qisong; Niu, Xiaogang; Huang, Xuhui; Xu, Naihan; Li, Zigang</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Science</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">3325-3330</span>CODEN:
                <span class="NLM_cas:coden">CSHCCN</span>;
        ISSN:<span class="NLM_cas:issn">2041-6520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Peptide stapling emerged as a versatile strategy to recapitulate the bioactive helical conformation of unstructured short peptides in water to improve their therapeutic properties in targeting intracellular "undruggable" targets.  Here, the authors present the development of photoinduced intramol. thiol-yne macrocyclization for rapid access to short stapled peptides with enhanced biophys. properties.  This new peptide stapling technique provides rapid access to conformationally constrained helixes with satisfying functional group tolerance.  Notably, the vinyl sulfide linkage shows distinct lipophilicity with reduced membrane toxicity compared to the corresponding all-hydrocarbon analog.  As a proof of principle, the authors constructed stabilized helixes modulating intracellular estrogen receptor (ER)-coactivator interactions with a nanomolar binding affinity, enhanced serum stability, a diffuse cellular distribution and selective cytotoxicity towards ER-pos. MCF-7 cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGopVBk7PWClZrVg90H21EOLACvtfcHk0lj8ETEgGf8xNQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitFGjurc%253D&md5=2749fb8c7b0b79210c9a74913c8b9002</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1039%2FC6SC00106H&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC6SC00106H%26sid%3Dliteratum%253Aachs%26aulast%3DTian%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DZhao%26aufirst%3DH.%26aulast%3DZeng%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DQ.%26aulast%3DNiu%26aufirst%3DX.%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DXu%26aufirst%3DN.%26aulast%3DLi%26aufirst%3DZ.%26atitle%3DStapling%2520of%2520Unprotected%2520Helical%2520Peptides%2520via%2520Photo-Induced%2520Intramolecular%2520Thiol-Yne%2520Hydrothiolation%26jtitle%3DChem.%2520Sci.%26date%3D2016%26volume%3D7%26spage%3D3325%26epage%3D3330%26doi%3D10.1039%2FC6SC00106H" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref64"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref64'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bruno, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cornillie, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nogieira, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheatham, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lim, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chou, D. H.-C.</span></span> <span> </span><span class="NLM_article-title">Application of Thiol-Yne/Thiol-Ene Reactions for Peptide and Protein Macrocyclizations</span>. <i>Chem. –Eur. J.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">7087</span>– <span class="NLM_lpage">7092</span>, <span class="refDoi"> DOI: 10.1002/chem.201700572</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1002%2Fchem.201700572" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=28345248" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC2sXms1Wks7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2017&pages=7087-7092&author=Y.+Wangauthor=B.+J.+Brunoauthor=S.+Cornillieauthor=J.+M.+Nogieiraauthor=D.+Chenauthor=T.+E.+Cheathamauthor=C.+S.+Limauthor=D.+H.-C.+Chou&title=Application+of+Thiol-Yne%2FThiol-Ene+Reactions+for+Peptide+and+Protein+Macrocyclizations&doi=10.1002%2Fchem.201700572"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Application of Thiol-yne/Thiol-ene Reactions for Peptide and Protein Macrocyclizations</span></div><div class="casAuthors">Wang, Yuanxiang; Bruno, Benjamin J.; Cornillie, Sean; Nogieira, Jason M.; Chen, Diao; Cheatham, Thomas E., III; Lim, Carol S.; Chou, Danny Hung-Chieh</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry - A European Journal</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">29</span>),
    <span class="NLM_cas:pages">7087-7092</span>CODEN:
                <span class="NLM_cas:coden">CEUJED</span>;
        ISSN:<span class="NLM_cas:issn">0947-6539</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The application of thiol-yne/thiol-ene reactions to synthesize mono- and bicyclic-stapled peptides and proteins is reported.  First, a thiol-ene-based peptide-stapling metho39713440Fd in aq. conditions was developed.  This method enabled the efficient stapling of recombinantly expressed coil-coiled proteins.  The resulting stapled protein demonstrated higher stability in its secondary structure than the unstapled version.  Furthermore, a thiol-yne coupling was performed by using an α,ω-diyne to react with two cysteine residues to synthesize a stapled peptide with two vinyl sulfide groups.  The stapled peptide could further react with another biscysteine peptide to yield a bicyclic stapled peptide with enhanced properties.  For example, the cell permeability of a stapled peptide was further increased by appending an oligoarginine cell-penetrating peptide.  The robustness and versatility of thiol-yne/thiol-ene reactions that can be applied to both synthetic and expressed peptides and proteins were demonstrated.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqI0I-p3I4MebVg90H21EOLACvtfcHk0lj8ETEgGf8xNQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXms1Wks7k%253D&md5=019af66e3a684454a84cf0115b4c2a24</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1002%2Fchem.201700572&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fchem.201700572%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DY.%26aulast%3DBruno%26aufirst%3DB.%2BJ.%26aulast%3DCornillie%26aufirst%3DS.%26aulast%3DNogieira%26aufirst%3DJ.%2BM.%26aulast%3DChen%26aufirst%3DD.%26aulast%3DCheatham%26aufirst%3DT.%2BE.%26aulast%3DLim%26aufirst%3DC.%2BS.%26aulast%3DChou%26aufirst%3DD.%2BH.-C.%26atitle%3DApplication%2520of%2520Thiol-Yne%252FThiol-Ene%2520Reactions%2520for%2520Peptide%2520and%2520Protein%2520Macrocyclizations%26jtitle%3DChem.%2520%25E2%2580%2593Eur.%2520J.%26date%3D2017%26volume%3D23%26spage%3D7087%26epage%3D7092%26doi%3D10.1002%2Fchem.201700572" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref65"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref65'); return false;" data-citation="" class="refNumLink">65</a></strong><div class="NLM_citation" id="cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chou, D. H.-C.</span></span> <span> </span><span class="NLM_article-title">A Thiol-Ene Coupling Approach to Native Peptide Stapling and Macrocyclization</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">10931</span>– <span class="NLM_lpage">10934</span>, <span class="refDoi"> DOI: 10.1002/anie.201503975</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1002%2Fanie.201503975" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=26189498" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFyqtb%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2015&pages=10931-10934&author=Y.+Wangauthor=D.+H.-C.+Chou&title=A+Thiol-Ene+Coupling+Approach+to+Native+Peptide+Stapling+and+Macrocyclization&doi=10.1002%2Fanie.201503975"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">A thiol-ene coupling approach to native peptide stapling and macrocyclization</span></div><div class="casAuthors">Wang, Yuanxiang; Chou, Danny Hung-Chieh</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">37</span>),
    <span class="NLM_cas:pages">10931-10934</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">We report the discovery of a peptide stapling and macrocyclization method using thiol-ene reactions between two cysteine residues and an α,ω-diene in high yields.  This new approach enabled us to selectively modify cysteine residues in native, unprotected peptides with a variety of stapling modifications for helix stabilization or general macrocyclization.  We synthesized stapled Axin mimetic analogs and demonstrated increased alpha helicity upon peptide stapling.  We then synthesized stapled p53 mimetic analogs using pure hydrocarbon linkers and demonstrated their abilities to block the p53-MDM2 interaction and selectively kill p53 wild-type colorectal carcinoma HCT-116 cells but not p53 null cells.  In summary, we demonstrated a robust and versatile peptide stapling method that could be potentially applied to both synthetic and expressed peptides.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdtZz8o6RMvbVg90H21EOLACvtfcHk0lj8ETEgGf8xNQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFyqtb%252FN&md5=2ea0ca9a99ab25c0cb3a1000b4cf618c</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1002%2Fanie.201503975&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201503975%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DY.%26aulast%3DChou%26aufirst%3DD.%2BH.-C.%26atitle%3DA%2520Thiol-Ene%2520Coupling%2520Approach%2520to%2520Native%2520Peptide%2520Stapling%2520and%2520Macrocyclization%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2015%26volume%3D54%26spage%3D10931%26epage%3D10934%26doi%3D10.1002%2Fanie.201503975" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref66"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref66'); return false;" data-citation="" class="refNumLink">66</a></strong><div class="NLM_citation" id="cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span> <span> </span><span class="NLM_article-title">Constructing Thioether-Tethered Cyclic Peptides via on-Resin Intra-Molecular Thiol-Ene Reaction</span>. <i>J. Pept. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">540</span>– <span class="NLM_lpage">544</span>, <span class="refDoi"> DOI: 10.1002/psc.2902</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1002%2Fpsc.2902" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=27378284" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFSiurzK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2016&pages=540-544&author=B.+Zhaoauthor=Q.+Zhangauthor=Z.+Li&title=Constructing+Thioether-Tethered+Cyclic+Peptides+via+on-Resin+Intra-Molecular+Thiol-Ene+Reaction&doi=10.1002%2Fpsc.2902"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">Constructing thioether-tethered cyclic peptides via on-resin intra-molecular thiol-ene reaction</span></div><div class="casAuthors">Zhao, Bingchuan; Zhang, Qingzhou; Li, Zigang</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Peptide Science</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">540-544</span>CODEN:
                <span class="NLM_cas:coden">JPSIEI</span>;
        ISSN:<span class="NLM_cas:issn">1075-2617</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons Ltd.</span>)
        </div><div class="casAbstract">Thiol-ene reactions have been used in a variety of applications that mostly involve an inter-mol. pathway.  Herein, we report a facile method to construct thioether-tethered cyclic peptides via an intra-mol. thiol-ene reaction.  This reaction is efficient, selective, and has good residue compatibility.  Short peptides with thioether tethers were constructed and were used to construct longer cyclic peptides.  This synthetic method may be useful for constructing bioactive peptides.  Copyright © 2016 European Peptide Society and John Wiley & Sons, Ltd.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMcNgwald7V7Vg90H21EOLACvtfcHk0lj8ETEgGf8xNQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFSiurzK&md5=084b999debfc1cd056ebeeb15452f448</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1002%2Fpsc.2902&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpsc.2902%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DB.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DLi%26aufirst%3DZ.%26atitle%3DConstructing%2520Thioether-Tethered%2520Cyclic%2520Peptides%2520via%2520on-Resin%2520Intra-Molecular%2520Thiol-Ene%2520Reaction%26jtitle%3DJ.%2520Pept.%2520Sci.%26date%3D2016%26volume%3D22%26spage%3D540%26epage%3D544%26doi%3D10.1002%2Fpsc.2902" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref67"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref67'); return false;" data-citation="" class="refNumLink">67</a></strong><div class="NLM_citation" id="cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fadzen, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolfe, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, C.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiocca, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawler, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pentelute, B. L.</span></span> <span> </span><span class="NLM_article-title">Perfluoroarene-Based Peptide Macrocycles to Enhance Penetration Across the Blood-Brain Barrier</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>139</i></span>,  <span class="NLM_fpage">15628</span>– <span class="NLM_lpage">15631</span>, <span class="refDoi"> DOI: 10.1021/jacs.7b09790</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.7b09790" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1amtL7O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=139&publication_year=2017&pages=15628-15631&author=C.+M.+Fadzenauthor=J.+M.+Wolfeauthor=C.-F.+Choauthor=E.+A.+Chioccaauthor=S.+E.+Lawlerauthor=B.+L.+Pentelute&title=Perfluoroarene-Based+Peptide+Macrocycles+to+Enhance+Penetration+Across+the+Blood-Brain+Barrier&doi=10.1021%2Fjacs.7b09790"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">Perfluoroarene-Based Peptide Macrocycles To Enhance Penetration across the Blood-Brain Barrier</span></div><div class="casAuthors">Fadzen, Colin M.; Wolfe, Justin M.; Cho, Choi-Fong; Chiocca, E. Antonio; Lawler, Sean E.; Pentelute, Bradley L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">139</span>
        (<span class="NLM_cas:issue">44</span>),
    <span class="NLM_cas:pages">15628-15631</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Here we describe the utility of peptide macrocyclization through perfluoroaryl-cysteine SNAr chem. in facilitating crossing the blood-brain barrier.  Multiple macrocyclic analogs of the peptide transportan-10 were investigated that displayed increased uptake in two different cell lines and improved proteolytic stability.  One of these analogs (M13) exhibited substantially increased delivery across a cellular spheroid model of the blood-brain barrier.  Through in vivo and ex vivo imaging of mouse brains, we demonstrate that this perfluoroarene-based macrocycle of TP10 exhibits increased penetration of the brain parenchyma following i.v. administration in mice.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplSoOwYi8wu7Vg90H21EOLACvtfcHk0lgUFg03NpGtqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1amtL7O&md5=9b5d8d1e53c7b94b3e0ff88096d2b96e</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1021%2Fjacs.7b09790&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.7b09790%26sid%3Dliteratum%253Aachs%26aulast%3DFadzen%26aufirst%3DC.%2BM.%26aulast%3DWolfe%26aufirst%3DJ.%2BM.%26aulast%3DCho%26aufirst%3DC.-F.%26aulast%3DChiocca%26aufirst%3DE.%2BA.%26aulast%3DLawler%26aufirst%3DS.%2BE.%26aulast%3DPentelute%26aufirst%3DB.%2BL.%26atitle%3DPerfluoroarene-Based%2520Peptide%2520Macrocycles%2520to%2520Enhance%2520Penetration%2520Across%2520the%2520Blood-Brain%2520Barrier%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2017%26volume%3D139%26spage%3D15628%26epage%3D15631%26doi%3D10.1021%2Fjacs.7b09790" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref68"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref68'); return false;" data-citation="" class="refNumLink">68</a></strong><div class="NLM_citation" id="cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jo, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meinhardt, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kulkarni, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Low, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeGrado, W. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenbaum, D. C.</span></span> <span> </span><span class="NLM_article-title">Development of α-Helical Calpain Probes by Mimicking a Natural Protein-Protein Interaction</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>134</i></span>,  <span class="NLM_fpage">17704</span>– <span class="NLM_lpage">17713</span>, <span class="refDoi"> DOI: 10.1021/ja307599z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja307599z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhtl2lsrfK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=134&publication_year=2012&pages=17704-17713&author=H.+Joauthor=N.+Meinhardtauthor=Y.+Wuauthor=S.+Kulkarniauthor=X.+Huauthor=K.+E.+Lowauthor=P.+L.+Daviesauthor=W.+F.+DeGradoauthor=D.+C.+Greenbaum&title=Development+of+%CE%B1-Helical+Calpain+Probes+by+Mimicking+a+Natural+Protein-Protein+Interaction&doi=10.1021%2Fja307599z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">Development of α-Helical Calpain Probes by Mimicking a Natural Protein-Protein Interaction</span></div><div class="casAuthors">Jo, Hyunil; Meinhardt, Nataline; Wu, Yibing; Kulkarni, Swapnil; Hu, Xiaozhen; Low, Kristin E.; Davies, Peter L.; DeGrado, William F.; Greenbaum, Doron C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">134</span>
        (<span class="NLM_cas:issue">42</span>),
    <span class="NLM_cas:pages">17704-17713</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A highly specific inhibitor of calpain was designed by mimicking a natural protein-protein interaction between calpain and its endogenous inhibitor calpastatin.  To enable this goal a new method of stabilizing an α-helix in a small peptide was established by screening 24 com. available cross-linkers for successful cysteine alkylation in a model peptide sequence.  The effects of crosslinking on the α-helicity of selected peptides were examd. by CD and NMR spectroscopy, and revealed structurally rigid cross-linkers to be the best at stabilizing α-helixes.  This strategy was applied to the design of inhibitors of calpain that are based on calpastatin, an intrinsically unstable polypeptide that becomes structured upon binding to the enzyme.  A two-turn α-helix that binds proximal to the active site cleft was stabilized, resulting in a potent and selective inhibitor for calpain.  The utility of this inhibitor was further expanded by developing irreversible calpain family activity-based probes (ABPs), which retained the specificity of the stabilized helical inhibitor.  It is believed that the inhibitor and ABPs will be useful for future investigation of calpains, while the crosslinking technique will enable exploration of other protein-protein interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpigseq6FFUU7Vg90H21EOLACvtfcHk0lgUFg03NpGtqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhtl2lsrfK&md5=2824ee6e61d4bb0e4202696fc57f1fa0</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1021%2Fja307599z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja307599z%26sid%3Dliteratum%253Aachs%26aulast%3DJo%26aufirst%3DH.%26aulast%3DMeinhardt%26aufirst%3DN.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DKulkarni%26aufirst%3DS.%26aulast%3DHu%26aufirst%3DX.%26aulast%3DLow%26aufirst%3DK.%2BE.%26aulast%3DDavies%26aufirst%3DP.%2BL.%26aulast%3DDeGrado%26aufirst%3DW.%2BF.%26aulast%3DGreenbaum%26aufirst%3DD.%2BC.%26atitle%3DDevelopment%2520of%2520%25CE%25B1-Helical%2520Calpain%2520Probes%2520by%2520Mimicking%2520a%2520Natural%2520Protein-Protein%2520Interaction%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2012%26volume%3D134%26spage%3D17704%26epage%3D17713%26doi%3D10.1021%2Fja307599z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref69"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref69'); return false;" data-citation="" class="refNumLink">69</a></strong><div class="NLM_citation" id="cit69"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Peraro, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siegert, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kritzer, J. A.</span></span> <span> </span><span class="NLM_article-title">Conformational Restriction of Peptides Using Dithiol Bis-Alkylation</span>. <i>Methods Enzymol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>580</i></span>,  <span class="NLM_fpage">303</span>– <span class="NLM_lpage">332</span>, <span class="refDoi"> DOI: 10.1016/bs.mie.2016.05.035</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1016%2Fbs.mie.2016.05.035" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=27586339" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A280%3ADC%252BC2szntVGktw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=580&publication_year=2016&pages=303-332&author=L.+Peraroauthor=T.+R.+Siegertauthor=J.+A.+Kritzer&title=Conformational+Restriction+of+Peptides+Using+Dithiol+Bis-Alkylation&doi=10.1016%2Fbs.mie.2016.05.035"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">Conformational Restriction of Peptides Using Dithiol Bis-Alkylation</span></div><div class="casAuthors">Peraro L; Siegert T R; Kritzer J A</div><div class="citationInfo"><span class="NLM_cas:title">Methods in enzymology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">580</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">303-32</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Macrocyclic peptides are highly promising as inhibitors of protein-protein interactions.  While many bond-forming reactions can be used to make cyclic peptides, most have limitations that make this chemical space challenging to access.  Recently, a variety of cysteine alkylation reactions have been used in rational design and library approaches for cyclic peptide discovery and development.  We and others have found that this chemistry is versatile and robust enough to produce a large variety of conformationally constrained cyclic peptides.  In this chapter, we describe applications, methods, mechanistic insights, and troubleshooting for dithiol bis-alkylation reactions for the production of cyclic peptides.  This method for efficient solution-phase macrocyclization is highly useful for the rapid production and screening of loop-based inhibitors of protein-protein interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRdlYGRAk0ein6oPH0mYWL1fW6udTcc2eZmD74-maP3sbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2szntVGktw%253D%253D&md5=5afc27d7b85b89daa26f140a2d3e47d1</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1016%2Fbs.mie.2016.05.035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fbs.mie.2016.05.035%26sid%3Dliteratum%253Aachs%26aulast%3DPeraro%26aufirst%3DL.%26aulast%3DSiegert%26aufirst%3DT.%2BR.%26aulast%3DKritzer%26aufirst%3DJ.%2BA.%26atitle%3DConformational%2520Restriction%2520of%2520Peptides%2520Using%2520Dithiol%2520Bis-Alkylation%26jtitle%3DMethods%2520Enzymol.%26date%3D2016%26volume%3D580%26spage%3D303%26epage%3D332%26doi%3D10.1016%2Fbs.mie.2016.05.035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref70"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref70'); return false;" data-citation="" class="refNumLink">70</a></strong><div class="NLM_citation" id="cit70"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rojas, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pentelute, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buchwald, S. L.</span></span> <span> </span><span class="NLM_article-title">Water-Soluble Palladium Reagents for Cysteine S-Arylation under Ambient Aqueous Conditions</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">4263</span>– <span class="NLM_lpage">4266</span>, <span class="refDoi"> DOI: 10.1021/acs.orglett.7b01911</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.orglett.7b01911" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1OlsbjN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2017&pages=4263-4266&author=A.+J.+Rojasauthor=B.+L.+Penteluteauthor=S.+L.+Buchwald&title=Water-Soluble+Palladium+Reagents+for+Cysteine+S-Arylation+under+Ambient+Aqueous+Conditions&doi=10.1021%2Facs.orglett.7b01911"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Water-Soluble Palladium Reagents for Cysteine S-Arylation under Ambient Aqueous Conditions</span></div><div class="casAuthors">Rojas, Anthony J.; Pentelute, Bradley L.; Buchwald, Stephen L.</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">4263-4266</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7052</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Authors report the use of a sulfonated biarylphosphine ligand (sSPhos) to promote the chemoselective modification of cysteine contg. proteins and peptides with palladium reagents in aq. medium.  The use of sSPhos allowed for the isolation of several air-stable and water-sol. mono- and bis-palladium reagents, which were used in an improved protocol for the rapid S-arylation of cysteines under benign and physiol. relevant conditions.  The cosolvent-free aq. conditions were applied to the conjugation of a variety of biomols. with affinity tags, heterocycles, fluorophores, and functional handles.  Addnl., bis-palladium reagents were used to perform macrocyclization of peptides bearing two cysteine residues.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpAr4KwsDJ5ZbVg90H21EOLACvtfcHk0lhcQJMywfnteg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1OlsbjN&md5=0f4c8407d151c034579097b745d81f10</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1021%2Facs.orglett.7b01911&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.orglett.7b01911%26sid%3Dliteratum%253Aachs%26aulast%3DRojas%26aufirst%3DA.%2BJ.%26aulast%3DPentelute%26aufirst%3DB.%2BL.%26aulast%3DBuchwald%26aufirst%3DS.%2BL.%26atitle%3DWater-Soluble%2520Palladium%2520Reagents%2520for%2520Cysteine%2520S-Arylation%2520under%2520Ambient%2520Aqueous%2520Conditions%26jtitle%3DOrg.%2520Lett.%26date%3D2017%26volume%3D19%26spage%3D4263%26epage%3D4266%26doi%3D10.1021%2Facs.orglett.7b01911" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref71"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref71'); return false;" data-citation="" class="refNumLink">71</a></strong><div class="NLM_citation" id="cit71"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rojas, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vinogradova, E. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buchwald, N. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reilly, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pentelute, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buchwald, S. L.</span></span> <span> </span><span class="NLM_article-title">Divergent Unprotected Peptide Macrocyclisation by Palladium-Mediated Cysteine Arylation</span>. <i>Chem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">4257</span>– <span class="NLM_lpage">4263</span>, <span class="refDoi"> DOI: 10.1039/C6SC05454D</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1039%2FC6SC05454D" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=29081961" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkvVOgtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=4257-4263&author=A.+J.+Rojasauthor=C.+Zhangauthor=E.+V.+Vinogradovaauthor=N.+H.+Buchwaldauthor=J.+Reillyauthor=B.+L.+Penteluteauthor=S.+L.+Buchwald&title=Divergent+Unprotected+Peptide+Macrocyclisation+by+Palladium-Mediated+Cysteine+Arylation&doi=10.1039%2FC6SC05454D"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">Divergent unprotected peptide macrocyclisation by palladium-mediated cysteine arylation</span></div><div class="casAuthors">Rojas, Anthony J.; Zhang, Chi; Vinogradova, Ekaterina V.; Buchwald, Nathan H.; Reilly, John; Pentelute, Bradley L.; Buchwald, Stephen L.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Science</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">4257-4263</span>CODEN:
                <span class="NLM_cas:coden">CSHCCN</span>;
        ISSN:<span class="NLM_cas:issn">2041-6520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Macrocyclic peptides are important therapeutic candidates due to their improved physicochem. properties in comparison to their linear counterparts.  Here we detail a method for a divergent macrocyclisation of unprotected peptides by crosslinking two cysteine residues with bis-palladium organometallic reagents.  These synthetic intermediates are prepd. in a single step from com. available aryl bis-halides.  Two bioactive linear peptides with cysteine residues at i, i + 4 and i, i + 7 positions, resp., were cyclized to introduce a diverse array of aryl and bi-aryl linkers.  These two series of macrocyclic peptides displayed similar linker-dependent lipophilicity, phospholipid affinity, and unique vol. of distributions.  Addnl., one of the bioactive peptides showed target binding affinity that was predominantly affected by the length of the linker.  Collectively, this divergent strategy allowed rapid and convenient access to various aryl linkers, enabling the systematic evaluation of the effect of appending unit on the medicinal properties of macrocyclic peptides.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr9-XconY4mOrVg90H21EOLACvtfcHk0lhcQJMywfnteg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkvVOgtbk%253D&md5=9eaa28f51178fa528e5bf959b0c16222</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1039%2FC6SC05454D&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC6SC05454D%26sid%3Dliteratum%253Aachs%26aulast%3DRojas%26aufirst%3DA.%2BJ.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DVinogradova%26aufirst%3DE.%2BV.%26aulast%3DBuchwald%26aufirst%3DN.%2BH.%26aulast%3DReilly%26aufirst%3DJ.%26aulast%3DPentelute%26aufirst%3DB.%2BL.%26aulast%3DBuchwald%26aufirst%3DS.%2BL.%26atitle%3DDivergent%2520Unprotected%2520Peptide%2520Macrocyclisation%2520by%2520Palladium-Mediated%2520Cysteine%2520Arylation%26jtitle%3DChem.%2520Sci.%26date%3D2017%26volume%3D8%26spage%3D4257%26epage%3D4263%26doi%3D10.1039%2FC6SC05454D" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref72"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref72'); return false;" data-citation="" class="refNumLink">72</a></strong><div class="NLM_citation" id="cit72"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Spokoyny, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ling, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Y.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pentelute, B. L.</span></span> <span> </span><span class="NLM_article-title">A Perfluoroaryl-Cysteine S(N)Ar Chemistry Approach to Unprotected Peptide Stapling</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>135</i></span>,  <span class="NLM_fpage">5946</span>– <span class="NLM_lpage">5949</span>, <span class="refDoi"> DOI: 10.1021/ja400119t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja400119t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlsV2gtrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=135&publication_year=2013&pages=5946-5949&author=A.+M.+Spokoynyauthor=Y.+Zouauthor=J.+J.+Lingauthor=H.+Yuauthor=Y.-S.+Linauthor=B.+L.+Pentelute&title=A+Perfluoroaryl-Cysteine+S%28N%29Ar+Chemistry+Approach+to+Unprotected+Peptide+Stapling&doi=10.1021%2Fja400119t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">A Perfluoroaryl-Cysteine SNAr Chemistry Approach to Unprotected Peptide Stapling</span></div><div class="casAuthors">Spokoyny, Alexander M.; Zou, Yekui; Ling, Jingjing J.; Yu, Hongtao; Lin, Yu-Shan; Pentelute, Bradley L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">135</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">5946-5949</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The authors report the discovery of a facile transformation between perfluoroarom. mols. and a cysteine thiolate, which is arylated at room temp.  This new approach enabled the authors to selectively modify cysteine residues in unprotected peptides, providing access to variants contg. rigid perfluoroarom. staples.  This stapling modification performed on a peptide sequence designed to bind the C-terminal domain of an HIV-1 (C-CA) showed enhancement in binding, cell permeability, and proteolytic stability properties, as compared to the unstapled analog.  Importantly, chem. stability of the formed staples allowed the authors to use this motif in the native chem. ligation-mediated synthesis of a small protein affibody that is capable of binding the human epidermal growth factor 2 receptor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqw-oA_BVUKOrVg90H21EOLACvtfcHk0lhcQJMywfnteg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXlsV2gtrc%253D&md5=126ac191059b0703757ef7838d90e183</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1021%2Fja400119t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja400119t%26sid%3Dliteratum%253Aachs%26aulast%3DSpokoyny%26aufirst%3DA.%2BM.%26aulast%3DZou%26aufirst%3DY.%26aulast%3DLing%26aufirst%3DJ.%2BJ.%26aulast%3DYu%26aufirst%3DH.%26aulast%3DLin%26aufirst%3DY.-S.%26aulast%3DPentelute%26aufirst%3DB.%2BL.%26atitle%3DA%2520Perfluoroaryl-Cysteine%2520S%2528N%2529Ar%2520Chemistry%2520Approach%2520to%2520Unprotected%2520Peptide%2520Stapling%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2013%26volume%3D135%26spage%3D5946%26epage%3D5949%26doi%3D10.1021%2Fja400119t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref73"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref73'); return false;" data-citation="" class="refNumLink">73</a></strong><div class="NLM_citation" id="cit73"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vinogradova, E. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spokoyny, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pentelute, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buchwald, S. L.</span></span> <span> </span><span class="NLM_article-title">Organometallic Palladium Reagents for Cysteine Bioconjugation</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>526</i></span>,  <span class="NLM_fpage">687</span>– <span class="NLM_lpage">691</span>, <span class="refDoi"> DOI: 10.1038/nature15739</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1038%2Fnature15739" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=26511579" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslCnu7zK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=526&publication_year=2015&pages=687-691&author=E.+V.+Vinogradovaauthor=C.+Zhangauthor=A.+M.+Spokoynyauthor=B.+L.+Penteluteauthor=S.+L.+Buchwald&title=Organometallic+Palladium+Reagents+for+Cysteine+Bioconjugation&doi=10.1038%2Fnature15739"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">Organometallic palladium reagents for cysteine bioconjugation</span></div><div class="casAuthors">Vinogradova, Ekaterina V.; Zhang, Chi; Spokoyny, Alexander M.; Pentelute, Bradley L.; Buchwald, Stephen L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">526</span>
        (<span class="NLM_cas:issue">7575</span>),
    <span class="NLM_cas:pages">687-691</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Peptides and proteins, contg. cysteine amino acid, were labeled by conjugation of the SH-group with arylpalladium 2-biphenylylphosphine hemilabile reagent, causing arylation the mercapto-group.  Reactions based on transition metals have found wide use in org. synthesis, in particular for the functionalization of small mols.  However, there are very few reports of using transition-metal-based reactions to modify complex biomols., which is due to the need for stringent reaction conditions (for example, aq. media, low temp. and mild pH) and the existence of multiple reactive functional groups found in biomols.  Here we report that palladium(II) complexes can be used for efficient and highly selective cysteine conjugation (bioconjugation) reactions that are rapid and robust under a range of bio-compatible reaction conditions.  The straightforward synthesis of the palladium reagents from diverse and easily accessible aryl halide and trifluoromethanesulfonate precursors makes the method highly practical, providing access to a large structural space for protein modification.  The resulting aryl bioconjugates are stable towards acids, bases, oxidants and external thiol nucleophiles.  The broad utility of the bioconjugation platform was further corroborated by the synthesis of new classes of stapled peptides and antibody-drug conjugates.  These palladium complexes show potential as benchtop reagents for diverse bioconjugation applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLRNU9i0UtEbVg90H21EOLACvtfcHk0lhcQJMywfnteg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslCnu7zK&md5=8fbb382c94600af312bb007d31b63056</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1038%2Fnature15739&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature15739%26sid%3Dliteratum%253Aachs%26aulast%3DVinogradova%26aufirst%3DE.%2BV.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DSpokoyny%26aufirst%3DA.%2BM.%26aulast%3DPentelute%26aufirst%3DB.%2BL.%26aulast%3DBuchwald%26aufirst%3DS.%2BL.%26atitle%3DOrganometallic%2520Palladium%2520Reagents%2520for%2520Cysteine%2520Bioconjugation%26jtitle%3DNature%26date%3D2015%26volume%3D526%26spage%3D687%26epage%3D691%26doi%3D10.1038%2Fnature15739" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref74"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref74'); return false;" data-citation="" class="refNumLink">74</a></strong><div class="NLM_citation" id="cit74"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morales-Sanfrutos, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Angelini, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cutting, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heinis, C.</span></span> <span> </span><span class="NLM_article-title">Structurally Diverse Cyclisation Linkers Impose Different Backbone Conformations in Bicyclic Peptides</span>. <i>ChemBioChem</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">1032</span>– <span class="NLM_lpage">1038</span>, <span class="refDoi"> DOI: 10.1002/cbic.201200049</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1002%2Fcbic.201200049" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=22492661" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC38Xlt1Kltb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2012&pages=1032-1038&author=S.+Chenauthor=J.+Morales-Sanfrutosauthor=A.+Angeliniauthor=B.+Cuttingauthor=C.+Heinis&title=Structurally+Diverse+Cyclisation+Linkers+Impose+Different+Backbone+Conformations+in+Bicyclic+Peptides&doi=10.1002%2Fcbic.201200049"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">Structurally Diverse Cyclisation Linkers Impose Different Backbone Conformations in Bicyclic Peptides</span></div><div class="casAuthors">Chen, Shiyu; Morales-Sanfrutos, Julia; Angelini, Alessandro; Cutting, Brian; Heinis, Christian</div><div class="citationInfo"><span class="NLM_cas:title">ChemBioChem</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1032-1038</span>CODEN:
                <span class="NLM_cas:coden">CBCHFX</span>;
        ISSN:<span class="NLM_cas:issn">1439-4227</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Combinatorial libraries of structurally diverse peptide macrocycles offer a rich source for the development of high-affinity ligands to targets of interest.  In this work we have developed linkers for the generation of genetically encoded bicyclic peptides and tested whether the peptides cyclised by them have significant variations in their backbone conformations.  Two new cyclisation reagents, each contg. three thiol-reactive groups, efficiently and selectively cyclised linear peptides contg. three cysteine moieties.  When the mesitylene linker of the bicyclic peptide PK15, a potent inhibitor of plasma kallikrein (Ki=2 nM), was replaced by the new linkers, its inhibitory activity dropped by a factor of more than 1000, suggesting that the linkers impose different conformations on the peptide.  Indeed, structural anal. by soln.-state NMR revealed different NOE constraints in the three bicyclic peptides, indicating that these relatively small linkers at the centers of bicyclic peptide structures significantly influence the conformations of the peptides.  These results demonstrate the prominent structural role of linkers in peptide macrocycles and suggest that application of different cyclisation linkers in a combinatorial fashion could be an attractive means to generate topol. diverse macrocycle libraries.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphdCp8KV5HeLVg90H21EOLACvtfcHk0liwd9HhfBCAew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xlt1Kltb4%253D&md5=ddb3f7e338a22ea1a1e3ced71624781d</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1002%2Fcbic.201200049&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbic.201200049%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DS.%26aulast%3DMorales-Sanfrutos%26aufirst%3DJ.%26aulast%3DAngelini%26aufirst%3DA.%26aulast%3DCutting%26aufirst%3DB.%26aulast%3DHeinis%26aufirst%3DC.%26atitle%3DStructurally%2520Diverse%2520Cyclisation%2520Linkers%2520Impose%2520Different%2520Backbone%2520Conformations%2520in%2520Bicyclic%2520Peptides%26jtitle%3DChemBioChem%26date%3D2012%26volume%3D13%26spage%3D1032%26epage%3D1038%26doi%3D10.1002%2Fcbic.201200049" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref75"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref75'); return false;" data-citation="" class="refNumLink">75</a></strong><div class="NLM_citation" id="cit75"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Timmerman, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beld, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Puijk, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meloen, R. H.</span></span> <span> </span><span class="NLM_article-title">Rapid and Quantitative Cyclization of Multiple Peptide Loops onto Synthetic Scaffolds for Structural Mimicry of Protein Surfaces</span>. <i>ChemBioChem</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">821</span>– <span class="NLM_lpage">824</span>, <span class="refDoi"> DOI: 10.1002/cbic.200400374</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1002%2Fcbic.200400374" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=15812852" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BD2MXksFajsL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2005&pages=821-824&author=P.+Timmermanauthor=J.+Beldauthor=W.+C.+Puijkauthor=R.+H.+Meloen&title=Rapid+and+Quantitative+Cyclization+of+Multiple+Peptide+Loops+onto+Synthetic+Scaffolds+for+Structural+Mimicry+of+Protein+Surfaces&doi=10.1002%2Fcbic.200400374"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">Rapid and quantitative cyclization of multiple peptide loops onto synthetic scaffolds for structural mimicry of protein surfaces</span></div><div class="casAuthors">Timmerman, Peter; Beld, Joris; Puijk, Wouter C.; Meloen, Rob H.</div><div class="citationInfo"><span class="NLM_cas:title">ChemBioChem</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">821-824</span>CODEN:
                <span class="NLM_cas:coden">CBCHFX</span>;
        ISSN:<span class="NLM_cas:issn">1439-4227</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">This work describes cyclization reaction of cysteine-contg. linear peptides onto bromomethylated arom. scaffolds without requiring side-chain protections.  The reaction is extremely fast and clean and runs to completion in less than 15 min at room temp.  The cyclic peptides are good mimics of protein surfaces.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqfSQnHmkXl_LVg90H21EOLACvtfcHk0liwd9HhfBCAew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXksFajsL0%253D&md5=52a1d3bb0bc8a483a683cdb900525c0f</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1002%2Fcbic.200400374&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbic.200400374%26sid%3Dliteratum%253Aachs%26aulast%3DTimmerman%26aufirst%3DP.%26aulast%3DBeld%26aufirst%3DJ.%26aulast%3DPuijk%26aufirst%3DW.%2BC.%26aulast%3DMeloen%26aufirst%3DR.%2BH.%26atitle%3DRapid%2520and%2520Quantitative%2520Cyclization%2520of%2520Multiple%2520Peptide%2520Loops%2520onto%2520Synthetic%2520Scaffolds%2520for%2520Structural%2520Mimicry%2520of%2520Protein%2520Surfaces%26jtitle%3DChemBioChem%26date%3D2005%26volume%3D6%26spage%3D821%26epage%3D824%26doi%3D10.1002%2Fcbic.200400374" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref76"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref76'); return false;" data-citation="" class="refNumLink">76</a></strong><div class="NLM_citation" id="cit76"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Diderich, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertoldo, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dessen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khan, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pizzitola, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Held, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huelsken, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heinis, C.</span></span> <span> </span><span class="NLM_article-title">Phage Selection of Chemically Stabilized α-Helical Peptide Ligands</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">1422</span>– <span class="NLM_lpage">1427</span>, <span class="refDoi"> DOI: 10.1021/acschembio.5b00963</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b00963" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC28XjsVagtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=1422-1427&author=P.+Diderichauthor=D.+Bertoldoauthor=P.+Dessenauthor=M.+M.+Khanauthor=I.+Pizzitolaauthor=W.+Heldauthor=J.+Huelskenauthor=C.+Heinis&title=Phage+Selection+of+Chemically+Stabilized+%CE%B1-Helical+Peptide+Ligands&doi=10.1021%2Facschembio.5b00963"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit76R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">Phage Selection of Chemically Stabilized α-Helical Peptide Ligands</span></div><div class="casAuthors">Diderich, Philippe; Bertoldo, Davide; Dessen, Pierre; Khan, Maola M.; Pizzitola, Irene; Held, Werner; Huelsken, Joerg; Heinis, Christian</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1422-1427</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Short α-helical peptides stabilized by linkages between constituent amino acids offer an attractive format for ligand development.  In recent years, a range of excellent ligands based on stabilized α-helixes were generated by rational design using α-helical peptides of natural proteins as templates.  Herein, we developed a method to engineer chem. stabilized α-helical ligands in a combinatorial fashion.  In brief, peptides contg. cysteines in position i and i + 4 are genetically encoded by phage display, the cysteines are modified with chem. bridges to impose α-helical conformations, and binders are isolated by affinity selection.  We applied the strategy to affinity mature an α-helical peptide binding β-catenin.  We succeeded in developing ligands with Kd's as low as 5.2 nM, having >200-fold improved affinity.  The strategy is generally applicable for affinity maturation of any α-helical peptide.  Compared to hydrocarbon stapled peptides, the herein evolved thioether-bridged peptide ligands can be synthesized more easily, as no unnatural amino acids are required and the cyclization reaction is more efficient and yields no stereoisomers.  A further advantage of the thioether-bridged peptide ligands is that they can be expressed recombinantly as fusion proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXjSfa_cgePrVg90H21EOLACvtfcHk0liwd9HhfBCAew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XjsVagtr8%253D&md5=a25d24d4733a0d89f4797c5313ef1870</span></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b00963&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b00963%26sid%3Dliteratum%253Aachs%26aulast%3DDiderich%26aufirst%3DP.%26aulast%3DBertoldo%26aufirst%3DD.%26aulast%3DDessen%26aufirst%3DP.%26aulast%3DKhan%26aufirst%3DM.%2BM.%26aulast%3DPizzitola%26aufirst%3DI.%26aulast%3DHeld%26aufirst%3DW.%26aulast%3DHuelsken%26aufirst%3DJ.%26aulast%3DHeinis%26aufirst%3DC.%26atitle%3DPhage%2520Selection%2520of%2520Chemically%2520Stabilized%2520%25CE%25B1-Helical%2520Peptide%2520Ligands%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2016%26volume%3D11%26spage%3D1422%26epage%3D1427%26doi%3D10.1021%2Facschembio.5b00963" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref77"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref77'); return false;" data-citation="" class="refNumLink">77</a></strong><div class="NLM_citation" id="cit77"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kale, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villequey, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong, X.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zorzi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deyle, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heinis, C.</span></span> <span> </span><span class="NLM_article-title">Cyclization of Peptides with Two Chemical Bridges Affords Large Scaffold Diversities</span>. <i>Nat. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">715</span>– <span class="NLM_lpage">723</span>, <span class="refDoi"> DOI: 10.1038/s41557-018-0042-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1038%2Fs41557-018-0042-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=29713035" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC1cXovVCkurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2018&pages=715-723&author=S.+S.+Kaleauthor=C.+Villequeyauthor=X.-D.+Kongauthor=A.+Zorziauthor=K.+Deyleauthor=C.+Heinis&title=Cyclization+of+Peptides+with+Two+Chemical+Bridges+Affords+Large+Scaffold+Diversities&doi=10.1038%2Fs41557-018-0042-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit77R"><div class="casContent"><span class="casTitleNuber">77</span><div class="casTitle"><span class="NLM_cas:atitle">Cyclization of peptides with two chemical bridges affords large scaffold diversities</span></div><div class="casAuthors">Kale, Sangram S.; Villequey, Camille; Kong, Xu-Dong; Zorzi, Alessandro; Deyle, Kaycie; Heinis, Christian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">715-723</span>CODEN:
                <span class="NLM_cas:coden">NCAHBB</span>;
        ISSN:<span class="NLM_cas:issn">1755-4330</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Successful screening campaigns depend on large and structurally diverse collections of compds.  In macrocycle screening, variation of the mol. scaffold is important for structural diversity, but so far it has been challenging to diversify this aspect in large combinatorial libraries.  Here, the authors report the cyclization of peptides with two chem. bridges to provide rapid access to thousands of different macrocyclic scaffolds in libraries that are easy to synthesize, screen and decode.  Application of this strategy to phage-encoded libraries allowed for the screening of an unprecedented structural diversity of macrocycles against plasma kallikrein, which is important in the swelling disorder hereditary angioedema.  These libraries yielded inhibitors with remarkable binding properties (subnanomolar Ki, >1000-fold selectivity) despite the small mol. mass (∼1200 Da).  An interlaced bridge format characteristic of this strategy provided high proteolytic stability (t1/2 in plasma of >3 days), making double-bridged peptides potentially amenable to topical or oral delivery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpmZOAVFLZRH7Vg90H21EOLACvtfcHk0lhythirrLBQEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXovVCkurg%253D&md5=66161a855b4d8317fb456a36765d9766</span></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=10.1038%2Fs41557-018-0042-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41557-018-0042-7%26sid%3Dliteratum%253Aachs%26aulast%3DKale%26aufirst%3DS.%2BS.%26aulast%3DVillequey%26aufirst%3DC.%26aulast%3DKong%26aufirst%3DX.-D.%26aulast%3DZorzi%26aufirst%3DA.%26aulast%3DDeyle%26aufirst%3DK.%26aulast%3DHeinis%26aufirst%3DC.%26atitle%3DCyclization%2520of%2520Peptides%2520with%2520Two%2520Chemical%2520Bridges%2520Affords%2520Large%2520Scaffold%2520Diversities%26jtitle%3DNat.%2520Chem.%26date%3D2018%26volume%3D10%26spage%3D715%26epage%3D723%26doi%3D10.1038%2Fs41557-018-0042-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref78"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref78'); return false;" data-citation="" class="refNumLink">78</a></strong><div class="NLM_citation" id="cit78"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kong, X.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriya, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carle, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pojer, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abriata, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deyle, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heinis, C.</span></span> <span> </span><span class="NLM_article-title">De Novo Development of Proteolytically Resistant Therapeutic Peptides for Oral Administration</span>. <i>Nat. Biomed. Eng.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">560</span>– <span class="NLM_lpage">571</span>, <span class="refDoi"> DOI: 10.1038/s41551-020-0556-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1038%2Fs41551-020-0556-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=32393891" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BB3cXpt1Oqur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2020&pages=560-571&author=X.-D.+Kongauthor=J.+Moriyaauthor=V.+Carleauthor=F.+Pojerauthor=L.+A.+Abriataauthor=K.+Deyleauthor=C.+Heinis&title=De+Novo+Development+of+Proteolytically+Resistant+Therapeutic+Peptides+for+Oral+Administration&doi=10.1038%2Fs41551-020-0556-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit78R"><div class="casContent"><span class="casTitleNuber">78</span><div class="casTitle"><span class="NLM_cas:atitle">De novo development of proteolytically resistant therapeutic peptides for oral administration</span></div><div class="casAuthors">Kong, Xu-Dong; Moriya, Jun; Carle, Vanessa; Pojer, Florence; Abriata, Luciano A.; Deyle, Kaycie; Heinis, Christian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biomedical Engineering</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">560-571</span>CODEN:
                <span class="NLM_cas:coden">NBEAB3</span>;
        ISSN:<span class="NLM_cas:issn">2157-846X</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Abstr.: The oral administration of peptide drugs is hampered by their metabolic instability and limited intestinal uptake.  Here, we describe a method for the generation of small target-specific peptides (less than 1,600 Da in size) that resist gastrointestinal proteases.  By using phage display to screen large libraries of genetically encoded double-bridged peptides on protease-resistant fd bacteriophages, we generated a peptide inhibitor of the coagulation Factor XIa with nanomolar affinity that resisted gastrointestinal proteases in all regions of the gastrointestinal tract of mice after oral administration, enabling more than 30% of the peptide to remain intact, and small quantities of it to reach the blood circulation.  We also developed a gastrointestinal-protease-resistant peptide antagonist for the interleukin-23 receptor, which has a role in the pathogenesis of Crohn's disease and ulcerative colitis.  The de novo generation of targeted peptides that resist proteolytic degrdn. in the gastrointestinal tract should help the development of effective peptides for oral delivery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGowjbT2rK-aWrVg90H21EOLACvtfcHk0lhythirrLBQEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXpt1Oqur0%253D&md5=d7db093aa00c687da06c68cc452680f4</span></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=10.1038%2Fs41551-020-0556-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41551-020-0556-3%26sid%3Dliteratum%253Aachs%26aulast%3DKong%26aufirst%3DX.-D.%26aulast%3DMoriya%26aufirst%3DJ.%26aulast%3DCarle%26aufirst%3DV.%26aulast%3DPojer%26aufirst%3DF.%26aulast%3DAbriata%26aufirst%3DL.%2BA.%26aulast%3DDeyle%26aufirst%3DK.%26aulast%3DHeinis%26aufirst%3DC.%26atitle%3DDe%2520Novo%2520Development%2520of%2520Proteolytically%2520Resistant%2520Therapeutic%2520Peptides%2520for%2520Oral%2520Administration%26jtitle%3DNat.%2520Biomed.%2520Eng.%26date%3D2020%26volume%3D4%26spage%3D560%26epage%3D571%26doi%3D10.1038%2Fs41551-020-0556-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref79"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref79'); return false;" data-citation="" class="refNumLink">79</a></strong><div class="NLM_citation" id="cit79"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baeriswyl, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heinis, C.</span></span> <span> </span><span class="NLM_article-title">Phage Selection of Cyclic Peptide Antagonists with Increased Stability toward Intestinal Proteases</span>. <i>Protein Eng., Des. Sel.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">81</span>– <span class="NLM_lpage">89</span>, <span class="refDoi"> DOI: 10.1093/protein/gzs085</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1093%2Fprotein%2Fgzs085" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=23100545" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVCisL%252FJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2013&pages=81-89&author=V.+Baeriswylauthor=C.+Heinis&title=Phage+Selection+of+Cyclic+Peptide+Antagonists+with+Increased+Stability+toward+Intestinal+Proteases&doi=10.1093%2Fprotein%2Fgzs085"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit79R"><div class="casContent"><span class="casTitleNuber">79</span><div class="casTitle"><span class="NLM_cas:atitle">Phage selection of cyclic peptide antagonists with increased stability toward intestinal proteases</span></div><div class="casAuthors">Baeriswyl, Vanessa; Heinis, Christian</div><div class="citationInfo"><span class="NLM_cas:title">Protein Engineering, Design & Selection</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">81-89</span>CODEN:
                <span class="NLM_cas:coden">PEDSBR</span>;
        ISSN:<span class="NLM_cas:issn">1741-0126</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The oral delivery of protein and peptide drugs is limited by their proteolytic degrdn. and the poor absorption across the intestinal epithelia.  In this work, we exposed a phage library of small bicyclic peptides (<1.5 kDa) to a pancreatic ext. of proteases prior to affinity selection to enrich binders with higher stability in the intestinal environment.  Panning with the therapeutic target plasma kallikrein yielded potent inhibitors (Kis between 5.6 and 336 nM) wherein bicyclic peptides isolated with proteolytic pressure were more stable.  A proline residue found in a specific position of several resistant bicyclic peptides proved to be a protective mark', rendering the bicyclic peptides resistant to significantly higher concns. of intestinal proteases while retaining essentially their inhibitory activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4LvGcBCGYWLVg90H21EOLACvtfcHk0lhythirrLBQEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVCisL%252FJ&md5=75e20392fb64e8cf24baa3f88c2955f1</span></div><a href="/servlet/linkout?suffix=cit79&amp;dbid=16384&amp;doi=10.1093%2Fprotein%2Fgzs085&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fprotein%252Fgzs085%26sid%3Dliteratum%253Aachs%26aulast%3DBaeriswyl%26aufirst%3DV.%26aulast%3DHeinis%26aufirst%3DC.%26atitle%3DPhage%2520Selection%2520of%2520Cyclic%2520Peptide%2520Antagonists%2520with%2520Increased%2520Stability%2520toward%2520Intestinal%2520Proteases%26jtitle%3DProtein%2520Eng.%252C%2520Des.%2520Sel.%26date%3D2013%26volume%3D26%26spage%3D81%26epage%3D89%26doi%3D10.1093%2Fprotein%2Fgzs085" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref80"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref80'); return false;" data-citation="" class="refNumLink">80</a></strong><div class="NLM_citation" id="cit80"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bernhagen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jungbluth, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quilis, N. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dostalek, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jalink, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Timmerman, P.</span></span> <span> </span><span class="NLM_article-title">Bicyclic RGD Peptides with Exquisite Selectivity for the Integrin Avβ3 Receptor Using a “Random Design” Approach</span>. <i>ACS Comb. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">198</span>– <span class="NLM_lpage">206</span>, <span class="refDoi"> DOI: 10.1021/acscombsci.8b00144</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acscombsci.8b00144" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A280%3ADC%252BB3cnpsF2hsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2019&pages=198-206&author=D.+Bernhagenauthor=V.+Jungbluthauthor=N.+G.+Quilisauthor=J.+Dostalekauthor=P.+B.+Whiteauthor=K.+Jalinkauthor=P.+Timmerman&title=Bicyclic+RGD+Peptides+with+Exquisite+Selectivity+for+the+Integrin+Av%CE%B23+Receptor+Using+a+%E2%80%9CRandom+Design%E2%80%9D+Approach&doi=10.1021%2Facscombsci.8b00144"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit80R"><div class="casContent"><span class="casTitleNuber">80</span><div class="casTitle"><span class="NLM_cas:atitle">Bicyclic RGD Peptides with Exquisite Selectivity for the Integrin αvβ3 Receptor Using a "Random Design" Approach</span></div><div class="casAuthors">Bernhagen Dominik; Timmerman Peter; Jungbluth Vanessa; Quilis Nestor Gisbert; Dostalek Jakub; White Paul B; Jalink Kees; Timmerman Peter</div><div class="citationInfo"><span class="NLM_cas:title">ACS combinatorial science</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">198-206</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">We describe the identification of bicyclic RGD peptides with high affinity and selectivity for integrin αvβ3 via high-throughput screening of partially randomized libraries.  Peptide libraries (672 different compounds) comprising the universal integrin-binding sequence Arg-Gly-Asp (RGD) in the first loop and a randomized sequence XXX (X being one of 18 canonical l-amino acids) in the second loop, both enclosed by either an l- or d-Cys residue, were converted to bicyclic peptides via reaction with 1,3,5-tris(bromomethyl)benzene (T3).  Screening of first-generation libraries yielded lead bicyclic inhibitors displaying submicromolar affinities for integrin αvβ3 (e.g., CT3HEQcT3RGDcT3, IC50 = 195 nM).  Next generation (second and third) libraries were obtained by partially varying the structure of the strongest lead inhibitors and screening for improved affinities and selectivities.  In this way, we identified the highly selective bicyclic αvβ3-binders CT3HPQcT3RGDcT3 (IC50 = 30 nM), CT3HPQCT3RGDcT3 (IC50 = 31 nM), and CT3HSQCT3RGDcT3 (IC50 = 42 nM) with affinities comparable to that of a knottin-RGD-type peptide (32 amino acids, IC50 = 38 nM) and outstanding selectivities over integrins αvβ5 (IC50 > 10000 nM) and α5β1 (IC50 > 10000 nM).  Affinity measurements using surface plasmon-enhanced fluorescence spectroscopy (SPFS) yielded Kd values of 0.4 and 0.6 nM for the Cy5-labeled bicycle CT3HPQcT3RGDcT3 and RGD "knottin" peptide, respectively.  In vitro staining of HT29 cells with Cy5-labeled bicycles using confocal microscopy revealed strong binding to integrins in their natural environment, which highlights the high potential of these peptides as markers of integrin expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSXZ9sPIKtyyyEJCPKhJPuqfW6udTcc2eZx_vawPaCcabntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3cnpsF2hsw%253D%253D&md5=8e2a6674879be4657dfff2792a9cb49c</span></div><a href="/servlet/linkout?suffix=cit80&amp;dbid=16384&amp;doi=10.1021%2Facscombsci.8b00144&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facscombsci.8b00144%26sid%3Dliteratum%253Aachs%26aulast%3DBernhagen%26aufirst%3DD.%26aulast%3DJungbluth%26aufirst%3DV.%26aulast%3DQuilis%26aufirst%3DN.%2BG.%26aulast%3DDostalek%26aufirst%3DJ.%26aulast%3DWhite%26aufirst%3DP.%2BB.%26aulast%3DJalink%26aufirst%3DK.%26aulast%3DTimmerman%26aufirst%3DP.%26atitle%3DBicyclic%2520RGD%2520Peptides%2520with%2520Exquisite%2520Selectivity%2520for%2520the%2520Integrin%2520Av%25CE%25B23%2520Receptor%2520Using%2520a%2520%25E2%2580%259CRandom%2520Design%25E2%2580%259D%2520Approach%26jtitle%3DACS%2520Comb.%2520Sci.%26date%3D2019%26volume%3D21%26spage%3D198%26epage%3D206%26doi%3D10.1021%2Facscombsci.8b00144" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref81"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref81'); return false;" data-citation="" class="refNumLink">81</a></strong><div class="NLM_citation" id="cit81"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Heinis, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rutherford, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freund, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G.</span></span> <span> </span><span class="NLM_article-title">Phage-Encoded Combinatorial Chemical Libraries Based on Bicyclic Peptides</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">502</span>– <span class="NLM_lpage">507</span>, <span class="refDoi"> DOI: 10.1038/nchembio.184</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1038%2Fnchembio.184" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=19483697" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BD1MXms1altbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2009&pages=502-507&author=C.+Heinisauthor=T.+Rutherfordauthor=S.+Freundauthor=G.+Winter&title=Phage-Encoded+Combinatorial+Chemical+Libraries+Based+on+Bicyclic+Peptides&doi=10.1038%2Fnchembio.184"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit81R"><div class="casContent"><span class="casTitleNuber">81</span><div class="casTitle"><span class="NLM_cas:atitle">Phage-encoded combinatorial chemical libraries based on bicyclic peptides</span></div><div class="casAuthors">Heinis, Christian; Rutherford, Trevor; Freund, Stephan; Winter, Greg</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">502-507</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Here we describe a phage strategy for the selection of ligands based on bicyclic or linear peptides attached covalently to an org. core.  We designed peptide repertoires with three reactive cysteine residues, each spaced apart by several random amino acid residues, and we fused the repertoires to the phage gene-3-protein.  Conjugation with tris-(bromomethyl)benzene via the reactive cysteines generated repertoires of peptide conjugates with two peptide loops anchored to a mesitylene core.  Iterative affinity selections yielded several enzyme inhibitors; after further mutagenesis and selection, we were able to chem. synthesize a lead inhibitor (PK15; Ki = 1.5 nM) specific to human plasma kallikrein that efficiently interrupted the intrinsic coagulation pathway in human plasma tested ex vivo.  This approach offers a powerful means of generating and selecting bicyclic macrocycles (or if cleaved, linear derivs. thereof) as ligands poised at the interface of small-mol. drugs and biologics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZhJRZyldisLVg90H21EOLACvtfcHk0ljkh8Z4juYGdw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXms1altbs%253D&md5=0634460f77b185fbcf85309c6dbf93e3</span></div><a href="/servlet/linkout?suffix=cit81&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.184&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.184%26sid%3Dliteratum%253Aachs%26aulast%3DHeinis%26aufirst%3DC.%26aulast%3DRutherford%26aufirst%3DT.%26aulast%3DFreund%26aufirst%3DS.%26aulast%3DWinter%26aufirst%3DG.%26atitle%3DPhage-Encoded%2520Combinatorial%2520Chemical%2520Libraries%2520Based%2520on%2520Bicyclic%2520Peptides%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2009%26volume%3D5%26spage%3D502%26epage%3D507%26doi%3D10.1038%2Fnchembio.184" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref82"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref82'); return false;" data-citation="" class="refNumLink">82</a></strong><div class="NLM_citation" id="cit82"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulligan, V. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mravic, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lemmin, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ford, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silva, D.-A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baker, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeGrado, W. F.</span></span> <span> </span><span class="NLM_article-title">De Novo Design of Covalently Constrained Mesosize Protein Scaffolds with Unique Tertiary Structures</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>114</i></span>,  <span class="NLM_fpage">10852</span>– <span class="NLM_lpage">10857</span>, <span class="refDoi"> DOI: 10.1073/pnas.1710695114</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1073%2Fpnas.1710695114" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=28973862" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFKjtL%252FL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2017&pages=10852-10857&author=B.+Dangauthor=H.+Wuauthor=V.+K.+Mulliganauthor=M.+Mravicauthor=Y.+Wuauthor=T.+Lemminauthor=A.+Fordauthor=D.-A.+Silvaauthor=D.+Bakerauthor=W.+F.+DeGrado&title=De+Novo+Design+of+Covalently+Constrained+Mesosize+Protein+Scaffolds+with+Unique+Tertiary+Structures&doi=10.1073%2Fpnas.1710695114"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit82R"><div class="casContent"><span class="casTitleNuber">82</span><div class="casTitle"><span class="NLM_cas:atitle">De novo design of covalently constrained mesosize protein scaffolds with unique tertiary structures</span></div><div class="casAuthors">Dang, Bobo; Wu, Haifan; Mulligan, Vikram Khipple; Mravic, Marco; Wu, Yibing; Lemmin, Thomas; Ford, Alexander; Silva, Daniel-Adriano; Baker, David; DeGrado, William F.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">114</span>
        (<span class="NLM_cas:issue">41</span>),
    <span class="NLM_cas:pages">10852-10857</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The folding of natural proteins typically relies on hydrophobic packing, metal binding, or disulfide bond formation in the protein core.  Alternatively, a 3D structure can be defined by incorporating a multivalent crosslinking agent, and this approach has been successfully developed for the selection of bicyclic peptides from large random-sequence libraries.  By contrast, there is no general method for the de novo computational design of multicross-linked proteins with predictable and well-defined folds, including ones not found in nature.  Here the authors use Rosetta and Tertiary Motifs (TERMs) to design small proteins that fold around multivalent crosslinkers.  The hydrophobic crosslinkers stabilize the fold by macrocyclic restraints, and they also form an integral part of a small apolar core.  The designed CovCore proteins were prepd. by chem. synthesis, and their structures were detd. by soln. NMR or x-ray crystallog.  These mesosized proteins, lying between conventional proteins and small peptides, are easily accessible either through biosynthetic precursors or chem. synthesis.  The unique tertiary structures and ease of synthesis of CovCore proteins indicate that they should provide versatile templates for developing inhibitors of protein-protein interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoT3cI2O-jid7Vg90H21EOLACvtfcHk0ljkh8Z4juYGdw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFKjtL%252FL&md5=27b2dd6ab774dd6f223971003bd65a6d</span></div><a href="/servlet/linkout?suffix=cit82&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1710695114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1710695114%26sid%3Dliteratum%253Aachs%26aulast%3DDang%26aufirst%3DB.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DMulligan%26aufirst%3DV.%2BK.%26aulast%3DMravic%26aufirst%3DM.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DLemmin%26aufirst%3DT.%26aulast%3DFord%26aufirst%3DA.%26aulast%3DSilva%26aufirst%3DD.-A.%26aulast%3DBaker%26aufirst%3DD.%26aulast%3DDeGrado%26aufirst%3DW.%2BF.%26atitle%3DDe%2520Novo%2520Design%2520of%2520Covalently%2520Constrained%2520Mesosize%2520Protein%2520Scaffolds%2520with%2520Unique%2520Tertiary%2520Structures%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2017%26volume%3D114%26spage%3D10852%26epage%3D10857%26doi%3D10.1073%2Fpnas.1710695114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref83"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref83'); return false;" data-citation="" class="refNumLink">83</a></strong><div class="NLM_citation" id="cit83"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Neubacher, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saya, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amore, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grossmann, T. N.</span></span> <span> </span><span class="NLM_article-title">In Situ Cyclization of Proteins (INCYPRO): Cross-Link Derivatization Modulates Protein Stability</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>85</i></span>,  <span class="NLM_fpage">14761476</span>– <span class="NLM_lpage">1483</span>, <span class="refDoi"> DOI: 10.1021/acs.joc.9b024901476</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1021%2Facs.joc.9b024901476" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=85&publication_year=2019&pages=14761476-1483&author=S.+Neubacherauthor=J.+M.+Sayaauthor=A.+Amoreauthor=T.+N.+Grossmann&title=In+Situ+Cyclization+of+Proteins+%28INCYPRO%29%3A+Cross-Link+Derivatization+Modulates+Protein+Stability&doi=10.1021%2Facs.joc.9b024901476"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit83&amp;dbid=16384&amp;doi=10.1021%2Facs.joc.9b024901476&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.joc.9b024901476%26sid%3Dliteratum%253Aachs%26aulast%3DNeubacher%26aufirst%3DS.%26aulast%3DSaya%26aufirst%3DJ.%2BM.%26aulast%3DAmore%26aufirst%3DA.%26aulast%3DGrossmann%26aufirst%3DT.%2BN.%26atitle%3DIn%2520Situ%2520Cyclization%2520of%2520Proteins%2520%2528INCYPRO%2529%253A%2520Cross-Link%2520Derivatization%2520Modulates%2520Protein%2520Stability%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2019%26volume%3D85%26spage%3D14761476%26epage%3D1483%26doi%3D10.1021%2Facs.joc.9b024901476" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref84"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref84'); return false;" data-citation="" class="refNumLink">84</a></strong><div class="NLM_citation" id="cit84"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pelay-Gimeno, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bange, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hennig, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grossmann, T. N.</span></span> <span> </span><span class="NLM_article-title">In Situ Cyclization of Native Proteins: Structure-Based Design of a Bicyclic Enzyme</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">11164</span>– <span class="NLM_lpage">11170</span>, <span class="refDoi"> DOI: 10.1002/anie.201804506</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1002%2Fanie.201804506" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtF2ls7%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2018&pages=11164-11170&author=M.+Pelay-Gimenoauthor=T.+Bangeauthor=S.+Hennigauthor=T.+N.+Grossmann&title=In+Situ+Cyclization+of+Native+Proteins%3A+Structure-Based+Design+of+a+Bicyclic+Enzyme&doi=10.1002%2Fanie.201804506"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit84R"><div class="casContent"><span class="casTitleNuber">84</span><div class="casTitle"><span class="NLM_cas:atitle">In Situ Cyclization of Native Proteins: Structure-Based Design of a Bicyclic Enzyme</span></div><div class="casAuthors">Pelay-Gimeno, Marta; Bange, Tanja; Hennig, Sven; Grossmann, Tom N.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">35</span>),
    <span class="NLM_cas:pages">11164-11170</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Increased tolerance of enzymes towards thermal and chem. stress is required for many applications and can be achieved by macrocyclization of the enzyme resulting in the stabilizing of its tertiary structure.  Thus far, macrocyclization approaches utilize a very limited structural diversity, which complicates the design process.  Herein, we report an approach that enables cyclization through the installation of modular crosslinks into native proteins composed entirely of proteinogenic amino acids.  Our stabilization procedure involves the introduction of three surface-exposed cysteine residues, which are reacted with a triselectrophile, resulting in the in situ cyclization of the protein (INCYPRO).  A bicyclic version of sortase A (SrtA) was designed that exhibits increased tolerance towards thermal as well as chem. denaturation, and proved to be efficient in protein labeling under denaturing conditions.  In addn., we applied INCYPRO to the KIX domain from the human CREB-binding protein (CBP), resulting in up to 24 °C increased thermal stability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9bVlewFNKi7Vg90H21EOLACvtfcHk0lipSQ-T7MHe4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtF2ls7%252FI&md5=ef6a00e4a22408647fbbb5158110564f</span></div><a href="/servlet/linkout?suffix=cit84&amp;dbid=16384&amp;doi=10.1002%2Fanie.201804506&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201804506%26sid%3Dliteratum%253Aachs%26aulast%3DPelay-Gimeno%26aufirst%3DM.%26aulast%3DBange%26aufirst%3DT.%26aulast%3DHennig%26aufirst%3DS.%26aulast%3DGrossmann%26aufirst%3DT.%2BN.%26atitle%3DIn%2520Situ%2520Cyclization%2520of%2520Native%2520Proteins%253A%2520Structure-Based%2520Design%2520of%2520a%2520Bicyclic%2520Enzyme%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2018%26volume%3D57%26spage%3D11164%26epage%3D11170%26doi%3D10.1002%2Fanie.201804506" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref85"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref85'); return false;" data-citation="" class="refNumLink">85</a></strong><div class="NLM_citation" id="cit85"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Blackwell, H. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sadowsky, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howard, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sampson, J. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chao, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steinmetz, W. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Leary, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grubbs, R. H.</span></span> <span> </span><span class="NLM_article-title">Ring-Closing Metathesis of Olefinic Peptides: Design, Synthesis, and Structural Characterization of Macrocyclic Helical Peptides</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">5291</span>– <span class="NLM_lpage">5302</span>, <span class="refDoi"> DOI: 10.1021/jo015533k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo015533k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BD3MXjtlCktr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2001&pages=5291-5302&author=H.+E.+Blackwellauthor=J.+D.+Sadowskyauthor=R.+J.+Howardauthor=J.+N.+Sampsonauthor=J.+A.+Chaoauthor=W.+E.+Steinmetzauthor=D.+J.+O%E2%80%99Learyauthor=R.+H.+Grubbs&title=Ring-Closing+Metathesis+of+Olefinic+Peptides%3A+Design%2C+Synthesis%2C+and+Structural+Characterization+of+Macrocyclic+Helical+Peptides&doi=10.1021%2Fjo015533k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit85R"><div class="casContent"><span class="casTitleNuber">85</span><div class="casTitle"><span class="NLM_cas:atitle">Ring-Closing Metathesis of Olefinic Peptides: Design, Synthesis, and Structural Characterization of Macrocyclic Helical Peptides</span></div><div class="casAuthors">Blackwell, Helen E.; Sadowsky, Jack D.; Howard, Rebecca J.; Sampson, Joshua N.; Chao, Jeffery A.; Steinmetz, Wayne E.; O'Leary, Daniel J.; Grubbs, Robert H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">5291-5302</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Heptapeptides contg. residues with terminal olefin-derivatized side chains (I) have been treated with ruthenium alkylidene and undergone facile ring-closing olefin metathesis (RCM) to give 21- and 23-membered macrocyclic peptides.  The primary structures of I were based upon a previously studied heptapeptide, which was shown to adopt a predominantly 310-helical conformation in CDCl3 soln. and an α-helical conformation in the solid state.  CD, IR, and soln.-phase 1H NMR studies strongly suggested that acyclic precursors I and the fully satd. macrocyclic products also adopted helical conformations in apolar org. solvents.  Single-crystal X-ray diffraction of one cyclic peptide showed it to exist as a right-handed 310-helix up to the fifth residue.  Soln.-phase NMR structures of both acyclic peptide I and cyclic peptide in CD2Cl2 indicated that the acyclic diene assumes a loosely 310-helical conformation, which is considerably rigidified upon macrocyclization.  The relative ease of introducing carbon-carbon bonds into peptide secondary structures by RCM and the predicted metabolic stability of these bonds renders olefin metathesis an exceptional methodol. for the synthesis of rigidified peptide architectures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoRIl-7fCxsfLVg90H21EOLACvtfcHk0lipSQ-T7MHe4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXjtlCktr0%253D&md5=c1f97d63de6a35d2e08068a89f3c4433</span></div><a href="/servlet/linkout?suffix=cit85&amp;dbid=16384&amp;doi=10.1021%2Fjo015533k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo015533k%26sid%3Dliteratum%253Aachs%26aulast%3DBlackwell%26aufirst%3DH.%2BE.%26aulast%3DSadowsky%26aufirst%3DJ.%2BD.%26aulast%3DHoward%26aufirst%3DR.%2BJ.%26aulast%3DSampson%26aufirst%3DJ.%2BN.%26aulast%3DChao%26aufirst%3DJ.%2BA.%26aulast%3DSteinmetz%26aufirst%3DW.%2BE.%26aulast%3DO%25E2%2580%2599Leary%26aufirst%3DD.%2BJ.%26aulast%3DGrubbs%26aufirst%3DR.%2BH.%26atitle%3DRing-Closing%2520Metathesis%2520of%2520Olefinic%2520Peptides%253A%2520Design%252C%2520Synthesis%252C%2520and%2520Structural%2520Characterization%2520of%2520Macrocyclic%2520Helical%2520Peptides%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2001%26volume%3D66%26spage%3D5291%26epage%3D5302%26doi%3D10.1021%2Fjo015533k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref86"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref86'); return false;" data-citation="" class="refNumLink">86</a></strong><div class="NLM_citation" id="cit86"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Galande, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bramlett, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trent, J. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burris, T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wittliff, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spatola, A. F.</span></span> <span> </span><span class="NLM_article-title">Potent Inhibitors of LXXLL-Based Protein-Protein Interactions</span>. <i>ChemBioChem</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">1991</span>– <span class="NLM_lpage">1998</span>, <span class="refDoi"> DOI: 10.1002/cbic.200500083</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1002%2Fcbic.200500083" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=16222726" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht1ehtLrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2005&pages=1991-1998&author=A.+K.+Galandeauthor=K.+S.+Bramlettauthor=J.+O.+Trentauthor=T.+P.+Burrisauthor=J.+L.+Wittliffauthor=A.+F.+Spatola&title=Potent+Inhibitors+of+LXXLL-Based+Protein-Protein+Interactions&doi=10.1002%2Fcbic.200500083"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit86R"><div class="casContent"><span class="casTitleNuber">86</span><div class="casTitle"><span class="NLM_cas:atitle">Potent inhibitors of LXXLL-based protein-protein interactions</span></div><div class="casAuthors">Galande, Amit K.; Bramlett, Kelli S.; Trent, John O.; Burris, Thomas P.; Wittliff, James L.; Spatola, Arno F.</div><div class="citationInfo"><span class="NLM_cas:title">ChemBioChem</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1991-1998</span>CODEN:
                <span class="NLM_cas:coden">CBCHFX</span>;
        ISSN:<span class="NLM_cas:issn">1439-4227</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Protein-protein interactions between estrogen receptors, ERα and ERβ, and their coactivators (CoAs) are an attractive target for drug intervention.  This interaction is mediated by a small pentapeptide motif (LXXLL), termed the NR box.  Based on this motif, a variety of cyclic and linear peptides were synthesized in order to gain a better understanding of the assocn. of CoA proteins with the ER isoforms.  Utilizing a time-resolved florescence-based coactivator interaction assay, we detd. the abilities of these peptides to inhibit this interaction.  Using mol. modeling and CD spectroscopy, we have examd. the structural basis of their bioactivities with both hormone receptor isoforms.  Either homocysteine or penicillamine was utilized as a substitute for cysteine in the disulfide-bridged peptides, while tertiary leucine and neopentyl glycine were used as the surrogates for the NR box leucines.  The most potent disufide-bridged peptide (Ki 70 pM, with ERα) incorporates neopentyl glycine in the NR box, while the most active peptide in this series with ERβ (Ki = 350 pM) incorporates tertiary leucine.  Surprisingly, several linear peptides contg. a single cysteine residue showed activities with low nanomolar Ki values.  Collectively, our results suggest a synthetic approach for designing potent and selective peptidomimetics for ERα and ERβ interactions with CoA proteins effecting estrogen action.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqRkazg_OTBybVg90H21EOLACvtfcHk0lgnwKiT9v_frQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXht1ehtLrK&md5=02351ff58df231cbd892cc30e19e50b7</span></div><a href="/servlet/linkout?suffix=cit86&amp;dbid=16384&amp;doi=10.1002%2Fcbic.200500083&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbic.200500083%26sid%3Dliteratum%253Aachs%26aulast%3DGalande%26aufirst%3DA.%2BK.%26aulast%3DBramlett%26aufirst%3DK.%2BS.%26aulast%3DTrent%26aufirst%3DJ.%2BO.%26aulast%3DBurris%26aufirst%3DT.%2BP.%26aulast%3DWittliff%26aufirst%3DJ.%2BL.%26aulast%3DSpatola%26aufirst%3DA.%2BF.%26atitle%3DPotent%2520Inhibitors%2520of%2520LXXLL-Based%2520Protein-Protein%2520Interactions%26jtitle%3DChemBioChem%26date%3D2005%26volume%3D6%26spage%3D1991%26epage%3D1998%26doi%3D10.1002%2Fcbic.200500083" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref87"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref87'); return false;" data-citation="" class="refNumLink">87</a></strong><div class="NLM_citation" id="cit87"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Harrison, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shepherd, N. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoang, H. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruiz-Gómez, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Driver, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Desai, V. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abbenante, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fairlie, D. P.</span></span> <span> </span><span class="NLM_article-title">Downsizing Human, Bacterial, and Viral Proteins to Short Water-Stable Alpha Helices That Maintain Biological Potency</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>107</i></span>,  <span class="NLM_fpage">11686</span>– <span class="NLM_lpage">11691</span>, <span class="refDoi"> DOI: 10.1073/pnas.1002498107</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1073%2Fpnas.1002498107" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=20543141" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC3cXovVGkur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=107&publication_year=2010&pages=11686-11691&author=R.+S.+Harrisonauthor=N.+E.+Shepherdauthor=H.+N.+Hoangauthor=G.+Ruiz-G%C3%B3mezauthor=T.+A.+Hillauthor=R.+W.+Driverauthor=V.+S.+Desaiauthor=P.+R.+Youngauthor=G.+Abbenanteauthor=D.+P.+Fairlie&title=Downsizing+Human%2C+Bacterial%2C+and+Viral+Proteins+to+Short+Water-Stable+Alpha+Helices+That+Maintain+Biological+Potency&doi=10.1073%2Fpnas.1002498107"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit87R"><div class="casContent"><span class="casTitleNuber">87</span><div class="casTitle"><span class="NLM_cas:atitle">Downsizing human, bacterial, and viral proteins to short water-stable alpha helices that maintain biological potency</span></div><div class="casAuthors">Harrison, Rosemary S.; Shepherd, Nicholas E.; Hoang, Huy N.; Ruiz-Gomez, Gloria; Hill, Timothy A.; Driver, Russell W.; Desai, Vishal S.; Young, Paul R.; Abbenante, Giovanni; Fairlie, David P.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">107</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">11686-11691, S11686/1-S11686/52</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Recombinant proteins are important therapeutics due to potent, highly specific, and nontoxic actions in vivo.  However, they are expensive medicines to manuf., chem. unstable, and difficult to administer with low patient uptake and compliance.  Small mol. drugs are cheaper and more bioavailable, but less target-specific in vivo and often have assocd. side effects.  Here we combine some advantages of proteins and small mols. by taking short amino acid sequences that confer potency and selectivity to proteins, and fixing them as small constrained mols. that are chem. and structurally stable and easy to make.  Proteins often use short α-helixes of just 1-4 helical turns (4-15 amino acids) to interact with biol. targets, but peptides this short usually have negligible α-helicity in water.  Here we show that short peptides, corresponding to helical epitopes from viral, bacterial, or human proteins, can be strategically fixed in highly α-helical structures in water.  These helix-constrained compds. have similar biol. potencies as proteins that bear the same helical sequences.  Examples are (i) a picomolar inhibitor of Respiratory Syncytial Virus F protein mediated fusion with host cells, (ii) a nanomolar inhibitor of RNA binding to the transporter protein HIV-Rev, (iii) a submicromolar inhibitor of Streptococcus pneumoniae growth induced by quorum sensing pheromone Competence Stimulating Peptide, and (iv) a picomolar agonist of the GPCR pain receptor opioid receptor like receptor ORL-1.  This approach can be generally applicable to downsizing helical regions of proteins with broad applications to biol. and medicine.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpjCkDABZP0wrVg90H21EOLACvtfcHk0lgnwKiT9v_frQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXovVGkur8%253D&md5=d06bf2eb3c61268f906ee9e26e013be1</span></div><a href="/servlet/linkout?suffix=cit87&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1002498107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1002498107%26sid%3Dliteratum%253Aachs%26aulast%3DHarrison%26aufirst%3DR.%2BS.%26aulast%3DShepherd%26aufirst%3DN.%2BE.%26aulast%3DHoang%26aufirst%3DH.%2BN.%26aulast%3DRuiz-G%25C3%25B3mez%26aufirst%3DG.%26aulast%3DHill%26aufirst%3DT.%2BA.%26aulast%3DDriver%26aufirst%3DR.%2BW.%26aulast%3DDesai%26aufirst%3DV.%2BS.%26aulast%3DYoung%26aufirst%3DP.%2BR.%26aulast%3DAbbenante%26aufirst%3DG.%26aulast%3DFairlie%26aufirst%3DD.%2BP.%26atitle%3DDownsizing%2520Human%252C%2520Bacterial%252C%2520and%2520Viral%2520Proteins%2520to%2520Short%2520Water-Stable%2520Alpha%2520Helices%2520That%2520Maintain%2520Biological%2520Potency%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2010%26volume%3D107%26spage%3D11686%26epage%3D11691%26doi%3D10.1073%2Fpnas.1002498107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref88"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref88'); return false;" data-citation="" class="refNumLink">88</a></strong><div class="NLM_citation" id="cit88"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jackson, D. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">King, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chmielewski, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schultz, P. G.</span></span> <span> </span><span class="NLM_article-title">General Approach to the Synthesis of Short .Alpha.-Helical Peptides</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">1991</span>,  <span class="NLM_volume"><i>113</i></span>,  <span class="NLM_fpage">9391</span>– <span class="NLM_lpage">9392</span>, <span class="refDoi"> DOI: 10.1021/ja00024a067</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja00024a067" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADyaK3MXmvVSmsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=1991&pages=9391-9392&author=D.+Y.+Jacksonauthor=D.+S.+Kingauthor=J.+Chmielewskiauthor=S.+Singhauthor=P.+G.+Schultz&title=General+Approach+to+the+Synthesis+of+Short+.Alpha.-Helical+Peptides&doi=10.1021%2Fja00024a067"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit88R"><div class="casContent"><span class="casTitleNuber">88</span><div class="casTitle"><span class="NLM_cas:atitle">General approach to the synthesis of short α-helical peptides</span></div><div class="casAuthors">Jackson, David Y.; King, David S.; Chmielewski, Jean; Singh, Sunil; Schultz, Peter G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">9391-2</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    </div><div class="casAbstract">Short peptides have been synthesized which contain a single intramol. disulfide bond that stabilizes two helical turns.  Peptides contg. D- and L-S-(acetamidomethyl)-2-amino-6-mercaptohexanoic acid at the i and i + 7 residue, resp., show only slight α-helicity in the reduced form in aq. soln.  On oxidn., these peptides exhibit a large increase in α-helicity in water both at 0° and 60°.  This approach has been used to generate eight and sixteen amino acid peptides with high helicity.  Oxidn. can be carried out under a wide variety of conditions with peptides that contain a large variety of functional groups.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGryvUu77IcBrrVg90H21EOLACvtfcHk0lgnwKiT9v_frQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3MXmvVSmsbk%253D&md5=c5e9be77bd7fbcbf428424de7cf9a862</span></div><a href="/servlet/linkout?suffix=cit88&amp;dbid=16384&amp;doi=10.1021%2Fja00024a067&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja00024a067%26sid%3Dliteratum%253Aachs%26aulast%3DJackson%26aufirst%3DD.%2BY.%26aulast%3DKing%26aufirst%3DD.%2BS.%26aulast%3DChmielewski%26aufirst%3DJ.%26aulast%3DSingh%26aufirst%3DS.%26aulast%3DSchultz%26aufirst%3DP.%2BG.%26atitle%3DGeneral%2520Approach%2520to%2520the%2520Synthesis%2520of%2520Short%2520.Alpha.-Helical%2520Peptides%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1991%26volume%3D113%26spage%3D9391%26epage%3D9392%26doi%3D10.1021%2Fja00024a067" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref89"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref89'); return false;" data-citation="" class="refNumLink">89</a></strong><div class="NLM_citation" id="cit89"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Leduc, A.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trent, J. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wittliff, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bramlett, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Briggs, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chirgadze, N. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burris, T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spatola, A. F.</span></span> <span> </span><span class="NLM_article-title">Helix-Stabilized Cyclic Peptides as Selective Inhibitors of Steroid Receptor-Coactivator Interactions</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>100</i></span>,  <span class="NLM_fpage">11273</span>– <span class="NLM_lpage">11278</span>, <span class="refDoi"> DOI: 10.1073/pnas.1934759100</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1073%2Fpnas.1934759100" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=13679575" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BD3sXotFKnu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=100&publication_year=2003&pages=11273-11278&author=A.-M.+Leducauthor=J.+O.+Trentauthor=J.+L.+Wittliffauthor=K.+S.+Bramlettauthor=S.+L.+Briggsauthor=N.+Y.+Chirgadzeauthor=Y.+Wangauthor=T.+P.+Burrisauthor=A.+F.+Spatola&title=Helix-Stabilized+Cyclic+Peptides+as+Selective+Inhibitors+of+Steroid+Receptor-Coactivator+Interactions&doi=10.1073%2Fpnas.1934759100"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit89R"><div class="casContent"><span class="casTitleNuber">89</span><div class="casTitle"><span class="NLM_cas:atitle">Helix-stabilized cyclic peptides as selective inhibitors of steroid receptor-coactivator interactions</span></div><div class="casAuthors">Leduc, Anne-Marie; Trent, John O.; Wittliff, James L.; Bramlett, Kelli S.; Briggs, Stephen L.; Chirgadze, Nikolay Y.; Wang, Yong; Burris, Thomas P.; Spatola, Arno F.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">100</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">11273-11278</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The interaction between nuclear receptors and coactivators provides an arena for testing whether protein-protein interactions may be inhibited by small mol. drug candidates.  We provide evidence that a short cyclic peptide, contg. a copy of the LXXLL nuclear receptor box pentapeptide, binds tightly and selectively to estrogen receptor α.  Furthermore, as shown by x-ray anal., the disulfide-bridged nonapeptide, nonhelical in aq. solns., is able to adopt a quasihelical conformer while binding to the groove created by ligand attachment to estrogen receptor α.  An i, i+3 linked analog, H-Lys-cyclo(D-Cys-Ile-Leu-Cys)-Arg-Leu-Leu-Gln-NH2 (peptidomimetic estrogen receptor modulator 1), binds with a Ki of 25 nM, significantly better than an i, i+4 bridged cyclic amide, as predicted by mol. modeling design criteria.  The induction of helical character, effective binding, and receptor selectivity exhibited by this peptide analog provide strong support for this strategy.  The stabilization of minimalist surface motifs may prove useful for the control of other macromol. assemblies, esp. when an amphiphilic helix is crucial for the strong binding interaction between two proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo68LUoV9WM-LVg90H21EOLACvtfcHk0ljmgD1rMaAtWA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXotFKnu7Y%253D&md5=a62f033a94c5e1cc4562e956d6cb6bfd</span></div><a href="/servlet/linkout?suffix=cit89&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1934759100&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1934759100%26sid%3Dliteratum%253Aachs%26aulast%3DLeduc%26aufirst%3DA.-M.%26aulast%3DTrent%26aufirst%3DJ.%2BO.%26aulast%3DWittliff%26aufirst%3DJ.%2BL.%26aulast%3DBramlett%26aufirst%3DK.%2BS.%26aulast%3DBriggs%26aufirst%3DS.%2BL.%26aulast%3DChirgadze%26aufirst%3DN.%2BY.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DBurris%26aufirst%3DT.%2BP.%26aulast%3DSpatola%26aufirst%3DA.%2BF.%26atitle%3DHelix-Stabilized%2520Cyclic%2520Peptides%2520as%2520Selective%2520Inhibitors%2520of%2520Steroid%2520Receptor-Coactivator%2520Interactions%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2003%26volume%3D100%26spage%3D11273%26epage%3D11278%26doi%3D10.1073%2Fpnas.1934759100" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref90"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref90'); return false;" data-citation="" class="refNumLink">90</a></strong><div class="NLM_citation" id="cit90"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Phelan, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skelton, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Braisted, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDowell, R. S.</span></span> <span> </span><span class="NLM_article-title">A General Method for Constraining Short Peptides to an α-Helical Conformation</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>119</i></span>,  <span class="NLM_fpage">455</span>– <span class="NLM_lpage">460</span>, <span class="refDoi"> DOI: 10.1021/ja9611654</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja9611654" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref90/cit90&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADyaK2sXotVWgsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=1997&pages=455-460&author=J.+C.+Phelanauthor=N.+J.+Skeltonauthor=A.+C.+Braistedauthor=R.+S.+McDowell&title=A+General+Method+for+Constraining+Short+Peptides+to+an+%CE%B1-Helical+Conformation&doi=10.1021%2Fja9611654"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit90R"><div class="casContent"><span class="casTitleNuber">90</span><div class="casTitle"><span class="NLM_cas:atitle">A General Method for Constraining Short Peptides to an α-Helical Conformation</span></div><div class="casAuthors">Phelan, J. Christopher; Skelton, Nicholas J.; Braisted, Andrew C.; McDowell, Robert S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">119</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">455-460</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A method for constraining short peptides (contg. fewer than 20 residues) of an arbitrary sequence to an α-helical conformation (∼100% helical in H2O at 25 °C) is presented.  Gln residues at positions i and i + 7 of the peptides were tethered with an alkanediyl chain between the side chain nitrogen atoms.  Peptides contg. this tether were readily synthesized on the solid phase by amide formation between an α,ω-diaminoalkane and the side chain carboxylates of Glu residues.  The resulting cyclic peptides were studied by NMR and CD and were found to adopt an α-helical conformation in aq. soln. and this α-helix was thermally stable to ≥40°.  Corresponding untethered control peptides with N-methylglutamine at the i and i + 7 positions lacked helicity under the same conditions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjjjTLuHs4P7Vg90H21EOLACvtfcHk0ljmgD1rMaAtWA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXotVWgsg%253D%253D&md5=57fc8604cb7ea3c545e5eafcf4a4519f</span></div><a href="/servlet/linkout?suffix=cit90&amp;dbid=16384&amp;doi=10.1021%2Fja9611654&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja9611654%26sid%3Dliteratum%253Aachs%26aulast%3DPhelan%26aufirst%3DJ.%2BC.%26aulast%3DSkelton%26aufirst%3DN.%2BJ.%26aulast%3DBraisted%26aufirst%3DA.%2BC.%26aulast%3DMcDowell%26aufirst%3DR.%2BS.%26atitle%3DA%2520General%2520Method%2520for%2520Constraining%2520Short%2520Peptides%2520to%2520an%2520%25CE%25B1-Helical%2520Conformation%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1997%26volume%3D119%26spage%3D455%26epage%3D460%26doi%3D10.1021%2Fja9611654" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref91"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref91'); return false;" data-citation="" class="refNumLink">91</a></strong><div class="NLM_citation" id="cit91"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schafmeister, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Po, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verdine, G. L.</span></span> <span> </span><span class="NLM_article-title">An All-Hydrocarbon Cross-Linking System for Enhancing the Helicity and Metabolic Stability of Peptides</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>122</i></span>,  <span class="NLM_fpage">5891</span>– <span class="NLM_lpage">5892</span>, <span class="refDoi"> DOI: 10.1021/ja000563a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja000563a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BD3cXjslajurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=122&publication_year=2000&pages=5891-5892&author=C.+E.+Schafmeisterauthor=J.+Poauthor=G.+L.+Verdine&title=An+All-Hydrocarbon+Cross-Linking+System+for+Enhancing+the+Helicity+and+Metabolic+Stability+of+Peptides&doi=10.1021%2Fja000563a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit91R"><div class="casContent"><span class="casTitleNuber">91</span><div class="casTitle"><span class="NLM_cas:atitle">An All-Hydrocarbon Cross-Linking System for Enhancing the Helicity and Metabolic Stability of Peptides</span></div><div class="casAuthors">Schafmeister, Christian E.; Po, Julia; Verdine, Gregory L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">122</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">5891-5892</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The authors have synthesized peptides contg. a hydrocarbon linkage connecting two unnatural amino acids having either R or S stereochem. at the methyl- and tether-bearing α-carbon, with tethers of various lengths to form a macrocyclic structure which either stabilized or destabilized the peptides α-helix formation, depending on length of tether and resulting size of macrocycle.  The unnatural amino acids were inserted in either i, i+4 or i, i+7 positions in synthetic C-peptide sequence from RNAse A, using solid-phase chem., metathesized while still linked to support, and the cleaved products tested for increased α-helix content as it affected soly. and resistance to trypsin digestion.  Length of tether and sepn. of insertion positions were both detg. factors in successful metathesis and α-helix enhancement; Ri,i+7S insertion with tether length of 11 had a trypsin cleavage rate const. of 0.058 M-1s-1, compared to 0.37 M-1s-1 for tether length of 9.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHKipXFsY9w7Vg90H21EOLACvtfcHk0ljmgD1rMaAtWA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXjslajurk%253D&md5=804271601bdd8a1c71e38a7e239a98d9</span></div><a href="/servlet/linkout?suffix=cit91&amp;dbid=16384&amp;doi=10.1021%2Fja000563a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja000563a%26sid%3Dliteratum%253Aachs%26aulast%3DSchafmeister%26aufirst%3DC.%2BE.%26aulast%3DPo%26aufirst%3DJ.%26aulast%3DVerdine%26aufirst%3DG.%2BL.%26atitle%3DAn%2520All-Hydrocarbon%2520Cross-Linking%2520System%2520for%2520Enhancing%2520the%2520Helicity%2520and%2520Metabolic%2520Stability%2520of%2520Peptides%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2000%26volume%3D122%26spage%3D5891%26epage%3D5892%26doi%3D10.1021%2Fja000563a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref92"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref92'); return false;" data-citation="" class="refNumLink">92</a></strong><div class="NLM_citation" id="cit92"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pelay-Gimeno, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glas, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koch, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grossmann, T. N.</span></span> <span> </span><span class="NLM_article-title">Structure-Based Design of Inhibitors of Protein–Protein Interactions: Mimicking Peptide Binding Epitopes</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">8896</span>– <span class="NLM_lpage">8927</span>, <span class="refDoi"> DOI: 10.1002/anie.201412070</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1002%2Fanie.201412070" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=26119925" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVOmu7rM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2015&pages=8896-8927&author=M.+Pelay-Gimenoauthor=A.+Glasauthor=O.+Kochauthor=T.+N.+Grossmann&title=Structure-Based+Design+of+Inhibitors+of+Protein%E2%80%93Protein+Interactions%3A+Mimicking+Peptide+Binding+Epitopes&doi=10.1002%2Fanie.201412070"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit92R"><div class="casContent"><span class="casTitleNuber">92</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Design of Inhibitors of Protein-Protein Interactions: Mimicking Peptide Binding Epitopes</span></div><div class="casAuthors">Pelay-Gimeno, Marta; Glas, Adrian; Koch, Oliver; Grossmann, Tom N.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">31</span>),
    <span class="NLM_cas:pages">8896-8927</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  Protein-protein interactions (PPIs) are involved at all levels of cellular organization, thus making the development of PPI inhibitors extremely valuable.  The identification of selective inhibitors is challenging because of the shallow and extended nature of PPI interfaces.  Inhibitors can be obtained by mimicking peptide binding epitopes in their bioactive conformation.  For this purpose, several strategies have been evolved to enable a projection of side chain functionalities in analogy to peptide secondary structures, thereby yielding mols. that are generally referred to as peptidomimetics.  Herein, we introduce a new classification of peptidomimetics (classes A-D) that enables a clear assignment of available approaches.  Based on this classification, the Review summarizes strategies that have been applied for the structure-based design of PPI inhibitors through stabilizing or mimicking turns, β-sheets, and helixes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPTEmZpatXbrVg90H21EOLACvtfcHk0ljmgD1rMaAtWA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVOmu7rM&md5=447c84aab6275b1057fda9a610918f2b</span></div><a href="/servlet/linkout?suffix=cit92&amp;dbid=16384&amp;doi=10.1002%2Fanie.201412070&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201412070%26sid%3Dliteratum%253Aachs%26aulast%3DPelay-Gimeno%26aufirst%3DM.%26aulast%3DGlas%26aufirst%3DA.%26aulast%3DKoch%26aufirst%3DO.%26aulast%3DGrossmann%26aufirst%3DT.%2BN.%26atitle%3DStructure-Based%2520Design%2520of%2520Inhibitors%2520of%2520Protein%25E2%2580%2593Protein%2520Interactions%253A%2520Mimicking%2520Peptide%2520Binding%2520Epitopes%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2015%26volume%3D54%26spage%3D8896%26epage%3D8927%26doi%3D10.1002%2Fanie.201412070" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref93"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref93'); return false;" data-citation="" class="refNumLink">93</a></strong><div class="NLM_citation" id="cit93"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bhardwaj, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulligan, V. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bahl, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilmore, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harvey, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheneval, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buchko, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pulavarti, S. V. S. R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaas, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eletsky, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, P.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnsen, W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greisen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rocklin, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Linsky, T. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watkins, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rettie, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonneau, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coutsias, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Correnti, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szyperski, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craik, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baker, D.</span></span> <span> </span><span class="NLM_article-title">Accurate de Novo Design of Hyperstable Constrained Peptides</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>538</i></span>,  <span class="NLM_fpage">329</span>– <span class="NLM_lpage">335</span>, <span class="refDoi"> DOI: 10.1038/nature19791</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1038%2Fnature19791" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=27626386" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFWiurnL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=538&publication_year=2016&pages=329-335&author=G.+Bhardwajauthor=V.+K.+Mulliganauthor=C.+D.+Bahlauthor=J.+M.+Gilmoreauthor=P.+J.+Harveyauthor=O.+Chenevalauthor=G.+W.+Buchkoauthor=S.+V.+S.+R.+K.+Pulavartiauthor=Q.+Kaasauthor=A.+Eletskyauthor=P.-S.+Huangauthor=W.+A.+Johnsenauthor=P.+Greisenauthor=G.+J.+Rocklinauthor=Y.+Songauthor=T.+W.+Linskyauthor=A.+Watkinsauthor=S.+A.+Rettieauthor=X.+Xuauthor=L.+P.+Carterauthor=R.+Bonneauauthor=J.+M.+Olsonauthor=E.+Coutsiasauthor=C.+E.+Correntiauthor=T.+Szyperskiauthor=D.+J.+Craikauthor=D.+Baker&title=Accurate+de+Novo+Design+of+Hyperstable+Constrained+Peptides&doi=10.1038%2Fnature19791"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit93R"><div class="casContent"><span class="casTitleNuber">93</span><div class="casTitle"><span class="NLM_cas:atitle">Accurate de novo design of hyperstable constrained peptides</span></div><div class="casAuthors">Bhardwaj, Gaurav; Mulligan, Vikram Khipple; Bahl, Christopher D.; Gilmore, Jason M.; Harvey, Peta J.; Cheneval, Olivier; Buchko, Garry W.; Pulavarti, Surya V. S. R. K.; Kaas, Quentin; Eletsky, Alexander; Huang, Po-Ssu; Johnsen, William A.; Greisen, Per Jr; Rocklin, Gabriel J.; Song, Yifan; Linsky, Thomas W.; Watkins, Andrew; Rettie, Stephen A.; Xu, Xianzhong; Carter, Lauren P.; Bonneau, Richard; Olson, James M.; Coutsias, Evangelos; Correnti, Colin E.; Szyperski, Thomas; Craik, David J.; Baker, David</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">538</span>
        (<span class="NLM_cas:issue">7625</span>),
    <span class="NLM_cas:pages">329-335</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Naturally occurring, pharmacol. active peptides constrained with covalent crosslinks generally have shapes that have evolved to fit precisely into binding pockets on their targets.  Such peptides can have excellent pharmaceutical properties, combining the stability and tissue penetration of small-mol. drugs with the specificity of much larger protein therapeutics.  The ability to design constrained peptides with precisely specified tertiary structures would enable the design of shape-complementary inhibitors of arbitrary targets.  Here we describe the development of computational methods for accurate de novo design of conformationally restricted peptides, and the use of these methods to design 18-47 residue, disulfide-crosslinked peptides, a subset of which are heterochiral and/or N-C backbone-cyclized.  Both genetically encodable and non-canonical peptides are exceptionally stable to thermal and chem. denaturation, and 12 exptl. detd. X-ray and NMR structures are nearly identical to the computational design models.  The computational design methods and stable scaffolds presented here provide the basis for development of a new generation of peptide-based drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHeMGaFeL_kbVg90H21EOLACvtfcHk0lgOHNDPrWyKdQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFWiurnL&md5=2d2b551ad78a2d76c99eeaba0c763c8e</span></div><a href="/servlet/linkout?suffix=cit93&amp;dbid=16384&amp;doi=10.1038%2Fnature19791&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature19791%26sid%3Dliteratum%253Aachs%26aulast%3DBhardwaj%26aufirst%3DG.%26aulast%3DMulligan%26aufirst%3DV.%2BK.%26aulast%3DBahl%26aufirst%3DC.%2BD.%26aulast%3DGilmore%26aufirst%3DJ.%2BM.%26aulast%3DHarvey%26aufirst%3DP.%2BJ.%26aulast%3DCheneval%26aufirst%3DO.%26aulast%3DBuchko%26aufirst%3DG.%2BW.%26aulast%3DPulavarti%26aufirst%3DS.%2BV.%2BS.%2BR.%2BK.%26aulast%3DKaas%26aufirst%3DQ.%26aulast%3DEletsky%26aufirst%3DA.%26aulast%3DHuang%26aufirst%3DP.-S.%26aulast%3DJohnsen%26aufirst%3DW.%2BA.%26aulast%3DGreisen%26aufirst%3DP.%26aulast%3DRocklin%26aufirst%3DG.%2BJ.%26aulast%3DSong%26aufirst%3DY.%26aulast%3DLinsky%26aufirst%3DT.%2BW.%26aulast%3DWatkins%26aufirst%3DA.%26aulast%3DRettie%26aufirst%3DS.%2BA.%26aulast%3DXu%26aufirst%3DX.%26aulast%3DCarter%26aufirst%3DL.%2BP.%26aulast%3DBonneau%26aufirst%3DR.%26aulast%3DOlson%26aufirst%3DJ.%2BM.%26aulast%3DCoutsias%26aufirst%3DE.%26aulast%3DCorrenti%26aufirst%3DC.%2BE.%26aulast%3DSzyperski%26aufirst%3DT.%26aulast%3DCraik%26aufirst%3DD.%2BJ.%26aulast%3DBaker%26aufirst%3DD.%26atitle%3DAccurate%2520de%2520Novo%2520Design%2520of%2520Hyperstable%2520Constrained%2520Peptides%26jtitle%3DNature%26date%3D2016%26volume%3D538%26spage%3D329%26epage%3D335%26doi%3D10.1038%2Fnature19791" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref94"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref94'); return false;" data-citation="" class="refNumLink">94</a></strong><div class="NLM_citation" id="cit94"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Est, C. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mangrolia, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, R. M.</span></span> <span> </span><span class="NLM_article-title">ROSETTA-Informed Design of Structurally Stabilized Cyclic Anti-Amyloid Peptides</span>. <i>Protein Eng., Des. Sel.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">47</span>– <span class="NLM_lpage">57</span>, <span class="refDoi"> DOI: 10.1093/protein/gzz016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1093%2Fprotein%2Fgzz016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=31650164" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A280%3ADC%252BB3MjhsFWhuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2019&pages=47-57&author=C.+B.+Estauthor=P.+Mangroliaauthor=R.+M.+Murphy&title=ROSETTA-Informed+Design+of+Structurally+Stabilized+Cyclic+Anti-Amyloid+Peptides&doi=10.1093%2Fprotein%2Fgzz016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit94R"><div class="casContent"><span class="casTitleNuber">94</span><div class="casTitle"><span class="NLM_cas:atitle">ROSETTA-informed design of structurally stabilized cyclic anti-amyloid peptides</span></div><div class="casAuthors">Est Chandler B; Mangrolia Parth; Murphy Regina M</div><div class="citationInfo"><span class="NLM_cas:title">Protein engineering, design & selection : PEDS</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">47-57</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">β-amyloid oligomers are thought to be the most toxic species formed en route to fibril deposition in Alzheimer's disease.  Transthyretin is a natural sequestering agent of β-amyloid oligomers: the binding site to β-amyloid has been traced to strands G/H of the inner β-sheet of transthyretin.  A linear peptide, with the same primary sequence as the β-amyloid binding domain on transthyretin, was moderately effective at inhibiting β-amyloid fibril growth.  Insertion of a β-turn template and cyclization greatly increased stability against proteolysis and improved efficacy as an amyloid inhibitor.  However, the cyclic peptide still contained a significant amount of disorder.  Using the Simple Cyclic Peptide Application within ROSETTA as an in silico predictor of cyclic peptide conformation and stability, we investigated putative structural enhancements, including stabilization by disulfide linkages and insertion of a second β-turn template.  Several candidates were synthesized and tested for secondary structure and ability to inhibit β-amyloid aggregation.  The results demonstrate that cyclization, β-sheet structure and conformational homogeneity are all preferable design features, whereas disulfide bond formation across the two β-strands is not preferable.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQEwXe3qQ7PkIvKgEt20y1XfW6udTcc2eZxeHZ73_tOb7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3MjhsFWhuw%253D%253D&md5=9aefa98f5739f90a698bdab680efda4c</span></div><a href="/servlet/linkout?suffix=cit94&amp;dbid=16384&amp;doi=10.1093%2Fprotein%2Fgzz016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fprotein%252Fgzz016%26sid%3Dliteratum%253Aachs%26aulast%3DEst%26aufirst%3DC.%2BB.%26aulast%3DMangrolia%26aufirst%3DP.%26aulast%3DMurphy%26aufirst%3DR.%2BM.%26atitle%3DROSETTA-Informed%2520Design%2520of%2520Structurally%2520Stabilized%2520Cyclic%2520Anti-Amyloid%2520Peptides%26jtitle%3DProtein%2520Eng.%252C%2520Des.%2520Sel.%26date%3D2019%26volume%3D32%26spage%3D47%26epage%3D57%26doi%3D10.1093%2Fprotein%2Fgzz016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref95"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref95'); return false;" data-citation="" class="refNumLink">95</a></strong><div class="NLM_citation" id="cit95"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guéret, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thavam, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carbajo, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potowski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larsson, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dahl, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dellsén, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grossmann, T. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plowright, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valeur, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lemurell, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waldmann, H.</span></span> <span> </span><span class="NLM_article-title">Macrocyclic Modalities Combining Peptide Epitopes and Natural Product Fragments</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>142</i></span>,  <span class="NLM_fpage">4904</span>– <span class="NLM_lpage">4915</span>, <span class="refDoi"> DOI: 10.1021/jacs.0c00269</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.0c00269" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BB3cXjtFGitLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=142&publication_year=2020&pages=4904-4915&author=S.+M.+Gu%C3%A9retauthor=S.+Thavamauthor=R.+J.+Carbajoauthor=M.+Potowskiauthor=N.+Larssonauthor=G.+Dahlauthor=A.+Dells%C3%A9nauthor=T.+N.+Grossmannauthor=A.+T.+Plowrightauthor=E.+Valeurauthor=M.+Lemurellauthor=H.+Waldmann&title=Macrocyclic+Modalities+Combining+Peptide+Epitopes+and+Natural+Product+Fragments&doi=10.1021%2Fjacs.0c00269"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit95R"><div class="casContent"><span class="casTitleNuber">95</span><div class="casTitle"><span class="NLM_cas:atitle">Macrocyclic Modalities Combining Peptide Epitopes and Natural Product Fragments</span></div><div class="casAuthors">Gueret, Stephanie M.; Thavam, Sasikala; Carbajo, Rodrigo J.; Potowski, Marco; Larsson, Niklas; Dahl, Goeran; Dellsen, Anita; Grossmann, Tom N.; Plowright, Alleyn T.; Valeur, Eric; Lemurell, Malin; Waldmann, Herbert</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">142</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4904-4915</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">"Hot loop" protein segments have variable structure and conformation and contribute crucially to protein-protein interactions.  We describe a new hot loop mimicking modality, termed PepNats, in which natural product (NP)-inspired structures are incorporated as conformation-detg. and -restricting structural elements into macrocyclic hot loop-derived peptides.  Macrocyclic PepNats representing hot loops of inducible nitric oxide synthase (iNOS) and human agouti-related protein (AGRP) were synthesized on solid support employing macrocyclization by imine formation and subsequent stereoselective 1,3-dipolar cycloaddn. as key steps.  PepNats derived from the iNOS DINNN hot loop and the AGRP RFF hot spot sequence yielded novel and potent ligands of the SPRY domain-contg. SOCS box protein 2 (SPSB2) that binds to iNOS, and selective ligands for AGRP-binding melanocortin (MC) receptors.  NP-inspired fragment abs. configuration dets. the conformation of the peptide part responsible for binding.  These results demonstrate that combination of NP-inspired scaffolds with peptidic epitopes enables identification of novel hot loop mimics with conformationally constrained and biol. relevant structure.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrMaFk_qnRn1LVg90H21EOLACvtfcHk0lgOHNDPrWyKdQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXjtFGitLk%253D&md5=fa5baf213b94ee1adfe05df41b2330af</span></div><a href="/servlet/linkout?suffix=cit95&amp;dbid=16384&amp;doi=10.1021%2Fjacs.0c00269&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.0c00269%26sid%3Dliteratum%253Aachs%26aulast%3DGu%25C3%25A9ret%26aufirst%3DS.%2BM.%26aulast%3DThavam%26aufirst%3DS.%26aulast%3DCarbajo%26aufirst%3DR.%2BJ.%26aulast%3DPotowski%26aufirst%3DM.%26aulast%3DLarsson%26aufirst%3DN.%26aulast%3DDahl%26aufirst%3DG.%26aulast%3DDells%25C3%25A9n%26aufirst%3DA.%26aulast%3DGrossmann%26aufirst%3DT.%2BN.%26aulast%3DPlowright%26aufirst%3DA.%2BT.%26aulast%3DValeur%26aufirst%3DE.%26aulast%3DLemurell%26aufirst%3DM.%26aulast%3DWaldmann%26aufirst%3DH.%26atitle%3DMacrocyclic%2520Modalities%2520Combining%2520Peptide%2520Epitopes%2520and%2520Natural%2520Product%2520Fragments%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2020%26volume%3D142%26spage%3D4904%26epage%3D4915%26doi%3D10.1021%2Fjacs.0c00269" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref96"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref96'); return false;" data-citation="" class="refNumLink">96</a></strong><div class="NLM_citation" id="cit96"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mulligan, V. K.</span></span> <span> </span><span class="NLM_article-title">The Emerging Role of Computational Design in Peptide Macrocycle Drug Discovery</span>. <i>Expert Opin. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">833</span>– <span class="NLM_lpage">852</span>, <span class="refDoi"> DOI: 10.1080/17460441.2020.1751117</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1080%2F17460441.2020.1751117" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=32345066" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BB3cXpsVKku7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2020&pages=833-852&author=V.+K.+Mulligan&title=The+Emerging+Role+of+Computational+Design+in+Peptide+Macrocycle+Drug+Discovery&doi=10.1080%2F17460441.2020.1751117"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit96R"><div class="casContent"><span class="casTitleNuber">96</span><div class="casTitle"><span class="NLM_cas:atitle">The emerging role of computational design in peptide macrocycle drug discovery</span></div><div class="casAuthors">Mulligan, Vikram K.</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Drug Discovery</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">833-852</span>CODEN:
                <span class="NLM_cas:coden">EODDBX</span>;
        ISSN:<span class="NLM_cas:issn">1746-0441</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">A review.  Drug discovery is a laborious process with rising cost per new drug.  Peptide macrocycles are promising therapeutics, though conformational flexibility can reduce target affinity and specificity.  Recent computational advancements address this problem by enabling rational design of rigidly folded peptide macrocycles.: This review summarizes currently approved peptide macrocycle therapeutics and discusses advantages of mesoscale drugs over small mols. or protein therapeutics.  It describes the history, rationale, and state of the art of computational tools, such as Rosetta, that allow the design of rigidly structured peptide macrocycles.  The emerging pipeline for designing peptide macrocycle drugs is described, including current challenges in designing permeable mols. that can emulate the chameleonic behavior of natural macrocycles.  Prospects for reducing computational cost and improving accuracy with emerging computational technologies are also discussed.: To embrace computational design of peptide macrocycle drugs, we must shift current attitudes regarding the role of computation in drug discovery, and move beyond Lipinski's rules.  This technol. has the potential to shift failures to earlier in silico stages of the drug discovery process, improving success rates in costly clin. trials.  Given the available tools, now is the time for drug developers to incorporate peptide macrocycle design into drug discovery pipelines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5iQ_xrAILPLVg90H21EOLACvtfcHk0lh1cByszQ1TIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXpsVKku7s%253D&md5=adee56703724078263c0aad5ff39f305</span></div><a href="/servlet/linkout?suffix=cit96&amp;dbid=16384&amp;doi=10.1080%2F17460441.2020.1751117&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F17460441.2020.1751117%26sid%3Dliteratum%253Aachs%26aulast%3DMulligan%26aufirst%3DV.%2BK.%26atitle%3DThe%2520Emerging%2520Role%2520of%2520Computational%2520Design%2520in%2520Peptide%2520Macrocycle%2520Drug%2520Discovery%26jtitle%3DExpert%2520Opin.%2520Drug%2520Discovery%26date%3D2020%26volume%3D15%26spage%3D833%26epage%3D852%26doi%3D10.1080%2F17460441.2020.1751117" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref97"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref97'); return false;" data-citation="" class="refNumLink">97</a></strong><div class="NLM_citation" id="cit97"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bonnet, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agrafiotis, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, E.</span></span> <span> </span><span class="NLM_article-title">Conformational Analysis of Macrocycles: Finding What Common Search Methods Miss</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">2242</span>– <span class="NLM_lpage">2259</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2013.10.003</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci900238a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1ahtbnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2009&pages=2242-2259&author=P.+Bonnetauthor=D.+K.+Agrafiotisauthor=F.+Zhuauthor=E.+Martin&title=Conformational+Analysis+of+Macrocycles%3A+Finding+What+Common+Search+Methods+Miss&doi=10.1016%2Fj.bmc.2013.10.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit97R"><div class="casContent"><span class="casTitleNuber">97</span><div class="casTitle"><span class="NLM_cas:atitle">Conformational Analysis of Macrocycles: Finding What Common Search Methods Miss</span></div><div class="casAuthors">Bonnet, Pascal; Agrafiotis, Dimitris K.; Zhu, Fangqiang; Martin, Eric</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2242-2259</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">As computational drug design becomes increasingly reliant on virtual screening and on high-throughput 3D modeling, the need for fast, robust, and reliable methods for sampling mol. conformations has become greater than ever.  Furthermore, chem. novelty is at a premium, forcing medicinal chemists to explore more complex structural motifs and unusual topologies.  This necessitates the use of conformational sampling techniques that work well in all cases.  Here, we compare the performance of several popular conformational search algorithms on three broad classes of macrocyclic mols.  These methods include Catalyst, CAESAR, MacroModel, MOE, Omega, Rubicon and two newer self-organizing algorithms known as stochastic proximity embedding (SPE) and self-organizing superimposition (SOS) that have been developed at Johnson & Johnson.  Our results show a compelling advantage for the three distance geometry methods (SOS, SPE, and Rubicon) followed to a lesser extent by MacroModel.  The remaining techniques, particularly those based on systematic search, often failed to identify any of the lowest energy conformations and are unsuitable for this class of structures.  Taken together with our previous study on drug-like mols., these results suggest that SPE and SOS are two of the most robust and universally applicable conformational search methods, with the latter being preferred because of its superior speed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhIMu1b7U477Vg90H21EOLACvtfcHk0lh1cByszQ1TIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht1ahtbnF&md5=9b4a5919415ef0660c4f9ff18d9b76f4</span></div><a href="/servlet/linkout?suffix=cit97&amp;dbid=16384&amp;doi=10.1021%2Fci900238a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci900238a%26sid%3Dliteratum%253Aachs%26aulast%3DBonnet%26aufirst%3DP.%26aulast%3DAgrafiotis%26aufirst%3DD.%2BK.%26aulast%3DZhu%26aufirst%3DF.%26aulast%3DMartin%26aufirst%3DE.%26atitle%3DConformational%2520Analysis%2520of%2520Macrocycles%253A%2520Finding%2520What%2520Common%2520Search%2520Methods%2520Miss%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2009%26volume%3D49%26spage%3D2242%26epage%3D2259%26doi%3D10.1016%2Fj.bmc.2013.10.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref98"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref98'); return false;" data-citation="" class="refNumLink">98</a></strong><div class="NLM_citation" id="cit98"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, I.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foloppe, N.</span></span> <span> </span><span class="NLM_article-title">Tackling the Conformational Sampling of Larger Flexible Compounds and Macrocycles in Pharmacology and Drug Discovery</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">7898</span>– <span class="NLM_lpage">7920</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2013.10.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1016%2Fj.bmc.2013.10.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=24184215" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslWltrfK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2013&pages=7898-7920&author=I.-J.+Chenauthor=N.+Foloppe&title=Tackling+the+Conformational+Sampling+of+Larger+Flexible+Compounds+and+Macrocycles+in+Pharmacology+and+Drug+Discovery&doi=10.1016%2Fj.bmc.2013.10.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit98R"><div class="casContent"><span class="casTitleNuber">98</span><div class="casTitle"><span class="NLM_cas:atitle">Tackling the conformational sampling of larger flexible compounds and macrocycles in pharmacology and drug discovery</span></div><div class="casAuthors">Chen, I-Jen; Foloppe, Nicolas</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">7898-7920</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Computational conformational sampling underpins much of mol. modeling and design in pharmaceutical work.  The sampling of smaller drug-like compds. has been an active area of research.  However, few studies have tested in details the sampling of larger more flexible compds., which are also relevant to drug discovery, including therapeutic peptides, macrocycles, and inhibitors of protein-protein interactions.  Here, we investigate extensively mainstream conformational sampling methods on three carefully curated compd. sets, namely the 'Drug-like', larger 'Flexible', and 'Macrocycle' compds.  These test mols. are chem. diverse with reliable X-ray protein-bound bioactive structures.  The compared sampling methods include Stochastic Search and the recent LowModeMD from MOE, all the low-mode based approaches from MacroModel, and MD/LLMOD recently developed for macrocycles.  In addn. to default settings, key parameters of the sampling protocols were explored.  The performance of the computational protocols was assessed via (i) the reprodn. of the X-ray bioactive structures, (ii) the size, coverage and diversity of the output conformational ensembles, (iii) the compactness/extendedness of the conformers, and (iv) the ability to locate the global energy min.  The influence of the stochastic nature of the searches on the results was also examd.  Much better results were obtained by adopting search parameters enhanced over the default settings, while maintaining computational tractability.  In MOE, the recent LowModeMD emerged as the method of choice.  Mixed torsional/low-mode from MacroModel performed as well as LowModeMD, and MD/LLMOD performed well for macrocycles.  The low-mode based approaches yielded very encouraging results with the flexible and macrocycle sets.  Thus, one can productively tackle the computational conformational search of larger flexible compds. for drug discovery, including macrocycles.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovwhGIpE1mMbVg90H21EOLACvtfcHk0lh1cByszQ1TIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslWltrfK&md5=2699227e69410a5673dec749169fd578</span></div><a href="/servlet/linkout?suffix=cit98&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2013.10.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2013.10.003%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DI.-J.%26aulast%3DFoloppe%26aufirst%3DN.%26atitle%3DTackling%2520the%2520Conformational%2520Sampling%2520of%2520Larger%2520Flexible%2520Compounds%2520and%2520Macrocycles%2520in%2520Pharmacology%2520and%2520Drug%2520Discovery%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2013%26volume%3D21%26spage%3D7898%26epage%3D7920%26doi%3D10.1016%2Fj.bmc.2013.10.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref99"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref99'); return false;" data-citation="" class="refNumLink">99</a></strong><div class="NLM_citation" id="cit99"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Coutsias, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lexa, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wester, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollock, S. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, M. P.</span></span> <span> </span><span class="NLM_article-title">Exhaustive Conformational Sampling of Complex Fused Ring Macrocycles Using Inverse Kinematics</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">4674</span>– <span class="NLM_lpage">4687</span>, <span class="refDoi"> DOI: 10.1021/acs.jctc.6b00250</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jctc.6b00250" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1WntLjI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=4674-4687&author=E.+A.+Coutsiasauthor=K.+W.+Lexaauthor=M.+J.+Westerauthor=S.+N.+Pollockauthor=M.+P.+Jacobson&title=Exhaustive+Conformational+Sampling+of+Complex+Fused+Ring+Macrocycles+Using+Inverse+Kinematics&doi=10.1021%2Facs.jctc.6b00250"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit99R"><div class="casContent"><span class="casTitleNuber">99</span><div class="casTitle"><span class="NLM_cas:atitle">Exhaustive Conformational Sampling of Complex Fused Ring Macrocycles Using Inverse Kinematics</span></div><div class="casAuthors">Coutsias, Evangelos A.; Lexa, Katrina W.; Wester, Michael J.; Pollock, Sara N.; Jacobson, Matthew P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4674-4687</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Natural product and synthetic macrocycles are chem. and topol. diverse.  An efficient, accurate, and general method for generating macrocycle conformations would enable structure-based design of macrocycle drugs or host-guest complexes.  Computational sampling also provides insight into transiently populated states, complementing crystallog. and NMR data.  Here, we report a new algorithm, BRIKARD, that addresses this challenge through computational algebraic geometry and inverse kinematics together with local energy minimization.  BRIKARD is demonstrated on 67 diverse macrocycles with structural data, encompassing various ring topologies.  We find this approach enumerates diverse structures with macrocyclic RMSD < 1.0 Å to the exptl. conformation for 85% of our data set in contrast to success rates of 67-75% with other approaches, while for the subset of 21 more challenging compds. in the data set, these rates are 57% and 10-29%, resp.  Because the algorithm can be efficiently run in parallel on many processors, exhaustive conformational sampling of complex cycles can be obtained in minutes rather than hours: with a 40 processor implementation protocol, BRIKARD samples the conformational diversity of a potential energy landscape in a median of 1.3 min of wallclock time, much faster than 3.1-10.3 h necessary with current programs.  By rigorously testing BRIKARD on a broad range of scaffolds with highly complex ring systems, we push the frontiers of macrocycle sampling to encompass multiring compds., including those with more than 50 ring atoms and up to seven interlaced flexible rings.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodPfQ1Rj98zLVg90H21EOLACvtfcHk0lhczVKnRDqE5g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1WntLjI&md5=75ba39a0489ad676e5e53c59931d1391</span></div><a href="/servlet/linkout?suffix=cit99&amp;dbid=16384&amp;doi=10.1021%2Facs.jctc.6b00250&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jctc.6b00250%26sid%3Dliteratum%253Aachs%26aulast%3DCoutsias%26aufirst%3DE.%2BA.%26aulast%3DLexa%26aufirst%3DK.%2BW.%26aulast%3DWester%26aufirst%3DM.%2BJ.%26aulast%3DPollock%26aufirst%3DS.%2BN.%26aulast%3DJacobson%26aufirst%3DM.%2BP.%26atitle%3DExhaustive%2520Conformational%2520Sampling%2520of%2520Complex%2520Fused%2520Ring%2520Macrocycles%2520Using%2520Inverse%2520Kinematics%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2016%26volume%3D12%26spage%3D4674%26epage%3D4687%26doi%3D10.1021%2Facs.jctc.6b00250" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref100"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref100'); return false;" data-citation="" class="refNumLink">100</a></strong><div class="NLM_citation" id="cit100"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Geng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.-D.</span></span> <span> </span><span class="NLM_article-title">Accurate Structure Prediction and Conformational Analysis of Cyclic Peptides with Residue-Specific Force Fields</span>. <i>J. Phys. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">1805</span>– <span class="NLM_lpage">1810</span>, <span class="refDoi"> DOI: 10.1021/acs.jpclett.6b00452</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jpclett.6b00452" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC28XntFCiu74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=1805-1810&author=H.+Gengauthor=F.+Jiangauthor=Y.-D.+Wu&title=Accurate+Structure+Prediction+and+Conformational+Analysis+of+Cyclic+Peptides+with+Residue-Specific+Force+Fields&doi=10.1021%2Facs.jpclett.6b00452"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit100R"><div class="casContent"><span class="casTitleNuber">100</span><div class="casTitle"><span class="NLM_cas:atitle">Accurate Structure Prediction and Conformational Analysis of Cyclic Peptides with Residue-Specific Force Fields</span></div><div class="casAuthors">Geng, Hao; Jiang, Fan; Wu, Yun-Dong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry Letters</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1805-1810</span>CODEN:
                <span class="NLM_cas:coden">JPCLCD</span>;
        ISSN:<span class="NLM_cas:issn">1948-7185</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Cyclic peptides (CPs) are promising candidates for drugs, chem. biol. tools, and self-assembling nanomaterials.  However, the development of reliable and accurate computational methods for their structure prediction has been challenging.  Here, 20 all-trans CPs of 5-12 residues selected from Cambridge Structure Database have been simulated using replica-exchange mol. dynamics with four different force fields.  The authors' recently developed residue-specific force fields RSFF1 and RSFF2 can correctly identify the crystal-like conformations of more than half CPs as the most populated conformation.  The RSFF2 performs the best, which consistently predicts the crystal structures of 17 out of 20 CPs with RMSD < 1.1 Å.  The authors also compared the backbone (φ,ψ) sampling of residues in CPs which those in short linear peptides and in globular proteins.  In general, unlike linear peptides, CPs have local conformational free energies and entropies quite similar to globular proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGooeUP-aaOZCrVg90H21EOLACvtfcHk0lhczVKnRDqE5g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XntFCiu74%253D&md5=d9c4be040be98396c50ae44a2fc97fbb</span></div><a href="/servlet/linkout?suffix=cit100&amp;dbid=16384&amp;doi=10.1021%2Facs.jpclett.6b00452&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jpclett.6b00452%26sid%3Dliteratum%253Aachs%26aulast%3DGeng%26aufirst%3DH.%26aulast%3DJiang%26aufirst%3DF.%26aulast%3DWu%26aufirst%3DY.-D.%26atitle%3DAccurate%2520Structure%2520Prediction%2520and%2520Conformational%2520Analysis%2520of%2520Cyclic%2520Peptides%2520with%2520Residue-Specific%2520Force%2520Fields%26jtitle%3DJ.%2520Phys.%2520Chem.%2520Lett.%26date%3D2016%26volume%3D7%26spage%3D1805%26epage%3D1810%26doi%3D10.1021%2Facs.jpclett.6b00452" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref101"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref101'); return false;" data-citation="" class="refNumLink">101</a></strong><div class="NLM_citation" id="cit101"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gil-Ley, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bussi, G.</span></span> <span> </span><span class="NLM_article-title">Enhanced Conformational Sampling Using Replica Exchange with Collective-Variable Tempering</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">1077</span>– <span class="NLM_lpage">1085</span>, <span class="refDoi"> DOI: 10.1021/ct5009087</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct5009087" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitlKrsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=1077-1085&author=A.+Gil-Leyauthor=G.+Bussi&title=Enhanced+Conformational+Sampling+Using+Replica+Exchange+with+Collective-Variable+Tempering&doi=10.1021%2Fct5009087"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit101R"><div class="casContent"><span class="casTitleNuber">101</span><div class="casTitle"><span class="NLM_cas:atitle">Enhanced Conformational Sampling Using Replica Exchange with Collective-Variable Tempering</span></div><div class="casAuthors">Gil-Ley, Alejandro; Bussi, Giovanni</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1077-1085</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The computational study of conformational transitions in RNA and proteins with atomistic mol. dynamics often requires suitable enhanced sampling techniques.  The authors here introduce a novel method where concurrent metadynamics are integrated in a Hamiltonian replica-exchange scheme.  The ladder of replicas is built with different strengths of the bias potential exploiting the tunability of well-tempered metadynamics.  Using this method, free-energy barriers of individual collective variables are significantly reduced compared with simple force-field scaling.  The introduced methodol. is flexible and allows adaptive bias potentials to be self-consistently constructed for a large no. of simple collective variables, such as distances and dihedral angles.  The method is tested on alanine dipeptide and applied to the difficult problem of conformational sampling in a tetranucleotide.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp5umzwBtWIALVg90H21EOLACvtfcHk0lhczVKnRDqE5g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitlKrsr8%253D&md5=e22aee8989f57a9d01a6f111cded9b5c</span></div><a href="/servlet/linkout?suffix=cit101&amp;dbid=16384&amp;doi=10.1021%2Fct5009087&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct5009087%26sid%3Dliteratum%253Aachs%26aulast%3DGil-Ley%26aufirst%3DA.%26aulast%3DBussi%26aufirst%3DG.%26atitle%3DEnhanced%2520Conformational%2520Sampling%2520Using%2520Replica%2520Exchange%2520with%2520Collective-Variable%2520Tempering%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2015%26volume%3D11%26spage%3D1077%26epage%3D1085%26doi%3D10.1021%2Fct5009087" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref102"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref102'); return false;" data-citation="" class="refNumLink">102</a></strong><div class="NLM_citation" id="cit102"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hosseinzadeh, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhardwaj, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulligan, V. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shortridge, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craven, T. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pardo-Avila, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rettie, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silva, D.-A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ibrahim, Y. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Webb, I. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cort, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adkins, J. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Varani, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baker, D.</span></span> <span> </span><span class="NLM_article-title">Comprehensive Computational Design of Ordered Peptide Macrocycles</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>358</i></span>,  <span class="NLM_fpage">1461</span>– <span class="NLM_lpage">1466</span>, <span class="refDoi"> DOI: 10.1126/science.aap7577</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1126%2Fscience.aap7577" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=29242347" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFGmtr3I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=358&publication_year=2017&pages=1461-1466&author=P.+Hosseinzadehauthor=G.+Bhardwajauthor=V.+K.+Mulliganauthor=M.+D.+Shortridgeauthor=T.+W.+Cravenauthor=F.+Pardo-Avilaauthor=S.+A.+Rettieauthor=D.+E.+Kimauthor=D.-A.+Silvaauthor=Y.+M.+Ibrahimauthor=I.+K.+Webbauthor=J.+R.+Cortauthor=J.+N.+Adkinsauthor=G.+Varaniauthor=D.+Baker&title=Comprehensive+Computational+Design+of+Ordered+Peptide+Macrocycles&doi=10.1126%2Fscience.aap7577"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit102R"><div class="casContent"><span class="casTitleNuber">102</span><div class="casTitle"><span class="NLM_cas:atitle">Comprehensive computational design of ordered peptide macrocycles</span></div><div class="casAuthors">Hosseinzadeh, Parisa; Bhardwaj, Gaurav; Mulligan, Vikram Khipple; Shortridge, Matthew D.; Craven, Timothy W.; Pardo-Avila, Fatima; Rettie, Stephen A.; Kim, David E.; Silva, Daniel-Adriano; Ibrahim, Yehia M.; Webb, Ian K.; Cort, John R.; Adkins, Joshua N.; Varani, Gabriele; Baker, David</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">358</span>
        (<span class="NLM_cas:issue">6369</span>),
    <span class="NLM_cas:pages">1461-1466</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Mixed-chirality peptide macrocycles such as cyclosporine are among the most potent therapeutics identified to date, but there is currently no way to systematically search the structural space spanned by such compds.  Natural proteins do not provide a useful guide: Peptide macrocycles lack regular secondary structures and hydrophobic cores, and can contain local structures not accessible with L-amino acids.  Here, we enumerate the stable structures that can be adopted by macrocyclic peptides composed of L- and D-amino acids by near-exhaustive backbone sampling followed by sequence design and energy landscape calcns.  We identify more than 200 designs predicted to fold into single stable structures, many times more than the no. of currently available unbound peptide macrocycle structures.  NMR structures of 9 of 12 designed 7- to 10-residue macrocycles, and three 11- to 14-residue bicyclic designs, are close to the computational models.  Our results provide a nearly complete coverage of the rich space of structures possible for short peptide macrocycles and vastly increase the available starting scaffolds for both rational drug design and library selection methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGorgJTppniqh7Vg90H21EOLACvtfcHk0lhczVKnRDqE5g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFGmtr3I&md5=fd13cbca720f657d91131e28e78fd435</span></div><a href="/servlet/linkout?suffix=cit102&amp;dbid=16384&amp;doi=10.1126%2Fscience.aap7577&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aap7577%26sid%3Dliteratum%253Aachs%26aulast%3DHosseinzadeh%26aufirst%3DP.%26aulast%3DBhardwaj%26aufirst%3DG.%26aulast%3DMulligan%26aufirst%3DV.%2BK.%26aulast%3DShortridge%26aufirst%3DM.%2BD.%26aulast%3DCraven%26aufirst%3DT.%2BW.%26aulast%3DPardo-Avila%26aufirst%3DF.%26aulast%3DRettie%26aufirst%3DS.%2BA.%26aulast%3DKim%26aufirst%3DD.%2BE.%26aulast%3DSilva%26aufirst%3DD.-A.%26aulast%3DIbrahim%26aufirst%3DY.%2BM.%26aulast%3DWebb%26aufirst%3DI.%2BK.%26aulast%3DCort%26aufirst%3DJ.%2BR.%26aulast%3DAdkins%26aufirst%3DJ.%2BN.%26aulast%3DVarani%26aufirst%3DG.%26aulast%3DBaker%26aufirst%3DD.%26atitle%3DComprehensive%2520Computational%2520Design%2520of%2520Ordered%2520Peptide%2520Macrocycles%26jtitle%3DScience%26date%3D2017%26volume%3D358%26spage%3D1461%26epage%3D1466%26doi%3D10.1126%2Fscience.aap7577" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref103"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref103'); return false;" data-citation="" class="refNumLink">103</a></strong><div class="NLM_citation" id="cit103"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kamenik, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lessel, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuchs, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fox, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liedl, K. R.</span></span> <span> </span><span class="NLM_article-title">Peptidic Macrocycles - Conformational Sampling and Thermodynamic Characterization</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">982</span>– <span class="NLM_lpage">992</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.8b00097</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.8b00097" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC1cXnsFantrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2018&pages=982-992&author=A.+S.+Kamenikauthor=U.+Lesselauthor=J.+E.+Fuchsauthor=T.+Foxauthor=K.+R.+Liedl&title=Peptidic+Macrocycles+-+Conformational+Sampling+and+Thermodynamic+Characterization&doi=10.1021%2Facs.jcim.8b00097"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit103R"><div class="casContent"><span class="casTitleNuber">103</span><div class="casTitle"><span class="NLM_cas:atitle">Peptidic Macrocycles - Conformational Sampling and Thermodynamic Characterization</span></div><div class="casAuthors">Kamenik, Anna S.; Lessel, Uta; Fuchs, Julian E.; Fox, Thomas; Liedl, Klaus R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">982-992</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Macrocycles are of considerable interest as highly specific drug candidates, yet they challenge std. conformer generators with their large no. of rotatable bonds and conformational restrictions.  Here, we present a mol. dynamics-based routine that bypasses current limitations in conformational sampling and extensively profiles the free energy landscape of peptidic macrocycles in soln.  We perform accelerated mol. dynamics simulations to capture a diverse conformational ensemble.  By applying an energetic cutoff, followed by geometric clustering, we demonstrate the striking robustness and efficiency of the approach in identifying highly populated conformational states of cyclic peptides.  The resulting structural and thermodn. information is benchmarked against interproton distances from NMR expts. and conformational states identified by X-ray crystallog.  Using three different model systems of varying size and flexibility, we show that the method reliably reproduces exptl. detd. structural ensembles and is capable of identifying key conformational states that include the bioactive conformation.  Thus, the described approach is a robust method to generate conformations of peptidic macrocycles and holds promise for structure-based drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqFlE0cJ41tmrVg90H21EOLACvtfcHk0lhFLl9A4V8h2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXnsFantrY%253D&md5=4c582a4036f17ac76c05d3e9e8718c02</span></div><a href="/servlet/linkout?suffix=cit103&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.8b00097&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.8b00097%26sid%3Dliteratum%253Aachs%26aulast%3DKamenik%26aufirst%3DA.%2BS.%26aulast%3DLessel%26aufirst%3DU.%26aulast%3DFuchs%26aufirst%3DJ.%2BE.%26aulast%3DFox%26aufirst%3DT.%26aulast%3DLiedl%26aufirst%3DK.%2BR.%26atitle%3DPeptidic%2520Macrocycles%2520-%2520Conformational%2520Sampling%2520and%2520Thermodynamic%2520Characterization%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2018%26volume%3D58%26spage%3D982%26epage%3D992%26doi%3D10.1021%2Facs.jcim.8b00097" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref104"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref104'); return false;" data-citation="" class="refNumLink">104</a></strong><div class="NLM_citation" id="cit104"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Labute, P.</span></span> <span> </span><span class="NLM_article-title">LowModeMD—Implicit Low-Mode Velocity Filtering Applied to Conformational Search of Macrocycles and Protein Loops</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">792</span>– <span class="NLM_lpage">800</span>, <span class="refDoi"> DOI: 10.1021/ci900508k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci900508k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref104/cit104&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC3cXlsVSksrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=792-800&author=P.+Labute&title=LowModeMD%E2%80%94Implicit+Low-Mode+Velocity+Filtering+Applied+to+Conformational+Search+of+Macrocycles+and+Protein+Loops&doi=10.1021%2Fci900508k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit104R"><div class="casContent"><span class="casTitleNuber">104</span><div class="casTitle"><span class="NLM_cas:atitle">LowModeMD - Implicit Low-Mode Velocity Filtering Applied to Conformational Search of Macrocycles and Protein Loops</span></div><div class="casAuthors">Labute, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">792-800</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We present a method for conformational search of complex mol. systems such as macrocycles and protein loops.  The method is based on perturbing an existing conformation along a mol. dynamics trajectory using initial at. velocities with kinetic energy concd. on the low-frequency vibrational modes, followed by energy minimization.  A novel Chebyshev polynomial filter is used to heavily dampen the high-frequency components of a randomly generated Maxwell-Boltzmann velocity vector.  The method is very efficient, even for large systems; it is straightforward to implement and requires only std. force-field energy and gradient evaluations.  The results of several computational expts. suggest that the method is capable of efficiently sampling low-strain energy conformations of complex systems with nontrivial nonbonded interaction networks.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfWZjZhFqqUbVg90H21EOLACvtfcHk0lhFLl9A4V8h2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXlsVSksrk%253D&md5=bb800eb2f9635a661578b978dae9da0a</span></div><a href="/servlet/linkout?suffix=cit104&amp;dbid=16384&amp;doi=10.1021%2Fci900508k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci900508k%26sid%3Dliteratum%253Aachs%26aulast%3DLabute%26aufirst%3DP.%26atitle%3DLowModeMD%25E2%2580%2594Implicit%2520Low-Mode%2520Velocity%2520Filtering%2520Applied%2520to%2520Conformational%2520Search%2520of%2520Macrocycles%2520and%2520Protein%2520Loops%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D792%26epage%3D800%26doi%3D10.1021%2Fci900508k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref105"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref105'); return false;" data-citation="" class="refNumLink">105</a></strong><div class="NLM_citation" id="cit105"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Olanders, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alogheli, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brandt, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karlén, A.</span></span> <span> </span><span class="NLM_article-title">Conformational Analysis of Macrocycles: Comparing General and Specialized Methods</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">231</span>– <span class="NLM_lpage">252</span>, <span class="refDoi"> DOI: 10.1007/s10822-020-00277-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref105/cit105&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1007%2Fs10822-020-00277-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref105/cit105&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=31965404" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref105/cit105&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BB3cXis1Sgtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2020&pages=231-252&author=G.+Olandersauthor=H.+Alogheliauthor=P.+Brandtauthor=A.+Karl%C3%A9n&title=Conformational+Analysis+of+Macrocycles%3A+Comparing+General+and+Specialized+Methods&doi=10.1007%2Fs10822-020-00277-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit105R"><div class="casContent"><span class="casTitleNuber">105</span><div class="casTitle"><span class="NLM_cas:atitle">Conformational analysis of macrocycles: comparing general and specialized methods</span></div><div class="casAuthors">Olanders, Gustav; Alogheli, Hiba; Brandt, Peter; Karlen, Anders</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">231-252</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Using macrocycles extd. from 44 macrocycle-protein X-ray crystallog. complexes, we evaluated each method based on their ability to (i) generate unique conformers, (ii) generate unique macrocycle ring conformations, (iii) identify global energy min., (iv) identify conformers similar to X-ray ligand conformation after Protein Prepn. Wizard treatment and to X-rayppw ring conformation.  In order to study relative energies of bioactive conformations, energy differences between global energy min. and energy minimized X-rayppw structures and global energy min. and MCMM-Exhaustive generated conformers closest to the X-rayppw structure, were calcd. and analyzed.  To assess the performance of the methods, they were compared to exhaustive MCMM search using 1,000,000 search steps for each of 44 macrocycles.  Prior to our anal., we also investigated if general search methods MCMM and MTLMOD could also be optimized for macrocycle conformational sampling.  Taken together, our work concludes that more general methods can be optimized for macrocycle modeling by slightly adjusting settings around ring closure bond.  In most cases, MCMM and MTLMOD with either std. or enhanced settings performed well in comparison to more specialized macrocycle sampling methods MD/LLMOD and PRIME-MCS.  MD/LLMOD emerged as most efficient method for generating global energy min.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpw7Rdc4sIY-LVg90H21EOLACvtfcHk0lhFLl9A4V8h2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXis1Sgtr4%253D&md5=4df4e45f11974acae7695919de562574</span></div><a href="/servlet/linkout?suffix=cit105&amp;dbid=16384&amp;doi=10.1007%2Fs10822-020-00277-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-020-00277-2%26sid%3Dliteratum%253Aachs%26aulast%3DOlanders%26aufirst%3DG.%26aulast%3DAlogheli%26aufirst%3DH.%26aulast%3DBrandt%26aufirst%3DP.%26aulast%3DKarl%25C3%25A9n%26aufirst%3DA.%26atitle%3DConformational%2520Analysis%2520of%2520Macrocycles%253A%2520Comparing%2520General%2520and%2520Specialized%2520Methods%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2020%26volume%3D34%26spage%3D231%26epage%3D252%26doi%3D10.1007%2Fs10822-020-00277-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref106"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref106'); return false;" data-citation="" class="refNumLink">106</a></strong><div class="NLM_citation" id="cit106"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sindhikara, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spronk, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Day, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borrelli, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheney, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Posy, S. L.</span></span> <span> </span><span class="NLM_article-title">Improving Accuracy, Diversity, and Speed with Prime Macrocycle Conformational Sampling</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">1881</span>– <span class="NLM_lpage">1894</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.7b00052</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.7b00052" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref106/cit106&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtF2ksbrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2017&pages=1881-1894&author=D.+Sindhikaraauthor=S.+A.+Spronkauthor=T.+Dayauthor=K.+Borrelliauthor=D.+L.+Cheneyauthor=S.+L.+Posy&title=Improving+Accuracy%2C+Diversity%2C+and+Speed+with+Prime+Macrocycle+Conformational+Sampling&doi=10.1021%2Facs.jcim.7b00052"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit106R"><div class="casContent"><span class="casTitleNuber">106</span><div class="casTitle"><span class="NLM_cas:atitle">Improving Accuracy, Diversity, and Speed with Prime Macrocycle Conformational Sampling</span></div><div class="casAuthors">Sindhikara, Dan; Spronk, Steven A.; Day, Tyler; Borrelli, Ken; Cheney, Daniel L.; Posy, Shana L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1881-1894</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A novel method for exploring macrocycle conformational space, Prime Macrocycle Conformational Sampling (Prime-MCS), is introduced and evaluated in the context of other available algorithms (Mol. Dynamics, LowModeMD in MOE, and MacroModel Baseline Search).  The algorithms were benchmarked on a dataset of 208 macrocycles which was curated for diversity from the Cambridge Structural Database, the Protein Databank, and the Biol. Interesting Mol. Ref. Dictionary.  The algorithms were evaluated in terms of accuracy (ability to reproduce the crystal structure), diversity (coverage of conformational space), and computational speed.  Prime-MCS most reliably reproduced crystallog. structures for RMSD thresholds >1.0 Å, most often produced the most diverse conformational ensemble, and was most often the fastest algorithm.  Detailed anal. and examn. of both typical and outlier cases were performed to reveal characteristics, shortcomings, expected performance, and complementarity of the methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfeDx3IcpZD7Vg90H21EOLACvtfcHk0ljYb87IFIgxdw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtF2ksbrJ&md5=a2b20d25c2ad2cede1c3a5fd0b5840ce</span></div><a href="/servlet/linkout?suffix=cit106&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.7b00052&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.7b00052%26sid%3Dliteratum%253Aachs%26aulast%3DSindhikara%26aufirst%3DD.%26aulast%3DSpronk%26aufirst%3DS.%2BA.%26aulast%3DDay%26aufirst%3DT.%26aulast%3DBorrelli%26aufirst%3DK.%26aulast%3DCheney%26aufirst%3DD.%2BL.%26aulast%3DPosy%26aufirst%3DS.%2BL.%26atitle%3DImproving%2520Accuracy%252C%2520Diversity%252C%2520and%2520Speed%2520with%2520Prime%2520Macrocycle%2520Conformational%2520Sampling%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2017%26volume%3D57%26spage%3D1881%26epage%3D1894%26doi%3D10.1021%2Facs.jcim.7b00052" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref107"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref107'); return false;" data-citation="" class="refNumLink">107</a></strong><div class="NLM_citation" id="cit107"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sindhikara, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borrelli, K.</span></span> <span> </span><span class="NLM_article-title">High Throughput Evaluation of Macrocyclization Strategies for Conformer Stabilization</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">6585</span> <span class="refDoi"> DOI: 10.1038/s41598-018-24766-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1038%2Fs41598-018-24766-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=29700331" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A280%3ADC%252BC1MjntFyksg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2018&author=D.+Sindhikaraauthor=K.+Borrelli&title=High+Throughput+Evaluation+of+Macrocyclization+Strategies+for+Conformer+Stabilization&doi=10.1038%2Fs41598-018-24766-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit107R"><div class="casContent"><span class="casTitleNuber">107</span><div class="casTitle"><span class="NLM_cas:atitle">High throughput evaluation of macrocyclization strategies for conformer stabilization</span></div><div class="casAuthors">Sindhikara Dan; Borrelli Ken</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">6585</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">While macrocyclization of a linear compound to stabilize a known bioactive conformation can be a useful strategy to increase binding potency, the difficulty of macrocycle synthesis can limit the throughput of such strategies.  Thus computational techniques may offer the higher throughput required to screen large numbers of compounds.  Here we introduce a method for evaluating the propensity of a macrocyclic compound to adopt a conformation similar that of a known active linear compound in the binding site.  This method can be used as a fast screening tool for prioritizing macrocycles by leveraging the assumption that the propensity for the known bioactive substructural conformation relates to the affinity.  While this method cannot to identify new interactions not present in the known linear compound, it could quickly differentiate compounds where the three dimensional geometries imposed by the macrocyclization prevent adoption of conformations with the same contacts as the linear compound in their conserved region.  Here we report the implementation of this method using an RMSD-based structural descriptor and a Boltzmann-weighted propensity calculation and apply it retrospectively to three macrocycle linker optimization design projects.  We found the method performs well in terms of prioritizing more potent compounds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSJc7XEAu3kJ9yzdcP07UmufW6udTcc2eZIKq48OFea-7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1MjntFyksg%253D%253D&md5=a2391f1e89175d1663342c8919e24743</span></div><a href="/servlet/linkout?suffix=cit107&amp;dbid=16384&amp;doi=10.1038%2Fs41598-018-24766-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-018-24766-5%26sid%3Dliteratum%253Aachs%26aulast%3DSindhikara%26aufirst%3DD.%26aulast%3DBorrelli%26aufirst%3DK.%26atitle%3DHigh%2520Throughput%2520Evaluation%2520of%2520Macrocyclization%2520Strategies%2520for%2520Conformer%2520Stabilization%26jtitle%3DSci.%2520Rep.%26date%3D2018%26volume%3D8%26doi%3D10.1038%2Fs41598-018-24766-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref108"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref108'); return false;" data-citation="" class="refNumLink">108</a></strong><div class="NLM_citation" id="cit108"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Watts, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dalal, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tebben, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheney, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shelley, J. C.</span></span> <span> </span><span class="NLM_article-title">Macrocycle Conformational Sampling with MacroModel</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">2680</span>– <span class="NLM_lpage">2696</span>, <span class="refDoi"> DOI: 10.1021/ci5001696</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci5001696" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref108/cit108&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsFKnsbbN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2014&pages=2680-2696&author=K.+S.+Wattsauthor=P.+Dalalauthor=A.+J.+Tebbenauthor=D.+L.+Cheneyauthor=J.+C.+Shelley&title=Macrocycle+Conformational+Sampling+with+MacroModel&doi=10.1021%2Fci5001696"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit108R"><div class="casContent"><span class="casTitleNuber">108</span><div class="casTitle"><span class="NLM_cas:atitle">Macrocycle Conformational Sampling with MacroModel</span></div><div class="casAuthors">Watts, K. Shawn; Dalal, Pranav; Tebben, Andrew J.; Cheney, Daniel L.; Shelley, John C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2680-2696</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Sampling low energy conformations of macrocycles is challenging due to the large size of many of these mols. and the constraints imposed by the macrocycle.  We present a new conformational search method (implemented in MacroModel) that uses brief MD simulations followed by minimization and normal-mode search steps.  The method was parametrized using a set of 100 macrocycles from the PDB and CSD.  It was then tested on a publicly available data set for which there are published results using alternative methods; we found that when the same force field is used (in this case MMFFs in vacuum), our method tended to identify conformations with lower energies than what the other methods identified.  The performance on a new set of 50 macrocycles from the PDB and CSD was also quite good; the mean and median RMSD values for just the ring atoms were 0.60 and 0.33 Å, resp.  However, the RMSD values for macrocycles with more than 30 ring-atoms were quite a bit larger compared to the smaller macrocycles.  Possible origins for this and ideas for improving the performance on very large macrocycles are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqaBKXPDzoVcLVg90H21EOLACvtfcHk0ljYb87IFIgxdw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsFKnsbbN&md5=612c949e25807bbf6f850c2a2c37b821</span></div><a href="/servlet/linkout?suffix=cit108&amp;dbid=16384&amp;doi=10.1021%2Fci5001696&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci5001696%26sid%3Dliteratum%253Aachs%26aulast%3DWatts%26aufirst%3DK.%2BS.%26aulast%3DDalal%26aufirst%3DP.%26aulast%3DTebben%26aufirst%3DA.%2BJ.%26aulast%3DCheney%26aufirst%3DD.%2BL.%26aulast%3DShelley%26aufirst%3DJ.%2BC.%26atitle%3DMacrocycle%2520Conformational%2520Sampling%2520with%2520MacroModel%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2014%26volume%3D54%26spage%3D2680%26epage%3D2696%26doi%3D10.1021%2Fci5001696" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref109"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref109'); return false;" data-citation="" class="refNumLink">109</a></strong><div class="NLM_citation" id="cit109"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Witek, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keller, B. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blatter, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meissner, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riniker, S.</span></span> <span> </span><span class="NLM_article-title">Kinetic Models of Cyclosporin A in Polar and Apolar Environments Reveal Multiple Congruent Conformational States</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">1547</span>– <span class="NLM_lpage">1562</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.6b00251</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.6b00251" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFaisL7L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2016&pages=1547-1562&author=J.+Witekauthor=B.+G.+Kellerauthor=M.+Blatterauthor=A.+Meissnerauthor=T.+Wagnerauthor=S.+Riniker&title=Kinetic+Models+of+Cyclosporin+A+in+Polar+and+Apolar+Environments+Reveal+Multiple+Congruent+Conformational+States&doi=10.1021%2Facs.jcim.6b00251"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit109R"><div class="casContent"><span class="casTitleNuber">109</span><div class="casTitle"><span class="NLM_cas:atitle">Kinetic Models of Cyclosporin A in Polar and Apolar Environments Reveal Multiple Congruent Conformational States</span></div><div class="casAuthors">Witek, Jagna; Keller, Bettina G.; Blatter, Markus; Meissner, Axel; Wagner, Trixie; Riniker, Sereina</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1547-1562</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The membrane permeability of cyclic peptides and peptidomimetics, which are generally larger and more complex than typical drug mols., is likely strongly influenced by the conformational behavior of these compds. in polar and apolar environments.  The size and complexity of peptides often limit their bioavailability, but there are known examples of peptide natural products such as cyclosporin A (CsA) that can cross cell membranes by passive diffusion.  CsA is an undecapeptide with seven methylated backbone amides.  Its crystal structure shows a "closed" twisted β-pleated sheet conformation with four intramol. hydrogen bonds that is also obsd. in NMR measurements of CsA in chloroform.  When binding to its target cyclophilin, on the other hand, CsA adopts an "open" conformation without intramol. hydrogen bonds.  In this study, we attempted to sample the complete conformational space of CsA in chloroform and in water by mol. dynamics simulations in order to better understand its conformational behavior in these two environments and to rationalize the good membrane permeability of CsA obsd. exptl.  From 10 μs mol. dynamics simulations in each solvent, Markov state models were constructed to characterize the metastable conformational states.  The model in chloroform is compared to nuclear Overhauser effect NMR spectroscopy data reported in this study and taken from the literature.  The conformational landscapes in the two solvents show significant overlap but also clearly distinct features.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr618Uz1zHQkrVg90H21EOLACvtfcHk0ljYb87IFIgxdw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFaisL7L&md5=0274259c6380217a27c5e0f4649dbf6a</span></div><a href="/servlet/linkout?suffix=cit109&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.6b00251&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.6b00251%26sid%3Dliteratum%253Aachs%26aulast%3DWitek%26aufirst%3DJ.%26aulast%3DKeller%26aufirst%3DB.%2BG.%26aulast%3DBlatter%26aufirst%3DM.%26aulast%3DMeissner%26aufirst%3DA.%26aulast%3DWagner%26aufirst%3DT.%26aulast%3DRiniker%26aufirst%3DS.%26atitle%3DKinetic%2520Models%2520of%2520Cyclosporin%2520A%2520in%2520Polar%2520and%2520Apolar%2520Environments%2520Reveal%2520Multiple%2520Congruent%2520Conformational%2520States%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2016%26volume%3D56%26spage%3D1547%26epage%3D1562%26doi%3D10.1021%2Facs.jcim.6b00251" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref110"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref110'); return false;" data-citation="" class="refNumLink">110</a></strong><div class="NLM_citation" id="cit110"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Allen, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dokholyan, N. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bowers, A. A.</span></span> <span> </span><span class="NLM_article-title">Dynamic Docking of Conformationally Constrained Macrocycles: Methods and Applications</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">10</span>– <span class="NLM_lpage">24</span>, <span class="refDoi"> DOI: 10.1021/acschembio.5b00663</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b00663" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref110/cit110&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVCjtbbL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=10-24&author=S.+E.+Allenauthor=N.+V.+Dokholyanauthor=A.+A.+Bowers&title=Dynamic+Docking+of+Conformationally+Constrained+Macrocycles%3A+Methods+and+Applications&doi=10.1021%2Facschembio.5b00663"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit110R"><div class="casContent"><span class="casTitleNuber">110</span><div class="casTitle"><span class="NLM_cas:atitle">Dynamic Docking of Conformationally Constrained Macrocycles: Methods and Applications</span></div><div class="casAuthors">Allen, Scott E.; Dokholyan, Nikolay V.; Bowers, Albert A.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">10-24</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Many natural products consist of large and flexible macrocycles that engage their targets via multiple contact points.  This combination of contained flexibility and large contact area often allows natural products to bind at target surfaces rather than deep pockets, making them attractive scaffolds for inhibiting protein-protein interactions and other challenging therapeutic targets.  The increasing ability to manipulate such compds. either biosynthetically or via semisynthetic modification means that these compds. can now be considered as starting points for medchem campaigns rather than solely as ends.  Modern medchem benefits substantially from rational improvements made on the basis of mol. docking.  As such, docking methods have been enhanced in recent years to deal with the complicated binding modalities and flexible scaffolds of macrocyclic natural products and natural product-like structures.  Here, we comprehensively review methods for treating and docking these large macrocyclic scaffolds and discuss some of the resulting advances in medicinal chem.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8sOyiW4teIbVg90H21EOLACvtfcHk0ljW563zZRYd6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVCjtbbL&md5=21f4cbf0a53f70cd0e7b7a7ad88cd5be</span></div><a href="/servlet/linkout?suffix=cit110&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b00663&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b00663%26sid%3Dliteratum%253Aachs%26aulast%3DAllen%26aufirst%3DS.%2BE.%26aulast%3DDokholyan%26aufirst%3DN.%2BV.%26aulast%3DBowers%26aufirst%3DA.%2BA.%26atitle%3DDynamic%2520Docking%2520of%2520Conformationally%2520Constrained%2520Macrocycles%253A%2520Methods%2520and%2520Applications%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2016%26volume%3D11%26spage%3D10%26epage%3D24%26doi%3D10.1021%2Facschembio.5b00663" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref111"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref111'); return false;" data-citation="" class="refNumLink">111</a></strong><div class="NLM_citation" id="cit111"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Alogheli, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olanders, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaal, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brandt, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karlén, A.</span></span> <span> </span><span class="NLM_article-title">Docking of Macrocycles: Comparing Rigid and Flexible Docking in Glide</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">190</span>– <span class="NLM_lpage">202</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.6b00443</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.6b00443" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref111/cit111&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC2sXosF2htA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2017&pages=190-202&author=H.+Alogheliauthor=G.+Olandersauthor=W.+Schaalauthor=P.+Brandtauthor=A.+Karl%C3%A9n&title=Docking+of+Macrocycles%3A+Comparing+Rigid+and+Flexible+Docking+in+Glide&doi=10.1021%2Facs.jcim.6b00443"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit111R"><div class="casContent"><span class="casTitleNuber">111</span><div class="casTitle"><span class="NLM_cas:atitle">Docking of Macrocycles: Comparing Rigid and Flexible Docking in Glide</span></div><div class="casAuthors">Alogheli, Hiba; Olanders, Gustav; Schaal, Wesley; Brandt, Peter; Karlen, Anders</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">190-202</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In recent years, there has been an increased interest in using macrocyclic compds. for drug discovery and development.  For docking of these commonly large and flexible compds. to be addressed, a screening and a validation set were assembled from the PDB consisting of 16 and 31 macrocycle-contg. protein complexes, resp.  The macrocycles were docked in Glide by rigid docking of pregenerated conformational ensembles produced by the macrocycle conformational sampling method (MCS) in Schr.ovrddot.odinger Release 2015-3 or by direct Glide flexible docking after performing ring-templating.  The two protocols were compared to rigid docking of pregenerated conformational ensembles produced by an exhaustive Monte Carlo multiple min. (MCMM) conformational search and a shorter MCMM conformational search (MCMM-short).  The docking accuracy was evaluated and expressed as the RMSD between the heavy atoms of the ligand as found in the X-ray structure after refinement and the poses obtained by the docking protocols.  The median RMSD values for top-scored poses of the screening set were 0.83, 0.80, 0.88, and 0.58 Å for MCMM, MCMM-short, MCS, and Glide flexible docking, resp.  There was no statistically significant difference in the performance between rigid docking of pregenerated conformations produced by the MCS and direct docking using Glide flexible docking.  However, the flexible docking protocol was 2-times faster in docking the screening set compared to that of the MCS protocol.  In a final study, the new Prime-MCS method was evaluated in Schr.ovrddot.odinger Release 2016-3.  This method is faster compared that of to MCS; however, the conformations generated were found to be suboptimal for rigid docking.  Therefore, on the basis of timing, accuracy, and ease of set up, std. Glide flexible docking with prior ring-templating is recommended over current gold std. protocols using rigid docking of pregenerated conformational ensembles.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVm5kNKieXLLVg90H21EOLACvtfcHk0ljW563zZRYd6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXosF2htA%253D%253D&md5=2e27fa8f5247904089cf50cb443706c7</span></div><a href="/servlet/linkout?suffix=cit111&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.6b00443&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.6b00443%26sid%3Dliteratum%253Aachs%26aulast%3DAlogheli%26aufirst%3DH.%26aulast%3DOlanders%26aufirst%3DG.%26aulast%3DSchaal%26aufirst%3DW.%26aulast%3DBrandt%26aufirst%3DP.%26aulast%3DKarl%25C3%25A9n%26aufirst%3DA.%26atitle%3DDocking%2520of%2520Macrocycles%253A%2520Comparing%2520Rigid%2520and%2520Flexible%2520Docking%2520in%2520Glide%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2017%26volume%3D57%26spage%3D190%26epage%3D202%26doi%3D10.1021%2Facs.jcim.6b00443" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref112"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref112'); return false;" data-citation="" class="refNumLink">112</a></strong><div class="NLM_citation" id="cit112"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Martin, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, I.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, A. W. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foloppe, N.</span></span> <span> </span><span class="NLM_article-title">Modelling the Binding Mode of Macrocycles: Docking and Conformational Sampling</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>28</i></span>, <span class="NLM_elocation-id">115143</span> <span class="refDoi"> DOI: 10.1016/j.bmc.2019.115143</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref112/cit112&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1016%2Fj.bmc.2019.115143" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref112/cit112&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=31771798" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref112/cit112&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC1MXit1Krtb7K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2020&author=S.+J.+Martinauthor=I.-J.+Chenauthor=A.+W.+E.+Chanauthor=N.+Foloppe&title=Modelling+the+Binding+Mode+of+Macrocycles%3A+Docking+and+Conformational+Sampling&doi=10.1016%2Fj.bmc.2019.115143"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit112R"><div class="casContent"><span class="casTitleNuber">112</span><div class="casTitle"><span class="NLM_cas:atitle">Modelling the binding mode of macrocycles: Docking and conformational sampling</span></div><div class="casAuthors">Martin, Sarah J.; Chen, I-Jen; Chan, A. W. Edith; Foloppe, Nicolas</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">115143</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Drug discovery is increasingly tackling challenging protein binding sites regarding mol. recognition and druggability, including shallow and solvent-exposed protein-protein interaction interfaces.  Macrocycles are emerging as promising chemotypes to modulate such sites.  Despite their chem. complexity, macrocycles comprise important drugs and offer advantages compared to non-cyclic analogs, hence the recent impetus in the medicinal chem. of macrocycles.  Elaboration of macrocycles, or constituent fragments, can strongly benefit from knowledge of their binding mode to a target.  When such information from X-ray crystallog. is elusive, computational docking can provide working models.  However, few studies have explored docking protocols for macrocycles, since conventional docking methods struggle with the conformational complexity of macrocycles, and also potentially with the shallower topol. of their binding sites.  Indeed, macrocycle binding mode prediction with the mainstream docking software GOLD has hardly been explored.  Here, we present an in-depth study of macrocycle docking with GOLD and the ChemPLP scores.  First, we summarize the thorough curation of a test set of 41 protein-macrocycle X-ray structures, raising the issue of lattice contacts with such systems.  Rigid docking of the known bioactive conformers was successful (three top ranked poses) for 92.7% of the systems, in absence of crystallog. waters.  Thus, without conformational search issues, scoring performed well.  However, docking success dropped to 29.3% with the GOLD built-in conformational search.  Yet, the success rate doubled to 58.5% when GOLD was supplied with extensive conformer ensembles docked rigidly.  The reasons for failure, sampling or scoring, were analyzed, exemplified with particular cases.  Overall, binding mode prediction of macrocycles remains challenging, but can be much improved with tailored protocols.  The anal. of the interplay between conformational sampling and docking will be relevant to the prospective modeling of macrocycles in general.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoO93W5YR9RgrVg90H21EOLACvtfcHk0ljW563zZRYd6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXit1Krtb7K&md5=de91562a32d648806253408e126af8d1</span></div><a href="/servlet/linkout?suffix=cit112&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2019.115143&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2019.115143%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DS.%2BJ.%26aulast%3DChen%26aufirst%3DI.-J.%26aulast%3DChan%26aufirst%3DA.%2BW.%2BE.%26aulast%3DFoloppe%26aufirst%3DN.%26atitle%3DModelling%2520the%2520Binding%2520Mode%2520of%2520Macrocycles%253A%2520Docking%2520and%2520Conformational%2520Sampling%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2020%26volume%3D28%26doi%3D10.1016%2Fj.bmc.2019.115143" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref113"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref113'); return false;" data-citation="" class="refNumLink">113</a></strong><div class="NLM_citation" id="cit113"><span> <i>Glide Schrödinger Release 2019-3 Glide</i>; <span class="NLM_publisher-name">Schrödinger, LLC</span>: <span class="NLM_publisher-loc">New York,
NY</span>, <span class="NLM_year">2019</span> (accessed Jan 31, 2018).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Glide+Schr%C3%B6dinger+Release+2019-3+Glide%3B+Schr%C3%B6dinger%2C+LLC%3A+New+York%2C%0ANY%2C+2019+%28accessed+Jan+31%2C+2018%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit113&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26btitle%3DGlide%2520Schr%25C3%25B6dinger%2520Release%25202019-3%2520Glide%26pub%3DSchr%25C3%25B6dinger%252C%2520LLC%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref114"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref114'); return false;" data-citation="" class="refNumLink">114</a></strong><div class="NLM_citation" id="cit114"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ugur, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schroft, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marion, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glaser, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Antes, I.</span></span> <span> </span><span class="NLM_article-title">Predicting the Bioactive Conformations of Macrocycles: A Molecular Dynamics-Based Docking Procedure with DynaDock</span>. <i>J. Mol. Model.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>25</i></span>, <span class="NLM_elocation-id">197</span> <span class="refDoi"> DOI: 10.1007/s00894-019-4077-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref114/cit114&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1007%2Fs00894-019-4077-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref114/cit114&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=31222506" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref114/cit114&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A280%3ADC%252BB3M3ptVOhtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2019&author=I.+Ugurauthor=M.+Schroftauthor=A.+Marionauthor=M.+Glaserauthor=I.+Antes&title=Predicting+the+Bioactive+Conformations+of+Macrocycles%3A+A+Molecular+Dynamics-Based+Docking+Procedure+with+DynaDock&doi=10.1007%2Fs00894-019-4077-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit114R"><div class="casContent"><span class="casTitleNuber">114</span><div class="casTitle"><span class="NLM_cas:atitle">Predicting the bioactive conformations of macrocycles: a molecular dynamics-based docking procedure with DynaDock</span></div><div class="casAuthors">Ugur Ilke; Schroft Maja; Marion Antoine; Glaser Manuel; Antes Iris; Ugur Ilke; Marion Antoine</div><div class="citationInfo"><span class="NLM_cas:title">Journal of molecular modeling</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">197</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Macrocyclic compounds are of growing interest as a new class of therapeutics, especially as inhibitors binding to protein-protein interfaces.  As molecular modeling is a well-established complimentary tool in modern drug design, the number of attempts to develop reliable docking strategies and algorithms to accurately predict the binding mode of macrocycles is rising continuously.  Standard molecular docking approaches need to be adapted to this application, as a comprehensive yet efficient sampling of all ring conformations of the macrocycle is necessary.  To overcome this issue, we designed a molecular dynamics-based docking protocol for macrocycles, in which the challenging sampling step is addressed by conventional molecular dynamics (750 ns) simulations performed at moderately high temperature (370 K).  Consecutive flexible docking with the DynaDock approach based on multiple, pre-sampled ring conformations yields highly accurate poses with ligand RMSD values lower than 1.8 ÅA.  We further investigated the value of molecular dynamics-based complex stability estimations for pose selection and discuss its applicability in combination with standard binding free energy estimations for assessing the quality of poses in future blind docking studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTIFcR2EWdUqLGbPKh4T9eAfW6udTcc2eayLVKIBZkCxbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3M3ptVOhtg%253D%253D&md5=0c50fc5a1d0a305c82f3dd62376f4438</span></div><a href="/servlet/linkout?suffix=cit114&amp;dbid=16384&amp;doi=10.1007%2Fs00894-019-4077-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00894-019-4077-5%26sid%3Dliteratum%253Aachs%26aulast%3DUgur%26aufirst%3DI.%26aulast%3DSchroft%26aufirst%3DM.%26aulast%3DMarion%26aufirst%3DA.%26aulast%3DGlaser%26aufirst%3DM.%26aulast%3DAntes%26aufirst%3DI.%26atitle%3DPredicting%2520the%2520Bioactive%2520Conformations%2520of%2520Macrocycles%253A%2520A%2520Molecular%2520Dynamics-Based%2520Docking%2520Procedure%2520with%2520DynaDock%26jtitle%3DJ.%2520Mol.%2520Model.%26date%3D2019%26volume%3D25%26doi%3D10.1007%2Fs00894-019-4077-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref115"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref115'); return false;" data-citation="" class="refNumLink">115</a></strong><div class="NLM_citation" id="cit115"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jantz, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Briem, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sommer, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rarey, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christ, C. D.</span></span> <span> </span><span class="NLM_article-title">Computational Macrocyclization: From de Novo Macrocycle Generation to Binding Affinity Estimation</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">1866</span>– <span class="NLM_lpage">1872</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201700478</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref115/cit115&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1002%2Fcmdc.201700478" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref115/cit115&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=28977738" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref115/cit115&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhslSktbnI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=1866-1872&author=V.+Wagnerauthor=L.+Jantzauthor=H.+Briemauthor=K.+Sommerauthor=M.+Rareyauthor=C.+D.+Christ&title=Computational+Macrocyclization%3A+From+de+Novo+Macrocycle+Generation+to+Binding+Affinity+Estimation&doi=10.1002%2Fcmdc.201700478"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit115R"><div class="casContent"><span class="casTitleNuber">115</span><div class="casTitle"><span class="NLM_cas:atitle">Computational Macrocyclization: From de novo Macrocycle Generation to Binding Affinity Estimation</span></div><div class="casAuthors">Wagner, Vincent; Jantz, Linda; Briem, Hans; Sommer, Kai; Rarey, Matthias; Christ, Clara D.</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">1866-1872</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Macrocycles play an increasing role in drug discovery, but their synthesis is often demanding.  Computational tools that suggest macrocyclization based on a known binding mode and that est. the binding affinity of these macrocycles could have a substantial impact on the medicinal chem. design process.  For both tasks, we established a workflow with high practical value.  For five diverse pharmaceutical targets we show that the effect of macrocyclization on binding can be calcd. robustly and accurately.  Applying this method to macrocycles designed by LigMac, a search tool for de novo macrocyclization, our results suggest that we have a robust protocol in hand to design macrocycles and prioritize them prior to synthesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonKcpGd8uDs7Vg90H21EOLACvtfcHk0ljjPg7Dc2UQGA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhslSktbnI&md5=30f49d08b16cd5473ea69de3e975ebee</span></div><a href="/servlet/linkout?suffix=cit115&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201700478&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201700478%26sid%3Dliteratum%253Aachs%26aulast%3DWagner%26aufirst%3DV.%26aulast%3DJantz%26aufirst%3DL.%26aulast%3DBriem%26aufirst%3DH.%26aulast%3DSommer%26aufirst%3DK.%26aulast%3DRarey%26aufirst%3DM.%26aulast%3DChrist%26aufirst%3DC.%2BD.%26atitle%3DComputational%2520Macrocyclization%253A%2520From%2520de%2520Novo%2520Macrocycle%2520Generation%2520to%2520Binding%2520Affinity%2520Estimation%26jtitle%3DChemMedChem%26date%3D2017%26volume%3D12%26spage%3D1866%26epage%3D1872%26doi%3D10.1002%2Fcmdc.201700478" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref116"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref116'); return false;" data-citation="" class="refNumLink">116</a></strong><div class="NLM_citation" id="cit116"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yu, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sindhikara, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rask, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kimura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span> <span> </span><span class="NLM_article-title">Accurate and Reliable Prediction of the Binding Affinities of Macrocycles to Their Protein Targets</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">6290</span>– <span class="NLM_lpage">6300</span>, <span class="refDoi"> DOI: 10.1021/acs.jctc.7b00885</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jctc.7b00885" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref116/cit116&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsl2gs7fP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&pages=6290-6300&author=H.+S.+Yuauthor=Y.+Dengauthor=Y.+Wuauthor=D.+Sindhikaraauthor=A.+R.+Raskauthor=T.+Kimuraauthor=R.+Abelauthor=L.+Wang&title=Accurate+and+Reliable+Prediction+of+the+Binding+Affinities+of+Macrocycles+to+Their+Protein+Targets&doi=10.1021%2Facs.jctc.7b00885"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit116R"><div class="casContent"><span class="casTitleNuber">116</span><div class="casTitle"><span class="NLM_cas:atitle">Accurate and Reliable Prediction of the Binding Affinities of Macrocycles to Their Protein Targets</span></div><div class="casAuthors">Yu, Haoyu S.; Deng, Yuqing; Wu, Yujie; Sindhikara, Dan; Rask, Amy R.; Kimura, Takayuki; Abel, Robert; Wang, Lingle</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">6290-6300</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Macrocycles have been emerging as a very important drug class in the past few decades largely due to their expanded chem. diversity benefiting from advances in synthetic methods.  Macrocyclization has been recognized as an effective way to restrict the conformational space of acyclic small mol. inhibitors with the hope of improving potency, selectivity, and metabolic stability.  Because of their relatively larger size as compared to typical small mol. drugs and the complexity of the structures, efficient sampling of the accessible macrocycle conformational space and accurate prediction of their binding affinities to their target protein receptors poses a great challenge of central importance in computational macrocycle drug design.  In this article, the authors present a novel method for relative binding free energy calcns. between macrocycles with different ring sizes and between the macrocycles and their corresponding acyclic counterparts.  The authors have applied the method to seven pharmaceutically interesting data sets taken from recent drug discovery projects including 33 macrocyclic ligands covering a diverse chem. space.  The predicted binding free energies are in good agreement with exptl. data with an overall root-mean-square error (RMSE) of 0.94 kcal/mol.  This is to the authors' knowledge the first time where the free energy of the macrocyclization of linear mols. has been directly calcd. with rigorous physics-based free energy calcn. methods, and the authors anticipate the outstanding accuracy demonstrated here across a broad range of target classes may have significant implications for macrocycle drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoibClnk_AFJrVg90H21EOLACvtfcHk0ljjPg7Dc2UQGA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsl2gs7fP&md5=62d194fa339aaadd9c46611730dee5ef</span></div><a href="/servlet/linkout?suffix=cit116&amp;dbid=16384&amp;doi=10.1021%2Facs.jctc.7b00885&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jctc.7b00885%26sid%3Dliteratum%253Aachs%26aulast%3DYu%26aufirst%3DH.%2BS.%26aulast%3DDeng%26aufirst%3DY.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DSindhikara%26aufirst%3DD.%26aulast%3DRask%26aufirst%3DA.%2BR.%26aulast%3DKimura%26aufirst%3DT.%26aulast%3DAbel%26aufirst%3DR.%26aulast%3DWang%26aufirst%3DL.%26atitle%3DAccurate%2520and%2520Reliable%2520Prediction%2520of%2520the%2520Binding%2520Affinities%2520of%2520Macrocycles%2520to%2520Their%2520Protein%2520Targets%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2017%26volume%3D13%26spage%3D6290%26epage%3D6300%26doi%3D10.1021%2Facs.jctc.7b00885" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref117"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref117'); return false;" data-citation="" class="refNumLink">117</a></strong><div class="NLM_citation" id="cit117"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wallraven, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holmelin, F. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glas, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hennig, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frolov, A. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grossmann, T. N.</span></span> <span> </span><span class="NLM_article-title">Adapting Free Energy Perturbation Simulations for Large Macrocyclic Ligands: How to Dissect Contributions from Direct Binding and Free Ligand Flexibility</span>. <i>Chem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">2269</span>– <span class="NLM_lpage">2276</span>, <span class="refDoi"> DOI: 10.1039/C9SC04705K</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref117/cit117&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1039%2FC9SC04705K" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref117/cit117&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=32180932" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref117/cit117&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtlWqsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=2269-2276&author=K.+Wallravenauthor=F.+L.+Holmelinauthor=A.+Glasauthor=S.+Hennigauthor=A.+I.+Frolovauthor=T.+N.+Grossmann&title=Adapting+Free+Energy+Perturbation+Simulations+for+Large+Macrocyclic+Ligands%3A+How+to+Dissect+Contributions+from+Direct+Binding+and+Free+Ligand+Flexibility&doi=10.1039%2FC9SC04705K"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit117R"><div class="casContent"><span class="casTitleNuber">117</span><div class="casTitle"><span class="NLM_cas:atitle">Adapting free energy perturbation simulations for large macrocyclic ligands: how to dissect contributions from direct binding and free ligand flexibility</span></div><div class="casAuthors">Wallraven, Kerstin; Holmelin, Fredrik L.; Glas, Adrian; Hennig, Sven; Frolov, Andrey I.; Grossmann, Tom N.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Science</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2269-2276</span>CODEN:
                <span class="NLM_cas:coden">CSHCCN</span>;
        ISSN:<span class="NLM_cas:issn">2041-6520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Large and flexible ligands gain increasing interest in the development of bioactive agents.  They challenge the applicability of computational ligand optimization strategies originally developed for small mols.  Free energy perturbation (FEP) is often used for predicting binding affinities of small mol. ligands, however, its use for more complex ligands remains limited.  Herein, we report the structure-based design of peptide macrocycles targeting the protein binding site of human adaptor protein 14-3-3.  We observe a surprisingly strong dependency of binding affinities on relatively small variations in substituent size.  FEP was performed to rationalize obsd. trends.  To account for insufficient convergence of FEP, restrained calcns. were performed and complemented with extensive REST MD simulations of the free ligands.  These calcns. revealed that changes in affinity originate both from altered direct interactions and conformational changes of the free ligand.  In addn., MD simulations provided the basis to rationalize unexpected trends in ligand lipophilicity.  We also verified the anticipated interaction site and binding mode for one of the high affinity ligands by X-ray crystallog.  The introduced fully-atomistic simulation protocol can be used to rationalize the development of structurally complex ligands which will support future ligand maturation efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrkl4saxYBokrVg90H21EOLACvtfcHk0ljjPg7Dc2UQGA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtlWqsLo%253D&md5=d2190c7ddb2725f9fa508724bb1d5773</span></div><a href="/servlet/linkout?suffix=cit117&amp;dbid=16384&amp;doi=10.1039%2FC9SC04705K&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC9SC04705K%26sid%3Dliteratum%253Aachs%26aulast%3DWallraven%26aufirst%3DK.%26aulast%3DHolmelin%26aufirst%3DF.%2BL.%26aulast%3DGlas%26aufirst%3DA.%26aulast%3DHennig%26aufirst%3DS.%26aulast%3DFrolov%26aufirst%3DA.%2BI.%26aulast%3DGrossmann%26aufirst%3DT.%2BN.%26atitle%3DAdapting%2520Free%2520Energy%2520Perturbation%2520Simulations%2520for%2520Large%2520Macrocyclic%2520Ligands%253A%2520How%2520to%2520Dissect%2520Contributions%2520from%2520Direct%2520Binding%2520and%2520Free%2520Ligand%2520Flexibility%26jtitle%3DChem.%2520Sci.%26date%3D2020%26volume%3D11%26spage%3D2269%26epage%3D2276%26doi%3D10.1039%2FC9SC04705K" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref118"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref118'); return false;" data-citation="" class="refNumLink">118</a></strong><div class="NLM_citation" id="cit118"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banks, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klicic, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mainz, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Repasky, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knoll, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shelley, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perry, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Francis, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shenkin, P. S.</span></span> <span> </span><span class="NLM_article-title">Glide: A New Approach for Rapid, Accurate Docking and Scoring. 1. Method and Assessment of Docking Accuracy</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">1739</span>– <span class="NLM_lpage">1749</span>, <span class="refDoi"> DOI: 10.1021/jm0306430</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0306430" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref118/cit118&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFyit74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1739-1749&author=R.+A.+Friesnerauthor=J.+L.+Banksauthor=R.+B.+Murphyauthor=T.+A.+Halgrenauthor=J.+J.+Klicicauthor=D.+T.+Mainzauthor=M.+P.+Repaskyauthor=E.+H.+Knollauthor=M.+Shelleyauthor=J.+K.+Perryauthor=D.+E.+Shawauthor=P.+Francisauthor=P.+S.+Shenkin&title=Glide%3A+A+New+Approach+for+Rapid%2C+Accurate+Docking+and+Scoring.+1.+Method+and+Assessment+of+Docking+Accuracy&doi=10.1021%2Fjm0306430"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit118R"><div class="casContent"><span class="casTitleNuber">118</span><div class="casTitle"><span class="NLM_cas:atitle">Glide: A new approach for rapid, accurate docking and scoring. 1. method and assessment of docking accuracy</span></div><div class="casAuthors">Friesner, Richard A.; Banks, Jay L.; Murphy, Robert B.; Halgren, Thomas A.; Klicic, Jasna J.; Mainz, Daniel T.; Repasky, Matthew P.; Knoll, Eric H.; Shelley, Mee; Perry, Jason K.; Shaw, David E.; Francis, Perry; Shenkin, Peter S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1739-1749</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Unlike other methods for docking ligands to the rigid 3D structure of a known protein receptor, Glide approximates a complete systematic search of the conformational, orientational, and positional space of the docked ligand.  In this search, an initial rough positioning and scoring phase that dramatically narrows the search space is followed by torsionally flexible energy optimization on an OPLS-AA nonbonded potential grid for a few hundred surviving candidate poses.  The very best candidates are further refined via a Monte Carlo sampling of pose conformation; in some cases, this is crucial to obtaining an accurate docked pose.  Selection of the best docked pose uses a model energy function that combines empirical and force-field-based terms.  Docking accuracy is assessed by redocking ligands from 282 cocrystd. PDB complexes starting from conformationally optimized ligand geometries that bear no memory of the correctly docked pose.  Errors in geometry for the top-ranked pose are less than 1 Å in nearly half of the cases and are greater than 2 Å in only about one-third of them.  Comparisons to published data on rms deviations show that Glide is nearly twice as accurate as GOLD and more than twice as accurate as FlexX for ligands having up to 20 rotatable bonds.  Glide is also found to be more accurate than the recently described Surflex method.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPuigu31qMMLVg90H21EOLACvtfcHk0lhPW0ulLWG1OA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFyit74%253D&md5=8cc2f0022318b12dd972e9c493375bf9</span></div><a href="/servlet/linkout?suffix=cit118&amp;dbid=16384&amp;doi=10.1021%2Fjm0306430&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0306430%26sid%3Dliteratum%253Aachs%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DBanks%26aufirst%3DJ.%2BL.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DKlicic%26aufirst%3DJ.%2BJ.%26aulast%3DMainz%26aufirst%3DD.%2BT.%26aulast%3DRepasky%26aufirst%3DM.%2BP.%26aulast%3DKnoll%26aufirst%3DE.%2BH.%26aulast%3DShelley%26aufirst%3DM.%26aulast%3DPerry%26aufirst%3DJ.%2BK.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26aulast%3DFrancis%26aufirst%3DP.%26aulast%3DShenkin%26aufirst%3DP.%2BS.%26atitle%3DGlide%253A%2520A%2520New%2520Approach%2520for%2520Rapid%252C%2520Accurate%2520Docking%2520and%2520Scoring.%25201.%2520Method%2520and%2520Assessment%2520of%2520Docking%2520Accuracy%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1739%26epage%3D1749%26doi%3D10.1021%2Fjm0306430" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref119"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref119'); return false;" data-citation="" class="refNumLink">119</a></strong><div class="NLM_citation" id="cit119"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beard, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollard, W. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banks, J. L.</span></span> <span> </span><span class="NLM_article-title">Glide: A New Approach for Rapid, Accurate Docking and Scoring. 2. Enrichment Factors in Database Screening</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">1750</span>– <span class="NLM_lpage">1759</span>, <span class="refDoi"> DOI: 10.1021/jm030644s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm030644s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref119/cit119&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFyit78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1750-1759&author=T.+A.+Halgrenauthor=R.+B.+Murphyauthor=R.+A.+Friesnerauthor=H.+S.+Beardauthor=L.+L.+Fryeauthor=W.+T.+Pollardauthor=J.+L.+Banks&title=Glide%3A+A+New+Approach+for+Rapid%2C+Accurate+Docking+and+Scoring.+2.+Enrichment+Factors+in+Database+Screening&doi=10.1021%2Fjm030644s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit119R"><div class="casContent"><span class="casTitleNuber">119</span><div class="casTitle"><span class="NLM_cas:atitle">Glide: A new approach for rapid, accurate docking and scoring. 2. Enrichment factors in database screening</span></div><div class="casAuthors">Halgren, Thomas A.; Murphy, Robert B.; Friesner, Richard A.; Beard, Hege S.; Frye, Leah L.; Pollard, W. Thomas; Banks, Jay L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1750-1759</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Glide's ability to identify active compds. in a database screen is characterized by applying Glide to a diverse set of nine protein receptors.  In many cases, two, or even three, protein sites are employed to probe the sensitivity of the results to the site geometry.  To make the database screens as realistic as possible, the screens use sets of "druglike" decoy ligands that have been selected to be representative of what we believe is likely to be found in the compd. collection of a pharmaceutical or biotechnol. company.  Results are presented for releases 1.8, 2.0, and 2.5 of Glide.  The comparisons show that av. measures for both "early" and "global" enrichment for Glide 2.5 are 3 times higher than for Glide 1.8 and more than 2 times higher than for Glide 2.0 because of better results for the least well-handled screens.  This improvement in enrichment stems largely from the better balance of the more widely parametrized GlideScore 2.5 function and the inclusion of terms that penalize ligand-protein interactions that violate established principles of phys. chem., particularly as it concerns the exposure to solvent of charged protein and ligand groups.  Comparisons to results for the thymidine kinase and estrogen receptors published by Rognan and co-workers (J. Med. Chem. 2000, 43, 4759-4767) show that Glide 2.5 performs better than GOLD 1.1, FlexX 1.8, or DOCK 4.01.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoT2ptubbKbvLVg90H21EOLACvtfcHk0lhPW0ulLWG1OA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFyit78%253D&md5=33d68dd968e65626b449df61e44e37be</span></div><a href="/servlet/linkout?suffix=cit119&amp;dbid=16384&amp;doi=10.1021%2Fjm030644s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm030644s%26sid%3Dliteratum%253Aachs%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DBeard%26aufirst%3DH.%2BS.%26aulast%3DFrye%26aufirst%3DL.%2BL.%26aulast%3DPollard%26aufirst%3DW.%2BT.%26aulast%3DBanks%26aufirst%3DJ.%2BL.%26atitle%3DGlide%253A%2520A%2520New%2520Approach%2520for%2520Rapid%252C%2520Accurate%2520Docking%2520and%2520Scoring.%25202.%2520Enrichment%2520Factors%2520in%2520Database%2520Screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1750%26epage%3D1759%26doi%3D10.1021%2Fjm030644s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref120"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref120'); return false;" data-citation="" class="refNumLink">120</a></strong><div class="NLM_citation" id="cit120"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kollman, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Massova, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reyes, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuhn, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chong, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donini, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cieplak, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Srinivasan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Case, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheatham, T. E.</span></span> <span> </span><span class="NLM_article-title">Calculating Structures and Free Energies of Complex Molecules: Combining Molecular Mechanics and Continuum Models</span>. <i>Acc. Chem. Res.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">889</span>– <span class="NLM_lpage">897</span>, <span class="refDoi"> DOI: 10.1021/ar000033j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ar000033j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref120/cit120&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BD3cXmvFGiu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2000&pages=889-897&author=P.+A.+Kollmanauthor=I.+Massovaauthor=C.+Reyesauthor=B.+Kuhnauthor=S.+Huoauthor=L.+Chongauthor=M.+Leeauthor=T.+Leeauthor=Y.+Duanauthor=W.+Wangauthor=O.+Doniniauthor=P.+Cieplakauthor=J.+Srinivasanauthor=D.+A.+Caseauthor=T.+E.+Cheatham&title=Calculating+Structures+and+Free+Energies+of+Complex+Molecules%3A+Combining+Molecular+Mechanics+and+Continuum+Models&doi=10.1021%2Far000033j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit120R"><div class="casContent"><span class="casTitleNuber">120</span><div class="casTitle"><span class="NLM_cas:atitle">Calculating Structures and Free Energies of Complex Molecules: Combining Molecular Mechanics and Continuum Models</span></div><div class="casAuthors">Kollman, Peter A.; Massova, Irina; Reyes, Carolina; Kuhn, Bernd; Huo, Shuanghong; Chong, Lillian; Lee, Matthew; Lee, Taisung; Duan, Yong; Wang, Wei; Donini, Oreola; Cieplak, Piotr; Srinivasan, Jaysharee; Case, David A.; Cheatham, Thomas E., III</div><div class="citationInfo"><span class="NLM_cas:title">Accounts of Chemical Research</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">889-897</span>CODEN:
                <span class="NLM_cas:coden">ACHRE4</span>;
        ISSN:<span class="NLM_cas:issn">0001-4842</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review, with 63 refs.  A historical perspective on the application of mol. dynamics (MD) to biol. macromols. is presented.  Recent developments combining state-of-the-art force fields with continuum solvation calcns. have allowed us to reach the fourth era of MD applications in which one can often derive both accurate structure and accurate relative free energies from mol. dynamics trajectories.  We illustrate such applications on nucleic acid duplexes, RNA hairpins, protein folding trajectories, and protein-ligand, protein-protein, and protein-nucleic acid interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXGhwlfA3kvLVg90H21EOLACvtfcHk0lhPW0ulLWG1OA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXmvFGiu7g%253D&md5=8436ee610ae145894428db1a1deff73c</span></div><a href="/servlet/linkout?suffix=cit120&amp;dbid=16384&amp;doi=10.1021%2Far000033j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Far000033j%26sid%3Dliteratum%253Aachs%26aulast%3DKollman%26aufirst%3DP.%2BA.%26aulast%3DMassova%26aufirst%3DI.%26aulast%3DReyes%26aufirst%3DC.%26aulast%3DKuhn%26aufirst%3DB.%26aulast%3DHuo%26aufirst%3DS.%26aulast%3DChong%26aufirst%3DL.%26aulast%3DLee%26aufirst%3DM.%26aulast%3DLee%26aufirst%3DT.%26aulast%3DDuan%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DDonini%26aufirst%3DO.%26aulast%3DCieplak%26aufirst%3DP.%26aulast%3DSrinivasan%26aufirst%3DJ.%26aulast%3DCase%26aufirst%3DD.%2BA.%26aulast%3DCheatham%26aufirst%3DT.%2BE.%26atitle%3DCalculating%2520Structures%2520and%2520Free%2520Energies%2520of%2520Complex%2520Molecules%253A%2520Combining%2520Molecular%2520Mechanics%2520and%2520Continuum%2520Models%26jtitle%3DAcc.%2520Chem.%2520Res.%26date%3D2000%26volume%3D33%26spage%3D889%26epage%3D897%26doi%3D10.1021%2Far000033j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref121"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref121'); return false;" data-citation="" class="refNumLink">121</a></strong><div class="NLM_citation" id="cit121"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Srinivasan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kollman, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Case, D. A.</span></span> <span> </span><span class="NLM_article-title">Continuum Solvent Studies of the Stability of RNA Hairpin Loops and Helices</span>. <i>J. Biomol. Struct. Dyn.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">671</span>– <span class="NLM_lpage">682</span>, <span class="refDoi"> DOI: 10.1080/07391102.1998.10508279</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref121/cit121&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1080%2F07391102.1998.10508279" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref121/cit121&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10052623" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref121/cit121&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADyaK1MXhtlams74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=1998&pages=671-682&author=J.+Srinivasanauthor=J.+Millerauthor=P.+A.+Kollmanauthor=D.+A.+Case&title=Continuum+Solvent+Studies+of+the+Stability+of+RNA+Hairpin+Loops+and+Helices&doi=10.1080%2F07391102.1998.10508279"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit121R"><div class="casContent"><span class="casTitleNuber">121</span><div class="casTitle"><span class="NLM_cas:atitle">Continuum solvent studies of the stability of RNA hairpin loops and helices</span></div><div class="casAuthors">Srinivasan, Jayashree; Miller, Jennifer; Kollman, Peter A.; Case, David A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Structure & Dynamics</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">671-682</span>CODEN:
                <span class="NLM_cas:coden">JBSDD6</span>;
        ISSN:<span class="NLM_cas:issn">0739-1102</span>.
    
            (<span class="NLM_cas:orgname">Adenine Press</span>)
        </div><div class="casAbstract">We apply continuum solvent models to investigate the relative stability of various conformational forms for two RNA sequences, GGAC(UUCG)GUCC and GGUG(UGAA)CACC.  In the first part, we compare alternate hairpin conformations to explore the reliability of these models to discriminate between different local conformations.  A second part looks at the hairpin-duplex conversion for the UUCG sequence, identifying major contributors to the thermodn. of a large scale transition.  Structures were taken as snapshots from multi-nanosecond mol. dynamics simulations computed in a consistent fashion using explicit solvent and with long-range electrostatics accounted for using the Particle-Mesh Ewald procedure.  The electrostatic contribution to solvation energies were computed using both a finite-difference Poisson-Boltzmann (PB) model and a pairwise Generalized Born model; non-electrostatic contributions were estd. with a surface-area dependent term.  To these solvation free energies were added the mean solute internal energies (detd. from a mol. mechanics potential) and ests. of the solute entropy (from a harmonic anal.).  Consistent with expt. and with earlier solvated mol. dynamics simulations, the UUCG hairpin was found to prefer conformers close to a recent NMR structure detn. in preference to those from an earlier NMR study.  Similarly, results for the UGAA hairpin favored an NMR-derived structure over that to be expected for a generic GNRA hairpin loop.  Exptl. free energies are not known for the hairpin/duplex conversion, but must be close to zero since hairpins are seen in soln. and duplexes in crystals; our calcns. find a value near zero and illustrate the expected interplay of solvation, salt effects and entropy in affecting this equil.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2ygn1MJoSNLVg90H21EOLACvtfcHk0ljQjGSv15Tz9Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXhtlams74%253D&md5=7843cabcbe7d2c75f4bc9455f2115f95</span></div><a href="/servlet/linkout?suffix=cit121&amp;dbid=16384&amp;doi=10.1080%2F07391102.1998.10508279&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F07391102.1998.10508279%26sid%3Dliteratum%253Aachs%26aulast%3DSrinivasan%26aufirst%3DJ.%26aulast%3DMiller%26aufirst%3DJ.%26aulast%3DKollman%26aufirst%3DP.%2BA.%26aulast%3DCase%26aufirst%3DD.%2BA.%26atitle%3DContinuum%2520Solvent%2520Studies%2520of%2520the%2520Stability%2520of%2520RNA%2520Hairpin%2520Loops%2520and%2520Helices%26jtitle%3DJ.%2520Biomol.%2520Struct.%2520Dyn.%26date%3D1998%26volume%3D16%26spage%3D671%26epage%3D682%26doi%3D10.1080%2F07391102.1998.10508279" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref122"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref122'); return false;" data-citation="" class="refNumLink">122</a></strong><div class="NLM_citation" id="cit122"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Paulsen, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sindhikara, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Appleby, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villaseñor, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmitz, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shivakumar, D.</span></span> <span> </span><span class="NLM_article-title">Evaluation of Free Energy Calculations for the Prioritization of Macrocycle Synthesis</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">3489</span>– <span class="NLM_lpage">3498</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.0c00132</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.0c00132" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref122/cit122&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtFGmt73N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2020&pages=3489-3498&author=J.+L.+Paulsenauthor=H.+S.+Yuauthor=D.+Sindhikaraauthor=L.+Wangauthor=T.+C.+Applebyauthor=A.+G.+Villase%C3%B1orauthor=U.+Schmitzauthor=D.+Shivakumar&title=Evaluation+of+Free+Energy+Calculations+for+the+Prioritization+of+Macrocycle+Synthesis&doi=10.1021%2Facs.jcim.0c00132"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit122R"><div class="casContent"><span class="casTitleNuber">122</span><div class="casTitle"><span class="NLM_cas:atitle">Evaluation of Free Energy Calculations for the Prioritization of Macrocycle Synthesis</span></div><div class="casAuthors">Paulsen, Janet L.; Yu, Haoyu S.; Sindhikara, Dan; Wang, Lingle; Appleby, Todd; Villasenor, Armando G.; Schmitz, Uli; Shivakumar, Devleena</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3489-3498</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A tremendous research and development effort was exerted toward combating chronic hepatitis C, ultimately leading to curative oral treatments, all of which are targeting viral proteins.  Despite the advantage of numerous targets allowing for broad hepatitis C virus (HCV) genotype coverage, the only host target inhibitors that advanced into clin. development were Cyclosporin A based cyclophilin inhibitors.  While cyclosporin-based mols. typically require a fermn. process, Gilead successfully pursued a fully synthetic, oral program based on Sanglifehrin A.  The drug discovery process, though greatly helped by facile crystallog., was still hampered by the limitations in the accuracy of predictive computational methods for prioritizing compd. ideas.  Recent advances in accuracy and speed of free energy perturbation (FEP) methods, however, are attractive for prioritizing and derisking synthetically challenging mols. and potentially could have had a significant impact on the speed of the development of this program.  Here in our simulated prospective study, the binding free energies of 26 macrocyclic cyclophilin inhibitors were blindly predicted using FEP+ to test this hypothesis.  The predictions had a low mean unsigned error (MUE) (1.1 kcal/mol) and accurately reproduced many design decisions from the program, suggesting that FEP+ has the potential to drive synthetic chem. efforts by more accurately ranking compds. with nonintuitive structure-activity relationships (SARs).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGogQ6nraJ7CZ7Vg90H21EOLACvtfcHk0ljQjGSv15Tz9Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtFGmt73N&md5=4926b12c085eb0f5181c1a265fac7571</span></div><a href="/servlet/linkout?suffix=cit122&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.0c00132&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.0c00132%26sid%3Dliteratum%253Aachs%26aulast%3DPaulsen%26aufirst%3DJ.%2BL.%26aulast%3DYu%26aufirst%3DH.%2BS.%26aulast%3DSindhikara%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DAppleby%26aufirst%3DT.%2BC.%26aulast%3DVillase%25C3%25B1or%26aufirst%3DA.%2BG.%26aulast%3DSchmitz%26aufirst%3DU.%26aulast%3DShivakumar%26aufirst%3DD.%26atitle%3DEvaluation%2520of%2520Free%2520Energy%2520Calculations%2520for%2520the%2520Prioritization%2520of%2520Macrocycle%2520Synthesis%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2020%26volume%3D60%26spage%3D3489%26epage%3D3498%26doi%3D10.1021%2Facs.jcim.0c00132" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref123"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref123'); return false;" data-citation="" class="refNumLink">123</a></strong><div class="NLM_citation" id="cit123"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cappel, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jerome, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hessler, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matter, H.</span></span> <span> </span><span class="NLM_article-title">Impact of Different Automated Binding Pose Generation Approaches on Relative Binding Free Energy Simulations</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">1432</span>– <span class="NLM_lpage">1444</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.9b01118</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.9b01118" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref123/cit123&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhs12ktb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2020&pages=1432-1444&author=D.+Cappelauthor=S.+Jeromeauthor=G.+Hesslerauthor=H.+Matter&title=Impact+of+Different+Automated+Binding+Pose+Generation+Approaches+on+Relative+Binding+Free+Energy+Simulations&doi=10.1021%2Facs.jcim.9b01118"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit123R"><div class="casContent"><span class="casTitleNuber">123</span><div class="casTitle"><span class="NLM_cas:atitle">Impact of Different Automated Binding Pose Generation Approaches on Relative Binding Free Energy Simulations</span></div><div class="casAuthors">Cappel, Daniel; Jerome, Steven; Hessler, Gerhard; Matter, Hans</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1432-1444</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Relative binding free energy (RBFE) prediction methods such as free energy perturbation (FEP) are important today for estg. protein-ligand binding affinities.  Significant hardware and algorithmic improvements now allow for simulating congeneric series within days.  Therefore, RBFE calcns. have an enormous potential for structure-based drug discovery.  As typically only a few representative crystal structures for a series are available, other ligands and design proposals must be reliably superimposed for meaningful results.  An obsd. significant effect of the alignment on FEP led us to develop an alignment approach combining docking with max. common substructure (MCS) derived core constraints from the most similar ref. pose, named MCS-docking workflow.  We then studied the effect of binding pose generation on the accuracy of RBFE predictions using six ligand series from five pharmaceutically relevant protein targets.  Overall, the MCS-docking workflow generated consistent poses for most of the ligands in the investigated series.  While multiple alignment methods often resulted in comparable FEP predictions, for most of the cases herein, the MCS-docking workflow produced the best accuracy in predictions.  Furthermore, the FEP anal. data strongly support the hypothesis that the accuracy of RBFE predictions depends on input poses to construct the perturbation map.  Therefore, an automated workflow without manual intervention minimizes potential errors and obtains the most useful predictions with significant impact for structure-based design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGob6kqQDqVs-7Vg90H21EOLACvtfcHk0ljQjGSv15Tz9Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhs12ktb4%253D&md5=c6c5e60560061e35da8e661f17c10a8e</span></div><a href="/servlet/linkout?suffix=cit123&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.9b01118&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.9b01118%26sid%3Dliteratum%253Aachs%26aulast%3DCappel%26aufirst%3DD.%26aulast%3DJerome%26aufirst%3DS.%26aulast%3DHessler%26aufirst%3DG.%26aulast%3DMatter%26aufirst%3DH.%26atitle%3DImpact%2520of%2520Different%2520Automated%2520Binding%2520Pose%2520Generation%2520Approaches%2520on%2520Relative%2520Binding%2520Free%2520Energy%2520Simulations%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2020%26volume%3D60%26spage%3D1432%26epage%3D1444%26doi%3D10.1021%2Facs.jcim.9b01118" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref124"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref124'); return false;" data-citation="" class="refNumLink">124</a></strong><div class="NLM_citation" id="cit124"><span>Daylight
Theory: SMARTS—A
Language for Describing Molecular Patterns. <a href="https://www.daylight.com/dayhtml/doc/theory/theory.smarts.html" class="extLink">https://www.daylight.com/dayhtml/doc/theory/theory.smarts.html</a> (accessed April 6, <span class="NLM_year">2020</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Daylight%0ATheory%3A+SMARTS%E2%80%94A%0ALanguage+for+Describing+Molecular+Patterns.+https%3A%2F%2Fwww.daylight.com%2Fdayhtml%2Fdoc%2Ftheory%2Ftheory.smarts.html+%28accessed+April+6%2C+2020%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit124&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs%26date%3D2020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref125"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref125'); return false;" data-citation="" class="refNumLink">125</a></strong><div class="NLM_citation" id="cit125"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Evers, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haack, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lorenz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bossart, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elvert, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henkel, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stengelin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glien, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dudda, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lorenz, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kadereit, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, M.</span></span> <span> </span><span class="NLM_article-title">Design of Novel Exendin-Based Dual Glucagon-like Peptide 1 (GLP-1)/Glucagon Receptor Agonists</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">4293</span>– <span class="NLM_lpage">4303</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00174</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00174" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref125/cit125&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmslKrsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=4293-4303&author=A.+Eversauthor=T.+Haackauthor=M.+Lorenzauthor=M.+Bossartauthor=R.+Elvertauthor=B.+Henkelauthor=S.+Stengelinauthor=M.+Kurzauthor=M.+Glienauthor=A.+Duddaauthor=K.+Lorenzauthor=D.+Kadereitauthor=M.+Wagner&title=Design+of+Novel+Exendin-Based+Dual+Glucagon-like+Peptide+1+%28GLP-1%29%2FGlucagon+Receptor+Agonists&doi=10.1021%2Facs.jmedchem.7b00174"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit125R"><div class="casContent"><span class="casTitleNuber">125</span><div class="casTitle"><span class="NLM_cas:atitle">Design of Novel Exendin-Based Dual Glucagon-like Peptide 1 (GLP-1)/Glucagon Receptor Agonists</span></div><div class="casAuthors">Evers, Andreas; Haack, Torsten; Lorenz, Martin; Bossart, Martin; Elvert, Ralf; Henkel, Bernd; Stengelin, Siegfried; Kurz, Michael; Glien, Maike; Dudda, Angela; Lorenz, Katrin; Kadereit, Dieter; Wagner, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4293-4303</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Dual activation of the glucagon-like peptide 1 (GLP-1) and glucagon receptor has the potential to lead to a novel therapy principle for the treatment of diabesity.  Here, we report a series of novel peptides with dual activity on these receptors that were discovered by rational design.  On the basis of sequence anal. and structure-based design, structural elements of glucagon were engineered into the selective GLP-1 receptor agonist exendin-4, resulting in hybrid peptides with potent dual GLP-1/glucagon receptor activity.  Detailed structure-activity relationship data are shown.  Further modifications with unnatural and modified amino acids resulted in novel metabolically stable peptides that demonstrated a significant dose-dependent decrease in blood glucose in chronic studies in diabetic db/db mice and reduced body wt. in diet-induced obese (DIO) mice.  Structural anal. by NMR spectroscopy confirmed that the peptides maintain an exendin-4-like structure with its characteristic tryptophan-cage fold motif that is responsible for favorable chem. and phys. stability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrewDgurwTT4LVg90H21EOLACvtfcHk0ljoi39nA6kRGQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmslKrsr8%253D&md5=b59a7a25cdd56516b78c988860b39211</span></div><a href="/servlet/linkout?suffix=cit125&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00174&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00174%26sid%3Dliteratum%253Aachs%26aulast%3DEvers%26aufirst%3DA.%26aulast%3DHaack%26aufirst%3DT.%26aulast%3DLorenz%26aufirst%3DM.%26aulast%3DBossart%26aufirst%3DM.%26aulast%3DElvert%26aufirst%3DR.%26aulast%3DHenkel%26aufirst%3DB.%26aulast%3DStengelin%26aufirst%3DS.%26aulast%3DKurz%26aufirst%3DM.%26aulast%3DGlien%26aufirst%3DM.%26aulast%3DDudda%26aufirst%3DA.%26aulast%3DLorenz%26aufirst%3DK.%26aulast%3DKadereit%26aufirst%3DD.%26aulast%3DWagner%26aufirst%3DM.%26atitle%3DDesign%2520of%2520Novel%2520Exendin-Based%2520Dual%2520Glucagon-like%2520Peptide%25201%2520%2528GLP-1%2529%252FGlucagon%2520Receptor%2520Agonists%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D4293%26epage%3D4303%26doi%3D10.1021%2Facs.jmedchem.7b00174" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref126"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref126'); return false;" data-citation="" class="refNumLink">126</a></strong><div class="NLM_citation" id="cit126"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Evers, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bossart, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pfeiffer-Marek, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elvert, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schreuder, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stengelin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lorenz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herling, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konkar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lukasczyk, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pfenninger, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lorenz, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haack, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kadereit, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, M.</span></span> <span> </span><span class="NLM_article-title">Dual Glucagon-like Peptide 1 (GLP-1)/Glucagon Receptor Agonists Specifically Optimized for Multidose Formulations</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">5580</span>– <span class="NLM_lpage">5593</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00292</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00292" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref126/cit126&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtF2isLrL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=5580-5593&author=A.+Eversauthor=M.+Bossartauthor=S.+Pfeiffer-Marekauthor=R.+Elvertauthor=H.+Schreuderauthor=M.+Kurzauthor=S.+Stengelinauthor=M.+Lorenzauthor=A.+Herlingauthor=A.+Konkarauthor=U.+Lukasczykauthor=A.+Pfenningerauthor=K.+Lorenzauthor=T.+Haackauthor=D.+Kadereitauthor=M.+Wagner&title=Dual+Glucagon-like+Peptide+1+%28GLP-1%29%2FGlucagon+Receptor+Agonists+Specifically+Optimized+for+Multidose+Formulations&doi=10.1021%2Facs.jmedchem.8b00292"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit126R"><div class="casContent"><span class="casTitleNuber">126</span><div class="casTitle"><span class="NLM_cas:atitle">Dual Glucagon-like Peptide 1 (GLP-1)/Glucagon Receptor Agonists Specifically Optimized for Multidose Formulations</span></div><div class="casAuthors">Evers, Andreas; Bossart, Martin; Pfeiffer-Marek, Stefania; Elvert, Ralf; Schreuder, Herman; Kurz, Michael; Stengelin, Siegfried; Lorenz, Martin; Herling, Andreas; Konkar, Anish; Lukasczyk, Ulrike; Pfenninger, Anja; Lorenz, Katrin; Haack, Torsten; Kadereit, Dieter; Wagner, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">5580-5593</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Novel peptidic dual agonists of the glucagon-like peptide 1 (GLP-1) and glucagon receptor are reported to have enhanced efficacy over pure GLP-1 receptor agonists with regard to treatment of obesity and diabetes.  We describe novel exendin-4 based dual agonists designed with an activity ratio favoring the GLP-1 vs. the glucagon receptor.  As result of an iterative optimization procedure that included mol. modeling, structural biol. studies (X-ray, NMR), peptide design and synthesis, exptl. activity, and soly. profiling, a candidate mol. was identified.  Novel SAR points are reported that allowed us to fine-tune the desired receptor activity ratio and increased soly. in the presence of antimicrobial preservatives, findings that can be of general applicability for any peptide discovery project.  The peptide was evaluated in chronic in vivo studies in obese diabetic monkeys as translational model for the human situation and demonstrated favorable blood glucose and body wt. lowering effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqj6Q6G9iY2lrVg90H21EOLACvtfcHk0ljoi39nA6kRGQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtF2isLrL&md5=cba1860c3a7b83427ee1cc0336b0e52c</span></div><a href="/servlet/linkout?suffix=cit126&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00292&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00292%26sid%3Dliteratum%253Aachs%26aulast%3DEvers%26aufirst%3DA.%26aulast%3DBossart%26aufirst%3DM.%26aulast%3DPfeiffer-Marek%26aufirst%3DS.%26aulast%3DElvert%26aufirst%3DR.%26aulast%3DSchreuder%26aufirst%3DH.%26aulast%3DKurz%26aufirst%3DM.%26aulast%3DStengelin%26aufirst%3DS.%26aulast%3DLorenz%26aufirst%3DM.%26aulast%3DHerling%26aufirst%3DA.%26aulast%3DKonkar%26aufirst%3DA.%26aulast%3DLukasczyk%26aufirst%3DU.%26aulast%3DPfenninger%26aufirst%3DA.%26aulast%3DLorenz%26aufirst%3DK.%26aulast%3DHaack%26aufirst%3DT.%26aulast%3DKadereit%26aufirst%3DD.%26aulast%3DWagner%26aufirst%3DM.%26atitle%3DDual%2520Glucagon-like%2520Peptide%25201%2520%2528GLP-1%2529%252FGlucagon%2520Receptor%2520Agonists%2520Specifically%2520Optimized%2520for%2520Multidose%2520Formulations%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D5580%26epage%3D5593%26doi%3D10.1021%2Facs.jmedchem.8b00292" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref127"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref127'); return false;" data-citation="" class="refNumLink">127</a></strong><div class="NLM_citation" id="cit127"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tarrasch, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun Kobilka, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobilka, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skiniotis, G.</span></span> <span> </span><span class="NLM_article-title">Cryo-EM Structure of the Activated GLP-1 Receptor in Complex with a G Protein</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>546</i></span>,  <span class="NLM_fpage">248</span>– <span class="NLM_lpage">253</span>, <span class="refDoi"> DOI: 10.1038/nature22394</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref127/cit127&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1038%2Fnature22394" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref127/cit127&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=28538729" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref127/cit127&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC2sXosVagtLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=546&publication_year=2017&pages=248-253&author=Y.+Zhangauthor=B.+Sunauthor=D.+Fengauthor=H.+Huauthor=M.+Chuauthor=Q.+Quauthor=J.+T.+Tarraschauthor=S.+Liauthor=T.+Sun+Kobilkaauthor=B.+K.+Kobilkaauthor=G.+Skiniotis&title=Cryo-EM+Structure+of+the+Activated+GLP-1+Receptor+in+Complex+with+a+G+Protein&doi=10.1038%2Fnature22394"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit127R"><div class="casContent"><span class="casTitleNuber">127</span><div class="casTitle"><span class="NLM_cas:atitle">Cryo-EM structure of the activated GLP-1 receptor in complex with a G protein</span></div><div class="casAuthors">Zhang, Yan; Sun, Bingfa; Feng, Dan; Hu, Hongli; Chu, Matthew; Qu, Qianhui; Tarrasch, Jeffrey T.; Li, Shane; Sun Kobilka, Tong; Kobilka, Brian K.; Skiniotis, Georgios</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">546</span>
        (<span class="NLM_cas:issue">7657</span>),
    <span class="NLM_cas:pages">248-253</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Glucagon-like peptide 1 (GLP-1) is a hormone with essential roles in regulating insulin secretion, carbohydrate metab. and appetite.  GLP-1 effects are mediated through binding to the GLP-1 receptor (GLP-1R), a class B G-protein-coupled receptor (GPCR) that signals primarily through the stimulatory G protein Gs.  Class B GPCRs are important therapeutic targets; however, our understanding of their mechanism of action is limited by the lack of structural information on activated and full-length receptors.  Here we report the cryo-electron microscopy structure of the peptide-activated GLP-1R-Gs complex at near at. resoln.  The peptide is clasped between the N-terminal domain and the transmembrane core of the receptor, and further stabilized by extracellular loops.  Conformational changes in the transmembrane domain result in a sharp kink in the middle of transmembrane helix 6, which pivots its intracellular half outward to accommodate the α5-helix of the Ras-like domain of Gs.  These results provide a structural framework for understanding class B GPCR activation through hormone binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmDRFGRwMs1bVg90H21EOLACvtfcHk0ljoi39nA6kRGQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXosVagtLo%253D&md5=5e7ebd94012ba47ecc299ad0502975de</span></div><a href="/servlet/linkout?suffix=cit127&amp;dbid=16384&amp;doi=10.1038%2Fnature22394&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature22394%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DSun%26aufirst%3DB.%26aulast%3DFeng%26aufirst%3DD.%26aulast%3DHu%26aufirst%3DH.%26aulast%3DChu%26aufirst%3DM.%26aulast%3DQu%26aufirst%3DQ.%26aulast%3DTarrasch%26aufirst%3DJ.%2BT.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DSun%2BKobilka%26aufirst%3DT.%26aulast%3DKobilka%26aufirst%3DB.%2BK.%26aulast%3DSkiniotis%26aufirst%3DG.%26atitle%3DCryo-EM%2520Structure%2520of%2520the%2520Activated%2520GLP-1%2520Receptor%2520in%2520Complex%2520with%2520a%2520G%2520Protein%26jtitle%3DNature%26date%3D2017%26volume%3D546%26spage%3D248%26epage%3D253%26doi%3D10.1038%2Fnature22394" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref128"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref128'); return false;" data-citation="" class="refNumLink">128</a></strong><div class="NLM_citation" id="cit128"><span> <i>Prime Schrödinger
Release 2019-3 Prime</i>; <span class="NLM_publisher-name">Schrödinger,
LLC</span>: <span class="NLM_publisher-loc">New York,
NY</span>, <span class="NLM_year">2019</span> (accessed Jan 31, 2018).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Prime+Schr%C3%B6dinger%0ARelease+2019-3+Prime%3B+Schr%C3%B6dinger%2C%0ALLC%3A+New+York%2C%0ANY%2C+2019+%28accessed+Jan+31%2C+2018%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit128&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DPrime%2520Schr%25C3%25B6dinger%250ARelease%25202019-3%2520Prime%26pub%3DSchr%25C3%25B6dinger%252C%250ALLC%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref129"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref129'); return false;" data-citation="" class="refNumLink">129</a></strong><div class="NLM_citation" id="cit129"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Day, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ottaway, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patterson, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gelfanov, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smiley, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gidda, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Findeisen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bruemmer, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drucker, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaudhary, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holland, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hembree, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abplanalp, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grant, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruehl, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirchner, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lockie, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hofmann, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woods, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nogueiras, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pfluger, P. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perez-Tilve, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DiMarchi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tschöp, M. H.</span></span> <span> </span><span class="NLM_article-title">A New Glucagon and GLP-1 Co-Agonist Eliminates Obesity in Rodents</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">749</span>– <span class="NLM_lpage">757</span>, <span class="refDoi"> DOI: 10.1038/nchembio.209</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref129/cit129&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1038%2Fnchembio.209" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref129/cit129&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=19597507" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref129/cit129&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BD1MXosFWhsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2009&pages=749-757&author=J.+W.+Dayauthor=N.+Ottawayauthor=J.+T.+Pattersonauthor=V.+Gelfanovauthor=D.+Smileyauthor=J.+Giddaauthor=H.+Findeisenauthor=D.+Bruemmerauthor=D.+J.+Druckerauthor=N.+Chaudharyauthor=J.+Hollandauthor=J.+Hembreeauthor=W.+Abplanalpauthor=E.+Grantauthor=J.+Ruehlauthor=H.+Wilsonauthor=H.+Kirchnerauthor=S.+H.+Lockieauthor=S.+Hofmannauthor=S.+C.+Woodsauthor=R.+Nogueirasauthor=P.+T.+Pflugerauthor=D.+Perez-Tilveauthor=R.+DiMarchiauthor=M.+H.+Tsch%C3%B6p&title=A+New+Glucagon+and+GLP-1+Co-Agonist+Eliminates+Obesity+in+Rodents&doi=10.1038%2Fnchembio.209"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit129R"><div class="casContent"><span class="casTitleNuber">129</span><div class="casTitle"><span class="NLM_cas:atitle">A new glucagon and GLP-1 co-agonist eliminates obesity in rodents</span></div><div class="casAuthors">Day, Jonathan W.; Ottaway, Nickki; Patterson, James T.; Gelfanov, Vasily; Smiley, David; Gidda, Jas; Findeisen, Hannes; Bruemmer, Dennis; Drucker, Daniel J.; Chaudhary, Nilika; Holland, Jenna; Hembree, Jazzminn; Abplanalp, William; Grant, Erin; Ruehl, Jennifer; Wilson, Hilary; Kirchner, Henriette; Lockie, Sarah Haas; Hofmann, Susanna; Woods, Stephen C.; Nogueiras, Ruben; Pfluger, Paul T.; Perez-Tilve, Diego; DiMarchi, Richard; Tschoep, Matthias H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">749-757</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">We report the efficacy of a new peptide with agonism at the glucagon and GLP-1 receptors that has potent, sustained satiation-inducing and lipolytic effects.  Selective chem. modification to glucagon resulted in a loss of specificity, with minimal change to inherent activity.  The structural basis for the co-agonism appears to be a combination of local positional interactions and a change in secondary structure.  Two co-agonist peptides differing from each other only in their level of glucagon receptor agonism were studied in rodent obesity models.  Administration of PEGylated peptides once per wk normalized adiposity and glucose tolerance in diet-induced obese mice.  Redn. of body wt. was achieved by a loss of body fat resulting from decreased food intake and increased energy expenditure.  These preclin. studies indicate that when full GLP-1 agonism is augmented with an appropriate degree of glucagon receptor activation, body fat redn. can be substantially enhanced without any overt adverse effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp4K1-_eVTTKbVg90H21EOLACvtfcHk0lhrFJ9diD3tSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXosFWhsLk%253D&md5=74b57672bfd375dd4d073624a25eab9b</span></div><a href="/servlet/linkout?suffix=cit129&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.209&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.209%26sid%3Dliteratum%253Aachs%26aulast%3DDay%26aufirst%3DJ.%2BW.%26aulast%3DOttaway%26aufirst%3DN.%26aulast%3DPatterson%26aufirst%3DJ.%2BT.%26aulast%3DGelfanov%26aufirst%3DV.%26aulast%3DSmiley%26aufirst%3DD.%26aulast%3DGidda%26aufirst%3DJ.%26aulast%3DFindeisen%26aufirst%3DH.%26aulast%3DBruemmer%26aufirst%3DD.%26aulast%3DDrucker%26aufirst%3DD.%2BJ.%26aulast%3DChaudhary%26aufirst%3DN.%26aulast%3DHolland%26aufirst%3DJ.%26aulast%3DHembree%26aufirst%3DJ.%26aulast%3DAbplanalp%26aufirst%3DW.%26aulast%3DGrant%26aufirst%3DE.%26aulast%3DRuehl%26aufirst%3DJ.%26aulast%3DWilson%26aufirst%3DH.%26aulast%3DKirchner%26aufirst%3DH.%26aulast%3DLockie%26aufirst%3DS.%2BH.%26aulast%3DHofmann%26aufirst%3DS.%26aulast%3DWoods%26aufirst%3DS.%2BC.%26aulast%3DNogueiras%26aufirst%3DR.%26aulast%3DPfluger%26aufirst%3DP.%2BT.%26aulast%3DPerez-Tilve%26aufirst%3DD.%26aulast%3DDiMarchi%26aufirst%3DR.%26aulast%3DTsch%25C3%25B6p%26aufirst%3DM.%2BH.%26atitle%3DA%2520New%2520Glucagon%2520and%2520GLP-1%2520Co-Agonist%2520Eliminates%2520Obesity%2520in%2520Rodents%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2009%26volume%3D5%26spage%3D749%26epage%3D757%26doi%3D10.1038%2Fnchembio.209" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref130"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref130'); return false;" data-citation="" class="refNumLink">130</a></strong><div class="NLM_citation" id="cit130"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Miranda, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winters, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gegg, C. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aral, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diamond, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guido, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stanislaus, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rose, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poppe, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Véniant, M. M.</span></span> <span> </span><span class="NLM_article-title">Design and Synthesis of Conformationally Constrained Glucagon-like Peptide-1 Derivatives with Increased Plasma Stability and Prolonged in Vivo Activity</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">2758</span>– <span class="NLM_lpage">2765</span>, <span class="refDoi"> DOI: 10.1021/jm701522b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm701522b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref130/cit130&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BD1cXks1Kmu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=2758-2765&author=L.+P.+Mirandaauthor=K.+A.+Wintersauthor=C.+V.+Geggauthor=A.+Patelauthor=J.+Aralauthor=J.+Longauthor=J.+Zhangauthor=S.+Diamondauthor=M.+Guidoauthor=S.+Stanislausauthor=M.+Maauthor=H.+Liauthor=M.+J.+Roseauthor=L.+Poppeauthor=M.+M.+V%C3%A9niant&title=Design+and+Synthesis+of+Conformationally+Constrained+Glucagon-like+Peptide-1+Derivatives+with+Increased+Plasma+Stability+and+Prolonged+in+Vivo+Activity&doi=10.1021%2Fjm701522b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit130R"><div class="casContent"><span class="casTitleNuber">130</span><div class="casTitle"><span class="NLM_cas:atitle">Design and Synthesis of Conformationally Constrained Glucagon-Like Peptide-1 Derivatives with Increased Plasma Stability and Prolonged in Vivo Activity</span></div><div class="casAuthors">Miranda, Les P.; Winters, Katherine A.; Gegg, Colin V.; Patel, Ankita; Aral, Jennifer; Long, Jason; Zhang, Jingwen; Diamond, Stephanie; Guido, Mark; Stanislaus, Shanaka; Ma, Mark; Li, Hongyan; Rose, Mark J.; Poppe, Leszek; Veniant, Murielle M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2758-2765</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A series of conformationally constrained derivs. of glucagon-like peptide-1 (GLP-1) were designed and evaluated.  By use of [Gly8]GLP-1(7-37)-NH2 (2) peptide as a starting point, 17 cyclic derivs. possessing i to i + 4, i to i + 5, or i to i + 7 side chain to side chain lactam bridges from positions 18 to 30 were prepd.  The effect of a helix-promoting α-amino-isobutyric acid (Aib) substitution at position 22 was also evaluated.  The introduction of i to i + 4 glutamic acid-lysine lactam constraints in c[Glu18-Lys22][Gly8]GLP-1(7-37)-NH2 (6), c[Glu22-Lys26][Gly8]GLP-1(7-37)-NH2 (10), and c[Glu23-Lys27][Gly8]GLP-1(7-37)-NH2 (11) resulted in potent functional activity and receptor affinities comparable to native GLP-1.  Selected GLP-1 peptides were chemoselectively PEGylated in order to prolong their in vivo activity.  PEGylated peptides [Gly8,Aib22]GLP-1(7-37)-Cys(PEG)-Ala-NH2 (23) and c[Glu22-Lys26][Gly8]GLP-1(7-37)-Cys(PEG)-Ser-Gly-NH2 (24) retained picomolar functional potency and avid receptor binding properties.  Importantly, PEGylated GLP-1 peptide 23 exhibited sustained in vivo efficacy with respect to blood glucose redn. and decreased body wt. for several days in nonhuman primates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGozaW6frxUWw7Vg90H21EOLACvtfcHk0lhrFJ9diD3tSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXks1Kmu7o%253D&md5=8526495a8a21e973a602a3cf55089d97</span></div><a href="/servlet/linkout?suffix=cit130&amp;dbid=16384&amp;doi=10.1021%2Fjm701522b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm701522b%26sid%3Dliteratum%253Aachs%26aulast%3DMiranda%26aufirst%3DL.%2BP.%26aulast%3DWinters%26aufirst%3DK.%2BA.%26aulast%3DGegg%26aufirst%3DC.%2BV.%26aulast%3DPatel%26aufirst%3DA.%26aulast%3DAral%26aufirst%3DJ.%26aulast%3DLong%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DDiamond%26aufirst%3DS.%26aulast%3DGuido%26aufirst%3DM.%26aulast%3DStanislaus%26aufirst%3DS.%26aulast%3DMa%26aufirst%3DM.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DRose%26aufirst%3DM.%2BJ.%26aulast%3DPoppe%26aufirst%3DL.%26aulast%3DV%25C3%25A9niant%26aufirst%3DM.%2BM.%26atitle%3DDesign%2520and%2520Synthesis%2520of%2520Conformationally%2520Constrained%2520Glucagon-like%2520Peptide-1%2520Derivatives%2520with%2520Increased%2520Plasma%2520Stability%2520and%2520Prolonged%2520in%2520Vivo%2520Activity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D2758%26epage%3D2765%26doi%3D10.1021%2Fjm701522b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref131"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref131'); return false;" data-citation="" class="refNumLink">131</a></strong><div class="NLM_citation" id="cit131"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murage, E. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bisello, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahn, J.-M.</span></span> <span> </span><span class="NLM_article-title">Development of Potent Glucagon-like Peptide-1 Agonists with High Enzyme Stability via Introduction of Multiple Lactam Bridges</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">6412</span>– <span class="NLM_lpage">6420</span>, <span class="refDoi"> DOI: 10.1021/jm100602m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm100602m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref131/cit131&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC3cXpvVarsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=6412-6420&author=E.+N.+Murageauthor=G.+Gaoauthor=A.+Biselloauthor=J.-M.+Ahn&title=Development+of+Potent+Glucagon-like+Peptide-1+Agonists+with+High+Enzyme+Stability+via+Introduction+of+Multiple+Lactam+Bridges&doi=10.1021%2Fjm100602m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit131R"><div class="casContent"><span class="casTitleNuber">131</span><div class="casTitle"><span class="NLM_cas:atitle">Development of Potent Glucagon-like Peptide-1 Agonists with High Enzyme Stability via Introduction of Multiple Lactam Bridges</span></div><div class="casAuthors">Murage, Eunice N.; Gao, Guangzu; Bisello, Alessandro; Ahn, Jung-Mo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">6412-6420</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Glucagon-like peptide-1 (GLP-1) has the ability to lower the blood glucose level, and its regulatory functions make it an attractive therapeutic agent for the treatment of type 2 diabetes.  However, its rapid degrdn. by enzymes like dipeptidyl peptidase-IV (DPP-IV) and neutral endopeptidase (NEP) 24.11 severely compromises its effective clin. use.  Whereas specific DPP-IV inhibitors have been developed, NEP 24.11 targets multiple sites in the GLP-1 sequence, which makes it difficult to block.  To address this drawback, we have designed and synthesized conformationally constrained GLP-1 analogs by introducing multiple lactam bridges that stabilized both α-helixes in the N- and C-terminal regions simultaneously.  In addn. to improving the receptor activation capability (up to 5-fold) by fixing the α-helical conformations required for optimal receptor interaction, the introduced lactam bridges provided outstanding shielding over NEP 24.11 (half-life of >96 h).  These highly constrained peptides are the first examples of NEP 24.11-resistant GLP-1 analogs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGQuuDP1-5D7Vg90H21EOLACvtfcHk0ljwm7H1Lds5OQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXpvVarsbw%253D&md5=a7a8385aa91eae3f44d1231c8bb937d7</span></div><a href="/servlet/linkout?suffix=cit131&amp;dbid=16384&amp;doi=10.1021%2Fjm100602m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm100602m%26sid%3Dliteratum%253Aachs%26aulast%3DMurage%26aufirst%3DE.%2BN.%26aulast%3DGao%26aufirst%3DG.%26aulast%3DBisello%26aufirst%3DA.%26aulast%3DAhn%26aufirst%3DJ.-M.%26atitle%3DDevelopment%2520of%2520Potent%2520Glucagon-like%2520Peptide-1%2520Agonists%2520with%2520High%2520Enzyme%2520Stability%2520via%2520Introduction%2520of%2520Multiple%2520Lactam%2520Bridges%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D6412%26epage%3D6420%26doi%3D10.1021%2Fjm100602m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref132"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref132'); return false;" data-citation="" class="refNumLink">132</a></strong><div class="NLM_citation" id="cit132"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murage, E. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schroeder, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beinborn, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahn, J.-M.</span></span> <span> </span><span class="NLM_article-title">Search for Alpha-Helical Propensity in the Receptor-Bound Conformation of Glucagon-like Peptide-1</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">10106</span>– <span class="NLM_lpage">10112</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2008.10.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref132/cit132&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1016%2Fj.bmc.2008.10.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref132/cit132&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=18952440" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref132/cit132&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsVSgt7bK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2008&pages=10106-10112&author=E.+N.+Murageauthor=J.+C.+Schroederauthor=M.+Beinbornauthor=J.-M.+Ahn&title=Search+for+Alpha-Helical+Propensity+in+the+Receptor-Bound+Conformation+of+Glucagon-like+Peptide-1&doi=10.1016%2Fj.bmc.2008.10.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit132R"><div class="casContent"><span class="casTitleNuber">132</span><div class="casTitle"><span class="NLM_cas:atitle">Search for α-helical propensity in the receptor-bound conformation of glucagon-like peptide-1</span></div><div class="casAuthors">Murage, Eunice N.; Schroeder, Jonathan C.; Beinborn, Martin; Ahn, Jung-Mo</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">10106-10112</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">To elucidate the receptor-bound conformation of glucagon-like peptide-1 (GLP-1), a series of conformationally constrained GLP-1 analogs were synthesized by introducing lactam bridges between Lysi and Glui +4 to form α-helixes at various positions.  The activity and affinity of these analogs to GLP-1 receptors suggested that the receptor-bound conformation comprises two α-helical segments between residues 11-21 and 23-34.  It is notable that the N-terminal α-helix is extended to Thr11, and that Gly22 plays a pivotal role in arranging the two α-helixes.  Based on these findings, a highly potent bicyclic GLP-1 analog was synthesized which is the most conformationally constrained GLP-1 analog reported to date.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9FswyXDckPrVg90H21EOLACvtfcHk0ljwm7H1Lds5OQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsVSgt7bK&md5=cd451fed8eedd20d058e63d0adc01bdf</span></div><a href="/servlet/linkout?suffix=cit132&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2008.10.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2008.10.006%26sid%3Dliteratum%253Aachs%26aulast%3DMurage%26aufirst%3DE.%2BN.%26aulast%3DSchroeder%26aufirst%3DJ.%2BC.%26aulast%3DBeinborn%26aufirst%3DM.%26aulast%3DAhn%26aufirst%3DJ.-M.%26atitle%3DSearch%2520for%2520Alpha-Helical%2520Propensity%2520in%2520the%2520Receptor-Bound%2520Conformation%2520of%2520Glucagon-like%2520Peptide-1%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2008%26volume%3D16%26spage%3D10106%26epage%3D10112%26doi%3D10.1016%2Fj.bmc.2008.10.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref133"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref133'); return false;" data-citation="" class="refNumLink">133</a></strong><div class="NLM_citation" id="cit133"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schreuder, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liesum, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lönze, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stump, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffmann, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiell, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toti, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bauer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kallus, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klemke-Jahn, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Czech, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kramer, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Enke, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niedermeyer, T. H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morrison, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brönstrup, M.</span></span> <span> </span><span class="NLM_article-title">Isolation, Co-Crystallization and Structure-Based Characterization of Anabaenopeptins as Highly Potent Inhibitors of Activated Thrombin Activatable Fibrinolysis Inhibitor (TAFIa)</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>6</i></span>, <span class="NLM_elocation-id">32958</span> <span class="refDoi"> DOI: 10.1038/srep32958</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref133/cit133&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1038%2Fsrep32958" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref133/cit133&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=27604544" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref133/cit133&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC2sXps1OktA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2016&author=H.+Schreuderauthor=A.+Liesumauthor=P.+L%C3%B6nzeauthor=H.+Stumpauthor=H.+Hoffmannauthor=M.+Schiellauthor=M.+Kurzauthor=L.+Totiauthor=A.+Bauerauthor=C.+Kallusauthor=C.+Klemke-Jahnauthor=J.+Czechauthor=D.+Kramerauthor=H.+Enkeauthor=T.+H.+J.+Niedermeyerauthor=V.+Morrisonauthor=V.+Kumarauthor=M.+Br%C3%B6nstrup&title=Isolation%2C+Co-Crystallization+and+Structure-Based+Characterization+of+Anabaenopeptins+as+Highly+Potent+Inhibitors+of+Activated+Thrombin+Activatable+Fibrinolysis+Inhibitor+%28TAFIa%29&doi=10.1038%2Fsrep32958"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit133R"><div class="casContent"><span class="casTitleNuber">133</span><div class="casTitle"><span class="NLM_cas:atitle">Isolation, Co-Crystallization and Structure-Based Characterization of Anabaenopeptins as Highly Potent Inhibitors of Activated Thrombin Activatable Fibrinolysis Inhibitor (TAFIa)</span></div><div class="casAuthors">Schreuder, Herman; Liesum, Alexander; Loenze, Petra; Stump, Heike; Hoffmann, Holger; Schiell, Matthias; Kurz, Michael; Toti, Luigi; Bauer, Armin; Kallus, Christopher; Klemke-Jahn, Christine; Czech, Joerg; Kramer, Dan; Enke, Heike; Niedermeyer, Timo H. J.; Morrison, Vincent; Kumar, Vasant; Broenstrup, Mark</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">32958</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Mature thrombin activatable fibrinolysis inhibitor (TAFIa) is a carboxypeptidase that stabilizes fibrin clots by removing C-terminal arginines and lysines from partially degraded fibrin.  Inhibition of TAFIa stimulates the degrdn. of fibrin clots and may help to prevent thrombosis.  Applying a lead finding approach based on literature-mining, we discovered that anabaenopeptins, cyclic peptides produced by cyanobacteria, were potent inhibitors of TAFIa with IC50 values as low as 1.5 nM.  We describe the isolation and structure elucidation of 20 anabaenopeptins, including 13 novel congeners, as well as their pronounced structure-activity relationships (SAR) with respect to inhibition of TAFIa.  Crystal structures of the anabaenopeptins B, C and F bound to the surrogate protease carboxypeptidase B revealed the binding modes of these large (∼850 Da) compds. in detail and explained the obsd. SAR, i.e. the strong dependence of the potency on a basic (Arg, Lys) exocyclic residue that addressed the S1' binding pocket, and a broad tolerance towards substitutions in the pentacyclic ring that acted as a plug of the active site.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNu4VixSn7RrVg90H21EOLACvtfcHk0ljwm7H1Lds5OQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXps1OktA%253D%253D&md5=4c7faf50dcdf4bb7b4f9b88c9e064a1d</span></div><a href="/servlet/linkout?suffix=cit133&amp;dbid=16384&amp;doi=10.1038%2Fsrep32958&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep32958%26sid%3Dliteratum%253Aachs%26aulast%3DSchreuder%26aufirst%3DH.%26aulast%3DLiesum%26aufirst%3DA.%26aulast%3DL%25C3%25B6nze%26aufirst%3DP.%26aulast%3DStump%26aufirst%3DH.%26aulast%3DHoffmann%26aufirst%3DH.%26aulast%3DSchiell%26aufirst%3DM.%26aulast%3DKurz%26aufirst%3DM.%26aulast%3DToti%26aufirst%3DL.%26aulast%3DBauer%26aufirst%3DA.%26aulast%3DKallus%26aufirst%3DC.%26aulast%3DKlemke-Jahn%26aufirst%3DC.%26aulast%3DCzech%26aufirst%3DJ.%26aulast%3DKramer%26aufirst%3DD.%26aulast%3DEnke%26aufirst%3DH.%26aulast%3DNiedermeyer%26aufirst%3DT.%2BH.%2BJ.%26aulast%3DMorrison%26aufirst%3DV.%26aulast%3DKumar%26aufirst%3DV.%26aulast%3DBr%25C3%25B6nstrup%26aufirst%3DM.%26atitle%3DIsolation%252C%2520Co-Crystallization%2520and%2520Structure-Based%2520Characterization%2520of%2520Anabaenopeptins%2520as%2520Highly%2520Potent%2520Inhibitors%2520of%2520Activated%2520Thrombin%2520Activatable%2520Fibrinolysis%2520Inhibitor%2520%2528TAFIa%2529%26jtitle%3DSci.%2520Rep.%26date%3D2016%26volume%3D6%26doi%3D10.1038%2Fsrep32958" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref134"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref134'); return false;" data-citation="" class="refNumLink">134</a></strong><div class="NLM_citation" id="cit134"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Halland, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brönstrup, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Czech, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Czechtizky, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evers, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Follmann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kohlmann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiell, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schreuder, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kallus, C.</span></span> <span> </span><span class="NLM_article-title">Novel Small Molecule Inhibitors of Activated Thrombin Activatable Fibrinolysis Inhibitor (TAFIa) from Natural Product Anabaenopeptin</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">4839</span>– <span class="NLM_lpage">4844</span>, <span class="refDoi"> DOI: 10.1021/jm501840b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501840b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref134/cit134&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC2MXovVSnt7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=4839-4844&author=N.+Hallandauthor=M.+Br%C3%B6nstrupauthor=J.+Czechauthor=W.+Czechtizkyauthor=A.+Eversauthor=M.+Follmannauthor=M.+Kohlmannauthor=M.+Schiellauthor=M.+Kurzauthor=H.+A.+Schreuderauthor=C.+Kallus&title=Novel+Small+Molecule+Inhibitors+of+Activated+Thrombin+Activatable+Fibrinolysis+Inhibitor+%28TAFIa%29+from+Natural+Product+Anabaenopeptin&doi=10.1021%2Fjm501840b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit134R"><div class="casContent"><span class="casTitleNuber">134</span><div class="casTitle"><span class="NLM_cas:atitle">Novel Small Molecule Inhibitors of Activated Thrombin Activatable Fibrinolysis Inhibitor (TAFIa) from Natural Product Anabaenopeptin</span></div><div class="casAuthors">Halland, Nis; Broenstrup, Mark; Czech, Joerg; Czechtizky, Werngard; Evers, Andreas; Follmann, Markus; Kohlmann, Markus; Schiell, Matthias; Kurz, Michael; Schreuder, Herman A.; Kallus, Christopher</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4839-4844</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Anabaenopeptins isolated from cyanobacteria were identified as inhibitors of carboxypeptidase TAFIa.  Cocrystal structures of these macrocyclic natural product inhibitors in a modified porcine carboxypeptidase B revealed their binding mode and provided the basis for the rational design of small mol. inhibitors with a previously unknown central urea motif.  Optimization based on these design concepts allowed for a rapid evaluation of the SAR and delivered potent small mol. inhibitors of TAFIa with a promising overall profile, e.g. I.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7kmWgfMLAA7Vg90H21EOLACvtfcHk0lj50ANs3Krs1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXovVSnt7c%253D&md5=6e1df1c771aec9982d046148cea91358</span></div><a href="/servlet/linkout?suffix=cit134&amp;dbid=16384&amp;doi=10.1021%2Fjm501840b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501840b%26sid%3Dliteratum%253Aachs%26aulast%3DHalland%26aufirst%3DN.%26aulast%3DBr%25C3%25B6nstrup%26aufirst%3DM.%26aulast%3DCzech%26aufirst%3DJ.%26aulast%3DCzechtizky%26aufirst%3DW.%26aulast%3DEvers%26aufirst%3DA.%26aulast%3DFollmann%26aufirst%3DM.%26aulast%3DKohlmann%26aufirst%3DM.%26aulast%3DSchiell%26aufirst%3DM.%26aulast%3DKurz%26aufirst%3DM.%26aulast%3DSchreuder%26aufirst%3DH.%2BA.%26aulast%3DKallus%26aufirst%3DC.%26atitle%3DNovel%2520Small%2520Molecule%2520Inhibitors%2520of%2520Activated%2520Thrombin%2520Activatable%2520Fibrinolysis%2520Inhibitor%2520%2528TAFIa%2529%2520from%2520Natural%2520Product%2520Anabaenopeptin%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D4839%26epage%3D4844%26doi%3D10.1021%2Fjm501840b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref135"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref135'); return false;" data-citation="" class="refNumLink">135</a></strong><div class="NLM_citation" id="cit135"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Halland, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Czech, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Czechtizky, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evers, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Follmann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kohlmann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schreuder, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kallus, C.</span></span> <span> </span><span class="NLM_article-title">Sulfamide as Zinc Binding Motif in Small Molecule Inhibitors of Activated Thrombin Activatable Fibrinolysis Inhibitor (TAFIa)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">9567</span>– <span class="NLM_lpage">9573</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01276</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01276" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref135/cit135&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhs1ylu7jE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=9567-9573&author=N.+Hallandauthor=J.+Czechauthor=W.+Czechtizkyauthor=A.+Eversauthor=M.+Follmannauthor=M.+Kohlmannauthor=H.+A.+Schreuderauthor=C.+Kallus&title=Sulfamide+as+Zinc+Binding+Motif+in+Small+Molecule+Inhibitors+of+Activated+Thrombin+Activatable+Fibrinolysis+Inhibitor+%28TAFIa%29&doi=10.1021%2Facs.jmedchem.6b01276"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit135R"><div class="casContent"><span class="casTitleNuber">135</span><div class="casTitle"><span class="NLM_cas:atitle">Sulfamide as Zinc Binding Motif in Small Molecule Inhibitors of Activated Thrombin Activatable Fibrinolysis Inhibitor (TAFIa)</span></div><div class="casAuthors">Halland, Nis; Czech, Joerg; Czechtizky, Werngard; Evers, Andreas; Follmann, Markus; Kohlmann, Markus; Schreuder, Herman A.; Kallus, Christopher</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">9567-9573</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Previously disclosed TAFIa inhibitors having a urea zinc-binding motif were used as the starting point for the development of a novel class of highly potent inhibitors having a sulfamide zinc-binding motif.  High-resoln. X-ray cocrystal structures were used to optimize the structures and reveal a highly unusual sulfamide configuration.  A selected sulfamide was profiled in vitro and in vivo and displayed a promising ADMET profile.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoj1LpJOb8JYrVg90H21EOLACvtfcHk0lj50ANs3Krs1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhs1ylu7jE&md5=8ffb8d0719f609351b9a67271d94fef4</span></div><a href="/servlet/linkout?suffix=cit135&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01276&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01276%26sid%3Dliteratum%253Aachs%26aulast%3DHalland%26aufirst%3DN.%26aulast%3DCzech%26aufirst%3DJ.%26aulast%3DCzechtizky%26aufirst%3DW.%26aulast%3DEvers%26aufirst%3DA.%26aulast%3DFollmann%26aufirst%3DM.%26aulast%3DKohlmann%26aufirst%3DM.%26aulast%3DSchreuder%26aufirst%3DH.%2BA.%26aulast%3DKallus%26aufirst%3DC.%26atitle%3DSulfamide%2520as%2520Zinc%2520Binding%2520Motif%2520in%2520Small%2520Molecule%2520Inhibitors%2520of%2520Activated%2520Thrombin%2520Activatable%2520Fibrinolysis%2520Inhibitor%2520%2528TAFIa%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D9567%26epage%3D9573%26doi%3D10.1021%2Facs.jmedchem.6b01276" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref136"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref136'); return false;" data-citation="" class="refNumLink">136</a></strong><div class="NLM_citation" id="cit136"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, Y.</span></span> <span> </span><span class="NLM_article-title">PROTACs: Great Opportunities for Academia and Industry</span>. <i>Signal Transduct. Target. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>4</i></span>, <span class="NLM_elocation-id">64</span> <span class="refDoi"> DOI: 10.1038/s41392-019-0101-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref136/cit136&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1038%2Fs41392-019-0101-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref136/cit136&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=31885879" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref136/cit136&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A280%3ADC%252BB3MbkvFCrtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2019&author=X.+Sunauthor=H.+Gaoauthor=Y.+Yangauthor=M.+Heauthor=Y.+Wuauthor=Y.+Songauthor=Y.+Tongauthor=Y.+Rao&title=PROTACs%3A+Great+Opportunities+for+Academia+and+Industry&doi=10.1038%2Fs41392-019-0101-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit136R"><div class="casContent"><span class="casTitleNuber">136</span><div class="casTitle"><span class="NLM_cas:atitle">PROTACs: great opportunities for academia and industry</span></div><div class="casAuthors">Sun Xiuyun; Gao Hongying; Yang Yiqing; He Ming; Wu Yue; Song Yugang; Tong Yan; Rao Yu; Sun Xiuyun; Gao Hongying; Yang Yiqing; Rao Yu</div><div class="citationInfo"><span class="NLM_cas:title">Signal transduction and targeted therapy</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">64</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Although many kinds of therapies are applied in the clinic, drug-resistance is a major and unavoidable problem.  Another disturbing statistic is the limited number of drug targets, which are presently only 20-25% of all protein targets that are currently being studied.  Moreover, the focus of current explorations of targets are their enzymatic functions, which ignores the functions from their scaffold moiety.  As a promising and appealing technology, PROteolysis TArgeting Chimeras (PROTACs) have attracted great attention both from academia and industry for finding available approaches to solve the above problems.  PROTACs regulate protein function by degrading target proteins instead of inhibiting them, providing more sensitivity to drug-resistant targets and a greater chance to affect the nonenzymatic functions.  PROTACs have been proven to show better selectivity compared to classic inhibitors.  PROTACs can be described as a chemical knockdown approach with rapidity and reversibility, which presents new and different biology compared to other gene editing tools by avoiding misinterpretations that arise from potential genetic compensation and/or spontaneous mutations.  PRTOACs have been widely explored throughout the world and have outperformed not only in cancer diseases, but also in immune disorders, viral infections and neurodegenerative diseases.  Although PROTACs present a very promising and powerful approach for crossing the hurdles of present drug discovery and tool development in biology, more efforts are needed to gain to get deeper insight into the efficacy and safety of PROTACs in the clinic.  More target binders and more E3 ligases applicable for developing PROTACs are waiting for exploration.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQretvERU2xT-_Q9URG6Dx8fW6udTcc2ebJY92RdknVRrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3MbkvFCrtA%253D%253D&md5=7d9545a21e3a78cededf2977a31e8f7d</span></div><a href="/servlet/linkout?suffix=cit136&amp;dbid=16384&amp;doi=10.1038%2Fs41392-019-0101-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41392-019-0101-6%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DX.%26aulast%3DGao%26aufirst%3DH.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DHe%26aufirst%3DM.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DSong%26aufirst%3DY.%26aulast%3DTong%26aufirst%3DY.%26aulast%3DRao%26aufirst%3DY.%26atitle%3DPROTACs%253A%2520Great%2520Opportunities%2520for%2520Academia%2520and%2520Industry%26jtitle%3DSignal%2520Transduct.%2520Target.%2520Ther.%26date%3D2019%26volume%3D4%26doi%3D10.1038%2Fs41392-019-0101-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref137"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref137'); return false;" data-citation="" class="refNumLink">137</a></strong><div class="NLM_citation" id="cit137"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Testa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucas, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wright, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Structure-Based Design of a Macrocyclic PROTAC</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">1727</span>– <span class="NLM_lpage">1734</span>, <span class="refDoi"> DOI: 10.1002/anie.201914396</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref137/cit137&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1002%2Fanie.201914396" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref137/cit137&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitlyitbjK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2020&pages=1727-1734&author=A.+Testaauthor=S.+J.+Hughesauthor=X.+Lucasauthor=J.+E.+Wrightauthor=A.+Ciulli&title=Structure-Based+Design+of+a+Macrocyclic+PROTAC&doi=10.1002%2Fanie.201914396"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit137R"><div class="casContent"><span class="casTitleNuber">137</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Design of a Macrocyclic PROTAC</span></div><div class="casAuthors">Testa, Andrea; Hughes, Scott J.; Lucas, Xavier; Wright, Jane E.; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1727-1734</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Constraining a mol. in its bioactive conformation via macrocyclization represents an attractive strategy to rationally design functional chem. probes.  While this approach has been applied to enzyme inhibitors or receptor antagonists, to date it remains unprecedented for bifunctional mols. that bring proteins together, such as PROTAC degraders.  Herein, the authors report the design and synthesis of a macrocyclic PROTAC by adding a cyclizing linker to the BET degrader MZ1.  A cocrystal structure of macroPROTAC-1 bound in a ternary complex with VHL and the second bromodomain of Brd4 validated the rational design.  Biophys. studies revealed enhanced discrimination between the second and the first bromodomains of BET proteins.  Despite a 12-fold loss of binary binding affinity for Brd4, macroPROTAC-1 exhibited cellular activity comparable to MZ1.  The findings support macrocyclization as an advantageous strategy to enhance PROTAC degrdn. potency and selectivity between homologous targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZjLgy_V3_abVg90H21EOLACvtfcHk0lhej-PdpRq8Kw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitlyitbjK&md5=6f3f279f6220ec30b30ca3d2c931da95</span></div><a href="/servlet/linkout?suffix=cit137&amp;dbid=16384&amp;doi=10.1002%2Fanie.201914396&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201914396%26sid%3Dliteratum%253Aachs%26aulast%3DTesta%26aufirst%3DA.%26aulast%3DHughes%26aufirst%3DS.%2BJ.%26aulast%3DLucas%26aufirst%3DX.%26aulast%3DWright%26aufirst%3DJ.%2BE.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DStructure-Based%2520Design%2520of%2520a%2520Macrocyclic%2520PROTAC%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2020%26volume%3D59%26spage%3D1727%26epage%3D1734%26doi%3D10.1002%2Fanie.201914396" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref138"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref138'); return false;" data-citation="" class="refNumLink">138</a></strong><div class="NLM_citation" id="cit138"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Molecular Recognition of Ternary Complexes: A New Dimension in the Structure-Guided Design of Chemical Degraders</span>. <i>Essays Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">505</span>– <span class="NLM_lpage">516</span>, <span class="refDoi"> DOI: 10.1042/EBC20170041</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref138/cit138&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1042%2FEBC20170041" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref138/cit138&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=29118097" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref138/cit138&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A280%3ADC%252BC1M7pt1Gmuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2017&pages=505-516&author=S.+J.+Hughesauthor=A.+Ciulli&title=Molecular+Recognition+of+Ternary+Complexes%3A+A+New+Dimension+in+the+Structure-Guided+Design+of+Chemical+Degraders&doi=10.1042%2FEBC20170041"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit138R"><div class="casContent"><span class="casTitleNuber">138</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular recognition of ternary complexes: a new dimension in the structure-guided design of chemical degraders</span></div><div class="casAuthors">Hughes Scott J; Ciulli Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Essays in biochemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">505-516</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Molecular glues and bivalent inducers of protein degradation (also known as PROTACs) represent a fascinating new modality in pharmacotherapeutics: the potential to knockdown previously thought 'undruggable' targets at sub-stoichiometric concentrations in ways not possible using conventional inhibitors.  Mounting evidence suggests these chemical agents, in concert with their target proteins, can be modelled as three-body binding equilibria that can exhibit significant cooperativity as a result of specific ligand-induced molecular recognition.  Despite this, many existing drug design and optimization regimens still fixate on binary target engagement, in part due to limited structural data on ternary complexes.  Recent crystal structures of protein complexes mediated by degrader molecules, including the first PROTAC ternary complex, underscore the importance of protein-protein interactions and intramolecular contacts to the mode of action of this class of compounds.  These discoveries have opened the door to a new paradigm for structure-guided drug design: borrowing surface area and molecular recognition from nature to elicit cellular signalling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTqZGQ3bfUY0b5XHngOXiV_fW6udTcc2eZxzE-4kwrZ2Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M7pt1Gmuw%253D%253D&md5=7470e559372f05d34c5924524a6d04bc</span></div><a href="/servlet/linkout?suffix=cit138&amp;dbid=16384&amp;doi=10.1042%2FEBC20170041&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FEBC20170041%26sid%3Dliteratum%253Aachs%26aulast%3DHughes%26aufirst%3DS.%2BJ.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DMolecular%2520Recognition%2520of%2520Ternary%2520Complexes%253A%2520A%2520New%2520Dimension%2520in%2520the%2520Structure-Guided%2520Design%2520of%2520Chemical%2520Degraders%26jtitle%3DEssays%2520Biochem.%26date%3D2017%26volume%3D61%26spage%3D505%26epage%3D516%26doi%3D10.1042%2FEBC20170041" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref139"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref139'); return false;" data-citation="" class="refNumLink">139</a></strong><div class="NLM_citation" id="cit139"><span><span class="NLM_contrib-group"><span class="NLM_string-name">De Guzman, R. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goto, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dyson, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wright, P. E.</span></span> <span> </span><span class="NLM_article-title">Structural Basis for Cooperative Transcription Factor Binding to the CBP Coactivator</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>355</i></span>,  <span class="NLM_fpage">1005</span>– <span class="NLM_lpage">1013</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2005.09.059</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref139/cit139&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1016%2Fj.jmb.2005.09.059" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref139/cit139&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=16253272" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref139/cit139&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtlGlsLzO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=355&publication_year=2006&pages=1005-1013&author=R.+N.+De+Guzmanauthor=N.+K.+Gotoauthor=H.+J.+Dysonauthor=P.+E.+Wright&title=Structural+Basis+for+Cooperative+Transcription+Factor+Binding+to+the+CBP+Coactivator&doi=10.1016%2Fj.jmb.2005.09.059"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit139R"><div class="casContent"><span class="casTitleNuber">139</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Basis for Cooperative Transcription Factor Binding to the CBP Coactivator</span></div><div class="casAuthors">De Guzman, Roberto N.; Goto, Natalie K.; Dyson, H. Jane; Wright, Peter E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">355</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1005-1013</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Regulation of transcription requires interactions between transcriptional activators and transcriptional co-activator CREB binding protein (CBP).  The KIX domain of CBP can bind simultaneously to two different proteins, providing an addnl. mechanism for transcriptional regulation.  Here we describe the soln. structure of the ternary complex formed by cooperative binding of activation domains from the c-Myb and mixed lineage leukemia (MLL) transcription factors to the KIX domain.  The MLL and c-Myb domains form helixes that bind to two distinct hydrophobic grooves on opposite faces of KIX.  Compared to the binary KIX:c-Myb complex, significant changes are obsd. in the structure of KIX at the MLL binding interface in the ternary complex.  Two regions of KIX that are disordered in the binary complex become structured in the ternary complex: a flexible loop forms intimate contacts with bound MLL, and the C-terminal helix is extended and stabilized by MLL binding.  This structural change results in the formation of addnl. electrostatic/polar interactions between KIX and the bound c-Myb, providing a structural basis for the cooperativity obsd. for the ternary complex.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2Ig7OPYgRFbVg90H21EOLACvtfcHk0lhej-PdpRq8Kw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtlGlsLzO&md5=a7cb1e86b16731b387558cb10bd62d8d</span></div><a href="/servlet/linkout?suffix=cit139&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2005.09.059&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2005.09.059%26sid%3Dliteratum%253Aachs%26aulast%3DDe%2BGuzman%26aufirst%3DR.%2BN.%26aulast%3DGoto%26aufirst%3DN.%2BK.%26aulast%3DDyson%26aufirst%3DH.%2BJ.%26aulast%3DWright%26aufirst%3DP.%2BE.%26atitle%3DStructural%2520Basis%2520for%2520Cooperative%2520Transcription%2520Factor%2520Binding%2520to%2520the%2520CBP%2520Coactivator%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2006%26volume%3D355%26spage%3D1005%26epage%3D1013%26doi%3D10.1016%2Fj.jmb.2005.09.059" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref140"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref140'); return false;" data-citation="" class="refNumLink">140</a></strong><div class="NLM_citation" id="cit140"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guedes, I. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pereira, F. S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dardenne, L. E.</span></span> <span> </span><span class="NLM_article-title">Empirical Scoring Functions for Structure-Based Virtual Screening: Applications, Critical Aspects, and Challenges</span>. <i>Front. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>, <span class="NLM_elocation-id">1089</span> <span class="refDoi"> DOI: 10.3389/fphar.2018.01089</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref140/cit140&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.3389%2Ffphar.2018.01089" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref140/cit140&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=30319422" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref140/cit140&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC1MXpt1WqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&author=I.+A.+Guedesauthor=F.+S.+S.+Pereiraauthor=L.+E.+Dardenne&title=Empirical+Scoring+Functions+for+Structure-Based+Virtual+Screening%3A+Applications%2C+Critical+Aspects%2C+and+Challenges&doi=10.3389%2Ffphar.2018.01089"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit140R"><div class="casContent"><span class="casTitleNuber">140</span><div class="casTitle"><span class="NLM_cas:atitle">Empirical scoring functions for structure-based virtual screening: applications, critical aspects, and challenges</span></div><div class="casAuthors">Guedes, Isabella A.; Pereira, Felipe S. S.; Dardenne, Laurent E.</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in Pharmacology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1089</span>CODEN:
                <span class="NLM_cas:coden">FPRHAU</span>;
        ISSN:<span class="NLM_cas:issn">1663-9812</span>.
    
            (<span class="NLM_cas:orgname">Frontiers Media S.A.</span>)
        </div><div class="casAbstract">A review.  Structure-based virtual screening (VS) is a widely used approach that employs the knowledge of the three-dimensional structure of the target of interest in the design of new lead compds. from large-scale mol. docking expts.  Through the prediction of the binding mode and affinity of a small mol. within the binding site of the target of interest, it is possible to understand important properties related to the binding process.  Empirical scoring functions are widely used for pose and affinity prediction.  Although pose prediction is performed with satisfactory accuracy, the correct prediction of binding affinity is still a challenging task and crucial for the success of structure-based VS expts.  There are several efforts in distinct fronts to develop even more sophisticated and accurate models for filtering and ranking large libraries of compds.  This paper will cover some recent successful applications and methodol. advances, including strategies to explore the ligand entropy and solvent effects, training with sophisticated machine-learning techniques, and the use of quantum mechanics.  Particular emphasis will be given to the discussion of crit. aspects and further directions for the development of more accurate empirical scoring functions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpuNmYBXPKivLVg90H21EOLACvtfcHk0lhej-PdpRq8Kw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXpt1WqsLk%253D&md5=6fd60ccc4d6c2c85a206030023f7c0d1</span></div><a href="/servlet/linkout?suffix=cit140&amp;dbid=16384&amp;doi=10.3389%2Ffphar.2018.01089&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffphar.2018.01089%26sid%3Dliteratum%253Aachs%26aulast%3DGuedes%26aufirst%3DI.%2BA.%26aulast%3DPereira%26aufirst%3DF.%2BS.%2BS.%26aulast%3DDardenne%26aufirst%3DL.%2BE.%26atitle%3DEmpirical%2520Scoring%2520Functions%2520for%2520Structure-Based%2520Virtual%2520Screening%253A%2520Applications%252C%2520Critical%2520Aspects%252C%2520and%2520Challenges%26jtitle%3DFront.%2520Pharmacol.%26date%3D2018%26volume%3D9%26doi%3D10.3389%2Ffphar.2018.01089" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref141"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref141'); return false;" data-citation="" class="refNumLink">141</a></strong><div class="NLM_citation" id="cit141"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Le Roux, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blaise, É.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boudreault, P.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Comeau, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doucet, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giarrusso, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collin, M.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neubauer, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kölling, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Göller, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seep, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tshitenge, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wittwer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kullmann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hillisch, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mittendorf, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marsault, E.</span></span> <span> </span><span class="NLM_article-title">Structure–Permeability Relationship of Semipeptidic Macrocycles—Understanding and Optimizing Passive Permeability and Efflux Ratio</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">6774</span>– <span class="NLM_lpage">6783</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c00013</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00013" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref141/cit141&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtVSht73F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=6774-6783&author=A.+Le+Rouxauthor=%C3%89.+Blaiseauthor=P.-L.+Boudreaultauthor=C.+Comeauauthor=A.+Doucetauthor=M.+Giarrussoauthor=M.-P.+Collinauthor=T.+Neubauerauthor=F.+K%C3%B6llingauthor=A.+H.+G%C3%B6llerauthor=L.+Seepauthor=D.+T.+Tshitengeauthor=M.+Wittwerauthor=M.+Kullmannauthor=A.+Hillischauthor=J.+Mittendorfauthor=E.+Marsault&title=Structure%E2%80%93Permeability+Relationship+of+Semipeptidic+Macrocycles%E2%80%94Understanding+and+Optimizing+Passive+Permeability+and+Efflux+Ratio&doi=10.1021%2Facs.jmedchem.0c00013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit141R"><div class="casContent"><span class="casTitleNuber">141</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Permeability Relationship of Semipeptidic Macrocycles-Understanding and Optimizing Passive Permeability and Efflux Ratio</span></div><div class="casAuthors">Le Roux, Antoine; Blaise, Emilie; Boudreault, Pierre-Luc; Comeau, Christian; Doucet, Annie; Giarrusso, Marilena; Collin, Marie-Pierre; Neubauer, Thomas; Kolling, Florian; Goller, Andreas H.; Seep, Lea; Tshitenge, Dieudonne T.; Wittwer, Matthias; Kullmann, Maximilian; Hillisch, Alexander; Mittendorf, Joachim; Marsault, Eric</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">6774-6783</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We herein report the first thorough anal. of the structure-permeability relationship of semipeptidic macrocycles.  In total, 47 macrocycles were synthesized using a hybrid solid-phase/soln. strategy, and then their passive and cellular permeability was assessed using the parallel artificial membrane permeability assay (PAMPA) and Caco-2 assay, resp.  The results indicate that semipeptidic macrocycles generally possess high passive permeability based on the PAMPA, yet their cellular permeability is governed by efflux, as reported in the Caco-2 assay.  Structural variations led to tractable structure-permeability and structure-efflux relationships, wherein the linker length, stereoinversion, N-methylation, and peptoids site-specifically impact the permeability and efflux.  Extensive NMR, mol. dynamics, and ensemble-based three-dimensional polar surface area (3D-PSA) studies showed that ensemble-based 3D-PSA is a good predictor of passive permeability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqCamxP_rXRLVg90H21EOLACvtfcHk0ljC364cqNnqDw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtVSht73F&md5=2794db50565ab0efbc123d0ec541ff32</span></div><a href="/servlet/linkout?suffix=cit141&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00013%26sid%3Dliteratum%253Aachs%26aulast%3DLe%2BRoux%26aufirst%3DA.%26aulast%3DBlaise%26aufirst%3D%25C3%2589.%26aulast%3DBoudreault%26aufirst%3DP.-L.%26aulast%3DComeau%26aufirst%3DC.%26aulast%3DDoucet%26aufirst%3DA.%26aulast%3DGiarrusso%26aufirst%3DM.%26aulast%3DCollin%26aufirst%3DM.-P.%26aulast%3DNeubauer%26aufirst%3DT.%26aulast%3DK%25C3%25B6lling%26aufirst%3DF.%26aulast%3DG%25C3%25B6ller%26aufirst%3DA.%2BH.%26aulast%3DSeep%26aufirst%3DL.%26aulast%3DTshitenge%26aufirst%3DD.%2BT.%26aulast%3DWittwer%26aufirst%3DM.%26aulast%3DKullmann%26aufirst%3DM.%26aulast%3DHillisch%26aufirst%3DA.%26aulast%3DMittendorf%26aufirst%3DJ.%26aulast%3DMarsault%26aufirst%3DE.%26atitle%3DStructure%25E2%2580%2593Permeability%2520Relationship%2520of%2520Semipeptidic%2520Macrocycles%25E2%2580%2594Understanding%2520and%2520Optimizing%2520Passive%2520Permeability%2520and%2520Efflux%2520Ratio%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D6774%26epage%3D6783%26doi%3D10.1021%2Facs.jmedchem.0c00013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref142"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref142'); return false;" data-citation="" class="refNumLink">142</a></strong><div class="NLM_citation" id="cit142"><span><span class="NLM_contrib-group"><span class="NLM_string-name">King, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fields, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fields, G. B.</span></span> <span> </span><span class="NLM_article-title">A Cleavage Method Which Minimizes Side Reactions Following Fmoc Solid Phase Peptide Synthesis</span>. <i>Int. J. Pept. Protein Res.</i> <span class="NLM_year" style="font-weight: bold;">1990</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">255</span>– <span class="NLM_lpage">266</span>, <span class="refDoi"> DOI: 10.1111/j.1399-3011.1990.tb00976.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref142/cit142&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1111%2Fj.1399-3011.1990.tb00976.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref142/cit142&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=2279849" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref142/cit142&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADyaK3MXisVGktg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=1990&pages=255-266&author=D.+S.+Kingauthor=C.+G.+Fieldsauthor=G.+B.+Fields&title=A+Cleavage+Method+Which+Minimizes+Side+Reactions+Following+Fmoc+Solid+Phase+Peptide+Synthesis&doi=10.1111%2Fj.1399-3011.1990.tb00976.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit142R"><div class="casContent"><span class="casTitleNuber">142</span><div class="casTitle"><span class="NLM_cas:atitle">A cleavage method which minimizes side reactions following Fmoc solid phase peptide synthesis</span></div><div class="casAuthors">King, David S.; Fields, Cynthia G.; Fields, Gregg B.</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Peptide & Protein Research</span>
        (<span class="NLM_cas:date">1990</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">255-66</span>CODEN:
                <span class="NLM_cas:coden">IJPPC3</span>;
        ISSN:<span class="NLM_cas:issn">0367-8377</span>.
    </div><div class="casAbstract">The success of solid phase peptide synthesis utilizing 9-fluorenylmethoxycarbonyl (Fmoc) amino acids is often limited by deleterious side reactions which occur during CF3CO2H (TFA) peptide-resin cleavage and side-chain deprotection.  The majority of these side reactions modify susceptible residues, such as Trp, Tyr, Met, and Cys, with TFA-liberated side-chain protecting groups and linkers.  The relative effectiveness of various scavengers in suppressing these side reactions was studied.  The cleavage mixt. 82.5% TFA : 5% H2O : 5% thioanisole : 2.5% 1,2-ethanedithiol (reagent K) was maximally efficient in inhibiting a great variety of side reactions.  Synthesis and cleavage of 10 peptides, each contg. 20-50 residues, demonstrated the complementarity of Fmoc chem. with reagent K for efficient synthesis of complex peptide.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqUwcooNm7oD7Vg90H21EOLACvtfcHk0ljC364cqNnqDw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3MXisVGktg%253D%253D&md5=ecf761fc6877559af82062c6345306ab</span></div><a href="/servlet/linkout?suffix=cit142&amp;dbid=16384&amp;doi=10.1111%2Fj.1399-3011.1990.tb00976.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1399-3011.1990.tb00976.x%26sid%3Dliteratum%253Aachs%26aulast%3DKing%26aufirst%3DD.%2BS.%26aulast%3DFields%26aufirst%3DC.%2BG.%26aulast%3DFields%26aufirst%3DG.%2BB.%26atitle%3DA%2520Cleavage%2520Method%2520Which%2520Minimizes%2520Side%2520Reactions%2520Following%2520Fmoc%2520Solid%2520Phase%2520Peptide%2520Synthesis%26jtitle%3DInt.%2520J.%2520Pept.%2520Protein%2520Res.%26date%3D1990%26volume%3D36%26spage%3D255%26epage%3D266%26doi%3D10.1111%2Fj.1399-3011.1990.tb00976.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref143"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref143'); return false;" data-citation="" class="refNumLink">143</a></strong><div class="NLM_citation" id="cit143"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cummings, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sekharan, S.</span></span> <span> </span><span class="NLM_article-title">Structure-Based Macrocycle Design in Small-Molecule Drug Discovery and Simple Metrics To Identify Opportunities for Macrocyclization of Small-Molecule Ligands</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">6843</span>– <span class="NLM_lpage">6853</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01985</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01985" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref143/cit143&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC1MXks1yktb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=6843-6853&author=M.+D.+Cummingsauthor=S.+Sekharan&title=Structure-Based+Macrocycle+Design+in+Small-Molecule+Drug+Discovery+and+Simple+Metrics+To+Identify+Opportunities+for+Macrocyclization+of+Small-Molecule+Ligands&doi=10.1021%2Facs.jmedchem.8b01985"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit143R"><div class="casContent"><span class="casTitleNuber">143</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Macrocycle Design in Small-Molecule Drug Discovery and Simple Metrics To Identify Opportunities for Macrocyclization of Small-Molecule Ligands</span></div><div class="casAuthors">Cummings, Maxwell D.; Sekharan, Sivakumar</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">6843-6853</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Interest is growing in the use of macrocycles in pharmaceutical discovery.  Macrocylization may provide a gateway to an expanded chem. space for small-mol. drug discovery, and this could be beneficial in prosecuting difficult targets, e.g., protein-protein interactions.  Most, but not all, macrocycle drugs are derived from natural products.  Studies on synthetic drug-like small-mol. macrocycles are limited, and our current understanding of macrocycle drugs is similarly limited.  Following some background discussion, we review several examples of the structure-based design of synthetic macrocycles.  Our opinion is that in conformationally suitable systems macrocycles are an analog class worthy of consideration.  We then summarize an approach for the initial evaluation of mols. as candidates for macrocyclization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCAQg11Q6Hk7Vg90H21EOLACvtfcHk0ljC364cqNnqDw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXks1yktb4%253D&md5=32ac78d837075171d19c4cea694dce3f</span></div><a href="/servlet/linkout?suffix=cit143&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01985&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01985%26sid%3Dliteratum%253Aachs%26aulast%3DCummings%26aufirst%3DM.%2BD.%26aulast%3DSekharan%26aufirst%3DS.%26atitle%3DStructure-Based%2520Macrocycle%2520Design%2520in%2520Small-Molecule%2520Drug%2520Discovery%2520and%2520Simple%2520Metrics%2520To%2520Identify%2520Opportunities%2520for%2520Macrocyclization%2520of%2520Small-Molecule%2520Ligands%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D6843%26epage%3D6853%26doi%3D10.1021%2Facs.jmedchem.8b01985" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref144"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref144'); return false;" data-citation="" class="refNumLink">144</a></strong><div class="NLM_citation" id="cit144"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jorgensen, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maxwell, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tirado-Rives, J.</span></span> <span> </span><span class="NLM_article-title">Development and Testing of the OPLS All-Atom Force Field on Conformational Energetics and Properties of Organic Liquids</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>118</i></span>,  <span class="NLM_fpage">11225</span>– <span class="NLM_lpage">11236</span>, <span class="refDoi"> DOI: 10.1021/ja9621760</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja9621760" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref144/cit144&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADyaK28XmtlOitrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=118&publication_year=1996&pages=11225-11236&author=W.+L.+Jorgensenauthor=D.+S.+Maxwellauthor=J.+Tirado-Rives&title=Development+and+Testing+of+the+OPLS+All-Atom+Force+Field+on+Conformational+Energetics+and+Properties+of+Organic+Liquids&doi=10.1021%2Fja9621760"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit144R"><div class="casContent"><span class="casTitleNuber">144</span><div class="casTitle"><span class="NLM_cas:atitle">Development and Testing of the OPLS All-Atom Force Field on Conformational Energetics and Properties of Organic Liquids</span></div><div class="casAuthors">Jorgensen, William L.; Maxwell, David S.; Tirado-Rives, Julian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">118</span>
        (<span class="NLM_cas:issue">45</span>),
    <span class="NLM_cas:pages">11225-11236</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The parametrization and testing of the OPLS all-atom force field for org. mols. and peptides are described.  Parameters for both torsional and nonbonded energetics have been derived, while the bond stretching and angle bending parameters have been adopted mostly from the AMBER all-atom force field.  The torsional parameters were detd. by fitting to rotational energy profiles obtained from ab initio MO calcns. at the RHF/6-31G*//RHF/6-31G* level for more than 50 org. mols. and ions.  The quality of the fits was high with av. errors for conformational energies of less than 0.2 kcal/mol.  The force-field results for mol. structures are also demonstrated to closely match the ab initio predictions.  The nonbonded parameters were developed in conjunction with Monte Carlo statistical mechanics simulations by computing thermodn. and structural properties for 34 pure org. liqs. including alkanes, alkenes, alcs., ethers, acetals, thiols, sulfides, disulfides, aldehydes, ketones, and amides.  Av. errors in comparison with exptl. data are 2% for heats of vaporization and densities.  The Monte Carlo simulations included sampling all internal and intermol. degrees of freedom.  It is found that such non-polar and monofunctional systems do not show significant condensed-phase effects on internal energies in going from the gas phase to the pure liqs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhGotSev_b1LVg90H21EOLACvtfcHk0ljRE2TSp6fFuA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XmtlOitrs%253D&md5=fef2924a69421881390282aa309ae91b</span></div><a href="/servlet/linkout?suffix=cit144&amp;dbid=16384&amp;doi=10.1021%2Fja9621760&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja9621760%26sid%3Dliteratum%253Aachs%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26aulast%3DMaxwell%26aufirst%3DD.%2BS.%26aulast%3DTirado-Rives%26aufirst%3DJ.%26atitle%3DDevelopment%2520and%2520Testing%2520of%2520the%2520OPLS%2520All-Atom%2520Force%2520Field%2520on%2520Conformational%2520Energetics%2520and%2520Properties%2520of%2520Organic%2520Liquids%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1996%26volume%3D118%26spage%3D11225%26epage%3D11236%26doi%3D10.1021%2Fja9621760" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref145"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref145'); return false;" data-citation="" class="refNumLink">145</a></strong><div class="NLM_citation" id="cit145"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jorgensen, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tirado-Rives, J.</span></span> <span> </span><span class="NLM_article-title">The OPLS [Optimized Potentials for Liquid Simulations] Potential Functions for Proteins, Energy Minimizations for Crystals of Cyclic Peptides and Crambin</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">1988</span>,  <span class="NLM_volume"><i>110</i></span>,  <span class="NLM_fpage">1657</span>– <span class="NLM_lpage">1666</span>, <span class="refDoi"> DOI: 10.1021/ja00214a001</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja00214a001" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref145/cit145&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADyaL1cXht1yjt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=1988&pages=1657-1666&author=W.+L.+Jorgensenauthor=J.+Tirado-Rives&title=The+OPLS+%5BOptimized+Potentials+for+Liquid+Simulations%5D+Potential+Functions+for+Proteins%2C+Energy+Minimizations+for+Crystals+of+Cyclic+Peptides+and+Crambin&doi=10.1021%2Fja00214a001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit145R"><div class="casContent"><span class="casTitleNuber">145</span><div class="casTitle"><span class="NLM_cas:atitle">The OPLS [optimized potentials for liquid simulations] potential functions for proteins, energy minimizations for crystals of cyclic peptides and crambin</span></div><div class="casAuthors">Jorgensen, William L.; Tirado-Rives, Julian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1988</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1657-66</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    </div><div class="casAbstract">A complete set of intermol. potential functions was developed for use in computer simulations of proteins in their native environment.  Parameters are reported for 25 peptide residues as well as the common neutral and charged terminal groups.  The potential functions have the simple Coulomb plus Lennard-Jones form and are compatible with the widely used models for water, TIP4P, TIP3P, and SPC.  The parameters were obtained and tested primarily in conjunction with Monte Carlo statistical mechanics simulations of 36 pure org. liqs. and numerous aq. solns. of org. ions representative of subunits in the side chains and backbones of proteins.  Bond stretch, angle bend, and torsional terms were adopted from the AMBER united-atom force field.  As reported here, further testing involved studies of conformational energy surfaces and optimizations of the crystal structures for 4 cyclic hexapeptides and a cyclic pentapeptide.  The av. root mean square deviation from the x-ray structures of the crystals is only 0.17 Å for the at. positions and 3% for the unit cell vols.  A more crit. test was then provided by performing energy minimizations for the complete crystal of the protein crambin, including 182 water mols. that were initially placed via a Monte Carlo simulation.  The resultant root mean square deviation for the non-H atoms is still only 0.17 Å and the variation in the errors for charged, polar, and nonpolar residues is small.  Significant improvement is apparent over the AMBER united-atom force field which was previously demonstrated to be superior to many alternatives.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZ6an38QWApbVg90H21EOLACvtfcHk0ljRE2TSp6fFuA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1cXht1yjt7Y%253D&md5=b6c901d8c295b3b37329a7faef527e12</span></div><a href="/servlet/linkout?suffix=cit145&amp;dbid=16384&amp;doi=10.1021%2Fja00214a001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja00214a001%26sid%3Dliteratum%253Aachs%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26aulast%3DTirado-Rives%26aufirst%3DJ.%26atitle%3DThe%2520OPLS%2520%255BOptimized%2520Potentials%2520for%2520Liquid%2520Simulations%255D%2520Potential%2520Functions%2520for%2520Proteins%252C%2520Energy%2520Minimizations%2520for%2520Crystals%2520of%2520Cyclic%2520Peptides%2520and%2520Crambin%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1988%26volume%3D110%26spage%3D1657%26epage%3D1666%26doi%3D10.1021%2Fja00214a001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref146"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref146'); return false;" data-citation="" class="refNumLink">146</a></strong><div class="NLM_citation" id="cit146"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shivakumar, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harder, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Damm, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sherman, W.</span></span> <span> </span><span class="NLM_article-title">Improving the Prediction of Absolute Solvation Free Energies Using the Next Generation OPLS Force Field</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">2553</span>– <span class="NLM_lpage">2558</span>, <span class="refDoi"> DOI: 10.1021/ct300203w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct300203w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref146/cit146&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC38XptFWjs7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&pages=2553-2558&author=D.+Shivakumarauthor=E.+Harderauthor=W.+Dammauthor=R.+A.+Friesnerauthor=W.+Sherman&title=Improving+the+Prediction+of+Absolute+Solvation+Free+Energies+Using+the+Next+Generation+OPLS+Force+Field&doi=10.1021%2Fct300203w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit146R"><div class="casContent"><span class="casTitleNuber">146</span><div class="casTitle"><span class="NLM_cas:atitle">Improving the Prediction of Absolute Solvation Free Energies Using the Next Generation OPLS Force Field</span></div><div class="casAuthors">Shivakumar, Devleena; Harder, Edward; Damm, Wolfgang; Friesner, Richard A.; Sherman, Woody</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2553-2558</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Explicit solvent mol. dynamics free energy perturbation simulations were performed to predict abs. solvation free energies of 239 diverse small mols.  We use OPLS2.0, the next generation OPLS force field, and compare the results with popular small mol. force fields-OPLS_2005, GAFF, and CHARMm-MSI.  OPLS2.0 produces the best correlation with exptl. data (R2 = 0.95, slope = 0.96) and the lowest av. unsigned errors (0.7 kcal/mol).  Important classes of compds. that performed suboptimally with OPLS_2005 show significant improvements.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqEngE5xV9Ev7Vg90H21EOLACvtfcHk0ljRE2TSp6fFuA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XptFWjs7g%253D&md5=eceb7dab3a8d0c6ac586c620848412d1</span></div><a href="/servlet/linkout?suffix=cit146&amp;dbid=16384&amp;doi=10.1021%2Fct300203w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct300203w%26sid%3Dliteratum%253Aachs%26aulast%3DShivakumar%26aufirst%3DD.%26aulast%3DHarder%26aufirst%3DE.%26aulast%3DDamm%26aufirst%3DW.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DSherman%26aufirst%3DW.%26atitle%3DImproving%2520the%2520Prediction%2520of%2520Absolute%2520Solvation%2520Free%2520Energies%2520Using%2520the%2520Next%2520Generation%2520OPLS%2520Force%2520Field%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2012%26volume%3D8%26spage%3D2553%26epage%3D2558%26doi%3D10.1021%2Fct300203w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref147"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref147'); return false;" data-citation="" class="refNumLink">147</a></strong><div class="NLM_citation" id="cit147"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hoover, W. G.</span></span> <span> </span><span class="NLM_article-title">Canonical Dynamics: Equilibrium Phase-Space Distributions</span>. <i>Phys. Rev. A</i> <span class="NLM_year" style="font-weight: bold;">1985</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">1695</span>– <span class="NLM_lpage">1697</span>, <span class="refDoi"> DOI: 10.1103/PhysRevA.31.1695</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref147/cit147&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1103%2FPhysRevA.31.1695" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref147/cit147&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=9895674" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref147/cit147&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A280%3ADC%252BC2sjotlWltA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=1985&pages=1695-1697&author=W.+G.+Hoover&title=Canonical+Dynamics%3A+Equilibrium+Phase-Space+Distributions&doi=10.1103%2FPhysRevA.31.1695"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit147R"><div class="casContent"><span class="casTitleNuber">147</span><div class="casTitle"><span class="NLM_cas:atitle">Canonical dynamics: Equilibrium phase-space distributions</span></div><div class="casAuthors">Hoover</div><div class="citationInfo"><span class="NLM_cas:title">Physical review. A, General physics</span>
        (<span class="NLM_cas:date">1985</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1695-1697</span>
        ISSN:<span class="NLM_cas:issn">0556-2791</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSFX8fOxXRWv36CEoQgWY10fW6udTcc2eZzD_vj5dxRcbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2sjotlWltA%253D%253D&md5=99a2477835b37592226a5d18a760685c</span></div><a href="/servlet/linkout?suffix=cit147&amp;dbid=16384&amp;doi=10.1103%2FPhysRevA.31.1695&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1103%252FPhysRevA.31.1695%26sid%3Dliteratum%253Aachs%26aulast%3DHoover%26aufirst%3DW.%2BG.%26atitle%3DCanonical%2520Dynamics%253A%2520Equilibrium%2520Phase-Space%2520Distributions%26jtitle%3DPhys.%2520Rev.%2520A%26date%3D1985%26volume%3D31%26spage%3D1695%26epage%3D1697%26doi%3D10.1103%2FPhysRevA.31.1695" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref148"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref148'); return false;" data-citation="" class="refNumLink">148</a></strong><div class="NLM_citation" id="cit148"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nosé, S.</span></span> <span> </span><span class="NLM_article-title">A Unified Formulation of the Constant Temperature Molecular Dynamics Methods</span>. <i>J. Chem. Phys.</i> <span class="NLM_year" style="font-weight: bold;">1984</span>,  <span class="NLM_volume"><i>81</i></span>,  <span class="NLM_fpage">511</span>– <span class="NLM_lpage">519</span>, <span class="refDoi"> DOI: 10.1063/1.447334</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref148/cit148&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1063%2F1.447334" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref148/cit148&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADyaL2cXkvFOrs7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=81&publication_year=1984&pages=511-519&author=S.+Nos%C3%A9&title=A+Unified+Formulation+of+the+Constant+Temperature+Molecular+Dynamics+Methods&doi=10.1063%2F1.447334"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit148R"><div class="casContent"><span class="casTitleNuber">148</span><div class="casTitle"><span class="NLM_cas:atitle">A unified formulation of the constant-temperature molecular-dynamics methods</span></div><div class="casAuthors">Nose, Shuichi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1984</span>),
    <span class="NLM_cas:volume">81</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">511-19</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    </div><div class="casAbstract">Three recently proposed const. temp. mol. dynamics methods [N., (1984) (1); W. G. Hoover et al., (1982) (2); D. J. Evans and G. P. Morris, (1983) (2); and J. M. Haile and S. Gupta, 1983) (3)] are examd. anal. via calcg. the equil. distribution functions and comparing them with that of the canonical ensemble.  Except for effects due to momentum and angular momentum conservation, method (1) yields the rigorous canonical distribution in both momentum and coordinate space.  Method (2) can be made rigorous in coordinate space, and can be derived from method (1) by imposing a specific constraint.  Method (3) is not rigorous and gives a deviation of order N-1/2 from the canonical distribution (N the no. of particles).  The results for the const. temp.-const. pressure ensemble are similar to the canonical ensemble case.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrsbAHxmqLp77Vg90H21EOLACvtfcHk0lgMD0bXkzOLqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL2cXkvFOrs7k%253D&md5=2974515ec89e5601868e35871c0f19c2</span></div><a href="/servlet/linkout?suffix=cit148&amp;dbid=16384&amp;doi=10.1063%2F1.447334&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.447334%26sid%3Dliteratum%253Aachs%26aulast%3DNos%25C3%25A9%26aufirst%3DS.%26atitle%3DA%2520Unified%2520Formulation%2520of%2520the%2520Constant%2520Temperature%2520Molecular%2520Dynamics%2520Methods%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1984%26volume%3D81%26spage%3D511%26epage%3D519%26doi%3D10.1063%2F1.447334" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref149"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref149'); return false;" data-citation="" class="refNumLink">149</a></strong><div class="NLM_citation" id="cit149"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Martyna, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tobias, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, M. L.</span></span> <span> </span><span class="NLM_article-title">Constant Pressure Molecular Dynamics Algorithms</span>. <i>J. Chem. Phys.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>101</i></span>,  <span class="NLM_fpage">4177</span>– <span class="NLM_lpage">4189</span>, <span class="refDoi"> DOI: 10.1063/1.467468</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref149/cit149&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1063%2F1.467468" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref149/cit149&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADyaK2cXmtFeht7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=101&publication_year=1994&pages=4177-4189&author=G.+J.+Martynaauthor=D.+J.+Tobiasauthor=M.+L.+Klein&title=Constant+Pressure+Molecular+Dynamics+Algorithms&doi=10.1063%2F1.467468"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit149R"><div class="casContent"><span class="casTitleNuber">149</span><div class="casTitle"><span class="NLM_cas:atitle">Constant pressure molecular dynamics algorithms</span></div><div class="casAuthors">Martyna, Glenn J.; Tobias, Douglas J.; Klein, Michael L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">101</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">4177-89</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    </div><div class="casAbstract">Modularly invariant equations of motion are derived that generate the isothermal-isobaric ensemble as their phase space avs.  Isotropic vol. fluctuations and fully flexible simulation cells as well as a hybrid scheme that naturally combines the two motions are considered.  The resulting methods are tested on two problems, a particle in a one-dimensional periodic potential and a spherical model of C60 in the solid/fluid phase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3YN7wMRJpjrVg90H21EOLACvtfcHk0lgMD0bXkzOLqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXmtFeht7o%253D&md5=c14bd79c6398b0b30541e3cbe92851b0</span></div><a href="/servlet/linkout?suffix=cit149&amp;dbid=16384&amp;doi=10.1063%2F1.467468&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.467468%26sid%3Dliteratum%253Aachs%26aulast%3DMartyna%26aufirst%3DG.%2BJ.%26aulast%3DTobias%26aufirst%3DD.%2BJ.%26aulast%3DKlein%26aufirst%3DM.%2BL.%26atitle%3DConstant%2520Pressure%2520Molecular%2520Dynamics%2520Algorithms%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1994%26volume%3D101%26spage%3D4177%26epage%3D4189%26doi%3D10.1063%2F1.467468" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref150"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref150'); return false;" data-citation="" class="refNumLink">150</a></strong><div class="NLM_citation" id="cit150"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Darden, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">York, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pedersen, L.</span></span> <span> </span><span class="NLM_article-title">Particle Mesh Ewald: An N·log(N) Method for Ewald Sums in Large Systems</span>. <i>J. Chem. Phys.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>98</i></span>,  <span class="NLM_fpage">10089</span>– <span class="NLM_lpage">10092</span>, <span class="refDoi"> DOI: 10.1063/1.464397</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref150/cit150&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1063%2F1.464397" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref150/cit150&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADyaK3sXks1Ohsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=1993&pages=10089-10092&author=T.+Dardenauthor=D.+Yorkauthor=L.+Pedersen&title=Particle+Mesh+Ewald%3A+An+N%C2%B7log%28N%29+Method+for+Ewald+Sums+in+Large+Systems&doi=10.1063%2F1.464397"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit150R"><div class="casContent"><span class="casTitleNuber">150</span><div class="casTitle"><span class="NLM_cas:atitle">Particle mesh Ewald: an N·log(N) method for Ewald sums in large systems</span></div><div class="casAuthors">Darden, Tom; York, Darrin; Pedersen, Lee</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">10089-92</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    </div><div class="casAbstract">An N·log(N) method for evaluating electrostatic energies and forces of large periodic systems is presented.  The method is based on interpolation of the reciprocal space Ewald sums and evaluation of the resulting convolution using fast Fourier transforms.  Timings and accuracies are presented for three large cryst. ionic systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDIJavf7AZPbVg90H21EOLACvtfcHk0lgMD0bXkzOLqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXks1Ohsr0%253D&md5=3c9f230bd01b7b714fd096d4d2e755f6</span></div><a href="/servlet/linkout?suffix=cit150&amp;dbid=16384&amp;doi=10.1063%2F1.464397&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.464397%26sid%3Dliteratum%253Aachs%26aulast%3DDarden%26aufirst%3DT.%26aulast%3DYork%26aufirst%3DD.%26aulast%3DPedersen%26aufirst%3DL.%26atitle%3DParticle%2520Mesh%2520Ewald%253A%2520An%2520N%25C2%25B7log%2528N%2529%2520Method%2520for%2520Ewald%2520Sums%2520in%2520Large%2520Systems%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1993%26volume%3D98%26spage%3D10089%26epage%3D10092%26doi%3D10.1063%2F1.464397" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref151"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref151'); return false;" data-citation="" class="refNumLink">151</a></strong><div class="NLM_citation" id="cit151"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ryckaert, J.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciccotti, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berendsen, H. J. C.</span></span> <span> </span><span class="NLM_article-title">Numerical Integration of the Cartesian Equations of Motion of a System with Constraints: Molecular Dynamics of n-Alkanes</span>. <i>J. Comput. Phys.</i> <span class="NLM_year" style="font-weight: bold;">1977</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">327</span>– <span class="NLM_lpage">341</span>, <span class="refDoi"> DOI: 10.1016/0021-9991(77)90098-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref151/cit151&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1016%2F0021-9991%2877%2990098-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref151/cit151&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADyaE2sXktVGhsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=1977&pages=327-341&author=J.-P.+Ryckaertauthor=G.+Ciccottiauthor=H.+J.+C.+Berendsen&title=Numerical+Integration+of+the+Cartesian+Equations+of+Motion+of+a+System+with+Constraints%3A+Molecular+Dynamics+of+n-Alkanes&doi=10.1016%2F0021-9991%2877%2990098-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit151R"><div class="casContent"><span class="casTitleNuber">151</span><div class="casTitle"><span class="NLM_cas:atitle">Numerical integration of the Cartesian equations of motion of a system with constraints:  molecular dynamics of n-alkanes</span></div><div class="casAuthors">Ryckaert, Jean Paul; Ciccotti, Giovanni; Berendsen, Herman J. C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Physics</span>
        (<span class="NLM_cas:date">1977</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">327-41</span>CODEN:
                <span class="NLM_cas:coden">JCTPAH</span>;
        ISSN:<span class="NLM_cas:issn">0021-9991</span>.
    </div><div class="casAbstract">A numerical algorithm integrating the 3N Cartesian equation of motion of a system of N points subject to holonomic constraints is applied to mol. dynamics simulation of a liq. of 64 butane mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYPlhG1U3Uw7Vg90H21EOLACvtfcHk0lhwBixZCB5D1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE2sXktVGhsL4%253D&md5=b4aecddfde149117813a5ea4f5353ce2</span></div><a href="/servlet/linkout?suffix=cit151&amp;dbid=16384&amp;doi=10.1016%2F0021-9991%2877%2990098-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0021-9991%252877%252990098-5%26sid%3Dliteratum%253Aachs%26aulast%3DRyckaert%26aufirst%3DJ.-P.%26aulast%3DCiccotti%26aufirst%3DG.%26aulast%3DBerendsen%26aufirst%3DH.%2BJ.%2BC.%26atitle%3DNumerical%2520Integration%2520of%2520the%2520Cartesian%2520Equations%2520of%2520Motion%2520of%2520a%2520System%2520with%2520Constraints%253A%2520Molecular%2520Dynamics%2520of%2520n-Alkanes%26jtitle%3DJ.%2520Comput.%2520Phys.%26date%3D1977%26volume%3D23%26spage%3D327%26epage%3D341%26doi%3D10.1016%2F0021-9991%2877%2990098-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63'],'ref64':['cit64'],'ref65':['cit65'],'ref66':['cit66'],'ref67':['cit67'],'ref68':['cit68'],'ref69':['cit69'],'ref70':['cit70'],'ref71':['cit71'],'ref72':['cit72'],'ref73':['cit73'],'ref74':['cit74'],'ref75':['cit75'],'ref76':['cit76'],'ref77':['cit77'],'ref78':['cit78'],'ref79':['cit79'],'ref80':['cit80'],'ref81':['cit81'],'ref82':['cit82'],'ref83':['cit83'],'ref84':['cit84'],'ref85':['cit85'],'ref86':['cit86'],'ref87':['cit87'],'ref88':['cit88'],'ref89':['cit89'],'ref90':['cit90'],'ref91':['cit91'],'ref92':['cit92'],'ref93':['cit93'],'ref94':['cit94'],'ref95':['cit95'],'ref96':['cit96'],'ref97':['cit97'],'ref98':['cit98'],'ref99':['cit99'],'ref100':['cit100'],'ref101':['cit101'],'ref102':['cit102'],'ref103':['cit103'],'ref104':['cit104'],'ref105':['cit105'],'ref106':['cit106'],'ref107':['cit107'],'ref108':['cit108'],'ref109':['cit109'],'ref110':['cit110'],'ref111':['cit111'],'ref112':['cit112'],'ref113':['cit113'],'ref114':['cit114'],'ref115':['cit115'],'ref116':['cit116'],'ref117':['cit117'],'ref118':['cit118'],'ref119':['cit119'],'ref120':['cit120'],'ref121':['cit121'],'ref122':['cit122'],'ref123':['cit123'],'ref124':['cit124'],'ref125':['cit125'],'ref126':['cit126'],'ref127':['cit127'],'ref128':['cit128'],'ref129':['cit129'],'ref130':['cit130'],'ref131':['cit131'],'ref132':['cit132'],'ref133':['cit133'],'ref134':['cit134'],'ref135':['cit135'],'ref136':['cit136'],'ref137':['cit137'],'ref138':['cit138'],'ref139':['cit139'],'ref140':['cit140'],'ref141':['cit141'],'ref142':['cit142'],'ref143':['cit143'],'ref144':['cit144'],'ref145':['cit145'],'ref146':['cit146'],'ref147':['cit147'],'ref148':['cit148'],'ref149':['cit149'],'ref150':['cit150'],'ref151':['cit151']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 2 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jeffrey Y.-K.  Wong</span>, <span class="hlFld-ContribAuthor ">Raja  Mukherjee</span>, <span class="hlFld-ContribAuthor ">Jiayuan  Miao</span>, <span class="hlFld-ContribAuthor ">Olena  Bilyk</span>, <span class="hlFld-ContribAuthor ">Vivian  Triana</span>, <span class="hlFld-ContribAuthor ">Mark  Miskolzie</span>, <span class="hlFld-ContribAuthor ">Antoine  Henninot</span>, <span class="hlFld-ContribAuthor ">John J.  Dwyer</span>, <span class="hlFld-ContribAuthor ">Serhii  Kharchenko</span>, <span class="hlFld-ContribAuthor ">Anna  Iampolska</span>, <span class="hlFld-ContribAuthor ">Dmitriy M.  Volochnyuk</span>, <span class="hlFld-ContribAuthor ">Yu-Shan  Lin</span>, <span class="hlFld-ContribAuthor ">Lynne-Marie  Postovit</span>, <span class="hlFld-ContribAuthor ">Ratmir  Derda</span>. </span><span class="cited-content_cbyCitation_article-title">Genetically-encoded discovery of proteolytically stable bicyclic inhibitors for morphogen NODAL. </span><span class="cited-content_cbyCitation_journal-name">Chemical Science</span><span> <strong>2021,</strong> <em>12 </em>
                                    (28)
                                     , 9694-9703. <a href="https://doi.org/10.1039/D1SC01916C" title="DOI URL">https://doi.org/10.1039/D1SC01916C</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D1SC01916C&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD1SC01916C%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Science%26atitle%3DGenetically-encoded%252Bdiscovery%252Bof%252Bproteolytically%252Bstable%252Bbicyclic%252Binhibitors%252Bfor%252Bmorphogen%252BNODAL%26aulast%3DWong%26aufirst%3DJeffrey%2BY.-K.%26date%3D2021%26date%3D2021%26volume%3D12%26issue%3D28%26spage%3D9694%26epage%3D9703" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Huadong  Hong</span>, <span class="hlFld-ContribAuthor ">Qianli  Zou</span>, <span class="hlFld-ContribAuthor ">Yamei  Liu</span>, <span class="hlFld-ContribAuthor ">Shaozhen  Wang</span>, <span class="hlFld-ContribAuthor ">Guizhi  Shen</span>, <span class="hlFld-ContribAuthor ">Xuehai  Yan</span>. </span><span class="cited-content_cbyCitation_article-title">Supramolecular Nanodrugs Based on Covalent Assembly of Therapeutic Peptides toward
              In Vitro
              Synergistic Anticancer Therapy. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2021,</strong> <em>33 </em><a href="https://doi.org/10.1002/cmdc.202100236" title="DOI URL">https://doi.org/10.1002/cmdc.202100236</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.202100236&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.202100236%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DSupramolecular%252BNanodrugs%252BBased%252Bon%252BCovalent%252BAssembly%252Bof%252BTherapeutic%252BPeptides%252Btoward%252BIn%252BVitro%252BSynergistic%252BAnticancer%252BTherapy%26aulast%3DHong%26aufirst%3DHuadong%26date%3D2021%26date%3D2021%26volume%3D33" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01500/20201019/images/medium/jm0c01500_0014.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01500/20201019/images/large/jm0c01500_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01500&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01500/20201019/images/medium/jm0c01500_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01500/20201019/images/large/jm0c01500_0002.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Procedure for the automated design of macrocycles. (Step 1) The user prepares a linear starting molecule. The attachment points may be either identified automatically or specified by the user. (Step 2) Generation of chemical linkers. These might either be provided as explicit lists, e.g., based on available chemical reagents, or can be grown automatically by combining and enumerating small breeding fragments. (Step 3) A full conformational ensemble of each linker is generated and (step 4) mapped with all attachment points to eliminate unfavorable cyclizations applying different geometric filter criteria. (Step 5) Finally, the cyclized ligands are ranked applying conformational and/or enthalpic scoring.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01500/20201019/images/large/jm0c01500_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01500&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01500/20201019/images/medium/jm0c01500_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01500/20201019/images/large/jm0c01500_0003.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Approach for modeling bicyclic peptide structures using a trisymmetrical linker. In step 1, all geometrically reasonable monocyclic structures are generated. In the second cyclization routine (step 2), the unlinked (third) attachment point of the linker is evaluated for attachment to the peptide backbone for construction of the second cycle.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01500/20201019/images/large/jm0c01500_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01500&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01500/20201019/images/medium/jm0c01500_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01500/20201019/images/large/jm0c01500_0004.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Cyclization of dual agonists of the GLP-1 and glucagon receptor. (a) Predicted binding mode of the dual agonist <b>4a</b> at the GLP-1R (pdb code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5VAI">5VAI</a>). (b) Automatically generated cyclization proposals with all chemical linkers shown in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>. (c) Cyclization proposals using a focused list of reagents and attachment positions. (d) Predicted binding modes of synthesized peptides obtained from MM/GBSA energy minimization.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01500/20201019/images/large/jm0c01500_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01500&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01500/20201019/images/medium/jm0c01500_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01500/20201019/images/large/jm0c01500_0005.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Macrocylization of a TAFIa inhibitor. (a) X-ray structure of anabaenopeptin C in complex with the surrogate protease carboxypeptidase B (pdb code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LRJ">5LRJ</a>). (b) Anabaenopepin C was modified to obtain the starting structure for rescaffolding of the macrocycle. The attachment vectors are indicated in magenta. (c) Breeding fragments that were combined and enumerated for cyclization. Predicted binding modes of (d) automatically generated cyclization proposals and (e) four cyclization proposals that were synthesized and experimentally tested.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01500/20201019/images/large/jm0c01500_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01500&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01500/20201019/images/medium/jm0c01500_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01500/20201019/images/large/jm0c01500_0006.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. (a) X-ray structure of the linear PROTAC MZ1 (<b>8a</b>) in complex with Brd4BD2 and VHL-EloB-EloC (pdb code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5T35">5T35</a>). (b) The PROTAC structure was modified to obtain the starting structure for cyclization. The attachment vectors are indicated in magenta. (c) Cyclized conformations comprising PEG units of different length, minimized in the protein environment. (d) Comparison of the predicted binding mode of <b>8b</b> with the experimental binding mode (pdb <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6SIS">6SIS</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01500/20201019/images/large/jm0c01500_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01500&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01500/20201019/images/medium/jm0c01500_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01500/20201019/images/large/jm0c01500_0007.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Generation of bicyclic variants of KIX domain from the human CREB binding protein. (a) NMR structure of KIX (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2AGH">2AGH</a>). The junction between the helical bundle (α1, α2, α3) and the C-terminal 310 helix (G1) that was subjected to covalent stabilization is indicated. (b) Cyclized construct that links residues 594, 599, and 646. (c, d) Three-dimensional alignment of snapshots from 100 ns simulations of the linear and cyclized proteins.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01500/20201019/images/large/jm0c01500_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01500&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01500/20201019/images/medium/jm0c01500_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01500/20201019/images/large/jm0c01500_0008.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Workflow for (a) peptides and (b) small molecules.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c01500/20201019/images/large/jm0c01500_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01500&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i41">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_60461" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_60461" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 151 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dougherty, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pei, D.</span></span> <span> </span><span class="NLM_article-title">Macrocycles as Protein-Protein Interaction Inhibitors</span>. <i>Biochem. J.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>474</i></span>,  <span class="NLM_fpage">1109</span>– <span class="NLM_lpage">1125</span>, <span class="refDoi"> DOI: 10.1042/BCJ20160619</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1042%2FBCJ20160619" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=28298556" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkvVygt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=474&publication_year=2017&pages=1109-1125&author=P.+G.+Doughertyauthor=Z.+Qianauthor=D.+Pei&title=Macrocycles+as+Protein-Protein+Interaction+Inhibitors&doi=10.1042%2FBCJ20160619"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Macrocycles as protein-protein interaction inhibitors</span></div><div class="casAuthors">Dougherty, Patrick G.; Qian, Ziqing; Pei, Dehua</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Journal</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">474</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1109-1125</span>CODEN:
                <span class="NLM_cas:coden">BIJOAK</span>;
        ISSN:<span class="NLM_cas:issn">0264-6021</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">Macrocyclic compds. such as cyclic peptides have emerged as a new and exciting class of drug candidates for inhibition of intracellular protein-protein interactions, which are challenging targets for conventional drug modalities (i.e. small mols. and proteins).  Over the past decade, several complementary technologies have been developed to synthesize macrocycle libraries and screen them for binding to therapeutically relevant targets.  Two different approaches have also been explored to increase the membrane permeability of cyclic peptides.  In this review, we discuss these methods and their applications in the discovery of macrocyclic compds. against protein-protein interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpyxFYrVReYebVg90H21EOLACvtfcHk0liTpEU9FnaQWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkvVygt7g%253D&md5=abd5d7af8383fbe5b553d9501c27ce9c</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1042%2FBCJ20160619&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBCJ20160619%26sid%3Dliteratum%253Aachs%26aulast%3DDougherty%26aufirst%3DP.%2BG.%26aulast%3DQian%26aufirst%3DZ.%26aulast%3DPei%26aufirst%3DD.%26atitle%3DMacrocycles%2520as%2520Protein-Protein%2520Interaction%2520Inhibitors%26jtitle%3DBiochem.%2520J.%26date%3D2017%26volume%3D474%26spage%3D1109%26epage%3D1125%26doi%3D10.1042%2FBCJ20160619" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yudin, A. K.</span></span> <span> </span><span class="NLM_article-title">Macrocycles: Lessons from the Distant Past, Recent Developments, and Future Directions</span>. <i>Chem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">30</span>– <span class="NLM_lpage">49</span>, <span class="refDoi"> DOI: 10.1039/C4SC03089C</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1039%2FC4SC03089C" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=28553456" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVGhsLbE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=30-49&author=A.+K.+Yudin&title=Macrocycles%3A+Lessons+from+the+Distant+Past%2C+Recent+Developments%2C+and+Future+Directions&doi=10.1039%2FC4SC03089C"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Macrocycles: lessons from the distant past, recent developments, and future directions</span></div><div class="casAuthors">Yudin, Andrei K.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Science</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">30-49</span>CODEN:
                <span class="NLM_cas:coden">CSHCCN</span>;
        ISSN:<span class="NLM_cas:issn">2041-6520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  A noticeable increase in mol. complexity of drug targets has created an unmet need in the therapeutic agents that are larger than traditional small mols.  Macrocycles, which are cyclic compds. comprising 12 atoms or more, are now recognized as mols. that "are up to the task" to interrogate extended protein interfaces.  However, because macrocycles (particularly the ones based on peptides) are equipped with large polar surface areas, achieving cellular permeability and bioavailability is anything but straightforward.  While one might consider this to be the Achilles' heel of this class of compds., the synthetic community continues to develop creative approaches toward the synthesis of macrocycles and their site-selective modification.  This perspective provides an overview of both mechanistic and structural issues that bear on macrocycles as a unique class of mols.  The reader is offered a historical foray into some of the classic studies that have resulted in the current renaissance of macrocycles.  In addn., an attempt is made to overview the more recent developments that give hope that macrocycles might indeed turn into a useful therapeutic modality.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrCRWplVAQ417Vg90H21EOLACvtfcHk0liTpEU9FnaQWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVGhsLbE&md5=48262b4650417f04063c0180b652a886</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1039%2FC4SC03089C&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC4SC03089C%26sid%3Dliteratum%253Aachs%26aulast%3DYudin%26aufirst%3DA.%2BK.%26atitle%3DMacrocycles%253A%2520Lessons%2520from%2520the%2520Distant%2520Past%252C%2520Recent%2520Developments%252C%2520and%2520Future%2520Directions%26jtitle%3DChem.%2520Sci.%26date%3D2015%26volume%3D6%26spage%3D30%26epage%3D49%26doi%3D10.1039%2FC4SC03089C" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zaretsky, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yudin, A. K.</span></span> <span> </span><span class="NLM_article-title">Recent Advances in the Synthesis of Cyclic Pseudopeptides</span>. <i>Drug Discovery Today Technol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">3</span>– <span class="NLM_lpage">10</span>, <span class="refDoi"> DOI: 10.1016/j.ddtec.2017.11.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1016%2Fj.ddtec.2017.11.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=29249240" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A280%3ADC%252BC1MzitlCjsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2017&pages=3-10&author=S.+Zaretskyauthor=A.+K.+Yudin&title=Recent+Advances+in+the+Synthesis+of+Cyclic+Pseudopeptides&doi=10.1016%2Fj.ddtec.2017.11.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Recent advances in the synthesis of cyclic pseudopeptides</span></div><div class="casAuthors">Zaretsky Serge; Yudin Andrei K</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today. Technologies</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">3-10</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Constrained peptides pose tremendous value in drug discovery.  For example, owing to their large surface areas, they offer novel ways at inhibiting protein-protein interactions.  As this field has grown, it has become desirable to introduce non-peptidic functionality into these rings to enable differentiated structure activity relationships and improved pharmacokinetic properties.  Recent advances in the synthesis of cyclic pseudopeptides include macrocyclization through cysteine alkylation, multicomponent reactions, decarboxylative couplings, and novel stapling chemistry.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRGLPLs76yr96-9TkgmBqTqfW6udTcc2eajrsY0twG357ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1MzitlCjsA%253D%253D&md5=c46d4e0edfd6f46413e0118e5187900c</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.ddtec.2017.11.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ddtec.2017.11.004%26sid%3Dliteratum%253Aachs%26aulast%3DZaretsky%26aufirst%3DS.%26aulast%3DYudin%26aufirst%3DA.%2BK.%26atitle%3DRecent%2520Advances%2520in%2520the%2520Synthesis%2520of%2520Cyclic%2520Pseudopeptides%26jtitle%3DDrug%2520Discovery%2520Today%2520Technol.%26date%3D2017%26volume%3D26%26spage%3D3%26epage%3D10%26doi%3D10.1016%2Fj.ddtec.2017.11.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mabonga, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kappo, A. P.</span></span> <span> </span><span class="NLM_article-title">Protein-Protein Interaction Modulators: Advances, Successes and Remaining Challenges</span>. <i>Biophys. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">559</span>– <span class="NLM_lpage">581</span>, <span class="refDoi"> DOI: 10.1007/s12551-019-00570-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1007%2Fs12551-019-00570-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=31301019" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtl2rsb3J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2019&pages=559-581&author=L.+Mabongaauthor=A.+P.+Kappo&title=Protein-Protein+Interaction+Modulators%3A+Advances%2C+Successes+and+Remaining+Challenges&doi=10.1007%2Fs12551-019-00570-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Protein-protein interaction modulators: advances, successes and remaining challenges</span></div><div class="casAuthors">Mabonga, Lloyd; Kappo, Abidemi Paul</div><div class="citationInfo"><span class="NLM_cas:title">Biophysical Reviews</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">559-581</span>CODEN:
                <span class="NLM_cas:coden">BRIECG</span>;
        ISSN:<span class="NLM_cas:issn">1867-2450</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  Modulating disease-relevant protein-protein interactions (PPIs) using small-mol. inhibitors is a quite indispensable diagnostic and therapeutic strategy in averting pathophysiol. cues and disease progression.  Over the years, targeting intracellular PPIs as drug design targets has been a challenging task owing to their highly dynamic and expansive interfacial areas (flat, featureless and relatively large).  However, advances in PPI-focused drug discovery technol. have been reported and a few drugs are already on the market, with some potential drug-like candidates already in clin. trials.  In this article, we review the advances, successes and remaining challenges in the application of small mols. as valuable PPI modulators in disease diagnosis and therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKCN2e4SrXuLVg90H21EOLACvtfcHk0lhlGTI2GwWxSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtl2rsb3J&md5=02c2c9b4b9d1ccb279c76c21776bb2ec</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1007%2Fs12551-019-00570-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs12551-019-00570-x%26sid%3Dliteratum%253Aachs%26aulast%3DMabonga%26aufirst%3DL.%26aulast%3DKappo%26aufirst%3DA.%2BP.%26atitle%3DProtein-Protein%2520Interaction%2520Modulators%253A%2520Advances%252C%2520Successes%2520and%2520Remaining%2520Challenges%26jtitle%3DBiophys.%2520Rev.%26date%3D2019%26volume%3D11%26spage%3D559%26epage%3D581%26doi%3D10.1007%2Fs12551-019-00570-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abdalla, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGaw, L. J.</span></span> <span> </span><span class="NLM_article-title">Natural Cyclic Peptides as an Attractive Modality for Therapeutics: A Mini Review</span>. <i>Molecules</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>23</i></span>, <span class="NLM_elocation-id">2080</span> <span class="refDoi"> DOI: 10.3390/molecules23082080</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.3390%2Fmolecules23082080" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitVaktrvJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2018&author=M.+A.+Abdallaauthor=L.+J.+McGaw&title=Natural+Cyclic+Peptides+as+an+Attractive+Modality+for+Therapeutics%3A+A+Mini+Review&doi=10.3390%2Fmolecules23082080"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Natural cyclic peptides as an attractive modality for therapeutics: a mini review</span></div><div class="casAuthors">Abdalla, Muna Ali; McGaw, Lyndy J.</div><div class="citationInfo"><span class="NLM_cas:title">Molecules</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2080/1-2080/19</span>CODEN:
                <span class="NLM_cas:coden">MOLEFW</span>;
        ISSN:<span class="NLM_cas:issn">1420-3049</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Peptides are important biomols. which facilitate the understanding of complex biol. processes, which in turn could be serendipitous biol. targets for future drugs.  They are classified as a unique therapeutic niche and will play an important role as fascinating agents in the pharmaceutical landscape.  Until now, more than 40 cyclic peptide drugs are currently in the market, and approx. one new cyclopeptide drug enters the market annually on av.  Interestingly, the majority of clin. approved cyclic peptides are derived from natural sources, such as peptide antibiotics and human peptide hormones.  In this report, the importance of cyclic peptides is discussed, and their role in drug discovery as interesting therapeutic biomols. will be highlighted.  Recently isolated naturally occurring cyclic peptides from microorganisms, sponges, and other sources with a wide range of pharmacol. properties are reviewed herein.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGopsXVTy7L4WrVg90H21EOLACvtfcHk0lhlGTI2GwWxSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitVaktrvJ&md5=4b71e28d331f6131595ac933b6a2f71a</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.3390%2Fmolecules23082080&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fmolecules23082080%26sid%3Dliteratum%253Aachs%26aulast%3DAbdalla%26aufirst%3DM.%2BA.%26aulast%3DMcGaw%26aufirst%3DL.%2BJ.%26atitle%3DNatural%2520Cyclic%2520Peptides%2520as%2520an%2520Attractive%2520Modality%2520for%2520Therapeutics%253A%2520A%2520Mini%2520Review%26jtitle%3DMolecules%26date%3D2018%26volume%3D23%26doi%3D10.3390%2Fmolecules23082080" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ali, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atmaj, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Oosterwijk, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groves, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dömling, A.</span></span> <span> </span><span class="NLM_article-title">Stapled Peptides Inhibitors: A New Window for Target Drug Discovery</span>. <i>Comput. Struct. Biotechnol. J.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">263</span>– <span class="NLM_lpage">281</span>, <span class="refDoi"> DOI: 10.1016/j.csbj.2019.01.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1016%2Fj.csbj.2019.01.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=30867891" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC1MXjvFWgsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2019&pages=263-281&author=A.+M.+Aliauthor=J.+Atmajauthor=N.+Van+Oosterwijkauthor=M.+R.+Grovesauthor=A.+D%C3%B6mling&title=Stapled+Peptides+Inhibitors%3A+A+New+Window+for+Target+Drug+Discovery&doi=10.1016%2Fj.csbj.2019.01.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Stapled Peptides Inhibitors: A New Window for Target Drug Discovery</span></div><div class="casAuthors">Ali, Ameena M.; Atmaj, Jack; Van Oosterwijk, Niels; Groves, Matthew R.; Domling, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Computational and Structural Biotechnology Journal</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">263-281</span>CODEN:
                <span class="NLM_cas:coden">CSBJAC</span>;
        ISSN:<span class="NLM_cas:issn">2001-0370</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Protein-protein interaction (PPI) is a hot topic in clin. research as protein networking has a major impact in human disease.  Such PPIs are potential drugs targets, leading to the need to inhibit/block specific PPIs.  While small mol. inhibitors have had some success and reached clin. trials, they have generally failed to address the flat and large nature of PPI surfaces.  As a result, larger biologics were developed for PPI surfaces and they have successfully targeted PPIs located outside the cell.  However, biologics have low bioavailability and cannot reach intracellular targets.  A novel class -hydrocarbon-stapled α-helical peptides that are synthetic mini-proteins locked into their bioactive structure through site-specific introduction of a chem. linker- has shown promise.  Stapled peptides show an ability to inhibit intracellular PPIs that previously have been intractable with traditional small mol. or biologics, suggesting that they offer a novel therapeutic modality.  In this review, we highlight what stapling adds to natural-mimicking peptides, describe the revolution of synthetic chem. techniques and how current drug discovery approaches have been adapted to stabilize active peptide conformations, including ring-closing metathesis (RCM), lactamisation, cycloaddns. and reversible reactions.  We provide an overview on the available stapled peptide high-resoln. structures in the protein data bank, with four selected structures discussed in details due to remarkable interactions of their staple with the target surface.  We believe that stapled peptides are promising drug candidates and open the doors for peptide therapeutics to reach currently "undruggable" space.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8MjXifoHwd7Vg90H21EOLACvtfcHk0lhlGTI2GwWxSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXjvFWgsrg%253D&md5=e2db5b05352df8c74231579cd0f05ad9</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.csbj.2019.01.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.csbj.2019.01.012%26sid%3Dliteratum%253Aachs%26aulast%3DAli%26aufirst%3DA.%2BM.%26aulast%3DAtmaj%26aufirst%3DJ.%26aulast%3DVan%2BOosterwijk%26aufirst%3DN.%26aulast%3DGroves%26aufirst%3DM.%2BR.%26aulast%3DD%25C3%25B6mling%26aufirst%3DA.%26atitle%3DStapled%2520Peptides%2520Inhibitors%253A%2520A%2520New%2520Window%2520for%2520Target%2520Drug%2520Discovery%26jtitle%3DComput.%2520Struct.%2520Biotechnol.%2520J.%26date%3D2019%26volume%3D17%26spage%3D263%26epage%3D281%26doi%3D10.1016%2Fj.csbj.2019.01.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Giordanetto, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kihlberg, J.</span></span> <span> </span><span class="NLM_article-title">Macrocyclic Drugs and Clinical Candidates: What Can Medicinal Chemists Learn from Their Properties?</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">278</span>– <span class="NLM_lpage">295</span>, <span class="refDoi"> DOI: 10.1021/jm400887j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm400887j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtlGnurjI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=278-295&author=F.+Giordanettoauthor=J.+Kihlberg&title=Macrocyclic+Drugs+and+Clinical+Candidates%3A+What+Can+Medicinal+Chemists+Learn+from+Their+Properties%3F&doi=10.1021%2Fjm400887j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Macrocyclic Drugs and Clinical Candidates: What Can Medicinal Chemists Learn from Their Properties?</span></div><div class="casAuthors">Giordanetto, Fabrizio; Kihlberg, Jan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">278-295</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Macrocycles are ideal in efforts to tackle difficult targets, but our understanding of what makes them cell permeable and orally bioavailable is limited.  Anal. of approx. 100 macrocyclic drugs and clin. candidates revealed that macrocycles are predominantly used for infectious disease and in oncol. and that most belong to the macrolide or cyclic peptide class.  A significant no. (N = 34) of these macrocycles are administered orally, revealing that oral bioavailability can be obtained at mol. wts. up to and above 1 kDa and polar surface areas ranging toward 250 Å2.  Moreover, insight from a group of de novo designed oral macrocycles in clin. studies and understanding of how cyclosporin A and model cyclic hexapeptides cross cell membranes may unlock wider opportunities in drug discovery.  However, the no. of oral macrocycles is still low and it remains to be seen if they are outliers or if macrocycles will open up novel oral druggable space.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpaeM2DXwvmiLVg90H21EOLACvtfcHk0lhlGTI2GwWxSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtlGnurjI&md5=a3e1743bd5ba34f73d8e78f8cb1bee5f</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1021%2Fjm400887j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm400887j%26sid%3Dliteratum%253Aachs%26aulast%3DGiordanetto%26aufirst%3DF.%26aulast%3DKihlberg%26aufirst%3DJ.%26atitle%3DMacrocyclic%2520Drugs%2520and%2520Clinical%2520Candidates%253A%2520What%2520Can%2520Medicinal%2520Chemists%2520Learn%2520from%2520Their%2520Properties%253F%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D278%26epage%3D295%26doi%3D10.1021%2Fjm400887j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, A. C.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khanna, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, J.-H.</span></span> <span> </span><span class="NLM_article-title">A Comprehensive Review on Current Advances in Peptide Drug Development and Design</span>. <i>Int. J. Mol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>20</i></span>, <span class="NLM_elocation-id">2383</span> <span class="refDoi"> DOI: 10.3390/ijms20102383</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.3390%2Fijms20102383" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BB3cXosVGiuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2019&author=A.+C.-L.+Leeauthor=J.+L.+Harrisauthor=K.+K.+Khannaauthor=J.-H.+Hong&title=A+Comprehensive+Review+on+Current+Advances+in+Peptide+Drug+Development+and+Design&doi=10.3390%2Fijms20102383"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">A comprehensive review on current advances in peptide drug development and design</span></div><div class="casAuthors">Lee, Andy Ch-Lung; Harris, Janelle Louise; Khanna, Kum Kum; Hong, Ji-Hong</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Molecular Sciences</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2383/1-2383/21</span>CODEN:
                <span class="NLM_cas:coden">IJMCFK</span>;
        ISSN:<span class="NLM_cas:issn">1422-0067</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Protein-protein interactions (PPIs) execute many fundamental cellular functions and have served as prime drug targets over the last two decades.  Interfering intracellular PPIs with small mols. has been extremely difficult for larger or flat binding sites, as antibodies cannot cross the cell membrane to reach such target sites.  In recent years, peptides smaller size and balance of conformational rigidity and flexibility have made them promising candidates for targeting challenging binding interfaces with satisfactory binding affinity and specificity.  Deciphering and characterizing peptide-protein recognition mechanisms is thus central for the invention of peptide-based strategies to interfere with endogenous protein interactions, or improvement of the binding affinity and specificity of existing approaches.  Importantly, a variety of computation-aided rational designs for peptide therapeutics have been developed, which aim to deliver comprehensive docking for peptide-protein interaction interfaces.  Over 60 peptides have been approved and administrated globally in clinics.  Despite this, advances in various docking models are only on the merge of making their contribution to peptide drug development.  In this review, we provide (i) a holistic overview of peptide drug development and the fundamental technologies utilized to date, and (ii) an updated review on key developments of computational modeling of peptide-protein interactions (PepPIs) with an aim to assist exptl. biologists exploit suitable docking methods to advance peptide interfering strategies against PPIs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbSlnJ71PNHbVg90H21EOLACvtfcHk0ljubvu4Jof5Ig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXosVGiuw%253D%253D&md5=344174f95cb22357d0b92d797874d81b</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.3390%2Fijms20102383&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fijms20102383%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DA.%2BC.-L.%26aulast%3DHarris%26aufirst%3DJ.%2BL.%26aulast%3DKhanna%26aufirst%3DK.%2BK.%26aulast%3DHong%26aufirst%3DJ.-H.%26atitle%3DA%2520Comprehensive%2520Review%2520on%2520Current%2520Advances%2520in%2520Peptide%2520Drug%2520Development%2520and%2520Design%26jtitle%3DInt.%2520J.%2520Mol.%2520Sci.%26date%3D2019%26volume%3D20%26doi%3D10.3390%2Fijms20102383" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Naylor, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bockus, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blanco, M.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lokey, R. S.</span></span> <span> </span><span class="NLM_article-title">Cyclic Peptide Natural Products Chart the Frontier of Oral Bioavailability in the Pursuit of Undruggable Targets</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">141</span>– <span class="NLM_lpage">147</span>, <span class="refDoi"> DOI: 10.1016/j.cbpa.2017.04.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1016%2Fj.cbpa.2017.04.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=28570865" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC2sXovVSgsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2017&pages=141-147&author=M.+R.+Naylorauthor=A.+T.+Bockusauthor=M.-J.+Blancoauthor=R.+S.+Lokey&title=Cyclic+Peptide+Natural+Products+Chart+the+Frontier+of+Oral+Bioavailability+in+the+Pursuit+of+Undruggable+Targets&doi=10.1016%2Fj.cbpa.2017.04.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Cyclic peptide natural products chart the frontier of oral bioavailability in the pursuit of undruggable targets</span></div><div class="casAuthors">Naylor, Matthew R.; Bockus, Andrew T.; Blanco, Maria-Jesus; Lokey, R. Scott</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">141-147</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">As interest in protein-protein interactions and other previously-undruggable targets increases, medicinal chemists are returning to natural products for design inspiration toward mols. that transcend the paradigm of small mol. drugs.  These compds., esp. peptides, often have poor ADME properties and thus require a more nuanced understanding of structure-property relationships to achieve desirable oral bioavailability.  Although there have been few clin. successes in this chem. space to date, recent work has identified opportunities to introduce favorable physicochem. properties to peptidic macrocycles that maintain activity and oral bioavailability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoanPrD1D-ePrVg90H21EOLACvtfcHk0ljubvu4Jof5Ig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXovVSgsLw%253D&md5=175671ba338608979d38d340866df885</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2017.04.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2017.04.012%26sid%3Dliteratum%253Aachs%26aulast%3DNaylor%26aufirst%3DM.%2BR.%26aulast%3DBockus%26aufirst%3DA.%2BT.%26aulast%3DBlanco%26aufirst%3DM.-J.%26aulast%3DLokey%26aufirst%3DR.%2BS.%26atitle%3DCyclic%2520Peptide%2520Natural%2520Products%2520Chart%2520the%2520Frontier%2520of%2520Oral%2520Bioavailability%2520in%2520the%2520Pursuit%2520of%2520Undruggable%2520Targets%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2017%26volume%3D38%26spage%3D141%26epage%3D147%26doi%3D10.1016%2Fj.cbpa.2017.04.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Poongavanam, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Danelius, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peintner, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alcaraz, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caron, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cummings, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wlodek, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erdelyi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hawkins, P. C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ermondi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kihlberg, J.</span></span> <span> </span><span class="NLM_article-title">Conformational Sampling of Macrocyclic Drugs in Different Environments: Can We Find the Relevant Conformations?</span>. <i>ACS Omega</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">11742</span>– <span class="NLM_lpage">11757</span>, <span class="refDoi"> DOI: 10.1021/acsomega.8b01379</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsomega.8b01379" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslKhtrnI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2018&pages=11742-11757&author=V.+Poongavanamauthor=E.+Daneliusauthor=S.+Peintnerauthor=L.+Alcarazauthor=G.+Caronauthor=M.+D.+Cummingsauthor=S.+Wlodekauthor=M.+Erdelyiauthor=P.+C.+D.+Hawkinsauthor=G.+Ermondiauthor=J.+Kihlberg&title=Conformational+Sampling+of+Macrocyclic+Drugs+in+Different+Environments%3A+Can+We+Find+the+Relevant+Conformations%3F&doi=10.1021%2Facsomega.8b01379"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Conformational Sampling of Macrocyclic Drugs in Different Environments: Can We Find the Relevant Conformations?</span></div><div class="casAuthors">Poongavanam, Vasanthanathan; Danelius, Emma; Peintner, Stefan; Alcaraz, Lilian; Caron, Giulia; Cummings, Maxwell D.; Wlodek, Stanislaw; Erdelyi, Mate; Hawkins, Paul C. D.; Ermondi, Giuseppe; Kihlberg, Jan</div><div class="citationInfo"><span class="NLM_cas:title">ACS Omega</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">11742-11757</span>CODEN:
                <span class="NLM_cas:coden">ACSODF</span>;
        ISSN:<span class="NLM_cas:issn">2470-1343</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Conformational flexibility is a major determinant of the properties of macrocycles and other drugs in beyond rule of 5 (bRo5) space.  Prediction of conformations is essential for design of drugs in this space and the authors have evaluated three tools for conformational sampling of a set of ten bRo5 drugs and clin. candidates in polar and apolar environments.  The distance-geometry based OMEGA was found to yield ensembles spanning larger structure and property spaces than the two MD based methods, MOE-LowMode MD (MOE) and MacroModel-Macrocycle Conformational Sampling (MC).  Both MC and OMEGA, but not MOE, generated different ensembles for polar and apolar environments.  All three conformational search methods generated conformers similar to the crystal structure conformers for nine of the ten compds., with OMEGA performing somewhat better than MOE and MC.  MOE and OMEGA found all six conformers of roxithromycin that were identified by NMR in aq. solns., while only OMEGA sampled the three conformers obsd. in chloroform.  The authors suggest that characterization of conformers using mol. descriptors, e.g. the radius of gyration (Rgyr) and polar surface area (PSA), is preferred to energy- or RMSD-based methods for selection of biol. relevant conformers in drug discovery in bRo5 space.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoetOG-yDeciLVg90H21EOLACvtfcHk0ljubvu4Jof5Ig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslKhtrnI&md5=a8722551b161e907daf55146f22b6d57</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Facsomega.8b01379&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsomega.8b01379%26sid%3Dliteratum%253Aachs%26aulast%3DPoongavanam%26aufirst%3DV.%26aulast%3DDanelius%26aufirst%3DE.%26aulast%3DPeintner%26aufirst%3DS.%26aulast%3DAlcaraz%26aufirst%3DL.%26aulast%3DCaron%26aufirst%3DG.%26aulast%3DCummings%26aufirst%3DM.%2BD.%26aulast%3DWlodek%26aufirst%3DS.%26aulast%3DErdelyi%26aufirst%3DM.%26aulast%3DHawkins%26aufirst%3DP.%2BC.%2BD.%26aulast%3DErmondi%26aufirst%3DG.%26aulast%3DKihlberg%26aufirst%3DJ.%26atitle%3DConformational%2520Sampling%2520of%2520Macrocyclic%2520Drugs%2520in%2520Different%2520Environments%253A%2520Can%2520We%2520Find%2520the%2520Relevant%2520Conformations%253F%26jtitle%3DACS%2520Omega%26date%3D2018%26volume%3D3%26spage%3D11742%26epage%3D11757%26doi%3D10.1021%2Facsomega.8b01379" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dougherty, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pei, D.</span></span> <span> </span><span class="NLM_article-title">Targeting Intracellular Protein-Protein Interactions with Cell-Permeable Cyclic Peptides</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">80</span>– <span class="NLM_lpage">86</span>, <span class="refDoi"> DOI: 10.1016/j.cbpa.2017.03.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1016%2Fj.cbpa.2017.03.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=28388463" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC2sXls1KlsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2017&pages=80-86&author=Z.+Qianauthor=P.+G.+Doughertyauthor=D.+Pei&title=Targeting+Intracellular+Protein-Protein+Interactions+with+Cell-Permeable+Cyclic+Peptides&doi=10.1016%2Fj.cbpa.2017.03.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting intracellular protein-protein interactions with cell-permeable cyclic peptides</span></div><div class="casAuthors">Qian, Ziqing; Dougherty, Patrick G.; Pei, Dehua</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">80-86</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Intracellular protein-protein interactions (PPIs) are challenging targets for conventional drug modalities, because small mols. generally do not bind to their large, flat binding sites with high affinity, whereas monoclonal antibodies cannot cross the cell membrane to reach the targets.  Cyclic peptides in the 700-2000 mol.-wt. range have the sufficient size and a balanced conformational flexibility/rigidity for binding to flat PPI interfaces with antibody-like affinity and specificity.  Several powerful cyclic peptide library technologies were developed over the past decade to rapidly discover potent, specific cyclic peptide ligands against proteins of interest including those involved in PPIs.  Methods are also being developed to enhance the membrane permeability of cyclic peptides through both passive diffusion and active transport mechanisms.  Integration of the permeability-enhancing elements into cyclic peptide design has led to an increasing no. of cell-permeable and biol. active cyclic peptides against intracellular PPIs.  In this account, we review the recent developments in the design and synthesis of cell-permeable cyclic peptides.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_IPrrU8rcMbVg90H21EOLACvtfcHk0ljR65qT9ldnRg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXls1KlsLg%253D&md5=2ea4ac695ee1f1651e5da4ed0bb7cf0b</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2017.03.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2017.03.011%26sid%3Dliteratum%253Aachs%26aulast%3DQian%26aufirst%3DZ.%26aulast%3DDougherty%26aufirst%3DP.%2BG.%26aulast%3DPei%26aufirst%3DD.%26atitle%3DTargeting%2520Intracellular%2520Protein-Protein%2520Interactions%2520with%2520Cell-Permeable%2520Cyclic%2520Peptides%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2017%26volume%3D38%26spage%3D80%26epage%3D86%26doi%3D10.1016%2Fj.cbpa.2017.03.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Robertson, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spring, D. R.</span></span> <span> </span><span class="NLM_article-title">Using Peptidomimetics and Constrained Peptides as Valuable Tools for Inhibiting Protein–Protein Interactions</span>. <i>Molecules</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>23</i></span> (<span class="NLM_issue">4</span>),  <span class="NLM_fpage">959</span>– <span class="NLM_lpage">976</span>, <span class="refDoi"> DOI: 10.3390/molecules23040959</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.3390%2Fmolecules23040959" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslGmsbrE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2018&pages=959-976&issue=4&author=N.+S.+Robertsonauthor=D.+R.+Spring&title=Using+Peptidomimetics+and+Constrained+Peptides+as+Valuable+Tools+for+Inhibiting+Protein%E2%80%93Protein+Interactions&doi=10.3390%2Fmolecules23040959"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Using peptidomimetics and constrained peptides as valuable tools for inhibiting protein-protein interactions</span></div><div class="casAuthors">Robertson, Naomi S.; Spring, David R.</div><div class="citationInfo"><span class="NLM_cas:title">Molecules</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">959/1-959/18</span>CODEN:
                <span class="NLM_cas:coden">MOLEFW</span>;
        ISSN:<span class="NLM_cas:issn">1420-3049</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Protein-protein interactions (PPIs) are tremendously important for the function of many biol. processes.  However, because of the structure of many protein-protein interfaces (flat, featureless and relatively large), they have largely been overlooked as potential drug targets.  In this review, we highlight the current tools used to study the mol. recognition of PPIs through the use of different peptidomimetics, from small mols. and scaffolds to peptides.  Then, we focus on constrained peptides, and in particular, ways to constrain -helixes through stapling using both one- and two-component techniques.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrwxPEMuLK6abVg90H21EOLACvtfcHk0ljR65qT9ldnRg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslGmsbrE&md5=480718d55ee9814eba533868e249faf1</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.3390%2Fmolecules23040959&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fmolecules23040959%26sid%3Dliteratum%253Aachs%26aulast%3DRobertson%26aufirst%3DN.%2BS.%26aulast%3DSpring%26aufirst%3DD.%2BR.%26atitle%3DUsing%2520Peptidomimetics%2520and%2520Constrained%2520Peptides%2520as%2520Valuable%2520Tools%2520for%2520Inhibiting%2520Protein%25E2%2580%2593Protein%2520Interactions%26jtitle%3DMolecules%26date%3D2018%26volume%3D23%26issue%3D4%26spage%3D959%26epage%3D976%26doi%3D10.3390%2Fmolecules23040959" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rubin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qvit, N.</span></span> <span> </span><span class="NLM_article-title">Cyclic Peptides for Protein-Protein Interaction Targets: Applications to Human Disease</span>. <i>Crit. Rev. Eukaryotic Gene Expression</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">199</span>– <span class="NLM_lpage">221</span>, <span class="refDoi"> DOI: 10.1615/CritRevEukaryotGeneExpr.2016016525</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1615%2FCritRevEukaryotGeneExpr.2016016525" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=27650985" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A280%3ADC%252BC2svis1ensg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2016&pages=199-221&author=S.+Rubinauthor=N.+Qvit&title=Cyclic+Peptides+for+Protein-Protein+Interaction+Targets%3A+Applications+to+Human+Disease&doi=10.1615%2FCritRevEukaryotGeneExpr.2016016525"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Cyclic Peptides for Protein-Protein Interaction Targets: Applications to Human Disease</span></div><div class="casAuthors">Rubin Samuel; Qvit Nir</div><div class="citationInfo"><span class="NLM_cas:title">Critical reviews in eukaryotic gene expression</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">199-221</span>
        ISSN:<span class="NLM_cas:issn">1045-4403</span>.
    </div><div class="casAbstract">Protein-protein interactions (PPIs) represent a significant portion of functionally relevant biological interactions, and therefore potential therapeutic targets.  Small molecules were traditionally used to target PPIs.  However, many PPI surfaces lack binding pockets due to their large and flat structures.  Antibodies can also be used to modulate PPIs, but they are expensive and not cell permeable.  Linear peptides are less expensive to produce than antibodies and are generally more selective than small molecules, but they are limited by decreased stability and poor permeability.  Modified peptides (peptidomimetics, e.g., cyclic peptides) can overcome these obstacles.  Advantages of using cyclic peptidomimetics to modulate PPIs derive from their conformational constraint, which supports target specificity, cell permeability, and metabolic stability.  Methods for rational design coupled with high-throughput techniques continue to support advances in the field.  Further development of cyclic peptidomimetics to modulate PPIs will improve treatment of human diseases, such as cancer, infection, neurodegeneration, and autoimmunity.  Here we describe several cyclic peptidomimetics that are currently used as drugs and many potential cyclic peptides PPI inhibitors in different stages of pre-clinical and clinical development.  Further development of cyclic peptidomimetics to modulate PPIs will continue to improve treatment of human diseases, such as cancer, infection, neurodegeneration, and autoimmunity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRv6Xi9HeQ4puQ4Pl9hzjQhfW6udTcc2eaM-1PhSykOarntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2svis1ensg%253D%253D&md5=69772e8646e86a3e209a4328b88a83a1</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1615%2FCritRevEukaryotGeneExpr.2016016525&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1615%252FCritRevEukaryotGeneExpr.2016016525%26sid%3Dliteratum%253Aachs%26aulast%3DRubin%26aufirst%3DS.%26aulast%3DQvit%26aufirst%3DN.%26atitle%3DCyclic%2520Peptides%2520for%2520Protein-Protein%2520Interaction%2520Targets%253A%2520Applications%2520to%2520Human%2520Disease%26jtitle%3DCrit.%2520Rev.%2520Eukaryotic%2520Gene%2520Expression%26date%3D2016%26volume%3D26%26spage%3D199%26epage%3D221%26doi%3D10.1615%2FCritRevEukaryotGeneExpr.2016016525" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Russo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aiello, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grieco, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marasco, D.</span></span> <span> </span><span class="NLM_article-title">Targeting “Undruggable” Proteins: Design of Synthetic Cyclopeptides</span>. <i>Curr. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">748</span>– <span class="NLM_lpage">762</span>, <span class="refDoi"> DOI: 10.2174/0929867323666160112122540</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.2174%2F0929867323666160112122540" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=26758797" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC28Xlt1KjtL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2016&pages=748-762&author=A.+Russoauthor=C.+Aielloauthor=P.+Griecoauthor=D.+Marasco&title=Targeting+%E2%80%9CUndruggable%E2%80%9D+Proteins%3A+Design+of+Synthetic+Cyclopeptides&doi=10.2174%2F0929867323666160112122540"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting "Undruggable" Proteins: Design of Synthetic Cyclopeptides</span></div><div class="casAuthors">Russo, Anna; Aiello, Carmela; Grieco, Paolo; Marasco, Daniela</div><div class="citationInfo"><span class="NLM_cas:title">Current Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">748-762</span>CODEN:
                <span class="NLM_cas:coden">CMCHE7</span>;
        ISSN:<span class="NLM_cas:issn">0929-8673</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">The development of synthetic macrocycles represents a powerful approach toward the identification of new protein binders or inhibitors of Protein-Protein Interactions (PPI) which are known to play key biol. roles in cancer signaling as well as in the regulation of cell division cycle.  Structural investigations led to identify "hot loops" sharing common motifs that are mainly involved in PPIs.  Most PPIs occur through large and flat surfaces; currently these protein complexes are defined as "undruggable" by conventional drug-discovery approaches, since the identification of small mols. to inhibit these targets is often unreachable.  Typically macrocycles are 500-2000 Da in size, having 12-membered, or more, ring architecture: they do not obey the Lipinski's rule but, for them nature offers many examples as therapeutic agents such as erythromycin (antibiotic), cyclosporin (immunosuppressant) and somatostatin (hormone).  Peptide-based macrocycles offer the advantages of directly mimicking secondary structures involved in PPIs and their pharmacol. application is related to the potential improvement of lead peptides in terms of potency, selectivity, stability and cell permeation.  The promising relevance of cyclopeptides prompted to develop new synthetic methods for cyclization: often biotechnol. approaches as well as regioselective reactions have been employed to cyclize peptides rapidly and nearly quant.  Moreover, different synthetic strategies in peptidomimetics' macrocyclization are actually available based on surrogate peptide bonds or NCL (Native Chem. Ligation) methods.  In this review we focus on the most common methods for the prepn. of cyclopeptides and interesting applications of the last decade.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZqHzNKiqC0bVg90H21EOLACvtfcHk0ljR65qT9ldnRg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xlt1KjtL0%253D&md5=c066116e3227a90037c0e3c77af6e50c</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.2174%2F0929867323666160112122540&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F0929867323666160112122540%26sid%3Dliteratum%253Aachs%26aulast%3DRusso%26aufirst%3DA.%26aulast%3DAiello%26aufirst%3DC.%26aulast%3DGrieco%26aufirst%3DP.%26aulast%3DMarasco%26aufirst%3DD.%26atitle%3DTargeting%2520%25E2%2580%259CUndruggable%25E2%2580%259D%2520Proteins%253A%2520Design%2520of%2520Synthetic%2520Cyclopeptides%26jtitle%3DCurr.%2520Med.%2520Chem.%26date%3D2016%26volume%3D23%26spage%3D748%26epage%3D762%26doi%3D10.2174%2F0929867323666160112122540" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stone, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deber, C. M.</span></span> <span> </span><span class="NLM_article-title">Therapeutic Design of Peptide Modulators of Protein-Protein Interactions in Membranes</span>. <i>Biochim. Biophys. Acta, Biomembr.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>1859</i></span>,  <span class="NLM_fpage">577</span>– <span class="NLM_lpage">585</span>, <span class="refDoi"> DOI: 10.1016/j.bbamem.2016.08.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1016%2Fj.bbamem.2016.08.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=27580024" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVegsrbE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1859&publication_year=2017&pages=577-585&author=T.+A.+Stoneauthor=C.+M.+Deber&title=Therapeutic+Design+of+Peptide+Modulators+of+Protein-Protein+Interactions+in+Membranes&doi=10.1016%2Fj.bbamem.2016.08.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Therapeutic design of peptide modulators of protein-protein interactions in membranes</span></div><div class="casAuthors">Stone, Tracy A.; Deber, Charles M.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Biomembranes</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">1859</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">577-585</span>CODEN:
                <span class="NLM_cas:coden">BBBMBS</span>;
        ISSN:<span class="NLM_cas:issn">0005-2736</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Membrane proteins play the central roles in a variety of cellular processes, ranging from nutrient uptake and signalling, to cell-cell communication.  Their biol. functions are directly related to how they fold and assemble; defects often lead to disease.  Protein-protein interactions (PPIs) within the membrane are therefore of great interest as therapeutic targets.  Here we review the progress in the application of membrane-insertable peptides for the disruption or stabilization of membrane-based PPIs.  We describe the design and prepn. of transmembrane peptide mimics; and of several categories of peptidomimetics used for study, including D-enantiomers, non-natural amino acids, peptoids, and β-peptides.  Further aspects of the review describe modifications to membrane-insertable peptides, including lipidation and cyclization via hydrocarbon stapling.  These approaches provide a pathway toward the development of metabolically stable, non-toxic, and efficacious peptide modulators of membrane-based PPIs.  This article is part of a Special Issue entitled: Lipid order/lipid defects and lipid-control of protein activity edited by Dirk Schneider.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoIYM33lXE3IbVg90H21EOLACvtfcHk0liZRE4yhUcVsw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVegsrbE&md5=c89943990f8a9f222e7cf6089f8b3f42</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.bbamem.2016.08.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbamem.2016.08.013%26sid%3Dliteratum%253Aachs%26aulast%3DStone%26aufirst%3DT.%2BA.%26aulast%3DDeber%26aufirst%3DC.%2BM.%26atitle%3DTherapeutic%2520Design%2520of%2520Peptide%2520Modulators%2520of%2520Protein-Protein%2520Interactions%2520in%2520Membranes%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Biomembr.%26date%3D2017%26volume%3D1859%26spage%3D577%26epage%3D585%26doi%3D10.1016%2Fj.bbamem.2016.08.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wójcik, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berlicki, Ł.</span></span> <span> </span><span class="NLM_article-title">Peptide-Based Inhibitors of Protein-Protein Interactions</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">707</span>– <span class="NLM_lpage">713</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2015.12.084</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1016%2Fj.bmcl.2015.12.084" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=26764190" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC28XitFKlsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2016&pages=707-713&author=P.+W%C3%B3jcikauthor=%C5%81.+Berlicki&title=Peptide-Based+Inhibitors+of+Protein-Protein+Interactions&doi=10.1016%2Fj.bmcl.2015.12.084"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Peptide-based inhibitors of protein-protein interactions</span></div><div class="casAuthors">Wojcik, Paulina; Berlicki, Lukasz</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">707-713</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Protein-protein interactions (PPIs) are key elements of several important biol. processes and have emerged as valuable targets in medicinal chem.  Importantly, numerous specific protein-protein interactions (e.g., p53-HDM2 and Bcl-2-BH3 domains) were found to be involved in the development of several diseases, including various types of cancer.  In general, the discovery of new synthetic PPI inhibitors is a challenging task because protein surfaces have not evolved in a manner that allows for specific binding of low mol. wt. compds.  Here, we review the discovery strategies for peptide-based PPI inhibitors.  Although peptide-based drug candidates exhibit significant drawbacks (in particular, low proteolytic stability), modifications of either the side chains or backbone could provide mols. of interest.  Moreover, due to the large mol. size of peptide-based compds., the discovery of mols. that specifically interact with extended protein surfaces is possible.  Two major strategies for constructing peptide-based PPI inhibitors are as follows: (a) cyclization (e.g., stapled peptides) and (b) modification of the backbone structure (e.g., β-peptides and peptoids).  These approaches for constructing PPI inhibitors enhance both the inhibitory activity and pharmacokinetic properties compared with non-modified α-peptides.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzQ6u5WIbqyrVg90H21EOLACvtfcHk0liZRE4yhUcVsw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitFKlsw%253D%253D&md5=e4c33a3743f9a6d6829ac3c2e5904f01</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2015.12.084&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2015.12.084%26sid%3Dliteratum%253Aachs%26aulast%3DW%25C3%25B3jcik%26aufirst%3DP.%26aulast%3DBerlicki%26aufirst%3D%25C5%2581.%26atitle%3DPeptide-Based%2520Inhibitors%2520of%2520Protein-Protein%2520Interactions%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2016%26volume%3D26%26spage%3D707%26epage%3D713%26doi%3D10.1016%2Fj.bmcl.2015.12.084" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yamagishi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoji, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miyagawa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawakami, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katoh, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goto, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suga, H.</span></span> <span> </span><span class="NLM_article-title">Natural Product-like Macrocyclic N-Methyl-Peptide Inhibitors against a Ubiquitin Ligase Uncovered from a Ribosome-Expressed de Novo Library</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">1562</span>– <span class="NLM_lpage">1570</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2011.09.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1016%2Fj.chembiol.2011.09.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=22195558" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs12htrnI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2011&pages=1562-1570&author=Y.+Yamagishiauthor=I.+Shojiauthor=S.+Miyagawaauthor=T.+Kawakamiauthor=T.+Katohauthor=Y.+Gotoauthor=H.+Suga&title=Natural+Product-like+Macrocyclic+N-Methyl-Peptide+Inhibitors+against+a+Ubiquitin+Ligase+Uncovered+from+a+Ribosome-Expressed+de+Novo+Library&doi=10.1016%2Fj.chembiol.2011.09.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Natural Product-Like Macrocyclic N-Methyl-Peptide Inhibitors against a Ubiquitin Ligase Uncovered from a Ribosome-Expressed De Novo Library</span></div><div class="casAuthors">Yamagishi, Yusuke; Shoji, Ikuo; Miyagawa, Shoji; Kawakami, Takashi; Katoh, Takayuki; Goto, Yuki; Suga, Hiroaki</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1562-1570</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Summary: Naturally occurring peptides often possess macrocyclic and N-methylated backbone.  These features grant them structural rigidity, high affinity to targets, proteolytic resistance, and occasionally membrane permeability.  Because such peptides are produced by either nonribosomal peptide synthetases or enzymic posttranslational modifications, it is yet a formidable challenge in degenerating sequence or length and prepg. libraries for screening bioactive mols.  Here, we report a new means of synthesizing a de novo library of "natural product-like" macrocyclic N-methyl-peptides using translation machinery under the reprogrammed genetic code, which is coupled with an in vitro display technique, referred to as RaPID (random nonstandard peptides integrated discovery) system.  This system allows for rapid selection of strong binders against an arbitrarily chosen therapeutic target.  Here, we have demonstrated the selection of anti-E6AP macrocyclic N-methyl-peptides, one of which strongly inhibits polyubiquitination of proteins such as p53.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5j7qHsUITV7Vg90H21EOLACvtfcHk0liZRE4yhUcVsw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs12htrnI&md5=14dd193429cd72bfb8dc87a9be5a7cf9</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2011.09.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2011.09.013%26sid%3Dliteratum%253Aachs%26aulast%3DYamagishi%26aufirst%3DY.%26aulast%3DShoji%26aufirst%3DI.%26aulast%3DMiyagawa%26aufirst%3DS.%26aulast%3DKawakami%26aufirst%3DT.%26aulast%3DKatoh%26aufirst%3DT.%26aulast%3DGoto%26aufirst%3DY.%26aulast%3DSuga%26aufirst%3DH.%26atitle%3DNatural%2520Product-like%2520Macrocyclic%2520N-Methyl-Peptide%2520Inhibitors%2520against%2520a%2520Ubiquitin%2520Ligase%2520Uncovered%2520from%2520a%2520Ribosome-Expressed%2520de%2520Novo%2520Library%26jtitle%3DChem.%2520Biol.%26date%3D2011%26volume%3D18%26spage%3D1562%26epage%3D1570%26doi%3D10.1016%2Fj.chembiol.2011.09.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zorzi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deyle, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heinis, C.</span></span> <span> </span><span class="NLM_article-title">Cyclic Peptide Therapeutics: Past, Present and Future</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">24</span>– <span class="NLM_lpage">29</span>, <span class="refDoi"> DOI: 10.1016/j.cbpa.2017.02.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1016%2Fj.cbpa.2017.02.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=28249193" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjsFGksLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2017&pages=24-29&author=A.+Zorziauthor=K.+Deyleauthor=C.+Heinis&title=Cyclic+Peptide+Therapeutics%3A+Past%2C+Present+and+Future&doi=10.1016%2Fj.cbpa.2017.02.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Cyclic peptide therapeutics: past, present and future</span></div><div class="casAuthors">Zorzi, Alessandro; Deyle, Kaycie; Heinis, Christian</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">24-29</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Cyclic peptides combine several favorable properties such as good binding affinity, target selectivity and low toxicity that make them an attractive modality for the development of therapeutics.  Over 40 cyclic peptide drugs are currently in clin. use and around one new cyclic peptide drug enters the market every year on av.  The vast majority of clin. approved cyclic peptides are derived from natural products, such as antimicrobials or human peptide hormones.  New powerful techniques based on rational design and in vitro evolution have enabled the de novo development of cyclic peptide ligands to targets for which nature does not offer solns.  A look at the cyclic peptides currently under clin. evaluation shows that several have been developed using such techniques.  This new source for cyclic peptide ligands introduces a freshness to the field, and it is likely that de novo developed cyclic peptides will be in clin. use in the near future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwchcMOj1bSbVg90H21EOLACvtfcHk0liWwPojEfPkcw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjsFGksLg%253D&md5=4b9581365b92b649dad8c7b22c5bd9c3</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2017.02.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2017.02.006%26sid%3Dliteratum%253Aachs%26aulast%3DZorzi%26aufirst%3DA.%26aulast%3DDeyle%26aufirst%3DK.%26aulast%3DHeinis%26aufirst%3DC.%26atitle%3DCyclic%2520Peptide%2520Therapeutics%253A%2520Past%252C%2520Present%2520and%2520Future%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2017%26volume%3D38%26spage%3D24%26epage%3D29%26doi%3D10.1016%2Fj.cbpa.2017.02.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ermert, P.</span></span> <span> </span><span class="NLM_article-title">Design, Properties and Recent Application of Macrocycles in Medicinal Chemistry</span>. <i>Chimia</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>71</i></span>,  <span class="NLM_fpage">678</span>– <span class="NLM_lpage">702</span>, <span class="refDoi"> DOI: 10.2533/chimia.2017.678</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.2533%2Fchimia.2017.678" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=29070413" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsFeitbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2017&pages=678-702&author=P.+Ermert&title=Design%2C+Properties+and+Recent+Application+of+Macrocycles+in+Medicinal+Chemistry&doi=10.2533%2Fchimia.2017.678"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Design, properties and recent application of macrocycles in medicinal chemistry</span></div><div class="casAuthors">Ermert, Philipp</div><div class="citationInfo"><span class="NLM_cas:title">Chimia</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">71</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">678-702</span>CODEN:
                <span class="NLM_cas:coden">CHIMAD</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Swiss Chemical Society</span>)
        </div><div class="casAbstract">Macrocyclic compds. have recently received increasing attention in drug discovery as these compds. offer the potential to modulate difficult targets and to access novel chemotypes.  Approaches towards libraries of macrocyclic compds. based on modular org. synthesis and applications of these technol. platforms to find and improve biol. active compds. are introduced in this minireview.  Alternatively, lead compds. may be obtained by truncation and modification of macrocyclic natural products.  Selected medicinal chem. programs are discussed, illustrating a macrocyclization approach toward ligands with improved properties.  The design of such ligands is often informed by X-ray crystal structures of protein-ligand complexes.  Efforts to understand cellular permeability and oral bioavailability of cyclic peptides and non-peptidic macrocycles are summarized.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqo_L_ef2BYLrVg90H21EOLACvtfcHk0liWwPojEfPkcw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsFeitbg%253D&md5=aca0ab87a30863de362d9002c3d9f6cc</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.2533%2Fchimia.2017.678&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2533%252Fchimia.2017.678%26sid%3Dliteratum%253Aachs%26aulast%3DErmert%26aufirst%3DP.%26atitle%3DDesign%252C%2520Properties%2520and%2520Recent%2520Application%2520of%2520Macrocycles%2520in%2520Medicinal%2520Chemistry%26jtitle%3DChimia%26date%3D2017%26volume%3D71%26spage%3D678%26epage%3D702%26doi%3D10.2533%2Fchimia.2017.678" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ermert, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luther, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zbinden, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Obrecht, D.</span></span> <span> </span><span class="NLM_article-title">Frontier Between Cyclic Peptides and Macrocycles</span>. <i>Methods Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>2001</i></span>,  <span class="NLM_fpage">147</span>– <span class="NLM_lpage">202</span>, <span class="refDoi"> DOI: 10.1007/978-1-4939-9504-2_9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1007%2F978-1-4939-9504-2_9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=31134572" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC1MXit1eis7zE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2001&publication_year=2019&pages=147-202&author=P.+Ermertauthor=A.+Lutherauthor=P.+Zbindenauthor=D.+Obrecht&title=Frontier+Between+Cyclic+Peptides+and+Macrocycles&doi=10.1007%2F978-1-4939-9504-2_9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Frontier between cyclic peptides and macrocycles</span></div><div class="casAuthors">Ermert, Philipp; Luther, Anatol; Zbinden, Peter; Obrecht, Daniel</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Molecular Biology (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">2001</span>
        (<span class="NLM_cas:issue">Cyclic Peptide Design</span>),
    <span class="NLM_cas:pages">147-202</span>CODEN:
                <span class="NLM_cas:coden">MMBIED</span>;
        ISSN:<span class="NLM_cas:issn">1940-6029</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">This review describes a selection of macrocyclic natural products and structurally modified analogs contg. peptidic and non-peptidic elements as structural features that potentially modulate cellular permeability.  Examples range from exclusively peptidic structures like cyclosporin A or phepropeptins to compds. with mostly non-peptidic character, such as telomestatin or largazole.  Furthermore, semisynthetic approaches and synthesis platforms to generate general and focused libraries of compds. at the interface of cyclic peptides and non-peptidic macrocycles are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomwEAzfYUWmbVg90H21EOLACvtfcHk0liWwPojEfPkcw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXit1eis7zE&md5=6526545a636d7b569524afab9a13b517</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1007%2F978-1-4939-9504-2_9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-1-4939-9504-2_9%26sid%3Dliteratum%253Aachs%26aulast%3DErmert%26aufirst%3DP.%26aulast%3DLuther%26aufirst%3DA.%26aulast%3DZbinden%26aufirst%3DP.%26aulast%3DObrecht%26aufirst%3DD.%26atitle%3DFrontier%2520Between%2520Cyclic%2520Peptides%2520and%2520Macrocycles%26jtitle%3DMethods%2520Mol.%2520Biol.%26date%3D2019%26volume%3D2001%26spage%3D147%26epage%3D202%26doi%3D10.1007%2F978-1-4939-9504-2_9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lau, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dunn, M. K.</span></span> <span> </span><span class="NLM_article-title">Therapeutic Peptides: Historical Perspectives, Current Development Trends, and Future Directions</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">2700</span>– <span class="NLM_lpage">2707</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2017.06.052</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1016%2Fj.bmc.2017.06.052" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=28720325" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtFygt7jP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2018&pages=2700-2707&author=J.+L.+Lauauthor=M.+K.+Dunn&title=Therapeutic+Peptides%3A+Historical+Perspectives%2C+Current+Development+Trends%2C+and+Future+Directions&doi=10.1016%2Fj.bmc.2017.06.052"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Therapeutic peptides: Historical perspectives, current development trends, and future directions</span></div><div class="casAuthors">Lau, Jolene L.; Dunn, Michael K.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2700-2707</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Peptide therapeutics have played a notable role in medical practice since the advent of insulin therapy in the 1920s.  Over 60 peptide drugs are approved in the United States and other major markets, and peptides continue to enter clin. development at a steady pace.  Peptide drug discovery has diversified beyond its traditional focus on endogenous human peptides to include a broader range of structures identified from other natural sources or through medicinal chem. efforts.  The authors maintain a comprehensive dataset on peptides that have entered human clin. studies that includes over 150 peptides in active development today.  Here the authors provide an overview of the peptide therapeutic landscape, including historical perspectives, mol. characteristics, regulatory benchmarks, and a therapeutic area breakdown.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqtmwxU3yil6LVg90H21EOLACvtfcHk0liWwPojEfPkcw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtFygt7jP&md5=c4dc775ee302737cfc5080ea3f1c8c74</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2017.06.052&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2017.06.052%26sid%3Dliteratum%253Aachs%26aulast%3DLau%26aufirst%3DJ.%2BL.%26aulast%3DDunn%26aufirst%3DM.%2BK.%26atitle%3DTherapeutic%2520Peptides%253A%2520Historical%2520Perspectives%252C%2520Current%2520Development%2520Trends%252C%2520and%2520Future%2520Directions%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2018%26volume%3D26%26spage%3D2700%26epage%3D2707%26doi%3D10.1016%2Fj.bmc.2017.06.052" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hruby, V. J.</span></span> <span> </span><span class="NLM_article-title">Peptide Science: Exploring the Use of Chemical Principles and Interdisciplinary Collaboration for Understanding Life Processes</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">4215</span>– <span class="NLM_lpage">4231</span>, <span class="refDoi"> DOI: 10.1021/jm0303103</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0303103" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BD3sXmvFegur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2003&pages=4215-4231&author=V.+J.+Hruby&title=Peptide+Science%3A+Exploring+the+Use+of+Chemical+Principles+and+Interdisciplinary+Collaboration+for+Understanding+Life+Processes&doi=10.1021%2Fjm0303103"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Peptide Science: Exploring the Use of Chemical Principles and Interdisciplinary Collaboration for Understanding Life Processes</span></div><div class="casAuthors">Hruby, Victor J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">4215-4231</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  We applied concepts of conformational constraint to all of the peptide hormone and neurotransmitter ligands we were investigating, in addn. to our work on oxytocin and vasopressin design.  A major focus of our work to study was peptide conformation and design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoQnFyHQ6L68rVg90H21EOLACvtfcHk0lg5ADN6PEU9GQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXmvFegur0%253D&md5=18c8eff8f94b7fbe6b6b29abe57d18f0</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Fjm0303103&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0303103%26sid%3Dliteratum%253Aachs%26aulast%3DHruby%26aufirst%3DV.%2BJ.%26atitle%3DPeptide%2520Science%253A%2520Exploring%2520the%2520Use%2520of%2520Chemical%2520Principles%2520and%2520Interdisciplinary%2520Collaboration%2520for%2520Understanding%2520Life%2520Processes%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2003%26volume%3D46%26spage%3D4215%26epage%3D4231%26doi%3D10.1021%2Fjm0303103" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hruby, V. J.</span></span> <span> </span><span class="NLM_article-title">Designing Peptide Receptor Agonists and Antagonists</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">847</span>– <span class="NLM_lpage">858</span>, <span class="refDoi"> DOI: 10.1038/nrd939</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1038%2Fnrd939" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=12415245" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BD38XotleqtLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2002&pages=847-858&author=V.+J.+Hruby&title=Designing+Peptide+Receptor+Agonists+and+Antagonists&doi=10.1038%2Fnrd939"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Designing peptide receptor agonists and antagonists</span></div><div class="casAuthors">Hruby, Victor J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">847-858</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The most ubiquitous mode for controlling and modulating cellular function, intercellular communication, immune response and information-transduction pathways is through peptide-protein non-covalent interactions.  Hormones, neurotransmitters, antigens, cytokines and growth factors represent key classes of such peptide ligands.  These ligands might either be processed fragments of larger precursor proteins or surface segments of larger proteins.  Although there are numerous exceptions, such as insulin, oxytocin and calcitonin, most ligands are not used directly as drugs, and often the most useful ligands for therapy would be analogs that act as antagonists of the native ligands.  A search for systematic structure-based or ligand-based approaches to designing such ligands has been an important concern.  Today, a robust strategy has been developed for the design of peptides as drugs, drug candidates and biol. tools.  This strategy includes structural, conformational, dynamic and topog. considerations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6bGXGVM4TrrVg90H21EOLACvtfcHk0lg5ADN6PEU9GQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XotleqtLg%253D&md5=a21f2c38a42e51059b4060c3f7c8966b</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1038%2Fnrd939&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd939%26sid%3Dliteratum%253Aachs%26aulast%3DHruby%26aufirst%3DV.%2BJ.%26atitle%3DDesigning%2520Peptide%2520Receptor%2520Agonists%2520and%2520Antagonists%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2002%26volume%3D1%26spage%3D847%26epage%3D858%26doi%3D10.1038%2Fnrd939" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kessler, H.</span></span> <span> </span><span class="NLM_article-title">Conformation and Biological Activity of Cyclic Peptides</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">1982</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">512</span>– <span class="NLM_lpage">523</span>, <span class="refDoi"> DOI: 10.1002/anie.198205121</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1002%2Fanie.198205121" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=1982&pages=512-523&author=H.+Kessler&title=Conformation+and+Biological+Activity+of+Cyclic+Peptides&doi=10.1002%2Fanie.198205121"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1002%2Fanie.198205121&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.198205121%26sid%3Dliteratum%253Aachs%26aulast%3DKessler%26aufirst%3DH.%26atitle%3DConformation%2520and%2520Biological%2520Activity%2520of%2520Cyclic%2520Peptides%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D1982%26volume%3D21%26spage%3D512%26epage%3D523%26doi%3D10.1002%2Fanie.198205121" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Purkayastha, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, T. J.</span></span> <span> </span><span class="NLM_article-title">Stabilization of Proteins by Covalent Cyclization</span>. <i>Biotechnol. Bioprocess Eng.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">702</span>– <span class="NLM_lpage">712</span>, <span class="refDoi"> DOI: 10.1007/s12257-019-0363-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1007%2Fs12257-019-0363-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitV2ht7vJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2019&pages=702-712&author=A.+Purkayasthaauthor=T.+J.+Kang&title=Stabilization+of+Proteins+by+Covalent+Cyclization&doi=10.1007%2Fs12257-019-0363-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Stabilization of Proteins by Covalent Cyclization</span></div><div class="casAuthors">Purkayastha, Anwesha; Kang, Taek Jin</div><div class="citationInfo"><span class="NLM_cas:title">Biotechnology and Bioprocess Engineering</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">702-712</span>CODEN:
                <span class="NLM_cas:coden">BBEIAU</span>;
        ISSN:<span class="NLM_cas:issn">1226-8372</span>.
    
            (<span class="NLM_cas:orgname">Korean Society for Biotechnology and Bioengineering</span>)
        </div><div class="casAbstract">Stabilizing proteins against the thermal stress or proteolytic attacks is an important goal in many protein engineering studies.  As the simplest approach to the rational engineering of proteins, cyclizing proteins covalently has attracted a great deal of attention.  The idea of stabilizing the folded state of a protein by connecting its loose ends came from the polymer theory and was evidenced by many super-stable cyclic peptides/proteins present in nature.  Lab. methodologies utilizing various tools such as inteins, transpeptidases, transglutaminases and split cellular anchoring proteins have been developed, engineered, and successfully adopted to protein cyclization reactions.  Depending on the method used, N- and C-termini could be joined together to yield backbone cyclized proteins, or side chains located near the ends might be crosslinked to yield side chain cyclized ones.  As each of the methods has its own pros and cons in its reaction scheme, the outcomes such as an increase in melting temp. of a given protein are different when different methods are applied.  In this review, we highlight the stabilizing effects exerted by protein cyclization focusing on not a specific cyclizing method but product proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNzPdVh_t6b7Vg90H21EOLACvtfcHk0lg5ADN6PEU9GQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitV2ht7vJ&md5=0847e36cc193c1cbcfb83815a8045fb9</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1007%2Fs12257-019-0363-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs12257-019-0363-4%26sid%3Dliteratum%253Aachs%26aulast%3DPurkayastha%26aufirst%3DA.%26aulast%3DKang%26aufirst%3DT.%2BJ.%26atitle%3DStabilization%2520of%2520Proteins%2520by%2520Covalent%2520Cyclization%26jtitle%3DBiotechnol.%2520Bioprocess%2520Eng.%26date%3D2019%26volume%3D24%26spage%3D702%26epage%3D712%26doi%3D10.1007%2Fs12257-019-0363-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moellering, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilinski, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grossmann, T. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeh, J. T.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verdine, G. L.</span></span> <span> </span><span class="NLM_article-title">Towards Understanding Cell Penetration by Stapled Peptides</span>. <i>Med. Chem. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">111</span>– <span class="NLM_lpage">119</span>, <span class="refDoi"> DOI: 10.1039/C4MD00131A</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1039%2FC4MD00131A" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=111-119&author=Q.+Chuauthor=R.+E.+Moelleringauthor=G.+J.+Hilinskiauthor=Y.-W.+Kimauthor=T.+N.+Grossmannauthor=J.+T.-H.+Yehauthor=G.+L.+Verdine&title=Towards+Understanding+Cell+Penetration+by+Stapled+Peptides&doi=10.1039%2FC4MD00131A"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1039%2FC4MD00131A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC4MD00131A%26sid%3Dliteratum%253Aachs%26aulast%3DChu%26aufirst%3DQ.%26aulast%3DMoellering%26aufirst%3DR.%2BE.%26aulast%3DHilinski%26aufirst%3DG.%2BJ.%26aulast%3DKim%26aufirst%3DY.-W.%26aulast%3DGrossmann%26aufirst%3DT.%2BN.%26aulast%3DYeh%26aufirst%3DJ.%2BT.-H.%26aulast%3DVerdine%26aufirst%3DG.%2BL.%26atitle%3DTowards%2520Understanding%2520Cell%2520Penetration%2520by%2520Stapled%2520Peptides%26jtitle%3DMed.%2520Chem.%2520Commun.%26date%3D2015%26volume%3D6%26spage%3D111%26epage%3D119%26doi%3D10.1039%2FC4MD00131A" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dougherty, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sahni, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pei, D.</span></span> <span> </span><span class="NLM_article-title">Understanding Cell Penetration of Cyclic Peptides</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>119</i></span>,  <span class="NLM_fpage">10241</span>– <span class="NLM_lpage">10287</span>, <span class="refDoi"> DOI: 10.1021/acs.chemrev.9b00008</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.9b00008" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC1MXptlCksLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=2019&pages=10241-10287&author=P.+G.+Doughertyauthor=A.+Sahniauthor=D.+Pei&title=Understanding+Cell+Penetration+of+Cyclic+Peptides&doi=10.1021%2Facs.chemrev.9b00008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Understanding Cell Penetration of Cyclic Peptides</span></div><div class="casAuthors">Dougherty, Patrick G.; Sahni, Ashweta; Pei, Dehua</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">119</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">10241-10287</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Approx. 75% of all disease-relevant human proteins, including those involved in intracellular protein-protein interactions (PPIs), are undruggable with the current drug modalities (i.e., small mols. and biologics).  Macrocyclic peptides provide a potential soln. to these undruggable targets because their larger sizes (relative to conventional small mols.) endow them the capability of binding to flat PPI interfaces with antibody-like affinity and specificity.  Powerful combinatorial library technologies have been developed to routinely identify cyclic peptides as potent, specific inhibitors against proteins including PPI targets.  However, with the exception of a very small set of sequences, the vast majority of cyclic peptides are impermeable to the cell membrane, preventing their application against intracellular targets.  This Review examines common structural features that render most cyclic peptides membrane impermeable, as well as the unique features that allow the minority of sequences to enter the cell interior by passive diffusion, endocytosis/endosomal escape, or other mechanisms.  We also present the current state of knowledge about the mol. mechanisms of cell penetration, the various strategies for designing cell-permeable, biol. active cyclic peptides against intracellular targets, and the assay methods available to quantify their cell-permeability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp0NDA9U7-1YrVg90H21EOLACvtfcHk0lha0knClqiYxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXptlCksLs%253D&md5=65f68a9b69f3f7f33641b6a91835b1dd</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.9b00008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.9b00008%26sid%3Dliteratum%253Aachs%26aulast%3DDougherty%26aufirst%3DP.%2BG.%26aulast%3DSahni%26aufirst%3DA.%26aulast%3DPei%26aufirst%3DD.%26atitle%3DUnderstanding%2520Cell%2520Penetration%2520of%2520Cyclic%2520Peptides%26jtitle%3DChem.%2520Rev.%26date%3D2019%26volume%3D119%26spage%3D10241%26epage%3D10287%26doi%3D10.1021%2Facs.chemrev.9b00008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Drucker, D. J.</span></span> <span> </span><span class="NLM_article-title">Advances in Oral Peptide Therapeutics</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">277</span>– <span class="NLM_lpage">289</span>, <span class="refDoi"> DOI: 10.1038/s41573-019-0053-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1038%2Fs41573-019-0053-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=31848464" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2019&pages=277-289&author=D.+J.+Drucker&title=Advances+in+Oral+Peptide+Therapeutics&doi=10.1038%2Fs41573-019-0053-0"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1038%2Fs41573-019-0053-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41573-019-0053-0%26sid%3Dliteratum%253Aachs%26aulast%3DDrucker%26aufirst%3DD.%2BJ.%26atitle%3DAdvances%2520in%2520Oral%2520Peptide%2520Therapeutics%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2019%26volume%3D19%26spage%3D277%26epage%3D289%26doi%3D10.1038%2Fs41573-019-0053-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jing, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, K.</span></span> <span> </span><span class="NLM_article-title">A Gold Mine for Drug Discovery: Strategies to Develop Cyclic Peptides into Therapies</span>. <i>Med. Res. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">753</span>– <span class="NLM_lpage">810</span>, <span class="refDoi"> DOI: 10.1002/med.21639</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1002%2Fmed.21639" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=31599007" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2019&pages=753-810&author=X.+Jingauthor=K.+Jin&title=A+Gold+Mine+for+Drug+Discovery%3A+Strategies+to+Develop+Cyclic+Peptides+into+Therapies&doi=10.1002%2Fmed.21639"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1002%2Fmed.21639&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmed.21639%26sid%3Dliteratum%253Aachs%26aulast%3DJing%26aufirst%3DX.%26aulast%3DJin%26aufirst%3DK.%26atitle%3DA%2520Gold%2520Mine%2520for%2520Drug%2520Discovery%253A%2520Strategies%2520to%2520Develop%2520Cyclic%2520Peptides%2520into%2520Therapies%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2019%26volume%3D40%26spage%3D753%26epage%3D810%26doi%3D10.1002%2Fmed.21639" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nielsen, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shepherd, N. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucke, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stoermer, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fairlie, D. P.</span></span> <span> </span><span class="NLM_article-title">Orally Absorbed Cyclic Peptides</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>117</i></span>,  <span class="NLM_fpage">8094</span>– <span class="NLM_lpage">8128</span>, <span class="refDoi"> DOI: 10.1021/acs.chemrev.6b00838</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.6b00838" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC2sXot12lurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=117&publication_year=2017&pages=8094-8128&author=D.+S.+Nielsenauthor=N.+E.+Shepherdauthor=W.+Xuauthor=A.+J.+Luckeauthor=M.+J.+Stoermerauthor=D.+P.+Fairlie&title=Orally+Absorbed+Cyclic+Peptides&doi=10.1021%2Facs.chemrev.6b00838"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Orally Absorbed Cyclic Peptides</span></div><div class="casAuthors">Nielsen, Daniel S.; Shepherd, Nicholas E.; Xu, Weijun; Lucke, Andrew J.; Stoermer, Martin J.; Fairlie, David P.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">117</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">8094-8128</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Peptides and proteins are not orally bioavailable in mammals, although a few peptides are intestinally absorbed in small amts.  Polypeptides are generally too large and polar to passively diffuse through lipid membranes, while most known active transport mechanisms facilitate cell uptake of only very small peptides.  Systematic evaluations of peptides with mol. wts. above 500 Da are needed to identify parameters that influence oral bioavailability.  Here we describe 125 cyclic peptides contg. four to thirty-seven amino acids that are orally absorbed by mammals.  Cyclization minimizes degrdn. in the gut, blood and tissues by removing cleavable N- and C-termini and by shielding components from metabolic enzymes.  Cyclization also folds peptides into bioactive conformations that det. exposure of polar atoms to solvation by water and lipids, and therefore should influence oral bioavailability.  Key chem. properties thought to influence oral absorption and bioavailability are analyzed, including mol. wt., octanol-water partitioning, hydrogen bond donors/acceptors, rotatable bonds and polar surface area.  The cyclic peptides violated to different degrees all of the limits traditionally considered to be important for oral bioavailability of drug-like small mols., although fewer hydrogen bond donors and reduced flexibility generally favored oral absorption.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprHf1YVkf7ErVg90H21EOLACvtfcHk0lha0knClqiYxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXot12lurY%253D&md5=38af1945471e5ddffd3ae630ebbe129b</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.6b00838&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.6b00838%26sid%3Dliteratum%253Aachs%26aulast%3DNielsen%26aufirst%3DD.%2BS.%26aulast%3DShepherd%26aufirst%3DN.%2BE.%26aulast%3DXu%26aufirst%3DW.%26aulast%3DLucke%26aufirst%3DA.%2BJ.%26aulast%3DStoermer%26aufirst%3DM.%2BJ.%26aulast%3DFairlie%26aufirst%3DD.%2BP.%26atitle%3DOrally%2520Absorbed%2520Cyclic%2520Peptides%26jtitle%3DChem.%2520Rev.%26date%3D2017%26volume%3D117%26spage%3D8094%26epage%3D8128%26doi%3D10.1021%2Facs.chemrev.6b00838" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nielsen, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohman, R.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoang, H. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucke, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fairlie, D. P.</span></span> <span> </span><span class="NLM_article-title">Flexibility versus Rigidity for Orally Bioavailable Cyclic Hexapeptides</span>. <i>ChemBioChem</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">2289</span>– <span class="NLM_lpage">2293</span>, <span class="refDoi"> DOI: 10.1002/cbic.201500441</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1002%2Fcbic.201500441" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=26336864" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1Wjt73I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2015&pages=2289-2293&author=D.+S.+Nielsenauthor=R.-J.+Lohmanauthor=H.+N.+Hoangauthor=T.+A.+Hillauthor=A.+Jonesauthor=A.+J.+Luckeauthor=D.+P.+Fairlie&title=Flexibility+versus+Rigidity+for+Orally+Bioavailable+Cyclic+Hexapeptides&doi=10.1002%2Fcbic.201500441"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Flexibility versus Rigidity for Orally Bioavailable Cyclic Hexapeptides</span></div><div class="casAuthors">Nielsen, Daniel S.; Lohman, Rink-Jan; Hoang, Huy N.; Hill, Timothy A.; Jones, Alun; Lucke, Andrew J.; Fairlie, David P.</div><div class="citationInfo"><span class="NLM_cas:title">ChemBioChem</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">2289-2293</span>CODEN:
                <span class="NLM_cas:coden">CBCHFX</span>;
        ISSN:<span class="NLM_cas:issn">1439-4227</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Cyclic peptides and macrocycles have the potential to be membrane permeable and orally bioavailable, despite often not complying with the "rule of five" used in medicinal chem. to guide the discovery of oral drugs.  Here we compare solvent-dependent three-dimensional structures of three cyclic hexapeptides contg. D-amino acids, prolines, and intramol. hydrogen bonds.  Conformational rigidity rather than flexibility resulted in higher membrane permeability, metabolic stability and oral bioavailability, consistent with less polar surface exposure to solvent and a reduced entropy penalty for transition between polar and nonpolar environments.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpArFfvUbUBA7Vg90H21EOLACvtfcHk0lha0knClqiYxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1Wjt73I&md5=b0cf19a1ee875648d518dcc0befd9c28</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1002%2Fcbic.201500441&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbic.201500441%26sid%3Dliteratum%253Aachs%26aulast%3DNielsen%26aufirst%3DD.%2BS.%26aulast%3DLohman%26aufirst%3DR.-J.%26aulast%3DHoang%26aufirst%3DH.%2BN.%26aulast%3DHill%26aufirst%3DT.%2BA.%26aulast%3DJones%26aufirst%3DA.%26aulast%3DLucke%26aufirst%3DA.%2BJ.%26aulast%3DFairlie%26aufirst%3DD.%2BP.%26atitle%3DFlexibility%2520versus%2520Rigidity%2520for%2520Orally%2520Bioavailable%2520Cyclic%2520Hexapeptides%26jtitle%3DChemBioChem%26date%3D2015%26volume%3D16%26spage%3D2289%26epage%3D2293%26doi%3D10.1002%2Fcbic.201500441" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Räder, A. F. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinmüller, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reichart, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schumacher-Klinger, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merzbach, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilon, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffman, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kessler, H.</span></span> <span> </span><span class="NLM_article-title">Orally Active Peptides: Is There a Magic Bullet?</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">14414</span>– <span class="NLM_lpage">14438</span>, <span class="refDoi"> DOI: 10.1002/anie.201807298</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1002%2Fanie.201807298" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A280%3ADC%252BB3c3hslegsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2018&pages=14414-14438&author=A.+F.+B.+R%C3%A4derauthor=M.+Weinm%C3%BCllerauthor=F.+Reichartauthor=A.+Schumacher-Klingerauthor=S.+Merzbachauthor=C.+Gilonauthor=A.+Hoffmanauthor=H.+Kessler&title=Orally+Active+Peptides%3A+Is+There+a+Magic+Bullet%3F&doi=10.1002%2Fanie.201807298"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Orally Active Peptides: Is There a Magic Bullet?</span></div><div class="casAuthors">Rader Andreas F B; Weinmuller Michael; Reichart Florian; Kessler Horst; Schumacher-Klinger Adi; Merzbach Shira; Gilon Chaim; Hoffman Amnon</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie (International ed. in English)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">44</span>),
    <span class="NLM_cas:pages">14414-14438</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">For decades, the development of peptides as potential drugs was aimed solely at peptides with the highest affinity, receptor selectivity, or stability against enzymatic degradation.  However, optimization of their oral availability is highly desirable to establish orally active peptides as potential drug candidates for everyday use.  A twofold optimization process is necessary to produce orally active peptides: 1) optimization of the affinity and selectivity and 2) optimization of the oral availability.  These two steps must be performed sequentially for the rational design of orally active peptides.  Nevertheless, additional knowledge is required to understand which structural changes increase oral availability, followed by incorporation of these elements into a peptide without changing its other biological properties.  Considerable efforts have been made to understand the influence of these modifications on oral availability.  One approach is to improve the oral availability of a peptide that has been previously optimized for biological activity, as described in (1) above.  The second approach is to first identify an intestinally permeable, metabolically stable peptide scaffold and then introduce the functional groups necessary for the desired biological function.  Previous approaches to achieving peptide oral availability have been claimed to have general applicability but, thus far, most of these solutions have not been successful in other cases.  This Review discusses diverse chemical modifications, model peptides optimized for bioavailability, and orally active peptides to summarize the state of the research on the oral activity of peptides.  We explain why no simple and straightforward strategy (i.e. a "magic bullet") exists for the design of an orally active peptide with a druglike biological function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSwS0O4JvOgT--MdeMhXAHkfW6udTcc2eb2lg-oRBtS_Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3c3hslegsg%253D%253D&md5=d5d266bb2a6a26ec27d413ab3afdc311</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1002%2Fanie.201807298&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201807298%26sid%3Dliteratum%253Aachs%26aulast%3DR%25C3%25A4der%26aufirst%3DA.%2BF.%2BB.%26aulast%3DWeinm%25C3%25BCller%26aufirst%3DM.%26aulast%3DReichart%26aufirst%3DF.%26aulast%3DSchumacher-Klinger%26aufirst%3DA.%26aulast%3DMerzbach%26aufirst%3DS.%26aulast%3DGilon%26aufirst%3DC.%26aulast%3DHoffman%26aufirst%3DA.%26aulast%3DKessler%26aufirst%3DH.%26atitle%3DOrally%2520Active%2520Peptides%253A%2520Is%2520There%2520a%2520Magic%2520Bullet%253F%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2018%26volume%3D57%26spage%3D14414%26epage%3D14438%26doi%3D10.1002%2Fanie.201807298" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Villar, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beglov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chennamadhavuni, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porco, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozakov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vajda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitty, A.</span></span> <span> </span><span class="NLM_article-title">How Proteins Bind Macrocycles</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">723</span>– <span class="NLM_lpage">731</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1584</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1038%2Fnchembio.1584" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=25038790" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFygs7jF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=723-731&author=E.+A.+Villarauthor=D.+Beglovauthor=S.+Chennamadhavuniauthor=J.+A.+Porcoauthor=D.+Kozakovauthor=S.+Vajdaauthor=A.+Whitty&title=How+Proteins+Bind+Macrocycles&doi=10.1038%2Fnchembio.1584"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">How proteins bind macrocycles</span></div><div class="casAuthors">Villar, Elizabeth A.; Beglov, Dmitri; Chennamadhavuni, Spandan; Porco, John A. Jr; Kozakov, Dima; Vajda, Sandor; Whitty, Adrian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">723-731</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The potential utility of synthetic macrocycles (MCs) as drugs, particularly against low-druggability targets such as protein-protein interactions, has been widely discussed.  There is little information, however, to guide the design of MCs for good target protein-binding activity or bioavailability.  To address this knowledge gap, we analyze the binding modes of a representative set of MC-protein complexes.  The results, combined with consideration of the physicochem. properties of approved macrocyclic drugs, allow us to propose specific guidelines for the design of synthetic MC libraries with structural and physicochem. features likely to favor strong binding to protein targets as well as good bioavailability.  We addnl. provide evidence that large, natural product-derived MCs can bind targets that are not druggable by conventional, drug-like compds., supporting the notion that natural product-inspired synthetic MCs can expand the no. of proteins that are druggable by synthetic small mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqa3yuNCHU5d7Vg90H21EOLACvtfcHk0li71nf2XPwmHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFygs7jF&md5=0d41fba9267abb3a3d0065487937075a</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1584&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1584%26sid%3Dliteratum%253Aachs%26aulast%3DVillar%26aufirst%3DE.%2BA.%26aulast%3DBeglov%26aufirst%3DD.%26aulast%3DChennamadhavuni%26aufirst%3DS.%26aulast%3DPorco%26aufirst%3DJ.%2BA.%26aulast%3DKozakov%26aufirst%3DD.%26aulast%3DVajda%26aufirst%3DS.%26aulast%3DWhitty%26aufirst%3DA.%26atitle%3DHow%2520Proteins%2520Bind%2520Macrocycles%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2014%26volume%3D10%26spage%3D723%26epage%3D731%26doi%3D10.1038%2Fnchembio.1584" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vinogradov, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suga, H.</span></span> <span> </span><span class="NLM_article-title">Macrocyclic Peptides as Drug Candidates: Recent Progress and Remaining Challenges</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>141</i></span>,  <span class="NLM_fpage">4167</span>– <span class="NLM_lpage">4181</span>, <span class="refDoi"> DOI: 10.1021/jacs.8b13178</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.8b13178" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC1MXjtFCqtb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=141&publication_year=2019&pages=4167-4181&author=A.+A.+Vinogradovauthor=Y.+Yinauthor=H.+Suga&title=Macrocyclic+Peptides+as+Drug+Candidates%3A+Recent+Progress+and+Remaining+Challenges&doi=10.1021%2Fjacs.8b13178"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Macrocyclic Peptides as Drug Candidates: Recent Progress and Remaining Challenges</span></div><div class="casAuthors">Vinogradov, Alexander A.; Yin, Yizhen; Suga, Hiroaki</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">141</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4167-4181</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Peptides as a therapeutic modality attract much attention due to their synthetic accessibility, high degree of specific binding, and the ability to target protein surfaces traditionally considered "undruggable".  Unfortunately, at the same time, other pharmacol. properties of a generic peptide, such as metabolic stability and cell permeability, are quite poor, which limits the success of de novo discovered biol. active peptides as drug candidates.  Here, we review how macrocyclization as well as the incorporation of nonproteogenic amino acids and various conjugation strategies may be utilized to improve on these characteristics to create better drug candidates.  We analyze recent progress and remaining challenges in improving individual pharmacol. properties of bioactive peptides, and offer our opinion on interfacing these, often conflicting, considerations, to create balanced drug candidates as a potential way to make further progress in this area.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXRPkCAGk8-rVg90H21EOLACvtfcHk0lh31oabx6QwjA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXjtFCqtb4%253D&md5=59b6197874d4709a74089367a09119cc</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Fjacs.8b13178&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.8b13178%26sid%3Dliteratum%253Aachs%26aulast%3DVinogradov%26aufirst%3DA.%2BA.%26aulast%3DYin%26aufirst%3DY.%26aulast%3DSuga%26aufirst%3DH.%26atitle%3DMacrocyclic%2520Peptides%2520as%2520Drug%2520Candidates%253A%2520Recent%2520Progress%2520and%2520Remaining%2520Challenges%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2019%26volume%3D141%26spage%3D4167%26epage%3D4181%26doi%3D10.1021%2Fjacs.8b13178" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Driggers, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hale, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terrett, N. K.</span></span> <span> </span><span class="NLM_article-title">The Exploration of Macrocycles for Drug Discovery--an Underexploited Structural Class</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">608</span>– <span class="NLM_lpage">624</span>, <span class="refDoi"> DOI: 10.1038/nrd2590</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1038%2Fnrd2590" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=18591981" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BD1cXnvFShsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2008&pages=608-624&author=E.+M.+Driggersauthor=S.+P.+Haleauthor=J.+Leeauthor=N.+K.+Terrett&title=The+Exploration+of+Macrocycles+for+Drug+Discovery%2D%2Dan+Underexploited+Structural+Class&doi=10.1038%2Fnrd2590"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">The exploration of macrocycles for drug discovery - an underexploited structural class</span></div><div class="casAuthors">Driggers, Edward M.; Hale, Stephen P.; Lee, Jinbo; Terrett, Nicholas K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">608-624</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Natural products comprised of a macrocycle ring structure have proven their therapeutic applications as antibiotics, immunosuppressants as well as anticancer agents.  Despite this, macrocyclic compds. remain under-explored.  Terrett and colleagues review the properties and features of current macrocycle drugs, emphasizing the vast potential of synthetic macrocycles in drug discovery.  Macrocyclic natural products have evolved to fulfil numerous biochem. functions, and their profound pharmacol. properties have led to their development as drugs.  A macrocycle provides diverse functionality and stereochem. complexity in a conformationally pre-organized ring structure.  This can result in high affinity and selectivity for protein targets, while preserving sufficient bioavailability to reach intracellular locations.  Despite these valuable characteristics, and the proven success of more than 100 marketed macrocycle drugs derived from natural products, this structural class has been poorly explored within drug discovery.  This is in part due to concerns about synthetic intractability and non-drug-like properties.  This Review describes the growing body of data in favor of macrocyclic therapeutics, and demonstrates that this class of compds. can be both fully drug-like in its properties and readily prepd. owing to recent advances in synthetic medicinal chem.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDtW8LpE4ru7Vg90H21EOLACvtfcHk0lhT23b9b2O_ig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXnvFShsbc%253D&md5=16e091818053e5b163b81078a8d03094</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1038%2Fnrd2590&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2590%26sid%3Dliteratum%253Aachs%26aulast%3DDriggers%26aufirst%3DE.%2BM.%26aulast%3DHale%26aufirst%3DS.%2BP.%26aulast%3DLee%26aufirst%3DJ.%26aulast%3DTerrett%26aufirst%3DN.%2BK.%26atitle%3DThe%2520Exploration%2520of%2520Macrocycles%2520for%2520Drug%2520Discovery--an%2520Underexploited%2520Structural%2520Class%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2008%26volume%3D7%26spage%3D608%26epage%3D624%26doi%3D10.1038%2Fnrd2590" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mallinson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collins, I.</span></span> <span> </span><span class="NLM_article-title">Macrocycles in New Drug Discovery</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">1409</span>– <span class="NLM_lpage">1438</span>, <span class="refDoi"> DOI: 10.4155/fmc.12.93</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.4155%2Ffmc.12.93" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=22857532" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtFCht7rO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2012&pages=1409-1438&author=J.+Mallinsonauthor=I.+Collins&title=Macrocycles+in+New+Drug+Discovery&doi=10.4155%2Ffmc.12.93"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Macrocycles in new drug discovery</span></div><div class="casAuthors">Mallinson, Jamie; Collins, Ian</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1409-1438</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">A review.  The use of drug-like macrocycles is emerging as an exciting area of medicinal chem., with several recent examples highlighting the favorable changes in biol. and physicochem. properties that macrocyclization can afford.  Natural product macrocycles and their synthetic derivs. have long been clin. useful and attention is now being focused on the wider use of macrocyclic scaffolds in medicinal chem. in the search for new drugs for increasingly challenging targets.  With the increasing awareness of concepts of drug-likeness and the dangers of mol. obesity', functionalized macrocyclic scaffolds could provide a way to generate ligand-efficient mols. with enhanced properties.  In this review we will sep. discuss the effects of macrocyclization upon potency, selectivity and physicochem. properties, concg. on recent case histories in oncol. drug discovery.  Addnl., we will highlight selected advances in the synthesis of macrocycles and provide an outlook on the future use of macrocyclic scaffolds in medicinal chem.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqSrzcbKW7Tc7Vg90H21EOLACvtfcHk0lhT23b9b2O_ig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtFCht7rO&md5=63ad275af92ac89ae6b36c5071b578f8</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.4155%2Ffmc.12.93&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.12.93%26sid%3Dliteratum%253Aachs%26aulast%3DMallinson%26aufirst%3DJ.%26aulast%3DCollins%26aufirst%3DI.%26atitle%3DMacrocycles%2520in%2520New%2520Drug%2520Discovery%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2012%26volume%3D4%26spage%3D1409%26epage%3D1438%26doi%3D10.4155%2Ffmc.12.93" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Whitty, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Viarengo, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, M.</span></span> <span> </span><span class="NLM_article-title">Progress towards the Broad Use of Non-Peptide Synthetic Macrocycles in Drug Discovery</span>. <i>Org. Biomol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">7729</span>– <span class="NLM_lpage">7735</span>, <span class="refDoi"> DOI: 10.1039/C7OB00056A</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1039%2FC7OB00056A" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=28876025" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsVKgsLrO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2017&pages=7729-7735&author=A.+Whittyauthor=L.+A.+Viarengoauthor=M.+Zhong&title=Progress+towards+the+Broad+Use+of+Non-Peptide+Synthetic+Macrocycles+in+Drug+Discovery&doi=10.1039%2FC7OB00056A"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Progress towards the broad use of non-peptide synthetic macrocycles in drug discovery</span></div><div class="casAuthors">Whitty, Adrian; Viarengo, Lauren A.; Zhong, Mengqi</div><div class="citationInfo"><span class="NLM_cas:title">Organic & Biomolecular Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">37</span>),
    <span class="NLM_cas:pages">7729-7735</span>CODEN:
                <span class="NLM_cas:coden">OBCRAK</span>;
        ISSN:<span class="NLM_cas:issn">1477-0520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  The authors discuss progress towards addressing three key questions pertaining to the design of screening libraries of synthetic nonpeptidic macrocycles (MCs) for drug discovery: What structural and physicochem. properties of MCs maximize the likelihood of achieving strong and specific binding to protein targets.  What features render a protein target suitable for binding MCs, and can this information be used to identify suitable targets for inhibition by MCs.  What properties of synthetic MCs confer good pharmaceutical properties, and particularly good aq. soly. coupled with passive membrane permeability.  The authors addnl. discuss how the criteria that define a meaningful MC screening hit are linked to the size of the screening library and the synthetic methodol. employed in its prepn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpeNdUQVdF6FrVg90H21EOLACvtfcHk0lhT23b9b2O_ig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsVKgsLrO&md5=451ddddc4457d5a2ee84b384aa15508e</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1039%2FC7OB00056A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC7OB00056A%26sid%3Dliteratum%253Aachs%26aulast%3DWhitty%26aufirst%3DA.%26aulast%3DViarengo%26aufirst%3DL.%2BA.%26aulast%3DZhong%26aufirst%3DM.%26atitle%3DProgress%2520towards%2520the%2520Broad%2520Use%2520of%2520Non-Peptide%2520Synthetic%2520Macrocycles%2520in%2520Drug%2520Discovery%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2017%26volume%3D15%26spage%3D7729%26epage%3D7735%26doi%3D10.1039%2FC7OB00056A" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dharanipragada, R.</span></span> <span> </span><span class="NLM_article-title">New Modalities in Conformationally Constrained Peptides for Potency, Selectivity and Cell Permeation</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">831</span>– <span class="NLM_lpage">849</span>, <span class="refDoi"> DOI: 10.4155/fmc.13.25</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.4155%2Ffmc.13.25" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=23651095" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC3sXntFyrsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2013&pages=831-849&author=R.+Dharanipragada&title=New+Modalities+in+Conformationally+Constrained+Peptides+for+Potency%2C+Selectivity+and+Cell+Permeation&doi=10.4155%2Ffmc.13.25"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">New modalities in conformationally constrained peptides for potency, selectivity and cell permeation</span></div><div class="casAuthors">Dharanipragada, Ram</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">831-849</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">A review.  There has been a resurgence of interest in peptide pharmaceuticals as they have an advantage of potency, selectivity and less toxicity compared with small-mol. therapeutics.  The main draw back of peptides is lack of stability to biol. media.  Constraining a peptide has been one of the approaches to improving in vivo stability of the peptides.  Several new modalities in constraining peptides have been developed over recent years and this review highlights some of the new developments.  The newer cyclization strategies have rendered, in some cases, oral activity, cell permeability, improved potency at the target receptor, selectivity against receptor subtypes and improved stability to enzymes.  As chemists further understand the rules governing cell permeability, oral absorption and enhancing stability of peptides, we can expect to see more peptides entering clinic for many unmet medical needs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoULvQN8YpaBbVg90H21EOLACvtfcHk0lhT23b9b2O_ig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXntFyrsrk%253D&md5=829f070989c1e8e9e729595ab92d30b5</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.4155%2Ffmc.13.25&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.13.25%26sid%3Dliteratum%253Aachs%26aulast%3DDharanipragada%26aufirst%3DR.%26atitle%3DNew%2520Modalities%2520in%2520Conformationally%2520Constrained%2520Peptides%2520for%2520Potency%252C%2520Selectivity%2520and%2520Cell%2520Permeation%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2013%26volume%3D5%26spage%3D831%26epage%3D849%26doi%3D10.4155%2Ffmc.13.25" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lau, Y. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Andrade, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spring, D. R.</span></span> <span> </span><span class="NLM_article-title">Peptide Stapling Techniques Based on Different Macrocyclisation Chemistries</span>. <i>Chem. Soc. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">91</span>– <span class="NLM_lpage">102</span>, <span class="refDoi"> DOI: 10.1039/C4CS00246F</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1039%2FC4CS00246F" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=25199043" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsV2mtb%252FL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2015&pages=91-102&author=Y.+H.+Lauauthor=P.+de+Andradeauthor=Y.+Wuauthor=D.+R.+Spring&title=Peptide+Stapling+Techniques+Based+on+Different+Macrocyclisation+Chemistries&doi=10.1039%2FC4CS00246F"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Peptide stapling techniques based on different macrocyclization chemistries</span></div><div class="casAuthors">Lau, Yu Heng; de Andrade, Peterson; Wu, Yuteng; Spring, David R.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Society Reviews</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">91-102</span>CODEN:
                <span class="NLM_cas:coden">CSRVBR</span>;
        ISSN:<span class="NLM_cas:issn">0306-0012</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  Peptide stapling is a strategy for constraining short peptides typically in an alpha-helical conformation.  Stapling is carried out by covalently linking the side-chains of two amino acids, thereby forming a peptide macrocycle.  There is an expanding repertoire of stapling techniques based on different macrocyclization chemistries.  In this tutorial review, the authors categorize and analyze key examples of peptide stapling in terms of their synthesis and applicability to biol. systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkqQ6yGsikIbVg90H21EOLACvtfcHk0liyA7AijzZlWA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsV2mtb%252FL&md5=0efb31241f9809b61a81474131a54de3</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1039%2FC4CS00246F&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC4CS00246F%26sid%3Dliteratum%253Aachs%26aulast%3DLau%26aufirst%3DY.%2BH.%26aulast%3Dde%2BAndrade%26aufirst%3DP.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DSpring%26aufirst%3DD.%2BR.%26atitle%3DPeptide%2520Stapling%2520Techniques%2520Based%2520on%2520Different%2520Macrocyclisation%2520Chemistries%26jtitle%3DChem.%2520Soc.%2520Rev.%26date%3D2015%26volume%3D44%26spage%3D91%26epage%3D102%26doi%3D10.1039%2FC4CS00246F" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ngambenjawong, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pineda, J. M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pun, S. H.</span></span> <span> </span><span class="NLM_article-title">Engineering an Affinity-Enhanced Peptide through Optimization of Cyclization Chemistry</span>. <i>Bioconjugate Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">2854</span>– <span class="NLM_lpage">2862</span>, <span class="refDoi"> DOI: 10.1021/acs.bioconjchem.6b00502</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.bioconjchem.6b00502" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhslamt7fO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2016&pages=2854-2862&author=C.+Ngambenjawongauthor=J.+M.+B.+Pinedaauthor=S.+H.+Pun&title=Engineering+an+Affinity-Enhanced+Peptide+through+Optimization+of+Cyclization+Chemistry&doi=10.1021%2Facs.bioconjchem.6b00502"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Engineering an Affinity-Enhanced Peptide through Optimization of Cyclization Chemistry</span></div><div class="casAuthors">Ngambenjawong, Chayanon; Pineda, Julio Marco B.; Pun, Suzie H.</div><div class="citationInfo"><span class="NLM_cas:title">Bioconjugate Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2854-2862</span>CODEN:
                <span class="NLM_cas:coden">BCCHES</span>;
        ISSN:<span class="NLM_cas:issn">1043-1802</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Peptide cyclization is a strategy used to improve stability and/or activity of peptides.  The most commonly used cyclization method is disulfide bridge formation of cysteine-contg. peptides as typically found in nature.  Over the years, an increasing no. of alternative chemistries for peptide cyclization with improved efficiency, kinetics, orthogonality, and stability have been reported.  However, there has been less appreciation for the opportunity to fine-tune peptide activity via the diverse chem. entities introduced at the site of linkage by different cyclization strategies.  Here, the authors demonstrate how cyclization optimization of an M2 "anti-inflammatory" macrophage-binding peptide (M2pep) resulted in a significant increase in binding affinity of the optimized analog to M2 macrophages while maintaining binding selectivity compared to M1 "pro-inflammatory" macrophages.  In this study, the authors report synthesis and evaluation of four cyclic M2pep(RY) analogs with diverse cyclization strategies; (1) Asp-[Amide]-Lys, (2) Azido-Lys-[Triazole (Copper(I)-catalyzed alkyne-azide cycloaddn. (CuAAC))]-Propargyl-Gly, (3) Cys-[Decafluorobiphenyl (DFBP)]-Cys, and (4) Cys-[Decafluorobiphenyl sulfone (DFS)]-Cys, whereby the chem. entity/linker at the linkage site is shown in the square bracket and is between the residues involved in cyclization.  These peptides are compared to a disulfide-cyclized M2pep(RY) that the authors previously reported as a serum-stable, affinity enhanced analog to the original linear M2pep.  DFBP-cyclized M2pep(RY) exhibits the highest binding activity to M2 macrophages with apparent dissocn. const. (KD) about 2.03 μM compared to 36.3 μM for the original disulfide-cyclized M2pep(RY) and 220 μM for the original linear peptide.  DFS-cyclized M2pep(RY) also binds more strongly than the original cyclized analog whereas Amide- and Triazole-cyclized M2pep(RY) analogs bind less strongly.  The authors verified that DFBP alone has negligible binding to M2 macrophages and the incorporation of diphenylalanine to the original sequence improves binding activity at the expense of soly. and increased toxicity.  In conclusion, the authors report development of cyclic M2pep(RY) analogs with diverse cyclization strategies leading to the discovery of DFBP-cyclized M2pep(RY) with super-enhanced M2 macrophage-binding activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoY-UeaKwT_xLVg90H21EOLACvtfcHk0liyA7AijzZlWA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhslamt7fO&md5=f0ed679dcb06b4bc8a29fe4c35a9500a</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Facs.bioconjchem.6b00502&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.bioconjchem.6b00502%26sid%3Dliteratum%253Aachs%26aulast%3DNgambenjawong%26aufirst%3DC.%26aulast%3DPineda%26aufirst%3DJ.%2BM.%2BB.%26aulast%3DPun%26aufirst%3DS.%2BH.%26atitle%3DEngineering%2520an%2520Affinity-Enhanced%2520Peptide%2520through%2520Optimization%2520of%2520Cyclization%2520Chemistry%26jtitle%3DBioconjugate%2520Chem.%26date%3D2016%26volume%3D27%26spage%3D2854%26epage%3D2862%26doi%3D10.1021%2Facs.bioconjchem.6b00502" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Skowron, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Speltz, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moore, T. W.</span></span> <span> </span><span class="NLM_article-title">Recent Structural Advances in Constrained Helical Peptides</span>. <i>Med. Res. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">749</span>– <span class="NLM_lpage">770</span>, <span class="refDoi"> DOI: 10.1002/med.21540</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1002%2Fmed.21540" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=30307621" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A280%3ADC%252BB3cznvF2itQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2019&pages=749-770&author=K.+J.+Skowronauthor=T.+E.+Speltzauthor=T.+W.+Moore&title=Recent+Structural+Advances+in+Constrained+Helical+Peptides&doi=10.1002%2Fmed.21540"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Recent structural advances in constrained helical peptides</span></div><div class="casAuthors">Skowron Kornelia J; Speltz Thomas E; Moore Terry W; Moore Terry W</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal research reviews</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">749-770</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Given the ubiquity of the  -helix in the proteome, there has been much research in developing mimics of  -helices, and most of this study has been toward developing protein-protein interaction inhibitors.  A common strategy for mimicking  -helices has been through the use of constrained, helical peptides.  The addition of a constraint typically provides for conformational and proteolytic stability and, in some cases, cell permeability.  Some of the most well-known strategies included are lactam formation and hydrocarbon "stapling." Beyond those strategies, there have been many recent advances in developing constrained peptides.  The purpose of this review is to highlight recent advances in the development of new helix-stabilizing technologies, constraint diversification strategies, tether diversification strategies, and combination strategies that create new bicyclic helical peptides.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSdKbhDWiU7TdLIORRPd5lbfW6udTcc2eY1MyqbQVjiorntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3cznvF2itQ%253D%253D&md5=9277ab4a2572b4d0d2b500e82c267936</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1002%2Fmed.21540&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmed.21540%26sid%3Dliteratum%253Aachs%26aulast%3DSkowron%26aufirst%3DK.%2BJ.%26aulast%3DSpeltz%26aufirst%3DT.%2BE.%26aulast%3DMoore%26aufirst%3DT.%2BW.%26atitle%3DRecent%2520Structural%2520Advances%2520in%2520Constrained%2520Helical%2520Peptides%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2019%26volume%3D39%26spage%3D749%26epage%3D770%26doi%3D10.1002%2Fmed.21540" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheng, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span> <span> </span><span class="NLM_article-title">Synthesis of Bioactive and Stabilized Cyclic Peptides by Macrocyclization Using C(Sp3)-H Activation</span>. <i>Chem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">4565</span>– <span class="NLM_lpage">4570</span>, <span class="refDoi"> DOI: 10.1039/C6SC05530C</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1039%2FC6SC05530C" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=28936334" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmt1ektbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=4565-4570&author=J.+Tangauthor=Y.+Heauthor=H.+Chenauthor=W.+Shengauthor=H.+Wang&title=Synthesis+of+Bioactive+and+Stabilized+Cyclic+Peptides+by+Macrocyclization+Using+C%28Sp3%29-H+Activation&doi=10.1039%2FC6SC05530C"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis of bioactive and stabilized cyclic peptides by macrocyclization using C(sp3)-H activation</span></div><div class="casAuthors">Tang, Jian; He, Yadong; Chen, Hongfei; Sheng, Wangjian; Wang, Huan</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Science</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">4565-4570</span>CODEN:
                <span class="NLM_cas:coden">CSHCCN</span>;
        ISSN:<span class="NLM_cas:issn">2041-6520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Cyclic peptides have attracted increasing attention in recent years due to their ability to inhibit protein-protein interactions.  Current strategies to prep. cyclic peptides often rely on functional amino acid side chains or the incorporation of unnatural amino acids, thus limiting their structural diversity.  Here, we describe the development of a highly versatile peptide macrocyclization strategy through a palladium-catalyzed C(sp3)-H activation and the synthesis of cyclic peptides featuring unique hydrocarbon linkages between the β-carbon of amino acids and the arom. side chains of Phe and Trp.  We demonstrate that such peptides exhibit improved biol. properties compared to their acyclic counterparts.  Finally, we applied this method in the synthesis of the natural product celogentin C.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqE3dkZ92oXRbVg90H21EOLACvtfcHk0liyA7AijzZlWA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmt1ektbk%253D&md5=47dfb53e3dff4be724b0480c248e280c</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1039%2FC6SC05530C&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC6SC05530C%26sid%3Dliteratum%253Aachs%26aulast%3DTang%26aufirst%3DJ.%26aulast%3DHe%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DSheng%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DH.%26atitle%3DSynthesis%2520of%2520Bioactive%2520and%2520Stabilized%2520Cyclic%2520Peptides%2520by%2520Macrocyclization%2520Using%2520C%2528Sp3%2529-H%2520Activation%26jtitle%3DChem.%2520Sci.%26date%3D2017%26volume%3D8%26spage%3D4565%26epage%3D4570%26doi%3D10.1039%2FC6SC05530C" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">White, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yudin, A. K.</span></span> <span> </span><span class="NLM_article-title">Contemporary Strategies for Peptide Macrocyclization</span>. <i>Nat. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">509</span>– <span class="NLM_lpage">524</span>, <span class="refDoi"> DOI: 10.1038/nchem.1062</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1038%2Fnchem.1062" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=21697871" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC3MXnvFWjtrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2011&pages=509-524&author=C.+J.+Whiteauthor=A.+K.+Yudin&title=Contemporary+Strategies+for+Peptide+Macrocyclization&doi=10.1038%2Fnchem.1062"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Contemporary strategies for peptide macrocyclization</span></div><div class="casAuthors">White, Christopher J.; Yudin, Andrei K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">509-524</span>CODEN:
                <span class="NLM_cas:coden">NCAHBB</span>;
        ISSN:<span class="NLM_cas:issn">1755-4330</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Peptide macrocycles have found applications that range from drug discovery to nanomaterials.  These ring-shaped mols. have shown remarkable capacity for functional fine-tuning.  Such capacity is enabled by the possibility of adjusting the peptide conformation using the techniques of chem. synthesis.  Cyclic peptides have been difficult, and often impossible, to prep. using traditional synthetic methods.  For macrocyclization to occur, the activated peptide must adopt an entropically disfavored pre-cyclization conformation before forming the desired product.  Here, recent solns. to some of the major challenges in this important area of contemporary synthesis were reviewed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqUTda6OkfQprVg90H21EOLACvtfcHk0ljqdn49i5Vhfw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXnvFWjtrc%253D&md5=32dfd94692ca20901dc8828bcd27ae8c</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1038%2Fnchem.1062&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchem.1062%26sid%3Dliteratum%253Aachs%26aulast%3DWhite%26aufirst%3DC.%2BJ.%26aulast%3DYudin%26aufirst%3DA.%2BK.%26atitle%3DContemporary%2520Strategies%2520for%2520Peptide%2520Macrocyclization%26jtitle%3DNat.%2520Chem.%26date%3D2011%26volume%3D3%26spage%3D509%26epage%3D524%26doi%3D10.1038%2Fnchem.1062" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Felix, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heimer, E. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lambros, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fournier, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mowles, T. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maines, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wegrzynski, B. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toome, V.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Synthesis, Biological Activity and Conformational Analysis of Cyclic GRF Analogs</span>. <i>Int. J. Pept. Protein Res.</i> <span class="NLM_year" style="font-weight: bold;">1988</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">441</span>– <span class="NLM_lpage">454</span>, <span class="refDoi"> DOI: 10.1111/j.1399-3011.1988.tb01375.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1111%2Fj.1399-3011.1988.tb01375.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=3149952" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADyaL1MXmtlemtLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=1988&pages=441-454&author=A.+M.+Felixauthor=E.+P.+Heimerauthor=C.+T.+Wangauthor=T.+J.+Lambrosauthor=A.+Fournierauthor=T.+F.+Mowlesauthor=S.+Mainesauthor=R.+M.+Campbellauthor=B.+B.+Wegrzynskiauthor=V.+Toome&title=Synthesis%2C+Biological+Activity+and+Conformational+Analysis+of+Cyclic+GRF+Analogs&doi=10.1111%2Fj.1399-3011.1988.tb01375.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis, biological activity and conformational analysis of cyclic GRF analogs</span></div><div class="casAuthors">Felix, Arthur M.; Heimer, Edgar P.; Wang, Ching Tso; Lambros, Theodore J.; Fournier, Alain; Mowles, Thomas F.; Maines, Sarah; Campbell, Robert M.; Wegrzynski, Bogda B.; et al.</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Peptide & Protein Research</span>
        (<span class="NLM_cas:date">1988</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">441-54</span>CODEN:
                <span class="NLM_cas:coden">IJPPC3</span>;
        ISSN:<span class="NLM_cas:issn">0367-8377</span>.
    </div><div class="casAbstract">A novel cyclic growth hormone-releasing factor (GRF) analog, cyclo(Asp8-Lys12)-[Asp8,Ala15]-GRF(1-29)-NH2 (I), was synthesized by the solid-phase procedure and found to retain significant biol. activity.  Solid-phase cyclization of Asp8 to Lys12 proceeded rapidly (∼2 h) using the BOP reagent.  Substitution of Ala2 with D-Ala2 and/or NH2-terminal replacement (desNH2-Tyr1 or N-MeTyr1) in I resulted in highly potent analogs that were also active in vivo.  Conformational anal. (CD and mol. dynamics calcns. based on NOE-derived distance constraints) demonstrated that I contains a long α-helical segment even in aq. soln.  A series of cyclo8,12 stereoisomers contg. D-Asp2 and/or D-Lys12 were prepd. and also found to be highly potent and to retain significant α-helical conformation.  The high biol. activity of cyclo8,12[N-MeTyr1,D-Ala2,Asp8,Ala15]-GRF(1-29)-NH2 may be explained on the basis of retention of a preferred bioactive conformation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrDIM5X6XSbp7Vg90H21EOLACvtfcHk0ljqdn49i5Vhfw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1MXmtlemtLg%253D&md5=a9471ff69ff82b7250e51992b4e0d409</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1111%2Fj.1399-3011.1988.tb01375.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1399-3011.1988.tb01375.x%26sid%3Dliteratum%253Aachs%26aulast%3DFelix%26aufirst%3DA.%2BM.%26aulast%3DHeimer%26aufirst%3DE.%2BP.%26aulast%3DWang%26aufirst%3DC.%2BT.%26aulast%3DLambros%26aufirst%3DT.%2BJ.%26aulast%3DFournier%26aufirst%3DA.%26aulast%3DMowles%26aufirst%3DT.%2BF.%26aulast%3DMaines%26aufirst%3DS.%26aulast%3DCampbell%26aufirst%3DR.%2BM.%26aulast%3DWegrzynski%26aufirst%3DB.%2BB.%26aulast%3DToome%26aufirst%3DV.%26atitle%3DSynthesis%252C%2520Biological%2520Activity%2520and%2520Conformational%2520Analysis%2520of%2520Cyclic%2520GRF%2520Analogs%26jtitle%3DInt.%2520J.%2520Pept.%2520Protein%2520Res.%26date%3D1988%26volume%3D32%26spage%3D441%26epage%3D454%26doi%3D10.1111%2Fj.1399-3011.1988.tb01375.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, J. W.</span></span> <span> </span><span class="NLM_article-title">The Synthesis and Study of Side-Chain Lactam-Bridged Peptides</span>. <i>Biopolymers</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">49</span>– <span class="NLM_lpage">75</span>, <span class="refDoi"> DOI: 10.1002/bip.10203</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1002%2Fbip.10203" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=12228920" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BD38Xnsl2rsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2002&pages=49-75&author=J.+W.+Taylor&title=The+Synthesis+and+Study+of+Side-Chain+Lactam-Bridged+Peptides&doi=10.1002%2Fbip.10203"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">The synthesis and study of side-chain lactam-bridged peptides</span></div><div class="casAuthors">Taylor, John W.</div><div class="citationInfo"><span class="NLM_cas:title">Biopolymers</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">49-75</span>CODEN:
                <span class="NLM_cas:coden">BIPMAA</span>;
        ISSN:<span class="NLM_cas:issn">0006-3525</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A review.  This article reviews some of the synthetic approaches that have been used to prep. peptides with side-chain lactam bridges, conformational effects of these bridges and their applications to studies of bioactive peptides.  For example, some lactam-bridged peptides are designed to have increased receptor binding affinity.  Peptides with side-chain lactam bridges may also serve as models for protein folding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqMgDLaY7mzdrVg90H21EOLACvtfcHk0ljqdn49i5Vhfw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xnsl2rsb4%253D&md5=9f0e8da81c533ac03a9f1f1bfca7200b</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1002%2Fbip.10203&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fbip.10203%26sid%3Dliteratum%253Aachs%26aulast%3DTaylor%26aufirst%3DJ.%2BW.%26atitle%3DThe%2520Synthesis%2520and%2520Study%2520of%2520Side-Chain%2520Lactam-Bridged%2520Peptides%26jtitle%3DBiopolymers%26date%3D2002%26volume%3D66%26spage%3D49%26epage%3D75%26doi%3D10.1002%2Fbip.10203" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Blackwell, H. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grubbs, R. H.</span></span> <span> </span><span class="NLM_article-title">Highly Efficient Synthesis of Covalently Cross-Linked Peptide Helices by Ring-Closing Metathesis</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>37</i></span>,  <span class="NLM_fpage">3281</span>– <span class="NLM_lpage">3284</span>, <span class="refDoi"> DOI: 10.1002/(SICI)1521-3773(19981217)37:23<3281::AID-ANIE3281>3.0.CO;2-V</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1002%2F%28SICI%291521-3773%2819981217%2937%3A23%3C3281%3A%3AAID-ANIE3281%3E3.0.CO%3B2-V" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADyaK1MXksVehtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=1998&pages=3281-3284&author=H.+E.+Blackwellauthor=R.+H.+Grubbs&title=Highly+Efficient+Synthesis+of+Covalently+Cross-Linked+Peptide+Helices+by+Ring-Closing+Metathesis&doi=10.1002%2F%28SICI%291521-3773%2819981217%2937%3A23%3C3281%3A%3AAID-ANIE3281%3E3.0.CO%3B2-V"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Highly efficient synthesis of covalently cross-linked peptide helices by ring-closing metathesis</span></div><div class="casAuthors">Blackwell, Helen E.; Grubbs, Robert H.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">3281-3284</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH</span>)
        </div><div class="casAbstract">Ring closing metathesis of acyclic peptide dienes Boc-Val-X-Leu-Aib-Val-X-Leu-OMe [I; Boc = Me3CO2C; X = Ser(CH2CH:CH2), Hse(CH2CH:CH2); Aib = NHCMe2CO], prepd. by std. solid-phase peptide chem., using olefin metathesis catalyst (PCy3)2Cl2Ru:CHPh gave 21- and 23-membered macrocyclic alkenes I in 85 and 90% yields, resp.  Hydrogenation of II gave the corresponding alkane-bridged cyclopeptides III in 98% yield.  Significant conformational change did not occur on cyclization of I to III, as evidenced by far-UV CD.  A crystal structure of III (X = Hse) is reported.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqFCptOTOzERbVg90H21EOLACvtfcHk0ljqdn49i5Vhfw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXksVehtA%253D%253D&md5=db2cc48a5a345762d90dc83f8287e67a</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291521-3773%2819981217%2937%3A23%3C3281%3A%3AAID-ANIE3281%3E3.0.CO%3B2-V&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291521-3773%252819981217%252937%253A23%253C3281%253A%253AAID-ANIE3281%253E3.0.CO%253B2-V%26sid%3Dliteratum%253Aachs%26aulast%3DBlackwell%26aufirst%3DH.%2BE.%26aulast%3DGrubbs%26aufirst%3DR.%2BH.%26atitle%3DHighly%2520Efficient%2520Synthesis%2520of%2520Covalently%2520Cross-Linked%2520Peptide%2520Helices%2520by%2520Ring-Closing%2520Metathesis%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D1998%26volume%3D37%26spage%3D3281%26epage%3D3284%26doi%3D10.1002%2F%28SICI%291521-3773%2819981217%2937%3A23%3C3281%3A%3AAID-ANIE3281%3E3.0.CO%3B2-V" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grossmann, T. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verdine, G. L.</span></span> <span> </span><span class="NLM_article-title">Synthesis of All-Hydrocarbon Stapled α-Helical Peptides by Ring-Closing Olefin Metathesis</span>. <i>Nat. Protoc.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">761</span>– <span class="NLM_lpage">771</span>, <span class="refDoi"> DOI: 10.1038/nprot.2011.324</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1038%2Fnprot.2011.324" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=21637196" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmvFyqurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&pages=761-771&author=Y.-W.+Kimauthor=T.+N.+Grossmannauthor=G.+L.+Verdine&title=Synthesis+of+All-Hydrocarbon+Stapled+%CE%B1-Helical+Peptides+by+Ring-Closing+Olefin+Metathesis&doi=10.1038%2Fnprot.2011.324"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis of all-hydrocarbon stapled α-helical peptides by ring-closing olefin metathesis</span></div><div class="casAuthors">Kim, Young-Woo; Grossmann, Tom N.; Verdine, Gregory L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">761-771</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">This protocol provides a detailed procedure for the prepn. of stapled α-helical peptides, which have proven their potential as useful mol. probes and as next-generation therapeutics.  Two crucial features of this protocol are the construction of peptide substrates contg. hindered α-Me, α-alkenyl amino acids and the ring-closing olefin metathesis (RCM) of the resulting resin-bound peptide substrates.  The stapling systems described in this protocol, namely bridging one or two turns of an α-helix, are highly adaptable to most peptide sequences, resulting in favorable RCM kinetics, helix stabilization and promotion of cellular uptake.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr2DCeMO4NgHLVg90H21EOLACvtfcHk0lgfr1RCZWFxOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmvFyqurg%253D&md5=02d269995eb1488632382c878df9fde4</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2011.324&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2011.324%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DY.-W.%26aulast%3DGrossmann%26aufirst%3DT.%2BN.%26aulast%3DVerdine%26aufirst%3DG.%2BL.%26atitle%3DSynthesis%2520of%2520All-Hydrocarbon%2520Stapled%2520%25CE%25B1-Helical%2520Peptides%2520by%2520Ring-Closing%2520Olefin%2520Metathesis%26jtitle%3DNat.%2520Protoc.%26date%3D2011%26volume%3D6%26spage%3D761%26epage%3D771%26doi%3D10.1038%2Fnprot.2011.324" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Verdine, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilinski, G. J.</span></span> <span> </span><span class="NLM_article-title">All-Hydrocarbon Stapled Peptides as Synthetic Cell-Accessible Mini-Proteins</span>. <i>Drug Discovery Today Technol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">e41</span>– <span class="NLM_lpage">e47</span>, <span class="refDoi"> DOI: 10.1016/j.ddtec.2012.01.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1016%2Fj.ddtec.2012.01.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=23264802" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC38XlvFaqsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2012&pages=e41-e47&author=G.+L.+Verdineauthor=G.+J.+Hilinski&title=All-Hydrocarbon+Stapled+Peptides+as+Synthetic+Cell-Accessible+Mini-Proteins&doi=10.1016%2Fj.ddtec.2012.01.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">All-hydrocarbon stapled peptides as Synthetic Cell-Accessible Mini-Proteins</span></div><div class="casAuthors">Verdine, Gregory L.; Hilinski, Gerard J.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today: Technologies</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">e41-e47</span>CODEN:
                <span class="NLM_cas:coden">DDTTB5</span>;
        ISSN:<span class="NLM_cas:issn">1740-6749</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  A majority of proteins responsible for the establishment and maintenance of human disease states are unable to be targeted therapeutically by mols. belonging to either of the two established classes of drugs, namely small mols. and protein therapeutics.  Recent efforts toward drugging these undruggable' proteins have led to greatly increased focus on cell-penetrating mini-proteins as a new class of agents for targeting intractable intracellular proteins.  These mols. are designed to combine the advantages intrinsic to each conventional therapeutic modality while overcoming their individual limitations.  One class of cell-penetrating mini-proteins, all-hydrocarbon stapled peptides, has recently demonstrated the ability to potently and specifically target previously intractable proteins such as transcription factors, vaulting this class to the forefront of a new wave of next-generation drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbIYOAoNrhCbVg90H21EOLACvtfcHk0lgfr1RCZWFxOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XlvFaqsLs%253D&md5=3d4b9e7c03349eab584c830e4252415f</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1016%2Fj.ddtec.2012.01.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ddtec.2012.01.004%26sid%3Dliteratum%253Aachs%26aulast%3DVerdine%26aufirst%3DG.%2BL.%26aulast%3DHilinski%26aufirst%3DG.%2BJ.%26atitle%3DAll-Hydrocarbon%2520Stapled%2520Peptides%2520as%2520Synthetic%2520Cell-Accessible%2520Mini-Proteins%26jtitle%3DDrug%2520Discovery%2520Today%2520Technol.%26date%3D2012%26volume%3D9%26spage%3De41%26epage%3De47%26doi%3D10.1016%2Fj.ddtec.2012.01.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bernal, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tyler, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korsmeyer, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walensky, L. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verdine, G. L.</span></span> <span> </span><span class="NLM_article-title">Reactivation of the P53 Tumor Suppressor Pathway by a Stapled P53 Peptide</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>129</i></span>,  <span class="NLM_fpage">2456</span>– <span class="NLM_lpage">2457</span>, <span class="refDoi"> DOI: 10.1021/ja0693587</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja0693587" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlGltrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=129&publication_year=2007&pages=2456-2457&author=F.+Bernalauthor=A.+F.+Tylerauthor=S.+J.+Korsmeyerauthor=L.+D.+Walenskyauthor=G.+L.+Verdine&title=Reactivation+of+the+P53+Tumor+Suppressor+Pathway+by+a+Stapled+P53+Peptide&doi=10.1021%2Fja0693587"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Reactivation of the p53 Tumor Suppressor Pathway by a Stapled p53 Peptide</span></div><div class="casAuthors">Bernal, Federico; Tyler, Andrew F.; Korsmeyer, Stanley J.; Walensky, Loren D.; Verdine, Gregory L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">129</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2456-2457</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The p53-hDM2 protein interaction is a validated therapeutic target in cancer.  We report the synthesis of stabilized alpha-helix of p53 (SAH-p53) compds. that antagonize the p53-hDM2 interaction.  We demonstrate that hydrocarbon stapling confers cellular permeability to a p53 peptide that is then capable of modulating transcriptional activity.  The lead SAH-p53 compd. triggers apoptosis in hDM2-overexpressing cancer cells by reactivating the native p53 signaling pathway.  SAH-p53 is the first example of an all-hydrocarbon i, i+7 stabilized peptide that subverts cancer through direct modulation of a transcriptional pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrU06ywOSa4CLVg90H21EOLACvtfcHk0lgfr1RCZWFxOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlGltrw%253D&md5=802905c5e6aaed4a224092f5603731fd</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Fja0693587&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja0693587%26sid%3Dliteratum%253Aachs%26aulast%3DBernal%26aufirst%3DF.%26aulast%3DTyler%26aufirst%3DA.%2BF.%26aulast%3DKorsmeyer%26aufirst%3DS.%2BJ.%26aulast%3DWalensky%26aufirst%3DL.%2BD.%26aulast%3DVerdine%26aufirst%3DG.%2BL.%26atitle%3DReactivation%2520of%2520the%2520P53%2520Tumor%2520Suppressor%2520Pathway%2520by%2520a%2520Stapled%2520P53%2520Peptide%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2007%26volume%3D129%26spage%3D2456%26epage%3D2457%26doi%3D10.1021%2Fja0693587" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bird, G. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madani, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perry, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Princiotto, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Supko, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gavathiotis, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sodroski, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walensky, L. D.</span></span> <span> </span><span class="NLM_article-title">Hydrocarbon Double-Stapling Remedies the Proteolytic Instability of a Lengthy Peptide Therapeutic</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>107</i></span>,  <span class="NLM_fpage">14093</span>– <span class="NLM_lpage">14098</span>, <span class="refDoi"> DOI: 10.1073/pnas.1002713107</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1073%2Fpnas.1002713107" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=20660316" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVCktrfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=107&publication_year=2010&pages=14093-14098&author=G.+H.+Birdauthor=N.+Madaniauthor=A.+F.+Perryauthor=A.+M.+Princiottoauthor=J.+G.+Supkoauthor=X.+Heauthor=E.+Gavathiotisauthor=J.+G.+Sodroskiauthor=L.+D.+Walensky&title=Hydrocarbon+Double-Stapling+Remedies+the+Proteolytic+Instability+of+a+Lengthy+Peptide+Therapeutic&doi=10.1073%2Fpnas.1002713107"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Hydrocarbon double-stapling remedies the proteolytic instability of a lengthy peptide therapeutic</span></div><div class="casAuthors">Bird, Gregory H.; Madani, Navid; Perry, Alisa F.; Princiotto, Amy M.; Supko, Jeffrey G.; He, Xiaoying; Gavathiotis, Evripidis; Sodroski, Joseph G.; Walensky, Loren D.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">107</span>
        (<span class="NLM_cas:issue">32</span>),
    <span class="NLM_cas:pages">14093-14098, S14093/1-S14093/8</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The pharmacol. utility of lengthy peptides can be hindered by loss of bioactive structure and rapid proteolysis, which limits bioavailability.  For example, enfuvirtide (Fuzeon, T20, DP178), a 36-amino acid peptide that inhibits human immunodeficiency virus type 1 (HIV-1) infection by effectively targeting the viral fusion app., has been relegated to a salvage treatment option mostly due to poor in vivo stability and lack of oral bioavailability.  To overcome the proteolytic shortcomings of long peptides as therapeutics, the authors examd. the biophys., biol., and pharmacol. impact of inserting all-hydrocarbon staples into an HIV-1 fusion inhibitor.  The authors find that peptide double-stapling confers striking protease resistance that translates into markedly improved pharmacokinetic properties, including oral absorption.  The authors detd. that the hydrocarbon staples create a proteolytic shield by combining reinforcement of overall α-helical structure, which slows the kinetics of proteolysis, with complete blockade of peptide cleavage at constrained sites in the immediate vicinity of the staple.  Importantly, double-stapling also optimizes the antiviral activity of HIV-1 fusion peptides and the antiproteolytic feature extends to other therapeutic peptide templates, such as the diabetes drug exenatide (Byetta).  Thus, hydrocarbon double-stapling may unlock the therapeutic potential of natural bioactive polypeptides by transforming them into structurally fortified agents with enhanced bioavailability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqme3pVM-fEB7Vg90H21EOLACvtfcHk0lh8UvokMm_UuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVCktrfJ&md5=95edd2112b56b564864e462eeb276e11</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1002713107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1002713107%26sid%3Dliteratum%253Aachs%26aulast%3DBird%26aufirst%3DG.%2BH.%26aulast%3DMadani%26aufirst%3DN.%26aulast%3DPerry%26aufirst%3DA.%2BF.%26aulast%3DPrinciotto%26aufirst%3DA.%2BM.%26aulast%3DSupko%26aufirst%3DJ.%2BG.%26aulast%3DHe%26aufirst%3DX.%26aulast%3DGavathiotis%26aufirst%3DE.%26aulast%3DSodroski%26aufirst%3DJ.%2BG.%26aulast%3DWalensky%26aufirst%3DL.%2BD.%26atitle%3DHydrocarbon%2520Double-Stapling%2520Remedies%2520the%2520Proteolytic%2520Instability%2520of%2520a%2520Lengthy%2520Peptide%2520Therapeutic%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2010%26volume%3D107%26spage%3D14093%26epage%3D14098%26doi%3D10.1073%2Fpnas.1002713107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Danial, N. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walensky, L. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fisher, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Molina, A. J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Datta, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pitter, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bird, G. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wikstrom, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deeney, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robertson, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morash, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kulkarni, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neschen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenberg, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corkey, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shirihai, O. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shulman, G. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lowell, B. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korsmeyer, S. J.</span></span> <span> </span><span class="NLM_article-title">Dual Role of Proapoptotic BAD in Insulin Secretion and Beta Cell Survival</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">144</span>– <span class="NLM_lpage">153</span>, <span class="refDoi"> DOI: 10.1038/nm1717</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1038%2Fnm1717" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=18223655" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhs1KhsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2008&pages=144-153&author=N.+N.+Danialauthor=L.+D.+Walenskyauthor=C.-Y.+Zhangauthor=C.+S.+Choiauthor=J.+K.+Fisherauthor=A.+J.+A.+Molinaauthor=S.+R.+Dattaauthor=K.+L.+Pitterauthor=G.+H.+Birdauthor=J.+D.+Wikstromauthor=J.+T.+Deeneyauthor=K.+Robertsonauthor=J.+Morashauthor=A.+Kulkarniauthor=S.+Neschenauthor=S.+Kimauthor=M.+E.+Greenbergauthor=B.+E.+Corkeyauthor=O.+S.+Shirihaiauthor=G.+I.+Shulmanauthor=B.+B.+Lowellauthor=S.+J.+Korsmeyer&title=Dual+Role+of+Proapoptotic+BAD+in+Insulin+Secretion+and+Beta+Cell+Survival&doi=10.1038%2Fnm1717"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Dual role of proapoptotic BAD in insulin secretion and beta cell survival</span></div><div class="casAuthors">Danial, Nika N.; Walensky, Loren D.; Zhang, Chen-Yu; Choi, Cheol Soo; Fisher, Jill K.; Molina, Anthony J. A.; Datta, Sandeep Robert; Pitter, Kenneth L.; Bird, Gregory H.; Wikstrom, Jakob D.; Deeney, Jude T.; Robertson, Kirsten; Morash, Joel; Kulkarni, Ameya; Neschen, Susanne; Kim, Sheene; Greenberg, Michael E.; Corkey, Barbara E.; Shirihai, Orian S.; Shulman, Gerald I.; Lowell, Bradford B.; Korsmeyer, Stanley J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">144-153</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The proapoptotic BCL-2 family member BAD resides in a glucokinase-contg. complex that regulates glucose-driven mitochondrial respiration.  Here, the authors present genetic evidence of a physiol. role for BAD in glucose-stimulated insulin secretion by beta cells.  This novel function of BAD is specifically dependent upon the phosphorylation of its BH3 sequence, previously defined as an essential death domain.  The authors highlight the pharmacol. relevance of phosphorylated BAD BH3 by using cell-permeable, hydrocarbon-stapled BAD BH3 helixes that target glucokinase, restore glucose-driven mitochondrial respiration and correct the insulin secretory response in Bad-deficient islets.  The authors' studies uncover an alternative target and function for the BAD BH3 domain and emphasize the therapeutic potential of phosphorylated BAD BH3 mimetics in selectively restoring beta cell function.  Furthermore, the authors show that BAD regulates the physiol. adaptation of beta cell mass during high-fat feeding.  The authors' findings provide genetic proof of the bifunctional activities of BAD in both beta cell survival and insulin secretion.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpQFhSE2I8jdLVg90H21EOLACvtfcHk0lh8UvokMm_UuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhs1KhsLo%253D&md5=204579cd0cabd3a8d372dbd23efdd869</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1038%2Fnm1717&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm1717%26sid%3Dliteratum%253Aachs%26aulast%3DDanial%26aufirst%3DN.%2BN.%26aulast%3DWalensky%26aufirst%3DL.%2BD.%26aulast%3DZhang%26aufirst%3DC.-Y.%26aulast%3DChoi%26aufirst%3DC.%2BS.%26aulast%3DFisher%26aufirst%3DJ.%2BK.%26aulast%3DMolina%26aufirst%3DA.%2BJ.%2BA.%26aulast%3DDatta%26aufirst%3DS.%2BR.%26aulast%3DPitter%26aufirst%3DK.%2BL.%26aulast%3DBird%26aufirst%3DG.%2BH.%26aulast%3DWikstrom%26aufirst%3DJ.%2BD.%26aulast%3DDeeney%26aufirst%3DJ.%2BT.%26aulast%3DRobertson%26aufirst%3DK.%26aulast%3DMorash%26aufirst%3DJ.%26aulast%3DKulkarni%26aufirst%3DA.%26aulast%3DNeschen%26aufirst%3DS.%26aulast%3DKim%26aufirst%3DS.%26aulast%3DGreenberg%26aufirst%3DM.%2BE.%26aulast%3DCorkey%26aufirst%3DB.%2BE.%26aulast%3DShirihai%26aufirst%3DO.%2BS.%26aulast%3DShulman%26aufirst%3DG.%2BI.%26aulast%3DLowell%26aufirst%3DB.%2BB.%26aulast%3DKorsmeyer%26aufirst%3DS.%2BJ.%26atitle%3DDual%2520Role%2520of%2520Proapoptotic%2520BAD%2520in%2520Insulin%2520Secretion%2520and%2520Beta%2520Cell%2520Survival%26jtitle%3DNat.%2520Med.%26date%3D2008%26volume%3D14%26spage%3D144%26epage%3D153%26doi%3D10.1038%2Fnm1717" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lawrence, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jourdan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vallée, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blandin, V.</span></span> <span> </span><span class="NLM_article-title">Peptide Cyclization via Ring-Closing Metathesis: The N-Alkenoxy Peptide Approach</span>. <i>Org. Biomol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">4575</span>– <span class="NLM_lpage">4581</span>, <span class="refDoi"> DOI: 10.1039/b812611a</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1039%2Fb812611a" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=19039366" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsVChtb7L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2008&pages=4575-4581&author=J.+Lawrenceauthor=M.+Jourdanauthor=Y.+Vall%C3%A9eauthor=V.+Blandin&title=Peptide+Cyclization+via+Ring-Closing+Metathesis%3A+The+N-Alkenoxy+Peptide+Approach&doi=10.1039%2Fb812611a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Peptide cyclization via ring-closing metathesis: the N-alkenoxy peptide approach</span></div><div class="casAuthors">Lawrence, James; Jourdan, Muriel; Vallee, Yannick; Blandin, Veronique</div><div class="citationInfo"><span class="NLM_cas:title">Organic & Biomolecular Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">4575-4581</span>CODEN:
                <span class="NLM_cas:coden">OBCRAK</span>;
        ISSN:<span class="NLM_cas:issn">1477-0520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">The prepn. of 17- and 18-membered cyclic hexapeptides from N-hydroxy tripeptides building blocks is described.  Introduction of an unsatd. chain on the hydroxamate oxygen followed by fragment coupling leads to N,N'-dialkenoxy hexapeptides that are efficiently cyclized through a ring-closing metathesis reaction.  The length of the alkene chains allows the modulation of the ring size.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8raTcvXy91rVg90H21EOLACvtfcHk0lh8UvokMm_UuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsVChtb7L&md5=d45a64eb1bc38d471c11e27a34357ab7</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1039%2Fb812611a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fb812611a%26sid%3Dliteratum%253Aachs%26aulast%3DLawrence%26aufirst%3DJ.%26aulast%3DJourdan%26aufirst%3DM.%26aulast%3DVall%25C3%25A9e%26aufirst%3DY.%26aulast%3DBlandin%26aufirst%3DV.%26atitle%3DPeptide%2520Cyclization%2520via%2520Ring-Closing%2520Metathesis%253A%2520The%2520N-Alkenoxy%2520Peptide%2520Approach%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2008%26volume%3D6%26spage%3D4575%26epage%3D4581%26doi%3D10.1039%2Fb812611a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sviridov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ikpot, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stonik, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drake, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amar, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osei-Hwedieh, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piszczek, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turner, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Remaley, A.</span></span> <span> </span><span class="NLM_article-title">Helix Stabilization of Amphipathic Peptides by Hydrocarbon Stapling Increases Cholesterol Efflux by the ABCA1 Transporter</span>. <i>Biochem. Biophys. Res. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>410</i></span>,  <span class="NLM_fpage">446</span>– <span class="NLM_lpage">451</span>, <span class="refDoi"> DOI: 10.1016/j.bbrc.2011.05.154</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1016%2Fj.bbrc.2011.05.154" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=21672528" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC3MXoslCns7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=410&publication_year=2011&pages=446-451&author=D.+Sviridovauthor=I.+Ikpotauthor=J.+Stonikauthor=S.+Drakeauthor=M.+Amarauthor=D.+Osei-Hwediehauthor=G.+Piszczekauthor=S.+Turnerauthor=A.+Remaley&title=Helix+Stabilization+of+Amphipathic+Peptides+by+Hydrocarbon+Stapling+Increases+Cholesterol+Efflux+by+the+ABCA1+Transporter&doi=10.1016%2Fj.bbrc.2011.05.154"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Helix stabilization of amphipathic peptides by hydrocarbon stapling increases cholesterol efflux by the ABCA1 transporter</span></div><div class="casAuthors">Sviridov, D. O.; Ikpot, I. Z.; Stonik, J.; Drake, S. K.; Amar, M.; Osei-Hwedieh, D. O.; Piszczek, G.; Turner, S.; Remaley, A. T.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical and Biophysical Research Communications</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">410</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">446-451</span>CODEN:
                <span class="NLM_cas:coden">BBRCA9</span>;
        ISSN:<span class="NLM_cas:issn">0006-291X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Apolipoprotein mimetic peptides are short amphipathic peptides that efflux cholesterol from cells by the ABCA1 transporter and are being investigated as therapeutic agents for cardiovascular disease.  We examd. the role of helix stabilization of these peptides in cholesterol efflux.  A 23-amino acid long peptide (Ac-VLEDSFKVSFLSALEEYTKKLNTQ-NH2) based on the last helix of apoA-I (A10) was synthesized, as well as two variants, S1A10 and S2A10, in which the third and fourth and third and fifth turn of each peptide, resp., were covalently joined by hydrocarbon staples.  By CD spectroscopy, the stapled variants at 24° were more helical in aq. buffer than A10 (A10 17%, S1A10 62%, S2A10 97%).  S1A10 and S2A10 unlike A10 were resistant to proteolysis by pepsin and chymotrypsin.  S1A10 and S2A10 showed more than a 10-fold increase in cholesterol efflux by the ABCA1 transporter compared to A10.  In summary, hydrocarbon stapling of amphipathic peptides increases their helicity, makes them resistant to proteolysis and enhances their ability to promote cholesterol efflux by the ABCA1 transporter, indicating that this peptide modification may be useful in the development of apolipoprotein mimetic peptides.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrK6lZ57TEU2bVg90H21EOLACvtfcHk0lhhD-mDbjZtOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXoslCns7s%253D&md5=4285715e1d9ba3410a36f2273b82d199</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1016%2Fj.bbrc.2011.05.154&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbrc.2011.05.154%26sid%3Dliteratum%253Aachs%26aulast%3DSviridov%26aufirst%3DD.%26aulast%3DIkpot%26aufirst%3DI.%26aulast%3DStonik%26aufirst%3DJ.%26aulast%3DDrake%26aufirst%3DS.%26aulast%3DAmar%26aufirst%3DM.%26aulast%3DOsei-Hwedieh%26aufirst%3DD.%26aulast%3DPiszczek%26aufirst%3DG.%26aulast%3DTurner%26aufirst%3DS.%26aulast%3DRemaley%26aufirst%3DA.%26atitle%3DHelix%2520Stabilization%2520of%2520Amphipathic%2520Peptides%2520by%2520Hydrocarbon%2520Stapling%2520Increases%2520Cholesterol%2520Efflux%2520by%2520the%2520ABCA1%2520Transporter%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D2011%26volume%3D410%26spage%3D446%26epage%3D451%26doi%3D10.1016%2Fj.bbrc.2011.05.154" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Walensky, L. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kung, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Escher, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malia, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbuto, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wright, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verdine, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korsmeyer, S. J.</span></span> <span> </span><span class="NLM_article-title">Activation of Apoptosis in Vivo by a Hydrocarbon-Stapled BH3 Helix</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>305</i></span>,  <span class="NLM_fpage">1466</span>– <span class="NLM_lpage">1470</span>, <span class="refDoi"> DOI: 10.1126/science.1099191</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1126%2Fscience.1099191" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=15353804" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BD2cXntFart7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=305&publication_year=2004&pages=1466-1470&author=L.+D.+Walenskyauthor=A.+L.+Kungauthor=I.+Escherauthor=T.+J.+Maliaauthor=S.+Barbutoauthor=R.+D.+Wrightauthor=G.+Wagnerauthor=G.+L.+Verdineauthor=S.+J.+Korsmeyer&title=Activation+of+Apoptosis+in+Vivo+by+a+Hydrocarbon-Stapled+BH3+Helix&doi=10.1126%2Fscience.1099191"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Activation of Apoptosis in Vivo by a Hydrocarbon-Stapled BH3 Helix</span></div><div class="casAuthors">Walensky, Loren D.; Kung, Andrew L.; Escher, Iris; Malia, Thomas J.; Barbuto, Scott; Wright, Renee D.; Wagner, Gerhard; Verdine, Gregory L.; Korsmeyer, Stanley J.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">305</span>
        (<span class="NLM_cas:issue">5689</span>),
    <span class="NLM_cas:pages">1466-1470</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">BCL-2 family proteins constitute a crit. control point for the regulation of apoptosis.  Protein interaction between BCL-2 members is a prominent mechanism of control and is mediated through the amphipathic α-helical BH3 segment, an essential death domain.  We used a chem. strategy, termed hydrocarbon stapling, to generate BH3 peptides with improved pharmacol. properties.  The stapled peptides, called "stabilized alpha-helix of BCL-2 domains" (SAHBs), proved to be helical, protease-resistant, and cell-permeable mols. that bound with increased affinity to multidomain BCL-2 member pockets.  A SAHB of the BH3 domain from the BID protein specifically activated the apoptotic pathway to kill leukemia cells.  In addn., SAHB effectively inhibited the growth of human leukemia xenografts in vivo.  Hydrocarbon stapling of native peptides may provide a useful strategy for exptl. and therapeutic modulation of protein-protein interactions in many signaling pathways.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrpfOpXrd8FVbVg90H21EOLACvtfcHk0lhhD-mDbjZtOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXntFart7c%253D&md5=53fca0524df7a584ccf9d0991bc1bb64</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1126%2Fscience.1099191&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1099191%26sid%3Dliteratum%253Aachs%26aulast%3DWalensky%26aufirst%3DL.%2BD.%26aulast%3DKung%26aufirst%3DA.%2BL.%26aulast%3DEscher%26aufirst%3DI.%26aulast%3DMalia%26aufirst%3DT.%2BJ.%26aulast%3DBarbuto%26aufirst%3DS.%26aulast%3DWright%26aufirst%3DR.%2BD.%26aulast%3DWagner%26aufirst%3DG.%26aulast%3DVerdine%26aufirst%3DG.%2BL.%26aulast%3DKorsmeyer%26aufirst%3DS.%2BJ.%26atitle%3DActivation%2520of%2520Apoptosis%2520in%2520Vivo%2520by%2520a%2520Hydrocarbon-Stapled%2520BH3%2520Helix%26jtitle%3DScience%26date%3D2004%26volume%3D305%26spage%3D1466%26epage%3D1470%26doi%3D10.1126%2Fscience.1099191" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhattacharya, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waheed, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tong, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heck, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curreli, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goger, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowburn, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freed, E. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Debnath, A. K.</span></span> <span> </span><span class="NLM_article-title">A Cell-Penetrating Helical Peptide as a Potential HIV-1 Inhibitor</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>378</i></span>,  <span class="NLM_fpage">565</span>– <span class="NLM_lpage">580</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2008.02.066</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1016%2Fj.jmb.2008.02.066" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=18374356" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BD1cXlslWks7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=378&publication_year=2008&pages=565-580&author=H.+Zhangauthor=Q.+Zhaoauthor=S.+Bhattacharyaauthor=A.+A.+Waheedauthor=X.+Tongauthor=A.+Hongauthor=S.+Heckauthor=F.+Curreliauthor=M.+Gogerauthor=D.+Cowburnauthor=E.+O.+Freedauthor=A.+K.+Debnath&title=A+Cell-Penetrating+Helical+Peptide+as+a+Potential+HIV-1+Inhibitor&doi=10.1016%2Fj.jmb.2008.02.066"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">A Cell-penetrating Helical Peptide as a Potential HIV-1 Inhibitor</span></div><div class="casAuthors">Zhang, Hongtao; Zhao, Qian; Bhattacharya, Shibani; Waheed, Abdul A.; Tong, Xiaohe; Hong, Anita; Heck, Susanne; Curreli, Francesca; Goger, Michael; Cowburn, David; Freed, Eric O.; Debnath, Asim K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">378</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">565-580</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The capsid domain of the human immunodeficiency virus type 1 (HIV-1) Gag polyprotein is a crit. determinant of virus assembly, and is therefore a potential target for developing drugs for AIDS therapy.  Recently, a 12-mer α-helical peptide (CAI) was reported to disrupt immature- and mature-like capsid particle assembly in vitro; however, it failed to inhibit HIV-1 in cell culture due to its inability to penetrate cells.  The same group reported the x-ray crystal structure of CAI in complex with the C-terminal domain of capsid (C-CA) at a resoln. of 1.7 Å.  Using this structural information, the authors have utilized a structure-based rational design approach to stabilize the α-helical structure of CAI and convert it to a cell-penetrating peptide (CPP).  The modified peptide (NYAD-1) showed enhanced α-helicity.  Expts. with laser scanning confocal microscopy indicated that NYAD-1 penetrated cells and colocalized with the Gag polyprotein during its trafficking to the plasma membrane where virus assembly takes place.  NYAD-1 disrupted the assembly of both immature- and mature-like virus particles in cell-free and cell-based in vitro systems.  NMR chem. shift perturbation anal. mapped the binding site of NYAD-1 to residues 169-191 of the C-terminal domain of HIV-1 capsid encompassing the hydrophobic cavity and the crit. dimerization domain with an improved binding affinity over CAI.  Furthermore, exptl. data indicate that NYAD-1 most likely targets capsid at a post-entry stage.  Most significantly, NYAD-1 inhibited a large panel of HIV-1 isolates in cell culture at low micromolar potency.  The study demonstrates how a structure-based rational design strategy can be used to convert a cell-impermeable peptide to a cell-permeable peptide that displays activity in cell-based assays without compromising its mechanism of action.  This proof-of-concept cell-penetrating peptide may aid validation of capsid as an anti-HIV-1 drug target and may help in designing peptidomimetics and small mol. drugs targeted to this protein.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGql5DKXGiSgWLVg90H21EOLACvtfcHk0lhhD-mDbjZtOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXlslWks7w%253D&md5=9f22fccdd6dc38c9e5c8841c63216436</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2008.02.066&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2008.02.066%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DZhao%26aufirst%3DQ.%26aulast%3DBhattacharya%26aufirst%3DS.%26aulast%3DWaheed%26aufirst%3DA.%2BA.%26aulast%3DTong%26aufirst%3DX.%26aulast%3DHong%26aufirst%3DA.%26aulast%3DHeck%26aufirst%3DS.%26aulast%3DCurreli%26aufirst%3DF.%26aulast%3DGoger%26aufirst%3DM.%26aulast%3DCowburn%26aufirst%3DD.%26aulast%3DFreed%26aufirst%3DE.%2BO.%26aulast%3DDebnath%26aufirst%3DA.%2BK.%26atitle%3DA%2520Cell-Penetrating%2520Helical%2520Peptide%2520as%2520a%2520Potential%2520HIV-1%2520Inhibitor%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2008%26volume%3D378%26spage%3D565%26epage%3D580%26doi%3D10.1016%2Fj.jmb.2008.02.066" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rostovtsev, V. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, L. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fokin, V. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharpless, K. B.</span></span> <span> </span><span class="NLM_article-title">A Stepwise Huisgen Cycloaddition Process: Copper(I)-Catalyzed Regioselective “Ligation” of Azides and Terminal Alkynes</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">2596</span>– <span class="NLM_lpage">2599</span>, <span class="refDoi"> DOI: 10.1002/1521-3773(20020715)41:14<2596::AID-ANIE2596>3.0.CO;2-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1002%2F1521-3773%2820020715%2941%3A14%3C2596%3A%3AAID-ANIE2596%3E3.0.CO%3B2-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=12203546" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BD38Xls1Ohsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2002&pages=2596-2599&author=V.+V.+Rostovtsevauthor=L.+G.+Greenauthor=V.+V.+Fokinauthor=K.+B.+Sharpless&title=A+Stepwise+Huisgen+Cycloaddition+Process%3A+Copper%28I%29-Catalyzed+Regioselective+%E2%80%9CLigation%E2%80%9D+of+Azides+and+Terminal+Alkynes&doi=10.1002%2F1521-3773%2820020715%2941%3A14%3C2596%3A%3AAID-ANIE2596%3E3.0.CO%3B2-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">A stepwise Huisgen cycloaddition process: copper(I)-catalyzed regioselective "ligation" of azides and terminal alkynes</span></div><div class="casAuthors">Rostovtsev, Vsevolod V.; Green, Luke G.; Fokin, Valery V.; Sharpless, K. Barry</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">2596-2599</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH</span>)
        </div><div class="casAbstract">1,4-Disubstituted 1,2,3-triazoles I (R1 = PhCH2, PhCH2OCH2, 1-adamantyl, etc.; R2 = HO2C, Ph, PhOCH2, Et2NCH2, etc.) were readily and cleanly prepd. via highly efficient and regioselective copper(I)-catalyzed cycloaddn. of azides R1N3 with terminal alkynes R2C≡CH in 82-93% yields.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptcW2JUNM1r7Vg90H21EOLACvtfcHk0lhhD-mDbjZtOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xls1Ohsr4%253D&md5=4603664be6639353b5e70f19b9f8d59f</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1002%2F1521-3773%2820020715%2941%3A14%3C2596%3A%3AAID-ANIE2596%3E3.0.CO%3B2-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F1521-3773%252820020715%252941%253A14%253C2596%253A%253AAID-ANIE2596%253E3.0.CO%253B2-4%26sid%3Dliteratum%253Aachs%26aulast%3DRostovtsev%26aufirst%3DV.%2BV.%26aulast%3DGreen%26aufirst%3DL.%2BG.%26aulast%3DFokin%26aufirst%3DV.%2BV.%26aulast%3DSharpless%26aufirst%3DK.%2BB.%26atitle%3DA%2520Stepwise%2520Huisgen%2520Cycloaddition%2520Process%253A%2520Copper%2528I%2529-Catalyzed%2520Regioselective%2520%25E2%2580%259CLigation%25E2%2580%259D%2520of%2520Azides%2520and%2520Terminal%2520Alkynes%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2002%26volume%3D41%26spage%3D2596%26epage%3D2599%26doi%3D10.1002%2F1521-3773%2820020715%2941%3A14%3C2596%3A%3AAID-ANIE2596%3E3.0.CO%3B2-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tornøe, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christensen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meldal, M.</span></span> <span> </span><span class="NLM_article-title">Peptidotriazoles on Solid Phase: [1,2,3]-Triazoles by Regiospecific Copper(I)-Catalyzed 1,3-Dipolar Cycloadditions of Terminal Alkynes to Azides</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">3057</span>– <span class="NLM_lpage">3064</span>, <span class="refDoi"> DOI: 10.1021/jo011148j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo011148j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BD38XisVeks7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2002&pages=3057-3064&author=C.+W.+Torn%C3%B8eauthor=C.+Christensenauthor=M.+Meldal&title=Peptidotriazoles+on+Solid+Phase%3A+%5B1%2C2%2C3%5D-Triazoles+by+Regiospecific+Copper%28I%29-Catalyzed+1%2C3-Dipolar+Cycloadditions+of+Terminal+Alkynes+to+Azides&doi=10.1021%2Fjo011148j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Peptidotriazoles on Solid Phase: [1,2,3]-Triazoles by Regiospecific Copper(I)-Catalyzed 1,3-Dipolar Cycloadditions of Terminal Alkynes to Azides</span></div><div class="casAuthors">Tornoe, Christian W.; Christensen, Caspar; Meldal, Morten</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3057-3064</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The cycloaddn. of azides to alkynes is one of the most important synthetic routes to 1H-[1,2,3]-triazoles.  This work reports a novel regiospecific copper(I)-catalyzed 1,3-dipolar cycloaddn. of terminal alkynes to azides on solid-phase.  Primary, secondary, and tertiary alkyl azides, aryl azides, and an azido sugar were used successfully in the copper(I)-catalyzed cycloaddn. producing diversely 1,4-substituted [1,2,3]-triazoles in peptide backbones or side chains.  The reaction conditions were fully compatible with solid-phase peptide synthesis on polar supports.  The copper(I) catalysis is mild and efficient (>95% conversion and purity in most cases) and furthermore, the x-ray structure of 2-azido-2-methylpropanoic acid has been solved, to yield structural information on the 1,3-dipoles entering the reaction.  Novel Fmoc-protected amino azides were prepd. from Fmoc-amino alcs. by Mitsunobu reaction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJXSF5l0g_lrVg90H21EOLACvtfcHk0lj4zaOCWWigVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XisVeks7w%253D&md5=6b3b805572783873952871f1b69f46dd</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1021%2Fjo011148j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo011148j%26sid%3Dliteratum%253Aachs%26aulast%3DTorn%25C3%25B8e%26aufirst%3DC.%2BW.%26aulast%3DChristensen%26aufirst%3DC.%26aulast%3DMeldal%26aufirst%3DM.%26atitle%3DPeptidotriazoles%2520on%2520Solid%2520Phase%253A%2520%255B1%252C2%252C3%255D-Triazoles%2520by%2520Regiospecific%2520Copper%2528I%2529-Catalyzed%25201%252C3-Dipolar%2520Cycloadditions%2520of%2520Terminal%2520Alkynes%2520to%2520Azides%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2002%26volume%3D67%26spage%3D3057%26epage%3D3064%26doi%3D10.1021%2Fjo011148j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kolb, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finn, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharpless, K. B.</span></span> <span> </span><span class="NLM_article-title">Click Chemistry: Diverse Chemical Function from a Few Good Reactions</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">2004</span>– <span class="NLM_lpage">2021</span>, <span class="refDoi"> DOI: 10.1002/1521-3773(20010601)40:11<2004::AID-ANIE2004>3.0.CO;2-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1002%2F1521-3773%2820010601%2940%3A11%3C2004%3A%3AAID-ANIE2004%3E3.0.CO%3B2-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=11433435" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BD3MXksVOis78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2001&pages=2004-2021&author=H.+C.+Kolbauthor=M.+G.+Finnauthor=K.+B.+Sharpless&title=Click+Chemistry%3A+Diverse+Chemical+Function+from+a+Few+Good+Reactions&doi=10.1002%2F1521-3773%2820010601%2940%3A11%3C2004%3A%3AAID-ANIE2004%3E3.0.CO%3B2-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Click chemistry: diverse chemical function from a few good reactions</span></div><div class="casAuthors">Kolb, Hartmuth C.; Finn, M. G.; Sharpless, K. Barry</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2004-2021</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH</span>)
        </div><div class="casAbstract">Review with > 88 refs.  Examn. of nature's favorite mols. reveals a striking preference for making carbon-heteroatom bonds over carbon-carbon bonds - surely no surprise given that carbon dioxide is nature's starting material and that most reactions are performed in water.  Nucleic acids, proteins, and polysaccharides are condensation polymers of small subunits stitched together by carbon-heteroatom bonds.  Even the 35 or so building blocks from which these crucial mols. are made each contain, at most, six contiguous C-C bonds, except for the three arom. amino acids.  Taking a cue from nature's approach, the development of a set of powerful, highly reliable, and selective reactions for the rapid synthesis of useful new compds. and combinatorial libraries through heteroatom links (C-X-C), an approach called "click chem." is addressed.  Click chem. is at once defined, enabled, and constrained by a handful of nearly perfect "spring-loaded" reactions.  The stringent criteria for a process to earn click chem. status are described along with examples of the mol. frameworks that are easily made using this spartan, but powerful, synthetic strategy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptrjo9nxXNRrVg90H21EOLACvtfcHk0lj4zaOCWWigVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXksVOis78%253D&md5=61c28b416c42c87821bc8a5bc2fdec36</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1002%2F1521-3773%2820010601%2940%3A11%3C2004%3A%3AAID-ANIE2004%3E3.0.CO%3B2-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F1521-3773%252820010601%252940%253A11%253C2004%253A%253AAID-ANIE2004%253E3.0.CO%253B2-5%26sid%3Dliteratum%253Aachs%26aulast%3DKolb%26aufirst%3DH.%2BC.%26aulast%3DFinn%26aufirst%3DM.%2BG.%26aulast%3DSharpless%26aufirst%3DK.%2BB.%26atitle%3DClick%2520Chemistry%253A%2520Diverse%2520Chemical%2520Function%2520from%2520a%2520Few%2520Good%2520Reactions%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2001%26volume%3D40%26spage%3D2004%26epage%3D2021%26doi%3D10.1002%2F1521-3773%2820010601%2940%3A11%3C2004%3A%3AAID-ANIE2004%3E3.0.CO%3B2-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cantel, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Isaad, A. L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scrima, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levy, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DiMarchi, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rovero, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halperin, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">D’Ursi, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Papini, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chorev, M.</span></span> <span> </span><span class="NLM_article-title">Synthesis and Conformational Analysis of a Cyclic Peptide Obtained via i to I+4 Intramolecular Side-Chain to Side-Chain Azide-Alkyne 1,3-Dipolar Cycloaddition</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>73</i></span>,  <span class="NLM_fpage">5663</span>– <span class="NLM_lpage">5674</span>, <span class="refDoi"> DOI: 10.1021/jo800142s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo800142s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BD1cXmtVGhs78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2008&pages=5663-5674&author=S.+Cantelauthor=A.+L.+C.+Isaadauthor=M.+Scrimaauthor=J.+J.+Levyauthor=R.+D.+DiMarchiauthor=P.+Roveroauthor=J.+A.+Halperinauthor=A.+M.+D%E2%80%99Ursiauthor=A.+M.+Papiniauthor=M.+Chorev&title=Synthesis+and+Conformational+Analysis+of+a+Cyclic+Peptide+Obtained+via+i+to+I%2B4+Intramolecular+Side-Chain+to+Side-Chain+Azide-Alkyne+1%2C3-Dipolar+Cycloaddition&doi=10.1021%2Fjo800142s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and conformational analysis of a cyclic peptide obtained via i to i+4 intramolecular side-chain to side-chain azide-alkyne 1,3-dipolar cycloaddition</span></div><div class="casAuthors">Cantel, Sonia; Le Chevalier Isaad, Alexandra; Scrima, Mario; Levy, Jay J.; DiMarchi, Richard D.; Rovero, Paolo; Halperin, Jose A.; D'Ursi, Anna Maria; Papini, Anna Maria; Chorev, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">5663-5674</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Intramol. side-chain to side-chain cyclization is an established approach to achieve stabilization of specific conformations and a recognized strategy to improve resistance toward proteolytic degrdn.  To this end, cyclizations, which are bioisosteric to the lactam-type side-chain to side-chain modification and do not require orthogonal protection schemes, are of great interest.  Herein, we report the employment of CuI-catalyzed 1,3-dipolar cycloaddn. of side chains modified with azido and alkynyl functions and explore alternative synthetic routes to efficiently generate 1,4-disubstituted [1,2,3]triazolyl-contg. cyclopeptides.  The solid-phase assembly of the linear precursor including ε-azido norleucine and the propargylglycine (Pra) in positions i and i+4, resp., was accomplished by either subjecting the resin-bound peptide to selective on-resin diazo transformation of a Lys into the Nle(ε-N3) or the incorporation of Fmoc-Nle(ε-N3)-OH during the stepwise build-up of the resin-bound peptide.  Soln.-phase CuI-catalyzed 1,3-dipolar cycloaddn. converts the linear precursor Ac-Lys-Gly-Nle(ε-N3)-Ser-Ile-Gln-Pra-Leu-Arg-NH2 into the 1,4-disubstituted [1,2,3]triazolyl-contg. cyclopeptide [Ac-Lys-Gly-Xaa(&1)-Ser-Ile-Gln-Yaa(&2)-Leu-Arg-NH2][(&1(CH2)4-1,4-[1,2,3]triazolyl-CH2&2)].  The conformational preferences of the latter model cyclopeptide, which is derived from the sequence of a highly helical and potent i to i+4 side-chain to side-chain lactam-contg. antagonist of parathyroid hormone-related peptide (PTHrP), are compared to the corresponding lactam analog Ac[Lys13(&1),Asp17(&2)]hPTHrP(11-19)NH2.  CD and NMR studies of [Ac-Lys-Gly-Xaa(&1)-Ser-Ile-Gln-Yaa(&2)-Leu-Arg-NH2][(&1(CH2)4-1,4-[1,2,3]triazolyl-CH2&2)] and Ac[Lys13(&1),Asp17(&2)]hPTHrP(11-19)NH2 in water/hexafluoroacetone (HFA) (50:50, vol./vol.) revealed a high prevalence of turn-helical structures involving in particular the cyclic regions of the mol.  Despite a slight difference of the backbone arrangement, the side-chains of Ser, Gln, and Ile located at the i+1 to i+3 of the ring-forming sequences share the same spatial orientation.  Both cyclopeptides differ regarding the location of the turn-helical segment, which in Ac[Lys13(&1),Asp17(&2)]hPTHrP(11-19)NH2 involves noncyclized residues while in [Ac-Lys-Gly-Xaa(&1)-Ser-Ile-Gln-Yaa(&2)-Leu-Arg-NH2][(&1(CH2)4-1,4-[1,2,3]triazolyl-CH2&2)] it overlaps with residues involved in the cyclic structure.  Therefore, the synthetic accessibility and conformational similarity of i to i+4 side-chain to side-chain cyclopeptide contg. the 1,4-disubstituted [1,2,3]triazolyl moiety to the lactam-type one may result in similar bioactivities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWO7f-VCjhQrVg90H21EOLACvtfcHk0lj4zaOCWWigVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXmtVGhs78%253D&md5=6f761bb8554306cf71d3facc87bb0683</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1021%2Fjo800142s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo800142s%26sid%3Dliteratum%253Aachs%26aulast%3DCantel%26aufirst%3DS.%26aulast%3DIsaad%26aufirst%3DA.%2BL.%2BC.%26aulast%3DScrima%26aufirst%3DM.%26aulast%3DLevy%26aufirst%3DJ.%2BJ.%26aulast%3DDiMarchi%26aufirst%3DR.%2BD.%26aulast%3DRovero%26aufirst%3DP.%26aulast%3DHalperin%26aufirst%3DJ.%2BA.%26aulast%3DD%25E2%2580%2599Ursi%26aufirst%3DA.%2BM.%26aulast%3DPapini%26aufirst%3DA.%2BM.%26aulast%3DChorev%26aufirst%3DM.%26atitle%3DSynthesis%2520and%2520Conformational%2520Analysis%2520of%2520a%2520Cyclic%2520Peptide%2520Obtained%2520via%2520i%2520to%2520I%252B4%2520Intramolecular%2520Side-Chain%2520to%2520Side-Chain%2520Azide-Alkyne%25201%252C3-Dipolar%2520Cycloaddition%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2008%26volume%3D73%26spage%3D5663%26epage%3D5674%26doi%3D10.1021%2Fjo800142s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Scrima, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chevalier-Isaad, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rovero, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Papini, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chorev, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">D’Ursi, A. M.</span></span> <span> </span><span class="NLM_article-title">CuI-Catalyzed Azide–Alkyne Intramolecular i-to-(I+4) Side-Chain-to-Side-Chain Cyclization Promotes the Formation of Helix-Like Secondary Structures</span>. <i>Eur. J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>2010</i></span>,  <span class="NLM_fpage">446</span>– <span class="NLM_lpage">457</span>, <span class="refDoi"> DOI: 10.1002/ejoc.200901157</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1002%2Fejoc.200901157" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2010&publication_year=2010&pages=446-457&author=M.+Scrimaauthor=A.+L.+Chevalier-Isaadauthor=P.+Roveroauthor=A.+M.+Papiniauthor=M.+Chorevauthor=A.+M.+D%E2%80%99Ursi&title=CuI-Catalyzed+Azide%E2%80%93Alkyne+Intramolecular+i-to-%28I%2B4%29+Side-Chain-to-Side-Chain+Cyclization+Promotes+the+Formation+of+Helix-Like+Secondary+Structures&doi=10.1002%2Fejoc.200901157"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1002%2Fejoc.200901157&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fejoc.200901157%26sid%3Dliteratum%253Aachs%26aulast%3DScrima%26aufirst%3DM.%26aulast%3DChevalier-Isaad%26aufirst%3DA.%2BL.%26aulast%3DRovero%26aufirst%3DP.%26aulast%3DPapini%26aufirst%3DA.%2BM.%26aulast%3DChorev%26aufirst%3DM.%26aulast%3DD%25E2%2580%2599Ursi%26aufirst%3DA.%2BM.%26atitle%3DCuI-Catalyzed%2520Azide%25E2%2580%2593Alkyne%2520Intramolecular%2520i-to-%2528I%252B4%2529%2520Side-Chain-to-Side-Chain%2520Cyclization%2520Promotes%2520the%2520Formation%2520of%2520Helix-Like%2520Secondary%2520Structures%26jtitle%3DEur.%2520J.%2520Org.%2520Chem.%26date%3D2010%26volume%3D2010%26spage%3D446%26epage%3D457%26doi%3D10.1002%2Fejoc.200901157" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kawamoto, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleska, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ran, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Design of Triazole-Stapled BCL9 α-Helical Peptides to Target the β-Catenin/B-Cell CLL/Lymphoma 9 (BCL9) Protein-Protein Interaction</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">1137</span>– <span class="NLM_lpage">1146</span>, <span class="refDoi"> DOI: 10.1021/jm201125d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm201125d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs1OjtLfL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=1137-1146&author=S.+A.+Kawamotoauthor=A.+Coleskaauthor=X.+Ranauthor=H.+Yiauthor=C.-Y.+Yangauthor=S.+Wang&title=Design+of+Triazole-Stapled+BCL9+%CE%B1-Helical+Peptides+to+Target+the+%CE%B2-Catenin%2FB-Cell+CLL%2FLymphoma+9+%28BCL9%29+Protein-Protein+Interaction&doi=10.1021%2Fjm201125d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Design of Triazole-Stapled BCL9 α-Helical Peptides to Target the β-Catenin/B-Cell CLL/lymphoma 9 (BCL9) Protein-Protein Interaction</span></div><div class="casAuthors">Kawamoto, Steven A.; Coleska, Adriana; Ran, Xu; Yi, Han; Yang, Chao-Yie; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1137-1146</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The interaction between β-catenin and B-cell CLL/lymphoma 9 (BCL9), crit. for the transcriptional activity of β-catenin, is mediated by a helical segment from BCL9 and a large binding groove in β-catenin.  Design of potent, metabolically stable BCL9 peptides represents an attractive approach to inhibit the activity of β-catenin.  In this study, we report the use of the Huisgen 1,3-dipolar cycloaddn. reaction to generate triazole-stapled BCL9 α-helical peptides.  The high efficiency and mild conditions of this 'click' reaction combined with the ease of synthesis of the necessary unnatural amino acids allows for facile synthesis of triazole-stapled peptides.  We have performed extensive optimization of this approach and identified the optimal combinations of azido and alkynyl linkers necessary for stapling BCL9 helixes.  The unsym. nature of the triazole staple also allowed the synthesis of double-stapled BCL9 peptides, which show a marked increase in helical character and an improvement in binding affinity and metabolic stability relative to wild-type and linear BCL9 peptides.  This study lays the foundation for further optimization of these triazole-stapled BCL9 peptides as potent, metabolically stable, and cell-permeable inhibitors to target the β-catenin and BCL9 interaction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqf88hHn2HxybVg90H21EOLACvtfcHk0lhCPMJU7nq6Nw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs1OjtLfL&md5=d0ab7aaf4074398402ae5f8b004a481b</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1021%2Fjm201125d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm201125d%26sid%3Dliteratum%253Aachs%26aulast%3DKawamoto%26aufirst%3DS.%2BA.%26aulast%3DColeska%26aufirst%3DA.%26aulast%3DRan%26aufirst%3DX.%26aulast%3DYi%26aufirst%3DH.%26aulast%3DYang%26aufirst%3DC.-Y.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DDesign%2520of%2520Triazole-Stapled%2520BCL9%2520%25CE%25B1-Helical%2520Peptides%2520to%2520Target%2520the%2520%25CE%25B2-Catenin%252FB-Cell%2520CLL%252FLymphoma%25209%2520%2528BCL9%2529%2520Protein-Protein%2520Interaction%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D1137%26epage%3D1146%26doi%3D10.1021%2Fjm201125d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Madden, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muppidi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Q.</span></span> <span> </span><span class="NLM_article-title">Synthesis of Cell-Permeable Stapled Peptide Dual Inhibitors of the P53-Mdm2/Mdmx Interactions via Photoinduced Cycloaddition</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">1472</span>– <span class="NLM_lpage">1475</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2011.01.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1016%2Fj.bmcl.2011.01.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=21277201" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC3MXit1Gmu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2011&pages=1472-1475&author=M.+M.+Maddenauthor=A.+Muppidiauthor=Z.+Liauthor=X.+Liauthor=J.+Chenauthor=Q.+Lin&title=Synthesis+of+Cell-Permeable+Stapled+Peptide+Dual+Inhibitors+of+the+P53-Mdm2%2FMdmx+Interactions+via+Photoinduced+Cycloaddition&doi=10.1016%2Fj.bmcl.2011.01.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis of cell-permeable stapled peptide dual inhibitors of the p53-Mdm2/Mdmx interactions via photoinduced cycloaddition</span></div><div class="casAuthors">Madden, Michael M.; Muppidi, Avinash; Li, Zhenyu; Li, Xiaolong; Chen, Jiandong; Lin, Qing</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1472-1475</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The authors report the application of photoinduced 1,3-dipolar cycloaddn. reaction to "staple" a peptide dual inhibitor of the p53-Mdm2/Mdmx interactions.  A series of "stapled" peptide inhibitors was efficiently synthesized and showed excellent dual inhibitory activity in ELISA assay.  Furthermore, the pos. charged, "stapled" peptides showed enhanced cellular uptake along with modest in vivo activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIUVyYPsFaS7Vg90H21EOLACvtfcHk0lhCPMJU7nq6Nw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXit1Gmu7w%253D&md5=8640cbbb835293bc90c551c083405d29</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2011.01.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2011.01.004%26sid%3Dliteratum%253Aachs%26aulast%3DMadden%26aufirst%3DM.%2BM.%26aulast%3DMuppidi%26aufirst%3DA.%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DLin%26aufirst%3DQ.%26atitle%3DSynthesis%2520of%2520Cell-Permeable%2520Stapled%2520Peptide%2520Dual%2520Inhibitors%2520of%2520the%2520P53-Mdm2%252FMdmx%2520Interactions%2520via%2520Photoinduced%2520Cycloaddition%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2011%26volume%3D21%26spage%3D1472%26epage%3D1475%26doi%3D10.1016%2Fj.bmcl.2011.01.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span> <span> </span><span class="NLM_article-title">Stapling of Unprotected Helical Peptides via Photo-Induced Intramolecular Thiol-Yne Hydrothiolation</span>. <i>Chem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">3325</span>– <span class="NLM_lpage">3330</span>, <span class="refDoi"> DOI: 10.1039/C6SC00106H</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1039%2FC6SC00106H" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=29997825" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC28XitFGjurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=3325-3330&author=Y.+Tianauthor=J.+Liauthor=H.+Zhaoauthor=X.+Zengauthor=D.+Wangauthor=Q.+Liuauthor=X.+Niuauthor=X.+Huangauthor=N.+Xuauthor=Z.+Li&title=Stapling+of+Unprotected+Helical+Peptides+via+Photo-Induced+Intramolecular+Thiol-Yne+Hydrothiolation&doi=10.1039%2FC6SC00106H"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Stapling of unprotected helical peptides via photo-induced intramolecular thiol-yne hydrothiolation</span></div><div class="casAuthors">Tian, Yuan; Li, Jingxu; Zhao, Hui; Zeng, Xiangze; Wang, Dongyuan; Liu, Qisong; Niu, Xiaogang; Huang, Xuhui; Xu, Naihan; Li, Zigang</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Science</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">3325-3330</span>CODEN:
                <span class="NLM_cas:coden">CSHCCN</span>;
        ISSN:<span class="NLM_cas:issn">2041-6520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Peptide stapling emerged as a versatile strategy to recapitulate the bioactive helical conformation of unstructured short peptides in water to improve their therapeutic properties in targeting intracellular "undruggable" targets.  Here, the authors present the development of photoinduced intramol. thiol-yne macrocyclization for rapid access to short stapled peptides with enhanced biophys. properties.  This new peptide stapling technique provides rapid access to conformationally constrained helixes with satisfying functional group tolerance.  Notably, the vinyl sulfide linkage shows distinct lipophilicity with reduced membrane toxicity compared to the corresponding all-hydrocarbon analog.  As a proof of principle, the authors constructed stabilized helixes modulating intracellular estrogen receptor (ER)-coactivator interactions with a nanomolar binding affinity, enhanced serum stability, a diffuse cellular distribution and selective cytotoxicity towards ER-pos. MCF-7 cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGopVBk7PWClZrVg90H21EOLACvtfcHk0lhCPMJU7nq6Nw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitFGjurc%253D&md5=2749fb8c7b0b79210c9a74913c8b9002</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1039%2FC6SC00106H&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC6SC00106H%26sid%3Dliteratum%253Aachs%26aulast%3DTian%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DZhao%26aufirst%3DH.%26aulast%3DZeng%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DQ.%26aulast%3DNiu%26aufirst%3DX.%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DXu%26aufirst%3DN.%26aulast%3DLi%26aufirst%3DZ.%26atitle%3DStapling%2520of%2520Unprotected%2520Helical%2520Peptides%2520via%2520Photo-Induced%2520Intramolecular%2520Thiol-Yne%2520Hydrothiolation%26jtitle%3DChem.%2520Sci.%26date%3D2016%26volume%3D7%26spage%3D3325%26epage%3D3330%26doi%3D10.1039%2FC6SC00106H" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref64"><div class="reference"><strong class="refLabel"><a href="#ref64" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bruno, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cornillie, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nogieira, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheatham, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lim, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chou, D. H.-C.</span></span> <span> </span><span class="NLM_article-title">Application of Thiol-Yne/Thiol-Ene Reactions for Peptide and Protein Macrocyclizations</span>. <i>Chem. –Eur. J.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">7087</span>– <span class="NLM_lpage">7092</span>, <span class="refDoi"> DOI: 10.1002/chem.201700572</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1002%2Fchem.201700572" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=28345248" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC2sXms1Wks7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2017&pages=7087-7092&author=Y.+Wangauthor=B.+J.+Brunoauthor=S.+Cornillieauthor=J.+M.+Nogieiraauthor=D.+Chenauthor=T.+E.+Cheathamauthor=C.+S.+Limauthor=D.+H.-C.+Chou&title=Application+of+Thiol-Yne%2FThiol-Ene+Reactions+for+Peptide+and+Protein+Macrocyclizations&doi=10.1002%2Fchem.201700572"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Application of Thiol-yne/Thiol-ene Reactions for Peptide and Protein Macrocyclizations</span></div><div class="casAuthors">Wang, Yuanxiang; Bruno, Benjamin J.; Cornillie, Sean; Nogieira, Jason M.; Chen, Diao; Cheatham, Thomas E., III; Lim, Carol S.; Chou, Danny Hung-Chieh</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry - A European Journal</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">29</span>),
    <span class="NLM_cas:pages">7087-7092</span>CODEN:
                <span class="NLM_cas:coden">CEUJED</span>;
        ISSN:<span class="NLM_cas:issn">0947-6539</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The application of thiol-yne/thiol-ene reactions to synthesize mono- and bicyclic-stapled peptides and proteins is reported.  First, a thiol-ene-based peptide-stapling metho39713440Fd in aq. conditions was developed.  This method enabled the efficient stapling of recombinantly expressed coil-coiled proteins.  The resulting stapled protein demonstrated higher stability in its secondary structure than the unstapled version.  Furthermore, a thiol-yne coupling was performed by using an α,ω-diyne to react with two cysteine residues to synthesize a stapled peptide with two vinyl sulfide groups.  The stapled peptide could further react with another biscysteine peptide to yield a bicyclic stapled peptide with enhanced properties.  For example, the cell permeability of a stapled peptide was further increased by appending an oligoarginine cell-penetrating peptide.  The robustness and versatility of thiol-yne/thiol-ene reactions that can be applied to both synthetic and expressed peptides and proteins were demonstrated.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqI0I-p3I4MebVg90H21EOLACvtfcHk0ljTYLU_sYsLEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXms1Wks7k%253D&md5=019af66e3a684454a84cf0115b4c2a24</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1002%2Fchem.201700572&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fchem.201700572%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DY.%26aulast%3DBruno%26aufirst%3DB.%2BJ.%26aulast%3DCornillie%26aufirst%3DS.%26aulast%3DNogieira%26aufirst%3DJ.%2BM.%26aulast%3DChen%26aufirst%3DD.%26aulast%3DCheatham%26aufirst%3DT.%2BE.%26aulast%3DLim%26aufirst%3DC.%2BS.%26aulast%3DChou%26aufirst%3DD.%2BH.-C.%26atitle%3DApplication%2520of%2520Thiol-Yne%252FThiol-Ene%2520Reactions%2520for%2520Peptide%2520and%2520Protein%2520Macrocyclizations%26jtitle%3DChem.%2520%25E2%2580%2593Eur.%2520J.%26date%3D2017%26volume%3D23%26spage%3D7087%26epage%3D7092%26doi%3D10.1002%2Fchem.201700572" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref65"><div class="reference"><strong class="refLabel"><a href="#ref65" class="rightTabRefNumLink">65</a></strong><div class="NLM_citation" id="rightTab-cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chou, D. H.-C.</span></span> <span> </span><span class="NLM_article-title">A Thiol-Ene Coupling Approach to Native Peptide Stapling and Macrocyclization</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">10931</span>– <span class="NLM_lpage">10934</span>, <span class="refDoi"> DOI: 10.1002/anie.201503975</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1002%2Fanie.201503975" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=26189498" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFyqtb%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2015&pages=10931-10934&author=Y.+Wangauthor=D.+H.-C.+Chou&title=A+Thiol-Ene+Coupling+Approach+to+Native+Peptide+Stapling+and+Macrocyclization&doi=10.1002%2Fanie.201503975"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">A thiol-ene coupling approach to native peptide stapling and macrocyclization</span></div><div class="casAuthors">Wang, Yuanxiang; Chou, Danny Hung-Chieh</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">37</span>),
    <span class="NLM_cas:pages">10931-10934</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">We report the discovery of a peptide stapling and macrocyclization method using thiol-ene reactions between two cysteine residues and an α,ω-diene in high yields.  This new approach enabled us to selectively modify cysteine residues in native, unprotected peptides with a variety of stapling modifications for helix stabilization or general macrocyclization.  We synthesized stapled Axin mimetic analogs and demonstrated increased alpha helicity upon peptide stapling.  We then synthesized stapled p53 mimetic analogs using pure hydrocarbon linkers and demonstrated their abilities to block the p53-MDM2 interaction and selectively kill p53 wild-type colorectal carcinoma HCT-116 cells but not p53 null cells.  In summary, we demonstrated a robust and versatile peptide stapling method that could be potentially applied to both synthetic and expressed peptides.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdtZz8o6RMvbVg90H21EOLACvtfcHk0ljTYLU_sYsLEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFyqtb%252FN&md5=2ea0ca9a99ab25c0cb3a1000b4cf618c</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1002%2Fanie.201503975&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201503975%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DY.%26aulast%3DChou%26aufirst%3DD.%2BH.-C.%26atitle%3DA%2520Thiol-Ene%2520Coupling%2520Approach%2520to%2520Native%2520Peptide%2520Stapling%2520and%2520Macrocyclization%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2015%26volume%3D54%26spage%3D10931%26epage%3D10934%26doi%3D10.1002%2Fanie.201503975" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref66"><div class="reference"><strong class="refLabel"><a href="#ref66" class="rightTabRefNumLink">66</a></strong><div class="NLM_citation" id="rightTab-cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span> <span> </span><span class="NLM_article-title">Constructing Thioether-Tethered Cyclic Peptides via on-Resin Intra-Molecular Thiol-Ene Reaction</span>. <i>J. Pept. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">540</span>– <span class="NLM_lpage">544</span>, <span class="refDoi"> DOI: 10.1002/psc.2902</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1002%2Fpsc.2902" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=27378284" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFSiurzK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2016&pages=540-544&author=B.+Zhaoauthor=Q.+Zhangauthor=Z.+Li&title=Constructing+Thioether-Tethered+Cyclic+Peptides+via+on-Resin+Intra-Molecular+Thiol-Ene+Reaction&doi=10.1002%2Fpsc.2902"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">Constructing thioether-tethered cyclic peptides via on-resin intra-molecular thiol-ene reaction</span></div><div class="casAuthors">Zhao, Bingchuan; Zhang, Qingzhou; Li, Zigang</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Peptide Science</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">540-544</span>CODEN:
                <span class="NLM_cas:coden">JPSIEI</span>;
        ISSN:<span class="NLM_cas:issn">1075-2617</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons Ltd.</span>)
        </div><div class="casAbstract">Thiol-ene reactions have been used in a variety of applications that mostly involve an inter-mol. pathway.  Herein, we report a facile method to construct thioether-tethered cyclic peptides via an intra-mol. thiol-ene reaction.  This reaction is efficient, selective, and has good residue compatibility.  Short peptides with thioether tethers were constructed and were used to construct longer cyclic peptides.  This synthetic method may be useful for constructing bioactive peptides.  Copyright © 2016 European Peptide Society and John Wiley & Sons, Ltd.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMcNgwald7V7Vg90H21EOLACvtfcHk0ljTYLU_sYsLEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFSiurzK&md5=084b999debfc1cd056ebeeb15452f448</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1002%2Fpsc.2902&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpsc.2902%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DB.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DLi%26aufirst%3DZ.%26atitle%3DConstructing%2520Thioether-Tethered%2520Cyclic%2520Peptides%2520via%2520on-Resin%2520Intra-Molecular%2520Thiol-Ene%2520Reaction%26jtitle%3DJ.%2520Pept.%2520Sci.%26date%3D2016%26volume%3D22%26spage%3D540%26epage%3D544%26doi%3D10.1002%2Fpsc.2902" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref67"><div class="reference"><strong class="refLabel"><a href="#ref67" class="rightTabRefNumLink">67</a></strong><div class="NLM_citation" id="rightTab-cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fadzen, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolfe, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, C.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiocca, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawler, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pentelute, B. L.</span></span> <span> </span><span class="NLM_article-title">Perfluoroarene-Based Peptide Macrocycles to Enhance Penetration Across the Blood-Brain Barrier</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>139</i></span>,  <span class="NLM_fpage">15628</span>– <span class="NLM_lpage">15631</span>, <span class="refDoi"> DOI: 10.1021/jacs.7b09790</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.7b09790" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1amtL7O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=139&publication_year=2017&pages=15628-15631&author=C.+M.+Fadzenauthor=J.+M.+Wolfeauthor=C.-F.+Choauthor=E.+A.+Chioccaauthor=S.+E.+Lawlerauthor=B.+L.+Pentelute&title=Perfluoroarene-Based+Peptide+Macrocycles+to+Enhance+Penetration+Across+the+Blood-Brain+Barrier&doi=10.1021%2Fjacs.7b09790"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">Perfluoroarene-Based Peptide Macrocycles To Enhance Penetration across the Blood-Brain Barrier</span></div><div class="casAuthors">Fadzen, Colin M.; Wolfe, Justin M.; Cho, Choi-Fong; Chiocca, E. Antonio; Lawler, Sean E.; Pentelute, Bradley L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">139</span>
        (<span class="NLM_cas:issue">44</span>),
    <span class="NLM_cas:pages">15628-15631</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Here we describe the utility of peptide macrocyclization through perfluoroaryl-cysteine SNAr chem. in facilitating crossing the blood-brain barrier.  Multiple macrocyclic analogs of the peptide transportan-10 were investigated that displayed increased uptake in two different cell lines and improved proteolytic stability.  One of these analogs (M13) exhibited substantially increased delivery across a cellular spheroid model of the blood-brain barrier.  Through in vivo and ex vivo imaging of mouse brains, we demonstrate that this perfluoroarene-based macrocycle of TP10 exhibits increased penetration of the brain parenchyma following i.v. administration in mice.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplSoOwYi8wu7Vg90H21EOLACvtfcHk0lhkBAwKdQNtaA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1amtL7O&md5=9b5d8d1e53c7b94b3e0ff88096d2b96e</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1021%2Fjacs.7b09790&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.7b09790%26sid%3Dliteratum%253Aachs%26aulast%3DFadzen%26aufirst%3DC.%2BM.%26aulast%3DWolfe%26aufirst%3DJ.%2BM.%26aulast%3DCho%26aufirst%3DC.-F.%26aulast%3DChiocca%26aufirst%3DE.%2BA.%26aulast%3DLawler%26aufirst%3DS.%2BE.%26aulast%3DPentelute%26aufirst%3DB.%2BL.%26atitle%3DPerfluoroarene-Based%2520Peptide%2520Macrocycles%2520to%2520Enhance%2520Penetration%2520Across%2520the%2520Blood-Brain%2520Barrier%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2017%26volume%3D139%26spage%3D15628%26epage%3D15631%26doi%3D10.1021%2Fjacs.7b09790" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref68"><div class="reference"><strong class="refLabel"><a href="#ref68" class="rightTabRefNumLink">68</a></strong><div class="NLM_citation" id="rightTab-cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jo, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meinhardt, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kulkarni, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Low, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeGrado, W. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenbaum, D. C.</span></span> <span> </span><span class="NLM_article-title">Development of α-Helical Calpain Probes by Mimicking a Natural Protein-Protein Interaction</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>134</i></span>,  <span class="NLM_fpage">17704</span>– <span class="NLM_lpage">17713</span>, <span class="refDoi"> DOI: 10.1021/ja307599z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja307599z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhtl2lsrfK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=134&publication_year=2012&pages=17704-17713&author=H.+Joauthor=N.+Meinhardtauthor=Y.+Wuauthor=S.+Kulkarniauthor=X.+Huauthor=K.+E.+Lowauthor=P.+L.+Daviesauthor=W.+F.+DeGradoauthor=D.+C.+Greenbaum&title=Development+of+%CE%B1-Helical+Calpain+Probes+by+Mimicking+a+Natural+Protein-Protein+Interaction&doi=10.1021%2Fja307599z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">Development of α-Helical Calpain Probes by Mimicking a Natural Protein-Protein Interaction</span></div><div class="casAuthors">Jo, Hyunil; Meinhardt, Nataline; Wu, Yibing; Kulkarni, Swapnil; Hu, Xiaozhen; Low, Kristin E.; Davies, Peter L.; DeGrado, William F.; Greenbaum, Doron C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">134</span>
        (<span class="NLM_cas:issue">42</span>),
    <span class="NLM_cas:pages">17704-17713</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A highly specific inhibitor of calpain was designed by mimicking a natural protein-protein interaction between calpain and its endogenous inhibitor calpastatin.  To enable this goal a new method of stabilizing an α-helix in a small peptide was established by screening 24 com. available cross-linkers for successful cysteine alkylation in a model peptide sequence.  The effects of crosslinking on the α-helicity of selected peptides were examd. by CD and NMR spectroscopy, and revealed structurally rigid cross-linkers to be the best at stabilizing α-helixes.  This strategy was applied to the design of inhibitors of calpain that are based on calpastatin, an intrinsically unstable polypeptide that becomes structured upon binding to the enzyme.  A two-turn α-helix that binds proximal to the active site cleft was stabilized, resulting in a potent and selective inhibitor for calpain.  The utility of this inhibitor was further expanded by developing irreversible calpain family activity-based probes (ABPs), which retained the specificity of the stabilized helical inhibitor.  It is believed that the inhibitor and ABPs will be useful for future investigation of calpains, while the crosslinking technique will enable exploration of other protein-protein interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpigseq6FFUU7Vg90H21EOLACvtfcHk0lhkBAwKdQNtaA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhtl2lsrfK&md5=2824ee6e61d4bb0e4202696fc57f1fa0</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1021%2Fja307599z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja307599z%26sid%3Dliteratum%253Aachs%26aulast%3DJo%26aufirst%3DH.%26aulast%3DMeinhardt%26aufirst%3DN.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DKulkarni%26aufirst%3DS.%26aulast%3DHu%26aufirst%3DX.%26aulast%3DLow%26aufirst%3DK.%2BE.%26aulast%3DDavies%26aufirst%3DP.%2BL.%26aulast%3DDeGrado%26aufirst%3DW.%2BF.%26aulast%3DGreenbaum%26aufirst%3DD.%2BC.%26atitle%3DDevelopment%2520of%2520%25CE%25B1-Helical%2520Calpain%2520Probes%2520by%2520Mimicking%2520a%2520Natural%2520Protein-Protein%2520Interaction%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2012%26volume%3D134%26spage%3D17704%26epage%3D17713%26doi%3D10.1021%2Fja307599z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref69"><div class="reference"><strong class="refLabel"><a href="#ref69" class="rightTabRefNumLink">69</a></strong><div class="NLM_citation" id="rightTab-cit69"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Peraro, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siegert, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kritzer, J. A.</span></span> <span> </span><span class="NLM_article-title">Conformational Restriction of Peptides Using Dithiol Bis-Alkylation</span>. <i>Methods Enzymol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>580</i></span>,  <span class="NLM_fpage">303</span>– <span class="NLM_lpage">332</span>, <span class="refDoi"> DOI: 10.1016/bs.mie.2016.05.035</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1016%2Fbs.mie.2016.05.035" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=27586339" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A280%3ADC%252BC2szntVGktw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=580&publication_year=2016&pages=303-332&author=L.+Peraroauthor=T.+R.+Siegertauthor=J.+A.+Kritzer&title=Conformational+Restriction+of+Peptides+Using+Dithiol+Bis-Alkylation&doi=10.1016%2Fbs.mie.2016.05.035"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">Conformational Restriction of Peptides Using Dithiol Bis-Alkylation</span></div><div class="casAuthors">Peraro L; Siegert T R; Kritzer J A</div><div class="citationInfo"><span class="NLM_cas:title">Methods in enzymology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">580</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">303-32</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Macrocyclic peptides are highly promising as inhibitors of protein-protein interactions.  While many bond-forming reactions can be used to make cyclic peptides, most have limitations that make this chemical space challenging to access.  Recently, a variety of cysteine alkylation reactions have been used in rational design and library approaches for cyclic peptide discovery and development.  We and others have found that this chemistry is versatile and robust enough to produce a large variety of conformationally constrained cyclic peptides.  In this chapter, we describe applications, methods, mechanistic insights, and troubleshooting for dithiol bis-alkylation reactions for the production of cyclic peptides.  This method for efficient solution-phase macrocyclization is highly useful for the rapid production and screening of loop-based inhibitors of protein-protein interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRdlYGRAk0ein6oPH0mYWL1fW6udTcc2eYQGz3-JU6p-7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2szntVGktw%253D%253D&md5=5afc27d7b85b89daa26f140a2d3e47d1</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1016%2Fbs.mie.2016.05.035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fbs.mie.2016.05.035%26sid%3Dliteratum%253Aachs%26aulast%3DPeraro%26aufirst%3DL.%26aulast%3DSiegert%26aufirst%3DT.%2BR.%26aulast%3DKritzer%26aufirst%3DJ.%2BA.%26atitle%3DConformational%2520Restriction%2520of%2520Peptides%2520Using%2520Dithiol%2520Bis-Alkylation%26jtitle%3DMethods%2520Enzymol.%26date%3D2016%26volume%3D580%26spage%3D303%26epage%3D332%26doi%3D10.1016%2Fbs.mie.2016.05.035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref70"><div class="reference"><strong class="refLabel"><a href="#ref70" class="rightTabRefNumLink">70</a></strong><div class="NLM_citation" id="rightTab-cit70"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rojas, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pentelute, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buchwald, S. L.</span></span> <span> </span><span class="NLM_article-title">Water-Soluble Palladium Reagents for Cysteine S-Arylation under Ambient Aqueous Conditions</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">4263</span>– <span class="NLM_lpage">4266</span>, <span class="refDoi"> DOI: 10.1021/acs.orglett.7b01911</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.orglett.7b01911" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1OlsbjN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2017&pages=4263-4266&author=A.+J.+Rojasauthor=B.+L.+Penteluteauthor=S.+L.+Buchwald&title=Water-Soluble+Palladium+Reagents+for+Cysteine+S-Arylation+under+Ambient+Aqueous+Conditions&doi=10.1021%2Facs.orglett.7b01911"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Water-Soluble Palladium Reagents for Cysteine S-Arylation under Ambient Aqueous Conditions</span></div><div class="casAuthors">Rojas, Anthony J.; Pentelute, Bradley L.; Buchwald, Stephen L.</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">4263-4266</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7052</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Authors report the use of a sulfonated biarylphosphine ligand (sSPhos) to promote the chemoselective modification of cysteine contg. proteins and peptides with palladium reagents in aq. medium.  The use of sSPhos allowed for the isolation of several air-stable and water-sol. mono- and bis-palladium reagents, which were used in an improved protocol for the rapid S-arylation of cysteines under benign and physiol. relevant conditions.  The cosolvent-free aq. conditions were applied to the conjugation of a variety of biomols. with affinity tags, heterocycles, fluorophores, and functional handles.  Addnl., bis-palladium reagents were used to perform macrocyclization of peptides bearing two cysteine residues.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpAr4KwsDJ5ZbVg90H21EOLACvtfcHk0lhkBAwKdQNtaA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1OlsbjN&md5=0f4c8407d151c034579097b745d81f10</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1021%2Facs.orglett.7b01911&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.orglett.7b01911%26sid%3Dliteratum%253Aachs%26aulast%3DRojas%26aufirst%3DA.%2BJ.%26aulast%3DPentelute%26aufirst%3DB.%2BL.%26aulast%3DBuchwald%26aufirst%3DS.%2BL.%26atitle%3DWater-Soluble%2520Palladium%2520Reagents%2520for%2520Cysteine%2520S-Arylation%2520under%2520Ambient%2520Aqueous%2520Conditions%26jtitle%3DOrg.%2520Lett.%26date%3D2017%26volume%3D19%26spage%3D4263%26epage%3D4266%26doi%3D10.1021%2Facs.orglett.7b01911" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref71"><div class="reference"><strong class="refLabel"><a href="#ref71" class="rightTabRefNumLink">71</a></strong><div class="NLM_citation" id="rightTab-cit71"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rojas, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vinogradova, E. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buchwald, N. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reilly, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pentelute, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buchwald, S. L.</span></span> <span> </span><span class="NLM_article-title">Divergent Unprotected Peptide Macrocyclisation by Palladium-Mediated Cysteine Arylation</span>. <i>Chem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">4257</span>– <span class="NLM_lpage">4263</span>, <span class="refDoi"> DOI: 10.1039/C6SC05454D</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1039%2FC6SC05454D" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=29081961" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkvVOgtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=4257-4263&author=A.+J.+Rojasauthor=C.+Zhangauthor=E.+V.+Vinogradovaauthor=N.+H.+Buchwaldauthor=J.+Reillyauthor=B.+L.+Penteluteauthor=S.+L.+Buchwald&title=Divergent+Unprotected+Peptide+Macrocyclisation+by+Palladium-Mediated+Cysteine+Arylation&doi=10.1039%2FC6SC05454D"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">Divergent unprotected peptide macrocyclisation by palladium-mediated cysteine arylation</span></div><div class="casAuthors">Rojas, Anthony J.; Zhang, Chi; Vinogradova, Ekaterina V.; Buchwald, Nathan H.; Reilly, John; Pentelute, Bradley L.; Buchwald, Stephen L.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Science</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">4257-4263</span>CODEN:
                <span class="NLM_cas:coden">CSHCCN</span>;
        ISSN:<span class="NLM_cas:issn">2041-6520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Macrocyclic peptides are important therapeutic candidates due to their improved physicochem. properties in comparison to their linear counterparts.  Here we detail a method for a divergent macrocyclisation of unprotected peptides by crosslinking two cysteine residues with bis-palladium organometallic reagents.  These synthetic intermediates are prepd. in a single step from com. available aryl bis-halides.  Two bioactive linear peptides with cysteine residues at i, i + 4 and i, i + 7 positions, resp., were cyclized to introduce a diverse array of aryl and bi-aryl linkers.  These two series of macrocyclic peptides displayed similar linker-dependent lipophilicity, phospholipid affinity, and unique vol. of distributions.  Addnl., one of the bioactive peptides showed target binding affinity that was predominantly affected by the length of the linker.  Collectively, this divergent strategy allowed rapid and convenient access to various aryl linkers, enabling the systematic evaluation of the effect of appending unit on the medicinal properties of macrocyclic peptides.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr9-XconY4mOrVg90H21EOLACvtfcHk0ljrlkAUMpMBMQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkvVOgtbk%253D&md5=9eaa28f51178fa528e5bf959b0c16222</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1039%2FC6SC05454D&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC6SC05454D%26sid%3Dliteratum%253Aachs%26aulast%3DRojas%26aufirst%3DA.%2BJ.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DVinogradova%26aufirst%3DE.%2BV.%26aulast%3DBuchwald%26aufirst%3DN.%2BH.%26aulast%3DReilly%26aufirst%3DJ.%26aulast%3DPentelute%26aufirst%3DB.%2BL.%26aulast%3DBuchwald%26aufirst%3DS.%2BL.%26atitle%3DDivergent%2520Unprotected%2520Peptide%2520Macrocyclisation%2520by%2520Palladium-Mediated%2520Cysteine%2520Arylation%26jtitle%3DChem.%2520Sci.%26date%3D2017%26volume%3D8%26spage%3D4257%26epage%3D4263%26doi%3D10.1039%2FC6SC05454D" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref72"><div class="reference"><strong class="refLabel"><a href="#ref72" class="rightTabRefNumLink">72</a></strong><div class="NLM_citation" id="rightTab-cit72"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Spokoyny, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ling, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Y.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pentelute, B. L.</span></span> <span> </span><span class="NLM_article-title">A Perfluoroaryl-Cysteine S(N)Ar Chemistry Approach to Unprotected Peptide Stapling</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>135</i></span>,  <span class="NLM_fpage">5946</span>– <span class="NLM_lpage">5949</span>, <span class="refDoi"> DOI: 10.1021/ja400119t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja400119t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlsV2gtrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=135&publication_year=2013&pages=5946-5949&author=A.+M.+Spokoynyauthor=Y.+Zouauthor=J.+J.+Lingauthor=H.+Yuauthor=Y.-S.+Linauthor=B.+L.+Pentelute&title=A+Perfluoroaryl-Cysteine+S%28N%29Ar+Chemistry+Approach+to+Unprotected+Peptide+Stapling&doi=10.1021%2Fja400119t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">A Perfluoroaryl-Cysteine SNAr Chemistry Approach to Unprotected Peptide Stapling</span></div><div class="casAuthors">Spokoyny, Alexander M.; Zou, Yekui; Ling, Jingjing J.; Yu, Hongtao; Lin, Yu-Shan; Pentelute, Bradley L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">135</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">5946-5949</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The authors report the discovery of a facile transformation between perfluoroarom. mols. and a cysteine thiolate, which is arylated at room temp.  This new approach enabled the authors to selectively modify cysteine residues in unprotected peptides, providing access to variants contg. rigid perfluoroarom. staples.  This stapling modification performed on a peptide sequence designed to bind the C-terminal domain of an HIV-1 (C-CA) showed enhancement in binding, cell permeability, and proteolytic stability properties, as compared to the unstapled analog.  Importantly, chem. stability of the formed staples allowed the authors to use this motif in the native chem. ligation-mediated synthesis of a small protein affibody that is capable of binding the human epidermal growth factor 2 receptor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqw-oA_BVUKOrVg90H21EOLACvtfcHk0ljrlkAUMpMBMQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXlsV2gtrc%253D&md5=126ac191059b0703757ef7838d90e183</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1021%2Fja400119t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja400119t%26sid%3Dliteratum%253Aachs%26aulast%3DSpokoyny%26aufirst%3DA.%2BM.%26aulast%3DZou%26aufirst%3DY.%26aulast%3DLing%26aufirst%3DJ.%2BJ.%26aulast%3DYu%26aufirst%3DH.%26aulast%3DLin%26aufirst%3DY.-S.%26aulast%3DPentelute%26aufirst%3DB.%2BL.%26atitle%3DA%2520Perfluoroaryl-Cysteine%2520S%2528N%2529Ar%2520Chemistry%2520Approach%2520to%2520Unprotected%2520Peptide%2520Stapling%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2013%26volume%3D135%26spage%3D5946%26epage%3D5949%26doi%3D10.1021%2Fja400119t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref73"><div class="reference"><strong class="refLabel"><a href="#ref73" class="rightTabRefNumLink">73</a></strong><div class="NLM_citation" id="rightTab-cit73"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vinogradova, E. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spokoyny, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pentelute, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buchwald, S. L.</span></span> <span> </span><span class="NLM_article-title">Organometallic Palladium Reagents for Cysteine Bioconjugation</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>526</i></span>,  <span class="NLM_fpage">687</span>– <span class="NLM_lpage">691</span>, <span class="refDoi"> DOI: 10.1038/nature15739</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1038%2Fnature15739" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=26511579" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslCnu7zK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=526&publication_year=2015&pages=687-691&author=E.+V.+Vinogradovaauthor=C.+Zhangauthor=A.+M.+Spokoynyauthor=B.+L.+Penteluteauthor=S.+L.+Buchwald&title=Organometallic+Palladium+Reagents+for+Cysteine+Bioconjugation&doi=10.1038%2Fnature15739"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">Organometallic palladium reagents for cysteine bioconjugation</span></div><div class="casAuthors">Vinogradova, Ekaterina V.; Zhang, Chi; Spokoyny, Alexander M.; Pentelute, Bradley L.; Buchwald, Stephen L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">526</span>
        (<span class="NLM_cas:issue">7575</span>),
    <span class="NLM_cas:pages">687-691</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Peptides and proteins, contg. cysteine amino acid, were labeled by conjugation of the SH-group with arylpalladium 2-biphenylylphosphine hemilabile reagent, causing arylation the mercapto-group.  Reactions based on transition metals have found wide use in org. synthesis, in particular for the functionalization of small mols.  However, there are very few reports of using transition-metal-based reactions to modify complex biomols., which is due to the need for stringent reaction conditions (for example, aq. media, low temp. and mild pH) and the existence of multiple reactive functional groups found in biomols.  Here we report that palladium(II) complexes can be used for efficient and highly selective cysteine conjugation (bioconjugation) reactions that are rapid and robust under a range of bio-compatible reaction conditions.  The straightforward synthesis of the palladium reagents from diverse and easily accessible aryl halide and trifluoromethanesulfonate precursors makes the method highly practical, providing access to a large structural space for protein modification.  The resulting aryl bioconjugates are stable towards acids, bases, oxidants and external thiol nucleophiles.  The broad utility of the bioconjugation platform was further corroborated by the synthesis of new classes of stapled peptides and antibody-drug conjugates.  These palladium complexes show potential as benchtop reagents for diverse bioconjugation applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLRNU9i0UtEbVg90H21EOLACvtfcHk0ljrlkAUMpMBMQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslCnu7zK&md5=8fbb382c94600af312bb007d31b63056</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1038%2Fnature15739&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature15739%26sid%3Dliteratum%253Aachs%26aulast%3DVinogradova%26aufirst%3DE.%2BV.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DSpokoyny%26aufirst%3DA.%2BM.%26aulast%3DPentelute%26aufirst%3DB.%2BL.%26aulast%3DBuchwald%26aufirst%3DS.%2BL.%26atitle%3DOrganometallic%2520Palladium%2520Reagents%2520for%2520Cysteine%2520Bioconjugation%26jtitle%3DNature%26date%3D2015%26volume%3D526%26spage%3D687%26epage%3D691%26doi%3D10.1038%2Fnature15739" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref74"><div class="reference"><strong class="refLabel"><a href="#ref74" class="rightTabRefNumLink">74</a></strong><div class="NLM_citation" id="rightTab-cit74"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morales-Sanfrutos, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Angelini, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cutting, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heinis, C.</span></span> <span> </span><span class="NLM_article-title">Structurally Diverse Cyclisation Linkers Impose Different Backbone Conformations in Bicyclic Peptides</span>. <i>ChemBioChem</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">1032</span>– <span class="NLM_lpage">1038</span>, <span class="refDoi"> DOI: 10.1002/cbic.201200049</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1002%2Fcbic.201200049" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=22492661" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC38Xlt1Kltb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2012&pages=1032-1038&author=S.+Chenauthor=J.+Morales-Sanfrutosauthor=A.+Angeliniauthor=B.+Cuttingauthor=C.+Heinis&title=Structurally+Diverse+Cyclisation+Linkers+Impose+Different+Backbone+Conformations+in+Bicyclic+Peptides&doi=10.1002%2Fcbic.201200049"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">Structurally Diverse Cyclisation Linkers Impose Different Backbone Conformations in Bicyclic Peptides</span></div><div class="casAuthors">Chen, Shiyu; Morales-Sanfrutos, Julia; Angelini, Alessandro; Cutting, Brian; Heinis, Christian</div><div class="citationInfo"><span class="NLM_cas:title">ChemBioChem</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1032-1038</span>CODEN:
                <span class="NLM_cas:coden">CBCHFX</span>;
        ISSN:<span class="NLM_cas:issn">1439-4227</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Combinatorial libraries of structurally diverse peptide macrocycles offer a rich source for the development of high-affinity ligands to targets of interest.  In this work we have developed linkers for the generation of genetically encoded bicyclic peptides and tested whether the peptides cyclised by them have significant variations in their backbone conformations.  Two new cyclisation reagents, each contg. three thiol-reactive groups, efficiently and selectively cyclised linear peptides contg. three cysteine moieties.  When the mesitylene linker of the bicyclic peptide PK15, a potent inhibitor of plasma kallikrein (Ki=2 nM), was replaced by the new linkers, its inhibitory activity dropped by a factor of more than 1000, suggesting that the linkers impose different conformations on the peptide.  Indeed, structural anal. by soln.-state NMR revealed different NOE constraints in the three bicyclic peptides, indicating that these relatively small linkers at the centers of bicyclic peptide structures significantly influence the conformations of the peptides.  These results demonstrate the prominent structural role of linkers in peptide macrocycles and suggest that application of different cyclisation linkers in a combinatorial fashion could be an attractive means to generate topol. diverse macrocycle libraries.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphdCp8KV5HeLVg90H21EOLACvtfcHk0lifzHhf0Kj6RQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xlt1Kltb4%253D&md5=ddb3f7e338a22ea1a1e3ced71624781d</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1002%2Fcbic.201200049&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbic.201200049%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DS.%26aulast%3DMorales-Sanfrutos%26aufirst%3DJ.%26aulast%3DAngelini%26aufirst%3DA.%26aulast%3DCutting%26aufirst%3DB.%26aulast%3DHeinis%26aufirst%3DC.%26atitle%3DStructurally%2520Diverse%2520Cyclisation%2520Linkers%2520Impose%2520Different%2520Backbone%2520Conformations%2520in%2520Bicyclic%2520Peptides%26jtitle%3DChemBioChem%26date%3D2012%26volume%3D13%26spage%3D1032%26epage%3D1038%26doi%3D10.1002%2Fcbic.201200049" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref75"><div class="reference"><strong class="refLabel"><a href="#ref75" class="rightTabRefNumLink">75</a></strong><div class="NLM_citation" id="rightTab-cit75"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Timmerman, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beld, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Puijk, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meloen, R. H.</span></span> <span> </span><span class="NLM_article-title">Rapid and Quantitative Cyclization of Multiple Peptide Loops onto Synthetic Scaffolds for Structural Mimicry of Protein Surfaces</span>. <i>ChemBioChem</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">821</span>– <span class="NLM_lpage">824</span>, <span class="refDoi"> DOI: 10.1002/cbic.200400374</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1002%2Fcbic.200400374" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=15812852" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BD2MXksFajsL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2005&pages=821-824&author=P.+Timmermanauthor=J.+Beldauthor=W.+C.+Puijkauthor=R.+H.+Meloen&title=Rapid+and+Quantitative+Cyclization+of+Multiple+Peptide+Loops+onto+Synthetic+Scaffolds+for+Structural+Mimicry+of+Protein+Surfaces&doi=10.1002%2Fcbic.200400374"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">Rapid and quantitative cyclization of multiple peptide loops onto synthetic scaffolds for structural mimicry of protein surfaces</span></div><div class="casAuthors">Timmerman, Peter; Beld, Joris; Puijk, Wouter C.; Meloen, Rob H.</div><div class="citationInfo"><span class="NLM_cas:title">ChemBioChem</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">821-824</span>CODEN:
                <span class="NLM_cas:coden">CBCHFX</span>;
        ISSN:<span class="NLM_cas:issn">1439-4227</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">This work describes cyclization reaction of cysteine-contg. linear peptides onto bromomethylated arom. scaffolds without requiring side-chain protections.  The reaction is extremely fast and clean and runs to completion in less than 15 min at room temp.  The cyclic peptides are good mimics of protein surfaces.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqfSQnHmkXl_LVg90H21EOLACvtfcHk0lifzHhf0Kj6RQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXksFajsL0%253D&md5=52a1d3bb0bc8a483a683cdb900525c0f</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1002%2Fcbic.200400374&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbic.200400374%26sid%3Dliteratum%253Aachs%26aulast%3DTimmerman%26aufirst%3DP.%26aulast%3DBeld%26aufirst%3DJ.%26aulast%3DPuijk%26aufirst%3DW.%2BC.%26aulast%3DMeloen%26aufirst%3DR.%2BH.%26atitle%3DRapid%2520and%2520Quantitative%2520Cyclization%2520of%2520Multiple%2520Peptide%2520Loops%2520onto%2520Synthetic%2520Scaffolds%2520for%2520Structural%2520Mimicry%2520of%2520Protein%2520Surfaces%26jtitle%3DChemBioChem%26date%3D2005%26volume%3D6%26spage%3D821%26epage%3D824%26doi%3D10.1002%2Fcbic.200400374" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref76"><div class="reference"><strong class="refLabel"><a href="#ref76" class="rightTabRefNumLink">76</a></strong><div class="NLM_citation" id="rightTab-cit76"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Diderich, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertoldo, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dessen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khan, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pizzitola, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Held, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huelsken, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heinis, C.</span></span> <span> </span><span class="NLM_article-title">Phage Selection of Chemically Stabilized α-Helical Peptide Ligands</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">1422</span>– <span class="NLM_lpage">1427</span>, <span class="refDoi"> DOI: 10.1021/acschembio.5b00963</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b00963" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC28XjsVagtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=1422-1427&author=P.+Diderichauthor=D.+Bertoldoauthor=P.+Dessenauthor=M.+M.+Khanauthor=I.+Pizzitolaauthor=W.+Heldauthor=J.+Huelskenauthor=C.+Heinis&title=Phage+Selection+of+Chemically+Stabilized+%CE%B1-Helical+Peptide+Ligands&doi=10.1021%2Facschembio.5b00963"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit76R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">Phage Selection of Chemically Stabilized α-Helical Peptide Ligands</span></div><div class="casAuthors">Diderich, Philippe; Bertoldo, Davide; Dessen, Pierre; Khan, Maola M.; Pizzitola, Irene; Held, Werner; Huelsken, Joerg; Heinis, Christian</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1422-1427</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Short α-helical peptides stabilized by linkages between constituent amino acids offer an attractive format for ligand development.  In recent years, a range of excellent ligands based on stabilized α-helixes were generated by rational design using α-helical peptides of natural proteins as templates.  Herein, we developed a method to engineer chem. stabilized α-helical ligands in a combinatorial fashion.  In brief, peptides contg. cysteines in position i and i + 4 are genetically encoded by phage display, the cysteines are modified with chem. bridges to impose α-helical conformations, and binders are isolated by affinity selection.  We applied the strategy to affinity mature an α-helical peptide binding β-catenin.  We succeeded in developing ligands with Kd's as low as 5.2 nM, having >200-fold improved affinity.  The strategy is generally applicable for affinity maturation of any α-helical peptide.  Compared to hydrocarbon stapled peptides, the herein evolved thioether-bridged peptide ligands can be synthesized more easily, as no unnatural amino acids are required and the cyclization reaction is more efficient and yields no stereoisomers.  A further advantage of the thioether-bridged peptide ligands is that they can be expressed recombinantly as fusion proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXjSfa_cgePrVg90H21EOLACvtfcHk0lifzHhf0Kj6RQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XjsVagtr8%253D&md5=a25d24d4733a0d89f4797c5313ef1870</span></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b00963&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b00963%26sid%3Dliteratum%253Aachs%26aulast%3DDiderich%26aufirst%3DP.%26aulast%3DBertoldo%26aufirst%3DD.%26aulast%3DDessen%26aufirst%3DP.%26aulast%3DKhan%26aufirst%3DM.%2BM.%26aulast%3DPizzitola%26aufirst%3DI.%26aulast%3DHeld%26aufirst%3DW.%26aulast%3DHuelsken%26aufirst%3DJ.%26aulast%3DHeinis%26aufirst%3DC.%26atitle%3DPhage%2520Selection%2520of%2520Chemically%2520Stabilized%2520%25CE%25B1-Helical%2520Peptide%2520Ligands%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2016%26volume%3D11%26spage%3D1422%26epage%3D1427%26doi%3D10.1021%2Facschembio.5b00963" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref77"><div class="reference"><strong class="refLabel"><a href="#ref77" class="rightTabRefNumLink">77</a></strong><div class="NLM_citation" id="rightTab-cit77"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kale, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villequey, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong, X.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zorzi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deyle, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heinis, C.</span></span> <span> </span><span class="NLM_article-title">Cyclization of Peptides with Two Chemical Bridges Affords Large Scaffold Diversities</span>. <i>Nat. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">715</span>– <span class="NLM_lpage">723</span>, <span class="refDoi"> DOI: 10.1038/s41557-018-0042-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1038%2Fs41557-018-0042-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=29713035" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC1cXovVCkurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2018&pages=715-723&author=S.+S.+Kaleauthor=C.+Villequeyauthor=X.-D.+Kongauthor=A.+Zorziauthor=K.+Deyleauthor=C.+Heinis&title=Cyclization+of+Peptides+with+Two+Chemical+Bridges+Affords+Large+Scaffold+Diversities&doi=10.1038%2Fs41557-018-0042-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit77R"><div class="casContent"><span class="casTitleNuber">77</span><div class="casTitle"><span class="NLM_cas:atitle">Cyclization of peptides with two chemical bridges affords large scaffold diversities</span></div><div class="casAuthors">Kale, Sangram S.; Villequey, Camille; Kong, Xu-Dong; Zorzi, Alessandro; Deyle, Kaycie; Heinis, Christian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">715-723</span>CODEN:
                <span class="NLM_cas:coden">NCAHBB</span>;
        ISSN:<span class="NLM_cas:issn">1755-4330</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Successful screening campaigns depend on large and structurally diverse collections of compds.  In macrocycle screening, variation of the mol. scaffold is important for structural diversity, but so far it has been challenging to diversify this aspect in large combinatorial libraries.  Here, the authors report the cyclization of peptides with two chem. bridges to provide rapid access to thousands of different macrocyclic scaffolds in libraries that are easy to synthesize, screen and decode.  Application of this strategy to phage-encoded libraries allowed for the screening of an unprecedented structural diversity of macrocycles against plasma kallikrein, which is important in the swelling disorder hereditary angioedema.  These libraries yielded inhibitors with remarkable binding properties (subnanomolar Ki, >1000-fold selectivity) despite the small mol. mass (∼1200 Da).  An interlaced bridge format characteristic of this strategy provided high proteolytic stability (t1/2 in plasma of >3 days), making double-bridged peptides potentially amenable to topical or oral delivery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpmZOAVFLZRH7Vg90H21EOLACvtfcHk0lifzHhf0Kj6RQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXovVCkurg%253D&md5=66161a855b4d8317fb456a36765d9766</span></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=10.1038%2Fs41557-018-0042-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41557-018-0042-7%26sid%3Dliteratum%253Aachs%26aulast%3DKale%26aufirst%3DS.%2BS.%26aulast%3DVillequey%26aufirst%3DC.%26aulast%3DKong%26aufirst%3DX.-D.%26aulast%3DZorzi%26aufirst%3DA.%26aulast%3DDeyle%26aufirst%3DK.%26aulast%3DHeinis%26aufirst%3DC.%26atitle%3DCyclization%2520of%2520Peptides%2520with%2520Two%2520Chemical%2520Bridges%2520Affords%2520Large%2520Scaffold%2520Diversities%26jtitle%3DNat.%2520Chem.%26date%3D2018%26volume%3D10%26spage%3D715%26epage%3D723%26doi%3D10.1038%2Fs41557-018-0042-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref78"><div class="reference"><strong class="refLabel"><a href="#ref78" class="rightTabRefNumLink">78</a></strong><div class="NLM_citation" id="rightTab-cit78"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kong, X.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriya, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carle, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pojer, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abriata, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deyle, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heinis, C.</span></span> <span> </span><span class="NLM_article-title">De Novo Development of Proteolytically Resistant Therapeutic Peptides for Oral Administration</span>. <i>Nat. Biomed. Eng.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">560</span>– <span class="NLM_lpage">571</span>, <span class="refDoi"> DOI: 10.1038/s41551-020-0556-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1038%2Fs41551-020-0556-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=32393891" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BB3cXpt1Oqur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2020&pages=560-571&author=X.-D.+Kongauthor=J.+Moriyaauthor=V.+Carleauthor=F.+Pojerauthor=L.+A.+Abriataauthor=K.+Deyleauthor=C.+Heinis&title=De+Novo+Development+of+Proteolytically+Resistant+Therapeutic+Peptides+for+Oral+Administration&doi=10.1038%2Fs41551-020-0556-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit78R"><div class="casContent"><span class="casTitleNuber">78</span><div class="casTitle"><span class="NLM_cas:atitle">De novo development of proteolytically resistant therapeutic peptides for oral administration</span></div><div class="casAuthors">Kong, Xu-Dong; Moriya, Jun; Carle, Vanessa; Pojer, Florence; Abriata, Luciano A.; Deyle, Kaycie; Heinis, Christian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biomedical Engineering</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">560-571</span>CODEN:
                <span class="NLM_cas:coden">NBEAB3</span>;
        ISSN:<span class="NLM_cas:issn">2157-846X</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Abstr.: The oral administration of peptide drugs is hampered by their metabolic instability and limited intestinal uptake.  Here, we describe a method for the generation of small target-specific peptides (less than 1,600 Da in size) that resist gastrointestinal proteases.  By using phage display to screen large libraries of genetically encoded double-bridged peptides on protease-resistant fd bacteriophages, we generated a peptide inhibitor of the coagulation Factor XIa with nanomolar affinity that resisted gastrointestinal proteases in all regions of the gastrointestinal tract of mice after oral administration, enabling more than 30% of the peptide to remain intact, and small quantities of it to reach the blood circulation.  We also developed a gastrointestinal-protease-resistant peptide antagonist for the interleukin-23 receptor, which has a role in the pathogenesis of Crohn's disease and ulcerative colitis.  The de novo generation of targeted peptides that resist proteolytic degrdn. in the gastrointestinal tract should help the development of effective peptides for oral delivery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGowjbT2rK-aWrVg90H21EOLACvtfcHk0lhlbeC3EnTs_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXpt1Oqur0%253D&md5=d7db093aa00c687da06c68cc452680f4</span></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=10.1038%2Fs41551-020-0556-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41551-020-0556-3%26sid%3Dliteratum%253Aachs%26aulast%3DKong%26aufirst%3DX.-D.%26aulast%3DMoriya%26aufirst%3DJ.%26aulast%3DCarle%26aufirst%3DV.%26aulast%3DPojer%26aufirst%3DF.%26aulast%3DAbriata%26aufirst%3DL.%2BA.%26aulast%3DDeyle%26aufirst%3DK.%26aulast%3DHeinis%26aufirst%3DC.%26atitle%3DDe%2520Novo%2520Development%2520of%2520Proteolytically%2520Resistant%2520Therapeutic%2520Peptides%2520for%2520Oral%2520Administration%26jtitle%3DNat.%2520Biomed.%2520Eng.%26date%3D2020%26volume%3D4%26spage%3D560%26epage%3D571%26doi%3D10.1038%2Fs41551-020-0556-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref79"><div class="reference"><strong class="refLabel"><a href="#ref79" class="rightTabRefNumLink">79</a></strong><div class="NLM_citation" id="rightTab-cit79"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baeriswyl, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heinis, C.</span></span> <span> </span><span class="NLM_article-title">Phage Selection of Cyclic Peptide Antagonists with Increased Stability toward Intestinal Proteases</span>. <i>Protein Eng., Des. Sel.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">81</span>– <span class="NLM_lpage">89</span>, <span class="refDoi"> DOI: 10.1093/protein/gzs085</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1093%2Fprotein%2Fgzs085" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=23100545" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVCisL%252FJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2013&pages=81-89&author=V.+Baeriswylauthor=C.+Heinis&title=Phage+Selection+of+Cyclic+Peptide+Antagonists+with+Increased+Stability+toward+Intestinal+Proteases&doi=10.1093%2Fprotein%2Fgzs085"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit79R"><div class="casContent"><span class="casTitleNuber">79</span><div class="casTitle"><span class="NLM_cas:atitle">Phage selection of cyclic peptide antagonists with increased stability toward intestinal proteases</span></div><div class="casAuthors">Baeriswyl, Vanessa; Heinis, Christian</div><div class="citationInfo"><span class="NLM_cas:title">Protein Engineering, Design & Selection</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">81-89</span>CODEN:
                <span class="NLM_cas:coden">PEDSBR</span>;
        ISSN:<span class="NLM_cas:issn">1741-0126</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The oral delivery of protein and peptide drugs is limited by their proteolytic degrdn. and the poor absorption across the intestinal epithelia.  In this work, we exposed a phage library of small bicyclic peptides (<1.5 kDa) to a pancreatic ext. of proteases prior to affinity selection to enrich binders with higher stability in the intestinal environment.  Panning with the therapeutic target plasma kallikrein yielded potent inhibitors (Kis between 5.6 and 336 nM) wherein bicyclic peptides isolated with proteolytic pressure were more stable.  A proline residue found in a specific position of several resistant bicyclic peptides proved to be a protective mark', rendering the bicyclic peptides resistant to significantly higher concns. of intestinal proteases while retaining essentially their inhibitory activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4LvGcBCGYWLVg90H21EOLACvtfcHk0lhlbeC3EnTs_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVCisL%252FJ&md5=75e20392fb64e8cf24baa3f88c2955f1</span></div><a href="/servlet/linkout?suffix=cit79&amp;dbid=16384&amp;doi=10.1093%2Fprotein%2Fgzs085&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fprotein%252Fgzs085%26sid%3Dliteratum%253Aachs%26aulast%3DBaeriswyl%26aufirst%3DV.%26aulast%3DHeinis%26aufirst%3DC.%26atitle%3DPhage%2520Selection%2520of%2520Cyclic%2520Peptide%2520Antagonists%2520with%2520Increased%2520Stability%2520toward%2520Intestinal%2520Proteases%26jtitle%3DProtein%2520Eng.%252C%2520Des.%2520Sel.%26date%3D2013%26volume%3D26%26spage%3D81%26epage%3D89%26doi%3D10.1093%2Fprotein%2Fgzs085" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref80"><div class="reference"><strong class="refLabel"><a href="#ref80" class="rightTabRefNumLink">80</a></strong><div class="NLM_citation" id="rightTab-cit80"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bernhagen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jungbluth, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quilis, N. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dostalek, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jalink, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Timmerman, P.</span></span> <span> </span><span class="NLM_article-title">Bicyclic RGD Peptides with Exquisite Selectivity for the Integrin Avβ3 Receptor Using a “Random Design” Approach</span>. <i>ACS Comb. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">198</span>– <span class="NLM_lpage">206</span>, <span class="refDoi"> DOI: 10.1021/acscombsci.8b00144</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acscombsci.8b00144" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A280%3ADC%252BB3cnpsF2hsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2019&pages=198-206&author=D.+Bernhagenauthor=V.+Jungbluthauthor=N.+G.+Quilisauthor=J.+Dostalekauthor=P.+B.+Whiteauthor=K.+Jalinkauthor=P.+Timmerman&title=Bicyclic+RGD+Peptides+with+Exquisite+Selectivity+for+the+Integrin+Av%CE%B23+Receptor+Using+a+%E2%80%9CRandom+Design%E2%80%9D+Approach&doi=10.1021%2Facscombsci.8b00144"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit80R"><div class="casContent"><span class="casTitleNuber">80</span><div class="casTitle"><span class="NLM_cas:atitle">Bicyclic RGD Peptides with Exquisite Selectivity for the Integrin αvβ3 Receptor Using a "Random Design" Approach</span></div><div class="casAuthors">Bernhagen Dominik; Timmerman Peter; Jungbluth Vanessa; Quilis Nestor Gisbert; Dostalek Jakub; White Paul B; Jalink Kees; Timmerman Peter</div><div class="citationInfo"><span class="NLM_cas:title">ACS combinatorial science</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">198-206</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">We describe the identification of bicyclic RGD peptides with high affinity and selectivity for integrin αvβ3 via high-throughput screening of partially randomized libraries.  Peptide libraries (672 different compounds) comprising the universal integrin-binding sequence Arg-Gly-Asp (RGD) in the first loop and a randomized sequence XXX (X being one of 18 canonical l-amino acids) in the second loop, both enclosed by either an l- or d-Cys residue, were converted to bicyclic peptides via reaction with 1,3,5-tris(bromomethyl)benzene (T3).  Screening of first-generation libraries yielded lead bicyclic inhibitors displaying submicromolar affinities for integrin αvβ3 (e.g., CT3HEQcT3RGDcT3, IC50 = 195 nM).  Next generation (second and third) libraries were obtained by partially varying the structure of the strongest lead inhibitors and screening for improved affinities and selectivities.  In this way, we identified the highly selective bicyclic αvβ3-binders CT3HPQcT3RGDcT3 (IC50 = 30 nM), CT3HPQCT3RGDcT3 (IC50 = 31 nM), and CT3HSQCT3RGDcT3 (IC50 = 42 nM) with affinities comparable to that of a knottin-RGD-type peptide (32 amino acids, IC50 = 38 nM) and outstanding selectivities over integrins αvβ5 (IC50 > 10000 nM) and α5β1 (IC50 > 10000 nM).  Affinity measurements using surface plasmon-enhanced fluorescence spectroscopy (SPFS) yielded Kd values of 0.4 and 0.6 nM for the Cy5-labeled bicycle CT3HPQcT3RGDcT3 and RGD "knottin" peptide, respectively.  In vitro staining of HT29 cells with Cy5-labeled bicycles using confocal microscopy revealed strong binding to integrins in their natural environment, which highlights the high potential of these peptides as markers of integrin expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSXZ9sPIKtyyyEJCPKhJPuqfW6udTcc2eamQ8czs0zG97ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3cnpsF2hsw%253D%253D&md5=8e2a6674879be4657dfff2792a9cb49c</span></div><a href="/servlet/linkout?suffix=cit80&amp;dbid=16384&amp;doi=10.1021%2Facscombsci.8b00144&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facscombsci.8b00144%26sid%3Dliteratum%253Aachs%26aulast%3DBernhagen%26aufirst%3DD.%26aulast%3DJungbluth%26aufirst%3DV.%26aulast%3DQuilis%26aufirst%3DN.%2BG.%26aulast%3DDostalek%26aufirst%3DJ.%26aulast%3DWhite%26aufirst%3DP.%2BB.%26aulast%3DJalink%26aufirst%3DK.%26aulast%3DTimmerman%26aufirst%3DP.%26atitle%3DBicyclic%2520RGD%2520Peptides%2520with%2520Exquisite%2520Selectivity%2520for%2520the%2520Integrin%2520Av%25CE%25B23%2520Receptor%2520Using%2520a%2520%25E2%2580%259CRandom%2520Design%25E2%2580%259D%2520Approach%26jtitle%3DACS%2520Comb.%2520Sci.%26date%3D2019%26volume%3D21%26spage%3D198%26epage%3D206%26doi%3D10.1021%2Facscombsci.8b00144" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref81"><div class="reference"><strong class="refLabel"><a href="#ref81" class="rightTabRefNumLink">81</a></strong><div class="NLM_citation" id="rightTab-cit81"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Heinis, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rutherford, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freund, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G.</span></span> <span> </span><span class="NLM_article-title">Phage-Encoded Combinatorial Chemical Libraries Based on Bicyclic Peptides</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">502</span>– <span class="NLM_lpage">507</span>, <span class="refDoi"> DOI: 10.1038/nchembio.184</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1038%2Fnchembio.184" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=19483697" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BD1MXms1altbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2009&pages=502-507&author=C.+Heinisauthor=T.+Rutherfordauthor=S.+Freundauthor=G.+Winter&title=Phage-Encoded+Combinatorial+Chemical+Libraries+Based+on+Bicyclic+Peptides&doi=10.1038%2Fnchembio.184"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit81R"><div class="casContent"><span class="casTitleNuber">81</span><div class="casTitle"><span class="NLM_cas:atitle">Phage-encoded combinatorial chemical libraries based on bicyclic peptides</span></div><div class="casAuthors">Heinis, Christian; Rutherford, Trevor; Freund, Stephan; Winter, Greg</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">502-507</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Here we describe a phage strategy for the selection of ligands based on bicyclic or linear peptides attached covalently to an org. core.  We designed peptide repertoires with three reactive cysteine residues, each spaced apart by several random amino acid residues, and we fused the repertoires to the phage gene-3-protein.  Conjugation with tris-(bromomethyl)benzene via the reactive cysteines generated repertoires of peptide conjugates with two peptide loops anchored to a mesitylene core.  Iterative affinity selections yielded several enzyme inhibitors; after further mutagenesis and selection, we were able to chem. synthesize a lead inhibitor (PK15; Ki = 1.5 nM) specific to human plasma kallikrein that efficiently interrupted the intrinsic coagulation pathway in human plasma tested ex vivo.  This approach offers a powerful means of generating and selecting bicyclic macrocycles (or if cleaved, linear derivs. thereof) as ligands poised at the interface of small-mol. drugs and biologics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZhJRZyldisLVg90H21EOLACvtfcHk0lgpnoAcL-4NIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXms1altbs%253D&md5=0634460f77b185fbcf85309c6dbf93e3</span></div><a href="/servlet/linkout?suffix=cit81&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.184&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.184%26sid%3Dliteratum%253Aachs%26aulast%3DHeinis%26aufirst%3DC.%26aulast%3DRutherford%26aufirst%3DT.%26aulast%3DFreund%26aufirst%3DS.%26aulast%3DWinter%26aufirst%3DG.%26atitle%3DPhage-Encoded%2520Combinatorial%2520Chemical%2520Libraries%2520Based%2520on%2520Bicyclic%2520Peptides%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2009%26volume%3D5%26spage%3D502%26epage%3D507%26doi%3D10.1038%2Fnchembio.184" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref82"><div class="reference"><strong class="refLabel"><a href="#ref82" class="rightTabRefNumLink">82</a></strong><div class="NLM_citation" id="rightTab-cit82"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulligan, V. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mravic, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lemmin, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ford, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silva, D.-A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baker, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeGrado, W. F.</span></span> <span> </span><span class="NLM_article-title">De Novo Design of Covalently Constrained Mesosize Protein Scaffolds with Unique Tertiary Structures</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>114</i></span>,  <span class="NLM_fpage">10852</span>– <span class="NLM_lpage">10857</span>, <span class="refDoi"> DOI: 10.1073/pnas.1710695114</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1073%2Fpnas.1710695114" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=28973862" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFKjtL%252FL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2017&pages=10852-10857&author=B.+Dangauthor=H.+Wuauthor=V.+K.+Mulliganauthor=M.+Mravicauthor=Y.+Wuauthor=T.+Lemminauthor=A.+Fordauthor=D.-A.+Silvaauthor=D.+Bakerauthor=W.+F.+DeGrado&title=De+Novo+Design+of+Covalently+Constrained+Mesosize+Protein+Scaffolds+with+Unique+Tertiary+Structures&doi=10.1073%2Fpnas.1710695114"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit82R"><div class="casContent"><span class="casTitleNuber">82</span><div class="casTitle"><span class="NLM_cas:atitle">De novo design of covalently constrained mesosize protein scaffolds with unique tertiary structures</span></div><div class="casAuthors">Dang, Bobo; Wu, Haifan; Mulligan, Vikram Khipple; Mravic, Marco; Wu, Yibing; Lemmin, Thomas; Ford, Alexander; Silva, Daniel-Adriano; Baker, David; DeGrado, William F.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">114</span>
        (<span class="NLM_cas:issue">41</span>),
    <span class="NLM_cas:pages">10852-10857</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The folding of natural proteins typically relies on hydrophobic packing, metal binding, or disulfide bond formation in the protein core.  Alternatively, a 3D structure can be defined by incorporating a multivalent crosslinking agent, and this approach has been successfully developed for the selection of bicyclic peptides from large random-sequence libraries.  By contrast, there is no general method for the de novo computational design of multicross-linked proteins with predictable and well-defined folds, including ones not found in nature.  Here the authors use Rosetta and Tertiary Motifs (TERMs) to design small proteins that fold around multivalent crosslinkers.  The hydrophobic crosslinkers stabilize the fold by macrocyclic restraints, and they also form an integral part of a small apolar core.  The designed CovCore proteins were prepd. by chem. synthesis, and their structures were detd. by soln. NMR or x-ray crystallog.  These mesosized proteins, lying between conventional proteins and small peptides, are easily accessible either through biosynthetic precursors or chem. synthesis.  The unique tertiary structures and ease of synthesis of CovCore proteins indicate that they should provide versatile templates for developing inhibitors of protein-protein interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoT3cI2O-jid7Vg90H21EOLACvtfcHk0lgpnoAcL-4NIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFKjtL%252FL&md5=27b2dd6ab774dd6f223971003bd65a6d</span></div><a href="/servlet/linkout?suffix=cit82&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1710695114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1710695114%26sid%3Dliteratum%253Aachs%26aulast%3DDang%26aufirst%3DB.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DMulligan%26aufirst%3DV.%2BK.%26aulast%3DMravic%26aufirst%3DM.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DLemmin%26aufirst%3DT.%26aulast%3DFord%26aufirst%3DA.%26aulast%3DSilva%26aufirst%3DD.-A.%26aulast%3DBaker%26aufirst%3DD.%26aulast%3DDeGrado%26aufirst%3DW.%2BF.%26atitle%3DDe%2520Novo%2520Design%2520of%2520Covalently%2520Constrained%2520Mesosize%2520Protein%2520Scaffolds%2520with%2520Unique%2520Tertiary%2520Structures%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2017%26volume%3D114%26spage%3D10852%26epage%3D10857%26doi%3D10.1073%2Fpnas.1710695114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref83"><div class="reference"><strong class="refLabel"><a href="#ref83" class="rightTabRefNumLink">83</a></strong><div class="NLM_citation" id="rightTab-cit83"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Neubacher, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saya, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amore, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grossmann, T. N.</span></span> <span> </span><span class="NLM_article-title">In Situ Cyclization of Proteins (INCYPRO): Cross-Link Derivatization Modulates Protein Stability</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>85</i></span>,  <span class="NLM_fpage">14761476</span>– <span class="NLM_lpage">1483</span>, <span class="refDoi"> DOI: 10.1021/acs.joc.9b024901476</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1021%2Facs.joc.9b024901476" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=85&publication_year=2019&pages=14761476-1483&author=S.+Neubacherauthor=J.+M.+Sayaauthor=A.+Amoreauthor=T.+N.+Grossmann&title=In+Situ+Cyclization+of+Proteins+%28INCYPRO%29%3A+Cross-Link+Derivatization+Modulates+Protein+Stability&doi=10.1021%2Facs.joc.9b024901476"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit83&amp;dbid=16384&amp;doi=10.1021%2Facs.joc.9b024901476&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.joc.9b024901476%26sid%3Dliteratum%253Aachs%26aulast%3DNeubacher%26aufirst%3DS.%26aulast%3DSaya%26aufirst%3DJ.%2BM.%26aulast%3DAmore%26aufirst%3DA.%26aulast%3DGrossmann%26aufirst%3DT.%2BN.%26atitle%3DIn%2520Situ%2520Cyclization%2520of%2520Proteins%2520%2528INCYPRO%2529%253A%2520Cross-Link%2520Derivatization%2520Modulates%2520Protein%2520Stability%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2019%26volume%3D85%26spage%3D14761476%26epage%3D1483%26doi%3D10.1021%2Facs.joc.9b024901476" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref84"><div class="reference"><strong class="refLabel"><a href="#ref84" class="rightTabRefNumLink">84</a></strong><div class="NLM_citation" id="rightTab-cit84"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pelay-Gimeno, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bange, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hennig, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grossmann, T. N.</span></span> <span> </span><span class="NLM_article-title">In Situ Cyclization of Native Proteins: Structure-Based Design of a Bicyclic Enzyme</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">11164</span>– <span class="NLM_lpage">11170</span>, <span class="refDoi"> DOI: 10.1002/anie.201804506</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1002%2Fanie.201804506" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtF2ls7%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2018&pages=11164-11170&author=M.+Pelay-Gimenoauthor=T.+Bangeauthor=S.+Hennigauthor=T.+N.+Grossmann&title=In+Situ+Cyclization+of+Native+Proteins%3A+Structure-Based+Design+of+a+Bicyclic+Enzyme&doi=10.1002%2Fanie.201804506"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit84R"><div class="casContent"><span class="casTitleNuber">84</span><div class="casTitle"><span class="NLM_cas:atitle">In Situ Cyclization of Native Proteins: Structure-Based Design of a Bicyclic Enzyme</span></div><div class="casAuthors">Pelay-Gimeno, Marta; Bange, Tanja; Hennig, Sven; Grossmann, Tom N.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">35</span>),
    <span class="NLM_cas:pages">11164-11170</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Increased tolerance of enzymes towards thermal and chem. stress is required for many applications and can be achieved by macrocyclization of the enzyme resulting in the stabilizing of its tertiary structure.  Thus far, macrocyclization approaches utilize a very limited structural diversity, which complicates the design process.  Herein, we report an approach that enables cyclization through the installation of modular crosslinks into native proteins composed entirely of proteinogenic amino acids.  Our stabilization procedure involves the introduction of three surface-exposed cysteine residues, which are reacted with a triselectrophile, resulting in the in situ cyclization of the protein (INCYPRO).  A bicyclic version of sortase A (SrtA) was designed that exhibits increased tolerance towards thermal as well as chem. denaturation, and proved to be efficient in protein labeling under denaturing conditions.  In addn., we applied INCYPRO to the KIX domain from the human CREB-binding protein (CBP), resulting in up to 24 °C increased thermal stability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9bVlewFNKi7Vg90H21EOLACvtfcHk0lgHPnzrS34fSg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtF2ls7%252FI&md5=ef6a00e4a22408647fbbb5158110564f</span></div><a href="/servlet/linkout?suffix=cit84&amp;dbid=16384&amp;doi=10.1002%2Fanie.201804506&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201804506%26sid%3Dliteratum%253Aachs%26aulast%3DPelay-Gimeno%26aufirst%3DM.%26aulast%3DBange%26aufirst%3DT.%26aulast%3DHennig%26aufirst%3DS.%26aulast%3DGrossmann%26aufirst%3DT.%2BN.%26atitle%3DIn%2520Situ%2520Cyclization%2520of%2520Native%2520Proteins%253A%2520Structure-Based%2520Design%2520of%2520a%2520Bicyclic%2520Enzyme%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2018%26volume%3D57%26spage%3D11164%26epage%3D11170%26doi%3D10.1002%2Fanie.201804506" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref85"><div class="reference"><strong class="refLabel"><a href="#ref85" class="rightTabRefNumLink">85</a></strong><div class="NLM_citation" id="rightTab-cit85"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Blackwell, H. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sadowsky, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howard, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sampson, J. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chao, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steinmetz, W. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Leary, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grubbs, R. H.</span></span> <span> </span><span class="NLM_article-title">Ring-Closing Metathesis of Olefinic Peptides: Design, Synthesis, and Structural Characterization of Macrocyclic Helical Peptides</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">5291</span>– <span class="NLM_lpage">5302</span>, <span class="refDoi"> DOI: 10.1021/jo015533k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo015533k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BD3MXjtlCktr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2001&pages=5291-5302&author=H.+E.+Blackwellauthor=J.+D.+Sadowskyauthor=R.+J.+Howardauthor=J.+N.+Sampsonauthor=J.+A.+Chaoauthor=W.+E.+Steinmetzauthor=D.+J.+O%E2%80%99Learyauthor=R.+H.+Grubbs&title=Ring-Closing+Metathesis+of+Olefinic+Peptides%3A+Design%2C+Synthesis%2C+and+Structural+Characterization+of+Macrocyclic+Helical+Peptides&doi=10.1021%2Fjo015533k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit85R"><div class="casContent"><span class="casTitleNuber">85</span><div class="casTitle"><span class="NLM_cas:atitle">Ring-Closing Metathesis of Olefinic Peptides: Design, Synthesis, and Structural Characterization of Macrocyclic Helical Peptides</span></div><div class="casAuthors">Blackwell, Helen E.; Sadowsky, Jack D.; Howard, Rebecca J.; Sampson, Joshua N.; Chao, Jeffery A.; Steinmetz, Wayne E.; O'Leary, Daniel J.; Grubbs, Robert H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">5291-5302</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Heptapeptides contg. residues with terminal olefin-derivatized side chains (I) have been treated with ruthenium alkylidene and undergone facile ring-closing olefin metathesis (RCM) to give 21- and 23-membered macrocyclic peptides.  The primary structures of I were based upon a previously studied heptapeptide, which was shown to adopt a predominantly 310-helical conformation in CDCl3 soln. and an α-helical conformation in the solid state.  CD, IR, and soln.-phase 1H NMR studies strongly suggested that acyclic precursors I and the fully satd. macrocyclic products also adopted helical conformations in apolar org. solvents.  Single-crystal X-ray diffraction of one cyclic peptide showed it to exist as a right-handed 310-helix up to the fifth residue.  Soln.-phase NMR structures of both acyclic peptide I and cyclic peptide in CD2Cl2 indicated that the acyclic diene assumes a loosely 310-helical conformation, which is considerably rigidified upon macrocyclization.  The relative ease of introducing carbon-carbon bonds into peptide secondary structures by RCM and the predicted metabolic stability of these bonds renders olefin metathesis an exceptional methodol. for the synthesis of rigidified peptide architectures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoRIl-7fCxsfLVg90H21EOLACvtfcHk0lgHPnzrS34fSg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXjtlCktr0%253D&md5=c1f97d63de6a35d2e08068a89f3c4433</span></div><a href="/servlet/linkout?suffix=cit85&amp;dbid=16384&amp;doi=10.1021%2Fjo015533k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo015533k%26sid%3Dliteratum%253Aachs%26aulast%3DBlackwell%26aufirst%3DH.%2BE.%26aulast%3DSadowsky%26aufirst%3DJ.%2BD.%26aulast%3DHoward%26aufirst%3DR.%2BJ.%26aulast%3DSampson%26aufirst%3DJ.%2BN.%26aulast%3DChao%26aufirst%3DJ.%2BA.%26aulast%3DSteinmetz%26aufirst%3DW.%2BE.%26aulast%3DO%25E2%2580%2599Leary%26aufirst%3DD.%2BJ.%26aulast%3DGrubbs%26aufirst%3DR.%2BH.%26atitle%3DRing-Closing%2520Metathesis%2520of%2520Olefinic%2520Peptides%253A%2520Design%252C%2520Synthesis%252C%2520and%2520Structural%2520Characterization%2520of%2520Macrocyclic%2520Helical%2520Peptides%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2001%26volume%3D66%26spage%3D5291%26epage%3D5302%26doi%3D10.1021%2Fjo015533k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref86"><div class="reference"><strong class="refLabel"><a href="#ref86" class="rightTabRefNumLink">86</a></strong><div class="NLM_citation" id="rightTab-cit86"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Galande, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bramlett, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trent, J. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burris, T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wittliff, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spatola, A. F.</span></span> <span> </span><span class="NLM_article-title">Potent Inhibitors of LXXLL-Based Protein-Protein Interactions</span>. <i>ChemBioChem</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">1991</span>– <span class="NLM_lpage">1998</span>, <span class="refDoi"> DOI: 10.1002/cbic.200500083</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1002%2Fcbic.200500083" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=16222726" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht1ehtLrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2005&pages=1991-1998&author=A.+K.+Galandeauthor=K.+S.+Bramlettauthor=J.+O.+Trentauthor=T.+P.+Burrisauthor=J.+L.+Wittliffauthor=A.+F.+Spatola&title=Potent+Inhibitors+of+LXXLL-Based+Protein-Protein+Interactions&doi=10.1002%2Fcbic.200500083"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit86R"><div class="casContent"><span class="casTitleNuber">86</span><div class="casTitle"><span class="NLM_cas:atitle">Potent inhibitors of LXXLL-based protein-protein interactions</span></div><div class="casAuthors">Galande, Amit K.; Bramlett, Kelli S.; Trent, John O.; Burris, Thomas P.; Wittliff, James L.; Spatola, Arno F.</div><div class="citationInfo"><span class="NLM_cas:title">ChemBioChem</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1991-1998</span>CODEN:
                <span class="NLM_cas:coden">CBCHFX</span>;
        ISSN:<span class="NLM_cas:issn">1439-4227</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Protein-protein interactions between estrogen receptors, ERα and ERβ, and their coactivators (CoAs) are an attractive target for drug intervention.  This interaction is mediated by a small pentapeptide motif (LXXLL), termed the NR box.  Based on this motif, a variety of cyclic and linear peptides were synthesized in order to gain a better understanding of the assocn. of CoA proteins with the ER isoforms.  Utilizing a time-resolved florescence-based coactivator interaction assay, we detd. the abilities of these peptides to inhibit this interaction.  Using mol. modeling and CD spectroscopy, we have examd. the structural basis of their bioactivities with both hormone receptor isoforms.  Either homocysteine or penicillamine was utilized as a substitute for cysteine in the disulfide-bridged peptides, while tertiary leucine and neopentyl glycine were used as the surrogates for the NR box leucines.  The most potent disufide-bridged peptide (Ki 70 pM, with ERα) incorporates neopentyl glycine in the NR box, while the most active peptide in this series with ERβ (Ki = 350 pM) incorporates tertiary leucine.  Surprisingly, several linear peptides contg. a single cysteine residue showed activities with low nanomolar Ki values.  Collectively, our results suggest a synthetic approach for designing potent and selective peptidomimetics for ERα and ERβ interactions with CoA proteins effecting estrogen action.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqRkazg_OTBybVg90H21EOLACvtfcHk0lgHPnzrS34fSg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXht1ehtLrK&md5=02351ff58df231cbd892cc30e19e50b7</span></div><a href="/servlet/linkout?suffix=cit86&amp;dbid=16384&amp;doi=10.1002%2Fcbic.200500083&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbic.200500083%26sid%3Dliteratum%253Aachs%26aulast%3DGalande%26aufirst%3DA.%2BK.%26aulast%3DBramlett%26aufirst%3DK.%2BS.%26aulast%3DTrent%26aufirst%3DJ.%2BO.%26aulast%3DBurris%26aufirst%3DT.%2BP.%26aulast%3DWittliff%26aufirst%3DJ.%2BL.%26aulast%3DSpatola%26aufirst%3DA.%2BF.%26atitle%3DPotent%2520Inhibitors%2520of%2520LXXLL-Based%2520Protein-Protein%2520Interactions%26jtitle%3DChemBioChem%26date%3D2005%26volume%3D6%26spage%3D1991%26epage%3D1998%26doi%3D10.1002%2Fcbic.200500083" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref87"><div class="reference"><strong class="refLabel"><a href="#ref87" class="rightTabRefNumLink">87</a></strong><div class="NLM_citation" id="rightTab-cit87"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Harrison, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shepherd, N. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoang, H. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruiz-Gómez, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Driver, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Desai, V. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abbenante, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fairlie, D. P.</span></span> <span> </span><span class="NLM_article-title">Downsizing Human, Bacterial, and Viral Proteins to Short Water-Stable Alpha Helices That Maintain Biological Potency</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>107</i></span>,  <span class="NLM_fpage">11686</span>– <span class="NLM_lpage">11691</span>, <span class="refDoi"> DOI: 10.1073/pnas.1002498107</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1073%2Fpnas.1002498107" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=20543141" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC3cXovVGkur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=107&publication_year=2010&pages=11686-11691&author=R.+S.+Harrisonauthor=N.+E.+Shepherdauthor=H.+N.+Hoangauthor=G.+Ruiz-G%C3%B3mezauthor=T.+A.+Hillauthor=R.+W.+Driverauthor=V.+S.+Desaiauthor=P.+R.+Youngauthor=G.+Abbenanteauthor=D.+P.+Fairlie&title=Downsizing+Human%2C+Bacterial%2C+and+Viral+Proteins+to+Short+Water-Stable+Alpha+Helices+That+Maintain+Biological+Potency&doi=10.1073%2Fpnas.1002498107"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit87R"><div class="casContent"><span class="casTitleNuber">87</span><div class="casTitle"><span class="NLM_cas:atitle">Downsizing human, bacterial, and viral proteins to short water-stable alpha helices that maintain biological potency</span></div><div class="casAuthors">Harrison, Rosemary S.; Shepherd, Nicholas E.; Hoang, Huy N.; Ruiz-Gomez, Gloria; Hill, Timothy A.; Driver, Russell W.; Desai, Vishal S.; Young, Paul R.; Abbenante, Giovanni; Fairlie, David P.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">107</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">11686-11691, S11686/1-S11686/52</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Recombinant proteins are important therapeutics due to potent, highly specific, and nontoxic actions in vivo.  However, they are expensive medicines to manuf., chem. unstable, and difficult to administer with low patient uptake and compliance.  Small mol. drugs are cheaper and more bioavailable, but less target-specific in vivo and often have assocd. side effects.  Here we combine some advantages of proteins and small mols. by taking short amino acid sequences that confer potency and selectivity to proteins, and fixing them as small constrained mols. that are chem. and structurally stable and easy to make.  Proteins often use short α-helixes of just 1-4 helical turns (4-15 amino acids) to interact with biol. targets, but peptides this short usually have negligible α-helicity in water.  Here we show that short peptides, corresponding to helical epitopes from viral, bacterial, or human proteins, can be strategically fixed in highly α-helical structures in water.  These helix-constrained compds. have similar biol. potencies as proteins that bear the same helical sequences.  Examples are (i) a picomolar inhibitor of Respiratory Syncytial Virus F protein mediated fusion with host cells, (ii) a nanomolar inhibitor of RNA binding to the transporter protein HIV-Rev, (iii) a submicromolar inhibitor of Streptococcus pneumoniae growth induced by quorum sensing pheromone Competence Stimulating Peptide, and (iv) a picomolar agonist of the GPCR pain receptor opioid receptor like receptor ORL-1.  This approach can be generally applicable to downsizing helical regions of proteins with broad applications to biol. and medicine.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpjCkDABZP0wrVg90H21EOLACvtfcHk0lioPQ639n1MFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXovVGkur8%253D&md5=d06bf2eb3c61268f906ee9e26e013be1</span></div><a href="/servlet/linkout?suffix=cit87&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1002498107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1002498107%26sid%3Dliteratum%253Aachs%26aulast%3DHarrison%26aufirst%3DR.%2BS.%26aulast%3DShepherd%26aufirst%3DN.%2BE.%26aulast%3DHoang%26aufirst%3DH.%2BN.%26aulast%3DRuiz-G%25C3%25B3mez%26aufirst%3DG.%26aulast%3DHill%26aufirst%3DT.%2BA.%26aulast%3DDriver%26aufirst%3DR.%2BW.%26aulast%3DDesai%26aufirst%3DV.%2BS.%26aulast%3DYoung%26aufirst%3DP.%2BR.%26aulast%3DAbbenante%26aufirst%3DG.%26aulast%3DFairlie%26aufirst%3DD.%2BP.%26atitle%3DDownsizing%2520Human%252C%2520Bacterial%252C%2520and%2520Viral%2520Proteins%2520to%2520Short%2520Water-Stable%2520Alpha%2520Helices%2520That%2520Maintain%2520Biological%2520Potency%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2010%26volume%3D107%26spage%3D11686%26epage%3D11691%26doi%3D10.1073%2Fpnas.1002498107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref88"><div class="reference"><strong class="refLabel"><a href="#ref88" class="rightTabRefNumLink">88</a></strong><div class="NLM_citation" id="rightTab-cit88"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jackson, D. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">King, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chmielewski, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schultz, P. G.</span></span> <span> </span><span class="NLM_article-title">General Approach to the Synthesis of Short .Alpha.-Helical Peptides</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">1991</span>,  <span class="NLM_volume"><i>113</i></span>,  <span class="NLM_fpage">9391</span>– <span class="NLM_lpage">9392</span>, <span class="refDoi"> DOI: 10.1021/ja00024a067</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja00024a067" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADyaK3MXmvVSmsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=1991&pages=9391-9392&author=D.+Y.+Jacksonauthor=D.+S.+Kingauthor=J.+Chmielewskiauthor=S.+Singhauthor=P.+G.+Schultz&title=General+Approach+to+the+Synthesis+of+Short+.Alpha.-Helical+Peptides&doi=10.1021%2Fja00024a067"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit88R"><div class="casContent"><span class="casTitleNuber">88</span><div class="casTitle"><span class="NLM_cas:atitle">General approach to the synthesis of short α-helical peptides</span></div><div class="casAuthors">Jackson, David Y.; King, David S.; Chmielewski, Jean; Singh, Sunil; Schultz, Peter G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">9391-2</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    </div><div class="casAbstract">Short peptides have been synthesized which contain a single intramol. disulfide bond that stabilizes two helical turns.  Peptides contg. D- and L-S-(acetamidomethyl)-2-amino-6-mercaptohexanoic acid at the i and i + 7 residue, resp., show only slight α-helicity in the reduced form in aq. soln.  On oxidn., these peptides exhibit a large increase in α-helicity in water both at 0° and 60°.  This approach has been used to generate eight and sixteen amino acid peptides with high helicity.  Oxidn. can be carried out under a wide variety of conditions with peptides that contain a large variety of functional groups.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGryvUu77IcBrrVg90H21EOLACvtfcHk0lioPQ639n1MFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3MXmvVSmsbk%253D&md5=c5e9be77bd7fbcbf428424de7cf9a862</span></div><a href="/servlet/linkout?suffix=cit88&amp;dbid=16384&amp;doi=10.1021%2Fja00024a067&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja00024a067%26sid%3Dliteratum%253Aachs%26aulast%3DJackson%26aufirst%3DD.%2BY.%26aulast%3DKing%26aufirst%3DD.%2BS.%26aulast%3DChmielewski%26aufirst%3DJ.%26aulast%3DSingh%26aufirst%3DS.%26aulast%3DSchultz%26aufirst%3DP.%2BG.%26atitle%3DGeneral%2520Approach%2520to%2520the%2520Synthesis%2520of%2520Short%2520.Alpha.-Helical%2520Peptides%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1991%26volume%3D113%26spage%3D9391%26epage%3D9392%26doi%3D10.1021%2Fja00024a067" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref89"><div class="reference"><strong class="refLabel"><a href="#ref89" class="rightTabRefNumLink">89</a></strong><div class="NLM_citation" id="rightTab-cit89"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Leduc, A.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trent, J. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wittliff, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bramlett, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Briggs, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chirgadze, N. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burris, T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spatola, A. F.</span></span> <span> </span><span class="NLM_article-title">Helix-Stabilized Cyclic Peptides as Selective Inhibitors of Steroid Receptor-Coactivator Interactions</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>100</i></span>,  <span class="NLM_fpage">11273</span>– <span class="NLM_lpage">11278</span>, <span class="refDoi"> DOI: 10.1073/pnas.1934759100</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1073%2Fpnas.1934759100" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=13679575" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BD3sXotFKnu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=100&publication_year=2003&pages=11273-11278&author=A.-M.+Leducauthor=J.+O.+Trentauthor=J.+L.+Wittliffauthor=K.+S.+Bramlettauthor=S.+L.+Briggsauthor=N.+Y.+Chirgadzeauthor=Y.+Wangauthor=T.+P.+Burrisauthor=A.+F.+Spatola&title=Helix-Stabilized+Cyclic+Peptides+as+Selective+Inhibitors+of+Steroid+Receptor-Coactivator+Interactions&doi=10.1073%2Fpnas.1934759100"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit89R"><div class="casContent"><span class="casTitleNuber">89</span><div class="casTitle"><span class="NLM_cas:atitle">Helix-stabilized cyclic peptides as selective inhibitors of steroid receptor-coactivator interactions</span></div><div class="casAuthors">Leduc, Anne-Marie; Trent, John O.; Wittliff, James L.; Bramlett, Kelli S.; Briggs, Stephen L.; Chirgadze, Nikolay Y.; Wang, Yong; Burris, Thomas P.; Spatola, Arno F.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">100</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">11273-11278</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The interaction between nuclear receptors and coactivators provides an arena for testing whether protein-protein interactions may be inhibited by small mol. drug candidates.  We provide evidence that a short cyclic peptide, contg. a copy of the LXXLL nuclear receptor box pentapeptide, binds tightly and selectively to estrogen receptor α.  Furthermore, as shown by x-ray anal., the disulfide-bridged nonapeptide, nonhelical in aq. solns., is able to adopt a quasihelical conformer while binding to the groove created by ligand attachment to estrogen receptor α.  An i, i+3 linked analog, H-Lys-cyclo(D-Cys-Ile-Leu-Cys)-Arg-Leu-Leu-Gln-NH2 (peptidomimetic estrogen receptor modulator 1), binds with a Ki of 25 nM, significantly better than an i, i+4 bridged cyclic amide, as predicted by mol. modeling design criteria.  The induction of helical character, effective binding, and receptor selectivity exhibited by this peptide analog provide strong support for this strategy.  The stabilization of minimalist surface motifs may prove useful for the control of other macromol. assemblies, esp. when an amphiphilic helix is crucial for the strong binding interaction between two proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo68LUoV9WM-LVg90H21EOLACvtfcHk0lioPQ639n1MFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXotFKnu7Y%253D&md5=a62f033a94c5e1cc4562e956d6cb6bfd</span></div><a href="/servlet/linkout?suffix=cit89&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1934759100&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1934759100%26sid%3Dliteratum%253Aachs%26aulast%3DLeduc%26aufirst%3DA.-M.%26aulast%3DTrent%26aufirst%3DJ.%2BO.%26aulast%3DWittliff%26aufirst%3DJ.%2BL.%26aulast%3DBramlett%26aufirst%3DK.%2BS.%26aulast%3DBriggs%26aufirst%3DS.%2BL.%26aulast%3DChirgadze%26aufirst%3DN.%2BY.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DBurris%26aufirst%3DT.%2BP.%26aulast%3DSpatola%26aufirst%3DA.%2BF.%26atitle%3DHelix-Stabilized%2520Cyclic%2520Peptides%2520as%2520Selective%2520Inhibitors%2520of%2520Steroid%2520Receptor-Coactivator%2520Interactions%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2003%26volume%3D100%26spage%3D11273%26epage%3D11278%26doi%3D10.1073%2Fpnas.1934759100" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref90"><div class="reference"><strong class="refLabel"><a href="#ref90" class="rightTabRefNumLink">90</a></strong><div class="NLM_citation" id="rightTab-cit90"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Phelan, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skelton, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Braisted, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDowell, R. S.</span></span> <span> </span><span class="NLM_article-title">A General Method for Constraining Short Peptides to an α-Helical Conformation</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>119</i></span>,  <span class="NLM_fpage">455</span>– <span class="NLM_lpage">460</span>, <span class="refDoi"> DOI: 10.1021/ja9611654</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja9611654" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref90/cit90&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADyaK2sXotVWgsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=1997&pages=455-460&author=J.+C.+Phelanauthor=N.+J.+Skeltonauthor=A.+C.+Braistedauthor=R.+S.+McDowell&title=A+General+Method+for+Constraining+Short+Peptides+to+an+%CE%B1-Helical+Conformation&doi=10.1021%2Fja9611654"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit90R"><div class="casContent"><span class="casTitleNuber">90</span><div class="casTitle"><span class="NLM_cas:atitle">A General Method for Constraining Short Peptides to an α-Helical Conformation</span></div><div class="casAuthors">Phelan, J. Christopher; Skelton, Nicholas J.; Braisted, Andrew C.; McDowell, Robert S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">119</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">455-460</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A method for constraining short peptides (contg. fewer than 20 residues) of an arbitrary sequence to an α-helical conformation (∼100% helical in H2O at 25 °C) is presented.  Gln residues at positions i and i + 7 of the peptides were tethered with an alkanediyl chain between the side chain nitrogen atoms.  Peptides contg. this tether were readily synthesized on the solid phase by amide formation between an α,ω-diaminoalkane and the side chain carboxylates of Glu residues.  The resulting cyclic peptides were studied by NMR and CD and were found to adopt an α-helical conformation in aq. soln. and this α-helix was thermally stable to ≥40°.  Corresponding untethered control peptides with N-methylglutamine at the i and i + 7 positions lacked helicity under the same conditions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjjjTLuHs4P7Vg90H21EOLACvtfcHk0lgY_AqOZtEgIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXotVWgsg%253D%253D&md5=57fc8604cb7ea3c545e5eafcf4a4519f</span></div><a href="/servlet/linkout?suffix=cit90&amp;dbid=16384&amp;doi=10.1021%2Fja9611654&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja9611654%26sid%3Dliteratum%253Aachs%26aulast%3DPhelan%26aufirst%3DJ.%2BC.%26aulast%3DSkelton%26aufirst%3DN.%2BJ.%26aulast%3DBraisted%26aufirst%3DA.%2BC.%26aulast%3DMcDowell%26aufirst%3DR.%2BS.%26atitle%3DA%2520General%2520Method%2520for%2520Constraining%2520Short%2520Peptides%2520to%2520an%2520%25CE%25B1-Helical%2520Conformation%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1997%26volume%3D119%26spage%3D455%26epage%3D460%26doi%3D10.1021%2Fja9611654" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref91"><div class="reference"><strong class="refLabel"><a href="#ref91" class="rightTabRefNumLink">91</a></strong><div class="NLM_citation" id="rightTab-cit91"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schafmeister, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Po, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verdine, G. L.</span></span> <span> </span><span class="NLM_article-title">An All-Hydrocarbon Cross-Linking System for Enhancing the Helicity and Metabolic Stability of Peptides</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>122</i></span>,  <span class="NLM_fpage">5891</span>– <span class="NLM_lpage">5892</span>, <span class="refDoi"> DOI: 10.1021/ja000563a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja000563a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BD3cXjslajurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=122&publication_year=2000&pages=5891-5892&author=C.+E.+Schafmeisterauthor=J.+Poauthor=G.+L.+Verdine&title=An+All-Hydrocarbon+Cross-Linking+System+for+Enhancing+the+Helicity+and+Metabolic+Stability+of+Peptides&doi=10.1021%2Fja000563a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit91R"><div class="casContent"><span class="casTitleNuber">91</span><div class="casTitle"><span class="NLM_cas:atitle">An All-Hydrocarbon Cross-Linking System for Enhancing the Helicity and Metabolic Stability of Peptides</span></div><div class="casAuthors">Schafmeister, Christian E.; Po, Julia; Verdine, Gregory L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">122</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">5891-5892</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The authors have synthesized peptides contg. a hydrocarbon linkage connecting two unnatural amino acids having either R or S stereochem. at the methyl- and tether-bearing α-carbon, with tethers of various lengths to form a macrocyclic structure which either stabilized or destabilized the peptides α-helix formation, depending on length of tether and resulting size of macrocycle.  The unnatural amino acids were inserted in either i, i+4 or i, i+7 positions in synthetic C-peptide sequence from RNAse A, using solid-phase chem., metathesized while still linked to support, and the cleaved products tested for increased α-helix content as it affected soly. and resistance to trypsin digestion.  Length of tether and sepn. of insertion positions were both detg. factors in successful metathesis and α-helix enhancement; Ri,i+7S insertion with tether length of 11 had a trypsin cleavage rate const. of 0.058 M-1s-1, compared to 0.37 M-1s-1 for tether length of 9.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHKipXFsY9w7Vg90H21EOLACvtfcHk0lgY_AqOZtEgIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXjslajurk%253D&md5=804271601bdd8a1c71e38a7e239a98d9</span></div><a href="/servlet/linkout?suffix=cit91&amp;dbid=16384&amp;doi=10.1021%2Fja000563a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja000563a%26sid%3Dliteratum%253Aachs%26aulast%3DSchafmeister%26aufirst%3DC.%2BE.%26aulast%3DPo%26aufirst%3DJ.%26aulast%3DVerdine%26aufirst%3DG.%2BL.%26atitle%3DAn%2520All-Hydrocarbon%2520Cross-Linking%2520System%2520for%2520Enhancing%2520the%2520Helicity%2520and%2520Metabolic%2520Stability%2520of%2520Peptides%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2000%26volume%3D122%26spage%3D5891%26epage%3D5892%26doi%3D10.1021%2Fja000563a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref92"><div class="reference"><strong class="refLabel"><a href="#ref92" class="rightTabRefNumLink">92</a></strong><div class="NLM_citation" id="rightTab-cit92"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pelay-Gimeno, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glas, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koch, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grossmann, T. N.</span></span> <span> </span><span class="NLM_article-title">Structure-Based Design of Inhibitors of Protein–Protein Interactions: Mimicking Peptide Binding Epitopes</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">8896</span>– <span class="NLM_lpage">8927</span>, <span class="refDoi"> DOI: 10.1002/anie.201412070</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1002%2Fanie.201412070" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=26119925" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVOmu7rM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2015&pages=8896-8927&author=M.+Pelay-Gimenoauthor=A.+Glasauthor=O.+Kochauthor=T.+N.+Grossmann&title=Structure-Based+Design+of+Inhibitors+of+Protein%E2%80%93Protein+Interactions%3A+Mimicking+Peptide+Binding+Epitopes&doi=10.1002%2Fanie.201412070"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit92R"><div class="casContent"><span class="casTitleNuber">92</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Design of Inhibitors of Protein-Protein Interactions: Mimicking Peptide Binding Epitopes</span></div><div class="casAuthors">Pelay-Gimeno, Marta; Glas, Adrian; Koch, Oliver; Grossmann, Tom N.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">31</span>),
    <span class="NLM_cas:pages">8896-8927</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  Protein-protein interactions (PPIs) are involved at all levels of cellular organization, thus making the development of PPI inhibitors extremely valuable.  The identification of selective inhibitors is challenging because of the shallow and extended nature of PPI interfaces.  Inhibitors can be obtained by mimicking peptide binding epitopes in their bioactive conformation.  For this purpose, several strategies have been evolved to enable a projection of side chain functionalities in analogy to peptide secondary structures, thereby yielding mols. that are generally referred to as peptidomimetics.  Herein, we introduce a new classification of peptidomimetics (classes A-D) that enables a clear assignment of available approaches.  Based on this classification, the Review summarizes strategies that have been applied for the structure-based design of PPI inhibitors through stabilizing or mimicking turns, β-sheets, and helixes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPTEmZpatXbrVg90H21EOLACvtfcHk0lgY_AqOZtEgIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVOmu7rM&md5=447c84aab6275b1057fda9a610918f2b</span></div><a href="/servlet/linkout?suffix=cit92&amp;dbid=16384&amp;doi=10.1002%2Fanie.201412070&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201412070%26sid%3Dliteratum%253Aachs%26aulast%3DPelay-Gimeno%26aufirst%3DM.%26aulast%3DGlas%26aufirst%3DA.%26aulast%3DKoch%26aufirst%3DO.%26aulast%3DGrossmann%26aufirst%3DT.%2BN.%26atitle%3DStructure-Based%2520Design%2520of%2520Inhibitors%2520of%2520Protein%25E2%2580%2593Protein%2520Interactions%253A%2520Mimicking%2520Peptide%2520Binding%2520Epitopes%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2015%26volume%3D54%26spage%3D8896%26epage%3D8927%26doi%3D10.1002%2Fanie.201412070" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref93"><div class="reference"><strong class="refLabel"><a href="#ref93" class="rightTabRefNumLink">93</a></strong><div class="NLM_citation" id="rightTab-cit93"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bhardwaj, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulligan, V. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bahl, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilmore, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harvey, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheneval, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buchko, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pulavarti, S. V. S. R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaas, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eletsky, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, P.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnsen, W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greisen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rocklin, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Linsky, T. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watkins, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rettie, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonneau, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coutsias, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Correnti, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szyperski, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craik, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baker, D.</span></span> <span> </span><span class="NLM_article-title">Accurate de Novo Design of Hyperstable Constrained Peptides</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>538</i></span>,  <span class="NLM_fpage">329</span>– <span class="NLM_lpage">335</span>, <span class="refDoi"> DOI: 10.1038/nature19791</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1038%2Fnature19791" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=27626386" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFWiurnL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=538&publication_year=2016&pages=329-335&author=G.+Bhardwajauthor=V.+K.+Mulliganauthor=C.+D.+Bahlauthor=J.+M.+Gilmoreauthor=P.+J.+Harveyauthor=O.+Chenevalauthor=G.+W.+Buchkoauthor=S.+V.+S.+R.+K.+Pulavartiauthor=Q.+Kaasauthor=A.+Eletskyauthor=P.-S.+Huangauthor=W.+A.+Johnsenauthor=P.+Greisenauthor=G.+J.+Rocklinauthor=Y.+Songauthor=T.+W.+Linskyauthor=A.+Watkinsauthor=S.+A.+Rettieauthor=X.+Xuauthor=L.+P.+Carterauthor=R.+Bonneauauthor=J.+M.+Olsonauthor=E.+Coutsiasauthor=C.+E.+Correntiauthor=T.+Szyperskiauthor=D.+J.+Craikauthor=D.+Baker&title=Accurate+de+Novo+Design+of+Hyperstable+Constrained+Peptides&doi=10.1038%2Fnature19791"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit93R"><div class="casContent"><span class="casTitleNuber">93</span><div class="casTitle"><span class="NLM_cas:atitle">Accurate de novo design of hyperstable constrained peptides</span></div><div class="casAuthors">Bhardwaj, Gaurav; Mulligan, Vikram Khipple; Bahl, Christopher D.; Gilmore, Jason M.; Harvey, Peta J.; Cheneval, Olivier; Buchko, Garry W.; Pulavarti, Surya V. S. R. K.; Kaas, Quentin; Eletsky, Alexander; Huang, Po-Ssu; Johnsen, William A.; Greisen, Per Jr; Rocklin, Gabriel J.; Song, Yifan; Linsky, Thomas W.; Watkins, Andrew; Rettie, Stephen A.; Xu, Xianzhong; Carter, Lauren P.; Bonneau, Richard; Olson, James M.; Coutsias, Evangelos; Correnti, Colin E.; Szyperski, Thomas; Craik, David J.; Baker, David</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">538</span>
        (<span class="NLM_cas:issue">7625</span>),
    <span class="NLM_cas:pages">329-335</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Naturally occurring, pharmacol. active peptides constrained with covalent crosslinks generally have shapes that have evolved to fit precisely into binding pockets on their targets.  Such peptides can have excellent pharmaceutical properties, combining the stability and tissue penetration of small-mol. drugs with the specificity of much larger protein therapeutics.  The ability to design constrained peptides with precisely specified tertiary structures would enable the design of shape-complementary inhibitors of arbitrary targets.  Here we describe the development of computational methods for accurate de novo design of conformationally restricted peptides, and the use of these methods to design 18-47 residue, disulfide-crosslinked peptides, a subset of which are heterochiral and/or N-C backbone-cyclized.  Both genetically encodable and non-canonical peptides are exceptionally stable to thermal and chem. denaturation, and 12 exptl. detd. X-ray and NMR structures are nearly identical to the computational design models.  The computational design methods and stable scaffolds presented here provide the basis for development of a new generation of peptide-based drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHeMGaFeL_kbVg90H21EOLACvtfcHk0lju-S1M1U5DnA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFWiurnL&md5=2d2b551ad78a2d76c99eeaba0c763c8e</span></div><a href="/servlet/linkout?suffix=cit93&amp;dbid=16384&amp;doi=10.1038%2Fnature19791&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature19791%26sid%3Dliteratum%253Aachs%26aulast%3DBhardwaj%26aufirst%3DG.%26aulast%3DMulligan%26aufirst%3DV.%2BK.%26aulast%3DBahl%26aufirst%3DC.%2BD.%26aulast%3DGilmore%26aufirst%3DJ.%2BM.%26aulast%3DHarvey%26aufirst%3DP.%2BJ.%26aulast%3DCheneval%26aufirst%3DO.%26aulast%3DBuchko%26aufirst%3DG.%2BW.%26aulast%3DPulavarti%26aufirst%3DS.%2BV.%2BS.%2BR.%2BK.%26aulast%3DKaas%26aufirst%3DQ.%26aulast%3DEletsky%26aufirst%3DA.%26aulast%3DHuang%26aufirst%3DP.-S.%26aulast%3DJohnsen%26aufirst%3DW.%2BA.%26aulast%3DGreisen%26aufirst%3DP.%26aulast%3DRocklin%26aufirst%3DG.%2BJ.%26aulast%3DSong%26aufirst%3DY.%26aulast%3DLinsky%26aufirst%3DT.%2BW.%26aulast%3DWatkins%26aufirst%3DA.%26aulast%3DRettie%26aufirst%3DS.%2BA.%26aulast%3DXu%26aufirst%3DX.%26aulast%3DCarter%26aufirst%3DL.%2BP.%26aulast%3DBonneau%26aufirst%3DR.%26aulast%3DOlson%26aufirst%3DJ.%2BM.%26aulast%3DCoutsias%26aufirst%3DE.%26aulast%3DCorrenti%26aufirst%3DC.%2BE.%26aulast%3DSzyperski%26aufirst%3DT.%26aulast%3DCraik%26aufirst%3DD.%2BJ.%26aulast%3DBaker%26aufirst%3DD.%26atitle%3DAccurate%2520de%2520Novo%2520Design%2520of%2520Hyperstable%2520Constrained%2520Peptides%26jtitle%3DNature%26date%3D2016%26volume%3D538%26spage%3D329%26epage%3D335%26doi%3D10.1038%2Fnature19791" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref94"><div class="reference"><strong class="refLabel"><a href="#ref94" class="rightTabRefNumLink">94</a></strong><div class="NLM_citation" id="rightTab-cit94"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Est, C. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mangrolia, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, R. M.</span></span> <span> </span><span class="NLM_article-title">ROSETTA-Informed Design of Structurally Stabilized Cyclic Anti-Amyloid Peptides</span>. <i>Protein Eng., Des. Sel.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">47</span>– <span class="NLM_lpage">57</span>, <span class="refDoi"> DOI: 10.1093/protein/gzz016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1093%2Fprotein%2Fgzz016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=31650164" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A280%3ADC%252BB3MjhsFWhuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2019&pages=47-57&author=C.+B.+Estauthor=P.+Mangroliaauthor=R.+M.+Murphy&title=ROSETTA-Informed+Design+of+Structurally+Stabilized+Cyclic+Anti-Amyloid+Peptides&doi=10.1093%2Fprotein%2Fgzz016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit94R"><div class="casContent"><span class="casTitleNuber">94</span><div class="casTitle"><span class="NLM_cas:atitle">ROSETTA-informed design of structurally stabilized cyclic anti-amyloid peptides</span></div><div class="casAuthors">Est Chandler B; Mangrolia Parth; Murphy Regina M</div><div class="citationInfo"><span class="NLM_cas:title">Protein engineering, design & selection : PEDS</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">47-57</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">β-amyloid oligomers are thought to be the most toxic species formed en route to fibril deposition in Alzheimer's disease.  Transthyretin is a natural sequestering agent of β-amyloid oligomers: the binding site to β-amyloid has been traced to strands G/H of the inner β-sheet of transthyretin.  A linear peptide, with the same primary sequence as the β-amyloid binding domain on transthyretin, was moderately effective at inhibiting β-amyloid fibril growth.  Insertion of a β-turn template and cyclization greatly increased stability against proteolysis and improved efficacy as an amyloid inhibitor.  However, the cyclic peptide still contained a significant amount of disorder.  Using the Simple Cyclic Peptide Application within ROSETTA as an in silico predictor of cyclic peptide conformation and stability, we investigated putative structural enhancements, including stabilization by disulfide linkages and insertion of a second β-turn template.  Several candidates were synthesized and tested for secondary structure and ability to inhibit β-amyloid aggregation.  The results demonstrate that cyclization, β-sheet structure and conformational homogeneity are all preferable design features, whereas disulfide bond formation across the two β-strands is not preferable.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQEwXe3qQ7PkIvKgEt20y1XfW6udTcc2eZJdHhc9_xhvbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3MjhsFWhuw%253D%253D&md5=9aefa98f5739f90a698bdab680efda4c</span></div><a href="/servlet/linkout?suffix=cit94&amp;dbid=16384&amp;doi=10.1093%2Fprotein%2Fgzz016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fprotein%252Fgzz016%26sid%3Dliteratum%253Aachs%26aulast%3DEst%26aufirst%3DC.%2BB.%26aulast%3DMangrolia%26aufirst%3DP.%26aulast%3DMurphy%26aufirst%3DR.%2BM.%26atitle%3DROSETTA-Informed%2520Design%2520of%2520Structurally%2520Stabilized%2520Cyclic%2520Anti-Amyloid%2520Peptides%26jtitle%3DProtein%2520Eng.%252C%2520Des.%2520Sel.%26date%3D2019%26volume%3D32%26spage%3D47%26epage%3D57%26doi%3D10.1093%2Fprotein%2Fgzz016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref95"><div class="reference"><strong class="refLabel"><a href="#ref95" class="rightTabRefNumLink">95</a></strong><div class="NLM_citation" id="rightTab-cit95"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guéret, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thavam, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carbajo, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potowski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larsson, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dahl, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dellsén, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grossmann, T. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plowright, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valeur, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lemurell, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waldmann, H.</span></span> <span> </span><span class="NLM_article-title">Macrocyclic Modalities Combining Peptide Epitopes and Natural Product Fragments</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>142</i></span>,  <span class="NLM_fpage">4904</span>– <span class="NLM_lpage">4915</span>, <span class="refDoi"> DOI: 10.1021/jacs.0c00269</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.0c00269" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BB3cXjtFGitLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=142&publication_year=2020&pages=4904-4915&author=S.+M.+Gu%C3%A9retauthor=S.+Thavamauthor=R.+J.+Carbajoauthor=M.+Potowskiauthor=N.+Larssonauthor=G.+Dahlauthor=A.+Dells%C3%A9nauthor=T.+N.+Grossmannauthor=A.+T.+Plowrightauthor=E.+Valeurauthor=M.+Lemurellauthor=H.+Waldmann&title=Macrocyclic+Modalities+Combining+Peptide+Epitopes+and+Natural+Product+Fragments&doi=10.1021%2Fjacs.0c00269"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit95R"><div class="casContent"><span class="casTitleNuber">95</span><div class="casTitle"><span class="NLM_cas:atitle">Macrocyclic Modalities Combining Peptide Epitopes and Natural Product Fragments</span></div><div class="casAuthors">Gueret, Stephanie M.; Thavam, Sasikala; Carbajo, Rodrigo J.; Potowski, Marco; Larsson, Niklas; Dahl, Goeran; Dellsen, Anita; Grossmann, Tom N.; Plowright, Alleyn T.; Valeur, Eric; Lemurell, Malin; Waldmann, Herbert</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">142</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4904-4915</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">"Hot loop" protein segments have variable structure and conformation and contribute crucially to protein-protein interactions.  We describe a new hot loop mimicking modality, termed PepNats, in which natural product (NP)-inspired structures are incorporated as conformation-detg. and -restricting structural elements into macrocyclic hot loop-derived peptides.  Macrocyclic PepNats representing hot loops of inducible nitric oxide synthase (iNOS) and human agouti-related protein (AGRP) were synthesized on solid support employing macrocyclization by imine formation and subsequent stereoselective 1,3-dipolar cycloaddn. as key steps.  PepNats derived from the iNOS DINNN hot loop and the AGRP RFF hot spot sequence yielded novel and potent ligands of the SPRY domain-contg. SOCS box protein 2 (SPSB2) that binds to iNOS, and selective ligands for AGRP-binding melanocortin (MC) receptors.  NP-inspired fragment abs. configuration dets. the conformation of the peptide part responsible for binding.  These results demonstrate that combination of NP-inspired scaffolds with peptidic epitopes enables identification of novel hot loop mimics with conformationally constrained and biol. relevant structure.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrMaFk_qnRn1LVg90H21EOLACvtfcHk0lju-S1M1U5DnA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXjtFGitLk%253D&md5=fa5baf213b94ee1adfe05df41b2330af</span></div><a href="/servlet/linkout?suffix=cit95&amp;dbid=16384&amp;doi=10.1021%2Fjacs.0c00269&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.0c00269%26sid%3Dliteratum%253Aachs%26aulast%3DGu%25C3%25A9ret%26aufirst%3DS.%2BM.%26aulast%3DThavam%26aufirst%3DS.%26aulast%3DCarbajo%26aufirst%3DR.%2BJ.%26aulast%3DPotowski%26aufirst%3DM.%26aulast%3DLarsson%26aufirst%3DN.%26aulast%3DDahl%26aufirst%3DG.%26aulast%3DDells%25C3%25A9n%26aufirst%3DA.%26aulast%3DGrossmann%26aufirst%3DT.%2BN.%26aulast%3DPlowright%26aufirst%3DA.%2BT.%26aulast%3DValeur%26aufirst%3DE.%26aulast%3DLemurell%26aufirst%3DM.%26aulast%3DWaldmann%26aufirst%3DH.%26atitle%3DMacrocyclic%2520Modalities%2520Combining%2520Peptide%2520Epitopes%2520and%2520Natural%2520Product%2520Fragments%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2020%26volume%3D142%26spage%3D4904%26epage%3D4915%26doi%3D10.1021%2Fjacs.0c00269" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref96"><div class="reference"><strong class="refLabel"><a href="#ref96" class="rightTabRefNumLink">96</a></strong><div class="NLM_citation" id="rightTab-cit96"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mulligan, V. K.</span></span> <span> </span><span class="NLM_article-title">The Emerging Role of Computational Design in Peptide Macrocycle Drug Discovery</span>. <i>Expert Opin. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">833</span>– <span class="NLM_lpage">852</span>, <span class="refDoi"> DOI: 10.1080/17460441.2020.1751117</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1080%2F17460441.2020.1751117" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=32345066" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BB3cXpsVKku7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2020&pages=833-852&author=V.+K.+Mulligan&title=The+Emerging+Role+of+Computational+Design+in+Peptide+Macrocycle+Drug+Discovery&doi=10.1080%2F17460441.2020.1751117"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit96R"><div class="casContent"><span class="casTitleNuber">96</span><div class="casTitle"><span class="NLM_cas:atitle">The emerging role of computational design in peptide macrocycle drug discovery</span></div><div class="casAuthors">Mulligan, Vikram K.</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Drug Discovery</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">833-852</span>CODEN:
                <span class="NLM_cas:coden">EODDBX</span>;
        ISSN:<span class="NLM_cas:issn">1746-0441</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">A review.  Drug discovery is a laborious process with rising cost per new drug.  Peptide macrocycles are promising therapeutics, though conformational flexibility can reduce target affinity and specificity.  Recent computational advancements address this problem by enabling rational design of rigidly folded peptide macrocycles.: This review summarizes currently approved peptide macrocycle therapeutics and discusses advantages of mesoscale drugs over small mols. or protein therapeutics.  It describes the history, rationale, and state of the art of computational tools, such as Rosetta, that allow the design of rigidly structured peptide macrocycles.  The emerging pipeline for designing peptide macrocycle drugs is described, including current challenges in designing permeable mols. that can emulate the chameleonic behavior of natural macrocycles.  Prospects for reducing computational cost and improving accuracy with emerging computational technologies are also discussed.: To embrace computational design of peptide macrocycle drugs, we must shift current attitudes regarding the role of computation in drug discovery, and move beyond Lipinski's rules.  This technol. has the potential to shift failures to earlier in silico stages of the drug discovery process, improving success rates in costly clin. trials.  Given the available tools, now is the time for drug developers to incorporate peptide macrocycle design into drug discovery pipelines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5iQ_xrAILPLVg90H21EOLACvtfcHk0lgFpKm7m1gFrA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXpsVKku7s%253D&md5=adee56703724078263c0aad5ff39f305</span></div><a href="/servlet/linkout?suffix=cit96&amp;dbid=16384&amp;doi=10.1080%2F17460441.2020.1751117&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F17460441.2020.1751117%26sid%3Dliteratum%253Aachs%26aulast%3DMulligan%26aufirst%3DV.%2BK.%26atitle%3DThe%2520Emerging%2520Role%2520of%2520Computational%2520Design%2520in%2520Peptide%2520Macrocycle%2520Drug%2520Discovery%26jtitle%3DExpert%2520Opin.%2520Drug%2520Discovery%26date%3D2020%26volume%3D15%26spage%3D833%26epage%3D852%26doi%3D10.1080%2F17460441.2020.1751117" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref97"><div class="reference"><strong class="refLabel"><a href="#ref97" class="rightTabRefNumLink">97</a></strong><div class="NLM_citation" id="rightTab-cit97"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bonnet, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agrafiotis, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, E.</span></span> <span> </span><span class="NLM_article-title">Conformational Analysis of Macrocycles: Finding What Common Search Methods Miss</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">2242</span>– <span class="NLM_lpage">2259</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2013.10.003</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci900238a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1ahtbnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2009&pages=2242-2259&author=P.+Bonnetauthor=D.+K.+Agrafiotisauthor=F.+Zhuauthor=E.+Martin&title=Conformational+Analysis+of+Macrocycles%3A+Finding+What+Common+Search+Methods+Miss&doi=10.1016%2Fj.bmc.2013.10.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit97R"><div class="casContent"><span class="casTitleNuber">97</span><div class="casTitle"><span class="NLM_cas:atitle">Conformational Analysis of Macrocycles: Finding What Common Search Methods Miss</span></div><div class="casAuthors">Bonnet, Pascal; Agrafiotis, Dimitris K.; Zhu, Fangqiang; Martin, Eric</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2242-2259</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">As computational drug design becomes increasingly reliant on virtual screening and on high-throughput 3D modeling, the need for fast, robust, and reliable methods for sampling mol. conformations has become greater than ever.  Furthermore, chem. novelty is at a premium, forcing medicinal chemists to explore more complex structural motifs and unusual topologies.  This necessitates the use of conformational sampling techniques that work well in all cases.  Here, we compare the performance of several popular conformational search algorithms on three broad classes of macrocyclic mols.  These methods include Catalyst, CAESAR, MacroModel, MOE, Omega, Rubicon and two newer self-organizing algorithms known as stochastic proximity embedding (SPE) and self-organizing superimposition (SOS) that have been developed at Johnson & Johnson.  Our results show a compelling advantage for the three distance geometry methods (SOS, SPE, and Rubicon) followed to a lesser extent by MacroModel.  The remaining techniques, particularly those based on systematic search, often failed to identify any of the lowest energy conformations and are unsuitable for this class of structures.  Taken together with our previous study on drug-like mols., these results suggest that SPE and SOS are two of the most robust and universally applicable conformational search methods, with the latter being preferred because of its superior speed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhIMu1b7U477Vg90H21EOLACvtfcHk0lgFpKm7m1gFrA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht1ahtbnF&md5=9b4a5919415ef0660c4f9ff18d9b76f4</span></div><a href="/servlet/linkout?suffix=cit97&amp;dbid=16384&amp;doi=10.1021%2Fci900238a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci900238a%26sid%3Dliteratum%253Aachs%26aulast%3DBonnet%26aufirst%3DP.%26aulast%3DAgrafiotis%26aufirst%3DD.%2BK.%26aulast%3DZhu%26aufirst%3DF.%26aulast%3DMartin%26aufirst%3DE.%26atitle%3DConformational%2520Analysis%2520of%2520Macrocycles%253A%2520Finding%2520What%2520Common%2520Search%2520Methods%2520Miss%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2009%26volume%3D49%26spage%3D2242%26epage%3D2259%26doi%3D10.1016%2Fj.bmc.2013.10.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref98"><div class="reference"><strong class="refLabel"><a href="#ref98" class="rightTabRefNumLink">98</a></strong><div class="NLM_citation" id="rightTab-cit98"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, I.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foloppe, N.</span></span> <span> </span><span class="NLM_article-title">Tackling the Conformational Sampling of Larger Flexible Compounds and Macrocycles in Pharmacology and Drug Discovery</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">7898</span>– <span class="NLM_lpage">7920</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2013.10.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1016%2Fj.bmc.2013.10.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=24184215" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslWltrfK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2013&pages=7898-7920&author=I.-J.+Chenauthor=N.+Foloppe&title=Tackling+the+Conformational+Sampling+of+Larger+Flexible+Compounds+and+Macrocycles+in+Pharmacology+and+Drug+Discovery&doi=10.1016%2Fj.bmc.2013.10.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit98R"><div class="casContent"><span class="casTitleNuber">98</span><div class="casTitle"><span class="NLM_cas:atitle">Tackling the conformational sampling of larger flexible compounds and macrocycles in pharmacology and drug discovery</span></div><div class="casAuthors">Chen, I-Jen; Foloppe, Nicolas</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">7898-7920</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Computational conformational sampling underpins much of mol. modeling and design in pharmaceutical work.  The sampling of smaller drug-like compds. has been an active area of research.  However, few studies have tested in details the sampling of larger more flexible compds., which are also relevant to drug discovery, including therapeutic peptides, macrocycles, and inhibitors of protein-protein interactions.  Here, we investigate extensively mainstream conformational sampling methods on three carefully curated compd. sets, namely the 'Drug-like', larger 'Flexible', and 'Macrocycle' compds.  These test mols. are chem. diverse with reliable X-ray protein-bound bioactive structures.  The compared sampling methods include Stochastic Search and the recent LowModeMD from MOE, all the low-mode based approaches from MacroModel, and MD/LLMOD recently developed for macrocycles.  In addn. to default settings, key parameters of the sampling protocols were explored.  The performance of the computational protocols was assessed via (i) the reprodn. of the X-ray bioactive structures, (ii) the size, coverage and diversity of the output conformational ensembles, (iii) the compactness/extendedness of the conformers, and (iv) the ability to locate the global energy min.  The influence of the stochastic nature of the searches on the results was also examd.  Much better results were obtained by adopting search parameters enhanced over the default settings, while maintaining computational tractability.  In MOE, the recent LowModeMD emerged as the method of choice.  Mixed torsional/low-mode from MacroModel performed as well as LowModeMD, and MD/LLMOD performed well for macrocycles.  The low-mode based approaches yielded very encouraging results with the flexible and macrocycle sets.  Thus, one can productively tackle the computational conformational search of larger flexible compds. for drug discovery, including macrocycles.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovwhGIpE1mMbVg90H21EOLACvtfcHk0lgFpKm7m1gFrA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslWltrfK&md5=2699227e69410a5673dec749169fd578</span></div><a href="/servlet/linkout?suffix=cit98&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2013.10.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2013.10.003%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DI.-J.%26aulast%3DFoloppe%26aufirst%3DN.%26atitle%3DTackling%2520the%2520Conformational%2520Sampling%2520of%2520Larger%2520Flexible%2520Compounds%2520and%2520Macrocycles%2520in%2520Pharmacology%2520and%2520Drug%2520Discovery%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2013%26volume%3D21%26spage%3D7898%26epage%3D7920%26doi%3D10.1016%2Fj.bmc.2013.10.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref99"><div class="reference"><strong class="refLabel"><a href="#ref99" class="rightTabRefNumLink">99</a></strong><div class="NLM_citation" id="rightTab-cit99"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Coutsias, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lexa, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wester, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollock, S. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, M. P.</span></span> <span> </span><span class="NLM_article-title">Exhaustive Conformational Sampling of Complex Fused Ring Macrocycles Using Inverse Kinematics</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">4674</span>– <span class="NLM_lpage">4687</span>, <span class="refDoi"> DOI: 10.1021/acs.jctc.6b00250</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jctc.6b00250" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1WntLjI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=4674-4687&author=E.+A.+Coutsiasauthor=K.+W.+Lexaauthor=M.+J.+Westerauthor=S.+N.+Pollockauthor=M.+P.+Jacobson&title=Exhaustive+Conformational+Sampling+of+Complex+Fused+Ring+Macrocycles+Using+Inverse+Kinematics&doi=10.1021%2Facs.jctc.6b00250"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit99R"><div class="casContent"><span class="casTitleNuber">99</span><div class="casTitle"><span class="NLM_cas:atitle">Exhaustive Conformational Sampling of Complex Fused Ring Macrocycles Using Inverse Kinematics</span></div><div class="casAuthors">Coutsias, Evangelos A.; Lexa, Katrina W.; Wester, Michael J.; Pollock, Sara N.; Jacobson, Matthew P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4674-4687</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Natural product and synthetic macrocycles are chem. and topol. diverse.  An efficient, accurate, and general method for generating macrocycle conformations would enable structure-based design of macrocycle drugs or host-guest complexes.  Computational sampling also provides insight into transiently populated states, complementing crystallog. and NMR data.  Here, we report a new algorithm, BRIKARD, that addresses this challenge through computational algebraic geometry and inverse kinematics together with local energy minimization.  BRIKARD is demonstrated on 67 diverse macrocycles with structural data, encompassing various ring topologies.  We find this approach enumerates diverse structures with macrocyclic RMSD < 1.0 Å to the exptl. conformation for 85% of our data set in contrast to success rates of 67-75% with other approaches, while for the subset of 21 more challenging compds. in the data set, these rates are 57% and 10-29%, resp.  Because the algorithm can be efficiently run in parallel on many processors, exhaustive conformational sampling of complex cycles can be obtained in minutes rather than hours: with a 40 processor implementation protocol, BRIKARD samples the conformational diversity of a potential energy landscape in a median of 1.3 min of wallclock time, much faster than 3.1-10.3 h necessary with current programs.  By rigorously testing BRIKARD on a broad range of scaffolds with highly complex ring systems, we push the frontiers of macrocycle sampling to encompass multiring compds., including those with more than 50 ring atoms and up to seven interlaced flexible rings.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodPfQ1Rj98zLVg90H21EOLACvtfcHk0lgFpKm7m1gFrA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1WntLjI&md5=75ba39a0489ad676e5e53c59931d1391</span></div><a href="/servlet/linkout?suffix=cit99&amp;dbid=16384&amp;doi=10.1021%2Facs.jctc.6b00250&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jctc.6b00250%26sid%3Dliteratum%253Aachs%26aulast%3DCoutsias%26aufirst%3DE.%2BA.%26aulast%3DLexa%26aufirst%3DK.%2BW.%26aulast%3DWester%26aufirst%3DM.%2BJ.%26aulast%3DPollock%26aufirst%3DS.%2BN.%26aulast%3DJacobson%26aufirst%3DM.%2BP.%26atitle%3DExhaustive%2520Conformational%2520Sampling%2520of%2520Complex%2520Fused%2520Ring%2520Macrocycles%2520Using%2520Inverse%2520Kinematics%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2016%26volume%3D12%26spage%3D4674%26epage%3D4687%26doi%3D10.1021%2Facs.jctc.6b00250" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref100"><div class="reference"><strong class="refLabel"><a href="#ref100" class="rightTabRefNumLink">100</a></strong><div class="NLM_citation" id="rightTab-cit100"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Geng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.-D.</span></span> <span> </span><span class="NLM_article-title">Accurate Structure Prediction and Conformational Analysis of Cyclic Peptides with Residue-Specific Force Fields</span>. <i>J. Phys. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">1805</span>– <span class="NLM_lpage">1810</span>, <span class="refDoi"> DOI: 10.1021/acs.jpclett.6b00452</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jpclett.6b00452" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC28XntFCiu74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=1805-1810&author=H.+Gengauthor=F.+Jiangauthor=Y.-D.+Wu&title=Accurate+Structure+Prediction+and+Conformational+Analysis+of+Cyclic+Peptides+with+Residue-Specific+Force+Fields&doi=10.1021%2Facs.jpclett.6b00452"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit100R"><div class="casContent"><span class="casTitleNuber">100</span><div class="casTitle"><span class="NLM_cas:atitle">Accurate Structure Prediction and Conformational Analysis of Cyclic Peptides with Residue-Specific Force Fields</span></div><div class="casAuthors">Geng, Hao; Jiang, Fan; Wu, Yun-Dong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry Letters</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1805-1810</span>CODEN:
                <span class="NLM_cas:coden">JPCLCD</span>;
        ISSN:<span class="NLM_cas:issn">1948-7185</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Cyclic peptides (CPs) are promising candidates for drugs, chem. biol. tools, and self-assembling nanomaterials.  However, the development of reliable and accurate computational methods for their structure prediction has been challenging.  Here, 20 all-trans CPs of 5-12 residues selected from Cambridge Structure Database have been simulated using replica-exchange mol. dynamics with four different force fields.  The authors' recently developed residue-specific force fields RSFF1 and RSFF2 can correctly identify the crystal-like conformations of more than half CPs as the most populated conformation.  The RSFF2 performs the best, which consistently predicts the crystal structures of 17 out of 20 CPs with RMSD < 1.1 Å.  The authors also compared the backbone (φ,ψ) sampling of residues in CPs which those in short linear peptides and in globular proteins.  In general, unlike linear peptides, CPs have local conformational free energies and entropies quite similar to globular proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGooeUP-aaOZCrVg90H21EOLACvtfcHk0lhRheSfYDDvsA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XntFCiu74%253D&md5=d9c4be040be98396c50ae44a2fc97fbb</span></div><a href="/servlet/linkout?suffix=cit100&amp;dbid=16384&amp;doi=10.1021%2Facs.jpclett.6b00452&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jpclett.6b00452%26sid%3Dliteratum%253Aachs%26aulast%3DGeng%26aufirst%3DH.%26aulast%3DJiang%26aufirst%3DF.%26aulast%3DWu%26aufirst%3DY.-D.%26atitle%3DAccurate%2520Structure%2520Prediction%2520and%2520Conformational%2520Analysis%2520of%2520Cyclic%2520Peptides%2520with%2520Residue-Specific%2520Force%2520Fields%26jtitle%3DJ.%2520Phys.%2520Chem.%2520Lett.%26date%3D2016%26volume%3D7%26spage%3D1805%26epage%3D1810%26doi%3D10.1021%2Facs.jpclett.6b00452" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref101"><div class="reference"><strong class="refLabel"><a href="#ref101" class="rightTabRefNumLink">101</a></strong><div class="NLM_citation" id="rightTab-cit101"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gil-Ley, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bussi, G.</span></span> <span> </span><span class="NLM_article-title">Enhanced Conformational Sampling Using Replica Exchange with Collective-Variable Tempering</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">1077</span>– <span class="NLM_lpage">1085</span>, <span class="refDoi"> DOI: 10.1021/ct5009087</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct5009087" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitlKrsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=1077-1085&author=A.+Gil-Leyauthor=G.+Bussi&title=Enhanced+Conformational+Sampling+Using+Replica+Exchange+with+Collective-Variable+Tempering&doi=10.1021%2Fct5009087"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit101R"><div class="casContent"><span class="casTitleNuber">101</span><div class="casTitle"><span class="NLM_cas:atitle">Enhanced Conformational Sampling Using Replica Exchange with Collective-Variable Tempering</span></div><div class="casAuthors">Gil-Ley, Alejandro; Bussi, Giovanni</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1077-1085</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The computational study of conformational transitions in RNA and proteins with atomistic mol. dynamics often requires suitable enhanced sampling techniques.  The authors here introduce a novel method where concurrent metadynamics are integrated in a Hamiltonian replica-exchange scheme.  The ladder of replicas is built with different strengths of the bias potential exploiting the tunability of well-tempered metadynamics.  Using this method, free-energy barriers of individual collective variables are significantly reduced compared with simple force-field scaling.  The introduced methodol. is flexible and allows adaptive bias potentials to be self-consistently constructed for a large no. of simple collective variables, such as distances and dihedral angles.  The method is tested on alanine dipeptide and applied to the difficult problem of conformational sampling in a tetranucleotide.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp5umzwBtWIALVg90H21EOLACvtfcHk0lhRheSfYDDvsA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitlKrsr8%253D&md5=e22aee8989f57a9d01a6f111cded9b5c</span></div><a href="/servlet/linkout?suffix=cit101&amp;dbid=16384&amp;doi=10.1021%2Fct5009087&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct5009087%26sid%3Dliteratum%253Aachs%26aulast%3DGil-Ley%26aufirst%3DA.%26aulast%3DBussi%26aufirst%3DG.%26atitle%3DEnhanced%2520Conformational%2520Sampling%2520Using%2520Replica%2520Exchange%2520with%2520Collective-Variable%2520Tempering%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2015%26volume%3D11%26spage%3D1077%26epage%3D1085%26doi%3D10.1021%2Fct5009087" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref102"><div class="reference"><strong class="refLabel"><a href="#ref102" class="rightTabRefNumLink">102</a></strong><div class="NLM_citation" id="rightTab-cit102"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hosseinzadeh, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhardwaj, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulligan, V. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shortridge, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craven, T. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pardo-Avila, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rettie, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silva, D.-A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ibrahim, Y. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Webb, I. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cort, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adkins, J. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Varani, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baker, D.</span></span> <span> </span><span class="NLM_article-title">Comprehensive Computational Design of Ordered Peptide Macrocycles</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>358</i></span>,  <span class="NLM_fpage">1461</span>– <span class="NLM_lpage">1466</span>, <span class="refDoi"> DOI: 10.1126/science.aap7577</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1126%2Fscience.aap7577" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=29242347" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFGmtr3I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=358&publication_year=2017&pages=1461-1466&author=P.+Hosseinzadehauthor=G.+Bhardwajauthor=V.+K.+Mulliganauthor=M.+D.+Shortridgeauthor=T.+W.+Cravenauthor=F.+Pardo-Avilaauthor=S.+A.+Rettieauthor=D.+E.+Kimauthor=D.-A.+Silvaauthor=Y.+M.+Ibrahimauthor=I.+K.+Webbauthor=J.+R.+Cortauthor=J.+N.+Adkinsauthor=G.+Varaniauthor=D.+Baker&title=Comprehensive+Computational+Design+of+Ordered+Peptide+Macrocycles&doi=10.1126%2Fscience.aap7577"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit102R"><div class="casContent"><span class="casTitleNuber">102</span><div class="casTitle"><span class="NLM_cas:atitle">Comprehensive computational design of ordered peptide macrocycles</span></div><div class="casAuthors">Hosseinzadeh, Parisa; Bhardwaj, Gaurav; Mulligan, Vikram Khipple; Shortridge, Matthew D.; Craven, Timothy W.; Pardo-Avila, Fatima; Rettie, Stephen A.; Kim, David E.; Silva, Daniel-Adriano; Ibrahim, Yehia M.; Webb, Ian K.; Cort, John R.; Adkins, Joshua N.; Varani, Gabriele; Baker, David</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">358</span>
        (<span class="NLM_cas:issue">6369</span>),
    <span class="NLM_cas:pages">1461-1466</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Mixed-chirality peptide macrocycles such as cyclosporine are among the most potent therapeutics identified to date, but there is currently no way to systematically search the structural space spanned by such compds.  Natural proteins do not provide a useful guide: Peptide macrocycles lack regular secondary structures and hydrophobic cores, and can contain local structures not accessible with L-amino acids.  Here, we enumerate the stable structures that can be adopted by macrocyclic peptides composed of L- and D-amino acids by near-exhaustive backbone sampling followed by sequence design and energy landscape calcns.  We identify more than 200 designs predicted to fold into single stable structures, many times more than the no. of currently available unbound peptide macrocycle structures.  NMR structures of 9 of 12 designed 7- to 10-residue macrocycles, and three 11- to 14-residue bicyclic designs, are close to the computational models.  Our results provide a nearly complete coverage of the rich space of structures possible for short peptide macrocycles and vastly increase the available starting scaffolds for both rational drug design and library selection methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGorgJTppniqh7Vg90H21EOLACvtfcHk0lhRheSfYDDvsA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFGmtr3I&md5=fd13cbca720f657d91131e28e78fd435</span></div><a href="/servlet/linkout?suffix=cit102&amp;dbid=16384&amp;doi=10.1126%2Fscience.aap7577&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aap7577%26sid%3Dliteratum%253Aachs%26aulast%3DHosseinzadeh%26aufirst%3DP.%26aulast%3DBhardwaj%26aufirst%3DG.%26aulast%3DMulligan%26aufirst%3DV.%2BK.%26aulast%3DShortridge%26aufirst%3DM.%2BD.%26aulast%3DCraven%26aufirst%3DT.%2BW.%26aulast%3DPardo-Avila%26aufirst%3DF.%26aulast%3DRettie%26aufirst%3DS.%2BA.%26aulast%3DKim%26aufirst%3DD.%2BE.%26aulast%3DSilva%26aufirst%3DD.-A.%26aulast%3DIbrahim%26aufirst%3DY.%2BM.%26aulast%3DWebb%26aufirst%3DI.%2BK.%26aulast%3DCort%26aufirst%3DJ.%2BR.%26aulast%3DAdkins%26aufirst%3DJ.%2BN.%26aulast%3DVarani%26aufirst%3DG.%26aulast%3DBaker%26aufirst%3DD.%26atitle%3DComprehensive%2520Computational%2520Design%2520of%2520Ordered%2520Peptide%2520Macrocycles%26jtitle%3DScience%26date%3D2017%26volume%3D358%26spage%3D1461%26epage%3D1466%26doi%3D10.1126%2Fscience.aap7577" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref103"><div class="reference"><strong class="refLabel"><a href="#ref103" class="rightTabRefNumLink">103</a></strong><div class="NLM_citation" id="rightTab-cit103"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kamenik, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lessel, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuchs, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fox, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liedl, K. R.</span></span> <span> </span><span class="NLM_article-title">Peptidic Macrocycles - Conformational Sampling and Thermodynamic Characterization</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">982</span>– <span class="NLM_lpage">992</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.8b00097</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.8b00097" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC1cXnsFantrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2018&pages=982-992&author=A.+S.+Kamenikauthor=U.+Lesselauthor=J.+E.+Fuchsauthor=T.+Foxauthor=K.+R.+Liedl&title=Peptidic+Macrocycles+-+Conformational+Sampling+and+Thermodynamic+Characterization&doi=10.1021%2Facs.jcim.8b00097"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit103R"><div class="casContent"><span class="casTitleNuber">103</span><div class="casTitle"><span class="NLM_cas:atitle">Peptidic Macrocycles - Conformational Sampling and Thermodynamic Characterization</span></div><div class="casAuthors">Kamenik, Anna S.; Lessel, Uta; Fuchs, Julian E.; Fox, Thomas; Liedl, Klaus R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">982-992</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Macrocycles are of considerable interest as highly specific drug candidates, yet they challenge std. conformer generators with their large no. of rotatable bonds and conformational restrictions.  Here, we present a mol. dynamics-based routine that bypasses current limitations in conformational sampling and extensively profiles the free energy landscape of peptidic macrocycles in soln.  We perform accelerated mol. dynamics simulations to capture a diverse conformational ensemble.  By applying an energetic cutoff, followed by geometric clustering, we demonstrate the striking robustness and efficiency of the approach in identifying highly populated conformational states of cyclic peptides.  The resulting structural and thermodn. information is benchmarked against interproton distances from NMR expts. and conformational states identified by X-ray crystallog.  Using three different model systems of varying size and flexibility, we show that the method reliably reproduces exptl. detd. structural ensembles and is capable of identifying key conformational states that include the bioactive conformation.  Thus, the described approach is a robust method to generate conformations of peptidic macrocycles and holds promise for structure-based drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqFlE0cJ41tmrVg90H21EOLACvtfcHk0liMjix_Ew0fkw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXnsFantrY%253D&md5=4c582a4036f17ac76c05d3e9e8718c02</span></div><a href="/servlet/linkout?suffix=cit103&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.8b00097&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.8b00097%26sid%3Dliteratum%253Aachs%26aulast%3DKamenik%26aufirst%3DA.%2BS.%26aulast%3DLessel%26aufirst%3DU.%26aulast%3DFuchs%26aufirst%3DJ.%2BE.%26aulast%3DFox%26aufirst%3DT.%26aulast%3DLiedl%26aufirst%3DK.%2BR.%26atitle%3DPeptidic%2520Macrocycles%2520-%2520Conformational%2520Sampling%2520and%2520Thermodynamic%2520Characterization%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2018%26volume%3D58%26spage%3D982%26epage%3D992%26doi%3D10.1021%2Facs.jcim.8b00097" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref104"><div class="reference"><strong class="refLabel"><a href="#ref104" class="rightTabRefNumLink">104</a></strong><div class="NLM_citation" id="rightTab-cit104"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Labute, P.</span></span> <span> </span><span class="NLM_article-title">LowModeMD—Implicit Low-Mode Velocity Filtering Applied to Conformational Search of Macrocycles and Protein Loops</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">792</span>– <span class="NLM_lpage">800</span>, <span class="refDoi"> DOI: 10.1021/ci900508k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci900508k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref104/cit104&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC3cXlsVSksrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=792-800&author=P.+Labute&title=LowModeMD%E2%80%94Implicit+Low-Mode+Velocity+Filtering+Applied+to+Conformational+Search+of+Macrocycles+and+Protein+Loops&doi=10.1021%2Fci900508k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit104R"><div class="casContent"><span class="casTitleNuber">104</span><div class="casTitle"><span class="NLM_cas:atitle">LowModeMD - Implicit Low-Mode Velocity Filtering Applied to Conformational Search of Macrocycles and Protein Loops</span></div><div class="casAuthors">Labute, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">792-800</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We present a method for conformational search of complex mol. systems such as macrocycles and protein loops.  The method is based on perturbing an existing conformation along a mol. dynamics trajectory using initial at. velocities with kinetic energy concd. on the low-frequency vibrational modes, followed by energy minimization.  A novel Chebyshev polynomial filter is used to heavily dampen the high-frequency components of a randomly generated Maxwell-Boltzmann velocity vector.  The method is very efficient, even for large systems; it is straightforward to implement and requires only std. force-field energy and gradient evaluations.  The results of several computational expts. suggest that the method is capable of efficiently sampling low-strain energy conformations of complex systems with nontrivial nonbonded interaction networks.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfWZjZhFqqUbVg90H21EOLACvtfcHk0liMjix_Ew0fkw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXlsVSksrk%253D&md5=bb800eb2f9635a661578b978dae9da0a</span></div><a href="/servlet/linkout?suffix=cit104&amp;dbid=16384&amp;doi=10.1021%2Fci900508k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci900508k%26sid%3Dliteratum%253Aachs%26aulast%3DLabute%26aufirst%3DP.%26atitle%3DLowModeMD%25E2%2580%2594Implicit%2520Low-Mode%2520Velocity%2520Filtering%2520Applied%2520to%2520Conformational%2520Search%2520of%2520Macrocycles%2520and%2520Protein%2520Loops%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D792%26epage%3D800%26doi%3D10.1021%2Fci900508k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref105"><div class="reference"><strong class="refLabel"><a href="#ref105" class="rightTabRefNumLink">105</a></strong><div class="NLM_citation" id="rightTab-cit105"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Olanders, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alogheli, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brandt, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karlén, A.</span></span> <span> </span><span class="NLM_article-title">Conformational Analysis of Macrocycles: Comparing General and Specialized Methods</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">231</span>– <span class="NLM_lpage">252</span>, <span class="refDoi"> DOI: 10.1007/s10822-020-00277-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref105/cit105&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1007%2Fs10822-020-00277-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref105/cit105&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=31965404" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref105/cit105&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BB3cXis1Sgtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2020&pages=231-252&author=G.+Olandersauthor=H.+Alogheliauthor=P.+Brandtauthor=A.+Karl%C3%A9n&title=Conformational+Analysis+of+Macrocycles%3A+Comparing+General+and+Specialized+Methods&doi=10.1007%2Fs10822-020-00277-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit105R"><div class="casContent"><span class="casTitleNuber">105</span><div class="casTitle"><span class="NLM_cas:atitle">Conformational analysis of macrocycles: comparing general and specialized methods</span></div><div class="casAuthors">Olanders, Gustav; Alogheli, Hiba; Brandt, Peter; Karlen, Anders</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">231-252</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Using macrocycles extd. from 44 macrocycle-protein X-ray crystallog. complexes, we evaluated each method based on their ability to (i) generate unique conformers, (ii) generate unique macrocycle ring conformations, (iii) identify global energy min., (iv) identify conformers similar to X-ray ligand conformation after Protein Prepn. Wizard treatment and to X-rayppw ring conformation.  In order to study relative energies of bioactive conformations, energy differences between global energy min. and energy minimized X-rayppw structures and global energy min. and MCMM-Exhaustive generated conformers closest to the X-rayppw structure, were calcd. and analyzed.  To assess the performance of the methods, they were compared to exhaustive MCMM search using 1,000,000 search steps for each of 44 macrocycles.  Prior to our anal., we also investigated if general search methods MCMM and MTLMOD could also be optimized for macrocycle conformational sampling.  Taken together, our work concludes that more general methods can be optimized for macrocycle modeling by slightly adjusting settings around ring closure bond.  In most cases, MCMM and MTLMOD with either std. or enhanced settings performed well in comparison to more specialized macrocycle sampling methods MD/LLMOD and PRIME-MCS.  MD/LLMOD emerged as most efficient method for generating global energy min.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpw7Rdc4sIY-LVg90H21EOLACvtfcHk0liMjix_Ew0fkw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXis1Sgtr4%253D&md5=4df4e45f11974acae7695919de562574</span></div><a href="/servlet/linkout?suffix=cit105&amp;dbid=16384&amp;doi=10.1007%2Fs10822-020-00277-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-020-00277-2%26sid%3Dliteratum%253Aachs%26aulast%3DOlanders%26aufirst%3DG.%26aulast%3DAlogheli%26aufirst%3DH.%26aulast%3DBrandt%26aufirst%3DP.%26aulast%3DKarl%25C3%25A9n%26aufirst%3DA.%26atitle%3DConformational%2520Analysis%2520of%2520Macrocycles%253A%2520Comparing%2520General%2520and%2520Specialized%2520Methods%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2020%26volume%3D34%26spage%3D231%26epage%3D252%26doi%3D10.1007%2Fs10822-020-00277-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref106"><div class="reference"><strong class="refLabel"><a href="#ref106" class="rightTabRefNumLink">106</a></strong><div class="NLM_citation" id="rightTab-cit106"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sindhikara, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spronk, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Day, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borrelli, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheney, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Posy, S. L.</span></span> <span> </span><span class="NLM_article-title">Improving Accuracy, Diversity, and Speed with Prime Macrocycle Conformational Sampling</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">1881</span>– <span class="NLM_lpage">1894</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.7b00052</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.7b00052" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref106/cit106&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtF2ksbrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2017&pages=1881-1894&author=D.+Sindhikaraauthor=S.+A.+Spronkauthor=T.+Dayauthor=K.+Borrelliauthor=D.+L.+Cheneyauthor=S.+L.+Posy&title=Improving+Accuracy%2C+Diversity%2C+and+Speed+with+Prime+Macrocycle+Conformational+Sampling&doi=10.1021%2Facs.jcim.7b00052"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit106R"><div class="casContent"><span class="casTitleNuber">106</span><div class="casTitle"><span class="NLM_cas:atitle">Improving Accuracy, Diversity, and Speed with Prime Macrocycle Conformational Sampling</span></div><div class="casAuthors">Sindhikara, Dan; Spronk, Steven A.; Day, Tyler; Borrelli, Ken; Cheney, Daniel L.; Posy, Shana L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1881-1894</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A novel method for exploring macrocycle conformational space, Prime Macrocycle Conformational Sampling (Prime-MCS), is introduced and evaluated in the context of other available algorithms (Mol. Dynamics, LowModeMD in MOE, and MacroModel Baseline Search).  The algorithms were benchmarked on a dataset of 208 macrocycles which was curated for diversity from the Cambridge Structural Database, the Protein Databank, and the Biol. Interesting Mol. Ref. Dictionary.  The algorithms were evaluated in terms of accuracy (ability to reproduce the crystal structure), diversity (coverage of conformational space), and computational speed.  Prime-MCS most reliably reproduced crystallog. structures for RMSD thresholds >1.0 Å, most often produced the most diverse conformational ensemble, and was most often the fastest algorithm.  Detailed anal. and examn. of both typical and outlier cases were performed to reveal characteristics, shortcomings, expected performance, and complementarity of the methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfeDx3IcpZD7Vg90H21EOLACvtfcHk0li5pAOgoIs8kg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtF2ksbrJ&md5=a2b20d25c2ad2cede1c3a5fd0b5840ce</span></div><a href="/servlet/linkout?suffix=cit106&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.7b00052&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.7b00052%26sid%3Dliteratum%253Aachs%26aulast%3DSindhikara%26aufirst%3DD.%26aulast%3DSpronk%26aufirst%3DS.%2BA.%26aulast%3DDay%26aufirst%3DT.%26aulast%3DBorrelli%26aufirst%3DK.%26aulast%3DCheney%26aufirst%3DD.%2BL.%26aulast%3DPosy%26aufirst%3DS.%2BL.%26atitle%3DImproving%2520Accuracy%252C%2520Diversity%252C%2520and%2520Speed%2520with%2520Prime%2520Macrocycle%2520Conformational%2520Sampling%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2017%26volume%3D57%26spage%3D1881%26epage%3D1894%26doi%3D10.1021%2Facs.jcim.7b00052" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref107"><div class="reference"><strong class="refLabel"><a href="#ref107" class="rightTabRefNumLink">107</a></strong><div class="NLM_citation" id="rightTab-cit107"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sindhikara, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borrelli, K.</span></span> <span> </span><span class="NLM_article-title">High Throughput Evaluation of Macrocyclization Strategies for Conformer Stabilization</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">6585</span> <span class="refDoi"> DOI: 10.1038/s41598-018-24766-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1038%2Fs41598-018-24766-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=29700331" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A280%3ADC%252BC1MjntFyksg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2018&author=D.+Sindhikaraauthor=K.+Borrelli&title=High+Throughput+Evaluation+of+Macrocyclization+Strategies+for+Conformer+Stabilization&doi=10.1038%2Fs41598-018-24766-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit107R"><div class="casContent"><span class="casTitleNuber">107</span><div class="casTitle"><span class="NLM_cas:atitle">High throughput evaluation of macrocyclization strategies for conformer stabilization</span></div><div class="casAuthors">Sindhikara Dan; Borrelli Ken</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">6585</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">While macrocyclization of a linear compound to stabilize a known bioactive conformation can be a useful strategy to increase binding potency, the difficulty of macrocycle synthesis can limit the throughput of such strategies.  Thus computational techniques may offer the higher throughput required to screen large numbers of compounds.  Here we introduce a method for evaluating the propensity of a macrocyclic compound to adopt a conformation similar that of a known active linear compound in the binding site.  This method can be used as a fast screening tool for prioritizing macrocycles by leveraging the assumption that the propensity for the known bioactive substructural conformation relates to the affinity.  While this method cannot to identify new interactions not present in the known linear compound, it could quickly differentiate compounds where the three dimensional geometries imposed by the macrocyclization prevent adoption of conformations with the same contacts as the linear compound in their conserved region.  Here we report the implementation of this method using an RMSD-based structural descriptor and a Boltzmann-weighted propensity calculation and apply it retrospectively to three macrocycle linker optimization design projects.  We found the method performs well in terms of prioritizing more potent compounds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSJc7XEAu3kJ9yzdcP07UmufW6udTcc2ebzZPucouW46Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1MjntFyksg%253D%253D&md5=a2391f1e89175d1663342c8919e24743</span></div><a href="/servlet/linkout?suffix=cit107&amp;dbid=16384&amp;doi=10.1038%2Fs41598-018-24766-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-018-24766-5%26sid%3Dliteratum%253Aachs%26aulast%3DSindhikara%26aufirst%3DD.%26aulast%3DBorrelli%26aufirst%3DK.%26atitle%3DHigh%2520Throughput%2520Evaluation%2520of%2520Macrocyclization%2520Strategies%2520for%2520Conformer%2520Stabilization%26jtitle%3DSci.%2520Rep.%26date%3D2018%26volume%3D8%26doi%3D10.1038%2Fs41598-018-24766-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref108"><div class="reference"><strong class="refLabel"><a href="#ref108" class="rightTabRefNumLink">108</a></strong><div class="NLM_citation" id="rightTab-cit108"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Watts, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dalal, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tebben, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheney, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shelley, J. C.</span></span> <span> </span><span class="NLM_article-title">Macrocycle Conformational Sampling with MacroModel</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">2680</span>– <span class="NLM_lpage">2696</span>, <span class="refDoi"> DOI: 10.1021/ci5001696</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci5001696" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref108/cit108&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsFKnsbbN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2014&pages=2680-2696&author=K.+S.+Wattsauthor=P.+Dalalauthor=A.+J.+Tebbenauthor=D.+L.+Cheneyauthor=J.+C.+Shelley&title=Macrocycle+Conformational+Sampling+with+MacroModel&doi=10.1021%2Fci5001696"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit108R"><div class="casContent"><span class="casTitleNuber">108</span><div class="casTitle"><span class="NLM_cas:atitle">Macrocycle Conformational Sampling with MacroModel</span></div><div class="casAuthors">Watts, K. Shawn; Dalal, Pranav; Tebben, Andrew J.; Cheney, Daniel L.; Shelley, John C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2680-2696</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Sampling low energy conformations of macrocycles is challenging due to the large size of many of these mols. and the constraints imposed by the macrocycle.  We present a new conformational search method (implemented in MacroModel) that uses brief MD simulations followed by minimization and normal-mode search steps.  The method was parametrized using a set of 100 macrocycles from the PDB and CSD.  It was then tested on a publicly available data set for which there are published results using alternative methods; we found that when the same force field is used (in this case MMFFs in vacuum), our method tended to identify conformations with lower energies than what the other methods identified.  The performance on a new set of 50 macrocycles from the PDB and CSD was also quite good; the mean and median RMSD values for just the ring atoms were 0.60 and 0.33 Å, resp.  However, the RMSD values for macrocycles with more than 30 ring-atoms were quite a bit larger compared to the smaller macrocycles.  Possible origins for this and ideas for improving the performance on very large macrocycles are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqaBKXPDzoVcLVg90H21EOLACvtfcHk0li5pAOgoIs8kg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsFKnsbbN&md5=612c949e25807bbf6f850c2a2c37b821</span></div><a href="/servlet/linkout?suffix=cit108&amp;dbid=16384&amp;doi=10.1021%2Fci5001696&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci5001696%26sid%3Dliteratum%253Aachs%26aulast%3DWatts%26aufirst%3DK.%2BS.%26aulast%3DDalal%26aufirst%3DP.%26aulast%3DTebben%26aufirst%3DA.%2BJ.%26aulast%3DCheney%26aufirst%3DD.%2BL.%26aulast%3DShelley%26aufirst%3DJ.%2BC.%26atitle%3DMacrocycle%2520Conformational%2520Sampling%2520with%2520MacroModel%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2014%26volume%3D54%26spage%3D2680%26epage%3D2696%26doi%3D10.1021%2Fci5001696" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref109"><div class="reference"><strong class="refLabel"><a href="#ref109" class="rightTabRefNumLink">109</a></strong><div class="NLM_citation" id="rightTab-cit109"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Witek, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keller, B. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blatter, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meissner, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riniker, S.</span></span> <span> </span><span class="NLM_article-title">Kinetic Models of Cyclosporin A in Polar and Apolar Environments Reveal Multiple Congruent Conformational States</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">1547</span>– <span class="NLM_lpage">1562</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.6b00251</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.6b00251" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFaisL7L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2016&pages=1547-1562&author=J.+Witekauthor=B.+G.+Kellerauthor=M.+Blatterauthor=A.+Meissnerauthor=T.+Wagnerauthor=S.+Riniker&title=Kinetic+Models+of+Cyclosporin+A+in+Polar+and+Apolar+Environments+Reveal+Multiple+Congruent+Conformational+States&doi=10.1021%2Facs.jcim.6b00251"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit109R"><div class="casContent"><span class="casTitleNuber">109</span><div class="casTitle"><span class="NLM_cas:atitle">Kinetic Models of Cyclosporin A in Polar and Apolar Environments Reveal Multiple Congruent Conformational States</span></div><div class="casAuthors">Witek, Jagna; Keller, Bettina G.; Blatter, Markus; Meissner, Axel; Wagner, Trixie; Riniker, Sereina</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1547-1562</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The membrane permeability of cyclic peptides and peptidomimetics, which are generally larger and more complex than typical drug mols., is likely strongly influenced by the conformational behavior of these compds. in polar and apolar environments.  The size and complexity of peptides often limit their bioavailability, but there are known examples of peptide natural products such as cyclosporin A (CsA) that can cross cell membranes by passive diffusion.  CsA is an undecapeptide with seven methylated backbone amides.  Its crystal structure shows a "closed" twisted β-pleated sheet conformation with four intramol. hydrogen bonds that is also obsd. in NMR measurements of CsA in chloroform.  When binding to its target cyclophilin, on the other hand, CsA adopts an "open" conformation without intramol. hydrogen bonds.  In this study, we attempted to sample the complete conformational space of CsA in chloroform and in water by mol. dynamics simulations in order to better understand its conformational behavior in these two environments and to rationalize the good membrane permeability of CsA obsd. exptl.  From 10 μs mol. dynamics simulations in each solvent, Markov state models were constructed to characterize the metastable conformational states.  The model in chloroform is compared to nuclear Overhauser effect NMR spectroscopy data reported in this study and taken from the literature.  The conformational landscapes in the two solvents show significant overlap but also clearly distinct features.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr618Uz1zHQkrVg90H21EOLACvtfcHk0li5pAOgoIs8kg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFaisL7L&md5=0274259c6380217a27c5e0f4649dbf6a</span></div><a href="/servlet/linkout?suffix=cit109&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.6b00251&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.6b00251%26sid%3Dliteratum%253Aachs%26aulast%3DWitek%26aufirst%3DJ.%26aulast%3DKeller%26aufirst%3DB.%2BG.%26aulast%3DBlatter%26aufirst%3DM.%26aulast%3DMeissner%26aufirst%3DA.%26aulast%3DWagner%26aufirst%3DT.%26aulast%3DRiniker%26aufirst%3DS.%26atitle%3DKinetic%2520Models%2520of%2520Cyclosporin%2520A%2520in%2520Polar%2520and%2520Apolar%2520Environments%2520Reveal%2520Multiple%2520Congruent%2520Conformational%2520States%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2016%26volume%3D56%26spage%3D1547%26epage%3D1562%26doi%3D10.1021%2Facs.jcim.6b00251" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref110"><div class="reference"><strong class="refLabel"><a href="#ref110" class="rightTabRefNumLink">110</a></strong><div class="NLM_citation" id="rightTab-cit110"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Allen, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dokholyan, N. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bowers, A. A.</span></span> <span> </span><span class="NLM_article-title">Dynamic Docking of Conformationally Constrained Macrocycles: Methods and Applications</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">10</span>– <span class="NLM_lpage">24</span>, <span class="refDoi"> DOI: 10.1021/acschembio.5b00663</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b00663" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref110/cit110&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVCjtbbL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=10-24&author=S.+E.+Allenauthor=N.+V.+Dokholyanauthor=A.+A.+Bowers&title=Dynamic+Docking+of+Conformationally+Constrained+Macrocycles%3A+Methods+and+Applications&doi=10.1021%2Facschembio.5b00663"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit110R"><div class="casContent"><span class="casTitleNuber">110</span><div class="casTitle"><span class="NLM_cas:atitle">Dynamic Docking of Conformationally Constrained Macrocycles: Methods and Applications</span></div><div class="casAuthors">Allen, Scott E.; Dokholyan, Nikolay V.; Bowers, Albert A.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">10-24</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Many natural products consist of large and flexible macrocycles that engage their targets via multiple contact points.  This combination of contained flexibility and large contact area often allows natural products to bind at target surfaces rather than deep pockets, making them attractive scaffolds for inhibiting protein-protein interactions and other challenging therapeutic targets.  The increasing ability to manipulate such compds. either biosynthetically or via semisynthetic modification means that these compds. can now be considered as starting points for medchem campaigns rather than solely as ends.  Modern medchem benefits substantially from rational improvements made on the basis of mol. docking.  As such, docking methods have been enhanced in recent years to deal with the complicated binding modalities and flexible scaffolds of macrocyclic natural products and natural product-like structures.  Here, we comprehensively review methods for treating and docking these large macrocyclic scaffolds and discuss some of the resulting advances in medicinal chem.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8sOyiW4teIbVg90H21EOLACvtfcHk0liEGYmhcEQfQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVCjtbbL&md5=21f4cbf0a53f70cd0e7b7a7ad88cd5be</span></div><a href="/servlet/linkout?suffix=cit110&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b00663&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b00663%26sid%3Dliteratum%253Aachs%26aulast%3DAllen%26aufirst%3DS.%2BE.%26aulast%3DDokholyan%26aufirst%3DN.%2BV.%26aulast%3DBowers%26aufirst%3DA.%2BA.%26atitle%3DDynamic%2520Docking%2520of%2520Conformationally%2520Constrained%2520Macrocycles%253A%2520Methods%2520and%2520Applications%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2016%26volume%3D11%26spage%3D10%26epage%3D24%26doi%3D10.1021%2Facschembio.5b00663" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref111"><div class="reference"><strong class="refLabel"><a href="#ref111" class="rightTabRefNumLink">111</a></strong><div class="NLM_citation" id="rightTab-cit111"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Alogheli, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olanders, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaal, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brandt, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karlén, A.</span></span> <span> </span><span class="NLM_article-title">Docking of Macrocycles: Comparing Rigid and Flexible Docking in Glide</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">190</span>– <span class="NLM_lpage">202</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.6b00443</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.6b00443" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref111/cit111&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC2sXosF2htA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2017&pages=190-202&author=H.+Alogheliauthor=G.+Olandersauthor=W.+Schaalauthor=P.+Brandtauthor=A.+Karl%C3%A9n&title=Docking+of+Macrocycles%3A+Comparing+Rigid+and+Flexible+Docking+in+Glide&doi=10.1021%2Facs.jcim.6b00443"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit111R"><div class="casContent"><span class="casTitleNuber">111</span><div class="casTitle"><span class="NLM_cas:atitle">Docking of Macrocycles: Comparing Rigid and Flexible Docking in Glide</span></div><div class="casAuthors">Alogheli, Hiba; Olanders, Gustav; Schaal, Wesley; Brandt, Peter; Karlen, Anders</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">190-202</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In recent years, there has been an increased interest in using macrocyclic compds. for drug discovery and development.  For docking of these commonly large and flexible compds. to be addressed, a screening and a validation set were assembled from the PDB consisting of 16 and 31 macrocycle-contg. protein complexes, resp.  The macrocycles were docked in Glide by rigid docking of pregenerated conformational ensembles produced by the macrocycle conformational sampling method (MCS) in Schr.ovrddot.odinger Release 2015-3 or by direct Glide flexible docking after performing ring-templating.  The two protocols were compared to rigid docking of pregenerated conformational ensembles produced by an exhaustive Monte Carlo multiple min. (MCMM) conformational search and a shorter MCMM conformational search (MCMM-short).  The docking accuracy was evaluated and expressed as the RMSD between the heavy atoms of the ligand as found in the X-ray structure after refinement and the poses obtained by the docking protocols.  The median RMSD values for top-scored poses of the screening set were 0.83, 0.80, 0.88, and 0.58 Å for MCMM, MCMM-short, MCS, and Glide flexible docking, resp.  There was no statistically significant difference in the performance between rigid docking of pregenerated conformations produced by the MCS and direct docking using Glide flexible docking.  However, the flexible docking protocol was 2-times faster in docking the screening set compared to that of the MCS protocol.  In a final study, the new Prime-MCS method was evaluated in Schr.ovrddot.odinger Release 2016-3.  This method is faster compared that of to MCS; however, the conformations generated were found to be suboptimal for rigid docking.  Therefore, on the basis of timing, accuracy, and ease of set up, std. Glide flexible docking with prior ring-templating is recommended over current gold std. protocols using rigid docking of pregenerated conformational ensembles.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVm5kNKieXLLVg90H21EOLACvtfcHk0liEGYmhcEQfQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXosF2htA%253D%253D&md5=2e27fa8f5247904089cf50cb443706c7</span></div><a href="/servlet/linkout?suffix=cit111&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.6b00443&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.6b00443%26sid%3Dliteratum%253Aachs%26aulast%3DAlogheli%26aufirst%3DH.%26aulast%3DOlanders%26aufirst%3DG.%26aulast%3DSchaal%26aufirst%3DW.%26aulast%3DBrandt%26aufirst%3DP.%26aulast%3DKarl%25C3%25A9n%26aufirst%3DA.%26atitle%3DDocking%2520of%2520Macrocycles%253A%2520Comparing%2520Rigid%2520and%2520Flexible%2520Docking%2520in%2520Glide%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2017%26volume%3D57%26spage%3D190%26epage%3D202%26doi%3D10.1021%2Facs.jcim.6b00443" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref112"><div class="reference"><strong class="refLabel"><a href="#ref112" class="rightTabRefNumLink">112</a></strong><div class="NLM_citation" id="rightTab-cit112"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Martin, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, I.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, A. W. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foloppe, N.</span></span> <span> </span><span class="NLM_article-title">Modelling the Binding Mode of Macrocycles: Docking and Conformational Sampling</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>28</i></span>, <span class="NLM_elocation-id">115143</span> <span class="refDoi"> DOI: 10.1016/j.bmc.2019.115143</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref112/cit112&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1016%2Fj.bmc.2019.115143" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref112/cit112&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=31771798" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref112/cit112&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC1MXit1Krtb7K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2020&author=S.+J.+Martinauthor=I.-J.+Chenauthor=A.+W.+E.+Chanauthor=N.+Foloppe&title=Modelling+the+Binding+Mode+of+Macrocycles%3A+Docking+and+Conformational+Sampling&doi=10.1016%2Fj.bmc.2019.115143"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit112R"><div class="casContent"><span class="casTitleNuber">112</span><div class="casTitle"><span class="NLM_cas:atitle">Modelling the binding mode of macrocycles: Docking and conformational sampling</span></div><div class="casAuthors">Martin, Sarah J.; Chen, I-Jen; Chan, A. W. Edith; Foloppe, Nicolas</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">115143</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Drug discovery is increasingly tackling challenging protein binding sites regarding mol. recognition and druggability, including shallow and solvent-exposed protein-protein interaction interfaces.  Macrocycles are emerging as promising chemotypes to modulate such sites.  Despite their chem. complexity, macrocycles comprise important drugs and offer advantages compared to non-cyclic analogs, hence the recent impetus in the medicinal chem. of macrocycles.  Elaboration of macrocycles, or constituent fragments, can strongly benefit from knowledge of their binding mode to a target.  When such information from X-ray crystallog. is elusive, computational docking can provide working models.  However, few studies have explored docking protocols for macrocycles, since conventional docking methods struggle with the conformational complexity of macrocycles, and also potentially with the shallower topol. of their binding sites.  Indeed, macrocycle binding mode prediction with the mainstream docking software GOLD has hardly been explored.  Here, we present an in-depth study of macrocycle docking with GOLD and the ChemPLP scores.  First, we summarize the thorough curation of a test set of 41 protein-macrocycle X-ray structures, raising the issue of lattice contacts with such systems.  Rigid docking of the known bioactive conformers was successful (three top ranked poses) for 92.7% of the systems, in absence of crystallog. waters.  Thus, without conformational search issues, scoring performed well.  However, docking success dropped to 29.3% with the GOLD built-in conformational search.  Yet, the success rate doubled to 58.5% when GOLD was supplied with extensive conformer ensembles docked rigidly.  The reasons for failure, sampling or scoring, were analyzed, exemplified with particular cases.  Overall, binding mode prediction of macrocycles remains challenging, but can be much improved with tailored protocols.  The anal. of the interplay between conformational sampling and docking will be relevant to the prospective modeling of macrocycles in general.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoO93W5YR9RgrVg90H21EOLACvtfcHk0liEGYmhcEQfQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXit1Krtb7K&md5=de91562a32d648806253408e126af8d1</span></div><a href="/servlet/linkout?suffix=cit112&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2019.115143&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2019.115143%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DS.%2BJ.%26aulast%3DChen%26aufirst%3DI.-J.%26aulast%3DChan%26aufirst%3DA.%2BW.%2BE.%26aulast%3DFoloppe%26aufirst%3DN.%26atitle%3DModelling%2520the%2520Binding%2520Mode%2520of%2520Macrocycles%253A%2520Docking%2520and%2520Conformational%2520Sampling%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2020%26volume%3D28%26doi%3D10.1016%2Fj.bmc.2019.115143" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref113"><div class="reference"><strong class="refLabel"><a href="#ref113" class="rightTabRefNumLink">113</a></strong><div class="NLM_citation" id="rightTab-cit113"><span> <i>Glide Schrödinger Release 2019-3 Glide</i>; <span class="NLM_publisher-name">Schrödinger, LLC</span>: <span class="NLM_publisher-loc">New York,
NY</span>, <span class="NLM_year">2019</span> (accessed Jan 31, 2018).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Glide+Schr%C3%B6dinger+Release+2019-3+Glide%3B+Schr%C3%B6dinger%2C+LLC%3A+New+York%2C%0ANY%2C+2019+%28accessed+Jan+31%2C+2018%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit113&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26btitle%3DGlide%2520Schr%25C3%25B6dinger%2520Release%25202019-3%2520Glide%26pub%3DSchr%25C3%25B6dinger%252C%2520LLC%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref114"><div class="reference"><strong class="refLabel"><a href="#ref114" class="rightTabRefNumLink">114</a></strong><div class="NLM_citation" id="rightTab-cit114"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ugur, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schroft, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marion, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glaser, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Antes, I.</span></span> <span> </span><span class="NLM_article-title">Predicting the Bioactive Conformations of Macrocycles: A Molecular Dynamics-Based Docking Procedure with DynaDock</span>. <i>J. Mol. Model.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>25</i></span>, <span class="NLM_elocation-id">197</span> <span class="refDoi"> DOI: 10.1007/s00894-019-4077-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref114/cit114&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1007%2Fs00894-019-4077-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref114/cit114&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=31222506" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref114/cit114&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A280%3ADC%252BB3M3ptVOhtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2019&author=I.+Ugurauthor=M.+Schroftauthor=A.+Marionauthor=M.+Glaserauthor=I.+Antes&title=Predicting+the+Bioactive+Conformations+of+Macrocycles%3A+A+Molecular+Dynamics-Based+Docking+Procedure+with+DynaDock&doi=10.1007%2Fs00894-019-4077-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit114R"><div class="casContent"><span class="casTitleNuber">114</span><div class="casTitle"><span class="NLM_cas:atitle">Predicting the bioactive conformations of macrocycles: a molecular dynamics-based docking procedure with DynaDock</span></div><div class="casAuthors">Ugur Ilke; Schroft Maja; Marion Antoine; Glaser Manuel; Antes Iris; Ugur Ilke; Marion Antoine</div><div class="citationInfo"><span class="NLM_cas:title">Journal of molecular modeling</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">197</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Macrocyclic compounds are of growing interest as a new class of therapeutics, especially as inhibitors binding to protein-protein interfaces.  As molecular modeling is a well-established complimentary tool in modern drug design, the number of attempts to develop reliable docking strategies and algorithms to accurately predict the binding mode of macrocycles is rising continuously.  Standard molecular docking approaches need to be adapted to this application, as a comprehensive yet efficient sampling of all ring conformations of the macrocycle is necessary.  To overcome this issue, we designed a molecular dynamics-based docking protocol for macrocycles, in which the challenging sampling step is addressed by conventional molecular dynamics (750 ns) simulations performed at moderately high temperature (370 K).  Consecutive flexible docking with the DynaDock approach based on multiple, pre-sampled ring conformations yields highly accurate poses with ligand RMSD values lower than 1.8 ÅA.  We further investigated the value of molecular dynamics-based complex stability estimations for pose selection and discuss its applicability in combination with standard binding free energy estimations for assessing the quality of poses in future blind docking studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTIFcR2EWdUqLGbPKh4T9eAfW6udTcc2eZF50RIZE5ga7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3M3ptVOhtg%253D%253D&md5=0c50fc5a1d0a305c82f3dd62376f4438</span></div><a href="/servlet/linkout?suffix=cit114&amp;dbid=16384&amp;doi=10.1007%2Fs00894-019-4077-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00894-019-4077-5%26sid%3Dliteratum%253Aachs%26aulast%3DUgur%26aufirst%3DI.%26aulast%3DSchroft%26aufirst%3DM.%26aulast%3DMarion%26aufirst%3DA.%26aulast%3DGlaser%26aufirst%3DM.%26aulast%3DAntes%26aufirst%3DI.%26atitle%3DPredicting%2520the%2520Bioactive%2520Conformations%2520of%2520Macrocycles%253A%2520A%2520Molecular%2520Dynamics-Based%2520Docking%2520Procedure%2520with%2520DynaDock%26jtitle%3DJ.%2520Mol.%2520Model.%26date%3D2019%26volume%3D25%26doi%3D10.1007%2Fs00894-019-4077-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref115"><div class="reference"><strong class="refLabel"><a href="#ref115" class="rightTabRefNumLink">115</a></strong><div class="NLM_citation" id="rightTab-cit115"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jantz, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Briem, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sommer, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rarey, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christ, C. D.</span></span> <span> </span><span class="NLM_article-title">Computational Macrocyclization: From de Novo Macrocycle Generation to Binding Affinity Estimation</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">1866</span>– <span class="NLM_lpage">1872</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201700478</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref115/cit115&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1002%2Fcmdc.201700478" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref115/cit115&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=28977738" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref115/cit115&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhslSktbnI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=1866-1872&author=V.+Wagnerauthor=L.+Jantzauthor=H.+Briemauthor=K.+Sommerauthor=M.+Rareyauthor=C.+D.+Christ&title=Computational+Macrocyclization%3A+From+de+Novo+Macrocycle+Generation+to+Binding+Affinity+Estimation&doi=10.1002%2Fcmdc.201700478"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit115R"><div class="casContent"><span class="casTitleNuber">115</span><div class="casTitle"><span class="NLM_cas:atitle">Computational Macrocyclization: From de novo Macrocycle Generation to Binding Affinity Estimation</span></div><div class="casAuthors">Wagner, Vincent; Jantz, Linda; Briem, Hans; Sommer, Kai; Rarey, Matthias; Christ, Clara D.</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">1866-1872</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Macrocycles play an increasing role in drug discovery, but their synthesis is often demanding.  Computational tools that suggest macrocyclization based on a known binding mode and that est. the binding affinity of these macrocycles could have a substantial impact on the medicinal chem. design process.  For both tasks, we established a workflow with high practical value.  For five diverse pharmaceutical targets we show that the effect of macrocyclization on binding can be calcd. robustly and accurately.  Applying this method to macrocycles designed by LigMac, a search tool for de novo macrocyclization, our results suggest that we have a robust protocol in hand to design macrocycles and prioritize them prior to synthesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonKcpGd8uDs7Vg90H21EOLACvtfcHk0ljAefFA7FBPeQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhslSktbnI&md5=30f49d08b16cd5473ea69de3e975ebee</span></div><a href="/servlet/linkout?suffix=cit115&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201700478&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201700478%26sid%3Dliteratum%253Aachs%26aulast%3DWagner%26aufirst%3DV.%26aulast%3DJantz%26aufirst%3DL.%26aulast%3DBriem%26aufirst%3DH.%26aulast%3DSommer%26aufirst%3DK.%26aulast%3DRarey%26aufirst%3DM.%26aulast%3DChrist%26aufirst%3DC.%2BD.%26atitle%3DComputational%2520Macrocyclization%253A%2520From%2520de%2520Novo%2520Macrocycle%2520Generation%2520to%2520Binding%2520Affinity%2520Estimation%26jtitle%3DChemMedChem%26date%3D2017%26volume%3D12%26spage%3D1866%26epage%3D1872%26doi%3D10.1002%2Fcmdc.201700478" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref116"><div class="reference"><strong class="refLabel"><a href="#ref116" class="rightTabRefNumLink">116</a></strong><div class="NLM_citation" id="rightTab-cit116"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yu, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sindhikara, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rask, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kimura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span> <span> </span><span class="NLM_article-title">Accurate and Reliable Prediction of the Binding Affinities of Macrocycles to Their Protein Targets</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">6290</span>– <span class="NLM_lpage">6300</span>, <span class="refDoi"> DOI: 10.1021/acs.jctc.7b00885</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jctc.7b00885" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref116/cit116&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsl2gs7fP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&pages=6290-6300&author=H.+S.+Yuauthor=Y.+Dengauthor=Y.+Wuauthor=D.+Sindhikaraauthor=A.+R.+Raskauthor=T.+Kimuraauthor=R.+Abelauthor=L.+Wang&title=Accurate+and+Reliable+Prediction+of+the+Binding+Affinities+of+Macrocycles+to+Their+Protein+Targets&doi=10.1021%2Facs.jctc.7b00885"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit116R"><div class="casContent"><span class="casTitleNuber">116</span><div class="casTitle"><span class="NLM_cas:atitle">Accurate and Reliable Prediction of the Binding Affinities of Macrocycles to Their Protein Targets</span></div><div class="casAuthors">Yu, Haoyu S.; Deng, Yuqing; Wu, Yujie; Sindhikara, Dan; Rask, Amy R.; Kimura, Takayuki; Abel, Robert; Wang, Lingle</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">6290-6300</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Macrocycles have been emerging as a very important drug class in the past few decades largely due to their expanded chem. diversity benefiting from advances in synthetic methods.  Macrocyclization has been recognized as an effective way to restrict the conformational space of acyclic small mol. inhibitors with the hope of improving potency, selectivity, and metabolic stability.  Because of their relatively larger size as compared to typical small mol. drugs and the complexity of the structures, efficient sampling of the accessible macrocycle conformational space and accurate prediction of their binding affinities to their target protein receptors poses a great challenge of central importance in computational macrocycle drug design.  In this article, the authors present a novel method for relative binding free energy calcns. between macrocycles with different ring sizes and between the macrocycles and their corresponding acyclic counterparts.  The authors have applied the method to seven pharmaceutically interesting data sets taken from recent drug discovery projects including 33 macrocyclic ligands covering a diverse chem. space.  The predicted binding free energies are in good agreement with exptl. data with an overall root-mean-square error (RMSE) of 0.94 kcal/mol.  This is to the authors' knowledge the first time where the free energy of the macrocyclization of linear mols. has been directly calcd. with rigorous physics-based free energy calcn. methods, and the authors anticipate the outstanding accuracy demonstrated here across a broad range of target classes may have significant implications for macrocycle drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoibClnk_AFJrVg90H21EOLACvtfcHk0ljAefFA7FBPeQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsl2gs7fP&md5=62d194fa339aaadd9c46611730dee5ef</span></div><a href="/servlet/linkout?suffix=cit116&amp;dbid=16384&amp;doi=10.1021%2Facs.jctc.7b00885&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jctc.7b00885%26sid%3Dliteratum%253Aachs%26aulast%3DYu%26aufirst%3DH.%2BS.%26aulast%3DDeng%26aufirst%3DY.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DSindhikara%26aufirst%3DD.%26aulast%3DRask%26aufirst%3DA.%2BR.%26aulast%3DKimura%26aufirst%3DT.%26aulast%3DAbel%26aufirst%3DR.%26aulast%3DWang%26aufirst%3DL.%26atitle%3DAccurate%2520and%2520Reliable%2520Prediction%2520of%2520the%2520Binding%2520Affinities%2520of%2520Macrocycles%2520to%2520Their%2520Protein%2520Targets%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2017%26volume%3D13%26spage%3D6290%26epage%3D6300%26doi%3D10.1021%2Facs.jctc.7b00885" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref117"><div class="reference"><strong class="refLabel"><a href="#ref117" class="rightTabRefNumLink">117</a></strong><div class="NLM_citation" id="rightTab-cit117"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wallraven, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holmelin, F. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glas, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hennig, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frolov, A. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grossmann, T. N.</span></span> <span> </span><span class="NLM_article-title">Adapting Free Energy Perturbation Simulations for Large Macrocyclic Ligands: How to Dissect Contributions from Direct Binding and Free Ligand Flexibility</span>. <i>Chem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">2269</span>– <span class="NLM_lpage">2276</span>, <span class="refDoi"> DOI: 10.1039/C9SC04705K</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref117/cit117&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1039%2FC9SC04705K" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref117/cit117&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=32180932" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref117/cit117&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtlWqsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=2269-2276&author=K.+Wallravenauthor=F.+L.+Holmelinauthor=A.+Glasauthor=S.+Hennigauthor=A.+I.+Frolovauthor=T.+N.+Grossmann&title=Adapting+Free+Energy+Perturbation+Simulations+for+Large+Macrocyclic+Ligands%3A+How+to+Dissect+Contributions+from+Direct+Binding+and+Free+Ligand+Flexibility&doi=10.1039%2FC9SC04705K"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit117R"><div class="casContent"><span class="casTitleNuber">117</span><div class="casTitle"><span class="NLM_cas:atitle">Adapting free energy perturbation simulations for large macrocyclic ligands: how to dissect contributions from direct binding and free ligand flexibility</span></div><div class="casAuthors">Wallraven, Kerstin; Holmelin, Fredrik L.; Glas, Adrian; Hennig, Sven; Frolov, Andrey I.; Grossmann, Tom N.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Science</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2269-2276</span>CODEN:
                <span class="NLM_cas:coden">CSHCCN</span>;
        ISSN:<span class="NLM_cas:issn">2041-6520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Large and flexible ligands gain increasing interest in the development of bioactive agents.  They challenge the applicability of computational ligand optimization strategies originally developed for small mols.  Free energy perturbation (FEP) is often used for predicting binding affinities of small mol. ligands, however, its use for more complex ligands remains limited.  Herein, we report the structure-based design of peptide macrocycles targeting the protein binding site of human adaptor protein 14-3-3.  We observe a surprisingly strong dependency of binding affinities on relatively small variations in substituent size.  FEP was performed to rationalize obsd. trends.  To account for insufficient convergence of FEP, restrained calcns. were performed and complemented with extensive REST MD simulations of the free ligands.  These calcns. revealed that changes in affinity originate both from altered direct interactions and conformational changes of the free ligand.  In addn., MD simulations provided the basis to rationalize unexpected trends in ligand lipophilicity.  We also verified the anticipated interaction site and binding mode for one of the high affinity ligands by X-ray crystallog.  The introduced fully-atomistic simulation protocol can be used to rationalize the development of structurally complex ligands which will support future ligand maturation efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrkl4saxYBokrVg90H21EOLACvtfcHk0ljumelDn8ly9Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtlWqsLo%253D&md5=d2190c7ddb2725f9fa508724bb1d5773</span></div><a href="/servlet/linkout?suffix=cit117&amp;dbid=16384&amp;doi=10.1039%2FC9SC04705K&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC9SC04705K%26sid%3Dliteratum%253Aachs%26aulast%3DWallraven%26aufirst%3DK.%26aulast%3DHolmelin%26aufirst%3DF.%2BL.%26aulast%3DGlas%26aufirst%3DA.%26aulast%3DHennig%26aufirst%3DS.%26aulast%3DFrolov%26aufirst%3DA.%2BI.%26aulast%3DGrossmann%26aufirst%3DT.%2BN.%26atitle%3DAdapting%2520Free%2520Energy%2520Perturbation%2520Simulations%2520for%2520Large%2520Macrocyclic%2520Ligands%253A%2520How%2520to%2520Dissect%2520Contributions%2520from%2520Direct%2520Binding%2520and%2520Free%2520Ligand%2520Flexibility%26jtitle%3DChem.%2520Sci.%26date%3D2020%26volume%3D11%26spage%3D2269%26epage%3D2276%26doi%3D10.1039%2FC9SC04705K" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref118"><div class="reference"><strong class="refLabel"><a href="#ref118" class="rightTabRefNumLink">118</a></strong><div class="NLM_citation" id="rightTab-cit118"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banks, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klicic, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mainz, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Repasky, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knoll, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shelley, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perry, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Francis, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shenkin, P. S.</span></span> <span> </span><span class="NLM_article-title">Glide: A New Approach for Rapid, Accurate Docking and Scoring. 1. Method and Assessment of Docking Accuracy</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">1739</span>– <span class="NLM_lpage">1749</span>, <span class="refDoi"> DOI: 10.1021/jm0306430</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0306430" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref118/cit118&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFyit74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1739-1749&author=R.+A.+Friesnerauthor=J.+L.+Banksauthor=R.+B.+Murphyauthor=T.+A.+Halgrenauthor=J.+J.+Klicicauthor=D.+T.+Mainzauthor=M.+P.+Repaskyauthor=E.+H.+Knollauthor=M.+Shelleyauthor=J.+K.+Perryauthor=D.+E.+Shawauthor=P.+Francisauthor=P.+S.+Shenkin&title=Glide%3A+A+New+Approach+for+Rapid%2C+Accurate+Docking+and+Scoring.+1.+Method+and+Assessment+of+Docking+Accuracy&doi=10.1021%2Fjm0306430"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit118R"><div class="casContent"><span class="casTitleNuber">118</span><div class="casTitle"><span class="NLM_cas:atitle">Glide: A new approach for rapid, accurate docking and scoring. 1. method and assessment of docking accuracy</span></div><div class="casAuthors">Friesner, Richard A.; Banks, Jay L.; Murphy, Robert B.; Halgren, Thomas A.; Klicic, Jasna J.; Mainz, Daniel T.; Repasky, Matthew P.; Knoll, Eric H.; Shelley, Mee; Perry, Jason K.; Shaw, David E.; Francis, Perry; Shenkin, Peter S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1739-1749</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Unlike other methods for docking ligands to the rigid 3D structure of a known protein receptor, Glide approximates a complete systematic search of the conformational, orientational, and positional space of the docked ligand.  In this search, an initial rough positioning and scoring phase that dramatically narrows the search space is followed by torsionally flexible energy optimization on an OPLS-AA nonbonded potential grid for a few hundred surviving candidate poses.  The very best candidates are further refined via a Monte Carlo sampling of pose conformation; in some cases, this is crucial to obtaining an accurate docked pose.  Selection of the best docked pose uses a model energy function that combines empirical and force-field-based terms.  Docking accuracy is assessed by redocking ligands from 282 cocrystd. PDB complexes starting from conformationally optimized ligand geometries that bear no memory of the correctly docked pose.  Errors in geometry for the top-ranked pose are less than 1 Å in nearly half of the cases and are greater than 2 Å in only about one-third of them.  Comparisons to published data on rms deviations show that Glide is nearly twice as accurate as GOLD and more than twice as accurate as FlexX for ligands having up to 20 rotatable bonds.  Glide is also found to be more accurate than the recently described Surflex method.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPuigu31qMMLVg90H21EOLACvtfcHk0ljumelDn8ly9Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFyit74%253D&md5=8cc2f0022318b12dd972e9c493375bf9</span></div><a href="/servlet/linkout?suffix=cit118&amp;dbid=16384&amp;doi=10.1021%2Fjm0306430&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0306430%26sid%3Dliteratum%253Aachs%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DBanks%26aufirst%3DJ.%2BL.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DKlicic%26aufirst%3DJ.%2BJ.%26aulast%3DMainz%26aufirst%3DD.%2BT.%26aulast%3DRepasky%26aufirst%3DM.%2BP.%26aulast%3DKnoll%26aufirst%3DE.%2BH.%26aulast%3DShelley%26aufirst%3DM.%26aulast%3DPerry%26aufirst%3DJ.%2BK.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26aulast%3DFrancis%26aufirst%3DP.%26aulast%3DShenkin%26aufirst%3DP.%2BS.%26atitle%3DGlide%253A%2520A%2520New%2520Approach%2520for%2520Rapid%252C%2520Accurate%2520Docking%2520and%2520Scoring.%25201.%2520Method%2520and%2520Assessment%2520of%2520Docking%2520Accuracy%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1739%26epage%3D1749%26doi%3D10.1021%2Fjm0306430" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref119"><div class="reference"><strong class="refLabel"><a href="#ref119" class="rightTabRefNumLink">119</a></strong><div class="NLM_citation" id="rightTab-cit119"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beard, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollard, W. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banks, J. L.</span></span> <span> </span><span class="NLM_article-title">Glide: A New Approach for Rapid, Accurate Docking and Scoring. 2. Enrichment Factors in Database Screening</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">1750</span>– <span class="NLM_lpage">1759</span>, <span class="refDoi"> DOI: 10.1021/jm030644s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm030644s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref119/cit119&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFyit78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1750-1759&author=T.+A.+Halgrenauthor=R.+B.+Murphyauthor=R.+A.+Friesnerauthor=H.+S.+Beardauthor=L.+L.+Fryeauthor=W.+T.+Pollardauthor=J.+L.+Banks&title=Glide%3A+A+New+Approach+for+Rapid%2C+Accurate+Docking+and+Scoring.+2.+Enrichment+Factors+in+Database+Screening&doi=10.1021%2Fjm030644s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit119R"><div class="casContent"><span class="casTitleNuber">119</span><div class="casTitle"><span class="NLM_cas:atitle">Glide: A new approach for rapid, accurate docking and scoring. 2. Enrichment factors in database screening</span></div><div class="casAuthors">Halgren, Thomas A.; Murphy, Robert B.; Friesner, Richard A.; Beard, Hege S.; Frye, Leah L.; Pollard, W. Thomas; Banks, Jay L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1750-1759</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Glide's ability to identify active compds. in a database screen is characterized by applying Glide to a diverse set of nine protein receptors.  In many cases, two, or even three, protein sites are employed to probe the sensitivity of the results to the site geometry.  To make the database screens as realistic as possible, the screens use sets of "druglike" decoy ligands that have been selected to be representative of what we believe is likely to be found in the compd. collection of a pharmaceutical or biotechnol. company.  Results are presented for releases 1.8, 2.0, and 2.5 of Glide.  The comparisons show that av. measures for both "early" and "global" enrichment for Glide 2.5 are 3 times higher than for Glide 1.8 and more than 2 times higher than for Glide 2.0 because of better results for the least well-handled screens.  This improvement in enrichment stems largely from the better balance of the more widely parametrized GlideScore 2.5 function and the inclusion of terms that penalize ligand-protein interactions that violate established principles of phys. chem., particularly as it concerns the exposure to solvent of charged protein and ligand groups.  Comparisons to results for the thymidine kinase and estrogen receptors published by Rognan and co-workers (J. Med. Chem. 2000, 43, 4759-4767) show that Glide 2.5 performs better than GOLD 1.1, FlexX 1.8, or DOCK 4.01.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoT2ptubbKbvLVg90H21EOLACvtfcHk0ljumelDn8ly9Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFyit78%253D&md5=33d68dd968e65626b449df61e44e37be</span></div><a href="/servlet/linkout?suffix=cit119&amp;dbid=16384&amp;doi=10.1021%2Fjm030644s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm030644s%26sid%3Dliteratum%253Aachs%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DBeard%26aufirst%3DH.%2BS.%26aulast%3DFrye%26aufirst%3DL.%2BL.%26aulast%3DPollard%26aufirst%3DW.%2BT.%26aulast%3DBanks%26aufirst%3DJ.%2BL.%26atitle%3DGlide%253A%2520A%2520New%2520Approach%2520for%2520Rapid%252C%2520Accurate%2520Docking%2520and%2520Scoring.%25202.%2520Enrichment%2520Factors%2520in%2520Database%2520Screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1750%26epage%3D1759%26doi%3D10.1021%2Fjm030644s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref120"><div class="reference"><strong class="refLabel"><a href="#ref120" class="rightTabRefNumLink">120</a></strong><div class="NLM_citation" id="rightTab-cit120"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kollman, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Massova, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reyes, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuhn, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chong, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donini, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cieplak, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Srinivasan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Case, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheatham, T. E.</span></span> <span> </span><span class="NLM_article-title">Calculating Structures and Free Energies of Complex Molecules: Combining Molecular Mechanics and Continuum Models</span>. <i>Acc. Chem. Res.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">889</span>– <span class="NLM_lpage">897</span>, <span class="refDoi"> DOI: 10.1021/ar000033j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ar000033j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref120/cit120&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BD3cXmvFGiu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2000&pages=889-897&author=P.+A.+Kollmanauthor=I.+Massovaauthor=C.+Reyesauthor=B.+Kuhnauthor=S.+Huoauthor=L.+Chongauthor=M.+Leeauthor=T.+Leeauthor=Y.+Duanauthor=W.+Wangauthor=O.+Doniniauthor=P.+Cieplakauthor=J.+Srinivasanauthor=D.+A.+Caseauthor=T.+E.+Cheatham&title=Calculating+Structures+and+Free+Energies+of+Complex+Molecules%3A+Combining+Molecular+Mechanics+and+Continuum+Models&doi=10.1021%2Far000033j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit120R"><div class="casContent"><span class="casTitleNuber">120</span><div class="casTitle"><span class="NLM_cas:atitle">Calculating Structures and Free Energies of Complex Molecules: Combining Molecular Mechanics and Continuum Models</span></div><div class="casAuthors">Kollman, Peter A.; Massova, Irina; Reyes, Carolina; Kuhn, Bernd; Huo, Shuanghong; Chong, Lillian; Lee, Matthew; Lee, Taisung; Duan, Yong; Wang, Wei; Donini, Oreola; Cieplak, Piotr; Srinivasan, Jaysharee; Case, David A.; Cheatham, Thomas E., III</div><div class="citationInfo"><span class="NLM_cas:title">Accounts of Chemical Research</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">889-897</span>CODEN:
                <span class="NLM_cas:coden">ACHRE4</span>;
        ISSN:<span class="NLM_cas:issn">0001-4842</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review, with 63 refs.  A historical perspective on the application of mol. dynamics (MD) to biol. macromols. is presented.  Recent developments combining state-of-the-art force fields with continuum solvation calcns. have allowed us to reach the fourth era of MD applications in which one can often derive both accurate structure and accurate relative free energies from mol. dynamics trajectories.  We illustrate such applications on nucleic acid duplexes, RNA hairpins, protein folding trajectories, and protein-ligand, protein-protein, and protein-nucleic acid interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXGhwlfA3kvLVg90H21EOLACvtfcHk0lhGMFtC8m-rsg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXmvFGiu7g%253D&md5=8436ee610ae145894428db1a1deff73c</span></div><a href="/servlet/linkout?suffix=cit120&amp;dbid=16384&amp;doi=10.1021%2Far000033j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Far000033j%26sid%3Dliteratum%253Aachs%26aulast%3DKollman%26aufirst%3DP.%2BA.%26aulast%3DMassova%26aufirst%3DI.%26aulast%3DReyes%26aufirst%3DC.%26aulast%3DKuhn%26aufirst%3DB.%26aulast%3DHuo%26aufirst%3DS.%26aulast%3DChong%26aufirst%3DL.%26aulast%3DLee%26aufirst%3DM.%26aulast%3DLee%26aufirst%3DT.%26aulast%3DDuan%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DDonini%26aufirst%3DO.%26aulast%3DCieplak%26aufirst%3DP.%26aulast%3DSrinivasan%26aufirst%3DJ.%26aulast%3DCase%26aufirst%3DD.%2BA.%26aulast%3DCheatham%26aufirst%3DT.%2BE.%26atitle%3DCalculating%2520Structures%2520and%2520Free%2520Energies%2520of%2520Complex%2520Molecules%253A%2520Combining%2520Molecular%2520Mechanics%2520and%2520Continuum%2520Models%26jtitle%3DAcc.%2520Chem.%2520Res.%26date%3D2000%26volume%3D33%26spage%3D889%26epage%3D897%26doi%3D10.1021%2Far000033j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref121"><div class="reference"><strong class="refLabel"><a href="#ref121" class="rightTabRefNumLink">121</a></strong><div class="NLM_citation" id="rightTab-cit121"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Srinivasan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kollman, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Case, D. A.</span></span> <span> </span><span class="NLM_article-title">Continuum Solvent Studies of the Stability of RNA Hairpin Loops and Helices</span>. <i>J. Biomol. Struct. Dyn.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">671</span>– <span class="NLM_lpage">682</span>, <span class="refDoi"> DOI: 10.1080/07391102.1998.10508279</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref121/cit121&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1080%2F07391102.1998.10508279" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref121/cit121&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10052623" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref121/cit121&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADyaK1MXhtlams74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=1998&pages=671-682&author=J.+Srinivasanauthor=J.+Millerauthor=P.+A.+Kollmanauthor=D.+A.+Case&title=Continuum+Solvent+Studies+of+the+Stability+of+RNA+Hairpin+Loops+and+Helices&doi=10.1080%2F07391102.1998.10508279"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit121R"><div class="casContent"><span class="casTitleNuber">121</span><div class="casTitle"><span class="NLM_cas:atitle">Continuum solvent studies of the stability of RNA hairpin loops and helices</span></div><div class="casAuthors">Srinivasan, Jayashree; Miller, Jennifer; Kollman, Peter A.; Case, David A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Structure & Dynamics</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">671-682</span>CODEN:
                <span class="NLM_cas:coden">JBSDD6</span>;
        ISSN:<span class="NLM_cas:issn">0739-1102</span>.
    
            (<span class="NLM_cas:orgname">Adenine Press</span>)
        </div><div class="casAbstract">We apply continuum solvent models to investigate the relative stability of various conformational forms for two RNA sequences, GGAC(UUCG)GUCC and GGUG(UGAA)CACC.  In the first part, we compare alternate hairpin conformations to explore the reliability of these models to discriminate between different local conformations.  A second part looks at the hairpin-duplex conversion for the UUCG sequence, identifying major contributors to the thermodn. of a large scale transition.  Structures were taken as snapshots from multi-nanosecond mol. dynamics simulations computed in a consistent fashion using explicit solvent and with long-range electrostatics accounted for using the Particle-Mesh Ewald procedure.  The electrostatic contribution to solvation energies were computed using both a finite-difference Poisson-Boltzmann (PB) model and a pairwise Generalized Born model; non-electrostatic contributions were estd. with a surface-area dependent term.  To these solvation free energies were added the mean solute internal energies (detd. from a mol. mechanics potential) and ests. of the solute entropy (from a harmonic anal.).  Consistent with expt. and with earlier solvated mol. dynamics simulations, the UUCG hairpin was found to prefer conformers close to a recent NMR structure detn. in preference to those from an earlier NMR study.  Similarly, results for the UGAA hairpin favored an NMR-derived structure over that to be expected for a generic GNRA hairpin loop.  Exptl. free energies are not known for the hairpin/duplex conversion, but must be close to zero since hairpins are seen in soln. and duplexes in crystals; our calcns. find a value near zero and illustrate the expected interplay of solvation, salt effects and entropy in affecting this equil.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2ygn1MJoSNLVg90H21EOLACvtfcHk0lhGMFtC8m-rsg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXhtlams74%253D&md5=7843cabcbe7d2c75f4bc9455f2115f95</span></div><a href="/servlet/linkout?suffix=cit121&amp;dbid=16384&amp;doi=10.1080%2F07391102.1998.10508279&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F07391102.1998.10508279%26sid%3Dliteratum%253Aachs%26aulast%3DSrinivasan%26aufirst%3DJ.%26aulast%3DMiller%26aufirst%3DJ.%26aulast%3DKollman%26aufirst%3DP.%2BA.%26aulast%3DCase%26aufirst%3DD.%2BA.%26atitle%3DContinuum%2520Solvent%2520Studies%2520of%2520the%2520Stability%2520of%2520RNA%2520Hairpin%2520Loops%2520and%2520Helices%26jtitle%3DJ.%2520Biomol.%2520Struct.%2520Dyn.%26date%3D1998%26volume%3D16%26spage%3D671%26epage%3D682%26doi%3D10.1080%2F07391102.1998.10508279" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref122"><div class="reference"><strong class="refLabel"><a href="#ref122" class="rightTabRefNumLink">122</a></strong><div class="NLM_citation" id="rightTab-cit122"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Paulsen, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sindhikara, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Appleby, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villaseñor, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmitz, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shivakumar, D.</span></span> <span> </span><span class="NLM_article-title">Evaluation of Free Energy Calculations for the Prioritization of Macrocycle Synthesis</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">3489</span>– <span class="NLM_lpage">3498</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.0c00132</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.0c00132" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref122/cit122&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtFGmt73N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2020&pages=3489-3498&author=J.+L.+Paulsenauthor=H.+S.+Yuauthor=D.+Sindhikaraauthor=L.+Wangauthor=T.+C.+Applebyauthor=A.+G.+Villase%C3%B1orauthor=U.+Schmitzauthor=D.+Shivakumar&title=Evaluation+of+Free+Energy+Calculations+for+the+Prioritization+of+Macrocycle+Synthesis&doi=10.1021%2Facs.jcim.0c00132"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit122R"><div class="casContent"><span class="casTitleNuber">122</span><div class="casTitle"><span class="NLM_cas:atitle">Evaluation of Free Energy Calculations for the Prioritization of Macrocycle Synthesis</span></div><div class="casAuthors">Paulsen, Janet L.; Yu, Haoyu S.; Sindhikara, Dan; Wang, Lingle; Appleby, Todd; Villasenor, Armando G.; Schmitz, Uli; Shivakumar, Devleena</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3489-3498</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A tremendous research and development effort was exerted toward combating chronic hepatitis C, ultimately leading to curative oral treatments, all of which are targeting viral proteins.  Despite the advantage of numerous targets allowing for broad hepatitis C virus (HCV) genotype coverage, the only host target inhibitors that advanced into clin. development were Cyclosporin A based cyclophilin inhibitors.  While cyclosporin-based mols. typically require a fermn. process, Gilead successfully pursued a fully synthetic, oral program based on Sanglifehrin A.  The drug discovery process, though greatly helped by facile crystallog., was still hampered by the limitations in the accuracy of predictive computational methods for prioritizing compd. ideas.  Recent advances in accuracy and speed of free energy perturbation (FEP) methods, however, are attractive for prioritizing and derisking synthetically challenging mols. and potentially could have had a significant impact on the speed of the development of this program.  Here in our simulated prospective study, the binding free energies of 26 macrocyclic cyclophilin inhibitors were blindly predicted using FEP+ to test this hypothesis.  The predictions had a low mean unsigned error (MUE) (1.1 kcal/mol) and accurately reproduced many design decisions from the program, suggesting that FEP+ has the potential to drive synthetic chem. efforts by more accurately ranking compds. with nonintuitive structure-activity relationships (SARs).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGogQ6nraJ7CZ7Vg90H21EOLACvtfcHk0lhGMFtC8m-rsg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtFGmt73N&md5=4926b12c085eb0f5181c1a265fac7571</span></div><a href="/servlet/linkout?suffix=cit122&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.0c00132&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.0c00132%26sid%3Dliteratum%253Aachs%26aulast%3DPaulsen%26aufirst%3DJ.%2BL.%26aulast%3DYu%26aufirst%3DH.%2BS.%26aulast%3DSindhikara%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DAppleby%26aufirst%3DT.%2BC.%26aulast%3DVillase%25C3%25B1or%26aufirst%3DA.%2BG.%26aulast%3DSchmitz%26aufirst%3DU.%26aulast%3DShivakumar%26aufirst%3DD.%26atitle%3DEvaluation%2520of%2520Free%2520Energy%2520Calculations%2520for%2520the%2520Prioritization%2520of%2520Macrocycle%2520Synthesis%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2020%26volume%3D60%26spage%3D3489%26epage%3D3498%26doi%3D10.1021%2Facs.jcim.0c00132" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref123"><div class="reference"><strong class="refLabel"><a href="#ref123" class="rightTabRefNumLink">123</a></strong><div class="NLM_citation" id="rightTab-cit123"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cappel, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jerome, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hessler, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matter, H.</span></span> <span> </span><span class="NLM_article-title">Impact of Different Automated Binding Pose Generation Approaches on Relative Binding Free Energy Simulations</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">1432</span>– <span class="NLM_lpage">1444</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.9b01118</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.9b01118" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref123/cit123&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhs12ktb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2020&pages=1432-1444&author=D.+Cappelauthor=S.+Jeromeauthor=G.+Hesslerauthor=H.+Matter&title=Impact+of+Different+Automated+Binding+Pose+Generation+Approaches+on+Relative+Binding+Free+Energy+Simulations&doi=10.1021%2Facs.jcim.9b01118"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit123R"><div class="casContent"><span class="casTitleNuber">123</span><div class="casTitle"><span class="NLM_cas:atitle">Impact of Different Automated Binding Pose Generation Approaches on Relative Binding Free Energy Simulations</span></div><div class="casAuthors">Cappel, Daniel; Jerome, Steven; Hessler, Gerhard; Matter, Hans</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1432-1444</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Relative binding free energy (RBFE) prediction methods such as free energy perturbation (FEP) are important today for estg. protein-ligand binding affinities.  Significant hardware and algorithmic improvements now allow for simulating congeneric series within days.  Therefore, RBFE calcns. have an enormous potential for structure-based drug discovery.  As typically only a few representative crystal structures for a series are available, other ligands and design proposals must be reliably superimposed for meaningful results.  An obsd. significant effect of the alignment on FEP led us to develop an alignment approach combining docking with max. common substructure (MCS) derived core constraints from the most similar ref. pose, named MCS-docking workflow.  We then studied the effect of binding pose generation on the accuracy of RBFE predictions using six ligand series from five pharmaceutically relevant protein targets.  Overall, the MCS-docking workflow generated consistent poses for most of the ligands in the investigated series.  While multiple alignment methods often resulted in comparable FEP predictions, for most of the cases herein, the MCS-docking workflow produced the best accuracy in predictions.  Furthermore, the FEP anal. data strongly support the hypothesis that the accuracy of RBFE predictions depends on input poses to construct the perturbation map.  Therefore, an automated workflow without manual intervention minimizes potential errors and obtains the most useful predictions with significant impact for structure-based design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGob6kqQDqVs-7Vg90H21EOLACvtfcHk0lhGMFtC8m-rsg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhs12ktb4%253D&md5=c6c5e60560061e35da8e661f17c10a8e</span></div><a href="/servlet/linkout?suffix=cit123&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.9b01118&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.9b01118%26sid%3Dliteratum%253Aachs%26aulast%3DCappel%26aufirst%3DD.%26aulast%3DJerome%26aufirst%3DS.%26aulast%3DHessler%26aufirst%3DG.%26aulast%3DMatter%26aufirst%3DH.%26atitle%3DImpact%2520of%2520Different%2520Automated%2520Binding%2520Pose%2520Generation%2520Approaches%2520on%2520Relative%2520Binding%2520Free%2520Energy%2520Simulations%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2020%26volume%3D60%26spage%3D1432%26epage%3D1444%26doi%3D10.1021%2Facs.jcim.9b01118" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref124"><div class="reference"><strong class="refLabel"><a href="#ref124" class="rightTabRefNumLink">124</a></strong><div class="NLM_citation" id="rightTab-cit124"><span>Daylight
Theory: SMARTS—A
Language for Describing Molecular Patterns. <a href="https://www.daylight.com/dayhtml/doc/theory/theory.smarts.html" class="extLink">https://www.daylight.com/dayhtml/doc/theory/theory.smarts.html</a> (accessed April 6, <span class="NLM_year">2020</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Daylight%0ATheory%3A+SMARTS%E2%80%94A%0ALanguage+for+Describing+Molecular+Patterns.+https%3A%2F%2Fwww.daylight.com%2Fdayhtml%2Fdoc%2Ftheory%2Ftheory.smarts.html+%28accessed+April+6%2C+2020%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit124&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs%26date%3D2020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref125"><div class="reference"><strong class="refLabel"><a href="#ref125" class="rightTabRefNumLink">125</a></strong><div class="NLM_citation" id="rightTab-cit125"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Evers, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haack, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lorenz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bossart, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elvert, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henkel, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stengelin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glien, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dudda, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lorenz, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kadereit, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, M.</span></span> <span> </span><span class="NLM_article-title">Design of Novel Exendin-Based Dual Glucagon-like Peptide 1 (GLP-1)/Glucagon Receptor Agonists</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">4293</span>– <span class="NLM_lpage">4303</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00174</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00174" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref125/cit125&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmslKrsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=4293-4303&author=A.+Eversauthor=T.+Haackauthor=M.+Lorenzauthor=M.+Bossartauthor=R.+Elvertauthor=B.+Henkelauthor=S.+Stengelinauthor=M.+Kurzauthor=M.+Glienauthor=A.+Duddaauthor=K.+Lorenzauthor=D.+Kadereitauthor=M.+Wagner&title=Design+of+Novel+Exendin-Based+Dual+Glucagon-like+Peptide+1+%28GLP-1%29%2FGlucagon+Receptor+Agonists&doi=10.1021%2Facs.jmedchem.7b00174"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit125R"><div class="casContent"><span class="casTitleNuber">125</span><div class="casTitle"><span class="NLM_cas:atitle">Design of Novel Exendin-Based Dual Glucagon-like Peptide 1 (GLP-1)/Glucagon Receptor Agonists</span></div><div class="casAuthors">Evers, Andreas; Haack, Torsten; Lorenz, Martin; Bossart, Martin; Elvert, Ralf; Henkel, Bernd; Stengelin, Siegfried; Kurz, Michael; Glien, Maike; Dudda, Angela; Lorenz, Katrin; Kadereit, Dieter; Wagner, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4293-4303</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Dual activation of the glucagon-like peptide 1 (GLP-1) and glucagon receptor has the potential to lead to a novel therapy principle for the treatment of diabesity.  Here, we report a series of novel peptides with dual activity on these receptors that were discovered by rational design.  On the basis of sequence anal. and structure-based design, structural elements of glucagon were engineered into the selective GLP-1 receptor agonist exendin-4, resulting in hybrid peptides with potent dual GLP-1/glucagon receptor activity.  Detailed structure-activity relationship data are shown.  Further modifications with unnatural and modified amino acids resulted in novel metabolically stable peptides that demonstrated a significant dose-dependent decrease in blood glucose in chronic studies in diabetic db/db mice and reduced body wt. in diet-induced obese (DIO) mice.  Structural anal. by NMR spectroscopy confirmed that the peptides maintain an exendin-4-like structure with its characteristic tryptophan-cage fold motif that is responsible for favorable chem. and phys. stability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrewDgurwTT4LVg90H21EOLACvtfcHk0ljn5SF4CfEJWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmslKrsr8%253D&md5=b59a7a25cdd56516b78c988860b39211</span></div><a href="/servlet/linkout?suffix=cit125&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00174&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00174%26sid%3Dliteratum%253Aachs%26aulast%3DEvers%26aufirst%3DA.%26aulast%3DHaack%26aufirst%3DT.%26aulast%3DLorenz%26aufirst%3DM.%26aulast%3DBossart%26aufirst%3DM.%26aulast%3DElvert%26aufirst%3DR.%26aulast%3DHenkel%26aufirst%3DB.%26aulast%3DStengelin%26aufirst%3DS.%26aulast%3DKurz%26aufirst%3DM.%26aulast%3DGlien%26aufirst%3DM.%26aulast%3DDudda%26aufirst%3DA.%26aulast%3DLorenz%26aufirst%3DK.%26aulast%3DKadereit%26aufirst%3DD.%26aulast%3DWagner%26aufirst%3DM.%26atitle%3DDesign%2520of%2520Novel%2520Exendin-Based%2520Dual%2520Glucagon-like%2520Peptide%25201%2520%2528GLP-1%2529%252FGlucagon%2520Receptor%2520Agonists%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D4293%26epage%3D4303%26doi%3D10.1021%2Facs.jmedchem.7b00174" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref126"><div class="reference"><strong class="refLabel"><a href="#ref126" class="rightTabRefNumLink">126</a></strong><div class="NLM_citation" id="rightTab-cit126"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Evers, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bossart, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pfeiffer-Marek, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elvert, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schreuder, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stengelin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lorenz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herling, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konkar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lukasczyk, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pfenninger, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lorenz, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haack, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kadereit, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, M.</span></span> <span> </span><span class="NLM_article-title">Dual Glucagon-like Peptide 1 (GLP-1)/Glucagon Receptor Agonists Specifically Optimized for Multidose Formulations</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">5580</span>– <span class="NLM_lpage">5593</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00292</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00292" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref126/cit126&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtF2isLrL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=5580-5593&author=A.+Eversauthor=M.+Bossartauthor=S.+Pfeiffer-Marekauthor=R.+Elvertauthor=H.+Schreuderauthor=M.+Kurzauthor=S.+Stengelinauthor=M.+Lorenzauthor=A.+Herlingauthor=A.+Konkarauthor=U.+Lukasczykauthor=A.+Pfenningerauthor=K.+Lorenzauthor=T.+Haackauthor=D.+Kadereitauthor=M.+Wagner&title=Dual+Glucagon-like+Peptide+1+%28GLP-1%29%2FGlucagon+Receptor+Agonists+Specifically+Optimized+for+Multidose+Formulations&doi=10.1021%2Facs.jmedchem.8b00292"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit126R"><div class="casContent"><span class="casTitleNuber">126</span><div class="casTitle"><span class="NLM_cas:atitle">Dual Glucagon-like Peptide 1 (GLP-1)/Glucagon Receptor Agonists Specifically Optimized for Multidose Formulations</span></div><div class="casAuthors">Evers, Andreas; Bossart, Martin; Pfeiffer-Marek, Stefania; Elvert, Ralf; Schreuder, Herman; Kurz, Michael; Stengelin, Siegfried; Lorenz, Martin; Herling, Andreas; Konkar, Anish; Lukasczyk, Ulrike; Pfenninger, Anja; Lorenz, Katrin; Haack, Torsten; Kadereit, Dieter; Wagner, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">5580-5593</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Novel peptidic dual agonists of the glucagon-like peptide 1 (GLP-1) and glucagon receptor are reported to have enhanced efficacy over pure GLP-1 receptor agonists with regard to treatment of obesity and diabetes.  We describe novel exendin-4 based dual agonists designed with an activity ratio favoring the GLP-1 vs. the glucagon receptor.  As result of an iterative optimization procedure that included mol. modeling, structural biol. studies (X-ray, NMR), peptide design and synthesis, exptl. activity, and soly. profiling, a candidate mol. was identified.  Novel SAR points are reported that allowed us to fine-tune the desired receptor activity ratio and increased soly. in the presence of antimicrobial preservatives, findings that can be of general applicability for any peptide discovery project.  The peptide was evaluated in chronic in vivo studies in obese diabetic monkeys as translational model for the human situation and demonstrated favorable blood glucose and body wt. lowering effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqj6Q6G9iY2lrVg90H21EOLACvtfcHk0ljn5SF4CfEJWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtF2isLrL&md5=cba1860c3a7b83427ee1cc0336b0e52c</span></div><a href="/servlet/linkout?suffix=cit126&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00292&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00292%26sid%3Dliteratum%253Aachs%26aulast%3DEvers%26aufirst%3DA.%26aulast%3DBossart%26aufirst%3DM.%26aulast%3DPfeiffer-Marek%26aufirst%3DS.%26aulast%3DElvert%26aufirst%3DR.%26aulast%3DSchreuder%26aufirst%3DH.%26aulast%3DKurz%26aufirst%3DM.%26aulast%3DStengelin%26aufirst%3DS.%26aulast%3DLorenz%26aufirst%3DM.%26aulast%3DHerling%26aufirst%3DA.%26aulast%3DKonkar%26aufirst%3DA.%26aulast%3DLukasczyk%26aufirst%3DU.%26aulast%3DPfenninger%26aufirst%3DA.%26aulast%3DLorenz%26aufirst%3DK.%26aulast%3DHaack%26aufirst%3DT.%26aulast%3DKadereit%26aufirst%3DD.%26aulast%3DWagner%26aufirst%3DM.%26atitle%3DDual%2520Glucagon-like%2520Peptide%25201%2520%2528GLP-1%2529%252FGlucagon%2520Receptor%2520Agonists%2520Specifically%2520Optimized%2520for%2520Multidose%2520Formulations%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D5580%26epage%3D5593%26doi%3D10.1021%2Facs.jmedchem.8b00292" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref127"><div class="reference"><strong class="refLabel"><a href="#ref127" class="rightTabRefNumLink">127</a></strong><div class="NLM_citation" id="rightTab-cit127"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tarrasch, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun Kobilka, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobilka, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skiniotis, G.</span></span> <span> </span><span class="NLM_article-title">Cryo-EM Structure of the Activated GLP-1 Receptor in Complex with a G Protein</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>546</i></span>,  <span class="NLM_fpage">248</span>– <span class="NLM_lpage">253</span>, <span class="refDoi"> DOI: 10.1038/nature22394</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref127/cit127&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1038%2Fnature22394" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref127/cit127&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=28538729" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref127/cit127&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC2sXosVagtLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=546&publication_year=2017&pages=248-253&author=Y.+Zhangauthor=B.+Sunauthor=D.+Fengauthor=H.+Huauthor=M.+Chuauthor=Q.+Quauthor=J.+T.+Tarraschauthor=S.+Liauthor=T.+Sun+Kobilkaauthor=B.+K.+Kobilkaauthor=G.+Skiniotis&title=Cryo-EM+Structure+of+the+Activated+GLP-1+Receptor+in+Complex+with+a+G+Protein&doi=10.1038%2Fnature22394"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit127R"><div class="casContent"><span class="casTitleNuber">127</span><div class="casTitle"><span class="NLM_cas:atitle">Cryo-EM structure of the activated GLP-1 receptor in complex with a G protein</span></div><div class="casAuthors">Zhang, Yan; Sun, Bingfa; Feng, Dan; Hu, Hongli; Chu, Matthew; Qu, Qianhui; Tarrasch, Jeffrey T.; Li, Shane; Sun Kobilka, Tong; Kobilka, Brian K.; Skiniotis, Georgios</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">546</span>
        (<span class="NLM_cas:issue">7657</span>),
    <span class="NLM_cas:pages">248-253</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Glucagon-like peptide 1 (GLP-1) is a hormone with essential roles in regulating insulin secretion, carbohydrate metab. and appetite.  GLP-1 effects are mediated through binding to the GLP-1 receptor (GLP-1R), a class B G-protein-coupled receptor (GPCR) that signals primarily through the stimulatory G protein Gs.  Class B GPCRs are important therapeutic targets; however, our understanding of their mechanism of action is limited by the lack of structural information on activated and full-length receptors.  Here we report the cryo-electron microscopy structure of the peptide-activated GLP-1R-Gs complex at near at. resoln.  The peptide is clasped between the N-terminal domain and the transmembrane core of the receptor, and further stabilized by extracellular loops.  Conformational changes in the transmembrane domain result in a sharp kink in the middle of transmembrane helix 6, which pivots its intracellular half outward to accommodate the α5-helix of the Ras-like domain of Gs.  These results provide a structural framework for understanding class B GPCR activation through hormone binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmDRFGRwMs1bVg90H21EOLACvtfcHk0ljFxz8GOzdQTQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXosVagtLo%253D&md5=5e7ebd94012ba47ecc299ad0502975de</span></div><a href="/servlet/linkout?suffix=cit127&amp;dbid=16384&amp;doi=10.1038%2Fnature22394&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature22394%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DSun%26aufirst%3DB.%26aulast%3DFeng%26aufirst%3DD.%26aulast%3DHu%26aufirst%3DH.%26aulast%3DChu%26aufirst%3DM.%26aulast%3DQu%26aufirst%3DQ.%26aulast%3DTarrasch%26aufirst%3DJ.%2BT.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DSun%2BKobilka%26aufirst%3DT.%26aulast%3DKobilka%26aufirst%3DB.%2BK.%26aulast%3DSkiniotis%26aufirst%3DG.%26atitle%3DCryo-EM%2520Structure%2520of%2520the%2520Activated%2520GLP-1%2520Receptor%2520in%2520Complex%2520with%2520a%2520G%2520Protein%26jtitle%3DNature%26date%3D2017%26volume%3D546%26spage%3D248%26epage%3D253%26doi%3D10.1038%2Fnature22394" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref128"><div class="reference"><strong class="refLabel"><a href="#ref128" class="rightTabRefNumLink">128</a></strong><div class="NLM_citation" id="rightTab-cit128"><span> <i>Prime Schrödinger
Release 2019-3 Prime</i>; <span class="NLM_publisher-name">Schrödinger,
LLC</span>: <span class="NLM_publisher-loc">New York,
NY</span>, <span class="NLM_year">2019</span> (accessed Jan 31, 2018).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Prime+Schr%C3%B6dinger%0ARelease+2019-3+Prime%3B+Schr%C3%B6dinger%2C%0ALLC%3A+New+York%2C%0ANY%2C+2019+%28accessed+Jan+31%2C+2018%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit128&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DPrime%2520Schr%25C3%25B6dinger%250ARelease%25202019-3%2520Prime%26pub%3DSchr%25C3%25B6dinger%252C%250ALLC%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref129"><div class="reference"><strong class="refLabel"><a href="#ref129" class="rightTabRefNumLink">129</a></strong><div class="NLM_citation" id="rightTab-cit129"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Day, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ottaway, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patterson, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gelfanov, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smiley, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gidda, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Findeisen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bruemmer, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drucker, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaudhary, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holland, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hembree, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abplanalp, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grant, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruehl, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirchner, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lockie, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hofmann, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woods, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nogueiras, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pfluger, P. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perez-Tilve, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DiMarchi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tschöp, M. H.</span></span> <span> </span><span class="NLM_article-title">A New Glucagon and GLP-1 Co-Agonist Eliminates Obesity in Rodents</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">749</span>– <span class="NLM_lpage">757</span>, <span class="refDoi"> DOI: 10.1038/nchembio.209</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref129/cit129&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1038%2Fnchembio.209" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref129/cit129&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=19597507" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref129/cit129&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BD1MXosFWhsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2009&pages=749-757&author=J.+W.+Dayauthor=N.+Ottawayauthor=J.+T.+Pattersonauthor=V.+Gelfanovauthor=D.+Smileyauthor=J.+Giddaauthor=H.+Findeisenauthor=D.+Bruemmerauthor=D.+J.+Druckerauthor=N.+Chaudharyauthor=J.+Hollandauthor=J.+Hembreeauthor=W.+Abplanalpauthor=E.+Grantauthor=J.+Ruehlauthor=H.+Wilsonauthor=H.+Kirchnerauthor=S.+H.+Lockieauthor=S.+Hofmannauthor=S.+C.+Woodsauthor=R.+Nogueirasauthor=P.+T.+Pflugerauthor=D.+Perez-Tilveauthor=R.+DiMarchiauthor=M.+H.+Tsch%C3%B6p&title=A+New+Glucagon+and+GLP-1+Co-Agonist+Eliminates+Obesity+in+Rodents&doi=10.1038%2Fnchembio.209"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit129R"><div class="casContent"><span class="casTitleNuber">129</span><div class="casTitle"><span class="NLM_cas:atitle">A new glucagon and GLP-1 co-agonist eliminates obesity in rodents</span></div><div class="casAuthors">Day, Jonathan W.; Ottaway, Nickki; Patterson, James T.; Gelfanov, Vasily; Smiley, David; Gidda, Jas; Findeisen, Hannes; Bruemmer, Dennis; Drucker, Daniel J.; Chaudhary, Nilika; Holland, Jenna; Hembree, Jazzminn; Abplanalp, William; Grant, Erin; Ruehl, Jennifer; Wilson, Hilary; Kirchner, Henriette; Lockie, Sarah Haas; Hofmann, Susanna; Woods, Stephen C.; Nogueiras, Ruben; Pfluger, Paul T.; Perez-Tilve, Diego; DiMarchi, Richard; Tschoep, Matthias H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">749-757</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">We report the efficacy of a new peptide with agonism at the glucagon and GLP-1 receptors that has potent, sustained satiation-inducing and lipolytic effects.  Selective chem. modification to glucagon resulted in a loss of specificity, with minimal change to inherent activity.  The structural basis for the co-agonism appears to be a combination of local positional interactions and a change in secondary structure.  Two co-agonist peptides differing from each other only in their level of glucagon receptor agonism were studied in rodent obesity models.  Administration of PEGylated peptides once per wk normalized adiposity and glucose tolerance in diet-induced obese mice.  Redn. of body wt. was achieved by a loss of body fat resulting from decreased food intake and increased energy expenditure.  These preclin. studies indicate that when full GLP-1 agonism is augmented with an appropriate degree of glucagon receptor activation, body fat redn. can be substantially enhanced without any overt adverse effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp4K1-_eVTTKbVg90H21EOLACvtfcHk0ljFxz8GOzdQTQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXosFWhsLk%253D&md5=74b57672bfd375dd4d073624a25eab9b</span></div><a href="/servlet/linkout?suffix=cit129&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.209&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.209%26sid%3Dliteratum%253Aachs%26aulast%3DDay%26aufirst%3DJ.%2BW.%26aulast%3DOttaway%26aufirst%3DN.%26aulast%3DPatterson%26aufirst%3DJ.%2BT.%26aulast%3DGelfanov%26aufirst%3DV.%26aulast%3DSmiley%26aufirst%3DD.%26aulast%3DGidda%26aufirst%3DJ.%26aulast%3DFindeisen%26aufirst%3DH.%26aulast%3DBruemmer%26aufirst%3DD.%26aulast%3DDrucker%26aufirst%3DD.%2BJ.%26aulast%3DChaudhary%26aufirst%3DN.%26aulast%3DHolland%26aufirst%3DJ.%26aulast%3DHembree%26aufirst%3DJ.%26aulast%3DAbplanalp%26aufirst%3DW.%26aulast%3DGrant%26aufirst%3DE.%26aulast%3DRuehl%26aufirst%3DJ.%26aulast%3DWilson%26aufirst%3DH.%26aulast%3DKirchner%26aufirst%3DH.%26aulast%3DLockie%26aufirst%3DS.%2BH.%26aulast%3DHofmann%26aufirst%3DS.%26aulast%3DWoods%26aufirst%3DS.%2BC.%26aulast%3DNogueiras%26aufirst%3DR.%26aulast%3DPfluger%26aufirst%3DP.%2BT.%26aulast%3DPerez-Tilve%26aufirst%3DD.%26aulast%3DDiMarchi%26aufirst%3DR.%26aulast%3DTsch%25C3%25B6p%26aufirst%3DM.%2BH.%26atitle%3DA%2520New%2520Glucagon%2520and%2520GLP-1%2520Co-Agonist%2520Eliminates%2520Obesity%2520in%2520Rodents%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2009%26volume%3D5%26spage%3D749%26epage%3D757%26doi%3D10.1038%2Fnchembio.209" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref130"><div class="reference"><strong class="refLabel"><a href="#ref130" class="rightTabRefNumLink">130</a></strong><div class="NLM_citation" id="rightTab-cit130"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Miranda, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winters, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gegg, C. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aral, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diamond, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guido, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stanislaus, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rose, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poppe, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Véniant, M. M.</span></span> <span> </span><span class="NLM_article-title">Design and Synthesis of Conformationally Constrained Glucagon-like Peptide-1 Derivatives with Increased Plasma Stability and Prolonged in Vivo Activity</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">2758</span>– <span class="NLM_lpage">2765</span>, <span class="refDoi"> DOI: 10.1021/jm701522b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm701522b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref130/cit130&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BD1cXks1Kmu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=2758-2765&author=L.+P.+Mirandaauthor=K.+A.+Wintersauthor=C.+V.+Geggauthor=A.+Patelauthor=J.+Aralauthor=J.+Longauthor=J.+Zhangauthor=S.+Diamondauthor=M.+Guidoauthor=S.+Stanislausauthor=M.+Maauthor=H.+Liauthor=M.+J.+Roseauthor=L.+Poppeauthor=M.+M.+V%C3%A9niant&title=Design+and+Synthesis+of+Conformationally+Constrained+Glucagon-like+Peptide-1+Derivatives+with+Increased+Plasma+Stability+and+Prolonged+in+Vivo+Activity&doi=10.1021%2Fjm701522b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit130R"><div class="casContent"><span class="casTitleNuber">130</span><div class="casTitle"><span class="NLM_cas:atitle">Design and Synthesis of Conformationally Constrained Glucagon-Like Peptide-1 Derivatives with Increased Plasma Stability and Prolonged in Vivo Activity</span></div><div class="casAuthors">Miranda, Les P.; Winters, Katherine A.; Gegg, Colin V.; Patel, Ankita; Aral, Jennifer; Long, Jason; Zhang, Jingwen; Diamond, Stephanie; Guido, Mark; Stanislaus, Shanaka; Ma, Mark; Li, Hongyan; Rose, Mark J.; Poppe, Leszek; Veniant, Murielle M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2758-2765</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A series of conformationally constrained derivs. of glucagon-like peptide-1 (GLP-1) were designed and evaluated.  By use of [Gly8]GLP-1(7-37)-NH2 (2) peptide as a starting point, 17 cyclic derivs. possessing i to i + 4, i to i + 5, or i to i + 7 side chain to side chain lactam bridges from positions 18 to 30 were prepd.  The effect of a helix-promoting α-amino-isobutyric acid (Aib) substitution at position 22 was also evaluated.  The introduction of i to i + 4 glutamic acid-lysine lactam constraints in c[Glu18-Lys22][Gly8]GLP-1(7-37)-NH2 (6), c[Glu22-Lys26][Gly8]GLP-1(7-37)-NH2 (10), and c[Glu23-Lys27][Gly8]GLP-1(7-37)-NH2 (11) resulted in potent functional activity and receptor affinities comparable to native GLP-1.  Selected GLP-1 peptides were chemoselectively PEGylated in order to prolong their in vivo activity.  PEGylated peptides [Gly8,Aib22]GLP-1(7-37)-Cys(PEG)-Ala-NH2 (23) and c[Glu22-Lys26][Gly8]GLP-1(7-37)-Cys(PEG)-Ser-Gly-NH2 (24) retained picomolar functional potency and avid receptor binding properties.  Importantly, PEGylated GLP-1 peptide 23 exhibited sustained in vivo efficacy with respect to blood glucose redn. and decreased body wt. for several days in nonhuman primates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGozaW6frxUWw7Vg90H21EOLACvtfcHk0ljFxz8GOzdQTQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXks1Kmu7o%253D&md5=8526495a8a21e973a602a3cf55089d97</span></div><a href="/servlet/linkout?suffix=cit130&amp;dbid=16384&amp;doi=10.1021%2Fjm701522b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm701522b%26sid%3Dliteratum%253Aachs%26aulast%3DMiranda%26aufirst%3DL.%2BP.%26aulast%3DWinters%26aufirst%3DK.%2BA.%26aulast%3DGegg%26aufirst%3DC.%2BV.%26aulast%3DPatel%26aufirst%3DA.%26aulast%3DAral%26aufirst%3DJ.%26aulast%3DLong%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DDiamond%26aufirst%3DS.%26aulast%3DGuido%26aufirst%3DM.%26aulast%3DStanislaus%26aufirst%3DS.%26aulast%3DMa%26aufirst%3DM.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DRose%26aufirst%3DM.%2BJ.%26aulast%3DPoppe%26aufirst%3DL.%26aulast%3DV%25C3%25A9niant%26aufirst%3DM.%2BM.%26atitle%3DDesign%2520and%2520Synthesis%2520of%2520Conformationally%2520Constrained%2520Glucagon-like%2520Peptide-1%2520Derivatives%2520with%2520Increased%2520Plasma%2520Stability%2520and%2520Prolonged%2520in%2520Vivo%2520Activity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D2758%26epage%3D2765%26doi%3D10.1021%2Fjm701522b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref131"><div class="reference"><strong class="refLabel"><a href="#ref131" class="rightTabRefNumLink">131</a></strong><div class="NLM_citation" id="rightTab-cit131"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murage, E. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bisello, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahn, J.-M.</span></span> <span> </span><span class="NLM_article-title">Development of Potent Glucagon-like Peptide-1 Agonists with High Enzyme Stability via Introduction of Multiple Lactam Bridges</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">6412</span>– <span class="NLM_lpage">6420</span>, <span class="refDoi"> DOI: 10.1021/jm100602m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm100602m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref131/cit131&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC3cXpvVarsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=6412-6420&author=E.+N.+Murageauthor=G.+Gaoauthor=A.+Biselloauthor=J.-M.+Ahn&title=Development+of+Potent+Glucagon-like+Peptide-1+Agonists+with+High+Enzyme+Stability+via+Introduction+of+Multiple+Lactam+Bridges&doi=10.1021%2Fjm100602m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit131R"><div class="casContent"><span class="casTitleNuber">131</span><div class="casTitle"><span class="NLM_cas:atitle">Development of Potent Glucagon-like Peptide-1 Agonists with High Enzyme Stability via Introduction of Multiple Lactam Bridges</span></div><div class="casAuthors">Murage, Eunice N.; Gao, Guangzu; Bisello, Alessandro; Ahn, Jung-Mo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">6412-6420</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Glucagon-like peptide-1 (GLP-1) has the ability to lower the blood glucose level, and its regulatory functions make it an attractive therapeutic agent for the treatment of type 2 diabetes.  However, its rapid degrdn. by enzymes like dipeptidyl peptidase-IV (DPP-IV) and neutral endopeptidase (NEP) 24.11 severely compromises its effective clin. use.  Whereas specific DPP-IV inhibitors have been developed, NEP 24.11 targets multiple sites in the GLP-1 sequence, which makes it difficult to block.  To address this drawback, we have designed and synthesized conformationally constrained GLP-1 analogs by introducing multiple lactam bridges that stabilized both α-helixes in the N- and C-terminal regions simultaneously.  In addn. to improving the receptor activation capability (up to 5-fold) by fixing the α-helical conformations required for optimal receptor interaction, the introduced lactam bridges provided outstanding shielding over NEP 24.11 (half-life of >96 h).  These highly constrained peptides are the first examples of NEP 24.11-resistant GLP-1 analogs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGQuuDP1-5D7Vg90H21EOLACvtfcHk0lgmx_AW7mO5ww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXpvVarsbw%253D&md5=a7a8385aa91eae3f44d1231c8bb937d7</span></div><a href="/servlet/linkout?suffix=cit131&amp;dbid=16384&amp;doi=10.1021%2Fjm100602m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm100602m%26sid%3Dliteratum%253Aachs%26aulast%3DMurage%26aufirst%3DE.%2BN.%26aulast%3DGao%26aufirst%3DG.%26aulast%3DBisello%26aufirst%3DA.%26aulast%3DAhn%26aufirst%3DJ.-M.%26atitle%3DDevelopment%2520of%2520Potent%2520Glucagon-like%2520Peptide-1%2520Agonists%2520with%2520High%2520Enzyme%2520Stability%2520via%2520Introduction%2520of%2520Multiple%2520Lactam%2520Bridges%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D6412%26epage%3D6420%26doi%3D10.1021%2Fjm100602m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref132"><div class="reference"><strong class="refLabel"><a href="#ref132" class="rightTabRefNumLink">132</a></strong><div class="NLM_citation" id="rightTab-cit132"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murage, E. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schroeder, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beinborn, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahn, J.-M.</span></span> <span> </span><span class="NLM_article-title">Search for Alpha-Helical Propensity in the Receptor-Bound Conformation of Glucagon-like Peptide-1</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">10106</span>– <span class="NLM_lpage">10112</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2008.10.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref132/cit132&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1016%2Fj.bmc.2008.10.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref132/cit132&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=18952440" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref132/cit132&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsVSgt7bK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2008&pages=10106-10112&author=E.+N.+Murageauthor=J.+C.+Schroederauthor=M.+Beinbornauthor=J.-M.+Ahn&title=Search+for+Alpha-Helical+Propensity+in+the+Receptor-Bound+Conformation+of+Glucagon-like+Peptide-1&doi=10.1016%2Fj.bmc.2008.10.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit132R"><div class="casContent"><span class="casTitleNuber">132</span><div class="casTitle"><span class="NLM_cas:atitle">Search for α-helical propensity in the receptor-bound conformation of glucagon-like peptide-1</span></div><div class="casAuthors">Murage, Eunice N.; Schroeder, Jonathan C.; Beinborn, Martin; Ahn, Jung-Mo</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">10106-10112</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">To elucidate the receptor-bound conformation of glucagon-like peptide-1 (GLP-1), a series of conformationally constrained GLP-1 analogs were synthesized by introducing lactam bridges between Lysi and Glui +4 to form α-helixes at various positions.  The activity and affinity of these analogs to GLP-1 receptors suggested that the receptor-bound conformation comprises two α-helical segments between residues 11-21 and 23-34.  It is notable that the N-terminal α-helix is extended to Thr11, and that Gly22 plays a pivotal role in arranging the two α-helixes.  Based on these findings, a highly potent bicyclic GLP-1 analog was synthesized which is the most conformationally constrained GLP-1 analog reported to date.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9FswyXDckPrVg90H21EOLACvtfcHk0lgmx_AW7mO5ww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsVSgt7bK&md5=cd451fed8eedd20d058e63d0adc01bdf</span></div><a href="/servlet/linkout?suffix=cit132&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2008.10.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2008.10.006%26sid%3Dliteratum%253Aachs%26aulast%3DMurage%26aufirst%3DE.%2BN.%26aulast%3DSchroeder%26aufirst%3DJ.%2BC.%26aulast%3DBeinborn%26aufirst%3DM.%26aulast%3DAhn%26aufirst%3DJ.-M.%26atitle%3DSearch%2520for%2520Alpha-Helical%2520Propensity%2520in%2520the%2520Receptor-Bound%2520Conformation%2520of%2520Glucagon-like%2520Peptide-1%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2008%26volume%3D16%26spage%3D10106%26epage%3D10112%26doi%3D10.1016%2Fj.bmc.2008.10.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref133"><div class="reference"><strong class="refLabel"><a href="#ref133" class="rightTabRefNumLink">133</a></strong><div class="NLM_citation" id="rightTab-cit133"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schreuder, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liesum, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lönze, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stump, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffmann, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiell, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toti, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bauer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kallus, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klemke-Jahn, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Czech, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kramer, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Enke, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niedermeyer, T. H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morrison, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brönstrup, M.</span></span> <span> </span><span class="NLM_article-title">Isolation, Co-Crystallization and Structure-Based Characterization of Anabaenopeptins as Highly Potent Inhibitors of Activated Thrombin Activatable Fibrinolysis Inhibitor (TAFIa)</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>6</i></span>, <span class="NLM_elocation-id">32958</span> <span class="refDoi"> DOI: 10.1038/srep32958</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref133/cit133&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1038%2Fsrep32958" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref133/cit133&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=27604544" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref133/cit133&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC2sXps1OktA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2016&author=H.+Schreuderauthor=A.+Liesumauthor=P.+L%C3%B6nzeauthor=H.+Stumpauthor=H.+Hoffmannauthor=M.+Schiellauthor=M.+Kurzauthor=L.+Totiauthor=A.+Bauerauthor=C.+Kallusauthor=C.+Klemke-Jahnauthor=J.+Czechauthor=D.+Kramerauthor=H.+Enkeauthor=T.+H.+J.+Niedermeyerauthor=V.+Morrisonauthor=V.+Kumarauthor=M.+Br%C3%B6nstrup&title=Isolation%2C+Co-Crystallization+and+Structure-Based+Characterization+of+Anabaenopeptins+as+Highly+Potent+Inhibitors+of+Activated+Thrombin+Activatable+Fibrinolysis+Inhibitor+%28TAFIa%29&doi=10.1038%2Fsrep32958"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit133R"><div class="casContent"><span class="casTitleNuber">133</span><div class="casTitle"><span class="NLM_cas:atitle">Isolation, Co-Crystallization and Structure-Based Characterization of Anabaenopeptins as Highly Potent Inhibitors of Activated Thrombin Activatable Fibrinolysis Inhibitor (TAFIa)</span></div><div class="casAuthors">Schreuder, Herman; Liesum, Alexander; Loenze, Petra; Stump, Heike; Hoffmann, Holger; Schiell, Matthias; Kurz, Michael; Toti, Luigi; Bauer, Armin; Kallus, Christopher; Klemke-Jahn, Christine; Czech, Joerg; Kramer, Dan; Enke, Heike; Niedermeyer, Timo H. J.; Morrison, Vincent; Kumar, Vasant; Broenstrup, Mark</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">32958</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Mature thrombin activatable fibrinolysis inhibitor (TAFIa) is a carboxypeptidase that stabilizes fibrin clots by removing C-terminal arginines and lysines from partially degraded fibrin.  Inhibition of TAFIa stimulates the degrdn. of fibrin clots and may help to prevent thrombosis.  Applying a lead finding approach based on literature-mining, we discovered that anabaenopeptins, cyclic peptides produced by cyanobacteria, were potent inhibitors of TAFIa with IC50 values as low as 1.5 nM.  We describe the isolation and structure elucidation of 20 anabaenopeptins, including 13 novel congeners, as well as their pronounced structure-activity relationships (SAR) with respect to inhibition of TAFIa.  Crystal structures of the anabaenopeptins B, C and F bound to the surrogate protease carboxypeptidase B revealed the binding modes of these large (∼850 Da) compds. in detail and explained the obsd. SAR, i.e. the strong dependence of the potency on a basic (Arg, Lys) exocyclic residue that addressed the S1' binding pocket, and a broad tolerance towards substitutions in the pentacyclic ring that acted as a plug of the active site.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNu4VixSn7RrVg90H21EOLACvtfcHk0lgmx_AW7mO5ww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXps1OktA%253D%253D&md5=4c7faf50dcdf4bb7b4f9b88c9e064a1d</span></div><a href="/servlet/linkout?suffix=cit133&amp;dbid=16384&amp;doi=10.1038%2Fsrep32958&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep32958%26sid%3Dliteratum%253Aachs%26aulast%3DSchreuder%26aufirst%3DH.%26aulast%3DLiesum%26aufirst%3DA.%26aulast%3DL%25C3%25B6nze%26aufirst%3DP.%26aulast%3DStump%26aufirst%3DH.%26aulast%3DHoffmann%26aufirst%3DH.%26aulast%3DSchiell%26aufirst%3DM.%26aulast%3DKurz%26aufirst%3DM.%26aulast%3DToti%26aufirst%3DL.%26aulast%3DBauer%26aufirst%3DA.%26aulast%3DKallus%26aufirst%3DC.%26aulast%3DKlemke-Jahn%26aufirst%3DC.%26aulast%3DCzech%26aufirst%3DJ.%26aulast%3DKramer%26aufirst%3DD.%26aulast%3DEnke%26aufirst%3DH.%26aulast%3DNiedermeyer%26aufirst%3DT.%2BH.%2BJ.%26aulast%3DMorrison%26aufirst%3DV.%26aulast%3DKumar%26aufirst%3DV.%26aulast%3DBr%25C3%25B6nstrup%26aufirst%3DM.%26atitle%3DIsolation%252C%2520Co-Crystallization%2520and%2520Structure-Based%2520Characterization%2520of%2520Anabaenopeptins%2520as%2520Highly%2520Potent%2520Inhibitors%2520of%2520Activated%2520Thrombin%2520Activatable%2520Fibrinolysis%2520Inhibitor%2520%2528TAFIa%2529%26jtitle%3DSci.%2520Rep.%26date%3D2016%26volume%3D6%26doi%3D10.1038%2Fsrep32958" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref134"><div class="reference"><strong class="refLabel"><a href="#ref134" class="rightTabRefNumLink">134</a></strong><div class="NLM_citation" id="rightTab-cit134"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Halland, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brönstrup, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Czech, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Czechtizky, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evers, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Follmann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kohlmann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiell, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schreuder, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kallus, C.</span></span> <span> </span><span class="NLM_article-title">Novel Small Molecule Inhibitors of Activated Thrombin Activatable Fibrinolysis Inhibitor (TAFIa) from Natural Product Anabaenopeptin</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">4839</span>– <span class="NLM_lpage">4844</span>, <span class="refDoi"> DOI: 10.1021/jm501840b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501840b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref134/cit134&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC2MXovVSnt7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=4839-4844&author=N.+Hallandauthor=M.+Br%C3%B6nstrupauthor=J.+Czechauthor=W.+Czechtizkyauthor=A.+Eversauthor=M.+Follmannauthor=M.+Kohlmannauthor=M.+Schiellauthor=M.+Kurzauthor=H.+A.+Schreuderauthor=C.+Kallus&title=Novel+Small+Molecule+Inhibitors+of+Activated+Thrombin+Activatable+Fibrinolysis+Inhibitor+%28TAFIa%29+from+Natural+Product+Anabaenopeptin&doi=10.1021%2Fjm501840b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit134R"><div class="casContent"><span class="casTitleNuber">134</span><div class="casTitle"><span class="NLM_cas:atitle">Novel Small Molecule Inhibitors of Activated Thrombin Activatable Fibrinolysis Inhibitor (TAFIa) from Natural Product Anabaenopeptin</span></div><div class="casAuthors">Halland, Nis; Broenstrup, Mark; Czech, Joerg; Czechtizky, Werngard; Evers, Andreas; Follmann, Markus; Kohlmann, Markus; Schiell, Matthias; Kurz, Michael; Schreuder, Herman A.; Kallus, Christopher</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4839-4844</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Anabaenopeptins isolated from cyanobacteria were identified as inhibitors of carboxypeptidase TAFIa.  Cocrystal structures of these macrocyclic natural product inhibitors in a modified porcine carboxypeptidase B revealed their binding mode and provided the basis for the rational design of small mol. inhibitors with a previously unknown central urea motif.  Optimization based on these design concepts allowed for a rapid evaluation of the SAR and delivered potent small mol. inhibitors of TAFIa with a promising overall profile, e.g. I.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7kmWgfMLAA7Vg90H21EOLACvtfcHk0lhALxybAZMhJQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXovVSnt7c%253D&md5=6e1df1c771aec9982d046148cea91358</span></div><a href="/servlet/linkout?suffix=cit134&amp;dbid=16384&amp;doi=10.1021%2Fjm501840b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501840b%26sid%3Dliteratum%253Aachs%26aulast%3DHalland%26aufirst%3DN.%26aulast%3DBr%25C3%25B6nstrup%26aufirst%3DM.%26aulast%3DCzech%26aufirst%3DJ.%26aulast%3DCzechtizky%26aufirst%3DW.%26aulast%3DEvers%26aufirst%3DA.%26aulast%3DFollmann%26aufirst%3DM.%26aulast%3DKohlmann%26aufirst%3DM.%26aulast%3DSchiell%26aufirst%3DM.%26aulast%3DKurz%26aufirst%3DM.%26aulast%3DSchreuder%26aufirst%3DH.%2BA.%26aulast%3DKallus%26aufirst%3DC.%26atitle%3DNovel%2520Small%2520Molecule%2520Inhibitors%2520of%2520Activated%2520Thrombin%2520Activatable%2520Fibrinolysis%2520Inhibitor%2520%2528TAFIa%2529%2520from%2520Natural%2520Product%2520Anabaenopeptin%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D4839%26epage%3D4844%26doi%3D10.1021%2Fjm501840b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref135"><div class="reference"><strong class="refLabel"><a href="#ref135" class="rightTabRefNumLink">135</a></strong><div class="NLM_citation" id="rightTab-cit135"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Halland, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Czech, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Czechtizky, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evers, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Follmann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kohlmann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schreuder, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kallus, C.</span></span> <span> </span><span class="NLM_article-title">Sulfamide as Zinc Binding Motif in Small Molecule Inhibitors of Activated Thrombin Activatable Fibrinolysis Inhibitor (TAFIa)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">9567</span>– <span class="NLM_lpage">9573</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01276</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01276" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref135/cit135&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhs1ylu7jE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=9567-9573&author=N.+Hallandauthor=J.+Czechauthor=W.+Czechtizkyauthor=A.+Eversauthor=M.+Follmannauthor=M.+Kohlmannauthor=H.+A.+Schreuderauthor=C.+Kallus&title=Sulfamide+as+Zinc+Binding+Motif+in+Small+Molecule+Inhibitors+of+Activated+Thrombin+Activatable+Fibrinolysis+Inhibitor+%28TAFIa%29&doi=10.1021%2Facs.jmedchem.6b01276"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit135R"><div class="casContent"><span class="casTitleNuber">135</span><div class="casTitle"><span class="NLM_cas:atitle">Sulfamide as Zinc Binding Motif in Small Molecule Inhibitors of Activated Thrombin Activatable Fibrinolysis Inhibitor (TAFIa)</span></div><div class="casAuthors">Halland, Nis; Czech, Joerg; Czechtizky, Werngard; Evers, Andreas; Follmann, Markus; Kohlmann, Markus; Schreuder, Herman A.; Kallus, Christopher</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">9567-9573</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Previously disclosed TAFIa inhibitors having a urea zinc-binding motif were used as the starting point for the development of a novel class of highly potent inhibitors having a sulfamide zinc-binding motif.  High-resoln. X-ray cocrystal structures were used to optimize the structures and reveal a highly unusual sulfamide configuration.  A selected sulfamide was profiled in vitro and in vivo and displayed a promising ADMET profile.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoj1LpJOb8JYrVg90H21EOLACvtfcHk0lhALxybAZMhJQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhs1ylu7jE&md5=8ffb8d0719f609351b9a67271d94fef4</span></div><a href="/servlet/linkout?suffix=cit135&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01276&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01276%26sid%3Dliteratum%253Aachs%26aulast%3DHalland%26aufirst%3DN.%26aulast%3DCzech%26aufirst%3DJ.%26aulast%3DCzechtizky%26aufirst%3DW.%26aulast%3DEvers%26aufirst%3DA.%26aulast%3DFollmann%26aufirst%3DM.%26aulast%3DKohlmann%26aufirst%3DM.%26aulast%3DSchreuder%26aufirst%3DH.%2BA.%26aulast%3DKallus%26aufirst%3DC.%26atitle%3DSulfamide%2520as%2520Zinc%2520Binding%2520Motif%2520in%2520Small%2520Molecule%2520Inhibitors%2520of%2520Activated%2520Thrombin%2520Activatable%2520Fibrinolysis%2520Inhibitor%2520%2528TAFIa%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D9567%26epage%3D9573%26doi%3D10.1021%2Facs.jmedchem.6b01276" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref136"><div class="reference"><strong class="refLabel"><a href="#ref136" class="rightTabRefNumLink">136</a></strong><div class="NLM_citation" id="rightTab-cit136"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, Y.</span></span> <span> </span><span class="NLM_article-title">PROTACs: Great Opportunities for Academia and Industry</span>. <i>Signal Transduct. Target. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>4</i></span>, <span class="NLM_elocation-id">64</span> <span class="refDoi"> DOI: 10.1038/s41392-019-0101-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref136/cit136&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1038%2Fs41392-019-0101-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref136/cit136&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=31885879" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref136/cit136&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A280%3ADC%252BB3MbkvFCrtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2019&author=X.+Sunauthor=H.+Gaoauthor=Y.+Yangauthor=M.+Heauthor=Y.+Wuauthor=Y.+Songauthor=Y.+Tongauthor=Y.+Rao&title=PROTACs%3A+Great+Opportunities+for+Academia+and+Industry&doi=10.1038%2Fs41392-019-0101-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit136R"><div class="casContent"><span class="casTitleNuber">136</span><div class="casTitle"><span class="NLM_cas:atitle">PROTACs: great opportunities for academia and industry</span></div><div class="casAuthors">Sun Xiuyun; Gao Hongying; Yang Yiqing; He Ming; Wu Yue; Song Yugang; Tong Yan; Rao Yu; Sun Xiuyun; Gao Hongying; Yang Yiqing; Rao Yu</div><div class="citationInfo"><span class="NLM_cas:title">Signal transduction and targeted therapy</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">64</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Although many kinds of therapies are applied in the clinic, drug-resistance is a major and unavoidable problem.  Another disturbing statistic is the limited number of drug targets, which are presently only 20-25% of all protein targets that are currently being studied.  Moreover, the focus of current explorations of targets are their enzymatic functions, which ignores the functions from their scaffold moiety.  As a promising and appealing technology, PROteolysis TArgeting Chimeras (PROTACs) have attracted great attention both from academia and industry for finding available approaches to solve the above problems.  PROTACs regulate protein function by degrading target proteins instead of inhibiting them, providing more sensitivity to drug-resistant targets and a greater chance to affect the nonenzymatic functions.  PROTACs have been proven to show better selectivity compared to classic inhibitors.  PROTACs can be described as a chemical knockdown approach with rapidity and reversibility, which presents new and different biology compared to other gene editing tools by avoiding misinterpretations that arise from potential genetic compensation and/or spontaneous mutations.  PRTOACs have been widely explored throughout the world and have outperformed not only in cancer diseases, but also in immune disorders, viral infections and neurodegenerative diseases.  Although PROTACs present a very promising and powerful approach for crossing the hurdles of present drug discovery and tool development in biology, more efforts are needed to gain to get deeper insight into the efficacy and safety of PROTACs in the clinic.  More target binders and more E3 ligases applicable for developing PROTACs are waiting for exploration.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQretvERU2xT-_Q9URG6Dx8fW6udTcc2ebsjNE7JKX15bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3MbkvFCrtA%253D%253D&md5=7d9545a21e3a78cededf2977a31e8f7d</span></div><a href="/servlet/linkout?suffix=cit136&amp;dbid=16384&amp;doi=10.1038%2Fs41392-019-0101-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41392-019-0101-6%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DX.%26aulast%3DGao%26aufirst%3DH.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DHe%26aufirst%3DM.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DSong%26aufirst%3DY.%26aulast%3DTong%26aufirst%3DY.%26aulast%3DRao%26aufirst%3DY.%26atitle%3DPROTACs%253A%2520Great%2520Opportunities%2520for%2520Academia%2520and%2520Industry%26jtitle%3DSignal%2520Transduct.%2520Target.%2520Ther.%26date%3D2019%26volume%3D4%26doi%3D10.1038%2Fs41392-019-0101-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref137"><div class="reference"><strong class="refLabel"><a href="#ref137" class="rightTabRefNumLink">137</a></strong><div class="NLM_citation" id="rightTab-cit137"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Testa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucas, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wright, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Structure-Based Design of a Macrocyclic PROTAC</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">1727</span>– <span class="NLM_lpage">1734</span>, <span class="refDoi"> DOI: 10.1002/anie.201914396</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref137/cit137&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1002%2Fanie.201914396" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref137/cit137&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitlyitbjK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2020&pages=1727-1734&author=A.+Testaauthor=S.+J.+Hughesauthor=X.+Lucasauthor=J.+E.+Wrightauthor=A.+Ciulli&title=Structure-Based+Design+of+a+Macrocyclic+PROTAC&doi=10.1002%2Fanie.201914396"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit137R"><div class="casContent"><span class="casTitleNuber">137</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Design of a Macrocyclic PROTAC</span></div><div class="casAuthors">Testa, Andrea; Hughes, Scott J.; Lucas, Xavier; Wright, Jane E.; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1727-1734</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Constraining a mol. in its bioactive conformation via macrocyclization represents an attractive strategy to rationally design functional chem. probes.  While this approach has been applied to enzyme inhibitors or receptor antagonists, to date it remains unprecedented for bifunctional mols. that bring proteins together, such as PROTAC degraders.  Herein, the authors report the design and synthesis of a macrocyclic PROTAC by adding a cyclizing linker to the BET degrader MZ1.  A cocrystal structure of macroPROTAC-1 bound in a ternary complex with VHL and the second bromodomain of Brd4 validated the rational design.  Biophys. studies revealed enhanced discrimination between the second and the first bromodomains of BET proteins.  Despite a 12-fold loss of binary binding affinity for Brd4, macroPROTAC-1 exhibited cellular activity comparable to MZ1.  The findings support macrocyclization as an advantageous strategy to enhance PROTAC degrdn. potency and selectivity between homologous targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZjLgy_V3_abVg90H21EOLACvtfcHk0liA8va1UJAThQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitlyitbjK&md5=6f3f279f6220ec30b30ca3d2c931da95</span></div><a href="/servlet/linkout?suffix=cit137&amp;dbid=16384&amp;doi=10.1002%2Fanie.201914396&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201914396%26sid%3Dliteratum%253Aachs%26aulast%3DTesta%26aufirst%3DA.%26aulast%3DHughes%26aufirst%3DS.%2BJ.%26aulast%3DLucas%26aufirst%3DX.%26aulast%3DWright%26aufirst%3DJ.%2BE.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DStructure-Based%2520Design%2520of%2520a%2520Macrocyclic%2520PROTAC%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2020%26volume%3D59%26spage%3D1727%26epage%3D1734%26doi%3D10.1002%2Fanie.201914396" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref138"><div class="reference"><strong class="refLabel"><a href="#ref138" class="rightTabRefNumLink">138</a></strong><div class="NLM_citation" id="rightTab-cit138"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Molecular Recognition of Ternary Complexes: A New Dimension in the Structure-Guided Design of Chemical Degraders</span>. <i>Essays Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">505</span>– <span class="NLM_lpage">516</span>, <span class="refDoi"> DOI: 10.1042/EBC20170041</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref138/cit138&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1042%2FEBC20170041" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref138/cit138&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=29118097" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref138/cit138&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A280%3ADC%252BC1M7pt1Gmuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2017&pages=505-516&author=S.+J.+Hughesauthor=A.+Ciulli&title=Molecular+Recognition+of+Ternary+Complexes%3A+A+New+Dimension+in+the+Structure-Guided+Design+of+Chemical+Degraders&doi=10.1042%2FEBC20170041"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit138R"><div class="casContent"><span class="casTitleNuber">138</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular recognition of ternary complexes: a new dimension in the structure-guided design of chemical degraders</span></div><div class="casAuthors">Hughes Scott J; Ciulli Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Essays in biochemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">505-516</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Molecular glues and bivalent inducers of protein degradation (also known as PROTACs) represent a fascinating new modality in pharmacotherapeutics: the potential to knockdown previously thought 'undruggable' targets at sub-stoichiometric concentrations in ways not possible using conventional inhibitors.  Mounting evidence suggests these chemical agents, in concert with their target proteins, can be modelled as three-body binding equilibria that can exhibit significant cooperativity as a result of specific ligand-induced molecular recognition.  Despite this, many existing drug design and optimization regimens still fixate on binary target engagement, in part due to limited structural data on ternary complexes.  Recent crystal structures of protein complexes mediated by degrader molecules, including the first PROTAC ternary complex, underscore the importance of protein-protein interactions and intramolecular contacts to the mode of action of this class of compounds.  These discoveries have opened the door to a new paradigm for structure-guided drug design: borrowing surface area and molecular recognition from nature to elicit cellular signalling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTqZGQ3bfUY0b5XHngOXiV_fW6udTcc2ebRBARSAg9AW7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M7pt1Gmuw%253D%253D&md5=7470e559372f05d34c5924524a6d04bc</span></div><a href="/servlet/linkout?suffix=cit138&amp;dbid=16384&amp;doi=10.1042%2FEBC20170041&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FEBC20170041%26sid%3Dliteratum%253Aachs%26aulast%3DHughes%26aufirst%3DS.%2BJ.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DMolecular%2520Recognition%2520of%2520Ternary%2520Complexes%253A%2520A%2520New%2520Dimension%2520in%2520the%2520Structure-Guided%2520Design%2520of%2520Chemical%2520Degraders%26jtitle%3DEssays%2520Biochem.%26date%3D2017%26volume%3D61%26spage%3D505%26epage%3D516%26doi%3D10.1042%2FEBC20170041" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref139"><div class="reference"><strong class="refLabel"><a href="#ref139" class="rightTabRefNumLink">139</a></strong><div class="NLM_citation" id="rightTab-cit139"><span><span class="NLM_contrib-group"><span class="NLM_string-name">De Guzman, R. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goto, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dyson, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wright, P. E.</span></span> <span> </span><span class="NLM_article-title">Structural Basis for Cooperative Transcription Factor Binding to the CBP Coactivator</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>355</i></span>,  <span class="NLM_fpage">1005</span>– <span class="NLM_lpage">1013</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2005.09.059</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref139/cit139&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1016%2Fj.jmb.2005.09.059" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref139/cit139&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=16253272" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref139/cit139&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtlGlsLzO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=355&publication_year=2006&pages=1005-1013&author=R.+N.+De+Guzmanauthor=N.+K.+Gotoauthor=H.+J.+Dysonauthor=P.+E.+Wright&title=Structural+Basis+for+Cooperative+Transcription+Factor+Binding+to+the+CBP+Coactivator&doi=10.1016%2Fj.jmb.2005.09.059"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit139R"><div class="casContent"><span class="casTitleNuber">139</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Basis for Cooperative Transcription Factor Binding to the CBP Coactivator</span></div><div class="casAuthors">De Guzman, Roberto N.; Goto, Natalie K.; Dyson, H. Jane; Wright, Peter E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">355</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1005-1013</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Regulation of transcription requires interactions between transcriptional activators and transcriptional co-activator CREB binding protein (CBP).  The KIX domain of CBP can bind simultaneously to two different proteins, providing an addnl. mechanism for transcriptional regulation.  Here we describe the soln. structure of the ternary complex formed by cooperative binding of activation domains from the c-Myb and mixed lineage leukemia (MLL) transcription factors to the KIX domain.  The MLL and c-Myb domains form helixes that bind to two distinct hydrophobic grooves on opposite faces of KIX.  Compared to the binary KIX:c-Myb complex, significant changes are obsd. in the structure of KIX at the MLL binding interface in the ternary complex.  Two regions of KIX that are disordered in the binary complex become structured in the ternary complex: a flexible loop forms intimate contacts with bound MLL, and the C-terminal helix is extended and stabilized by MLL binding.  This structural change results in the formation of addnl. electrostatic/polar interactions between KIX and the bound c-Myb, providing a structural basis for the cooperativity obsd. for the ternary complex.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2Ig7OPYgRFbVg90H21EOLACvtfcHk0liA8va1UJAThQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtlGlsLzO&md5=a7cb1e86b16731b387558cb10bd62d8d</span></div><a href="/servlet/linkout?suffix=cit139&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2005.09.059&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2005.09.059%26sid%3Dliteratum%253Aachs%26aulast%3DDe%2BGuzman%26aufirst%3DR.%2BN.%26aulast%3DGoto%26aufirst%3DN.%2BK.%26aulast%3DDyson%26aufirst%3DH.%2BJ.%26aulast%3DWright%26aufirst%3DP.%2BE.%26atitle%3DStructural%2520Basis%2520for%2520Cooperative%2520Transcription%2520Factor%2520Binding%2520to%2520the%2520CBP%2520Coactivator%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2006%26volume%3D355%26spage%3D1005%26epage%3D1013%26doi%3D10.1016%2Fj.jmb.2005.09.059" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref140"><div class="reference"><strong class="refLabel"><a href="#ref140" class="rightTabRefNumLink">140</a></strong><div class="NLM_citation" id="rightTab-cit140"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guedes, I. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pereira, F. S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dardenne, L. E.</span></span> <span> </span><span class="NLM_article-title">Empirical Scoring Functions for Structure-Based Virtual Screening: Applications, Critical Aspects, and Challenges</span>. <i>Front. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>, <span class="NLM_elocation-id">1089</span> <span class="refDoi"> DOI: 10.3389/fphar.2018.01089</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref140/cit140&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.3389%2Ffphar.2018.01089" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref140/cit140&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=30319422" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref140/cit140&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC1MXpt1WqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&author=I.+A.+Guedesauthor=F.+S.+S.+Pereiraauthor=L.+E.+Dardenne&title=Empirical+Scoring+Functions+for+Structure-Based+Virtual+Screening%3A+Applications%2C+Critical+Aspects%2C+and+Challenges&doi=10.3389%2Ffphar.2018.01089"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit140R"><div class="casContent"><span class="casTitleNuber">140</span><div class="casTitle"><span class="NLM_cas:atitle">Empirical scoring functions for structure-based virtual screening: applications, critical aspects, and challenges</span></div><div class="casAuthors">Guedes, Isabella A.; Pereira, Felipe S. S.; Dardenne, Laurent E.</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in Pharmacology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1089</span>CODEN:
                <span class="NLM_cas:coden">FPRHAU</span>;
        ISSN:<span class="NLM_cas:issn">1663-9812</span>.
    
            (<span class="NLM_cas:orgname">Frontiers Media S.A.</span>)
        </div><div class="casAbstract">A review.  Structure-based virtual screening (VS) is a widely used approach that employs the knowledge of the three-dimensional structure of the target of interest in the design of new lead compds. from large-scale mol. docking expts.  Through the prediction of the binding mode and affinity of a small mol. within the binding site of the target of interest, it is possible to understand important properties related to the binding process.  Empirical scoring functions are widely used for pose and affinity prediction.  Although pose prediction is performed with satisfactory accuracy, the correct prediction of binding affinity is still a challenging task and crucial for the success of structure-based VS expts.  There are several efforts in distinct fronts to develop even more sophisticated and accurate models for filtering and ranking large libraries of compds.  This paper will cover some recent successful applications and methodol. advances, including strategies to explore the ligand entropy and solvent effects, training with sophisticated machine-learning techniques, and the use of quantum mechanics.  Particular emphasis will be given to the discussion of crit. aspects and further directions for the development of more accurate empirical scoring functions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpuNmYBXPKivLVg90H21EOLACvtfcHk0liut1E-aMWceQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXpt1WqsLk%253D&md5=6fd60ccc4d6c2c85a206030023f7c0d1</span></div><a href="/servlet/linkout?suffix=cit140&amp;dbid=16384&amp;doi=10.3389%2Ffphar.2018.01089&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffphar.2018.01089%26sid%3Dliteratum%253Aachs%26aulast%3DGuedes%26aufirst%3DI.%2BA.%26aulast%3DPereira%26aufirst%3DF.%2BS.%2BS.%26aulast%3DDardenne%26aufirst%3DL.%2BE.%26atitle%3DEmpirical%2520Scoring%2520Functions%2520for%2520Structure-Based%2520Virtual%2520Screening%253A%2520Applications%252C%2520Critical%2520Aspects%252C%2520and%2520Challenges%26jtitle%3DFront.%2520Pharmacol.%26date%3D2018%26volume%3D9%26doi%3D10.3389%2Ffphar.2018.01089" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref141"><div class="reference"><strong class="refLabel"><a href="#ref141" class="rightTabRefNumLink">141</a></strong><div class="NLM_citation" id="rightTab-cit141"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Le Roux, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blaise, É.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boudreault, P.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Comeau, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doucet, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giarrusso, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collin, M.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neubauer, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kölling, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Göller, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seep, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tshitenge, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wittwer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kullmann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hillisch, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mittendorf, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marsault, E.</span></span> <span> </span><span class="NLM_article-title">Structure–Permeability Relationship of Semipeptidic Macrocycles—Understanding and Optimizing Passive Permeability and Efflux Ratio</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">6774</span>– <span class="NLM_lpage">6783</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c00013</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00013" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref141/cit141&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtVSht73F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=6774-6783&author=A.+Le+Rouxauthor=%C3%89.+Blaiseauthor=P.-L.+Boudreaultauthor=C.+Comeauauthor=A.+Doucetauthor=M.+Giarrussoauthor=M.-P.+Collinauthor=T.+Neubauerauthor=F.+K%C3%B6llingauthor=A.+H.+G%C3%B6llerauthor=L.+Seepauthor=D.+T.+Tshitengeauthor=M.+Wittwerauthor=M.+Kullmannauthor=A.+Hillischauthor=J.+Mittendorfauthor=E.+Marsault&title=Structure%E2%80%93Permeability+Relationship+of+Semipeptidic+Macrocycles%E2%80%94Understanding+and+Optimizing+Passive+Permeability+and+Efflux+Ratio&doi=10.1021%2Facs.jmedchem.0c00013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit141R"><div class="casContent"><span class="casTitleNuber">141</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Permeability Relationship of Semipeptidic Macrocycles-Understanding and Optimizing Passive Permeability and Efflux Ratio</span></div><div class="casAuthors">Le Roux, Antoine; Blaise, Emilie; Boudreault, Pierre-Luc; Comeau, Christian; Doucet, Annie; Giarrusso, Marilena; Collin, Marie-Pierre; Neubauer, Thomas; Kolling, Florian; Goller, Andreas H.; Seep, Lea; Tshitenge, Dieudonne T.; Wittwer, Matthias; Kullmann, Maximilian; Hillisch, Alexander; Mittendorf, Joachim; Marsault, Eric</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">6774-6783</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We herein report the first thorough anal. of the structure-permeability relationship of semipeptidic macrocycles.  In total, 47 macrocycles were synthesized using a hybrid solid-phase/soln. strategy, and then their passive and cellular permeability was assessed using the parallel artificial membrane permeability assay (PAMPA) and Caco-2 assay, resp.  The results indicate that semipeptidic macrocycles generally possess high passive permeability based on the PAMPA, yet their cellular permeability is governed by efflux, as reported in the Caco-2 assay.  Structural variations led to tractable structure-permeability and structure-efflux relationships, wherein the linker length, stereoinversion, N-methylation, and peptoids site-specifically impact the permeability and efflux.  Extensive NMR, mol. dynamics, and ensemble-based three-dimensional polar surface area (3D-PSA) studies showed that ensemble-based 3D-PSA is a good predictor of passive permeability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqCamxP_rXRLVg90H21EOLACvtfcHk0liut1E-aMWceQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtVSht73F&md5=2794db50565ab0efbc123d0ec541ff32</span></div><a href="/servlet/linkout?suffix=cit141&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00013%26sid%3Dliteratum%253Aachs%26aulast%3DLe%2BRoux%26aufirst%3DA.%26aulast%3DBlaise%26aufirst%3D%25C3%2589.%26aulast%3DBoudreault%26aufirst%3DP.-L.%26aulast%3DComeau%26aufirst%3DC.%26aulast%3DDoucet%26aufirst%3DA.%26aulast%3DGiarrusso%26aufirst%3DM.%26aulast%3DCollin%26aufirst%3DM.-P.%26aulast%3DNeubauer%26aufirst%3DT.%26aulast%3DK%25C3%25B6lling%26aufirst%3DF.%26aulast%3DG%25C3%25B6ller%26aufirst%3DA.%2BH.%26aulast%3DSeep%26aufirst%3DL.%26aulast%3DTshitenge%26aufirst%3DD.%2BT.%26aulast%3DWittwer%26aufirst%3DM.%26aulast%3DKullmann%26aufirst%3DM.%26aulast%3DHillisch%26aufirst%3DA.%26aulast%3DMittendorf%26aufirst%3DJ.%26aulast%3DMarsault%26aufirst%3DE.%26atitle%3DStructure%25E2%2580%2593Permeability%2520Relationship%2520of%2520Semipeptidic%2520Macrocycles%25E2%2580%2594Understanding%2520and%2520Optimizing%2520Passive%2520Permeability%2520and%2520Efflux%2520Ratio%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D6774%26epage%3D6783%26doi%3D10.1021%2Facs.jmedchem.0c00013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref142"><div class="reference"><strong class="refLabel"><a href="#ref142" class="rightTabRefNumLink">142</a></strong><div class="NLM_citation" id="rightTab-cit142"><span><span class="NLM_contrib-group"><span class="NLM_string-name">King, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fields, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fields, G. B.</span></span> <span> </span><span class="NLM_article-title">A Cleavage Method Which Minimizes Side Reactions Following Fmoc Solid Phase Peptide Synthesis</span>. <i>Int. J. Pept. Protein Res.</i> <span class="NLM_year" style="font-weight: bold;">1990</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">255</span>– <span class="NLM_lpage">266</span>, <span class="refDoi"> DOI: 10.1111/j.1399-3011.1990.tb00976.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref142/cit142&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1111%2Fj.1399-3011.1990.tb00976.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref142/cit142&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=2279849" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref142/cit142&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADyaK3MXisVGktg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=1990&pages=255-266&author=D.+S.+Kingauthor=C.+G.+Fieldsauthor=G.+B.+Fields&title=A+Cleavage+Method+Which+Minimizes+Side+Reactions+Following+Fmoc+Solid+Phase+Peptide+Synthesis&doi=10.1111%2Fj.1399-3011.1990.tb00976.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit142R"><div class="casContent"><span class="casTitleNuber">142</span><div class="casTitle"><span class="NLM_cas:atitle">A cleavage method which minimizes side reactions following Fmoc solid phase peptide synthesis</span></div><div class="casAuthors">King, David S.; Fields, Cynthia G.; Fields, Gregg B.</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Peptide & Protein Research</span>
        (<span class="NLM_cas:date">1990</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">255-66</span>CODEN:
                <span class="NLM_cas:coden">IJPPC3</span>;
        ISSN:<span class="NLM_cas:issn">0367-8377</span>.
    </div><div class="casAbstract">The success of solid phase peptide synthesis utilizing 9-fluorenylmethoxycarbonyl (Fmoc) amino acids is often limited by deleterious side reactions which occur during CF3CO2H (TFA) peptide-resin cleavage and side-chain deprotection.  The majority of these side reactions modify susceptible residues, such as Trp, Tyr, Met, and Cys, with TFA-liberated side-chain protecting groups and linkers.  The relative effectiveness of various scavengers in suppressing these side reactions was studied.  The cleavage mixt. 82.5% TFA : 5% H2O : 5% thioanisole : 2.5% 1,2-ethanedithiol (reagent K) was maximally efficient in inhibiting a great variety of side reactions.  Synthesis and cleavage of 10 peptides, each contg. 20-50 residues, demonstrated the complementarity of Fmoc chem. with reagent K for efficient synthesis of complex peptide.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqUwcooNm7oD7Vg90H21EOLACvtfcHk0liut1E-aMWceQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3MXisVGktg%253D%253D&md5=ecf761fc6877559af82062c6345306ab</span></div><a href="/servlet/linkout?suffix=cit142&amp;dbid=16384&amp;doi=10.1111%2Fj.1399-3011.1990.tb00976.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1399-3011.1990.tb00976.x%26sid%3Dliteratum%253Aachs%26aulast%3DKing%26aufirst%3DD.%2BS.%26aulast%3DFields%26aufirst%3DC.%2BG.%26aulast%3DFields%26aufirst%3DG.%2BB.%26atitle%3DA%2520Cleavage%2520Method%2520Which%2520Minimizes%2520Side%2520Reactions%2520Following%2520Fmoc%2520Solid%2520Phase%2520Peptide%2520Synthesis%26jtitle%3DInt.%2520J.%2520Pept.%2520Protein%2520Res.%26date%3D1990%26volume%3D36%26spage%3D255%26epage%3D266%26doi%3D10.1111%2Fj.1399-3011.1990.tb00976.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref143"><div class="reference"><strong class="refLabel"><a href="#ref143" class="rightTabRefNumLink">143</a></strong><div class="NLM_citation" id="rightTab-cit143"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cummings, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sekharan, S.</span></span> <span> </span><span class="NLM_article-title">Structure-Based Macrocycle Design in Small-Molecule Drug Discovery and Simple Metrics To Identify Opportunities for Macrocyclization of Small-Molecule Ligands</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">6843</span>– <span class="NLM_lpage">6853</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01985</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01985" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref143/cit143&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC1MXks1yktb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=6843-6853&author=M.+D.+Cummingsauthor=S.+Sekharan&title=Structure-Based+Macrocycle+Design+in+Small-Molecule+Drug+Discovery+and+Simple+Metrics+To+Identify+Opportunities+for+Macrocyclization+of+Small-Molecule+Ligands&doi=10.1021%2Facs.jmedchem.8b01985"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit143R"><div class="casContent"><span class="casTitleNuber">143</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Macrocycle Design in Small-Molecule Drug Discovery and Simple Metrics To Identify Opportunities for Macrocyclization of Small-Molecule Ligands</span></div><div class="casAuthors">Cummings, Maxwell D.; Sekharan, Sivakumar</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">6843-6853</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Interest is growing in the use of macrocycles in pharmaceutical discovery.  Macrocylization may provide a gateway to an expanded chem. space for small-mol. drug discovery, and this could be beneficial in prosecuting difficult targets, e.g., protein-protein interactions.  Most, but not all, macrocycle drugs are derived from natural products.  Studies on synthetic drug-like small-mol. macrocycles are limited, and our current understanding of macrocycle drugs is similarly limited.  Following some background discussion, we review several examples of the structure-based design of synthetic macrocycles.  Our opinion is that in conformationally suitable systems macrocycles are an analog class worthy of consideration.  We then summarize an approach for the initial evaluation of mols. as candidates for macrocyclization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCAQg11Q6Hk7Vg90H21EOLACvtfcHk0lhaPX3aZi8ebQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXks1yktb4%253D&md5=32ac78d837075171d19c4cea694dce3f</span></div><a href="/servlet/linkout?suffix=cit143&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01985&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01985%26sid%3Dliteratum%253Aachs%26aulast%3DCummings%26aufirst%3DM.%2BD.%26aulast%3DSekharan%26aufirst%3DS.%26atitle%3DStructure-Based%2520Macrocycle%2520Design%2520in%2520Small-Molecule%2520Drug%2520Discovery%2520and%2520Simple%2520Metrics%2520To%2520Identify%2520Opportunities%2520for%2520Macrocyclization%2520of%2520Small-Molecule%2520Ligands%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D6843%26epage%3D6853%26doi%3D10.1021%2Facs.jmedchem.8b01985" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref144"><div class="reference"><strong class="refLabel"><a href="#ref144" class="rightTabRefNumLink">144</a></strong><div class="NLM_citation" id="rightTab-cit144"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jorgensen, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maxwell, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tirado-Rives, J.</span></span> <span> </span><span class="NLM_article-title">Development and Testing of the OPLS All-Atom Force Field on Conformational Energetics and Properties of Organic Liquids</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>118</i></span>,  <span class="NLM_fpage">11225</span>– <span class="NLM_lpage">11236</span>, <span class="refDoi"> DOI: 10.1021/ja9621760</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja9621760" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref144/cit144&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADyaK28XmtlOitrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=118&publication_year=1996&pages=11225-11236&author=W.+L.+Jorgensenauthor=D.+S.+Maxwellauthor=J.+Tirado-Rives&title=Development+and+Testing+of+the+OPLS+All-Atom+Force+Field+on+Conformational+Energetics+and+Properties+of+Organic+Liquids&doi=10.1021%2Fja9621760"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit144R"><div class="casContent"><span class="casTitleNuber">144</span><div class="casTitle"><span class="NLM_cas:atitle">Development and Testing of the OPLS All-Atom Force Field on Conformational Energetics and Properties of Organic Liquids</span></div><div class="casAuthors">Jorgensen, William L.; Maxwell, David S.; Tirado-Rives, Julian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">118</span>
        (<span class="NLM_cas:issue">45</span>),
    <span class="NLM_cas:pages">11225-11236</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The parametrization and testing of the OPLS all-atom force field for org. mols. and peptides are described.  Parameters for both torsional and nonbonded energetics have been derived, while the bond stretching and angle bending parameters have been adopted mostly from the AMBER all-atom force field.  The torsional parameters were detd. by fitting to rotational energy profiles obtained from ab initio MO calcns. at the RHF/6-31G*//RHF/6-31G* level for more than 50 org. mols. and ions.  The quality of the fits was high with av. errors for conformational energies of less than 0.2 kcal/mol.  The force-field results for mol. structures are also demonstrated to closely match the ab initio predictions.  The nonbonded parameters were developed in conjunction with Monte Carlo statistical mechanics simulations by computing thermodn. and structural properties for 34 pure org. liqs. including alkanes, alkenes, alcs., ethers, acetals, thiols, sulfides, disulfides, aldehydes, ketones, and amides.  Av. errors in comparison with exptl. data are 2% for heats of vaporization and densities.  The Monte Carlo simulations included sampling all internal and intermol. degrees of freedom.  It is found that such non-polar and monofunctional systems do not show significant condensed-phase effects on internal energies in going from the gas phase to the pure liqs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhGotSev_b1LVg90H21EOLACvtfcHk0lhaPX3aZi8ebQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XmtlOitrs%253D&md5=fef2924a69421881390282aa309ae91b</span></div><a href="/servlet/linkout?suffix=cit144&amp;dbid=16384&amp;doi=10.1021%2Fja9621760&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja9621760%26sid%3Dliteratum%253Aachs%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26aulast%3DMaxwell%26aufirst%3DD.%2BS.%26aulast%3DTirado-Rives%26aufirst%3DJ.%26atitle%3DDevelopment%2520and%2520Testing%2520of%2520the%2520OPLS%2520All-Atom%2520Force%2520Field%2520on%2520Conformational%2520Energetics%2520and%2520Properties%2520of%2520Organic%2520Liquids%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1996%26volume%3D118%26spage%3D11225%26epage%3D11236%26doi%3D10.1021%2Fja9621760" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref145"><div class="reference"><strong class="refLabel"><a href="#ref145" class="rightTabRefNumLink">145</a></strong><div class="NLM_citation" id="rightTab-cit145"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jorgensen, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tirado-Rives, J.</span></span> <span> </span><span class="NLM_article-title">The OPLS [Optimized Potentials for Liquid Simulations] Potential Functions for Proteins, Energy Minimizations for Crystals of Cyclic Peptides and Crambin</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">1988</span>,  <span class="NLM_volume"><i>110</i></span>,  <span class="NLM_fpage">1657</span>– <span class="NLM_lpage">1666</span>, <span class="refDoi"> DOI: 10.1021/ja00214a001</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja00214a001" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref145/cit145&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADyaL1cXht1yjt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=1988&pages=1657-1666&author=W.+L.+Jorgensenauthor=J.+Tirado-Rives&title=The+OPLS+%5BOptimized+Potentials+for+Liquid+Simulations%5D+Potential+Functions+for+Proteins%2C+Energy+Minimizations+for+Crystals+of+Cyclic+Peptides+and+Crambin&doi=10.1021%2Fja00214a001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit145R"><div class="casContent"><span class="casTitleNuber">145</span><div class="casTitle"><span class="NLM_cas:atitle">The OPLS [optimized potentials for liquid simulations] potential functions for proteins, energy minimizations for crystals of cyclic peptides and crambin</span></div><div class="casAuthors">Jorgensen, William L.; Tirado-Rives, Julian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1988</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1657-66</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    </div><div class="casAbstract">A complete set of intermol. potential functions was developed for use in computer simulations of proteins in their native environment.  Parameters are reported for 25 peptide residues as well as the common neutral and charged terminal groups.  The potential functions have the simple Coulomb plus Lennard-Jones form and are compatible with the widely used models for water, TIP4P, TIP3P, and SPC.  The parameters were obtained and tested primarily in conjunction with Monte Carlo statistical mechanics simulations of 36 pure org. liqs. and numerous aq. solns. of org. ions representative of subunits in the side chains and backbones of proteins.  Bond stretch, angle bend, and torsional terms were adopted from the AMBER united-atom force field.  As reported here, further testing involved studies of conformational energy surfaces and optimizations of the crystal structures for 4 cyclic hexapeptides and a cyclic pentapeptide.  The av. root mean square deviation from the x-ray structures of the crystals is only 0.17 Å for the at. positions and 3% for the unit cell vols.  A more crit. test was then provided by performing energy minimizations for the complete crystal of the protein crambin, including 182 water mols. that were initially placed via a Monte Carlo simulation.  The resultant root mean square deviation for the non-H atoms is still only 0.17 Å and the variation in the errors for charged, polar, and nonpolar residues is small.  Significant improvement is apparent over the AMBER united-atom force field which was previously demonstrated to be superior to many alternatives.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZ6an38QWApbVg90H21EOLACvtfcHk0lhaPX3aZi8ebQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1cXht1yjt7Y%253D&md5=b6c901d8c295b3b37329a7faef527e12</span></div><a href="/servlet/linkout?suffix=cit145&amp;dbid=16384&amp;doi=10.1021%2Fja00214a001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja00214a001%26sid%3Dliteratum%253Aachs%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26aulast%3DTirado-Rives%26aufirst%3DJ.%26atitle%3DThe%2520OPLS%2520%255BOptimized%2520Potentials%2520for%2520Liquid%2520Simulations%255D%2520Potential%2520Functions%2520for%2520Proteins%252C%2520Energy%2520Minimizations%2520for%2520Crystals%2520of%2520Cyclic%2520Peptides%2520and%2520Crambin%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1988%26volume%3D110%26spage%3D1657%26epage%3D1666%26doi%3D10.1021%2Fja00214a001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref146"><div class="reference"><strong class="refLabel"><a href="#ref146" class="rightTabRefNumLink">146</a></strong><div class="NLM_citation" id="rightTab-cit146"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shivakumar, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harder, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Damm, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sherman, W.</span></span> <span> </span><span class="NLM_article-title">Improving the Prediction of Absolute Solvation Free Energies Using the Next Generation OPLS Force Field</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">2553</span>– <span class="NLM_lpage">2558</span>, <span class="refDoi"> DOI: 10.1021/ct300203w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct300203w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref146/cit146&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADC%252BC38XptFWjs7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&pages=2553-2558&author=D.+Shivakumarauthor=E.+Harderauthor=W.+Dammauthor=R.+A.+Friesnerauthor=W.+Sherman&title=Improving+the+Prediction+of+Absolute+Solvation+Free+Energies+Using+the+Next+Generation+OPLS+Force+Field&doi=10.1021%2Fct300203w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit146R"><div class="casContent"><span class="casTitleNuber">146</span><div class="casTitle"><span class="NLM_cas:atitle">Improving the Prediction of Absolute Solvation Free Energies Using the Next Generation OPLS Force Field</span></div><div class="casAuthors">Shivakumar, Devleena; Harder, Edward; Damm, Wolfgang; Friesner, Richard A.; Sherman, Woody</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2553-2558</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Explicit solvent mol. dynamics free energy perturbation simulations were performed to predict abs. solvation free energies of 239 diverse small mols.  We use OPLS2.0, the next generation OPLS force field, and compare the results with popular small mol. force fields-OPLS_2005, GAFF, and CHARMm-MSI.  OPLS2.0 produces the best correlation with exptl. data (R2 = 0.95, slope = 0.96) and the lowest av. unsigned errors (0.7 kcal/mol).  Important classes of compds. that performed suboptimally with OPLS_2005 show significant improvements.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqEngE5xV9Ev7Vg90H21EOLACvtfcHk0lgSoIrEECTz3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XptFWjs7g%253D&md5=eceb7dab3a8d0c6ac586c620848412d1</span></div><a href="/servlet/linkout?suffix=cit146&amp;dbid=16384&amp;doi=10.1021%2Fct300203w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct300203w%26sid%3Dliteratum%253Aachs%26aulast%3DShivakumar%26aufirst%3DD.%26aulast%3DHarder%26aufirst%3DE.%26aulast%3DDamm%26aufirst%3DW.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DSherman%26aufirst%3DW.%26atitle%3DImproving%2520the%2520Prediction%2520of%2520Absolute%2520Solvation%2520Free%2520Energies%2520Using%2520the%2520Next%2520Generation%2520OPLS%2520Force%2520Field%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2012%26volume%3D8%26spage%3D2553%26epage%3D2558%26doi%3D10.1021%2Fct300203w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref147"><div class="reference"><strong class="refLabel"><a href="#ref147" class="rightTabRefNumLink">147</a></strong><div class="NLM_citation" id="rightTab-cit147"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hoover, W. G.</span></span> <span> </span><span class="NLM_article-title">Canonical Dynamics: Equilibrium Phase-Space Distributions</span>. <i>Phys. Rev. A</i> <span class="NLM_year" style="font-weight: bold;">1985</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">1695</span>– <span class="NLM_lpage">1697</span>, <span class="refDoi"> DOI: 10.1103/PhysRevA.31.1695</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref147/cit147&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1103%2FPhysRevA.31.1695" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref147/cit147&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=9895674" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref147/cit147&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A280%3ADC%252BC2sjotlWltA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=1985&pages=1695-1697&author=W.+G.+Hoover&title=Canonical+Dynamics%3A+Equilibrium+Phase-Space+Distributions&doi=10.1103%2FPhysRevA.31.1695"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit147R"><div class="casContent"><span class="casTitleNuber">147</span><div class="casTitle"><span class="NLM_cas:atitle">Canonical dynamics: Equilibrium phase-space distributions</span></div><div class="casAuthors">Hoover</div><div class="citationInfo"><span class="NLM_cas:title">Physical review. A, General physics</span>
        (<span class="NLM_cas:date">1985</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1695-1697</span>
        ISSN:<span class="NLM_cas:issn">0556-2791</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSFX8fOxXRWv36CEoQgWY10fW6udTcc2eZwfMVhU_JPjLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2sjotlWltA%253D%253D&md5=99a2477835b37592226a5d18a760685c</span></div><a href="/servlet/linkout?suffix=cit147&amp;dbid=16384&amp;doi=10.1103%2FPhysRevA.31.1695&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1103%252FPhysRevA.31.1695%26sid%3Dliteratum%253Aachs%26aulast%3DHoover%26aufirst%3DW.%2BG.%26atitle%3DCanonical%2520Dynamics%253A%2520Equilibrium%2520Phase-Space%2520Distributions%26jtitle%3DPhys.%2520Rev.%2520A%26date%3D1985%26volume%3D31%26spage%3D1695%26epage%3D1697%26doi%3D10.1103%2FPhysRevA.31.1695" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref148"><div class="reference"><strong class="refLabel"><a href="#ref148" class="rightTabRefNumLink">148</a></strong><div class="NLM_citation" id="rightTab-cit148"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nosé, S.</span></span> <span> </span><span class="NLM_article-title">A Unified Formulation of the Constant Temperature Molecular Dynamics Methods</span>. <i>J. Chem. Phys.</i> <span class="NLM_year" style="font-weight: bold;">1984</span>,  <span class="NLM_volume"><i>81</i></span>,  <span class="NLM_fpage">511</span>– <span class="NLM_lpage">519</span>, <span class="refDoi"> DOI: 10.1063/1.447334</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref148/cit148&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1063%2F1.447334" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref148/cit148&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADyaL2cXkvFOrs7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=81&publication_year=1984&pages=511-519&author=S.+Nos%C3%A9&title=A+Unified+Formulation+of+the+Constant+Temperature+Molecular+Dynamics+Methods&doi=10.1063%2F1.447334"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit148R"><div class="casContent"><span class="casTitleNuber">148</span><div class="casTitle"><span class="NLM_cas:atitle">A unified formulation of the constant-temperature molecular-dynamics methods</span></div><div class="casAuthors">Nose, Shuichi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1984</span>),
    <span class="NLM_cas:volume">81</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">511-19</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    </div><div class="casAbstract">Three recently proposed const. temp. mol. dynamics methods [N., (1984) (1); W. G. Hoover et al., (1982) (2); D. J. Evans and G. P. Morris, (1983) (2); and J. M. Haile and S. Gupta, 1983) (3)] are examd. anal. via calcg. the equil. distribution functions and comparing them with that of the canonical ensemble.  Except for effects due to momentum and angular momentum conservation, method (1) yields the rigorous canonical distribution in both momentum and coordinate space.  Method (2) can be made rigorous in coordinate space, and can be derived from method (1) by imposing a specific constraint.  Method (3) is not rigorous and gives a deviation of order N-1/2 from the canonical distribution (N the no. of particles).  The results for the const. temp.-const. pressure ensemble are similar to the canonical ensemble case.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrsbAHxmqLp77Vg90H21EOLACvtfcHk0lgSoIrEECTz3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL2cXkvFOrs7k%253D&md5=2974515ec89e5601868e35871c0f19c2</span></div><a href="/servlet/linkout?suffix=cit148&amp;dbid=16384&amp;doi=10.1063%2F1.447334&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.447334%26sid%3Dliteratum%253Aachs%26aulast%3DNos%25C3%25A9%26aufirst%3DS.%26atitle%3DA%2520Unified%2520Formulation%2520of%2520the%2520Constant%2520Temperature%2520Molecular%2520Dynamics%2520Methods%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1984%26volume%3D81%26spage%3D511%26epage%3D519%26doi%3D10.1063%2F1.447334" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref149"><div class="reference"><strong class="refLabel"><a href="#ref149" class="rightTabRefNumLink">149</a></strong><div class="NLM_citation" id="rightTab-cit149"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Martyna, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tobias, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, M. L.</span></span> <span> </span><span class="NLM_article-title">Constant Pressure Molecular Dynamics Algorithms</span>. <i>J. Chem. Phys.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>101</i></span>,  <span class="NLM_fpage">4177</span>– <span class="NLM_lpage">4189</span>, <span class="refDoi"> DOI: 10.1063/1.467468</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref149/cit149&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1063%2F1.467468" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref149/cit149&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADyaK2cXmtFeht7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=101&publication_year=1994&pages=4177-4189&author=G.+J.+Martynaauthor=D.+J.+Tobiasauthor=M.+L.+Klein&title=Constant+Pressure+Molecular+Dynamics+Algorithms&doi=10.1063%2F1.467468"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit149R"><div class="casContent"><span class="casTitleNuber">149</span><div class="casTitle"><span class="NLM_cas:atitle">Constant pressure molecular dynamics algorithms</span></div><div class="casAuthors">Martyna, Glenn J.; Tobias, Douglas J.; Klein, Michael L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">101</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">4177-89</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    </div><div class="casAbstract">Modularly invariant equations of motion are derived that generate the isothermal-isobaric ensemble as their phase space avs.  Isotropic vol. fluctuations and fully flexible simulation cells as well as a hybrid scheme that naturally combines the two motions are considered.  The resulting methods are tested on two problems, a particle in a one-dimensional periodic potential and a spherical model of C60 in the solid/fluid phase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3YN7wMRJpjrVg90H21EOLACvtfcHk0lgSoIrEECTz3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXmtFeht7o%253D&md5=c14bd79c6398b0b30541e3cbe92851b0</span></div><a href="/servlet/linkout?suffix=cit149&amp;dbid=16384&amp;doi=10.1063%2F1.467468&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.467468%26sid%3Dliteratum%253Aachs%26aulast%3DMartyna%26aufirst%3DG.%2BJ.%26aulast%3DTobias%26aufirst%3DD.%2BJ.%26aulast%3DKlein%26aufirst%3DM.%2BL.%26atitle%3DConstant%2520Pressure%2520Molecular%2520Dynamics%2520Algorithms%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1994%26volume%3D101%26spage%3D4177%26epage%3D4189%26doi%3D10.1063%2F1.467468" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref150"><div class="reference"><strong class="refLabel"><a href="#ref150" class="rightTabRefNumLink">150</a></strong><div class="NLM_citation" id="rightTab-cit150"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Darden, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">York, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pedersen, L.</span></span> <span> </span><span class="NLM_article-title">Particle Mesh Ewald: An N·log(N) Method for Ewald Sums in Large Systems</span>. <i>J. Chem. Phys.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>98</i></span>,  <span class="NLM_fpage">10089</span>– <span class="NLM_lpage">10092</span>, <span class="refDoi"> DOI: 10.1063/1.464397</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref150/cit150&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1063%2F1.464397" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref150/cit150&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADyaK3sXks1Ohsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=1993&pages=10089-10092&author=T.+Dardenauthor=D.+Yorkauthor=L.+Pedersen&title=Particle+Mesh+Ewald%3A+An+N%C2%B7log%28N%29+Method+for+Ewald+Sums+in+Large+Systems&doi=10.1063%2F1.464397"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit150R"><div class="casContent"><span class="casTitleNuber">150</span><div class="casTitle"><span class="NLM_cas:atitle">Particle mesh Ewald: an N·log(N) method for Ewald sums in large systems</span></div><div class="casAuthors">Darden, Tom; York, Darrin; Pedersen, Lee</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">10089-92</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    </div><div class="casAbstract">An N·log(N) method for evaluating electrostatic energies and forces of large periodic systems is presented.  The method is based on interpolation of the reciprocal space Ewald sums and evaluation of the resulting convolution using fast Fourier transforms.  Timings and accuracies are presented for three large cryst. ionic systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDIJavf7AZPbVg90H21EOLACvtfcHk0liWHMoFuvh-Og"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXks1Ohsr0%253D&md5=3c9f230bd01b7b714fd096d4d2e755f6</span></div><a href="/servlet/linkout?suffix=cit150&amp;dbid=16384&amp;doi=10.1063%2F1.464397&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.464397%26sid%3Dliteratum%253Aachs%26aulast%3DDarden%26aufirst%3DT.%26aulast%3DYork%26aufirst%3DD.%26aulast%3DPedersen%26aufirst%3DL.%26atitle%3DParticle%2520Mesh%2520Ewald%253A%2520An%2520N%25C2%25B7log%2528N%2529%2520Method%2520for%2520Ewald%2520Sums%2520in%2520Large%2520Systems%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1993%26volume%3D98%26spage%3D10089%26epage%3D10092%26doi%3D10.1063%2F1.464397" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref151"><div class="reference"><strong class="refLabel"><a href="#ref151" class="rightTabRefNumLink">151</a></strong><div class="NLM_citation" id="rightTab-cit151"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ryckaert, J.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciccotti, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berendsen, H. J. C.</span></span> <span> </span><span class="NLM_article-title">Numerical Integration of the Cartesian Equations of Motion of a System with Constraints: Molecular Dynamics of n-Alkanes</span>. <i>J. Comput. Phys.</i> <span class="NLM_year" style="font-weight: bold;">1977</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">327</span>– <span class="NLM_lpage">341</span>, <span class="refDoi"> DOI: 10.1016/0021-9991(77)90098-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref151/cit151&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=10.1016%2F0021-9991%2877%2990098-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref151/cit151&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;key=1%3ACAS%3A528%3ADyaE2sXktVGhsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=1977&pages=327-341&author=J.-P.+Ryckaertauthor=G.+Ciccottiauthor=H.+J.+C.+Berendsen&title=Numerical+Integration+of+the+Cartesian+Equations+of+Motion+of+a+System+with+Constraints%3A+Molecular+Dynamics+of+n-Alkanes&doi=10.1016%2F0021-9991%2877%2990098-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit151R"><div class="casContent"><span class="casTitleNuber">151</span><div class="casTitle"><span class="NLM_cas:atitle">Numerical integration of the Cartesian equations of motion of a system with constraints:  molecular dynamics of n-alkanes</span></div><div class="casAuthors">Ryckaert, Jean Paul; Ciccotti, Giovanni; Berendsen, Herman J. C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Physics</span>
        (<span class="NLM_cas:date">1977</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">327-41</span>CODEN:
                <span class="NLM_cas:coden">JCTPAH</span>;
        ISSN:<span class="NLM_cas:issn">0021-9991</span>.
    </div><div class="casAbstract">A numerical algorithm integrating the 3N Cartesian equation of motion of a system of N points subject to holonomic constraints is applied to mol. dynamics simulation of a liq. of 64 butane mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYPlhG1U3Uw7Vg90H21EOLACvtfcHk0liWHMoFuvh-Og"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE2sXktVGhsL4%253D&md5=b4aecddfde149117813a5ea4f5353ce2</span></div><a href="/servlet/linkout?suffix=cit151&amp;dbid=16384&amp;doi=10.1016%2F0021-9991%2877%2990098-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0021-9991%252877%252990098-5%26sid%3Dliteratum%253Aachs%26aulast%3DRyckaert%26aufirst%3DJ.-P.%26aulast%3DCiccotti%26aufirst%3DG.%26aulast%3DBerendsen%26aufirst%3DH.%2BJ.%2BC.%26atitle%3DNumerical%2520Integration%2520of%2520the%2520Cartesian%2520Equations%2520of%2520Motion%2520of%2520a%2520System%2520with%2520Constraints%253A%2520Molecular%2520Dynamics%2520of%2520n-Alkanes%26jtitle%3DJ.%2520Comput.%2520Phys.%26date%3D1977%26volume%3D23%26spage%3D327%26epage%3D341%26doi%3D10.1016%2F0021-9991%2877%2990098-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5VAI" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5VAI','PDB','5VAI'); return false;">PDB: 5VAI</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LRJ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LRJ','PDB','5LRJ'); return false;">PDB: 5LRJ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6SIS" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6SIS','PDB','6SIS'); return false;">PDB: 6SIS</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5T35" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5T35','PDB','5T35'); return false;">PDB: 5T35</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2AGH" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2AGH','PDB','2AGH'); return false;">PDB: 2AGH</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-2"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i37"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01500">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_11684"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.0c01500?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01500</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Analytical data with RP-HPLC retention times and molecular masses of the synthesized peptides; IUPAC names of the synthesized peptides; detailed synthesis procedures and analytical data for TAFIa inhibitors; two additional retrospective case studies: a peptide example (hydrocarbon-stapled peptides targeting β-catenin) and a small molecule example (macrocyclic factor XIa inhibitors) (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01500/suppl_file/jm0c01500_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01500/suppl_file/jm0c01500_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01500/suppl_file/jm0c01500_si_001.pdf">jm0c01500_si_001.pdf (525.83 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01500/suppl_file/jm0c01500_si_002.csv">jm0c01500_si_002.csv (10.6 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c01500&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2020.63.issue-20%3BpageGroup%3Astring%3APublication+Pages%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c01500%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c01500" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                6MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"67992742daf524cb","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
